
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Changes in cellular composition shape the inductive properties of Hensen’s Node - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152538178AF1D9A3053817003D6A6F00.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371044/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Changes in cellular composition shape the inductive properties of Hensen’s Node">
<meta name="citation_author" content="Tatiane Y Kanno">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author_institution" content="Department of Pathology, Boston Children’s Hospital, Boston, MA USA">
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY USA">
<meta name="citation_author" content="Megan Rothstein">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author_institution" content="Department of Pathology, Boston Children’s Hospital, Boston, MA USA">
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY USA">
<meta name="citation_author" content="Marcos Simoes-Costa">
<meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, MA USA">
<meta name="citation_author_institution" content="Department of Pathology, Boston Children’s Hospital, Boston, MA USA">
<meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY USA">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7824">
<meta name="citation_doi" content="10.1038/s41467-025-63154-2">
<meta name="citation_pmid" content="40841542">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371044/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371044/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371044/pdf/41467_2025_Article_63154.pdf">
<meta name="description" content="The establishment of the vertebrate body plan is orchestrated by the organizer, a specialized group of cells with inductive properties that guide axial specification during early development. In avian embryos, organizer cells reside within Hensen’s ...">
<meta name="og:title" content="Changes in cellular composition shape the inductive properties of Hensen’s Node">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The establishment of the vertebrate body plan is orchestrated by the organizer, a specialized group of cells with inductive properties that guide axial specification during early development. In avian embryos, organizer cells reside within Hensen’s ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371044/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371044">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63154-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63154.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371044%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371044/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371044/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371044/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 22;16:7824. doi: <a href="https://doi.org/10.1038/s41467-025-63154-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63154-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Changes in cellular composition shape the inductive properties of Hensen’s Node</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kanno%20TY%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Tatiane Y Kanno</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Tatiane Y Kanno</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">
<sup>2</sup>Department of Pathology, Boston Children’s Hospital, Boston, MA USA </div>
<div class="p">
<sup>3</sup>Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kanno%20TY%22%5BAuthor%5D" class="usa-link"><span class="name western">Tatiane Y Kanno</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rothstein%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Megan Rothstein</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Megan Rothstein</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">
<sup>2</sup>Department of Pathology, Boston Children’s Hospital, Boston, MA USA </div>
<div class="p">
<sup>3</sup>Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rothstein%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Megan Rothstein</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Simoes-Costa%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Marcos Simoes-Costa</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Marcos Simoes-Costa</span></h3>
<div class="p">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div class="p">
<sup>2</sup>Department of Pathology, Boston Children’s Hospital, Boston, MA USA </div>
<div class="p">
<sup>3</sup>Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Simoes-Costa%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marcos Simoes-Costa</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Systems Biology, Harvard Medical School, Boston, MA USA </div>
<div id="Aff2">
<sup>2</sup>Department of Pathology, Boston Children’s Hospital, Boston, MA USA </div>
<div id="Aff3">
<sup>3</sup>Department of Molecular Biology and Genetics, Cornell University, Ithaca, NY USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Sep 24; Accepted 2025 Aug 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371044  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841542/" class="usa-link">40841542</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The establishment of the vertebrate body plan is orchestrated by the organizer, a specialized group of cells with inductive properties that guide axial specification during early development. In avian embryos, organizer cells reside within Hensen’s node, a transient structure located at the tip of the primitive streak. Despite its pivotal role during gastrulation, the cellular architecture of the Hensen’s node remains poorly understood. Here, we show that Hensen’s node is composed of two transcriptionally and functionally distinct organizer populations. In addition to anterior <em>GSC</em>-expressing cells associated with head induction, we identify a posterior population co-expressing organizer and mesodermal genes. These posterior cells exhibit trunk-inducing activity when transplanted into naïve tissue. Our findings reveal that the organizer is a dynamic and spatially compartmentalized structure, and that temporal changes in the relative abundance of anterior and posterior populations underlie shifts in its inductive capacity, ensuring coordinated patterning along the body axis.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Embryonic induction, Gastrulation</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">The organizer is a signaling center in the embryo that orchestrates the formation of body axes. Here they show that Hensen’s Node contains two main cell types and that changes in their abundance alter its ability to induce head versus trunk structures.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Two crucial steps in normal vertebrate development are the formation of the three germ layers and the specification of the body axes in the early embryo. Both processes are orchestrated by the organizer, a transient and dynamic structure in the gastrula formed by a group of cells with inductive properties<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. The organizer was first identified a century ago in classical transplantation experiments, which showed that this structure could induce a secondary axis when grafted into host embryos<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Organizer cells regulate various signaling pathways to instruct nearby cells to form neural and mesodermal derivatives, which are organized along the antero-posterior axis of the embryo<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. The organizer is highly conserved among vertebrates, and it can be identified by the expression of a set of genes that include the homeobox transcription factor <em>GOOSECOID</em> (<em>GSC</em>)<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup> and the BMP-antagonist <em>CHORDIN</em> (<em>CHRD</em>)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Notably, the organizer function is dynamic, and its inductive properties have been shown to change over developmental time<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. However, the molecular mechanisms driving these changes in inductive properties remain poorly understood.</p>
<p id="Par4">In avian embryos, organizer cells reside within the Hensen’s node<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Characterized in rabbit and guinea pig embryos by Victor Hensen in 1876, the node is a crucial signaling center that also differentiates into various embryonic tissues<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Classic grafting experiments have demonstrated its powerful inductive capacity, showing that when transplanted into a host embryo, the Hensen’s node can induce the formation of a secondary body axis<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. The node forms at the anterior end of the primitive streak in the early gastrula<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> and subsequently moves posteriorly during neurulation<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Fate mapping studies have shown that node cells contribute to the central nervous system and the prechordal plate, as well as axial structures such as the notochord and somites<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Thus, the Hensen’s node plays a critical role in forming anterior structures and in the sequential development of posterior tissues as it regresses. Despite its importance, the cellular composition of the node and the genetic programs that define its distinct cell populations are still largely unexplored.</p>
<p id="Par5">In this work, we employed single-cell transcriptomics, high-resolution hybridization chain reaction fluorescence in situ hybridization (HCR-FISH), and experimental embryology to uncover the cellular composition and molecular identity of Hensen’s node. Our results reveal that the node is divided into two spatially distinct populations along the anterior-posterior axis, each defined by unique molecular signatures and inductive capacities. The anterior population, marked by <em>GSC</em> expression, is involved in the induction of cephalic structures<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. In contrast, the posterior population expresses mesodermal genes and is involved in trunk and caudal development. Both populations possess organizer activity, as shown by their ability to induce secondary axes when grafted into host embryos. Notably, the composition of the node is dynamic: during early gastrulation, the node is dominated by anterior organizer cells. In later stages, as the node regresses along the primitive streak, the emergence (and eventual predominance) of the posterior population of cells is observed. We propose that such cellular composition shifts underlie the node’s changing inductive properties, ensuring the coordinated formation of both anterior and posterior embryonic structures.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">The Hensen’s node is composed of two <em>CHRD</em>-positive cell populations</h3>
<p id="Par6">To characterize the cellular composition of the Hensen’s node, we microdissected the anterior portion of the primitive streak from gastrula stage chick embryos (Hamburger and Hamilton Stage 4; HH4, <em>n</em> = 38)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> and processed samples for single-cell RNA-seq (scRNA-seq) (Fig. <a href="#Fig1" class="usa-link">1a</a>). This resulted in a dataset composed of 8134 high-quality cells. Unsupervised clustering and uniform manifold approximation and projection (UMAP)<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> identified four cell clusters (Fig. <a href="#Fig1" class="usa-link">1b</a>) within our samples. To determine the identity of these cell populations, we mapped the expression of specific markers of each cluster (Fig. <a href="#Fig1" class="usa-link">1c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a> and Supplementary Data <a href="#MOESM4" class="usa-link">1</a>) with HCR-FISH<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. We found that the upper cluster in the UMAP (in green), characterized by high expression of epithelial genes such as <em>CDH11</em> (Fig. <a href="#Fig1" class="usa-link">1d–e</a>), <em>CLDN1</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>), and <em>SOX3</em> (Supplementary Data <a href="#MOESM4" class="usa-link">1</a>), contained the epiblast cells adjacent to the node. In contrast, the cluster occupying the lower part of the UMAP (in magenta) was composed of endodermal cells, identified by the expression of specific markers such as <em>SOX17</em> (Fig. <a href="#Fig1" class="usa-link">1f, g</a>). This was expected since the tissue samples we microdissected from gastrula-stage embryos also included the cells surrounding the node. Nevertheless, most of the cells in our dataset displayed robust <em>CHRD</em> expression and were thus part of the Hensen’s node (Fig. <a href="#Fig1" class="usa-link">1h, i</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Single-cell transcriptomic analysis reveals molecular heterogeneity in the Hensen’s node.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371044_41467_2025_63154_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5703/12371044/5d8f97220604/41467_2025_63154_Fig1_HTML.jpg" loading="lazy" id="d33e488" height="544" width="800" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic representation of the experimental design: 38 Hensen′s nodes were dissected from stage HH4 embryos and subjected to single-cell RNA-seq analysis. <strong>b</strong> UMAP visualization of all clusters from the dataset, colored by cell identity, depicting 8134 individual cells. <strong>c</strong> Dot plot showing the expression levels and percent expression of key marker genes for each cluster, indicating specific gene expression in each cell population. <strong>d</strong> UMAP displaying the expression of <em>CDH11</em> in the epiblast cluster. <strong>e</strong> HCR-FISH showing <em>CDH11</em> expression in the anterior epiblast around the Hensen′s node (<em>n</em> = 3/3 biological replicates with similar results). <strong>f</strong> UMAP displaying the expression of <em>SOX17</em> in the endoderm cluster. <strong>g</strong> HCR-FISH showing <em>SOX17</em> expression in the Hensen′s node and surrounding endoderm at HH4+ stage (<em>n</em> = 3/3 biological replicates with similar results). <strong>h</strong> UMAP displaying the expression of <em>CHRD</em> in the anterior and posterior node clusters. <strong>i</strong> HCR-FISH showing <em>CHRD</em> expression in the Hensen′s node at stage HH4 (arrow, <em>n</em> = 10/10 biological replicates with similar results). <strong>j</strong> UMAP displaying the expression of <em>GSC</em>, an organizer-specific gene, in the anterior node cluster. <strong>k</strong> HCR-FISH showing <em>GSC</em> expression in the Hensen’s node at stage HH4 (<em>n</em> = 10/10 biological replicates with similar results). <strong>l</strong> UMAP displaying the expression of <em>LMO1</em> in the posterior node cells. <strong>m</strong> HCR-FISH showing <em>LMO1</em> expression in the Hensen′s node at stage HH4 (<em>n</em> = 10/10 biological replicates with similar results). Scale bar = 200 μm.</p></figcaption></figure><p id="Par7"><em>CHRD-</em>positive cells could be divided into two clusters within the UMAP, corresponding to anterior and posterior cell populations within the node (Fig. <a href="#Fig1" class="usa-link">1b, c</a>). The anterior node cluster expressed high levels of transcription factor <em>GSC</em> (Fig. <a href="#Fig1" class="usa-link">1</a>c, j, k), a canonical marker of the vertebrate head organizer<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. <em>GSC</em> plays a role in regulating gastrulation movements and represses the expression of <em>WNT</em>, <em>BMP</em>, and genes encoding ventral fates<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. This cluster also expressed other anterior genes such as <em>CER1, ADMP</em>, and <em>OTX2</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>, <a href="#MOESM1" class="usa-link">b</a>). The second <em>CHRD-</em>positive population did not contain <em>GSC</em> transcripts and instead could be identified by the presence of <em>LMO1</em> (Fig. <a href="#Fig1" class="usa-link">1l</a>), which was expressed in the posterior portion of the node (Fig. <a href="#Fig1" class="usa-link">1m</a>). This group of cells also displayed high expression levels of transcription factors known to regulate the formation and development of mesoderm, such as <em>MSGN1</em> and <em>MESP1</em>, and mesenchymal markers, including <em>ZEB2</em> and <em>RND3</em> (Fig. <a href="#Fig1" class="usa-link">1c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>). Gene ontology analysis of genes enriched in this posterior cell cluster revealed significant enrichment for terms associated with mesodermal programs, including mesoderm development and morphogenesis (Supplementary Fig. <a href="#MOESM1" class="usa-link">1c</a>). These results suggest that the HH4 Hensen’s node is composed of two <em>CHRD-</em>positive cell populations with distinct molecular profiles.</p></section><section id="Sec4"><h3 class="pmc_sec_title">The cellular organization of the Hensen’s node</h3>
<p id="Par8">Next, we mapped the spatial distribution of the two <em>CHRD</em>-positive cell populations within Hensen’s node with multiplexed HCR-FISH, histological analysis, and image segmentation. Co-localization of <em>CHRD</em> (Fig. <a href="#Fig2" class="usa-link">2a</a>) and genes enriched in the anterior and posterior node clusters allowed us to define how these cell populations are organized within the node. Triple staining for <em>CHRD, GSC</em>, and <em>LMO1</em> confirmed that <em>CHRD</em>-positive node cells can be divided into anterior and posterior cell populations (Fig. <a href="#Fig2" class="usa-link">2a–d</a>). <em>GSC</em>-positive cells were positioned anteriorly (Fig. <a href="#Fig2" class="usa-link">2b</a>). In contrast, <em>LMO1</em>-positive cells were organized in two bilateral territories that surround the anterior portion of the primitive streak (Fig. <a href="#Fig2" class="usa-link">2c</a>). Thus, <em>GSC</em> and <em>LMO1</em> demarcate two spatial compartments within the node that are organized along the anterior-posterior axis (Fig. <a href="#Fig2" class="usa-link">2d–e</a>). To confirm this, we examined additional markers of the posterior node cell cluster (Fig. <a href="#Fig1" class="usa-link">1c</a>) with HCR-FISH. We found that <em>CXCL14</em>, <em>POMC</em>, and <em>RND3</em> were also enriched in the posterior portion of the node (Fig. <a href="#Fig2" class="usa-link">2f–h</a>). Multiplexed HCR confirmed co-expression of anterior (<em>GSC</em> and <em>ADMP</em>; Supplementary Fig. <a href="#MOESM1" class="usa-link">2a–b</a>) and posterior (<em>LMO1</em>, <em>RND3</em>, and <em>CXCL14</em>; Supplementary Fig. <a href="#MOESM1" class="usa-link">2c–f</a>) node markers in their respective populations. Other mesodermal genes, such as <em>MSGN1</em>, were expressed in the posterior node and also detected in the neighboring mesoderm and primitive streak (Supplementary Fig. <a href="#MOESM1" class="usa-link">2g, h</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Spatial segregation of organizer subpopulations in stage HH4 Hensen’s node.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371044_41467_2025_63154_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5703/12371044/653001f4f22c/41467_2025_63154_Fig2_HTML.jpg" loading="lazy" id="d33e792" height="825" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Whole-mount HCR-FISH showing <em>CHRD</em> expression throughout the Hensen’s node at stage HH4. <strong>b</strong>–<strong>d</strong> Anterior organizer cells marked by <em>GSC</em> are localized to the anterior region (<strong>b</strong>, cyan arrowhead), while posterior organizer markers such as <em>LMO1</em> (<strong>c</strong>, magenta arrowhead) are enriched in lateral and posterior regions. <strong>d</strong> Merged image showing spatial segregation of <em>GSC</em> (cyan) and <em>LMO1</em> (magenta). <strong>e</strong> Schematic summarizing the anterior-posterior compartmentalization of the node based on marker gene expression. The dotted line corresponds to the axial level of the transverse sections in (<strong>i</strong>, <strong>j</strong>, and <strong>k</strong>). <strong>f</strong>–<strong>h</strong> Additional posterior node markers (<em>CXCL14</em>, <em>POMC</em>, and <em>RND3</em>) are also enriched in lateral and posterior domains of the node (magenta arrowheads, <em>n</em> = 5/5 biological replicates with similar results). <strong>i</strong>, <strong>j</strong> Transverse section showing <em>CHRD</em> (<strong>i</strong>) and <em>GSC</em> (<strong>j</strong>) expression at stage HH4 (<em>n</em> = 3/3 biological replicates with similar results). <strong>k</strong> Segmentation analysis of transverse sections showing <em>GSC</em>-positive anterior organizer cells positioned ventrally and <em>RND3</em>-positive posterior organizer cells enriched in dorsal-lateral regions. <strong>l</strong> Schematic depicting the distribution of anterior (cyan) and posterior (magenta) node populations along the dorsal–ventral axis. D, dorsal; V, ventral. <strong>a</strong>–<strong>d</strong> (<em>n</em> = 3/3 biological replicates with similar results). Scale bar = 100 μm.</p></figcaption></figure><p id="Par9">We next determined how the anterior and posterior node populations were organized along the dorsal-ventral axis of the embryo. To accomplish this, we performed multiplexed HCR-FISH for node genes on transverse cross-sections of gastrula-stage embryos. Analysis of the expressions of <em>CHRD</em> (pan-node marker), <em>GSC</em> (anterior node marker), and <em>RND3</em> (posterior node marker) indicated that the anterior and posterior cell populations also occupied distinct dorsoventral positions within the stage HH4 node (Fig. <a href="#Fig2" class="usa-link">2i, j</a>). To confirm this, we analyzed these samples with CellProfiler image segmentation<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a> and Supplementary Data <a href="#MOESM8" class="usa-link">5</a>), which allowed us to computationally isolate individual <em>CHRD</em>-positive cells (Fig. <a href="#Fig2" class="usa-link">2i</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>) and compare the expression of <em>GSC</em> (Fig. <a href="#Fig2" class="usa-link">2j</a>) and <em>RND3</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>) within the node. This analysis revealed that <em>GSC</em>-positive cells are positioned in the ventral portion of the node, while most <em>RND3</em>-positive cells occupy a dorsolateral position within the structure adjacent to epiblast cells (Fig. <a href="#Fig2" class="usa-link">2k</a>). While the two populations display distinct spatial biases, we also observe some intermingling of cell types at the boundaries of their territories (Fig. <a href="#Fig2" class="usa-link">2k</a>), suggesting a gradual transition between anterior and posterior node domains. Similar to this, we also detect the co-expression of markers such as <em>GSC</em> and <em>SOX17</em> in cells near the anterior node, consistent with transitional states as some organizer-derived cells begin to adopt mesendodermal fates (Supplementary Fig. <a href="#MOESM1" class="usa-link">2i–n</a>).</p>
<p id="Par10">We confirmed this dorsoventral distribution of anterior/posterior node cells by examining embryos co-stained with <em>GSC</em> and <em>TBXT</em>, which in avian embryos is expressed in both node populations<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a, b</a>). Confocal microscopy revealed that <em>GSC-</em>positive cells could only be detected at deeper dorsoventral Z-sections within the node (Supplementary Fig. <a href="#MOESM1" class="usa-link">4c–i</a>), consistent with our image segmentation results (Fig. <a href="#Fig2" class="usa-link">2k</a>). Together, these results allowed us to map the cellular organization of the Hensen’s node at the mid-gastrula (Fig. <a href="#Fig2" class="usa-link">2l</a>). We found that the anterior and posterior node subpopulations are spatially segregated along the anterior-posterior and dorsal-ventral axes (Fig. <a href="#Fig2" class="usa-link">2</a>e, <a href="#Fig2" class="usa-link">l</a>). At this developmental stage, anterior node cells are positioned at the center of the node. In contrast, posterior node cells occupy a more lateral position before and after ingression into the blastocoel (Fig. <a href="#Fig2" class="usa-link">2</a>e, <a href="#Fig2" class="usa-link">l</a>). RNA velocity analysis<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> of stage HH4 dataset revealed early transcriptional divergence between anterior and posterior node populations, with vector fields indicating distinct developmental trajectories emerging within the node region (Supplementary Fig. <a href="#MOESM1" class="usa-link">4j</a>). This analysis also supports a gradual transition from a subset of anterior node cells toward the endodermal cluster, suggesting a developmental trajectory linking these organizer cells to the mesendodermal lineage, consistent with their known fate contribution<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Together, the results show that by stage HH4, anterior and posterior node cells are already following separate transcriptional paths that may underlie their distinct behaviors and inductive properties.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Temporal segregation of node cell populations</h3>
<p id="Par11">As development proceeds, the Hensen’s node regresses toward the posterior end of the embryo<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. This event marks the transition from gastrula to neurula and is crucial for notochord formation and axial elongation<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. To define how the cellular composition of the node changes upon regression, we conducted scRNA-seq analysis of dissected stage HH6 nodes (Fig. <a href="#Fig3" class="usa-link">3a</a>, <em>n</em> = 20). Hensen’s nodes from neurula stages were dissected and dissociated for scRNA-seq, which resulted in a dataset composed of 12,753 high-quality cells. This dataset contained the posterior node populations and neighboring cells from the notochord, neural tissue, endoderm, and presomitic mesoderm (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>, b, <a href="#MOESM1" class="usa-link">d</a> and Supplementary Data <a href="#MOESM5" class="usa-link">2</a>). To identify the node cells, we isolated and re-clustered the <em>CHRD</em>-positive cells in our dataset (Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>, <a href="#MOESM1" class="usa-link">e</a>). Since differentiating notochord cells also express <em>CHRD</em>, we examined the expression of the notochord marker <em>NOTO</em> in the resulting UMAP (Supplementary Fig. <a href="#MOESM1" class="usa-link">5f</a>). This allowed us to distinguish notochord cells from the regressing Hensen’s node by identifying the <em>CHRD</em>-positive and <em>NOTO</em>-negative (Fig. <a href="#Fig3" class="usa-link">3b</a>). Differential analysis of gene expression revealed that these cells retain the molecular signature of stage HH4 posterior node, including <em>CXCL14, MSGN1, RND3</em>, and <em>POMC</em> (Fig. <a href="#Fig3" class="usa-link">3b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5d</a>, <a href="#MOESM1" class="usa-link">g–j</a>). HCR-FISH revealed that these cells remain positioned at the posterior-lateral part of the node (Fig. <a href="#Fig3" class="usa-link">3c, d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5k, n</a>). These results show that the regressing node cells retain the molecular program of the stage HH4 posterior node (Fig. <a href="#Fig3" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Temporal segregation of Hensen’s node cell populations.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371044_41467_2025_63154_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5703/12371044/600b056bfd11/41467_2025_63154_Fig3_HTML.jpg" loading="lazy" id="d33e1125" height="840" width="696" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic representation of the experimental design for single-cell RNA-seq, performed on 20 Hensen’s nodes at stage HH6. <strong>b</strong> UMAP plot showing the expression of <em>CXCL14</em> and <em>NOTO</em> within <em>CHRD</em>-positive node cells. <strong>c</strong>, <strong>d</strong> HCR-FISH showing <em>CXCL14</em> expression in the posterior node (node is outlined by a dotted line). White arrows highlight the localization of <em>CXCL14/CHRD</em>-positive cells in the posterior region of the node (<em>n</em> = 5/5 biological replicates with similar results). <strong>e</strong>–<strong>g </strong>UMAP plots displaying <em>GSC</em> and <em>LMO1</em> expression in <em>CHRD</em>-positive cells from nodes at stages HH3 (<strong>e</strong>), HH4 (<strong>f</strong>), and HH6 (<strong>g</strong>). <em>GSC</em>-positive cells mark the anterior node present on HH3-HH4 stages, while <em>LMO1</em>-positive posterior cells emerge in the node by stage HH4 and remain at stage HH6. <strong>h</strong>, <strong>i</strong> HCR-FISH for <em>CHRD</em>, <em>GSC</em>, and <em>RND3</em> at stage HH5 (<em>n</em> = 3/3 biological replicates with similar results). <em>CHRD</em> marks the entire Hensen’s node, <em>GSC</em> labels the anterior cells (cyan arrows), and <em>RND3</em> is predominantly expressed in the lateral regions of the regressing node (magenta arrows). <strong>j</strong>–<strong>l</strong> Schematic depicting the spatial segregation of the anterior node (AN) and posterior (PN) node populations along the anterior-posterior axis at stages HH3 (<strong>j</strong>), HH4 (<strong>k</strong>), and HH6 (<strong>l</strong>). Scale bar = 100 μm.</p></figcaption></figure><p id="Par12">Next, we utilized scRNA-seq to investigate the temporal segregation of anterior and posterior node populations. To complement our stages HH4 and HH6 datasets, we analyzed Hensen’s nodes dissected from early gastrula-stage embryos (HH3, Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>, <em>n</em> = 20) when the primitive streak is still elongating and the node is just starting to form<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Analysis of the molecular signatures of cell clusters in this dataset allowed us to identify the distinct cell types present at this developmental stage (Supplementary Fig. <a href="#MOESM1" class="usa-link">6b</a>, <a href="#MOESM1" class="usa-link">d</a> and Supplementary Data <a href="#MOESM6" class="usa-link">3</a>). To examine how the composition of the node changes during these developmental stages, we projected the molecular signatures of the anterior (<em>GSC</em> and <em>ADMP</em>) and posterior (<em>LMO1, CXCL14, MSGN1, POMC, and MESP1</em>) organizer cells in the UMAPs of <em>CHRD</em>-positive cells of the three developmental stages (Fig. <a href="#Fig3" class="usa-link">3e–g</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">5g–j</a>, <a href="#MOESM1" class="usa-link">6e–m</a>). At stage HH3, we could only detect the anterior signature within the <em>CHRD</em>-positive cells (Fig. <a href="#Fig3" class="usa-link">3e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">6f–j</a>), the earliest identifiable node population in our dataset. Posterior node cells were first detected at stage HH4 when both anterior and posterior cell populations co-exist within the structure (Fig. <a href="#Fig3" class="usa-link">3f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">6k–m</a>). Finally, at stage HH6, the anterior node signature is no longer detectable, indicating that only posterior cells remain in the regressing node (Fig. <a href="#Fig3" class="usa-link">3g</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">5d</a>, <a href="#MOESM1" class="usa-link">m, n</a>). Indeed, HCR-FISH analysis at stage HH5 shows that <em>GSC</em>-positive cells begin to leave the node and migrate toward the cranial region of the embryo (Fig. <a href="#Fig3" class="usa-link">3h, i</a>). Previous fate mapping analysis shows that these cells will contribute to derivatives such as the prechordal plate and ventral foregut endoderm<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. These findings indicate that the node is initially established as an anterior structure (Fig. <a href="#Fig3" class="usa-link">3j</a>) and that its cellular composition changes to include posterior cells once the primitive streak is fully extended (Fig. <a href="#Fig3" class="usa-link">3k</a>). Subsequently, posterior cells dominate the node as it regresses along the primitive streak (Fig. <a href="#Fig3" class="usa-link">3l</a>). These results illustrate how the cellular composition of the node changes during development and suggest that its inductive properties may also change accordingly.</p>
<p id="Par13">To reconstruct the developmental trajectory of organizer cells, we computationally isolated <em>CHRD</em>-positive cells from our scRNA-seq datasets spanning stages HH3, HH4, and HH6 (Fig. <a href="#Fig4" class="usa-link">4a</a>). Dimensionality reduction and re-clustering of this subset revealed six transcriptionally distinct populations corresponding to epiblast (EPI), anterior node (AN), intermediate node (IN), posterior node (PN), endoderm (EN), and notochord (NC) (Fig. <a href="#Fig4" class="usa-link">4b</a>). To infer potential lineage relationships among these populations, we performed both RNA velocity and Monocle3 pseudotime analysis. RNA velocity (Fig. <a href="#Fig4" class="usa-link">4c</a>) suggested directional transitions from epiblast into the anterior node, followed by divergence toward the posterior node, notochord, and endodermal fates. This general structure was supported by Monocle3 trajectory inference, which positioned the epiblast upstream of branching lineages corresponding to the same three terminal fates (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). Consistent with our HCR-FISH analyses, projection of developmental stage data onto the UMAP revealed that the anterior node cluster was primarily composed of stage HH3 and HH4 cells, while the posterior and intermediate node populations were largely made up of stage HH6 cells (Fig. <a href="#Fig4" class="usa-link">4d, e</a>), indicating a shift in cell identity over time. The developmental trajectories inferred from this analysis support a model in which early anterior node cells give rise to endodermal, posterior node cells, and the notochord (Fig. <a href="#Fig4" class="usa-link">4f</a>). This hierarchical organization positions epiblast cells upstream in the lineage and places posterior node derivatives at the terminus. These findings suggest that Hensen’s node is a temporally assembled and transcriptionally dynamic structure, where distinct organizer populations emerge in succession to coordinate sequential phases of axial induction.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Developmental trajectories of Hensen’s node cells across different stages.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371044_41467_2025_63154_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5703/12371044/0b078ff51a64/41467_2025_63154_Fig4_HTML.jpg" loading="lazy" id="d33e1283" height="395" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic representation of the Hensen’s node collected for scRNA-seq analysis at stages HH3, HH4, and HH6. <strong>b</strong> UMAP plot displaying 10,422 aggregated <em>CHRD</em>-positive cells from all three developmental stages (HH3, HH4, and HH6). <strong>c</strong> RNA velocity analysis showing the predicted lineage trajectories of <em>CHRD</em>-positive cells across developmental time. <strong>d</strong> UMAP plot showing the cell identities within the <em>CHRD</em>-positive population. <strong>e</strong> Stacked bar plot representing the proportion of each stage within the identified populations. <strong>f</strong> Proposed model of cell fate trajectory: Epiblast (EPI) cells give rise to the anterior node (AN) cells and contribute to notochord (NC). Anterior node cells further differentiate into endoderm (EN), notochord (NC), and posterior node (PN), which in turn give rise to paraxial mesoderm (PM).</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Changes in neural-inducing properties of Hensen’s node</h3>
<p id="Par14">Previous studies have demonstrated that grafting the Hensen’s node into gastrula-stage embryos can induce the formation of a secondary body axis<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. <em>GSC</em> plays a critical role in this process<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, as it directly activates the transcription of secreted factors involved in embryonic induction, such as <em>CHRD</em> and <em>NOG</em><sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. However, our findings indicate that <em>GSC</em> is absent from the <em>CHRD</em>-positive posterior cells in the node (Fig. <a href="#Fig2" class="usa-link">2</a>). We thus investigated whether posterior node cells exhibit organizer activity by performing transplantation experiments using transgenic chick embryos. As positive control, we first grafted entire stage HH4 Hensen’s nodes, which contain both anterior and posterior node regions, from GFP-positive donors into stage-matched wild-type hosts (Fig. <a href="#Fig5" class="usa-link">5a</a>). These grafts induced the formation of a secondary axis, including anterior neural tissues marked by <em>OTX2</em> expression (Fig. <a href="#Fig5" class="usa-link">5b, c</a>and Supplementary Fig. <a href="#MOESM1" class="usa-link">8a, b</a>). Next, we grafted stage HH6 Hensen’s nodes, which lack the anterior domain, into stage HH4 hosts. These posterior node grafts induced the formation of a secondary trunk, including a notochord and somites, but failed to generate anterior neural structures (Fig. <a href="#Fig5" class="usa-link">5d, e</a>), as confirmed by the absence of <em>OTX2</em> expression (Supplementary Fig. <a href="#MOESM1" class="usa-link">8c, d</a>). These findings, along with previous studies using chicken-quail chimeras<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, demonstrate that posterior node cells retain organizer activity and can induce neural identity, but do not promote anterior neural fate.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Induction of posterior neural identity by posterior organizer cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371044_41467_2025_63154_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5703/12371044/9083a58fce87/41467_2025_63154_Fig5_HTML.jpg" loading="lazy" id="d33e1437" height="791" width="700" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic of the grafting experiment: the Hensen′s node from a GFP + donor embryo at stage HH4 or HH6 was transplanted into the marginal zone of a wild-type stage HH4 host. <strong>b</strong>, <strong>c</strong> Whole-mount immunostaining for <em>OTX2</em> (magenta) shows that stage HH4 node grafts induce a secondary axis (arrowhead) that contains anterior neural tissue (pink arrow, <em>n</em> = 5/5 biological replicates with similar results). <strong>d</strong>, <strong>e</strong> In contrast, HH6 node grafts induce posterior axial structures (arrowhead), including <em>CHRD</em>-expressing notochord (magenta arrow, <em>n </em>= 5/5 biological replicates with similar results). <strong>f</strong> Schematic of enhancer reporter experiment: stage HH4 or HH6 donor nodes were grafted into transgenic stage HH4 host embryos, electroporated with <em>SOX2</em>N1::GFP (N1) and <em>SOX2</em>N2::mCherry (N2) reporter constructs. <strong>g</strong>, <strong>h</strong> Representative images showing differential enhancer activation following grafting of stage HH4 (<strong>g</strong>, <em>n</em> = 12/16 biological replicates with similar results) or HH6 (<strong>h</strong>, <em>n</em> = 11/19 biological replicates with similar results) node tissue. <strong>i</strong> Box plots showing the distribution of N1-GFP+ and N2-mCherry+ cells per embryo after transplantation of nodes at stages HH4 or HH6. Each dot represents a single embryo (biological replicate); <em>n </em>= 12 embryos (HH4), <em>n</em> = 11 embryos (HH6). Boxes indicate the median (center line), 25th and 75th percentiles (box), and whiskers denote values within 1.5 × interquartile range. A two-way binomial generalized linear model (GLM) was used to assess the effect of stage on marker expression. For GFP+ cells, the stage effect estimate (HH6 vs HH4) was − 1.203 (z = − 9.78, <em>p</em> = 1.5 × 10<sup>-22</sup>, 95% CI: − 1.444 to − 0.962). For mCherry+ cells, the estimate was + 1.203 (<em>z</em> = 9.78, <em>p</em> = 1.5 ×  10<sup>-22</sup>, 95% CI: 0.962 to 1.444). Source data is provided as a Source Data file.</p></figcaption></figure><p id="Par15">To test whether distinct node populations not only orchestrate neural induction but also contribute to the spatial patterning of neural tissue, we used enhancer-reporter constructs derived from the <em>SOX2</em> locus. The <em>SOX2</em>N2 and <em>SOX2</em>N1 enhancers are active in anterior and posterior regions of the neural plate, respectively, and can serve as readouts for spatial identity during neural induction<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Specifically, <em>SOX2</em>N2 enhancer is active in the anterior neural plate, encompassing prospective forebrain and midbrain regions, while <em>SOX2</em>N1 enhancer is active in the posterior neural plate, marking areas fated to become the spinal cord. We generated stage HH4 transgenic hosts carrying <em>SOX2</em>N1::GFP (N1-GFP) and <em>SOX2</em>N2::mCherry (N2-mChe) constructs and grafted wild-type HH4 or HH6 nodes into these embryos (Fig. <a href="#Fig5" class="usa-link">5f</a>). Notably, because only host epiblast cells carry the enhancer reporters, these experiments allow direct visualization of neural induction in the host tissue. Grafts of stage HH4 nodes led to robust activation of the <em>SOX2</em>N2 in the secondary axis (Fig. <a href="#Fig5" class="usa-link">5g</a>and Supplementary Fig. <a href="#MOESM1" class="usa-link">8e–h</a>), consistent with the induction of anterior neural identities. Imaging of the secondary axis showed spatial separation between <em>SOX2</em>N2 expression in anterior regions and some <em>SOX2</em>N1 expression in more caudal regions of the induced axis (Supplementary Fig. <a href="#MOESM1" class="usa-link">8e, f</a>). In contrast, stage HH6 node grafts predominantly activated the <em>SOX2</em>N1 enhancer (Fig. <a href="#Fig5" class="usa-link">5h</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">8g, h</a>), suggesting that posterior node cells preferentially induce neural fates with posterior character. Indeed, quantification of enhancer activation across multiple embryos revealed that stage HH4 grafts (<em>n</em> = 12, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">8f</a>) consistently induced both enhancers, whereas stage HH6 grafts (<em>n</em> = 11, Supplementary Table <a href="#MOESM1" class="usa-link">1</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">8h</a>) led almost exclusively to <em>SOX2</em>N1 activation (Fig. <a href="#Fig5" class="usa-link">5i</a>). These results indicate that anterior and posterior node populations differ in their ability to induce and pattern neural tissue, with posterior node cells biased toward the caudal neural fates.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Induction of caudal mesoderm by posterior node cells</h3>
<p id="Par16">Next, we determined the cohort of cell types induced by the posterior node by performing scRNA-seq on secondary axes. In this experiment, we inverted our transplantation scheme and grafted wild-type stage HH6 nodes into GFP-positive stage HH4 host embryos (Fig. <a href="#Fig6" class="usa-link">6a</a>). This approach enabled us to confidently identify induced cells even with lower-depth single-cell transcriptomes, as the presence of GFP transcripts ensured they originated from the host. After a 16 h incubation, we dissected secondary axes from five embryos, dissociated the tissue, and profiled a total of 7750 cells. These were grouped into 16 clusters (Supplementary Fig. <a href="#MOESM1" class="usa-link">9a</a>), including mesodermal, neural, ectodermal, and extra-embryonic populations. To examine the diversity of host contributions, we isolated and re-clustered 2372 GFP-positive cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">9b, c</a>). This dataset revealed 14 distinct clusters corresponding to the cell types induced by the posterior node population. These included posterior mesoderm cells expressing notochord and presomitic mesoderm genes, as well as axial-specific markers like <em>HOXB5</em> and <em>CDX4</em>, indicating their posterior identity (Fig. <a href="#Fig6" class="usa-link">6b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">9d</a>and Supplementary Data <a href="#MOESM7" class="usa-link">4</a>). In addition, neural plate progenitors, non-neural ectoderm, and neural cells were identified among the induced populations, marked by the expression of genes such as <em>SFRP2</em>, <em>TFAP2A</em>, <em>SOX2</em>, and <em>PAX6</em> (Supplementary Data <a href="#MOESM7" class="usa-link">4</a>). Non-induced cell types from the surrounding graft region, such as blood cells, endodermal cells, and extra-embryonic tissue, were also present in the dataset (Supplementary Fig. <a href="#MOESM1" class="usa-link">9e–g</a>). To examine the extent of host cell induction, we focused on the subset of 3453 cells with ectodermal or mesodermal identity from the initial clustering analysis (Supplementary Fig. <a href="#MOESM1" class="usa-link">9a</a>). This analysis showed that 1188 cells (34%) from this group were GFP-positive, indicating that a substantial portion of the secondary axis was derived from induced host cells.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Posterior node cells can induce caudal mesoderm.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371044_41467_2025_63154_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5703/12371044/1880d6814ca0/41467_2025_63154_Fig6_HTML.jpg" loading="lazy" id="d33e1662" height="644" width="667" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic depicting the transplantation assay, where wild-type (WT) stage HH6 nodes were transplanted into GFP-positive HH4 hosts, followed by dissociation for single-cell RNA-seq (<em>n</em> = 5). <strong>b</strong> UMAP plot displaying GFP-positive host cells across different clusters, showing the formation of ectodermal and mesodermal cells. Gray dots represent unconverted cells. <strong>c</strong>, <strong>d</strong> Schematic of node transplantation experiments into transgenic embryos. WT stage HH4 embryos were electroporated with <em>CHRD-E</em> (notochord enhancer) or <em>TCF15-E</em> (somite enhancer), followed by node grafting from GFP-positive HH6 embryos. <strong>e</strong> After ~ 16 h, <em>CHRD-E</em> activity is detected in the notochord, confirming the induction of trunk-specific structures in the host (<em>n</em> = 10/17 biological replicates with similar results). <strong>f</strong> High magnification image of the marked region within the transplanted cells shown in panel (<strong>e</strong>), showing activation of <em>CHRD</em>-E within the induced secondary axis. <strong>g </strong><em>TCF15-E</em> activity is observed in the somite region (arrows), confirming the induction of paraxial mesoderm in host embryos (<em>n</em> = 6/10 biological replicates with similar results). <strong>h</strong> HCR-FISH showing <em>CHRD</em> (notochord – magenta, arrowhead) and <em>LMO2</em> (blood islands - blue) expression in embryos after 16 h post-transplantation of stage HH6 GFP-positive node (<em>n</em> = 5/5 biological replicates with similar results). <strong>i</strong>, <strong>j </strong>High magnification image of the marked region shown in panel (<strong>h</strong>). GFP-positive cells from the donor form posterior structures (somites, notochord) and are shown in green (<strong>i</strong>). HCR-FISH for <em>CHRD</em> (magenta) showing that some notochord cells are GFP-negative (<strong>j</strong>). Scale bar = 100 μm. NC = Notochord, S = Somite.</p></figcaption></figure><p id="Par17">Since notochord and paraxial mesoderm cells were relatively rare in our scRNA-seq dataset, we also investigated the capacity of the regressing node to induce posterior mesoderm with additional experiments. We grafted a GFP-positive regressing Hensen’s node (HH6) into stage HH4 embryos electroporated with reporter constructs containing <em>CHRD</em> or <em>TCF15</em> enhancers (Fig. <a href="#Fig6" class="usa-link">6c, d</a>). We identified the <em>CHRD</em> enhancer using our stage HH4 Hensen’s node ATAC-seq dataset and demonstrated its activity in the notochord (Supplementary Fig. <a href="#MOESM1" class="usa-link">9h, i</a>). <em>TCF15</em> enhancer was previously described, and it is active in presomitic mesoderm and somites<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Since only host cells have been transfected with the enhancer construct, the reporter gene activity indicates mesoderm induction. Consistent with the induction of notochord cells by the graft, we observed activation of the <em>CHRD</em> enhancer in the midline of the induced secondary axis (Fig. <a href="#Fig6" class="usa-link">6e, f</a>, <em>n</em> = 10/17, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). Adjacent to the notochord, we also observed activation of <em>TCF15</em> enhancer in the somite epithelium (Fig. <a href="#Fig6" class="usa-link">6g</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">9j–l</a><em>n</em> = 6/10 and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). These cells also expressed endogenous <em>LEF1</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">9j–l</a>), which is expressed at high levels by paraxial mesoderm<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. As expected, we also observe the contribution of the GFP donor tissue to the somites (Fig. <a href="#Fig6" class="usa-link">6f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">9j</a>, <a href="#MOESM1" class="usa-link">l</a>), consistent with previous reports that the medial part of these structures is derived from the node<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. HCR-FISH for <em>CHRD</em> in the secondary axis also revealed that the notochord presented GFP-negative cells and was thus induced from host tissue (Fig. <a href="#Fig6" class="usa-link">6h, i</a>).</p>
<p id="Par18">Our transplantation experiments indicate that the posterior node displays organizer activity and that these cells can drive cellular diversification in stage HH4 epiblast cells. We next examined the regressing node’s ability to induce naïve cells, such as the early blastoderm and embryonic stem cells, into mesoderm. In the first experiment, we isolated the anterior blastoderm from HH3+ host embryos by dissecting away the primitive streak (Supplementary Fig. <a href="#MOESM1" class="usa-link">10a</a>). We then grafted stage HH6 GFP-positive nodes in the isolated blastoderm, which was free of any mesoderm-derived cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">10a</a>). The transplanted node successfully induced the expression of mesodermal and ectodermal markers, specifically <em>BRACHYURY (BRA)</em> and <em>SOX2</em>, respectively (Supplementary Fig. <a href="#MOESM1" class="usa-link">10b–f</a>'', <em>n</em> = 5/7), demonstrating its capacity to direct naïve cells toward these lineages. In the second experiment, we co-cultured stage HH6 Hensen’s nodes with human embryonic stem cells (hESCs) engineered with reporters for the three germ layers: <em>SOX2</em>-mCitrine (ectoderm), <em>BRA</em>-mCerulean (mesoderm), and <em>SOX17</em>-tdTomato (endoderm) (RUES2-GLR)<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. This system allowed us to visually track the induction of germ layer markers in response to the transplanted node (Supplementary Fig. <a href="#MOESM1" class="usa-link">10g</a>, <em>n</em> = 3/3). Pluripotent cells express <em>SOX2</em>, and a few cells have begun to express <em>SOX17</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">10h</a>), while <em>BRA</em> expression is absent (Supplementary Fig. <a href="#MOESM1" class="usa-link">10i, j</a>). Upon transplantation, the hESCs adjacent to the explants expressed the mesodermal marker <em>BRA</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">10k–o</a>), further confirming the node’s inductive capacity. Together, these results demonstrate that the posterior cell population in Hensen’s node possesses the capacity to induce the formation of posterior structures and cell types, including both axial and paraxial mesoderm, thereby functioning as an organizer in the avian embryo.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par19">Here, we provide a detailed characterization of the molecular signatures and inductive properties of the cell populations within Hensen’s node. Our findings reveal that the node is composed of two molecularly distinct populations, anterior and posterior node cells, which function as the head and trunk organizers, respectively. These populations exhibit distinct transcriptional identities, underscoring the molecular and functional heterogeneity of the organizer. Fate map experiments conducted by Storey and colleagues<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> indicate that distinct portions of the node also have distinct developmental fates<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Their work identified the medial-anterior region of the node as giving rise to notochord cells, while the posterior-lateral regions contribute to somite formation<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Our study builds upon these findings by demonstrating that these anatomical territories correspond to molecularly distinct cell populations with unique transcriptional programs and inductive capacities.</p>
<p id="Par20">The division of the organizer into transcriptionally distinct anterior and posterior populations is not unique to the chick and appears to be a deeply conserved feature of vertebrate development. In zebrafish, Saude and colleagues (2000)<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> demonstrated that anterior and posterior organizer activities are spatially segregated within the embryonic shield, the zebrafish equivalent of the node in amniotes and the blastopore lip in amphibians. This compartmentalization is reflected in distinct transcription factor expression: <em>gsc</em> marks cells that induce anterior structures, while <em>floating head/noto</em> marks cells associated with posterior fates. Gritsman et al. <sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> showed that this anteroposterior polarity is established before gastrulation and is regulated by differential Nodal signaling, with <em>gsc</em>-positive prechordal plate progenitors positioned closer to the margin than <em>noto</em>-expressing notochordal cells. Comparable spatial organization has also been observed in Xenopus, where fate-mapping and functional experiments have long distinguished between anterior and trunk organizer domains within the organizer. The early Xenopus organizer expresses <em>gsc</em>, <em>chordin</em>, and <em>cerberus</em> and is required for head induction<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>, whereas posterior regions express <em>xnot</em> and <em>brachyury</em>, contributing to notochord and mesoderm formation<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>.</p>
<p id="Par21">In the mouse, recent single-cell atlases<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup> have also examined anterior and posterior compartments within the node and axial mesendoderm. These studies revealed early segregation of <em>Gsc</em> and <em>Foxa2</em> expression domains, alongside Nodal-dependent specification of the anterior mesendoderm, which contributes to head structures, and posterior domains that give rise to the notochord<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. The anterior primitive streak and node harbor organizer-like populations that express <em>Gsc</em> and <em>Otx2</em>, whereas posterior node regions express <em>T</em> (<em>Brachyury</em>), <em>Mixl1</em>, and EMT-associated factors, resembling the molecular signature we report in chick. Our identification of anterior <em>GSC</em>-positive and posterior <em>LMO1/RND3</em>-positive populations in Hensen’s node reinforces the view that the subdivision of organizer function is a conserved feature of vertebrate development. Nevertheless, our findings also highlight possible species-specific aspects of organizer regulation. For example, in the chick, <em>BRACHYURY</em> is expressed in both head and trunk organizer cells, while <em>NOTO</em> expression appears restricted to the forming notochord and is absent from posterior node cells. Alternatively, these patterns may reflect differences in developmental timing or sampling. For example, scRNA-seq analysis in zebrafish<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> identified an intermediate population co-expressing <em>gsc</em>, <em>brachyury (ta)</em>, and <em>noto</em>, suggesting similar transcriptional states may exist in other species but remain undetected. High-resolution comparative studies will be essential to distinguish biological variation from technical limitations.</p>
<p id="Par22">These findings provide a mechanistic foundation for a long-standing model of organizer function. Our results align with the hypothesis first proposed by Otto Mangold<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>, who suggested that the organizer has distinct head and trunk inductive capacities. Mangold’s experiments demonstrated that early-stage organizers induce head structures when transplanted, while later-stage organizers promote trunk formation<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Our findings provide a cellular and molecular basis for Mangold’s hypothesis by identifying two distinct populations in the node - one responsible for head induction and the other for trunk development. As the node regresses during gastrulation, its cellular composition shifts, transitioning from predominantly anterior organizer cells to posterior cells. This shift in cell populations likely underlies the changing inductive properties of the node, explaining why early-stage nodes induce head structures while later-stage nodes promote trunk formation. Our data suggests that distinct gene regulatory circuits govern these two populations. <em>GSC</em> is a key regulator in the anterior cells, orchestrating a program geared toward head formation<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>,<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. In contrast, the posterior cells appear to rely on a genetic circuit involving <em>LMO1</em>, mesodermal transcription factors, and EMT-related genes. While our transcriptomic analysis suggests that these populations may also have differing secretory profiles, which could contribute to their distinct inductive roles, this aspect requires further investigation.</p>
<p id="Par23">The shifts in cellular composition we observed within the Hensen’s node also explain the temporal regulation of its inductive abilities. These changes are driven by extensive morphogenetic events during node development. First, there is the migration of anterior node cells that leave the structure to contribute to head formation. This is followed by the regression of the node, which moves posteriorly along the primitive streak during notochord formation. These morphogenetic movements lead to changes in the node’s composition, which we have observed across different developmental stages, as anterior organizer cells are gradually replaced by posterior, trunk-inducing cells. Trajectory analyses using RNA velocity and Monocle3 pseudotime inference suggest dynamic cell state transitions within the node, with epiblast cells giving rise to anterior node cells, which subsequently contribute to posterior node, notochord, and endodermal derivatives. Experimental lineage tracing will be required to validate these trajectories. These observations raise important questions about how trunk organizer cells are specified and how their emergence influences the behavior of head organizer cells. For example, the appearance of trunk organizer cells may initiate signaling events that prompt anterior cells to exit the node. Taken together, our findings support a model in which the node undergoes continuous remodeling through cell-cell interactions and cell-stage transitions, allowing it to coordinate successive inductive events along the body axis.</p>
<p id="Par24">Beyond providing insights into embryonic development, our findings also have important implications for regenerative medicine and evolutionary biology. Our single-cell analysis of secondary axes induced by node grafts, complemented by enhancer-reporter assays in host tissue, demonstrates that the trunk organizer population is capable of inducing a broad range of caudal cell fates, including both mesodermal and ectodermal derivatives (Fig. <a href="#Fig6" class="usa-link">6</a>). Understanding how specific cell populations within the organizer control tissue induction and axis formation could inform strategies for engineering tissues and organs from pluripotent stem cells<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. For instance, harnessing the inductive properties of these cells may provide a foundation for developing therapies to repair or replace damaged tissues, particularly in cases of congenital defects linked to early patterning errors<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Moreover, our work contributes to the broader understanding of how the organizer has evolved across species, offering clues into how vertebrate body plans have diversified through evolutionary time. The identification of specialized organizer populations within the Hensen’s node raises intriguing questions about how these mechanisms may vary across different organisms and what evolutionary pressures shaped the emergence of such distinct yet coordinated inductive functions.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Ethics statement</h3>
<p id="Par25">This work was conducted with ethical regulations and guidelines approved by BCH ESCRO (BCHSC#2022.10.07).</p></section><section id="Sec11"><h3 class="pmc_sec_title">Embryo collection and tissue dissociation</h3>
<p id="Par26">Fertilized White Leghorn chicken eggs were obtained from the Department of Animal Science, University of Connecticut. Transgenic GFP chicken eggs were obtained from Clemson University through Dr. Susan Chapman. Eggs were incubated at 37-38 °C and 50% humidity until reaching the desired developmental stage. Embryos were collected using filter paper as a carrier<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup> and staged according to Hamburger and Hamilton<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The Hensen’s node of 20 embryos at HH3 (12-13 h), 38 embryos at HH4 (18–20 h), and 20 embryos at HH6 (25-26 h) were fine dissected using spring scissors and dissociated using Accumax (Sigma #A7089) for 15–20 min at room temperature to generate a single-cell suspension.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Single-cell RNA-Seq library preparation and sequencing</h3>
<p id="Par27">For single-cell RNA sequencing (scRNA-seq), the dissociated cells were washed with 1X PBS containing 0.04% of BSA and filtered using a Flowmi cell strainer (Millipore-Sigma), porosity of 40 µM to remove cell debris and clumps. The gene expression library preparation and sequencing were conducted by Cornell University BRC Genomics Core Facility, following the manufacturer’s instructions (10x Genomics). The libraries were prepared using the NextSeq 500/550 75 bp kit and sequenced using the NextSeq 500 instrument. The sequencing run was configured with the following read lengths: an initial read of 28 bases, an 8-base pair index read for sample multiplexing, and a final read of 56 bases.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Single-cell RNA-Seq data processing and analysis</h3>
<p id="Par28">FASTq files of each sample generated from the 10x Genomics Chromium platform were aligned to the <em>Gallus gallus</em> GRCg6a.101 reference genome using 10x Genomics Cell Ranger software v5.0.1 with default parameters. Stages HH3, HH4, and HH6 Hensen’s node-filtered expression matrices were analyzed using the R package Seurat (version 4.1.3). We filtered out the low-quality cells by removing single cells with less than 2000 (HH3), 1000 (HH4), and 500 (HH6) genes, and with a percentage of reads that map to the mitochondrial genome above 5%. After the filtering step, 4849 (HH3), 8694 (HH4), and 15,095 (HH6) cells remained, and the data were normalized using the scale factor = 10,000. Next, we identified the highly variable expressed genes that were used in the downstream analysis by using the function FindVariableFeatures. The Seurat pipeline was used to perform linear dimensional reduction and unsupervised clustering. To generate the UMAP of the 4849 (HH3), 8694 (HH4), and 15,095 (HH6) cells, we used the first 20 principal components. In the clustering step, the number of dimensions used to find the neighbors was 20, and the resolution was 0.3 (HH3 and HH4) and 0.5 (HH6). Next, to identify and remove the potential doublets, we used the DoubletFinder tool. A total of 250 (HH3), 466 (HH4), and 868 (HH6) doublets were identified and removed from the datasets. A total of 4599 (HH3), 8228 (HH4), and 14,227 (HH6) cells passed the criteria described above. Cells with high content of mitochondria biogenesis and cell cycle genes were excluded from the downstream analysis. This adjustment resulted in a final cell count of 4512 for HH3, 8134 for HH4, and 12,753 for HH6. Differential expression analysis for each cluster was then conducted using the FindAllMarkers function. Genes on the female chromosome W were excluded from the HH3 and HH6 datasets to prevent sex-biased analysis and ensure the integrity of cell populations. Cell identity for each cluster was assigned based on the expression profile of known marker genes. <em>CHRD</em>-positive cells were identified from the scRNA-seq dataset by selecting cells with a scaled expression level of <em>CHRD</em> greater than 0.5. The subset of <em>CHRD</em>-positive cells was re-clustered using the FindNeighbors function on the first 20 principal components with a resolution of 0.5. The total <em>CHRD</em>-positive cells were 1759 for HH3, 2215 for HH4, and 6538 for HH6.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Whole-mount HCR-FISH in chick embryos</h3>
<p id="Par29">Hybridization chain reaction RNA fluorescent in situ hybridization (HCR-FISH, Molecular Instruments HCR protocol v3.0) was performed on stage HH4 (18–20 h) and HH6 (25-26 h) whole mount chick embryos. Briefly, embryos were collected and fixed in 4% paraformaldehyde for 1-2 h at room temperature. After fixation, embryos were dissected to remove the vitelline membrane, washed with PBS containing 0.1% Tween (PBST) on ice, and then stored in methanol at − 20 °C until use. For the rehydration process, embryos were subjected to a series of graded Methanol/PBST washes on ice. During the detection step, embryos were pre-hybridized with probe hybridization buffer for 30 min at 37 °C, followed by incubation with 4 pmol of each probe set overnight. Excess probes were removed by washing the embryos with probe wash buffer at 37 °C. Embryos were then pre-amplified with amplification buffer and incubated with hairpin overnight at room temperature, protected from light. The excess hairpins were removed by washing the embryos with 5X SSCT at room temperature. Embryos were incubated with DAPI for 1 h, followed by sample mounting for microscopy. Imaging was conducted at Cornell University BRC imaging core facility using a Zeiss LSM880 confocal multiphoton inverted microscope with a 20x objective lens. Additional imaging was also performed at the Core for Imaging Technology &amp; Education (CITE) at Harvard Medical School, using a Nikon AX R point scanning confocal microscope, also equipped with a 20x objective lens. In addition, embryos were imaged using the upright Zeiss Axio Zoom fluorescence microscope available in our lab. Image analysis and processing were performed using Fiji software. The target mRNA probe sets, amplifiers, and buffers were purchased from Molecular Instruments (<a href="https://www.molecularinstruments.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.molecularinstruments.com/</a>). Probes purchased from IDT were designed using the custom software created by the Özpolat Lab<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. These probes were amplified with Molecular Instruments hairpins. All HCR-FISH experiments were performed in multiple embryos (<em>n</em> ≤ 10) per condition to ensure reproducibility.</p></section><section id="Sec15"><h3 class="pmc_sec_title">HCR-FISH on tissue sections</h3>
<p id="Par30">For HCR in tissue sections, embryos were collected and fixed in 4% paraformaldehyde for 1 h at room temperature. Following fixation, embryos were dissected, washed with DEPC-PBS, and incubated with 5% sucrose/PBS for 3 h at room temperature. Embryos were then transferred to 15% sucrose/PBS overnight. The next day, embryos were incubated with a 1:1 mixture of OCT and 30% Sucrose/PBS, embedded in disposable plastic cryomolds, and stored at − 80 °C until cryosectioning. Tissue sections of 10 μM were obtained using the CryoStar NX50 (Thermo Fisher). HCR-FISH was performed following the HCR RNA-FISH protocol for fixed tissue sections provided by Molecular Instruments (<a href="https://www.molecularinstruments.com/hcr-rnafish-protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.molecularinstruments.com/hcr-rnafish-protocols</a>). In brief, sections on slides were fixed in ice-cold 4% paraformaldehyde for 15 min at 4 °C. The slides were then washed with a series of ethanol at room temperature and then incubated in 1X PBS. For the detection stage, slides were pre-hybridized for 10 min inside a humidified chamber pre-warmed to 37 °C, followed by incubation with 0.4 pmol of each probe set overnight. Excess probes were removed by washing the embryos with probe wash buffer at 37 °C. Next, the sections were pre-amplified with amplification buffer in a humidified chamber for 30 min at room temperature and incubated with hairpin solution in the dark overnight, also at room temperature. Finally, excess hairpins were removed by washing the embryos with 5X SSCT at room temperature. The slides were incubated with DAPI for 1 h, followed by sample mounting for imaging. Tissue sections were imaged using a Zeiss Imager.Z2 fluorescent microscope with a 20x objective lens.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Image segmentation and analysis</h3>
<p id="Par31">Cell segmentation was performed using the open-source image software Cellprofiler (v.4.2.1 <a href="http://www.cellprofiler.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.cellprofiler.org</a>). A custom pipeline (Supplementary Data <a href="#MOESM8" class="usa-link">5</a>) was designed to segment the nuclei, define the <em>CHRD</em>-positive domain, and quantify marker expression at the single-cell level. The Hensen’s node cells can be identified by <em>CHRD</em> expression. Non-Hensen’s node cells were excluded, and the intensity of the expression for anterior and posterior node markers were analyzed to characterize the spatial distribution of these two populations within the Hensen’s node. This pipeline effectively detects the distinct compartments where each cell population is present.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Pseudotime trajectory analysis</h3>
<p id="Par32">RNA velocity analysis was performed using the Python package scVelo<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Initially, the spliced and unspliced counts matrix of stage HH4 Hensen’s node was constructed using the Velocyto command line tool<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Primary data processing of stage HH4 Hensen’s node scRNA-seq was performed using Seurat in R. To convert the Seurat object into a format compatible with scVelo, we exported all necessary metadata, including the metadata table, expression counts matrix, dimensionality reduction matrix, and gene names. Next, the data was loaded in Python, where the spliced/unspliced counts matrix and the Seurat metadata were integrated. RNA velocity was computed using the scVelo dynamical model. The above-described method was also employed for RNA velocity analysis of stages HH3, HH4, and HH6 Hensen’s node-integrated dataset. Single-cell RNA-seq Seurat objects from the three developmental stages were integrated, and the batch effect was corrected using the Seurat package. Trajectory analysis was also performed using <span class="text-underline">Monocle 3</span> (v1.4.25) to reconstruct developmental progression across stages HH3, HH4, and HH6. The Seurat object containing integrated single-cell RNA-seq data was converted to a Monocle object. Dimensionality reduction and clustering results from Seurat were transferred to ensure consistency between analyses. Bootstrap-based cluster stability was performed to assess the robustness of clustering across developmental stages (HH3, HH4, and HH6). A total of 200 bootstrap iterations were conducted, each time randomly sampling 1750 cells from each developmental stage from the integrated Seurat object. Each subsample was clustered independently, and the cluster assignments were compared to the original clustering using the Adjusted Rand Index (ARI). The ARI distribution (median ≈ 0.222) indicates moderate but consistent cluster reproducibility.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Enhancer cloning</h3>
<p id="Par33">The <em>CHRD</em> enhancer was identified from ATAC-seq datasets generated by the Simoes-Costa lab in stage HH4 Hensen’s node tissue. A region of high accessibility in the <em>CHRD</em> locus was amplified from HH10 chicken genomic DNA and cloned into the pTK-mCherry reporter vector<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. The DNA fragment was amplified using primers with overlapping ends to facilitate cloning via Gibson assembly (Invitrogen #<a href="https://www.ncbi.nlm.nih.gov/protein/A46627" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A46627</a>). Primers used: Fw - tcttacgcgtgctagcccACCGGACATGCAGGCAAATA and Rv - atcgcagatctcgagcccCACGTAGCCCCTCACAGC. When tested in-vivo, this element drove specific reporter activity in the notochord, consistent with the endogenous <em>CHRD</em> expression pattern. A cis-regulatory element (CRE) for <em>TCF15</em> previously described by Mok and colleagues<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> was amplified and cloned into the pTK-mCherry reporter vector as described above. This CRE drives reporter activity in presomitic mesoderm, somites, and lateral plate mesoderm. The following primers were used to clone the DNA fragment: FW - tcttacgcgtgctagcccTCTCCAGCTCTGTTCAAGGG and RV - atcgcagatctcgagcccGCAGTGGAAAATCCAAGTCCC. <em>SOX2</em> enhancers N1 and N2, previously described by Uchikawa and colleagues<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, were also cloned into pTK-GFP<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup> and pTK-mcherry<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> expression vectors.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Embryo preparation and enhancer electroporation</h3>
<p id="Par34">Wild-type host embryos at stages HH3 + /HH4 were collected using the filter paper-based easy-culture method<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. <em>CHRD</em> and <em>TCF15</em> enhancer plasmids were transfected into the embryos by <em>ex-ovo</em> electroporation. Briefly, 1-2 μg/μL of the expression vector was injected into the space between the epiblast tissue and vitelline membrane to cover the entire embryo. Embryos were electroporated with platinum electrodes using the following parameters: 5 pulses of 5 V, 50 ms on, 100 ms off. Electroporated host embryos were then transferred to 35 mm Petri dishes containing Bacto-agar+albumen (1:1) substrate supplemented with 10% glucose and were kept at 37 °C until use. Bacto-agar solution adjusted to 0.8%.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Transplantation of Hensen’s Node into electroporated hosts</h3>
<p id="Par35">Hensen’s nodes from stage HH6 GFP donor embryos were finely dissected and transplanted into the marginal zone between the area opaca and area pellucida of wild-type stage HH4 host embryos. These host embryos had been previously electroporated with either <em>CHRD</em> (<em>n</em> = 10/17) or <em>TCF15</em> (<em>n</em> = 6/10) enhancers. To allow further development, embryos were cultured at 37 °C with 50% humidity for 16–18 h. Subsequently, embryos were fixed, stained with DAPI, and processed for imaging.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Immunofluorescence on transplanted embryos</h3>
<p id="Par36">Grafted embryos were fixed in 4% paraformaldehyde for 30 min at room temperature, followed by three washes with PBST (1X PBS + 0.5% Triton X-100) for 10 min each. Next, embryos were blocked and permeabilized with PBST supplemented with 3% normal donkey serum (NDS) and 1% bovine serum albumin (BSA) at room temperature for 1 h. After the blocking step, wild-type host embryos were incubated with anti-<em>OTX2</em> antibody (1:300, R&amp;D Systems #AF1979), and host embryos electroporated with <em>TCF15</em> were incubated with anti-<em>LEF1</em> antibody (1:200, Abcam #ab137872) diluted in blocking buffer overnight at 4 °C. The following day, embryos were washed three times with PBST for 15 min each and then incubated with secondary donkey anti-goat antibody conjugated with Alexa Fluor 647 (Molecular Probes) for 2 h at room temperature. After the secondary antibody incubation, embryos were washed, stained with DAPI, and mounted with fluoromount for imaging. Images were acquired using a Nikon AX R point scanning confocal microscope, also equipped with a 20x objective lens. In addition, embryos were imaged using the upright Zeiss Axio Zoom fluorescence microscope available in our lab.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Single-cell RNA-seq of induced secondary axes</h3>
<p id="Par37">GFP host embryos at stage HH3 + /HH4 were collected and cultured as previously described (see Embryo preparation and enhancer electroporation). Wild-type HH6 Hensen’s nodes were then dissected and grafted into the marginal zone between the area opaca and area pellucida of the GFP host embryos. Embryos were cultured for 16–18 h to allow further development. Next, secondary axes (<em>n</em> = 5) induced by the graft were micro-dissected and dissociated into a single-cell suspension as described above. scRNA-seq library preparation and sequencing were performed according to the manufacturer’s instructions (10x Genomics). Libraries were prepared using the Illumina NextSeq 500/550 High Output kit v2.5 (75 Cycles) and sequenced using the NextSeq 500/550 instrument at the Boston Children’s Hospital Molecular Genetics Core facility. The final library was paired-end sequenced at (28 × 10 ×10 × 44).</p></section><section id="Sec23"><h3 class="pmc_sec_title">Single-cell RNA-seq data processing of secondary axes</h3>
<p id="Par38">To identify the GFP host cells, we modified the <em>Gallus gallus</em> GRCg6a.99 reference genome to include the eGFP sequence. The FASTq files were then aligned to this modified reference genome. The filtered expression matrix was analyzed using the R package Seurat. Low-quality cells with fewer than 200 genes and a mitochondrial read percentage above 5% were removed, leaving a total of 8249 cells. The data was normalized using the scale factor = 10000. Highly variable-expressed genes for downstream analysis were identified using the FindVariableFeatures function. The Seurat pipeline was employed for linear dimensional reduction and unsupervised clustering. The first 20 principal components were used to generate the UMAP plot. For clustering, the number of dimensions used to find the neighbors was set to 20, and the resolution was 0.3. DoubletFinder was then used to filter out 499 doublets. From the 7750 remaining singlets, 2372 GFP-positive cells were isolated, and differential expression analysis for each cluster was performed using the FindAllMarkers function. Cell identities were assigned to each population based on the expression of known marker genes found in their gene expression profiles.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Quantification of <em>SOX2</em> enhancer activity following node grafts</h3>
<p id="Par39">Wild-type host embryos at stages HH3 + /HH4 were collected and electroporated with <em>SOX2</em>N1::GFP (N1-GFP) and <em>SOX2</em>N2::mCherry (N2-mChe) enhancer constructs at a final concentration of 1 µg/µL, as previously described (see <em>Embryo preparation and enhancer electroporation</em>). Following electroporation, stage HH4 and HH6 Hensen’s nodes were dissected from wild-type embryos and grafted into the marginal zone between the area opaca and area pellucida of the transgenic host embryos. Embryos were cultured for 15–16 h, then collected, fixed, and mounted for imaging. Grafted tissues were imaged using a Nikon AX R point scanning confocal microscope, also equipped with a 20x objective lens. Quantification of <em>SOX2</em>N1::GFP- and <em>SOX2</em>N2::mCherry-positive cells was performed using Fiji’s particle analysis tool for each individual embryo. A fluorescence threshold was first applied to distinguish labeled cells from the background signal. The <em>watershed</em> function was then used to separate adjacent cells, followed by segmentation and counting using the analyze particles function, which selected objects within the 3–10 µm size range. A binomial generalized linear model (GLM) analysis was performed to evaluate changes in the proportion of <em>SOX2</em>N1::GFP and <em>SOX2</em>N2<em>::mCherry</em>-labeled cells following stages HH4 or HH6 node transplantation. For each embryo, the proportion of labeled cells was calculated by dividing the number of GFP + or mCherry + cells by the total number of fluorescent cells. The total number of fluorescent cells per embryo was used as a weighting factor to account for variation in sample size across embryos. Predicted proportions and 95% confidence intervals were calculated on the logit scale and converted to probabilities using the inverse logit function. Statistical significance was determined from model coefficients, with <em>p</em>-values &lt; 0.05 considered significant.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Co-culture of human ESC reporter line with chick Hensen’s node</h3>
<p id="Par40">RUES2-GLR<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>, a human embryonic stem cell (hESC) reporter line containing <em>SOX2</em>-mCitrine, <em>BRACHYURY</em>-mCerulean, and <em>SOX17</em>-tdTomato, was kindly donated by the Brivanlou lab at The Rockefeller University. RUES2-GLR cells were cultured in 35 mm ibidi dishes with mTser plus medium for 24 h before the addition of the Hensen’s node. Hensen’s nodes at stage HH5 + /6 were dissected and washed in PBS, then added to the hESC culture. Co-cultured cells were incubated for 14 h, fixed with 4% paraformaldehyde for 15 min at room temperature, and then washed with 1X PBS and stained with DAPI. Images were acquired using a widefield inverted Nikon Ti2 fluorescence microscope with monochrome and color cameras.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Transplantation of Hensen’s node into naïve anterior epiblast</h3>
<p id="Par41">Embryos at stage HH3+ were collected using filter paper and cultured in Petri dishes containing Bacto-agar+albumen (1:1) substrate supplemented with 10% glucose. The entire primitive streak and surrounding epiblast were then removed using a tungsten needle, leaving the naïve anterior epiblast as the host tissue. The Hensen’s node from stage HH6 GFP embryos was dissected and grafted into the marginal zone of the previously dissected host epiblast. The tissue was cultured for around 16 h at 37 °C and 50% humidity and then fixed in 4% paraformaldehyde for 15 min at room temperature. For immunostaining, the samples were incubated with anti-<em>SOX2</em> (1:200, Abcam #ab97959) and anti-<em>BRACHYURY</em> (1:300, R&amp;D Systems #AF2085) antibodies. Images were acquired using a Nikon point scanner confocal AX R microscope with tunable GaAsP PMTs and resonant scanner at 20x magnification.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par42">Further information on research design is available in the <a href="#MOESM9" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec28"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Peer Review file</a><sup> (560.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63154_MOESM3_ESM.pdf</a><sup> (564.2KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (55.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (45.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (30.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (500KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM8_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (4.4KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (96.2KB, pdf) </sup>
</div></div></section></section><section id="Sec29"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (12KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors thank Peter Schweitzer from Cornell University BRC Genomics Core Facility for all his help in generating the single-cell RNA-seq libraries and next-generation sequencing. We also thank Rebecca Williams and Joana de la Cruz from Cornell University BRC Imaging Core Facility for helping with the image acquisition using the Zeiss 880 confocal microscope. We acknowledge the Core for Imaging Technology &amp; Education at Harvard Medical School for microscopy and image analysis resources. We acknowledge image analysis assistance from the Center for Open Bioimage Analysis (COBA), which is supported by the National Institute of General Medical Sciences, NIH P41 GM135019. We thank Dr. Ali Brivanlou at The Rockefeller University for kindly donating the RUES2-GLR hESC reporter line. Finally, we thank the Harvard Chan Bioinformatics Core for their bioinformatics and statistics consultation services. This project was supported by National Institutes of Health grant DP2HD102043 (MSC) and Cornell University Center for Vertebrate Genomics Distinguished Scholar Award (TYK).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: T.Y.K. and M.S.C.; Methodology: T.Y.K., M.R. and M.S.C.; Investigation: T.Y.K.; Visualization: T.Y.K. and M.S.C.; Funding acquisition: M.S.C. and T.Y.K.; Project administration: M.S.C.; Supervision: M.S.C.; Writing: T.Y.K. and M.S.C.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par43"><em>Nature Communications</em> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data generated in this study is available in the Gene Expression Omnibus (GEO) repository under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228354" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE228354</a>, including raw data as well as processed/normalized files. <a href="#Sec29" class="usa-link">Source data</a> are provided in this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par44">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec30"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63154-2.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Spemann, H. &amp; Mangold, H. über Induktion von Embryonalanlagen durch Implantation artfremder Organisatoren. <em>Arch. Mikrosk. Anat. Entwicklmech.</em><strong>100</strong>, 599–638 (1924).</cite> [<a href="https://scholar.google.com/scholar_lookup?Spemann,%20H.%20&amp;%20Mangold,%20H.%20%C3%BCber%20Induktion%20von%20Embryonalanlagen%20durch%20Implantation%20artfremder%20Organisatoren.%20Arch.%20Mikrosk.%20Anat.%20Entwicklmech.100,%20599%E2%80%93638%20(1924)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Lemaire, P. &amp; Kodjabachian, L. The vertebrate organizer: structure and molecules. <em>Trends Genet.</em><strong>12</strong>, 525–531 (1996).
</cite> [<a href="https://doi.org/10.1016/s0168-9525(97)81401-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9257536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lemaire,%20P.%20&amp;%20Kodjabachian,%20L.%20The%20vertebrate%20organizer:%20structure%20and%20molecules.%20Trends%20Genet.12,%20525%E2%80%93531%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Martinez Arias, A. &amp; Steventon, B. On the nature and function of organizers. <em>Development</em><strong>145</strong>, 10.1242/dev.159525 (2018).</cite> [<a href="https://doi.org/10.1242/dev.159525" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5868996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29523654/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Kumar, V., Park, S., Lee, U. &amp; Kim, J. The organizer and its signaling in embryonic development. <em>J. Dev. Biol.</em><strong>9</strong>, 10.3390/jdb9040047 (2021).</cite> [<a href="https://doi.org/10.3390/jdb9040047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8628936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34842722/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Yu, J. K. et al. Axial patterning in cephalochordates and the evolution of the organizer. <em>Nature</em><strong>445</strong>, 613–617 (2007).
</cite> [<a href="https://doi.org/10.1038/nature05472" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17237766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20J.%20K.%20et%20al.%20Axial%20patterning%20in%20cephalochordates%20and%20the%20evolution%20of%20the%20organizer.%20Nature445,%20613%E2%80%93617%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Anderson, C. &amp; Stern, C. D. Organizers in development. <em>Curr. Top. Dev. Biol.</em><strong>117</strong>, 435–454 (2016).
</cite> [<a href="https://doi.org/10.1016/bs.ctdb.2015.11.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26969994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Anderson,%20C.%20&amp;%20Stern,%20C.%20D.%20Organizers%20in%20development.%20Curr.%20Top.%20Dev.%20Biol.117,%20435%E2%80%93454%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Asashima, M. &amp; Satou-Kobayashi, Y. Spemann-Mangold organizer and mesoderm induction. <em>Cells Dev.</em><strong>178</strong>, 203903 (2024).
</cite> [<a href="https://doi.org/10.1016/j.cdev.2024.203903" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38295873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Asashima,%20M.%20&amp;%20Satou-Kobayashi,%20Y.%20Spemann-Mangold%20organizer%20and%20mesoderm%20induction.%20Cells%20Dev.178,%20203903%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>De Roberts, E. M., Blum, M., Niehrs, C. &amp; Steinbeisser, H. Goosecoid and the organizer. <em>Dev. Suppl</em>. <strong>116</strong>, 167–171 (1992).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/1363720/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Izpisua-Belmonte, J. C., De Robertis, E. M., Storey, K. G. &amp; Stern, C. D. The homeobox gene goosecoid and the origin of organizer cells in the early chick blastoderm. <em>Cell</em><strong>74</strong>, 645–659 (1993).
</cite> [<a href="https://doi.org/10.1016/0092-8674(93)90512-o" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7916659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Izpisua-Belmonte,%20J.%20C.,%20De%20Robertis,%20E.%20M.,%20Storey,%20K.%20G.%20&amp;%20Stern,%20C.%20D.%20The%20homeobox%20gene%20goosecoid%20and%20the%20origin%20of%20organizer%20cells%20in%20the%20early%20chick%20blastoderm.%20Cell74,%20645%E2%80%93659%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Joubin, K. &amp; Stern, C. D. Molecular interactions continuously define the organizer during the cell movements of gastrulation. <em>Cell</em><strong>98</strong>, 559–571 (1999).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(00)80044-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10490096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Joubin,%20K.%20&amp;%20Stern,%20C.%20D.%20Molecular%20interactions%20continuously%20define%20the%20organizer%20during%20the%20cell%20movements%20of%20gastrulation.%20Cell98,%20559%E2%80%93571%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Niehrs, C. Regionally specific induction by the Spemann-Mangold organizer. <em>Nat. Rev. Genet.</em><strong>5</strong>, 425–434 (2004).
</cite> [<a href="https://doi.org/10.1038/nrg1347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15153995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Niehrs,%20C.%20Regionally%20specific%20induction%20by%20the%20Spemann-Mangold%20organizer.%20Nat.%20Rev.%20Genet.5,%20425%E2%80%93434%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Sasai, Y. et al. Xenopus chordin: a novel dorsalizing factor activated by organizer-specific homeobox genes. <em>Cell</em><strong>79</strong>, 779–790 (1994).
</cite> [<a href="https://doi.org/10.1016/0092-8674(94)90068-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3082463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8001117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sasai,%20Y.%20et%20al.%20Xenopus%20chordin:%20a%20novel%20dorsalizing%20factor%20activated%20by%20organizer-specific%20homeobox%20genes.%20Cell79,%20779%E2%80%93790%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Stern, C. D. et al. Head-tail patterning of the vertebrate embryo: One, two or many unresolved problems?. <em>Int. J. Dev. Biol.</em><strong>50</strong>, 3–15 (2006).
</cite> [<a href="https://doi.org/10.1387/ijdb.052095cs" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16323073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stern,%20C.%20D.%20et%20al.%20Head-tail%20patterning%20of%20the%20vertebrate%20embryo:%20One,%20two%20or%20many%20unresolved%20problems?.%20Int.%20J.%20Dev.%20Biol.50,%203%E2%80%9315%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Saude, L., Woolley, K., Martin, P., Driever, W. &amp; Stemple, D. L. Axis-inducing activities and cell fates of the zebrafish organizer. <em>Development</em><strong>127</strong>, 3407–3417 (2000).
</cite> [<a href="https://doi.org/10.1242/dev.127.16.3407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10903167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saude,%20L.,%20Woolley,%20K.,%20Martin,%20P.,%20Driever,%20W.%20&amp;%20Stemple,%20D.%20L.%20Axis-inducing%20activities%20and%20cell%20fates%20of%20the%20zebrafish%20organizer.%20Development127,%203407%E2%80%933417%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Boettger, T., Knoetgen, H., Wittler, L. &amp; Kessel, M. The avian organizer. <em>Int. J. Dev. Biol.</em><strong>45</strong>, 281–287 (2001).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/11291858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boettger,%20T.,%20Knoetgen,%20H.,%20Wittler,%20L.%20&amp;%20Kessel,%20M.%20The%20avian%20organizer.%20Int.%20J.%20Dev.%20Biol.45,%20281%E2%80%93287%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Wittler, L., Spieler, D. &amp; Kessel, M. in <em>The Vertebrate Organizer,</em> 395–408 (2003).</cite>
</li>
<li id="CR17">
<span class="label">17.</span><cite>Beddington, R. S. &amp; Robertson, E. J. Axis development and early asymmetry in mammals. <em>Cell</em><strong>96</strong>, 195–209 (1999).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(00)80560-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9988215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beddington,%20R.%20S.%20&amp;%20Robertson,%20E.%20J.%20Axis%20development%20and%20early%20asymmetry%20in%20mammals.%20Cell96,%20195%E2%80%93209%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Waddington, C. H. Induction by the endoderm in birds. <em>Wilhelm. Roux Arch. Entwickl. Mech. Org.</em><strong>128</strong>, 502–521 (1933).
</cite> [<a href="https://doi.org/10.1007/BF00649862" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28353959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waddington,%20C.%20H.%20Induction%20by%20the%20endoderm%20in%20birds.%20Wilhelm.%20Roux%20Arch.%20Entwickl.%20Mech.%20Org.128,%20502%E2%80%93521%20(1933)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Viebahn, C. Hensen’s node. <em>Genesis</em><strong>29</strong>, 96–103 (2001).
</cite> [<a href="https://doi.org/10.1002/1526-968x(200102)29:2&lt;96::aid-gene1010&gt;3.0.co;2-h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11170350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Viebahn,%20C.%20Hensen%E2%80%99s%20node.%20Genesis29,%2096%E2%80%93103%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Stern, C. D. Grafting Hensen’s node. <em>Methods Mol. Biol.</em><strong>461</strong>, 265–276 (2008).
</cite> [<a href="https://doi.org/10.1007/978-1-60327-483-8_18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19030803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stern,%20C.%20D.%20Grafting%20Hensen%E2%80%99s%20node.%20Methods%20Mol.%20Biol.461,%20265%E2%80%93276%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Hatada, Y. &amp; Stern, C. D. A fate map of the epiblast of the early chick embryo. <em>Development</em><strong>120</strong>, 2879–2889 (1994).
</cite> [<a href="https://doi.org/10.1242/dev.120.10.2879" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7607078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hatada,%20Y.%20&amp;%20Stern,%20C.%20D.%20A%20fate%20map%20of%20the%20epiblast%20of%20the%20early%20chick%20embryo.%20Development120,%202879%E2%80%932889%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Le Douarin, N. Patterning of the early neural primordium in the avian embryo. <em>Int. J. Dev. Biol.</em><strong>1</strong>, 51S (1996).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/9087692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Le%20Douarin,%20N.%20Patterning%20of%20the%20early%20neural%20primordium%20in%20the%20avian%20embryo.%20Int.%20J.%20Dev.%20Biol.1,%2051S%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Gros, J., Feistel, K., Viebahn, C., Blum, M. &amp; Tabin, C. J. Cell movements at Hensen’s node establish left/right asymmetric gene expression in the chick. <em>Science</em><strong>324</strong>, 941–944 (2009).
</cite> [<a href="https://doi.org/10.1126/science.1172478" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2993078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19359542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gros,%20J.,%20Feistel,%20K.,%20Viebahn,%20C.,%20Blum,%20M.%20&amp;%20Tabin,%20C.%20J.%20Cell%20movements%20at%20Hensen%E2%80%99s%20node%20establish%20left/right%20asymmetric%20gene%20expression%20in%20the%20chick.%20Science324,%20941%E2%80%93944%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Selleck, M. A. J. &amp; Stern, C. D. Fate mapping and cell lineage analysis of Hensen’s node in the chick embryo. <em>Development</em><strong>112</strong>, 615–626 (1991).
</cite> [<a href="https://doi.org/10.1242/dev.112.2.615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1794328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Selleck,%20M.%20A.%20J.%20&amp;%20Stern,%20C.%20D.%20Fate%20mapping%20and%20cell%20lineage%20analysis%20of%20Hensen%E2%80%99s%20node%20in%20the%20chick%20embryo.%20Development112,%20615%E2%80%93626%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Selleck, M. A. J. &amp; Stern, C. D. Commitment of mesoderm cells in Hensen’s node of the chick embryo to notochord and somite. <em>Development</em><strong>114</strong>, 403–415 (1992).</cite> [<a href="https://scholar.google.com/scholar_lookup?Selleck,%20M.%20A.%20J.%20&amp;%20Stern,%20C.%20D.%20Commitment%20of%20mesoderm%20cells%20in%20Hensen%E2%80%99s%20node%20of%20the%20chick%20embryo%20to%20notochord%20and%20somite.%20Development114,%20403%E2%80%93415%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Storey, K. G., Selleck, M. A. J. &amp; Stern, C. D. Neural induction and regionalisation by different subpopulations of cells in Hensen’s node. <em>Development</em><strong>121</strong>, 417–428 (1995).
</cite> [<a href="https://doi.org/10.1242/dev.121.2.417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7768183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Storey,%20K.%20G.,%20Selleck,%20M.%20A.%20J.%20&amp;%20Stern,%20C.%20D.%20Neural%20induction%20and%20regionalisation%20by%20different%20subpopulations%20of%20cells%20in%20Hensen%E2%80%99s%20node.%20Development121,%20417%E2%80%93428%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Streit, A. &amp; Stern, C. D. Mesoderm patterning and somite formation during node regression: differential effects of chordin and noggin. <em>Mech. Dev.</em><strong>85</strong>, 85–96 (1999).
</cite> [<a href="https://doi.org/10.1016/s0925-4773(99)00085-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10415349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Streit,%20A.%20&amp;%20Stern,%20C.%20D.%20Mesoderm%20patterning%20and%20somite%20formation%20during%20node%20regression:%20differential%20effects%20of%20chordin%20and%20noggin.%20Mech.%20Dev.85,%2085%E2%80%9396%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Streit, A., Berliner, A. J., Papanayotou, C., Sirulnik, A. &amp; Stern, C. D. Initiation of neural induction by FGF signalling before gastrulation. <em>Nature</em><strong>406</strong>, 74–78 (2000).
</cite> [<a href="https://doi.org/10.1038/35017617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10894544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Streit,%20A.,%20Berliner,%20A.%20J.,%20Papanayotou,%20C.,%20Sirulnik,%20A.%20&amp;%20Stern,%20C.%20D.%20Initiation%20of%20neural%20induction%20by%20FGF%20signalling%20before%20gastrulation.%20Nature406,%2074%E2%80%9378%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Yao, J. &amp; Kessler, D. S. Goosecoid promotes head organizer activity by direct repression of Xwnt8 in Spemann’s organizer. <em>Development</em><strong>128</strong>, 2975–2987 (2001).
</cite> [<a href="https://doi.org/10.1242/dev.128.15.2975" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11532920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yao,%20J.%20&amp;%20Kessler,%20D.%20S.%20Goosecoid%20promotes%20head%20organizer%20activity%20by%20direct%20repression%20of%20Xwnt8%20in%20Spemann%E2%80%99s%20organizer.%20Development128,%202975%E2%80%932987%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Umair, Z. et al. Goosecoid controls neuroectoderm specification via dual circuits of direct repression and indirect stimulation in xenopus embryos. <em>Mol. Cells</em><strong>44</strong>, 723–735 (2021).
</cite> [<a href="https://doi.org/10.14348/molcells.2021.0055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8560583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34711690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Umair,%20Z.%20et%20al.%20Goosecoid%20controls%20neuroectoderm%20specification%20via%20dual%20circuits%20of%20direct%20repression%20and%20indirect%20stimulation%20in%20xenopus%20embryos.%20Mol.%20Cells44,%20723%E2%80%93735%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Hamburger, V. &amp; Hamilton, H. L. A series of normal stages in the development of the chick embryo. <em>J. Morphol.</em><strong>88</strong>, 49–92 (1951).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/24539719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hamburger,%20V.%20&amp;%20Hamilton,%20H.%20L.%20A%20series%20of%20normal%20stages%20in%20the%20development%20of%20the%20chick%20embryo.%20J.%20Morphol.88,%2049%E2%80%9392%20(1951)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Becht, E. et al. Dimensionality reduction for visualizing single-cell data using UMAP. <em>Nat. Biotechnol.</em><strong>37</strong>, 38–47 (2019).</cite> [<a href="https://doi.org/10.1038/nbt.4314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30531897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Becht,%20E.%20et%20al.%20Dimensionality%20reduction%20for%20visualizing%20single-cell%20data%20using%20UMAP.%20Nat.%20Biotechnol.37,%2038%E2%80%9347%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Choi, H. M. T., Beck, V. A. &amp; Pierce, N. A. Next-generation in situ hybridization chain reaction: Higher gain, lower cost, greater durability. <em>ACS Nano</em><strong>8</strong>, 4284–4294 (2014).
</cite> [<a href="https://doi.org/10.1021/nn405717p" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4046802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24712299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choi,%20H.%20M.%20T.,%20Beck,%20V.%20A.%20&amp;%20Pierce,%20N.%20A.%20Next-generation%20in%20situ%20hybridization%20chain%20reaction:%20Higher%20gain,%20lower%20cost,%20greater%20durability.%20ACS%20Nano8,%204284%E2%80%934294%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Choi, H. M. T. et al. Third-generation in situ hybridization chain reaction: multiplexed, quantitative, sensitive, versatile, robust. <em>Development</em><strong>145</strong>, 10.1242/dev.165753 (2018).</cite> [<a href="https://doi.org/10.1242/dev.165753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6031405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29945988/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>De Robertis, E. M., Blum, M., Niehrs, C. &amp; Steinbeisser, H. Goosecoid and the organizer. <em>J. Neurosci.</em><strong>13</strong>, 167–171 (1993).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/1363720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Robertis,%20E.%20M.,%20Blum,%20M.,%20Niehrs,%20C.%20&amp;%20Steinbeisser,%20H.%20Goosecoid%20and%20the%20organizer.%20J.%20Neurosci.13,%20167%E2%80%93171%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Yasuo, H. &amp; Lemaire, P. Role of Goosecoid, Xnot and Wnt antagonists in the maintenance of the notochord genetic programme in Xenopus gastrulae. <em>Development</em><strong>128</strong>, 3783–3793 (2001).
</cite> [<a href="https://doi.org/10.1242/dev.128.19.3783" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11585804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yasuo,%20H.%20&amp;%20Lemaire,%20P.%20Role%20of%20Goosecoid,%20Xnot%20and%20Wnt%20antagonists%20in%20the%20maintenance%20of%20the%20notochord%20genetic%20programme%20in%20Xenopus%20gastrulae.%20Development128,%203783%E2%80%933793%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Stirling, D. R. et al. CellProfiler 4: improvements in speed, utility and usability. <em>BMC Bioinform.</em><strong>22</strong>, 433 (2021).</cite> [<a href="https://doi.org/10.1186/s12859-021-04344-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8431850/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34507520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stirling,%20D.%20R.%20et%20al.%20CellProfiler%204:%20improvements%20in%20speed,%20utility%20and%20usability.%20BMC%20Bioinform.22,%20433%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Knezevic, V., De Santo, R. &amp; Mackem, S. Two novel chick T-box genes related to mouse Brachyury are expressed in different, non-overlapping mesodermal domains during gastrulation. <em>Development</em><strong>124</strong>, 411–419 (1997).
</cite> [<a href="https://doi.org/10.1242/dev.124.2.411" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9053317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knezevic,%20V.,%20De%20Santo,%20R.%20&amp;%20Mackem,%20S.%20Two%20novel%20chick%20T-box%20genes%20related%20to%20mouse%20Brachyury%20are%20expressed%20in%20different,%20non-overlapping%20mesodermal%20domains%20during%20gastrulation.%20Development124,%20411%E2%80%93419%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>La Manno, G. et al. RNA velocity of single cells. <em>Nature</em><strong>560</strong>, 494–498 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0414-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6130801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30089906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?La%20Manno,%20G.%20et%20al.%20RNA%20velocity%20of%20single%20cells.%20Nature560,%20494%E2%80%93498%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Bergen, V., Lange, M., Peidli, S., Wolf, F. A. &amp; Theis, F. J. Generalizing RNA velocity to transient cell states through dynamical modeling. <em>Nat. Biotechnol.</em><strong>38</strong>, 1408–1414 (2020).
</cite> [<a href="https://doi.org/10.1038/s41587-020-0591-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32747759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bergen,%20V.,%20Lange,%20M.,%20Peidli,%20S.,%20Wolf,%20F.%20A.%20&amp;%20Theis,%20F.%20J.%20Generalizing%20RNA%20velocity%20to%20transient%20cell%20states%20through%20dynamical%20modeling.%20Nat.%20Biotechnol.38,%201408%E2%80%931414%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Wells, J. M. &amp; Melton, D. A. Vertebrate endoderm development. <em>Annu. Rev. Cell Dev. Biol.</em><strong>15</strong>, 393–410 (1999).
</cite> [<a href="https://doi.org/10.1146/annurev.cellbio.15.1.393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10611967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wells,%20J.%20M.%20&amp;%20Melton,%20D.%20A.%20Vertebrate%20endoderm%20development.%20Annu.%20Rev.%20Cell%20Dev.%20Biol.15,%20393%E2%80%93410%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Kimura, W., Yasugi, S., Stern, C. D. &amp; Fukuda, K. Fate and plasticity of the endoderm in the early chick embryo. <em>Dev. Biol.</em><strong>289</strong>, 283–295 (2006).
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2005.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16337933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kimura,%20W.,%20Yasugi,%20S.,%20Stern,%20C.%20D.%20&amp;%20Fukuda,%20K.%20Fate%20and%20plasticity%20of%20the%20endoderm%20in%20the%20early%20chick%20embryo.%20Dev.%20Biol.289,%20283%E2%80%93295%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Benazeraf, B. Dynamics and mechanisms of posterior axis elongation in the vertebrate embryo. <em>Cell Mol. Life Sci.</em><strong>76</strong>, 89–98 (2019).
</cite> [<a href="https://doi.org/10.1007/s00018-018-2927-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11105343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30283977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benazeraf,%20B.%20Dynamics%20and%20mechanisms%20of%20posterior%20axis%20elongation%20in%20the%20vertebrate%20embryo.%20Cell%20Mol.%20Life%20Sci.76,%2089%E2%80%9398%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Lawson, A. &amp; Schoenwolf, G. C. Cell populations and morphogenetic movements underlying formation of the avian primitive streak and organizer. <em>Genesis</em><strong>29</strong>, 188–195 (2001).
</cite> [<a href="https://doi.org/10.1002/gene.1023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11309852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lawson,%20A.%20&amp;%20Schoenwolf,%20G.%20C.%20Cell%20populations%20and%20morphogenetic%20movements%20underlying%20formation%20of%20the%20avian%20primitive%20streak%20and%20organizer.%20Genesis29,%20188%E2%80%93195%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Chuai, M. et al. Cell movement during chick primitive streak formation. <em>Dev. Biol.</em><strong>296</strong>, 137–149 (2006).
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2006.04.451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2556955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16725136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chuai,%20M.%20et%20al.%20Cell%20movement%20during%20chick%20primitive%20streak%20formation.%20Dev.%20Biol.296,%20137%E2%80%93149%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Kirby, M. L. et al. Hensen’s node gives rise to the ventral midline of the foregut: implications for organizing head and heart development. <em>Dev. Biol.</em><strong>253</strong>, 175–188 (2003).
</cite> [<a href="https://doi.org/10.1016/s0012-1606(02)00024-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12645923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kirby,%20M.%20L.%20et%20al.%20Hensen%E2%80%99s%20node%20gives%20rise%20to%20the%20ventral%20midline%20of%20the%20foregut:%20implications%20for%20organizing%20head%20and%20heart%20development.%20Dev.%20Biol.253,%20175%E2%80%93188%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Cho, K. W., Blumberg, B., Steinbeisser, H. &amp; De Robertis, E. M. Molecular nature of Spemann’s organizer: the role of the Xenopus homeobox gene goosecoid. <em>Cell</em><strong>67</strong>, 1111–1120 (1991).
</cite> [<a href="https://doi.org/10.1016/0092-8674(91)90288-a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3102583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1684739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cho,%20K.%20W.,%20Blumberg,%20B.,%20Steinbeisser,%20H.%20&amp;%20De%20Robertis,%20E.%20M.%20Molecular%20nature%20of%20Spemann%E2%80%99s%20organizer:%20the%20role%20of%20the%20Xenopus%20homeobox%20gene%20goosecoid.%20Cell67,%201111%E2%80%931120%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Storey, K. G., Crossley, J. M., De Robertis, E. M., Norris, W. E. &amp; Stern, C. D. Neural induction and regionalisation in the chick embryo. <em>Development</em><strong>114</strong>, 729–741 (1992).
</cite> [<a href="https://doi.org/10.1242/dev.114.3.729" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1618139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Storey,%20K.%20G.,%20Crossley,%20J.%20M.,%20De%20Robertis,%20E.%20M.,%20Norris,%20W.%20E.%20&amp;%20Stern,%20C.%20D.%20Neural%20induction%20and%20regionalisation%20in%20the%20chick%20embryo.%20Development114,%20729%E2%80%93741%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Uchikawa, M., Ishida, Y., Takemoto, T., Kamachi, Y. &amp; Kondoh, H. Functional analysis of chicken Sox2 enhancers highlights an array of diverse regulatory elements that are conserved in mammals. <em>Dev. Cell</em><strong>4</strong>, 509–519 (2003).
</cite> [<a href="https://doi.org/10.1016/s1534-5807(03)00088-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12689590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uchikawa,%20M.,%20Ishida,%20Y.,%20Takemoto,%20T.,%20Kamachi,%20Y.%20&amp;%20Kondoh,%20H.%20Functional%20analysis%20of%20chicken%20Sox2%20enhancers%20highlights%20an%20array%20of%20diverse%20regulatory%20elements%20that%20are%20conserved%20in%20mammals.%20Dev.%20Cell4,%20509%E2%80%93519%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Mok, G. F. et al. Characterising open chromatin in chick embryos identifies cis-regulatory elements important for paraxial mesoderm formation and axis extension. <em>Nat. Commun.</em><strong>12</strong>, 1157 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-21426-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7895974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33608545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mok,%20G.%20F.%20et%20al.%20Characterising%20open%20chromatin%20in%20chick%20embryos%20identifies%20cis-regulatory%20elements%20important%20for%20paraxial%20mesoderm%20formation%20and%20axis%20extension.%20Nat.%20Commun.12,%201157%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Schmidt, M., Tanaka, M. &amp; Munsterberg, A. Expression of β-catenin in the developing chick myotome is regulated by myogenic signals. <em>Development</em><strong>127</strong>, 4105–4113 (2000).
</cite> [<a href="https://doi.org/10.1242/dev.127.19.4105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10976043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmidt,%20M.,%20Tanaka,%20M.%20&amp;%20Munsterberg,%20A.%20Expression%20of%20%CE%B2-catenin%20in%20the%20developing%20chick%20myotome%20is%20regulated%20by%20myogenic%20signals.%20Development127,%204105%E2%80%934113%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Schmidt, M., Patterson, M., Farrell, E. &amp; Münsterberg, A. Dynamic expression of Lef/Tcf family members and β-catenin during chick gastrulation, neurulation, and early limb development. <em>Dev. Dyn.</em><strong>229</strong>, 703–707 (2004).
</cite> [<a href="https://doi.org/10.1002/dvdy.20010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14991726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmidt,%20M.,%20Patterson,%20M.,%20Farrell,%20E.%20&amp;%20M%C3%BCnsterberg,%20A.%20Dynamic%20expression%20of%20Lef/Tcf%20family%20members%20and%20%CE%B2-catenin%20during%20chick%20gastrulation,%20neurulation,%20and%20early%20limb%20development.%20Dev.%20Dyn.229,%20703%E2%80%93707%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Martyn, I., Kanno, T. Y., Ruzo, A., Siggia, E. D. &amp; Brivanlou, A. H. Self-organization of a human organizer by combined Wnt and Nodal signalling. <em>Nature</em><strong>558</strong>, 132–135 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0150-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6077985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29795348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martyn,%20I.,%20Kanno,%20T.%20Y.,%20Ruzo,%20A.,%20Siggia,%20E.%20D.%20&amp;%20Brivanlou,%20A.%20H.%20Self-organization%20of%20a%20human%20organizer%20by%20combined%20Wnt%20and%20Nodal%20signalling.%20Nature558,%20132%E2%80%93135%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Psychoyos, D. &amp; Stern, C. D. Fates and migratory routes of primitive streak cells in the chick embryo. <em>Development</em><strong>122</strong>, 1523–1534 (1996).
</cite> [<a href="https://doi.org/10.1242/dev.122.5.1523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8625839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Psychoyos,%20D.%20&amp;%20Stern,%20C.%20D.%20Fates%20and%20migratory%20routes%20of%20primitive%20streak%20cells%20in%20the%20chick%20embryo.%20Development122,%201523%E2%80%931534%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Schoenwolf, G. C., Garcia-Martinez, V. &amp; Dias, M. S. Mesoderm movement and fate during avian gastrulation and neurulation. <em>Dev. Dyn.</em><strong>193</strong>, 235–248 (1992).
</cite> [<a href="https://doi.org/10.1002/aja.1001930304" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1600242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schoenwolf,%20G.%20C.,%20Garcia-Martinez,%20V.%20&amp;%20Dias,%20M.%20S.%20Mesoderm%20movement%20and%20fate%20during%20avian%20gastrulation%20and%20neurulation.%20Dev.%20Dyn.193,%20235%E2%80%93248%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Wittler, L. &amp; Kessel, M. The acquisition of neural fate in the chick. <em>Mech. Dev.</em><strong>121</strong>, 1031–1042 (2004).
</cite> [<a href="https://doi.org/10.1016/j.mod.2004.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15296969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wittler,%20L.%20&amp;%20Kessel,%20M.%20The%20acquisition%20of%20neural%20fate%20in%20the%20chick.%20Mech.%20Dev.121,%201031%E2%80%931042%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Gritsman, K., Talbot, W. S. &amp; Schier, A. F. Nodal signaling patterns the organizer. <em>Development</em><strong>127</strong>, 921–932 (2000).
</cite> [<a href="https://doi.org/10.1242/dev.127.5.921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10662632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gritsman,%20K.,%20Talbot,%20W.%20S.%20&amp;%20Schier,%20A.%20F.%20Nodal%20signaling%20patterns%20the%20organizer.%20Development127,%20921%E2%80%93932%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Glinka, A., Wu, W., Onichtchouk, D., Blumenstock, C. &amp; Niehrs, C. Head induction by simultaneous repression of Bmp and Wnt signalling in Xenopus. <em>Nature</em><strong>389</strong>, 517–519 (1997).
</cite> [<a href="https://doi.org/10.1038/39092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9333244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Glinka,%20A.,%20Wu,%20W.,%20Onichtchouk,%20D.,%20Blumenstock,%20C.%20&amp;%20Niehrs,%20C.%20Head%20induction%20by%20simultaneous%20repression%20of%20Bmp%20and%20Wnt%20signalling%20in%20Xenopus.%20Nature389,%20517%E2%80%93519%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>von Dassow, G., Schmidt, J. E. &amp; Kimelman, D. Induction of the Xenopus organizer: expression and regulation of Xnot, a novel FGF and activin-regulated homeo box gene. <em>Genes Dev.</em><strong>7</strong>, 355–366 (1993).
</cite> [<a href="https://doi.org/10.1101/gad.7.3.355" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8095482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?von%20Dassow,%20G.,%20Schmidt,%20J.%20E.%20&amp;%20Kimelman,%20D.%20Induction%20of%20the%20Xenopus%20organizer:%20expression%20and%20regulation%20of%20Xnot,%20a%20novel%20FGF%20and%20activin-regulated%20homeo%20box%20gene.%20Genes%20Dev.7,%20355%E2%80%93366%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Artinger, M., Blitz, I., Inoue, K., Tran, U. &amp; Cho, K. W. Y. Interaction of goosecoid and brachyury in Xenopus mesoderm patterning. <em>Mech. Dev.</em><strong>65</strong>, 187–196 (1997).
</cite> [<a href="https://doi.org/10.1016/s0925-4773(97)00073-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9256355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Artinger,%20M.,%20Blitz,%20I.,%20Inoue,%20K.,%20Tran,%20U.%20&amp;%20Cho,%20K.%20W.%20Y.%20Interaction%20of%20goosecoid%20and%20brachyury%20in%20Xenopus%20mesoderm%20patterning.%20Mech.%20Dev.65,%20187%E2%80%93196%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Wen, J. et al. Single-cell analysis reveals lineage segregation in early post-implantation mouse embryos. <em>J. Biol. Chem.</em><strong>292</strong>, 9840–9854 (2017).
</cite> [<a href="https://doi.org/10.1074/jbc.M117.780585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5465504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28298438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wen,%20J.%20et%20al.%20Single-cell%20analysis%20reveals%20lineage%20segregation%20in%20early%20post-implantation%20mouse%20embryos.%20J.%20Biol.%20Chem.292,%209840%E2%80%939854%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Farrell, J. A. et al. Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis. <em>Science</em><strong>360</strong>, 10.1126/science.aar3131 (2018).</cite> [<a href="https://doi.org/10.1126/science.aar3131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6247916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29700225/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Mangold, O. Über die Induktionsfähigkeit der verschiedenen Bezirke der Neurula von Urodelen. <em>Die Naturwissenschaften</em><strong>21</strong>, 761–766 (1933).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mangold,%20O.%20%C3%9Cber%20die%20Induktionsf%C3%A4higkeit%20der%20verschiedenen%20Bezirke%20der%20Neurula%20von%20Urodelen.%20Die%20Naturwissenschaften21,%20761%E2%80%93766%20(1933)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Sander, V., Reversade, B. &amp; De Robertis, E. M. The opposing homeobox genes Goosecoid and Vent1/2 self-regulate Xenopus patterning. <em>EMBO J.</em><strong>26</strong>, 2955–2965 (2007).
</cite> [<a href="https://doi.org/10.1038/sj.emboj.7601705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1894760/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17525737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sander,%20V.,%20Reversade,%20B.%20&amp;%20De%20Robertis,%20E.%20M.%20The%20opposing%20homeobox%20genes%20Goosecoid%20and%20Vent1/2%20self-regulate%20Xenopus%20patterning.%20EMBO%20J.26,%202955%E2%80%932965%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Seiliez, I., Thisse, B. &amp; Thisse, C. FoxA3 and goosecoid promote anterior neural fate through inhibition of Wnt8a activity before the onset of gastrulation. <em>Dev. Biol.</em><strong>290</strong>, 152–163 (2006).
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2005.11.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16364286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seiliez,%20I.,%20Thisse,%20B.%20&amp;%20Thisse,%20C.%20FoxA3%20and%20goosecoid%20promote%20anterior%20neural%20fate%20through%20inhibition%20of%20Wnt8a%20activity%20before%20the%20onset%20of%20gastrulation.%20Dev.%20Biol.290,%20152%E2%80%93163%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Blatchley, M. R. &amp; Anseth, K. S. Middle-out methods for spatiotemporal tissue engineering of organoids. <em>Nat. Rev. Bioeng.</em><strong>1</strong>, 329–345 (2023).
</cite> [<a href="https://doi.org/10.1038/s44222-023-00039-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10010248/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37168734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blatchley,%20M.%20R.%20&amp;%20Anseth,%20K.%20S.%20Middle-out%20methods%20for%20spatiotemporal%20tissue%20engineering%20of%20organoids.%20Nat.%20Rev.%20Bioeng.1,%20329%E2%80%93345%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Liu, Y. et al. Embryonic organizer formation disorder leads to multiorgan dysplasia in Down syndrome. <em>Cell Death Dis.</em><strong>13</strong>, 1054 (2022).
</cite> [<a href="https://doi.org/10.1038/s41419-022-05517-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9763398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36535930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20Embryonic%20organizer%20formation%20disorder%20leads%20to%20multiorgan%20dysplasia%20in%20Down%20syndrome.%20Cell%20Death%20Dis.13,%201054%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Chapman, S. C., Collignon, J., Schoenwolf, G. C. &amp; Lumsden, A. Improved method for chick whole-embryo culture using a filter paper carrier. <em>Dev. Dyn.</em><strong>220</strong>, 284–289 (2001).
</cite> [<a href="https://doi.org/10.1002/1097-0177(20010301)220:3&lt;284::AID-DVDY1102&gt;3.0.CO;2-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11241836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chapman,%20S.%20C.,%20Collignon,%20J.,%20Schoenwolf,%20G.%20C.%20&amp;%20Lumsden,%20A.%20Improved%20method%20for%20chick%20whole-embryo%20culture%20using%20a%20filter%20paper%20carrier.%20Dev.%20Dyn.220,%20284%E2%80%93289%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Kuehn, E. et al. Segment number threshold determines juvenile onset of germline cluster expansion in Platynereis dumerilii. <em>J. Exp. Zool. B Mol. Dev. Evol.</em><strong>338</strong>, 225–240 (2022).
</cite> [<a href="https://doi.org/10.1002/jez.b.23100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9114164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34793615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuehn,%20E.%20et%20al.%20Segment%20number%20threshold%20determines%20juvenile%20onset%20of%20germline%20cluster%20expansion%20in%20Platynereis%20dumerilii.%20J.%20Exp.%20Zool.%20B%20Mol.%20Dev.%20Evol.338,%20225%E2%80%93240%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Betancur, P., Bronner-Fraser, M. &amp; Sauka-Spengler, T. Genomic code for Sox10 activation reveals a key regulatory enhancer for cranial neural crest. <em>Proc. Natl. Acad. Sci. USA</em><strong>107</strong>, 3570–3575 (2010).
</cite> [<a href="https://doi.org/10.1073/pnas.0906596107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2840498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20139305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Betancur,%20P.,%20Bronner-Fraser,%20M.%20&amp;%20Sauka-Spengler,%20T.%20Genomic%20code%20for%20Sox10%20activation%20reveals%20a%20key%20regulatory%20enhancer%20for%20cranial%20neural%20crest.%20Proc.%20Natl.%20Acad.%20Sci.%20USA107,%203570%E2%80%933575%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Peer Review file</a><sup> (560.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63154_MOESM3_ESM.pdf</a><sup> (564.2KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (55.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (45.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (30.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (500KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM8_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (4.4KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (96.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371044/bin/41467_2025_63154_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (12KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data generated in this study is available in the Gene Expression Omnibus (GEO) repository under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE228354" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE228354</a>, including raw data as well as processed/normalized files. <a href="#Sec29" class="usa-link">Source data</a> are provided in this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63154-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63154.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371044/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371044/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371044%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371044/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371044/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371044/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841542/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371044/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841542/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371044/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371044/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="KZoCsPBEk9j1MNNfTj8r225pXqgFwzDrZHGnmRuPAcICbVsi5HeC07AIm1q9zems">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
