
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Evaluation of the HumanMethylationEPIC v2.0 Bead Chip Using Low Quality and Quantity DNA Samples - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532D6D8AF20503052D6D0043046F3F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="biolproc">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369268/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Biological Procedures Online">
<meta name="citation_title" content="Evaluation of the HumanMethylationEPIC v2.0 Bead Chip Using Low Quality and Quantity DNA Samples">
<meta name="citation_author" content="Brando Poggiali">
<meta name="citation_author_institution" content="Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark">
<meta name="citation_author" content="Mikkel Eriksen Dupont">
<meta name="citation_author_institution" content="Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark">
<meta name="citation_author" content="Marie-Louise Kampmann">
<meta name="citation_author_institution" content="Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark">
<meta name="citation_author" content="Athina Vidaki">
<meta name="citation_author_institution" content="Department of Clinical Genetics, Maastricht University Medical Center, Maastricht, The Netherlands">
<meta name="citation_author_institution" content="Department of Genetics &amp; Cell Biology, GROW and CARIM Institutes, Maastricht University, Maastricht, The Netherlands">
<meta name="citation_author" content="Vania Pereira">
<meta name="citation_author_institution" content="Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark">
<meta name="citation_author" content="Claus Børsting">
<meta name="citation_author_institution" content="Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark">
<meta name="citation_author" content="Jacob Tfelt-Hansen">
<meta name="citation_author_institution" content="Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark">
<meta name="citation_author_institution" content="Department of Cardiology, The Heart Centre, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark">
<meta name="citation_author" content="Jeppe Dyrberg Andersen">
<meta name="citation_author_institution" content="Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="27">
<meta name="citation_firstpage" content="34">
<meta name="citation_doi" content="10.1186/s12575-025-00292-3">
<meta name="citation_pmid" content="40841873">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369268/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369268/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369268/pdf/12575_2025_Article_292.pdf">
<meta name="description" content="The HumanMethylationEPIC v2.0 BeadChip (EPIC v2.0) microarray is a widely used tool for genome-wide DNA methylation (DNAm) analysis, designed for high-quality human DNA with a recommended input of 250 ng. However, in clinical and forensic settings, ...">
<meta name="og:title" content="Evaluation of the HumanMethylationEPIC v2.0 Bead Chip Using Low Quality and Quantity DNA Samples">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The HumanMethylationEPIC v2.0 BeadChip (EPIC v2.0) microarray is a widely used tool for genome-wide DNA methylation (DNAm) analysis, designed for high-quality human DNA with a recommended input of 250 ng. However, in clinical and forensic settings, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369268/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12369268">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12575-025-00292-3"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12575_2025_Article_292.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369268%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12369268/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12369268/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369268/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-biolproc.png" alt="Biological Procedures Online logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Biological Procedures Online" title="Link to Biological Procedures Online" shape="default" href="https://biologicalproceduresonline.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Biol Proced Online</button></div>. 2025 Aug 21;27:34. doi: <a href="https://doi.org/10.1186/s12575-025-00292-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12575-025-00292-3</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Biol%20Proced%20Online%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Biol%20Proced%20Online%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Biol%20Proced%20Online%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Biol%20Proced%20Online%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Evaluation of the HumanMethylationEPIC v2.0 Bead Chip Using Low Quality and Quantity DNA Samples</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Poggiali%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Brando Poggiali</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Brando Poggiali</span></h3>
<div class="p">
<sup>1</sup>Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Poggiali%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Brando Poggiali</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dupont%20ME%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mikkel Eriksen Dupont</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mikkel Eriksen Dupont</span></h3>
<div class="p">
<sup>1</sup>Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dupont%20ME%22%5BAuthor%5D" class="usa-link"><span class="name western">Mikkel Eriksen Dupont</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kampmann%20ML%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Marie-Louise Kampmann</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Marie-Louise Kampmann</span></h3>
<div class="p">
<sup>1</sup>Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kampmann%20ML%22%5BAuthor%5D" class="usa-link"><span class="name western">Marie-Louise Kampmann</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vidaki%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Athina Vidaki</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Athina Vidaki</span></h3>
<div class="p">
<sup>3</sup>Department of Clinical Genetics, Maastricht University Medical Center, Maastricht, The Netherlands </div>
<div class="p">
<sup>4</sup>Department of Genetics &amp; Cell Biology, GROW and CARIM Institutes, Maastricht University, Maastricht, The Netherlands </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vidaki%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Athina Vidaki</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pereira%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Vania Pereira</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Vania Pereira</span></h3>
<div class="p">
<sup>1</sup>Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pereira%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vania Pereira</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C3%B8rsting%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Claus Børsting</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Claus Børsting</span></h3>
<div class="p">
<sup>1</sup>Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C3%B8rsting%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Claus Børsting</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tfelt-Hansen%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jacob Tfelt-Hansen</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jacob Tfelt-Hansen</span></h3>
<div class="p">
<sup>1</sup>Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark </div>
<div class="p">
<sup>2</sup>Department of Cardiology, The Heart Centre, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tfelt-Hansen%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jacob Tfelt-Hansen</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Andersen%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jeppe Dyrberg Andersen</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jeppe Dyrberg Andersen</span></h3>
<div class="p">
<sup>1</sup>Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Andersen%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeppe Dyrberg Andersen</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen, Denmark </div>
<div id="Aff2">
<sup>2</sup>Department of Cardiology, The Heart Centre, Copenhagen University Hospital, Rigshospitalet, Copenhagen, Denmark </div>
<div id="Aff3">
<sup>3</sup>Department of Clinical Genetics, Maastricht University Medical Center, Maastricht, The Netherlands </div>
<div id="Aff4">
<sup>4</sup>Department of Genetics &amp; Cell Biology, GROW and CARIM Institutes, Maastricht University, Maastricht, The Netherlands </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 28; Accepted 2025 Jul 7; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12369268  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841873/" class="usa-link">40841873</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">The HumanMethylationEPIC v2.0 BeadChip (EPIC v2.0) microarray is a widely used tool for genome-wide DNA methylation (DNAm) analysis, designed for high-quality human DNA with a recommended input of 250 ng. However, in clinical and forensic settings, DNA samples may be of low quality and/or quantity (highly fragmented and/or available in low amounts). This study assessed the performance of the EPIC v2.0 on DNA samples with various combinations of average DNA fragment size (350, 230, 165, and 95 bp) and DNA input amount (100, 50, 20, and 10 ng), compared to a paired control sample analyzed under optimal conditions (high-quality DNA and 250 ng DNA input).</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">The best performance was obtained for samples with average DNA fragment size of 350 bp and 100 ng DNA input (~ 90% probe detection rate, <em>r</em> = 0.995, and median absolute beta value differences|Δ<em>β</em>| = 0.012 when compared with the control sample). Samples with lower average DNA fragment sizes and DNA input amount performed worse, with the lowest probe detection rate (~ 43%), <em>r</em> = 0.946, and the highest|Δ<em>β</em>| (0.038). Samples with average DNA fragment sizes of 95 bp and those with 165 bp at 10 ng DNA input failed to pass sample quality control (QC). CpG sites with intermediate DNAm values (<em>β</em> = 0.1–0.9) showed higher|Δ<em>β</em>| than the extreme DNAm values (<em>β</em> = 0–0.1, and <em>β</em> = 0.9–1). Finally, we assessed an application of DNAm by performing epigenetic age analysis, and observed mean absolute errors (MAEs) below 10 years for 350 bp samples across four epigenetic clocks.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">Both DNA fragment size and DNA input amounts affect DNAm analysis on the EPIC v2.0, with the investigated DNA fragment size having a greater impact than the investigated DNA input amount. DNAm measurements were achieved with the EPIC v2.0 microarray down to an average DNA fragment size of 165 bp and a 20 ng DNA input. Highly fragmented DNA (95 bp) did not result in usable DNAm analysis as all samples failed QC. Overall, our study demonstrates the potential and limitations of EPIC v2.0 microarray with low quality and quantity DNA samples.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12575-025-00292-3.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> DNA methylation, EPIC v2.0 microarray, Fragmented DNA, Low-input DNA, Clinical epigenetics, Forensic genetics, Epigenetic age, Degraded DNA</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par4">DNA methylation (DNAm) is an epigenetic modification that regulates gene expression and is implicated in various biological processes and diseases [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. DNAm is mainly found on cytosine bases that are followed by a guanine (CpG sites). Various technologies can measure DNAm, with whole-genome bisulfite sequencing and DNAm microarrays being the most widely used for genome-wide investigations. Microarray-based approaches offer a good compromise between genome-wide coverage and affordability [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Several microarrays are available for DNAm analysis, including the Agilent SurePrint Methylation Microarray, Affymetrix GeneChip Human Promoter Array, and Illumina HumanMethylationEPIC BeadChip microarray. These arrays have been used to generate data from large cohorts such as The Cancer Genome Atlas (TCGA) [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>] and the Accessible Resource for Integrated Epigenomics Studies (ARIES) [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>] to investigate environmental exposures [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>], ageing [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>], and various diseases [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>].</p>
<p id="Par5">Illumina developed multiple versions of the Infinium Human Methylation BeadChip microarray over time. The latest version, the HumanMethylationEPIC v2.0 BeadChip (EPIC v2.0), targets ~ 935,000 CpG sites, with 77.63% homology to its predecessor (EPIC v1.0) and over 200,000 new CpGs located in enhancers and open-chromatin regions [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. This new and improved release contains fewer probes subjected to cross-reactivity and sequence polymorphism issues and introduces probes targeting cancer somatic mutations [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. The EPIC v2.0 microarray requires high-quality DNA samples, and high DNA quantities are recommended (250 ng input). In a clinical and forensic setting, it may be essential to be able to work on fragmented and/or low-quantity DNA material. For example, circulating free DNA (cfDNA), also known as cell-free DNA, consists of small circulating fragments of DNA (median size of 167 bp) present in plasma, saliva, urine, and seminal fluid [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. cfDNA is released by cells due to cell death and can be considered as a marker of inflammation and tissue injury. It has been used as a noninvasive diagnostic and disease-monitoring tool in oncology [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] and autoinflammatory conditions [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Additionally, cfDNA methylation patterns could be used to localize the tissue of origin [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>] or to detect cancer [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Another source of degraded DNA in clinical diagnostics is formalin-fixed, paraffin-embedded (FFPE) tissue, where DNA is often fragmented and present in low quantities, making DNAm analysis challenging [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p>
<p id="Par6">In forensic genetics, researchers aim to extract as much information as possible from minute DNA traces, including the estimated chronological age of a donor using DNAm. Illumina methylation microarrays have been explored as a potential tool for DNAm age prediction [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. However, forensic traces are often exposed to harsh environmental conditions, leading to poor DNA quality and low yields. This makes the applicability of DNAm arrays in forensic casework challenging if the recommended input of 250 ng DNA must be adhered to. Also, even more degraded samples could be found in ancient DNA material, where DNA is exposed to harsher conditions for much longer periods of time. In recent years, the interest in DNAm analysis for ancient DNA has grown [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>], with DNAm array technologies successfully applied to infer age at death and cell composition in mummified human remains from the eighteenth century, using EPIC microarray DNA inputs of 320 and 220 ng [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>].</p>
<p id="Par7">These fields would benefit from a detailed investigation into the potential and limitations of the EPIC v2.0 microarray technology when applied to DNA samples of low quality and quantity. Previously, DNA input of less than 250 ng yielded accurate results for the EPIC microarrays [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>], but so far there is no study systematically assessing how the combination of low DNA quality (fragmented DNA) and quantity (input amount) influences the reproducibility, precision, accuracy, and applicability of the EPIC v2.0 microarray. In this study, we measured genome-wide DNAm in duplicate using the EPIC v2.0 microarray and samples with different average DNA fragment sizes (350, 230, 165, and 95 bp) and input amounts (100, 50, 20, and 10 ng). We assessed data quality, probe detection rate, reproducibility, precision, and absolute methylation (beta) value differences (|Δ<em>β</em>|) compared to a control DNA sample. Additionally, we assessed the impact of these parameters on a key DNAm application: DNAm age prediction. Finally, we compared a recently released DNAm detection calling method, the ELBAR algorithm [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>], which was specifically designed for suboptimal DNA input samples, against the previous established pOOBAH algorithm [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>] developed for high quality DNA.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Study Design and DNA Sample Preparation</h3>
<p id="Par8">Figure <a href="#Fig1" class="usa-link">1</a> shows the design of the study. Peripheral blood from a male individual was collected in a EDTA whole blood collection tube and stored at − 80 °C until DNA extraction. Genomic DNA extraction was performed using the QIAamp DNA Blood Mini Kit (Qiagen, Hilden, Germany) according to the manufacturer’s guidelines. Extracted genomic DNA was quantified using the Qubit™ dsDNA HS Assay Kit (Thermo Fisher Scientific, Waltham, MA, USA) following the manufacturer’s recommendations. Qubit measurements were used to prepare a DNA solution containing 2000 ng in 210 µL. This DNA sample was equally split into four different tubes. Each tube was subjected to DNA fragmentation with the Covaris S220 instrument (Covaris, Woburn, MA, USA) for time periods of 10, 14, 21, and 113 min to achieve DNA fragment sizes of ∼350, ∼230, ∼165, and ∼95 bp, respectively. To confirm the obtained fragment sizes, fragmented DNA samples were analyzed using the High Sensitivity DNA Kit for 2100 Bioanalyzer Systems (Agilent Technologies, Santa Clara, CA, USA), according to manufacturer’s instructions. Average DNA fragment sizes were calculated using the 2100 Expert Software package (Agilent Technologies, Santa Clara, CA, USA) and setting a range from 38 to 9330 bp. Degradation Indexes (DIs) were obtained by quantifying fragmented DNA samples with Quantifiler™ Trio DNA Quantification kit (Thermo Fisher Scientific, Waltham, MA, USA) using the Applied Biosystems™ 7500 real-time PCR system (Thermo Fisher Scientific, Waltham, MA, USA) according to the manufacturer’s protocol. DIs were plotted against average DNA fragment size using ggplot2 R-library (version 3.4.4) [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. From each of the fragmented DNA extracts, we prepared four different dilutions with DNA amounts of 100, 50, 20, and 10 ng in duplicate.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369268_12575_2025_292_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d34c/12369268/2a3fef857b1a/12575_2025_292_Fig1_HTML.jpg" loading="lazy" id="d33e370" height="1016" width="746" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Study design. Whole blood was collected from one individual. The extracted DNA was fragmented to obtain four target DNA fragment sizes (350, 230, 165, and 95 bp). The different fragment size samples were diluted to reach total DNA input amounts of 100, 50, 20, and 10 ng. All samples were analyzed in duplicates (x2). A non-fragmented DNA sample with 250 ng DNA input amount was used as a control for the analysis</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">DNA Methylation Profiling</h3>
<p id="Par10">Genomic DNA samples were bisulfite-converted using the EZ DNA Methylation™ Kit (Zymo Research, Irvine, CA, USA) according to the manufacturer’s instructions, with a final elution volume of 10 µl. Genome-wide DNAm levels were measured using the Infinium MethylationEPIC kit v2.0 (Illumina, San Diego, CA, USA), following the manufacturer’s protocol. Samples were randomly distributed on the EPIC bead chips and chip positions (rows 1–8). The arrays were scanned using the iScan™ System (Illumina, San Diego, CA, USA) to obtain raw Intensity Data files (.<em>idat</em>).</p>
<p id="Par11">Data analysis was carried out in the R statistical environment (version 4.3.1) [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] with packages from the Bioconductor project (version 3.18). Raw.<em>idat</em> files were imported into R and processed using the Sensible Step-wise Analysis of DNA Methylation BeadChip (<em>SeSAMe</em>) package, version 1.20.0 [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. DNAm levels (<em>β</em>-values), ranging from 0 to 1, and quality metrics were obtained using the <em>SeSAMe</em> function <em>openSesame()</em>, which provides end-to-end preprocessing that converts IDATs to DNA methylation levels. In brief, the selected preprocessing functions <em>qualityMask()</em>, <em>inferInfiniumIChannel()</em>, <em>dyeBiasNL()</em>, <em>pOOBAH()</em>, and <em>noob()</em> masked the probes of poor-quality design (32,896 probes), inferred the channel of Infinium-I probes, corrected for dye bias, removed probes with low-quality signal (detection <em>p</em>-value ≥ 0.05), and implemented a background subtraction. The analysis using ELBAR (Eliminating BAckground-dominated Reading) algorithm was performed using preprocessing functions <em>qualityMask()</em>, <em>inferInfiniumIChannel()</em>, <em>dyeBiasNL()</em>, and <em>ELBAR()</em>. Quality control (QC) was performed using the standard <em>SeSAMe</em> QC quality metrics on raw and preprocessed data. More specifically, we assessed the relationship between red and green background intensities, median intensities, the ratio between red and green median intensities, probe detection rate (calculated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d34c/12369268/4b1ab100687b/d33e438.gif" loading="lazy" id="d33e438" alt="Inline graphic"></span>), <em>β</em>-value distributions, and Principal Component Analysis (PCA). Probe detection rate was calculated considering cg, ch, and rs probes, excluding from the calculation masked, control, and negative probes, using a total number of 903,335 probes. PCA was used to assess the overall variability of the data. Samples that did not meet the following criteria in the raw and preprocessed data were removed from downstream analysis: (i) Median signal intensity &lt; 2,000; (ii) Ratio of red and green median signal intensity &lt; 0.5 or &gt;2; (iii) deviation from bimodal <em>β</em>-value distribution shape. Data from the 65 single nucleotide polymorphism (SNP) probes included in the EPIC v2.0 microarray were used to verify the identity of the sample donor and to assess the impact of degraded samples on EPIC v2.0 genotyping. All plots were generated using the ggplot2 R-library (version 3.4.4) [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>].</p></section><section id="Sec5"><h3 class="pmc_sec_title">DNAm Data Analysis</h3>
<p id="Par12">To investigate the impact of fragmented and low-input DNA samples (hereafter referred to as degraded samples) on precision, reproducibility, and accuracy of the EPIC v2.0 microarray, we calculated: (1) the Pearson correlation coefficient (<em>r</em>) between sample replicates, (2) <em>r</em> between degraded samples and control sample, and (3) the absolute differences in beta values (<em>|∆β|</em>) compared to the control sample and between sample replicates. To investigate the precision of the EPIC v2.0 microarray in degraded samples, we calculated within-replicate correlations, r, using the shared CpGs between replicate pairs. To assess the EPIC v2.0 microarray reproducibility we calculated pairwise correlations between degraded samples and the control sample using whole-array <em>r</em> on β-values with the <em>cor()</em> function in R. To ensure comparability of <em>r</em> between degraded samples, missing values were removed using the <em>na.omit()</em> function, resulting in a total of 202,439 common CpGs. For each combination of DNA fragment size and input amount, average DNAm values between the two replicates were used to calculate <em>r</em>. <em>P</em>-values were adjusted for multiple testing using the <em>r</em> function <em>p.adjust()</em> using the Benjamini-Hochberg method [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] to control the False Discovery Rate (FDR) at a threshold of <em>p</em> = 0.05. Additionally, we calculated the absolute differences in beta values (<em>|∆β|</em>) of the 202,439 common CpGs between the average DNAm value obtained for the degraded DNA samples that passed QC and the one of the control sample. We calculated median, interquartile range (IQR), and percentage of CpG sites with <em>|∆β|</em> ≥ 0.05 and 0.1 from <em>|∆β|</em> distributions. Additionally, we investigated <em>|∆β|</em> in CpGs using <em>β</em>-value intervals of 0.1 (0.0-0.1, 0.1–0.2, 0.2–0.3, 0.4–0.5, 0.5–0.6, 0.6–0.7, 0.7–0.8, 0.8–0.9, and 0.9–1.0). Within-replicate <em>|∆β|</em> statistics (median, IQR, <em>|∆β|</em> ≥ 0.05 and 0,1) were also calculated using the shared CpGs between the replicate pair.</p>
<p id="Par13">Information related to the type of probe (type I and II) and genomic location of each CpG site (island, shore, shelf, and open sea) was extracted from the <em>IlluminaHumanMethylationEPICv2anno.20a1.hg38</em> Bioconductor package [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Gene location (gene body, intergenic, and promoter regions) for each CpG was obtained from <em>EnsDb.Hsapiens.v111</em> Bioconductor package [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>].</p></section><section id="Sec6"><h3 class="pmc_sec_title">Age Prediction of Degraded Samples</h3>
<p id="Par14">To evaluate the impact of DNA fragment size and input amount on a DNAm application, we performed DNAm-based age prediction for all degraded samples using the function <em>DNAmAge()</em> from <em>methylclock</em> Bioconductor package (version 1.8.0) [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. We predicted the chronological age of the samples’ donor using four different epigenetic clocks: the multi-tissue age estimator developed by Horvarth et al. [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] (Horvath clock), which contains 353 CpGs; the skinHorvath clock [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>], with 391 CpGs; the BLUP clock [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>], containing 319,607 CpGs; and the EN clock [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>], with 514 CpGs. To assess age prediction accuracy on degraded samples, we calculated the mean absolute error (MAE) by averaging the absolute differences between predicted and chronological ages across the two sample replicates.</p></section></section><section id="Sec7"><h2 class="pmc_sec_title">Results</h2>
<p id="Par15">We investigated the performance of the Infinium MethylationEPIC v2.0 microarray (EPIC v2.0) using a combination of fragmented (average sizes of 350, 230, 165, and 95 bp) and low-input (100, 50, 20, 10 ng) DNA material. Supplementary Fig. <a href="#MOESM2" class="usa-link">1</a> shows the obtained fragment size distributions of the four DNA sample sizes. These distributions are represented by Gaussian-like curves with elongated tails extending towards longer DNA fragments (right skewed), with shorter DNA fragment samples exhibiting a narrower distribution. The obtained DIs for the DNA samples with average fragment sizes of 350, 230, 165, and 95 bp were 2.09, 3.44, 8.88, and 8008.45, respectively. Supplementary Fig. <a href="#MOESM2" class="usa-link">2</a> illustrates the relationship between the average DNA fragment size and DI.</p>
<section id="Sec8"><h3 class="pmc_sec_title">Quality Control of Degraded DNA Methylation Data</h3>
<p id="Par16">It was key to carry out a thorough QC procedure of the Illumina EPIC v2.0 microarray data to ensure reliable results given that samples of suboptimal DNA quality and quantity were employed.</p>
<p id="Par17">Illumina EPIC v2.0 microarray utilizes beads with multiple copies of 50 bp oligonucleotide probes targeting specific loci in the genome. Once a DNA fragment hybridizes with its complementary probe, single-base extension (SBE) incorporates labeled nucleotides (green or red), which emits a signal. The intensities of the signals from probes targeting a given CpG site provide information about its DNAm level. Non-specific binding of negative control probes and fluorescence intrinsically associated with the microarray surface generate background signal intensities, which could interfere with the analysis if not properly removed. Figure <a href="#Fig2" class="usa-link">2</a>A shows the relationship between the two mean background signal intensities (green and red) across all samples in the raw data. Degraded samples with average DNA fragment sizes of 350 bp, 230 bp, and 165 bp (except those with a 10 ng input) showed higher red than green background signal intensities. This imbalance was corrected after preprocessing with the pOOBAH algorithm (Supplementary Fig. <a href="#MOESM2" class="usa-link">3</a>), after which these samples clustered closely with the high-quality DNA control (grey dot). In contrast, degraded samples with an average DNA fragment size of 95 bp and 165 bp with 10 ng DNA input showed low background signal intensity before and after preprocessing (Fig. <a href="#Fig2" class="usa-link">2</a>A and Supplementary Fig. <a href="#MOESM2" class="usa-link">3</a>), with at least one background signal (green or red) close to zero after preprocessing, indicating potential issues. Figure <a href="#Fig2" class="usa-link">2</a>B shows the mean signal intensities of all probes across all samples in the raw data. The control sample exhibited a mean signal intensity of approximately 6,000, followed by a progressive decline in degraded samples with lower DNA quality and quantity. Degraded samples with average DNA fragment sizes of 95 bp and 165 bp with 20 ng and 10 ng input had mean signal intensities lower than 2,000, falling below the QC threshold for passing samples. However, after preprocessing, samples with average DNA fragment size of 165 bp and 20 ng input showed improved mean signal intensity (Supplementary Fig. <a href="#MOESM2" class="usa-link">4</a>A). The low mean red and green background signal intensity observed for degraded samples with a DNA fragment size of 95 bp and 165 bp with 10 ng DNA input was desirable, however, the mean signal intensity was low, which indicates poor DNA hybridization. Another important QC metric is the ratio of median red to green signal intensities, that represents the relative balance between the two fluorescent dye channels (green and red) used in the microarray. A balanced ratio (close to 1) indicates good microarray performance, while an imbalanced ratio suggests potential technical issues. In the raw data, this ratio ranged between 0.5 and 2 (QC thresholds) for all degraded samples (Fig. <a href="#Fig2" class="usa-link">2</a>C). After preprocessing with pOOBAH, samples with 95 bp and 165 bp fragment sizes at 10 ng input deviated notably from a ratio of 1, again suggesting potential technical issues (Supplementary Fig. <a href="#MOESM2" class="usa-link">4</a>B). Additionally, we assessed the genotyping performance of the 65 SNP probes included in the EPIC v2.0 microarray for sample matching (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). When comparing the SNP genotypes of the degraded samples with the control sample, we observed an increase in discordant genotype calls with decreasing length of DNA fragment size and lower DNA input. Particularly, more than half of the SNP genotypes were discordant in samples with an average fragment size of 165 bp at 10 ng input, as well as samples with DNA fragment size of 95 bp.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369268_12575_2025_292_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d34c/12369268/bfb0f597f3dc/12575_2025_292_Fig2_HTML.jpg" loading="lazy" id="d33e632" height="1015" width="742" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Quality control (QC) of the obtained EPIC v2.0 data obtained. <strong>A</strong> Correlation of the mean (red and green) background signal intensities in the raw data. Diagonal dashed line represents a ratio of 1. <strong>B</strong> Mean signal intensity in raw data. The red dashed line represents the pass/fail QC threshold. <strong>C</strong> Ratio of red/green median intensity in the raw data. The red dashed lines represent the pass/fail QC thresholds. <strong>D</strong> Principal Component analysis (PCA) based on 6,160 shared CpGs among all samples (preprocessed with POOBAH algorithm), including samples that failed QC. <strong>E</strong> PCA based on 202,439 shared CpGs among samples that passed QC (preprocessed with POOBAH algorithm). <strong>F</strong> Probe detection rate and number of probes passing filters in degraded samples passing QC, preprocessed with the pOOBAH algorithm</p></figcaption></figure><p id="Par19">We performed PCA using beta values of the 6,160 CpGs that were shared among all samples (including the samples that failed QC) and preprocessed with the pOOBAH algorithm (Fig. <a href="#Fig2" class="usa-link">2</a>D). The aim was to assess potential outliers, batch effect, and degradation patterns (DNA fragment size and input amount) in the data. We observed that the control sample and the samples with average fragment sizes higher than 95 bp (except for the combination of 10 ng input and average fragment size of 165 bp DNA) clustered together, while the other samples were scattered across the PCA plane. PC1 and PC2 represented 44.41% and 19.12% of the total variance in the data, respectively, indicating that average DNA fragment size and input amount were the primary drivers of the variance. We also examined the <em>β-</em>value distribution in all samples. Degraded samples that clustered together with the control sample showed the expected bimodal <em>β-</em>value distribution shape (Supplementary Fig. <a href="#MOESM2" class="usa-link">5</a>). In contrast, degraded samples with average fragment sizes of 95 bp and 165 bp (with a DNA input of 10 ng) presented a deviation from the bimodal <em>β</em>-value distribution shape in both replicates. Based on these results, we excluded all DNA samples with an average DNA fragment size of 95 bp, and DNA samples with an average DNA fragment size of 165 bp and a DNA input of 10 ng. Thus, 23 out of 33 samples, including both control and degraded samples, passed QC and were included in subsequent analyses. Looking at the corresponding PCA plot considering the 202,439 common CpGs (Fig. <a href="#Fig2" class="usa-link">2</a>E), degraded samples with an average DNA fragment size of 350 bp clustered together with the control sample, whereas samples with DNA fragment size of 230 bp and 165 bp were more spread out, indicating a different degradation pattern. PC1 and PC2 represented 20.25% and 8.94% of the total variance, respectively. Finally, considering the probe detection rate in samples passing QC and preprocessed with pOOBAH, 903,211 probes (99.9%) passed filtering for the control sample. A decrease in probe detection rate was observed with both shorter DNA fragment size and lower DNA input (Fig. <a href="#Fig2" class="usa-link">2</a>F). Degraded samples with an average DNA fragment size of 350 bp and 100 ng DNA input had a probe detection rate of 90.3%, corresponding to 815,754 probes passing filtering. The most degraded samples to pass QC (average DNA fragment size of 165 bp and 20 ng DNA input) had the lowest probe detection rate (41.86.%) and the smallest number of probes passing filtering (378,116). We also investigated the failing probes across the different DNA fragment sizes and DNA input amounts. <em>The SeSAMe</em> pipeline masked 32,896 poorly designed probes, which were therefore excluded from this analysis. A total of 6,066 failing probes were common between all the samples passing QC (including the control sample), however, when considering only the degraded samples, we obtained 57,910 commonly failing probes. We calculated the number of common failing probes for each DNA fragment size and input amount. A total of 71,820, 132,622, and 173,396 common failing probes were detected in samples with average DNA fragment sizes of 350, 230, and 165 bp, respectively (Supplementary Fig. <a href="#MOESM2" class="usa-link">6</a>A). Regarding DNA input amounts, we obtained 67,563, 93,651, 130,661, and 166,795 shared failing probes for input amounts of 100, 50, 20, and 10 ng, respectively (Supplementary Fig. <a href="#MOESM2" class="usa-link">6</a>B). Lists with the names of the commonly failing probes across all samples, all degraded samples, DNA fragment sizes, and DNA input amounts are shown in Supplementary File 1.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Degraded DNA Methylation Data Correlations and Absolute Differences</h3>
<p id="Par20">To investigate the impact of low quality and quantity DNA samples on EPIC v2.0 microarray, we calculated the within-replicate correlation (precision) for each combination of average DNA fragment size and input amount preprocessed with the pOOBAH algorithm (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Degraded samples with an average DNA fragment size of 350 bp and DNA input of 100 ng had within-replicate <em>r</em> value of 0.990 (804,732 shared CpGs). We observed a progressive decrease in <em>r</em> values and number of shared CpGs with shorter DNA fragment sizes and lower DNA inputs. The lowest within-replicate <em>r</em> value (0.851) and number of shared CpGs (259,031) was observed in degraded samples with average DNA fragment size of 165 bp and DNA input of 20 ng. All correlation tests had an adjusted p-value below 0.05. We also assessed the correlation between degraded samples and control sample (reproducibility) for each combination of average DNA fragment size and input amount. Figure <a href="#Fig3" class="usa-link">3</a> shows the <em>r</em> values obtained from 202,439 common CpGs among all samples passing QC and the control sample. First, the correlation between the control sample and sample with average DNA fragment size of 350 bp and DNA input of 100 ng was <em>r</em> = 0.995. For the rest of the degraded samples, we observed a progressive decrease in <em>r</em> values with shorter DNA fragment sizes and lower DNA inputs. The lowest correlation value, <em>r</em> = 0.946, was observed in sample with an average DNA fragment size of 165 bp and DNA input of 20 ng. A drop in <em>r</em> values (from 0.995 to 0.989) was observed in degraded samples with DNA fragment size of 350 bp when reducing the DNA input to 20 ng. Degraded samples with DNA fragment size of 165 bp showed a larger <em>r</em> value drop (from 0.971 to 0.946) (Supplementary Fig. <a href="#MOESM2" class="usa-link">7</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369268_12575_2025_292_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d34c/12369268/cd8b96ec8e02/12575_2025_292_Fig3_HTML.jpg" loading="lazy" id="d33e737" height="288" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>EPIC v2.0 performance on degraded samples. <strong>A</strong> Pearson correlation coefficients (<em>r</em>) based on mean <em>β</em>-value of 202,439 common CpGs between control sample and degraded samples that passed QC. <strong>B</strong> Absolute <em>β</em>-value differences (|∆<em>β</em>|) distributions based on 202,439 common CpGs between control and degraded samples. Vertical black lines represent median values of the distributions</p></figcaption></figure><p id="Par22">Furthermore, we aimed to estimate the deviation of <em>β</em>-values from the control sample (accuracy) in all degraded samples that passed QC. Quantifying the impact of sample degradation on DNAm measurement could provide a criterion for determining whether to proceed with DNAm analysis. Therefore, we calculated the absolute difference of <em>β</em>-values (<em>|∆β|</em>) between the control and degraded samples, using mean <em>β</em>-values between sample replicates that passed QC. A distribution of <em>|∆β|</em> is generated for each pair of degraded samples (Fig. <a href="#Fig4" class="usa-link">4</a>). All distributions showed a peak close to 0 and elongated tails towards higher values of <em>|∆β|</em>. We observed a lower median <em>|∆β|</em> and percentage of probes with a <em>|∆β|</em> ≥ 0.05 and 0.10 for samples with larger average DNA fragment sizes and higher DNA inputs (Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). Samples with average DNA fragment size of 350 bp and DNA input amount of 100 ng had a median <em>|∆β|</em> of 0.016 and 1.95% of probes with a <em>|∆β|</em> ≥ 0.10. While samples with average DNA fragment size of 165 bp and DNA input amount of 20 ng had a median <em>|∆β|</em> of 0.051 and 31.70% of probes with a <em>|∆β|</em> ≥ 0.10. We further investigated <em>|∆β|</em> in <em>β</em>-value intervals of 0.1 to comprehend whether EPIC v2.0 measurements of CpG sites with intermediate <em>β</em>-values were more affected by low quality and quantity DNA samples. We observed higher <em>|∆β|</em> values for intermediate <em>β</em>-values intervals (0.1–0.9) compared to the extreme <em>β</em>-values intervals (0.0–0.1 and 0.9–1.0) in all DNA fragment sizes investigated (Supplementary Fig. <a href="#MOESM2" class="usa-link">8</a>). Samples with shorter average DNA fragment sizes and lower DNA inputs exhibited higher <em>|∆β|</em> in intermediate <em>β</em>-value intervals, while extreme <em>β</em>-values intervals were less affected by the quality and quantity of the DNA sample. We also calculated <em>|∆β|</em> between sample replicates and estimated median <em>|∆β|</em> and percentage of probes with a <em>|∆β|</em> ≥ 0.05 and 0.10. We observed an increase of these parameters with shorter average DNA fragment sizes and lower DNA inputs (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). Degraded samples with average DNA fragment size of 350 bp and DNA input of 100 ng had median <em>|∆β</em>| of 0.018 and percentage of probes with <em>|∆β|</em> ≥ 0.10 of 0.068 (804,732 shared CpGs). While for the sample with average DNA fragment size of 165 bp and DNA input of 20 ng, the median <em>|∆β</em>| was 0.03 and 0.23% of probes had <em>|∆β</em>| ≥ 0.10 (259,031 shared CpGs).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369268_12575_2025_292_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d34c/12369268/6623f943a155/12575_2025_292_Fig4_HTML.jpg" loading="lazy" id="d33e850" height="197" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Age prediction accuracies. Mean absolute errors (MAEs) of predicted ages from degraded samples using four epigenetic clocks (BLUP, EN, Horvath, and skinHorvath clock)</p></figcaption></figure><p id="Par24">Then, we investigated the type of probes and target regions that were more robust in low quality and quantity DNA samples. Illumina EPIC v2.0 microarray uses two types of oligonucleotide probes, Type I probes target methylated and unmethylated epialleles of a CpG site using two different probes, while Type II probes use a single probe able to target both methylated and unmethylated epialleles. We calculated the proportion of Type I and II probes in probes passing filtering for each DNA sample that passed QC. The relative proportion of Type I probes that passed filtering increased in samples with shorter average DNA fragment sizes and lower DNA input (Supplementary Fig. <a href="#MOESM2" class="usa-link">9</a>A), which indicated that these probes were more efficient than the Type II probes. We also assessed the genomic (CpG island, shore, shelf, and open sea regions) and gene (promoters, gene body, and intergenic regions) distribution of probes passing filtering (Supplementary Fig. <a href="#MOESM2" class="usa-link">9</a>B, C). We observed an increase in the proportion of successfully typed probes targeting CpG islands and gene promoters in samples with shorter average DNA fragment sizes and lower DNA input when compared to the control sample. To determine whether this enrichment was simply due to the higher retention of Type I probes (which are inherently enriched in CpG islands and promoter regions), we stratified the analysis by probe type. This revealed that the increase in the proportion of probes passing filtering targeting CpG islands and promoters was evident within both Type I and Type II probes (Supplementary Figs. <a href="#MOESM2" class="usa-link">10</a> and <a href="#MOESM2" class="usa-link">11</a>).</p></section><section id="Sec10"><h3 class="pmc_sec_title">Age Prediction of Degraded Samples</h3>
<p id="Par27">Finally, we wanted to test the impact of DNA fragment sizes and DNA inputs on a DNAm application. We chose to estimate DNAm age because it is a potential analysis of interest in the medical and forensic fields. We calculated MAE for the control and degraded samples using four epigenetic clocks: BLUP (319,607 CpGs), EN (514 CpGs), Horvath (353 CpGs), and skinHorvath (391 CpGs) clocks. The control sample had an absolute error (AE) of 0.19, 1.83, 8.77, and 0.21 years for BLUP, EN, Horvath, and skinHorvath clocks, respectively. For the degraded samples, we obtained higher MAE in all samples, especially for the BLUP, EN, and skinHorvath clocks (Fig. <a href="#Fig5" class="usa-link">5</a>). We observed a general increase in MAE with shorter DNA fragment size and lower DNA input amounts in all epigenetic clocks tested, indicating an impact of sample degradation on DNAm age prediction analysis. Samples with average DNA fragment size of 350 bp had a MAE &lt; 10 years for all DNA inputs and epigenetic clocks, except for the age predicted with the Horvath clock in 20 ng DNA input samples. In contrast, degraded samples with average DNA fragment size of 165 had MAE &gt; 10 years, except for age predicted from 20 ng DNA input samples with the BLUP clock and the 50 ng DNA input sample with the skinHorvath clock. Supplementary Table <a href="#MOESM1" class="usa-link">5</a> shows the number of missing CpGs in each epigenetic clock across all samples that passed QC. As expected, the number of missing CpGs in the four epigenetic clocks increased with shorter DNA fragment size and lower DNA input amounts.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369268_12575_2025_292_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d34c/12369268/332e2408ddfd/12575_2025_292_Fig5_HTML.jpg" loading="lazy" id="d33e910" height="1016" width="760" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of pOOBAH and ELBAR algorithm performance on degraded samples passing QC. <strong>A</strong> Probe detection rate and number of probes passing filters. <strong>B</strong> Pearson correlation coefficients (<em>r</em>) between degraded samples and the control, calculated using both algorithms. <strong>C</strong> Median absolute <em>β</em>-value differences (|∆<em>β</em>|) between degraded samples and the control, calculated using both algorithms. <strong>D</strong> Mean absolute errors (MAEs) of predicted chronological ages from degraded samples preprocessed with pOOBAH and ELBAR, using the BLUP clock. <strong>E</strong> MAEs of predicted ages from degraded samples preprocessed with pOOBAH and ELBAR, using the EN clock</p></figcaption></figure></section><section id="Sec11"><h3 class="pmc_sec_title">ELBAR Algorithm Performance on Degraded Samples</h3>
<p id="Par29">The ELBAR algorithm is a detection method specifically designed for low-input DNA samples. It aims to exclude only those probes fully dominated by background signal while retaining probes with true biological signal. We compared ELBAR with the established pOOBAH algorithm across our set of degraded DNA samples that passed QC. ELBAR consistently achieved higher probe detection rates in all degraded samples (Fig. <a href="#Fig5" class="usa-link">5</a>A), detecting over 99.98% of probes in samples with an average DNA fragment size of 350 bp (at 100 ng, 50 ng, and 20 ng input) and 230 bp (at 100 ng and 50 ng input) (Supplementary Table <a href="#MOESM1" class="usa-link">6</a>). These results suggest that ELBAR retains nearly all probes, even in degraded samples. To assess the quality of the retained probes, we examined the whole-array <em>r</em> and the median|∆β|. Similar to pOOBAH, ELBAR showed a decline in correlation with decreasing average DNA fragment size and input amount (Fig. <a href="#Fig5" class="usa-link">5</a>B; Supplementary Table <a href="#MOESM1" class="usa-link">6</a>). Nevertheless, samples with average DNA fragment size of 350 bp and 230 bp at 100 ng and 50 ng input maintained comparable correlation values, while a slight reduction was observed in samples with shorter fragments and lower input. Notably, all samples had an <em>r</em> above 0.93. Median|∆β| values were generally higher in degraded samples preprocess with ELBAR (Fig. <a href="#Fig5" class="usa-link">5</a>C; Supplementary Table <a href="#MOESM1" class="usa-link">6</a>), indicating a modest reduction in precision compared to pOOBAH, likely due to the inclusion of probes with weaker signal intensities. Finally, we evaluated whether the probes retained by ELBAR algorithm could improve age prediction. Using the BLUP clock, ELBAR outperformed pOOBAH, yielding lower or comparable MAEs across all combinations of average DNA fragment size and input amount. However, this improvement was not observed for the other three epigenetic clocks (EN, Horvath, and skinHorvath), where no consistent performance differences were found between the two algorithms (Fig. <a href="#Fig5" class="usa-link">5</a>D; Supplementary Fig. <a href="#MOESM2" class="usa-link">12</a>A, B). This may suggest that clocks relying on fewer, highly age-correlated CpGs may not benefit from ELBAR’s broader probe retention strategy.</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par30">The HumanMethylationEPIC v2.0 BeadChip is the latest update in the Illumina DNAm microarray series. The EPIC microarray has been successfully employed in clinical and forensic contexts to study DNAm patterns using high-quality and quantity DNA samples [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. However, clinical and forensic DNA samples often encounter suboptimal conditions, such as fragmented and/or scarce DNA. Here, we comprehensively investigated the EPIC v2.0 microarray’s performance on human DNA samples of varying fragment sizes (350, 230, 165, and 95 bp) and input amounts (100, 50, 20, and 10 ng). Overall, the EPIC v2.0 microarray maintained high reproducibility (<em>r</em> &gt; 0.94 with control sample), precision (<em>r</em> &gt; 0.86 between sample replicates), accuracy (median <em>|∆β</em>| &lt; 0.051), and probe detection rate &gt; 41.86% (~ 378,116 probes passing filtering) for degraded samples down to an average DNA fragment size of 165 bp with a DNA input of 20 ng when preprocessed with pOOBAH algorithm. However, as expected, we observed a progressive decrease in reproducibility, precision, probe detection rate, and age prediction accuracy as DNA fragment sizes and input amounts decreased, with DNA fragment size having a greater impact than input amount. This was suggested by the larger drop in reproducibility with shorter DNA fragment size and the QC failure of highly fragmented DNA samples (95 bp).</p>
<p id="Par31">The analysis of EPIC v2.0 microarray data necessitates upstream QC to ensure robust and reproducible results; hence, samples and probes that do not meet QC standards must be excluded from downstream analyses. Key QC metrics for sample inclusion are probe detection rates, background and median signal intensities, red-to-green median signal intensity ratios, bisulfite conversion rates, <em>β</em>-value distributions, sex and identity matching, and PCA [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. In our study, degraded samples with an average DNA fragment size of 95 bp failed QC since they displayed altered signal intensities and <em>β</em>-value distributions despite having probes that initially passed filtering criteria. We observed altered <em>β</em>-value distributions, particularly a shift towards intermediate β-values. Consistent with our findings, Lee et al. [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] reported similar skewing of <em>β</em>-value towards intermediate values in suboptimal samples, such as those from FFPE tissues and cfDNA. This trend was attributed to the low DNA input, resulting in reduced overall signal intensities. Our data showed that short DNA fragments, combined with low DNA input, may drive the shift towards intermediate β-values. This phenomenon likely reflects the limited availability of DNA molecules for probe hybridization. EPIC v2.0 microarray technology relies on beads bound to 50 bp oligonucleotides that hybridize with specific bisulfite-converted DNA regions. Technically, DNA fragments as small as 95 bp should be successfully targeted by these probes. However, the initial step of the Infinium array protocol, bisulfite conversion, is a harsh chemical reaction necessary to distinguish methylated from unmethylated cytosines, which causes substantial DNA damage. We propose that the failure of 95 bp DNA samples in the EPIC v2.0 analysis is likely due to further DNA degradation caused during the bisulfite conversion step, which significantly reduces the pool of DNA fragments available for hybridization by the EPIC v2.0 probes. This hypothesis is further supported by our observation that degraded samples with 95 bp DNA fragment size and relatively high input amounts (100 and 50 ng) did not yield reliable data. Reducing the harshness of bisulfite treatment (such as lower temperatures or shorter incubation times) [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] or using enzyme-based conversion methods [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>] could potentially enhance the analysis of highly fragmented DNA samples and improve the overall detection of DNAm measurement.</p>
<p id="Par32">Additionally, although Illumina recommends a DNA input of 250 ng for EPIC v2.0 analysis, several authors assessed the EPIC v2.0 performance using low-input DNA samples [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. Kaur et al. [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>] obtained valuable data from as little as 1 ng DNA input, though probe detection rates (still exceeding 50%) and reproducibility (Spearman’s rank correlation of 0.918) decreased with lower DNA input amounts. Our findings align with these observations, showing a decrease in probe detection rate, reproducibility, and precision with lower DNA input amounts and shorter DNA fragment sizes. Nevertheless, the deviation of these parameters with lower DNA input amount should not only be attributed to EPIC v2.0 performance but also by the intrinsic nature of DNAm level calculation. The DNAm level represents the proportion of methylated cytosines in a sample, and stochastic variations of DNA molecule sampling can lead to fluctuation in the analysis of DNAm levels in samples with low DNA input amounts. Naue et al. [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] reported that a DNA sample of 10 ng, representing 2,785 copies prior to bisulfite conversion, had a 95% confidence interval for Δ<em>β</em> relative to the true DNAm value of ± 0.019. We expect that part of the|Δ<em>β|</em> observed in low-input DNA samples (10 ng) in this study reflects the inherent stochastic variations in DNA molecule sampling rather than only the performance of EPIC v2.0 microarray. This deviation from the real DNAm in a low-input DNA sample cannot be reduced by using other DNAm measurement technologies, but only by increasing DNA input amounts. When performing analyses of differentially methylated positions (DMPs) and regions (DMRs) the commonly used threshold between the two sample groups (e.g., cases/controls or exposed/non-exposed individuals) is a|Δ<em>β|</em> of 0.1. In our study, we observed an increase in the percentage of CpG sites where|Δ<em>β|</em> ≥ 0.1, from 1.95% in samples with average DNA fragment size of 350 bp and a DNA input amount of 100 ng to 31.70% in samples with average sizes of 165 bp and 20 ng DNA input. This significant increase in CpG sites with a|Δ<em>β|</em> ≥ 0.1 indicates that DNA fragment size and input amount could impact DMP and DMR analyses. These variables should be taken into consideration when performing DMP and DMR analyses by matching the two sample groups or by including these variables as covariates. Furthermore, we noted that DNAm measurement at CpG sites with intermediate DNAm value (0.1–0.9) were more affected by DNA fragment size and lower input amount as measured by <em>|∆β|</em>. Higher <em>|∆β|</em> in intermediate DNAm values were previously observed in other studies when assessing EPIC microarray in low-input DNA samples [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] and tissues stored in suboptimal conditions [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. Eventually, an enrichment of Type I probes and probes targeting CpG island and gene promoter regions was observed in degraded samples.</p>
<p id="Par33">DNAm age inference represents one of the most promising and extensively researched applications within the field of epigenetics. In the medical context, DNAm age was applied to predict lifespan (time-to-death due to all-cause mortality) [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>–<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>], but also to better understand the impact of lifestyle and environment on aging [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. While in the forensic field, inferring the age of an unknown sample donor at a crime scene can provide crucial leads in police investigations to narrow down the suspect pool. Our analysis revealed that array-based DNAm age prediction is feasible when DNA samples have average DNA fragment sizes of 350 bp (DI = ~ 2.1) and a DNA input amount of 100 ng is used. However, when the average DNA fragment size is 165 bp (DI = ~ 8.88) or 95 bp (DI = ~ 8008.4), accurate age prediction was not possible. We believe that a combination of missing data and larger <em>|∆β|</em> is the reason of the lower age prediction accuracies observed in degraded samples with short DNA fragment size and low DNA inputs. Lee et al. [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>] applied EPIC microarray technology in forensics to predict the age of an individual from fresh, fragmented, and low-input DNA samples. However, fragmented and low-input DNA were tested separately. In our study, we demonstrated that the combination of these two variables could alter DNAm measurement more significantly than each variable alone. We recommend a thorough evaluation of the combined effect of fragmented and low-input DNA samples on forensic estimations before applying EPIC v2.0 technology in forensic case work.</p>
<p id="Par34">DNAm analysis of cfDNA in plasma or other body fluids could provide valuable information about tumor diagnosis, monitoring and prognosis [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. cfDNA is retrieved in limited amounts from both healthy individuals and cancer patients. Additionally, cfDNA typically presents a peak at 167 bp for healthy individuals and 143 bp for cancer patients in the distribution of DNA fragment size [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. Analysis of DNAm in cfDNA provides a dual challenge due to the low DNA input and short fragment size. Although cfDNA was not directly used in this study, our analysis indicates that EPIC v2.0 technology could yield altered DNAm measurement when analyzing DNA fragment sizes of 165 bp using low-input amounts. Therefore, we propose to increase DNA input if the DNA is very fragmented. Our evaluation also has implications for the analysis of FFPE samples, which are primarily used for histological examinations but also for long-term tissue preservation. FFPE-derived DNA often exhibits low quality, which can impact DNAm analysis with EPIC arrays [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Dupont et al. [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>] observed that FFPE samples had a mean DI = 2.51, close to the DNA samples with average DNA fragment size of 350 bp (DI = 2.09) used in this study, and that DI was correlated with EPIC array probe detection rate [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. This correlation was observed also in our study, and we suggest that the DI parameter could be used as a preliminary quality screening tool to perform EPIC v2.0 analysis and as a predictor of probe detection rate.</p>
<p id="Par35">Finally, we evaluated the performance of ELBAR, a recently developed methylation calling algorithm designed for low-input DNA samples [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Consistent with the findings of Lee et al. [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>], ELBAR applied less stringent probe masking compared to the pOOBAH method. However, we observed a slight reduction in reproducibility (lower whole-array r) and accuracy (higher|∆β|), suggesting that the additional probes retained by ELBAR may yield less reliable DNA methylation measurements—an observation also noted by Lee et al. [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. The higher detection rate thus appears to come at the expense of reduced reproducibility and accuracy. We recommend that the choice of preprocessing algorithm be guided by the specific downstream application: whether the priority is a higher detection rate or more reproducible and accurate DNAm values. No improvements in age prediction application were observed across most epigenetic clocks, except for the BLUP clock, where prediction accuracy was significantly enhanced. The BLUP clock contains a large number of CpGs (319,607 CpGs), many of which show weak correlation with age. In this context, the increased probe retention of ELBAR may be advantageous compared to the imputation strategy employed by the <em>methylclock</em> package. Nevertheless, larger sample sizes are needed to confirm this finding.</p>
<p id="Par36">There are a few limitations in our work. Our dataset was composed of DNA samples coming from a single individual and the investigated average DNA fragment size was limited to the range of 350–95 bp. Furthermore, the distribution of the four average DNA fragment sizes showed slight variations, with samples with average DNA fragment size of 350 bp having broader peak compared to those with average size of 95 bp samples. DNA samples used in this study were artificially fragmented. The starting DNA used in this study was of high quality. Clinical and forensic samples, however, may present additional layers of complexity, such as DNA alterations or damage (e.g., chemical crosslinking in FFPE samples [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]) or different DNA fragment size distributions. These differences could make DNAm analysis more complex and limit the direct applicability of our results to forensic and clinical scenarios. Future work should focus on investigating wider DNA fragment sizes, clinical and mock forensic degraded samples, and alternative experimental and computational approaches to improve the detection of DNAm in suboptimal samples. Lee et al. [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] proposed different bisulfite conversion kits, elution volumes, and input volumes for the Infinium DNAm microarray. These alternative workflows could be applied to fragmented and low-input DNA samples in future studies.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par37">DNA fragment size and input amount affect DNAm measurements as analyzed with the EPIC v2.0 microarray, with DNA fragment size having a greater impact than DNA input amount. Reliable DNAm data could be obtained down to an average DNA fragment size of 165 bp with 20 ng input amount. However, reproducibility, precision, probe detection rate, and age prediction are compromised by shorter DNA fragment size and input amounts. When dealing with degraded samples, we recommended evaluation of DNA quality and quantity before conducting DNAm analysis with the EPIC v2.0 microarray. Overall, we showed the potential and limitations of the EPIC v2.0 microarray with low quality and quantity DNA samples.</p></section><section id="Sec14"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12369268/bin/12575_2025_292_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (26.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12369268/bin/12575_2025_292_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (2.2MB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We would like to thank Anja Ladegaard Jørgensen for assistance with this work.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>EPIC v2.0</dt>
<dd><p id="Par41">HumanMethylationEPIC v2.0 BeadChip</p></dd>
<dt>DNAm</dt>
<dd><p id="Par42">DNA methylation</p></dd>
<dt>DMRs</dt>
<dd><p id="Par43">Differentially methylated regions</p></dd>
<dt>DMPs</dt>
<dd><p id="Par44">Differentially methylated positions</p></dd>
<dt>ELBAR</dt>
<dd><p id="Par45">Eliminating BAckground-dominated Reading</p></dd>
<dt>QC</dt>
<dd><p id="Par46">Quality control</p></dd>
<dt>|Δβ|</dt>
<dd><p id="Par47">Absolute beta value difference</p></dd>
<dt>TCGA</dt>
<dd><p id="Par48">The Cancer Genome Atlas</p></dd>
<dt>ARIES</dt>
<dd><p id="Par49">Accessible Resource for Integrated Epigenomics Studies</p></dd>
<dt>cfDNA</dt>
<dd><p id="Par50">Circulating free DNA</p></dd>
<dt>FFPE</dt>
<dd><p id="Par51">Formalin-fixed, paraffin-embedded</p></dd>
<dt>Di</dt>
<dd><p id="Par52">Degradation index</p></dd>
<dt>SeSAMe</dt>
<dd><p id="Par53">Sensible Step-wise Analysis of DNA Methylation BeadChip</p></dd>
<dt>PCA</dt>
<dd><p id="Par54">Principal component analysis</p></dd>
<dt>SNP</dt>
<dd><p id="Par55">Single nucleotide polymorphism</p></dd>
<dt><em>r</em></dt>
<dd><p id="Par56">Pearson correlation coefficient</p></dd>
<dt>FDR</dt>
<dd><p id="Par57">False discovery rate</p></dd>
<dt>IQR</dt>
<dd><p id="Par58">Interquartile range</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>B.P.: designing the experiment, formal analysis, investigation, data curation, and writing original draft. M.E.D.: designing the experiment, investigation, reviewing and editing the manuscript. M.L.K: designing the experiment, reviewing and editing the manuscript. A.V.: reviewing and editing the manuscript. C.B: supervision, reviewing and editing the manuscript. V.P.: supervision, reviewing and editing the manuscript. J.T.H.: reviewing and editing the manuscript, funding acquisition. J.D.A.: designing the experiment, investigation, supervision, and funding acquisition. All authors approved the final version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open access funding provided by Copenhagen University</p></section><section id="notes3"><h2 class="pmc_sec_title">Data Availability</h2>
<p>The datasets used and/or analysed during the current study are available from the corresponding author on reasonable request. The R script used for data analysis in this study is publicly available at https://doi.org/10.5281/zenodo.15368234.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics Approval and Consent to Participate</h3>
<p id="Par38">The study conformed to the Declaration of Helsinki and was approved by the Committees of Health Research Ethics in the Capital Region of Denmark (H-22034131). Informed written consent was collected from the individual. Patient material and data were pseudonymized.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for Publication</h3>
<p id="Par39">Consent for publication was collected from the individual and included in the informed written consent.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par40">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Brando Poggiali, Email: brando.poggiali@sund.ku.dk.</p>
<p>Jeppe Dyrberg Andersen, Email: jeppe.dyrberg.andersen@sund.ku.dk.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Dhar GA, Saha S, Mitra P, Nag Chaudhuri R. DNA methylation and regulation of gene expression: guardian of our health. Nucleus. 2021;64(3):259–70.</cite> [<a href="https://doi.org/10.1007/s13237-021-00367-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8366481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34421129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dhar%20GA,%20Saha%20S,%20Mitra%20P,%20Nag%20Chaudhuri%20R.%20DNA%20methylation%20and%20regulation%20of%20gene%20expression:%20guardian%20of%20our%20health.%20Nucleus.%202021;64(3):259%E2%80%9370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Robertson KD. DNA methylation and human disease. Nat Rev Genet. 2005;6(8):597–610.
</cite> [<a href="https://doi.org/10.1038/nrg1655" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16136652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robertson%20KD.%20DNA%20methylation%20and%20human%20disease.%20Nat%20Rev%20Genet.%202005;6(8):597%E2%80%93610." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Bibikova M, Le J, Barnes B, Saedinia-Melnyk S, Zhou L, Shen R, et al. Genome-Wide Dna Methylation Profiling Using Infinium <sup>®</sup> Assay Epigenomics. 2009;1(1):177–200.
</cite> [<a href="https://doi.org/10.2217/epi.09.14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22122642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bibikova%20M,%20Le%20J,%20Barnes%20B,%20Saedinia-Melnyk%20S,%20Zhou%20L,%20Shen%20R,%20et%20al.%20Genome-Wide%20Dna%20Methylation%20Profiling%20Using%20Infinium%20%C2%AE%20Assay%20Epigenomics.%202009;1(1):177%E2%80%93200." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Weisenberger DJ. Characterizing DNA methylation alterations from the cancer genome atlas. J Clin Invest. 2014;124(1):17–23.
</cite> [<a href="https://doi.org/10.1172/JCI69740" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3871233/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24382385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weisenberger%20DJ.%20Characterizing%20DNA%20methylation%20alterations%20from%20the%20cancer%20genome%20atlas.%20J%20Clin%20Invest.%202014;124(1):17%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Relton CL, Gaunt T, McArdle W, Ho K, Duggirala A, Shihab H, et al. Data resource profile: accessible resource for integrated epigenomic studies (ARIES). Int J Epidemiol. 2015;44(4):1181–90.
</cite> [<a href="https://doi.org/10.1093/ije/dyv072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5593097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25991711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Relton%20CL,%20Gaunt%20T,%20McArdle%20W,%20Ho%20K,%20Duggirala%20A,%20Shihab%20H,%20et%20al.%20Data%20resource%20profile:%20accessible%20resource%20for%20integrated%20epigenomic%20studies%20(ARIES).%20Int%20J%20Epidemiol.%202015;44(4):1181%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Plusquin M, Guida F, Polidoro S, Vermeulen R, Raaschou-Nielsen O, Campanella G, et al. DNA methylation and exposure to ambient air pollution in two prospective cohorts. Environ Int. 2017;108:127–36.
</cite> [<a href="https://doi.org/10.1016/j.envint.2017.08.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6139298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28843141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Plusquin%20M,%20Guida%20F,%20Polidoro%20S,%20Vermeulen%20R,%20Raaschou-Nielsen%20O,%20Campanella%20G,%20et%20al.%20DNA%20methylation%20and%20exposure%20to%20ambient%20air%20pollution%20in%20two%20prospective%20cohorts.%20Environ%20Int.%202017;108:127%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Lai CQ, Parnell LD, Lee YC, Zeng H, Smith CE, McKeown NM, et al. The impact of alcoholic drinks and dietary factors on epigenetic markers associated with triglyceride levels. Front Genet. 2023;14:1117778.
</cite> [<a href="https://doi.org/10.3389/fgene.2023.1117778" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9975169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36873949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lai%20CQ,%20Parnell%20LD,%20Lee%20YC,%20Zeng%20H,%20Smith%20CE,%20McKeown%20NM,%20et%20al.%20The%20impact%20of%20alcoholic%20drinks%20and%20dietary%20factors%20on%20epigenetic%20markers%20associated%20with%20triglyceride%20levels.%20Front%20Genet.%202023;14:1117778." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Dhingra R, Kwee LC, Diaz-Sanchez D, Devlin RB, Cascio W, Hauser ER, et al. Evaluating DNA methylation age on the illumina methylationepic bead chip. PLoS ONE. 2019;14(4):e0207834.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0207834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6474589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31002714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dhingra%20R,%20Kwee%20LC,%20Diaz-Sanchez%20D,%20Devlin%20RB,%20Cascio%20W,%20Hauser%20ER,%20et%20al.%20Evaluating%20DNA%20methylation%20age%20on%20the%20illumina%20methylationepic%20bead%20chip.%20PLoS%20ONE.%202019;14(4):e0207834." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Li M, Li Y, Qin H, Tubbs JD, Li M, Qiao C, et al. Genome-wide DNA methylation analysis of peripheral blood cells derived from patients with first-episode schizophrenia in the Chinese Han population. Mol Psychiatry. 2021;26(8):4475–85.
</cite> [<a href="https://doi.org/10.1038/s41380-020-00968-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33279932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20M,%20Li%20Y,%20Qin%20H,%20Tubbs%20JD,%20Li%20M,%20Qiao%20C,%20et%20al.%20Genome-wide%20DNA%20methylation%20analysis%20of%20peripheral%20blood%20cells%20derived%20from%20patients%20with%20first-episode%20schizophrenia%20in%20the%20Chinese%20Han%20population.%20Mol%20Psychiatry.%202021;26(8):4475%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Zhang X, Xiang Y, He D, Liang B, Wang C, Luo J, et al. Identification of potential biomarkers for CAD using integrated expression and methylation data. Front Genet. 2020;11:778.
</cite> [<a href="https://doi.org/10.3389/fgene.2020.00778" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7509170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33033488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20X,%20Xiang%20Y,%20He%20D,%20Liang%20B,%20Wang%20C,%20Luo%20J,%20et%20al.%20Identification%20of%20potential%20biomarkers%20for%20CAD%20using%20integrated%20expression%20and%20methylation%20data.%20Front%20Genet.%202020;11:778." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Draškovič T, Hauptman N. Discovery of novel DNA methylation biomarker panels for the diagnosis and differentiation between common adenocarcinomas and their liver metastases. Sci Rep. 2024;14(1):3095.
</cite> [<a href="https://doi.org/10.1038/s41598-024-53754-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10850119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38326602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dra%C5%A1kovi%C4%8D%20T,%20Hauptman%20N.%20Discovery%20of%20novel%20DNA%20methylation%20biomarker%20panels%20for%20the%20diagnosis%20and%20differentiation%20between%20common%20adenocarcinomas%20and%20their%20liver%20metastases.%20Sci%20Rep.%202024;14(1):3095." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Noguera-Castells A, García-Prieto CA, Álvarez-Errico D, Esteller M. Validation of the new EPIC DNA methylation microarray (900K EPIC v2) for high-throughput profiling of the human DNA methylome. Epigenetics. 2023;18(1):2185742.
</cite> [<a href="https://doi.org/10.1080/15592294.2023.2185742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9988339/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36871255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Noguera-Castells%20A,%20Garc%C3%ADa-Prieto%20CA,%20%C3%81lvarez-Errico%20D,%20Esteller%20M.%20Validation%20of%20the%20new%20EPIC%20DNA%20methylation%20microarray%20(900K%20EPIC%20v2)%20for%20high-throughput%20profiling%20of%20the%20human%20DNA%20methylome.%20Epigenetics.%202023;18(1):2185742." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Kaur D, Lee SM, Goldberg D, Spix NJ, Hinoue T, Li HT, et al. Comprehensive evaluation of the infinium human methylationepic v2 BeadChip. Epigenetics Commun. 2023;3(1):6.
</cite> [<a href="https://doi.org/10.1186/s43682-023-00021-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10919401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38455390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaur%20D,%20Lee%20SM,%20Goldberg%20D,%20Spix%20NJ,%20Hinoue%20T,%20Li%20HT,%20et%20al.%20Comprehensive%20evaluation%20of%20the%20infinium%20human%20methylationepic%20v2%20BeadChip.%20Epigenetics%20Commun.%202023;3(1):6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Udomruk S, Orrapin S, Pruksakorn D, Chaiyawat P. Size distribution of cell-free DNA in oncology. Crit Rev Oncol Hematol. 2021;166:103455.
</cite> [<a href="https://doi.org/10.1016/j.critrevonc.2021.103455" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34464717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Udomruk%20S,%20Orrapin%20S,%20Pruksakorn%20D,%20Chaiyawat%20P.%20Size%20distribution%20of%20cell-free%20DNA%20in%20oncology.%20Crit%20Rev%20Oncol%20Hematol.%202021;166:103455." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Khurram I, Khan MU, Ibrahim S, Saleem A, Khan Z, Mubeen M, et al. Efficacy of cell-free DNA as a diagnostic biomarker in breast cancer patients. Sci Rep. 2023;13(1):15347.
</cite> [<a href="https://doi.org/10.1038/s41598-023-42726-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10504267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37715016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khurram%20I,%20Khan%20MU,%20Ibrahim%20S,%20Saleem%20A,%20Khan%20Z,%20Mubeen%20M,%20et%20al.%20Efficacy%20of%20cell-free%20DNA%20as%20a%20diagnostic%20biomarker%20in%20breast%20cancer%20patients.%20Sci%20Rep.%202023;13(1):15347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Shen W, Dong H, Tang H, Zhang Y, Jia S, Luo Y. Diagnosis of prostate cancer using cell-free DNA methylation profiles from expressed prostatic secretions. J Clin Oncol. 2023;41(6suppl):389–389.</cite> [<a href="https://scholar.google.com/scholar_lookup?Shen%20W,%20Dong%20H,%20Tang%20H,%20Zhang%20Y,%20Jia%20S,%20Luo%20Y.%20Diagnosis%20of%20prostate%20cancer%20using%20cell-free%20DNA%20methylation%20profiles%20from%20expressed%20prostatic%20secretions.%20J%20Clin%20Oncol.%202023;41(6suppl):389%E2%80%93389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Lauková L, Konečná B, Vlková B, Mlynáriková V, Celec P, Šteňová E. Anti-cytokine therapy and plasma DNA in patients with rheumatoid arthritis. Rheumatol Int. 2018;38(8):1449–54.
</cite> [<a href="https://doi.org/10.1007/s00296-018-4055-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29796906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laukov%C3%A1%20L,%20Kone%C4%8Dn%C3%A1%20B,%20Vlkov%C3%A1%20B,%20Mlyn%C3%A1rikov%C3%A1%20V,%20Celec%20P,%20%C5%A0te%C5%88ov%C3%A1%20E.%20Anti-cytokine%20therapy%20and%20plasma%20DNA%20in%20patients%20with%20rheumatoid%20arthritis.%20Rheumatol%20Int.%202018;38(8):1449%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Raptis L, Menard HA. Quantitation and characterization of plasma DNA in normals and patients with systemic lupus erythematosus. J Clin Invest. 1980;66(6):1391–9.
</cite> [<a href="https://doi.org/10.1172/JCI109992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC371625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7440721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raptis%20L,%20Menard%20HA.%20Quantitation%20and%20characterization%20of%20plasma%20DNA%20in%20normals%20and%20patients%20with%20systemic%20lupus%20erythematosus.%20J%20Clin%20Invest.%201980;66(6):1391%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Xu Y, Song Y, Chang J, Zhou X, Qi Q, Tian X, et al. High levels of Circulating cell-free &lt; scp &gt; DNA are a biomarker of active &lt; scp &gt; sle. Eur J Clin Invest. 2018;48(11):e13015.
</cite> [<a href="https://doi.org/10.1111/eci.13015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30079480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20Y,%20Song%20Y,%20Chang%20J,%20Zhou%20X,%20Qi%20Q,%20Tian%20X,%20et%20al.%20High%20levels%20of%20Circulating%20cell-free%E2%80%89&lt;%E2%80%89scp%E2%80%89&gt;%E2%80%89DNA%E2%80%89are%20a%20biomarker%20of%20active%E2%80%89&lt;%E2%80%89scp%E2%80%89&gt;%E2%80%89sle.%20Eur%20J%20Clin%20Invest.%202018;48(11):e13015." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Spector BL, Harrell L, Sante D, Wyckoff GJ, Willig L. The methylome and cell-free DNA: current applications in medicine and pediatric disease. Pediatr Res. 2023;94(1):89–95.
</cite> [<a href="https://doi.org/10.1038/s41390-022-02448-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9842217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36646885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spector%20BL,%20Harrell%20L,%20Sante%20D,%20Wyckoff%20GJ,%20Willig%20L.%20The%20methylome%20and%20cell-free%20DNA:%20current%20applications%20in%20medicine%20and%20pediatric%20disease.%20Pediatr%20Res.%202023;94(1):89%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Medina JE, Dracopoli NC, Bach PB, Lau A, Scharpf RB, Meijer GA, et al. Cell-free DNA approaches for cancer early detection and interception. J Immunother Cancer. 2023;11(9):e006013.
</cite> [<a href="https://doi.org/10.1136/jitc-2022-006013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10496721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37696619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Medina%20JE,%20Dracopoli%20NC,%20Bach%20PB,%20Lau%20A,%20Scharpf%20RB,%20Meijer%20GA,%20et%20al.%20Cell-free%20DNA%20approaches%20for%20cancer%20early%20detection%20and%20interception.%20J%20Immunother%20Cancer.%202023;11(9):e006013." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>McDonough SJ, Bhagwate A, Sun Z, Wang C, Zschunke M, Gorman JA, et al. Use of FFPE-derived DNA in next generation sequencing: DNA extraction methods. PLoS ONE. 2019;14(4):e0211400.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0211400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6459541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30973937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McDonough%20SJ,%20Bhagwate%20A,%20Sun%20Z,%20Wang%20C,%20Zschunke%20M,%20Gorman%20JA,%20et%20al.%20Use%20of%20FFPE-derived%20DNA%20in%20next%20generation%20sequencing:%20DNA%20extraction%20methods.%20PLoS%20ONE.%202019;14(4):e0211400." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Lee JM, Park SU, Lee SD, Lee HY. Application of array-based age prediction models to post-mortem tissue samples. Forensic Sci Int Genet. 2024;68:102940.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2023.102940" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37857127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20JM,%20Park%20SU,%20Lee%20SD,%20Lee%20HY.%20Application%20of%20array-based%20age%20prediction%20models%20to%20post-mortem%20tissue%20samples.%20Forensic%20Sci%20Int%20Genet.%202024;68:102940." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Lee HY, Hong SR, Lee JE, Hwang IK, Kim NY, Lee JM, et al. Epigenetic age signatures in bones. Forensic Sci Int Genet. 2020;46:102261.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2020.102261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32087494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20HY,%20Hong%20SR,%20Lee%20JE,%20Hwang%20IK,%20Kim%20NY,%20Lee%20JM,%20et%20al.%20Epigenetic%20age%20signatures%20in%20bones.%20Forensic%20Sci%20Int%20Genet.%202020;46:102261." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Pruszkowska-Przybylska P, Dupont ME, Jacobsen SB, Smerup M, Tfelt-Hansen J, Morling N, et al. Evaluation of DNAmAge in paired fresh, frozen, and formalin-fixed paraffin-embedded heart tissues. PLoS ONE. 2024;19(5):e0299557.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0299557" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11078437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pruszkowska-Przybylska%20P,%20Dupont%20ME,%20Jacobsen%20SB,%20Smerup%20M,%20Tfelt-Hansen%20J,%20Morling%20N,%20et%20al.%20Evaluation%20of%20DNAmAge%20in%20paired%20fresh,%20frozen,%20and%20formalin-fixed%20paraffin-embedded%20heart%20tissues.%20PLoS%20ONE.%202024;19(5):e0299557." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Sawyer S, Gelabert P, Yakir B, Llanos-Lizcano A, Sperduti A, Bondioli L, et al. Improved detection of methylation in ancient DNA. Genome Biol. 2024;25(1):261.
</cite> [<a href="https://doi.org/10.1186/s13059-024-03405-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11465500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39390557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sawyer%20S,%20Gelabert%20P,%20Yakir%20B,%20Llanos-Lizcano%20A,%20Sperduti%20A,%20Bondioli%20L,%20et%20al.%20Improved%20detection%20of%20methylation%20in%20ancient%20DNA.%20Genome%20Biol.%202024;25(1):261." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Wagner S, Plomion C, Orlando L. Uncovering signatures of DNA methylation in ancient plant remains from patterns of Post-mortem DNA damage. Front Ecol Evol. 2020;8:11.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wagner%20S,%20Plomion%20C,%20Orlando%20L.%20Uncovering%20signatures%20of%20DNA%20methylation%20in%20ancient%20plant%20remains%20from%20patterns%20of%20Post-mortem%20DNA%20damage.%20Front%20Ecol%20Evol.%202020;8:11." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Gokhman D, Malul A, Carmel L. Inferring past environments from ancient epigenomes. Mol Biol Evol. 2017;34(10):2429–38.
</cite> [<a href="https://doi.org/10.1093/molbev/msx211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5850778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28957511/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gokhman%20D,%20Malul%20A,%20Carmel%20L.%20Inferring%20past%20environments%20from%20ancient%20epigenomes.%20Mol%20Biol%20Evol.%202017;34(10):2429%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Schmidt M, Maixner F, Hotz G, Pap I, Szikossy I, Pálfi G, et al. DNA methylation profiling in mummified human remains from the eighteenth-century. Sci Rep. 2021;11(1):15493.
</cite> [<a href="https://doi.org/10.1038/s41598-021-95021-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8322318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34326450/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmidt%20M,%20Maixner%20F,%20Hotz%20G,%20Pap%20I,%20Szikossy%20I,%20P%C3%A1lfi%20G,%20et%20al.%20DNA%20methylation%20profiling%20in%20mummified%20human%20remains%20from%20the%20eighteenth-century.%20Sci%20Rep.%202021;11(1):15493." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Christiansen SN, Andersen JD, Kampmann ML, Liu J, Andersen MM, Tfelt-Hansen J, et al. Reproducibility of the infinium methylationepic BeadChip assay using low DNA amounts. Epigenetics. 2022;17(12):1636–45.
</cite> [<a href="https://doi.org/10.1080/15592294.2022.2051861" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9621037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35356867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Christiansen%20SN,%20Andersen%20JD,%20Kampmann%20ML,%20Liu%20J,%20Andersen%20MM,%20Tfelt-Hansen%20J,%20et%20al.%20Reproducibility%20of%20the%20infinium%20methylationepic%20BeadChip%20assay%20using%20low%20DNA%20amounts.%20Epigenetics.%202022;17(12):1636%E2%80%9345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Lee SM, Loo CE, Prasasya RD, Bartolomei MS, Kohli RM, Zhou W. Low-input and single-cell methods for infinium DNA methylation beadchips. Nucleic Acids Res. 2024;52(7):e38–38.
</cite> [<a href="https://doi.org/10.1093/nar/gkae127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11040145/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38407446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20SM,%20Loo%20CE,%20Prasasya%20RD,%20Bartolomei%20MS,%20Kohli%20RM,%20Zhou%20W.%20Low-input%20and%20single-cell%20methods%20for%20infinium%20DNA%20methylation%20beadchips.%20Nucleic%20Acids%20Res.%202024;52(7):e38%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Zhou W, Triche TJ, Laird PW, Shen H. SeSAMe: reducing artifactual detection of DNA methylation by infinium beadchips in genomic deletions. Nucleic Acids Res. 2018;46:e123.
</cite> [<a href="https://doi.org/10.1093/nar/gky691" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6237738/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30085201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20W,%20Triche%20TJ,%20Laird%20PW,%20Shen%20H.%20SeSAMe:%20reducing%20artifactual%20detection%20of%20DNA%20methylation%20by%20infinium%20beadchips%20in%20genomic%20deletions.%20Nucleic%20Acids%20Res.%202018;46:e123." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Wickham H. ggplot2: Elegant Graphics for Data Analysis [Internet]. Springer-Verlag New York; 2016. Available from: <a href="https://ggplot2.tidyverse.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ggplot2.tidyverse.org</a></cite>
</li>
<li id="CR34">
<span class="label">34.</span><cite>R Core Team. R: A Language and Environment for Statistical Computing [Internet]. Vienna, Austria. 2021. Available from: <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R-project.org/</a></cite>
</li>
<li id="CR35">
<span class="label">35.</span><cite>Benjamini Y, Hochberg Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. J R Stat Soc Ser B Stat Methodol. 1995;57(1):289–300.</cite> [<a href="https://scholar.google.com/scholar_lookup?Benjamini%20Y,%20Hochberg%20Y.%20Controlling%20the%20false%20discovery%20rate:%20A%20practical%20and%20powerful%20approach%20to%20multiple%20testing.%20J%20R%20Stat%20Soc%20Ser%20B%20Stat%20Methodol.%201995;57(1):289%E2%80%93300." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Gu Z. IlluminaHumanMethylationEPICv2anno.20a1.hg38: Annotation for Illumina’s EPIC v2.0 methylation arrays [Internet]. 2024. Available from: <a href="https://www.illumina.com/products/by-type/microarray-kits/infinium-methylation-epic.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.illumina.com/products/by-type/microarray-kits/infinium-methylation-epic.html</a></cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Rainer J. EnsDb.Hsapiens.v86: Ensembl based annotation package. 2017.</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite>Pelegí-Sisó D, de Prado P, Ronkainen J, Bustamante M, González JR. <em>methylclock</em>: a bioconductor package to estimate DNA methylation age. Bioinformatics. 2021;37(12):1759–60.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32960939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peleg%C3%AD-Sis%C3%B3%20D,%20de%20Prado%20P,%20Ronkainen%20J,%20Bustamante%20M,%20Gonz%C3%A1lez%20JR.%20methylclock:%20a%20bioconductor%20package%20to%20estimate%20DNA%20methylation%20age.%20Bioinformatics.%202021;37(12):1759%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Horvath S. DNA methylation age of human tissues and cell types. Genome Biol. 2013;14(10):R115.
</cite> [<a href="https://doi.org/10.1186/gb-2013-14-10-r115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4015143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24138928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Horvath%20S.%20DNA%20methylation%20age%20of%20human%20tissues%20and%20cell%20types.%20Genome%20Biol.%202013;14(10):R115." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Horvath S, Oshima J, Martin GM, Lu AT, Quach A, Cohen H, et al. Epigenetic clock for skin and blood cells applied to Hutchinson Gilford Progeria syndrome and ex vivo studies. Aging. 2018;10(7):1758–75.
</cite> [<a href="https://doi.org/10.18632/aging.101508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6075434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30048243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Horvath%20S,%20Oshima%20J,%20Martin%20GM,%20Lu%20AT,%20Quach%20A,%20Cohen%20H,%20et%20al.%20Epigenetic%20clock%20for%20skin%20and%20blood%20cells%20applied%20to%20Hutchinson%20Gilford%20Progeria%20syndrome%20and%20ex%20vivo%20studies.%20Aging.%202018;10(7):1758%E2%80%9375." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Zhang Q, Vallerga CL, Walker RM, Lin T, Henders AK, Montgomery GW, et al. Improved precision of epigenetic clock estimates across tissues and its implication for biological ageing. Genome Med. 2019;11(1):54.
</cite> [<a href="https://doi.org/10.1186/s13073-019-0667-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6708158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31443728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Q,%20Vallerga%20CL,%20Walker%20RM,%20Lin%20T,%20Henders%20AK,%20Montgomery%20GW,%20et%20al.%20Improved%20precision%20of%20epigenetic%20clock%20estimates%20across%20tissues%20and%20its%20implication%20for%20biological%20ageing.%20Genome%20Med.%202019;11(1):54." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Rodriguez-Casanova A, Costa-Fraga N, Castro-Carballeira C, González-Conde M, Abuin C, Bao-Caamano A, et al. A genome-wide cell-free DNA methylation analysis identifies an episignature associated with metastatic luminal B breast cancer. Front Cell Dev Biol. 2022;10:1016955.
</cite> [<a href="https://doi.org/10.3389/fcell.2022.1016955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9641197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36393855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rodriguez-Casanova%20A,%20Costa-Fraga%20N,%20Castro-Carballeira%20C,%20Gonz%C3%A1lez-Conde%20M,%20Abuin%20C,%20Bao-Caamano%20A,%20et%20al.%20A%20genome-wide%20cell-free%20DNA%20methylation%20analysis%20identifies%20an%20episignature%20associated%20with%20metastatic%20luminal%20B%20breast%20cancer.%20Front%20Cell%20Dev%20Biol.%202022;10:1016955." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Hahn J, Bressler J, Domingo-Relloso A, Chen MH, McCartney DL, Teumer A, et al. DNA methylation analysis is used to identify novel genetic loci associated with Circulating fibrinogen levels in blood. J Thromb Haemost. 2023;21(5):1135–47.
</cite> [<a href="https://doi.org/10.1016/j.jtha.2023.01.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11556295/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36716967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hahn%20J,%20Bressler%20J,%20Domingo-Relloso%20A,%20Chen%20MH,%20McCartney%20DL,%20Teumer%20A,%20et%20al.%20DNA%20methylation%20analysis%20is%20used%20to%20identify%20novel%20genetic%20loci%20associated%20with%20Circulating%20fibrinogen%20levels%20in%20blood.%20J%20Thromb%20Haemost.%202023;21(5):1135%E2%80%9347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Hop PJ, Zwamborn RAJ, Hannon EJ, Dekker AM, van Eijk KR, Walker EM, et al. Cross-reactive probes on illumina DNA methylation arrays: a large study on ALS shows that a cautionary approach is warranted in interpreting epigenome-wide association studies. NAR Genom Bioinform. 2020;2(4):105.</cite> [<a href="https://doi.org/10.1093/nargab/lqaa105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7745769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33554115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hop%20PJ,%20Zwamborn%20RAJ,%20Hannon%20EJ,%20Dekker%20AM,%20van%20Eijk%20KR,%20Walker%20EM,%20et%20al.%20Cross-reactive%20probes%20on%20illumina%20DNA%20methylation%20arrays:%20a%20large%20study%20on%20ALS%20shows%20that%20a%20cautionary%20approach%20is%20warranted%20in%20interpreting%20epigenome-wide%20association%20studies.%20NAR%20Genom%20Bioinform.%202020;2(4):105." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Sahoo K, Sundararajan V. Methods in DNA methylation array dataset analysis: A review. Comput Struct Biotechnol J. 2024;23:2304–25.
</cite> [<a href="https://doi.org/10.1016/j.csbj.2024.05.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11153885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38845821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sahoo%20K,%20Sundararajan%20V.%20Methods%20in%20DNA%20methylation%20array%20dataset%20analysis:%20A%20review.%20Comput%20Struct%20Biotechnol%20J.%202024;23:2304%E2%80%9325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Dai Q, Ye C, Irkliyenko I, Wang Y, Sun HL, Gao Y, et al. Ultrafast bisulfite sequencing detection of 5-methylcytosine in DNA and RNA. Nat Biotechnol. 2024;42:1559.
</cite> [<a href="https://doi.org/10.1038/s41587-023-02034-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11217147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38168991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dai%20Q,%20Ye%20C,%20Irkliyenko%20I,%20Wang%20Y,%20Sun%20HL,%20Gao%20Y,%20et%20al.%20Ultrafast%20bisulfite%20sequencing%20detection%20of%205-methylcytosine%20in%20DNA%20and%20RNA.%20Nat%20Biotechnol.%202024;42:1559." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Simons RB, Karkala F, Kukk MM, Adams HHH, Kayser M, Vidaki A. Comparative performance evaluation of bisulfite- and enzyme-based DNA conversion methods. Clin Epigenetics. 2025;17(1):56.
</cite> [<a href="https://doi.org/10.1186/s13148-025-01855-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11969950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40181442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simons%20RB,%20Karkala%20F,%20Kukk%20MM,%20Adams%20HHH,%20Kayser%20M,%20Vidaki%20A.%20Comparative%20performance%20evaluation%20of%20bisulfite-%20and%20enzyme-based%20DNA%20conversion%20methods.%20Clin%20Epigenetics.%202025;17(1):56." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Peters TJ, Meyer B, Ryan L, Achinger-Kawecka J, Song J, Campbell EM, et al. Characterisation and reproducibility of the humanmethylationepic v2.0 BeadChip for DNA methylation profiling. BMC Genomics. 2024;25(1):251.
</cite> [<a href="https://doi.org/10.1186/s12864-024-10027-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10916044/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38448820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peters%20TJ,%20Meyer%20B,%20Ryan%20L,%20Achinger-Kawecka%20J,%20Song%20J,%20Campbell%20EM,%20et%20al.%20Characterisation%20and%20reproducibility%20of%20the%20humanmethylationepic%20v2.0%20BeadChip%20for%20DNA%20methylation%20profiling.%20BMC%20Genomics.%202024;25(1):251." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Naue J, Hoefsloot HCJ, Kloosterman AD, Verschure PJ. Forensic DNA methylation profiling from minimal traces: how low can we go? Forensic Sci Int Genet. 2018;33:17–23.
</cite> [<a href="https://doi.org/10.1016/j.fsigen.2017.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29175600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naue%20J,%20Hoefsloot%20HCJ,%20Kloosterman%20AD,%20Verschure%20PJ.%20Forensic%20DNA%20methylation%20profiling%20from%20minimal%20traces:%20how%20low%20can%20we%20go?%20Forensic%20Sci%20Int%20Genet.%202018;33:17%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Poggiali B, Dupont ME, Jacobsen SB, Smerup MH, Christiansen SNN, Tfelt-Hansen J, et al. DNA methylation stability in cardiac tissues kept at different temperatures and time intervals. Sci Rep. 2024;14(1):25170.
</cite> [<a href="https://doi.org/10.1038/s41598-024-76027-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11502879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39448773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poggiali%20B,%20Dupont%20ME,%20Jacobsen%20SB,%20Smerup%20MH,%20Christiansen%20SNN,%20Tfelt-Hansen%20J,%20et%20al.%20DNA%20methylation%20stability%20in%20cardiac%20tissues%20kept%20at%20different%20temperatures%20and%20time%20intervals.%20Sci%20Rep.%202024;14(1):25170." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Chen BH, Marioni RE, Colicino E, Peters MJ, Ward-Caviness CK, Tsai PC, et al. DNA methylation-based measures of biological age: meta-analysis predicting time to death. Aging. 2016;8(9):1844–65.
</cite> [<a href="https://doi.org/10.18632/aging.101020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5076441/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27690265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20BH,%20Marioni%20RE,%20Colicino%20E,%20Peters%20MJ,%20Ward-Caviness%20CK,%20Tsai%20PC,%20et%20al.%20DNA%20methylation-based%20measures%20of%20biological%20age:%20meta-analysis%20predicting%20time%20to%20death.%20Aging.%202016;8(9):1844%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Zhang Y, Wilson R, Heiss J, Breitling LP, Saum KU, Schöttker B, et al. DNA methylation signatures in peripheral blood strongly predict all-cause mortality. Nat Commun. 2017;8(1):14617.
</cite> [<a href="https://doi.org/10.1038/ncomms14617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5357865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28303888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Y,%20Wilson%20R,%20Heiss%20J,%20Breitling%20LP,%20Saum%20KU,%20Sch%C3%B6ttker%20B,%20et%20al.%20DNA%20methylation%20signatures%20in%20peripheral%20blood%20strongly%20predict%20all-cause%20mortality.%20Nat%20Commun.%202017;8(1):14617." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Levine ME, Lu AT, Quach A, Chen BH, Assimes TL, Bandinelli S, et al. An epigenetic biomarker of aging for lifespan and healthspan. Aging. 2018;10(4):573–91.
</cite> [<a href="https://doi.org/10.18632/aging.101414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5940111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29676998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Levine%20ME,%20Lu%20AT,%20Quach%20A,%20Chen%20BH,%20Assimes%20TL,%20Bandinelli%20S,%20et%20al.%20An%20epigenetic%20biomarker%20of%20aging%20for%20lifespan%20and%20healthspan.%20Aging.%202018;10(4):573%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Ryan J, Wrigglesworth J, Loong J, Fransquet PD, Woods RL. A systematic review and Meta-analysis of environmental, lifestyle, and health factors associated with DNA methylation age. Journals Gerontology: Ser A. 2020;75(3):481–94.</cite> [<a href="https://doi.org/10.1093/gerona/glz099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7328212/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31001624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ryan%20J,%20Wrigglesworth%20J,%20Loong%20J,%20Fransquet%20PD,%20Woods%20RL.%20A%20systematic%20review%20and%20Meta-analysis%20of%20environmental,%20lifestyle,%20and%20health%20factors%20associated%20with%20DNA%20methylation%20age.%20Journals%20Gerontology:%20Ser%20A.%202020;75(3):481%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Cisneros-Villanueva M, Hidalgo-Pérez L, Rios-Romero M, Cedro-Tanda A, Ruiz-Villavicencio CA, Page K, et al. Cell-free DNA analysis in current cancer clinical trials: a review. Br J Cancer. 2022;126(3):391–400.
</cite> [<a href="https://doi.org/10.1038/s41416-021-01696-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8810765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35027672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cisneros-Villanueva%20M,%20Hidalgo-P%C3%A9rez%20L,%20Rios-Romero%20M,%20Cedro-Tanda%20A,%20Ruiz-Villavicencio%20CA,%20Page%20K,%20et%20al.%20Cell-free%20DNA%20analysis%20in%20current%20cancer%20clinical%20trials:%20a%20review.%20Br%20J%20Cancer.%202022;126(3):391%E2%80%93400." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Qi T, Pan M, Shi H, Wang L, Bai Y, Ge Q. Cell-Free DNA fragmentomics: the novel promising biomarker. Int J Mol Sci. 2023;24(2):1503.
</cite> [<a href="https://doi.org/10.3390/ijms24021503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9866579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36675018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qi%20T,%20Pan%20M,%20Shi%20H,%20Wang%20L,%20Bai%20Y,%20Ge%20Q.%20Cell-Free%20DNA%20fragmentomics:%20the%20novel%20promising%20biomarker.%20Int%20J%20Mol%20Sci.%202023;24(2):1503." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Dupont ME, Jacobsen SB, Christiansen SNN, Tfelt-Hansen J, Smerup MH, Andersen JD, et al. Fresh and frozen cardiac tissue are comparable in DNA methylation array β-values, but formalin-fixed, paraffin-embedded tissue May overestimate DNA methylation levels. Sci Rep. 2023;13(1):16381.
</cite> [<a href="https://doi.org/10.1038/s41598-023-43788-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10541404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37773256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dupont%20ME,%20Jacobsen%20SB,%20Christiansen%20SNN,%20Tfelt-Hansen%20J,%20Smerup%20MH,%20Andersen%20JD,%20et%20al.%20Fresh%20and%20frozen%20cardiac%20tissue%20are%20comparable%20in%20DNA%20methylation%20array%20%CE%B2-values,%20but%20formalin-fixed,%20paraffin-embedded%20tissue%20May%20overestimate%20DNA%20methylation%20levels.%20Sci%20Rep.%202023;13(1):16381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Dupont ME, Christiansen SN, Jacobsen SB, Kampmann ML, Olsen KB, Tfelt-Hansen J, et al. DNA quality evaluation of formalin-fixed paraffin-embedded heart tissue for DNA methylation array analysis. Sci Rep. 2023;13(1):2004.
</cite> [<a href="https://doi.org/10.1038/s41598-023-29120-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9898234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36737451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dupont%20ME,%20Christiansen%20SN,%20Jacobsen%20SB,%20Kampmann%20ML,%20Olsen%20KB,%20Tfelt-Hansen%20J,%20et%20al.%20DNA%20quality%20evaluation%20of%20formalin-fixed%20paraffin-embedded%20heart%20tissue%20for%20DNA%20methylation%20array%20analysis.%20Sci%20Rep.%202023;13(1):2004." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Mathieson W, Thomas GA. Why Formalin-fixed, Paraffin-embedded biospecimens must be used in genomic medicine: an Evidence-based review and conclusion. J Histochem Cytochem. 2020;68(8):543–52.
</cite> [<a href="https://doi.org/10.1369/0022155420945050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7400666/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32697619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mathieson%20W,%20Thomas%20GA.%20Why%20Formalin-fixed,%20Paraffin-embedded%20biospecimens%20must%20be%20used%20in%20genomic%20medicine:%20an%20Evidence-based%20review%20and%20conclusion.%20J%20Histochem%20Cytochem.%202020;68(8):543%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369268/bin/12575_2025_292_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (26.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369268/bin/12575_2025_292_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (2.2MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets used and/or analysed during the current study are available from the corresponding author on reasonable request. The R script used for data analysis in this study is publicly available at https://doi.org/10.5281/zenodo.15368234.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Biological Procedures Online are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12575-025-00292-3"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12575_2025_Article_292.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12369268/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12369268/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369268%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369268/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12369268/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12369268/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841873/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12369268/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841873/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12369268/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12369268/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="b1SH73vspvfgr7T6jh0ssp5QbO2qDGD7Nkr1dzblIQeAG1H6SsGJKsQAk8UIZ6XU">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
