
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Understanding the genetic basis of heat stress tolerance in wheat (Triticum aestivum L.) through genome‐wide association studies - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E482168AEECDE30582160025CC485F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12241833/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Understanding the genetic basis of heat stress tolerance in wheat (Triticum aestivum L.) through genome‐wide association studies">
<meta name="citation_author" content="Santosh Gudi">
<meta name="citation_author_institution" content="Department of Plant Pathology, North Dakota State University, Fargo, North Dakota, USA">
<meta name="citation_author_institution" content="Edward T. Schafer Agricultural Research Center, USDA‐ARS, Fargo, North Dakota, USA">
<meta name="citation_author" content="Jatinder Singh">
<meta name="citation_author_institution" content="Department of Plant Pathology, North Dakota State University, Fargo, North Dakota, USA">
<meta name="citation_author" content="Harsimardeep Gill">
<meta name="citation_author_institution" content="Department of Agronomy, Horticulture, and Plant Science, South Dakota State University, Brookings, South Dakota, USA">
<meta name="citation_author" content="Sunish Sehgal">
<meta name="citation_author_institution" content="Department of Agronomy, Horticulture, and Plant Science, South Dakota State University, Brookings, South Dakota, USA">
<meta name="citation_author" content="Justin D Faris">
<meta name="citation_author_institution" content="Edward T. Schafer Agricultural Research Center, USDA‐ARS, Fargo, North Dakota, USA">
<meta name="citation_author" content="Upinder Gill">
<meta name="citation_author_institution" content="Department of Plant Pathology, North Dakota State University, Fargo, North Dakota, USA">
<meta name="citation_author" content="Rajeev Gupta">
<meta name="citation_author_institution" content="Edward T. Schafer Agricultural Research Center, USDA‐ARS, Fargo, North Dakota, USA">
<meta name="citation_publication_date" content="2025 Jul 9">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70071">
<meta name="citation_doi" content="10.1002/tpg2.70071">
<meta name="citation_pmid" content="40635288">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12241833/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12241833/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12241833/pdf/TPG2-18-e70071.pdf">
<meta name="description" content="Heat stress can reduce the production potential of wheat (Triticum aestivum L.) by affecting the various developmental stages of wheat including the seedling stage. Understanding the genetic basis of heat stress tolerance can help in breeding ...">
<meta name="og:title" content="Understanding the genetic basis of heat stress tolerance in wheat (Triticum aestivum L.) through genome‐wide association studies">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Heat stress can reduce the production potential of wheat (Triticum aestivum L.) by affecting the various developmental stages of wheat including the seedling stage. Understanding the genetic basis of heat stress tolerance can help in breeding ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12241833/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12241833">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70071"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70071.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12241833%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12241833/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12241833/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12241833/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70071" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 9;18(3):e70071. doi: <a href="https://doi.org/10.1002/tpg2.70071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70071</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Understanding the genetic basis of heat stress tolerance in wheat (<em>Triticum aestivum</em> L.) through genome‐wide association studies</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gudi%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Santosh Gudi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Santosh Gudi</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Edward T. Schafer Agricultural Research Center, USDA‐ARS, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gudi%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Santosh Gudi</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jatinder Singh</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jatinder Singh</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jatinder Singh</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gill%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Harsimardeep Gill</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Harsimardeep Gill</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Agronomy, Horticulture, and Plant Science, South Dakota State University, Brookings, South Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gill%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Harsimardeep Gill</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sehgal%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sunish Sehgal</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sunish Sehgal</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Agronomy, Horticulture, and Plant Science, South Dakota State University, Brookings, South Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sehgal%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sunish Sehgal</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Faris%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Justin D Faris</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Justin D Faris</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Edward T. Schafer Agricultural Research Center, USDA‐ARS, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Faris%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">Justin D Faris</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gill%20U%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Upinder Gill</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Upinder Gill</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gill%20U%22%5BAuthor%5D" class="usa-link"><span class="name western">Upinder Gill</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gupta%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Rajeev Gupta</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Rajeev Gupta</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Edward T. Schafer Agricultural Research Center, USDA‐ARS, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gupta%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rajeev Gupta</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270071-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Plant Pathology, North Dakota State University, Fargo, North Dakota, USA</div>
<div id="tpg270071-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Agronomy, Horticulture, and Plant Science, South Dakota State University, Brookings, South Dakota, USA</div>
<div id="tpg270071-aff-0003">
<sup>
<sup>3</sup>
</sup>Edward T. Schafer Agricultural Research Center, USDA‐ARS, Fargo, North Dakota, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Rajeev Gupta, Edward T. Schafer Agricultural Research Center, USDA‐ARS, Fargo, ND, USA. Email: <span>Rajeev.Gupta@usda.gov</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 May 9; Received 2024 Dec 6; Accepted 2025 Jun 6; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America. This article has been contributed to by U.S. Government employees and their work is in the public domain in the USA.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12241833  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40635288/" class="usa-link">40635288</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Heat stress can reduce the production potential of wheat (<em>Triticum aestivum</em> L.) by affecting the various developmental stages of wheat including the seedling stage. Understanding the genetic basis of heat stress tolerance can help in breeding resilient wheat cultivars with improved productivity. Here, evaluation of a diverse panel of spring wheat landraces and cultivars under non‐heat stress (23°C) and heat stress (36°C) treatments in a controlled environment revealed large phenotypic and genetic variations. Heat stress negatively affected all seedling traits with the maximum reduction in root length (85.6%) and the least reduction in coleoptile length (15.44%). Moreover, based on seedling performance, we identified six highly heat tolerant (PI 366905, Kzyl Sark, Rang, Perico S, Bohr Gamh, and PI 620689) and six highly heat susceptible (CItr 17470, CItr 13270, Coeruleum, Shashi, Hallany, and Currawa) genotypes. Genome‐wide association analysis using 302,524 single nucleotide polymorphisms identified 23 marker‐trait associations (MTAs), of which 16 were associated with various seedling traits under heat stress. Gene annotation and expression analysis indicated 35 differentially expressed genes, of which 13 were considered as high‐confidence genes with functional relevance to heat stress including protein kinase, basic‐leucine zipper, UDP‐glucosyltransferase, pyrophosphate‐energized proton pump, fatty acid hydroxylase, and other classes of proteins. The MTAs and candidate genes identified in this study hold promise for developing heat‐resilient wheat cultivars through the selection of favorable alleles with gene‐specific molecular markers.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270071-list-10001" class="list" style="list-style-type:disc">
<li><p>Evaluating global spring wheat diversity panel identified highly heat‐tolerant lines.</p></li>
<li><p>A genome‐wide association studies (GWAS) identified 23 marker‐trait associations (MTAs) associated with different seedling traits in wheat.</p></li>
<li><p>Expression analysis pinpointed candidate genes (CGs) putatively involved in heat stress tolerance.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270071-dl-0001">
<dt>BLUE</dt>
<dd><p>best linear unbiased estimates</p></dd>
<dt>CGs</dt>
<dd><p>candidate genes</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association studies</p></dd>
<dt>MLM</dt>
<dd><p>mixed linear model</p></dd>
<dt>MTAs</dt>
<dd><p>marker‐trait associations</p></dd>
<dt>QTLs</dt>
<dd><p>quantitative trait loci</p></dd>
</dl>
<section id="tpg270071-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Wheat (<em>Triticum aestivum</em> L.) is the third most grown food crop of the United States following corn and soybean with an annual production of 1.65 billion bushels from 47.2 million acres (<a href="https://www.nass.usda.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nass.usda.gov/</a>). North Dakota is the second largest producer of wheat in the United States with production totaling 307 million bushels during 2023–2024 (<a href="https://www.ndwheat.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ndwheat.com</a>). However, global wheat production is constrained by several biotic (such as rust, Fusarium head blight, powdery mildew, tan spot, etc.) and abiotic stresses (such as heat, drought, salinity, ion toxicity, etc.) (Rajamanickam et al., <a href="#tpg270071-bib-0042" class="usa-link" aria-describedby="tpg270071-bib-0042">2024</a>; Gudi et al., <a href="#tpg270071-bib-0055" class="usa-link" aria-describedby="tpg270071-bib-0055">2025</a>). Heat stress is the major abiotic stress that negatively impacts wheat growth at various developmental stages. For instance, heat stress at the early developmental stages affects seed germination and seedling establishment (Farhad et al., <a href="#tpg270071-bib-0016" class="usa-link" aria-describedby="tpg270071-bib-0016">2023</a>; Gudi, Halladakeri, et al., <a href="#tpg270071-bib-0020" class="usa-link" aria-describedby="tpg270071-bib-0020">2024</a>). Heat stress reduces the photosynthetic efficiency, respiration, stomatal conductance, net assimilation rate, and water use efficiency (Coast et al., <a href="#tpg270071-bib-0006" class="usa-link" aria-describedby="tpg270071-bib-0006">2024</a>; Gudi et al., <a href="#tpg270071-bib-0022" class="usa-link" aria-describedby="tpg270071-bib-0022">2023</a>). Additionally, heat stress ruptures cell membranes and inactivates key enzymes involved in growth and development, thereby reducing photo‐assimilation, shoot and root biomass production, and tiller number per unit area (Djanaguiraman et al., <a href="#tpg270071-bib-0011" class="usa-link" aria-describedby="tpg270071-bib-0011">2020</a>; Dwivedi et al., <a href="#tpg270071-bib-0013" class="usa-link" aria-describedby="tpg270071-bib-0013">2024</a>; Gudi, Jain, et al., <a href="#tpg270071-bib-0021" class="usa-link" aria-describedby="tpg270071-bib-0021">2024</a>). Heat stress also results in the increased production of reactive oxygen species, which are detrimental to cellular metabolisms, such as damage to nucleic acids and membrane peroxidation (Abasi et al., <a href="#tpg270071-bib-0001" class="usa-link" aria-describedby="tpg270071-bib-0001">2024</a>; Bita &amp; Gerats, <a href="#tpg270071-bib-0004" class="usa-link" aria-describedby="tpg270071-bib-0004">2013</a>).</p>
<p>Additionally, in the later developmental stages, heat stress leads to pollen sterility and subsequently decreases grain number per spike, grain‐filling duration, and grain quality (Sihag et al., <a href="#tpg270071-bib-0045" class="usa-link" aria-describedby="tpg270071-bib-0045">2024</a>; Singh et al., <a href="#tpg270071-bib-0046" class="usa-link" aria-describedby="tpg270071-bib-0046">2023</a>; Tanin et al., <a href="#tpg270071-bib-0047" class="usa-link" aria-describedby="tpg270071-bib-0047">2022</a>). For instance, exposing wheat to high temperature at the anthesis stage linearly reduced the floret fertility from 85% at 24°C to 0% (i.e., complete abortion of pollen grains) at 35°C (Prasad &amp; Djanaguiraman, <a href="#tpg270071-bib-0039" class="usa-link" aria-describedby="tpg270071-bib-0039">2014</a>). Studies have also reported that every 1°C increase in temperature during anthesis and grain filling reduces grain yield by 6% (Zhao et al., <a href="#tpg270071-bib-0054" class="usa-link" aria-describedby="tpg270071-bib-0054">2017</a>) and harvest index by 5% (Lawlor &amp; Mitchell, <a href="#tpg270071-bib-0030" class="usa-link" aria-describedby="tpg270071-bib-0030">2000</a>) in wheat. Additionally, heat stress reduces endosperm counts in the developing grains, thereby reducing the grain size and 1000‐grain weight, which results in reduced grain quality (Girousse, <a href="#tpg270071-bib-0019" class="usa-link" aria-describedby="tpg270071-bib-0019">2023</a>).</p>
<p>Heat stress threatens global food security by reducing wheat production in major wheat‐growing areas of the world. Therefore, it is necessary to understand the genetic basis of heat stress tolerance as well as to develop heat‐resilient wheat cultivars. A first step toward achieving this is to tap into native genetic and phenotypic diversity to identify candidate genomic regions associated with heat stress tolerance. Several genome‐wide association studies (GWAS) and/or quantitative trait loci (QTLs) studies have been conducted in wheat to identify significant marker‐trait associations (MTAs) or QTLs associated with various agronomic, physiological, and biochemical traits under heat stress (Devate et al., <a href="#tpg270071-bib-0008" class="usa-link" aria-describedby="tpg270071-bib-0008">2022</a>; Gudi et al., <a href="#tpg270071-bib-0023" class="usa-link" aria-describedby="tpg270071-bib-0023">2022</a>; Khan et al., <a href="#tpg270071-bib-0029" class="usa-link" aria-describedby="tpg270071-bib-0029">2022</a>; Maulana et al., <a href="#tpg270071-bib-0035" class="usa-link" aria-describedby="tpg270071-bib-0035">2018</a>; Muhammad et al., <a href="#tpg270071-bib-0036" class="usa-link" aria-describedby="tpg270071-bib-0036">2020</a>; Qaseem et al., <a href="#tpg270071-bib-0041" class="usa-link" aria-describedby="tpg270071-bib-0041">2019</a>; Tanin et al., <a href="#tpg270071-bib-0048" class="usa-link" aria-describedby="tpg270071-bib-0048">2023</a>; Tian et al., <a href="#tpg270071-bib-0050" class="usa-link" aria-describedby="tpg270071-bib-0050">2023</a>). Most of these studies were conducted under terminal heat stress or at the grain‐filling stage with only few studies focusing on seedling heat stress tolerance (Fu et al., <a href="#tpg270071-bib-0017" class="usa-link" aria-describedby="tpg270071-bib-0017">2024</a>; Khan et al., <a href="#tpg270071-bib-0029" class="usa-link" aria-describedby="tpg270071-bib-0029">2022</a>; Maulana et al., <a href="#tpg270071-bib-0035" class="usa-link" aria-describedby="tpg270071-bib-0035">2018</a>). Despite these efforts, the genetic mechanism(s) governing heat stress tolerance is not thoroughly understood. Moreover, most of these studies failed to identify the beneficial alleles and to develop the functional markers that can be utilized to develop heat‐resilient wheat cultivars. Therefore, it is necessary to identify the source of seedling heat stress‐tolerant lines and their genomic regions putatively involved in improved heat stress tolerance. To accomplish this, we evaluated a diverse panel of hexaploid spring wheat accessions, including landraces and cultivars, under heat stress and identified extremely heat‐tolerant lines. Moreover, a comprehensive GWAS, candidate gene (CG) mining underneath significant MTAs, and expression analysis helped us to identify the potential CGs involved in improved seedling heat stress tolerance.</p></section><section id="tpg270071-sec-0020"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270071-sec-0030"><h3 class="pmc_sec_title">2.1. Plant materials</h3>
<p>The experimental material consisted of 216 hexaploid wheat accessions with spring type growth habit (Table <a href="#tpg270071-supinfo-0001" class="usa-link">S1</a>). These germplasm lines were selected from the exome‐sequenced wheat panel, which consisted of diverse accessions of tetraploid and hexaploid wheat lines belonging to landraces and cultivars (He et al., <a href="#tpg270071-bib-0026" class="usa-link" aria-describedby="tpg270071-bib-0026">2019</a>). Seed materials were sourced from the United States Department of Agriculture‐National Small Grains Collection gene bank and grown for one round of purification and seed increase. Genotypic data were accessed from the online repository (<a href="http://wheatgenomics.plantpath.ksu.edu/1000EC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://wheatgenomics.plantpath.ksu.edu/1000EC</a>).</p></section><section id="tpg270071-sec-0040"><h3 class="pmc_sec_title">2.2. Evaluation for seedling heat stress tolerance</h3>
<p>The experiment was conducted using a two‐factor randomized complete block design with three replications. Seeds from each genotype were surface‐sterilized using 10% Clorox (for 20 min) and 70% ethanol (for 5 min) followed by rinsing three times with deionized water (diH<sub>2</sub>O). Disinfected seeds were kept in refrigerator (4°C) for 48 h to ensure uniform seed germination. Seeds from each genotype were sown in six cones (30 cm length and 5‐cm diameter) filled with autoclaved vermiculite. Three of the six cones representing three replications were placed in a growth chamber adjusted to 23°C (i.e., non‐heat stress treatment; T1), and the remaining three cones were placed in a separate growth chamber adjusted to 36°C (i.e., heat stress treatment; T2). Once the seeds germinated, growth chambers were adjusted to a 16‐h photoperiod (i.e., 16 h of light and 8 h of dark) for 12 days. On the 13th day, data on seedling characteristics such as shoot length (SL; cm), root length (RL; cm), root number (RN), coleoptile length (CL; cm), shoot fresh weight (SFW; mg), and root fresh weight (RFW; mg) were recorded. Precautions were taken to avoid water stress by providing sufficient amounts of water (via water containers) at 23°C for the non‐heat stress treatment (T1) and 36°C for the heat stress treatment (T2). Furthermore, containers were replaced with fresh water after every third day to avoid fungal growth.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270071-sec-0050"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270071-list-0001" class="list" style="list-style-type:disc">
<li><p>Evaluating global spring wheat diversity panel identified highly heat‐tolerant lines.</p></li>
<li><p>A genome‐wide association studies (GWAS) identified 23 marker‐trait associations (MTAs) associated with different seedling traits in wheat.</p></li>
<li><p>Expression analysis pinpointed candidate genes (CGs) putatively involved in heat stress tolerance.</p></li>
</ul></section></section></section><section id="tpg270071-sec-0060"><h3 class="pmc_sec_title">2.3. Phenotypic data analysis</h3>
<p>Phenotypic data collected on seedling traits under non‐heat stress (T1) and heat stress (T2) treatments were subjected to mixed linear model (MLM) using META‐R. The META‐R relies on LME4 R‐package (Alvarado et al., <a href="#tpg270071-bib-0003" class="usa-link" aria-describedby="tpg270071-bib-0003">2020</a>) to calculate the genotypic best linear unbiased estimates (BLUEs) for individual treatments. BLUE values were estimated by using the following model:
</p>
<table class="disp-formula p" id="tpg270071-disp-0001"><tr><td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>G</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>R</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></td></tr></table>
<p>where, <em>Y<sub>ij</sub>
</em> is the trait of interest, <em>μ</em> is the grand mean, <em>G<sub>i</sub>
</em> is the effect of <em>i</em>th genotype, <em>R<sub>j</sub>
</em> is the effect of <em>j</em>th replication, <em>e<sub>ij</sub>
</em> is the error associated with <em>i</em>th genotype and <em>j</em>th replication, which is assumed to be normally and independently distributed, with mean zero and homogeneity of variances as <em>σ</em>
<sup>2</sup>.</p>
<p>The broad‐sense heritability (h<sup>2</sup>
<sub>bs</sub>) for seedling traits was calculated by using the following formula in META‐R:
</p>
<table class="disp-formula p" id="tpg270071-disp-0002"><tr><td class="formula"><math id="jats-math-2" display="block"><mrow><mrow><msup><msub><mi>h</mi><mi mathvariant="bold">bs</mi></msub><mn>2</mn></msup><mo linebreak="badbreak">=</mo><mfrac><msup><msub><mi mathvariant="normal">σ</mi><mi mathvariant="bold">g</mi></msub><mn>2</mn></msup><mrow><msup><msub><mi mathvariant="normal">σ</mi><mi mathvariant="bold">g</mi></msub><mn>2</mn></msup><mo>+</mo><msup><msub><mi mathvariant="normal">σ</mi><mi mathvariant="bold">e</mi></msub><mn>2</mn></msup><mo>/</mo><mi>R</mi></mrow></mfrac></mrow></mrow></math></td></tr></table>
<p>where, <em>h</em>
<sub>bs</sub>
<sup>2</sup> is the broad‐sense heritability, <em>σ</em>
<sub>g</sub>
<sup>2</sup> is the genetic variance, <em>σ</em>
<sub>e</sub>
<sup>2</sup> error variance, <em>R</em> is the number of replications.</p>
<p>BLUE values from heat stress treatment were divided from those from non‐heat stress treatments for each genotype to check the impact of heat stress. Moreover, this ratio was used to select extremely tolerant and sensitive genotypes for heat stress. We used paired <em>t</em>‐test to check the significant difference for each genotype under non‐heat stress and heat stress.</p>
<p>Pearson's correlation coefficient analysis among the seedling traits was assessed based on BLUEs using the “Corrplot” package built in the RStudio (R Core Team, <a href="#tpg270071-bib-0049" class="usa-link" aria-describedby="tpg270071-bib-0049">2018</a>).</p></section><section id="tpg270071-sec-0070"><h3 class="pmc_sec_title">2.4. Single nucleotide polymorphisms (SNPs) genotyping and population statistics</h3>
<p>The wheat accessions used in the present investigation were previously genotyped by capturing the exonic regions using the NimbleGen SeqCap EZ Reagent Kit Plus v2 (Roche) and sequenced on an Illumina HiSeq2000 instrument (He et al., <a href="#tpg270071-bib-0026" class="usa-link" aria-describedby="tpg270071-bib-0026">2019</a>). Variant calling file containing a filtered set of single nucleotide polymorphisms (SNPs) (i.e., 3,573,809) was used to extract the genotypic information for the selected genotypes. Obtained genotypic information was subjected to data filtering using “TASSEL v5.2.84” to remove the SNPs with minor allele frequency of &lt;5%, missing data of &gt;20%, and heterozygosity of &gt;20%. Beagle v4.1 was used to assign the missing genotypes from selected SNPs (Browning &amp; Browning, <a href="#tpg270071-bib-0005" class="usa-link" aria-describedby="tpg270071-bib-0005">2007</a>). Finally, the imputed SNPs (i.e., 302,524) were used for genome‐wide association analysis.</p>
<p>Population stratification was analyzed using Bayesian model‐based clustering, unsupervised principal component analysis (PCA), and kinship analysis. “STRUCTURE v.2.3.4” software was used for Bayesian model‐based clustering. In brief, structure analysis was done by setting the number of fixed sub‐populations to 10 (i.e., <em>k</em> = 10) with 10,000 MCMC (Markov chain Monte Carlo) replications and 10,000 burn‐in iterations (Pritchard et al., <a href="#tpg270071-bib-0040" class="usa-link" aria-describedby="tpg270071-bib-0040">2000</a>). Then the web‐based program “STRUCTURE HARVESTER v0.6.9450” was used to determine the most likely number of sub‐populations by plotting ∆K and number of sub‐populations (<a href="https://taylor0.biology.ucla.edu/structureHarvester/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://taylor0.biology.ucla.edu/structureHarvester/</a>). The unsupervised PCA was performed in “PLINK,” and the scatter plot showing grouping of genotypes was generated using R‐package. Kinship analysis was carried out to see the familial relationship among the genotypes using “GAPIT” through pairwise SNP comparison. Kinship plot depicting the genetic relatedness among genotypes was generated from square symmetric matrix.</p>
<p>Short‐range linkage disequilibrium (LD) was assessed by calculating squared allelic correlation (<em>r</em>
<sup>2</sup>) between SNP pairs within a 5000 Kb distance by utilizing “PopLDdecay software.” The LD‐decay plot was generated by (i) plotting estimated <em>r</em>
<sup>2</sup> values against the genetic distance (in Kb) between SNPs and (ii) calculating the LD decay by fitting smooth locally weighted polynomial regression (LOESS) curve. Point at which the critical <em>r</em>
<sup>2</sup> value of 0.2 intersects with the LOESS curve was identified as the LD decay point. LD decay analysis was conducted at the sub‐genomes and across the whole genome.</p></section><section id="tpg270071-sec-0080"><h3 class="pmc_sec_title">2.5. Association analysis and CG mining</h3>
<p>Genome‐wide association analysis was carried out by utilizing the filtered set of SNPs (302,524) to identify significant MTAs. Analysis was performed using the general‐linear model (GLM), MLM, fixed and random model circulating probability unification (FarmCPU), and Bayesian information and LD iteratively nested keyway (BLINK) built in the R‐environment, “GAPIT v 3.0” (Genome association and prediction integrated tool) (Wang &amp; Zhang, <a href="#tpg270071-bib-0051" class="usa-link" aria-describedby="tpg270071-bib-0051">2021</a>). GLM and MLM control the false positives by incorporating the cofactors such as population structure (<em>Q</em>) in GLM and population structure (<em>Q</em>) and kinship (<em>K</em>) in the MLM. FarmCPU is a multi‐locus model that reduces false negatives by eliminating the confounding effects generated between test markers and the covariates (X. Liu et al., <a href="#tpg270071-bib-0032" class="usa-link" aria-describedby="tpg270071-bib-0032">2016</a>). To avoid the overfitting of testing markers, FarmCPU considers associated markers as the fixed effects in the fixed effect model and was optimized in a maximum likelihood method in the MLM. BLINK improves the statistical power by employing LD and uses the Bayesian information content in a fixed‐effect model to overcome the computational load. Comparing all four models using quantile–quantile plots (Q–Q plots) suggested that FarmCPU performed better in fixing false positive and false negative associations. Therefore, FarmCPU was employed to identify significant MTAs. The Bonferroni correction's threshold for declaring significant MTAs is typically too stringent as it considers all SNPs in the dataset rather than independent tests. Therefore, we used the less stringent, arbitrary threshold of −log<sub>10</sub>(<em>p</em>) ≥ 5.0 to consider significant MTAs as suggested by Halder et al. (<a href="#tpg270071-bib-0024" class="usa-link" aria-describedby="tpg270071-bib-0024">2023</a>).</p>
<p>CG models within a 1 Mb region (500 Kb on either side) of each MTA were extracted using the BioMart tool present in Ensembl Plants (<a href="https://plants.ensembl.org/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plants.ensembl.org/index.html</a>). The 1 Mb window size for CG analysis was decided based on the results of LD decay analysis, which indicated an average whole genome LD block size of 1.5 Mb. Moreover, previous studies also suggested using a 1 Mb genomic region around significant MTA or QTL for CG analysis (Halder et al., <a href="#tpg270071-bib-0024" class="usa-link" aria-describedby="tpg270071-bib-0024">2023</a>; Halladakeri et al., <a href="#tpg270071-bib-0025" class="usa-link" aria-describedby="tpg270071-bib-0025">2023</a>). Functional descriptions of the identified CG models were obtained from the InterPro database (<a href="https://www.ebi.ac.uk/interpro/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/interpro/</a>). The extracted CGs were subjected to in silico expression analysis using the wheat expression browser (i.e., ExpVIP) (<a href="http://www.wheat-expression.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.wheat‐expression.com/</a>). For this purpose, the expression dataset involving the transcriptomic analysis of seedling heat stress (1 and 6 h) in the heat‐tolerant wheat variety TAM‐107 was used (Z. Liu et al., <a href="#tpg270071-bib-0033" class="usa-link" aria-describedby="tpg270071-bib-0033">2015</a>). Finally, the CGs with at least two‐fold change in the expression value were considered as differentially expressed genes (DEGs) as suggested in Gudi et al. (<a href="#tpg270071-bib-0023" class="usa-link" aria-describedby="tpg270071-bib-0023">2022</a>).</p></section></section><section id="tpg270071-sec-0090"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270071-sec-0100"><h3 class="pmc_sec_title">3.1. Phenotypic evaluation of wheat seedlings for heat stress tolerance</h3>
<p>Hexaploid wheat accessions selected from the exome‐sequenced panel were assessed for seedling heat stress tolerance by measuring multiple seedling traits (SL, RL, CL, RN, SFW, and RFW) under heat stress (36°C) and non‐heat stress (23°C) treatments. The BLUE values estimated from the phenotypic data showed significant variations for all seedling traits under non‐heat stress (23°C) and heat stress (36°C) treatments (Table <a href="#tpg270071-tbl-0001" class="usa-link">1</a>; Figure <a href="#tpg270071-fig-0001" class="usa-link">1</a>). Higher phenotypic variation was observed under non‐heat stress (23°C) treatment than under heat stress (36°C) for most of the seedling traits (except CL and RN). Heat stress negatively affected all seedling traits with the maximum effect on RL (85.6% reduction) and the minimum effect on CL (15.44% reduction). Furthermore, a significant increase in RN (20%) was observed under heat stress compared to the non‐heat stress treatment. To check the stability of genotype performance under heat stress, we compared the top 25% (i.e., 54 genotypes) ranked under non‐heat stress with their ranking under heat stress for each trait. Among these top performers, only 22 genotypes maintained their top 25% ranking under heat stress for CL, 15 for SL, 16 for RL, 22 for RN, 24 for SFW, and 19 for RFW (Figure <a href="#tpg270071-fig-0001" class="usa-link">1</a>). The remaining genotypes failed to maintain their ranking under heat stress, indicating a reduction in performance for these genotypes. Moreover, based on performance of genotypes under heat stress, we identified six highly tolerant (i.e., PI 366905, Kzyl Sark, Rang, Perico S, Bohr Gamh, and PI 620689) and six highly susceptible (i.e., CItr 17470, CItr 13270, Coeruleum, Shashi, Hallany, and Currawa) genotypes (Figure <a href="#tpg270071-fig-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="tpg270071-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Summary statistics for the seedling traits under non‐heat stress (23°C) and heat stress (36°C) treatments.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Statistic</th>
<th align="left" rowspan="1" colspan="1">CL</th>
<th align="left" rowspan="1" colspan="1">SL</th>
<th align="left" rowspan="1" colspan="1">RL</th>
<th align="left" rowspan="1" colspan="1">RN</th>
<th align="left" rowspan="1" colspan="1">SFW</th>
<th align="left" rowspan="1" colspan="1">RFW</th>
</tr></thead>
<tbody>
<tr><td colspan="7" align="left" rowspan="1">
<strong>Non‐heat stress (23°C)</strong>
</td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">Mean ± LSD</td>
<td align="left" rowspan="1" colspan="1">2.72 ± 0.5***</td>
<td align="left" rowspan="1" colspan="1">18.92 ± 4***</td>
<td align="left" rowspan="1" colspan="1">33.76 ± 6.05***</td>
<td align="left" rowspan="1" colspan="1">5.35 ± 1.2***</td>
<td align="left" rowspan="1" colspan="1">13.42 ± 4.94***</td>
<td align="left" rowspan="1" colspan="1">23.23 ± 4.24***</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Range</td>
<td align="left" rowspan="1" colspan="1">1.87–3.68</td>
<td align="left" rowspan="1" colspan="1">10.8–31.92</td>
<td align="left" rowspan="1" colspan="1">17.33–48.5</td>
<td align="left" rowspan="1" colspan="1">3.43–7.67</td>
<td align="left" rowspan="1" colspan="1">1.14–23.4</td>
<td align="left" rowspan="1" colspan="1">4–46.82</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CV (%)</td>
<td align="left" rowspan="1" colspan="1">11.55</td>
<td align="left" rowspan="1" colspan="1">13.17</td>
<td align="left" rowspan="1" colspan="1">11.17</td>
<td align="left" rowspan="1" colspan="1">13.94</td>
<td align="left" rowspan="1" colspan="1">22.96</td>
<td align="left" rowspan="1" colspan="1">11.39</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Heritability</td>
<td align="left" rowspan="1" colspan="1">0.77</td>
<td align="left" rowspan="1" colspan="1">0.83</td>
<td align="left" rowspan="1" colspan="1">0.89</td>
<td align="left" rowspan="1" colspan="1">0.66</td>
<td align="left" rowspan="1" colspan="1">0.82</td>
<td align="left" rowspan="1" colspan="1">0.96</td>
</tr>
<tr><td colspan="7" align="left" rowspan="1">
<strong>Heat stress (36°C)</strong>
</td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">Grand mean</td>
<td align="left" rowspan="1" colspan="1">2.3 ± 0.46***</td>
<td align="left" rowspan="1" colspan="1">11.38 ± 2.81***</td>
<td align="left" rowspan="1" colspan="1">4.86 ± 1.49***</td>
<td align="left" rowspan="1" colspan="1">6.42 ± 1.8***</td>
<td align="left" rowspan="1" colspan="1">7.38 ± 3.04***</td>
<td align="left" rowspan="1" colspan="1">4.18 ± 2.64***</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Range</td>
<td align="left" rowspan="1" colspan="1">1.1–3.42</td>
<td align="left" rowspan="1" colspan="1">5.17–20.67</td>
<td align="left" rowspan="1" colspan="1">1.6–10.6</td>
<td align="left" rowspan="1" colspan="1">3.67–9.67</td>
<td align="left" rowspan="1" colspan="1">1.33–17</td>
<td align="left" rowspan="1" colspan="1">0.4–16</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CV (%)</td>
<td align="left" rowspan="1" colspan="1">12.39</td>
<td align="left" rowspan="1" colspan="1">15.37</td>
<td align="left" rowspan="1" colspan="1">19.15</td>
<td align="left" rowspan="1" colspan="1">17.51</td>
<td align="left" rowspan="1" colspan="1">25.69</td>
<td align="left" rowspan="1" colspan="1">39.42</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Heritability</td>
<td align="left" rowspan="1" colspan="1">0.82</td>
<td align="left" rowspan="1" colspan="1">0.83</td>
<td align="left" rowspan="1" colspan="1">0.93</td>
<td align="left" rowspan="1" colspan="1">0.71</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
<td align="left" rowspan="1" colspan="1">0.85</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Percent reduction</strong>
</td>
<td align="left" rowspan="1" colspan="1">15.44</td>
<td align="left" rowspan="1" colspan="1">39.85</td>
<td align="left" rowspan="1" colspan="1">85.6</td>
<td align="left" rowspan="1" colspan="1">−20</td>
<td align="left" rowspan="1" colspan="1">45.01</td>
<td align="left" rowspan="1" colspan="1">82.01</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270071-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270071-tbl1-note-0001"><p>Abbreviations: CL, coleoptile length (cm); CV, coefficient of variance; LSD, least significant difference; RFW: root fresh weight (mg); RN, root number; RL, root length (cm); SFW, shoot fresh weight (mg); SL, shoot length (cm).</p></div>
<div class="fn" id="tpg270071-tbl1-note-0002"><p>***Significant at <em>p</em> &lt; 0.001.</p></div>
</div></section><figure class="fig xbox font-sm" id="tpg270071-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ee2c/12241833/3357887a884f/TPG2-18-e70071-g003.jpg" loading="lazy" id="jats-graphic-1" height="696" width="700" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270071-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypic variation for seedling traits under non‐heat stress (23°C) (blue) and heat stress (36°C) (green) treatment: (a) coleoptile length (cm); (b) shoot length (cm); (c) root length (cm); (d) root number; (e) shoot fresh weight (mg); (f) root fresh weight (mg). In each treatment, the kernel density plot represents the data distribution; box plot represents the minimum, first quartile, mean/median, third quartile, and maximum values in the dataset; and dots represent the individual genotypes. The gray lines depict the performance of same genotype in non‐heat stress and heat stress treatments with connecting dots. ***Significant at <em>p</em> &lt; 0.001.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270071-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ee2c/12241833/71f6d690deac/TPG2-18-e70071-g005.jpg" loading="lazy" id="jats-graphic-3" height="714" width="800" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270071-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparing the performance of six highly tolerant (light green box) and six highly sensitive (light blue box) lines under non‐heat stress (23°C) (blue bars) and heat stress (36°C) (orange bars) treatments for (a) coleoptile length (cm); (b) shoot length (cm); (c) root length (cm); (d) root number; (e) shoot fresh weight (mg); (f) root fresh weight (mg). A paired <em>t</em>‐test was used to check the level of significance between non‐heat stress (23°C) and heat stress (36°C) treatment for each genotype (level of significance: * &lt; 0.05%; ** &lt; 0.01%; *** &lt; 0.001; ns, non‐significance).</p></figcaption></figure><p>The h<sup>2</sup>
<sub>bs</sub> estimated using BLUE values revealed the different genetic architecture of seedling traits under non‐heat stress and heat stress treatments (Table <a href="#tpg270071-tbl-0001" class="usa-link">1</a>). The maximum and minimum heritability was observed for RFW (96%) and RN (66%), respectively, under the non‐heat stress treatment. However, under heat stress, the maximum heritability was observed for RL (93%) and the minimum heritability was observed for RN (71%). Furthermore, the heritability estimates were higher for all seedling traits (except RFW) under heat stress compared to the non‐heat stress treatment. Pearson's correlation coefficient analysis revealed a significant positive association (<em>p</em>‐value &lt; 0.05) among seedling traits under non‐heat stress and heat stress treatments as well as for the combined dataset except for RL with CL, RN, and SFW under non‐heat stress and combined dataset (Figure <a href="#tpg270071-fig-0003" class="usa-link">3a–c</a>). Correlation coefficient analysis revealed a notable shift in the trait relationship under non‐heat stress and heat stress treatments. Most prominent changes were observed for SL versus RL and SFW versus RL (Figure <a href="#tpg270071-fig-0003" class="usa-link">3a,b</a>). Under non‐heat stress, SL and RL showed a weak association (<em>r</em> = 0.18; <em>p</em>‐value &lt; 0.01), whereas this relationship was strengthened under heat stress treatment (<em>r</em> = 0.58; <em>p</em>‐value &lt; 0.001). Likewise, RL was not correlated with SFW under non‐heat stress, but showed a strong positive correlation under heat stress (<em>r</em> = 0.18; <em>p</em>‐value &lt; 0.01). These results suggests that genotypes maintaining long RL will support the better shoot growth and biomass accumulation under heat stress. Correlation results were further confirmed by PCA biplots, which showed higher levels of association among the seedling traits (Figure <a href="#tpg270071-fig-0003" class="usa-link">3d–f</a>).</p>
<figure class="fig xbox font-sm" id="tpg270071-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ee2c/12241833/4fa2e0192af7/TPG2-18-e70071-g004.jpg" loading="lazy" id="jats-graphic-5" height="536" width="800" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270071-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pearson's correlation coefficient among the seedling traits under non‐heat stress (23°C) (a), heat stress (36°C) (b), and combined dataset from non‐heat stress and heat stress treatments (c); and the principal component analysis (PCA) analysis for seedling traits under non‐heat stress (d), heat stress (e), and combined dataset from non‐heat stress (23°C) and heat stress (36°C) treatments (f). CL, coleoptile length (cm); RFW, root fresh weight (mg); RN, root number; RL, root length (cm); SFW, shoot fresh weight (mg); SL, shoot length (cm). *Significant at <em>p</em> &lt; 0.05; **Significant at <em>p</em> &lt; 0.01; ***Significant at <em>p</em> &lt; 0.001.</p></figcaption></figure></section><section id="tpg270071-sec-0110"><h3 class="pmc_sec_title">3.2. Genotypic information, population structure, and LD analysis</h3>
<p>Of the total 7.3 million SNPs, 302,524 filtered SNPs were used for the GWAS. The chromosome with the most SNPs was chromosome 2B (i.e., 26,385), whereas chromosome 4D had the least (i.e., 919) (Table <a href="#tpg270071-supinfo-0001" class="usa-link">S2</a>). Among the sub‐genomes, the B‐genome had the highest number of SNPs (i.e., 148,165) and the D‐genome had the lowest number of SNPs (i.e., 27,052) (Table <a href="#tpg270071-supinfo-0001" class="usa-link">S2</a>).</p>
<p>Bayesian model‐based clustering and PCA revealed two subgroups in the association panel (Figure <a href="#tpg270071-fig-0004" class="usa-link">4a,b</a>). The first two PCs explained 28.3% (i.e., 17.5% by PC1 and 10.8% by PC2) of the total genetic variation. These PCs were used to differentiate the genotypes based on type (i.e., cultivated, landraces, and unknown) and origin (i.e., continent wise) (Figure <a href="#tpg270071-fig-0004" class="usa-link">4c</a>). Group I consisted mainly of landraces collected from Asia and Africa with a few genotypes collected from Europe and the Americas (Figure <a href="#tpg270071-fig-0004" class="usa-link">4d</a>). However, Group II included both cultivated and landraces collected from Europe and the Americas with a few genotypes from Asia and Africa. These results were further confirmed by kinship (<em>K</em>) analysis using “GAPIT,” which revealed two subgroups in the association panel (Figure <a href="#tpg270071-fig-0004" class="usa-link">4e</a>). The LD‐decay analysis using critical <em>r</em>
<sup>2</sup> value identified the LD block size of 1.5 Mb at the whole genome level (Figure <a href="#tpg270071-supinfo-0001" class="usa-link">S1</a>). However, the LD‐decay analysis showed a differential pattern across the three sub‐genomes; that is, with the A (∼1.2 Mb) and B (∼1.3 Mb) sub‐genomes showed smaller LD block sizes compared to that of the D sub‐genome (i.e., &gt;9 Mb) (Figure <a href="#tpg270071-supinfo-0001" class="usa-link">S1</a>).</p>
<figure class="fig xbox font-sm" id="tpg270071-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ee2c/12241833/cceacbc8a4e0/TPG2-18-e70071-g006.jpg" loading="lazy" id="jats-graphic-7" height="462" width="800" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270071-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Population structure analysis of the diverse hexaploid spring wheat collection: (a) Evanno plot of Delta‐K statistic obtained from Bayesian model‐based clustering in the STRUCTURE software; (b) scree plot for first 20 principal components obtained from principal component analysis (PCA); (c) PCA‐based scatter plot showing the grouping of wheat accessions based on type of germplasm lines; (d) PCA‐based scatter plot showing the grouping of wheat accessions based on origin; (e) kinship plot showing the familial relationship among the wheat germplasm lines.</p></figcaption></figure></section><section id="tpg270071-sec-0120"><h3 class="pmc_sec_title">3.3. Genome‐wide association analysis</h3>
<p>The BLUE values obtained from phenotypic data on seedling traits were subjected to GWAS using GLM, MLM, FarmCPU, and BLINK. However, comparing GWAS results using Q–Q plots showed better results for FarmCPU, and hence we proceed with FarmCPU. GWAS identified 23 significant MTAs associated with different seedling traits, of which 21 are unique MTAs. Moreover, among 23 MTAs, seven were identified under non‐heat stress and 16 under heat stress treatment (Table <a href="#tpg270071-tbl-0001" class="usa-link">1</a>; Figure <a href="#tpg270071-supinfo-0001" class="usa-link">S2</a> and <a href="#tpg270071-supinfo-0001" class="usa-link">S3</a>). Of the seven MTAs identified under non‐heat stress treatment, two were associated with SL, three were associated with RN, and two were associated with RFW (Figure <a href="#tpg270071-fig-0005" class="usa-link">5</a>). MTAs associated with RFW had the highest SNP effect of 4.68 (scaffold10127_4250444) and 6.19 (scaffold4077_4635398). Of the 16 MTAs identified under heat stress treatment, one was associated with SFW, eight were associated with RL, and seven were associated with RFW (Figure <a href="#tpg270071-fig-0006" class="usa-link">6</a>). Four of these MTAs, scaffold468‐1_279183, scaffold5611_406983, scaffold38798_23175, and scaffold146879_1644623, can be considered as most important MTAs, as they possess high allelic effect of &gt;1 (Table <a href="#tpg270071-tbl-0002" class="usa-link">2</a>). Interestingly, a heat‐tolerant line, Bohr Gamh (PI 462106), collected from middle Asia possesses positive alleles for all these MTAs. Similarly, tolerant lines, PI 366905, have positive alleles for three MTAs (scaffold468‐1_279183, scaffold5611_406983, and scaffold38798_23175), Kzyl Sark has positive alleles for scaffold468‐1_279183, and Perico S has positive alleles for scaffold5611_406983. However, two tolerant lines, Rang and PI 620689, did not have positive alleles for these four MTAs, suggesting that there might be other genomic regions giving tolerance in these genotypes. Interestingly, it was noticed that all six sensitive genotypes possess negative alleles for these four MTAs, further supporting the possible involvement of these MTAs in heat stress tolerance. Furthermore, allelic effect estimation revealed that the favorable alleles of MTAs identified under non‐heat stress improved seedling performance under non‐heat stress only (Figure <a href="#tpg270071-supinfo-0001" class="usa-link">S4a–f</a>), except for MTA, scaffold4077_4635398, which improved seedling performance under both non‐heat stress and heat stress treatments (Figure <a href="#tpg270071-supinfo-0001" class="usa-link">S4g</a>). Similarly, of the 16 MTAs identified under heat stress, favorable alleles for 12 MTAs improved seedling performance under heat stress only (Figure <a href="#tpg270071-supinfo-0001" class="usa-link">S4</a>), whereas the favorable alleles of scaffold468‐1_279183 (Figure <a href="#tpg270071-supinfo-0001" class="usa-link">S4h</a>), scaffold21647_3085029 (Figure <a href="#tpg270071-supinfo-0001" class="usa-link">S4l</a>), scaffold2159_4802103 (Figure <a href="#tpg270071-supinfo-0001" class="usa-link">S4p</a>), and scaffold22822_3587896 (Figure <a href="#tpg270071-supinfo-0001" class="usa-link">S4s</a>) improved seedling performance under both non‐heat stress and heat stress treatments. Therefore, only 12 MTAs identified under heat stress whose favorable alleles did not change the performance across treatments are useful in breeding for heat tolerance. Furthermore, we estimated the additive effect of favorable alleles on the phenotypic performance, which indicated that the genotypes with more favorable alleles had higher performance under heat stress treatment (Figure <a href="#tpg270071-fig-0007" class="usa-link">7</a>).</p>
<figure class="fig xbox font-sm" id="tpg270071-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ee2c/12241833/63b814878edb/TPG2-18-e70071-g001.jpg" loading="lazy" id="jats-graphic-9" height="943" width="800" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270071-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Marker‐trait associations (MTAs) identified for seedling traits under non‐heat stress (23°C) treatment: (a) MTAs associated with shoot length (SL; cm); (b) MTAs associated with root number (RN); (c) MTAs associated with root fresh weight (RFW; mg).</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270071-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ee2c/12241833/1be6eb87ee6f/TPG2-18-e70071-g008.jpg" loading="lazy" id="jats-graphic-11" height="940" width="800" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270071-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Marker‐trait associations (MTAs) identified for seedling traits under heat stress (36°C) treatment: (a) MTAs associated with shoot fresh weight (SFW; mg); (b) MTAs associated with root length (RL; cm); (c) MTAs associated with root fresh weight (RFW; mg).</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270071-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>List of marker‐trait associations (MTAs) identified for seedling traits under non‐heat stress (23°C) and heat stress (36°C) treatments.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">MTAs/SNPs</th>
<th align="left" rowspan="1" colspan="1">Chromosome</th>
<th align="left" rowspan="1" colspan="1">Physical position (bp)</th>
<th align="left" rowspan="1" colspan="1">Allelic effect</th>
<th align="left" rowspan="1" colspan="1">LOD score</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="6" rowspan="1">
<strong>Non‐heat stress (23°C)</strong>
</td></tr>
<tr>
<td rowspan="2" align="left" colspan="1">SL</td>
<td align="left" rowspan="1" colspan="1">scaffold62159_9784648</td>
<td align="left" rowspan="1" colspan="1">4A</td>
<td align="left" rowspan="1" colspan="1">586339854</td>
<td align="center" rowspan="1" colspan="1">1.46</td>
<td align="center" rowspan="1" colspan="1">5.19</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold62159_9693200</td>
<td align="left" rowspan="1" colspan="1">4A</td>
<td align="left" rowspan="1" colspan="1">586431302</td>
<td align="center" rowspan="1" colspan="1">−1.46</td>
<td align="center" rowspan="1" colspan="1">5.24</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">RN</td>
<td align="left" rowspan="1" colspan="1">scaffold63259_2360029</td>
<td align="left" rowspan="1" colspan="1">2B</td>
<td align="left" rowspan="1" colspan="1">639219929</td>
<td align="center" rowspan="1" colspan="1">−0.35</td>
<td align="center" rowspan="1" colspan="1">5.01</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold63259_2360398</td>
<td align="left" rowspan="1" colspan="1">2B</td>
<td align="left" rowspan="1" colspan="1">639220298</td>
<td align="center" rowspan="1" colspan="1">0.27</td>
<td align="center" rowspan="1" colspan="1">5</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold128405_6019031</td>
<td align="left" rowspan="1" colspan="1">4B</td>
<td align="left" rowspan="1" colspan="1">603166442</td>
<td align="center" rowspan="1" colspan="1">0.6</td>
<td align="center" rowspan="1" colspan="1">6.12</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1">RFW</td>
<td align="left" rowspan="1" colspan="1">scaffold10127_4250444</td>
<td align="left" rowspan="1" colspan="1">1A</td>
<td align="left" rowspan="1" colspan="1">499798735</td>
<td align="center" rowspan="1" colspan="1">4.68</td>
<td align="center" rowspan="1" colspan="1">5.36</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold4077_4635398</td>
<td align="left" rowspan="1" colspan="1">7D</td>
<td align="left" rowspan="1" colspan="1">404242254</td>
<td align="center" rowspan="1" colspan="1">6.19</td>
<td align="center" rowspan="1" colspan="1">6.32</td>
</tr>
<tr><td colspan="6" align="left" rowspan="1">
<strong>Heat stress (36°C)</strong>
</td></tr>
<tr>
<td align="left" rowspan="1" colspan="1">SFW</td>
<td align="left" rowspan="1" colspan="1">scaffold468‐1_279183</td>
<td align="left" rowspan="1" colspan="1">7B</td>
<td align="left" rowspan="1" colspan="1">701158105</td>
<td align="center" rowspan="1" colspan="1">−1.43</td>
<td align="center" rowspan="1" colspan="1">5.42</td>
</tr>
<tr>
<td rowspan="8" align="left" colspan="1">RL</td>
<td align="left" rowspan="1" colspan="1">scaffold76743_1777606</td>
<td align="left" rowspan="1" colspan="1">3A</td>
<td align="left" rowspan="1" colspan="1">26518037</td>
<td align="center" rowspan="1" colspan="1">−0.6</td>
<td align="center" rowspan="1" colspan="1">6.01</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold9496_2310787</td>
<td align="left" rowspan="1" colspan="1">3B</td>
<td align="left" rowspan="1" colspan="1">454645221</td>
<td align="center" rowspan="1" colspan="1">0.4</td>
<td align="center" rowspan="1" colspan="1">5.02</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold142879_1376183</td>
<td align="left" rowspan="1" colspan="1">3D</td>
<td align="left" rowspan="1" colspan="1">595273895</td>
<td align="center" rowspan="1" colspan="1">0.65</td>
<td align="center" rowspan="1" colspan="1">7.67</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold21647_3085029</td>
<td align="left" rowspan="1" colspan="1">6B</td>
<td align="left" rowspan="1" colspan="1">683822994</td>
<td align="center" rowspan="1" colspan="1">−0.66</td>
<td align="center" rowspan="1" colspan="1">7.66</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold26910_6990976</td>
<td align="left" rowspan="1" colspan="1">7A</td>
<td align="left" rowspan="1" colspan="1">464572920</td>
<td align="center" rowspan="1" colspan="1">0.79</td>
<td align="center" rowspan="1" colspan="1">6.72</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold4982_9981508</td>
<td align="left" rowspan="1" colspan="1">7A</td>
<td align="left" rowspan="1" colspan="1">612703273</td>
<td align="center" rowspan="1" colspan="1">0.97</td>
<td align="center" rowspan="1" colspan="1">9.76</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold2159_4802103</td>
<td align="left" rowspan="1" colspan="1">7A</td>
<td align="left" rowspan="1" colspan="1">652687632</td>
<td align="center" rowspan="1" colspan="1">0.77</td>
<td align="center" rowspan="1" colspan="1">5.21</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold71558_289995</td>
<td align="left" rowspan="1" colspan="1">7A</td>
<td align="left" rowspan="1" colspan="1">735776454</td>
<td align="center" rowspan="1" colspan="1">0.66</td>
<td align="center" rowspan="1" colspan="1">6.36</td>
</tr>
<tr>
<td rowspan="7" align="left" colspan="1">RFW</td>
<td align="left" rowspan="1" colspan="1">scaffold5611_406983</td>
<td align="left" rowspan="1" colspan="1">1B</td>
<td align="left" rowspan="1" colspan="1">638934103</td>
<td align="center" rowspan="1" colspan="1">1.15</td>
<td align="center" rowspan="1" colspan="1">7.31</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold131852_2297774</td>
<td align="left" rowspan="1" colspan="1">2B</td>
<td align="left" rowspan="1" colspan="1">751183272</td>
<td align="center" rowspan="1" colspan="1">−0.89</td>
<td align="center" rowspan="1" colspan="1">7.17</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold22822_3587896</td>
<td align="left" rowspan="1" colspan="1">3A</td>
<td align="left" rowspan="1" colspan="1">661835680</td>
<td align="center" rowspan="1" colspan="1">0.78</td>
<td align="center" rowspan="1" colspan="1">5.82</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold6791_22287451</td>
<td align="left" rowspan="1" colspan="1">5A</td>
<td align="left" rowspan="1" colspan="1">657547002</td>
<td align="center" rowspan="1" colspan="1">−0.97</td>
<td align="center" rowspan="1" colspan="1">5.88</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold43799_1029672</td>
<td align="left" rowspan="1" colspan="1">5B</td>
<td align="left" rowspan="1" colspan="1">599554479</td>
<td align="center" rowspan="1" colspan="1">0.5</td>
<td align="center" rowspan="1" colspan="1">5.07</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold38798_23175</td>
<td align="left" rowspan="1" colspan="1">5D</td>
<td align="left" rowspan="1" colspan="1">506095232</td>
<td align="center" rowspan="1" colspan="1">−1.47</td>
<td align="center" rowspan="1" colspan="1">6.17</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">scaffold146879_1644623</td>
<td align="left" rowspan="1" colspan="1">7B</td>
<td align="left" rowspan="1" colspan="1">678622161</td>
<td align="center" rowspan="1" colspan="1">−1.29</td>
<td align="center" rowspan="1" colspan="1">9.68</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270071-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270071-tbl2-note-0001"><p>Abbreviations: LOD, logarithm of odds; MTAs, marker‐trait associations; RFW, root fresh weight (mg); RN, root number; RL, root length (cm); SFW, shoot fresh weight (mg); SNPs, single nucleotide polymorphism; SL, shoot length (cm).</p></div></div></section><figure class="fig xbox font-sm" id="tpg270071-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ee2c/12241833/54e49a8c91d8/TPG2-18-e70071-g007.jpg" loading="lazy" id="jats-graphic-13" height="400" width="800" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270071-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Additive effect of favorable alleles on: (a) shoot length (SL; cm) under non‐heat stress (23°C) treatment; (b) root fresh weight (RFW; mg) under non‐heat stress (23°C) treatment; (c) root number under non‐heat stress (23°C) treatment; (d) root length (RL; cm) under heat stress (36°C) treatment; (e) root fresh weight (RFW; mg) under heat stress (36°C) treatment. Letters (a, b, c) in the boxplots represent the statistical significance difference (at 5%) between group means based on Tukey's HSD test.</p></figcaption></figure></section><section id="tpg270071-sec-0130"><h3 class="pmc_sec_title">3.4. CG and expression analysis</h3>
<p>CG analysis from the genomic regions spanning 21 significant MTAs identified 310 unique gene models (Table <a href="#tpg270071-supinfo-0001" class="usa-link">S3</a>). The highest number of CGs (27) was identified in scaffold6791_22287451, located on chromosome 5A, whereas the lowest number of CGs (4) was identified in scaffold26910_6990976, located on chromosome 7A. Of the 310 unique gene models, 263 had functional annotation, whereas the remaining 47 genes were uncharacterized. Multiple gene models with similar gene functions were frequently detected from the genomic regions involving different MTAs. For instance, 19 gene models harbored protein kinase (PK) domains, eight had ubiquitin‐like domains, six had zinc finger and RNA recognition motif domains, five had NB‐ARC and EF‐hand domains, four were similar to cytochrome P450 proteins, three had F‐box domains, and others are listed in Table <a href="#tpg270071-supinfo-0001" class="usa-link">S3</a>.</p>
<p>CG models (i.e., 310) extracted from genomic region spanning MTAs were subjected to in silico expression analysis using transcriptomic dataset involving seedling heat stress (1 and 6 h) and identified total of 35 DEGs (Z. Liu et al., <a href="#tpg270071-bib-0033" class="usa-link" aria-describedby="tpg270071-bib-0033">2015</a>) (Figure <a href="#tpg270071-fig-0008" class="usa-link">8</a>; Table <a href="#tpg270071-supinfo-0001" class="usa-link">S4</a>). Among the 35 DEGs, 15 were down‐regulated and 20 were up‐regulated under heat stress treatment. Of the 15 down‐regulated genes, the expression of two genes was reduced after 1 h of heat stress, 12 genes were reduced after 6 h of heat stress, and one gene was reduced after both 1 and 6 h of heat stress. Similarly, of the 20 up‐regulated genes, the expression of 10 genes increased after 1 h of heat stress, two genes increased after 6 h of heat stress, and eight genes expressed after both 1 and 6 h of heat stress.</p>
<figure class="fig xbox font-sm" id="tpg270071-fig-0008"><h4 class="obj_head">FIGURE 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ee2c/12241833/d4e6218c50ab/TPG2-18-e70071-g002.jpg" loading="lazy" id="jats-graphic-15" height="998" width="498" alt="FIGURE 8"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270071-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heat map showing the differentially expressed genes (DEGs) under non‐heat stress (23°C) and heat stress (36°C) treatment (1 and 6 h).</p></figcaption></figure><p>Based on the gene functional description and the information available from the published literature, 13 of the 35 DEGs were found to be involved in heat stress tolerance, and these genes were considered as high‐confidence CGs (Table <a href="#tpg270071-tbl-0003" class="usa-link">3</a>). These CGs were predicted to encode proteins belonging to the following classes: PK, fatty acid hydroxylase, pectinacetylesterase, UDP‐glucuronosyl/UDP‐glucosyltransferase, pyrophosphate‐energized proton pump, EF‐hand, basic‐leucine zipper (bZIP), DNA methylase, N‐6 adenine‐specific conserved site, and thioredoxin.</p>
<section class="tw xbox font-sm" id="tpg270071-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Differentially expressed high‐confidence candidate genes (CGs) associated with seedling heat stress tolerance.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Sl. No.</th>
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Gene ID</th>
<th align="left" rowspan="1" colspan="1">Gene position (bp)</th>
<th align="left" rowspan="1" colspan="1">GO term name</th>
<th align="left" rowspan="1" colspan="1">Interpro description</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">1</td>
<td rowspan="6" align="left" colspan="1">RL</td>
<td align="left" rowspan="1" colspan="1">TraesCS3A02G049600</td>
<td align="left" rowspan="1" colspan="1">26229850‐26234135</td>
<td align="left" rowspan="1" colspan="1">ATP binding</td>
<td align="left" rowspan="1" colspan="1">Protein kinase domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">TraesCS3A02G050211</td>
<td align="left" rowspan="1" colspan="1">26469173‐26482725</td>
<td align="left" rowspan="1" colspan="1">ATP binding</td>
<td align="left" rowspan="1" colspan="1">Protein kinase domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">TraesCS3B02G283600</td>
<td align="left" rowspan="1" colspan="1">454656825‐454657602</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">TraesCS3B02G283700</td>
<td align="left" rowspan="1" colspan="1">454668034‐454672347</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">TraesCS6B02G409800</td>
<td align="left" rowspan="1" colspan="1">684220072‐684221496</td>
<td align="left" rowspan="1" colspan="1">Protein‐disulfide reductase activity</td>
<td align="left" rowspan="1" colspan="1">Thioredoxin domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">TraesCS7A02G422500</td>
<td align="left" rowspan="1" colspan="1">612990989‐612994043</td>
<td align="left" rowspan="1" colspan="1">ATP binding</td>
<td align="left" rowspan="1" colspan="1">Protein kinase domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">SFW</td>
<td align="left" rowspan="1" colspan="1">TraesCS7B02G433100</td>
<td align="left" rowspan="1" colspan="1">700723840‐700724301</td>
<td align="left" rowspan="1" colspan="1">Calcium ion binding</td>
<td align="left" rowspan="1" colspan="1">EF‐hand domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">TraesCS7B02G433800</td>
<td align="left" rowspan="1" colspan="1">701185893‐701191043</td>
<td align="left" rowspan="1" colspan="1">Integral component of membrane</td>
<td align="left" rowspan="1" colspan="1">Pyrophosphate‐energised proton pump</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">9</td>
<td rowspan="5" align="left" colspan="1">RFW</td>
<td align="left" rowspan="1" colspan="1">TraesCS2B02G556600</td>
<td align="left" rowspan="1" colspan="1">751317909‐751324133</td>
<td align="left" rowspan="1" colspan="1">DNA‐binding transcription factor activity</td>
<td align="left" rowspan="1" colspan="1">Basic‐leucine zipper domain</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">TraesCS5A02G486000</td>
<td align="left" rowspan="1" colspan="1">657181008‐657184937</td>
<td align="left" rowspan="1" colspan="1">Hydrolase activity</td>
<td align="left" rowspan="1" colspan="1">Pectinacetylesterase/NOTUM</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">TraesCS5A02G486400</td>
<td align="left" rowspan="1" colspan="1">657205537‐657208108</td>
<td align="left" rowspan="1" colspan="1">Nucleic acid binding</td>
<td align="left" rowspan="1" colspan="1">DNA methylase, N‐6 adenine‐specific, conserved site</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">TraesCS5A02G486800</td>
<td align="left" rowspan="1" colspan="1">657394482‐657396507</td>
<td align="left" rowspan="1" colspan="1">Hexosyltransferase activity</td>
<td align="left" rowspan="1" colspan="1">UDP‐glucuronosyl/UDP‐glucosyltransferase</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">TraesCS7B02G408900</td>
<td align="left" rowspan="1" colspan="1">678124719‐678128411</td>
<td align="left" rowspan="1" colspan="1">Integral component of membrane</td>
<td align="left" rowspan="1" colspan="1">Fatty acid hydroxylase</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270071-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270071-tbl3-note-0001"><p>Abbreviations: GO, gene ontology; RFW, root fresh weight; RL, root length; SFW, shoot fresh weight.</p></div></div></section></section></section><section id="tpg270071-sec-0140"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Heat stress significantly reduces grain yield and quality by affecting various developmental stages of wheat, including seedling and reproductive stages. Phenotypic screening at early developmental stages for seed germination and seedling vigor enables high‐throughput evaluation of a large number of germplasm lines under controlled condition. Moreover, seedling evaluation accelerates the breeding process by allowing early selection of promising genotypes. Although seedling performance observed under controlled environment does not always repeat during the adult stage or at field conditions (due to involvement of combination of stresses), several studies reported that the genotypes performing well under seedling heat stress also tend to exhibit improved tolerance at the adult stage with a strong positive correlation (<em>r</em>  =  0.6930) (Alsamadany, <a href="#tpg270071-bib-0002" class="usa-link" aria-describedby="tpg270071-bib-0002">2016</a>; Collins et al., <a href="#tpg270071-bib-0007" class="usa-link" aria-describedby="tpg270071-bib-0007">2017</a>; Lu et al., <a href="#tpg270071-bib-0034" class="usa-link" aria-describedby="tpg270071-bib-0034">2022</a>). Studies also reported the collocated QTLs and/or heat induced genes for the seedling stage and reproductive stage (Li et al., <a href="#tpg270071-bib-0031" class="usa-link" aria-describedby="tpg270071-bib-0031">2012</a>). Therefore, we studied the effect of heat stress on six seedling traits (SL, RL, CL, RN, SFW, and RFW) associated with seedling vigor on 216 diverse spring wheat accessions. We observed a huge genetic variation for studied traits under heat stress. On average, wheat accessions showed reduced performance under heat stress for SL, RL, CL, SFW, and RFW, whereas RN was substantially increased. Compared to optimum temperature, high‐temperature stress slows down seed imbibition and oxygen uptake and inhibits the mobilization of seed reserves from endosperm to embryo, thereby delays or reduces the seed germination (Essemine et al., <a href="#tpg270071-bib-0014" class="usa-link" aria-describedby="tpg270071-bib-0014">2010</a>). Moreover, direct exposure of germinating seeds to high‐temperature stress affects the rate of development of radicle and coleoptile tissues, thereby reduces seedling vigor by reducing root and shoot traits, respectively. Similarly, we observed a negative effect of heat stress on shoot and root traits. Furthermore, we observed that the impact of heat stress was more on root traits (RL and RFW) compared to shoot traits (CL, SL, and SFW). This is owing to the direct exposure of root tissue to increased soil temperature, which disrupts the cell division and meristematic tissue growth, thereby reduce primary root growth. Moreover, heat stress affects roots indirectly by creating increased water and nutrient demand for shoots or by decreasing carbon supply from shoots to roots, thereby reduces root biomass. In contrast, shoots have some protection from temperature fluctuations due to their exposure to air and the ability to adjust leaf orientation or canopy temperature to reduce heat absorption.</p>
<p>Identifying contrasting germplasm lines is crucial for understanding the molecular and genetic mechanisms underlying heat stress tolerance in crops (Janni et al., <a href="#tpg270071-bib-0028" class="usa-link" aria-describedby="tpg270071-bib-0028">2020</a>). Moreover, heat‐tolerant lines can serve as valuable starting material for developing heat‐resilient wheat cultivars. In this study, we observed less impact of heat stress on six germplasm lines (i.e., PI 366905, Kzyl Sark, Rang, Perico S, Bohr Gamh, and PI 620689). These lines can be utilized in future breeding program to develop the heat‐resilient wheat cultivars.</p>
<p>Correlation studies reveal the intrinsic relationship among different traits and also help in indirect selection for stress‐tolerant traits. In this study, we observed a strong positive correlation among shoot and root traits, indicating that heat‐induced damage to roots impairs their ability to absorb sufficient water and nutrients, thereby limiting photosynthesis and resulting in a proportional decrease in shoot growth.</p>
<p>Plants have developed a range of adaptive mechanisms to mitigate the effects of heat stress. Identifying genomic regions associated with improved heat stress tolerance and incorporating them into cultivars helps in developing stress‐resilient crops. In this study, we used GWAS to identify 23 MTAs associated with seedling traits on 15 chromosomes (Figures <a href="#tpg270071-fig-0005" class="usa-link">5</a> and <a href="#tpg270071-fig-0006" class="usa-link">6</a>; Table <a href="#tpg270071-tbl-0002" class="usa-link">2</a>). Additionally, by comparing our results with published results of Khan et al. (<a href="#tpg270071-bib-0029" class="usa-link" aria-describedby="tpg270071-bib-0029">2022</a>), we identified five overlapping QTLs/MTAs (i.e., scaffold10127_4250444, scaffold4077_4635398, scaffold21647_3085029, scaffold5611_406983, and scaffold38798_23175) for seedling traits. Significant MTAs identified through GWAS serve as potential targets for extracting putative CGs associated with targeted traits (Devate et al., <a href="#tpg270071-bib-0008" class="usa-link" aria-describedby="tpg270071-bib-0008">2022</a>; Rathan et al., <a href="#tpg270071-bib-0043" class="usa-link" aria-describedby="tpg270071-bib-0043">2022</a>).</p>
<p>Integrating favorable alleles from multiple QTLs in same genetic background is an effective breeding strategy to optimize plant traits for stress tolerance (Fan et al., <a href="#tpg270071-bib-0015" class="usa-link" aria-describedby="tpg270071-bib-0015">2019</a>; You et al., <a href="#tpg270071-bib-0052" class="usa-link" aria-describedby="tpg270071-bib-0052">2021</a>). In this study, we noticed the significant additive effect of favorable alleles for different seedling traits under non‐heat stress (for SL, RFW, and RN) (Figure <a href="#tpg270071-fig-0007" class="usa-link">7a–c</a>) and heat stress (for RL and RFW) (Figure <a href="#tpg270071-fig-0007" class="usa-link">7d,e</a>) treatments. Genotypes with &gt;1 favorable allele improved the seeding performance more than those with no favorable alleles, which signifies the importance of combining multiple favorable alleles to improve heat stress tolerance.</p>
<p>A SNP located on chromosome 3A (scaffold76743_1777606) is linked with two putative genes (<em>TraesCS3A02G049600</em> and <em>TraesCS3A02G050211</em>) encoding PK domain. PKs, including receptor‐like kinases (RLKs), mitogen‐activated PKs (MAPKs) cascades, calcium‐dependent PKs (CDPKs/CPKs), and calcineurin B like interacting PKs, play important roles in stress sensing and signal transduction and help in responding to prevailing stresses. For instance, MAPKs regulate the plant's response to multiple abiotic stresses, including heat and drought stress (de Zelicourt et al., <a href="#tpg270071-bib-0009" class="usa-link" aria-describedby="tpg270071-bib-0009">2016</a>), and RLKs such as guard cell hydrogen peroxide‐resistant 1 are necessary for closing stomata under drought stress by regulating the synthesis of abscisic acid and hydrogen peroxide (Ding et al., <a href="#tpg270071-bib-0010" class="usa-link" aria-describedby="tpg270071-bib-0010">2013</a>). Similarly, CPKs such as <em>cpk28</em> regulate the phosphorylation of ascorbate peroxidase (POD) and thereby improve heat stress tolerance (Hu et al., <a href="#tpg270071-bib-0027" class="usa-link" aria-describedby="tpg270071-bib-0027">2021</a>). SNP located on chromosome 7B (scaffold468‐1_279183) linked to a gene, <em>TraesCS7B02G4338000</em>, which encodes for the proton‐pumping pyrophosphatases. These pyrophosphatases act as a backup system for converting the energy stored in pyrophosphates into electrochemical gradients under various stresses. For instance, the overexpression of H<sup>+</sup>‐PPase <em>AVP1</em> (Arabidopsis vacuolar H+‐PPase) results in increased SUMOylation, which ultimately leads to improved plant growth and development under heat, cold, drought, salt, and nutrient stresses (Gaxiola et al., <a href="#tpg270071-bib-0018" class="usa-link" aria-describedby="tpg270071-bib-0018">2012</a>; Park et al., <a href="#tpg270071-bib-0037" class="usa-link" aria-describedby="tpg270071-bib-0037">2005</a>; Patir‐Nebioglu et al., <a href="#tpg270071-bib-0038" class="usa-link" aria-describedby="tpg270071-bib-0038">2019</a>; Schilling et al., <a href="#tpg270071-bib-0044" class="usa-link" aria-describedby="tpg270071-bib-0044">2017</a>). Similarly, we identified a gene (<em>TraesCS2B02G556600</em>) linked to scaffold131852_2297774 encoded to bZIP transcription factors (TFs). These TFs regulate the expression of genes involved in stress tolerance and increase the plant's adaptability to heat, drought, salt, and cold stresses by enhancing the activity of antioxidant enzymes such as catalase, POD, and superoxide dismutase (Yu et al., <a href="#tpg270071-bib-0053" class="usa-link" aria-describedby="tpg270071-bib-0053">2020</a>). Similarly, genes encoding a thioredoxin domain‐containing protein, a fatty acid hydroxylase, UDP‐glucosyltransferase, pectinacetylesterase, and an EF‐hand domain‐containing proteins. These proteins are known to provide heat stress tolerance through calcium signaling and protein‐disulfide reductase activity (Dong et al., <a href="#tpg270071-bib-0012" class="usa-link" aria-describedby="tpg270071-bib-0012">2020</a>; Patir‐Nebioglu et al., <a href="#tpg270071-bib-0038" class="usa-link" aria-describedby="tpg270071-bib-0038">2019</a>). We also identified two high‐confidence DEGs (<em>TraesCS3B02G283600</em> and <em>TraesCS3B02G283700</em>) with unknown functions. These might represent new classes of genes that confer tolerance to heat stress. Future studies may lead to better functional characterization of these genes to identify the underlying mechanisms of stress tolerance. Moreover, the combination of favorable alleles for multiple MTAs identified in this study can be integrated into common genetic background to develop heat‐resilient wheat varieties.</p></section><section id="tpg270071-sec-0150"><h2 class="pmc_sec_title">5. CONCLUSION</h2>
<p>In this study, we observed a negative impact of heat stress on seedling traits and also identified highly tolerant lines for heat stress. These lines can be utilized in breeding programs to develop resilient wheat varieties with sustainable yield under changing climatic conditions. Additionally, our study identified significant MTAs and putative CGs associated with heat stress tolerance. Heat‐tolerant lines and the putative genes identified in this study provide the foundation for developing functional markers and also help in marker‐assisted breeding of heat‐resilient wheat varieties. Moreover, these results provide the foundation for future studies focusing to understand the genetic basis of heat stress tolerance in wheat.</p></section><section id="tpg270071-sec-0160"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Santosh Gudi</strong>: Conceptualization; data curation; formal analysis; methodology; project administration; software; visualization; writing—original draft. <strong>Jatinder Singh</strong>: Formal analysis; visualization; writing—original draft. <strong>Harsimardeep Gill</strong>: Formal analysis; visualization. <strong>Sunish Sehgal</strong>: Writing—review and editing. <strong>Justin D. Faris</strong>: Writing—review and editing. <strong>Upinder Gill</strong>: Conceptualization; investigation; supervision; writing—review and editing. <strong>Rajeev Gupta</strong>: Conceptualization; funding acquisition; investigation; project administration; resources; supervision; writing—review and editing.</p></section><section id="tpg270071-sec-0180"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec18"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270071-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270071-supitem-0001"><div class="caption p">
<p>
<strong>Supplementary Table 1</strong>. List of 216 germplasm lines used for association analysis in the present study.</p>
<p>
<strong>Supplementary Table 2</strong>. Number of single nucleotide polymorphisms (SNPs) distributed along the chromosomes and the sub‐genomes, and the total number of SNPs in whole genome.</p>
<p>
<strong>Supplementary Table 3</strong>. List of unique gene models identified from the significant marker‐trait associations (MTAs) associated with seedling traits under non‐heat stress (23°C) (blue) and heat stress (36°C) (green) treatment.</p>
<p>
<strong>Supplementary Table 4</strong>. List of differentially expressed genes (DEGs) identified in the present study.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12241833/bin/TPG2-18-e70071-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70071-s001.xlsx</a><sup> (53.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270071-supitem-0002"><div class="caption p">
<p>
<strong>Supplementary Figure 1</strong>. Analysis of linkage disequilibrium (LD) at the whole genome and sub‐genome levels in the hexaploid spring wheat collection. <strong>(a)</strong> LD decay analysis of the whole genome; <strong>(b)</strong> LD decay analysis for the A‐sub‐genome; <strong>(c)</strong> LD decay analysis for the B‐sub‐genome; <strong>(d)</strong> LD decay analysis for the D‐sub‐genome.</p>
<p>
<strong>Supplementary Figure 2</strong>. Manhattan plot for the: <strong>(a)</strong> shoot fresh weight (SFW; mg); <strong>(b)</strong> root length (RL; cm); <strong>(c)</strong> coleoptile length (CL; cm) under non‐heat stress (23°C) treatment.</p>
<p>
<strong>Supplementary Figure 3</strong>. Manhattan plot for the: <strong>(a)</strong> shoot length (SL; cm); <strong>(b)</strong> root number (RN); <strong>(c)</strong> coleoptile length (CL; cm) under heat stress (36°C) treatment.</p>
<p>
<strong>Supplementary Figure 4</strong>. Estimating the allelic effects of significant marker‐trait associations (MTAs) identified in the present study: <strong>(a‐g)</strong> allelic effects of MTAs, scaffold62159_9784648 (a), scaffold62159_9693200 (b), scaffold63259_2360029 (c), scaffold63259_2360398 (d), scaffold128405_6019031 (e), scaffold10127_4250444 (f), and scaffold4077_4635398 (g) identified under non‐heat stress (23°C) treatment; <strong>(h‐w)</strong> allelic effects of MTAs, scaffold468‐1_279183 (h), scaffold76743_1777606 (i), scaffold9496_2310787 (j), scaffold142879_1376183 (k), scaffold21647_3085029 (l), scaffold26910_6990976 (m), scaffold4982_9981508 (n), scaffold2159_4802103 (o), scaffold71558_289995 (p), scaffold5611_406983 (q), scaffold131852_2297774 (r), scaffold22822_3587896 (s), scaffold6791_22287451 (t), scaffold43799_1029672 (u), scaffold38798_23175 (v), and scaffold146879_1644623 (w) identified under heat stress (36°C) treatment.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12241833/bin/TPG2-18-e70071-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70071-s002.docx</a><sup> (1.2MB, docx) </sup>
</div></div></section></section><section id="tpg270071-sec-0170" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>The authors acknowledge the Agricultural Experiment Station, Department of Plant Pathology at North Dakota State University, Fargo, ND, USA and Cereal Crops Improvement Research, Edward T. Schafer Agricultural Research Center, USDA‐ARS, Fargo, ND, USA for providing support.</p></section><section id="notes1"><p>


Gudi, S.
, 
Singh, J.
, 
Gill, H.
, 
Sehgal, S.
, 
Faris, J. D.
, 
Gill, U.
, &amp; 
Gupta, R.
 (2025). Understanding the genetic basis of heat stress tolerance in wheat (<em>Triticum aestivum</em> L.) through genome‐wide association studies. The Plant Genome, 18, e70071. 10.1002/tpg2.70071

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270071-note-0001"><p>Assigned to Associate Editor Susanne Dreisigacker.</p></div></div></section></section><section id="tpg270071-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270071-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270071-bib-0001">
<cite>

Abasi, F.
, 
Raja, N. I.
, 
Mashwani, Z.‐R.
, 
Ehsan, M.
, 
Ali, H.
, &amp; 
Shahbaz, M.
 (2024). Heat and wheat: Adaptation strategies with respect to heat shock proteins and antioxidant potential; an era of climate change. International Journal of Biological Macromolecules, 256, 128379. 10.1016/j.ijbiomac.2023.128379

</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2023.128379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38000583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Biological%20Macromolecules&amp;title=Heat%20and%20wheat:%20Adaptation%20strategies%20with%20respect%20to%20heat%20shock%20proteins%20and%20antioxidant%20potential;%20an%20era%20of%20climate%20change&amp;author=F.%20Abasi&amp;author=N.%20I.%20Raja&amp;author=Z.%E2%80%90R.%20Mashwani&amp;author=M.%20Ehsan&amp;author=H.%20Ali&amp;volume=256&amp;publication_year=2024&amp;pages=128379&amp;pmid=38000583&amp;doi=10.1016/j.ijbiomac.2023.128379&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0002">
<cite>

Alsamadany, H.
 (2016). Diversity and genetic studies of heat tolerance in wheat. The University of Westen Australia.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Diversity%20and%20genetic%20studies%20of%20heat%20tolerance%20in%20wheat&amp;author=H.%20Alsamadany&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0003">
<cite>

Alvarado, G.
, 
Rodríguez, F. M.
, 
Pacheco, A.
, 
Burgueño, J.
, 
Crossa, J.
, 
Vargas, M.
, 
Pérez‐Rodríguez, P.
, &amp; 
Lopez‐Cruz, M. A.
 (2020). META‐R: A software to analyze data from multi‐environment plant breeding trials. Crop Journal, 8, 745–756. 10.1016/J.CJ.2020.03.010
</cite> [<a href="https://doi.org/10.1016/J.CJ.2020.03.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Journal&amp;title=META%E2%80%90R:%20A%20software%20to%20analyze%20data%20from%20multi%E2%80%90environment%20plant%20breeding%20trials&amp;author=G.%20Alvarado&amp;author=F.%20M.%20Rodr%C3%ADguez&amp;author=A.%20Pacheco&amp;author=J.%20Burgue%C3%B1o&amp;author=J.%20Crossa&amp;volume=8&amp;publication_year=2020&amp;pages=745-756&amp;doi=10.1016/J.CJ.2020.03.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0004">
<cite>

Bita, C. E.
, &amp; 
Gerats, T.
 (2013). Plant tolerance to high temperature in a changing environment: Scientific fundamentals and production of heat stress‐tolerant crops. Frontiers in Plant Science, 4, 273. 10.3389/FPLS.2013.00273

</cite> [<a href="https://doi.org/10.3389/FPLS.2013.00273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3728475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23914193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Plant%20tolerance%20to%20high%20temperature%20in%20a%20changing%20environment:%20Scientific%20fundamentals%20and%20production%20of%20heat%20stress%E2%80%90tolerant%20crops&amp;author=C.%20E.%20Bita&amp;author=T.%20Gerats&amp;volume=4&amp;publication_year=2013&amp;pages=273&amp;pmid=23914193&amp;doi=10.3389/FPLS.2013.00273&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0005">
<cite>

Browning, S. R.
, &amp; 
Browning, B. L.
 (2007). Rapid and accurate haplotype phasing and missing‐data inference for whole‐genome association studies by use of localized haplotype clustering. American Journal of Human Genetics, 81, 1084–1097. 10.1086/521987

</cite> [<a href="https://doi.org/10.1086/521987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2265661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17924348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Rapid%20and%20accurate%20haplotype%20phasing%20and%20missing%E2%80%90data%20inference%20for%20whole%E2%80%90genome%20association%20studies%20by%20use%20of%20localized%20haplotype%20clustering&amp;author=S.%20R.%20Browning&amp;author=B.%20L.%20Browning&amp;volume=81&amp;publication_year=2007&amp;pages=1084-1097&amp;pmid=17924348&amp;doi=10.1086/521987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0006">
<cite>

Coast, O.
, 
Scafaro, A. P.
, 
Bramley, H.
, 
Taylor, N. L.
, &amp; 
Atkin, O. K.
 (2024). Photosynthesis in newly developed leaves of heat‐tolerant wheat acclimates to long‐term nocturnal warming. Journal of Experimental Botany, 75, 962–978. 10.1093/jxb/erad437

</cite> [<a href="https://doi.org/10.1093/jxb/erad437" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10837020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37935881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Photosynthesis%20in%20newly%20developed%20leaves%20of%20heat%E2%80%90tolerant%20wheat%20acclimates%20to%20long%E2%80%90term%20nocturnal%20warming&amp;author=O.%20Coast&amp;author=A.%20P.%20Scafaro&amp;author=H.%20Bramley&amp;author=N.%20L.%20Taylor&amp;author=O.%20K.%20Atkin&amp;volume=75&amp;publication_year=2024&amp;pages=962-978&amp;pmid=37935881&amp;doi=10.1093/jxb/erad437&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0007">
<cite>

Collins, N.
, 
Hildebrand, S.
, 
Taylor, K.
, 
Taylor, H.
, 
Pleming, D.
, 
Lohraseb, I.
, 
Shirdelmoghanloo, H.
, 
Erena, M.
, 
Rahman, M.
, 
Taylor, J.
, 
Munoz‐Santa, S.
, 
Mather, D.
, 
Heuer, S.
, 
Sissons, M.
, &amp; 
Emebiri, L.
 (2017). Understanding heat impacts on wheat to breed future tolerance. Grains Research and Development Corporation. <a href="https://grdc.com.au/resources-and-publications/grdc-update-papers/tab-content/grdc-update-papers/2017/02/understanding-heat-impacts-on-wheat-to-breedfuture-tolerance" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://grdc.com.au/resources‐and‐publications/grdc‐update‐papers/tab‐content/grdc‐update‐papers/2017/02/understanding‐heat‐impacts‐on‐wheat‐to‐breedfuture‐tolerance</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Understanding%20heat%20impacts%20on%20wheat%20to%20breed%20future%20tolerance&amp;author=N.%20Collins&amp;author=S.%20Hildebrand&amp;author=K.%20Taylor&amp;author=H.%20Taylor&amp;author=D.%20Pleming&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0008">
<cite>

Devate, N. B.
, 
Krishna, H.
, 
Sunilkumar, V. P.
, 
Manjunath, K. K.
, 
Mishra, C. N.
, 
Jain, N.
, 
Singh, G. P.
, &amp; 
Singh, P. K.
 (2022). Identification of genomic regions of wheat associated with grain Fe and Zn content under drought and heat stress using genome‐wide association study. Frontiers in Genetics, 13, 1034947. 10.3389/FGENE.2022.1034947

</cite> [<a href="https://doi.org/10.3389/FGENE.2022.1034947" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9634069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36338980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Identification%20of%20genomic%20regions%20of%20wheat%20associated%20with%20grain%20Fe%20and%20Zn%20content%20under%20drought%20and%20heat%20stress%20using%20genome%E2%80%90wide%20association%20study&amp;author=N.%20B.%20Devate&amp;author=H.%20Krishna&amp;author=V.%20P.%20Sunilkumar&amp;author=K.%20K.%20Manjunath&amp;author=C.%20N.%20Mishra&amp;volume=13&amp;publication_year=2022&amp;pages=1034947&amp;pmid=36338980&amp;doi=10.3389/FGENE.2022.1034947&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0009">
<cite>

de Zelicourt, A.
, 
Colcombet, J.
, &amp; 
Hirt, H.
 (2016). The role of MAPK modules and ABA during abiotic stress signaling. Trends in Plant Science, 21, 677–685. 10.1016/j.tplants.2016.04.004

</cite> [<a href="https://doi.org/10.1016/j.tplants.2016.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27143288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=The%20role%20of%20MAPK%20modules%20and%20ABA%20during%20abiotic%20stress%20signaling&amp;author=A.%20de%20Zelicourt&amp;author=J.%20Colcombet&amp;author=H.%20Hirt&amp;volume=21&amp;publication_year=2016&amp;pages=677-685&amp;pmid=27143288&amp;doi=10.1016/j.tplants.2016.04.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0010">
<cite>

Ding, Y.
, 
Cao, J.
, 
Ni, L.
, 
Zhu, Y.
, 
Zhang, A.
, 
Tan, M.
, &amp; 
Jiang, M.
 (2013). ZmCPK11 is involved in abscisic acid‐induced antioxidant defence and functions upstream of ZmMPK5 in abscisic acid signalling in maize. Journal of Experimental Botany, 64, 871–884. 10.1093/jxb/ers366

</cite> [<a href="https://doi.org/10.1093/jxb/ers366" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3580805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23268839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=ZmCPK11%20is%20involved%20in%20abscisic%20acid%E2%80%90induced%20antioxidant%20defence%20and%20functions%20upstream%20of%20ZmMPK5%20in%20abscisic%20acid%20signalling%20in%20maize&amp;author=Y.%20Ding&amp;author=J.%20Cao&amp;author=L.%20Ni&amp;author=Y.%20Zhu&amp;author=A.%20Zhang&amp;volume=64&amp;publication_year=2013&amp;pages=871-884&amp;pmid=23268839&amp;doi=10.1093/jxb/ers366&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0011">
<cite>

Djanaguiraman, M.
, 
Narayanan, S.
, 
Erdayani, E.
, &amp; 
Prasad, P. V. V
 (2020). Effects of high temperature stress during anthesis and grain filling periods on photosynthesis, lipids and grain yield in wheat. BMC Plant Biology, 20, Article 268. 10.1186/s12870-020-02479-0

</cite> [<a href="https://doi.org/10.1186/s12870-020-02479-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7285450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32517754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Effects%20of%20high%20temperature%20stress%20during%20anthesis%20and%20grain%20filling%20periods%20on%20photosynthesis,%20lipids%20and%20grain%20yield%20in%20wheat&amp;author=M.%20Djanaguiraman&amp;author=S.%20Narayanan&amp;author=E.%20Erdayani&amp;author=P.%20V.%20V%20Prasad&amp;volume=20&amp;publication_year=2020&amp;pages=268&amp;pmid=32517754&amp;doi=10.1186/s12870-020-02479-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0012">
<cite>

Dong, N.‐Q.
, 
Sun, Y.
, 
Guo, T.
, 
Shi, C.‐L.
, 
Zhang, Y.‐M.
, 
Kan, Y.
, 
Xiang, Y.‐H.
, 
Zhang, H.
, 
Yang, Y.‐B.
, 
Li, Y.‐C.
, 
Zhao, H.‐Y.
, 
Yu, H.‐X.
, 
Lu, Z.‐Q.
, 
Wang, Y.
, 
Ye, W.‐W.
, 
Shan, J.‐X.
, &amp; 
Lin, H.‐X.
 (2020). UDP‐glucosyltransferase regulates grain size and abiotic stress tolerance associated with metabolic flux redirection in rice. Nature Communications, 11, Article 2629. 10.1038/s41467-020-16403-5
</cite> [<a href="https://doi.org/10.1038/s41467-020-16403-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7250897/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32457405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=UDP%E2%80%90glucosyltransferase%20regulates%20grain%20size%20and%20abiotic%20stress%20tolerance%20associated%20with%20metabolic%20flux%20redirection%20in%20rice&amp;author=N.%E2%80%90Q.%20Dong&amp;author=Y.%20Sun&amp;author=T.%20Guo&amp;author=C.%E2%80%90L.%20Shi&amp;author=Y.%E2%80%90M.%20Zhang&amp;volume=11&amp;publication_year=2020&amp;pages=2629&amp;pmid=32457405&amp;doi=10.1038/s41467-020-16403-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0013">
<cite>

Dwivedi, S. K.
, 
Soni, S. K.
, 
Mishra, J. S.
, 
Koley, T. K.
, &amp; 
Kumar, S.
 (2024). Assessment of terminal heat tolerance ability of wheat genotypes based on chemometric analysis and agro‐physiological traits. Acta Physiologiae Plantarum, 46, Article 48. 10.1007/s11738-024-03677-1
</cite> [<a href="https://doi.org/10.1007/s11738-024-03677-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Physiologiae%20Plantarum&amp;title=Assessment%20of%20terminal%20heat%20tolerance%20ability%20of%20wheat%20genotypes%20based%20on%20chemometric%20analysis%20and%20agro%E2%80%90physiological%20traits&amp;author=S.%20K.%20Dwivedi&amp;author=S.%20K.%20Soni&amp;author=J.%20S.%20Mishra&amp;author=T.%20K.%20Koley&amp;author=S.%20Kumar&amp;volume=46&amp;publication_year=2024&amp;pages=48&amp;doi=10.1007/s11738-024-03677-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0014">
<cite>

Essemine, J.
, 
Ammar, S.
, &amp; 
Bouzid, S.
 (2010). Effect of temperature on root and shoot development in wheat seedlings during early growth stage. Asian Journal of Plant Sciences, 9, 375–379. 10.3923/ajps.2010.375.379
</cite> [<a href="https://doi.org/10.3923/ajps.2010.375.379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Asian%20Journal%20of%20Plant%20Sciences&amp;title=Effect%20of%20temperature%20on%20root%20and%20shoot%20development%20in%20wheat%20seedlings%20during%20early%20growth%20stage&amp;author=J.%20Essemine&amp;author=S.%20Ammar&amp;author=S.%20Bouzid&amp;volume=9&amp;publication_year=2010&amp;pages=375-379&amp;doi=10.3923/ajps.2010.375.379&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0015">
<cite>

Fan, X.
, 
Cui, F.
, 
Ji, J.
, 
Zhang, W.
, 
Zhao, X.
, 
Liu, J.
, 
Meng, D.
, 
Tong, Y.
, 
Wang, T.
, &amp; 
Li, J.
 (2019). Dissection of pleiotropic QTL regions controlling wheat spike characteristics under different nitrogen treatments using traditional and conditional QTL mapping. Frontiers in Plant Science, 10, 187. 10.3389/fpls.2019.00187

</cite> [<a href="https://doi.org/10.3389/fpls.2019.00187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6400075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30863417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Dissection%20of%20pleiotropic%20QTL%20regions%20controlling%20wheat%20spike%20characteristics%20under%20different%20nitrogen%20treatments%20using%20traditional%20and%20conditional%20QTL%20mapping&amp;author=X.%20Fan&amp;author=F.%20Cui&amp;author=J.%20Ji&amp;author=W.%20Zhang&amp;author=X.%20Zhao&amp;volume=10&amp;publication_year=2019&amp;pages=187&amp;pmid=30863417&amp;doi=10.3389/fpls.2019.00187&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0016">
<cite>

Farhad, M.
, 
Kumar, U.
, 
Tomar, V.
, 
Bhati, P. K.
, 
Krishnan, J. N.
, Kishowar‐E‐Mustarin, 
Barek, V.
, 
Brestic, M.
, &amp; 
Hossain, A.
 (2023). Heat stress in wheat: A global challenge to feed billions in the current era of the changing climate. Frontiers in Sustainable Food Systems, 7, 1203721. 10.3389/fsufs.2023.1203721
</cite> [<a href="https://doi.org/10.3389/fsufs.2023.1203721" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Sustainable%20Food%20Systems&amp;title=Heat%20stress%20in%20wheat:%20A%20global%20challenge%20to%20feed%20billions%20in%20the%20current%20era%20of%20the%20changing%20climate&amp;author=M.%20Farhad&amp;author=U.%20Kumar&amp;author=V.%20Tomar&amp;author=P.%20K.%20Bhati&amp;author=J.%20N.%20Krishnan&amp;volume=7&amp;publication_year=2023&amp;pages=1203721&amp;doi=10.3389/fsufs.2023.1203721&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0017">
<cite>

Fu, C.
, 
Zhou, Y.
, 
Liu, A.
, 
Chen, R.
, 
Yin, L.
, 
Li, C.
, &amp; 
Mao, H.
 (2024). Genome‐wide association study for seedling heat tolerance under two temperature conditions in bread wheat (<em>Triticum aestivum</em> L.). BMC Plant Biology, 24, Article 430. 10.1186/s12870-024-05116-2

</cite> [<a href="https://doi.org/10.1186/s12870-024-05116-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11107014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38773371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Genome%E2%80%90wide%20association%20study%20for%20seedling%20heat%20tolerance%20under%20two%20temperature%20conditions%20in%20bread%20wheat%20(Triticum%20aestivum%20L.)&amp;author=C.%20Fu&amp;author=Y.%20Zhou&amp;author=A.%20Liu&amp;author=R.%20Chen&amp;author=L.%20Yin&amp;volume=24&amp;publication_year=2024&amp;pages=430&amp;pmid=38773371&amp;doi=10.1186/s12870-024-05116-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0018">
<cite>

Gaxiola, R. A.
, 
Sanchez, C. A.
, 
Paez‐Valencia, J.
, 
Ayre, B. G.
, &amp; 
Elser, J. J.
 (2012). Genetic manipulation of a “vacuolar” H+‐PPase: From salt tolerance to yield enhancement under phosphorus‐deficient soils. Plant Physiology, 159, 3–11. 10.1104/pp.112.195701

</cite> [<a href="https://doi.org/10.1104/pp.112.195701" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3375966/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22434041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Genetic%20manipulation%20of%20a%20%E2%80%9Cvacuolar%E2%80%9D%20H+%E2%80%90PPase:%20From%20salt%20tolerance%20to%20yield%20enhancement%20under%20phosphorus%E2%80%90deficient%20soils&amp;author=R.%20A.%20Gaxiola&amp;author=C.%20A.%20Sanchez&amp;author=J.%20Paez%E2%80%90Valencia&amp;author=B.%20G.%20Ayre&amp;author=J.%20J.%20Elser&amp;volume=159&amp;publication_year=2012&amp;pages=3-11&amp;pmid=22434041&amp;doi=10.1104/pp.112.195701&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0019">
<cite>

Girousse, C.
 (2023). Heat shock exposure during early wheat grain development can reduce maximum endosperm cell number but not necessarily final grain dry mass. PLoS One, 18, e0285218. 10.1371/journal.pone.0285218

</cite> [<a href="https://doi.org/10.1371/journal.pone.0285218" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10146457/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37115800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Heat%20shock%20exposure%20during%20early%20wheat%20grain%20development%20can%20reduce%20maximum%20endosperm%20cell%20number%20but%20not%20necessarily%20final%20grain%20dry%20mass&amp;author=C.%20Girousse&amp;volume=18&amp;publication_year=2023&amp;pages=e0285218&amp;pmid=37115800&amp;doi=10.1371/journal.pone.0285218&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0055">
<cite>

Gudi, S.
, 
Gill, H. S.
, 
Collins, S.
, 
Singh, J.
, 
Sandhu, D.
, 
Sehgal, S. K.
, 
Gill, G.
, &amp; 
Gupta, R.
 (2025). Association analysis identified superior haplotypes for improved salt stress tolerance in wheat (<em>Triticum aestivum</em> L.). Plant Stress, 16, 100900. 10.1016/j.stress.2025.100900
</cite> [<a href="https://doi.org/10.1016/j.stress.2025.100900" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Stress&amp;title=Association%20analysis%20identified%20superior%20haplotypes%20for%20improved%20salt%20stress%20tolerance%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=S.%20Gudi&amp;author=H.%20S.%20Gill&amp;author=S.%20Collins&amp;author=J.%20Singh&amp;author=D.%20Sandhu&amp;volume=16&amp;publication_year=2025&amp;pages=100900&amp;doi=10.1016/j.stress.2025.100900&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0020">
<cite>

Gudi, S.
, 
Halladakeri, P.
, 
Singh, G.
, 
Kumar, P.
, 
Singh, S.
, 
Alwutayd, K. M.
, 
Abd El‐Moneim, D.
, &amp; 
Sharma, A.
 (2024). Deciphering the genetic landscape of seedling drought stress tolerance in wheat (<em>Triticum aestivum</em> L.) through genome‐wide association studies. Frontiers in Plant Science, 15, 1351075. 10.3389/fpls.2024.1351075

</cite> [<a href="https://doi.org/10.3389/fpls.2024.1351075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10952099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38510445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Deciphering%20the%20genetic%20landscape%20of%20seedling%20drought%20stress%20tolerance%20in%20wheat%20(Triticum%20aestivum%20L.)%20through%20genome%E2%80%90wide%20association%20studies&amp;author=S.%20Gudi&amp;author=P.%20Halladakeri&amp;author=G.%20Singh&amp;author=P.%20Kumar&amp;author=S.%20Singh&amp;volume=15&amp;publication_year=2024&amp;pages=1351075&amp;pmid=38510445&amp;doi=10.3389/fpls.2024.1351075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0021">
<cite>

Gudi, S.
, 
Jain, M.
, 
Singh, S.
, 
Kaur, S.
, 
Srivastava, P.
, 
Mavi, G. S.
, 
Chhuneja, P.
, 
Sohu, V. S.
, 
Safhi, F. A.
, 
El‐Moneim, D. A.
, &amp; 
Sharma, A.
 (2024). Stress adaptive plasticity from <em>Aegilops tauschii</em> introgression lines improves drought and heat stress tolerance in bread wheat (<em>Triticum aestivum</em> L.). PeerJ, 12, e17528. 10.7717/peerj.17528

</cite> [<a href="https://doi.org/10.7717/peerj.17528" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11177856/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38881860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Stress%20adaptive%20plasticity%20from%20Aegilops%20tauschii%20introgression%20lines%20improves%20drought%20and%20heat%20stress%20tolerance%20in%20bread%20wheat%20(Triticum%20aestivum%20L.)&amp;author=S.%20Gudi&amp;author=M.%20Jain&amp;author=S.%20Singh&amp;author=S.%20Kaur&amp;author=P.%20Srivastava&amp;volume=12&amp;publication_year=2024&amp;pages=e17528&amp;pmid=38881860&amp;doi=10.7717/peerj.17528&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0022">
<cite>

Gudi, S.
, 
Saini, D. K.
, 
Halladakeri, P.
, 
Singh, G.
, 
Singh, S.
, 
Kaur, S.
, 
Goyal, P.
, 
Srivastava, P.
, 
Mavi, G. S.
, &amp; 
Sharma, A.
 (2023). Genome‐wide association study unravels genomic regions associated with chlorophyll fluorescence parameters in wheat (<em>Triticum aestivum</em> L.) under different sowing conditions. Plant Cell Reports, 42, 1453–1472. 10.1007/s00299-023-03041-6

</cite> [<a href="https://doi.org/10.1007/s00299-023-03041-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37338572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Reports&amp;title=Genome%E2%80%90wide%20association%20study%20unravels%20genomic%20regions%20associated%20with%20chlorophyll%20fluorescence%20parameters%20in%20wheat%20(Triticum%20aestivum%20L.)%20under%20different%20sowing%20conditions&amp;author=S.%20Gudi&amp;author=D.%20K.%20Saini&amp;author=P.%20Halladakeri&amp;author=G.%20Singh&amp;author=S.%20Singh&amp;volume=42&amp;publication_year=2023&amp;pages=1453-1472&amp;pmid=37338572&amp;doi=10.1007/s00299-023-03041-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0023">
<cite>

Gudi, S.
, 
Saini, D. K.
, 
Singh, G.
, 
Halladakeri, P.
, 
Kumar, P.
, 
Shamshad, M.
, 
Tanin, M. J.
, 
Singh, S.
, &amp; 
Sharma, A.
 (2022). Unravelling consensus genomic regions associated with quality traits in wheat using meta‐analysis of quantitative trait loci. Planta, 255, Article 115. 10.1007/s00425-022-03904-4

</cite> [<a href="https://doi.org/10.1007/s00425-022-03904-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35508739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=Unravelling%20consensus%20genomic%20regions%20associated%20with%20quality%20traits%20in%20wheat%20using%20meta%E2%80%90analysis%20of%20quantitative%20trait%20loci&amp;author=S.%20Gudi&amp;author=D.%20K.%20Saini&amp;author=G.%20Singh&amp;author=P.%20Halladakeri&amp;author=P.%20Kumar&amp;volume=255&amp;publication_year=2022&amp;pages=115&amp;pmid=35508739&amp;doi=10.1007/s00425-022-03904-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0024">
<cite>

Halder, J.
, 
Gill, H. S.
, 
Zhang, J.
, 
Altameemi, R.
, 
Olson, E.
, 
Turnipseed, B.
, &amp; 
Sehgal, S. K.
 (2023). Genome‐wide association analysis of spike and kernel traits in the U.S. hard winter wheat. The Plant Genome, 16, e20300. 10.1002/tpg2.20300

</cite> [<a href="https://doi.org/10.1002/tpg2.20300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36636831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genome%E2%80%90wide%20association%20analysis%20of%20spike%20and%20kernel%20traits%20in%20the%20U.S.%20hard%20winter%20wheat&amp;author=J.%20Halder&amp;author=H.%20S.%20Gill&amp;author=J.%20Zhang&amp;author=R.%20Altameemi&amp;author=E.%20Olson&amp;volume=16&amp;publication_year=2023&amp;pages=e20300&amp;pmid=36636831&amp;doi=10.1002/tpg2.20300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0025">
<cite>

Halladakeri, P.
, 
Gudi, S.
, 
Akhtar, S.
, 
Singh, G.
, 
Saini, D. K.
, 
Hilli, H. J.
, &amp; 
Mir, R. R.
 (2023). Meta‐analysis of the quantitative trait loci associated with agronomic traits, fertility restoration, disease resistance, and seed quality traits in pigeonpea (<em>Cajanus cajan</em> L.). The Plant Genome, 16(3), e20342. 10.1002/tpg2.20342

</cite> [<a href="https://doi.org/10.1002/tpg2.20342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37328945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Meta%E2%80%90analysis%20of%20the%20quantitative%20trait%20loci%20associated%20with%20agronomic%20traits,%20fertility%20restoration,%20disease%20resistance,%20and%20seed%20quality%20traits%20in%20pigeonpea%20(Cajanus%20cajan%20L.)&amp;author=P.%20Halladakeri&amp;author=S.%20Gudi&amp;author=S.%20Akhtar&amp;author=G.%20Singh&amp;author=D.%20K.%20Saini&amp;volume=16&amp;issue=3&amp;publication_year=2023&amp;pages=e20342&amp;pmid=37328945&amp;doi=10.1002/tpg2.20342&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0026">
<cite>

He, F.
, 
Pasam, R.
, 
Shi, F.
, 
Kant, S.
, 
Keeble‐Gagnere, G.
, 
Kay, P.
, 
Forrest, K.
, 
Fritz, A.
, 
Hucl, P.
, 
Wiebe, K.
, 
Knox, R.
, 
Cuthbert, R.
, 
Pozniak, C.
, 
Akhunova, A.
, 
Morrell, P. L.
, 
Davies, J. P.
, 
Webb, S. R.
, 
Spangenberg, G.
, 
Hayes, B.
, … 
Akhunov, E.
 (2019). Exome sequencing highlights the role of wild‐relative introgression in shaping the adaptive landscape of the wheat genome. Nature Genetics, 51, 896–904. 10.1038/s41588-019-0382-2

</cite> [<a href="https://doi.org/10.1038/s41588-019-0382-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31043759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Exome%20sequencing%20highlights%20the%20role%20of%20wild%E2%80%90relative%20introgression%20in%20shaping%20the%20adaptive%20landscape%20of%20the%20wheat%20genome&amp;author=F.%20He&amp;author=R.%20Pasam&amp;author=F.%20Shi&amp;author=S.%20Kant&amp;author=G.%20Keeble%E2%80%90Gagnere&amp;volume=51&amp;publication_year=2019&amp;pages=896-904&amp;pmid=31043759&amp;doi=10.1038/s41588-019-0382-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0027">
<cite>

Hu, Z.
, 
Li, J.
, 
Ding, S.
, 
Cheng, F.
, 
Li, X.
, 
Jiang, Y.
, 
Yu, J.
, 
Foyer, C. H.
, &amp; 
Shi, K.
 (2021). The protein kinase CPK28 phosphorylates ascorbate peroxidase and enhances thermotolerance in tomato. Plant Physiology, 186, 1302–1317. 10.1093/plphys/kiab120

</cite> [<a href="https://doi.org/10.1093/plphys/kiab120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8195530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33711164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=The%20protein%20kinase%20CPK28%20phosphorylates%20ascorbate%20peroxidase%20and%20enhances%20thermotolerance%20in%20tomato&amp;author=Z.%20Hu&amp;author=J.%20Li&amp;author=S.%20Ding&amp;author=F.%20Cheng&amp;author=X.%20Li&amp;volume=186&amp;publication_year=2021&amp;pages=1302-1317&amp;pmid=33711164&amp;doi=10.1093/plphys/kiab120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0028">
<cite>

Janni, M.
, 
Gullì, M.
, 
Maestri, E.
, 
Marmiroli, M.
, 
Valliyodan, B.
, 
Nguyen, H. T.
, &amp; 
Marmiroli, N.
 (2020). Molecular and genetic bases of heat stress responses in crop plants and breeding for increased resilience and productivity. Journal of Experimental Botany, 71, 3780–3802. 10.1093/jxb/eraa034

</cite> [<a href="https://doi.org/10.1093/jxb/eraa034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7316970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31970395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Molecular%20and%20genetic%20bases%20of%20heat%20stress%20responses%20in%20crop%20plants%20and%20breeding%20for%20increased%20resilience%20and%20productivity&amp;author=M.%20Janni&amp;author=M.%20Gull%C3%AC&amp;author=E.%20Maestri&amp;author=M.%20Marmiroli&amp;author=B.%20Valliyodan&amp;volume=71&amp;publication_year=2020&amp;pages=3780-3802&amp;pmid=31970395&amp;doi=10.1093/jxb/eraa034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0029">
<cite>

Khan, M. I.
, 
Kainat, Z.
, 
Maqbool, S.
, 
Mehwish, A.
, 
Ahmad, S.
, 
Suleman, H. M.
, 
Mahmood, Z.
, 
Ali, M.
, 
Aziz, A.
, 
Rasheed, A.
, &amp; 
Li, H.
 (2022). Genome‐wide association for heat tolerance at seedling stage in historical spring wheat cultivars. Frontiers in Plant Science, 13, 972481. 10.3389/fpls.2022.972481

</cite> [<a href="https://doi.org/10.3389/fpls.2022.972481" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9453861/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36092407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%E2%80%90wide%20association%20for%20heat%20tolerance%20at%20seedling%20stage%20in%20historical%20spring%20wheat%20cultivars&amp;author=M.%20I.%20Khan&amp;author=Z.%20Kainat&amp;author=S.%20Maqbool&amp;author=A.%20Mehwish&amp;author=S.%20Ahmad&amp;volume=13&amp;publication_year=2022&amp;pages=972481&amp;pmid=36092407&amp;doi=10.3389/fpls.2022.972481&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0030">
<cite>

Lawlor, D. W.
, &amp; 
Mitchell, R. A. C.
 (2000). Crop ecosystem responses to climatic change: Wheat. In Reddy K. R. &amp; Hodges H. F. (Eds.), Climate change and global crop productivity (pp. 57–80). CABI International.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Climate%20change%20and%20global%20crop%20productivity&amp;author=D.%20W.%20Lawlor&amp;author=R.%20A.%20C.%20Mitchell&amp;publication_year=2000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0031">
<cite>

Li, S.
, 
Chang, X.
, 
Wang, C.
, &amp; 
Jing, R.
 (2012). Mapping QTLs for seedling traits and heat tolerance indices in common wheat. Acta Botanica Boreali‐Occidentalia Sinica, 32(2012), 1525–1533.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Botanica%20Boreali%E2%80%90Occidentalia%20Sinica&amp;title=Mapping%20QTLs%20for%20seedling%20traits%20and%20heat%20tolerance%20indices%20in%20common%20wheat&amp;author=S.%20Li&amp;author=X.%20Chang&amp;author=C.%20Wang&amp;author=R.%20Jing&amp;volume=32&amp;issue=2012&amp;publication_year=2012&amp;pages=1525-1533&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0032">
<cite>

Liu, X.
, 
Huang, M.
, 
Fan, B.
, 
Buckler, E. S.
, &amp; 
Zhang, Z.
 (2016). Iterative usage of fixed and random effect models for powerful and efficient genome‐wide association studies. PLOS Genetics, 12, e1005767. 10.1371/journal.pgen.1005767

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005767" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4734661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26828793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=Iterative%20usage%20of%20fixed%20and%20random%20effect%20models%20for%20powerful%20and%20efficient%20genome%E2%80%90wide%20association%20studies&amp;author=X.%20Liu&amp;author=M.%20Huang&amp;author=B.%20Fan&amp;author=E.%20S.%20Buckler&amp;author=Z.%20Zhang&amp;volume=12&amp;publication_year=2016&amp;pages=e1005767&amp;pmid=26828793&amp;doi=10.1371/journal.pgen.1005767&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0033">
<cite>

Liu, Z.
, 
Xin, M.
, 
Qin, J.
, 
Peng, H.
, 
Ni, Z.
, 
Yao, Y.
, &amp; 
Sun, Q.
 (2015). Temporal transcriptome profiling reveals expression partitioning of homeologous genes contributing to heat and drought acclimation in wheat (<em>Triticum aestivum</em> L.). BMC Plant Biology, 15, Article 152. 10.1186/s12870-015-0511-8

</cite> [<a href="https://doi.org/10.1186/s12870-015-0511-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4474349/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26092253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Temporal%20transcriptome%20profiling%20reveals%20expression%20partitioning%20of%20homeologous%20genes%20contributing%20to%20heat%20and%20drought%20acclimation%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=Z.%20Liu&amp;author=M.%20Xin&amp;author=J.%20Qin&amp;author=H.%20Peng&amp;author=Z.%20Ni&amp;volume=15&amp;publication_year=2015&amp;pages=152&amp;pmid=26092253&amp;doi=10.1186/s12870-015-0511-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0034">
<cite>

Lu, L.
, 
Liu, H.
, 
Wu, Y.
, &amp; 
Yan, G.
 (2022). Wheat genotypes tolerant to heat at seedling stage tend to be also tolerant at adult stage: The possibility of early selection for heat tolerance breeding. The Crop Journal, 10(4), 1006–1013. 10.1016/j.cj.2022.01.005
</cite> [<a href="https://doi.org/10.1016/j.cj.2022.01.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Crop%20Journal&amp;title=Wheat%20genotypes%20tolerant%20to%20heat%20at%20seedling%20stage%20tend%20to%20be%20also%20tolerant%20at%20adult%20stage:%20The%20possibility%20of%20early%20selection%20for%20heat%20tolerance%20breeding&amp;author=L.%20Lu&amp;author=H.%20Liu&amp;author=Y.%20Wu&amp;author=G.%20Yan&amp;volume=10&amp;issue=4&amp;publication_year=2022&amp;pages=1006-1013&amp;doi=10.1016/j.cj.2022.01.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0035">
<cite>

Maulana, F.
, 
Ayalew, H.
, 
Anderson, J. D.
, 
Kumssa, T. T.
, 
Huang, W.
, &amp; 
Ma, X.‐F.
 (2018). Genome‐Wide Association Mapping of Seedling Heat Tolerance in Winter Wheat. Frontiers in Plant Science, 9, 1272. 10.3389/fpls.2018.01272

</cite> [<a href="https://doi.org/10.3389/fpls.2018.01272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6131858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30233617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%E2%80%90Wide%20Association%20Mapping%20of%20Seedling%20Heat%20Tolerance%20in%20Winter%20Wheat&amp;author=F.%20Maulana&amp;author=H.%20Ayalew&amp;author=J.%20D.%20Anderson&amp;author=T.%20T.%20Kumssa&amp;author=W.%20Huang&amp;volume=9&amp;publication_year=2018&amp;pages=1272&amp;pmid=30233617&amp;doi=10.3389/fpls.2018.01272&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0036">
<cite>

Muhammad, A.
, 
Hu, W.
, 
Li, Z.
, 
Li, J.
, 
Xie, G.
, 
Wang, J.
, &amp; 
Wang, L.
 (2020). Appraising the genetic architecture of kernel traits in hexaploid wheat using GWAS. International Journal of Molecular Sciences, 21, 5649. 10.3390/ijms21165649

</cite> [<a href="https://doi.org/10.3390/ijms21165649" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7460857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32781752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Appraising%20the%20genetic%20architecture%20of%20kernel%20traits%20in%20hexaploid%20wheat%20using%20GWAS&amp;author=A.%20Muhammad&amp;author=W.%20Hu&amp;author=Z.%20Li&amp;author=J.%20Li&amp;author=G.%20Xie&amp;volume=21&amp;publication_year=2020&amp;pages=5649&amp;pmid=32781752&amp;doi=10.3390/ijms21165649&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0037">
<cite>

Park, S.
, 
Li, J.
, 
Pittman, J. K.
, 
Berkowitz, G. A.
, 
Yang, H.
, 
Undurraga, S.
, 
Morris, J.
, 
Hirschi, K. D.
, &amp; 
Gaxiola, R. A.
 (2005). Up‐regulation of a H + ‐pyrophosphatase (H + ‐PPase) as a strategy to engineer drought‐resistant crop plants. Proceedings of the National Academy of Sciences, 102, 18830–18835. 10.1073/pnas.0509512102
</cite> [<a href="https://doi.org/10.1073/pnas.0509512102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1323196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16361442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Up%E2%80%90regulation%20of%20a%20H%20+%20%E2%80%90pyrophosphatase%20(H%20+%20%E2%80%90PPase)%20as%20a%20strategy%20to%20engineer%20drought%E2%80%90resistant%20crop%20plants&amp;author=S.%20Park&amp;author=J.%20Li&amp;author=J.%20K.%20Pittman&amp;author=G.%20A.%20Berkowitz&amp;author=H.%20Yang&amp;volume=102&amp;publication_year=2005&amp;pages=18830-18835&amp;pmid=16361442&amp;doi=10.1073/pnas.0509512102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0038">
<cite>

Patir‐Nebioglu, M. G.
, 
Andrés, Z.
, 
Krebs, M.
, 
Fink, F.
, 
Drzewicka, K.
, 
Stankovic‐Valentin, N.
, 
Segami, S.
, 
Schuck, S.
, 
Büttner, M.
, 
Hell, R.
, 
Maeshima, M.
, 
Melchior, F.
, &amp; 
Schumacher, K.
 (2019). RETRACTED: Pyrophosphate modulates plant stress responses via SUMOylation. Elife, 8, e44213. 10.7554/eLife.44213

</cite> [<a href="https://doi.org/10.7554/eLife.44213" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6382351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30785397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=RETRACTED:%20Pyrophosphate%20modulates%20plant%20stress%20responses%20via%20SUMOylation&amp;author=M.%20G.%20Patir%E2%80%90Nebioglu&amp;author=Z.%20Andr%C3%A9s&amp;author=M.%20Krebs&amp;author=F.%20Fink&amp;author=K.%20Drzewicka&amp;volume=8&amp;publication_year=2019&amp;pages=e44213&amp;pmid=30785397&amp;doi=10.7554/eLife.44213&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="/articles/PMC6435317/" class="text-red">Retracted</a>]</li>
<li id="tpg270071-bib-0039">
<cite>

Prasad, P. V. V.
, &amp; 
Djanaguiraman, M.
 (2014). Response of floret fertility and individual grain weight of wheat to high temperature stress: Sensitive stages and thresholds for temperature and duration. Functional Plant Biology, 41, 1261–1269. 10.1071/FP14061

</cite> [<a href="https://doi.org/10.1071/FP14061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32481075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=Response%20of%20floret%20fertility%20and%20individual%20grain%20weight%20of%20wheat%20to%20high%20temperature%20stress:%20Sensitive%20stages%20and%20thresholds%20for%20temperature%20and%20duration&amp;author=P.%20V.%20V.%20Prasad&amp;author=M.%20Djanaguiraman&amp;volume=41&amp;publication_year=2014&amp;pages=1261-1269&amp;pmid=32481075&amp;doi=10.1071/FP14061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0040">
<cite>

Pritchard, J. K.
, 
Stephens, M.
, &amp; 
Donnelly, P.
 (2000). Inference of population structure using multilocus genotype data. Genetics, 155, 945–959. 10.1093/genetics/155.2.945

</cite> [<a href="https://doi.org/10.1093/genetics/155.2.945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10835412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=J.%20K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;doi=10.1093/genetics/155.2.945&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0041">
<cite>

Qaseem, M. F.
, 
Qureshi, R.
, 
Shaheen, H.
, &amp; 
Shafqat, N.
 (2019). Genome‐wide association analyses for yield and yield‐related traits in bread wheat (<em>Triticum aestivum</em> L.) under pre‐anthesis combined heat and drought stress in field conditions. PLoS One, 14, e0213407. 10.1371/journal.pone.0213407

</cite> [<a href="https://doi.org/10.1371/journal.pone.0213407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6422278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30883588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Genome%E2%80%90wide%20association%20analyses%20for%20yield%20and%20yield%E2%80%90related%20traits%20in%20bread%20wheat%20(Triticum%20aestivum%20L.)%20under%20pre%E2%80%90anthesis%20combined%20heat%20and%20drought%20stress%20in%20field%20conditions&amp;author=M.%20F.%20Qaseem&amp;author=R.%20Qureshi&amp;author=H.%20Shaheen&amp;author=N.%20Shafqat&amp;volume=14&amp;publication_year=2019&amp;pages=e0213407&amp;pmid=30883588&amp;doi=10.1371/journal.pone.0213407&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0042">
<cite>

Rajamanickam, V.
, 
Sevanthi, A. M.
, 
Swarbreck, S. M.
, 
Gudi, S.
, 
Singh, N.
, 
Singh, V. K.
, &amp; 
Pandey, R.
 (2024). High‐throughput root phenotyping and association analysis identified potential genomic regions for phosphorus use efficiency in wheat (<em>Triticum aestivum</em> L.). Planta, 260(6), Article 142. 10.1007/s00425-024-04577-x

</cite> [<a href="https://doi.org/10.1007/s00425-024-04577-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39557700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Planta&amp;title=High%E2%80%90throughput%20root%20phenotyping%20and%20association%20analysis%20identified%20potential%20genomic%20regions%20for%20phosphorus%20use%20efficiency%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=V.%20Rajamanickam&amp;author=A.%20M.%20Sevanthi&amp;author=S.%20M.%20Swarbreck&amp;author=S.%20Gudi&amp;author=N.%20Singh&amp;volume=260&amp;issue=6&amp;publication_year=2024&amp;pages=142&amp;pmid=39557700&amp;doi=10.1007/s00425-024-04577-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0043">
<cite>

Rathan, N. D.
, 
Krishna, H.
, 
Ellur, R. K.
, 
Sehgal, D.
, 
Govindan, V.
, 
Ahlawat, A. K.
, 
Krishnappa, G.
, 
Jaiswal, J. P.
, 
Singh, J. B.
, 
SV, S.
, 
Ambati, D.
, 
Singh, S. K.
, 
Bajpai, K.
, &amp; 
Mahendru‐Singh, A.
 (2022). Genome‐wide association study identifies loci and candidate genes for grain micronutrients and quality traits in wheat (<em>Triticum aestivum</em> L.). Scientific Reports, 12, Article 7037. 10.1038/s41598-022-10618-w

</cite> [<a href="https://doi.org/10.1038/s41598-022-10618-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9054743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35487909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Genome%E2%80%90wide%20association%20study%20identifies%20loci%20and%20candidate%20genes%20for%20grain%20micronutrients%20and%20quality%20traits%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=N.%20D.%20Rathan&amp;author=H.%20Krishna&amp;author=R.%20K.%20Ellur&amp;author=D.%20Sehgal&amp;author=V.%20Govindan&amp;volume=12&amp;publication_year=2022&amp;pages=7037&amp;pmid=35487909&amp;doi=10.1038/s41598-022-10618-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0044">
<cite>

Schilling, R. K.
, 
Tester, M.
, 
Marschner, P.
, 
Plett, D. C.
, &amp; 
Roy, S. J.
 (2017). AVP1: One protein, many roles. Trends in Plant Science, 22, 154–162. 10.1016/j.tplants.2016.11.012

</cite> [<a href="https://doi.org/10.1016/j.tplants.2016.11.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27989652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=AVP1:%20One%20protein,%20many%20roles&amp;author=R.%20K.%20Schilling&amp;author=M.%20Tester&amp;author=P.%20Marschner&amp;author=D.%20C.%20Plett&amp;author=S.%20J.%20Roy&amp;volume=22&amp;publication_year=2017&amp;pages=154-162&amp;pmid=27989652&amp;doi=10.1016/j.tplants.2016.11.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0045">
<cite>

Sihag, P.
, 
Kumar, U.
, 
Sagwal, V.
, 
Kapoor, P.
, 
Singh, Y.
, 
Mehla, S.
, 
Balyan, P.
, 
Mir, R. R.
, 
Varshney, R. K.
, 
Singh, K. P.
, &amp; 
Dhankher, O. P.
 (2024). Effect of terminal heat stress on osmolyte accumulation and gene expression during grain filling in bread wheat (<em>Triticum aestivum</em> L.). The Plant Genome, 17, e20307. 10.1002/tpg2.20307

</cite> [<a href="https://doi.org/10.1002/tpg2.20307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36751876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Effect%20of%20terminal%20heat%20stress%20on%20osmolyte%20accumulation%20and%20gene%20expression%20during%20grain%20filling%20in%20bread%20wheat%20(Triticum%20aestivum%20L.)&amp;author=P.%20Sihag&amp;author=U.%20Kumar&amp;author=V.%20Sagwal&amp;author=P.%20Kapoor&amp;author=Y.%20Singh&amp;volume=17&amp;publication_year=2024&amp;pages=e20307&amp;pmid=36751876&amp;doi=10.1002/tpg2.20307&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0046">
<cite>

Singh, J.
, 
Bedi, S.
, 
Gudi, S.
, 
Kumar, P.
, &amp; 
Sharma, A.
 (2023). Coronal roots and stem lignin content as significant contributors for lodging tolerance in wheat (<em>Triticum aestivum</em> L.). Cereal Research Communications, 51, 639–647. 10.1007/s42976-022-00337-3
</cite> [<a href="https://doi.org/10.1007/s42976-022-00337-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cereal%20Research%20Communications&amp;title=Coronal%20roots%20and%20stem%20lignin%20content%20as%20significant%20contributors%20for%20lodging%20tolerance%20in%20wheat%20(Triticum%20aestivum%20L.)&amp;author=J.%20Singh&amp;author=S.%20Bedi&amp;author=S.%20Gudi&amp;author=P.%20Kumar&amp;author=A.%20Sharma&amp;volume=51&amp;publication_year=2023&amp;pages=639-647&amp;doi=10.1007/s42976-022-00337-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0047">
<cite>

Tanin, M. J.
, 
Saini, D. K.
, 
Sandhu, K. S.
, 
Pal, N.
, 
Gudi, S.
, 
Chaudhary, J.
, &amp; 
Sharma, A.
 (2022). Consensus genomic regions associated with multiple abiotic stress tolerance in wheat and implications for wheat breeding. Scientific Reports, 12, Article 13680. 10.1038/s41598-022-18149-0

</cite> [<a href="https://doi.org/10.1038/s41598-022-18149-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9372038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35953529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Consensus%20genomic%20regions%20associated%20with%20multiple%20abiotic%20stress%20tolerance%20in%20wheat%20and%20implications%20for%20wheat%20breeding&amp;author=M.%20J.%20Tanin&amp;author=D.%20K.%20Saini&amp;author=K.%20S.%20Sandhu&amp;author=N.%20Pal&amp;author=S.%20Gudi&amp;volume=12&amp;publication_year=2022&amp;pages=13680&amp;pmid=35953529&amp;doi=10.1038/s41598-022-18149-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0048">
<cite>

Tanin, M. J.
, 
Sharma, A.
, 
Ram, H.
, 
Singh, S.
, 
Srivastava, P.
, 
Mavi, G. S.
, 
Saini, D. K.
, 
Gudi, S.
, 
Kumar, P.
, 
Goyal, P.
, &amp; 
Sohu, V. S.
 (2023). Application of potassium nitrate and salicylic acid improves grain yield and related traits by delaying leaf senescence in Gpc‐B1 carrying advanced wheat genotypes. Frontiers in Plant Science, 14, 1107705. 10.3389/fpls.2023.1107705

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1107705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10389087/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37528976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Application%20of%20potassium%20nitrate%20and%20salicylic%20acid%20improves%20grain%20yield%20and%20related%20traits%20by%20delaying%20leaf%20senescence%20in%20Gpc%E2%80%90B1%20carrying%20advanced%20wheat%20genotypes&amp;author=M.%20J.%20Tanin&amp;author=A.%20Sharma&amp;author=H.%20Ram&amp;author=S.%20Singh&amp;author=P.%20Srivastava&amp;volume=14&amp;publication_year=2023&amp;pages=1107705&amp;pmid=37528976&amp;doi=10.3389/fpls.2023.1107705&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0049">
<cite>
R Core Team
. (2018). 
R: A language and environment for statistical computing
. R Foundation for Statistical Computing.</cite> [<a href="https://scholar.google.com/scholar_lookup?R%20Core%20Team%20.%20(2018).%20R:%20A%20language%20and%20environment%20for%20statistical%20computing%20.%20R%20Foundation%20for%20Statistical%20Computing." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0050">
<cite>

Tian, G.
, 
Wang, S.
, 
Wu, J.
, 
Wang, Y.
, 
Wang, X.
, 
Liu, S.
, 
Han, D.
, 
Xia, G.
, &amp; 
Wang, M.
 (2023). Allelic variation of TaWD40‐4B.1 contributes to drought tolerance by modulating catalase activity in wheat. Nature Communications, 14, Article 1200. 10.1038/s41467-023-36901-6
</cite> [<a href="https://doi.org/10.1038/s41467-023-36901-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9981739/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36864053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Allelic%20variation%20of%20TaWD40%E2%80%904B.1%20contributes%20to%20drought%20tolerance%20by%20modulating%20catalase%20activity%20in%20wheat&amp;author=G.%20Tian&amp;author=S.%20Wang&amp;author=J.%20Wu&amp;author=Y.%20Wang&amp;author=X.%20Wang&amp;volume=14&amp;publication_year=2023&amp;pages=1200&amp;pmid=36864053&amp;doi=10.1038/s41467-023-36901-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0051">
<cite>

Wang, J.
, &amp; 
Zhang, Z.
 (2021). GAPIT Version 3: Boosting power and accuracy for genomic association and prediction. Genomics, Proteomics &amp; Bioinformatics, 19, 629–640. 10.1016/j.gpb.2021.08.005
</cite> [<a href="https://doi.org/10.1016/j.gpb.2021.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9121400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34492338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics,%20Proteomics%20&amp;%20Bioinformatics&amp;title=GAPIT%20Version%203:%20Boosting%20power%20and%20accuracy%20for%20genomic%20association%20and%20prediction&amp;author=J.%20Wang&amp;author=Z.%20Zhang&amp;volume=19&amp;publication_year=2021&amp;pages=629-640&amp;pmid=34492338&amp;doi=10.1016/j.gpb.2021.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0052">
<cite>

You, J.
, 
Liu, H.
, 
Wang, S.
, 
Luo, W.
, 
Gou, L.
, 
Tang, H.
, 
Mu, Y.
, 
Deng, M.
, 
Jiang, Q.
, 
Chen, G.
, 
Qi, P.
, 
Peng, Y.
, 
Tang, L.
, 
Habib, A.
, 
Wei, Y.
, 
Zheng, Y.
, 
Lan, X.
, &amp; 
Ma, J.
 (2021). Spike density quantitative trait loci detection and analysis in tetraploid and hexaploid wheat recombinant inbred line populations. Frontiers in Plant Science, 12, 796397. 10.3389/fpls.2021.796397

</cite> [<a href="https://doi.org/10.3389/fpls.2021.796397" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8716915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34975986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Spike%20density%20quantitative%20trait%20loci%20detection%20and%20analysis%20in%20tetraploid%20and%20hexaploid%20wheat%20recombinant%20inbred%20line%20populations&amp;author=J.%20You&amp;author=H.%20Liu&amp;author=S.%20Wang&amp;author=W.%20Luo&amp;author=L.%20Gou&amp;volume=12&amp;publication_year=2021&amp;pages=796397&amp;pmid=34975986&amp;doi=10.3389/fpls.2021.796397&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0053">
<cite>

Yu, Y.
, 
Qian, Y.
, 
Jiang, M.
, 
Xu, J.
, 
Yang, J.
, 
Zhang, T.
, 
Gou, L.
, &amp; 
Pi, E.
 (2020). Regulation mechanisms of plant basic leucine zippers to various abiotic stresses. Frontiers in Plant Science, 11, 1258. 10.3389/fpls.2020.01258

</cite> [<a href="https://doi.org/10.3389/fpls.2020.01258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7468500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32973828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Regulation%20mechanisms%20of%20plant%20basic%20leucine%20zippers%20to%20various%20abiotic%20stresses&amp;author=Y.%20Yu&amp;author=Y.%20Qian&amp;author=M.%20Jiang&amp;author=J.%20Xu&amp;author=J.%20Yang&amp;volume=11&amp;publication_year=2020&amp;pages=1258&amp;pmid=32973828&amp;doi=10.3389/fpls.2020.01258&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270071-bib-0054">
<cite>

Zhao, C.
, 
Liu, B.
, 
Piao, S.
, 
Wang, X.
, 
Lobell, D. B.
, 
Huang, Y.
, 
Huang, M.
, 
Yao, Y.
, 
Bassu, S.
, 
Ciais, P.
, 
Durand, J.‐L.
, 
Elliott, J.
, 
Ewert, F.
, 
Janssens, I. A.
, 
Li, T.
, 
Lin, E.
, 
Liu, Q.
, 
Martre, P.
, 
Müller, C.
, … 
Asseng, S.
 (2017). Temperature increase reduces global yields of major crops in four independent estimates. Proceedings of the National Academy of Sciences, 114, 9326–9331. 10.1073/pnas.1701762114
</cite> [<a href="https://doi.org/10.1073/pnas.1701762114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5584412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28811375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Temperature%20increase%20reduces%20global%20yields%20of%20major%20crops%20in%20four%20independent%20estimates&amp;author=C.%20Zhao&amp;author=B.%20Liu&amp;author=S.%20Piao&amp;author=X.%20Wang&amp;author=D.%20B.%20Lobell&amp;volume=114&amp;publication_year=2017&amp;pages=9326-9331&amp;pmid=28811375&amp;doi=10.1073/pnas.1701762114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>
<strong>Supplementary Table 1</strong>. List of 216 germplasm lines used for association analysis in the present study.</p>
<p>
<strong>Supplementary Table 2</strong>. Number of single nucleotide polymorphisms (SNPs) distributed along the chromosomes and the sub‐genomes, and the total number of SNPs in whole genome.</p>
<p>
<strong>Supplementary Table 3</strong>. List of unique gene models identified from the significant marker‐trait associations (MTAs) associated with seedling traits under non‐heat stress (23°C) (blue) and heat stress (36°C) (green) treatment.</p>
<p>
<strong>Supplementary Table 4</strong>. List of differentially expressed genes (DEGs) identified in the present study.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12241833/bin/TPG2-18-e70071-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70071-s001.xlsx</a><sup> (53.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<p>
<strong>Supplementary Figure 1</strong>. Analysis of linkage disequilibrium (LD) at the whole genome and sub‐genome levels in the hexaploid spring wheat collection. <strong>(a)</strong> LD decay analysis of the whole genome; <strong>(b)</strong> LD decay analysis for the A‐sub‐genome; <strong>(c)</strong> LD decay analysis for the B‐sub‐genome; <strong>(d)</strong> LD decay analysis for the D‐sub‐genome.</p>
<p>
<strong>Supplementary Figure 2</strong>. Manhattan plot for the: <strong>(a)</strong> shoot fresh weight (SFW; mg); <strong>(b)</strong> root length (RL; cm); <strong>(c)</strong> coleoptile length (CL; cm) under non‐heat stress (23°C) treatment.</p>
<p>
<strong>Supplementary Figure 3</strong>. Manhattan plot for the: <strong>(a)</strong> shoot length (SL; cm); <strong>(b)</strong> root number (RN); <strong>(c)</strong> coleoptile length (CL; cm) under heat stress (36°C) treatment.</p>
<p>
<strong>Supplementary Figure 4</strong>. Estimating the allelic effects of significant marker‐trait associations (MTAs) identified in the present study: <strong>(a‐g)</strong> allelic effects of MTAs, scaffold62159_9784648 (a), scaffold62159_9693200 (b), scaffold63259_2360029 (c), scaffold63259_2360398 (d), scaffold128405_6019031 (e), scaffold10127_4250444 (f), and scaffold4077_4635398 (g) identified under non‐heat stress (23°C) treatment; <strong>(h‐w)</strong> allelic effects of MTAs, scaffold468‐1_279183 (h), scaffold76743_1777606 (i), scaffold9496_2310787 (j), scaffold142879_1376183 (k), scaffold21647_3085029 (l), scaffold26910_6990976 (m), scaffold4982_9981508 (n), scaffold2159_4802103 (o), scaffold71558_289995 (p), scaffold5611_406983 (q), scaffold131852_2297774 (r), scaffold22822_3587896 (s), scaffold6791_22287451 (t), scaffold43799_1029672 (u), scaffold38798_23175 (v), and scaffold146879_1644623 (w) identified under heat stress (36°C) treatment.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12241833/bin/TPG2-18-e70071-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70071-s002.docx</a><sup> (1.2MB, docx) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70071"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70071.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12241833/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12241833/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12241833%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12241833/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12241833/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12241833/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40635288/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12241833/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40635288/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12241833/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12241833/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="TZZQptkrXdOZ0cHAyjQCoM9ck5VXYqtSv2MggDlxcbK2khWUb5f4I1SAbk5GfZuO">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
