
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Identification of copper metabolism-related subtypes, the development of a prognosis model, and characterization of the immune landscape in colorectal cancer - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5342EE8AF3F7C30542EE00428B3704.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="discon">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364797/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Discover Oncology">
<meta name="citation_title" content="Identification of copper metabolism-related subtypes, the development of a prognosis model, and characterization of the immune landscape in colorectal cancer">
<meta name="citation_author" content="Geng Peng">
<meta name="citation_author_institution" content="Department of Gastrointestinal Surgery, The Eighth Affiliated Hospital of Southern Medical University, (The First People’s Hospital of Shunde Foshan), Shunde, Foshan, 528300 Guangdong Province China">
<meta name="citation_author_institution" content="The Second School of Clinical Medicine, Southern Medical University, Guangzhou, 510080 Guangdong Province China">
<meta name="citation_author" content="Lin Zhong">
<meta name="citation_author_institution" content="Department of Gastrointestinal Surgery, The Eighth Affiliated Hospital of Southern Medical University, (The First People’s Hospital of Shunde Foshan), Shunde, Foshan, 528300 Guangdong Province China">
<meta name="citation_author_institution" content="The Second School of Clinical Medicine, Southern Medical University, Guangzhou, 510080 Guangdong Province China">
<meta name="citation_author" content="Nan Lai">
<meta name="citation_author_institution" content="Department of Medicine Ultrasonics, Nanfang Hospital, Southern Medical University, Guangzhou, 510515 P.R. China">
<meta name="citation_author" content="Lina Luo">
<meta name="citation_author_institution" content="Department of Gastrointestinal Surgery, The Eighth Affiliated Hospital of Southern Medical University, (The First People’s Hospital of Shunde Foshan), Shunde, Foshan, 528300 Guangdong Province China">
<meta name="citation_author_institution" content="The Second School of Clinical Medicine, Southern Medical University, Guangzhou, 510080 Guangdong Province China">
<meta name="citation_author" content="Fu Cheng">
<meta name="citation_author_institution" content="Department of Gastrointestinal Surgery, The Eighth Affiliated Hospital of Southern Medical University, (The First People’s Hospital of Shunde Foshan), Shunde, Foshan, 528300 Guangdong Province China">
<meta name="citation_author_institution" content="The Second School of Clinical Medicine, Southern Medical University, Guangzhou, 510080 Guangdong Province China">
<meta name="citation_author" content="Manzhao Ouyang">
<meta name="citation_author_institution" content="Department of Gastrointestinal Surgery, The Eighth Affiliated Hospital of Southern Medical University, (The First People’s Hospital of Shunde Foshan), Shunde, Foshan, 528300 Guangdong Province China">
<meta name="citation_author_institution" content="The Second School of Clinical Medicine, Southern Medical University, Guangzhou, 510080 Guangdong Province China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="1584">
<meta name="citation_doi" content="10.1007/s12672-025-03183-x">
<meta name="citation_pmid" content="40830698">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364797/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364797/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364797/pdf/12672_2025_Article_3183.pdf">
<meta name="description" content="Abnormalities in copper metabolism are implicated in colorectal cancer (CRC) progression and unfavorable survival outcomes. Although there has been a recent surge of studies on cuprotosis-related genes, most have not elucidated the relationship ...">
<meta name="og:title" content="Identification of copper metabolism-related subtypes, the development of a prognosis model, and characterization of the immune landscape in colorectal cancer">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Abnormalities in copper metabolism are implicated in colorectal cancer (CRC) progression and unfavorable survival outcomes. Although there has been a recent surge of studies on cuprotosis-related genes, most have not elucidated the relationship ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364797/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12364797">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1007/s12672-025-03183-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12672_2025_Article_3183.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12364797%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12364797/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12364797/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364797/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-discon.png" alt="Discover Oncology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Discover Oncology" title="Link to Discover Oncology" shape="default" href="https://doi.org/10.1007/s12672-025-03183-x" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Discov Oncol</button></div>. 2025 Aug 19;16:1584. doi: <a href="https://doi.org/10.1007/s12672-025-03183-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1007/s12672-025-03183-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Discov%20Oncol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Discov%20Oncol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Discov%20Oncol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Discov%20Oncol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Identification of copper metabolism-related subtypes, the development of a prognosis model, and characterization of the immune landscape in colorectal cancer</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Geng Peng</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Geng Peng</span></h3>
<div class="p">
<sup>1</sup>Department of Gastrointestinal Surgery, The Eighth Affiliated Hospital of Southern Medical University, (The First People’s Hospital of Shunde Foshan), Shunde, Foshan, 528300 Guangdong Province China </div>
<div class="p">
<sup>2</sup>The Second School of Clinical Medicine, Southern Medical University, Guangzhou, 510080 Guangdong Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Geng Peng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhong%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Lin Zhong</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Lin Zhong</span></h3>
<div class="p">
<sup>1</sup>Department of Gastrointestinal Surgery, The Eighth Affiliated Hospital of Southern Medical University, (The First People’s Hospital of Shunde Foshan), Shunde, Foshan, 528300 Guangdong Province China </div>
<div class="p">
<sup>2</sup>The Second School of Clinical Medicine, Southern Medical University, Guangzhou, 510080 Guangdong Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhong%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lin Zhong</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lai%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Nan Lai</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Nan Lai</span></h3>
<div class="p">
<sup>3</sup>Department of Medicine Ultrasonics, Nanfang Hospital, Southern Medical University, Guangzhou, 510515 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lai%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nan Lai</span></a>
</div>
</div>
<sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Lina Luo</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Lina Luo</span></h3>
<div class="p">
<sup>1</sup>Department of Gastrointestinal Surgery, The Eighth Affiliated Hospital of Southern Medical University, (The First People’s Hospital of Shunde Foshan), Shunde, Foshan, 528300 Guangdong Province China </div>
<div class="p">
<sup>2</sup>The Second School of Clinical Medicine, Southern Medical University, Guangzhou, 510080 Guangdong Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lina Luo</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Fu Cheng</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Fu Cheng</span></h3>
<div class="p">
<sup>1</sup>Department of Gastrointestinal Surgery, The Eighth Affiliated Hospital of Southern Medical University, (The First People’s Hospital of Shunde Foshan), Shunde, Foshan, 528300 Guangdong Province China </div>
<div class="p">
<sup>2</sup>The Second School of Clinical Medicine, Southern Medical University, Guangzhou, 510080 Guangdong Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fu Cheng</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ouyang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Manzhao Ouyang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Manzhao Ouyang</span></h3>
<div class="p">
<sup>1</sup>Department of Gastrointestinal Surgery, The Eighth Affiliated Hospital of Southern Medical University, (The First People’s Hospital of Shunde Foshan), Shunde, Foshan, 528300 Guangdong Province China </div>
<div class="p">
<sup>2</sup>The Second School of Clinical Medicine, Southern Medical University, Guangzhou, 510080 Guangdong Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ouyang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Manzhao Ouyang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Gastrointestinal Surgery, The Eighth Affiliated Hospital of Southern Medical University, (The First People’s Hospital of Shunde Foshan), Shunde, Foshan, 528300 Guangdong Province China </div>
<div id="Aff2">
<sup>2</sup>The Second School of Clinical Medicine, Southern Medical University, Guangzhou, 510080 Guangdong Province China </div>
<div id="Aff3">
<sup>3</sup>Department of Medicine Ultrasonics, Nanfang Hospital, Southern Medical University, Guangzhou, 510515 P.R. China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 4; Accepted 2025 Jul 9; Collection date 2025 Dec.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12364797  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830698/" class="usa-link">40830698</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Objective</h3>
<p id="Par1">Abnormalities in copper metabolism are implicated in colorectal cancer (CRC) progression and unfavorable survival outcomes. Although there has been a recent surge of studies on cuprotosis-related genes, most have not elucidated the relationship between copper and tumors from the perspective of copper metabolism in tumors. This study explores the prognostic value of copper metabolism-related genes (CMGs) in CRC and their molecular characteristics within the tumor immune microenvironment.</p></section><section id="sec2"><h3 class="pmc_sec_title">Method</h3>
<p id="Par2">We investigated the expression profile of 112 copper CMGs which were obtained from MSigDB in 1340 CRC patients from four independent datasets (TCGA-COAD/READ, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17536" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE17536</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40967" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE40967</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103479" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE103479</a>). Cox regression analysis was used to identify CMGs related to prognosis genes. Then, we used consensus unsupervised clustering analysis to classify the 1340 patients into different CMGs subtypes. Differences in prognosis, clinicopathological features, tumor microenvironment score, abundance of immune cells, and DEGs between different CMGs subtypes were systematically analyzed. Further we classified patients into different gene subtypes based on the expression of DEGs in different CMGs subtypes. The Lasso Cox regression algorithm was used to calculate CMGs risk scores which was used to stratify CRC patients into high and low-risk groups. The OS time, clinicopathological features, stromal-immune score, abundance of immune cells, expression of immune checkpoints and drugs sensitivity between the two groups were compared. Finally, the nomogram incorporating CMGs risk scores, patient age, and TNM staging was constructed and validated.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">1340 CRC patients were classified into 3 distinct subtypes: CMGs subtype A exhibited the most active copper metabolism, followed by CMGs subtype B, with CMGs subtype C being the least active. Notably, patients in CMGs subtype A exhibited reduced overall survival (OS) compared to subtypes B and C, with an immune microenvironment enriched in TAMs and TANs, a paucity of CD8<sup>+</sup> T cells and plasma cells. The expression of immune checkpoints was highest in CMGs subtype A, including PD1, PDL1, TIGIT, and B7H3. Based on the DEGs between different CMGs subtypes, we identified two gene subtypes and gene subtype A demonstrated a strong association with poor survival outcomes and shared a similar tumor microenvironment with CMGs subtype A. CRC patients were further stratified into high and low-risk groups based on median CMGs risk scores. Patients in the high-risk group were associated with shorter OS, unfavorable survival outcomes and most of them were in advanced stages. Moreover, the high-risk group showed elevated stromal-immune scores, greater prevalence of TAMs and TANs, and higher expression of immune checkpoints, including PD1, TIM3, B7H3, and SIGLEC15. Significantly, high-risk group also had a higher incidence of microsatellite instability-high (MSI-H), tumor mutational burden (TMB) and somatic mutation, suggesting enhanced responsiveness to immunotherapy. Finally, The AUC values of the nomogram at 1, 3, 5, 10 years were 0.769, 0.745, 0.730, and 0.799. The calibration curves demonstrated substantial concordance with the ideal model in OS predictions.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par4">Heterogeneity in copper metabolism exists within CRC, and abnormalities in copper metabolism levels influence CRC progression and the immune microenvironment. CMGs are effective biomarkers for predicting the prognosis of CRC patient and guiding immunotherapy.</p></section><section id="sec5"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1007/s12672-025-03183-x.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Copper metabolism<sub>1</sub>, Tumor immunity<sub>2</sub>, Molecular subtypes<sub>3</sub>, Tumor microenvironment<sub>4</sub>, Colorectal cancer<sub>5</sub>, Risk score model<sub>6</sub></p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par5">Colorectal cancer (CRC) currently stands as the third most prevalent malignant tumor worldwide, with the second-highest mortality rate, posing a significant challenge to the global population’s health [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Despite substantial progress in the prevention, diagnosis, and treatment of CRC over the past few decades, the absence of effective early diagnostic methods leads to the majority of patients being diagnosed at advanced stages [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Therefore, there is an ongoing need for more in-depth research to explore the molecular and pathological mechanisms of CRC, seeking reliable and potent therapeutic approaches to further improve patient survival rates and quality of life [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>].</p>
<p id="Par6">Metabolic abnormalities are fundamental characteristics of tumors, influencing their tumorigenesis and progression [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. In recent years, metal ions such as manganese, zinc, and calcium have garnered widespread attention for their regulatory roles in the proliferation, apoptosis, invasion, metastasis, and immune modulation of tumors [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. In 2022, Tsvetkov and his colleagues unveiled the specific mechanism of cuprotosis that copper directly binds to the lipoylated components of the tricarboxylic acid (TCA) cycle, resulting in the aggregation of lipoylated proteins and loss of iron-sulfur cluster proteins [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. This study forced the world’s attention to focus on copper once again, revisiting the function of copper metabolism in tumors. Copper, as one of the essential metal elements in the human body, is intricately linked to growth, development, and disease states [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. For healthy adults, the recommended daily dietary copper intake is 0.9 mg [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. However, an excess or deficiency of copper within the body can lead to diseases such as Wilson’s disease or Menkes disease, resulting in associated pathological changes and functional impairments in the brain, liver, kidneys, and muscles [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. The current findings indicate that the concentration of copper within CRC is significantly higher than in normal tissues [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], and it is closely associated with clinical prognosis [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Consequently, the active metabolism of copper emerges as a fundamental mechanism promoting the progression of CRC. Ongoing studies elucidate that copper profoundly influences tumor biology, including proliferation, migration, metastasis, energy metabolism, and angiogenesis [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Copper promotes tumor growth by activating the MAPK signaling pathway [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] and induces tumor autophagy through the mediation of ULK1/2 [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. Acting as a crucial catalytic cofactor for the activation of LOX and LOXL, copper participates in the cross-linking of collagen and elastin [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. This engagement enables the reshaping of the extracellular matrix and facilitates epithelial-mesenchymal transition (EMT) [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Furthermore, by inhibiting the activity of phosphodiesterase 3B (PDE3B), copper suppresses the degradation of intracellular cAMP to regulate lipolysis [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Moreover, activated by copper, various pro-angiogenic factors (such as VEGF, angiogenin, basic fibroblast growth factor, TGF-β) are released through the NF-κB or HIF-α pathways, influencing the generation of neovascularization around the tumor [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>].</p>
<p id="Par7">Although copper can promote tumor progression through a variety of pathways, there are still many key issues that have not yet been investigated. For example, copper activates signaling pathways by directly binding to MEK1 and ULK1/2, which may be one of the bases for the function of copper, but the specific mechanism remains unknown. Furthermore, the role of copper in tumor immunity is also poorly understood. Although it has been shown that copper plays a crucial role in maintaining normal immune function, such as enhancing T cell proliferation and cytotoxic function, improving neutrophil phagocytosis, and reducing plasma cell antibody secretion, studies in the field of tumor immunity are limited [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. With the discovery of the mechanisms underlying cuprotosis, there has been a recent surge in studies about cuprotosis-related genes and tumor development. However, these studies have not been grounded in the context of copper metabolism in tumors, thus lacking a more comprehensive perspective to interpret the influence of copper metabolism on tumors’ biological behaviors, particularly concerning the tumor immune microenvironment.</p>
<p id="Par8">Consequently, we applied copper metabolism-related genes (CMGs) as prognostic indicators for CRC, with exploration of their intrinsic relationships, metabolic pathways, and assessment of immunotherapeutic responsiveness. This approach holds immense promise for practical application. In this study, we comprehensively evaluated the expression of CMGs in CRC using consensus unsupervised clustering analysis to classify 1340 patients into 3 CMGs subtypes, and then further classified them into 2 gene subtypes based on differential genes (DEGs) between the CMGs subtypes. Using the CIBERSORT and ESTIMATE algorithms, the immune landscapes among CMGs subtypes and between gene subtypes were evaluated. Finally, a CMGs risk score model was constructed from the prognostic DEGs, which, combined with clinical characteristics to predict the survival prognosis, immune landscape features, and responsiveness to immunotherapy in CRC patients.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Data source</h3>
<p id="Par9">Figure <a href="#Fig1" class="usa-link">1</a> illustrates the workflow of this study. Gene expression data (fragments per kilobase million, FPKM), along with relevant prognostic and clinical pathological data, were obtained from The Cancer Genome Atlas (TCGA) (<a href="https://portal.gdc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov/</a>) and the Gene Expression Omnibus (GEO) (<a href="https://www.ncbi.nlm.nih.gov/geo/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/</a>). A total of three GEO datasets (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17536" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE17536</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40967" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE40967</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103479" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE103479</a>) and TCGA (colon adenocarcinoma/rectum adenocarcinoma) data were included in the analysis. Patients with incomplete clinical information or unclear survival status were excluded. Only treatment-naive primary tumor samples were included to ensure consistency. The FPKM from TCGA was transformed into transcripts per kilobase million (TPM) and merged with the GEO gene expression data after batch effect removal using the “ComBat” package, resulting in a gene expression matrix containing 1373 samples. Prognostic data and clinical pathological characteristics were extracted from all four datasets, including survival status, overall survival (OS) time, age, gender, stage, TP53 mutation, KRAS mutation, and BRAF mutation. Data with incomplete prognostic information were excluded, resulting in a clinical dataset comprising 1340 patients.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12364797_12672_2025_3183_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9a06/12364797/336d423f2248/12672_2025_3183_Fig1_HTML.jpg" loading="lazy" id="d33e380" height="895" width="709" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Workflow of this study</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Consensus clustering analysis of CMGs</h3>
<p id="Par11">Tengenesets(GOBP_CELLULAR_RESPONSE_TO_COPPER_ION,GOBP_COPPER_ION_HOMEOSTASIS,GOBP_COPPER_ION_IMPORT, GOBP_COPPER_ION_TRANSMEMBRANE_TRANSPORT,GOBP_COPPER_ION_TRANSPORT,GOBP_INTRACELLULAR_COPPER_ION_HOMEOSTASIS,GOBP_RESPONSE_TO_COPPER_ION,GOMF_COPPER_CHAPERONE_ACTIVITY,GOMF_COPPER_ION_BINDING,GOMF_COPPER_ION_TRANSMEMBRANE_TRANSPORTER_ACTIVITY) were downloaded from MSigDB (<a href="https://www.gsea-msigdb.org/gsea/msigdb/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.gsea-msigdb.org/gsea/msigdb/</a>) to obtain CMGs. The gene sets were merged to remove duplicate genes, and the intersection with the gene list from the gene expression data was taken, resulting in 112 CMGs included for further analysis (Table <a href="#MOESM1" class="usa-link">S1</a>). After conducting univariate Cox regression analysis on CMGs, Spearman correlation analysis was employed to calculate the r-values of CMGs, which were then visualized using Cytoscape (v3.9.1). The R package “ConsensusClusterPlus” was used for consensus unsupervised clustering analysis to classify the 1340 patients into different CMGs subtypes. The clustering procedure adhered to the following criteria: Firstly, the cumulative distribution function (CDF) curve exhibited a gradual and smooth increase. Secondly, none of the groups were characterized by a small sample size. Finally, following clustering, an increase in intra-group correlation and a decrease in inter-group correlation were observed. To further validate the clustering of CMGs subtypes, the expression of the 112 CMGs was dimensionally reduced using the R package “Rtsne”. Finally, Gene set variation analysis (GSVA) was performed using the hallmark gene set (c2.cp.kegg.v7.2) sourced from the MSigDB database.</p></section><section id="Sec5"><h3 class="pmc_sec_title">The differences between molecular subtypes with the prognosis and clinicopathological features of CRC</h3>
<p id="Par12">Using the R package “pheatmap”, the expression of CMGs or DEGs in CRC patients, molecular subtypes, TP53 mutation, KRAS mutation, BRAF mutation, stage, gender, survival status, and OS time were displayed in heatmap format. The “survival” and “survminer” R packages were utilized for Kaplan-Meier analysis to calculate differences in OS time between different molecular subtypes.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Assessment of tumor microenvironment (TME) score, abundance of immune cells, and expression of immune-related molecules between molecular subtypes</h3>
<p id="Par13">We utilized the “ESTIMATE” algorithm to derive stromal and immune scores for each CRC patient. Moreover, employing the “CIBERSORT” algorithm, we computed the infiltration abundance of 22 distinct immune cell types in CRC patients. Based on molecular subtypes, we conducted a comparison of immune checkpoint molecule expressions, including PDL1, PD1, CD276 (B7H3), TIGIT, and others.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Identification and functional annotation of DEGs</h3>
<p id="Par14">DEGs among distinct CMGs subtypes were identified using the “limma” package in R, with a fold-change of 1.5 and an adjusted p-value of &lt; 0.05. Following DEGs identification, functional enrichment analyses were conducted to elucidate potential functions, identify associated gene functions, and reveal enriched pathways. These analyses were performed using the “clusterprofiler” package in R.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Construction of the prognostic CMGs risk score</h3>
<p id="Par15">To better investigate the relationship between CMGs and clinical prognosis, we initially subjected the obtained CMGs-related DEGs to univariate Cox regression analysis. Subsequently, based on the expression of prognostic DEGs, patients were grouped into different gene subtypes using an unsupervised clustering method, followed by further analysis of the differences in TME and expression of immune-related molecules. Then, the “caret” package in R randomly divided all CRC patients into a training set (<em>n</em> = 938) and a test set (<em>n</em> = 397) in a 7:3 ratio.</p>
<p id="Par16">The Lasso Cox regression algorithm was employed to identify CMGs-related prognostic genes, aiming to minimize the risk of overfitting. This analysis was executed using the “glmnet” package in R. Finally, based on the minimum partial likelihood deviance, a group of candidate genes was subjected to a multivariate regression analysis to obtain the coefficient (coef) for each gene. The calculation of the CMGs risk score followed this formula: CMGs risk score = Σ(Expi * coefi).</p>
<p id="Par17">After obtaining the CMGs risk score for each patient in the training set through the formula, the 938 patients were divided into high and low-risk groups based on the median. Subsequently, Kaplan-Meier analysis was performed on the OS time of the two groups. Finally, the “timeROC” package in R was utilized to calculate the area under the time-dependent receiver operating characteristic (ROC) curve for the high and low-risk groups at 1-, 3-, 5-, and 10-years. The same analysis steps were applied to the test set data.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Tissue samples</h3>
<p id="Par18">24 pairs of CRC and adjacent non-tumor tissue samples were collected from CRC patients at Shunde Hospital of Southern Medical University. A portion of the samples was preserved at −80 °C using RNAlater (Invitrogen, Carlsbad, CA, USA), while the remaining samples were fixed in formalin and embedded in paraffin.</p></section><section id="Sec10"><h3 class="pmc_sec_title">RNA isolation and quantitative real-time polymerase chain reaction PCR (RT-qPCR)</h3>
<p id="Par19">Total RNA was extracted from CRC patient tissues using TRIzol reagent (Invitrogen, Carlsbad, CA, USA). Complementary DNA (cDNA) was synthesized from the extracted RNA using the HiScript Q RT SuperMix reagent kit (Vazyme, Nanjing, China). RT-qPCR was performed using ChamQ Universal SYBR qPCR Master Mix (Vazyme) on a 7500 RT-qPCR System (Applied Biosystems, Waltham, MA, USA). The data were normalized using ACTIN as the internal control and analyzed with the 2^(-ΔCt) method. Statistical comparisons were conducted using paired Wilcoxon tests and data visualization was performed with the “ggpubr” R package. The primer sequences used for qRT-PCR in this study are provided in Table S6.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Immunohistochemical (IHC) staining analysis of clinical tissue specimens</h3>
<p id="Par20">IHC staining analysis was performed on paraffin-embedded tissues to detect the protein expression of SLC2A3 and HOXC6 in 24 tumor tissues and adjacent normal tissues according to the standard immunoperoxidase staining procedure. All slides were incubated with anti- SLC2A3 (1:1000; 20403-1-AP, Proteintech) and anti- HOXC6 (1:500, ab41587, Abcam) according to the manufacturer’s instructions. The expression score was independently assessed in a blinded manner by two pathologists using the immunoreactivity score, which incorporates both the proportion of positively stained cells and staining intensity. A third pathologist was asked to re-evaluate the slides, when discrepancy occurred between the two pathologists. The percentage of positive cells was graded as follows: 0 (negative), 1 (&lt; 10%), 2 (10–50%), 3 (51–80%), and 4 (&gt; 80%). The intensity of staining was graded as follows: 0 (no color reaction), 1 (mild reaction), (moderate reaction), and 3 (intense reaction). The percentage of positive tumor cells and the staining intensity were multiplied to produce a weighted score for each sample. Immunoreactivity score of 8–12 was defined as high expression level and immunoreactivity score of 0–7 was defined as low expression. McNemar’s test was used to statistical analysis.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Clinical correlation and stratified analyses of the prognostic CMGs risk score</h3>
<p id="Par21">Chi-square tests were employed to examine the relationships between the CMGs risk groups and various clinical characteristics, including age, gender, stage, TP53 mutation, KRAS mutation, and BRAF mutation. Wilcoxon tests were utilized to compare CMGs risk scores between different groups based on the aforementioned clinical characteristics. Furthermore, a stratified Kaplan-Meier analysis was performed to assess whether the CMGs risk score maintained its predictive efficacy across diverse subgroups based on clinical characteristics.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Assessment of correlation between CMGs risk score and microsatellite instability (MSI), as well as cancer stem cell (CSC) index</h3>
<p id="Par22">To assess the proportions of tumor-infiltrating immune cells in the TME, CIBERSORT was performed to quantify the abundance of 22 infiltrating immune cells in heterogeneous samples within the low and high-risk groups. We assessed the stromal score and immune score in low and high-risk groups by the ESTIMATE algorithm, and the results were visualized with a jitter boxplot. The correlation between the fractions of the 22 infiltrating immune cells and the CMGs risk score as well as the expression of 11 CMGs were explored. The expression of immune checkpoints between the low and high-risk groups was visualized using boxplots. We utilized the “cBioPortalData” package to acquire MSI data (MANTIS score) for CRC. The MANTIS score is a predictive measure of a patient’s MSI status, as introduced in an article by Bonneville et al. [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. The CSC index was obtained through the “synapser” package from <a href="http://www.synapse.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.synapse.org</a>. Subsequently, we processed the data, extracted the expression matrix, and then used the “gelnet” package to construct a model and predict the CSC index of the CRC expression matrix. Lastly, we evaluated the correlations between the CMGs risk score and MSI, as well as CSC.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Mutation and drug susceptibility analysis</h3>
<p id="Par23">To identify somatic mutations in CRC patients between high and low-risk groups, we utilized the Mutation Annotation Format (MAF) extracted from the TCGA database, by using the “maftools” R package. Furthermore, the TMB score was computed for each CRC patient in the two groups. To investigate different therapeutic responses to chemotherapeutic drugs among CRC patients in the two groups, the half maximal inhibitory concentration (IC50) values were calculated using the “pRRophetic” package.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Construction and validation of a nomogram-based scoring system</h3>
<p id="Par24">Clinical characteristics and CMGs risk scores were utilized to formulate a predictive nomogram through the “rms” package, aligning with the results of independent prognosis analysis. In this nomogram scoring system, each variable was assigned a corresponding score, and the cumulative score was calculated by summing the scores across all variables for each sample. Time-dependent ROC curves for 1-, 3-, 5-, and 10-years survivals were used to evaluate the nomogram’s performance. Calibration plots of the nomogram were generated to illustrate the predictive accuracy concerning the anticipated 1-, 3-, 5- and 10-year survival outcomes in comparison to the ideal model.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Statistical analyses</h3>
<p id="Par25">All statistical analyses were performed using R version 4.3.2. Statistical significance was set at <em>p</em> &lt; 0.05.</p></section></section><section id="Sec18"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec19"><h3 class="pmc_sec_title">Identification of CMGs subtypes in CRC</h3>
<p id="Par26">The workflow of our study is illustrated in Fig. <a href="#Fig1" class="usa-link">1</a>. To fully understand the expression profile of CMGs, 1340 patients from four eligible cohorts (TCGA-COAD/READ, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17536" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE17536</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40967" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE40967</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103479" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE103479</a>) were included for further analysis. Detailed information on the 1340 CRC patients is presented in Table <a href="#MOESM1" class="usa-link">S1</a>. Univariate Cox regression was performed to reveal the prognostic values of 112 CMGs (<em>p</em> &lt; 0.05). 27 prognostic CMGs were selected for analysis using Multivariate Cox regression, 8 of which were identified as independent predictive factors (Fig. <a href="#Fig2" class="usa-link">2</a>A). The correlation analysis of prognostic CMGs is shown in Fig. <a href="#Fig2" class="usa-link">2</a>B.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12364797_12672_2025_3183_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9a06/12364797/f5f84114c9d7/12672_2025_3183_Fig2_HTML.jpg" loading="lazy" id="d33e511" height="936" width="709" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Three CMGs subtypes identified by consistent clustering and differences in clinicopathological and biological characteristics. (<strong>A)</strong> Forest plot exhibiting the prognostic CMGs following univariate and multivariate Cox regression analyses. 27 CMGs included in the analysis of multivariate COX regression and 8 CMGs identified as independent risk factors (p.val &lt; 0.05). (<strong>B</strong>) Correlation analysis of 27 prognostic CMGs visualized by Cytoscape. Genes represented in orange pies indicate risk factors, while those in green pies represent favorable factors. The sizes of the pies denote significance. Orange lines represent positive correlations and blue lines represent negative correlations. (<strong>C</strong>) Consensus matrix of all CRC cohorts for k = 3. (<strong>D</strong>) t-SNE analysis for the 1340 CRC patients, each dot represents a single sample. (<strong>E</strong>) Kaplan-Meier survival analysis for the overall survival of the three CMGs subtypes in CRC. (<strong>F</strong>) Heatmap illustrating the differences in clinicopathological characteristics and expression levels of CMGs between the three distinct subtypes. CMGs, copper metabolism-related gene; t-SNE, t-distributed stochastic neighbor embedding; CRC, colorectal cancer</p></figcaption></figure><p id="Par28">For further exploration of the expression profile of CMGs in CRC, we used the consensus unsupervised clustering algorithm to classify CRC patients according to the expression of the 112 CMGs. The optimal number of clusters was determined as k = 3, based on the gradual increase and subsequent plateau of the cumulative distribution function (CDF) curve, distinct separation between clusters, and sufficient sample size within each group (Fig. <a href="#MOESM1" class="usa-link">S1</a>). Detailed selection criteria are provided in the Materials and Methods section. As a result, 1340 CRC patients were classified into 3 unique subtypes and significant differences in the expression of CMGs were demonstrated by t-distributed stochastic neighbor embedding (t-SNE) analysis (Fig. <a href="#Fig2" class="usa-link">2</a>C and D). The Kaplan-Meier curves showed that patients in CMGs subtypes A had a shorter OS time compared to those in CMGs subtypes B or C (log-rank test, <em>p</em> = 0.019; Fig. <a href="#Fig2" class="usa-link">2</a>E). Kruskal-Wallis test revealed significant differences (<em>p</em> &lt; 0.05) among 97 out of 112 CMGs in three subtypes. The expression of 45 CMGs was highest in CMGs subtype A, 30 CMGs were highest in CMGs subtype B, and 22 CMGs were highest in CMGs subtype C, based on the comparison of mean rank. (Table <a href="#MOESM2" class="usa-link">S2</a>). Furthermore, scoring based on mean rank (the highest mean rank is assigned 3 points, medium is assigned 2 points, and the lowest is assigned 1 point), CMGs subtype A also scored higher than CMGs subtype B and CMGs subtype C (Table <a href="#MOESM2" class="usa-link">S2</a>). Collectively, these findings show that CMGs subtype A have the most active copper metabolism, followed by subtype B, while subtype C have a copper metabolism that was considerably less active. Expression of CMGs, BRAF mutation, KRAS mutation, TP53 mutation, and clinical characteristics in the three subtypes are shown by heatmap (Fig. <a href="#Fig2" class="usa-link">2</a>F).</p></section><section id="Sec20"><h3 class="pmc_sec_title">Characteristics of the TME in CMGs subtypes</h3>
<p id="Par29">GSVA enrichment analysis showed that three CMGs subtypes display distinctive differences in KEGG pathways (Table <a href="#MOESM3" class="usa-link">S3</a>). As shown in Fig. <a href="#Fig3" class="usa-link">3</a>A, CMGs subtype A is significantly enriched in glycosaminoglycan biosynthesis chondroitin sulfate, focal adhesion, extracellular matrix (ECM) receptor interaction, leishmania infection, cell adhesion molecules cams, viral myocarditis, regulation of actin cytoskeleton, pathways in cancer, leukocyte transendothelial migration, and chemokine signaling pathway compared to the other two subtypes. Using the CIBERSORT algorithm, we assessed the 22 human immune cell subsets in each CRC sample to gain insights into the function of CMGs in the TME of CRC. Among the three subtypes, there were clear differences in the infiltration of most immune cells in TME (Fig. <a href="#Fig3" class="usa-link">3</a>B). Gamma delta T cells, neutrophils, M0 macrophages, M1 macrophages, and M2 macrophages, as well as resting mast cells, were more prevalent in CMGs subtype A than subtype B or subtype C (Dunn-Bonferroni test, adj.<em>p</em> &lt; 0.05). Meanwhile, CMGs subtype A contains fewer plasma cells, CD8<sup>+</sup> T cells, resting and activated memory CD4<sup>+</sup> T cells, regulatory T cells (Tregs), resting NK cells, monocytes, resting and activated dendritic cells, and activated mast cells (Dunn-Bonferroni test, adj.<em>p</em> &lt; 0.05). CMGs subtypes B and C exhibit similar immune cell compositions in plasma cells, Tregs, M0 macrophages, as well as resting and activated dendritic cells. CMGs subtype C demonstrates the highest immune cell composition in plasma cells, resting and activated memory CD4<sup>+</sup> T cells, resting NK cells, monocytes, and resting and activated dendritic cells. However, it notably lacks follicular helper T cells, gamma delta T cells, eosinophils, neutrophils, M0 macrophages, M1 macrophages, and M2 macrophages compared to the other two subtypes (Dunn-Bonferroni test, adj.<em>p</em> &lt; 0.05).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12364797_12672_2025_3183_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9a06/12364797/56c0e49d0832/12672_2025_3183_Fig3_HTML.jpg" loading="lazy" id="d33e593" height="564" width="709" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differences in biological pathways and TME between distinct subtypes (<strong>A</strong>) GSVA of biological pathways between three CMGs subtypes. (<strong>B</strong>) Fraction of tumor-infiltrating immune cells in CMGs subtypes. (<strong>C</strong>,<strong> D</strong>) Expression of PDL1 and PD1 between distinct subtypes. (<strong>E</strong>) TME scores in different CMGs subtypes. TME, tumor microenvironment; GSVA, gene set variation analysis; CMGs, copper metabolism-related gene; PDL1, programmed death ligand 1; PD1, programmed cell death protein 1</p></figcaption></figure><p id="Par31">We further analyzed the expression of immune-related molecules in the three subtypes. PDL1 exhibited the highest expression in subtype A, followed by subtype B, and was lowest in subtype C (Fig. <a href="#Fig3" class="usa-link">3</a>C). Additionally, the expressions of PD1 in subtypes A and B was significantly higher than in subtype C. The expression of LAG3 in subtypes A and B were higher than in subtype C, with its ligands LGALS3 and CLEC4G showing higher expression in subtype B (Fig. <a href="#MOESM2" class="usa-link">S2</a>A). The distribution of TIGIT, TIM3, and B7H3 was similar to that of PDL1, with higher expression observed in subtype A compared to the other two groups (Fig. <a href="#MOESM2" class="usa-link">S2</a>B-S2D). The expression of TIGIT ligand (PVR), TIM3 ligands (HMGB1, LGALS9), HVEM and its ligand (BTLA), Siglec15, B2M, TAP1, and TAP2 varied significantly among the three subtypes (Fig. <a href="#MOESM2" class="usa-link">S2</a>B-S2G).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Using DEGs to identify different gene subtypes</h3>
<p id="Par32">By using the R package “limma”, we identified the DEGs based on the three CMGs subtypes. There are 98 DEGs between CMGs subtype A and subtype B, 32 DEGs between CMGs subtype B and subtype C, and 213 DEGs between CMGs subtype A and subtype C (Table S4). In total, 250 DEGs were included for further analysis. GO and KEGG enrichment analyses were performed to explore the potential biological functions. The GO enrichment analysis results of CMGs subtype A-B had 26 identical enrichments with the results of CMGs subtype A-C (Fig. <a href="#Fig4" class="usa-link">4</a>A and B), and most of these identical enrichment results were related to extracellular matrix composition, cell-extracellular matrix attachment, and titanase activity, which implies that copper metabolism in CRC may be involved in biological behaviors such as extracellular matrix remodeling, metastasis, and invasion. It is noteworthy that in CMGs subtype A (highly active copper metabolism) compared to CMGs subtype C (inactive copper metabolism), the GO results were enriched in neutrophil and granulocyte chemotaxis, migration, complement components, chemokine, and cytokine activity, suggesting that copper metabolism may have a significant influence on the immune response in CRC. Similarly, in the KEGG analysis, similar immune-related pathways are displayed (Fig. <a href="#Fig4" class="usa-link">4</a>D). However, in CMGs subtype A (highly active copper metabolism) compared to CMGs subtype B (moderate active copper metabolism), there is no such similar enrichment in GO and KEGG results (Fig. <a href="#Fig4" class="usa-link">4</a>A and C). Subsequently, we conducted univariate Cox regression analysis on 250 DEGs, resulting in 152 OS-related DEGs (<em>p</em> &lt; 0.05, Table S5). To further analyze the potential functions and mechanisms of copper metabolism related DEGs, we employed the consensus unsupervised clustering algorithm to classify CRC patients into two gene subtypes based on the 152 prognostic DEGs (Fig. <a href="#Fig4" class="usa-link">4</a>E, Fig. <a href="#MOESM3" class="usa-link">S3</a>). Kaplan-Meier curves demonstrated that the OS time of gene subtype A was significantly lower than that of gene subtype B (log-rank test, <em>p</em> = 0.007; Fig. <a href="#Fig4" class="usa-link">4</a>F). Moreover, the expression of most CMGs was significantly higher in gene subtype A compared to gene subtype B (Fig. S4). This observation suggests a strong correlation between active copper metabolism and unfavorable clinical outcomes. There were significant differences between gene subtype A and gene subtype B in terms of BRAF mutation, TNM stage, gender, survival time, and outcome (Wilcoxon test, <em>p</em> &lt; 0.05; Fig. <a href="#Fig5" class="usa-link">5</a>A).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12364797_12672_2025_3183_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9a06/12364797/a2a61de82b40/12672_2025_3183_Fig4_HTML.jpg" loading="lazy" id="d33e668" height="1021" width="709" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differences in biological pathways and identification of gene subtypes based on DEGs. (<strong>A</strong>,<strong> C</strong>) GO (above) and KEGG (below) enrichment analyses of DEGs between CMGs subtype A and B. (<strong>B</strong>,<strong> D</strong>) GO (above) and KEGG (below) enrichment analyses of DEGs between CMGs subtype A and C. (<strong>E</strong>) Consensus matrix of all CRC cohorts for k = 2. (F) Kaplan-Meier survival analysis for the overall survival of the two gene subtypes in CRC. DEG, differentially expressed gene; GO, Gene Ontology; KEGG, Kyoto Encyclopedia of Genes and Genomes; CMGs, copper metabolism-related gene; CRC, colorectal cancer</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12364797_12672_2025_3183_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9a06/12364797/b2fd4b2d2e1f/12672_2025_3183_Fig5_HTML.jpg" loading="lazy" id="d33e691" height="551" width="709" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differences in biological pathways and TME between two gene subtypes. (<strong>A</strong>) The differences in clinicopathologic features and expression levels of DEGs. (<strong>B</strong>) The distribution of 22 infiltrating immune cell types across the two subtypes of CRC. (<strong>C</strong>) Expression of PDL1 and PD1 between distinct two subtypes. (<strong>D</strong>) TME scores in different gene subtypes. TME, tumor microenvironment; DEG, differentially expressed gene; CRC, colorectal cancer; PDL1, programmed death ligand 1; PD1, programmed cell death protein 1</p></figcaption></figure><p id="Par35">Additionally, significant differences were observed in immune cells and immune-related molecules between gene subtype A and gene subtype B. Plasma cells, CD8<sup>+</sup> T cells, resting CD4<sup>+</sup> memory T cells, activated memory CD4<sup>+</sup> T cells, helper follicular T cells, activated NK cells, monocytes, activated dendritic cells, and activated mast cells were more abundant in the TME of gene subtype B, while gamma delta T cells, M0 macrophages, M1 macrophages, M2 macrophages, resting mast cells, eosinophils, and neutrophils were more abundant in the TME of gene subtype A (Wilcoxon test, <em>p</em> &lt; 0.05; Fig. <a href="#Fig5" class="usa-link">5</a>B). Immune-related molecules such as PDL1, PD1, LAG3, LGALS3, CLEC4G, TIGIT, PVR, B7H3, TIM3, HVEM, BTLA, B2M, TAP1, and TAP2 showed significantly higher expression in gene subtype A compared to gene subtype B (Fig. <a href="#Fig5" class="usa-link">5</a>C, Fig. S5). According to the CIBERSORT algorithm, stromal scores, immune scores, and estimate scores were significantly higher in gene subtype A compared to gene subtype B (Fig. <a href="#Fig5" class="usa-link">5</a>D).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Construction and validation of prognostic CMGs risk scores</h3>
<p id="Par36">To explore the relationship between CMGs and their prognostic value in CRC, we utilized 152 prognostic DEGs to construct a prognosis-related model. Initially, CRC patients were divided into a training set (<em>n</em> = 962) and a test set (<em>n</em> = 411) at a ratio of 7:3 using the “caret” package. Subsequently, an analysis of the 152 prognostic DEGs was conducted using Lasso regression and multivariate Cox regression in the training set. Following Lasso regression analysis, 21 OS-associated prognostic genes were determined based on the minimum partial likelihood deviance (Fig. S6). Finally, after performing multivariate Cox regression analysis, we identified 11 prognostic genes (MT2A, SLC2A3, FABP4, PXDN, CXCL13, HOXC6, SCG2, THBS4, CRYAB, SPARCL1, and CCDC80), consisting of seven high-risk genes (MT2A, SLC2A3, FABP4, HOXC6, SCG2, CRYAB, and SPARCL1) and four low-risk genes (PXDN, CXCL13, THBS4, and CCDC80) (Fig. <a href="#Fig6" class="usa-link">6</a>A). According to the results of the multivariate Cox regression analysis, the CMGs risk score was constructed as follows: CMGs risk score = (0.1437023 *expression of MT2A) + (0.238056 *expression of SLC2A3) + (0.1567226 *expression of FABP4) + (−0.26443 *expression of PXDN) + (−0.1650416 *expression of CXCL13) + (0.1692485 *expression of HOXC6) + (0.2083284 *expression of SCG2) + (−0.198839 *expression of THBS4) + (0.1904191 *expression of CRYAB) + (0.2497736 *expression of SPARCL1) + (−0.2458789 *expression of CCDC80).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12364797_12672_2025_3183_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9a06/12364797/685c9a1f514a/12672_2025_3183_Fig6_HTML.jpg" loading="lazy" id="d33e761" height="733" width="709" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Construction of CMGs risk score and classification in the high- and low-risk groups in the training set. (<strong>A</strong>) Forest plot exhibiting the 11 prognostic DEGs identified by multivariate Cox regression analyses (p.val &lt; 0.05). (<strong>B</strong>) Kaplan-Meier curve for patients in the high- and low-risk groups. (<strong>C</strong>) Time-dependent ROC analysis for the CMGs risk score. (<strong>D</strong>) mRNA Expression of SLC2A3 in tumor and non-tumor tissues of CRC patients. (<strong>E</strong>) mRNA Expression of HOXC6 in tumor and non-tumor tissues of CRC patients. (<strong>F</strong>) mRNA Expression of CXCL13 in tumor and non-tumor tissues of CRC patients. (<strong>G</strong>) Representative images of the SLC2A3 in tumor and non-tumor tissues of CRC patients (Scale bars indicate 100 μm). (<strong>H</strong>) Representative images of the HOXC6 in tumor and non-tumor tissues of CRC patients (Scale bars indicate 100 μm). CMGs, copper metabolism-related gene; DEG, differentially expressed gene; ROC, receiver operating characteristic; CRC, colorectal cancer</p></figcaption></figure><p id="Par38">Through the CMGs risk score model, patients within the training set were stratified into high and low-risk groups based on the median CMGs risk score. Kaplan-Meier analysis demonstrated that patients in the high-risk group exhibited lower OS time compared to those in the low-risk group (log-rank test, <em>p</em> &lt; 0.001; Fig. <a href="#Fig6" class="usa-link">6</a>B). Furthermore, the 1-, 3-, 5-, and 10-year survival rates associated with the CMGs risk score were indicated by Area Under the Curve (AUC) values of 0.712, 0.681, 0.658, and 0.698, respectively (Fig. <a href="#Fig6" class="usa-link">6</a>C). To validate the prognostic performance of the CMGs risk score, we calculated CMGs risk scores across the test set and validation set (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87211" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87211</a>). Based on the median CMGs risk score of the training set, both datasets were stratified into high and low-risk groups. The ROC and Kaplan-Meier survival analysis results were consistent with the results observed in the training set (Fig. S7A-S7D). Besides that, we extracted total mRNA from tumor tissues and adjacent normal tissues from 24 clinical samples and detected the mRNA expression levels of SLC2A3, HOXC6, and CXCL13 by using RT-qPCR. The results showed that SLC2A3 and HOXC6 were highly expressed in tumor tissues (Wilcoxon test, <em>p</em> &lt; 0.05, Fig. <a href="#Fig6" class="usa-link">6</a>D and E), while the expression of CXCL13 showed no significant difference between tumor and adjacent normal tissues (Fig. <a href="#Fig6" class="usa-link">6</a>F). Immunohistochemistry (IHC) was performed to assess the expression of SLC2A3 and HOXC6 in paired tumor and adjacent normal tissues from 24 clinical specimens. Both SLC2A3 and HOXC6 demonstrated significantly higher expression in tumor tissues compared to adjacent normal tissues (Fig. <a href="#Fig6" class="usa-link">6</a>G and H, <em>p</em> &lt; 0.05). Due to sample limitations, only three genes were validated by RT-qPCR, further validation in larger patient cohorts is warranted in future studies.</p>
<p id="Par39">The CMGs risk score significantly varied across different subtypes. Figure <a href="#Fig7" class="usa-link">7</a>A illustrates the distribution of patients based on CMGs subtypes, gene subtypes, CMGs risk groups, and survival outcomes. Among the CMGs subtypes, subtype A displayed the highest CMGs risk score, followed by subtype B, with subtype C exhibiting the lowest score (<em>p</em> &lt; 0.001; Fig. <a href="#Fig7" class="usa-link">7</a>B). Similarly, within the gene subtypes, gene subtype A had a significantly higher CMGs risk score than gene subtype B (<em>p</em> &lt; 0.001; Fig. <a href="#Fig7" class="usa-link">7</a>C). The rank dot plot revealed the distribution of patients with ranked CMGs risk scores (Fig. <a href="#Fig7" class="usa-link">7</a>D). Patients with higher CMGs risk scores had lower survival time and a higher probability of unfavorable survival outcomes (Fig. <a href="#Fig7" class="usa-link">7</a>E and F). We also analyzed the differences in clinical characteristics between the high and low-risk groups. There were no differences in age, gender, KRAS mutation, or TP53 mutation between the high and low-risk groups. However, the percentage of stage 3–4 and BRAF mutation was significantly higher in the high-risk group compared to the low-risk group (chi-square test, <em>p</em> &lt; 0.001; Fig. S8A-S8F). Subsequently, stratified Kaplan-Meier survival analysis was conducted based on clinical characteristics (Fig. S8G-S8K). The results indicated that the high-risk group with age &gt; 60 years had the lowest survival time, followed by the high-risk group with age &lt; 60 years, the low-risk group with age &gt; 60 years, and the low-risk group with age &lt; 60 years (log-rank test, <em>p</em> &lt; 0.001; Fig. S8G). Additionally, the survival time of the high-risk group with stage 3–4 was significantly shorter than that of the high-risk group with stage 1–2, the low-risk group with stage 3–4, and the low-risk group with stage 1–2 (log-rank test, <em>p</em> &lt; 0.001; Fig. S8I). By performing univariate and multivariate Cox regression analyses of patients’ age, gender, TMN stage, and CMGs risk score, we found that age, TNM stage, and CMGs risk score were independent risk factors (<em>p</em> &lt; 0.001, Fig. S9).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12364797_12672_2025_3183_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9a06/12364797/ee7419439516/12672_2025_3183_Fig7_HTML.jpg" loading="lazy" id="d33e854" height="463" width="709" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Alluvial diagram illustrating the distributions of CRC patients in CMGs subtypes, gene subtypes, CMGs risk groups, and survival outcomes. (<strong>B</strong>) CMGs risk scores in different CMGs subtypes. (<strong>C</strong>) CMGs risk scores in different gene subtypes. (<strong>D</strong>) Distribution of CMR risk score in different risk groups. (<strong>E</strong>) Correlation between CMGs risk groups and survival outcomes. (<strong>F</strong>) Differences in CMGs risk scores with different survival outcomes. CMGs, copper metabolism-related gene; CRC, colorectal cancer</p></figcaption></figure></section><section id="Sec23"><h3 class="pmc_sec_title">Assessment of TME and immune checkpoints between the high-risk group and low-risk group</h3>
<p id="Par41">To analyze the relationship between CMGs risk score and tumor immunity, we utilized the CIBERSORT algorithm to assess the correlation between CMGs risk score and the abundance of immune cells in each sample. The high-risk group positively correlated with the infiltration of resting CD4<sup>+</sup> T cells, gamma delta T cells, activated mast cells, monocytes, M0 macrophages, M2 macrophages, and neutrophils (Fig. <a href="#Fig8" class="usa-link">8</a>A). Moreover, the high-risk group demonstrated higher stromal score and immune score in comparison to the low-risk group (Fig. <a href="#Fig8" class="usa-link">8</a>B). Continuing our exploration into the influence of CMGs risk score on tumor immunity, we performed a Spearman correlation analysis between the 11 prognostic genes and immune cell infiltration abundance, uncovering notable correlations between these genes and most immune cell abundances (Fig. <a href="#Fig8" class="usa-link">8</a>C), especially with M0 macrophages, M1 macrophages, and neutrophils. Particularly noteworthy was the significant positive correlation between CXCL13 and CD8<sup>+</sup> T cells, follicular helper T cells, and M1 macrophages. Moreover, the expression of HAVCR2 (TIM3), LAIR1, CD274 (PDL1), CD276 (B7H3), SELPLG, TNFSF9, NRP1, SIGLEC15, TNFSF4, CD86, and COLEC12 were higher in the high-risk group, whereas CTLA4, TNFRSF14, LGALS9, PDCD1, HMGB1, and PVR exhibited lower expression (Wilcoxon test, <em>p</em> &lt; 0.05; Fig. <a href="#Fig8" class="usa-link">8</a>D).</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12364797_12672_2025_3183_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9a06/12364797/bc6a1edcb5ba/12672_2025_3183_Fig8_HTML.jpg" loading="lazy" id="d33e901" height="1047" width="709" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Evaluation of TME in high- and low-risk groups. (<strong>A</strong>) Correlation between CMGs risk score and abundance of infiltrating immune cells. (<strong>B</strong>) TME score in high- and low-risk groups. (<strong>C</strong>) Correlations between the abundance of infiltrating immune cells and eleven genes in the proposed model. (<strong>D</strong>) Expression of immune checkpoints in the high and low-risk groups. TME, tumor microenvironment</p></figcaption></figure></section><section id="Sec24"><h3 class="pmc_sec_title">The relationship between CMGs risk score and MSI and CSC index</h3>
<p id="Par43">Tumors with higher MSI tend to generate neoantigens, leading to increased infiltration of immune cells, thereby making them more likely to benefit from immunotherapy and suppress tumor progression [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. We designated MANTIS scores greater than 0.4 as MSI-H (high MSI), while those less than or equal to 0.4 were classified as MSI-L/MSS (low or microsatellite stable) [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. The results indicated a significantly higher proportion of MSI-H in the high-risk group compared to the low-risk group (chi-square test, <em>p</em> &lt; 0.05; Fig. <a href="#Fig9" class="usa-link">9</a>A). Additionally, the CMGs risk score in the MSI-H group was markedly higher than that in MSI-L/MSS (<em>p</em> &lt; 0.001, Fig. <a href="#Fig9" class="usa-link">9</a>B). Patients were stratified into four groups based on CMGs risk group and MSI, and Kaplan-Meier analysis revealed significantly lower OS times in the high-risk group compared to the low-risk group, with the lowest survival time observed in the high-risk + MSS-H subgroup (Fig. <a href="#Fig9" class="usa-link">9</a>C). However, a notable negative correlation was observed between CMGs risk score and CSC index (<em>r</em> = −0.35, <em>p</em> &lt; 0.001), indicating a higher presence of cancer stem cell characteristics in tumors with low CMGs scores and a lower degree of differentiation (Fig. <a href="#Fig9" class="usa-link">9</a>D).</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12364797_12672_2025_3183_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9a06/12364797/0273d101a64e/12672_2025_3183_Fig9_HTML.jpg" loading="lazy" id="d33e978" height="1002" width="709" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comprehensive analysis of the CMGs risk score in CRC. (<strong>A</strong>) Compositions of MSI status in the two groups. (<strong>B</strong>) CMGs risk scores in different MSI statuses. <strong>(C</strong>) Stratified Kaplan Meier analysis for CRC patients with different CMGs risk scores and MSI statuses. (<strong>D</strong>) Correlation between CSC index and CMGs risk score. (<strong>E</strong>) TMB in different risk groups. (<strong>F</strong>) Spearman correlation analysis of the CMGs risk score and TMB. (<strong>G</strong>) Top 10 genes with mutation frequency in high-risk group. (<strong>H</strong>) Top 10 genes with mutation frequency in the low-risk group. (<strong>I</strong>) Top 12 significant mutation genes in the different risk groups. (<strong>J-L</strong>) Chemotherapeutic sensitivity of anti-tumor drugs in different risk groups. CMGs, copper metabolism-related gene; MSI, microsatellite instability; CSC, cancer stem cell; TMB, tumor mutation burden</p></figcaption></figure></section><section id="Sec25"><h3 class="pmc_sec_title">Mutation and drug susceptibility analysis</h3>
<p id="Par45">TMB serves as a crucial biomarker for evaluating the potential benefit of immunotherapy, with higher TMB associated with increased neoantigen production, thus affecting the efficacy of immunotherapy [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. We observed a significant positive correlation between CMGs risk score and TMB (Fig. <a href="#Fig9" class="usa-link">9</a>E and F). Further analysis of somatic mutation data in tumors from the high and low-risk groups revealed significant disparities in mutated genes between the two cohorts. Firstly, among the top 10 mutated genes in both groups, TP53, APC, TTN, KRAS, SYNE1, MUC16, FAT4, and DNAH5 were commonly observed, while OBSCN and ZFHX4 were more prevalent in the high-risk group, and the low-risk group exhibited more mutations in PIK3CA and DNAH11 (Fig. <a href="#Fig9" class="usa-link">9</a>G and H). Secondly, the number of mutations in genes such as KIAA1671, TBC1D9, BRAF, SRCAP, UNC6B, PCNXL3, WDR33, IGSF5, LRRTM1, and SAFB was significantly higher in the high-risk group compared to the low-risk group, whereas mutations in APC and NPAP1 were more prevalent in the low-risk group (Fig. <a href="#Fig9" class="usa-link">9</a>I). These results suggest that tumors in the high-risk group have a significantly higher TMB and somatic mutation rate, indicating a potential for better response to immunotherapy. Subsequently, we analyzed the relationship between CMGs risk score and IC50 of anti-tumor drugs to identify drug susceptibility in tumors across different risk groups. The high-risk group showed significantly lower IC50 for Elesclomol (Fig. <a href="#Fig9" class="usa-link">9</a>J, <em>p</em> &lt; 0.001), but higher IC50 for Rapamycin and Doxorubicin (Fig. <a href="#Fig9" class="usa-link">9</a>K and L, <em>p</em> &lt; 0.001). These findings suggest that tumors with different CMGs risk scores exhibit varying susceptibility to different drugs.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Development and assessment of nomogram for survival prediction</h3>
<p id="Par46">To better utilize the CMGs risk score in guiding clinical treatment, we integrated it with the TNM stage and developed a nomogram to predict the survival probability at 1-, 3-, 5-, and 10-years (Fig. <a href="#Fig10" class="usa-link">10</a>A). Initially, we evaluated the AUC curves for 1-, 3-, 5-, and 10-years using training, testing, internal validation (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40967" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE40967</a>), and external validation (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87211" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87211</a>) datasets (Fig. <a href="#Fig10" class="usa-link">10</a>B and E). Compared to utilizing TNM stage or age alone, as well as their combination as a predictive indicator, the combined model demonstrated superior performance (Fig. S10A-S10L). The AUC values for the nomogram in the training set were 0.769, 0.745, 0.730, and 0.799 for 1-, 3-, 5-, and 10-years, respectively. In comparison, the AUC values for both age and TNM stage were 0.691, 0.671, 0.670, 0.731 (Fig. S10C). In the external validation set, the AUC values for the nomogram were 0.731 0.705, 0.726 and 0.582 for 1-, 3-, 5- and 10-years respectively, while for both age and TNM stage, the AUC values were 0.649, 0.672, 0.573, 0.530 (Fig. S10L). Compared to the ideal model, the multiple calibration curves for the training, testing, internal validation, and external validation sets showed good performance (Fig. <a href="#Fig10" class="usa-link">10</a>F and I).</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12364797_12672_2025_3183_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9a06/12364797/ebd00ba8b8e9/12672_2025_3183_Fig10_HTML.jpg" loading="lazy" id="d33e1040" height="960" width="709" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Construction and validation of a nomogram. (<strong>A</strong>) Nomogram for predicting the 1-, 3-, 5-, and 10-year OS of CRC patients in the training set. (<strong>B-E</strong>) Time-dependent ROC analysis of the nomogram in the training, testing, internal validation (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40967" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE40967</a>), and external validation (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87211" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87211</a>) sets. (<strong>F-I</strong>) Calibration curves for evaluating the agreement in the predicted and actual OS in the training, testing, internal validation (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40967" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE40967</a>), and external validation (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87211" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87211</a>) sets. OS, overall survival; CRC, colorectal cancer; ROC, receiver operating characteristic</p></figcaption></figure></section></section><section id="Sec27"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par48">Copper is an important trace element in the human body. Due to the redox properties of copper, which exist in the form of 1-valent and 2-valent copper, it can act as an electron transfer carrier and catalytic factor for enzymes, and participate in a variety of reactions in the body, such as energy metabolism, mitochondrial respiration, antioxidant, collagen cross-linking, pigmentation, and catecholamine biosynthesis [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Dysregulation of copper metabolism can lead to copper overloaded or copper deficiency, causing damage and dysfunction in tissues and organs</p>
<p id="Par49">Beyond its physiological roles, copper plays an important role in tumor progression. Numerous studies have demonstrated elevated copper levels in the serum and tumor tissues of patients with various cancers, including colorectal cancer (CRC), breast cancer, lung cancer, and ovarian cancer, compared to normal tissues [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Notably, elevated serum copper was observed to correlate with advanced TNM stage and progression of CRC and breast cancer [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. The concentration of copper in CRC was also determined to be higher than in normal tissues, while the concentration of zinc decreases in progressive CRC [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Thus, the Cu/Zn ratio emerges as a crucial prognostic indicator for CRC. Experimental studies have discovered that copper can regulate tumor progression through various pathways, including proliferation, metastasis, angiogenesis, and energy metabolism [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Despite studies confirming the role of copper in maintaining immune regulation, its direct effects on tumor immunity remain underexplored [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Recent research found that copper can promote the expression of PDL1 in tumors to induce immune evasion [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. This research provides a foundation and direction for exploring the correlation between copper and tumor immunity, as well as for developing new drugs to enhance immunotherapeutic responsiveness. Following the discovery of cuproptosis [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], numerous studies have focused on elucidating the prognostic implications of cuproptosis-associated genes and their modulatory roles within the tumor microenvironment [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. However, these researchers have not interpreted the findings from the perspective of copper metabolism in tumors. In our study, we meticulously scrutinized the expression patterns of 112 CMGs across CRC cohorts. Based on the expression profiles of CMGs, CRC patients were methodically stratified into three discernible CMGs subtypes. Subsequent identified the prognostic DEGs between CMGs subtypes, we further classified CRC into gene subtypes A and B and elucidated substantial variations. Additionally, a CMGs-risk scoring model was constructed based on prognostic DEGs after affirming its robust predictive efficacy. By utilizing this model, CRC patients were dichotomized into categories of high-risk group or low-risk group. Patients in different groups showed significantly distinct prognoses, clinicopathological characteristics, TME, immune checkpoint expression, mutation profiles, MSI, CSC index, and drug susceptibility. Finally, by integrating the CMGs risk score, age and tumor stage, we constructed a quantitative nomogram, further enhancing the performance of the CMGs risk score and facilitating its clinical application. This comprehensive analysis has revealed the profound regulatory influence of copper metabolism on biological pathways, TME formation, and drug susceptibility in CRC, providing a novel perspective and groundwork for understanding the interplay between copper metabolism and tumor immunity.</p>
<p id="Par50">The level of active copper metabolism correlates with the progression of CRC. In this study, CRC patients were categorized into three different CMGs subtypes based on the expression of CMGs (Fig. <a href="#Fig2" class="usa-link">2</a>C-D). Most CMGs exhibited the highest expression in CMGs subtype A, yet the OS time was the shortest among the three subtypes (Fig. <a href="#Fig2" class="usa-link">2</a>E). CMGs subtype B demonstrated higher expression compared to CMGs subtype C, and the OS time of CMGs subtype B was shorter than that of CMGs subtype C (Fig. <a href="#Fig2" class="usa-link">2</a>E). Similarly, the OS time of gene subtype A with high expression of DEGs was also significantly shorter than that of gene subtype B with low expression of DEGs (Fig. <a href="#Fig4" class="usa-link">4</a>F). Using prognostic DEGs, we constructed a CMGs risk score with robust clinical predictive efficacy. The CMGs risk score was consistent with the distribution trends of CMGs subtypes and gene subtypes (Fig. <a href="#Fig7" class="usa-link">7</a>B-C), manifesting a positive correlation with the levels of activation in copper metabolism. Kaplan-Meier analysis revealed that patients with high CMGs risk scores had significantly worse OS than those with low scores (Fig. <a href="#Fig6" class="usa-link">6</a>B), and higher TNM stages were significantly associated with elevated CMGs scores (Fig. S8C). Notably, patients with both a high CMGs score and advanced TNM stage (Stage 3–4) had the worst survival outcomes (Fig. S8I), consistent with previous findings that active copper metabolism is a poor prognostic marker in CRC.</p>
<p id="Par51">CRCs with varying levels of copper metabolism displayed distinct molecular functions and metabolic pathways. GSVA analysis revealed that CMGs subtype A had elevated activity in pathways associated with extracellular matrix (ECM) remodeling, such as glycosaminoglycan biosynthesis, focal adhesion, ECM-receptor interaction, and cell adhesion molecules (Fig. <a href="#Fig3" class="usa-link">3</a>A). Specifically, glycosaminoglycan biosynthesis chondroitin sulfate, focal adhesion, ECM receptor interaction, and cell adhesion molecules were involved in the activity of ECM. Similar results were observed for GO and KEGG enrichment results for DEGs of CMGs subtypes (Fig. <a href="#Fig4" class="usa-link">4</a>A-D). The ECM is composed of fibrous proteins (e.g., collagen, elastin, fibronectin) and glycosaminoglycans (e.g., hyaluronic acid, chondroitin sulfate) [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. It plays a key role in tumor proliferation, invasion, angiogenesis, and immune regulation [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>, <a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Previous studies have identified that copper promotes the remodeling of ECM by lysyl oxidase (LOX) and lysyl oxidase-like proteins (LOXL). The LOX and LOXL families are recognized as copper-dependent enzymes that facilitate the crosslinking of collagen and elastin, thereby promoting tissue stiffness [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. In addition to serving as a catalytic cofactor for the LOX and LOXL families, copper also regulates the hypoxic environment in TME by promoting the expression of LOX family proteins through the activation of HIF-1α, such as LOX, LOXL2, and LOXL4 [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. It was found that LOX promoted the expression of HIF-1α, thereby establishing a positive feedback loop that facilitated tumor development [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. Researchers have demonstrated that LOX is associated with the progression of CRC, contributing to invasion and metastasis. IHC demonstrated that the expression of ECM and LOX were positively correlated with the TNM stage of CRC [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. RT-PCR analysis of tumor samples revealed that the expression and nuclear translocation of LOX were associated with the invasion and metastasis of CRC [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>, <a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. LOX and LOXL facilitate the invasion and metastasis of CRC through various pathways involving IL-6, FAK/SRC, and EMT [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>–<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>].</p>
<p id="Par52">Another important copper-related regulator of ECM remodeling is the COMMD family (Copper Metabolism MURR1 Domain-containing), which maintains copper homeostasis [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. COMMD1 deficiency leads to intracellular copper accumulation [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] and reduced ATP7B expression, limiting copper efflux [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. CTR1 knockdown has also been shown to decrease COMMD1, XIAP and NF-κB levels [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Recently, Hancock et al. found that loss of COMMD3 increases copper burden in tumors and enhances the ability of tumors to remodel the ECM, thereby promoting tumor growth and invasion [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. In our study, we observed that the proportion of stage 3–4 patients in the high-risk group was significantly higher than that in the low-risk group and the OS time was shortest for patients with Stage 3–4 in the high-CMGs group (Fig. S8C, Fig. S8I). In summary, these results suggest that active copper metabolism plays an important role in regulating invasion and metastasis in CRC by remodeling the ECM.</p>
<p id="Par53">Active copper metabolism promotes the formation of immunosuppressive TME in CRC. Utilizing the ESTIMATE package, we discovered that CMGs subtypes A had significantly higher stromal and immunity scores than the other subtypes (Fig. <a href="#Fig3" class="usa-link">3</a>E). Despite elevated immune cell infiltration, this group also showed increased expression of immune checkpoints including PDL1, PD1, B7H3, TIGIT, TIM3, and LAG3 (Fig. <a href="#Fig3" class="usa-link">3</a>C-D, Fig. <a href="#MOESM2" class="usa-link">S2</a>). The gene subtypes and CMGs risk groups showed a similar immune landscape (Fig. <a href="#Fig5" class="usa-link">5</a>C-D, Fig. S5, Fig. <a href="#Fig7" class="usa-link">7</a>B, and Fig. <a href="#Fig7" class="usa-link">7</a>D). These collective results suggested that the majority of infiltrating immune cells in CMGs subtype A, gene subtype A, and the high-risk group could be in a functionally inhibitory state. Voli et al. discovered that copper promoted the expression of PDL1 in tumors to induce immune evasion [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. This finding elucidates one of the underlying mechanisms contributing to the high expression of immune checkpoints in CRC with active copper metabolism. TME is classified into three main types: Infiltrated-excluded, Infiltrated-inflamed, and Infiltrated-TLS [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. Infiltrated-inflamed TME is characterized by abundant infiltrating lymphocytes expressing PD1 and tumor cells expressing PDL1. While these tumors have high immunogenicity, the functionality of immune cells is diminished, and they are termed “Hot Tumors”. The TME of CMGs subtype A, gene subtype A, and high-risk group resembles the Infiltrated-inflamed TME, implying that these types of tumors have good responsiveness to PD1/PDL1 or other immune checkpoint inhibitors (ICIs). Therefore, drugs targeting copper metabolism may have the potential to become sensitizers for immunotherapy.</p>
<p id="Par54">Besides that, we also observed a significantly positive correlation between the CMGs risk score and the proportion of M0 macrophages, M2 macrophages, and neutrophils. In contrast, there was a negative correlation with the proportion of M1 macrophages, CD8<sup>+</sup> T cells, and activated NK cells (Fig. <a href="#Fig7" class="usa-link">7</a>A). Tumor-associated macrophages (TAMs) are a group of immune cells with phagocytic, antigen-presenting, and cytokine-secreting [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. M1 macrophages exert anti-tumor effects, whereas M2 macrophages promote tumor growth and immunosuppression [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>–<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. Our results suggest that active copper metabolism skews macrophage polarization toward the M2 phenotype, contributing to poor prognosis. Previous studies have shown that copper metabolism regulates the polarization of TAMs in direct or indirect manners. Copper directly promotes M2 polarization by increasing oxidative stress, modulating glutathione synthesis, and inhibiting the arginase pathway in Macrophages [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>–<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. Secondly, Copper can upregulate intratumoral ROS levels, which indirectly serve as crucial factors in the activation and functionality of M2 macrophages [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>, <a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. However, studies based on the crosstalk between copper metabolism in tumors and TAMs are relatively insufficient, and further studies are needed to explore the underlying mechanisms. The effectiveness of anti-PD1/PDL1 therapies was initially believed to be primarily attributed to their regulation of T cells. However, in recent years, it has been revealed that TAMs are one of the targets for ICIs. TAMs expressing PDL1 exhibited a positive correlation with survival outcomes and emerged as independent prognostic factor in cancer [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. CRC promotes M2 polarization and upregulates PDL1 expression by secreting miR-21-5p and miR-200a [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. Conversely, the blockade of PDL1 promotes the proliferation of TAMs and upregulates multiple macrophage inflammatory pathways [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. Gordon et al. demonstrated in vivo that blocking the PD1/PDL1 increased phagocytosis of macrophages, consequently suppressing tumor growth [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. The combination of anti-PDL1 with PD1-IL2v has been observed to prolong the response phase, enhancing therapeutic efficacy by the reprogramming of immunosuppressive TAMs [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. We found that PD1 and PDL1 were highly expressed in CMGs subtype A and gene subtype A. PDL1 was significantly upregulated in the high-risk group and was accompanied by a high proportion of M2 macrophage infiltration. We hypothesize that M2 macrophages with high expression of PD1/PDL1 were present within the TME of CRC with active copper metabolism, and that treatment with anti-PD1/PDL1 may exert its effect through these immune cells.</p>
<p id="Par55">In addition to TAMs, neutrophils are also immune cells worthy of our attention. In our study, we observed a positive correlation between active copper metabolism (CMGs subtype A, gene subtype A, and high CMGs risk group) and the presence of neutrophils in TME. Blood neutrophils serve a critical function within the immune system, defending against pathogens, orchestrating inflammation, and impacting a range of immune and non-immune processes [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. However, Tumor-associated neutrophils (TANs) with distinctly different functions are widely believed to promote tumor progression, such as tumorigenesis, proliferation, metastasis, angiogenesis, formation of neutrophil extracellular traps (NETs), and immunosuppression [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>, <a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. TANs inhibit CD8⁺ T cell proliferation and IFN-γ production, as well as induce T cell apoptosis [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>, <a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. Gr1<sup>+</sup> neutrophils were reported to reduce T cell homing and alter angiogenesis, consequently inducing hypoxia and sustained Snail expression to promote tumor proliferation and metastasis [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. Moreover, NETs have emerged in recent years as crucial molecules discovered to play a significant role in facilitating tumor progression by TANs. NETs are adhesive, mesh-like structures secreted by neutrophils during the defense against external pathogens, serving the function of capturing and eliminating extracellular pathogens [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. However, researchers have found that NETs promote tumor proliferation, migration, and metastasis through various mechanisms [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>, <a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. NETs activate the Toll-like receptor 9 (TLR9) pathway and facilitate the process of EMT to promote the migration and invasion of tumors [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>–<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. Furthermore, NETs can create an immunosuppressive TME by inhibiting the recruitment and activation of T cells and NK cells, and promote the differentiation of regulatory Tregs [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. Additionally, TANs share similar functionalities and expression profiles with polymorphonuclear myeloid-derived suppressor cells (PMN-MDSCs), collectively exerting immunosuppressive effects [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. Nevertheless, there is still a notable scarcity of investigations regarding the connection between copper metabolism genes and the infiltration and activation of tumor-associated neutrophils (TANs). Further studies are needed to elucidate and validate these findings. In conclusion, CRC with active copper metabolism may establish an immunosuppressive TME by upregulating the expression of immune checkpoints, promoting the M2 polarization, increasing the infiltration of TANs or MDSCs and decreasing the infiltration of CD8<sup>+</sup> T cells.</p>
<p id="Par56">Copper metabolism is also associated with microsatellite instability (MSI), which is driven by defects in the mismatch repair (MMR) system [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Consequently, a deficient MMR (d-MMR) system is prone to MSI-H. Approximately 15% of CRCs exhibit MSI-high (MSI-H) status, which is linked to increased immunogenicity and favorable responses to immune checkpoint inhibitors (ICIs) [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>, <a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. Due to the favorable response to ICIs therapy, d-MMR and MSI-H are considered biomarkers for a favorable prognosis in CRC. We found that high CMGs risk scores correlated with MSI-H status (Fig. <a href="#Fig8" class="usa-link">8</a>A-B). This finding aligns with the higher expression of PD1 and PDL1 observed in the high- risk group. Stratified analysis revealed that low-risk MSS/MSI-L patients exhibited the most favorable prognosis, while high-risk MSI-H patients had the poorest outcomes (Fig. <a href="#Fig8" class="usa-link">8</a>C). Therefore, as previously stated, we believe that ICIs have the potential to significantly improve survival rates in the high-risk group. In addition, we found that CRCs with different CMGs risk scores had different susceptibility to antitumor drugs (Fig. <a href="#Fig9" class="usa-link">9</a>J-L). Additionally, drug sensitivity analysis showed that CRCs with high CMGs scores were more responsive to Elesclomol—a copper ionophore that induces copper-mediated oxidative stress and cell death [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>, <a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. Thus, CRC with active copper metabolism can potentially benefit from the use of copper ionophores to enhance anti-tumor effects. Although our model predicts immunotherapy responsiveness based on immune checkpoint expression and TMB/MSI, the absence of datasets from patients who have received immunotherapy limits direct validation of its predictive accuracy. However, the combination therapy of copper-modulating agents and ICIs for CRC holds crucial research significance, requiring extensive exploration in this domain.</p>
<p id="Par57">This study has several limitations. Firstly, all analyses are based on retrospective data from public databases, and all samples used in our study are retrospective. Therefore, inherent case selection bias may have influenced the results. Large-scale prospective studies and additional in vitro and in vivo experiments are needed to confirm our findings. Additionally, some crucial clinical variables, such as surgical, neoadjuvant chemotherapy, and radiotherapy data, were unavailable for analysis in most datasets, potentially impacting immune response and the status of copper metabolism. Although a nomogram was constructed based on the risk score and clinical characteristics of CRC patients, the current method for obtaining an accurate risk score remains costly and technically challenging. Additionally, the cutoff used to distinguish high- and low-risk groups was based on the median value of the CMGs risk score in the training cohort. Therefore, further optimization of the model is necessary to enhance its clinical feasibility and applicability. Future studies should explore clinically actionable thresholds through prospective validation.</p></section><section id="Sec28"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par58">We innovatively unveiled the heterogeneity in copper metabolism within CRC through the comprehensive analysis of multiple cohort data based on currently known CMGs. This analysis allowed us to investigate the variation in physiological pathways, TME, clinicopathological characteristics, and drug susceptibility across CRC with different statuses of copper metabolism. This study emphasized the CMGs as promising biological targets for antitumor therapy and immunotherapy, thereby laying the groundwork for the development of relevant targeted drugs or combination therapies. Additionally, we established a predictive model based on CMGs, which can be utilized for the evaluation and personalized treatment of CRC patients, ultimately enhancing the efficiency of clinical treatment.</p></section><section id="Sec29"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12364797/bin/12672_2025_3183_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (353.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12364797/bin/12672_2025_3183_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (30.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12364797/bin/12672_2025_3183_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (1.1MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Foshan Gastrointestinal Tumor Diagnosis and Treatment Engineering Technology Research Center, Foshan City’s 14th Five-Year Key Specialty Project. Funded by the key medical talents training project of Shunde District, Foshan.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Manzhao Ouyang is responsible for the concept and design of this study. Geng Peng, Lin Zhong and Nan Lai drafted the article and revised it critically for important intellectual content. Geng Peng, Lin Zhong, and Lina Luo retrieved the data and revised the article. Geng Peng, Nan Lai and Fu Cheng participated in manuscript writing and editing. These authors contributed equally to this work. The final manuscript was read and approved by all authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was funded by the 2022 Guangdong Province Basic and Applied Basic Research Fund Project (No. 2022A1515012315), Medical Science and Technology Research Fund Project of Guangdong Province (A2023185), Southern Medical University Shunde Hospital 2022 Research Initiation Programme Project.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data presented in the study are presented in the manuscript and Additional files. Further inquiries of datasets are available directed to the corresponding author.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par59">The study was approved by the Ethics Committee of Shunde Hospital of Southern Medical University and was performed in accordance with the principles of the Declaration of Helsinki. All patients signed informed consent forms before the study.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par60">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Geng Peng, Lin Zhong and Nan Lai contributed equally to this work.</p></div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Siegel RL, Miller KD, Wagle NS, Jemal A, Cancer statistics. 2023. Ca-Cancer J Clin. 2023;73(1):17–48. 10.3322/caac.21763</cite> [<a href="https://doi.org/10.3322/caac.21763" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36633525/" class="usa-link">PubMed</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Dekker E, Tanis PJ, Vleugels J, Kasi PM, Wallace MB. Colorectal cancer. Lancet. 2019;394(10207):1467–80. 10.1016/S0140-6736(19)32319-0.
</cite> [<a href="https://doi.org/10.1016/S0140-6736(19)32319-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31631858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dekker%20E,%20Tanis%20PJ,%20Vleugels%20J,%20Kasi%20PM,%20Wallace%20MB.%20Colorectal%20cancer.%20Lancet.%202019;394(10207):1467%E2%80%9380.%2010.1016/S0140-6736(19)32319-0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Biller LH, Schrag D. Diagnosis and treatment of metastatic colorectal cancer: A review. Jama-J Am Med Assoc. 2021;325(7):669–85. 10.1001/jama.2021.0106.</cite> [<a href="https://doi.org/10.1001/jama.2021.0106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33591350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Biller%20LH,%20Schrag%20D.%20Diagnosis%20and%20treatment%20of%20metastatic%20colorectal%20cancer:%20A%20review.%20Jama-J%20Am%20Med%20Assoc.%202021;325(7):669%E2%80%9385.%2010.1001/jama.2021.0106." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Johdi NA, Sukor NF. Colorectal Cancer immunotherapy: options and strategies. Front Immunol. 2020;11:1624. 10.3389/fimmu.2020.01624.
</cite> [<a href="https://doi.org/10.3389/fimmu.2020.01624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7530194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33042104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johdi%20NA,%20Sukor%20NF.%20Colorectal%20Cancer%20immunotherapy:%20options%20and%20strategies.%20Front%20Immunol.%202020;11:1624.%2010.3389/fimmu.2020.01624." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Hanahan D. Hallmarks of cancer: new dimensions. Cancer Discov. 2022;12(1):31–46. 10.1158/2159-8290.CD-21-1059.
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-21-1059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35022204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanahan%20D.%20Hallmarks%20of%20cancer:%20new%20dimensions.%20Cancer%20Discov.%202022;12(1):31%E2%80%9346.%2010.1158/2159-8290.CD-21-1059." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Lv M, Chen M, Zhang R, Zhang W, Wang C, Zhang Y, Wei X, Guan Y, Liu J, Feng K, Jing M, Wang X, Liu YC, Mei Q, Han W, Jiang Z. Manganese is critical for antitumor immune responses via cGAS-STING and improves the efficacy of clinical immunotherapy. Cell Res. 2020;30(11):966–79. 10.1038/s41422-020-00395-4.
</cite> [<a href="https://doi.org/10.1038/s41422-020-00395-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7785004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32839553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lv%20M,%20Chen%20M,%20Zhang%20R,%20Zhang%20W,%20Wang%20C,%20Zhang%20Y,%20Wei%20X,%20Guan%20Y,%20Liu%20J,%20Feng%20K,%20Jing%20M,%20Wang%20X,%20Liu%20YC,%20Mei%20Q,%20Han%20W,%20Jiang%20Z.%20Manganese%20is%20critical%20for%20antitumor%20immune%20responses%20via%20cGAS-STING%20and%20improves%20the%20efficacy%20of%20clinical%20immunotherapy.%20Cell%20Res.%202020;30(11):966%E2%80%9379.%2010.1038/s41422-020-00395-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Wang C, Sun Z, Zhao C, Zhang Z, Wang H, Liu Y, Guo Y, Zhang B, Gu L, Yu Y, Hu Y, Wu J. Maintaining manganese in tumor to activate cGAS-STING pathway evokes a robust abscopal anti-tumor effect. J Control Release. 2021;331:480–90. 10.1016/j.jconrel.2021.01.036.
</cite> [<a href="https://doi.org/10.1016/j.jconrel.2021.01.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33545219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20C,%20Sun%20Z,%20Zhao%20C,%20Zhang%20Z,%20Wang%20H,%20Liu%20Y,%20Guo%20Y,%20Zhang%20B,%20Gu%20L,%20Yu%20Y,%20Hu%20Y,%20Wu%20J.%20Maintaining%20manganese%20in%20tumor%20to%20activate%20cGAS-STING%20pathway%20evokes%20a%20robust%20abscopal%20anti-tumor%20effect.%20J%20Control%20Release.%202021;331:480%E2%80%9390.%2010.1016/j.jconrel.2021.01.036." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Martins AJ, Gubert P, Villas BG, Meirelles PM, Santamaria A, Lee E, Tinkov AA, Bowman AB, Aschner M. Manganese-induced neurodegenerative diseases and possible therapeutic approaches. Expert Rev Neurother. 2020;20(11):1109–21. 10.1080/14737175.2020.1807330.
</cite> [<a href="https://doi.org/10.1080/14737175.2020.1807330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7657997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32799578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martins%20AJ,%20Gubert%20P,%20Villas%20BG,%20Meirelles%20PM,%20Santamaria%20A,%20Lee%20E,%20Tinkov%20AA,%20Bowman%20AB,%20Aschner%20M.%20Manganese-induced%20neurodegenerative%20diseases%20and%20possible%20therapeutic%20approaches.%20Expert%20Rev%20Neurother.%202020;20(11):1109%E2%80%9321.%2010.1080/14737175.2020.1807330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Rink L, Kirchner H. Zinc-altered immune function and cytokine production. J Nutr. 2000;130(5S Suppl). 10.1093/jn/130.5.1407S.:1407S-1411S.</cite> [<a href="https://doi.org/10.1093/jn/130.5.1407S" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10801952/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Bafaro E, Liu Y, Xu Y, Dempski RE. The emerging role of zinc transporters in cellular homeostasis and cancer. Signal Transduct Tar. 2017;2:17029. 10.1038/sigtrans.2017.29.</cite> [<a href="https://doi.org/10.1038/sigtrans.2017.29" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5661630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29218234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bafaro%20E,%20Liu%20Y,%20Xu%20Y,%20Dempski%20RE.%20The%20emerging%20role%20of%20zinc%20transporters%20in%20cellular%20homeostasis%20and%20cancer.%20Signal%20Transduct%20Tar.%202017;2:17029.%2010.1038/sigtrans.2017.29." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Hogstrand C, Kille P, Ackland ML, Hiscox S, Taylor KM. A mechanism for epithelial-mesenchymal transition and Anoikis resistance in breast cancer triggered by zinc channel ZIP6 and STAT3 (signal transducer and activator of transcription 3). Biochem J. 2013;455(2):229–37. 10.1042/BJ20130483.
</cite> [<a href="https://doi.org/10.1042/BJ20130483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3789231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23919497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hogstrand%20C,%20Kille%20P,%20Ackland%20ML,%20Hiscox%20S,%20Taylor%20KM.%20A%20mechanism%20for%20epithelial-mesenchymal%20transition%20and%20Anoikis%20resistance%20in%20breast%20cancer%20triggered%20by%20zinc%20channel%20ZIP6%20and%20STAT3%20(signal%20transducer%20and%20activator%20of%20transcription%203).%20Biochem%20J.%202013;455(2):229%E2%80%9337.%2010.1042/BJ20130483." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Silvestri R, Nicoli V, Gangadharannambiar P, Crea F, Bootman MD. Calcium signalling pathways in prostate cancer initiation and progression. Nat Rev Urol. 2023;20(9):524–43. 10.1038/s41585-023-00738-x.
</cite> [<a href="https://doi.org/10.1038/s41585-023-00738-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36964408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Silvestri%20R,%20Nicoli%20V,%20Gangadharannambiar%20P,%20Crea%20F,%20Bootman%20MD.%20Calcium%20signalling%20pathways%20in%20prostate%20cancer%20initiation%20and%20progression.%20Nat%20Rev%20Urol.%202023;20(9):524%E2%80%9343.%2010.1038/s41585-023-00738-x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>So CL, Saunus JM, Roberts-Thomson SJ, Monteith GR. Calcium signalling and breast cancer. Semin Cell Dev Biol. 2019;94:74–83. 10.1016/j.semcdb.2018.11.001.
</cite> [<a href="https://doi.org/10.1016/j.semcdb.2018.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30439562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?So%20CL,%20Saunus%20JM,%20Roberts-Thomson%20SJ,%20Monteith%20GR.%20Calcium%20signalling%20and%20breast%20cancer.%20Semin%20Cell%20Dev%20Biol.%202019;94:74%E2%80%9383.%2010.1016/j.semcdb.2018.11.001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Tsvetkov P, Coy S, Petrova B, Dreishpoon M, Verma A, Abdusamad M, Rossen J, Joesch-Cohen L, Humeidi R, Spangler RD, Eaton JK, Frenkel E, Kocak M, Corsello SM, Lutsenko S, Kanarek N, Santagata S, Golub TR. Copper induces cell death by targeting lipoylated TCA cycle proteins. Science. 2022;375(6586):1254–61. 10.1126/science.abf0529.
</cite> [<a href="https://doi.org/10.1126/science.abf0529" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9273333/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35298263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsvetkov%20P,%20Coy%20S,%20Petrova%20B,%20Dreishpoon%20M,%20Verma%20A,%20Abdusamad%20M,%20Rossen%20J,%20Joesch-Cohen%20L,%20Humeidi%20R,%20Spangler%20RD,%20Eaton%20JK,%20Frenkel%20E,%20Kocak%20M,%20Corsello%20SM,%20Lutsenko%20S,%20Kanarek%20N,%20Santagata%20S,%20Golub%20TR.%20Copper%20induces%20cell%20death%20by%20targeting%20lipoylated%20TCA%20cycle%20proteins.%20Science.%202022;375(6586):1254%E2%80%9361.%2010.1126/science.abf0529." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Chen J, Jiang Y, Shi H, Peng Y, Fan X, Li C. The molecular mechanisms of copper metabolism and its roles in human diseases. Pflug Arch Eur J Phy. 2020;472(10):1415–29. 10.1007/s00424-020-02412-2.</cite> [<a href="https://doi.org/10.1007/s00424-020-02412-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32506322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20J,%20Jiang%20Y,%20Shi%20H,%20Peng%20Y,%20Fan%20X,%20Li%20C.%20The%20molecular%20mechanisms%20of%20copper%20metabolism%20and%20its%20roles%20in%20human%20diseases.%20Pflug%20Arch%20Eur%20J%20Phy.%202020;472(10):1415%E2%80%9329.%2010.1007/s00424-020-02412-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Ge EJ, Bush AI, Casini A, Cobine PA, Cross JR, DeNicola GM, Dou QP, Franz KJ, Gohil VM, Gupta S, Kaler SG, Lutsenko S, Mittal V, Petris MJ, Polishchuk R, Ralle M, Schilsky ML, Tonks NK, Vahdat LT, Van Aelst L, Xi D, Yuan P, Brady DC, Chang CJ. Connecting copper and cancer: from transition metal signalling to metalloplasia. Nat Rev Cancer. 2022;22(2):102–13. 10.1038/s41568-021-00417-2.
</cite> [<a href="https://doi.org/10.1038/s41568-021-00417-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8810673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34764459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ge%20EJ,%20Bush%20AI,%20Casini%20A,%20Cobine%20PA,%20Cross%20JR,%20DeNicola%20GM,%20Dou%20QP,%20Franz%20KJ,%20Gohil%20VM,%20Gupta%20S,%20Kaler%20SG,%20Lutsenko%20S,%20Mittal%20V,%20Petris%20MJ,%20Polishchuk%20R,%20Ralle%20M,%20Schilsky%20ML,%20Tonks%20NK,%20Vahdat%20LT,%20Van%20Aelst%20L,%20Xi%20D,%20Yuan%20P,%20Brady%20DC,%20Chang%20CJ.%20Connecting%20copper%20and%20cancer:%20from%20transition%20metal%20signalling%20to%20metalloplasia.%20Nat%20Rev%20Cancer.%202022;22(2):102%E2%80%9313.%2010.1038/s41568-021-00417-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Jiang Z, Sha G, Zhang W, Zhang Z, Liu T, Wang D, Tang D. The huge potential of targeting copper status in the treatment of colorectal cancer. Clin Transl Oncol. 2023;25(7):1977–90. 10.1007/s12094-023-03107-7.
</cite> [<a href="https://doi.org/10.1007/s12094-023-03107-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36781599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20Z,%20Sha%20G,%20Zhang%20W,%20Zhang%20Z,%20Liu%20T,%20Wang%20D,%20Tang%20D.%20The%20huge%20potential%20of%20targeting%20copper%20status%20in%20the%20treatment%20of%20colorectal%20cancer.%20Clin%20Transl%20Oncol.%202023;25(7):1977%E2%80%9390.%2010.1007/s12094-023-03107-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Cheng F, Peng G, Lu Y, Wang K, Ju Q, Ju Y, Ouyang M. Relationship between copper and immunity: the potential role of copper in tumor immunity. Front Oncol. 2022;12:1019153. 10.3389/fonc.2022.1019153.
</cite> [<a href="https://doi.org/10.3389/fonc.2022.1019153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9676660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36419894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng%20F,%20Peng%20G,%20Lu%20Y,%20Wang%20K,%20Ju%20Q,%20Ju%20Y,%20Ouyang%20M.%20Relationship%20between%20copper%20and%20immunity:%20the%20potential%20role%20of%20copper%20in%20tumor%20immunity.%20Front%20Oncol.%202022;12:1019153.%2010.3389/fonc.2022.1019153." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Czlonkowska A, Litwin T, Dusek P, Ferenci P, Lutsenko S, Medici V, Rybakowski JK, Weiss KH, Schilsky ML. Wilson disease. Nat Rev Dis Primers. 2018;4(1):21. 10.1038/s41572-018-0018-3.
</cite> [<a href="https://doi.org/10.1038/s41572-018-0018-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6416051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30190489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Czlonkowska%20A,%20Litwin%20T,%20Dusek%20P,%20Ferenci%20P,%20Lutsenko%20S,%20Medici%20V,%20Rybakowski%20JK,%20Weiss%20KH,%20Schilsky%20ML.%20Wilson%20disease.%20Nat%20Rev%20Dis%20Primers.%202018;4(1):21.%2010.1038/s41572-018-0018-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Denoyer D, Masaldan S, La Fontaine S, Cater MA. Targeting copper in cancer therapy: ‘copper that cancer’. Metallomics. 2015;7(11):1459–76. 10.1039/c5mt00149h.
</cite> [<a href="https://doi.org/10.1039/c5mt00149h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26313539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Denoyer%20D,%20Masaldan%20S,%20La%20Fontaine%20S,%20Cater%20MA.%20Targeting%20copper%20in%20cancer%20therapy:%20%E2%80%98copper%20that%20cancer%E2%80%99.%20Metallomics.%202015;7(11):1459%E2%80%9376.%2010.1039/c5mt00149h." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Gupta SK, Shukla VK, Vaidya MP, Roy SK, Gupta S. Serum and tissue trace elements in colorectal cancer. J Surg Oncol. 1993;52(3):172–5. 10.1002/jso.2930520311.
</cite> [<a href="https://doi.org/10.1002/jso.2930520311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8441275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gupta%20SK,%20Shukla%20VK,%20Vaidya%20MP,%20Roy%20SK,%20Gupta%20S.%20Serum%20and%20tissue%20trace%20elements%20in%20colorectal%20cancer.%20J%20Surg%20Oncol.%201993;52(3):172%E2%80%935.%2010.1002/jso.2930520311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Gupte A, Mumper RJ. Elevated copper and oxidative stress in cancer cells as a target for cancer treatment. Cancer Treat Rev. 2009;35(1):32–46. 10.1016/j.ctrv.2008.07.004.
</cite> [<a href="https://doi.org/10.1016/j.ctrv.2008.07.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18774652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gupte%20A,%20Mumper%20RJ.%20Elevated%20copper%20and%20oxidative%20stress%20in%20cancer%20cells%20as%20a%20target%20for%20cancer%20treatment.%20Cancer%20Treat%20Rev.%202009;35(1):32%E2%80%9346.%2010.1016/j.ctrv.2008.07.004." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Khoshdel Z, Naghibalhossaini F, Abdollahi K, Shojaei S, Moradi M, Malekzadeh M. Serum copper and zinc levels among Iranian colorectal Cancer patients. Biol Trace Elem Res. 2016;170(2):294–9. 10.1007/s12011-015-0483-4.
</cite> [<a href="https://doi.org/10.1007/s12011-015-0483-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26329996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khoshdel%20Z,%20Naghibalhossaini%20F,%20Abdollahi%20K,%20Shojaei%20S,%20Moradi%20M,%20Malekzadeh%20M.%20Serum%20copper%20and%20zinc%20levels%20among%20Iranian%20colorectal%20Cancer%20patients.%20Biol%20Trace%20Elem%20Res.%202016;170(2):294%E2%80%939.%2010.1007/s12011-015-0483-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Sharma K, Mittal DK, Kesarwani RC, Kamboj VP. Chowdhery. Diagnostic and prognostic significance of serum and tissue trace elements in breast malignancy. Indian J Med Sci. 1994;48(10):227–32.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/7829172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma%20K,%20Mittal%20DK,%20Kesarwani%20RC,%20Kamboj%20VP.%20Chowdhery.%20Diagnostic%20and%20prognostic%20significance%20of%20serum%20and%20tissue%20trace%20elements%20in%20breast%20malignancy.%20Indian%20J%20Med%20Sci.%201994;48(10):227%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Brady DC, Crowe MS, Turski ML, Hobbs GA, Yao X, Chaikuad A, Knapp S, Xiao K, Campbell SL, Thiele DJ, Counter CM. Copper is required for oncogenic BRAF signalling and tumorigenesis. Nature. 2014;509(7501):492–6. 10.1038/nature13180.
</cite> [<a href="https://doi.org/10.1038/nature13180" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4138975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24717435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brady%20DC,%20Crowe%20MS,%20Turski%20ML,%20Hobbs%20GA,%20Yao%20X,%20Chaikuad%20A,%20Knapp%20S,%20Xiao%20K,%20Campbell%20SL,%20Thiele%20DJ,%20Counter%20CM.%20Copper%20is%20required%20for%20oncogenic%20BRAF%20signalling%20and%20tumorigenesis.%20Nature.%202014;509(7501):492%E2%80%936.%2010.1038/nature13180." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Tsang T, Posimo JM, Gudiel AA, Cicchini M, Feldser DM, Brady DC. Copper is an essential regulator of the autophagic kinases ULK1/2 to drive lung adenocarcinoma. Nat Cell Biol. 2020;22(4):412–24. 10.1038/s41556-020-0481-4.
</cite> [<a href="https://doi.org/10.1038/s41556-020-0481-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7610258/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32203415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsang%20T,%20Posimo%20JM,%20Gudiel%20AA,%20Cicchini%20M,%20Feldser%20DM,%20Brady%20DC.%20Copper%20is%20an%20essential%20regulator%20of%20the%20autophagic%20kinases%20ULK1/2%20to%20drive%20lung%20adenocarcinoma.%20Nat%20Cell%20Biol.%202020;22(4):412%E2%80%9324.%2010.1038/s41556-020-0481-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Chen W, Yang A, Jia J, Popov YV, Schuppan D, You H. Lysyl oxidase (LOX) family members: rationale and their potential as therapeutic targets for liver fibrosis. Hepatology. 2020;72(2):729–41. 10.1002/hep.31236.
</cite> [<a href="https://doi.org/10.1002/hep.31236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32176358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20W,%20Yang%20A,%20Jia%20J,%20Popov%20YV,%20Schuppan%20D,%20You%20H.%20Lysyl%20oxidase%20(LOX)%20family%20members:%20rationale%20and%20their%20potential%20as%20therapeutic%20targets%20for%20liver%20fibrosis.%20Hepatology.%202020;72(2):729%E2%80%9341.%2010.1002/hep.31236." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>El-Haibi CP, Bell GW, Zhang J, Collmann AY, Wood D, Scherber CM, Csizmadia E, Mariani O, Zhu C, Campagne A, Toner M, Bhatia SN, Irimia D, Vincent-Salomon A, Karnoub AE. Critical role for Lysyl oxidase in mesenchymal stem cell-driven breast cancer malignancy. P Natl Acad Sci Usa. 2012;109(43):17460–5. 10.1073/pnas.1206653109.</cite> [<a href="https://doi.org/10.1073/pnas.1206653109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3491529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23033492/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Haibi%20CP,%20Bell%20GW,%20Zhang%20J,%20Collmann%20AY,%20Wood%20D,%20Scherber%20CM,%20Csizmadia%20E,%20Mariani%20O,%20Zhu%20C,%20Campagne%20A,%20Toner%20M,%20Bhatia%20SN,%20Irimia%20D,%20Vincent-Salomon%20A,%20Karnoub%20AE.%20Critical%20role%20for%20Lysyl%20oxidase%20in%20mesenchymal%20stem%20cell-driven%20breast%20cancer%20malignancy.%20P%20Natl%20Acad%20Sci%20Usa.%202012;109(43):17460%E2%80%935.%2010.1073/pnas.1206653109." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Salvador F, Martin A, Lopez-Menendez C, Moreno-Bueno G, Santos V, Vazquez-Naharro A, Santamaria PG, Morales S, Dubus PR, Muinelo-Romay L, Lopez-Lopez R, Tung JC, Weaver VM, Portillo F, Cano A. Lysyl Oxidase-like protein LOXL2 promotes lung metastasis of breast Cancer. Cancer Res. 2017;77(21):5846–59. 10.1158/0008-5472.CAN-16-3152.
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-16-3152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5656180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28720577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Salvador%20F,%20Martin%20A,%20Lopez-Menendez%20C,%20Moreno-Bueno%20G,%20Santos%20V,%20Vazquez-Naharro%20A,%20Santamaria%20PG,%20Morales%20S,%20Dubus%20PR,%20Muinelo-Romay%20L,%20Lopez-Lopez%20R,%20Tung%20JC,%20Weaver%20VM,%20Portillo%20F,%20Cano%20A.%20Lysyl%20Oxidase-like%20protein%20LOXL2%20promotes%20lung%20metastasis%20of%20breast%20Cancer.%20Cancer%20Res.%202017;77(21):5846%E2%80%9359.%2010.1158/0008-5472.CAN-16-3152." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Krishnamoorthy L, Cotruvo JJ, Chan J, Kaluarachchi H, Muchenditsi A, Pendyala VS, Jia S, Aron AT, Ackerman CM, Wal MN, Guan T, Smaga LP, Farhi SL, New EJ, Lutsenko S, Chang CJ. Copper regulates cyclic-AMP-dependent lipolysis. Nat Chem Biol. 2016;12(8):586–92. 10.1038/nchembio.2098.
</cite> [<a href="https://doi.org/10.1038/nchembio.2098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4955676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27272565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krishnamoorthy%20L,%20Cotruvo%20JJ,%20Chan%20J,%20Kaluarachchi%20H,%20Muchenditsi%20A,%20Pendyala%20VS,%20Jia%20S,%20Aron%20AT,%20Ackerman%20CM,%20Wal%20MN,%20Guan%20T,%20Smaga%20LP,%20Farhi%20SL,%20New%20EJ,%20Lutsenko%20S,%20Chang%20CJ.%20Copper%20regulates%20cyclic-AMP-dependent%20lipolysis.%20Nat%20Chem%20Biol.%202016;12(8):586%E2%80%9392.%2010.1038/nchembio.2098." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Pan Q, Kleer CG, van Golen KL, Irani J, Bottema KM, Bias C, De Carvalho M, Mesri EA, Robins DM, Dick RD, Brewer GJ, Merajver SD. Copper deficiency induced by Tetrathiomolybdate suppresses tumor growth and angiogenesis. Cancer Res. 2002;62(17):4854–9.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/12208730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan%20Q,%20Kleer%20CG,%20van%20Golen%20KL,%20Irani%20J,%20Bottema%20KM,%20Bias%20C,%20De%20Carvalho%20M,%20Mesri%20EA,%20Robins%20DM,%20Dick%20RD,%20Brewer%20GJ,%20Merajver%20SD.%20Copper%20deficiency%20induced%20by%20Tetrathiomolybdate%20suppresses%20tumor%20growth%20and%20angiogenesis.%20Cancer%20Res.%202002;62(17):4854%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Kim KK, Abelman S, Yano N, Ribeiro JR, Singh RK, Tipping M, Moore RG. Tetrathiomolybdate inhibits mitochondrial complex IV and mediates degradation of hypoxia-inducible factor-1alpha in cancer cells. Sci Rep-Uk. 2015;5:14296. 10.1038/srep14296.</cite> [<a href="https://doi.org/10.1038/srep14296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4606568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26469226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20KK,%20Abelman%20S,%20Yano%20N,%20Ribeiro%20JR,%20Singh%20RK,%20Tipping%20M,%20Moore%20RG.%20Tetrathiomolybdate%20inhibits%20mitochondrial%20complex%20IV%20and%20mediates%20degradation%20of%20hypoxia-inducible%20factor-1alpha%20in%20cancer%20cells.%20Sci%20Rep-Uk.%202015;5:14296.%2010.1038/srep14296." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Bonneville R, Krook MA, Kautto EA, Miya J, Wing MR, Chen HZ, Reeser JW, Yu L, Roychowdhury S. Landscape of microsatellite instability across 39 Cancer types. Jco Precis Oncol. 2017;2017. 10.1200/PO.17.00073.</cite> [<a href="https://doi.org/10.1200/PO.17.00073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5972025/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29850653/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Gelsomino F, Barbolini M, Spallanzani A, Pugliese G, Cascinu S. The evolving role of microsatellite instability in colorectal cancer: A review. Cancer Treat Rev. 2016;51:19–26. 10.1016/j.ctrv.2016.10.005.
</cite> [<a href="https://doi.org/10.1016/j.ctrv.2016.10.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27838401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gelsomino%20F,%20Barbolini%20M,%20Spallanzani%20A,%20Pugliese%20G,%20Cascinu%20S.%20The%20evolving%20role%20of%20microsatellite%20instability%20in%20colorectal%20cancer:%20A%20review.%20Cancer%20Treat%20Rev.%202016;51:19%E2%80%9326.%2010.1016/j.ctrv.2016.10.005." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Li Y. Copper homeostasis: emerging target for cancer treatment. IUBMB Life. 2020;72(9):1900–8. 10.1002/iub.2341.
</cite> [<a href="https://doi.org/10.1002/iub.2341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32599675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Y.%20Copper%20homeostasis:%20emerging%20target%20for%20cancer%20treatment.%20IUBMB%20Life.%202020;72(9):1900%E2%80%938.%2010.1002/iub.2341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Zowczak M, Iskra M, Torlinski L, Cofta S. Analysis of serum copper and zinc concentrations in cancer patients. Biol Trace Elem Res. 2001;82(1–3):1–8. 10.1385/BTER:82:1-3:001.
</cite> [<a href="https://doi.org/10.1385/BTER:82:1-3:001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11697759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zowczak%20M,%20Iskra%20M,%20Torlinski%20L,%20Cofta%20S.%20Analysis%20of%20serum%20copper%20and%20zinc%20concentrations%20in%20cancer%20patients.%20Biol%20Trace%20Elem%20Res.%202001;82(1%E2%80%933):1%E2%80%938.%2010.1385/BTER:82:1-3:001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Voli F, Valli E, Lerra L, Kimpton K, Saletta F, Giorgi FM, Mercatelli D, Rouaen J, Shen S, Murray JE, Ahmed-Cox A, Cirillo G, Mayoh C, Beavis PA, Haber M, Trapani JA, Kavallaris M, Vittorio O. Intratumoral copper modulates PD-L1 expression and influences tumor immune evasion. Cancer Res. 2020;80(19):4129–44. 10.1158/0008-5472.CAN-20-0471.
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-20-0471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32816860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Voli%20F,%20Valli%20E,%20Lerra%20L,%20Kimpton%20K,%20Saletta%20F,%20Giorgi%20FM,%20Mercatelli%20D,%20Rouaen%20J,%20Shen%20S,%20Murray%20JE,%20Ahmed-Cox%20A,%20Cirillo%20G,%20Mayoh%20C,%20Beavis%20PA,%20Haber%20M,%20Trapani%20JA,%20Kavallaris%20M,%20Vittorio%20O.%20Intratumoral%20copper%20modulates%20PD-L1%20expression%20and%20influences%20tumor%20immune%20evasion.%20Cancer%20Res.%202020;80(19):4129%E2%80%9344.%2010.1158/0008-5472.CAN-20-0471." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Xu X, Ding C, Zhong H, Qin W, Shu D, Yu M, Abuduaini N, Zhang S, Yang X, Feng B. Integrative analysis revealed that distinct cuprotosis patterns reshaped tumor microenvironment and responses to immunotherapy of colorectal cancer. Front Immunol. 2023;14:1165101. 10.3389/fimmu.2023.1165101.
</cite> [<a href="https://doi.org/10.3389/fimmu.2023.1165101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10060625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37006250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20X,%20Ding%20C,%20Zhong%20H,%20Qin%20W,%20Shu%20D,%20Yu%20M,%20Abuduaini%20N,%20Zhang%20S,%20Yang%20X,%20Feng%20B.%20Integrative%20analysis%20revealed%20that%20distinct%20cuprotosis%20patterns%20reshaped%20tumor%20microenvironment%20and%20responses%20to%20immunotherapy%20of%20colorectal%20cancer.%20Front%20Immunol.%202023;14:1165101.%2010.3389/fimmu.2023.1165101." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Huang J, Zhang L, Wan D, Zhou L, Zheng S, Lin S, Qiao Y. Extracellular matrix and its therapeutic potential for cancer treatment. Signal Transduct Tar. 2021;6(1):153. 10.1038/s41392-021-00544-0.</cite> [<a href="https://doi.org/10.1038/s41392-021-00544-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8062524/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33888679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20J,%20Zhang%20L,%20Wan%20D,%20Zhou%20L,%20Zheng%20S,%20Lin%20S,%20Qiao%20Y.%20Extracellular%20matrix%20and%20its%20therapeutic%20potential%20for%20cancer%20treatment.%20Signal%20Transduct%20Tar.%202021;6(1):153.%2010.1038/s41392-021-00544-0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Yuan Z, Li Y, Zhang S, Wang X, Dou H, Yu X, Zhang Z, Yang S, Xiao M. Extracellular matrix remodeling in tumor progression and immune escape: from mechanisms to treatments. Mol Cancer. 2023;22(1):48. 10.1186/s12943-023-01744-8.
</cite> [<a href="https://doi.org/10.1186/s12943-023-01744-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10007858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36906534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan%20Z,%20Li%20Y,%20Zhang%20S,%20Wang%20X,%20Dou%20H,%20Yu%20X,%20Zhang%20Z,%20Yang%20S,%20Xiao%20M.%20Extracellular%20matrix%20remodeling%20in%20tumor%20progression%20and%20immune%20escape:%20from%20mechanisms%20to%20treatments.%20Mol%20Cancer.%202023;22(1):48.%2010.1186/s12943-023-01744-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Barker HE, Cox TR, Erler JT. The rationale for targeting the LOX family in cancer. Nat Rev Cancer. 2012;12(8):540–52. 10.1038/nrc3319.
</cite> [<a href="https://doi.org/10.1038/nrc3319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22810810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barker%20HE,%20Cox%20TR,%20Erler%20JT.%20The%20rationale%20for%20targeting%20the%20LOX%20family%20in%20cancer.%20Nat%20Rev%20Cancer.%202012;12(8):540%E2%80%9352.%2010.1038/nrc3319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Pez F, Dayan F, Durivault J, Kaniewski B, Aimond G, Le Provost GS, Deux B, Clezardin P, Sommer P, Pouyssegur J, Reynaud C. The HIF-1-inducible Lysyl oxidase activates HIF-1 via the Akt pathway in a positive regulation loop and synergizes with HIF-1 in promoting tumor cell growth. Cancer Res. 2011;71(5):1647–57. 10.1158/0008-5472.CAN-10-1516.
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-10-1516" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21239473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pez%20F,%20Dayan%20F,%20Durivault%20J,%20Kaniewski%20B,%20Aimond%20G,%20Le%20Provost%20GS,%20Deux%20B,%20Clezardin%20P,%20Sommer%20P,%20Pouyssegur%20J,%20Reynaud%20C.%20The%20HIF-1-inducible%20Lysyl%20oxidase%20activates%20HIF-1%20via%20the%20Akt%20pathway%20in%20a%20positive%20regulation%20loop%20and%20synergizes%20with%20HIF-1%20in%20promoting%20tumor%20cell%20growth.%20Cancer%20Res.%202011;71(5):1647%E2%80%9357.%2010.1158/0008-5472.CAN-10-1516." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Wong CC, Tse AP, Huang YP, Zhu YT, Chiu DK, Lai RK, Au SL, Kai AK, Lee JM, Wei LL, Tsang FH, Lo RC, Shi J, Zheng YP, Wong CM, Ng IO. Lysyl oxidase-like 2 is critical to tumor microenvironment and metastatic niche formation in hepatocellular carcinoma. Hepatology. 2014;60(5):1645–58. 10.1002/hep.27320.
</cite> [<a href="https://doi.org/10.1002/hep.27320" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25048396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wong%20CC,%20Tse%20AP,%20Huang%20YP,%20Zhu%20YT,%20Chiu%20DK,%20Lai%20RK,%20Au%20SL,%20Kai%20AK,%20Lee%20JM,%20Wei%20LL,%20Tsang%20FH,%20Lo%20RC,%20Shi%20J,%20Zheng%20YP,%20Wong%20CM,%20Ng%20IO.%20Lysyl%20oxidase-like%202%20is%20critical%20to%20tumor%20microenvironment%20and%20metastatic%20niche%20formation%20in%20hepatocellular%20carcinoma.%20Hepatology.%202014;60(5):1645%E2%80%9358.%2010.1002/hep.27320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Wei B, Zhou X, Liang C, Zheng X, Lei P, Fang J, Han X, Wang L, Qi C, Wei H. Human colorectal cancer progression correlates with LOX-induced ECM stiffening. Int J Biol Sci. 2017;13(11):1450–7. 10.7150/ijbs.21230.
</cite> [<a href="https://doi.org/10.7150/ijbs.21230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5715527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29209148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei%20B,%20Zhou%20X,%20Liang%20C,%20Zheng%20X,%20Lei%20P,%20Fang%20J,%20Han%20X,%20Wang%20L,%20Qi%20C,%20Wei%20H.%20Human%20colorectal%20cancer%20progression%20correlates%20with%20LOX-induced%20ECM%20stiffening.%20Int%20J%20Biol%20Sci.%202017;13(11):1450%E2%80%937.%2010.7150/ijbs.21230." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Liu Y, Wang G, Liang Z, Mei Z, Wu T, Cui A, Liu C, Cui L. Lysyl oxidase: A colorectal cancer biomarker of lung and hepatic metastasis. Thorac Cancer. 2018;9(7):785–93. 10.1111/1759-7714.12645.
</cite> [<a href="https://doi.org/10.1111/1759-7714.12645" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6026604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29766649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Y,%20Wang%20G,%20Liang%20Z,%20Mei%20Z,%20Wu%20T,%20Cui%20A,%20Liu%20C,%20Cui%20L.%20Lysyl%20oxidase:%20A%20colorectal%20cancer%20biomarker%20of%20lung%20and%20hepatic%20metastasis.%20Thorac%20Cancer.%202018;9(7):785%E2%80%9393.%2010.1111/1759-7714.12645." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Kim Y, Roh S, Park JY, Kim Y, Cho DH, Kim JC. Differential expression of the LOX family genes in human colorectal adenocarcinomas. Oncol Rep. 2009;22(4):799–804. 10.3892/or_00000502.
</cite> [<a href="https://doi.org/10.3892/or_00000502" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19724858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20Y,%20Roh%20S,%20Park%20JY,%20Kim%20Y,%20Cho%20DH,%20Kim%20JC.%20Differential%20expression%20of%20the%20LOX%20family%20genes%20in%20human%20colorectal%20adenocarcinomas.%20Oncol%20Rep.%202009;22(4):799%E2%80%93804.%2010.3892/or_00000502." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Baker AM, Cox TR, Bird D, Lang G, Murray GI, Sun XF, Southall SM, Wilson JR, Erler JT. The role of Lysyl oxidase in SRC-dependent proliferation and metastasis of colorectal cancer. Jnci-J Natl Cancer I. 2011;103(5):407–24. 10.1093/jnci/djq569.</cite> [<a href="https://doi.org/10.1093/jnci/djq569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21282564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baker%20AM,%20Cox%20TR,%20Bird%20D,%20Lang%20G,%20Murray%20GI,%20Sun%20XF,%20Southall%20SM,%20Wilson%20JR,%20Erler%20JT.%20The%20role%20of%20Lysyl%20oxidase%20in%20SRC-dependent%20proliferation%20and%20metastasis%20of%20colorectal%20cancer.%20Jnci-J%20Natl%20Cancer%20I.%202011;103(5):407%E2%80%9324.%2010.1093/jnci/djq569." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Baker AM, Bird D, Lang G, Cox TR, Erler JT. Lysyl oxidase enzymatic function increases stiffness to drive colorectal cancer progression through FAK. Oncogene. 2013;32(14):1863–8. 10.1038/onc.2012.202.
</cite> [<a href="https://doi.org/10.1038/onc.2012.202" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22641216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baker%20AM,%20Bird%20D,%20Lang%20G,%20Cox%20TR,%20Erler%20JT.%20Lysyl%20oxidase%20enzymatic%20function%20increases%20stiffness%20to%20drive%20colorectal%20cancer%20progression%20through%20FAK.%20Oncogene.%202013;32(14):1863%E2%80%938.%2010.1038/onc.2012.202." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Park PG, Jo SJ, Kim MJ, Kim HJ, Lee JH, Park CK, Kim H, Lee KY, Kim H, Park JH, Dong SM, Lee JM. Role of LOXL2 in the epithelial-mesenchymal transition and colorectal cancer metastasis. Oncotarget. 2017;8(46):80325–35. 10.18632/oncotarget.18170.
</cite> [<a href="https://doi.org/10.18632/oncotarget.18170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5655201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29113306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Park%20PG,%20Jo%20SJ,%20Kim%20MJ,%20Kim%20HJ,%20Lee%20JH,%20Park%20CK,%20Kim%20H,%20Lee%20KY,%20Kim%20H,%20Park%20JH,%20Dong%20SM,%20Lee%20JM.%20Role%20of%20LOXL2%20in%20the%20epithelial-mesenchymal%20transition%20and%20colorectal%20cancer%20metastasis.%20Oncotarget.%202017;8(46):80325%E2%80%9335.%2010.18632/oncotarget.18170." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>You G, Zhou C, Wang L, Liu Z, Fang H, Yao X, Zhang X. COMMD proteins function and their regulating roles in tumors. Front Oncol. 2023;13:1067234. 10.3389/fonc.2023.1067234.
</cite> [<a href="https://doi.org/10.3389/fonc.2023.1067234" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9910083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36776284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?You%20G,%20Zhou%20C,%20Wang%20L,%20Liu%20Z,%20Fang%20H,%20Yao%20X,%20Zhang%20X.%20COMMD%20proteins%20function%20and%20their%20regulating%20roles%20in%20tumors.%20Front%20Oncol.%202023;13:1067234.%2010.3389/fonc.2023.1067234." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Klomp AE, van de Sluis B, Klomp LW, Wijmenga C. The ubiquitously expressed MURR1 protein is absent in canine copper toxicosis. J Hepatol. 2003;39(5):703–9. 10.1016/s0168-8278(03)00380-5.
</cite> [<a href="https://doi.org/10.1016/s0168-8278(03)00380-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14568250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klomp%20AE,%20van%20de%20Sluis%20B,%20Klomp%20LW,%20Wijmenga%20C.%20The%20ubiquitously%20expressed%20MURR1%20protein%20is%20absent%20in%20canine%20copper%20toxicosis.%20J%20Hepatol.%202003;39(5):703%E2%80%939.%2010.1016/s0168-8278(03)00380-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Miyayama T, Hiraoka D, Kawaji F, Nakamura E, Suzuki N, Ogra Y. Roles of COMM-domain-containing 1 in stability and recruitment of the copper-transporting ATPase in a mouse hepatoma cell line. Biochem J. 2010;429(1):53–61. 10.1042/BJ20100223.
</cite> [<a href="https://doi.org/10.1042/BJ20100223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20433422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miyayama%20T,%20Hiraoka%20D,%20Kawaji%20F,%20Nakamura%20E,%20Suzuki%20N,%20Ogra%20Y.%20Roles%20of%20COMM-domain-containing%201%20in%20stability%20and%20recruitment%20of%20the%20copper-transporting%20ATPase%20in%20a%20mouse%20hepatoma%20cell%20line.%20Biochem%20J.%202010;429(1):53%E2%80%9361.%2010.1042/BJ20100223." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Materia S, Cater MA, Klomp LW, Mercer JF, La Fontaine S. Clusterin and COMMD1 independently regulate degradation of the mammalian copper ATPases ATP7A and ATP7B. J Biol Chem. 2012;287(4):2485–99. 10.1074/jbc.M111.302216.
</cite> [<a href="https://doi.org/10.1074/jbc.M111.302216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3268409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22130675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Materia%20S,%20Cater%20MA,%20Klomp%20LW,%20Mercer%20JF,%20La%20Fontaine%20S.%20Clusterin%20and%20COMMD1%20independently%20regulate%20degradation%20of%20the%20mammalian%20copper%20ATPases%20ATP7A%20and%20ATP7B.%20J%20Biol%20Chem.%202012;287(4):2485%E2%80%9399.%2010.1074/jbc.M111.302216." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Ilyechova EY, Bonaldi E, Orlov IA, Skomorokhova EA, Puchkova LV, Broggini M. CRISP-R/Cas9 mediated deletion of copper transport genes CTR1 and DMT1 in NSCLC cell line H1299. Biological and Pharmacological consequences. Cells-Basel. 2019;8(4). 10.3390/cells8040322.</cite> [<a href="https://doi.org/10.3390/cells8040322" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6523758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30959888/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Hancock JL, Kalimutho M, Straube J, Lim M, Gresshoff I, Saunus JM, Lee JS, Lakhani SR, Simpson KJ, Bush AI, Anderson RL, Khanna KK. COMMD3 loss drives invasive breast cancer growth by modulating copper homeostasis. J Exp Clin Canc Res. 2023;42(1):90. 10.1186/s13046-023-02663-8.</cite> [<a href="https://doi.org/10.1186/s13046-023-02663-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10111822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37072858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hancock%20JL,%20Kalimutho%20M,%20Straube%20J,%20Lim%20M,%20Gresshoff%20I,%20Saunus%20JM,%20Lee%20JS,%20Lakhani%20SR,%20Simpson%20KJ,%20Bush%20AI,%20Anderson%20RL,%20Khanna%20KK.%20COMMD3%20loss%20drives%20invasive%20breast%20cancer%20growth%20by%20modulating%20copper%20homeostasis.%20J%20Exp%20Clin%20Canc%20Res.%202023;42(1):90.%2010.1186/s13046-023-02663-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Binnewies M, Roberts EW, Kersten K, Chan V, Fearon DF, Merad M, Coussens LM, Gabrilovich DI, Ostrand-Rosenberg S, Hedrick CC, Vonderheide RH, Pittet MJ, Jain RK, Zou W, Howcroft TK, Woodhouse EC, Weinberg RA, Krummel MF. Understanding the tumor immune microenvironment (TIME) for effective therapy. Nat Med. 2018;24(5):541–50. 10.1038/s41591-018-0014-x.
</cite> [<a href="https://doi.org/10.1038/s41591-018-0014-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5998822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29686425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Binnewies%20M,%20Roberts%20EW,%20Kersten%20K,%20Chan%20V,%20Fearon%20DF,%20Merad%20M,%20Coussens%20LM,%20Gabrilovich%20DI,%20Ostrand-Rosenberg%20S,%20Hedrick%20CC,%20Vonderheide%20RH,%20Pittet%20MJ,%20Jain%20RK,%20Zou%20W,%20Howcroft%20TK,%20Woodhouse%20EC,%20Weinberg%20RA,%20Krummel%20MF.%20Understanding%20the%20tumor%20immune%20microenvironment%20(TIME)%20for%20effective%20therapy.%20Nat%20Med.%202018;24(5):541%E2%80%9350.%2010.1038/s41591-018-0014-x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Wang H, Tian T, Zhang J. Tumor-Associated macrophages (TAMs) in colorectal Cancer (CRC): from mechanism to therapy and prognosis. Int J Mol Sci. 2021;22(16). 10.3390/ijms22168470.</cite> [<a href="https://doi.org/10.3390/ijms22168470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8395168/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34445193/" class="usa-link">PubMed</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Xiang X, Wang J, Lu D, Xu X. Targeting tumor-associated macrophages to synergize tumor immunotherapy. Signal Transduct Tar. 2021;6(1):75. 10.1038/s41392-021-00484-9.</cite> [<a href="https://doi.org/10.1038/s41392-021-00484-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7900181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33619259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiang%20X,%20Wang%20J,%20Lu%20D,%20Xu%20X.%20Targeting%20tumor-associated%20macrophages%20to%20synergize%20tumor%20immunotherapy.%20Signal%20Transduct%20Tar.%202021;6(1):75.%2010.1038/s41392-021-00484-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Lin Y, Xu J, Lan H. Tumor-associated macrophages in tumor metastasis: biological roles and clinical therapeutic applications. J Hematol Oncol. 2019;12(1):76. 10.1186/s13045-019-0760-3.
</cite> [<a href="https://doi.org/10.1186/s13045-019-0760-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6626377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31300030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20Y,%20Xu%20J,%20Lan%20H.%20Tumor-associated%20macrophages%20in%20tumor%20metastasis:%20biological%20roles%20and%20clinical%20therapeutic%20applications.%20J%20Hematol%20Oncol.%202019;12(1):76.%2010.1186/s13045-019-0760-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Vayrynen JP, Haruki K, Lau MC, Vayrynen SA, Zhong R, Dias CA, Borowsky J, Zhao M, Fujiyoshi K, Arima K, Twombly TS, Kishikawa J, Gu S, Aminmozaffari S, Shi S, Baba Y, Akimoto N, Ugai T, Da SA, Guerriero JL, Song M, Wu K, Chan AT, Nishihara R, Fuchs CS, Meyerhardt JA, Giannakis M, Ogino S, Nowak JA. The prognostic role of macrophage polarization in the colorectal Cancer microenvironment. Cancer Immunol Res. 2021;9(1):8–19. 10.1158/2326-6066.CIR-20-0527.
</cite> [<a href="https://doi.org/10.1158/2326-6066.CIR-20-0527" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7785652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33023967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vayrynen%20JP,%20Haruki%20K,%20Lau%20MC,%20Vayrynen%20SA,%20Zhong%20R,%20Dias%20CA,%20Borowsky%20J,%20Zhao%20M,%20Fujiyoshi%20K,%20Arima%20K,%20Twombly%20TS,%20Kishikawa%20J,%20Gu%20S,%20Aminmozaffari%20S,%20Shi%20S,%20Baba%20Y,%20Akimoto%20N,%20Ugai%20T,%20Da%20SA,%20Guerriero%20JL,%20Song%20M,%20Wu%20K,%20Chan%20AT,%20Nishihara%20R,%20Fuchs%20CS,%20Meyerhardt%20JA,%20Giannakis%20M,%20Ogino%20S,%20Nowak%20JA.%20The%20prognostic%20role%20of%20macrophage%20polarization%20in%20the%20colorectal%20Cancer%20microenvironment.%20Cancer%20Immunol%20Res.%202021;9(1):8%E2%80%9319.%2010.1158/2326-6066.CIR-20-0527." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Arancibia S, Barrientos A, Torrejon J, Escobar A, Beltran CJ. Copper oxide nanoparticles recruit macrophages and modulate nitric oxide, Proinflammatory cytokines and PGE2 production through arginase activation. Nanomedicine-Uk. 2016;11(10):1237–51. 10.2217/nnm.16.39.</cite> [<a href="https://doi.org/10.2217/nnm.16.39" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27079258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arancibia%20S,%20Barrientos%20A,%20Torrejon%20J,%20Escobar%20A,%20Beltran%20CJ.%20Copper%20oxide%20nanoparticles%20recruit%20macrophages%20and%20modulate%20nitric%20oxide,%20Proinflammatory%20cytokines%20and%20PGE2%20production%20through%20arginase%20activation.%20Nanomedicine-Uk.%202016;11(10):1237%E2%80%9351.%2010.2217/nnm.16.39." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Dukhinova MS, Prilepskii AY, Shtil AA, Vinogradov VV. Metal oxide nanoparticles in therapeutic regulation of macrophage functions. Nanomaterials-Basel. 2019;9(11). 10.3390/nano9111631.</cite> [<a href="https://doi.org/10.3390/nano9111631" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6915518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31744137/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Triboulet S, Aude-Garcia C, Carriere M, Diemer H, Proamer F, Habert A, Chevallet M, Collin-Faure V, Strub JM, Hanau D, Van Dorsselaer A, Herlin-Boime N, Rabilloud T. Molecular responses of mouse macrophages to copper and copper oxide nanoparticles inferred from proteomic analyses. Mol Cell Proteom. 2013;12(11):3108–22. 10.1074/mcp.M113.030742.</cite> [<a href="https://doi.org/10.1074/mcp.M113.030742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3820927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23882024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Triboulet%20S,%20Aude-Garcia%20C,%20Carriere%20M,%20Diemer%20H,%20Proamer%20F,%20Habert%20A,%20Chevallet%20M,%20Collin-Faure%20V,%20Strub%20JM,%20Hanau%20D,%20Van%20Dorsselaer%20A,%20Herlin-Boime%20N,%20Rabilloud%20T.%20Molecular%20responses%20of%20mouse%20macrophages%20to%20copper%20and%20copper%20oxide%20nanoparticles%20inferred%20from%20proteomic%20analyses.%20Mol%20Cell%20Proteom.%202013;12(11):3108%E2%80%9322.%2010.1074/mcp.M113.030742." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Huang R, Chen H, Liang J, Li Y, Yang J, Luo C, Tang Y, Ding Y, Liu X, Yuan Q, Yu H, Ye Y, Xu W, Xie X. Dual role of reactive oxygen species and their application in Cancer therapy. J Cancer. 2021;12(18):5543–61. 10.7150/jca.54699.
</cite> [<a href="https://doi.org/10.7150/jca.54699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8364652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34405016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20R,%20Chen%20H,%20Liang%20J,%20Li%20Y,%20Yang%20J,%20Luo%20C,%20Tang%20Y,%20Ding%20Y,%20Liu%20X,%20Yuan%20Q,%20Yu%20H,%20Ye%20Y,%20Xu%20W,%20Xie%20X.%20Dual%20role%20of%20reactive%20oxygen%20species%20and%20their%20application%20in%20Cancer%20therapy.%20J%20Cancer.%202021;12(18):5543%E2%80%9361.%2010.7150/jca.54699." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Zhang J, Li H, Wu Q, Chen Y, Deng Y, Yang Z, Zhang L, Liu B. Tumoral NOX4 recruits M2 tumor-associated macrophages via ROS/PI3K signaling-dependent various cytokine production to promote NSCLC growth. Redox Biol. 2019;22:101116. 10.1016/j.redox.2019.101116.
</cite> [<a href="https://doi.org/10.1016/j.redox.2019.101116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6374999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30769285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20J,%20Li%20H,%20Wu%20Q,%20Chen%20Y,%20Deng%20Y,%20Yang%20Z,%20Zhang%20L,%20Liu%20B.%20Tumoral%20NOX4%20recruits%20M2%20tumor-associated%20macrophages%20via%20ROS/PI3K%20signaling-dependent%20various%20cytokine%20production%20to%20promote%20NSCLC%20growth.%20Redox%20Biol.%202019;22:101116.%2010.1016/j.redox.2019.101116." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Liu CQ, Xu J, Zhou ZG, Jin LL, Yu XJ, Xiao G, Lin J, Zhuang SM, Zhang YJ, Zheng L. Expression patterns of programmed death ligand 1 correlate with different microenvironments and patient prognosis in hepatocellular carcinoma. Brit J Cancer. 2018;119(1):80–8. 10.1038/s41416-018-0144-4.
</cite> [<a href="https://doi.org/10.1038/s41416-018-0144-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6035200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29921949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20CQ,%20Xu%20J,%20Zhou%20ZG,%20Jin%20LL,%20Yu%20XJ,%20Xiao%20G,%20Lin%20J,%20Zhuang%20SM,%20Zhang%20YJ,%20Zheng%20L.%20Expression%20patterns%20of%20programmed%20death%20ligand%201%20correlate%20with%20different%20microenvironments%20and%20patient%20prognosis%20in%20hepatocellular%20carcinoma.%20Brit%20J%20Cancer.%202018;119(1):80%E2%80%938.%2010.1038/s41416-018-0144-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Yin Y, Liu B, Cao Y, Yao S, Liu Y, Jin G, Qin Y, Chen Y, Cui K, Zhou L, Bian Z, Fei B, Huang S, Huang Z. Colorectal Cancer-Derived small extracellular vesicles promote tumor immune evasion by upregulating PD-L1 expression in tumor-Associated macrophages. Adv Sci. 2022;9(9):2102620. 10.1002/advs.202102620.</cite> [<a href="https://doi.org/10.1002/advs.202102620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8948581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35356153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yin%20Y,%20Liu%20B,%20Cao%20Y,%20Yao%20S,%20Liu%20Y,%20Jin%20G,%20Qin%20Y,%20Chen%20Y,%20Cui%20K,%20Zhou%20L,%20Bian%20Z,%20Fei%20B,%20Huang%20S,%20Huang%20Z.%20Colorectal%20Cancer-Derived%20small%20extracellular%20vesicles%20promote%20tumor%20immune%20evasion%20by%20upregulating%20PD-L1%20expression%20in%20tumor-Associated%20macrophages.%20Adv%20Sci.%202022;9(9):2102620.%2010.1002/advs.202102620." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Hartley GP, Chow L, Ammons DT, Wheat WH, Dow SW. Programmed cell death ligand 1 (PD-L1) signaling regulates macrophage proliferation and activation. Cancer Immunol Res. 2018;6(10):1260–73. 10.1158/2326-6066.CIR-17-0537.
</cite> [<a href="https://doi.org/10.1158/2326-6066.CIR-17-0537" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30012633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hartley%20GP,%20Chow%20L,%20Ammons%20DT,%20Wheat%20WH,%20Dow%20SW.%20Programmed%20cell%20death%20ligand%201%20(PD-L1)%20signaling%20regulates%20macrophage%20proliferation%20and%20activation.%20Cancer%20Immunol%20Res.%202018;6(10):1260%E2%80%9373.%2010.1158/2326-6066.CIR-17-0537." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Gordon SR, Maute RL, Dulken BW, Hutter G, George BM, McCracken MN, Gupta R, Tsai JM, Sinha R, Corey D, Ring AM, Connolly AJ, Weissman IL. PD-1 expression by tumour-associated macrophages inhibits phagocytosis and tumour immunity. Nature. 2017;545(7655):495–9. 10.1038/nature22396.
</cite> [<a href="https://doi.org/10.1038/nature22396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5931375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28514441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gordon%20SR,%20Maute%20RL,%20Dulken%20BW,%20Hutter%20G,%20George%20BM,%20McCracken%20MN,%20Gupta%20R,%20Tsai%20JM,%20Sinha%20R,%20Corey%20D,%20Ring%20AM,%20Connolly%20AJ,%20Weissman%20IL.%20PD-1%20expression%20by%20tumour-associated%20macrophages%20inhibits%20phagocytosis%20and%20tumour%20immunity.%20Nature.%202017;545(7655):495%E2%80%939.%2010.1038/nature22396." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Tichet M, Wullschleger S, Chryplewicz A, Fournier N, Marcone R, Kauzlaric A, Homicsko K, Deak LC, Umana P, Klein C, Hanahan D. Bispecific PD1-IL2v and anti-PD-L1 break tumor immunity resistance by enhancing stem-like tumor-reactive CD8(+) T cells and reprogramming macrophages. Immunity. 2023;56(1):162–79. 10.1016/j.immuni.2022.12.006.
</cite> [<a href="https://doi.org/10.1016/j.immuni.2022.12.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36630914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tichet%20M,%20Wullschleger%20S,%20Chryplewicz%20A,%20Fournier%20N,%20Marcone%20R,%20Kauzlaric%20A,%20Homicsko%20K,%20Deak%20LC,%20Umana%20P,%20Klein%20C,%20Hanahan%20D.%20Bispecific%20PD1-IL2v%20and%20anti-PD-L1%20break%20tumor%20immunity%20resistance%20by%20enhancing%20stem-like%20tumor-reactive%20CD8(+)%20T%20cells%20and%20reprogramming%20macrophages.%20Immunity.%202023;56(1):162%E2%80%9379.%2010.1016/j.immuni.2022.12.006." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Que H, Fu Q, Lan T, Tian X, Wei X. Tumor-associated neutrophils and neutrophil-targeted cancer therapies. Bba-Rev Cancer. 2022;1877(5):188762. 10.1016/j.bbcan.2022.188762.</cite> [<a href="https://doi.org/10.1016/j.bbcan.2022.188762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35853517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Que%20H,%20Fu%20Q,%20Lan%20T,%20Tian%20X,%20Wei%20X.%20Tumor-associated%20neutrophils%20and%20neutrophil-targeted%20cancer%20therapies.%20Bba-Rev%20Cancer.%202022;1877(5):188762.%2010.1016/j.bbcan.2022.188762." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Lecot P, Sarabi M, Pereira AM, Mussard J, Koenderman L, Caux C, Bendriss-Vermare N, Michallet MC. Neutrophil heterogeneity in cancer: from biology to therapies. Front Immunol. 2019;10:2155. 10.3389/fimmu.2019.02155.
</cite> [<a href="https://doi.org/10.3389/fimmu.2019.02155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6764113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31616408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lecot%20P,%20Sarabi%20M,%20Pereira%20AM,%20Mussard%20J,%20Koenderman%20L,%20Caux%20C,%20Bendriss-Vermare%20N,%20Michallet%20MC.%20Neutrophil%20heterogeneity%20in%20cancer:%20from%20biology%20to%20therapies.%20Front%20Immunol.%202019;10:2155.%2010.3389/fimmu.2019.02155." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Michaeli J, Shaul ME, Mishalian I, Hovav AH, Levy L, Zolotriov L, Granot Z, Fridlender ZG. Tumor-associated neutrophils induce apoptosis of non-activated CD8 T-cells in a TNFalpha and NO-dependent mechanism, promoting a tumor-supportive environment. Oncoimmunology. 2017;6(11):e1356965. 10.1080/2162402X.2017.1356965.
</cite> [<a href="https://doi.org/10.1080/2162402X.2017.1356965" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5674962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29147615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michaeli%20J,%20Shaul%20ME,%20Mishalian%20I,%20Hovav%20AH,%20Levy%20L,%20Zolotriov%20L,%20Granot%20Z,%20Fridlender%20ZG.%20Tumor-associated%20neutrophils%20induce%20apoptosis%20of%20non-activated%20CD8%20T-cells%20in%20a%20TNFalpha%20and%20NO-dependent%20mechanism,%20promoting%20a%20tumor-supportive%20environment.%20Oncoimmunology.%202017;6(11):e1356965.%2010.1080/2162402X.2017.1356965." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Faget J, Groeneveld S, Boivin G, Sankar M, Zangger N, Garcia M, Guex N, Zlobec I, Steiner L, Piersigilli A, Xenarios I, Meylan E. Neutrophils and snail orchestrate the establishment of a Pro-tumor microenvironment in lung Cancer. Cell Rep. 2017;21(11):3190–204. 10.1016/j.celrep.2017.11.052.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.11.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29241546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Faget%20J,%20Groeneveld%20S,%20Boivin%20G,%20Sankar%20M,%20Zangger%20N,%20Garcia%20M,%20Guex%20N,%20Zlobec%20I,%20Steiner%20L,%20Piersigilli%20A,%20Xenarios%20I,%20Meylan%20E.%20Neutrophils%20and%20snail%20orchestrate%20the%20establishment%20of%20a%20Pro-tumor%20microenvironment%20in%20lung%20Cancer.%20Cell%20Rep.%202017;21(11):3190%E2%80%93204.%2010.1016/j.celrep.2017.11.052." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Adrover JM, McDowell S, He XY, Quail DF, Egeblad M. NETworking with cancer: the bidirectional interplay between cancer and neutrophil extracellular traps. Cancer Cell. 2023;41(3):505–26. 10.1016/j.ccell.2023.02.001.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2023.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10280682/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36827980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adrover%20JM,%20McDowell%20S,%20He%20XY,%20Quail%20DF,%20Egeblad%20M.%20NETworking%20with%20cancer:%20the%20bidirectional%20interplay%20between%20cancer%20and%20neutrophil%20extracellular%20traps.%20Cancer%20Cell.%202023;41(3):505%E2%80%9326.%2010.1016/j.ccell.2023.02.001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Khan U, Chowdhury S, Billah MM, Islam K, Thorlacius H, Rahman M. Neutrophil extracellular traps in colorectal Cancer progression and metastasis. Int J Mol Sci. 2021;22(14). 10.3390/ijms22147260.</cite> [<a href="https://doi.org/10.3390/ijms22147260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8307027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34298878/" class="usa-link">PubMed</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Nie M, Yang L, Bi X, Wang Y, Sun P, Yang H, Liu P, Li Z, Xia Y, Jiang W. Neutrophil extracellular traps induced by IL8 promote diffuse large B-cell lymphoma progression via the TLR9 signaling. Clin Cancer Res. 2019;25(6):1867–79. 10.1158/1078-0432.CCR-18-1226.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-18-1226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30446590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nie%20M,%20Yang%20L,%20Bi%20X,%20Wang%20Y,%20Sun%20P,%20Yang%20H,%20Liu%20P,%20Li%20Z,%20Xia%20Y,%20Jiang%20W.%20Neutrophil%20extracellular%20traps%20induced%20by%20IL8%20promote%20diffuse%20large%20B-cell%20lymphoma%20progression%20via%20the%20TLR9%20signaling.%20Clin%20Cancer%20Res.%202019;25(6):1867%E2%80%9379.%2010.1158/1078-0432.CCR-18-1226." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Tohme S, Yazdani HO, Al-Khafaji AB, Chidi AP, Loughran P, Mowen K, Wang Y, Simmons RL, Huang H, Tsung A. Neutrophil extracellular traps promote the development and progression of liver metastases after surgical stress. Cancer Res. 2016;76(6):1367–80. 10.1158/0008-5472.CAN-15-1591.
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-15-1591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4794393/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26759232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tohme%20S,%20Yazdani%20HO,%20Al-Khafaji%20AB,%20Chidi%20AP,%20Loughran%20P,%20Mowen%20K,%20Wang%20Y,%20Simmons%20RL,%20Huang%20H,%20Tsung%20A.%20Neutrophil%20extracellular%20traps%20promote%20the%20development%20and%20progression%20of%20liver%20metastases%20after%20surgical%20stress.%20Cancer%20Res.%202016;76(6):1367%E2%80%9380.%2010.1158/0008-5472.CAN-15-1591." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Stehr AM, Wang G, Demmler R, Stemmler MP, Krug J, Tripal P, Schmid B, Geppert CI, Hartmann A, Munoz LE, Schoen J, Volkl S, Merkel S, Becker C, Schett G, Grutzmann R, Naschberger E, Herrmann M, Sturzl M. Neutrophil extracellular traps drive epithelial-mesenchymal transition of human colon cancer. J Pathol. 2022;256(4):455–67. 10.1002/path.5860.
</cite> [<a href="https://doi.org/10.1002/path.5860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34939675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stehr%20AM,%20Wang%20G,%20Demmler%20R,%20Stemmler%20MP,%20Krug%20J,%20Tripal%20P,%20Schmid%20B,%20Geppert%20CI,%20Hartmann%20A,%20Munoz%20LE,%20Schoen%20J,%20Volkl%20S,%20Merkel%20S,%20Becker%20C,%20Schett%20G,%20Grutzmann%20R,%20Naschberger%20E,%20Herrmann%20M,%20Sturzl%20M.%20Neutrophil%20extracellular%20traps%20drive%20epithelial-mesenchymal%20transition%20of%20human%20colon%20cancer.%20J%20Pathol.%202022;256(4):455%E2%80%9367.%2010.1002/path.5860." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Jin W, Yin H, Li H, Yu XJ, Xu HX, Liu L. Neutrophil extracellular DNA traps promote pancreatic cancer cells migration and invasion by activating EGFR/ERK pathway. J Cell Mol Med. 2021;25(12):5443–56. 10.1111/jcmm.16555.
</cite> [<a href="https://doi.org/10.1111/jcmm.16555" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8184670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33955688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin%20W,%20Yin%20H,%20Li%20H,%20Yu%20XJ,%20Xu%20HX,%20Liu%20L.%20Neutrophil%20extracellular%20DNA%20traps%20promote%20pancreatic%20cancer%20cells%20migration%20and%20invasion%20by%20activating%20EGFR/ERK%20pathway.%20J%20Cell%20Mol%20Med.%202021;25(12):5443%E2%80%9356.%2010.1111/jcmm.16555." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Le DT, Durham JN, Smith KN, Wang H, Bartlett BR, Aulakh LK, Lu S, Kemberling H, Wilt C, Luber BS, Wong F, Azad NS, Rucki AA, Laheru D, Donehower R, Zaheer A, Fisher GA, Crocenzi TS, Lee JJ, Greten TF, Duffy AG, Ciombor KK, Eyring AD, Lam BH, Joe A, Kang SP, Holdhoff M, Danilova L, Cope L, Meyer C, Zhou S, Goldberg RM, Armstrong DK, Bever KM, Fader AN, Taube J, Housseau F, Spetzler D, Xiao N, Pardoll DM, Papadopoulos N, Kinzler KW, Eshleman JR, Vogelstein B, Anders RA, Diaz LJ. Mismatch repair deficiency predicts response of solid tumors to PD-1 Blockade. Science. 2017;357(6349):409–13. 10.1126/science.aan6733.
</cite> [<a href="https://doi.org/10.1126/science.aan6733" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5576142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28596308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Le%20DT,%20Durham%20JN,%20Smith%20KN,%20Wang%20H,%20Bartlett%20BR,%20Aulakh%20LK,%20Lu%20S,%20Kemberling%20H,%20Wilt%20C,%20Luber%20BS,%20Wong%20F,%20Azad%20NS,%20Rucki%20AA,%20Laheru%20D,%20Donehower%20R,%20Zaheer%20A,%20Fisher%20GA,%20Crocenzi%20TS,%20Lee%20JJ,%20Greten%20TF,%20Duffy%20AG,%20Ciombor%20KK,%20Eyring%20AD,%20Lam%20BH,%20Joe%20A,%20Kang%20SP,%20Holdhoff%20M,%20Danilova%20L,%20Cope%20L,%20Meyer%20C,%20Zhou%20S,%20Goldberg%20RM,%20Armstrong%20DK,%20Bever%20KM,%20Fader%20AN,%20Taube%20J,%20Housseau%20F,%20Spetzler%20D,%20Xiao%20N,%20Pardoll%20DM,%20Papadopoulos%20N,%20Kinzler%20KW,%20Eshleman%20JR,%20Vogelstein%20B,%20Anders%20RA,%20Diaz%20LJ.%20Mismatch%20repair%20deficiency%20predicts%20response%20of%20solid%20tumors%20to%20PD-1%20Blockade.%20Science.%202017;357(6349):409%E2%80%9313.%2010.1126/science.aan6733." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Ganesh K, Stadler ZK, Cercek A, Mendelsohn RB, Shia J, Segal NH, Diaz LJ. Immunotherapy in colorectal cancer: rationale, challenges and potential. Nat Rev Gastro Hepat. 2019;16(6):361–75. 10.1038/s41575-019-0126-x.</cite> [<a href="https://doi.org/10.1038/s41575-019-0126-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7295073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30886395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ganesh%20K,%20Stadler%20ZK,%20Cercek%20A,%20Mendelsohn%20RB,%20Shia%20J,%20Segal%20NH,%20Diaz%20LJ.%20Immunotherapy%20in%20colorectal%20cancer:%20rationale,%20challenges%20and%20potential.%20Nat%20Rev%20Gastro%20Hepat.%202019;16(6):361%E2%80%9375.%2010.1038/s41575-019-0126-x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Nagai M, Vo NH, Shin OL, Chimmanamada D, Inoue T, Chu J, Beaudette-Zlatanova BC, Lu R, Blackman RK, Barsoum J, Koya K, Wada Y. The oncology drug elesclomol selectively transports copper to the mitochondria to induce oxidative stress in cancer cells. Free Radical Bio Med. 2012;52(10):2142–50. 10.1016/j.freeradbiomed.2012.03.017.
</cite> [<a href="https://doi.org/10.1016/j.freeradbiomed.2012.03.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22542443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nagai%20M,%20Vo%20NH,%20Shin%20OL,%20Chimmanamada%20D,%20Inoue%20T,%20Chu%20J,%20Beaudette-Zlatanova%20BC,%20Lu%20R,%20Blackman%20RK,%20Barsoum%20J,%20Koya%20K,%20Wada%20Y.%20The%20oncology%20drug%20elesclomol%20selectively%20transports%20copper%20to%20the%20mitochondria%20to%20induce%20oxidative%20stress%20in%20cancer%20cells.%20Free%20Radical%20Bio%20Med.%202012;52(10):2142%E2%80%9350.%2010.1016/j.freeradbiomed.2012.03.017." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Gao W, Huang Z, Duan J, Nice EC, Lin J, Huang C. Elesclomol induces copper-dependent ferroptosis in colorectal cancer cells via degradation of ATP7A. Mol Oncol. 2021;15(12):3527–44. 10.1002/1878-0261.13079.
</cite> [<a href="https://doi.org/10.1002/1878-0261.13079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8637554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34390123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20W,%20Huang%20Z,%20Duan%20J,%20Nice%20EC,%20Lin%20J,%20Huang%20C.%20Elesclomol%20induces%20copper-dependent%20ferroptosis%20in%20colorectal%20cancer%20cells%20via%20degradation%20of%20ATP7A.%20Mol%20Oncol.%202021;15(12):3527%E2%80%9344.%2010.1002/1878-0261.13079." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12364797/bin/12672_2025_3183_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (353.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12364797/bin/12672_2025_3183_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (30.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12364797/bin/12672_2025_3183_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (1.1MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data presented in the study are presented in the manuscript and Additional files. Further inquiries of datasets are available directed to the corresponding author.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Discover Oncology are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1007/s12672-025-03183-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12672_2025_Article_3183.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12364797/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12364797/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12364797%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364797/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12364797/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12364797/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830698/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12364797/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830698/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12364797/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12364797/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="sqo8a4vZnAD7gqzkADCvTW4f5LFSSjb5VtXh3LiWQlh21h7R45hBJlyRtq0gcNZF">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
