
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Exploring the space of self-reproducing ribozymes using generative models - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E484428AEF08F3058442005D1A6C0D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373953/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Exploring the space of self-reproducing ribozymes using generative models">
<meta name="citation_author" content="Camille N Lambert">
<meta name="citation_author_institution" content="Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France">
<meta name="citation_author" content="Vaitea Opuu">
<meta name="citation_author_institution" content="Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France">
<meta name="citation_author_institution" content="Max Planck Institute for Mathematics in the Sciences, Leipzig, Germany">
<meta name="citation_author" content="Francesco Calvanese">
<meta name="citation_author_institution" content="Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France">
<meta name="citation_author_institution" content="Sorbonne Université, CNRS, Laboratoire de Biologie Computationnelle, Quantitative et Synthétique, UMR 7238, Paris, France">
<meta name="citation_author" content="Polina Pavlinova">
<meta name="citation_author_institution" content="Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France">
<meta name="citation_author" content="Francesco Zamponi">
<meta name="citation_author_institution" content="Dipartimento di Fisica, Sapienza Università di Roma, Rome, Italy">
<meta name="citation_author" content="Eric J Hayden">
<meta name="citation_author_institution" content="Biomolecular Sciences Graduate Programs, Boise State University, Boise, ID USA">
<meta name="citation_author_institution" content="Department of Biological Sciences, Boise State University, Boise, ID USA">
<meta name="citation_author" content="Martin Weigt">
<meta name="citation_author_institution" content="Sorbonne Université, CNRS, Laboratoire de Biologie Computationnelle, Quantitative et Synthétique, UMR 7238, Paris, France">
<meta name="citation_author" content="Matteo Smerlak">
<meta name="citation_author_institution" content="Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France">
<meta name="citation_author_institution" content="Max Planck Institute for Mathematics in the Sciences, Leipzig, Germany">
<meta name="citation_author_institution" content="Capital Fund Management, Paris, France">
<meta name="citation_author" content="Philippe Nghe">
<meta name="citation_author_institution" content="Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7836">
<meta name="citation_doi" content="10.1038/s41467-025-63151-5">
<meta name="citation_pmid" content="40846705">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373953/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373953/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373953/pdf/41467_2025_Article_63151.pdf">
<meta name="description" content="Estimating the plausibility of RNA self-reproduction is central to origin-of-life scenarios. However, this property has been shown in only a handful of catalytic RNAs. Here, we compare models for their generative power in diversifying a reference ...">
<meta name="og:title" content="Exploring the space of self-reproducing ribozymes using generative models">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Estimating the plausibility of RNA self-reproduction is central to origin-of-life scenarios. However, this property has been shown in only a handful of catalytic RNAs. Here, we compare models for their generative power in diversifying a reference ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373953/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373953">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63151-5"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63151.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373953%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373953/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373953/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373953/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 22;16:7836. doi: <a href="https://doi.org/10.1038/s41467-025-63151-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63151-5</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Exploring the space of self-reproducing ribozymes using generative models</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lambert%20CN%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Camille N Lambert</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Camille N Lambert</span></h3>
<div class="p">
<sup>1</sup>Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lambert%20CN%22%5BAuthor%5D" class="usa-link"><span class="name western">Camille N Lambert</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Opuu%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Vaitea Opuu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Vaitea Opuu</span></h3>
<div class="p">
<sup>1</sup>Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France </div>
<div class="p">
<sup>2</sup>Max Planck Institute for Mathematics in the Sciences, Leipzig, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Opuu%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vaitea Opuu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calvanese%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Francesco Calvanese</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Francesco Calvanese</span></h3>
<div class="p">
<sup>1</sup>Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France </div>
<div class="p">
<sup>3</sup>Sorbonne Université, CNRS, Laboratoire de Biologie Computationnelle, Quantitative et Synthétique, UMR 7238, Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calvanese%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Francesco Calvanese</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pavlinova%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Polina Pavlinova</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Polina Pavlinova</span></h3>
<div class="p">
<sup>1</sup>Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pavlinova%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Polina Pavlinova</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zamponi%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Francesco Zamponi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Francesco Zamponi</span></h3>
<div class="p">
<sup>4</sup>Dipartimento di Fisica, Sapienza Università di Roma, Rome, Italy </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zamponi%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Francesco Zamponi</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hayden%20EJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Eric J Hayden</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Eric J Hayden</span></h3>
<div class="p">
<sup>5</sup>Biomolecular Sciences Graduate Programs, Boise State University, Boise, ID USA </div>
<div class="p">
<sup>6</sup>Department of Biological Sciences, Boise State University, Boise, ID USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hayden%20EJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Eric J Hayden</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weigt%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Martin Weigt</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Martin Weigt</span></h3>
<div class="p">
<sup>3</sup>Sorbonne Université, CNRS, Laboratoire de Biologie Computationnelle, Quantitative et Synthétique, UMR 7238, Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weigt%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Weigt</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smerlak%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Matteo Smerlak</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Matteo Smerlak</span></h3>
<div class="p">
<sup>1</sup>Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France </div>
<div class="p">
<sup>2</sup>Max Planck Institute for Mathematics in the Sciences, Leipzig, Germany </div>
<div class="p">
<sup>7</sup>Capital Fund Management, Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smerlak%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Matteo Smerlak</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nghe%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Philippe Nghe</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Philippe Nghe</span></h3>
<div class="p">
<sup>1</sup>Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nghe%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Philippe Nghe</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Laboratoire de Biophysique et Evolution, UMR CNRS-ESPCI 8231 Chimie Biologie Innovation, ESPCI Paris, Université PSL, Paris, France </div>
<div id="Aff2">
<sup>2</sup>Max Planck Institute for Mathematics in the Sciences, Leipzig, Germany </div>
<div id="Aff3">
<sup>3</sup>Sorbonne Université, CNRS, Laboratoire de Biologie Computationnelle, Quantitative et Synthétique, UMR 7238, Paris, France </div>
<div id="Aff4">
<sup>4</sup>Dipartimento di Fisica, Sapienza Università di Roma, Rome, Italy </div>
<div id="Aff5">
<sup>5</sup>Biomolecular Sciences Graduate Programs, Boise State University, Boise, ID USA </div>
<div id="Aff6">
<sup>6</sup>Department of Biological Sciences, Boise State University, Boise, ID USA </div>
<div id="Aff7">
<sup>7</sup>Capital Fund Management, Paris, France </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 21; Accepted 2025 Aug 12; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373953  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846705/" class="usa-link">40846705</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Estimating the plausibility of RNA self-reproduction is central to origin-of-life scenarios. However, this property has been shown in only a handful of catalytic RNAs. Here, we compare models for their generative power in diversifying a reference ribozyme, based on statistical covariation and secondary structure prediction, and experimentally test model predictions using high-throughput sequencing. Leveraging statistical physics methods, we compute the number of ribozymes capable of autocatalytic self-reproduction from oligonucleotide fragments to be over 10<sup>39</sup>, with sequences found up to 65 mutations from the original sequence and 99 mutations away from each other, far beyond the 10 mutations achieved by deep mutational scanning. The findings demonstrate an efficient method for exploring RNA sequence space, and provide quantitative data on self-reproducing RNA that further illuminates the potential pathways to abiogenesis.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Origin of life, High-throughput screening</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">The spontaneous emergence of autocatalytic RNAs is central to origin of life. Here, the authors use machine learning, high-throughput screening and statistical physics to explore large neutral space of catalytic RNAs.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The RNA world hypothesis surmises that early evolving systems consisted of reactions between RNAs catalyzed by RNAs<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. In this scenario, the first ribozymes (catalytic RNAs) would have arisen by chance from random polymerization, possibly helped by covalent assembly reactions between partially complementary RNAs<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Among those, certain ribozymes could catalyze their own production, a process called autocatalysis<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, which has been tested experimentally. For instance, ligase ribozymes selected in vitro were shown to catalyze the joining of two RNA oligonucleotides, such that the product forms an additional copy of the ribozyme<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. In another system, naturally occurring self-splicing group I intron RNAs were engineered to make copies of themselves by splicing together four oligonucleotide substrates<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. It is envisioned that a gradual process of evolution could have led from autocatalytic RNAs<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> to template-based replication of RNA catalyzed by polymerase ribozymes<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, thereby initiating a mode of evolution similar to biological evolution. While some ribozymes relevant to the origin of life still need to be discovered, another important question is whether there exists a diversity of sequences that can carry out a given activity among all possible sequences (the sequence space), an ensemble called the <em>neutral network</em>, or <em>neutral set</em><sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. In particular, the larger the diversity of self-reproducing RNAs, the more likely are transitions between self-reproducing systems, thus enabling primordial modes of evolution<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<p id="Par4">One challenge to studying the neutral set of ribozymes is the extremely large number of sequences that need to be tested to determine which ones can perform a specific function. A strategy is to diversify RNAs known to have the desired property, and assess how many of the resulting sequences conserve this property. Neutral sets of RNA have been explored computationally using secondary structure prediction algorithms<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, and experimentally using deep mutational scanning (random mutagenesis combined with a functional assay)<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. The latter studies demonstrated the existence of connected neutral sets in the neighborhood of reference sequences, but remain limited to a few mutations away from it. More recently, machine learning approaches were combined with experimental testing to reach larger diversification, mutagenizing a 20-nucleotide-long region of a ribozyme<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> and reproducing the diversity of naturally occurring ribozymes<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. However, despite the progress made in the case of protein design<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, generative models still perform poorly for larger RNAs that are the size of the autocatalytic ligase and group I introns<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Furthermore, predicting catalysis remains difficult in general, and none of these studies have characterized the neutral set for autocatalytic RNA self-reproduction.</p>
<p id="Par5">Yet, the two autocatalytic RNAs mentioned above have been slightly diversified to create several sequences that collectively reproduce through cross-reactivity networks<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. In these experiments, sequence changes were made primarily to the substrate-binding regions of the ribozyme to change interactions between sequences and cause them to make copies of other sequences more efficiently than their own sequence. However, the majority of the ribozyme sequence outside of the substrate-binding sites remained unchanged in these experiments, and the vast number of sequences capable of these types of reactions remains unknown, limiting our understanding of this type of RNA reproduction. Analysis of evolutionary conservation and structure in naturally occurring self-splicing group I introns (Fig. <a href="#Fig1" class="usa-link">1a</a>) suggests a large potential neutral set that has not yet been experimentally evaluated<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Here, we devised generative probabilistic models based on statistical learning and structure prediction (Fig. <a href="#Fig1" class="usa-link">1b</a>), validated with a high-throughput catalytic assay that serves as a proxy for autocatalytic reproduction (Fig. <a href="#Fig1" class="usa-link">1c, d</a>). This combination of prediction and experimentation enabled the exploration of a very large neutral space of catalytic RNAs derived from Group I Introns, together with estimations of its size. The results demonstrate the potential of this type of generative model and apply it to better understand the vast number of sequences that could contribute to the origin of life.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Computational and experimental workflow for the design of artificial group I introns.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373953_41467_2025_63151_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/db90/12373953/75908fa6c320/41467_2025_63151_Fig1_HTML.jpg" loading="lazy" id="d33e422" height="528" width="800" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>
<em>Azoarcus</em> ribozyme secondary structure. More conserved positions among homologous sequences of the Multiple Sequence Alignment appear in darker red. <strong>b</strong> RNAs were designed using evolutionary models (DCA, VAE, PRO), structural models (SB, SB-3D, BPR), and a combination thereof (DCA-SB). The input of evolutionary models is a Multiple Sequence Alignment. The input of structural models is ab initio predictions of secondary structure or experimental 3D contacts of Azo. Designs are generated based on probabilistic scores and/or structure scores, depending on the model. <strong>c</strong> Assay mimicking self-splicing: candidate ribozymes (black) are synthesized with an exon sequence (yellow); RNAs are mixed and incubated with the gray and red substrates; In Step 1, the ribozyme covalently attaches its 3′ yellow to the gray substrate; Then, the product, consisting of the covalently joined gray and yellow sequences, is exchanged for the red substrate; In step 2, the ribozyme ligates the red substrate onto its 3′ extremity. <strong>d</strong> Screening workflow: after computational generation (<strong>b</strong>), the ribozymes are transcribed from a DNA pool, and tested by the screening assay (<strong>c</strong>); after screening, the active ribozymes are amplified with a substrate complementary primer before being sequenced; the frequency of active sequences (carrying S2) post-assay is normalized by the variant frequency pre-assay.</p></figcaption></figure></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">High-throughput assay for self-splicing</h3>
<p id="Par6">To assess the generative power of computational models, we developed a deep sequencing assay, with which we analyzed 24,220 unique RNA sequences (Fig. <a href="#Fig1" class="usa-link">1c, d</a>, “High-throughput self-splicing assay” section in Methods). This assay mimics the two steps of the natural self-splicing activity of Group I Introns, from which autocatalytic self-reproduction has been engineered<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. RNA candidates are first synthesized together with a tRNA fragment exon at their 3′ end (yellow in Fig. <a href="#Fig1" class="usa-link">1c</a>). The RNA libraries are then incubated together with two substrate RNAs (gray and red in Fig. <a href="#Fig1" class="usa-link">1c</a>) at 37 °C. As a proxy for autocatalysis, the RNA is considered active if it transfers its exon to a first substrate, exchanges it for a second substrate, and attaches the extremity of the latter to its own 3′ end (plain red in Fig. <a href="#Fig1" class="usa-link">1c</a>). Active RNAs are thus distinguished from inactive ones by deep sequencing, based on the fragment carried at their 3′ end (Fig. <a href="#Fig1" class="usa-link">1d</a>). For each variant, we defined the activity score as the logarithm of the fraction of active sequences after screening, divided by the fraction of sequences before incubation, setting as zero the <em>Azoarcus</em> ribozyme reference score (Fig. <a href="#Fig2" class="usa-link">2a</a>). The assay was highly reproducible with a Pearson correlation of 0.99 between independent triplicates (<em>p</em> &lt; 10<sup>−</sup><sup>5</sup>) (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). As the reaction consists of attaching fragments to the catalysts themselves via specific binding to substrates, cross-catalysis is expected to be negligible, which is supported by the absence of activity for a large number of mutations (see below). Additionally, we verified <em>a posteriori</em> that cross-catalysis negligibly affected the activity score by assaying variants separately by denaturing gel electrophoresis (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>) and sequencing in separate batches (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>), finding that band intensities significantly correlated with sequencing scores (Pearson correlation of 0.61, <em>p</em>-value = 0.008, <em>N</em> = 17). Consistently, mutants that displayed no visible product on the gel had a sequencing score below the activity threshold (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). We furthermore tested that incubation in sub-pools yielded the same sequencing scores as within the total pool (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>, “Cross-catalysis tests” section).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Comparison of the generative power of computational models.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373953_41467_2025_63151_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/db90/12373953/a65fa33c057f/41467_2025_63151_Fig2_HTML.jpg" loading="lazy" id="d33e544" height="654" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Experimental activity as a function of the number of mutations relative to Azo, by bins of 5 mutations (<em>N</em> = 150 per bin). Top: Statistical learning and hybrid models; Bottom: structure-based models. Per bin, dots are the mean activity, and quartile bars correspond to the first and third quartiles. <em>N</em> per bin per model provided in Source Data file. The red dashed line is the active threshold set at a z-score of 3.09 or equivalently a <em>p</em>-value = 10<sup>−</sup><sup>3</sup>, which corresponds to an activity of −2.76. <strong>b</strong> Fraction of designs that were catalytically active for each model (Active Fraction) as a function of the number of mutations, by bins of 5 mutations. Dots are the active fraction. Error bars upper (lower) bound is the active fraction including (excluding) activity scores within the 98.5 percentile of the measurement error distribution around the threshold. <em>N</em> per bin per model is provided in the Source Data file. <strong>c</strong> Effective support size of models as a function of mutational distance, showing how many different sequences the model can generate at any given distance. The star indicates the estimation at <em>L</em><sub>max</sub>. The star values have been corrected with the experimental active fraction at <em>L</em><sub>max</sub>, which is of order 1%. Inset: zoom on the experimental correction at <em>L</em><sub>max</sub> for DCA <em>T</em> = 1. <strong>d</strong> Principal Component Analysis (PCA), with the first two principal components, PC1 and PC2, shown for the generated sequences (gray) overlayed with active sequences (color). All panels are projected on the same axis system, which is the first two principal components of chimeric sequences (analogs of natural Group I introns), with the <em>Azoarcus</em> sequence located at the origin. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par7">We next tested several models by evenly populating bins of 10 mutations up to 150 mutations away from the <em>Azoarcus</em> ribozyme (Fig. <a href="#Fig2" class="usa-link">2a</a>, 150 sequences or more per model per bin). All models displayed a loss of activity as more mutations were introduced in Azo, with the score reaching a lower plateau corresponding to inactive sequences (Fig. <a href="#Fig2" class="usa-link">2a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). For random uniform mutagenesis (denoted RUM, see Table <a href="#Tab1" class="usa-link">1</a> for model acronyms), activity was indetectable after 15 mutations, confirming previous studies<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. The measurement noise distribution was taken as the distribution of scores at more than 100 mutations overall all models (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>), and the threshold for significant activity was set at a z-score of 3.09 corresponding to an activity score of −2.76 (corresponding to 10<sup>−</sup><sup>2.76</sup> = 0,17% of the <em>Azoarcus</em> ribozyme activity) and a <em>p</em>-value &lt; 10<sup>−</sup><sup>3</sup> for strict positivity (“Activity threshold and active fraction” section). Figure <a href="#Fig2" class="usa-link">2b</a> displays the active fraction, defined as the fraction of variants from each model that are above the noise threshold, plotted as a function of the number of mutations introduced relative to the reference <em>Azoarcus</em> ribozyme. For each generative model, we report <em>L</em><sub>50</sub> as the number of mutations relative to <em>Azoarcus</em> that maintains an active fraction of 50% and <em>L</em><sub>max</sub> as the number of mutations that maintain 1% (Fig. <a href="#Fig2" class="usa-link">2</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>, ‘Definitions of <em>L</em><sub>50</sub> and <em>L</em><sub>max</sub>, significance levels’ in Methods). <em>L</em><sub>50</sub> can be interpreted as the mutational distance that can be accurately predicted by a given model, beyond which its predictive power rapidly decreases. <em>L</em><sub>max</sub> represents the maximum number of mutations that we could reliably assess given our experimental resolution per bin. Note that deeper sequencing of larger RNA pools may allow assessment of functional sequences at frequencies below 1%, which would lead to larger <em>L</em><sub>max</sub> values.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Acronym list of the computational models</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Acronym</th>
<th colspan="1" rowspan="1">Description</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>RUM</strong></td>
<td align="left" colspan="1" rowspan="1">Mutations are introduced by randomly selecting positions and assigning with uniform probability one nucleotide.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>BPR</strong></td>
<td align="left" colspan="1" rowspan="1">Mutations with the Base-Pair Replacement method are introduced following the WT structure: at paired positions, both nucleotides are mutated simultaneously into G–C, G–U, or A–U pairs; otherwise, mutations are random.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>BPR</strong> + <strong>3D</strong>
</td>
<td align="left" colspan="1" rowspan="1">Mutations are introduced as in BPR except positions involved in tertiary interactions (detected by analyzing the tertiary structure), which are kept unchanged.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>SB</strong></td>
<td align="left" colspan="1" rowspan="1">Mutations are introduced with the MCMC algorithm following the Structure-Based (SB) score, which penalizes mispairing under the thermodynamic folding model.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>SB-3D</strong></td>
<td align="left" colspan="1" rowspan="1">Mutations are introduced as in SB except positions involved in tertiary interactions (detected by analyzing the tertiary structure), which are kept unchanged.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>PRO</strong></td>
<td align="left" colspan="1" rowspan="1">Mutations are introduced by randomly selecting positions and assigning nucleotides according to the frequency of the mutation observed in the MSA.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>DCA T</strong> = <strong>1</strong>
</td>
<td align="left" colspan="1" rowspan="1">Mutations are introduced with the MCMC algorithm at <em>T</em> = 1 following the DCA model trained on the MSA of homologs, ensuring designs match single and pairwise frequencies observed in the MSA.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>DCA T</strong> = <strong>0.3</strong>
</td>
<td align="left" colspan="1" rowspan="1">Mutations are introduced with the MCMC algorithm at <em>T</em> = 0.3 following the DCA model, which concentrates the sampling onto the most likely sequences under the DCA model.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>DCA-SB</strong></td>
<td align="left" colspan="1" rowspan="1">Mutations are introduced with the MCMC algorithm at <em>T</em> = 0.3 following the DCA model combined with the SB score, optimizing simultaneously the DCA score and the folding into the WT fold.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>VAE</strong></td>
<td align="left" colspan="1" rowspan="1">Mutations are introduced with the neural network called Variational AutoEncoder (VAE), which is trained on the MSA of homologs. Similarly to DCA, it is trained to maximize the likelihood of the observed MSA.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>CHI</strong></td>
<td align="left" colspan="1" rowspan="1">Chimeric sequences are derived from natural sequences aligned to <em>Azoarcus</em> by replacing deletions with <em>Azoarcus</em> nucleotides, using only aligned positions to maintain the same length.</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Populating the neutral space with Direct Coupling Analysis</h3>
<p id="Par8">We first explored the neutral space covered by Group I Introns using Direct Coupling Analysis (DCA), which has been proven to generate in vivo neutral variants of protein enzymes<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. DCA is a statistical learning approach that accounts for nucleotide conservation and covariation via networks of pairwise couplings<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. It describes a probability distribution over the space of functional nucleotide sequences. Our model was trained on a Multiple Sequence Alignment of 815 Group I introns sequences that share the same domain composition and secondary structure as the <em>Azoarcus</em> ribozyme (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>). Although sequences of the alignment may differ in length due to deletions or insertions relative to Azo, nucleotide covariations can be learned at positions aligned with the <em>Azoarcus</em> ribozyme. Sequences of the same length as the <em>Azoarcus</em> ribozyme (197 nucleotides) were sampled by Markov Chain Monte Carlo up to 90 mutations. We used the sequencing-based assay to identify active variants at increasing mutational distance.</p>
<p id="Par9">We found that the DCA-generated sequences retained activity at higher mutational distances, with <em>L</em><sub>50</sub> = 20 and <em>L</em><sub>max</sub> = 60. This was a large improvement as compared to random mutations, for which <em>L</em><sub>50</sub> = 5 and <em>L</em><sub>max</sub> = 10 (Fig. <a href="#Fig2" class="usa-link">2b</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). We also compared the DCA-generated sequences to “chimeric sequences” (CHI), which were generated by adding increasing numbers of mutations to the <em>Azoarcus</em> ribozyme, but only using nucleotide diversity at positions found in other group I introns in our data set, while also removing insertions and deletions to keep a constant length (Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a>). These chimeric sequences never reached the 50% success rate per bin (Fig. <a href="#Fig2" class="usa-link">2b</a>). Overall, only 6% of them were active, possibly due to the disruption of base-pairs, where this percentage on average corresponds to 7 over the 59 base-pairs found in the <em>Azoarcus</em> ribozyme structure. From these comparisons, it is clear that the DCA model can generate active ribozymes at high mutational distances, far beyond what is obvious from random mutagenesis or a naive application of evolutionary conservation.</p>
<p id="Par10">We next set out to use this DCA model to estimate the size of the neutral set of this type of autocatalytic self-reproducing RNA. It is important to note that the number of potential autocatalytic self-reproducers Ω<sub>DCA</sub> predicted from the DCA model cannot be simply calculated by multiplying the total number of possible sequences at mutational distance L by the measured active fraction generated by DCA. Indeed, the DCA model generates predictions focused on a restricted subset of sequences <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e821"><mi>x</mi></math></span> as compared to random uniform mutagenesis, namely the sequences to which DCA assigns a high probability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e830"><mi>P</mi><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow></math></span>. Thus, the number of sequences among which the DCA model samples is, in practice, smaller than the total number of possible sequences. On the other hand, to restrict the size of space given the distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e844"><mi>P</mi></math></span> is not straightforward because any sequence may have a non-zero, however small, probability.</p>
<p id="Par11">In information theory, the “effective support size” is a term used to define the effective size of a sampling set<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. The effective support size can be defined for any probability distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e859"><mi>P</mi></math></span> over all sequences <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e868"><mi>x</mi></math></span> as Ω = exp(<em>S</em>), where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e880"><mi>S</mi><mo>=</mo><mo>−</mo><mo>∑</mo><mi>P</mi><mfenced close=")" open="("><mrow><mi>x</mi></mrow></mfenced><mi>log</mi><mi>P</mi><mrow><mo>(</mo><mrow><mi>x</mi></mrow><mo>)</mo></mrow></math></span> is the Shannon entropy, the sum being taken over all sequences <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e904"><mi>x</mi></math></span>. This definition relies on the following mathematical result: when sampling <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e913"><mi>N</mi></math></span> sequences <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e922"><mi>x</mi></math></span> from the probability distribution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e931"><mi>P</mi></math></span>, the sample probability behaves like <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e940"><msup><mrow><mfenced close=")" open="("><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mi mathvariant="normal">Ω</mi></mrow></mfrac></mrow></mfenced></mrow><mrow><mi>N</mi></mrow></msup></math></span> for large enough <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e960"><mi>N</mi></math></span> (see Supplementary Information for more technical details<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>). Intuitively, this means that sampling from the model follows the same statistics as if one were sampling uniformly from a subset of size Ω. In other words, the vast majority of the probability is concentrated in a subset of size Ω, interpreted as the effective number of different sequences that the model can generate.</p>
<p id="Par12">The full process consists of computing the effective number of sequences Ω(<em>L</em>) that can be generated by the model (e.g., DCA) at a mutational distance <em>L</em>, then multiplying it by the active fraction experimentally measured at <em>L</em>. The maximum of the number at the particular distance L constitutes a lower bound for the whole neutral-set size Ω. In practice, this maximum was determined as the largest <em>L</em> such that we could measure the active fraction to be significantly larger than zero. Although it is not generally known how to compute Ω for probabilistic or generative models, we devised a semi-analytical method exploiting the structure of DCA (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>, Supplementary Information “Support size computations”)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, which we applied at each <em>L</em> to obtain Ω(<em>L</em>). This theoretical number increases sub-exponentially with <em>L</em>, reaching 10<sup>41</sup> sequences at 60 mutations (Fig. <a href="#Fig2" class="usa-link">2c</a>). The success rate at <em>L</em><sub>max</sub> = 60 mutations was determined to be larger than 1% with <em>p</em> &lt; 4.10<sup>−5</sup> confidence. Thus, correcting Ω<sub>DCA</sub> with the experimental success rate leads to Ω<sub>DCA</sub> ≃ 10<sup>39</sup>.</p>
<p id="Par13">We next set out to further evaluate the diversity of the sequences generated by DCA. First, we used a Principal Component Analysis (PCA) projection of the DCA-generated sequences and chimeras. We found that the chimeras (representative of natural diversity) clustered near the <em>Azoarcus</em> ribozyme in these projections (Fig. <a href="#Fig2" class="usa-link">2d</a>), while the DCA-generated sequences bridge the sequence space between the <em>Azoarcus</em> ribozyme and natural chimeras, from the perspective of the first two principal components. We note that PC1 strongly correlates with the distance to the <em>Azoarcus</em> ribozyme (Pearson ρ = 0.86, two-sided <em>p</em>-value = 10<sup>−</sup><sup>16</sup> corresponding to the numerical precision, under the null hypothesis <em>ρ</em> = 0 using the Beta distribution). To further compare the variety of sequences generated by each model, we computed two different mutational distance distributions. First, we computed the distance from each DCA sequence to the closest chimera, including Azo. This can be interpreted as a measurement of how different the samples are from any natural sequences (Fig. <a href="#Fig3" class="usa-link">3a</a>, top). Second, we computed the distances between all pairs of sequences within each model (Fig. <a href="#Fig3" class="usa-link">3a</a>, bottom). Experimentally active DCA sequences were found to be on average 25 mutations away from each other (Fig. <a href="#Fig3" class="usa-link">3a</a>, bottom), on average 18 mutations away from any chimera, and up to 55 mutations away from any chimera (Fig. <a href="#Fig3" class="usa-link">3a</a>, top, Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). These distances were comparable to those found between all chimeras (active and non-active), which had 32 mutations between them on average and up to 59 mutations away from each other (Fig. <a href="#Fig3" class="usa-link">3a</a>, Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). It thus appears that DCA achieves a form of interpolation, by generating a diversity of experimentally active sequences similar to the diversity of natural Group I introns (Fig. <a href="#Fig3" class="usa-link">3b, c</a>), as well as a form of extrapolation, by generating active sequences as far away from any chimeras as the latter are from each other. Extrapolation is further confirmed by the largest distance between active DCA designs and chimeras being 99 mutations (Supplementary Table <a href="#MOESM1" class="usa-link">3</a>), well beyond the largest distance between any two chimeras.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Diversity explored by the generated sequences.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373953_41467_2025_63151_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/db90/12373953/829ed9ea3e15/41467_2025_63151_Fig3_HTML.jpg" loading="lazy" id="d33e1129" height="852" width="788" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Upper panel: Violin plots of minimal distance to the set of chimeric sequences for generated sequences (light shades) and active sequences (solid shades), per model, with identical colors as in Fig. <a href="#Fig2" class="usa-link">2b</a>; The average of distances for chimeric sequences is represented by a dashed line. Lower panel: Violin plot of distances between all pairs for generated sequences (light shades) and active sequences (solid shades), per model. <strong>b</strong> Mutational enrichment from chimeric sequences. Top: At each position (x-axis), <em>Azoarcus</em> ribozyme nucleotides are marked by a black dot, and shades are the log ratio of mutation frequencies relative to <em>Azoarcus</em> ribozyme in chimeras (green) and in active variants over all models (red). Bottom: effective number of nucleotides in chimera (green), candidate designs (black), and active designs (red). <strong>c</strong> Mutational enrichment due to the structure: Effective number of nucleotides per position in DCA <em>T</em> = 1 (blue) and DCA-SB (purple). <strong>d</strong> Top: PCA projection of DCA (left) and DCA-SB sequences (right), on the 2 principal components of the DCA <em>T</em> = 1 set. Active sequences are colored and inactive ones are gray. Bottom: same but in the principal components of the DCA-SB set, revealing an upper cluster (C<sup>u</sup>) populated by DCA-SB only, distinct from the cluster below (C<sup>d</sup>) that comprises all DCA sequences. Inset: Distribution of structure scores in the two clusters, showing that C<sup>u</sup> has a significantly better structure score (***, one-sided <em>t</em>-test, <em>p</em>-value = 10<sup>−</sup><sup>16</sup>, numerical precision). Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Generative power of statistical and structural methods</h3>
<p id="Par14">These initial results suggest a trade-off between extensively exploring all regions of genotype space and increasing the probability of successfully finding active sequences. For instance, random mutagenesis considers all possible mutations and would theoretically yield the highest possible Ω, but its success rate decreases very quickly below our ability to experimentally detect active fractions. In general, a less restricted model should eventually discover a larger total number of active sequences, but with a much lower success rate, leading to inefficient experiments. In contrast, more constrained models like DCA generate functional sequences with a higher number of mutations, but at the cost of exploring more limited regions of the sequence space. This further suggests that for a given model, different choices could be made to fine tune the model to achieve a desired outcome, such as identifying active sequences with a larger number of mutations (as quantified by <em>L</em><sub>50</sub> and <em>L</em><sub>max</sub>), or to yield a larger effective number of generated sequences at a given distance (as quantified by Ω(<em>L</em>)). Overall, the estimation of the number of active sequences by calculating effective support size is not obvious because, for any given model, it depends on how Ω increases with the number of mutations as well as on the <em>L</em><sub>max</sub> that can be validated experimentally.</p>
<p id="Par15">To explore this trade-off, we tested more relaxed models, which we expected to access a larger diversity but at a lower mutational distance. The profile model (PRO) was a randomization approach where each nucleotide was replaced according to its position-specific frequency in the Multiple Sequence Alignment. It is less constrained compared to DCA, but still more constrained than random (RUM) because it utilizes some evolutionary information from nucleotide conservation. This PRO model was only marginally more accurate than RUM, with <em>L</em><sub>50</sub> = 5 (Fig. <a href="#Fig2" class="usa-link">2b</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>), <em>L</em><sub>max</sub> = 15, and Ω<sub>PRO</sub> = 10<sup>22</sup>. Recall that the success rate of random mutagenesis (RUM) was nearly identical to this with <em>L</em><sub>50</sub> = 5, <em>L</em><sub>max</sub> = 10, and Ω<sub>RUM</sub> = 10<sup>20</sup>, and these metrics were much lower for both models than for DCA (Fig. <a href="#Fig2" class="usa-link">2b</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). Limitations of the profile model likely result from a high probability of disrupting the secondary structure when selecting nucleotides independently, as 60% of <em>Azoarcus</em> ribozyme positions are base-paired (Fig. <a href="#Fig1" class="usa-link">1a</a>). To assess the impact of base pairing, we tested the random replacement of base pairs of <em>Azoarcus</em> ribozyme with other canonical pairs, combined with randomized nucleotides at unpaired positions, which we termed the Base-Pair Replacement model (BPR). This model performed significantly better than the profile model, with L<sub>50</sub> = 15, <em>L</em><sub>max</sub> = 20, and Ω<sub>BPR</sub> = 10<sup>29</sup>. Further constraining the model by forbidding variations at tertiary contacts (30 positions in Azo, Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>, BPR-3D model) better populated the tail of the distribution of active RNAs, with <em>L</em><sub>50</sub> = 15 and <em>L</em><sub>max</sub> = 35. Ultimately, Ω<sub>BPR-3D</sub> = 10<sup>37</sup>, which was still smaller but nearing Ω<sub>DCA</sub>.</p>
<p id="Par16">Because the base-pair replacement model (BPR) improved prediction success at higher mutational distances, we next explored the impact of using secondary structure prediction. We generated sequences that were predicted to form the correct secondary structure from a thermodynamic model (SB model for ‘structure-based’)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>–<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, using a heuristic that retained the pseudo-knot as a probabilistic signature in the contact map (Supplementary Fig. <a href="#MOESM1" class="usa-link">6e</a>)<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>–<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. This model performed only slightly better than RUM, with <em>L</em><sub>50</sub> = 10 and <em>L</em><sub>max</sub> = 20. This limited performance could be due to discrepancies between the predicted and the actual structure (Supplementary Fig. <a href="#MOESM1" class="usa-link">6c</a>) or to structure energy minimization being non-optimal for catalysis (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). Fixing positions involved in 3D contacts<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> (SB-3D model) again improved the performance at higher distances, with <em>L</em><sub>50</sub> = 10 and <em>L</em><sub>max</sub> = 40. The higher <em>L</em><sub>max</sub> was beyond that of the BPR-3D model, suggesting that cooperative effects that determine structure stability in the SB model may make a cumulative difference, but only when critical tertiary interactions are unperturbed.</p>
<p id="Par17">Despite the success of the DCA model, we considered that pairwise statistical covariation alone might miss the importance of higher-order interactions of three or more nucleotides. Such higher-order interactions are known to be important for group I intron ribozyme catalysis. For example, there are several base-triples in the “G-binding pocket” of the active site<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. To further explore more constrained models, we used Variational AutoEncoders<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> (VAE) to encode high-order interactions in a neural network and generated sequences from this model. Overall, VAE covered a similar diversity to DCA as shown by PCA projections (Fig. <a href="#Fig2" class="usa-link">2d</a>) and only a slightly lower capacity for mutagenesis, with <em>L</em><sub>50</sub> = 15 and <em>L</em><sub>max</sub> = 60. The absence of improvement compared to DCA may be explained by the modest number of training sequences (<em>N</em> = 816), which might be insufficient to resolve higher-order couplings<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>.</p>
<p id="Par18">We next set out to further improve the DCA-based models by introducing some constraints. We sampled sequences with higher probability according to DCA by lowering a parameter termed “sampling temperature”, which controls the degree of random diversification during Monte Carlo sampling, from <em>T</em> = 1 to <em>T</em> = 0.3. This alone showed a dramatic improvement in model success at higher mutational distances with <em>L</em><sub>50</sub> = 45 and <em>L</em><sub>max</sub> = 60. In a separate approach, we combined the simple secondary structure maintaining approach (SB) with DCA to generate the DCA-SB model, where DCA is sampled at <em>T</em> = 0.3. This model also showed a very high success rate at higher mutational distances, with <em>L</em><sub>50</sub> = 55 and <em>L</em><sub>max</sub> = 65, with a significantly higher success rate in the range 60–70 compared to DCA alone (two-sample <em>t</em>-test, <em>p</em> &lt; 10<sup>−</sup><sup>5</sup>). Interestingly, the DCA-SB sequences introduced diversity in the loops P5 and P6 (Figs. <a href="#Fig1" class="usa-link">1</a>a, <a href="#Fig3" class="usa-link">3c</a>), a diversity that was not found in DCA alone. The better exploration by DCA-SB was confirmed by PCA projections, where the DCA-SB sequences covered the DCA cluster plus an additional cluster that was characterized by a better structure score (Fig. <a href="#Fig3" class="usa-link">3d</a>).</p>
<p id="Par19">Focusing on the models that identified activity at higher mutational distances than the standard DCA, namely DCA <em>T</em> = 0.3 and DCA-SB, we computed the support size for these models over a limited range of mutational distances (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Over the explored range of mutations, Ω<sub>DCA T=0.3</sub> and Ω<sub>DCA-SB</sub> were found to be many orders of magnitude below Ω<sub>DCA</sub>, consistent with stronger design constraints (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). This result illustrates how models better for introducing many mutations at once are also more constrained, resulting in a reduction in the total effective number of sequences that can be discovered.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Robustness and generality of the approach</h3>
<p id="Par20">We have used a two-step splicing assay as a proxy for the self-reproduction from fragments because it uses the same catalytic mechanism. To confirm that the DCA-generated sequences can, in fact, self-reproduce from fragments, we chose individual sequences at varying distances from the <em>Azoarcus</em> ribozyme up to 60 mutations (Supplementary Material ‘Self-reproduction assay’). We split sequences into either two or four RNA fragments, allowed them to react, and monitored the appearance of the full-length ribozyme (Fig. <a href="#Fig4" class="usa-link">4</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>)<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. More than 60% of the sequences (<em>N</em> = 17) that had a score larger than the activity threshold were found to covalently self-reproduce from fragments. Importantly, we observed self-reproduction for DCA sequences with up to 60 mutational differences from the <em>Azoarcus</em> ribozyme. We note that the self-reproduction reaction can be sensitive to the structural context of the CAU tag sequences, and many more sequences might be able to self-reproduce with minor changes to the precise tag location. These results confirm that a majority of catalytically active group I intron ribozymes artificially generated by DCA are capable of covalent self-reproduction from fragments.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Evidence for self-reproduction.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373953_41467_2025_63151_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/db90/12373953/f61f15862c0d/41467_2025_63151_Fig4_HTML.jpg" loading="lazy" id="d33e1428" height="724" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Two-fragment self-reproduction experimental assay. The ribozymes are first fragmented into two pieces (WXY and Z) at position 145, then incubated at defined time points. The formation of longer covalent copies is monitored by gel electrophoresis. <strong>b</strong> Two-fragment self-reproduction assay for 5 designs and Azo, assayed for 0, 1, 6 and 24 h. The assayed ribozymes are documented in the table on the right of the gel: the number of mutations from the <em>Azoarcus</em> ribozyme is indicated in their name, and their activity score for the self-splicing assay in the last column. The molecular scale in nucleotides. The red arrows are pointing at the recombined covalent ribozymes. For other designed ribozymes with 15–60 mutations, see Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>. <strong>c</strong> Four-fragment self-reproduction assay for one design and Azo, assayed for 1, 2, 4 and 24 h. The dca.T03.19.1 design and <em>Azoarcus</em> ribozyme are first fragmented into four fragments (W, X, Y and Z) at the positions 62, 99 and 145. The molecular scale in nucleotides. Two conditions were assayed: the four fragments in the absence of the covalent ribozyme; and the four fragments in the presence of the covalent ribozyme (condition +). The red arrow shows the presence of the fully recombined covalent ribozymes. <strong>d</strong> The dca.T03.19.1 ribozyme. The colors represent the 4 different fragments, and the nucleotides colored with a darker shade highlight the position of mutations compared to the <em>Azoarcus</em> ribozyme.</p></figcaption></figure><p id="Par21">We next thought to strengthen our estimation of the size of the neutral set obtained with the DCA model. As mentioned, the estimation is based on the support size curve combined with the number of active sequences found at different mutational distances (Fig. <a href="#Fig2" class="usa-link">2</a>), and is sensitive to deviations at the largest mutational distances. To better establish our estimation of Ω<sub>DCA</sub>, we generated a DCA pool of 1000 sequences at 60 mutations and experimentally determined the active fraction. The results showed that 2.2% of the sequences were functional, consistent with our initial estimate. The larger sample size enhances the level of significance, leading to a <em>p</em>-value &lt; 10<sup>−</sup><sup>21</sup> for having active sequences in the 60-mutation bin (Fig. <a href="#Fig5" class="usa-link">5a</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Robustness and generality.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373953_41467_2025_63151_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/db90/12373953/71c67e20d3b4/41467_2025_63151_Fig5_HTML.jpg" loading="lazy" id="d33e1486" height="465" width="700" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Designed DCA sequences at 60 mutations from the WT. The histogram shows the activity scores of 991 detected designs (out of 1000 designs) with 22 active sequences beyond the activity threshold (equal to −2.76, red dotted line), corresponding to 2% of the pool. <strong>b</strong> Comparison of active fraction as a function of the number of mutations for the RUM and DCA-SB models, for different MgCl<sub>2</sub> concentrations: 60 mM as our standard condition (solid lines) and a lower concentration of 5 mM (dashed lines). <em>N</em> per bin per model provided in the Source Data file. Dots are the active fraction. Error bars upper (lower) bound is the active fraction including (excluding) activity scores within the 98.5 percentile of the measurement error distribution around the threshold. <strong>c</strong> Sequencing score at 37 °C versus 60 °C for the same DCA pool. Due to lower sequencing depth as compared to assays of Fig. <a href="#Fig2" class="usa-link">2</a>, a more stringent activity threshold of −1.5 was used (see “Methods” section). The Pearson correlation is 0.64, two-sided <em>p</em>-value = 10<sup>−</sup><sup>16</sup> (numerical precision). <em>N</em> = 558, comprising 484 active for both conditions (upper-right quadrant), 25 inactive for both (lower-left), 19 active at 60 °C but not 37 °C, 30 active at 37 °C but not 60 °C. <strong>d</strong> Active fraction as a function of the number of mutations for the DCA model at 37 °C (plain line) and 60 °C (dashed line). Sequences with a score below the activity threshold in at least one condition are in light color. Note that the fraction is computed over all designed sequences, including those that were not active enough to be detected by sequencing (thus counted as inactive). <em>N</em> per bin per model provided in the Source Data file. Dots are the active fraction. Error bars upper (lower) bound of vertical bars is the active fraction including (excluding) activity scores within the 98.5 percentile of the measurement error distribution around the threshold. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par22">To relate our findings to possible prebiotic scenarios, we assessed the impact of environmental conditions, magnesium concentration, and temperature, which are both known to affect structural stability and catalysis in RNA. While the experiments reported so far were done at 60 mM magnesium concentration, we used a 5 mM magnesium concentration to run the assay for the pools of the RUM and DCA-SB, the models that introduced the least and the most mutations, respectively. As expected, the active fraction decreased in both cases, consistently in each mutation bin (Fig. <a href="#Fig5" class="usa-link">5b</a>). The impact on the RUM was dramatic, with <em>L</em><sub>50</sub> = 0 and <em>L</em><sub>max</sub> = 5, due to initially low metrics. However, the maximum diversification was found to be robust for the DCA-SB model, with <em>L</em><sub>50</sub> = 30 and <em>L</em><sub>max</sub> = 60.</p>
<p id="Par23">Regarding the effect of temperature, the <em>Azoarcus</em> ribozyme strongly degrades above 70 °C (half-life shorter than 30 min). We thus tested the standard DCA model pool at 60 °C, below this upper limit, but well above the 37 °C of the initial test. Sequencing scores correlated well between the 2 conditions (<em>r</em> = 0.64, <em>p</em>-value &lt; 10<sup>−</sup><sup>5</sup>, Fig. <a href="#Fig5" class="usa-link">5c</a>). The region below the x = y diagonal appears more populated than above it (Fig. <a href="#Fig5" class="usa-link">5c</a>), confirming the expectation that a given sequence should have a better activity at 37 °C than at 60 °C. Qualitatively, a large majority of the mutants detected by sequencing were found to be above the activity threshold for both temperatures (484 over 558 detected sequences, those below the threshold or non-detected being considered inactive). Consistently, the active fraction as a function of the number of mutations was slightly lower at 60 °C than 37 °C, yet both temperatures followed the same trend (Fig. <a href="#Fig5" class="usa-link">5d</a>), leading to <em>L</em><sub>50</sub> = 15 and <em>L</em><sub>max</sub> = 45 at 60 °C (compared to <em>L</em><sub>50</sub> = 20 and <em>L</em><sub>max</sub> = 60 at 37 °C).</p>
<p id="Par24">Finally, we wondered if the results were specific to the <em>Azoarcus</em> ribozyme model system, or if it could be extrapolated to other ribozymes. First, we applied the DCA model using a group I intron from <em>Phormidium</em> sp. as a starting point for mutagenesis (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>, Supplementary Information ‘Phormidium’). We measured <em>L</em><sub>max</sub> = 25 for the DCA model and <em>L</em><sub>max</sub> = 35 for the DCA-SB model. Although the degree of diversification was lower than for <em>Azoarcus</em>, this indicates that the diversification approach can be generalized to extend the neutral set from other starting points. The next question is whether these findings might be applied to ribozymes that differ by their activity, structure, or length. A recent analysis shows that the effective number of RNAs with a given activity within a family of structural analogs can be estimated, according to DCA, using a power law <em>d</em> <sup>L</sup>, where <em>L</em> is the length of the RNA and d is an effective nucleotide diversity per position found to be 1.74<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Following this analysis, we computationally examined three other ribozyme families with diverse lengths: Hammerhead (66 nucleotides), glmS (167 nucleotides), and Bacterial RNase P class A (367 nucleotides). We observed comparable trend of the support size versus number of mutations, which furthermore matched the scaling law of support size versus length formerly found on a large number of naturally occurring RNAs<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>). These results confirm the generality of generative approaches for functional RNAs, also found by a deep learning method<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, and indicates a rule of thumb to extrapolate our DCA-based estimation of diversity to other ribozymes.</p></section></section><section id="Sec7"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par25">The mapping of molecular function to sequence space is important for understanding many biological questions, ranging from biotechnology applications to the origin of life. Sequence space increases exponentially with the length of a genetic sequence (DNA, RNA, or protein), and this presents several challenges for studying larger RNA molecules that are important models for origins of life research. For the 197-nucleotide-long <em>Azoarcus</em> group I intron RNA studied here, there exist 10<sup>118</sup> possible sequences that make up the sequence space. It is not possible to synthesize all of these RNA sequences, and we must therefore develop methods to maximize information gained from experiments. Here, we demonstrated an approach that allows an efficient exploration of this sequence space to map genotypes capable of an autocatalytic self-reproduction reaction. By comparing several statistical models and reporting the consistency of their predictive power over thousands of measurements each, we provided a method for future applications of this approach to other systems.</p>
<p id="Par26">Based on the best-performing DCA model and the measured activity of randomly sampled sequences, we suggest that there exist over 10<sup>39</sup> autocatalytic self-reproducers that make up the neutral set for this function. This estimation is based on our experimental detection limit and therefore represents a lower bound. While this is a lower bound of potentially autocatalytic sequences, it provides a quantitative evaluation of the sequence space to guide future questions and research. At the moment, the frequency of other ribozyme activities may be estimated from randomization techniques. Ligase ribozymes, from which replicases were derived, were selected from random pools of size 10<sup>15</sup> sequences<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. The diversification potential of aminoacylation ribozymes, relevant to the origin of translation, was explored by mutagenesis in a subregion of length 21 nucleotides, showing that 10<sup>5</sup> sequences were active among the 4.10<sup>12</sup> possible ones<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. In general, these studies and the current work suggest that ribozymes are abundant in sequence space, but suggest that different structures and functions likely have different frequencies.</p>
<p id="Par27">The Direct Coupling Analysis approach can be applied to additional RNA systems to further advance origins of life research or other RNA research. We found that diversification was equally efficient when starting from different group I intron ribozymes (<em>Azoarcus</em> and Phormidium ribozymes). Smaller group I intron analogs are also known to exist, such as a 140-nucleotide-long ribozyme derived from a shortened SunY ribozyme. In the long term, generating RNAs with the same activity but much less sequence identity or without length constraints should be possible, and has been demonstrated in proteins<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Beyond group I intron analogs, relaxing the structural constraints would further expand the range of RNAs capable of self-reproducing. For instance, the family of group II intron ribozymes is structurally distinct but also catalyzes splicing reactions between RNAs. Further, smaller and simpler instances of autocatalytic RNAs exist, but have not been extensively characterized experimentally. For instance, engineered autocatalytic ligase ribozymes are 67 nucleotides long, overall relying on an enzyme-substrate complex of 134 nucleotides<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Even smaller ribozymes were recently found to emerge from random mixtures of activated oligomers, with lengths as short as 20 nucleotides<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>, relying on 40-nucleotide complexes. By studying reproducing ribozymes of different sizes and structures, it should be possible to determine scaling laws relating ribozyme length and diversity, and allow estimates of the size and connectedness of the neutral sets<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. We started with only 815 unique sequences to train our models, suggesting that any RNA structure could be similarly studied with only minimal known natural examples, or by generating a good initial data set experimentally. We note that the choice of training data is important, but the optimal criteria might change for different sequences and structures. The methods for selecting training data remain an active area of research in itself<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>.</p>
<p id="Par28">Autocatalytic RNAs should also be considered in the larger picture of prebiotic environments, which would likely alter the activity of certain sequences and impact the mapping of genotype to function<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Certain prebiotic processes could have actually facilitated the production or preservation of autocatalytic RNA. For instance, interactions with the mineral surfaces can favor structured RNAs that are prone to catalysis<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Peptide-RNA interactions have been shown to form coacervate structures that can improve RNA functions<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. On the other hand, the prebiotic milieu could be constraining. In particular, the concentrations of different metal ions can impact ribozyme activity. Lower magnesium concentration tends to reduce the fraction of active molecules, although we found that magnesium concentration in the millimolar range had a limited impact on our identification of active sequences.</p>
<p id="Par29">Generative approaches, as developed in this work, could uncover the full extent of RNA sequences capable of achieving functions relevant to the RNA world. The highly active developments in artificial intelligence, including large language models, for RNA structure<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> and function<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> hold the promise to systematically design RNAs and ultimately reveal the variety of accessible pathways for prebiotic evolution.</p></section><section id="Sec8"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec9"><h3 class="pmc_sec_title">High-throughput self-splicing assay</h3>
<p id="Par30">The assay aims at discriminating against an activity similar to self-splicing, the reaction catalyzed by the wild-type <em>Azoarcus</em> GII. For the natural self-splicing reaction, the GII binds a free GTP, allowing the attack of A. Then, the 3′ of A attacks the 3′ of the GII to allow the formation of a covalent bond between the two exons, while releasing the GII.</p>
<p id="Par31">To mimic this self-splicing mechanism, we devised an experimental assay comprising two steps, as shown in Fig. <a href="#Fig1" class="usa-link">1c</a>. All ribozymes were produced with a 15-nt-long sequence at their 3′ extremity, called sequence A (yellow region in Fig. <a href="#Fig1" class="usa-link">1c</a>), which corresponds to the wild-type exon of <em>Azoarcus</em> GII. First, the sequence A is transferred at the end of the substrate called S1 (grey region in Fig. <a href="#Fig1" class="usa-link">1c</a>). Then, a second substrate called B-S2 (red substrate in Fig. <a href="#Fig1" class="usa-link">1c</a>) binds into the recognition site of the RNA, leading to the transfer of a cleaved section S2 of B-S2 at the 3′ end of the ribozyme and the release of a smaller B fragment (fraction of the red substrate that remains attached to the 5′ GUG extremity of the ribozyme, right-most panel of Fig. <a href="#Fig1" class="usa-link">1c</a>). The ribozymes with the sequence S2 (fraction of the red substrate attached to the 3′ extremity of the ribozyme, right-most panel of Fig. <a href="#Fig1" class="usa-link">1c</a>), thus considered active, were selected during the later steps.</p>
<p id="Par32">We performed three large pooled experiments, wherein we assayed thousands of sequences simultaneously. For each computationally designed pool of RNA sequences, we ordered the corresponding DNA templates with the exon at the 3′ end and the T7 promoter in the 5′ end. The single-strand DNA oligo pools containing 12,000 or 18,000 sequences were purchased from Twist Bioscience. We amplified the DNA pools by PCR (~15 cycles) with the KAPA HiFi HotStart ReadyMix (Roche), and purified the samples with the NucleoSpin Gel and PCR cleanup (Macherey Nagel). The DNA templates were then transcribed using the HiScribe T7 High Yield RNA Synthesis Kit (New England Biolabs) for 4 h at 37 °C to produce RNA molecules. The samples were then subjected to phenol-chloroform extraction and ethanol precipitation using 0.1 volume of 3 M Sodium Acetate (Sigma) and 2.5 volumes of cold 100% ethanol. After extraction, the samples were treated with the DNAse I (New England Biolabs) and PAGE purified on an 8% urea PAGE.</p>
<p id="Par33">For each designed RNA pool, we performed two sub-experiments. The first sub-experiment is the self-splicing assay. The second one is a control experiment to correct for the biases of the relative quantity of each synthesized ribozyme within the corresponding pool (before reaction), used to compute the activity scores. For the self-splicing assay, 2 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e1729"><mi mathvariant="normal">μM</mi></math></span> of ribozymes were incubated with 25 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e1739"><mi mathvariant="normal">μM</mi></math></span> of substrates S1 and B-S2 in a buffer (30 mM EPPS, pH 7.5, 60 mM MgCl2) at 37 °C in a final volume of 20 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e1749"><mi mathvariant="normal">μL</mi></math></span>. Two samples were taken during the incubation, at 0 and 60 min, mixed with loading solution (70% formamide, 130 mM EDTA, 0.1% xylene cyanol, 0.1% bromophenol blue), and loaded on an 8% Urea PAGE. The reaction was quenched by adding 60 mM of EDTA, and the ribozymes were cleaned using the Monarch RNA cleanup kit (New England Biolabs) with an adjusted volume of ethanol and binding buffer. For the control experiment, no substrate was used, and the ribozyme pool was directly cleaned with the Monarch RNA cleanup kit.</p>
<p id="Par34">The RNA samples were then prepared for sequencing, using the NEBNext Ultra II Directional RNA Library Prep Kit for Illumina (New England Biolabs). For the self-splicing experiment, the primer used during the reverse transcription corresponds to the sequence complementary to the S2 substrate, whereas the sequence complementary to sequence A was used for the control experiment. The samples were sequenced on a NovaSeq SP flow cell (2∗250 nt, 2∗800 M reads) in paired ends and with 25% of PhiX by the NGS platform at Institut du Cerveau et de la Moelle épinière (ICM, Paris).</p>
<p id="Par35">To test the robustness of our experimental protocol across the 3 pools, we introduced a common set of 355 sequences, which showed a very consistent activity computation across all pools (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e1763"><mi>ρ</mi><mo>=</mo></math></span> 0.99 correlation between each pair of pools, Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>).</p></section><section id="Sec10"><h3 class="pmc_sec_title">Cross-catalysis tests</h3>
<p id="Par36">Cross-catalysis within a pool would result in false positives (apparently active ribozymes that are in reality inactive), which in turn could lead to overestimation of the lower bound on the number of ribozymes. To test for this, we performed a series of experiments ensuring that cross-catalysis was negligible.</p>
<p id="Par37">First, we assayed 24 sequences by denaturing gel electrophoresis, individually (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). We used identical reaction conditions as the pooled high-throughput assay: 2 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e1785"><mi mathvariant="normal">μM</mi></math></span> of ribozymes incubated with 25 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e1795"><mi mathvariant="normal">μM</mi></math></span> substrates at 37 °C for 1 h. These sequences were selected among the pool designed by DCA to cover the range of mutations 15–60 from Azo, with various scores of the assay as measured by sequencing. Overall, all the sequences that had a sequencing score above the threshold also displayed a form of catalytic activity visible on the gel. 1 in the 24 sequences was a false negative according to sequencing: it displayed products on the gel but a sequencing score below the threshold. However, neither false negatives impact our lower bound estimation of the number of ribozymes, nor do they indicate the presence of  cross-catalysis as the latter results in false positives.</p>
<p id="Par38">Second, we measured by the sequencing assay the activity score of a set of sequences spanning a range of activity scores, comparing the score obtained in a pool and as separate individual measurements. The sequences were produced, purified, and tested separately. The scores reproduced well the relative experimental activity measured in the pool, with ρ = 0.95 with <em>p</em>-value &lt; 10<sup>−</sup><sup>5</sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3b</a>).</p>
<p id="Par39">We further tested that active ribozymes did not affect the measured activity of each other within a pool as a function of their relative concentration, by taking a subset of 12 only active sequences, tested as a subpool. The activity score of this subset incubated as a separate, smaller pool correlated strongly with the scores within the total pool, with <em>ρ</em> = 0.98 with <em>p</em>-value &lt; 10<sup>−</sup><sup>5</sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3c</a>), showing that the overall concentration of active variants did not affect the activity score.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Self-reproduction assay</h3>
<p id="Par40">The fragmentation strategy previously reported for fragmenting the <em>Azoarcus</em> ribozyme<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> was generalized to the artificial ribozymes. We fragmented the ribozymes at the nucleotide positions 145–147, which we referred to as the Y/Z junction. This fragmentation site is located in the loop of the P8 paired region (Fig. <a href="#Fig4" class="usa-link">4a</a>). For each ribozyme, two fragments were produced with the addition of short sequence fragments necessary for the recombination reaction to occur. For the first fragment (WXY fragment), a ‘CAU’ tag was inserted at the 3′-end, and for the second fragment (Z fragment), a ‘GGCAU’ tag was inserted at the 5′-end by PCR.</p>
<p id="Par41">The self-reproduction assays were carried out as follows. 1 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e1846"><mi mathvariant="normal">μM</mi></math></span> of each ribozyme fragment was incubated in a buffer (30 mM EPPS, pH 7.5, 60 mM MgCl2) at 37 °C. Samples were taken at different time points during the incubation, mixed with loading solution (70% formamide, 130 mM EDTA, 0.1% xylene cyanol, 0.1% bromophenol blue), and loaded on an 8% Urea PAGE. The gels were stained with SyBr Gold (ThermoFisher) diluted in TBE buffer.</p>
<p id="Par42">To fragment a ribozyme into 4 fragments<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> (W, X, Y Z fragments, Fig. <a href="#Fig4" class="usa-link">4d</a>), the ‘CAU’ tag was inserted at the 3′-end of the fragments W, X and Y and the ‘GGCAU’ tag was inserted at the 5′-end of the fragments X, Y and Z. Similarly, the self-reproduction reaction was performed by incubating 1 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e1865"><mi mathvariant="normal">μM</mi></math></span> of each 4 fragments in the incubation buffer (30 mM EPPS, pH 7.5, 60 mM MgCl2) at 37 °C. The course of the reaction was followed by analyzing samples on an 8% Urea PAGE.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Construction of the MSA</h3>
<p id="Par43">To build the MSA, we used the wild-type sequence (197 nucleotides long) of <em>Azoarcus</em> GI Intron and its known secondary structure derived from the PDB X-ray structure 1G9B<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup> in order to detect homologous sequences from the RFAM database<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. To account for similarity but also secondary structure compatibility, we built a seed alignment containing only the <em>Azoarcus</em> sequence and its secondary structure, which we fed to the alignment algorithm implemented in the package Infernal (version 1.1.4)<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> to search for homologs in the 2611 sequences of the RF00028 RFAM family. We filtered out all the sequences for which the computed e-value is &gt;10<sup>−</sup><sup>3</sup>, which is the probability of finding the obtained alignment score randomly. 816 sequences were found below the homology threshold of 10<sup>−</sup><sup>3</sup> and aligned using Infernal. Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a> shows the diversity per position in this alignment, measured as the exponential of Shannon entropy, and the frequency of gaps per nucleotide position.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Calculation of activities from sequencing results</h3>
<p id="Par44">To estimate the experimental activities, we computed the frequencies of designed sequences in: (i) the reference condition before the catalytic reaction, and (ii) the reacted condition where the substrate has been mixed with the designs and then incubated. For both conditions, we mapped each paired-end read to the closest designed sequence using the software Blast (version 2.12)<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Then, we selected the reads that covered at least 70% of the mapped designed sequence with full identity, which allowed us to obtain the occurrence of each design in the sample. We computed the frequencies <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e1915"><msub><mrow><mi>f</mi></mrow><mrow><mi>r</mi><mi>e</mi><mi>f</mi></mrow></msub></math></span> of designs in each pool before catalysis, allowing us to quantify the bias in RNA molecules in the initially synthesized pool. Then, we computed the frequencies of designs <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e1930"><msub><mrow><mi>f</mi></mrow><mrow><mi>s</mi><mi>e</mi><mi>l</mi></mrow></msub></math></span> in the sample with the substrate. To do so, we counted the reads when the substrate was attached right after the 3′ end of the design, which indicates that the ribozyme was able to excise the exon and then ligate the substrate. Finally, we computed the experimental activity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e1945"><mi>a</mi><mi>c</mi><mi>t</mi><mo>=</mo><mi>log</mi><mfenced close=")" open="("><mrow><mfrac><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>s</mi><mi>e</mi><mi>l</mi></mrow></msub></mrow><mrow><msub><mrow><mi>f</mi></mrow><mrow><mi>r</mi><mi>e</mi><mi>f</mi></mrow></msub></mrow></mfrac></mrow></mfenced></math></span>. Analyzing reverse reads was sufficient for calculating the activity score. Sequences such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e1977"><msub><mrow><mi>f</mi></mrow><mrow><mi>r</mi><mi>e</mi><mi>f</mi></mrow></msub></math></span> &gt; 0 but <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e1993"><msub><mrow><mi>f</mi></mrow><mrow><mi>s</mi><mi>e</mi><mi>l</mi></mrow></msub></math></span> = 0 were considered inactive. Sequences such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e2008"><msub><mrow><mi>f</mi></mrow><mrow><mi>r</mi><mi>e</mi><mi>f</mi></mrow></msub></math></span> &lt; 5 were excluded from the analysis.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Activity threshold and active fraction</h3>
<p id="Par45">As depicted in Fig. <a href="#Fig2" class="usa-link">2a</a>, as more mutations are introduced, the activity score decreases from zero (the Azo score reference) to a lower plateau. This plateau is reached for all models beyond 70 mutations, characteristic of non-functional sequences. We considered the distribution of scores at this plateau representative of the experimental noise, taking the score distribution over the 543 sequences comprising at least 100 mutations (red in Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). The noise distribution is found to be Gaussian (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). Significant activity was taken for a <em>p</em>-value &lt; 10<sup>−</sup><sup>3</sup>, resulting in a threshold of −2.76. Note that this p-value guarantees a z-score larger than 3.09 (3 standard deviations above the mean). For reference, activities greater than −2.45 corresponded to <em>p</em>-value &lt; 10<sup>−</sup><sup>4</sup>, and activities greater than −2.16 to <em>p</em>-values≪10<sup>−</sup><sup>5</sup>. Those more stringent thresholds are verified for most of the active sequences found in the <em>L</em><sub>max</sub> bin, and the presence of multiple sequences above the threshold further lowers the level of evidence, thus lowering the p-value (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). In the case of incubation at different temperatures, we used a more stringent threshold of −1.5 to account for the increased noise level due to shallower sequencing, determined using the same reference pool measured at different sequencing depths. For each model, we computed the active fraction per bin spanning 5 mutations of distance from Azo. For each bin, the active fraction is defined as the number of designs with activity levels above the threshold divided by the total number of designs in that bin.</p>
<p id="Par46">The error bars on the bins' active fraction are estimated using the error on the individual sequence activity measurements. For this, we used data from 355 overlapping sequences, whose activity has been independently measured three times. As already pointed out, these measurements displayed high consistency, with a correlation coefficient close to 0.99 across different experiments. For each of these 355 sequences, we calculated the standard deviation of the activity measurements. The vast majority of the standard deviations are below 0.3 (with just 5/355 designs exceeding this value). Consequently, we considered 0.3 as the error on the single activity measurement. Using this error estimate, we computed the error bars for the active fraction in each bin by counting the number of designs whose activity values crossed the threshold, considering the ±0.3 measurement error.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Definitions of <em>L</em><sub>50</sub> and <em>L</em><sub>max</sub>, significance levels</h3>
<p id="Par47"><em>L</em><sub>50</sub> was estimated as the largest number of mutations such that a given model achieved at least 50% active sequences within a bin of 5 mutations with <em>p</em>-value &lt; 10<sup>−</sup><sup>3</sup> (binomial test, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). <em>L</em><sub>max</sub> was estimated as the largest number of mutations such that a given model achieved at least 1% success rate within a bin of 5 mutations with <em>p</em>-value &lt; 10<sup>−</sup><sup>3</sup> (binomial test, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>).</p>
<p id="Par48">However, as the <em>L</em><sub>max</sub> bin generally comprises more than 1 sequence, and with scores clearly above the threshold, we computed and reported the p-value given all the available data, rejecting the hypothesis that all sequence activities in this bin are drawn from the noise, as explained below and reported in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>. To compute this actual <em>p</em>-value for <em>L</em><sub>max</sub>, we assumed a Gaussian null model for the measured activities <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e2121"><mi>a</mi></math></span> of non-functional sequences, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e2130"><mi>a</mi><mo>~</mo><mi>N</mi><mrow><mo>(</mo><mrow><mi>μ</mi><mo>,</mo><mi>σ</mi></mrow><mo>)</mo></mrow></math></span>, which we call the noise model. As described above in the “Activity threshold and active fraction” section, the model mean <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e2148"><mi>μ</mi></math></span> and standard deviation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e2157"><mi>σ</mi></math></span> are estimated from a sample of 543 non-functional sequences having at least 100 mutations. We assume here that all measured activity is due to experimental noise.</p>
<p id="Par49">Assume now a sample of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e2168"><mi>N</mi></math></span> sequences <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e2177"><mrow><mo>{</mo><mrow><msub><mrow><mi>s</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>.</mo><mo>.</mo><mo>.</mo><mo>,</mo><mi>N</mi></mrow><mo>}</mo></mrow></math></span> with experimentally measured activities <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e2204"><mrow><mo>{</mo><mrow><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>.</mo><mo>.</mo><mo>.</mo><mo>,</mo><mi>N</mi></mrow><mo>}</mo></mrow></math></span> (typically the sequences of any bin in mutational distance to Azo). For any arbitrary <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e2231"><mi>z</mi></math></span>-value, we can find the number <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e2240"><mi>n</mi><mrow><mo>(</mo><mrow><mi>z</mi></mrow><mo>)</mo></mrow><mo>=</mo><mrow><mo>{</mo><mrow><mi>i</mi><mo>,</mo><msub><mrow><mi>a</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>&gt;</mo><mi>μ</mi><mo>+</mo><mi>z</mi><mi>σ</mi></mrow><mo>}</mo></mrow><mo>∨</mo></math></span> of sequences with measured activity being at least <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e2273"><mi>z</mi></math></span> standard deviations above the mean of the noise model. We can also calculate the probability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e2282"><mi>π</mi><mrow><mo>(</mo><mrow><mi>z</mi></mrow><mo>)</mo></mrow><mo>=</mo><mi>P</mi><mrow><mo>(</mo><mrow><mi>a</mi><mo>&gt;</mo><mi>μ</mi><mo>+</mo><mi>z</mi><mi>σ</mi></mrow><mo>)</mo></mrow></math></span> that a randomly chosen activity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e2308"><mi>a</mi><mo>~</mo><mi>N</mi><mrow><mo>(</mo><mrow><mi>μ</mi><mo>,</mo><mi>σ</mi></mrow><mo>)</mo></mrow></math></span> from the noise model is beyond that activity threshold.</p>
<p id="Par50">These two numbers, via a one-sided binomial test, give access to a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e2328"><mi>z</mi></math></span>-dependent <em>p</em>-value (which allows to refuse or not the hypothesis that all data are drawn from the null model):</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><math id="d33e2340" display="block"><mi>p</mi><mo>=</mo><msubsup><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>i</mi><mo>=</mo><mi>n</mi><mrow><mo>(</mo><mrow><mi>z</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>N</mi></mrow></msubsup><mfrac><mrow><mi>N</mi><mo>!</mo></mrow><mrow><mi>i</mi><mo>!</mo><mrow><mo>(</mo><mrow><mi>N</mi><mo>−</mo><mi>i</mi></mrow><mo>)</mo></mrow><mo>!</mo></mrow></mfrac><msup><mrow><mi>π</mi><mrow><mo>(</mo><mrow><mi>z</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>i</mi></mrow></msup><mspace width="0.25em"></mspace><msup><mrow><mfenced close="]" open="["><mrow><mn>1</mn><mo>−</mo><mi>π</mi><mrow><mo>(</mo><mrow><mi>z</mi></mrow><mo>)</mo></mrow></mrow></mfenced></mrow><mrow><mi>N</mi><mo>−</mo><mi>i</mi></mrow></msup></math></td></tr></table>
<p id="Par51">In coherence with our selection threshold chosen according to the Method section “Activity threshold and active fraction”, we report <em>p</em>-values for sequences above our standard threshold, i.e., for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e2411"><mi>π</mi><mrow><mo>(</mo><mrow><mi>z</mi></mrow><mo>)</mo></mrow><mo>=</mo><mn>0.001</mn></math></span>, reached at. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e2427"><mi>z</mi><mo>≃</mo><mn>3.09</mn></math></span>.</p>
<p id="Par52">In principle, any <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e2440"><mi>z</mi></math></span>-value could be used for calculating <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e2449"><mi>p</mi></math></span>-values. For smaller <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e2458"><mi>z</mi></math></span>, we would have more super-threshold activities (larger <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e2467"><mi>n</mi><mrow><mo>(</mo><mrow><mi>z</mi></mrow><mo>)</mo></mrow></math></span>) but of smaller individual significance (smaller <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e2481"><mi>π</mi><mrow><mo>(</mo><mrow><mi>z</mi></mrow><mo>)</mo></mrow></math></span>), while a larger <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e2496"><mi>z</mi></math></span> would lead to fewer selected sequences of higher individual significance. Note that fixing our activity threshold (and thus considering one value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e2505"><mi>z</mi></math></span>) leads to an upper bound of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e2514"><mi>p</mi></math></span>-values.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par53">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373953/bin/41467_2025_63151_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373953/bin/41467_2025_63151_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (6.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373953/bin/41467_2025_63151_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.3MB, pdf) </sup>
</div></div></section></section><section id="Sec18"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12373953/bin/41467_2025_63151_MOESM4_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (1.8MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors acknowledge Vincent Messow, Nono S. C. Merleau, and Owen Terpstra for preliminary work, Sandeep Ameta, and Michal Matyjasik for discussions. The authors acknowledge the following funding sources: Human Frontier Science Program RGY0077 / 2019 (E.H., M.S., P.N.), Institut Pierre-Gilles de Gennes ANR-10-EQPX-34 (P.N.), -EU H2020 Grant ERC AbioEvo 101002075 (P.N.), EU H2020 grant MSCA-RISE In-ferNet 734439 (M.W.).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: E.H., M.W., M.S., P.N. Experimental Methodology: C.L., P.P., E.H., P.N. Computational Methodology: V.O., F.C., F.Z., M.W., M.S. Investigation: C.L., V.O., F.C., P.P. Funding acquisition: E.H., M.W., M.S., P.N. Project administration: P.N. Supervision: F.Z., E.H., M.W., M.S., P.N. Writing – original draft: C.L., V.O., P.N. Writing – review &amp; editing: C.L., V.O., F.C., F.Z., E.H., M.W., M.S., P.N.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par54"><em>Nature Communications</em> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Source data for the figures are provided with this paper as a zip folder. Additional data generated in this study have been deposited in the Zenodo database under accession code 10.5281/zenodo.16531362. <a href="#Sec18" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>The code used in this study to generate and analyze sequences has been deposited in the Zenodo database under accession code 10.5281/zenodo.16531362</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par55">Authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Camille N. Lambert, Vaitea Opuu, Francesco Calvanese.</p></div>
</div></section><section id="sec19"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63151-5.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Gilbert, W. Origin of life: the RNA world. <em>Nature</em><strong>319</strong>, 618–618 (1986).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gilbert,%20W.%20Origin%20of%20life:%20the%20RNA%20world.%20Nature319,%20618%E2%80%93618%20(1986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Briones, C., Stich, M. &amp; Manrubia, S. C. The dawn of the RNA World: toward functional complexity through ligation of random RNA oligomers. <em>RNA</em><strong>15</strong>, 743–749 (2009).
</cite> [<a href="https://doi.org/10.1261/rna.1488609" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2673073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19318464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Briones,%20C.,%20Stich,%20M.%20&amp;%20Manrubia,%20S.%20C.%20The%20dawn%20of%20the%20RNA%20World:%20toward%20functional%20complexity%20through%20ligation%20of%20random%20RNA%20oligomers.%20RNA15,%20743%E2%80%93749%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Wu, L. et al. Selective nonenzymatic formation of biologically common RNA hairpins. <em>Angew. Chem.</em><strong>137</strong>, e202417370 (2025).</cite> [<a href="https://doi.org/10.1002/anie.202417370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11773311/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39568250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20L.%20et%20al.%20Selective%20nonenzymatic%20formation%20of%20biologically%20common%20RNA%20hairpins.%20Angew.%20Chem.137,%20e202417370%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Kauffman, S. A. Autocatalytic sets of proteins. <em>J. Theor. Biol.</em><strong>119</strong>, 1–24 (1986).
</cite> [<a href="https://doi.org/10.1016/s0022-5193(86)80047-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3713221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kauffman,%20S.%20A.%20Autocatalytic%20sets%20of%20proteins.%20J.%20Theor.%20Biol.119,%201%E2%80%9324%20(1986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Nghe, P. et al. Prebiotic network evolution: six key parameters. <em>Mol. Biosyst.</em><strong>11</strong>, 3206–3217 (2015).
</cite> [<a href="https://doi.org/10.1039/c5mb00593k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26490759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nghe,%20P.%20et%20al.%20Prebiotic%20network%20evolution:%20six%20key%20parameters.%20Mol.%20Biosyst.11,%203206%E2%80%933217%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Paul, N. &amp; Joyce, G. F. A self-replicating ligase ribozyme. <em>Proc. Natl. Acad. Sci. USA</em><strong>99</strong>, 12733–12740 (2002).
</cite> [<a href="https://doi.org/10.1073/pnas.202471099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC130529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12239349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paul,%20N.%20&amp;%20Joyce,%20G.%20F.%20A%20self-replicating%20ligase%20ribozyme.%20Proc.%20Natl.%20Acad.%20Sci.%20USA99,%2012733%E2%80%9312740%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Robertson, M. P. &amp; Joyce, G. F. Highly efficient self-replicating RNA enzymes. <em>Chem. Biol.</em><strong>21</strong>, 238–245 (2014).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2013.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3943892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24388759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robertson,%20M.%20P.%20&amp;%20Joyce,%20G.%20F.%20Highly%20efficient%20self-replicating%20RNA%20enzymes.%20Chem.%20Biol.21,%20238%E2%80%93245%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Hayden, E. J. &amp; Lehman, N. Self-assembly of a group I intron from inactive oligonucleotide fragments. <em>Chem. Biol.</em><strong>13</strong>, 909–918 (2006).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2006.06.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16931340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hayden,%20E.%20J.%20&amp;%20Lehman,%20N.%20Self-assembly%20of%20a%20group%20I%20intron%20from%20inactive%20oligonucleotide%20fragments.%20Chem.%20Biol.13,%20909%E2%80%93918%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Pavlinova, P., Lambert, C. N., Malaterre, C. &amp; Nghe, P. Abiogenesis through gradual evolution of autocatalysis into template-based replication. <em>FEBS Lett.</em><strong>597</strong>, 344–379 (2023).
</cite> [<a href="https://doi.org/10.1002/1873-3468.14507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36203246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pavlinova,%20P.,%20Lambert,%20C.%20N.,%20Malaterre,%20C.%20&amp;%20Nghe,%20P.%20Abiogenesis%20through%20gradual%20evolution%20of%20autocatalysis%20into%20template-based%20replication.%20FEBS%20Lett.597,%20344%E2%80%93379%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Attwater, J., Raguram, A., Morgunov, A. S., Gianni, E. &amp; Holliger, P. Ribozyme-catalysed RNA synthesis using triplet building blocks. <em>eLife</em><strong>7</strong>, 35255 (2018).</cite> [<a href="https://doi.org/10.7554/eLife.35255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6003772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29759114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Attwater,%20J.,%20Raguram,%20A.,%20Morgunov,%20A.%20S.,%20Gianni,%20E.%20&amp;%20Holliger,%20P.%20Ribozyme-catalysed%20RNA%20synthesis%20using%20triplet%20building%20blocks.%20eLife7,%2035255%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Papastavrou, N., Horning, D. P. &amp; Joyce, G. F. RNA-catalyzed evolution of catalytic RNA. <em>Proc. Natl. Acad. Sci. USA</em><strong>121</strong>, 2321592121 (2024).</cite> [<a href="https://doi.org/10.1073/pnas.2321592121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10945747/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38437533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Papastavrou,%20N.,%20Horning,%20D.%20P.%20&amp;%20Joyce,%20G.%20F.%20RNA-catalyzed%20evolution%20of%20catalytic%20RNA.%20Proc.%20Natl.%20Acad.%20Sci.%20USA121,%202321592121%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Fontana, W. et al. RNA folding and combinatory landscapes. <em>Phys. Rev. E</em><strong>47</strong>, 2083–2099 (1993).</cite> [<a href="https://doi.org/10.1103/physreve.47.2083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9960229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fontana,%20W.%20et%20al.%20RNA%20folding%20and%20combinatory%20landscapes.%20Phys.%20Rev.%20E47,%202083%E2%80%932099%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Nghe, P. A stepwise emergence of evolution in the RNA world. <em>FEBS Lett.</em>10.1002/1873-3468.70065 (2025).</cite> [<a href="https://doi.org/10.1002/1873-3468.70065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40353364/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Schuster, P., Fontana, W., Stadler, P. F. &amp; Hofacker, I. L. From sequences to shapes and back: a case study in RNA secondary structures. <em>Proc. R. Soc. Lond. Ser. B Biol. Sci.</em><strong>255</strong>, 279–284 (1994).</cite> [<a href="https://doi.org/10.1098/rspb.1994.0040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7517565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schuster,%20P.,%20Fontana,%20W.,%20Stadler,%20P.%20F.%20&amp;%20Hofacker,%20I.%20L.%20From%20sequences%20to%20shapes%20and%20back:%20a%20case%20study%20in%20RNA%20secondary%20structures.%20Proc.%20R.%20Soc.%20Lond.%20Ser.%20B%20Biol.%20Sci.255,%20279%E2%80%93284%20(1994)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Fontana, W. &amp; Schuster, P. Shaping space: the possible and the attainable in RNA genotype-phenotype mapping. <em>J. Theor. Biol.</em><strong>194</strong>, 491–515 (1998).
</cite> [<a href="https://doi.org/10.1006/jtbi.1998.0771" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9790826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fontana,%20W.%20&amp;%20Schuster,%20P.%20Shaping%20space:%20the%20possible%20and%20the%20attainable%20in%20RNA%20genotype-phenotype%20mapping.%20J.%20Theor.%20Biol.194,%20491%E2%80%93515%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Jiménez, J. I., Xulvi-Brunet, R., Campbell, G. W., Turk-MacLeod, R. &amp; Chen, I. A. Comprehensive experimental fitness landscape and evolutionary network for small RNA. <em>Proc. Natl. Acad. Sci. USA</em><strong>110</strong>, 14984–14989 (2013).
</cite> [<a href="https://doi.org/10.1073/pnas.1307604110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3773786/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23980164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jim%C3%A9nez,%20J.%20I.,%20Xulvi-Brunet,%20R.,%20Campbell,%20G.%20W.,%20Turk-MacLeod,%20R.%20&amp;%20Chen,%20I.%20A.%20Comprehensive%20experimental%20fitness%20landscape%20and%20evolutionary%20network%20for%20small%20RNA.%20Proc.%20Natl.%20Acad.%20Sci.%20USA110,%2014984%E2%80%9314989%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Kobori, S. &amp; Yokobayashi, Y. High-Throughput mutational analysis of a twister ribozyme. <em>Angew. Chem. Int. Ed.</em><strong>55</strong>, 10354–10357 (2016).</cite> [<a href="https://doi.org/10.1002/anie.201605470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5113685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27461281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kobori,%20S.%20&amp;%20Yokobayashi,%20Y.%20High-Throughput%20mutational%20analysis%20of%20a%20twister%20ribozyme.%20Angew.%20Chem.%20Int.%20Ed.55,%2010354%E2%80%9310357%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Puchta, O. et al. Molecular evolution: network of epistatic interactions within a yeast snoRNA. <em>Science</em><strong>352</strong>, 840–844 (2016).
</cite> [<a href="https://doi.org/10.1126/science.aaf0965" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5137784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27080103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Puchta,%20O.%20et%20al.%20Molecular%20evolution:%20network%20of%20epistatic%20interactions%20within%20a%20yeast%20snoRNA.%20Science352,%20840%E2%80%93844%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Li, C., Qian, W., Maclean, C. J. &amp; Zhang, J. The fitness landscape of a tRNA gene. <em>Science</em><strong>352</strong>, 837–840 (2016).
</cite> [<a href="https://doi.org/10.1126/science.aae0568" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4894649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27080104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20C.,%20Qian,%20W.,%20Maclean,%20C.%20J.%20&amp;%20Zhang,%20J.%20The%20fitness%20landscape%20of%20a%20tRNA%20gene.%20Science352,%20837%E2%80%93840%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Domingo, J., Diss, G. &amp; Lehner, B. Pairwise and higher-order genetic interactions during the evolution of a tRNA. <em>Nature</em><strong>558</strong>, 117–121 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0170-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6193533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29849145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Domingo,%20J.,%20Diss,%20G.%20&amp;%20Lehner,%20B.%20Pairwise%20and%20higher-order%20genetic%20interactions%20during%20the%20evolution%20of%20a%20tRNA.%20Nature558,%20117%E2%80%93121%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Hayden, E. J., Bendixsen, D. P. &amp; Wagner, A. Intramolecular phenotypic capacitance in a modular RNA molecule. <em>Proc. Natl. Acad. Sci. USA</em><strong>112</strong>, 12444–12449 (2015).
</cite> [<a href="https://doi.org/10.1073/pnas.1420902112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4603511/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26401020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hayden,%20E.%20J.,%20Bendixsen,%20D.%20P.%20&amp;%20Wagner,%20A.%20Intramolecular%20phenotypic%20capacitance%20in%20a%20modular%20RNA%20molecule.%20Proc.%20Natl.%20Acad.%20Sci.%20USA112,%2012444%E2%80%9312449%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Pressman, A. D. et al. Mapping a systematic ribozyme fitness landscape reveals a frustrated evolutionary network for self-aminoacylating RNA. <em>J. Am. Chem. Soc.</em><strong>141</strong>, 6213–6223 (2019).
</cite> [<a href="https://doi.org/10.1021/jacs.8b13298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6548421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30912655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pressman,%20A.%20D.%20et%20al.%20Mapping%20a%20systematic%20ribozyme%20fitness%20landscape%20reveals%20a%20frustrated%20evolutionary%20network%20for%20self-aminoacylating%20RNA.%20J.%20Am.%20Chem.%20Soc.141,%206213%E2%80%936223%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Andreasson, J. O., Savinov, A., Block, S. M. &amp; Greenleaf, W. J. Comprehensive sequence-to-function mapping of cofactor-dependent RNA catalysis in the glmS ribozyme. <em>Nat. Commun.</em><strong>11</strong>, 1663 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-15540-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7125110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32245964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andreasson,%20J.%20O.,%20Savinov,%20A.,%20Block,%20S.%20M.%20&amp;%20Greenleaf,%20W.%20J.%20Comprehensive%20sequence-to-function%20mapping%20of%20cofactor-dependent%20RNA%20catalysis%20in%20the%20glmS%20ribozyme.%20Nat.%20Commun.11,%201663%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Roberts, J. M., Beck, J. D., Pollock, T. B., Bendixsen, D. P. &amp; Hayden, E. J. RNA sequence to structure analysis from comprehensive pairwise mutagenesis of multiple self-cleaving ribozymes. <em>eLife</em><strong>12</strong>, 80360 (2023).</cite> [<a href="https://doi.org/10.7554/eLife.80360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9901934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36655987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roberts,%20J.%20M.,%20Beck,%20J.%20D.,%20Pollock,%20T.%20B.,%20Bendixsen,%20D.%20P.%20&amp;%20Hayden,%20E.%20J.%20RNA%20sequence%20to%20structure%20analysis%20from%20comprehensive%20pairwise%20mutagenesis%20of%20multiple%20self-cleaving%20ribozymes.%20eLife12,%2080360%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Rotrattanadumrong, R. &amp; Yokobayashi, Y. Experimental exploration of a ribozyme neutral network using evolutionary algorithm and deep learning. <em>Nat. Commun.</em><strong>13</strong>, 4847 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-32538-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9385714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35977956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rotrattanadumrong,%20R.%20&amp;%20Yokobayashi,%20Y.%20Experimental%20exploration%20of%20a%20ribozyme%20neutral%20network%20using%20evolutionary%20algorithm%20and%20deep%20learning.%20Nat.%20Commun.13,%204847%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Sumi, S., Hamada, M. &amp; Saito, H. Deep generative design of RNA family sequences. <em>Nat. Methods</em><strong>21</strong>, 435–443 (2024).
</cite> [<a href="https://doi.org/10.1038/s41592-023-02148-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38238559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sumi,%20S.,%20Hamada,%20M.%20&amp;%20Saito,%20H.%20Deep%20generative%20design%20of%20RNA%20family%20sequences.%20Nat.%20Methods21,%20435%E2%80%93443%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Dauparas, J. et al. Robust deep learning–based protein sequence design using ProteinMPNN. <em>Science</em><strong>378</strong>, 49–56 (2022).
</cite> [<a href="https://doi.org/10.1126/science.add2187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9997061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36108050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dauparas,%20J.%20et%20al.%20Robust%20deep%20learning%E2%80%93based%20protein%20sequence%20design%20using%20ProteinMPNN.%20Science378,%2049%E2%80%9356%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Abramson, J. et al. Accurate structure prediction of biomolecular interactions with AlphaFold 3. <em>Nature</em><strong>630</strong>, 493–500 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abramson,%20J.%20et%20al.%20Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20AlphaFold%203.%20Nature630,%20493%E2%80%93500%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kretsch, R. C. et al. Assessment of nucleic acid structure prediction in CASP16. Preprint at <em>bioRxiv</em>10.1101/2025.05.06.652459 (2025).</cite>
</li>
<li id="CR30">
<span class="label">30.</span><cite>Vaidya, N. et al. Spontaneous network formation among cooperative RNA replicators. <em>Nature</em><strong>491</strong>, 72–77 (2012).
</cite> [<a href="https://doi.org/10.1038/nature11549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23075853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vaidya,%20N.%20et%20al.%20Spontaneous%20network%20formation%20among%20cooperative%20RNA%20replicators.%20Nature491,%2072%E2%80%9377%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Sczepanski, J. T. &amp; Joyce, G. F. Synthetic evolving systems that implement a user-specified genetic code of arbitrary design. <em>Chem. Biol.</em><strong>19</strong>, 1324–1332 (2012).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2012.08.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3483592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23102225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sczepanski,%20J.%20T.%20&amp;%20Joyce,%20G.%20F.%20Synthetic%20evolving%20systems%20that%20implement%20a%20user-specified%20genetic%20code%20of%20arbitrary%20design.%20Chem.%20Biol.19,%201324%E2%80%931332%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Michel, F. &amp; Westhof, E. Modelling of the three-dimensional architecture of group I catalytic introns based on comparative sequence analysis. <em>J. Mol. Biol.</em><strong>216</strong>, 585–610 (1990).
</cite> [<a href="https://doi.org/10.1016/0022-2836(90)90386-Z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2258934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michel,%20F.%20&amp;%20Westhof,%20E.%20Modelling%20of%20the%20three-dimensional%20architecture%20of%20group%20I%20catalytic%20introns%20based%20on%20comparative%20sequence%20analysis.%20J.%20Mol.%20Biol.216,%20585%E2%80%93610%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Russ, W. P. et al. An evolution-based model for designing chorismate mutase enzymes. <em>Science</em><strong>369</strong>, 440–445 (2020).
</cite> [<a href="https://doi.org/10.1126/science.aba3304" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32703877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Russ,%20W.%20P.%20et%20al.%20An%20evolution-based%20model%20for%20designing%20chorismate%20mutase%20enzymes.%20Science369,%20440%E2%80%93445%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Cuturello, F., Tiana, G. &amp; Bussi, G. Assessing the accuracy of direct-coupling analysis for RNA contact prediction. <em>RNA</em><strong>26</strong>, 637–647 (2020).
</cite> [<a href="https://doi.org/10.1261/rna.074179.119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7161351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32115426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cuturello,%20F.,%20Tiana,%20G.%20&amp;%20Bussi,%20G.%20Assessing%20the%20accuracy%20of%20direct-coupling%20analysis%20for%20RNA%20contact%20prediction.%20RNA26,%20637%E2%80%93647%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Cover, T. M. &amp; Thomas, J. A. <em>Elements of Information Theory</em> (Wiley, New York, 1999).</cite>
</li>
<li id="CR36">
<span class="label">36.</span><cite>Kardar, M. <em>Statistical Physics of Fields</em> (Cambridge University Press, 2007).</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Calvanese, F., Lambert, C. N., Nghe, P., Zamponi, F. &amp; Weigt, M. Towards parsimonious generative modeling of RNA families. <em>Nucleic Acids Res.</em><strong>52</strong>, 5465–5477 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkae289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11162787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38661206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calvanese,%20F.,%20Lambert,%20C.%20N.,%20Nghe,%20P.,%20Zamponi,%20F.%20&amp;%20Weigt,%20M.%20Towards%20parsimonious%20generative%20modeling%20of%20RNA%20families.%20Nucleic%20Acids%20Res.52,%205465%E2%80%935477%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Lorenz, R. et al. ViennaRNA package 2.0. <em>Algorithms Mol. Biol</em>. <strong>6</strong>, 26 (2011).
</cite> [<a href="https://doi.org/10.1186/1748-7188-6-26" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3319429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22115189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lorenz,%20R.%20et%20al.%20ViennaRNA%20package%202.0.%20Algorithms%20Mol.%20Biol.%206,%2026%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Zuker, M. &amp; Stiegler, P. Optimal computer folding of large RNA sequences using thermodynamics and auxiliary information. <em>Nucleic Acids Res.</em><strong>9</strong>, 133–148 (1981).
</cite> [<a href="https://doi.org/10.1093/nar/9.1.133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC326673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6163133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zuker,%20M.%20&amp;%20Stiegler,%20P.%20Optimal%20computer%20folding%20of%20large%20RNA%20sequences%20using%20thermodynamics%20and%20auxiliary%20information.%20Nucleic%20Acids%20Res.9,%20133%E2%80%93148%20(1981)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Mathews, D. H. Incorporating chemical modification constraints into a dynamic programming algorithm for prediction of RNA secondary structure. <em>Proc. Natl. Acad. Sci. USA</em><strong>101</strong>, 7287–7292 (2004).
</cite> [<a href="https://doi.org/10.1073/pnas.0401799101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC409911/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15123812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mathews,%20D.%20H.%20Incorporating%20chemical%20modification%20constraints%20into%20a%20dynamic%20programming%20algorithm%20for%20prediction%20of%20RNA%20secondary%20structure.%20Proc.%20Natl.%20Acad.%20Sci.%20USA101,%207287%E2%80%937292%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>McCaskill, J. S. The equilibrium partition function and base pair binding probabilities for RNA secondary structure. <em>Biopolymers</em><strong>29</strong>, 1105–1119 (1990).
</cite> [<a href="https://doi.org/10.1002/bip.360290621" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1695107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McCaskill,%20J.%20S.%20The%20equilibrium%20partition%20function%20and%20base%20pair%20binding%20probabilities%20for%20RNA%20secondary%20structure.%20Biopolymers29,%201105%E2%80%931119%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Zadeh, J. N. et al. NUPACK: Analysis and design of nucleic acid systems. <em>J. Comput Chem.</em><strong>32</strong>, 170–173 (2011).
</cite> [<a href="https://doi.org/10.1002/jcc.21596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20645303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zadeh,%20J.%20N.%20et%20al.%20NUPACK:%20Analysis%20and%20design%20of%20nucleic%20acid%20systems.%20J.%20Comput%20Chem.32,%20170%E2%80%93173%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Dirks, R. M. &amp; Pierce, N. A. An algorithm for computing nucleic acid base-pairing probabilities including pseudoknots. <em>J. Comput Chem.</em><strong>25</strong>, 1295–1304 (2004).
</cite> [<a href="https://doi.org/10.1002/jcc.20057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15139042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dirks,%20R.%20M.%20&amp;%20Pierce,%20N.%20A.%20An%20algorithm%20for%20computing%20nucleic%20acid%20base-pairing%20probabilities%20including%20pseudoknots.%20J.%20Comput%20Chem.25,%201295%E2%80%931304%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Mustoe, A. M., Al-Hashimi, H. M. &amp; Brooks, C. L. Secondary structure encodes a cooperative tertiary folding funnel in the Azoarcus ribozyme. <em>Nucleic Acids Res.</em><strong>44</strong>, 402–412 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkv1055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4705646/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26481360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mustoe,%20A.%20M.,%20Al-Hashimi,%20H.%20M.%20&amp;%20Brooks,%20C.%20L.%20Secondary%20structure%20encodes%20a%20cooperative%20tertiary%20folding%20funnel%20in%20the%20Azoarcus%20ribozyme.%20Nucleic%20Acids%20Res.44,%20402%E2%80%93412%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Li, S., Palo, M. Z., Zhang, X., Pintilie, G. &amp; Zhang, K. Snapshots of the second-step self-splicing of Tetrahymena ribozyme revealed by cryo-EM. <em>Nat. Commun.</em><strong>14</strong>, 1294 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-36724-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10020454/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36928031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20S.,%20Palo,%20M.%20Z.,%20Zhang,%20X.,%20Pintilie,%20G.%20&amp;%20Zhang,%20K.%20Snapshots%20of%20the%20second-step%20self-splicing%20of%20Tetrahymena%20ribozyme%20revealed%20by%20cryo-EM.%20Nat.%20Commun.14,%201294%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Ding, X., Zou, Z. &amp; Brooks Iii, C. L. Deciphering protein evolution and fitness landscapes with latent space models. <em>Nat. Commun.</em><strong>10</strong>, 5644 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-13633-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6904478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31822668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ding,%20X.,%20Zou,%20Z.%20&amp;%20Brooks%20Iii,%20C.%20L.%20Deciphering%20protein%20evolution%20and%20fitness%20landscapes%20with%20latent%20space%20models.%20Nat.%20Commun.10,%205644%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Poelwijk, F. J., Socolich, M. &amp; Ranganathan, R. Learning the pattern of epistasis linking genotype and phenotype in a protein. <em>Nat. Commun.</em><strong>10</strong>, 4213 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-12130-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6746860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31527666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poelwijk,%20F.%20J.,%20Socolich,%20M.%20&amp;%20Ranganathan,%20R.%20Learning%20the%20pattern%20of%20epistasis%20linking%20genotype%20and%20phenotype%20in%20a%20protein.%20Nat.%20Commun.10,%204213%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Bartel, D. P. &amp; Szostak, J. W. Isolation of new ribozymes from a large pool of random sequences. <em>Science</em><strong>261</strong>, 1411–1418 (1993).
</cite> [<a href="https://doi.org/10.1126/science.7690155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7690155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bartel,%20D.%20P.%20&amp;%20Szostak,%20J.%20W.%20Isolation%20of%20new%20ribozymes%20from%20a%20large%20pool%20of%20random%20sequences.%20Science261,%201411%E2%80%931418%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Madani, A. et al. Large language models generate functional protein sequences across diverse families. <em>Nat. Biotechnol.</em><strong>41</strong>, 1099–1106 (2023).
</cite> [<a href="https://doi.org/10.1038/s41587-022-01618-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10400306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36702895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madani,%20A.%20et%20al.%20Large%20language%20models%20generate%20functional%20protein%20sequences%20across%20diverse%20families.%20Nat.%20Biotechnol.41,%201099%E2%80%931106%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Lincoln, T. A. &amp; Joyce, G. F. Self-sustained replication of an RNA enzyme. <em>Science</em><strong>323</strong>, 1229–1232 (2009).
</cite> [<a href="https://doi.org/10.1126/science.1167856" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2652413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19131595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lincoln,%20T.%20A.%20&amp;%20Joyce,%20G.%20F.%20Self-sustained%20replication%20of%20an%20RNA%20enzyme.%20Science323,%201229%E2%80%931232%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Mizuuchi, R. &amp; Ichihashi, N. Minimal RNA self-reproduction discovered from a random pool of oligomers. <em>Chem. Sci.</em><strong>14</strong>, 7656–7664 (2023).
</cite> [<a href="https://doi.org/10.1039/d3sc01940c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10355099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37476714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mizuuchi,%20R.%20&amp;%20Ichihashi,%20N.%20Minimal%20RNA%20self-reproduction%20discovered%20from%20a%20random%20pool%20of%20oligomers.%20Chem.%20Sci.14,%207656%E2%80%937664%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Wagner, A. Genotype sampling for deep-learning assisted experimental mapping of a combinatorially complete fitness landscape. <em>Bioinformatics</em><strong>40</strong>, btae317 (2024).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btae317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11132821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38745436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wagner,%20A.%20Genotype%20sampling%20for%20deep-learning%20assisted%20experimental%20mapping%20of%20a%20combinatorially%20complete%20fitness%20landscape.%20Bioinformatics40,%20btae317%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Peri, G., Gibard, C., Shults, N. H., Crossin, K. &amp; Hayden, E. J. Dynamic RNA fitness landscapes of a group I ribozyme during changes to the experimental environment. <em>Mol. Biol. Evolution.</em>10.1093/molbev/msab373 (2022).</cite> [<a href="https://doi.org/10.1093/molbev/msab373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8890501/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35020916/" class="usa-link">PubMed</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Mizuuchi, R. et al. Mineral surfaces select for longer RNA molecules. <em>Chem. Commun.</em><strong>55</strong>, 2090–2093 (2019).</cite> [<a href="https://doi.org/10.1039/c8cc10319d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6377063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30694272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mizuuchi,%20R.%20et%20al.%20Mineral%20surfaces%20select%20for%20longer%20RNA%20molecules.%20Chem.%20Commun.55,%202090%E2%80%932093%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Poudyal, R. R. et al. Template-directed RNA polymerization and enhanced ribozyme catalysis inside membraneless compartments formed by coacervates. <em>Nat. Commun.</em><strong>10</strong>, 490 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-08353-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6353945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30700721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poudyal,%20R.%20R.%20et%20al.%20Template-directed%20RNA%20polymerization%20and%20enhanced%20ribozyme%20catalysis%20inside%20membraneless%20compartments%20formed%20by%20coacervates.%20Nat.%20Commun.10,%20490%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Le Vay, K., Song, E. Y., Ghosh, B., Tang, T.-Y. D. &amp; Mutschler, H. Enhanced ribozyme-catalyzed recombination and oligonucleotide assembly in peptide-RNA condensates. <em>Angew. Chem. Int. Ed. Engl.</em><strong>60</strong>, 26096–26104 (2021).
</cite> [<a href="https://doi.org/10.1002/anie.202109267" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9299051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34569680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Le%20Vay,%20K.,%20Song,%20E.%20Y.,%20Ghosh,%20B.,%20Tang,%20T.-Y.%20D.%20&amp;%20Mutschler,%20H.%20Enhanced%20ribozyme-catalyzed%20recombination%20and%20oligonucleotide%20assembly%20in%20peptide-RNA%20condensates.%20Angew.%20Chem.%20Int.%20Ed.%20Engl.60,%2026096%E2%80%9326104%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Kagaya, Y. et al. NuFold: end-to-end approach for RNA tertiary structure prediction with flexible nucleobase center representation. <em>Nat. Commun.</em><strong>16</strong>, 881 (2025).
</cite> [<a href="https://doi.org/10.1038/s41467-025-56261-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11751094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39837861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kagaya,%20Y.%20et%20al.%20NuFold:%20end-to-end%20approach%20for%20RNA%20tertiary%20structure%20prediction%20with%20flexible%20nucleobase%20center%20representation.%20Nat.%20Commun.16,%20881%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Joshi, C. K. et al. gRNAde: geometric deep learning for 3D RNA inverse design. Preprint at 10.48550/ARXIV.2305.14749 (2023).</cite>
</li>
<li id="CR59">
<span class="label">59.</span><cite>Penić, R. J., Vlašić, T., Huber, R. G., Wan, Y. &amp; Šikić, M. RiNALMo: general-purpose RNA language models can generalize well on structure prediction tasks. <em>Nat. Commun</em>. <strong>16</strong>, 1 (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-60872-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12219582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40593636/" class="usa-link">PubMed</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Zou, S. et al. A large-scale foundation model for RNA function and structure prediction. Preprint at <em>bioRxiv</em>10.1101/2024.11.28.625345 (2024).</cite>
</li>
<li id="CR61">
<span class="label">61.</span><cite>Adams, P. L., Stahley, M. R., Kosek, A. B., Wang, J. &amp; Strobel, S. A. Crystal structure of a self-splicing group I intron with both exons. <em>Nature</em><strong>430</strong>, 45–50 (2004).
</cite> [<a href="https://doi.org/10.1038/nature02642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15175762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adams,%20P.%20L.,%20Stahley,%20M.%20R.,%20Kosek,%20A.%20B.,%20Wang,%20J.%20&amp;%20Strobel,%20S.%20A.%20Crystal%20structure%20of%20a%20self-splicing%20group%20I%20intron%20with%20both%20exons.%20Nature430,%2045%E2%80%9350%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Kalvari, I. et al. 10.1093/nar/gkaa1047.</cite>
</li>
<li id="CR63">
<span class="label">63.</span><cite>Nawrocki, E. P. &amp; Eddy, S. Infernal 1.1: 100-fold faster RNA homology searches. <em>Bioinformatics</em><strong>29</strong>, 2933–2935 (2013).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt509" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3810854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24008419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nawrocki,%20E.%20P.%20&amp;%20Eddy,%20S.%20Infernal%201.1:%20100-fold%20faster%20RNA%20homology%20searches.%20Bioinformatics29,%202933%E2%80%932935%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Altschul, S. F., Gish, W., Miller, W., Myers, E. W. &amp; Lipman, D. J. Basic local alignment search tool. <em>J. Mol. Biol.</em><strong>215</strong>, 403–410 (1990).
</cite> [<a href="https://doi.org/10.1016/S0022-2836(05)80360-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Altschul,%20S.%20F.,%20Gish,%20W.,%20Miller,%20W.,%20Myers,%20E.%20W.%20&amp;%20Lipman,%20D.%20J.%20Basic%20local%20alignment%20search%20tool.%20J.%20Mol.%20Biol.215,%20403%E2%80%93410%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373953/bin/41467_2025_63151_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373953/bin/41467_2025_63151_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (6.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373953/bin/41467_2025_63151_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373953/bin/41467_2025_63151_MOESM4_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (1.8MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Source data for the figures are provided with this paper as a zip folder. Additional data generated in this study have been deposited in the Zenodo database under accession code 10.5281/zenodo.16531362. <a href="#Sec18" class="usa-link">Source data</a> are provided with this paper.</p>
<p>The code used in this study to generate and analyze sequences has been deposited in the Zenodo database under accession code 10.5281/zenodo.16531362</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63151-5"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63151.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373953/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373953/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373953%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373953/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373953/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373953/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846705/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373953/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846705/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373953/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373953/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="eoq0pAvU6UkcSbL0HknDHLgry5t4IEHa1n7ABxUi6L7J1zPWlJfYEXI0nyPkJ9rI">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
