
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Serological proteomic characterization for monitoring liver fibrosis regression in chronic hepatitis B patients on treatment - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4B56E8AF3A6E305B56E002F3BA297.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365141/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Serological proteomic characterization for monitoring liver fibrosis regression in chronic hepatitis B patients on treatment">
<meta name="citation_author" content="Mengyang Zhang">
<meta name="citation_author_institution" content="Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author" content="Shuyan Chen">
<meta name="citation_author_institution" content="Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author" content="Xiaoning Wu">
<meta name="citation_author_institution" content="Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author" content="Jialing Zhou">
<meta name="citation_author_institution" content="Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author" content="Bingqiong Wang">
<meta name="citation_author_institution" content="Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author" content="Tongtong Meng">
<meta name="citation_author_institution" content="Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author" content="Rongxuan Hua">
<meta name="citation_author_institution" content="Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author" content="Yameng Sun">
<meta name="citation_author_institution" content="Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="Chinese Institutes for Medical Research (CIMR), Beijing, China">
<meta name="citation_author" content="Hong You">
<meta name="citation_author_institution" content="Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author" content="Wei Chen">
<meta name="citation_author_institution" content="State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="Chinese Institutes for Medical Research (CIMR), Beijing, China">
<meta name="citation_author_institution" content="Experimental and Translational Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_author_institution" content="Beijing Clinical Research Institute, Beijing Friendship Hospital, Capital Medical University, Beijing, China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7714">
<meta name="citation_doi" content="10.1038/s41467-025-63006-z">
<meta name="citation_pmid" content="40830346">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365141/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365141/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365141/pdf/41467_2025_Article_63006.pdf">
<meta name="description" content="Longitudinal serological proteomic dynamics during antiviral therapy (AVT) in chronic hepatitis B (CHB) patients with liver fibrosis remain poorly characterized. Here, using four-dimensional data-independent acquisition mass spectrometry ...">
<meta name="og:title" content="Serological proteomic characterization for monitoring liver fibrosis regression in chronic hepatitis B patients on treatment">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Longitudinal serological proteomic dynamics during antiviral therapy (AVT) in chronic hepatitis B (CHB) patients with liver fibrosis remain poorly characterized. Here, using four-dimensional data-independent acquisition mass spectrometry ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365141/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365141">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63006-z"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63006.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365141%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365141/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365141/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365141/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7714. doi: <a href="https://doi.org/10.1038/s41467-025-63006-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63006-z</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Serological proteomic characterization for monitoring liver fibrosis regression in chronic hepatitis B patients on treatment</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mengyang Zhang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mengyang Zhang</span></h3>
<div class="p">
<sup>1</sup>Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>2</sup>State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>3</sup>National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mengyang Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Shuyan Chen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Shuyan Chen</span></h3>
<div class="p">
<sup>1</sup>Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>2</sup>State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>3</sup>National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shuyan Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Xiaoning Wu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Xiaoning Wu</span></h3>
<div class="p">
<sup>1</sup>Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>2</sup>State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>3</sup>National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoning Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jialing Zhou</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jialing Zhou</span></h3>
<div class="p">
<sup>1</sup>Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>2</sup>State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>3</sup>National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jialing Zhou</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Bingqiong Wang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Bingqiong Wang</span></h3>
<div class="p">
<sup>1</sup>Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>2</sup>State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>3</sup>National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bingqiong Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meng%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Tongtong Meng</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Tongtong Meng</span></h3>
<div class="p">
<sup>1</sup>Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>2</sup>State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>3</sup>National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meng%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tongtong Meng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hua%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Rongxuan Hua</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Rongxuan Hua</span></h3>
<div class="p">
<sup>1</sup>Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>2</sup>State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>3</sup>National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hua%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rongxuan Hua</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Yameng Sun</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Yameng Sun</span></h3>
<div class="p">
<sup>1</sup>Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>2</sup>State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>3</sup>National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>4</sup>Chinese Institutes for Medical Research (CIMR), Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yameng Sun</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22You%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Hong You</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Hong You</span></h3>
<div class="p">
<sup>1</sup>Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>2</sup>State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>3</sup>National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22You%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hong You</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Wei Chen</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Wei Chen</span></h3>
<div class="p">
<sup>2</sup>State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>3</sup>National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>4</sup>Chinese Institutes for Medical Research (CIMR), Beijing, China </div>
<div class="p">
<sup>5</sup>Experimental and Translational Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">
<sup>6</sup>Beijing Clinical Research Institute, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wei Chen</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>6,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Liver Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div id="Aff2">
<sup>2</sup>State Key Lab of Digestive Health, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div id="Aff3">
<sup>3</sup>National Clinical Research Center of Digestive Diseases, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div id="Aff4">
<sup>4</sup>Chinese Institutes for Medical Research (CIMR), Beijing, China </div>
<div id="Aff5">
<sup>5</sup>Experimental and Translational Research Center, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div id="Aff6">
<sup>6</sup>Beijing Clinical Research Institute, Beijing Friendship Hospital, Capital Medical University, Beijing, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 29; Accepted 2025 Aug 6; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365141Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830346/" class="usa-link">40830346</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Longitudinal serological proteomic dynamics during antiviral therapy (AVT) in chronic hepatitis B (CHB) patients with liver fibrosis remain poorly characterized. Here, using four-dimensional data-independent acquisition mass spectrometry (4D-DIA-MS), paired liver biopsy (LBx)-proven serum samples from 130 CHB liver fibrosis patients undergoing short-term (78 weeks) or long-term (260 weeks) AVT are analyzed. Our findings show that prolonged AVT drives progressive serological proteomic remodeling in fibrosis regressors, characterized by a temporal inversion in the activation of the complement and coagulation cascades. Using machine learning algorithms trained on the 4D-DIA-MS discovery cohort, we develop a logistic regression model incorporating a seven-protein panel for short-term AVT and a three-protein panel for long-term AVT, respectively, both of which demonstrate moderate discriminatory capabilities for fibrosis regression. Subsequent external validation in an independent cohort (nâ€‰=â€‰54) with serial LBx assessments at baseline, 78 weeks, and 260 weeks, where serological proteins are quantified using parallel reaction monitoring mass spectrometry (PRM-MS), further confirms their generalizability. Furthermore, our longitudinal trajectory analysis highlights that the long-term proteomic signature exhibits greater stability compared to the short-term panel. This study proposes and validates duration-adapted serological proteomic panels as non-invasive tools for monitoring histological fibrosis regression in on-treatment CHB patients.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Liver fibrosis, Diagnostic markers, Hepatitis B virus, Proteomic analysis</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Liver fibrosis regression during antiviral therapy in chronic hepatitis B patients is not well understood. Here, the authors use mass spectrometry and machine learning to develop and validate serological proteomic panels, offering non-invasive tools for monitoring fibrosis regression.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Chronic hepatitis B (CHB) infection remains a principal driver of liver fibrosis burden worldwide<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. While liver fibrosis regression has been well-documented in CHB patients following successful suppression of hepatitis B virus (HBV), only 40%-60% of patients achieve regression by antiviral therapy (AVT)<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Recent studies indicate that histological non-regression of fibrosis is positively associated with increased risks of hepatocellular carcinoma and liver-related mortality<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, which can be distressing for patients after prolonged treatment when their fibrosis fails to regress or even worsens. This clinical reality underscores the critical need for longitudinal monitoring strategies to assess liver fibrosis regression during AVT.</p>
<p id="Par4">Although liver biopsy (LBx) remains the gold standard for assessing the histological status of HBV-related liver fibrosis, its inherent drawbacks (e.g., sampling variability and procedural risks) limit its use for serial monitoring of liver fibrosis<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. In recent years, transient elastography (TE), a non-invasive test (NIT), has been developed and shown to be effective for evaluating liver fibrosis in treatment-naÃ¯ve patients due to its reliability and reproducibility<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. However, liver stiffness measured by TE is largely influenced by inflammation rather than fibrotic remodeling during AVT<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, indicating that TE is not suitable for monitoring liver fibrosis regression in CHB patients on AVT. Similarly, established serological indices and emerging biomarkers, such as platelet counts, platelet-based models (aspartate aminotransferase to platelet ratio index [APRI] and fibrosis-4 index [FIB4]), Wisteria floribunda agglutinin-positive Mac-2-binding protein (M2BPGi), Chitinase-3-like protein 1 (CHI3L1), and Golgi protein 73 (GP73), show good diagnostic potential for treatment-naÃ¯ve liver fibrosis<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>â€“<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. However, they have proven inadequate or nonspecific for accurately evaluating liver fibrosis regression<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<p id="Par5">Previous studies have revealed that baseline heterogeneity in host and viral factorsâ€”such as age, gender, alcohol consumption, comorbidities, and the natural course of HBVâ€”significantly affects antiviral outcomes, including virologic suppression, biochemical remission, and fibrosis regression<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. This variability complicates the effectiveness of routine NITs and the identification of novel biomarkers for monitoring liver fibrosis regression during AVT. Our prior clinical studies also demonstrated that the serological biochemical indices related to HBV infection and liver inflammation significantly improved following a period of AVT, regardless of changes in histological fibrosis<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Therefore, we posit that the changes in the serological proteomic profile may be also highly sensitive to the recovery of virology and inflammation, posing a critical barrier to the discovery of serological proteomic biomarkers for liver fibrosis regression in CHB patients.</p>
<p id="Par6">To our knowledge, the dynamic serological proteomic profiling of CHB liver fibrosis patients during AVT remains largely unexplored, and there are currently no effective NITs that accurately reflect fibrosis regression in on-treatment CHB patients. However, recent advancements in omics methodologies and the capabilities of machine learning (ML) enable hypothesis-free screening of even thousands of potential biomarker candidates for disease diagnosis and treatment efficiency evaluation<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Consequently, this study employed four-dimensional data-independent acquisition mass spectrometry (4D-DIA-MS) to characterize the comprehensive serological proteomic dynamics during AVT in CHB patients with liver fibrosis, using paired LBx-proven serum samples (nâ€‰=â€‰130). Additionally, multiple ML models were developed to identify potential serological biomarkers that can distinguish between regressive and non-regressive CHB liver fibrosis patients following AVT. The performance of these models was validated internally through bootstrap resampling and calibration curves, as well as externally in an independent cohort (nâ€‰=â€‰54) where serological proteins were quantified using parallel reaction monitoring mass spectrometry (PRM-MS).</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Study design and patient characteristics</h3>
<p id="Par7">The study design workflow is illustrated in Fig.Â <a href="#Fig1" class="usa-link">1</a>. We analyzed paired serum samples from 130 CHB patients with liver fibrosis receiving AVT, comprising two treatment duration groups: short-term (78 weeks, nâ€‰=â€‰94) and long-term (260 weeks, nâ€‰=â€‰36). Paired pre-/post-treatment LBx were obtained for all short-term patients and 15 long-term patients; the remaining 21 long-term patients with baseline clinical diagnosis of compensated cirrhosis provided only post-treatment LBx. Fibrosis regression occurred in 59.6% (56/94) of short-term and 69.4% (25/36) of long-term treated patients, as determined by standardized histological criteria (see â€œMethodsâ€). Longitudinal serological proteomic profiling was performed using 4D-DIA-MS (discovery cohort), with data randomly divided into training (60%) and testing (40%) sets through stratified sampling on the regression status for ML development. We then employed bootstrap resampling (1000 iterations) and calibration curve analysis for internal validation, complemented by external validation in an independent PRM-MS cohort (nâ€‰=â€‰54) with serial LBx at baseline (Ishak scoreâ€‰â‰¥â€‰2), 78 weeks, and 260 weeks. This validation cohort demonstrated fibrosis regression rates of 59.3% (32/54) and 77.8% (42/54) following short-term and long-term treatment, respectively. Furthermore, the model performance was assessed in subgroups of patients from both the 4D-DIA-MS discovery cohort (training and testing sets) and the PRM-MS validation cohort, specifically focusing on three clinically relevant populations: patients with significant baseline fibrosis (Ishak â‰¥ 3), patients without steatosis at baseline (non-steatosis), and patients with substantial histological improvement versus non-regression following AVT (designated as Î”Ishak â‰¥ 1). Finally, model performance was compared against routine NITs, including liver stiffness measurement (LSM), APRI, and FIB4, across both the 4D-DIA-MS discovery and PRM-MS validation cohorts. The demographic and clinical characteristics of the enrolled patients in the 4D-DIA-MS discovery cohort (training and testing sets) and PRM-MS validation cohort are summarized in TableÂ <a href="#Tab1" class="usa-link">1</a>. Detailed subgroup sample sizes are provided in Supplementary TableÂ <a href="#MOESM1" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Study design framework.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365141_41467_2025_63006_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5d4a/12365141/03085ce44442/41467_2025_63006_Fig1_HTML.jpg" loading="lazy" id="d33e368" height="674" width="800" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Paired serological proteomes from CHB liver fibrosis patients undergoing either short-term or long-term AVT were profiled using 4D-DIA-MS technology. The resulting 4D-DIA-MS data were randomly divided into training and testing sets. ML models, incorporating five feature selection methods and four ML algorithms, were developed using the training set and subsequently validated in the testing set to identify optimal panels for differentiating fibrosis regression status during AVT. The performance of the prioritized models was internally validated through bootstrap resampling and calibration curves, and externally assessed in an independent cohort where serological proteins were quantified using PRM-MS. Additionally, the models were compared with routine NITs and evaluated in clinically relevant subgroups of patients. <em>Abbreviations</em>: 4D-DIA-MS four-dimensional data-independent acquisition mass spectrometry, APRI aspartate aminotransferase to platelet ratio index, ALT alanine aminotransferase; AST aspartate aminotransferase, AVT antiviral therapy, FIB4 fibrosis-4 index, LSM liver stiffness measurement, ML machine learning, NIT non-invasive test, PLS partial least squares, PLT platelet, PIR predominantly progressive, indeterminate, predominantly regressive, PRM-MS parallel reaction monitoring mass spectrometry, SVM support vector machine. Created in BioRender. Chen, W. (2025) <a href="https://BioRender.com/84noo7b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/84noo7b</a>.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Clinical characteristics of 4D-DIA-MS discovery and PRM-MS validation cohorts</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th colspan="1" rowspan="1"></th>
<th colspan="2" rowspan="1">4D-DIA-MS: Short-term</th>
<th rowspan="2" colspan="1">PRM-MS: Short-term (nâ€‰=â€‰54)</th>
<th rowspan="2" colspan="1"><em>p</em></th>
<th colspan="2" rowspan="1">4D-DIA-MS: Long-term</th>
<th rowspan="2" colspan="1">PRM-MS: Long-term (nâ€‰=â€‰54)</th>
<th rowspan="2" colspan="1"><em>p</em></th>
</tr>
<tr>
<th colspan="1" rowspan="1"></th>
<th colspan="1" rowspan="1">Train (nâ€‰=â€‰57)</th>
<th colspan="1" rowspan="1">Test (nâ€‰=â€‰37)</th>
<th colspan="1" rowspan="1">Train (nâ€‰=â€‰22)</th>
<th colspan="1" rowspan="1">Test (nâ€‰=â€‰14)</th>
</tr>
</thead>
<tbody>
<tr><td colspan="9" rowspan="1">Demographics</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAge, years</td>
<td align="left" colspan="1" rowspan="1">32 (27, 41)</td>
<td align="left" colspan="1" rowspan="1">40 (29, 46)</td>
<td align="left" colspan="1" rowspan="1">37 (28, 42)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰3.816</p>
<p>pâ€‰=â€‰0.148</p>
</td>
<td align="left" colspan="1" rowspan="1">44 (37, 51)</td>
<td align="left" colspan="1" rowspan="1">41 (37, 49)</td>
<td align="left" colspan="1" rowspan="1">37 (28, 42)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>H</strong>â€‰=â€‰<strong>12.548</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.002</strong></p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒMale, n (%)</td>
<td align="left" colspan="1" rowspan="1">45 (78.9)</td>
<td align="left" colspan="1" rowspan="1">25 (67.6)</td>
<td align="left" colspan="1" rowspan="1">42 (77.8)</td>
<td align="left" colspan="1" rowspan="1">
<p>Ï‡Â² = 1.783</p>
<p>pâ€‰=â€‰0.410</p>
</td>
<td align="left" colspan="1" rowspan="1">16 (72.7)</td>
<td align="left" colspan="1" rowspan="1">12 (85.7)</td>
<td align="left" colspan="1" rowspan="1">42 (77.8)</td>
<td align="left" colspan="1" rowspan="1">
<p>Ï‡Â² = 0.835</p>
<p>pâ€‰=â€‰0.659</p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">Laboratory tests</td></tr>
<tr><td colspan="9" rowspan="1">ALT, U/L</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">89.5 (42.0, 140.6)</td>
<td align="left" colspan="1" rowspan="1">84.0 (35.0, 145.0)</td>
<td align="left" colspan="1" rowspan="1">72.7 (49.5, 110.0)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.282</p>
<p>pâ€‰=â€‰0.869</p>
</td>
<td align="left" colspan="1" rowspan="1">51.5 (36.0, 63.0)</td>
<td align="left" colspan="1" rowspan="1">52.7 (34.0, 79.7)</td>
<td align="left" colspan="1" rowspan="1">72.7 (49.5, 110.0)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰5.926</p>
<p>pâ€‰=â€‰0.052</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒWeek 78/260</td>
<td align="left" colspan="1" rowspan="1">21.7 (15.0, 31.0)</td>
<td align="left" colspan="1" rowspan="1">25.0 (16.8, 32.6)</td>
<td align="left" colspan="1" rowspan="1">26.5 (20.0, 38.7)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰2.988</p>
<p>pâ€‰=â€‰0.225</p>
</td>
<td align="left" colspan="1" rowspan="1">24.0 (18.0, 32.0)</td>
<td align="left" colspan="1" rowspan="1">26.5 (21.9, 40.0)</td>
<td align="left" colspan="1" rowspan="1">21.7 (16.5, 28.0)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰90.000</p>
<p>pâ€‰=â€‰0.132</p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">AST, U/L</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">53.3 (33.4, 80.0)</td>
<td align="left" colspan="1" rowspan="1">51.0 (37.2, 98.4)</td>
<td align="left" colspan="1" rowspan="1">53.5 (41.2, 82.0)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.211</p>
<p>pâ€‰=â€‰0.900</p>
</td>
<td align="left" colspan="1" rowspan="1">42.2 (35.0, 53.0)</td>
<td align="left" colspan="1" rowspan="1">43.4 (30.0, 83.0)</td>
<td align="left" colspan="1" rowspan="1">53.5 (41.2, 82.0)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰3.390</p>
<p>pâ€‰=â€‰0.184</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒWeek 78/260</td>
<td align="left" colspan="1" rowspan="1">21.7 (19.0, 27.2)</td>
<td align="left" colspan="1" rowspan="1">25.5 (20.0, 30.6)</td>
<td align="left" colspan="1" rowspan="1">26.7 (21.0, 31.0)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰4.346</p>
<p>pâ€‰=â€‰0.114</p>
</td>
<td align="left" colspan="1" rowspan="1">23.1 (19.0, 26.9)</td>
<td align="left" colspan="1" rowspan="1">26.0 (22.5, 28.0)</td>
<td align="left" colspan="1" rowspan="1">20.2 (18.0, 25.0)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>H</strong>â€‰=â€‰<strong>6.614</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.037</strong></p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">Total bilirubin, umol/L</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">13.8 (10.8, 20.0)</td>
<td align="left" colspan="1" rowspan="1">14.4 (12.1, 19.5)</td>
<td align="left" colspan="1" rowspan="1">16.2 (11.9, 19.1)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.517</p>
<p>pâ€‰=â€‰0.772</p>
</td>
<td align="left" colspan="1" rowspan="1">17.2 (12.9, 20.5)</td>
<td align="left" colspan="1" rowspan="1">13.9 (10.9, 21.0)</td>
<td align="left" colspan="1" rowspan="1">16.2 (11.9, 19.1)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰4.044</p>
<p>pâ€‰=â€‰0.132</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒWeek 78/260</td>
<td align="left" colspan="1" rowspan="1">14.3 (11.2, 16.7)</td>
<td align="left" colspan="1" rowspan="1">12.6 (9.0, 17.7)</td>
<td align="left" colspan="1" rowspan="1">12.4 (9.7, 15.2)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰3.017</p>
<p>pâ€‰=â€‰0.221</p>
</td>
<td align="left" colspan="1" rowspan="1">18.0 (13.2, 21.7)</td>
<td align="left" colspan="1" rowspan="1">14.8 (11.6, 19.4)</td>
<td align="left" colspan="1" rowspan="1">14.0 (9.9, 17.5)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰3.877</p>
<p>pâ€‰=â€‰0.144</p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">Albumin, g/L</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">43.0 (40.0, 46.0)</td>
<td align="left" colspan="1" rowspan="1">42.6 (39.0, 47.4)</td>
<td align="left" colspan="1" rowspan="1">42.6 (40.0, 45.3)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.422</p>
<p>pâ€‰=â€‰0.810</p>
</td>
<td align="left" colspan="1" rowspan="1">43.6 (40.5, 46.5)</td>
<td align="left" colspan="1" rowspan="1">42.5 (39.2, 45.5)</td>
<td align="left" colspan="1" rowspan="1">42.6 (40.0, 45.3)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.685</p>
<p>pâ€‰=â€‰0.710</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒWeek 78/260</td>
<td align="left" colspan="1" rowspan="1">45.4 (43.0, 47.5)</td>
<td align="left" colspan="1" rowspan="1">45.5 (43.6, 46.9)</td>
<td align="left" colspan="1" rowspan="1">45.4 (43.0, 48.0)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.023</p>
<p>pâ€‰=â€‰0.988</p>
</td>
<td align="left" colspan="1" rowspan="1">47.3 (43.5, 48.6)</td>
<td align="left" colspan="1" rowspan="1">44.2 (43.9, 50.0)</td>
<td align="left" colspan="1" rowspan="1">46.3 (43.7, 48.3)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.580</p>
<p>pâ€‰=â€‰0.748</p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">Platelet, 10^9/L</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">162 (132, 223)</td>
<td align="left" colspan="1" rowspan="1">137 (111, 192)</td>
<td align="left" colspan="1" rowspan="1">167 (121, 189)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰1.963</p>
<p>pâ€‰=â€‰0.375</p>
</td>
<td align="left" colspan="1" rowspan="1">128.5 (94, 147)</td>
<td align="left" colspan="1" rowspan="1">127 (111, 151.5)</td>
<td align="left" colspan="1" rowspan="1">166.5 (121, 189)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>H</strong>â€‰=â€‰<strong>9.643</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.008</strong></p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒWeek 78/260</td>
<td align="left" colspan="1" rowspan="1">161 (121, 202)</td>
<td align="left" colspan="1" rowspan="1">142.5 (115, 175.5)</td>
<td align="left" colspan="1" rowspan="1">148 (119, 199)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰1.854</p>
<p>pâ€‰=â€‰0.396</p>
</td>
<td align="left" colspan="1" rowspan="1">146 (126, 193)</td>
<td align="left" colspan="1" rowspan="1">164 (103, 255)</td>
<td align="left" colspan="1" rowspan="1">195 (171, 223)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>H</strong>â€‰=â€‰<strong>11.844</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.003</strong></p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">International normalized ratio</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">1.06 (1.00, 1.14)</td>
<td align="left" colspan="1" rowspan="1">1.09 (1.00, 1.14)</td>
<td align="left" colspan="1" rowspan="1">1.05 (1.00, 1.13)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰1.562</p>
<p>pâ€‰=â€‰0.458</p>
</td>
<td align="left" colspan="1" rowspan="1">1.12 (1.05, 1.20)</td>
<td align="left" colspan="1" rowspan="1">1.16 (1.03, 1.21)</td>
<td align="left" colspan="1" rowspan="1">1.05 (1.00, 1.13)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>H</strong>â€‰=â€‰<strong>6.853</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.033</strong></p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒWeek 78/260</td>
<td align="left" colspan="1" rowspan="1">1.00 (0.96, 1.06)</td>
<td align="left" colspan="1" rowspan="1">1.00 (0.96, 1.03)</td>
<td align="left" colspan="1" rowspan="1">1.00 (0.93, 1.04)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.319</p>
<p>pâ€‰=â€‰0.852</p>
</td>
<td align="left" colspan="1" rowspan="1">1.01 (0.96, 1.11)</td>
<td align="left" colspan="1" rowspan="1">0.98 (0.95, 1.02)</td>
<td align="left" colspan="1" rowspan="1">1.01 (0.96, 1.04)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰2.041</p>
<p>pâ€‰=â€‰0.360</p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">HBsAg, 10<sup>3</sup>IU/mL</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">4.8 (1.6, 9.6)</td>
<td align="left" colspan="1" rowspan="1">2.8 (0.8, 5.0)</td>
<td align="left" colspan="1" rowspan="1">4.0 (1.9, 13.4)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰4.312</p>
<p>pâ€‰=â€‰0.116</p>
</td>
<td align="left" colspan="1" rowspan="1">1.7 (0.8, 4.7)</td>
<td align="left" colspan="1" rowspan="1">2.8 (2.0, 5.0)</td>
<td align="left" colspan="1" rowspan="1">4.0 (1.9, 13.4)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>H</strong>â€‰=â€‰<strong>7.955</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.019</strong></p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">HBeAg, n (%)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">36 (67.9)</td>
<td align="left" colspan="1" rowspan="1">26 (72.2)</td>
<td align="left" colspan="1" rowspan="1">43 (82.7)</td>
<td align="left" colspan="1" rowspan="1">
<p>Ï‡Â² = 3.139</p>
<p>pâ€‰=â€‰0.208</p>
</td>
<td align="left" colspan="1" rowspan="1">10 (45.5)</td>
<td align="left" colspan="1" rowspan="1">6 (50.0)</td>
<td align="left" colspan="1" rowspan="1">43 (82.7)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>Ï‡Â² = 12.194</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.002</strong></p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">HBV DNA, Log IU/mL</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">6.6 (4.8, 7.6)</td>
<td align="left" colspan="1" rowspan="1">6.3 (4.7, 7.2)</td>
<td align="left" colspan="1" rowspan="1">6.7 (5.4, 7.5)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.834</p>
<p>pâ€‰=â€‰0.659</p>
</td>
<td align="left" colspan="1" rowspan="1">4.7 (3.2, 6.0)</td>
<td align="left" colspan="1" rowspan="1">6.1 (4.3, 6.9)</td>
<td align="left" colspan="1" rowspan="1">6.7 (5.4, 7.5)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>H</strong>â€‰=â€‰<strong>16.447</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.000</strong></p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒWeek 78/260</td>
<td align="left" colspan="1" rowspan="1">0.0 (0.0, 1.4)</td>
<td align="left" colspan="1" rowspan="1">0.0 (0.0, 0.0)</td>
<td align="left" colspan="1" rowspan="1">0 (0, 1.4)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰1.814</p>
<p>pâ€‰=â€‰0.404</p>
</td>
<td align="left" colspan="1" rowspan="1">0.0 (0.0, 0.0)</td>
<td align="left" colspan="1" rowspan="1">0.0 (0.0, 0.0)</td>
<td align="left" colspan="1" rowspan="1">0.0 (0.0, 0.0)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.660</p>
<p>pâ€‰=â€‰0.719</p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">LSM, kPa</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">9.9 (7.2, 14.4)</td>
<td align="left" colspan="1" rowspan="1">10.7 (8.5, 15.2)</td>
<td align="left" colspan="1" rowspan="1">12.6 (8.6, 18.6)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰4.373</p>
<p>pâ€‰=â€‰0.112</p>
</td>
<td align="left" colspan="1" rowspan="1">13.5 (10.7, 19.6)</td>
<td align="left" colspan="1" rowspan="1">14.0 (8.7, 20.4)</td>
<td align="left" colspan="1" rowspan="1">12.6 (8.6, 18.6)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.279</p>
<p>pâ€‰=â€‰0.870</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒWeek 78/260</td>
<td align="left" colspan="1" rowspan="1">6.1 (5.3, 9.1)</td>
<td align="left" colspan="1" rowspan="1">6.8 (6.1, 8.2)</td>
<td align="left" colspan="1" rowspan="1">6.9 (5.6, 9.1)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰1.409</p>
<p>pâ€‰=â€‰0.494</p>
</td>
<td align="left" colspan="1" rowspan="1">8.5 (6.7, 10.2)</td>
<td align="left" colspan="1" rowspan="1">7.4 (6.1, 11.2)</td>
<td align="left" colspan="1" rowspan="1">6.0 (4.6, 8.3)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>H</strong>â€‰=â€‰<strong>13.563</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.001</strong></p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">APRI</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">0.8 (0.5, 1.4)</td>
<td align="left" colspan="1" rowspan="1">0.9 (0.6, 1.9)</td>
<td align="left" colspan="1" rowspan="1">0.9 (0.6, 1.4)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰1.198</p>
<p>pâ€‰=â€‰0.549</p>
</td>
<td align="left" colspan="1" rowspan="1">0.9 (0.7, 1.4)</td>
<td align="left" colspan="1" rowspan="1">0.9 (0.5, 1.6)</td>
<td align="left" colspan="1" rowspan="1">0.9 (0.6, 1.4)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰0.061</p>
<p>pâ€‰=â€‰0.970</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒWeek 78/260</td>
<td align="left" colspan="1" rowspan="1">0.3 (0.2, 0.5)</td>
<td align="left" colspan="1" rowspan="1">0.4 (0.3, 0.6)</td>
<td align="left" colspan="1" rowspan="1">0.5 (0.3, 0.6)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰4.054</p>
<p>pâ€‰=â€‰0.132</p>
</td>
<td align="left" colspan="1" rowspan="1">0.4 (0.3, 0.6)</td>
<td align="left" colspan="1" rowspan="1">0.4 (0.3, 0.5)</td>
<td align="left" colspan="1" rowspan="1">0.3 (0.2, 0.3)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>H</strong>â€‰=â€‰<strong>13.315</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.001</strong></p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">FIB4</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1">1.2 (0.9, 1.6)</td>
<td align="left" colspan="1" rowspan="1">1.8 (1.0, 3.1)</td>
<td align="left" colspan="1" rowspan="1">1.5 (1, 2.2)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰4.428</p>
<p>pâ€‰=â€‰0.109</p>
</td>
<td align="left" colspan="1" rowspan="1">2.3 (1.6, 3.2)</td>
<td align="left" colspan="1" rowspan="1">2.3 (1.7, 2.8)</td>
<td align="left" colspan="1" rowspan="1">1.5 (1, 2.2)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>H</strong>â€‰=â€‰<strong>9.089</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.011</strong></p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒWeek 78/260</td>
<td align="left" colspan="1" rowspan="1">1.0 (0.8, 1.6)</td>
<td align="left" colspan="1" rowspan="1">1.4 (0.9, 2.4)</td>
<td align="left" colspan="1" rowspan="1">1.1 (0.8, 1.6)</td>
<td align="left" colspan="1" rowspan="1">
<p>Hâ€‰=â€‰5.693</p>
<p>pâ€‰=â€‰0.058</p>
</td>
<td align="left" colspan="1" rowspan="1">1.3 (1.1, 2.3)</td>
<td align="left" colspan="1" rowspan="1">1.3 (1.0, 1.7)</td>
<td align="left" colspan="1" rowspan="1">0.9 (0.7, 1.3)</td>
<td align="left" colspan="1" rowspan="1">
<p><strong>H</strong>â€‰=â€‰<strong>22.381</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.000</strong></p>
</td>
</tr>
<tr><td colspan="9" rowspan="1">Histological assessments</td></tr>
<tr><td colspan="9" rowspan="1">Ishak fibrosis score, n (%)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒBaseline</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">
<p>Ï‡Â² = 7.600</p>
<p>pâ€‰=â€‰0.107</p>
</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">
<p><strong>Ï‡Â² = 46.499</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.000</strong></p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒ0/1/2</td>
<td align="left" colspan="1" rowspan="1">17 (29.8)</td>
<td align="left" colspan="1" rowspan="1">10 (27.0)</td>
<td align="left" colspan="1" rowspan="1">7 (13.0)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">2 (9.1)</td>
<td align="left" colspan="1" rowspan="1">3 (21.4)</td>
<td align="left" colspan="1" rowspan="1">7 (13.0)</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒ3/4</td>
<td align="left" colspan="1" rowspan="1">34 (59.6)</td>
<td align="left" colspan="1" rowspan="1">20 (54.1)</td>
<td align="left" colspan="1" rowspan="1">33 (61.1)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4 (18.2)</td>
<td align="left" colspan="1" rowspan="1">2 (14.3)</td>
<td align="left" colspan="1" rowspan="1">33 (61.1)</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒ5/6</td>
<td align="left" colspan="1" rowspan="1">6 (10.5)</td>
<td align="left" colspan="1" rowspan="1">7 (18.9)</td>
<td align="left" colspan="1" rowspan="1">14 (25.9)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4 (18.2)</td>
<td align="left" colspan="1" rowspan="1">0 (0.0)</td>
<td align="left" colspan="1" rowspan="1">14 (25.9)</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒUnknown</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">12 (54.5)</td>
<td align="left" colspan="1" rowspan="1">9 (64.3)</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒWeek 78/260</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">
<p>Ï‡Â² = 6.311</p>
<p>pâ€‰=â€‰0.177</p>
</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">
<p><strong>Ï‡Â² = 10.622</strong></p>
<p><strong>p</strong>â€‰=â€‰<strong>0.031</strong></p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒ0/1/2</td>
<td align="left" colspan="1" rowspan="1">25 (43.9)</td>
<td align="left" colspan="1" rowspan="1">15 (40.5)</td>
<td align="left" colspan="1" rowspan="1">15 (27.8)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">4 (18.2)</td>
<td align="left" colspan="1" rowspan="1">3 (21.4)</td>
<td align="left" colspan="1" rowspan="1">22 (40.7)</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒ3/4</td>
<td align="left" colspan="1" rowspan="1">26 (45.6)</td>
<td align="left" colspan="1" rowspan="1">17 (45.9)</td>
<td align="left" colspan="1" rowspan="1">25 (46.3)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">12 (54.5)</td>
<td align="left" colspan="1" rowspan="1">6 (42.9)</td>
<td align="left" colspan="1" rowspan="1">28 (51.9)</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒ5/6</td>
<td align="left" colspan="1" rowspan="1">6 (10.5)</td>
<td align="left" colspan="1" rowspan="1">5 (13.5)</td>
<td align="left" colspan="1" rowspan="1">14 (25.9)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">6 (27.3)</td>
<td align="left" colspan="1" rowspan="1">5 (35.7)</td>
<td align="left" colspan="1" rowspan="1">4 (0.07)</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p114"><p>Note: Continuous variables were expressed as meanâ€‰Â±â€‰standard deviation or median (interquartile range), whereas categorical variables were expressed as counts and percentages. For comparisons across three independent groups, continuous variables were analyzed using one-way ANOVA (for normally distributed data with homogeneity of variance) or the Kruskal-Wallis test (for non-normally distributed data). Categorical variables were compared using the chi-square test (when expected frequenciesâ€‰â‰¥â€‰5) or the Fisher-Freeman-Halton exact test (when expected frequencies &lt;5). Statistical significance was defined as pâ€‰&lt;â€‰0.05, with significant values highlighted in bold.</p></div></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Comparable serological proteomic kinetics in fibrosis regressors versus non-regressors during AVT</h3>
<p id="Par8">Our 4D-DIA-MS analysis quantified a total of 816 serological proteins, which underwent rigorous quality filtering (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">1</a>). This process involved the exclusion of immunoglobulins and their fractions, proteins with more than 50% missing values across all samples. As a result, approximately 36.5% of the initial identifications were removed, yielding 518 high-confidence proteins with median coefficients of variation of around 10% across baseline and post-treatment time points (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">2</a>). Treatment-induced serological proteomic changes were quantified as post-/pre-treatment (post/pre) ratios for subsequent analyses. Based on the post/pre ratios of 518 robust proteins, hierarchical clustering (HCL) with average linkage and the Euclidean distance metric revealed that the variation trends in the serological proteome were generally comparable between fibrosis regressors and non-regressors in both short-term and long-term AVT groups (Fig.Â <a href="#Fig2" class="usa-link">2a</a>). Principal component analysis (PCA) of the post/pre ratios confirmed this observation, showing no systematic separation between regression status groups or treatment duration cohorts (Fig.Â <a href="#Fig2" class="usa-link">2b</a>). To further assess inter-patient proteomic divergence, we implemented the scale-invariant â€œ1-râ€ similarity metric. However, the integration of â€œ1-râ€ values into HCL algorithm failed to distinguish fibrosis regression status in either treatment duration group (Fig.Â <a href="#Fig2" class="usa-link">2c</a>). These complementary analytical approaches demonstrate fundamental similarities in AVT-induced proteomic dynamics between patients with differing fibrosis outcomes. This conserved kinetic signature across regression status groups underscores the challenges of conventional hypothesis-driven approaches for identifying serological biomarkers of histological fibrosis improvement during AVT.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Comparison of serological proteomic kinetics between fibrosis regressors and non-regressors during AVT.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365141_41467_2025_63006_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5d4a/12365141/f5b61f66c7e8/41467_2025_63006_Fig2_HTML.jpg" loading="lazy" id="d33e1298" height="623" width="799" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The heatmap illustrates the overall dynamics of serological proteomes between fibrosis regressors and non-regressors following AVT, represented by the Log<sub>2</sub>-transformed post/pre ratios of protein abundance (nâ€‰=â€‰81 for R; nâ€‰=â€‰49 for NR; nâ€‰=â€‰94 for short-term; nâ€‰=â€‰36 for long-term). Darker shades of grey indicate lower ratios, while darker shades of red indicate higher ratios. Both the row post/pre ratios and the column patient samples were clustered using HCL with average linkage and Euclidean distance metrics. <strong>b</strong> PCA plots compare short-term versus long-term patients and fibrosis regressors versus non-regressors, based on the Log<sub>2</sub>-transformed post/pre ratios of protein abundance (nâ€‰=â€‰81 for R; nâ€‰=â€‰49 for NR; nâ€‰=â€‰94 for short-term; nâ€‰=â€‰36 for long-term). Grouped samples are distinguished by distinct colors. <strong>c</strong> The heatmap depicts the divergence of serological proteomic kinetics between any two patients undergoing AVT, based on the scale-invariant measure â€œ1-Pearsonâ€™s correlation coefficient (r)â€. Total patients: nâ€‰=â€‰94 for short-term; nâ€‰=â€‰36 for long-term; nâ€‰=â€‰81 for R; nâ€‰=â€‰49 for NR. Short-term patients: nâ€‰=â€‰56 for R; nâ€‰=â€‰38 for NR. Long-term patients: nâ€‰=â€‰25 for R; nâ€‰=â€‰11 for NR. Darker shades of grey signify lower divergence, while darker shades of red signify higher divergence. Both row and column â€œ1-râ€ values were clustered using HCL with average linkage and Euclidean distance metrics. Sample groups are color-coded. <em>Abbreviations</em>: AVT antiviral therapy, NR non-regressor, R regressor.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Prolonged AVT induces progressive serological proteomic shifts in fibrosis regressors and non-regressors</h3>
<p id="Par9">Following short-term AVT, fibrosis regressors exhibited 25 uniquely decreased and 9 increased proteins, while non-regressors showed 38 exclusively decreased and 17 increased proteins, with 50 proteins commonly downregulated and 11 upregulated in both groups (Fig.Â <a href="#Fig3" class="usa-link">3a, b</a> and Supplementary TableÂ <a href="#MOESM1" class="usa-link">2</a>). However, post/pre ratios of these differentially abundant proteins (DAPs) demonstrated limited discriminatory capacity, with all area under the curve (AUC) valuesâ€‰&lt;â€‰0.70 for distinguishing regression status (Fig.Â <a href="#Fig3" class="usa-link">3a, b</a> and Supplementary TableÂ <a href="#MOESM1" class="usa-link">2</a>). In contrast, fibrosis regressors following long-term AVT displayed pronounced serological proteomic remodeling, characterized by 90 uniquely decreased and 33 increased proteins, whereas non-regressors showed minimal specific changes (13 decreased and 14 increased proteins), alongside 22 commonly downregulated and 10 upregulated proteins (Fig.Â <a href="#Fig3" class="usa-link">3c, d</a> and Supplementary TableÂ <a href="#MOESM1" class="usa-link">3</a>). Notably, 33 DAPs in regressors following long-term AVT achieved clinically relevant discriminatory power (AUCâ€‰&gt;â€‰0.70) using post/pre ratios, highlighting their potential as biomarkers for fibrosis regression (Fig.Â <a href="#Fig3" class="usa-link">3c, d</a> and Supplementary TableÂ <a href="#MOESM1" class="usa-link">3</a>). Comparatively, the extent of serological proteomic changes was more pronounced in fibrosis regressors than non-regressors after long-term AVT.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Abundance and functional alterations of serological proteomes in fibrosis regressors and non-regressors receiving AVT.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365141_41467_2025_63006_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5d4a/12365141/0b1457d8a809/41467_2025_63006_Fig3_HTML.jpg" loading="lazy" id="d33e1366" height="961" width="799" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, <strong>c</strong> Venn diagrams illustrate the specifically or commonly upregulated (<strong>â†‘</strong>) and downregulated (â†“) DAPs between fibrosis regressors and non-regressors following short-term or long-term AVT. DAPs post-AVT were identified using a two-sided paired Studentâ€™s t-test or a two-sided Wilcoxon signed-rank test, in conjunction with a FC criterion. Proteins meeting both criteria (p-valueâ€‰&lt;â€‰0.05 and FCâ€‰&gt;â€‰1.2) were defined as statistically significant. Short-term patients: nâ€‰=â€‰56 for R; nâ€‰=â€‰38 for NR. Long-term patients: nâ€‰=â€‰25 for R; nâ€‰=â€‰11 for NR. <strong>b</strong>, <strong>d</strong> Heatmaps depict the average abundance of DAPs in fibrosis regressors and non-regressors before and after AVT. The intensity of red indicates higher abundance, while darker shades of grey represent lower abundance. The AUC values, calculated based on the post/pre ratios of DAPs to differentiate fibrosis regressors from non-regressors, are visualized (Short-term patients: nâ€‰=â€‰56 for R; nâ€‰=â€‰38 for NR. Long-term patients: nâ€‰=â€‰25 for R; nâ€‰=â€‰11 for NR). Darker colors represent higher AUC values, and lighter colors correspond to lower AUC values. DAPs with AUC values Ã¥ 0.7 are marked. DAP groups are color-coded. <strong>e</strong>, <strong>f</strong> Significantly <strong>e</strong>nriched KEGG pathways of DAPs that were either unique to or shared between R and NR after short-term or long-term AVT. DAVID employs a modified Fisherâ€™s exact test (one-sided) for KEGG pathway enrichment analysis, with Benjamini-Hochberg correction for multiple testing. An adj.p valueâ€‰&lt;â€‰0.05 was considered statistically significant. <strong>g</strong> Venn diagrams show the representative pathways that were specifically or commonly enriched (â†“, decrease; â†‘, increase) between fibrosis regressors and non-regressors in the short-term or long-term patient groups. <em>Abbreviations</em>: AUC area under the curve, DAP differentially abundant protein, Post post-treatment, Pre pre-treatment, Short-R regressor in the short-term group, Short-NR non-regressor in the short-term group, Long-R regressor in the long-term group, Long-NR non-regressor in the long-term group.</p></figcaption></figure><p id="Par10">Subsequentially, Kyoto Encyclopedia of Genes and Genomes <strong>(</strong>KEGG) pathway enrichment analysis of DAPs stratified by fibrosis regression status and AVT duration revealed temporal pathway divergence. Specifically, decreased DAPsâ€”whether regressor-specific, shared, or non-regressor-specificâ€”triggered by short-term AVT were predominantly enriched in cell adhesion molecules; in contrast, increased DAPs that were specific to non-regressors or shared by both groups were significantly associated with complement-coagulation cascades (Fig.Â <a href="#Fig3" class="usa-link">3e</a>). Following long-term AVT, decreased DAPs (regressor-specific, shared, or non-regressor-specific) converged on fibrogenesis pathways, including cell adhesion molecules and extracellular matrix (ECM)-receptor interactions, while increased DAPs specific to regressors or shared by both groups sustained complement and coagulation cascades activation (Fig.Â <a href="#Fig3" class="usa-link">3f</a>). Notably, suppression of fibrogenesis persisted across both groups regardless of therapy duration; however, the complement-coagulation responses exhibited a temporal inversion: non-regressors dominated after short-term AVT, whereas regressors prevailed after long-term AVT (Fig.Â <a href="#Fig3" class="usa-link">3g</a>). This divergence underscores the capacity of sustained AVT to drive progressive serological proteomic remodeling in fibrosis regressors, surpassing the changes observed in non-regressors.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Serological proteomic panels for monitoring liver fibrosis regression in CHB patients following short-term AVT</h3>
<p id="Par11">Based on post/pre ratios of the specific DAPs identified in fibrosis regressors and non-regressors in response to short-term AVT from the training cohort, we employed five feature selection methodologies: univariate logistic regression, backward stepwise regression, least absolute shrinkage and selection operator (Lasso) regression, partial least squares (PLS) regression, and support vector machine (SVM). The selected protein panels were subsequently evaluated through four ML algorithms (logistic regression, PLS regression, SVM, and random forest) using ten iterations of fivefold cross-validation. Performance was independently validated in the corresponding testing cohort. As shown in Fig.Â <a href="#Fig4" class="usa-link">4a</a>, the logistic regression model incorporating DAPs selected through univariate feature selection demonstrated optimal performance, achieving training and testing AUCs of 0.85 (95% confidence interval [CI]: 0.74â€“0.95) and 0.66 (95% CI: 0.47â€“0.84), respectively, for distinguishing fibrosis regression status. The final panel comprised seven proteins (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3a, b</a>): cholesteryl ester transfer protein (CETP), contactin 1 (CNTN1), paraoxonase 3 (PON3), isocitrate dehydrogenase 1 (IDH1), prosaposin (PSAP), serpin family A member 5 (SERPINA5), and complement factor H related 4 (CFHR4).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. ML models for discriminating liver fibrosis regressors following short-term AVT.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365141_41467_2025_63006_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5d4a/12365141/e23fdac864ca/41467_2025_63006_Fig4_HTML.jpg" loading="lazy" id="d33e1413" height="579" width="749" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> AUC values (95% CI) of ML models developed using five feature selection methods and four ML algorithms. These models were constructed from the training set and validated in the testing set derived from short-term patients in the 4D-DIA-MS discovery cohort. The AUC value (95% CI) of the optimal short-term panel is highlighted in dark red. Training set: nâ€‰=â€‰34 for R; nâ€‰=â€‰23 for NR. Testing set: nâ€‰=â€‰22 for R; nâ€‰=â€‰15 for NR. <strong>b</strong> Receiver operating characteristic (ROC) curve analysis of the selected short-term panel and ML model for monitoring fibrosis regression status in short-term patients from the PRM-MS validation cohort (IDH1 was not detected, which was imputed using the mean abundance of IDH1 in the training set). Data were presented as AUC values with 95% CI. PRM-MS validation cohort: nâ€‰=â€‰32 for R; nâ€‰=â€‰22 for NR. <strong>c</strong> AUC values (95% CI) of the selected optimal short-term panel and ML model across clinically relevant populations, including patients with baseline Ishak scores â‰¥ 3, non-steatosis patients at baseline, and those with Î”Ishak â‰¥ 1 post-short-term AVT. These values are presented for both the 4D-DIA-MS discovery cohort (including the training set, testing set, and the total of both sets) and the PRM-MS validation cohort (IDH1 was not detected, which was imputed using the mean abundance of IDH1 in the training set). Detailed subgroup sample sizes are provided in Supplementary TableÂ <a href="#MOESM1" class="usa-link">1</a>. <em>Abbreviations</em>: 4D-DIA-MS four-dimensional data-independent acquisition mass spectrometry, AUC area under the curve, PLS partial least squares, PRM-MS parallel reaction monitoring mass spectrometry, SVM support vector machine.</p></figcaption></figure><p id="Par12">To address potential overfitting, we first performed internal validation using 1,000 bootstrap resamples in the training set (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">4a, b</a>). The optimism-corrected AUC remained stable at 0.85 (medium), with calibration analysis showing strong agreement between predicted and observed outcomes (Brier score = 0.154). This performance substantially exceeded that of conventional NITs, including the post/pre ratios of LSM (AUCâ€‰=â€‰0.64, 95% CI: 0.56â€“0.72; Brier score = 0.231), APRI (AUCâ€‰=â€‰0.56, 95% CI: 0.50â€“0.61; Brier score = 0.238), and FIB4 (AUCâ€‰=â€‰0.56, 95% CI: 0.50â€“0.62; Brier score = 0.241) in predicting fibrosis regression. Then, external validation in an independent PRM-MS cohort confirmed the generalizability of the model, yielding an AUC of 0.79 (95% CI: 0.66-0.91) for fibrosis regression status discrimination (Fig.Â <a href="#Fig4" class="usa-link">4b</a>). Moreover, subgroup analyses demonstrated consistent performance across clinically relevant populations: baseline Ishak â‰¥ 3, non-steatosis patients at baseline, and those with Î”Ishak â‰¥ 1 post-AVT in both the 4D-DIA-MS discovery cohort (training set, testing set, and the total of both sets) and the PRM-MS validation cohort (Fig.Â <a href="#Fig4" class="usa-link">4c</a>). Collectively, this optimized serological proteomic panel demonstrates robust performance across multiple validation frameworks and clinical subgroups, establishing its potential as a non-invasive biomarker for monitoring short-term AVT-induced fibrosis regression in CHB patients.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Serological proteomic panels for monitoring liver fibrosis regression in CHB patients after long-term AVT</h3>
<p id="Par13">To identify serological biomarkers of fibrosis regression following long-term AVT, we extended our analytical pipeline to evaluate post/pre ratios of specific DAPs in fibrosis regressors and non-regressors undergoing prolonged AVT. Using five feature selection methods and four ML algorithms with ten iterations of fivefold cross-validation, univariate logistic regression identified a minimal serological panel comprising three proteins: serpin family A member 7 (SERPINA7), CD163 molecule (CD163), and CFHR4 (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3c, d</a>). While the random forest model incorporating this three-protein panel showed peak training performance (AUCâ€‰=â€‰0.93, 95% CI: 0.84â€“1.00) and testing performance (AUCâ€‰=â€‰0.70, 95% CI: 0.40â€“1.00), it exhibited limited generalizability in the external PRM-MS validation cohort (AUCâ€‰=â€‰0.55, 95% CI: 0.47â€“0.63) (Fig.Â <a href="#Fig5" class="usa-link">5a, b</a>). In contrast, the logistic regression model incorporating the same three-protein panel demonstrated consistent discriminative capacity across the training set (AUCâ€‰=â€‰0.87, 95% CI: 0.71â€“1.00), testing set (AUCâ€‰=â€‰0.68, 95% CI: 0.28â€“1.00), and external PRM-MS validation cohort (AUCâ€‰=â€‰0.75, 95% CI 0.63â€“0.88) (Fig.Â <a href="#Fig5" class="usa-link">5a, b</a>). Internal validation via 1,000 bootstrap resamples confirmed the robustness of the model, with an optimism-adjusted AUC of 0.87 and excellent calibration (Brier score = 0.138), surpassing conventional NITs, including the post/pre ratios of LSM (AUCâ€‰=â€‰0.59, Brier = 0.214), APRI (AUCâ€‰=â€‰0.72, Brier = 0.184), and FIB-4 (AUCâ€‰=â€‰0.77, Brier = 0.195) (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">4c, d</a>). Additionally, subgroup analyses further validated stable performance among clinically relevant strata (baseline Ishak â‰¥ 3, non-steatosis at baseline, and Î”Ishak â‰¥ 1 post-AVT) across both discovery and validation cohorts (Fig.Â <a href="#Fig5" class="usa-link">5c</a>), underscoring the stability and generalizability of the model for diverse patient populations.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. ML models for discriminating liver fibrosis regressors following long-term AVT.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365141_41467_2025_63006_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5d4a/12365141/122e0bf15238/41467_2025_63006_Fig5_HTML.jpg" loading="lazy" id="d33e1467" height="581" width="749" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> AUC values (95% CI) for ML models developed using five feature selection methods in conjunction with four ML algorithms. These models were constructed from the training set and subsequently validated in the testing set derived from long-term patients in the 4D-DIA-MS discovery cohort. The AUC values (95% CI) for the optimal long-term panels are highlighted in dark red. Training set: nâ€‰=â€‰15 for R; nâ€‰=â€‰7 for NR. Testing set: nâ€‰=â€‰10 for R; nâ€‰=â€‰4 for NR. <strong>b</strong> ROC curve analysis of the selected long-term panel, used to monitor fibrosis regression status in long-term patients from the PRM-MS validation cohort (CD163 was not detected, which was imputed using the mean abundance of IDH1 in the training set). Data were presented as AUC values with 95% CI. PRM-MS validation cohort: nâ€‰=â€‰42 for R; nâ€‰=â€‰12 for NR. <strong>c</strong> AUC values (95% CI) of the ML models across clinically relevant populations, including patients with baseline Ishak scores â‰¥ 3, non-steatosis patients at baseline, and those exhibiting a Î”Ishak â‰¥ 1 post-long-term AVT. These values are presented for both the 4D-DIA-MS discovery cohort (comprising the training set, testing set, and the aggregate of both sets) and the PRM-MS validation cohort (CD163 was not detected, which was imputed using the mean abundance of IDH1 in the training set). Detailed subgroup sample sizes are provided in Supplementary TableÂ <a href="#MOESM1" class="usa-link">1</a>. <em>Abbreviations</em>: 4D-DIA-MS four-dimensional data-independent acquisition mass spectrometry, AUC area under the curve, PLS partial least squares, PRM-MS parallel reaction monitoring mass spectrometry, SVM support vector machine.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Comparison of short-term and long-term serological proteomic panels for monitoring liver fibrosis regression in on-treatment CHB patients</h3>
<p id="Par14">A systematic comparison of short-term and long-term proteomic panels revealed distinct temporal dynamics in their ability to monitor AVT-induced fibrosis regression. As illustrated in Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5</a>, the short-term panel lost its discriminative capacity for regression status after prolonged AVT, while the long-term panel similarly failed to stratify outcomes following short-term AVT (AUCâ€‰&lt;â€‰0.60 across discovery and its subgroup cohorts). This finding confirms the duration-dependent remodeling of serological proteomes between fibrosis regressors and non-regressors. Subsequently, cross-panel analysis identified CFHR4 as the sole conserved biomarker shared between the long-term and short-term panels. CFHR4 demonstrated moderate diagnostic performance for long-term AVT response across the training set (AUCâ€‰=â€‰0.81, 95% CI: 0.60â€“1.00), testing set (AUCâ€‰=â€‰0.70, 95% CI: 0.39â€“1.00), and combined discovery cohort (AUCâ€‰=â€‰0.72, 95% CI: 0.53â€“0.90), but a poor diagnostic performance in the external PRM-MS validation cohort (AUCâ€‰=â€‰0.60, 95% CI: 0.43â€“0.77) (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">6</a>).</p>
<p id="Par15">Additionally, longitudinal trajectory analysis revealed critical divergences in protein behavior. Long-term panel proteins, including SERPINA7, CD163, and CFHR4, exhibited consistent directional changes across both the 4D-DIA-MS discovery and PRM-MS validation cohorts. Specifically, SERPINA7 (CD163 was not detected) was downregulated, while CFHR4 was upregulated in regressors after long-term AVT (Fig.Â <a href="#Fig6" class="usa-link">6a</a>). Notably, these proteins displayed a trend of inverse trajectories during pre-treatment fibrogenesisâ€”SERPINA7 and CD163 (pâ€‰&lt;â€‰0.05) were elevated, whereas CFHR4 was suppressed in cases of progressive fibrosis (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">7</a>)â€”establishing their dual relevance as biomarkers for both fibrogenesis and regression. In stark contrast, short-term panel proteins exhibited cohort-specific variability during AVT (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">8</a>), with inconsistent inverse relationships to baseline fibrotic progression (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">7</a>). This result underscores superior biological fidelity of long-term proteomic signatures, which consistently reflect bidirectional fibrosis dynamics compared to the transient and context-dependent fluctuations observed in short-term markers. Furthermore, functional characterization through gene set enrichment analysis (GSEA) linked the dynamics of the long-term panel (CFHR4 upregulation and CD163 downregulation) to the regulation of complement-coagulation cascades during sustained AVT (Fig.Â <a href="#Fig6" class="usa-link">6b</a>). Collectively, these findings establish the temporal specificity of proteomic biomarkers in fibrosis monitoring and highlight the clinical necessity for duration-adapted serological panels. Additionally, long-term signatures offer robust mechanistic insights into AVT-induced fibrosis regression, potentially through the modulation of coagulation and complement pathways.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Longitudinal dynamics and functional characterization of long-term proteomic panel biomarkers.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365141_41467_2025_63006_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5d4a/12365141/f8104dfea6df/41467_2025_63006_Fig6_HTML.jpg" loading="lazy" id="d33e1518" height="789" width="789" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Box plots with overlaid longitudinal trajectories illustrate the dynamic alterations in serological levels of long-term panel proteins (SERPINA7, CFHR4, CD163) between baseline (Week 0) and post-long-term AVT (Week 260) in fibrosis regressors versus non-regressors. Data are derived from the 4D-DIA-MS discovery cohort (training set: nâ€‰=â€‰15 pairs for R and nâ€‰=â€‰7 pairs for NR; testing set: nâ€‰=â€‰10 pairs for R and nâ€‰=â€‰4 pairs for NR) and the PRM-MS validation cohort (nâ€‰=â€‰42 pairs for R and nâ€‰=â€‰12 pairs for NR). All paired serum samples were biological replicates, with each pair originating from an individual patient. Normalized protein abundances were compared using a two-sided paired Studentâ€™s t-test (for normally distributed data) or a two-sided Wilcoxon signed-rank tests (for non-parametric data). A p-valueâ€‰&lt;â€‰0.05 was considered statistically significant. The box plots display the median (50<sup>th</sup> percentile; centre line), the 25<sup>th</sup> and 75<sup>th</sup> percentiles (box bounds), and the minima and maxima (whiskers) within 1.5 times the interquartile range (IQR) from the box bounds. <strong>b</strong> GSEA was performed on the complement and coagulation cascade pathway using proteins (CFHR4 and CD163) stratified into high- and low-expression groups based on median abundance thresholds in the 4D-DIA-MS discovery cohort. GSEA uses permutation-based significance testing to evaluate directional enrichment of gene sets in ranked gene lists, inherently employing a one-sided approach to detect coordinated over-representation at extremes of the expression profile. The enrichment plot displays the trajectory of the enrichment score across rank-ordered proteins, with the leading edge subset (indicated by the vertical red line) representing the proteins driving pathway enrichment. The p-value for the enrichment analysis is shown in the plot, highlighting the statistical significance of the enrichment. <em>Abbreviations</em>: 4D-DIA-MS four-dimensional data-independent acquisition mass spectrometry, NR non-regressor, PRM-MS parallel reaction monitoring mass spectrometry, R regressor.</p></figcaption></figure></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par16">The high prevalence of HBV-related liver fibrosis remains a significant contributor to end-stage cirrhosis and hepatocellular carcinoma<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, highlighting the growing need for reliable NITs to assess liver fibrosis. Current diagnostic approaches prioritize NITs such as APRI, FIB-4, and imaging elastography as first-line evaluations for antiviral-naÃ¯ve patients with HBV-related liver fibrosis<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. However, these methods present notable limitations, including indeterminate results and a stronger correlation with liver inflammatory activity than with histological fibrotic remodeling<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>â€“<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, which diminishes their utility in reflecting LBx-verified fibrosis regression following AVT. Additionally, emerging evidence suggests that baseline heterogeneity in host-pathogen interactions may also influence AVT-induced liver fibrosis regression<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, posing challenges for both the implementation of conventional NITs and the development of novel biomarkers to monitor fibrotic resolution during treatment.</p>
<p id="Par17">Our prior investigation established the LBx-dependent â€œBeijing Classificationâ€ system as a validated framework for assessing post-AVT fibrosis regression<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. While histological evaluation remains the gold standard, serum-based NITs may offer distinct advantages in clinical practicality and patient acceptability<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Leveraging advancements in high-throughput proteomics, we employed cutting-edge 4D-DIA-MS to analyze paired serum samples from CHB patients with histologically confirmed liver fibrosis undergoing AVT. The 4D-DIA-MS technique combines data-independent acquisition with ion mobility separation, enhancing the accuracy and depth of proteomic quantification<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Using this innovative platform, we identified over 500 reproducibly quantified proteins across longitudinal serum samples following rigorous quality control. However, unsupervised clustering approaches were unable to differentiate between fibrosis regressors and non-regressors after AVT, based on the post/pre ratios or scale-invariant measures of changes in these robust proteins. This convergence in serological proteome dynamics likely reflects near-universal virological suppression (undetectable HBV DNA) and inflammatory normalization (ALT normalization) across our cohort, even with short-term AVT. These findings further support the notion that serological proteomic fluctuations during AVT primarily reflect virological and inflammatory normalization, presenting substantial challenges in identifying serological protein biomarkers of histological fibrosis improvement.</p>
<p id="Par18">While serological proteomic trajectories showed broad similarities between fibrosis regressors and non-regressors during AVT, longitudinal analysis revealed that prolonged AVT drives divergent proteomic remodeling in regressors, surpassing the changes observed in non-regressors. This aligns with our recent clinical and transcriptomic observations demonstrating the superior fibrosis regression rates following long-term versus short-term AVT (80.4% versus 65.2%, respectively)<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, coupled with the enhanced restoration of ECM-related gene signatures specifically in regressors during prolonged therapy<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. These temporal patterns suggest that the increased detection of reduced DAPs in regressors after long-term AVT likely reflects cumulative histological improvements achieved through sustained AVT. Intriguingly, proteomic profiling uncovered treatment-duration-dependent shifts in pathway activation. Although fibrogenesis suppression persisted in both groups irrespective of AVT duration, short-term AVT preferentially amplified complement-coagulation cascades in non-regressorsâ€”a pattern reversed with prolonged therapy, where regressors displayed dominant activation of these pathways. Studies have demonstrated that the complement and coagulation system plays a pivotal role in liver homeostasis and immune responses, with its activation contributing to beneficial effects on liver immune response and regeneration<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Therefore, the temporal switch in complement-coagulation activation implies that long-term AVT may synergistically enhance immunoregulatory competence and microenvironmental homeostasis, even without overt fibrosis regression.</p>
<p id="Par19">Currently, while ML offers transformative potential for biomarker prioritization in omics discovery pipelines, translational implementation remains challenging due to frequent overfitting and lack of clinical validation<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. To address these limitations, we implemented rigorous partitioning of our 4D-DIA-MS proteomic dataset into independent training and testing cohorts prior to feature selection, ensuring no data leakage during model development<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. This strategy capitalized on the strength of 4D-DIA-MS in generating reproducible, untargeted proteomic profilesâ€”ideal for unbiased discoveryâ€”while leveraging the capacity of ML to decode complex biomarker signatures. Notably, the optimized logistic regression models, incorporating a seven-protein panel for short-term AVT and a three-protein panel for long-term AVT, demonstrated superior discrimination of fibrosis regression compared to conventional NITs in internal validation. Critically, we bridged discovery to clinical applicability through PRM-MS-based external validation in an independent cohort, exploiting the precision of PRM for targeted quantification<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, and confirmed the model robustness across platforms. Subgroup analyses further reinforced clinical utility, showing consistent performance across diverse populations (baseline Ishak â‰¥ 3, non-steatosis patients at baseline) and treatment responders (Î”Ishak â‰¥ 1) in both 4D-DIA-MS discovery and PRM-MS validation phases. This multilayered validation frameworkâ€”spanning technical reproducibility (4D-DIA-to-PRM transition), cohort independence (training-testing-external cohorts), and clinical heterogeneity (subgroup consistency)â€”establishes a paradigm for translating ML-derived proteomic signatures into clinically actionable tools. Moreover, we systematically evaluated the incremental value of clinical parameters alongside serological proteomic features during model training. Unfortunately, integrative models combining protein and clinical data failed to outperform serological protein-only classifiers in discriminating fibrosis regression (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">9</a>), suggesting that proteomic signatures may capture latent biological signals beyond conventional clinical metricsâ€”a hypothesis requiring further mechanistic investigation.</p>
<p id="Par20">Our longitudinal proteomic analyses revealed notable differences in AVT-induced biological remodeling based on treatment duration. Long-term AVT resulted in more pronounced serological proteome reorganization and pathway activation in fibrosis regressors compared to short-term AVT, highlighting duration-dependent functional adaptations. This temporal divergence provides a mechanistic basis for the observed specificity of our diagnostic panels: the short-term model (7-protein panel) was unable to effectively discriminate fibrosis regression status at long-term endpoints, while the long-term panel (3-protein signature) demonstrated limited utility during the early treatment phases. Importantly, the proteins included in the long-term panel exhibited consistent directional changes across both the 4D-DIA discovery and PRM validation cohorts, showing inverse trajectories during pre-treatment fibrogenesis. In contrast, the constituents of the short-term panel displayed inter-cohort variability in expression patterns, showing inconsistent inverse relationships with baseline fibrotic progression. The modest proteomic perturbations observed during the early treatment phases, combined with the biological noise inherent to acute therapeutic responses, likely contribute to the inter-cohort inconsistency and temporal instability of the short-term signatures. In contrast, the robustness of the long-term panelâ€”validated across orthogonal platforms and diverse populationsâ€”positions it as a clinically actionable tool for identifying non-regressors after prolonged AVT. Except for the superior stability, the long-term panel also exhibited mechanistic relevance to complement-coagulation cascades, aligning with sustained functional modulation following sustained AVT. This stratification capability has direct translational relevance, enabling intensified monitoring of high-risk patients to mitigate the incidence of decompensated cirrhosis and hepatocellular carcinoma through timely clinical interventions.</p>
<p id="Par21">To our knowledge, this study represents the pioneering longitudinal proteomic investigation characterizing on-treatment serological dynamics and biomarker trajectories associated with histologically confirmed fibrosis regression in CHB patients. The study presents several notable advantages. First, we used paired LBx-validated serum samples from multicenter randomized controlled trials, ensuring clinical relevance through standardized enrollment criteria and antiviral regimens<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The histopathological rigor was reinforced by robust inter-observer agreement (Îºâ€‰=â€‰0.65; ICCâ€‰=â€‰0.71, 95% CI: 0.65â€“0.76), exceeding the thresholds necessary for reproducible fibrosis staging<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Second, we employed a dual-phase discovery-validation framework: 4D-DIA-MS facilitated unbiased proteome-wide profiling, while PRM-MS provided targeted verification, leveraging their complementary strengths in discovery depth and quantification precision<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Third, our multilayered validation strategy, which included internal bootstrapping, external PRM-MS validation, and subgroup analyses across different populations, confirmed the generalizability of the models while mitigating the risk of overfitting. However, our study also has limitations. 1. There was cohort asymmetry, with the long-term group having fewer paired serum samples than the short-term group in the 4D-DIA-MS discovery cohort, which may limit the power to detect subtle proteomic shifts. 2. While 4D-DIA-MS demonstrated robust reproducibility, its detection threshold may have excluded low-abundance proteins that are critical to fibrotic resolution. 3. The failure in quantifying IDH1 in the short-term panel and CD163 in the long-term panel by PRM-MS technique, which may have led to an underestimation of diagnostic performance. 4. Although serological biomarkers offer non-invasive clinical utility, their expression levels and regulatory dynamics may differ from those in liver tissue due to compartmentalized pathophysiology. Future studies integrating paired tissue-serum proteomics are needed to resolve this critical knowledge gap.</p>
<p id="Par22">In conclusion, our study addressed a critical unmet need in CHB-related liver fibrosis management by linking proteomic dynamics to histological outcomes, offering actionable tools for monitoring fibrosis regression; the association of the long-term panel with complement-coagulation activation offers valuable mechanistic insights into AVT-induced functional restoration (Fig.Â <a href="#Fig7" class="usa-link">7</a>). This study advances non-invasive fibrosis monitoring by integrating cutting-edge proteomics with ML-driven biomarker discovery. Its strengths lie in technical rigor, validation depth, and biological plausibility of identified pathways. However, limitations in cohort size, diagnostic performance, and mechanistic elucidation highlight the need for multicenter validation and functional studies to translate these findings into clinical practice.</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7. Dynamic serological proteomic signatures and biomarker panels for monitoring fibrosis regression in on-treatment CHB patients.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365141_41467_2025_63006_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5d4a/12365141/7a33e092c901/41467_2025_63006_Fig7_HTML.jpg" loading="lazy" id="d33e1640" height="424" width="698" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prolonged AVT induced progressive serological proteomic and functional shifts in both fibrosis regressors and non-regressors. Notably, serological proteomic recovery was more pronounced in fibrosis regressors compared to non-regressors following long-term AVT. Additionally, a seven-protein panel was identified for monitoring fibrosis regression during short-term AVT, while a three-protein panel was established for long-term AVT. <em>Abbreviations</em>: AVT antiviral therapy, CHB chronic hepatitis B, DAP differentially abundant protein, NR non-regressor, R regressor.</p></figcaption></figure></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Study population</h3>
<p id="Par23">CHB patients with liver fibrosis undergoing AVT were screened from our previous nationwide multicenter randomized controlled clinical trials and their extension studies (ClinicalTrials.gov identifiers: <a href="https://clinicaltrials.gov/ct2/show/NCT01938781" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT01938781</a>, <a href="https://clinicaltrials.gov/ct2/show/NCT01938820" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT01938820</a>, <a href="https://clinicaltrials.gov/ct2/show/NCT03777969" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT03777969</a>, and <a href="https://clinicaltrials.gov/ct2/show/NCT02849132" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT02849132</a>). Inclusion criteria: age 18 years or older; availability of paired LBx pre- and post-treatment (either 78 or 260 weeks of AVT) or clinically diagnosed compensated cirrhosis at baseline with post-treatment LBx; baseline fibrosis Ishak score â‰¥ 2 if the pre-treatment LBx was available; availability of serum samples at both baseline and the time of post-treatment LBx; hepatitis B surface antigen (HBsAg) positive for at least 6 months before AVT; HBV DNA levels &gt; 20,000 IU/ml for hepatitis B e antigen (HBeAg)-positive patients or &gt; 2000 IU/ml for HBeAg-negative patients; and length of the LBx core â‰¥ 0.5â€‰cm with at least five portal tracts. Exclusion criteria: current or historical liver decompensation (ascites, variceal bleeding, or encephalopathy); elevated serological alpha-fetoprotein (&gt;â€‰100â€‰ng/mL) or creatinine (â€‰&gt;â€‰1.5 times the upper normal limit); coinfection with hepatitis C or human immunodeficiency virus; other severe liver conditions or serious organ disorders; and pregnancy or lactation. Patients who met the inclusion and exclusion criteria were divided into a discovery group and a validation group for subsequent serological biomarker discovery. To ensure internal consistency across all participating centers, we implemented standardized protocols that included the following aspects: all centers uniformly applied identical eligibility criteria for patient enrollment; all participants received a standardized AVT regimen with entecavir as the therapeutic backbone; systematic clinical evaluations were conducted at six-month intervals following established follow-up protocols; and the primary outcomes focused on histological fibrosis regression and the incidence of predefined clinical endpoints. All enrolled patients provided written informed consent. The study was approved by the Ethics Committee of Beijing Friendship Hospital, Capital Medical University (2016-P2-021-04), and conducted in accordance with the Declaration of Helsinki. Patients and/or the public were not involved in the design, conduct, reporting, or dissemination plans of our research.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Clinical examination</h3>
<p id="Par24">The clinical examination encompasses the collection of demographic data, routine blood tests, liver biochemistry analyses, and LSM. Detailed methodologies for these procedures are thoroughly documented in our prior publications<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Histological assessment</h3>
<p id="Par25">Formalin-fixed paraffin-embedded blocks of LBx were sectioned into 5-Î¼m-thick slices, stained with reticulin, and assessed by two pathologists who were blinded to the experimental conditions, timing of the LBx, and other clinical details. Inter-observer agreement for histopathological scoring was assessed using ICC and Cohenâ€™s kappa (Îº), with 95% confidence intervals calculated via bootstrapping (1000 iterations). Discordant cases were reviewed by a third senior pathologist to reach a consensus. Histological changes were evaluated using Ishak and/or PIR scores<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. The PIR score represents a novel pathological classification designed to assess the dynamic changes in fibrosis quality<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. It categorizes fibrosis into three types: predominantly <em>progressive</em> (characterized by thick, broad, loose, and pale septa with inflammation), predominantly <em>regressive</em> (marked by delicate, thin, dense, and splitting septa), and <em>indeterminate</em> (exhibiting a balance between progressive and regressive scarring). This classification underscores the significance of evaluating and recognizing the evolving nature of fibrosis, particularly in the context of AVT. For patients with paired LBx, a fibrosis regressor was defined as either a decrease in the Ishak score by â‰¥ 1 after treatment (Î”Ishak â‰¥ 1) or a stable Ishak score accompanied by predominantly <em>regressive</em> changes according to the post-treatment PIR score. For patients clinically diagnosed with compensated cirrhosis before treatment, a fibrosis regressor was identified as having an Ishak stage â‰¤ 4 post-treatment (Î”Ishak â‰¥ 1) or exhibiting predominantly <em>regressive</em> changes in those with Ishak fibrosis stages 5â€“6 after treatment. Conversely, a fibrosis non-regressor was defined as a patient who did not meet the criteria for fibrosis regression. Additionally, patients without liver steatosis at baseline (non-steatosis) were defined as having histological steatosis of less than 5% before treatment for those with paired LBx. For patients clinically diagnosed with compensated cirrhosis prior to treatment, non-steatosis was determined based on the absence of signs of fatty liver detected on ultrasound.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Clinical diagnosis of compensated cirrhosis</h3>
<p id="Par26">Patients diagnosed with compensated cirrhosis should meet at least one of the following criteria. (1) Presence of endoscopic examination: An endoscopic examination revealed esophageal and/or gastric varices, with the exclusion of non-cirrhotic portal hypertension. (2) Absence of endoscopic examination: If an endoscopic examination has not been performed, at least two of the following four criteria must be satisfied. i. Imaging examination: Ultrasound, computed tomography, or magnetic resonance imaging indicated typical changes associated with cirrhosis, including an irregular liver surface, granular or nodular liver parenchyma, with or without splenomegaly (defined as spleen thickness exceeding 4.0â€‰cm or greater than 5 rib units). ii. Hematological indicators: Routine blood tests showed a platelet count of &lt;100â€‰Ã—â€‰10<sup>9</sup>/L, with no alternative explanation. iii. Serum and coagulation abnormalities: Serum albumin levels &lt;35â€‰g/L or prothrombin time extended by more than 3â€‰s (after discontinuation of thrombolytic or anticoagulant drugs for more than 7 days), or decreased cholinesterase levels (excluding the influence of anti-cholinesterase drugs). iv. Liver stiffness: LSMâ€‰&gt;â€‰12.4â€‰kPa.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Serum sample preparation</h3>
<p id="Par27">Blood samples were collected using standard venepuncture protocols into pro-coagulation vacuum tubes. Serum was obtained by centrifugation at 376â€‰Ã—â€‰<em>g</em> for 10â€‰min at 4â€‰Â°C and immediately frozen at âˆ’80â€‰Â°C until use. Highly abundant serological proteins in each 40â€‰Î¼L sample were removed using the ProteoExtractÂ® Albumin/IgG Removal Kit (MilliPore, MA, USA) following the vendorâ€™s protocol, as albumin constitutes 70â€“90% of the serological proteome. The depleted serum was freeze-dried and reconstituted in SDS lysis buffer (Beyotime, Hangzhou, China). After centrifugation at 12,000â€‰Ã—â€‰<em>g</em> for 10â€‰min at room temperature, the supernatant was collected, and protein concentration was detected using a bicinchoninic acid assay. Each sample containing 10â€‰Î¼g of protein was loaded onto a 12% SDS-PAGE gel for separation. The gel was stained with Coomassie Brilliant Blue using eStain LG (GenScript, Nanjing, China) and analyzed with an automatic digital gel imaging system (Tanon, Shanghai, China).</p>
<p id="Par28">Equal amounts of protein (50â€‰Î¼g) were adjusted to a consistent concentration and volume based on the measured protein concentration. Samples were then treated with dithiothreitol (Thermo Fisher Scientific, MA, USA) at a final concentration of 5â€‰mM and incubated at 55â€‰Â°C for 30â€‰min. Subsequently, 10â€‰mM iodoacetamide (Thermo Fisher Scientific) was added and allowed to react in the dark for 15â€‰min at room temperature. Protein precipitation was achieved by mixing samples with six volumes of pre-cooled acetone and incubating at âˆ’20â€‰Â°C for overnight. After centrifugation at 8000â€‰Ã—â€‰<em>g</em> for 10â€‰min at 4â€‰Â°C, the resulting pellets were dissolved in 100â€‰Î¼L of 50â€‰mM ammonium bicarbonate solution (Thermo Fisher Scientific) and digested with Trypsin (1â€‰Î¼g, Likuso, Shanghai, China) at 37â€‰Â°C overnight. The enzymatic reaction was halted by adjusting pH to 3 with phosphoric acid (Sigma, MO, USA). Peptides were desalted using SOLAâ„¢ SPE (Thermo Fisher Scientific), freeze-dried in a vacuum freeze dryer, and then reconstituted with iRT peptides (1:10, Biognosys, Zug, Switzerland).</p>
<p id="Par29">The peptide samples underwent fractionation using an Agilent 1100 HPLC System (Agilent, CA, USA). A gradient of mobile phases A (2% acetonitrile in HPLC-grade water) and B (90% acetonitrile in HPLC-grade water) was employed for reverse-phase chromatography. The gradient was programmed as follows: from 0 to 10â€‰min, 2% B; from 10 to 10.01â€‰min, 2% to 5% B; from 10.01 to 37â€‰min, 5% to 20% B; from 37 to 48â€‰min, 20% to 40% B; from 48 to 48.01â€‰min, 40% to 90% B; from 48.01 to 58â€‰min, 90% B; from 58 to 58.01â€‰min, 90% to 2% B; from 58.01 to 63â€‰min, 2% B. Separation of tryptic peptides occurred at a flow rate of 250â€‰Î¼L/min and was monitored at 210â€‰nm and 280â€‰nm. Fractions were collected every minute from 10 to 50â€‰minutes into centrifuge tubes 1â€“10 sequentially, repeating this collection sequence until the gradient was complete. The separated peptides were subsequently lyophilized for mass spectrometry (MS) analysis.</p></section><section id="Sec16"><h3 class="pmc_sec_title">4D-DIA-MS analysis</h3>
<p id="Par30">Serological proteome of the discovery cohort was quantified using 4D-DIA-MS by Shanghai OE Biological Technology Co., Ltd. (Shanghai, China), following established protocols. Liquid chromatography-tandem mass spectrometry (LC-MS/MS) was performed using an Evosep One system (Evosep Biosystems, Odense, Denmark) coupled to a timsTOF Pro 2 mass spectrometer (Bruker Daltonics, MA, USA). Samples were loaded onto Evotip C18 traps at 1â€‰Î¼L/min and separated on a 15-cm analytical column using the standardized 30SPD method (15-min effective gradient). Mobile phases consisted of 0.1% (v/v) formic acid in water (Solvent A) and 0.1% (v/v) formic acid in acetonitrile (Solvent B). Mass spectrometry operated in PASEF mode<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> with shared parameters: capillary voltage at 1.5â€‰kV, dry gas flow of 3.0â€‰L/min at 180â€‰Â°C, m/z range of 100â€“1700, and ramped collision energy (20â€“59â€‰eV). For data-dependent acquisition (DDA), precursors were fragmented across ion mobility (1/K<sub>0</sub>) values of 0.85â€“1.3â€‰Vs/cm<sup>2</sup> using intensity-triggered MS/MS (isolation width: Â±1â€‰m/z; topN=12). Data-independent acquisition (DIA) employed extended ion mobility (0.6â€“1.6â€‰Vs/cm<sup>2</sup>) with variable isolation windows covering the full m/z range, synchronizing cycle time with chromatographic peak width. DDA established spectral libraries, while DIA enabled quantitative profiling.</p>
<p id="Par31">MS/MS spectra preprocessing was performed using Spectronaut Pulsarâ„¢ (v15.3, Biognosys) under the following parameters: Trypsin was specified as the proteolytic enzyme with allowance for up to two missed cleavages; methionine oxidation and N-terminal carbamidomethylation were set as variable and fixed modifications, respectively; a 1% false discovery rate (FDR) threshold was applied at three hierarchical levels (peptide-spectrum match, peptide, and protein) using the Target-Decoy strategy with mProphet algorithm implementation<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. DDA-derived spectral libraries were used for DIA data processing, with precursor, peptide, and protein FDR thresholds set to 1%. Protein identification was conducted against the UniProt-reviewed human proteome database (uniprot-reviewed_yes+taxonomy_9606.fasta). Quantification was executed at the MS2 level using an intensity-based local normalization strategy, where peptide quantities were derived from the mean intensity of the top 1â€“3 ranked precursors, and protein quantities were subsequently calculated as the mean of 1â€“3 highest-abundance peptides<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. To ensure data quality, the abundance matrix of serological proteins was cleaned by removing immunoglobulin or its fractions and proteins with over 50% missing data. Missing data imputation and proteomic data normalization were performed using the <em>missForest</em> and <em>vsn</em> packages, respectively. Additionally, the coefficient of variation was employed to evaluate the abundance variability of each serological protein among individuals.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Expression and functional analyses of serological proteome</h3>
<p id="Par32">Changes in serological proteome post-AVT were analyzed by calculating the Post/Pre ratios of MS signal intensity and visualized using the <em>ComplexHeatmap</em> package. The divergence in serological protein abundance among individuals was assessed using the one minus Pearsonâ€™s Correlation Coefficient (1-r) with the <em>corrplot</em> package. HCL with average linkage and Euclidean distance metric was used to visualize similarities and differences in serological protein abundance among individuals using the <em>pheatmap</em> package. PCA was performed to identify categorical variables (principal components, PCs) in the data based on observed variables, serving to verify the results of the HCL. PCA was implemented using the <em>PCA</em> function from the <em>FactoMineR</em> package, with confidence ellipse drawn using the <em>factoextra</em> package with default parameters. DAPs post-AVT were identified using a two-sided paired Studentâ€™s t-test or a two-sided Wilcoxon signed-rank test, combined with the fold change (FC) criterion. Proteins with a p-valueâ€‰&lt;â€‰0.05 and FCâ€‰&gt;â€‰1.2 were considered statistically significant. Venn analysis was performed to identify specific and common DAPs between or among groups using the VENNY 2.1 tool (<a href="https://bioinfogp.cnb.csic.es/tools/venny/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioinfogp.cnb.csic.es/tools/venny/index.html</a>). Functional interpretation of DAPs was performed using KEGG pathways through the DAVID web server (<a href="https://davidbioinformatics.nih.gov" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://davidbioinformatics.nih.gov</a>)<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, with an adjusted p-valueâ€‰&lt;â€‰0.05 considered statistically significant. GSEA was performed to interpret the molecular functions of proteins of interest based on Sangerbox 3.0<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Samples were categorized into low and high subgroups based on the median Post/Pre ratio of protein of interest, with a p-valueâ€‰&lt;â€‰0.05 considered statistically significant. Visualization of DAPs of interest was implemented using <em>ggplot2</em> package.</p></section><section id="Sec18"><h3 class="pmc_sec_title">ML models</h3>
<p id="Par33">The ML analysis was conducted with the aim to identify potential serological proteins that could distinguish histological fibrosis regression in CHB patients with liver fibrosis following short-term or long-term AVT. Initially, the short-term or long-term data from 4D-DIA-MS discovery cohort were randomly divided into training and testing sets through stratified sampling on the regression status, comprising 60% and 40% of the patients, respectively, using the <em>caret</em> package. The training set was employed for model development, while the testing set was reserved for subsequent validation. Feature selection was then performed using five methods: univariate logistic regression, backward stepwise regression, Lasso regression, PLS regression, and SVM applied to specific DAPs in regressors or non-regressors from the training set. This process employed packages including <em>stats</em>, <em>MASS</em>, <em>glmnet</em>, <em>ropls</em>, and <em>e1071</em>. Following feature selection, ML models, including regularized Logistic regression with Ridge penalization, PLS regression, SVM, and random forest, were established based on the five selected features using the <em>glmnet</em>, <em>kernlab</em>, <em>e1071</em>, <em>randomForest</em>, <em>caret</em>, <em>MLmetrics</em>, and <em>pROC</em> packages. Models that incorporated relatively fewer serological proteins but demonstrated good performance in both training and testing sets were prioritized for further internal and external validations. Internal validation was conducted in the training set using bootstrap resampling (drawing datasets of equal size to the original training cohort with replacement over 1,000 iterations) and calibration curves by the <em>boot</em> and <em>calibrate</em> packages.</p></section><section id="Sec19"><h3 class="pmc_sec_title">PRM-MS analysis</h3>
<p id="Par34">To validate candidate short-term and long-term biomarker panels in the independent validation cohort, a customized PRM-MS assay was performed by Shanghai OE Biological Technology Co., Ltd. (Shanghai, China). This targeted approach prioritized maximal coverage of candidate proteins. Serum protein preparation and DDA-based spectral library construction followed identical protocols to those described for 4D-DIA-MS experiments, ensuring methodological consistency. Targeted proteomics via PRM-MS was performed on a timsTOF Pro 2 instrument (Bruker Daltonics) coupled to a nanoflow liquid chromatography system. Tryptic peptides were separated on a reversed-phase C18 column (75â€‰Î¼mâ€‰Ã—â€‰25â€‰cm, 1.6â€‰Î¼m, 120â€‰Ã…; IonOpticks, Melbourne, Australia) at 300â€‰nL/min using mobile phase A (0.1% formic acid in H<sub>2</sub>O) and B (0.1% formic acid in acetonitrile) with a 60â€‰min nonlinear gradient: 2â€‰â†’â€‰22% B (0â€“45â€‰min), 22â€‰â†’â€‰37% B (45â€“50â€‰min), 37â€‰â†’â€‰80% B (50â€“55â€‰min), and 80% B (55â€“60â€‰min). Mass spectrometry operated in positive-ion PRM-PASEF mode with precursor lists configured in timsControl PRM module. Key parameters included: 1.4â€‰kV capillary voltage, 100â€“1700â€‰m/z scan range, ion mobility window (1/K<sub>0</sub>) of 0.6â€“1.6â€‰Vs/cm<sup>2</sup>, 100â€‰ms ion accumulation/release time, and 10â€‰min scheduled retention time windows, enabling high-efficiency targeted quantification.</p>
<p id="Par35">Raw PRM data were analyzed with SpectroDive software (v11.8, Biognosys) through a two-step workflow: (1) automated peak extraction with subsequent manual validation and retention time alignment to ensure peptide identification accuracy and cross-sample consistency, followed by (2) protein quantification using the mean peptide quantity algorithm (arithmetic mean of normalized peptide intensities) embedded in the software. Sparse data points were imputed non-parametrically via the <em>missForest</em> package, and global normalization was applied using the <em>vsn</em> package.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par36">Statistical analysis was performed using R 4.4.3 (<a href="https://www.r-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.r-project.org/</a>) or GraphPad Prism 9 (GraphPad Software Inc., CA, USA). Continuous variables were presented as meanâ€‰Â±â€‰standard deviation or median (interquartile range), while categorical variables were summarized with counts and percentages. For comparisons between paired groups, continuous variables were analyzed using a two-sided paired Studentâ€™s t-test or a two-sided Wilcoxon signed-rank test, depending on the normality of data distribution. Continuous variables among three independent groups were compared using one-way ANOVA (for normally distributed data with equal variances) or the Kruskal-Wallis H test (for non-parametric comparisons). Categorical variables among three independent groups were analyzed using the Chi-square test (when expected frequencies â‰¥ 5) or the Fisher-Freeman-Halton exact test (when expected frequencies &lt;5). A p-value of &lt;0.05 was considered statistically significant.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par37">Further information on research design is available in theÂ <a href="#MOESM3" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365141/bin/41467_2025_63006_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365141/bin/41467_2025_63006_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Peer Review file</a><sup> (921.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365141/bin/41467_2025_63006_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (848.5KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by National Key Research and Development Program (2023YFC2306900 [to H.Y.]), National Natural Science Foundation of China (82430023 [to H.Y.], 82130018 [to HY], 82470646 [to W.C.], 82170613 [to W.C.], 82300696 [to S.C.]), Beijing Municipal Science &amp; Technology Commission (Z221100007422115 [to Y.S.]), and Young Talent Funds of Chinese Institutes for Medical Research (CX23YQB08 [to W.C.], CX23YQB12 [to Y.S.]).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conception and design: H.Y., W.C.; project supervision: H.Y., W.C.; sample collection: S.Y.C., Y.M.S., X.N.W., J.L.Z., B.Q.W., T.T.M., R.X.H.; data acquisition: M.Y.Z., W.C., H.Y., S.Y.C., Y.M.S.; data analysis: M.Y.Z., W.C.; data interpretation: M.Y.Z., W.C., H.Y., Y.M.S., S.Y.C.; manuscript drafting: W.C., M.Y.Z.; critical revision of the manuscript: W.C., M.Y.Z., H.Y.; all authors reviewed and edited this manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par38"><em>Nature Communications</em> thanks Jordan J. Feld and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The raw 4D-DIA-MS data, PRM-MS data, and their processed peptide and protein abundance data generated in this study have been deposited in the Proteomics Identifications Database (PRIDE, <a href="https://www.ebi.ac.uk/pride/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/pride/</a>) under accession codes PXD059174 and PXD063833.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>The R codes generated in this study are available in the Figshare database (10.6084/m9.figshare.27087253). All R packages used in this study are publicly available through the Comprehensive R Archive Network (CRAN) or Bioconductor, with specific version numbers and persistent digital object identifiers (DOIs) provided in Supplementary TableÂ <a href="#MOESM1" class="usa-link">4</a>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par39">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisherâ€™s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Mengyang Zhang, Shuyan Chen.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Yameng Sun, Email: sunyamenggo@163.com.</p>
<p>Hong You, Email: youhong30@sina.com.</p>
<p>Wei Chen, Email: cw_2011@126.com.</p></section><section id="sec23"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63006-z.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Polaris Observatory, C Global prevalence, treatment, and prevention of hepatitis B virus infection in 2016: a modelling study. <em>Lancet Gastroenterol. Hepatol.</em><strong>3</strong>, 383â€“403 (2018).
</cite> [<a href="https://doi.org/10.1016/S2468-1253(18)30056-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29599078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Polaris%20Observatory,%20C%20Global%20prevalence,%20treatment,%20and%20prevention%20of%20hepatitis%20B%20virus%20infection%20in%202016:%20a%20modelling%20study.%20Lancet%20Gastroenterol.%20Hepatol.3,%20383%E2%80%93403%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Sun, Y. et al. New classification of liver biopsy assessment for fibrosis in chronic hepatitis B patients before and after treatment. <em>Hepatology</em><strong>65</strong>, 1438â€“1450 (2017).
</cite> [<a href="https://doi.org/10.1002/hep.29009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28027574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20Y.%20et%20al.%20New%20classification%20of%20liver%20biopsy%20assessment%20for%20fibrosis%20in%20chronic%20hepatitis%20B%20patients%20before%20and%20after%20treatment.%20Hepatology65,%201438%E2%80%931450%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Chang, T. T. et al. Long-term entecavir therapy results in the reversal of fibrosis/cirrhosis and continued histological improvement in patients with chronic hepatitis B. <em>Hepatology</em><strong>52</strong>, 886â€“893 (2010).
</cite> [<a href="https://doi.org/10.1002/hep.23785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20683932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang,%20T.%20T.%20et%20al.%20Long-term%20entecavir%20therapy%20results%20in%20the%20reversal%20of%20fibrosis/cirrhosis%20and%20continued%20histological%20improvement%20in%20patients%20with%20chronic%20hepatitis%20B.%20Hepatology52,%20886%E2%80%93893%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Sun, Y. et al. Regression of liver fibrosis in patients on hepatitis B therapy is associated with decreased liver-related events. <em>Clin. Gastroenterol. Hepatol.</em><strong>22</strong>, 591â€“601 e593 (2024).
</cite> [<a href="https://doi.org/10.1016/j.cgh.2023.11.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38040276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20Y.%20et%20al.%20Regression%20of%20liver%20fibrosis%20in%20patients%20on%20hepatitis%20B%20therapy%20is%20associated%20with%20decreased%20liver-related%20events.%20Clin.%20Gastroenterol.%20Hepatol.22,%20591%E2%80%93601%20e593%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Chang, X. et al. The utility of P-I-R classification in predicting the on-treatment histological and clinical outcomes of patients with hepatitis B and advanced liver fibrosis. <em>Hepatology</em><strong>79</strong>, 425â€“437 (2024).
</cite> [<a href="https://doi.org/10.1097/HEP.0000000000000563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10789381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37611260/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang,%20X.%20et%20al.%20The%20utility%20of%20P-I-R%20classification%20in%20predicting%20the%20on-treatment%20histological%20and%20clinical%20outcomes%20of%20patients%20with%20hepatitis%20B%20and%20advanced%20liver%20fibrosis.%20Hepatology79,%20425%E2%80%93437%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Schmoyer, C. J., Kumar, D., Gupta, G. &amp; Sterling, R. K. Diagnostic accuracy of noninvasive tests to detect advanced hepatic fibrosis in patients with hepatitis C and end-stage renal disease. <em>Clin. Gastroenterol. Hepatol.</em><strong>18</strong>, 2332â€“2339 e2331 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cgh.2020.02.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32084602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmoyer,%20C.%20J.,%20Kumar,%20D.,%20Gupta,%20G.%20&amp;%20Sterling,%20R.%20K.%20Diagnostic%20accuracy%20of%20noninvasive%20tests%20to%20detect%20advanced%20hepatic%20fibrosis%20in%20patients%20with%20hepatitis%20C%20and%20end-stage%20renal%20disease.%20Clin.%20Gastroenterol.%20Hepatol.18,%202332%E2%80%932339%20e2331%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Martinez, S. M., Crespo, G., Navasa, M. &amp; Forns, X. Noninvasive assessment of liver fibrosis. <em>Hepatology</em><strong>53</strong>, 325â€“335 (2011).
</cite> [<a href="https://doi.org/10.1002/hep.24013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21254180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martinez,%20S.%20M.,%20Crespo,%20G.,%20Navasa,%20M.%20&amp;%20Forns,%20X.%20Noninvasive%20assessment%20of%20liver%20fibrosis.%20Hepatology53,%20325%E2%80%93335%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Dong, X. Q. et al. Declining in liver stiffness cannot indicate fibrosis regression in patients with chronic hepatitis B: a 78-week prospective study. <em>J. Gastroenterol. Hepatol.</em><strong>34</strong>, 755â€“763 (2019).
</cite> [<a href="https://doi.org/10.1111/jgh.14498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30290019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong,%20X.%20Q.%20et%20al.%20Declining%20in%20liver%20stiffness%20cannot%20indicate%20fibrosis%20regression%20in%20patients%20with%20chronic%20hepatitis%20B:%20a%2078-week%20prospective%20study.%20J.%20Gastroenterol.%20Hepatol.34,%20755%E2%80%93763%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Wong, G. L. et al. On-treatment monitoring of liver fibrosis with transient elastography in chronic hepatitis B patients. <em>Antivir. Ther.</em><strong>16</strong>, 165â€“172 (2011).
</cite> [<a href="https://doi.org/10.3851/IMP1726" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21447865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wong,%20G.%20L.%20et%20al.%20On-treatment%20monitoring%20of%20liver%20fibrosis%20with%20transient%20elastography%20in%20chronic%20hepatitis%20B%20patients.%20Antivir.%20Ther.16,%20165%E2%80%93172%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Lian, M. J., Chen, Z. Q., Wang, Q. M., Zheng, G. S. &amp; Hong, G. L. Diagnostic accuracy of mac-2-binding protein glycosylation isomer for diagnosing hepatitis B-related fibrosis: a meta-analysis. <em>J. Dig. Dis.</em><strong>23</strong>, 550â€“560 (2022).
</cite> [<a href="https://doi.org/10.1111/1751-2980.13140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36251470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lian,%20M.%20J.,%20Chen,%20Z.%20Q.,%20Wang,%20Q.%20M.,%20Zheng,%20G.%20S.%20&amp;%20Hong,%20G.%20L.%20Diagnostic%20accuracy%20of%20mac-2-binding%20protein%20glycosylation%20isomer%20for%20diagnosing%20hepatitis%20B-related%20fibrosis:%20a%20meta-analysis.%20J.%20Dig.%20Dis.23,%20550%E2%80%93560%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Das, A. et al. Evaluation of Chitinase 3-like 1 (CHI3L1) as a noninvasive biomarker of hepatic fibrosis in patients with Hepatitis B virus-related compensated chronic liver disease. <em>J. Fam. Med Prim. Care</em><strong>10</strong>, 1694â€“1698 (2021).</cite> [<a href="https://doi.org/10.4103/jfmpc.jfmpc_1922_20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8144771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34123914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Das,%20A.%20et%20al.%20Evaluation%20of%20Chitinase%203-like%201%20(CHI3L1)%20as%20a%20noninvasive%20biomarker%20of%20hepatic%20fibrosis%20in%20patients%20with%20Hepatitis%20B%20virus-related%20compensated%20chronic%20liver%20disease.%20J.%20Fam.%20Med%20Prim.%20Care10,%201694%E2%80%931698%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Cao, Z. et al. Assessment of serum Golgi protein 73 as a biomarker for the diagnosis of significant fibrosis in patients with chronic HBV infection. <em>J. Viral Hepat.</em><strong>24</strong>, 57â€“65 (2017).
</cite> [<a href="https://doi.org/10.1111/jvh.12786" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29082644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cao,%20Z.%20et%20al.%20Assessment%20of%20serum%20Golgi%20protein%2073%20as%20a%20biomarker%20for%20the%20diagnosis%20of%20significant%20fibrosis%20in%20patients%20with%20chronic%20HBV%20infection.%20J.%20Viral%20Hepat.24,%2057%E2%80%9365%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Sun, Y. M., Chen, S. Y. &amp; You, H. Regression of liver fibrosis: evidence and challenges. <em>Chin. Med. J.</em><strong>133</strong>, 1696â€“1702 (2020).
</cite> [<a href="https://doi.org/10.1097/CM9.0000000000000835" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7401741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32568866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20Y.%20M.,%20Chen,%20S.%20Y.%20&amp;%20You,%20H.%20Regression%20of%20liver%20fibrosis:%20evidence%20and%20challenges.%20Chin.%20Med.%20J.133,%201696%E2%80%931702%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Darmadi, D. et al. Factors affecting HBV DNA suppression in chronic hepatitis B patients treated with tenofovir disoproxil fumarate. <em>F1000Res</em><strong>11</strong>, 1521 (2022).
</cite> [<a href="https://doi.org/10.12688/f1000research.128116.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10521109/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37767077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Darmadi,%20D.%20et%20al.%20Factors%20affecting%20HBV%20DNA%20suppression%20in%20chronic%20hepatitis%20B%20patients%20treated%20with%20tenofovir%20disoproxil%20fumarate.%20F1000Res11,%201521%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Sun, Y. et al. Persistent low level of hepatitis B virus promotes fibrosis progression during therapy. <em>Clin. Gastroenterol. Hepatol.</em><strong>18</strong>, 2582â€“2591 e2586 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cgh.2020.03.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32147592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20Y.%20et%20al.%20Persistent%20low%20level%20of%20hepatitis%20B%20virus%20promotes%20fibrosis%20progression%20during%20therapy.%20Clin.%20Gastroenterol.%20Hepatol.18,%202582%E2%80%932591%20e2586%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Thiele, M. et al. Opportunities and barriers in omics-based biomarker discovery for steatotic liver diseases. <em>J. Hepatol.</em><strong>81</strong>, 345â€“359 (2024).
</cite> [<a href="https://doi.org/10.1016/j.jhep.2024.03.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38552880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thiele,%20M.%20et%20al.%20Opportunities%20and%20barriers%20in%20omics-based%20biomarker%20discovery%20for%20steatotic%20liver%20diseases.%20J.%20Hepatol.81,%20345%E2%80%93359%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Shen, C., Jiang, X., Li, M. &amp; Luo, Y. Hepatitis virus and hepatocellular carcinoma: recent advances. <em>Cancers</em><strong>15</strong>, 10.3390/cancers15020533 (2023).</cite> [<a href="https://doi.org/10.3390/cancers15020533" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9856776/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36672482/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Sterling, R. K. et al. AASLD Practice Guideline on imaging-based noninvasive liver disease assessment of hepatic fibrosis and steatosis. <em>Hepatology</em>, 10.1097/HEP.0000000000000843 (2024).</cite> [<a href="https://doi.org/10.1097/HEP.0000000000000843" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38489518/" class="usa-link">PubMed</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Huang, L. L. et al. Liver stiffness measurement can predict liver inflammation in chronic hepatitis B patients with normal alanine transaminase. <em>J. Clin. Transl. Hepatol.</em><strong>11</strong>, 817â€“826 (2023).
</cite> [<a href="https://doi.org/10.14218/JCTH.2022.00329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10318296/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37408816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20L.%20L.%20et%20al.%20Liver%20stiffness%20measurement%20can%20predict%20liver%20inflammation%20in%20chronic%20hepatitis%20B%20patients%20with%20normal%20alanine%20transaminase.%20J.%20Clin.%20Transl.%20Hepatol.11,%20817%E2%80%93826%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Abu-Freha, N. et al. Liver fibrosis regression and associated factors in HCV patients treated with direct-acting antiviral agents. <em>Life</em><strong>13</strong>, 10.3390/life13091872 (2023).</cite> [<a href="https://doi.org/10.3390/life13091872" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10533124/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37763276/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Kim, W. R. et al. Evaluation of APRI and FIB-4 scoring systems for non-invasive assessment of hepatic fibrosis in chronic hepatitis B patients. <em>J. Hepatol.</em><strong>64</strong>, 773â€“780 (2016).
</cite> [<a href="https://doi.org/10.1016/j.jhep.2015.11.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26626497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20W.%20R.%20et%20al.%20Evaluation%20of%20APRI%20and%20FIB-4%20scoring%20systems%20for%20non-invasive%20assessment%20of%20hepatic%20fibrosis%20in%20chronic%20hepatitis%20B%20patients.%20J.%20Hepatol.64,%20773%E2%80%93780%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Wong, V. W., Adams, L. A., de Ledinghen, V., Wong, G. L. &amp; Sookoian, S. Noninvasive biomarkers in NAFLD and NASHâ€”current progress and future promise. <em>Nat. Rev. Gastroenterol. Hepatol.</em><strong>15</strong>, 461â€“478 (2018).
</cite> [<a href="https://doi.org/10.1038/s41575-018-0014-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29844588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wong,%20V.%20W.,%20Adams,%20L.%20A.,%20de%20Ledinghen,%20V.,%20Wong,%20G.%20L.%20&amp;%20Sookoian,%20S.%20Noninvasive%20biomarkers%20in%20NAFLD%20and%20NASH%E2%80%94current%20progress%20and%20future%20promise.%20Nat.%20Rev.%20Gastroenterol.%20Hepatol.15,%20461%E2%80%93478%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Li, J., Smith, L. S. &amp; Zhu, H. J. Data-independent acquisition (DIA): An emerging proteomics technology for analysis of drug-metabolizing enzymes and transporters. <em>Drug Discov. Today Technol.</em><strong>39</strong>, 49â€“56 (2021).
</cite> [<a href="https://doi.org/10.1016/j.ddtec.2021.06.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8674493/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34906325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20J.,%20Smith,%20L.%20S.%20&amp;%20Zhu,%20H.%20J.%20Data-independent%20acquisition%20(DIA):%20An%20emerging%20proteomics%20technology%20for%20analysis%20of%20drug-metabolizing%20enzymes%20and%20transporters.%20Drug%20Discov.%20Today%20Technol.39,%2049%E2%80%9356%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Chen, S. et al. Liver fibrosis showed a two-phase regression rate during long-term anti-HBV therapy by three-time biopsies assessments. <em>Hepatol. Int.</em><strong>18</strong>, 904â€“916 (2024).
</cite> [<a href="https://doi.org/10.1007/s12072-024-10643-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38565833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20S.%20et%20al.%20Liver%20fibrosis%20showed%20a%20two-phase%20regression%20rate%20during%20long-term%20anti-HBV%20therapy%20by%20three-time%20biopsies%20assessments.%20Hepatol.%20Int.18,%20904%E2%80%93916%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Chen, W. et al. Matrisome gene-based subclassification of patients with liver fibrosis identifies clinical and molecular heterogeneities. <em>Hepatology</em><strong>78</strong>, 1118â€“1132 (2023).
</cite> [<a href="https://doi.org/10.1097/HEP.0000000000000423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10524702/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37098756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20W.%20et%20al.%20Matrisome%20gene-based%20subclassification%20of%20patients%20with%20liver%20fibrosis%20identifies%20clinical%20and%20molecular%20heterogeneities.%20Hepatology78,%201118%E2%80%931132%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Thorgersen, E. B. et al. The role of complement in liver injury, regeneration, and transplantation. <em>Hepatology</em><strong>70</strong>, 725â€“736 (2019).
</cite> [<a href="https://doi.org/10.1002/hep.30508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6771474/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30653682/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thorgersen,%20E.%20B.%20et%20al.%20The%20role%20of%20complement%20in%20liver%20injury,%20regeneration,%20and%20transplantation.%20Hepatology70,%20725%E2%80%93736%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Desaire, H. How (Not) to generate a highly predictive biomarker panel using machine learning. <em>J. Proteome Res.</em><strong>21</strong>, 2071â€“2074 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.2c00117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9680826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36004690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Desaire,%20H.%20How%20(Not)%20to%20generate%20a%20highly%20predictive%20biomarker%20panel%20using%20machine%20learning.%20J.%20Proteome%20Res.21,%202071%E2%80%932074%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Guergues, J. et al. A semi-automated workflow for DIA-based global discovery to pathway-driven PRM analysis. <em>Proteomics</em><strong>25</strong>, e2400129 (2025).
</cite> [<a href="https://doi.org/10.1002/pmic.202400129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11798701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39235396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guergues,%20J.%20et%20al.%20A%20semi-automated%20workflow%20for%20DIA-based%20global%20discovery%20to%20pathway-driven%20PRM%20analysis.%20Proteomics25,%20e2400129%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Pavlides, M. et al. Interobserver variability in histologic evaluation of liver fibrosis using categorical and quantitative scores. <em>Am. J. Clin. Pathol.</em><strong>147</strong>, 364â€“369 (2017).
</cite> [<a href="https://doi.org/10.1093/ajcp/aqx011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5848246/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28340131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pavlides,%20M.%20et%20al.%20Interobserver%20variability%20in%20histologic%20evaluation%20of%20liver%20fibrosis%20using%20categorical%20and%20quantitative%20scores.%20Am.%20J.%20Clin.%20Pathol.147,%20364%E2%80%93369%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Zhang, M. et al. Persistent steatosis correlates with decreased fibrosis regression during anti-HBV treatment in patients with chronic HBV infection. <em>J. Med. Virol.</em><strong>95</strong>, e29156 (2023).
</cite> [<a href="https://doi.org/10.1002/jmv.29156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37822064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20M.%20et%20al.%20Persistent%20steatosis%20correlates%20with%20decreased%20fibrosis%20regression%20during%20anti-HBV%20treatment%20in%20patients%20with%20chronic%20HBV%20infection.%20J.%20Med.%20Virol.95,%20e29156%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Meier, F. et al. Online parallel accumulation-serial fragmentation (PASEF) with a novel trapped ion mobility mass spectrometer. <em>Mol. Cell Proteom.</em><strong>17</strong>, 2534â€“2545 (2018).</cite> [<a href="https://doi.org/10.1074/mcp.TIR118.000900" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6283298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30385480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meier,%20F.%20et%20al.%20Online%20parallel%20accumulation-serial%20fragmentation%20(PASEF)%20with%20a%20novel%20trapped%20ion%20mobility%20mass%20spectrometer.%20Mol.%20Cell%20Proteom.17,%202534%E2%80%932545%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. &amp; Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. <em>Nat. Methods</em><strong>17</strong>, 41â€“44 (2020).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0638-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6949130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31768060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Demichev,%20V.,%20Messner,%20C.%20B.,%20Vernardis,%20S.%20I.,%20Lilley,%20K.%20S.%20&amp;%20Ralser,%20M.%20DIA-NN:%20neural%20networks%20and%20interference%20correction%20enable%20deep%20proteome%20coverage%20in%20high%20throughput.%20Nat.%20Methods17,%2041%E2%80%9344%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Zhang, H. &amp; Bensaddek, D. Narrow precursor mass range for DIA-MS enhances protein identification and quantification in Arabidopsis. <em>Life</em><strong>11</strong>, 10.3390/life11090982 (2021).</cite> [<a href="https://doi.org/10.3390/life11090982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8469718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34575131/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Sherman, B. T. et al. DAVID: a web server for functional enrichment analysis and functional annotation of gene lists (2021 update). <em>Nucleic Acids Res.</em><strong>50</strong>, W216â€“W221 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkac194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9252805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35325185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sherman,%20B.%20T.%20et%20al.%20DAVID:%20a%20web%20server%20for%20functional%20enrichment%20analysis%20and%20functional%20annotation%20of%20gene%20lists%20(2021%20update).%20Nucleic%20Acids%20Res.50,%20W216%E2%80%93W221%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Shen, W. et al. Sangerbox: A comprehensive, interaction-friendly clinical bioinformatics analysis platform. <em>Imeta</em><strong>1</strong>, e36 (2022).
</cite> [<a href="https://doi.org/10.1002/imt2.36" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10989974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38868713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20W.%20et%20al.%20Sangerbox:%20A%20comprehensive,%20interaction-friendly%20clinical%20bioinformatics%20analysis%20platform.%20Imeta1,%20e36%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365141/bin/41467_2025_63006_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365141/bin/41467_2025_63006_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Peer Review file</a><sup> (921.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365141/bin/41467_2025_63006_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (848.5KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The raw 4D-DIA-MS data, PRM-MS data, and their processed peptide and protein abundance data generated in this study have been deposited in the Proteomics Identifications Database (PRIDE, <a href="https://www.ebi.ac.uk/pride/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/pride/</a>) under accession codes PXD059174 and PXD063833.</p>
<p>The R codes generated in this study are available in the Figshare database (10.6084/m9.figshare.27087253). All R packages used in this study are publicly available through the Comprehensive R Archive Network (CRAN) or Bioconductor, with specific version numbers and persistent digital object identifiers (DOIs) provided in Supplementary TableÂ <a href="#MOESM1" class="usa-link">4</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63006-z"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63006.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.0Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365141/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365141/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365141%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365141/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365141/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365141/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830346/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365141/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830346/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365141/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365141/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="CalQxXjXMX8zs2oMBziGjHrwW3mG1T93JcmChrWTBfOMOwtkA7oXojtDsacnYwCk">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
