
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genome sequencing and physiological characterization of three Neoarthrinium moseri strains - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A2E78AF2005305A2E7004FEBDE25.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcmicr">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369201/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Microbiology">
<meta name="citation_title" content="Genome sequencing and physiological characterization of three Neoarthrinium moseri strains">
<meta name="citation_author" content="Nadine J Hochenegger">
<meta name="citation_author_institution" content="Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria">
<meta name="citation_author" content="Gabriel A Vignolle">
<meta name="citation_author_institution" content="Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria">
<meta name="citation_author_institution" content="The Vatche and Tamar Manoukian Division of Digestive Diseases, Department of Medicine, David Geffen School of Medicine at UCLA, Los Angeles, CA USA">
<meta name="citation_author_institution" content="Goodman-Luskine Microbiome Center, University of California, Los Angeles, CA USA">
<meta name="citation_author" content="Matthias Schmal">
<meta name="citation_author_institution" content="Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria">
<meta name="citation_author" content="Robert L Mach">
<meta name="citation_author_institution" content="Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria">
<meta name="citation_author" content="Astrid R Mach-Aigner">
<meta name="citation_author_institution" content="Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria">
<meta name="citation_author" content="Mohammad Javad Rahimi">
<meta name="citation_author_institution" content="Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria">
<meta name="citation_author" content="Chin Mei Chan">
<meta name="citation_author_institution" content="Chemical Sciences, Faculty of Science, Universiti Brunei Darussalam, Bandar Seri Begawan, Jalan Tungku Link, Brunei Darussalam">
<meta name="citation_author" content="Feng M Cai">
<meta name="citation_author_institution" content="School of Ecology, Sun Yat-sen University, Shenzhen, 518107 China">
<meta name="citation_author" content="Irina S Druzhinina">
<meta name="citation_author_institution" content="Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria">
<meta name="citation_author_institution" content="Royal Botanic Gardens, Kew Green, Richmond Surry TW9 3AE UK">
<meta name="citation_author" content="Christian Zimmermann">
<meta name="citation_author_institution" content="Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="526">
<meta name="citation_doi" content="10.1186/s12866-025-04274-z">
<meta name="citation_pmid" content="40841932">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369201/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369201/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369201/pdf/12866_2025_Article_4274.pdf">
<meta name="description" content="Fungi play essential ecological roles and have been utilized by humans for diverse applications such as industrial enzyme production or as sources of bioactive compounds. Recent research has highlighted the Amphisphaeriales order (Ascomycota) as ...">
<meta name="og:title" content="Genome sequencing and physiological characterization of three Neoarthrinium moseri strains">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Fungi play essential ecological roles and have been utilized by humans for diverse applications such as industrial enzyme production or as sources of bioactive compounds. Recent research has highlighted the Amphisphaeriales order (Ascomycota) as ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369201/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12369201">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12866-025-04274-z"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12866_2025_Article_4274.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369201%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12369201/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12369201/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369201/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcmicr.png" alt="BMC Microbiology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Microbiology" title="Link to BMC Microbiology" shape="default" href="https://bmcmicrobiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Microbiol</button></div>. 2025 Aug 21;25:526. doi: <a href="https://doi.org/10.1186/s12866-025-04274-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12866-025-04274-z</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Microbiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Microbiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genome sequencing and physiological characterization of three <em>Neoarthrinium moseri</em> strains</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hochenegger%20NJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Nadine J Hochenegger</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Nadine J Hochenegger</span></h3>
<div class="p">
<sup>1</sup>Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hochenegger%20NJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Nadine J Hochenegger</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vignolle%20GA%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Gabriel A Vignolle</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Gabriel A Vignolle</span></h3>
<div class="p">
<sup>1</sup>Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria </div>
<div class="p">
<sup>2</sup>The Vatche and Tamar Manoukian Division of Digestive Diseases, Department of Medicine, David Geffen School of Medicine at UCLA, Los Angeles, CA USA </div>
<div class="p">
<sup>3</sup>Goodman-Luskine Microbiome Center, University of California, Los Angeles, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vignolle%20GA%22%5BAuthor%5D" class="usa-link"><span class="name western">Gabriel A Vignolle</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schmal%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Matthias Schmal</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Matthias Schmal</span></h3>
<div class="p">
<sup>1</sup>Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schmal%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Matthias Schmal</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mach%20RL%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Robert L Mach</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Robert L Mach</span></h3>
<div class="p">
<sup>1</sup>Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mach%20RL%22%5BAuthor%5D" class="usa-link"><span class="name western">Robert L Mach</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mach-Aigner%20AR%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Astrid R Mach-Aigner</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Astrid R Mach-Aigner</span></h3>
<div class="p">
<sup>1</sup>Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mach-Aigner%20AR%22%5BAuthor%5D" class="usa-link"><span class="name western">Astrid R Mach-Aigner</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rahimi%20MJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Mohammad Javad Rahimi</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Mohammad Javad Rahimi</span></h3>
<div class="p">
<sup>1</sup>Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rahimi%20MJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohammad Javad Rahimi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chan%20CM%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Chin Mei Chan</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Chin Mei Chan</span></h3>
<div class="p">
<sup>4</sup>Chemical Sciences, Faculty of Science, Universiti Brunei Darussalam,  Bandar Seri Begawan, Jalan Tungku Link, Brunei Darussalam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chan%20CM%22%5BAuthor%5D" class="usa-link"><span class="name western">Chin Mei Chan</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20FM%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Feng M Cai</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Feng M Cai</span></h3>
<div class="p">
<sup>5</sup>School of Ecology, Sun Yat-sen University, Shenzhen, 518107 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20FM%22%5BAuthor%5D" class="usa-link"><span class="name western">Feng M Cai</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Druzhinina%20IS%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Irina S Druzhinina</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Irina S Druzhinina</span></h3>
<div class="p">
<sup>1</sup>Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria </div>
<div class="p">
<sup>6</sup>Royal Botanic Gardens, Kew Green, Richmond Surry TW9 3AE UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Druzhinina%20IS%22%5BAuthor%5D" class="usa-link"><span class="name western">Irina S Druzhinina</span></a>
</div>
</div>
<sup>1,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zimmermann%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Christian Zimmermann</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Christian Zimmermann</span></h3>
<div class="p">
<sup>1</sup>Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zimmermann%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian Zimmermann</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Institute of Chemical, Environmental and Bioscience Engineering, TU Wien, Gumpendorfer Strasse 1a, Wien, 1060 Austria </div>
<div id="Aff2">
<sup>2</sup>The Vatche and Tamar Manoukian Division of Digestive Diseases, Department of Medicine, David Geffen School of Medicine at UCLA, Los Angeles, CA USA </div>
<div id="Aff3">
<sup>3</sup>Goodman-Luskine Microbiome Center, University of California, Los Angeles, CA USA </div>
<div id="Aff4">
<sup>4</sup>Chemical Sciences, Faculty of Science, Universiti Brunei Darussalam,  Bandar Seri Begawan, Jalan Tungku Link, Brunei Darussalam </div>
<div id="Aff5">
<sup>5</sup>School of Ecology, Sun Yat-sen University, Shenzhen, 518107 China </div>
<div id="Aff6">
<sup>6</sup>Royal Botanic Gardens, Kew Green, Richmond Surry TW9 3AE UK </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 29; Accepted 2025 Jul 28; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12369201  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841932/" class="usa-link">40841932</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Fungi play essential ecological roles and have been utilized by humans for diverse applications such as industrial enzyme production or as sources of bioactive compounds. Recent research has highlighted the <em>Amphisphaeriales</em> order (<em>Ascomycota</em>) as promising producers of secondary metabolites of pharmaceutical importance. Within this family, the recently established genus <em>Neoarthrinium</em> includes species such as <em>N. brasiliense</em>,<em> N. lithocarpicola</em>,<em> N. moseri</em>,<em> N. trachycarpi</em>, and <em>N. urticae</em>. Existing literature has primarily focused on the taxonomy and phylogeny of <em>Neoarthrinium</em>, leaving its physiology, ecology, and metabolic potential unexplored.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">This study presents the first investigation of the metabolic and genomic potential of <em>N. moseri</em>. We describe the isolation of two South-Asian <em>N. moseri</em> strains and the genome sequencing of these strains alongside the Colombian ex-type strain for the species. Comparative genome analysis reveals an exceptionally high number of biosynthetic gene clusters (BGCs), surpassing those of many other fungi in the <em>Amphisphaeriales</em> order. Additionally, the genome of <em>N. moseri</em> contains a diverse repertoire of carbohydrate-active enzymes (CAZymes), supporting its hypothesized ecological role as a phyllosphere fungus (putatively an endophyte and/or saprotroph). Ecophysiological assays, including BIOLOG phenotyping, demonstrate its ability to utilize a broad range of carbon sources, emphasizing ecological versatility.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">This study highlights <em>N. moseri</em> as a promising candidate for secondary metabolite discovery, providing foundational insights into the metabolic and genomic potential of the <em>Neoarthrinium</em> genus and related fungi.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12866-025-04274-z.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Amphisphaeriales, Ascomycota, Comparative genomics, Fungi, Genome mining, Microbiological assessment, Secondary metabolism</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par29">Fungi are a diverse kingdom with a broad range of ecological roles, e.g. decomposers of organic matter and symbiotic partners of plants – may it be in mutualistic and parasitic relations. Humankind has been using fungi for different purposes, such as food and feed fermentation, agricultural applications, enzymes production, and as source of bioactive compounds for medicine and industry [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. The fungal secondary metabolism is generally considered to be a large untapped reservoir for novel bioactive compounds and drug leads [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. The ongoing efforts to find new pharmaceuticals encompass the search for new fungi and mining their genomes [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>].</p>
<p id="Par30">In the recent years, the order <em>Amphisphaeriales</em> (<em>Ascomycota</em>) gained increasing attention as promising secondary metabolite producers [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>], especially, fungi related to such genera as <em>Apiospora</em> and <em>Arthrinium</em> (in the family of <em>Apiosporaceae</em>) [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. These fungi are known to produce a broad range of bioactive compounds, including cytochalasins with cytotoxic activity, polyketides such as apiosporic acid and hexylitaconic acid derivatives, and antimicrobial agents like apiosporamide. Additional compounds isolated from these genera exhibit antioxidant, tyrosinase-inhibitory, and quorum-sensing-disrupting properties. These findings highlight the chemical diversity of <em>Amphisphaeriales</em> and suggest that lesser-known genera such as <em>Neoarthrinium</em> may also harbor unexplored metabolic potential [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>].</p>
<p id="Par31">The genus <em>Neoarthrinium</em>, established in 2022 within the family <em>Apiosporaceae (vide infra)</em>, originally comprised four species: <em>N. lithocarpicola</em>, <em>N. moseri</em>, <em>N. trachycarpi</em>, and <em>N. urticae</em> [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. These taxa were isolated from diverse terrestrial plant hosts across Asia and South America: <em>N. lithocarpicola</em> from diseased leaves of <em>Lithocarpus glaber</em> in China; <em>N. trachycarpi</em> from <em>Trachycarpus fortunei</em>, also in China; <em>N. moseri</em> from a dead plant in Colombia; and <em>N. urticae</em> from leaf litter in India. W. Gams isolated the <em>N. moseri</em> type strain CBS 164.80 from the dead petiole of <em>Mauritia minor</em> in Colombia in 1995 [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. It was originally described as an unusual species of the genus <em>Wardomyces</em> (<em>Microascales</em>) until Jiang et al. realized that this isolate belongs to <em>Amphisphaeriales</em> and assigned it to <em>N. moseri</em> in 2022 [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. The reclassification of <em>N. urticae</em> (syn. <em>Arthrinium urticae</em>) was based on sequence data from a single isolate, and its taxonomic placement remains tentative due to uncertainties regarding its representativeness [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Since 2022, three additional species have been described. <em>N. brasiliense</em>, added in 2024, further expanded the genus into South America [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. In 2025, two ecologically and geographically distinct taxa were introduced: <em>N. lewisiae</em> was isolated from necrotic leaf spots on <em>Pandanus tectorius</em> (screwpine) in coastal Australia [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>] and <em>N. aquaticum</em> was described from submerged plant tissue of the golden leather fern (<em>Acrostichum aureum</em>) in a freshwater habitat in Thailand [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], representing the first aquatic species in the genus. Notably, all <em>Neoarthrinium</em> strains have been isolated from the surface of plants [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Mukhopadhyay et al. also proposed the establishment of the new family <em>Neoarthriniaceae</em>, to accommodate the genus <em>Neoarthrinium</em> [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. These findings highlight the ecological versatility of <em>Neoarthrinium</em>, encompassing both saprobic and potentially pathogenic lifestyles across tropical and subtropical regions, with substrates ranging from terrestrial leaf litter and petioles to aquatic ferns and coastal monocots.</p>
<p id="Par32">Limited to mostly taxonomic and phylogenetic studies, existing literature provides little insight into the physiology, ecology, or metabolic potential of this genus. This study provides the first investigation of the metabolic and genomic potential of a species within the genus <em>Neoarthrinium</em>, namely <em>N. moseri</em>. We describe the isolation of two new <em>N. moseri</em> strains from Borneo and genome sequencing of these strains and the Colombian ex-type strain CBS 164.80. We have mined the genomes for carbohydrate-active enzymes (CAZymes) and biosynthetic gene clusters (BGCs) and compared them to other fungi in this regard. Further, we sequenced and annotated also the mitochondrial genome. We explored basic growth characteristics and substrate utilization of <em>N. moseri</em> to gain physiological insights using a BIOLOG Phenotype microarray. Additionally, we suggest that <em>N. trachycarpi</em> should not be considered a separate species but strains of <em>N. moseri</em> based on the genomic data together with already existing phylogenetics and a reassessment of the spore sizes.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Sampling and strain purification</h3>
<p id="Par33">The epiphytic fungi TUCIM 5799 and TUCIM 5827 were isolated from the same environmental sample, the adaxial surface of the healthy leaf of <em>Rubroshorea johorensis</em> (<em>Dipterocarpaceae</em>, <em>Malvales</em>; DNA BarCode maturase K (<em>matK</em>) deposited in NCBI GenBank <a href="https://www.ncbi.nlm.nih.gov/nuccore/MF993320.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MF993320.1</a> [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]), sampled in the high canopy (40–60 m above ground) of the lowland tropical rain forest surrounding the Kuala Belalong Field Studies Center (KBFSC, 4°32’48.2"N 115°09’27.9"E) located in the Temburong District of Brunei Darussalam (Borneo). For this purpose, the adaxial surface of a freshly sampled leaf was scratched by the sterile electric toothbrush (2 min) in 25 ml of sterile water supplemented with Tween-20 (0.01%) in large sterile Petri plate (20 cm in diameter). The resulting suspension was collected in 50 ml falcons and centrifuged at 4 °C for 15 min at 14 000 rpm. The resulting pellet was resuspended in 4 ml of sterile water and used for serial dilution and plating on potato dextrose agar (PDA, Carl Roth) supplemented with 200 mg/l of chloramphenicol. Young single spore fungal colonies were detected with the use of a stereo microscope and aseptically transferred to fresh PDA plates and cultivated at 28 °C in darkness, resulting in the isolation of TUCIM 5799 and TUCIM 5827. Agar plugs with pure mature cultures were preserved in 40% glycerol and stored at −80 °C in TU Wien Collection of Industrial Microorganisms (TUCIM).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Maintenance and morphological characterization of strains</h3>
<p id="Par34">While the TUCIM strains were isolated in the course of this study, the CBS 164.80 strain was obtained from the CBS Filamentous fungi and Yeast Collection at the Westerdijk Fungal Biodiversity Institute. CBS 164.80, TUCIM 5827, and TUCIM 5799 were maintained on agar plates containing 30 g/l oatmeal (S-Budget, SPAR Österreichische Warenhandels-AG; shredded to ∅ 0.25 mm) and 15 g/l agar.</p>
<p id="Par35">Three different media were used to initially evaluate the morphology of the three strains: MEA (20 g/l malt extract, 15 g/l agar), CYAS (Czapek yeast autolysate agar with 50 g/l NaCl; 3 g/l NaNO<sub>3</sub>, 5 g/l yeast extract, 30 g/l sucrose, 1.3 g/l K<sub>2</sub>HPO<sub>4</sub>,</p>
<p id="Par36">0.5 g/l KCl, 0.5 g MgSO<sub>4</sub> • 7 H<sub>2</sub>O, 0.01 g FeSO<sub>4</sub> • 5 H<sub>2</sub>O, 0.005 g CuSO<sub>4</sub> • 5 H<sub>2</sub>O,</p>
<p id="Par37">15 g/l agar), and Oat (as above). 5 µl of spore solution.</p>
<p id="Par38">(8 g/l NaCl, 0.05%(v/v) Tween-80) with an OD<sub>600</sub> of 3.0 were applied to the middle of the agar plates, which were subsequently incubated at 28 °C for 13–14 days.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Microscopy</h3>
<p id="Par39">Brightfield microscopy (BF) was performed using VWR Microscope TR 500 (VWR International GmbH, Darmstadt, Germany).</p>
<p id="Par40">Scanning electron microscopy (SEM) of conidia was performed using COXEM EM-30AX PLUS with a SPT-20 Sputter. For sample preparation, conidia of the respective strain were softly scratched off an overgrown oatmeal-plate with a cotton swab. Conidia were then carefully distributed over a silver stripe which was attached to the stage of the device. Further proceedings were done according to the manufacturer’s instructions. Pictures were processed using the device’s own software Nanostation 3.0.4. The sizes of the spores were measured using FIJI (ImageJ2, Version 2.9.0).</p></section><section id="Sec6"><h3 class="pmc_sec_title">DNA extraction and library preparation</h3>
<p id="Par41">The <em>N. moseri</em> strains were cultivated in malt extract medium at 28 °C and 180 rpm for 10 days in an orbital shaking incubator. The biomass was filtered through miracloth (EMD Millipore Corp., Burlington, MA, USA), frozen in liquid nitrogen, lyophilized. For DNA extraction, the lyophilized biomass was disrupted using a Fast-Prep-24 (MP Biomedicals, Santa Ana/, CA, USA) with 0.37 g of glass beads ∅ 0.1 mm, 0.25 g of glass beads ∅ 1 mm, and a glass bead ∅ 5 mm at 6 m/s for 30 s. After the addition of 1 ml CTAB buffer (100 mM Tris.Cl, 20 mM EDTA, 1.4 M NaCl, 2% (w/v) CTAB, pH = 8.0) and 4 µl β-mercaptoethanol, the samples were subjected to two further disruption treatments on the Fast-Prep-24 at 5 m/s for 30 s and then incubated at 65 °C for 20 min. The supernatant was extracted with phenol, chloroform, isoamylalcohol (25:24:1) followed by a chloroform extraction. The supernatant was treated with RNase A (Thermo Fisher Scientific, Inc., Waltham, MA, USA) according to the manufacturer’s instructions. Finally, the DNA was precipitated with ethanol and dissolved in 10 mM Tris.Cl (pH = 8.0).</p>
<p id="Par42">The DNA was sheared in a Diagenode Bioruptor<sup>®</sup> Pico (Diagenode s.a., Liège, Belgium) with the settings set to high and three cycles of 15 s “on” and 60 s “off”. The sheared DNA was purified using PCR purification kit (Thermo Fisher Scientific, Inc., Waltham, MA, USA) and then double side size selected with “NEBNext Ultra™ sample purification beads” (New England Biolabs, Ipswich, MA, USA) for 800 bp fragments. The library preparation was performed following the protocol of “NEBNext<sup>®</sup> Ultra™ II DNA Library Kit with Purification Beads” and “NEBNext<sup>®</sup> Multiplex Oligos for Illumina (Index Primer Set1 and Set2)” (New England Biolabs, Ipswich, MA, USA). The average size in bp of the library was measured with the fragment analyzer from Advanced Analytical Technologies using the Agilent dsDNA 915 Reagent Kit (35–5000 bp) and analyzed with the PRO size software (Agilent Technologies, Santa Clara, California, USA). The exact DNA concentrations were measured with an “invitrogen™ Qubit™ fluorometer” in ng/µl (Thermo Fisher Scientific, Inc., Waltham, MA, USA) using a “Quant-iT™ dsDNA BR Assay” kit (Thermo Fisher Scientific, Inc., Waltham, MA, USA). Specifically, two libraries were created with a DNA fragment length of 1293 ± 6 bp and 1136 ± 7 bp, the average DNA concentrations were 34.93 ± 0.25 ng/µl and 10.63 ± 0.23 ng/µl, resulting in a 40.899 nM and a 14.178 nM library, respectively. The libraries were diluted to the appropriate 4 nM concentration for sequencing.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Sequencing</h3>
<p id="Par43">The sequencing of the <em>N. moseri</em> library was performed on an Illumina MiSeq platform using two V3 Reagent Kit (600 cycles) and one V2 Nano Reagent Kit (500 cycles) following the standard protocol of Illumina sequencing protocol without adding PhiX control to the runs (Illumina, San Diego, California, USA), resulting in a total of 67,670,936 paired end-reads. The raw data were deposited at the Sequence Read Archive (SRA) under the accession SRR13570309 (CBS 164.80), SRR13747339 (TUCIM 5827) and SRR13747338 (TUCIM 5799). The quality profiles and all further figures, if not specified otherwise, were visualized in R [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>].</p></section><section id="Sec8"><h3 class="pmc_sec_title">Extracting the mitochondrial genome and cleaning the raw reads</h3>
<p id="Par44">First, a preliminary assembly was performed using SPAdes v3.13.1 [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>] with default parameters for each strain separately. Mitochondrial sequences were identified in each strain by performing a sequence similarity analysis using BLAST [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>] (non-redundant nucleotide database). Contigs ranging from 500 to 1000 bp were then used as seed input for NOVOplasty v3.7 [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] to extract the whole circularized mitochondrial genome of <em>N. moseri</em> CBS 164.80, TUCIM 5799 and TUCIM 5827. This was performed in an iterative manner. The mitochondrial genomes were visualized with CGViewer [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The mitochondrial genomes were annotated with the automated MITOS2 web pipeline. The mitochondrial genomes were deposited at GenBank with accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/MW554918" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW554918</a> (CBS 164.80), <a href="https://www.ncbi.nlm.nih.gov/nuccore/MW660808" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW660808</a> (TUCIM 5827), and <a href="https://www.ncbi.nlm.nih.gov/nuccore/MW660809" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW660809</a> (TUCIM 5799).</p>
<p id="Par45">Using the mitochondrial genomes of the strains as input an index was built with bowtie v1.2.2 [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>], respectively, and the mitochondrial flagged reads were extracted using --un option from each raw reads file. The clean raw reads were then re-paired with Fastq-pair [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] to use paired end read assemblers.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Whole - genome assembly</h3>
<p id="Par46">For each strain respectively, the raw cleaned paired end reads were quality trimmed using Trimmomatic [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] in the command line and specifying PE for paired end reads and ILLUMINACLIP: Adapter-PE.fa:2:30:10:2:keepBothReads LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36 to ensure high quality adapter-free reads. Then the cleaned raw reads were assembled using SPAdes v3.13.1 [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>], for each strain separately. Furthermore, the high quality trimmed cleaned paired end reads were used for scaffolding with SSPACE-Standard v3.0 in an iterative manner with following command line options -x 1 -m 50 -o 20 -k 8 -a 0.70 -n 30 -z 150 –b and –k 6. Ns introduced during the assemblies and the scaffolding, so called gaps, were closed with GapFiller v1-10 [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>] using following commands -m 30 -o 6 -r 0.7 -n 10 -d 50 -t 10 -g 0 -i 5 -b.</p>
<p id="Par47">The assemblies were further improved by using Pilon v1.21 [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] iteratively. We first indexed the assemblies with bwa [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>], SAMtools v1.7 [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] and picard [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. The high quality trimmed cleaned paired end reads were mapped to the matching indexed assemblies of the individual <em>N. moseri</em> strains with bwa. The reads were mapped and combined in one step. Next, we sorted and created bam files from the sam files using SAMtools. Together with the paired sequencing reads, these were used as input for Pilon to iteratively improve each genome.</p>
<p id="Par48">The genome assemblies were deposited at GenBank the accession no. GCA_022829205.1 (CBS 164.80), GCA_022829195.1 (TUCIM 5799), and GCA_022829225.1 (TUCIM 5827).</p></section><section id="Sec10"><h3 class="pmc_sec_title">Phylogenetic analysis</h3>
<p id="Par49">To confirm the taxonomic identity of the two newly isolated strains (TUCIM 5799 and TUCIM 5827), we performed molecular identification using a multi-locus sequence analysis based on the internal transcribed spacer (ITS), the large subunit ribosomal RNA gene (LSU), and the beta-tubulin gene (<em>tub2</em>). Sequences used for the multiple sequence alignment can be seen in Table <a href="#Tab1" class="usa-link">1</a>. Multiple sequence alignment was performed by using MAFFT [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. The alignments for each gene were manually curated and concatenated using MEGA [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Based on this MSA iqtree2 was used to generate phylogenetic trees using the “K2P + I” nt substitute model [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Trees were generated by applying 1000 bootstraps in 10 individual runs each. Numbers at nodes indicate bootstrap support values in %. The tree was visualized using figtree software [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>].</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Isolates and GenBank accession numbers used in the phylogenetic analyses. NA, not available</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="3" rowspan="1">GenBank accession number</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Species</th>
<th align="left" colspan="1" rowspan="1">Strain</th>
<th align="left" colspan="1" rowspan="1">ITS</th>
<th align="left" colspan="1" rowspan="1">LSU</th>
<th align="left" colspan="1" rowspan="1">tub2</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium moseri</em>
</td>
<td align="left" colspan="1" rowspan="1">CBS 164.80</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/LN850995" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">LN850995</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/LN851049" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">LN851049</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/LN851154" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">LN851154</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium moseri</em>
</td>
<td align="left" colspan="1" rowspan="1">TUCIM 5799</td>
<td align="left" colspan="3" rowspan="1">Additional File 1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium moseri</em>
</td>
<td align="left" colspan="1" rowspan="1">TUCIM 5827</td>
<td align="left" colspan="3" rowspan="1">Additional File 1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium urticae</em>
</td>
<td align="left" colspan="1" rowspan="1">IMI 326,344</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/AB220245" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AB220245</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/AB220339" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AB220339</a></td>
<td align="left" colspan="1" rowspan="1">NA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium trachycarpi</em>
</td>
<td align="left" colspan="1" rowspan="1">CFCC 53,038</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MK301098" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MK301098</a></td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MK303394" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MK303394</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium trachycarpi</em>
</td>
<td align="left" colspan="1" rowspan="1">CFCC 53,039</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MK301099" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MK301099</a></td>
<td align="left" colspan="1" rowspan="1">NA</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MK303395" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MK303395</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium lithocarpicola</em>
</td>
<td align="left" colspan="1" rowspan="1">CFCC 54,456</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/ON427580" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ON427580</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/ON427582" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ON427582</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/ON456914" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ON456914</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium lithocarpicola</em>
</td>
<td align="left" colspan="1" rowspan="1">CFCC 55,883</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/ON427581" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ON427581</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/ON427583" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ON427583</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/ON456915" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ON456915</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium brasiliense</em>
</td>
<td align="left" colspan="1" rowspan="1">URM 8364</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/OQ540770" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OQ540770</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/OQ540773" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OQ540773</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/OQ473592" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OQ473592</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium brasiliense</em>
</td>
<td align="left" colspan="1" rowspan="1">V150.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/OQ540771" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OQ540771</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/OQ540774" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OQ540774</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/OQ473593" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OQ473593</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium aquaticum</em>
</td>
<td align="left" colspan="1" rowspan="1">P4A41</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ481181" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ481181</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ481182" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ481182</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ639434" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ639434</a></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Neoarthrinium lewisiae</em>
</td>
<td align="left" colspan="1" rowspan="1">BRIP72527g</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV364379" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV364379</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PV364384" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV364384</a></td>
<td align="left" colspan="1" rowspan="1">NA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Lepteutypa fuckelii</em> (outgroup)</td>
<td align="left" colspan="1" rowspan="1">CBS 140,409</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_154123" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_154123</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/KT949902" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">KT949902</a></td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MH554677" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MH554677</a></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec11"><h3 class="pmc_sec_title">Gene prediction</h3>
<p id="Par51">To predict the genes, we first masked the repetitive elements in the nuclear genomes of <em>N. moseri</em> CBS 164.80 and our two new isolates to reduce the number of false positives during the subsequent gene prediction using RepeatMasker [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] Further, we performed an tRNA prediction with tRNAscan-SE v1.3.1 [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] using the unmasked genome. tRNAscan-SE.</p>
<p id="Par52">For the gene prediction, we used Augustus v3.3.2 [44], because no transcriptome data was available. Augustus v3.3.2 [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] was trained with the genome of <em>Pestalotiopsis fici</em> (assembly PFICI; BioSample accession: SAMN02369365) following the protocol by Hoff &amp; Stanke [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. The genomes and the gene sets were evaluated using Quast v5.0.2 [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Quast v5.0.2 includes benchmarking with Benchmarking Universal Single-Copy Orthologs (BUSCO) v3.0.2, this was performed with the eukaryote dataset of 303 BUSCOs from 100 species. We further evaluated the gene predictions by aligning the amino acid sequences using Blastp v2.9.0+ [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>] against the UniProt database [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>].</p></section><section id="Sec12"><h3 class="pmc_sec_title">Annotation</h3>
<p id="Par53">The gene sets were first annotated using Blastp against the UniProt protein database. Protein ANNotation with Z-scoRE (PANNZER2) [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] was used to provide both GO and free text DE producing an accurate functional annotation. CAZymes were annotated using the dbCAN2 [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>] meta server by applying a HMMer (Hidden Markov model) search [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>], a DIAMOND [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] search and a Hotpep [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] search and combining the three outputs.</p></section><section id="Sec13"><h3 class="pmc_sec_title">BGC Genome mining and comparison</h3>
<p id="Par54">The antiSMASH 7.1.0 fungal-version web version [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] was used for genome mining for secondary metabolite BGCs with following extra features applied: KnownClusterBlast, ClusterBlast, SubClusterBlast, MIBiG cluster comparison, ActiveSiteFinder, RREFinder, Cluster Pfam analysis, Pfam-based GO term annotation, TIGRFam analysis (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Genomes used for comparative antismash genome mining</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Organism</th>
<th align="left" colspan="1" rowspan="1">Strain</th>
<th align="left" colspan="1" rowspan="1">GenBank</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Aspergillus flavus</em>
</td>
<td align="left" colspan="1" rowspan="1">NRRL3357</td>
<td align="left" colspan="1" rowspan="1">GCA_014117465.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Aspergillus fumigatus</em>
</td>
<td align="left" colspan="1" rowspan="1">Af293</td>
<td align="left" colspan="1" rowspan="1">GCA_000002655.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Fusarium oxysporum</em>
</td>
<td align="left" colspan="1" rowspan="1">Fo47</td>
<td align="left" colspan="1" rowspan="1">GCA_013085055.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Apiospora malaysiana</em>
</td>
<td align="left" colspan="1" rowspan="1">STlab-iicb</td>
<td align="left" colspan="1" rowspan="1">GCA_006508115.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Hypoxylon fragiforme</em>
</td>
<td align="left" colspan="1" rowspan="1">CBS 206.31</td>
<td align="left" colspan="1" rowspan="1">GCA_022984875.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Microdochium bolleyi</em>
</td>
<td align="left" colspan="1" rowspan="1">J235TASD1</td>
<td align="left" colspan="1" rowspan="1">GCA_001566295.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Pestalotiopsis fici</em>
</td>
<td align="left" colspan="1" rowspan="1">W106-1-2</td>
<td align="left" colspan="1" rowspan="1">GCA_000516985.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Poronia punctata</em>
</td>
<td align="left" colspan="1" rowspan="1">CBS 180.79</td>
<td align="left" colspan="1" rowspan="1">GCA_022579005.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Rosellinia necatrix</em>
</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/CMW50482" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CMW50482</a></td>
<td align="left" colspan="1" rowspan="1">GCA_026420105.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Xylaria longipes</em>
</td>
<td align="left" colspan="1" rowspan="1">CBS 148.73</td>
<td align="left" colspan="1" rowspan="1">GCA_025201785.1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par56">The detected BGCs within the <em>N. moseri</em> genomes were further compared to each other using BiG-SLiCE v2.0.0 [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] (with the HMM database ‘bigslice-models-2022-11-30’) under default feature extraction, clustering settings, cutoffs, and GCF clustering thresholds. Additionally, BiG-SCAPE v1.1.8 [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] was used for comparative analysis, employing the latest release of the Pfam-A.hmm database [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>] (processed with hmmpress from HMMER), with a distance threshold of 0.3 for GCF clustering and MIBiG comparison enabled.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Ecophysiological profiling</h3>
<p id="Par57">MEX-medium (30 g/l malt extract, 1 g/l peptone, 15 g/l agar) was used as basis to investigate growth of the three <em>N. moseri</em> strains under different stress conditions.</p>
<p id="Par58">The strains were incubated at different temperatures (37 °C, 28 °C, and 21 °C) in order to narrow down the possible optimal growth temperature of the strains.</p>
<p id="Par59">For the purpose of testing the tolerance to increasing salinity, NaCl was added to the MEX plates to a final concentration of 0, 0.5, 1, 1.5, 2, and 2.5 M, respectively.</p>
<p id="Par60">Additionally, the tolerance of the strains to varying pH in the medium was tested. Therefore, the MEX plates were adjusted to pH 2, 3, 5, 7, 8, and 9 with HCl and NaOH, respectively, under sterile conditions. 10 g/l Phytagel + 5 mM MgCl<sub>2</sub> was used instead of agar in those plates. The plates used for salinity- and pH-tolerance-testing were incubated at 28 °C. All plates were inoculated by applying 5 µl of a spore solution (OD<sub>600</sub> of 3) to the center of the plates and were incubated at the according temperatures for 11 days.</p>
<p id="Par61">The resulting colony radii were measured using FIJI (ImageJ2, Version 2.9.0).</p></section><section id="Sec15"><h3 class="pmc_sec_title">BIOLOG assay</h3>
<p id="Par62">Growth of <em>N. moseri</em> strains on 95 different carbon sources was performed using the BIOLOG FF (Filamentous Fungi) MicroPlate™ (Art.Nr. 1006) panels (Biolog, Hayward, CA, United States). Spore solution was applied to FF Inoculating Fluid (Art.Nr. 72106) to a turbidity-increase of 20% (Turbidity Deice Name) and incubated for 16 days at 28 °C. Fungal growth was determined by measuring optical density at 750 nm (OD<sub>750</sub>) using a plate reader (TECAN Spark<sup>®</sup> Multimode Microplate Reader) after each 24 h, starting after inoculation (day 0). The assay was performed in technical triplicates. We compared OD<sub>750</sub>max independent from growth rate as a simple method to estimate the potential biomass formation on carbon sources. Data was evaluated using GraphPad Prism 9.1.2 (GraphPad Software, LLC.). Statistics were performed using GraphPad Prism 9.1.2 (GraphPad Software, LLC.).</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec17"><h3 class="pmc_sec_title">
<em>Isolation and DNA Barcoding of two epiphytic</em> N. moseri <em>strains</em>
</h3>
<p id="Par63">In 2008, we isolated the two epiphytic fungi TUCIM 5799 and TUCIM 5827 from the adaxial surface of the healthy high canopy leaf of <em>Shorea johorensis</em> (<em>Dipterocarpaceae</em>, <em>Malvales</em>; DNA BarCode maturase K (<em>matK</em>) deposited in NCBI GenBank <a href="https://www.ncbi.nlm.nih.gov/nuccore/MF993320.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MF993320.1</a> [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]) on Borneo.</p>
<p id="Par64">The two new isolates as well as the reference strain CBS 164.80 form a light-colored mycelium (beige on malt extract and Czapek yeast autolysate plates, white on oatmeal after 14 days, Fig. <a href="#Fig1" class="usa-link">1</a>A-I). The texture of the mycelia and the size of the colonies differ amongst the strains and depend on the culture media. Further, we observed the formation of a large quantity of conidia on oatmeal plates. The conidia were present in a slimy layer on the surface of the colonies. The strain TUCIM 5799 also produced smaller amounts of conidia on malt extract and Czapek yeast autolysate plates. The conidia of the two new isolates and the <em>N. moseri</em> reference strain look alike: They are melanized, dark colored, and are most frequently pear-shaped, with a length of 4.2–6.7 μm long and a width of 3–3.9 μm, in contrast to essentially larger dimensions previously reported for this species [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>] (Fig. <a href="#Fig1" class="usa-link">1</a>J-O; Table <a href="#Tab3" class="usa-link">3</a>) and more consistent with the values obtained for <em>N. trachycarpi</em> (6.1–8.5 × 4.2–5.3 μm) [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Thus, we re-assessed Gams’ SEM images [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>], and measured actually similar sizes we obtained from our own picture (Table <a href="#Tab3" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369201_12866_2025_4274_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1758/12369201/f6d848c28a74/12866_2025_4274_Fig1_HTML.jpg" loading="lazy" id="MO1" height="1245" width="744" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Morphology of <em>N. moseri</em> CBS 164.80 (A, D, G), TUCIM 5799 (B, E, H), and TUCIM 5827 (C, F, I) on malt extract (A-C), Czapek yeast autolysate (D-F), and oatmeal (G-I) plates after incubation at 28 °C for 14 days. Brightfield and scanning electron microscopy of spores of <em>N. moseri</em> CBS. 164.80 (J, M), TUCIM 5799 (K, N), and TUCIM 5827 (L, O).</p></figcaption></figure><section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Average conidia dimensions</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Strain</th>
<th align="left" colspan="1" rowspan="1">Length [µm]</th>
<th align="left" colspan="1" rowspan="1">Width [µm]</th>
<th align="left" colspan="1" rowspan="1">Reference</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. moseri</em> CBS 164.80</td>
<td align="left" colspan="1" rowspan="1">10–14</td>
<td align="left" colspan="1" rowspan="1">3–4.5</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>] (original values)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. moseri</em> CBS 164.80</td>
<td align="left" colspan="1" rowspan="1">4.5–7.1</td>
<td align="left" colspan="1" rowspan="1">3.1–4.3</td>
<td align="left" colspan="1" rowspan="1">Re-assessment of pictures from [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. moseri</em> CBS 164.80</td>
<td align="left" colspan="1" rowspan="1">4.4–6</td>
<td align="left" colspan="1" rowspan="1">3.1–4.1</td>
<td align="left" colspan="1" rowspan="1">This publication</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. moseri</em> TUCIM 5799</td>
<td align="left" colspan="1" rowspan="1">4.2–5.7</td>
<td align="left" colspan="1" rowspan="1">3.2–3.8</td>
<td align="left" colspan="1" rowspan="1">This publication</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. moseri</em> TUCIM 5827</td>
<td align="left" colspan="1" rowspan="1">4.3–6.7</td>
<td align="left" colspan="1" rowspan="1">3–3.9</td>
<td align="left" colspan="1" rowspan="1">This publication</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. trachycarpi</em>
</td>
<td align="left" colspan="1" rowspan="1">6.1–8.5</td>
<td align="left" colspan="1" rowspan="1">4.2–5.8</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. lithocarpicola</em>
</td>
<td align="left" colspan="1" rowspan="1">5–8.5</td>
<td align="left" colspan="1" rowspan="1">4.5–6</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. brasiliense</em>
</td>
<td align="left" colspan="1" rowspan="1">4–5</td>
<td align="left" colspan="1" rowspan="1">3–4</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. aquaticum</em>
</td>
<td align="left" colspan="1" rowspan="1">8–13</td>
<td align="left" colspan="1" rowspan="1">6–10</td>
<td align="left" colspan="1" rowspan="1">[<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. urticae</em>
</td>
<td align="left" colspan="1" rowspan="1">n/a</td>
<td align="left" colspan="1" rowspan="1">n/a</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. lewisiae</em>
</td>
<td align="left" colspan="1" rowspan="1">n/a</td>
<td align="left" colspan="1" rowspan="1">n/a</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par66">To confirm the classification of the new isolates as <em>N. moseri</em> strains, we performed a multiple sequence alignment of the available ITS, LSU and <em>tub2</em> sequences of the <em>Neoarthrinium</em> strains and constructed a phylogenetic tree (Fig. <a href="#Fig2" class="usa-link">2</a>). <em>N. moseri</em> CBS 164.80 clusters together with two <em>N. trachycarpi</em> strains, <em>N. urticae</em> and the two new isolates, with the two Borneo isolates forming a distinct subclade (Fig. <a href="#Fig2" class="usa-link">2</a>). Notably, all pairwise comparisons within the <em>N. moseri</em> clade showed very high sequence identity across the ITS, LSU, and <em>tub2</em> loci, generally exceeding 99%, with the lowest observed identity being 97.77% (Additional File 2).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369201_12866_2025_4274_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1758/12369201/ed307b01b56b/12866_2025_4274_Fig2_HTML.jpg" loading="lazy" id="MO2" height="419" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylogenetic tree based on the concatenated multiple sequence alignment of ITS, LSU and <em>tub2 </em>of the indicated fungal isolates (type strains in bold)<em>.</em> The rooted phylogenetic tree is the consensus of 10 individual runs applying 1000 bootstraps utilizing the maximum-likelihood approach. Values at nodes indicate bootstrap support values in %.</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">Mitochondrial and nuclear genome of N. moseri</h3>
<p id="Par67">To assess the genetic makeup and potential metabolic capabilities of <em>N. moseri</em>, we sequenced and assembled the complete mitochondrial and nuclear genomes of three strains using Illumina paired-end sequencing followed by SPAdes assembly, with additional refinement steps including NOVOPlasty for mitochondria and Pilon for genome polishing. The extracted circularized mitochondrial genomes have a length of 42,769 bp, 43,978 bp, and 42,769 bp and with GC contents of 27.52%, 27.52%, and 27.53% for the strain CBS 164.80, TUCIM 5799, and TUCIM 5827, respectively (Fig. <a href="#MOESM1" class="usa-link">S1</a>). The respective average sequencing coverages were at 364x, 8,939x, and 464x.</p>
<p id="Par68">The size of <em>N. moseri</em> nuclear genomes is between 43.7 Mbp and 46.1 Mbp with average sequencing coverages between 32x and 141x. The detailed results of the genomes and assembly characteristics (size, GC content, characteristics for scaffold number and size, N<sub>50</sub> and L<sub>50</sub>) are summarized in Table <a href="#MOESM1" class="usa-link">S1</a>. To evaluate the completeness of the genome assembly, we performed a Benchmarking Universal Single-Copy Orthologues (BUSCO) analysis with the eukaryote dataset [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. 100% complete BUSCOs without duplicates were found in all three assemblies (Table <a href="#MOESM1" class="usa-link">S1</a>). Further, we calculated the average nucleotide identity (ANI) and found the three strains to be highly similar (Table <a href="#MOESM2" class="usa-link">S2</a>). Additionally, the genomes of the three strains exhibited a similar GC content of around 52.7%, and masked element analysis indicated a low level of repetitive sequences, with simple repeats and low complexity regions occupying less than 1% of the genomes.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Gene prediction and annotation</h3>
<p id="Par69">First, we identified and masked the repetitive elements in the nuclear genomes of the three <em>N. moseri</em> strains (Table <a href="#Tab4" class="usa-link">4</a>). Additionally, we performed a tRNA prediction and found a total of 196, 190 and 189 tRNA genes, respectively (Table <a href="#Tab4" class="usa-link">4</a>, Additional Files 3–5). The analysis of repetitive elements revealed that less than 1% of each <em>N. moseri</em> genome was composed of transposable elements or low-complexity regions, indicating a relatively compact genome architecture with few repetitive sequences. Among the identified elements, short interspersed nuclear elements (SINEs) and long interspersed nuclear elements (LINEs) are known to be non-coding mobile elements that can influence genome evolution and gene regulation. Simple repeats and low complexity regions often play roles in genome structure, microsatellite formation, or replication slippage but are typically underrepresented in compact fungal genomes. The consistent tRNA gene counts (189–196) across strains suggest a conserved translational capacity among the three sequenced <em>N. moseri</em> strains.</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Masked repetitive elements and tRNA genes found in the genomes of the <em>N. moseri</em> strains CBS 164.80 (CBS), TUCIM 5799 (5799) and TUCIM 5827 (5827)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Masked element</th>
<th align="left" colspan="3" rowspan="1">Number of elements*</th>
<th align="left" colspan="3" rowspan="1">Length occupied in bp</th>
<th align="left" colspan="3" rowspan="1">Percentage of sequence</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Strain</td>
<td align="left" colspan="1" rowspan="1">CBS</td>
<td align="left" colspan="1" rowspan="1">5799</td>
<td align="left" colspan="1" rowspan="1">5827</td>
<td align="left" colspan="1" rowspan="1">CBS</td>
<td align="left" colspan="1" rowspan="1">5799</td>
<td align="left" colspan="1" rowspan="1">5827</td>
<td align="left" colspan="1" rowspan="1">CBS</td>
<td align="left" colspan="1" rowspan="1">5799</td>
<td align="left" colspan="1" rowspan="1">5827</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SINEs</td>
<td align="left" colspan="1" rowspan="1">35</td>
<td align="left" colspan="1" rowspan="1">35</td>
<td align="left" colspan="1" rowspan="1">33</td>
<td align="left" colspan="1" rowspan="1">2,289</td>
<td align="left" colspan="1" rowspan="1">2,404</td>
<td align="left" colspan="1" rowspan="1">2,231</td>
<td align="left" colspan="1" rowspan="1">0.01%</td>
<td align="left" colspan="1" rowspan="1">0.01%</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LINEs</td>
<td align="left" colspan="1" rowspan="1">223</td>
<td align="left" colspan="1" rowspan="1">222</td>
<td align="left" colspan="1" rowspan="1">220</td>
<td align="left" colspan="1" rowspan="1">16,838</td>
<td align="left" colspan="1" rowspan="1">17,399</td>
<td align="left" colspan="1" rowspan="1">16,757</td>
<td align="left" colspan="1" rowspan="1">0.04%</td>
<td align="left" colspan="1" rowspan="1">0.04%</td>
<td align="left" colspan="1" rowspan="1">0.04%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LTR elements</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">300</td>
<td align="left" colspan="1" rowspan="1">204</td>
<td align="left" colspan="1" rowspan="1">200</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DNA elements</td>
<td align="left" colspan="1" rowspan="1">50</td>
<td align="left" colspan="1" rowspan="1">49</td>
<td align="left" colspan="1" rowspan="1">55</td>
<td align="left" colspan="1" rowspan="1">3,751</td>
<td align="left" colspan="1" rowspan="1">3,598</td>
<td align="left" colspan="1" rowspan="1">4,260</td>
<td align="left" colspan="1" rowspan="1">0.01%</td>
<td align="left" colspan="1" rowspan="1">0.01%</td>
<td align="left" colspan="1" rowspan="1">0.01%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Unclassified</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">142</td>
<td align="left" colspan="1" rowspan="1">142</td>
<td align="left" colspan="1" rowspan="1">72</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Small RNA</td>
<td align="left" colspan="1" rowspan="1">86</td>
<td align="left" colspan="1" rowspan="1">78</td>
<td align="left" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1">12,181</td>
<td align="left" colspan="1" rowspan="1">12,157</td>
<td align="left" colspan="1" rowspan="1">11,815</td>
<td align="left" colspan="1" rowspan="1">0.05%</td>
<td align="left" colspan="1" rowspan="1">0.03%</td>
<td align="left" colspan="1" rowspan="1">0.05%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Simple repeats</td>
<td align="left" colspan="1" rowspan="1">7,572</td>
<td align="left" colspan="1" rowspan="1">7,432</td>
<td align="left" colspan="1" rowspan="1">7,532</td>
<td align="left" colspan="1" rowspan="1">306,538</td>
<td align="left" colspan="1" rowspan="1">294,925</td>
<td align="left" colspan="1" rowspan="1">297,695</td>
<td align="left" colspan="1" rowspan="1">0.70%</td>
<td align="left" colspan="1" rowspan="1">0.66%</td>
<td align="left" colspan="1" rowspan="1">0.64%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Low complexity</td>
<td align="left" colspan="1" rowspan="1">652</td>
<td align="left" colspan="1" rowspan="1">600</td>
<td align="left" colspan="1" rowspan="1">652</td>
<td align="left" colspan="1" rowspan="1">30,991</td>
<td align="left" colspan="1" rowspan="1">27,670</td>
<td align="left" colspan="1" rowspan="1">30,995</td>
<td align="left" colspan="1" rowspan="1">0.06%</td>
<td align="left" colspan="1" rowspan="1">0.06%</td>
<td align="left" colspan="1" rowspan="1">0.07%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">tRNA</td>
<td align="left" colspan="1" rowspan="1">196</td>
<td align="left" colspan="1" rowspan="1">189</td>
<td align="left" colspan="1" rowspan="1">190</td>
<td align="left" colspan="1" rowspan="1">17,154</td>
<td align="left" colspan="1" rowspan="1">16,788</td>
<td align="left" colspan="1" rowspan="1">16,884</td>
<td align="left" colspan="1" rowspan="1">0.04%</td>
<td align="left" colspan="1" rowspan="1">0.04%</td>
<td align="left" colspan="1" rowspan="1">0.04%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par71">As no transcriptome data were available, the gene prediction was performed on the masked genome using a model trained with the genome of <em>P. fici</em>. We obtained approx. 14,000 genes for <em>N. moseri</em> (Table <a href="#Tab5" class="usa-link">5</a>). A significant portion of the predicted genes (34.4–36.7%) did not match any sequences in public databases below the E<sup>−5</sup> threshold (Table <a href="#Tab5" class="usa-link">5</a>), suggesting the presence of potentially novel genes unique to <em>N. moseri</em>.</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Gene predictions</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Strain</th>
<th align="left" colspan="1" rowspan="1">Predicted putative genes</th>
<th align="left" colspan="1" rowspan="1">genes without BLAST hits below E<sup>−5</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">CBS 164.80</td>
<td align="left" colspan="1" rowspan="1">13,929</td>
<td align="left" colspan="1" rowspan="1">4,797 (34.4%)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TUCIM 5799</td>
<td align="left" colspan="1" rowspan="1">14,160</td>
<td align="left" colspan="1" rowspan="1">4,964 (35.0%)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TUCIM 5827</td>
<td align="left" colspan="1" rowspan="1">14,595</td>
<td align="left" colspan="1" rowspan="1">5,352 (36.7%)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par73">The predicted gene sets were annotated by blasting them against the UniProt database and via the PANNZER2 web interface. The combined functional annotations are given in Additional Files 6–8.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Genome mining for CAZymes</h3>
<p id="Par74">The ability to decompose organic matter and the saprotrophic lifestyle are hallmarks of fungal biology. Fungi thrive on plant biomass and other natural materials by degrading complex and simple carbohydrates using so-called carbohydrate active enzymes (CAZymes) [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. We used dbCAN2 (a meta-server for CAZyme annotation) and a HMMer (Hidden Markov model) search [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>], a DIAMOND search [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>], and a Hotpep search [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] to predict the CAZymes in the three <em>N. moseri</em> genomes (Fig. <a href="#Fig3" class="usa-link">3</a>; Table <a href="#Tab6" class="usa-link">6</a>). In total, 1,005, 1,011, and 1,018 CAZymes were predicted by all three methods in CBS 164.80, TUCIM 5799, and TUCIM 5827, respectively (Fig. <a href="#Fig3" class="usa-link">3</a>, Additional Files 9–11, including 455, 455, and 460 genes predicted by all three methods (Fig. <a href="#Fig3" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369201_12866_2025_4274_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1758/12369201/290cba9f143c/12866_2025_4274_Fig3_HTML.jpg" loading="lazy" id="MO3" height="222" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The genomes of the three sequences <em>N. moseri</em> strains were mined for putative CaZymes using Diamond, HMMER, and Hotpep.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>The carbohydrate active enzymes (CAZymes) found with dbCAN2 a meta-server for cazyme annotation. Glycosyltransferases (GT); glycoside hydrolases (GH); carbohydrate esterases (CE); polysaccharide lyases (PL); redox enzymes with auxiliary activities (AA)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Strain</th>
<th align="left" colspan="1" rowspan="1">Total</th>
<th align="left" colspan="1" rowspan="1">GT</th>
<th align="left" colspan="1" rowspan="1">GH</th>
<th align="left" colspan="1" rowspan="1">CE</th>
<th align="left" colspan="1" rowspan="1">PL</th>
<th align="left" colspan="1" rowspan="1">AA</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. moseri</em> CBS 164.80</td>
<td align="left" colspan="1" rowspan="1">1005</td>
<td align="left" colspan="1" rowspan="1">148</td>
<td align="left" colspan="1" rowspan="1">476</td>
<td align="left" colspan="1" rowspan="1">93</td>
<td align="left" colspan="1" rowspan="1">27</td>
<td align="left" colspan="1" rowspan="1">222</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. moseri</em> TUCIM 5799</td>
<td align="left" colspan="1" rowspan="1">1011</td>
<td align="left" colspan="1" rowspan="1">152</td>
<td align="left" colspan="1" rowspan="1">479</td>
<td align="left" colspan="1" rowspan="1">94</td>
<td align="left" colspan="1" rowspan="1">27</td>
<td align="left" colspan="1" rowspan="1">222</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>N. moseri</em> TUCIM 5827</td>
<td align="left" colspan="1" rowspan="1">1018</td>
<td align="left" colspan="1" rowspan="1">151</td>
<td align="left" colspan="1" rowspan="1">476</td>
<td align="left" colspan="1" rowspan="1">95</td>
<td align="left" colspan="1" rowspan="1">27</td>
<td align="left" colspan="1" rowspan="1">231</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>P. fici</em>
</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">121</td>
<td align="left" colspan="1" rowspan="1">460</td>
<td align="left" colspan="1" rowspan="1">138</td>
<td align="left" colspan="1" rowspan="1">39</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par75">The dbCAN2 server also predicts certain subclasses of CAZyme. Glycosyltransferases (GT families) catalyze glycosidic bond formation and inversion and are part of the posttranslational modification steps in different compound formation processes. Glycoside hydrolases (GH families) is a large group of enzyme families which hydrolyse glycosidic bonds. Carbohydrate esterases (e.g., CE1, CE10 families) catalyze de-N or de-O-acylation of ester bonds in saccharides like in pectin. Polysaccharide lyases (e.g., PL1, PL7 families) cleave polysaccharide chains via β-elimination. Redox enzymes with auxiliary activities are involved in the breakdown processes of polysaccharides and lignin. The respective numbers of the predicted CAZymes subclasses (<em>sensu</em> dbCAN2) are also listed in Table <a href="#Tab6" class="usa-link">6</a>. For comparative purposes, we selected <em>P. fici</em> as a reference because it belongs to the same order (<em>Amphisphaeriales</em>) and has a publicly available, well-annotated genome with CAZyme predictions. While broader comparative analyses across more taxa could offer additional insight, our focus here was to provide a contextually relevant benchmark from a phylogenetically close relative.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Genome mining for secondary metabolites</h3>
<p id="Par77">We used antiSMASH [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] to mine the genomes of the three <em>N. moseri</em> strains for genes that might be involved in the production of secondary metabolites and compared them to a few fungi of the same order (<em>Amphisphaeriales</em>) and the sister-taxon, the <em>Xylariales</em>, as well as the proficient secondary metabolite-producers <em>Aspergillus flavus</em>, <em>A. fumigatus</em> (<em>Eurotiales</em>) [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>] and <em>Fusarium oxysporum</em> species complex (<em>Hypocreales</em>) [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Notably, the three <em>N. moseri</em> strains exhibited the highest BGC-count among the compared strains (85 in CBS 164.80, 88 in TUCIM 5799, 90 in TUCIM 5827) (Fig. <a href="#Fig4" class="usa-link">4</a>, Additional File 12).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369201_12866_2025_4274_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1758/12369201/4cb617f5961b/12866_2025_4274_Fig4_HTML.jpg" loading="lazy" id="MO4" height="647" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of the predicted BGCs (antiSMASH 7.0) in the genomes of the indicated fungal species, including the three sequenced <em>N. moseri </em>strains (<em>Amphisphaeriales</em> genus<em> incertae sedis</em>). NRPS, non-ribosomal peptide synthetase; PKS, polyketide synthase; RiPP, ribosomally synthesized and post-translationally modified peptides; RCDP, arginine-containing cyclodipeptide synthase; hybrids, BGCs that contain core enzymes with characteristics for different classes, e.g. PKS-NRPS fusion enyzmes.</p></figcaption></figure><p id="Par78">Next, we compared the identified BGCs in three <em>N. moseri</em> strains using BiG-SLiCE (Additional Files 13 and 14). A total of 244 BGCs were used as input. The analysis grouped them into 88 Gene Cluster Families (GCFs). The majority of GCFs (76) contained matching BGCs from all three strains, reflecting high conservation of biosynthetic loci within the species. Eight GCFs were exclusively found in one strain, and four GCFs were shared between two strains. Notably, the singleton GCFs were predominantly RiPP-like BGCs (Additional File 14), suggesting limited conservation or lower comparability in this class.</p>
<p id="Par79">To validate and complement these results, we additionally performed a BiG-SCAPE analysis on the same set of BGCs (Additional file 15). The results were largely consistent with the BiG-SLiCE output. One previously ungrouped RiPP BGC could be matched to a homologous pair, while another singleton BGC was found to be part of a larger, split BGC locus that had been predicted as two separate BGCs by antiSMASH (Additional File 14). Interestingly, among all detected BGCs, only one GCF could be associated with a known MIBiG cluster, namely the scytalone/T3HN BGC (Additional File 14).To get a better understanding of the secondary metabolite potential, we manually compared the predicted BGCs of the <em>N. moseri</em> strains to the MIBIG 4.0 database [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>] and assessed the predictions by a manual BGC comparison using the cblaster tool [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>] if the BGC contained more than one gene (Additional File 16). We found 14 BGCs similar to previously characterized BGCs (Additional File 17). Additional to BGCs for common compounds, such as siderophores or choline, we also found BGCs for antibacterial and antifungal substances, such as citridone A and related compounds, fusaric acid, and (-)-mellein. We also found BGCs highly similar to BGCs reported from plant pathogenic fungi, e.g. brassicicene C and koraiol, and the plant growth hormone gibberellin. Further, we detected BGCs for pharmaceutically interesting compounds such as the histone deacetylase inhibitor depudecin, the immunomodulator swainsonine, and the cytotoxin eupenifeldin.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Growth optima and stress tolerance</h3>
<p id="Par80">To gain some insights into the ecophysiology of <em>N. moseri</em>, we cultivated the three strains on malt extract plates (MEX) with varying NaCl concentrations, pH values, or at different temperatures (Fig. <a href="#Fig5" class="usa-link">5</a>A-C, individual growth curves are depicted in Fig. <a href="#MOESM2" class="usa-link">S2</a>). All three <em>N. moseri</em> strains grew on MEX with NaCl concentrations ranging from 0 to 2.5 M (Fig. <a href="#Fig5" class="usa-link">5</a>A), with optimal growth at 0 M and the least at 2.5 M. Mycelial growth progressively decreased as NaCl concentration increased.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369201_12866_2025_4274_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1758/12369201/6b632ab177e4/12866_2025_4274_Fig5_HTML.jpg" loading="lazy" id="MO5" height="539" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Growth response of the<em>N. moseri</em> strains to different NaCl-concentrations (A), pH (B), and temperatures (C), respectively, after 11 days of incubation on MEX. Data shows the mean of three experiments ± SD. Corresponding statistics are provided in Additional File 18.</p></figcaption></figure><p id="Par81">All three <em>N. moseri</em> strains grew on the media adjusted to any of the pH values between 2 and 9 (Fig. <a href="#Fig5" class="usa-link">5</a>B), with pH 2 as the least tolerated condition. All three strains grew comparatively well at pH 3 with CBS 164.80 and TUCIM 5827 having their optima at this condition. In contrast, TUCIM 5799 has its growth optimum at pH 5, which was the second-least favorite condition for the other two strains. Regarding pH 7, 8, and 9, the three strains behaved similar to each other and showed modest growth. The optimum in the alkaline pH range was surprisingly at pH 9.</p>
<p id="Par82">All three <em>N. moseri</em> strains grew at 21 °C and 28 °C (Fig. <a href="#Fig5" class="usa-link">5</a>C). There was no difference in growth of TUCIM 5799 and TUCIM 5827 between these incubation temperatures. CBS 164.80 grew slightly better at 28 °C than at 21 °C. Only TUCIM 5799 was able to tolerate 37 °C but showed poor growth.</p>
<p id="Par83">Although all three <em>N. moseri</em> strains showed robust growth under a wide range of conditions, notable physiological differences were observed. For example, TUCIM 5799 was the only strain to grow, albeit weakly, at 37 °C, indicating a slightly higher thermotolerance compared to the other strains (Fig. <a href="#Fig5" class="usa-link">5</a>C). Under acidic conditions, CBS 164.80 and TUCIM 5827 exhibited optimal growth at pH 3, whereas TUCIM 5799 grew best at pH 5. These observations suggest some degree of strain-specific adaptation to environmental pH.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Utilization of different carbon sources</h3>
<p id="Par84">We performed a BIOLOG Phenotype microarray to assess the carbon source utilization profiles of the three <em>N. moseri</em> strains. The growth was monitored over 16 days, and we used the maximal produced biomass throughout the growth period (OD<sub>750</sub>max) for comparison (Fig. <a href="#Fig6" class="usa-link">6</a>; Table <a href="#Tab7" class="usa-link">7</a>). The individual growth curves on each carbon source are shown in the supplements (Fig. <a href="#MOESM3" class="usa-link">S3</a>-<a href="#MOESM10" class="usa-link">S10</a>). The carbon sources were grouped as described previously [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. “No growth” indicates OD<sub>750</sub>max values less than or equal to the respective OD<sub>750</sub>max on water. Neither of the three <em>N. moseri</em> strains grew on sedoheptulosan, L-sorbose, glucuronamide, N-acetyl-D-mannosamine, and 2-amino ethanol. TUCIM 5827 did not grow on N-acetyl-galactosamine and α-methyl-D-glucoside, either. On all other tested carbon sources we observed differently strong growth, suggesting that <em>N. moseri</em> has an adaptive and diverse primary metabolism.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369201_12866_2025_4274_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1758/12369201/f3ef361f6e2f/12866_2025_4274_Fig6_HTML.jpg" loading="lazy" id="MO6" height="1077" width="744" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative heatmap of the OD<sub>750</sub>max of the three tested <em>N. moseri</em> strains on the indicated carbon sources in BIOLOG FF microplates. Grayscale indicates respective OD<sub>750</sub>max.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab7"><h4 class="obj_head">Table 7.</h4>
<div class="caption p"><p>Growth of the <em>N. moseri</em> strains on different carbon sources. Strong growth was defined as an OD<sub>750</sub>max of 15% above the median OD<sub>750</sub>max for each strain, weak growth as 15% below the median OD<sub>750</sub>max, medium in between these two thresholds</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Carbon-Source</th>
<th align="left" colspan="1" rowspan="1">CBS 164.80</th>
<th align="left" colspan="1" rowspan="1">TUCIM_5799</th>
<th align="left" colspan="1" rowspan="1">TUCIM_5827</th>
<th align="left" colspan="1" rowspan="1">Carbon-Source</th>
<th align="left" colspan="1" rowspan="1">TUCIM_5799</th>
<th align="left" colspan="2" rowspan="1">TUCIM_5827</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Water</td>
<td align="left" colspan="3" rowspan="1">no</td>
<td align="left" colspan="4" rowspan="1">Monosaccharide-related compounds: Sugar acids</td>
</tr>
<tr>
<td align="left" colspan="4" rowspan="1">Monosaccharides: Heptose</td>
<td align="left" colspan="1" rowspan="1">D-Galacturonic Acid</td>
<td align="left" colspan="3" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Sedoheptulosan</td>
<td align="left" colspan="3" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">D-Gluconic Acid</td>
<td align="left" colspan="3" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="4" rowspan="1">Monosaccharides: Hexoses</td>
<td align="left" colspan="1" rowspan="1">Glucuronamide</td>
<td align="left" colspan="3" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> D-Fructose</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">D-Glucuronic Acid</td>
<td align="left" colspan="3" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> L-Fucose</td>
<td align="left" colspan="2" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">2-Keto-D-Gluconic Acid</td>
<td align="left" colspan="3" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> D-Galactose</td>
<td align="left" colspan="3" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">D-Saccharic Acid</td>
<td align="left" colspan="3" rowspan="1">slight</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> α-D-Glucose</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" colspan="4" rowspan="1">Monosaccharide-related compounds: Hexosamines</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> D-Mannose</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">N-Acetyl-D-Galactosamine</td>
<td align="left" colspan="1" rowspan="1">slight</td>
<td align="left" colspan="2" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> L-Rhamnose</td>
<td align="left" colspan="2" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">N-Acetyl-D-Glucosamine</td>
<td align="left" colspan="3" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> L-Sorbose</td>
<td align="left" colspan="3" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">N-Acetyl-D-Mannosamine</td>
<td align="left" colspan="3" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> D-Tagatose</td>
<td align="left" colspan="1" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">D-Glucosamine</td>
<td align="left" colspan="3" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="4" rowspan="1">Monosaccarides: Pentoses</td>
<td align="left" colspan="4" rowspan="1">Monosaccharide-related compounds: Polyols</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> D-Arabinose</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">Adonitol</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="2" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> L-Arabinose</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="2" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">D-Arabitol</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="2" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> D-Psicose</td>
<td align="left" colspan="2" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1">i-Erythritol</td>
<td align="left" colspan="3" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> D-Ribose</td>
<td align="left" colspan="1" rowspan="1">slight</td>
<td align="left" colspan="2" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">Glycerol</td>
<td align="left" colspan="3" rowspan="1">slight</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> D-Xylose</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="2" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">m-Inositol</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="2" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="4"></td>
<td align="left" colspan="1" rowspan="1">Maltitol</td>
<td align="left" colspan="3" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D-Mannitol</td>
<td align="left" colspan="3" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D-Sorbitol</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="2" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Xylitol</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="2" rowspan="1">slight</td>
</tr>
<tr>
<td align="left" colspan="4" rowspan="1">Other sugars: Disaccharides</td>
<td align="left" colspan="4" rowspan="1">Other sugars: Glucosides</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D-Cellobiose</td>
<td align="left" colspan="3" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">Amygdalin</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="2" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Gentiobiose</td>
<td align="left" colspan="3" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">Arbutin</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">α-D-Lactose</td>
<td align="left" colspan="3" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">α-Methyl-D-Galactoside</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="2" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Lactulose</td>
<td align="left" colspan="3" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">β-Methyl-D-Galactoside</td>
<td align="left" colspan="3" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Maltose</td>
<td align="left" colspan="2" rowspan="1"> strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">α-Methyl-D-Glucoside</td>
<td align="left" colspan="2" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D-Melibiose</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">β-Methyl-D-Glucoside</td>
<td align="left" colspan="3" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Palatinose</td>
<td align="left" colspan="3" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">Salicin</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D-Raffinose</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">Stachyose</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="4" rowspan="1">Other sugars: Trisaccharides</td>
<td align="left" colspan="1" rowspan="1">Sucrose</td>
<td align="left" colspan="2" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Maltotriose</td>
<td align="left" colspan="2" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">D-Trehalose</td>
<td align="left" colspan="3" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">D-Melezitose</td>
<td align="left" colspan="3" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">Turanose</td>
<td align="left" colspan="3" rowspan="1">strong</td>
</tr>
<tr>
<td align="left" colspan="4" rowspan="1">Other Sugars: Polysaccharides</td>
<td align="left" colspan="4" rowspan="1">Nitrogen containing compounds: Peptides</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">α-Cyclodextrin</td>
<td align="left" colspan="3" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1">L-Alanyl-Glycine</td>
<td align="left" colspan="3" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">β-Cyclodextrin</td>
<td align="left" colspan="1" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1">Glycyl-L-Glutamic Acid</td>
<td align="left" colspan="3" rowspan="1">slight</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Dextrin</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" rowspan="2" colspan="4"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Glycogen</td>
<td align="left" colspan="3" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="4" rowspan="1">Nitrogen containing compounds: L-amino acids</td>
<td align="left" colspan="4" rowspan="1">Nitrogen containing compounds: TCA-Cylce Intermediates</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">γ-Amino-butyric Acid</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="1" rowspan="1">Fumaric Acid</td>
<td align="left" colspan="3" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">N-Acetyl-L-Glutamic Acid</td>
<td align="left" colspan="3" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1">α-Keto-Glutaric Acid</td>
<td align="left" colspan="2" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">slight</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L-Alanine</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">D-Malic Acid</td>
<td align="left" colspan="2" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L-Asparagine</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">L-Malic Acid</td>
<td align="left" colspan="2" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L-Aspartic Acid</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">Succinic Acid</td>
<td align="left" colspan="2" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L-Glutamic Acid</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" colspan="4" rowspan="1">Nitrogen containing compounds: Aliphatic Organic Acids</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L-Ornithine</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">Bromosuccinic Acid</td>
<td align="left" colspan="1" rowspan="1"> slight</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L-Phenylalanine</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="2" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">β-Hydroxy-butyric Acid</td>
<td align="left" colspan="1" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L-Proline</td>
<td align="left" colspan="1" rowspan="1">strong</td>
<td align="left" colspan="2" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">γ-Hydroxy-butyric Acid</td>
<td align="left" colspan="2" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L-Pyroglutamic Acid</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="2" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L-Serine</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" colspan="4" rowspan="1">Others</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">L-Threonine</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">Tween 80</td>
<td align="left" colspan="2" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">slight</td>
</tr>
<tr>
<td align="left" colspan="4" rowspan="1">Nitrogen containing compounds: Biogene and Heterocyclic Amines</td>
<td align="left" colspan="1" rowspan="1">Glucose-1-Phosphate</td>
<td align="left" colspan="3" rowspan="1">slight</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2-Amino Ethanol</td>
<td align="left" colspan="3" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">p-Hydroxyphenyl-acetic Acid</td>
<td align="left" colspan="3" rowspan="1">slight</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Putrescine</td>
<td align="left" colspan="3" rowspan="1">medium</td>
<td align="left" colspan="1" rowspan="1">D-Lactic Acid Methyl Ester</td>
<td align="left" colspan="3" rowspan="1">slight </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Adenosine</td>
<td align="left" colspan="3" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1">Quinic Acid</td>
<td align="left" colspan="3" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Uridine</td>
<td align="left" colspan="3" rowspan="1">slight</td>
<td align="left" colspan="1" rowspan="1">Sebacic Acid</td>
<td align="left" colspan="3" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="4"></td>
<td align="left" colspan="1" rowspan="1">Succinamic Acid</td>
<td align="left" colspan="3" rowspan="1">medium</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Succinic Acid Mono-Methyl Ester</td>
<td align="left" colspan="3" rowspan="1">slight</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Alaninamide</td>
<td align="left" colspan="3" rowspan="1">slight</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Adenosine-5’-Monophosphate</td>
<td align="left" colspan="3" rowspan="1">slight</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par85">All three <em>N. moseri</em> strains showed strong growth on saccharides (Fig. <a href="#Fig6" class="usa-link">6</a>). In contrast, nitrogen-containing compounds were poorly utilized. For CBS 164.80, we observed strong growth only on γ-amino-butyric acid, L-phenylalanine, and L-proline. TUCIM 5827 grew well only on γ-amino-butyric acid and TUCIM 5799 did not metabolize any of the tested nitrogen-containing compounds well (Table <a href="#Tab7" class="usa-link">7</a>).</p>
<p id="Par86">The carbon source utilization assay further revealed subtle but meaningful differences among the strains. Notably, CBS 164.80 and TUCIM 5827 grew strongly on γ-aminobutyric acid (GABA), whereas TUCIM 5799 showed only moderate growth (Table <a href="#Tab7" class="usa-link">7</a>). As GABA is a stress-related signaling compound in plants, this utilization pattern may indicate adaptive metabolic flexibility in response to host- or habitat-derived compounds. Additionally, TUCIM 5827 exhibited consistently lower biomass production across many carbon sources, suggesting reduced growth vigor or a more specialized metabolic strategy.</p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par88">In this study, we isolated and characterized two new strains of <em>N. moseri</em> (TUCIM 5799 and TUCIM 5827). Morphological and genetic analyses confirmed the attribution of these strains to <em>N. moseri</em>, while also revealing strain-specific variations in growth and morphological characteristics. <em>N. moseri</em> exhibited high adaptability across a range of growth conditions, including varying pH levels and salt concentrations, and demonstrated a diverse and adaptable carbon utilization pattern. Genome mining further uncovered an exceptional number of BGCs, highlighting the species’ considerable secondary metabolism potential. Additionally, an in-depth analysis of CAZymes revealed a rich repertoire consistent with a saprotrophic lifestyle, suggesting <em>N. moseri</em> plays a significant role in decomposing plant materials in its natural environment.</p>
<section id="Sec25"><h3 class="pmc_sec_title">Phylogeny and Taxonomic Insights</h3>
<p id="Par89">The type strain of <em>N. moseri</em> (CBS 164.80) was isolated in Colombia in 1995 and originally described as an unusual <em>Wardomyces</em> [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. In 2022, Jiang et al. created the genus <em>Neoarthrinium</em> upon isolation of several new fungi in China [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>].</p>
<p id="Par90">Importantly, <em>N. moseri</em>, <em>N. trachycarpi</em>, and <em>N. urticae</em> possess highly similar ITS, LSU, and <em>tub2</em> sequences (. 2), as already discussed by Jiang et al. and Mukhopadhyay et al. [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. However, Jiang et al. proposed keeping <em>N. moseri</em> and <em>N. trachycarpi</em> as separate species based on conidial size differences. Importantly, they compared their own measurements of <em>N. trachycarpi</em> to the values of <em>N. moseri</em> as reported by W. Gams. Our reassessment of Gams’ SEM images [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>], along with our own conidial size measurements of the CBS strain and our two new isolates, revealed inconsistencies with previously reported values. It appears the spores of <em>N. moseri</em> and <em>N. trachycarpi</em> are similar in size (Table <a href="#Tab3" class="usa-link">3</a>), suggesting that <em>N. trachycarpi</em> is not a distinct species. Further, the high ANI values among the CBS strain and our isolates (Table <a href="#MOESM2" class="usa-link">S2</a>) strongly suggest that they belong to the same species. This fact, taken together with the phylogenetic tree based on the ITS, LSU, and <em>tub2</em> sequences (Fig. <a href="#Fig2" class="usa-link">2</a>) and the analysis by Mukhopadhyay et al. [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. suggest that <em>N. moseri</em>, <em>N. trachycarpi</em>, and <em>N. urticae</em> belong to the same species.</p>
<p id="Par91">Notably, the high ANI values among the sequenced <em>N. moseri</em> strains also indicate a stable genomic architecture and strong relatedness across the strains, despite spatiotemporal differences in their isolation.</p>
<p id="Par92">The available sequences for <em>N. urticae</em> are from a single isolate from leaf litter in India (IMI 326344) but not from isolates from the type host <em>Urtica dioica</em> L. (<em>Urticaceae</em>). Jiang et al. already stated that “Additional molecular studies on verified isolates from <em>Urtica</em> collected in Europe are necessary to reveal whether IMI 326344 represents true <em>N. urticae</em>. However, <em>N. urticae</em> appears to be very rare and we are unaware of any additional collections with the exception of the type.” [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. We second this opinion. Resolving these taxonomic uncertainties will also require more comprehensive sampling, sequence datasets, including less-conserved regions or population genomic analyses.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Adaptability and Physiological Insights</h3>
<p id="Par93">Although the three sequenced <em>N. moseri</em> strains are genetically similar, they displayed distinct morphological traits under varying culture conditions (Fig. <a href="#Fig1" class="usa-link">1</a>) and slight differences regarding their growth condition tolerance (Fig. <a href="#Fig5" class="usa-link">5</a>) and carbon source utilization (Fig. <a href="#Fig6" class="usa-link">6</a>; Table <a href="#Tab7" class="usa-link">7</a>). However, all strains exhibited strong conidiation on oat medium, making it a reliable choice for conidia collection in future experiments.</p>
<p id="Par94">The BIOLOG assays revealed that all strains preferentially metabolized sugars, though with noticeable differences in biomass production. TUCIM 5827 consistently showed lower biomass accumulation across all substrates (Fig. <a href="#Fig6" class="usa-link">6</a>; Table <a href="#Tab7" class="usa-link">7</a>), correlating with its weaker growth on MEX and oat media (Fig. <a href="#Fig2" class="usa-link">2</a>). The wide range of metabolized carbon sources suggests a highly versatile and adaptive catabolic system, reinforcing the idea that <em>N. moseri</em> is a saprotroph with considerable ecological resilience.</p>
<p id="Par95">We also observed that γ-aminobutyric acid (GABA) was efficiently metabolized by CBS 164.80 and TUCIM 5827, while TUCIM 5799 did so moderately. This is particularly intriguing as plants produce GABA in response to stress, including fungal infections [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. While GABA can inhibit the growth of certain plant pathogens [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>], the ability of <em>N. moseri</em> to utilize GABA may confer an ecological advantage and may be a potential adaptation to an endophytic or epiphytic lifestyle. Further studies are needed to determine whether <em>N. moseri</em> can withstand inhibitory GABA concentrations or if it exclusively utilizes it as a nutrient.</p>
<p id="Par96">The strains also exhibited a certain degree of halotolerance, characterized by optimal growth at low NaCl concentrations and reduced growth at higher concentrations (Fig. <a href="#Fig3" class="usa-link">3</a>A). Acidotolerance was observed, with CBS 164.80 and TUCIM 5827 thriving at pH 3, while TUCIM 5799 preferred pH 5 (Fig. <a href="#Fig3" class="usa-link">3</a>B). Importantly, the strains demonstrated growth across a broad pH range including even very high pH values, indicating their ability to survive in diverse environmental conditions. Thermotolerance varied among the strains, with TUCIM 5799 being the only one able to grow at 37 °C (Fig. <a href="#Fig3" class="usa-link">3</a>C). This observation is significant given the shared habitat of TUCIM 5799 and TUCIM 5827, suggesting a localized adaptation or microevolutionary divergence. In general, <em>N. moseri</em> can be classified as a mesophilic species, with strain-specific physiological adaptations that are likely to contribute to its ecological success.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Mitochondrial genome</h3>
<p id="Par97">The mitochondrial genome of <em>N. moseri</em> contains 14 conserved protein-coding genes (Fig. <a href="#MOESM1" class="usa-link">S1</a>), as expected for fungi, with one exception: the <em>atp8</em> gene (encoding for the ATP synthase F0 subunit 8) is absent. This gene was presumably transferred to the nuclear genome, as a single gene encoding for a putative ATP synthase subunit can be found in each genome of the three strains (JN550g13373 in <em>N. moseri</em> CBS 164.80; JX265g13592 in <em>N. moseri</em> TUCIM 5799; JX266g13823 in <em>N. moseri</em> TUCIM 5827). Such gene transfers are well-documented in fungi and highlight evolutionary genomic plasticity. This phenomenon may influence mitochondrial function and warrant further investigation into its implications for energy metabolism and strain-specific adaptation.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Genome mining and metabolic potential</h3>
<p id="Par98">The extensive CAZyme repertoire in <em>N. moseri</em> (Table <a href="#Tab6" class="usa-link">6</a>) supports the notion of a saprotrophic lifestyle, and indicates metabolic flexibility, allowing <em>N. moseri</em> to utilize a variety of complex and simple carbon sources, a trait confirmed by BIOLOG analysis that suggests an oligotrophic lifestyle.</p>
<p id="Par99">Genome mining for BGCs revealed a striking potential for secondary metabolite production, consistent with its classification in the <em>Amphisphaeriales</em> order [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. <em>N. moseri</em> demonstrates a biosynthetic potential comparable to or even exceeding that of fungi known for and studied partly due to their secondary metabolism, such as <em>Aspergillus</em> and <em>Fusarium</em>. We found that most BGCs are present in all three strains, suggesting a highly conserved secondary metabolite profile, matching the high ANI. In general, secondary metabolites can be useful under certain conditions and contribute to an organism’s fitness. We found BGCs for common metabolites such as choline, the siderophore dimethylcoprogen, and the protective DHN-melanin, which contribute to the basic fitness of <em>N. moseri</em>, Additionally, <em>N. moseri</em> also possesses BGCs that are most likely responsible for the production of antimicrobial compounds, which can contribute to fitness in competitive situations. Interestingly, we also found BGCs likely to produce brassicicene C and koraiol, and the plant growth hormone gibberellin, suggesting that <em>N. moseri</em> might not only be an inert epiphyte, but might directly interact with its plant host. Importantly, for most of the predicted BGCs, we could not detect similar BGCs in the MIBIG 4.0 database, suggesting that further exploration of these BGCs could lead to the discovery of novel compounds with pharmaceutical or agricultural applications. Additionally, future studies using expanded and related taxa will be needed to resolve BGC conservation across the order <em>Amphisphaeriales</em>.</p></section></section><section id="Sec29"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par100">This study provides the first genomic and physiological characterization of <em>N. moseri</em>, revealing its exceptional biosynthetic potential and its metabolic versatility. The three sequenced strains exhibit a remarkably high number of biosynthetic gene clusters (BGCs), surpassing those of many fungi within the <em>Amphisphaeriales</em>. Genomic analysis also uncovered an extensive repertoire of carbohydrate-active enzymes (CAZymes), supporting its role as a versatile saprotroph or epiphyte capable of degrading a broad array of plant-derived substrates. Ecophysiological assays further demonstrate the species’ adaptability to diverse environmental conditions, including variations in salinity, temperature, and pH, as well as its capacity to metabolize a wide range of carbon sources.</p>
<p id="Par101">Morphological and genetic analyses challenge the current taxonomy, suggesting that <em>N. trachycarpi</em> and <em>N. urticae</em> may not be distinct from <em>N. moseri</em>. Together, these findings not only expand our understanding of the genus <em>Neoarthrinium</em> but also position <em>N. moseri</em> as a promising candidate for future biotechnological exploration and drug discovery efforts. The presence of unique, uncharacterized BGCs in the genome provides compelling motivation for further studies into the metabolic products of this species. Overall, this work lays the foundation for integrating <em>N. moseri</em> into broader studies of fungal ecology, evolution, and natural product biosynthesis.</p></section><section id="Sec30"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (1.7MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (11.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM3_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (3.8KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM4_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4.</a><sup> (11.5KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM5_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5.</a><sup> (11.7KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM6_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6.</a><sup> (11.6KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM7_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7.</a><sup> (40.8KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8.</a><sup> (13.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM9_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9.</a><sup> (40.9KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 10.</a><sup> (11.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM11_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 11.</a><sup> (41.2KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 12.</a><sup> (161.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM13"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 13.</a><sup> (13.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM14"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM14_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 14.</a><sup> (12.4MB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM15"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM15_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 15.</a><sup> (12.7MB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM16"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM16_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 16.</a><sup> (13.1MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM17"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM17_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 17.</a><sup> (12.6MB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM18"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM18_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 18.</a><sup> (22.8MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM19"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM19_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 19.</a><sup> (97.3MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We kindly thank Gerd Mauschitz and Wolfgang Ipsmiller (both TU Wien, Vienna, Austria) for providing access to the scanning electron microscope and supporting us with the measurements. The authors are thankful for the arboreal field work performed by Alexey Kopchinskiy (Austria) in a frame of WWTF-LS13-048 project with kind assistance of the staff from the Kuala Belalong Field Studies Centre (Brunei Darussalam). We thank Kamariah A. Salim and Linda B.L. Lim (Universiti Brunei Darussalam) for their contribution in taxonomic identification of the host plants.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>ANI</dt>
<dd><p id="Par4">Average Nucleotide Identity</p></dd>
<dt>BGC</dt>
<dd><p id="Par5">Biosynthetic Gene Cluster</p></dd>
<dt>BUSCO</dt>
<dd><p id="Par6">Benchmarking Universal Single-Copy Orthologs</p></dd>
<dt>CAZymes</dt>
<dd><p id="Par7">Carbohydrate-Active Enzymes</p></dd>
<dt>CE</dt>
<dd><p id="Par8">Carbohydrate Esterase</p></dd>
<dt>CYAS</dt>
<dd><p id="Par9">Czapek Yeast Autolysate Agar with Salt</p></dd>
<dt>DBCAN</dt>
<dd><p id="Par10">Database for automated Carbohydrate-active enzyme ANnotation</p></dd>
<dt>GCF</dt>
<dd><p id="Par11">Gene cluster family</p></dd>
<dt>GH</dt>
<dd><p id="Par12">Glycoside Hydrolase</p></dd>
<dt>GO</dt>
<dd><p id="Par13">Gene Ontology</p></dd>
<dt>GT</dt>
<dd><p id="Par14">Glycosyltransferase</p></dd>
<dt>HMMER</dt>
<dd><p id="Par15">Hidden Markov Model Search Tool</p></dd>
<dt>ITS</dt>
<dd><p id="Par16">Internal Transcribed Spacer</p></dd>
<dt>LSU</dt>
<dd><p id="Par17">Large Subunit Ribosomal RNA</p></dd>
<dt>MEA</dt>
<dd><p id="Par18">Malt Extract Agar</p></dd>
<dt>MEX</dt>
<dd><p id="Par19">Malt Extract Medium</p></dd>
<dt>OUT</dt>
<dd><p id="Par20">Operational Taxonomic Unit</p></dd>
<dt>PDA</dt>
<dd><p id="Par21">Potato Dextrose Agar</p></dd>
<dt>PKS</dt>
<dd><p id="Par22">Polyketide Synthase</p></dd>
<dt>PL</dt>
<dd><p id="Par23">Polysaccharide Lyase</p></dd>
<dt>RCDPS</dt>
<dd><p id="Par24">Arginine-Containing Cyclodipeptide Synthase</p></dd>
<dt>RiPP</dt>
<dd><p id="Par25">Ribosomally Synthesized and Post-Translationally Modified Peptide</p></dd>
<dt>RNA</dt>
<dd><p id="Par26">Ribonucleic Acid</p></dd>
<dt>SINE/LINE/LTR, Short/Long</dt>
<dd><p id="Par27">Interspersed Nuclear Elements/Long Terminal Repeat</p></dd>
<dt>SEM</dt>
<dd><p id="Par28">Scanning Electron Microscopy</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>NJH performed the microbiological assessment, the antiSMASH-analysis, and co-drafted the manuscriptGAV performed the genome sequencing, assembly, annotation, and co-drafted the manuscriptMS performed the phylogenetic analysisRLM was involved in the study design, supervision, and provided resourcesARMA was involved in the study design, supervision, and provided resourcesMJR isolated the two new strainsCMC provided resources and taxonomic identification of the host plantsFC was involved in taxonomic evaluationISD was involved in the study design, taxonomic evaluation and and co-drafted the manuscript CZ was involved in the study design, supervision, provided resources, performed the comparative BGC analyses, and co-drafted the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open access funding provided by TU Wien (TUW). This research was funded in whole or in part by the Austrian Science Fund (FWF) [10.55776/P 34036] and TU Wien [PhD program TU Wien bioactive]. For open access purposes, the author has applied a CC BY public copyright license to any author-accepted manuscript version arising from this submission.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The following genomic sequences are available at NCBI GenBank: Rubroshorea johorensis (Dipterocarpaceae, Malvales; DNA BarCode maturase K (matK) with accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/MF993320.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MF993320.1</a>; The mitochondrial genomes with accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/MW554918" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW554918</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/MW660808" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW660808</a>, and <a href="https://www.ncbi.nlm.nih.gov/nuccore/MW660809" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW660809</a>; the genome assemblies with accession no. GCA_022829205.1, GCA_022829195.1, and GCA_022829225.1.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par102">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par103">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par104">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Nadine J. Hochenegger and Gabriel A. Vignolle contributed equally to this work.</p></div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Hyde KD, Xu J, Rapior S, Jeewon R, Lumyong S, Niego AGT, et al. The amazing potential of fungi: 50 ways we can exploit fungi industrially. Fungal Divers. 2019;97(1):1–136.</cite> [<a href="https://scholar.google.com/scholar_lookup?Hyde%20KD,%20Xu%20J,%20Rapior%20S,%20Jeewon%20R,%20Lumyong%20S,%20Niego%20AGT,%20et%20al.%20The%20amazing%20potential%20of%20fungi:%2050%20ways%20we%20can%20exploit%20fungi%20industrially.%20Fungal%20Divers.%202019;97(1):1%E2%80%93136." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Keller NP, Turner G, Bennett JW. Fungal secondary metabolism — from biochemistry to genomics. Nat Rev Microbiol. 2005;3(12):937–47.
</cite> [<a href="https://doi.org/10.1038/nrmicro1286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16322742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keller%20NP,%20Turner%20G,%20Bennett%20JW.%20Fungal%20secondary%20metabolism%20%E2%80%94%20from%20biochemistry%20to%20genomics.%20Nat%20Rev%20Microbiol.%202005;3(12):937%E2%80%9347." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Conrado R, Gomes TC, Roque GSC, De Souza AO. Overview of bioactive fungal secondary metabolites: cytotoxic and antimicrobial compounds. Antibiotics. 2022;11(11):1604.
</cite> [<a href="https://doi.org/10.3390/antibiotics11111604" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9687038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36421247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Conrado%20R,%20Gomes%20TC,%20Roque%20GSC,%20De%20Souza%20AO.%20Overview%20of%20bioactive%20fungal%20secondary%20metabolites:%20cytotoxic%20and%20antimicrobial%20compounds.%20Antibiotics.%202022;11(11):1604." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Schüller A, Studt-Reinhold L, Strauss J. How to completely squeeze a fungus—advanced genome mining tools for novel bioactive substances. Pharmaceutics. 2022;14(9): 1837.
</cite> [<a href="https://doi.org/10.3390/pharmaceutics14091837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9505985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36145585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sch%C3%BCller%20A,%20Studt-Reinhold%20L,%20Strauss%20J.%20How%20to%20completely%20squeeze%20a%20fungus%E2%80%94advanced%20genome%20mining%20tools%20for%20novel%20bioactive%20substances.%20Pharmaceutics.%202022;14(9):%201837." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Xue M, Hou X, Fu J, Zhang J, Wang J, Zhao Z, et al. Recent advances in search of bioactive secondary metabolites from fungi triggered by chemical epigenetic modifiers. J Fungi. 2023;9(2): 172.</cite> [<a href="https://doi.org/10.3390/jof9020172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9961798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36836287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xue%20M,%20Hou%20X,%20Fu%20J,%20Zhang%20J,%20Wang%20J,%20Zhao%20Z,%20et%20al.%20Recent%20advances%20in%20search%20of%20bioactive%20secondary%20metabolites%20from%20fungi%20triggered%20by%20chemical%20epigenetic%20modifiers.%20J%20Fungi.%202023;9(2):%20172." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Shabana S, Lakshmi KR, Satya AK. An updated review of secondary metabolites from marine fungi. Mini Rev Med Chem. 2021;21(5):602–42.
</cite> [<a href="https://doi.org/10.2174/1389557520666200925142514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32981503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shabana%20S,%20Lakshmi%20KR,%20Satya%20AK.%20An%20updated%20review%20of%20secondary%20metabolites%20from%20marine%20fungi.%20Mini%20Rev%20Med%20Chem.%202021;21(5):602%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Becker K, Stadler M. Recent progress in biodiversity research on the <em>Xylariales</em> and their secondary metabolism. J Antibiot (Tokyo). 2021;74(1):1–23.
</cite> [<a href="https://doi.org/10.1038/s41429-020-00376-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7732752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33097836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Becker%20K,%20Stadler%20M.%20Recent%20progress%20in%20biodiversity%20research%20on%20the%20Xylariales%20and%20their%20secondary%20metabolism.%20J%20Antibiot%20(Tokyo).%202021;74(1):1%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Verekar SA, Gupta MK, Deshmukh SK. Discovery of Bioactive Metabolites from the Genus <em>Arthrinium</em>. In: Satyanarayana T, Deshmukh SK, editors. Fungi and Fungal Products in Human Welfare and Biotechnology. Singapore: Springer Nature; 2023 [cited 2025 Jan 24]. pp. 257–87. Available from: 10.1007/978-981-19-8853-0_9</cite>
</li>
<li id="CR9">
<span class="label">9.</span><cite>Overgaard ML, Aalborg T, Zeuner EJ, Westphal KR, Lau FA, Nielsen VS, et al. Quick guide to secondary metabolites from <em>Apiospora</em> and <em>Arthrinium</em>. Fungal Biol Rev. 2023;43: 100288.</cite> [<a href="https://scholar.google.com/scholar_lookup?Overgaard%20ML,%20Aalborg%20T,%20Zeuner%20EJ,%20Westphal%20KR,%20Lau%20FA,%20Nielsen%20VS,%20et%20al.%20Quick%20guide%20to%20secondary%20metabolites%20from%20Apiospora%20and%20Arthrinium.%20Fungal%20Biol%20Rev.%202023;43:%20100288." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Jiang N, Voglmayr H, Ma CY, Xue H, Piao CG, Li Y. A new <em>Arthrinium</em>-like genus of <em>Amphisphaeriales</em> in China. MycoKeys. 2022;92:27–43.
</cite> [<a href="https://doi.org/10.3897/mycokeys.92.86521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9849059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36761316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20N,%20Voglmayr%20H,%20Ma%20CY,%20Xue%20H,%20Piao%20CG,%20Li%20Y.%20A%20new%20Arthrinium-like%20genus%20of%20Amphisphaeriales%20in%20China.%20MycoKeys.%202022;92:27%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Gams W. An unusual species of <em>Wardomyces</em> (<em>Hyphomycetes</em>). Sydowia. 1995 [cited 2025 Jan 24]; Available from: <a href="https://www.semanticscholar.org/paper/An-unusual-species-of-Wardomyces-(Hyphomycetes).-Gams/b38e303e464efe266c4739d2624a5e921fe50f90" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.semanticscholar.org/paper/An-unusual-species-of-Wardomyces-(Hyphomycetes).-Gams/b38e303e464efe266c4739d2624a5e921fe50f90</a></cite>
</li>
<li id="CR12">
<span class="label">12.</span><cite>Deyse Viana dos S. Neoarthrinium brasiliense, sp. nov. (Apiosporaceae, Sordariomycetes), a new endophytic fungal species from Lafoensia pacari tree. Sydowia. 2024;76:179–85.</cite>
</li>
<li id="CR13">
<span class="label">13.</span><cite>Tan YP, Bishop-Hurley SL, Marney TS, Shivas RG. Index of Australian Fungi no. 53. 2025 Mar 25 [cited 2025 Apr 3]; Available from: <a href="https://zenodo.org/records/15080534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/15080534</a></cite>
</li>
<li id="CR14">
<span class="label">14.</span><cite>Mukhopadhyay S, Phukhamsakda C, Bhunjun CS, Bahkali AH, Jones EBG, Hyde KD. Exploring fungi on Mangrove associates: novel family <em>Neoarthriniaceae</em> and two novel species in <em>Neoarthrinium</em> and <em>Oncopodiella</em>. Mycol Progress. 2025;24(1): 18.</cite> [<a href="https://scholar.google.com/scholar_lookup?Mukhopadhyay%20S,%20Phukhamsakda%20C,%20Bhunjun%20CS,%20Bahkali%20AH,%20Jones%20EBG,%20Hyde%20KD.%20Exploring%20fungi%20on%20Mangrove%20associates:%20novel%20family%20Neoarthriniaceae%20and%20two%20novel%20species%20in%20Neoarthrinium%20and%20Oncopodiella.%20Mycol%20Progress.%202025;24(1):%2018." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Yan H, Jiang N, Liang LY, Yang Q, Tian CM. <em>Arthrinium trachycarpum</em> sp. <em>nov</em>. from <em>Trachycarpus Fortunei</em> in China. Phytotaxa. 2019;400(3):203–10.</cite> [<a href="https://scholar.google.com/scholar_lookup?Yan%20H,%20Jiang%20N,%20Liang%20LY,%20Yang%20Q,%20Tian%20CM.%20Arthrinium%20trachycarpum%20sp.%20nov.%20from%20Trachycarpus%20Fortunei%20in%20China.%20Phytotaxa.%202019;400(3):203%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Minter DW, Cannon PF. Arthrinium urticae. [Descriptions of Fungi and Bacteria]. Descriptions of Fungi and Bacteria. 2018;2160. </cite>
</li>
<li id="CR17">
<span class="label">17.</span><cite>Laciny A, Zettel H, Kopchinskiy A, Pretzer C, Pal A, Salim KA et al. <em>Colobopsis explodens</em> sp. n., model species for studies on exploding ants (<em>Hymenoptera</em>, <em>Formicidae</em>), with biological notes and first illustrations of males of the <em>Colobopsis cylindrica</em> group. ZooKeys. 2018;(751):1–40.</cite> [<a href="https://doi.org/10.3897/zookeys.751.22661" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5919914/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29706783/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Team R. core. R: A language and environment for statistical computing. 2019.<a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R-project.org/</a>.</cite>
</li>
<li id="CR19">
<span class="label">19.</span><cite>Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS, et al. 2012 SPAdes: a new genome assembly algorithm and its applications to single-cell sequencing. J Comput Biol J Comput Mol Cell Biol. 19(5):455–77.</cite> [<a href="https://doi.org/10.1089/cmb.2012.0021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3342519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22506599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bankevich%20A,%20Nurk%20S,%20Antipov%20D,%20Gurevich%20AA,%20Dvorkin%20M,%20Kulikov%20AS,%20et%20al.%C2%A02012%20SPAdes:%20a%20new%20genome%20assembly%20algorithm%20and%20its%20applications%20to%20single-cell%20sequencing.%20J%20Comput%20Biol%20J%20Comput%20Mol%20Cell%20Biol.%2019(5):455%E2%80%9377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Johnson M, Zaretskaya I, Raytselis Y, Merezhuk Y, McGinnis S, Madden TL. NCBI BLAST: a better web interface. Nucleic Acids Res. 2008;36(Web Server issue):W5-9.
</cite> [<a href="https://doi.org/10.1093/nar/gkn201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2447716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18440982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johnson%20M,%20Zaretskaya%20I,%20Raytselis%20Y,%20Merezhuk%20Y,%20McGinnis%20S,%20Madden%20TL.%20NCBI%20BLAST:%20a%20better%20web%20interface.%20Nucleic%20Acids%20Res.%202008;36(Web%20Server%20issue):W5-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Dierckxsens N, Mardulyn P, Smits G. NOVOPlasty: de Novo assembly of organelle genomes from whole genome data. Nucleic Acids Res. 2016;45(4):e18–18.</cite> [<a href="https://doi.org/10.1093/nar/gkw955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5389512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28204566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dierckxsens%20N,%20Mardulyn%20P,%20Smits%20G.%20NOVOPlasty:%20de%20Novo%20assembly%20of%20organelle%20genomes%20from%20whole%20genome%20data.%20Nucleic%20Acids%20Res.%202016;45(4):e18%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Grant JR, Stothard P. The CGView Server: a comparative genomics tool for circular genomes. Nucleic Acids Res. 2008;36(suppl_2):W181-4. </cite> [<a href="https://doi.org/10.1093/nar/gkn179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2447734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18411202/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Langmead B, Trapnell C, Pop M, Salzberg SL. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol. 2009;10(3):R25.
</cite> [<a href="https://doi.org/10.1186/gb-2009-10-3-r25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2690996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19261174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead%20B,%20Trapnell%20C,%20Pop%20M,%20Salzberg%20SL.%20Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome.%20Genome%20Biol.%202009;10(3):R25." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Edwards JA, Edwards RA. Fastq-pair: efficient synchronization of paired-end fastq files. bioRxiv; 2019;552885. 10.1101/552885.</cite>
</li>
<li id="CR25">
<span class="label">25.</span><cite>Trimmomatic. a flexible trimmer for Illumina sequence data | Bioinformatics | Oxford Academic [Internet]. [cited 2025 Jan 24]. Available from: <a href="https://academic.oup.com/bioinformatics/article/30/15/2114/2390096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/bioinformatics/article/30/15/2114/2390096</a></cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Boetzer M, Pirovano W. Toward almost closed genomes with gapfiller. Genome Biol. 2012;13(6):R56.
</cite> [<a href="https://doi.org/10.1186/gb-2012-13-6-r56" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3446322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22731987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boetzer%20M,%20Pirovano%20W.%20Toward%20almost%20closed%20genomes%20with%20gapfiller.%20Genome%20Biol.%202012;13(6):R56." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Walker BJ, Abeel T, Shea T, Priest M, Abouelliel A, Sakthikumar S, et al. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. PLoS One. 2014;9(11):e112963.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0112963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4237348/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25409509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Walker%20BJ,%20Abeel%20T,%20Shea%20T,%20Priest%20M,%20Abouelliel%20A,%20Sakthikumar%20S,%20et%20al.%20Pilon:%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement.%20PLoS%20One.%202014;9(11):e112963." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Li H, Durbin R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics. 2009;25(14):1754–60.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2705234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H,%20Durbin%20R.%20Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform.%20Bioinformatics.%202009;25(14):1754%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. 2009 The Sequence Alignment/Map format and SAMtools. Bioinformatics. 25(16):2078–9.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2723002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H,%20Handsaker%20B,%20Wysoker%20A,%20Fennell%20T,%20Ruan%20J,%20Homer%20N,%20et%20al.%C2%A02009%20The%20Sequence%20Alignment/Map%20format%20and%20SAMtools.%20Bioinformatics.%2025(16):2078%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Picard Tools -. By Broad Institute [Internet]. [cited 2025 Jan 24]. Available from: <a href="https://broadinstitute.github.io/picard/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://broadinstitute.github.io/picard/</a></cite>
</li>
<li id="CR31">
<span class="label">31.</span><cite>MAFFT. a novel method for rapid multiple sequence alignment based on fast Fourier transform | Nucleic Acids Research | Oxford Academic [Internet]. [cited 2025 Jan 24]. Available from: <a href="https://academic.oup.com/nar/article/30/14/3059/2904316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/nar/article/30/14/3059/2904316</a></cite> [<a href="https://doi.org/10.1093/nar/gkf436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC135756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12136088/" class="usa-link">PubMed</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>MEGA11. Molecular Evolutionary Genetics Analysis Version 11 | Molecular Biology and Evolution | Oxford Academic [Internet]. [cited 2025 Jan 24]. Available from: <a href="https://academic.oup.com/mbe/article/38/7/3022/6248099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/mbe/article/38/7/3022/6248099</a></cite> [<a href="https://doi.org/10.1093/molbev/msab120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8233496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33892491/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>IQ-TREE 2. New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era | Molecular Biology and Evolution | Oxford Academic [Internet]. [cited 2025 Jan 24]. Available from: <a href="https://academic.oup.com/mbe/article/37/5/1530/5721363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/mbe/article/37/5/1530/5721363</a></cite>
</li>
<li id="CR34">
<span class="label">34.</span><cite>FigTree [Internet]. [cited 2025 Jan 24]. Available from: <a href="http://tree.bio.ed.ac.uk/software/figtree/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://tree.bio.ed.ac.uk/software/figtree/</a></cite>
</li>
<li id="CR35">
<span class="label">35.</span><cite>Smit A, Hubley R, Green P. RepeatMasker Open-4.0. 2013-2015 . <a href="http://www.repeatmasker.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.repeatmasker.org</a>.</cite>
</li>
<li id="CR36">
<span class="label">36.</span><cite> Lowe TM, Eddy SR. tRNAscan-SE: A Program for Improved Detection of Transfer RNA Genes in Genomic Sequence. Nucleic Acids Res. 1997;25(5):955–64. 10.1093/nar/25.5.955.</cite> [<a href="https://doi.org/10.1093/nar/25.5.955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC146525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9023104/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Stanke M, Keller O, Gunduz I, Hayes A, Waack S, Morgenstern B. AUGUSTUS: ab initio prediction of alternative transcripts. Nucleic Acids Res. 2006;34:W435-9.
</cite> [<a href="https://doi.org/10.1093/nar/gkl200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1538822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16845043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stanke%20M,%20Keller%20O,%20Gunduz%20I,%20Hayes%20A,%20Waack%20S,%20Morgenstern%20B.%20AUGUSTUS:%20ab%20initio%20prediction%20of%20alternative%20transcripts.%20Nucleic%20Acids%20Res.%202006;34:W435-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Hoff KJ, Stanke M. Predicting genes in single genomes with AUGUSTUS. Curr Protoc Bioinformatics. 2019;65(1): e57.
</cite> [<a href="https://doi.org/10.1002/cpbi.57" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30466165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoff%20KJ,%20Stanke%20M.%20Predicting%20genes%20in%20single%20genomes%20with%20AUGUSTUS.%20Curr%20Protoc%20Bioinformatics.%202019;65(1):%20e57." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>QUAST. quality assessment tool for genome assemblies | Bioinformatics | Oxford Academic [Internet]. [cited 2025 Jan 24]. Available from: <a href="https://academic.oup.com/bioinformatics/article/29/8/1072/228832" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/bioinformatics/article/29/8/1072/228832</a></cite> [<a href="https://doi.org/10.1093/bioinformatics/btt086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3624806/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>UniProt. a worldwide hub of protein knowledge | Nucleic Acids Research | Oxford Academic [Internet]. [cited 2025 Jan 24]. Available from: <a href="https://academic.oup.com/nar/article/47/D1/D506/5160987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/nar/article/47/D1/D506/5160987</a></cite> [<a href="https://doi.org/10.1093/nar/gky1049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30395287/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Törönen P, Medlar A, Holm L. PANNZER2: a rapid functional annotation web server. Nucleic Acids Res. 2018;46(W1):W84-8.
</cite> [<a href="https://doi.org/10.1093/nar/gky350" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6031051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29741643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?T%C3%B6r%C3%B6nen%20P,%20Medlar%20A,%20Holm%20L.%20PANNZER2:%20a%20rapid%20functional%20annotation%20web%20server.%20Nucleic%20Acids%20Res.%202018;46(W1):W84-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>dbCAN2. a meta server for automated carbohydrate-active enzyme annotation | Nucleic Acids Research | Oxford Academic [Internet]. [cited 2025 Jan 24]. Available from: <a href="https://academic.oup.com/nar/article/46/W1/W95/4996582?login=true" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/nar/article/46/W1/W95/4996582?login=true</a></cite> [<a href="https://doi.org/10.1093/nar/gky418" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6031026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29771380/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Finn RD, Clements J, Eddy SR. HMMER web server: interactive sequence similarity searching. Nucleic Acids Res 05/18 ed. 2011;39:W29-37.</cite> [<a href="https://doi.org/10.1093/nar/gkr367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3125773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21593126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Finn%20RD,%20Clements%20J,%20Eddy%20SR.%20HMMER%20web%20server:%20interactive%20sequence%20similarity%20searching.%20Nucleic%20Acids%20Res%2005/18%20ed.%202011;39:W29-37." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Buchfink B, Xie C, Huson DH. Fast and sensitive protein alignment using DIAMOND. Nat Methods. 2015;12(1):59–60.
</cite> [<a href="https://doi.org/10.1038/nmeth.3176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25402007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buchfink%20B,%20Xie%20C,%20Huson%20DH.%20Fast%20and%20sensitive%20protein%20alignment%20using%20DIAMOND.%20Nat%20Methods.%202015;12(1):59%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Busk PK, Pilgaard B, Lezyk MJ, Meyer AS, Lange L. Homology to peptide pattern for annotation of carbohydrate-active enzymes and prediction of function. BMC Bioinformatics. 2017;18(1):214.
</cite> [<a href="https://doi.org/10.1186/s12859-017-1625-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5389127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28403817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Busk%20PK,%20Pilgaard%20B,%20Lezyk%20MJ,%20Meyer%20AS,%20Lange%20L.%20Homology%20to%20peptide%20pattern%20for%20annotation%20of%20carbohydrate-active%20enzymes%20and%20prediction%20of%20function.%20BMC%20Bioinformatics.%202017;18(1):214." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Blin K, Shaw S, Augustijn HE, Reitz ZL, Biermann F, Alanjary M et al. antiSMASH 7.0: new and improved predictions for detection, regulation, chemical structures and visualisation. Nucleic Acids Res. 2023;51(W1):W46–50. <a href="https://academic.oup.com/nar/article/51/W1/W46/7151336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/nar/article/51/W1/W46/7151336</a>.</cite> [<a href="https://doi.org/10.1093/nar/gkad344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37140036/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Kautsar SA, Biermann F, Louwers J, de Hollander M, Sahu P, Somerville W. medema-group/bigslice: Version 2.0.0. 2024. <a href="https://zenodo.org/records/10783493" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/10783493</a>.</cite>
</li>
<li id="CR48">
<span class="label">48.</span><cite> Kautsar SA, van der Hooft JJJ, de Ridder D, Medema MH.BiG-SLiCE. A highly scalable tool maps the diversity of 1.2 million biosynthetic gene clusters. Gigascience. 2021;10(1):giaa154. <a href="https://academic.oup.com/gigascience/article/10/1/giaa154/6092777?" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/gigascience/article/10/1/giaa154/6092777?</a>.</cite> [<a href="https://doi.org/10.1093/gigascience/giaa154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7804863/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33438731/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Navarro-Muñoz JC, Selem-Mojica N, Mullowney MW, Kautsar SA, Tryon JH, Parkinson EI, et al. A computational framework to explore large-scale biosynthetic diversity. Nat Chem Biol. 2020;16(1):60–8.
</cite> [<a href="https://doi.org/10.1038/s41589-019-0400-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6917865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31768033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Navarro-Mu%C3%B1oz%20JC,%20Selem-Mojica%20N,%20Mullowney%20MW,%20Kautsar%20SA,%20Tryon%20JH,%20Parkinson%20EI,%20et%20al.%20A%20computational%20framework%20to%20explore%20large-scale%20biosynthetic%20diversity.%20Nat%20Chem%20Biol.%202020;16(1):60%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Mistry J, Chuguransky S, Williams L, Qureshi M, Salazar GA, Sonnhammer ELL, et al. Pfam: the protein families database in 2021. Nucleic Acids Res. 2021;49(D1):D412–9.
</cite> [<a href="https://doi.org/10.1093/nar/gkaa913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7779014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33125078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mistry%20J,%20Chuguransky%20S,%20Williams%20L,%20Qureshi%20M,%20Salazar%20GA,%20Sonnhammer%20ELL,%20et%20al.%20Pfam:%20the%20protein%20families%20database%20in%202021.%20Nucleic%20Acids%20Res.%202021;49(D1):D412%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Simão FA, Waterhouse RM, Ioannidis P, Kriventseva EV, Zdobnov EM. BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs. Bioinformatics. 2015;31(19):3210–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26059717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sim%C3%A3o%20FA,%20Waterhouse%20RM,%20Ioannidis%20P,%20Kriventseva%20EV,%20Zdobnov%20EM.%20BUSCO:%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs.%20Bioinformatics.%202015;31(19):3210%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>van den Brink J, de Vries RP. Fungal enzyme sets for plant polysaccharide degradation. Appl Microbiol Biotechnol. 2011;91(6):1477–92.
</cite> [<a href="https://doi.org/10.1007/s00253-011-3473-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3160556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21785931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20den%20Brink%20J,%20de%20Vries%20RP.%20Fungal%20enzyme%20sets%20for%20plant%20polysaccharide%20degradation.%20Appl%20Microbiol%20Biotechnol.%202011;91(6):1477%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Sanchez JF, Somoza AD, Keller NP, Wang CCC. Advances in <em>Aspergillus</em> secondary metabolite research in the post-genomic era. Nat Prod Rep. 2012;29(3):351–71.
</cite> [<a href="https://doi.org/10.1039/c2np00084a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4568942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22228366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanchez%20JF,%20Somoza%20AD,%20Keller%20NP,%20Wang%20CCC.%20Advances%20in%20Aspergillus%20secondary%20metabolite%20research%20in%20the%20post-genomic%20era.%20Nat%20Prod%20Rep.%202012;29(3):351%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Ibrahim SRM, Sirwi A, Eid BG, Mohamed SGA, Mohamed GA. Bright side of <em>Fusarium oxysporum</em>: secondary metabolites bioactivities and industrial relevance in biotechnology and nanotechnology. J Fungi. 2021;7(11): 943.</cite> [<a href="https://doi.org/10.3390/jof7110943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8625159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34829230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ibrahim%20SRM,%20Sirwi%20A,%20Eid%20BG,%20Mohamed%20SGA,%20Mohamed%20GA.%20Bright%20side%20of%20Fusarium%20oxysporum:%20secondary%20metabolites%20bioactivities%20and%20industrial%20relevance%20in%20biotechnology%20and%20nanotechnology.%20J%20Fungi.%202021;7(11):%20943." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>MIBiG 4. 0: advancing biosynthetic gene cluster curation through global collaboration | Nucleic Acids Research | Oxford Academic [Internet]. [cited 2025 Mar 11]. Available from: <a href="https://academic.oup.com/nar/article/53/D1/D678/7919508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/nar/article/53/D1/D678/7919508</a></cite> [<a href="https://doi.org/10.1093/nar/gkae1115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701617/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39657789/" class="usa-link">PubMed</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>van den Belt M, Gilchrist C, Booth TJ, Chooi YH, Medema MH, Alanjary M. Cagecat: the comparative gene cluster analysis toolbox for rapid search and visualisation of homologous gene clusters. BMC Bioinformatics. 2023;24(1):181.
</cite> [<a href="https://doi.org/10.1186/s12859-023-05311-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10155394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37131131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20den%20Belt%20M,%20Gilchrist%20C,%20Booth%20TJ,%20Chooi%20YH,%20Medema%20MH,%20Alanjary%20M.%20Cagecat:%20the%20comparative%20gene%20cluster%20analysis%20toolbox%20for%20rapid%20search%20and%20visualisation%20of%20homologous%20gene%20clusters.%20BMC%20Bioinformatics.%202023;24(1):181." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Atanasova L, Druzhinina IS. Global nutrient profiling by phenotype microarrays: a tool complementing genomic and proteomic studies in conidial fungi. J Zhejiang Univ Sci B. 2010;11(3):151–68.
</cite> [<a href="https://doi.org/10.1631/jzus.B1000007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2833400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20205302/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Atanasova%20L,%20Druzhinina%20IS.%20Global%20nutrient%20profiling%20by%20phenotype%20microarrays:%20a%20tool%20complementing%20genomic%20and%20proteomic%20studies%20in%20conidial%20fungi.%20J%20Zhejiang%20Univ%20Sci%20B.%202010;11(3):151%E2%80%9368." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Bouché N, Fromm H. GABA in plants: just a metabolite? Trends Plant Sci. 2004;9(3):110–5.
</cite> [<a href="https://doi.org/10.1016/j.tplants.2004.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15003233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bouch%C3%A9%20N,%20Fromm%20H.%20GABA%20in%20plants:%20just%20a%20metabolite?%20Trends%20Plant%20Sci.%202004;9(3):110%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Mead O, Thynne E, Winterberg B, Solomon PS. Characterising the role of GABA and its metabolism in the wheat pathogen <em>Stagonospora nodorum</em>. PLoS One. 2013;8(11):e78368.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0078368" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3827059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24265684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mead%20O,%20Thynne%20E,%20Winterberg%20B,%20Solomon%20PS.%20Characterising%20the%20role%20of%20GABA%20and%20its%20metabolism%20in%20the%20wheat%20pathogen%20Stagonospora%20nodorum.%20PLoS%20One.%202013;8(11):e78368." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Yu S, Zhen C, Zhao P, Li J, Qin Z, Gao H. Antifungal mechanisms of γ-aminobutyric acid against the postharvest pathogen <em>Alternaria alternata</em>. LWT. 2023;173:114314.</cite> [<a href="https://scholar.google.com/scholar_lookup?Yu%20S,%20Zhen%20C,%20Zhao%20P,%20Li%20J,%20Qin%20Z,%20Gao%20H.%20Antifungal%20mechanisms%20of%20%CE%B3-aminobutyric%20acid%20against%20the%20postharvest%20pathogen%20Alternaria%20alternata.%20LWT.%202023;173:114314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Salvatore MM, Félix C, Lima F, Ferreira V, Duarte AS, Salvatore F, et al. Effect of γ-aminobutyric acid (GABA) on the metabolome of two strains of <em>Lasiodiplodia theobromae</em> isolated from grapevine. Molecules. 2020;25(17): 3833.
</cite> [<a href="https://doi.org/10.3390/molecules25173833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7503889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32842542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Salvatore%20MM,%20F%C3%A9lix%20C,%20Lima%20F,%20Ferreira%20V,%20Duarte%20AS,%20Salvatore%20F,%20et%20al.%20Effect%20of%20%CE%B3-aminobutyric%20acid%20(GABA)%20on%20the%20metabolome%20of%20two%20strains%20of%20Lasiodiplodia%20theobromae%20isolated%20from%20grapevine.%20Molecules.%202020;25(17):%203833." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (1.7MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (11.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM3_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (3.8KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM4_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4.</a><sup> (11.5KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM5_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5.</a><sup> (11.7KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM6_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6.</a><sup> (11.6KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM7_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7.</a><sup> (40.8KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8.</a><sup> (13.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM9_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9.</a><sup> (40.9KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 10.</a><sup> (11.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM11_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 11.</a><sup> (41.2KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM12_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 12.</a><sup> (161.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 13.</a><sup> (13.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material14_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM14_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 14.</a><sup> (12.4MB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material15_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM15_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 15.</a><sup> (12.7MB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material16_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM16_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 16.</a><sup> (13.1MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material17_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM17_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 17.</a><sup> (12.6MB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material18_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM18_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 18.</a><sup> (22.8MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material19_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369201/bin/12866_2025_4274_MOESM19_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 19.</a><sup> (97.3MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The following genomic sequences are available at NCBI GenBank: Rubroshorea johorensis (Dipterocarpaceae, Malvales; DNA BarCode maturase K (matK) with accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/MF993320.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MF993320.1</a>; The mitochondrial genomes with accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/MW554918" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW554918</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/MW660808" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW660808</a>, and <a href="https://www.ncbi.nlm.nih.gov/nuccore/MW660809" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MW660809</a>; the genome assemblies with accession no. GCA_022829205.1, GCA_022829195.1, and GCA_022829225.1.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Microbiology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12866-025-04274-z"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12866_2025_Article_4274.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12369201/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12369201/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369201%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369201/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12369201/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12369201/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841932/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12369201/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841932/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12369201/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12369201/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="4SVdkXzMVKTZ0Mr0OoQFRdEMBSVKDq0FpgmJMKp6Uy5xceqb7VlvOtWhxcQfYOAr">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
