
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Arabidopsis MACP2 contributes to autophagy induction by modulating starvation-induced reactive oxygen species homeostasis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152546AB8AF32E130546AB0029865B2C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="advbio">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367599/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Advanced Biotechnology">
<meta name="citation_title" content="Arabidopsis MACP2 contributes to autophagy induction by modulating starvation-induced reactive oxygen species homeostasis">
<meta name="citation_author" content="Ying Zhou">
<meta name="citation_author_institution" content="State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China">
<meta name="citation_author" content="Xue Zhang">
<meta name="citation_author_institution" content="State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China">
<meta name="citation_author" content="Tiancong Qi">
<meta name="citation_author_institution" content="Tsinghua-Peking Center for Life Sciences, MOE Key Laboratory of Bioinformatics, School of Life Sciences, Tsinghua University, Beijing, 100084 China">
<meta name="citation_author" content="Zi-Han Wang">
<meta name="citation_author_institution" content="College of Life Science and Technology, Central South University of Forestry and Technology, Changsha, 410004 China">
<meta name="citation_author" content="Yao Wang">
<meta name="citation_author_institution" content="State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China">
<meta name="citation_author" content="Lin-Na Wang">
<meta name="citation_author_institution" content="State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China">
<meta name="citation_author" content="Yong-Lun Zeng">
<meta name="citation_author_institution" content="School of Life Sciences, Centre for Cell &amp; Developmental Biology, State Key Laboratory of Agrobiotechnology, The Chinese University of Hong Kong, New Territories, Shatin, Hong Kong, China">
<meta name="citation_author" content="Hanjie He">
<meta name="citation_author_institution" content="College of Life Science and Technology, Central South University of Forestry and Technology, Changsha, 410004 China">
<meta name="citation_author" content="Liwen Jiang">
<meta name="citation_author_institution" content="School of Life Sciences, Centre for Cell &amp; Developmental Biology, State Key Laboratory of Agrobiotechnology, The Chinese University of Hong Kong, New Territories, Shatin, Hong Kong, China">
<meta name="citation_author" content="Daoxin Xie">
<meta name="citation_author_institution" content="Tsinghua-Peking Center for Life Sciences, MOE Key Laboratory of Bioinformatics, School of Life Sciences, Tsinghua University, Beijing, 100084 China">
<meta name="citation_author" content="Shi Xiao">
<meta name="citation_author_institution" content="State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China">
<meta name="citation_author" content="Lu-Jun Yu">
<meta name="citation_author_institution" content="State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China">
<meta name="citation_author" content="Qin-Fang Chen">
<meta name="citation_author_institution" content="State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="3">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="25">
<meta name="citation_doi" content="10.1007/s44307-025-00078-4">
<meta name="citation_pmid" content="40833484">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367599/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367599/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367599/pdf/44307_2025_Article_78.pdf">
<meta name="description" content="In plants, autophagy is a conserved recycling system essential for development and stress responses by targeting cellular components for massive degradation in the vacuole. Our previous work suggested that autophagy contributes to Arabidopsis ...">
<meta name="og:title" content="Arabidopsis MACP2 contributes to autophagy induction by modulating starvation-induced reactive oxygen species homeostasis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="In plants, autophagy is a conserved recycling system essential for development and stress responses by targeting cellular components for massive degradation in the vacuole. Our previous work suggested that autophagy contributes to Arabidopsis ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367599/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12367599">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1007/s44307-025-00078-4"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/44307_2025_Article_78.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367599%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12367599/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12367599/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367599/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-advbio.png" alt="Advanced Biotechnology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Advanced Biotechnology" title="Link to Advanced Biotechnology" shape="default" href="https://doi.org/10.1007/s44307-025-00078-4" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Adv Biotechnol (Singap)</button></div>. 2025 Aug 20;3(3):25. doi: <a href="https://doi.org/10.1007/s44307-025-00078-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1007/s44307-025-00078-4</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Adv%20Biotechnol%20(Singap)%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Adv%20Biotechnol%20(Singap)%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Adv%20Biotechnol%20(Singap)%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Adv%20Biotechnol%20(Singap)%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Arabidopsis MACP2 contributes to autophagy induction by modulating starvation-induced reactive oxygen species homeostasis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ying Zhou</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ying Zhou</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Ying Zhou</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Xue Zhang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Xue Zhang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xue Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qi%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Tiancong Qi</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Tiancong Qi</span></h3>
<div class="p">
<sup>2</sup>Tsinghua-Peking Center for Life Sciences, MOE Key Laboratory of Bioinformatics, School of Life Sciences, Tsinghua University, Beijing, 100084 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qi%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tiancong Qi</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20ZH%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Zi-Han Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Zi-Han Wang</span></h3>
<div class="p">
<sup>3</sup>College of Life Science and Technology, Central South University of Forestry and Technology, Changsha, 410004 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20ZH%22%5BAuthor%5D" class="usa-link"><span class="name western">Zi-Han Wang</span></a>
</div>
</div>
<sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yao Wang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yao Wang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yao Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20LN%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Lin-Na Wang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Lin-Na Wang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20LN%22%5BAuthor%5D" class="usa-link"><span class="name western">Lin-Na Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20YL%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Yong-Lun Zeng</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Yong-Lun Zeng</span></h3>
<div class="p">
<sup>4</sup>School of Life Sciences, Centre for Cell &amp; Developmental Biology, State Key Laboratory of Agrobiotechnology, The Chinese University of Hong Kong, New Territories, Shatin, Hong Kong, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20YL%22%5BAuthor%5D" class="usa-link"><span class="name western">Yong-Lun Zeng</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Hanjie He</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Hanjie He</span></h3>
<div class="p">
<sup>3</sup>College of Life Science and Technology, Central South University of Forestry and Technology, Changsha, 410004 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hanjie He</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Liwen Jiang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Liwen Jiang</span></h3>
<div class="p">
<sup>4</sup>School of Life Sciences, Centre for Cell &amp; Developmental Biology, State Key Laboratory of Agrobiotechnology, The Chinese University of Hong Kong, New Territories, Shatin, Hong Kong, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Liwen Jiang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Daoxin Xie</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Daoxin Xie</span></h3>
<div class="p">
<sup>2</sup>Tsinghua-Peking Center for Life Sciences, MOE Key Laboratory of Bioinformatics, School of Life Sciences, Tsinghua University, Beijing, 100084 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xie%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daoxin Xie</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiao%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Shi Xiao</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Shi Xiao</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiao%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shi Xiao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20LJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Lu-Jun Yu</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Lu-Jun Yu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20LJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Lu-Jun Yu</span></a>
</div>
</div>
<sup>1,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20QF%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Qin-Fang Chen</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Qin-Fang Chen</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20QF%22%5BAuthor%5D" class="usa-link"><span class="name western">Qin-Fang Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>State Key Laboratory of Biocontrol, Guangdong Provincial Key Laboratory of Plant Stress Biology, Innovation Center for Evolutionary Synthetic Biology, School of Life Sciences, Sun Yat-Sen University, Guangzhou, 510275 China </div>
<div id="Aff2">
<sup>2</sup>Tsinghua-Peking Center for Life Sciences, MOE Key Laboratory of Bioinformatics, School of Life Sciences, Tsinghua University, Beijing, 100084 China </div>
<div id="Aff3">
<sup>3</sup>College of Life Science and Technology, Central South University of Forestry and Technology, Changsha, 410004 China </div>
<div id="Aff4">
<sup>4</sup>School of Life Sciences, Centre for Cell &amp; Developmental Biology, State Key Laboratory of Agrobiotechnology, The Chinese University of Hong Kong, New Territories, Shatin, Hong Kong, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 3; Revised 2025 Jul 18; Accepted 2025 Jul 24; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12367599Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40833484/" class="usa-link">40833484</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">In plants, autophagy is a conserved recycling system essential for development and stress responses by targeting cellular components for massive degradation in the vacuole. Our previous work suggested that autophagy contributes to Arabidopsis (<em>Arabidopsis thaliana</em>) stress responses by modulating NADPH-oxidase-mediated reactive oxygen species (ROS) homeostasis; however, the molecular link between extracellular ROS and autophagy remains unknown. We performed a yeast two-hybrid screen to identify components involved in autophagy, using the central autophagy component ATG8e as a bait. We identified MEMBRANE ATTACK COMPLEX/PERFORIN-LIKE 2 (MACP2) as an interactor of ATG8e via its the ATG8-interacting motif and confirmed this interaction by co-immunoprecipitation and bimolecular fluorescence complementation assays. <em>MACP2-</em>overexpressing lines showed enhanced sensitivity to nutritional starvation, accelerated leaf senescence, and increased hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>) levels, resembling the phenotypes of <em>atg</em> mutants defective in autophagy. Conversely, <em>macp2</em> knockouts exhibited diminished starvation-induced H<sub>2</sub>O<sub>2</sub> accumulation and attenuated autophagosome formation and fully suppressed the starvation-hypersensitive phenotypes of the <em>atg5-1</em> mutant. In particular, MACP2 was degraded through the autophagy machinery during prolonged starvation, suggesting a feedback regulatory mechanism for maintaining MACP2 homeostasis. Our findings suggest that MACP2 acts as a key regulator in autophagy induction by controlling influx of extracellular H<sub>2</sub>O<sub>2</sub> in Arabidopsis.Â </p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1007/s44307-025-00078-4.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Autophagy,  MACP2,  ROS,  Arabidopsis</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">In plants, autophagy is an evolutionarily conserved mechanism that modulates responses to environmental stresses and development cues by targeting intracellular components for degradation (Bassham et al. <a href="#CR5" class="usa-link" aria-describedby="CR5">2006</a>; Huang et al. <a href="#CR23" class="usa-link" aria-describedby="CR23">2019</a>; Qi et al. <a href="#CR49" class="usa-link" aria-describedby="CR49">2021</a>; <a href="#CR51" class="usa-link" aria-describedby="CR51">2023</a>). The hallmark of autophagy is the formation of a double-membrane vesicle called the autophagosome, which facilitates the degradation of enclosed cargoes through the resident acid hydrolases of vacuoles (Liu &amp; Bassham <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>; Zhuang et al. <a href="#CR81" class="usa-link" aria-describedby="CR81">2018</a>; Qi et al. <a href="#CR49" class="usa-link" aria-describedby="CR49">2021</a>; <a href="#CR51" class="usa-link" aria-describedby="CR51">2023</a>). Several autophagy-related proteins (ATGs) have been shown to play indispensable roles at different stages of autophagosome formation via the assembly of four protein complexes: the ATG1â€“ATG13 protein kinase complex, the ATG6â€“phosphatidylinositol 3-kinase (PI3K) complex, the ATG9 membrane delivery complex, and the ATG5â€“ATG12 and ATG8â€“phosphatidylethanolamine (PE) complexes (Li &amp; Vierstra <a href="#CR28" class="usa-link" aria-describedby="CR28">2012</a>; Qi et al. <a href="#CR49" class="usa-link" aria-describedby="CR49">2021</a>; <a href="#CR51" class="usa-link" aria-describedby="CR51">2023</a>; Huang &amp; Guo <a href="#CR22" class="usa-link" aria-describedby="CR22">2024</a>). Among ATGs, the ubiquitin-like conjugation protein ATG8 is a key regulator in mediating selective recruitment of cargoes to the autophagosome (Kellner et al. <a href="#CR26" class="usa-link" aria-describedby="CR26">2017</a>). During selective autophagy, receptors interact with ATG8 through specific ATG8-interacting motif (AIM) or LIR-interacting motifs, characterized by the consensus amino acid sequence W/F/Y-X-X-L/I/V (Liu &amp; Bassham <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>; Marshall et al. <a href="#CR36" class="usa-link" aria-describedby="CR36">2019</a>; Acheampong et al. <a href="#CR1" class="usa-link" aria-describedby="CR1">2020</a>). Upon autophagy induction, ATG8 also interacts with several ATGs, such as ATG1 (Nakatogawa et al. <a href="#CR41" class="usa-link" aria-describedby="CR41">2012</a>), ATG4s (Yoshimoto et al. <a href="#CR75" class="usa-link" aria-describedby="CR75">2004</a>), ATG6, and ATG7 (Kaufmann et al. <a href="#CR25" class="usa-link" aria-describedby="CR25">2014</a>; Kaufmann &amp; Wollert <a href="#CR24" class="usa-link" aria-describedby="CR24">2014</a>), for either cargo recruitment or self-maintenance.</p>
<p id="Par3">Reactive oxygen species (ROS) are pivotal signaling molecules during plant interactions with their environment (Mittler et al. <a href="#CR37" class="usa-link" aria-describedby="CR37">2022</a>). As they are by-products of cellular metabolic pathways, the reactivity of ROS is a double-edged sword; at low levels, ROS function as key secondary messengers to maintain plant growth and development, while the excessive accumulation of ROS poses a substantial threat to cellular macromolecules by causing their irreversible oxidation (Halliwell <a href="#CR17" class="usa-link" aria-describedby="CR17">2007</a>; Castro et al. <a href="#CR6" class="usa-link" aria-describedby="CR6">2021</a>). Moreover, ROS production in different subcellular compartments, including the apoplast and intracellular organelles, initiates specific oxidative signaling pathways in plant cells (Castro et al. <a href="#CR6" class="usa-link" aria-describedby="CR6">2021</a>).</p>
<p id="Par4">Increasing evidence suggests that autophagy modulates cellular ROS homeostasis <em>in planta</em> (Chen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2015a</a>; Huang et al. <a href="#CR23" class="usa-link" aria-describedby="CR23">2019</a>; Qi et al. <a href="#CR49" class="usa-link" aria-describedby="CR49">2021</a>). ROS accumulation inside cells is responsible for the oxidative damage of proteins, lipids, or even organelles, which activates autophagy for their clearance (Xiong et al. <a href="#CR72" class="usa-link" aria-describedby="CR72">2007</a>; Qi et al. <a href="#CR49" class="usa-link" aria-describedby="CR49">2021</a>). In particular, autophagy distinguishes itself by its capacity to eliminate entire mitochondria and peroxisomes, two ROS-generating organelles, thereby limiting intracellular ROS production (Zhou et al. <a href="#CR80" class="usa-link" aria-describedby="CR80">2022</a>). By contrast, apoplastic ROS are primarily produced by plasma membraneâ€“localized flavin-containing NADPH oxidases, with the NADPH oxidases RESPIRATORY BURST OXIDASE HOMOLOG D (RBOHD) and RBOHF being major players in apoplast ROS production (Castro et al. <a href="#CR6" class="usa-link" aria-describedby="CR6">2021</a>). Previous studies have suggested that RBOHs are required in the leaves and roots of Arabidopsis (<em>Arabidopsis thaliana</em>) plants for autophagosome biogenesis in response to hypoxia and for plant tolerance to submergence or waterlogging (Chen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2015a</a>; Guan et al. <a href="#CR15" class="usa-link" aria-describedby="CR15">2019</a>). Thus, apoplastic ROS likely contribute to the regulation of stress responses through the functionality of autophagy. However, the molecular link between extracellular ROS and autophagy induction is still unknown.</p>
<p id="Par5">Members of the MEMBRANE ATTACK COMPLEX AND PERFORIN (MACP) family are major regulators during pathogen infections and the microbe-associated molecular pattern (MAMP)-triggered immunity in a variety of organisms (Rosado et al. <a href="#CR53" class="usa-link" aria-describedby="CR53">2008</a>; Fukunaga et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2017</a>). A signature motif, comprising the sequence Y/S-G-T/Sâ€“H-X7-G-G (X), has been identified in MACPFs from animals and plants (Ni &amp; Gilbert <a href="#CR44" class="usa-link" aria-describedby="CR44">2017</a>; Yu et al. <a href="#CR77" class="usa-link" aria-describedby="CR77">2020</a>; Chen et al. <a href="#CR9" class="usa-link" aria-describedby="CR9">2021</a>; Holmes et al. <a href="#CR21" class="usa-link" aria-describedby="CR21">2021</a>). Upon their oligomerization, MACPFs play key roles in animal development and immunity by forming pores in cellular membranes (Lukoyanova et al. <a href="#CR34" class="usa-link" aria-describedby="CR34">2016</a>; Moreno-Hagelsieb et al. <a href="#CR38" class="usa-link" aria-describedby="CR38">2017</a>). However, the biological function and molecular mechanism of plant MACPFs are poorly understood. In Arabidopsis, four genes encode proteins with a conserved MACPF motif: <em>CONSTITUTIVELY ACTIVATED CELL DEATH 1</em> (<em>CAD1</em>), <em>NECROTIC SPOTTED LESION 1</em> (<em>NSL1</em>), <em>MACP1</em>, and <em>MACP2</em> (Yu et al. <a href="#CR77" class="usa-link" aria-describedby="CR77">2020</a>; Zhang et al. <a href="#CR79" class="usa-link" aria-describedby="CR79">2022</a>). Among these genes, mutation of <em>CAD1</em> or <em>NSL1</em> constitutively activates plant defense responses and leads to programmed cell death in a salicylic acid-(SA) dependent manner (Morita-Yamamuro et al. <a href="#CR40" class="usa-link" aria-describedby="CR40">2005</a>; Noutoshi et al. <a href="#CR45" class="usa-link" aria-describedby="CR45">2006</a>; Tsutsui et al. <a href="#CR65" class="usa-link" aria-describedby="CR65">2006</a>; Asada et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2011</a>; Fukunaga et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2017</a>; Chen et al. <a href="#CR9" class="usa-link" aria-describedby="CR9">2021</a>). MACP2 regulates programmed cell death and participates in bacterial pathogen resistance and necrotrophic fungal pathogen susceptibility (Zhang et al. <a href="#CR79" class="usa-link" aria-describedby="CR79">2022</a>). These observations suggest that MACPFs are likely key factors in regulating plant defense signaling.</p>
<p id="Par6">In this study, we performed a yeast two-hybrid screen to uncover the ATG8-interaction network and identified MACP2 as a potential ATG8 interactor in Arabidopsis. We determined that MACP2 modulates extracellular H<sub>2</sub>O<sub>2</sub>-induced autophagosome biogenesis in response to carbon or nitrogen starvation. Under long-term starvation, however, the MACP2â€“ATG8 association led to MACP2 degradation via autophagy, revealing a maintenance mechanism for MACP2 protein abundance during prolonged nutritional starvation in plant cells.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Screening of ATG8-interacting proteins by yeast two-hybrid assay</h3>
<p id="Par7">Previous work suggested that during plant stress responses, autophagy acts as a protective mechanism to maintain the cellular ROS homeostasis (Chen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2015a</a>; Guan et al. <a href="#CR15" class="usa-link" aria-describedby="CR15">2019</a>). To investigate the molecular link between plant autophagy and stress-induced ROS accumulation, we screened for interactors of ATG8e by conducting a yeast two-hybrid screen. We identified 15 candidate interacting proteins with functions related to the categories of protein kinase, protein transport family, and decarboxylase, to name a few (TableÂ <a href="#Tab1" class="usa-link">1</a>). Among the 15 candidates, we noticed the H<sub>2</sub>O<sub>2</sub>-scavenging enzyme CATALASE 3 (CAT3) (Zou et al. <a href="#CR82" class="usa-link" aria-describedby="CR82">2015</a>) and a receptor-like cytoplasmic kinase (RLCK) from subgroup VI_A2 participating in plant growth and morphogenesis (Valkai et al. <a href="#CR66" class="usa-link" aria-describedby="CR66">2020</a>). We also obtained ENDOPLASMIC RETICULUM MORPHOLOGY 2 (ERMO2), a Sec23/Sec24 transport family protein involved in endoplasmic reticulumâ€“toâ€“Golgi trafficking (Qu et al. <a href="#CR52" class="usa-link" aria-describedby="CR52">2014</a>), confirming the connection between autophagy and endomembrane systems (Xia et al. <a href="#CR70" class="usa-link" aria-describedby="CR70">2020</a>). In particular, the protein MACP2 caught our attention because it regulates programmed cell death and participates in bacterial pathogen resistance and necrotrophic fungal pathogen susceptibility (Zhang et al. <a href="#CR79" class="usa-link" aria-describedby="CR79">2022</a>). Moreover, <em>MACP2</em> is widely expressed in various plant tissues and its transcript levels are upregulated in response to several biotic and abiotic stresses (Yu et al. <a href="#CR77" class="usa-link" aria-describedby="CR77">2020</a>). Thus, we chose MACP2 to explore the significance of its association with ATG8.
</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">TableÂ 1.</h4>
<div class="caption p"><p>Screening of ATG8-interacting proteins by Y2H assay</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Gene locus</th>
<th align="left" colspan="1" rowspan="1">Description</th>
<th align="left" colspan="1" rowspan="1">Name</th>
<th align="left" colspan="1" rowspan="1">AIM (xLIR)</th>
<th align="left" colspan="1" rowspan="1">PSSM Score</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT1G03130</em></td>
<td align="left" colspan="1" rowspan="1">Photosystem I reaction center subunit II</td>
<td align="left" colspan="1" rowspan="1"><em>PsaD-2</em></td>
<td align="left" colspan="1" rowspan="1">EEFYVI</td>
<td align="left" colspan="1" rowspan="1">10</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT5G45775</em></td>
<td align="left" colspan="1" rowspan="1">Ribosomal L5P family protein</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">KEYELL</td>
<td align="left" colspan="1" rowspan="1">13</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT1G01970</em></td>
<td align="left" colspan="1" rowspan="1">Tetratricopeptide repeat (TPR)-like superfamily protein</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">
<p>ADWLSI</p>
<p>RDYTKI</p>
<p>RVFDAV</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>21</p>
<p>16</p>
<p>7</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT4G04320</em></td>
<td align="left" colspan="1" rowspan="1">Malonyl-CoA decarboxylase family protein</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">
<p>RVYDVV</p>
<p>PTYEVL</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>9</p>
<p>13</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT3G07100</em></td>
<td align="left" colspan="1" rowspan="1">Sec23/Sec24 protein transport family protein</td>
<td align="left" colspan="1" rowspan="1"><em>ERMO2</em></td>
<td align="left" colspan="1" rowspan="1">EGFLLL</td>
<td align="left" colspan="1" rowspan="1">11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT2G18890</em></td>
<td align="left" colspan="1" rowspan="1">Protein kinase superfamily protein</td>
<td align="left" colspan="1" rowspan="1"><em>RLCK VI_A2</em></td>
<td align="left" colspan="1" rowspan="1">FSFQEI</td>
<td align="left" colspan="1" rowspan="1">12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT5G28840</em></td>
<td align="left" colspan="1" rowspan="1">GDP-D-mannose 3',5'-epimerase</td>
<td align="left" colspan="1" rowspan="1"><em>GME</em></td>
<td align="left" colspan="1" rowspan="1">REFTFI</td>
<td align="left" colspan="1" rowspan="1">12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT4G20330</em></td>
<td align="left" colspan="1" rowspan="1">Transcription initiation factor TFIIE, beta subunit</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">AVFDSL</td>
<td align="left" colspan="1" rowspan="1">7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT1G54040</em></td>
<td align="left" colspan="1" rowspan="1">Epithiospecifier protein</td>
<td align="left" colspan="1" rowspan="1"><em>ESP</em></td>
<td align="left" colspan="1" rowspan="1">
<p>RSYDTV</p>
<p>SEWTFL</p>
<p>LVWEKL</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>13</p>
<p>20</p>
<p>13</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT5G46250</em></td>
<td align="left" colspan="1" rowspan="1">RNA-binding protein</td>
<td align="left" colspan="1" rowspan="1"><em>LARP6a</em></td>
<td align="left" colspan="1" rowspan="1">
<p>KGFVPI</p>
<p>SSFLVV</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>11</p>
<p>11</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT4G24290</em></td>
<td align="left" colspan="1" rowspan="1">MAC/Perforin domain-containing protein</td>
<td align="left" colspan="1" rowspan="1"><em>MACP2</em></td>
<td align="left" colspan="1" rowspan="1">GVFISL</td>
<td align="left" colspan="1" rowspan="1">6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT1G20620</em></td>
<td align="left" colspan="1" rowspan="1">Catalase</td>
<td align="left" colspan="1" rowspan="1"><em>CAT3</em></td>
<td align="left" colspan="1" rowspan="1">FDFDPL</td>
<td align="left" colspan="1" rowspan="1">16</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT4G21110</em></td>
<td align="left" colspan="1" rowspan="1">G10 family protein</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">EGWELI</td>
<td align="left" colspan="1" rowspan="1">19</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT3G26650</em></td>
<td align="left" colspan="1" rowspan="1">glyceraldehyde 3-phosphate dehydrogenase A subunit</td>
<td align="left" colspan="1" rowspan="1"><em>GAPA</em></td>
<td align="left" colspan="1" rowspan="1">DEFVSI</td>
<td align="left" colspan="1" rowspan="1">15</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><em>AT1G07470</em></td>
<td align="left" colspan="1" rowspan="1">Transcription factor IIA, alpha/beta subunit</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">AVYIHV</td>
<td align="left" colspan="1" rowspan="1">6</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p30"><p>PSSM score represents the relative frequency at which a specific residue appears at that position</p></div></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">MACP2 interacts with ATG8e via the AIM</h3>
<p id="Par8">To explore the potential relationship between MACP2 and ATG8, we performed a phylogenetic analysis of MACP2 and related proteins and examined its protein structure and sequence. In Arabidopsis and rice (<em>Oryza sativa</em>), MACP members clustered into three groups based on the phylogenetic tree topology (Fig.Â <a href="#Fig1" class="usa-link">1</a>a). Among them, MACP2 is closely related to NSL1 in the second group of the MACP family. A comparison of the MACP2 sequence to those of EXO70Ds (Acheampong et al. <a href="#CR1" class="usa-link" aria-describedby="CR1">2020</a>) revealed a characteristic xLIR (G132â€“L137) domain, representing the primary ATG8-interacting motif (AIM), located within the MACPF domain of MACP2 and NSL1, as determined by the iLIR database (Fig.Â <a href="#Fig1" class="usa-link">1</a>b, Fig. S1). Moreover, structural predictions confirmed the presence of an xLIR domain within a Î²-pleated sheet in MACP2 and an Î±-helix in NSL1 (Fig.Â <a href="#Fig1" class="usa-link">1</a>c, Fig. S1). By contrast, the other two MACPs of Arabidopsis, MACP1 and CAD1, lacked this AIM (Fig. S1).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig.Â 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367599_44307_2025_78_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e986/12367599/9c0b3b6320ac/44307_2025_78_Fig1_HTML.jpg" loading="lazy" id="MO1" height="772" width="778" alt="Fig.Â 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Identification of MACP2 as an ATG8e interactor. <strong>a</strong> Phylogenetic tree of MACPF proteins in Arabidopsis (<em>Arabidopsis thaliana</em>) and rice (<em>Oryza sativa</em>)<em>.</em> The phylogenetic tree was reconstructed via the MEGA X software package with the maximum likelihood method based on the MACPF domain. <strong>b</strong> Sequence alignment of the conserved ATG8-interacting motif (AIM) in MACP2, NLS1, and EXO70Ds of Arabidopsis. EXO70Ds are confirmed ATG8-interacting proteins with AIM. The core AIM is indicated by a gray background. Conserved amino acids in AIM are shown with black backgrounds. Numbers indicate the corresponding amino acid positions. <strong>c</strong> Structure of the AIM in MACP2. AIM located within the MACP2 was analyzed by iLIR database, the protein structure was predicted by Alphafold and visualized using PyMOL software. The AIM is shown in red. <strong>d</strong> Co-immunoprecipitation (Co-IP) assay showing the interaction between ATG8e and MACP2 or MACP2 M1. The constructs <em>MACP2-HA</em>, <em>MACP2 M1-HA (MACP2</em><sup><em>F134A</em>,<em>L137A</em></sup><em>-HA</em>), and <em>FLAG-eGFP-ATG8e</em> were transfected into mesophyll protoplasts isolated from wild-type Arabidopsis and immunoprecipitated with FLAG affinity magnetic beads. <strong>e</strong> Bimolecular fluorescence complementation (BiFC) assay showing the interaction between MACP2 and ATG8e in vivo. The constructs <em>MACP2-nYFP</em> or <em>MACP2 M1-nYFP</em> (<em>MACP2</em><sup><em>F134A,L137A</em></sup><em>-nYFP</em>) were co-transfected with <em>cYFP-ATG8e</em> in protoplasts and incubated for 12Â h under continuous light conditions, which was followed by treatment with 100Â Î¼M benzo [1,2,3] thiadiazole-7-carbothioic acid S-methyl ester (BTH) and 1Â Î¼M concanamycin A (ConA) for 6Â h. MACP2 M1, MACP2 with F134A and L137A mutations in the AIM. The construct pairs <em>MACP2-nYFP</em>â€‰+â€‰<em>cYFP</em> and <em>nYFP</em>â€‰+â€‰<em>cYFP-ATG8e</em> were similarly co-transfected into Arabidopsis protoplasts as negative controls. Confocal images for YFP, chlorophyll autofluorescence, and bright field are shown. The numbers in the image indicated the ratio of YFP signal (the number of YFP signal to total protoplasts). The percentages are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three independent replicates. For each experiment, 15 images were used for the calculation for each coexpression combination. Scale bars, 10Â Î¼m</p></figcaption></figure><p id="Par9">To test the interaction between MACP2 and ATG8e <em>in planta</em>, we performed co-immunoprecipitation (Co-IP) and bimolecular fluorescence complementation (BiFC) assays. To this end, we co-transfected mesophyll protoplasts prepared from wild-type Arabidopsis plants with the constructs <em>MACP2-HA</em> (encoding MACP2 with a HA tag) and <em>FLAG-eGFP-ATG8e</em> (encoding a fusion between the green fluorescent protein [eGFP] and ATG8e, together with a FLAG tag) and incubated the transfected cells in darkness for 16Â h. When we precipitated ATG8e with anti-FLAG antibodies, we detected MACP2-HA among the immunoprecipitated proteins. To determine the dependence of MACP2-ATG8e interaction on the AIM domain of MACP2, we generated a point mutant form of MACP2 in which the conserved Phe-134 (F134) and Leu-137 (L137) residues were changed to Ala (MACP2<sup>F134A, L137A</sup>, M1) to disrupt the AIM domain. As expected, when we co-transfected the constructs <em>MACP2 M1-HA</em> and <em>FLAG-eGFP-ATG8e</em> in protoplast cells, FLAG-eGFP-ATG8e did not co-precipitate with MACP2 M1-HA (Fig.Â <a href="#Fig1" class="usa-link">1</a>d). As negative controls, FLAG-eGFP did not co-precipitate with MACP2-HA and MACP2 M1-HA (Fig.Â <a href="#Fig1" class="usa-link">1</a>d).</p>
<p id="Par10">Similarly, BiFC assays showed that after incubation in continuous light for 12Â h, protoplasts co-transfected with <em>MACP2-nYFP</em> and <em>cYFP-ATG8e</em> demonstrated reconstitution of functional YFP at the plasma membrane, indicative of the MACP2â€“ATG8e interaction (Fig.Â <a href="#Fig1" class="usa-link">1</a>e). By contrast, co-transfection of <em>MACP2 M1-nYFP</em> and <em>cYFP-ATG8e</em> failed to produce an intact YFP signal (Fig.Â <a href="#Fig1" class="usa-link">1</a>e). Furthermore, after treatment with the autophagy inducer benzo [1,2,3] thiadiazole-7-carbothioic acid S-methyl ester (BTH, salicylic acid agonist) and the inhibitor of vacuole-type ATPase concanamycin A (ConA), which inhibits the degradation of autophagic bodies, leading to their accumulation (Chen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2015a</a>; Qi et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2017</a>; <a href="#CR51" class="usa-link" aria-describedby="CR51">2023</a>), we observed the accumulation of YFP signal in punctate structures (Fig.Â <a href="#Fig1" class="usa-link">1</a>e). As controls, the <em>nYFP</em>â€‰+â€‰<em>cYFP</em>-<em>ATG8e</em> and <em>MACP2-nYFP</em>â€‰+â€‰<em>cYFP</em> combinations did not yield any YFP signal (Fig.Â <a href="#Fig1" class="usa-link">1</a>e). Taken together, these results suggest that MACP2 interacts with ATG8 through its AIM in vivo.</p></section><section id="Sec5"><h3 class="pmc_sec_title">
<em>MACP2</em>-overexpressing lines show accelerated leaf senescence and increased sensitivity to nutrient starvation</h3>
<p id="Par11">Previous studies have indicated that the autophagic degradation of chloroplasts occurs during natural premature leaf senescence and in response to nitrogen or carbon deprivation (Wada et al. <a href="#CR67" class="usa-link" aria-describedby="CR67">2009</a>; Avila-Ospina et al. <a href="#CR3" class="usa-link" aria-describedby="CR3">2014</a>; Qi et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2017</a>; <a href="#CR51" class="usa-link" aria-describedby="CR51">2023</a>). Conversely, autophagy-defective mutants are characterized by premature leaf senescence and hypersensitivity to nutrient deprivation (Hanaoka et al. <a href="#CR18" class="usa-link" aria-describedby="CR18">2002</a>; Chung et al. <a href="#CR10" class="usa-link" aria-describedby="CR10">2010</a>; Qi et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2017</a>). Given that MACP2 interacts with ATG8e, we sought genetic evidence to support the potential role of <em>MACP2</em> in autophagy. Accordingly, we obtained previously published knockout mutants (<em>macp2-1</em> and <em>macp2-2</em>) and overexpression lines (<em>MACP2-</em>OEs) for <em>MACP2</em> (Zhang et al. <a href="#CR79" class="usa-link" aria-describedby="CR79">2022</a>) for phenotypic analysis. 4-week-old wild-type, <em>macp2</em>, and <em>MACP2-</em>OE plants did not exhibit leaf senescence phenotypes when grown under normal conditions in a 16-h-light/8-h-dark photoperiod (Fig.Â <a href="#Fig2" class="usa-link">2</a>a). Notably, the cotyledons and several true leaves of <em>MACP2-</em>OE plants turned yellow in 5-week-old plants, while those in wild-type and <em>macp2</em> plants remained green (middle image; Fig.Â <a href="#Fig2" class="usa-link">2</a>a). Moreover, 6-week-old <em>macp2</em> and <em>MACP2-</em>OE plants displayed delayed and accelerated senescence, respectively, of their true leaves compared to those of wild-type plants (bottom image, Fig.Â <a href="#Fig2" class="usa-link">2</a>a). We confirmed the above findings by measuring the relative chlorophyll contents of all genotypes and we observed no difference in the chlorophyll contents of 4-week-old plants, while those of 6-week-old <em>macp2</em> mutants were significantly higher than that of the wild type. Similarly, the chlorophyll contents of 5- and 6-week-old <em>MACP2-</em>OE lines were significantly lower than in the wild type at the corresponding stage (Fig.Â <a href="#Fig2" class="usa-link">2</a>b).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig.Â 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e986/12367599/7d2787c0be02/44307_2025_78_Fig2_HTML.jpg" loading="lazy" id="MO2" height="938" width="787" alt="Fig.Â 2"></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypes of <em>macp2</em> knockout and <em>MACP2</em>-overexpressing lines in response to age-dependent senescence and nutrient starvation. <strong>a</strong> and <strong>b</strong> Representative photographs (<strong>a</strong>) and relative chlorophyll contents (<strong>b</strong>) showing the leaf senescence phenotypes of the wild type (WT), <em>macp2</em> mutants (<em>macp2-1</em> and <em>macp2</em>-<em>2</em>), and <em>MACP2-</em>OE lines (OE-1 and OE-2) grown under a 16-h-light/8-h-dark photoperiod for the indicated number of weeks. Photographs were taken and chlorophyll contents were analyzed at 4, 5, and 6Â weeks after germination. For each experiment, three entire plants were used per genotype. The experiments were performed as three biological replicates, each with similar results. Data are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three biological replicates. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA via STAT software, <em>P</em>â€‰&lt;â€‰0.05. <strong>c</strong> and <strong>e</strong> Representative photographs of WT, <em>macp2</em> and <em>MACP2-</em>OE (OE-1 and OE-2) plants in response to carbon (<strong>c</strong>) or nitrogen starvation (<strong>e</strong>). For carbon starvation, 4-week-old WT, <em>macp2</em>, and <em>MACP2-</em>OE plants were transferred to continuous darkness (Câ€‰âˆ’) for 7 d and photographed after a 7-d recovery period. For nitrogen starvation, WT, <em>macp2</em>, and <em>MACP2-</em>OE seedlings grown on half-strength solid Murashige and Skoog (MS) medium for 7 d were transferred to N-rich (Nâ€‰+) or N-free (Nâ€‰âˆ’) liquid medium and photographed at 5 d into treatment. <strong>d</strong> and <strong>f</strong> Relative chlorophyll contents of plants in response to carbon starvation (<strong>d</strong>) or nitrogen starvation (<strong>f</strong>) for the indicated times. For each experiment, three entire plants were used per genotype for carbon starvation (<strong>d</strong>) or 20 seedlings were treated per genotype for nitrogen starvation (<strong>f</strong>). All experiments were performed as three biological replicates, each with similar results, and representative data from one replicate are shown. Data are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three technical replicates. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em>â€‰&lt;â€‰0.05</p></figcaption></figure><p id="Par12">To more directly assess the involvement of MACP2 in the autophagy pathway, we subjected the wild type, the <em>macp2</em> mutants, and the <em>MACP2-</em>OE lines to carbon or nitrogen starvation by transfer into constant darkness or to nitrogen-free growth medium, respectively. We observed that 4-week-old <em>macp2</em> and <em>MACP2-</em>OE lines transferred to constant darkness for 7 d displayed enhanced and attenuated resistance, respectively, to fixed carbon starvation compared to wild type (Fig.Â <a href="#Fig2" class="usa-link">2</a>c). In agreement, the chlorophyll contents of the <em>macp2</em> and <em>MACP2-</em>OE lines were significantly higher and lower, respectively, than that of wild-type plants upon carbon starvation (Fig.Â <a href="#Fig2" class="usa-link">2</a>d). As with carbon starvation, 7-d-old <em>macp2</em> and <em>MACP2-</em>OE seedlings displayed increased and decreased tolerance of nitrogen deprivation, respectively, relative to the wild type when transferred to nitrogen-free Murashige and Skoog (MS) medium for 5 d (Fig.Â <a href="#Fig2" class="usa-link">2</a>e). Again, we confirmed these results by measuring the relative chlorophyll contents of all genotypes (Fig.Â <a href="#Fig2" class="usa-link">2</a>f). Together, these results suggest that MACP2 is likely a positive regulator of autophagy in plants.</p></section><section id="Sec6"><h3 class="pmc_sec_title">The accelerated senescence of <strong><em>MACP2-</em></strong>OE plants is associated with the salicylic acid and H<sub>2</sub>O<sub>2</sub> pathways</h3>
<p id="Par13">Autophagy has previously been shown to be essential for the disposal of damaged organelles and the maintenance of cellular ROS homeostasis in plants under stress conditions; these responses are controlled by the SA signaling pathway (Liu &amp; Bassham <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>; Chen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2015a</a>; Guo et al. <a href="#CR16" class="usa-link" aria-describedby="CR16">2021</a>). Given that SA and ROS accumulate in Arabidopsis <em>atg</em> mutants (Yoshimoto et al. <a href="#CR76" class="usa-link" aria-describedby="CR76">2009</a>), we measured the endogenous contents for SA and SA glycoside (SAG) of 4-, 5-, and 6-week-old wild-type, <em>macp2</em>, and <em>MACP2-</em>OE plants grown under normal conditions. We observed few significant differences in SA and SAG contents among 4-week-old wild-type, <em>macp2</em>, and <em>MACP2-</em>OE plants (Fig.Â <a href="#Fig3" class="usa-link">3</a>a). However, in 5- and 6-week-old plants, the SA and SAG contents in <em>macp2</em> and <em>MACP2-</em>OE plants were significantly lower and higher, respectively, than those in the wild type (Fig.Â <a href="#Fig3" class="usa-link">3</a>a).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig.Â 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367599_44307_2025_78_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e986/12367599/9d7d721397e3/44307_2025_78_Fig3_HTML.jpg" loading="lazy" id="MO3" height="910" width="709" alt="Fig.Â 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The accelerated senescence phenotypes of <em>MACP2-</em>OE plants are associated with salicylic acid pathway. <strong>a</strong> Measurement of salicylic acid (SA) and SA glycoside (SAG) contents in the rosettes of 4-, 5-, and 6-week-old wild type (WT), <em>macp2</em>, and <em>MACP2-</em>OE (OE-1 and OE-2) plants. The experiments were performed as three biological replicates, each with similar results, and representative data from one replicate are shown. Data are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three technical replicates. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em>â€‰&lt;â€‰0.05.<strong> b</strong> Hierarchical cluster analysis applied to the 4 DEGs (more than onefold and false discovery rateâ€‰&lt;â€‰0.05) representing senescence-related genes, and 5 DEGs in the SA pathway by comparing 5- and 6-week-old <em>macp2</em> and <em>MACP2-OEs</em> rosettes to that of the WT. The transcriptional profiles of relative gene expression values were analyzed using the TB tools command in R. Red and blue represent upregulated and downregulated genes, respectively. <strong>c</strong> Representative photographs showing the leaf senescence phenotypes of 4- and 5-week-old WT, <em>MACP2-</em>OE-1, <em>eds1-22</em>,Â OE-1 <em>eds1-22</em>, <em>pad4</em>, andÂ OE-1 <em>pad4</em> plants grown under a 16-h-light/8-h-dark photoperiod. Photographs were taken at 4 and 5Â weeks after germination. <strong>e</strong> Representative photographs of WT, <em>MACP2-</em>OE-1, <em>eds1-22</em>,Â OE-1 <em>eds1-22</em>, <em>pad4</em>, andÂ OE-1 <em>pad4</em> plants in response to carbon starvation. Four-week-old plants were kept in the light/dark normal condition (Câ€‰+) or transferred to continuous darkness (Câ€‰âˆ’) for 7 d to induce carbon starvation and photographed after a 7-d recovery in a 16-h-light/8-h-dark photoperiod. <strong>d</strong> and<strong> f</strong> Relative chlorophyll contents of wild-type (WT), <em>MACP2-</em>OE-1, <em>eds1-22</em>, <em>pad4</em>, OE-1 <em>eds1-22</em>, and OE-1 <em>pad4</em> plants upon natural senescence <strong>(d)</strong>, of 4-week-old plants under dark starvation (<strong>f</strong>), expressed as a percentage relative to control plants or seedlings. For each experiment, 3 entire plants (<strong>d</strong> and <strong>f</strong>) were used per genotype. All experiments were performed as three biological replicates, each with similar results. Relative chlorophyll contents are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three biological replicates. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em>â€‰&lt;â€‰0.05</p></figcaption></figure><p id="Par14">To investigate the correlation between SA and the leaf senescence phenotype of <em>macp2</em> and <em>MACP2-</em>OE plants, we assessed the transcript levels of senescence-associated genes and of SA-associated defense-responsive genes. Consistent with the senescence phenotypes, we detected a significant upregulation in the transcript levels of the senescence-associated genes <em>SENESCENCE-ASSOCIATED GENE13</em> (<em>SAG13</em>), <em>SAG29</em>, <em>SAG113</em>, and <em>SENESCENCE4</em> (<em>SEN4</em>), as well as of the defense genes <em>PATHOGENESIS-RELATED1</em> (<em>PR1</em>), <em>PR5</em>, <em>SULFURTRANSFERASE1</em> (<em>ST1</em>), <em>ENHANCED DISEASE RESISTANCE2</em> (<em>EDR2</em>) and <em>WRKY53</em> in the <em>MACP2-</em>OE lines compared to the wild type (Fig.Â <a href="#Fig3" class="usa-link">3</a>b). By contrast, the transcript levels of <em>SAG113</em>, <em>PR1</em>, and <em>EDR2</em> were lower in the 6-week-old <em>macp2</em> lines than in the wild type (Fig.Â <a href="#Fig3" class="usa-link">3</a>b).</p>
<p id="Par15">To explore the regulation of MACP2-mediated leaf senescence by SA signaling, we obtained the <em>MACP2</em>-OE <em>pad4</em> (OE <em>pad4</em>) and <em>MACP2</em>-OE <em>eds1</em> (OE <em>eds1</em>) lines (Zhang et al. <a href="#CR79" class="usa-link" aria-describedby="CR79">2022</a>) for phenotypic analysis. Compared to wild-type, <em>eds1</em>, and <em>pad4</em> plants, the accelerated natural senescence (Fig.Â <a href="#Fig3" class="usa-link">3</a>c) and starvation hypersensitivity (Fig.Â <a href="#Fig3" class="usa-link">3</a>e, Fig. S2a) phenotypes seen in <em>MACP2</em>-OE (OE-1) were attenuated by the <em>eds1</em> and <em>pad4</em> mutations in theÂ OE <em>eds1</em> andÂ OE <em>pad4</em> lines, which was also reflected in the relative chlorophyll contents of these genotypes (Fig.Â <a href="#Fig3" class="usa-link">3</a>d, f, Fig. S2b). Thus, the accelerated aging senescence seen in <em>MACP2-</em>OE plants is dependent on the SA pathway.</p>
<p id="Par16">To examine a possible association between the senescence phenotypes and ROS levels in <em>macp2</em> and <em>MACP2-</em>OE lines, we measured the total hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>) levels in the plants of all genotypes plants during senescence and nutritional starvation using an Amplex Red-based fluorescence assay. The 5- and 6-week-old <em>MACP2-</em>OE plants exhibited significantly higher H<sub>2</sub>O<sub>2</sub> levels than the wild type, while 6-week-old <em>macp2</em> mutant plants displayed significantly lower H<sub>2</sub>O<sub>2</sub> levels when grown under normal conditions (Fig.Â <a href="#Fig4" class="usa-link">4</a>a). Following 7 d of carbon or nitrogen starvation, <em>macp2</em> and <em>MACP2-</em>OE plants exhibited significantly lower and higher H<sub>2</sub>O<sub>2</sub> levels, respectively, than the wild type (Fig.Â <a href="#Fig4" class="usa-link">4</a>b, c), suggesting that MACP2 may be involved in maintaining cellular ROS homeostasis in response to senescence and nutrient starvation.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig.Â 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367599_44307_2025_78_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e986/12367599/f7e9bd30df72/44307_2025_78_Fig4_HTML.jpg" loading="lazy" id="MO4" height="514" width="708" alt="Fig.Â 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The accelerated senescence phenotypes of <em>MACP2-</em>OE plants are associated with H<sub>2</sub>O<sub>2</sub> influx. <strong>a</strong> to<strong> c</strong> Relative H<sub>2</sub>O<sub>2</sub> contents of wild-type (WT), <em>macp2</em>, and <em>MACP2-</em>OE (OE-1 and OE-2) plants grown under a 16-h-light/8-h-dark photoperiod for the indicated number of weeks (<strong>a</strong>), 4-week-old plants with or without a 7-d carbon starvation treatment (<strong>b</strong>), or of 7-d-old seedlings with or without a 7-d nitrogen starvation (<strong>c</strong>) treatment. For each experiment, three entire plants <strong>(a</strong> and <strong>b)</strong> or 20 seedlings (<strong>c</strong>) were used per genotype. All experiments were performed as three biological replicates, each with similar results. Data are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three biological replicates. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em>â€‰&lt;â€‰0.05. <strong>d</strong> and <strong>e</strong> Arabidopsis plants expressing the <em>Hyper</em> molecular probe in WT and <em>macp2-1</em> backgrounds to generate transgenic lines (WT <em>pHyper</em> and <em>macp2 pHyper</em>). The 7-d-old WT <em>pHyper</em> and <em>macp2 pHyper</em> seedlings were treated with or without a 24-h carbon starvation. Images represent the region of root cells excited at 488 and 405 nm, respectively (<strong>d</strong>). Scale bars, 50 Î¼m. (<strong>e</strong>) Quantification of <em>pHyPer</em> fluorescent signals in root cells from the WT <em>pHyper</em> and <em>macp2 pHyper</em> in (<strong>d</strong>). All experiments were performed as three biological replicates, each with similar results. Data are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three biological replicates. For each biological replicate, 15 seedlings were used for the calculation for per genotype. Different lowercase letters indicate significant differences between groups as determined by one-way ANOVA, <em>P</em>â€‰&lt;â€‰0.05</p></figcaption></figure><p id="Par17">To examine whether the phenotypes of nutrient starvation in <em>macp2</em> mutants were associated with intracellular ROS contents, we measured H<sub>2</sub>O<sub>2</sub> levels in the roots of 7-d-old <em>pHyPer</em> transgenic lines in wild type and <em>macp2</em> backgrounds (WT <em>pHyPer</em>, <em>macp2 pHyPer</em>) upon carbon starvation. HyPer is a genetically H<sub>2</sub>O<sub>2</sub>-specific biosensor to be nicely correlated with intracellular H<sub>2</sub>O<sub>2</sub> changes by determining the ratio of fluorescent signals at an excitation wavelength of 488Â nm relative to that at 405Â nm (HernÃ¡ndez-Barrera et al. <a href="#CR20" class="usa-link" aria-describedby="CR20">2013</a>; Shi et al. <a href="#CR58" class="usa-link" aria-describedby="CR58">2022</a>; KrÃ¼ger et al. <a href="#CR27" class="usa-link" aria-describedby="CR27">2021</a>). As shown in Fig.Â <a href="#Fig4" class="usa-link">4</a>, we observed that the ratio of fluorescent signals was enhanced after carbon starvation in the roots of wild type (Fig.Â <a href="#Fig4" class="usa-link">4</a>d, e). Moreover, we showed that the ratio of fluorescent signals in <em>macp2</em> was substantially lower than that of wild type before and after carbon starvation (Fig.Â <a href="#Fig4" class="usa-link">4</a>d, e), indicating that the mutation of <em>MACP2</em> in plants inhibits the influx of H<sub>2</sub>O<sub>2</sub> in plant cells.</p>
<p id="Par18">In plant cells, the excessÂ ROSÂ activates the production of antioxidants, including glutathione (GSH), to eliminate ROS-induced damage and maintain cellular homeostasis (Sharma et al. <a href="#CR57" class="usa-link" aria-describedby="CR57">2012</a>; Yuan et al. <a href="#CR78" class="usa-link" aria-describedby="CR78">2017</a>). Given that elevated levels of H<sub>2</sub>O<sub>2</sub> were detected in <em>MACP2-</em>OE plants under nutritional starvation stress, we further investigated the effects of GSH application on the sensitivity of carbon-deficient treatment in wild-type, <em>MACP2-</em>OE, and <em>atg5-1</em> seedlings. Under normal growth conditions (CK), the <em>MACP2-</em>OE (OE-1 and OE-2), and <em>atg5-1</em> seedlings exhibited little morphological changes from that of wild type with or without 500Â Î¼M GSH application for 12 d. However, the increased sensitivities of <em>MACP2-</em>OE plants to the carbon starvation (Câ€‰âˆ’) were completely rescued by supplying GSH in carbon-free solid half-strength MS growth medium (Fig. S3a). As a control, the hyper-sensitive phenotypes of <em>atg5-1</em> mutant were also partly restored by 500Â Î¼M GSH application (Fig. S3a). These findings were further confirmed by measuring the relative chlorophyll contents of all genotypes, which showed that <em>MACP2-</em>OE (OE-1, OE-2) and <em>atg5-1</em> plants accumulated significantly less chlorophylls than the wild type upon carbon starvation. With comparison, we did not observe the difference between wild type and <em>MACP2-</em>OE lines upon starvation with GSH supplied (Fig. S3b).</p></section><section id="Sec7"><h3 class="pmc_sec_title">MACP2 is required for starvation-induced autophagosome formation</h3>
<p id="Par19">Given that MACP2 is important for plant senescence and susceptibility to nutrient starvation, we investigated the genetic relationship between MACP2 and the autophagy pathway. To this end, we generated two <em>macp2Â atg5</em> (<em>macp2-1Â atg5-1</em> and <em>macp2-2Â atg5-1</em>) double mutants by crossing each <em>macp2</em> mutant to <em>atg5-1</em>. As shown in Fig.Â <a href="#Fig5" class="usa-link">5</a>a, compared to wild type, the <em>macp2</em> mutants had delayed leaf senescence, while the <em>atg5-1</em> mutants showed an accelerated leaf senescence phenotype, confirming previous findings (Liu &amp; Bassham <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>). Importantly, the <em>macp2Â atg5</em> double mutants displayed a partial rescue of the accelerated senescence phenotype seen in <em>atg5-1</em>, with fewer yellow leaves than <em>atg5-1</em> at both 5 and 6Â weeks (Fig.Â <a href="#Fig5" class="usa-link">5</a>a). We confirmed this observation by quantification of relative chlorophyll contents. The <em>macp2Â atg5</em> double mutants accumulated significantly more chlorophyll than <em>atg5-1</em>, with levels comparable to that of the wild type in 6-week-old plants (Fig.Â <a href="#Fig5" class="usa-link">5</a>b). Similarly, the <em>atg5-1</em> mutant was hypersensitive to carbon or nitrogen starvation (Fig.Â <a href="#Fig5" class="usa-link">5</a>c, Fig. S4a), as previously reported (Liu &amp; Bassham <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>). However, 4-week-old <em>macp2Â atg5</em> plants displayed enhanced tolerance of starvation, with more non-senescing green leaves, resembling the <em>macp2</em> mutants (Fig.Â <a href="#Fig5" class="usa-link">5</a>c, Fig. S4a). We confirmed these results by determining relative chlorophyll contents (Fig.Â <a href="#Fig5" class="usa-link">5</a>d, Fig. S4b).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig.Â 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367599_44307_2025_78_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e986/12367599/2392cc9313d9/44307_2025_78_Fig5_HTML.jpg" loading="lazy" id="MO5" height="751" width="709" alt="Fig.Â 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Knockout of <em>MACP2</em> rescues the accelerated senescence and ROS accumulation phenotypes of the <em>atg5</em> mutant.<strong> a</strong> and<strong> b</strong> Representative photographs (<strong>a</strong>) and relative chlorophyll contents (<strong>b</strong>) showing leaf senescence in the wild type (WT), <em>macp2</em>, <em>atg5-1</em>, and <em>macp2</em>Â <em>atg5</em> double mutants (<em>macp2-1Â atg5-1</em> and <em>macp2-2Â atg5-1</em>) grown under a 16-h-light/8-h-dark photoperiod. Photographs and the relative chlorophyll contents were obtained at 5 and 6 weeks after germination. For each experiment, three entire plants were used per genotype. The experiments were performed as three biological replicates, each with similar results, and representative data from one replicate are shown. Data are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three technical replicates. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em>â€‰&lt;â€‰0.05. <strong>c</strong> and<strong> d</strong> Representative photographs of (<strong>c</strong>) and relative chlorophyll contents (<strong>d</strong>) showing the WT, <em>macp2</em>, <em>atg5-1</em>, and <em>macp2</em>Â <em>atg5</em> double mutants in response to constant darkness treatment. Four-week-old WT, <em>macp2</em>, <em>atg5-1</em>, and <em>macp2</em>Â <em>atg5-1</em> double mutant plants were transferred to continuous darkness for 7 d and photographed after a 7-d recovery in a 16-h-light/8-h-dark photoperiod. <strong>e</strong> and<strong> f</strong> H<sub>2</sub>O<sub>2</sub> contents in WT, <em>macp2</em>, <em>atg5-1</em>, and <em>macp2</em>Â <em>atg5</em> double mutant plants during senescence (<strong>e</strong>) or following a carbon starvation treatment (<strong>f</strong>). For each experiment (<strong>b</strong> and <strong>dâ€“f</strong>), three entire plants were used per genotype. All experiments were performed as three biological replicates, each with similar results. Relative chlorophyll and H<sub>2</sub>O<sub>2</sub> contents are average valuesâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three biological replicates. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em>â€‰&lt;â€‰0.05</p></figcaption></figure><p id="Par20">Several autophagy-deficient mutants including <em>atg5-1</em> were shown to constitutively accumulate ROS, which contributes to the age- and starvation-dependent promotion of leaf senescence in these mutants (Yoshimoto et al. <a href="#CR76" class="usa-link" aria-describedby="CR76">2009</a>). Since the <em>macp2</em> mutants accumulate lower ROS levels in response to starvation (Fig.Â <a href="#Fig4" class="usa-link">4</a>b, c), we speculated that the recovery of the senescence phenotypes seen in <em>atg5-1</em> by <em>macp2</em> may be related to H<sub>2</sub>O<sub>2</sub> contents. To test this possibility, we examined the H<sub>2</sub>O<sub>2</sub> contents in wild-type, <em>macp2</em>, <em>atg5-1</em> and <em>macp2Â atg5</em> plants during natural senescence and upon nutrient starvation. Compared to the <em>atg5-1</em> mutant, H<sub>2</sub>O<sub>2</sub> levels were significantly lower in 5- and 6-week-old <em>macp2Â atg5</em> double mutant plants experiencing natural senescence (Fig.Â <a href="#Fig5" class="usa-link">5</a>e), as in <em>macp2Â atg5</em> double mutant plants under fixed carbon starvation for 7 d (Fig.Â <a href="#Fig5" class="usa-link">5</a>f). These results support the idea that the phenotypes observed in the <em>macp2Â atg5</em> double mutants are due to their reduced ROS production. These findings indicate that MACP2 may regulate autophagy by modulating ROS dynamics in response to age- and starvation-mediated senescence.</p>
<p id="Par21">To further explore the involvement of MACP2 in regulating autophagy, we crossed the <em>macp2-1</em> mutant and <em>MACP2</em>-OE line to a transgenic line expressing <em>eGFP-ATG8e</em>, which is widely used as an autophagosome marker (Xiao et al. <a href="#CR71" class="usa-link" aria-describedby="CR71">2010</a>; Chen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2015a</a>), to generate the <em>macp2-1</em>Â <em>eGFP-ATG8e</em> and <em>MACP2</em>-OE <em>eGFP-ATG8e</em> lines. We subjected the <em>eGFP-ATG8e</em> (<em>eGFP-ATG8e</em> in the wild-type background), and <em>macp2-1</em>Â <em>eGFP-ATG8e</em> and <em>MACP2</em>-OE <em>eGFP-ATG8e</em> lines to normal growth conditions or to constant darkness in the presence or absence of 1 Î¼M ConA for 16 h. An examination of fluorescence patterns by confocal microscopy suggested that in the absence of ConA treatment, the numbers of eGFP-ATG8e-labeled autophagic puncta in the root cells of <em>eGFP-ATG8e</em> and <em>macp2-1Â eGFP-ATG8e</em> lines were comparable regardless of fixed carbon starvation (Fig.Â <a href="#Fig6" class="usa-link">6</a>a, b). Upon 12-h carbon starvation, however, the numbers of autophagic puncta in the root cells of <em>MACP2</em>-OE <em>eGFP-ATG8e</em> was significantly increased than that of <em>eGFP-ATG8e</em> cells (Fig.Â <a href="#Fig6" class="usa-link">6</a>a, b). Moreover, the puncta in the <em>eGFP-ATG8e</em> line were markedly increased upon carbon starvation in the presence of ConA (Câ€‰âˆ’ +â€‰ConA; Fig.Â <a href="#Fig6" class="usa-link">6</a>a, b). Notably, starvation-induced autophagic puncta accumulated in the root cells of <em>macp2-1Â eGFP-ATG8e</em> and <em>MACP2</em>-OE <em>eGFP-ATG8e</em> were significantly lower and higher, respectively, than that of <em>eGFP-ATG8e</em> with ConA application (Fig.Â <a href="#Fig6" class="usa-link">6</a>a, b). As an independent method to measure autophagic flux, we employed the release of free eGFP to monitor the degradation of the eGFP-ATG8e reporter into the vacuole by autophagy (Qi et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2017</a>; <a href="#CR51" class="usa-link" aria-describedby="CR51">2023</a>). Consistent with the above confocal microscopy results, the ratio of free eGFP/eGFP-ATG8e following carbon starvation was lower in <em>macp2-1Â eGFP-ATG8e</em> and higher in <em>MACP2</em>-OE <em>eGFP-ATG8e</em> seedlings, compared to the <em>eGFP-ATG8e</em> line (Fig.Â <a href="#Fig6" class="usa-link">6</a>c), indicating that MACP2 is required for the starvation-induced formation of autophagosomes in Arabidopsis.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig.Â 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367599_44307_2025_78_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e986/12367599/716687fed2ea/44307_2025_78_Fig6_HTML.jpg" loading="lazy" id="MO6" height="566" width="778" alt="Fig.Â 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MACP2 modulates carbon starvationâ€“induced autophagosome formation. <strong>a</strong> and<strong> b</strong> Confocal analysis showing eGFP-ATG8e labeled autophagosomes in the root cells of wild-type (WT), <em>macp2-1</em>, and <em>MACP2</em>-OE seedlings. 7-d-old <em>eGFP-ATG8e</em>, <em>macp2-1 eGFP-ATG8e</em>, and <em>MACP2-</em>OE <em>eGFP-ATG8e</em> (<em>MACP2-FLAG eGFP-ATG8e</em>) lines were subjected to normal (CK) or carbon starvation (Câ€‰âˆ’) conditions, alone or with 1 ÂµM ConA treatment for 16 h. The formation of autophagosomes was visualized by fluorescence confocal microscopy. Scale bars, 10 Î¼m. (<strong>b</strong>) Number of puncta per root section in root cells from the <em>eGFP-ATG8e</em>, <em>macp2-1 eGFP-ATG8e</em>, and <em>MACP2-</em>OE <em>eGFP-ATG8e</em> lines in (<strong>a</strong>). Data are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three biological replicates. For each experiment, 15 sections were used for calculation per genotype. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em>â€‰&lt;â€‰0.05. <strong>c</strong> Immunoblot analysis showing the processing of eGFP-ATG8e in WT, <em>macp2-1</em>, and <em>MACP2</em>-OE seedlings in response to carbon starvation (Câ€‰âˆ’). Seven-d-old <em>eGFP-ATG8e</em>, <em>macp2-1 eGFP-ATG8e</em>, and <em>MACP2-</em>OE <em>eGFP-ATG8e</em> (<em>MACP2-FLAG eGFP-ATG8e</em>) seedlings were collected at 0, 12, 24, or 36 h post transfer (hpt) to constant darkness. Anti-eGFP antibodies were used for immunoblotting. The positions of the eGFP-ATG8e fusion and of free eGFP are indicated at right. The experiments were performed as three biological replicates, each with similar results. Data are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three biological replicates. The ratio between free eGFP and eGFP-ATG8e is shown below. The numbers at left indicate the molecular weight (kD) of each size marker. Anti-ACTIN antibodies and Ponceau Sâ€“stained membranes are shown below the blot to indicate the amount of protein loaded per lane</p></figcaption></figure><p id="Par22">Given that MACP2 influences the production of autophagosomes in plant responses to nutritional starvation, we further investigate the potential role of MACP2 in modulating ROS-induced autophagy biogenesis. To this end, we subjected the <em>eGFP-ATG8e</em>, <em>macp2-1</em>Â <em>eGFP-ATG8e</em>, and <em>MACP2</em>-OE <em>eGFP-ATG8e</em> lines to half-strength MS liquid medium with 100 Î¼M H<sub>2</sub>O<sub>2</sub> and SA agonist BTH (benzo [1,2,3] thiadiazole-7-carbothioic acid S-methyl ester) for the indicated times and then observed by confocal microscopy. We suggested that both treatments induced the formation of eGFP-ATG8e-labbled autophagic puncta significantly compared to the control in WT background (Fig. S5a-d), confirming previous findings (Xiong et al. <a href="#CR72" class="usa-link" aria-describedby="CR72">2007</a>; Yoshimoto et al. <a href="#CR76" class="usa-link" aria-describedby="CR76">2009</a>; He et al. <a href="#CR19" class="usa-link" aria-describedby="CR19">2023</a>). Moreover, we showed that compared to the WT plants, the <em>macp2-1</em> mutant and <em>MACP2</em>-OE accumulated significant less or more autophagic puncta in response to either H<sub>2</sub>O<sub>2</sub> or BTH (Fig. S5a-d).</p></section><section id="Sec8"><h3 class="pmc_sec_title">Autophagy contributes to MACP2 degradation upon long-term stress exposure</h3>
<p id="Par23">To verify the link between MACP2 and autophagy, we determined the subcellular localization of MACP2 in transgenic lines expressing <em>MACP2-YFP</em> (Zhang et al. <a href="#CR79" class="usa-link" aria-describedby="CR79">2022</a>). Confocal microscopy showed that under normal growth conditions, the MACP2-YFP fusion predominantly localized at the plasma membrane in the root cells of 7-d-old seedlings (Fig.Â <a href="#Fig7" class="usa-link">7</a>a). However, when the seedlings were transferred to constant darkness (Câ€‰âˆ’) or subjected to nitrogen starvation treatment (Nâ€‰âˆ’) for 12Â h, fluorescent signals for MACP2-YFP appeared to accumulate as vesicles in the cytoplasm (Fig.Â <a href="#Fig7" class="usa-link">7</a>a-c). In the <em>YFP</em> control line, we detected YFP signals in the cytoplasm and the nucleus under both normal growth conditions and nutrition starvation (Câ€’ and Nâ€‰âˆ’; Fig.Â <a href="#Fig7" class="usa-link">7</a>a).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig.Â 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367599_44307_2025_78_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e986/12367599/ccc6f7c2b100/44307_2025_78_Fig7_HTML.jpg" loading="lazy" id="MO7" height="740" width="778" alt="Fig.Â 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Feedback regulation of MACP2 by autophagy upon long-term stress exposure.<strong> a</strong> and<strong> c</strong> Subcellular localization of MACP2-YFP (upper images) in the root cells of 7-d-old <em>MACP2-YFP</em> seedlings under normal growth conditions (CK) or upon 12-h carbon (Câ€‰âˆ’â€‰12 h) or nitrogen starvation (Nâ€‰âˆ’â€‰12 h) in the presence of 1 Î¼M ConA. The localization of YFP alone (lower images) under similar conditions was used as a control. Scale bars, 10 Î¼m. (<strong>c</strong>) Number of puncta per root section in root cells from <em>MACP2-YFP</em> seedlings under normal growth conditions (CK), 12-h carbon (Câ€‰âˆ’â€‰12 hâ€‰+â€‰ConA) and nitrogen starvation (Nâ€‰âˆ’â€‰12 hâ€‰+â€‰ConA) in (<strong>a</strong>). Data are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three biological replicates. For each experiment, 15 sections were used for calculation per genotype. Different lowercase letters indicate significant differences between groups as determined by one-way ANOVA, <em>P</em>â€‰&lt;â€‰0.05. <strong>b</strong> Colocalization of MACP2-YFP with mCherry-ATG8f in response to carbon starvation in the presence of ConA (Câ€‰âˆ’â€‰12 hâ€‰+â€‰ConA). The <em>MACP2-YFP</em> line was crossed to <em>mCherry-ATG8f</em> to generate the double transgenic line <em>mCherry-ATG8f MACP2-YFP</em>. Colocalization assay of MACP2-YFP and mCherry-ATG8f in transgenic <em>mCherry-ATG8f MACP2-YFP</em> seedlings after treatment with 1 ÂµM ConA under continuous darkness for 12 h is shown. The experiments were performed as three biological replicates, each with similar results. For each experiment, 15 sections were used for observation for per genotype. Scale bars, 10 Î¼m. <strong>d</strong> Treatment with the autophagy inhibitor ConA attenuates the degradation of MACP2. 7-d-old <em>MACP2-FLAG</em> transgenic seedlings were exposed to carbon starvation in the presence and absence of 1 Î¼M ConA or 50 Î¼M MG132 for the indicated times (0, 6, 12, 24, or 48 h). hpt, hours post-treatment. The experiments were performed as three biological replicates, each with similar results. The abundance of ACTIN was used as a control. The relative quantification of MACP2-FLAG protein intensity was normalized to the corresponding ACTIN control and then compared to its level at 0 h. Data are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three biological replicates. <strong>e</strong> Degradation pattern of MACP2-HA in the wild type (WT), <em>atg5-1</em>, and <em>rpn10-1</em>. The <em>MACP2-HA</em> and e<em>GFP-FLAG</em> plasmids were co-transfected into protoplasts isolated from WT, <em>atg5-1</em>, and <em>rpn10-1</em> mutants. After overnight culture, protoplasts were treated with 50 Î¼M cycloheximide (CHX) under darkness for the indicated times (0, 2, 4, or 6 h). The experiments were performed as three biological replicates, each with similar results. The abundance of eGFP-FALG was used as a control. The relative quantification of MACP2-HA protein intensity was normalized to the corresponding eGFP-FALG control and then compared to its level at 0 h. Data are meansâ€‰Â±â€‰SD (<em>n</em>â€‰=â€‰3) calculated from three biological replicates. The numbers on the left indicate the molecular weight (kD) of each size marker. hpt, h post-treatment. <strong>f</strong> A working model for MACP2 in the regulation of autophagy induction by controlling influx of extracellular ROS in Arabidopsis. During leaf senescence or in plant response to nutrient deprivation, MACP2 protein forms a homo- or hetero-polymer at the plasma membrane to form a membrane attack complex (MAC) pore, which mediates the transport of the extracellular H<sub>2</sub>O<sub>2</sub> produced through RBOHs into the cytoplasm. During the early senescence or starvation stages, H<sub>2</sub>O<sub>2</sub> acts as a signal molecule to stimulate autophagy, which maintains cellular homeostasis and energy supply, ensuring cell growth (left panel). By contrast, the continuous accumulation of H<sub>2</sub>O<sub>2</sub> triggers the association of MACP2 with autophagosomes and targets MACP2 for its vacuolar degradation (right panel), thus mediating cellular ROS homeostasis and promoting cell survival under long-term starvation conditions</p></figcaption></figure><p id="Par24">To determine the nature of the punctate MACP2-YFP signals, we tested the colocalization of MACP2 with the autophagosome marker ATG8. To this end, we generated a <em>MACP2-YFP mCherry-ATG8f</em> double transgenic line by crossing <em>MACP2-YFP</em> with <em>mCherry-ATG8f</em> encoding a fusion between the red fluorescent protein mCherry and ATG8f (Xia et al. <a href="#CR70" class="usa-link" aria-describedby="CR70">2020</a>). Indeed, the starvation-induced punctate dots of MACP2-YFP primarily colocalized with mCherry-ATG8f in the root cells of 7-d-old seedlings upon 12Â h of carbon or nitrogen starvation in the presence of ConA (Fig.Â <a href="#Fig7" class="usa-link">7</a>b, Fig. S6), suggesting that MACP2 associates with starvation-induced autophagosomes <em>in planta</em>.</p>
<p id="Par25">To explore the potential regulatory mechanism linking autophagy and MACP2, we assayed the stability of MACP2 by using 7-d-old <em>pUBQ10:MACP2-FLAG</em> transgenic seedlings (Fig. S7). Immunoblot analysis revealed the accumulation of MACP2-FLAG for the first 24Â h following transfer to constant darkness, followed by a marked drop in abundance after 48Â h of darkness (Fig.Â <a href="#Fig7" class="usa-link">7</a>d), indicating that MACP2 itself is degraded under long-term stress exposure. When we repeated the experiment in the presence of the proteasome inhibitor MG132 or ConA, which inhibits the degradation of autophagic bodies (Qi et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2017</a>, <a href="#CR51" class="usa-link" aria-describedby="CR51">2023</a>; Xia et al. <a href="#CR70" class="usa-link" aria-describedby="CR70">2020</a>), we discovered that the application of ConA, but not MG132, suppresses the degradation of MACP2 during darkness for 48Â h (Fig.Â <a href="#Fig7" class="usa-link">7</a>d). To verify this finding, we transfected the <em>MACP2-HA</em> construct into Arabidopsis protoplasts isolated from the wild type, the autophagy mutant <em>atg5-1</em>, and the dual ATG8/Ubiquitin receptor mutant <em>rpn10-1</em> defective in REGULATORY PARTICLE NON-ATPASE 10 function (Smalle et al. <a href="#CR61" class="usa-link" aria-describedby="CR61">2003</a>; Marshall et al. <a href="#CR35" class="usa-link" aria-describedby="CR35">2015</a>). We then incubated all protoplasts for up to 6Â h in the presence of the protein translation inhibitor cycloheximide (CHX; Nazio et al. <a href="#CR42" class="usa-link" aria-describedby="CR42">2016</a>; <a href="#CR43" class="usa-link" aria-describedby="CR43">2017</a>). A protein stability assay revealed that MACP2-HA accumulated during the first 4Â h of CHX treatment, but decreased after 6Â h in continuous darkness in wild-type protoplasts (Fig.Â <a href="#Fig7" class="usa-link">7</a>e). However, this degradation of MACP2-HA was substantially attenuated in the <em>atg5-1</em> and <em>rpn10-1</em> mutants under the same conditions (Fig.Â <a href="#Fig7" class="usa-link">7</a>e), indicating that the autophagy pathway mediates the degradation of MACP2 under darkness. Furthermore, we investigated the protein abundance of MACP2 and MACP2<sup>F134A,L137A</sup> variant (MACP2 M1) in wild-type and <em>atg8e</em> Arabidopsis protoplasts under darkness for up to 8 h in the presence of CHX. Immunoblot analysis showed the degradation of MACP2-HA was faster in <em>atg8e</em> than in wild type. By contrast, the protein abundance of MACP2 M1-HA showed no difference in wild type and <em>atg8e</em> (Fig. S8). Taken together, these findings demonstrate that autophagy regulates MACP2 protein stability during darkness starvation exposure.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par26">In plants, autophagy is a degradation pathway that recycles damaged or unwanted cell materials and other macromolecules during development or upon exposure to abiotic and biotic stress (Liu &amp; Bassham <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>; Signorelli et al. <a href="#CR60" class="usa-link" aria-describedby="CR60">2019</a>). Increasing evidence demonstrates that autophagy and ROS are interconnected; that is, ROS can induce autophagy, and autophagy in turn contributes to lowering excessive ROS accumulation (Signorelli et al. <a href="#CR60" class="usa-link" aria-describedby="CR60">2019</a>; Zhou et al. <a href="#CR80" class="usa-link" aria-describedby="CR80">2022</a>). ROS are important early signaling molecules in plant responses to environmental stresses, but excess ROS may cause severe damage to cellular components. Under these circumstances, autophagy is a key mechanism by which ROS levels are kept under control during plant stress responses, possibly by modulating the degradation of ROS-generating organelles (Signorelli et al. <a href="#CR60" class="usa-link" aria-describedby="CR60">2019</a>; Castro et al. <a href="#CR6" class="usa-link" aria-describedby="CR6">2021</a>). However, how ROS, and particularly extracellular H<sub>2</sub>O<sub>2</sub>, induce autophagy remains largely unknown to date.</p>
<p id="Par27">In this study, we present several lines of evidence to support the idea that the membrane attack complex and perforin family protein MACP2 functions as a key regulator of autophagy induction by controlling the influx of extracellular H<sub>2</sub>O<sub>2</sub> in Arabidopsis. First, MACP2 interacted with ATG8e via its conserved AIM in vivo and translocated from the plasma membrane to autophagosomes in response to nutrient starvation (Fig.Â <a href="#Fig1" class="usa-link">1</a>, <a href="#Fig7" class="usa-link">7</a>). Second, knockout of <em>MACP2</em> (<em>macp2</em>) enhanced plant tolerance to nutrient starvation and delayed leaf senescence. Conversely, <em>MACP2</em>-overexpressing lines (<em>MACP2-</em>OEs) showed increased sensitivity to nutrient starvation and accelerated leaf senescence (Fig.Â <a href="#Fig2" class="usa-link">2</a>, <a href="#Fig3" class="usa-link">3</a>, and <a href="#Fig5" class="usa-link">5</a>). Third, <em>macp2</em> and <em>MACP2-</em>OEs lines showed significantly lower or higher SA and H<sub>2</sub>O<sub>2</sub> levels, respectively, than wild-type plants during natural age-dependent senescence or in response to nutrient starvation in an SA-dependent manner (Fig.Â <a href="#Fig3" class="usa-link">3</a>, <a href="#Fig4" class="usa-link">4</a>). Fourth, upon nutrient starvation, autophagosome formation was compromised in <em>macp2</em> compared to wild type (Fig.Â <a href="#Fig6" class="usa-link">6</a>). Finally, MACP2 was degraded through the autophagy machinery during prolonged nutrient starvation (Fig.Â <a href="#Fig7" class="usa-link">7</a>). Thus, our findings demonstrate that MACP2 is involved in autophagy induction by mediating the influx of extracellular ROS, whereas MACP2 itself is feedback regulated by autophagy to control a proper cellular ROS balance.</p>
<p id="Par28">In animal cells, MACPFs form oligomeric rings in target membranes during cell development and immunity (Lukoyanova et al. <a href="#CR34" class="usa-link" aria-describedby="CR34">2016</a>; Moreno-Hagelsieb et al. <a href="#CR38" class="usa-link" aria-describedby="CR38">2017</a>). We recently suggested that in plants, <em>MACP2</em> is widely expressed in various plant tissues, and its expression is upregulated in response to biotic and abiotic stresses (Yu et al. <a href="#CR77" class="usa-link" aria-describedby="CR77">2020</a>). Consistent with this notion, recent findings indicate that MACP2 induces programmed cell death essential for plant susceptibility to bacterial and necrotrophic fungal pathogens (Zhang et al. <a href="#CR79" class="usa-link" aria-describedby="CR79">2022</a>). Moreover, our findings revealed that the conserved residues in the AIM of MACP2 link its function to autophagy induction by interacting with ATG8 (Fig.Â <a href="#Fig1" class="usa-link">1</a>), suggesting that MACP2 is likely an important upstream regulator of the autophagy pathway involved in diverse stresses responses in plants.</p>
<p id="Par29">Autophagy is crucial for the maintenance of cellular homeostasis, acting as a quality control mechanism under both normal and stress conditions in plant cells (Qi et al. <a href="#CR51" class="usa-link" aria-describedby="CR51">2023</a>). Previous studies have shown that H<sub>2</sub>O<sub>2</sub> and the ROS-producing agent methylviologen (MV) trigger autophagy, the transcript level of <em>AtATG18a</em> increased after MV treatment in wild-type seedlings (Scherz-Shouval et al. <a href="#CR54" class="usa-link" aria-describedby="CR54">2007</a>; Shin et al. <a href="#CR59" class="usa-link" aria-describedby="CR59">2009</a>; Xiong et al. <a href="#CR72" class="usa-link" aria-describedby="CR72">2007</a>; Signorelli et al. <a href="#CR60" class="usa-link" aria-describedby="CR60">2019</a>; Lin et al. <a href="#CR32" class="usa-link" aria-describedby="CR32">2021</a>). In yeast (<em>Saccharomyces cerevisiae</em>) and animal cells, ROS activate autophagy by directly regulating the stability of ATG4 protein (Scherz-Shouval et al. <a href="#CR54" class="usa-link" aria-describedby="CR54">2007</a>; Essick &amp; Flora <a href="#CR12" class="usa-link" aria-describedby="CR12">2010</a>; Lin et al. <a href="#CR32" class="usa-link" aria-describedby="CR32">2021</a>). Therefore, we conceive that ROS-induced autophagy in plants may be mediated through either transcriptional upregulation of core autophagy-related genes, such as <em>AtATG18a</em>, or post-translational regulation of ATG proteins stability, as observed in yeast and animals, although the underlyingÂ mechanism remains to be elucidated. In Arabidopsis, most autophagy-defective mutants display conventional phenotypes with ROS accumulation, early leaf senescence, and increased sensitivity to nutrient deprivation (Liu &amp; Bassham <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>; Signorelli et al. <a href="#CR60" class="usa-link" aria-describedby="CR60">2019</a>). Similarly, we observed that <em>macp2</em> mutants displayed enhanced tolerance of nutrient starvation and delayed leaf senescence with lower ROS accumulation. By contrast, the <em>MACP2</em>-OE lines displayed increased sensitivity to nutrient starvation and accelerated leaf senescence with higher ROS accumulation (Fig.Â <a href="#Fig2" class="usa-link">2</a>â€“<a href="#Fig4" class="usa-link">4</a>), suggesting that MACP2 is likely a positive regulator in ROS production in planta. It is worthy to note that like the <em>MACP2</em>-OE lines, the <em>atg</em> mutants also displayed accelerated senescence phenotypes (Hanaoka et al. <a href="#CR18" class="usa-link" aria-describedby="CR18">2002</a>; Yoshimoto et al. <a href="#CR76" class="usa-link" aria-describedby="CR76">2009</a>; Qi et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2017</a>), which is likely due to their cellular accumulation of ROS (Signorelli et al. <a href="#CR60" class="usa-link" aria-describedby="CR60">2019</a>), supporting that ROS is a double-edged sword in autophagy induction and plant tolerance to starvation. Notably, the accelerated natural and starvation-induced senescence phenotypes of <em>atg5-1</em> were partially rescued by mutation of <em>MACP2</em>, which was also correlated with ROS levels <em>in planta</em> (Fig.Â <a href="#Fig5" class="usa-link">5</a>), suggesting that the cellular ROS level is a key determinant factor for induction of the accelerated senescence phenotypes in both the <em>MACP2</em>-OE lines and <em>atg5-1</em> mutants. Consistent with this notion, the knockout of <em>MACP2</em> blocks the intracellular ROS accumulation upon starvation or during senescence, which therefore attenuates its enhanced senescent phenotypes.</p>
<p id="Par30">RBOHD and RBOHF are two major NADPH oxidases localizing at the plasma membrane to catalyze the production of apoplastic ROS in plant cells (Castro et al. <a href="#CR6" class="usa-link" aria-describedby="CR6">2021</a>). We and other groups have previously observed that in Arabidopsis, the autophagosome-like structures decrease in the <em>rbohd</em>, <em>rbohf</em>, and <em>rbohd rbohf</em> mutants in comparison to the wild type under hypoxia stress (Chen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2015a</a>; Guan et al. <a href="#CR15" class="usa-link" aria-describedby="CR15">2019</a>), supporting the role of RBOH-generated ROS in inducing autophagy in plant cells. Given that <em>macp2</em> accumulates less H<sub>2</sub>O<sub>2</sub> than the wild type under natural senescence and nutrient starvation conditions (Fig.Â <a href="#Fig4" class="usa-link">4</a>), it is likely that the diminished H<sub>2</sub>O<sub>2</sub> influx in the <em>macp2</em> mutant contributes to its rescue of the <em>atg5-1</em> phenotypes (Fig.Â <a href="#Fig5" class="usa-link">5</a>e, f). Previous works suggest that membrane attack complex (MAC) allows the free passage of water, ions (e.g. Ca<sup>2+</sup>), and/or other molecules, including small proteins across membranes to induce cell death (Morgan <a href="#CR39" class="usa-link" aria-describedby="CR39">1989</a>; Bakopoulos et al. <a href="#CR4" class="usa-link" aria-describedby="CR4">2019</a>). We speculate that, in response to environmental cues, MACP2 forms a homo- or hetero-polymer at the plasma membrane to mediate influx of extracellular ROS, which act as signals to stimulate the initiation of autophagosome formation. Consistent with our findings, a recently study shows that the mitochondrial GSDMD (pore-forming protein gasdermin D) protein mediates pore formation to release mitochondrial ROS across the membrane (Weindel et al. <a href="#CR69" class="usa-link" aria-describedby="CR69">2022</a>). Thus, it is likely that the pore-forming proteins play an important role in mediating the transport of H<sub>2</sub>O<sub>2</sub> across the membrane in cells. Consistent with our findings, several previous studies reveal that multiple plant aquaporins facilitate H<sub>2</sub>O<sub>2</sub> transport (Waszczak et al. <a href="#CR68" class="usa-link" aria-describedby="CR68">2018</a>; Castro et al. <a href="#CR6" class="usa-link" aria-describedby="CR6">2021</a>). For instance, Arabidopsis PIP1;4 (plasma membrane intrinsic protein 1;4) and PIP2;1 aquaporins mediate H<sub>2</sub>O<sub>2</sub> influx into the cytoplasm during plant immunity and stomatal closure, respectively (Grondin et al. <a href="#CR14" class="usa-link" aria-describedby="CR14">2015</a>; Tian et al. <a href="#CR64" class="usa-link" aria-describedby="CR64">2016</a>). Intriguingly, recent findings reveal that autophagy is involved in the degradation of PIP2;7 under drought stress conditions (Li et al. <a href="#CR30" class="usa-link" aria-describedby="CR30">2020</a>; Tang &amp; Bassham. <a href="#CR63" class="usa-link" aria-describedby="CR63">2022</a>), indicating autophagy may function with several membrane-localized proteins modulating cellular ROS trafficking in response to biotic and abiotic stresses. Interestingly, another MACP family member NSL1 also contains the conserved AIM domain (Fig. S1), which is involved in modulating ROS production in plant immunity response (Fukunaga et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2017</a>). However, whether NSL1 shares a redundant function with MACP2 in maintaining cellular ROS homeostasis and regulating autophagy biogenesis remain to be further investigated.</p>
<p id="Par31">Moreover, several studies have revealed that autophagy likely mediates the maintenance of ROS homeostasis under stresses via the NONEXPRESSER OF PR GENES1 (NPR1)â€“PHYTOALEXIN-DEFICIENT4 (PAD4)â€“ENHANCED DISEASE SUSCEPTIBILITY1 (EDS1) SA pathway (Liu &amp; Bassham <a href="#CR33" class="usa-link" aria-describedby="CR33">2012</a>; Chen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2015a</a>; Qi et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2017</a>; Guo et al. <a href="#CR16" class="usa-link" aria-describedby="CR16">2021</a>). Previous studies have demonstrated that SA agonist BTH induces autophagosome formation (Yoshimoto et al. <a href="#CR76" class="usa-link" aria-describedby="CR76">2009</a>; He et al. <a href="#CR19" class="usa-link" aria-describedby="CR19">2023</a>; Qi et al. <a href="#CR51" class="usa-link" aria-describedby="CR51">2023</a>). In Arabidopsis, mutation of <em>ATG</em> genes results in upregulation of the SA biosynthesis genes and consequent SA accumulation, thereby influencing both senescence processes and pathogen defense responses (Yoshimoto et al. <a href="#CR76" class="usa-link" aria-describedby="CR76">2009</a>; Liao &amp; Bassham. <a href="#CR31" class="usa-link" aria-describedby="CR31">2020</a>). Consistent with this notion, the SA and SAG contents of <em>macp2</em> and <em>MACP2-</em>OE plants were significantly lower and higher, respectively, than in wild type (Fig.Â <a href="#Fig3" class="usa-link">3</a>a). Moreover, the accelerated natural senescence and hypersensitivity to nutrition starvation seen in <em>MACP2-</em>OE lines were completely suppressed in the <em>eds1</em> and <em>pad4</em> mutant backgrounds (Fig.Â <a href="#Fig3" class="usa-link">3</a>c-f, Fig. S2). Moreover, the SA signaling mutant <em>npr1</em> is observed to suppress the early senescence phenotype in <em>atg5</em>, suggesting that the senescence phenotype in <em>atg5</em> mutant is SA dependent (Yoshimoto et al. <a href="#CR76" class="usa-link" aria-describedby="CR76">2009</a>). Therefore, the observations that the <em>macp2</em>Â mutation rescued the accelerated senescence phenotype and the sensitivity to nutrient starvation of <em>atg5-1</em> in the <em>macp2Â atg5</em> double mutants (Fig.Â <a href="#Fig5" class="usa-link">5</a>a-d), may be explained by the decreased SA contents in the <em>macp2</em> mutants during senescence or upon starvation. Furthermore, previous works have suggested that the accumulation of ROS in <em>atg</em> mutants is mediated by SA pathway (Yoshimoto et al. <a href="#CR76" class="usa-link" aria-describedby="CR76">2009</a>; Chen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2015a</a>). Likewise, the accumulation of ROS can stimulate SA biosynthesis, suggesting these two signaling molecules likely form a positive amplification loop (Shah <a href="#CR55" class="usa-link" aria-describedby="CR55">2003</a>; Yoshimoto et al. <a href="#CR76" class="usa-link" aria-describedby="CR76">2009</a>). Collectively, these findings establish that SA and ROS both function as autophagy inducers and mutually reinforcing signaling molecules within an integrated regulatory network. It is worth noting that mutation of either MACPF family gene, <em>CAD1</em> or <em>NSL1</em>, constitutively activates plant defense responses and programmed cell death in an SA-dependent manner (Morita-Yamamuro et al. <a href="#CR40" class="usa-link" aria-describedby="CR40">2005</a>; Noutoshi et al. <a href="#CR45" class="usa-link" aria-describedby="CR45">2006</a>; Tsutsui et al. <a href="#CR65" class="usa-link" aria-describedby="CR65">2006</a>; Asada et al. <a href="#CR2" class="usa-link" aria-describedby="CR2">2011</a>; Fukunaga et al. <a href="#CR13" class="usa-link" aria-describedby="CR13">2017</a>; Chen et al. <a href="#CR9" class="usa-link" aria-describedby="CR9">2021</a>). Thus, SA is an important regulatory signaling molecule that modulates MACPF function and ROS-triggered autophagy induction.</p>
<p id="Par32">We further showed that under normal growth conditions, MACP2-YFP predominately localized to the plasma membrane; however, MACP2-YFP translocated to autophagosomes in response to nutrient starvation, as evidenced by the colocalization of MACP2-YFP punctate dots with mCherry-ATG8f-labeled vesicles (Fig.Â <a href="#Fig7" class="usa-link">7</a>a-c, S6). The association of MACP2 with autophagosomes may reflect its feedback degradation by the autophagy machinery under long-term stresses (Fig.Â <a href="#Fig7" class="usa-link">7</a>d, e, Fig. S8), indicating that MACP2 probably regulates and is regulated by the autophagy pathway, which is potentially a key mechanism in controlling a proper cell ROS balance, especially during prolonged starvation exposure. This phenomenon has been frequently observed in the autophagy- and vacuolar sortingâ€“mediated degradation pathways. For instance, the two central protein complexes ATG1â€“ATG13 and SEVEN IN ABSENTIA OF ARABIDOPSIS (SINATs)â€“FYVE DOMAIN PROTEIN REQUIRED FOR ENDOSOMAL SORTING1 (FREE1) modulate autophagosome biogenesis and are degraded via the autophagy pathway (Suttangkakul et al. <a href="#CR62" class="usa-link" aria-describedby="CR62">2011</a>; Xia et al. <a href="#CR70" class="usa-link" aria-describedby="CR70">2020</a>), thus providing feedback control of autophagy initiation. Notably, our results demonstrated that the MACP2 protein stability is mediated by autophagy pathway, but not through the 26S proteasome pathway during carbon starvation (Fig.Â <a href="#Fig7" class="usa-link">7</a>d, e). However, the degradation of MACP2 was suppressed in the dual ATG8/Ubiquitin receptor mutant <em>rpn10-1</em> (Fig.Â <a href="#Fig7" class="usa-link">7</a>e). A previous work demonstrates that the proteasome subunit RPN10 simultaneously binds to both ATG8 and ubiquitin (Marshall et al. <a href="#CR35" class="usa-link" aria-describedby="CR35">2015</a>). Thus, RPN10 likely acts as a selective autophagy receptor to target inactive 26S proteasomes by concurrent interactions with ubiquitylated proteasome subunits and lipidated ATG8 lining the enveloping autophagic membranes (Marshall et al. <a href="#CR35" class="usa-link" aria-describedby="CR35">2015</a>). Thus, we considered that RPN10 may mediate the degradation of MACP2 by activation of proteophagy during carbon starvation, rather than proteasome-mediated degradation. However, the precise molecular mechanism underlying RPN10-mediated MACP2 degradation requires further elucidation.</p>
<p id="Par33">In conclusion, we elucidated the roles of MACP2 in the regulation of autophagy induction through control of the influx of extracellular ROS in Arabidopsis (Fig.Â <a href="#Fig7" class="usa-link">7</a>f). During leaf senescence or in response to nutrient deprivation, MACP2 forms a homo- or hetero-polymer at the plasma membrane to form a membrane attack complex (MAC) pore, which mediates the transport of the extracellular H<sub>2</sub>O<sub>2</sub> produced through RBOHs into the cytoplasm. During early senescence or nutrient starvation, H<sub>2</sub>O<sub>2</sub> acts as a signal molecule to stimulate the induction of autophagy, which maintains cellular homeostasis and energy supply, ensuring cell growth (left panel, Fig.Â <a href="#Fig7" class="usa-link">7</a>f). By contrast, the continuous accumulation of H<sub>2</sub>O<sub>2</sub> triggers the association of MACP2 with autophagosomes and targets MACP2 for its vacuolar degradation (right panel, Fig.Â <a href="#Fig7" class="usa-link">7</a>f), thus mediating cell ROS homeostasis and promoting cell survival under long-term starvation. Our findings therefore provide insights into the molecular link between extracellular ROS and autophagy induction and suggest that MACP2-composed MAC pores at the plasma membrane may serve as functional channels for ROS influx in response to environmental or physiological stresses, including nutrient starvation.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Materials and method</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Plant materials and growth conditions</h3>
<p id="Par34">Arabidopsis (<em>Arabidopsis thaliana</em>) accession Columbia-0 (Col-0) was used as the wild type in this study. The T-DNA insertion <em>macp2-1</em> (SALK_040186) and <em>macp2-2</em> (SALK_052845C) mutants were obtained from The Arabidopsis Information Resource (TAIR; <a href="http://www.arabidopsis.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.arabidopsis.org</a>). The <em>eds1-22</em>, <em>pad4</em>, <em>atg5-1</em>, <em>atg8e</em>, and <em>rpn10-1</em> mutants used in this study have been described previously (Shah <a href="#CR56" class="usa-link" aria-describedby="CR56">1997</a>; Chen et al. <a href="#CR8" class="usa-link" aria-describedby="CR8">2015b</a>; Xia et al. <a href="#CR70" class="usa-link" aria-describedby="CR70">2020</a>). For genetic analysis, the <em>eds1-22</em> and <em>pad4</em> mutants were crossed to <em>MACP2-</em>OEs to generate theÂ OE <em>eds1</em> (Zhang et al. <a href="#CR79" class="usa-link" aria-describedby="CR79">2022</a>) andÂ OE <em>pad4</em> lines; similarly, the <em>atg5-1</em> mutant was crossed to <em>macp2</em> mutants to generate the double mutants <em>macp2-1Â atg5-1</em> and <em>macp2-2Â atg5-1</em>. To investigate H<sub>2</sub>O<sub>2</sub> influx in plants, <em>p35S::HyPer</em> transgenic line (Shi et al. <a href="#CR58" class="usa-link" aria-describedby="CR58">2022</a>) was crossed to <em>macp2-1</em> mutant to generate the <em>macp2-1 p35S::HyPer</em> line. Fluorescence intensity o<em>f pHyPer</em> transgenic plants was determined in the roots of 1-week-old seedlings by the excitation wavelength at 488Â nm or 405Â nm and the emission at 530Â nm (band pass 20). A fluorescence ratio was calculated as 488/405.</p>
<p id="Par35">For seed germination assays, Arabidopsis seeds were surface-sterilized with 20% (v/v) bleach containing 0.1% (v/v) Tween 20 (Sigma, P2287) for 15Â min, washed with distilled water at least six times, and plated on half-strength Murashige and Skoog (MS, Sigma, M5519) medium with 1% (w/v) sucrose and 0.8% (w/v) agar. The plates were incubated at 4Â Â°C for 2 d in the darkness, before being transferred to a growth room under a 16-h-light/8-h-dark (20Â Â°C/24Â Â°CÂ day/night) photoperiod. At 7 d after germination, seedlings were transplanted to soil (Pansa, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PS008321" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PS008321</a>) for further growth under the same conditions.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Treatments</h3>
<p id="Par36">The treatment of nutrient starvation was performed as described previously (Qi et al. <a href="#CR51" class="usa-link" aria-describedby="CR51">2023</a>). For nitrogen starvation tests, 7-d-old seedlings grown on solid half-strength MS medium were transferred to liquid half-strength MS medium or nitrogen-free liquid half-strength MS medium and grown under normal growth conditions for the indicated times. For carbon starvation, 4-week-old adult plants were transferred to continuous darkness for the indicated times and then allowed to recover under normal growth conditions for 7 d. For seedling treatment, 7-d-old seedlings grew in half-strength MS were transferred to solid sucrose-free medium with or without 500Â Î¼M GSH and incubated under continuous dark conditions for 12 d, followed by recovery under normal growth conditions for 5 d. Samples were collected or photographed at the indicated time points. The relative chlorophyll content was recorded as described below.</p>
<p id="Par37">For chemical treatments, 7-d-old Arabidopsis seedlings grown on solid half-strength MS medium with 1% (w/v) sucrose were transferred to sterile 12-well plates filled with liquid half-strength MS medium with no added sucrose (âˆ’â€‰C) and 50Â Î¼M MG132 or 1Â Î¼M concanamycin A (ConA) in continuous darkness at the indicated time points (0, 6, 12, 24, or 48Â h). After the specified treatments, seedlings were collected for confocal analysis or flash-frozen in liquid nitrogen for protein extraction and immunoblot analysis. For transient expression analysis, the plasmid <em>MACP2-HA</em> and <em>MACP2 M1-HA</em> were transfected into wild-type, <em>atg5-1</em>, <em>atg8e</em>, and <em>rpn10-1</em> protoplasts; after overnight culture under continuous light, protoplasts were treated with 50Â Î¼M cycloheximide (CHX) in darkness for the indicated times (0, 2, 4, or 6Â h). Subsequently, the protoplasts were collected and the protein extracted for immunoblot analysis.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Yeast Two-Hybrid (Y2H) screening</h3>
<p id="Par38">For conducting the yeast two-hybrid assay, the cDNA of <em>Arabidopsis ATG8e</em> was inserted into the pLexA vector, which was fused with the LexA DNA-binding domain (BD), thereby serving as the bait. The screening was carried out using the Matchmaker LexA two-hybrid system (Clontech) along with the EGY48 yeast strain to probe an Arabidopsis cDNA library (Xu et al. <a href="#CR73" class="usa-link" aria-describedby="CR73">2002</a>). The transformed yeast cells were then cultured on 2% Gal/Raf/X-Î²-gal (SD-Ura/-His/-Trp/-Leu) selective medium in accordance with the guidelines provided by Clontech.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Plasmid construction and transgenic plant generation</h3>
<p id="Par39">All constructs were generated using a ClonExpress II One Step Cloning Kit (catalog no. C112; Vazyme). All gene-specific primers with about 15-bp extensions homologous to the corresponding vectors are listed in Table S1. For the <em>MACP2-HA</em> and <em>MACP2 M1-HA</em> constructs, the full-length coding sequence of <em>MACP2</em> and <em>MACP2</em> with mutations (<em>MACP2</em><sup><em>F134A</em>,<em>L137A</em></sup>) were cloned into <em>Bam</em>HI- and <em>Stu</em>I-digested pUC119 vector (containing the Arabidopsis <em>UBIQUITIN10</em> [<em>UBQ10</em>] promoter; Li et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2013</a>; Xia et al. <a href="#CR70" class="usa-link" aria-describedby="CR70">2020</a>) to generate the <em>MACP2-HA</em> and <em>MACP2 M1-HA</em> constructs. The full-length coding sequence ATG8e was cloned into <em>Bam</em>HI-digested pUC120 (harboring the Arabidopsis <em>UBQ10</em> promoter; Xia et al. <a href="#CR70" class="usa-link" aria-describedby="CR70">2020</a>) to generate the <em>FLAG-eGFP-ATG8e</em> construct. To generate plasmids for the bimolecular fluorescence complementation (BiFC) assay, the full-length coding sequence of <em>MACP2</em> and its variant (<em>MACP2</em><sup><em>F134A</em>, <em>L137A</em></sup>) was cloned into <em>Bam</em>HI-digested pHBT-YN or pHBT-YC vector (Qi et al. <a href="#CR48" class="usa-link" aria-describedby="CR48">2017</a>), respectively, to generate <em>MACP2-nYFP</em> and M<em>ACP2 M1-nYFP</em> constructs. The full-length coding sequence fragment of <em>ATG8e</em> was cloned into <em>Bam</em>HI-digested pUC120-YC (Xia et al. <a href="#CR70" class="usa-link" aria-describedby="CR70">2020</a>) to generate <em>cYFP-ATG8e</em> construct.</p>
<p id="Par40">To generate stable transgenic plants expressing <em>MACP2-YFP</em> or <em>MACP2-FLAG</em>, the fusion cassettes derived from the above pUC119 constructs were cloned into the binary vector pFGC-RCS (Li et al. <a href="#CR29" class="usa-link" aria-describedby="CR29">2013</a>; Zhang et al. <a href="#CR79" class="usa-link" aria-describedby="CR79">2022</a>). The resulting clones were subsequently introduced into wild-type Arabidopsis (Col-0) or the mutants <em>eds1</em> and <em>pad4</em> via Agrobacterium (<em>Agrobacterium tumefaciens</em>) â€“mediated transformation using the floral dip method (CloughÂ &amp; Bent (Clough, et al., <a href="#CR11" class="usa-link" aria-describedby="CR11">1998</a>)) to generate <em>MACP2-FLAG</em>, <em>MACP2-YFP</em> (<em>MACP2-</em>OEs),Â <em>MACP2-YFP eds1</em> (OE <em>eds1</em>), andÂ <em>MACP2-YFP pad4</em> (OE <em>pad4</em>) transgenic plants. The <em>MACP2-YFP</em> line was crossed to <em>mCherry-ATG8f</em> to generate the double transgenic line <em>mCherry-ATG8f MACP2-YFP.</em> The <em>macp2-1 eGFP-ATG8e</em> double mutant was obtained by crossing <em>macp2-1</em> to the <em>eGFP-ATG8e</em> line, and the transgenic plant <em>MACP2-</em>OEÂ <em>eGFP-ATG8e</em> (<em>MACP2-FLAG eGFP-ATG8e</em>) was produced by introduction of <em>MACP2-FLAG</em> into <em>eGFP-ATG8e</em> line.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Co-immunoprecipitation (Co-IP) and BiFC assays</h3>
<p id="Par41">Preparation and transfection of Arabidopsis mesophyll protoplasts were performed according to Yoo et al. (<a href="#CR74" class="usa-link" aria-describedby="CR74">2007</a>). Protoplasts isolated from the rosettes of 4-week-old plants were transfected with the indicated plasmids and cultured for 16Â h in the light before protein extraction. For the Co-IP assays, the transfected protoplasts were centrifuged and collected and lysed in 200 Î¼L immunoprecipitation (IP) buffer (10Â mM HEPES at pH 7.4, 150Â mM NaCl, 2Â mM EDTA, and 10% [v/v] glycerol) with 0.5% (v/v) Triton X-100. A 10% volume of total lysate was used as input, and the remainder was mixed with 300 Î¼L IP buffer without Triton X-100 and incubated with FLAG affinity beads (Sigma-Aldrich) for 4Â h at 4Â Â°C. The beads were centrifuged, collected, and washed six times with IP buffer containing 0.1% (v/v) Triton X-100, followed by addition of SDS-PAGE loading buffer and heating at 95Â Â°C for 5Â min before immunoblot analysis.</p>
<p id="Par42">For the BiFC assay, the appropriate pairs of <em>nYFP</em> and <em>cYFP</em> plasmids were transfected in leaf mesophyll protoplasts prepared from wild-type rosettes, incubated for 12Â h under continuous light conditions to allow transgene expression, and then incubated with 100Â Î¼M benzo [1,2,3] thiadiazole-7-carbothioic acid S-methyl ester (BTH) and 1Â Î¼M ConA for 6Â h for treated cells. The YFP fluorescence was detected by confocal microscopy (ZEISS 880) with an excitation wavelength of 514Â nm and emission collected with a bandpass infrared (IR) filter from 524 to 550Â nm.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Measurement of chlorophyll content</h3>
<p id="Par43">Chlorophyll contents were determined as described previously (Qi et al. <a href="#CR50" class="usa-link" aria-describedby="CR50">2022</a>). Arabidopsis leaves were harvested at the natural senescence stage or during carbon or nitrogen starvation at different growth periods. Total chlorophylls were extracted by incubating the samples in 2Â mL anhydrous ethanol in the dark at 4Â Â°C for 48Â h. Absorbance was determined at 664 and 647Â nm, and the total chlorophyll content was measured and normalized to fresh weight per sample.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Measurement of phytohormone contents</h3>
<p id="Par44">Phytohormones were extracted and their concentrations measured as described previously (Pan et al. <a href="#CR46" class="usa-link" aria-describedby="CR46">2010</a>; Chen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2015a</a>). Powdered tissue was weighed in a 2-mL centrifuge tube and extracted in 800 Î¼L extraction buffer (2-propanol/water/concentrated HCl [2:1:0.005, v/v/v]) with internal standards (10Â ng d4-salicylic acid [SA] and 10Â ng d5-jasmonic acid [JA]; Sigma-Aldrich). The mixtures were mixed with shaking at 100Â g for 30Â min at 4Â Â°C, after which 1Â mL dichloromethane was added and shaking continued for an additional 30Â min at 4Â Â°C. The samples were then centrifuged at 13,000Â g for 10Â min at 4Â Â°C. One milliliter from the lower phase was collected and dried using a nitrogen evaporator with nitrogen flow. The samples were dissolved in 200 Î¼L 60% (v/v) methanol in sterile ultrapure water. Quantitative detection and analysis of phytohormones were performed with a chromatograph (Shimadzu) coupled to a mass spectrometer (Triple TOF 5600 [AB SCIEX]) (Chen et al. <a href="#CR7" class="usa-link" aria-describedby="CR7">2015a</a>).</p></section><section id="Sec18"><h3 class="pmc_sec_title">Measurement of H<sub>2</sub>O<sub>2</sub> content</h3>
<p id="Par45">H<sub>2</sub>O<sub>2</sub> was extracted from the leaves of 4-, 5-, and 6-week-old Arabidopsis plants using an Amplex Red Hydrogen Peroxide/Peroxidase Assay Kit (Invitrogen) according to the manufacturerâ€™s instructions, as described previously (Pucciariello et al. <a href="#CR47" class="usa-link" aria-describedby="CR47">2012</a>). Briefly, 50 Î¼L of extracted supernatant and 50 Î¼L of the reaction working solution provided in the kit were transferred to a transparent plate and incubated at room temperature in darkness for 30Â min. The absorption was determined at 560Â nm on a multimode reader. A standard curve was generated by diluting the H<sub>2</sub>O<sub>2</sub> working solution provided with the kit to concentrations of 2, 4, 6, 8, 10, 12, 15, or 20Â Î¼M in a final volume of 50 Î¼L 1â€‰Ã—â€‰reaction buffer. Each dilution was mixed with 50 Î¼L reaction working solution and incubated as above.</p></section><section id="Sec19"><h3 class="pmc_sec_title">RNA extraction and RT-qPCR</h3>
<p id="Par46">Total RNA was extracted from the leaves of 4-, 5-, and 6-week-old Arabidopsis plants using a HiPure Plant RNA Mini kit (Magen) according to the manufacturerâ€™s instructions. The isolated total RNA was converted into first-strand cDNA with a HiScript II QRT Super Mix kit with gDNA Wiper (Vazyme). Quantitative PCR (qPCR) was performed with ChamQ SYBR Color qPCR Master Mix (Vazyme) on a StepOne Plus Real-time PCR System (Applied Biosystems).</p>
<p id="Par47">qPCR was performed with the following settings: 95Â Â°C for 5Â min followed by 40 cycles of 95Â Â°C for 15Â s, 55Â Â°C for 15Â s, and 72Â Â°C for 30Â s, and a subsequent standard dissociation protocol to validate the presence of a unique polymerase chain reaction (PCR) product. For calculation of the relative expression levels of different genes, three technical replicates were performed per reaction. The delta of threshold cycle (â–³Ct) values were calculated by subtracting the arithmetic mean Ct values of the targets from the normalizing <em>ACTIN2</em>. The relative transcript level (2<sup>âˆ’â–³â–³Ct</sup>) was calculated from three independent experiments. The gene-specific primers used for qPCR are listed in Table S1.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Protein extraction and immunoblot analysis</h3>
<p id="Par48">For total protein extraction, plant tissues were ground in liquid nitrogen and homogenized in ice-cold protein extraction buffer (50Â mM sodium phosphate [pH 7.0], 200Â mM NaCl, 10Â mM MgCl<sub>2</sub>, 0.2% [v/v] Î²-mercaptoethanol, and 10% [v/v] glycerol) with protease inhibitor cocktail (Roche, 04693132001). The samples were placed on ice for 30Â min and centrifuged at 4Â Â°C for 10Â min at 12,000Â g. The supernatant was transferred to a new microfuge tube before electrophoresis. For total protein extraction from protoplasts, collected cells were flash-frozen in liquid nitrogen and extracted in IP buffer.</p>
<p id="Par49">For immunoblot analysis, the supernatants from protein extracts were subjected to SDS-PAGE and transferred to a Hybond-C membrane (Amersham). Specific anti-HA (cat. no. H6533, 1:5,000; Sigma-Aldrich), anti-FLAG (cat. no. A8592, 1:5,000; Sigma-Aldrich), anti-eGFP (cat. no. M20004, 1:3,000; Abmart), and anti-ACTIN (cat. no. 58169; 1:5,000; Cell Signaling Technology) antibodies were used for immunoblotting analysis. Quantification of the immunoblot signal was determined with ImageJ software.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Microscopy analyses</h3>
<p id="Par50">To determine the effect of <em>macp2</em> and <em>MACP2</em>-OE plants on autophagosome formation during carbon starvation, 7-d-old <em>eGFP-ATG8e</em>, <em>macp2-1 eGFP-ATG8e</em>, and <em>MACP2-</em>OE<em> eGFP-ATG8e</em> seedlings grown on half-strength MS medium were transferred to sucrose-free MS medium with or without 1Â Î¼M ConA and incubated in constant dark conditions for fixed-C starvation treatment. For H<sub>2</sub>O<sub>2</sub> and BTH treatment, 7-d-old <em>eGFP-ATG8e</em>, <em>macp2-1 eGFP-ATG8e</em> and <em>MACP2-</em>OE<em> eGFP-ATG8e</em> seedlings were transferred to half-strength MS liquid medium with or without 100Â Î¼M H<sub>2</sub>O<sub>2</sub> or BTH (SA agonist, benzo [1,2,3] thiadiazole-7-carbothioic acid S-methyl ester) for 12Â h or 8Â h, respectively.</p>
<p id="Par51">For the subcellular localization of MACP2-YFP and the colocalization of MACP2-YFP with mCherry-ATG8f in response to nutrient starvation, 7-d-old <em>MACP2-YFP</em> and <em>MACP2-YFP mCherry-ATG8f</em> seedlings grown on solid half-strength MS medium were transferred to sucrose-free liquid half-strength MS medium (âˆ’â€‰C) containing 1Â Î¼M ConA and incubated in constant dark conditions for fixed-C starvation treatment, or seedlings were transferred to nitrogen-free liquid half-strength (âˆ’â€‰N) containing 1Â Î¼M ConA for indicated times. The eGFP, YFP and mCherry signals were detected by confocal microscopy (ZEISS 880) with an excitation wavelength of 488Â nm (eGFP), 514Â nm (YFP) or 561Â nm (mCherry) and emission collected with a bandpass infrared (IR) filter from 500 to 530Â nm (GFP, YFP), or 560 to 610Â nm (mCherry).</p></section><section id="Sec23"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par52">Data presented in this study are meansâ€‰Â±â€‰standard deviation (SD) of three independent experiments unless indicated otherwise. Significant differences within or between groups were assessed by one-way analysis of variance (ANOVA) with Tukeyâ€™s honestly significant difference (HSD) test via STAT software, and a <em>P</em>-valueâ€‰&lt;â€‰0.05 was considered significant.</p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Accession numbers</h2>
<p id="Par53">Sequence data from this article can be found in the Arabidopsis Genome Initiative database under the following accession numbers: <em>MACP1</em> (AT1G14780), <em>MACP2</em> (AT4G24290), <em>NSL1</em> (AT1G28380), <em>CAD1</em> (AT1G29690), <em>EXO70D1</em> (AT1G72470),</p>
<p id="Par54"><em>EXO70D2</em> (AT1G54090), <em>EXO70D3</em> (AT3G14090), <em>ATG8e</em> (AT2G45170), <em>ATG8f</em> (AT4G16520), <em>ATG5</em> (AT5G17290), <em>EDS1</em> (AT3G48090), <em>PAD4</em> (AT3G52430), <em>SAG13</em> (AT2G29350), <em>SAG29</em> (AT5G13170), <em>SAG113</em> (AT5G59220), <em>SEN4</em> (AT4G30270), <em>PR1</em> (At2g14610), <em>PR5</em> (At1g75040), <em>ST1</em> (AT1G79230), <em>EDR2</em> (AT4G19040), and <em>WRKY53</em> (AT4G23810).</p></section><section id="Sec25"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12367599/bin/44307_2025_78_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">44307_2025_78_MOESM1_ESM.zip</a><sup> (14.3MB, zip) </sup><p>Supplementary Material 1:Â Fig. S1.Structures of MACP proteins from Arabidopsis. Structures of NSL1 (AT1G28380), MACP1 (AT1G14780), and CAD1 (AT1G29690) were predicted by Alphafold and visualized using PyMOL software. AIM located within the proteins were analyzed by iLIR database. The predicted AIM is shown in red.Â Fig. S2. The accelerated senescence phenotypes of <em>MACP2-</em>OE plants are dependent on the salicylic acid pathway. a Representative photograph of Wild-type (WT), <em>MACP2-</em>OE-2,<em>eds1-22</em>,<em>pad4</em>, OE-2<em>eds1</em>, OE<em>-2</em><em>pad4 seedlings</em> in response to nitrogen starvation (Nâˆ’). For nitrogen starvation treatment, 7-d-old WT, OE-2,<em>eds1-22, pad4</em>, OE-2<em>eds1</em>, and OE-2<em>pad4</em>seedlings grown on half-strength solid MS medium were transferred to N-rich (N+) or N-free (Nâˆ’) medium and photographed at 5 d into treatment. b Relative chlorophyll contents of 7-d-old wild-type (WT), <em>MACP2-</em>OE-1,<em>eds1-22</em>,<em>pad4</em>,OE-1<em>eds1-22</em>, and OE-1<em>pad4</em>plants seedlings under nitrogen deficiency, expressed as a percentage relative to control plants or seedlings. For each experiment,20 seedlings were used per genotype. All experiments were performed as three biological replicates, each with similar results. Relative chlorophyll contents are means Â± SD (<em>n</em> = 3) calculated from three biological replicates. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em> &lt; 0.05.Â Fig. S3. The starvation sensitive phenotypes of <em>MACP2-</em>OE plants are rescued by GSH application. a Wild-type (WT), <em>MACP2-</em>OE (OE-1 and OE-2), and <em>atg5-1</em> seedlings grown on half-strength solid MS medium for 7 d. The seedlings were transferred to solid sucrose-rich medium (CK) or sucrose-free medium (Câˆ’) with or without 500 Î¼M GSH and incubated under normal light/dark conditions or continuous dark conditions for 12 d, respectively, followed by recovery under normal growth conditions for 5 d. b Relative chlorophyll contents of 7-d-old WT, <em>MACP2-</em>OE (OE-1 and OE-2), and <em>atg5-1</em>plants in response to carbon starvation (a) for 12 d, followed by recovery under normal growth conditions for 5 d. For each experiment, 20 seedlings were used per genotype. All experiments were performed as three biological replicates, each with similar results. Relative chlorophyll contents are means Â± SD (<em>n</em> = 3) calculated from three biological replicates. Different lowercase letters indicate significant differences between groups as determined by one-way ANOVA, <em>P</em> &lt; 0.05.Â Fig. S4.Knockout of <em>MACP2</em> partially rescues the hypersensitive phenotypes of <em>atg5-1</em>to nitrogen deficiency. a and b Representative photographs (a) and relative chlorophyll contents (b) of wild-type (WT), <em>macp2-1</em>, <em>macp2-2</em>,<em>atg5-1</em>,<em>macp2-1Â atg5-1</em>,and <em>macp2-2Â atg5-1</em>double mutant seedlings in response to nitrogen starvation. 7-d-old WT<em>macp2</em>,<em>atg5-1</em>,<em>macp2-1Â atg5-1</em>,and <em>macp2-2Â atg5</em><em>-1</em>double mutant seedlings grown on half-strength solid MS medium were transferred to N-rich (N+) or N-deficient (Nâˆ’) medium and photographed at 5 d into treatment. For each experiment, 20 seedlings were used per genotype. The experiments were performed as three biological replicates, each with similar results. Data are means Â± SD (<em>n</em> = 3) calculated from three biological replicates. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em> &lt; 0.05.Â Fig. S5. MACP2 modulates H<sub>2</sub>O<sub>2</sub>â€“ or BTHâ€“induced autophagosome formation. a and c Confocal analysis showing eGFP-ATG8e labeled autophagosomes in the root cells of wild-type (WT),<em>macp2-1</em>, and <em>MACP2</em>-OE seedlings. Seven-d-old<em>eGFP-ATG8e</em>, <em>macp2-1 eGFP-ATG8e</em>, and <em>MACP2-</em>OE<em>eGFP-ATG8e</em> (<em>MACP2-FLAG eGFP-ATG8e</em>) seedlings were transferred to half-strength MS liquid medium with or without 100 Î¼M H<sub>2</sub>O<sub>2</sub> or BTH (SA agonist, benzo [1,2,3] thiadiazole-7-carbothioic acid S-methyl ester) for 12 h or 8 h, respectively. The formation of autophagosomes was visualized by fluorescence confocal microscopy. Scale bars, 10 Î¼m. banddNumber of puncta per root section in root cells from the <em>eGFP-ATG8e</em>, <em>macp2-1 eGFP-ATG8e</em>,and<em>MACP2</em>OE<em> eGFP-ATG8e</em>in (a and c). Data are means Â± SD (<em>n</em>= 3) calculated from three biological replicates. For each experiment, 15 sections were used for calculation per genotype. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em> &lt; 0.05.Â Fig. S6. Colocalization of MACP2-YFP with mCherry-ATG8f in response to nitrogen starvation. A <em>MACP2-YFP</em> plant was crossed to <em>mCherry-ATG8f</em> to generate the double transgenic line <em>mCherry-ATG8f MACP2-YFP</em>. Colocalization assay of MACP2-YFP and mCherry-ATG8f in transgenic plants <em>mCherry-ATG8f MACP2-YFP</em>after 12 h of nitrogen starvation. YFP/mCherry was excited at 514/561 nm and the emissions were collected using a 554/601 nm band-pass emission filter on ZEISS LSM 880 confocal laser scanning microscope. Scale bars, 10 Î¼m.Â Fig. S7. Identification of <em>MACP2-FLAG</em>transgenic plants. a and b Relative <em>MACP2</em>transcript levels (a) and accumulation of MACP2-FLAG(b) in 4-week-old wild-type (WT) and <em>pUBQ10:MACP2-FLAG</em>lines, as determined by RT-qPCR or immunoblot analysis, respectively. Anti-HA and anti-Actin antibodies were used for immunoblotting. Actin is shown as a loading control. The experiments were performed three times (biological replicates), each with similar results, and representative data from one replicate are shown. Data are means Â± SD (<em>n</em> = 3) calculated from three technical replicates. Asterisks indicate significant differences from the WT (**<em>P</em> &lt; 0.01, *<em>P</em> &lt; 0.05 by Studentâ€™s <em>t</em> test).Â Fig. S8. Protein abundance of MACP2 and MACP2 M1 in wild type and <em>atg8e</em> mutant. The <em>GFP-FLAG</em> and <em>MACP2-HA</em> or <em>MACP2 M1-HA</em> plasmids were co-transfected into protoplasts isolated from wild type (WT) and <em>atg8e</em> mutant. After overnight culture, protoplasts were treated with 50 Î¼M cycloheximide (CHX) under darkness for the indicated times (0, 4, 6, or 8 h). The experiments were performed as three biological replicates, each with similar results. The abundance of eGFP-FLAG was used as a control. The relative quantification of MACP2-HA and MACP2 MA-HA proteins were normalized to the corresponding eGFP-FLAG control and then compared to its level at 0 h. Data are means Â± SD (<em>n</em> = 3) calculated from three biological replicates. The numbers on the left indicate the molecular weight (kD) of each size marker. hpt, h post-treatment.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12367599/bin/44307_2025_78_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (10.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12367599/bin/44307_2025_78_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (414KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the Arabidopsis Biological Resources Center (<a href="http://www.arabidopsis.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.arabidopsis.org</a>) for providing <em>macp2</em> mutant seeds, Prof. Xing-Liang Hou (South China Botanical Garden, Chinese Academy of Sciences) for providing <em>eds1</em>, Prof. Ming-Yi Bai (Shandong University) for providing <em>p35S::HyPer</em>, and Prof. Nan Yao (Sun Yat-Sen University) for providing <em>pad4</em>. This work was supported by the National Natural Science Foundation of China (Projects 32321163646, 32000215 and 32370293) and the Natural Science Foundation of Guangdong Province (Projects 2023A1515011779, 2023A1515012038, andÂ 2024A1515012940Â ).</p>
<section id="FPar1"><h3 class="pmc_sec_title">One-sentence summary</h3>
<p id="Par55">An Arabidopsis member of the MEMBRANE ATTACK COMPLEX/PERFORIN family, MACP2, is a key regulator of autophagy, controlling the influx of extracellular ROS.</p></section></section><section id="notes1"><h2 class="pmc_sec_title">Authorsâ€™ contributions</h2>
<p>Q.F.C., L.J.Y., S.X., and D.X.X. conceived and designed the study. Y.Z., X.Z., T.C.Q. Z.H.W., Y.W., L.N.W., and Y.L.Z. performed the experiments. Q.F.C., X.Z., and Z.H.W. conducted result analysis and data organization. S.X., Y.Z., Q.F.C., X.Z., H.J.H., L.W.J., D.X.X., and T.C.Q. discussed and interpreted the results. S.X., Y.Z., Q.F.C., X.Z., and L.J.Y. wrote the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>The original data and materials in the paper are available via contacting the corresponding author.</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par56">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par57">All authors agreed to the publication of this manuscript.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par58">Authors S.X. and L.J. are members of the Editorial Board for Advanced Biotechnology, and all these authors are not involved in the journalâ€™s review of and decisions related to this manuscript.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Ying Zhou, Xue Zhang, Tiancong Qi and Zi-Han Wang contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Lu-Jun Yu, Email: yulujun2@mail.sysu.edu.cn.</p>
<p>Qin-Fang Chen, Email: chenqf3@mail.sysu.edu.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ol class="ref-list font-sm">
<li id="CR1">
<cite>Acheampong AK, Shanks C, Cheng CY, Schaller GE, Dagdas Y, Kieber JJ. EXO70D isoforms mediate selective autophagic degradation of type-A ARR proteins to regulate cytokinin sensitivity. Proc Natl Acad Sci U S A. 2020;117:27034â€“43.
</cite> [<a href="https://doi.org/10.1073/pnas.2013161117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7604425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33051300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Acheampong%20AK,%20Shanks%20C,%20Cheng%20CY,%20Schaller%20GE,%20Dagdas%20Y,%20Kieber%20JJ.%20EXO70D%20isoforms%20mediate%20selective%20autophagic%20degradation%20of%20type-A%20ARR%20proteins%20to%20regulate%20cytokinin%20sensitivity.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202020;117:27034%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<cite>Asada Y, Yamamoto M, Tsutsui T, Yamaguchi J. The <em>Arabidopsis</em> NSL2 negatively controls systemic acquired resistance via hypersensitive response. Plant Biotechnol. 2011;28:9â€“15.</cite> [<a href="https://scholar.google.com/scholar_lookup?Asada%20Y,%20Yamamoto%20M,%20Tsutsui%20T,%20Yamaguchi%20J.%20The%20Arabidopsis%20NSL2%20negatively%20controls%20systemic%20acquired%20resistance%20via%20hypersensitive%20response.%20Plant%20Biotechnol.%202011;28:9%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<cite>Avila-Ospina L, Moison M, Yoshimoto K, Masclaux-Daubresse C. Autophagy, plant senescence, and nutrient recycling. J Exp Bot. 2014;65:3799â€“811.
</cite> [<a href="https://doi.org/10.1093/jxb/eru039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24687977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Avila-Ospina%20L,%20Moison%20M,%20Yoshimoto%20K,%20Masclaux-Daubresse%20C.%20Autophagy,%20plant%20senescence,%20and%20nutrient%20recycling.%20J%20Exp%20Bot.%202014;65:3799%E2%80%93811." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<cite>Bakopoulos D, Whisstock JC, Johnson TK. Control of growth factor signalling by MACPF proteins. Biochem Soc Trans. 2019;47:801â€“10.
</cite> [<a href="https://doi.org/10.1042/BST20180179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31209154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bakopoulos%20D,%20Whisstock%20JC,%20Johnson%20TK.%20Control%20of%20growth%20factor%20signalling%20by%20MACPF%20proteins.%20Biochem%20Soc%20Trans.%202019;47:801%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<cite>Bassham DC, Laporte M, Marty F, Moriyasu Y, Ohsumi Y, Olsen LJ, Yoshimoto K. Autophagy in development and stress responses of plants. Autophagy. 2006;2:2â€“11.
</cite> [<a href="https://doi.org/10.4161/auto.2092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16874030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bassham%20DC,%20Laporte%20M,%20Marty%20F,%20Moriyasu%20Y,%20Ohsumi%20Y,%20Olsen%20LJ,%20Yoshimoto%20K.%20Autophagy%20in%20development%20and%20stress%20responses%20of%20plants.%20Autophagy.%202006;2:2%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<cite>Castro B, Citterico M, Kimura S, Stevens DM, Wrzaczek M, Coaker G. Stress-induced reactive oxygen species compartmentalization, perception and signalling. Nat Plants. 2021;7:403â€“12.
</cite> [<a href="https://doi.org/10.1038/s41477-021-00887-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8751180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33846592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Castro%20B,%20Citterico%20M,%20Kimura%20S,%20Stevens%20DM,%20Wrzaczek%20M,%20Coaker%20G.%20Stress-induced%20reactive%20oxygen%20species%20compartmentalization,%20perception%20and%20signalling.%20Nat%20Plants.%202021;7:403%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<cite>Chen L, Liao B, Qi H, Xie LJ, Huang L, Tan WJ, Zhai N, Yuan LB, Zhou Y, Yu LJ, Chen QF, Shu WS, Xiao S. Autophagy contributes to regulation of the hypoxia response during submergence in <em>Arabidopsis thaliana</em>. Autophagy. 2015a;11:2233â€“46.
</cite> [<a href="https://doi.org/10.1080/15548627.2015.1112483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4835207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26566261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20L,%20Liao%20B,%20Qi%20H,%20Xie%20LJ,%20Huang%20L,%20Tan%20WJ,%20Zhai%20N,%20Yuan%20LB,%20Zhou%20Y,%20Yu%20LJ,%20Chen%20QF,%20Shu%20WS,%20Xiao%20S.%20Autophagy%20contributes%20to%20regulation%20of%20the%20hypoxia%20response%20during%20submergence%20in%20Arabidopsis%20thaliana.%20Autophagy.%202015a;11:2233%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<cite>Chen QF, Xu L, Tan WJ, Chen L, Qi H, Xie LJ, Chen MX, Liu BY, Yu LJ, Yao N, Zhang JH, Shu W, Xiao S. Disruption of the Arabidopsis defense regulator genes SAG101, EDS1, and PAD4 confers enhanced freezing tolerance. Mol Plant. 2015b;8:1536â€“49.
</cite> [<a href="https://doi.org/10.1016/j.molp.2015.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5321072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26149542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20QF,%20Xu%20L,%20Tan%20WJ,%20Chen%20L,%20Qi%20H,%20Xie%20LJ,%20Chen%20MX,%20Liu%20BY,%20Yu%20LJ,%20Yao%20N,%20Zhang%20JH,%20Shu%20W,%20Xiao%20S.%20Disruption%20of%20the%20Arabidopsis%20defense%20regulator%20genes%20SAG101,%20EDS1,%20and%20PAD4%20confers%20enhanced%20freezing%20tolerance.%20Mol%20Plant.%202015b;8:1536%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<cite>Chen P, Jian H, Wei F, Gu L, Hu T, Lv X, Guo X, Lu J, Ma L, Wang H, Wu A, Mao G, Yu S, Wei H. Phylogenetic analysis of the membrane attack complex/perforin domain-containing proteins in <em>Gossypium</em> and the role of <em>GhMACPF26</em> in cotton under cold stress. Front Plant Sci. 2021;12: 684227.
</cite> [<a href="https://doi.org/10.3389/fpls.2021.684227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8641546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34868097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20P,%20Jian%20H,%20Wei%20F,%20Gu%20L,%20Hu%20T,%20Lv%20X,%20Guo%20X,%20Lu%20J,%20Ma%20L,%20Wang%20H,%20Wu%20A,%20Mao%20G,%20Yu%20S,%20Wei%20H.%20Phylogenetic%20analysis%20of%20the%20membrane%20attack%20complex/perforin%20domain-containing%20proteins%20in%20Gossypium%20and%20the%20role%20of%20GhMACPF26%20in%20cotton%20under%20cold%20stress.%20Front%20Plant%20Sci.%202021;12:%20684227." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<cite>Chung T, Phillips AR, Vierstra RD. ATG8 lipidation and ATG8-mediated autophagy in Arabidopsis require ATG12 expressed from the differentially controlled ATG12A and ATG12B loci. Plant J. 2010;62:483â€“93.
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2010.04166.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20136727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chung%20T,%20Phillips%20AR,%20Vierstra%20RD.%20ATG8%20lipidation%20and%20ATG8-mediated%20autophagy%20in%20Arabidopsis%20require%20ATG12%20expressed%20from%20the%20differentially%20controlled%20ATG12A%20and%20ATG12B%20loci.%20Plant%20J.%202010;62:483%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<cite>Clough SJ, Bent AF. Floral dip: a simplified method for <em>Agrobacterium</em>-mediated transformation of <em>Arabidopsis thaliana</em>. Plant J. 1998;16:735â€“43.
</cite> [<a href="https://doi.org/10.1046/j.1365-313x.1998.00343.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10069079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clough%20SJ,%20Bent%20AF.%20Floral%20dip:%20a%20simplified%20method%20for%20Agrobacterium-mediated%20transformation%20of%20Arabidopsis%20thaliana.%20Plant%20J.%201998;16:735%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<cite>Essick EE, Flora S. Oxidative stress and autophagy in cardiac disease, neurological disorders, aging and cancer. Oxid Med Cell Longev. 2010;3:168â€“77.
</cite> [<a href="https://doi.org/10.4161/oxim.3.3.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2952075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20716941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Essick%20EE,%20Flora%20S.%20Oxidative%20stress%20and%20autophagy%20in%20cardiac%20disease,%20neurological%20disorders,%20aging%20and%20cancer.%20Oxid%20Med%20Cell%20Longev.%202010;3:168%E2%80%9377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<cite>Fukunaga S, Sogame M, Hata M, Singkaravanit-Ogawa S, Pislewska-Bednarek M, Onozawa-Komori M, Nishiuchi T, Hiruma K, Saitoh H, Terauchi R, Kitakura S, Inoue Y, Bednarek P, Schulze-Lefert P, Takano Y. Dysfunction of <em>Arabidopsis</em> MACPF domain protein activates programmed cell death via tryptophan metabolism in MAMP-triggered immunity. Plant J. 2017;89:381â€“93.
</cite> [<a href="https://doi.org/10.1111/tpj.13391" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27711985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fukunaga%20S,%20Sogame%20M,%20Hata%20M,%20Singkaravanit-Ogawa%20S,%20Pislewska-Bednarek%20M,%20Onozawa-Komori%20M,%20Nishiuchi%20T,%20Hiruma%20K,%20Saitoh%20H,%20Terauchi%20R,%20Kitakura%20S,%20Inoue%20Y,%20Bednarek%20P,%20Schulze-Lefert%20P,%20Takano%20Y.%20Dysfunction%20of%20Arabidopsis%20MACPF%20domain%20protein%20activates%20programmed%20cell%20death%20via%20tryptophan%20metabolism%20in%20MAMP-triggered%20immunity.%20Plant%20J.%202017;89:381%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<cite>Grondin A, Rodrigues O, Verdoucq L, Merlot S, Leonhardt N, Maurel C. Aquaporins contribute to ABA-triggered stomatal closure through OST1-mediated phosphorylation. Plant Cell. 2015;27:1945â€“54.
</cite> [<a href="https://doi.org/10.1105/tpc.15.00421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4531361/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26163575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grondin%20A,%20Rodrigues%20O,%20Verdoucq%20L,%20Merlot%20S,%20Leonhardt%20N,%20Maurel%20C.%20Aquaporins%20contribute%20to%20ABA-triggered%20stomatal%20closure%20through%20OST1-mediated%20phosphorylation.%20Plant%20Cell.%202015;27:1945%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<cite>Guan B, Lin Z, Liu D, Li C, Zhou Z, Mei F, Li J, Deng X. Effect of waterlogging-induced autophagy on programmed cell death in Arabidopsis roots. Front Plant Sci. 2019;10: 468.
</cite> [<a href="https://doi.org/10.3389/fpls.2019.00468" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6470631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31031792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guan%20B,%20Lin%20Z,%20Liu%20D,%20Li%20C,%20Zhou%20Z,%20Mei%20F,%20Li%20J,%20Deng%20X.%20Effect%20of%20waterlogging-induced%20autophagy%20on%20programmed%20cell%20death%20in%20Arabidopsis%20roots.%20Front%20Plant%20Sci.%202019;10:%20468." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<cite>Guo Y, Ren G, Zhang K, Li Z, Miao Y, Guo H. Leaf senescence: progression, regulation, and application. Mol Hortic. 2021;1:5.
</cite> [<a href="https://doi.org/10.1186/s43897-021-00006-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10509828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37789484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20Y,%20Ren%20G,%20Zhang%20K,%20Li%20Z,%20Miao%20Y,%20Guo%20H.%20Leaf%20senescence:%20progression,%20regulation,%20and%20application.%20Mol%20Hortic.%202021;1:5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<cite>Halliwell B. Biochemistry of oxidative stress. Biochem Soc Trans. 2007;35:1147â€“50.
</cite> [<a href="https://doi.org/10.1042/BST0351147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17956298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Halliwell%20B.%20Biochemistry%20of%20oxidative%20stress.%20Biochem%20Soc%20Trans.%202007;35:1147%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<cite>Hanaoka H, Noda T, Shirano Y, Kato T, Hayashi H, Shibata D, Tabata S, Ohsumi Y. Leaf senescence and starvation-induced chlorosis are accelerated by the disruption of an Arabidopsis autophagy gene. Plant Physiol. 2002;129:1181â€“93.
</cite> [<a href="https://doi.org/10.1104/pp.011024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC166512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12114572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanaoka%20H,%20Noda%20T,%20Shirano%20Y,%20Kato%20T,%20Hayashi%20H,%20Shibata%20D,%20Tabata%20S,%20Ohsumi%20Y.%20Leaf%20senescence%20and%20starvation-induced%20chlorosis%20are%20accelerated%20by%20the%20disruption%20of%20an%20Arabidopsis%20autophagy%20gene.%20Plant%20Physiol.%202002;129:1181%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<cite>He Y, Gao J, Luo M, Gao C, Lin Y, Wong HY, Cui Y, Zhuang X, Jiang L. VAMP724 and VAMP726 are involved in autophagosome formation in Arabidopsis thaliana. Autophagy. 2023;13:19(5):1406â€“1423.</cite> [<a href="https://doi.org/10.1080/15548627.2022.2127240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10240985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36130166/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<cite>HernÃ¡ndez-Barrera A, Quinto C, Johnson EA, Wu HM, Cheung AY, CÃ¡rdenas L. Using hyper as a molecular probe to visualize hydrogen peroxide in living plant cells: a method with virtually unlimited potential in plant biology. Methods Enzymol. 2013;527:275â€“90.
</cite> [<a href="https://doi.org/10.1016/B978-0-12-405882-8.00015-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23830637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hern%C3%A1ndez-Barrera%20A,%20Quinto%20C,%20Johnson%20EA,%20Wu%20HM,%20Cheung%20AY,%20C%C3%A1rdenas%20L.%20Using%20hyper%20as%20a%20molecular%20probe%20to%20visualize%20hydrogen%20peroxide%20in%20living%20plant%20cells:%20a%20method%20with%20virtually%20unlimited%20potential%20in%20plant%20biology.%20Methods%20Enzymol.%202013;527:275%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<cite>Holmes DR, Bredow M, Thor K, Pascetta SA, Sementchoukova I, Siegel KR, Zipfel C, Monaghan J. A novel allele of the Arabidopsis thaliana MACPF protein CAD1 results in deregulated immune signaling. Genetics. 2021;217:iyab022.</cite> [<a href="https://doi.org/10.1093/genetics/iyab022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8049551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33779749/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<cite>Huang L, Guo H. Acetylation modifcation in the regulation of macroautophagy. Adv Biotech. 2024;2: 19.</cite> [<a href="https://doi.org/10.1007/s44307-024-00027-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11740868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39883319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20L,%20Guo%20H.%20Acetylation%20modifcation%20in%20the%20regulation%20of%20macroautophagy.%20Adv%20Biotech.%202024;2:%2019." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<cite>Huang L, Yu LJ, Zhang X, Fan B, Wang FZ, Dai YS, Qi H, Zhou Y, Xie LJ, Xiao S. Autophagy regulates glucose-mediated root meristem activity by modulating ROS production in Arabidopsis. Autophagy. 2019;15:407â€“22.
</cite> [<a href="https://doi.org/10.1080/15548627.2018.1520547" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6351127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30208757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20L,%20Yu%20LJ,%20Zhang%20X,%20Fan%20B,%20Wang%20FZ,%20Dai%20YS,%20Qi%20H,%20Zhou%20Y,%20Xie%20LJ,%20Xiao%20S.%20Autophagy%20regulates%20glucose-mediated%20root%20meristem%20activity%20by%20modulating%20ROS%20production%20in%20Arabidopsis.%20Autophagy.%202019;15:407%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<cite>Kaufmann A, Wollert T. Scaffolding the expansion of autophagosomes. Autophagy. 2014;10:1343â€“5.
</cite> [<a href="https://doi.org/10.4161/auto.28980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4203560/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24963637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaufmann%20A,%20Wollert%20T.%20Scaffolding%20the%20expansion%20of%20autophagosomes.%20Autophagy.%202014;10:1343%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<cite>Kaufmann A, Beier V, Franquelim HG, Wollert T. Molecular mechanism of autophagic membrane-scaffold assembly and disassembly. Cell. 2014;156:469â€“81.
</cite> [<a href="https://doi.org/10.1016/j.cell.2013.12.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24485455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaufmann%20A,%20Beier%20V,%20Franquelim%20HG,%20Wollert%20T.%20Molecular%20mechanism%20of%20autophagic%20membrane-scaffold%20assembly%20and%20disassembly.%20Cell.%202014;156:469%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<cite>Kellner R, de la Concepcion JC, Maqbool A, Kamoun S, Dagdas YF. ATG8 expansion: a driver of selective autophagy diversification? Trends Plant Sci. 2017;22:204â€“14.
</cite> [<a href="https://doi.org/10.1016/j.tplants.2016.11.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28038982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kellner%20R,%20de%20la%20Concepcion%20JC,%20Maqbool%20A,%20Kamoun%20S,%20Dagdas%20YF.%20ATG8%20expansion:%20a%20driver%20of%20selective%20autophagy%20diversification?%20Trends%20Plant%20Sci.%202017;22:204%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<cite>KrÃ¼ger C, Waldeck-Weiermair M, Kaynert J, Pokrant T, Komaragiri Y, Otto O, Michel T, Elsner M. AQP8 is a crucial H<sub>2</sub>O<sub>2</sub> transporter in insulin-producing RINm5F cells. Redox Biol. 2021;43: 101962.
</cite> [<a href="https://doi.org/10.1016/j.redox.2021.101962" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8082690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33892285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kr%C3%BCger%20C,%20Waldeck-Weiermair%20M,%20Kaynert%20J,%20Pokrant%20T,%20Komaragiri%20Y,%20Otto%20O,%20Michel%20T,%20Elsner%20M.%20AQP8%20is%20a%20crucial%20H2O2%20transporter%20in%20insulin-producing%20RINm5F%20cells.%20Redox%20Biol.%202021;43:%20101962." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<cite>Li F, Vierstra RD. Autophagy: a multifaceted intracellular system for bulk and selective recycling. Trends Plant Sci. 2012;17:526â€“37.
</cite> [<a href="https://doi.org/10.1016/j.tplants.2012.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22694835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20F,%20Vierstra%20RD.%20Autophagy:%20a%20multifaceted%20intracellular%20system%20for%20bulk%20and%20selective%20recycling.%20Trends%20Plant%20Sci.%202012;17:526%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<cite>Li JF, Chung HS, Niu Y, Bush J, Mc Cormack M, Sheen J. Comprehensive protein-based artificial microRNA screens for effective gene silencing in plants. Plant Cell. 2013;25:1507â€“22.
</cite> [<a href="https://doi.org/10.1105/tpc.113.112235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3694689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23645631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20JF,%20Chung%20HS,%20Niu%20Y,%20Bush%20J,%20Mc%20Cormack%20M,%20Sheen%20J.%20Comprehensive%20protein-based%20artificial%20microRNA%20screens%20for%20effective%20gene%20silencing%20in%20plants.%20Plant%20Cell.%202013;25:1507%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<cite>Li X, Liu Q, Feng H, Deng J, Zhang R, Wen J, Dong J, Wang T. Dehydrin MtCAS31 promotes autophagic degradation under drought stress. Autophagy. 2020;16:862â€“77.
</cite> [<a href="https://doi.org/10.1080/15548627.2019.1643656" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7144882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31362589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Liu%20Q,%20Feng%20H,%20Deng%20J,%20Zhang%20R,%20Wen%20J,%20Dong%20J,%20Wang%20T.%20Dehydrin%20MtCAS31%20promotes%20autophagic%20degradation%20under%20drought%20stress.%20Autophagy.%202020;16:862%E2%80%9377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<cite>Liao CY, Bassham DC. Combating stress: the interplay between hormone signaling and autophagy in plants. J Exp Bot. 2020;71:1723â€“33.
</cite> [<a href="https://doi.org/10.1093/jxb/erz515" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7067298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31725881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao%20CY,%20Bassham%20DC.%20Combating%20stress:%20the%20interplay%20between%20hormone%20signaling%20and%20autophagy%20in%20plants.%20J%20Exp%20Bot.%202020;71:1723%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<cite>Lin Z, Wang YL, Cheng LS, Zhou LL, Xu QT, Liu DC, Deng XY, Mei FZ, Zhou ZQ. Mutual regulation of ROS accumulation and cell autophagy in wheat roots under hypoxia stress. Plant Physiol Biochem. 2021;158:91â€“102.
</cite> [<a href="https://doi.org/10.1016/j.plaphy.2020.11.049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33302125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20Z,%20Wang%20YL,%20Cheng%20LS,%20Zhou%20LL,%20Xu%20QT,%20Liu%20DC,%20Deng%20XY,%20Mei%20FZ,%20Zhou%20ZQ.%20Mutual%20regulation%20of%20ROS%20accumulation%20and%20cell%20autophagy%20in%20wheat%20roots%20under%20hypoxia%20stress.%20Plant%20Physiol%20Biochem.%202021;158:91%E2%80%93102." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<cite>Liu Y, Bassham DC. Autophagy: pathways for self-eating in plant cells. Annu Rev Plant Biol. 2012;63:215â€“37.
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-042811-105441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22242963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Y,%20Bassham%20DC.%20Autophagy:%20pathways%20for%20self-eating%20in%20plant%20cells.%20Annu%20Rev%20Plant%20Biol.%202012;63:215%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<cite>Lukoyanova N, Hoogenboom BW, Saibil HR. The membrane attack complex, perforin and cholesterol-dependent cytolysin superfamily of pore-forming proteins. J Cell Sci. 2016;129:2125â€“33.
</cite> [<a href="https://doi.org/10.1242/jcs.182741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27179071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lukoyanova%20N,%20Hoogenboom%20BW,%20Saibil%20HR.%20The%20membrane%20attack%20complex,%20perforin%20and%20cholesterol-dependent%20cytolysin%20superfamily%20of%20pore-forming%20proteins.%20J%20Cell%20Sci.%202016;129:2125%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<cite>Marshall RS, Li F, Gemperline DC, Book AJ, Vierstra RD. Autophagic degradation of the 26S proteasome is mediated by the dual ATG8/ubiquitin receptor RPN10 in Arabidopsis. Mol Cell. 2015;58:1053â€“66.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2015.04.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4903074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26004230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marshall%20RS,%20Li%20F,%20Gemperline%20DC,%20Book%20AJ,%20Vierstra%20RD.%20Autophagic%20degradation%20of%20the%2026S%20proteasome%20is%20mediated%20by%20the%20dual%20ATG8/ubiquitin%20receptor%20RPN10%20in%20Arabidopsis.%20Mol%20Cell.%202015;58:1053%E2%80%9366." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<cite>Marshall RS, Hua Z, Mali S, McLoughlin F, Vierstra RD. ATG8-binding UIM proteins define a new class of autophagy adaptors and receptors. Cell. 2019;177:766â€“81.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.02.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6810650/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30955882/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marshall%20RS,%20Hua%20Z,%20Mali%20S,%20McLoughlin%20F,%20Vierstra%20RD.%20ATG8-binding%20UIM%20proteins%20define%20a%20new%20class%20of%20autophagy%20adaptors%20and%20receptors.%20Cell.%202019;177:766%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<cite>Mittler R, Zandalinas SI, Fichman Y, Van Breusegem F. Reactive oxygen species signalling in plant stress responses. Nat Rev Mol Cell Biol. 2022;23:663â€“79.
</cite> [<a href="https://doi.org/10.1038/s41580-022-00499-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35760900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mittler%20R,%20Zandalinas%20SI,%20Fichman%20Y,%20Van%20Breusegem%20F.%20Reactive%20oxygen%20species%20signalling%20in%20plant%20stress%20responses.%20Nat%20Rev%20Mol%20Cell%20Biol.%202022;23:663%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<cite>Moreno-Hagelsieb G, Vitug B, Medrano-Soto A, Saier MH Jr. The membrane attack complex/perforin superfamily. J Mol Microb Biotech. 2017;27:252â€“67.</cite> [<a href="https://doi.org/10.1159/000481286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5796659/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29145176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moreno-Hagelsieb%20G,%20Vitug%20B,%20Medrano-Soto%20A,%20Saier%20MH%20Jr.%20The%20membrane%20attack%20complex/perforin%20superfamily.%20J%20Mol%20Microb%20Biotech.%202017;27:252%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<cite>Morgan BP. Complement membrane attack on nucleated cells: resistance, recovery and non-lethal effects. Biochem J. 1989;264:1â€“14.
</cite> [<a href="https://doi.org/10.1042/bj2640001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1133540/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2690818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morgan%20BP.%20Complement%20membrane%20attack%20on%20nucleated%20cells:%20resistance,%20recovery%20and%20non-lethal%20effects.%20Biochem%20J.%201989;264:1%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<cite>Morita-Yamamuro C, Tsutsui T, Sato M, Yoshioka H, Tamaoki M, Ogawa D, Matsuura H, Yoshihara T, Ikeda A, Uyeda I, Yamaguchi J. The Arabidopsis gene CAD1 controls programmed cell death in the plant immune system and encodes a protein containing a MACPF domain. Plant Cell Physiol. 2005;46:902â€“12.
</cite> [<a href="https://doi.org/10.1093/pcp/pci095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15799997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morita-Yamamuro%20C,%20Tsutsui%20T,%20Sato%20M,%20Yoshioka%20H,%20Tamaoki%20M,%20Ogawa%20D,%20Matsuura%20H,%20Yoshihara%20T,%20Ikeda%20A,%20Uyeda%20I,%20Yamaguchi%20J.%20The%20Arabidopsis%20gene%20CAD1%20controls%20programmed%20cell%20death%20in%20the%20plant%20immune%20system%20and%20encodes%20a%20protein%20containing%20a%20MACPF%20domain.%20Plant%20Cell%20Physiol.%202005;46:902%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<cite>Nakatogawa H, Ohbayashi S, Sakoh-Nakatogawa M, Kakuta S, Suzuki SW, Kirisako H, Kondo-Kakuta C, Noda NN, Yamamoto H, Ohsumi Y. The autophagy-related protein kinase Atg1 interacts with the ubiquitin-like protein Atg8 via the Atg8 family interacting motif to facilitate autophagosome formation. J Biol Chem. 2012;287:28503â€“7.
</cite> [<a href="https://doi.org/10.1074/jbc.C112.387514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3436563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22778255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakatogawa%20H,%20Ohbayashi%20S,%20Sakoh-Nakatogawa%20M,%20Kakuta%20S,%20Suzuki%20SW,%20Kirisako%20H,%20Kondo-Kakuta%20C,%20Noda%20NN,%20Yamamoto%20H,%20Ohsumi%20Y.%20The%20autophagy-related%20protein%20kinase%20Atg1%20interacts%20with%20the%20ubiquitin-like%20protein%20Atg8%20via%20the%20Atg8%20family%20interacting%20motif%20to%20facilitate%20autophagosome%20formation.%20J%20Biol%20Chem.%202012;287:28503%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<cite>Nazio F, Carinci M, Valacca C, Bielli P, Strappazzon F, Antonioli M, Ciccosanti F, Rodolfo C, Campello S, Fimia GM, Sette C, Bonaldo P, Cecconi F. Fine-tuning of ULK1 mRNA and protein levels is required for autophagy oscillation. J Cell Biol. 2016;215:841â€“56.
</cite> [<a href="https://doi.org/10.1083/jcb.201605089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5166502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27932573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nazio%20F,%20Carinci%20M,%20Valacca%20C,%20Bielli%20P,%20Strappazzon%20F,%20Antonioli%20M,%20Ciccosanti%20F,%20Rodolfo%20C,%20Campello%20S,%20Fimia%20GM,%20Sette%20C,%20Bonaldo%20P,%20Cecconi%20F.%20Fine-tuning%20of%20ULK1%20mRNA%20and%20protein%20levels%20is%20required%20for%20autophagy%20oscillation.%20J%20Cell%20Biol.%202016;215:841%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<cite>Nazio F, Carinci M, Cecconi F. ULK1 ubiquitylation is regulated by phosphorylation on its carboxy terminus. Cell Cycle. 2017;16:1744â€“7.
</cite> [<a href="https://doi.org/10.1080/15384101.2017.1361063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5628632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28820317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nazio%20F,%20Carinci%20M,%20Cecconi%20F.%20ULK1%20ubiquitylation%20is%20regulated%20by%20phosphorylation%20on%20its%20carboxy%20terminus.%20Cell%20Cycle.%202017;16:1744%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<cite>Ni T, Gilbert RJC. Repurposing a pore: highly conserved perforin-like proteins with alternative mechanisms. Philos Trans R Soc Lond B Biol Sci. 2017;372: 20160212.
</cite> [<a href="https://doi.org/10.1098/rstb.2016.0212" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5483515/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28630152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ni%20T,%20Gilbert%20RJC.%20Repurposing%20a%20pore:%20highly%20conserved%20perforin-like%20proteins%20with%20alternative%20mechanisms.%20Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci.%202017;372:%2020160212." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<cite>Noutoshi Y, Kuromori T, Wada T, Hirayama T, Kamiya A, Imura Y, Yasuda M, Nakashita H, Shirasu K, Shinozaki K. Loss of necrotic spotted lesions 1 associates with cell death and defense responses in <em>Arabidopsis thaliana</em>. Plant Mol Biol. 2006;62:29â€“42.
</cite> [<a href="https://doi.org/10.1007/s11103-006-9001-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16900325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Noutoshi%20Y,%20Kuromori%20T,%20Wada%20T,%20Hirayama%20T,%20Kamiya%20A,%20Imura%20Y,%20Yasuda%20M,%20Nakashita%20H,%20Shirasu%20K,%20Shinozaki%20K.%20Loss%20of%20necrotic%20spotted%20lesions%201%20associates%20with%20cell%20death%20and%20defense%20responses%20in%20Arabidopsis%20thaliana.%20Plant%20Mol%20Biol.%202006;62:29%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<cite>Pan X, Welti R, Wang X. Quantitative analysis of major plant hormones in crude plant extracts by high-performance liquid chromatography-mass spectrometry. Nat Protoc. 2010;5:986â€“92.
</cite> [<a href="https://doi.org/10.1038/nprot.2010.37" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20448544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan%20X,%20Welti%20R,%20Wang%20X.%20Quantitative%20analysis%20of%20major%20plant%20hormones%20in%20crude%20plant%20extracts%20by%20high-performance%20liquid%20chromatography-mass%20spectrometry.%20Nat%20Protoc.%202010;5:986%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<cite>Pucciariello C, Parlanti S, Banti V, Novi G, Perata P. Reactive oxygen species-driven transcription in Arabidopsis under oxygen deprivation. Plant Physiol. 2012;159:184â€“96.
</cite> [<a href="https://doi.org/10.1104/pp.111.191122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3375960/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22415514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pucciariello%20C,%20Parlanti%20S,%20Banti%20V,%20Novi%20G,%20Perata%20P.%20Reactive%20oxygen%20species-driven%20transcription%20in%20Arabidopsis%20under%20oxygen%20deprivation.%20Plant%20Physiol.%202012;159:184%E2%80%9396." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<cite>Qi H, Xia FN, Xie LJ, Yu LJ, Chen QF, Zhuang XH, Wang Q, Li F, Jiang L, Xie Q, Xiao S. TRAF family proteins regulate autophagy dynamics by modulating AUTOPHAGY PROTEIN6 stability in Arabidopsis. Plant Cell. 2017;29:890â€“911.
</cite> [<a href="https://doi.org/10.1105/tpc.17.00056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5435438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28351989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qi%20H,%20Xia%20FN,%20Xie%20LJ,%20Yu%20LJ,%20Chen%20QF,%20Zhuang%20XH,%20Wang%20Q,%20Li%20F,%20Jiang%20L,%20Xie%20Q,%20Xiao%20S.%20TRAF%20family%20proteins%20regulate%20autophagy%20dynamics%20by%20modulating%20AUTOPHAGY%20PROTEIN6%20stability%20in%20Arabidopsis.%20Plant%20Cell.%202017;29:890%E2%80%93911." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<cite>Qi H, Xia FN, Xiao S. Autophagy in plants: Physiological roles and post-translational regulation. J Integr Plant Biol. 2021;63:161â€“79.
</cite> [<a href="https://doi.org/10.1111/jipb.12941" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32324339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qi%20H,%20Xia%20FN,%20Xiao%20S.%20Autophagy%20in%20plants:%20Physiological%20roles%20and%20post-translational%20regulation.%20J%20Integr%20Plant%20Biol.%202021;63:161%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<cite>Qi H, Lei X, Wang Y, Yu S, Liu T, Zhou SK, Chen JY, Chen QF, Qiu RL, Jiang L, Xiao S. 14-3-3 proteins contribute to autophagy by modulating SINAT-mediated degradation of ATG13. Plant Cell. 2022;34:4857â€“76.
</cite> [<a href="https://doi.org/10.1093/plcell/koac273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9709989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36053201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qi%20H,%20Lei%20X,%20Wang%20Y,%20Yu%20S,%20Liu%20T,%20Zhou%20SK,%20Chen%20JY,%20Chen%20QF,%20Qiu%20RL,%20Jiang%20L,%20Xiao%20S.%2014-3-3%20proteins%20contribute%20to%20autophagy%20by%20modulating%20SINAT-mediated%20degradation%20of%20ATG13.%20Plant%20Cell.%202022;34:4857%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<cite>Qi H, Wang Y, Bao Y, Bassham DC, Chen L, Chen QF, Hou SW, Hwang I, Huang L, Lai ZB, Li F, Liu Y, Qiu R, Wang H, Wang P, Xie Q, Zeng Y, Zhuang X, Gao C, Jiang L, Xiao S. Studying plant autophagy: challenges and recommended methodologies. Adv Biotechnol. 2023;1:2.</cite> [<a href="https://doi.org/10.1007/s44307-023-00002-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11727600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39883189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qi%20H,%20Wang%20Y,%20Bao%20Y,%20Bassham%20DC,%20Chen%20L,%20Chen%20QF,%20Hou%20SW,%20Hwang%20I,%20Huang%20L,%20Lai%20ZB,%20Li%20F,%20Liu%20Y,%20Qiu%20R,%20Wang%20H,%20Wang%20P,%20Xie%20Q,%20Zeng%20Y,%20Zhuang%20X,%20Gao%20C,%20Jiang%20L,%20Xiao%20S.%20Studying%20plant%20autophagy:%20challenges%20and%20recommended%20methodologies.%20Adv%20Biotechnol.%202023;1:2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<cite>Qu X, Chatty PR, Roeder AHK. Endomembrane trafficking protein SEC24A regulates cell size patterning in Arabidopsis. Plant Physiol. 2014;166:1877â€“90.
</cite> [<a href="https://doi.org/10.1104/pp.114.246033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4256882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25315606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qu%20X,%20Chatty%20PR,%20Roeder%20AHK.%20Endomembrane%20trafficking%20protein%20SEC24A%20regulates%20cell%20size%20patterning%20in%20Arabidopsis.%20Plant%20Physiol.%202014;166:1877%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<cite>Rosado CJ, Kondos S, Bull TE, Kuiper MJ, Law RH, Buckle AM, Voskoboinik I, Bird PI, Trapani JA, Whisstock JC, Dunstone MA. The MACPF/CDC family of pore-forming toxins. Cell Microbiol. 2008;10:1765â€“74.
</cite> [<a href="https://doi.org/10.1111/j.1462-5822.2008.01191.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2654483/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18564372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosado%20CJ,%20Kondos%20S,%20Bull%20TE,%20Kuiper%20MJ,%20Law%20RH,%20Buckle%20AM,%20Voskoboinik%20I,%20Bird%20PI,%20Trapani%20JA,%20Whisstock%20JC,%20Dunstone%20MA.%20The%20MACPF/CDC%20family%20of%20pore-forming%20toxins.%20Cell%20Microbiol.%202008;10:1765%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<cite>Scherz-Shouval R, Shvets E, Fass E, Shorer H, Gil L, Elazar Z. Reactive oxygen species are essential for autophagy and specifically regulate the activity of Atg4. EMBO J. 2007;26:1749â€“60.
</cite> [<a href="https://doi.org/10.1038/sj.emboj.7601623" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1847657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17347651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scherz-Shouval%20R,%20Shvets%20E,%20Fass%20E,%20Shorer%20H,%20Gil%20L,%20Elazar%20Z.%20Reactive%20oxygen%20species%20are%20essential%20for%20autophagy%20and%20specifically%20regulate%20the%20activity%20of%20Atg4.%20EMBO%20J.%202007;26:1749%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<cite>Shah J. The salicylic acid loop in plant defense. Curr Opin Plant Biol. 2003;6:365â€“71.
</cite> [<a href="https://doi.org/10.1016/s1369-5266(03)00058-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12873532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shah%20J.%20The%20salicylic%20acid%20loop%20in%20plant%20defense.%20Curr%20Opin%20Plant%20Biol.%202003;6:365%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<cite>Shah J, Tsui F, Klessig DF. Characterization of a salicylic acid-insensitive mutant (<em>sai1</em>) of <em>Arabidopsis thaliana</em>, identified in a selective screen utilizing the SA-inducible expression of the <em>tms2</em> gene. Mol Plant Microbe Interact. 1997;10:69â€“78.
</cite> [<a href="https://doi.org/10.1094/MPMI.1997.10.1.69" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9002272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shah%20J,%20Tsui%20F,%20Klessig%20DF.%20Characterization%20of%20a%20salicylic%20acid-insensitive%20mutant%20(sai1)%20of%20Arabidopsis%20thaliana,%20identified%20in%20a%20selective%20screen%20utilizing%20the%20SA-inducible%20expression%20of%20the%20tms2%20gene.%20Mol%20Plant%20Microbe%20Interact.%201997;10:69%E2%80%9378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57"><cite>Sharma P, Jha AB, Dubey RS, Pessarakli M. Reactive oxygen species, oxidative damage, and antioxidative defense mechanism in plants under stressful conditions. J Bot.Â 2012;2012:217037.Â 10.1155/2012/217037.</cite></li>
<li id="CR58">
<cite>Shi W, Wang L, Yao L, Hao W, Han C, Fan M, Wang W, Bai MY. Spatially patterned hydrogen peroxide orchestrates stomatal development in Arabidopsis. Nat Commun. 2022;13(1): 5040.
</cite> [<a href="https://doi.org/10.1038/s41467-022-32770-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9418256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36028510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi%20W,%20Wang%20L,%20Yao%20L,%20Hao%20W,%20Han%20C,%20Fan%20M,%20Wang%20W,%20Bai%20MY.%20Spatially%20patterned%20hydrogen%20peroxide%20orchestrates%20stomatal%20development%20in%20Arabidopsis.%20Nat%20Commun.%202022;13(1):%205040." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<cite>Shin JH, Yoshimoto K, Ohsumi Y, Jeon JS, An G. OsATG10b, an autophagosome component, is needed for cell survival against oxidative stresses in rice. Mol Cell. 2009;27:67â€“74.</cite> [<a href="https://doi.org/10.1007/s10059-009-0006-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19214435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shin%20JH,%20Yoshimoto%20K,%20Ohsumi%20Y,%20Jeon%20JS,%20An%20G.%20OsATG10b,%20an%20autophagosome%20component,%20is%20needed%20for%20cell%20survival%20against%20oxidative%20stresses%20in%20rice.%20Mol%20Cell.%202009;27:67%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<cite>Signorelli S, Tarkowski ÅP, Van den Ende W, Bassham DC. Linking autophagy to abiotic and biotic stress responses. Trends Plant Sci. 2019;24:413â€“30.
</cite> [<a href="https://doi.org/10.1016/j.tplants.2019.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6475611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30824355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Signorelli%20S,%20Tarkowski%20%C5%81P,%20Van%20den%20Ende%20W,%20Bassham%20DC.%20Linking%20autophagy%20to%20abiotic%20and%20biotic%20stress%20responses.%20Trends%20Plant%20Sci.%202019;24:413%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<cite>Smalle J, Kurepa J, Yang P, Emborg TJ, Babiychuk E, Kushnir S, Vierstra RD. The pleiotropic role of the 26S proteasome subunit RPN10 in Arabidopsis growth and development supports a substrate-s pecific function in abscisic acid signaling. Plant Cell. 2003;2003(15):965â€“80.</cite> [<a href="https://doi.org/10.1105/tpc.009217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC152342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12671091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smalle%20J,%20Kurepa%20J,%20Yang%20P,%20Emborg%20TJ,%20Babiychuk%20E,%20Kushnir%20S,%20Vierstra%20RD.%20The%20pleiotropic%20role%20of%20the%2026S%20proteasome%20subunit%20RPN10%20in%20Arabidopsis%20growth%20and%20development%20supports%20a%20substrate-s%20pecific%20function%20in%20abscisic%20acid%20signaling.%20Plant%20Cell.%202003;2003(15):965%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<cite>Suttangkakul A, Li F, Chung T, Vierstra RD. The ATG1/ATG13 protein kinase complex is both a regulator and a target of autophagic recycling in Arabidopsis. Plant Cell. 2011;23(10):3761â€“79.
</cite> [<a href="https://doi.org/10.1105/tpc.111.090993" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3229148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21984698/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suttangkakul%20A,%20Li%20F,%20Chung%20T,%20Vierstra%20RD.%20The%20ATG1/ATG13%20protein%20kinase%20complex%20is%20both%20a%20regulator%20and%20a%20target%20of%20autophagic%20recycling%20in%20Arabidopsis.%20Plant%20Cell.%202011;23(10):3761%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<cite>Tang J, Bassham DC. Autophagy during drought: function, regulation, and potential application. Plant J. 2022;109:390â€“401.
</cite> [<a href="https://doi.org/10.1111/tpj.15481" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34469611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20J,%20Bassham%20DC.%20Autophagy%20during%20drought:%20function,%20regulation,%20and%20potential%20application.%20Plant%20J.%202022;109:390%E2%80%93401." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<cite>Tian S, Wang X, Li P, Wang H, Ji H, Xie J, Qiu Q, Shen D, Dong H. Plant aquaporin AtPIP1;4 links apoplastic H<sub>2</sub>O<sub>2</sub> induction to disease immunity pathways. Plant Physiol. 2016;171(3):1635â€“50.
</cite> [<a href="https://doi.org/10.1104/pp.15.01237" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4936539/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26945050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tian%20S,%20Wang%20X,%20Li%20P,%20Wang%20H,%20Ji%20H,%20Xie%20J,%20Qiu%20Q,%20Shen%20D,%20Dong%20H.%20Plant%20aquaporin%20AtPIP1;4%20links%20apoplastic%20H2O2%20induction%20to%20disease%20immunity%20pathways.%20Plant%20Physiol.%202016;171(3):1635%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<cite>Tsutsui T, Morita-Yamamuro C, Asada Y, Minami E, Shibuya N, Ikeda A, Yamaguchi J. Salicylic acid and a chitin elicitor both control expression of the CAD1 gene involved in the plant immunity of Arabidopsis. Biosci Biotechnol Biochem. 2006;70:2042â€“8.
</cite> [<a href="https://doi.org/10.1271/bbb.50700" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16960394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsutsui%20T,%20Morita-Yamamuro%20C,%20Asada%20Y,%20Minami%20E,%20Shibuya%20N,%20Ikeda%20A,%20Yamaguchi%20J.%20Salicylic%20acid%20and%20a%20chitin%20elicitor%20both%20control%20expression%20of%20the%20CAD1%20gene%20involved%20in%20the%20plant%20immunity%20of%20Arabidopsis.%20Biosci%20Biotechnol%20Biochem.%202006;70:2042%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<cite>Valkai I, KÃ©nesi F, Domonkos I, Ayaydin F, TarkowskÃ¡ D, Strnad M, FaragÃ³ A, Bodai L, FehÃ©r A. The Arabidopsis RLCK VI_A2 kinase controls seedling and plant growth in parallel with gibberellin. Int J Mol Sci. 2020;21: 7266.
</cite> [<a href="https://doi.org/10.3390/ijms21197266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7582978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33019674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Valkai%20I,%20K%C3%A9nesi%20F,%20Domonkos%20I,%20Ayaydin%20F,%20Tarkowsk%C3%A1%20D,%20Strnad%20M,%20Farag%C3%B3%20A,%20Bodai%20L,%20Feh%C3%A9r%20A.%20The%20Arabidopsis%20RLCK%20VI_A2%20kinase%20controls%20seedling%20and%20plant%20growth%20in%20parallel%20with%20gibberellin.%20Int%20J%20Mol%20Sci.%202020;21:%207266." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<cite>Wada S, Ishida H, Izumi M, Yoshimoto K, Ohsumi Y, Mae T, Makino A. Autophagy plays a role in chloroplast degradation during senescence in individually darkened leaves. Plant Physiol. 2009;149:885â€“93.
</cite> [<a href="https://doi.org/10.1104/pp.108.130013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2633819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19074627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wada%20S,%20Ishida%20H,%20Izumi%20M,%20Yoshimoto%20K,%20Ohsumi%20Y,%20Mae%20T,%20Makino%20A.%20Autophagy%20plays%20a%20role%20in%20chloroplast%20degradation%20during%20senescence%20in%20individually%20darkened%20leaves.%20Plant%20Physiol.%202009;149:885%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<cite>Waszczak C, Carmody M, Kangasjarvi J. Reactive oxygen species in plant signaling. Annu Rev Plant Biol. 2018;69:209â€“36.
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-042817-040322" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29489394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waszczak%20C,%20Carmody%20M,%20Kangasjarvi%20J.%20Reactive%20oxygen%20species%20in%20plant%20signaling.%20Annu%20Rev%20Plant%20Biol.%202018;69:209%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<cite>Weindel CG, Martinez EL, Zhao X, Mabry CJ, Bell SL, Vail KJ, Coleman AK, VanPortfliet JJ, Zhao B, Wagner AR, Azam S, Scott HM, Li P, West AP, Karpac J, Patrick KL, Watson RO. Mitochondrial ROS promotes susceptibility to infection via gasdermin D-mediated necroptosis. Cell. 2022;185:3214â€“31.
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.06.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9531054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35907404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weindel%20CG,%20Martinez%20EL,%20Zhao%20X,%20Mabry%20CJ,%20Bell%20SL,%20Vail%20KJ,%20Coleman%20AK,%20VanPortfliet%20JJ,%20Zhao%20B,%20Wagner%20AR,%20Azam%20S,%20Scott%20HM,%20Li%20P,%20West%20AP,%20Karpac%20J,%20Patrick%20KL,%20Watson%20RO.%20Mitochondrial%20ROS%20promotes%20susceptibility%20to%20infection%20via%20gasdermin%20D-mediated%20necroptosis.%20Cell.%202022;185:3214%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<cite>Xia FN, Zeng B, Liu HS, Qi H, Xie LJ, Yu LJ, Chen QF, Li JF, Chen YQ, Jiang L, Xiao S. SINAT E3 ubiquitin ligases mediate FREE1 and VPS23A degradation to modulate abscisic acid signaling. Plant Cell. 2020;32:3290â€“310.
</cite> [<a href="https://doi.org/10.1105/tpc.20.00267" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7534459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32753431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia%20FN,%20Zeng%20B,%20Liu%20HS,%20Qi%20H,%20Xie%20LJ,%20Yu%20LJ,%20Chen%20QF,%20Li%20JF,%20Chen%20YQ,%20Jiang%20L,%20Xiao%20S.%20SINAT%20E3%20ubiquitin%20ligases%20mediate%20FREE1%20and%20VPS23A%20degradation%20to%20modulate%20abscisic%20acid%20signaling.%20Plant%20Cell.%202020;32:3290%E2%80%93310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<cite>Xiao S, Gao W, Chen QF, Chan SW, Zheng SX, Ma J, Wang M, Welti R, Chye ML. Overexpression of Arabidopsis Acyl-CoA binding protein ACBP3 promotes starvation-induced and age-dependent leaf senescence. Plant Cell. 2010;22:1463â€“82.
</cite> [<a href="https://doi.org/10.1105/tpc.110.075333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2899868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20442372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiao%20S,%20Gao%20W,%20Chen%20QF,%20Chan%20SW,%20Zheng%20SX,%20Ma%20J,%20Wang%20M,%20Welti%20R,%20Chye%20ML.%20Overexpression%20of%20Arabidopsis%20Acyl-CoA%20binding%20protein%20ACBP3%20promotes%20starvation-induced%20and%20age-dependent%20leaf%20senescence.%20Plant%20Cell.%202010;22:1463%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<cite>Xiong Y, Contento AL, Nguyen PQ, Bassham DC. Degradation of oxidized proteins by autophagy during oxidative stress in Arabidopsis. Plant Physiol. 2007;143:291â€“9.
</cite> [<a href="https://doi.org/10.1104/pp.106.092106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1761971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17098847/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiong%20Y,%20Contento%20AL,%20Nguyen%20PQ,%20Bassham%20DC.%20Degradation%20of%20oxidized%20proteins%20by%20autophagy%20during%20oxidative%20stress%20in%20Arabidopsis.%20Plant%20Physiol.%202007;143:291%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<cite>Xu L, Liu F, Lechner E, Genschik P, Crosby WL, Ma H, Peng W, Huang D, Xie D. The SCF (COI1) ubiquitin-ligase complexes are required for jasmonate response in Arabidopsis. Plant Cell. 2002;14:1919â€“35.
</cite> [<a href="https://doi.org/10.1105/tpc.003368" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC151474/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12172031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20L,%20Liu%20F,%20Lechner%20E,%20Genschik%20P,%20Crosby%20WL,%20Ma%20H,%20Peng%20W,%20Huang%20D,%20Xie%20D.%20The%20SCF%20(COI1)%20ubiquitin-ligase%20complexes%20are%20required%20for%20jasmonate%20response%20in%20Arabidopsis.%20Plant%20Cell.%202002;14:1919%E2%80%9335." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<cite>Yoo SD, Cho YH, Sheen J. <em>Arabidopsis</em> mesophyll protoplasts: a versatile cell system for transient gene expression analysis. Nat Protoc. 2007;2:1565â€“72.
</cite> [<a href="https://doi.org/10.1038/nprot.2007.199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17585298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoo%20SD,%20Cho%20YH,%20Sheen%20J.%20Arabidopsis%20mesophyll%20protoplasts:%20a%20versatile%20cell%20system%20for%20transient%20gene%20expression%20analysis.%20Nat%20Protoc.%202007;2:1565%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<cite>Yoshimoto K, Hanaoka H, Sato S, Kato T, Tabata S, Noda T, Ohsumi Y. Processing of ATG8s, ubiquitin-like proteins, and their deconjugation by ATG4s are essential for plant autophagy. Plant Cell. 2004;16:2967â€“83.
</cite> [<a href="https://doi.org/10.1105/tpc.104.025395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC527192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15494556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshimoto%20K,%20Hanaoka%20H,%20Sato%20S,%20Kato%20T,%20Tabata%20S,%20Noda%20T,%20Ohsumi%20Y.%20Processing%20of%20ATG8s,%20ubiquitin-like%20proteins,%20and%20their%20deconjugation%20by%20ATG4s%20are%20essential%20for%20plant%20autophagy.%20Plant%20Cell.%202004;16:2967%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<cite>Yoshimoto K, Jikumaru Y, Kamiya Y, Kusano M, Consonni C, Panstruga R, Ohsumi Y, Shirasu K. Autophagy negatively regulates cell death by controlling NPR1-dependent salicylic acid signaling during senescence and the innate immune response in Arabidopsis. Plant Cell. 2009;21:2914â€“27.
</cite> [<a href="https://doi.org/10.1105/tpc.109.068635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2768913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19773385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshimoto%20K,%20Jikumaru%20Y,%20Kamiya%20Y,%20Kusano%20M,%20Consonni%20C,%20Panstruga%20R,%20Ohsumi%20Y,%20Shirasu%20K.%20Autophagy%20negatively%20regulates%20cell%20death%20by%20controlling%20NPR1-dependent%20salicylic%20acid%20signaling%20during%20senescence%20and%20the%20innate%20immune%20response%20in%20Arabidopsis.%20Plant%20Cell.%202009;21:2914%E2%80%9327." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<cite>Yu L, Liu D, Chen S, Dai Y, Guo W, Zhang X, Wang L, Ma S, Xiao M, Qi H, Xiao S, Chen QF. Evolution and expression of the membrane attack complex and perforin gene family in the Poaceae. Int J Mol Sci. 2020;21: 5736.
</cite> [<a href="https://doi.org/10.3390/ijms21165736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7460961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32785137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20L,%20Liu%20D,%20Chen%20S,%20Dai%20Y,%20Guo%20W,%20Zhang%20X,%20Wang%20L,%20Ma%20S,%20Xiao%20M,%20Qi%20H,%20Xiao%20S,%20Chen%20QF.%20Evolution%20and%20expression%20of%20the%20membrane%20attack%20complex%20and%20perforin%20gene%20family%20in%20the%20Poaceae.%20Int%20J%20Mol%20Sci.%202020;21:%205736." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<cite>Yuan LB, Dai YS, Xie LJ, Yu LJ, Zhou Y, Lai YX, Yang YC, Xu L, Chen QF, Xiao S. Jasmonate regulates plant responses to postsubmergence reoxygenation through transcriptional activation of antioxidant synthesis. Plant Physiol. 2017;173:1864â€“80.
</cite> [<a href="https://doi.org/10.1104/pp.16.01803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5338657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28082717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan%20LB,%20Dai%20YS,%20Xie%20LJ,%20Yu%20LJ,%20Zhou%20Y,%20Lai%20YX,%20Yang%20YC,%20Xu%20L,%20Chen%20QF,%20Xiao%20S.%20Jasmonate%20regulates%20plant%20responses%20to%20postsubmergence%20reoxygenation%20through%20transcriptional%20activation%20of%20antioxidant%20synthesis.%20Plant%20Physiol.%202017;173:1864%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<cite>Zhang X, Dai YS, Wang YX, Su ZZ, Yu LJ, Zhang ZF, Xiao S, Chen QF. Overexpression of the Arabidopsis MACPF protein AtMACP2 promotes pathogen resistance by activating SA signaling. Int J Mol Sci.Â 2022;23:8784. 10.3390/ijms23158784.</cite> [<a href="https://doi.org/10.3390/ijms23158784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9369274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35955922/" class="usa-link">PubMed</a>]</li>
<li id="CR80">
<cite>Zhou J, Li XY, Liu YJ, Feng J, Wu Y, Shen HM, Lu GD. Full-coverage regulations of autophagy by ROS: from induction to maturation. Autophagy. 2022;18:1240â€“55.
</cite> [<a href="https://doi.org/10.1080/15548627.2021.1984656" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9225210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34662529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20J,%20Li%20XY,%20Liu%20YJ,%20Feng%20J,%20Wu%20Y,%20Shen%20HM,%20Lu%20GD.%20Full-coverage%20regulations%20of%20autophagy%20by%20ROS:%20from%20induction%20to%20maturation.%20Autophagy.%202022;18:1240%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<cite>Zhuang X, Chung KP, Luo M, Jiang L. Autophagosome biogenesis and the endoplasmic reticulum: a plant perspective. Trends Plant Sci. 2018;23:677â€“92.
</cite> [<a href="https://doi.org/10.1016/j.tplants.2018.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29929776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhuang%20X,%20Chung%20KP,%20Luo%20M,%20Jiang%20L.%20Autophagosome%20biogenesis%20and%20the%20endoplasmic%20reticulum:%20a%20plant%20perspective.%20Trends%20Plant%20Sci.%202018;23:677%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<cite>Zou JJ, Li XD, Ratnasekera D, Wang C, Liu WX, Song LF, Zhang WZ, Wu WH. <em>Arabidopsis</em> CALCIUM-DEPENDENT PROTEIN KINASE8 and CATALASE3 function in abscisic acid-mediated signaling and H<sub>2</sub>O<sub>2</sub> homeostasis in stomatal guard cells under drought stress. Plant Cell. 2015;27:1445â€“60.
</cite> [<a href="https://doi.org/10.1105/tpc.15.00144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4456645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25966761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zou%20JJ,%20Li%20XD,%20Ratnasekera%20D,%20Wang%20C,%20Liu%20WX,%20Song%20LF,%20Zhang%20WZ,%20Wu%20WH.%20Arabidopsis%20CALCIUM-DEPENDENT%20PROTEIN%20KINASE8%20and%20CATALASE3%20function%20in%20abscisic%20acid-mediated%20signaling%20and%20H2O2%20homeostasis%20in%20stomatal%20guard%20cells%20under%20drought%20stress.%20Plant%20Cell.%202015;27:1445%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12367599/bin/44307_2025_78_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">44307_2025_78_MOESM1_ESM.zip</a><sup> (14.3MB, zip) </sup><p>Supplementary Material 1:Â Fig. S1.Structures of MACP proteins from Arabidopsis. Structures of NSL1 (AT1G28380), MACP1 (AT1G14780), and CAD1 (AT1G29690) were predicted by Alphafold and visualized using PyMOL software. AIM located within the proteins were analyzed by iLIR database. The predicted AIM is shown in red.Â Fig. S2. The accelerated senescence phenotypes of <em>MACP2-</em>OE plants are dependent on the salicylic acid pathway. a Representative photograph of Wild-type (WT), <em>MACP2-</em>OE-2,<em>eds1-22</em>,<em>pad4</em>, OE-2<em>eds1</em>, OE<em>-2</em><em>pad4 seedlings</em> in response to nitrogen starvation (Nâˆ’). For nitrogen starvation treatment, 7-d-old WT, OE-2,<em>eds1-22, pad4</em>, OE-2<em>eds1</em>, and OE-2<em>pad4</em>seedlings grown on half-strength solid MS medium were transferred to N-rich (N+) or N-free (Nâˆ’) medium and photographed at 5 d into treatment. b Relative chlorophyll contents of 7-d-old wild-type (WT), <em>MACP2-</em>OE-1,<em>eds1-22</em>,<em>pad4</em>,OE-1<em>eds1-22</em>, and OE-1<em>pad4</em>plants seedlings under nitrogen deficiency, expressed as a percentage relative to control plants or seedlings. For each experiment,20 seedlings were used per genotype. All experiments were performed as three biological replicates, each with similar results. Relative chlorophyll contents are means Â± SD (<em>n</em> = 3) calculated from three biological replicates. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em> &lt; 0.05.Â Fig. S3. The starvation sensitive phenotypes of <em>MACP2-</em>OE plants are rescued by GSH application. a Wild-type (WT), <em>MACP2-</em>OE (OE-1 and OE-2), and <em>atg5-1</em> seedlings grown on half-strength solid MS medium for 7 d. The seedlings were transferred to solid sucrose-rich medium (CK) or sucrose-free medium (Câˆ’) with or without 500 Î¼M GSH and incubated under normal light/dark conditions or continuous dark conditions for 12 d, respectively, followed by recovery under normal growth conditions for 5 d. b Relative chlorophyll contents of 7-d-old WT, <em>MACP2-</em>OE (OE-1 and OE-2), and <em>atg5-1</em>plants in response to carbon starvation (a) for 12 d, followed by recovery under normal growth conditions for 5 d. For each experiment, 20 seedlings were used per genotype. All experiments were performed as three biological replicates, each with similar results. Relative chlorophyll contents are means Â± SD (<em>n</em> = 3) calculated from three biological replicates. Different lowercase letters indicate significant differences between groups as determined by one-way ANOVA, <em>P</em> &lt; 0.05.Â Fig. S4.Knockout of <em>MACP2</em> partially rescues the hypersensitive phenotypes of <em>atg5-1</em>to nitrogen deficiency. a and b Representative photographs (a) and relative chlorophyll contents (b) of wild-type (WT), <em>macp2-1</em>, <em>macp2-2</em>,<em>atg5-1</em>,<em>macp2-1Â atg5-1</em>,and <em>macp2-2Â atg5-1</em>double mutant seedlings in response to nitrogen starvation. 7-d-old WT<em>macp2</em>,<em>atg5-1</em>,<em>macp2-1Â atg5-1</em>,and <em>macp2-2Â atg5</em><em>-1</em>double mutant seedlings grown on half-strength solid MS medium were transferred to N-rich (N+) or N-deficient (Nâˆ’) medium and photographed at 5 d into treatment. For each experiment, 20 seedlings were used per genotype. The experiments were performed as three biological replicates, each with similar results. Data are means Â± SD (<em>n</em> = 3) calculated from three biological replicates. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em> &lt; 0.05.Â Fig. S5. MACP2 modulates H<sub>2</sub>O<sub>2</sub>â€“ or BTHâ€“induced autophagosome formation. a and c Confocal analysis showing eGFP-ATG8e labeled autophagosomes in the root cells of wild-type (WT),<em>macp2-1</em>, and <em>MACP2</em>-OE seedlings. Seven-d-old<em>eGFP-ATG8e</em>, <em>macp2-1 eGFP-ATG8e</em>, and <em>MACP2-</em>OE<em>eGFP-ATG8e</em> (<em>MACP2-FLAG eGFP-ATG8e</em>) seedlings were transferred to half-strength MS liquid medium with or without 100 Î¼M H<sub>2</sub>O<sub>2</sub> or BTH (SA agonist, benzo [1,2,3] thiadiazole-7-carbothioic acid S-methyl ester) for 12 h or 8 h, respectively. The formation of autophagosomes was visualized by fluorescence confocal microscopy. Scale bars, 10 Î¼m. banddNumber of puncta per root section in root cells from the <em>eGFP-ATG8e</em>, <em>macp2-1 eGFP-ATG8e</em>,and<em>MACP2</em>OE<em> eGFP-ATG8e</em>in (a and c). Data are means Â± SD (<em>n</em>= 3) calculated from three biological replicates. For each experiment, 15 sections were used for calculation per genotype. Different lowercase letters indicate significant differences within each group as determined by one-way ANOVA, <em>P</em> &lt; 0.05.Â Fig. S6. Colocalization of MACP2-YFP with mCherry-ATG8f in response to nitrogen starvation. A <em>MACP2-YFP</em> plant was crossed to <em>mCherry-ATG8f</em> to generate the double transgenic line <em>mCherry-ATG8f MACP2-YFP</em>. Colocalization assay of MACP2-YFP and mCherry-ATG8f in transgenic plants <em>mCherry-ATG8f MACP2-YFP</em>after 12 h of nitrogen starvation. YFP/mCherry was excited at 514/561 nm and the emissions were collected using a 554/601 nm band-pass emission filter on ZEISS LSM 880 confocal laser scanning microscope. Scale bars, 10 Î¼m.Â Fig. S7. Identification of <em>MACP2-FLAG</em>transgenic plants. a and b Relative <em>MACP2</em>transcript levels (a) and accumulation of MACP2-FLAG(b) in 4-week-old wild-type (WT) and <em>pUBQ10:MACP2-FLAG</em>lines, as determined by RT-qPCR or immunoblot analysis, respectively. Anti-HA and anti-Actin antibodies were used for immunoblotting. Actin is shown as a loading control. The experiments were performed three times (biological replicates), each with similar results, and representative data from one replicate are shown. Data are means Â± SD (<em>n</em> = 3) calculated from three technical replicates. Asterisks indicate significant differences from the WT (**<em>P</em> &lt; 0.01, *<em>P</em> &lt; 0.05 by Studentâ€™s <em>t</em> test).Â Fig. S8. Protein abundance of MACP2 and MACP2 M1 in wild type and <em>atg8e</em> mutant. The <em>GFP-FLAG</em> and <em>MACP2-HA</em> or <em>MACP2 M1-HA</em> plasmids were co-transfected into protoplasts isolated from wild type (WT) and <em>atg8e</em> mutant. After overnight culture, protoplasts were treated with 50 Î¼M cycloheximide (CHX) under darkness for the indicated times (0, 4, 6, or 8 h). The experiments were performed as three biological replicates, each with similar results. The abundance of eGFP-FLAG was used as a control. The relative quantification of MACP2-HA and MACP2 MA-HA proteins were normalized to the corresponding eGFP-FLAG control and then compared to its level at 0 h. Data are means Â± SD (<em>n</em> = 3) calculated from three biological replicates. The numbers on the left indicate the molecular weight (kD) of each size marker. hpt, h post-treatment.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12367599/bin/44307_2025_78_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (10.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12367599/bin/44307_2025_78_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3.</a><sup> (414KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The original data and materials in the paper are available via contacting the corresponding author.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Advanced Biotechnology are provided here courtesy of <strong>Springer</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1007/s44307-025-00078-4"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/44307_2025_Article_78.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.6Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12367599/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12367599/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367599%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367599/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12367599/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12367599/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40833484/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12367599/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40833484/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12367599/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12367599/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="vPfcVj1HmgQuRXs1zKkg9d4sw4YM95HMFcnXtCme5BAXRaYVenQWCgDHW40iFtwD">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
