
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Insights into type IX secretion from PorKN cogwheel structure bound to PorG and attachment complexes - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE533E9C8AF391E3053E9C0047835996.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365241/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Insights into type IX secretion from PorKN cogwheel structure bound to PorG and attachment complexes">
<meta name="citation_author" content="Dhana G Gorasia">
<meta name="citation_author_institution" content="Oral Health Cooperative Research Centre, Melbourne Dental School, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author" content="Eric Hanssen">
<meta name="citation_author_institution" content="Ian Holmes Imaging Centre, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author_institution" content="ARC Industrial Training Centre for Cryo Electron Microscopy of Membrane Proteins, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author_institution" content="Department of Biochemistry and Pharmacology, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author" content="Manasi Mudaliyar">
<meta name="citation_author_institution" content="ARC Industrial Training Centre for Cryo Electron Microscopy of Membrane Proteins, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author_institution" content="Department of Biochemistry and Pharmacology, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author" content="Craig J Morton">
<meta name="citation_author_institution" content="CSIRO Biomedical Manufacturing, Clayton, VIC Australia">
<meta name="citation_author" content="Sepideh Valimehr">
<meta name="citation_author_institution" content="Ian Holmes Imaging Centre, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author_institution" content="ARC Industrial Training Centre for Cryo Electron Microscopy of Membrane Proteins, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author" content="Christine Seers">
<meta name="citation_author_institution" content="Oral Health Cooperative Research Centre, Melbourne Dental School, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author" content="Lianyi Zhang">
<meta name="citation_author_institution" content="Oral Health Cooperative Research Centre, Melbourne Dental School, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author" content="Matthew T Doyle">
<meta name="citation_author_institution" content="Sydney Infectious Diseases Institute, The University of Sydney, Darlington, NSW Australia">
<meta name="citation_author_institution" content="Centre for Drug Discovery Innovation, The University of Sydney, Darlington, NSW Australia">
<meta name="citation_author_institution" content="School of Medical Sciences, The University of Sydney, Darlington, NSW Australia">
<meta name="citation_author" content="Debnath Ghosal">
<meta name="citation_author_institution" content="ARC Industrial Training Centre for Cryo Electron Microscopy of Membrane Proteins, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author_institution" content="Department of Biochemistry and Pharmacology, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author" content="Paul D Veith">
<meta name="citation_author_institution" content="Oral Health Cooperative Research Centre, Melbourne Dental School, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_author" content="Eric C Reynolds">
<meta name="citation_author_institution" content="Oral Health Cooperative Research Centre, Melbourne Dental School, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7735">
<meta name="citation_doi" content="10.1038/s41467-025-63163-1">
<meta name="citation_pmid" content="40830366">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365241/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365241/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365241/pdf/41467_2025_Article_63163.pdf">
<meta name="description" content="The Type IX Secretion System exports proteins across the outer membrane (OM) of bacteria in the Bacteroidota phylum, however, the mechanistic details remain unknown. In Porphyromonas gingivalis the core components of the multi-protein complex are ...">
<meta name="og:title" content="Insights into type IX secretion from PorKN cogwheel structure bound to PorG and attachment complexes">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The Type IX Secretion System exports proteins across the outer membrane (OM) of bacteria in the Bacteroidota phylum, however, the mechanistic details remain unknown. In Porphyromonas gingivalis the core components of the multi-protein complex are ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365241/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365241">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63163-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63163.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365241%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365241/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365241/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365241/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7735. doi: <a href="https://doi.org/10.1038/s41467-025-63163-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63163-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Insights into type IX secretion from PorKN cogwheel structure bound to PorG and attachment complexes</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gorasia%20DG%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Dhana G Gorasia</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Dhana G Gorasia</span></h3>
<div class="p">
<sup>1</sup>Oral Health Cooperative Research Centre, Melbourne Dental School, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gorasia%20DG%22%5BAuthor%5D" class="usa-link"><span class="name western">Dhana G Gorasia</span></a>
</div>
</div>
<sup>1,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hanssen%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Eric Hanssen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Eric Hanssen</span></h3>
<div class="p">
<sup>2</sup>Ian Holmes Imaging Centre, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">
<sup>3</sup>ARC Industrial Training Centre for Cryo Electron Microscopy of Membrane Proteins, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">
<sup>4</sup>Department of Biochemistry and Pharmacology, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hanssen%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eric Hanssen</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mudaliyar%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Manasi Mudaliyar</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Manasi Mudaliyar</span></h3>
<div class="p">
<sup>3</sup>ARC Industrial Training Centre for Cryo Electron Microscopy of Membrane Proteins, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">
<sup>4</sup>Department of Biochemistry and Pharmacology, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mudaliyar%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Manasi Mudaliyar</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Morton%20CJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Craig J Morton</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Craig J Morton</span></h3>
<div class="p">
<sup>5</sup>CSIRO Biomedical Manufacturing, Clayton, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Morton%20CJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Craig J Morton</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Valimehr%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sepideh Valimehr</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sepideh Valimehr</span></h3>
<div class="p">
<sup>2</sup>Ian Holmes Imaging Centre, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">
<sup>3</sup>ARC Industrial Training Centre for Cryo Electron Microscopy of Membrane Proteins, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Valimehr%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sepideh Valimehr</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seers%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Christine Seers</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Christine Seers</span></h3>
<div class="p">
<sup>1</sup>Oral Health Cooperative Research Centre, Melbourne Dental School, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seers%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christine Seers</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Lianyi Zhang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Lianyi Zhang</span></h3>
<div class="p">
<sup>1</sup>Oral Health Cooperative Research Centre, Melbourne Dental School, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lianyi Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doyle%20MT%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Matthew T Doyle</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Matthew T Doyle</span></h3>
<div class="p">
<sup>6</sup>Sydney Infectious Diseases Institute, The University of Sydney, Darlington, NSW Australia </div>
<div class="p">
<sup>7</sup>Centre for Drug Discovery Innovation, The University of Sydney, Darlington, NSW Australia </div>
<div class="p">
<sup>8</sup>School of Medical Sciences, The University of Sydney, Darlington, NSW Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doyle%20MT%22%5BAuthor%5D" class="usa-link"><span class="name western">Matthew T Doyle</span></a>
</div>
</div>
<sup>6,</sup><sup>7,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ghosal%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Debnath Ghosal</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Debnath Ghosal</span></h3>
<div class="p">
<sup>3</sup>ARC Industrial Training Centre for Cryo Electron Microscopy of Membrane Proteins, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">
<sup>4</sup>Department of Biochemistry and Pharmacology, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ghosal%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Debnath Ghosal</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Veith%20PD%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Paul D Veith</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Paul D Veith</span></h3>
<div class="p">
<sup>1</sup>Oral Health Cooperative Research Centre, Melbourne Dental School, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Veith%20PD%22%5BAuthor%5D" class="usa-link"><span class="name western">Paul D Veith</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reynolds%20EC%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Eric C Reynolds</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Eric C Reynolds</span></h3>
<div class="p">
<sup>1</sup>Oral Health Cooperative Research Centre, Melbourne Dental School, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reynolds%20EC%22%5BAuthor%5D" class="usa-link"><span class="name western">Eric C Reynolds</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Oral Health Cooperative Research Centre, Melbourne Dental School, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div id="Aff2">
<sup>2</sup>Ian Holmes Imaging Centre, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div id="Aff3">
<sup>3</sup>ARC Industrial Training Centre for Cryo Electron Microscopy of Membrane Proteins, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div id="Aff4">
<sup>4</sup>Department of Biochemistry and Pharmacology, The Bio21 Molecular Science and Biotechnology Institute, The University of Melbourne, Parkville, VIC Australia </div>
<div id="Aff5">
<sup>5</sup>CSIRO Biomedical Manufacturing, Clayton, VIC Australia </div>
<div id="Aff6">
<sup>6</sup>Sydney Infectious Diseases Institute, The University of Sydney, Darlington, NSW Australia </div>
<div id="Aff7">
<sup>7</sup>Centre for Drug Discovery Innovation, The University of Sydney, Darlington, NSW Australia </div>
<div id="Aff8">
<sup>8</sup>School of Medical Sciences, The University of Sydney, Darlington, NSW Australia </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 3; Accepted 2025 Aug 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365241  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830366/" class="usa-link">40830366</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The Type IX Secretion System exports proteins across the outer membrane (OM) of bacteria in the Bacteroidota phylum, however, the mechanistic details remain unknown. In <em>Porphyromonas gingivalis</em> the core components of the multi-protein complex are the Sov translocon, Attachment Complexes (PorQ, U, V, Z), PorLM molecular motors and PorKN rings. Here, we present a ~ 3.5 Å cryo-EM structure of the periplasmic rings comprising 32–33 subunits each of PorK and PorN. Additionally, we show the presence of a critical disulfide bond between PorK and the OM protein PorG that is essential for protein secretion and demonstrate that the Attachment Complexes bind to, and are localized above, the PorKN rings. Overall, each ring resembles a cogwheel with PorN forming cog-like projections that we propose engage with the PorLM motor to drive the rotation of the PorKN cogwheel together with PorG and associated Attachment Complexes, thus providing the energy to complete protein secretion and the coordinated cell surface attachment of the secreted cargo.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Bacteriology, Permeation and transport, Bacterial secretion, Cryoelectron microscopy</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Type IX secretion system (T9SS) is required for protein export in the Bacteroidota. Here, the authors obtain structure of T9SS PorKN ring which resembles a cogwheel that may have a rotational movement and is disulfide bonded to outer membrane beta-barrel PorG.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Periodontitis is a prevalent, chronic inflammatory disease associated with a dysbiotic subgingival plaque that results in the destruction of the tooth’s supporting tissues and is additionally linked with a range of systemic diseases like diabetes, cardiovascular diseases, certain cancers and dementia<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. <em>Porphyromonas gingivalis</em>, an obligate anaerobe, is strongly associated with severe periodontitis at sites refractory to treatment and is now recognised as a keystone pathobiont of this disease<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. <em>P. gingivalis</em> uses the Type IX Secretion System (T9SS) to translocate at least 35 cargo proteins to the cell surface, including hallmark virulence factors such as the gingipain proteases RgpA, RgpB, and Kgp<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. The T9SS secreted cargo proteins play an important role in virulence of the bacterium, including biofilm formation, nutrition acquisition, antibiotic resistance, adhesion and degradation of host proteins<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>.</p>
<p id="Par4">The T9SS is exclusively present in members of the <em>Bacteroidota</em> phylum<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. The T9SS cargo proteins have an N-terminal signal peptide that facilitates export across the inner membrane by the Sec system and have a conserved C-terminal domain (CTD), referred to as the CTD signal, that enables them to pass through the outer membrane (OM) via the T9SS<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. The fate of proteins secreted by the T9SS is determined by their CTD signal type. Proteins with type A CTD signals undergo a complex process once they reach the cell surface. Their CTD signals are removed and replaced with anionic-lipopolysaccharide (A-LPS), which anchors the proteins to the cell surface, resulting in the formation of an electron-dense surface layer (EDSL)<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. In contrast, proteins possessing type B CTD signals are anchored by binding to their cognate carrier proteins<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Besides regulatory proteins, the T9SS is composed of at least 18 proteins in <em>P. gingivalis</em>, namely: PorK, PorL, PorM, PorN, Sov, PorT, PorU, PorW, PorP, PorV, PorQ, PorZ, PorE, PorF, PorG, Plug, PorD, PorA<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Recent studies have begun to shed light on the structure and function of some of these components (Fig. <a href="#Fig1" class="usa-link">1A</a>). The PorL-PorM motor complex is anchored to the inner membrane and powered by the proton motive force<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. The PorM rotor extends into the periplasm to drive secretion in non-motile <em>P. gingivalis</em> and both secretion and motility in <em>Flavobacterium johnsoniae</em><sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Additionally, in-situ structural studies of the T9SS and bacterial two-hybrid studies have shown an interaction between PorM and PorN<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. PorK and PorN form large ring-shaped structures in the periplasm that are tethered to the OM by the lipidation of the PorK N-terminus and by association with an 8-stranded OM β-barrel protein, PorG<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Evidence from cross-linking studies indicated that a periplasmic loop of PorG is in close proximity to both PorK and PorN<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. SprA (an orthologue of <em>P. gingivalis</em> Sov) has been shown to form a large 36-stranded β-barrel that is either bound to PorV or Plug and is the OM translocon through which the T9SS cargo proteins are secreted<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. We demonstrated that the Sov interactome in <em>P. gingivalis</em> included PorV, PorA, Plug, PorW, PorD, and the PorKN rings, proposing that PorW forms a bridge between the PorKN rings and the Sov translocon<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Recent findings in <em>F. johnsoniae</em> validated our results, where an extended-translocon including SprE (PorW orthologue), PorD, and trapped substrates was purified from the <em>gldL</em> (<em>porL</em>) mutant<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. The accumulation of substrates in the translocon when energetic input was removed suggested that energy is required to release PorV-bound substrates from the translocon<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. PorU and PorZ are anchored to the OM via binding to the PorV and PorQ OM β-barrel proteins respectively, and together the four proteins form the Attachment Complex<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. PorU is a sortase that cleaves the CTD signal on the cell surface and swaps it with A-LPS provided by PorZ, thereby anchoring cargo proteins onto the cell surface<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Structure of the PorKN ring.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365241_41467_2025_63163_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c92b/12365241/994c777099ab/41467_2025_63163_Fig1_HTML.jpg" loading="lazy" id="d33e521" height="983" width="675" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Cartoon representation of the T9SS architecture showing the location of PorKN rings and PorG, PorLM motor, and the Sov translocon bound to PorV and PorW. <strong>B</strong> A bottom view of 33-fold symmetry cryo-EM volume of the PorKN ring (Pink: PorK, Blue: PorN). <strong>C</strong> Side views of the PorKN ring showing the flat upper surface and an outer rim formed by PorK with cog-like projections formed by PorN. <strong>D</strong> Top-view of the PorKN ring. <strong>E</strong> Enlarged views of the outer surface of the PorKN ring showing the fitting of PorK (pink) and PorN (blue) and interlock of PorK subunits. <strong>F</strong> Ribbon fold structures of PorK (pink) and PorN (blue), outer and side views showing the glycans on PorK. <strong>G</strong> Inner ring cross section showing the interface of PorN with the cavity formed by two PorK subunits. PorK (purple and pink) surface rendered. PorN (blue) in surface rendered and ribbon representation. <strong>H</strong> Detailed structure of the two glycans on PorK.</p></figcaption></figure><p id="Par5">Although, the PorKN ring and its association with PorG was identified nearly a decade ago, their precise role remains unclear. The proteins that comprise the Attachment Complex have been identified, but the exact location of this complex in relation to the other T9SS components is unknown. In this work, we present a high-resolution structure of the PorKN ring, offering insights into their function. We also demonstrate that PorG is associated with PorK through a disulphide bond, which is crucial for the function of the T9SS and show that the Attachment Complexes are associated with, and co-localised in-vivo above the PorKN rings providing insight into the co-ordinated mechanism of T9SS secretion and attachment in <em>P. gingivalis</em>.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">High-resolution structure of the PorKN rings</h3>
<p id="Par6">Previously, we purified the PorKNG complex from <em>P. gingivalis</em> and demonstrated that it forms large, 50 nm ring-shaped structures<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. However, due to one preferred orientation on the EM grid and protein aggregation, we were only able to achieve a resolution of ~40 Å. At this resolution, we could discern that the structure was composed of one large ring and one slightly smaller ring, but the organisation of the PorK, PorN and PorG subunits within the ring structure remained unclear. With optimisation of buffer conditions, which included addition of 1 M urea prior to applying the sample on an EM grid, we were able to mitigate strong aggregation of the PorKN rings and capture various orientations of the rings on the EM grid by focusing acquisition on thicker vitreous ice. This improvement enabled us to achieve ~3.5 Å maps of the PorKN rings (Fig. <a href="#Fig1" class="usa-link">1B–D</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). Several rounds of classifications were performed that resulted in two classes with distinct symmetry of C32 (46% of final particles) and C33 (54% of final particles). A very small number of top views with C34 and C35 fold-symmetry were observed early in the 2D classification process but did not lead to 3D classes suggesting that these two species are too rare to bear on the classification. <em>P. gingivalis</em> is therefore capable of assembling rings of different stoichiometry. The interfaces between adjacent heterodimers were identical in the C32 and C33 fold symmetries, however, they did make a difference to the measured diameter of the ring: an increase of 16 Å was observed in the C33 fold symmetry (529 Å) compared to the C32 fold-symmetry (513 Å). In both instance each heterodimer had the periodicity of 50.36 Å.</p>
<p id="Par7">The AlphaFold<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> predicted structure of the PorK and PorN heterodimer was used as a starting point for structure refinement. The structure of PorK was solved from Glu<sup>45</sup> to Ala<sup>481</sup> missing the N-terminal 21 residues (not including signal peptide) and the C-terminal 10 residues. PorN was solved from Leu<sup>50</sup> to Gln<sup>297</sup> missing the N-terminal 25 residues (not including signal peptide) and the C-terminal 63 residues. Although present in the purified samples, PorG was unable to be resolved (see below)<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. The orientation of the ring was easily solved by comparison to the published in-situ structures of the ring where the larger diameter “upper ring” is adjacent to the OM<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. The top and side views of the ring show that PorK forms the upper ring, featuring a large, flat surface orientated towards the OM that is bound by a discontinuous “outer rim” (Fig. <a href="#Fig1" class="usa-link">1C, D</a>). PorN forms the second ring of a slightly smaller external diameter underneath PorK and it folds to create cog-like projections that in vivo would be on the periplasmic side (Fig. <a href="#Fig1" class="usa-link">1C</a>). Enlarged side-views show how PorK subunits interlock through a large wedge-shaped protrusion (~Asp<sup>186</sup>-Ile<sup>218</sup>), which extends diagonally down and across to fit into a crevice on the adjacent monomer (Fig. <a href="#Fig1" class="usa-link">1E, G</a>). Interestingly, the outside surface of the PorK outer rim resembled a three-toed claw that exhibited highly conserved negatively charged residues (Supplementary Fig. <a href="#MOESM1" class="usa-link">2A, B</a>). In <em>P. gingivalis</em> PorK, these were largely contributed by Glu<sup>121-2</sup>, Glu<sup>127</sup>, and Glu<sup>152-4</sup>. A small N-terminal region of PorK (residues Gly<sup>50</sup>-Val<sup>103</sup>) and the C-terminal part of PorK (residues Trp<sup>261</sup>-Arg<sup>471</sup>) together adopt the fold of a formylglycine generating enzyme (FGE)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> with a root mean square deviation (RMSD) of 0.951. This structural arrangement makes up most of the core of PorK and a large part of the cavity that binds PorN (Supplementary Fig. <a href="#MOESM1" class="usa-link">2C</a>). In contrast, the region Arg<sup>104</sup>-Cys<sup>260</sup> is unique to PorK and includes the interlocking wedge, the negatively charged outer rim and the outside wall of the PorN-binding cavity (Supplementary Fig. <a href="#MOESM1" class="usa-link">2C</a>).</p>
<p id="Par8">A cross-section view of the PorKN heterodimer (Fig. <a href="#Fig1" class="usa-link">1F</a>) highlights the location of the most N- and C-terminal parts solved. For PorK, both termini appear on the OM side, consistent with the N-terminal Cys being lipidated and inserted into the membrane. The most C-terminal part of PorN was the α-helix extending from Thr<sup>271</sup>-Arg<sup>290</sup> located on the inside of the ring extending toward the OM (Fig. <a href="#Fig1" class="usa-link">1F</a>). This helix can be seen in the inside views appearing as vertical bars (Fig. <a href="#Fig1" class="usa-link">1C</a>). The unsolved N- and C-terminal regions of PorK and PorN are likely to be flexible in the purified rings. The unresolved C-terminal region of PorN (residues Thr<sup>291</sup>-Lys<sup>361</sup>) is highly positively charged and is predicted to be mostly α-helical.</p>
<p id="Par9">PorN was found to insert into a cavity produced underneath each pair of PorK subunits (Fig. <a href="#Fig1" class="usa-link">1G</a>). Three linear regions of PorN are involved in the binding. The first region from Leu<sup>50</sup> to Trp<sup>70</sup> is mostly α-helix and binds deeply within the cavity of PorK (Supplementary Fig. <a href="#MOESM1" class="usa-link">3A</a>). Within this region, Arg<sup>53</sup> and Ser/Thr<sup>51</sup> are absolutely conserved among 45 diverse PorN homologues within the <em>Bacteroidia</em> class (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). These residues penetrate deeply into a pocket of PorK and are modelled to interact mainly through hydrogen bonds (Supplementary Fig. <a href="#MOESM1" class="usa-link">3A</a>). The second and third regions from Phe<sup>179</sup> to Ser<sup>184</sup> and Glu<sup>285</sup> to Val<sup>295</sup> respectively form one deeply penetrating loop, and several residues involved in hydrophobic and aromatic stacking interactions (Supplementary Fig. <a href="#MOESM1" class="usa-link">3A</a>). Together, the nature and extent of these interactions suggest a stable interaction between PorK and PorN. Consistent with this, the computationally estimated dissociation constant (K<sub>d</sub>) of one PorN subunit binding to two PorK subunits at 37 °C was computed to be 2 × 10<sup>−11 </sup>M with a ΔG of −15.2 kcal/mol, indicative of a strong association. Due to technical limitations in obtaining native PorK without PorN, the K<sup>d</sup> could not be determined experimentally.</p>
<p id="Par10">The cog-like projections of PorN suggest potential interaction points with the dimeric PorM rotor. We therefore predicted the binding site for this known interaction using AlphaFold. The C-terminal d4 domain of PorM was predicted to bind between the cog-like projections of PorN, potentially through electrostatic interactions between the conserved negatively charged cogs and the positively charged rotor tip (Supplementary Fig. <a href="#MOESM1" class="usa-link">3B</a>). While the ipTM score was 0.47, the prediction was consistent, and the geometry was a good fit. The poor score may reflect that the binding is dynamic- the main requirement being that the PorM shaft can slot in-between the cogs that can drive rotation of the rings.</p></section><section id="Sec4"><h3 class="pmc_sec_title">O-glycosylation of PorK and PorN</h3>
<p id="Par11">PorK and PorN were predicted to be <em>O</em>-glycosylated at three and two sites respectively, on the basis of the published (D)(S/T)(A/I/L/V/M/T/S/C/F/G) glycosylation motif<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. We performed mass spectrometry analysis on the purified PorK and PorN protein bands resulting in the identification of glycosylated peptides that covered all 5 sites (Table <a href="#Tab1" class="usa-link">1</a>). All sites were modified with the known major heptasaccharide O-glycans of Δmass 1436 and 1394 Da, while the minor O-glycans of Δmass 1352 and 792 were observed for only some of the sites. The level of site occupancy was high since non-glycosylated forms of the same peptides were not observed, with the exception of non-glycosylated PorN <sup>264</sup>ALAEYCPTPDSMK<sup>280</sup> which was detected at a low level. The collision-induced decay (CID) MS2 spectra clearly show the presence of the heptasaccharide associated with each of the five glycosylated peptides (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>).</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Glycosylated peptides of PorK and PorN</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Peptide sequence</th>
<th colspan="1" rowspan="1">Glycan masses</th>
<th colspan="1" rowspan="1">Best Score (<em>E</em> value)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">PorK: GSIVLGNKEAD<strong><span class="text-underline">S</span></strong><sup><strong><span class="text-underline">68</span></strong></sup>LWGIPAESR</td>
<td align="left" colspan="1" rowspan="1">1352, 1394, 1436</td>
<td align="left" colspan="1" rowspan="1">41 (1.3E-04)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorK: D<strong><span class="text-underline">S</span></strong><sup><strong><span class="text-underline">106</span></strong></sup>IIR</td>
<td align="left" colspan="1" rowspan="1">1352, 1394, 1436</td>
<td align="left" colspan="1" rowspan="1">
<sup>b</sup>y1-4, b2-4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorK: D<strong><span class="text-underline">T</span></strong><sup><strong><span class="text-underline">222</span></strong></sup>AFVDESGNIISETITR</td>
<td align="left" colspan="1" rowspan="1">1394, 1436</td>
<td align="left" colspan="1" rowspan="1">46 (4.5E-05)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<sup>a</sup>PorN: ALAEYCPTPD<strong><span class="text-underline">S</span></strong><sup><strong><span class="text-underline">274</span></strong></sup>MKMESK</td>
<td align="left" colspan="1" rowspan="1">1352, 1394, 1436</td>
<td align="left" colspan="1" rowspan="1">37 (3.7E-04)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorN: DGLFVTQD<strong><span class="text-underline">T</span></strong><sup><strong><span class="text-underline">299</span></strong></sup>TWMK</td>
<td align="left" colspan="1" rowspan="1">792, 1352, 1394, 1436</td>
<td align="left" colspan="1" rowspan="1">47 (5.0E-05)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p14">
<p><sup>a</sup>Minor levels of non-glycosylated peptide were also detected.</p>
<p><sup>b</sup>The PorK peptide DSIIR was too small to generate a sufficient number of MS fragments for Mascot scoring. However, manual searching of the raw data revealed glycosylated peptides that exhibited the expected fragment ions (y1-4, b2-4) for this sequence with very high mass accuracy.</p>
</div></div></section><p id="Par12">The PorK glycans were also observed in the cryo-EM map. PorK Ser<sup>68</sup> was observed with only one sugar, while PorK Ser<sup>106</sup> and Thr<sup>222</sup> were observed with full-length (heptasaccharide) glycans with only the acetyl glycerol phosphate portion unassigned (Fig. <a href="#Fig1" class="usa-link">1H</a>). The glycans on Ser<sup>106</sup> and Thr<sup>222</sup> are located in the PorK-specific region under the outer rim, with the terminal sugars potentially interacting with adjacent PorK subunits. In contrast, the glycans on PorN were not visualised on the cryo-EM map, suggesting high flexibility.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Functional role of PorG in the T9SS</h3>
<p id="Par13">Previously, we have shown that PorG interacts with the PorKN ring (Fig. <a href="#Fig1" class="usa-link">1A</a>)<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. To further elucidate PorG’s role within these PorKN rings and the T9SS, we generated a <em>porG</em> deletion mutant in the <em>P. gingivalis</em> W50 strain. We assessed the impact of the deletion on T9SS function through phenotypic analysis of pigmentation, gingipain activity assay and the ability of T9SS cargo proteins to attach to the cell surface. The W50 <em>porG</em> mutant was apigmented on blood agar plates as opposed to the black pigmented WT, consistent with earlier reports of a <em>porG</em> deletion mutant in the <em>P. gingivalis</em> ATCC 33277 strain<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. This suggests that PorG is an essential component of the T9SS (Fig. <a href="#Fig2" class="usa-link">2A</a>). The <em>porG</em> mutant displayed negligible &lt;5% cell surface and supernatant gingipain (Kgp and Rgp) activity compared to the WT (Fig. <a href="#Fig2" class="usa-link">2B</a>), consistent with an inability of these T9SS-dependent proteases to be translocated across the OM. In contrast to the WT, anti-A-LPS western blot showed no signal above 60 kDa in the <em>porG</em> mutant (Fig. <a href="#Fig2" class="usa-link">2C</a>), indicating the inability of T9SS cargo proteins to be attached to the cell surface by modification with A-LPS. Accordingly, the layer of cargo proteins on the cell surface known as the EDSL was absent in cryo-electron microscopy images of <em>porG</em> mutant cells but present in the WT (Fig. <a href="#Fig2" class="usa-link">2D</a>). We also constructed <em>porG</em> complementation strains (<em>porG</em><sup>+</sup>) by chromosomal integration of <em>porG</em> into the <em>fimA</em> (<em>pg2132</em>) locus. The <em>porG</em>-complemented strain demonstrated pigmentation, increased protease activities, A-LPS signal above 60 kDa and EDSL (Fig. <a href="#Fig2" class="usa-link">2A–D</a>), indicating the phenotype observed in the <em>porG</em> mutant is specifically due to the lack of PorG. Jointly, these results indicate that PorG is an essential component of the T9SS.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. PorG is an essential component of the T9SS.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365241_41467_2025_63163_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c92b/12365241/ae85b0fea63f/41467_2025_63163_Fig2_HTML.jpg" loading="lazy" id="d33e974" height="509" width="796" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong><em>P. gingivalis</em> WT, <em>porG</em> deletant, and <em>porG</em><sup><em>+</em></sup> complementation strain pigmentation on blood agar plates after 7 days of incubation. <strong>B</strong> Intact whole cells and supernatant samples were analysed for endoproteolytic activities of the T9SS RgpA, RgpB, and Kgp cargo proteins using Arg- and Lys-specific chromogenic substrates. Relative gingipain activities are represented as a percentage of the WT activity for all fractions and strains, <em>n </em>= 3 biological replicates, error bars mean ± SD. Significant difference is observed between WT and <em>porG</em> mutant, two-tailed unpaired Student’s <em>t</em> test (<em>p</em> value: 6 × 10<sup>−7</sup>). <strong>C</strong> Cell lysates from WT and the porG mutant were electrophoresed using SDS-PAGE and the proteins were transferred onto a nitrocellulose membrane and probed with mouse monoclonal antibody mAb 1B5 (anti-A-LPS). Coomassie stained gel shows the protein load. <strong>D</strong> Cryo-electron micrographs showing the presence of an electron-dense surface layer (<em>EDSL</em>) in WT cells, and the absence of this layer in the <em>porG</em> mutant cells (white arrows). Arrows show outer membrane (OM) and inner membrane (IM). Scale bars, 200 nm. Large protein complexes were prepared from <strong>E</strong> ABK- and <strong>F</strong><em>porG</em> mutant cells, the samples were negatively stained and viewed under the electron microscope. The electron micrographs show the presence of large (50 nm) rings (black arrows). Scale bars, 200 nm. Source data are provided as a <a href="#MOESM8" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par14">Next, we investigated if the PorKN rings were affected by the absence of PorG. We applied our partial purification protocol for macromolecular complexes, that we have previously used to assess the presence of the rings in various T9SS mutants<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. For comparison with the <em>porG</em> mutant, we used the <em>P. gingivalis</em> ABK- strain which lacks all three gingipains but has a functional T9SS. The samples were stained with uranyl acetate and visualised under electron microscopy. The 50 nm PorKN rings were visible in the ABK- cells (Fig. <a href="#Fig2" class="usa-link">2E</a>) and the <em>porG</em> deletion mutant (Fig. <a href="#Fig2" class="usa-link">2F</a>). Notably, fewer PorKN rings were visible in the <em>porG</em> mutant, possibly due to a reduction in the abundance of T9SS protein components in this strain (see below).</p>
<p id="Par15">We also performed gene cluster analysis and found <em>P. gingivalis porG</em> (<em>pg0189</em>) is located adjacent to genes coding a BamA-associated lipoprotein (PG0188)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> and undecaprenyl diphosphate synthase (UppS, IspU, PG0190). Nearby are also the genes coding BamA (PG0191), and two Skp proteins (PG0192 and PG0193)<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. This six-gene cluster was found to be conserved across numerous genera in the Bacteroidota phylum, especially in class <em>Bacteroidia</em>, but also extending to other classes such as <em>Cytophagia</em>, <em>Sphingobacteriia</em>, and <em>Flavobacteriia</em>, with the exception of the lipoprotein orthologs, which were sometimes undetected (Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). Besides UppS, which is involved in peptidoglycan and LPS biosynthesis, the other four encoded proteins are involved in Omp biogenesis<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, with Skp functioning as a periplasmic chaperone and the BAM complex responsible for the insertion of OMPs into the OM<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. The conserved location of <em>porG</em> within this cluster, therefore, suggested a possible involvement with OMP biogenesis that is specific to the T9SS. Some species of <em>Bacteroides</em>, such as <em>B. fragilis</em>, while closely related to <em>P. gingivalis</em>, lack PorG and the T9SS. A gene cluster analysis of <em>B. fragilis bamA</em> (BF0504), shows an extended gene cluster that includes all the genes noted above in the same order except for <em>porG</em> (Supplementary Data <a href="#MOESM4" class="usa-link">2</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">PorG not required for T9SS OM β-Barrel localization</h3>
<p id="Par16">To explore if the T9SS components are affected in the absence of PorG, we performed label free quantitative (LFQ) proteomics on the whole cell lysate of WT and the <em>porG</em> mutant as per our published protocol<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. In the <em>porG</em> mutant, all the T9SS components were decreased in abundance compared to the WT (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>), including 6 OM β-barrels, however, OM β-barrels unrelated to the T9SS were unaffected. This suggests that the T9SS is downregulated in the absence of PorG.</p>
<p id="Par17">BamA is a highly conserved protein responsible for folding and inserting OM β-barrel proteins into the OM<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Since the <em>porG</em> gene (<em>pg0189</em>) is localised near the <em>bamA</em> gene (<em>pg0191</em>), we explored whether PorG was involved in the localisation and folding of the T9SS specific OM β-barrels Sov, PorV, PorQ, PorT, PorF, and PorP. The cell envelope (CE) proteins from both the wild-type (WT) and <em>porG</em> mutant strains were subjected to trypsin digestion and analysed by mass spectrometry. The data were analysed by MaxQuant software, and the values of the ratio of LFQ intensities of the <em>porG</em> mutant to WT was normalised to other OM β-barrels that are not related to the T9SS. The T9SS OM β-barrels were identified in the CE of the <em>porG</em> mutant albeit at reduced levels (Fig. <a href="#Fig3" class="usa-link">3A</a>), consistent with the whole cell lysate results (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). Together, this suggests that T9SS OM β-barrels are able to integrate into the OM in the absence of PorG. We also investigated the protein interactions of the Sov translocon in the <em>porG</em> mutant by blue native PAGE western blot and affinity purification mass spectrometry using Sov specific antibodies. In the native western blot, two Sov specific bands were observed, which we previously assigned to mainly PorV-Sov (~500 kDa) and PorV-Sov-PorW (~750 kDa) (Fig. <a href="#Fig3" class="usa-link">3B</a>)<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> consistent with the PorV and Sov β-barrels being properly integrated in the OM. Of note, to obtain the signal in the <em>porG</em> lane, the blot was exposed for longer, supporting the finding of lower Sov abundance in the <em>porG</em> mutant. Similarly, we previously employed Sov co-immunoprecipitation (Co-IP) studies in several strains to show that the Sov interactome includes PorA, PorD, Plug, PorK, and PorN in addition to PorV and PorW<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. To test if Sov can make the same interactions in the <em>porG</em> mutant, the Sov affinity-enriched material from ATCC 33277 <em>porG</em> mutant (gift from Prof Naito) was analysed by mass spectrometry and found to contain Sov, PorV, PorA, PorW, and PorK at more than 10 times the level observed in the negative control, while PorD was also present but its observed ratio was not significant (Fig. <a href="#Fig3" class="usa-link">3C</a>). The ATCC 33277 <em>porG</em> mutant was selected for this work to maintain consistency with our previous published work in the ATCC 33277 <em>P. gingivalis</em> strain<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Taken together, these results strongly suggest that the two key T9SS OMPs, Sov, and PorV can integrate into the OM and interact with their binding partners, suggesting that the defect in the T9SS function due to lack of PorG is not due to the inability of OMPs to be incorporated into the OM.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. PorG is not involved in localising T9SS OM β-barrels to the CE.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365241_41467_2025_63163_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c92b/12365241/13c530b2d6f6/41467_2025_63163_Fig3_HTML.jpg" loading="lazy" id="d33e1199" height="222" width="700" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Cell envelope fractions from WT and the <em>porG</em> mutant were analysed by mass spectrometry and MaxQuant software. The LFQ ratios of <em>porG</em> mutant to WT of the T9SS OM β-barrels were plotted relative to other OMPs not related to the T9SS. <em>n</em> = 3 biological replicates, Error bars: mean ± SD. <strong>B</strong> Cell lysates from ABK- and the <em>porG</em> mutant were electrophoresed on a blue native gel. The proteins were transferred onto a PVDF membrane and probed with anti-Sov antibodies. Coomassie stained gel shows the protein load. <strong>C</strong> Cell lysates from <em>porG</em> and <em>sov</em> mutants were subjected to co-immunoprecipitation using anti-Sov antibodies. The enriched material was analysed by mass spectrometry and MaxQuant software. The graph is a plot of LFQ intensities ratio of <em>porG</em> to <em>sov</em> mutant (log scale). The values of abundance of each protein relative to Sov (iBAQ) is written on top of the bar graph. * LFQ ratios &lt;10 were deemed insignificant but shown to enable the display of PorD for completeness. Source data are provided as a <a href="#MOESM8" class="usa-link">Source Data</a> file.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">PorG forms a disulfide bond with PorK</h3>
<p id="Par18">To further understand the nature of the interaction between PorK, PorN, and PorG, we used AlphaFold Multimer<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> to predict the structure of the PorKNG complex and its relationship with the lipid bilayer. The modelling revealed that PorK anchors to the membrane via its lipid moiety and interacts with PorG, while also showing a gap of ~20 Å between the flat surface of PorK and the OM (Fig. <a href="#Fig4" class="usa-link">4A</a>). Assuming a conventional OM β-barrel architecture where the N- and C- termini are located in the periplasm, the model shows the periplasmic end of the PorG barrel interacting with PorK (Fig. <a href="#Fig4" class="usa-link">4A</a>). The PorG C-terminal extension containing Cys<sup>232</sup> was predicted to run on the surface of PorK with the Cys placed in close proximity to PorK Cys<sup>356</sup>, suggesting potential for disulphide bond formation (Fig. <a href="#Fig4" class="usa-link">4A</a>). The structure of PorKN derived from the cryo-EM analysis confirmed the location of the PorK Cys<sup>356</sup> and its availability for creating a disulphide bond with PorG.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Conserved Cys<sup>232</sup> of PorG forms a disulphide bond with a conserved Cys<sup>356</sup> of PorK.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365241_41467_2025_63163_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c92b/12365241/38be0087917d/41467_2025_63163_Fig4_HTML.jpg" loading="lazy" id="d33e1267" height="994" width="695" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> AlphaFold3 model of PorG (green)-PorK(pink) interaction showing the close proximity of the PorG Cys<sup>232</sup> and PorK Cys<sup>356</sup> (circled). Cys residues are shown in ball rendering <strong>B</strong> SDS-PAGE of purified PorKNG complex electrophoresed under reducing (DTT+) and non-reducing (DTT−) conditions. <strong>C</strong> MS/MS spectra consistent with a disulphide bond peptide between PorG (<sup>230</sup>TFCNKQ<sup>235</sup>) and PorK (<sup>355</sup>GCFLGNFKPGEGDYTADGHLIPSR<sup>378</sup>). <strong>D</strong> SDS-PAGE of purified PorKNG complex under reducing conditions. <strong>E</strong> Cryo-EM images of the PorKN rings showing the purified dual ring structure (NK-KN) in the presence (top panel) and absence of DTT (bottom panel). Scale bars: 50 nm. Source data are provided as a <a href="#MOESM8" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par19">To verify the presence of a disulphide bond between PorK and PorG, the purified PorKNG complex sample was analysed by SDS-PAGE under denatured reducing and nonreducing conditions (Fig. <a href="#Fig4" class="usa-link">4B</a>). There were no obvious differences in the migration profiles of the two prominent protein bands of PorK and PorN under reducing and nonreducing conditions. However, under nonreducing condition the intensity of the PorG (~25 kDa) (band 4) (Fig. <a href="#Fig4" class="usa-link">4B</a>) decreased significantly and a band at ~90 kDa appeared (band 1) (Fig. <a href="#Fig4" class="usa-link">4B</a>). This suggested the presence of a disulphide bond between PorG and its associated partner. To investigate which proteins are present in the ~90 kDa band, this protein band, together with the PorK and PorN bands in the nonreducing gel lane, was excised and cut in half. Half of each band was subjected to trypsin digestion in the absence of dithiothreitol (DTT), while the other half was reduced and alkylated before trypsin digestion. The tryptic fragments were analysed by mass spectrometry. Analysis of the band at ~90 kDa identified both PorK and PorG proteins, however, pLink software did not identify any disulphide bonded peptides. Thus, the MS/MS spectra were analysed manually to identify the disulphide bond. PorG only contains one cysteine, Cys<sup>232</sup>. In the reduced gel band analysis of the ~90 kDa band a PorK peptide containing Cys<sup>356</sup> was identified “<sup>355</sup>GCFLGNFKPGEGDYTADGHLIPSR<sup>378</sup>”. Of note, this peptide was absent in the nonreduced gel analysis of the same band. Thus, we created a table of expected masses containing the PorK peptide “<sup>355</sup>GCFLGNFKPGEGDYTADGHLIPSR<sup>378</sup>” combined with four differentially cleaved PorG peptides “<sup>230</sup>TFCNK<sup>234</sup>”, “<sup>230</sup>TFCNKQ<sup>235</sup>”, “<sup>229</sup>KTFCNK<sup>234</sup>” and “<sup>229</sup>KTFCNKQ<sup>235</sup>” (Table <a href="#Tab2" class="usa-link">2</a>). Analysis of the MS1-level spectra accurately identified MS peaks corresponding to the PorK peptide cross-linked to all four of the variant PorG peptides (Table <a href="#Tab2" class="usa-link">2</a>). Each of these peaks was only found in the nonreduced band, supporting their identity as cross-linked peptides. The assignments were confirmed from multiple MS/MS spectra, from various charge-states with the best spectrum obtained for the crosslinked peptides PorK- <sup>355</sup>GCFLGNFKPGEGDYTADGHLIPSR<sup>378</sup> and PorG- <sup>229</sup>KTFCNKQ<sup>235</sup> with a parent ion mass of 822.8873 (4+) (Fig. <a href="#Fig4" class="usa-link">4C</a>). Bioinformatic analysis and BLAST searches of PorG and PorK identified PorG, Cys<sup>232</sup> and PorK Cys<sup>356</sup> to be conserved suggesting that the crosslink between these two Cys residues may be a conserved and important feature (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). Since PorG was not resolved in the high-resolution structure of the PorKN rings we determined its stoichiometry by densitometry analysis. Densitometry analysis of coomassie-stained gels of the purified complex estimated ~6–7 PorG molecules per PorKN ring (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Expected and observed masses (M/Z) of PorK peptide (GCFLGNFKPGEGDYTADGHLIPSR) + PorG peptides</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">PorK peptide crosslinked with:</th>
<th colspan="1" rowspan="1">Calculated m/z</th>
<th colspan="1" rowspan="1">Observed m/z non-reduced</th>
<th colspan="1" rowspan="1">Observed m/z reduced</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">PorG: TFCNK</td>
<td align="left" colspan="1" rowspan="1">790.8714 (4+)</td>
<td align="left" colspan="1" rowspan="1">790.8713 (4+)</td>
<td align="left" colspan="1" rowspan="1">Not observed</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorG: TFCNKQ</td>
<td align="left" colspan="1" rowspan="1">822.8860 (4+)</td>
<td align="left" colspan="1" rowspan="1">822.8873 (4+)</td>
<td align="left" colspan="1" rowspan="1">Not observed</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorG: KTFCNK</td>
<td align="left" colspan="1" rowspan="1">822.8951 (4+)</td>
<td align="left" colspan="1" rowspan="1">822.8946 (4+)</td>
<td align="left" colspan="1" rowspan="1">Not observed</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorG: KTFCNKQ</td>
<td align="left" colspan="1" rowspan="1">854.9098 (4+)</td>
<td align="left" colspan="1" rowspan="1">854.9090 (4+)</td>
<td align="left" colspan="1" rowspan="1">Not observed</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par20">To investigate a structural role of disulphide bond formation between PorK and PorG, we employed our PorKNG complex purification protocol using reducing conditions at all steps and electrophoresed the purified sample on an SDS-PAGE gel. We identified the PorK and PorN bands, however, the PorG band was not detected (Fig. <a href="#Fig4" class="usa-link">4D</a>). PorN appeared to be degraded slightly, perhaps due to some protease activity. The PorKN containing complex isolated under reducing conditions was then visualised under cryo-EM to investigate the effect of reduction and the removal of PorG on the structure of the rings. The rings under reducing conditions appeared to be same as the rings without DTT (Fig. <a href="#Fig4" class="usa-link">4E</a>), and the dual ring structure (NK-KN) was also unaffected. This suggests that once the rings have formed, the removal of PorG does not impact the PorKN ring structure. In the presence of DTT, the PorG is removed, but the dual ring structure stays intact, likely due to the lipid interactions of the PorK lipoprotein. In the absence of DTT PorG is likely to be present in between the lipids and due to its low stoichiometry, is difficult to visualise. This suggests that the artefact of the dual ring formation between the rings is attributable to the lipids in PorK as we had speculated previously<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> and is not related to the presence of PorG.</p></section><section id="Sec8"><h3 class="pmc_sec_title">The PorG-PorK disulfide is essential for the function of the T9SS</h3>
<p id="Par21">To further understand the role of the disulphide bond between PorG and PorK, we mutated PorG Cys<sup>232</sup> to Ser<sup>232</sup>. We mutated the Cys<sup>232</sup> codon of <em>porG</em> with the insertion of <em>ermF</em> 3’ as a recombinant marker gene, producing the strain designated PorG<sup>C232S</sup>. As a control, a recombinant cassette in which the Cys<sup>232</sup> codon was unchanged was also prepared, this strain was named PorG<sup>Cys-Cys</sup>. The control PorG<sup>Cys-Cys</sup> strain retained a WT like black pigmentation on blood agar, had A-LPS signal above 60 kDa and EDSL was present, indicating the recombination strategy did not affect the function of the T9SS (Fig. <a href="#Fig5" class="usa-link">5A–C</a>). In contrast, the PorG<sup>C232S</sup> strain was similar to the <em>porG</em> mutant in that it was not pigmented on blood agar (Fig. <a href="#Fig5" class="usa-link">5A</a>), had no A-LPS signal above 60 kDa in the mAb-1B5 western blot (Fig. <a href="#Fig5" class="usa-link">5B</a>), and had no EDSL layer (Fig. <a href="#Fig5" class="usa-link">5C</a>). All these findings indicate that the T9SS is non-functional in the PorG<sup>C232S</sup> strain. To explore if the PorK and PorN were localised to the CE, we prepared CE fractions from WT and PorG<sup>C232S</sup> mutant. The sample was subjected to Western blot analysis using PorK and PorN antibodies. The immunoblot showed the presence of both PorK and PorN with similar abundances in both WT and PorG<sup>C232S</sup> mutant (Fig. <a href="#Fig5" class="usa-link">5D</a>). Next, we investigated if the PorKN rings were formed in the PorG<sup>C232S</sup> mutant by isolating the complex using our published purification protocol under non-reducing conditions<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. The sample was analysed on the gel to assess its purity and the identity of the proteins present (Fig. <a href="#Fig5" class="usa-link">5E</a>). Both PorK and PorN bands were identified, however, the PorG band was missing on the gel. This suggested that the disulphide bond between PorG and PorK is required for the stable interaction. The sample was then negatively stained and visualised under the electron microscope. The rings were visualised and appeared to be of unaltered size and shape (Fig. <a href="#Fig5" class="usa-link">5F</a>), thus Cys<sup>232</sup> of PorG is not essential for PorKN ring formation. Jointly, these results suggest that PorG may play an active role in the mechanism of type IX secretion that is dependent on its disulphide bond to PorK.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. The disulfide bond between PorG and PorK has a functional role in the T9SS.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365241_41467_2025_63163_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c92b/12365241/f40df5003836/41467_2025_63163_Fig5_HTML.jpg" loading="lazy" id="d33e1525" height="477" width="798" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong><em>P. gingivalis</em> PorG<sup>Cys232-Cys</sup>, PorG<sup>Cys232-Ser</sup> pigmentation on blood agar plates after 7 days of incubation. <strong>B</strong> Cell lysates from WT and the PorG<sup>Cys-Ser</sup> were electrophoresed on an SDS-PAGE gel and the proteins were transferred onto a nitrocellulose membrane and probed with mouse monoclonal antibody mAb 1B5 (anti-A-LPS). Coomassie stained gel shows protein load. <strong>C</strong> Cryo-electron micrographs showing the presence of an EDSL (white arrow) on WT and PorG<sup>Cys-Cys</sup> mutants cells and absence in the PorG<sup>Cys-Ser</sup> mutant cells. <strong>D</strong> Cell envelope fractions from the WT, PorG<sup>Cys-Ser</sup>, and PorG<sup>Cys-Cys</sup> were electrophoresed on an SDS-PAGE gel. The proteins were transferred onto a nitrocellulose membrane and probed with PorK and PorN antibodies. The Coomassie stained gel shows the protein load. <strong>E</strong> SDS-PAGE gel of purified PorKN complex from the PorG<sup>Cys232-Ser</sup> mutant cells. The protein band observed beneath PorK was identified as PG1881, which co-purified with the complex. <strong>F</strong> Purified PorKN complex from PorG<sup>Cys-Ser</sup> was negatively stained and visualised by cryo-electron microscopy, showing ring structures. Scale bar: 100 nm. Source data are provided as a <a href="#MOESM8" class="usa-link">Source Data</a> file.</p></figcaption></figure></section><section id="Sec9"><h3 class="pmc_sec_title">The attachment complex associates with the PorKN rings</h3>
<p id="Par22">Recent in-situ structural studies of the T9SS revealed distinct surface densities positioned above the PorKN rings<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Given that PorZ and PorU are known to be localised on the cell surface (Fig. <a href="#Fig1" class="usa-link">1A</a>), we investigated whether these observed densities could represent components of the Attachment Complexes. Additionally, while the function of the Attachment Complex is well-established, its structural organisation and spatial relationship to other T9SS sub-complexes remain unexplored. Therefore, we investigated whether the Attachment Complexes were associated with the PorKN rings. To address this question, we performed Co-IP with PorZ antibodies using lysates from ABK-, <em>porV</em>, <em>porU</em>, and <em>wbaP</em> mutant <em>P. gingivalis</em> cells. The immunoprecipitated material was quantified by mass spectrometry and MaxQuant relative to <em>porZ</em> mutant negative control. The label-free quantitation (LFQ) intensity ratios of the top 30 proteins in the ABK Co-IP were plotted on a heatmap, along with the LFQ intensity ratios for the <em>porV</em>, <em>porU</em>, and <em>wbaP</em> mutants (Fig. <a href="#Fig6" class="usa-link">6A</a>). The <em>wbaP</em> mutant lacks A-LPS and, consequently, while possessing a functional T9SS, fails to anchor cargo proteins to the cell surface, resulting in their release into the culture fluid (CF)<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. As expected, the other Attachment Complex components PorQ, PorU, and PorV were observed in the pulldown assay with high LFQ intensity ratios (Fig. <a href="#Fig6" class="usa-link">6A</a>). PorK, PorN, PorG, PorE, PorT, and Sov also exhibited high LFQ intensity ratios (Fig. <a href="#Fig6" class="usa-link">6A</a>), indicating that their presence in the pulldown assay was specific to the presence of PorZ. The iBAQ metric was used to quantify the relative abundance of proteins in the pulldown. After correction of the iBAQ values using the <em>porZ</em> mutant negative control, the values were quantified relative to PorQ. PorZ, PorU, PorQ, and PorV were the most abundant proteins present in the immunoprecipitated material, followed by PorK, PorN, and PorG, consistent with the Attachment Complexes being associated with the PorKN rings and PorG (Fig. <a href="#Fig6" class="usa-link">6B</a>). The Co-IP using PorZ antibodies was also performed on <em>porV</em>, <em>porU</em> and <em>wbaP</em> mutant cells (Table <a href="#Tab3" class="usa-link">3</a>). In the <em>porV</em> and <em>porU</em> mutants, PorU and PorV proteins were absent in the immunoprecipitated material as expected, but the PorKN rings and PorG were still present (Table <a href="#Tab3" class="usa-link">3</a>), suggesting that the PorQZ subcomplex<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> can form in the absence of PorU and PorV and still associate with the PorKN rings and PorG. A-LPS binds to PorZ and is a substrate of PorU<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. In the <em>wbaP</em> mutant, which does not express A-LPS, all four components of the attachment complex were enriched together with the components of the PorKN rings and PorG (Table <a href="#Tab3" class="usa-link">3</a>), indicating that A-LPS was not required for their association. Collectively, this suggests that the Attachment Complex associates directly with the PorKN ring and PorG.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Association of the attachment complexes with the PorKN rings.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365241_41467_2025_63163_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c92b/12365241/4125d7000bfa/41467_2025_63163_Fig6_HTML.jpg" loading="lazy" id="d33e1679" height="393" width="691" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The <em>P. gingivalis</em> strain lysates were subjected to coimmunoprecipitation using PorZ-specific antibodies. The <em>porZ</em> mutant was used as a negative control. The immunoprecipitated samples were digested with trypsin, analysed by mass spectrometry, and quantified using MaxQuant software. The ratio of LFQ intensity relative to the <em>porZ</em> control was determined. <strong>A</strong> The top 30 proteins for (ABK<sup>−</sup>/<em>porZ</em>) were plotted on a heatmap along with LFQ intensity ratios for the same proteins calculated for <em>porV</em>, <em>porU</em>, and <em>wbaP</em> mutants. <strong>B</strong> The <em>porV</em> mutant cells were labelled with PorZ primary antibodies followed by anti-rabbit gold (6 nm) secondary antibodies. The <em>porZ</em> mutant cells were used as a negative control. Cryo-ET was performed on the immunolabeled cells. In cryotomograms, PorKN rings were observed directly underneath the gold clusters. White arrows point to gold clusters and the yellow arrows point to the PorKN rings. Scale bars: 50 nm. <strong>C</strong> A cartoon representation of the PorKN rings connected to PorM (M) on the periplasmic side, and to the outer membrane (OM) β-barrels, including PorG (G), PorQ-PorZ (Q, Z), and PorV-PorU (V, U) on the OM side. <strong>D</strong> A schematic illustrating the proposed mechanism of the T9SS secretion and the attachment of cargo proteins to the cell surface. The rotary movement of PorM causes the PorKN ring and its associated proteins (PorG and attachment complexes) to rotate along with it. The Sov-PorV-cargo is positioned adjacent to the PorKN rings, held in place by the transient interaction between PorW and PorN. As PorG rotates, it provides the energy to push the PorV-cargo out of the Sov translocon and place it next to the attachment complex for sortase (PorU) reaction and anchorage to the cell surface via A-LPS. Source data are provided as a <a href="#MOESM8" class="usa-link">Source Data</a> file.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Relative abundance of PorZ binding partners in various T9SS mutants</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th colspan="1" rowspan="1"></th>
<th colspan="1" rowspan="1"></th>
<th colspan="4" rowspan="1"><em>iBAQ (%)</em></th>
</tr>
<tr>
<th colspan="1" rowspan="1">Protein name</th>
<th colspan="1" rowspan="1">LFQ ratio ABK-/<em>porZ</em>
</th>
<th colspan="1" rowspan="1">ABK-</th>
<th colspan="1" rowspan="1"><em>porV</em></th>
<th colspan="1" rowspan="1"><em>porU</em></th>
<th colspan="1" rowspan="1"><em>wbaP</em></th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">PorZ</td>
<td align="left" colspan="1" rowspan="1">655</td>
<td align="left" colspan="1" rowspan="1">255</td>
<td align="left" colspan="1" rowspan="1">324.8</td>
<td align="left" colspan="1" rowspan="1">389</td>
<td align="left" colspan="1" rowspan="1">679</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorQ</td>
<td align="left" colspan="1" rowspan="1">11810</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorU</td>
<td align="left" colspan="1" rowspan="1">3003</td>
<td align="left" colspan="1" rowspan="1">70.8</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">93</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorV</td>
<td align="left" colspan="1" rowspan="1">43</td>
<td align="left" colspan="1" rowspan="1">91.7</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">80</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorK</td>
<td align="left" colspan="1" rowspan="1">327</td>
<td align="left" colspan="1" rowspan="1">7.7</td>
<td align="left" colspan="1" rowspan="1">12.4</td>
<td align="left" colspan="1" rowspan="1">16.2</td>
<td align="left" colspan="1" rowspan="1">25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorN</td>
<td align="left" colspan="1" rowspan="1">67</td>
<td align="left" colspan="1" rowspan="1">4.6</td>
<td align="left" colspan="1" rowspan="1">7.8</td>
<td align="left" colspan="1" rowspan="1">12.8</td>
<td align="left" colspan="1" rowspan="1">25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorG</td>
<td align="left" colspan="1" rowspan="1">42</td>
<td align="left" colspan="1" rowspan="1">0.4</td>
<td align="left" colspan="1" rowspan="1">1.4</td>
<td align="left" colspan="1" rowspan="1">1.2</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorT</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">0.1</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
<td align="left" colspan="1" rowspan="1">0.3</td>
<td align="left" colspan="1" rowspan="1">0.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PorE</td>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1">0.1</td>
<td align="left" colspan="1" rowspan="1">0.1</td>
<td align="left" colspan="1" rowspan="1">11.9</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sov</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par23">To validate this finding and investigate if multiple Attachment Complexes were co-located and are above the PorKN rings in vivo, <em>porV</em> mutant cells were labelled with anti-PorZ and secondary antibodies conjugated to gold and imaged by cryo-electron tomography (cryo-ET). The <em>porV</em> mutant was selected for this work because it lacks an EDSL, providing better access of the antibodies to the PorZ antigen. From 62 tomograms of <em>porV</em>- cells we observed an average of 4 clusters of gold particles per cell. Within each cluster were up to 4 gold particles and tomogram sections showed the PorKN rings directly beneath the gold clusters (Fig. <a href="#Fig6" class="usa-link">6B</a> and Supplementary Movie <a href="#MOESM5" class="usa-link">1</a>). In comparison, nanogold cryo-ET on a <em>porZ</em> mutant only showed a small number of labelled cells relative to the <em>porV</em> mutant, indicating the specificity of the PorZ antibodies. Together, these results indicate that multiple Attachment Complexes are co-located directly above the PorKN rings.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par24">The T9SS in <em>P. gingivalis</em> is comprised of at least 20 proteins. We previously showed that the PorK and PorN components of the T9SS form large 50 nm ring-shaped structures, which are intimately associated with PorG<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. The exact role of these rings in the mechanistic function of the T9SS is unknown. We proposed that the large PorKN rings create a diffusion barrier that retains all the T9SS components within the ring to promote efficient and coordinated protein secretion and cell surface attachment<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The PorKN ring makes a direct contact with the PorL/M motor<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> that powers the T9SS, therefore, the ring is also presumably involved in distributing the energy<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>.</p>
<p id="Par25">The high-resolution structure of the PorKN rings presented here reveals that PorK forms a core domain resembling a formylglycine-generating enzyme (FGE). These enzymes catalyse the conversion of a protein cysteine to a protein formylglycine via a proposed copper oxidase mechanism<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The two Cys residues in the FGE active site are missing in PorK, indicating that it cannot perform this function. The PorK-specific portion is inserted within the FGE sequence between residue 103 and 261 and is located on the outer side of each subunit, forming the outer rim and the interlocking wedge that fits snuggly within the adjacent PorK subunits. Hence, one of the functions of the PorK-specific appendage appears to be ring formation. The ring structures in other secretion systems either assemble around a solid core or possess intrinsic properties that facilitate self-assembly. For instance, FliF of Flagella motor self-assembles to form the MS-ring in the inner membrane<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Since there is no solid core present in the T9SS, PorK may possess intrinsic properties that enable it to self-assemble and form a ring structure in the periplasm.</p>
<p id="Par26">MS analysis showed that the <em>O</em>-glycosylation sites of PorK and PorN were glycosylated with full length glycans (7 sugars), however in the structure, only the first sugar was resolved at PorK Ser<sup>68</sup>. No sugars were resolved for the two identified glycosylation sites in PorN. The resolution of the structure was not sufficient to resolve the isomeric conformations of the sugars. Presumably, the hidden sugars were not visible due to having high flexibility. The visible sugars participated in multiple hydrogen bonds to different parts of the protein sequence, potentially contributing to the stability of the structure. The role of <em>O</em>-glycans on PorK and PorN is unclear, however, SprE, a homologue of PorW in <em>P. gingivalis</em>, was also found to be <em>O</em>-glycosylated, and this glycosylation was essential for its stability<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. We therefore predict a similar role for <em>O</em>-glycosylation in PorK and PorN.</p>
<p id="Par27">The structure of PorN was similar to the published crystal structure<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> (RMSD: 1.112), with the following notable exceptions (Supplementary Fig. <a href="#MOESM1" class="usa-link">2C</a>). The “cog” was displaced by ~13 Å, and the N-terminal segment (Leu<sup>50</sup>-Asn<sup>67</sup>) as well as the C-terminal segment (Arg<sup>263</sup>-Lys<sup>303</sup>) were resolved here. PorN binds deeply within a cavity produced by two PorK subunits, suggesting that in the ring assembly process, PorK multimerization (at least dimer) may precede the incorporation of PorN subunits. The stability of the PorKN interaction is underscored computationally with a very low estimated K<sup>d</sup>, and experimentally by the ability of the PorKN rings to withstand the harsh purification protocol, including overnight centrifugation in 30% w/v CsCl and short periods in 0.5% DDM, 500 mM NaCl and 1 M urea. Together, these data indicate a solid ring under physiological conditions. The cog-like projections of PorN in the periplasm suggest a mechanism by which the PorLM motor could transfer its rotational energy. AlphaFold modelling of PorM with PorN positioned PorM between the cog-like projections of PorN. Since PorLM motors have been proposed to exhibit rotary movement<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, the rotation of PorM rotors interlocked with PorN cogs would cause the whole PorKN ring to rotate.</p>
<p id="Par28">PorG co-purifies with the PorKN rings, and we found it to associate with PorK via a disulphide bond between conserved cysteine residues. The inability to resolve PorG in the structure may be due to its low stoichiometry, small size and/or flexibility. Disruption of the disulphide bond made the T9SS non-functional, however, the rings now comprising only PorK and PorN appeared similar in structure to the native PorKN rings. This suggests that the disulphide bond with PorG plays a crucial mechanistic role in the function of the T9SS rather than in the assembly of the PorKN rings. PorG has three periplasmic loops. Two of these are the more common short loops of 3–5 aa that are modelled to be buried amongst the hydrophilic phospholipid head-groups, but one loop, Ser<sup>121</sup>-Arg<sup>131</sup> and the C-terminal extension are long enough to traverse the gap and interact with the surface of PorK. The model predicts PorG to be located on the inside edge of PorK. This location is supported by our previously observed Lys-Lys crosslinks between the periplasmic loop of PorG (Lys<sup>125</sup>) and Lys<sup>480</sup> and Lys<sup>483</sup> of PorK<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>, with a modelled distance of only 7 Å and 15 Å, respectively.</p>
<p id="Par29">We also demonstrate that the Attachment Complex, consisting of PorU, PorV, PorQ, and PorZ, associates with the PorKN rings. Using immunogold and cryo-ET, we show that this complex is co-located directly above the PorKN rings. Recently, Song et al. solved a low-resolution in-situ structure of the <em>P. gingivalis</em> T9SS<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. An 8-fold symmetry ring of cell surface densities was observed directly above the PorKN periplasmic rings. Based on the observation that these surface densities disappeared in a <em>sov</em> mutant, the authors proposed that the surface densities were individual Sov subunits. Interestingly, the densities were also absent in <em>porL</em> and <em>porM</em> mutants<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Our data however, show that Sov is properly located in the OM in <em>porL</em> and <em>porM</em> mutants maintaining its functional interaction network including PorV, PorW, PorK and PorN (Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>). Similarly, a recent study showed that SprA, the Sov orthologue in <em>F. johnsoniae</em>, maintains its complex with PorV and SprE (PorW orthologue) in the <em>gldL</em> (<em>porL)</em> mutant<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Therefore, if the surface densities observed in the in-situ structure by Song et al. were Sov, then those densities shouldn’t disappear in the <em>porL</em> mutant. These results suggest that the cell surface rings are unlikely to be the Sov translocons alone. PorU and PorZ of the Attachment Complexes are also T9SS cargo proteins that cannot be secreted to the cell surface in any T9SS component mutants, such as <em>sov</em>, <em>porL</em> and <em>porM</em>, and therefore it is likely that the observed cell surface densities include the Attachment Complexes, which are of similar overall dimensions to Sov.</p>
<p id="Par30">Collectively, based on our results, we propose the following. PorG OMPs are symmetrically arranged on top of the flat PorK ring surface, secured firmly via disulphide bond linkages with the PorK subunits. The OM β- barrels PorQ (bound to PorZ) and PorV (bound to PorU) are also positioned above PorK monomers in close proximity to PorG (Fig. <a href="#Fig6" class="usa-link">6C</a>). The rotation of the PorKN rings would cause the tethered PorGs and the associated Attachment Complexes to also rotate. Recently, it was proposed that energy is needed to detach cargo bound PorV from the Sov translocon<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. The loaded Sov-PorV-cargo complex is likely to be positioned near the ring via dynamic interaction with the Sov/PorN bridging protein PorW<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> leading to our proposed model of T9SS secretion and attachment in <em>P. gingivalis</em> (Fig. <a href="#Fig6" class="usa-link">6D</a>).</p>
<p id="Par31">In the model the energy of rotation provided by the T9SS PorLM motors is conveyed to the PorKN rings. The bent PorM rotors fit into the PorN cogs to drive the rotation of the ring along with the disulphide-bonded PorG and associated Attachment Complexes. With Sov in close proximity to the rotating ring, it positions the PorV-cargo in the path of the rotating PorG. When the PorG and Attachment Complexes are driven past Sov-PorV, PorG dislodges PorV, pushing it away from Sov, and simultaneously pulling the cargo through Sov’s lateral gate, thereby completing secretion (Fig. <a href="#Fig6" class="usa-link">6D</a>). The extracted PorV-cargo complex may then bind to the Attachment Complex, also rotating past, leading to PorU-catalysed CTD cleavage and A-LPS attachment of the cargo. This represents an efficient and coordinated model of secretion and attachment, since the cargo substrates would be transferred directly from Sov to the Attachment Complexes by the action of PorG, rather than utilising the less efficient shuttle model that we previously suggested<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Our previous data can nevertheless be explained, whereby in the absence of PorU or PorZ of the Attachment Complexes, the PorV cargo complexes accumulate<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. In these mutants, PorG can presumably still function to extract the PorV-cargo complexes from Sov, but without functional Attachment Complexes, the PorV-cargo cannot be processed and therefore accumulate. The model also allows for one or more Sov translocons per ring.</p>
<p id="Par32">While our study provides insights into the structure of the PorKN ring as a potential cogwheel, it is limited by the lack of direct evidence of rotational movement. Hence our future work will focus on targeted mutagenesis of key residues at the PorK-PorN interface and PorN-PorM interface as well as demonstrating rotation of the PorKN cogwheel with PorG attached to validate our model of T9SS cargo protein secretion. In conclusion, we present a high-resolution structure of the T9SS PorKN rings and demonstrate a close association with other components of the T9SS, in particular PorG and the Attachment Complexes. Based on this ring structure and association, we propose that the PorLM motor drives the rotation of the PorKN cogwheel in the periplasm together with the OM PorG and associated Attachment Complexes, providing the energy to complete protein secretion and the coordinated cell surface attachment of the secreted cargo. The movement of PorG and the Attachment Complexes rotationally through the OM, driven by rotating periplasmic cog wheels of a protein nanomachine, implies a paradigm shift from the widely considered rigid OM theory to one of fluidity. These findings provide insights into the molecular mechanism of T9SS secretion and attachment in <em>P. gingivalis</em>.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">
<em>P. gingivalis</em> culture conditions</h3>
<p id="Par33"><em>P. gingivalis</em> strains were grown on solid medium (TSBHI agar) containing trypticase soy agar (40 g/L), brain heart infusion (5 g/L), 5% (v/v) lysed defibrinated horse blood, cysteine hydrochloride (0.5 g/L), hemin (5 mg/L) and menadione (5 µg/mL) or in TSBHI broth (25 g/L tryptic soy, 30 g/L brain heart infusion, 0.5 g/L cysteine hydrochloride, 5 mg/L haemin and 1 mg/L menadione) under anaerobic conditions as described previously<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The appropriate antibiotic was included in the media to select for the relevant antibiotic marker of the strain, but omitted in the final harvested liquid culture.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Construction of <em>porG</em> deletion mutant</h3>
<p id="Par34">The <em>porG</em> deletion mutant in the W50 strain was generated by double cross-over recombination using the suicide plasmid pΔPorG-ermF that contained upstream and downstream regions of <em>porG</em> (<em>pg0189</em>) directionally cloned adjacent to an erythromycin resistance cassette (<em>ermF</em>). The upstream and downstream regions of <em>porG</em> were amplified by PCR using W50 genomic DNA as a template and the following primer pairs: Upstream-F (ACGC<span class="text-underline">GCATGC</span>CCACAGAACAAGCCGAAGGTATAA (SphI site underlined)/ Upstream-R (ACGC<span class="text-underline">CCATGG</span>TGAGCAGCTATCCGGCCGTTAC) (NcoI site underlined) and Downstream-F (ACGC<span class="text-underline">CTGCAG</span>GGATGCGCTGACGGATAAAAGTG) (PstI site underlined)/ Downstream-R (ACGC<span class="text-underline">ATGCAT</span>ACCATCCATGACCAAAGCGATATG) (NsiI site underlined). The PCR amplicons were digested with the appropriate restriction endonucleases and sequentially cloned upstream and downstream of <em>ermF</em> in pAL30<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Once the sequence integrity of the cloned DNA was confirmed pΔPorG-ermF was linearised using NsiI and electroporated into <em>P</em>. <em>gingivalis</em> W50 with transformants selected on TSBHI agar plates supplemented with 10 mg/L of erythromycin. The correct homologous recombination of transformants was verified using PCR with primer pair TACTATTATGCCACAGAAC and GACAACCACCCGACTTTGAA. The absence of PorG protein was verified by mass spectrometry.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Construction of PorG complementation strain</h3>
<p id="Par35">To construct the <em>porG</em> complementation plasmid, pKD713-PstI that contains a recombinant cassette of the <em>fimA</em> neighbouring gene regions and the <em>tetQ</em> gene selection marker (a gift from Dr Mikio Shoji) was utilised. DNA spanning from a 3′ region of the <em>pg0188</em> locus to <em>porG</em> was PCR-amplified using primers -ACGC<span class="text-underline">CTGCAG</span>CCATGGTTTGGCCACAGGGAA and - ACGC<span class="text-underline">CTGCAG</span>GCACTATAACTGCTGACTGGAG (PstI site underlined). The amplicon was cloned into the pKD713-PstI through PstI digestion and ligation to create pKD713-PorG. Once it was verified that the cloned DNA sequence was correct, pKD713-PorG was linearised using NotI and introduced into the <em>porG</em> mutant cells by electroporation and plated on TSBHI agar supplemented with 1 mg/L tetracycline to select complementation clones. Insertion of <em>porG</em> into the <em>fimA</em> locus was verified by genomic PCR using primers corresponding to <em>porG</em>, the tetracycline resistance gene (<em>tetQ</em>) and the <em>pg2133</em> locus.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Construction of targeted PorG<sup>Cys232-Ser</sup><em>P. gingivalis</em> mutant strain</h3>
<p id="Par36">A series of PCR and SOE-PCR reactions were conducted to produce <em>pg0189</em> Cys232Ser codon mutation and non-mutated control recombination cassettes. The <em>pg0188</em> open reading frame 3’ overlaps the 5’ of the <em>pg0189</em> open reading frame by 14 nucleotides, so the recombination marker <em>ermF</em> was placed 3’ to <em>pg0189</em> to not disrupt the putative operon. Template DNAs were pAL30<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> (PCR-1) or <em>P. gingivalis</em> W50 genomic DNA (PCR-2, PCR-3 and PCR-4). PCR-1 used primers ermFprom_for (GATAGCTTCCGCTATTGCTT) and ermF-Rev (GACAACCACCCGACTTTGAA) to produce <em>ermF</em> with the <em>ermF</em> promoter. The PCR-2 amplicon used a forward primer with a fusion of ermF 5’ and <em>pg0189-pg0190</em> intergenic nucleotides. The PCR-2 primers were ermF_pg0190_For, <span class="text-underline">AAGCAATAGCGGAAGCTATC</span>CCTCGAAACATTCTAAGGGAAT (<em>ermF</em> nucleotides underlined) and a reverse primer that annealed within <em>pg0190</em>, Reverse_3_PstI, ACGC<span class="text-underline">CTGCAG</span>CTTCCGGACGATTCCAATTCT (PstI site underlined). PCR-3 and PCR-4 used forward primer that annealed to <em>pg0189</em>, Forward Oli_1_SphI (ACGC<span class="text-underline">GCATGC</span>GGCCGGGTTGTCTTTGGGTT, SphI site underlined) and respective reverse primers that annealed to <em>pg0189</em> and had an <em>ermF</em> tag at 5’ (underlined below), Reverse-1A-WT (<span class="text-underline">TTCAAAGTCGGGTGGTTGT</span>CCCCTATTGTTTATTACAAAAAGTC) and Reverse-1B-Ser (<span class="text-underline">TTCAAAGTCGGGTGGTTGT</span>CCCCTATTGTTTATTACTAAAAGTC). PCR-1 and PCR-2 were denatured and annealed by the homologous nucleotides, extended, then amplified with external primers ermF-Rev and Reverse_3_PstI to produce SOE-1 product. SOE-1 and either PCR-3 or PCR-4 were denatured and annealed, extended, then amplified with external primers Forward Oli_1_SphI and Reverse_3_PstI to produce SOE-2 and SOE-3, respectively. SOE-2 and SOE-3 were digested using SphI and PstI and ligated to the pAL30 vector. After verification of the fidelity of the inserted DNA by sequencing, the plasmid was digested with SacI and transformed into <em>P. gingivalis</em> W50 by electroporation. Erythromycin-resistant transformants were selected from TSBHI agar plates, and correct integration into the genome was determined using PCR with primer pair Forward Check (GCGGATTTTAAGGTCGGCTT) and Reverse-Check (AGCATCTCATTCATGATAGC).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Gingipain activity assay</h3>
<p id="Par37">Arg- and Lys-specific proteolytic activity was determined using the synthetic chromogenic substrates N-benzoyl-DL-arginine-4-nitroanilide hydrochloride (BapNA) and N-(<em>p</em>-tosyl)-Gly-Pro-Lys 4-nitroanilide acetate salt (GPKNA) (Sigma Aldrich), respectively. Briefly, <em>P. gingivalis</em> in broth culture were pelleted at 8000 g for 10 min then resuspended in a buffer containing 50 mM Tris-HCl, pH 7.4, 5 mM CaCl<sub>2</sub> and 150 mM NaCl. The supernatant (culture fluid, CF) containing vesicles were also collected. Assays were conducted in 96-well microtiter plates with 5.6 × 10<sup>6</sup> cells or 140 µL of CF per well to which a final concentration of 20 mM L-cysteine, pH 8.0 was added. After the mixture was incubated at 37 °C for 10 min, a final concentration of 1 mM substrate (BapNA or GPKNA) was added to it and substrate cleavage was determined by measuring the absorbance at 405 nm at 10 s intervals for ~60 min at 37 °C using a PerkinElmer 1420 Multilabel Counter VICTOR3™. For CF activity assay, 140 µL of each sample was used instead of the cells.</p></section><section id="Sec17"><h3 class="pmc_sec_title">SDS-PAGE and western blots</h3>
<p id="Par38">Protein samples were mixed with SDS- loading dye and analysed on NuPAGE Bis-Tris precast gels electrophoresed using MOPS SDS running buffer. The gels were stained overnight with 0.1% w/v Coomassie Blue G-250 in 17% w/v ammonium sulphate, 34% v/v methanol and 3% v/v o-phosphoric acid. For western blotting, protein samples were resolved on SDS-PAGE gels and transferred onto a nitrocellulose membrane. The membrane was blocked with 5% skim milk powder in PBS-tween (0.05% (v/v) Tween-20 in PBS). The membranes were probed with mAb-1B5 (a kind gift from Professor M.A Curtis), PorK or PorN primary antibodies followed by goat anti-mouse or anti-rabbit HRP conjugated secondary antibodies. The signal was developed using SuperSignal West Pico chemiluminescent substrate and visualised with a LAS-3000 imaging system. For an example of presentation of full scan blots, see the <a href="#MOESM8" class="usa-link">Source Data</a> file.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Cell envelope (CE) preparation</h3>
<p id="Par39">The cell envelope was prepared as described in Glew et al.<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Briefly, <em>P. gingivalis</em> cells were washed and resuspended in 0.5× PBS, 5 mM MgCl<sub>2</sub> in a 50 mL tube and placed in a beaker containing an ice/water mixture and sonicated using an ultrasonic processor (model CPX 750, Cole Parmer) fitted with a 6.5 mm tapered microtip. The amplitude was set to 40%, pulser to 1 s on, 2 s off for a total of 30 min. The CE was collected by centrifugation at 48,400 <em>g</em> for 20 min and washed in 1× PBS before storing in −80 °C for further analysis.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Blue native PAGE immunoblot and co-immunoprecipitation</h3>
<p id="Par40">Blue native PAGE immunoblot and Co-IP were conducted using cell lysates in buffers containing 1% DDM as described by Gorasia et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Purification of PorKNG complex</h3>
<p id="Par41">PorKNG complexes were purified as per our published protocol<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. For the complex preparation under reducing conditions, 5 mM DTT was added to the cell lysate and 1 mM DTT in all the subsequent steps. For the cryo-EM single particle analysis, 1 M urea was added to the purified sample immediately before application to the EM grid.</p>
<p id="Par42">For the crude preparation, the <em>P</em>. <em>gingivalis</em> cells (ABK- (<em>P. gingivalis</em> cells lacking gingipains) and <em>porG</em> mutant) were lysed and digested with lysozyme as per our published protocol<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> and the pelleted complexes were resuspended in 50 mM Tris-HCL, pH 7.5, 500 mM NaCl, 1% DDM and 5 mM EDTA. Following centrifugation at 17,000 <em>g</em> for 10 min, the resulting supernatant was layered on a 30% sucrose cushion and centrifuged at 104,677 <em>g</em> for 2 h at 4 °C in a TLS-55 rotor (Beckman Coulter). The pellet was resuspended in 10 mM Tris-HCl, pH 7.5, 50 mM NaCl and 0.5% DDM and stored at 4 °C.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Electron microscopy</h3>
<p id="Par43">For negative staining of protein complexes samples were adsorbed onto glow discharged Formvar-carbon films supported on 400 mesh copper grids. Samples were negatively stained with aqueous uranyl acetate and viewed under an FEI Tecnai F30 operated at 200 kV equipped with a CETA 4kx4k camera (FEI, NL). Micrographs were obtained under low-dose conditions (~20 e<sup>−</sup>/Å<sup>2</sup>) with defocus values of ~2 μm. For cryo-electron microscopy of protein complexes and <em>P</em>. <em>gingivalis</em> cells, samples were applied onto a glow discharged Quantifoil (respectively R1.2/1.3 and R3.5/1) holey carbon film mounted on a 200 mesh copper grid. The grid was blotted for 4 s with a blot force of −1 using a Vitrobot Mark IV (Thermo Fisher Scientific) set to 22 °C and 100% humidity and plunge frozen into liquid ethane. For cells, transmission electron microscopy was carried out under cryogenic conditions using a FEI Tecnai F30 operated as described above. Micrographs were obtained under low-dose conditions (~25 e<sup>−</sup>/Å<sup>2</sup>) with defocus values of ~2 μm. For protein complex, 1 M urea was added to the complex to limit strong self-aggregation and cryo-EM data collection was carried out on a Titan Krios G4 cryo-electron microscope equipped with a K3 detector and a BioQuantum imaging filter (Gatan) at 300 kV in counting mode. Movie stacks were collected at a nominal magnification of 64,000×, a pixel size of 1.32 Å, and a defocus of −0.6 to −2 μm with a fluency of 13.5 e<sup>−</sup>/pixel/s. Each movie was a result of 5.34 s exposures with a total accumulated electron exposure of 54 e<sup>−</sup>/Å<sup>2</sup> which were fractioned into 40 frames. Automated data collection in AFIS mode was performed using EPU (Thermo Fisher Scientific). An energy filter slit width of 10 eV and a 50 μm C2 condenser aperture were used during imaging.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Cryo-EM data processing</h3>
<p id="Par44">Movies were collected (8909), motion-corrected and CTF fitted using CryoSPARC Patch motion correction and Patch CTF jobs<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Manual picking was used to pick ~200 particles, subject them to class average with 3 classes corresponding to top, side and tilted views. Picking using these templates yielded 556,522 initial particles from which 121,937 final particles were selected using two rounds of 2D classification. Ab initio modelling with 50,000 particles was carried out and output 4 model. The top two models were used in heterogeneous refinement with the 121,937 particles. 77,661 particles were selected and subjected to homogenous refinement in C1 then C2. Symmetry expansion and subtraction on one of the ring was carried out. The resulting particles were subjected to C1 homogenous refinement and 3D classification in C1, 47.3% resulted in a map with 33 repeats and 52.3% in a ring with 32 repeats. Both maps were subjected to C1 refinement then respectively C33 and C32 refinement. Particles were further cleaned up with a round of 2D classes and a final round of local refinement (masking the detergent) was carried out in respectively C33 (3.57 Å, EMD-48741) [<a href="https://www.ebi.ac.uk/emdb/EMD-48741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/emdb/EMD-48741</a>] and C32 (3.63 Å, EMD-48722) [<a href="https://www.ebi.ac.uk/emdb/EMD-48722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/emdb/EMD-48722</a>].</p></section><section id="Sec23"><h3 class="pmc_sec_title">Structure refinement</h3>
<p id="Par45">Initial model of PorK and PorN were generated using Alpha Fold3 with three copies as a hexamer (three copies of the PorK-N dimer) and fitted to the experimental map with manual modification in Coot<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup> and real space refinement with Phenix<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> with statistics reported for the central dimer of the modelled hexamer (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). Complete PorKN ring models were generated by duplication of the refined dimer structure using C33 symmetry (PDB: 9MYJ).</p></section><section id="Sec24"><h3 class="pmc_sec_title">Immunogold labelling and cryo-electron tomography</h3>
<p id="Par46">PorZ antibodies were generated by Genscript against amino acid residues 29-776 of <em>P. gingivalis</em> PorZ. <em>P. gingivalis</em> cells were labelled with PorZ antibodies as detailed in Chen et al.<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> with the following modifications. PorZ antibodies were used at a concentration of 30 µg/mL, and three washes were included after the incubation with anti-rabbit secondary gold (6 nm) antibody. The immunogold-treated positive and appropriate negative control samples were vitrified using a Vitrobot Mark IV (Thermo Fisher Scientific). 4 μL of the sample was pipetted on a glow discharged Quantifoil extra thick copper EM-grids (R2/2 200 mesh, Electron Microscopy Sciences). The excess liquid was blotted (blot force 7, blot time 7 s) using a Whatman filter paper and plunge frozen into liquid ethane.</p>
<p id="Par47">Data was collected on Titan Krios G4, 300 keV FEG cryo transmission electron microscope, equipped with a Gatan K3 direct electron detector equipped with an energy filter set to 20 eV. The tilt series was acquired in movie mode from −60° to +60° at 3° intervals with a total electron dose of 120 e<sup>−</sup>/Å<sup>2</sup>, defocus of −8 μm and a pixel size of 3.4 Å. Subsequently, tilt series were aligned using IMOD (Version 4.11.5)<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. The 2 K binned micrographs were reconstructed into 500 nm thick 1 K SIRT tomograms by Tomo3D (Version 2.2)<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>.</p></section><section id="Sec25"><h3 class="pmc_sec_title">In gel digestion and mass spectrometry</h3>
<p id="Par48">In-gel digestions and LC-MS/MS analysis were performed as essentially described in refs. <sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Relative abundances of proteins were quantified by MaxQuant (version 1.5.3.30)<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Raw MS/MS files were searched against the <em>P. gingivalis</em> W83 or ATCC 33277 database. The default MaxQuant parameters were used, except the LFQ minimum ratio count was set to 1 and the match between runs was selected. MaxQuant normalised the data set as part of data processing.</p>
<p id="Par49">To identify the disulphide bond, raw MS/MS files were analysed using pLink software and manually reviewed to detect the disulphide linkage between PorG and PorK.</p>
<p id="Par50">To identify glycosylation sites on PorK and PorN, the purified PorKNG complex was electrophoresed on an SDS-PAGE gel. The bands corresponding to PorK and PorN were subjected to trypsin digestion and the tryptic fragments analysed as essentially described in ref. <sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Proteins and peptides were identified using Mascot software. All searches were conducted using trypsin with up to two missed cleavages allowed, peptide mass tolerance was set to 10 ppm, fragment mass tolerance to 0.04 Da, and a fixed modification of carbamidomethyl (C), variable modifications of oxidation (M), 792.25 (S, T), 1352.42 (S, T), 1394.42 (S, T) and 1436.43 (S, T).</p></section><section id="Sec26"><h3 class="pmc_sec_title">Gene cluster analysis</h3>
<p id="Par51">Gene cluster analysis was performed using the Kegg SSDB gene cluster tool, accessible from <a href="https://www.genome.jp/kegg/genes.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genome.jp/kegg/genes.html</a>, using default parameters (gap size = 0; Threshold = 100).</p></section><section id="Sec27"><h3 class="pmc_sec_title">AlphaFold modelling and dissociation constant (K<sub>d</sub>) calculations</h3>
<p id="Par52">Model structures for various components of T9SS and their complexes were generated using AlphaFold3 through the AlphaFold3 server interface<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Where necessary, membrane effects on proteins during model building were simulated by the inclusion of multiple copies (usually 100) of myristic acid in the modelling process. Full membrane complexes were generated for protein models using the OPM<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> and CHARMM-GUI<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> servers. Interfacial energies in protein-protein complexes were predicted using the PRODIGY server<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> using default settings other than the temperature, which was set to 37 °C.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par53">All results presented in the paper are representative with similar data obtained for at least three biological replicates unless otherwise specified. Data were analysed using a two-tailed unpaired Student’s <em>t</em> test to compare means between two groups. Differences were considered statistically significant at <em>p</em> &lt; 0.05.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par54">Further information on research design is available in the <a href="#MOESM6" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec30"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63163_MOESM2_ESM.pdf</a><sup> (32.8KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM3_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (27.4KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (140.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM5_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (11.4MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer review file</a><sup> (472.2KB, pdf) </sup>
</div></div></section></section><section id="Sec31"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM8_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (105.5KB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by the Australian Government, Department of Industry, Innovation and Science (Grant ID: 20080108), E.C.R., the Australian National Health and Medical Research Council (Grant IDs: 1193647, 1123866 and APP1196924), E.C.R. and D.G. the Australian Research Council (Grant ID: DP200100914), E.C.R. and the Australian Dental Research Foundation (Grant ID: 492-2019), D.G.G. and a Cumming Global Centre for Pandemic Therapeutics, Foundation Grant, D.G. We acknowledge the use of the Mass Spectrometry and Proteomics Facility and the Ian Holmes Imaging Centre both at the Bio21 Institute, The University of Melbourne, Australia.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>D.G.G. conceptualisation, methodology, data acquisition, investigation, formal analysis, data curation, funding acquisition, and writing—original draft; E.H. data acquisition, investigation, formal analysis, data curation, C.M., M.M., S.V. and D.G. data acquisition and writing—review and editing, C.S., L.Z. and M.T.D. methodology and writing—review and editing; P.D.V., conceptualisation, investigation, data curation, supervision, funding acquisition and writing—review and editing; E.C.R. conceptualisation, supervision, investigation, funding acquisition and writing—review and editing. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par55"><em>Nature Communications</em> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Cryo-EM density maps and atomic coordinates are deposited in the Electron Microscopy Databank (EMDB) with the following accession number: <a href="https://www.ebi.ac.uk/emdb/EMD-48741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-48741</a>, <a href="https://www.ebi.ac.uk/emdb/EMD-48722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-48722</a>. Atomic coordinates are deposited in the Protein Data Bank (PDB) with the following accession number PDB:9MYJ [10.2210/pdb9MYJ/pdb]. The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD062825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD062825</a> and <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD062800" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD062800</a>. <a href="#Sec31" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par56">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Dhana G. Gorasia, Eric Hanssen.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Dhana G. Gorasia, Email: gorasiad@unimelb.edu.au</p>
<p>Paul D. Veith, Email: pdv@unimelb.edu.au</p>
<p>Eric C. Reynolds, Email: e.reynolds@unimelb.edu.au</p></section><section id="sec32"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63163-1.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Arigbede, A. O., Babatope, B. O. &amp; Bamidele, M. K. Periodontitis and systemic diseases: a literature review. <em>J. Indian Soc. Periodontol.</em><strong>16</strong>, 487–491 (2012).
</cite> [<a href="https://doi.org/10.4103/0972-124X.106878" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3590713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23493942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arigbede,%20A.%20O.,%20Babatope,%20B.%20O.%20&amp;%20Bamidele,%20M.%20K.%20Periodontitis%20and%20systemic%20diseases:%20a%20literature%20review.%20J.%20Indian%20Soc.%20Periodontol.16,%20487%E2%80%93491%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Bui, F. Q. et al. Association between periodontal pathogens and systemic disease. <em>Biomed. J.</em><strong>42</strong>, 27–35 (2019).
</cite> [<a href="https://doi.org/10.1016/j.bj.2018.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6468093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30987702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bui,%20F.%20Q.%20et%20al.%20Association%20between%20periodontal%20pathogens%20and%20systemic%20disease.%20Biomed.%20J.42,%2027%E2%80%9335%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Byrne, S. J. et al. Microbiome profiles of non-responding and responding paired periodontitis sites within the same participants following non-surgical treatment. <em>J. Oral Microbiol.</em><strong>14</strong>, 2043595 (2022).
</cite> [<a href="https://doi.org/10.1080/20002297.2022.2043595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8920355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35295980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Byrne,%20S.%20J.%20et%20al.%20Microbiome%20profiles%20of%20non-responding%20and%20responding%20paired%20periodontitis%20sites%20within%20the%20same%20participants%20following%20non-surgical%20treatment.%20J.%20Oral%20Microbiol.14,%202043595%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Byrne, S. J. et al. Progression of chronic periodontitis can be predicted by the levels of <em>Porphyromonas gingivalis</em> and <em>Treponema denticola</em> in subgingival plaque. <em>Oral Microbiol. Immunol.</em><strong>24</strong>, 469–477 (2009).
</cite> [<a href="https://doi.org/10.1111/j.1399-302X.2009.00544.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19832799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Byrne,%20S.%20J.%20et%20al.%20Progression%20of%20chronic%20periodontitis%20can%20be%20predicted%20by%20the%20levels%20of%20Porphyromonas%20gingivalis%20and%20Treponema%20denticola%20in%20subgingival%20plaque.%20Oral%20Microbiol.%20Immunol.24,%20469%E2%80%93477%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Hajishengallis, G., Darveau, R. P. &amp; Curtis, M. A. The keystone-pathogen hypothesis. <em>Nat. Rev. Microbiol.</em><strong>10</strong>, 717–725 (2012).
</cite> [<a href="https://doi.org/10.1038/nrmicro2873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3498498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22941505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hajishengallis,%20G.,%20Darveau,%20R.%20P.%20&amp;%20Curtis,%20M.%20A.%20The%20keystone-pathogen%20hypothesis.%20Nat.%20Rev.%20Microbiol.10,%20717%E2%80%93725%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Sato, K. et al. A protein secretion system linked to bacteroidete gliding motility and pathogenesis. <em>Proc. Natl. Acad. Sci. USA</em><strong>107</strong>, 276–281 (2010).
</cite> [<a href="https://doi.org/10.1073/pnas.0912010107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2806738/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19966289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sato,%20K.%20et%20al.%20A%20protein%20secretion%20system%20linked%20to%20bacteroidete%20gliding%20motility%20and%20pathogenesis.%20Proc.%20Natl.%20Acad.%20Sci.%20USA107,%20276%E2%80%93281%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Paillat, M., Lunar Silva, I., Cascales, E. &amp; Doan, T. A journey with type IX secretion system effectors: selection, transport, processing and activities. <em>Microbiology</em><strong>169</strong>, 001320 (2023).
</cite> [<a href="https://doi.org/10.1099/mic.0.001320" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10202324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37043368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paillat,%20M.,%20Lunar%20Silva,%20I.,%20Cascales,%20E.%20&amp;%20Doan,%20T.%20A%20journey%20with%20type%20IX%20secretion%20system%20effectors:%20selection,%20transport,%20processing%20and%20activities.%20Microbiology169,%20001320%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Veith, P. D., Glew, M. D., Gorasia, D. G., Cascales, E. &amp; Reynolds, E. C. The type IX secretion system and its role in bacterial function and pathogenesis. <em>J. Dent. Res.</em><strong>101</strong>, 374–383 (2022).
</cite> [<a href="https://doi.org/10.1177/00220345211051599" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34889148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Veith,%20P.%20D.,%20Glew,%20M.%20D.,%20Gorasia,%20D.%20G.,%20Cascales,%20E.%20&amp;%20Reynolds,%20E.%20C.%20The%20type%20IX%20secretion%20system%20and%20its%20role%20in%20bacterial%20function%20and%20pathogenesis.%20J.%20Dent.%20Res.101,%20374%E2%80%93383%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Lasica, A. M., Ksiazek, M., Madej, M. &amp; Potempa, J. The type IX secretion system (T9SS): highlights and recent insights into its structure and function. <em>Front. Cell Infect. Microbiol.</em><strong>7</strong>, 215 (2017).
</cite> [<a href="https://doi.org/10.3389/fcimb.2017.00215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5445135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28603700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lasica,%20A.%20M.,%20Ksiazek,%20M.,%20Madej,%20M.%20&amp;%20Potempa,%20J.%20The%20type%20IX%20secretion%20system%20(T9SS):%20highlights%20and%20recent%20insights%20into%20its%20structure%20and%20function.%20Front.%20Cell%20Infect.%20Microbiol.7,%20215%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Veith, P. D., Glew, M. D., Gorasia, D. G. &amp; Reynolds, E. C. Type IX secretion: the generation of bacterial cell surface coatings involved in virulence, gliding motility and the degradation of complex biopolymers. <em>Mol. Microbiol.</em><strong>106</strong>, 35–53 (2017).
</cite> [<a href="https://doi.org/10.1111/mmi.13752" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28714554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Veith,%20P.%20D.,%20Glew,%20M.%20D.,%20Gorasia,%20D.%20G.%20&amp;%20Reynolds,%20E.%20C.%20Type%20IX%20secretion:%20the%20generation%20of%20bacterial%20cell%20surface%20coatings%20involved%20in%20virulence,%20gliding%20motility%20and%20the%20degradation%20of%20complex%20biopolymers.%20Mol.%20Microbiol.106,%2035%E2%80%9353%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>McBride, M. J. &amp; Zhu, Y. Gliding motility and Por secretion system genes are widespread among members of the phylum Bacteroidetes. <em>J. Bacteriol.</em><strong>195</strong>, 270–278 (2013).
</cite> [<a href="https://doi.org/10.1128/JB.01962-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3553832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23123910/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McBride,%20M.%20J.%20&amp;%20Zhu,%20Y.%20Gliding%20motility%20and%20Por%20secretion%20system%20genes%20are%20widespread%20among%20members%20of%20the%20phylum%20Bacteroidetes.%20J.%20Bacteriol.195,%20270%E2%80%93278%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Shoji, M. et al. Por secretion system-dependent secretion and glycosylation of Porphyromonas gingivalis hemin-binding protein 35. <em>PLoS ONE</em><strong>6</strong>, e21372 (2011).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0021372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3120885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21731719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shoji,%20M.%20et%20al.%20Por%20secretion%20system-dependent%20secretion%20and%20glycosylation%20of%20Porphyromonas%20gingivalis%20hemin-binding%20protein%2035.%20PLoS%20ONE6,%20e21372%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Seers, C. A. et al. The RgpB C-terminal domain has a role in attachment of RgpB to the outer membrane and belongs to a novel C-terminal-domain family found in Porphyromonas gingivalis. <em>J. Bacteriol.</em><strong>188</strong>, 6376–6386 (2006).
</cite> [<a href="https://doi.org/10.1128/JB.00731-06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1595369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16923905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seers,%20C.%20A.%20et%20al.%20The%20RgpB%20C-terminal%20domain%20has%20a%20role%20in%20attachment%20of%20RgpB%20to%20the%20outer%20membrane%20and%20belongs%20to%20a%20novel%20C-terminal-domain%20family%20found%20in%20Porphyromonas%20gingivalis.%20J.%20Bacteriol.188,%206376%E2%80%936386%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Veith, P. D. et al. Protein substrates of a novel secretion system are numerous in the bacteroidetes phylum and have in common a cleavable C-terminal secretion signal, extensive post-translational modification, and cell-surface attachment. <em>J. Proteome Res.</em><strong>12</strong>, 4449–4461 (2013).
</cite> [<a href="https://doi.org/10.1021/pr400487b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24007199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Veith,%20P.%20D.%20et%20al.%20Protein%20substrates%20of%20a%20novel%20secretion%20system%20are%20numerous%20in%20the%20bacteroidetes%20phylum%20and%20have%20in%20common%20a%20cleavable%20C-terminal%20secretion%20signal,%20extensive%20post-translational%20modification,%20and%20cell-surface%20attachment.%20J.%20Proteome%20Res.12,%204449%E2%80%934461%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Gorasia, D. G. et al. <em>Porphyromonas gingivalis</em> type IX secretion substrates are cleaved and modified by a sortase-like mechanism. <em>PLoS Pathog.</em><strong>11</strong>, e1005152 (2015).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1005152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4560394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26340749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gorasia,%20D.%20G.%20et%20al.%20Porphyromonas%20gingivalis%20type%20IX%20secretion%20substrates%20are%20cleaved%20and%20modified%20by%20a%20sortase-like%20mechanism.%20PLoS%20Pathog.11,%20e1005152%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Veith, P. D. et al. Type IX secretion system cargo proteins are glycosylated at the C terminus with a novel linking sugar of the Wbp/Vim pathway. <em>mBio</em><strong>11</strong>, e01497–01420 (2020).
</cite> [<a href="https://doi.org/10.1128/mBio.01497-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7468200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32873758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Veith,%20P.%20D.%20et%20al.%20Type%20IX%20secretion%20system%20cargo%20proteins%20are%20glycosylated%20at%20the%20C%20terminus%20with%20a%20novel%20linking%20sugar%20of%20the%20Wbp/Vim%20pathway.%20mBio11,%20e01497%E2%80%9301420%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Gorasia, D. G. et al. Type B CTD proteins secreted by the type IX secretion system associate with PorP-like proteins for cell surface anchorage. <em>Int. J. Mol. Sci.</em><strong>23</strong>, 5681 (2022).
</cite> [<a href="https://doi.org/10.3390/ijms23105681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9143113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35628493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gorasia,%20D.%20G.%20et%20al.%20Type%20B%20CTD%20proteins%20secreted%20by%20the%20type%20IX%20secretion%20system%20associate%20with%20PorP-like%20proteins%20for%20cell%20surface%20anchorage.%20Int.%20J.%20Mol.%20Sci.23,%205681%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Naito, M., Tominaga, T., Shoji, M. &amp; Nakayama, K. PGN_0297 is an essential component of the type IX secretion system (T9SS) in Porphyromonas gingivalis: Tn-seq analysis for exhaustive identification of T9SS-related genes. <em>Microbiol. Immunol.</em><strong>63</strong>, 11–20 (2019).
</cite> [<a href="https://doi.org/10.1111/1348-0421.12665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6590471/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30599082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naito,%20M.,%20Tominaga,%20T.,%20Shoji,%20M.%20&amp;%20Nakayama,%20K.%20PGN_0297%20is%20an%20essential%20component%20of%20the%20type%20IX%20secretion%20system%20(T9SS)%20in%20Porphyromonas%20gingivalis:%20Tn-seq%20analysis%20for%20exhaustive%20identification%20of%20T9SS-related%20genes.%20Microbiol.%20Immunol.63,%2011%E2%80%9320%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Yukitake, H. et al. PorA, a conserved C-terminal domain-containing protein, impacts the PorXY-SigP signaling of the type IX secretion system. <em>Sci. Rep.</em><strong>10</strong>, 21109 (2020).
</cite> [<a href="https://doi.org/10.1038/s41598-020-77987-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7712824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33273542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yukitake,%20H.%20et%20al.%20PorA,%20a%20conserved%20C-terminal%20domain-containing%20protein,%20impacts%20the%20PorXY-SigP%20signaling%20of%20the%20type%20IX%20secretion%20system.%20Sci.%20Rep.10,%2021109%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Heath, J. E. et al. PG1058 is a novel multidomain protein component of the bacterial type IX secretion system. <em>PLoS ONE</em><strong>11</strong>, e0164313 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0164313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5053529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27711252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heath,%20J.%20E.%20et%20al.%20PG1058%20is%20a%20novel%20multidomain%20protein%20component%20of%20the%20bacterial%20type%20IX%20secretion%20system.%20PLoS%20ONE11,%20e0164313%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Vincent, M. S. et al. Characterization of the <em>Porphyromonas gingivalis</em> type IX secretion trans-envelope PorKLMNP core complex. <em>J. Biol. Chem.</em><strong>292</strong>, 3252–3261 (2017).
</cite> [<a href="https://doi.org/10.1074/jbc.M116.765081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5336160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28057754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vincent,%20M.%20S.%20et%20al.%20Characterization%20of%20the%20Porphyromonas%20gingivalis%20type%20IX%20secretion%20trans-envelope%20PorKLMNP%20core%20complex.%20J.%20Biol.%20Chem.292,%203252%E2%80%933261%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Gorasia, D. G. et al. Structural insights into the PorK and PorN components of the <em>Porphyromonas gingivalis</em> type IX secretion system. <em>PLoS Pathog.</em><strong>12</strong>, e1005820 (2016).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1005820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4980022/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27509186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gorasia,%20D.%20G.%20et%20al.%20Structural%20insights%20into%20the%20PorK%20and%20PorN%20components%20of%20the%20Porphyromonas%20gingivalis%20type%20IX%20secretion%20system.%20PLoS%20Pathog.12,%20e1005820%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Hennell James, R. et al. Structure and mechanism of the proton-driven motor that powers type 9 secretion and gliding motility. <em>Nat. Microbiol.</em><strong>6</strong>, 221–233 (2021).
</cite> [<a href="https://doi.org/10.1038/s41564-020-00823-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7116788/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33432152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hennell%20James,%20R.%20et%20al.%20Structure%20and%20mechanism%20of%20the%20proton-driven%20motor%20that%20powers%20type%209%20secretion%20and%20gliding%20motility.%20Nat.%20Microbiol.6,%20221%E2%80%93233%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Song, L. et al. A unique bacterial secretion machinery with multiple secretion centers. <em>Proc. Natl. Acad. Sci. USA</em><strong>119</strong>, e2119907119 (2022).
</cite> [<a href="https://doi.org/10.1073/pnas.2119907119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9170169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35471908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20L.%20et%20al.%20A%20unique%20bacterial%20secretion%20machinery%20with%20multiple%20secretion%20centers.%20Proc.%20Natl.%20Acad.%20Sci.%20USA119,%20e2119907119%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Lauber, F., Deme, J. C., Lea, S. M. &amp; Berks, B. C. Type 9 secretion system structures reveal a new protein transport mechanism. <em>Nature</em><strong>564</strong>, 77–82 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0693-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6927815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30405243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lauber,%20F.,%20Deme,%20J.%20C.,%20Lea,%20S.%20M.%20&amp;%20Berks,%20B.%20C.%20Type%209%20secretion%20system%20structures%20reveal%20a%20new%20protein%20transport%20mechanism.%20Nature564,%2077%E2%80%9382%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Gorasia, D. G. et al. Protein interactome analysis of the type IX secretion system identifies PorW as the missing link between the PorK/N ring complex and the Sov translocon. <em>Microbiol. Spectr.</em><strong>10</strong>, e0160221 (2022).
</cite> [<a href="https://doi.org/10.1128/spectrum.01602-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8754138/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35019767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gorasia,%20D.%20G.%20et%20al.%20Protein%20interactome%20analysis%20of%20the%20type%20IX%20secretion%20system%20identifies%20PorW%20as%20the%20missing%20link%20between%20the%20PorK/N%20ring%20complex%20and%20the%20Sov%20translocon.%20Microbiol.%20Spectr.10,%20e0160221%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Lauber, F. et al. Structural insights into the mechanism of protein transport by the Type 9 Secretion system translocon. <em>Nat. Microbiol.</em><strong>9</strong>, 1089–1102 (2024).
</cite> [<a href="https://doi.org/10.1038/s41564-024-01644-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10994853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38538833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lauber,%20F.%20et%20al.%20Structural%20insights%20into%20the%20mechanism%20of%20protein%20transport%20by%20the%20Type%209%20Secretion%20system%20translocon.%20Nat.%20Microbiol.9,%201089%E2%80%931102%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Glew, M. D. et al. PorV is an outer membrane shuttle protein for the type IX secretion system. <em>Sci. Rep.</em><strong>7</strong>, 8790 (2017).
</cite> [<a href="https://doi.org/10.1038/s41598-017-09412-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5562754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28821836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Glew,%20M.%20D.%20et%20al.%20PorV%20is%20an%20outer%20membrane%20shuttle%20protein%20for%20the%20type%20IX%20secretion%20system.%20Sci.%20Rep.7,%208790%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Madej, M. et al. PorZ, an essential component of the type IX secretion system of <em>Porphyromonas gingivalis</em>, delivers anionic lipopolysaccharide to the PorU sortase for transpeptidase processing of T9SS cargo proteins. <em>mBio</em><strong>12</strong>, e02262–20 (2021).
</cite> [<a href="https://doi.org/10.1128/mBio.02262-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8545088/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33622730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madej,%20M.%20et%20al.%20PorZ,%20an%20essential%20component%20of%20the%20type%20IX%20secretion%20system%20of%20Porphyromonas%20gingivalis,%20delivers%20anionic%20lipopolysaccharide%20to%20the%20PorU%20sortase%20for%20transpeptidase%20processing%20of%20T9SS%20cargo%20proteins.%20mBio12,%20e02262%E2%80%9320%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Abramson, J. et al. Accurate structure prediction of biomolecular interactions with AlphaFold 3. <em>Nature</em><strong>630</strong>, 493–500 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abramson,%20J.%20et%20al.%20Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20AlphaFold%203.%20Nature630,%20493%E2%80%93500%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Holder, P. G. et al. Reconstitution of formylglycine-generating enzyme with copper(II) for aldehyde tag conversion. <em>J. Biol. Chem.</em><strong>290</strong>, 15730–15745 (2015).
</cite> [<a href="https://doi.org/10.1074/jbc.M115.652669" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4505483/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25931126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Holder,%20P.%20G.%20et%20al.%20Reconstitution%20of%20formylglycine-generating%20enzyme%20with%20copper(II)%20for%20aldehyde%20tag%20conversion.%20J.%20Biol.%20Chem.290,%2015730%E2%80%9315745%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Veith, P. D., Shoji, M., Scott, N. E. &amp; Reynolds, E. C. Characterization of the O-glycoproteome of <em>Porphyromonas gingivalis</em>. <em>Microbiol. Spectr.</em><strong>10</strong>, e0150221 (2022).
</cite> [<a href="https://doi.org/10.1128/spectrum.01502-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8729774/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34985300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Veith,%20P.%20D.,%20Shoji,%20M.,%20Scott,%20N.%20E.%20&amp;%20Reynolds,%20E.%20C.%20Characterization%20of%20the%20O-glycoproteome%20of%20Porphyromonas%20gingivalis.%20Microbiol.%20Spectr.10,%20e0150221%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Glew, M. D. et al. Blue native-PAGE analysis of membrane protein complexes in <em>Porphyromonas gingivalis</em>. <em>J. Proteom.</em><strong>110</strong>, 72–92 (2014).</cite> [<a href="https://doi.org/10.1016/j.jprot.2014.07.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25111759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Glew,%20M.%20D.%20et%20al.%20Blue%20native-PAGE%20analysis%20of%20membrane%20protein%20complexes%20in%20Porphyromonas%20gingivalis.%20J.%20Proteom.110,%2072%E2%80%9392%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Veith, P. D., Gorasia, D. G. &amp; Reynolds, E. C. Towards defining the outer membrane proteome of <em>Porphyromonas gingivalis</em>. <em>Mol. Oral Microbiol.</em><strong>36</strong>, 25–36 (2021).
</cite> [<a href="https://doi.org/10.1111/omi.12320" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33124778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Veith,%20P.%20D.,%20Gorasia,%20D.%20G.%20&amp;%20Reynolds,%20E.%20C.%20Towards%20defining%20the%20outer%20membrane%20proteome%20of%20Porphyromonas%20gingivalis.%20Mol.%20Oral%20Microbiol.36,%2025%E2%80%9336%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Diederichs, K. A., Buchanan, S. K. &amp; Botos, I. Building better barrels-beta-barrel biogenesis and insertion in bacteria and mitochondria. <em>J. Mol. Biol.</em><strong>433</strong>, 166894 (2021).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2021.166894" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8292188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33639212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diederichs,%20K.%20A.,%20Buchanan,%20S.%20K.%20&amp;%20Botos,%20I.%20Building%20better%20barrels-beta-barrel%20biogenesis%20and%20insertion%20in%20bacteria%20and%20mitochondria.%20J.%20Mol.%20Biol.433,%20166894%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Gorasia, D. G., Glew, M. D., Veith, P. D. &amp; Reynolds, E. C. Quantitative proteomic analysis of the type IX secretion system mutants in <em>Porphyromonas gingivalis</em>. <em>Mol. Oral. Microbiol.</em><strong>35</strong>, 78–84 (2020).
</cite> [<a href="https://doi.org/10.1111/omi.12283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32040252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gorasia,%20D.%20G.,%20Glew,%20M.%20D.,%20Veith,%20P.%20D.%20&amp;%20Reynolds,%20E.%20C.%20Quantitative%20proteomic%20analysis%20of%20the%20type%20IX%20secretion%20system%20mutants%20in%20Porphyromonas%20gingivalis.%20Mol.%20Oral.%20Microbiol.35,%2078%E2%80%9384%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Knowles, T. J., Scott-Tucker, A., Overduin, M. &amp; Henderson, I. R. Membrane protein architects: the role of the BAM complex in outer membrane protein assembly. <em>Nat. Rev. Microbiol.</em><strong>7</strong>, 206–214 (2009).
</cite> [<a href="https://doi.org/10.1038/nrmicro2069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19182809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knowles,%20T.%20J.,%20Scott-Tucker,%20A.,%20Overduin,%20M.%20&amp;%20Henderson,%20I.%20R.%20Membrane%20protein%20architects:%20the%20role%20of%20the%20BAM%20complex%20in%20outer%20membrane%20protein%20assembly.%20Nat.%20Rev.%20Microbiol.7,%20206%E2%80%93214%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Gorasia, D. G., Veith, P. D. &amp; Reynolds, E. C. Protein interactome mapping of <em>Porphyromonas gingivalis</em> provides insights into the formation of the PorQ-Z complex of the type IX secretion system. <em>Mol. Oral Microbiol.</em>10.1111/omi.12383 (2022).</cite> [<a href="https://doi.org/10.1111/omi.12383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10947112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35862235/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Gorasia, D. G., Veith, P. D. &amp; Reynolds, E. C. The type IX secretion system: advances in structure, function and organisation. <em>Microorganisms</em><strong>8</strong>, 1173 (2020).
</cite> [<a href="https://doi.org/10.3390/microorganisms8081173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7463736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32752268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gorasia,%20D.%20G.,%20Veith,%20P.%20D.%20&amp;%20Reynolds,%20E.%20C.%20The%20type%20IX%20secretion%20system:%20advances%20in%20structure,%20function%20and%20organisation.%20Microorganisms8,%201173%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Minamino, T., Imada, K. &amp; Namba, K. Molecular motors of the bacterial flagella. <em>Curr. Opin. Struct. Biol.</em><strong>18</strong>, 693–701 (2008).
</cite> [<a href="https://doi.org/10.1016/j.sbi.2008.09.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18848888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Minamino,%20T.,%20Imada,%20K.%20&amp;%20Namba,%20K.%20Molecular%20motors%20of%20the%20bacterial%20flagella.%20Curr.%20Opin.%20Struct.%20Biol.18,%20693%E2%80%93701%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Mariano, G. et al. Oligomerization of the FliF domains suggests a coordinated assembly of the bacterial flagellum MS ring. <em>Front. Microbiol.</em><strong>12</strong>, 781960 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.781960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8786727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35087486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mariano,%20G.%20et%20al.%20Oligomerization%20of%20the%20FliF%20domains%20suggests%20a%20coordinated%20assembly%20of%20the%20bacterial%20flagellum%20MS%20ring.%20Front.%20Microbiol.12,%20781960%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Fuchsbauer, O., Lunar Silva, I., Cascales, E., Roussel, A. &amp; Leone, P. Structural and functional analyses of the <em>Porphyromonas gingivalis</em> type IX secretion system PorN protein. <em>J. Biol. Chem.</em><strong>298</strong>, 101618 (2022).
</cite> [<a href="https://doi.org/10.1016/j.jbc.2022.101618" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8861641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35065963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fuchsbauer,%20O.,%20Lunar%20Silva,%20I.,%20Cascales,%20E.,%20Roussel,%20A.%20&amp;%20Leone,%20P.%20Structural%20and%20functional%20analyses%20of%20the%20Porphyromonas%20gingivalis%20type%20IX%20secretion%20system%20PorN%20protein.%20J.%20Biol.%20Chem.298,%20101618%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Glew, M. D. et al. PG0026 is the C-terminal signal peptidase of a novel secretion system of <em>Porphyromonas gingivalis</em>. <em>J. Biol. Chem.</em><strong>287</strong>, 24605–24617 (2012).
</cite> [<a href="https://doi.org/10.1074/jbc.M112.369223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3397888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22593568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Glew,%20M.%20D.%20et%20al.%20PG0026%20is%20the%20C-terminal%20signal%20peptidase%20of%20a%20novel%20secretion%20system%20of%20Porphyromonas%20gingivalis.%20J.%20Biol.%20Chem.287,%2024605%E2%80%9324617%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Chen, Y. Y. et al. The outer membrane protein LptO is essential for the O-deacylation of LPS and the co-ordinated secretion and attachment of A-LPS and CTD proteins in <em>Porphyromonas gingivalis</em>. <em>Mol. Microbiol.</em><strong>79</strong>, 1380–1401 (2011).
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2010.07530.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21244528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Y.%20Y.%20et%20al.%20The%20outer%20membrane%20protein%20LptO%20is%20essential%20for%20the%20O-deacylation%20of%20LPS%20and%20the%20co-ordinated%20secretion%20and%20attachment%20of%20A-LPS%20and%20CTD%20proteins%20in%20Porphyromonas%20gingivalis.%20Mol.%20Microbiol.79,%201380%E2%80%931401%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Dashper, S. G. et al. Response of Porphyromonas gingivalis to heme limitation in continuous culture. <em>J. Bacteriol.</em><strong>191</strong>, 1044–1055 (2009).
</cite> [<a href="https://doi.org/10.1128/JB.01270-08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2632107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19028886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dashper,%20S.%20G.%20et%20al.%20Response%20of%20Porphyromonas%20gingivalis%20to%20heme%20limitation%20in%20continuous%20culture.%20J.%20Bacteriol.191,%201044%E2%80%931055%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Punjani, A., Rubinstein, J. L., Fleet, D. J. &amp; Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. <em>Nat. Methods</em><strong>14</strong>, 290–296 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28165473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Punjani,%20A.,%20Rubinstein,%20J.%20L.,%20Fleet,%20D.%20J.%20&amp;%20Brubaker,%20M.%20A.%20cryoSPARC:%20algorithms%20for%20rapid%20unsupervised%20cryo-EM%20structure%20determination.%20Nat.%20Methods14,%20290%E2%80%93296%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Emsley, P., Lohkamp, B., Scott, W. G. &amp; Cowtan, K. Features and development of Coot. <em>Acta Crystallogr. D. Biol. Crystallogr.</em><strong>66</strong>, 486–501 (2010).
</cite> [<a href="https://doi.org/10.1107/S0907444910007493" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2852313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Emsley,%20P.,%20Lohkamp,%20B.,%20Scott,%20W.%20G.%20&amp;%20Cowtan,%20K.%20Features%20and%20development%20of%20Coot.%20Acta%20Crystallogr.%20D.%20Biol.%20Crystallogr.66,%20486%E2%80%93501%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Liebschner, D. et al. Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix. <em>Acta Crystallogr. D Struct. Biol.</em><strong>75</strong>, 861–877 (2019).
</cite> [<a href="https://doi.org/10.1107/S2059798319011471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6778852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31588918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liebschner,%20D.%20et%20al.%20Macromolecular%20structure%20determination%20using%20X-rays,%20neutrons%20and%20electrons:%20recent%20developments%20in%20Phenix.%20Acta%20Crystallogr.%20D%20Struct.%20Biol.75,%20861%E2%80%93877%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Mastronarde, D. N. &amp; Held, S. R. Automated tilt series alignment and tomographic reconstruction in IMOD. <em>J. Struct. Biol.</em><strong>197</strong>, 102–113 (2017).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2016.07.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5247408/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27444392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mastronarde,%20D.%20N.%20&amp;%20Held,%20S.%20R.%20Automated%20tilt%20series%20alignment%20and%20tomographic%20reconstruction%20in%20IMOD.%20J.%20Struct.%20Biol.197,%20102%E2%80%93113%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Agulleiro, J. I. &amp; Fernandez, J. J. Tomo3D 2.0-exploitation of advanced vector extensions (AVX) for 3D reconstruction. <em>J. Struct. Biol.</em><strong>189</strong>, 147–152 (2015).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2014.11.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25528570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Agulleiro,%20J.%20I.%20&amp;%20Fernandez,%20J.%20J.%20Tomo3D%202.0-exploitation%20of%20advanced%20vector%20extensions%20(AVX)%20for%203D%20reconstruction.%20J.%20Struct.%20Biol.189,%20147%E2%80%93152%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Cox, J. et al. Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ. <em>Mol. Cell Proteom.</em><strong>13</strong>, 2513–2526 (2014).</cite> [<a href="https://doi.org/10.1074/mcp.M113.031591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4159666/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24942700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cox,%20J.%20et%20al.%20Accurate%20proteome-wide%20label-free%20quantification%20by%20delayed%20normalization%20and%20maximal%20peptide%20ratio%20extraction,%20termed%20MaxLFQ.%20Mol.%20Cell%20Proteom.13,%202513%E2%80%932526%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Lomize, M. A., Pogozheva, I. D., Joo, H., Mosberg, H. I. &amp; Lomize, A. L. OPM database and PPM web server: resources for positioning of proteins in membranes. <em>Nucleic Acids Res.</em><strong>40</strong>, D370–D376 (2012).
</cite> [<a href="https://doi.org/10.1093/nar/gkr703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3245162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21890895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lomize,%20M.%20A.,%20Pogozheva,%20I.%20D.,%20Joo,%20H.,%20Mosberg,%20H.%20I.%20&amp;%20Lomize,%20A.%20L.%20OPM%20database%20and%20PPM%20web%20server:%20resources%20for%20positioning%20of%20proteins%20in%20membranes.%20Nucleic%20Acids%20Res.40,%20D370%E2%80%93D376%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Jo, S., Kim, T., Iyer, V. G. &amp; Im, W. CHARMM-GUI: a web-based graphical user interface for CHARMM. <em>J. Comput. Chem.</em><strong>29</strong>, 1859–1865 (2008).
</cite> [<a href="https://doi.org/10.1002/jcc.20945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18351591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jo,%20S.,%20Kim,%20T.,%20Iyer,%20V.%20G.%20&amp;%20Im,%20W.%20CHARMM-GUI:%20a%20web-based%20graphical%20user%20interface%20for%20CHARMM.%20J.%20Comput.%20Chem.29,%201859%E2%80%931865%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Wu, E. L. et al. CHARMM-GUI membrane builder toward realistic biological membrane simulations. <em>J. Comput. Chem.</em><strong>35</strong>, 1997–2004 (2014).
</cite> [<a href="https://doi.org/10.1002/jcc.23702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4165794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25130509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20E.%20L.%20et%20al.%20CHARMM-GUI%20membrane%20builder%20toward%20realistic%20biological%20membrane%20simulations.%20J.%20Comput.%20Chem.35,%201997%E2%80%932004%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Vangone, A. &amp; Bonvin, A. M. Contacts-based prediction of binding affinity in protein-protein complexes. <em>Elife</em><strong>4</strong>, e07454 (2015).
</cite> [<a href="https://doi.org/10.7554/eLife.07454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4523921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26193119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vangone,%20A.%20&amp;%20Bonvin,%20A.%20M.%20Contacts-based%20prediction%20of%20binding%20affinity%20in%20protein-protein%20complexes.%20Elife4,%20e07454%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Xue, L. C., Rodrigues, J. P., Kastritis, P. L., Bonvin, A. M. &amp; Vangone, A. PRODIGY: a web server for predicting the binding affinity of protein-protein complexes. <em>Bioinformatics</em><strong>32</strong>, 3676–3678 (2016).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27503228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xue,%20L.%20C.,%20Rodrigues,%20J.%20P.,%20Kastritis,%20P.%20L.,%20Bonvin,%20A.%20M.%20&amp;%20Vangone,%20A.%20PRODIGY:%20a%20web%20server%20for%20predicting%20the%20binding%20affinity%20of%20protein-protein%20complexes.%20Bioinformatics32,%203676%E2%80%933678%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (2.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63163_MOESM2_ESM.pdf</a><sup> (32.8KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM3_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (27.4KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (140.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM5_ESM.mov" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Movie 1</a><sup> (11.4MB, mov) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM6_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer review file</a><sup> (472.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365241/bin/41467_2025_63163_MOESM8_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (105.5KB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Cryo-EM density maps and atomic coordinates are deposited in the Electron Microscopy Databank (EMDB) with the following accession number: <a href="https://www.ebi.ac.uk/emdb/EMD-48741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-48741</a>, <a href="https://www.ebi.ac.uk/emdb/EMD-48722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-48722</a>. Atomic coordinates are deposited in the Protein Data Bank (PDB) with the following accession number PDB:9MYJ [10.2210/pdb9MYJ/pdb]. The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD062825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD062825</a> and <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD062800" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD062800</a>. <a href="#Sec31" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63163-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63163.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365241/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365241/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365241%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365241/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365241/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365241/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830366/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365241/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830366/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365241/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365241/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="pyys3nrXo8QODUtg3GmCrG2wxCZV4N8MItV4WK0YJGFtJy5MHZtNRfZrHJ7w6vci">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
