
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Phylogenetic diversity and molecular evolution of Hantaan virus harbored by Apodemus chejuensis on Jeju Island, Republic of Korea, 2022–2023 - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5343268AF3FB43054326002F0432FC.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosntd">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373272/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS Neglected Tropical Diseases">
<meta name="citation_title" content="Phylogenetic diversity and molecular evolution of Hantaan virus harbored by Apodemus chejuensis on Jeju Island, Republic of Korea, 2022–2023">
<meta name="citation_author" content="Juyoung Noh">
<meta name="citation_author_institution" content="Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Kyungmin Park">
<meta name="citation_author_institution" content="Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author_institution" content="Institute for Viral Diseases, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Seong-Gyu Kim">
<meta name="citation_author_institution" content="Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author_institution" content="BK21 Graduate Program, Department of Biomedical Sciences, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Ye-rin Seo">
<meta name="citation_author_institution" content="Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Jongwoo Kim">
<meta name="citation_author_institution" content="Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author_institution" content="BK21 Graduate Program, Department of Biomedical Sciences, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Hee-Kyung Cho">
<meta name="citation_author_institution" content="Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author_institution" content="BK21 Graduate Program, Department of Biomedical Sciences, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Won-Keun Kim">
<meta name="citation_author_institution" content="Department of Microbiology, College of Medicine, Hallym University, Chuncheon, Republic of Korea">
<meta name="citation_author_institution" content="Institute of Medical Research, College of Medicine, Hallym University, Chuncheon, Republic of Korea">
<meta name="citation_author" content="Jin-Won Song">
<meta name="citation_author_institution" content="Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author_institution" content="BK21 Graduate Program, Department of Biomedical Sciences, Korea University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="19">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0013459">
<meta name="citation_doi" content="10.1371/journal.pntd.0013459">
<meta name="citation_pmid" content="40828876">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373272/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373272/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373272/pdf/pntd.0013459.pdf">
<meta name="description" content="Hantaan virus (HTNV), hosted by Apodemus spp., is a well-recognized causative agent of hemorrhagic fever with renal syndrome (HFRS) and poses a crucial global public health concern. Based on the current evidence, HTNV carried by A. chejuensis is ...">
<meta name="og:title" content="Phylogenetic diversity and molecular evolution of Hantaan virus harbored by Apodemus chejuensis on Jeju Island, Republic of Korea, 2022–2023">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Hantaan virus (HTNV), hosted by Apodemus spp., is a well-recognized causative agent of hemorrhagic fever with renal syndrome (HFRS) and poses a crucial global public health concern. Based on the current evidence, HTNV carried by A. chejuensis is ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373272/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373272">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pntd.0013459"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pntd.0013459.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373272%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373272/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373272/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373272/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosntd.png" alt="PLOS Neglected Tropical Diseases logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS Neglected Tropical Diseases" title="Link to PLOS Neglected Tropical Diseases" shape="default" href="https://doi.org/10.1371/journal.pntd.0013459" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS Negl Trop Dis</button></div>. 2025 Aug 19;19(8):e0013459. doi: <a href="https://doi.org/10.1371/journal.pntd.0013459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pntd.0013459</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20Negl%20Trop%20Dis%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20Negl%20Trop%20Dis%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20Negl%20Trop%20Dis%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20Negl%20Trop%20Dis%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Phylogenetic diversity and molecular evolution of Hantaan virus harbored by <em>Apodemus chejuensis</em> on Jeju Island, Republic of Korea, 2022–2023</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Noh%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Juyoung Noh</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Juyoung Noh</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Resources, Software, Visualization, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Noh%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Juyoung Noh</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Kyungmin Park</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Kyungmin Park</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div class="p">
<sup>2</sup>Institute for Viral Diseases, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div>Conceptualization, Data curation, Formal analysis, Resources, Validation, Visualization, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kyungmin Park</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SG%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Seong-Gyu Kim</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Seong-Gyu Kim</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div class="p">
<sup>3</sup>BK21 Graduate Program, Department of Biomedical Sciences, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div>Formal analysis, Methodology, Resources, Software, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SG%22%5BAuthor%5D" class="usa-link"><span class="name western">Seong-Gyu Kim</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seo%20YR%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ye-rin Seo</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ye-rin Seo</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div>Data curation, Methodology, Software, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seo%20YR%22%5BAuthor%5D" class="usa-link"><span class="name western">Ye-rin Seo</span></a>
</div>
</div>
<sup>1,</sup><sup>¤a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jongwoo Kim</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jongwoo Kim</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div class="p">
<sup>3</sup>BK21 Graduate Program, Department of Biomedical Sciences, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div>Methodology, Resources, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jongwoo Kim</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cho%20HK%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Hee-Kyung Cho</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Hee-Kyung Cho</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div class="p">
<sup>3</sup>BK21 Graduate Program, Department of Biomedical Sciences, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div>Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cho%20HK%22%5BAuthor%5D" class="usa-link"><span class="name western">Hee-Kyung Cho</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20WK%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Won-Keun Kim</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Won-Keun Kim</span></h3>
<div class="p">
<sup>4</sup>Department of Microbiology, College of Medicine, Hallym University, Chuncheon, Republic of Korea</div>
<div class="p">
<sup>5</sup>Institute of Medical Research, College of Medicine, Hallym University, Chuncheon, Republic of Korea</div>
<div>Funding acquisition, Validation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20WK%22%5BAuthor%5D" class="usa-link"><span class="name western">Won-Keun Kim</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20JW%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jin-Won Song</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jin-Won Song</span></h3>
<div class="p">
<sup>1</sup>Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div class="p">
<sup>3</sup>BK21 Graduate Program, Department of Biomedical Sciences, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div>Funding acquisition, Project administration, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20JW%22%5BAuthor%5D" class="usa-link"><span class="name western">Jin-Won Song</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Stephanie N Seifert</span><sup>6</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Department of Microbiology, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div id="aff002">
<sup>2</sup>Institute for Viral Diseases, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div id="aff003">
<sup>3</sup>BK21 Graduate Program, Department of Biomedical Sciences, Korea University College of Medicine, Seoul, Republic of Korea</div>
<div id="aff004">
<sup>4</sup>Department of Microbiology, College of Medicine, Hallym University, Chuncheon, Republic of Korea</div>
<div id="aff005">
<sup>5</sup>Institute of Medical Research, College of Medicine, Hallym University, Chuncheon, Republic of Korea</div>
<div id="edit1">
<sup>6</sup>Washington State University, UNITED STATES OF AMERICA</div>
<div class="author-notes p">
<div class="fn" id="coi001"><p>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="currentaff001"><p>¤a Present address: Department of Preventive Medicine, Korea University College of Medicine, Seoul, Republic of Korea</p></div>
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>jwsong@korea.ac.kr</span></p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Juyoung Noh</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Resources, Software, Visualization, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Kyungmin Park</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Resources, Validation, Visualization, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Seong-Gyu Kim</span></strong>: <span class="role">Formal analysis, Methodology, Resources, Software, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ye-rin Seo</span></strong>: <span class="role">Data curation, Methodology, Software, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jongwoo Kim</span></strong>: <span class="role">Methodology, Resources, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Hee-Kyung Cho</span></strong>: <span class="role">Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Won-Keun Kim</span></strong>: <span class="role">Funding acquisition, Validation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jin-Won Song</span></strong>: <span class="role">Funding acquisition, Project administration, Supervision, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Stephanie N Seifert</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 28; Accepted 2025 Aug 9; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Noh et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373272  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40828876/" class="usa-link">40828876</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<section id="sec001"><h3 class="pmc_sec_title">Background</h3>
<p>Hantaan virus (HTNV), hosted by <em>Apodemus</em> spp., is a well-recognized causative agent of hemorrhagic fever with renal syndrome (HFRS) and poses a crucial global public health concern. Based on the current evidence, HTNV carried by <em>A. chejuensis</em> is proposed as the likely etiological agent of HFRS on Jeju Island, Republic of Korea (ROK).</p></section><section id="sec002"><h3 class="pmc_sec_title">Methodology/Principal findings</h3>
<p>In this study, 50 small mammals were collected from five locations in Seogwipo-si and Jeju-si on Jeju Island, ROK, during 2022–2023. Serological and molecular analyses revealed HTNV prevalence rates of 34% (16/47) and 27.7% (13/47), respectively. Using a multiplex polymerase chain reaction-based nanopore sequencing approach, nine complete HTNV genomes were sequenced from the lung tissues of <em>A. chejuensis</em>, representing the first comprehensive genomic characterization of HTNV from Seogwipo-si (Hogeun-dong) and Jeju-si (Sangdae-ri). Phylodynamic analyses suggest evolutionary divergence and phylogeographic diversity, with four unique amino acid substitutions identified in HTNV genomes from Seogwipo-si.</p></section><section id="sec003"><h3 class="pmc_sec_title">Conclusion/Significance</h3>
<p>This study provides important insights into the genomic surveillance, genetic diversity, and evolutionary dynamics of orthohantaviruses, which are essential for guiding effective public health strategies to control and prevent future HFRS outbreaks in the ROK.</p></section></section><section class="abstract" id="abstract2"><h2>Author summary</h2>
<p>The integration of genomic surveillance with spatial and temporal data offers a powerful approach to reconstruct viral transmission patterns across both geographic and temporal dimensions. Whole-genome sequencing is essential for elucidating the complex dynamics of viral evolution, genomic epidemiology, and the transmission pathways of emerging pathogens. Hantaan virus (HTNV) and the related genotypes classified under the species <em>Orthohantavirus hantaense,</em> primarily hosted by <em>Apodemus</em> species, are well-documented causative agents of hemorrhagic fever with renal syndrome (HFRS), posing a substantial global public health concern. Current evidence suggests that HTNV associated with <em>A. chejuensis</em> is the likely etiological agent responsible for HFRS cases on Jeju Island, Republic of Korea (ROK). This study conducted extensive HTNV surveillance, sampling 50 small mammals from five locations in Seogwipo-si and Jeju-si on Jeju Island between 2022 and 2023. Nine complete HTNV genomes were sequenced from lung tissues of <em>A. chejuensis</em>, constituting the first comprehensive genomic characterization of HTNV from Seogwipo-si (Hogeun-dong) and Jeju-si (Sangdae-ri). Phylodynamic analyses suggest evolutionary divergence and phylogeographic diversity, with four distinctive amino acid substitutions identified in HTNV genomes from Seogwipo-si. These findings contribute to our understanding of orthohantavirus genomic diversity and evolutionary dynamics, providing useful insights to inform public health strategies aimed at controlling HFRS outbreaks in the ROK.</p></section><section id="sec004"><h2 class="pmc_sec_title">Introduction</h2>
<p>Orthohantaviruses, members of the family Hantaviridae within the order Elliovirales, are enveloped RNA viruses with a negative-sense, single-stranded genome [<a href="#pntd.0013459.ref001" class="usa-link" aria-describedby="pntd.0013459.ref001">1</a>]. The genome is segmented into three parts: small (S), medium (M), and large (L), which encode a nucleocapsid protein (N), two membrane glycoproteins (Gn and Gc), and RNA-dependent RNA polymerase (RdRp), respectively. In Eurasia, orthohantaviruses are the primary cause of hemorrhagic fever with renal syndrome (HFRS), a disease with a fatality rate of 1–15% [<a href="#pntd.0013459.ref002" class="usa-link" aria-describedby="pntd.0013459.ref002">2</a>]. The viruses most commonly associated with HFRS include Hantaan virus (HTNV), Seoul virus (SEOV), Dobrava virus (DOBV), and Puumala virus, all of which belong to different <em>Orthohantavirus</em> species [<a href="#pntd.0013459.ref003" class="usa-link" aria-describedby="pntd.0013459.ref003">3</a>]. Humans are typically infected by inhalation of aerosolized viral particles from the saliva, urine, or feces of infected rodents [<a href="#pntd.0013459.ref004" class="usa-link" aria-describedby="pntd.0013459.ref004">4</a>].</p>
<p>Approximately 400 HFRS cases are reported annually in the Republic of Korea (ROK), with the case fatality rate averaging between 1% and 4% (accessible online at <a href="https://dportal.kdca.go.kr/pot/is/rginEDW.do" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dportal.kdca.go.kr/pot/is/rginEDW.do</a>). HTNV is the primary pathogen, accounting for 70% of these infections, followed by SEOV, which is responsible for 20%, with the remaining 10% attributed to unidentified viral agents [<a href="#pntd.0013459.ref005" class="usa-link" aria-describedby="pntd.0013459.ref005">5</a>]. HTNV was first isolated in 1978 from a striped field mouse (<em>Apodemus agrarius</em>) in the ROK, confirming its role as a causative agent of HFRS [<a href="#pntd.0013459.ref006" class="usa-link" aria-describedby="pntd.0013459.ref006">6</a>]. Currently, HTNV is classified into four distinct genotypes: two variants of the Hantaan virus associated with <em>A. agrarius</em> and <em>A. chejuensis</em> and the Soochong and Amur viruses, which are carried by <em>A. peninsulae</em> [<a href="#pntd.0013459.ref006" class="usa-link" aria-describedby="pntd.0013459.ref006">6</a>–<a href="#pntd.0013459.ref010" class="usa-link" aria-describedby="pntd.0013459.ref010">10</a>]. <em>A. agrarius</em>, the most prevalent rodent species, is widely distributed across the Korean Peninsula, whereas <em>A. chejuensis</em> is restricted to Jeju Island, the southernmost region of the ROK [<a href="#pntd.0013459.ref011" class="usa-link" aria-describedby="pntd.0013459.ref011">11</a>]. In 2012, Jeju virus was discovered and characterized from a shrew, <em>Crocidura shantungensis,</em> on Jeju Island [<a href="#pntd.0013459.ref012" class="usa-link" aria-describedby="pntd.0013459.ref012">12</a>,<a href="#pntd.0013459.ref013" class="usa-link" aria-describedby="pntd.0013459.ref013">13</a>]. However, its pathogenic potential remains uncertain. Based on current evidence, HTNV harbored by <em>A. chejuensis</em> is tentatively proposed as the etiological agent of HFRS on Jeju Island, ROK.</p>
<p>Next-generation sequencing (NGS) technologies have significantly advanced virology, enabling whole-genome sequencing, metagenomics, transmission tracking, genomic epidemiological studies, and vaccine development [<a href="#pntd.0013459.ref014" class="usa-link" aria-describedby="pntd.0013459.ref014">14</a>–<a href="#pntd.0013459.ref018" class="usa-link" aria-describedby="pntd.0013459.ref018">18</a>]. Sequencing the complete genomes of emerging viruses is crucial for clinical diagnostics, virulence assessment, and sources during outbreaks [<a href="#pntd.0013459.ref019" class="usa-link" aria-describedby="pntd.0013459.ref019">19</a>–<a href="#pntd.0013459.ref021" class="usa-link" aria-describedby="pntd.0013459.ref021">21</a>]. The MinION sequencer (Oxford Nanopore Technologies [ONT], London, UK) is a portable, long-read sequencing device that provides a compact, cost-effective alternative to conventional NGS platforms [<a href="#pntd.0013459.ref022" class="usa-link" aria-describedby="pntd.0013459.ref022">22</a>,<a href="#pntd.0013459.ref023" class="usa-link" aria-describedby="pntd.0013459.ref023">23</a>]. Nanopore sequencing has been successfully used to obtain full-length genomic sequences of two New World hantaviruses, Sin Nombre virus and Prospect Hill virus [<a href="#pntd.0013459.ref024" class="usa-link" aria-describedby="pntd.0013459.ref024">24</a>]. In addition, amplicon-based nanopore sequencing has been explored as a diagnostic method for rapidly generating near-complete genomic sequences of HTNV and SEOV from patients with HFRS and their rodent reservoirs [<a href="#pntd.0013459.ref025" class="usa-link" aria-describedby="pntd.0013459.ref025">25</a>–<a href="#pntd.0013459.ref029" class="usa-link" aria-describedby="pntd.0013459.ref029">29</a>].</p>
<p>Genomic surveillance, when combined with data on geographic location and sampling time, allows for the reconstruction of viral transmission patterns over space and time [<a href="#pntd.0013459.ref030" class="usa-link" aria-describedby="pntd.0013459.ref030">30</a>,<a href="#pntd.0013459.ref031" class="usa-link" aria-describedby="pntd.0013459.ref031">31</a>]. To achieve this, both heuristic and model-driven phylogeographic approaches have been developed, which offer a range of models to address key issues related to the spread and evolution of infectious diseases [<a href="#pntd.0013459.ref032" class="usa-link" aria-describedby="pntd.0013459.ref032">32</a>]. The coronavirus disease (COVID-19) pandemic has underscored the urgent need for rapid pathogen detection and monitoring through genomic surveillance integrated with epidemiological analysis [<a href="#pntd.0013459.ref033" class="usa-link" aria-describedby="pntd.0013459.ref033">33</a>]. During the 2014–2015 Ebola virus outbreak in West Africa, phylogenetic studies and evolutionary analyses were used in mapping viral transmission routes by integrating genomic data with spatial and temporal epidemiological information [<a href="#pntd.0013459.ref034" class="usa-link" aria-describedby="pntd.0013459.ref034">34</a>]. Similarly, the combination of clinical sequencing and epidemiology during the 2013–2015 Lassa virus outbreak provided real-time insights into transmission patterns, particularly rodent-to-human spread [<a href="#pntd.0013459.ref035" class="usa-link" aria-describedby="pntd.0013459.ref035">35</a>]. These approaches, which utilize both discrete and continuous geographic data, are becoming increasingly important in viral phylodynamics, offering a deeper understanding of the evolution and spread [<a href="#pntd.0013459.ref036" class="usa-link" aria-describedby="pntd.0013459.ref036">36</a>,<a href="#pntd.0013459.ref037" class="usa-link" aria-describedby="pntd.0013459.ref037">37</a>].</p>
<p>In this study, an epidemiological survey was conducted to assess the serological and viral prevalence of hantaviruses in rodents and shrews captured on Jeju Island, ROK, between 2022 and 2023. Amplicon-based NGS was employed to sequence the complete genome of HTNV to better understand its evolutionary diversification in Seogwipo-si and Jeju-si, Jeju Island. Phylodynamic analyses were performed to examine the evolutionary divergence and geographic variation of HTNV across Jeju Island, with an emphasis on identifying the key mutation dynamics. This report provides important insights into the genomic surveillance, phylogenetic diversity, and evolutionary dynamics of orthohantaviruses that are essential for developing effective strategies to mitigate HFRS outbreaks in the ROK.</p></section><section id="sec005"><h2 class="pmc_sec_title">Methods</h2>
<section id="sec006"><h3 class="pmc_sec_title">Ethics statement</h3>
<p>All small mammals were handled in accordance with the ethical guidelines approved by the Korea University Institutional Animal Care and Use Committee (Approval number: KUIACUC #2022–0034). All necropsies and laboratory experiments were performed in a biosafety level 3 facility at the Korea University College of Medicine, Seoul, ROK.</p></section><section id="sec007"><h3 class="pmc_sec_title">Sample collection</h3>
<p>Between 2022 and 2023, small mammals—including rodents and shrews—were captured at five locations on Jeju Island, ROK: Hogeun-dong and Seohong-dong in Seogwipo-si, and Bongseong-ri, Sangdae-ri, and Ora-dong in Jeju-si. The procedures for trapping and transportation have been described previously [<a href="#pntd.0013459.ref038" class="usa-link" aria-describedby="pntd.0013459.ref038">38</a>]. Briefly, field sampling was conducted once annually at each site over the study period, with each trapping session lasting exactly two consecutive nights. At each location, 20–25 collapsible, live-capture Sherman traps (7.7 × 9 × 23 cm; H.B. Sherman, Tallahassee, FL, USA) were deployed each afternoon along transects spaced at 4–5 meter intervals and retrieved the following morning. Traps were placed in ecotones and transitional zones across a variety of habitats, including unmanaged grasslands, herbaceous vegetation, farmlands, and forest edges. Given the five sites sampled over two years, with two-night trapping sessions at each visit, the total trapping effort was estimated at approximately 400–500 trap-nights, depending on the number of traps deployed per site. Captured animals were individually labeled, placed in secure transport containers, and transferred to Korea University in Seoul. Upon arrival, individuals were humanely euthanized and identified to species based on external morphological characteristics; sex and body weight were also recorded. Whole blood and internal organs (lung, liver, spleen, and kidney) were aseptically collected in accordance with protocols approved by the KUIACUC and stored at −80°C until further analysis.</p></section><section id="sec008"><h3 class="pmc_sec_title">Mitochondrial DNA (mtDNA) analysis</h3>
<p>Total DNA was extracted from the homogenized liver tissues using either the High Pure PCR Template Preparation Kit (Roche, Basel, Switzerland) or TRI Reagent Solution (Ambion, Austin, Texas, USA) according to the manufacturer’s instructions. To verify the taxonomic classification of small mammals, PCR amplification was performed targeting the mtDNA cytochrome <em>b</em> (CYTB) gene. The PCR employed universal primers for mammals and followed the cycling conditions outlined previously [<a href="#pntd.0013459.ref039" class="usa-link" aria-describedby="pntd.0013459.ref039">39</a>].</p></section><section id="sec009"><h3 class="pmc_sec_title">Indirect immunofluorescence antibody test</h3>
<p>Serum and heart fluids were diluted with phosphate buffered saline (PBS) to final dilution factors of 1:32 and 1:2, respectively. The diluted samples were then applied to ten-well slides (PTFE Printed Slide 10 well; Electron Microscopy Science, USA) containing acetone-fixed HTNV-infected Vero E6 cells and incubated at 37°C for 30 min. Following incubation, the plates were washed sequentially with PBS and distilled water (D.W.). The cells were then treated with fluorescein isothiocyanate (FITC)-conjugated goat antibodies specific to mouse immunoglobulin G (IgG) for rodents or mouse or rat IgG for shrews (MP Bio, CA, USA). The slides were then incubated at 37°C for 30 min and washed. Hantavirus-specific fluorescence was identified using a fluorescence microscope (Axio Scope; Zeiss, Berlin, Germany). Antibody titers were quantified as the highest dilution of the sample that exhibited weak yet specific fluorescence, as determined through a two-fold serial dilution.</p></section><section id="sec010"><h3 class="pmc_sec_title">Reverse transcription-polymerase chain reaction (RT-PCR)</h3>
<p>Total RNA was isolated from 50–100 mg of homogenized rodent lung tissue using TRI Reagent Solution (Ambion), in accordance with the manufacturer’s protocol. cDNA was synthesized from 1 µg of total RNA employing the High-Capacity RNA-to-cDNA Kit (Applied Biosystems, Foster City, CA, USA), using a random hexamer primer in conjunction with OSM55 (5′–TAG TAG TAG ACT CC–3′). Subsequent amplification of the target regions was conducted through first-round and nested PCR in a total volume of 25 µL. Each reaction mixture comprised 0.625 U of Ex Taq DNA polymerase (TaKaRa Bio, Japan), 2.5 µL of 10 × reaction buffer, 2 µL of a 2 mM dNTP mix, 10 pmol of each primer (final concentration: 0.4 µM), and 1.5 µL of the cDNA template. Thermal cycling conditions included an initial denaturation at 94°C for 5 min, followed by six cycles of denaturation at 94°C for 30 s, annealing at 37°C for 40 s, and extension at 72°C for 1 min. This was succeeded by 32 cycles of denaturation at 94°C for 30 s, annealing at 42°C for 40 s, and extension at 72°C for 1 min, concluding with a final extension step at 72°C for 5 min. PCR amplifications were carried out using the ProFlex PCR System (Life Technologies, CA, USA). Amplified products were purified with the MinElute PCR Purification Kit (Qiagen, Hilden, Germany). Bidirectional Sanger sequencing was performed using the BigDye Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems) on an ABI 3730XL DNA Analyzer (Applied Biosystems). The primers used for amplification have been previously described [<a href="#pntd.0013459.ref009" class="usa-link" aria-describedby="pntd.0013459.ref009">9</a>].</p></section><section id="sec011"><h3 class="pmc_sec_title">Quantitative PCR (qPCR)</h3>
<p>qPCR was conducted in a 10 µL reaction mixture consisting of 5 µL of SYBR Green PCR Master Mix (Applied Biosystems), 3 µL of D.W., 1 µL of cDNA, and 5 pmol of each primer (final concentration: 0.4 µM), using the QuantStudio 5 Flex Real-Time PCR System (Applied Biosystems). The cycling conditions and primer sequences have been previously described [<a href="#pntd.0013459.ref009" class="usa-link" aria-describedby="pntd.0013459.ref009">9</a>]. Viral copy numbers were quantified by generating a standard curve from recombinant plasmid DNA containing the S segment of HTNV, as outlined in prior studies [<a href="#pntd.0013459.ref040" class="usa-link" aria-describedby="pntd.0013459.ref040">40</a>].</p></section><section id="sec012"><h3 class="pmc_sec_title">Multiplex PCR-based nanopore sequencing</h3>
<p>cDNA was amplified using previously described multiplex HTNV-specific primer mixtures designed for southern Korean strains, in combination with the Solg 2X Uh-Taq PCR Smart Mix (Solgent, Daejeon, ROK), following the protocol established by Prayitno et al (2024) [<a href="#pntd.0013459.ref027" class="usa-link" aria-describedby="pntd.0013459.ref027">27</a>]. Library preparation was performed using a Native Barcoding Kit 24 V14 (SQK-NBD 114.24; ONT). Following end preparation, the libraries were purified using AMPure XP beads (Beckman Coulter, CA, USA). The purified barcoded libraries were pooled, ligated to sequencing adapters, and subsequently sequenced on a MinION MK1C device (ONT) using a FLO-MIN114 (R10.4.1) flow cell for 14 h.</p>
<p>Raw reads were demultiplexed and adapter-trimmed using Porechop (v0.9.0), and only reads with a minimum Q-score of 7 were retained. Index sequences were removed, and to minimize the risk of artificial primer-derived sequence similarity, the first and last 20 nucleotides at the 5′ and 3′ ends of each read were trimmed using CLC Genomics Workbench (v22.0.2; Qiagen, Hilden, Germany). Filtered reads were then mapped to the HTNV reference genome (strain Ac20–5) using the following alignment parameters: match score = 2, mismatch cost = 3, length fraction = 0.7, similarity fraction = 0.8, and non-specific match handling = ignore. Genome coverage depth was subsequently calculated, and consensus sequences were generated based on majority-rule nucleotide calling at positions with a minimum depth of 5 × ; positions with coverage &lt;5 × were excluded and denoted as ‘N’. Indels and frameshift errors within open reading frames were manually corrected according to previously established error correction criteria by comparing the mapped reads with the reference genome [<a href="#pntd.0013459.ref025" class="usa-link" aria-describedby="pntd.0013459.ref025">25</a>]. Genome regions not covered by the reads were amplified and confirmed using conventional nested RT-PCR. Final consensus sequences were deposited in GenBank.</p></section><section id="sec013"><h3 class="pmc_sec_title">Rapid amplification of cDNA ends (RACE) PCR</h3>
<p>To acquire the 3’ and 5’ terminal sequences of the HTNV genome, RACE PCR was carried out using the 3’- and 5’-RACE System for Rapid Amplification of cDNA Ends, Version 2.0 (Invitrogen, Carlsbad, CA, USA), following the manufacturer’s protocol.</p></section><section id="sec014"><h3 class="pmc_sec_title">Phylogenetic analysis</h3>
<p>The genomic sequences of HTNV tripartite segments were aligned using the ClustalW algorithm in MegAlign (v5; DNASTAR, Madison, WI, USA). The most appropriate substitution model was determined to be GTR + G + I using the MEGA 7 software. Phylogenetic trees for each segment were constructed using Bayesian Evolutionary Analysis by Sampling Trees (BEAST v2.6.7) under a strict molecular clock model [<a href="#pntd.0013459.ref041" class="usa-link" aria-describedby="pntd.0013459.ref041">41</a>]. The analysis was conducted over 80 million iterations, ensuring an effective sample size of greater than 200.</p></section><section id="sec015"><h3 class="pmc_sec_title">Network analysis</h3>
<p>Phylogenetic networks were constructed using the median-joining algorithm implemented in Network software (version 10.2.0.0; Fluxus Engineering, Cambridge, UK), with the epsilon set to 0 to reduce reticulations and simplify network topology. The networks were generated from multiple sequence alignments of HTNV genomes, including 55 sequences for the S and M segments and 51 sequences for the L segment. Analyses were based on aligned nucleotide positions spanning the S segment (49–1,343 nt), the M segment (1–3,431 nt), and the L segment (28–6,506 nt). The median-joining method integrates minimum spanning trees with inferred median vectors to represent potential ancestral relationships and mutational steps among sequences. Lineages were delineated based on genetic distance, resulting in the identification of two major groups. Network visualization was performed using Network Publisher (version 2.1.2.5) and finalized in Adobe Illustrator 2021.</p></section><section id="sec016"><h3 class="pmc_sec_title">Amino acid substitution analysis</h3>
<p>The amino acid sequences of the three HTNV genes were aligned using the ClustalW method in MegAlign (v5; DNASTAR). Amino acid mutations in HTNV were identified for each variant by comparison with the reference HTNV Ac20–5. The lollipop plot was visualized using the <em>dplyr</em> and <em>ggplot2</em> packages in R version 4.3.3.</p></section></section><section id="sec017"><h2 class="pmc_sec_title">Results</h2>
<section id="sec018"><h3 class="pmc_sec_title">Field trapping for small mammals on Jeju Island, 2022–2023</h3>
<p>We collected 50 small mammals from five locations: across Seogwipo-si (Hogeun-dong and Seohong-dong) and Jeju-si (Bongseong-ri, Sangdae-ri, and Ora-dong), Jeju Island, ROK, during 2022–2023 (<a href="#pntd.0013459.g001" class="usa-link">Fig 1</a>). The captured animals included 47 <em>Apodemus chejuensis</em> (94%) and three <em>Crocidura shantungensis</em> individuals (6%) (<a href="#pntd.0013459.s003" class="usa-link">S1 Table</a>). The species identification of <em>A. chejuensis</em> was validated by phylogenetic analysis based on mitochondrial sequencing (<a href="#pntd.0013459.s001" class="usa-link">S1 Fig</a>).</p>
<figure class="fig xbox font-sm" id="pntd.0013459.g001"><h4 class="obj_head">Fig 1. Geographic locations of Hantaan virus (HTNV) carried by <em>Apodemus chejuensis</em> captured on Jeju Island, Republic of Korea, in 2022–2023.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373272_pntd.0013459.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/863b/12373272/473c909b29b1/pntd.0013459.g001.jpg" loading="lazy" height="635" width="750" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pntd.0013459.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The map delineates the five sampling sites: <strong>(A)</strong> Hogeun-dong, <strong>(B)</strong> Bongseong-ri, <strong>(C)</strong> Sangdae-ri, <strong>(D)</strong> Seohong-dong, and <strong>(E)</strong> Ora-dong. Locations where HTNV was detected are marked by red and blue circles, whereas black circles denote sites where HTNV was not detected. The initial map was generated using Quantum Geographical Information System (QGIS) 3.10 for Windows and subsequently refined in Adobe Illustrator 2020 (Adobe Systems Incorporated, California, USA). The base layer of the map was sourced from <a href="https://www.naturalearthdata.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.naturalearthdata.com/</a> and is freely available for use in any project without the need for permission.</p></figcaption></figure></section><section id="sec019"><h3 class="pmc_sec_title">Epidemiological surveillance of HTNV</h3>
<p>Serological testing of <em>A. chejuensis</em> revealed a positivity rate of 34% (16/47), whereas HTNV RNA was detected in 27.7% (13/47) (<a href="#pntd.0013459.t001" class="usa-link">Table 1</a>). HTNV RNA positivity rates were 27.6% (8/29) in Hogeun-dong, Seogwipo-si; 57.1% (4/7) in Bongseong-ri, Jeju-si; and 12.5% (1/8) in Sangdae-ri, Jeju-si. Males exhibited a higher viral positivity rate (37.5%, 9/24) than females (17.4%, 4/23). The prevalence also increased with weight: 17.4% (4/23) for 21–30 g, 54.5% (6/11) for 31–40 g, and 75% (3/4) for &gt;40 g. No HTNV RNA was detected in rodents weighing 11–20 g. The epidemiological characteristics of HTNV-infected <em>A. chejuensis</em> are shown in <a href="#pntd.0013459.s004" class="usa-link">S2 Table</a>. In this study, the shrews tested negative in both serological and molecular assays.</p>
<section class="tw xbox font-sm" id="pntd.0013459.t001"><h4 class="obj_head">Table 1. Serological and molecular prevalence of Hantaan virus (HTNV) in <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea, 2022– 2023.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" colspan="2" rowspan="1">Characteristic</th>
<th align="left" rowspan="1" colspan="1">Number of<br> captured rodents</th>
<th align="left" rowspan="1" colspan="1">HTNV Seroprevalence (%)</th>
<th align="left" rowspan="1" colspan="1">HTNV<br> Prevalence (%)<sup>a</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="2" rowspan="1">
<strong>Region, n = 47</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Seogwipo-si</td>
<td align="left" rowspan="1" colspan="1">Hogeun-dong</td>
<td align="left" rowspan="1" colspan="1">29</td>
<td align="left" rowspan="1" colspan="1">9/29 (31)</td>
<td align="left" rowspan="1" colspan="1">8/29 (27.6)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Seohong-dong</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">0/1</td>
<td align="left" rowspan="1" colspan="1">0/1</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Jeju-si</td>
<td align="left" rowspan="1" colspan="1">Bongseong-ri</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">6/7 (85.7)</td>
<td align="left" rowspan="1" colspan="1">4/7 (57.1)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ora-dong</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">0/2</td>
<td align="left" rowspan="1" colspan="1">0/2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Sangdae-ri</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">1/8 (12.5)</td>
<td align="left" rowspan="1" colspan="1">1/8 (12.5)</td>
</tr>
<tr>
<td align="left" colspan="2" rowspan="1">
<strong>Sex, n = 47</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" colspan="2" rowspan="1">Male</td>
<td align="left" rowspan="1" colspan="1">24</td>
<td align="left" rowspan="1" colspan="1">11/24 (45.8)</td>
<td align="left" rowspan="1" colspan="1">9/24 (37.5)</td>
</tr>
<tr>
<td align="left" colspan="2" rowspan="1">Female</td>
<td align="left" rowspan="1" colspan="1">23</td>
<td align="left" rowspan="1" colspan="1">5/23 (21.7)</td>
<td align="left" rowspan="1" colspan="1">4/23 (17.4)</td>
</tr>
<tr>
<td align="left" colspan="2" rowspan="1">
<strong>Weight, g, n = 47</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" colspan="2" rowspan="1">11-20</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">2/9 (22.2)</td>
<td align="left" rowspan="1" colspan="1">0/9</td>
</tr>
<tr>
<td align="left" colspan="2" rowspan="1">21-30</td>
<td align="left" rowspan="1" colspan="1">23</td>
<td align="left" rowspan="1" colspan="1">5/23 (21.7)</td>
<td align="left" rowspan="1" colspan="1">4/23 (17.4)</td>
</tr>
<tr>
<td align="left" colspan="2" rowspan="1">31-40</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">6/11 (54.5)</td>
<td align="left" rowspan="1" colspan="1">6/11 (54.5)</td>
</tr>
<tr>
<td align="left" colspan="2" rowspan="1">&gt; 40</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">3/4 (75)</td>
<td align="left" rowspan="1" colspan="1">3/4 (75)</td>
</tr>
<tr>
<td align="left" colspan="2" rowspan="1">
<strong>Total</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>47</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>16/47 (34)</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>13/47 (27.7)</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pntd.0013459.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t001fn001"><p><sup>a</sup>HTNV prevalence was confirmed using a nested reverse transcription-polymerase chain reaction.</p></div></div></section></section><section id="sec020"><h3 class="pmc_sec_title">Determination of HTNV RNA copy numbers from <em>A. chejuensis</em> lung tissues</h3>
<p>HTNV RNA loads in the lung tissues of 13 <em>A. chejuensis</em> were measured (<a href="#pntd.0013459.s005" class="usa-link">S3 Table</a>). Five rodents (Ac23–18, Ac23–20, Ac23–15, Ac23–19, and Ac22–24) exhibited the highest viral RNA loads, with the Ct values ranging from 15.9 to 20.3, corresponding to 10⁵–10⁶ copies/μL. Three individuals (Ac23–1, Ac23–17, and Ac23–19) showed Ct values ranging from 23.0 to 24.2, indicating viral RNA amounts of 10⁴–10⁵ copies/μL. Ac23–12 and Ac23–22 had viral RNA quantities of 10²–10³ copies/μL, with Ct values between 27.2 and 29.0. Additionally, three <em>A. chejuensis</em> strains (Ac23–14, Ac22–20, and Ac22–23) exhibited the lowest viral loads, ranging from 0 to 1 copy, with Ct values &gt;38.0.</p></section><section id="sec021"><h3 class="pmc_sec_title">Whole-genome sequencing of HTNV</h3>
<p>Using amplicon-based nanopore sequencing, nearly complete genome sequences of HTNV were obtained from lung tissues of <em>A. chejuensis</em> collected from Jeju Island, ROK, between 2022 and 2023 (<a href="#pntd.0013459.t002" class="usa-link">Table 2</a>). Five rodents (Ac23–18, Ac23–20, Ac23–15, Ac23–19, and Ac22–24) exhibited the highest viral loads, containing 10⁵ to 10⁶ copies/µL, with genome coverages of 96.4%–98.5%, 97.5%–99.3%, and 97.1%–99.6% for the S, M, and L segments, respectively. Three individuals (Ac23–1, Ac23–17, and Ac23–19) with RNA copy numbers ranging from 10⁴ to 10⁵ copies/µL showed coverages of 98.5%, 95.3%–99.8%, and 95.8%–98.6% for the S, M, and L segments, respectively. Two variants (Ac23–12 and Ac23–22) with 10² to 10³ copies/µL displayed coverages of 95.7%–96.3%, 87.6%–91.6%, and 93.7%–93.8% for the S, M, and L segments, respectively. The three samples with the lowest viral RNA amounts (0–1 copy/µL; Ac23–14, Ac22–20, and Ac22–23) exhibited genome coverages of 86.0%–96.3%, 80.8%–91.2%, and 71.8%–93.0% for the S, M, and L segments, respectively. The average number of mapped viral reads and sequencing depths are listed in <a href="#pntd.0013459.s006" class="usa-link">S4 Table</a>.</p>
<section class="tw xbox font-sm" id="pntd.0013459.t002"><h4 class="obj_head">Table 2. Genome coverage rate of Hantaan virus (HTNV) in lung tissues of <em>Apodemus chejuensis</em> collected from Jeju Island, based on viral RNA copy number.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Viral RNA<br>copy number<br>(copies/uL)</th>
<th align="left" rowspan="2" colspan="1">Sample</th>
<th align="left" colspan="2" rowspan="2">Collection site</th>
<th align="left" colspan="3" rowspan="1">HTNV genomes, coverage rate<sup>a</sup>
</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1">S segment</th>
<th align="left" rowspan="1" colspan="1">M segment</th>
<th align="left" rowspan="1" colspan="1">L segment</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="5" colspan="1">10<sup>5</sup> to 10<sup>6</sup>
</td>
<td align="left" rowspan="1" colspan="1">Ac23–18</td>
<td align="left" rowspan="1" colspan="1">Seogwipo-si</td>
<td align="left" rowspan="1" colspan="1">Hogeun-dong</td>
<td align="left" rowspan="1" colspan="1">96.4</td>
<td align="left" rowspan="1" colspan="1">97.5</td>
<td align="left" rowspan="1" colspan="1">97.1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ac23–20</td>
<td align="left" rowspan="1" colspan="1">Seogwipo-si</td>
<td align="left" rowspan="1" colspan="1">Hogeun-dong</td>
<td align="left" rowspan="1" colspan="1">98.1</td>
<td align="left" rowspan="1" colspan="1">99.1</td>
<td align="left" rowspan="1" colspan="1">99.6</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ac23–15</td>
<td align="left" rowspan="1" colspan="1">Seogwipo-si</td>
<td align="left" rowspan="1" colspan="1">Hogeun-dong</td>
<td align="left" rowspan="1" colspan="1">98.5</td>
<td align="left" rowspan="1" colspan="1">99.1</td>
<td align="left" rowspan="1" colspan="1">98.8</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ac23–19</td>
<td align="left" rowspan="1" colspan="1">Seogwipo-si</td>
<td align="left" rowspan="1" colspan="1">Hogeun-dong</td>
<td align="left" rowspan="1" colspan="1">98.5</td>
<td align="left" rowspan="1" colspan="1">99.1</td>
<td align="left" rowspan="1" colspan="1">98.7</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ac22–24</td>
<td align="left" rowspan="1" colspan="1">Jeju-si</td>
<td align="left" rowspan="1" colspan="1">Bongseong-ri</td>
<td align="left" rowspan="1" colspan="1">98.5</td>
<td align="left" rowspan="1" colspan="1">99.3</td>
<td align="left" rowspan="1" colspan="1">99.6</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">10<sup>4</sup> to 10<sup>5</sup>
</td>
<td align="left" rowspan="1" colspan="1">Ac23–1</td>
<td align="left" rowspan="1" colspan="1">Jeju-si</td>
<td align="left" rowspan="1" colspan="1">Sangdae-ri</td>
<td align="left" rowspan="1" colspan="1">98.5</td>
<td align="left" rowspan="1" colspan="1">96.5</td>
<td align="left" rowspan="1" colspan="1">98.6</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ac23–17</td>
<td align="left" rowspan="1" colspan="1">Seogwipo-si</td>
<td align="left" rowspan="1" colspan="1">Hogeun-dong</td>
<td align="left" rowspan="1" colspan="1">98.5</td>
<td align="left" rowspan="1" colspan="1">99.8</td>
<td align="left" rowspan="1" colspan="1">95.8</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ac22–19</td>
<td align="left" rowspan="1" colspan="1">Jeju-si</td>
<td align="left" rowspan="1" colspan="1">Bongseong-ri</td>
<td align="left" rowspan="1" colspan="1">98.5</td>
<td align="left" rowspan="1" colspan="1">95.3</td>
<td align="left" rowspan="1" colspan="1">98.3</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">10<sup>2</sup> to 10<sup>3</sup>
</td>
<td align="left" rowspan="1" colspan="1">Ac23–12</td>
<td align="left" rowspan="1" colspan="1">Seogwipo-si</td>
<td align="left" rowspan="1" colspan="1">Hogeun-dong</td>
<td align="left" rowspan="1" colspan="1">96.3</td>
<td align="left" rowspan="1" colspan="1">87.6</td>
<td align="left" rowspan="1" colspan="1">93.7</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ac23–22</td>
<td align="left" rowspan="1" colspan="1">Seogwipo-si</td>
<td align="left" rowspan="1" colspan="1">Hogeun-dong</td>
<td align="left" rowspan="1" colspan="1">95.7</td>
<td align="left" rowspan="1" colspan="1">91.6</td>
<td align="left" rowspan="1" colspan="1">93.8</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">0 to 1</td>
<td align="left" rowspan="1" colspan="1">Ac23–14</td>
<td align="left" rowspan="1" colspan="1">Seogwipo-si</td>
<td align="left" rowspan="1" colspan="1">Hogeun-dong</td>
<td align="left" rowspan="1" colspan="1">96.3</td>
<td align="left" rowspan="1" colspan="1">91.2</td>
<td align="left" rowspan="1" colspan="1">93.0</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ac22–20</td>
<td align="left" rowspan="1" colspan="1">Jeju-si</td>
<td align="left" rowspan="1" colspan="1">Bongseong-ri</td>
<td align="left" rowspan="1" colspan="1">91.9</td>
<td align="left" rowspan="1" colspan="1">80.8</td>
<td align="left" rowspan="1" colspan="1">73.9</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ac22–23</td>
<td align="left" rowspan="1" colspan="1">Jeju-si</td>
<td align="left" rowspan="1" colspan="1">Bongseong-ri</td>
<td align="left" rowspan="1" colspan="1">86.0</td>
<td align="left" rowspan="1" colspan="1">81.3</td>
<td align="left" rowspan="1" colspan="1">71.8</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pntd.0013459.t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="t002fn001"><p><sup>a</sup>Coverage was estimated by mapping viral reads to the HTNV reference genome (strain Ac20–5) using CLC Genomics Workbench (v22.0.2; Qiagen). Nucleotide positions with coverage depth &lt;5 were excluded and denoted as ‘N’. The coverage rate for each segment was calculated using the formula: (covered genome length/ reference genome size) × 100.</p></div>
<div class="fn" id="t002fn002"><p>Ac, <em>Apodemus chejuensis</em><span class="text-strike">.</span></p></div>
</div></section><p>Nine full-length genomic sequences of HTNV were recovered from rodents captured at three locations: Hogeun-dong in Seogwipo-si and Sangdae-ri and Bongseong-ri in Jeju-si, Jeju Island, ROK, in 2022–2023. Gaps in the HTNV tripartite RNA genome sequences were filled using conventional nested RT-PCR. To achieve a whole-genome sequence of HTNV, 5′ and 3′ RACE PCR was conducted on HTNV Ac23–18, a specimen from Seogwipo-si on Jeju Island with the highest viral copy number. The 5′ terminal sequence of HTNV tripartite RNA was determined to be 5′-GGA GUC UAC UAC UA-3′, whereas the 3′ end sequence was identified as 5′-UAG UAG UAU GCU CC-3, ’ corresponding to conserved termini sequences of the reference genome sequence (HTNV Ac20–5) [<a href="#pntd.0013459.ref009" class="usa-link" aria-describedby="pntd.0013459.ref009">9</a>]. Terminal sequences of the remaining HTNV specimens were recovered empirically.</p></section><section id="sec022"><h3 class="pmc_sec_title">Genome comparison of HTNV</h3>
<p>Nucleotide comparison analysis revealed that the HTNV genome shared 99.2% identity in the S segment, 99.1%–99.2% identity in the M segment, and 99.4–99.8% identity in the L segment with the reference HTNV Ac20–5, which was originally detected in <em>A. chejuensis</em> captured in Ora-dong, Jeju-si, ROK (<a href="#pntd.0013459.s007" class="usa-link">S5 Table</a>). The amino acid sequences showed similarities of 99.4%–99.6% for the N protein, 99.5%–99.6% for the glycoproteins, and 99.5%–99.6% for the RdRp.</p></section><section id="sec023"><h3 class="pmc_sec_title">Phylodynamic analysis of HTNV</h3>
<p>A phylodynamic analysis was conducted to explore the genetic diversity and mutation dynamics among the tripartite genomes of HTNV derived from <em>A. chejuensis</em> collected from Jeju Island, ROK, from 2022 to 2023 (<a href="#pntd.0013459.g002" class="usa-link">Fig 2</a>). Phylogenetic analysis of nine HTNV variants, including Ac23–15, Ac23–17, Ac23–18, Ac23–19, Ac23–20, and Ac23–22 from Hogeun-dong in Seogwipo-si; Ac23–1 from Sangdae-ri; and Ac22–19 and Ac22–24 from Bongseong-ri in Jeju-si, demonstrated consistent clustering into a homologous genetic group along with other HTNV variants collected from Jeju Island, ROK. However, discordance in evolutionary divergence was observed within the S segment of the HTNV variant Ac23–22 from Hogeun-dong in Seogwipo-si compared to the M and L segments.</p>
<figure class="fig xbox font-sm" id="pntd.0013459.g002"><h4 class="obj_head">Fig 2. Phylodynamic analysis of Hantaan virus (HTNV) S, M, and L segments from <em>Apodemus chejuensis</em> captured on Jeju Island, Republic of Korea (ROK), in 2022–2023.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373272_pntd.0013459.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/863b/12373272/3b35c3d65d1d/pntd.0013459.g002.jpg" loading="lazy" height="268" width="746" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pntd.0013459.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Nine whole-genome sequences of HTNV were newly obtained in this study. Phylogenetic inference was performed using an alignment based on the HTNV <strong>(A</strong>) S (49–1,343 nt), <strong>(B)</strong> M (1-3,431 nt), and <strong>(C)</strong> L (28-6,506 nt) genomic sequence, using the strict clock method within the Bayesian Evolutionary Analysis Sampling Trees (BEAST, v2.6.7). A total of 80 million iterations were conducted to ensure an effective sample size greater than 200. Posterior probabilities are displayed at the tips of each node, with colored circles indicating nodes with posterior values. The arrowhead marks the branch point of differentiation associated with a key amino acid variation in each segment, which was notably found in Seogwipo-si, Jeju Island, ROK. Sequence colors correspond to their respective collection sites at the city level, with the newly acquired HTNV sequences from this study denoted by stars. Accession numbers for the hantaviral sequences from this study are listed in <a href="#pntd.0013459.s008" class="usa-link">S6 Table</a>.</p></figcaption></figure><p>Moreover, four unique amino acid substitutions were identified in the tripartite genomes of HTNV variants from Seogwipo-si (<a href="#pntd.0013459.t003" class="usa-link">Table 3</a> and <a href="#pntd.0013459.s002" class="usa-link">S2 Fig</a>). These mutations comprised one in the S segment (N protein; R199K), two in the M segment (glycoproteins; V563L and E1019K), and one in the L segment (RdRp; F439S) when compared to the corresponding sequences of HTNV variants from Jeju-si.</p>
<section class="tw xbox font-sm" id="pntd.0013459.t003"><h4 class="obj_head">Table 3. Key amino acid substitutions in the S, M, and L segments of Hantaan virus (HTNV) carried by <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="1" colspan="1">Segment</th>
<th align="left" colspan="3" rowspan="1">Amino acid substitution<sup>a</sup>
</th>
<th align="left" rowspan="1" colspan="1">Variation site</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">Seogwipo-si</th>
<th align="left" rowspan="1" colspan="1">Jeju-si</th>
<th align="left" rowspan="1" colspan="1">Reference</th>
<th align="left" rowspan="1" colspan="1"></th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">S</td>
<td align="left" rowspan="1" colspan="1">K</td>
<td align="left" rowspan="1" colspan="1">R</td>
<td align="left" rowspan="1" colspan="1">R</td>
<td align="left" rowspan="1" colspan="1">199</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">M</td>
<td align="left" rowspan="1" colspan="1">L</td>
<td align="left" rowspan="1" colspan="1">V</td>
<td align="left" rowspan="1" colspan="1">V</td>
<td align="left" rowspan="1" colspan="1">563</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">K</td>
<td align="left" rowspan="1" colspan="1">E</td>
<td align="left" rowspan="1" colspan="1">E</td>
<td align="left" rowspan="1" colspan="1">1019</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">L</td>
<td align="left" rowspan="1" colspan="1">S</td>
<td align="left" rowspan="1" colspan="1">F</td>
<td align="left" rowspan="1" colspan="1">F</td>
<td align="left" rowspan="1" colspan="1">439</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pntd.0013459.t003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t003fn001"><p><sup>a</sup>, Key amino acid substitutions were determined in the genomic sequences of HTNV from Seogwipo-si and compared to the reference HTNV Ac20–5.</p></div></div></section></section><section id="sec024"><h3 class="pmc_sec_title">Phylogeographic network analysis of HTNV</h3>
<p>Network analyses revealed a well-supported evolutionary divergence and genetic diversity among the HTNV S, M, and L genomes across various geographic regions of Jeju Island and the Korean Peninsula (<a href="#pntd.0013459.g003" class="usa-link">Fig 3</a>). Two distinct phylogenetic lineages were identified in the ROK. Lineage 1 encompasses regions in the northern ROK, including Pocheon-si, Yeoncheon-gun, Dongducheon-si, and Paju-si in Gyeonggi Province, and Chuncheon-si, Hwacheon-gun, Cheorwon-gun, and Yanggu-gun in Gangwon Province. Conversely, lineage 2 was associated with the southern regions of the ROK, including Seogwipo-si and Jeju-si on Jeju Island, and Gwangju-si and Boseong-gun in Jeollanam Province. These two lineages were defined based on consistent phylogeographic clustering and regional separation observed across all three genomic segments. The mean nucleotide divergence between Lineage 1 and Lineage 2 ranged from approximately 12% to 18%, depending on the segment.</p>
<figure class="fig xbox font-sm" id="pntd.0013459.g003"><h4 class="obj_head">Fig 3. Phylogeographic network analysis of Hantaan virus (HTNV) from <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea (ROK), in 2022–2023.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373272_pntd.0013459.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/863b/12373272/8adcf94a63d5/pntd.0013459.g003.jpg" loading="lazy" height="728" width="733" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pntd.0013459.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The phylogeographical network was generated from alignments of HTNV tripartite genomes for 55 genomic sequences (for S and M segments) and 51 genomic sequences (for L segment). <strong>(A)</strong> The geographic distributions of HTNV variants are indicated by colored circles at the collection sites on the map. The initial map was generated using Quantum Geographical Information System (QGIS) 3.10 for Windows. The base layer of the map was sourced from <a href="https://www.naturalearthdata.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.naturalearthdata.com/</a> and is freely available for use in any project without the need for permission. Networks were constructed using the median-joining method in Network software (version 10.2.0.0; Fluxus Engineering, Cambridge, UK) with an epsilon value of 0, based on the HTNV <strong>(B)</strong> S (49–1,343 nt), <strong>(C)</strong> M (1–3,431 nt), and <strong>(D)</strong> L (28–6,506 nt) segments. Color-coded symbols represent individual HTNV variants according to their collection sites, with circle sizes proportional to the connection frequency between viruses. The nodes illustrate the relationships among viral strains, with red numbers on the nodes indicating the total nucleotide mutations between strains. Lineage 1 spans regions in northern ROK, including Pocheon-si, Yeoncheon-gun, Dongducheon-si, and Paju-si in Gyeonggi Province, and Chuncheon-si, Hwacheon-gun, Cheorwon-gun, and Yanggu-gun in Gangwon Province. Lineage 2 covers areas in southern ROK, including Seogwipo-si and Jeju-si on Jeju Island, as well as Gwangju-si and Boseong-gun in Jeollanam Province. HTNV genomes from China were used as the outgroup in this study. Accession numbers for the hantaviral sequences from this study are listed in <a href="#pntd.0013459.s008" class="usa-link">S6 Table</a>.</p></figcaption></figure></section></section><section id="sec025"><h2 class="pmc_sec_title">Discussion</h2>
<p>Genomic surveillance, in conjunction with epidemiology, has played a crucial role in informing public health strategies for HFRS by elucidating the transmission pathways between patients and their infection sources [<a href="#pntd.0013459.ref042" class="usa-link" aria-describedby="pntd.0013459.ref042">42</a>]. For instance, in 2005, clinical sequencing of HTNV from four U.S. army soldiers diagnosed with HFRS during their deployment with U.S. Forces Korea (USFK) demonstrated a phylogeographic association between the viral sequences and those recovered from rodents captured at military training sites where the soldiers had been active [<a href="#pntd.0013459.ref043" class="usa-link" aria-describedby="pntd.0013459.ref043">43</a>]. Subsequent genomic investigations conducted between 2013 and 2015, where near-complete HTNV genome sequences from both ROK and USFK soldiers, further substantiated these epidemiological correlations [<a href="#pntd.0013459.ref044" class="usa-link" aria-describedby="pntd.0013459.ref044">44</a>]. Notably, active surveillance efforts involving targeted rodent trapping at sites suspected of orthohantavirus outbreaks linked to HFRS have allowed researchers to distinguish between viral lineages within short distances (approximately 5 km), making it possible to track specific strains of the virus and identify likely sites of exposure more accurately [<a href="#pntd.0013459.ref045" class="usa-link" aria-describedby="pntd.0013459.ref045">45</a>]. In the present study, nine whole-genome sequences of HTNV were obtained from <em>A. chejuensis</em> collected from Seogwipo-si and Jeju-si, Jeju Island, ROK, between 2022 and 2023. These findings represent the first comprehensive characterization of the complete genome sequences of HTNV from Seogwipo-si (Hogeun-dong) and Jeju-si (Sangdae-ri), Jeju Island. Jeju Island has been recognized as a HFRS-endemic region since the official identification of the first clinical case in 2005 (accessible online at <a href="https://dportal.kdca.go.kr/pot/is/rginEDW.do" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dportal.kdca.go.kr/pot/is/rginEDW.do</a>). To date, 28 clinical cases have been reported on this island. However, only a single clinical case has been documented as a case report on the island, without any molecular clues provided by genomic epidemiology owing to the lack of direct clinical sequencing of the patient samples [<a href="#pntd.0013459.ref046" class="usa-link" aria-describedby="pntd.0013459.ref046">46</a>]. Recently, genomic surveillance revealed the etiological agent and clinical characteristics of HFRS in the southern areas of the ROK (Gwangju Metropolitan City and Boseong-gun), providing molecular evidence of a novel HTNV genotype that causes hantaviral diseases, which was clustered as lineage 2 in our study [<a href="#pntd.0013459.ref027" class="usa-link" aria-describedby="pntd.0013459.ref027">27</a>]. To clarify the transmission dynamics and pathogenicity of HTNV, further studies should focus on sequencing and characterization of the clinical features of HFRS in patients from Jeju Island, ROK.</p>
<p>In this study, we conducted a comparative genomic analysis to explore the evolutionary dynamics of HTNV carried by <em>A. chejuensis</em> on Jeju Island, ROK. Phylogeographic analyses revealed geographically structured evolutionary divergence among HTNV variants sampled from different regions of the island. Phylogenetic incongruence was observed in the tripartite genome of HTNV Ac23–22, with the S segment diverging from the cluster formed by its M and L segments, which grouped with strains from Hogeun-dong, Seogwipo-si. Such patterns may result from segment-specific evolutionary pressures or past reassortment events, as genome reassortment among closely related strains has been reported in hantaviruses [<a href="#pntd.0013459.ref047" class="usa-link" aria-describedby="pntd.0013459.ref047">47</a>–<a href="#pntd.0013459.ref049" class="usa-link" aria-describedby="pntd.0013459.ref049">49</a>]. In the ROK, a putative hybrid zone has been proposed where lineage overlap may facilitate segment exchange [<a href="#pntd.0013459.ref050" class="usa-link" aria-describedby="pntd.0013459.ref050">50</a>,<a href="#pntd.0013459.ref051" class="usa-link" aria-describedby="pntd.0013459.ref051">51</a>]. GiRaF analysis did not detect any statistically supported reassortment events in our dataset. While earlier studies have provided an important genomic foundation, the number of complete HTNV sequences from Jeju Island remains limited. Continued efforts to expand genomic sampling will be essential for refining our understanding of HTNV phylogeography and evolutionary dynamics in this under-sampled region.</p>
<p>The classification of DOBV lineages (Dobrava, Sochi, Kurkino, and Saaremaa) is primarily based on distinctions in their phylogenetic relationships, associated rodent reservoirs, and geographic distribution [<a href="#pntd.0013459.ref052" class="usa-link" aria-describedby="pntd.0013459.ref052">52</a>]. Despite the high degree of genetic similarity among these genotypes, they elicit HFRS of varying severities. The case fatality rate (CFR) of infections attributed to the Dobrava genotype, which is carried by <em>A. flavicollis</em>, ranges between 10% and 12% [<a href="#pntd.0013459.ref053" class="usa-link" aria-describedby="pntd.0013459.ref053">53</a>]. Conversely, the HFRS cases linked to the Sochi genotype hosted by <em>A. ponticus</em> exhibited a CFR of approximately 6% [<a href="#pntd.0013459.ref054" class="usa-link" aria-describedby="pntd.0013459.ref054">54</a>]. Moreover, orthohantavirus outbreaks related to the Kurkino genotype harbored by <em>A. agrarius</em> demonstrated a CFR of less than 1% in regions of European Russia. Notably, infections caused by the Saaremaa genotype, which is also carried by <em>A. agrarius</em>, tend to be subclinical [<a href="#pntd.0013459.ref055" class="usa-link" aria-describedby="pntd.0013459.ref055">55</a>]. Although these DOBV lineages differ in clinical severity and host specificity, genetic diversity also plays a key role in shaping the pathogenic potential of other orthohantaviruses such as HTNV. In this study, pairwise sequence comparisons revealed high nucleotide and amino acid identity between the newly obtained HTNV genomes from Seogwipo-si and the reference strain Ac20–5, with divergence not exceeding 1% across all segments. However, four unique amino acid substitutions were identified in the HTNV genomes from Seogwipo-si—R199K in the S segment (N protein), V563L and E1019K in the M segment (glycoproteins), and F439S in the L segment (RdRp)—compared to those from Jeju-si. These mutations, while possibly resulting from natural genetic variability, were geographically distinct and may warrant further investigation. Although HTNV strains from Jeju Island have not yet been isolated, future studies incorporating virus isolation and functional analyses will be important to determine the potential phenotypic relevance of these substitutions.</p>
<p>While this study provides valuable genomic insights into HTNV diversity on Jeju Island, several limitations should be acknowledged. 1) The total number of small mammals collected was limited, despite a systematic trapping design across ecologically distinct sites. As a result, the current findings offer only a preliminary indication of HTNV prevalence, and more extensive, longitudinal sampling will be necessary to better define its geographic distribution and infection dynamics. 2) Several amino acid substitutions were identified, but their potential structural or functional implications could not be evaluated within the scope of this study. Future work should incorporate protein modelling and virus isolation followed by experimental validation to clarify the biological significance of these variations. 3) The phylodynamic analysis was based on a relatively small number of sequences from a narrow temporal and geographic window, limiting the resolution of inferred evolutionary patterns. Broader and longer-term sampling will be essential to strengthen future phylodynamic inferences. 4) The origin of HTNV strains detected on Jeju Island remains unclear. While sequence similarity with strains from the Korean mainland suggests a possible introduction from the peninsula, phylogeographic interpretation is limited by the lack of comparative genomic data from nearby regions, particularly southeastern China. Further regional sampling will be needed to better clarify the virus’s evolutionary history.</p>
<p>In conclusion, nine complete HTNV genomic sequences were acquired from rodents collected from Jeju Island, ROK, between 2022 and 2023. This is the first complete characterization of HTNV genomes from Seogwipo-si (Hogeun-dong) and Jeju-si (Sangdae-ri), enhancing the phylogeographic resolution of the region. Phylodynamic analysis revealed a well-supported evolutionary divergence and geographic diversity across Jeju Island, with four unique amino acid substitutions identified in the HTNV genomes from Seogwipo-si. These findings provide valuable insights into the genomic surveillance, phylogenetic diversity, and evolutionary dynamics of orthohantaviruses, which are crucial for developing effective strategies to mitigate HFRS outbreaks in the ROK.</p></section><section id="sec026"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pntd.0013459.s001"><div class="caption p">
<span>S1 Fig. Phylogeny based on the mitochondrial DNA cytochrome <em>b</em> (CYTB) gene of <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea (ROK), in 2022–2023.</span><p>A phylogenetic tree generated from the mitochondrial DNA CYTB gene sequences (686 bps) of <em>A. chejuensis</em> captured on Jeju Island, ROK. A phylogenetic analysis was conducted with the HKY + G substitution model using the maximum likelihood method in MEGA 7.0. The branch lengths in the phylogenetic tree correspond to the number of nucleotide substitutions, while the vertical distances are adjusted for improved visual clarity. Bootstrap probabilities, calculated from 1,000 iterations, are indicated at each node. In this figure, the genomic sequences of the CYTB gene from <em>A. chejuensis</em> are displayed, with the newly obtained sequences highlighted in bold red font.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s001.tif" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s001.tif</a><sup> (3.9MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pntd.0013459.s002"><div class="caption p">
<span>S2 Fig. Amino acid substitutions in the tripartite genomes of Hantaan virus (HTNV) carried by <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea, in 2022–2023.</span><p>The lollipop plot illustrates the positions of amino acid mutations in the nine HTNV sequences obtained in this study, compared to the reference HTNV Ac20–5 genomic sequences. The plot was visualized using the <em>dplyr</em> and <em>ggplot2</em> packages in R version 4.3.3. Each node represents a point of amino acid variations, with the node length indicating the frequency of HTNV amino acid substitutions across different rodent collection sites. The color-coded bars correspond to the tripartite segments: S segment (blue), M segment (green), and L segment (red). Red nodes highlight key amino acid changes uniquely found in sequences from Seogwipo-si. The scale bar represents the length of HTNV amino acids and nucleotides.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s002.tif" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s002.tif</a><sup> (4MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pntd.0013459.s003"><div class="caption p">
<span>S1 Table. Species composition of 50 small mammals captured on Jeju Island, Republic of Korea, in 2022–2023.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s003.pdf</a><sup> (320KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pntd.0013459.s004"><div class="caption p">
<span>S2 Table. Characteristics of Hantaan virus (HTNV)-infected <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea, in 2022–2023.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s004.pdf</a><sup> (319.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pntd.0013459.s005"><div class="caption p">
<span>S3 Table. Hantaan virus RNA copy numbers from the lung tissues of <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s005.pdf</a><sup> (319.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pntd.0013459.s006"><div class="caption p">
<span>S4 Table. Summary of mapped reads and sequencing depth of amplicon-based nanopore sequencing for Hantaan virus harbored by <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea, in 2022–2023.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s006.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s006.pdf</a><sup> (320KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pntd.0013459.s007"><div class="caption p">
<span>S5 Table. Nucleotide and amino acid sequence similarities of Hantaan virus (HTNV) carried by <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea, in 2022–2023.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s007.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s007.pdf</a><sup> (319.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pntd.0013459.s008"><div class="caption p">
<span>S6 Table. Accession numbers of genomic sequences of Hantaan virus (HTNV) S, M, and L segments in the current study.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s008.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s008.pdf</a><sup> (319.4KB, pdf) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>The data presented and supporting information in this study are available from GenBank (accession numbers: PQ410874–PQ410913).</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This work was supported by the National Research Foundation of Korea (NRF) grant funded by the Korean government (MSIT) (2023R1A2C2006105 to J.-W.S.), Korea University (to J.-W.S.), and Government-wide R&amp;D to Advance Infectious Disease Prevention and Control, Republic of Korea (grant number RS-2023-KH140418 to W.-K.K.). The funders played no role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pntd.0013459.ref001">
<span class="label">1.</span><cite>Vaheri A, Strandin T, Hepojoki J, Sironen T, Henttonen H, Mäkelä S, et al. Uncovering the mysteries of hantavirus infections. Nat Rev Microbiol. 2013;11(8):539–50. doi: 10.1038/nrmicro3066

</cite> [<a href="https://doi.org/10.1038/nrmicro3066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24020072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol&amp;title=Uncovering%20the%20mysteries%20of%20hantavirus%20infections&amp;author=A%20Vaheri&amp;author=T%20Strandin&amp;author=J%20Hepojoki&amp;author=T%20Sironen&amp;author=H%20Henttonen&amp;volume=11&amp;issue=8&amp;publication_year=2013&amp;pages=539-50&amp;pmid=24020072&amp;doi=10.1038/nrmicro3066&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref002">
<span class="label">2.</span><cite>Peters CJ, Simpson GL, Levy H. Spectrum of hantavirus infection: hemorrhagic fever with renal syndrome and hantavirus pulmonary syndrome. Annu Rev Med. 1999;50:531–45. doi: 10.1146/annurev.med.50.1.531

</cite> [<a href="https://doi.org/10.1146/annurev.med.50.1.531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10073292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Med&amp;title=Spectrum%20of%20hantavirus%20infection:%20hemorrhagic%20fever%20with%20renal%20syndrome%20and%20hantavirus%20pulmonary%20syndrome&amp;author=CJ%20Peters&amp;author=GL%20Simpson&amp;author=H%20Levy&amp;volume=50&amp;publication_year=1999&amp;pages=531-45&amp;pmid=10073292&amp;doi=10.1146/annurev.med.50.1.531&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref003">
<span class="label">3.</span><cite>Kabwe E, Davidyuk Y, Shamsutdinov A, Garanina E, Martynova E, Kitaeva K, et al. Orthohantaviruses, Emerging Zoonotic Pathogens. Pathogens. 2020;9(9):775. doi: 10.3390/pathogens9090775

</cite> [<a href="https://doi.org/10.3390/pathogens9090775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7558059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32971887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pathogens&amp;title=Orthohantaviruses,%20Emerging%20Zoonotic%20Pathogens&amp;author=E%20Kabwe&amp;author=Y%20Davidyuk&amp;author=A%20Shamsutdinov&amp;author=E%20Garanina&amp;author=E%20Martynova&amp;volume=9&amp;issue=9&amp;publication_year=2020&amp;pages=775&amp;pmid=32971887&amp;doi=10.3390/pathogens9090775&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref004">
<span class="label">4.</span><cite>Forbes KM, Sironen T, Plyusnin A. Hantavirus maintenance and transmission in reservoir host populations. Curr Opin Virol. 2018;28:1–6. doi: 10.1016/j.coviro.2017.09.003

</cite> [<a href="https://doi.org/10.1016/j.coviro.2017.09.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29024905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Virol&amp;title=Hantavirus%20maintenance%20and%20transmission%20in%20reservoir%20host%20populations&amp;author=KM%20Forbes&amp;author=T%20Sironen&amp;author=A%20Plyusnin&amp;volume=28&amp;publication_year=2018&amp;pages=1-6&amp;pmid=29024905&amp;doi=10.1016/j.coviro.2017.09.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref005">
<span class="label">5.</span><cite>Noh Y-T, Cho J-E, Han MG, Lee N-Y, Kim S-Y, Chu C, et al. Seroepidemiological Characteristics of Haemorrhagic Fever with Renal Syndrome from 1996 to 2005 in Korea. J Bacteriol Virol. 2006;36(4):263. doi: 10.4167/jbv.2006.36.4.263</cite> [<a href="https://doi.org/10.4167/jbv.2006.36.4.263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol%20Virol&amp;title=Seroepidemiological%20Characteristics%20of%20Haemorrhagic%20Fever%20with%20Renal%20Syndrome%20from%201996%20to%202005%20in%20Korea&amp;author=Y-T%20Noh&amp;author=J-E%20Cho&amp;author=MG%20Han&amp;author=N-Y%20Lee&amp;author=S-Y%20Kim&amp;volume=36&amp;issue=4&amp;publication_year=2006&amp;pages=263&amp;doi=10.4167/jbv.2006.36.4.263&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref006">
<span class="label">6.</span><cite>Lee HW, Lee PW, Johnson KM. Isolation of the etiologic agent of Korean Hemorrhagic fever. J Infect Dis. 1978;137(3):298–308. doi: 10.1093/infdis/137.3.298

</cite> [<a href="https://doi.org/10.1093/infdis/137.3.298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&amp;title=Isolation%20of%20the%20etiologic%20agent%20of%20Korean%20Hemorrhagic%20fever&amp;author=HW%20Lee&amp;author=PW%20Lee&amp;author=KM%20Johnson&amp;volume=137&amp;issue=3&amp;publication_year=1978&amp;pages=298-308&amp;pmid=24670&amp;doi=10.1093/infdis/137.3.298&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref007">
<span class="label">7.</span><cite>Baek LJ, Kariwa H, Lokugamage K, Yoshimatsu K, Arikawa J, Takashima I, et al. Soochong virus: an antigenically and genetically distinct hantavirus isolated from Apodemus peninsulae in Korea. J Med Virol. 2006;78(2):290–7. doi: 10.1002/jmv.20538

</cite> [<a href="https://doi.org/10.1002/jmv.20538" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16372283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Med%20Virol&amp;title=Soochong%20virus:%20an%20antigenically%20and%20genetically%20distinct%20hantavirus%20isolated%20from%20Apodemus%20peninsulae%20in%20Korea&amp;author=LJ%20Baek&amp;author=H%20Kariwa&amp;author=K%20Lokugamage&amp;author=K%20Yoshimatsu&amp;author=J%20Arikawa&amp;volume=78&amp;issue=2&amp;publication_year=2006&amp;pages=290-7&amp;pmid=16372283&amp;doi=10.1002/jmv.20538&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref008">
<span class="label">8.</span><cite>Yashina L, Mishin V, Zdanovskaya N, Schmaljohn C, Ivanov L. A newly discovered variant of a hantavirus in Apodemus peninsulae, far Eastern Russia. Emerg Infect Dis. 2001;7(5):912–3. doi: 10.3201/eid0705.017530

</cite> [<a href="https://doi.org/10.3201/eid0705.017530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2631886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11747715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Emerg%20Infect%20Dis&amp;title=A%20newly%20discovered%20variant%20of%20a%20hantavirus%20in%20Apodemus%20peninsulae,%20far%20Eastern%20Russia&amp;author=L%20Yashina&amp;author=V%20Mishin&amp;author=N%20Zdanovskaya&amp;author=C%20Schmaljohn&amp;author=L%20Ivanov&amp;volume=7&amp;issue=5&amp;publication_year=2001&amp;pages=912-3&amp;pmid=11747715&amp;doi=10.3201/eid0705.017530&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref009">
<span class="label">9.</span><cite>Park K, Kim W-K, Lee S-H, Kim J, Lee J, Cho S, et al. A novel genotype of Hantaan orthohantavirus harbored by Apodemus agrarius chejuensis as a potential etiologic agent of hemorrhagic fever with renal syndrome in Republic of Korea. PLoS Negl Trop Dis. 2021;15(5):e0009400. doi: 10.1371/journal.pntd.0009400

</cite> [<a href="https://doi.org/10.1371/journal.pntd.0009400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8143423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33979351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Negl%20Trop%20Dis&amp;title=A%20novel%20genotype%20of%20Hantaan%20orthohantavirus%20harbored%20by%20Apodemus%20agrarius%20chejuensis%20as%20a%20potential%20etiologic%20agent%20of%20hemorrhagic%20fever%20with%20renal%20syndrome%20in%20Republic%20of%20Korea&amp;author=K%20Park&amp;author=W-K%20Kim&amp;author=S-H%20Lee&amp;author=J%20Kim&amp;author=J%20Lee&amp;volume=15&amp;issue=5&amp;publication_year=2021&amp;pmid=33979351&amp;doi=10.1371/journal.pntd.0009400&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref010">
<span class="label">10.</span><cite>Park K, No JS, Prayitno SP, Seo Y-R, Lee S-H, Noh J, et al. Epidemiological Surveillance and Genomic Characterization of Soochong Virus From Apodemus Species Using Multiplex PCR-Based Next-Generation Sequencing, Republic of Korea. J Med Virol. 2024;96(11):e70077. doi: 10.1002/jmv.70077

</cite> [<a href="https://doi.org/10.1002/jmv.70077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39588784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Med%20Virol&amp;title=Epidemiological%20Surveillance%20and%20Genomic%20Characterization%20of%20Soochong%20Virus%20From%20Apodemus%20Species%20Using%20Multiplex%20PCR-Based%20Next-Generation%20Sequencing,%20Republic%20of%20Korea&amp;author=K%20Park&amp;author=JS%20No&amp;author=SP%20Prayitno&amp;author=Y-R%20Seo&amp;author=S-H%20Lee&amp;volume=96&amp;issue=11&amp;publication_year=2024&amp;pmid=39588784&amp;doi=10.1002/jmv.70077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref011">
<span class="label">11.</span><cite>Koh HS, Lee WJ, Kocher TD. The genetic relationships of two subspecies of striped field mice, Apodemus agrarius coreae and Apodemus agrarius chejuensis. Heredity (Edinb). 2000;85 ( Pt 1):30–6. doi: 10.1046/j.1365-2540.2000.00723.x

</cite> [<a href="https://doi.org/10.1046/j.1365-2540.2000.00723.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10971688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity%20(Edinb)&amp;title=The%20genetic%20relationships%20of%20two%20subspecies%20of%20striped%20field%20mice,%20Apodemus%20agrarius%20coreae%20and%20Apodemus%20agrarius%20chejuensis&amp;author=HS%20Koh&amp;author=WJ%20Lee&amp;author=TD%20Kocher&amp;publication_year=2000&amp;pages=30-6&amp;pmid=10971688&amp;doi=10.1046/j.1365-2540.2000.00723.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref012">
<span class="label">12.</span><cite>Arai S, Gu SH, Baek LJ, Tabara K, Bennett SN, Oh H-S, et al. Divergent ancestral lineages of newfound hantaviruses harbored by phylogenetically related crocidurine shrew species in Korea. Virology. 2012;424(2):99–105. doi: 10.1016/j.virol.2011.11.013

</cite> [<a href="https://doi.org/10.1016/j.virol.2011.11.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3288197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22230701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virology&amp;title=Divergent%20ancestral%20lineages%20of%20newfound%20hantaviruses%20harbored%20by%20phylogenetically%20related%20crocidurine%20shrew%20species%20in%20Korea&amp;author=S%20Arai&amp;author=SH%20Gu&amp;author=LJ%20Baek&amp;author=K%20Tabara&amp;author=SN%20Bennett&amp;volume=424&amp;issue=2&amp;publication_year=2012&amp;pages=99-105&amp;pmid=22230701&amp;doi=10.1016/j.virol.2011.11.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref013">
<span class="label">13.</span><cite>Lee S-H, Kim W-K, Park K, No JS, Lee G-Y, Kim H-C, et al. Genetic diversity and phylogeography of Jeju Orthohantavirus (Hantaviridae) in the Republic of Korea. Virology. 2020;543:13–9. doi: 10.1016/j.virol.2020.01.012

</cite> [<a href="https://doi.org/10.1016/j.virol.2020.01.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32056842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virology&amp;title=Genetic%20diversity%20and%20phylogeography%20of%20Jeju%20Orthohantavirus%20(Hantaviridae)%20in%20the%20Republic%20of%20Korea&amp;author=S-H%20Lee&amp;author=W-K%20Kim&amp;author=K%20Park&amp;author=JS%20No&amp;author=G-Y%20Lee&amp;volume=543&amp;publication_year=2020&amp;pages=13-9&amp;pmid=32056842&amp;doi=10.1016/j.virol.2020.01.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref014">
<span class="label">14.</span><cite>Radford AD, Chapman D, Dixon L, Chantrey J, Darby AC, Hall N. Application of next-generation sequencing technologies in virology. J Gen Virol. 2012;93(Pt 9):1853–68. doi: 10.1099/vir.0.043182-0

</cite> [<a href="https://doi.org/10.1099/vir.0.043182-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3709572/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22647373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Gen%20Virol&amp;title=Application%20of%20next-generation%20sequencing%20technologies%20in%20virology&amp;author=AD%20Radford&amp;author=D%20Chapman&amp;author=L%20Dixon&amp;author=J%20Chantrey&amp;author=AC%20Darby&amp;volume=93&amp;publication_year=2012&amp;pages=1853-68&amp;pmid=22647373&amp;doi=10.1099/vir.0.043182-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref015">
<span class="label">15.</span><cite>Park K, Kim J, Noh J, Kim K, Yang E, Kim S-G, et al. First detection and characterization of hepatitis E virus (Rocahepevirus ratti) from urban Norway rats (Rattus norvegicus) in the Republic of Korea. J Med Virol. 2024;96(1):e29401. doi: 10.1002/jmv.29401

</cite> [<a href="https://doi.org/10.1002/jmv.29401" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38235603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Med%20Virol&amp;title=First%20detection%20and%20characterization%20of%20hepatitis%20E%20virus%20(Rocahepevirus%20ratti)%20from%20urban%20Norway%20rats%20(Rattus%20norvegicus)%20in%20the%20Republic%20of%20Korea&amp;author=K%20Park&amp;author=J%20Kim&amp;author=J%20Noh&amp;author=K%20Kim&amp;author=E%20Yang&amp;volume=96&amp;issue=1&amp;publication_year=2024&amp;pmid=38235603&amp;doi=10.1002/jmv.29401&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref016">
<span class="label">16.</span><cite>Lee G-Y, Kim W-K, Cho S, Park K, Kim J, Lee S-H, et al. Genotyping and Molecular Diagnosis of Hepatitis A Virus in Human Clinical Samples Using Multiplex PCR-Based Next-Generation Sequencing. Microorganisms. 2022;10(1):100. doi: 10.3390/microorganisms10010100

</cite> [<a href="https://doi.org/10.3390/microorganisms10010100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8779169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35056549/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=Genotyping%20and%20Molecular%20Diagnosis%20of%20Hepatitis%20A%20Virus%20in%20Human%20Clinical%20Samples%20Using%20Multiplex%20PCR-Based%20Next-Generation%20Sequencing&amp;author=G-Y%20Lee&amp;author=W-K%20Kim&amp;author=S%20Cho&amp;author=K%20Park&amp;author=J%20Kim&amp;volume=10&amp;issue=1&amp;publication_year=2022&amp;pages=100&amp;pmid=35056549&amp;doi=10.3390/microorganisms10010100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref017">
<span class="label">17.</span><cite>Lee S-H, No JS, Kim K, Budhathoki S, Park K, Lee G-Y, et al. Novel Paju Apodemus paramyxovirus 1 and 2, harbored by Apodemus agrarius in the Republic of Korea. Virology. 2021;562:40–9. doi: 10.1016/j.virol.2021.06.011

</cite> [<a href="https://doi.org/10.1016/j.virol.2021.06.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34256243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Virology&amp;title=Novel%20Paju%20Apodemus%20paramyxovirus%201%20and%202,%20harbored%20by%20Apodemus%20agrarius%20in%20the%20Republic%20of%20Korea&amp;author=S-H%20Lee&amp;author=JS%20No&amp;author=K%20Kim&amp;author=S%20Budhathoki&amp;author=K%20Park&amp;volume=562&amp;publication_year=2021&amp;pages=40-9&amp;pmid=34256243&amp;doi=10.1016/j.virol.2021.06.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref018">
<span class="label">18.</span><cite>Park K, Lee J, Kim J, Kim S-G, Song D-H, Gu SH, et al. Discovery and genomic characterization of Ulleung virus harbored by Crocidura utsuryoensis on Ulleung Island in Republic of Korea. Sci Rep. 2025;15(1):21748. doi: 10.1038/s41598-025-06845-6

</cite> [<a href="https://doi.org/10.1038/s41598-025-06845-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12215722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40593191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Discovery%20and%20genomic%20characterization%20of%20Ulleung%20virus%20harbored%20by%20Crocidura%20utsuryoensis%20on%20Ulleung%20Island%20in%20Republic%20of%20Korea&amp;author=K%20Park&amp;author=J%20Lee&amp;author=J%20Kim&amp;author=S-G%20Kim&amp;author=D-H%20Song&amp;volume=15&amp;issue=1&amp;publication_year=2025&amp;pages=21748&amp;pmid=40593191&amp;doi=10.1038/s41598-025-06845-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref019">
<span class="label">19.</span><cite>Smith GJD, Vijaykrishna D, Bahl J, Lycett SJ, Worobey M, Pybus OG, et al. Origins and evolutionary genomics of the 2009 swine-origin H1N1 influenza A epidemic. Nature. 2009;459(7250):1122–5. doi: 10.1038/nature08182

</cite> [<a href="https://doi.org/10.1038/nature08182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19516283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Origins%20and%20evolutionary%20genomics%20of%20the%202009%20swine-origin%20H1N1%20influenza%20A%20epidemic&amp;author=GJD%20Smith&amp;author=D%20Vijaykrishna&amp;author=J%20Bahl&amp;author=SJ%20Lycett&amp;author=M%20Worobey&amp;volume=459&amp;issue=7250&amp;publication_year=2009&amp;pages=1122-5&amp;pmid=19516283&amp;doi=10.1038/nature08182&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref020">
<span class="label">20.</span><cite>Gire SK, Goba A, Andersen KG, Sealfon RSG, Park DJ, Kanneh L, et al. Genomic surveillance elucidates Ebola virus origin and transmission during the 2014 outbreak. Science. 2014;345(6202):1369–72. doi: 10.1126/science.1259657

</cite> [<a href="https://doi.org/10.1126/science.1259657" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4431643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25214632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genomic%20surveillance%20elucidates%20Ebola%20virus%20origin%20and%20transmission%20during%20the%202014%20outbreak&amp;author=SK%20Gire&amp;author=A%20Goba&amp;author=KG%20Andersen&amp;author=RSG%20Sealfon&amp;author=DJ%20Park&amp;volume=345&amp;issue=6202&amp;publication_year=2014&amp;pages=1369-72&amp;pmid=25214632&amp;doi=10.1126/science.1259657&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref021">
<span class="label">21.</span><cite>Park K, Shin M, Natasha A, Kim J, Noh J, Kim S-G, et al. Novel human coronavirus in an infant patient with pneumonia, Republic of Korea. Emerg Microbes Infect. 2025;14(1):2466705. doi: 10.1080/22221751.2025.2466705

</cite> [<a href="https://doi.org/10.1080/22221751.2025.2466705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11849027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39945663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Emerg%20Microbes%20Infect&amp;title=Novel%20human%20coronavirus%20in%20an%20infant%20patient%20with%20pneumonia,%20Republic%20of%20Korea&amp;author=K%20Park&amp;author=M%20Shin&amp;author=A%20Natasha&amp;author=J%20Kim&amp;author=J%20Noh&amp;volume=14&amp;issue=1&amp;publication_year=2025&amp;pages=2466705&amp;pmid=39945663&amp;doi=10.1080/22221751.2025.2466705&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref022">
<span class="label">22.</span><cite>Lee G-Y, Park K, Lee Y-S, Kim JH, Byun KS, Kim J, et al. Molecular diagnosis of patients with hepatitis A virus infection using amplicon-based nanopore sequencing. PLoS One. 2023;18(7):e0288361. doi: 10.1371/journal.pone.0288361

</cite> [<a href="https://doi.org/10.1371/journal.pone.0288361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10337952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37437048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Molecular%20diagnosis%20of%20patients%20with%20hepatitis%20A%20virus%20infection%20using%20amplicon-based%20nanopore%20sequencing&amp;author=G-Y%20Lee&amp;author=K%20Park&amp;author=Y-S%20Lee&amp;author=JH%20Kim&amp;author=KS%20Byun&amp;volume=18&amp;issue=7&amp;publication_year=2023&amp;pmid=37437048&amp;doi=10.1371/journal.pone.0288361&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref023">
<span class="label">23.</span><cite>Lee J, Park K, Kim J, Lee S-H, Lee G-Y, Cho S, et al. Whole-genome sequencing and genetic diversity of severe fever with thrombocytopenia syndrome virus using multiplex PCR-based nanopore sequencing, Republic of Korea. PLoS Negl Trop Dis. 2022;16(9):e0010763. doi: 10.1371/journal.pntd.0010763

</cite> [<a href="https://doi.org/10.1371/journal.pntd.0010763" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9499217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36094957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Negl%20Trop%20Dis&amp;title=Whole-genome%20sequencing%20and%20genetic%20diversity%20of%20severe%20fever%20with%20thrombocytopenia%20syndrome%20virus%20using%20multiplex%20PCR-based%20nanopore%20sequencing,%20Republic%20of%20Korea&amp;author=J%20Lee&amp;author=K%20Park&amp;author=J%20Kim&amp;author=S-H%20Lee&amp;author=G-Y%20Lee&amp;volume=16&amp;issue=9&amp;publication_year=2022&amp;pmid=36094957&amp;doi=10.1371/journal.pntd.0010763&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref024">
<span class="label">24.</span><cite>Taylor MK, Williams EP, Wongsurawat T, Jenjaroenpun P, Nookaew I, Jonsson CB. Amplicon-Based, Next-Generation Sequencing Approaches to Characterize Single Nucleotide Polymorphisms of Orthohantavirus Species. Front Cell Infect Microbiol. 2020;10:565591. doi: 10.3389/fcimb.2020.565591

</cite> [<a href="https://doi.org/10.3389/fcimb.2020.565591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7591466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33163416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Infect%20Microbiol&amp;title=Amplicon-Based,%20Next-Generation%20Sequencing%20Approaches%20to%20Characterize%20Single%20Nucleotide%20Polymorphisms%20of%20Orthohantavirus%20Species&amp;author=MK%20Taylor&amp;author=EP%20Williams&amp;author=T%20Wongsurawat&amp;author=P%20Jenjaroenpun&amp;author=I%20Nookaew&amp;volume=10&amp;publication_year=2020&amp;pages=565591&amp;pmid=33163416&amp;doi=10.3389/fcimb.2020.565591&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref025">
<span class="label">25.</span><cite>Park K, Lee S-H, Kim J, Lee J, Lee G-Y, Cho S, et al. Multiplex PCR-Based Nanopore Sequencing and Epidemiological Surveillance of Hantaan orthohantavirus in Apodemus agrarius, Republic of Korea. Viruses. 2021;13(5):847. doi: 10.3390/v13050847

</cite> [<a href="https://doi.org/10.3390/v13050847" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8148566/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34066592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=Multiplex%20PCR-Based%20Nanopore%20Sequencing%20and%20Epidemiological%20Surveillance%20of%20Hantaan%20orthohantavirus%20in%20Apodemus%20agrarius,%20Republic%20of%20Korea&amp;author=K%20Park&amp;author=S-H%20Lee&amp;author=J%20Kim&amp;author=J%20Lee&amp;author=G-Y%20Lee&amp;volume=13&amp;issue=5&amp;publication_year=2021&amp;pages=847&amp;pmid=34066592&amp;doi=10.3390/v13050847&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref026">
<span class="label">26.</span><cite>Park K, Noh J, Kim K, Kim J, Cho H-K, Kim S-G, et al. A Development of Rapid Whole-Genome Sequencing of Seoul orthohantavirus Using a Portable One-Step Amplicon-Based High Accuracy Nanopore System. Viruses. 2023;15(7):1542. doi: 10.3390/v15071542

</cite> [<a href="https://doi.org/10.3390/v15071542" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10386077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37515228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Viruses&amp;title=A%20Development%20of%20Rapid%20Whole-Genome%20Sequencing%20of%20Seoul%20orthohantavirus%20Using%20a%20Portable%20One-Step%20Amplicon-Based%20High%20Accuracy%20Nanopore%20System&amp;author=K%20Park&amp;author=J%20Noh&amp;author=K%20Kim&amp;author=J%20Kim&amp;author=H-K%20Cho&amp;volume=15&amp;issue=7&amp;publication_year=2023&amp;pages=1542&amp;pmid=37515228&amp;doi=10.3390/v15071542&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref027">
<span class="label">27.</span><cite>Prayitno SP, Natasha A, Lee S, Kim C-M, Lee YM, Park K, et al. Etiological agent and clinical characteristics of haemorrhagic fever with renal syndrome in the southern Republic of Korea: a genomic surveillance study. Clin Microbiol Infect. 2024;30(6):795–802. doi: 10.1016/j.cmi.2024.02.014

</cite> [<a href="https://doi.org/10.1016/j.cmi.2024.02.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38402954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Microbiol%20Infect&amp;title=Etiological%20agent%20and%20clinical%20characteristics%20of%20haemorrhagic%20fever%20with%20renal%20syndrome%20in%20the%20southern%20Republic%20of%20Korea:%20a%20genomic%20surveillance%20study&amp;author=SP%20Prayitno&amp;author=A%20Natasha&amp;author=S%20Lee&amp;author=C-M%20Kim&amp;author=YM%20Lee&amp;volume=30&amp;issue=6&amp;publication_year=2024&amp;pages=795-802&amp;pmid=38402954&amp;doi=10.1016/j.cmi.2024.02.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref028">
<span class="label">28.</span><cite>Kim J, Park K, Kim K, Noh J, Kim S-G, Yang E, et al. High-resolution phylogeographical surveillance of Hantaan orthohantavirus using rapid amplicon-based Flongle sequencing, Republic of Korea. J Med Virol. 2024;96(1):e29346. doi: 10.1002/jmv.29346

</cite> [<a href="https://doi.org/10.1002/jmv.29346" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38178580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Med%20Virol&amp;title=High-resolution%20phylogeographical%20surveillance%20of%20Hantaan%20orthohantavirus%20using%20rapid%20amplicon-based%20Flongle%20sequencing,%20Republic%20of%20Korea&amp;author=J%20Kim&amp;author=K%20Park&amp;author=K%20Kim&amp;author=J%20Noh&amp;author=S-G%20Kim&amp;volume=96&amp;issue=1&amp;publication_year=2024&amp;pmid=38178580&amp;doi=10.1002/jmv.29346&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref029">
<span class="label">29.</span><cite>Park K, Kim J, Noh J, Kim S-G, Cho H-K, Kim K, et al. Epidemiological surveillance and phylogenetic diversity of Orthohantavirus hantanense using high-fidelity nanopore sequencing, Republic of Korea. PLoS Negl Trop Dis. 2025;19(2):e0012859. doi: 10.1371/journal.pntd.0012859

</cite> [<a href="https://doi.org/10.1371/journal.pntd.0012859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11828426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39919119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Negl%20Trop%20Dis&amp;title=Epidemiological%20surveillance%20and%20phylogenetic%20diversity%20of%20Orthohantavirus%20hantanense%20using%20high-fidelity%20nanopore%20sequencing,%20Republic%20of%20Korea&amp;author=K%20Park&amp;author=J%20Kim&amp;author=J%20Noh&amp;author=S-G%20Kim&amp;author=H-K%20Cho&amp;volume=19&amp;issue=2&amp;publication_year=2025&amp;pmid=39919119&amp;doi=10.1371/journal.pntd.0012859&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref030">
<span class="label">30.</span><cite>Hadfield J, Megill C, Bell SM, Huddleston J, Potter B, Callender C, et al. Nextstrain: real-time tracking of pathogen evolution. Bioinformatics. 2018;34(23):4121–3. doi: 10.1093/bioinformatics/bty407

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6247931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29790939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Nextstrain:%20real-time%20tracking%20of%20pathogen%20evolution&amp;author=J%20Hadfield&amp;author=C%20Megill&amp;author=SM%20Bell&amp;author=J%20Huddleston&amp;author=B%20Potter&amp;volume=34&amp;issue=23&amp;publication_year=2018&amp;pages=4121-3&amp;pmid=29790939&amp;doi=10.1093/bioinformatics/bty407&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref031">
<span class="label">31.</span><cite>Park K, Kim K, Kim J, Noh J, Kim S-G, Cho H-K, et al. Web Visualization for Spatiotemporal Genomic Epidemiology, Annotation, and Mutation Dynamics of Orthohantavirus hantanense Using Nextstrain. Am J Trop Med Hyg. 2025;112(4):871–4. doi: 10.4269/ajtmh.24-0432

</cite> [<a href="https://doi.org/10.4269/ajtmh.24-0432" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11965749/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39874598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20J%20Trop%20Med%20Hyg&amp;title=Web%20Visualization%20for%20Spatiotemporal%20Genomic%20Epidemiology,%20Annotation,%20and%20Mutation%20Dynamics%20of%20Orthohantavirus%20hantanense%20Using%20Nextstrain&amp;author=K%20Park&amp;author=K%20Kim&amp;author=J%20Kim&amp;author=J%20Noh&amp;author=S-G%20Kim&amp;volume=112&amp;issue=4&amp;publication_year=2025&amp;pages=871-4&amp;pmid=39874598&amp;doi=10.4269/ajtmh.24-0432&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref032">
<span class="label">32.</span><cite>Baele G, Dellicour S, Suchard MA, Lemey P, Vrancken B. Recent advances in computational phylodynamics. Curr Opin Virol. 2018;31:24–32. doi: 10.1016/j.coviro.2018.08.009

</cite> [<a href="https://doi.org/10.1016/j.coviro.2018.08.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30248578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Virol&amp;title=Recent%20advances%20in%20computational%20phylodynamics&amp;author=G%20Baele&amp;author=S%20Dellicour&amp;author=MA%20Suchard&amp;author=P%20Lemey&amp;author=B%20Vrancken&amp;volume=31&amp;publication_year=2018&amp;pages=24-32&amp;pmid=30248578&amp;doi=10.1016/j.coviro.2018.08.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref033">
<span class="label">33.</span><cite>Saxena SK, Kumar S, Ansari S, Paweska JT, Maurya VK, Tripathi AK, et al. Characterization of the novel SARS-CoV-2 Omicron (B.1.1.529) variant of concern and its global perspective. J Med Virol. 2022;94(4):1738–44. doi: 10.1002/jmv.27524

</cite> [<a href="https://doi.org/10.1002/jmv.27524" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34905235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Med%20Virol&amp;title=Characterization%20of%20the%20novel%20SARS-CoV-2%20Omicron%20(B.1.1.529)%20variant%20of%20concern%20and%20its%20global%20perspective&amp;author=SK%20Saxena&amp;author=S%20Kumar&amp;author=S%20Ansari&amp;author=JT%20Paweska&amp;author=VK%20Maurya&amp;volume=94&amp;issue=4&amp;publication_year=2022&amp;pages=1738-44&amp;pmid=34905235&amp;doi=10.1002/jmv.27524&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref034">
<span class="label">34.</span><cite>Carroll MW, Matthews DA, Hiscox JA, Elmore MJ, Pollakis G, Rambaut A, et al. Temporal and spatial analysis of the 2014-2015 Ebola virus outbreak in West Africa. Nature. 2015;524(7563):97–101. doi: 10.1038/nature14594

</cite> [<a href="https://doi.org/10.1038/nature14594" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10601607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26083749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Temporal%20and%20spatial%20analysis%20of%20the%202014-2015%20Ebola%20virus%20outbreak%20in%20West%20Africa&amp;author=MW%20Carroll&amp;author=DA%20Matthews&amp;author=JA%20Hiscox&amp;author=MJ%20Elmore&amp;author=G%20Pollakis&amp;volume=524&amp;issue=7563&amp;publication_year=2015&amp;pages=97-101&amp;pmid=26083749&amp;doi=10.1038/nature14594&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref035">
<span class="label">35.</span><cite>Andersen KG, Shapiro BJ, Matranga CB, Sealfon R, Lin AE, Moses LM, et al. Clinical Sequencing Uncovers Origins and Evolution of Lassa Virus. Cell. 2015;162(4):738–50. doi: 10.1016/j.cell.2015.07.020

</cite> [<a href="https://doi.org/10.1016/j.cell.2015.07.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4537774/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26276630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Clinical%20Sequencing%20Uncovers%20Origins%20and%20Evolution%20of%20Lassa%20Virus&amp;author=KG%20Andersen&amp;author=BJ%20Shapiro&amp;author=CB%20Matranga&amp;author=R%20Sealfon&amp;author=AE%20Lin&amp;volume=162&amp;issue=4&amp;publication_year=2015&amp;pages=738-50&amp;pmid=26276630&amp;doi=10.1016/j.cell.2015.07.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref036">
<span class="label">36.</span><cite>Grenfell BT, Pybus OG, Gog JR, Wood JLN, Daly JM, Mumford JA, et al. Unifying the epidemiological and evolutionary dynamics of pathogens. Science. 2004;303(5656):327–32. doi: 10.1126/science.1090727

</cite> [<a href="https://doi.org/10.1126/science.1090727" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14726583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Unifying%20the%20epidemiological%20and%20evolutionary%20dynamics%20of%20pathogens&amp;author=BT%20Grenfell&amp;author=OG%20Pybus&amp;author=JR%20Gog&amp;author=JLN%20Wood&amp;author=JM%20Daly&amp;volume=303&amp;issue=5656&amp;publication_year=2004&amp;pages=327-32&amp;pmid=14726583&amp;doi=10.1126/science.1090727&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref037">
<span class="label">37.</span><cite>Park K, Kim SG, Kim JA, Prayitno SP, Kim J, Lee SH. Molecular evolution and zoonotic potential of Muju virus (Orthohantavirus puumalaense) in Craseomys regulus, Republic of Korea. Journal of Medical Virology. 2025;97(7):e70503.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Medical%20Virology&amp;title=Molecular%20evolution%20and%20zoonotic%20potential%20of%20Muju%20virus%20(Orthohantavirus%20puumalaense)%20in%20Craseomys%20regulus,%20Republic%20of%20Korea&amp;author=K%20Park&amp;author=SG%20Kim&amp;author=JA%20Kim&amp;author=SP%20Prayitno&amp;author=J%20Kim&amp;volume=97&amp;issue=7&amp;publication_year=2025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref038">
<span class="label">38.</span><cite>Klein TA, Kim H-C, Chong S-T, Kim J-A, Lee S-Y, Kim W-K, et al. Hantaan virus surveillance targeting small mammals at nightmare range, a high elevation military training area, Gyeonggi Province, Republic of Korea. PLoS One. 2015;10(4):e0118483. doi: 10.1371/journal.pone.0118483

</cite> [<a href="https://doi.org/10.1371/journal.pone.0118483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4398386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25874643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Hantaan%20virus%20surveillance%20targeting%20small%20mammals%20at%20nightmare%20range,%20a%20high%20elevation%20military%20training%20area,%20Gyeonggi%20Province,%20Republic%20of%20Korea&amp;author=TA%20Klein&amp;author=H-C%20Kim&amp;author=S-T%20Chong&amp;author=J-A%20Kim&amp;author=S-Y%20Lee&amp;volume=10&amp;issue=4&amp;publication_year=2015&amp;pmid=25874643&amp;doi=10.1371/journal.pone.0118483&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref039">
<span class="label">39.</span><cite>Irwin DM, Kocher TD, Wilson AC. Evolution of the cytochrome b gene of mammals. J Mol Evol. 1991;32(2):128–44. doi: 10.1007/BF02515385

</cite> [<a href="https://doi.org/10.1007/BF02515385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1901092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Evol&amp;title=Evolution%20of%20the%20cytochrome%20b%20gene%20of%20mammals&amp;author=DM%20Irwin&amp;author=TD%20Kocher&amp;author=AC%20Wilson&amp;volume=32&amp;issue=2&amp;publication_year=1991&amp;pages=128-44&amp;pmid=1901092&amp;doi=10.1007/BF02515385&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref040">
<span class="label">40.</span><cite>No JS, Kim W-K, Cho S, Lee S-H, Kim J-A, Lee D, et al. Comparison of targeted next-generation sequencing for whole-genome sequencing of Hantaan orthohantavirus in Apodemus agrarius lung tissues. Sci Rep. 2019;9(1):16631. doi: 10.1038/s41598-019-53043-2

</cite> [<a href="https://doi.org/10.1038/s41598-019-53043-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6851128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31719616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Comparison%20of%20targeted%20next-generation%20sequencing%20for%20whole-genome%20sequencing%20of%20Hantaan%20orthohantavirus%20in%20Apodemus%20agrarius%20lung%20tissues&amp;author=JS%20No&amp;author=W-K%20Kim&amp;author=S%20Cho&amp;author=S-H%20Lee&amp;author=J-A%20Kim&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=16631&amp;pmid=31719616&amp;doi=10.1038/s41598-019-53043-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref041">
<span class="label">41.</span><cite>Drummond AJ, Rambaut A. BEAST: Bayesian evolutionary analysis by sampling trees. BMC Evol Biol. 2007;7:214. doi: 10.1186/1471-2148-7-214

</cite> [<a href="https://doi.org/10.1186/1471-2148-7-214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2247476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17996036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evol%20Biol&amp;title=BEAST:%20Bayesian%20evolutionary%20analysis%20by%20sampling%20trees&amp;author=AJ%20Drummond&amp;author=A%20Rambaut&amp;volume=7&amp;publication_year=2007&amp;pages=214&amp;pmid=17996036&amp;doi=10.1186/1471-2148-7-214&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref042">
<span class="label">42.</span><cite>Kim W-K, Cho S, Lee S-H, No JS, Lee G-Y, Park K, et al. Genomic Epidemiology and Active Surveillance to Investigate Outbreaks of Hantaviruses. Front Cell Infect Microbiol. 2021;10:532388. doi: 10.3389/fcimb.2020.532388

</cite> [<a href="https://doi.org/10.3389/fcimb.2020.532388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7819890/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33489927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Infect%20Microbiol&amp;title=Genomic%20Epidemiology%20and%20Active%20Surveillance%20to%20Investigate%20Outbreaks%20of%20Hantaviruses&amp;author=W-K%20Kim&amp;author=S%20Cho&amp;author=S-H%20Lee&amp;author=JS%20No&amp;author=G-Y%20Lee&amp;volume=10&amp;publication_year=2021&amp;pages=532388&amp;pmid=33489927&amp;doi=10.3389/fcimb.2020.532388&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref043">
<span class="label">43.</span><cite>Song JW, Moon SS, Gu SH, Song KJ, Baek LJ, Kim HC, et al. Hemorrhagic fever with renal syndrome in 4 US soldiers, South Korea, 2005. Emerg Infect Dis. 2009;15(11):1833–6. doi: 10.3201/eid1511.090076

</cite> [<a href="https://doi.org/10.3201/eid1511.090076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2857219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19891878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Emerg%20Infect%20Dis&amp;title=Hemorrhagic%20fever%20with%20renal%20syndrome%20in%204%20US%20soldiers,%20South%20Korea,%202005&amp;author=JW%20Song&amp;author=SS%20Moon&amp;author=SH%20Gu&amp;author=KJ%20Song&amp;author=LJ%20Baek&amp;volume=15&amp;issue=11&amp;publication_year=2009&amp;pages=1833-6&amp;pmid=19891878&amp;doi=10.3201/eid1511.090076&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref044">
<span class="label">44.</span><cite>Kim W-K, Kim J-A, Song DH, Lee D, Kim YC, Lee S-Y, et al. Phylogeographic analysis of hemorrhagic fever with renal syndrome patients using multiplex PCR-based next generation sequencing. Sci Rep. 2016;6:26017. doi: 10.1038/srep26017

</cite> [<a href="https://doi.org/10.1038/srep26017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4879520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27221218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Phylogeographic%20analysis%20of%20hemorrhagic%20fever%20with%20renal%20syndrome%20patients%20using%20multiplex%20PCR-based%20next%20generation%20sequencing&amp;author=W-K%20Kim&amp;author=J-A%20Kim&amp;author=DH%20Song&amp;author=D%20Lee&amp;author=YC%20Kim&amp;volume=6&amp;publication_year=2016&amp;pages=26017&amp;pmid=27221218&amp;doi=10.1038/srep26017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref045">
<span class="label">45.</span><cite>Kim W-K, No JS, Lee D, Jung J, Park H, Yi Y, et al. Active Targeted Surveillance to Identify Sites of Emergence of Hantavirus. Clin Infect Dis. 2020;70(3):464–73. doi: 10.1093/cid/ciz234

</cite> [<a href="https://doi.org/10.1093/cid/ciz234" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30891596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&amp;title=Active%20Targeted%20Surveillance%20to%20Identify%20Sites%20of%20Emergence%20of%20Hantavirus&amp;author=W-K%20Kim&amp;author=JS%20No&amp;author=D%20Lee&amp;author=J%20Jung&amp;author=H%20Park&amp;volume=70&amp;issue=3&amp;publication_year=2020&amp;pages=464-73&amp;pmid=30891596&amp;doi=10.1093/cid/ciz234&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref046">
<span class="label">46.</span><cite>Lim CM, Heo ST, Kim J, Yu JR, Kim YR, Lee KH. Case of Hemorrhagic Fever with Renal Syndrome in Jeju Island. Infect Chemother. 2012;44(6):454. doi: 10.3947/ic.2012.44.6.454</cite> [<a href="https://doi.org/10.3947/ic.2012.44.6.454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect%20Chemother&amp;title=Case%20of%20Hemorrhagic%20Fever%20with%20Renal%20Syndrome%20in%20Jeju%20Island&amp;author=CM%20Lim&amp;author=ST%20Heo&amp;author=J%20Kim&amp;author=JR%20Yu&amp;author=YR%20Kim&amp;volume=44&amp;issue=6&amp;publication_year=2012&amp;pages=454&amp;doi=10.3947/ic.2012.44.6.454&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref047">
<span class="label">47.</span><cite>Cho S, Kim W-K, No JS, Lee S-H, Jung J, Yi Y, et al. Urinary genome detection and tracking of Hantaan virus from hemorrhagic fever with renal syndrome patients using multiplex PCR-based next-generation sequencing. PLoS Negl Trop Dis. 2021;15(9):e0009707. doi: 10.1371/journal.pntd.0009707

</cite> [<a href="https://doi.org/10.1371/journal.pntd.0009707" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8478167/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34582439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Negl%20Trop%20Dis&amp;title=Urinary%20genome%20detection%20and%20tracking%20of%20Hantaan%20virus%20from%20hemorrhagic%20fever%20with%20renal%20syndrome%20patients%20using%20multiplex%20PCR-based%20next-generation%20sequencing&amp;author=S%20Cho&amp;author=W-K%20Kim&amp;author=JS%20No&amp;author=S-H%20Lee&amp;author=J%20Jung&amp;volume=15&amp;issue=9&amp;publication_year=2021&amp;pmid=34582439&amp;doi=10.1371/journal.pntd.0009707&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref048">
<span class="label">48.</span><cite>Lee S-H, Kim W-K, No JS, Kim J-A, Kim JI, Gu SH, et al. Dynamic Circulation and Genetic Exchange of a Shrew-borne Hantavirus, Imjin virus, in the Republic of Korea. Sci Rep. 2017;7:44369. doi: 10.1038/srep44369

</cite> [<a href="https://doi.org/10.1038/srep44369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5353647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28295052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Dynamic%20Circulation%20and%20Genetic%20Exchange%20of%20a%20Shrew-borne%20Hantavirus,%20Imjin%20virus,%20in%20the%20Republic%20of%20Korea&amp;author=S-H%20Lee&amp;author=W-K%20Kim&amp;author=JS%20No&amp;author=J-A%20Kim&amp;author=JI%20Kim&amp;volume=7&amp;publication_year=2017&amp;pages=44369&amp;pmid=28295052&amp;doi=10.1038/srep44369&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref049">
<span class="label">49.</span><cite>Kim J-A, Kim W-K, No JS, Lee S-H, Lee S-Y, Kim JH, et al. Genetic Diversity and Reassortment of Hantaan Virus Tripartite RNA Genomes in Nature, the Republic of Korea. PLoS Negl Trop Dis. 2016;10(6):e0004650. doi: 10.1371/journal.pntd.0004650

</cite> [<a href="https://doi.org/10.1371/journal.pntd.0004650" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4912082/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27315053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Negl%20Trop%20Dis&amp;title=Genetic%20Diversity%20and%20Reassortment%20of%20Hantaan%20Virus%20Tripartite%20RNA%20Genomes%20in%20Nature,%20the%20Republic%20of%20Korea&amp;author=J-A%20Kim&amp;author=W-K%20Kim&amp;author=JS%20No&amp;author=S-H%20Lee&amp;author=S-Y%20Lee&amp;volume=10&amp;issue=6&amp;publication_year=2016&amp;pmid=27315053&amp;doi=10.1371/journal.pntd.0004650&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref050">
<span class="label">50.</span><cite>Lee G-Y, Kim W-K, Park K, Lee S-H, Hwang J, No JS, et al. Phylogeographic diversity and hybrid zone of Hantaan orthohantavirus collected in Gangwon Province, Republic of Korea. PLoS Negl Trop Dis. 2020;14(10):e0008714. doi: 10.1371/journal.pntd.0008714

</cite> [<a href="https://doi.org/10.1371/journal.pntd.0008714" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7588125/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33035222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Negl%20Trop%20Dis&amp;title=Phylogeographic%20diversity%20and%20hybrid%20zone%20of%20Hantaan%20orthohantavirus%20collected%20in%20Gangwon%20Province,%20Republic%20of%20Korea&amp;author=G-Y%20Lee&amp;author=W-K%20Kim&amp;author=K%20Park&amp;author=S-H%20Lee&amp;author=J%20Hwang&amp;volume=14&amp;issue=10&amp;publication_year=2020&amp;pmid=33035222&amp;doi=10.1371/journal.pntd.0008714&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref051">
<span class="label">51.</span><cite>Park K, Kim J, Kim S-G, Kim W-K, Song J-W. Molecular evolution and reassortment dynamics of Orthohantavirus hantanense revealed through longitudinal genomic surveillance in the Republic of Korea. Sci Rep. 2025;15(1):24672. doi: 10.1038/s41598-025-11003-z

</cite> [<a href="https://doi.org/10.1038/s41598-025-11003-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12241637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40634677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Molecular%20evolution%20and%20reassortment%20dynamics%20of%20Orthohantavirus%20hantanense%20revealed%20through%20longitudinal%20genomic%20surveillance%20in%20the%20Republic%20of%20Korea&amp;author=K%20Park&amp;author=J%20Kim&amp;author=S-G%20Kim&amp;author=W-K%20Kim&amp;author=J-W%20Song&amp;volume=15&amp;issue=1&amp;publication_year=2025&amp;pages=24672&amp;pmid=40634677&amp;doi=10.1038/s41598-025-11003-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref052">
<span class="label">52.</span><cite>Klempa B, Avsic-Zupanc T, Clement J, Dzagurova TK, Henttonen H, Heyman P, et al. Complex evolution and epidemiology of Dobrava-Belgrade hantavirus: definition of genotypes and their characteristics. Arch Virol. 2013;158(3):521–9. doi: 10.1007/s00705-012-1514-5

</cite> [<a href="https://doi.org/10.1007/s00705-012-1514-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3586401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23090188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arch%20Virol&amp;title=Complex%20evolution%20and%20epidemiology%20of%20Dobrava-Belgrade%20hantavirus:%20definition%20of%20genotypes%20and%20their%20characteristics&amp;author=B%20Klempa&amp;author=T%20Avsic-Zupanc&amp;author=J%20Clement&amp;author=TK%20Dzagurova&amp;author=H%20Henttonen&amp;volume=158&amp;issue=3&amp;publication_year=2013&amp;pages=521-9&amp;pmid=23090188&amp;doi=10.1007/s00705-012-1514-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref053">
<span class="label">53.</span><cite>Avsic-Zupanc T, Petrovec M, Furlan P, Kaps R, Elgh F, Lundkvist A. Hemorrhagic fever with renal syndrome in the Dolenjska region of Slovenia--a 10-year survey. Clin Infect Dis. 1999;28(4):860–5. doi: 10.1086/515185

</cite> [<a href="https://doi.org/10.1086/515185" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10825051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&amp;title=Hemorrhagic%20fever%20with%20renal%20syndrome%20in%20the%20Dolenjska%20region%20of%20Slovenia--a%2010-year%20survey&amp;author=T%20Avsic-Zupanc&amp;author=M%20Petrovec&amp;author=P%20Furlan&amp;author=R%20Kaps&amp;author=F%20Elgh&amp;volume=28&amp;issue=4&amp;publication_year=1999&amp;pages=860-5&amp;pmid=10825051&amp;doi=10.1086/515185&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref054">
<span class="label">54.</span><cite>Klempa B, Tkachenko EA, Dzagurova TK, Yunicheva YV, Morozov VG, Okulova NM, et al. Hemorrhagic fever with renal syndrome caused by 2 lineages of Dobrava hantavirus, Russia. Emerg Infect Dis. 2008;14(4):617–25. doi: 10.3201/eid1404.071310

</cite> [<a href="https://doi.org/10.3201/eid1404.071310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2570905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18394280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Emerg%20Infect%20Dis&amp;title=Hemorrhagic%20fever%20with%20renal%20syndrome%20caused%20by%202%20lineages%20of%20Dobrava%20hantavirus,%20Russia&amp;author=B%20Klempa&amp;author=EA%20Tkachenko&amp;author=TK%20Dzagurova&amp;author=YV%20Yunicheva&amp;author=VG%20Morozov&amp;volume=14&amp;issue=4&amp;publication_year=2008&amp;pages=617-25&amp;pmid=18394280&amp;doi=10.3201/eid1404.071310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pntd.0013459.ref055">
<span class="label">55.</span><cite>Golovljova I, Vasilenko V, Mittzenkov V, Prükk T, Seppet E, Vene S, et al. Characterization of hemorrhagic fever with renal syndrome caused by hantaviruses, Estonia. Emerg Infect Dis. 2007;13(11):1773–6. doi: 10.3201/eid1311.070221

</cite> [<a href="https://doi.org/10.3201/eid1311.070221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3375801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18217569/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Emerg%20Infect%20Dis&amp;title=Characterization%20of%20hemorrhagic%20fever%20with%20renal%20syndrome%20caused%20by%20hantaviruses,%20Estonia&amp;author=I%20Golovljova&amp;author=V%20Vasilenko&amp;author=V%20Mittzenkov&amp;author=T%20Pr%C3%BCkk&amp;author=E%20Seppet&amp;volume=13&amp;issue=11&amp;publication_year=2007&amp;pages=1773-6&amp;pmid=18217569&amp;doi=10.3201/eid1311.070221&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Fig. Phylogeny based on the mitochondrial DNA cytochrome <em>b</em> (CYTB) gene of <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea (ROK), in 2022–2023.</span><p>A phylogenetic tree generated from the mitochondrial DNA CYTB gene sequences (686 bps) of <em>A. chejuensis</em> captured on Jeju Island, ROK. A phylogenetic analysis was conducted with the HKY + G substitution model using the maximum likelihood method in MEGA 7.0. The branch lengths in the phylogenetic tree correspond to the number of nucleotide substitutions, while the vertical distances are adjusted for improved visual clarity. Bootstrap probabilities, calculated from 1,000 iterations, are indicated at each node. In this figure, the genomic sequences of the CYTB gene from <em>A. chejuensis</em> are displayed, with the newly obtained sequences highlighted in bold red font.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s001.tif" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s001.tif</a><sup> (3.9MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S2 Fig. Amino acid substitutions in the tripartite genomes of Hantaan virus (HTNV) carried by <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea, in 2022–2023.</span><p>The lollipop plot illustrates the positions of amino acid mutations in the nine HTNV sequences obtained in this study, compared to the reference HTNV Ac20–5 genomic sequences. The plot was visualized using the <em>dplyr</em> and <em>ggplot2</em> packages in R version 4.3.3. Each node represents a point of amino acid variations, with the node length indicating the frequency of HTNV amino acid substitutions across different rodent collection sites. The color-coded bars correspond to the tripartite segments: S segment (blue), M segment (green), and L segment (red). Red nodes highlight key amino acid changes uniquely found in sequences from Seogwipo-si. The scale bar represents the length of HTNV amino acids and nucleotides.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s002.tif" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s002.tif</a><sup> (4MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S1 Table. Species composition of 50 small mammals captured on Jeju Island, Republic of Korea, in 2022–2023.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s003.pdf</a><sup> (320KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S2 Table. Characteristics of Hantaan virus (HTNV)-infected <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea, in 2022–2023.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s004.pdf</a><sup> (319.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S3 Table. Hantaan virus RNA copy numbers from the lung tissues of <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s005.pdf</a><sup> (319.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>S4 Table. Summary of mapped reads and sequencing depth of amplicon-based nanopore sequencing for Hantaan virus harbored by <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea, in 2022–2023.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s006.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s006.pdf</a><sup> (320KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>S5 Table. Nucleotide and amino acid sequence similarities of Hantaan virus (HTNV) carried by <em>Apodemus chejuensis</em> collected on Jeju Island, Republic of Korea, in 2022–2023.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s007.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s007.pdf</a><sup> (319.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>S6 Table. Accession numbers of genomic sequences of Hantaan virus (HTNV) S, M, and L segments in the current study.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373272/bin/pntd.0013459.s008.pdf" data-ga-action="click_feat_suppl" class="usa-link">pntd.0013459.s008.pdf</a><sup> (319.4KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data presented and supporting information in this study are available from GenBank (accession numbers: PQ410874–PQ410913).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS Neglected Tropical Diseases are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pntd.0013459"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pntd.0013459.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373272/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373272/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373272%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373272/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373272/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373272/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40828876/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373272/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40828876/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373272/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373272/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="tdpx0q1LUgNDljZ44MUmfN8zA7aZwY44iyiVhieEUuTyWqdGy8rRWSE2gPHl7suU">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
