
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Promiscuous and multivalent interactions between Eps15 and partner protein Dab2 generate a complex interaction network - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A4DC8AF229F305A4DC002548EA45.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370899/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Promiscuous and multivalent interactions between Eps15 and partner protein Dab2 generate a complex interaction network">
<meta name="citation_author" content="Andromachi Papagiannoula">
<meta name="citation_author_institution" content="Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, Berlin, Germany">
<meta name="citation_author_institution" content="Freie Universität Berlin, Department of Biology, Chemistry, Pharmacy, Berlin, Germany">
<meta name="citation_author" content="Ida Marie Vedel">
<meta name="citation_author_institution" content="Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, Berlin, Germany">
<meta name="citation_author" content="Kathrin Motzny">
<meta name="citation_author_institution" content="Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, Berlin, Germany">
<meta name="citation_author" content="Maud Tengo">
<meta name="citation_author_institution" content="Université Grenoble Alpes, CNRS, CEA, IBS, Grenoble, France">
<meta name="citation_author" content="Arbesa Saiti">
<meta name="citation_author_institution" content="Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, Berlin, Germany">
<meta name="citation_author" content="Sigrid Milles">
<meta name="citation_author_institution" content="Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, Berlin, Germany">
<meta name="citation_author_institution" content="Université Grenoble Alpes, CNRS, CEA, IBS, Grenoble, France">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7783">
<meta name="citation_doi" content="10.1038/s41467-025-63090-1">
<meta name="citation_pmid" content="40841375">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370899/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370899/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370899/pdf/41467_2025_Article_63090.pdf">
<meta name="description" content="Clathrin-mediated endocytosis depends on complex protein interactions. Eps15 plays a key role through interactions of its three EH domains with Asn-Pro-Phe (NPF) motifs in intrinsically disordered regions (IDRs) of other endocytic proteins. Using ...">
<meta name="og:title" content="Promiscuous and multivalent interactions between Eps15 and partner protein Dab2 generate a complex interaction network">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Clathrin-mediated endocytosis depends on complex protein interactions. Eps15 plays a key role through interactions of its three EH domains with Asn-Pro-Phe (NPF) motifs in intrinsically disordered regions (IDRs) of other endocytic proteins. Using ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370899/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370899">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63090-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63090.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370899%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370899/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370899/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370899/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7783. doi: <a href="https://doi.org/10.1038/s41467-025-63090-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63090-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Promiscuous and multivalent interactions between Eps15 and partner protein Dab2 generate a complex interaction network</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Papagiannoula%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Andromachi Papagiannoula</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Andromachi Papagiannoula</span></h3>
<div class="p">
<sup>1</sup>Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, Berlin, Germany </div>
<div class="p">
<sup>2</sup>Freie Universität Berlin, Department of Biology, Chemistry, Pharmacy, Berlin, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Papagiannoula%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andromachi Papagiannoula</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vedel%20IM%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ida Marie Vedel</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ida Marie Vedel</span></h3>
<div class="p">
<sup>1</sup>Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, Berlin, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vedel%20IM%22%5BAuthor%5D" class="usa-link"><span class="name western">Ida Marie Vedel</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Motzny%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Kathrin Motzny</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Kathrin Motzny</span></h3>
<div class="p">
<sup>1</sup>Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, Berlin, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Motzny%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kathrin Motzny</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tengo%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Maud Tengo</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Maud Tengo</span></h3>
<div class="p">
<sup>3</sup>Université Grenoble Alpes, CNRS, CEA, IBS, Grenoble, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tengo%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maud Tengo</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saiti%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Arbesa Saiti</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Arbesa Saiti</span></h3>
<div class="p">
<sup>1</sup>Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, Berlin, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saiti%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arbesa Saiti</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Milles%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Sigrid Milles</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Sigrid Milles</span></h3>
<div class="p">
<sup>1</sup>Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, Berlin, Germany </div>
<div class="p">
<sup>3</sup>Université Grenoble Alpes, CNRS, CEA, IBS, Grenoble, France </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Milles%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sigrid Milles</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Leibniz-Forschungsinstitut für Molekulare Pharmakologie, Robert-Rössle-Straße 10, Berlin, Germany </div>
<div id="Aff2">
<sup>2</sup>Freie Universität Berlin, Department of Biology, Chemistry, Pharmacy, Berlin, Germany </div>
<div id="Aff3">
<sup>3</sup>Université Grenoble Alpes, CNRS, CEA, IBS, Grenoble, France </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 25; Accepted 2025 Aug 8; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370899  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841375/" class="usa-link">40841375</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Clathrin-mediated endocytosis depends on complex protein interactions. Eps15 plays a key role through interactions of its three EH domains with Asn-Pro-Phe (NPF) motifs in intrinsically disordered regions (IDRs) of other endocytic proteins. Using nuclear magnetic resonance spectroscopy, we investigate the interaction between Eps15’s EH domains and a highly disordered Dab2 fragment (Dab2<sub>320-495</sub>). We find that the EH domains exhibit binding promiscuity, recognizing not only the NPF motif of Dab2 but also other phenylalanine containing motifs. This promiscuity enables interactions with Eps15’s own IDR (Eps15<sub>IDR</sub>), which lacks NPF motifs, suggesting a self-inhibitory state that promotes liquid-liquid phase separation. Despite competing for the same EH domain binding sites, Eps15<sub>IDR</sub> and Dab2<sub>320-495</sub> can bind EH123 simultaneously, forming a highly dynamic interaction network that facilitates the recruitment of Dab2<sub>320-495</sub> into Eps15 condensates. Our findings provide molecular insights into the competitive interactions shaping the early stages of clathrin-mediated endocytosis.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Intrinsically disordered proteins, Endocytosis, Solution-state NMR</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Eps15, a protein involved in endocytosis initiation, uses small folded EH domains to bind disordered partner proteins and its own disordered tail, thus forming a dynamic network via phase separation that may help organize early endocytic events.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Clathrin-mediated endocytosis (CME) is the major pathway for cargo uptake into the eukaryotic cell, often with transmembrane receptors as cargoes. The important cellular process requires a complex network of interactions, which finally results in the formation and uptake of a clathrin-coated vesicle. During the early phases of the process that takes roughly one to two minutes<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>, intrinsically disordered protein regions (IDRs) of the endocytic initiators play a major role<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. While folded proteins mainly exist in one stable conformation, IDRs interconvert rapidly between many different conformations due to their relatively flat energy landscape. IDRs often recognize their folded partner proteins through small interaction regions of only a few residues, also called linear motifs<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. This is also the case in CME, where IDRs of endocytic initiators interact with, for example, the major adapter protein complex AP2 or clathrin<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, allowing them to establish a robust multivalent interaction network which allows for rapid rearrangement of the different protein members. Eps15, together with FCHo1/2 proteins, counts among the first proteins to arrive at the endocytic pit<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. It has three N-terminal Eps15 homology domains (EH domains), which share the common fold of two helix-loop-helix motifs (EF-hands) spaced by a short β-sheet<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. The EH domains are followed by a central coiled-coil domain, through which Eps15 dimerizes<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, and a large IDR of around 400 residues in length (Fig. <a href="#Fig1" class="usa-link">1A</a>)<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Eps15 tetramers, formed by two anti-parallel Eps15 dimers, have also been observed, seemingly requiring interaction between the EH domains and the IDR from opposite dimers<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. FCHo1/2, which also forms dimers, uses its C-terminal µ homology domain (µHD) to interact with the multiple Asp-Pro-Phe (DPF) motifs contained in Eps15’s IDR<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The multivalent interaction network between Eps15 and FCHo1/2 enables the formation of liquid-like protein droplets, which has been proposed as a process to catalyze the early steps of CME<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. The EH domains of Eps15, for example, can bind to Asn-Pro-Phe (NPF) motif containing proteins<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, such as Epsin1, Stonin2<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> or Dab2 (disabled homolog 2)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. While interactions between EH domains and both Epsin1 and Stonin2 have been characterized at molecular detail, the nature of Dab2’s interaction with EH domains remains to date elusive. Dab2 has an N-terminal folded phosphotyrosine interacting domain (PID), which it uses to anchor itself to the membrane and transmembrane cargo, followed by a large IDR of around 500 residues in length, which comprises both DPF as well as NPF motifs (Fig. <a href="#Fig1" class="usa-link">1B</a>)<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Both Eps15 and Dab2 belong to the class of clathrin-associated sorting proteins (CLASPs), which regulate cargo sorting during the early phases of CME and usually rely on downstream factors, such as the major adaptor protein complex AP2 or clathrin<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Dab2 has special properties as a CLASP since it can lead to successful CME of specific cargoes even in the absence of AP2. These cargoes, for example, integrin β1, are specifically recognized by the Dab2 PID. Importantly, binding of Eps15 to Dab2 is required for the AP2-independent internalization of these protein cargoes<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, highlighting the importance of understanding the communication between Eps15 and Dab2.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Interaction between Dab2<sub>320–495</sub> and the individual EH domains of Eps15.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370899_41467_2025_63090_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0bcc/12370899/24fe38ec0e48/41467_2025_63090_Fig1_HTML.jpg" loading="lazy" id="d33e338" height="398" width="693" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic illustration of <strong>(A)</strong> Eps15 and <strong>(B)</strong> Dab2. The disorder prediction (IUPred2A<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>) is shown as a black curve along the sequence. Above horizontal line: predicted disordered, Below horizontal line: predicted ordered. The folded domains are shown as gray boxes above the disorder prediction. Putative interaction motifs are shown as colored circles along the sequence (legend on the right of Dab2). DLL/DLF: clathrin binding; DPF/DPW: AP2α binding; FxDxF: AP2β binding; NPF: EH domain binding. The stretches used in this study are boxed with gray dashed lines. <strong>C</strong> Zoom into a <sup>1</sup>H−<sup>15</sup>N HSQC spectrum of Dab2<sub>320–495</sub> in the absence and presence of EH2 (left) and EH1 (right). <strong>D–F</strong> CSPs of 100 μM Dab2<sub>320–495</sub> in the presence versus the absence of increasing concentrations of EH1 (D), EH2 (E) and EH3 (F), respectively. Color legends are displayed in the respective plots. Filled pink circles denote positions of phenylalanines.</p></figcaption></figure><p id="Par4">Comprehending an interaction between two proteins that comprise a significant content of IDRs is challenging. In this work, we therefore chose to use nuclear magnetic resonance (NMR) spectroscopy, which is the only experimental technique able to provide amino-acid resolution of such dynamic protein systems. Using NMR spectroscopy, we thus investigated the region of Dab2 that is predicted to be most disordered, residues 320–495 (Dab2<sub>320–495</sub>), its conformational sampling, as well as its interactions with the EH domains of Eps15. All EH domains interact preferentially with the only NPF motif within Dab2<sub>320–495</sub>, but EH2 and EH3 demonstrate a high level of binding promiscuity towards other phenylalanine-containing motifs. This promiscuity is maintained for the combined EH domains, comprising EH1, EH2 and EH3 (EH123), and surprisingly enables Eps15’s own IDR (Eps15<sub>IDR</sub>), which does not possess any NPF motifs, to bind to EH123 and thereby partially occupy the binding pockets of the different EH domains. We provide a detailed molecular characterization of these intra-molecular interactions within Eps15, which are likely key to the liquid-liquid phase separation of Eps15 in the absence of other endocytic proteins. Our NMR results indicate that both Dab2<sub>320–495</sub> and Eps15<sub>IDR</sub> can bind EH123 simultaneously, but partially compete with each other. In line with this, we observe that Dab2<sub>320–495</sub> is recruited into liquid-like condensates formed by full-length Eps15. Our results provide a detailed description of the multivalent and promiscuous interactions between Eps15 and Dab2, which appear to be key to a dynamic protein network that can phase separate and evolve over time, which is required in the early stages of CME.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Dab2<sub>320–495</sub> is an IDR with mild helical propensities</h3>
<p id="Par5">The around 500 residue long IDR of Dab2 comprises a central region, which is predicted by IUPred2A<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> to be significantly more disordered than the remainder of the IDR (Fig. <a href="#Fig1" class="usa-link">1B</a>). We designed a construct encompassing this region, reaching from residue 320 to 495 (Dab2<sub>320–495</sub>). This region contains 2 putative binding sites for clathrin (DLL/DLF) and one NPF motif, prone to bind to EH domains contained in Eps15. In order to assess this construct by NMR spectroscopy and describe its conformational sampling, which is fundamental to its function, we first expressed and purified the construct, and assigned the NMR backbone resonances of a <sup>1</sup>H, <sup>15</sup>N, <sup>13</sup>C labeled sample (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). We were initially not able to assign two small regions within Dab2<sub>320–495</sub>, for which we designed two new stretches (Dab2<sub>328–360</sub> and Dab2<sub>358–390</sub>), whose <sup>1</sup>H-<sup>15</sup>N heteronuclear single quantum coherence (HSQC) spectrum overlaid well with the one of Dab2<sub>320–495</sub> (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>), enabling the transfer of assignments from Dab2<sub>328–360</sub> and Dab2<sub>358–390</sub> to Dab2<sub>320–495.</sub> As predicted, Dab2<sub>320–495</sub> appears to be disordered along its entire length, testified by secondary chemical shifts (SCSs) close to zero (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). However, calculating a conformational ensemble on the basis of chemical shifts using a combination of the statistical coil generator flexible meccano<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> and the selection algorithm ASTEROIDS<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> revealed three regions with mild helical propensity as compared to random coil: 339–354, 454–460, and 486–490 (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). Around the NPF motif, no particular structural propensity could be observed. We then recorded <sup>15</sup>N spin relaxation rates, exquisitely sensitive to the fast motions that intrinsically disordered proteins (IDPs) and IDRs undergo, in order to understand the dynamics of the intrinsically disordered chain of Dab2<sub>320–495</sub>. In agreement with the calculated SCSs, <sup>15</sup>N spin relaxation rates (R<sub>1ρ</sub>, R<sub>1</sub>, {<sup>1</sup>H}-<sup>15</sup>N heteronuclear Overhauser effect - hetNOE) are reminiscent of those of an intrinsically disordered protein (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>), with increased rigidity, testified by increased R<sub>1ρ</sub> and hetNOE rates, apparent around the transient helix between residues 328 and 360 (helix<sub>N</sub>). This helix is connected with the remainder of the disordered chain by a few amino acids of more rapid mobility. The more C-terminal transient helices are also characterized by slightly increased R<sub>1ρ</sub> and hetNOE rates, although less pronounced than those of helix<sub>N</sub>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">EH2 and EH3 are promiscuous binders</h3>
<p id="Par6">We then titrated the different Eps15 EH domains (EH1, EH2, or EH3) into <sup>15</sup>N Dab2<sub>320–495</sub> and recorded <sup>1</sup>H-<sup>15</sup>N HSQC spectra at the different titration steps to assess which residues in Dab2 take part in the interaction. While small chemical shift perturbations (CSPs) are observed very locally upon addition of EH1, the spectrum of Dab2<sub>320–495</sub> is more significantly perturbed when EH2 and EH3 are present at comparable concentrations (Fig. <a href="#Fig1" class="usa-link">1C–F</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">5</a>, <a href="#MOESM1" class="usa-link">6</a>). Plotting the CSPs of Dab2<sub>320–495</sub> upon interaction with the different EH domains along its sequence (Fig. <a href="#Fig1" class="usa-link">1D–F</a>) clearly illustrates that EH1, EH2, and EH3 have different binding patterns: EH1 is the most selective of all Eps15 EH domains, binding mainly to the NPF motif of <sup>15</sup>N Dab2<sub>320–495,</sub> while very small CSPs occur around residue F366 only at 200% EH1. EH2 and EH3, on the other hand, were able to interact with multiple small stretches on Dab2<sub>320–495</sub>, revealing a high degree of binding promiscuity. This binding behavior is also mirrored by decreased peak intensities (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>) and increased R<sub>1ρ</sub> rates around the respective interaction sites (Fig. <a href="#Fig2" class="usa-link">2A, B</a>). Since a spin-lock field of 1500 Hz was used in the R<sub>1ρ</sub> experiments, effectively quenching contributions of intermediate exchange to the relaxation rates, the observed increases in R<sub>1ρ</sub> reflect the slowed rotational tumbling times of the interacting residues when in contact with the respective EH domain. Residues in between the interaction sites have R<sub>1ρ</sub> rates similar to those of the unbound Dab2<sub>320–495</sub>, suggesting that the different interaction sites act independently from each other. The acquisition of R<sub>1ρ</sub> experiments of the different titration steps thus provides critical information on the dynamic behavior of Dab2’s intrinsically disordered chain and how this is affected by binding to the folded – and much more slowly tumbling – EH domains. Even though the largest CSPs were observed around the NPF motif when EH2 and EH3 were added, also residues around DYF, WPF, DLF, and DIF motifs were in contact with those two EH domains at higher EH:Dab2 ratios. This is also reflected by the <sup>15</sup>N R<sub>1ρ</sub> rates measured (Fig. <a href="#Fig2" class="usa-link">2A, B</a>). Common to all motifs is the phenylalanine residue, suggesting an importance for this residue type in the binding to EH2 and EH3 (Fig. <a href="#Fig1" class="usa-link">1E, F</a>). Interestingly, most of these motifs are located in the regions with increased helical propensity of Dab2<sub>320–495</sub> (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. <sup>15</sup>N spin relaxation of Dab2<sub>320–495</sub> upon interaction with the EH domains.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0bcc/12370899/c01575062b30/41467_2025_63090_Fig2_HTML.jpg" loading="lazy" id="d33e592" height="736" width="612" alt="Fig. 2"></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><sup>15</sup>N R<sub>1ρ</sub> spin relaxation of Dab2<sub>320–495</sub> (100 µM) in the absence and presence of <strong>(A)</strong> EH1 or EH3 and <strong>(B)</strong> increasing concentrations of EH2. The experiments were recorded at a <sup>1</sup>H frequency of 600 MHz. The relaxation rates in (<strong>A</strong>) and (<strong>B</strong>) were derived from a fit of peak intensities against the relaxation delay. Errors of the fitted rates were derived from the experimental uncertainty. <strong>C</strong> Zoom into the peak corresponding to V399 from Dab2<sub>320–495</sub>
<sup>1</sup>H-<sup>15</sup>N HSQC spectra alone and in the presence of increasing concentrations of EH1 (left) and EH2 (right). <strong>D</strong> Zoom into the peak corresponding to L486 from Dab2<sub>320–495</sub>
<sup>1</sup>H-<sup>15</sup>N HSQC spectra alone and in the presence of increasing concentrations of EH1 (left) and EH2 (right). Color legends are displayed in the respective panels. <strong>E</strong> ΔR<sub>2, eff</sub> calculated from effective R<sub>2</sub> values (R<sub>2, eff</sub>) of relaxation dispersion experiments at a CPMG frequency of 31 Hz subtracted with R<sub>2, eff</sub> at a CPMG frequency of 1000 Hz recorded on a sample of 100 µM Dab2<sub>320–495</sub> in the presence of 10% EH2 at a <sup>1</sup>H frequency of 600 MHz and 900 MHz. Error bars, propagated from the errors of R<sub>2, eff</sub> at 31 Hz and 1000 Hz determined from repeat measurements and with a minimum error of 0.5 (see Supplementary Fig. <a href="#MOESM1" class="usa-link">7B</a> for the corresponding decay curves), are shown for values that are significantly larger than 0 .</p></figcaption></figure><p id="Par7">The Dab2<sub>320–495</sub> motifs were furthermore observed to bind to the EH domains with different dynamics. For example, V399, just next to the NPF motif, showed only mild peak broadening upon interaction with EH1, justified by the slowed rotational tumbling times when in complex. Upon interaction with EH2, however, significantly stronger broadening was observed (Fig. <a href="#Fig2" class="usa-link">2C</a>). The other interacting regions show very little broadening and are mainly characterized by CSPs. One example is the residue L486, which does not interact with EH1 and shows essentially no line broadening combined with small CSPs in the presence of EH2 (Fig. <a href="#Fig2" class="usa-link">2D</a>). These differential behaviors prompted us to record Carr-Purcell-Meiboom-Gill (CPMG) relaxation dispersion of Dab2<sub>320–495</sub> to investigate the possibility of intermediate exchange in the microsecond to millisecond regime between the bound and unbound states in the presence of EH2 (Fig. <a href="#Fig2" class="usa-link">2E</a>). These experiments confirm that the NPF motif binds EH2 in intermediate exchange on the NMR chemical shift timescale. Surprisingly, many residues throughout the helix<sub>N</sub> also displayed dynamics on the intermediate timescale upon binding. The remaining motifs, DIF and DLF, were concluded to bind in fast exchange.</p>
<p id="Par8">In order to quantify the affinity of the interaction between Dab2<sub>320–495</sub> and the EH domains, we attempted to extract dissociation constants (K<sub>D</sub>) for the different Dab2<sub>320–495</sub> interaction motifs. Even though fitting of CSPs as a function of EH domain concentration was difficult due to the low affinities and peak broadening (especially around the NPF motif upon interaction with EH2), they can be estimated to be in the high micromolar to millimolar range. The highest affinity of ~340 μM was determined between residue G400 and EH2—this residue is close to the NPF motif, which broadens too rapidly in the titration to determine a K<sub>D</sub>. EH1 interacts with the NPF motif (G400) with an affinity similar to that of EH2 (~350 μM), while the affinity of EH3 was estimated to only ~1.4 mM. The other, promiscuous binding sites, interact even more weakly (Supplementary Fig. <a href="#MOESM1" class="usa-link">7A</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). Analysis of the CPMG relaxation dispersion curves of Dab2<sub>320–495</sub> upon interaction with EH2 using a two-site exchange model allowed us to extract an exchange rate of 149 ± 13 s<sup>−1</sup> and a percentage of bound Dab2<sub>320–495</sub> of 3.3 ± 0.3% for protein concentrations of 100 μM Dab2<sub>320–495</sub> and 10 μM EH2 (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). This results in a K<sub>D</sub> value of 196 μM, which is in a similar range as the affinity between Dab2<sub>320–495</sub> and EH2 determined by CSPs. The dispersion in the helix<sub>N</sub> and around the NPF motif were fit together in the fits of the CPMG experiments (Supplementary Fig. <a href="#MOESM1" class="usa-link">7B</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Dab2<sub>320–495</sub> binds to the hydrophobic pocket of EH domains</h3>
<p id="Par9">We wondered whether differences in the EH domains could explain the distinct interactions observed with the different motifs of Dab2<sub>320–495</sub>. We thus set out to determine the binding pockets on each of the EH domains that engage in interaction with Dab2<sub>320–495</sub>. Even though NMR assignments of the three EH domains exist<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, we assigned all three EH domains under our experimental conditions (Supplementary Figs. <a href="#MOESM1" class="usa-link">8</a>–<a href="#MOESM1" class="usa-link">10</a>). From the carbon SCSs, we calculated secondary structure propensities (SSPs)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, which are in agreement with previously determined NMR structures of the proteins (Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Additionally, <sup>15</sup>N R<sub>1</sub> and R<sub>1ρ</sub> relaxation rates reflect small folded domains with flexible N- and C-termini, as expected (Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>). We then titrated Dab2<sub>320–495</sub> into the different <sup>15</sup>N labeled EH domains and recorded <sup>1</sup>H-<sup>15</sup>N Transverse relaxation-optimized spectroscopy (TROSY) HSQC spectra of the EH domains at the different titration steps. We plotted the respective CSPs along the sequence of the individual EH domains (Fig. <a href="#Fig3" class="usa-link">3A–C</a>) and then colored the structures of the three EH domains relative to the magnitude of the observed CSPs (Fig. <a href="#Fig3" class="usa-link">3D</a>). Zooms into the spectra of EH2 and EH3 illustrate the spectral changes upon Dab2<sub>320–495</sub> binding (Fig. <a href="#Fig3" class="usa-link">3E</a>). The largest CSPs occur around the previously identified binding pockets<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> and are comparable between all three EH domains (Fig. <a href="#Fig3" class="usa-link">3D</a>). An additional binding region, a hydrophobic patch previously shown to bind a second NPF motif in Stonin2<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, is clearly visible in EH2. The region seems to participate in binding also within EH1 and EH3, albeit in a less pronounced way (Fig. <a href="#Fig3" class="usa-link">3A-D</a>, region around residues 185–195 in EH2). R<sub>1ρ</sub> relaxation rates systematically increase throughout all residues when Dab2<sub>320–495</sub> is titrated into the <sup>15</sup>N EH domains, likely reflecting mildly increased tumbling times resulting from the motional drag that the bound IDR exerts (Fig. <a href="#Fig3" class="usa-link">3F</a>)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Interaction between the individual EH domains of Eps15 and Dab2<sub>320–495</sub>.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370899_41467_2025_63090_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0bcc/12370899/bfbfccdae415/41467_2025_63090_Fig3_HTML.jpg" loading="lazy" id="d33e821" height="372" width="705" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A–C</strong> CSPs of 100 μM EH1 (<strong>A</strong>), EH2 (<strong>B</strong>) and EH3 (<strong>C</strong>), respectively, in the absence versus the presence of increasing amounts of Dab2<sub>320–495</sub>. Color legends are displayed in the respective plots. <strong>D</strong> Structures of EH1 (AlphaFold2<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>), EH2 (PDB <a href="https://doi.org/10.2210/pdb1FF1/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1FF1</a><sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>) and EH3 (PDB <a href="https://doi.org/10.2210/pdb1C07/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1C07</a><sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>) with the CSP at 100% Dab2<sub>320–495</sub> mapped onto them. The linear gradient starts from 0% (white), which represents no CSP, to the 100% (black), which is 0.75 ppm (the biggest CSP observed between EH2 and Dab2<sub>320–495</sub>). <strong>E</strong> Zoom into a <sup>1</sup>H-<sup>15</sup>N TROSY-HSQC spectrum of EH2 (left) and EH3 (right) alone and in the presence of Dab2<sub>320–495</sub>. <strong>F </strong><sup>15</sup>N R<sub>1ρ</sub> spin relaxation of EH3 (100 µM) in the absence and presence of Dab2<sub>320–495</sub> at a <sup>1</sup>H frequency of 600 MHz. Color legends are indicated in the figures. The relaxation rates were derived from a fit of peak intensities against the relaxation delay. Errors of the fitted rates were derived from the experimental uncertainty.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">EH123 shows increased binding promiscuity</h3>
<p id="Par10">In order to better reflect the physiological scenario, where the different EH domains in Eps15 are not isolated, but expressed in row, we investigated the interaction between Dab2<sub>320–495</sub> and the full EH-domain containing region (EH1-EH2-EH3, also called EH123) by titrating increasing concentrations of EH123 into <sup>15</sup>N Dab2<sub>320–495</sub>. As expected from the interactions of the individual EH domains, both CSPs and R<sub>1ρ</sub> rates reveal that EH123 is able to interact with the same regions of Dab2<sub>320–495</sub> as observed for EH2 and EH3 (Fig. <a href="#Fig4" class="usa-link">4A, B</a>). Interestingly, the R<sub>1ρ</sub> relaxation rates of Dab2<sub>320–495</sub> in the presence of EH123 show increased relaxation not only of the residues of the five Dab2<sub>320–495</sub> binding regions, but the residues located in between the binding motifs also show mildly increased rates. This is particularly evident for the residues located between the first three binding motifs (DYFDQQF, WPF and NPF) and suggests an overall stiffening of the chain, potentially due to binding of the motifs to different EH domains within the same EH123 molecule.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Interaction between EH123 and Dab2<sub>320–495</sub>.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370899_41467_2025_63090_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0bcc/12370899/10b7b863f081/41467_2025_63090_Fig4_HTML.jpg" loading="lazy" id="d33e911" height="292" width="750" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong><sup> 15</sup>N R<sub>1ρ</sub> spin relaxation of Dab2<sub>320–495</sub> (100 µM) in the absence and presence of increasing concentrations of EH123 at a <sup>1</sup>H frequency of 600 MHz. <strong>B</strong> CSPs between the <sup>1</sup>H-<sup>15</sup>N HSQC of <sup>15</sup>N Dab2<sub>320–495</sub> in the absence and presence of increasing amounts of EH123. The filled pink dots represent the phenylalanines present in the Dab2<sub>320–495</sub>. <strong>C</strong><sup> 15</sup>N R<sub>1ρ</sub> spin relaxation of the individual <sup>15</sup>N EH domains (EH1, EH2, EH3) as well as EH123 in the absence and presence of 100% Dab2<sub>320–495</sub> at a <sup>1</sup>H frequency of 600 MHz. <strong>D</strong> CSPs between the <sup>1</sup>H-<sup>15</sup>N TROSY-HSQC of <sup>15</sup>N EH123 in the absence and presence of increasing amounts of the Dab2<sub>320–495</sub>. The color legends are indicated in the respective figure panels. The relaxation rates in (<strong>A</strong>) and (<strong>C</strong>) were derived from a fit of peak intensities against the relaxation delay. Errors of the fitted rates were derived from the experimental uncertainty.</p></figcaption></figure><p id="Par11">While CSPs around the NPF motif remain the largest upon interaction with EH123, the difference between the interaction of EH123 with the NPF motif and the other motifs appears smaller than for interaction with the individual EH domains. Indeed, R<sub>1ρ</sub> rates increase much more significantly around these promiscuous sites, albeit part of the increase should be attributed to the larger size of EH123 relative to the individual domains. Nonetheless, binding affinities approximated from the CSPs for EH123 binding are significantly higher than those of the individual EH domains, ranging from around 100 to around 400 μM (Supplementary Fig. <a href="#MOESM1" class="usa-link">7A</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). This suggests that once one EH domain within EH123 is bound to Dab2<sub>320–495</sub>, the two other EH domains of the same molecule remain available for (preferential) binding to other motifs, an effect reminiscent of avidity, effectively increasing the total level of binding promiscuity. A much higher concentration of individual EH domains (3x as many) is needed to partially recover the chemical shift differences and R<sub>1ρ</sub> rates of Dab2<sub>320–495</sub> in the presence of EH123 (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">All EH domains within EH123 engage in interaction with Dab2</h3>
<p id="Par12">To investigate whether Dab2<sub>320–495</sub> binds to the same sites on the EH domains when they are expressed in row rather than independently, we also aimed to characterize the interaction between Dab2<sub>320–495</sub> and EH123 from the side of EH123. We therefore produced <sup>15</sup>N labeled EH123 and recorded a <sup>1</sup>H–<sup>15</sup>N TROSY-HSQC spectrum. If the different EH domains were unaffected by being connected to each other, we would expect this spectrum to overlay perfectly with those of the individual EH domains. However, small CSPs were observed for all three EH domains, and the peaks of EH2 and EH3 appeared broadened (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>), revealing that each EH domain is affected by the presence of the two additional domains, potentially due to intra-molecular interactions. Despite the changes in the spectrum of EH123 compared to those of EH1, EH2, and EH3, the assignments from the individual domains could be fully transferred to the combined construct. In order to examine the broadening of the peaks originating from EH2 and EH3, we recorded <sup>15</sup>N spin relaxation of EH123 (R<sub>1ρ</sub>). All R<sub>1ρ</sub> rates of EH123 were significantly increased as compared to the individual EH domains, which was expected for this larger protein construct (Fig. <a href="#Fig4" class="usa-link">4C</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">14C</a>). The rates of EH2 and EH3, however, were significantly higher overall than those from EH1. Indeed, the increased R<sub>1ρ</sub> rates were similar for EH2 and EH3, suggesting that EH2 and EH3 tumble together as one entity, while EH1 can move independently within EH123. To further investigate this behavior, we decided to address the very fast, picosecond motions that are normally present in dynamic linkers between folded domains by measuring a hetNOE experiment with EH123. The recorded hetNOEs show quite similar values all along EH123, reflecting the similar fold and internal dynamics of the individual domains (Supplementary Fig. <a href="#MOESM1" class="usa-link">14C</a>). Only the linker between EH1 and EH2 displays much lower hetNOE values, demonstrating that this linker is undergoing much more rapid motion than the rest of the protein construct, also compared to the linker connecting EH2 and EH3. The hetNOE data are thus in very good agreement with EH2 and EH3 tumbling together within the EH123 construct.</p>
<p id="Par13">We then added Dab2<sub>320–495</sub> to <sup>15</sup>N labeled EH123 to assess whether EH123 bound differently to Dab2 than the individual EH domains. We observed CSPs for all three domains, similar to those of the individual EH domains when interacting with Dab2<sub>320–495</sub> (Fig. <a href="#Fig4" class="usa-link">4D</a>), suggesting that the binding sites on all EH domains are also available within EH123, despite the fact that EH2 and EH3 tumble together within EH123. R<sub>1ρ</sub> rates increased throughout EH123 when Dab2<sub>320–495</sub> was added (Fig. <a href="#Fig4" class="usa-link">4C</a>), and the rates of EH1 remained lower than those of EH2 and EH3, suggesting that the overall conformation of the EH domains within EH123 is preserved upon binding of Dab2<sub>320–495</sub>.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Eps15’s own IDR interacts with EH123</h3>
<p id="Par14">The remarkable binding promiscuity towards phenylalanine-containing motifs observed between EH123 and Dab2<sub>320–495</sub>, prompted us to investigate whether EH123 could also interact with Eps15’s own IDR (Eps15<sub>IDR</sub>). Eps15<sub>IDR</sub> does not contain any canonical NPF motifs, but contains 14 DPF motifs (Fig. <a href="#Fig1" class="usa-link">1A</a>), which have previously been suggested to interact with Eps15’s own EH domains<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. When we added equimolar amounts of Eps15<sub>IDR</sub> to <sup>15</sup>N EH123, CSPs were observed throughout the <sup>1</sup>H–<sup>15</sup>N TROSY-HSQC spectrum of EH123, confirming the interaction and further suggesting that all EH domains bind to Eps15<sub>IDR</sub> (Fig. <a href="#Fig5" class="usa-link">5A</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>). Upon interaction, most peaks of EH2 and EH3 were severely broadened, likely because of the motional drag the long Eps15<sub>IDR</sub> exerts on EH123. As a consequence, many of the signals were too weak to calculate CSPs and extract relaxation rates for EH2 and EH3 within EH123 upon interaction with Eps15<sub>IDR</sub> (Fig. <a href="#Fig5" class="usa-link">5A</a>). However, the CSPs originating from EH1 within EH123 were all smaller than those in presence of the same amount of Dab2<sub>320–495</sub>, indicating that EH1 interacts less well with Eps15<sub>IDR</sub> than with Dab2<sub>320–495</sub>, in agreement with EH1 being the least promiscuous of the domains, binding more selectively to NPF motifs than EH2 and EH3 (Fig. <a href="#Fig5" class="usa-link">5A</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Competitive binding of Dab2<sub>320–495</sub> and Eps15<sub>IDR</sub> to EH123.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370899_41467_2025_63090_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0bcc/12370899/7f013432cf20/41467_2025_63090_Fig5_HTML.jpg" loading="lazy" id="d33e1163" height="517" width="763" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> CSPs calculated between the <sup>1</sup>H–<sup>15</sup>N TROSY-HSQC of 200 µM <sup>15</sup>N EH123 in the absence and presence of 200 µM Dab2<sub>320–495</sub> and/or 200 µM Eps15<sub>IDR.</sub> Below are <sup>15</sup>N R<sub>1ρ</sub> spin relaxation rates of <sup>15</sup>N EH123 in the absence and presence of 100% Eps15<sub>IDR</sub> and both 100% Dab2<sub>320–495</sub> and Eps15<sub>IDR</sub> at a <sup>1</sup>H frequency of 1200 MHz. Only the R<sub>1ρ</sub> rates within EH1 in the presence of Eps15<sub>IDR</sub> and Eps15<sub>IDR</sub> with Dab2<sub>320–495</sub> are shown as the peaks of EH2 and EH3 were severely broadened. The parts of EH123 corresponding to EH1, EH2, and EH3 are illustrated above the plots. <strong>B</strong> CSPs calculated between the <sup>1</sup>H-<sup>15</sup>N HSQC of 100 µM <sup>15</sup>N Eps15<sub>IDR</sub> in the absence and presence of 100 µM EH123 and 100 µM EH123 + 100 µM Dab2<sub>320–495</sub>, and <sup>15</sup>N R<sub>1ρ</sub> spin relaxation of <sup>15</sup>N Eps15<sub>IDR</sub> in the absence and presence of 100% EH123 and 100% EH123 + 100% Dab2<sub>320–495</sub> at a <sup>1</sup>H frequency of 1200 MHz. <strong>C</strong>
<sup>1</sup>H–<sup>15</sup>N HSQC spectra of 100 µM <sup>15</sup>N Eps15<sub>IDR</sub> alone and in the presence of 100 µM EH123. <strong>D</strong> Zoom into <sup>1</sup>H-<sup>15</sup>N HSQC spectrum of 100 µM <sup>15</sup>N Eps15<sub>IDR</sub> alone and in the presence of 100 µM EH123 and 100 µM EH123 + 100 µM Dab2<sub>320–495</sub>. <strong>E</strong> Zoom into <sup>1</sup>H-<sup>15</sup>N HSQC spectrum of 100 µM <sup>15</sup>N Dab2<sub>320–495</sub> alone and in the presence of 100 µM EH123 and 100 µM EH123 + 100 µM Eps15<sub>IDR</sub>. Color codes are denoted in the respective panels. The relaxation rates in (<strong>A</strong>) and (<strong>B</strong>) (lower panels) were derived from a fit of peak intensities against the relaxation delay. Errors of the fitted rates were derived from the experimental uncertainty.</p></figcaption></figure></section><section id="Sec9"><h3 class="pmc_sec_title">EH123 binds phenylalanine-containing motifs in Eps15<sub>IDR</sub>
</h3>
<p id="Par15">In order to further characterize the interaction between Eps15<sub>IDR</sub> and EH123, we assigned the backbone resonances of <sup>1</sup>H, <sup>15</sup>N and <sup>13</sup>C labeled Eps15<sub>IDR</sub>. To circumvent the spectral overlap of the long Eps15<sub>IDR</sub>, we designed four overlapping smaller stretches (Eps15<sub>IDR 481-581</sub>, Eps15<sub>IDR 569–671</sub>, Eps15<sub>IDR 648-780</sub>, and Eps15<sub>IDR 761–896</sub>). The <sup>1</sup>H-<sup>15</sup>N HSQC spectra of these stretches overlayed nicely with the spectrum of the full Eps15<sub>IDR</sub> (Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>), allowing the transfer of <sup>1</sup>H and <sup>15</sup>N resonance assignments between the spectra. We assigned 242 out of 393 resonances distributed across the IDR except for the N-terminal residues 481-521 to which no resonances could be assigned (Supplementary Fig. <a href="#MOESM1" class="usa-link">17</a>). This is likely because these residues (481–504) are part of the coiled-coil domain as predicted by AlphaFold2<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, giving rise to weak peaks in the 3D assignment spectra. Low R<sub>1</sub> and R<sub>1ρ</sub> relaxation rates and SCSs around 0 for both Eps15<sub>IDR</sub> and the smaller stretches agree well with a disordered protein (Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>). The C-terminal part however, (~residues 850–885) has increased R<sub>1ρ</sub> rates and SSP values close to 1, revealing a stable alpha helix. This alpha helix, which is also predicted by AlphaFold2<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, contains the two ubiquitin interaction motifs (UIMs) present in Eps15<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. In general, the R<sub>1ρ</sub> relaxation rates are slightly larger within Eps15<sub>IDR</sub> compared to the smaller stretches, potentially resulting from small hydrophobic clusters in these regions leading to transient self-interactions within the chain—an effect that has been observed for transiently folded elements in other IDPs<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>.</p>
<p id="Par16">We next recorded <sup>1</sup>H-<sup>15</sup>N HSQC spectra of <sup>15</sup>N Eps15<sub>IDR</sub> with equimolar amounts of unlabeled EH123 in order to pinpoint the exact interaction regions on Eps15<sub>IDR</sub>. R<sub>1ρ</sub> rates, CSPs, and intensity ratios compared to Eps15<sub>IDR</sub> on its own revealed two relatively large interaction regions (Fig. <a href="#Fig5" class="usa-link">5B, C</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>). These regions, 620–680 and 700–730, are characterized by a high density of phenylalanines. While the first region is enriched in DPF motifs, which have previously been proposed to bind to EH123, the second region is characterized by fewer DPF motifs and contains multiple other small linear motifs with phenylalanine, underlining the promiscuity of the EH domains. Moreover, small CSPs and increases in R<sub>1ρ</sub> rates are observed around three smaller regions, indicating additional interaction sites for EH123: residues ~800–809 and ~820–827 containing DPF(s), and residues ~843–846 containing phenylalanines. While increases in R<sub>1ρ</sub> relaxation rates and decreases in intensity ratios in the C-terminal helix are also observed (Fig. <a href="#Fig5" class="usa-link">5B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>), the small CSPs accompanying these changes suggest that this may be through binding to the first helical residues of hydrophobic nature (843–846), which then affects the tumbling time of the entire alpha helical element.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Dab2<sub>320–495</sub> and Eps15<sub>IDR</sub> partially compete for EH123 binding</h3>
<p id="Par17">The interaction observed between EH123 and Eps15<sub>IDR</sub> suggests that intra-molecular interactions within Eps15 may occupy Dab2 binding sites on the EH domains in the native context. We therefore conducted a competition experiment, acquiring a spectrum of <sup>15</sup>N EH123 with equimolar amounts of both Eps15<sub>IDR</sub> and Dab2<sub>320–495</sub>. The resulting CSPs of EH123 showed that in the presence of both Dab2<sub>320–495</sub> and Eps15<sub>IDR</sub>, the CSPs of EH1 were similar to those in the presence of Dab2 only. R<sub>1ρ</sub> rates around EH1 were increased mildly as compared to binding of Eps15<sub>IDR</sub> or Dab2<sub>320–495</sub> alone to EH123, while the peaks of EH2 and EH3 were broadened severely, also as observed upon Eps15<sub>IDR</sub> interaction (Fig. <a href="#Fig5" class="usa-link">5A</a> and see Supplementary Fig. <a href="#MOESM1" class="usa-link">15B</a>). While it is difficult to disentangle the individual contributions of Dab2<sub>320–495</sub> and Eps15<sub>IDR</sub> binding to the spectral changes, this suggest that both IDRs may bind EH123 at the same time. To further investigate this, we recorded a <sup>1</sup>H-<sup>15</sup>N HSQC spectrum of <sup>15</sup>N Eps15<sub>IDR</sub> and added first EH123 and then Dab2<sub>320–495</sub>. While most peaks were unaffected by the addition of Dab2<sub>320–495</sub>, some peaks shifted very slightly back towards the unbound state of Eps15<sub>IDR</sub> (Fig. <a href="#Fig5" class="usa-link">5B, D</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">19</a>). This suggests that binding between Eps15<sub>IDR</sub> and EH123 is largely unaffected by the presence of Dab2<sub>320–495</sub>. We then conducted the same competition experiment, but this time using <sup>15</sup>N Dab2<sub>320–495</sub>. Again, when adding equimolar amounts of Eps15<sub>IDR</sub> to <sup>15</sup>N Dab2<sub>320–495</sub> bound to EH123, the spectrum of Dab2<sub>320–495</sub> was only slightly affected, with peaks from bound Dab2<sub>320–495</sub> moving towards the unbound form (Fig. <a href="#Fig5" class="usa-link">5E</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">20</a>), likely due to competition with Eps15<sub>IDR</sub>. To rule out that any of the observed spectral changes could originate from a potential interaction between Dab2<sub>320–495</sub> and Eps15<sub>IDR</sub>, we recorded a <sup>1</sup>H-<sup>15</sup>N HSQC spectrum of <sup>15</sup>N Dab2<sub>320–495</sub> with Eps15<sub>IDR</sub>, confirming that the two IDRs do not interact (Supplementary Fig. <a href="#MOESM1" class="usa-link">21</a>). Taken together, the competition experiments conducted from the side of each binding partner (EH123, Eps15<sub>IDR</sub>, and Dab2<sub>320–495</sub>) suggest that both Eps15<sub>IDR</sub> and Dab2<sub>320–495</sub> can bind to EH123 at the same time, even though they are competing for the same EH domain interaction sites. This could be possible due to a fast on and off rate of the low-affinity interaction motifs, provided binding sites in both Dab2<sub>320–495</sub> and Eps15<sub>IDR</sub> bind with similar affinities, thereby creating a complex and dynamic interaction network allowing both IDRs to bind EH123.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Dab2<sub>320–495</sub> enters into Eps15 protein condensates</h3>
<p id="Par18">Eps15 has recently been shown to form liquid-like droplets, and interactions between the EH domains and Eps15<sub>IDR</sub> were proposed to contribute to droplet formation<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. The seemingly simultaneous interactions of Eps15<sub>IDR</sub> and Dab2<sub>320–495</sub> with EH123 observed here made us question whether Dab2<sub>320–495</sub> could be recruited to Eps15 condensates. We therefore expressed and purified full-length Eps15, from here on just called Eps15, and labeled its cysteines with the fluorophore AZDye594. In order to fluorescently label Dab2<sub>320–495</sub>, which does not contain any cysteines, we created a single cysteine mutant (S328C Dab2<sub>320–495</sub>), and labeled it with AZDye488. We first examined if we could achieve liquid-like droplet formation of Eps15 under our experimental conditions. Indeed, upon addition of 3% PEG8000, we observed droplet formation starting from a protein concentration of as little as 0.5 μM (Fig. <a href="#Fig6" class="usa-link">6A</a>). Droplets increased in size with increasing Eps15 concentrations (Fig. <a href="#Fig6" class="usa-link">6A, C</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">22A</a>). Unlike Eps15, Dab2<sub>320–495</sub> was not able to form any droplets on its own (Supplementary Fig. <a href="#MOESM1" class="usa-link">23B</a>). When Dab2<sub>320–495</sub> was added to Eps15 droplets, it entered readily and clearly enriched in the Eps15 condensed phase (Fig. <a href="#Fig6" class="usa-link">6B</a>). While the presence of Dab2<sub>320–495</sub> did not majorly alter the size or number of the droplets overall, we did not manage to observe phase separation at the lowest tested concentration of 0.5 μM Eps15 when 7 μM Dab2<sub>320–495</sub> was present in addition (Fig. <a href="#Fig6" class="usa-link">6B, C</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a>). We were thus wondering, how Dab2<sub>320–495</sub> concentration affected Eps15 condensation. Therefore, we added increasing concentrations of Dab2<sub>320–495</sub> (0.5 to 10 μM) to droplets formed by 7 μM Eps15—a concentration at which many and relatively large droplets can be observed. Dab2<sub>320–495</sub> entered into the droplets at all concentrations added, with the intensity of Dab2<sub>320–495</sub> inside the droplets increasing with higher Dab2<sub>320–495</sub> concentration (Fig. <a href="#Fig6" class="usa-link">6D</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a>). No major effects on condensate size or number of condensates were observed (Fig. <a href="#Fig6" class="usa-link">6E</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">22</a>), and even 20 times molar excess of Dab2<sub>320–495</sub> did not seem to majorly alter the condensates (Supplementary Fig. <a href="#MOESM1" class="usa-link">23</a>). Since droplets made in the presence of smaller concentrations of Dab2<sub>320–495</sub> were significantly less bright than those made in the presence of higher Dab2<sub>320–495</sub> concentrations, we also analyzed the partition coefficient of both proteins into the droplets. While Eps15 partitioned better into droplets with increasing Eps15 concentration, its partitioning was less dependent on Dab2<sub>320–495</sub> concentration (Supplementary Fig. <a href="#MOESM1" class="usa-link">22C, D</a>). Partitioning of Dab2<sub>320–495</sub> into the droplets depended on both its own concentration in the sample, as well as the concentration of Eps15 (Supplementary Figs. <a href="#MOESM1" class="usa-link">22C, D</a>). Even though the interactions between Dab2<sub>320–495</sub> and EH123 are rather weak, recruitment into Eps15 droplets is specific, illustrated by a fluorescently labeled DNA, which is not recruited into Eps15 droplets (Supplementary Fig. <a href="#MOESM1" class="usa-link">23E</a>). This indicates, in line with our NMR results, that the inter-molecular Eps15 interactions responsible for droplet formation can be maintained in the presence of Dab2<sub>320–495</sub>, while Eps15 also remains available for interaction with Dab2.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Dab2<sub>320–495</sub> is recruited into Eps15 droplets.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370899_41467_2025_63090_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0bcc/12370899/5aab1e064f73/41467_2025_63090_Fig6_HTML.jpg" loading="lazy" id="d33e1555" height="1250" width="709" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Representative droplets imaged from Eps15 alone at different concentrations (0.5 to 10 μM, indicated above). <strong>B</strong> Eps15 droplets formed under the same conditions in the presence of 7 μM Dab2<sub>320–495</sub>. <strong>C</strong> Droplet sizes plotted in a histogram for different concentrations of Eps15 in the presence and absence of 7 μM Dab2<sub>320–495</sub>, corresponding to the microscopy images in (<strong>A</strong>) and (<strong>B</strong>). <strong>D</strong> Eps15 droplets formed with 7 μM protein, to which different concentrations of Dab2<sub>320–495</sub> (0.5 to 10 μM, indicated above) was added. <strong>E</strong> Droplet sizes plotted in a histogram for different concentrations of Dab2<sub>320–495</sub> in the presence and absence of 7 μM Eps15, corresponding to the microscopy images in (<strong>D</strong>). 10% of the protein used in the microscopy experiments was fluorescently labeled. Eps15 was labeled using AZDye594 (displayed in red). Dab2<sub>320–495 S328C</sub> was labeled using AZDye488 (displayed in green). The scale bar is 5 μm and applies to all images in the figure. Images were acquired by confocal microscopy. The number of images used for the histograms is indicated as ‘n = x’ in each plot. Source data are the same as for Supplementary Fig. <a href="#MOESM1" class="usa-link">22A and B</a> and are provided as a Source Data file.</p></figcaption></figure></section></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par19">The dynamic network formed by intrinsically disordered regions (IDRs) and folded proteins interacting with short linear motifs within these IDRs, is key to a number of biological processes<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Clathrin-mediated endocytosis is a prime example of a process that comprises many of those short linear motifs<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, leading to the formation of a complex interaction network. Known examples are the α and β2 appendage domains of the major adapter protein AP2 that interact with DPF/DPW and FxDxF motifs (x = any amino acid)<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, the clathrin heavy chain terminal domain that interacts with DLL/DLF motifs<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> or the EH domains found in Eps15, which interact with NPF motifs of diverse IDR partners<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Although extensive studies have identified these consensus sequences, it has meanwhile become clear that additional binding sites are yet to be discovered and that this requires a strategy by which focus lies on studying long endocytic IDRs. This has recently allowed identifying a particularly long interaction site between the neuronal AP180 and the AP2β2 appendage domain<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. In addition, many endocytic IDRs seem to bind to the same partners<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup> and understanding how the different binding modes synergize or compete with each other is crucial for a molecular comprehension of clathrin-mediated endocytosis.</p>
<p id="Par20">In this study, we shed light onto those partially competitive interactions by investigating the binding of a large intrinsically disordered region stemming from the CLASP Dab2 (Dab2<sub>320–495</sub>) and its interactions with Eps15 EH domains, as well as its competition for binding with Eps15’s own IDR (Eps15<sub>IDR</sub>). In line with the previous literature and extensive peptide screens to assess binding patterns of Eps15 EH domains<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, all three EH domains of Eps15 (EH1, EH2 and EH3) prefer to bind to the only NPF motif within the sequence of Dab2<sub>320–495</sub> (Fig. <a href="#Fig1" class="usa-link">1</a>, <a href="#Fig2" class="usa-link">2</a>). In the presence of EH2, significant exchange in the microsecond to millisecond time scale is observed around the NPF motif and an N-terminal transient helix in Dab2<sub>320–495</sub> (helix<sub>N</sub>), allowing to estimate a dissociation constant between Dab2<sub>320–495</sub> (NPF motif and helix<sub>N</sub>) and EH2 on the order of hundreds of micromolar (Fig. <a href="#Fig2" class="usa-link">2</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">7B</a> and Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). These affinities are in agreement with affinities of other small linear motif interactions determined in the context of clathrin-mediated endocytosis<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> and also with those determined from chemical shift perturbations between Dab2<sub>320–495</sub> and EH domains (Supplementary Fig. <a href="#MOESM1" class="usa-link">7A</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). However, they are orders of magnitude weaker than those observed between Stonin2 and EH2 (K<sub>D</sub> in the nanomolar range)<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Although the EH domains bind preferentially to the NPF motif of Dab2<sub>320–495</sub>, as it is expected from the literature<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, the atomic resolution provided by our NMR experiments also points toward additional, slightly weaker binding sites including DYF, WPF, DLF, and DIF motifs (Fig. <a href="#Fig1" class="usa-link">1</a>). Such sites will certainly play a role in the crowded environment of the endocytic pit, and these motifs further testify to an extremely promiscuous binding behavior of EH2 and EH3. This is in line with the observation that other EH domains, such as the EH domain of EHD1 or of POB-1 can bind to the motif xPF<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. By investigating the EH domain interactions with small linear motifs from the side of a bona fide IDR, we can conclude that the promiscuity of EH2 and EH3 is even larger than previously anticipated, binding to most phenylalanine containing motifs (Fig. <a href="#Fig7" class="usa-link">7A</a>). It is remarkable that, overall, the dissociation constants determined for binding between EH domains and the NPF motif in Dab2<sub>320–495</sub> are not extremely different to those between EH domains and non-NPF binding sites (~1.5 mM versus up to ~4 mM for EH3 for example, ~0.3 mM versus up to ~10 mM for EH2), suggesting that promiscuity can indeed play an important role physiologically.</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7. Schematic of Eps15 and Dab2<sub>320–495</sub> interactions and phase separation.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0bcc/12370899/87410c636591/41467_2025_63090_Fig7_HTML.jpg" loading="lazy" id="d33e1720" height="1062" width="666" alt="Fig. 7"></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> illustration of the complex interaction network between Eps15 and Dab2<sub>320–495</sub>, including intra-molecular interactions within Eps15. Interactions are visualized with red dashed lines. <strong>B</strong> Cartoon of self-interactions within Eps15 and of interactions between Eps15 and Dab2<sub>320–495</sub>. Eps15’s EH domains interact with Eps15<sub>IDR</sub> and with Dab2<sub>320–495</sub>. EH2 and EH3 tumble together. All three EH domains interact with the NPF motif within Dab2<sub>320–495</sub>, while EH2 and EH3 are promiscuous binders. <strong>C</strong> The weak multivalent interaction network of the two IDRs with EH123 likely contributes to co-phase separation of Eps15 and Dab2<sub>320–495</sub>. Eps15:Dab2 interactions are highlighted in red shading, and Eps15:Eps15 interactions are highlighted in pink shading.</p></figcaption></figure><p id="Par21">Even though some proteins contain individual EH domains, this is not the case for Eps15, which contains EH1, EH2 and EH3 sequentially at its N-terminus. Unexpectedly, we observe that EH2 and EH3 move together as one entity in a construct comprising all three EH domains (EH123, Fig. <a href="#Fig4" class="usa-link">4C</a>), suggesting a potential interaction between EH2 and EH3. EH1, on the other hand, tumbles independently from the other two EH domains. Nonetheless, EH123 makes all EH domain binding sites available for interaction with Dab2<sub>320–495</sub>. From the side of Dab2<sub>320–495</sub>, binding to EH123 is strikingly different than binding to the individual EH domains. While the same NPF and promiscuous motif interactions maintain, the non-NPF interactions seem to gain importance, leading to affinities between the different Dab2<sub>320–495</sub> motifs in the hundreds of μM range for both NPF and non-NPF motifs (Fig. <a href="#Fig4" class="usa-link">4</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">7A, 13</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). This could be due to avidity effects of the three EH domains in close spatial proximity compared to the individual domains.</p>
<p id="Par22">The present promiscuity of interactions, particularly in the context of the full EH123 construct, suggests that the EH domains might bind to Eps15’s own IDR, which has also been suggested previously<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Addition of Eps15<sub>IDR</sub> to <sup>15</sup>N EH123 indeed led to CSPs and increased <sup>15</sup>N R<sub>1ρ</sub> relaxation rates confirming this intra-molecular interaction (Fig. <a href="#Fig5" class="usa-link">5A</a>). Our data recorded on <sup>15</sup>N labeled Eps15<sub>IDR</sub> show that EH123 binds to all phenylalanine containing regions of Eps15<sub>IDR</sub>, highlighting the promiscuity of EH123 also observed in the binding to Dab2<sub>320–495</sub> (Fig. <a href="#Fig5" class="usa-link">5B</a>). While one would expect that an NPF containing IDR, such as Dab2<sub>320–495</sub>, might replace Eps15<sub>IDR</sub> from EH123 due to the preferred interaction of EH domains with NPF motifs compared to more promiscuous binding sites, this seems not to be the case. When both Eps15<sub>IDR</sub> and Dab2<sub>320–495</sub> are added to <sup>15</sup>N labeled EH123, the spectrum and corresponding relaxation rates show signatures of binding to both proteins, although the individual contributions cannot be disentangled due to their similar interaction sites on EH123 (Fig. <a href="#Fig5" class="usa-link">5A</a>). A similar behavior is observed when a sample containing all three proteins is detected from the side of <sup>15</sup>N Eps15<sub>IDR</sub> or <sup>15</sup>N Dab2<sub>320–495</sub>. Only a few regions within the spectra of Eps15<sub>IDR</sub> and Dab2<sub>320–495</sub> slightly revert towards their unbound state when both EH123 and the respective other IDR (Dab2<sub>320–495</sub> or Eps15<sub>IDR</sub>) are added (Fig. <a href="#Fig5" class="usa-link">5D, E</a>), suggesting that both Eps15<sub>IDR</sub> and Dab2<sub>320–495</sub> can bind EH123 at the same time.</p>
<p id="Par23">How is this possible when Dab2<sub>320–495</sub> and Eps15<sub>IDR</sub> occupy the same binding sites on EH123? Key to this question is likely the multivalency by which both IDRs interact with EH123, such that, while some motifs of Dab2<sub>320–495</sub> might be displaced from EH123 in a dynamic fashion when Eps15<sub>IDR</sub> is added—or vice versa—others maintain, thereby creating a dynamic trimeric (or even higher order) complex. Which of the motifs remain bound to EH123 and which get displaced is certainly a function of the individual motif’s affinities. This is easiest visualized as a dynamic equilibrium where motifs in Dab2 are competing with Eps15’s own IDR for binding to the EH domains, forming a complex and dynamic network (Fig. <a href="#Fig7" class="usa-link">7A, B</a>). The fact that neither Dab2<sub>320–495</sub> nor Eps15<sub>IDR</sub> can effectively out-compete the other suggests that the strengths of the individual interactions between EH123 and Dab2<sub>320–495</sub> or Eps15<sub>IDR</sub> are not vastly different. The affinity between the EH domains and Eps15<sub>IDR</sub> might be increased in native full-length Eps15, as a result of proximity effects. However, our phase separation experiments with full-length Eps15 and Dab2<sub>320–495</sub> show recruitment of Dab2 into the Eps15 condensates, arguing for a physiological relevance of the interactions seen between Dab2<sub>320–495</sub>, the EH domains and Eps15<sub>IDR</sub> and the competition effects observed.</p>
<p id="Par24">While binding of EH domains to an intrinsically disordered linker of the same protein has been observed for the EH domain contained in EHD2<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, the interaction network established by Eps15 is certainly remarkable, since so many interactions take place and compete with each other (Fig. <a href="#Fig7" class="usa-link">7</a>). This has implications for all other binding partners of Eps15, whether binding to the EH domains or Eps15’s IDR, since any binding partner will be competing with these intra-molecular interactions. Indeed, the interactions observed between Eps15 EH domains and Eps15<sub>IDR</sub> could constitute some kind of auto-inhibitory mechanism, such as recently described for WW domain proteins<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>, which could be (partially) released by other interaction partners, such as Dab2. In this context, it should be noted that the interactions between EH123 and Eps15<sub>IDR</sub> could be both intra-molecular within one Eps15 molecule and inter-molecular between different Eps15 molecules. Indeed, Eps15-Eps15 interactions have previously been suggested to drive Eps15 phase separation in the context of clathrin-mediated endocytosis. For example, based on previous literature on the interaction between the EH domain of POB1 and DPF motifs<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, which are also contained in Eps15<sub>IDR</sub>, interaction between Eps15 EH domains and Eps15<sub>IDR</sub> has been suggested to be important for liquid-liquid phase separation<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. While our molecular data show that not only DPF motifs, but also other F-rich protein regions are involved in this interaction, other interactions of similar strengths between Eps15 and its binding partners may drive this partner into the liquid-like droplets, such as it has been observed for FCHo1/2<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> or ubiquitin<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Of note, phenylalanines have been identified as critical ‘stickers’ to promote weak interactions in liquid-liquid phase separation of IDPs<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Usually, they are thought to interact with phenylalanines in other IDPs, thereby creating a dynamic interaction network. In the case of Eps15, we have observed phenylalanines to contact small folded domains (EH domains)—interactions that may also drive Dab2<sub>320–495</sub> into Eps15 droplets. In good agreement with this hypothesis, we observe that Dab2<sub>320–495</sub>, which does not form droplets on its own, also enters liquid-like droplets of Eps15, likely due to the weak multivalent network between the two (Fig. <a href="#Fig7" class="usa-link">7C</a>). Therefore, while current literature points to Eps15 as the main initiator of condensate formation during the early phases of CME<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> our data suggest that Dab2 may be recruited into such condensates also in the cellular context. Interestingly, even though Eps15’s IDR and Dab2<sub>320–495</sub> seem to compete for the same interaction sites on EH123, the presence of Dab2<sub>320–495</sub> had only negligible effects on the Eps15 condensates (Fig, <a href="#Fig6" class="usa-link">6</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">22</a>, <a href="#MOESM1" class="usa-link">23</a>) and even a large excess of Dab2<sub>320–495</sub> did not dissolve Eps15 droplets (Supplementary Fig. <a href="#MOESM1" class="usa-link">23</a>). This behavior may make sense in the context of the early clathrin coated pit: Eps15 condensates are indeed thought to function as an initiator<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, responsible to accumulate downstream client proteins that make endocytosis progress. Eps15 only moves away from the pit later in the process, when many other CLASPs/accessory proteins will have enriched at the pit<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. It is thus possible that a much higher concentration of competitive interaction partners from different regions of Dab2, but also other NPF (and non-NPF) interaction partners are needed to affect phase separation or segregate these proteins out of Eps15 condensates. Indeed, it will be interesting to see whether the interactions between EH domains and Eps15<sub>IDR</sub> are maintained also in the presence of proteins containing multiple NPF motifs, binding with higher affinities<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, and what consequences this has on liquid-liquid phase separation at the endocytic pit and thus progression of productive endocytosis.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Cloning</h3>
<p id="Par25">Eps15-pmCherryN1 was a gift from Christien Merrifield (Addgene plasmid # 27696; <a href="http://n2t.net/addgene:27696" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://n2t.net/addgene:27696</a>; RRID: Addgene_27696)<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. The three individual EH domains (EH1, EH2, EH3) and Eps15<sub>IDR 761-896</sub> were cloned into the pET41c vector leading to constructs with a non-cleavable C-terminal His-tag. Full length Eps15, EH123, Eps15<sub>IDR</sub>, Eps15<sub>IDR 481–581</sub>, Eps15<sub>IDR 569-671</sub> and Eps15<sub>IDR 648–780</sub> were cloned into the pET28a vector with an N-terminal GB1 solubility tag pET28-6His-GB1. This leads to the expression of a TEV (tobacco etch virus) cleavable 6His-GB1-TEV site construct followed by the protein of interest. The gene of Dab2<sub>320–495</sub> was purchased from Twist Bioscience and cloned into a pET28a vector with non-cleavable C-terminal His-tag. For the purpose of fluorescence labeling, a single cysteine mutant of Dab2<sub>320–495</sub> was constructed using site-directed mutagenesis (Dab2<sub>320–495 S328C</sub>). Dab2<sub>328–360</sub> and Dab2<sub>358–390</sub> were cloned into a pET28a vector with an N-terminal GB1 solubility tag pET28-6His-GB1. The UniProt IDs of Eps15 and Dab2 used in this study are <a href="https://www.ncbi.nlm.nih.gov/protein/P42566" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P42566</a> and <a href="https://www.ncbi.nlm.nih.gov/protein/P98082" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P98082</a>, respectively. The primer list for all cloning performed can be found below:</p>
<section class="tw xbox font-sm" id="Taba"><div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups"><tbody>
<tr><td colspan="2" rowspan="1"><strong>Restriction enzyme cloning</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_120_Xho1_rv</td>
<td align="left" colspan="1" rowspan="1">GTG GTG CTC GAG CTC AGC TGC AGA GGT TCC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_215_Xho1_rv</td>
<td align="left" colspan="1" rowspan="1">GTG GTG CTC GAG TCT CTT AGA TGG TGG CAC CAA GGC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_310_Xho1_rv</td>
<td align="left" colspan="1" rowspan="1">GTG GTG CTC GAG TGG TGG AAT CAT TTC AGG AGT AAG AAC GTG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_671_Xho1_rv</td>
<td align="left" colspan="1" rowspan="1">AGT GCC TCG AGT TAA GTG CTT GAA GTG GCA AAA GGA TCA G</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_780_Not1_rv</td>
<td align="left" colspan="1" rowspan="1">GTG GTG GCG GCC GCT CTT GTT GGA GTT CCG ATC TTT GGT GG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_896_Not1_rv</td>
<td align="left" colspan="1" rowspan="1">GTG GTG GCG GCC GCC CGT GCT TCT GAT ATC TCA GAT TTG CTG AG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Dab2_360_Not1_rv</td>
<td align="left" colspan="1" rowspan="1">GCA CTG CGG CCG CTT ATT GGG CCT CTT GCT TCC CG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Dab2_390_Not_rv</td>
<td align="left" colspan="1" rowspan="1">GCA CTG CGG CCG CTT AGG AAA ATC CGT TTT GCT CGC GC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_1_Nde1_fw</td>
<td align="left" colspan="1" rowspan="1">GAT ATA CAT ATG GCT GCG GCG GC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_121_Nde1_fw</td>
<td align="left" colspan="1" rowspan="1">GAT ATA CAT ATG CCA TGG GCT GTA AAA CCT GAA GAT AAG GC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_216_Nde1_fw</td>
<td align="left" colspan="1" rowspan="1">GAT ATA CAT ATG AAA ACG TGG GTT GTA TCC CCT GCA G</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_481_Nde1_fw</td>
<td align="left" colspan="1" rowspan="1">GAT ATA CAT ATG CAC CTA CAA GAT TCA CAA CAG GAA ATT AGT TCA ATG C</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_761_Nde1_fw</td>
<td align="left" colspan="1" rowspan="1">GATATA CATATG TCG GTC AAA AGT GAA GAT GAA CCC CC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15_569_Nde1_fw</td>
<td align="left" colspan="1" rowspan="1">GTG CGC ATA TGT CTG GTG TGA CTG ATG AAA ATG AGG TG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Dab2_328_Nde_fw</td>
<td align="left" colspan="1" rowspan="1">GTG CGC ATA TGA GCA CCC CCT TGT CCA ATG G</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Dab2_358_Nde_fw</td>
<td align="left" colspan="1" rowspan="1">GTG CGC ATA TGG AGG CCC AAG CGG GC</td>
</tr>
<tr><td colspan="2" rowspan="1"><strong>Q5® Site-Directed Mutagenesis Kit (NEB)</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15-IDR1(481-581)_F</td>
<td align="left" colspan="1" rowspan="1">CTC GAG CAC CAC CAC CAC CAC C</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15-IDR1(481-581)_R</td>
<td align="left" colspan="1" rowspan="1">AAC AGC TGT AGT CAC CTC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15-IDR1(648-780)_F</td>
<td align="left" colspan="1" rowspan="1">AAA GGT TCA GAT CCA TTT G</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Eps15-IDR1(648-780)_R</td>
<td align="left" colspan="1" rowspan="1">CAT ATG ACC CTG GAA GTA C</td>
</tr>
<tr><td colspan="2" rowspan="1"><strong>Site directed mutagenesis</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Dab2_S328C_fw</td>
<td align="left" colspan="1" rowspan="1">CTA GCA GCT GTA CCC CCT TGT CCA ATG GTC C</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Dab2_S328C_rv</td>
<td align="left" colspan="1" rowspan="1">CAA GGG GGT ACA GCT GCT AGA ACT ATT CTC CTT CTT CAT GGT ATA TC</td>
</tr>
</tbody></table></div>
<div class="p text-right font-secondary"><a href="table/Taba/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec15"><h3 class="pmc_sec_title">Protein expression and purification</h3>
<p id="Par26">The proteins were expressed in the <em>E. coli</em> Rosetta (DE3) strain and grown in LB medium with 30 mg/L Kanamycin and 30 mg/L Chloramphenicol at 37 °C. When the optical density (OD) at 600 nm was around 0.6–1, expression was induced with 1 mM isopropyl-β-D-thiogalactopyranoside (IPTG) and the expression was then continued at 20 °C overnight. For isotope labeling (<sup>15</sup>N, <sup>13</sup>C) M9 minimal medium was used and supplemented with 1 g/L <sup>15</sup>NH<sub>4</sub>Cl and/or with 2 g/L <sup>13</sup>C-glucose.</p>
<p id="Par27">Cells were lysed by sonication in lysis buffer (20 mM Tris, 150 mM NaCl pH 8, with Roche Ethylenediaminetetraacetic Acid (EDTA)-free protease inhibitor cocktail (Sigma-Aldrich Chemie GmbH)). Purification involved a two-step process: initial nickel purification followed by size-exclusion chromatography (SEC). The nickel column was equilibrated in lysis buffer before the filtered lysate was applied to a nickel column. The column was then washed with lysis buffer containing 20 mM imidazole, and the protein was eluted using lysis buffer supplemented with 400 mM imidazole. For all constructs expressed in the pET28-6His-GB1 vector, the eluted fraction, which contained the target protein (validated by SDS-PAGE and Coomassie staining), underwent overnight dialysis with 1 mg TEV protease at 4 °C in 500 mL of lysis buffer supplemented with 5 mM β-mercaptoethanol before proceeding to the SEC purification step. Proteins were further purified using a Superdex 75 or a Superdex 200 column, equilibrated in NMR buffer (50 mM Na-phosphate pH 6, 150 mM NaCl, and 2 mM dithiothreitol (DTT)). Fractions containing pure protein (validated by SDS-PAGE) were concentrated and frozen with final protein concentrations determined by absorbance at 280 nm using extinction coefficients determined by Expasy Protparam<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>.</p></section><section id="Sec16"><h3 class="pmc_sec_title">NMR spectroscopy</h3>
<p id="Par28">NMR experiments were measured at the Leibniz-Forschungsinstitut für Molekulare Pharmakologie (FMP), Berlin, Germany (<sup>1</sup>H frequencies of 600, 750, 900, 1200 MHz), and at the Institute of Structural Biology (IBS), Grenoble, France (<sup>1</sup>H frequencies of 600 MHz). The spectrometers were equipped with either room-temperature- (750 MHz) or cryo-probes (600, 900 MHz, 1200 MHz). All experiments were measured in NMR buffer (50 mM Na-phosphate pH 6, 150 mM NaCl, 2 mM DTT) at 25 °C. Addition of up to 5 mM CaCl<sub>2</sub> did not affect our results. Spectra were processed with NMRPipe<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>, using qMDD<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup> for non-uniformly sampled assignment spectra, and analyzed with CCPN<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. <sup>1</sup>H-<sup>15</sup>N HSQC, TROSY-HSQC and triple resonance experiments were acquired using NMRlib<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. The software TopSpin 3.5 and 4.4.1 (Bruker) were used for data acquisition.</p></section><section id="Sec17"><h3 class="pmc_sec_title">
<sup>15</sup>N, <sup>13</sup>C Backbone assignment</h3>
<p id="Par29">Almost complete assignments could be obtained for all protein constructs. The assignment spectra of the <sup>15</sup>N, <sup>13</sup>C labeled EH1, EH2, EH3, Dab2<sub>328–360</sub>, and Dab2<sub>358–390</sub> were acquired at a <sup>1</sup>H frequency of 750 MHz. Assignment spectra of Eps15<sub>IDR</sub>, Eps15<sub>IDR 480–581</sub>, Eps15<sub>IDR 569–671</sub>, Eps15<sub>IDR 648–780</sub>, Eps15<sub>IDR 761–896</sub>, and Dab2<sub>320–495</sub> were acquired at a <sup>1</sup>H frequency of 600 MHz. Standard BEST-TROSY triple resonance experiments correlating CO, Cα, Cβ resonances (HNCO, HNCOCA, HNCA, iHNCA, HNCOCACB, iHNCACB)<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> were acquired. The assignments were done in CCPN<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> and then validated using MARS<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Secondary chemical shifts and secondary structure propensities<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> were calculated using random coil values from refDB<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Overlapping the 3 TROSY-HSQC spectra of the individual EH domains (EH1, EH2, EH3) we transferred the <sup>1</sup>H and <sup>15</sup>N resonances to the TROSY-HSQC of the EH123 domain. The assignment of the different Eps15<sub>IDR</sub> stretches was transferred to the spectrum of the full Eps15<sub>IDR</sub> in a similar way. Dab2<sub>320–495</sub> comprises many prolines, and some peaks within its <sup>1</sup>H-<sup>15</sup>N HSQC spectrum could be attributed to appear due to cis/trans isomerization of neighboring prolines. We have not observed differences in binding due to cis/trans proline isomerization for the interactions tested and therefore all plots within the manuscript refer to the main conformational state of the protein.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Titrations, <sup>15</sup>N relaxation, and relaxation dispersion</h3>
<p id="Par30">Extraction of peak intensities (I) as well as <sup>1</sup>H and <sup>15</sup>N chemical shifts, were carried out from <sup>1</sup>H-<sup>15</sup>N HSQC or BEST-TROSY-HSQC spectra. Combined chemical shift perturbations (CSPs) were calculated using</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e2252" display="block"><mi>C</mi><mi>S</mi><mi>P</mi><mo>=</mo><msqrt><mrow><msup><mrow><mfenced close=")" open="("><mrow><mi>δ</mi><mmultiscripts><mi>H</mi><none></none><none></none><mprescripts></mprescripts><none></none><mrow><mn>1</mn></mrow></mmultiscripts><mo>⋅</mo><mn>6.5</mn></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup><mo>+</mo><msup><mrow><mfenced close=")" open="("><mrow><mi>δ</mi><mmultiscripts><mi>N</mi><none></none><none></none><mprescripts></mprescripts><none></none><mrow><mn>15</mn></mrow></mmultiscripts></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup></mrow></msqrt></math></td>
<td class="label">1</td>
</tr></table>
<p>The specific concentrations used in the different titrations are indicated in the respective figures, with the percentage of partner indicating the molar ratio of partner protein:observed protein.</p>
<p id="Par31">Residue specific <em>K</em><sub>d</sub> values were estimated by fitting the following equation to the CSPs as a function of concentration:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e2306" display="block"><mi mathvariant="normal">Δ</mi><mi>C</mi><mi>S</mi><mi>P</mi><mo>=</mo><mi mathvariant="normal">Δ</mi><msub><mrow><mi>C</mi><mi>S</mi><mi>P</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>⋅</mo><mfrac><mrow><mfenced close="]" open="["><mrow><mi>D</mi><mi>a</mi><mi>b</mi><mn>2</mn></mrow></mfenced><mo>+</mo><mfenced close="]" open="["><mrow><mi>E</mi><mi>H</mi><mi>x</mi></mrow></mfenced><mo>+</mo><mi>K</mi><mi>d</mi><mo>−</mo><msqrt><mrow><msup><mrow><mrow><mo>(</mo><mrow><mfenced close="]" open="["><mrow><mi>D</mi><mi>a</mi><mi>b</mi><mn>2</mn></mrow></mfenced><mo>+</mo><mfenced close="]" open="["><mrow><mi>E</mi><mi>H</mi><mi>x</mi></mrow></mfenced><mo>+</mo><mi>K</mi><mi>d</mi></mrow><mo>)</mo></mrow></mrow><mrow><mn>2</mn></mrow></msup><mo>−</mo><mn>4</mn><mo>⋅</mo><mfenced close="]" open="["><mrow><mi>D</mi><mi>a</mi><mi>b</mi><mn>2</mn></mrow></mfenced><mo>⋅</mo><mrow><mo>[</mo><mrow><mi>E</mi><mi>H</mi><mi>x</mi></mrow><mo>]</mo></mrow></mrow></msqrt></mrow><mrow><mn>2</mn><mo>⋅</mo><mrow><mo>[</mo><mrow><mi>D</mi><mi>a</mi><mi>b</mi><mn>2</mn></mrow><mo>]</mo></mrow></mrow></mfrac></math></td>
<td class="label">2</td>
</tr></table>
<p id="Par32">Fitting was performed with Python.</p>
<p id="Par33"><sup>15</sup>N R<sub>1ρ</sub>, R<sub>1</sub> and {<sup>1</sup>H}-<sup>15</sup>N HetNOE relaxation rates<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> were assessed at 600 MHz and 1200 MHz <sup>1</sup>H Larmor frequency. The spin-lock field for the R<sub>1ρ</sub> experiment was set to 1500 Hz or 2000 Hz for the 600 MHz and 1200 MHz magnets, respectively, and 6–7 delays, between 10 and 230 ms (for the disordered Dab2<sub>320–495</sub> and Eps15<sub>IDR</sub>) and between 10 and 70 ms (for the folded EH domains) were used to sample the decay of magnetization. The relaxation rates were determined from an exponential fit to the peak intensity relative to the delay time. Errors of the fitted rates were derived from the experimental uncertainty. Inter scan delays were typically of 1.9 s. R<sub>1</sub> was measured using 6–7 delays between 0 to 1.2 s with an inter-scan delay of typically 1.9 or 1.3 s. Inter scan delays of HetNOE experiments were 1 s.</p>
<p id="Par34">Relaxation dispersion experiments<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> were conducted at 600 and 900 MHz, employing 14 CPMG frequencies ranging from 31 to 1000 Hz, with a constant-time relaxation of 32 ms. R<sub>2</sub> uncertainty in the CPMG experiments was estimated via Monte Carlo sampling from a normal distribution based on the experimental noise. The error bars represent the 15.9th and 84.1st percentiles of the resulting R<sub>2</sub> distribution and were determined by the software ChemEx (<a href="https://github.com/gbouvignies/chemex" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/gbouvignies/chemex</a>). For the experiment with 100 µM Dab2<sub>320–495</sub> in the presence of 10% EH2, 12 residues within the helix<sub>N</sub> and the NPF motif (Val340, Asp341, Gln345, Gln346, Gln349, Ser351, Thr354, Lys356, Phe366, Phe 398, Val399, Asp426) were fit globally using a 2 site exchange model with the software ChemEx (<a href="https://github.com/gbouvignies/chemex" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/gbouvignies/chemex</a>). An exchange rate of 149 ± 13 s<sup>−1</sup> and a percentage bound of 3 ± 0.3% were obtained from the fit.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Conformational ensemble of Dab2<sub>320–495</sub>
</h3>
<p id="Par35">A conformational ensemble was calculated from Dab2<sub>320–495</sub> and Dab2<sub>328-360</sub> using a combination of the statistical coil generator, flexible meccano<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, and the genetic algorithm ASTEROIDS<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. From a statistical coil ensemble of 10,000 conformers, 200 conformations that together best described the H<sub>N</sub>, N, CO, Cα and Cβ chemical shifts of the proteins were selected. A new ensemble of 8500 conformers was generated using the Φ and ψ angles from the previous selection and supplemented with 1500 conformations from the previous pool of conformers. A new selection of 200 conformations was performed on the new pool. This iteration was repeated 4 times. Ensemble-averaged chemical shifts were generated using SPARTA<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>, and secondary chemical shifts were calculated based on RefDB<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Structures of the individual EH domains</h3>
<p id="Par36">The structures that were used to plot the CSPs for EH2 and EH3 are the PDB codes <a href="https://doi.org/10.2210/pdb1FF1/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1FF1</a><sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> and <a href="https://doi.org/10.2210/pdb1C07/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1C07</a><sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, respectively. For EH1, we use an AlphaFold2<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> prediction since at the time of writing no <em>human</em> Eps15 EH1 structure was deposited in the PDB. The CSPs of each EH domain with 100 μM Dab2<sub>320–495</sub> (100%) were plotted onto the structures. The linear gradient is from 0%, which represents no CSP, up to 100%, which is the highest CSP observed at 0.75 ppm. The structures were visualized using Pymol.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Protein labeling with fluorescent dyes</h3>
<p id="Par37">Protein labeling was performed using malemeide dyes; AZDye488 (Vectorlabs, CA, USA), for Dab2<sub>320–495 S328C</sub> and AZDye594 (Vectorlabs, CA, USA) for Eps15 at random cysteine side chains, essentially as described previously<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>,<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. After adding 10 mM DTT overnight to fully reduce the proteins, they were dialyzed into a phosphate-buffer (50 mM Na-phosphate pH 7, 150 mM NaCl) for 2 × 1 h at room temperature. At least 5 times molar excess of the dyes was used for the labeling reaction, which, after mixing, contained a maximum of 10% V/V DMSO. The protein/dye mixtures were allowed to react for 1 h at room temperature and incubated overnight at 4 °C. To stop the reaction, 10 mM of DTT were added before injecting the mixtures on a SEC70 or 650 equilibrated in NMR buffer to remove any unconjugated dye.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Protein droplets and microscopy imaging</h3>
<p id="Par38">All droplet formation assays were performed in a buffer of 50 mM Na-phosphate pH 6, 150 mM NaCl, and 2 mM DTT, 3% w/v PEG8000, using different ratios of Eps15 (0.5 µM to 10 µM) and Dab2<sub>320–495</sub> (0 µM to 140 µM). For both proteins, a ratio of 10:1 of non-labeled: fluorescently labeled protein was used. A total sample volume of 20 µL was placed in 8-well polystyrene chambers with 1.5 borosilicate coverglass (Nunc Lab-Tek). Epi-fluorescence imaging was performed with a Nikon Ti Eclipse microscope with a 60x (PLAN APO, NA:1.40, WD 0.13 mm) oil immersion objective. The setup was controlled by the imaging software NIS (Nikon). Confocal microscopy was performed with the LSM780, and the setup was controlled by Zeiss Zen Black software. Dab2<sub>320–495 S328C</sub> AZDye488 (Ex.: 488 nm; EmF.: 490–535 nm) and Eps15 AZDye594 (Ex.: 561<strong>;</strong> EmF.: 569–631 nm) were imaged. A PL APO DIC M27 63 × /1.40 NA oil objective (Carl Zeiss Microscopy) at a zoom factor of 1 or 3 and a line average of 1 was used, acquiring images of 512 × 512/1024 × 1024 pixels, respectively. Microscopy images were imported and analyzed using ImageJ/Fiji<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. The intensity ratio of the droplets versus background was calculated by manually selecting droplets and comparing the intensity to an identical area of the background. The final ratio was calculated as an average over all droplets picked across the different images for each condition. The number and size of the droplets were determined by applying an Otsu threshold to each image. The number of droplets was counted in each individual image for each condition, while the size of the droplets was calculated as an average of droplet size across all images of each condition. For the determination of the droplet size and count as well as the partitioning of Eps15 and Dab2<sub>320–495</sub> into droplets, images at a zoom factor of 1 (512 × 512 pixels) were used, and the analyses from ImageJ/Fiji were plotted and visualized using Python. The number of images included in the analysis for each condition can be found in Fig. <a href="#Fig6" class="usa-link">6</a>. The images were acquired at different spatial positions of the same samples, respectively.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par39">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370899/bin/41467_2025_63090_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (10.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370899/bin/41467_2025_63090_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370899/bin/41467_2025_63090_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (2.2MB, pdf) </sup>
</div></div></section></section><section id="Sec25"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370899/bin/41467_2025_63090_MOESM4_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (1.6MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank all members of the Milles group for fruitful discussions and critical proofreading. We thank M.R. Jensen for providing the pET28-6His-GB1 plasmid, M. Blackledge and M.R. Jensen for providing NMR analysis scripts, P. Schmieder, N. Trieloff and M. Beerbaum for technical assistance on the NMR spectrometers and T. Soykan, M. Biek and M. Lehmann from the Imaging Facility of FMP for technical support and assistance with image analysis. The Institut de Biologie Structurale acknowledges integration into the Interdisciplinary Research Institute of Grenoble. This work was supported by the Leibniz-Forschungsinstitut für Molekulare Pharmakologie (FMP) (to S.M.). This project has received funding from the European Research Council (ERC) Starting Grant MultiMotif to S.M. under the European Union’s Horizon 2020 research and innovation program (grant agreement no. 802209).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>S.M., A.P., and I.M.V. designed the research. A.P., I.M.V., K.M., M.T., and A.S. performed research. A.P., I.M.V., and S.M. analyzed the data. A.P., I.M.V., and S.M. wrote the paper.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par40"><em>Nature Communications</em> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Funding</h2>
<p>Open Access funding enabled and organized by Projekt DEAL.</p></section><section id="notes4"><h2 class="pmc_sec_title">Data availability</h2>
<p>All study data are included in the article, supporting information, the Source Data files and/or can be obtained from the corresponding author upon request. The chemical shift assignments generated in this study have been deposited in the Biological Magnetic Resonance Data Bank (BMRB) under the accession numbers <a href="https://bmrb.io/data_library/summary/index.php?bmrbId=52613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">52613</a> (Dab2<sub>320–495</sub>), <a href="https://bmrb.io/data_library/summary/index.php?bmrbId=52866" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">52866</a> (Eps15<sub>IDR 481-581</sub>), <a href="https://bmrb.io/data_library/summary/index.php?bmrbId=52864" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">52864</a> (Eps15<sub>IDR 569-671</sub>), <a href="https://bmrb.io/data_library/summary/index.php?bmrbId=52863" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">52863</a> (Eps15<sub>IDR 648-780</sub>), and <a href="https://bmrb.io/data_library/summary/index.php?bmrbId=52867" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">52867</a> (Eps15<sub>IDR 761-896</sub>), respectively. The NMR and imaging statistics generated in this study are provided in the Source Data files. The PDB entries of EH2 and EH3 used in this work are: <a href="https://doi.org/10.2210/pdb1FF1/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1FF1</a> (EH2) and <a href="https://doi.org/10.2210/pdb1C07/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1C07</a> (EH3).  <a href="#Sec25" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par41">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Andromachi Papagiannoula, Ida Marie Vedel.</p></div>
</div></section><section id="sec26"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63090-1.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Kaksonen, M. &amp; Roux, A. Mechanisms of clathrin-mediated endocytosis. <em>Nat. Rev. Mol. Cell Biol.</em><strong>19</strong>, 313–326 (2018).
</cite> [<a href="https://doi.org/10.1038/nrm.2017.132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29410531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaksonen,%20M.%20&amp;%20Roux,%20A.%20Mechanisms%20of%20clathrin-mediated%20endocytosis.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.19,%20313%E2%80%93326%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Maldonado-Báez, L. &amp; Wendland, B. Endocytic adaptors: recruiters, coordinators and regulators. <em>Trends Cell Biol.</em><strong>16</strong>, 505–513 (2006).
</cite> [<a href="https://doi.org/10.1016/j.tcb.2006.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16935508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maldonado-B%C3%A1ez,%20L.%20&amp;%20Wendland,%20B.%20Endocytic%20adaptors:%20recruiters,%20coordinators%20and%20regulators.%20Trends%20Cell%20Biol.16,%20505%E2%80%93513%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Brett, T. J., Traub, L. M. &amp; Fremont, D. H. Accessory protein recruitment motifs in clathrin-mediated endocytosis. <em>Structure</em><strong>10</strong>, 797–809 (2002).
</cite> [<a href="https://doi.org/10.1016/s0969-2126(02)00784-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12057195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brett,%20T.%20J.,%20Traub,%20L.%20M.%20&amp;%20Fremont,%20D.%20H.%20Accessory%20protein%20recruitment%20motifs%20in%20clathrin-mediated%20endocytosis.%20Structure10,%20797%E2%80%93809%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Van Roey, K. et al. Short linear motifs: ubiquitous and functionally diverse protein interaction modules directing cell regulation. <em>Chem. Rev.</em><strong>114</strong>, 6733–6778 (2014).
</cite> [<a href="https://doi.org/10.1021/cr400585q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24926813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Roey,%20K.%20et%20al.%20Short%20linear%20motifs:%20ubiquitous%20and%20functionally%20diverse%20protein%20interaction%20modules%20directing%20cell%20regulation.%20Chem.%20Rev.114,%206733%E2%80%936778%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Schmid, E. M. et al. Role of the AP2 beta-appendage hub in recruiting partners for clathrin-coated vesicle assembly. <em>PLoS Biol.</em><strong>4</strong>, e262 (2006).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.0040262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1540706/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16903783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmid,%20E.%20M.%20et%20al.%20Role%20of%20the%20AP2%20beta-appendage%20hub%20in%20recruiting%20partners%20for%20clathrin-coated%20vesicle%20assembly.%20PLoS%20Biol.4,%20e262%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Praefcke, G. J. K. et al. Evolving nature of the AP2 alpha-appendage hub during clathrin-coated vesicle endocytosis. <em>EMBO J.</em><strong>23</strong>, 4371–4383 (2004).
</cite> [<a href="https://doi.org/10.1038/sj.emboj.7600445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC526462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15496985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Praefcke,%20G.%20J.%20K.%20et%20al.%20Evolving%20nature%20of%20the%20AP2%20alpha-appendage%20hub%20during%20clathrin-coated%20vesicle%20endocytosis.%20EMBO%20J.23,%204371%E2%80%934383%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Naudi-Fabra, S. et al. An extended interaction site determines binding between AP180 and AP2 in clathrin-mediated endocytosis. <em>Nat. Commun.</em><strong>15</strong>, 5884 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-50212-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11246429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39003270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naudi-Fabra,%20S.%20et%20al.%20An%20extended%20interaction%20site%20determines%20binding%20between%20AP180%20and%20AP2%20in%20clathrin-mediated%20endocytosis.%20Nat.%20Commun.15,%205884%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zhuo, Y. et al. Dynamic interactions between clathrin and locally structured elements in a disordered protein mediate clathrin lattice assembly. <em>J. Mol. Biol.</em><strong>404</strong>, 274–290 (2010).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2010.09.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2981644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20875424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhuo,%20Y.%20et%20al.%20Dynamic%20interactions%20between%20clathrin%20and%20locally%20structured%20elements%20in%20a%20disordered%20protein%20mediate%20clathrin%20lattice%20assembly.%20J.%20Mol.%20Biol.404,%20274%E2%80%93290%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Zhuo, Y. et al. Nuclear magnetic resonance structural mapping reveals promiscuous interactions between clathrin-box motif sequences and the n-terminal domain of the clathrin heavy chain. <em>Biochemistry</em><strong>54</strong>, 2571–2580 (2015).
</cite> [<a href="https://doi.org/10.1021/acs.biochem.5b00065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4429812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25844500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhuo,%20Y.%20et%20al.%20Nuclear%20magnetic%20resonance%20structural%20mapping%20reveals%20promiscuous%20interactions%20between%20clathrin-box%20motif%20sequences%20and%20the%20n-terminal%20domain%20of%20the%20clathrin%20heavy%20chain.%20Biochemistry54,%202571%E2%80%932580%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Henne, W. M. et al. FCHo proteins are nucleators of clathrin-mediated endocytosis. <em>Science</em><strong>328</strong>, 1281–1284 (2010).
</cite> [<a href="https://doi.org/10.1126/science.1188462" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2883440/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20448150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Henne,%20W.%20M.%20et%20al.%20FCHo%20proteins%20are%20nucleators%20of%20clathrin-mediated%20endocytosis.%20Science328,%201281%E2%80%931284%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Confalonieri, S. &amp; Di Fiore, P. P. The Eps15 homology (EH) domain. <em>FEBS Lett.</em><strong>513</strong>, 24–29 (2002).
</cite> [<a href="https://doi.org/10.1016/s0014-5793(01)03241-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11911876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Confalonieri,%20S.%20&amp;%20Di%20Fiore,%20P.%20P.%20The%20Eps15%20homology%20(EH)%20domain.%20FEBS%20Lett.513,%2024%E2%80%9329%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Tebar, F., Confalonieri, S., Carter, R. E., Di Fiore, P. P. &amp; Sorkin, A. Eps15 is constitutively oligomerized due to homophilic interaction of its coiled-coil region*. <em>J. Biol. Chem.</em><strong>272</strong>, 15413–15418 (1997).
</cite> [<a href="https://doi.org/10.1074/jbc.272.24.15413" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9182572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tebar,%20F.,%20Confalonieri,%20S.,%20Carter,%20R.%20E.,%20Di%20Fiore,%20P.%20P.%20&amp;%20Sorkin,%20A.%20Eps15%20is%20constitutively%20oligomerized%20due%20to%20homophilic%20interaction%20of%20its%20coiled-coil%20region*.%20J.%20Biol.%20Chem.272,%2015413%E2%80%9315418%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Cupers, P., ter Haar, E., Boll, W. &amp; Kirchhausen, T. Parallel dimers and anti-parallel tetramers formed by epidermal growth factor receptor pathway substrate clone 15. <em>J. Biol. Chem.</em><strong>272</strong>, 33430–33434 (1997).
</cite> [<a href="https://doi.org/10.1074/jbc.272.52.33430" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9407139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cupers,%20P.,%20ter%20Haar,%20E.,%20Boll,%20W.%20&amp;%20Kirchhausen,%20T.%20Parallel%20dimers%20and%20anti-parallel%20tetramers%20formed%20by%20epidermal%20growth%20factor%20receptor%20pathway%20substrate%20clone%2015.%20J.%20Biol.%20Chem.272,%2033430%E2%80%9333434%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>van Bergen en Henegouwen, P. M. Eps15: a multifunctional adaptor protein regulating intracellular trafficking. <em>Cell Commun. Signal.</em><strong>7</strong>, 24 (2009).
</cite> [<a href="https://doi.org/10.1186/1478-811X-7-24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2767346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19814798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20Bergen%20en%20Henegouwen,%20P.%20M.%20Eps15:%20a%20multifunctional%20adaptor%20protein%20regulating%20intracellular%20trafficking.%20Cell%20Commun.%20Signal.7,%2024%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Reider, A. et al. Syp1 is a conserved endocytic adaptor that contains domains involved in cargo selection and membrane tubulation. <em>EMBO J.</em><strong>28</strong>, 3103–3116 (2009).
</cite> [<a href="https://doi.org/10.1038/emboj.2009.248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2771086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19713939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reider,%20A.%20et%20al.%20Syp1%20is%20a%20conserved%20endocytic%20adaptor%20that%20contains%20domains%20involved%20in%20cargo%20selection%20and%20membrane%20tubulation.%20EMBO%20J.28,%203103%E2%80%933116%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Day, K. J. et al. Liquid-like protein interactions catalyse assembly of endocytic vesicles. <em>Nat. Cell Biol.</em><strong>23</strong>, 366–376 (2021).
</cite> [<a href="https://doi.org/10.1038/s41556-021-00646-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8035231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33820972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Day,%20K.%20J.%20et%20al.%20Liquid-like%20protein%20interactions%20catalyse%20assembly%20of%20endocytic%20vesicles.%20Nat.%20Cell%20Biol.23,%20366%E2%80%93376%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Salcini, A. E. et al. Binding specificity and in vivo targets of the EH domain, a novel protein–protein interaction module. <em>Genes Dev.</em><strong>11</strong>, 2239–2249 (1997).
</cite> [<a href="https://doi.org/10.1101/gad.11.17.2239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC275390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9303539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Salcini,%20A.%20E.%20et%20al.%20Binding%20specificity%20and%20in%20vivo%20targets%20of%20the%20EH%20domain,%20a%20novel%20protein%E2%80%93protein%20interaction%E2%80%89module.%20Genes%20Dev.11,%202239%E2%80%932249%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Rumpf, J. et al. Structure of the Eps15-stonin2 complex provides a molecular explanation for EH-domain ligand specificity. <em>EMBO J.</em><strong>27</strong>, 558–569 (2008).
</cite> [<a href="https://doi.org/10.1038/sj.emboj.7601980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2241652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18200045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rumpf,%20J.%20et%20al.%20Structure%20of%20the%20Eps15-stonin2%20complex%20provides%20a%20molecular%20explanation%20for%20EH-domain%20ligand%20specificity.%20EMBO%20J.27,%20558%E2%80%93569%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Teckchandani, A., Mulkearns, E. E., Randolph, T. W., Toida, N. &amp; Cooper, J. A. The clathrin adaptor Dab2 recruits EH domain scaffold proteins to regulate integrin β1 endocytosis. <em>Mol. Biol. Cell</em><strong>23</strong>, 2905–2916 (2012).
</cite> [<a href="https://doi.org/10.1091/mbc.E11-12-1007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3408417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22648170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teckchandani,%20A.,%20Mulkearns,%20E.%20E.,%20Randolph,%20T.%20W.,%20Toida,%20N.%20&amp;%20Cooper,%20J.%20A.%20The%20clathrin%20adaptor%20Dab2%20recruits%20EH%20domain%20scaffold%20proteins%20to%20regulate%20integrin%20%CE%B21%20endocytosis.%20Mol.%20Biol.%20Cell23,%202905%E2%80%932916%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Finkielstein, C. V. &amp; Capelluto, D. G. S. Disabled-2: A modular scaffold protein with multifaceted functions in signaling. <em>Cell</em><strong>1</strong>, 48–58 (2016).</cite> [<a href="https://doi.org/10.1002/bies.201670907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27417122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Finkielstein,%20C.%20V.%20&amp;%20Capelluto,%20D.%20G.%20S.%20Disabled-2:%20A%20modular%20scaffold%20protein%20with%20multifaceted%20functions%20in%20signaling.%20Cell1,%2048%E2%80%9358%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Smith, S. M., Baker, M., Halebian, M. &amp; Smith, C. J. Weak molecular interactions in clathrin-mediated endocytosis. Front. Mol. Biosci. 4, 72 (2017).
</cite> [<a href="https://doi.org/10.3389/fmolb.2017.00072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5694535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29184887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20S.%20M.,%20Baker,%20M.,%20Halebian,%20M.%20&amp;%20Smith,%20C.%20J.%20Weak%20molecular%20interactions%20in%20clathrin-mediated%20endocytosis.%20Front.%20Mol.%20Biosci.%204,%2072%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Takei, K. &amp; Haucke, V. Clathrin-mediated endocytosis: membrane factors pull the trigger. <em>Trends Cell Biol.</em><strong>11</strong>, 385–391 (2001).
</cite> [<a href="https://doi.org/10.1016/s0962-8924(01)02082-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11514193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takei,%20K.%20&amp;%20Haucke,%20V.%20Clathrin-mediated%20endocytosis:%20membrane%20factors%20pull%20the%20trigger.%20Trends%20Cell%20Biol.11,%20385%E2%80%93391%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Erdős, G. &amp; Dosztányi, Z. Analyzing protein disorder with IUPred2A. <em>Curr. Protoc. Bioinforma.</em><strong>70</strong>, e99 (2020).</cite> [<a href="https://doi.org/10.1002/cpbi.99" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32237272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Erd%C5%91s,%20G.%20&amp;%20Doszt%C3%A1nyi,%20Z.%20Analyzing%20protein%20disorder%20with%20IUPred2A.%20Curr.%20Protoc.%20Bioinforma.70,%20e99%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Ozenne, V. et al. Flexible-Meccano: a tool for the generation of explicit ensemble descriptions of intrinsically disordered proteins and their associated experimental observables. <em>Bioinformatics</em><strong>28</strong>, 1463–1470 (2012).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22613562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ozenne,%20V.%20et%20al.%20Flexible-Meccano:%20a%20tool%20for%20the%20generation%20of%20explicit%20ensemble%20descriptions%20of%20intrinsically%20disordered%20proteins%20and%20their%20associated%20experimental%20observables.%20Bioinformatics28,%201463%E2%80%931470%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Jensen, M. R., Salmon, L., Nodet, G. &amp; Blackledge, M. Defining conformational ensembles of intrinsically disordered and partially folded proteins directly from chemical shifts. <em>J. Am. Chem. Soc.</em><strong>132</strong>, 1270–1272 (2010).
</cite> [<a href="https://doi.org/10.1021/ja909973n" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20063887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jensen,%20M.%20R.,%20Salmon,%20L.,%20Nodet,%20G.%20&amp;%20Blackledge,%20M.%20Defining%20conformational%20ensembles%20of%20intrinsically%20disordered%20and%20partially%20folded%20proteins%20directly%20from%20chemical%20shifts.%20J.%20Am.%20Chem.%20Soc.132,%201270%E2%80%931272%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Rumpf, J., Simon, B., Groemping, Y. &amp; Sattler, M. 1H, 13C, and 15N chemical shift assignments for the Eps15-EH2-stonin 2 complex. <em>Biomol. NMR Assign.</em><strong>2</strong>, 55–58 (2008).
</cite> [<a href="https://doi.org/10.1007/s12104-008-9083-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19636924/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rumpf,%20J.,%20Simon,%20B.,%20Groemping,%20Y.%20&amp;%20Sattler,%20M.%201H,%2013C,%20and%2015N%20chemical%20shift%20assignments%20for%20the%20Eps15-EH2-stonin%202%20complex.%20Biomol.%20NMR%20Assign.2,%2055%E2%80%9358%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Carotenuto, A. et al. Sequence-specific 1H, 13C and 15N assignment and secondary structure of the apo EH2 domain of mouse Eps15. <em>J. Biomol. NMR</em><strong>14</strong>, 97–98 (1999).
</cite> [<a href="https://doi.org/10.1023/a:1008394326526" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10382315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carotenuto,%20A.%20et%20al.%20Sequence-specific%201H,%2013C%20and%2015N%20assignment%20and%20secondary%20structure%20of%20the%20apo%20EH2%20domain%20of%20mouse%20Eps15.%20J.%20Biomol.%20NMR14,%2097%E2%80%9398%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Whitehead, B. et al. Sequence-specific 1H, 13C and 15N assignment of the EH1 domain of mouse Eps15. <em>J. Biomol. NMR</em><strong>12</strong>, 465–466 (1998).
</cite> [<a href="https://doi.org/10.1023/a:1008382305918" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9835057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whitehead,%20B.%20et%20al.%20Sequence-specific%201H,%2013C%20and%2015N%20assignment%20of%20the%20EH1%20domain%20of%20mouse%20Eps15.%20J.%20Biomol.%20NMR12,%20465%E2%80%93466%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Marsh, J. A., Singh, V. K., Jia, Z. &amp; Forman-Kay, J. D. Sensitivity of secondary structure propensities to sequence differences between α- and γ-synuclein: implications for fibrillation. <em>Protein Sci.</em><strong>15</strong>, 2795–2804 (2006).
</cite> [<a href="https://doi.org/10.1110/ps.062465306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2242444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17088319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marsh,%20J.%20A.,%20Singh,%20V.%20K.,%20Jia,%20Z.%20&amp;%20Forman-Kay,%20J.%20D.%20Sensitivity%20of%20secondary%20structure%20propensities%20to%20sequence%20differences%20between%20%CE%B1-%20and%20%CE%B3-synuclein:%20implications%20for%20fibrillation.%20Protein%20Sci.15,%202795%E2%80%932804%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Whitehead, B., Tessari, M., Carotenuto, A., van Bergen en Henegouwen, P. M. P. &amp; Vuister, G. W. The EH1 domain of Eps15 is structurally classified as a member of the s100 subclass of EF-hand-containing proteins. <em>Biochemistry</em><strong>38</strong>, 11271–11277 (1999).
</cite> [<a href="https://doi.org/10.1021/bi990922i" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10471276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whitehead,%20B.,%20Tessari,%20M.,%20Carotenuto,%20A.,%20van%20Bergen%20en%20Henegouwen,%20P.%20M.%20P.%20&amp;%20Vuister,%20G.%20W.%20The%20EH1%20domain%20of%20Eps15%20is%20structurally%20classified%20as%20a%20member%20of%20the%20s100%20subclass%20of%20EF-hand-containing%20proteins.%20Biochemistry38,%2011271%E2%80%9311277%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>de Beer, T. et al. Molecular mechanism of NPF recognition by EH domains. <em>Nat. Struct. Biol.</em><strong>7</strong>, 1018–1022 (2000).
</cite> [<a href="https://doi.org/10.1038/80924" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11062555/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Beer,%20T.%20et%20al.%20Molecular%20mechanism%20of%20NPF%20recognition%20by%20EH%20domains.%20Nat.%20Struct.%20Biol.7,%201018%E2%80%931022%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Enmon, J. L., de Beer, T. &amp; Overduin, M. Solution structure of Eps15’s third EH domain reveals coincident Phe−Trp and Asn−Pro−Phe binding sites. <em>Biochemistry</em><strong>39</strong>, 4309–4319 (2000).
</cite> [<a href="https://doi.org/10.1021/bi9927383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10757979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Enmon,%20J.%20L.,%20de%20Beer,%20T.%20&amp;%20Overduin,%20M.%20Solution%20structure%20of%20Eps15%E2%80%99s%20third%20EH%20domain%20reveals%20coincident%20Phe%E2%88%92Trp%20and%20Asn%E2%88%92Pro%E2%88%92Phe%20binding%20sites.%20Biochemistry39,%204309%E2%80%934319%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Bae, S.-H., Dyson, H. J. &amp; Wright, P. E. Prediction of the rotational tumbling time for proteins with disordered segments. <em>J. Am. Chem. Soc.</em><strong>131</strong>, 6814–6821 (2009).
</cite> [<a href="https://doi.org/10.1021/ja809687r" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2694746/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19391622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bae,%20S.-H.,%20Dyson,%20H.%20J.%20&amp;%20Wright,%20P.%20E.%20Prediction%20of%20the%20rotational%20tumbling%20time%20for%20proteins%20with%20disordered%20segments.%20J.%20Am.%20Chem.%20Soc.131,%206814%E2%80%936821%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <em>Nature</em><strong>596</strong>, 583–589 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumper,%20J.%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold.%20Nature596,%20583%E2%80%93589%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Klapisz, E. et al. A ubiquitin-interacting motif (UIM) is essential for Eps15 and Eps15R ubiquitination. <em>J. Biol. Chem.</em><strong>277</strong>, 30746–30753 (2002).
</cite> [<a href="https://doi.org/10.1074/jbc.M203004200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12072436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Klapisz,%20E.%20et%20al.%20A%20ubiquitin-interacting%20motif%20(UIM)%20is%20essential%20for%20Eps15%20and%20Eps15R%20ubiquitination.%20J.%20Biol.%20Chem.277,%2030746%E2%80%9330753%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Hofmann, K. &amp; Falquet, L. A ubiquitin-interacting motif conserved in components of the proteasomal and lysosomal protein degradation systems. <em>Trends Biochem Sci.</em><strong>26</strong>, 347–350 (2001).
</cite> [<a href="https://doi.org/10.1016/s0968-0004(01)01835-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11406394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hofmann,%20K.%20&amp;%20Falquet,%20L.%20A%20ubiquitin-interacting%20motif%20conserved%20in%20components%20of%20the%20proteasomal%20and%20lysosomal%20protein%20degradation%20systems.%20Trends%20Biochem%20Sci.26,%20347%E2%80%93350%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Abyzov, A. et al. Identification of dynamic modes in an intrinsically disordered protein using temperature-dependent NMR relaxation. <em>J. Am. Chem. Soc.</em><strong>138</strong>, 6240–6251 (2016).
</cite> [<a href="https://doi.org/10.1021/jacs.6b02424" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27112095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abyzov,%20A.%20et%20al.%20Identification%20of%20dynamic%20modes%20in%20an%20intrinsically%20disordered%20protein%20using%20temperature-dependent%20NMR%20relaxation.%20J.%20Am.%20Chem.%20Soc.138,%206240%E2%80%936251%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Fung, H. Y. J., Birol, M. &amp; Rhoades, E. IDPs in macromolecular complexes: the roles of multivalent interactions in diverse assemblies. <em>Curr. Opin. Struct. Biol.</em><strong>49</strong>, 36–43 (2018).
</cite> [<a href="https://doi.org/10.1016/j.sbi.2017.12.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5915967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29306779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fung,%20H.%20Y.%20J.,%20Birol,%20M.%20&amp;%20Rhoades,%20E.%20IDPs%20in%20macromolecular%20complexes:%20the%20roles%20of%20multivalent%20interactions%20in%20diverse%20assemblies.%20Curr.%20Opin.%20Struct.%20Biol.49,%2036%E2%80%9343%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Duan, D., Hanson, M., Holland, D. O. &amp; Johnson, M. E. Integrating protein copy numbers with interaction networks to quantify stoichiometry in clathrin-mediated endocytosis. <em>Sci. Rep.</em><strong>12</strong>, 5413 (2022).
</cite> [<a href="https://doi.org/10.1038/s41598-022-09259-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8967901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35354856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duan,%20D.,%20Hanson,%20M.,%20Holland,%20D.%20O.%20&amp;%20Johnson,%20M.%20E.%20Integrating%20protein%20copy%20numbers%20with%20interaction%20networks%20to%20quantify%20stoichiometry%20in%20clathrin-mediated%20endocytosis.%20Sci.%20Rep.12,%205413%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Paoluzi, S. et al. Recognition specificity of individual EH domains of mammals and yeast. <em>EMBO J.</em><strong>17</strong>, 6541–6550 (1998).
</cite> [<a href="https://doi.org/10.1093/emboj/17.22.6541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1171001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9822599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paoluzi,%20S.%20et%20al.%20Recognition%20specificity%20of%20individual%20EH%20domains%20of%20mammals%20and%20yeast.%20EMBO%20J.17,%206541%E2%80%936550%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Kieken, F. et al. Structural insight into the interaction of proteins containing NPF, DPF, and GPF motifs with the C-terminal EH-domain of EHD1. <em>Protein Sci.</em><strong>18</strong>, 2471–2479 (2009).
</cite> [<a href="https://doi.org/10.1002/pro.258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2821266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19798736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kieken,%20F.%20et%20al.%20Structural%20insight%20into%20the%20interaction%20of%20proteins%20containing%20NPF,%20DPF,%20and%20GPF%20motifs%20with%20the%20C-terminal%20EH-domain%20of%20EHD1.%20Protein%20Sci.18,%202471%E2%80%932479%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Santonico, E., Panni, S., Falconi, M., Castagnoli, L. &amp; Cesareni, G. Binding to DPF-motif by the POB1 EH domain is responsible for POB1-Eps15 interaction. <em>BMC Biochem.</em><strong>8</strong>, 29 (2007).
</cite> [<a href="https://doi.org/10.1186/1471-2091-8-29" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2238750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18154663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santonico,%20E.,%20Panni,%20S.,%20Falconi,%20M.,%20Castagnoli,%20L.%20&amp;%20Cesareni,%20G.%20Binding%20to%20DPF-motif%20by%20the%20POB1%20EH%20domain%20is%20responsible%20for%20POB1-Eps15%20interaction.%20BMC%20Biochem.8,%2029%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Daumke, O. et al. Architectural and mechanistic insights into an EHD ATPase involved in membrane remodelling. <em>Nature</em><strong>449</strong>, 923–927 (2007).
</cite> [<a href="https://doi.org/10.1038/nature06173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17914359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daumke,%20O.%20et%20al.%20Architectural%20and%20mechanistic%20insights%20into%20an%20EHD%20ATPase%20involved%20in%20membrane%20remodelling.%20Nature449,%20923%E2%80%93927%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Martínez-Lumbreras, S. et al. Intramolecular autoinhibition regulates the selectivity of PRPF40A tandem WW domains for proline-rich motifs. <em>Nat. Commun.</em><strong>15</strong>, 3888 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-48004-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11079029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38719828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mart%C3%ADnez-Lumbreras,%20S.%20et%20al.%20Intramolecular%20autoinhibition%20regulates%20the%20selectivity%20of%20PRPF40A%20tandem%20WW%20domains%20for%20proline-rich%20motifs.%20Nat.%20Commun.15,%203888%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Yuan, F. et al. Ubiquitin-driven protein condensation stabilizes clathrin-mediated endocytosis. <em>PNAS Nexus</em><strong>3</strong>, pgae342 (2024).
</cite> [<a href="https://doi.org/10.1093/pnasnexus/pgae342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11382290/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39253396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan,%20F.%20et%20al.%20Ubiquitin-driven%20protein%20condensation%20stabilizes%20clathrin-mediated%20endocytosis.%20PNAS%20Nexus3,%20pgae342%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Martin, E. W. et al. Valence and patterning of aromatic residues determine the phase behavior of prion-like domains. <em>Science</em><strong>367</strong>, 694–699 (2020).
</cite> [<a href="https://doi.org/10.1126/science.aaw8653" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7297187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32029630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martin,%20E.%20W.%20et%20al.%20Valence%20and%20patterning%20of%20aromatic%20residues%20determine%20the%20phase%20behavior%20of%20prion-like%20domains.%20Science367,%20694%E2%80%93699%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Sochacki, K. A., Dickey, A. M., Strub, M.-P. &amp; Taraska, J. W. Endocytic proteins are partitioned at the edge of the clathrin lattice in mammalian cells. <em>Nat. Cell Biol.</em><strong>19</strong>, 352–361 (2017).
</cite> [<a href="https://doi.org/10.1038/ncb3498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7509982/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28346440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sochacki,%20K.%20A.,%20Dickey,%20A.%20M.,%20Strub,%20M.-P.%20&amp;%20Taraska,%20J.%20W.%20Endocytic%20proteins%20are%20partitioned%20at%20the%20edge%20of%20the%20clathrin%20lattice%20in%20mammalian%20cells.%20Nat.%20Cell%20Biol.19,%20352%E2%80%93361%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Taylor, M. J., Perrais, D. &amp; Merrifield, C. J. A high precision survey of the molecular dynamics of mammalian clathrin-mediated endocytosis. <em>PLOS Biol.</em><strong>9</strong>, e1000604 (2011).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.1000604" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3062526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21445324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taylor,%20M.%20J.,%20Perrais,%20D.%20&amp;%20Merrifield,%20C.%20J.%20A%20high%20precision%20survey%20of%20the%20molecular%20dynamics%20of%20mammalian%20clathrin-mediated%20endocytosis.%20PLOS%20Biol.9,%20e1000604%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Gasteiger, E. et al. Protein Identification and Analysis Tools on the ExPASy Server. in The Proteomics Protocols Handbook (ed. Walker, J. M.) 571–607 (Humana Press, Totowa, NJ, 2005).</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>Delaglio, F. et al. NMRPipe: a multidimensional spectral processing system based on UNIX pipes. <em>J. Biomol. NMR</em><strong>6</strong>, 277–293 (1995).
</cite> [<a href="https://doi.org/10.1007/BF00197809" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8520220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Delaglio,%20F.%20et%20al.%20NMRPipe:%20a%20multidimensional%20spectral%20processing%20system%20based%20on%20UNIX%20pipes.%20J.%20Biomol.%20NMR6,%20277%E2%80%93293%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Kazimierczuk, K. &amp; Orekhov, V. Yu. Accelerated NMR spectroscopy by using compressed sensing. <em>Angew. Chem. Int. Ed.</em><strong>50</strong>, 5556–5559 (2011).</cite> [<a href="https://doi.org/10.1002/anie.201100370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21538743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kazimierczuk,%20K.%20&amp;%20Orekhov,%20V.%20Yu.%20Accelerated%20NMR%20spectroscopy%20by%20using%20compressed%20sensing.%20Angew.%20Chem.%20Int.%20Ed.50,%205556%E2%80%935559%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Skinner, S. P. et al. CcpNmr AnalysisAssign: a flexible platform for integrated NMR analysis. <em>J. Biomol. NMR</em><strong>66</strong>, 111–124 (2016).
</cite> [<a href="https://doi.org/10.1007/s10858-016-0060-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5095159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27663422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Skinner,%20S.%20P.%20et%20al.%20CcpNmr%20AnalysisAssign:%20a%20flexible%20platform%20for%20integrated%20NMR%20analysis.%20J.%20Biomol.%20NMR66,%20111%E2%80%93124%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Favier, A. &amp; Brutscher, B. NMRlib: user-friendly pulse sequence tools for Bruker NMR spectrometers. <em>J. Biomol. NMR</em><strong>73</strong>, 199–211 (2019).
</cite> [<a href="https://doi.org/10.1007/s10858-019-00249-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31076970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Favier,%20A.%20&amp;%20Brutscher,%20B.%20NMRlib:%20user-friendly%20pulse%20sequence%20tools%20for%20Bruker%20NMR%20spectrometers.%20J.%20Biomol.%20NMR73,%20199%E2%80%93211%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Solyom, Z. et al. BEST-TROSY experiments for time-efficient sequential resonance assignment of large disordered proteins. <em>J. Biomol. NMR</em><strong>55</strong>, 311–321 (2013).
</cite> [<a href="https://doi.org/10.1007/s10858-013-9715-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23435576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Solyom,%20Z.%20et%20al.%20BEST-TROSY%20experiments%20for%20time-efficient%20sequential%20resonance%20assignment%20of%20large%20disordered%20proteins.%20J.%20Biomol.%20NMR55,%20311%E2%80%93321%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Jung, Y.-S. &amp; Zweckstetter, M. Mars - robust automatic backbone assignment of proteins. <em>J. Biomol. NMR</em><strong>30</strong>, 11–23 (2004).
</cite> [<a href="https://doi.org/10.1023/B:JNMR.0000042954.99056.ad" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15452431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jung,%20Y.-S.%20&amp;%20Zweckstetter,%20M.%20Mars%20-%20robust%20automatic%20backbone%20assignment%20of%20proteins.%20J.%20Biomol.%20NMR30,%2011%E2%80%9323%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Zhang, H., Neal, S. &amp; Wishart, D. S. RefDB: a database of uniformly referenced protein chemical shifts. <em>J. Biomol. NMR</em><strong>25</strong>, 173–195 (2003).
</cite> [<a href="https://doi.org/10.1023/a:1022836027055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12652131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20H.,%20Neal,%20S.%20&amp;%20Wishart,%20D.%20S.%20RefDB:%20a%20database%20of%20uniformly%20referenced%20protein%20chemical%20shifts.%20J.%20Biomol.%20NMR25,%20173%E2%80%93195%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Lakomek, N.-A., Ying, J. &amp; Bax, A. Measurement of 15N relaxation rates in perdeuterated proteins by TROSY-based methods. <em>J. Biomol. NMR</em><strong>53</strong>, 209–221 (2012).
</cite> [<a href="https://doi.org/10.1007/s10858-012-9626-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3412688/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22689066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lakomek,%20N.-A.,%20Ying,%20J.%20&amp;%20Bax,%20A.%20Measurement%20of%2015N%20relaxation%20rates%20in%20perdeuterated%20proteins%20by%20TROSY-based%20methods.%20J.%20Biomol.%20NMR53,%20209%E2%80%93221%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Hansen, D. F., Vallurupalli, P. &amp; Kay, L. E. An improved 15N relaxation dispersion experiment for the measurement of millisecond time-scale dynamics in proteins. <em>J. Phys. Chem. B</em><strong>112</strong>, 5898–5904 (2008).
</cite> [<a href="https://doi.org/10.1021/jp074793o" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18001083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hansen,%20D.%20F.,%20Vallurupalli,%20P.%20&amp;%20Kay,%20L.%20E.%20An%20improved%2015N%20relaxation%20dispersion%20experiment%20for%20the%20measurement%20of%20millisecond%20time-scale%20dynamics%20in%20proteins.%20J.%20Phys.%20Chem.%20B112,%205898%E2%80%935904%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Shen, Y. &amp; Bax, A. Protein backbone chemical shifts predicted from searching a database for torsion angle and sequence homology. <em>J. Biomol. NMR</em><strong>38</strong>, 289–302 (2007).
</cite> [<a href="https://doi.org/10.1007/s10858-007-9166-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17610132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20Y.%20&amp;%20Bax,%20A.%20Protein%20backbone%20chemical%20shifts%20predicted%20from%20searching%20a%20database%20for%20torsion%20angle%20and%20sequence%20homology.%20J.%20Biomol.%20NMR38,%20289%E2%80%93302%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Mirdita, M. et al. ColabFold: making protein folding accessible to all. <em>Nat. Methods</em><strong>19</strong>, 679–682 (2022).
</cite> [<a href="https://doi.org/10.1038/s41592-022-01488-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9184281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35637307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mirdita,%20M.%20et%20al.%20ColabFold:%20making%20protein%20folding%20accessible%20to%20all.%20Nat.%20Methods19,%20679%E2%80%93682%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Guseva, S. et al. Measles virus nucleo- and phosphoproteins form liquid-like phase-separated compartments that promote nucleocapsid assembly. <em>Sci. Adv.</em><strong>6</strong>, eaaz7095 (2020).
</cite> [<a href="https://doi.org/10.1126/sciadv.aaz7095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7112944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32270045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guseva,%20S.%20et%20al.%20Measles%20virus%20nucleo-%20and%20phosphoproteins%20form%20liquid-like%20phase-separated%20compartments%20that%20promote%20nucleocapsid%20assembly.%20Sci.%20Adv.6,%20eaaz7095%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Lemke, E. A. Site-specific labeling of proteins for single-molecule FRET measurements using genetically encoded ketone functionalities. <em>Methods Mol. Biol.</em><strong>751</strong>, 3–15 (2011).
</cite> [<a href="https://doi.org/10.1007/978-1-61779-151-2_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21674321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lemke,%20E.%20A.%20Site-specific%20labeling%20of%20proteins%20for%20single-molecule%20FRET%20measurements%20using%20genetically%20encoded%20ketone%20functionalities.%20Methods%20Mol.%20Biol.751,%203%E2%80%9315%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Schindelin, J. et al. Fiji: an open-source platform for biological-image analysis. <em>Nat. Methods</em><strong>9</strong>, 676–682 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3855844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schindelin,%20J.%20et%20al.%20Fiji:%20an%20open-source%20platform%20for%20biological-image%20analysis.%20Nat.%20Methods9,%20676%E2%80%93682%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370899/bin/41467_2025_63090_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (10.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370899/bin/41467_2025_63090_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370899/bin/41467_2025_63090_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (2.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370899/bin/41467_2025_63090_MOESM4_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (1.6MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All study data are included in the article, supporting information, the Source Data files and/or can be obtained from the corresponding author upon request. The chemical shift assignments generated in this study have been deposited in the Biological Magnetic Resonance Data Bank (BMRB) under the accession numbers <a href="https://bmrb.io/data_library/summary/index.php?bmrbId=52613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">52613</a> (Dab2<sub>320–495</sub>), <a href="https://bmrb.io/data_library/summary/index.php?bmrbId=52866" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">52866</a> (Eps15<sub>IDR 481-581</sub>), <a href="https://bmrb.io/data_library/summary/index.php?bmrbId=52864" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">52864</a> (Eps15<sub>IDR 569-671</sub>), <a href="https://bmrb.io/data_library/summary/index.php?bmrbId=52863" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">52863</a> (Eps15<sub>IDR 648-780</sub>), and <a href="https://bmrb.io/data_library/summary/index.php?bmrbId=52867" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">52867</a> (Eps15<sub>IDR 761-896</sub>), respectively. The NMR and imaging statistics generated in this study are provided in the Source Data files. The PDB entries of EH2 and EH3 used in this work are: <a href="https://doi.org/10.2210/pdb1FF1/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1FF1</a> (EH2) and <a href="https://doi.org/10.2210/pdb1C07/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">1C07</a> (EH3).  <a href="#Sec25" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63090-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63090.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370899/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370899/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370899%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370899/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370899/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370899/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841375/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370899/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841375/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370899/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370899/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="BYTsQIsudF75rhQFSzOauwtMQUG3JcxiE6BOG2vImx8BesCzgqa65gLJwHoCg94O">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
