
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Metagenomic analysis reveals how multiple stressors disrupt virus–host interactions in multi-trophic freshwater mesocosms - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532DAA8AF209D3052DAA004315B88A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370912/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Metagenomic analysis reveals how multiple stressors disrupt virus–host interactions in multi-trophic freshwater mesocosms">
<meta name="citation_author" content="Tao Wang">
<meta name="citation_author_institution" content="Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, PR China">
<meta name="citation_author_institution" content="Xiangxi River Ecosystem Research Station in the Three Gorges Reservoir Region, Institute of Hydrobiology, Chinese Academy of Sciences, Yixing, PR China">
<meta name="citation_author" content="Peiyu Zhang">
<meta name="citation_author_institution" content="Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, PR China">
<meta name="citation_author_institution" content="Xiangxi River Ecosystem Research Station in the Three Gorges Reservoir Region, Institute of Hydrobiology, Chinese Academy of Sciences, Yixing, PR China">
<meta name="citation_author" content="Karthik Anantharaman">
<meta name="citation_author_institution" content="Department of Bacteriology, University of Wisconsin-Madison, Madison, WI USA">
<meta name="citation_author_institution" content="Department of Integrative Biology, University of Wisconsin-Madison, Madison, WI USA">
<meta name="citation_author_institution" content="Department of Data Science and AI, Indian Institute of Technology Madras, Chennai, India">
<meta name="citation_author" content="Huan Wang">
<meta name="citation_author_institution" content="Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, PR China">
<meta name="citation_author_institution" content="School of Marine Biology and Fisheries, Hainan University, Haikou, PR China">
<meta name="citation_author_institution" content="State Key Laboratory of Marine Resource Utilization in South China Sea, Hainan University, Haikou, PR China">
<meta name="citation_author" content="Huan Zhang">
<meta name="citation_author_institution" content="Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, PR China">
<meta name="citation_author_institution" content="Xiangxi River Ecosystem Research Station in the Three Gorges Reservoir Region, Institute of Hydrobiology, Chinese Academy of Sciences, Yixing, PR China">
<meta name="citation_author" content="Min Zhang">
<meta name="citation_author_institution" content="Hubei Provincial Engineering Laboratory for Pond Aquaculture, Engineering Research Center of Green Development for Conventional Aquatic Biological Industry in the Yangtze River Economic Belt, College of Fisheries, Huazhong Agricultural University, Wuhan, PR China">
<meta name="citation_author" content="Jun Xu">
<meta name="citation_author_institution" content="Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, PR China">
<meta name="citation_author_institution" content="Xiangxi River Ecosystem Research Station in the Three Gorges Reservoir Region, Institute of Hydrobiology, Chinese Academy of Sciences, Yixing, PR China">
<meta name="citation_author_institution" content="State Key Laboratory of Marine Resource Utilization in South China Sea, Hainan University, Haikou, PR China">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7806">
<meta name="citation_doi" content="10.1038/s41467-025-63162-2">
<meta name="citation_pmid" content="40841555">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370912/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370912/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370912/pdf/41467_2025_Article_63162.pdf">
<meta name="description" content="Virus–host interactions are vital to microbiome ecology and evolution, yet their responses to environmental stressors under global change remain poorly understood. We perform a 10-month outdoor mesocosm experiment simulating multi-trophic freshwater ...">
<meta name="og:title" content="Metagenomic analysis reveals how multiple stressors disrupt virus–host interactions in multi-trophic freshwater mesocosms">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Virus–host interactions are vital to microbiome ecology and evolution, yet their responses to environmental stressors under global change remain poorly understood. We perform a 10-month outdoor mesocosm experiment simulating multi-trophic freshwater ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370912/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370912">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63162-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63162.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370912%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370912/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370912/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370912/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7806. doi: <a href="https://doi.org/10.1038/s41467-025-63162-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63162-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Metagenomic analysis reveals how multiple stressors disrupt virus–host interactions in multi-trophic freshwater mesocosms</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Tao Wang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Tao Wang</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, PR China </div>
<div class="p">
<sup>2</sup>Xiangxi River Ecosystem Research Station in the Three Gorges Reservoir Region, Institute of Hydrobiology, Chinese Academy of Sciences, Yixing, PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tao Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Peiyu Zhang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Peiyu Zhang</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, PR China </div>
<div class="p">
<sup>2</sup>Xiangxi River Ecosystem Research Station in the Three Gorges Reservoir Region, Institute of Hydrobiology, Chinese Academy of Sciences, Yixing, PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peiyu Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anantharaman%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Karthik Anantharaman</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Karthik Anantharaman</span></h3>
<div class="p">
<sup>3</sup>Department of Bacteriology, University of Wisconsin-Madison, Madison, WI USA </div>
<div class="p">
<sup>4</sup>Department of Integrative Biology, University of Wisconsin-Madison, Madison, WI USA </div>
<div class="p">
<sup>5</sup>Department of Data Science and AI, Indian Institute of Technology Madras, Chennai, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anantharaman%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Karthik Anantharaman</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Huan Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Huan Wang</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, PR China </div>
<div class="p">
<sup>6</sup>School of Marine Biology and Fisheries, Hainan University, Haikou, PR China </div>
<div class="p">
<sup>7</sup>State Key Laboratory of Marine Resource Utilization in South China Sea, Hainan University, Haikou, PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huan Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Huan Zhang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Huan Zhang</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, PR China </div>
<div class="p">
<sup>2</sup>Xiangxi River Ecosystem Research Station in the Three Gorges Reservoir Region, Institute of Hydrobiology, Chinese Academy of Sciences, Yixing, PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huan Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Min Zhang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Min Zhang</span></h3>
<div class="p">
<sup>8</sup>Hubei Provincial Engineering Laboratory for Pond Aquaculture, Engineering Research Center of Green Development for Conventional Aquatic Biological Industry in the Yangtze River Economic Belt, College of Fisheries, Huazhong Agricultural University, Wuhan, PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Min Zhang</span></a>
</div>
</div>
<sup>8,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jun Xu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jun Xu</span></h3>
<div class="p">
<sup>1</sup>Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, PR China </div>
<div class="p">
<sup>2</sup>Xiangxi River Ecosystem Research Station in the Three Gorges Reservoir Region, Institute of Hydrobiology, Chinese Academy of Sciences, Yixing, PR China </div>
<div class="p">
<sup>7</sup>State Key Laboratory of Marine Resource Utilization in South China Sea, Hainan University, Haikou, PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jun Xu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>7,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Key Laboratory of Lake and Watershed Science for Water Security, Institute of Hydrobiology, Chinese Academy of Sciences, Wuhan, PR China </div>
<div id="Aff2">
<sup>2</sup>Xiangxi River Ecosystem Research Station in the Three Gorges Reservoir Region, Institute of Hydrobiology, Chinese Academy of Sciences, Yixing, PR China </div>
<div id="Aff3">
<sup>3</sup>Department of Bacteriology, University of Wisconsin-Madison, Madison, WI USA </div>
<div id="Aff4">
<sup>4</sup>Department of Integrative Biology, University of Wisconsin-Madison, Madison, WI USA </div>
<div id="Aff5">
<sup>5</sup>Department of Data Science and AI, Indian Institute of Technology Madras, Chennai, India </div>
<div id="Aff6">
<sup>6</sup>School of Marine Biology and Fisheries, Hainan University, Haikou, PR China </div>
<div id="Aff7">
<sup>7</sup>State Key Laboratory of Marine Resource Utilization in South China Sea, Hainan University, Haikou, PR China </div>
<div id="Aff8">
<sup>8</sup>Hubei Provincial Engineering Laboratory for Pond Aquaculture, Engineering Research Center of Green Development for Conventional Aquatic Biological Industry in the Yangtze River Economic Belt, College of Fisheries, Huazhong Agricultural University, Wuhan, PR China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 3; Accepted 2025 Aug 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370912  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841555/" class="usa-link">40841555</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Virus–host interactions are vital to microbiome ecology and evolution, yet their responses to environmental stressors under global change remain poorly understood. We perform a 10-month outdoor mesocosm experiment simulating multi-trophic freshwater shallow lake ecosystems. Using a fully factorial design comprising eight treatments with six replicates each, we assess the individual and combined effects of climate warming, nutrient loading, and pesticide loading on DNA viral communities and their interactions with microbial hosts. Metagenomic sequencing recovers 12,359 viral OTUs and 1628 unique prokaryotic metagenome-assembled genomes. Our analysis shows that combined nutrient and pesticide loading causes significant disruption by synergistically reducing viral alpha diversity while altering beta diversity and predator-prey linkages. Stressors lead to the simplification of virus-bacteria cross-kingdom networks, with nutrient-pesticide combinations exerting the strongest influence, although warming impacts diminish in the presence of pesticides. Stressor-driven changes also affect the abundance and composition of viral auxiliary metabolic genes, leading to complex shifts in virus-mediated metabolic pathways under multiple stress conditions. These findings underscore the importance of understanding the regulatory role of viruses on microbial communities to effectively address the challenges posed by global change.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Water microbiology, Freshwater ecology, Environmental impact, Virus-host interactions, Microbial ecology</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">The impact of global change-related stressors on microbial communities is poorly studied. Here, the authors use mesocosm experiments to show that combined environmental stressors such as nutrient and pesticide loading disrupt freshwater viral communities and virus-host interactions even when single stressors do not.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Viruses, as both regulators and predators in the microbial world, are numerically dominant and play crucial roles in host adaptation, biodiversity, horizontal gene transfer, biogeochemical cycling, ecological processes, and overall ecosystem functionality<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. In marine ecosystems, viruses are estimated to release at least 145 Gt of carbon annually by lysing prokaryotic cells in tropical and subtropical oceans<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>, suggested to contribute to the daily mortality of approximately 20-40% of microbial hosts<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. A recent study in freshwater ecosystems has also demonstrated that carbon cycling in these environments relies heavily on virus-host interactions<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Despite the significant ecological roles viruses play, our understanding of their adaptive strategies in response to environmental changes—especially within the context of global climate change—remains limited. This knowledge gap presents challenges for developing effective ecosystem conservation and restoration strategies.</p>
<p id="Par4">The influence of environmental stressors on virus-host interactions can occur through various potential pathways. Since bacteria are the most abundant cellular organisms, most viruses are likely to be bacteriophages (i.e., phages—viruses that infect bacteria), which are highly abundant and play critical roles in bacterial mortality and biogeochemical cycling<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Two key ecological hypotheses describe the distinct interactions between phages and their hosts, corresponding to the two primary lifestyles of phages: the kill the winner<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup> and piggyback the winner<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> models. In the kill the winner scenario, phages predominantly exhibit a lytic (virulent) lifestyle, preying on fast-growing bacteria and thus freeing up space and nutrient resources for slower-growing bacteria, akin to a predator-prey interaction. In contrast, the piggyback the winner hypothesis involves phages adopting a temperate lifestyle, in which their genomes are integrated into the host and replicated during cell division without immediately lysing the host cell. This lysogenic state persists until internal or external signals trigger the lytic cycle<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Under stressful environmental conditions, virulent phages promote nutrient cycling, drive community evolution, and maintain host diversity and ecosystem stability, while temperate phages may modulate bacterial metabolism or traits to enhance host survival, fostering a mutualistic relationship that aids adaptation to adverse environments<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Additionally, phages possess the ability to encode auxiliary metabolic genes (AMGs), which can influence their hosts and contribute to microbial-driven biogeochemical cycles<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. For instance, research in soil ecosystems has shown that AMGs may help host bacteria adapt to pollutants like pesticides and heavy metals<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Similarly, studies from aquatic environments have highlighted the significant role of AMGs in biogeochemical processes<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>.</p>
<p id="Par5">Environmental stressors rarely occur in isolation, and the complex, often unpredictable interactions between multiple stressors can further complicate virus-host interactions. In recent decades, climate scientists have observed an unprecedented acceleration of global warming<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Under a high greenhouse gas emissions scenario (SSP5-8.5), global surface temperatures are projected to rise by 6.6–14.1 °C by 2300 compared to the period between 1850-1900<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Evidence from a five-year study of experimental warming in permafrost soils suggests that viruses may influence microbial community responses to warming through functional genes and infection strategies<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Alongside climate change, global eutrophication is expected to intensify during the 21st century<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Furthermore, increased pesticide use is anticipated as a response to rising insect pest populations and metabolic rates driven by warming, which threatens crop yields<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. These stressors pose a growing threat to ecosystems in the context of future climate scenarios, underscoring the urgency of investigating their ecological impacts. Laboratory microcosm experiments have shown that nutrient availability can influence phage lysis rates, with lysis being enhanced in nutrient-rich environments<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Pesticides, meanwhile, present substantial threats to microbial communities<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, though viruses may mediate host responses and influence resistance to pesticide contamination<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. In addition, pesticides could indirectly disrupt viral communities in freshwater ecosystems through cascade effects on the food web, specifically by targeting protozoa or other plankton<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. The combined effects of multiple stressors can potentially reshape the microbial world. A deeper understanding of the regulatory roles of viruses may offer new insights into evaluating the effects of these stressors and elucidating the mechanisms underlying their effects. Despite the growing evidence, our understanding of how environmental stressors affect viruses remains fragmented and incomplete, particularly regarding the joint effects of multiple stressors on virus-host interactions.</p>
<p id="Par6">In this work, we conduct a replicated, factorial outdoor mesocosm experiment to explore how climate warming, nutrient loading, and pesticide loading individually and collectively impact viral communities and virus-host interactions. Our investigation focuses on the response strategies of DNA virus-host interactions under multiple stressors, including (1) viral and host community structures, (2) virus-host linkages, (3) virus-host cross-kingdom network structures, and (4) variations in viral AMGs. These analyses aim to deepen our understanding of how these globally pervasive environmental stressors affect biogeochemical cycles and ecosystem functions, ultimately informing strategies to mitigate the impacts of environmental change. Here, we show that multiple environmental stressors impose complex and non-additive effects on viral response strategies and virus–host interactions. Notably, combined stressors—such as nutrient and pesticide loading—induce significant alterations in viral community composition, even when individual stressors exert limited influence. These results emphasize the necessity of accounting for interactive effects among stressors and highlight their potential to restructure microbial communities in freshwater ecosystems.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<p id="Par7">To realistically simulate natural freshwater ecosystems, we constructed a complex, multi-trophic mesocosm and applied environmental stressors that reflect real-world variation patterns (see Methods section and Fig. <a href="#Fig1" class="usa-link">1</a>). After a continuous 10-month experiment, we obtained 12,359 unique DNA viral operational taxonomic units (vOTUs) and 1628 unique prokaryotic metagenome-assembled genomes (MAGs) through metagenomic sequencing. Approximately 99% of the identified viruses belong to the class Caudoviricetes (Supplementary Figs. <a href="#MOESM1" class="usa-link">1</a> and <a href="#MOESM1" class="usa-link">2</a>). These double-stranded DNA-tailed phages are widely distributed in aquatic ecosystems<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> and are also frequently reported in soil studies<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. The MAGs were taxonomically classified as bacteria, with the dominant phyla being Actinomycetota, Bacteroidota, Cyanobacteriota, and Pseudomonadota (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). Subsequently, we examined the impact of environmental stressors on viral communities and their interactions with bacteria.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. The experimental design of this study.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370912_41467_2025_63162_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96a7/12370912/77fe393dbaaa/41467_2025_63162_Fig1_HTML.jpg" loading="lazy" id="d33e474" height="402" width="800" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Photo of the mesocosm taken in June 2021. Letter C represents controls, W represents warming, E represents nutrient loading, P represents imidacloprid loading, and the letter combinations represent environmental stressor combinations. Each group has six replicates. <strong>B</strong> Warming treatment design: The temperature was maintained 3.5 °C above ambient, with the occurrence of multiple heatwaves. <strong>C</strong> Nutrient and <strong>D</strong> imidacloprid loading design: Nutrient (nitrogen and phosphorus) and imidacloprid doses were adjusted over time to reflect seasonal variations in agricultural practices and precipitation intensities in the middle and lower reaches of the Yangtze River throughout the year. Source data are provided as a Source Data file.</p></figcaption></figure><section id="Sec3"><h3 class="pmc_sec_title">Responses of viral and host community structures</h3>
<p id="Par8">Intersection analyses showed that 1733 vOTUs were shared among all treatments, representing the largest observed intersection (Fig. <a href="#Fig2" class="usa-link">2A</a>). A total of 980 vOTUs were shared across six treatments, excluding the EP (nutrient-insecticide co-loading) and WEP (warming plus nutrient-insecticide co-loading) groups. In contrast, 909 vOTUs were uniquely associated with individual treatments, with the highest number found in the C (control) group (171 vOTUs), followed by the E (nutrient loading) group (161 vOTUs). The distribution of vOTUs shared across different numbers of treatments was relatively uniform, ranging from 10.4% to 16.4% (Fig. <a href="#Fig2" class="usa-link">2A</a>). Analysis of the relative abundance of broadly shared vOTUs (i.e., those present across all eight treatments) revealed that interactions involving EP or WEP were the primary drivers influencing their relative abundance (Fig. <a href="#Fig2" class="usa-link">2B</a>; Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). Specifically, the WEP group exhibited a significantly higher relative abundance of these shared vOTUs compared to the C group, whereas differences among the other groups were relatively minor. Regarding MAGs, a total of 284 were shared among all treatments (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). Additionally, 119 MAGs were shared across six treatments, excluding the EP and WEP groups, while 105 were shared among seven treatments, excluding the C group. The number of MAGs uniquely associated with individual treatments was relatively low, totaling only 80 (Supplementary Fig. <a href="#MOESM1" class="usa-link">5A</a>). The EP group had the most unique MAGs among all treatments, with 16 identified. Analysis of the relative abundance of broadly shared MAGs (i.e., those present in all eight treatments) revealed a decreased relative abundance in the EP group compared to the others (Supplementary Fig. <a href="#MOESM1" class="usa-link">5B</a>; Supplementary Table <a href="#MOESM1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Effects of multiple stressors on the viral community.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370912_41467_2025_63162_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96a7/12370912/26e2b44deb4b/41467_2025_63162_Fig2_HTML.jpg" loading="lazy" id="d33e567" height="551" width="799" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> An UpSet plot shows the intersection sizes of vOTUs, with each vertical bar indicating vOTUs uniquely detected in specific treatment combinations. Presence/absence data were aggregated across replicates, and presence indicates vOTUs found in at least one sample per group. Highlighted bars represent broadly shared vOTUs (i.e., those present across all eight groups). Only the top 50 vertical bars are shown; for the complete set of groups, please refer to Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>. The accompanying donut chart depicts the distribution of vOTUs shared among varying numbers of groups, with annotated counts and proportions. <strong>B</strong> The relative abundance of shared vOTUs. Data are shown with 6 biological replicates. The center line in the boxplot denotes the median, with the box edges representing the 25th and 75th percentiles (IQR). The whiskers extend to ±1.5×IQR. <strong>C</strong> Relative abundance of temperate viruses. Temperate viruses were identified via bioinformatic prediction methods. <strong>D</strong> Effect of stressors on viral richness. In <strong>C</strong>, <strong>D</strong>, data are presented for each group (<em>n</em> = 6 biological replicates), with error bars representing the mean ± 95% confidence interval. In <strong>B</strong>–<strong>D</strong>, the two-sided <em>p</em>-values from the emmeans tests were adjusted using the Benjamini-Hochberg method. Different lowercase letters between groups indicate significant differences at <em>p</em> &lt; 0.05. Exact <em>p</em>-values are provided in Supplementary Table <a href="#MOESM1" class="usa-link">14</a>. The letter C on the <em>x</em>-axis represents controls, W represents warming, E represents nutrient loading, P represents imidacloprid loading, and the letter combinations represent stressor combinations. <strong>E</strong> Shifts in viral community composition under multiple stressors. Analysis of similarities (Anosim) and permutational multivariate analysis of variance (see Supplementary Table <a href="#MOESM1" class="usa-link">3</a>) with a two-sided 999-permutation test were used to assess significant differences (<em>n</em> = 6 biological replicates). Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par9">Temperate viruses were identified via bioinformatic predictions based on the detection of proviral integration sites, integrase genes, and characteristic patterns of protein composition and associations (see “Methods” for details). Under nutrient loading conditions, the proportion of temperate viruses increased slightly (Fig. <a href="#Fig2" class="usa-link">2C</a>; Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). In scenarios involving simultaneous nutrient (nitrogen and phosphorus) and imidacloprid (a common insecticide) loading, the proportion of temperate viruses displayed considerable variability, ranging from over 50% to below 30% (Fig. <a href="#Fig2" class="usa-link">2C</a>).</p>
<p id="Par10">The results indicated that viral alpha diversity was significantly disrupted by the combined effects of nutrient and imidacloprid loading; however, neither factor alone had a notable impact (Fig. <a href="#Fig2" class="usa-link">2D</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>; Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Under their combined influence, both the richness and Shannon indices demonstrated significant reductions. Furthermore, the Simpson and Pielou indices revealed increased intragroup variability compared to other groups (Supplementary Fig. <a href="#MOESM1" class="usa-link">6B</a>, C). Additionally, warming did not influence the combined effect of nutrient and imidacloprid loading on alpha diversity, as changes observed in the EP group were consistent with those in the WEP group (Fig. <a href="#Fig2" class="usa-link">2D</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>).</p>
<p id="Par11">Warming, nutrient loading, and imidacloprid loading all resulted in significant shifts in viral community structure, which in turn altered viral beta diversity (Fig. <a href="#Fig2" class="usa-link">2E</a>, Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). Multiple stressors significantly affect viral beta diversity, with notable combined effects observed between nutrient and imidacloprid loading (Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). Our findings indicate that the alpha diversity of both viruses and bacteria did not exhibit a significant response to warming (Fig. <a href="#Fig2" class="usa-link">2D</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">6</a> and <a href="#MOESM1" class="usa-link">7A–D</a>; Supplementary Tables <a href="#MOESM1" class="usa-link">2</a> and <a href="#MOESM1" class="usa-link">4</a>). Notably, viral beta diversity responded more sensitively than alpha diversity, reflecting trends observed in bacterial beta diversity (Fig. <a href="#Fig2" class="usa-link">2E</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7E</a>; Supplementary Table <a href="#MOESM1" class="usa-link">5</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Responses of virus-host linkages</h3>
<p id="Par12">Using a combination of bioinformatic methods for virus-host prediction (see “Methods” for details), we identified 3455 vOTUs-MAGs linkages, comprising 2031 vOTUs and 772 MAGs across 17 host phyla (Fig. <a href="#Fig3" class="usa-link">3A</a>; Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). Our findings indicate that multiple stressors can disrupt predator-prey linkages between viruses and hosts (Fig. <a href="#Fig3" class="usa-link">3B</a>), with these effects varying according to viral lifestyle (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>; Supplementary Table <a href="#MOESM1" class="usa-link">6</a>). A significantly elevated log<sub>10</sub>-transformed virus-to-host ratio (VHR; calculated as the abundance ratio of viruses with identified putative hosts to their corresponding hosts) was observed in the P (imidacloprid loading), EP, WP, and WEP groups compared to the control group (Fig. <a href="#Fig3" class="usa-link">3B</a>). For virulent viruses, similar trends were observed within the overall viral community (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>), likely attributable to a higher proportion of virulent viruses within the abundance composition (Fig. <a href="#Fig2" class="usa-link">2C</a>). For temperate viruses, the combined effects of nutrient and imidacloprid loading significantly disrupted predator-prey linkages; however, this effect was less pronounced under warming conditions (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>; Supplementary Table <a href="#MOESM1" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Effects of multiple stressors on virus–host linkages.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370912_41467_2025_63162_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96a7/12370912/0acab5bf1863/41467_2025_63162_Fig3_HTML.jpg" loading="lazy" id="d33e717" height="996" width="754" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Phylogenetic tree of MAGs at the phylum level recovered from metagenomes. Green circles indicate MAG lineages predicted to be infected by viruses, with the number of identified vOTUs shown inside the circles. The numbers in parentheses represent the number of MAGs associated with identified viruses relative to the total number of MAGs. <strong>B</strong> Effects of multiple stressors on the log<sub>10</sub> virus-host abundance ratio (VHR) of viruses. The ratio was calculated for viruses with identified putative hosts using bioinformatic prediction methods. The number of putative virus–host species pairs identified across six biological replicates for each treatment is shown above the plot. The center line in the half-boxplot denotes the median, with the box edges representing the 25th and 75th percentiles (IQR). The whiskers extend to ±1.5×IQR. The two-sided <em>p</em>-values from the emmeans tests were adjusted using the Benjamini-Hochberg method. Different lowercase letters between groups indicate significant differences at <em>p</em> &lt; 0.05. Exact <em>p</em>-values are provided in Supplementary Table <a href="#MOESM1" class="usa-link">15</a>. The letter C on the x-axis represents controls, W represents warming, E represents nutrient loading, P represents imidacloprid loading, and the letter combinations represent environmental stressor combinations. <strong>C</strong> The correlation between viral and host relative abundances at the host family level, where cell size is positively correlated with the Pearson correlation coefficient (r). Asterisks indicate significance levels from a two-tailed t-test: * for <em>p</em> &lt; 0.05, ** for <em>p</em> &lt; 0.01, and *** for <em>p</em> &lt; 0.001. The figure was divided at the phylum level based on the host, with the y-axis representing the host at the family level. Exact <em>p</em> values and source data are provided as a Source Data file.</p></figcaption></figure><p id="Par13">Variability in correlations at the host family level (Fig. <a href="#Fig3" class="usa-link">3C</a>) suggests that the effects of multiple stressors on virus-host interactions may differ among taxonomic groups. For instance, warming reduced correlations in lineage-specific interactions within families such as UBA1268 (Planctomycetota) and Nostocaceae (Cyanobacteriota), while enhancing correlations in families such as UBA5976 and Nanopelagicaceae (Actinomycetota) relative to the control group. The influence of multiple stressors introduces complexity to lineage-specific interactions (Fig. <a href="#Fig3" class="usa-link">3C</a>), highlighting the necessity to consider how these stressors differentially impact viral control mechanisms within microbial communities.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Responses of virus–host network</h3>
<p id="Par14">The interactions of multiple stressors-warming, nutrient loading, and imidacloprid loading-significantly altered the cross-kingdom co-occurrence network of viruses and bacteria, leading to significant simplification of the network (Fig. <a href="#Fig4" class="usa-link">4A</a>; Supplementary Table <a href="#MOESM1" class="usa-link">7</a>). These stressors generally decreased the network’s complexity and stability, with the exception of nutrient loading alone (Fig. <a href="#Fig4" class="usa-link">4B-I</a>; Supplementary Table <a href="#MOESM1" class="usa-link">8</a>). Nutrient loading increased the network’s complexity (e.g., average K, in Fig. <a href="#Fig4" class="usa-link">4D</a>) and enhanced its robustness (Fig. <a href="#Fig4" class="usa-link">4I</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Effects of multiple stressors on the virus-bacteria cross-kingdom co-occurrence network.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370912_41467_2025_63162_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96a7/12370912/a81de30a5159/41467_2025_63162_Fig4_HTML.jpg" loading="lazy" id="d33e790" height="759" width="799" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Network nodes are colored by virus and bacteria, while edges are colored to represent positive and negative interactions. The size of the nodes is positively correlated with their degree. <strong>B</strong>–<strong>H</strong> Changes in subnetwork properties across different groups are illustrated. Data are shown for 6 biological replicates, with error bars representing the mean ± 95% confidence interval. <strong>I</strong> Variations in the robustness of the cross-kingdom co-occurrence network are presented. Data are shown with 100 repetitions of the simulation. The center line in the boxplot denotes the median, with the box edges representing the 25th and 75th percentiles (IQR). The whiskers extend to ±1.5×IQR. The two-sided <em>p</em> values from the emmeans tests (<strong>B</strong>–<strong>H</strong>) and the pairwise Wilcoxon test (<strong>I</strong>) were adjusted using the Benjamini-Hochberg method. Different lowercase letters between groups indicate significant differences at <em>p</em> &lt; 0.05. Exact <em>p</em>-values are provided in Supplementary Table <a href="#MOESM1" class="usa-link">16</a>. The letter C on the x-axis represents controls, W represents warming, E represents nutrient loading, P represents imidacloprid loading, and the letter combinations represent environmental stressor combinations. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par15">When applied independently, both warming and imidacloprid loading significantly reduced the network’s complexity, including a decrease in the number of nodes, the number of links, and the average K (Fig. <a href="#Fig4" class="usa-link">4B–D</a>). These stressors also increased average path length, the number of connected components and relative modularity (Fig. <a href="#Fig4" class="usa-link">4E–G</a>). Additionally, they decreased network stability, as indicated by increased vulnerability and reduced robustness (Fig. <a href="#Fig4" class="usa-link">4H, I</a>). However, the interaction between warming and imidacloprid resulted in antagonistic effects, alleviating the impact of each stressor on network complexity and stability when applied individually (Fig. <a href="#Fig4" class="usa-link">4B–I</a>; Supplementary Tables <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">8</a>). While nutrient loading notably enhanced the complexity and robustness of the network, significant disruption was observed when combined with imidacloprid loading. A simplified and less robust network was observed in the EP or WEP groups (Fig. <a href="#Fig4" class="usa-link">4A–I</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Responses of viral auxiliary metabolic genes</h3>
<p id="Par16">Through multiple screening phases (see “Methods”), we identified 2996 AMGs linked to potential metabolic functions in the KEGG database, corresponding to 100 KOs, the majority of which are involved in metabolic activities (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">10A</a> and <a href="#MOESM1" class="usa-link">11</a>; Supplementary Data <a href="#MOESM3" class="usa-link">2</a>). The richness of AMGs significantly declined under the combined effects of nutrient and imidacloprid loading (Supplementary Fig. <a href="#MOESM1" class="usa-link">10B</a>), while the composition of these genes underwent notable changes due to the stressors (Supplementary Fig. <a href="#MOESM1" class="usa-link">10C</a>). Each stressor—warming, nutrient loading, and imidacloprid loading—independently caused significant shifts in AMG composition, with a pronounced interaction between nutrient and imidacloprid loading that further amplified these changes (Supplementary Table <a href="#MOESM1" class="usa-link">9</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Effects of multiple stressors on viral auxiliary metabolic genes (AMGs).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370912_41467_2025_63162_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96a7/12370912/388ce5a08c69/41467_2025_63162_Fig5_HTML.jpg" loading="lazy" id="d33e894" height="489" width="799" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Variation in AMGs across different groups (<em>n</em> = 6 biological replicates). Only AMGs in treatment groups with significant differences from the control group and potential contributions to metabolism are shown. Data are presented as the mean abundance for each group. The abundance of AMGs was calculated based on the abundance of viral contigs containing AMGs (see Methods). The corresponding KEGG level 2 metabolic pathways for each gene are marked with red dots in the lower panel. Asterisks mark genes with Kruskal–Wallis <em>p</em>  &lt;  0.05 and LDA score &gt; 2. All analyses used two-sided <em>p</em>-values. Black asterisks (*) indicate genes with higher abundance in the treatment group compared to the control, while red asterisks indicate genes with lower abundance. Exact <em>p</em>-values are provided in Supplementary Table <a href="#MOESM1" class="usa-link">10</a>. The letter C on the y-axis represents controls, W represents warming, E represents nutrient loading, P represents imidacloprid loading, and the letter combinations represent environmental stressor combinations. <strong>B</strong> The roles of these AMGs in major KEGG level 3 metabolic pathways are shown, as along with the downstream and upstream compounds or pathways. Genes are shown in red font, while the names of metabolic pathways and compounds are displayed in black. Different metabolic pathways are distinguished by color blocks. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par17">Stressors can alter the abundance of genes linked to various metabolic pathways, thereby affecting the regulation of host metabolism by viruses, with effects differing based on the specific type of stressor (Fig. <a href="#Fig5" class="usa-link">5A</a>, Supplementary Table <a href="#MOESM1" class="usa-link">10</a>). The presence of multiple stressors leads to more complex changes in the abundance of viral AMGs (Fig. <a href="#Fig5" class="usa-link">5A</a>). Under combined nutrient and imidacloprid conditions, the changes in AMG abundance diverged from those observed with each stressor alone. Climate warming amplifies these variations, resulting in increased complexity. For example, under WEP conditions, some auxiliary metabolic genes (e.g., <em>acpP, cobS, deoC, EARS, queC, wbpD</em>) exhibit smaller abundance differences compared to the control group relative to EP conditions, while differences in other genes (e.g., <em>argH, dut, glnA, kdsB</em>) are further amplified. Additionally, the direction of abundance change for certain genes (e.g., <em>pyrB</em>) is reversed. Furthermore, environmental stress not only directly affects the abundance of auxiliary metabolic genes and specific pathways but also indirectly influences genes regulating upstream and downstream pathways, potentially altering substance formation within these pathways and thus impacting overall metabolic activity (Fig. <a href="#Fig5" class="usa-link">5B</a>).</p></section></section><section id="Sec7"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par18">In the context of ecosystems subjected to multiple stressors, our understanding of how these stressors individually or collectively influence viral communities and their interactions with other microbial communities remains limited. This limitation hinders our ability to effectively manage and protect ecosystems in the context of climate change. Our findings indicate that the combined loadings of environmental pollutants, such as nutrient and insecticide loading, significantly disrupt DNA viral communities and their interactions with hosts. Climate warming reduces the complexity and stability of virus-host interaction networks; however, it exhibits an antagonistic effect when combined with insecticide loading. The impact of multiple stressors results in diverse changes in viral AMGs, potentially influencing biogeochemical cycles. To effectively address the impacts of climate change and environmental stressors, there is an urgent need to investigate viruses and their interactions with other microbes within complex ecosystems.</p>
<p id="Par19">For the viral and host community structures we investigated, we observed that combinations of environmental stressors—such as the interaction between nutrient and insecticide loading (EP), or the three-way interaction involving warming (W), nutrient loading (E), and insecticide loading (P)—significantly altered both viral and host community structures. These stressor combinations significantly affected viral species’ composition and community diversity. Notably, only 14.0% of viral operational taxonomic units (vOTUs) were shared across all treatment groups, indicating strong treatment-specific niche preferences. This is consistent with the well-known ecological dependence of viruses on both host availability and local environmental conditions<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Similarly, bacterial communities showed limited overlap across treatments, with only 17.4% of metagenome-assembled genomes (MAGs) common to all conditions. Together, these findings suggest that both viral and bacterial communities are highly sensitive to environmental stressors. The vOTUs shared across multiple treatments can be regarded as habitat generalists<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, which are viral taxa capable of maintaining detectable abundance and occurrence across a range of environmental stress conditions. These generalist viruses may possess higher environmental tolerance or enhanced mechanisms for persistence under disturbance<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, as evidenced by their significantly elevated relative abundance in the WEP treatment group. Such responses highlight their potential ecological advantage in complex, multi-stressor environments.</p>
<p id="Par20">Previous studies in soil suggest that viral alpha diversity may exhibit lower sensitivity to warming than that of other prokaryotes<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. In contrast, viral communities often respond more strongly to environmental pollutants, such as heavy metal contamination<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, which is consistent with our observations under insecticide and nutrient enrichment treatments. In our study, viral beta diversity exhibited more pronounced changes than alpha diversity, a trend also observed in bacterial communities. This divergence implies that different ecological mechanisms may govern alpha and beta diversities within viral and bacterial communities. Moreover, viral beta diversity was closely associated with changes in prokaryotic beta diversity, reflecting a strong coupling between viral and host community structures. This aligns with findings from soil systems, where viral beta diversity is significantly correlated with both abiotic factors and the beta diversity of prokaryotic (bacteria and archaea) communities<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Our findings from freshwater ecosystems reveal that viral community diversity closely tracks changes in prokaryotic beta diversity, suggesting a consistent regulatory role of biotic and abiotic factors across ecosystems. These results emphasize the complexity of virus-prokaryote interactions and underscore the need for further research on multiple stressors to elucidate the mechanisms driving these variations across broader spatial and temporal scales.</p>
<p id="Par21">An increase in the abundance of temperate viruses was observed in the EP condition, alongside a rise in intra-group variation. The increased abundance of temperate viruses under nutrient and imidacloprid loading potentially reflects a viral strategy to evade environmental stress<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Under nutrient loading conditions, this increase may also be attributed to the enhanced density and growth of host microorganisms resulting from nutrient loading<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. In environments characterized by high host density and rapid growth, viruses tend to adopt a lysogenic lifestyle, consistent with the piggyback the winner hypothesis—a phenomenon observed in coral reef viromes<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Although adopting a temperate lifestyle may facilitate evasion of adverse conditions imposed by multiple stressors<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, the inconsistency of this response strategy and the diverse patterns complicate our understanding of ecosystem responses.</p>
<p id="Par22">As for predator-prey linkages, imidacloprid loading alone significantly influenced the linkages between viruses and their hosts, likely through cascading effects on consumers, such as protists or copepods<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, across the food web. In addition to these cascading effects, even the mere presence of protists—at low abundance and with minimal predation pressure—can elevate the energy and resource demands of virus-infected cells (e.g., cyanobacterial virocells), inducing metabolic reprogramming and altering exometabolite release<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. These indirect effects may modulate the progression of viral infections and their ecological ramifications. Although not direct targets, freshwater microbes are nevertheless affected by pesticide exposure. Consequently, studying environmental pollutants within complex, multi-trophic systems is crucial to better capturing their broader ecological consequences, highlighting the urgency of incorporating viruses and other microbes into food web studies<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. The combination of nutrient and imidacloprid loading resulted in an increased proportion of viruses exhibiting temperate strategies, which theoretically should decrease log<sub>10</sub>(VHR) due to increased lysogeny<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. However, our study revealed a significant decline in species richness under the EP condition. Environmental stressors filtered viral species, with species homogenization resulting in stronger top-down controls on specific microbial groups. Warming also caused a significant decrease in log<sub>10</sub>(VHR) for temperate viruses but had no significant effect on virulent viruses. These specific responses may introduce unexpected variations in biogeochemical cycles as a result of climate change. While adopting lysogeny can serve as an adaptive response to stressful conditions<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, the combined effect of stressors, such as nutrient and imidacloprid loading, may exceed the adaptive capacity of lysogenic viruses, indicating potentially severe consequences of multiple environmental stressors. Given that different bacterial communities perform specialized functions, a deeper understanding of how stressors affect virus-bacteria interactions is essential for comprehending their broader ecological implications. Such insights may also enhance our understanding of prokaryotic succession within these microbial ecosystems.</p>
<p id="Par23">Different environmental stressors and their combinations variably impact network complexity and stability. Nutrient loading increases network complexity (e.g., average K) and stability, likely by stimulating the growth and reproduction of basal organisms (e.g., bacteria)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, which may enhance resource availability for organisms at other trophic levels (e.g., viruses) and foster interactions among them. In resource-rich environments, frequent interactions, such as predation, competition, and symbiosis, contribute to a more complex and stable ecological network, supporting multi-trophic interdependence<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. In contrast, warming and imidacloprid loading independently simplify and destabilize the network. Viral infection processes in microbial food webs are often temperature-dependent<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, and warming may destabilize certain virus-host relationships, reducing connections within the virus-bacteria network and disrupting its structure. Additionally, warming alters the metabolic rates of prokaryotic microbes<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>, shifts resource use strategies, and changes community composition, which may destabilize specific virus-bacteria associations and decrease network complexity and stability. The impact of imidacloprid loading on network complexity and stability likely arises from its effects on invertebrate zooplankton, inducing trophic cascades that influence viral and bacterial interactions<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>.</p>
<p id="Par24">The combined effects of multiple stressors led to complex changes in network parameters, distinct from the impacts observed for each stressor alone. Interaction between warming and imidacloprid slightly increased network complexity and stability, possibly due to co-adaptive species responses, where exposure to one stressor enhances adaptation to additional stressors<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>–<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Alternatively, warming-induced external pesticide degradation might contribute to this effect<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. In contrast, combined nutrient and imidacloprid loading simplified the network, reducing nodes and links, which corresponded with decreased diversity. Nutrients and imidacloprid exert bottom-up and top-down effects, respectively, on our complex multi-trophic ecosystem, potentially causing greater disturbances when both are present. These results align with findings that certain stressors interact non-additively<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Previous research shows that environmental stressors can both destabilize and stabilize microbial networks, though most studies focus on bacteria, fungi, and cercozoans, with virus-bacteria interactions largely unexplored<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>–<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Stressors may alter microbial hierarchies and cross-kingdom network linkages, depending on specific microbial domain combinations<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Understanding virus-bacteria interactions across kingdoms is critical to addressing climate change threats. Our findings indicate that the effects of stressors on the virus-bacteria network depend on both the type of stressors and their interactions. While warming and imidacloprid each pose significant threats to the cross-kingdom network, nutrient loading increased network complexity and robustness. However, combined nutrient and imidacloprid loading resulted in a simpler network with fewer nodes and links, raising concerns about the combined effects of environmental pollutants on biogeochemical cycles and aquatic ecosystems. Additionally, the potential antagonistic effects of warming and imidacloprid loading under certain conditions present challenges for ecosystem management, underscoring the need for a more comprehensive understanding of the implications of multiple stressors for effective ecosystem management<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>.</p>
<p id="Par25">For viral auxiliary metabolic genes (AMGs), we found that their abundance was influenced by multiple environmental stressors. Previous research suggests that virus-associated AMGs may contribute to biogeochemical cycles, including those of carbon, nitrogen, phosphorus, and sulfur<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. Our study provides evidence that key ecological processes are affected by environmental stressors through the metabolic regulation of hosts in response to viruses. Genes involved in carbon cycle pathways<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>, such as the pentose phosphate pathway (<em>deoC</em>, <em>TALDO1</em>, <em>hxlA</em>, <em>hxlB</em>, <em>PGD</em>), methane metabolism (<em>AGXT, hxlA, hxlB</em>), and glyoxylate and dicarboxylate metabolism (<em>glnA, AGXT</em>), were significantly affected by stressors, particularly under conditions of multiple stressors (Fig. <a href="#Fig5" class="usa-link">5</a>). For instance, in the control group, the abundance of <em>TALDO1</em>, <em>PGD</em>, <em>hxlA</em>, and <em>hxlB</em> were notably higher than in the EP, WP, and WEP groups, indicating that the ability of viruses to modulate the microbial carbon cycle is compromised by the combined effects of multiple stressors. It is important to note that the findings of this study are primarily based on metagenomic sequencing, which reveals the presence of viral AMGs but does not provide information on their actual expression. While previous studies have provided evidence that AMGs are actively involved in the host’s metabolic activities<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>, our analysis of partial metatranscriptomic data also demonstrates that a significant number of viral sequences recovered from the metagenome are active (Supplementary Table <a href="#MOESM1" class="usa-link">11</a>). Furthermore, these AMGs are abundantly expressed in multiple metabolic pathways, indicating their regulatory role in the host’s metabolic processes (Supplementary Discussion; Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>). Future studies incorporating transcriptomics will further explore the expression differences of AMGs under multiple stressors, providing a more comprehensive understanding of their functional relevance.</p>
<p id="Par26">Some genes are implicated in multiple metabolic processes, suggesting that the metabolic changes resulting from variations in these auxiliary metabolic genes require further experimental validation in future studies to fully understand their true impacts. However, it is crucial to note that environmental stressors can influence multiple viral AMGs upstream and downstream of metabolic pathways, potentially leading to significant ecological risks in environments exposed to multiple stressors. Therefore, it is urgent to incorporate virus-mediated auxiliary metabolic regulation mechanisms into the assessment of ecosystem impacts under multiple stressors.</p>
<p id="Par27">Although viruses play a vital role in regulating the structure and function of microbial communities, their responses to multiple environmental stressors remain underexplored. This knowledge gap hinders our capacity to effectively address the challenges posed by global climate change and various environmental stressors. Our research, conducted in near-natural ecosystems and accounting for the temporal variability of stressors, offers significant insights into the responses of microbial communities to these challenges. Our findings indicate that viral communities and their interactions with hosts are profoundly affected by environmental stressors. Climate warming altered the structure of viral communities and the regulation of auxiliary metabolic processes, resulting in simplified virus-host networks that increase vulnerability and decrease robustness. While nutrient loading and pesticide application alone did not significantly disrupt viral diversity, predator-prey linkages, or co-occurrence networks, their combined effects severely disrupted viral communities and virus-host interactions. Furthermore, the interaction between warming and pesticide exposure mitigated the individual impacts of each under specific conditions. This evidence underscores the intricate complexity of ecosystem responses to multiple stressors. Viral auxiliary metabolic genes, known to regulate diverse metabolic pathways, showed stressor-induced shifts in their abundance across pathways. These findings emphasize the importance of considering viruses and their interactions with other organisms when evaluating the ecological impacts of multiple stressors. The combined effects of nutrient and pesticide loading observed in our study suggest that interactions between environmental pollutants may lead to greater disruptions in microbial communities, thereby interfering with essential biogeochemical cycling processes. However, pesticide loading demonstrates an antagonistic effect when combined with climate warming. In the conservation of shallow lakes, it is crucial to consider the cumulative effects of multiple pollutants and their potential interactions with climate change.</p>
<p id="Par28">Our study highlights the significant interactive effects of multiple environmental stressors on viral and bacterial community structures, as well as their ecological linkages. These important effects would likely have been overlooked if the stressors had been examined in isolation. Given that most ecosystems are simultaneously exposed to various anthropogenic pressures, it is essential to consider their combined impacts—both within freshwater systems and across broader ecological contexts. Several limitations of the current study suggest directions for future research. First, our analysis primarily relied on metagenomic data, focusing on DNA viruses, particularly double-stranded DNA viruses. The ecological roles of single-stranded DNA and RNA viruses remain poorly understood. Future research may integrate viromics and metatranscriptomics, while also refining methods to enrich free viruses, thereby achieving more comprehensive taxonomic and functional characterization. Metatranscriptomic data are especially valuable for verifying the expression of auxiliary metabolic genes. The combination of metagenomics and metatranscriptomics will enhance our ability to assess the functional consequences of environmental stressors on microbial communities. Second, because our pollutant addition simulation was designed to closely mimic natural environmental conditions by incorporating fluctuating inputs that reflect temporal variations in pollutant loading, the study employed end-point sampling. However, incorporating temporal sampling would provide greater insight into ecologically relevant successional dynamics and transient microbial community shifts. Lastly, although this study focused on virus–bacteria interactions, other microbial taxa (e.g., fungi, archaea, and protists) may also influence virus–host interactions. Including these groups in future investigations will help to develop a more holistic understanding of microbiome responses to multiple stressors.</p></section><section id="Sec8"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec9"><h3 class="pmc_sec_title">Ethics statement</h3>
<p id="Par29">This experiment was conducted in compliance with all relevant ethical regulations and was approved and supervised by the Animal Ethical and Welfare Committee, Institute of Hydrobiology, Chinese Academy of Sciences.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Experiment design</h3>
<p id="Par30">The mesocosm system used for this experiment was located at Huazhong Agricultural University in Wuhan, Central China (30°29′N; 114°22′E). It consisted of 48 insulated cylindrical polyethylene tanks (1.5 m in diameter and 1.4 m in height) equipped with an automatic dynamic temperature control system<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. Starting in early February 2021, the mesocosms were gradually filled with sediment, water, and various aquatic organisms to mimic the shallow lake ecosystem. Each mesocosm was half-filled with a 10 cm layer of muddy sediment collected from Lake Liangzi (30°11′3″N, 114°37′59″E). The sediment was homogenized and sieved through a 5 × 5 mm metal mesh to remove large debris, macrophyte seeds, and mollusks. To closely mimic the ecosystem conditions of natural lakes, fish, shrimp, snails, plankton, and submerged macrophytes were introduced. Each mesocosm received four bitterlings (<em>Rhodeus sinensis</em>, ~3 cm), four crucian carp (<em>Carassius auratus</em>, ~4 cm), five freshwater shrimps (<em>Macrobrachium nipponense</em>, ~4 cm), fourteen snails (<em>Radix swinhoei</em>, 1-2.5 cm), and twenty snails (<em>Bellamya aeruginosa</em>, ~2.5 cm). Additionally, 50 g each of <em>Potamogeton crispus</em> and <em>Hydrilla verticillata</em> were planted in the sediment to establish submerged macrophytes. Ten liters of lake water from nearby Lake Nanhu (30°28′57″N, 114°22′34″E) were added to each mesocosm to inoculate plankton and other microbes. These species, commonly found in lakes in the middle and lower reaches of the Yangtze River Basin, were introduced at densities and biomass levels consistent with those occurring in nature<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>–<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>.</p>
<p id="Par31">The experiment spanned from February to November 2021, with environmental stressors applied starting on April 8, 2021, after a two-month acclimation period. The study employed a fully factorial design involving three stressors: warming (W), nutrient loading (E), and the insecticide imidacloprid (P), with controls (C). This design resulted in eight treatments, each with six replicates. To minimize potential grouping biases, mesocosms were randomly assigned to one of the eight treatment groups prior to the acclimation period. All mesocosms were then acclimated under identical ambient conditions for two months to ensure they evolved in the same direction. On April 7, 2021, we systematically assessed the environmental conditions of each mesocosm. Dissolved oxygen, conductivity, and pH were measured in situ using a HACH HQD portable meter (HQ60d, HACH, USA). Nutrient concentrations (total nitrogen, total phosphorus, nitrate and phosphate) and phytoplankton chlorophyll-a content in depth-integrated water were quantified using a spectrophotometer<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. Periphytic algae chlorophyll-a content was determined by suspending a polyethylene board (rough surface, 10 × 10 cm) in each mesocosm, positioned 60 cm below the water surface. Additionally, we calculated the plant volume infested (PVI), an indicator of submerged macrophyte community biomass<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. No noticeable differences were observed in these parameters across treatment groups (Supplementary Table <a href="#MOESM1" class="usa-link">12</a>), confirming that all systems were in a comparable and consistent state prior to treatment application.</p>
<p id="Par32">The warming treatment elevated the water temperature by +3.5 °C above ambient levels and included multiple heatwave events throughout the experiment (Fig. <a href="#Fig1" class="usa-link">1B</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). The frequency and intensity of these heatwaves were based on model projections from historical meteorological data for the middle and lower Yangtze River Basin and are expected to occur in this region by the end of the century due to ongoing climate warming<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. The nutrient loading treatment involved the biweekly addition of nitrogen (N) and phosphorus (P) at a 10:1 mass ratio, achieved by dissolving NaNO<sub>3</sub> and KH<sub>2</sub>PO<sub>4</sub> in demineralized water (Fig. <a href="#Fig1" class="usa-link">1C</a>). The insecticide treatment involved biweekly applications of imidacloprid (70% active ingredient, PD20120072, Bayer, Germany) to simulate agricultural pesticide loading (Fig. <a href="#Fig1" class="usa-link">1D</a>). Imidacloprid, a common insecticide belonging to the neonicotinoid class, is one of the most widely used agricultural pesticides globally and poses a higher risk to freshwater ecosystems<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. The intensity of both nutrient and pesticide applications was adjusted to reflect agricultural activities and precipitation patterns in the region<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>,<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. During the experiment, the average nutrient loading concentrations were 0.90 mg L<sup>−1</sup> for nitrogen (ranging from 0.25 to 1.6 mg L<sup>−1</sup>) and 0.09 mg L<sup>−1</sup> for phosphorus (ranging from 0.025 to 0.16 mg L<sup>−1</sup>). The average insecticide loading concentration was 32.67 µg L<sup>−1</sup> (ranging from 10 to 50 µg L<sup>−1</sup>), consistent with levels typically observed in natural water bodies in agricultural regions worldwide<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>,<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Sampling collection</h3>
<p id="Par33">At the end of the experiment in November 2021, ten depth-integrated water samples were taken from each mesocosm using a transparent plexiglas tube (70 mm in diameter, 1 m in length). These samples were combined and thoroughly mixed in a cleaned, sterile water bucket. The mixed water samples were then filtered through 0.22 μm membrane filters (Merck millipore, Germany) using a peristaltic pump. The filters were stored at −80 °C prior to DNA extraction. A single water sample from each mesocosm was used for metagenomic sequencing, resulting in a total of 48 samples.</p></section><section id="Sec12"><h3 class="pmc_sec_title">DNA extraction and metagenomic sequencing</h3>
<p id="Par34">Total genomic DNA was extracted from water samples using the CretMag<sup>TM</sup> Power Soil DNA Kit (CretBiotech, China) following the manufacturer’s protocol. Concentration and purity of extracted DNA was determined with TBS-380 and NanoDrop2000, respectively. The quality of the DNA extracts was assessed by electrophoresis on a 1% agarose gel. The DNA was then fragmented to an average size of approximately 400 bp using a Covaris M220 (Gene Company Limited, China) to prepare for paired-end library construction. The paired-end library was built using the NEXTflex<sup>TM</sup> Rapid DNA-Seq Kit (Bioo Scientific, Austin, TX, USA), where adapters containing the full complement of sequencing primer hybridization sites were ligated to the blunt-end of fragments. For each sample, 200 ng of DNA was used as input for library preparation, including 6 cycles of PCR amplification. Paired-end sequencing was subsequently carried out on the DNBSEQ-T7 platform at Wefind Biotechnology Co., Ltd. (Wuhan, China) using the DNBSEQ-T7RS Reagent kit (FCL PE150) V2.0, in accordance with the manufacturer’s instructions (<a href="https://www.mgi-tech.com/products/reagents_info/43/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.mgi-tech.com/products/reagents_info/43/</a>). For detailed information on the sequencing data, please refer to Supplementary Table <a href="#MOESM1" class="usa-link">13</a>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Sequence quality control, genome assembly and binning</h3>
<p id="Par35">The raw reads obtained from metagenomic sequencing underwent initial trimming and contaminant removal using the Read_qc module (with default parameters) in metaWRAP v1.3.2<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>. Afterward, the clean reads from each water sample were separately assembled into contigs through the metaWRAP assembly module, which employs the MegaHit<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup> with default parameters. Subsequently, these contigs were binned into species-level metagenome-assembled genomes (MAGs) using the metaWRAP Binning module. This process utilized three distinct binning algorithms, namely CONCOCT<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>, MaxBin2<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>, and metaBAT2<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. The results of the multiple binning predictions were refined and consolidated into a more accurate bin set with the metaWRAP Binning refinement module, applying thresholds of at least 70% completeness and less than 10% contamination (-c 70 -x 10)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>. Remaining MAGs were dereplicated using dRep v3.4.5 with the parameters -S_algorithm ANImf -sa 0.99 -nc 0.1 -clusterAlg single<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>.</p>
<p id="Par36">For taxonomic classification, the classify_wf workflow in GTDB-TK v2.3.2 was used with the parameters -skip_ani_screen –extension fa<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>, referencing the GTDB database release 214. A phylogenetic tree was then generated using the infer command of GTDB-TK with default parameters. The tree was subsequently visualized and annotated using iTOL v6<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Viral contig identification, taxonomic classification and lifestyle prediction</h3>
<p id="Par37">Viral contigs (primarily DNA viruses) were recovered from the metagenomic assemblies using a combination of tools, including DeepVirFinder<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup>, VirSorter2<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>, VIBRANT<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>, and CheckV<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>, with strict quality thresholds. First, metagenomic contigs of at least 10,000 bp in length were identified through DeepVirFinder v1.0 (-l 10000) and VirSorter2 v2.2.4 (-min-length 10000 -keep-original-seq). Contigs were retained if they achieved a score ≥ 0.9 and <em>p</em> ≤ 0.01 in DeepVirFinder, or if they had a score ≥ 0.9 in VirSorter2, or a score ≥ 0.7 with hallmark genes present<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Next, the selected contigs underwent further classification using VIBRANT v1.2.1 with the -virome parameter, where only high- and medium-quality contigs were kept<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. These contigs were assessed (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>) and trimmed using CheckV v1.0.1 with default parameters to eliminate potential host regions, discarding those containing host genes without any viral genes detected. Clean viral contigs were clustered into species-level viral operational taxonomic units (vOTUs) at 95% nucleotide identity using CD-HIT v4.8.1<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup> with the parameters -c 0.95 and -aS 0.8.</p>
<p id="Par38">Taxonomic classification of viral sequences was initially performed using the end-to-end command in geNomad v1.7.1 with default parameters<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup>, yielding a relatively high classification rate at the class level but lower resolution at the family level (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). To improve annotation, vContact3 v3.0.0b65 with the v220 database (<a href="https://bitbucket.org/MAVERICLab/vcontact3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bitbucket.org/MAVERICLab/vcontact3</a>) was subsequently employed (-db-domain prokaryotes -db-version 220), which also resulted in a higher annotation rate at the class level but limited success at the family level (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>).</p>
<p id="Par39">The viral lifestyle was predicted using a combination of CheckV<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>, VIBRANT<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>, and PhaTYP<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup>. Specifically, vOTUs identified as proviruses by CheckV, labeled as lysogenic by VIBRANT, or predicted to be temperate (score of ≥ 0.8) by PhaTYP v0.3.0 with default parameters were categorized as temperate viruses. CheckV and VIBRANT detected temperate viruses by identifying contigs with proviral integration sites or integrase genes<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup>, while PhaTYP used machine learning to identify characteristic patterns of protein composition and associations to predict temperate lifestyles<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup>. A virus was classified as temperate if any of the tools identified it as lysogenic, while those not identified as temperate were classified as potential virulent viruses. Analysis of a subset of metatranscriptomic sequencing data confirmed the transcriptional activity of lysogeny-related genes in temperate viruses, with many of these genes showing active expression (Supplementary Methods, Supplementary Disscussion, and Supplementary Table <a href="#MOESM1" class="usa-link">11</a>).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Abundance quantification of vOTUs and MAGs</h3>
<p id="Par40">The abundances of vOTUs and MAGs were determined using transcripts per million (TPM) via the coverM tool (v0.6.1; <a href="https://github.com/wwood/CoverM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/wwood/CoverM</a>)<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>. For this, quality-trimmed metagenomic reads were mapped to dereplicated vOTUs and MAGs, employing the contig and genome modules, respectively. Alignments with a read identity of ≤ 95% or an aligned percentage of ≤ 75% were discarded to ensure quality (-min-read-percent-identity 95 -min-read-aligned-percent 75)<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup>. A threshold of 10 was set as a confidence filter (-min-covered-fraction 10), and values below this threshold were excluded from the analysis<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>. The relative abundance of each vOTU and MAG was expressed as their proportion of the total TPM count within each sample.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Host prediction of viruses</h3>
<p id="Par41">Four approaches were employed to establish potential links between vOTUs and their prokaryotic host genomes<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>–<a href="#CR98" class="usa-link" aria-describedby="CR98">98</a></sup>: (i) CRISPR spacer matches, (ii) tRNA sequence similarity, (iii) nucleotide homology, and (iv) oligonucleotide frequency (ONF) comparison. For the CRISPR-based method, MAGs were screened for CRISPR spacers using CRT v1.2 with the parameters -minNR 4 -maxRL 55 -maxSL 70<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a>,<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup>. Virus-host associations were subsequently identified with SpacePHARER v5.c2e680a through spacer matches<sup><a href="#CR100" class="usa-link" aria-describedby="CR100">100</a></sup>. In the tRNA approach, tRNA sequences from both vOTUs and MAGs were detected using tRNAscan-SE v2.0.12 with the -G -Q parameters<sup><a href="#CR101" class="usa-link" aria-describedby="CR101">101</a></sup>, followed by BLASTn v2.15.0 comparisons, where results with 100% identity and query coverage were considered<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup>. For nucleotide sequence homology, BLASTn was used to compare vOTUs and MAGs, with virus-host pairings confirmed if the matches had an e-value ≤ 10<sup>−5</sup>, nucleotide identity ≥ 90%, and alignment length ≥ 2000 bp<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>. Lastly, for ONF analysis, VirHostMatcher v1.0.0 with default parameters was applied to compute distances between vOTUs and MAGs based on ONF profiles, identifying potential linkages for d<sub>2</sub>* values ≤ 0.2<sup><a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>,<a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>,<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a></sup>. To investigate the potential impacts of multiple stressors on virus-host predator-prey linkages, the virus-to-host ratio (VHR) was calculated using the linked virus-host pairs, with values log<sub>10</sub>-transformed for analysis<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Identification of viral auxiliary metabolic genes and functional annonation</h3>
<p id="Par42">All vOTUs were annotated using the DRAM-v workflow<sup><a href="#CR105" class="usa-link" aria-describedby="CR105">105</a></sup>. First, vOTUs were processed through VirSorter2 to generate affi-contigs.tab files, which were then used as input for DRAM-v.py annotate with the parameters -use_uniref -min_contig_size 0 in DRAM-v v1.5.0. Since some sequences identified by DeepVirFinder were not classified as viral by VirSorter2, we applied the -min-score 0 parameter<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>. Putative auxiliary metabolic genes (AMGs) were then summarized using DRAM-v.py distill (default parameters) with the following criteria: viral genes flagged for metabolism (-M) and assigned confidence scores between 1 and 3<sup><a href="#CR105" class="usa-link" aria-describedby="CR105">105</a></sup>. For further refinement, we manually curated candidate AMGs using these methods: (i) genomic context of the candidate AMGs was assessed based on CheckV and VirSorter2 annotations to confirm the presence of viral hallmark or viral-like genes upstream or downstream of the AMG<sup><a href="#CR106" class="usa-link" aria-describedby="CR106">106</a></sup>; (ii) illegitimate AMGs linked to nucleotide metabolism, ribosomal proteins, transcriptional/translational regulators, glycoside hydrolases, peptidases, glycosyl transferases, adenylyltransferases, methyltransferases, and structural proteins were excluded, as these are involved in viral replication, invasion, and structure, as noted in prior studies<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR107" class="usa-link" aria-describedby="CR107">107</a>–<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a></sup>. The finalized AMGs were then annotated using the KEGG database<sup><a href="#CR110" class="usa-link" aria-describedby="CR110">110</a></sup> to evaluate their roles in various metabolic pathways.</p>
<p id="Par43">To ensure AMG abundances were derived solely from viruses and not confounded by host-derived sequences, the sum of abundances of viral contigs containing AMGs was used to represent AMG abundances<sup><a href="#CR111" class="usa-link" aria-describedby="CR111">111</a></sup>. To enable competitive mapping, microbial contigs from MAGs containing AMG counterpart genes—defined by identical KEGG Orthology annotations—were also incorporated into the reference database<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. AMGs were grouped at the gene level for further analyses, including diversity, composition, and abundance. Gene-specific abundances were determined by aggregating the abundances of all AMGs associated with each gene.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par44">All statistical analyses were conducted using R software v4.1.3. Alpha diversity metrics, including richness, Shannon’s index, Simpson’s index, and Pielou’s index, as well as beta diversity, were evaluated using the vegan package v2.6-4<sup><a href="#CR112" class="usa-link" aria-describedby="CR112">112</a></sup>. Non-metric multidimensional scaling (NMDS) based on Bray-Curtis distances was performed using the metaMDS function in vegan, and significance was tested through analysis of similarity and permutational multivariate analysis of variance with 999 permutations. Correlations between log<sub>10</sub>-transformed virus and host abundances within each group were calculated using the corr.test function from the psych package v2.3.3<sup><a href="#CR113" class="usa-link" aria-describedby="CR113">113</a></sup>. To visualize the shared vOTUs or MAGs across treatments, we utilized an Upset plot, implemented using the ggupset v0.4.1<sup><a href="#CR114" class="usa-link" aria-describedby="CR114">114</a></sup> and ggplot2 v3.5.1<sup><a href="#CR115" class="usa-link" aria-describedby="CR115">115</a></sup> packages.</p>
<p id="Par45">Cross-kingdom co-occurrence networks constructed using the corAndPvalue function from the WGCNA package v1.72.1<sup><a href="#CR116" class="usa-link" aria-describedby="CR116">116</a></sup>. Only species (MAGs and vOTUs) present in at least four replicates per treatment were included in the analysis<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Significant correlations were defined by a Spearman’s correlation coefficient &gt; 0.6 and a FDR-corrected <em>p</em>-value &lt; 0.01<sup><a href="#CR117" class="usa-link" aria-describedby="CR117">117</a></sup>. Network visualization was performed using the igraph package v1.3.4<sup><a href="#CR118" class="usa-link" aria-describedby="CR118">118</a></sup> and Gephi v0.10.1. Nodes represent species that exhibit significant interspecies interactions within a given treatment, with edges representing these relationships. Subnetworks were extracted by retaining nodes (and corresponding edges) from individual mesocosms using the induced_subgraph function in igraph<sup><a href="#CR119" class="usa-link" aria-describedby="CR119">119</a></sup>. Network topological properties—including nodes, links (edges), average K, average path length, number of connected components, and relative modularity—were calculated using the net_properties.4 function in the ggClusterNet package v0.1.0<sup><a href="#CR120" class="usa-link" aria-describedby="CR120">120</a></sup>. An increased values of nodes, links, average K, and shorter average path lengths indicate greater potential complexity within a microbiome network<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>,<a href="#CR122" class="usa-link" aria-describedby="CR122">122</a></sup>. Increased modularity or a greater number of connected components signifies a shift toward a high-modularity structure in microbial networks, which aids in mitigating the effects of species extinction within the network<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>,<a href="#CR123" class="usa-link" aria-describedby="CR123">123</a>–<a href="#CR126" class="usa-link" aria-describedby="CR126">126</a></sup>. Network stability metrics (i.e., robustness and vulnerability) were assessed using previously established methods<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>, with the corresponding R scripts available at <a href="https://github.com/Mengting-Maggie-Yuan/warming-network-complexity-stability" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Mengting-Maggie-Yuan/warming-network-complexity-stability</a>. Network robustness was defined as the proportion of species remaining after the random removal of nodes. To assess this, each network underwent 100 simulations in which 50% of its nodes were randomly removed<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Robustness values were then calculated for each simulation, and differences in robustness across networks were compared. Vulnerability of each node measures the relative contribution of the node to the global efficiency. Network-level vulnerability is defined by the highest individual node vulnerability<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Global efficiency was calculated as the average efficiency across all node pairs. Variations in network parameters for both the overall network and its subnetworks in response to stressors were found to be consistent (Fig. <a href="#Fig4" class="usa-link">4B–H</a>, Supplementary Table <a href="#MOESM1" class="usa-link">7</a>).</p>
<p id="Par46">Alpha diversity, the relative abundance of broadly shared vOTUs or MAGs (i.e., those found in all eight treatments), log<sub>10</sub> (VHR), and network properties were analyzed using generalized linear models (GLMs) with Gaussian distribution, with warming, nutrients, and pesticides (individually and in combination) as fixed factors. Residual normality was checked using quantile–quantile plots and Shapiro-Wilk tests<sup><a href="#CR127" class="usa-link" aria-describedby="CR127">127</a></sup>. Post hoc pairwise comparisons were performed using the emmeans package v1.8.6<sup><a href="#CR128" class="usa-link" aria-describedby="CR128">128</a></sup>. If data failed to meet normality assumptions (e.g., Simpson’s index, robustness), Aligned Rank Transform (ART) was applied using the ARTool package v0.11.1<sup><a href="#CR129" class="usa-link" aria-describedby="CR129">129</a></sup>, followed by post hoc pairwise comparisons via the rstatix package v0.7.2 using the pairwise_wilcox_test function<sup><a href="#CR130" class="usa-link" aria-describedby="CR130">130</a></sup>. The Benjamini-Hochberg method was used to adjust <em>p</em>-values to control the false discovery rate<sup><a href="#CR131" class="usa-link" aria-describedby="CR131">131</a></sup>. Statistical significance was set at <em>p</em> &lt; 0.05. Differences in gene abundance were assessed using linear discriminant analysis (LDA) effect size in LEfSe v1.1.01 with default settings<sup><a href="#CR132" class="usa-link" aria-describedby="CR132">132</a></sup>, and LEfSe was applied to identify genes differentially represented between treatments and controls.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par47">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec20"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63162_MOESM2_ESM.pdf</a><sup> (7.6KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63162_MOESM3_ESM.xlsx</a><sup> (346.3KB, xlsx) </sup><p>Supplementary Data 1 and Supplementary Data 2</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (110.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.9MB, pdf) </sup>
</div></div></section></section><section id="Sec21"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (4.2MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This research was supported by the National Natural Science Foundation of China (Grant Nos. 42377469 [J.X.] and 32001151 [H.W.]); the International Cooperation Project of the Chinese Academy of Sciences (Grant No. 152342KYSB20190025 [J.X.]); the Hainan University Start-up Funding for Scientific Research (Grant Nos. KYQD (ZR)−23, 086 [J.X.] and KYQD (ZR)-23, 087 [H.W.]); and the Key Research and Development Program of Jiangxi province (Grant No. 20243BBH81037 [H.Z.]).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>T.W., P.Z., H.Z., H.W., M.Z., and J.X. designed and performed the research; T.W., M.Z., and J.X. analyzed the data; T.W., M.Z. and J.X. wrote the original manuscript; and T.W., P.Z., K.A., M.Z., and J.X. contributed to revisions. All authors reviewed and approved the final version.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par48"><em>Nature Communications</em> thanks Amy Zimmerman, and the other, anonymous, reviewer for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All raw sequencing data generated in this study have been deposited in the NCBI Sequence Read Archive (SRA) database under project ID <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1189868" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA1189868</a>. The primary data supporting the results of this study are publicly available on Figshare<sup><a href="#CR133" class="usa-link" aria-describedby="CR133">133</a></sup> [10.6084/m9.figshare.29040674] and KNB [10.5063/F1SJ1J3F]. <a href="#Sec21" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>The code and programs used for analyses are described in the Methods, and are also available at 10.6084/m9.figshare.29040674.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par49">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Min Zhang, Email: zhm7875@mail.hzau.edu.cn.</p>
<p>Jun Xu, Email: xujun@ihb.ac.cn.</p></section><section id="sec22"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63162-2.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Breitbart, M., Bonnain, C., Malki, K. &amp; Sawaya, N. A. Phage puppet masters of the marine microbial realm. <em>Nat. Microbiol.</em><strong>3</strong>, 754–766 (2018).
</cite> [<a href="https://doi.org/10.1038/s41564-018-0166-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29867096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Breitbart,%20M.,%20Bonnain,%20C.,%20Malki,%20K.%20&amp;%20Sawaya,%20N.%20A.%20Phage%20puppet%20masters%20of%20the%20marine%20microbial%20realm.%20Nat.%20Microbiol.3,%20754%E2%80%93766%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Jansson, J. K. &amp; Wu, R. N. Soil viral diversity, ecology and climate change. <em>Nat. Rev. Microbiol.</em><strong>21</strong>, 296–311 (2023).
</cite> [<a href="https://doi.org/10.1038/s41579-022-00811-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36352025/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jansson,%20J.%20K.%20&amp;%20Wu,%20R.%20N.%20Soil%20viral%20diversity,%20ecology%20and%20climate%20change.%20Nat.%20Rev.%20Microbiol.21,%20296%E2%80%93311%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Breitbart, M. Marine viruses: truth or dare. <em>Annu. Rev. Mar. Sci.</em><strong>4</strong>, 425–448 (2012).</cite> [<a href="https://doi.org/10.1146/annurev-marine-120709-142805" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22457982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Breitbart,%20M.%20Marine%20viruses:%20truth%20or%20dare.%20Annu.%20Rev.%20Mar.%20Sci.4,%20425%E2%80%93448%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Zhang, Q. Y. &amp; Gui, J. F. Diversity, evolutionary contribution and ecological roles of aquatic viruses. <em>Sci. China Life Sci.</em><strong>61</strong>, 1486–1502 (2018).
</cite> [<a href="https://doi.org/10.1007/s11427-018-9414-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30443861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Q.%20Y.%20&amp;%20Gui,%20J.%20F.%20Diversity,%20evolutionary%20contribution%20and%20ecological%20roles%20of%20aquatic%20viruses.%20Sci.%20China%20Life%20Sci.61,%201486%E2%80%931502%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Carreira, C. et al. Integrating viruses into soil food web biogeochemistry. <em>Nat. Microbiol.</em><strong>9</strong>, 1918–1928 (2024).
</cite> [<a href="https://doi.org/10.1038/s41564-024-01767-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39095499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carreira,%20C.%20et%20al.%20Integrating%20viruses%20into%20soil%20food%20web%20biogeochemistry.%20Nat.%20Microbiol.9,%201918%E2%80%931928%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Santos-Medellín, C., Blazewicz, S. J., Pett-Ridge, J., Firestone, M. K. &amp; Emerson, J. B. Viral but not bacterial community successional patterns reflect extreme turnover shortly after rewetting dry soils. <em>Nat. Ecol. Evol.</em><strong>7</strong>, 1809–1822 (2023).
</cite> [<a href="https://doi.org/10.1038/s41559-023-02207-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37770548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santos-Medell%C3%ADn,%20C.,%20Blazewicz,%20S.%20J.,%20Pett-Ridge,%20J.,%20Firestone,%20M.%20K.%20&amp;%20Emerson,%20J.%20B.%20Viral%20but%20not%20bacterial%20community%20successional%20patterns%20reflect%20extreme%20turnover%20shortly%20after%20rewetting%20dry%20soils.%20Nat.%20Ecol.%20Evol.7,%201809%E2%80%931822%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Lara, E. et al. Unveiling the role and life strategies of viruses from the surface to the dark ocean. <em>Sci. Adv.</em><strong>3</strong>, e1602565 (2017).
</cite> [<a href="https://doi.org/10.1126/sciadv.1602565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5587022/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28913418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lara,%20E.%20et%20al.%20Unveiling%20the%20role%20and%20life%20strategies%20of%20viruses%20from%20the%20surface%20to%20the%20dark%20ocean.%20Sci.%20Adv.3,%20e1602565%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Suttle, C. A. Marine viruses - major players in the global ecosystem. <em>Nat. Rev. Microbiol.</em><strong>5</strong>, 801–812 (2007).
</cite> [<a href="https://doi.org/10.1038/nrmicro1750" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17853907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suttle,%20C.%20A.%20Marine%20viruses%20-%20major%20players%20in%20the%20global%20ecosystem.%20Nat.%20Rev.%20Microbiol.5,%20801%E2%80%93812%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Braga, L. P. P. et al. Viruses direct carbon cycling in lake sediments under global change. <em>Proc. Natl. Acad. Sci. USA</em><strong>119</strong>, e2202261119 (2022).
</cite> [<a href="https://doi.org/10.1073/pnas.2202261119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9564219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36206369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Braga,%20L.%20P.%20P.%20et%20al.%20Viruses%20direct%20carbon%20cycling%20in%20lake%20sediments%20under%20global%20change.%20Proc.%20Natl.%20Acad.%20Sci.%20USA119,%20e2202261119%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Thingstad, T. F. Elements of a theory for the mechanisms controlling abundance, diversity, and biogeochemical role of lytic bacterial viruses in aquatic systems. <em>Limnol. Oceanogr.</em><strong>45</strong>, 1320–1328 (2000).</cite> [<a href="https://scholar.google.com/scholar_lookup?Thingstad,%20T.%20F.%20Elements%20of%20a%20theory%20for%20the%20mechanisms%20controlling%20abundance,%20diversity,%20and%20biogeochemical%20role%20of%20lytic%20bacterial%20viruses%20in%20aquatic%20systems.%20Limnol.%20Oceanogr.45,%201320%E2%80%931328%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Silveira, C. B. &amp; Rohwer, F. L. Piggyback-the-Winner in host-associated microbial communities. <em>Npj Biofilms Microbiomes</em><strong>2</strong>, 16010 (2016).
</cite> [<a href="https://doi.org/10.1038/npjbiofilms.2016.10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5515262/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28721247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Silveira,%20C.%20B.%20&amp;%20Rohwer,%20F.%20L.%20Piggyback-the-Winner%20in%20host-associated%20microbial%20communities.%20Npj%20Biofilms%20Microbiomes2,%2016010%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Knowles, B. et al. Lytic to temperate switching of viral communities. <em>Nature</em><strong>531</strong>, 466–470 (2016).
</cite> [<a href="https://doi.org/10.1038/nature17193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26982729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Knowles,%20B.%20et%20al.%20Lytic%20to%20temperate%20switching%20of%20viral%20communities.%20Nature531,%20466%E2%80%93470%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Correa, A. M. S. et al. Revisiting the rules of life for viruses of microorganisms. <em>Nat. Rev. Microbiol.</em><strong>19</strong>, 501–513 (2021).
</cite> [<a href="https://doi.org/10.1038/s41579-021-00530-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33762712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Correa,%20A.%20M.%20S.%20et%20al.%20Revisiting%20the%20rules%20of%20life%20for%20viruses%20of%20microorganisms.%20Nat.%20Rev.%20Microbiol.19,%20501%E2%80%93513%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Huang, D. et al. Adaptive strategies and ecological roles of phages in habitats under physicochemical stress. <em>Trends Microbiol.</em><strong>32</strong>, 902–916 (2024).
</cite> [<a href="https://doi.org/10.1016/j.tim.2024.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38433027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20D.%20et%20al.%20Adaptive%20strategies%20and%20ecological%20roles%20of%20phages%20in%20habitats%20under%20physicochemical%20stress.%20Trends%20Microbiol.32,%20902%E2%80%93916%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Howard-Varona, C., Hargreaves, K. R., Abedon, S. T. &amp; Sullivan, M. B. Lysogeny in nature: mechanisms, impact and ecology of temperate phages. <em> ISME J.</em><strong>11</strong>, 1511–1520 (2017).
</cite> [<a href="https://doi.org/10.1038/ismej.2017.16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5520141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28291233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Howard-Varona,%20C.,%20Hargreaves,%20K.%20R.,%20Abedon,%20S.%20T.%20&amp;%20Sullivan,%20M.%20B.%20Lysogeny%20in%20nature:%20mechanisms,%20impact%20and%20ecology%20of%20temperate%20phages.%20ISME%20J.11,%201511%E2%80%931520%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Luo, X. Q. et al. Viral community-wide auxiliary metabolic genes differ by lifestyles, habitats, and hosts. <em>Microbiome</em><strong>10</strong>, 190 (2022).
</cite> [<a href="https://doi.org/10.1186/s40168-022-01384-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9636769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36333738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo,%20X.%20Q.%20et%20al.%20Viral%20community-wide%20auxiliary%20metabolic%20genes%20differ%20by%20lifestyles,%20habitats,%20and%20hosts.%20Microbiome10,%20190%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Hesketh-Best, P. J. et al. Viruses of sulfur oxidizing phototrophs encode genes for pigment, carbon, and sulfur metabolisms. <em>Commun. Earth Environ.</em><strong>4</strong>, 126 (2023).
</cite> [<a href="https://doi.org/10.1038/s43247-023-00796-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11041744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38665202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hesketh-Best,%20P.%20J.%20et%20al.%20Viruses%20of%20sulfur%20oxidizing%20phototrophs%20encode%20genes%20for%20pigment,%20carbon,%20and%20sulfur%20metabolisms.%20Commun.%20Earth%20Environ.4,%20126%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Zheng, X. X. et al. Organochlorine contamination enriches virus-encoded metabolism and pesticide degradation associated auxiliary genes in soil microbiomes. <em> ISME J.</em><strong>16</strong>, 1397–1408 (2022).
</cite> [<a href="https://doi.org/10.1038/s41396-022-01188-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9038774/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35039616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20X.%20X.%20et%20al.%20Organochlorine%20contamination%20enriches%20virus-encoded%20metabolism%20and%20pesticide%20degradation%20associated%20auxiliary%20genes%20in%20soil%20microbiomes.%20ISME%20J.16,%201397%E2%80%931408%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Wu, Z. Z. et al. Distinct adaptive strategies and microbial interactions of soil viruses under different metal(loid) contaminations. <em>J. Hazard. Mater.</em><strong>460</strong>, 132347 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jhazmat.2023.132347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37619274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20Z.%20Z.%20et%20al.%20Distinct%20adaptive%20strategies%20and%20microbial%20interactions%20of%20soil%20viruses%20under%20different%20metal(loid)%20contaminations.%20J.%20Hazard.%20Mater.460,%20132347%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Kieft, K. et al. Ecology of inorganic sulfur auxiliary metabolism in widespread bacteriophages. <em>Nat. Commun.</em><strong>12</strong>, 3503 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-23698-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8190135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34108477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kieft,%20K.%20et%20al.%20Ecology%20of%20inorganic%20sulfur%20auxiliary%20metabolism%20in%20widespread%20bacteriophages.%20Nat.%20Commun.12,%203503%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>IPCC. <em>Climate Change 2014: Synthesis Report, Contribution of Working Groups I, II and III to the Fifth Assessment Report of the Intergovernmental Panel on Climate Change</em> (eds Pachauri, R. K. &amp; Meyer, L. A.) (IPCC, 2014).</cite>
</li>
<li id="CR22">
<span class="label">22.</span><cite>IPCC. <em>Climate Change 2021: the Physical Science Basis. Contribution of Working Group I to the Sixth Assessment Report of the Intergovernmental Panel on Climate Change</em> (eds Masson-Delmotte V. et al.) (Cambridge University Press, 2021).</cite>
</li>
<li id="CR23">
<span class="label">23.</span><cite>Ji, M. et al. Tundra soil viruses mediate responses of microbial communities to climate warming. <em>mBio</em>. <strong>14</strong>, e03009–e03022 (2023).
</cite> [<a href="https://doi.org/10.1128/mbio.03009-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10127799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36786571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ji,%20M.%20et%20al.%20Tundra%20soil%20viruses%20mediate%20responses%20of%20microbial%20communities%20to%20climate%20warming.%20mBio.%2014,%20e03009%E2%80%93e03022%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Sinha, E., Michalak, A. M. &amp; Balaji, V. Eutrophication will increase during the 21st century as a result of precipitation changes. <em>Science</em><strong>357</strong>, 405–408 (2017).
</cite> [<a href="https://doi.org/10.1126/science.aan2409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28751610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sinha,%20E.,%20Michalak,%20A.%20M.%20&amp;%20Balaji,%20V.%20Eutrophication%20will%20increase%20during%20the%2021st%20century%20as%20a%20result%20of%20precipitation%20changes.%20Science357,%20405%E2%80%93408%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Deutsch, C. A. et al. Increase in crop losses to insect pests in a warming climate. <em>Science</em><strong>361</strong>, 916–919 (2018).
</cite> [<a href="https://doi.org/10.1126/science.aat3466" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30166490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deutsch,%20C.%20A.%20et%20al.%20Increase%20in%20crop%20losses%20to%20insect%20pests%20in%20a%20warming%20climate.%20Science361,%20916%E2%80%93919%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Wu, H. Q. et al. Phage-bacterium interactions and nutrient availability can shape C and N retention in microbial biomass. <em>Eur. J. Soil Sci.</em><strong>73</strong>, e13296 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wu,%20H.%20Q.%20et%20al.%20Phage-bacterium%20interactions%20and%20nutrient%20availability%20can%20shape%20C%20and%20N%20retention%20in%20microbial%20biomass.%20Eur.%20J.%20Soil%20Sci.73,%20e13296%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Ruuskanen, S. et al. Ecosystem consequences of herbicides: the role of microbiome. <em>Trends Ecol. Evol.</em><strong>38</strong>, 35–43 (2023).
</cite> [<a href="https://doi.org/10.1016/j.tree.2022.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36243622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruuskanen,%20S.%20et%20al.%20Ecosystem%20consequences%20of%20herbicides:%20the%20role%20of%20microbiome.%20Trends%20Ecol.%20Evol.38,%2035%E2%80%9343%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Sun, M. M., Yuan, S. J., Xia, R., Ye, M. &amp; Balcázar, J. L. Underexplored viral auxiliary metabolic genes in soil: diversity and eco-evolutionary significance. <em>Environ. Microbiol.</em><strong>25</strong>, 800–810 (2023).
</cite> [<a href="https://doi.org/10.1111/1462-2920.16329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36571495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20M.%20M.,%20Yuan,%20S.%20J.,%20Xia,%20R.,%20Ye,%20M.%20&amp;%20Balc%C3%A1zar,%20J.%20L.%20Underexplored%20viral%20auxiliary%20metabolic%20genes%20in%20soil:%20diversity%20and%20eco-evolutionary%20significance.%20Environ.%20Microbiol.25,%20800%E2%80%93810%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Staley, Z. R., Harwood, V. J. &amp; Rohr, J. R. A synthesis of the effects of pesticides on microbial persistence in aquatic ecosystems. <em>Crit. Rev. Toxicol.</em><strong>45</strong>, 813–836 (2015).
</cite> [<a href="https://doi.org/10.3109/10408444.2015.1065471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4750050/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26565685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Staley,%20Z.%20R.,%20Harwood,%20V.%20J.%20&amp;%20Rohr,%20J.%20R.%20A%20synthesis%20of%20the%20effects%20of%20pesticides%20on%20microbial%20persistence%20in%20aquatic%20ecosystems.%20Crit.%20Rev.%20Toxicol.45,%20813%E2%80%93836%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Ram, A. S. P. &amp; Sime-Ngando, T. Functional responses of prokaryotes and viruses to grazer effects and nutrient additions in freshwater microcosms. <em> ISME J.</em><strong>2</strong>, 498–509 (2008).
</cite> [<a href="https://doi.org/10.1038/ismej.2008.15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18273065/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ram,%20A.%20S.%20P.%20&amp;%20Sime-Ngando,%20T.%20Functional%20responses%20of%20prokaryotes%20and%20viruses%20to%20grazer%20effects%20and%20nutrient%20additions%20in%20freshwater%20microcosms.%20ISME%20J.2,%20498%E2%80%93509%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Gregory, A. C. et al. Marine DNA viral macro- and microdiversity from pole to pole. <em>Cell</em><strong>177</strong>, 1109–1123 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.03.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6525058/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31031001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gregory,%20A.%20C.%20et%20al.%20Marine%20DNA%20viral%20macro-%20and%20microdiversity%20from%20pole%20to%20pole.%20Cell177,%201109%E2%80%931123%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>López-Pérez, M., Haro-Moreno, J. M., de la Torre, J. R. &amp; Rodriguez-Valera, F. Novel associated with Marine Group I Thaumarchaeota assembled from metagenomes. <em>Environ. Microbiol.</em><strong>21</strong>, 1980–1988 (2019).
</cite> [<a href="https://doi.org/10.1111/1462-2920.14462" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30370610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%B3pez-P%C3%A9rez,%20M.,%20Haro-Moreno,%20J.%20M.,%20de%20la%20Torre,%20J.%20R.%20&amp;%20Rodriguez-Valera,%20F.%20Novel%20associated%20with%20Marine%20Group%20I%20Thaumarchaeota%20assembled%20from%20metagenomes.%20Environ.%20Microbiol.21,%201980%E2%80%931988%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Malki, K. et al. Prokaryotic and viral community composition of freshwater springs in Florida, USA. <em>mBio</em><strong>11</strong>, e00436–00420 (2020).
</cite> [<a href="https://doi.org/10.1128/mBio.00436-20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7157768/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32265327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Malki,%20K.%20et%20al.%20Prokaryotic%20and%20viral%20community%20composition%20of%20freshwater%20springs%20in%20Florida,%20USA.%20mBio11,%20e00436%E2%80%9300420%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Moon, K. et al. Freshwater viral metagenome reveals novel and functional phage-borne antibiotic resistance genes. <em>Microbiome</em><strong>8</strong>, 75 (2020).
</cite> [<a href="https://doi.org/10.1186/s40168-020-00863-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7265639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32482165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moon,%20K.%20et%20al.%20Freshwater%20viral%20metagenome%20reveals%20novel%20and%20functional%20phage-borne%20antibiotic%20resistance%20genes.%20Microbiome8,%2075%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Bonachela, J. A. Viral plasticity facilitates host diversity in challenging environments. <em>Nat. Commun.</em><strong>15</strong>, 7473 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-51344-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11362530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39209841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bonachela,%20J.%20A.%20Viral%20plasticity%20facilitates%20host%20diversity%20in%20challenging%20environments.%20Nat.%20Commun.15,%207473%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Mojica, K. D. A. &amp; Brussaard, C. P. D. Factors affecting virus dynamics and microbial host–virus interactions in marine environments. <em>FEMS Microbiol. Ecol.</em><strong>89</strong>, 495–515 (2014).
</cite> [<a href="https://doi.org/10.1111/1574-6941.12343" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24754794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mojica,%20K.%20D.%20A.%20&amp;%20Brussaard,%20C.%20P.%20D.%20Factors%20affecting%20virus%20dynamics%20and%20microbial%20host%E2%80%93virus%20interactions%20in%20marine%20environments.%20FEMS%20Microbiol.%20Ecol.89,%20495%E2%80%93515%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Bell, T. H. &amp; Bell, T. Many roads to bacterial generalism. <em>FEMS Microbiol. Ecol.</em><strong>97</strong>, fiaa240 (2021).</cite> [<a href="https://doi.org/10.1093/femsec/fiaa240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33238305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bell,%20T.%20H.%20&amp;%20Bell,%20T.%20Many%20roads%20to%20bacterial%20generalism.%20FEMS%20Microbiol.%20Ecol.97,%20fiaa240%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Meijenfeldt, F. A. B. V., Hogeweg, P. &amp; Dutilh, B. E. A social niche breadth score reveals niche range strategies of generalists and specialists. <em>Nat. Ecol. Evol.</em><strong>7</strong>, 768–781 (2023).
</cite> [<a href="https://doi.org/10.1038/s41559-023-02027-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10172124/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37012375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meijenfeldt,%20F.%20A.%20B.%20V.,%20Hogeweg,%20P.%20&amp;%20Dutilh,%20B.%20E.%20A%20social%20niche%20breadth%20score%20reveals%20niche%20range%20strategies%20of%20generalists%20and%20specialists.%20Nat.%20Ecol.%20Evol.7,%20768%E2%80%93781%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Tan, Y. et al. Enhanced bacterium–phage symbiosis in attached microbial aggregates on a membrane surface facing elevated hydraulic stress. <em>Environ. Sci. Technol.</em><strong>57</strong>, 17324–17337 (2023).
</cite> [<a href="https://doi.org/10.1021/acs.est.3c05452" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37930060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan,%20Y.%20et%20al.%20Enhanced%20bacterium%E2%80%93phage%20symbiosis%20in%20attached%20microbial%20aggregates%20on%20a%20membrane%20surface%20facing%20elevated%20hydraulic%20stress.%20Environ.%20Sci.%20Technol.57,%2017324%E2%80%9317337%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Johnston, E. R. et al. Responses of tundra soil microbial communities to half a decade of experimental warming at two critical depths. <em>Proc. Natl Acad. Sci. USA</em><strong>116</strong>, 15096–15105 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1901307116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6660733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31285347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johnston,%20E.%20R.%20et%20al.%20Responses%20of%20tundra%20soil%20microbial%20communities%20to%20half%20a%20decade%20of%20experimental%20warming%20at%20two%20critical%20depths.%20Proc.%20Natl%20Acad.%20Sci.%20USA116,%2015096%E2%80%9315105%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Feng, J. J. et al. Warming-induced permafrost thaw exacerbates tundra soil carbon decomposition mediated by microbial community. <em>Microbiome</em><strong>8</strong>, 3 (2020).
</cite> [<a href="https://doi.org/10.1186/s40168-019-0778-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6969446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31952472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng,%20J.%20J.%20et%20al.%20Warming-induced%20permafrost%20thaw%20exacerbates%20tundra%20soil%20carbon%20decomposition%20mediated%20by%20microbial%20community.%20Microbiome8,%203%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Santos-Medellín, C. et al. Spatial turnover of soil viral populations and genotypes overlain by cohesive responses to moisture in grasslands. <em>Proc. Natl. Acad. Sci. USA</em><strong>119</strong>, e2209132119 (2022).
</cite> [<a href="https://doi.org/10.1073/pnas.2209132119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9659419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36322723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santos-Medell%C3%ADn,%20C.%20et%20al.%20Spatial%20turnover%20of%20soil%20viral%20populations%20and%20genotypes%20overlain%20by%20cohesive%20responses%20to%20moisture%20in%20grasslands.%20Proc.%20Natl.%20Acad.%20Sci.%20USA119,%20e2209132119%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Chrzanowski, T. H., Sterner, R. W. &amp; Elser, J. J. Nutrient enrichment and nutrient regeneration stimulate bacterioplankton growth. <em>Microb. Ecol.</em><strong>29</strong>, 221–230 (1995).
</cite> [<a href="https://doi.org/10.1007/BF00164886" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24185342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chrzanowski,%20T.%20H.,%20Sterner,%20R.%20W.%20&amp;%20Elser,%20J.%20J.%20Nutrient%20enrichment%20and%20nutrient%20regeneration%20stimulate%20bacterioplankton%20growth.%20Microb.%20Ecol.29,%20221%E2%80%93230%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Yuan, G. et al. Responses of protozoan communities to multiple environmental stresses (warming, eutrophication, and pesticide pollution). <em>Animals</em><strong>14</strong>, 1293 (2024).
</cite> [<a href="https://doi.org/10.3390/ani14091293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11083714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38731297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan,%20G.%20et%20al.%20Responses%20of%20protozoan%20communities%20to%20multiple%20environmental%20stresses%20(warming,%20eutrophication,%20and%20pesticide%20pollution).%20Animals14,%201293%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Sumon, K. A. et al. Effects of imidacloprid on the ecology of sub-tropical freshwater microcosms. <em>Environ. Pollut.</em><strong>236</strong>, 432–441 (2018).
</cite> [<a href="https://doi.org/10.1016/j.envpol.2018.01.102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29414368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sumon,%20K.%20A.%20et%20al.%20Effects%20of%20imidacloprid%20on%20the%20ecology%20of%20sub-tropical%20freshwater%20microcosms.%20Environ.%20Pollut.236,%20432%E2%80%93441%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Howard-Varona, C. et al. Protist impacts on marine cyanovirocell metabolism. <em> ISME Commun.</em><strong>2</strong>, 94 (2022).
</cite> [<a href="https://doi.org/10.1038/s43705-022-00169-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9723779/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37938263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Howard-Varona,%20C.%20et%20al.%20Protist%20impacts%20on%20marine%20cyanovirocell%20metabolism.%20ISME%20Commun.2,%2094%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Chen, X., Weinbauer, M. G., Jiao, N. &amp; Zhang, R. Revisiting marine lytic and lysogenic virus-host interactions: Kill-the-Winner and Piggyback-the-Winner. <em>Sci. Bull.</em><strong>66</strong>, 871–874 (2021).</cite> [<a href="https://doi.org/10.1016/j.scib.2020.12.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36654234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20X.,%20Weinbauer,%20M.%20G.,%20Jiao,%20N.%20&amp;%20Zhang,%20R.%20Revisiting%20marine%20lytic%20and%20lysogenic%20virus-host%20interactions:%20Kill-the-Winner%20and%20Piggyback-the-Winner.%20Sci.%20Bull.66,%20871%E2%80%93874%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Zhu, L. Y. et al. Resource-dependent biodiversity and potential multi-trophic interactions determine belowground functional trait stability. <em>Microbiome</em><strong>11</strong>, 95 (2023).
</cite> [<a href="https://doi.org/10.1186/s40168-023-01539-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10150482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37127665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20L.%20Y.%20et%20al.%20Resource-dependent%20biodiversity%20and%20potential%20multi-trophic%20interactions%20determine%20belowground%20functional%20trait%20stability.%20Microbiome11,%2095%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Wieczynski, D. J. et al. Viral infections likely mediate microbial controls on ecosystem responses to global warming. <em>FEMS Microbiol. Ecol.</em><strong>99</strong>, fiad016 (2023).
</cite> [<a href="https://doi.org/10.1093/femsec/fiad016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36828391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wieczynski,%20D.%20J.%20et%20al.%20Viral%20infections%20likely%20mediate%20microbial%20controls%20on%20ecosystem%20responses%20to%20global%20warming.%20FEMS%20Microbiol.%20Ecol.99,%20fiad016%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Seidel, L. et al. Climate change-related warming reduces thermal sensitivity and modifies metabolic activity of coastal benthic bacterial communities. <em> ISME J.</em><strong>17</strong>, 855–869 (2023).
</cite> [<a href="https://doi.org/10.1038/s41396-023-01395-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10202955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36977742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seidel,%20L.%20et%20al.%20Climate%20change-related%20warming%20reduces%20thermal%20sensitivity%20and%20modifies%20metabolic%20activity%20of%20coastal%20benthic%20bacterial%20communities.%20ISME%20J.17,%20855%E2%80%93869%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Smith, T. P. et al. Community-level respiration of prokaryotic microbes may rise with global warming. <em>Nat. Commun.</em><strong>10</strong>, 5124 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-13109-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6851113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31719536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20T.%20P.%20et%20al.%20Community-level%20respiration%20of%20prokaryotic%20microbes%20may%20rise%20with%20global%20warming.%20Nat.%20Commun.10,%205124%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Blanck, H. A critical review of procedures and approaches used for assessing pollution-induced community tolerance (PICT) in biotic communities. <em>Hum. Ecol. Risk Assess.</em><strong>8</strong>, 1003–1034 (2002).</cite> [<a href="https://scholar.google.com/scholar_lookup?Blanck,%20H.%20A%20critical%20review%20of%20procedures%20and%20approaches%20used%20for%20assessing%20pollution-induced%20community%20tolerance%20(PICT)%20in%20biotic%20communities.%20Hum.%20Ecol.%20Risk%20Assess.8,%201003%E2%80%931034%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Vinebrooke, R. D. et al. Impacts of multiple stressors on biodiversity and ecosystem functioning: the role of species co-tolerance. <em>Oikos</em><strong>104</strong>, 451–457 (2004).</cite> [<a href="https://scholar.google.com/scholar_lookup?Vinebrooke,%20R.%20D.%20et%20al.%20Impacts%20of%20multiple%20stressors%20on%20biodiversity%20and%20ecosystem%20functioning:%20the%20role%20of%20species%20co-tolerance.%20Oikos104,%20451%E2%80%93457%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Chara-Serna, A. M. &amp; Richardson, J. S. Chlorpyrifos interacts with other agricultural stressors to alter stream communities in laboratory microcosms. <em>Ecol. Appl.</em><strong>28</strong>, 162–176 (2018).
</cite> [<a href="https://doi.org/10.1002/eap.1637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29024139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chara-Serna,%20A.%20M.%20&amp;%20Richardson,%20J.%20S.%20Chlorpyrifos%20interacts%20with%20other%20agricultural%20stressors%20to%20alter%20stream%20communities%20in%20laboratory%20microcosms.%20Ecol.%20Appl.28,%20162%E2%80%93176%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Meng, S. D., Tran, T. T., Dinh, K. V., Delnat, V. &amp; Stoks, R. Acute warming increases pesticide toxicity more than transgenerational warming by reducing the energy budget. <em>Sci. Total Environ.</em><strong>805</strong>, 150373 (2022).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2021.150373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34818764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meng,%20S.%20D.,%20Tran,%20T.%20T.,%20Dinh,%20K.%20V.,%20Delnat,%20V.%20&amp;%20Stoks,%20R.%20Acute%20warming%20increases%20pesticide%20toxicity%20more%20than%20transgenerational%20warming%20by%20reducing%20the%20energy%20budget.%20Sci.%20Total%20Environ.805,%20150373%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Vijayaraj, V. et al. Multiple-stressor exposure of aquatic food webs: Nitrate and warming modulate the effect of pesticides. <em>Water Res.</em><strong>216</strong>, 118325 (2022).
</cite> [<a href="https://doi.org/10.1016/j.watres.2022.118325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35349923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vijayaraj,%20V.%20et%20al.%20Multiple-stressor%20exposure%20of%20aquatic%20food%20webs:%20Nitrate%20and%20warming%20modulate%20the%20effect%20of%20pesticides.%20Water%20Res.216,%20118325%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Wang, T. et al. Interactions between climate warming, herbicides, and eutrophication in the aquatic food web. <em>J. Environ. Manag.</em><strong>345</strong>, 118753 (2023).</cite> [<a href="https://doi.org/10.1016/j.jenvman.2023.118753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37625285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20T.%20et%20al.%20Interactions%20between%20climate%20warming,%20herbicides,%20and%20eutrophication%20in%20the%20aquatic%20food%20web.%20J.%20Environ.%20Manag.345,%20118753%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Wu, H., Gao, T., Hu, A. &amp; Wang, J. Network complexity and stability of microbes enhanced by microplastic diversity. <em>Environ. Sci. Technol.</em><strong>58</strong>, 4334–4345 (2024).
</cite> [<a href="https://doi.org/10.1021/acs.est.3c08704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38382548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20H.,%20Gao,%20T.,%20Hu,%20A.%20&amp;%20Wang,%20J.%20Network%20complexity%20and%20stability%20of%20microbes%20enhanced%20by%20microplastic%20diversity.%20Environ.%20Sci.%20Technol.58,%204334%E2%80%934345%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Yuan, M. M. et al. Climate warming enhances microbial network complexity and stability. <em>Nat. Clim. change</em><strong>11</strong>, 343–348 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yuan,%20M.%20M.%20et%20al.%20Climate%20warming%20enhances%20microbial%20network%20complexity%20and%20stability.%20Nat.%20Clim.%20change11,%20343%E2%80%93348%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Hernandez, D. J., David, A. S., Menges, E. S., Searcy, C. A. &amp; Afkhami, M. E. Environmental stress destabilizes microbial networks. <em> ISME J.</em><strong>15</strong>, 1722–1734 (2021).
</cite> [<a href="https://doi.org/10.1038/s41396-020-00882-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8163744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33452480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hernandez,%20D.%20J.,%20David,%20A.%20S.,%20Menges,%20E.%20S.,%20Searcy,%20C.%20A.%20&amp;%20Afkhami,%20M.%20E.%20Environmental%20stress%20destabilizes%20microbial%20networks.%20ISME%20J.15,%201722%E2%80%931734%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Zhou, Y. et al. Warming reshaped the microbial hierarchical interactions. <em>Glob. Change Biol.</em><strong>27</strong>, 6331–6347 (2021).</cite> [<a href="https://doi.org/10.1111/gcb.15891" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34544207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Y.%20et%20al.%20Warming%20reshaped%20the%20microbial%20hierarchical%20interactions.%20Glob.%20Change%20Biol.27,%206331%E2%80%936347%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Brown, C. J., Saunders, M. I., Possingham, H. P. &amp; Richardson, A. J. Managing for interactions between local and global stressors of ecosystems. <em>PLoS ONE</em><strong>8</strong>, e65765 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0065765" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3680442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23776542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20C.%20J.,%20Saunders,%20M.%20I.,%20Possingham,%20H.%20P.%20&amp;%20Richardson,%20A.%20J.%20Managing%20for%20interactions%20between%20local%20and%20global%20stressors%20of%20ecosystems.%20PLoS%20ONE8,%20e65765%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Zhou, L. &amp; Wang, S. The bright side of ecological stressors. <em>Trends Ecol. Evol.</em><strong>38</strong>, 568–578 (2023).
</cite> [<a href="https://doi.org/10.1016/j.tree.2023.01.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36906435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20L.%20&amp;%20Wang,%20S.%20The%20bright%20side%20of%20ecological%20stressors.%20Trends%20Ecol.%20Evol.38,%20568%E2%80%93578%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Liao, H. P. et al. Mesophilic and thermophilic viruses are associated with nutrient cycling during hyperthermophilic composting. <em> ISME J.</em><strong>17</strong>, 916–930 (2023).
</cite> [<a href="https://doi.org/10.1038/s41396-023-01404-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10202948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37031344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao,%20H.%20P.%20et%20al.%20Mesophilic%20and%20thermophilic%20viruses%20are%20associated%20with%20nutrient%20cycling%20during%20hyperthermophilic%20composting.%20ISME%20J.17,%20916%E2%80%93930%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Zhou, Z. et al. Unravelling viral ecology and evolution over 20 years in a freshwater lake. <em>Nat. Microbiol.</em><strong>10</strong>, 231–245 (2025).
</cite> [<a href="https://doi.org/10.1038/s41564-024-01876-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39753667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Z.%20et%20al.%20Unravelling%20viral%20ecology%20and%20evolution%20over%2020%E2%80%89years%20in%20a%20freshwater%20lake.%20Nat.%20Microbiol.10,%20231%E2%80%93245%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Wang, T. et al. A dynamic temperature difference control recording system in shallow lake mesocosm. <em>MethodsX</em><strong>7</strong>, 100930 (2020).
</cite> [<a href="https://doi.org/10.1016/j.mex.2020.100930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7303671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32577407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20T.%20et%20al.%20A%20dynamic%20temperature%20difference%20control%20recording%20system%20in%20shallow%20lake%20mesocosm.%20MethodsX7,%20100930%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Gong, Z., Li, Y. &amp; Xie, P. Population dynamics and production of <em>Bellamya aeruginosa</em> (Reeve) (Mollusca: Viviparidae) in Lake Donghu, Wuhan. <em>J. Lake Sci.</em><strong>21</strong>, 401–407 (2009).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gong,%20Z.,%20Li,%20Y.%20&amp;%20Xie,%20P.%20Population%20dynamics%20and%20production%20of%20Bellamya%20aeruginosa%20(Reeve)%20(Mollusca:%20Viviparidae)%20in%20Lake%20Donghu,%20Wuhan.%20J.%20Lake%20Sci.21,%20401%E2%80%93407%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Mao, Z. et al. Pelagic energy flow supports the food web of a shallow lake following a dramatic regime shift driven by water level changes. <em>Sci. Total Environ.</em><strong>756</strong>, 143642 (2021).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2020.143642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33302070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao,%20Z.%20et%20al.%20Pelagic%20energy%20flow%20supports%20the%20food%20web%20of%20a%20shallow%20lake%20following%20a%20dramatic%20regime%20shift%20driven%20by%20water%20level%20changes.%20Sci.%20Total%20Environ.756,%20143642%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Yu, J. et al. Changes in pelagic fish community composition, abundance, and biomass along a productivity gradient in subtropical lakes. <em>Water</em><strong>13</strong>, 858 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yu,%20J.%20et%20al.%20Changes%20in%20pelagic%20fish%20community%20composition,%20abundance,%20and%20biomass%20along%20a%20productivity%20gradient%20in%20subtropical%20lakes.%20Water13,%20858%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Zhi, Y., Liu, Y., Li, W. &amp; Cao, Y. Responses of four submerged macrophytes to freshwater snail density (<em>Radix swinhoei</em>) under clear-water conditions: a mesocosm study. <em>Ecol. Evol.</em><strong>10</strong>, 7644–7653 (2020).
</cite> [<a href="https://doi.org/10.1002/ece3.6489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7391322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32760554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhi,%20Y.,%20Liu,%20Y.,%20Li,%20W.%20&amp;%20Cao,%20Y.%20Responses%20of%20four%20submerged%20macrophytes%20to%20freshwater%20snail%20density%20(Radix%20swinhoei)%20under%20clear-water%20conditions:%20a%20mesocosm%20study.%20Ecol.%20Evol.10,%207644%E2%80%937653%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Standards CN. <em>Chinese National Standards for Surface Water Quality Analysis</em>. (Standards Press of China, 1996).</cite>
</li>
<li id="CR72">
<span class="label">72.</span><cite>Sayer, C. D., Davidson, T. A. &amp; Jones, J. I. Seasonal dynamics of macrophytes and phytoplankton in shallow lakes: a eutrophication-driven pathway from plants to plankton?. <em>Freshw. Biol.</em><strong>55</strong>, 500–513 (2010).</cite> [<a href="https://scholar.google.com/scholar_lookup?Sayer,%20C.%20D.,%20Davidson,%20T.%20A.%20&amp;%20Jones,%20J.%20I.%20Seasonal%20dynamics%20of%20macrophytes%20and%20phytoplankton%20in%20shallow%20lakes:%20a%20eutrophication-driven%20pathway%20from%20plants%20to%20plankton?.%20Freshw.%20Biol.55,%20500%E2%80%93513%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Egli, M. et al. A One-Health environmental risk assessment of contaminants of emerging concern in London’s waterways throughout the SARS-CoV-2 pandemic. <em>Environ. Int.</em><strong>180</strong>, 108210 (2023).
</cite> [<a href="https://doi.org/10.1016/j.envint.2023.108210" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37778289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Egli,%20M.%20et%20al.%20A%20One-Health%20environmental%20risk%20assessment%20of%20contaminants%20of%20emerging%20concern%20in%20London%E2%80%99s%20waterways%20throughout%20the%20SARS-CoV-2%20pandemic.%20Environ.%20Int.180,%20108210%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Zhu, B. Y., Chen, J., Gu, L., Zhuan, M. J. &amp; Xu, W. X. Runoff response to 1.5°C and 2.0°C global warming for the Yangtze River basin. <em>Adv. Clim. Change Res.</em><strong>16</strong>, 690–705 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhu,%20B.%20Y.,%20Chen,%20J.,%20Gu,%20L.,%20Zhuan,%20M.%20J.%20&amp;%20Xu,%20W.%20X.%20Runoff%20response%20to%201.5%C2%B0C%20and%202.0%C2%B0C%20global%20warming%20for%20the%20Yangtze%20River%20basin.%20Adv.%20Clim.%20Change%20Res.16,%20690%E2%80%93705%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Jackson, M. C., Pawar, S. &amp; Woodward, G. The temporal dynamics of multiple stressor effects: from individuals to ecosystems. <em>Trends Ecol. Evol.</em><strong>36</strong>, 402–410 (2021).
</cite> [<a href="https://doi.org/10.1016/j.tree.2021.01.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33583600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jackson,%20M.%20C.,%20Pawar,%20S.%20&amp;%20Woodward,%20G.%20The%20temporal%20dynamics%20of%20multiple%20stressor%20effects:%20from%20individuals%20to%20ecosystems.%20Trends%20Ecol.%20Evol.36,%20402%E2%80%93410%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Jeppesen, E., Sondergaard, M., Meerhoff, M., Lauridsen, T. L. &amp; Jensen, J. P. Shallow lake restoration by nutrient loading reduction - some recent findings and challenges ahead. <em>Hydrobiologia</em><strong>584</strong>, 239–252 (2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jeppesen,%20E.,%20Sondergaard,%20M.,%20Meerhoff,%20M.,%20Lauridsen,%20T.%20L.%20&amp;%20Jensen,%20J.%20P.%20Shallow%20lake%20restoration%20by%20nutrient%20loading%20reduction%20-%20some%20recent%20findings%20and%20challenges%20ahead.%20Hydrobiologia584,%20239%E2%80%93252%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Morrissey, C. A. et al. Neonicotinoid contamination of global surface waters and associated risk to aquatic invertebrates: a review. <em>Environ. Int.</em><strong>74</strong>, 291–303 (2015).
</cite> [<a href="https://doi.org/10.1016/j.envint.2014.10.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25454246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morrissey,%20C.%20A.%20et%20al.%20Neonicotinoid%20contamination%20of%20global%20surface%20waters%20and%20associated%20risk%20to%20aquatic%20invertebrates:%20a%20review.%20Environ.%20Int.74,%20291%E2%80%93303%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Uritskiy, G. V., DiRuggiero, J. &amp; Taylor, J. MetaWRAP-a flexible pipeline for genome-resolved metagenomic data analysis. <em>Microbiome</em><strong>6</strong>, 158 (2018).
</cite> [<a href="https://doi.org/10.1186/s40168-018-0541-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6138922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30219103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uritskiy,%20G.%20V.,%20DiRuggiero,%20J.%20&amp;%20Taylor,%20J.%20MetaWRAP-a%20flexible%20pipeline%20for%20genome-resolved%20metagenomic%20data%20analysis.%20Microbiome6,%20158%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Li, D. H. et al. MEGAHIT v1.0: A fast and scalable metagenome assembler driven by advanced methodologies and community practices. <em>Methods</em><strong>102</strong>, 3–11 (2016).
</cite> [<a href="https://doi.org/10.1016/j.ymeth.2016.02.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27012178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20D.%20H.%20et%20al.%20MEGAHIT%20v1.0:%20A%20fast%20and%20scalable%20metagenome%20assembler%20driven%20by%20advanced%20methodologies%20and%20community%20practices.%20Methods102,%203%E2%80%9311%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Alneberg, J. et al. Binning metagenomic contigs by coverage and composition. <em>Nat. Methods</em><strong>11</strong>, 1144–1146 (2014).
</cite> [<a href="https://doi.org/10.1038/nmeth.3103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25218180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alneberg,%20J.%20et%20al.%20Binning%20metagenomic%20contigs%20by%20coverage%20and%20composition.%20Nat.%20Methods11,%201144%E2%80%931146%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Wu, Y. W., Simmons, B. A. &amp; Singer, S. W. MaxBin 2.0: an automated binning algorithm to recover genomes from multiple metagenomic datasets. <em>Bioinformatics</em><strong>32</strong>, 605–607 (2016).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26515820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20Y.%20W.,%20Simmons,%20B.%20A.%20&amp;%20Singer,%20S.%20W.%20MaxBin%202.0:%20an%20automated%20binning%20algorithm%20to%20recover%20genomes%20from%20multiple%20metagenomic%20datasets.%20Bioinformatics32,%20605%E2%80%93607%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Kang, D. W. D., Froula, J., Egan, R. &amp; Wang, Z. MetaBAT, an efficient tool for accurately reconstructing single genomes from complex microbial communities. <em>PeerJ</em><strong>3</strong>, e1165 (2015).
</cite> [<a href="https://doi.org/10.7717/peerj.1165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4556158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26336640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kang,%20D.%20W.%20D.,%20Froula,%20J.,%20Egan,%20R.%20&amp;%20Wang,%20Z.%20MetaBAT,%20an%20efficient%20tool%20for%20accurately%20reconstructing%20single%20genomes%20from%20complex%20microbial%20communities.%20PeerJ3,%20e1165%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. &amp; Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. <em>Genome Res.</em><strong>25</strong>, 1043–1055 (2015).
</cite> [<a href="https://doi.org/10.1101/gr.186072.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4484387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25977477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Parks,%20D.%20H.,%20Imelfort,%20M.,%20Skennerton,%20C.%20T.,%20Hugenholtz,%20P.%20&amp;%20Tyson,%20G.%20W.%20CheckM:%20assessing%20the%20quality%20of%20microbial%20genomes%20recovered%20from%20isolates,%20single%20cells,%20and%20metagenomes.%20Genome%20Res.25,%201043%E2%80%931055%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Olm, M. R., Brown, C. T., Brooks, B. &amp; Banfield, J. F. dRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication. <em> ISME J.</em><strong>11</strong>, 2864–2868 (2017).
</cite> [<a href="https://doi.org/10.1038/ismej.2017.126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5702732/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28742071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Olm,%20M.%20R.,%20Brown,%20C.%20T.,%20Brooks,%20B.%20&amp;%20Banfield,%20J.%20F.%20dRep:%20a%20tool%20for%20fast%20and%20accurate%20genomic%20comparisons%20that%20enables%20improved%20genome%20recovery%20from%20metagenomes%20through%20de-replication.%20ISME%20J.11,%202864%E2%80%932868%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Chaumeil, P. A., Mussig, A. J., Hugenholtz, P. &amp; Parks, D. H. GTDB-Tk v2: memory friendly classification with the genome taxonomy database. <em>Bioinformatics</em><strong>38</strong>, 5315–5316 (2022).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btac672" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9710552/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36218463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaumeil,%20P.%20A.,%20Mussig,%20A.%20J.,%20Hugenholtz,%20P.%20&amp;%20Parks,%20D.%20H.%20GTDB-Tk%20v2:%20memory%20friendly%20classification%20with%20the%20genome%20taxonomy%20database.%20Bioinformatics38,%205315%E2%80%935316%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Letunic, I. &amp; Bork, P. Interactive Tree of Life (iTOL) v6: recent updates to the phylogenetic tree display and annotation tool. <em>Nucleic Acids Res.</em><strong>52</strong>, W78–W82 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkae268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38613393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Letunic,%20I.%20&amp;%20Bork,%20P.%20Interactive%20Tree%20of%20Life%20(iTOL)%20v6:%20recent%20updates%20to%20the%20phylogenetic%20tree%20display%20and%20annotation%20tool.%20Nucleic%20Acids%20Res.52,%20W78%E2%80%93W82%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Ren, J. et al. Identifying viruses from metagenomic data using deep learning. <em>Quant. Biol.</em><strong>8</strong>, 64–77 (2020).
</cite> [<a href="https://doi.org/10.1007/s40484-019-0187-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8172088/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34084563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ren,%20J.%20et%20al.%20Identifying%20viruses%20from%20metagenomic%20data%20using%20deep%20learning.%20Quant.%20Biol.8,%2064%E2%80%9377%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Guo, J. R. et al. VirSorter2: a multi-classifier, expert-guided approach to detect diverse DNA and RNA viruses. <em>Microbiome</em><strong>9</strong>, 37 (2021).
</cite> [<a href="https://doi.org/10.1186/s40168-020-00990-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7852108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33522966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20J.%20R.%20et%20al.%20VirSorter2:%20a%20multi-classifier,%20expert-guided%20approach%20to%20detect%20diverse%20DNA%20and%20RNA%20viruses.%20Microbiome9,%2037%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Kieft, K., Zhou, Z. C. &amp; Anantharaman, K. VIBRANT: automated recovery, annotation and curation of microbial viruses, and evaluation of viral community function from genomic sequences. <em>Microbiome</em><strong>8</strong>, 90 (2020).
</cite> [<a href="https://doi.org/10.1186/s40168-020-00867-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7288430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32522236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kieft,%20K.,%20Zhou,%20Z.%20C.%20&amp;%20Anantharaman,%20K.%20VIBRANT:%20automated%20recovery,%20annotation%20and%20curation%20of%20microbial%20viruses,%20and%20evaluation%20of%20viral%20community%20function%20from%20genomic%20sequences.%20Microbiome8,%2090%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Nayfach, S. et al. CheckV assesses the quality and completeness of metagenome-assembled viral genomes. <em>Nat. Biotechnol.</em><strong>39</strong>, 578–585 (2021).
</cite> [<a href="https://doi.org/10.1038/s41587-020-00774-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8116208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33349699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nayfach,%20S.%20et%20al.%20CheckV%20assesses%20the%20quality%20and%20completeness%20of%20metagenome-assembled%20viral%20genomes.%20Nat.%20Biotechnol.39,%20578%E2%80%93585%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Fu, L. M., Niu, B. F., Zhu, Z. W., Wu, S. T. &amp; Li, W. Z. CD-HIT: accelerated for clustering the next-generation sequencing data. <em>Bioinformatics</em><strong>28</strong>, 3150–3152 (2012).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3516142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23060610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fu,%20L.%20M.,%20Niu,%20B.%20F.,%20Zhu,%20Z.%20W.,%20Wu,%20S.%20T.%20&amp;%20Li,%20W.%20Z.%20CD-HIT:%20accelerated%20for%20clustering%20the%20next-generation%20sequencing%20data.%20Bioinformatics28,%203150%E2%80%933152%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Camargo, A. P. et al. Identification of mobile genetic elements with geNomad. <em>Nat. Biotechnol.</em><strong>42</strong>, 1303–1312 (2023).
</cite> [<a href="https://doi.org/10.1038/s41587-023-01953-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11324519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37735266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camargo,%20A.%20P.%20et%20al.%20Identification%20of%20mobile%20genetic%20elements%20with%20geNomad.%20Nat.%20Biotechnol.42,%201303%E2%80%931312%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Shang, J. Y., Tang, X. B. &amp; Sun, Y. N. PhaTYP: predicting the lifestyle for bacteriophages using BERT. <em>Brief. Bioinform.</em><strong>24</strong>, bbac487 (2023).
</cite> [<a href="https://doi.org/10.1093/bib/bbac487" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9851330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36659812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shang,%20J.%20Y.,%20Tang,%20X.%20B.%20&amp;%20Sun,%20Y.%20N.%20PhaTYP:%20predicting%20the%20lifestyle%20for%20bacteriophages%20using%20BERT.%20Brief.%20Bioinform.24,%20bbac487%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Li, Z. X. et al. Deep sea sediments associated with cold seeps are a subsurface reservoir of viral diversity. <em> ISME J.</em><strong>15</strong>, 2366–2378 (2021).
</cite> [<a href="https://doi.org/10.1038/s41396-021-00932-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8319345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33649554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Z.%20X.%20et%20al.%20Deep%20sea%20sediments%20associated%20with%20cold%20seeps%20are%20a%20subsurface%20reservoir%20of%20viral%20diversity.%20ISME%20J.15,%202366%E2%80%932378%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Aroney, S. T. et al. CoverM: Read alignment statistics for metagenomics. <em>Bioinformatics</em><strong>41</strong>, btaf147 (2025).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaf147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11993303/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40193404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aroney,%20S.%20T.%20et%20al.%20CoverM:%20Read%20alignment%20statistics%20for%20metagenomics.%20Bioinformatics41,%20btaf147%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Shi, L. D. et al. A mixed blessing of viruses in wastewater treatment plants. <em>Water Res.</em><strong>215</strong>, 118237 (2022).
</cite> [<a href="https://doi.org/10.1016/j.watres.2022.118237" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35245718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi,%20L.%20D.%20et%20al.%20A%20mixed%20blessing%20of%20viruses%20in%20wastewater%20treatment%20plants.%20Water%20Res.215,%20118237%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Liu, Y. Q. et al. Diversity and function of mountain and polar supraglacial DNA viruses. <em>Sci. Bull.</em><strong>68</strong>, 2418–2433 (2023).</cite> [<a href="https://doi.org/10.1016/j.scib.2023.09.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37739838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20Q.%20et%20al.%20Diversity%20and%20function%20of%20mountain%20and%20polar%20supraglacial%20DNA%20viruses.%20Sci.%20Bull.68,%202418%E2%80%932433%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Gios, E., Mosley, O. E., Hoggard, M. &amp; Handley, K. M. High niche specificity and host genetic diversity of groundwater viruses. <em> ISME J.</em><strong>18</strong>, wrae035 (2024).
</cite> [<a href="https://doi.org/10.1093/ismejo/wrae035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10980836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38452204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gios,%20E.,%20Mosley,%20O.%20E.,%20Hoggard,%20M.%20&amp;%20Handley,%20K.%20M.%20High%20niche%20specificity%20and%20host%20genetic%20diversity%20of%20groundwater%20viruses.%20ISME%20J.18,%20wrae035%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Bland, C. et al. CRISPR Recognition Tool (CRT): a tool for automatic detection of clustered regularly interspaced palindromic repeats. <em>BMC Bioinform.</em><strong>8</strong>, 209 (2007).</cite> [<a href="https://doi.org/10.1186/1471-2105-8-209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1924867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17577412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bland,%20C.%20et%20al.%20CRISPR%20Recognition%20Tool%20(CRT):%20a%20tool%20for%20automatic%20detection%20of%20clustered%20regularly%20interspaced%20palindromic%20repeats.%20BMC%20Bioinform.8,%20209%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Zhang, R. S. et al. SpacePHARER: sensitive identification of phages from CRISPR spacers in prokaryotic hosts. <em>Bioinformatics</em><strong>37</strong>, 3364–3366 (2021).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8504623/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33792634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20R.%20S.%20et%20al.%20SpacePHARER:%20sensitive%20identification%20of%20phages%20from%20CRISPR%20spacers%20in%20prokaryotic%20hosts.%20Bioinformatics37,%203364%E2%80%933366%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Chan, P. P., Lin, B. Y., Mak, A. J. &amp; Lowe, T. M. tRNAscan-SE 2.0: improved detection and functional classification of transfer RNA genes. <em>Nucleic Acids Res.</em><strong>49</strong>, 9077–9096 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab688" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8450103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34417604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chan,%20P.%20P.,%20Lin,%20B.%20Y.,%20Mak,%20A.%20J.%20&amp;%20Lowe,%20T.%20M.%20tRNAscan-SE%202.0:%20improved%20detection%20and%20functional%20classification%20of%20transfer%20RNA%20genes.%20Nucleic%20Acids%20Res.49,%209077%E2%80%939096%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Ghaly, T. M. et al. Exploring virus-host-environment interactions in a chemotrophic-based underground estuary. <em>Environ. Microbiome</em><strong>19</strong>, 9 (2024).
</cite> [<a href="https://doi.org/10.1186/s40793-024-00549-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10829341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38291480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ghaly,%20T.%20M.%20et%20al.%20Exploring%20virus-host-environment%20interactions%20in%20a%20chemotrophic-based%20underground%20estuary.%20Environ.%20Microbiome19,%209%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Martins, P. D. et al. Viral and metabolic controls on high rates of microbial sulfur and carbon cycling in wetland ecosystems. <em>Microbiome</em><strong>6</strong>, 138 (2018).
</cite> [<a href="https://doi.org/10.1186/s40168-018-0522-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6081815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30086797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martins,%20P.%20D.%20et%20al.%20Viral%20and%20metabolic%20controls%20on%20high%20rates%20of%20microbial%20sulfur%20and%20carbon%20cycling%20in%20wetland%20ecosystems.%20Microbiome6,%20138%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Ahlgren, N. A., Ren, J., Lu, Y. Y., Fuhrman, J. A. &amp; Sun, F. Z. Alignment-free d<sub>2</sub>* oligonucleotide frequency dissimilarity measure improves prediction of hosts from metagenomically-derived viral sequences. <em>Nucleic Acids Res.</em><strong>45</strong>, 39–53 (2017).
</cite> [<a href="https://doi.org/10.1093/nar/gkw1002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5224470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27899557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahlgren,%20N.%20A.,%20Ren,%20J.,%20Lu,%20Y.%20Y.,%20Fuhrman,%20J.%20A.%20&amp;%20Sun,%20F.%20Z.%20Alignment-free%20d2*%20oligonucleotide%20frequency%20dissimilarity%20measure%20improves%20prediction%20of%20hosts%20from%20metagenomically-derived%20viral%20sequences.%20Nucleic%20Acids%20Res.45,%2039%E2%80%9353%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Shaffer, M. et al. DRAM for distilling microbial metabolism to automate the curation of microbiome function. <em>Nucleic Acids Res.</em><strong>48</strong>, 8883–8900 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa621" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7498326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32766782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shaffer,%20M.%20et%20al.%20DRAM%20for%20distilling%20microbial%20metabolism%20to%20automate%20the%20curation%20of%20microbiome%20function.%20Nucleic%20Acids%20Res.48,%208883%E2%80%938900%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Pratama, A. A. et al. Expanding standards in viromics: in silico evaluation of dsDNA viral genome identification, classification, and auxiliary metabolic gene curation. <em>PeerJ</em><strong>9</strong>, e11447 (2021).
</cite> [<a href="https://doi.org/10.7717/peerj.11447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8210812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34178438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pratama,%20A.%20A.%20et%20al.%20Expanding%20standards%20in%20viromics:%20in%20silico%20evaluation%20of%20dsDNA%20viral%20genome%20identification,%20classification,%20and%20auxiliary%20metabolic%20gene%20curation.%20PeerJ9,%20e11447%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>ter Horst, A. M. et al. Minnesota peat viromes reveal terrestrial and aquatic niche partitioning for local and global viral populations. <em>Microbiome</em><strong>9</strong>, 233 (2021).
</cite> [<a href="https://doi.org/10.1186/s40168-021-01156-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8626947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34836550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?ter%20Horst,%20A.%20M.%20et%20al.%20Minnesota%20peat%20viromes%20reveal%20terrestrial%20and%20aquatic%20niche%20partitioning%20for%20local%20and%20global%20viral%20populations.%20Microbiome9,%20233%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Yu, M. S. et al. Diversity and potential host-interactions of viruses inhabiting deep-sea seamount sediments. <em>Nat. Commun.</em><strong>15</strong>, 3228 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-47600-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11018836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38622147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20M.%20S.%20et%20al.%20Diversity%20and%20potential%20host-interactions%20of%20viruses%20inhabiting%20deep-sea%20seamount%20sediments.%20Nat.%20Commun.15,%203228%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Yi, Y. et al. A systematic analysis of marine lysogens and proviruses. <em>Nat. Commun.</em><strong>14</strong>, 6013 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-41699-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10533544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37758717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yi,%20Y.%20et%20al.%20A%20systematic%20analysis%20of%20marine%20lysogens%20and%20proviruses.%20Nat.%20Commun.14,%206013%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Kanehisa, M. &amp; Goto, S. KEGG: Kyoto Encyclopedia of Genes and Genomes. <em>Nucleic Acids Res.</em><strong>28</strong>, 27–30 (2000).
</cite> [<a href="https://doi.org/10.1093/nar/28.1.27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.%20&amp;%20Goto,%20S.%20KEGG:%20Kyoto%20Encyclopedia%20of%20Genes%20and%20Genomes.%20Nucleic%20Acids%20Res.28,%2027%E2%80%9330%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Jian, H. H. et al. Diversity and distribution of viruses inhabiting the deepest ocean on Earth. <em> ISME J.</em><strong>15</strong>, 3094–3110 (2021).
</cite> [<a href="https://doi.org/10.1038/s41396-021-00994-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8443753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33972725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jian,%20H.%20H.%20et%20al.%20Diversity%20and%20distribution%20of%20viruses%20inhabiting%20the%20deepest%20ocean%20on%20Earth.%20ISME%20J.15,%203094%E2%80%933110%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Oksanen, J. et al. vegan: Community Ecology Package. Version 2.6-4, <a href="http://CRAN.R-project.org/package=vegan" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R-project.org/package=vegan</a> (2022).</cite>
</li>
<li id="CR113">
<span class="label">113.</span><cite>Revelle, W. psych: Procedures for personality and psychological research. Version 2.3.3. <a href="http://cran.r-project.org/package=psych" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r-project.org/package=psych</a> (2023).</cite>
</li>
<li id="CR114">
<span class="label">114.</span><cite>Ahlmann-Eltze, C. ggupset: Combination Matrix Axis for ‘ggplot2’ to Create ‘UpSet’ Plots. R package version 0.4.0.9000, <a href="http://github.com/const-ae/ggupset" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/const-ae/ggupset</a> (2025).</cite>
</li>
<li id="CR115">
<span class="label">115.</span><cite>Wickham, H. <em>Ggplot2: Elegant Graphics for Data Analysis (2nd ed.)</em> (Springer International Publishing, 2016).</cite>
</li>
<li id="CR116">
<span class="label">116.</span><cite>Langfelder, P. &amp; Horvath, S. WGCNA: an R package for weighted correlation network analysis. <em>BMC Bioinform.</em><strong>9</strong>, 559 (2008).</cite> [<a href="https://doi.org/10.1186/1471-2105-9-559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2631488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19114008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langfelder,%20P.%20&amp;%20Horvath,%20S.%20WGCNA:%20an%20R%20package%20for%20weighted%20correlation%20network%20analysis.%20BMC%20Bioinform.9,%20559%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Barberán, A., Bates, S. T., Casamayor, E. O. &amp; Fierer, N. Using network analysis to explore co-occurrence patterns in soil microbial communities. <em> ISME J.</em><strong>6</strong>, 343–351 (2012).
</cite> [<a href="https://doi.org/10.1038/ismej.2011.119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3260507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21900968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barber%C3%A1n,%20A.,%20Bates,%20S.%20T.,%20Casamayor,%20E.%20O.%20&amp;%20Fierer,%20N.%20Using%20network%20analysis%20to%20explore%20co-occurrence%20patterns%20in%20soil%20microbial%20communities.%20ISME%20J.6,%20343%E2%80%93351%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>Csárdi, G. &amp; Nepusz, T. The igraph software package for complex network research. <em>InterJournal, Complex Syst.</em><strong>1695</strong>, 1–9 (2006).</cite> [<a href="https://scholar.google.com/scholar_lookup?Cs%C3%A1rdi,%20G.%20&amp;%20Nepusz,%20T.%20The%20igraph%20software%20package%20for%20complex%20network%20research.%20InterJournal,%20Complex%20Syst.1695,%201%E2%80%939%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Jiao, S., Lu, Y. H. &amp; Wei, G. H. Soil multitrophic network complexity enhances the link between biodiversity and multifunctionality in agricultural systems. <em>Glob. Change Biol.</em><strong>28</strong>, 140–153 (2022).</cite> [<a href="https://doi.org/10.1111/gcb.15917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34610173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiao,%20S.,%20Lu,%20Y.%20H.%20&amp;%20Wei,%20G.%20H.%20Soil%20multitrophic%20network%20complexity%20enhances%20the%20link%20between%20biodiversity%20and%20multifunctionality%20in%20agricultural%20systems.%20Glob.%20Change%20Biol.28,%20140%E2%80%93153%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Wen, T. et al. ggClusterNet: an R package for microbiome network analysis and modularity-based multiple network layouts. <em>iMeta</em>. <strong>1</strong>, e32 (2022).
</cite> [<a href="https://doi.org/10.1002/imt2.32" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10989811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38868720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wen,%20T.%20et%20al.%20ggClusterNet:%20an%20R%20package%20for%20microbiome%20network%20analysis%20and%20modularity-based%20multiple%20network%20layouts.%20iMeta.%201,%20e32%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Wang, X. et al. Decreased soil multifunctionality is associated with altered microbial network properties under precipitation reduction in a semiarid grassland. <em>iMeta</em><strong>2</strong>, e106 (2023).
</cite> [<a href="https://doi.org/10.1002/imt2.106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10989785/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38868425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.%20et%20al.%20Decreased%20soil%20multifunctionality%20is%20associated%20with%20altered%20microbial%20network%20properties%20under%20precipitation%20reduction%20in%20a%20semiarid%20grassland.%20iMeta2,%20e106%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR122">
<span class="label">122.</span><cite>Xun, W. et al. Alteration of soil bacterial interaction networks driven by different longterm fertilization management practices in the red soil of South China. <em>Appl. Soil Ecol.</em><strong>120</strong>, 128–134 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Xun,%20W.%20et%20al.%20Alteration%20of%20soil%20bacterial%20interaction%20networks%20driven%20by%20different%20longterm%20fertilization%20management%20practices%20in%20the%20red%20soil%20of%20South%20China.%20Appl.%20Soil%20Ecol.120,%20128%E2%80%93134%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR123">
<span class="label">123.</span><cite>Stouffer, D. B. &amp; Bascompte, J. Compartmentalization increases food-web persistence. <em>Proc. Natl. Acad. Sci. USA</em><strong>108</strong>, 3648–3652 (2011).
</cite> [<a href="https://doi.org/10.1073/pnas.1014353108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3048152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21307311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stouffer,%20D.%20B.%20&amp;%20Bascompte,%20J.%20Compartmentalization%20increases%20food-web%20persistence.%20Proc.%20Natl.%20Acad.%20Sci.%20USA108,%203648%E2%80%933652%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR124">
<span class="label">124.</span><cite>Wang, M. et al. Strong succession in prokaryotic association networks and community assembly mechanisms in an acid mine drainage-impacted riverine ecosystem. <em>Water Res.</em><strong>243</strong>, 120343 (2023).
</cite> [<a href="https://doi.org/10.1016/j.watres.2023.120343" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37482007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20M.%20et%20al.%20Strong%20succession%20in%20prokaryotic%20association%20networks%20and%20community%20assembly%20mechanisms%20in%20an%20acid%20mine%20drainage-impacted%20riverine%20ecosystem.%20Water%20Res.243,%20120343%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR125">
<span class="label">125.</span><cite>Basu, M., Bhattacharyya, N. P. &amp; Mohanty, P. K. Modules of human micro-RNA co-target network. <em>J. Phys. Confer Ser.</em><strong>297</strong>, 012002 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Basu,%20M.,%20Bhattacharyya,%20N.%20P.%20&amp;%20Mohanty,%20P.%20K.%20Modules%20of%20human%20micro-RNA%20co-target%20network.%20J.%20Phys.%20Confer%20Ser.297,%20012002%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR126">
<span class="label">126.</span><cite>Deschildre, J., Vandemoortele, B., Loers, J. U., Preter, K. D. &amp; Vermeirssen, V. Evaluation of single-sample network inference methods for precision oncology. <em>npj Syst. Biol. Appl.</em><strong>10</strong>, 18 (2024).
</cite> [<a href="https://doi.org/10.1038/s41540-024-00340-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10869342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38360881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deschildre,%20J.,%20Vandemoortele,%20B.,%20Loers,%20J.%20U.,%20Preter,%20K.%20D.%20&amp;%20Vermeirssen,%20V.%20Evaluation%20of%20single-sample%20network%20inference%20methods%20for%20precision%20oncology.%20npj%20Syst.%20Biol.%20Appl.10,%2018%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR127">
<span class="label">127.</span><cite>Feld, C. K., Segurado, P. &amp; Gutierrez-Canovas, C. Analysing the impact of multiple stressors in aquatic biomonitoring data: a ‘cookbook’ with applications in R. <em>Sci. Total Environ.</em><strong>573</strong>, 1320–1339 (2016).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2016.06.243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27499499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feld,%20C.%20K.,%20Segurado,%20P.%20&amp;%20Gutierrez-Canovas,%20C.%20Analysing%20the%20impact%20of%20multiple%20stressors%20in%20aquatic%20biomonitoring%20data:%20a%20%E2%80%98cookbook%E2%80%99%20with%20applications%20in%20R.%20Sci.%20Total%20Environ.573,%201320%E2%80%931339%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR128">
<span class="label">128.</span><cite>Lenth, R. et al. emmeans: estimated marginal means, aka least-squares mean. Version 1.8.6. <a href="http://cran.r-project.org/web/packages/emmeans/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cran.r-project.org/web/packages/emmeans/index.html</a> (2023).</cite>
</li>
<li id="CR129">
<span class="label">129.</span><cite>Wobbrock J. O., Findlater L., Gergle D. &amp; Higgins J. J. The Aligned Rank Transform for nonparametric factorial analyses using only ANOVA procedures. In <em>Proceedings of the SIGCHI Conference on Human Factors in Computing Systems.</em>Association for Computing Machinery, New York, NY, USA, 143–146 (2011).</cite>
</li>
<li id="CR130">
<span class="label">130.</span><cite>Kassambara, A. rstatix: pipe-friendly framework for basic statistical tests. Version 0.7.2. <a href="http://CRAN.R-project.org/package=rstatix" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R-project.org/package=rstatix</a> (2023).</cite>
</li>
<li id="CR131">
<span class="label">131.</span><cite>Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate - a practical and powerful approach to multiple testing. <em>J. R. Stat. Soc. Ser. B-Stat. Methodol.</em><strong>57</strong>, 289–300 (1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?Benjamini,%20Y.%20&amp;%20Hochberg,%20Y.%20Controlling%20the%20false%20discovery%20rate%20-%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing.%20J.%20R.%20Stat.%20Soc.%20Ser.%20B-Stat.%20Methodol.57,%20289%E2%80%93300%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR132">
<span class="label">132.</span><cite>Segata, N. et al. Metagenomic biomarker discovery and explanation. <em>Genome Biol.</em><strong>12</strong>, R60 (2011).
</cite> [<a href="https://doi.org/10.1186/gb-2011-12-6-r60" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3218848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21702898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Segata,%20N.%20et%20al.%20Metagenomic%20biomarker%20discovery%20and%20explanation.%20Genome%20Biol.12,%20R60%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR133">
<span class="label">133.</span><cite>Wang, T. et al. Metagenomic analysis reveals how multiple stressors disrupt virus–host interactions in multi-trophic freshwater mesocosms. <em>Figshare</em>. 10.6084/m9.figshare.29040674 (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-63162-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12370912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40841555/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63162_MOESM2_ESM.pdf</a><sup> (7.6KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63162_MOESM3_ESM.xlsx</a><sup> (346.3KB, xlsx) </sup><p>Supplementary Data 1 and Supplementary Data 2</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (110.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370912/bin/41467_2025_63162_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (4.2MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All raw sequencing data generated in this study have been deposited in the NCBI Sequence Read Archive (SRA) database under project ID <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1189868" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA1189868</a>. The primary data supporting the results of this study are publicly available on Figshare<sup><a href="#CR133" class="usa-link" aria-describedby="CR133">133</a></sup> [10.6084/m9.figshare.29040674] and KNB [10.5063/F1SJ1J3F]. <a href="#Sec21" class="usa-link">Source data</a> are provided with this paper.</p>
<p>The code and programs used for analyses are described in the Methods, and are also available at 10.6084/m9.figshare.29040674.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63162-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63162.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370912/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370912/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370912%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370912/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370912/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370912/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841555/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370912/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841555/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370912/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370912/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="brdkjNWwSqkibVSvi4kAHkdbbd9uoyH8E1Pzkg1CR8pUJaScPSej0GfEdC7RQA9H">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
