
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Optimal dosing of anti-cancer treatment under drug-induced plasticity - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5311028AEF0FB305110200235ACD27.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="npjsba">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375710/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="NPJ Systems Biology and Applications">
<meta name="citation_title" content="Optimal dosing of anti-cancer treatment under drug-induced plasticity">
<meta name="citation_author" content="Einar Bjarki Gunnarsson">
<meta name="citation_author_institution" content="Division of Applied Mathematics, Science Institute, University of Iceland, Reykjavik, Iceland">
<meta name="citation_author_institution" content="School of Mathematics, University of Minnesota, Minneapolis, MN USA">
<meta name="citation_author" content="Benedikt Vilji Magnússon">
<meta name="citation_author_institution" content="Division of Applied Mathematics, Science Institute, University of Iceland, Reykjavik, Iceland">
<meta name="citation_author" content="Jasmine Foo">
<meta name="citation_author_institution" content="School of Mathematics, University of Minnesota, Minneapolis, MN USA">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="11">
<meta name="citation_firstpage" content="98">
<meta name="citation_doi" content="10.1038/s41540-025-00571-5">
<meta name="citation_pmid" content="40850942">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375710/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375710/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375710/pdf/41540_2025_Article_571.pdf">
<meta name="description" content="While cancer has traditionally been considered a genetic disease, mounting evidence indicates an important role for non-genetic (epigenetic) mechanisms. Common anti-cancer drugs have recently been observed to induce the adoption of non-genetic ...">
<meta name="og:title" content="Optimal dosing of anti-cancer treatment under drug-induced plasticity">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="While cancer has traditionally been considered a genetic disease, mounting evidence indicates an important role for non-genetic (epigenetic) mechanisms. Common anti-cancer drugs have recently been observed to induce the adoption of non-genetic ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375710/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375710">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41540-025-00571-5"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41540_2025_Article_571.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375710%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375710/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375710/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375710/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npjsba.jpg" alt="NPJ Systems Biology and Applications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NPJ Systems Biology and Applications" title="Link to NPJ Systems Biology and Applications" shape="default" href="http://www.nature.com/npjsba/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">NPJ Syst Biol Appl</button></div>. 2025 Aug 25;11:98. doi: <a href="https://doi.org/10.1038/s41540-025-00571-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41540-025-00571-5</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22NPJ%20Syst%20Biol%20Appl%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22NPJ%20Syst%20Biol%20Appl%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22NPJ%20Syst%20Biol%20Appl%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22NPJ%20Syst%20Biol%20Appl%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Optimal dosing of anti-cancer treatment under drug-induced plasticity</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gunnarsson%20EB%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Einar Bjarki Gunnarsson</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Einar Bjarki Gunnarsson</span></h3>
<div class="p">
<sup>1</sup>Division of Applied Mathematics, Science Institute, University of Iceland, Reykjavik, Iceland </div>
<div class="p">
<sup>2</sup>School of Mathematics, University of Minnesota, Minneapolis, MN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gunnarsson%20EB%22%5BAuthor%5D" class="usa-link"><span class="name western">Einar Bjarki Gunnarsson</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Magn%C3%BAsson%20BV%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Benedikt Vilji Magnússon</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Benedikt Vilji Magnússon</span></h3>
<div class="p">
<sup>1</sup>Division of Applied Mathematics, Science Institute, University of Iceland, Reykjavik, Iceland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Magn%C3%BAsson%20BV%22%5BAuthor%5D" class="usa-link"><span class="name western">Benedikt Vilji Magnússon</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foo%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jasmine Foo</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jasmine Foo</span></h3>
<div class="p">
<sup>2</sup>School of Mathematics, University of Minnesota, Minneapolis, MN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foo%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jasmine Foo</span></a>
</div>
</div>
<sup>2</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Division of Applied Mathematics, Science Institute, University of Iceland, Reykjavik, Iceland </div>
<div id="Aff2">
<sup>2</sup>School of Mathematics, University of Minnesota, Minneapolis, MN USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 6; Accepted 2025 Jul 21; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375710  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40850942/" class="usa-link">40850942</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">While cancer has traditionally been considered a genetic disease, mounting evidence indicates an important role for non-genetic (epigenetic) mechanisms. Common anti-cancer drugs have recently been observed to induce the adoption of non-genetic drug-tolerant cell states, thereby accelerating the evolution of drug resistance. This confounds conventional high-dose treatment strategies aimed at maximal tumor reduction, since high doses can simultaneously promote non-genetic resistance. In this work, we study optimal dosing of anti-cancer treatment under drug-induced cell plasticity. We show that the optimal dosing strategy steers the tumor to a fixed equilibrium composition between sensitive and tolerant cells, while precisely balancing the trade-off between cell kill and tolerance induction. The optimal equilibrium strategy ranges from applying a low dose continuously to applying the maximum dose intermittently, depending on the dynamics of tolerance induction. We finally discuss how our approach can be integrated with in vitro data to derive patient-specific treatment insights.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Cancer, Computational biology and bioinformatics, Systems biology, Applied mathematics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Cancer is the result of a complex evolutionary process which results in cells gaining the ability to divide uncontrollably<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. While genetic mutations have long been understood to be important drivers of cancer evolution, it has more recently become clear that epigenetic mechanisms such as methylation and acetylation are sufficient to fulfill the traditional hallmarks of cancer<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Epigenetic mechanisms are both heritable and reversible, and they can enable cells to switch dynamically between two or more phenotypic states, which commonly show differential responses to drug treatment. In fact, tumor cells in many cancer types have been observed to adopt reversible drug-tolerant or stem-like states in vitro, which enables the tumor to temporarily evade drug treatment and sets the stage for the evolution of more permanent resistance mechanisms<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. While these drug-tolerant states can exist independently of treatment<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, recent evidence indicates that common anti-cancer drugs can also induce or accelerate their adoption, a phenomenon we refer to as <em>drug-induced plasticity</em><sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Drug induction is already well known in the context of genetic mutations, where both anti-cancer and anti-bacterial drugs have been observed to cause genomic instability or <em>mutagenesis</em>, referring to an elevated mutation rate occurring as a result of the stress imposed by the treatment<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>.</p>
<p id="Par3">Anti-cancer drug treatment has traditionally been administered under the “maximum tolerated dose” (MTD) paradigm, where the goal is to eradicate the tumor as quickly as possible and minimize the probability of spontaneous resistance-conferring mutations<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Non-genetic cell plasticity fundamentally confounds the reasoning behind this strategy, especially if it is induced by the drug. First, if drug-tolerant states are adopted independently of the drug, it is likely that drug-tolerant cells will preexist treatment in large numbers, even if they form only a small fraction of the tumor bulk<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. In addition, since non-genetic mechanisms usually operate much faster than genetic mutations<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, a non-negligible fraction of drug-sensitive cells will adopt drug tolerance under treatment. This means that it can be impossible to kill the tumor, no matter how large a dose is applied<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Second, if the drug induces tolerance adoption in a dose-dependent manner, applying larger doses with the aim of maximizing cell kill has the downside of promoting resistance evolution. This trade-off has been explored in several recent mathematical modeling works, which generally find that continuous low-dose or intermittent strategies outperform MTD treatment<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. However, these works usually focus on one potential model of drug-induced tolerance and some only consider the case of irreversible genetic resistance. In addition, many of these works only consider constant or intermittent strategies. Overall, there is a lack of generalized insights into how different potential forms of drug-induced tolerance influence optimal treatment decisions.</p>
<p id="Par4">In this work, we seek a more systematic understanding of optimal dosing strategies under drug-induced cell plasticity. We seek both a qualitative understanding of the characteristics of optimal strategies and simple quantitative methods for computing them. We consider many different possible forms of drug-induced tolerance and explore how and why different forms lead to different optimal strategies. Our work differs from previous work in several important ways. First, whereas prior work generally assumes that tolerance is induced through elevated transitions from drug-sensitivity to drug-tolerance, it is also biologically plausible that the drug inhibits transitions from tolerance back to sensitivity, or that it affects both transitions simultaneously. For example, hypermethylation in the promoter regions of tumor suppressor genes has been associated with gene silencing and drug resistance in many cancers<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. When a cell divides, existing methylation patterns are preserved by the maintenance methyltransferase DNMT1, while de novo methylation is carried out by the methyltransferases DNMT3A/3B<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. If a drug-tolerant state is conferred via methylation, drug-induced upregulation of DNMT3A/3B will induce the adoption of the tolerant state, while drug-induced upregulation of DNMT1 will inhibit transitions out of it. The notion that the drug may not only elevate transitions from sensitivity to tolerance is further supported by experimental evidence indicating that the drug can influence all possible phenotypic transitions simultaneously<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Second, whereas many prior works assume that the level of tolerance induction is linear as a function of the drug dose, we also consider the case where tolerance is induced uniformly in the presence of drug. These two forms of drug-induced tolerance were observed in a recent experimental work by Russo et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, and they emerge as limiting cases of a more general Michaelis-Menten form of tolerance induction (Methods). Third, while most previous work is computational in nature, we establish mathematical results which provide generalized insights that apply across all biologically relevant parameter regimes. Finally, whereas some previous works focus on irreversible transitions from drug-sensitivity to drug-resistance, we study the case of reversible phenotypic switching between sensitivity and tolerance. Our results are compared with previous work in more detail in Supplementary Note <a href="#MOESM1" class="usa-link">1</a>.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Mathematical model of drug-induced tolerance enables derivation of optimal dosing strategies</h3>
<p id="Par5">We propose a mathematical model of a tumor undergoing anti-cancer drug treatment, in which cells are able to transition between two cell states, a drug-sensitive (type-0) and a drug-tolerant (type-1) state. Cells of type-<em>i</em> divide at rate <em>b</em><sub><em>i</em></sub> and die at rate <em>d</em><sub><em>i</em></sub>, with net division rate <em>λ</em><sub><em>i</em></sub> = <em>b</em><sub><em>i</em></sub> − <em>d</em><sub><em>i</em></sub>. Each drug-sensitive cell is assumed to transition to the drug-tolerant state at rate <em>μ</em>, and each tolerant cell transitions back to the sensitive state at rate <em>ν</em> (Fig. <a href="#Fig1" class="usa-link">1</a>a). To model the effect of the drug on cell proliferation, we assume that the sensitive cell death rate <em>d</em><sub>0</sub>(<em>c</em>) increases with the drug dose <em>c</em> (Fig. <a href="#Fig1" class="usa-link">1</a>b). Equivalently, it can be assumed that the drug decreases the division rate of sensitive cells (Methods). To model drug-induced tolerance, the transition rates <em>μ</em>(<em>c</em>) and <em>ν</em>(<em>c</em>) between sensitivity and tolerance are assumed to depend on the drug dose <em>c</em>, either in a linear or uniform fashion (Fig. <a href="#Fig1" class="usa-link">1</a>c, d). A detailed description of the mathematical model is provided in the Methods section.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Mathematical model of drug-induced tolerance.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375710_41540_2025_571_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/491e/12375710/46a0d06d7cdd/41540_2025_571_Fig1_HTML.jpg" loading="lazy" id="d33e568" height="801" width="725" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> In the mathematical model, cells transition between two states, a drug-sensitive (type-0) state and a drug-tolerant (type-1) state. Sensitive cells divide at rate <em>b</em><sub>0</sub>, die at rate <em>d</em><sub>0</sub> and transition to tolerance at rate <em>μ</em>. Tolerant cells divide at rate <em>b</em><sub>1</sub>, die at rate <em>d</em><sub>1</sub> and transition to sensitivity at rate <em>ν</em>. The sensitive cell death rate <em>d</em><sub>0</sub> and the transition rates <em>μ</em> and <em>ν</em> depend on the drug dose <em>c</em>. <strong>b</strong> The sensitive cell death rate follows a Michaelis-Menten equation of the form <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e462"><mrow><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>−</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mi>c</mi><mo>/</mo><mrow><mo>(</mo><mrow><mi>c</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></math></span>, where <em>d</em><sub>0</sub> is the death rate in the absence of drug and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e517"><msub><mrow><mi>d</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span> is the saturation death rate under an arbitrarily large drug dose. The dose <em>c</em> is normalized to the EC<sub>50</sub> dose for <em>d</em><sub>0</sub>, meaning that the drug has half the maximal effect at dose <em>c</em> = 1. <strong>c</strong> The transition rate <em>μ</em>(<em>c</em>) from drug-sensitivity to drug-tolerance is assumed either a linearly increasing function of <em>c</em> or to be uniformly elevated in the presence of drug. These two forms of drug-induced tolerance were observed in a recent study by Russo et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> and they also emerge as limiting cases of more general Michaelis-Menten dynamics. <strong>d</strong> The transition rate <em>ν</em>(<em>c</em>) from drug-tolerance to drug-sensitivity is similarly assumed either a linearly decreasing function of <em>c</em> or uniformly inhibited in the presence of drug.</p></figcaption></figure><p id="Par6">The time evolution of the number of drug-sensitive (type-0) and drug-tolerant (type-1) cells can be described by the differential equations</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><math id="d33e576" display="block"><mtable><mtr><mtd columnalign="right"><mfrac><mrow><mi>d</mi><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mtd><mtd columnalign="center"><mo>=</mo></mtd><mtd columnalign="left"><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo></mtd></mtr><mtr><mtd columnalign="right"><mfrac><mrow><mi>d</mi><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mtd><mtd columnalign="center"><mo>=</mo></mtd><mtd columnalign="left"><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>+</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo></mtd></mtr></mtable></math></td></tr></table>
<p>where the drug dose is a function of time, <em>c</em>(<em>t</em>) (Methods). The proportion of drug-sensitive cells over time, <em>f</em><sub>0</sub>(<em>t</em>) = <em>n</em><sub>0</sub>(<em>t</em>)/(<em>n</em><sub>0</sub>(<em>t</em>) + <em>n</em><sub>1</sub>(<em>t</em>)), then follows the differential equation</p>
<table class="disp-formula p" id="Equb"><tr><td class="formula"><math id="d33e726" display="block"><mrow><mfrac><mrow><mi>d</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfenced><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfenced><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>.</mo></mrow></math></td></tr></table>
<p>If a constant dose is applied, <em>c</em>(<em>t</em>) = <em>c</em> for all <em>t</em> ≥ 0, the tumor eventually grows exponentially at a fixed rate <em>σ</em>(<em>c</em>) given by</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e838" display="block"><mrow><mi>σ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><msqrt><mrow><msup><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mrow><mn>2</mn></mrow></msup><mo>+</mo><mn>4</mn><mi>μ</mi><mspace width="0.25em"></mspace><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></msqrt></mrow></mfenced><mo>.</mo></mrow></math></td>
<td class="label">1</td>
</tr></table>
<p>Moreover, the intratumor composition eventually reaches an equilibrium, where the proportion of sensitive cells in the population becomes fixed at</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e950" display="block"><mrow><msub><mrow><mover accent="true"><mrow><mi>f</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mrow><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>σ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfrac><mo>.</mo></mrow></math></td>
<td class="label">2</td>
</tr></table>
<p>Thus, under a constant dose <em>c</em>, it is possible to maintain the tumor at a stable intratumor composition, which leads to a fixed exponential growth rate <em>σ</em>(<em>c</em>) (Methods).</p>
<p id="Par7">We consider a dosing strategy (<em>c</em>(<em>t</em>))<sub><em>t</em>∈[0, <em>T</em>]</sub> optimal if it minimizes the total number of cells, <em>n</em><sub>0</sub>(<em>T</em>) + <em>n</em><sub>1</sub>(<em>T</em>), at the end of a finite time horizon [0, <em>T</em>]. When analyzing which dosing strategy is optimal in the long run, we also consider an infinite-horizon problem where the goal is to minimize the long-run average tumor growth rate (Methods). We can show that to determine the optimal strategy, it is sufficient to consider its effect on the proportion of sensitive cells <em>f</em><sub>0</sub>(<em>t</em>) over time, as opposed to its effect on the individual subpopulation counts <em>n</em><sub>0</sub>(<em>t</em>) and <em>n</em><sub>1</sub>(<em>t</em>) (Methods). This enables us to reduce an optimal control problem involving two state variables <em>n</em><sub>0</sub>(<em>t</em>) and <em>n</em><sub>1</sub>(<em>t</em>) to a single-state-variable problem, which simplifies our computational algorithms and mathematical analysis. In line with this insight, we use the proportion of sensitive cells over time to demonstrate the effects of optimal dosing strategies on tumor evolution in the following sections.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Constant low dosing is optimal in the long run under linear induction of tolerance</h3>
<p id="Par8">We now investigate optimal dosing strategies under linear induction of tolerance. We consider three cases, depicted in Fig. <a href="#Fig2" class="usa-link">2</a>a, depending on whether the drug only affects transitions from sensitivity to tolerance (Case I), it only affects transitions from tolerance to sensitivity (Case II), or it affects both transitions simultaneously (Case III). In Fig. <a href="#Fig2" class="usa-link">2</a>b, we show the optimal strategy for each case, computed using our own implementation of the forward–backward sweep method in MATLAB (Methods). For Case I, the optimal strategy applies a constant low dose <em>c</em>* from the beginning. For Case II, the optimal strategy starts with the maximum dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e1108"><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span>, and then gradually decreases it to a constant low dose <em>c</em>*. In this case, higher doses can be used at the onset of treatment to reduce the predominantly sensitive tumor more quickly, without the downside of promoting transitions from sensitivity to tolerance. For Case III, the optimal strategy has characteristics of both previous cases, as it does apply larger doses in the beginning, but it does not start with the maximum dose.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Optimal dosing under linear induction of tolerance.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375710_41540_2025_571_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/491e/12375710/dd33d0ce8483/41540_2025_571_Fig2_HTML.jpg" loading="lazy" id="d33e1274" height="1012" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> In our investigation of linear induction of tolerance, we consider three cases. In Case I, the drug only increases the transition rate from sensitivity to tolerance (<em>μ</em>(<em>c</em>) = <em>μ</em><sub>0; </sub>+ <em>k</em><em>c</em> and <em>ν</em>(<em>c</em>) = <em>ν</em><sub>0</sub>), in Case II, the drug only decreases the transition rate from tolerance to sensitivity (<em>μ</em>(<em>c</em>) = <em>μ</em><sub>0</sub> and <em>ν</em>(<em>c</em>) = <em>ν</em><sub>0</sub> − <em>m</em><em>c</em>), and in Case III, the drug does both (<em>μ</em>(<em>c</em>) = <em>μ</em><sub>0</sub> + <em>k</em><em>c</em> and <em>ν</em>(<em>c</em>) = <em>ν</em><sub>0</sub> − <em>m</em><em>c</em>). <strong>b</strong> Optimal dosing strategies under linear induction of tolerance computed using the forward-backward sweep method. In all cases, the optimal dosing strategy involves both a transient phase and an equilibrium phase. During the equilibrium phase, a constant dose <em>c</em>* is applied, which minimizes the long-run tumor growth rate <em>σ</em>(<em>c</em>) under a constant dose <em>c</em>, as indicated by the dotted lines. The grayed-out part of each dosing strategy is a boundary effect which arises due to the optimization goal of minimizing the tumor size at a specific time <em>T</em>, with no regard for what is best in the long run. If we were to extend the treatment horizon beyond time <em>T</em>, the period applying the constant dose <em>c</em>* would be extended (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>). We therefore ignore this boundary effect and focus on what is optimal over a longer time horizon. <strong>c</strong> Proportion of sensitive cells <em>f</em><sub>0</sub>(<em>t</em>) over time under the optimal dosing strategy. The constant equilibrium dose <em>c</em>* maintains the tumor at a fixed proportion between sensitive and tolerant cells. <strong>d</strong> Tumor size evolution under the optimal strategy compared with the tumor size evolution under the constant-dose strategy applying <em>c</em>* throughout. For Case I, the two strategies are the same. For Cases II/III, the optimal strategy results in 85.6%/55.4% greater tumor reduction during the initial stages of treatment, and 59.4%/24.2% greater reduction long-term. <strong>e</strong> If the proportion of sensitive cells <em>f</em><sub>0</sub>(0) at the start of treatment is changed, only the transient phase of the optimal treatment is affected, whereas the optimal equilibrium dose <em>c</em>* remains the same. For Case I, the transient phase is independent of the initial condition.</p></figcaption></figure><p id="Par9">By combining these results with mathematical analysis, we can derive several general treatment insights. First, the optimal dosing strategy always involves both a transient phase and an equilibrium phase. During the equilibrium phase, a constant dose <em>c</em>* is applied, which maintains the tumor at a fixed proportion between sensitive and tolerant cells (Fig. <a href="#Fig2" class="usa-link">2</a>(c)). This is true irrespective of the model parameter values (Supplementary Note <a href="#MOESM1" class="usa-link">1</a>). To demonstrate the importance of distinguishing these two treatment phases, we compare in Fig. <a href="#Fig2" class="usa-link">2</a>d the optimal strategy with the constant-dose strategy applying the equilibrium dose <em>c</em>* throughout. Eventually, the tumor reduces at the same exponential rate under both strategies. However, during the transient phase, the optimal strategy reduces the tumor much faster for Cases II and III, and this difference in tumor size is maintained throughout the course of treatment.</p>
<p id="Par10">Second, we can prove mathematically that the optimal equilibrium dose <em>c</em>* minimizes the long-run tumor growth rate <em>σ</em>(<em>c</em>) under a constant dose <em>c</em> (Supplementary Note <a href="#MOESM1" class="usa-link">1</a>). This means that the explicit expression (<a href="#Equ1" class="usa-link">1</a>) for <em>σ</em>(<em>c</em>) has clinical value on its own, as it can be used to address questions concerning ultimate treatment success. For example, if for a particular cancer type and a particular drug, we are interested in knowing whether there exists <em>any</em> dosing schedule capable of driving long-run tumor reduction (however complicated), we simply need to check whether there exists a constant dose <em>c</em> such that <em>σ</em>(<em>c</em>) &lt; 0. We also note that since the equilibrium dose <em>c</em>* is determined by long-run tumor growth dynamics, it is not affected by changes in the initial proportion between sensitive and tolerant cells (Fig. <a href="#Fig2" class="usa-link">2</a>e).</p>
<p id="Par11">Third, we can say that the goal of the transient phase is to steer the tumor to a desired composition, and the goal of the equilibrium phase is to maintain it at that composition. We can show that during the transient phase, once the proportion of sensitive cells reaches some value <em>f</em><sub>0</sub>, the optimal dose <em>c</em> to give at that moment must maximize</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e1353" display="block"><mrow><mfrac><mrow><mi>σ</mi><mrow><mo>(</mo><mrow><msup><mrow><mi>c</mi></mrow><mrow><mo>*</mo></mrow></msup></mrow><mo>)</mo></mrow><mo>−</mo><mi>u</mi><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow></mrow><mrow><mo>−</mo><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mo>′</mo></mrow></msubsup></mrow></mfrac><mo>,</mo></mrow></math></td>
<td class="label">3</td>
</tr></table>
<p>where <em>u</em>(<em>c</em>, <em>f</em><sub>0</sub>) = (<em>λ</em><sub>0</sub>(<em>c</em>) − <em>λ</em><sub>1</sub>)<em>f</em><sub>0</sub> + <em>λ</em><sub>1</sub> is the instantaneous tumor growth rate (Supplementary Note <a href="#MOESM1" class="usa-link">1</a>). Applying the maximum dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e1431"><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span> throughout the transient phase would kill the sensitive cells as quickly as possible, which would lead to the lowest possible growth rate <em>u</em>(<em>c</em>, <em>f</em><sub>0</sub>) and thus the largest possible numerator in (<a href="#Equ3" class="usa-link">3</a>). However, the maximum dose would simultaneously induce a large number of sensitive cells to adopt tolerance, which would lead to a large reduction <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e1458"><mrow><mo>−</mo><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mo>′</mo></mrow></msubsup></mrow></math></span> in the proportion of sensitive cells in the denominator of (<a href="#Equ3" class="usa-link">3</a>). Overall, expression (<a href="#Equ3" class="usa-link">3</a>) describes precisely how the optimal strategy must balance the trade-off between sensitive cell kill and tolerance induction as it steers the tumor to the optimal equilibrium composition.</p>
<p id="Par12">Finally, our mathematical analysis suggests a simple and efficient method to compute the optimal dosing strategy, which simultaneously provides an insight into how the doses are selected. First, the optimal equilibrium dose <em>c</em>* can be derived by minimizing expression (<a href="#Equ1" class="usa-link">1</a>), and the associated equilibrium proportion of sensitive cells <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e1490"><mrow><msub><mrow><mover accent="true"><mrow><mi>f</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><msup><mrow><mi>c</mi></mrow><mrow><mo>*</mo></mrow></msup></mrow><mo>)</mo></mrow></mrow></math></span> can be computed using expression (<a href="#Equ2" class="usa-link">2</a>). Then, the transient phase of the optimal strategy can be computed by iteratively determining the maximizing dose <em>c</em> in (<a href="#Equ3" class="usa-link">3</a>) and updating the value of <em>f</em><sub>0</sub>, as visualized in Supplementary Fig. <a href="#MOESM1" class="usa-link">2a</a>. In Supplementary Fig. <a href="#MOESM1" class="usa-link">2b</a>, we confirm that the transient phase computed using this approach agrees with the transient phase shown in Fig. <a href="#Fig2" class="usa-link">2</a> (obtained using the forward-backward sweep method).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Long treatment breaks significantly deteriorate treatment efficacy</h3>
<p id="Par13">Under linear induction of tolerance, it is optimal in the long run to expose the tumor continuously to the drug at a majority drug-tolerant composition. We are next interested in investigating how the optimal treatment compares with pulsed treatments, where the tumor is taken periodically off drug to allow tolerant cells to revert back to sensitivity. This comparison is made using the long-run average growth rate of the tumor (Methods).</p>
<p id="Par14">We consider a pulsed schedule as a triple (<em>c</em>, <em>t</em><sub>cycle</sub>, <em>φ</em>) encoding the drug dose <em>c</em>, the duration of each treatment cycle <em>t</em><sub>cycle</sub>, and the proportion of time <em>φ</em> the drug is applied during each cycle. A constant schedule is considered a pulsed schedule with <em>φ</em> = 1. It is relatively simple to show that in our model, for any pulsed schedule (<em>c</em>, <em>t</em><sub>cycle</sub>, <em>φ</em><sub>1</sub>), there exists a schedule (<em>c</em>, <em>t</em><sub>cycle</sub>/2, <em>φ</em><sub>2</sub>) with half the cycle length which yields the same or lower long-run growth rate (Supplementary Note <a href="#MOESM1" class="usa-link">1</a>). Therefore, when investigating pulsed schedules, it is sufficient to consider idealized schedules with an arbitrarily short cycle length. However, it is also simple to show that once the cycle length has become sufficiently small, the long-run average growth rate becomes effectively invariant to the cycle length (Supplementary Note <a href="#MOESM1" class="usa-link">1</a>). Thus, even if the best pulsed schedule involves switching arbitrarily fast between drug and no drug, a comparable outcome can be obtained using a much longer cycle time. This is an important insight, as it means that the model-recommended pulsed schedule is more clinically feasible.</p>
<p id="Par15">For a given pulsed schedule (<em>c</em>, <em>φ</em>) with an arbitrarily short cycle length, the long-run average growth rate of the tumor can be computed using the relatively simple formula (<a href="#Equ24" class="usa-link">24</a>). In Fig. <a href="#Fig3" class="usa-link">3</a>a, we show the long-run growth rate for a wide range of possible pulsed schedules (<em>c</em>, <em>φ</em>) under Cases I, II and III. The red lines show for each proportion <em>φ</em> of drug exposure the optimal dose <em>c</em>*(<em>φ</em>) to administer, and the red dots indicate the optimal schedules. As expected, a constant low dose is optimal in the long run. Figure <a href="#Fig3" class="usa-link">3</a>b, c show that for all three cases, the optimal schedule is quite sensitive to the duration of drug exposure. Instituting treatment breaks of 20–30% or longer results in a significantly larger tumor growth rate and a larger average dose being required to achieve that growth rate. From a clinical perspective, understanding to what extent the optimal strategy can be perturbed to insert treatment breaks can be useful for reducing potential side effects of continuous drug exposure. Plots like those in Fig. <a href="#Fig3" class="usa-link">3</a> could help a clinician understand the implications of instituting treatment breaks of a certain length, and to determine which dose is best for that length.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Continuous and pulsed schedules under linear induction of tolerance.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375710_41540_2025_571_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/491e/12375710/5cf3d568b20a/41540_2025_571_Fig3_HTML.jpg" loading="lazy" id="d33e1691" height="832" width="798" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Long-run average growth rate of the tumor under a range of possible pulsed schedules of the form (<em>c</em>, <em>φ</em>), where <em>c</em> is the drug dose applied and <em>φ</em> is the proportion of drug exposure. The red curves show for each exposure proportion <em>φ</em> the optimal dose <em>c</em>*(<em>φ</em>) to give. The red dots show the optimal pulsed schedules, which are continuous for Cases I, II and III, consistent with our results in Fig. <a href="#Fig2" class="usa-link">2</a>. <strong>b</strong> Long-run average growth rate under the optimal dose <em>c</em>*(<em>φ</em>), shown as a function of the proportion <em>φ</em> of drug exposure. <strong>c</strong> Average dose applied during each treatment cycle under the optimal dose <em>c</em>*(<em>φ</em>), shown as a function of the proportion <em>φ</em> of drug exposure, normalized to the average dose under the optimal schedule.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Pulsed maximum dosing is optimal in the long run under uniform induction of tolerance</h3>
<p id="Par16">We now consider the case where drug tolerance is induced in a uniform fashion, meaning that the drug effect is constant whenever the drug is present. In this case, the forward–backward sweep method does not converge to a unique solution, due to the discontinuity of the transition rate functions <em>μ</em>(<em>c</em>) and <em>ν</em>(<em>c</em>). We therefore rely on mathematical analysis.</p>
<p id="Par17">First, we can show that in the long run, it is optimal to apply a pulsed schedule alternating arbitrarily quickly between the maximum dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e1715"><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span> and no dose (Supplementary Note <a href="#MOESM1" class="usa-link">1</a>). To verify this result computationally, we compare in Fig. <a href="#Fig4" class="usa-link">4</a>b the long-run average tumor growth rate over a wide range of possible pulsed schedules (<em>c</em>, <em>φ</em>). For each given proportion of drug exposure <em>φ</em>, it is optimal to apply the maximum dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e1744"><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span> whenever the drug is present, as indicated by the red lines. For Case I, it is optimal to apply the maximum dose continuously, and for Cases II and III, it is optimal to alternate between the maximum dose and no dose, as indicated by the red dots. Thus, whereas low continuous dosing is optimal in the long run for linear induction of tolerance, for uniform induction, it is optimal to apply the maximum dose either continuously or intermittently. This result is intuitive since under uniform induction, once any drug dose is applied, there is no downside in applying a larger dose.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Optimal dosing under uniform induction of tolerance.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375710_41540_2025_571_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/491e/12375710/6e42164cec9e/41540_2025_571_Fig4_HTML.jpg" loading="lazy" id="d33e2001" height="775" width="667" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> In our investigation of uniform induction of tolerance, we consider three cases. In Case I, the drug only increases the transition rate from sensitivity to tolerance (<em>μ</em>(0) = <em>μ</em><sub>0</sub> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e1773"><mrow><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>μ</mi></mrow><mrow><mi>max</mi></mrow></msub></mrow></math></span> for <em>c</em> &gt; 0, <em>ν</em>(<em>c</em>) = <em>ν</em><sub>0</sub> for <em>c</em> ≥ 0), in Case II, the drug only decreases the transition rate from tolerance to sensitivity (<em>μ</em>(<em>c</em>) = <em>μ</em><sub>0</sub> for <em>c</em> ≥ 0, <em>ν</em>(0) = <em>ν</em><sub>0</sub> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e1833"><mrow><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>ν</mi></mrow><mrow><mi>min</mi></mrow></msub></mrow></math></span> for <em>c</em> &gt; 0), and in Case III, the drug does both (<em>μ</em>(0) = <em>μ</em><sub>0</sub> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e1864"><mrow><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>μ</mi></mrow><mrow><mi>max</mi></mrow></msub></mrow></math></span> for <em>c</em> &gt; 0, <em>ν</em>(0) = <em>ν</em><sub>0</sub> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e1896"><mrow><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>ν</mi></mrow><mrow><mi>min</mi></mrow></msub></mrow></math></span> for <em>c</em> &gt; 0). <strong>b</strong> The heatmaps show the long-run average tumor growth rate for a range of possible pulsed schedules of the form (<em>c</em>, <em>φ</em>) for Cases I, II and III. The red curves show each for exposure proportion <em>φ</em> the optimal dose <em>c</em>*(<em>φ</em>) to give, and the red dots show the optimal pulsed schedules. For each possible exposure proportion <em>φ</em>, it is optimal to apply the maximum dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e1943"><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span>. For Case I, where the drug only affects transitions from sensitivity to tolerance, it is optimal to apply the maximum dose continuously. For Cases II and III, it is optimal to alternate between the maximum dose and no dose. <strong>c</strong> Under uniform induction of tolerance, we can prove that during the transient phase, it is optimal to apply the maximum dose throughout, and during the equilibrium phase, it is optimal to apply a pulsed schedule <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e1959"><mrow><mo>(</mo><mrow><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></math></span> alternating between the maximum allowed dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e1978"><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span> and no dose (possibly with <em>φ</em> = 1). Examples of optimal strategies are shown for Cases I, II and III. For Case I, where the drug only increases transitions from sensitivity to tolerance, it is optimal to apply the maximum dose continuously throughout the entire treatment period. <strong>d</strong> Proportion of sensitive cells <em>f</em><sub>0</sub>(<em>t</em>) over time under the optimal dosing strategy. Same as for linear induction of tolerance, the optimal dosing strategy maintains the tumor at a fixed proportion between sensitive and tolerant cells in the long run.</p></figcaption></figure><p id="Par18">Second, we can show that the optimal transient strategy is to apply the maximum dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e2009"><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span> continuously under Cases I, II and III (Supplementary Note <a href="#MOESM1" class="usa-link">1</a>). This enables us to derive optimal dosing strategies for all cases under uniform induction of tolerance (Fig. <a href="#Fig4" class="usa-link">4</a>c). For Case I, it is optimal to apply the maximum dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e2028"><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span> continuously from the beginning. This implies that drug-induced tolerance does not necessarily call for low or intermittent dosing. For Cases II and III, it is optimal to start with the maximum dose and then alternate between the maximum dose and no dose. For all cases, the optimal strategy eventually maintains the tumor at a fixed composition between sensitive and tolerant cells (Fig. <a href="#Fig4" class="usa-link">4</a>d). Thus, same as for the case of linear induction, the optimal strategy involves a transient and an equilibrium phase, where the goal of the transient phase is to steer the tumor to a desired composition, and the goal of the equilibrium phase is to maintain it at that composition.</p>
<p id="Par19">In this section, we have referred to a strategy alternating arbitrarily quickly between the maximum dose and no dose as the optimal long-run strategy. Since this is an idealized schedule, it is more precise to say that there exists a sequence of pulsed strategies which achieve the optimal long-run growth rate as <em>t</em><sub>cycle</sub> approaches zero. However, same as for the case of linear induction, the cycle length can be extended significantly without affecting the long-run average growth rate of the tumor (Supplementary Note <a href="#MOESM1" class="usa-link">1</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Integration with in vitro patient tumor data produces individualized clinical insights</h3>
<p id="Par20">Linear and uniform induction of tolerance represent extreme cases where the level of tolerance induction either increases gradually with the dose or is insensitive to the dose. In the former case, lower doses are generally preferred, whereas in the latter case, the maximum dose should be applied whenever the drug is present. If a clinician understands which case applies better to a particular cancer type, for example, based on experience with prior patients, these general insights can help guide clinical decision-making. If the biological mechanism of drug-induced tolerance is furthermore known, it may become possible to distinguish between Cases I, II and III. For linear induction, this can help decide whether to apply low or high doses during the initial stages of treatment, and for uniform induction, it can help decide whether to apply the maximum dose continuously or intermittently in the long run.</p>
<p id="Par21">Our approach can yield even more specific clinical insights when integrated with experiments involving in vitro models of patient tumors. Technologies to derive clinically relevant cell culture models from patient tumor samples are advancing rapidly<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>–<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, and their development has recently been put into focus by the FDA Modernization Act 2.0, which seeks to replace animal models in drug safety and effectiveness testing by in vitro human cell models and in silico computational modeling<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. As a case study, we now apply our insights to in vitro experiments from Russo et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, where the authors treated two colorectal cancer cell lines with increasing doses of anti-cancer agents. They found that when BRAF V600-E mutated WiDr colorectal cancer cells were treated with a combination of dabrafenib and cetuximab, the drugs linearly induced transitions from a drug-sensitive to a drug-persistent state (Case I under linear induction). For RAS/RAF wild-type DiFi cells treated with cetuximab, the drug uniformly induced the same transitions (Case I under uniform induction). We assume that the drug does not affect transitions from persistence to sensitivity, since no such effect is indicated in ref. <sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>.</p>
<p id="Par22">According to our analysis in the previous section, we already know that for the DiFi cells, it is optimal to apply the maximum dose continuously. We therefore focus on the WiDr cells and compare the optimal strategy with dosing regimens considered in ref. <sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. For the purposes of this analysis, we adopt the model of drug effect on cell proliferation from ref. <sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> (Methods). As expected from our analysis in the section on linear induction of tolerance, the optimal strategy over a treatment horizon of <em>T</em> = 100 days applies a constant dose from the beginning (Fig. <a href="#Fig5" class="usa-link">5</a>a). The optimal dose is even smaller here than for the examples considered in Fig. <a href="#Fig2" class="usa-link">2</a> or <em>c</em>* = 0.85 as a proportion of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e2104"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose (in absolute terms, the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e2120"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose is 2.3779 × 10<sup>−7 </sup><em>M</em> and the optimal dose is 2 × 10<sup>−7 </sup><em>M</em>). One reason that the optimal dose is lower than the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e2145"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose in this case is that the dose at which the net sensitive cell division rate <em>λ</em><sub>0</sub>(<em>c</em>) becomes negative is much smaller than the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e2168"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose, meaning that the tumor reduces in size even at <em>c</em>* = 0.85 (Methods).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Optimal dosing for an in vitro colorectal cancer system.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375710_41540_2025_571_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/491e/12375710/b57bd7ceb251/41540_2025_571_Fig5_HTML.jpg" loading="lazy" id="d33e2303" height="500" width="800" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> For BRAF V600-E mutated WiDr cells, Russo et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> observed linearly increasing transitions from sensitivity to tolerance under dabrafenib + cetuximab treatment. Under a sufficiently long time horizon, the optimal dosing strategy applies a low constant dose equal to 85% of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e2201"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose. <strong>b</strong> In ref. <sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, the authors consider both a constant strategy and a linearly increasing strategy with average dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e2224"><mrow><mover accent="true"><mrow><mi>c</mi></mrow><mo>¯</mo></mover><mo>=</mo><mn>4.2</mn></mrow></math></span> (relative to the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e2240"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose) using a time horizon of <em>T</em> = 10 days. The optimal strategy over <em>T</em> = 10 days applies small doses during most of the period and increases the dose towards the end of the horizon. <strong>c</strong> The linearly increasing schedule with average dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e2265"><mrow><mover accent="true"><mrow><mi>c</mi></mrow><mo>¯</mo></mover><mo>=</mo><mn>4.2</mn></mrow></math></span> outperforms the constant-dose schedule with the same average dose in terms of tumor reduction over <em>T</em> = 10 days, which is consistent with the results of ref. <sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. However, the optimal strategy over <em>T</em> = 10 days achieves a 19% greater tumor reduction while applying half as much dose. Moreover, the optimal long-run constant dose <em>c</em>* = 0.85 achieves similar tumor reduction to the linear strategy at less than 20% of the cumulative dose. <strong>d</strong> Out of the four schedules considered in part <strong>b</strong>, the optimal long-run constant dose <em>c</em>* = 0.85 induces the fewest sensitive cells to adopt persistence over <em>T</em> = 10 days.</p></figcaption></figure><p id="Par23">In ref. <sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, the authors considered a shorter time horizon of <em>T</em> = 10 days, and they compared a constant dose schedule with <em>c</em> = 4.2, relative to the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e2321"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose, with a linearly increasing dose schedule involving the same average dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e2337"><mrow><mover accent="true"><mrow><mi>c</mi></mrow><mo>¯</mo></mover><mo>=</mo><mn>4.2</mn></mrow></math></span> over the horizon (Fig. <a href="#Fig5" class="usa-link">5</a>b). They observed that the linearly increasing schedule significantly outperformed the constant-dose schedule, which is confirmed by our analysis (Fig. <a href="#Fig5" class="usa-link">5</a>c). However, we note that the considerable preference for the linear schedule is due to the constant-dose schedule with <em>c</em> = 4.2 being significantly suboptimal. For example, the long-run optimal dose <em>c</em>* = 0.85 leads to a similar tumor decay over <em>T</em> = 10 days as the linear schedule with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e2368"><mrow><mover accent="true"><mrow><mi>c</mi></mrow><mo>¯</mo></mover><mo>=</mo><mn>4.2</mn></mrow></math></span> (Fig. <a href="#Fig5" class="usa-link">5</a>c). The optimal short-term strategy over <em>T</em> = 10 days applies small doses for most of the period, starting at <em>c</em> = 0.90, before increasing the dose at the very end (Fig. <a href="#Fig5" class="usa-link">5</a>b). The optimal short-term strategy achieves a 19% greater tumor reduction than the linear strategy, while applying less than half as much cumulative dose, or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e2396"><mrow><mover accent="true"><mrow><mi>c</mi></mrow><mo>¯</mo></mover><mo>=</mo><mn>2.0</mn></mrow></math></span> on average.</p>
<p id="Par24">Here, we have evaluated the treatment schedules in terms of the tumor size at the end of the treatment horizon. In ref. <sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, the authors considered a different metric of treatment success, the number of persister cells induced under treatment, assuming no persister cells at the beginning of treatment. Even under this metric, the optimal short-term strategy outperforms the linearly increasing schedule, and in fact, the optimal long-run dose <em>c</em>* = 0.85 performs the best (Fig. <a href="#Fig5" class="usa-link">5</a>d). From one perspective, this is not surprising, since if the goal is to minimize the number of sensitive cells that the drug induces to adopt persistence, it is optimal to give no drug and thus induce no sensitive cells to adopt persistence. On the other hand, the fact that the optimal long-run dose induces fewer persister cells than the other three schedules in the short term is precisely why it is better in the long run.</p>
<p id="Par25">In summary, the constant-dose strategy with <em>c</em> = 0.85 is both successful at restricting persister evolution in the short term and optimally controlling the tumor in the long term. In addition, it applies around an 80% lower cumulative dose than the linear schedule with average dose <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e2428"><mrow><mover accent="true"><mrow><mi>c</mi></mrow><mo>¯</mo></mover><mo>=</mo><mn>4.2</mn></mrow></math></span>. These results show how applying our approach to experiments involving in vitro models of patient tumors can yield individualized clinical insights. Determining the optimal dose, understanding how it affects the tumor in the short and long term, and comparing it with other potential dosing strategies would be difficult without the mathematical model.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par26">In this work, we have investigated optimal dosing of anti-cancer treatment under drug-induced plasticity. We found that the optimal strategy always involves both a transient phase and an equilibrium phase, where during the equilibrium phase, the tumor is maintained at a fixed intratumor composition that minimizes its long-run growth rate. Under linear induction of tolerance, the optimal equilibrium strategy is to apply a low constant dose, while under uniform induction of tolerance, it is optimal to apply the maximum dose either continuously or intermittently. We proved this mathematically and provided simple methods for computing the optimal strategies. We also showed that during the transient phase, the optimal strategy steers the tumor to the desired long-run composition while precisely balancing the trade-off between cell kill and tolerance induction. We discussed how these general insights can help guide clinical decision-making, and we showed how more specific insights can be obtained by integrating our methodology with experiments involving in vitro models of patient tumors.</p>
<p id="Par27">We view this work as an initial step towards a more complete understanding of optimal dosing under drug-induced plasticity. We have left many important questions unaddressed which serve as a guide for future work. First, we have only considered the extreme cases where drug tolerance is induced in a linear or uniform fashion. If the transition rate functions <em>μ</em>(<em>c</em>) and <em>ν</em>(<em>c</em>) follow more general Michaelis-Menten or Hill dynamics<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, it may be difficult to apply optimal control analysis, but the optimal pulsed strategy can always be determined using the explicit expression (<a href="#Equ24" class="usa-link">24</a>). Second, while we have assumed that cells transition directly between a drug-sensitive and a drug-tolerant state, these transitions may occur through intermediate states or in a more continuous fashion. In addition, there is evidence that prolonged drug exposure can induce epigenetic reprogramming of tolerant cells and eventually lead to stable drug resistance<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Tolerant cells can furthermore form a reservoir for the evolution of eventual genetic resistance<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. This suggests the need to include more cell states in the model, which is easily accommodated within our framework and we plan to do in future work. Alternatively, the cell state can be treated as a continuous variable, representing gradual evolution towards resistance<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>–<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Nevertheless, we believe that our analysis of the dynamics between sensitive and tolerant cells yields valuable insights that will be useful for understanding the more complex dynamics. Third, we have considered an exponential growth model without competitive or cooperative dynamics. If tumor growth is constrained by a carrying capacity, our insights will continue to hold whenever it is possible to kill the tumor, since then the carrying capacity will not significantly affect the dynamics. However, if recurrence cannot be avoided, it may become optimal to keep the tumor close to the carrying capacity, in order to avoid releasing tolerant cells from competition with sensitive cells<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. This will be important to explore in future work. Finally, we have equated drug dose with drug concentration throughout and ignored the effect of pharmacokinetics, as well as unwanted toxic side effects of the drug. Confirming the clinical applicability of our approach would require an investigation of these dynamics, particularly for the case of uniform tolerance induction, where the dynamics are significantly different depending on whether the drug is present or not.</p>
<p id="Par28">Taking full advantage of our approach for a specific cancer type or a specific patient requires understanding the quantitative dynamics of drug induction for that specific case. Russo et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> have suggested a Bayesian approach for inferring the transition rate function <em>μ</em>(<em>c</em>) from sensitivity to tolerance using in vitro tumor bulk data, but their work only distinguishes between the extreme cases of linear and uniform induction of tolerance. In a recent work, we have suggested a maximum likelihood framework for inferring more general dose-response dynamics for <em>μ</em>(<em>c</em>) based on the Hill equation<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. However, as we have discussed, it is also plausible that the drug inhibits the transition rate <em>ν</em>(<em>c</em>) from tolerance back to sensitivity, or that the drug simultaneously affects both rates. In general, the present work shows that the optimal strategy varies significantly depending on the exact dynamics of tolerance induction. This suggests the need to develop integrated experimental and mathematical tools capable of jointly inferring <em>d</em><sub>0</sub>(<em>c</em>), <em>μ</em>(<em>c</em>) and <em>ν</em>(<em>c</em>) from experimental data, which may require data on the phenotypic composition of the tumor<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Ultimately, these integrated tools can help usher in a new era of precision medicine, where the dynamics of drug induction are determined and dosing strategies are optimized on an individual patient basis<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Model of cell proliferation and phenotypic switching</h3>
<p id="Par29">Our baseline model is a multi-type branching process model in continuous time<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. In the model, cells switch stochastically between two distinct cell states, a drug-sensitive (type-0) and a drug-tolerant (type-1) state. In the absence of drug, a type-0 cell divides into two type-0 cells at rate <em>b</em><sub>0</sub>, it dies at rate <em>d</em><sub>0,</sub> and it transitions to the tolerant type-1 state at rate <em>μ</em> &gt; 0. More precisely, each type-0 cell waits an exponentially distributed amount of time with rate <em>a</em><sub>0</sub> ≔ <em>b</em><sub>0</sub> + <em>d</em><sub>0</sub> + <em>μ</em> before it either divides with probability <em>b</em><sub>0</sub>/<em>a</em><sub>0</sub>, dies with probability <em>d</em><sub>0</sub>/<em>a</em><sub>0</sub>, or transitions to type-1 with probability <em>μ</em>/<em>a</em><sub>0</sub>. A type-1 cell divides at rate <em>b</em><sub>1</sub>, it dies at rate <em>d</em><sub>1</sub>, and it reverts to type-0 at rate <em>ν</em> &gt; 0 (Fig. <a href="#Fig1" class="usa-link">1</a>a). The net birth rates of the two types are denoted by <em>λ</em><sub>0</sub> ≔ <em>b</em><sub>1</sub> − <em>d</em><sub>1</sub> and <em>λ</em><sub>1</sub> ≔ <em>b</em><sub>1</sub> − <em>d</em><sub>1</sub>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Drug effect on cell proliferation</h3>
<p id="Par30">To model the effect of the anti-cancer drug on cell proliferation, we assume that the sensitive cell death rate <em>d</em><sub>0</sub> is an increasing function of the current drug dose <em>c</em>. For simplicity, we assume that the drug-tolerant cells are unaffected by the drug. The specific functional form for <em>d</em><sub>0</sub> is the Michaelis-Menten function, which is commonly used for this purpose:</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e2681" display="block"><mrow><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>−</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>⋅</mo><mfrac><mrow><mi>c</mi></mrow><mrow><mi>c</mi><mo>+</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></mrow></mfrac><mo>=</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>⋅</mo><mfrac><mrow><mi>c</mi></mrow><mrow><mi>c</mi><mo>+</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></mrow></mfrac><mo>.</mo></mrow></math></td>
<td class="label">4</td>
</tr></table>
<p>Here, <em>d</em><sub>0</sub> is the sensitive cell death rate in the absence of drug, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e2773"><msub><mrow><mi>d</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span> is the saturation death rate under an arbitrarily large drug dose, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e2786"><mrow><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>:</mo><mo>=</mo><msub><mrow><mi>d</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>−</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>≥</mo><mn>0</mn></mrow></math></span> is their difference, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e2817"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> is the dose at which the drug has half the maximal effect on sensitive cell proliferation. Note that we write simply <em>d</em><sub>0</sub> instead of <em>d</em><sub>0</sub>(0) to signify the death rate in the absence of drug. The function in (<a href="#Equ4" class="usa-link">4</a>) is concave, but it is worth noting that if the the drug dose is viewed on a log scale, then (<a href="#Equ4" class="usa-link">4</a>) becomes an <em>S</em>-shaped dose-response curve which is characteristic of viability curves in the biological literature (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Drug effect on sensitive cell death rate <em>d</em><sub>0</sub>.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12375710_41540_2025_571_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/491e/12375710/52b5259939cf/41540_2025_571_Fig6_HTML.jpg" loading="lazy" id="d33e2875" height="253" width="675" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The Michaelis-Menten function (<a href="#Equ4" class="usa-link">4</a>) describing the drug effect on cell proliferation is concave in the drug dose <em>c</em> (left panel). When the dose <em>c</em> is viewed on a log scale, the function becomes takes on an <em>S</em>-shape which is characteristic of viability curves in the biological literature (right panel).</p></figcaption></figure><p id="Par31">Expression (<a href="#Equ4" class="usa-link">4</a>) can be rewritten as</p>
<table class="disp-formula p" id="Equc"><tr><td class="formula"><math id="d33e2886" display="block"><mrow><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>⋅</mo><mfrac><mrow><mrow><mo>(</mo><mrow><mi>c</mi><mo>/</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></mrow><mo>)</mo></mrow></mrow><mrow><mrow><mo>(</mo><mrow><mi>c</mi><mo>/</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></mrow><mo>)</mo></mrow><mo>+</mo><mn>1</mn></mrow></mfrac><mo>.</mo></mrow></math></td></tr></table>
<p>Thus, if we measure the drug dose as a proportion of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e2954"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose, we obtain the simpler expression</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e2971" display="block"><mrow><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>⋅</mo><mfrac><mrow><mi>c</mi></mrow><mrow><mi>c</mi><mo>+</mo><mn>1</mn></mrow></mfrac><mo>.</mo></mrow></math></td>
<td class="label">5</td>
</tr></table>
<p>We will usually adopt this scaling of the drug dose and use the simpler form for <em>d</em><sub>0</sub>(<em>c</em>). Similarly, the net birth rate of sensitive cells is given by</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><math id="d33e3022" display="block"><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>b</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>⋅</mo><mfrac><mrow><mi>c</mi></mrow><mrow><mi>c</mi><mo>+</mo><mn>1</mn></mrow></mfrac><mo>,</mo></mrow></math></td>
<td class="label">6</td>
</tr></table>
<p>where we simply write <em>λ</em><sub>0</sub> instead of <em>λ</em><sub>0</sub>(0).</p>
<p id="Par32">While we have assumed in the preceding discussion that the drug increases the death rate of sensitive cells (cytotoxic drug), we can equivalently assume that the drug decreases their division rate (cytostatic drug). This is because our results depend on the division rate and death rate of sensitive cells only through the net birth rate <em>λ</em><sub>0</sub>(<em>c</em>). If we assume that a cytostatic drug influences the division rate as follows:</p>
<table class="disp-formula p" id="Equd"><tr><td class="formula"><math id="d33e3099" display="block"><mrow><msub><mrow><mi>b</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>b</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi mathvariant="normal">Δ</mi><msub><mrow><mi>b</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>⋅</mo><mfrac><mrow><mi>c</mi></mrow><mrow><mi>c</mi><mo>+</mo><mn>1</mn></mrow></mfrac><mo>,</mo></mrow></math></td></tr></table>
<p>the net division rate becomes</p>
<table class="disp-formula p" id="Eque"><tr><td class="formula"><math id="d33e3141" display="block"><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>b</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi mathvariant="normal">Δ</mi><msub><mrow><mi>b</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>⋅</mo><mfrac><mrow><mi>c</mi></mrow><mrow><mi>c</mi><mo>+</mo><mn>1</mn></mrow></mfrac><mo>,</mo></mrow></math></td></tr></table>
<p>which has the same form as expression (<a href="#Equ6" class="usa-link">6</a>).</p></section><section id="Sec12"><h3 class="pmc_sec_title">Drug effect on switching rates</h3>
<p id="Par33">To model the possibility of drug-induced tolerance, we assume that in the presence of the anti-cancer drug, one or both of the transition rates <em>μ</em> and <em>ν</em> change with the drug dose <em>c</em>. If we focus first on the transition rate <em>μ</em> from sensitivity to tolerance, it is natural to assume that it follows a Michaelis-Menten function of the form (<a href="#Equ4" class="usa-link">4</a>):</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><math id="d33e3225" display="block"><mrow><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>μ</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>−</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>⋅</mo><mfrac><mrow><mi>c</mi></mrow><mrow><mi>c</mi><mo>+</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup></mrow></mfrac><mo>=</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi mathvariant="normal">Δ</mi><mi>μ</mi><mo>⋅</mo><mfrac><mrow><mi>c</mi></mrow><mrow><mi>c</mi><mo>+</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup></mrow></mfrac><mo>,</mo></mrow></math></td>
<td class="label">7</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e3304"><mrow><mi mathvariant="normal">Δ</mi><mi>μ</mi><mo>:</mo><mo>=</mo><msub><mrow><mi>μ</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>−</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>≥</mo><mn>0</mn></mrow></math></span>. The dose at which the drug has half the maximal effect on <em>μ</em>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e3334"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup></math></span>, is in general distinct from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e3350"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span>. Here, we will focus on the two extreme cases where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e3366"><mrow><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup><mspace width="0.25em"></mspace><mo>≫</mo><mspace width="0.25em"></mspace><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e3395"><mrow><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup><mspace width="0.25em"></mspace><mo>≪</mo><mspace width="0.25em"></mspace><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></mrow></math></span>. The case <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e3423"><mrow><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup><mspace width="0.25em"></mspace><mo>≫</mo><mspace width="0.25em"></mspace><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></mrow></math></span> means that the drug induces tolerance at doses which are already very effective at killing the sensitive cells. In this case, if the drug dose <em>c</em> is similar in magnitude to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e3454"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span>, then <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e3470"><mrow><mi>c</mi><mspace width="0.25em"></mspace><mo>≪</mo><mspace width="0.25em"></mspace><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup></mrow></math></span> and by Taylor expansion:</p>
<table class="disp-formula p" id="Equf"><tr><td class="formula"><math id="d33e3491" display="block"><mrow><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi mathvariant="normal">Δ</mi><mi>μ</mi><mo>⋅</mo><mfrac><mrow><mi>c</mi><mo>/</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup></mrow><mrow><mi>c</mi><mo>/</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup><mo>+</mo><mn>1</mn></mrow></mfrac><mo>≈</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mfrac><mrow><mi mathvariant="normal">Δ</mi><mi>μ</mi></mrow><mrow><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup></mrow></mfrac><mo>⋅</mo><mi>c</mi><mo>.</mo></mrow></math></td></tr></table>
<p>In other words, <em>μ</em>(<em>c</em>) can be treated as linearly increasing in <em>c</em>. The case <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e3577"><mrow><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup><mspace width="0.25em"></mspace><mo>≪</mo><mspace width="0.25em"></mspace><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></mrow></math></span> means that the drug has fully induced tolerance at low doses which are not yet effective at killing the sensitive cells. In this case, if the drug dose <em>c</em> is similar in magnitude to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e3608"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span>, then <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e3625"><mrow><mi>c</mi><mspace width="0.25em"></mspace><mo>≫</mo><mspace width="0.25em"></mspace><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup></mrow></math></span> and by Taylor expansion for <em>c</em> &gt; 0:</p>
<table class="disp-formula p" id="Equg"><tr><td class="formula"><math id="d33e3649" display="block"><mrow><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi mathvariant="normal">Δ</mi><mi>μ</mi><mo>⋅</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mn>1</mn><mo>+</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>μ</mi></mrow></msubsup><mo>/</mo><mi>c</mi></mrow></mfrac><mo>≈</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi mathvariant="normal">Δ</mi><mi>μ</mi><mo>=</mo><msub><mrow><mi>μ</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>.</mo></mrow></math></td></tr></table>
<p>In other words, <em>μ</em> is effectively constant in the presence of drug.</p>
<p id="Par34">Based on the preceding, we assume the following two functional forms for <em>μ</em>:</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par35"><em>μ</em>(<em>c</em>) = <em>μ</em><sub>0</sub> + <em>k</em><em>c</em>, where <em>k</em> ≥ 0 (tolerance is linearly induced).</p></li>
<li><p id="Par36"><em>μ</em>(0) = <em>μ</em><sub>0</sub> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e3751"><mrow><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>μ</mi></mrow><mrow><mi>max</mi></mrow></msub></mrow></math></span> for <em>c</em> &gt; 0 (tolerance is uniformly induced).</p></li>
</ol>
<p>The reason we focus on these two extremes is twofold. First, these two exact forms were observed empirically in the recent investigation by Russo et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, which is the only work we know of which has attempted to quantify drug-induced tolerance in a dose-dependent manner using in vitro data. Second, due to their simplicity relative to the general case (<a href="#Equ7" class="usa-link">7</a>), we are able to establish rigorous mathematical results on optimal dosing strategies.</p>
<p id="Par37">For the transition rate <em>ν</em> from tolerance back to sensitivity, the corresponding Michaelis-Menten function has the following form:</p>
<table class="disp-formula p" id="Equh"><tr><td class="formula"><math id="d33e3788" display="block"><mrow><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>ν</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>ν</mi></mrow><mrow><mi>min</mi></mrow></msub><mo>−</mo><msub><mrow><mi>ν</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>⋅</mo><mfrac><mrow><mi>c</mi></mrow><mrow><mi>c</mi><mo>+</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>ν</mi></mrow></msubsup></mrow></mfrac><mo>=</mo><msub><mrow><mi>ν</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi mathvariant="normal">Δ</mi><mi>ν</mi><mo>⋅</mo><mfrac><mrow><mi>c</mi></mrow><mrow><mi>c</mi><mo>+</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>ν</mi></mrow></msubsup></mrow></mfrac><mo>,</mo></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e3867"><mrow><mi mathvariant="normal">Δ</mi><mi>ν</mi><mo>:</mo><mo>=</mo><msub><mrow><mi>ν</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><msub><mrow><mi>ν</mi></mrow><mrow><mi>min</mi></mrow></msub><mo>≥</mo><mn>0</mn></mrow></math></span>. We will also assume the following two functional forms for <em>ν</em>:</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par38"><em>ν</em>(<em>c</em>) = <em>ν</em><sub>0</sub> − <em>m</em><em>c</em>, where <em>m</em> ≥ 0.</p></li>
<li><p id="Par39"><em>ν</em>(0) = <em>ν</em><sub>0</sub> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e3931"><mrow><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>ν</mi></mrow><mrow><mi>min</mi></mrow></msub></mrow></math></span> for <em>c</em> &gt; 0.</p></li>
</ol>
<p>By assumption, <em>ν</em> is a decreasing function of <em>c</em>, since we are modeling the case where the drug inhibits transitions from drug-tolerance to drug-sensitivity.</p>
<p id="Par40">Throughout, we assume that <em>μ</em><sub>0</sub> &gt; 0 and <em>ν</em><sub>0</sub> &gt; 0, meaning that cells are able to transition between the two states even in the absence of drug.</p></section><section id="Sec13"><h3 class="pmc_sec_title">System equations</h3>
<p id="Par41">Let <em>Z</em><sub>0</sub>(<em>t</em>) and <em>Z</em><sub>1</sub>(<em>t</em>) denote the number of sensitive and tolerant cells, respectively, at time <em>t</em>. Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e3995"><mrow><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow><mo>:</mo><mo>=</mo><mi mathvariant="double-struck">E</mi><mrow><mo>[</mo><mrow><msub><mrow><mi>Z</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow><mo>]</mo></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><math id="d33e4032"><mrow><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow><mo>:</mo><mo>=</mo><mi mathvariant="double-struck">E</mi><mrow><mo>[</mo><mrow><msub><mrow><mi>Z</mi></mrow><mrow><mn>1</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow><mo>]</mo></mrow></mrow></math></span> be the mean number of sensitive and tolerant cells at time <em>t</em>. These mean functions satisfy the differential equations</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><math id="d33e4073" display="block"><mtable columnspacing="1em" rowspacing="4pt"><mtr><mtd columnalign="right"><mstyle displaystyle="true" scriptlevel="0"><mfrac><mrow><mi>d</mi><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mstyle></mtd><mtd><mo>=</mo></mtd><mtd columnalign="left"><mo>(</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>(</mo><mi>c</mi><mo>)</mo><mo>−</mo><mi>μ</mi><mo>(</mo><mi>c</mi><mo>)</mo><mo>)</mo><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>ν</mi><mo>(</mo><mi>c</mi><mo>)</mo><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo></mtd></mtr><mtr><mtd columnalign="right"><mstyle displaystyle="true" scriptlevel="0"><mfrac><mrow><mi>d</mi><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mstyle></mtd><mtd><mo>=</mo></mtd><mtd columnalign="left"><mo>(</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><mi>ν</mi><mo>(</mo><mi>c</mi><mo>)</mo><mo>)</mo><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>+</mo><mi>μ</mi><mo>(</mo><mi>c</mi><mo>)</mo><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>.</mo></mtd></mtr></mtable></math></td>
<td class="label">8</td>
</tr></table>
<p>Here, the dose <em>c</em> should be viewed as a function of time, <em>c</em>(<em>t</em>). The total population size <em>n</em>(<em>t</em>) ≔ <em>n</em><sub>0</sub>(<em>t</em>) + <em>n</em><sub>1</sub>(<em>t</em>) obeys the differential equation</p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><math id="d33e4209" display="block"><mrow><mfrac><mrow><mi>d</mi><mi>n</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>.</mo></mrow></math></td>
<td class="label">9</td>
</tr></table>
<p>Let <em>f</em><sub>0</sub>(<em>t</em>) ≔ <em>n</em><sub>0</sub>(<em>t</em>)/<em>n</em>(<em>t</em>) and <em>f</em><sub>1</sub>(<em>t</em>) ≔ 1 − <em>f</em><sub>0</sub>(<em>t</em>) be the proportions of each cell type in the population at time <em>t</em>. Then</p>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><math id="d33e4294" display="block"><mtable><mtr><mtd columnalign="left"><mfrac><mrow><mi>d</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac></mtd><mtd columnalign="left"><mo>=</mo></mtd><mtd columnalign="left"><mfrac><mrow><mrow><mo>(</mo><mrow><mi>d</mi><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>/</mo><mi>d</mi><mi>t</mi></mrow><mo>)</mo></mrow><mo>⋅</mo><mi>n</mi><mo>−</mo><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>⋅</mo><mrow><mo>(</mo><mrow><mi>d</mi><mi>n</mi><mo>/</mo><mi>d</mi><mi>t</mi></mrow><mo>)</mo></mrow></mrow><mrow><msup><mrow><mi>n</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac></mtd></mtr><mtr><mtd columnalign="left"></mtd><mtd columnalign="left"><mo>=</mo></mtd><mtd columnalign="left"><mfrac><mrow><mfenced close=")" open="("><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></mfenced><mi>n</mi><mo>−</mo><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></mfenced></mrow><mrow><msup><mrow><mi>n</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac></mtd></mtr><mtr><mtd columnalign="left"></mtd><mtd columnalign="left"><mo>=</mo></mtd><mtd columnalign="left"><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow></mrow></mfenced></mtd></mtr><mtr><mtd columnalign="left"></mtd><mtd columnalign="left"><mo>=</mo></mtd><mtd columnalign="left"><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfenced><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfenced><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>.</mo></mtd></mtr></mtable></math></td>
<td class="label">10</td>
</tr></table>
<p>Expression (<a href="#Equ9" class="usa-link">9</a>) can be rewritten as follows:</p>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><math id="d33e4615" display="block"><mrow><mfrac><mrow><mi>d</mi><mi>n</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow></mrow></mfenced><mi>n</mi><mo>=</mo><mfenced close=")" open="("><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow><mo>)</mo></mrow><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></mfenced><mi>n</mi><mo>.</mo></mrow></math></td>
<td class="label">11</td>
</tr></table>
<p>Solving this differential equation, the mean tumor size at time <em>T</em> is given by the explicit expression</p>
<table class="disp-formula p" id="Equ12"><tr>
<td class="formula"><math id="d33e4708" display="block"><mrow><mi>n</mi><mrow><mo>(</mo><mrow><mi>T</mi></mrow><mo>)</mo></mrow><mo>=</mo><mi>n</mi><mrow><mo>(</mo><mrow><mn>0</mn></mrow><mo>)</mo></mrow><mi>exp</mi><mfenced close=")" open="("><mrow><msubsup><mrow><mo>∫</mo></mrow><mrow><mn>0</mn></mrow><mrow><mi>T</mi></mrow></msubsup><mfenced close=")" open="("><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow><mo>)</mo></mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></mfenced><mi>d</mi><mi>t</mi></mrow></mfenced><mo>.</mo></mrow></math></td>
<td class="label">12</td>
</tr></table>
<p>Expressions (<a href="#Equ11" class="usa-link">11</a>) and (<a href="#Equ12" class="usa-link">12</a>) indicate that the instantaneous growth rate of the population at time <em>t</em> is given by</p>
<table class="disp-formula p" id="Equ13"><tr>
<td class="formula"><math id="d33e4788" display="block"><mrow><mi>u</mi><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>:</mo><mo>=</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow><mo>)</mo></mrow><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo></mrow></math></td>
<td class="label">13</td>
</tr></table>
<p>which will be an important quantity in our investigation of optimal treatment strategies.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Behavior under a constant dose</h3>
<p id="Par42">When a constant drug dose is applied, <em>c</em>(<em>t</em>) = <em>c</em>, all model parameters <em>λ</em><sub>0</sub>(<em>c</em>), <em>λ</em><sub>1</sub>, <em>μ</em>(<em>c</em>) and <em>ν</em>(<em>c</em>) are constant. In this case, the system equations (<a href="#Equ8" class="usa-link">8</a>) admit explicit solutions. To simplify the notation, we let <em>λ</em><sub>0</sub>, <em>λ</em><sub>1</sub>, <em>μ</em> and <em>ν</em> denote the constant parameters. We first note that the dynamics of the system can be encoded in the so-called <em>infinitesimal generator</em></p>
<table class="disp-formula p" id="Equi"><tr><td class="formula"><math id="d33e4901" display="block"><mrow><mi mathvariant="bold">A</mi><mo>:</mo><mo>=</mo><mfenced close="]" open="["><mrow><mtable><mtr><mtd columnalign="center"><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>μ</mi></mtd><mtd columnalign="center"><mi>μ</mi></mtd></mtr><mtr><mtd columnalign="center"><mi>ν</mi></mtd><mtd columnalign="center"><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><mi>ν</mi><mo>,</mo></mtd></mtr></mtable></mrow></mfenced><mo>,</mo></mrow></math></td></tr></table>
<p>where the (<em>i</em> + 1, <em>j</em> + 1)-th element describes the net rate at which a cell of type-<em>i</em> produces a cell of type-<em>j</em>. The infinitesimal generator has distinct real eigenvalues <em>ρ</em> &lt; <em>σ</em> given by</p>
<table class="disp-formula p" id="Equj"><tr><td class="formula"><math id="d33e4960" display="block"><mtable><mtr><mtd columnalign="right"><mi>σ</mi></mtd><mtd columnalign="center"><mo>=</mo></mtd><mtd columnalign="left"><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><mfenced close=")" open="("><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>μ</mi></mrow><mo>)</mo></mrow><mo>+</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><mi>ν</mi></mrow><mo>)</mo></mrow><mo>+</mo><msqrt><mrow><msup><mrow><mfenced close=")" open="("><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>μ</mi></mrow><mo>)</mo></mrow><mo>−</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><mi>ν</mi></mrow><mo>)</mo></mrow></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup><mo>+</mo><mn>4</mn><mi>μ</mi><mi>ν</mi></mrow></msqrt></mrow></mfenced><mo>,</mo></mtd></mtr><mtr><mtd columnalign="right"><mi>ρ</mi></mtd><mtd columnalign="center"><mo>=</mo></mtd><mtd columnalign="left"><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><mfenced close=")" open="("><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>μ</mi></mrow><mo>)</mo></mrow><mo>+</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><mi>ν</mi></mrow><mo>)</mo></mrow><mo>−</mo><msqrt><mrow><msup><mrow><mfenced close=")" open="("><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>μ</mi></mrow><mo>)</mo></mrow><mo>−</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><mi>ν</mi></mrow><mo>)</mo></mrow></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup><mo>+</mo><mn>4</mn><mi>μ</mi><mi>ν</mi></mrow></msqrt></mrow></mfenced><mo>.</mo></mtd></mtr></mtable></math></td></tr></table>
<p>Now define</p>
<table class="disp-formula p" id="Equk"><tr><td class="formula"><math id="d33e5115" display="block"><mtable><mtr><mtd columnalign="right"><mi>δ</mi></mtd><mtd columnalign="center"><mo>:</mo><mo>=</mo></mtd><mtd columnalign="left"><mfrac><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>μ</mi></mrow><mo>)</mo></mrow><mo>−</mo><mi>ρ</mi></mrow><mrow><mi>ν</mi></mrow></mfrac><mo>,</mo></mtd></mtr><mtr><mtd columnalign="right"><mi>β</mi></mtd><mtd columnalign="center"><mo>:</mo><mo>=</mo></mtd><mtd columnalign="left"><mfrac><mrow><mi>σ</mi><mo>−</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>μ</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>ν</mi></mrow></mfrac><mo>.</mo></mtd></mtr></mtable></math></td></tr></table>
<p>If <em>n</em><sub>0</sub>(0) = <em>n</em> and <em>n</em><sub>1</sub>(0) = <em>m</em>, the mean number of cells of each type can be written explicitly as</p>
<table class="disp-formula p" id="Equ14"><tr>
<td class="formula"><math id="d33e5190" display="block"><mtable><mtr><mtd columnalign="right"><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mtd><mtd columnalign="center"><mo>=</mo></mtd><mtd columnalign="left"><mfrac><mrow><mi>n</mi><mi>δ</mi><mo>+</mo><mi>m</mi></mrow><mrow><mi>δ</mi><mo>+</mo><mi>β</mi></mrow></mfrac><msup><mrow><mi>e</mi></mrow><mrow><mi>σ</mi><mi>t</mi></mrow></msup><mo>+</mo><mfrac><mrow><mi>n</mi><mi>β</mi><mo>−</mo><mi>m</mi></mrow><mrow><mi>δ</mi><mo>+</mo><mi>β</mi></mrow></mfrac><msup><mrow><mi>e</mi></mrow><mrow><mi>ρ</mi><mi>t</mi></mrow></msup><mo>,</mo></mtd></mtr><mtr><mtd columnalign="right"><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mtd><mtd columnalign="center"><mo>=</mo></mtd><mtd columnalign="left"><mfrac><mrow><mi>β</mi><mrow><mo>(</mo><mrow><mi>n</mi><mi>δ</mi><mo>+</mo><mi>m</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>δ</mi><mo>+</mo><mi>β</mi></mrow></mfrac><msup><mrow><mi>e</mi></mrow><mrow><mi>σ</mi><mi>t</mi></mrow></msup><mo>−</mo><mfrac><mrow><mi>δ</mi><mrow><mo>(</mo><mrow><mi>n</mi><mi>β</mi><mo>−</mo><mi>m</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>δ</mi><mo>+</mo><mi>β</mi></mrow></mfrac><msup><mrow><mi>e</mi></mrow><mrow><mi>ρ</mi><mi>t</mi></mrow></msup><mo>.</mo></mtd></mtr></mtable></math></td>
<td class="label">14</td>
</tr></table>
<p>These expressions indicate two important aspects of the long-run dynamics. First, the tumor eventually grows (or decays) at exponential rate <em>σ</em>. Second, the intratumor composition eventually reaches an equilibrium where the proportion between type-1 and type-0 becomes the constant <em>β</em>. We denote the equilibrium proportion of the type-0 cells as</p>
<table class="disp-formula p" id="Equ15"><tr>
<td class="formula"><math id="d33e5315" display="block"><mrow><msub><mrow><mover accent="true"><mrow><mi>f</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mo>:</mo><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mn>1</mn><mo>+</mo><mi>β</mi></mrow></mfrac><mo>=</mo><mfrac><mrow><mi>ν</mi></mrow><mrow><mi>σ</mi><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>μ</mi><mo>+</mo><mi>ν</mi></mrow></mfrac><mo>.</mo></mrow></math></td>
<td class="label">15</td>
</tr></table>
<p>When we are considering a constant treatment, <em>c</em>(<em>t</em>) = <em>c</em>, we will write <em>σ</em>(<em>c</em>) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><math id="d33e5374"><mrow><msub><mrow><mover accent="true"><mrow><mi>f</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></math></span> to explicitly denote the dependence on the dose <em>c</em>. For detailed derivations of the above formulas, we refer to Appendix A.2 of our previous work<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Optimal control problem</h3>
<p id="Par43">To determine the optimal dosing strategy, we assume a finite treatment horizon [0, <em>T</em>] with <em>T</em> &gt; 0. Let PC([0, <em>T</em>]) denote the set of all piecewise continuous functions on [0, <em>T</em>]. The space of allowable dosing strategies is</p>
<table class="disp-formula p" id="Equl"><tr><td class="formula"><math id="d33e5419" display="block"><mrow><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi>T</mi></mrow></msub><mo>:</mo><mo>=</mo><mrow><mo>{</mo><mrow><mi>c</mi><mo>∈</mo><mi mathvariant="normal">PC</mi><mrow><mo>(</mo><mrow><mrow><mo>[</mo><mrow><mn>0</mn><mo>,</mo><mi>T</mi></mrow><mo>]</mo></mrow></mrow><mo>)</mo></mrow><mo>:</mo><mi>c</mi><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow><mo>≤</mo><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>,</mo><mspace width="0.25em"></mspace><mo>∀</mo><mi>t</mi><mo>∈</mo><mrow><mo>[</mo><mrow><mn>0</mn><mo>,</mo><mi>T</mi></mrow><mo>]</mo></mrow></mrow><mo>}</mo></mrow><mo>,</mo></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><math id="d33e5483"><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span> is the maximum allowable instantaneous dose. For simplicity, when deriving optimal dosing strategies, we assume that the drug dose corresponds perfectly to the drug concentration reaching the tumor, thereby ignoring pharmacokinetic effects. Our main objective is to find the dosing strategy <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><math id="d33e5496"><mrow><mi>c</mi><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi>T</mi></mrow></msub></mrow></math></span> which minimizes the expected tumor size at the end of the treatment period. More specifically, we aim to solve the optimal control problem</p>
<table class="disp-formula p" id="Equm"><tr><td class="formula"><math id="d33e5513" display="block"><mrow><munder><mrow><mi>inf</mi></mrow><mrow><mi>c</mi><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi>T</mi></mrow></msub></mrow></munder><mi>n</mi><mrow><mo>(</mo><mrow><mi>T</mi></mrow><mo>)</mo></mrow><mo>=</mo><munder><mrow><mi>inf</mi></mrow><mrow><mi>c</mi><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi>T</mi></mrow></msub></mrow></munder><mrow><mo>(</mo><mrow><msub><mrow><mi>n</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>T</mi></mrow><mo>)</mo></mrow><mo>+</mo><msub><mrow><mi>n</mi></mrow><mrow><mn>1</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>T</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>.</mo></mrow></math></td></tr></table>
<p>By (<a href="#Equ12" class="usa-link">12</a>) and (<a href="#Equ13" class="usa-link">13</a>), it is equivalent to solve the problem</p>
<table class="disp-formula p" id="Equ16"><tr>
<td class="formula"><math id="d33e5586" display="block"><mrow><munder><mrow><mi>inf</mi></mrow><mrow><mi>c</mi><mo>∈</mo><msub><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">C</mi></mrow><mrow><mi>T</mi></mrow></msub></mrow></munder><msubsup><mrow><mo>∫</mo></mrow><mrow><mn>0</mn></mrow><mrow><mi>T</mi></mrow></msubsup><mi>u</mi><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mi>d</mi><mi>t</mi><mo>,</mo></mrow></math></td>
<td class="label">16</td>
</tr></table>
<p>where <em>u</em>(<em>c</em>, <em>f</em><sub>0</sub>) ≔ (<em>λ</em><sub>0</sub>(<em>c</em>) − <em>λ</em><sub>1</sub>)<em>f</em><sub>0</sub> + <em>λ</em><sub>1</sub> is the instantaneous growth rate of the population and <em>f</em><sub>0</sub>(<em>t</em>) is governed by the differential equation</p>
<table class="disp-formula p" id="Equn"><tr><td class="formula"><math id="d33e5668" display="block"><mrow><mfrac><mrow><mi>d</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfenced><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfenced><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>.</mo></mrow></math></td></tr></table>
<p>Note that under the assumption that <em>c</em> is piecewise continuous on [0, <em>T</em>], <em>f</em><sub>0</sub> is piecewise continuously differentiable on [0, <em>T</em>]. Over a finite treatment horizon [0, <em>T</em>], the average growth rate of the population is given by</p>
<table class="disp-formula p" id="Equo"><tr><td class="formula"><math id="d33e5777" display="block"><mrow><msub><mrow><mi>ū</mi></mrow><mrow><mi>T</mi></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>T</mi></mrow></mfrac><msubsup><mrow><mo>∫</mo></mrow><mrow><mn>0</mn></mrow><mrow><mi>T</mi></mrow></msubsup><mi>u</mi><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mi>d</mi><mi>t</mi><mo>.</mo></mrow></math></td></tr></table>
<p>In our investigation of the optimal control problem (<a href="#Equ16" class="usa-link">16</a>), we will be interested in knowing which treatment is the best in the long run as <em>T</em> → <em>∞</em>. Since the integral in (<a href="#Equ16" class="usa-link">16</a>) can be unbounded as <em>T</em> → <em>∞</em>, we will also consider the <em>long-run average growth rate</em></p>
<table class="disp-formula p" id="Equ17"><tr>
<td class="formula"><math id="d33e5846" display="block"><mrow><msub><mrow><mi>ū</mi></mrow><mrow><mi>∞</mi></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><munder><mrow><mi mathvariant="normal">lim</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">sup</mi></mrow><mrow><mi>T</mi><mo>→</mo><mi>∞</mi></mrow></munder><msub><mrow><mi>ū</mi></mrow><mrow><mi>T</mi></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><munder><mrow><mi mathvariant="normal">lim</mi><mspace width="0.25em"></mspace><mi mathvariant="normal">sup</mi></mrow><mrow><mi>T</mi><mo>→</mo><mi>∞</mi></mrow></munder><mfrac><mrow><mn>1</mn></mrow><mrow><mi>T</mi></mrow></mfrac><msubsup><mrow><mo>∫</mo></mrow><mrow><mn>0</mn></mrow><mrow><mi>T</mi></mrow></msubsup><mi>u</mi><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mi>d</mi><mi>t</mi><mo>,</mo></mrow></math></td>
<td class="label">17</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><math id="d33e5927"><mrow><mi>c</mi><mo>:</mo><mrow><mo>[</mo><mrow><mn>0</mn><mo>,</mo><mi>∞</mi></mrow><mo>)</mo></mrow><mo>→</mo><mrow><mo>[</mo><mrow><mn>0</mn><mo>,</mo><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub></mrow><mo>]</mo></mrow></mrow></math></span> is considered a treatment with an infinite time horizon.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Implementation of forward–backward sweep method</h3>
<p id="Par44">For the case of linear induction of tolerance, we use the forward–backward sweep method<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> to solve the optimal control problem (<a href="#Equ16" class="usa-link">16</a>). If we set</p>
<table class="disp-formula p" id="Equp"><tr><td class="formula"><math id="d33e5971" display="block"><mrow><mi>g</mi><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><mi>c</mi></mrow><mo>)</mo></mrow><mo>:</mo><mo>=</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfenced><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfenced><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>,</mo></mrow></math></td></tr></table>
<p>then <em>f</em><sub>0</sub>(<em>t</em>) is governed by the differential equation <em>d</em><em>f</em><sub>0</sub>/<em>d</em><em>t</em> = <em>g</em>(<em>f</em><sub>0</sub>, <em>c</em>). The so-called <em>Hamiltonian</em> for this problem is</p>
<table class="disp-formula p" id="Equ18"><tr>
<td class="formula"><math id="d33e6101" display="block"><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">H</mi><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><mi>c</mi><mo>,</mo><mi>γ</mi></mrow><mo>)</mo></mrow><mo>=</mo><mi>u</mi><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>γ</mi><mi>g</mi><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><mi>c</mi></mrow><mo>)</mo></mrow><mo>,</mo></mrow></math></td>
<td class="label">18</td>
</tr></table>
<p>where <em>γ</em> = (<em>γ</em>(<em>t</em>))<sub><em>t</em>∈[0, <em>T</em>]</sub> is a Lagrangian multiplier function. By Pontryagin’s principle, the optimal solution <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><math id="d33e6174"><mrow><msup><mrow><mi>c</mi></mrow><mrow><mo>*</mo></mrow></msup><mo>=</mo><msub><mrow><mrow><mo>(</mo><mrow><msup><mrow><mi>c</mi></mrow><mrow><mo>*</mo></mrow></msup><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mrow><mi>t</mi><mo>∈</mo><mrow><mo>[</mo><mrow><mn>0</mn><mo>,</mo><mi>T</mi></mrow><mo>]</mo></mrow></mrow></msub></mrow></math></span> must satisfy</p>
<table class="disp-formula p" id="Equq"><tr><td class="formula"><math id="d33e6215" display="block"><mrow><msup><mrow><mi>c</mi></mrow><mrow><mo>*</mo></mrow></msup><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi mathvariant="normal">argmin</mi></mrow><mrow><mi>c</mi></mrow></msub><mi class="MJX-tex-caligraphic" mathvariant="script">H</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><mi>c</mi><mo>,</mo><mi>γ</mi></mrow><mo>)</mo></mrow></mrow></math></td></tr></table>
<p>for each <em>t</em> ∈ [0, <em>T</em>], where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><math id="d33e6262"><mrow><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><msub><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mrow><mi>t</mi><mo>∈</mo><mrow><mo>[</mo><mrow><mn>0</mn><mo>,</mo><mi>T</mi></mrow><mo>]</mo></mrow></mrow></msub></mrow></math></span> and (<em>γ</em>(<em>t</em>))<sub><em>t</em>∈[0, <em>T</em>]</sub> satisfy the differential equations</p>
<table class="disp-formula p" id="Equr"><tr><td class="formula"><math id="d33e6318" display="block"><mtable><mtr><mtd columnalign="right"><mfrac><mrow><mi>∂</mi><mi class="MJX-tex-caligraphic" mathvariant="script">H</mi></mrow><mrow><mi>∂</mi><mi>γ</mi></mrow></mfrac></mtd><mtd columnalign="center"><mo>=</mo></mtd><mtd columnalign="left"><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mo>′</mo></mrow></msubsup><mo>,</mo></mtd></mtr><mtr><mtd columnalign="right"><mfrac><mrow><mi>∂</mi><mi class="MJX-tex-caligraphic" mathvariant="script">H</mi></mrow><mrow><mi>∂</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow></mfrac></mtd><mtd columnalign="center"><mo>=</mo></mtd><mtd columnalign="left"><mo>−</mo><msup><mrow><mi>γ</mi></mrow><mrow><mo>′</mo></mrow></msup><mo>.</mo></mtd></mtr></mtable></math></td></tr></table>
<p>The boundary conditions are <em>f</em><sub>0</sub>(0) = <em>α</em> for some fixed <em>α</em> ∈ [0, 1] and <em>γ</em>(<em>T</em>) = 0. For our problem, the two differenial equations above become</p>
<table class="disp-formula p" id="Equ19"><tr>
<td class="formula"><math id="d33e6390" display="block"><mrow><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mo>′</mo></mrow></msubsup><mo>=</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mspace width="0.25em"></mspace><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>,</mo><mspace width="1.0em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mn>0</mn></mrow><mo>)</mo></mrow><mo>=</mo><mi>α</mi><mo>,</mo></mrow></math></td>
<td class="label">19</td>
</tr></table>
<table class="disp-formula p" id="Equ20"><tr>
<td class="formula"><math id="d33e6498" display="block"><mtable><mtr><mtd columnalign="left"><msup><mrow><mi>γ</mi></mrow><mrow><mo>′</mo></mrow></msup><mo>=</mo><mo>−</mo><mfenced close=")" open="("><mrow><mn>2</mn><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow></mfenced></mtd></mtr><mtr><mtd columnalign="left"><mspace width="1.0em"></mspace><mi>γ</mi><mo>−</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>,</mo><mspace width="1.0em"></mspace><mi>γ</mi><mrow><mo>(</mo><mrow><mi>T</mi></mrow><mo>)</mo></mrow><mo>=</mo><mn>0</mn><mo>.</mo></mtd></mtr></mtable></math></td>
<td class="label">20</td>
</tr></table>
<p>To compute the optimal solution, we apply the forward-backward sweep method, which is an iterative procedure that proceeds as follows:</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par45">We discretize time and consider a set of time points <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><math id="d33e6623"><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">T</mi><mo>=</mo><mrow><mo>[</mo><mrow><mn>0</mn><mo>,</mo><msub><mrow><mi>t</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mrow><mi>t</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><mi>T</mi></mrow><mo>]</mo></mrow></mrow></math></span>.</p></li>
<li><p id="Par46">We make an initial guess for the optimal policy <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><math id="d33e6661"><msub><mrow><mrow><mo>(</mo><mrow><mi>c</mi><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mrow><mi>t</mi><mo>∈</mo><mi class="MJX-tex-caligraphic" mathvariant="script">T</mi></mrow></msub></math></span>. We simply use <em>c</em>(<em>t</em>) = 0 for all <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><math id="d33e6692"><mrow><mi>t</mi><mo>∈</mo><mi class="MJX-tex-caligraphic" mathvariant="script">T</mi></mrow></math></span>.</p></li>
<li><p id="Par47">We solve (<a href="#Equ19" class="usa-link">19</a>) using the initial condition <em>f</em><sub>0</sub>(0) = <em>α</em>, assuming the initial guess for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><math id="d33e6720"><msub><mrow><mrow><mo>(</mo><mrow><mi>c</mi><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mrow><mi>t</mi><mo>∈</mo><mi class="MJX-tex-caligraphic" mathvariant="script">T</mi></mrow></msub></math></span>.</p></li>
<li><p id="Par48">We solve (<a href="#Equ20" class="usa-link">20</a>) <em>backwards in time</em> using the initial condition <em>γ</em>(<em>T</em>) = 0, assuming the initial guess for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><math id="d33e6762"><msub><mrow><mrow><mo>(</mo><mrow><mi>c</mi><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mrow><mi>t</mi><mo>∈</mo><mi class="MJX-tex-caligraphic" mathvariant="script">T</mi></mrow></msub></math></span> and the trajectory for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><math id="d33e6788"><msub><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mrow><mi>t</mi><mo>∈</mo><mi class="MJX-tex-caligraphic" mathvariant="script">T</mi></mrow></msub></math></span> from step (3).</p></li>
<li><div class="p" id="Par49">Using the obtained trajectories for (<em>f</em><sub>0</sub>(<em>t</em>)) and (<em>γ</em>(<em>t</em>)), for each fixed <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><math id="d33e6835"><mrow><mi>t</mi><mo>∈</mo><mi class="MJX-tex-caligraphic" mathvariant="script">T</mi></mrow></math></span> we compute the optimal dose at time <em>t</em> as<table class="disp-formula p" id="Equs"><tr><td class="formula"><math id="d33e6852" display="block"><mrow><msup><mrow><mi>c</mi></mrow><mrow><mo>*</mo></mrow></msup><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi mathvariant="normal">argmax</mi></mrow><mrow><mi>c</mi></mrow></msub><mi class="MJX-tex-caligraphic" mathvariant="script">H</mi><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><mi>c</mi><mo>,</mo><mi>γ</mi></mrow><mo>)</mo></mrow><mo>,</mo></mrow></math></td></tr></table>which boils down to solving<table class="disp-formula p" id="Equ21"><tr>
<td class="formula"><math id="d33e6896" display="block"><mrow><mfrac><mrow><mi>∂</mi><mi class="MJX-tex-caligraphic" mathvariant="script">H</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>=</mo><mn>0</mn><mo>.</mo></mrow></math></td>
<td class="label">21</td>
</tr></table>For our problem, this condition can be written as a second-degree polynomial equation in <em>c</em>. Since<table class="disp-formula p" id="Equt"><tr><td class="formula"><math id="d33e6919" display="block"><mtable><mtr><mtd columnalign="left"><mfrac><mrow><mi>∂</mi><mi>u</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mspace width="0.25em"></mspace><mo>=</mo><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mfrac><mrow><mi>∂</mi><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>,</mo></mtd></mtr><mtr><mtd columnalign="left"><mfrac><mrow><mi>∂</mi><mi>g</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mspace width="0.16em"></mspace><mo>=</mo><mspace width="0.16em"></mspace><mo>−</mo><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mfrac><mrow><mi>∂</mi><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>+</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mfrac><mrow><mi>∂</mi><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>−</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mfrac><mrow><mi>∂</mi><mi>μ</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>−</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mfrac><mrow><mi>∂</mi><mi>ν</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>+</mo><mfrac><mrow><mi>∂</mi><mi>ν</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>,</mo></mtd></mtr></mtable></math></td></tr></table>we have<table class="disp-formula p" id="Equu"><tr><td class="formula"><math id="d33e7041" display="block"><mtable><mtr><mtd columnalign="left"><mfrac><mrow><mi>∂</mi><mi class="MJX-tex-caligraphic" mathvariant="script">H</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac></mtd><mtd columnalign="left"><mo>=</mo></mtd><mtd columnalign="left"><mfrac><mrow><mi>∂</mi><mi>u</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>+</mo><mi>γ</mi><mfrac><mrow><mi>∂</mi><mi>g</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac></mtd></mtr><mtr><mtd columnalign="left"></mtd><mtd columnalign="left"><mo>=</mo></mtd><mtd columnalign="left"><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>γ</mi><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mo>+</mo><mi>γ</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mfrac><mrow><mi>∂</mi><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>−</mo><mi>γ</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mfrac><mrow><mi>∂</mi><mi>μ</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>+</mo><mi>γ</mi><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mfrac><mrow><mi>∂</mi><mi>ν</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>.</mo></mtd></mtr></mtable></math></td></tr></table>Now,<table class="disp-formula p" id="Equv"><tr><td class="formula"><math id="d33e7157" display="block"><mtable><mtr><mtd columnalign="left"><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>⋅</mo><mfrac><mrow><mi>c</mi></mrow><mrow><mi>c</mi><mo>+</mo><mn>1</mn></mrow></mfrac><mspace width="1.0em"></mspace><mo>⇒</mo><mspace width="1.0em"></mspace><mfrac><mrow><mi>∂</mi><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>=</mo><mo>−</mo><mfrac><mrow><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><msup><mrow><mrow><mo>(</mo><mrow><mi>c</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac><mo>,</mo></mtd></mtr><mtr><mtd columnalign="left"><mspace width="2.0em"></mspace><mspace width="1.0em"></mspace><mspace width="0.16em"></mspace><mspace width="0.16em"></mspace><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>k</mi><mi>c</mi><mspace width="1.0em"></mspace><mo>⇒</mo><mspace width="1.0em"></mspace><mfrac><mrow><mi>∂</mi><mi>μ</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>=</mo><mi>k</mi><mo>,</mo></mtd></mtr><mtr><mtd columnalign="left"><mspace width="2.0em"></mspace><mspace width="1.0em"></mspace><mspace width="0.16em"></mspace><mspace width="0.16em"></mspace><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>ν</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>m</mi><mi>c</mi><mspace width="0.16em"></mspace><mspace width="0.16em"></mspace><mspace width="0.16em"></mspace><mo>⇒</mo><mspace width="1.0em"></mspace><mfrac><mrow><mi>∂</mi><mi>ν</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>=</mo><mo>−</mo><mi>m</mi><mo>.</mo></mtd></mtr></mtable></math></td></tr></table>If we set<table class="disp-formula p" id="Equw"><tr><td class="formula"><math id="d33e7312" display="block"><mtable><mtr><mtd columnalign="right"><mi>a</mi><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><mi>γ</mi></mrow><mo>)</mo></mrow></mtd><mtd columnalign="center"><mo>:</mo><mo>=</mo></mtd><mtd columnalign="left"><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>γ</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mn>1</mn><mo>−</mo><mi>γ</mi></mrow><mo>)</mo></mrow><mo>,</mo></mtd></mtr><mtr><mtd columnalign="right"><mi>b</mi><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><mi>γ</mi></mrow><mo>)</mo></mrow></mtd><mtd columnalign="center"><mo>:</mo><mo>=</mo></mtd><mtd columnalign="left"><mo>−</mo><mi>γ</mi><mi>k</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>γ</mi><mi>m</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mn>1</mn></mrow><mo>)</mo></mrow><mo>,</mo></mtd></mtr><mtr><mtd columnalign="right"></mtd></mtr></mtable></math></td></tr></table>we then have<table class="disp-formula p" id="Equx"><tr><td class="formula"><math id="d33e7412" display="block"><mrow><mfrac><mrow><mi>∂</mi><mi class="MJX-tex-caligraphic" mathvariant="script">H</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>=</mo><mfrac><mrow><mi>a</mi></mrow><mrow><msup><mrow><mfenced close=")" open="("><mrow><mi>c</mi><mo>+</mo><mn>1</mn></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac><mo>+</mo><mi>b</mi><mo>,</mo></mrow></math></td></tr></table>and we obtain the second-degree polynomial equation<table class="disp-formula p" id="Equy"><tr><td class="formula"><math id="d33e7446" display="block"><mrow><mi>b</mi><msup><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msup><mo>+</mo><mn>2</mn><mi>b</mi><mi>c</mi><mo>+</mo><mrow><mo>(</mo><mrow><mi>a</mi><mo>+</mo><mi>b</mi></mrow><mo>)</mo></mrow><mo>=</mo><mn>0</mn><mo>.</mo></mrow></math></td></tr></table>
</div></li>
<li><p id="Par50">Steps (3)–(5) are repeated until convergence.</p></li>
</ol>
<p id="Par51">Figure <a href="#Fig2" class="usa-link">2</a> shows optimal dosing strategies under linear induction of tolerance computed using our implementation of the forward-backward sweep method. These results are consistent with computations obtained using the Dymos optimal control library in Python together with the IPOPT solver, as shown in Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Rapid pulsed strategies</h3>
<p id="Par52">Pulsed schedules are simple treatment strategies which alternate between giving a fixed drug dose and no drug. We identify a pulsed schedule as a three-dimensional vector (<em>c</em>, <em>t</em><sub>cycle</sub>, <em>φ</em>) where <em>c</em> is the dose applied, <em>t</em><sub>cycle</sub> is the duration of each treatment cycle, and <em>φ</em> is the proportion of drug application during each cycle. We will be particularly interested in idealized strategies where the treatment cycles are arbitrarily short. We note that if the cycle time <em>t</em><sub>cycle</sub> = <em>d</em><em>t</em> is infinitesimal, then using (<a href="#Equ10" class="usa-link">10</a>), the infinitesimal change in <em>f</em><sub>0</sub> is</p>
<table class="disp-formula p" id="Equz"><tr><td class="formula"><math id="d33e7530" display="block"><mtable><mtr><mtd columnalign="right"><mi>d</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mtd><mtd columnalign="center"><mo>=</mo></mtd><mtd columnalign="left"><mi>φ</mi><mi>d</mi><mi>t</mi><mfenced close=")" open="("><mrow><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfenced><mspace width="0.25em"></mspace><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfenced><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></mfenced></mtd></mtr><mtr><mtd columnalign="right"></mtd><mtd columnalign="center"></mtd><mtd columnalign="left"><mo>+</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mi>d</mi><mi>t</mi><mfenced close=")" open="("><mrow><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow></mfenced><mspace width="0.25em"></mspace><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>ν</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow></mfenced><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>ν</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow></mfenced><mo>.</mo></mtd></mtr></mtable></math></td></tr></table>
<p>If we define the average rates over each cycle,</p>
<table class="disp-formula p" id="Equ22"><tr>
<td class="formula"><math id="d33e7703" display="block"><mtable><mtr><mtd columnalign="right"><msub><mrow><mover accent="true"><mrow><mi>λ</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mtd><mtd columnalign="center"><mo>:</mo><mo>=</mo></mtd><mtd columnalign="left"><mi>φ</mi><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>φ</mi></mrow><mo>)</mo></mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>φ</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>,</mo></mtd></mtr><mtr><mtd columnalign="right"><mover accent="true"><mrow><mi>μ</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mtd><mtd columnalign="center"><mo>:</mo><mo>=</mo></mtd><mtd columnalign="left"><mi>φ</mi><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>φ</mi></mrow><mo>)</mo></mrow><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>φ</mi><mrow><mo>(</mo><mrow><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>μ</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>,</mo></mtd></mtr><mtr><mtd columnalign="right"><mover accent="true"><mrow><mi>ν</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mtd><mtd columnalign="center"><mo>:</mo><mo>=</mo></mtd><mtd columnalign="left"><mi>φ</mi><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>+</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>φ</mi></mrow><mo>)</mo></mrow><msub><mrow><mi>ν</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>=</mo><msub><mrow><mi>ν</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi>φ</mi><mrow><mo>(</mo><mrow><mi>ν</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>ν</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>,</mo></mtd></mtr></mtable></math></td>
<td class="label">22</td>
</tr></table>
<p>then <em>f</em><sub>0</sub>(<em>t</em>) obeys the differential equation</p>
<table class="disp-formula p" id="Equ23"><tr>
<td class="formula"><math id="d33e7922" display="block"><mtable><mtr><mtd columnalign="left"><mfrac><mrow><mi>d</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mover accent="true"><mrow><mi>λ</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow></mfenced><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mo>−</mo><mfenced close=")" open="("><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><msub><mrow><mover accent="true"><mrow><mi>λ</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mo>+</mo><mover accent="true"><mrow><mi>μ</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mo>+</mo><mover accent="true"><mrow><mi>ν</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow></mfenced></mtd></mtr><mtr><mtd columnalign="left"><mspace width="2.0em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mover accent="true"><mrow><mi>ν</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mo>.</mo></mtd></mtr></mtable></math></td>
<td class="label">23</td>
</tr></table>
<p>Similarly, the instantaneous growth rate of the population is given by</p>
<table class="disp-formula p" id="Equaa"><tr><td class="formula"><math id="d33e8044" display="block"><mrow><mi>ū</mi><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi><mo>,</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>=</mo><mrow><mo>(</mo><mrow><msub><mrow><mover accent="true"><mrow><mi>λ</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow><mo>)</mo></mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>.</mo></mrow></math></td></tr></table>
<p>Thus, under an arbitrarily fast pulsed schedule, the associated model dynamics can be approximated by a constant-dose model with parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><math id="d33e8106"><mrow><msub><mrow><mover accent="true"><mrow><mi>λ</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow></math></span>, <em>λ</em><sub>1</sub>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><math id="d33e8134"><mrow><mover accent="true"><mrow><mi>μ</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><math id="d33e8155"><mrow><mover accent="true"><mrow><mi>ν</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow></math></span>. By the section on behavior under a constant dose above, in the long run, the population grows at exponential rate</p>
<table class="disp-formula p" id="Equ24"><tr>
<td class="formula"><math id="d33e8176" display="block"><mtable><mtr><mtd columnalign="left"><mi>σ</mi><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mspace width="0.16em"></mspace><mo>:</mo><mo>=</mo><mspace width="0.25em"></mspace><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><mstyle mathsize="2.03em"><mfenced open="("><mrow></mrow></mfenced></mstyle><mrow><mo>(</mo><mrow><msub><mrow><mover accent="true"><mrow><mi>λ</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mo>−</mo><mover accent="true"><mrow><mi>μ</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>+</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><mover accent="true"><mrow><mi>ν</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mtd></mtr><mtr><mtd columnalign="left"><mspace width="2.0em"></mspace><mspace width="2.0em"></mspace><mspace width="1.0em"></mspace><mo>+</mo><mfenced close=")"><mrow><msqrt><mrow><msup><mrow><mfenced close=")" open="("><mrow><mrow><mo>(</mo><mrow><msub><mrow><mover accent="true"><mrow><mi>λ</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mo>−</mo><mover accent="true"><mrow><mi>μ</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>−</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>−</mo><mover accent="true"><mrow><mi>ν</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup><mo>+</mo><mn>4</mn><mover accent="true"><mrow><mi>μ</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mover accent="true"><mrow><mi>ν</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow></msqrt></mrow></mfenced><mo>.</mo></mtd></mtr></mtable></math></td>
<td class="label">24</td>
</tr></table>
<p>In addition, the population eventually reaches an equilibrium composition, where the proportion of sensitive cells becomes</p>
<table class="disp-formula p" id="Equ25"><tr>
<td class="formula"><math id="d33e8355" display="block"><mrow><msub><mrow><mover accent="true"><mrow><mi>f</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mo>:</mo><mo>=</mo><mfrac><mrow><mover accent="true"><mrow><mi>ν</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow><mrow><mi>σ</mi><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mo>−</mo><msub><mrow><mover accent="true"><mrow><mi>λ</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mo>+</mo><mover accent="true"><mrow><mi>μ</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow><mo>+</mo><mover accent="true"><mrow><mi>ν</mi></mrow><mo>¯</mo></mover><mrow><mo>(</mo><mrow><mi>c</mi><mo>,</mo><mi>φ</mi></mrow><mo>)</mo></mrow></mrow></mfrac><mo>.</mo></mrow></math></td>
<td class="label">25</td>
</tr></table></section><section id="Sec18"><h3 class="pmc_sec_title">Baseline parameters</h3>
<p id="Par53">Our computational results are shown using a parametrization of the model inspired by experimental investigations of drug tolerance in cancer and bacteria. The parameter values are shown in Table <a href="#Tab1" class="usa-link">1</a>. First, we assume that in the absence of drug, the proliferation rate of sensitive cells is <em>λ</em><sub>0</sub> = 0.04, and that the maximal drug effect on the death rate is Δ<em>d</em><sub>0</sub> = 0.08. Our motivating example is PC9 cells treated with a large dose of erlotinib, for which the proliferation rate of sensitive cells in the absence of drug has been estimated as <em>λ</em><sub>0</sub> = 0.04 per h and the drug effect as Δ<em>d</em><sub>0</sub> = 0.08 per h<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. We note that for a cohort of patients with metastatic melanoma treated with vemurafenib, the typical value of <em>λ</em><sub>0</sub> was 0.01 per day and the typical drug effect was Δ<em>d</em><sub>0</sub> = 0.04 per day. Thus, the time unit for the baseline regime can be considered to be hours in the in vitro setting and days in the clinical setting.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Baseline parameter values used in the study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Parameter</th>
<th colspan="1" rowspan="1">
<em>λ</em><sub>0</sub>
</th>
<th colspan="1" rowspan="1">Δ<em>d</em><sub>0</sub>
</th>
<th colspan="1" rowspan="1">
<em>λ</em><sub>1</sub>
</th>
<th colspan="1" rowspan="1">
<em>μ</em><sub>0</sub>
</th>
<th colspan="1" rowspan="1">
<em>ν</em><sub>0</sub>
</th>
<th colspan="1" rowspan="1"><em>k</em></th>
<th colspan="1" rowspan="1"><em>m</em></th>
<th colspan="1" rowspan="1">Δ<em>μ</em>
</th>
<th colspan="1" rowspan="1">Δ<em>ν</em>
</th>
<th colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><math id="d33e8548"><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub></math></span></th>
</tr></thead>
<tbody><tr>
<td colspan="1" rowspan="1">Value</td>
<td colspan="1" rowspan="1">0.04</td>
<td colspan="1" rowspan="1">0.08</td>
<td colspan="1" rowspan="1">0.001</td>
<td colspan="1" rowspan="1">0.0004</td>
<td colspan="1" rowspan="1">0.004</td>
<td colspan="1" rowspan="1">0.0004</td>
<td colspan="1" rowspan="1">0.0004</td>
<td colspan="1" rowspan="1">0.004</td>
<td colspan="1" rowspan="1">0.003</td>
<td colspan="1" rowspan="1">10</td>
</tr></tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par54">Second, we assume that in the absence of drug, <em>λ</em><sub>1</sub> ≪ <em>λ</em><sub>0</sub>, meaning that the drug-tolerant cells proliferate much slower than the drug-sensitive cells, and that they only make up a small proportion of the tumor at the start of the treatment. This is consistent with experimental evidence showing that drug-tolerant cells are generally slow-cycling<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Third, we assume that <em>μ</em><sub>0</sub> ≪ <em>λ</em><sub>0</sub>, meaning that transitions from sensitivity to tolerance are rare compared to cell divisions. This is consistent with evidence that epigenetic modifications are commonly retained for 10 − 10<sup>5</sup> cell divisions<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>–<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Fourth, we assume that <em>ν</em><sub>0</sub> is significantly larger than <em>μ</em><sub>0</sub>, meaning that the tolerant state is lost faster than it is adopted. This is consistent with experimental evidence both from cancer<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> and bacteria<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>.</p>
<p id="Par55">In Figs. <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig3" class="usa-link">3</a>, we assume that <em>k</em> = <em>m</em> = 0.0004 and that the maximum allowable dose is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><math id="d33e8665"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>=</mo><mn>10</mn></mrow></math></span> as a proportion of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><math id="d33e8682"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose. In Fig. <a href="#Fig4" class="usa-link">4</a>, we assume that Δ<em>μ</em> = 0.004, Δ<em>ν</em> = 0.003 and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><math id="d33e8707"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>=</mo><mn>10</mn></mrow></math></span>. Unless otherwise noted, we assume that the proportion of sensitive cells at the start of treatment is the equilibrium proportion <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><math id="d33e8723"><mrow><msub><mrow><mover accent="true"><mrow><mi>f</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mn>0</mn></mrow><mo>)</mo></mrow></mrow></math></span> in the absence of drug.</p>
<p id="Par56">We note that with this parametrization, the net growth rate of tolerant cells taking phenotypic switching into account is <em>λ</em><sub>1</sub> − <em>ν</em><sub>0</sub> = −0.003 &lt; 0. Thus, in the presence of drug, the tolerant population decays over time, but much slower than the sensitive cells.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Implementation of the Russo et al. model</h3>
<section id="Sec20"><h4 class="pmc_sec_title">Model and parametrization</h4>
<p id="Par57">In Russo et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, the authors use their experimental data to parametrize a mathematical model involving two cell types, sensitive and tolerant. The drug effect on the sensitive cell death rate <em>d</em><sub>0</sub> is modeled using a function of the form</p>
<table class="disp-formula p" id="Equab"><tr><td class="formula"><math id="d33e8769" display="block"><mrow><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>max</mi></mrow></msub><mo>−</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>exp</mi><mrow><mo>(</mo><mrow><mo>−</mo><mi>a</mi><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>exp</mi><mrow><mo>(</mo><mrow><mo>−</mo><mi>a</mi><mi>c</mi></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>,</mo></mrow></math></td></tr></table>
<p>where <em>c</em> is the drug dose measured in <em>M</em>. Under this model, the dose which has half the maximal effect on the sensitive cell death rate is given by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><math id="d33e8861"><mrow><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup><mo>=</mo><mi>log</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow><mo>/</mo><mi>a</mi></mrow></math></span>. If we measure the drug dose as a proportion of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><math id="d33e8887"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose, we can rewrite the above function as</p>
<table class="disp-formula p" id="Equ26"><tr>
<td class="formula"><math id="d33e8904" display="block"><mrow><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>+</mo><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>exp</mi><mrow><mo>(</mo><mrow><mo>−</mo><mi>c</mi><mi>log</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>.</mo></mrow></math></td>
<td class="label">26</td>
</tr></table>
<p>We note that this version of <em>d</em><sub>0</sub>(<em>c</em>) is concave in <em>c</em>, same as the functional form we assume in the section on the drug effect on cell proliferation above.</p>
<p id="Par58">For the WiDr cell line, Russo et al. infer that the transition rate <em>μ</em>(<em>c</em>) from sensitivity to tolerance is given by the linear function</p>
<table class="disp-formula p" id="Equac"><tr><td class="formula"><math id="d33e8976" display="block"><mrow><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msup><mrow><mi>k</mi></mrow><mrow><mo>′</mo></mrow></msup><mi>c</mi><mo>,</mo></mrow></math></td></tr></table>
<p>where <em>c</em> is the drug dose measured in <em>M</em>. If we again measure the drug dose as a proportion of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><math id="d33e9006"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose and define <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><math id="d33e9022"><mrow><mi>k</mi><mo>:</mo><mo>=</mo><msup><mrow><mi>k</mi></mrow><mrow><mo>′</mo></mrow></msup><msub><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow></msub></mrow></math></span>, we can rewrite this function as</p>
<table class="disp-formula p" id="Equad"><tr><td class="formula"><math id="d33e9045" display="block"><mrow><mi>μ</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><mi>k</mi><mi>c</mi><mo>.</mo></mrow></math></td></tr></table>
<p>With this modeling setup, the authors in ref. <sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> use their experimental data to derive the parameter estimates <em>λ</em><sub>0</sub> = <em>b</em><sub>0</sub> − <em>d</em><sub>0</sub> = 0.048, <em>a</em> = 2.91495 × 10<sup>6</sup>, Δ<em>d</em><sub>0</sub> = 1.095, <em>λ</em><sub>1</sub> − <em>ν</em><sub>0</sub> = −0.073 and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><math id="d33e9099"><mrow><msup><mrow><mi>k</mi></mrow><mrow><mo>′</mo></mrow></msup><mo>=</mo><mn>136935</mn></mrow></math></span> for the WiDr B7 clone. For these estimates, the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><math id="d33e9115"><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup></math></span> dose is given by</p>
<table class="disp-formula p" id="Equae"><tr><td class="formula"><math id="d33e9132" display="block"><mrow><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup><mo>=</mo><mfrac><mrow><mi>log</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow></mrow><mrow><mn>2.91495</mn><mo>⋅</mo><mn>1</mn><msup><mrow><mn>0</mn></mrow><mrow><mn>6</mn></mrow></msup></mrow></mfrac><mo>=</mo><mn>2.3779</mn><mo>⋅</mo><mn>1</mn><msup><mrow><mn>0</mn></mrow><mrow><mo>−</mo><mn>7</mn></mrow></msup><mspace width="0.33em"></mspace><mi>M</mi><mo>,</mo></mrow></math></td></tr></table>
<p>and the rescaled slope <em>k</em> is given by</p>
<table class="disp-formula p" id="Equaf"><tr><td class="formula"><math id="d33e9183" display="block"><mrow><mi>k</mi><mo>=</mo><mn>136935</mn><mo>⋅</mo><msubsup><mrow><mi mathvariant="normal">EC</mi></mrow><mrow><mn>50</mn></mrow><mrow><mi>d</mi></mrow></msubsup><mo>=</mo><mn>0.03256</mn><mo>.</mo></mrow></math></td></tr></table></section><section id="Sec21"><h4 class="pmc_sec_title">Implementation of forward-backward sweep method</h4>
<p id="Par59">To solve the optimal control problem (<a href="#Equ16" class="usa-link">16</a>) using the drug effect function in (<a href="#Equ26" class="usa-link">26</a>), we can apply the forward–backward sweep method as already laid out with one modification. In the fifth step, we have to solve <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><math id="d33e9216"><mrow><mi>∂</mi><mi class="MJX-tex-caligraphic" mathvariant="script">H</mi><mo>/</mo><mi>∂</mi><mi>c</mi><mo>=</mo><mn>0</mn></mrow></math></span>. We again have that</p>
<table class="disp-formula p" id="Equag"><tr><td class="formula"><math id="d33e9233" display="block"><mrow><mfrac><mrow><mi>∂</mi><mi class="MJX-tex-caligraphic" mathvariant="script">H</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>=</mo><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>γ</mi><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mo>+</mo><mi>γ</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mfrac><mrow><mi>∂</mi><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>−</mo><mi>γ</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mfrac><mrow><mi>∂</mi><mi>μ</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>+</mo><mi>γ</mi><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mfrac><mrow><mi>∂</mi><mi>ν</mi></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>,</mo></mrow></math></td></tr></table>
<p>but we must modify our calculation of ∂<em>λ</em><sub>0</sub>/∂<em>c</em>. In this case, we have</p>
<table class="disp-formula p" id="Equah"><tr><td class="formula"><math id="d33e9332" display="block"><mrow><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>=</mo><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>exp</mi><mrow><mo>(</mo><mrow><mo>−</mo><mi>c</mi><mi>log</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>,</mo></mrow></math></td></tr></table>
<p>which implies that</p>
<table class="disp-formula p" id="Equai"><tr><td class="formula"><math id="d33e9385" display="block"><mrow><mfrac><mrow><mi>∂</mi><msub><mrow><mi>λ</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mrow><mi>∂</mi><mi>c</mi></mrow></mfrac><mo>=</mo><mo>−</mo><mi>log</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow><mrow><mo>(</mo><mrow><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mi>exp</mi><mrow><mo>(</mo><mrow><mo>−</mo><mi>c</mi><mi>log</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>,</mo></mrow></math></td></tr></table>
<p>and we obtain the equation</p>
<table class="disp-formula p" id="Equaj"><tr><td class="formula"><math id="d33e9438" display="block"><mrow><mo>−</mo><mi>log</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow><mrow><mo>(</mo><mrow><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mi>exp</mi><mrow><mo>(</mo><mrow><mo>−</mo><mi>c</mi><mi>log</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mi>γ</mi><msubsup><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow><mrow><mn>2</mn></mrow></msubsup><mo>+</mo><mi>γ</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>−</mo><mi>γ</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mi>k</mi><mo>=</mo><mn>0</mn><mo>.</mo></mrow></math></td></tr></table>
<p>If we now set</p>
<table class="disp-formula p" id="Equak"><tr><td class="formula"><math id="d33e9516" display="block"><mtable><mtr><mtd columnalign="left"><mi>a</mi><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><mi>γ</mi></mrow><mo>)</mo></mrow><mspace width="0.16em"></mspace><mo>:</mo><mo>=</mo><mspace width="0.25em"></mspace><mi>log</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow><mrow><mo>(</mo><mrow><mi mathvariant="normal">Δ</mi><msub><mrow><mi>d</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow><mo>)</mo></mrow><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mi>γ</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>−</mo><mn>1</mn><mo>−</mo><mi>γ</mi></mrow><mo>)</mo></mrow><mo>,</mo></mtd></mtr><mtr><mtd columnalign="left"><mi>b</mi><mrow><mo>(</mo><mrow><mspace width="0.25em"></mspace><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><mi>γ</mi></mrow><mo>)</mo></mrow><mspace width="0.16em"></mspace><mspace width="0.25em"></mspace><mo>:</mo><mo>=</mo><mspace width="0.25em"></mspace><mo>−</mo><mi>γ</mi><msub><mrow><mi>f</mi></mrow><mrow><mn>0</mn></mrow></msub><mi>k</mi><mo>,</mo></mtd></mtr></mtable></math></td></tr></table>
<p>we obtain</p>
<table class="disp-formula p" id="Equal"><tr><td class="formula"><math id="d33e9612" display="block"><mrow><mi>exp</mi><mrow><mo>(</mo><mrow><mo>−</mo><mi>c</mi><mi>log</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>=</mo><mo>−</mo><mfrac><mrow><mi>b</mi></mrow><mrow><mi>a</mi></mrow></mfrac><mo>,</mo></mrow></math></td></tr></table>
<p>which yields <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><math id="d33e9643"><mrow><mi>c</mi><mo>=</mo><mo>−</mo><mi>log</mi><mrow><mo>(</mo><mrow><mo>−</mo><mi>b</mi><mo>/</mo><mi>a</mi></mrow><mo>)</mo></mrow><mo>/</mo><mi>log</mi><mrow><mo>(</mo><mrow><mn>2</mn></mrow><mo>)</mo></mrow></mrow></math></span>. This is the solution we use in the fifth step.</p></section><section id="Sec22"><h4 class="pmc_sec_title">Natural transitions between phenotypes</h4>
<p id="Par60">In Russo et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, the authors assume no transitions from sensitivity to tolerance in the absence of drug, <em>μ</em><sub>0</sub> = 0. For the WiDr cell line (clone B7), their point estimate for the proportion of sensitive cells at the start of treatment is 0.992, which would indicate transitions between phenotypes before treatment. However, the proportion is indistinguishable from zero when statistical uncertainty is taken into account. The authors point out that transitions from tolerance to sensitivity may occur even in the presence of drug, but that these transitions would effectively contribute to the death rate of tolerant cells. Overall, it is uncertain to what extent cells transition between sensitivity and tolerance in the absence of drug.</p>
<p id="Par61">To be consistent with our modeling framework, we assume transitions between types do occur in the absence of drug, so that they lead to an equilibrium proportion <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><math id="d33e9685"><mrow><msub><mrow><mover accent="true"><mrow><mi>f</mi></mrow><mo>¯</mo></mover></mrow><mrow><mn>0</mn></mrow></msub><mrow><mo>(</mo><mrow><mn>0</mn></mrow><mo>)</mo></mrow><mo>=</mo><mn>0.992</mn></mrow></math></span> of sensitive cells in the absence of drug. This leads us to take <em>μ</em><sub>0</sub> = 0.001, <em>λ</em><sub>1</sub> = 0 and <em>ν</em><sub>0</sub> = 0.073, which is still consistent with Russo et al.’s finding that drug tolerance is primarily induced by the drug and that <em>λ</em><sub>1</sub> − <em>ν</em><sub>0</sub> = −0.073. A summary of the parameter values used for our analysis of the Russo et al. model is given in Table <a href="#Tab2" class="usa-link">2</a>.</p>
<section class="tw xbox font-sm" id="Tab2"><h5 class="obj_head">Table 2.</h5>
<div class="caption p"><p>Parameter values used in our analysis of the Russo et al. model</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Parameter</th>
<th colspan="1" rowspan="1">
<em>λ</em><sub>0</sub>
</th>
<th colspan="1" rowspan="1">Δ<em>d</em><sub>0</sub>
</th>
<th colspan="1" rowspan="1">
<em>λ</em><sub>1</sub>
</th>
<th colspan="1" rowspan="1">
<em>μ</em><sub>0</sub>
</th>
<th colspan="1" rowspan="1">
<em>ν</em><sub>0</sub>
</th>
<th colspan="1" rowspan="1"><em>k</em></th>
</tr></thead>
<tbody><tr>
<td colspan="1" rowspan="1">Value</td>
<td colspan="1" rowspan="1">0.048</td>
<td colspan="1" rowspan="1">1.095</td>
<td colspan="1" rowspan="1">0</td>
<td colspan="1" rowspan="1">0.001</td>
<td colspan="1" rowspan="1">0.073</td>
<td colspan="1" rowspan="1">0.03256</td>
</tr></tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section></section><section id="Sec23"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12375710/bin/41540_2025_571_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.1MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The work of E.B.G. was supported in part by NIH/NCI grant R01 CA241137. The work of J.F. was supported in part by NSF grants CMMI-2228034 and DMS-2052465.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: E.B.G. and J.F. Methodology: E.B.G., B.V.M. and J.F. Investigation: E.B.G. and B.V.M. Formal analysis: B.V.M. and E.B.G. Software: B.V.M. and E.B.G. Writing: E.B.G. (main and supplementary text, review and editing), B.V.M. (supplementary text, review and editing), J.F. (review and editing). Visualization: E.B.G. and B.V.M. Funding acquisition: J.F.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>No datasets were generated or analysed during the current study.</p></section><section id="notes3"><h2 class="pmc_sec_title">Code availability</h2>
<p>All codes necessary to reproduce the results of this study are available in the GitHub repository <a href="https://github.com/egunnars/optimal_dosing_drug-induced_plasticity" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/egunnars/optimal_dosing_drug-induced_plasticity</a>.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par62">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec24"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41540-025-00571-5.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Merlo, L. M., Pepper, J. W., Reid, B. J. &amp; Maley, C. C. Cancer as an evolutionary and ecological process. <em>Nat. Rev. Cancer</em><strong>6</strong>, 924 (2006).
</cite> [<a href="https://doi.org/10.1038/nrc2013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17109012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Merlo,%20L.%20M.,%20Pepper,%20J.%20W.,%20Reid,%20B.%20J.%20&amp;%20Maley,%20C.%20C.%20Cancer%20as%20an%20evolutionary%20and%20ecological%20process.%20Nat.%20Rev.%20Cancer6,%20924%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Greaves, M. Evolutionary determinants of cancer. <em>Cancer Discov.</em><strong>5</strong>, 806–820 (2015).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-15-0439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4539576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26193902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greaves,%20M.%20Evolutionary%20determinants%20of%20cancer.%20Cancer%20Discov.5,%20806%E2%80%93820%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Flavahan, W. A., Gaskell, E. &amp; Bernstein, B. E. Epigenetic plasticity and the hallmarks of cancer. <em>Science</em><strong>357</strong>, eaal2380 (2017).
</cite> [<a href="https://doi.org/10.1126/science.aal2380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5940341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28729483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flavahan,%20W.%20A.,%20Gaskell,%20E.%20&amp;%20Bernstein,%20B.%20E.%20Epigenetic%20plasticity%20and%20the%20hallmarks%20of%20cancer.%20Science357,%20eaal2380%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Jones, P. A. &amp; Baylin, S. B. The epigenomics of cancer. <em>Cell</em><strong>128</strong>, 683–692 (2007).
</cite> [<a href="https://doi.org/10.1016/j.cell.2007.01.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3894624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17320506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jones,%20P.%20A.%20&amp;%20Baylin,%20S.%20B.%20The%20epigenomics%20of%20cancer.%20Cell128,%20683%E2%80%93692%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Brock, A., Chang, H. &amp; Huang, S. Non-genetic heterogeneity—a mutation-independent driving force for the somatic evolution of tumours. <em>Nat. Rev. Genet.</em><strong>10</strong>, 336 (2009).
</cite> [<a href="https://doi.org/10.1038/nrg2556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19337290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brock,%20A.,%20Chang,%20H.%20&amp;%20Huang,%20S.%20Non-genetic%20heterogeneity%E2%80%94a%20mutation-independent%20driving%20force%20for%20the%20somatic%20evolution%20of%20tumours.%20Nat.%20Rev.%20Genet.10,%20336%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Brown, R. &amp; Strathdee, G. Epigenomics and epigenetic therapy of cancer. <em>Trends Mol. Med.</em><strong>8</strong>, S43–S48 (2002).
</cite> [<a href="https://doi.org/10.1016/s1471-4914(02)02314-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11927287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20R.%20&amp;%20Strathdee,%20G.%20Epigenomics%20and%20epigenetic%20therapy%20of%20cancer.%20Trends%20Mol.%20Med.8,%20S43%E2%80%93S48%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Sharma, S. V. et al. A chromatin-mediated reversible drug-tolerant state in cancer cell subpopulations. <em>Cell</em><strong>141</strong>, 69–80 (2010).
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.02.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2851638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20371346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma,%20S.%20V.%20et%20al.%20A%20chromatin-mediated%20reversible%20drug-tolerant%20state%20in%20cancer%20cell%20subpopulations.%20Cell141,%2069%E2%80%9380%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Goldman, A. et al. Temporally sequenced anticancer drugs overcome adaptive resistance by targeting a vulnerable chemotherapy-induced phenotypic transition. <em>Nat. Commun.</em><strong>6</strong>, 6139 (2015).
</cite> [<a href="https://doi.org/10.1038/ncomms7139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4339891/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25669750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goldman,%20A.%20et%20al.%20Temporally%20sequenced%20anticancer%20drugs%20overcome%20adaptive%20resistance%20by%20targeting%20a%20vulnerable%20chemotherapy-induced%20phenotypic%20transition.%20Nat.%20Commun.6,%206139%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Ramirez, M. et al. Diverse drug-resistance mechanisms can emerge from drug-tolerant cancer persister cells. <em>Nat. Commun.</em><strong>7</strong>, 10690 (2016).
</cite> [<a href="https://doi.org/10.1038/ncomms10690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4762880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26891683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ramirez,%20M.%20et%20al.%20Diverse%20drug-resistance%20mechanisms%20can%20emerge%20from%20drug-tolerant%20cancer%20persister%20cells.%20Nat.%20Commun.7,%2010690%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Hata, A. N. et al. Tumor cells can follow distinct evolutionary paths to become resistant to epidermal growth factor receptor inhibition. <em>Nat. Med.</em><strong>22</strong>, 262–269 (2016).
</cite> [<a href="https://doi.org/10.1038/nm.4040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4900892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26828195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hata,%20A.%20N.%20et%20al.%20Tumor%20cells%20can%20follow%20distinct%20evolutionary%20paths%20to%20become%20resistant%20to%20epidermal%20growth%20factor%20receptor%20inhibition.%20Nat.%20Med.22,%20262%E2%80%93269%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Shaffer, S. M. et al. Rare cell variability and drug-induced reprogramming as a mode of cancer drug resistance. <em>Nature</em><strong>546</strong>, 431 (2017).
</cite> [<a href="https://doi.org/10.1038/nature22794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5542814/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28607484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shaffer,%20S.%20M.%20et%20al.%20Rare%20cell%20variability%20and%20drug-induced%20reprogramming%20as%20a%20mode%20of%20cancer%20drug%20resistance.%20Nature546,%20431%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Su, Y. et al. Single-cell analysis resolves the cell state transition and signaling dynamics associated with melanoma drug-induced resistance. <em>Proc. Natl. Acad. Sci. USA</em><strong>114</strong>, 13679–13684 (2017).
</cite> [<a href="https://doi.org/10.1073/pnas.1712064115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5748184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29229836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su,%20Y.%20et%20al.%20Single-cell%20analysis%20resolves%20the%20cell%20state%20transition%20and%20signaling%20dynamics%20associated%20with%20melanoma%20drug-induced%20resistance.%20Proc.%20Natl.%20Acad.%20Sci.%20USA114,%2013679%E2%80%9313684%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Neftel, C. et al. An integrative model of cellular states, plasticity, and genetics for glioblastoma. <em>Cell</em><strong>178</strong>, 835–849 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.06.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6703186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Neftel,%20C.%20et%20al.%20An%20integrative%20model%20of%20cellular%20states,%20plasticity,%20and%20genetics%20for%20glioblastoma.%20Cell178,%20835%E2%80%93849%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Gunnarsson, E. B., De, S., Leder, K. &amp; Foo, J. Understanding the role of phenotypic switching in cancer drug resistance. <em>J. Theor. Biol.</em><strong>490</strong>, 110162 (2020).
</cite> [<a href="https://doi.org/10.1016/j.jtbi.2020.110162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7785289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31953135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gunnarsson,%20E.%20B.,%20De,%20S.,%20Leder,%20K.%20&amp;%20Foo,%20J.%20Understanding%20the%20role%20of%20phenotypic%20switching%20in%20cancer%20drug%20resistance.%20J.%20Theor.%20Biol.490,%20110162%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Pisco, A. O. et al. Non-darwinian dynamics in therapy-induced cancer drug resistance. <em>Nat. Commun.</em><strong>4</strong>, 2467 (2013).
</cite> [<a href="https://doi.org/10.1038/ncomms3467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4657953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24045430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pisco,%20A.%20O.%20et%20al.%20Non-darwinian%20dynamics%20in%20therapy-induced%20cancer%20drug%20resistance.%20Nat.%20Commun.4,%202467%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Vipparthi, K. et al. Emergence of hybrid states of stem-like cancer cells correlates with poor prognosis in oral cancer. <em>IScience</em><strong>25</strong>, 104317 (2022).</cite> [<a href="https://doi.org/10.1016/j.isci.2022.104317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9114525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35602941/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Russo, M. et al. A modified fluctuation-test framework characterizes the population dynamics and mutation rate of colorectal cancer persister cells. <em>Nat. Genet.</em><strong>54</strong>, 976–984 (2022).
</cite> [<a href="https://doi.org/10.1038/s41588-022-01105-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9279152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35817983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Russo,%20M.%20et%20al.%20A%20modified%20fluctuation-test%20framework%20characterizes%20the%20population%20dynamics%20and%20mutation%20rate%20of%20colorectal%20cancer%20persister%20cells.%20Nat.%20Genet.54,%20976%E2%80%93984%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Szikriszt, B. et al. A comprehensive survey of the mutagenic impact of common cancer cytotoxics. <em>Genome Biol.</em><strong>17</strong>, 1–16 (2016).
</cite> [<a href="https://doi.org/10.1186/s13059-016-0963-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4862131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27161042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szikriszt,%20B.%20et%20al.%20A%20comprehensive%20survey%20of%20the%20mutagenic%20impact%20of%20common%20cancer%20cytotoxics.%20Genome%20Biol.17,%201%E2%80%9316%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Venkatesan, S., Swanton, C., Taylor, B. S. &amp; Costello, J. F. Treatment-induced mutagenesis and selective pressures sculpt cancer evolution. <em>Cold Spring Harb. Perspect. Med.</em><strong>7</strong>, a026617 (2017).
</cite> [<a href="https://doi.org/10.1101/cshperspect.a026617" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5538404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28289245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Venkatesan,%20S.,%20Swanton,%20C.,%20Taylor,%20B.%20S.%20&amp;%20Costello,%20J.%20F.%20Treatment-induced%20mutagenesis%20and%20selective%20pressures%20sculpt%20cancer%20evolution.%20Cold%20Spring%20Harb.%20Perspect.%20Med.7,%20a026617%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Russo, M. et al. Adaptive mutability of colorectal cancers in response to targeted therapies. <em>Science</em><strong>366</strong>, 1473–1480 (2019).
</cite> [<a href="https://doi.org/10.1126/science.aav4474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31699882/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Russo,%20M.%20et%20al.%20Adaptive%20mutability%20of%20colorectal%20cancers%20in%20response%20to%20targeted%20therapies.%20Science366,%201473%E2%80%931480%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Cipponi, A. et al. Mtor signaling orchestrates stress-induced mutagenesis, facilitating adaptive evolution in cancer. <em>Science</em><strong>368</strong>, 1127–1131 (2020).
</cite> [<a href="https://doi.org/10.1126/science.aau8768" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32499442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cipponi,%20A.%20et%20al.%20Mtor%20signaling%20orchestrates%20stress-induced%20mutagenesis,%20facilitating%20adaptive%20evolution%20in%20cancer.%20Science368,%201127%E2%80%931131%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Kerbel, R. S. &amp; Kamen, B. A. The anti-angiogenic basis of metronomic chemotherapy. <em>Nat. Rev. Cancer</em><strong>4</strong>, 423–436 (2004).
</cite> [<a href="https://doi.org/10.1038/nrc1369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15170445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kerbel,%20R.%20S.%20&amp;%20Kamen,%20B.%20A.%20The%20anti-angiogenic%20basis%20of%20metronomic%20chemotherapy.%20Nat.%20Rev.%20Cancer4,%20423%E2%80%93436%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Norton, L. The Norton-Simon hypothesis revisited. <em>Cancer Treat. Rep.</em><strong>70</strong>, 163–169 (1986).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/3510732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Norton,%20L.%20The%20Norton-Simon%20hypothesis%20revisited.%20Cancer%20Treat.%20Rep.70,%20163%E2%80%93169%20(1986)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Brown, R., Curry, E., Magnani, L., Wilhelm-Benartzi, C. S. &amp; Borley, J. Poised epigenetic states and acquired drug resistance in cancer. <em>Nat. Rev. Cancer</em><strong>14</strong>, 747–753 (2014).
</cite> [<a href="https://doi.org/10.1038/nrc3819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25253389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20R.,%20Curry,%20E.,%20Magnani,%20L.,%20Wilhelm-Benartzi,%20C.%20S.%20&amp;%20Borley,%20J.%20Poised%20epigenetic%20states%20and%20acquired%20drug%20resistance%20in%20cancer.%20Nat.%20Rev.%20Cancer14,%20747%E2%80%93753%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Greene, J. M., Gevertz, J. L. &amp; Sontag, E. D. Mathematical approach to differentiate spontaneous and induced evolution to drug resistance during cancer treatment. <em>JCO Clin. Cancer Inform.</em><strong>3</strong>, 1–20 (2019).
</cite> [<a href="https://doi.org/10.1200/CCI.18.00087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6873992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30969799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greene,%20J.%20M.,%20Gevertz,%20J.%20L.%20&amp;%20Sontag,%20E.%20D.%20Mathematical%20approach%20to%20differentiate%20spontaneous%20and%20induced%20evolution%20to%20drug%20resistance%20during%20cancer%20treatment.%20JCO%20Clin.%20Cancer%20Inform.3,%201%E2%80%9320%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Akhmetzhanov, A. R. et al. Modelling bistable tumour population dynamics to design effective treatment strategies. <em>J. Theor. Biol.</em><strong>474</strong>, 88–102 (2019).
</cite> [<a href="https://doi.org/10.1016/j.jtbi.2019.05.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9534689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31077681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akhmetzhanov,%20A.%20R.%20et%20al.%20Modelling%20bistable%20tumour%20population%20dynamics%20to%20design%20effective%20treatment%20strategies.%20J.%20Theor.%20Biol.474,%2088%E2%80%93102%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Greene, J. M., Sanchez-Tapia, C. &amp; Sontag, E. D. Mathematical details on a cancer resistance model. <em>Front. Bioeng. Biotechnol.</em><strong>8</strong>, 501 (2020).
</cite> [<a href="https://doi.org/10.3389/fbioe.2020.00501" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7325889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32656186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greene,%20J.%20M.,%20Sanchez-Tapia,%20C.%20&amp;%20Sontag,%20E.%20D.%20Mathematical%20details%20on%20a%20cancer%20resistance%20model.%20Front.%20Bioeng.%20Biotechnol.8,%20501%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Kuosmanen, T. et al. Drug-induced resistance evolution necessitates less aggressive treatment. <em>PLoS Comput. Biol.</em><strong>17</strong>, e1009418 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1009418" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8491903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34555024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuosmanen,%20T.%20et%20al.%20Drug-induced%20resistance%20evolution%20necessitates%20less%20aggressive%20treatment.%20PLoS%20Comput.%20Biol.17,%20e1009418%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Angelini, E., Wang, Y., Zhou, J. X., Qian, H. &amp; Huang, S. A model for the intrinsic limit of cancer therapy: duality of treatment-induced cell death and treatment-induced stemness. <em>PLOS Comput. Biol.</em><strong>18</strong>, e1010319 (2022).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1010319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9352192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35877695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Angelini,%20E.,%20Wang,%20Y.,%20Zhou,%20J.%20X.,%20Qian,%20H.%20&amp;%20Huang,%20S.%20A%20model%20for%20the%20intrinsic%20limit%20of%20cancer%20therapy:%20duality%20of%20treatment-induced%20cell%20death%20and%20treatment-induced%20stemness.%20PLOS%20Comput.%20Biol.18,%20e1010319%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Corigliano, M., Di Bernardo, A., Cosentino Lagomarsino, M. &amp; Pompei, S. Optimal treatment for drug-induced cancer persisters involves release periods and intermediate drug doses. <em>PRX Life</em><strong>3</strong>, 023018 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Corigliano,%20M.,%20Di%20Bernardo,%20A.,%20Cosentino%20Lagomarsino,%20M.%20&amp;%20Pompei,%20S.%20Optimal%20treatment%20for%20drug-induced%20cancer%20persisters%20involves%20release%20periods%20and%20intermediate%20drug%20doses.%20PRX%20Life3,%20023018%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Esteller, M. Cpg island hypermethylation and tumor suppressor genes: a booming present, a brighter future. <em>Oncogene</em><strong>21</strong>, 5427–5440 (2002).
</cite> [<a href="https://doi.org/10.1038/sj.onc.1205600" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12154405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Esteller,%20M.%20Cpg%20island%20hypermethylation%20and%20tumor%20suppressor%20genes:%20a%20booming%20present,%20a%20brighter%20future.%20Oncogene21,%205427%E2%80%935440%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Romero-Garcia, S., Prado-Garcia, H. &amp; Carlos-Reyes, A. Role of DNA methylation in the resistance to therapy in solid tumors. <em>Front. Oncol.</em><strong>10</strong>, 1152 (2020).
</cite> [<a href="https://doi.org/10.3389/fonc.2020.01152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7426728/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32850327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Romero-Garcia,%20S.,%20Prado-Garcia,%20H.%20&amp;%20Carlos-Reyes,%20A.%20Role%20of%20DNA%20methylation%20in%20the%20resistance%20to%20therapy%20in%20solid%20tumors.%20Front.%20Oncol.10,%201152%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Okano, M., Bell, D. W., Haber, D. A. &amp; Li, E. Dna methyltransferases dnmt3a and dnmt3b are essential for de novo methylation and mammalian development. <em>Cell</em><strong>99</strong>, 247–257 (1999).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(00)81656-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10555141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Okano,%20M.,%20Bell,%20D.%20W.,%20Haber,%20D.%20A.%20&amp;%20Li,%20E.%20Dna%20methyltransferases%20dnmt3a%20and%20dnmt3b%20are%20essential%20for%20de%20novo%20methylation%20and%20mammalian%20development.%20Cell99,%20247%E2%80%93257%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Esteller, M. Epigenetics in cancer. <em>N. Engl. J. Med.</em><strong>358</strong>, 1148–1159 (2008).
</cite> [<a href="https://doi.org/10.1056/NEJMra072067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18337604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Esteller,%20M.%20Epigenetics%20in%20cancer.%20N.%20Engl.%20J.%20Med.358,%201148%E2%80%931159%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wensink, G. E. et al. Patient-derived organoids as a predictive biomarker for treatment response in cancer patients. <em>npj Precis. Oncol.</em><strong>5</strong>, 30 (2021).
</cite> [<a href="https://doi.org/10.1038/s41698-021-00168-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8042051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33846504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wensink,%20G.%20E.%20et%20al.%20Patient-derived%20organoids%20as%20a%20predictive%20biomarker%20for%20treatment%20response%20in%20cancer%20patients.%20npj%20Precis.%20Oncol.5,%2030%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Jiang, S. et al. An automated organoid platform with inter-organoid homogeneity and inter-patient heterogeneity. <em>Cell Rep. Med.</em><strong>1</strong>, 100161 (2020).</cite> [<a href="https://doi.org/10.1016/j.xcrm.2020.100161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7762778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33377132/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Tebon, P. J. et al. Drug screening at single-organoid resolution via bioprinting and interferometry. <em>Nat. Commun.</em><strong>14</strong>, 3168 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-38832-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10244450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37280220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tebon,%20P.%20J.%20et%20al.%20Drug%20screening%20at%20single-organoid%20resolution%20via%20bioprinting%20and%20interferometry.%20Nat.%20Commun.14,%203168%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Walsh, A. J. et al. Quantitative optical imaging of primary tumor organoid metabolism predicts drug response in breast cancer. <em>Cancer Res.</em><strong>74</strong>, 5184–5194 (2014).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-14-0663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4167558/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25100563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Walsh,%20A.%20J.%20et%20al.%20Quantitative%20optical%20imaging%20of%20primary%20tumor%20organoid%20metabolism%20predicts%20drug%20response%20in%20breast%20cancer.%20Cancer%20Res.74,%205184%E2%80%935194%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Gunnarsson, E. B. et al. Understanding patient-derived tumor organoid growth through an integrated imaging and mathematical modeling framework. <em>PLoS Comput. Biol.</em><strong>20</strong>, e1012256 (2024).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1012256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11324155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39093897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gunnarsson,%20E.%20B.%20et%20al.%20Understanding%20patient-derived%20tumor%20organoid%20growth%20through%20an%20integrated%20imaging%20and%20mathematical%20modeling%20framework.%20PLoS%20Comput.%20Biol.20,%20e1012256%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>FDA Modernization Act 2.0. <a href="https://www.congress.gov/bill/117th-congress/senate-bill/5002/text" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.congress.gov/bill/117th-congress/senate-bill/5002/text</a> (2022).</cite>
</li>
<li id="CR41">
<span class="label">41.</span><cite>Zushin, P.-J. H. et al. FDA modernization act 2.0: transitioning beyond animal models with human cells, organoids, and AI/ML-based approaches. <em>J. Clin. Investig.</em><strong>133</strong>, 21(2023).</cite> [<a href="https://doi.org/10.1172/JCI175824" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10617761/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37909337/" class="usa-link">PubMed</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Ahmed, S. M., Shivnaraine, R. V. &amp; Wu, J. C. FDA modernization act 2.0 paves the way to computational biology and clinical trials in a dish. <em>Circulation</em><strong>148</strong>, 309–311 (2023).
</cite> [<a href="https://doi.org/10.1161/CIRCULATIONAHA.123.065585" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10902210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37486997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahmed,%20S.%20M.,%20Shivnaraine,%20R.%20V.%20&amp;%20Wu,%20J.%20C.%20FDA%20modernization%20act%202.0%20paves%20the%20way%20to%20computational%20biology%20and%20clinical%20trials%20in%20a%20dish.%20Circulation148,%20309%E2%80%93311%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Goutelle, S. et al. The Hill equation: a review of its capabilities in pharmacological modelling. <em>Fundam. Clin. Pharmacol.</em><strong>22</strong>, 633–648 (2008).
</cite> [<a href="https://doi.org/10.1111/j.1472-8206.2008.00633.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19049668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goutelle,%20S.%20et%20al.%20The%20Hill%20equation:%20a%20review%20of%20its%20capabilities%20in%20pharmacological%20modelling.%20Fundam.%20Clin.%20Pharmacol.22,%20633%E2%80%93648%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Chisholm, R. H. et al. Emergence of drug tolerance in cancer cell populations: an evolutionary outcome of selection, nongenetic instability, and stress-induced adaptation. <em>Cancer Res.</em><strong>75</strong>, 930–939 (2015).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-14-2103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25627977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chisholm,%20R.%20H.%20et%20al.%20Emergence%20of%20drug%20tolerance%20in%20cancer%20cell%20populations:%20an%20evolutionary%20outcome%20of%20selection,%20nongenetic%20instability,%20and%20stress-induced%20adaptation.%20Cancer%20Res.75,%20930%E2%80%93939%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Chisholm, R. H., Lorenzi, T. &amp; Clairambault, J. Cell population heterogeneity and evolution towards drug resistance in cancer: biological and mathematical assessment, theoretical treatment optimisation. <em>Biochim. Biophys. Acta (BBA)-Gen. Subj.</em><strong>1860</strong>, 2627–2645 (2016).</cite> [<a href="https://doi.org/10.1016/j.bbagen.2016.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27339473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chisholm,%20R.%20H.,%20Lorenzi,%20T.%20&amp;%20Clairambault,%20J.%20Cell%20population%20heterogeneity%20and%20evolution%20towards%20drug%20resistance%20in%20cancer:%20biological%20and%20mathematical%20assessment,%20theoretical%20treatment%20optimisation.%20Biochim.%20Biophys.%20Acta%20(BBA)-Gen.%20Subj.1860,%202627%E2%80%932645%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Pouchol, C., Clairambault, J., Lorz, A. &amp; Trélat, E. Asymptotic analysis and optimal control of an integro-differential system modelling healthy and cancer cells exposed to chemotherapy. <em>J. Math. Pures Appl.</em><strong>116</strong>, 268–308 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Pouchol,%20C.,%20Clairambault,%20J.,%20Lorz,%20A.%20&amp;%20Tr%C3%A9lat,%20E.%20Asymptotic%20analysis%20and%20optimal%20control%20of%20an%20integro-differential%20system%20modelling%20healthy%20and%20cancer%20cells%20exposed%20to%20chemotherapy.%20J.%20Math.%20Pures%20Appl.116,%20268%E2%80%93308%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Almeida, L., Bagnerini, P., Fabrini, G., Hughes, B. D. &amp; Lorenzi, T. Evolution of cancer cell populations under cytotoxic therapy and treatment optimisation: insight from a phenotype-structured model. <em>ESAIM Math. Model. Numer. Anal.</em><strong>53</strong>, 1157–1190 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Almeida,%20L.,%20Bagnerini,%20P.,%20Fabrini,%20G.,%20Hughes,%20B.%20D.%20&amp;%20Lorenzi,%20T.%20Evolution%20of%20cancer%20cell%20populations%20under%20cytotoxic%20therapy%20and%20treatment%20optimisation:%20insight%20from%20a%20phenotype-structured%20model.%20ESAIM%20Math.%20Model.%20Numer.%20Anal.53,%201157%E2%80%931190%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Jarrett, A. M. et al. Optimal control theory for personalized therapeutic regimens in oncology: Background, history, challenges, and opportunities. <em>J. Clin. Med.</em><strong>9</strong>, 1314 (2020).
</cite> [<a href="https://doi.org/10.3390/jcm9051314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7290915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32370195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jarrett,%20A.%20M.%20et%20al.%20Optimal%20control%20theory%20for%20personalized%20therapeutic%20regimens%20in%20oncology:%20Background,%20history,%20challenges,%20and%20opportunities.%20J.%20Clin.%20Med.9,%201314%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Gatenby, R. A. A change of strategy in the war on cancer. <em>Nature</em><strong>459</strong>, 508–509 (2009).
</cite> [<a href="https://doi.org/10.1038/459508a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19478766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gatenby,%20R.%20A.%20A%20change%20of%20strategy%20in%20the%20war%20on%20cancer.%20Nature459,%20508%E2%80%93509%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Wu, C., Gunnarsson, E. B., Foo, J. &amp; Leder, K. A statistical framework for detecting therapy-induced resistance from drug screens. <em>npj Syst. Biol. Appl.</em><strong>11</strong>, 88 (2025).
</cite> [<a href="https://doi.org/10.1038/s41540-025-00560-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12328638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40769982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20C.,%20Gunnarsson,%20E.%20B.,%20Foo,%20J.%20&amp;%20Leder,%20K.%20A%20statistical%20framework%20for%20detecting%20therapy-induced%20resistance%20from%20drug%20screens.%20npj%20Syst.%20Biol.%20Appl.11,%2088%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Gunnarsson, E. B., Foo, J. &amp; Leder, K. Statistical inference of the rates of cell proliferation and phenotypic switching in cancer. <em>J. Theor. Biol.</em><strong>568</strong>, 111497 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jtbi.2023.111497" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10372878/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37087049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gunnarsson,%20E.%20B.,%20Foo,%20J.%20&amp;%20Leder,%20K.%20Statistical%20inference%20of%20the%20rates%20of%20cell%20proliferation%20and%20phenotypic%20switching%20in%20cancer.%20J.%20Theor.%20Biol.568,%20111497%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>McGuire, M. F. et al. Formalizing an integrative, multidisciplinary cancer therapy discovery workflow. <em>Cancer Res.</em><strong>73</strong>, 6111–6117 (2013).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-13-0310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4040396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23955390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McGuire,%20M.%20F.%20et%20al.%20Formalizing%20an%20integrative,%20multidisciplinary%20cancer%20therapy%20discovery%20workflow.%20Cancer%20Res.73,%206111%E2%80%936117%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Hamis, S., Powathil, G. G. &amp; Chaplain, M. A. Blackboard to bedside: a mathematical modeling bottom-up approach toward personalized cancer treatments. <em>JCO Clin. Cancer Inform.</em><strong>3</strong>, 1–11 (2019).
</cite> [<a href="https://doi.org/10.1200/CCI.18.00068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30742485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hamis,%20S.,%20Powathil,%20G.%20G.%20&amp;%20Chaplain,%20M.%20A.%20Blackboard%20to%20bedside:%20a%20mathematical%20modeling%20bottom-up%20approach%20toward%20personalized%20cancer%20treatments.%20JCO%20Clin.%20Cancer%20Inform.3,%201%E2%80%9311%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Athreya, K. B. &amp; Ney, P. E. <em>Branching processes</em> (Courier Corporation, 2004).</cite>
</li>
<li id="CR55">
<span class="label">55.</span><cite>McAsey, M., Mou, L. &amp; Han, W. Convergence of the forward-backward sweep method in optimal control. <em>Comput. Optim. Appl.</em><strong>53</strong>, 207–226 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?McAsey,%20M.,%20Mou,%20L.%20&amp;%20Han,%20W.%20Convergence%20of%20the%20forward-backward%20sweep%20method%20in%20optimal%20control.%20Comput.%20Optim.%20Appl.53,%20207%E2%80%93226%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Sharp, J. A.<em>Numerical Methods for Optimal Control and Parameter Estimation in the Life Sciences</em>. Ph.D. thesis, Queensland University of Technology (2022).</cite>
</li>
<li id="CR57">
<span class="label">57.</span><cite>Roesch, A. et al. A temporarily distinct subpopulation of slow-cycling melanoma cells is required for continuous tumor growth. <em>Cell</em><strong>141</strong>, 583–594 (2010).
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.04.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2882693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20478252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roesch,%20A.%20et%20al.%20A%20temporarily%20distinct%20subpopulation%20of%20slow-cycling%20melanoma%20cells%20is%20required%20for%20continuous%20tumor%20growth.%20Cell141,%20583%E2%80%93594%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Roesch, A. et al. Overcoming intrinsic multidrug resistance in melanoma by blocking the mitochondrial respiratory chain of slow-cycling jarid1bhigh cells. <em>Cancer Cell</em><strong>23</strong>, 811–825 (2013).
</cite> [<a href="https://doi.org/10.1016/j.ccr.2013.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3810180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23764003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roesch,%20A.%20et%20al.%20Overcoming%20intrinsic%20multidrug%20resistance%20in%20melanoma%20by%20blocking%20the%20mitochondrial%20respiratory%20chain%20of%20slow-cycling%20jarid1bhigh%20cells.%20Cancer%20Cell23,%20811%E2%80%93825%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Niepel, M., Spencer, S. L. &amp; Sorger, P. K. Non-genetic cell-to-cell variability and the consequences for pharmacology. <em>Curr. Opin. Chem. Biol.</em><strong>13</strong>, 556–561 (2009).
</cite> [<a href="https://doi.org/10.1016/j.cbpa.2009.09.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2975492/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19833543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Niepel,%20M.,%20Spencer,%20S.%20L.%20&amp;%20Sorger,%20P.%20K.%20Non-genetic%20cell-to-cell%20variability%20and%20the%20consequences%20for%20pharmacology.%20Curr.%20Opin.%20Chem.%20Biol.13,%20556%E2%80%93561%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Sigal, A. et al. Variability and memory of protein levels in human cells. <em>Nature</em><strong>444</strong>, 643 (2006).
</cite> [<a href="https://doi.org/10.1038/nature05316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17122776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sigal,%20A.%20et%20al.%20Variability%20and%20memory%20of%20protein%20levels%20in%20human%20cells.%20Nature444,%20643%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Cohen, A. A. et al. Dynamic proteomics of individual cancer cells in response to a drug. <em>Science</em><strong>322</strong>, 1511–1516 (2008).
</cite> [<a href="https://doi.org/10.1126/science.1160165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19023046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cohen,%20A.%20A.%20et%20al.%20Dynamic%20proteomics%20of%20individual%20cancer%20cells%20in%20response%20to%20a%20drug.%20Science322,%201511%E2%80%931516%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Gupta, P. B. et al. Stochastic state transitions give rise to phenotypic equilibrium in populations of cancer cells. <em>Cell</em><strong>146</strong>, 633–644 (2011).
</cite> [<a href="https://doi.org/10.1016/j.cell.2011.07.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21854987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gupta,%20P.%20B.%20et%20al.%20Stochastic%20state%20transitions%20give%20rise%20to%20phenotypic%20equilibrium%20in%20populations%20of%20cancer%20cells.%20Cell146,%20633%E2%80%93644%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Paryad-Zanjani, S., Saint-Antoine, M. M. &amp; Singh, A. Optimal scheduling of therapy to delay cancer drug resistance. <em>IFAC-PapersOnLine</em><strong>54</strong>, 239–244 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Paryad-Zanjani,%20S.,%20Saint-Antoine,%20M.%20M.%20&amp;%20Singh,%20A.%20Optimal%20scheduling%20of%20therapy%20to%20delay%20cancer%20drug%20resistance.%20IFAC-PapersOnLine54,%20239%E2%80%93244%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Norman, T. M., Lord, N. D., Paulsson, J. &amp; Losick, R. Stochastic switching of cell fate in microbes. <em>Annu. Rev. Microbiol.</em><strong>69</strong>, 381–403 (2015).
</cite> [<a href="https://doi.org/10.1146/annurev-micro-091213-112852" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26332088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Norman,%20T.%20M.,%20Lord,%20N.%20D.,%20Paulsson,%20J.%20&amp;%20Losick,%20R.%20Stochastic%20switching%20of%20cell%20fate%20in%20microbes.%20Annu.%20Rev.%20Microbiol.69,%20381%E2%80%93403%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12375710/bin/41540_2025_571_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (1.1MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>No datasets were generated or analysed during the current study.</p>
<p>All codes necessary to reproduce the results of this study are available in the GitHub repository <a href="https://github.com/egunnars/optimal_dosing_drug-induced_plasticity" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/egunnars/optimal_dosing_drug-induced_plasticity</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from NPJ Systems Biology and Applications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41540-025-00571-5"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41540_2025_Article_571.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375710/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375710/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375710%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375710/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375710/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375710/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40850942/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375710/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40850942/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375710/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375710/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="PGHWfZfMw3ExvlPWBYydCKrurF4MOrkqqbpHtaEQyXlDAHAWpAMprflRBjPBAOOG">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
