
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Glycosylated N‑Acyl Phosphoethanolamines as Bacterial Food-Dependent Signaling Molecules in Caenorhabditis Nematodes - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A2668AF1E83305A2660015030DCE.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsbmcau">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371488/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Bio &amp; Med Chem Au">
<meta name="citation_title" content="Glycosylated N‑Acyl Phosphoethanolamines as Bacterial Food-Dependent Signaling Molecules in Caenorhabditis Nematodes">
<meta name="citation_author" content="Siva Bandi">
<meta name="citation_author_institution" content="Laboratory for Bioanalytical Chemistry, University of Neuchâtel, Avenue de Bellevaux 51, CH-2000 Neuchâtel, Switzerland">
<meta name="citation_author" content="Marie-Désirée Schlemper-Scheidt">
<meta name="citation_author_institution" content="Laboratory for Bioanalytical Chemistry, University of Neuchâtel, Avenue de Bellevaux 51, CH-2000 Neuchâtel, Switzerland">
<meta name="citation_author" content="Rocío Rivera Sánchez">
<meta name="citation_author_institution" content="Laboratory for Bioanalytical Chemistry, University of Neuchâtel, Avenue de Bellevaux 51, CH-2000 Neuchâtel, Switzerland">
<meta name="citation_author" content="Sylvain Sutour">
<meta name="citation_author_institution" content="Neuchatel Platform for Analytical Chemistry (NPAC), University of Neuchâtel, Avenue de Bellevaux 51, CH-2000 Neuchâtel, Switzerland">
<meta name="citation_author" content="Gaétan Glauser">
<meta name="citation_author_institution" content="Neuchatel Platform for Analytical Chemistry (NPAC), University of Neuchâtel, Avenue de Bellevaux 51, CH-2000 Neuchâtel, Switzerland">
<meta name="citation_author" content="Yojiro Ishida">
<meta name="citation_author_institution" content="Center for Advanced Biotechnology and Medicine (CABM), Rutgers University, 679 Hoes Lane West, Piscataway, New Jersey 08854, United States">
<meta name="citation_author" content="Stephan H von Reuß">
<meta name="citation_author_institution" content="Laboratory for Bioanalytical Chemistry, University of Neuchâtel, Avenue de Bellevaux 51, CH-2000 Neuchâtel, Switzerland">
<meta name="citation_author_institution" content="Neuchatel Platform for Analytical Chemistry (NPAC), University of Neuchâtel, Avenue de Bellevaux 51, CH-2000 Neuchâtel, Switzerland">
<meta name="citation_publication_date" content="2025 Jun 30">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="602">
<meta name="citation_doi" content="10.1021/acsbiomedchemau.5c00012">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371488/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371488/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371488/pdf/bg5c00012.pdf">
<meta name="description" content="N-acyl ethanolamines represent conserved lipophilic signaling molecules that function as endogenous ligands at G-protein-coupled receptors, ion channels, and nuclear receptors. Using a combination of comparative ultrahigh-performance liquid ...">
<meta name="og:title" content="Glycosylated N‑Acyl Phosphoethanolamines as Bacterial Food-Dependent Signaling Molecules in Caenorhabditis Nematodes">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="N-acyl ethanolamines represent conserved lipophilic signaling molecules that function as endogenous ligands at G-protein-coupled receptors, ion channels, and nuclear receptors. Using a combination of comparative ultrahigh-performance liquid ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371488/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371488">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsbiomedchemau.5c00012"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/bg5c00012.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371488%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371488/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371488/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371488/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsbmcau.png" alt="ACS Bio &amp; Med Chem Au logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Bio &amp; Med Chem Au" title="Link to ACS Bio &amp; Med Chem Au" shape="default" href="https://pubs.acs.org/journal/abmcb8" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Bio Med Chem Au</button></div>. 2025 Jun 30;5(4):602–619. doi: <a href="https://doi.org/10.1021/acsbiomedchemau.5c00012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsbiomedchemau.5c00012</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Glycosylated <em>N</em>‑Acyl Phosphoethanolamines
as Bacterial Food-Dependent Signaling Molecules in <em>Caenorhabditis</em> Nematodes</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bandi%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Siva Bandi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Siva Bandi</span></h3>
<div class="p">
<sup>†</sup>
Laboratory
for Bioanalytical Chemistry, University
of Neuchâtel, Avenue de Bellevaux 51, CH-2000
Neuchâtel, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bandi%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Siva Bandi</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schlemper-Scheidt%20MD%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Marie-Désirée Schlemper-Scheidt</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Marie-Désirée Schlemper-Scheidt</span></h3>
<div class="p">
<sup>†</sup>
Laboratory
for Bioanalytical Chemistry, University
of Neuchâtel, Avenue de Bellevaux 51, CH-2000
Neuchâtel, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schlemper-Scheidt%20MD%22%5BAuthor%5D" class="usa-link"><span class="name western">Marie-Désirée Schlemper-Scheidt</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rivera%20S%C3%A1nchez%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Rocío Rivera Sánchez</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Rocío Rivera Sánchez</span></h3>
<div class="p">
<sup>†</sup>
Laboratory
for Bioanalytical Chemistry, University
of Neuchâtel, Avenue de Bellevaux 51, CH-2000
Neuchâtel, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rivera%20S%C3%A1nchez%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rocío Rivera Sánchez</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sutour%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sylvain Sutour</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sylvain Sutour</span></h3>
<div class="p">
<sup>‡</sup>
Neuchatel
Platform for Analytical Chemistry (NPAC), University of Neuchâtel, Avenue de Bellevaux 51, CH-2000
Neuchâtel, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sutour%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sylvain Sutour</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Glauser%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Gaétan Glauser</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Gaétan Glauser</span></h3>
<div class="p">
<sup>‡</sup>
Neuchatel
Platform for Analytical Chemistry (NPAC), University of Neuchâtel, Avenue de Bellevaux 51, CH-2000
Neuchâtel, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Glauser%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gaétan Glauser</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishida%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yojiro Ishida</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yojiro Ishida</span></h3>
<div class="p">
<sup>§</sup>
Center
for Advanced Biotechnology and Medicine (CABM), Rutgers University, 679 Hoes Lane West, Piscataway, New Jersey
08854, United States
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishida%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yojiro Ishida</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22von%20Reu%C3%9F%20SH%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Stephan H von Reuß</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Stephan H von Reuß</span></h3>
<div class="p">
<sup>†</sup>
Laboratory
for Bioanalytical Chemistry, University
of Neuchâtel, Avenue de Bellevaux 51, CH-2000
Neuchâtel, Switzerland
</div>
<div class="p">
<sup>‡</sup>
Neuchatel
Platform for Analytical Chemistry (NPAC), University of Neuchâtel, Avenue de Bellevaux 51, CH-2000
Neuchâtel, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22von%20Reu%C3%9F%20SH%22%5BAuthor%5D" class="usa-link"><span class="name western">Stephan H von Reuß</span></a>
</div>
</div>
<sup>†,</sup><sup>‡,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Laboratory
for Bioanalytical Chemistry, University
of Neuchâtel, Avenue de Bellevaux 51, CH-2000
Neuchâtel, Switzerland
</div>
<div id="aff2">
<sup>‡</sup>
Neuchatel
Platform for Analytical Chemistry (NPAC), University of Neuchâtel, Avenue de Bellevaux 51, CH-2000
Neuchâtel, Switzerland
</div>
<div id="aff3">
<sup>§</sup>
Center
for Advanced Biotechnology and Medicine (CABM), Rutgers University, 679 Hoes Lane West, Piscataway, New Jersey
08854, United States
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>stephan.vonreuss@unine.ch</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 20; Accepted 2025 Apr 29; Revised 2025 Apr 29; Collection date 2025 Aug 20.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371488</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>
<em>N</em>-acyl ethanolamines represent conserved lipophilic
signaling molecules that function as endogenous ligands at G-protein-coupled receptors, ion channels, and nuclear
receptors. Using a combination of comparative ultrahigh-performance
liquid chromatography electrospray ionization high-resolution tandem
mass spectrometry (UHPLC-ESI-HR-MS<sup>E</sup>) analysis and microreactions,
a diversity of glycosylated <em>N</em>-acyl phosphoethanolamines
were characterized in <em>Caenorhabditis</em> nematodes. Representative
examples were enriched by RP-C18 chromatography and identified by
NMR spectroscopy. Comparative metabolomics and isotope incorporation
experiments revealed that the biosynthesis of the homologous <em>N</em>-acyl building blocks (approximately 50 compounds) depends
on the bacterial food source, chain elongation and desaturation of
food-derived fatty acids, or their de novo biosynthesis by the nematode,
whereas the biosynthesis of medium-chain <em>N</em>-acyl units
depends on the peroxisomal β-oxidation cycle via the 3-ketoacyl-<em>S</em>-CoA thiolase <em>daf-22</em>. Glycosylation of
these lipophilic <em>N</em>-acyl ethanolamines results in
amphiphilic modular metabolites (approximately 100 identified compounds)
that are released into the environment and exhibit potential signaling
functions. Exclusively male-produced β-sophorosyl <em>N-</em>acyl-phosphoethanolamines like SNAP-13:1cyclo retain females of and , and its biosynthesis requires bacterial
cyclo fatty acids 17:1cyclo and 19:1cyclo, thereby translating growth
phase-dependent bacterial lipogeneses into a behavioral signal. Amphiphilic
2-(β-glucosyl)-glyceryl <em>N</em>-eicosapentaenoyl
phosphoethanolamine (GGp-NAE-20:5), a dominating component of the metabolome, represents a water-soluble
derivative of <em>N</em>-eicosapentaenoyl ethanolamine (NAE
20:5), potentially enabling intra- and interspecies endocannabinoid
signaling.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> nematode-derived modular metabolites, peroxisomal
β-oxidation, 
<em>N</em>-acyl ethanolamine, endocannabinoid
signaling, lipogenesis</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371488_bg5c00012_0010.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cbef/12371488/151f2fa2f64d/bg5c00012_0010.jpg" loading="lazy" height="416" width="773" alt="graphic file with name bg5c00012_0010.jpg"></a></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371488_bg5c00012_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cbef/12371488/151f2fa2f64d/bg5c00012_0008.jpg" loading="lazy" id="tgr1" height="416" width="773" alt="graphic file with name bg5c00012_0008.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Lipophilic long-chain <em>N</em>-acyl ethanolamines (<strong>1</strong>, NAEs) constitute conserved lipid
signaling molecules in
plants, animals, and microorganisms (<a href="#fig1" class="usa-link">Figure 
</a>a).<em>
−


</em> Polyunsaturated <em>N</em>-acyl ethanolamines
(<strong>1</strong>) like anandamide (NAE-20:4 Δ<sup>5,8,11,14</sup>) and <em>N</em>-eicosapentaenoyl ethanolamine (NAE-20:5
Δ<sup>5,8,11,14,17</sup>), as well as related NAEs with <em>N</em>-oleoyl (18:1 Δ<sup>9</sup>), <em>N</em>-linoleoyl
(18:2 Δ<sup>9,12</sup>), <em>N</em>-α-linolenoyl
(18:3 Δ<sup>9,12,15</sup>), or <em>N</em>-palmitoyl
(16:0) residues (<a href="#fig1" class="usa-link">Figure 
</a>c) function as endogenous ligands at G-protein-coupled receptors
(CB1, CB2, GPR55, GPR110, GPR119),<em>
,
</em> ion channels
(TRPV1),<em>
,
</em> and nuclear receptors (PPARα).<em>
,
</em> Biogenesis of <em>N</em>-acyl ethanolamines (<strong>1</strong>) is initiated by <em>N</em>-acylation of membrane forming phosphatidylethanolamines (PEs, <strong>3</strong>),
and the resulting <em>N</em>-acyl phosphatidylethanolamines (NAPEs, <strong>5</strong>) are converted into NAEs (<strong>1</strong>)
via different pathways that include (a) direct cleavage by NAPE-specific
phospholipase D (NAPE-PLD), (b) cleavage to <em>O</em>,<em>O</em>-diacyl glycerols (DAG) and <em>N</em>-acyl phospho­ethanolamines
(NAE-P, <strong>2</strong>) by phospholipase C (PLC), or (c) sequential
cleavage via the corresponding <em>lyso</em>-NAPE (<strong>6</strong>) intermediates by β-hydrolase 4 (ABHD4) and <em>lyso</em>-NAPE-specific phospholipase D<em>
−





</em> (<a href="#fig1" class="usa-link">Figure 
</a>a). NAE
abundance is further regulated via their hydrolysis by fatty acid
amide hydrolases (FAAH).<em>
,
</em>
</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cbef/12371488/bbcf76b883e3/bg5c00012_0001.jpg" loading="lazy" id="gr1" height="4106" width="3500" alt="1"></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Homologous <em>N</em>-acyl ethanolamines (NAEs, <strong>1</strong>) derived from <em>N</em>-acylation of phosphatidylethanolamines
(<strong>3</strong>) via <em>N</em>-acyl phosphatidylethanolamines
(NAPEs, <strong>5</strong>), either directly or via <em>N</em>-acyl phosphoethanolamine (<strong>2</strong>) or <em>lyso</em>-<em>N</em>-acyl phosphatidylethanolamine (<em>lyso</em>-NAPE, <strong>6</strong>) and <em>N</em>-acyl glycerylphosphoethanolamine
(Gp-NAE, <strong>7</strong>) intermediates; (b) nematode-derived modular
metabolites carrying homologous <em>N</em>-acyl ethanolamines
building blocks: β-sophorosyl <em>N</em>-acyl phosphoethanolamines
(<strong>8</strong>, SNAP), β-glucosyl <em>N</em>-acyl phosphoethanolamines
(<strong>9</strong>, GNAP), and 2-(β-glucosyl)­glyceryl <em>N</em>-acyl phosphoethanolamines (<strong>10</strong>, GGp-NAE); (c) Long-chain <em>N</em>-acyl residues of NAEs (<strong>1</strong>) involved in endocannabinoid
signaling; (<strong>d</strong>) modular metabolites (<strong>11</strong>–<strong>15</strong>) isolated from JU1904 with <em>N</em>-acyl building blocks derived from
peroxisomal β-oxidation; (<strong>e</strong>) Distribution of β-sophorosyl <em>N</em>-acyl phosphoethanolamines (<strong>8</strong>) in gonochoristic JU1904, PB2801, PB4641, and JU1422.</p></figcaption></figure><p>The bacterivorous nematode serves as a model organism
to study lipogenesis and lipid metabolism,
and their effect on lifespan and development.<em>
,
</em> Long-chain <em>N</em>-acyl ethanolamines (<strong>1</strong>), including NAE-20:4 and NAE-20:5, have previously been identified
in <em>
−

</em> and shown to
modulate thermal avoidance, olfactory
sensitivity, and cholesterol homeostasis and coordinate nutrient status with metabolic
changes that ultimately determine lifespan.<em>
−

</em> Two functional
NAPE-PLD orthologues have been identified along with NAE sensing G protein-coupled receptors such as NPR-19
and NPR-32.
</p>
<p>Biosynthesis of NAE-20:5
and NAE-20:4 (<strong>1</strong>) and related
compounds is reduced in peroxisomal β-oxidation mutants such
as <em>daf-22</em> (3-ketoacyl-<em>S</em>-CoA thiolase),
presumably due to depletion of ethanolamine pools upon accumulation
of copious amounts of ethanolamine conjugates of very long-chain ascarosides,
the precursors of a conserved class of nematode signaling molecules.<em>
,
</em> Other peroxisomal β-oxidation mutants such as <em>acox-1</em> (acyl-CoA oxidase) exhibit no effect on NAE-20:5 and NAE-20:4 biosynthesis but display a strongly reduced abundance of <em>lyso</em>-NAPEs (<strong>6</strong>) potentially involved in NAE
biosynthesis. While these results demonstrate
the implication of peroxisomal β-oxidation in ’ NAE biosynthesis, the relationship
between lipogenesis, β-oxidation, and NAE biogenesis has remained
largely enigmatic.</p>
<p>Here, we demonstrate that <em>N</em>-acyl ethanolamine
metabolisms in various <em>Caenorhabditis</em> spp., including
the hermaphroditic model organism as well as the gonochoristic and , involve multiple
homologous series of fatty acid building blocks that either originate
directly from the bacterial food source and the elongation of food-derived
precursors or from de novo lipogenesis. A diversity of <em>N</em>-acyl units of medium chain length are derived from chain shortening
of long-chain precursors via the <em>daf-22</em>-dependent
peroxisomal β-oxidation cycle. Lipophilic <em>N</em>-acyl ethanolamines serve as building blocks for amphiphilic β-glycosyl <em>N</em>-acyl phosphoethanolamines (<strong>8</strong>, <strong>9</strong>) and 2-(β-glucosyl)-glyceryl <em>N</em>-acyl phosphoethanolamines
(<strong>10</strong>) that are subsequently released into the environment,
indicating potential signaling functions (<a href="#fig1" class="usa-link">Figure 
</a>b). For example, the male-specific β-sophorosyl <em>N</em>-<em>cis</em>-5,6-methylenedodecanoyl phosphoethanolamine
(<strong>11</strong>), which encodes the availability and growth phase
of the bacterial food
source, retains females of and . Our results demonstrate
that <em>N</em>-acyl ethanolamine metabolism is not only involved
in endogenous signaling but also participates in intraspecies and
potentially even interspecies interactions via amphiphilic signaling
molecules released into the environment.</p></section><section id="sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Identification
of β-Sophorosyl <em>N</em>-Acyl
Phosphoethanolamines from </h3>
<p>Comparative ultrahigh-performance liquid chromatography
electrospray ionization high-resolution tandem mass spectrometry (UHPLC-ESI-HR-MS<sup>E</sup>) analysis of exometabolome extracts from seven <em>Caenorhabditis</em> species of the Elegans group, N2, PB2801, AF16, JU1422, PB4641, JU1373, and JU1904 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>) highlighted a novel class of nematode-derived modular metabolites
(NDMMs) that carry homologous <em>N</em>-acyl ethanolamine
building blocks. Inspection of HR-MS/MS spectra (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2</a>) suggested modular β-glycosyl <em>N</em>-acyl phosphoethanolamine structures
that integrate building blocks from carbohydrate, <em>N</em>-acyl ethanolamine, and energy metabolism. Two dominating representatives
(<strong>11</strong> and <strong>12</strong>) were found to be particularly
abundant in the exometabolome of the tropical species (previously called <em>C. sp. 16</em>) strain JU1904 from Central Bali, Indonesia, and were subsequently isolated from 2 L of the liquid culture
supernatant using a combination of reverse phase chromatography and
semipreparative high-performance liquid chromatography on RP-C18.
Partial structures were identified by analysis of <sup>1</sup>H NMR, <em>dqf</em>-COSY, and HSQC spectra (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>), which indicated the presence of two β-glucopyranosyl units
along with an ethanolamine moiety, whereas the variable <em>N</em>-acyl building blocks were identified as a cyclopropyl group containing <em>cis</em>-5,6-methylenedodecanoyl unit (13:1cyclo (Δ<sup>5,6</sup>)) or a monounsaturated (5<em>Z</em>)-dodecenoyl
unit (12:1 (Δ<sup>5</sup>)) for <strong>11</strong> and <strong>12</strong>, respectively (<a href="#fig1" class="usa-link">Figure 
</a>d). The absolute configuration of the (5<em>R</em>,6<em>S</em>)-<em>cis-</em>cyclopropyl moiety of <strong>11</strong> was subsequently assigned based on its biosynthetic origin from -derived (9<em>R</em>,10<em>S</em>)-<em>cis</em>-9,10-methylenehexadecanoic acid (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S3</a>) of known absolute configuration. The characteristic disaccharide building block
was identified as a β-sophorose (1,2-β-glucopyranosyl-glucopyranoside)
unit based on long-range HMBC correlations (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4</a>). The connectivity of the modular building blocks as deduced
from the HR-MS/MS fragmentation patterns (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S2 and S5</a>) was finally confirmed by long-range H,H-COSY and
H,C-HMBC correlations to reveal β-<span class="text-underline">S</span>ophorosyl <em>
<span class="text-underline">N</span>
</em>-<span class="text-underline">A</span>cyl <span class="text-underline">P</span>hosphoethanolamine (SNAP) structures that we like to
call SNAP<em>-</em>13:1cyclo (<strong>11</strong>) and SNAP-12:1
(<strong>12</strong>). The energy-rich and rather labile β-sophorosyl-1-phosphate
linkage was unambiguously established based on the vicinal <sup>3</sup>
<em>J</em>-(H,P)-coupling constant of 8.8 Hz observed for
an anomeric proton signal with <em>dd</em> multiplicity (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S6</a>). The β-glycosyl-1-phosphate
linkage in SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>) is prone to hydrolysis to furnish the corresponding <em>N</em>-acyl phosphoethanolamines (<strong>2</strong>) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S7</a>).</p></section><section id="sec2.2"><h3 class="pmc_sec_title">β-Glycosyl <em>N</em>-Acyl
Phosphoethanolamines
are Conserved in <em>Caenorhabditis</em> spp.</h3>
<p>Having
identified the two dominating β-sophorosyl <em>N</em>-acyl phosphoethanolamines from as SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>), we utilized their common characteristic ESI-(−)-HR-MS/MS
fragment ion signal at <em>m</em>/<em>z</em> 78.9591 [PO<sub>3</sub>]<sup>−</sup> for
the phosphate unit to screen for additional minor derivatives (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8</a>). These analyses confirmed that SNAP-13:1cyclo
(<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>) are representatives
of a diverse class of β-sophorosyl <em>N</em>-acyl phosphoethanolamines
that share a common fragment ion of low intensity at <em>m</em>/<em>z</em> 421.0753 [C<sub>12</sub>H<sub>22</sub>O<sub>14</sub>P]<sup>−</sup> for the β-sophorose-1-phosphate unit
and carry homologues <em>N</em>-acyl residues of medium chain
lengths ranging from 10 to 15 carbons (see <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S3</a> for 17 assigned structures). Comparative analysis of their
chromatographic retention times indicated two homologous series with
saturated straight chains or iso-branched <em>N</em>-acyl
residues (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S9</a>). Identification of
the straight chain series was confirmed by chemical correlation of
saturated SNAP-12:0 with SNAP-12:1 (<strong>12</strong>) using palladium-catalyzed
hydrogenation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S10</a>). The assignment
of iso-branched <em>N-</em>iso-acyl chains was subsequently
confirmed by one and two-dimensional <sup>1</sup>H NMR spectroscopy
of SNAP-13:0iso (<strong>13</strong>) and SNAP-11:0iso (<strong>14</strong>) isolated from JU1904
and PB2801 (<a href="#fig1" class="usa-link">Figure 
</a>d and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). Furthermore, using microreactions with palladium-catalyzed
hydrogenation, a series of (odd-numbered) inert cyclopropyl group
containing homologues were distinguished from the (even-numbered)
double bond containing derivatives that are readily hydrogenated (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S11</a>).</p>
<p>In addition, HR-MS/MS screening
for <em>m</em>/<em>z</em> 78.9591 [PO<sub>3</sub>]<sup>−</sup> (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8</a>) revealed a
second series of closely related minor derivatives that are characterized
by a monosaccharide building block instead of the β-sophorose
unit (see <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S4</a>). Analysis of HR-MS/MS
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S12</a>) and one and two-dimensional <sup>1</sup>H NMR spectra (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S13 and 14</a>) of isolated GNAP-13:1cyclo (<strong>15</strong>) (<a href="#fig1" class="usa-link">Figure 
</a>d) as a representative for
these β-<span class="text-underline">G</span>lucosyl <em>
<span class="text-underline">N</span>
</em>-<span class="text-underline">A</span>cyl <span class="text-underline">P</span>hosphoethanolamines
(GNAP, <strong>9</strong>) confirmed the presence of a β-glucose-1-phosphate
moiety (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). Targeted screening of
MS<sup>E</sup> chromatograms for the β-glucose-1-phosphate fragment
ion at <em>m</em>/<em>z</em> 259.0224 [C<sub>6</sub>H<sub>12</sub>O<sub>9</sub>P]<sup>−</sup> demonstrated that
the β-glucosyl <em>N</em>-acyl phosphoethanolamines
(<strong>9</strong>) represent a diverse class of metabolites that carry
homologous <em>N</em>-acyl moieties ranging from 7 to 20 carbons,
including saturated straight and iso-branched building blocks, as
well as cyclopropyl group containing and mono and polyunsaturated <em>N</em>-acyl units (see <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S4</a> for
46 assigned structures, some of which have been previously detected). Comparative UHPLC-ESI-HR-MS<sup>E</sup> analysis
of exometabolome extracts of seven <em>Caenorhabditis</em> species from the Elegans group revealed that modular β-glycosyl <em>N</em>-acyl phosphoethanolamines (<strong>8</strong> &amp; <strong>9</strong>) are highly conserved. The gonochoristic species represented
by , , , and produce predominantly
β-sophorosyl <em>N</em>-acyl phosphoethanolamines (<strong>8</strong>) with SNAP-13:1cyclo (<strong>11</strong>) as the dominating
component in all four species (<a href="#fig1" class="usa-link">Figure 
</a>e) along with a diversity of species-specific minor
β-glycosyl <em>N</em>-acyl phosphoethanolamines (<strong>8</strong> &amp; <strong>9</strong>). In contrast, all three androdiecious <em>Caenorhabditis</em> species described so far, , , and were found to
produce only traces of the β-sophorosyl <em>N</em>-acyl
phosphoethanolamines (SNAP, <strong>8</strong>) but instead release the
corresponding β-glucosides (GNAP, <strong>9</strong>). The β-glycosyl <em>N</em>-acyl phosphoethanolamine profile of the model organism N2 is dominated by compounds containing <em>N</em>-cyclopropyl-acyl moieties like 11:1cyclo, 13:1cyclo, and
15:1cyclo moieties, as well as the polyunsaturated <em>N</em>-eicosapentaenoyl moiety (20:5), along with minor amounts of straight
and iso-branched-chain derivatives (<a href="#fig6" class="usa-link">Figure 
</a>f).</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cbef/12371488/180c757f70d1/bg5c00012_0006.jpg" loading="lazy" id="gr6" height="4429" width="4175" alt="6"></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Structures of <em>N</em>-acyl
glycerylphosphoethanolamine
(Gp-NAE-15:1cyclo, <strong>17</strong>) and the corresponding homologous
series of <em>O</em>-acyl <em>N</em>-acyl glycerylphosphoethanolamines
(<em>lyso</em>-NAPE-15:1cyclo, <strong>18</strong>), and the dominating <em>N</em>-acyl 2-(β-glucosyl)-glycerylphosphoethanolamines
(GGp-NAEs, <strong>19</strong>–<strong>21</strong>) isolated from the exometabolome; (b) ESI-(−)-HR-MS
and MS<sup>E</sup> spectra of Gp-NAE-15:1cyclo (<strong>17</strong>) showing
the distribution of <sup>13</sup>C isotopomers for [C<sub>15</sub>,C<sub>5</sub>]-Gp-NAE-15:1cyclo (<strong>17</strong>) as [M –
H]<sup>−</sup> and dehydrated [C<sub>15</sub>,C<sub>2</sub>]-NAE-15:1cyclo fragment ion upon mixed isotope labeling by feeding JU1904 with a 1:1 mixture of [U–<sup>12</sup>C]- and [U–<sup>13</sup>C]-, along with natural abundance spectra as control; (c) extracted
ion chromatogram for <em>m</em>/<em>z</em> 266.248
for NAE-<em>cy</em>15:1 fragment ions reveals diverse <em>lyso</em>-NAPE<em>-cy</em>15:1 (<strong>18</strong>) with
homologous 1,2-isomeric <em>O</em>-acyl building blocks; (d)
while Gp-NAEs (<strong>7</strong>) are enriched in the endolipidome, their
release rates drastically increase upon 2-β-glucosylation to
the corresponding GGp-NAEs (<strong>10</strong>); (e) biosynthesis of
GGp-NAE-15:1cyclo (<strong>19</strong>) and GGp-NAE-18:5 (<strong>21</strong>) is almost abolished in the <em>daf-22</em> metabolome,
whereas those of GGp-NAE-17:1cyclo and GGp-NAE 20:5 (<strong>20</strong>) is only slightly reduced, demonstrating the implication of <em>daf-22</em>-dependent peroxisomal β-oxidation; and (f)
composition of homologous β-glucosyl <em>N</em>-acyl
phosphoethanolamines (GNAP, <strong>9</strong>) and <em>N</em>-acyl 2-(β-glucosyl)-glycerylphosphoethanolamines (GGp-NAE, <strong>10</strong>) in the exometabolome.</p></figcaption></figure></section><section id="sec2.3"><h3 class="pmc_sec_title">Male-Produced β-Sophorosyl <em>N</em>-Acyl Phosphoethanolamines
Attract Females</h3>
<p>Sex-specific exometabolome analysis of gonochoristic JU1904
demonstrated that the three representative β-sophorosyl <em>N</em>-acyl phosphoethanolamines SNAP-13:1cyclo (<strong>11</strong>), SNAP-12:1 (<strong>12</strong>), and SNAP-13:0iso (<strong>13</strong>) are predominantly produced by males (<a href="#fig2" class="usa-link">Figure 
</a>a). Male-specific production of SNAP-13:1cyclo
(<strong>11</strong>), SNAP-12:1 (<strong>12</strong>), and SNAP-13:0iso (<strong>13</strong>) was also demonstrated for gonochoristic PB2801, JU1422, and PB4641 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S15</a>). In addition, male specificity could
also be demonstrated for most of the minor β-sophorosyl <em>N</em>-acyl phosphoethanolamines (<strong>8</strong>) carrying straight
chain, iso-branched, unsaturated, and cyclopropyl groups containing <em>N</em>-acyl residues (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S16</a>). In contrast, the corresponding monosaccharides
such as GNAP-13:1cyclo (<strong>15</strong>) were detected in males and
females of the gonochoristic <em>Caenorhabditis</em> species
and even appeared to be somewhat enriched in females (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S17</a>). These β-glucosyl <em>N</em>-acyl phosphoethanolamines (<strong>9</strong>) are abundant in androdiecious , , and . Their production
in is largely unaffected
by the <em>him-5­(e1490)</em> mutation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S18</a>) that is characterized by a high incidence of males,
approximately 33% versus 0.3% for the N2 wild type, which confirms that β-glucosyl <em>N</em>-acyl phosphoethanolamines (<strong>9</strong>) such as GNAP-13:1cyclo
(<strong>15</strong>) are predominantly produced by hermaphrodites.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cbef/12371488/3ef845dbf772/bg5c00012_0002.jpg" loading="lazy" id="gr2" height="2624" width="3500" alt="2"></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>a</strong>) Sex-specific analysis of JU1904 indicated that the dominating β-sophorosyl <em>N</em>-acyl phosphoethanolamines SNAP-13:1cyclo (<strong>11</strong>), SNAP-12:1 (<strong>12</strong>), and SNAP-13:0iso (<strong>13</strong>) are exclusively produced by males; (b) holding assays demonstrate
that females of JU1904
are retained in scoring regions conditioned with male-produced SNAP-13:1cyclo
(<strong>11</strong>) but not with SNAP-12:1 (<strong>12</strong>), whereas
(c) females of PB2801 are
retained by both SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>) (Welch′s <em>t</em> test, <em>ns</em> = not significant, see <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Tables S9 and S10</a> for data).</p></figcaption></figure><p>The biological activity of male-produced
SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>) on adult behavior was evaluated using a
holding assay that quantifies
nematode retention in conditioned scoring regions in comparison to
the solvent control (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S9</a>). Whereas
males are not affected, females are retained by ecologically relevant
amounts of SNAP-13:1cyclo (<strong>11</strong>) ranging from 10 fmol to
10 pmol (<a href="#fig2" class="usa-link">Figure 
</a>b).
In contrast, SNAP-12:1 (<strong>12</strong>) did not elicit any significant
response in males or females at any of the concentrations tested,
which demonstrates that the presence of the cyclopropyl moiety is
essential for the behavioral activity of SNAP-13:1cyclo (<strong>11</strong>) in females. Females
of are retained by 100 fmol of both SNAP-13:1cyclo (<strong>11</strong>) or SNAP-12:1 (<strong>12</strong>) (<a href="#fig2" class="usa-link">Figure 
</a>c), although male production of SNAP-13:1cyclo
(<strong>11</strong>) is considerably more pronounced than those of SNAP-12:1
(<strong>12</strong>) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S15</a>). males
did not respond to any of the compounds tested (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S10</a>).</p></section><section id="sec2.4"><h3 class="pmc_sec_title">Biogenesis of β-Sophorosyl <em>N</em>-Acyl Phosphoethanolamines
Depends on Bacterial Food Availability</h3>
<p>To decipher the ecological
significance of female retention by male-produced SNAP-13:1cyclo (<strong>11</strong>), the biogenesis of the β-glycosyl <em>N</em>-acyl phosphoethanolamines was investigated. Bacterial food dependency
was evaluated by comparative UHPLC-HR-MS analysis of the conditioned
media supernatant collected from well-fed versus starved cultures, which demonstrates that SNAP-13:1cyclo
(<strong>11</strong>), SNAP-12:1 (<strong>12</strong>), and SNAP-13:0iso (<strong>13</strong>) accumulate under well-fed conditions, whereas their production
is largely abolished upon starvation (<a href="#fig3" class="usa-link">Figure 
</a>a). Similar results were obtained for a diversity
of homologous β-glycosyl <em>N</em>-acyl phosphoethanolamines
(<strong>8</strong>, <strong>9</strong>) in and (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S19 and S20</a>). This pronounced food dependency resembles
those observed for the dominating 1- and 2-isomeric <em>lyso</em>-phosphatidylethanolamines (LPEs, <strong>4</strong>) with homologous
long-chain <em>O</em>-acyl side chains such as 14:0, 16:0,
18:0, 16:1, 18:1, 17:1cyclo, and 19:1cyclo (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S21 and Table S5</a>) that are derived from partial hydrolysis
of bacterial membrane forming phosphatidylethanolamines (PEs, <strong>3</strong>). In contrast, the biosynthesis of ascarosides, a class
of conserved nematode glycolipids with diverse signaling functions,<em>
,
</em> is largely maintained when shifting from well-fed to starved conditions
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S22</a>), suggesting that their fatty
acid-derived aglycones originate from stored lipid reservoirs (or
de novo lipogenesis), whereas the fatty acid moieties of the β-glycosyl <em>N</em>-acyl phosphoethanolamines (<strong>8</strong>, <strong>9</strong>) are more directly derived from the bacterial diet.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cbef/12371488/89ab8ed29d49/bg5c00012_0003.jpg" loading="lazy" id="gr3" height="3147" width="4175" alt="3"></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Food-dependent biosynthesis
of β-sophorosyl <em>N</em>-acyl phosphoethanolamine
(SNAP) in JU1904. (a) Biosynthesis
of SNAP<em>-</em>13:1cyclo (<strong>11</strong>) SNAP-12:1 (<strong>12</strong>), and SNAP-13:0iso (<strong>13</strong>) subsides upon starvation
(relative intensity, mean ± 1 SD, <em>n</em> = 4); (b)
Biosynthesis of SNAP<em>-</em>13:1cyclo
(<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>) strongly depends
on the diet, whereas that of
SNAP-13:0iso (<strong>13</strong>) is independent of the bacterial food
source (relative intensity, mean ± 1 SD; Welch′s <em>t</em> test, <em>n</em> = 3).</p></figcaption></figure><p>Time-dependent quantitative analysis based on the comparison of
released versus retained metabolites, the exo- and endometabolome,
respectively, indicated that under well-fed conditions, approximately
5-20% of SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>) present in worm bodies is being released into the environment every
hour (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S23</a>), which is comparable
to the release rates observed for most ascaroside signals (asc-C5-C9:10-40%/h)
but considerably lower than those observed for the dominating LPEs
(<strong>4</strong>) derived from direct hydrolysis of PEs (e.g., LPE-14:0, LPE-16:0, LPE-17:1cyclo,
LPE-19:1cyclo with 80-160%/h). Starved nematodes retain only limited
reservoirs of SNAP-13:1cyclo (<strong>11</strong>, 9% of well-fed), SNAP-12:1
(<strong>12</strong>, 26%), or SNAP-13:0iso (<strong>13</strong>, 16%) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S24</a>), suggesting that the β-sophorosyl <em>N</em>-acyl phosphoethanolamines represent (slowly) released
environmental signals that potentially serve as a measure for food
availability.</p></section><section id="sec2.5"><h3 class="pmc_sec_title">Assembly of β-Sophorosyl <em>N</em>-Acyl Phosphoethanolamines
Combines Bacterial Lipogenesis and Nematode Gluconeogenesis</h3>
<p>Aiming to decipher the biosynthetic origin of the β-sophorosyl <em>N</em>-acyl phosphoethanolamines, we utilized the mixed [U–<sup>12</sup>C]/[U–<sup>13</sup>C]-isotope labeling strategy,<em>
−

</em> in order to discriminate food-derived and de novo biosynthesized
metabolic building blocks. After feeding with a 1:1 mixture of natural abundance [U–<sup>12</sup>C]
and uniformly carbon-13 labeled [U–<sup>13</sup>C]-, the exometabolome extract was fractionated
by RP-C18 chromatography to enrich the target compounds. Analysis
of ESI-(−)-HR-MS<sup>E</sup> spectra revealed a bimodal distribution
of <sup>13</sup>C isotopomers for the molecular ion signal of SNAP-13:1cyclo
(<strong>11</strong>), whereas the corresponding 13:1cyclo (Δ<sup>5,6</sup>) <em>N</em>-acyl phosphoethanolamine (<strong>2</strong>) fragment ion from loss of the β-sophorose unit shows discrete
signals for [U–<sup>12</sup>C], [<sup>13</sup>C<sub>2</sub>], [<sup>13</sup>C<sub>13</sub>], and [U–<sup>13</sup>C<sub>15</sub>] isotopomers (<a href="#fig4" class="usa-link">Figures 
</a>a and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S25</a>), which demonstrates
that SNAP-13:1cyclo (<strong>11</strong>) is produced by the nematode
upon combining a [C<sub>13</sub>]-13:1cyclo fatty acid and a [C<sub>2</sub>]-ethanolamine building block derived from bacterial metabolism,
along with two glucose units derived from de novo gluconeogenesis.
A similar bimodal <sup>13</sup>C isotopomer distribution was observed
for SNAP-12:1 (<strong>12</strong>) along with discrete signals for [U–<sup>12</sup>C], [<sup>13</sup>C<sub>2</sub>], [<sup>13</sup>C<sub>12</sub>], and [U–<sup>13</sup>C<sub>14</sub>] isotopomers for the
monounsaturated 12:1 (Δ<sup>5</sup>) <em>N</em>-acyl
phosphoethanolamine fragment ion (<strong>2</strong>) (<a href="#fig4" class="usa-link">Figures 
</a>b and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S25</a>), indicating its origin from the bacterial diet. However,
minor signals for <sup>13</sup>C isotopomers with a bell shape distribution
also suggested some incorporation of de novo biosynthesized 16:1 (Δ<sup>9</sup>) or 18:1 (Δ<sup>11</sup>). The bacterial origin of both 12:1 (Δ<sup>5</sup>) and 13:1cyclo
(Δ<sup>5,6</sup>) <em>N</em>-acyl units and the ethanolamine
building block along with their assembly by the nematode could also
be confirmed for [C<sub>13</sub>,C<sub>2</sub>]-NAE-P-13:1cyclo and
[C<sub>12</sub>,C<sub>2</sub>]-NAEP-12:1, the corresponding <em>N</em>-acyl phosphoethanolamines (<strong>2</strong>) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S26</a>) derived from hydrolysis of SNAP-13:1cyclo
(<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>), respectively (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S7</a>).</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cbef/12371488/a960b107505c/bg5c00012_0004.jpg" loading="lazy" id="gr4" height="3708" width="4175" alt="4"></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>ESI-(−)-HR-MS<sup>E</sup> spectra
showing the distribution
of <sup>13</sup>C isotopomers for molecular ions [M – H]<sup>−</sup> and CID-derived <em>N</em>-acyl phosphoethanolamine
fragment ions [M – C<sub>12</sub>H<sub>21</sub>O<sub>10</sub>]<sup>−</sup> of SNAP-13:1cyclo (<strong>11</strong>), SNAP-12:1
(<strong>12</strong>), and SNAP-13:0iso (<strong>13</strong>) upon mixed isotope
labeling by feeding JU1904
with a 1:1 mixture of [U–<sup>12</sup>C]- and [U–<sup>13</sup>C]-, along with natural
abundance spectra as control. (a, b) Biogenesis of SNAP-13:1cyclo
(<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>) depends on the assembly
of bacteria-derived <em>N</em>-acyl and ethanolamine units
along with a sophorose unit from gluconeogenesis. (c) In contrast,
SNAP-13:1iso (<strong>13</strong>) depends on de novo lipogenesis of the <em>N</em>-iso-acyl moiety, which incorporates a [<sup>13</sup>C<sub>5</sub>]-isovalerate building block upon feeding with 50% <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>6</sub>]-leucine enriched but is not labeled by <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>5</sub>]-valine.</p></figcaption></figure></section><section id="sec2.6"><h3 class="pmc_sec_title">Biogenesis of β-Sophorosyl <em>N</em>-Acyl Phosphoethanolamines
Depends on the Bacterial Diet</h3>
<p>In order to explore how the
lipid composition of the bacterial diet affects the biosynthesis of
β-sophorosyl <em>N</em>-acyl phosphoethanolamines, was cultivated with
either the standard OP50 diet
or with , an alternative
food source that has been shown to affect nematode reproduction, fitness,
and lifespan. Characterization of the
fatty acid composition of the OP50 lipidome using fatty acid methyl ester (FAME) analysis by GC-EIMS
showed two cyclopropyl fatty acids, 17:1cyclo (Δ<sup>9,10</sup>) and 19:1cyclo (Δ<sup>11,12</sup>), along with monounsaturated
palmitoleic acid (16:1 (Δ<sup>9</sup>)) and <em>cis</em>-vaccenic acid (18:1 (Δ<sup>11</sup>)) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S27a</a>), which represent potential precursors for the <em>N</em>-acyl units of SNAP-13:1cyclo (Δ<sup>5,6</sup>) (<strong>11</strong>) and SNAP-12:1 (Δ<sup>5</sup>) (<strong>12</strong>),
respectively. In contrast, the lipidome of is characterized by the absence of cyclopropyl and monounsaturated
fatty acids and instead represents a rich source of iso- and anteiso<em>-</em>fatty acids such as 15:0iso, 17:0iso, 15:0anteiso, and
17:0anteiso (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S27b</a>). Shifting ’s
diet from to resulted in specific and strong attenuation
of SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>) (<a href="#fig3" class="usa-link">Figure 
</a>b), which
indicates that their production depends on the bacterial diet and
requires bacterial cyclopropyl and monounsaturated fatty acids, respectively.
Bacterial food-dependent biosynthesis was also observed for GNAP<em>-</em>13:1cyclo (<strong>15</strong>) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S28</a>), a dominating β-glycosyl <em>N</em>-acyl
phosphoethanolamine in the model organism . In contrast, branched-chain β-sophorosyl <em>N-</em>iso-acyl phosphoethanolamines such as SNAP-13:0iso (<strong>13</strong>) are not affected by shifting from to an iso-fatty acid rich diet (<a href="#fig3" class="usa-link">Figure 
</a>b).
These results indicate that the bacterivorous nematode requires -derived cyclopropyl fatty acids (17:1cyclo (Δ<sup>9,10</sup>) and 19:1cyclo (Δ<sup>11,12</sup>)) and their monounsaturated
precursors, palmitoleic (16:1 (Δ<sup>9</sup>)) and <em>cis</em>-vaccenic acid (18:1 (Δ<sup>11</sup>)) for biosynthesis of
SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>),
respectively, which further implies chain shortening of these long-chain
fatty acids (potentially via β-oxidation) to produce the 13:1cyclo
(Δ<sup>5,6</sup>) and 12:1 (Δ<sup>5</sup>) <em>N</em>-acyl building blocks of SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1
(<strong>12</strong>), respectively. Considering that represents an artificial food source for bacterivorous <em>Caenorhabditis</em> spp. and that bacterial fatty acid components
are required for the biosynthesis of SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>), we screened the fatty acid composition
of a set of 12 bacterial strains isolated from the microbiome (the CeMBio collection) by GC-EIMS (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S29</a>), which demonstrated that both cyclopropyl fatty acid and palmitoleic
acid are abundant lipid components in <em>Caenorhabditis’</em> natural environment.</p></section><section id="sec2.7"><h3 class="pmc_sec_title">Iso-Branched β-Glycosyl <em>N</em>-Iso-acyl Phosphoethanolamines
Depend on De Novo Lipogenesis by <em>elo-5</em>
</h3>
<p>In
contrast to the bacteria-derived <em>N</em>-acyl building
blocks of SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>) (<a href="#fig4" class="usa-link">Figure 
</a>a,b), mixed
isotope labeling of SNAP-13:0iso (<strong>13</strong>) in using a 1:1 mixture of natural abundance
[U–<sup>12</sup>C]- and carbon-13 labeled [U–<sup>13</sup>C]- (<a href="#fig4" class="usa-link">Figure 
</a>c) furnished a bell-shaped distribution of <sup>13</sup>C isotopomers for the <em>N</em>-iso-tridecanoyl
phosphoethanolamine fragment ion, indicating a de novo origin for
the branched-chain <em>N</em>-iso-acyl moiety of SNAP-13:0iso
(<strong>13</strong>). Production of iso-branched SNAP-13:0iso (<strong>13</strong>) in is independent
of the iso-fatty acid content of the bacterial diet (<a href="#fig3" class="usa-link">Figure 
</a>b), in agreement with a de
novo origin. Feeding with
an auxotrophic <em>Δval,
Δleu</em>, Δ<em>ile</em> mutant specifically
enriched with either 50% <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>6</sub>]-leucine or 50% <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>5</sub>]-valine
revealed that iso-branched SNAP-13:0iso (<strong>13</strong>) (<a href="#fig4" class="usa-link">Figures 
</a>c, and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S30</a>) and GNAP-11:0iso (<strong>16</strong>) (<a href="#fig5" class="usa-link">Figure 
</a>a) are [<sup>13</sup>C<sub>5</sub>]-labeled upon incorporation of <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>6</sub>]-leucine but not by <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>5</sub>]-valine. Incorporation of <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>6</sub>]-leucine enriched also furnished [<sup>13</sup>C<sub>5</sub>]-labeled iso-fatty acids
[<sup>13</sup>C<sub>5</sub>]-15:0iso and [<sup>13</sup>C<sub>5</sub>]-17:0iso (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S31</a>), thus suggesting
that the <em>N</em>-iso-acyl moiety of SNAP-13:0iso (<strong>13</strong>) and GNAP-11:0iso (<strong>16</strong>) might originate from
de novo lipogenesis of odd-numbered iso-fatty acids, which represent
essential building blocks for branched-chain d17:1iso sphingosine
and related sphingolipids characteristic for nematodes.<em>
−


</em> De novo lipogenesis of iso-fatty acids involves the chain elongation
of a <span class="font-variant-small-caps">l</span>-leucine-derived isovalerate unit<em>
,
</em> and depends on the iso-fatty acid elongase <em>elo-5</em> that participates in the elongation of 13:0iso to 15:0iso.<em>
,,
</em> Implication of <em>elo-5</em> in the biogenesis of the β-glucosyl <em>N</em>-iso-acyl
phosphoethanolamine GNAP-11:0iso (<strong>16</strong>) was demonstrated
using the <em>elo-5­(gk208)</em> loss of function mutant that is known to arrest larval development
at the L1 stage unless supplemented with an exogenous source of iso-branched
fatty acids such as 15:0iso or 17:0iso. Comparative analysis of <em>elo-5­(gk208)</em> cultures supplemented
with either synthetic 17:0iso or its stable isotope labeled [D<sub>6</sub>]-isotopomer<em>
,
</em> furnished exclusively natural
abundance GNAP-11:0iso (<strong>16</strong>) or isotope labeled [D<sub>6</sub>]-GNAP-11:0iso ([D<sub>6</sub>]-<strong>16</strong>), respectively
(<a href="#fig5" class="usa-link">Figure 
</a>b), which
not only demonstrated that GNAP-11:0iso (<strong>16</strong>) biogenesis
in depends on <em>elo-5</em> but also revealed that it proceeds exclusively via chain shortening
of a long-chain iso-fatty acid intermediate such as 17:0iso and not
directly via de novo lipogenesis of the medium-chain 11:0iso fatty
acid building block (<a href="#fig5" class="usa-link">Figure 
</a>d).</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cbef/12371488/bb89c49e2a81/bg5c00012_0005.jpg" loading="lazy" id="gr5" height="2802" width="4175" alt="5"></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Incorporation of <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>6</sub>]-leucine labeled <em>E. coli</em> (50% enrichment) into
[<sup>13</sup>C<sub>5</sub>]-GNAP-11:0iso ([<sup>13</sup>C<sub>5</sub>]-<strong>16</strong>) in PB2801;
(b) incorporation of natural abundance 17:0iso or [D<sub>6</sub>]-17:0iso
by <em>elo-5­(gk208)</em> provides exclusively GNAP-11:0iso (<strong>16</strong>) or [D<sub>6</sub>]-GNAP-11:0iso ([D<sub>6</sub>]-<strong>16</strong>), respectively; (c)
comparative UHPLC-ESI-(−)-HR-MS analysis of wild-type (N2) and β-oxidation mutant
exometabolomes reveals that β-glycosyl <em>N</em>-acyl
ethanolamine phosphate (GNAP) biosynthesis is independent of the mitochondrial
3-ketoacyl-<em>S</em>-CoA thiolase <em>kat-1</em> but
proceeds via the peroxisomal 3-ketoacyl-<em>S</em>-CoA thiolase <em>daf-22</em> (relative intensity, mean ± 1 SD, <em>n</em> = 3); and (d) biosynthetic pathway from <span class="font-variant-small-caps">l</span>-leucine to GNAP-11:0iso
(<strong>16</strong>) via <em>elo-5</em>-dependent de novo lipogenesis
of iso-fatty acids such as 17:0iso and subsequent chain shortening
via peroxisomal β-oxidation by <em>daf-22</em>.</p></figcaption></figure></section><section id="sec2.8"><h3 class="pmc_sec_title">Biosynthesis of β-Glycosyl <em>N</em>-Acyl Phosphoethanolamines
Depends on Peroxisomal β-Oxidation by <em>daf-22</em>
</h3>
<p>Considering that the biosynthesis of the various <em>N</em>-acyl building blocks of β-glycosyl <em>N</em>-acyl phosphoethanolamines in <em>Caenorhabditis</em> spp.
appears to depend on chain shortening of long-chain fatty acids derived
from either the bacterial diet or de novo lipogenesis, we screened β-oxidation mutants carrying deletions
in the peroxisomal 3-ketoacyl-<em>S</em>-CoA thiolase <em>daf-22</em>
<em>
−

</em> or the mitochondrial 3-ketoacyl-<em>S</em>-CoA thiolase <em>kat-1</em>. Comparative exometabolome analysis of wild-type N2, <em>daf-22­(ok693)</em>, and <em>kat-1­(tm1037)</em> demonstrates that the biosynthesis of β-glucosyl <em>N</em>-acyl phosphoethanolamines does not require mitochondrial <em>kat-1</em> but depends on the peroxisomal β-oxidation cycle
that involves <em>daf-22</em> (<a href="#fig5" class="usa-link">Figure 
</a>c). Production of β-glycosyl <em>N</em>-acyl phospho­ethanolamines in the <em>daf-22</em> mutant is strongly attenuated for straight chain and iso-branched
derivatives like GNAP-11:0iso (<strong>16</strong>) and almost abolished
for the cyclopropyl group containing GNAP-13:1cyclo (<strong>15</strong>). These results demonstrate that both β-glycosyl <em>N</em>-acyl phosphoethanolamine (<a href="#fig5" class="usa-link">Figure 
</a>c) and ascaroside biosynthesis depend on <em>daf-22</em> (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S32</a>) and utilize the same primary metabolic pathway, the peroxisomal
β-oxidation of long-chain fatty acids (LCFAs), to regulate the
chain lengths of lipid-derived homologous acyl building blocks. However,
while the <em>daf-22</em> mutant accumulates homologous long-chain
ascarosides that represent the direct precursors for the short-chain
components characteristic for the wild type (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S32</a>),<em>
,
</em> no accumulation of any long-chain
homologue was observed for the β-glycosyl <em>N</em>-acyl phosphoethanolamines, suggesting that the subsequent biosynthetic
steps that connect peroxisomal β-oxidation and β-glycosyl <em>N</em>-acyl phosphoethanolamine biosynthesis are selective for
specific medium-chain fatty acid (MCFAs) building blocks.</p></section><section id="sec2.9"><h3 class="pmc_sec_title">Characterization
of <em>N</em>-Acyl Ethanolamines as
Putative Biosynthetic Intermediates</h3>
<p>Having shown that the
biosynthesis of β-glycosyl <em>N</em>-acyl phosphoethanolamine
(<strong>8</strong> and <strong>9</strong>) with medium-length <em>N</em>-acyl building blocks depends on peroxisomal β-oxidation of
fatty acids, we aimed to characterize putative biosynthetic intermediates.
Targeted ESI-(+)-HR-MS<sup>E</sup> analysis by screening for [C<sub>2</sub>H<sub>8</sub>NO]<sup>+</sup> at <em>m</em>/<em>z</em> 62.061 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S33</a>) enabled
the detection of previously described long-chain <em>N</em>-acyl ethanolamines (NAEs, <strong>1</strong>)<em>
,−





</em> as part of a homologous series carrying <em>N</em>-acyl
units ranging from 10 to 20 carbons (see <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S6</a> for 55 assigned structures) that are highly conserved in <em>Caenorhabditis</em> spp. NAE-15:1cyclo is dominating in fed with , while the other androdiecious <em>Caenorhabditis</em> species
produce predominantly NAE-20:5 () or NAE-18:1 () (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S34</a>). Biosynthesis of the homologous <em>N</em>-acyl chains in NAEs (<strong>1</strong>) parallels those of
SNAPs (<strong>8</strong>) and GNAPs (<strong>9</strong>). De novo produced
branched-chain <em>N-</em>iso-acyl ethanolamines incorporate
a [<sup>13</sup>C<sub>5</sub>]-isovalerate unit upon feeding with <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>6</sub>]-leucine enriched (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S6 and Figure S35</a>). Production of long-chain polyunsaturated <em>N</em>-acyl ethanolamines is partially downregulated in <em>daf-22</em> as previously reported,<em>
,
</em> whereas the biogenesis
of several chain-shortened homologues, including NAE-15:1cyclo, NAE-13:1cyclo,
NAE-15:0iso, NAE-13:0iso, and NAE-18:5, is almost abolished in the <em>daf-22</em> mutant (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S36</a>), demonstrating
their origin from chain shortening of long-chain precursors via the
peroxisomal β-oxidation cycle. Mixed isotope labeling by feeding with a 1:1 mixture of natural abundance
[U–<sup>12</sup>C]- and carbon-13 labeled [U–<sup>13</sup>C]- afforded a diversity of
[C<em>
<sub>n</sub>
</em>,C<sub>2</sub>]-labeled <em>N</em>-acyl ethanolamines (<em>n</em> = carbon number of <em>N</em>-acyl unit) such as [C<sub>16</sub>,C<sub>2</sub>]-NAE-16:0,
[C<sub>19</sub>,C<sub>2</sub>]-NAE-19:1cyclo, [C<sub>17</sub>,C<sub>2</sub>]-NAE-17:1cyclo, [C<sub>15</sub>,C<sub>2</sub>]-NAE-15:1cyclo,
and [C<sub>13</sub>,C<sub>2</sub>]-NAE-13:1cyclo, as well as chain
elongated homologues with polyunsaturated <em>N</em>-octadecanoyl
or <em>N</em>-eicosanoyl moieties such as [C<sub>16</sub>,C<sub>2</sub>,C<sub>2</sub>]-NAE-18:X (X = 3-5), [C<sub>16</sub>,C<sub>2</sub>,C<sub>2</sub>,C<sub>2</sub>]-NAE-20:4 (anandamide) and [C<sub>16</sub>,C<sub>2</sub>,C<sub>2</sub>,C<sub>2</sub>]-NAE-20:5, along
with de novo biosynthesized iso-branched derivatives such as NAE-15:0iso
and NAE-17:0iso (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S37</a>), which demonstrates
that <em>N</em>-acyl ethanolamines are produced by the nematode
via assembly of an ethanolamine unit with a diversity of homologous <em>N</em>-acyl building blocks derived from either bacterial lipogenesis,
chain elongation of bacteria-derived palmitic acid (16:0), or de novo
lipogenesis. Furthermore, identification of chain-shortened homologues,
such as [C<sub>15</sub>,C<sub>2</sub>]-NAE-15:1cyclo and [C<sub>13</sub>,C<sub>2</sub>]-NAE-13:1cyclo (originating from -derived 17:1cyclo and 19:1cyclo) or [C<sub>16</sub>,C<sub>2</sub>,C<sub>2</sub>]-NAE-18:5 (originating from 20:5), and
their <em>daf-22</em> dependence, demonstrates that <em>N</em>-acyl ethanolamine metabolism also incorporates building
blocks derived from peroxisomal β-oxidation of long-chain fatty
acid precursors.</p></section><section id="sec2.10"><h3 class="pmc_sec_title">Glyceryl <em>N</em>-Acyl Phosphoethanolamines
(Gp-NAEs)
are Putative Intermediates in β-Glycosyl <em>N</em>-Acyl
Phosphoethanolamine Biosynthesis</h3>
<p>Aiming to identify the biosynthetic
intermediates that link <em>daf-22</em>-dependent peroxisomal
β-oxidation of long-chain fatty acids with <em>N</em>-acyl ethanolamine metabolism and β-glycosyl <em>N</em>-acyl phosphoethanolamine biosynthesis, the exo- and endometabolomes
of <em>Caenorhabditis</em> spp. were analyzed for modular
metabolites that carry the characteristic <em>N</em>-acyl
ethanolamine building blocks. Targeted ESI-(+)-HR-MS/MS screening
revealed a limited collection of glyceryl <em>N</em>-acyl phosphoethanolamines (Gp-NAEs, <strong>7</strong>) with
11:1cyclo, 13:1cyclo, 15:1cyclo, 12:0, 14:0, 16:1, and 14:1 <em>N</em>-acyl units (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S7</a>), which
are particularly enriched in the endolipidome (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S38</a>). Isomeric Gp-NAEs (<strong>7</strong>) and LPEs (<strong>4</strong>) were distinguished based on their composition of <em>N</em>- or <em>O</em>-acyl building blocks (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S38</a>), MS/MS fragmentation patterns (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S39</a>), and the distribution of <sup>13</sup>C isotopomers upon mixed isotope labeling by feeding with a 1:1 mixture
of [U–<sup>12</sup>C] and [U–<sup>13</sup>C]- (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S40 and S41</a>). ESI-(+)-HR-MS<sup>E</sup> analysis of bacterial food-derived [U–C<sub>n</sub>]-LPEs (<strong>4</strong>) with homologous long-chain <em>O</em>-acyl side chains, such as 14:0, 16:0, 18:0, 16:1, 18:1,
17:1cyclo, and 19:1cyclo, furnished signals for uniformly [U–<sup>12</sup>C] and [U–<sup>13</sup>C] labeled isotopomers (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S40 and Table S5</a>), in agreement with their
origin from partial hydrolysis of bacterial phosphatidyl­ethanolamines
(<strong>6</strong>, PE). In contrast, ESI-(+)-HR-MS<sup>E</sup> of the
Gp-NAEs (<strong>7</strong>) such as Gp-NAE-15:1cyclo (<strong>17</strong>) (<a href="#fig6" class="usa-link">Figure 
</a>a) furnished signals corresponding to [U–<sup>12</sup>C], [<sup>13</sup>C<sub>5</sub>], [<sup>13</sup>C<sub>15</sub>], and [U–<sup>13</sup>C<sub>20</sub>] isotopomers for the
molecular ion [C<sub>20</sub>H<sub>41</sub>NO<sub>7</sub>P]<sup>+</sup>, as well as signals for [U–<sup>12</sup>C], [<sup>13</sup>C<sub>2</sub>], [<sup>13</sup>C<sub>15</sub>], and [U–<sup>13</sup>C<sub>17</sub>] isotopomers of the [C<sub>15</sub>,C<sub>2</sub>]-labeled <em>N</em>-acyl ethanolamine fragment ion
[C<sub>17</sub>H<sub>32</sub>NO]<sup>+</sup> (<a href="#fig6" class="usa-link">Figure 
</a>b and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S41</a>). These results demonstrate that [C<sub>15</sub>,C<sub>5</sub>]-Gp-NAE-15:1cyclo
(<strong>17</strong>) is biosynthesized by the nematode through attachment
of the (chain-shortened) bacteria-derived [C<sub>15</sub>]-15:1cyclo <em>N</em>-acyl unit to an uniformly labeled [C<sub>5</sub>]-glycerylphosphoethanolamine
building block (presumably via <em>N</em>-acylation of PEs
(<strong>3</strong>) to furnish NAPEs (<strong>5</strong>)), which places
the biosynthesis of Gp-NAEs (<strong>7</strong>) upstream of NAEs (<strong>1</strong>) as well as SNAPs (<strong>8</strong>) and GNAPs (<strong>9</strong>). While these results demonstrate the integration of (chain-shortened)
bacteria-derived <em>N</em>-acyl units into endometabolome
enriched Gp-NAEs (<strong>7</strong>), homologous series with medium-length <em>N</em>-acyl units ranging from 8 to 14 carbons have previously
been described in starved L1 larvae utilizing ethanol as sole carbon source, indicating that in the absence of bacterial lipids, the
Gp-NAEs (<strong>7</strong>) can also be produced by de novo lipogenesis.
Their <em>daf-22</em> dependence has previously been described, but their potential function as intermediates
in ’ NAE biosynthesis
has not yet been demonstrated.</p>
<p>Targeted ESI-(+)-HR-MS/MS screening for the corresponding <em>lyso</em>-NAPEs (<strong>6</strong>) in the lipidome revealed the presence of multiple homologous series (<a href="#fig6" class="usa-link">Figures 
</a>c and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S42</a>) that could be distinguished from the isomeric
phosphatidylethanolamines (PEs, <strong>3</strong>) based on their MS/MS
fragmentation patterns (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S43</a>) and
composition of (<em>N</em>- or) <em>O</em>-acyl units. <em>Lyso</em>-NAPEs carrying 15:1cyclo as the <em>N</em>-acyl
unit (<strong>18</strong>) (<a href="#fig6" class="usa-link">Figure 
</a>a) and branched-chain iso-fatty acids such as 15:0iso and
17:0iso as <em>O</em>-acyl unit are particularly abundant
in the lipidome (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S42</a>), demonstrating that <em>lyso</em>-NAPE-dependent <em>N-</em>acyl ethanolamine metabolism also
includes chain-shortened fatty acid building blocks.</p></section><section id="sec2.11"><h3 class="pmc_sec_title">Characterization
of 2-(β-Glucosyl)-glyceryl <em>N</em>-Acyl Phosphoethanolamines
(GGp-NAEs)</h3>
<p>Having identified
Gp-NAEs (<strong>7</strong>) like Gp-NAE-15:1cyclo (<strong>17</strong>) (<a href="#fig6" class="usa-link">Figure 
</a>a) as potential intermediates
in NAE (<strong>1</strong>), SNAP (<strong>8</strong>), and GNAP (<strong>9</strong>) biosynthesis, we screened for additional modular metabolites carrying
these building blocks. ESI-(+)-HR-MS/MS analysis of the exometabolome
extract highlighted a series of structurally related modular 2-(β-glucosyl)-glyceryl <em>N</em>-acyl phosphoethanolamines (GGp-NAEs, <strong>10</strong>)
carrying homologous <em>N</em>-acyl moieties ranging from
10 to 20 carbons (see <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S8</a> for 39 assigned
structures), some of which have previously been detected.<em>
,,
</em> Comparative LC-MS analysis indicated
that these GGp-NAEs (<strong>10</strong>) are abundant in hermaphroditic , , , as well as gonochoristic , , and . In contrast to
male-produced SNAP (<strong>8</strong>), the biosynthesis of GGp-NAEs
(<strong>10</strong>) in gonochoristic <em>Caenorhabditis</em> species is not sex-specific (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S44</a>). The highly unsaturated GGp-NAE 20:5 (Δ<sup>5,8,11,14,17</sup>) (<strong>20</strong>) (<a href="#fig6" class="usa-link">Figure 
</a>a), an amphiphilic derivative of the endocannabinoid <em>N</em>-eicosapentaenoyl ethanolamine (NAE 20:5), constitutes
the most abundant GGp-NAE (<strong>10</strong>) representative in (<a href="#fig6" class="usa-link">Figure 
</a>f) and related nematodes. Dominating GGp-NAEs were
enriched from 4 L of a liquid
culture medium and the structures of GGp-NAE<em>-</em>15:1cyclo
(Δ<sup>7,8</sup>) (<strong>19</strong>), GGp-NAE-20:5 (Δ<sup>5,8,11,14,17</sup>) (<strong>20</strong>), and GGp-NAE-18:5 (Δ<sup>3,6,9,12,15</sup>) (<strong>21</strong>) were established by one and
two-dimensional NMR spectroscopy (<a href="#fig6" class="usa-link">Figure 
</a>a and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>). Comparative analysis of exo- and endometabolome extracts demonstrated
that release rates of the strongly retained Gp-NAEs (<strong>7</strong>) are dramatically increased upon conversion to the corresponding
2-<em>O</em>-β-glucosides (GGp-NAEs, <strong>10</strong>) (<a href="#fig6" class="usa-link">Figure 
</a>d), suggesting
that 2-<em>O</em>-glucosylation converts them into amphiphilic
environmental signals. In contrast to GGp-NAE-20:5 (<strong>20</strong>) and GGp-NAE-17:1cyclo, the relative amounts of GGp-NAE-18:5 (<strong>21</strong>) and GGp-NAE-15:1cyclo (<strong>19</strong>) are strongly reduced
in the <em>daf-22­(ok693)</em> mutant metabolome (<a href="#fig6" class="usa-link">Figure 
</a>e), which demonstrates
that the biosynthesis of these modular metabolites integrates <em>N</em>-acyl building blocks derived from the peroxisomal β-oxidation
of long-chain fatty acid precursors such as 20:5 or -derived 17:1cyclo, respectively. Comparative
analysis of GNAP (<strong>9</strong>) and GGp-NAE (<strong>10</strong>) profiles
in the exometabolome indicated
that relative abundancies of their <em>N</em>-acyl building
blocks are similar but not identical (<a href="#fig6" class="usa-link">Figure 
</a>f). The eicosapentaenoyl (20:5) unit constitutes
the dominating <em>N</em>-acyl group for both <strong>9</strong> and <strong>10</strong>, suggesting a potential function in endocannabinoid
signaling. Cyclopropyl fatty acids represent another group of dominating <em>N</em>-acyl residues, but their distribution differs between
GNAPs (<strong>9</strong>) and GGp-NAEs (<strong>10</strong>), with short
chains (11:1cyclo, 13:1cyclo, and 15:1cyclo) dominating for GNAPs
(<strong>9</strong>), and longer chains (15:1cyclo and 17:1cyclo) dominating
for GGp-NAEs (<strong>10</strong>). The same predominance for shorter <em>N</em>-acyl moieties for GNAPs (<strong>9</strong>) versus GGp-NAEs
(<strong>10</strong>) is also observed for saturated straight and branched-chain
components (<a href="#fig6" class="usa-link">Figure 
</a>f), indicating that GNAPs (<strong>9</strong>) and GGp-NAEs (<strong>10</strong>) are derived from similar but not identical <em>N</em>-aycl
ethanolamine pools or that their biosynthesis is carefully controlled.</p></section><section id="sec2.12"><h3 class="pmc_sec_title">Biosynthesis of Glycosylated <em>N</em>-Acyl Phosphoethanolamines
in <em>Caenorhabditis spp.</em>
</h3>
<p>In conclusion, using
a combination of comparative metabolomics and stable isotope labeling
experiments, the biosynthesis of β-glycosylated <em>N</em>-acyl phosphoethanolamines (<strong>8</strong>, <strong>9</strong>, <strong>10</strong>) in <em>Caenorhabditis</em> spp. was elucidated
(<a href="#fig7" class="usa-link">Figure 
</a>). Biosynthesis
of the two dominating β-sophorosyl <em>N</em>-acyl phosphoethanolamines
SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>) depends
on bacterial lipogenesis and thereby encodes the growth phase of the
bacterial food source (<a href="#fig7" class="usa-link">Figure 
</a>a). Male-specific biosynthesis of female attracting SNAP-13:1cyclo
(<strong>11</strong>) requires bacteria-derived cyclopropyl fatty acids
(17:1cyclo or 19:1cyclo), which are exclusively produced upon stationary
phase-dependent expression of cyclopropyl fatty acid synthase (<em>cfa</em>) that converts
all monounsaturated 16:1 and 18:1 into 17:1cyclo and 19:1cyclo, respectively.
Because biosynthesis of SNAP-12:1 (<strong>12</strong>) requires the bacteria-derived
monounsaturated fatty acids 16:1 and 18:1, sex-specific SNAP-13:1cyclo
(<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>) signaling in gonochoristic <em>Caenorhabditis spp</em>. encodes information about the availability
and growth phase of the bacterial food source(s) (<a href="#fig7" class="usa-link">Figure 
</a>a). In contrast, the biosynthesis
of iso-branched SNAPs such as SNAP-13:0iso (<strong>13</strong>) involves
de novo lipogenesis of iso-fatty acids including chain elongation
of a <span class="font-variant-small-caps">l</span>-leucine-derived [C<sub>5</sub>]-isovalerate building block via elongase <em>elo-5</em>, to furnish long-chain iso-fatty acid precursors (15:0iso and 17:0iso)
(<a href="#fig7" class="usa-link">Figure 
</a>b).</p>
<figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cbef/12371488/634236dfd735/bg5c00012_0007.jpg" loading="lazy" id="gr7" height="2708" width="4175" alt="7"></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Biogenesis
of β-glycosyl <em>N</em>-acyl phosphoethanolamines
in <em>Caenorhabditis</em> spp. involves the integration of
fatty acid building blocks from growth phase-dependent bacterial lipogenesis,
as well as (b) <em>elo-5</em>-dependent de novo lipogenesis,
along with lipolysis, peroxisomal β-oxidation via <em>daf-22</em> and NAE metabolism; (c) biosynthetic pathway from exogenous or endogenous
lipids to β-glycosyl <em>N</em>-acyl phosphoethanolamines
involves lipolysis and chain shortening of resulting long-chain fatty
acids (LCFAs) by <em>daf-22</em>-dependent peroxisomal-β-oxidation
to furnish medium-chain fatty acids (MCFAs), which are attached to
phosphatidylethanolamines (PE, <strong>3</strong>) by <em>N</em>-acyltransferases (<em>NAT</em>) to furnish <em>N</em>-acyl phosphatidylethanolamines (NAPEs, <strong>5</strong>). Hydrolysis
of NAPEs provides <em>N</em>-acyl <em>lyso-</em>phosphatidylethanolamines
(<em>lyso</em>-NAPEs, <strong>6</strong>), glycerylphosphate <em>N</em>-acyl ethanolamines (Gp-NAEs, <strong>7</strong>), and <em>N</em>-acyl ethanolamines (NAEs, <strong>1</strong>), which serve
as precursors for the β-sophorosyl <em>N</em>-acyl ethanolamine
phosphates (SNAP, <strong>8</strong>), β-glucosyl <em>N</em>-acyl ethanolamine phosphates (GNAP, <strong>9</strong>), and 2-β-glucosyl
glycerylphosphate <em>N</em>-acyl ethanolamines (GGp-NAEs, <strong>10</strong>) that are subsequently released into the environment.</p></figcaption></figure><p>The various long-chain fatty acid (LCFA) building
blocks originating
from bacterial lipogenesis or de novo lipogenesis are released by
lipolysis (<a href="#fig7" class="usa-link">Figure 
</a>c) and potentially enter the <em>daf-22</em>-dependent peroxisomal
β-oxidation cycle. Resulting medium-chain fatty acids (MCFA)
and their LCFA precursors are subsequently utilized for <em>N</em>-acylation of phosphatidylethanolamines (PEs, <strong>3</strong>) in
a tightly controlled manner to furnish <em>N</em>-acyl phosphatidylethanolamines
(NAPEs, <strong>5</strong>). NAPEs (<strong>5</strong>) with long and medium-chain <em>N</em>-acyl moieties are hydrolyzed, either directly or via the
corresponding <em>lyso</em>-NAPE (<strong>6</strong>) and Gp-NAE
(<strong>7</strong>) intermediates, to furnish lipophilic NAEs (<strong>1</strong>) that act as endogenous signaling molecules, and in addition, also serve as precursors for
amphiphilic β-glycosyl <em>N</em>-acyl phosphoethanolamines
such as the exclusively male-produced SNAPs (<strong>8</strong>), as well
as GNAPs (<strong>9</strong>). Furthermore, 2-<em>O</em>-β-glucosylation
of endometabolome enriched Gp-NAEs (<strong>7</strong>) furnishes the
amphiphilic 2-(β-glucosyl)-glyceryl <em>N</em>-acyl
phosphoethanolamines (GGp-NAEs, <strong>10</strong>) (predominantly 20:5
and 15:1cyclo in ), which
are subsequently released into the environment to function as precursors
for the corresponding NAEs (<strong>1</strong>), thereby enabling intra-
and potentially even interspecies signaling.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">Discussion</h2>
<p>
<em>N</em>-Acyl ethanolamines such as the endocannabinoids
NAE-20:5 and NAE-20:4 are known as endogenous lipid signaling molecules
that regulate various aspects of biology, including reproductive development, TGF-β signaling,
cholesterol transport, axon regeneration, behavior, and nociception. Our chemical analysis revealed that the structural
diversity of homologous <em>N</em>-acyl ethanolamines with
55 assigned structures in and related nematodes is considerably larger than previously anticipated.
This structural diversity originates from the integration of diverse
fatty acid building blocks derived directly from bacterial lipogenesis,
from chain elongation and desaturation of bacteria-derived precursors
(mainly palmitic acid, 16:0), or from <em>elo-5</em>-dependent
de novo lipogenesis of iso-fatty acids by the nematode. Structural
diversity is further increased by chain shortening of long-chain fatty
acids via the <em>daf-22</em>-dependent peroxisomal β-oxidation
cycle, demonstrating that NAE signaling involves building blocks from
anabolic and catabolic pathways. Because biological activities of
endocannabinoid signals depend most strongly on the homologous <em>N</em>-acyl side chains, the structural diversity of NAE metabolites
might be paralleled by a yet undisclosed functional diversity.</p>
<p>Furthermore, our results demonstrate that the lipophilic <em>N</em>-acyl ethanolamines serve as building blocks for three
classes of amphiphilic nematode-derived modular metabolites (NDMMs),
including 17 members of β-sophorosyl <em>N</em>-acyl
phosphoethanolamines (SNAPs, <strong>8</strong>), 46 members of β-glucosyl <em>N</em>-acyl phosphoethanolamines (GNAPs, <strong>9</strong>) and
39 members of 2-(β-glucosyl)-glyceryl <em>N</em>-acyl
phosphoethanolamines (GGp-NAEs, <strong>10</strong>). While these compounds
are conserved in <em>Caenorhabditis</em>, their individual
compound profiles are highly species-specific.</p>
<p>Comparative analysis
of exo- and endometabolomes revealed that
the lipophilic NAE (<strong>1</strong>) and Gp-NAE (<strong>7</strong>) precursors
are retained in the endolipidome, whereas conversion into the amphiphilic <em>O</em>-glycosyl derivates, SNAPs (<strong>8</strong>), GNAPs (<strong>9</strong>), and GGp-NAEs (<strong>10</strong>) results in their subsequent
release into the exometabolome, thus suggesting potential signaling
functions. Behavioral assays revealed that ecologically relevant amounts
of the predominantly male-produced SNAP-13:1cyclo (<strong>11</strong>) retains females of gonocho­ristic . Females of are retained
by both SNAP-13:1cyclo (<strong>11</strong>) and SNAP-12:1 (<strong>12</strong>), indicating species-specific communication. In addition, it is
conceivable that the various glycosylated <em>N</em>-acyl
phosphoethanolamines, SNAP (<strong>8</strong>), GNAP (<strong>9</strong>),
and GGp-NAEs (<strong>10</strong>), elicit additional biological activities
on nematode development or behavior that have not yet been characterized.
Considering their accumulation in the exometabolome, facile hydrolysis,
and the fact that water-soluble <em>N</em>-acyl phosphoethanolamines
(<strong>2</strong>) have previously been described as prodrugs for <em>N</em>-acyl ethanolamine (<strong>1</strong>) based endocannabinoids, the glycosylated <em>N</em>-acyl phosphoethanolamines
(<strong>8</strong>, <strong>9</strong>, <strong>10</strong>) might represent
a general mechanism for endogenous <em>N</em>-acyl ethanolamine
signaling to be shared with the environment, and thus potentially
be involved in the regulation of intra and interspecies interactions.
Additional bioassays to characterize the potential bioactivities of
the amphiphilic glycosylated <em>N</em>-acyl phosphoethanolamines
(<strong>8</strong>, <strong>9</strong>, and <strong>10</strong>) and their role
as precursors for lipophilic <em>N</em>-acyl ethanolamines
will be required to evaluate their ecological significance.</p>
<p>The female retention by male-produced SNAP-13:1cyclo (<strong>11</strong>) in depends on the presence
of the cyclopropyl moiety derived exclusively from the bacterial food
source, which provides further evidence for the importance of bacterial
cyclopropyl fatty acids in the chemical signaling of bacterivorous
nematodes. We have previously shown that female production of the
highly species-specific male attracting sex-pheromone fasc-C4-cycloC11
(SMID: fasc#1) in depends on bacteria-derived 17:1cyclo and 19:1cyclo
produced by upon entering the
stationary growth phase. Its biosynthetic
intermediate, 11:1cyclo (SMID: becyp#1) has subsequently been identified
as an endogenous activator of <em>fat-7</em> expression in . In contrast,
the monounsaturated palmitoleic acid (18:1 (Δ<sup>9</sup>)),
which serves as a precursor for 19:1cyclo in and is dominating its lipidome during the exponential growth phase,
enhances recovery of dauer
larva. Growth phase-dependent cyclopropanation
of unsaturated fatty acids is prominent among bacteria<em>
,
</em> and abundant within the microbiome, as shown by FAME analysis of the “CeMBio”
model microbiome, in which 10 out of
12 bacterial isolates were found to be rich in cyclopropyl fatty acids
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S29</a>). Retention of females by male-produced SNAP-13:1cyclo
(<strong>11</strong>), the production of which requires bacterial food-derived
17:1cyclo or 19:1cyclo, thus encodes not only bacterial food availability
but also bacterial growth phase-dependent fatty acid cyclopropanation
as a widespread indicator for the exhaustive conversion of environmental
nutrients into bacterial biomass, which is required to make these
resources accessible for bacterivorous nematodes. Taken together,
our results demonstrate how the metabolism, development, and behavior
of bacterivorous nematodes like the <em>Caenorhabditis</em> spp. are shaped by the lipid composition of its bacterial food source.</p>
<p>Characterization of dietary and lipogenesis pathways along with
complementary chain shortening via peroxisomal β-oxidation in <em>N</em>-acyl ethanolamine biosynthesis represents an important
step toward understanding the metabolic regulation of the lipid and
endocannabinoid network. Additional species-specific glycosylation
of endogenous NAEs and their subsequent release into the environment
facilitates intraspecies and potentially even interspecies interactions
via amphiphilic exocannabinoid signaling molecules.</p></section><section id="sec4"><h2 class="pmc_sec_title">Experimental Procedures</h2>
<section id="sec4.1"><h3 class="pmc_sec_title">Organisms</h3>
<p>All <em>Caenorhabditis</em> nematode
strains, including the wild-type isolates of N2 (United Kingdom), JU1904
(Indonesia), PB2801 (Costa
Rica), AF16 (India), JU1422 (India), PB4641 (United States), and JU1373 (La Réunion), as well as the knockout mutants <em>daf-22(ok693)</em>strain RB859, <em>kat-1­(tm1037)</em> strain VS24, <em>elo-5­(gk208)</em> strain VC410, and <em>him-5­(e1490)</em> strain DR466, were obtained from the Caenorhabditis Genetics Center
(CGC). The uracil auxotroph strain OP50 and the CeMBio bacterial strains (CEent1), (JUb66), (MYb10), (JUb134), (JUb19), (MYb11), (MSPm1), (BIGb0172), (BIGb0393), (MYb71), (BIGb0170), and (JUb44) were obtained from the Caenorhabditis
Genetics Center (CGC). The <em>Δleu Δile</em> Δ<em>val</em> auxotroph <em>ilvB2 leuA169</em> strain GB7044 was obtained from the
Bacillus Genetics Strain Center (BGSC 1A452). The <em>Δleu
Δile</em> Δ<em>val</em> auxotroph BL21­(DE3) <em>ΔilvD ΔleuB
ΔavtA</em> Δ<em>ilvE</em> was generated by
Yojiro Ishida (Rutgers University).</p></section><section id="sec4.2"><h3 class="pmc_sec_title">Cultivation of and </h3>
<p> OP50 and GB7044 were
precultured on solidified Lysogeny Broth agar (15.0 g/L agar with
10.0 g/L tryptone, 5.0 g/L yeast extract, 5.0 g/L NaCl) at 37 °C
for 12 h, and single colonies were used to inoculate 1.0 L of Lysogeny
Broth medium, which was incubated overnight (12–14 h) at 37
°C and 180 rpm. Bacterial cells were harvested by centrifugation
at 4700g for 10 min, washed with M9 phosphate buffer (11.3 g/L Na<sub>2</sub>HPO<sub>4</sub>·7H<sub>2</sub>O, 5.0 g/L NaCl, 3.0 g/L
KH<sub>2</sub>PO<sub>4</sub>, 247 mg/L MgSO<sub>4</sub>·7H<sub>2</sub>O), and stored at 7 °C until being used as food for nematode
cultures (maximum of 2 weeks).</p></section><section id="sec4.3"><h3 class="pmc_sec_title">Cultivation of CeMBio Bacteria</h3>
<p>Bacteria isolated from
the microbiome (CeMBio) were precultured on solidified Lysogeny Broth
agar at 25 °C for 12 h, and single colonies were used to inoculate
1.0 L of Lysogeny Broth (LB) medium, which was incubated for 24 h
at 25 °C and 180 rpm. Bacterial cells were harvested by centrifugation
at 4700<em>g</em> for 10 min, washed with M9 phosphate buffer
(11.3 g/L Na<sub>2</sub>HPO<sub>4</sub>·7H<sub>2</sub>O, 5.0
g/L NaCl, 3.0 g/L KH<sub>2</sub>PO<sub>4</sub>, 247 mg/L MgSO<sub>4</sub>·7H<sub>2</sub>O), frozen at −20 °C, and
lyophilized for subsequent FAME analysis.</p></section><section id="sec4.4"><h3 class="pmc_sec_title">Cultivation of Nematodes</h3>
<p>Nematodes were cultivated
on solidified Nematode Growth Medium (NGM) agar (15 g/L agar, 2.5
g/L peptone, 3.0 g/L NaCl, 2.71 g/L KH<sub>2</sub>PO<sub>4</sub>,
0.89 g/L K<sub>2</sub>HPO<sub>4</sub>, 240 mg/L MgSO<sub>4</sub>·7H<sub>2</sub>O, 147 mg/L CaCl<sub>2</sub>·2H<sub>2</sub>O, 5 mg cholesterol
in 1 mL ethanol, 17.0 g/L agar) seeded
with bacterial food. Wild-type isolates of , , , , , and were cultured at
23 °C, and wild-type
strain N2 and mutants <em>daf-22­(ok693)</em>, <em>kat-1­(tm1037),
elo-5­(gk208)</em>, and <em>him-5­(e1490)</em> were cultured
at 20 °C.</p></section><section id="sec4.5"><h3 class="pmc_sec_title">Preparation of Nematode Metabolome Extracts</h3>
<p>Mixed stage
nematodes from four 10 cm NGM agar plates collected in S-medium were
used as inoculums for liquid cultures grown in 100 mL S-medium (5.85
g/L NaCl, 1.0 g/L K<sub>2</sub>HPO<sub>4</sub>, 6.0 g/L KH<sub>2</sub>PO<sub>4</sub>, 2.94 g/L K<sub>3</sub>[citrate]·H<sub>2</sub>O, 200 mg/L citric acid·H<sub>2</sub>O, 738 mg/L MgSO<sub>4</sub>, 441 mg/L CaCl<sub>2</sub>·2H<sub>2</sub>O, 18.6 mg/L Na<sub>2</sub>H<sub>2</sub>EDTA, 6.9 mg/L FeSO<sub>4</sub>·7H<sub>2</sub>O, 2.0 mg/L MnCl<sub>2</sub>·4H<sub>2</sub>O, 2.9 mg/L ZnSO<sub>4</sub>·7H<sub>2</sub>O, 0.25 mg/L CuSO<sub>4</sub>·5H<sub>2</sub>O, 5 mg/L cholesterol) at 150
rpm and 20 °C (for wild-type
and mutant strains) or 23 °C (for all other <em>Caenorhabditis</em> species). A 1 cm<sup>3</sup> portion of concentrated OP50 bacterial pellets (from an overnight
culture in LB medium at 37 °C) was provided daily as food from
day 1 to 7, after which the cultures were starved for an additional
7 days. Nematodes were separated by centrifugation at 1000<em>g</em> for 10 min, frozen at −20 °C, lyophilized,
powdered by crushing with solid NaCl, and extracted with 3 ×
5 mL methanol to furnish the endometabolome extract. The filtered
supernatant was frozen at −20 °C, lyophilized, and extracted
with 3 × 50 mL methanol for 12 h each to furnish the exometabolome
extract. The extracts were filtered, concentrated to dryness under
reduced pressure at 40 °C, and reconstituted in 1 mL of methanol.
Aliquots were diluted with methanol and analyzed by UHPLC-ESI-HR-MS<sup>E</sup>. All measurements were performed by using at least three
biological replicates.</p></section><section id="sec4.6"><h3 class="pmc_sec_title">Enrichment of Target Compounds</h3>
<p>Target
compounds were
enriched by reverse phase solid phase extraction on 0.5 g of RP-C18
(Chromabond, Macherey-Nagel) using a stepwise gradient of methanol
in water as eluent (10 mL each, from 0% to 100% methanol using 10
increments of +10%). The resulting RP-C18 SPE fractions were concentrated
under reduced pressure using a Vacufuge Plus Concentrator (Eppendorf),
and aliquots were analyzed by UHPLC-ESI-HR-MS.</p></section><section id="sec4.7"><h3 class="pmc_sec_title">Isolation of β-Glycosyl <em>N</em>-Acyl Phosphoethanolamines
(SNAP (8) and GNAP (9))</h3>
<p>The β-glycosyl <em>N</em>-acyl phosphoethanolamine was enriched from 2 L of the liquid culture
supernatant of strain JU1904
grown as described before. The filtered supernatant was frozen at
−20 °C and lyophilized, and the residue was extracted
with 3 × 200 mL of methanol for 12 h each. The combined extract
was filtered and concentrated to dryness under reduced pressure at
40 °C. The resulting residue was reconstituted in 10 mL of methanol
and filtered to remove insoluble salt. The concentrated exometabolome
extract was reconstituted in 5 mL of water and fractionated by reverse
phase chromatography on 5 g of RP-C18 cartridges (Chromabond, Macherey-Nagel)
using a stepwise gradient of methanol in water as eluent (from 0 to
100% methanol using increments of +10% and fractions of 10 mL). The
resulting RP-C18 fractions were concentrated under reduced pressure
using a Vacufuge Plus Concentrator (Eppendorf), and aliquots were
analyzed by UHPLC-ESI-(−)-HR-MS<sup>E</sup> and <sup>1</sup>H NMR. SPE fractions enriched in target compounds (eluted with 70
and 80% methanol) were submitted to semipreparative HPLC using a 1525
EF pump (Waters) connected to a 2487 DAD UV detector (Waters) with
a detection wavelength of 305 nm and a FC203B fraction collector (Gilson).
Chromatographic separation was achieved using an Xterra MS C18 column
(150 mm, 19 mm inner diameter, 5 μm particle size) (Waters).
Optimized HPLC conditions were developed on UHPLC and then transferred
to semipreparative scale. A water/acetonitrile solvent system was
used with 0.05% formic acid at a flow rate of 8 mL/min. The temperature
of the column was maintained at 45 °C. Fractions (8 mL) were
collected every minute. Aliquots of 10 μL were diluted and analyzed
by UHPLC-ESI-(−)-HR-MS<sup>E</sup> with isocratic elution.
HPLC fractions were concentrated under reduced pressure using a Centrivap
vacuum concentrator (Labconco). Fractions rich in target compounds
SNAP-13:1cyclo (<strong>11</strong>), SNAP-12:1 (<strong>12</strong>), SNAP-13:0iso
(<strong>13</strong>), SNAP-11:0iso (<strong>14</strong>), and GNAP-13:1cyclo
(<strong>15</strong>) were dissolved in CD<sub>3</sub>OD and analyzed
by NMR spectroscopy (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>).</p></section><section id="sec4.8"><h3 class="pmc_sec_title">Isolation
of 2-(β-Glucosyl)-glyceryl <em>N</em>-Acyl Phosphoethanolamines
(GGp-NAEs, 10)</h3>
<p>The 2-(β-glucosyl)-glyceryl <em>N</em>-acyl phosphoethanolamines were enriched from 4 L of the
liquid culture supernatant of strain N2 grown as described before. The filtered supernatant was
frozen at −20 °C and lyophilized, and the residue was
extracted with 3 × 200 mL methanol for 12 h each. The combined
extract was filtered and concentrated to dryness under reduced pressure
at 40 °C. The resulting residue was reconstituted in 10 mL methanol
and filtered to remove insoluble salt. The concentrated exometabolome
extract was reconstituted in 5 mL water and fractionated by reverse
phase chromatography on 5 g RP-C18 cartridges (Chromabond, Macherey-Nagel)
using a stepwise gradient of methanol in water as eluent (from 0%
to 100% methanol using increments of +10% and fractions of 10 mL).
The resulting RP-C18 fractions were concentrated under reduced pressure
using a Vacufuge Plus Concentrator (Eppendorf), and aliquots were
analyzed by UHPLC-ESI-(−)-HR-MS<sup>E</sup> and <sup>1</sup>H NMR. SPE fractions enriched in target compounds (eluted with 80%
methanol) were submitted to semipreparative HPLC using an LC-20AR
pump (Shimadzu, Japan) connected to an SPD-20A UV–vis detector
(Shimadzu) with detection at 305 nm and a FRC-40 fraction collector
(Shimadzu). Chromatographic separation was achieved using an Xterra
MS C18 column (150 mm, 19 mm i.d., 5 μm particle size) (Waters).
Optimized HPLC conditions were developed on UHPLC and then transferred
to semipreparative scale. A water/acetonitrile solvent system was
used with 0.05% formic acid at a flow rate of 8 mL/min. The temperature
of the column was maintained at 45 °C. Fractions (8 mL) were
collected every minute. Aliquots of 10 μL were diluted and analyzed
by UHPLC-ESI-(−)-HR-MS<sup>E</sup> with isocratic elution.
HPLC fractions were concentrated under reduced pressure using a Centrivap
vacuum concentrator (Labconco). Fractions rich in target compounds
GGp-NAE-15:1cyclo (<strong>19</strong>), GGp-NAE-20:5 (<strong>20</strong>), and GGp-NAE-18:5 (<strong>21</strong>) were dissolved in CD<sub>3</sub>OD and analyzed by NMR spectroscopy (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>).</p></section><section id="sec4.9"><h3 class="pmc_sec_title">Chemical Correlation by Palladium-Catalyzed Hydrogenation</h3>
<p>Aliquots (10 μL) of semipreparative HPLC fractions enriched
in β-glycosyl <em>N</em>-acyl phosphoethanolamines (from
separation of the JU1904 exometabolome) were diluted with 1 mL ethanol
and treated with 10 mg palladium on carbon (10%, w/w), and hydrogen
gas was bubbled through the vigorously stirred mixture. After 1 h,
the mixtures were filtered over cotton and concentrated under reduced
pressure, and the residues were taken up in 100 μL of methanol
for comparative UHPLC-ESI-(−)-HR-MS<sup>E</sup> analysis using
the diluted untreated exometabolome fraction (10 μL in 100 μL
of methanol) as control.</p></section><section id="sec4.10"><h3 class="pmc_sec_title">Sex-Specific Metabolome Analysis</h3>
<p>For sex-specific exometabolome
analysis of JU1904, PB2801, JU1422, or PB4641, a total number of 100 male and 100 female adult nematodes
were collected from nonstarved cultures on NGM agar seeded with OP50 using a 0.25 mm platinum–iridium
(90:10 wt %) wire pick. Nematodes were incubated for 15 h at 20 °C
and 180 rpm in 500 μL of S-medium supplemented with 100 μL
of an OP50 overnight culture
in LB medium. Cultures were filtered over cotton to remove the nematodes,
and the medium was frozen at −20 °C, lyophilized, and
extracted with 500 μL methanol. Filtered extracts were concentrated
under reduced pressure, and the residues were taken up in 100 μL
of methanol for UHPLC-ESI-(−)-HR-MS<sup>E</sup> analysis.</p></section><section id="sec4.11"><h3 class="pmc_sec_title">Comparative Analysis of Well-Fed and Starved Cultures</h3>
<p>Mixed
stage nematodes of JU1904
or PB2801 from
four 10 cm agar plates collected in S-medium were used as inoculums
for liquid cultures grown in 100 mL of S-medium at 180 rpm and 23
°C. Each day, 1 cm<sup>3</sup> of concentrated OP50 bacterial pellets (from an overnight
culture in LB medium at 37 °C) was provided as food from day
1 to 7, after which worms were separated by centrifugation at 1000<em>g</em> for 10 min, and the filtered supernatant representing
the exometabolome under well-fed conditions was frozen at −20
°C. The separated nematodes were transferred into 100 mL fresh
S-medium and cultured for an additional 7 days without food, after
which worms were separated again by centrifugation at 1000<em>g</em> for 10 min. The filtered supernatant representing the
exometabolome under starved conditions was frozen at −20 °C.
The remaining nematodes were transferred into 100 mL of fresh S-medium
and cultured at 23 °C and 180 rpm for another 7 days without
food, after which worms were separated by centrifugation at 1000<em>g</em> for 10 min. The filtered supernatant representing the
exometabolome under severely starved conditions was frozen at −20
°C. The frozen exometabolome samples were lyophilized and extracted
with 3 × 50 mL methanol for 12 h each. The combined extracts
were filtered, concentrated to dryness under reduced pressure at 40
°C, and reconstituted in 1 mL of methanol for UHPLC-ESI-(−)-HR-MS<sup>E</sup> analysis. All experiments were performed in triplicate.</p></section><section id="sec4.12"><h3 class="pmc_sec_title">Comparative Analysis of Exo- and Endometabolomes</h3>
<p>Mixed
stage JU1904 or PB2801 from four 10 cm agar plates collected
in S-medium were used as inoculums for liquid cultures grown in 100
mL of S-medium at 180 rpm and 23 °C. Each day, 1 cm<sup>3</sup> of concentrated OP50 bacterial
pellets (from an overnight culture in LB medium at 37 °C) was
provided as food from day 1 to 5, after which worms were separated
by centrifugation at 1000<em>g</em> for 10 min. The separated
nematodes were transferred to 100 mL of fresh S-medium supplemented
with 1 cm<sup>3</sup> concentrated OP50 bacterial pellets and cultivated
at 23 °C and 180 rpm for 5 h, after which the worm pellet representing
the endometabolome was separated by centrifugation (10 min at 4700<em>g</em>), frozen at −20 °C, lyophilized, and extracted
with 3 × 50 mL of methanol for 12 h each. The filtered supernatant
representing the exometabolome was frozen at −20 °C, lyophilized,
and extracted with 3 × 50 mL methanol for 12 h each. The combined
extract was filtered, concentrated to dryness under reduced pressure
at 40 °C, and reconstituted in 1 mL methanol. An aliquot of 50
μL was diluted with 50 μL methanol and submitted for LC-MS
analysis. All experiments were performed in triplicate.</p></section><section id="sec4.13"><h3 class="pmc_sec_title">Comparative
Analysis of Wild-Type and Mutant Metabolomes</h3>
<p> mutants <em>daf-22­(ok693)</em> strain RB859, <em>kat-1­(tm1037)</em> strain VS24, <em>elo-5­(gk208)</em> strain VC410, and <em>him-5­(e1490)</em> strain DR466 along with the wild-type strain N2 as control were
precultured on NGM agar plates seeded with OP50. Mixed stage nematodes from four 10 cm agar plates collected
in S-medium were used as inoculums for liquid cultures grown in 100
mL of S-medium at 150 rpm and 20 °C. 1 cm<sup>3</sup> of OP50 bacterial pellets (from overnight cultures
in LB medium) was provided as food from day 1 to 7, and the cultures
were starved for an additional 7 days, after which the cultures were
harvested and processed as previously described. The harvested nematodes
were lyophilized, and their weight was measured for normalization.
All measurements were performed in triplicate.</p></section><section id="sec4.14"><h3 class="pmc_sec_title">Comparative
analysis of and Fed Cultures</h3>
<p> JU1904 or PB2801 were precultured on NGM agar plates
seeded with OP50 or GB7044. Mixed stage nematodes from four
10 cm agar plates collected in S-medium were used as inoculums for
liquid cultures grown in 100 mL of S-medium at 150 rpm and 23 °C.
1 cm<sup>3</sup> of OP50 or GB7044 bacterial pellets (from overnight
cultures in LB medium) was provided as food from day 1 to 7, and the
cultures were starved for an additional 7 days, after which the cultures
were harvested and processed as previously described. All measurements
were performed in triplicate.</p></section><section id="sec4.15"><h3 class="pmc_sec_title">Cultivation of [U–<sup>12</sup>C]/[U–<sup>13</sup>C]-</h3>
<p> BL21­(DE3) was precultured
on solidified M9
minimal salt agar (15.0 g agar/L with 5.0 g/L NaCl, 6.0 g/L Na<sub>2</sub>HPO<sub>4</sub>, 3.0 g/L KH<sub>2</sub>PO<sub>4</sub>, 1.0
g/L NH<sub>4</sub>Cl, 247 mg/L MgSO<sub>4</sub>·7H<sub>2</sub>O, 11.1 mg CaCl<sub>2</sub>·2H<sub>2</sub>O, 3.3 mg/L FeSO<sub>4</sub>·7H<sub>2</sub>O) supplemented with 5.0 g/L <span class="font-variant-small-caps">d</span>-glucose at 37 °C for 12 h, and single colonies were used to
inoculate 1 L of M9 minimal salt medium supplemented with 5.0 g/L <span class="font-variant-small-caps">d</span>-glucose. Cultures were incubated overnight (12–14 h)
at 37 °C with 180 rpm. Bacterial cells were harvested by centrifugation
at 4700<em>g</em> for 10 min, washed with M9 phosphate buffer
(11.3 g/L Na<sub>2</sub>HPO<sub>4</sub>·7H<sub>2</sub>O, 5.0 g/L NaCl, 3.0 g/L KH<sub>2</sub>PO<sub>4</sub>, 247 mg/L MgSO<sub>4</sub>·7H<sub>2</sub>O), and stored at
7 °C until being used as food for nematode cultures (maximum
of 2 weeks). Isotope labeled [U–<sup>13</sup>C]- was prepared by using <span class="font-variant-small-caps">d</span>-[U–<sup>13</sup>C<sub>6</sub>]-glucose as the sole carbon source.</p></section><section id="sec4.16"><h3 class="pmc_sec_title">Mixed
[<sup>12</sup>C]/[<sup>13</sup>C]-Isotope Labeling Experiments</h3>
<p> JU1904 was precultured
at 23 °C on solidified peptone-free Nematode Growth Medium (NGM)
agar (15 g/L agar, 3.0 g/L NaCl, 2.71 g/L KH<sub>2</sub>PO<sub>4</sub>, 0.89 g/L K<sub>2</sub>HPO<sub>4</sub>, 246 mg/L MgSO<sub>4</sub>·7H<sub>2</sub>O, 147 mg/L CaCl<sub>2</sub>·2H<sub>2</sub>O, 5 mg cholesterol in 1 mL ethanol, 17.0 g/L agar) seeded with a
1:1 mixture (w/w) of natural abundance [U–<sup>12</sup>C]-
and [U–<sup>13</sup>C]-. Mixed stage nematodes from four 10 cm peptone-free NGM agar plates collected in citrate-free S-medium
were used as inoculums for liquid cultures grown in 100 mL of citrate-free
S-medium (5.85 g/L NaCl, 1.0 g/L K<sub>2</sub>HPO<sub>4</sub>, 6.0
g/L KH<sub>2</sub>PO<sub>4</sub>, 738 mg/L MgSO<sub>4</sub>, 441 mg/L
CaCl<sub>2</sub>·2H<sub>2</sub>O, 18.6 mg/L Na<sub>2</sub>H<sub>2</sub>EDTA, 6.9 mg/L FeSO<sub>4</sub>·7H<sub>2</sub>O, 2.0
mg/L MnCl<sub>2</sub>·4H<sub>2</sub>O, 2.9 mg/L ZnSO<sub>4</sub>·7H<sub>2</sub>O, 0.25 mg/L CuSO<sub>4</sub>·5H<sub>2</sub>O, 5 mg/L cholesterol) at 150 rpm and 23 °C. 1 cm<sup>3</sup> portion of a 1:1 mixture (w/w) of natural abundance [U–<sup>12</sup>C] and [U–<sup>13</sup>C]- bacterial pellets were provided as food from day 1 to 7. Nematodes
were separated by centrifugation at 1000<em>g</em> for 10
min, frozen at −20 °C,
lyophilized, powdered by crushing with solid NaCl, and extracted with
3 × 5 mL methanol to furnish the endometabolome extract. The
filtered supernatant was frozen at −20 °C, lyophilized,
and extracted with 3 × 50 mL methanol for 12 h each to furnish
the exometabolome extract. The extracts were filtered, concentrated
to dryness under reduced pressure at 40 °C, and reconstituted
in 1 mL of methanol. Aliquots were diluted with methanol and analyzed
by UHPLC-ESI-HR-MS<sup>E</sup>. All measurements were performed in
triplicate.</p></section><section id="sec4.17"><h3 class="pmc_sec_title">Cultivation of <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>5</sub>]-Valine or <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>6</sub>]-Leucine
Labeled </h3>
<p> BL21­(DE3) <em>ΔilvD</em> Δ<em>leuB</em> Δ<em>avtA</em> Δ<em>ilvE</em> was precultured on solidified (15.0 g/L agar) supplemented M9 minimal
salt agar at 37 °C for 12 h and used to inoculate 1 L of M9 minimal
salt medium (12.8 g/L g Na<sub>2</sub>HPO<sub>4</sub>·7H<sub>2</sub>O, 3.0 g/L g KH<sub>2</sub>PO<sub>4</sub>, 1.0 g/L NH<sub>4</sub>Cl, 0.5 g/L NaCl, 240 mg/L MgSO<sub>4</sub>·7H<sub>2</sub>O, 14.7 mg CaCl<sub>2</sub>·2H<sub>2</sub>O) supplemented with
5.0 g/L <span class="font-variant-small-caps">d</span>-glucose as carbon source, as well as <span class="font-variant-small-caps">l</span>-valine (20 mg/L), <span class="font-variant-small-caps">l</span>-leucine (20 mg/L), <span class="font-variant-small-caps">l</span>-isoleucine
(20 mg/L), and kanamycin (50 mg/L). Cultures were incubated overnight
(12–14 h) at 37 °C with 180 rpm. Bacterial cells were
harvested by centrifugation at 4700<em>g</em> for 10 min,
washed with M9 phosphate buffer (11.3 g/L Na<sub>2</sub>HPO<sub>4</sub>·7H<sub>2</sub>O, 5.0 g/L NaCl, 3.0 g/L KH<sub>2</sub>PO<sub>4</sub>, 247 mg/L MgSO<sub>4</sub>·7H<sub>2</sub>O), and stored
at 7 °C until being used as food for nematode cultures. <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>5</sub>]-Valine and <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>6</sub>]-leucine labeled was prepared by substituting the corresponding amino acid supplement
with a 1:1 mixture (w/w) of natural abundance [U–<sup>12</sup>C] and the [U–<sup>13</sup>C]-labeled isotopomer.</p></section><section id="sec4.18"><h3 class="pmc_sec_title">Incorporation
of <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>5</sub>]-Valine and <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>6</sub>]-Leucine
Labeled </h3>
<p> PB2801 was precultured on peptone-free
NGM agar seeded with natural abundance BL21­(DE3) <em>ΔilvD</em> Δ<em>leuB</em> Δ<em>avtA</em> Δ<em>ilvE</em> as control
or a 1:1 mixture (w/w) of natural abundance and <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>5</sub>]-valine or <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>6</sub>]-leucine labeled BL21­(DE3) <em>ΔilvD</em> Δ<em>leuB</em> Δ<em>avtA</em> Δ<em>ilvE</em>. Mixed
stage nematodes from four 10 cm agar plates collected in S-medium
were used as inoculums for liquid cultures grown in 100 mL S-medium
at 150 rpm and 23 °C. 1 cm<sup>3</sup> of natural abundance or a 1:1 mixture (w/w) of natural abundance
and <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>5</sub>]-valine or <span class="font-variant-small-caps">l</span>-[U–<sup>13</sup>C<sub>6</sub>]-leucine labeled BL21­(DE3) <em>ΔilvD</em> Δ<em>leuB</em> Δ<em>avtA</em> Δ<em>ilvE</em> bacterial pellets was provided as food from day 1 to 7, after which
the cultures were harvested and processed as previously described.
All measurements were performed in triplicate.</p></section><section id="sec4.19"><h3 class="pmc_sec_title">Incorporation
of [H]- &amp; [D<sub>6</sub>]-17:0iso by <em>elo-5­(gk208)</em>
</h3>
<p>
<em>elo-5­(gk208)</em> strain VC410 was repeatedly axenized
using egg bleaching to remove the concomitant <em>Stenotrophomonas</em> bacterium that supports its growth. Eggs
were hatched on NGM agar
seeded with OP50 and supplemented
with 10 mM 17:0iso (1 mL of 2.7 mg/mL of fatty acid in EtOH per 9
mL of OP50 culture) synthesized
as previously described. Monoxenic conditions
were established by L1 developmental arrest of the F2 generation after
being depleted of the 17:0iso supplement. L1 arrested nematodes were
subsequently rescued by transfer to plates with 10 mM 17:0iso or [D<sub>6</sub>]-17:0iso. Nematodes were precultured for 4 rounds on NGM
agar seeded with OP50 supplemented
with 10 mM 17:0iso or [D<sub>6</sub>]-17:0iso. Mixed stage nematodes
from six 6 cm agar plates collected in S-medium were used as inoculums
for liquid cultures grown in 100 mL of S-medium at 180 rpm and 23
°C. Each day, 0.5 cm<sup>3</sup>of concentrated OP50 bacterial
pellets (from an overnight culture in LB medium at 37 °C) supplemented
with 1 mM 17iso or [D<sub>6</sub>]-17iso was provided as food from
day 1 to 5, after which the cultures were harvested and processed
as previously described. All experiments were performed in triplicate.</p></section><section id="sec4.20"><h3 class="pmc_sec_title">Lipid Extraction of Nematodes</h3>
<p>Lyophilized nematodes
(10 mg dry weight) were treated with 1.7 mL methyl <em>tert</em>-butyl methyl ether (MTBE), and samples were vortexed vigorously. Worms were sonicated for 30 min at 0 °C,
treated with 0.42 mL of water, and sonicated for another 15 min. Phases
were separated by centrifugation at 4 °C and 4700<em>g</em> for 15 min. The upper organic phase was transferred to a 4 mL glass
vial, and the remaining lower phase was re-extracted with an additional
1.7 mL of MTBE for 15 min. After centrifugation, the organic layers
were combined and evaporated under reduced pressure. The residue was
dissolved in 0.1 mL ACN/iPrOH/water (65:30:5, v/v/v) at a final concentration
of 100 mg dry worms/mL and analyzed by UHPLC-ESI-HR-MS. All measurements
were performed in triplicate.</p></section><section id="sec4.21"><h3 class="pmc_sec_title">Preparation of Fatty Acid
Methyl Esters (FAMEs)</h3>
<p>Lyophilized
nematodes or bacteria (10 mg dry weight) were treated with 1 mL of
2.5% (v/v) sulfuric acid in methanol and incubated at 70 °C for 2 h. After cooling to room temperature,
1.5 mL water and 0.2 mLhexane were
added. The mixture was vortexed for 30 s and centrifuged at 1000<em>g</em> for 5 min. The upper organic layer was separated and utilized
for fatty acid methyl ester analysis by GC-EIMS.</p></section><section id="sec4.22"><h3 class="pmc_sec_title">Gas Chromatography
Electron Ionization Mass Spectrometry (GC-EIMS)</h3>
<p>Gas chromatography
electron ionization mass spectrometry (GC-EIMS)
was performed using a GC 7890B (Agilent) coupled with a Gerstel Multi-Purpose
Sampler (MPS) and connected to an MSD 5977A mass spectrometer (Agilent).
Chromatographic separations were achieved using an HP 5 ms Ultra Inert
column (30 m, 0.25 mm ID, 0.25 μm film thickness) with helium
as the carrier gas at a flow rate of 1 mL/min. A temperature program
starting at 50 °C for 5 min, followed by a linear gradient of
+10 °C/min to 320 and 320 °C for 10 min, was used. A total volume of 1 μL was injected spitless at
an injector temperature of 270 °C. 70 eV mass spectra were acquired
from <em>m</em>/<em>z</em> 35–650 amu. Instruments
were controlled by using Gerstel Maestro software (V1). Data were
processed using MassHunter software version B.07 (2014) from Agilent
Technologies.</p></section><section id="sec4.23"><h3 class="pmc_sec_title">Liquid Chromatography Mass Spectrometry (LC-MS)</h3>
<p>Ultrahigh-performance
liquid chromatography electrospray ionization high-resolution tandem
mass spectrometry (UHPLC-ESI-HR-MS/MS) analyses were performed using
an Acquity ultrahigh-pressure liquid chromatography (UHPLC) system
coupled to a Synapt G2 QTOF mass spectrometer (Waters, Milford, MA)
controlled by MassLynx 4.1 software. Chromatographic separations were
achieved using a Waters Acquity HSS T3 UHPLC column (100 mm ×
2.1 mm, 1.8 μm particle diameter) with a flow rate of 500 μL/min using a binary solvent mixture composed
of eluent A (water containing 0.05% formic acid) and eluent B (98%
acetonitrile and 2% water containing 0.05% formic acid). A linear
gradient from 0 to 100% B in 10 min was applied, followed by a hold
at 100% for 5 min and re-equilibration at 0% B for another 5 min.
Analysis was performed using electrospray ionization (ESI) in positive
and negative ionization modes using a source temperature of 120 °C,
desolvation gas at 15 L/min and 450 °C, and capillary voltage
at 2.5 kV for ESI-(+) or 2.0 kV for ESI-(−). Mass spectra were
recorded from <em>m</em>/<em>z</em> 50 to <em>m</em>/<em>z</em> 1200. Fragmentation data were collected
in data-independent (broad band collision induced dissociation) mode
using a collision energy of 8-55 V in positive mode and 10–60
V in negative mode. Data were analyzed with MassLynx software (Waters,
Milford, MA).</p></section><section id="sec4.24"><h3 class="pmc_sec_title">Nuclear Magnetic Resonance Spectroscopy (NMR)</h3>
<p>NMR spectra
were recorded at 400 MHz for <sup>1</sup>H and 100 MHz for <sup>13</sup>C using a Bruker AVANCE II 400 instrument equipped with a 5 mm BBFO+
probe or at 600 MHz for <sup>1</sup>H and 125 MHz for <sup>13</sup>C using a Bruker Ascend 600 instrument with a 5 mm BBI probe (NPAC,
UniNE). CD<sub>3</sub>OD (99.96% or 99.8% D) or CDCl<sub>3</sub> (99.8%
D) was used as the solvent. Residual solvent signals were used as
an internal reference with <sup>1</sup>H at 3.31 ppm and <sup>13</sup>C at 49.05 ppm for CD<sub>3</sub>OD or <sup>1</sup>H at 7.26 ppm
and <sup>13</sup>C at 77.16 ppm for CDCl<sub>3</sub>. Two-dimensional
high-resolution double quantum filtered (<em>dqf</em>)-COSY
spectra were recorded using the <em>cosydfph</em> pulse sequence
with phase cycling for coherence selection by acquiring 32 scans (NS
64 for GNAP) using 8192 data points along f2 and 256 increments along
f1 for a sweep width of 7.5 ppm. HSQC spectra were recorded using
the <em>hsqcetgpsi2</em> pulse sequence by acquiring 448 scans
using 1024 data points in f2 and 64 increments in f1. HMBC spectra
were recorded by using the <em>hmbcgpndqf</em> pulse sequence
by acquiring 432 scans using 2048 data points in f2 and 128 increments
in f1. Spectra were zero-filled to 8192 × 512 for COSY, 2048
× 128 for HSCQ, and 2048 × 128 for HMBC prior to Fourier
transformation and then phased manually and baseline corrected if
required. Post processing and data analysis were performed using the
TopSpin 4.1.3 (2021, Bruker Biospin), MestReNova 9.0.0 (2013, Mestrelab
Research), and MestReNova 14.1.2 (2020, Mestrelab Research) software.</p></section><section id="sec4.25"><h3 class="pmc_sec_title">Retention Assay</h3>
<p>Nematode preference for environments
conditioned with known amounts of SNAP-13:1cyclo (<strong>11</strong>)
or SNAP-12:1 (<strong>12</strong>) was measured by using a retention assay
with JU1904 and PB2801. On a 6 cm Petri dish filled with
6 mL peptone-free NGM agar, circular scoring regions with 9 mm diameter
were marked. Next, 1 μL of 10% aqueous methanol (as solvent
control) or solutions of SNAP-13:1cyclo (<strong>11</strong>) and of SNAP-12:1
(<strong>12</strong>) in biologically relevant concentrations ranging
from 10 nM–10 μM in 10% aqueous methanol were placed
in the center of the scoring areas and left to dry for 5 min. Young
adult nematodes from nonstarved and noncrowded 6 cm NGM agar plates
seeded with OP50 were separated
by sex and transferred to peptone-free NGM agar without food for approximately
1 h before being used for the experiment in order to minimize the
amount of concomitant bacteria. Individual worms were placed into
the center of the conditioned scoring region, and the time required
for the nematodes to leave the scoring region was recorded. Nematodes
were defined to have left the scoring area when no part of their body
was within the circular boundary. A total number of 20 worms per condition
were analyzed, and experiments were repeated at least twice on separate
days. A Welch’s <em>t</em> test was performed using
SPSS Statistics software version 26 (IBM) to evaluate the effect of
β-glycosyl <em>N</em>-acyl phosphoethanolamines on mean
times nematode spent in scoring regions.</p></section></section><section id="sec42"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371488/bin/bg5c00012_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00012_si_001.pdf</a><sup> (10.6MB, pdf) </sup>
</div></div></section></section><section id="ACK-d14e3936-autogenerated" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors thank Dr. Armelle Vallat (Neuchâtel
Platform
for Analytical Chemistry, NPAC) for GC-EIMS measurements. mutants <em>daf-22(ok693)</em>, <em>kat-1(tm1037)</em>, <em>elo-5(gk208)</em>,
and <em>him-5(e1490)</em> were generated by the International Gene Knockout Consortium, which is gratefully
acknowledged. All nematode strains were provided by the Caenorhabditis
Genetics Center, which is funded by the NIH Office of Research Infrastructure
Programs (P40 OD010440).</p></section><section id="notes-1"><p>The Supporting Information
is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00012?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00012</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Phylogeny of <em>Caenorhabditis</em> species;
HR-MS/MS spectra of SNAPs and GNAPS; hydrolysis of SNAPs ; chemical
correlations via micro reactions; sex-specific analysis; bacterial
food-dependent analysis; release rates; comprative analysis of endo-
and exo-emetabolomes; MS spectra for isotopomers from stable isotope
labelling experiments; MS fragmentation pathways; NAE and lyso-NAPE
profiling; NMR and MS data; retention assay data; and NMR spectra
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00012/suppl_file/bg5c00012_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes-2"><h2>∥.</h2>
<p>S.B. and M.D.S.S. contributed equally to this work. S.B., M.D.S.S.,
R.R.S., and S.v.R conceived the study, designed experiments, and analyzed
the data. S.B., R.R.S., M.D.S.S., S.S., and G.G. performed experiments.
R.R.S. and Y.I. provided research materials. S.v.R., S.B., and M.D.S.
wrote the manuscript with contributions from all authors. All authors
have given approval to the final version of the manuscript.</p></section><section id="notes-3"><p>Financial support
by the Swiss National Science Foundation (SNSF-169700; SNSF-197228)
and the University of Neuchâtel (UniNE) is gratefully acknowledged.</p></section><section id="NOTES-d14e3952-autogenerated"><p>The authors declare no
competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>

Elphick, M. R.
; 
Egertová, M.
. 
The
Phylogenetic Distribution and Evolutionary Origins of Endocannabinoid
Signalling. In Handbook of Experimental Pharmacology; Springer, 2005; Vol. 168, pp 283–297.</cite> [<a href="https://doi.org/10.1007/3-540-26573-2_9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16596778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Handbook%20of%20Experimental%20Pharmacology&amp;publication_year=2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Blancaflor E. B., Kilaru A., Keereetaweep J., Khan B. R., Faure L., Chapman K. D.. 
N-Acyl ethanolamines: lipid metabolites
with functions in plant growth and development. Plant J. 2014;79(4):568–583. doi: 10.1111/tpj.12427.</cite> [<a href="https://doi.org/10.1111/tpj.12427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24397856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20J.&amp;title=N-Acyl%20ethanolamines:%20lipid%20metabolites%20with%20functions%20in%20plant%20growth%20and%20development&amp;author=E.%20B.%20Blancaflor&amp;author=A.%20Kilaru&amp;author=J.%20Keereetaweep&amp;author=B.%20R.%20Khan&amp;author=L.%20Faure&amp;volume=79&amp;issue=4&amp;publication_year=2014&amp;pages=568-583&amp;pmid=24397856&amp;doi=10.1111/tpj.12427&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Silver R. J.. The Endocannabinoid
System of Animals. Animals. 2019;9:686. doi: 10.3390/ani9090686.</cite> [<a href="https://doi.org/10.3390/ani9090686" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6770351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31527410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Animals&amp;title=The%20Endocannabinoid%20System%20of%20Animals&amp;author=R.%20J.%20Silver&amp;volume=9&amp;publication_year=2019&amp;pages=686&amp;pmid=31527410&amp;doi=10.3390/ani9090686&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Cannon A. E., Chapman K. D.. Lipid Signaling through G Proteins. Trends Plant Sci. 2021;26(7):720–728. doi: 10.1016/j.tplants.2020.12.012.</cite> [<a href="https://doi.org/10.1016/j.tplants.2020.12.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33468433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Plant%20Sci.&amp;title=Lipid%20Signaling%20through%20G%20Proteins&amp;author=A.%20E.%20Cannon&amp;author=K.%20D.%20Chapman&amp;volume=26&amp;issue=7&amp;publication_year=2021&amp;pages=720-728&amp;pmid=33468433&amp;doi=10.1016/j.tplants.2020.12.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Syed S. K., Bui H. H., Beavers L. S., Farb T. B., Ficorilli J., Chesterfield A. K., Kuo M. S., Bokvist K., Barrett D. G., Efanov A. M.. Regulation
of GPR119 receptor activity with endocannabinoid-like
lipids. Am. J. Physiol.-Endocrinol. Metab. 2012;303(12):E1469–E1478. doi: 10.1152/ajpendo.00269.2012.</cite> [<a href="https://doi.org/10.1152/ajpendo.00269.2012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23074242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.-Endocrinol.%20Metab.&amp;title=Regulation%20of%20GPR119%20receptor%20activity%20with%20endocannabinoid-like%20lipids&amp;author=S.%20K.%20Syed&amp;author=H.%20H.%20Bui&amp;author=L.%20S.%20Beavers&amp;author=T.%20B.%20Farb&amp;author=J.%20Ficorilli&amp;volume=303&amp;issue=12&amp;publication_year=2012&amp;pages=E1469-E1478&amp;pmid=23074242&amp;doi=10.1152/ajpendo.00269.2012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Im D. S.. GPR119
and GPR55 as Receptors for Fatty Acid Ethanolamides, Oleoylethanolamide
and Palmitoylethanolamide. Int. J. Mol. Sci. 2021;22(3):1034. doi: 10.3390/ijms22031034.</cite> [<a href="https://doi.org/10.3390/ijms22031034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7864322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33494185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=GPR119%20and%20GPR55%20as%20Receptors%20for%20Fatty%20Acid%20Ethanolamides,%20Oleoylethanolamide%20and%20Palmitoylethanolamide&amp;author=D.%20S.%20Im&amp;volume=22&amp;issue=3&amp;publication_year=2021&amp;pages=1034&amp;pmid=33494185&amp;doi=10.3390/ijms22031034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Movahed P., Jönsson B. A., Birnir B., Wingstrand J. A., Jørgensen T. D., Ermund A., Sterner O., Zygmunt P. M., Högestätt E. D.. Endogenous
unsaturated C18 N-acyl ethanolamines are vanilloid
receptor (TRPV1) agonists. J. Biol. Chem. 2005;280(46):38496–38504. doi: 10.1074/jbc.M507429200.</cite> [<a href="https://doi.org/10.1074/jbc.M507429200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16081411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Endogenous%20unsaturated%20C18%20N-acyl%20ethanolamines%20are%20vanilloid%20receptor%20(TRPV1)%20agonists&amp;author=P.%20Movahed&amp;author=B.%20A.%20J%C3%B6nsson&amp;author=B.%20Birnir&amp;author=J.%20A.%20Wingstrand&amp;author=T.%20D.%20J%C3%B8rgensen&amp;volume=280&amp;issue=46&amp;publication_year=2005&amp;pages=38496-38504&amp;pmid=16081411&amp;doi=10.1074/jbc.M507429200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Thabuis C., Tissot-Favre D., Bezelgues J. B., Martin J. C., Cruz-Hernandez C., Dionisi F., Destaillats F.. Biological functions and metabolism
of oleoylethanolamide. Lipids. 2008;43(10):887–894. doi: 10.1007/s11745-008-3217-y.</cite> [<a href="https://doi.org/10.1007/s11745-008-3217-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18704536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lipids&amp;title=Biological%20functions%20and%20metabolism%20of%20oleoylethanolamide&amp;author=C.%20Thabuis&amp;author=D.%20Tissot-Favre&amp;author=J.%20B.%20Bezelgues&amp;author=J.%20C.%20Martin&amp;author=C.%20Cruz-Hernandez&amp;volume=43&amp;issue=10&amp;publication_year=2008&amp;pages=887-894&amp;pmid=18704536&amp;doi=10.1007/s11745-008-3217-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Hansen H. S., Diep T. A.. 
N-acyl ethanolamines, anandamide and
food intake. Biochem. Pharmacol. 2009;78(6):553–560. doi: 10.1016/j.bcp.2009.04.024.</cite> [<a href="https://doi.org/10.1016/j.bcp.2009.04.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19413995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Pharmacol.&amp;title=N-acyl%20ethanolamines,%20anandamide%20and%20food%20intake&amp;author=H.%20S.%20Hansen&amp;author=T.%20A.%20Diep&amp;volume=78&amp;issue=6&amp;publication_year=2009&amp;pages=553-560&amp;pmid=19413995&amp;doi=10.1016/j.bcp.2009.04.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Leung D., Saghatelian A., Simon G. M., Cravatt B. F.. Inactivation of N-acyl phosphatidylethanolamine phospholipase D reveals
multiple mechanisms for the biosynthesis of endocannabinoids. Biochemistry. 2006;45(15):4720–4726. doi: 10.1021/bi060163l.</cite> [<a href="https://doi.org/10.1021/bi060163l" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1538545/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16605240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Inactivation%20of%20N-acyl%20phosphatidylethanolamine%20phospholipase%20D%20reveals%20multiple%20mechanisms%20for%20the%20biosynthesis%20of%20endocannabinoids&amp;author=D.%20Leung&amp;author=A.%20Saghatelian&amp;author=G.%20M.%20Simon&amp;author=B.%20F.%20Cravatt&amp;volume=45&amp;issue=15&amp;publication_year=2006&amp;pages=4720-4726&amp;pmid=16605240&amp;doi=10.1021/bi060163l&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Liu J., Wang L., Harvey-White J., Osei-Hyiaman D., Razdan R., Gong Q., Chan A. C., Zhou Z., Huang B. X., Kim H. Y., Kunos G.. A biosynthetic
pathway
for anandamide. Proc. Natl. Acad. Sci. U.S.A. 2006;103(36):13345–13350. doi: 10.1073/pnas.0601832103.</cite> [<a href="https://doi.org/10.1073/pnas.0601832103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1557387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16938887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=A%20biosynthetic%20pathway%20for%20anandamide&amp;author=J.%20Liu&amp;author=L.%20Wang&amp;author=J.%20Harvey-White&amp;author=D.%20Osei-Hyiaman&amp;author=R.%20Razdan&amp;volume=103&amp;issue=36&amp;publication_year=2006&amp;pages=13345-13350&amp;pmid=16938887&amp;doi=10.1073/pnas.0601832103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Ueda N., Tsuboi K., Uyama T.. Metabolism
of endocannabinoids and
related N-acyl ethanolamines: canonical and alternative
pathways. FEBS J. 2013;280(9):1874–1894. doi: 10.1111/febs.12152.</cite> [<a href="https://doi.org/10.1111/febs.12152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23425575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20J.&amp;title=Metabolism%20of%20endocannabinoids%20and%20related%20N-acyl%20ethanolamines:%20canonical%20and%20alternative%20pathways&amp;author=N.%20Ueda&amp;author=K.%20Tsuboi&amp;author=T.%20Uyama&amp;volume=280&amp;issue=9&amp;publication_year=2013&amp;pages=1874-1894&amp;pmid=23425575&amp;doi=10.1111/febs.12152&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Rahman I. A. S., Tsuboi K., Uyama T., Ueda N.. New players in the
fatty acyl ethanolamide metabolism. Pharmacol.
Res. 2014;86:1–10. doi: 10.1016/j.phrs.2014.04.001.</cite> [<a href="https://doi.org/10.1016/j.phrs.2014.04.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24747663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Res.&amp;title=New%20players%20in%20the%20fatty%20acyl%20ethanolamide%20metabolism&amp;author=I.%20A.%20S.%20Rahman&amp;author=K.%20Tsuboi&amp;author=T.%20Uyama&amp;author=N.%20Ueda&amp;volume=86&amp;publication_year=2014&amp;pages=1-10&amp;pmid=24747663&amp;doi=10.1016/j.phrs.2014.04.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Tsuboi K., Okamoto Y., Rahman I. A., Uyama T., Inoue T., Tokumura A., Ueda N.. Glycerophosphodiesterase
GDE4 as
a novel lysophospholipase D: a possible involvement in bioactive N-acyl
ethanolamine biosynthesis. Biochim. Biophys.
Acta, Mol. Cell Biol. Lipids. 2015;1851(5):537–548. doi: 10.1016/j.bbalip.2015.01.002.</cite> [<a href="https://doi.org/10.1016/j.bbalip.2015.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25596343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta,%20Mol.%20Cell%20Biol.%20Lipids&amp;title=Glycerophosphodiesterase%20GDE4%20as%20a%20novel%20lysophospholipase%20D:%20a%20possible%20involvement%20in%20bioactive%20N-acyl%20ethanolamine%20biosynthesis&amp;author=K.%20Tsuboi&amp;author=Y.%20Okamoto&amp;author=I.%20A.%20Rahman&amp;author=T.%20Uyama&amp;author=T.%20Inoue&amp;volume=1851&amp;issue=5&amp;publication_year=2015&amp;pages=537-548&amp;pmid=25596343&amp;doi=10.1016/j.bbalip.2015.01.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Lee H. C., Simon G. M., Cravatt B. F.. ABHD4 regulates
multiple classes
of N-acyl phospholipids in the mammalian central nervous system. Biochemistry. 2015;54(15):2539–2549. doi: 10.1021/acs.biochem.5b00207.</cite> [<a href="https://doi.org/10.1021/acs.biochem.5b00207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4767004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25853435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=ABHD4%20regulates%20multiple%20classes%20of%20N-acyl%20phospholipids%20in%20the%20mammalian%20central%20nervous%20system&amp;author=H.%20C.%20Lee&amp;author=G.%20M.%20Simon&amp;author=B.%20F.%20Cravatt&amp;volume=54&amp;issue=15&amp;publication_year=2015&amp;pages=2539-2549&amp;pmid=25853435&amp;doi=10.1021/acs.biochem.5b00207&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Ogura Y., Parsons W. H., Kamat S. S., Cravatt B. F.. A calcium-dependent
acyltransferase that produces N-acyl phosphatidylethanolamines. Nat. Chem. Biol. 2016;12(9):669–671. doi: 10.1038/nchembio.2127.</cite> [<a href="https://doi.org/10.1038/nchembio.2127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4990470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27399000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&amp;title=A%20calcium-dependent%20acyltransferase%20that%20produces%20N-acyl%20phosphatidylethanolamines&amp;author=Y.%20Ogura&amp;author=W.%20H.%20Parsons&amp;author=S.%20S.%20Kamat&amp;author=B.%20F.%20Cravatt&amp;volume=12&amp;issue=9&amp;publication_year=2016&amp;pages=669-671&amp;pmid=27399000&amp;doi=10.1038/nchembio.2127&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Tsuboi K., Takezaki N., Ueda N.. The N-acyl ethanolamine-hydrolyzing
acid amidase (NAAA) Chem. Biodivers. 2007;4(8):1914–1925. doi: 10.1002/cbdv.200790159.</cite> [<a href="https://doi.org/10.1002/cbdv.200790159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17712833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biodivers.&amp;title=The%20N-acyl%20ethanolamine-hydrolyzing%20acid%20amidase%20(NAAA)&amp;author=K.%20Tsuboi&amp;author=N.%20Takezaki&amp;author=N.%20Ueda&amp;volume=4&amp;issue=8&amp;publication_year=2007&amp;pages=1914-1925&amp;pmid=17712833&amp;doi=10.1002/cbdv.200790159&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Ueda N., Tsuboi K., Uyama T.. 
N-acyl ethanolamine
metabolism with special reference to N-acyl ethanolamine-hydrolyzing
acid amidase (NAAA) Prog. Lipid Res. 2010;49(4):299–315. doi: 10.1016/j.plipres.2010.02.003.</cite> [<a href="https://doi.org/10.1016/j.plipres.2010.02.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20152858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Prog.%20Lipid%20Res.&amp;title=N-acyl%20ethanolamine%20metabolism%20with%20special%20reference%20to%20N-acyl%20ethanolamine-hydrolyzing%20acid%20amidase%20(NAAA)&amp;author=N.%20Ueda&amp;author=K.%20Tsuboi&amp;author=T.%20Uyama&amp;volume=49&amp;issue=4&amp;publication_year=2010&amp;pages=299-315&amp;pmid=20152858&amp;doi=10.1016/j.plipres.2010.02.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Yue Y., Li S., Shen P., Park Y..  as a model
for obesity research. Curr. Res.
Food Sci. 2021;4:692–697. doi: 10.1016/j.crfs.2021.09.008.</cite> [<a href="https://doi.org/10.1016/j.crfs.2021.09.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8501670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34647034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Res.%20Food%20Sci.&amp;title=as%20a%20model%20for%20obesity%20research&amp;author=Y.%20Yue&amp;author=S.%20Li&amp;author=P.%20Shen&amp;author=Y.%20Park&amp;volume=4&amp;publication_year=2021&amp;pages=692-697&amp;pmid=34647034&amp;doi=10.1016/j.crfs.2021.09.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>An L., Fu X., Chen J., Ma J.. Application of in Lipid Metabolism Research. Int. J. Mol. Sci. 2023;24(2):1173. doi: 10.3390/ijms24021173.</cite> [<a href="https://doi.org/10.3390/ijms24021173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9860639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36674689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Application%20of%20in%20Lipid%20Metabolism%20Research&amp;author=L.%20An&amp;author=X.%20Fu&amp;author=J.%20Chen&amp;author=J.%20Ma&amp;volume=24&amp;issue=2&amp;publication_year=2023&amp;pages=1173&amp;pmid=36674689&amp;doi=10.3390/ijms24021173&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Vrablik T. L., Watts J. L.. Polyunsaturated fatty acid derived signaling in reproduction
and development: insights from and . Mol. Reprod. Dev. 2013;80(4):244–259. doi: 10.1002/mrd.22167.</cite> [<a href="https://doi.org/10.1002/mrd.22167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4350910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23440886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Reprod.%20Dev.&amp;title=Polyunsaturated%20fatty%20acid%20derived%20signaling%20in%20reproduction%20and%20development:%20insights%20from%20and&amp;author=T.%20L.%20Vrablik&amp;author=J.%20L.%20Watts&amp;volume=80&amp;issue=4&amp;publication_year=2013&amp;pages=244-259&amp;pmid=23440886&amp;doi=10.1002/mrd.22167&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>

Estrada-Valencia, R.
; 
de Lima, M. E.
; 
Colonnello, A.
; 
Rangel-López, E.
; 
Saraiva, N. R.
; 
de Ávila, D. S.
; 
Aschner, M.
; 
Santamaría, A.
. 
The
Endocannabinoid System in . In Reviews of Physiology, Biochemistry
and Pharmacology; Springer, 2023; Vol. 184, pp 1–31.
</cite> [<a href="https://doi.org/10.1007/112_2021_64" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8850531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34401955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Reviews%20of%20Physiology,%20Biochemistry%20and%20Pharmacology&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Helf M. J., Fox B. W., Artyukhin A. B., Zhang Y. K., Schroeder F. C.. Comparative
metabolomics with Metaboseek reveals functions of a conserved fat
metabolism pathway in . Nat. Commun. 2022;13(1):782. doi: 10.1038/s41467-022-28391-9.</cite> [<a href="https://doi.org/10.1038/s41467-022-28391-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8831614/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35145075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Comparative%20metabolomics%20with%20Metaboseek%20reveals%20functions%20of%20a%20conserved%20fat%20metabolism%20pathway%20in&amp;author=M.%20J.%20Helf&amp;author=B.%20W.%20Fox&amp;author=A.%20B.%20Artyukhin&amp;author=Y.%20K.%20Zhang&amp;author=F.%20C.%20Schroeder&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=782&amp;pmid=35145075&amp;doi=10.1038/s41467-022-28391-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Abdollahi M., Castaño J. D., Salem J. B., Beaudry F.. Anandamide
Modulates
Thermal Avoidance in Through Vanilloid and Cannabinoid Receptor Interplay. Neurochem. Res. 2024;49(9):2423–2439. doi: 10.1007/s11064-024-04186-w.</cite> [<a href="https://doi.org/10.1007/s11064-024-04186-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38847909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurochem.%20Res.&amp;title=Anandamide%20Modulates%20Thermal%20Avoidance%20in%20Through%20Vanilloid%20and%20Cannabinoid%20Receptor%20Interplay&amp;author=M.%20Abdollahi&amp;author=J.%20D.%20Casta%C3%B1o&amp;author=J.%20B.%20Salem&amp;author=F.%20Beaudry&amp;volume=49&amp;issue=9&amp;publication_year=2024&amp;pages=2423-2439&amp;pmid=38847909&amp;doi=10.1007/s11064-024-04186-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Levichev A., Faumont S., Berner R. Z., Purcell Z., White A. M., Chicas-Cruz K., Lockery S. R.. The conserved endocannabinoid anandamide
modulates olfactory sensitivity to induce hedonic feeding in . Curr. Biol. 2023;33(9):1625–1639.e4. doi: 10.1016/j.cub.2023.03.013.</cite> [<a href="https://doi.org/10.1016/j.cub.2023.03.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10175219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37084730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=The%20conserved%20endocannabinoid%20anandamide%20modulates%20olfactory%20sensitivity%20to%20induce%20hedonic%20feeding%20in&amp;author=A.%20Levichev&amp;author=S.%20Faumont&amp;author=R.%20Z.%20Berner&amp;author=Z.%20Purcell&amp;author=A.%20M.%20White&amp;volume=33&amp;issue=9&amp;publication_year=2023&amp;pages=1625-1639.e4&amp;pmid=37084730&amp;doi=10.1016/j.cub.2023.03.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Galles C., Prez G. M., Penkov S., Boland S., Porta E. O. J., Altabe S. G., Labadie G. R., Schmidt U., Knölker H. J., Kurzchalia T. V., de Mendoza D.. Endocannabinoids in are essential for the mobilization
of cholesterol from internal reserves. Sci.
Rep. 2018;8(1):6398. doi: 10.1038/s41598-018-24925-8.</cite> [<a href="https://doi.org/10.1038/s41598-018-24925-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5913221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29686301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Endocannabinoids%20in%20are%20essential%20for%20the%20mobilization%20of%20cholesterol%20from%20internal%20reserves&amp;author=C.%20Galles&amp;author=G.%20M.%20Prez&amp;author=S.%20Penkov&amp;author=S.%20Boland&amp;author=E.%20O.%20J.%20Porta&amp;volume=8&amp;issue=1&amp;publication_year=2018&amp;pages=6398&amp;pmid=29686301&amp;doi=10.1038/s41598-018-24925-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Lehtonen M., Reisner K., Auriola S., Wong G., Callaway J. C.. Mass-spectrometric
identification of anandamide and 2-arachidonoylglycerol in nematodes. Chem. Biodivers. 2008;5(11):2431–2441. doi: 10.1002/cbdv.200890208.</cite> [<a href="https://doi.org/10.1002/cbdv.200890208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19035572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biodivers.&amp;title=Mass-spectrometric%20identification%20of%20anandamide%20and%202-arachidonoylglycerol%20in%20nematodes&amp;author=M.%20Lehtonen&amp;author=K.%20Reisner&amp;author=S.%20Auriola&amp;author=G.%20Wong&amp;author=J.%20C.%20Callaway&amp;volume=5&amp;issue=11&amp;publication_year=2008&amp;pages=2431-2441&amp;pmid=19035572&amp;doi=10.1002/cbdv.200890208&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Lucanic M., Held J. M., Vantipalli M. C., Klang I. M., Graham J. B., Gibson B. W., Lithgow G. J., Gill M. S.. 
N-acyl ethanolamine signalling mediates
the effect of diet on lifespan
in . Nature. 2011;473(7346):226–229. doi: 10.1038/nature10007.</cite> [<a href="https://doi.org/10.1038/nature10007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3093655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21562563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=N-acyl%20ethanolamine%20signalling%20mediates%20the%20effect%20of%20diet%20on%20lifespan%20in&amp;author=M.%20Lucanic&amp;author=J.%20M.%20Held&amp;author=M.%20C.%20Vantipalli&amp;author=I.%20M.%20Klang&amp;author=J.%20B.%20Graham&amp;volume=473&amp;issue=7346&amp;publication_year=2011&amp;pages=226-229&amp;pmid=21562563&amp;doi=10.1038/nature10007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Annibal A., Karalay Ö., Latza C., Antebi A.. A novel EI-GC/MS method
for the accurate quantification of anti-aging compound oleoylethanolamine
in . Anal. Methods. 2018;10(22):2551–2559. doi: 10.1039/C8AY00396C.</cite> [<a href="https://doi.org/10.1039/C8AY00396C" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Methods&amp;title=A%20novel%20EI-GC/MS%20method%20for%20the%20accurate%20quantification%20of%20anti-aging%20compound%20oleoylethanolamine%20in&amp;author=A.%20Annibal&amp;author=%C3%96.%20Karalay&amp;author=C.%20Latza&amp;author=A.%20Antebi&amp;volume=10&amp;issue=22&amp;publication_year=2018&amp;pages=2551-2559&amp;doi=10.1039/C8AY00396C&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Harrison N., Lone M. A., Kaul T. K., Rodrigues P. R., Ogungbe I. V., Gill M. S.. Characterization
of N-acyl phosphatidylethanolamine-specific phospholipase-D
isoforms
in the nematode . PLoS One. 2014;9(11):e113007. doi: 10.1371/journal.pone.0113007.</cite> [<a href="https://doi.org/10.1371/journal.pone.0113007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4244089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25423491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Characterization%20of%20N-acyl%20phosphatidylethanolamine-specific%20phospholipase-D%20isoforms%20in%20the%20nematode&amp;author=N.%20Harrison&amp;author=M.%20A.%20Lone&amp;author=T.%20K.%20Kaul&amp;author=P.%20R.%20Rodrigues&amp;author=I.%20V.%20Ogungbe&amp;volume=9&amp;issue=11&amp;publication_year=2014&amp;pages=e113007&amp;pmid=25423491&amp;doi=10.1371/journal.pone.0113007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Pastuhov S. I., Matsumoto K., Hisamoto N.. Endocannabinoid signaling regulates
regenerative axon navigation in via the GPCRs NPR-19 and NPR-32. Genes Cells. 2016;21(7):696–705. doi: 10.1111/gtc.12377.</cite> [<a href="https://doi.org/10.1111/gtc.12377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27193416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Cells&amp;title=Endocannabinoid%20signaling%20regulates%20regenerative%20axon%20navigation%20in%20via%20the%20GPCRs%20NPR-19%20and%20NPR-32&amp;author=S.%20I.%20Pastuhov&amp;author=K.%20Matsumoto&amp;author=N.%20Hisamoto&amp;volume=21&amp;issue=7&amp;publication_year=2016&amp;pages=696-705&amp;pmid=27193416&amp;doi=10.1111/gtc.12377&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Izrayelit Y., Robinette S. L., Bose N., von Reuss S. H., Schroeder F. C.. 2D NMR-based
metabolomics uncovers interactions between
conserved biochemical pathways in the model organism . ACS
Chem. Biol. 2013;8(2):314–319. doi: 10.1021/cb3004644.</cite> [<a href="https://doi.org/10.1021/cb3004644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3574178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23163760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Chem.%20Biol.&amp;title=2D%20NMR-based%20metabolomics%20uncovers%20interactions%20between%20conserved%20biochemical%20pathways%20in%20the%20model%20organism&amp;author=Y.%20Izrayelit&amp;author=S.%20L.%20Robinette&amp;author=N.%20Bose&amp;author=S.%20H.%20von%20Reuss&amp;author=F.%20C.%20Schroeder&amp;volume=8&amp;issue=2&amp;publication_year=2013&amp;pages=314-319&amp;pmid=23163760&amp;doi=10.1021/cb3004644&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Artyukhin A. B., Zhang Y. K., Akagi A. E., Panda O., Sternberg P. W., Schroeder F. C.. Metabolomic ″Dark Matter″ Dependent on
Peroxisomal β-Oxidation in . J. Am. Chem. Soc. 2018;140(8):2841–2852. doi: 10.1021/jacs.7b11811.</cite> [<a href="https://doi.org/10.1021/jacs.7b11811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5890438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29401383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Metabolomic%20%E2%80%B3Dark%20Matter%E2%80%B3%20Dependent%20on%20Peroxisomal%20%CE%B2-Oxidation%20in&amp;author=A.%20B.%20Artyukhin&amp;author=Y.%20K.%20Zhang&amp;author=A.%20E.%20Akagi&amp;author=O.%20Panda&amp;author=P.%20W.%20Sternberg&amp;volume=140&amp;issue=8&amp;publication_year=2018&amp;pages=2841-2852&amp;pmid=29401383&amp;doi=10.1021/jacs.7b11811&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Bouagnon A. D., Lin L., Srivastava S., Liu C. C., Panda O., Schroeder F. C., Srinivasan S., Ashrafi K.. Intestinal peroxisomal fatty acid
β-oxidation regulates neural serotonin signaling through a feedback
mechanism. PLoS Biol. 2019;17(12):e3000242. doi: 10.1371/journal.pbio.3000242.</cite> [<a href="https://doi.org/10.1371/journal.pbio.3000242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6917301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31805041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&amp;title=Intestinal%20peroxisomal%20fatty%20acid%20%CE%B2-oxidation%20regulates%20neural%20serotonin%20signaling%20through%20a%20feedback%20mechanism&amp;author=A.%20D.%20Bouagnon&amp;author=L.%20Lin&amp;author=S.%20Srivastava&amp;author=C.%20C.%20Liu&amp;author=O.%20Panda&amp;volume=17&amp;issue=12&amp;publication_year=2019&amp;pages=e3000242&amp;pmid=31805041&amp;doi=10.1371/journal.pbio.3000242&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<strong>a</strong> <cite>Félix M.-A., Braendle C., Cutter A. D.. A streamlined
system for species diagnosis in Caenorhabditis (Nematoda: Rhabditidae)
with name designations for 15 distinct biological species. PLoS One. 2014;9(4):e94723.  doi: 10.1371/journal.pone.0094723.</cite> [<a href="https://doi.org/10.1371/journal.pone.0094723" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3984244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24727800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=A%20streamlined%20system%20for%20species%20diagnosis%20in%20Caenorhabditis%20(Nematoda:%20Rhabditidae)%20with%20name%20designations%20for%2015%20distinct%20biological%20species&amp;author=M.-A.%20F%C3%A9lix&amp;author=C.%20Braendle&amp;author=A.%20D.%20Cutter&amp;volume=9&amp;issue=4&amp;publication_year=2014&amp;pages=e94723&amp;pmid=24727800&amp;doi=10.1371/journal.pone.0094723&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]; <cite>Erratum in:</cite>; <strong>b</strong> <cite>Félix M.-A., Braendle C., Cutter A. D.. PLoS One. 2015;10(3):e0118327. doi: 10.1371/journal.pone.0118327.</cite> [<a href="https://doi.org/10.1371/journal.pone.0118327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4351106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25742442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;author=M.-A.%20F%C3%A9lix&amp;author=C.%20Braendle&amp;author=A.%20D.%20Cutter&amp;volume=10&amp;issue=3&amp;publication_year=2015&amp;pages=e0118327&amp;pmid=25742442&amp;doi=10.1371/journal.pone.0118327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Stuart L. J., Buck J. P., Tremblay A. E., Buist P. H.. Configurational
analysis of cyclopropyl fatty acids isolated from . Org. Lett. 2006;8(1):79–81. doi: 10.1021/ol052550d.</cite> [<a href="https://doi.org/10.1021/ol052550d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16381572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&amp;title=Configurational%20analysis%20of%20cyclopropyl%20fatty%20acids%20isolated%20from&amp;author=L.%20J.%20Stuart&amp;author=J.%20P.%20Buck&amp;author=A.%20E.%20Tremblay&amp;author=P.%20H.%20Buist&amp;volume=8&amp;issue=1&amp;publication_year=2006&amp;pages=79-81&amp;pmid=16381572&amp;doi=10.1021/ol052550d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Hodgkin J., Horvitz H. R., Brenner S.. Nondisjunction
Mutants of the Nematode . Genetics. 1979;91(1):67–94. doi: 10.1093/genetics/91.1.67.</cite> [<a href="https://doi.org/10.1093/genetics/91.1.67" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1213932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17248881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics.&amp;title=Nondisjunction%20Mutants%20of%20the%20Nematode&amp;author=J.%20Hodgkin&amp;author=H.%20R.%20Horvitz&amp;author=S.%20Brenner&amp;volume=91&amp;issue=1&amp;publication_year=1979&amp;pages=67-94&amp;pmid=17248881&amp;doi=10.1093/genetics/91.1.67&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Choe A., von Reuss S. H., Kogan D., Gasser R. B., Platzer E. G., Schroeder F. C., Sternberg P. W.. Ascaroside signaling is widely conserved
among nematodes. Curr. Biol. 2012;22(9):772–780. doi: 10.1016/j.cub.2012.03.024.</cite> [<a href="https://doi.org/10.1016/j.cub.2012.03.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3360977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22503501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=Ascaroside%20signaling%20is%20widely%20conserved%20among%20nematodes&amp;author=A.%20Choe&amp;author=S.%20H.%20von%20Reuss&amp;author=D.%20Kogan&amp;author=R.%20B.%20Gasser&amp;author=E.%20G.%20Platzer&amp;volume=22&amp;issue=9&amp;publication_year=2012&amp;pages=772-780&amp;pmid=22503501&amp;doi=10.1016/j.cub.2012.03.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Dong C., Reilly D. K., Bergame C., Dolke F., Srinivasan J., von Reuss S. H.. Comparative
Ascaroside Profiling of Caenorhabditis Exometabolomes
Reveals Species-Specific (ω) and (ω -
2)-Hydroxylation Downstream of Peroxisomal β-Oxidation. J. Org. Chem. 2018;83(13):7109–7120. doi: 10.1021/acs.joc.8b00094.</cite> [<a href="https://doi.org/10.1021/acs.joc.8b00094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6035057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29480728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Org.%20Chem.&amp;title=Comparative%20Ascaroside%20Profiling%20of%20Caenorhabditis%20Exometabolomes%20Reveals%20Species-Specific%20(%CF%89)%20and%20(%CF%89%20-%202)-Hydroxylation%20Downstream%20of%20Peroxisomal%20%CE%B2-Oxidation&amp;author=C.%20Dong&amp;author=D.%20K.%20Reilly&amp;author=C.%20Bergame&amp;author=F.%20Dolke&amp;author=J.%20Srinivasan&amp;volume=83&amp;issue=13&amp;publication_year=2018&amp;pages=7109-7120&amp;pmid=29480728&amp;doi=10.1021/acs.joc.8b00094&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Perez C. L., Van Gilst M. R.. A 13C isotope
labeling strategy reveals the influence
of insulin signaling on lipogenesis in . Cell Metab. 2008;8(3):266–274. doi: 10.1016/j.cmet.2008.08.007.</cite> [<a href="https://doi.org/10.1016/j.cmet.2008.08.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18762027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab.&amp;title=A%2013C%20isotope%20labeling%20strategy%20reveals%20the%20influence%20of%20insulin%20signaling%20on%20lipogenesis%20in&amp;author=C.%20L.%20Perez&amp;author=M.%20R.%20Van%20Gilst&amp;volume=8&amp;issue=3&amp;publication_year=2008&amp;pages=266-274&amp;pmid=18762027&amp;doi=10.1016/j.cmet.2008.08.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Ratnappan R., Amrit F. R., Chen S. W., Gill H., Holden K., Ward J., Yamamoto K. R., Olsen C. P., Ghazi A.. Germline signals
deploy NHR-49 to modulate fatty-acid β-oxidation and desaturation
in somatic tissues of . PLoS Genet. 2014;10(12):e1004829. doi: 10.1371/journal.pgen.1004829.</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4256272/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25474470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Germline%20signals%20deploy%20NHR-49%20to%20modulate%20fatty-acid%20%CE%B2-oxidation%20and%20desaturation%20in%20somatic%20tissues%20of&amp;author=R.%20Ratnappan&amp;author=F.%20R.%20Amrit&amp;author=S.%20W.%20Chen&amp;author=H.%20Gill&amp;author=K.%20Holden&amp;volume=10&amp;issue=12&amp;publication_year=2014&amp;pages=e1004829&amp;pmid=25474470&amp;doi=10.1371/journal.pgen.1004829&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Dancy B. C. R., Chen S. W., Drechsler R., Gafken P. R., Olsen C. P.. 13C- and
15N-Labeling Strategies Combined with Mass Spectrometry Comprehensively
Quantify Phospholipid Dynamics in . PLoS One. 2015;10(11):e0141850. doi: 10.1371/journal.pone.0141850.</cite> [<a href="https://doi.org/10.1371/journal.pone.0141850" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4631354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26528916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One.&amp;title=13C-%20and%2015N-Labeling%20Strategies%20Combined%20with%20Mass%20Spectrometry%20Comprehensively%20Quantify%20Phospholipid%20Dynamics%20in&amp;author=B.%20C.%20R.%20Dancy&amp;author=S.%20W.%20Chen&amp;author=R.%20Drechsler&amp;author=P.%20R.%20Gafken&amp;author=C.%20P.%20Olsen&amp;volume=10&amp;issue=11&amp;publication_year=2015&amp;pages=e0141850&amp;pmid=26528916&amp;doi=10.1371/journal.pone.0141850&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Hoang K. L., Gerardo N. M., Morran L. T.. The effects
of on fitness after heat stress. Ecol. Evol. 2019;9(6):3491–3499. doi: 10.1002/ece3.4983.</cite> [<a href="https://doi.org/10.1002/ece3.4983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6434544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30962907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecol.%20Evol.&amp;title=The%20effects%20of%20on%20fitness%20after%20heat%20stress&amp;author=K.%20L.%20Hoang&amp;author=N.%20M.%20Gerardo&amp;author=L.%20T.%20Morran&amp;volume=9&amp;issue=6&amp;publication_year=2019&amp;pages=3491-3499&amp;pmid=30962907&amp;doi=10.1002/ece3.4983&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Nickels J.
D., Chatterjee S., Mostofian B., Stanley C. B., Ohl M., Zolnierczuk P., Schulz R., Myles D. A. A., Standaert R. F., Elkins J. G., Cheng X., Katsaras J..  Lipid Extract, A Branched-Chain Fatty Acid
Model Membrane. J. Phys. Chem. Lett. 2017;8(17):4214–4217. doi: 10.1021/acs.jpclett.7b01877.</cite> [<a href="https://doi.org/10.1021/acs.jpclett.7b01877" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28825491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20Lett.&amp;title=Lipid%20Extract,%20A%20Branched-Chain%20Fatty%20Acid%20Model%20Membrane&amp;author=J.%20D.%20Nickels&amp;author=S.%20Chatterjee&amp;author=B.%20Mostofian&amp;author=C.%20B.%20Stanley&amp;author=M.%20Ohl&amp;volume=8&amp;issue=17&amp;publication_year=2017&amp;pages=4214-4217&amp;pmid=28825491&amp;doi=10.1021/acs.jpclett.7b01877&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Dirksen P., Assié A., Zimmermann J., Zhang F., Tietje A. M., Marsh S. A., Félix M. A., Shapira M., Kaleta C., Schulenburg H., Samuel B. S.. CeMbio - The Microbiome Resource. G3:Genes, Genomes, Genet. 2020;10(9):3025–3039. doi: 10.1534/g3.120.401309.</cite> [<a href="https://doi.org/10.1534/g3.120.401309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7466993/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32669368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:Genes,%20Genomes,%20Genet.&amp;title=CeMbio%20-%20The%20Microbiome%20Resource&amp;author=P.%20Dirksen&amp;author=A.%20Assi%C3%A9&amp;author=J.%20Zimmermann&amp;author=F.%20Zhang&amp;author=A.%20M.%20Tietje&amp;volume=10&amp;issue=9&amp;publication_year=2020&amp;pages=3025-3039&amp;pmid=32669368&amp;doi=10.1534/g3.120.401309&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Zhu H., Shen H., Sewell A. K., Kniazeva M., Han M.. A novel sphingolipid-TORC1
pathway critically promotes postembryonic development in . Elife. 2013;2:e00429. doi: 10.7554/eLife.00429.</cite> [<a href="https://doi.org/10.7554/eLife.00429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3660743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23705068/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=A%20novel%20sphingolipid-TORC1%20pathway%20critically%20promotes%20postembryonic%20development%20in&amp;author=H.%20Zhu&amp;author=H.%20Shen&amp;author=A.%20K.%20Sewell&amp;author=M.%20Kniazeva&amp;author=M.%20Han&amp;volume=2&amp;publication_year=2013&amp;pages=e00429&amp;pmid=23705068&amp;doi=10.7554/eLife.00429&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Zhu H., Sewell A. K., Han M.. Intestinal
apical polarity mediates
regulation of TORC1 by glucosylceramide in . Genes Dev. 2015;29(12):1218–1223. doi: 10.1101/gad.263483.115.</cite> [<a href="https://doi.org/10.1101/gad.263483.115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4495394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26109047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Intestinal%20apical%20polarity%20mediates%20regulation%20of%20TORC1%20by%20glucosylceramide%20in&amp;author=H.%20Zhu&amp;author=A.%20K.%20Sewell&amp;author=M.%20Han&amp;volume=29&amp;issue=12&amp;publication_year=2015&amp;pages=1218-1223&amp;pmid=26109047&amp;doi=10.1101/gad.263483.115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Kniazeva M., Zhu H., Sewell A. K., Han M.. A Lipid-TORC1 Pathway Promotes Neuronal
Development and Foraging Behavior under Both Fed and Fasted Conditions
in . Dev. Cell. 2015;33(3):260–271. doi: 10.1016/j.devcel.2015.02.015.</cite> [<a href="https://doi.org/10.1016/j.devcel.2015.02.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25892013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Cell&amp;title=A%20Lipid-TORC1%20Pathway%20Promotes%20Neuronal%20Development%20and%20Foraging%20Behavior%20under%20Both%20Fed%20and%20Fasted%20Conditions%20in&amp;author=M.%20Kniazeva&amp;author=H.%20Zhu&amp;author=A.%20K.%20Sewell&amp;author=M.%20Han&amp;volume=33&amp;issue=3&amp;publication_year=2015&amp;pages=260-271&amp;pmid=25892013&amp;doi=10.1016/j.devcel.2015.02.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Sánchez R. R., Bandi S., Scheidt M.-D., Laaroussi H., Fox B. W., Ishida Y., Glauser G., Sutour S., von Reuss S. H.. iso-Fatty Acid Metabolism in ’ Ceramide Biosynthesis. Helv. Chim. Acta. 2023;106(11):e202300131. doi: 10.1002/hlca.202300131.</cite> [<a href="https://doi.org/10.1002/hlca.202300131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Helv.%20Chim.%20Acta&amp;title=iso-Fatty%20Acid%20Metabolism%20in%20%E2%80%99%20Ceramide%20Biosynthesis&amp;author=R.%20R.%20S%C3%A1nchez&amp;author=S.%20Bandi&amp;author=M.-D.%20Scheidt&amp;author=H.%20Laaroussi&amp;author=B.%20W.%20Fox&amp;volume=106&amp;issue=11&amp;publication_year=2023&amp;pages=e202300131&amp;doi=10.1002/hlca.202300131&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Hannich J. T., Mellal D., Feng S., Zumbuehl A., Riezman H.. Structure
and conserved function of iso-branched sphingoid bases from the nematode . Chem.
Sci. 2017;8(5):3676–3686. doi: 10.1039/C6SC04831E.</cite> [<a href="https://doi.org/10.1039/C6SC04831E" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6094178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30155209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Sci.&amp;title=Structure%20and%20conserved%20function%20of%20iso-branched%20sphingoid%20bases%20from%20the%20nematode&amp;author=J.%20T.%20Hannich&amp;author=D.%20Mellal&amp;author=S.%20Feng&amp;author=A.%20Zumbuehl&amp;author=H.%20Riezman&amp;volume=8&amp;issue=5&amp;publication_year=2017&amp;pages=3676-3686&amp;pmid=30155209&amp;doi=10.1039/C6SC04831E&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Tanno H., Sassa T., Sawai M., Kihara A.. Production of branched-chain
very-long-chain fatty acids by fatty acid elongases and their tissue
distribution in mammals. Biochim. Biophys. Acta,
Mol. Cell Biol. Lipids. 2021;1866(1):158842. doi: 10.1016/j.bbalip.2020.158842.</cite> [<a href="https://doi.org/10.1016/j.bbalip.2020.158842" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33069870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta,%20Mol.%20Cell%20Biol.%20Lipids&amp;title=Production%20of%20branched-chain%20very-long-chain%20fatty%20acids%20by%20fatty%20acid%20elongases%20and%20their%20tissue%20distribution%20in%20mammals&amp;author=H.%20Tanno&amp;author=T.%20Sassa&amp;author=M.%20Sawai&amp;author=A.%20Kihara&amp;volume=1866&amp;issue=1&amp;publication_year=2021&amp;pages=158842&amp;pmid=33069870&amp;doi=10.1016/j.bbalip.2020.158842&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>Kniazeva M., Crawford Q. T., Seiber M., Wang C. Y., Han M.. Monomethyl
branched-chain fatty acids play an essential role in development. PLoS Biol. 2004;2:E257. doi: 10.1371/journal.pbio.0020257.</cite> [<a href="https://doi.org/10.1371/journal.pbio.0020257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC514883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15340492/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&amp;title=Monomethyl%20branched-chain%20fatty%20acids%20play%20an%20essential%20role%20in%20development&amp;author=M.%20Kniazeva&amp;author=Q.%20T.%20Crawford&amp;author=M.%20Seiber&amp;author=C.%20Y.%20Wang&amp;author=M.%20Han&amp;volume=2&amp;publication_year=2004&amp;pages=E257&amp;pmid=15340492&amp;doi=10.1371/journal.pbio.0020257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>Kniazeva M., Euler T., Han M.. A branched-chain
fatty acid is involved
in post-embryonic growth control in parallel to the insulin receptor
pathway and its biosynthesis is feedback-regulated in . Genes Dev. 2008;22(15):2102–2110. doi: 10.1101/gad.1692008.</cite> [<a href="https://doi.org/10.1101/gad.1692008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2492746/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18676815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=A%20branched-chain%20fatty%20acid%20is%20involved%20in%20post-embryonic%20growth%20control%20in%20parallel%20to%20the%20insulin%20receptor%20pathway%20and%20its%20biosynthesis%20is%20feedback-regulated%20in&amp;author=M.%20Kniazeva&amp;author=T.%20Euler&amp;author=M.%20Han&amp;volume=22&amp;issue=15&amp;publication_year=2008&amp;pages=2102-2110&amp;pmid=18676815&amp;doi=10.1101/gad.1692008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Richardson M. B., Williams S. J.. A practical synthesis of long-chain iso-fatty acids
(iso-C12-C19) and related natural products. Beilstein J. Org. Chem. 2013;9:1807–1812. doi: 10.3762/bjoc.9.210.</cite> [<a href="https://doi.org/10.3762/bjoc.9.210" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3778388/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24062846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Beilstein%20J.%20Org.%20Chem.&amp;title=A%20practical%20synthesis%20of%20long-chain%20iso-fatty%20acids%20(iso-C12-C19)%20and%20related%20natural%20products&amp;author=M.%20B.%20Richardson&amp;author=S.%20J.%20Williams&amp;volume=9&amp;publication_year=2013&amp;pages=1807-1812&amp;pmid=24062846&amp;doi=10.3762/bjoc.9.210&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Golden J. W., Riddle D. L.. A gene affecting production of the dauer-inducing pheromone. Mol. Gen. Genet. 1985;198(3):534–536. doi: 10.1007/BF00332953.</cite> [<a href="https://doi.org/10.1007/BF00332953" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3859733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Gen.%20Genet.&amp;title=A%20gene%20affecting%20production%20of%20the%20dauer-inducing%20pheromone&amp;author=J.%20W.%20Golden&amp;author=D.%20L.%20Riddle&amp;volume=198&amp;issue=3&amp;publication_year=1985&amp;pages=534-536&amp;pmid=3859733&amp;doi=10.1007/BF00332953&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Butcher R. A., Ragains J. R., Li W., Ruvkun G., Clardy J., Mak H. Y.. Biosynthesis of the dauer pheromone. Proc. Natl. Acad. Sci. U.S.A. 2009;106(6):1875–1879. doi: 10.1073/pnas.0810338106.</cite> [<a href="https://doi.org/10.1073/pnas.0810338106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2631283/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19174521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Biosynthesis%20of%20the%20dauer%20pheromone&amp;author=R.%20A.%20Butcher&amp;author=J.%20R.%20Ragains&amp;author=W.%20Li&amp;author=G.%20Ruvkun&amp;author=J.%20Clardy&amp;volume=106&amp;issue=6&amp;publication_year=2009&amp;pages=1875-1879&amp;pmid=19174521&amp;doi=10.1073/pnas.0810338106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>Joo H. J., Yim Y. H., Jeong P. Y., Jin Y. X., Lee J. E., Kim H., Jeong S. K., Chitwood D. J., Paik Y. K..  utilizes dauer pheromone biosynthesis to dispose
of toxic peroxisomal fatty acids for cellular homoeostasis. Biochem. J. 2009;422(1):61–71. doi: 10.1042/BJ20090513.</cite> [<a href="https://doi.org/10.1042/BJ20090513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19496754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=utilizes%20dauer%20pheromone%20biosynthesis%20to%20dispose%20of%20toxic%20peroxisomal%20fatty%20acids%20for%20cellular%20homoeostasis&amp;author=H.%20J.%20Joo&amp;author=Y.%20H.%20Yim&amp;author=P.%20Y.%20Jeong&amp;author=Y.%20X.%20Jin&amp;author=J.%20E.%20Lee&amp;volume=422&amp;issue=1&amp;publication_year=2009&amp;pages=61-71&amp;pmid=19496754&amp;doi=10.1042/BJ20090513&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref58">
<cite>Mak H. Y., Nelson L. S., Basson M., Johnson C. D., Ruvkun G.. Polygenic
control of fat
storage. Nat. Genet. 2006;38(3):363–368. doi: 10.1038/ng1739.</cite> [<a href="https://doi.org/10.1038/ng1739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16462744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=Polygenic%20control%20of%20fat%20storage&amp;author=H.%20Y.%20Mak&amp;author=L.%20S.%20Nelson&amp;author=M.%20Basson&amp;author=C.%20D.%20Johnson&amp;author=G.%20Ruvkun&amp;volume=38&amp;issue=3&amp;publication_year=2006&amp;pages=363-368&amp;pmid=16462744&amp;doi=10.1038/ng1739&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref59">
<cite>von
Reuss S. H., Bose N., Srinivasan J., Yim J. J., Judkins J. C., Sternberg P. W., Schroeder F. C.. Comparative Metabolomics Reveals Biogenesis of Ascarosides,
a Modular Library of Small-Molecule Signals in . J. Am. Chem. Soc. 2012;134(3):1817–1824. doi: 10.1021/ja210202y.</cite> [<a href="https://doi.org/10.1021/ja210202y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3269134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22239548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Comparative%20Metabolomics%20Reveals%20Biogenesis%20of%20Ascarosides,%20a%20Modular%20Library%20of%20Small-Molecule%20Signals%20in&amp;author=S.%20H.%20von%20Reuss&amp;author=N.%20Bose&amp;author=J.%20Srinivasan&amp;author=J.%20J.%20Yim&amp;author=J.%20C.%20Judkins&amp;volume=134&amp;issue=3&amp;publication_year=2012&amp;pages=1817-1824&amp;pmid=22239548&amp;doi=10.1021/ja210202y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref60">
<cite>von
Reuss S. H., Dolke F., Dong C.. Ascaroside Profiling
of Using Gas
Chromatography-Electron Ionization Mass Spectrometry. Anal. Chem. 2017;89(19):10570–10577. doi: 10.1021/acs.analchem.7b02803.</cite> [<a href="https://doi.org/10.1021/acs.analchem.7b02803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28866881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Chem.&amp;title=Ascaroside%20Profiling%20of%20Using%20Gas%20Chromatography-Electron%20Ionization%20Mass%20Spectrometry&amp;author=S.%20H.%20von%20Reuss&amp;author=F.%20Dolke&amp;author=C.%20Dong&amp;volume=89&amp;issue=19&amp;publication_year=2017&amp;pages=10570-10577&amp;pmid=28866881&amp;doi=10.1021/acs.analchem.7b02803&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref61">
<cite>Artyukhin A. B., Yim J. J., Cheong
Cheong M., Avery L.. Starvation-induced
collective behavior in . Sci. Rep. 2015;5:10647. doi: 10.1038/srep10647.</cite> [<a href="https://doi.org/10.1038/srep10647" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4445038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26013573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Starvation-induced%20collective%20behavior%20in&amp;author=A.%20B.%20Artyukhin&amp;author=J.%20J.%20Yim&amp;author=M.%20Cheong%20Cheong&amp;author=L.%20Avery&amp;volume=5&amp;publication_year=2015&amp;pages=10647&amp;pmid=26013573&amp;doi=10.1038/srep10647&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref62">
<cite>Juntunen J., Huuskonen J., Laine K., Niemi R., Taipale H., Nevalainen T., Pate D. W., Järvinen T.. Anandamide
prodrugs. 1. Water-soluble phosphate esters of arachidonylethanolamide
and R-methanandamide. Eur. J. Pharm. Sci. 2003;19(1):37–43. doi: 10.1016/S0928-0987(03)00044-7.</cite> [<a href="https://doi.org/10.1016/S0928-0987(03)00044-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12729860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharm.%20Sci.&amp;title=Anandamide%20prodrugs.%201.%20Water-soluble%20phosphate%20esters%20of%20arachidonylethanolamide%20and%20R-methanandamide&amp;author=J.%20Juntunen&amp;author=J.%20Huuskonen&amp;author=K.%20Laine&amp;author=R.%20Niemi&amp;author=H.%20Taipale&amp;volume=19&amp;issue=1&amp;publication_year=2003&amp;pages=37-43&amp;pmid=12729860&amp;doi=10.1016/S0928-0987(03)00044-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref63">
<cite>Dolke F., Dong C., Bandi S., Paetz C., Glauser G., von Reuß S. H.. Ascaroside Signaling in the Bacterivorous Nematode Encodes the Growth Phase of
Its Bacterial Food Source. Org. Lett. 2019;21(15):5832–5837. doi: 10.1021/acs.orglett.9b01914.</cite> [<a href="https://doi.org/10.1021/acs.orglett.9b01914" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31305087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Org.%20Lett.&amp;title=Ascaroside%20Signaling%20in%20the%20Bacterivorous%20Nematode%20Encodes%20the%20Growth%20Phase%20of%20Its%20Bacterial%20Food%20Source&amp;author=F.%20Dolke&amp;author=C.%20Dong&amp;author=S.%20Bandi&amp;author=C.%20Paetz&amp;author=G.%20Glauser&amp;volume=21&amp;issue=15&amp;publication_year=2019&amp;pages=5832-5837&amp;pmid=31305087&amp;doi=10.1021/acs.orglett.9b01914&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref64">
<cite>Fox B. W., Helf M. J., Burkhardt R. N., Artyukhin A. B., Curtis B. J., Palomino D. F., Schroeder A. F., Chaturbedi A., Tauffenberger A., Wrobel C. J. J., Zhang Y. K., Lee S. S., Schroeder F. C.. Evolutionarily
related host and microbial
pathways regulate fat desaturation in . Nat. Commun. 2024;15(1):1520. doi: 10.1038/s41467-024-45782-2.</cite> [<a href="https://doi.org/10.1038/s41467-024-45782-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10876521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38374083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Evolutionarily%20related%20host%20and%20microbial%20pathways%20regulate%20fat%20desaturation%20in&amp;author=B.%20W.%20Fox&amp;author=M.%20J.%20Helf&amp;author=R.%20N.%20Burkhardt&amp;author=A.%20B.%20Artyukhin&amp;author=B.%20J.%20Curtis&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=1520&amp;pmid=38374083&amp;doi=10.1038/s41467-024-45782-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref65">
<cite>Kaul T. K., Reis Rodrigues P., Ogungbe I. V., Kapahi P., Gill M. S.. Bacterial
fatty acids enhance recovery from the dauer larva in . PLoS
One. 2014;9(1):e86979. doi: 10.1371/journal.pone.0086979.</cite> [<a href="https://doi.org/10.1371/journal.pone.0086979" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3901721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24475206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Bacterial%20fatty%20acids%20enhance%20recovery%20from%20the%20dauer%20larva%20in&amp;author=T.%20K.%20Kaul&amp;author=P.%20Reis%20Rodrigues&amp;author=I.%20V.%20Ogungbe&amp;author=P.%20Kapahi&amp;author=M.%20S.%20Gill&amp;volume=9&amp;issue=1&amp;publication_year=2014&amp;pages=e86979&amp;pmid=24475206&amp;doi=10.1371/journal.pone.0086979&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref66">
<cite>Grogan D. W., Cronan J. E. Jr. Cyclopropane ring formation in membrane
lipids of bacteria. Microbiol. Mol. Biol. Rev. 1997;61(4):429–441. doi: 10.1128/mmbr.61.4.429-441.1997.</cite> [<a href="https://doi.org/10.1128/mmbr.61.4.429-441.1997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC232619/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9409147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;title=Cyclopropane%20ring%20formation%20in%20membrane%20lipids%20of%20bacteria&amp;author=D.%20W.%20Grogan&amp;author=J.%20E.%20Cronan&amp;volume=61&amp;issue=4&amp;publication_year=1997&amp;pages=429-441&amp;pmid=9409147&amp;doi=10.1128/mmbr.61.4.429-441.1997&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref67">
<cite>Cronan J. E.. Bacterial
membrane lipids: where do we stand. Annu. Rev.
Microbiol. 2003;57:203–224. doi: 10.1146/annurev.micro.57.030502.090851.</cite> [<a href="https://doi.org/10.1146/annurev.micro.57.030502.090851" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14527277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Microbiol.&amp;title=Bacterial%20membrane%20lipids:%20where%20do%20we%20stand&amp;author=J.%20E.%20Cronan&amp;volume=57&amp;publication_year=2003&amp;pages=203-224&amp;pmid=14527277&amp;doi=10.1146/annurev.micro.57.030502.090851&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref68">
<cite>

Stiernagle, T.

Maintenance
of <em>C. elegans</em>
. In C. elegans
Research Community; WormBook, 2006. 
<a href="http://www.wormbook.org/chapters/www_strainmaintain/strainmaintain.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.wormbook.org/chapters/www_strainmaintain/strainmaintain.html</a>.</cite> [<a href="https://doi.org/10.1895/wormbook.1.101.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4781397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18050451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=C.%20elegans%20Research%20Community&amp;publication_year=2006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref69">
<cite>Matyash V., Liebisch G., Kurzchalia T. V., Shevchenko A., Schwudke D.. Lipid extraction by methyl-tert-butyl
ether for high-throughput lipidomics. J. Lipid
Res. 2008;49(5):1137–1146. doi: 10.1194/jlr.D700041-JLR200.</cite> [<a href="https://doi.org/10.1194/jlr.D700041-JLR200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2311442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18281723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Lipid%20Res.&amp;title=Lipid%20extraction%20by%20methyl-tert-butyl%20ether%20for%20high-throughput%20lipidomics&amp;author=V.%20Matyash&amp;author=G.%20Liebisch&amp;author=T.%20V.%20Kurzchalia&amp;author=A.%20Shevchenko&amp;author=D.%20Schwudke&amp;volume=49&amp;issue=5&amp;publication_year=2008&amp;pages=1137-1146&amp;pmid=18281723&amp;doi=10.1194/jlr.D700041-JLR200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371488/bin/bg5c00012_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00012_si_001.pdf</a><sup> (10.6MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Bio &amp; Med Chem Au are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsbiomedchemau.5c00012"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/bg5c00012.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371488/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371488/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371488%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371488/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371488/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371488/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371488/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371488/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371488/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Ll3olOeUhhLxmWoEboKl23rd5BvoZnlUi2U1gmZiw6t4Up2D6Ikn9VysQ69nYeUc">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
