
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            An alternative cytoplasmic SFPQ isoform with reduced phase separation potential is up-regulated in ALS - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A0C38AF1CA9305A0C30055339158.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sciadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372870/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Science Advances">
<meta name="citation_title" content="An alternative cytoplasmic SFPQ isoform with reduced phase separation potential is up-regulated in ALS">
<meta name="citation_author" content="Jacob Neeves">
<meta name="citation_author_institution" content="Department of Neuromuscular Diseases, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.">
<meta name="citation_author_institution" content="The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.">
<meta name="citation_author" content="Marija Petrić Howe">
<meta name="citation_author_institution" content="Department of Neuromuscular Diseases, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.">
<meta name="citation_author_institution" content="The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.">
<meta name="citation_author" content="Oliver J Ziff">
<meta name="citation_author_institution" content="Department of Neuromuscular Diseases, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.">
<meta name="citation_author_institution" content="The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.">
<meta name="citation_author" content="Beth Callaghan">
<meta name="citation_author_institution" content="The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.">
<meta name="citation_author" content="Daniel Jutzi">
<meta name="citation_author_institution" content="UK Dementia Research Institute at King’s College London, London, UK.">
<meta name="citation_author_institution" content="Department of Basic and Clinical Neuroscience, Institute of Psychiatry, Psychology &amp; Neuroscience, King’s College London, 125 Coldharbour Lane, London SE5 9NU, UK.">
<meta name="citation_author" content="Koustav Pal">
<meta name="citation_author_institution" content="Department of Neuromuscular Diseases, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.">
<meta name="citation_author_institution" content="The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.">
<meta name="citation_author" content="Theodoros I Roumeliotis">
<meta name="citation_author_institution" content="Functional Proteomics team, The Institute of Cancer Research, 237 Fulham Road, London SW3 6JB, UK.">
<meta name="citation_author" content="Jyoti Choudhary">
<meta name="citation_author_institution" content="Functional Proteomics team, The Institute of Cancer Research, 237 Fulham Road, London SW3 6JB, UK.">
<meta name="citation_author" content="Adrian M Isaacs">
<meta name="citation_author_institution" content="UK Dementia Research Institute at University College London, London, UK.">
<meta name="citation_author_institution" content="Department of Neurodegenerative Disease, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.">
<meta name="citation_author" content="Frank Rigo">
<meta name="citation_author_institution" content="Ionis Pharmaceuticals, Carlsbad, CA 92010, USA.">
<meta name="citation_author" content="C Frank Bennett">
<meta name="citation_author_institution" content="Ionis Pharmaceuticals, Carlsbad, CA 92010, USA.">
<meta name="citation_author" content="Marc-David Ruepp">
<meta name="citation_author_institution" content="UK Dementia Research Institute at King’s College London, London, UK.">
<meta name="citation_author_institution" content="Department of Basic and Clinical Neuroscience, Institute of Psychiatry, Psychology &amp; Neuroscience, King’s College London, 125 Coldharbour Lane, London SE5 9NU, UK.">
<meta name="citation_author" content="Rickie Patani">
<meta name="citation_author_institution" content="Department of Neuromuscular Diseases, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.">
<meta name="citation_author_institution" content="The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="11">
<meta name="citation_issue" content="34">
<meta name="citation_firstpage" content="eadt4814">
<meta name="citation_doi" content="10.1126/sciadv.adt4814">
<meta name="citation_pmid" content="40845103">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372870/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372870/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372870/pdf/sciadv.adt4814.pdf">
<meta name="description" content="Splicing factor proline- and glutamine-rich (SFPQ) is an RNA binding protein that broadly regulates RNA metabolism. Although its nuclear roles are well studied, evidence of SFPQ’s cytoplasmic functionality is emerging. Altered expression and ...">
<meta name="og:title" content="An alternative cytoplasmic SFPQ isoform with reduced phase separation potential is up-regulated in ALS">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Splicing factor proline- and glutamine-rich (SFPQ) is an RNA binding protein that broadly regulates RNA metabolism. Although its nuclear roles are well studied, evidence of SFPQ’s cytoplasmic functionality is emerging. Altered expression and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372870/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12372870">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1126/sciadv.adt4814"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/sciadv.adt4814.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372870%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12372870/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12372870/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372870/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sciadv.gif" alt="Science Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Science Advances" title="Link to Science Advances" shape="default" href="http://advances.sciencemag.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Adv</button></div>. 2025 Aug 22;11(34):eadt4814. doi: <a href="https://doi.org/10.1126/sciadv.adt4814" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1126/sciadv.adt4814</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>An alternative cytoplasmic SFPQ isoform with reduced phase separation potential is up-regulated in ALS</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Neeves%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Jacob Neeves</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Jacob Neeves</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Neuromuscular Diseases, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.</div>
<div class="p">
<sup><sup>2</sup></sup>The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Project administration, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Neeves%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jacob Neeves</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Petri%C4%87%20Howe%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Marija Petrić Howe</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Marija Petrić Howe</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Neuromuscular Diseases, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.</div>
<div class="p">
<sup><sup>2</sup></sup>The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.</div>
<div>Investigation, Methodology, Validation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Petri%C4%87%20Howe%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marija Petrić Howe</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ziff%20OJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Oliver J Ziff</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Oliver J Ziff</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Neuromuscular Diseases, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.</div>
<div class="p">
<sup><sup>2</sup></sup>The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.</div>
<div>Data curation, Formal analysis, Investigation, Methodology, Resources, Software, Validation, Visualization, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ziff%20OJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Oliver J Ziff</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Callaghan%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Beth Callaghan</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Beth Callaghan</span></h3>
<div class="p">
<sup><sup>2</sup></sup>The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.</div>
<div>Investigation, Validation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Callaghan%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Beth Callaghan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jutzi%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Daniel Jutzi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Daniel Jutzi</span></h3>
<div class="p">
<sup><sup>3</sup></sup>UK Dementia Research Institute at King’s College London, London, UK.</div>
<div class="p">
<sup><sup>4</sup></sup>Department of Basic and Clinical Neuroscience, Institute of Psychiatry, Psychology &amp; Neuroscience, King’s College London, 125 Coldharbour Lane, London SE5 9NU, UK.</div>
<div>Investigation, Methodology, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jutzi%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel Jutzi</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pal%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Koustav Pal</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Koustav Pal</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Neuromuscular Diseases, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.</div>
<div class="p">
<sup><sup>2</sup></sup>The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.</div>
<div>Formal analysis, Software</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pal%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Koustav Pal</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roumeliotis%20TI%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Theodoros I Roumeliotis</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Theodoros I Roumeliotis</span></h3>
<div class="p">
<sup><sup>5</sup></sup>Functional Proteomics team, The Institute of Cancer Research, 237 Fulham Road, London SW3 6JB, UK.</div>
<div>Data curation, Formal analysis, Methodology, Software, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roumeliotis%20TI%22%5BAuthor%5D" class="usa-link"><span class="name western">Theodoros I Roumeliotis</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Choudhary%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jyoti Choudhary</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jyoti Choudhary</span></h3>
<div class="p">
<sup><sup>5</sup></sup>Functional Proteomics team, The Institute of Cancer Research, 237 Fulham Road, London SW3 6JB, UK.</div>
<div>Formal analysis</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Choudhary%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jyoti Choudhary</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Isaacs%20AM%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Adrian M Isaacs</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Adrian M Isaacs</span></h3>
<div class="p">
<sup><sup>6</sup></sup>UK Dementia Research Institute at University College London, London, UK.</div>
<div class="p">
<sup><sup>7</sup></sup>Department of Neurodegenerative Disease, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.</div>
<div>Resources, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Isaacs%20AM%22%5BAuthor%5D" class="usa-link"><span class="name western">Adrian M Isaacs</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rigo%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Frank Rigo</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Frank Rigo</span></h3>
<div class="p">
<sup><sup>8</sup></sup>Ionis Pharmaceuticals, Carlsbad, CA 92010, USA.</div>
<div>Resources</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rigo%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Frank Rigo</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bennett%20CF%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">C Frank Bennett</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">C Frank Bennett</span></h3>
<div class="p">
<sup><sup>8</sup></sup>Ionis Pharmaceuticals, Carlsbad, CA 92010, USA.</div>
<div>Resources</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bennett%20CF%22%5BAuthor%5D" class="usa-link"><span class="name western">C Frank Bennett</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruepp%20MD%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Marc-David Ruepp</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Marc-David Ruepp</span></h3>
<div class="p">
<sup><sup>3</sup></sup>UK Dementia Research Institute at King’s College London, London, UK.</div>
<div class="p">
<sup><sup>4</sup></sup>Department of Basic and Clinical Neuroscience, Institute of Psychiatry, Psychology &amp; Neuroscience, King’s College London, 125 Coldharbour Lane, London SE5 9NU, UK.</div>
<div>Funding acquisition, Resources, Supervision, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ruepp%20MD%22%5BAuthor%5D" class="usa-link"><span class="name western">Marc-David Ruepp</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patani%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Rickie Patani</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Rickie Patani</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Neuromuscular Diseases, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.</div>
<div class="p">
<sup><sup>2</sup></sup>The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.</div>
<div>Conceptualization, Funding acquisition, Methodology, Project administration, Resources, Supervision, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patani%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rickie Patani</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>*,</sup><sup>†,</sup><sup>‡,</sup><sup>§</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>Department of Neuromuscular Diseases, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.</div>
<div id="aff2">
<sup><sup>2</sup></sup>The Francis Crick Institute, 1 Midland Road, London NW1 1AT, UK.</div>
<div id="aff3">
<sup><sup>3</sup></sup>UK Dementia Research Institute at King’s College London, London, UK.</div>
<div id="aff4">
<sup><sup>4</sup></sup>Department of Basic and Clinical Neuroscience, Institute of Psychiatry, Psychology &amp; Neuroscience, King’s College London, 125 Coldharbour Lane, London SE5 9NU, UK.</div>
<div id="aff5">
<sup><sup>5</sup></sup>Functional Proteomics team, The Institute of Cancer Research, 237 Fulham Road, London SW3 6JB, UK.</div>
<div id="aff6">
<sup><sup>6</sup></sup>UK Dementia Research Institute at University College London, London, UK.</div>
<div id="aff7">
<sup><sup>7</sup></sup>Department of Neurodegenerative Disease, Queen Square Institute of Neurology, University College London, London WC1N 3BG, UK.</div>
<div id="aff8">
<sup><sup>8</sup></sup>Ionis Pharmaceuticals, Carlsbad, CA 92010, USA.</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">Corresponding author. Email: <span>jacob.neeves.17@ucl.ac.uk</span> (J.N.); <span>rickie.patani@ucl.ac.uk</span>, <span>rickie.patani@nus.edu.sg</span> (R.P.)</p>
</div>
<div class="fn" id="afn1">
<sup>†</sup><p class="display-inline">Present address: Neurobiology Programme, Life Sciences Institute, Centre for Life Sciences, National University of Singapore, Singapore, Singapore.</p>
</div>
<div class="fn" id="afn2">
<sup>‡</sup><p class="display-inline">Present address: Department of Medicine, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore.</p>
</div>
<div class="fn" id="afn3">
<sup>§</sup><p class="display-inline">Present address: Department of Anatomy, Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Jacob Neeves</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Project administration, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Marija Petrić Howe</span></strong>: <span class="role">Investigation, Methodology, Validation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Oliver J Ziff</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Methodology, Resources, Software, Validation, Visualization, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Beth Callaghan</span></strong>: <span class="role">Investigation, Validation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Daniel Jutzi</span></strong>: <span class="role">Investigation, Methodology, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Koustav Pal</span></strong>: <span class="role">Formal analysis, Software</span>
</div>
<div>
<strong class="contrib"><span class="name western">Theodoros I Roumeliotis</span></strong>: <span class="role">Data curation, Formal analysis, Methodology, Software, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jyoti Choudhary</span></strong>: <span class="role">Formal analysis</span>
</div>
<div>
<strong class="contrib"><span class="name western">Adrian M Isaacs</span></strong>: <span class="role">Resources, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Frank Rigo</span></strong>: <span class="role">Resources</span>
</div>
<div>
<strong class="contrib"><span class="name western">C Frank Bennett</span></strong>: <span class="role">Resources</span>
</div>
<div>
<strong class="contrib"><span class="name western">Marc-David Ruepp</span></strong>: <span class="role">Funding acquisition, Resources, Supervision, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Rickie Patani</span></strong>: <span class="role">Conceptualization, Funding acquisition, Methodology, Project administration, Resources, Supervision, Writing - review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 1; Accepted 2025 Jul 18; Collection date 2025 Aug 22.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. Distributed under a Creative Commons Attribution License 4.0 (CC BY).</div>
<p>This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution license</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12372870  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40845103/" class="usa-link">40845103</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Splicing factor proline- and glutamine-rich (SFPQ) is an RNA binding protein that broadly regulates RNA metabolism. Although its nuclear roles are well studied, evidence of SFPQ’s cytoplasmic functionality is emerging. Altered expression and nuclear-to-cytoplasmic redistribution of SFPQ have been recognized in amyotrophic lateral sclerosis (ALS) pathology, yet the mechanistic bases for these phenomena remain undetermined. We identified altered <em>SFPQ</em> splicing in ALS, increasing the expression of an alternative mRNA isoform lacking a nuclear localization sequence, which we termed “<em>altSFPQ</em>.” We find that <em>altSFPQ</em> mRNA contributes to SFPQ autoregulation and is highly unstable yet exhibits context-specific translation with cytoplasm-predominant localization. Notably, reduced canonical <em>SFPQ</em> coincides with increased <em>altSFPQ</em> transcript expression in familial and sporadic ALS models, providing a mechanistic basis for SFPQ nuclear-to-cytoplasmic redistribution in patients with ALS. Last, we observe that the altSFPQ protein has reduced phase separation potential and differential protein binding compared to its canonical counterpart, providing insight into its mechanistic relevance to physiology and ALS pathogenesis.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>Mechanisms driving nucleocytoplasmic distribution of SFPQ proteins in health and ALS are uncovered.</p></section><section id="sec1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>Amyotrophic lateral sclerosis (ALS) is a devastating disease with a lifetime risk of between 1:300 and 1:400. It is among the most rapidly progressive and relentless neurodegenerative diseases, with an average survival of only 3 to 5 years. The need for effective disease-modifying treatment targeting the full spectrum of ALS is apparent and urgent. However, a prerequisite for achieving this is a deeper understanding of its molecular pathomechanisms. Recognized pathological hallmarks of ALS include the nuclear-to-cytoplasmic redistribution of RNA binding proteins (RBPs), including TAR DNA binding protein of 43 kDa (TDP-43), fused in sarcoma (FUS), and splicing factor proline- and glutamine-rich (SFPQ) (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>–<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>). Autosomal-dominant mutations are prevalent within the nuclear localization signal (NLS) region in <em>FUS</em>-ALS and less common, but reported, in <em>TARDBP</em>-ALS. Beyond these cases, the mechanistic basis for RBP redistribution in ALS, including for SFPQ, is incompletely resolved.</p>
<p>SFPQ is an abundant, nuclear-predominant RBP, involved in many aspects of nuclear RNA metabolism (<a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>), including transcriptional regulation (<a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>), pre-mRNA splicing (<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>), and paraspeckle formation (<a href="#R9" class="usa-link" aria-describedby="R9"><em>9</em></a>). Consequently, its dysregulated expression and localization cause widespread dysfunction, particularly in neurons, which is typically attributed to loss of its nuclear functions. Loss of SFPQ results in the inclusion of numerous cryptic last exons in genes with neuronal functions (<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>), as well as neuronal apoptosis due to reduced RNA polymerase II processivity through long genes (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>). Furthermore, spatial dissociation of nuclear SFPQ and FUS, which normally form a high–molecular weight (HMW) complex, causes increased four-repeat tau isoform expression, which, in turn, leads to frontotemporal dementia (FTD)–like phenotypes in mice (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>) and neurodegeneration in human cortical neurons (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>). In addition to its nuclear functions, cytoplasmic roles of SFPQ, particularly in neurons, have also been described (<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>–<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>). Notably, SFPQ knockout primarily affects neuronal development, manifesting in axonal malformation, which is abrogated through expression of a cytoplasmically restricted SFPQ protein (ΔNLS-wtSFPQ) (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>). This demonstrates the crucial role played by a non-nuclear pool of SFPQ in neurons.</p>
<p>Pathophysiological increases in cytoplasmic SFPQ, including aggregate formation, have been described in several neurodegenerative diseases (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>, <a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>–<a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>). However, the effect of these phenomena on affected cell types and the mechanistic basis for altered localization remains undetermined. Recently, <em>SFPQ</em> gene polymorphisms were identified in patients with ALS, found in the coiled-coil region implicated in zinc-mediated subcellular localization (<a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>, <a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>). Considering that these regions of the protein are shared with the other two members of the <em>Drosophila</em> behavior/human splicing (DBHS) proteins, PSPC1 and NONO, which do not exhibit significant redistribution in ALS (<a href="#R23" class="usa-link" aria-describedby="R23"><em>23</em></a>), it follows that alternative regulation may drive cytoplasmic localization of SFPQ. Against this background, we hypothesized that an alternative splicing event determines SFPQ nucleocytoplasmic distribution. Along with protein dysregulation, we and others recently described aberrant alternative splicing of <em>SFPQ</em>, in the form of intron retention (IR), in multiple ALS subtypes (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>, <a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>, <a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>). Although <em>SFPQ</em> transcript isoforms have previously been described (<a href="#R26" class="usa-link" aria-describedby="R26"><em>26</em></a>), overall regulation of <em>SFPQ</em> pre-mRNA splicing and the roles of its RNA products remain unresolved.</p>
<p>In this study, we show that an alternative distal exon event in <em>SFPQ</em> is expressed throughout motor neuron development, anticorrelates with the major coding mRNA, and is highly unstable due to nonsense-mediated mRNA decay (NMD). This mRNA also exhibits translation potential and produces a cytoplasmic-predominant protein with differential phase separation propensity and protein binding partners compared to the canonical SFPQ protein. This RNA isoform is up-regulated in human induced pluripotent stem cell (hiPSC)–derived familial ALS neurons (<em>VCP</em>-, <em>FUS</em>-, and <em>SOD1</em>- mutations) and in motor neurons derived from patients with sporadic ALS, thus representing the vast majority of ALS cases. Therefore, we propose that aberrant regulation of <em>SFPQ</em> gene expression ultimately contributes to driving its nuclear protein loss and cytoplasmic accumulation through the preferential production of a cytoplasmically localized protein.</p></section><section id="sec2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="sec3"><h3 class="pmc_sec_title">
<em>SFPQ</em> alternative distal exon splicing is ubiquitous and well conserved</h3>
<p>To gain initial insight into <em>SFPQ</em> gene expression, we searched the UCSC genome browser for annotated <em>SFPQ</em> transcripts. The NCBI RefSeq collection includes the major protein-coding mRNA and two noncoding transcripts, which use alternative terminal exons; these two transcripts differ by just three nucleotides (fig. S1A, in blue). Whereas the first alternative distal exon is well conserved, two additional downstream exons have arisen in mammals. Ensembl includes at least 10 annotated human transcripts; ENST00000357214 is denoted as coding, all others as noncoding (fig. S1A, in red). To broadly assess the relative expression of these transcripts, we used the ExonSkipDB web browser focusing on the Genotype-Tissue Expression (GTEx) database of RNA sequencing (RNA-seq) across 31 normal human tissues [<a href="https://ccsm.uth.edu/ExonSkipDB/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ccsm.uth.edu/ExonSkipDB/</a>; (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>)] (fig. S1B). This demonstrated that three transcripts are ubiquitously expressed: the only annotated coding transcript (henceforth termed <em>wtSFPQ</em>, ENST00000357214), an alternative distal exon(s) transcript (henceforth termed <em>altSFPQ</em>, ENST00000470472), and an additional noncoding transcript consistent with the intron-retaining RNA we previously identified [ENST00000468598; (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>)]. ENST00000470472 corresponds to RefSeq <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_136703.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_136703.2</a>, whereas the lowly expressed ENST00000460428 corresponds to RefSeq <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_136702.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_136702.2</a>. The three transcripts are expressed in all tissues, with relatively high expression in nerve tissue (fig. S1B). Last, we performed MAJIQ splicing analysis (<a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>, <a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>) on our previously generated RNA-seq data derived from samples taken throughout directed motor neurogenesis from hiPSCs (six stages in total, including two postmitotic neuronal stages), with both nuclear and cytoplasmic fractions analyzed at each stage (fig. S1C). MAJIQ quantifies both annotated and unannotated splicing events. The local splicing variant (LSV) which used multiple well-expressed events in both cell compartments and across all cell stages was chr1:35187001-35187122. This LSV diverges from exon 9 and contains (i) exon 9 to 10 splicing, consistent with canonical protein-coding <em>wtSFPQ</em>; (ii) exon 9 to 11 splicing, consistent with <em>altSFPQ</em>; and (iii) previously identified intron 9 retention (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>) (a simplified SFPQ schema demonstrating the divergent events described, and primer design to specifically assay them, is shown in fig. S1D). Each isoform is detected in both cell compartments throughout motor neurogenesis, but the relative expression varies in a stage-dependent manner. An anticorrelation was observed with respect to the relative levels of exon 9 to 10 (<em>wt</em>) and exon 9 to 11 (<em>alt</em>) splicing in both compartments. Intron 9 inclusion (<em>ir</em>) also exhibits an anticorrelation with exon 9 to 10 splicing at most stages, particularly within cytoplasmic samples (<a href="#F1" class="usa-link">Fig. 1A</a>). Therefore, our data suggest that these splicing events in <em>SFPQ</em> are ubiquitous across tissues and expressed continuously through neurodevelopmental lineage restriction of motor neurons. Notably, relative to <em>wtSFPQ</em> expression, absolute levels of <em>altSFPQ</em>, as determined by reverse transcription quantitative polymerase chain reaction (RT-qPCR), increase through motor neurogenesis from iPSCs, to DIV14 neural precursor cells (NPCs), then to DIV35 terminally differentiated motor neurons (fig. S2A). Nevertheless, relatively low expression of <em>altSFPQ</em> across tissues and at earlier stages of our neurogenesis paradigm implies a lower nascent transcription rate and/or reduced posttranscriptional stability compared to <em>wtSFPQ</em>.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Fig. 1. <em>AltSFPQ</em> is an NMD target and contributes to SFPQ autoregulation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372870_sciadv.adt4814-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/55cd/12372870/fdf7a17edc5d/sciadv.adt4814-f1.jpg" loading="lazy" height="767" width="780" alt="Fig. 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Mean percentage splicing (PSI) for each of the three splicing events across six stages of neuronal differentiation (see fig. S1C) in either nuclear or cytoplasmic fractions derived from four healthy hiPSC lines. (<strong>B</strong>) RNA expression (qPCR) relative to time point 0 when DRB treatment was administered, normalized at each time point over <em>GAPDH</em>, in HEK293T cells (<em>n</em> = 3). (<strong>C</strong>) <em>WtSFPQ</em> and <em>altSFPQ</em> mRNA levels (qPCR) normalized over <em>GAPDH</em> and <em>POLR2B</em>, in SMG1 inhibitor (SMG1i) and mock-treated HEK293T cells (<em>n</em> = 3; unpaired <em>t</em> tests). (<strong>D</strong>) Representative images of BaseScope RNA-FISH on untreated and SMG1i-treated HEK293T cells, probed for either <em>wtSFPQ</em> or <em>altSFPQ</em>. (<strong>E</strong>) Quantification of BaseScope <em>SFPQ</em> RNA puncta per cell in untreated (UT) and SMG1i-treated HEK293T cells (<em>n</em> = 3; average of ≥4 fields per replicate; unpaired <em>t</em> tests). (<strong>F</strong>) Ratio between <em>altSFPQ</em> and <em>wtSFPQ</em> transcript expression as measured by qPCR in SFPQ siRNA-treated hiPSC-derived DIV3 motor neurons (<em>n</em> = 3; unpaired <em>t</em> test). (<strong>G</strong>) <em>WtSFPQ</em> and <em>altSFPQ</em> mRNA expression levels (qPCR) normalized over <em>GAPDH</em> and <em>POLR2B</em>, in the HA-wtSFPQ plasmid (pSFPQ) and control plasmid (pEV) transfected HeLa cells (<em>n</em> = 3; unpaired <em>t</em> tests). (<strong>H</strong>) <em>WtSFPQ</em> and <em>altSFPQ</em> mRNA expression levels (qPCR) normalized over <em>GAPDH</em>, in wtSFPQ-T2A-mAPPLE and mAPPLE control lentiviral transduced DIV14 (day 11 posttransduction) i3Neurons (<em>n</em> = 3; unpaired <em>t</em> tests). Graphs are presented as means ± SEM. HKGs, housekeeping genes.</p></figcaption></figure></section><section id="sec4"><h3 class="pmc_sec_title">
<em>AltSFPQ</em> is an NMD target and contributes to SFPQ autoregulation</h3>
<p>To infer posttranscriptional stability of the <em>altSFPQ</em> transcript, we treated human embryonic kidney (HEK) 293T cells with the transcriptional inhibitor 5,6-dichloro-1-β-ribofuranosyl benzimidazole (DRB) and assessed RNA expression over time by RT-qPCR. Compared to the relatively stable transcript <em>GAPDH</em>, <em>altSFPQ</em> was highly unstable, exhibiting down-regulation almost comparable to an <em>SFPQ</em> constitutively spliced intron and at a faster rate than the established NMD-sensitive target <em>GADD45A</em> (<a href="#F1" class="usa-link">Fig. 1B</a>). We orthogonally validated this finding by using the highly selective RNA polymerase II irreversible inhibitor alpha-amanitin (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>, <a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>) (fig. S2B). When considered together with the cytoplasmic presence of <em>altSFPQ</em> mRNA, cytoplasmic degradation is implicated. <em>AltSFPQ</em> contains a premature translation termination codon (PTC), followed by two downstream introns, which suggests that it is a target for NMD (<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>, <a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>). The <em>SFPQ</em> intron-retaining transcript also introduces PTCs and is present in the cytoplasm (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>, <a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>). To evaluate the NMD sensitivity of these transcripts, HEK293T cells were treated with the mRNA translation inhibitor cycloheximide (CHX). <em>AltSFPQ</em> mRNA exhibited a marked 20-fold up-regulation upon inhibition of translation (fig. S2C), which was threefold greater than <em>GADD45A</em>. Intron-retaining <em>SFPQ</em> (henceforth referred to as <em>irSFPQ</em>) exhibited no change in expression, whereas <em>wtSFPQ</em>, unexpectedly, exhibited a small but significant up-regulation. To confirm that exon 9 to 11 splicing reflects the full-length <em>altSFPQ</em> transcript, we performed Northern blot analysis on CHX-treated HEK293T RNA. Using a probe designed to span the last three exons of <em>altSFPQ</em>, we demonstrated clear up-regulation of a 2.3-kb transcript consistent with <em>altSFPQ</em> transcript size (fig. S2D). To orthogonally validate that <em>altSFPQ</em>, but not <em>wtSFPQ</em>, is an authentic NMD target (<a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>), we used a chemical inhibitor of SMG1 kinase, a crucial component of the NMD pathway (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>). Following a 24-hour treatment, <em>altSFPQ</em> transcript levels rose markedly in HEK293T cells as measured by RT-qPCR, whereas <em>wtSFPQ</em> was unaltered (<a href="#F1" class="usa-link">Fig. 1C</a>). This was further orthogonally demonstrated by BaseScope RNA-FISH on fixed HEK293T cultures. Average <em>altSFPQ</em> RNA puncta per cell rose from 2.23 to 26.24 in SMG1i treatment, whereas <em>wtSFPQ</em> RNA levels nonsignificantly decreased from 42.86 to 32.06 per cell (<a href="#F1" class="usa-link">Fig. 1, D and E</a>). Recognizing cell type–specific differences in NMD activity, we further confirmed these findings in hiPSC-derived motor neurons treated with the SMG1 inhibitor. In this context, this finding was coupled with a significant reduction in <em>wtSFPQ</em> expression. Lack of <em>GADD45A</em> response in neurons, but stabilization in HEK293T cells may reflect a reduced concentration of the inhibitor used in neurons or cell type–specific differences in NMD activity (fig. S2, E and F). Nonetheless, together, these data strongly suggest that <em>altSFPQ</em> is an authentic NMD target, but <em>irSFPQ</em> is not.</p>
<p>Many RBPs autoregulate (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>–<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>). Noting that (i) the SFPQ protein binds extensively to its own transcripts (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>) and (ii) <em>altSFPQ</em> transcript is highly unstable, we hypothesized that SFPQ protein autoregulates by directly modulating the splicing decision between <em>altSFPQ</em> and <em>wtSFPQ</em>. To address this, we performed small interfering RNA (siRNA)–mediated knockdown of SFPQ protein in hiPSC-derived motor neurons. We found that, although both <em>wtSFPQ</em> and <em>altSFPQ</em> transcripts were substantially down-regulated, there was a greater magnitude of reduction in <em>altSFPQ</em> (<a href="#F1" class="usa-link">Fig. 1F</a> and fig. S2G). These changes occurred in the absence of altered <em>SFPQ pre-mRNA</em> levels, as inferred by measuring expression of a constitutively spliced intron. We next investigated the effects of SFPQ overexpression on endogenous <em>SFPQ</em> RNAs via transient transfection of a hemagglutinin (HA)–tagged wtSFPQ construct in HeLa cells (fig. S2H). We observed a significant down-regulation of endogenous <em>wtSFPQ</em> transcript 48 hours posttransfection along with significantly increased <em>altSFPQ</em> but no change in <em>irSFPQ</em> or <em>pre-mRNA</em> (<a href="#F1" class="usa-link">Fig. 1G</a> and fig. S2I). Last, we overexpressed HA-wtSFPQ in i3Neurons via lentiviral transduction; again, <em>wtSFPQ</em> was significantly down-regulated and <em>altSFPQ</em> up-regulated (<a href="#F1" class="usa-link">Fig. 1H</a>). Together, these data support a model in which SFPQ protein autoregulates through favored splicing of an mRNA which is targeted for degradation via NMD.</p></section><section id="sec5"><h3 class="pmc_sec_title">
<em>AltSFPQ</em> encodes a previously unidentified SFPQ protein</h3>
<p>Lack of NMD sensitivity of <em>irSFPQ</em> and partial cytoplasmic localization prompted us to examine its coding potential. To do so, we performed polysome profiling (<a href="#F2" class="usa-link">Fig. 2A</a>) on hiPSC-derived NPCs and subsequent RT-qPCR of transcripts on the resultant fractions. As expected, <em>wtSFPQ</em> and positive control <em>GAPDH</em> transcripts predominantly associate with HMW polysomal fractions, consistent with highly efficient translation. In contrast, <em>irSFPQ</em> was only localized to LMW polysome fractions. Considering that NMD is dependent on transcript engagement with ribosomal machinery, whereupon it is typically degraded during early rounds of translation, we also expected to see predominant <em>altSFPQ</em> engagement with individual ribosomes (monosomes) and low–molecular weight (LMW) polysomes but minimal engagement with HMW polysomes (<a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>, <a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>). Unexpectedly, a relatively large pool of <em>altSFPQ</em> mRNA associates with HMW polysomes, indicative of translation. The proportion of the transcript within these fractions is comparable to that of <em>GAPDH</em> (<a href="#F2" class="usa-link">Fig. 2B</a>). To ensure that these profiles reflected true ribosomal engagement and not cosedimentation, we first assessed the impact of EDTA treatment on HEK293T lysates to induce ribosomal dissociation. Both <em>wtSFPQ</em> and <em>altSFPQ</em> mRNAs shifted substantially from HMW polysome fractions toward lighter fractions (fig. S3A). The same effect was then observed following EDTA treatment of NPC lysates (<a href="#F2" class="usa-link">Fig. 2C</a>); notably, no shift was observed for <em>irSFPQ</em> (fig. S3B). To determine that <em>altSFPQ</em> polysomal association did not reflect a nonspecific or artifactual profile, we examined the <em>SRSF6</em> PTC+ NMD-sensitive transcript (<a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>). The <em>SRSF6</em> transcript was predominantly identified in the monosomal fraction with progressively diminishing presence through disome and LMW polysome fractions. Furthermore, the different profile compared to <em>altSFPQ</em> is not simply a consequence of differential expression of the two transcripts (fig. S3C). Together, these data suggest <em>altSFPQ</em> translation potential in mammalian cells.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2. <em>AltSFPQ</em> encodes a previously unidentified SFPQ protein.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372870_sciadv.adt4814-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/55cd/12372870/f0830cc4811e/sciadv.adt4814-f2.jpg" loading="lazy" height="845" width="780" alt="Fig. 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Polysome profiling schema, with example absorbance profile. RNP, ribonucleoprotein. (<strong>B</strong>) Analysis of <em>SFPQ</em> transcripts and <em>GAPDH</em> mRNA by qPCR from hiPSC-derived NPCs (DIV7), plotted as the percentage of total (<em>n</em> = 3; one cell line derived from a healthy donor and two cell lines derived from a patient with VCP-ALS). (<strong>C</strong>) Polysome profiles obtained from hiPSC-derived DIV14 NPCs with (red) or without (black) EDTA treatment; EDTA treatment caused substantial loss of polysomes. Analysis (right) of <em>wtSFPQ</em> or <em>altSFPQ</em> mRNA expression by qPCR from each fraction, plotted as the percentage of total (<em>n</em> = 3; two cell lines from healthy donors and one cell line from a patient with VCP-ALS). (<strong>D</strong>) Schematic depiction of SFPQ isoforms and antibodies used for IP and immunoblotting. One antibody recognizes an epitope common to both proteins (“N”-term SFPQ antibody), whereas a C-terminally targeting antibody recognizes only wtSFPQ (“C”-term SFPQ antibody). RRMs, RNA recognition motifs. (<strong>E</strong>) Western blot showing the altSFPQ isoform expression in SMG1i-treated HEK293T cells; <em>n</em> = 3, stars demarcate the proteins. (<strong>F</strong>) <em>AltSFPQ</em> mRNA levels (qPCR, normalized over <em>GAPDH</em>) in nontargeting control ASO (NTC)–treated and ASO20-treated HEK293T cells (<em>n</em> = 3; unpaired <em>t</em> test). (<strong>G</strong>) Representative Western blot of the endogenous SFPQ protein expression in NTC-treated or ASO20-treated HEK293T cells; relates to (F). Initial IP using ab177149 (C terminus targeting) SFPQ antibody removes wtSFPQ and enables visualization of the altSFPQ protein in “cleared” fractions (using N-terminal SFPQ antibody). An 8-μg input was loaded along with increasing amounts of cleared lysate; an input equivalent amount of the bead eluate was loaded. (<strong>H</strong>) Quantification of altSFPQ (normalized over vinculin) in (G), using the highest-quantity cleared lysate within each replicate (<em>n</em> = 3; unpaired <em>t</em> test). Graphs are presented as means ± SEM.</p></figcaption></figure><p>Specific identification of the alternative protein is complicated by (i) low expression in the basal state, (ii) similar molecular weight to the canonical SFPQ protein (wtSFPQ), and (iii) lack of tryptic digest sites to enable unique peptide identification, of relevance to most proteomics library preparation approaches. Commercially available antibodies include one that only targets the wtSFPQ C terminus and others that target structured domains common to both isoforms (hereafter denoted as N-terminal antibody) (<a href="#F2" class="usa-link">Fig. 2D</a>). HA-tagged wtSFPQ and altSFPQ proteins run at such similar weights as to overlap on SDS–polyacrylamide gel electrophoresis (PAGE) (fig. S3D). Consequently, and considering the relatively low expression of altSFPQ compared to wtSFPQ, higher input requirements will typically result in masking of the alternative protein. To circumvent this problem, we made use of SMG1 inhibition–mediated <em>altSFPQ</em> up-regulation in conjunction with long SDS-PAGE runtimes on fixed percentage gels and Western blot detection using both the N- and C-terminal antibodies. This demonstrated an α-N-terminal SFPQ-positive, α-C-terminal SFPQ-negative protein running at the expected molecular weight and therefore consistent with the novel isoform (<a href="#F2" class="usa-link">Fig. 2E</a>). To further confirm that this is an SFPQ protein, we pretreated cells with an SFPQ-targeting siRNA pool before SMG1i treatment. This reduced expression of the SMG1i-induced protein to less than 40% (fig. S3E), confirming that it is an SFPQ protein.</p>
<p>Reduced expression of NMD regulators and reduced UPF1 phosphorylation may result in broad translation of truncated proteins deriving from NMD-sensitive transcripts (<a href="#R41" class="usa-link" aria-describedby="R41"><em>41</em></a>, <a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>). To examine whether the altSFPQ protein is translated in the absence of NMD inhibition, we sought specific means of <em>altSFPQ</em> up-regulation. We reasoned that splicing of its encoding transcript might be determined by intronic splicing regulatory sequences in close proximity to the splice sites. Therefore, we designed a panel of uniformly phosphorothioate (PS)- and 2′-<em>O</em>-methoxyethyl (2′MOE)–modified antisense oligonucleotides (ASOs) with the aim of specifically increasing <em>altSFPQ</em> expression. In total, 20 ASOs were designed and tested: 2 ASOs targeting either the 5′ or 3′ splice site, with respect to <em>altSFPQ</em> exons 9 and 10, and a further 18 ASOs “tiled” in nine-nucleotide steps forward or backward from the 5′ and 3′ splice sites, respectively (fig. S3F). ASOs were initially screened in HEK293T cells for 48 hours at a 200 nM concentration, and <em>SFPQ</em> mRNA expression was assessed by RT-qPCR. One ASO (ASO20) successfully increased <em>altSFPQ</em> mRNA expression (fig. S3G). To maximize accuracy of altSFPQ protein detection, we made use of the wtSFPQ C-terminal–specific antibody to immunoprecipitate this protein out of solution. Following this, immunoblotting using an SFPQ N-terminal–specific antibody revealed a protein ~97 kDa in size, with increased expression in an ASO20-dependent manner (<a href="#F2" class="usa-link">Fig. 2, F to H</a>). Of note, ASO20 treatment also reduced the expression of the <em>wtSFPQ</em> mRNA and wtSFPQ protein (fig. S3, H and I). Last, we used the previously described C-terminal antibody immunoprecipitation (IP) approach to confirm that the altSFPQ protein is expressed in hiPSC-derived motor neurons (fig. S3J). Together, this shows that an altSFPQ protein is expressed and that its levels increase when <em>altSFPQ</em> transcript expression is up-regulated.</p></section><section id="sec6"><h3 class="pmc_sec_title">
<em>AltSFPQ</em> encodes a cytoplasm-predominant protein, which attenuates subcellular distribution of DBHS proteins</h3>
<p>AltSFPQ and wtSFPQ proteins share an identical sequence until amino acid 663; altSFPQ has a further seven unique amino acids (total length of 669 amino acids), whereas wtSFPQ has an additional 45 unique amino acids (total length of 707 amino acids). In the altSFPQ protein, this difference dictates an omission of the classical nuclear localization signal (cNLS) located at the extreme C terminus of wtSFPQ (<a href="#F2" class="usa-link">Fig. 2D</a>), which is required for its nuclear localization (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>). To test the localization of the altSFPQ protein, we transiently expressed HA-tagged wtSFPQ, altSFPQ, and a ΔNLS SFPQ (but otherwise unaltered from wtSFPQ) in HEK293T cells and assessed distribution by immunofluorescence (<a href="#F3" class="usa-link">Fig. 3A</a>). As expected, wtSFPQ was predominantly localized in the nucleus, whereas omission of the NLS markedly shifts the protein into the cytoplasm, although some nuclear localization remains. The altSFPQ protein exhibited a highly similar subcellular distribution as for the ΔNLS protein, demonstrating that the unique C terminus does not encode an alternative functional NLS. To assess the nuclear-cytoplasmic distribution of endogenous wtSFPQ and altSFPQ proteins, we used SMG1i treatment of HEK293T cells followed by biochemical fractionation and Western blotting. In line with the distribution of the recombinant proteins, wtSFPQ is predominantly nuclear, whereas SMG1i-induced altSFPQ is predominantly cytoplasmic, although both are present within both compartments (<a href="#F3" class="usa-link">Fig. 3, B and C</a>). To address whether altSFPQ can modulate <em>SFPQ</em> expression in a similar manner to wtSFPQ, we assessed the effect of altSFPQ overexpression on endogenous <em>SFPQ</em> RNAs. Transfected HA-altSFPQ did not significantly alter expression of <em>wtSFPQ</em> and <em>altSFPQ</em>, or <em>irSFPQ</em>, in HeLa cells (fig. S4A). Furthermore, lentiviral-derived HA-altSFPQ overexpression in i3Neurons also had no significant effect on the expression of <em>SFPQ</em> RNAs (fig. S4B). This implies that altSFPQ does not autoregulate.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3. <em>AltSFPQ</em> encodes a cytoplasm-predominant protein, which attenuates subcellular distribution of DBHS proteins.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372870_sciadv.adt4814-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/55cd/12372870/3445d724e455/sciadv.adt4814-f3.jpg" loading="lazy" height="916" width="779" alt="Fig. 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Representative immunofluorescence images showing the subcellular localization of HA-tagged recombinant SFPQ proteins in transfected HEK293T cells. (<strong>B</strong>) Western blot of nuclear-cytoplasmic fractionated SMG1i-treated HEK293T cells; vinculin and lamin B1 act as subcellular markers. (<strong>C</strong>) Quantification of the nuclear-cytoplasmic ratio of SFPQ proteins in fractionated SMG1i Western blots; the C-term SFPQ antibody signal is used to quantify wtSFPQ, and the N-term antibody is used to quantify the smaller visible SFPQ protein (i.e., altSFPQ); <em>n</em> = 3; one-way ANOVA, Tukey’s multiple comparisons. (<strong>D</strong>) HEK293T cells were transfected with 100 ng of eGFP-wtSFPQ (+100 ng of EV plasmid), 100 ng of mAPPLE-wtSFPQ (+100 ng of EV plasmid), or 100 ng of each tagged SFPQ protein; representative zoomed-in images are displayed (arrowheads show co-localized cytoplasmic eGFP and mAPPLE signals; stars show the nuclei exhibiting both eGFP and mAPPLE signal). (<strong>E</strong>) Image analysis quantification relating to (D), measuring the nuclear-cytoplasmic ratio (using a 20-μm perinuclear ring region) of the eGFP signal (reflecting the eGFP-wtSFPQ protein) in eGFP-wtSFPQ transfected versus eGFP-wtSFPQ/mAPPLE-altSFPQ cotransfected cells (<em>n</em> = 3; each data point represents the average across ≥5 fields of view for each replicate; unpaired <em>t</em> test). a.u., arbitrary units. (<strong>F</strong>) As for (E) but assessing the nuclear-cytoplasmic ratio of the mAPPLE signal (reflecting the mAPPLE-altSFPQ protein) (<em>n</em> = 3; each data point represents the average of ≥5 fields of view for each replicate; unpaired <em>t</em> test).</p></figcaption></figure><p>The DBHS proteins (SFPQ, NONO, and PSPC1) primarily exist in homo- and heterodimeric forms (<a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>). To determine whether altSFPQ expression modulates the subcellular distribution of wtSFPQ, we individually or coexpressed both with different fluorescent tags to visualize the two proteins simultaneously. Intriguingly, the recombinant proteins both affected one another’s localization, with redistribution of wtSFPQ to the cytoplasm and altSFPQ to the nucleus when coexpressed (<a href="#F3" class="usa-link">Fig. 3, D to F</a>). Furthermore, overexpressed altSFPQ, as well as a cytoplasmic wtSFPQ protein (ΔNLS-wtSFPQ), redistributed endogenous NONO, the highest-affinity heterodimeric partner of SFPQ protein (<a href="#R45" class="usa-link" aria-describedby="R45"><em>45</em></a>), into the cytoplasm (fig. S4, C and D). Therefore, <em>altSFPQ</em> encodes a cytoplasmic protein, which can modulate the subcellular distribution of other DBHS proteins, likely through dimerization.</p></section><section id="sec7"><h3 class="pmc_sec_title">AltSFPQ exhibits reduced phase separation propensity compared to wtSFPQ and is up-regulated during neuronal stress</h3>
<p>As well as subcellular distribution, we reasoned that the altered C-terminal protein sequence may affect the behavior of altSFPQ relative to wtSFPQ protein. AlphaFold prediction suggests that the region unique to wtSFPQ protein (highlighted in red) is unstructured (<a href="#F4" class="usa-link">Fig. 4A</a>). All structured domains common to the DBHS protein family are shared, and the differential C termini are of low complexity. Intriguingly, it was recently reported that the C-terminal low-complexity region (LCR) drives SFPQ liquid-liquid phase separation (LLPS), whereas the N-terminal LCR attenuates this (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). The altSFPQ protein loses a large proportion of this C-terminal LCR, reducing its predicted disorder (<a href="#F4" class="usa-link">Fig. 4B</a>). To directly test the relative LLPS propensity of the two proteins, we generated N-terminally enhanced green fluorescent protein (eGFP)–tagged SFPQ proteins with solubilizing C-terminal maltose-binding protein (MBP) tags (<a href="#F4" class="usa-link">Fig. 4C</a>). Recombinant SFPQ proteins were purified from bacterial cultures and concentrated in a storage buffer containing 300 mM NaCl (fig. S5A) at final concentrations of 4.32 mg/ml (GFP-wtSFPQ-MBP) and 4.33 mg/ml (GFP-altSFPQ-MBP); both exhibited high purity (260/280 absorbance ratios of 0.57 for both). For in vitro droplet assays, proteins were diluted to a physiological salt concentration (150 mM NaCl), MBP tags were enzymatically removed, and common crowding agent polyethylene glycol (PEG) was added at increasing concentrations to a fixed 3 μM protein. Micrometer-sized droplets increasingly formed in a PEG concentration–dependent manner (<a href="#F4" class="usa-link">Fig. 4D</a>); both proteins settled to the slide surface as round droplets and fused over time, demonstrative of their liquid-like state (fig. S5B). For quantification, images were taken once droplets had settled to the bottom of the slide but before droplet fusion, enabling quantification of the condensed phase. We quantified droplets and their characteristics from triplicate experiments in the presence of 1% PEG (<a href="#F4" class="usa-link">Fig. 4D</a> and fig. S5C). eGFP-altSFPQ homotypic samples consistently formed fewer droplets with a trend toward reduced size and amount of eGFP protein within droplets compared to eGFP-wtSFPQ (<a href="#F4" class="usa-link">Fig. 4, E and F</a>, and fig. S5). To orthogonally validate our findings, we used a sedimentation assay, whereby samples are centrifuged following tobacco etch virus (TEV) cleavage, and the partitioning of SFPQ proteins into the pelleted fraction is used as a measure of phase separation. A crowding agent increased phase separation of SFPQ proteins, as expected, but protein was also present in the pellet fraction in the absence of any crowding agent. Quantitative comparison of the two SFPQ proteins showed that both without and with a crowding agent altSFPQ exhibited reduced phase separation (<a href="#F4" class="usa-link">Fig. 4, G and H</a>, and fig. S5E). These data demonstrate that, although both SFPQ proteins have LLPS behavior, this is reduced for altSFPQ due to a reduced C-terminal LCR.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4. AltSFPQ exhibits reduced phase separation compared to wtSFPQ and is up-regulated during neuronal stress.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372870_sciadv.adt4814-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/55cd/12372870/62e3d40de0e5/sciadv.adt4814-f4.jpg" loading="lazy" height="916" width="766" alt="Fig. 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) AlphaFold-predicted wtSFPQ protein structure showing the isoform-specific low-complexity C-terminal region (red). (<strong>B</strong>) Schematic depiction of wtSFPQ and altSFPQ protein domains and IUPred disorder prediction graphs. aa, amino acids. (<strong>C</strong>) SFPQ protein purification strategy for LLPS assays. (<strong>D</strong>) Representative images of the eGFP-(wt/alt)SFPQ protein homotypic liquid droplet formation through increasing PEG and fixed protein (3 μM) concentrations, in 150 mM salt buffer; zoomed insets demonstrate spherical forms. (<strong>E</strong>) Quantification of the LLPS droplet number for 3 μM eGFP-(wt/alt)SFPQ proteins in 150 mM salt buffer and 1% PEG (<em>n</em> = 3; unpaired <em>t</em> test). (<strong>F</strong>) Ratio of GFP intensity inside/outside LLPS droplets for 3 μM eGFP-(wt/alt)SFPQ proteins in 150 mM salt buffer and 1% PEG (<em>n</em> = 3; unpaired <em>t</em> test). (<strong>G</strong>) Representative image of the sedimentation assay, gel loaded with supernatant and pellet fractions of TEV cleaved eGFP-SFPQ proteins ± PEG. (<strong>H</strong>) Quantification of the pellet/supernatant ratio of cleaved eGFP-SFPQ proteins, for samples processed without PEG (<em>n</em> = 3; unpaired <em>t</em> test). (<strong>I</strong>) <em>AltSFPQ</em> expression (qPCR) in nuclear fractions of motor neurons (DIV6) in response to sorbitol-induced osmotic (OSM) and sodium arsenite–induced oxidative (OX) stresses, normalized over <em>Nit1</em> and <em>NFX1</em> fraction housekeeping genes (Frac_HKGs). Data are expressed as FC over untreated samples per line and presented as means ± SEM from three control lines; ANOVA with Dunnett’s multiple comparisons. (<strong>J</strong>) As for (I) but in cytoplasmic fractions. (<strong>K</strong>) Representative images of untreated and sodium arsenite–treated motor neurons (DIV6), with BaseScope <em>altSFPQ</em> RNA-FISH, DAPI, and bright-field differential interference contrast (DIC) overlay; arrowheads show RNA puncta within axons. (<strong>L</strong>) Quantification of BaseScope RNA-FISH puncta per cell in untreated versus sodium arsenite–induced oxidative stress condition in motor neurons (DIV6); data points reflect fields of view (<em>n</em> = 5 to 7), multiple unpaired <em>t</em> tests.</p></figcaption></figure><p>LCRs are suggested to be important mediators of functional biocondensation in vivo (<a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>). SFPQ is found in cellular stress–induced biocondensates such as cytoplasmic stress granules and nuclear paraspeckles (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>). We have previously shown that cytoplasmic stress granules form during osmotic and oxidative neuronal stress responses (<a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>). Considering divergent LLPS, we hypothesized that altSFPQ expression might be increased during cellular stress. Osmotic (sorbitol) and oxidative (arsenite) stressors were applied to day 6 terminally differentiated hiPSC-derived motor neurons, and <em>SFPQ</em> transcript expression was assessed by RT-qPCR. Both stressors caused an ~2-fold increase in <em>altSFPQ</em> expression; neither was associated with a significant increase in <em>wtSFPQ</em>. Intriguingly, in oxidative and, to a greater extent, osmotic stress, <em>irSFPQ</em> levels were significantly reduced (fig. S5, F and G). To gain further insight, we assessed the responses in the nucleus and cytoplasm separately following biochemical fractionation, with the rationale that changes in the nucleus are more likely to arise from posttranscriptional splicing, particularly if coupled to reduced IR, whereas cytoplasmic up-regulation only might indicate stabilization through stress-induced NMD inhibition. In line with the whole-cell analysis, <em>altSFPQ</em> expression was significantly increased in the nucleus during osmotic stress, whereas <em>irSFPQ</em> was down-regulated (<a href="#F4" class="usa-link">Fig. 4I</a> and fig. S5H). We also quantified the expression of an abundant retained intron derived from the <em>OGT</em> gene and its spliced counterpart. This intron is posttranscriptionally spliced out in response to numerous stimuli to enable rapid modulation of O-linked glycosylation (<a href="#R50" class="usa-link" aria-describedby="R50"><em>50</em></a>–<a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>). The <em>OGT</em> protein-coding transcript was increased, whereas the <em>OGT</em> intron-retaining transcript is reciprocally down-regulated (fig. S5I), implicating a similar mode of regulation for <em>irSFPQ</em> and <em>altSFPQ</em> during the hyperosmotic stress response. In the cytoplasm, <em>altSFPQ</em> expression was increased in response to oxidative stress only (<a href="#F4" class="usa-link">Fig. 4J</a>); <em>irSFPQ</em> was markedly down-regulated in the cytoplasm during osmotic stress (fig. S5J). To orthogonally validate our findings, we performed BaseScope RNA-FISH using 1xZZ probes targeting a unique exon-exon junction for either <em>wtSFPQ</em> or <em>altSFPQ</em> on fixed day 6 terminally differentiated motor neurons subjected to arsenite-induced oxidative stress. <em>AltSFPQ</em> RNAs were detected in cell nuclei, somas, and neurites under untreated and stressed conditions (<a href="#F4" class="usa-link">Fig. 4K</a>). Oxidative stress caused an increase in <em>altSFPQ</em> RNA puncta (average of 1.55 RNAs per cell across biological lines to 3.31 under treatment; <a href="#F4" class="usa-link">Fig. 4L</a>) in both the nucleus (average untreated = 0.80, oxidative = 1.82) and extranuclear cell regions (average untreated = 1.00, oxidative = 2.04), broadly in line with qPCR analyses, whereas <em>wtSFPQ</em> RNA levels (whole-cell average untreated = 6.20, oxidative = 7.94) were increased in nuclei only (nuclear average untreated = 3.12, oxidative = 5.01; extranuclear untreated = 4.58, oxidative = 5.25) (fig. S6, A and B).</p>
<p>Together, these data demonstrate that the altSFPQ protein exhibits reduced phase separation propensity compared to wtSFPQ, and <em>altSFPQ</em> mRNA expression is increased in the context of acute neuronal stresses, raising the possibility that altSFPQ is functional during neuronal stress.</p></section><section id="sec8"><h3 class="pmc_sec_title">Altered C terminus drives differential protein binding partners for altSFPQ</h3>
<p>SFPQ contains two RNA recognition motifs within the core DBHS region of the protein, which mediate its binding to RNAs. It is noteworthy that the altSFPQ protein retains this region entirely, raising the hypothesis that it maintains the same theoretical RNA interactome. To test whether the altSFPQ protein retains capacity to bind the same RNAs in vivo, we performed RNA immunoprecipitation (RIP) following transient expression of the HA-SFPQ variants in neuro-2A (N2A) neuroblastoma cells. altSFPQ binds to established SFPQ RNA interactors: lncRNA (long noncoding RNA) <em>NEAT1_2</em> (<a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>), <em>SFPQ retained intron 9</em> (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>), and <em>LMNB2</em> and <em>BCL2L2</em>, which are anterogradely trafficked by SFPQ within axonal granules to synapses (<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>) (<a href="#F5" class="usa-link">Fig. 5A</a> and fig. S7A). A trend toward reduced binding compared to wtSFPQ may be mediated by different levels of these RNAs within the nucleus and cytoplasm, as well as cotranscriptional association of wtSFPQ to these transcripts. Considering the binding of altSFPQ to these RNAs, and that RNA binding can modulate protein biocondensation, we assessed the impact of RNA addition on in vitro phase separation. Addition of an 800–base pair (bp) region of <em>SFPQ intron 9</em> induced visible biocondensate formation for both wtSFPQ and altSFPQ proteins in the absence of any crowding agent; this took the form of “beads-on-a-string” fibrillar structures rather than globular droplets (fig. S7B). Furthermore, RNA addition increased sedimentation of both proteins similarly in vitro (fig. S7, C and D). These results suggest that RNA binding enhances the propensity for both wtSFPQ and altSFPQ to form biomolecular condensates, which might alter the nucleocytoplasmic or subcompartmental distribution of both.</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Fig. 5. Altered C terminus drives differential protein binding partners for altSFPQ.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372870_sciadv.adt4814-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/55cd/12372870/4e4c5dc5c984/sciadv.adt4814-f5.jpg" loading="lazy" height="608" width="800" alt="Fig. 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) RIP performed on whole-cell lysates from N2A cells expressing HA-SFPQ variants or an EV as a negative control. Levels of associated mRNAs were assessed by qPCR using primers against the indicated targets and expressed as the percentage of total input (<em>n</em> = 3; data represent means ± SEM). (<strong>B</strong>) Three-way Venn diagram showing numbers of proteins significantly bound by WT-SFPQ, ALT-SFPQ, and NLS-SFPQ proteins over the EV condition (EV; log<sub>2</sub>FC &gt; 0.5, <em>P</em> &lt; 0.05) from affinity purification proteomics data from untreated cells. (<strong>C</strong>) ORA of 808 SFPQ-bound proteins (subsetted based on &gt;0.3 positive correlation with the bait HA-proteins) comparing ALT versus WT binding enrichment from untreated cells (top 10 gene ontology pathways and Kyoto Encyclopedia of Genes and Genomes terms displayed). (<strong>D</strong>) As for (C) but for terms associated with increased ALT-SFPQ over NLS-SFPQ protein binders. (<strong>E</strong>) As for (B) but from oxidative-stressed cells. (<strong>F</strong>) As for (C) in oxidative-stressed cells. (<strong>G</strong>) As for (D) but from oxidative-stressed cells. Some GO terms are shortened.</p></figcaption></figure><p>Having established that RNA binding is maintained, we hypothesized that the differential low-complexity C terminus of altSFPQ alters its protein interactome. We performed coimmunoprecipitation (co-IP) on lysates from N2A cells expressing WT- and ALT-SFPQ variants, as well as a ΔNLS-wtSFPQ to infer differential binding not conferred by altered nucleocytoplasmic localization alone; empty vector (EV) expression acted as a negative control. Obligate SFPQ binder NONO was efficiently pulled down with each SFPQ variant, whereas GAPDH protein was not detected (fig. S7E); four replicate experiments were subsequently processed for mass spectrometry (MS). Similar numbers of proteins were identified as significantly increased over EV [<em>P</em> &lt; 0.05, log<sub>2</sub>FC (fold change) &gt; 0.5] across variants (457 proteins for WT-SFPQ, 418 for ALT-SFPQ, and 386 for ΔNLS-SFPQ), most of which bind to all three proteins (63.1%; <a href="#F5" class="usa-link">Fig. 5B</a> and table S1). Use of human SFPQ (hSFPQ) bait proteins in mouse cells enabled us to use hSFPQ peptide counts to regress out spurious or artifactual binders and minimize the impact of variability in the amount of immunoprecipitated bait protein between variants and replicates. As a result, a total of 808 proteins were identified as binders of any variant based on a positive correlation cutoff of 0.3 target:bait (table S2). We used this subset to carry out overrepresentation analysis (ORA) [STRING; (<a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>)] to determine whether ALT-SFPQ binds to different groups of proteins. Compared to WT-SFPQ, ALT-SFPQ exhibits enriched binding to ribosomal and translation-associated, as well as cell junction and synapse-associated proteins (<a href="#F5" class="usa-link">Fig. 5C</a>), whereas terms enriched for WT-SFPQ include spliceosomal, mRNA splicing, and mitochondrial complex (fig. S7F). ΔNLS-SFPQ also exhibited enriched binding to cytosolic ribosome–associated proteins compared to WT-SFPQ, although to a lesser extent, and reduced binding to splicing/spliceosome proteins, as well as ribosome biogenesis terms but not mitochondrial proteins. Furthermore, when we directly compared ΔNLS to ALT-SFPQ, the most enriched terms in ALT were cytosolic ribosome and translation, whereas splicing and mitochondrial proteins were enriched in the ΔNLS interactome (<a href="#F5" class="usa-link">Fig. 5D</a> and fig. S7G). In addition, considering the increased expression of <em>altSFPQ</em> in stress, we subjected N2A cells to sodium arsenite–induced oxidative stress and assessed the protein interactomes of SFPQ variants. NONO was efficiently pulled down with each SFPQ variant under stress, whereas GAPDH protein was once more unbound (fig. S7H). Slightly fewer proteins were identified as significantly increased over EV (<em>P</em> &lt; 0.05, log<sub>2</sub>FC &gt; 0.5) compared to untreated samples (354 proteins for WT-SFPQ, 306 for ALT-SFPQ, and 382 for ΔNLS-SFPQ), and almost half of all binders were shared by the three proteins (49.9%; <a href="#F5" class="usa-link">Fig. 5E</a> and table S1). ORA revealed that, during stress, ALT-SFPQ still exhibits enriched binding to ribosomal proteins compared to both WT-SFPQ and ΔNLS-SFPQ proteins (<a href="#F5" class="usa-link">Fig. 5, F and G</a>) and decreased binding to splicing-associated terms (fig. S7, I and J).</p>
<p>Together, these data demonstrate differential protein binding of SFPQ variants, which manifests through not only the divergent nucleocytoplasmic localization of endogenous SFPQ proteins but also differences within the cytoplasm driven by alternative C-terminal protein sequences. These may arise directly from alternative C-terminal protein binding, divergent biocondensation properties, or likely a combination of these.</p></section><section id="sec9"><h3 class="pmc_sec_title">
<em>AltSFPQ</em> mRNA is up-regulated in familial and sporadic ALS iPSMNs</h3>
<p>To test whether <em>SFPQ</em> is alternatively spliced during motor neuron development in the context of ALS, we examined splicing in our poly(A) RNA-seq dataset of <em>VCP</em> mutation–related ALS (<em>VCP</em>-ALS hereafter) compared to CTRL samples undergoing motor neurogenesis (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>), with a focus on LSV chr1:35187001-35187122. Although the broad pattern of splicing in <em>VCP</em>-ALS cultures is comparable to that of the controls throughout neurogenesis (fig. S8A), direct comparison between the two groups revealed dysregulation at multiple stages (<a href="#F6" class="usa-link">Fig. 6A</a> and fig. S8, B and C). Aberrant expression of <em>intron 9</em>–containing and <em>wtSFPQ</em> transcripts has previously been validated (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>). Increased <em>altSFPQ</em> splicing was additionally validated by RT-qPCR, in nuclear and cytoplasmic samples. Furthermore, the abundance of the transcript was found to be increased in both compartments in <em>VCP</em>-ALS (<a href="#F6" class="usa-link">Fig. 6B</a>). This represents, to our knowledge, the first report of such altered <em>SFPQ</em> splicing in a model derived from patients with ALS.</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">Fig. 6. <em>AltSFPQ</em> mRNA is up-regulated in familial and sporadic ALS iPSMNs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372870_sciadv.adt4814-f6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/55cd/12372870/5bc564520e3b/sciadv.adt4814-f6.jpg" loading="lazy" height="334" width="785" alt="Fig. 6."></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Line graph showing the mean delta PSI value (<em>VCP</em> mutant–control) for <em>wtSFPQ</em> and <em>altSFPQ</em> splicing events across six stages of neuronal differentiation in either nuclear (left) or cytoplasmic (right) fractions derived from four control and four VCP mutant hiPSC lines. (<strong>B</strong>) Bar graphs showing SFPQ exon 9 to 11 splicing levels analyzed by qPCR for NPC nuclear and cytoplasmic fractions from control and <em>VCP</em> mutant samples. Splicing was measured by normalizing target expression over the gene expression (constitutive exon) level for each line (left). The right panel depicts the same but for transcript abundance (target normalized over <em>NIT1</em> and <em>NFX1</em> fraction housekeeping genes). Data expressed as FC over controls and presented as means ± SEM from four lines per group, with data points representing the mean value for each biological line across two technical replicates; unpaired <em>t</em> tests. (<strong>C</strong>) Violin plots showing CTRL-FUS delta PSI values (<em>y</em> axis) for <em>SFPQ</em> splicing events (<em>wtSFPQ</em>/exon 9 to 10 in red; <em>altSFPQ</em>/exon 9 to 11 in blue) in <em>FUS</em> R521G mutant motor neurons versus controls (<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>). (<strong>D</strong>) As for (C) in <em>SOD1</em> A4V mutant motor neurons versus controls (<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>). (<strong>E</strong>) Violin plots showing PSI values (<em>y</em> axis) for <em>SFPQ</em> splicing events (<em>wtSFPQ</em>/exon 9 to 10 in red; <em>altSFPQ</em>/exon 9 to 11 in blue) in &gt;200 sporadic ALS iPSMNs and &gt;50 control samples (<a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>, <a href="#R58" class="usa-link" aria-describedby="R58"><em>58</em></a>).</p></figcaption></figure><p>Having established that <em>altSFPQ</em> is increased in <em>VCP</em>-related ALS, we next sought to understand whether it is increased across familial ALS. We therefore analyzed publicly available poly(A) RNA-seq datasets of iPSC-derived motor neurons (iPSMNs) from patients with <em>FUS</em> or <em>SOD1</em> ALS (<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>, <a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>). <em>AltSFPQ</em> splicing was significantly increased in both familial ALS datasets [<em>FUS</em> R521G delta percent spliced in (dPSI) 0.146, <em>P</em> = 0.0212; <em>SOD1</em> A4V dPSI 0.047, <em>P</em> = 4.18 × 10<sup>−5</sup>], whereas <em>wtSFPQ</em> splice junction usage was significantly reduced (<em>FUS</em> R521G dPSI −0.146, <em>P</em> = 0.0212; <em>SOD1</em> A4V dPSI −0.378, <em>P</em> = 1.66 × 10<sup>−13</sup>) (<a href="#F6" class="usa-link">Fig. 6, C and D</a>); <em>irSFPQ</em> was increased in SOD1 A4V neurons (dPSI 0.323, <em>P</em> = 5.99 × 10<sup>−12</sup>) (fig. S8D).</p>
<p>To understand the generalizability of <em>altSFPQ</em> expression further, we examined sporadic ALS, which comprises 90% of all patients with ALS. To this end, we analyzed <em>SFPQ</em> splicing in RNA-seq data from 208 sporadic ALS and 50 control iPSMN lines from combined NeuroLINCS (<a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>) (8 sporadic and 8 control lines) and Answer ALS (<a href="#R58" class="usa-link" aria-describedby="R58"><em>58</em></a>) (200 sporadic and 42 control lines) studies. It should be noted that these data derive from ribo-depleted but not poly(A) selected RNA libraries. Splicing analysis revealed that the LSV which exhibited the most significant alteration in ALS was the same as in the previous analyses (i.e., exon 9 to 10, exon 9 to 11, and intron 9 inclusive). Splicing of <em>wtSFPQ</em> mRNA was markedly reduced in sporadic ALS (dPSI −0.106, <em>P</em> = 2.76 × 10<sup>−5</sup>). Conversely, <em>altSFPQ</em> splicing (dPSI 0.036, <em>P</em> = 2.19 × 10<sup>−3</sup>) was significantly up-regulated (<a href="#F6" class="usa-link">Fig. 6E</a>); IR was also up-regulated (dPSI 0.050, <em>P</em> = 5.22 × 10<sup>−4</sup>) (fig. S8E). This result therefore implicates dysregulated <em>altSFPQ</em> mRNA expression, along with potential translation, broadly throughout ALS, which may explain the established nuclear loss and cytoplasmic gain of SFPQ protein observed in familial and sporadic postmortem patient motor neurons (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>).</p></section></section><section id="sec10"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>The nuclear-to-cytoplasmic redistribution of SFPQ is an established phenotype of diseased neurons in ALS. However, the mechanistic basis for this change in subcellular localization has remained undetermined. In this study, we show that, in hiPSC-derived neuronal models of ALS, the <em>SFPQ</em> gene undergoes a splicing switch from the canonical protein-coding mRNA (<em>wtSFPQ</em>) toward an alternative isoform with a unique 3′ terminal sequence (<em>altSFPQ</em>). Despite NMD sensitivity, this isoform undergoes translation to generate a protein omitting a classical NLS sequence (<a href="#R59" class="usa-link" aria-describedby="R59"><em>59</em></a>) and exhibiting cytoplasmic-predominant localization, therefore recapitulating the signature redistribution profile found in ALS (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>). The concept of alternative isoforms leading to a change in subcellular localization of ALS-related RBPs is supported by a recent report of a C terminus truncated short TDP-43 (sTDP-43) isoform that introduces a nuclear export signal and exhibits up-regulation and cytosolic aggregation in a hyperexcitability-dependent manner (<a href="#R60" class="usa-link" aria-describedby="R60"><em>60</em></a>). Other C terminus truncated TDP-43 splice isoforms have been identified with similar localization characteristics (<a href="#R61" class="usa-link" aria-describedby="R61"><em>61</em></a>). Notably, SFPQ is redistributed in other neurodegenerative diseases such as Alzheimer’s disease and FTD (<a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>, <a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>, <a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>). Whether SFPQ splicing is similarly dysregulated in these other diseases is of notable interest. Our findings are also supported by a recent preprint from Zeng <em>et al.</em> (<a href="#R62" class="usa-link" aria-describedby="R62"><em>62</em></a>), demonstrating that knockdown of the ALS-associated TDP-43 protein shifts poly(A) site usage from proximal to distal in the <em>SFPQ</em> gene, consistent with the shift from <em>wtSFPQ</em> to <em>altSFPQ</em> expression, which we have identified across models derived from patients with familial and sporadic ALS. This may provide a mechanism contributing to <em>SFPQ</em> dysregulation, although SOD1 and FUS fALS should progress by a different mechanism given that these cases typically lack TDP-43 pathology.</p>
<p>Seminal work identified SFPQ (PSF as it was previously known) as a spliceosome-associated nuclear protein and described multiple SFPQ transcript isoforms including one consistent with <em>altSFPQ</em> mRNA (<a href="#R26" class="usa-link" aria-describedby="R26"><em>26</em></a>, <a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>, <a href="#R63" class="usa-link" aria-describedby="R63"><em>63</em></a>). However, although proteolytically truncated SFPQ proteins have been identified in specific contexts (<a href="#R64" class="usa-link" aria-describedby="R64"><em>64</em></a>), no alternative splicing determined proteoform has been demonstrated to the best of our knowledge. Our data suggest that, under normal conditions, <em>wtSFPQ</em> mRNA expression markedly outweighs that of <em>altSFPQ</em>. Consistent with this, SFPQ is typically observed as a highly expressed nuclear protein wherein it carries out numerous functions including transcriptional regulation, splicing and polyadenylation, DNA damage repair, and paraspeckle formation (<a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>, <a href="#R65" class="usa-link" aria-describedby="R65"><em>65</em></a>). Nonetheless, cytoplasmic functionality of SFPQ was recently reported in neurons, including roles in mRNA transport (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>, <a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>, <a href="#R66" class="usa-link" aria-describedby="R66"><em>66</em></a>). Furthermore, axonal developmental defects in zebrafish conferred by total loss of SFPQ are at least partially rescued by a ΔNLS cytoplasmic SFPQ protein (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>). Although altSFPQ loses a classical NLS, it retains two RNA recognition motifs, a NonA/paraspeckles (NOPS) domain, and a coiled-coil domain (<a href="#R67" class="usa-link" aria-describedby="R67"><em>67</em></a>). We have demonstrated that it retains capacity to bind established SFPQ RNA targets, including those with which it associates in axonal RNA transport granules (<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>). Gene distal alternative last exon isoforms broadly display preferential localization to neurites (<a href="#R68" class="usa-link" aria-describedby="R68"><em>68</em></a>), promoting isoform-specific axonal translation and formation of a distinct translatome (<a href="#R69" class="usa-link" aria-describedby="R69"><em>69</em></a>). We have observed <em>altSFPQ</em> mRNA localization within neurites of hiPSC-derived motor neurons; it follows that <em>altSFPQ</em> may undergo local translation to carry out known SFPQ functions in axonal and synaptic maintenance (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>). Notably, low-level expression of an ALS-linked C-terminally truncated FUS protein, which thereby exhibits cytoplasmic localization, causes marked changes in the spinal cord transcriptome (<a href="#R70" class="usa-link" aria-describedby="R70"><em>70</em></a>), implicating small changes in cytoplasmic SFPQ expression as potentially impactful. However, the contribution of increased <em>altSFPQ</em> to ALS pathogenesis via a cytoplasmic gain of function is currently undetermined.</p>
<p>High conservation of the <em>altSFPQ</em> first alternative exon implies important functionality. The two additional downstream exons display much less conservation. Considering that its NMD sensitivity is likely conferred by the presence of downstream introns within the 3′ untranslated region, it is possible that, in lower-order organisms, the alternative protein is expressed from a relatively stable transcript, assuming that lower conservation predicts lower usage. As such, the protein function of the altSFPQ isoform may have preceded a more complex role involving autoregulation as well as alternative protein localization and function. Of note, although PTC presence is generally predictive of NMD sensitivity, detectable protein can still be generated from such transcripts despite their low levels (<a href="#R71" class="usa-link" aria-describedby="R71"><em>71</em></a>).</p>
<p>In addition to a structured central region, SFPQ has low-complexity N- and C-terminal regions, the latter of which drives its LLPS (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). AltSFPQ lacks a large proportion of the C-terminal LCR present in wtSFPQ. We predicted that this might confer reduced phase separation propensity and confirmed as such in vitro. Alternative isoforms exhibiting differential phase separation mediated by differential LCR inclusion have been demonstrated for at least one other gene (<a href="#R72" class="usa-link" aria-describedby="R72"><em>72</em></a>), supporting our model whereby alternative splicing regulates SFPQ protein biocondensation properties and thereby presumably its cytoplasmic function. LLPS droplets of ALS-relevant RBPs may transition into aggregates with time (<a href="#R73" class="usa-link" aria-describedby="R73"><em>73</em></a>, <a href="#R74" class="usa-link" aria-describedby="R74"><em>74</em></a>). The biocondensates (spherical droplets) we observed in our phase separation studies for both SFPQ proteins did not completely fuse once settled to the slide surfaces, thereby resulting in the appearance of many “deformed” spherical structures. This is consistent with initial LLPS formation, which progresses to a viscous liquid-like or even a gel-like state (<a href="#R75" class="usa-link" aria-describedby="R75"><em>75</em></a>); this has been directly related to progression toward pathological fibrillization in the context of other RBPs relevant to ALS (<a href="#R76" class="usa-link" aria-describedby="R76"><em>76</em></a>). The degree to which this state is reversible has not been examined in this study, but it is noteworthy that cytoplasmic aggregates containing SFPQ are sometimes observed in ALS postmortem tissue (<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>). AltSFPQ does not lack biocondensate properties but rather displays reduced propensity to form biocondensates compared to wtSFPQ. As such, whereas differential biocondensation proclivity may subserve differential functions, increased altSFPQ in ALS over time may still lead to the formation of cytoplasmic aggregates or contribute to an “aggregation-prone” environment for other proteins. On the other hand, the low-level diffuse cytoplasmic signal, rather than aggregates, also described for SFPQ in ALS pathology is also consistent with altSFPQ protein expression.</p>
<p>Mechanisms underlying direct wtSFPQ nuclear-to-cytoplasmic relocalization have been described (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>); these may occur alongside altSFPQ expression or represent context-specific requirements for the cytoplasmic localization of SFPQ proteins with different behaviors and thereby functions. In line with this, we observed increased <em>altSFPQ</em> expression, at least partially independent of <em>wtSFPQ</em> changes during acute neuronal stress responses. Furthermore, we identified divergent classes of protein binding partners for the different SFPQ proteins. Some of these are likely conferred by the differential nuclear-to-cytoplasmic localization of wtSFPQ versus altSFPQ in the contexts that we have assessed. However, differential binding of altSFPQ and ΔNLS-SFPQ to ribosomal large subunit proteins, as an example, implies divergent behavior within the cytoplasm. This may be a direct consequence of the differential C-terminal sequences conferring different binding, reduced stochastic low-complexity binding, altered biocondensation, or a combination. In line with this, despite potential assay-related biocondensate disruption, it was intriguing to observe reduced binding of altSFPQ to mitochondrial proteins, which were previously found to interact with the RBP FUS in an LLPS-dependent manner (<a href="#R74" class="usa-link" aria-describedby="R74"><em>74</em></a>). In addition, SFPQ has previously been identified as a binder of the large ribosomal subunit in a large omics study (<a href="#R77" class="usa-link" aria-describedby="R77"><em>77</em></a>); it is plausible that this might have been altSFPQ, considering that altSFPQ peptides would be indistinguishable from those of wtSFPQ. Whether increased binding of altSFPQ to the ribosome confers altered translation in vivo is an interesting outstanding question, particularly considering translation defects identified in ALS (<a href="#R78" class="usa-link" aria-describedby="R78"><em>78</em></a>).</p>
<p>AltSFPQ readily interacted with endogenous SFPQ in our dataset, implying in vivo heterodimerization; SFPQ and the other DBHS proteins form both homo- and heterodimers with one another (<a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>). Furthermore, coexpression of wtSFPQ and altSFPQ proteins resulted in subcellular redistribution of both, with clear signal colocalization in the cytoplasm, whereas the overexpressed altSFPQ protein caused redistribution of endogenous NONO toward the cytoplasm. DBHS domain–containing proteins in the dipteran <em>Chironomus tentans</em> regulate one another’s subcellular localization (<a href="#R79" class="usa-link" aria-describedby="R79"><em>79</em></a>), which raises the intriguing prospect of human DBHS protein heterodimerization regulating subcellular localization of SFPQ proteins, as well as NONO and possibly PSPC1. The relative impact will depend on expression level and heterodimer affinities, which are nonuniform across different cells and tissues (<a href="#R65" class="usa-link" aria-describedby="R65"><em>65</em></a>). In the context of ALS, this implies that expression of altSFPQ will not immediately “drag” wtSFPQ from the nucleus, but as dysregulated isoform expression shifts increasingly toward altSFPQ during the course of disease, this will progressively affect nuclear wtSFPQ levels through direct protein-protein interactions. As well as subcellular localization, it follows that altSFPQ dimerization with DBHS proteins might regulate their biocondensation, given that intrinsically disordered protein composition determines phase separation and aggregation propensity of these proteins (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>), and thereby alter their molecular functions (<a href="#R80" class="usa-link" aria-describedby="R80"><em>80</em></a>). We have also described the formation of fibrillar structures, rather than droplets, upon addition of RNA to eGFP-SFPQ proteins in vitro. A similar behavior has been observed for NONO and PSPC1 proteins (<a href="#R80" class="usa-link" aria-describedby="R80"><em>80</em></a>, <a href="#R81" class="usa-link" aria-describedby="R81"><em>81</em></a>) but not FUS: RNA initially enhances the formation of globular droplets, until high levels of RNA are added, whereupon droplets are solubilized (<a href="#R82" class="usa-link" aria-describedby="R82"><em>82</em></a>). This implies a shared mechanism by which RNA-dependent modulation of biocondensation regulates DBHS protein function, including altSFPQ.</p>
<p>Our stress data raise the possibility that <em>altSFPQ</em> derives from a reservoir of “poised” <em>SFPQ</em> intron-retaining transcripts, which we and others have previously identified as increased in expression in familial and sporadic ALS (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>, <a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>, <a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>). Broadly, stress-responsive splicing out of normally retained introns has been proposed as a mechanism for cellular adaptation to stress (<a href="#R50" class="usa-link" aria-describedby="R50"><em>50</em></a>, <a href="#R83" class="usa-link" aria-describedby="R83"><em>83</em></a>, <a href="#R84" class="usa-link" aria-describedby="R84"><em>84</em></a>). Candidate-based single-molecule fluorescence in situ hybridization (smFISH) studies have suggested that alternatively spliced exons are posttranscriptionally spliced at a greater frequency than their constitutively spliced counterparts (<a href="#R85" class="usa-link" aria-describedby="R85"><em>85</em></a>). This possibility has been orthogonally supported by metabolic labeling studies using 4-thiouridine to identify and isolate RNA within minutes of transcription, which revealed that alternative splicing is slower than constitutive splicing (<a href="#R86" class="usa-link" aria-describedby="R86"><em>86</em></a>). Further evidence supporting this phenomenon whereby a reservoir of IR transcripts are “poised” and then posttranscriptionally spliced in a stimulus-specific manner comes from live-cell reporters, which have convincingly demonstrated that introns with relatively weak splice sites are removed over a longer time frame compared to introns with stronger and canonical splice sites (<a href="#R87" class="usa-link" aria-describedby="R87"><em>87</em></a>). Additionally, retained introns are more evolutionarily conserved than their constitutive counterparts, which is consistent with functional significance (<a href="#R50" class="usa-link" aria-describedby="R50"><em>50</em></a>).</p>
<p>Together, we have highlighted dysregulated splicing of <em>SFPQ</em> in ALS, the translation potential of a previously identified mRNA isoform, and present this as a potential mechanism driving the nuclear loss and cytoplasmic gain of SFPQ protein in this disease (<a href="#F7" class="usa-link">Fig. 7</a>). Many questions remain, including the extent of altSFPQ pathology in ALS, the generalizability of altered SFPQ splicing in neurodegeneration, the functions of altSFPQ, and whether these are cell type- and context-specific. Further investigations addressing these points will determine the potential therapeutic benefit of targeting this SFPQ splicing isoform in ALS.</p>
<figure class="fig xbox font-sm" id="F7"><h3 class="obj_head">Fig. 7. Proposed model of SFPQ protein redistribution in ALS.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372870_sciadv.adt4814-f7.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/55cd/12372870/bad298f9fed5/sciadv.adt4814-f7.jpg" loading="lazy" height="829" width="800" alt="Fig. 7."></a></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Under control conditions, the <em>SFPQ</em> gene expresses at least two splicing isoforms; <em>wtSFPQ</em> (red RNA) encodes a nuclear predominant protein, the other (<em>altSFPQ</em>; blue RNA) is largely degraded but encodes a primarily cytoplasmic protein. The altSFPQ protein exhibits reduced phase separation propensity and partially distinct protein interactome, including translation-associated machinery. In ALS models, <em>wtSFPQ</em> is reduced whereas <em>altSFPQ</em> is increased, possibly resulting in reduced nuclear and increased cytoplasmic SFPQ proteins, manifesting as the nuclear-to-cytoplasmic redistribution of total SFPQ protein in disease.</p></figcaption></figure></section><section id="sec11"><h2 class="pmc_sec_title">MATERIALS AND METHODS</h2>
<section id="sec12"><h3 class="pmc_sec_title">Experimental models</h3>
<section id="sec13"><h4 class="pmc_sec_title">
Ethics statement
</h4>
<p>Informed consent was obtained from healthy control donors and patients in this study. Experimental protocols were enacted according to regulations and guidelines outlined by UCLH’s National Hospital for Neurology and Neurosurgery and UCL’s Institute of Neurology Joint Research Ethics Committee (09/0272).</p></section><section id="sec14"><h4 class="pmc_sec_title">
hiPSCs and motor neurons
</h4>
<p>hiPSC lines used are detailed in table S3. Three control lines are commercially available: Coriell (ND41866*C), Thermo Fisher Scientific (A18945), and Cedars Sinai (CS02iCTR-NTn4). HiPSCs were maintained on Geltrex (Life Technologies) with Essential 8 media (Life Technologies) and passaged using EDTA (Life Technologies, 0.5 mM). Directed differentiation from hiPSCs to motor neurons was performed using a previously published protocol (<a href="#R88" class="usa-link" aria-describedby="R88"><em>88</em></a>). Briefly, hiPSCs were first grown to 100% confluency and then differentiated to the neuroepithelium in chemically defined medium consisting of Dulbecco’s modified Eagle’s medium (DMEM)/F12 GlutaMAX, Neurobasal, <span class="font-variant-small-caps">l</span>-glutamine, N2 supplement, nonessential amino acids, B27 supplement, β-mercaptoethanol (βME) (all from Life Technologies) and insulin (Sigma-Aldrich). Treatment with small molecules from days 0 to 7 was as follows: 1 μM dorsomorphin (Millipore), 2 μM SB431542 (Tocris Bioscience), and 3.3 μM CHIR99021 (Miltenyi Biotec). Starting from day 8, the neuroepithelial layer was patterned for 7 days with 0.5 μM retinoic acid and 1 μM purmorphamine. At day 14, NPCs were treated with 0.1 μM purmorphamine for a further 4 days before being terminally differentiated in 0.1 μM compound E (Enzo Life Sciences) to promote cell cycle exit. All cultures were maintained at 37°C and 5% CO<sub>2</sub>.</p>
<p>For RNA interference, NPCs were plated onto 12-well plates in N2B27 media at density of 5 × 10<sup>5</sup> cells per well and then transfected with 30 pmol of siRNA targeting SFPQ (pool of four siRNAs, ON-TARGETplus SmartPool SFPQ siRNA, L-006455-00-0005, Horizon) or nontargeting control siRNA (pool of four siRNAs, ON-TARGETplus Non-targeting Control Pool, D-001810-10-05, Horizon). Two microliters of Lipofectamine RNAiMax was used as a transfection reagent according to the manufacturer’s protocol. Following overnight incubation, cell media were changed to fresh N2B27 neural media supplemented with 0.1 μM compound E to push cells into terminal differentiation. After 3 days in this medium (96 hours posttransfection), samples were harvested. For CHX treatments, CHX was added directly to growth media at a final concentration of 100 μg/ml and cells were maintained in culture for 6 hours. For SMG1 inhibition, 0.5 μM hSMG-1 inhibitor 11e (a kind gift from O. Muehlemann, University of Bern) was added directly to growth media on cells and cells were maintained in culture for 24 hours. For CHX and SMG1i treatments, dimethyl sulfoxide (DMSO) was used as the vehicle control. For osmotic stress experiments, 0.4 M sorbitol was added onto cells in normal motor neuronal media and incubated for 1 hour in a standard 37°C incubator. For oxidative stress experiments, 0.5 mM sodium arsenite (Sigma-Aldrich) was added, again for 1 hour at 37°C. Treatment compounds were diluted in cell media. Cells from each condition plus untreated cultures were then harvested.</p></section><section id="sec15"><h4 class="pmc_sec_title">
HEK293T, HeLa, and N2A cell lines
</h4>
<p>The HEK293T, HeLa, and N2A lines were obtained from the Francis Crick Cell Services facility, following mycoplasma testing. Cells were cultured in DMEM containing 10% fetal bovine serum and penicillin/streptomycin and grown at 37°C and 5% CO<sub>2</sub>. Cells were transfected with Lipofectamine 2000 in Opti-MEM according to the manufacturers’ guidelines. For CHX treatments, CHX was added directly to growth media at a final concentration of 100 μg/ml and cells were maintained in culture for 6 hours. For SMG1 inhibition, 1 μM SMG1i was added directly to growth media on cells and cells were maintained in culture for 24 hours. DMSO was used as the vehicle control for CHX and SMG1i treatments. For DRB (Sigma-Aldrich) treatments, DRB was added directly to growth media at a final concentration of 75 nM and cells were maintained in culture for indicated times. For alpha-amanitin treatments, alpha-amanitin (Roche) was added directly to growth media at a final concentration of 5 μg/ml and cells were maintained in culture for indicated times.</p></section><section id="sec16"><h4 class="pmc_sec_title">
iNeuron cell culture
</h4>
<p>Stably expressing doxycycline-inducible BS6 2H9 <em>Neurogenin2</em> (<em>Ngn2</em>) i3-iPS cells were used for rapid differentiation into cortical neurons (i3Neurons) using a previously described method (<a href="#R89" class="usa-link" aria-describedby="R89"><em>89</em></a>). Briefly, i3iPS cells were grown to 80% confluency, washed with phosphate-buffered saline (PBS), lifted with Accutase (Gibco), and plated at 1 × 10<sup>7</sup> cells in a 10-cm Geltrex-coated dish (DIV0). Cells were maintained from DIV0–3 in an induction medium consisting of DMEM/F12 (Gibco), 1× N2 (Thermo Fisher Scientific), 1× GlutaMAX (Gibco), 1× Hepes (Gibco), 1× nonessential amino acids (Gibco), doxycycline (2 μg ml<sup>−1</sup>), and 10 μM Y-27632 (DIV0 only; Tocris), which was exchanged daily. On DIV3, cells were dissociated with Accutase and replated on to poly(ethylenimine)-coated (Sigma-Aldrich) and laminin-coated (Sigma-Aldrich) 12-well plates at 7.5 × 10<sup>5</sup> cells per well in neuronal maintenance medium consisting of Neurobasal medium (Gibco), supplemented with 1× B27 (Gibco), brain-derived neurotrophic factor (10 ng ml<sup>−1</sup>; PeproTech), NT-3 (10 ng ml<sup>−1</sup>; PeproTech), laminin (1 μg ml<sup>−1</sup>), and 5-fluor-2′-deoxyuridine (FDU; 2 μM DIV4, 0.5 μM thereafter; Gibco). From DIV3 to DIV14, cells were maintained in neuronal maintenance medium. Lentiviral transduction (multiplicity of infection: 7) to overexpress SFPQ-T2A-mAPPLE proteins was carried out 1 hour after DIV3 replating, with a full media change carried out the next day. On day 11 posttransduction (DIV14 overall), cells were washed once in PBS, then lifted with EDTA, pelleted at 2000<em>g</em> for 5 min at 4°C, and then processed for RNA extraction.</p></section></section><section id="sec17"><h3 class="pmc_sec_title">Method details</h3>
<section id="sec18"><h4 class="pmc_sec_title">
Generation of plasmids
</h4>
<p>pcDNA3.1-HA-SFPQ was a gift from R. Segal (Addgene plasmid no. 166959; <a href="http://n2t.net/addgene:166959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://n2t.net/addgene:166959</a>; RRID:Addgene_166959). For the HA-altSFPQ plasmid, a unique C-terminal altSFPQ coding sequence was amplified from HEK293T cDNA with esp3I and ecoRI overhangs and cloned into the HA-SFPQ plasmid with the esp3I-ecoRI insert removed. For the HA-NLS-SFPQ plasmid, a portion of the wtSFPQ coding sequence up to, but not including, the terminal nuclear localization sequence was amplified from HEK293T cDNA with esp3I and ecoRI overhangs and cloned into the HA-SFPQ plasmid with the esp3I-ecoRI insert removed. For the EV plasmid, a multiple cloning site with kpn1-ecoRI overhangs was cloned into the HA-SFPQ plasmid with the kpn1-ecoRI insert (entire HA-wtSFPQ sequence) removed. eGFP-wtSFPQ and mAPPLE-altSFPQ expression constructs were generated by cloning eGFP or mAPPLE sequences into pcDNA3.1-HA-(wt/alt)SFPQ plasmids using NEBuilder (NEB).</p>
<p>To generate His6-<em>Tev</em>-eGFP-SFPQ-<em>Tev</em>-MBP plasmids ORF (open reading frame) codon optimized for <em>Escherichia coli</em>, wtSFPQ and altSFPQ coding sequences (GeneArt, Thermo Fisher Scientific) were cloned into pMal-C5-<em>Tev</em>-eGFP-<em>Tev</em>--His6 (generated by gene synthesis and cloning of <em>Tev</em>-eGFP-<em>Tev</em>-His6 cDNA into SacI-HindIII sites of pMalC5x, General Biosystems) using XbaI and BamHI restriction sites to generate pMal-C5-<em>Tev</em>-SFPQ-eGFP-<em>Tev-</em>-His6. pMal-C5-<em>Tev</em>-SFPQ-eGFP-<em>Tev</em>--His6 was then used to PCR and then combine the following four fragments using Gibson assembly (NEB): ATG-His6-<em>TEV</em>-GFP-SFPQ, (wt/alt)SFPQ-<em>TEV</em>-MBP, MBP-STOP, and MBP-STOP-vector_backbone-ATG-His6-<em>TEV</em>-GFP, giving final His6-<em>Tev</em>-eGFP-SFPQ-<em>Tev</em>--MBP constructs. For in vitro transcription of <em>SFPQ intron 9</em> RNA, an 800-bp region (GTAATGTATCC-UGAAUGUGGAU), which is bound by SFPQ (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>), was generated by gene synthesis and cloned into a pMA vector, with a T3 promoter upstream and an EcoRV restriction site downstream of the <em>SFPQ</em> intron sequence (GeneArt, Thermo Fisher Scientific).</p></section><section id="sec19"><h4 class="pmc_sec_title">
Lentivirus production
</h4>
<p>A pCDH-hSYN lentiviral plasmid was used as the vector to create the control and SFPQ lentiviral constructs. Subcloning of SFPQ constructs was undertaken as a three-fragment insertion of the (i) HA-SFPQ coding sequence, (ii) T2A, and (iii) mAPPLE using NEBuilder (NEB) to yield final pCDH-hSYN-HA_SFPQ-T2A-mAPPLE products. Control was generated by cloning the mAPPLE only into the vector to generate the pCDH-hSYN-mAPPLE plasmid.</p>
<p>Third-generation VSV-G lentiviruses were prepared in HEK293T cells: SFPQ or control lentiviral transfer plasmids transfected with pLP1, pLP2, and pVSV-G lentiviral packaging and envelope plasmids, using Lipofectamine 2000 (Thermo Fisher Scientific). Virus-containing supernatants were collected 48 and 72 hours posttransfection, combined, and concentrated by precipitation with PEG-8000–NaCl (final 8% and 80 mM, respectively) for &gt;4 hours at 4°C. Virus containing pellets were resuspended in PBS and titered using ddPCR (from Addgene; <a href="https://addgene.org/protocols/lentivirus-ddpcr-titration/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://addgene.org/protocols/lentivirus-ddpcr-titration/</a>) and then aliquoted and stored at −80°C until use.</p></section><section id="sec20"><h4 class="pmc_sec_title">
Recombinant protein expression and purification
</h4>
<p>Expression and purification of recombinant His6-eGFP-SFPQ-MBP (wt and alt) were performed using a protocol adapted from (<a href="#R90" class="usa-link" aria-describedby="R90"><em>90</em></a>). In brief, the bacterial expression vectors were transformed into Rosetta-2(DE3)-pLysS <em>E. coli</em> and grown in standard Lysogeny broth (LB) medium supplemented with chloramphenicol (33 μg/ml) and carbenicillin (100 μg/ml). At an optical density at 600 nm (OD<sub>600</sub>) of ∼0.8, cells were induced with 0.1 mM IPTG (isopropyl-β-D-thiogalactopyranoside) for 24 hours at 12°C. Cells were lysed by sonication (5 x 1-min pulses with 5-min breaks between pulses) in lysis buffer [50 mM Na<sub>2</sub>HPO<sub>4</sub>/NaH<sub>2</sub>PO<sub>4</sub> (pH 8.0), 300 mM NaCl, 20 μM ZnCl<sub>2</sub>, 20 mM imidazole, and 10% glycerol] supplemented with 4 mM βME, DNase1 (20 U/ml; Roche), and RNase A (200 μg/ml; Sigma-Aldrich), and the debris was removed by centrifugation at 24,500<em>g</em> and 10°C for 1 hour. The cleared lysates were then incubated with Ni-NTA agarose beads (Qiagen) and washed with lysis buffer before elution in HisTrap buffer [50 mM Na<sub>2</sub>HPO<sub>4</sub>/NaH<sub>2</sub>PO<sub>4</sub> (pH 8.0), 300 mM NaCl, 20 μM ZnCl<sub>2</sub>, 250 mM imidazole, and 4 mM βME]. The eluate was then incubated with amylose resin (NEB), washed with wash buffer [50 mM Na<sub>2</sub>HPO<sub>4</sub>/NaH<sub>2</sub>PO<sub>4</sub> (pH 8.0), 300 mM NaCl, 20 μM ZnCl<sub>2</sub>, 20 mM imidazole, and 4 mM βME], and subsequently eluted in wash buffer (+ 20 mM maltose). Last, proteins were concentrated using 100-kDa molecular weight cut-off Viva spin concentrator columns, then aliquoted, and stored at −80°C for one time use per aliquot. Protein concentrations were determined from their absorbance at 280 nm using ε predicted by the ProtParam tool; 260-/280-nm ratios of all purified proteins were ∼0.6.</p></section><section id="sec21"><h4 class="pmc_sec_title">
In vitro phase separation assays
</h4>
<p>Purified full-length His6-eGFP-SFPQ-MBP stock proteins (wt or alt) were centrifuged at 17,000<em>g</em> for 10 min at 4°C to remove the precipitated/aggregated protein, and then protein concentrations were reestablished by measuring the absorbance at 280 nm by NanoDrop using the respective extinction coefficients (ε). Samples were then diluted in salt-free droplet buffer [50 mM Na<sub>2</sub>HPO<sub>4</sub>/NaH<sub>2</sub>PO<sub>4</sub> (pH 8.0) and 20 μM ZnCl<sub>2</sub>] to a final 150 mM NaCl concentration.</p>
<p>For droplet assays, phase separation was induced by addition of TEV protease (Sigma-Aldrich) at 25°C for 60 min, followed by addition of PEG-8000 crowding agent and transfer to μ-slide 18-well–flat chambers (IBIDI). Imaging for quantification of eGFP-tagged SFPQ proteins was performed by wide-field fluorescence microscopy; one 20x magnification image was taken per condition in each experimental batch. Flat-field illumination correction, filtering, noise reduction, object identification, and output measurements were performed with CellProfiler. Imaging for figs. S4B and S6B were taken on an 880 laser scanning confocal microscope (Zeiss) at 63x objective.</p>
<p>For in vitro sedimentation assays, samples were processed as described in (<a href="#R91" class="usa-link" aria-describedby="R91"><em>91</em></a>). Briefly, the TEV protease was added and the sample was incubated at 30°C for 2 hours to cleave the MBP tag and then incubated a further 30 min at room temperature with or without addition of PEG-8000 crowding agent. For experiments measuring the impact of RNA addition, an 800-bp section of <em>SFPQ intron 9</em> was added for 30 min at room temperature without a crowding agent, following MBP cleavage. The RNA was in vitro transcribed from T3-<em>SFPQintron9</em> using the T3 MEGAscript kit (Thermo Fisher Scientific) following linearization of pMA-T3-<em>SFPQintron9</em>-EcoRV by EcoRV-HF restriction enzyme digest and phenol/chloroform cleanup; the reaction was incubated with TurboDNAse for 30 min at 37°C after transcription, RNA was precipitated by phenol/chloroform, and a single 800-bp product was confirmed by running on an agarose gel before use. After 30 min, room temperature incubation samples were centrifuged for 15 min at 17,000<em>g</em> at 4°C to pellet-formed condensates. Supernatant fractions were collected, and pellets were resuspended in the same volume of 150 mM NaCl droplet buffer; a 4x NuPAGE LDS sample buffer and 50 mM dithiothreitol (DTT) final concentration were added, and protein samples were denatured at 80°C for 10 min. Samples were equally loaded onto Invitrogen NuPAGE 4 to 12% Bis-Tris gels, along with an unstained protein ladder, and standard SDS-PAGE runs proceeded. Gels were then fixed for 30 min in 50% ethanol and 10% acetic acid and then stained overnight (SYPRO Ruby). The following day, gels were destained with 10% ethanol and 10% acetic acid and then imaged on a GelDoc XR+ (Bio-Rad). Intensities of cleaved, full-length eGFP protein bands were quantified in ImageJ.</p></section><section id="sec22"><h4 class="pmc_sec_title">
Cell fractionation
</h4>
<p>Biochemical subcellular fractionation was achieved using the Ambion PARIS kit cell fractionation buffer, following the manufacturer’s general protocol, and an 8 M urea nuclear lysis buffer was prepared in-house. Initially, cells were washed once using ice-cold PBS. Cytosolic fraction was then obtained by lysing cells directly in ice-cold cell fractionation buffer for 10 min, thus disrupting plasma membranes while leaving nuclear membranes intact. Lysates were then centrifuged for 3 min at 500<em>g</em> and 4°C. The supernatant was then collected and further centrifuged at maximum speed and 4°C, and the resulting supernatant was then processed as cytosolic fraction. Nuclear pellets from the first centrifugation step were washed once with cell fractionation buffer and then lysed on ice for 30 min in 8 M urea nuclear lysis buffer, containing 50 mM tris-HCl (pH 8), 100 mM NaCl, 0.1% SDS, and 1 mM DTT. The resulting nuclear fraction was then homogenized using a QIAshredder (QIAGEN) to shred chromatin and reduce viscosity, before being further processed for RNA extraction. Both lysis buffers were supplemented with RiboLock RNase Inhibitor (0.1 U/μl) and Halt Protease Inhibitor Cocktail.</p></section><section id="sec23"><h4 class="pmc_sec_title">
Northern blotting
</h4>
<p>For <em>altSFPQ</em> Northern blot analysis, 5 μg of total RNA per condition was denatured and electrophoresed on a 1% formaldehyde-agarose gel for 2 hours at 80 V. Following washes in RNase-free H<sub>2</sub>O and 20x SSC buffer, the blot was transferred onto a Hybond nylon membrane (Amersham Biosciences) overnight via capillary transfer at room temperature. The next day, RNA was cross-linked onto the membrane by ultraviolet light (254 nm) exposure at 120 J. Then, the membrane was blocked in prehybridization buffer (Sigma-Aldrich) plus preboiled salmon sperm single-stranded DNA (ssDNA) (100 μg/ml) at 68°C for 4 hours, following which the prehybridization solution was replaced by PerfectHyb Plus Hybridization Buffer (Sigma-Aldrich) [plus denatured ssDNA (100 μg/ml)]. A purified probe was added, and the membrane was incubated at 68°C overnight. Probes were synthesized as follows: The target region was amplified from HEK293T cDNA using 5′-3′ (TGATTGATGTTGGCTGATATTGGA) and 3′-5′ (ACGTTCATTCCTCCTTCCACT) primers and standard PCR. Probes were labeled using the Random Primer DNA Labeling Kit v2 (Takara), according to the manufacturer’s instructions. Briefly, 20 ng of the probe (template), 2 μl of a random primer, and nuclease-free water were combined in a 14-μl reaction, which was then heated at 95°C for 3 min to denature template/primers. To this reaction, the following was added: 2.5 μl of 10x buffer, 2.5 μl of a deoxynucleoside triphosphate mixture [containing 0.2 mM dGTP (deoxyguanosine triphosphate), dATP (deoxyadenosine triphosphate), and dTTP (deoxythymidine triphosphate) each] and 5 μl of labeled deoxycytidine triphosphate (dCTP) (32P-dCTP, 9.25 megabecquerels). Last, following the addition of 1 μl of exo-free Klenow fragment (2 U/μl), the reaction was incubated at 37°C for 2 hours. Labeled probes were then purified using MiniQuick Spin DNA Columns (Roche) according to the manufacturer’s instructions. The purified probe was denatured by heating at 100°C for 5 min before addition to the hybridization buffer.</p>
<p>On the following day, the prehybridization solution was removed and the membrane was washed three times in 2x SSC and 0.1% SDS at 68°C, followed by final high stringency wash in 0.2x SSC and 0.1% SDS. The hybridized membrane was then exposed to a storage phosphor screen for a few hours, capturing the hybridized signal, and then developed using a Molecular Imager FX Pro Plus System and Quantity One 1-D software (Bio-Rad).</p></section><section id="sec24"><h4 class="pmc_sec_title">
Polysome profiling
</h4>
<p>Sucrose gradients (10 to 50%) were prepared by mixing 10 and 50% sucrose solutions [prepared in 20 mM tris-acetate (pH 7.5), 150 mM NaCl, and 4 mM MgCl<sub>2</sub>], using a Biocomp Gradient Master Base Unit (Wolf Laboratories), according to the manufacturer’s instructions. Before usage, gradients were stored at 4°C. Cells were cultured until ~70 to 80% confluent, unless alternative confluency was unavoidable (i.e., neural induction), and then washed once in ice-cold PBS + CHX (100 μg/ml) before lysis in lysis buffer [20 mM tris-HCl, 150 mM NaCl, 4 mM MgCl<sub>2</sub>, 1% Triton X-100, 1 mM DTT, 1x protease inhibitor cocktail (EDTA-free), 1x RNase inhibitor (RiboLock and SUPERase·In), and CHX (100 μg/ml)]. Nuclei were cleared with a brief centrifugation at 500<em>g</em>; the supernatant was then further cleared by centrifugation at max speed to remove any insoluble material. In the case of EDTA treatment, 30 mM EDTA was added to the cleared lysate and incubated on ice for 30 min. Samples, containing equal quantities of RNA, quantified using the QuantiFluor RNA system (Promega), were then loaded onto the sucrose gradient and subjected to ultracentrifugation in a Beckman SW41 rotor at 39,000 rpm for 1.5 hours at 4°C. Fractions were collected using the Biocomp machine (Wolf Laboratories) and on-screen profiles to guide manual dispensal. Overnight RNA precipitation using 0.1 volume of 3 M sodium acetate and 2.5 volumes of 100% ethanol was followed by Turbo DNase (Invitrogen) treatment and lastly TRIzol:chloroform RNA extraction.</p></section><section id="sec25"><h4 class="pmc_sec_title">
Splice-modulating ASOs
</h4>
<p>Before experimentation, all ASOs were dissolved in nuclease-free H<sub>2</sub>O and the concentration was determined by OD<sub>260</sub> absorbance. PS backbone–, 2′MOE-modified 18-nucleotide ASOs were designed (IONIS Pharmaceuticals) to target the SFPQ gene and modulate splicing through steric blocking, without recruitment of RNase H.</p>
<p>For the ASO screen, HEK293T cells were seeded onto 24-well plates such that the confluency was around 30% the next day. Lipofectamine 2000 and Opti-MEM were used, according to the manufacturer’s guidelines, to transfect 200 nM ASOs, and cells were harvested 48 hours after transfection. Expression levels for target mRNAs were compared to mock treatment in which Lipofectamine reagent, but no ASO, was introduced. For follow-up experiments using a candidate ASO, nontargeting control (NC2; IDT) was additionally included, having the same nucleotide length and chemistry, to account for nonspecific effects.</p></section><section id="sec26"><h4 class="pmc_sec_title">
RNA extraction and RT-qPCR
</h4>
<p>Cells were washed with PBS, and then total RNA was typically extracted using the Maxwell RSC simplyRNA Cells kit (Promega), using associated reagents including DNase treatment and a Maxwell RSC 48 machine, unless otherwise specified in associated methods. RNA concentrations were determined via NanoDrop; 260/280 and 260/230 ratios provided estimates of purity. Total RNA was reverse transcribed using the RevertAid First Strand cDNA synthesis kit (Thermo Fisher Scientific) or Superscript IV (Thermo Fisher Scientific) using random hexamer primers. qPCR was performed using the PowerUP SYBR Green Master Mix (Thermo Fisher Scientific) and the QuantStudio Flex Real-Time PCR System (Applied Biosystems). Primers used are listed in table S4. Before experimental use all primers used underwent quality control testing, involving assessment of melting curves, amplification characteristics, and agarose gel electrophoresis of the PCR product. Within each experiment, an RT-minus sample was used as a negative control. The relative transcript expression level was determined through normalization of target expression to that of a housekeeping gene(s); where multiple housekeeping genes were used, the geometric mean of these is used for normalization. “Proportional transcript isoform expression” (e.g., percent spliced in/splicing ratio) was determined through normalization of the target isoform to the expression of a constitutively expressed region within the host gene. Data were analyzed according to the ddCt method, typically expressed as FC over the control/untreated group.</p></section><section id="sec27"><h4 class="pmc_sec_title">
Protein extraction, quantification, and Western blotting
</h4>
<p>To extract total protein for Western blotting, cells were lysed in RIPA (radioimmunoprecipitation assay) buffer [50 mM tris-HCl (pH 7.4), 150 mM NaCl, 0.5% sodium deoxycholate, and 1% NP-40], supplemented with Halt Protease Inhibitor Cocktail for 15 min on ice, followed by 10 cycles of sonication (Bioruptor) with alternating 30-s on/off. Following a further 15-min incubation on ice, lysates were cleared by centrifugation at max speed and 4°C. The protein was quantified using DC Assay (Bio-Rad) as per the manufacturer’s instructions before downstream processing. When necessary, protein samples were concentrated using methanol-chloroform precipitation. A 4x NuPAGE LDS sample buffer and 50 mM DTT final concentration was added to the protein sample and placed at 70°C for 10 min for denaturation. Samples were loaded onto Invitrogen NuPAGE 4 to 12% Bis-Tris or 7% tris-acetate protein gels and run at 100 to 180 V for 1 to 5 hours in NuPAGE MOPS or tris-acetate running buffer, respectively. The protein was then transferred onto a polyvinylidene difluoride (Immobilon F-PL) membrane at 330 mA for 90 min at 4°C. Membranes were blocked for 1 hour in 5% milk in 0.1% PBS–Tween 20 (PBS-T) and then incubated with primary antibodies (see table S5) for 1 hour at room temperature or overnight at 4°C. After three washes in PBS-T, IRdye 680 and 800CW secondary antibodies (1:10,000; LI-COR Biosciences), in PBS-T, were added as appropriate for 1 hour at room temperature. Following a further three washes, membranes were imaged using a LI-COR FC Odyssey CLx. Blot densitometry was analyzed using ImageStudio Lite 5.2 or EmpiriaStudio.</p></section><section id="sec28"><h4 class="pmc_sec_title">
RNA immunoprecipitation
</h4>
<p>Cells were lysed in lysis buffer [50 mM Hepes (pH 7.4), 150 mM NaCl,1% IGEPAL, and 1 mM EDTA] supplemented with RiboLock RNase Inhibitor (0.1 U/μl) and Halt Protease Inhibitor Cocktail and subjected to 10 cycles of sonication (Bioruptor) with alternating 30-s on/off. Lysates were cleared of insoluble material by centrifugation and quantified using DC Assay. One thousand micrograms of the total protein lysate was prepared for each condition. The lysate (10%) was kept aside for total input RNA extraction. Pierce anti-HA magnetic beads (1 mg/ml; Thermo Fisher Scientific) were washed two times with lysis buffer and then added to the protein lysate and rotated for 2 hours at 4°C. Beads were washed four times with lysis buffer and once with PBS each at 4°C followed by elution of RNA in extraction buffer (0.2 M sodium acetate, 1 mM EDTA, and 0.2% SDS), supplemented with RiboLock RNase Inhibitor (0.1 U/μl), for 5 min at 70°C. Total input RNA and immunoprecipitated RNA were purified using the PureLink RNA Micro Scale Kit according to the manufacturer’s instructions with on-column DNase digestion, reverse transcribed using Superscript IV and random hexamers, and then analyzed by qPCR as described.</p></section><section id="sec29"><h4 class="pmc_sec_title">
Immunoprecipitation
</h4>
<p>For SFPQ IP, 0.71 μg of C-terminal SFPQ antibody (Abcam, ab177149) was incubated with prewashed protein G Dynabeads (Invitrogen) in lysis buffer [50 mM tris-HCl (pH 7.4), 150 mM NaCl, 1% IGEPAL + 1 mM EDTA + HALT Protease Inhibitor Cocktail] for 1 hour at 4°C. Eighty to 100 μg of the HEK293T cell lysate, generated by incubating cells with lysis buffer and 10 cycles of sonication (Bioruptor) with alternating 30-s on/off, was added to antibody-conjugated Dynabeads and rotated overnight at 4°C. The input lysate and three times washed beads were boiled at 95°C in 1x LDS buffer + 50 mM DTT. The cleared supernatant was subjected to methanol-chloroform precipitation, before boiling in 1x LDS buffer + 50 mM DTT, to concentrate the protein to enable adequate loading quantity for Western blotting. For HA-SFPQ co-IP, cells were lysed in lysis buffer [50 mM Hepes (pH 7.4), 150 mM NaCl, 1% IGEPAL, and 1 mM EDTA] supplemented with Halt Protease Inhibitor Cocktail and subjected to 10 cycles of sonication (Bioruptor) with alternating 30-s on/off. Lysates were cleared of insoluble material by centrifugation and quantified using DC Assay. One thousand micrograms of the total protein lysate was prepared for each condition. Some lysate was kept aside as an input. Pierce anti-HA magnetic beads (1 mg/ml; Thermo Fisher Scientific) were washed two times with lysis buffer and then added to the protein lysate and rotated overnight at 4°C. The supernatant was kept aside as cleared material. Beads were washed four times with lysis buffer and once with PBS each at 4°C followed by storage at −80°C for MS or boiled at 95°C in 1x LDS buffer + 50 mM DTT and resolved by SDS-PAGE.</p></section><section id="sec30"><h4 class="pmc_sec_title">
Immunocytochemistry
</h4>
<p>HEK293T cells were plated onto poly-<span class="font-variant-small-caps">d</span>-lysine–coated IBIDI 8-well chamber slides (IBIDI) or 96-well plates (Thermo Fisher Scientific). Before cell staining, cells were fixed using 4% paraformaldehyde for 10 min at room temperature and then washed and maintained in PBS. Fixed cells were permeabilized by 0.3% Triton X-100 in PBS for 10 min and then blocked with 5% bovine serum albumin (BSA) (Sigma-Aldrich) in PBS for 1 hour at room temperature. Immunolabeling was performed with primary antibodies in 5% BSA at 4°C overnight, followed by species-specific Alexa Fluor–conjugated secondary antibodies (1:1000) in 5% BSA for 1 hour at room temperature. 4′,6-Diamidino-2-phenylindole (DAPI) counterstain (100 ng/ml) was subsequently applied for 10 min at room temperature, and then cells were maintained in PBS until imaged. Primary antibodies used are detailed in table S5. Microscopy was carried out using an 880 laser scanning confocal microscope (Zeiss), at 63x objective, 1.4–numerical aperture oil immersion objective, or on an Opera Phenix at 40x objective.</p></section><section id="sec31"><h4 class="pmc_sec_title">
BaseScope ISH analysis
</h4>
<p>Cells cultured on 8-well chamber slides (Thermo Fisher Scientific) were fixed in 10% neutral buffered formalin, dehydrated through increasing concentrations of ethanol, and stored in 100% ethanol at −20°C. On the day of assaying, slides were air-dried, loaded onto the BOND RX (Leica Biosystems), and stained according to the BaseScope LS Reagent Assay protocol. Briefly, samples were rehydrated, subjected to 5-min protease (ACDbio) pretreatment at room temperature to retrieve RNA, and incubated with custom wtSFPQ (TV1) or altSFPQ (TV2) 1xZZ BaseScope probes for 2 hours at 40°C. Probe design and manufacture were carried out by ACDbio/Bio-Techne; probes were designed to span the exon-exon junctions unique to either <em>wtSFPQ</em> or <em>altSFPQ</em>. The TV1 probe targets <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_005066.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_005066.3</a> (transcript variant 1) and should not target <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_136702.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_136702.2</a> (transcript variant 2) or <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_136703.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_136703.2</a> (transcript variant 3), whereas the TV2 probe targets both <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_136702.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_136702.2</a> (tv2) and <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_136703.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NR_136703.2</a> (tv3) and should not target <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_005066.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_005066.3</a> (tv1). RNA signal visualization was achieved with AP–Fast Red, and cell nuclei were counterstained with DAPI. Slides were mounted using Vectamount aqueous mourning medium (Vector), and images were acquired on an 880 laser scanning confocal microscope (Zeiss) at 40x objective. Images were initially processed in ImageJ to max project and segment nuclei (StarDist plugin), and then a CellProfiler pipeline was used to identify SFPQ RNA foci using a minimal intensity threshold filter.</p></section><section id="sec32"><h4 class="pmc_sec_title">
Statistics
</h4>
<p>Statistical analysis and associated graph generation were conducted using GraphPad Prism 8. Statistical tests used for each experimental dataset are reported within the figure legends, when significant differences were identified, alongside repeat numbers. When comparing between two sets of data, an unpaired two-tailed Student’s <em>t</em> test was used. When comparing between more than two groups, one-way analysis of variance (ANOVA) with Tukey or Dunnett’s post hoc comparison was used, assuming normal distribution. Data are presented as means ± SEM. Statistical significance was accepted at a <em>P</em> value of &lt;0.05 and either displayed as actual value or star (*<em>P</em> ≤ 0.05, **<em>P</em> ≤ 0.01, ***<em>P</em> ≤ 0.001, or ****<em>P</em> ≤ 0.0001).</p></section><section id="sec33"><h4 class="pmc_sec_title">
RNA-seq analysis
</h4>
<section id="sec34"><h5 class="pmc_sec_title">
RNA-seq processing, integration, and quality control
</h5>
<p>Raw RNA-seq reads (fastq files) and accompanying metadata were downloaded using the nfcore/fetchngs v1.9 pipeline (<a href="#R92" class="usa-link" aria-describedby="R92"><em>92</em></a>) and pysradb v1.3 using the sample sequence read archive accession number. Reads were processed using the nfcore/rnaseq v3.9 pipeline (<a href="#R92" class="usa-link" aria-describedby="R92"><em>92</em></a>). Raw reads underwent adaptor trimming with Trim Galore, removal of ribosomal RNA with SortMeRNA, alignment to Ensembl GRCh38.99 human reference genome using splice-aware aligner, STAR v2.7.1, and BAM-level quantification with Salmon. Gene counts were normalized for library size and transformed on a log<sub>2</sub> scale using the variance stabilizing transformation function in DESeq2. We excluded three control samples in Answer ALS that whole-genome sequencing revealed to have pathogenic ALS mutations. In addition, four Answer ALS iPSMNs from patients with non-ALS motor neuron diseases were excluded. NeuroLINCS consists of three distinct iPSC protocols [induced motor neurons (iMNs), directly induced motor neurons (diMNs), and undifferentiated iPSCs], of which only the iMN and diMN batches were included.</p></section><section id="sec35"><h5 class="pmc_sec_title">
Alternative splicing analysis
</h5>
<p>All modes of alternative splicing were analyzed using MAJIQ v2.4 (<a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>, <a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>) on iPSMN RNA library samples. STAR-aligned BAMs were used as an input to the MAJIQ splice graph builder using Ensembl GRCh38.99 transcript annotation. For Tyzack <em>et al.</em> (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>), VCP-ALS; Kapeli <em>et al.</em> (<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>), FUS-ALS; and Kiskinis <em>et al.</em> (<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>), SOD1-ALS datasets, differential splicing was calculated using the MAJIQ deltaPSI function, which is most appropriate for small single-batch homogeneous experimental replicate datasets. For nonmutant sporadic ALS datasets, Answer ALS and NeuroLINCS, differential splicing was calculated using the MAJIQ heterogen function, designed for examining splicing across large and heterogeneous datasets. A threshold of TNOM (thresholded nonparametric overlap measure) <em>P</em> value &lt; 0.05 was used to call significant splicing changes between groups. Changes in splicing were examined using the Voila modulize function that breaks down the complex local splice variants into the classic binary splicing events (e.g., exon skipping or IR).</p></section><section id="sec36"><h5 class="pmc_sec_title">
Data availability
</h5>
<p>HiPSC-derived motor neuron raw sequencing data used in this study are available in public repositories under the following accession numbers: Tyzack <em>et al.</em> [(<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>); <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE152983" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE152983</a>], Kapeli <em>et al.</em> [(<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>); <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE77702" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE77702</a>], Kiskinis <em>et al.</em> [(<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>); <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54409" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE54409</a>], NeuroLINCS Consortium <em>et al.</em> [(<a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>); phs001231.v2.p1], and Baxi <em>et al.</em> [(<a href="#R58" class="usa-link" aria-describedby="R58"><em>58</em></a>); Answer ALS data portal]. Some raw data have restricted access (NeuroLINCS dbGaP accession number: phs0001231.v2.p1; Answer ALS database). Granting access to these is beyond the control of the authors. Access can be obtained by applying to the relevant Data Access Committees. Answer ALS requires a signed data use agreement to have full access.</p></section></section></section><section id="sec37"><h3 class="pmc_sec_title">Mass spectrometry</h3>
<p>For on-bead digestion of the pull-down samples, 100 μl of 100 mM triethylammonium bicarbonate (TEAB) was added to the beads followed by simultaneous reduction and alkylation and trypsin digestion with 5 mM tris(2-carboxyethyl)phosphine, 10 mM IAA (indole-3-acetic acid), and a trypsin concentration of 50 ng/μl with overnight shaking at 900 rpm at room temperature. The bead-free digests were SpeedVac dried and reconstituted in 25 μl of 100 mM TEAB, and half of the volume was labeled with the TMTpro reagents by adding 5 μl of the reagent (25 μg/μl). The peptide pool was fractionated with high-pH reversed-phase chromatography using the XBridge C18 column (1.0 mm by 100 mm, 3.5 μm, Waters) on an UltiMate 3000 HPLC system over a 1% gradient in 35 min. Mobile phase A was 0.1% (v/v) ammonium hydroxide, and mobile phase B was 0.1% ammonium hydroxide (v/v) in acetonitrile. Peptide fractions were pooled into eight samples for liquid chromatography–mass spectrometry (LC-MS) analysis. LC-MS analysis was performed on an UltiMate 3000 system coupled with the Orbitrap Ascend mass spectrometer (Thermo Fisher Scientific) using a 25-cm capillary column (Waters, nanoE MZ PST BEH130 C18, 1.7 μm, 75 μm by 250 mm) over a 100-min gradient 5 to 35% of mobile phase B composed of 80% acetonitrile and 0.1% formic acid. Peptides were preconcentrated onto an Acclaim PepMap 100 (100 μm by 2 cm C18, 5 μm) trapping column at 10 μl/min of 0.1% trifluoroacetic acid, and the analytical column was connected to an EASY-Spray emitter (Thermo Fisher Scientific, ES991). MS spectra were collected at an Orbitrap mass resolution of 120k, and precursors were targeted for higher-energy collision dissociation (HCD) fragmentation in the top speed mode (3 s) with a collision energy of 32% and ion trap detection in turbo scan rate. MS3 scans were triggered by Real-Time Search (RTS) against a fasta file containing UniProt <em>Mus musculus</em> reviewed canonical and isoform sequences concatenated with the variant sequences of human SFPQ (HA-tag), with multinotch isolation (10 notches) and HCD fragmentation with a collision energy of 55% at 45k Orbitrap resolution. Targeted precursors were dynamically excluded from further activation for 45 s with a mass tolerance of 10 parts per million (ppm), and RTS close-out was disabled. Static modifications for RTS were TMTpro16plex at K/n-term (+304.2071) and carbamidomethyl at C (+57.0215), and variable modifications were deamidated NQ (+0.984) and oxidation of M (+15.9949) with maximum of one missed cleavage and two variable modifications per peptide. The Sequest HT and Comet nodes in Proteome Discoverer 3.0 (Thermo Fisher Scientific) were used to search the raw mass spectra against the same fasta file used in the RTS analysis. The precursor mass tolerance was set at 20 ppm and the fragment ion mass tolerance at 0.5 Da (or 1 Da for Comet) with up to two trypsin missed cleavages allowed. TMTpro at N terminus/K and carbamidomethyl at C were defined as static modifications. Dynamic modifications were oxidation of M and deamidation of N/Q. Peptide confidence was estimated with the Percolator node, and peptide FDR (false discovery rate) was set at 0.01 based on target-decoy search. Only unique peptides were used for quantification, considering protein groups for peptide uniqueness. Peptides with an average reporter signal-to-noise greater than 3 were used for protein quantification.</p>
<section id="sec38"><h4 class="pmc_sec_title">
Data availability
</h4>
<p>The MS proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE (<a href="#R93" class="usa-link" aria-describedby="R93"><em>93</em></a>) partner repository with the dataset identifier PXD051918.</p></section><section id="sec39"><h4 class="pmc_sec_title">
LC-MS/MS analysis
</h4>
<p>Statistical data analysis was performed in the Perseus software (<a href="#R94" class="usa-link" aria-describedby="R94"><em>94</em></a>). Normalized protein abundances were exported from Proteome Discoverer and scaled to row mean (log<sub>2</sub>) per replicate batch. Two-sample <em>t</em> tests were then performed for WT, ALT, and ΔNLS against the EV samples to identify enriched proteins for each hSFPQ bait protein under both untreated and oxidative stress conditions. In addition, the hSFPQ protein profile was regressed out from the log<sub>2</sub> scaled data for WT, ALT, and ΔNLS, excluding EVs from the regression; i.e., a linear regression model was applied, using hSFPQ relative abundances as the independent variable to normalize target protein abundances, assuming a linear relationship between bait and prey signals. The hSFPQ scaled log<sub>2</sub> values were used as the independent variable (<em>x</em>), and the log<sub>2</sub> scaled values of each protein were used as the dependent variable (<em>y</em>). The residuals of the <em>y</em> = <em>ax</em> + <em>b</em> (<em>a</em> is the slope, and <em>b</em> is the intercept) model for each protein were used as the corrected relative log<sub>2</sub> ratios. Pearson correlation values for each protein to hSFPQ were also computed. Matched log<sub>2</sub> ratios comparing ALT and ΔNLS against WT as well as ΔNLS versus ALT were further computed followed by one-sample <em>t</em> tests to identify differences between the hSFPQ conditions. Last, π values (log<sub>2</sub>FC × −log<sub>10</sub><em>P</em> value) were computed (<a href="#R95" class="usa-link" aria-describedby="R95"><em>95</em></a>) only for proteins with positive correlation to hSFPQ (Pearson &gt; 0.3) as a means of significance ranking. The ranking scores were used for network cluster enrichment in STRING (<a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>).</p></section></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p><strong>Funding:</strong> We wish to thank the control donors and patients for fibroblast donations. This work was supported by the Francis Crick Institute, which receives its core funding from Cancer Research UK (FC010110), the UK Medical Research Council (FC010110), and the Wellcome Trust (FC010110). This work was also supported by an MRC Senior Clinical Fellowship (MR/S006591/1) and a Lister Institute Research Prize Fellowship to R.P. Furthermore, R.P. gratefully acknowledges the generous support from S. Redgwell, L. Iles, Challenging MND, the Motor Neuron Disease Association (Patani/Dec22/957-793), My Name’5 Doddie Foundation (MN5DF/2022/003), and Target ALS (BB-2024-C4-L4). This research and related results were also made possible through the support of the NOMIS Foundation (to M.-D.R.) as well as the UK Dementia Research Institute (UK DRI-6005, to M.-D.R.) through UK DRI Ltd., principally funded by the UK Medical Research Council. We are grateful for the help and support provided by the following platforms at the Francis Crick Institute: Cell Services, High Throughput Imaging (in particular, C. Queval), Vectorcore, and Experimental Histopathology (in particular, L. Meader and M. Green). We thank all members of the Patani lab for helpful discussion and inputs.</p>
<section id="sec40"><p><strong>Author contributions:</strong> Conceptualization: J.N. and R.P. Investigation: J.N., M.P.H., O.J.Z., B.C., and D.J. Methodology: J.N., M.P.H., D.J., T.I.R., and R.P. Validation: J.N., M.P.H., and B.C. Formal analysis: J.N., O.J.Z., K.P., T.I.R., and J.C. Software: O.J.Z., K.P., and T.I.R. Data curation: J.N., O.J.Z., and T.I.R. Project administration: J.N. and R.P. Writing—original draft: J.N. Writing—review and editing: J.N. and R.P. with contributions by all coauthors. Visualization: J.N. and O.J.Z. Resources: A.M.I., F.R., C.F.B., M.-D.R., and R.P. Funding acquisition: M.-D.R. and R.P. Supervision: M.-D.R. and R.P.</p></section><section id="sec41"><p><strong>Competing interests:</strong> F.R. and C.F.B. are employees of Ionis Pharmaceuticals. All other authors declare that they have no competing interests.</p></section><section id="sec42"><p><strong>Data and materials availability:</strong> All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials. Accession numbers for reanalyzed public RNA-seq datasets have been stated in the appropriate Materials and Methods section. The MS proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE partner repository with the dataset identifier PXD051918. Last, all other raw data associated with this study have been deposited in the database Dryad (DOI:<a href="http://dx.doi.org/10.5061/dryad.gb5mkkx1z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.5061/dryad.gb5mkkx1z</a>). The VCP mutant iPSC lines can be provided by S. Wray at UCL pending scientific review and a completed material transfer agreement (MTA). Requests for these should be submitted to UCL Business (<span>info@uclb.com</span>). The BS6 2H9 line is subject to an MTA with the University of Edinburgh. This line can be provided by the University of Edinburgh pending review and a completed MTA. Requests for the BS6 2H9 line should be submitted to S. Chandran (University of Edinburgh). The ASOs targeting <em>SFPQ</em> are subject to an MTA to R. Patani from Ionis Pharmaceuticals. These ASOs can be provided by Ionis Pharmaceuticals pending scientific review and a completed MTA. Requests for the ASOs targeting <em>SFPQ</em> should be submitted to P. Jafar-Nejad (<span>pjafar-nejad@ionis.com</span>).</p></section></section><section id="sec43"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section id="sec44"><h3 class="pmc_sec_title">The PDF file includes:</h3>
<section class="sm xbox font-sm" id="supplementary-material1"><div class="caption p">
<p>Figs. S1 to S8</p>
<p>Legends for tables S1 and S2</p>
<p>Tables S3 to S5</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372870/bin/sciadv.adt4814_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adt4814_sm.pdf</a><sup> (6MB, pdf) </sup>
</div></div></section></section><section id="sec45"><h3 class="pmc_sec_title">Other Supplementary Material for this manuscript includes the following:</h3>
<section class="sm xbox font-sm" id="supplementary-material2"><div class="caption p"><p>Tables S1 and S2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372870/bin/sciadv.adt4814_tables_s1_and_s2.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adt4814_tables_s1_and_s2.zip</a><sup> (7.1MB, zip) </sup>
</div></div></section></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES AND NOTES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Neumann M., Sampathu D. M., Kwong L. K., Truax A. C., Micsenyi M. C., Chou T. T., Bruce J., Schuck T., Grossman M., Clark C. M., McCluskey L. F., Miller B. L., Masliah E., Mackenzie I. R., Feldman H., Feiden W., Kretzschmar H. A., Trojanowski J. Q., Lee V. M., 
Ubiquitinated TDP-43 in frontotemporal lobar degeneration and amyotrophic lateral sclerosis. Science
314, 
130–133 (2006).
</cite> [<a href="https://doi.org/10.1126/science.1134108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17023659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Ubiquitinated%20TDP-43%20in%20frontotemporal%20lobar%20degeneration%20and%20amyotrophic%20lateral%20sclerosis&amp;volume=314&amp;publication_year=2006&amp;pages=130-133&amp;pmid=17023659&amp;doi=10.1126/science.1134108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Kwiatkowski T. J. Jr., Bosco D. A., Leclerc A. L., Tamrazian E., Vanderburg C. R., Russ C., Davis A., Gilchrist J., Kasarskis E. J., Munsat T., Valdmanis P., Rouleau G. A., Hosler B. A., Cortelli P., de Jong P. J., Yoshinaga Y., Haines J. L., Pericak-Vance M. A., Yan J., Ticozzi N., Siddique T., McKenna-Yasek D., Sapp P. C., Horvitz H. R., Landers J. E., Brown R. H. Jr., 
Mutations in the FUS/TLS gene on chromosome 16 cause familial amyotrophic lateral sclerosis. Science
323, 
1205–1208 (2009).
</cite> [<a href="https://doi.org/10.1126/science.1166066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19251627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Mutations%20in%20the%20FUS/TLS%20gene%20on%20chromosome%2016%20cause%20familial%20amyotrophic%20lateral%20sclerosis&amp;volume=323&amp;publication_year=2009&amp;pages=1205-1208&amp;pmid=19251627&amp;doi=10.1126/science.1166066&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Vance C., Rogelj B., Hortobágyi T., De Vos K. J., Nishimura A. L., Sreedharan J., Hu X., Smith B., Ruddy D., Wright P., Ganesalingam J., Williams K. L., Tripathi V., Al-Saraj S., Al-Chalabi A., Leigh P. N., Blair I. P., Nicholson G., de Belleroche J., Gallo J.-M., Miller C. C., Shaw C. E., 
Mutations in FUS, an RNA processing protein, cause familial amyotrophic lateral sclerosis type 6. Science
323, 
1208–1211 (2009).
</cite> [<a href="https://doi.org/10.1126/science.1165942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4516382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19251628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Mutations%20in%20FUS,%20an%20RNA%20processing%20protein,%20cause%20familial%20amyotrophic%20lateral%20sclerosis%20type%206&amp;volume=323&amp;publication_year=2009&amp;pages=1208-1211&amp;pmid=19251628&amp;doi=10.1126/science.1165942&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Tyzack G. E., Luisier R., Taha D. M., Neeves J., Modic M., Mitchell J. S., Meyer I., Greensmith L., Newcombe J., Ule J., Luscombe N. M., Patani R., 
Widespread FUS mislocalization is a molecular hallmark of amyotrophic lateral sclerosis. Brain
142, 
2572–2580 (2019).
</cite> [<a href="https://doi.org/10.1093/brain/awz217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6735815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31368485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Widespread%20FUS%20mislocalization%20is%20a%20molecular%20hallmark%20of%20amyotrophic%20lateral%20sclerosis&amp;volume=142&amp;publication_year=2019&amp;pages=2572-2580&amp;pmid=31368485&amp;doi=10.1093/brain/awz217&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Luisier R., Tyzack G. E., Hall C. E., Mitchell J. S., Devine H., Taha D. M., Malik B., Meyer I., Greensmith L., Newcombe J., Ule J., Luscombe N. M., Patani R., 
Intron retention and nuclear loss of SFPQ are molecular hallmarks of ALS. Nat. Commun.
9, 
2010 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-04373-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5964114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29789581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Intron%20retention%20and%20nuclear%20loss%20of%20SFPQ%20are%20molecular%20hallmarks%20of%20ALS&amp;volume=9&amp;publication_year=2018&amp;pages=2010&amp;pmid=29789581&amp;doi=10.1038/s41467-018-04373-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Yarosh C. A., Iacona J. R., Lutz C. S., Lynch K. W., 
PSF: Nuclear busy-body or nuclear facilitator?
Wiley Interdiscip. Rev. RNA
6, 
351–367 (2015).
</cite> [<a href="https://doi.org/10.1002/wrna.1280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4478221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25832716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Wiley%20Interdiscip.%20Rev.%20RNA&amp;title=PSF:%20Nuclear%20busy-body%20or%20nuclear%20facilitator?&amp;volume=6&amp;publication_year=2015&amp;pages=351-367&amp;pmid=25832716&amp;doi=10.1002/wrna.1280&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Roepcke S., Stahlberg S., Klein H., Schulz M. H., Theobald L., Gohlke S., Vingron M., Walther D. J., 
A tandem sequence motif acts as a distance-dependent enhancer in a set of genes involved in translation by binding the proteins NonO and SFPQ. BMC Genomics
12, 
624 (2011).
</cite> [<a href="https://doi.org/10.1186/1471-2164-12-624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3262029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22185324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=A%20tandem%20sequence%20motif%20acts%20as%20a%20distance-dependent%20enhancer%20in%20a%20set%20of%20genes%20involved%20in%20translation%20by%20binding%20the%20proteins%20NonO%20and%20SFPQ&amp;volume=12&amp;publication_year=2011&amp;pages=624&amp;pmid=22185324&amp;doi=10.1186/1471-2164-12-624&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Heyd F., Lynch K. W., 
Phosphorylation-dependent regulation of PSF by GSK3 controls CD45 alternative splicing. Mol. Cell
40, 
126–137 (2010).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2010.09.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2954053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20932480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Phosphorylation-dependent%20regulation%20of%20PSF%20by%20GSK3%20controls%20CD45%20alternative%20splicing&amp;volume=40&amp;publication_year=2010&amp;pages=126-137&amp;pmid=20932480&amp;doi=10.1016/j.molcel.2010.09.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Fox A. H., Lam Y. W., Leung A. K. L., Lyon C. E., Andersen J., Mann M., Lamond A. I., 
Paraspeckles: A novel nuclear domain. Curr. Biol.
12, 
13–25 (2002).
</cite> [<a href="https://doi.org/10.1016/s0960-9822(01)00632-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11790299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=Paraspeckles:%20A%20novel%20nuclear%20domain&amp;volume=12&amp;publication_year=2002&amp;pages=13-25&amp;pmid=11790299&amp;doi=10.1016/s0960-9822(01)00632-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Gordon P. M., Hamid F., Makeyev E. V., Houart C., 
A conserved role for the ALS-linked splicing factor SFPQ in repression of pathogenic cryptic last exons. Nat. Commun.
12, 
1918 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-22098-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7997972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33771997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=A%20conserved%20role%20for%20the%20ALS-linked%20splicing%20factor%20SFPQ%20in%20repression%20of%20pathogenic%20cryptic%20last%20exons&amp;volume=12&amp;publication_year=2021&amp;pages=1918&amp;pmid=33771997&amp;doi=10.1038/s41467-021-22098-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Takeuchi A., Iida K., Tsubota T., Hosokawa M., Denawa M., Brown J. B., Ninomiya K., Ito M., Kimura H., Abe T., Kiyonari H., Ohno K., Hagiwara M., 
Loss of Sfpq causes long-gene transcriptopathy in the brain. Cell Rep.
23, 
1326–1341 (2018).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.03.141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29719248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=Loss%20of%20Sfpq%20causes%20long-gene%20transcriptopathy%20in%20the%20brain&amp;volume=23&amp;publication_year=2018&amp;pages=1326-1341&amp;pmid=29719248&amp;doi=10.1016/j.celrep.2018.03.141&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Ishigaki S., Fujioka Y., Okada Y., Riku Y., Udagawa T., Honda D., Yokoi S., Endo K., Ikenaka K., Takagi S., Iguchi Y., Sahara N., Takashima A., Okano H., Yoshida M., Warita H., Aoki M., Watanabe H., Okado H., Katsuno M., Sobue G., 
Altered tau isoform ratio caused by loss of FUS and SFPQ function leads to FTLD-like phenotypes. Cell Rep.
18, 
1118–1131 (2017).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.01.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28147269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=Altered%20tau%20isoform%20ratio%20caused%20by%20loss%20of%20FUS%20and%20SFPQ%20function%20leads%20to%20FTLD-like%20phenotypes&amp;volume=18&amp;publication_year=2017&amp;pages=1118-1131&amp;pmid=28147269&amp;doi=10.1016/j.celrep.2017.01.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Hung C., Patani R., 
Elevated 4R tau contributes to endolysosomal dysfunction and neurodegeneration in VCP-related frontotemporal dementia. Brain
147, 
970–979 (2024).
</cite> [<a href="https://doi.org/10.1093/brain/awad370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10907086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37882537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Elevated%204R%20tau%20contributes%20to%20endolysosomal%20dysfunction%20and%20neurodegeneration%20in%20VCP-related%20frontotemporal%20dementia&amp;volume=147&amp;publication_year=2024&amp;pages=970-979&amp;pmid=37882537&amp;doi=10.1093/brain/awad370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Kanai Y., Dohmae N., Hirokawa N., 
Kinesin transports RNA: Isolation and characterization of an RNA-transporting granule. Neuron
43, 
513–525 (2004).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2004.07.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15312650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Kinesin%20transports%20RNA:%20Isolation%20and%20characterization%20of%20an%20RNA-transporting%20granule&amp;volume=43&amp;publication_year=2004&amp;pages=513-525&amp;pmid=15312650&amp;doi=10.1016/j.neuron.2004.07.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Furukawa M. T., Sakamoto H., Inoue K., 
Interaction and colocalization of HERMES/RBPMS with NonO, PSF, and G3BP1 in neuronal cytoplasmic RNP granules in mouse retinal line cells. Genes Cells
20, 
257–266 (2015).
</cite> [<a href="https://doi.org/10.1111/gtc.12224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25651939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Cells&amp;title=Interaction%20and%20colocalization%20of%20HERMES/RBPMS%20with%20NonO,%20PSF,%20and%20G3BP1%20in%20neuronal%20cytoplasmic%20RNP%20granules%20in%20mouse%20retinal%20line%20cells&amp;volume=20&amp;publication_year=2015&amp;pages=257-266&amp;pmid=25651939&amp;doi=10.1111/gtc.12224&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Cosker K. E., Fenstermacher S. J., Pazyra-Murphy M. F., Elliott H. L., Segal R. A., 
The RNA-binding protein SFPQ orchestrates an RNA regulon to promote axon viability. Nat. Neurosci.
19, 
690–696 (2016).
</cite> [<a href="https://doi.org/10.1038/nn.4280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5505173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27019013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Neurosci.&amp;title=The%20RNA-binding%20protein%20SFPQ%20orchestrates%20an%20RNA%20regulon%20to%20promote%20axon%20viability&amp;volume=19&amp;publication_year=2016&amp;pages=690-696&amp;pmid=27019013&amp;doi=10.1038/nn.4280&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Thomas-Jinu S., Gordon P. M., Fielding T., Taylor R., Smith B. N., Snowden V., Blanc E., Vance C., Topp S., Wong C.-H., Bielen H., Williams K. L., McCann E. P., Nicholson G. A., Pan-Vazquez A., Fox A. H., Bond C. S., Talbot W. S., Blair I. P., Shaw C. E., Houart C., 
Non-nuclear pool of splicing factor SFPQ regulates axonal transcripts required for normal motor development. Neuron
94, 
322–336.e5 (2017).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2017.03.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5405110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28392072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Non-nuclear%20pool%20of%20splicing%20factor%20SFPQ%20regulates%20axonal%20transcripts%20required%20for%20normal%20motor%20development&amp;volume=94&amp;publication_year=2017&amp;pages=322-336.e5&amp;pmid=28392072&amp;doi=10.1016/j.neuron.2017.03.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Ke Y. D., Dramiga J., Schütz U., Kril J. J., Ittner L. M., Schröder H., Götz J., 
Tau-mediated nuclear depletion and cytoplasmic accumulation of SFPQ in Alzheimer’s and Pick’s disease. PLOS ONE
7, 
e35678 (2012).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0035678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3338448/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22558197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&amp;title=Tau-mediated%20nuclear%20depletion%20and%20cytoplasmic%20accumulation%20of%20SFPQ%20in%20Alzheimer%E2%80%99s%20and%20Pick%E2%80%99s%20disease&amp;volume=7&amp;publication_year=2012&amp;pages=e35678&amp;pmid=22558197&amp;doi=10.1371/journal.pone.0035678&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Widagdo J., Udagedara S., Bhembre N., Tan J. Z. A., Neureiter L., Huang J., Anggono V., Lee M., 
Familial ALS-associated <em>SFPQ</em> variants promote the formation of SFPQ cytoplasmic aggregates in primary neurons. Open Biol.
12, 
220187 (2022).
</cite> [<a href="https://doi.org/10.1098/rsob.220187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9516340/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36168806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Open%20Biol.&amp;title=Familial%20ALS-associated%20SFPQ%20variants%20promote%20the%20formation%20of%20SFPQ%20cytoplasmic%20aggregates%20in%20primary%20neurons&amp;volume=12&amp;publication_year=2022&amp;pages=220187&amp;pmid=36168806&amp;doi=10.1098/rsob.220187&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Younas N., Zafar S., Shafiq M., Noor A., Siegert A., Arora A. S., Galkin A., Zafar A., Schmitz M., Stadelmann C., Andreoletti O., Ferrer I., Zerr I., 
SFPQ and Tau: Critical factors contributing to rapid progression of Alzheimer’s disease. Acta Neuropathol.
140, 
317–339 (2020).
</cite> [<a href="https://doi.org/10.1007/s00401-020-02178-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7423812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32577828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Neuropathol.&amp;title=SFPQ%20and%20Tau:%20Critical%20factors%20contributing%20to%20rapid%20progression%20of%20Alzheimer%E2%80%99s%20disease&amp;volume=140&amp;publication_year=2020&amp;pages=317-339&amp;pmid=32577828&amp;doi=10.1007/s00401-020-02178-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Lu J., Shu R., Zhu Y., 
Dysregulation and dislocation of SFPQ disturbed DNA organization in Alzheimer’s disease and frontotemporal dementia. J. Alzheimers Dis.
61, 
1311–1321 (2018).
</cite> [<a href="https://doi.org/10.3233/JAD-170659" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29376859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Alzheimers%20Dis.&amp;title=Dysregulation%20and%20dislocation%20of%20SFPQ%20disturbed%20DNA%20organization%20in%20Alzheimer%E2%80%99s%20disease%20and%20frontotemporal%20dementia&amp;volume=61&amp;publication_year=2018&amp;pages=1311-1321&amp;pmid=29376859&amp;doi=10.3233/JAD-170659&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Huang J., Ringuet M., Whitten A. E., Caria S., Lim Y. W., Badhan R., Anggono V., Lee M., 
Structural basis of the zinc-induced cytoplasmic aggregation of the RNA-binding protein SFPQ. Nucleic Acids Res.
48, 
3356–3365 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7102971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32034402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Structural%20basis%20of%20the%20zinc-induced%20cytoplasmic%20aggregation%20of%20the%20RNA-binding%20protein%20SFPQ&amp;volume=48&amp;publication_year=2020&amp;pages=3356-3365&amp;pmid=32034402&amp;doi=10.1093/nar/gkaa076&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Tyzack G. E., Manferrari G., Newcombe J., Luscombe N. M., Luisier R., Patani R., 
Paraspeckle components NONO and PSPC1 are not mislocalized from motor neuron nuclei in sporadic ALS. Brain
143, 
e66 (2020).
</cite> [<a href="https://doi.org/10.1093/brain/awaa205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7447511/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32844195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Paraspeckle%20components%20NONO%20and%20PSPC1%20are%20not%20mislocalized%20from%20motor%20neuron%20nuclei%20in%20sporadic%20ALS&amp;volume=143&amp;publication_year=2020&amp;pages=e66&amp;pmid=32844195&amp;doi=10.1093/brain/awaa205&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Hogan A. L., Grima N., Fifita J. A., McCann E. P., Heng B., Fat S. C. M., Wu S., Maharjan R., Cain A. K., Henden L., Rayner S., Tarr I., Zhang K. Y., Zhao Q., Zhang Z.-H., Wright A., Lee A., Morsch M., Yang S., Williams K. L., Blair I. P., 
Splicing factor proline and glutamine rich intron retention, reduced expression and aggregate formation are pathological features of amyotrophic lateral sclerosis. Neuropathol. Appl. Neurobiol.
47, 
990–1003 (2021).
</cite> [<a href="https://doi.org/10.1111/nan.12749" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34288034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuropathol.%20Appl.%20Neurobiol.&amp;title=Splicing%20factor%20proline%20and%20glutamine%20rich%20intron%20retention,%20reduced%20expression%20and%20aggregate%20formation%20are%20pathological%20features%20of%20amyotrophic%20lateral%20sclerosis&amp;volume=47&amp;publication_year=2021&amp;pages=990-1003&amp;pmid=34288034&amp;doi=10.1111/nan.12749&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Tyzack G. E., Neeves J., Crerar H., Klein P., Ziff O., Taha D. M., Luisier R., Luscombe N. M., Patani R., 
Aberrant cytoplasmic intron retention is a blueprint for RNA binding protein mislocalization in VCP-related amyotrophic lateral sclerosis. Brain
144, 
1985–1993 (2021).
</cite> [<a href="https://doi.org/10.1093/brain/awab078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8370440/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33693641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Aberrant%20cytoplasmic%20intron%20retention%20is%20a%20blueprint%20for%20RNA%20binding%20protein%20mislocalization%20in%20VCP-related%20amyotrophic%20lateral%20sclerosis&amp;volume=144&amp;publication_year=2021&amp;pages=1985-1993&amp;pmid=33693641&amp;doi=10.1093/brain/awab078&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Patton J. G., Porro E. B., Galceran J., Tempst P., Nadal-Ginard B., 
Cloning and characterization of PSF, a novel pre-mRNA splicing factor. Genes Dev.
7, 
393–406 (1993).
</cite> [<a href="https://doi.org/10.1101/gad.7.3.393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8449401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Cloning%20and%20characterization%20of%20PSF,%20a%20novel%20pre-mRNA%20splicing%20factor&amp;volume=7&amp;publication_year=1993&amp;pages=393-406&amp;pmid=8449401&amp;doi=10.1101/gad.7.3.393&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Kim P., Yang M., Yiya K., Zhao W., Zhou X., 
ExonSkipDB: Functional annotation of exon skipping event in human. Nucleic Acids Res.
48, 
D896–D907 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkz917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31642488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=ExonSkipDB:%20Functional%20annotation%20of%20exon%20skipping%20event%20in%20human&amp;volume=48&amp;publication_year=2020&amp;pages=D896-D907&amp;pmid=31642488&amp;doi=10.1093/nar/gkz917&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Vaquero-Garcia J., Barrera A., Gazzara M. R., Gonzalez-Vallinas J., Lahens N. F., Hogenesch J. B., Lynch K. W., Barash Y., 
A new view of transcriptome complexity and regulation through the lens of local splicing variations. eLife
5, 
e11752 (2016).
</cite> [<a href="https://doi.org/10.7554/eLife.11752" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4801060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26829591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=A%20new%20view%20of%20transcriptome%20complexity%20and%20regulation%20through%20the%20lens%20of%20local%20splicing%20variations&amp;volume=5&amp;publication_year=2016&amp;pages=e11752&amp;pmid=26829591&amp;doi=10.7554/eLife.11752&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Vaquero-Garcia J., Aicher J. K., Jewell S., Gazzara M. R., Radens C. M., Jha A., Norton S. S., Lahens N. F., Grant G. R., Barash Y., 
RNA splicing analysis using heterogeneous and large RNA-seq datasets. Nat. Commun.
14, 
1230 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-36585-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9984406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36869033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=RNA%20splicing%20analysis%20using%20heterogeneous%20and%20large%20RNA-seq%20datasets&amp;volume=14&amp;publication_year=2023&amp;pages=1230&amp;pmid=36869033&amp;doi=10.1038/s41467-023-36585-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Nguyen V. T., Giannoni F., Dubois M. F., Seo S. J., Vigneron M., Kédinger C., Bensaude O., 
In vivo degradation of RNA polymerase II largest subunit triggered by alpha-amanitin. Nucleic Acids Res.
24, 
2924–2929 (1996).
</cite> [<a href="https://doi.org/10.1093/nar/24.15.2924" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC146057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8760875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=In%20vivo%20degradation%20of%20RNA%20polymerase%20II%20largest%20subunit%20triggered%20by%20alpha-amanitin&amp;volume=24&amp;publication_year=1996&amp;pages=2924-2929&amp;pmid=8760875&amp;doi=10.1093/nar/24.15.2924&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Bushnell D. A., Cramer P., Kornberg R. D., 
Structural basis of transcription: α-amanitin-RNA polymerase II cocrystal at 2.8 Å resolution. Proc. Natl. Acad. Sci. U.S.A.
99, 
1218–1222 (2002).
</cite> [<a href="https://doi.org/10.1073/pnas.251664698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC122170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11805306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Structural%20basis%20of%20transcription:%20%CE%B1-amanitin-RNA%20polymerase%20II%20cocrystal%20at%202.8%20%C3%85%20resolution&amp;volume=99&amp;publication_year=2002&amp;pages=1218-1222&amp;pmid=11805306&amp;doi=10.1073/pnas.251664698&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Nagy E., Maquat L. E., 
A rule for termination-codon position within intron-containing genes: When nonsense affects RNA abundance. Trends Biochem. Sci.
23, 
198–199 (1998).
</cite> [<a href="https://doi.org/10.1016/s0968-0004(98)01208-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9644970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem.%20Sci.&amp;title=A%20rule%20for%20termination-codon%20position%20within%20intron-containing%20genes:%20When%20nonsense%20affects%20RNA%20abundance&amp;volume=23&amp;publication_year=1998&amp;pages=198-199&amp;pmid=9644970&amp;doi=10.1016/s0968-0004(98)01208-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Lejeune F., Maquat L. E., 
Mechanistic links between nonsense-mediated mRNA decay and pre-mRNA splicing in mammalian cells. Curr. Opin. Cell Biol.
17, 
309–315 (2005).
</cite> [<a href="https://doi.org/10.1016/j.ceb.2005.03.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15901502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Cell%20Biol.&amp;title=Mechanistic%20links%20between%20nonsense-mediated%20mRNA%20decay%20and%20pre-mRNA%20splicing%20in%20mammalian%20cells&amp;volume=17&amp;publication_year=2005&amp;pages=309-315&amp;pmid=15901502&amp;doi=10.1016/j.ceb.2005.03.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Santos D. A., Shi L., Tu B. P., Weissman J. S., 
Cycloheximide can distort measurements of mRNA levels and translation efficiency. Nucleic Acids Res.
47, 
4974–4985 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gkz205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6547433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30916348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Cycloheximide%20can%20distort%20measurements%20of%20mRNA%20levels%20and%20translation%20efficiency&amp;volume=47&amp;publication_year=2019&amp;pages=4974-4985&amp;pmid=30916348&amp;doi=10.1093/nar/gkz205&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Langer L. M., Bonneau F., Gat Y., Conti E., 
Cryo-EM reconstructions of inhibitor-bound SMG1 kinase reveal an autoinhibitory state dependent on SMG8. Elife
10, 
e72353 (2021).
</cite> [<a href="https://doi.org/10.7554/eLife.72353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8592573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34698635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Cryo-EM%20reconstructions%20of%20inhibitor-bound%20SMG1%20kinase%20reveal%20an%20autoinhibitory%20state%20dependent%20on%20SMG8&amp;volume=10&amp;publication_year=2021&amp;pages=e72353&amp;pmid=34698635&amp;doi=10.7554/eLife.72353&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Lareau L. F., Brooks A. N., Soergel D. A. W., Meng Q., Brenner S. E., 
The coupling of alternative splicing and nonsense-mediated mRNA decay. Adv. Exp. Med. Biol.
623, 
190–211 (2007).
</cite> [<a href="https://doi.org/10.1007/978-0-387-77374-2_12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18380348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Exp.%20Med.%20Biol.&amp;title=The%20coupling%20of%20alternative%20splicing%20and%20nonsense-mediated%20mRNA%20decay&amp;volume=623&amp;publication_year=2007&amp;pages=190-211&amp;pmid=18380348&amp;doi=10.1007/978-0-387-77374-2_12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Ni J. Z., Grate L., Donohue J. P., Preston C., Nobida N., O’Brien G., Shiue L., Clark T. A., Blume J. E., Ares M. Jr., 
Ultraconserved elements are associated with homeostatic control of splicing regulators by alternative splicing and nonsense-mediated decay. Genes Dev.
21, 
708–718 (2007).
</cite> [<a href="https://doi.org/10.1101/gad.1525507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1820944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17369403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Ultraconserved%20elements%20are%20associated%20with%20homeostatic%20control%20of%20splicing%20regulators%20by%20alternative%20splicing%20and%20nonsense-mediated%20decay&amp;volume=21&amp;publication_year=2007&amp;pages=708-718&amp;pmid=17369403&amp;doi=10.1101/gad.1525507&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Campagne S., Jutzi D., Malard F., Matoga M., Romane K., Feldmuller M., Colombo M., Ruepp M.-D., Allain F. H.-T., 
Molecular basis of RNA-binding and autoregulation by the cancer-associated splicing factor RBM39. Nat. Commun.
14, 
5366 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-40254-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10477243/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37666821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Molecular%20basis%20of%20RNA-binding%20and%20autoregulation%20by%20the%20cancer-associated%20splicing%20factor%20RBM39&amp;volume=14&amp;publication_year=2023&amp;pages=5366&amp;pmid=37666821&amp;doi=10.1038/s41467-023-40254-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>J. P. B. Lloyd, C. E. French, S. E. Brenner, Polysome fractionation analysis reveals features important for human nonsense-mediated mRNA decay. bioRxiv 981811 [Preprint] (2020). 10.1101/2020.03.08.981811.</cite> [<a href="https://doi.org/10.1101/2020.03.08.981811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Maquat L. E., Tarn W.-Y., Isken O., 
The pioneer round of translation: Features and functions. Cell
142, 
368–374 (2010).
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.07.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2950652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20691898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The%20pioneer%20round%20of%20translation:%20Features%20and%20functions&amp;volume=142&amp;publication_year=2010&amp;pages=368-374&amp;pmid=20691898&amp;doi=10.1016/j.cell.2010.07.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Isken O., Kim Y. K., Hosoda N., Mayeur G. L., Hershey J. W., Maquat L. E., 
Upf1 phosphorylation triggers translational repression during nonsense-mediated mRNA decay. Cell
133, 
314–327 (2008).
</cite> [<a href="https://doi.org/10.1016/j.cell.2008.02.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4193665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18423202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Upf1%20phosphorylation%20triggers%20translational%20repression%20during%20nonsense-mediated%20mRNA%20decay&amp;volume=133&amp;publication_year=2008&amp;pages=314-327&amp;pmid=18423202&amp;doi=10.1016/j.cell.2008.02.030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Kim W. K., Yun S., Kwon Y., You K. T., Shin N., Kim J., Kim H., 
mRNAs containing NMD-competent premature termination codons are stabilized and translated under UPF1 depletion. Sci. Rep.
7, 
15833 (2017).
</cite> [<a href="https://doi.org/10.1038/s41598-017-16177-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5696521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29158530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=mRNAs%20containing%20NMD-competent%20premature%20termination%20codons%20are%20stabilized%20and%20translated%20under%20UPF1%20depletion&amp;volume=7&amp;publication_year=2017&amp;pages=15833&amp;pmid=29158530&amp;doi=10.1038/s41598-017-16177-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Dye B. T., Patton J. G., 
An RNA recognition motif (RRM) is required for the localization of PTB-associated splicing factor (PSF) to subnuclear speckles. Exp. Cell Res.
263, 
131–144 (2001).
</cite> [<a href="https://doi.org/10.1006/excr.2000.5097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11161712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp.%20Cell%20Res.&amp;title=An%20RNA%20recognition%20motif%20(RRM)%20is%20required%20for%20the%20localization%20of%20PTB-associated%20splicing%20factor%20(PSF)%20to%20subnuclear%20speckles&amp;volume=263&amp;publication_year=2001&amp;pages=131-144&amp;pmid=11161712&amp;doi=10.1006/excr.2000.5097&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Knott G. J., Chong Y. S., Passon D. M., Liang X.-H., Deplazes E., Conte M. R., Marshall A. C., Lee M., Fox A. H., Bond C. S., 
Structural basis of dimerization and nucleic acid binding of human DBHS proteins NONO and PSPC1. Nucleic Acids Res.
50, 
522–535 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8754649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34904671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Structural%20basis%20of%20dimerization%20and%20nucleic%20acid%20binding%20of%20human%20DBHS%20proteins%20NONO%20and%20PSPC1&amp;volume=50&amp;publication_year=2022&amp;pages=522-535&amp;pmid=34904671&amp;doi=10.1093/nar/gkab1216&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Huang J., Casas Garcia G. P., Perugini M. A., Fox A. H., Bond C. S., Lee M., 
Crystal structure of a SFPQ/PSPC1 heterodimer provides insights into preferential heterodimerization of human DBHS family proteins. J. Biol. Chem.
293, 
6593–6602 (2018).
</cite> [<a href="https://doi.org/10.1074/jbc.RA117.001451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5925804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29530979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Crystal%20structure%20of%20a%20SFPQ/PSPC1%20heterodimer%20provides%20insights%20into%20preferential%20heterodimerization%20of%20human%20DBHS%20family%20proteins&amp;volume=293&amp;publication_year=2018&amp;pages=6593-6602&amp;pmid=29530979&amp;doi=10.1074/jbc.RA117.001451&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Marshall A. C., Cummins J., Kobelke S., Zhu T., Widagdo J., Anggono V., Hyman A., Fox A. H., Bond C. S., Lee M., 
Different low-complexity regions of SFPQ play distinct roles in the formation of biomolecular condensates. J. Mol. Biol.
435, 
168364 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2023.168364" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37952770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Different%20low-complexity%20regions%20of%20SFPQ%20play%20distinct%20roles%20in%20the%20formation%20of%20biomolecular%20condensates&amp;volume=435&amp;publication_year=2023&amp;pages=168364&amp;pmid=37952770&amp;doi=10.1016/j.jmb.2023.168364&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Lee J., Cho H., Kwon I., 
Phase separation of low-complexity domains in cellular function and disease. Exp. Mol. Med.
54, 
1412–1422 (2022).
</cite> [<a href="https://doi.org/10.1038/s12276-022-00857-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9534829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36175485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp.%20Mol.%20Med.&amp;title=Phase%20separation%20of%20low-complexity%20domains%20in%20cellular%20function%20and%20disease&amp;volume=54&amp;publication_year=2022&amp;pages=1412-1422&amp;pmid=36175485&amp;doi=10.1038/s12276-022-00857-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Naganuma T., Nakagawa S., Tanigawa A., Sasaki Y. F., Goshima N., Hirose T., 
Alternative 3′-end processing of long noncoding RNA initiates construction of nuclear paraspeckles. EMBO J.
31, 
4020–4034 (2012).
</cite> [<a href="https://doi.org/10.1038/emboj.2012.251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3474925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22960638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&amp;title=Alternative%203%E2%80%B2-end%20processing%20of%20long%20noncoding%20RNA%20initiates%20construction%20of%20nuclear%20paraspeckles&amp;volume=31&amp;publication_year=2012&amp;pages=4020-4034&amp;pmid=22960638&amp;doi=10.1038/emboj.2012.251&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Harley J., Patani R., 
Stress-specific spatiotemporal responses of RNA-binding proteins in human stem-cell-derived motor neurons. Int. J. Mol. Sci.
21, 
8346 (2020).
</cite> [<a href="https://doi.org/10.3390/ijms21218346" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7664327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33172210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Stress-specific%20spatiotemporal%20responses%20of%20RNA-binding%20proteins%20in%20human%20stem-cell-derived%20motor%20neurons&amp;volume=21&amp;publication_year=2020&amp;pages=8346&amp;pmid=33172210&amp;doi=10.3390/ijms21218346&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Boutz P. L., Bhutkar A., Sharp P. A., 
Detained introns are a novel, widespread class of post-transcriptionally spliced introns. Genes Dev.
29, 
63–80 (2015).
</cite> [<a href="https://doi.org/10.1101/gad.247361.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4281565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25561496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Detained%20introns%20are%20a%20novel,%20widespread%20class%20of%20post-transcriptionally%20spliced%20introns&amp;volume=29&amp;publication_year=2015&amp;pages=63-80&amp;pmid=25561496&amp;doi=10.1101/gad.247361.114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Park S.-K., Zhou X., Pendleton K. E., Hunter O. V., Kohler J. J., O’Donnell K. A., Conrad N. K., 
A conserved splicing silencer dynamically regulates O-GlcNAc transferase intron retention and O-GlcNAc homeostasis. Cell Rep.
20, 
1088–1099 (2017).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.07.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5588854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28768194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=A%20conserved%20splicing%20silencer%20dynamically%20regulates%20O-GlcNAc%20transferase%20intron%20retention%20and%20O-GlcNAc%20homeostasis&amp;volume=20&amp;publication_year=2017&amp;pages=1088-1099&amp;pmid=28768194&amp;doi=10.1016/j.celrep.2017.07.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Tan Z.-W., Fei G., Paulo J. A., Bellaousov S., Martin S. E. S., Duveau D. Y., Thomas C. J., Gygi S. P., Boutz P. L., Walker S., 
O-GlcNAc regulates gene expression by controlling detained intron splicing. Nucleic Acids Res.
48, 
5656–5669 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7261177/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32329777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=O-GlcNAc%20regulates%20gene%20expression%20by%20controlling%20detained%20intron%20splicing&amp;volume=48&amp;publication_year=2020&amp;pages=5656-5669&amp;pmid=32329777&amp;doi=10.1093/nar/gkaa263&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Yamazaki T., Souquere S., Chujo T., Kobelke S., Chong Y. S., Fox A. H., Bond C. S., Nakagawa S., Pierron G., Hirose T., 
Functional domains of NEAT1 architectural lncRNA induce paraspeckle assembly through phase separation. Mol. Cell
70, 
1038–1053.e7 (2018).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2018.05.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29932899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Functional%20domains%20of%20NEAT1%20architectural%20lncRNA%20induce%20paraspeckle%20assembly%20through%20phase%20separation&amp;volume=70&amp;publication_year=2018&amp;pages=1038-1053.e7&amp;pmid=29932899&amp;doi=10.1016/j.molcel.2018.05.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Szklarczyk D., Morris J. H., Cook H., Kuhn M., Wyder S., Simonovic M., Santos A., Doncheva N. T., Roth A., Bork P., Jensen L. J., von Mering C., 
The STRING database in 2017: Quality-controlled protein-protein association networks, made broadly accessible. Nucleic Acids Res.
45, 
D362–D368 (2017).
</cite> [<a href="https://doi.org/10.1093/nar/gkw937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5210637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27924014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20STRING%20database%20in%202017:%20Quality-controlled%20protein-protein%20association%20networks,%20made%20broadly%20accessible&amp;volume=45&amp;publication_year=2017&amp;pages=D362-D368&amp;pmid=27924014&amp;doi=10.1093/nar/gkw937&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>Kapeli K., Pratt G. A., Vu A. Q., Hutt K. R., Martinez F. J., Sundararaman B., Batra R., Freese P., Lambert N. J., Huelga S. C., Chun S. J., Liang T. Y., Chang J., Donohue J. P., Shiue L., Zhang J., Zhu H., Cambi F., Kasarskis E., Hoon S., Ares M. Jr., Burge C. B., Ravits J., Rigo F., Yeo G. W., 
Distinct and shared functions of ALS-associated proteins TDP-43, FUS and TAF15 revealed by multisystem analyses. Nat. Commun.
7, 
12143 (2016).
</cite> [<a href="https://doi.org/10.1038/ncomms12143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4935974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27378374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Distinct%20and%20shared%20functions%20of%20ALS-associated%20proteins%20TDP-43,%20FUS%20and%20TAF15%20revealed%20by%20multisystem%20analyses&amp;volume=7&amp;publication_year=2016&amp;pages=12143&amp;pmid=27378374&amp;doi=10.1038/ncomms12143&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Kiskinis E., Sandoe J., Williams L. A., Boulting G. L., Moccia R., Wainger B. J., Han S., Peng T., Thams S., Mikkilineni S., Mellin C., Merkle F. T., Davis-Dusenbery B. N., Ziller M., Oakley D., Ichida J., Costanzo S. D., Atwater N., Maeder M. L., Goodwin M. J., Nemesh J., Handsaker R. E., Paull D., Noggle S., McCarroll S. A., Joung J. K., Woolf C. J., Brown R. H., Eggan K., 
Pathways disrupted in human ALS motor neurons identified through genetic correction of mutant SOD1. Cell Stem Cell
14, 
781–795 (2014).
</cite> [<a href="https://doi.org/10.1016/j.stem.2014.03.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4653065/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24704492/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&amp;title=Pathways%20disrupted%20in%20human%20ALS%20motor%20neurons%20identified%20through%20genetic%20correction%20of%20mutant%20SOD1&amp;volume=14&amp;publication_year=2014&amp;pages=781-795&amp;pmid=24704492&amp;doi=10.1016/j.stem.2014.03.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>NeuroLINCS Consortium, Li J., Lim R. G., Kaye J. A., Dardov V., Coyne A. N., Wu J., Milani P., Cheng A., Thompson T. G., Ornelas L., Frank A., Adam M., Banuelos M. G., Casale M., Cox V., Escalante-Chong R., Daigle J. G., Gomez E., Hayes L., Holewenski R., Lei S., Lenail A., Lima L., Mandefro B., Matlock A., Panther L., Patel-Murray N. L., Pham J., Ramamoorthy D., Sachs K., Shelley B., Stocksdale J., Trost H., Wilhelm M., Venkatraman V., Wassie B. T., Wyman S., Yang S., NYGC ALS Consortium, Van Eyk J. E., Lloyd T. E., Finkbeiner S., Fraenkel E., Rothstein J. D., Sareen D., Svendsen C. N., Thompson L. M., 
An integrated multi-omic analysis of iPSC-derived motor neurons from C9ORF72 ALS patients. iScience
24, 
103221 (2021).
</cite> [<a href="https://doi.org/10.1016/j.isci.2021.103221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8554488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34746695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=An%20integrated%20multi-omic%20analysis%20of%20iPSC-derived%20motor%20neurons%20from%20C9ORF72%20ALS%20patients&amp;volume=24&amp;publication_year=2021&amp;pages=103221&amp;pmid=34746695&amp;doi=10.1016/j.isci.2021.103221&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Baxi E. G., Thompson T., Li J., Kaye J. A., Lim R. G., Wu J., Ramamoorthy D., Lima L., Vaibhav V., Matlock A., Frank A., Coyne A. N., Landin B., Ornelas L., Mosmiller E., Thrower S., Farr S. M., Panther L., Gomez E., Galvez E., Perez D., Meepe I., Lei S., Mandefro B., Trost H., Pinedo L., Banuelos M. G., Liu C., Moran R., Garcia V., Workman M., Ho R., Wyman S., Roggenbuck J., Harms M. B., Stocksdale J., Miramontes R., Wang K., Venkatraman V., Holewenski R., Sundararaman N., Pandey R., Manalo D.-M., Donde A., Huynh N., Adam M., Wassie B. T., Vertudes E., Amirani N., Raja K., Thomas R., Hayes L., Lenail A., Cerezo A., Luppino S., Farrar A., Pothier L., Prina C., Morgan T., Jamil A., Heintzman S., Jockel-Balsarotti J., Karanja E., Markway J., Callum M. M., Joslin B., Alibazoglu D., Kolb S., Ajroud-Driss S., Baloh R., Heitzman D., Miller T., Glass J. D., Patel-Murray N. L., Yu H., Sinani E., Vigneswaran P., Sherman A. V., Ahmad O., Roy P., Beavers J. C., Zeiler S., Krakauer J. W., Agurto C., Cecchi G., Bellard M., Raghav Y., Sachs K., Ehrenberger T., Bruce E., Cudkowicz M. E., Maragakis N., Norel R., Van Eyk J. E., Finkbeiner S., Berry J., Sareen D., Thompson L. M., Fraenkel E., Svendsen C. N., Rothstein J. D., 
Answer ALS, a large-scale resource for sporadic and familial ALS combining clinical and multi-omics data from induced pluripotent cell lines. Nat. Neurosci.
25, 
226–237 (2022).
</cite> [<a href="https://doi.org/10.1038/s41593-021-01006-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8825283/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35115730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Neurosci.&amp;title=Answer%20ALS,%20a%20large-scale%20resource%20for%20sporadic%20and%20familial%20ALS%20combining%20clinical%20and%20multi-omics%20data%20from%20induced%20pluripotent%20cell%20lines&amp;volume=25&amp;publication_year=2022&amp;pages=226-237&amp;pmid=35115730&amp;doi=10.1038/s41593-021-01006-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>Dingwall C., Laskey R. A., 
Nuclear targeting sequences—A consensus?
Trends Biochem. Sci.
16, 
478–481 (1991).
</cite> [<a href="https://doi.org/10.1016/0968-0004(91)90184-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1664152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem.%20Sci.&amp;title=Nuclear%20targeting%20sequences%E2%80%94A%20consensus?&amp;volume=16&amp;publication_year=1991&amp;pages=478-481&amp;pmid=1664152&amp;doi=10.1016/0968-0004(91)90184-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Weskamp K., Tank E. M., Miguez R., McBride J. P., Gómez N. B., White M., Lin Z., Gonzalez C. M., Serio A., Sreedharan J., Barmada S. J., 
Shortened TDP43 isoforms upregulated by neuronal hyperactivity drive TDP43 pathology in ALS. J. Clin. Invest.
130, 
1139–1155 (2020).
</cite> [<a href="https://doi.org/10.1172/JCI130988" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7269575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31714900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest.&amp;title=Shortened%20TDP43%20isoforms%20upregulated%20by%20neuronal%20hyperactivity%20drive%20TDP43%20pathology%20in%20ALS&amp;volume=130&amp;publication_year=2020&amp;pages=1139-1155&amp;pmid=31714900&amp;doi=10.1172/JCI130988&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Shenouda M., Xiao S., MacNair L., Lau A., Robertson J., 
A C-terminally truncated TDP-43 splice isoform exhibits neuronal specific cytoplasmic aggregation and contributes to TDP-43 pathology in ALS. Front. Neurosci.
16, 
868556 (2022).
</cite> [<a href="https://doi.org/10.3389/fnins.2022.868556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9253772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35801182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Neurosci.&amp;title=A%20C-terminally%20truncated%20TDP-43%20splice%20isoform%20exhibits%20neuronal%20specific%20cytoplasmic%20aggregation%20and%20contributes%20to%20TDP-43%20pathology%20in%20ALS&amp;volume=16&amp;publication_year=2022&amp;pages=868556&amp;pmid=35801182&amp;doi=10.3389/fnins.2022.868556&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Y. Zeng, A. Lovchykova, T. Akiyama, C. Liu, C. Guo, V. M. Jawahar, O. Sianto, A. Calliari, M. Prudencio, D. W. Dickson, L. Petrucelli, A. D. Gitler, TDP-43 nuclear loss in FTD/ALS causes widespread alternative polyadenylation changes. bioRxiv 576625 [Preprint] (2024). 10.1101/2024.01.22.575730.</cite> [<a href="https://doi.org/10.1101/2024.01.22.575730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R63">
<span class="label">63.</span><cite>Patton J. G., Mayer S. A., Tempst P., Nadal-Ginard B., 
Characterization and molecular cloning of polypyrimidine tract-binding protein: A component of a complex necessary for pre-mRNA splicing. Genes Dev.
5, 
1237–1251 (1991).
</cite> [<a href="https://doi.org/10.1101/gad.5.7.1237" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1906036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Characterization%20and%20molecular%20cloning%20of%20polypyrimidine%20tract-binding%20protein:%20A%20component%20of%20a%20complex%20necessary%20for%20pre-mRNA%20splicing&amp;volume=5&amp;publication_year=1991&amp;pages=1237-1251&amp;pmid=1906036&amp;doi=10.1101/gad.5.7.1237&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R64">
<span class="label">64.</span><cite>Flather D., Nguyen J. H. C., Semler B. L., Gershon P. D., 
Exploitation of nuclear functions by human rhinovirus, a cytoplasmic RNA virus. PLOS Pathog.
14, 
e1007277 (2018).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1007277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6126879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30142213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Pathog.&amp;title=Exploitation%20of%20nuclear%20functions%20by%20human%20rhinovirus,%20a%20cytoplasmic%20RNA%20virus&amp;volume=14&amp;publication_year=2018&amp;pages=e1007277&amp;pmid=30142213&amp;doi=10.1371/journal.ppat.1007277&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R65">
<span class="label">65.</span><cite>Knott G. J., Bond C. S., Fox A. H., 
The DBHS proteins SFPQ, NONO and PSPC1: A multipurpose molecular scaffold. Nucleic Acids Res.
44, 
3989–4004 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkw271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4872119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27084935/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20DBHS%20proteins%20SFPQ,%20NONO%20and%20PSPC1:%20A%20multipurpose%20molecular%20scaffold&amp;volume=44&amp;publication_year=2016&amp;pages=3989-4004&amp;pmid=27084935&amp;doi=10.1093/nar/gkw271&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R66">
<span class="label">66.</span><cite>Fukuda Y., Pazyra-Murphy M. F., Silagi E. S., Tasdemir-Yilmaz O. E., Li Y., Rose L., Yeoh Z. C., Vangos N. E., Geffken E. A., Seo H.-S., Adelmant G., Bird G. H., Walensky L. D., Marto J. A., Dhe-Paganon S., Segal R. A., 
Binding and transport of SFPQ-RNA granules by KIF5A/KLC1 motors promotes axon survival. J. Cell Biol.
220, 
e202005051 (2021).
</cite> [<a href="https://doi.org/10.1083/jcb.202005051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7721913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33284322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&amp;title=Binding%20and%20transport%20of%20SFPQ-RNA%20granules%20by%20KIF5A/KLC1%20motors%20promotes%20axon%20survival&amp;volume=220&amp;publication_year=2021&amp;pages=e202005051&amp;pmid=33284322&amp;doi=10.1083/jcb.202005051&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R67">
<span class="label">67.</span><cite>Lee M., Sadowska A., Bekere I., Ho D., Gully B. S., Lu Y., Iyer K. S., Trewhella J., Fox A. H., Bond C. S., 
The structure of human SFPQ reveals a coiled-coil mediated polymer essential for functional aggregation in gene regulation. Nucleic Acids Res.
43, 
3826–3840 (2015).
</cite> [<a href="https://doi.org/10.1093/nar/gkv156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4402515/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25765647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20structure%20of%20human%20SFPQ%20reveals%20a%20coiled-coil%20mediated%20polymer%20essential%20for%20functional%20aggregation%20in%20gene%20regulation&amp;volume=43&amp;publication_year=2015&amp;pages=3826-3840&amp;pmid=25765647&amp;doi=10.1093/nar/gkv156&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R68">
<span class="label">68.</span><cite>Taliaferro J. M., Vidaki M., Oliveira R., Olson S., Zhan L., Saxena T., Wang E. T., Graveley B. R., Gertler F. B., Swanson M. S., Burge C. B., 
Distal alternative last exons localize mRNAs to neural projections. Mol. Cell
61, 
821–833 (2016).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2016.01.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4798900/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26907613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Distal%20alternative%20last%20exons%20localize%20mRNAs%20to%20neural%20projections&amp;volume=61&amp;publication_year=2016&amp;pages=821-833&amp;pmid=26907613&amp;doi=10.1016/j.molcel.2016.01.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R69">
<span class="label">69.</span><cite>Shigeoka T., Jung H., Jung J., Turner-Bridger B., Ohk J., Lin J. Q., Amieux P. S., Holt C. E., 
Dynamic axonal translation in developing and mature visual circuits. Cell
166, 
181–192 (2016).
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.05.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4930487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27321671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Dynamic%20axonal%20translation%20in%20developing%20and%20mature%20visual%20circuits&amp;volume=166&amp;publication_year=2016&amp;pages=181-192&amp;pmid=27321671&amp;doi=10.1016/j.cell.2016.05.029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R70">
<span class="label">70.</span><cite>Lysikova E. A., Funikov S., Rezvykh A. P., Chaprov K. D., Kukharsky M. S., Ustyugov A., Deykin A. V., Flyamer I. M., Boyle S., Bachurin S. O., Ninkina N., Buchman V. L., 
Low level of expression of C-terminally truncated human FUS causes extensive changes in the spinal cord transcriptome of asymptomatic transgenic mice. Neurochem. Res.
45, 
1168–1179 (2020).
</cite> [<a href="https://doi.org/10.1007/s11064-020-02999-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32157564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurochem.%20Res.&amp;title=Low%20level%20of%20expression%20of%20C-terminally%20truncated%20human%20FUS%20causes%20extensive%20changes%20in%20the%20spinal%20cord%20transcriptome%20of%20asymptomatic%20transgenic%20mice&amp;volume=45&amp;publication_year=2020&amp;pages=1168-1179&amp;pmid=32157564&amp;doi=10.1007/s11064-020-02999-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R71">
<span class="label">71.</span><cite>Reber S., Mechtersheimer J., Nasif S., Benitez J. A., Colombo M., Domanski M., Jutzi D., Hedlund E., Ruepp M. D., 
CRISPR-Trap: A clean approach for the generation of gene knockouts and gene replacements in human cells. Mol. Biol. Cell
29, 
75–83 (2018).
</cite> [<a href="https://doi.org/10.1091/mbc.E17-05-0288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5909934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29167381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Cell&amp;title=CRISPR-Trap:%20A%20clean%20approach%20for%20the%20generation%20of%20gene%20knockouts%20and%20gene%20replacements%20in%20human%20cells&amp;volume=29&amp;publication_year=2018&amp;pages=75-83&amp;pmid=29167381&amp;doi=10.1091/mbc.E17-05-0288&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R72">
<span class="label">72.</span><cite>Batlle C., Yang P., Coughlin M., Messing J., Pesarrodona M., Szulc E., Salvatella X., Kim H. J., Taylor J. P., Ventura S., 
hnRNPDL phase separation is regulated by alternative splicing and disease-causing mutations accelerate its aggregation. Cell Rep.
30, 
1117–1128.e5 (2020).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.12.080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6996132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31995753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=hnRNPDL%20phase%20separation%20is%20regulated%20by%20alternative%20splicing%20and%20disease-causing%20mutations%20accelerate%20its%20aggregation&amp;volume=30&amp;publication_year=2020&amp;pages=1117-1128.e5&amp;pmid=31995753&amp;doi=10.1016/j.celrep.2019.12.080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R73">
<span class="label">73.</span><cite>Patel A., Lee H. O., Jawerth L., Maharana S., Jahnel M., Hein M. Y., Stoynov S., Mahamid J., Saha S., Franzmann T. M., Pozniakovski A., Poser I., Maghelli N., Royer L. A., Weigert M., Myers E. W., Grill S., Drechsel D., Hyman A. A., Alberti S., 
A liquid-to-solid phase transition of the ALS protein FUS accelerated by disease mutation. Cell
162, 
1066–1077 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.07.047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26317470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20liquid-to-solid%20phase%20transition%20of%20the%20ALS%20protein%20FUS%20accelerated%20by%20disease%20mutation&amp;volume=162&amp;publication_year=2015&amp;pages=1066-1077&amp;pmid=26317470&amp;doi=10.1016/j.cell.2015.07.047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R74">
<span class="label">74.</span><cite>Reber S., Jutzi D., Lindsay H., Devoy A., Mechtersheimer J., Levone B. R., Domanski M., Bentmann E., Dormann D., Muhlemann O., Barabino S. M. L., Ruepp M.-D., 
The phase separation-dependent FUS interactome reveals nuclear and cytoplasmic function of liquid-liquid phase separation. Nucleic Acids Res.
49, 
7713–7731 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8287939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34233002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20phase%20separation-dependent%20FUS%20interactome%20reveals%20nuclear%20and%20cytoplasmic%20function%20of%20liquid-liquid%20phase%20separation&amp;volume=49&amp;publication_year=2021&amp;pages=7713-7731&amp;pmid=34233002&amp;doi=10.1093/nar/gkab582&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R75">
<span class="label">75.</span><cite>Alberti S., Gladfelter A., Mittag T., 
Considerations and challenges in studying liquid-liquid phase separation and biomolecular condensates. Cell
176, 
419–434 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.12.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6445271/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30682370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Considerations%20and%20challenges%20in%20studying%20liquid-liquid%20phase%20separation%20and%20biomolecular%20condensates&amp;volume=176&amp;publication_year=2019&amp;pages=419-434&amp;pmid=30682370&amp;doi=10.1016/j.cell.2018.12.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R76">
<span class="label">76.</span><cite>Molliex A., Temirov J., Lee J., Coughlin M., Kanagaraj A. P., Kim H. J., Mittag T., Taylor J. P., 
Phase separation by low complexity domains promotes stress granule assembly and drives pathological fibrillization. Cell
163, 
123–133 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.09.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5149108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26406374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Phase%20separation%20by%20low%20complexity%20domains%20promotes%20stress%20granule%20assembly%20and%20drives%20pathological%20fibrillization&amp;volume=163&amp;publication_year=2015&amp;pages=123-133&amp;pmid=26406374&amp;doi=10.1016/j.cell.2015.09.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R77">
<span class="label">77.</span><cite>Simsek D., Tiu G. C., Flynn R. A., Byeon G. W., Leppek K., Xu A. F., Chang H. Y., Barna M., 
The mammalian ribo-interactome reveals ribosome functional diversity and heterogeneity. Cell
169, 
1051–1065.e18 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.05.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5548193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28575669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The%20mammalian%20ribo-interactome%20reveals%20ribosome%20functional%20diversity%20and%20heterogeneity&amp;volume=169&amp;publication_year=2017&amp;pages=1051-1065.e18&amp;pmid=28575669&amp;doi=10.1016/j.cell.2017.05.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R78">
<span class="label">78.</span><cite>Sun S., Sun Y., Ling S. C., Ferraiuolo L., McAlonis-Downes M., Zou Y., Drenner K., Wang Y., Ditsworth D., Tokunaga S., Kopelevich A., Kaspar B. K., Lagier-Tourenne C., Cleveland D. W., 
Translational profiling identifies a cascade of damage initiated in motor neurons and spreading to glia in mutant SOD1-mediated ALS. Proc. Natl. Acad. Sci. U.S.A.
112, 
E6993–E7002 (2015).
</cite> [<a href="https://doi.org/10.1073/pnas.1520639112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4687558/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26621731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Translational%20profiling%20identifies%20a%20cascade%20of%20damage%20initiated%20in%20motor%20neurons%20and%20spreading%20to%20glia%20in%20mutant%20SOD1-mediated%20ALS&amp;volume=112&amp;publication_year=2015&amp;pages=E6993-E7002&amp;pmid=26621731&amp;doi=10.1073/pnas.1520639112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R79">
<span class="label">79.</span><cite>Kiesler E., Miralles F., Ostlund Farrants A.-K., Visa N., 
The Hrp65 self-interaction is mediated by an evolutionarily conserved domain and is required for nuclear import of Hrp65 isoforms that lack a nuclear localization signal. J. Cell Sci.
116, 
3949–3956 (2003).
</cite> [<a href="https://doi.org/10.1242/jcs.00690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12928329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Sci.&amp;title=The%20Hrp65%20self-interaction%20is%20mediated%20by%20an%20evolutionarily%20conserved%20domain%20and%20is%20required%20for%20nuclear%20import%20of%20Hrp65%20isoforms%20that%20lack%20a%20nuclear%20localization%20signal&amp;volume=116&amp;publication_year=2003&amp;pages=3949-3956&amp;pmid=12928329&amp;doi=10.1242/jcs.00690&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R80">
<span class="label">80.</span><cite>Zhang S., Cooper J. A., Chong Y. S., Naveed A., Mayoh C., Jayatilleke N., Liu T., Amos S., Kobelke S., Marshall A. C., Meers O., Choi Y. S., Bond C. S., Fox A. H., 
NONO enhances mRNA processing of super-enhancer-associated GATA2 and HAND2 genes in neuroblastoma. EMBO Rep.
24, 
e54977 (2023).
</cite> [<a href="https://doi.org/10.15252/embr.202254977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9900351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36416237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Rep.&amp;title=NONO%20enhances%20mRNA%20processing%20of%20super-enhancer-associated%20GATA2%20and%20HAND2%20genes%20in%20neuroblastoma&amp;volume=24&amp;publication_year=2023&amp;pages=e54977&amp;pmid=36416237&amp;doi=10.15252/embr.202254977&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R81">
<span class="label">81.</span><cite>Shao W., Bi X., Pan Y., Gao B., Wu J., Yin Y., Liu Z., Peng M., Zhang W., Jiang X., Ren W., Xu Y., Wu Z., Wang K., Zhan G., Lu J. Y., Han X., Li T., Wang J., Li G., Deng H., Li B., Shen X., 
Phase separation of RNA-binding protein promotes polymerase binding and transcription. Nat. Chem. Biol.
18, 
70–80 (2022).
</cite> [<a href="https://doi.org/10.1038/s41589-021-00904-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34916619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&amp;title=Phase%20separation%20of%20RNA-binding%20protein%20promotes%20polymerase%20binding%20and%20transcription&amp;volume=18&amp;publication_year=2022&amp;pages=70-80&amp;pmid=34916619&amp;doi=10.1038/s41589-021-00904-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R82">
<span class="label">82.</span><cite>Maharana S., Wang J., Papadopoulos D. K., Richter D., Pozniakovsky A., Poser I., Bickle M., Rizk S., Guillén-Boixet J., Franzmann T. M., Jahnel M., Marrone L., Chang Y.-T., Sterneckert J., Tomancak P., Hyman A. A., Alberti S., 
RNA buffers the phase separation behavior of prion-like RNA binding proteins. Science
360, 
918–921 (2018).
</cite> [<a href="https://doi.org/10.1126/science.aar7366" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6091854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29650702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=RNA%20buffers%20the%20phase%20separation%20behavior%20of%20prion-like%20RNA%20binding%20proteins&amp;volume=360&amp;publication_year=2018&amp;pages=918-921&amp;pmid=29650702&amp;doi=10.1126/science.aar7366&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R83">
<span class="label">83.</span><cite>Ninomiya K., Kataoka N., Hagiwara M., 
Stress-responsive maturation of Clk1/4 pre-mRNAs promotes phosphorylation of SR splicing factor. J. Cell Biol.
195, 
27–40 (2011).
</cite> [<a href="https://doi.org/10.1083/jcb.201107093" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3187705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21949414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&amp;title=Stress-responsive%20maturation%20of%20Clk1/4%20pre-mRNAs%20promotes%20phosphorylation%20of%20SR%20splicing%20factor&amp;volume=195&amp;publication_year=2011&amp;pages=27-40&amp;pmid=21949414&amp;doi=10.1083/jcb.201107093&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R84">
<span class="label">84.</span><cite>Naro C., Jolly A., Persio S. D., Bielli P., Setterblad N., Alberdi A. J., Vicini E., Geremia R., De la Grange P., Sette C., 
An orchestrated intron retention program in meiosis controls timely usage of transcripts during germ cell differentiation. Dev. Cell
41, 
82–93.e4 (2017).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2017.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5392497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28366282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Cell&amp;title=An%20orchestrated%20intron%20retention%20program%20in%20meiosis%20controls%20timely%20usage%20of%20transcripts%20during%20germ%20cell%20differentiation&amp;volume=41&amp;publication_year=2017&amp;pages=82-93.e4&amp;pmid=28366282&amp;doi=10.1016/j.devcel.2017.03.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R85">
<span class="label">85.</span><cite>Vargas D. Y., Shah K., Batish M., Levandoski M., Sinha S., Marras S. A., Schedl P., Tyagi S., 
Single-molecule imaging of transcriptionally coupled and uncoupled splicing. Cell
147, 
1054–1065 (2011).
</cite> [<a href="https://doi.org/10.1016/j.cell.2011.10.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3245879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22118462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Single-molecule%20imaging%20of%20transcriptionally%20coupled%20and%20uncoupled%20splicing&amp;volume=147&amp;publication_year=2011&amp;pages=1054-1065&amp;pmid=22118462&amp;doi=10.1016/j.cell.2011.10.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R86">
<span class="label">86.</span><cite>Pai A. A., Henriques T., McCue K., Burkholder A., Adelman K., Burge C. B., 
The kinetics of pre-mRNA splicing in the Drosophila genome and the influence of gene architecture. Elife
6, 
e32537 (2017).
</cite> [<a href="https://doi.org/10.7554/eLife.32537" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5762160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29280736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=The%20kinetics%20of%20pre-mRNA%20splicing%20in%20the%20Drosophila%20genome%20and%20the%20influence%20of%20gene%20architecture&amp;volume=6&amp;publication_year=2017&amp;pages=e32537&amp;pmid=29280736&amp;doi=10.7554/eLife.32537&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R87">
<span class="label">87.</span><cite>Martin R. M., Rino J., Carvalho C., Kirchhausen T., Carmo-Fonseca M., 
Live-cell visualization of pre-mRNA splicing with single-molecule sensitivity. Cell Rep.
4, 
1144–1155 (2013).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2013.08.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3805459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24035393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=Live-cell%20visualization%20of%20pre-mRNA%20splicing%20with%20single-molecule%20sensitivity&amp;volume=4&amp;publication_year=2013&amp;pages=1144-1155&amp;pmid=24035393&amp;doi=10.1016/j.celrep.2013.08.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R88">
<span class="label">88.</span><cite>Hall C. E., Yao Z., Choi M., Tyzack G. E., Serio A., Luisier R., Harley J., Preza E., Arber C., Crisp S. J., Watson P. M. D., Kullmann D. M., Abramov A. Y., Wray S., Burley R., Loh S. H. Y., Martins L. M., Stevens M. M., Luscombe N. M., Sibley C. R., Lakatos A., Ule J., Gandhi S., Patani R., 
Progressive motor neuron pathology and the role of astrocytes in a human stem cell model of VCP-related ALS. Cell Rep.
19, 
1739–1749 (2017).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.05.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5464993/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28564594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=Progressive%20motor%20neuron%20pathology%20and%20the%20role%20of%20astrocytes%20in%20a%20human%20stem%20cell%20model%20of%20VCP-related%20ALS&amp;volume=19&amp;publication_year=2017&amp;pages=1739-1749&amp;pmid=28564594&amp;doi=10.1016/j.celrep.2017.05.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R89">
<span class="label">89.</span><cite>Giblin A., Cammack A. J., Blomberg N., Anoar S., Mikheenko A., Carcolé M., Atilano M. L., Hull A., Shen D., Wei X., Coneys R., Zhou L., Mohammed Y., Olivier-Jimenez D., Wang L. Y., Kinghorn K. J., Niccoli T., Coyne A. N., van der Kant R., Lashley T., Giera M., Partridge L., Isaacs A. M., 
Neuronal polyunsaturated fatty acids are protective in ALS/FTD. Nat. Neurosci.
28, 
737–747 (2025).
</cite> [<a href="https://doi.org/10.1038/s41593-025-01889-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11976277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40000803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Neurosci.&amp;title=Neuronal%20polyunsaturated%20fatty%20acids%20are%20protective%20in%20ALS/FTD&amp;volume=28&amp;publication_year=2025&amp;pages=737-747&amp;pmid=40000803&amp;doi=10.1038/s41593-025-01889-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R90">
<span class="label">90.</span><cite>Hofweber M., Hutten S., Bourgeois B., Spreitzer E., Niedner-Boblenz A., Schifferer M., Ruepp M.-D., Simons M., Niessing D., Madl T., Dormann D., 
Phase separation of FUS is suppressed by its nuclear import receptor and arginine methylation. Cell
173, 
706–719.e13 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.03.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29677514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Phase%20separation%20of%20FUS%20is%20suppressed%20by%20its%20nuclear%20import%20receptor%20and%20arginine%20methylation&amp;volume=173&amp;publication_year=2018&amp;pages=706-719.e13&amp;pmid=29677514&amp;doi=10.1016/j.cell.2018.03.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R91">
<span class="label">91.</span><cite>Gruijs da Silva L. A., Dormann D., 
Sedimentation assays to assess the impact of posttranslational modifications on phase separation of RNA-binding proteins in vitro and in cells. Methods Mol. Biol.
2563, 
325–339 (2023).
</cite> [<a href="https://doi.org/10.1007/978-1-0716-2663-4_16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36227481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&amp;title=Sedimentation%20assays%20to%20assess%20the%20impact%20of%20posttranslational%20modifications%20on%20phase%20separation%20of%20RNA-binding%20proteins%20in%20vitro%20and%20in%20cells&amp;volume=2563&amp;publication_year=2023&amp;pages=325-339&amp;pmid=36227481&amp;doi=10.1007/978-1-0716-2663-4_16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R92">
<span class="label">92.</span><cite>Ewels P. A., Peltzer A., Fillinger S., Patel H., Alneberg J., Wilm A., Garcia M. U., Di Tommaso P., Nahnsen S., 
The nf-core framework for community-curated bioinformatics pipelines. Nat. Biotechnol.
38, 
276–278 (2020).
</cite> [<a href="https://doi.org/10.1038/s41587-020-0439-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32055031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=The%20nf-core%20framework%20for%20community-curated%20bioinformatics%20pipelines&amp;volume=38&amp;publication_year=2020&amp;pages=276-278&amp;pmid=32055031&amp;doi=10.1038/s41587-020-0439-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R93">
<span class="label">93.</span><cite>Perez-Riverol Y., Bai J., Bandla C., García-Seisdedos D., Hewapathirana S., Kamatchinathan S., Kundu D. J., Prakash A., Frericks-Zipper A., Eisenacher M., Walzer M., Wang S., Brazma A., Vizcaíno J. A., 
The PRIDE database resources in 2022: A hub for mass spectrometry-based proteomics evidences. Nucleic Acids Res.
50, 
D543–D552 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728295/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34723319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=The%20PRIDE%20database%20resources%20in%202022:%20A%20hub%20for%20mass%20spectrometry-based%20proteomics%20evidences&amp;volume=50&amp;publication_year=2022&amp;pages=D543-D552&amp;pmid=34723319&amp;doi=10.1093/nar/gkab1038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R94">
<span class="label">94.</span><cite>Tyanova S., Temu T., Sinitcyn P., Carlson A., Hein M. Y., Geiger T., Mann M., Cox J., 
The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat. Methods
13, 
731–740 (2016).
</cite> [<a href="https://doi.org/10.1038/nmeth.3901" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27348712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=The%20Perseus%20computational%20platform%20for%20comprehensive%20analysis%20of%20(prote)omics%20data&amp;volume=13&amp;publication_year=2016&amp;pages=731-740&amp;pmid=27348712&amp;doi=10.1038/nmeth.3901&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R95">
<span class="label">95.</span><cite>Xiao Y., Hsiao T. H., Suresh U., Chen H. I., Wu X., Wolf S. E., Chen Y., 
A novel significance score for gene selection and ranking. Bioinformatics
30, 
801–807 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr671" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3957066/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22321699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20novel%20significance%20score%20for%20gene%20selection%20and%20ranking&amp;volume=30&amp;publication_year=2014&amp;pages=801-807&amp;pmid=22321699&amp;doi=10.1093/bioinformatics/btr671&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>Figs. S1 to S8</p>
<p>Legends for tables S1 and S2</p>
<p>Tables S3 to S5</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372870/bin/sciadv.adt4814_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adt4814_sm.pdf</a><sup> (6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Tables S1 and S2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372870/bin/sciadv.adt4814_tables_s1_and_s2.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adt4814_tables_s1_and_s2.zip</a><sup> (7.1MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>HiPSC-derived motor neuron raw sequencing data used in this study are available in public repositories under the following accession numbers: Tyzack <em>et al.</em> [(<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>); <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE152983" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE152983</a>], Kapeli <em>et al.</em> [(<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>); <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE77702" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE77702</a>], Kiskinis <em>et al.</em> [(<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>); <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54409" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE54409</a>], NeuroLINCS Consortium <em>et al.</em> [(<a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>); phs001231.v2.p1], and Baxi <em>et al.</em> [(<a href="#R58" class="usa-link" aria-describedby="R58"><em>58</em></a>); Answer ALS data portal]. Some raw data have restricted access (NeuroLINCS dbGaP accession number: phs0001231.v2.p1; Answer ALS database). Granting access to these is beyond the control of the authors. Access can be obtained by applying to the relevant Data Access Committees. Answer ALS requires a signed data use agreement to have full access.</p>
<p>The MS proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE (<a href="#R93" class="usa-link" aria-describedby="R93"><em>93</em></a>) partner repository with the dataset identifier PXD051918.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Science Advances are provided here courtesy of <strong>American Association for the Advancement of Science</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1126/sciadv.adt4814"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/sciadv.adt4814.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12372870/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12372870/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372870%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372870/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12372870/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12372870/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40845103/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12372870/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40845103/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12372870/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12372870/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="rPmkdqhEeolJBAw5jivRObaIBwdd85RUcm6pkUwXRSwmSlbYCjq3hvnMhZVvaVRB">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
