
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Coordinated changes in midkine expression and midkine-associated multiomic profile in glioma microenvironment - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AE928AF30C9305AE920022EC1CD0.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368106/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Coordinated changes in midkine expression and midkine-associated multiomic profile in glioma microenvironment">
<meta name="citation_author" content="Mieszko Lachota">
<meta name="citation_author_institution" content="Department of Ophthalmology, Children’s Memorial Health Institute, Warsaw, Poland">
<meta name="citation_author_institution" content="Laboratory of Cellular and Genetic Therapies, Medical University of Warsaw, Warsaw, Poland">
<meta name="citation_author" content="Katarzyna Zielniok">
<meta name="citation_author_institution" content="Laboratory of Cellular and Genetic Therapies, Medical University of Warsaw, Warsaw, Poland">
<meta name="citation_author" content="Agata Góźdź">
<meta name="citation_author_institution" content="Department of Histology and Embryology, Centre for Biostructure Research, Medical University of Warsaw, Warsaw, Poland">
<meta name="citation_author" content="Patrycja Szpak">
<meta name="citation_author_institution" content="Department of Histology and Embryology, Centre for Biostructure Research, Medical University of Warsaw, Warsaw, Poland">
<meta name="citation_author" content="Ilona Kalaszczyńska">
<meta name="citation_author_institution" content="Department of Histology and Embryology, Centre for Biostructure Research, Medical University of Warsaw, Warsaw, Poland">
<meta name="citation_author" content="Radosław Zagożdżon">
<meta name="citation_author_institution" content="Laboratory of Cellular and Genetic Therapies, Medical University of Warsaw, Warsaw, Poland">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30587">
<meta name="citation_doi" content="10.1038/s41598-025-16253-5">
<meta name="citation_pmid" content="40835683">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368106/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368106/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368106/pdf/41598_2025_Article_16253.pdf">
<meta name="description" content="Midkine (MDK), a multifunctional growth factor, has been implicated in promoting tumor progression, yet its role in glioblastoma (GBM) remains insufficiently characterized. To investigate MDK’s function in glioma, we integrated four RNA-Seq datasets ...">
<meta name="og:title" content="Coordinated changes in midkine expression and midkine-associated multiomic profile in glioma microenvironment">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Midkine (MDK), a multifunctional growth factor, has been implicated in promoting tumor progression, yet its role in glioblastoma (GBM) remains insufficiently characterized. To investigate MDK’s function in glioma, we integrated four RNA-Seq datasets ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368106/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368106">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-16253-5"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_16253.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368106%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368106/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368106/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368106/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 20;15:30587. doi: <a href="https://doi.org/10.1038/s41598-025-16253-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-16253-5</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Coordinated changes in midkine expression and midkine-associated multiomic profile in glioma microenvironment</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lachota%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mieszko Lachota</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mieszko Lachota</span></h3>
<div class="p">
<sup>1</sup>Department of Ophthalmology, Children’s Memorial Health Institute, Warsaw, Poland </div>
<div class="p">
<sup>2</sup>Laboratory of Cellular and Genetic Therapies, Medical University of Warsaw, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lachota%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mieszko Lachota</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zielniok%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Katarzyna Zielniok</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Katarzyna Zielniok</span></h3>
<div class="p">
<sup>2</sup>Laboratory of Cellular and Genetic Therapies, Medical University of Warsaw, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zielniok%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Katarzyna Zielniok</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%B3%C5%BAd%C5%BA%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Agata Góźdź</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Agata Góźdź</span></h3>
<div class="p">
<sup>3</sup>Department of Histology and Embryology, Centre for Biostructure Research, Medical University of Warsaw, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%B3%C5%BAd%C5%BA%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Agata Góźdź</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Szpak%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Patrycja Szpak</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Patrycja Szpak</span></h3>
<div class="p">
<sup>3</sup>Department of Histology and Embryology, Centre for Biostructure Research, Medical University of Warsaw, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Szpak%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Patrycja Szpak</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kalaszczy%C5%84ska%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ilona Kalaszczyńska</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ilona Kalaszczyńska</span></h3>
<div class="p">
<sup>3</sup>Department of Histology and Embryology, Centre for Biostructure Research, Medical University of Warsaw, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kalaszczy%C5%84ska%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ilona Kalaszczyńska</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zago%C5%BCd%C5%BCon%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Radosław Zagożdżon</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Radosław Zagożdżon</span></h3>
<div class="p">
<sup>2</sup>Laboratory of Cellular and Genetic Therapies, Medical University of Warsaw, Warsaw, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zago%C5%BCd%C5%BCon%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Radosław Zagożdżon</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Ophthalmology, Children’s Memorial Health Institute, Warsaw, Poland </div>
<div id="Aff2">
<sup>2</sup>Laboratory of Cellular and Genetic Therapies, Medical University of Warsaw, Warsaw, Poland </div>
<div id="Aff3">
<sup>3</sup>Department of Histology and Embryology, Centre for Biostructure Research, Medical University of Warsaw, Warsaw, Poland </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 9; Accepted 2025 Aug 13; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368106  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835683/" class="usa-link">40835683</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par7">Midkine (MDK), a multifunctional growth factor, has been implicated in promoting tumor progression, yet its role in glioblastoma (GBM) remains insufficiently characterized. To investigate MDK’s function in glioma, we integrated four RNA-Seq datasets into a harmonized cohort of 1,017 adult gliomas, including 256 GBM samples. We complemented this with freshly collected human GBM tissues and matched primary cell cultures to evaluate MDK expression and secretion patterns, further contextualized using single-cell RNA-Seq. Finally, we tested the impact of GBM-derived MDK on macrophage secretome composition to validate our in silico observations. We found that <em>MDK</em> expression increases with tumor grade in <em>IDH</em><sup>wildtype</sup> gliomas, accompanied by a shift in isoform proportions favoring the canonical <em>MDK</em> transcript. High <em>MDK</em> expression was associated with poor prognosis specifically in GBM, where the <em>MDK</em><sup>high</sup> subgroup comprised 75% of cases. <em>MDK</em><sup>high</sup> GBMs exhibited a distinctive multiomic signature, including elevated chemokine and cytokine expression. Functionally, GBM-derived MDK induced macrophages to secrete multiple cytokines and chemokines, suggesting its role in reshaping the tumor microenvironment. Our findings reveal MDK’s previously underappreciated role in GBM aggressiveness and immune modulation, underscoring its potential as a biomarker and actionable therapeutic target for most GBM patients.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-16253-5.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Midkine, MDK, Glioblastoma, GBM, Glioma, Tumor microenvironment, TME, TCGA</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> CNS cancer, Cytokines</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par8">Glioblastoma (GBM) is the most common malignant glial tumor in adults and the leading cause of cancer deaths among patients aged 15 to 34<sup>1,2</sup>. According to the 2021 World Health Organization (WHO) Classification of Gliomas, adult-type diffuse astrocytic tumors are grouped based on isocitrate dehydrogenase (<em>IDH</em>) mutation status, with only <em>IDH</em><sup>wildtype</sup> tumors with either characteristic morphology or molecular features of glioblastoma being classified as “glioblastoma, <em>IDH</em><sup>wildtype</sup>"<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. First-line treatment, consisting of radical and safe resection supported by chemoradiotherapy, yields a median overall survival (OS) of approximately 16 months, which has not improved since 1990s<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. There is no standard second-line therapy to prolong OS, with virtually inevitable recurrence less than a year after surgery<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. While for many systemic cancers, targeted treatments became part of the standard of care, in grade IV <em>IDH</em><sup>wildtype</sup> gliomas (GBM), the clinical benefit is only seen in 1–2% of patients with v-Raf murine sarcoma viral oncogene homolog B1 (BRAF) mutations treated with <em>BRAF</em> p.V600E inhibitors<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. There is a pressing need for new targeted therapies and biomarkers that could allow for early detection and identification of patients benefiting from additional targeted treatment. Despite the advances in understanding glioblastoma biology and promising results of several preclinical studies, new therapies for glioblastoma have failed to improve patient survival<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>.</p>
<p id="Par9">There are several reasons why treating glioblastoma is so difficult. Its infiltrative nature and fragility of surrounding brain tissues make complete resection of tumor cells nearly impossible, leading to tumor recurrence at the resection margin<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Another is that the presence of mutations in genes such as epidermal growth factor receptor (EGFR), isocitrate dehydrogenase (IDH), protein kinase C (PKC), tumor protein p53 (p53), phosphatase and tensin homolog (PTEN), retinoblastoma protein (pRB), and vascular endothelial growth factor (VEGF) leads to the deregulation of major cellular pathways, ultimately resulting in high proliferative activity of GBM cells and the promotion of continuous neovascularization in GBM tumors<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Moreover, the blood-brain barrier limits the penetration of chemotherapeutics to the tumor area<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Treatment is further impaired by substantial heterogeneity of the tumor and the presence of a population of chemo- and radiotherapy-resistant cancer-stem-like cells (CSCs)<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Finally, GBM cells secrete numerous soluble factors that reshape the tumor microenvironment, enhancing cancer cell proliferation and impairing antitumor immune response<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Some studies have proposed midkine (MDK), a soluble heparin-binding growth factor, as one of them<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>.</p>
<p id="Par10">In a healthy central nervous system (CNS), MDK is produced by fetal astrocytes during neurogenesis<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Then, its expression is progressively decreasing to low levels in adults<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. However, MDK can be conditionally expressed in adults upon tissue damage due to its involvement in regenerative processes inducing cell proliferation, monocyte chemotaxis, and chemokine expression<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. MDK signals through multiple receptors, including proteoglycans such as syndecans and receptor protein tyrosine phosphatase-β (RPTP-β), as well as non-proteoglycan receptors such as anaplastic lymphoma kinase (ALK), low-density lipoprotein receptor-related protein-1 (Lrp1) and integrins (α4β1 and α6β1)<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Its multifaceted properties, normally utilized during embryogenesis or tissue regeneration, can be hijacked in cancer<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. MDK is overexpressed and has diagnostic and prognostic potential in several cancers, including lung, bladder, esophageal, and breast cancer<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. MDK pleiotropic downstream signaling events transduced through its vast array of receptors are involved in multiple hallmarks of cancer, including proliferative signaling via direct activation of its receptors, inducing neovascularization, avoiding immune destruction via a chemotactic effect on macrophages, which are then being M2 polarized, and direct effect on promoting invasion and metastasis, as reviewed in detail by Filippou et al.<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p>
<p id="Par11">In gliomas, studies of midkine (MDK) started in 1997 when Mishima et al. showed that MDK mRNA and protein expression levels were higher in high-grade astrocytomas, e.g., anaplastic astrocytomas and glioblastomas, compared to low-grade astrocytomas and healthy brain<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Further studies indicated that high MDK expression is associated with a worse prognosis in GBM patients<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. MDK was also shown to increase GBM cell proliferation, migration, and invasion and induce temozolomide resistance by improving the stem-like properties of GBM through ALK and Notch homolog 1 (Notch1)/phosphorylated c-Jun N-terminal kinase (p-JNK) signaling pathways<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Additionally, a recent study revealed that MDK rewires tumor-infiltrating T cells to secrete chemokines contributing to protumorigenic microenvironment<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. The mechanism of its overexpression in glioma has not been elucidated. However, it’s been suggested that retinoic acid, Wnt/β-catenin, and hypoxia signaling play a role<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Despite evidence of its importance in GBM, large-scale studies confirming MDK’s expression patterns and cellular source are lacking. Moreover, many of its biological functions in glioblastoma remain poorly understood.</p>
<p id="Par12">In view of its multifaceted role and insights from recent preclinical and Phase I clinical studies, we rationalized performing a comprehensive characterization of <em>MDK</em> expression patterns together with MDK-associated multiomic footprint in glioblastoma to lay the ground for future investigations and allow for informed design of clinical trials based on identifying patients that will likely benefit from the treatment. We collected primary tumor samples, established matched primary GBM cell cultures, and performed the transcriptomics and proteomics using RNA sequencing, enzyme-linked immunosorbent assay (ELISA), and Luminex. We then harnessed the power of multiple glioma datasets: The Cancer Genome Atlas (TCGA)-GBM, TCGA-LGG, Chinese Glioma Genome Atlas (CGGA), ERP010930, and SRP044668 to create a large, harmonized RNA-seq dataset with 1017 gliomas and healthy brain tissue samples. Single-cell RNA-seq, methylation, and mutational analysis of primary glioblastoma tumors from the TCGA-GBM cohort further complemented the study. For additional validation, we stimulated macrophages with either MDK or GBM cell culture supernatants to confirm MDKs’ impact on macrophage secretome.</p>
<p id="Par13">Overall, our study confirmed earlier findings of increased MDK expression in gliomas, proportional to the tumor grade, as well as its association with poor survival in GBM with a diverse 1000 + patient cohort. Additionally, we have discovered a dominant MDK isoform with evidence for selection pressure towards its expression along with an increase in tumor grade. Finally, our multiomic analyses revealed an unfavorable MDK-associated footprint and its role in reshaping the TME by inducing an immunosuppressive secretome of macrophages. Altogether, this underscores the importance of MDK research in developing targeted therapies for GBM.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Human samples</h3>
<p id="Par14">The study was conducted according to the principles of the Declaration of Helsinki and local regulations. Biological material from patients with glioblastoma was obtained and processed as described in Gozdz et al.<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Briefly, glioblastoma (GBM) tissue and plasma samples were collected from patients undergoing surgery at the Medical University of Warsaw Neurosurgery Clinic and the Regional Hospital of the Medical University of Silesia in Sosnowiec. The tissue samples were used to establish primary GBM cell cultures and for bulk RNA-seq, while the plasma was used for proteomics, as described below.</p>
<p id="Par15">The study was approved by the Bioethics Committee of the Medical University of Silesia (permit number KNW/0022/KB1/2/I/17). All patients participating in the study provided informed consent before the surgery. Shortly before surgical resection, venous blood samples were collected on K2 EDTA from 9 patients (7 males, 2 females; median age 59 years, min. 41, max. 71). Plasma was then obtained by centrifugation (2000 × <em>g</em> for 15 min), after which it was partitioned and frozen at −80 °C until further analysis. All tumors were classified histopathologically as glioblastoma. For RNA-Seq analysis, we used tumor samples from 12 donors (8 males, 4 females; median age 61 years, min. 41, max. 71), and 12 ex-vivo cultures established from these tumors.</p>
<p id="Par16">Tumor tissue was processed one to six hours after resection. A portion of the tissue was quick frozen for later nucleic acids isolation, and the rest was washed twice with PBS containing 5% BSA and antibiotics, then cut into small fragments with a scalpel with blade no.10 until it was well minced (adopted from Leelatian et al.<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The tissue was then transferred to 15 mL conical tubes by flushing the dish with the 5 mL activated dissociation medium (containing collagenase IV and DNAase I) and incubated at 37 °C for 60 min. After this time, the tube was centrifuged for 5 min at 300 × <em>g</em> and resuspended in ACK lysis buffer for 10 min. After another centrifugation and passing through a 70 μm cell strainer and then a 40 μm cell strainer, the cells were resuspended in a tumor sphere culture medium. The medium was composed of DMEM-F12 with Glutamax (#31331028, Gibco), 2% B27 supplement (#0080085SA, Gibco), 20 ng/mL EGF (#PHG0311, Gibco), 20 ng/mL bFGF (#PHG0026, Gibco), 0,5% Penicilin-Streptomycin solution (#15140122, Gibco), and 1 IU/mL heparin (#7980, STEMCELL Technologies). The culture was started by seeding 2 × 10<sup>6</sup> cells in 10 mL of tumor sphere culture medium in a T25cm<sup>2</sup> culture flask. Immediately upon reaching an exponential growth rate, cells were propagated for cryopreservation, and the day of the first cryopreservation was acknowledged as the date of cell line establishment.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Maintenance of spherical ex vivo cultures derived from glioblastoma and culture of commercially available GBM cell lines</h3>
<p id="Par17">The primary ex vivo glioblastoma spherical cell cultures were maintained in tumor sphere culture medium composed of DMEM/F12 with Glutamax, 2% B27 supplement, 20 ng/ml EGF, 20 ng/ml bFGF (all Gibco, Thermo Fischer Scientific), 0.5% Penicillin-Streptomycin solution (Gibco) and 1 IU/ml heparin (STEMCELL Technologies) in an incubator under standard culture conditions (37 °C, 5% CO<sup>2</sup>). Cells were grown as non-adherent, three-dimensional spheroids on 25 cm<sup>2</sup> cell culture flasks (Sarstaedt), or (2 populations which from the beginning showed a tendency to adhere to surfaces) in an adherent form on laminin-coated 25 cm<sup>2</sup> culture flasks (Sigma Aldrich). The culture medium was replaced twice a week by centrifugation (5 min at 350 × g). Spheroids were passaged by accutase digestion (ACCUMAX, Sigma Aldrich) every 14 days. Cells between 7 and 14 passages were used for analyses.</p>
<p id="Par18">To analyze the effect of conditioned medium from GBM culture on the secretion profile of macrophages, cells from the 9 GBM populations were seeded at a density of 3 million cells per well in a 6-well plate in standard GBM medium in three replicates and cultured for 2 days until spheroids were formed. The medium was then replaced, and the cells were cultured in 2 ml of medium per well for 72 h, after which the medium was collected, pooled from 3 replicates into a single sample, aliquoted, and frozen at −80 °C for further use. At the time of medium collection, there were 5–8 million GBM cells in each well, depending on the population.</p>
<p id="Par19">The U-251 cell line was purchased from Sigma-Aldrich (Merck), the DKMG line from DSMZ (GmbH) and the U87 line from ATCC. The lines s were cultured in MEM Complete Medium, with non-essential amino acids, 1 mM sodium pyruvate, 2 mM L-Glutamine and 10% FBS and 1% penicillin-streptomycin (all Gibco, Thermo Fischer Scientific). Cells were maintained at 37 °C in a humidified atmosphere and 5% CO2 until 80% confluency was achieved and then detached with TrypLE™ Express (Thermo Fischer Scientific).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Analysis of the secretory profile of macrophages</h3>
<p id="Par20">The effect of midkine (MDK) and conditioned media from GBM spherical cultures on the secretion profile of macrophages was investigated using cells obtained from buffy coats from three healthy donors purchased from the Regional Blood Donation and Blood Treatment Centre in Warsaw, in accordance with the consent of the Bioethics Committee at the Medical University of Warsaw No. AKBE/20/2022. PBMC isolation was performed using Histopaque<sup>®</sup>−1077 (Merck) with SepMate™ PBMC Isolation Tubes (STEMCELL Technologies) according to the manufacturer’s protocol. The PBMC were then seeded into 48-well plates for non-adherent culture (Sarsteadt) at 550,000 cells per well (two complete plates from each donor) in RPMI (Gibco) with 10% human serum (Merck). After two hours in the incubator, the plates were removed, the medium was aspirated, and all wells were thoroughly rinsed twice with DPBS (Gibco). The cells adhering to the bottom were cultured for another 4 days in RPMI with 10% human serum. After four days, the cells reached the macrophage phenotype, which was confirmed by flow cytometry, and then treatment with midkine and conditioned medium from GBM culture was initiated. After removing the existing medium, 200 µl of RPMI medium with 10% human serum enriched with 4 concentrations of MDK (#450-16-20UG, Peprotech) were added to the wells: 100 ng, 500 ng, 1 µg, and 2.5 µg. The control cells for these treatments were those cultured in 200 µl RPMI with 10% human serum. To the cells treated with conditioned medium from the culture of 9 primary GBM populations, 200 µl of medium consisting of a 1:1 mixture of conditioned GBM medium (DMEM/F12 + 2% B27 Supplement + 20ng/ml bFGF + 20ng/ml EGF + 1IU/ml heparin, collected after 72 h of spherical culture) and RPMI with 10% human serum were added. To the cells treated with midkine neutralizing antibody (#MA5-32538, Invitrogen, Thermo Scientific) before adding 200 µl of the mixture of conditioned GBM medium with RPMI to the wells with macrophages, neutralizing antibodies were added to the conditioned medium and incubated for 15 min, then mixed with the appropriate RPMI medium and applied to the wells. The final concentration of neutralizing antibody in the GBM and RPMI media mixture was 15 µg/ml. The control cells for this treatment were those treated with 200 µl of a mixture of pure GBM medium and RPMI 10% human serum in a 1:1 ratio. Each treatment was performed in 3 technical replicates (3 wells per treatment) for each of the 3 donors. The treatments were conducted for another 72 h, after which the medium was collected, pooled from 3 technical replicates into one sample, centrifuged at 800 x <em>g</em> for 4 min, and frozen in Eppendorf tubes at −80 °C for Luminex analysis.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Western blotting</h3>
<p id="Par21">To confirm intracellular midkine expression at the protein level by Western blot analysis, spherical cultures of 10 GBM populations were maintained under standard GBM culture conditions (T25cm<sup>2</sup> bottles, DMEM/F12 medium with 2% B27 Supplement, 20ng/ml EGF, 20ng/ml bFGF and 1IU/ml heparin, at 5% CO2 and 37 °C) for at least three consecutive weeks until approximately 6 million cells were obtained from each bottle. The cultures were then centrifuged (350 x <em>g</em>, 5 min), the media were discarded, and the spheroids were digested with StemPro Accutase Cell Dissociation Reagent (#A1110501, Gibco, Thermo Scientific). The cells were then washed twice in DPBS (Gibco), and after decanting, the cell pellet was frozen at −80°C. Protein isolation was performed by suspending the cell pellets in 100 µl of RIPA Lysis and Extraction Buffer (Thermo Scientific) with the addition of protease and phosphatase inhibitors (Cell Signalling, #5871 and #5870, respectively) for 30 min on ice. The cell lysates were then centrifuged (14,000 rpm for 20 min at 4 °C), transferred to new Eppendorf tubes, and subjected to protein quantification using the Pierce BCA Protein Assay Kit (#23225, Thermo Scientific). After isolation and determination of protein levels, the lysates were frozen at −80°C. For WB analysis, 4xLaemmli Sample buffer (#1610747, BioRad) with 355 mM β-mercaptoethanol was added to samples containing 15 µg of total protein and denatured at 100 °C for 10 min before loading onto the gel. Electrophoretic separation of proteins was performed on a 12% polyacrylamide gel (29:1) in TRIS-HCL, Glycine, SDS buffer at a constant voltage of 180 V using PageRuler™ Plus Prestained Protein Ladder (Thermo Scientific) protein marker, applying 15 µg of protein lysate per well. Next, the proteins were electrotransferred to a methanol-activated Immobilon-FL PVDF Membrane (Millipore, Merck) in TRIS-HCL, Glycine, 20% methanol buffer for 110 min at a constant voltage of 100V. The membranes were then washed, and the protein bands were detected by staining with Ponceau S red and imaged with the ChemiDoc MP System (Bio-Rad) using the colorimetric detection method. The membranes were then rinsed in water until the red dye was thoroughly washed out and blocked for 1 hour in a 5% solution of non-fat milk powder in TBST. The blocked membranes were then incubated overnight at 4 °C in a solution of primary antibodies. The next day, the membranes were washed in TBST (4 × 10 min) and incubated in a solution of secondary antibodies conjugated with HRP for 1 hour at room temperature. After another cycle of washing in TBST, the membranes were developed using Clarity Max ECL (BioRad) for 5 min and imaged on a ChemiDoc MP Imaging System (Bio-Rad) with the ‘Blots’ and ‘Chemiluminescence’ application. Densitometric data were acquired using Image Lab software (Bio-Rad, v.6.1.0 build 7 Standard Edition for Windows, Bio-Rad Laboratories Inc., Hercules, California, USA, www.bio-rad.com/en-pl/product/image-lab-software? ID=KRE6P5E8Z) with data analysis, graphing and statistics performed using GraphPad Prism software (v.10.5.0 for Windows GraphPad Software, Boston, Massachusetts USA, <a href="http://www.graphpad.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.graphpad.com</a>). The list of utilized antibodies can be found in Supplementary Table 1.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Luminex and ELISA</h3>
<p id="Par22">Quantitative assessment of MDK secretion in patient plasma and media from primary ex-vivo cultures and GBM cell lines culture was performed on undiluted samples by Luminex’s technology on a LUMINEX 200 platform (Luminex Corporation) using the Human Luminex Discovery Assay kit (#LXSAHM, R&amp;D Systems) according to the manufacturer’s protocols. For this purpose, cells from 12 GBM populations (approximately 3 million per population) were cultured for 48 h in 5 mL of tumor sphere culture medium, under standard conditions (37 °C, 5% CO<sup>2</sup>), on 25 cm<sup>2</sup> flasks. The culture medium was then collected and centrifuged (350 × <em>g</em> for 5 min), divided into aliquots, and frozen at −80 °C for further analysis. The culture medium of the U251, DKMG, and U87 cell lines for secretion assays was collected after 48 h of cell culture in 2 ml of complete MEM medium in 6-well plates after centrifugation of 450 × <em>g</em>. On the day of analysis, the collected medium was thawed on ice. The assay procedure was performed according to the manufacturer’s protocol. The results were normalized by the values obtained in the basal growth medium and the number of cells collected from the culture flask. As the results of midkine concentrations in GBM patients’ plasma and primary cultures were very high, we decided to verify the analysis with the Human Midkine ELISA kit (Abcam, #<a href="https://www.ncbi.nlm.nih.gov/nuccore/Ab193761" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Ab193761</a>) according to the manufacturer’s protocol on 5-fold diluted samples.</p>
<p id="Par23">A Luminex analysis evaluating the effect of midkine and conditioned media from primary GBM cultures on the secretory profile of macrophages was performed using the Human Luminex Discovery Assay kit (#LXSAHM, BioTechne) with the following analytes: CCL2, MIP-1α (CCL3), CCL8, CCL13, CCL14, CCL17, CCL18, CCL26, CXCL5, CXCL6, CXCL10, CXCL11, interleukin 1 alpha (IL-1α), interleukin 10 (IL-10), interleukin 15 (IL-15), interleukin 33 (IL-33), interferon γ (IFN-γ), granulocyte-macrophage colony-stimulating factor (GM-CSF), granzyme B, programmed death-ligand 1 (PD-L1), and platelet-derived growth factor AA/BB (PDGF-AA/BB), selected based on transcriptomic data. On the day of analysis, frozen portions of macrophage culture medium and control media were thawed on ice and applied to the plate without diluting. The analysis was performed according to the manufacturer’s protocol. The results of the analysis were processed using Belysa Immunoassay Curve Fitting Software (v.1.2 for Windows, Cat. No. 40–122, Merck KGaA, Darmstadt, Germany <a href="https://www.sigmaaldrich.com/PL/pl/product/mm/40122?srsltid=AfmBOorlbA0a8uh9IhRwaGOPL0JvCiPBc5kNGdiHuWGCs-53IAgFYnFQ" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sigmaaldrich.com/PL/pl/product/mm/40122?srsltid=AfmBOorlbA0a8uh9IhRwaGOPL0JvCiPBc5kNGdiHuWGCs-53IAgFYnFQ</a>).</p></section><section id="Sec8"><h3 class="pmc_sec_title">Isolation of RNA from tissues and cell cultures and RNA sequencing</h3>
<p id="Par24">Cryopreserved tumor sections and cells cultured at 20% and 5% oxygen were harvested at the same or adjacent passage, mechanically disrupted in TRIzol Reagent (Thermo Fisher Scientific), and processed along the manufacturer’s instructions to obtain material for RNA sequencing. The integrity of the isolated RNA was assessed using a Bioanalyzer (Agilent Technologies). RNA sequencing was performed on high-quality RNA samples isolated from 12 GBM tumors and 12 tumor-derived ex-vivo cultures. Sequencing was performed on a HiSeq 1500 (Illumina) in a rapid run flow cell with paired-end settings (2 × 76 bp). RNA sequencing reads were processed using nf-core/rnaseq; they were aligned to the human genome (GRCh38) using the HISAT2 (v2.2.0) algorithm, then transcripts were assembled and quantified using StringTie (v2.1.7)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Data mining and preprocessing</h3>
<p id="Par25">All analyses were performed in R (<a href="https://www.r-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.r-project.org/</a>) using Bioconductor (<a href="https://www.bioconductor.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioconductor.org/</a>) packages. Tidyverse R package was used for data preparation. ggplot2, Nebulosa, and ComplexHeatmap R packages were used for data visualization<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p>
<p id="Par26">The bulk gene expression profiles (raw counts from RNA-seq) of four glioma datasets as well as healthy brain tissues were downloaded from recount-brain, a curated repository for human brain RNA-Seq datasets, with cohort names: TCGA-GBM, TCGA-LGG, Chinese Glioma Genome Atlas (CGGA), ERP010930, and SRP044668<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>–<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. All available metadata, including age, grade, race, sec, <em>IDH</em> mutation status, and overall survival data, were downloaded from recount-brain, the Cancer Genome Atlas (TCGA) and CGGA repositories. Isocitrate dehydrogenase (<em>IDH</em>) mutation status in SRP044668 was kindly provided by Dr. Peter A. Sims. Data from <a href="http://www.brainrnaseq.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.brainrnaseq.org</a> was kindly provided by Dr. Steven Sloan<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Retrieving data from the recount ensured that consistent bioinformatic pipelines were used for these four datasets<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. To further ensure no batch effects between the datasets, we performed batch effect removal by the ComBat_seq method from the R package “sva” (v3.48) to remove unwanted variation in our combined dataset. Tumor grade was included as a covariate<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>.</p>
<p id="Par27">The transcript isoform percentage of glioblastoma, low-grade glioma, and healthy brain tissues were downloaded from UCSC Xena (<a href="https://xenabrowser.net/datapages/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://xenabrowser.net/datapages/</a>) - cohort TCGA TARGET GTEx filtered for gliomas (grade II-IV), and healthy brain tissues, accession date 25th July 2022.</p>
<p id="Par28">The single-cell gene expression profiles of glioblastoma from 28 patients (24131 cells) were downloaded from Single Cell Portal, accession date 26th October 2021 (SPC393, <a href="https://singlecell.broadinstitute.org/single_cell" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://singlecell.broadinstitute.org/single_cell</a>)<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Group assignment and survival analysis</h3>
<p id="Par29">Patients in the combined RNA-seq dataset were dichotomized for survival analysis into two groups based on MDK expression. The optimal cutoff point was determined using maximally selected rank statistics, defined as the expression threshold that yields the most significant split based on the standardized log-rank test, implemented via the maxstat and survminer (v0.4.9) R packages. Based on this cutoff, patients were assigned to either the <em>MDK</em><sup>high</sup> or <em>MDK</em><sup>low</sup> group. The dataset was then stratified by tumor grade or by both grade and <em>IDH</em> mutation status. Kaplan–Meier survival analysis with log-rank testing was performed to assess differences in overall survival between the two groups.</p>
<p id="Par30">A global cutoff was used to dichotomize samples for survival analysis across all glioma grades and <em>IDH</em> statuses, ensuring consistent stratification and enabling direct comparison of MDK’s prognostic value across glioma subtypes. For downstream multiomic analyses focused on grade IV <em>IDH</em><sup>wildtype</sup> gliomas—where MDK showed significant prognostic relevance—a subgroup-specific cutoff calculated in an identical manner was used to better capture biologically meaningful differences within this cohort.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Differential gene expression analysis</h3>
<p id="Par31">For differential gene expression analysis, the RNA-seq expression data was either raw count type (combined bulk RNA-seq dataset) or imported from StringTie (our dataset)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Patients were divided into two groups based on either MDK expression as described above (recount v2 datasets) or divided based on sample type (primary tumor vs. primary cell culture). After assigning the samples to two groups based on risk score, we performed differential gene expression analysis using <em>DESeq2</em> R package<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. To optimize power, we used Independent Hypothesis Weighting (IHW) with an adjusted p-value threshold &lt; 0.05 to report differentially expressed genes (DEGs) and further gene-set enrichment analysis<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Gene set enrichment analysis was performed using <em>fgsea</em> R package (v1.26)<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Estimation of immune cell type abundance in tumor tissue</h3>
<p id="Par32">Patients in the combined RNA-seq dataset were divided into two groups based on either MDK expression as described above. We used <em>CIBERSORTx</em> (<a href="https://cibersortx.stanford.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cibersortx.stanford.edu/</a>) to evaluate the relative abundance of predefined cell types in mixed solid tissues<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Normalized and harmonized TPM gene expression data was used for this analysis. We employed the LM22 leukocyte gene signature matrix downloaded from the <em>CIBERSORTx</em> website. LM22 contains 547 genes distinguishing 22 types of immune-related cells. We set the number of permutations to 1000 for robust analysis. The B-mode of batch correction supplied with LM22 GEP was used. <em>CIBERSORTx</em> enumerated the abundance scores of the 22 infiltrating immune cells, including B cells, dendritic cells, T cells, natural killer cells, macrophages, and others. To simplify the text, we referred to the cell type abundance estimates as “abundance” or “infiltration” alone. The results were filtered with a <em>p-value</em> &lt; 0.05 threshold.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Methylation and mutation analysis</h3>
<p id="Par33">Patients in the combined RNA-seq dataset were divided into two groups based on either MDK expression as described above. The methylation data for the TCGA-GBM cohort was obtained using <em>ELMER</em> (v2.24.0, accession date 8th October 2024) and processed as described in Silva et al.<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. The mutation data for the TCGA-GBM cohort was obtained using <em>maftools</em> (v2.16.0, accession date 16th June 2022) and processed as described in Silva et al.<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Single-cell RNA sequencing analysis</h3>
<p id="Par34">Data was accessed as described above and imported into Seurat (v4.1.0) along with meta- and clustering data<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Malignant cells were split based on MDK expression (positive or negative). Differential gene expression analysis was performed using FindMarkers (Seurat). Gene set enrichment analysis was performed using fgsea (v1.26)<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. We utilized cell cycle markers from Kowalczyk et al. 2015<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>, to assign cell cycle state to each malignant cell. Cell cycle analysis was performed using CellCycleScoring implemented in Seurat.</p></section></section><section id="Sec15"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec16"><h3 class="pmc_sec_title">Midkine expression in healthy brain tissues and gliomas</h3>
<p id="Par35">To characterize and better understand the role of midkine (MDK) in gliomas, we downloaded uniformly processed RNA-Seq abundance values from recount-brain, a curated repository for human brain RNA-Seq datasets, for four different uniformly processed datasets of adult gliomas and healthy brain tissue controls: 643 glioma samples from the cancer genome atlas (TCGA), 268 glioma samples from Chinese glioma genome atlas (CGGA), 93 glioma samples from SRP044668 and 48 glioma samples from ERP010930. For expression analysis, 35 samples from the resection margin were excluded. For the survival analyses, we included only patients with complete survival data. Retrieving data from the recount ensured that consistent bioinformatic pipelines were used for these four datasets. To further ensure no batch effects between the datasets, we performed batch effect removal using negative binomial regression. Then we retrieved all available metadata and used isocitrate dehydrogenase (<em>IDH</em>) mutation status and tumor grade to align our cohort to the 2021 World Health Organization (WHO) Classification of Gliomas, which groups adult-type diffuse astrocytic tumors based on <em>IDH</em> mutation status, with only <em>IDH</em><sup>wildtype</sup> tumors with either characteristic morphology or molecular features of glioblastoma being classified as "glioblastoma, <em>IDH</em><sup>wildtype</sup>"<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>.</p>
<p id="Par36">Hijacking cytokine-signaling is a hallmark of cancer, and understanding the role of these hijacked cytokines in tumor samples can help researchers design novel treatments. Therefore, we evaluated <em>MDK</em> expression in healthy brain tissues and gliomas in our combined dataset. At first, our analysis revealed a consistent picture of high <em>MDK</em> expression in grade II, III, and IV gliomas compared to healthy brain tissues, with its expression increasing proportionally to the tumor grade (Fig. <a href="#Fig1" class="usa-link">1</a>a). Stratifying the dataset based on <em>IDH</em> mutation status revealed that MDK expression is low and relatively stable across different tumor grades in <em>IDH</em><sup>mutant</sup> tumors (Fig. <a href="#Fig1" class="usa-link">1</a>a). Conversely, it is higher and increases proportionally to the tumor grade in <em>IDH</em><sup>wildtype</sup> tumors (Fig. <a href="#Fig1" class="usa-link">1</a>a, Supplementary Fig. <a href="#Fig1" class="usa-link">1</a>). Overall, MDK expression was higher in <em>IDH</em><sup>wildtype</sup> grade III and IV tumors than <em>IDH</em><sup>mutant</sup> tumors of corresponding grade (Supplementary Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368106_41598_2025_16253_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4b3/12368106/21d996342e20/41598_2025_16253_Fig1_HTML.jpg" loading="lazy" id="d33e854" height="748" width="774" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Midkine expression in healthy brain tissues and glial tumors. (<strong>a</strong>) Midkine expression (log2(DESeq2 normalized counts + 0.001) in the combined dataset in all gliomas and healthy brain tissues stratified by grade (<em>N</em> = 1017) (left), in <em>IDH</em><sup>mutant</sup> gliomas (<em>N</em> = 542) (center), and <em>IDH</em><sup>wildtype</sup> gliomas and healthy brain tissues stratified by grade (<em>N</em> = 409). (<strong>b</strong>) <em>MDK</em> isoform proportions in grade II, III, and IV gliomas and healthy brain tissues from TCGA (672 samples) and GTEX (1146 samples) datasets. Shannon diversity index was calculated for each group. (<strong>c</strong>) Midkine expression (log2(DESeq2 normalized counts + 0.001) in the tumor tissues (<em>N</em> = 12) and the primary GBM cell cultures (<em>N</em> = 12) in the WUM cohort. (<strong>d</strong>) <em>MDK</em> isoform proportions in the tumor tissues (<em>N</em> = 12) and the primary GBM cell cultures (<em>N</em> = 12) in the WUM cohort. Wilcoxon rank-sum test was applied to evaluate the differences between sample types in (<strong>a</strong>) and (<strong>c</strong>). Significance thresholds: NS p &gt; = 0.05; ** <em>p</em> &lt; 0.01; *** <em>p</em> &lt; 0.001; **** <em>p</em> &lt; 0.0001.</p></figcaption></figure><p id="Par38">Healthy brain tissues were characterized by scarce <em>MDK</em> expression, with the fetal astrocytes being the only central nervous system (CNS) cell type expressing <em>MDK</em> under physiologic conditions (Supplementary Fig. <a href="#Fig2" class="usa-link">2</a>). mRNA isoform analysis revealed that the most prevalent <em>MDK</em> isoform remained the same (ENST00000395566) in all sample types. However, along with the tumor grade, the prevalence of ENST00000395566 increased, while the diversity of <em>MDK</em> isoforms gradually decreased, as assessed by the Shannon diversity index (Fig. <a href="#Fig1" class="usa-link">1</a>b).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368106_41598_2025_16253_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4b3/12368106/72a9fccfde1a/41598_2025_16253_Fig2_HTML.jpg" loading="lazy" id="d33e983" height="765" width="733" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cross-section of MDK in different samples. (<strong>a</strong>) Midkine expression on a single-cell level from primary glioblastoma tumors (<em>N</em> = 28) projected on tSNE plot (<em>N</em> = 7930 cells). (<strong>b</strong>) Midkine protein expression in lysates from primary glioblastoma cell cultures from the WUM cohort (<em>N</em> = 10) assessed by Western blot. Representative image of the chemiluminescent signal of midkine bands followed by membrane stripping and probing for b-actin. The mean integrated optical density (IOD) from three experiments is presented as a barplot on the right side of the panel (<em>N</em> = 10). Error bars represent the SEM from the average of three separate experiments. (<strong>c</strong>) Midkine secretion by primary glioblastoma cell cultures (<em>N</em> = 12) and DKMG, U87, and U251 cell lines. Secretion was evaluated on cell culture medium using Luminex. Wilcoxon rank-sum test was applied to assess the differences between cell types. (<strong>d</strong>) Correlation of MDK serum concentration (pg/ml, y-axis) with <em>MDK</em> mRNA expression log2(normalized counts + 1) on the x-axis (<em>N</em> = 9). Spearman’s rank correlation was used to assess the relationship between variables.</p></figcaption></figure><p id="Par39">Subsequently, we collected 12 primary glioblastoma (GBM) tumors and derived 12 primary tumor ex vivo cultures. We then performed RNA sequencing on both tumor tissues and matched primary cultures. <em>MDK</em> expression was comparable in the tumor samples and respective primary cell cultures, suggesting that malignant cells are the predominant MDK source in GBM tumor microenvironment (TME) (Fig. <a href="#Fig1" class="usa-link">1</a>c). Notably, the mean <em>MDK</em> isoform proportions were similar to gliomas in the combined dataset (Fig. <a href="#Fig1" class="usa-link">1</a>d). The primary tumor samples had greater <em>MDK</em> isoform diversity compared to primary tumor ex vivo cultures (Fig. <a href="#Fig1" class="usa-link">1</a>d).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Profile of midkine expression in glioblastoma</h3>
<p id="Par40">In search for other cell types potentially contributing to the MDK pool, we analyzed single-cell RNA-seq dataset from 28 GBM patients (7930 cells) that included myeloid cells, oligodendrocytes, and T cells along with malignant cells (Fig. <a href="#Fig2" class="usa-link">2</a>a). The expression data indicates that malignant cells, indeed, are the predominant source of <em>MDK</em> in the tumor microenvironment. To further validate these findings, we performed western blotting of ten GBM primary cell cultures, which confirmed expression of MDK in GBM cells (Fig. <a href="#Fig2" class="usa-link">2</a>b, Supplementary Figs. <a href="#Fig3" class="usa-link">3</a> and <a href="#Fig4" class="usa-link">4</a>), as well as western blotting of macrophages and macrophages stimulated with GBM-conditioned media, both with no signs of MDK expression (Supplementary Fig. <a href="#Fig4" class="usa-link">4</a>). To investigate potential mechanisms leading to high MDK expression, we examined the correlation of MDK with retinol binding protein 1 (<em>RBP1</em>) expression as well as with HALLMARK_HYPOXIA and HALLMARK_ESTROGEN_RESPONSE_LATE signature scores (Supplementary Fig. <a href="#Fig5" class="usa-link">5</a>a-c). The correlation coefficients were <em>R</em> = 0.59, <em>R</em> = 0.32, and <em>R</em> = 0.28, respectively (<em>p</em> &lt; 2.2e-16 for all).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368106_41598_2025_16253_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4b3/12368106/00c39214c1a4/41598_2025_16253_Fig3_HTML.jpg" loading="lazy" id="d33e1150" height="609" width="796" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prognostic role of <em>MDK</em> in gliomas. Kaplan–Meier survival analysis of overall survival based on MDK expression in combined glioma dataset. In (<strong>a</strong>) the combined dataset is stratified by grade only into Grade II (N = 316), Grade III (N = 311), and Grade IV (N = 263) subgroups. In (<strong>b</strong>) <em>IDH</em><sup>mutant</sup> tumors are stratified by grade into Grade II <em>IDH</em><sup>mutant</sup> (N = 284), Grade III <em>IDH</em><sup>mutant</sup> (N = 211), and Grade IV <em>IDH</em><sup>mutant</sup> (N = 38). In (<strong>c</strong>) <em>IDH</em><sup>wildtype</sup> tumors are stratified by grade into Grade II <em>IDH</em><sup>wildtype</sup> (N = 30), Grade III <em>IDH</em><sup>wildtype</sup> (N = 100), and Grade IV <em>IDH</em><sup>wildtype</sup> (N = 218). Below each plot is a ‘number at risk’ table - the cumulative number of events table and the cumulative number of censored subjects table. Statistical comparison was performed by using the log-rank test.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368106_41598_2025_16253_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4b3/12368106/8b88a2f10e5a/41598_2025_16253_Fig4_HTML.jpg" loading="lazy" id="d33e1248" height="395" width="790" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of <em>MDK</em><sup>positive</sup> and <em>MDK</em><sup>negative</sup> malignant cells. (<strong>a</strong>) MDK expression at single-cell level in glioblastoma tumor tissue (<em>N</em> = 28 donors and 7930 cells). (<strong>b</strong>) Differential gene expression analysis between <em>MDK</em><sup>positive</sup> (<em>N</em> = 5041 cells) and <em>MDK</em><sup>negative</sup> (<em>N</em> = 1822 cells) malignant cells (<strong>c</strong>) Gene-set enrichment analysis (Hallmark gene set) of differentially expressed genes between <em>MDK</em><sup>positive</sup> (<em>N</em> = 5041 cells) and <em>MDK</em><sup>negative</sup> (<em>N</em> = 1822 cells) cells. Significantly enriched pathways are shown, with normalized enrichment scores and adjusted p-values.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368106_41598_2025_16253_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4b3/12368106/df1faa037e3f/41598_2025_16253_Fig5_HTML.jpg" loading="lazy" id="d33e1458" height="788" width="785" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of <em>MDK</em><sup>high</sup> and <em>MDK</em><sup>low</sup> tumors. (<strong>a</strong>) Distribution of immune cells across the samples in <em>MDK</em><sup>high</sup> (<em>N</em> = 221) and <em>MDK</em><sup>low</sup> (<em>N</em> = 35) tumors in the combined cohort. Additionally, graphs displaying MDK expression, MDK group assignment, original dataset, and CIBERSORTx p-value are displayed below the heatmaps. Above the heatmap, overall survival is displayed as both length in years and the occurrence of death. (<strong>b</strong>) Differential gene expression analysis between <em>MDK</em><sup>high</sup> (<em>N</em> = 221) and <em>MDK</em><sup>low</sup> (<em>N</em> = 35) GBM tumors from the combined dataset (<em>N</em> = 256). (<strong>c</strong>) A table plot of gene-set enrichment analysis (Hallmark gene set) of differentially expressed genes between <em>MDK</em><sup>high</sup> (<em>N</em> = 221) and <em>MDK</em><sup>low</sup> (<em>N</em> = 35) tumors. Significantly enriched pathways are shown, with normalized enrichment scores and adjusted p-values.</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">Midkine secretion by malignant cells</h3>
<p id="Par42">After confirming malignant GBM cells as the primary source of <em>MDK</em> mRNA expression, we proceeded to evaluate MDK secretion to determine whether its expression is reflected at the secretion level. For this purpose, we performed Luminex and Enzyme-Linked Immunosorbent Assay (ELISA) assays on supernatants from 12 primary cell cultures and three commercially available GBM cell lines: DKMG, U251, and U87. MDK was secreted by all the primary cell cultures, reaching a median concentration of 2049 pg/ml (per 1 million cells) after a 48-hour culture in 5 ml (Fig. <a href="#Fig2" class="usa-link">2</a>c). Among commercially available cell lines, DKMG was the only one to secrete significant quantities of MDK (median concentration of 3244 pg/ml per 1 million cells), whereas U87 and U251 only secreted negligible amounts (140pg/ml and 178pg/ml per 1 million cells, respectively).</p>
<p id="Par43">With the substantial potential of primary glioblastoma cells for MDK secretion, we decided to assess whether local expression correlates to a systemic rise of MDK levels, which could potentially be used for the non-invasive identification of <em>MDK</em><sup>high</sup> tumors. For this purpose, we analyzed plasma samples from 9 GBM patients with matched tumor bulk RNA-seq data. While midkine serum concentration was higher than the norms reported for healthy people, there was no correlation between MDK expression and MDK serum concentration (<em>R</em> = −0.23, <em>p</em> = 0.55) (Fig. <a href="#Fig2" class="usa-link">2</a>d).</p></section><section id="Sec19"><h3 class="pmc_sec_title">Prognostic value of midkine in glioblastoma</h3>
<p id="Par44">The existing selective pressure for higher <em>MDK</em> expression, favoring its specific isoform (ENST00000395566), suggests its tumor-promoting role in gliomas. To assess its prognostic significance, we have performed Kaplan-Meier univariate Cox regression analyses in gliomas using <em>MDK</em> expression as an overall survival (OS) predictor. First, patients were split into <em>MDK</em><sup>high</sup> and <em>MDK</em><sup>low</sup> groups based on the abundance of <em>MDK</em> using the maxstat algorithm, as described in detail in materials and methods. Then, the tumors were stratified by either grade (Fig. <a href="#Fig3" class="usa-link">3</a>a) or grade and <em>IDH</em> mutation status (Fig. <a href="#Fig3" class="usa-link">3</a>b &amp; c). Both Kaplan-Meier and Cox regression survival analyses showed that patients with <em>MDK</em><sup>high</sup> tumors had significantly shorter OS than those with <em>MDK</em><sup>low</sup> tumors in Grade III and Grade IV gliomas (both <em>IDH</em><sup>wildtype</sup> and <em>IDH</em><sup>mutant</sup> tumors combined) (Fig. <a href="#Fig3" class="usa-link">3</a>a, Supplementary Fig. <a href="#Fig6" class="usa-link">6</a>a), as well as <em>IDH</em><sup>wildtype</sup> gliomas in general (Grades II, III and IV) (Supplementary Fig. <a href="#Fig6" class="usa-link">6</a>b and Supplementary Fig. <a href="#Fig7" class="usa-link">7</a>a-b). Further analysis stratified by both grade and <em>IDH</em> mutation status revealed that only in grade IV <em>IDH</em><sup>wildtype</sup> tumors high <em>MDK</em> expression was significantly associated with poor prognosis (HR = 1.94, CI = 1.32–2.85, <em>p</em> &lt; 0.001) (Fig. <a href="#Fig3" class="usa-link">3</a>b-c and Supplementary Fig. <a href="#Fig6" class="usa-link">6</a>c-d).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368106_41598_2025_16253_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4b3/12368106/8c9e1acbd840/41598_2025_16253_Fig6_HTML.jpg" loading="lazy" id="d33e1548" height="797" width="776" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The effect of Midkine on macrophage secretome. The macrophages from different donors (<em>N</em> = 3) were stimulated with a control medium and rising physiological concentrations of midkine (0.1, 0.5, 1, and 2.5 ug/ml, the left plot of each panel) or conditioned medium from different primary GBM cell cultures (<em>N</em> = 9) with a neutralizing anti-MDK antibody added (right plot of each panel), with GBM culture medium serving as a control. The secretion of the following analytes was analyzed: (<strong>a</strong>) CCL14, (<strong>b</strong>) CCL17, (<strong>c</strong>) CCL18, (<strong>d</strong>) CXCL11, (<strong>e</strong>) IL-10, (<strong>f</strong>) Granzyme B, (<strong>g</strong>) PD-L1, (<strong>h</strong>) IFN-y, and (<strong>i</strong>) GM-CSF. The results are presented as bar plots with a median and a dot plot with each data point. The concentration of analytes is displayed in ng/ml.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368106_41598_2025_16253_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4b3/12368106/38fca05e7c0c/41598_2025_16253_Fig7_HTML.jpg" loading="lazy" id="d33e1703" height="749" width="800" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Somatic mutations in <em>MDK</em><sup>high</sup> and <em>MDK</em><sup>low</sup> tumors. (<strong>a</strong>) The most frequently mutated genes in <em>MDK</em><sup>high</sup> (<em>N</em> = 124) and <em>MDK</em><sup>low</sup> (<em>N</em> = 28) tumors. (<strong>b</strong>) Differentially mutated genes between <em>MDK</em><sup>high</sup> (<em>N</em> = 124) and <em>MDK</em><sup>low</sup> (<em>N</em> = 28) displayed as a forest plot. Bars indicate a 95% confidence interval of the odds ratio. The adjacent table includes the number of samples in both groups with the mutations in the highlighted gene. Statistical significance was assessed by Fisher’s exact test. (<strong>c</strong>) Differentially methylated probes in <em>MDK</em><sup>high</sup> (<em>N</em> = 42) and <em>MDK</em><sup>low</sup> (<em>N</em> = 12) tumors. (d) Heatmap of DNA methylation and linked mRNA expression patterns in <em>MDK</em><sup>high</sup> (<em>N</em> = 42) and <em>MDK</em><sup>low</sup> (<em>N</em> = 12) tumors in the TCGA-GBM cohort.</p></figcaption></figure></section><section id="Sec20"><h3 class="pmc_sec_title">Single-cell map of <em>MDK</em><sup>positive</sup> and <em>MDK</em><sup>negative</sup> glioblastoma cells</h3>
<p id="Par46">After confirming MDK’s role in GBM prognosis, we explored its biological effects through multiomic analysis. First, we investigated the <em>MDK</em> expression in GBM cells at single-cell level in single cell RNA-seq data. The <em>MDK</em> expression was heterogeneous in most donors, however, the vast majority of malignant cells were <em>MDK</em> positive (Fig. <a href="#Fig4" class="usa-link">4</a>a). That prompted us to explore the differences between <em>MDK</em><sup>negative</sup> and <em>MDK</em><sup>positive</sup> cells through differential gene expression analysis followed by pathway enrichment analysis (Fig. <a href="#Fig4" class="usa-link">4</a>b).</p>
<p id="Par48">Some of the most downregulated genes in <em>MDK</em><sup>positive</sup> cells were <em>HBB</em> (hemoglobin beta chain), <em>ITM2A</em> (Integral Membrane Protein 2 A), and <em>LOX</em> (Lysyl Oxidase) (Fig. <a href="#Fig3" class="usa-link">3</a>b and Supplementary File 1). Conversely, <em>MDK</em><sup>positive</sup> cells were enriched in <em>MSMP</em> (Microseminoprotein, Prostate Associated), <em>COL1A2</em> (Collagen Type I Alpha 2 Chain) and <em>COL3A1</em> (Collagen Type III Alpha 1 Chain), and <em>PLAU</em> (urokinase) (Fig. <a href="#Fig4" class="usa-link">4</a>b and Supplementary File 1). Then, we studied <em>MDK</em> expression in different cell states, as well as assessed the proportion of malignant cells in each phase of the cell cycle in <em>MDK</em><sup>negative</sup> and <em>MDK</em><sup>positive</sup> cells. A higher proportion of <em>MDK</em><sup>positive</sup> cells were in the S phase, and similarly, mean MDK expression was higher in cells in the S phase (Supplementary Fig. 8). Holistically, <em>MDK</em><sup>positive</sup> cells were enriched in coagulation and epithelial-mesenchymal transition pathways (Fig. <a href="#Fig4" class="usa-link">4</a>c).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Gene-expression networks of <em>MDK</em><sup>high</sup> and <em>MDK</em><sup>low</sup> glioblastomas</h3>
<p id="Par49">To further investigate MDK-related footprint we divided the GBM from the combined dataset based on <em>MDK</em> expression into “High” (<em>n</em> = 221) and “Low” (<em>n</em> = 35) groups. The cutoff point was determined using the maxstat algorithm, as described in detail in materials and methods. We then estimated the cellular composition of GBM tumors with CIBERSORTx using the LM22 signature matrix. Surprisingly, the immune cell composition in <em>MDK</em><sup>high</sup> and <em>MDK</em><sup>low</sup> tumors remained similar, with only differences in the estimated abundance of activated NK cells (1.9% vs. 1.2%, <em>p</em> = 0.006, <em>MDK</em><sup>low</sup> and <em>MDK</em><sup>high</sup> respectively), regulatory T cells (Tregs, 1.1% vs. 1.8%, <em>p</em> = 0.016, <em>MDK</em><sup>low</sup> and <em>MDK</em><sup>high</sup> respectively) and naïve CD4 T cells (0.32% vs. 0.18%, <em>p</em> = 0.02, <em>MDK</em><sup>low</sup> and <em>MDK</em><sup>high</sup> respectively) (Fig. <a href="#Fig5" class="usa-link">5</a>a). The estimated proportions of infiltrating cell types, together with metadata including MDK status, MDK expression, overall survival (OS) status, OS time, and source dataset, are presented as a heatmap in Fig. <a href="#Fig5" class="usa-link">5</a>a.</p>
<p id="Par51">We then performed differential gene expression analysis, followed by exploring the gene-expression pathways associated with <em>MDK</em><sup>high</sup> and <em>MDK</em><sup>low</sup> GBM tumors. One of the most upregulated group of genes in <em>MDK</em><sup>high</sup> tumors were C-C chemokines CCL18, CCL2, CCL13, CCL20, CCL14, CCL8, CCL26, CCL17, CCL25, CCL5, CCL3L1, CCL3, and CCL22, as well as C-X-C chemokines such as CXCL3, CXCL8, CXCL5, CXCL11, CXCL10, CXCL2, CXCL6, CXCL1, CXCL13, CXCL16, CXCL9 (Supplementary File 2). Other cytokines and protumorigenic factors such as interleukin 1a (IL-1α), interleukin 15 (IL-15), PD-L1, granzyme B, interleukin 10 (IL-10), interferon γ (IFN-γ), and granulocyte macrophage colony-stimulating factor (GM-CSF) were also upregulated in <em>MDK</em><sup>high</sup> tumors (Supplementary File 2). Pathway enrichment analysis revealed that <em>MDK</em><sup>high</sup> tumors are characterized by a mixed inflammatory profile with evidence of epithelial-mesenchymal transition, DNA repair, oxidative phosphorylation, angiogenesis, and hypoxia. (Fig. <a href="#Fig5" class="usa-link">5</a>b, c).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Chemokine and cytokine-inducing role of MDK</h3>
<p id="Par52">To validate the cytokine- and chemokine-rich profile observed in <em>MDK</em><sup>high</sup> GBM tumors, we performed Luminex-based secretome profiling of human macrophages, the most abundant stromal cells in GBM, and key cytokine producers. Macrophages derived from three independent donors were stimulated with physiological concentrations of recombinant MDK, GBM-conditioned media, or GBM-conditioned media supplemented with a neutralizing anti-MDK antibody.</p>
<p id="Par53">We analyzed the secretion of 21 selected soluble factors—primarily chemokines and immunomodulatory cytokines—chosen based on their transcriptomic enrichment in <em>MDK</em><sup>high</sup> tumors. These analytes included: CCL2, MIP-1α (CCL3), CCL8, CCL13, CCL14, CCL17, CCL18, CCL26, CXCL5, CXCL6, CXCL10, CXCL11, interleukin 1 alpha (IL-1α), interleukin 10 (IL-10), interleukin 15 (IL-15), interleukin 33 (IL-33), interferon γ (IFN-γ), granulocyte-macrophage colony-stimulating factor (GM-CSF), granzyme B, programmed death-ligand 1 (PD-L1), and platelet-derived growth factor AA/BB (PDGF-AA/BB).</p>
<p id="Par54">Stimulation with recombinant MDK led to a marked increase in the secretion of chemokines such as MIP-1α (CCL3), CCL17, CCL18, CXCL5, CXCL10, and CXCL11, as well as immunomodulatory and proinflammatory factors including IL-1α, IL-15, IL-33, IL-10, GM-CSF, granzyme B, PD-L1, PDGF-AA/BB, and IFN-γ (Fig. <a href="#Fig6" class="usa-link">6</a>, Supplementary Figs. 9 &amp; 10). These effects were dose-dependent and proportional to the concentration of MDK.</p>
<p id="Par56">Exposure to GBM-conditioned media also significantly increased the secretion of several factors, including CCL17, CXCL11, IL-10, granzyme B, PD-L1, IFN-γ, GM-CSF, CCL13, CXCL5, CXCL6, CXCL10, PDGF-AA/BB, IL-1α, IL-15, and IL-33 (Fig. <a href="#Fig6" class="usa-link">6</a>, Supplementary Figs. 9 &amp; 10). Notably, this upregulation was partially reversed by the addition of a neutralizing anti-MDK antibody, implicating MDK as an effector in mediating these responses. Interestingly, GBM-conditioned media alone did not elevate secretion of CCL14, CCL18, or MIP-1α; however, their levels decreased upon anti-MDK treatment, suggesting that while mixed factors affect their expression, MDK also plays a role in their upregulation. In contrast, MDK stimulation had no apparent effect on the secretion of CCL2, CCL8, CCL13, or CCL26 within the concentration range tested (Fig. <a href="#Fig6" class="usa-link">6</a>, Supplementary Figs. 9 &amp; 10).</p>
<p id="Par57">Taken together, both the recombinant MDK stimulation and the GBM-conditioned media experiments point to MDK as a potent modulator of macrophage function, promoting the secretion of multiple immunologically relevant soluble factors—most notably CCL17, CXCL10, PD-L1, IFN-γ, GM-CSF, CXCL5, and to a lesser extent granzyme B, CCL18, CXCL6, CXCL11 (Fig. <a href="#Fig6" class="usa-link">6</a>, Supplementary Figs. 9 &amp; 10). Importantly, all of the abovementioned factors were significantly upregulated in the differential gene expression analysis in <em>MDK</em><sup>high</sup> GBM tumors.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Mutational and methylation profiles of <em>MDK</em><sup>high</sup> and <em>MDK</em><sup>low</sup> tumors</h3>
<p id="Par58">We next examined the somatic mutational profiles of <em>MDK</em><sup>high</sup> and <em>MDK</em><sup>low</sup> tumors. <em>PTEN</em>, <em>EGFR</em>, and <em>TP53</em> were, predictably, the most frequently point-mutated genes. (Fig. <a href="#Fig7" class="usa-link">7</a>a). While their prevalence remained relatively similar in both <em>MDK</em><sup>high</sup> and <em>MDK</em><sup>low</sup> tumors, <em>MDK</em><sup>low</sup> tumors were enriched in GPR132, PTCH1, TEAD1, and XKR7 mutations, which were exclusively identified in that group at a frequency of 11% (vs. 0%) (Fig. <a href="#Fig7" class="usa-link">7</a>b).</p>
<p id="Par60">Finally, we investigated the genome-wide DNA methylation patterns between <em>MDK</em><sup>high</sup> and <em>MDK</em><sup>low</sup> tumors. The latter were characterized by a unique hypermethylated genome, as indicated by the vast majority of probes with only sporadic islands of mild hypomethylation (Fig. <a href="#Fig7" class="usa-link">7</a>c). Conversely, <em>MDK</em><sup>high</sup> tumors were a heterogeneous group with a large set of hypomethylated probes, where the methylation pattern divided the samples into two groups based on four different sets of probes (Fig. <a href="#Fig7" class="usa-link">7</a>c &amp; d).</p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par61">In this study, we harnessed the power of a large, harmonized RNA-seq dataset complemented with primary tumor samples and matched primary cell cultures to comprehensively characterize midkine (MDK) expression, secretion, and MDK-associated multiomic footprint in grade IV <em>IDH</em><sup>wildtype</sup> gliomas (glioblastomas, GBM). We showed that gliomas express and secrete MDK in excessive quantities with higher expression in <em>IDH</em><sup>wildtype</sup> gliomas. Our study also indicates that MDK expression in <em>IDH</em><sup>wildtype</sup> gliomas correlates with tumor grade, potentially reflecting selective pressure favoring the ENST00000395566 isoform, and that high midkine expression is associated with a poor prognosis in <em>IDH</em><sup>wildtype</sup> grade IV gliomas (GBM). Utilization of a dataset combined from ethnically and geographically diverse cohorts ensures the robustness and clinical applicability of the expression patterns and survival analyses. Its apparent prognostic effect and heterogenous expression in GBM allowed for the identification of a large group of <em>MDK</em><sup>high</sup> tumors constituting 75% of all GBM, representing a cohort potentially benefiting from MDK-targeting treatment. Additionally, identification of the ENST00000395566 MDK isoform as the most prevalent and selectively increased along the glioma grade shows that while the truncated form of MDK has been proposed to be specific to certain tumors, in gliomas, the full-length transcript is the most prevalent and favored<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. The ENST00000395566 transcript corresponds to the MANE-selected canonical isoform and contains all functional domains of midkine, including heparin-binding and growth factor activity regions. In contrast, some less abundant isoforms may lack domains necessary for full signaling capacity, suggesting that selection pressure in GBM favors the fully functional transcript.</p>
<p id="Par62">The mechanism underlying the widespread yet variable MDK expression in GBM has not been elucidated. Originally, <em>MDK</em> has been described as a retinoic acid-inducible gene<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Given that retinoic acid signaling is commonly overactivated in high-grade gliomas and correlates with adverse prognosis, our findings suggest a plausible involvement of MDK in mediating these effects, warranting further mechanistic investigation. We have found that retinol binding protein 1 (<em>RBP1)</em> expression, in fact, strongly correlates with <em>MDK</em> expression in our combined dataset. Another possible mechanism is related to estrogen, as <em>MDK</em> is an estrogen-responsive gene<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. We show that <em>MDK</em> expression weakly correlates with the HALLMARK_ESTROGEN_RESPONSE_LATE gene signature. Hypoxia can further contribute to increased MDK expression due to hypoxia-responsive element in <em>MDK</em> promoter<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. In our study, <em>MDK</em> expression also weakly correlates with hypoxia signature, suggesting a possible contribution of retinoic acid signaling, potentially acting in concert with other pathways.</p>
<p id="Par63">Analysis on a single-cell level revealed that <em>MDK</em> expression is associated with increased cell proliferation as well as pathways related to epithelial-mesenchymal transition (EMT) and coagulation. The EMT increases the mobility potential of malignant cells and is a critical step in cancer invasion and metastasis. The increased proliferation, migration, and EMT are likely mediated through MDK-induced (anaplastic lymphoma kinase) ALK and MET pathway activation<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>–<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. High expression of pro-coagulation factors and matrix metalloproteinases can contribute to dysfunctional microvasculature, one of the hallmarks of glioblastoma<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>,<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. On a tissue level, we found that <em>MDK</em><sup>high</sup> tumors are enriched in pathways related to epithelial-mesenchymal transition, DNA repair, oxidative phosphorylation, angiogenesis, and hypoxia. Additionally, <em>MDK</em><sup>low</sup> tumors were enriched in GPR132, PTCH1, TEAD1, and XKR7 mutations, which were not identified in <em>MDK</em><sup>high</sup> tumors. Mutations in, e.g., PTCH1 are associated with favorable response to checkpoint inhibitor therapy, which might affect the treatment of <em>MDK</em><sup>low</sup> tumors in the future<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>.</p>
<p id="Par64">MDK is an established regulator of cell migration. It was shown to directly facilitate the migration of monocytes and polymorphonuclear neutrophils (PMNs) in soluble and substrate-bound forms. Additionally, MDK was shown to induce chemokine secretion in human T cells<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>,<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. In our study, we found that most highly expressed group of genes in <em>MDK</em><sup>high</sup> tumors in our study were chemokines including C-C chemokines such as CCL18, CCL2, CCL13, CCL20, CCL14, CCL8, CCL26, CCL17, CCL25, CCL5, CCL3L1, CCL3, and CCL22, as well as C-X-C chemokines such as CXCL1-3, CXCL5-6, CXCL8-11, CXCL13, CXCL16, which are known to attract TAMs, protumorigenic neutrophils and immature NK cells to the TME, as well as synergistically promoting angiogenesis along with MDK<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>–<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. Interestingly, chemokines were not upregulated in <em>MDK</em><sup>positive</sup> GBM cells, indicating that the chemokine-inducing effect of MDK is mediated by tumor-infiltrating cells, as suggested by a recent study by Guo et al.<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. <em>MDK</em><sup>high</sup> tumors were also characterized by upregulated expression of interleukin 1a (IL-1α), interleukin 15 (IL-15), PD-L1, granzyme B, interleukin 10 (IL-10), interferon-γ (IFN-γ), and granulocyte macrophage colony-stimulating factor (GM-CSF).</p>
<p id="Par65">Macrophages are key cytokine producers that shape their surrounding microenvironment. In glioblastoma (GBM), they constitute the predominant stromal component, contributing to multiple tumor-promoting processes<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. Here, we provide functional evidence that GBM-secreted MDK modulates the macrophage secretome, skewing it toward a pro-tumorigenic and immunosuppressive phenotype. Using Luminex profiling of macrophages from three independent donors, we obtained data supporting the results of differential gene expression analysis between <em>MDK</em><sup>high</sup> and <em>MDK</em><sup>low</sup> GBM tumors, and identified 20 soluble factors modulated by MDK, including chemokines: MIP-1a (CCL3), CCL13, CCL14, CCL17, CCL18, CXCL5, CXCL6, CXCL10, CXCL10, CXCL11, cytokines interleukin 1 alpha (IL-1α), interleukin 10 (IL-10), interleukin 15 (IL-15), interleukin 33 (IL-33), interferon γ (IFN-γ), granulocyte-macrophage colony-stimulating factor (GM-CSF), platelet-derived growth factor AA/BB (PDGF-AA/BB), and other soluble factors such as programmed death-ligand 1 (PD-L1), granzyme B.</p>
<p id="Par66">Among the prominently MDK-induced chemokines, CCL17, a known CCR4 ligand, is associated with the recruitment of regulatory T cells (Tregs) and immunosuppression in various tumors. Our RNA-seq deconvolution analysis revealed a higher proportion of Tregs in <em>MDK</em><sup>high</sup> tumors compared to <em>MDK</em><sup>low</sup>, implicating the role of MDK-induced CCL17 in promoting a Treg-enriched microenvironment<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. CCL18, predominantly expressed by tumor-associated macrophages, has been shown to promote epithelial-to-mesenchymal transition (EMT), tumor cell migration, and invasion<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. Using a humanized glioma model, CCL18 was recently demonstrated to drive glioma progression via CCR8-ACP5 signaling<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. IL-10, an anti-inflammatory cytokine produced by myeloid cells, has been implicated in mediating T-cell dysfunction in the GBM microenvironment<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>. GM-CSF, another MDK-induced cytokine, supports the expansion of immunosuppressive myeloid populations in GBM through IL-4R alpha and is associated with the tumor’s characteristic lymphopenia and systemic immune dysfunction<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>,<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. Furthermore, PD-L1, a key immune checkpoint molecule, was significantly upregulated in response to MDK, potentially contributing to T-cell exhaustion in GBM<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>. Finally, Granzyme B, traditionally associated with cytotoxic lymphocytes, was also increased in MDK-stimulated macrophages. Emerging evidence suggests it may participate in tissue remodeling and immune modulation within the tumor microenvironment<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. Crucially, the use of a neutralizing anti-MDK antibody indicated that these effects were MDK-specific: the upregulation of these factors in response to GBM-conditioned media was partially reversed upon MDK blockade. Taken together, our findings identify MDK as a potent regulator of the macrophage secretome, which may indicate that it also has a similar effect in tumor-associated macrophages (TAM) in GBM by promoting the expression of factors that collectively enhance immune suppression, tumor cell invasion, and therapy resistance.</p>
<p id="Par67">At the single-cell level, the most upregulated genes in <em>MDK</em><sup>positive</sup> cells were <em>MSMP</em> (microseminoprotein), <em>PLAU</em> (urokinase), and the collagen-encoding genes <em>COL1A2</em> and <em>COL3A1</em>. <em>MSMP</em> has been implicated in the chemotaxis of CCR2-expressing cells and resistance to anti-VEGF therapy in ovarian cancer<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>,<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>. While its role in GBM remains unclear, it has been detected at both mRNA and protein levels and is suggested to be associated with tumor stemness<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. <em>PLAU</em> and its receptor <em>PLAUR</em> have been shown to induce a mesenchymal gene expression signature in GBM cells, promoting tumor cell survival and correlating with poor prognosis<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>. Collagens and extracellular matrix components are critical in GBM biology; inhibition of <em>COL1A2</em> and <em>COL3A1</em> significantly suppresses cell proliferation, colony formation, and invasiveness. Notably, <em>COL1A2</em> expression is reduced by phosphoinositide 3-kinase (PI3K) inhibition, and since MDK signals through the PI3K pathway, this may explain the observed upregulation of <em>COL1A2</em> in MDK-positive cells. Earlier studies also support the role of MDK in promoting collagen expression<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>–<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>. Conversely, <em>HBB</em> (hemoglobin subunit beta) and <em>LOX</em> (lysyl oxidase) were among the most downregulated genes in <em>MDK</em><sup>positive</sup> cells. <em>HBB</em>, encoding beta-globin, has been reported to act as both a tumor suppressor and promoter, depending on the cancer type<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>,<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup>. The downregulation of <em>LOX</em> is unexpected, given its established role in enhancing glioma invasiveness via extracellular matrix crosslinking and remodeling through enhancing epidermal growth factor receptor (EGFR) signaling<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>–<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup>. We hypothesize that in <em>MDK</em><sup>positive</sup> cells, where the MDK receptor LRP-1 (low density lipoprotein receptor-related protein 1) stabilizes activated EGFR, the EGFR-entrapping function of LOX may become redundant, potentially explaining its decreased expression<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>. Further investigation is warranted to elucidate the interplay between MDK and LOX signaling in GBM.</p>
<p id="Par68">All of the above-mentioned rationales may explain the lower survival rate and worse prognostic outcomes in the <em>MDK</em><sup>high</sup> group, indicating that MDK plays a central role in GBM biology, and its inhibition might be a feasible strategy to increase the treatment efficacy of larger treatment regimens through inhibition of GBM cell proliferation and EMT as well as rebalancing of the tumor microenvironment. After MDK was discovered in 1988, it soon became a spotlight target in cancer research<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup>. However, the discovered complexity of MDK signaling and its receptor network halted further research due to a need for advanced molecular biology techniques that were yet unavailable<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. Recent advancements in molecular biology and genetic engineering techniques have made targeting MDK feasible. Several <em>MDK</em> inhibition strategies, including MDK inhibitor iMDK, siRNA, and CRISPR-Cas9 liposomes, were evaluated in preclinical studies<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>. These initial reports show that MDK inhibition restores temozolomide (TMZ) sensitivity and attenuates the growth of GBM cell lines and patient-derived GBM models<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>.</p>
<p id="Par69">A particularly promising approach is based on crizotinib, an orally available 1st generation ATP-competitive ALK and c-MET dual inhibitor<sup><a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>,<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup>. After encouraging preclinical reports showing potent antitumor activity, a phase I clinical study (<a href="https://clinicaltrials.gov/ct2/show/NCT02270034" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT02270034</a>) has been launched and completed<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>–<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup>. The treatment was well-tolerated and the initial efficacy report showed that crizotinib combined with standard of care – TMZ (temzolomide) + radiotherapy (RT), resulted in a median progression-free survival (PFS) of 10.7 months and a median overall survival (OS) of 22.6 months<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup>. While no direct comparison with only TMZ + RT was made, both OS and PFS were higher than expected and higher than in other studies combining other small-molecule inhibitors with TMZ and RT<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>–<a href="#CR107" class="usa-link" aria-describedby="CR107">107</a></sup>. Notably, the pharmacotherapy was initiated 5.3 weeks (median) after surgery, and post-treatment MDK serum levels were similar to those observed at baseline. Since MDK is an injury-inducible cytokine, its levels most likely rise after surgery-induced tissue damage, promoting survival and proliferation of residual GBM cells<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup>. This indicates that any MDK-targeted therapy should be initiated at the diagnosis if it does not interfere with surgery and post-surgical care. A possible prognostic value of serum MDK concentrations has not been observed. We show that, while <em>MDK</em> mRNA levels have robust prognostic value, they do not correlate to MDK serum levels. However, both our study and the <a href="https://clinicaltrials.gov/ct2/show/NCT02270034" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT02270034</a> clinical trial are limited by a small sample size, and further research is needed to evaluate the correlation of MDK mRNA expression and MDK serum levels and a possible prognostic value of the latter.</p>
<p id="Par70">This study has several limitations. While our aim was to conduct an explorative analysis of MDK expression patterns, survival associations, and its putative effects on GBM biology, much of our work is based on in silico analyses. As such, inferred pathway activation, cell-type deconvolution, and expression correlations are hypothesis-generating and require further validation both in vivo through manipulating <em>MDK</em>, as well as in additional ex vivo systems. Additionally, macrophage-stimulation experiments were limited to three macrophage donors, which may not fully capture inter-individual variability in immune responses. While our transcriptomic analyses were based on large, geographically, and ethnically diverse datasets corrected for batch effects, functional validation (including serum and conditioned media experiments) was conducted on a relatively narrow patient cohort, limiting generalizability. Importantly, our study does not include in vivo validation of MDK-targeted therapies in GBM models, a critical step before clinical translation. Despite these limitations, we addressed potential biases by integrating multiple independent datasets, applying rigorous statistical controls, and complementing in silico findings with functional macrophage assays and protein-level analyses.</p>
<p id="Par71">In summary, our study’s combination of computational and functional analyses provides novel insight into MDK as a key contributor to a distinct, aggressive glioblastoma phenotype. We characterized MDK expression patterns in a wide range of gliomas from ethnically different cohorts and revealed coordinated changes in its expression and isoform proportions. We showed that the primary source of MDK in GBM TME are the malignant cells, producing MDK likely upon retinoic acid-, estrogen-, and hypoxia-induced signaling. Multiomic analysis shows that MDK has strong, pleiotropic protumorigenic effects excreted through both direct effects on malignant cells and indirect through reshaping the TME, as confirmed by uncovering MDK’s potential to induce protumorigenic cytokine and chemokine profile in macrophages. These facts underline the potential for MDK inhibition in a large subset of GBM patients, particularly in combination treatment.</p></section><section id="Sec25"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368106/bin/41598_2025_16253_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (1.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12368106/bin/41598_2025_16253_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (1.2MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12368106/bin/41598_2025_16253_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (2MB, docx) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>M.L. conceptualized and designed the study, performed data collection, analyzed data, drafted the initial manuscript, reviewed, and finalized the manuscript. K.Z. conceptualized and designed the study, performed data collection, analyzed data, reviewed, and finalized the manuscript. A.G. performed data collection and reviewed the manuscript. P.S. performed data collection and reviewed the manuscript. I.K. performed data collection and reviewed the manuscript. R.Z. conceptualized the study, reviewed, and finalized the manuscript. The authors approved the final manuscript as submitted and have agreed to be accountable for all aspects of the work.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the National Science Centre (2020/37/B/NZ6/02191); Medical Research Agency (2020/ABM/04/00002); National Centre for Research and Development (STRATEGMED3/307326/6/NCBR/2017).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Sequence data have been deposited at the European Genome-phenome Archive (EGA), hosted by the EBI and the CRG, under accession number EGAS00001006267. M.L. will provide the code used for data analysis upon request from the corresponding author.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par72">Radoslaw Zagożdżon is an <em>ad hoc</em> scientific consultant for 4Cell Therapies S. A. (Gliwice, Poland). The remaining authors declare no conflict of interest.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Hanif, F., Muzaffar, K., Perveen, K., Malhi, S. M. &amp; Simjee, S. U. Glioblastoma multiforme: A review of its epidemiology and pathogenesis through clinical presentation and treatment. <em>Asian Pac. J. Cancer Prev.</em><strong>18</strong>, 3–9 (2017).
</cite> [<a href="https://doi.org/10.22034/APJCP.2017.18.1.3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5563115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28239999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanif,%20F.,%20Muzaffar,%20K.,%20Perveen,%20K.,%20Malhi,%20S.%20M.%20&amp;%20Simjee,%20S.%20U.%20Glioblastoma%20multiforme:%20A%20review%20of%20its%20epidemiology%20and%20pathogenesis%20through%20clinical%20presentation%20and%20treatment.%20Asian%20Pac.%20J.%20Cancer%20Prev.18,%203%E2%80%939%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Davis, M. E. &amp; Glioblastoma Overview of disease and treatment. <em>Clin. J. Oncol. Nurs.</em><strong>20</strong>, S2–S8 (2016).
</cite> [<a href="https://doi.org/10.1188/16.CJON.S1.2-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5123811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27668386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davis,%20M.%20E.%20&amp;%20Glioblastoma%20Overview%20of%20disease%20and%20treatment.%20Clin.%20J.%20Oncol.%20Nurs.20,%20S2%E2%80%93S8%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Louis, D. N. et al. The 2021 WHO classification of tumors of the central nervous system: a summary. <em>Neuro Oncol.</em><strong>23</strong>, 1231–1251 (2021).
</cite> [<a href="https://doi.org/10.1093/neuonc/noab106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8328013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34185076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Louis,%20D.%20N.%20et%20al.%20The%202021%20WHO%20classification%20of%20tumors%20of%20the%20central%20nervous%20system:%20a%20summary.%20Neuro%20Oncol.23,%201231%E2%80%931251%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Poon, M. T. C., Sudlow, C. L. M., Figueroa, J. D. &amp; Brennan, P. M. Longer-term (≥ 2 years) survival in patients with glioblastoma in population-based studies pre- and post-2005: a systematic review and meta-analysis. <em>Sci. Rep.</em><strong>10</strong>, 11622 (2020).
</cite> [<a href="https://doi.org/10.1038/s41598-020-68011-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7363854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32669604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poon,%20M.%20T.%20C.,%20Sudlow,%20C.%20L.%20M.,%20Figueroa,%20J.%20D.%20&amp;%20Brennan,%20P.%20M.%20Longer-term%20(%E2%89%A5%E2%80%892%20years)%20survival%20in%20patients%20with%20glioblastoma%20in%20population-based%20studies%20pre-%20and%20post-2005:%20a%20systematic%20review%20and%20meta-analysis.%20Sci.%20Rep.10,%2011622%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Di Nunno, V., Gatto, L., Tosoni, A., Bartolini, S. &amp; Franceschi, E. Implications of BRAF V600E mutation in gliomas: molecular considerations, prognostic value and treatment evolution. <em>Front. Oncol.</em><strong>12</strong>, 1067252 (2023).
</cite> [<a href="https://doi.org/10.3389/fonc.2022.1067252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9846085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36686797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Di%20Nunno,%20V.,%20Gatto,%20L.,%20Tosoni,%20A.,%20Bartolini,%20S.%20&amp;%20Franceschi,%20E.%20Implications%20of%20BRAF%20V600E%20mutation%20in%20gliomas:%20molecular%20considerations,%20prognostic%20value%20and%20treatment%20evolution.%20Front.%20Oncol.12,%201067252%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Kanemaru, Y. et al. Dramatic response of BRAF V600E-mutant epithelioid glioblastoma to combination therapy with BRAF and MEK inhibitor: establishment and xenograft of a cell line to predict clinical efficacy. <em>Acta Neuropathol. Commun.</em><strong>7</strong>, 119 (2019).
</cite> [<a href="https://doi.org/10.1186/s40478-019-0774-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6659204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31345255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanemaru,%20Y.%20et%20al.%20Dramatic%20response%20of%20BRAF%20V600E-mutant%20epithelioid%20glioblastoma%20to%20combination%20therapy%20with%20BRAF%20and%20MEK%20inhibitor:%20establishment%20and%20xenograft%20of%20a%20cell%20line%20to%20predict%20clinical%20efficacy.%20Acta%20Neuropathol.%20Commun.7,%20119%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Pearson, J. R. D. et al. Immune escape in glioblastoma multiforme and the adaptation of immunotherapies for treatment. <em>Frontiers Immunology</em><strong>11</strong>, 582106 (2020).</cite> [<a href="https://doi.org/10.3389/fimmu.2020.582106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7594513/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33178210/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>DeCordova, S. et al. Molecular heterogeneity and immunosuppressive microenvironment in glioblastoma. <em>Frontiers Immunology</em><strong>11</strong>, 1402 (2020).</cite> [<a href="https://doi.org/10.3389/fimmu.2020.01402" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7379131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32765498/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>McLendon, R. et al. Comprehensive genomic characterization defines human glioblastoma genes and core pathways. <em>Nature</em><strong>455</strong>, 1061–1068 (2008).
</cite> [<a href="https://doi.org/10.1038/nature07385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2671642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18772890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McLendon,%20R.%20et%20al.%20Comprehensive%20genomic%20characterization%20defines%20human%20glioblastoma%20genes%20and%20core%20pathways.%20Nature455,%201061%E2%80%931068%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Brennan, C. W. et al. The somatic genomic landscape of glioblastoma. <em>Cell</em><strong>155</strong>, 462–477 (2013).
</cite> [<a href="https://doi.org/10.1016/j.cell.2013.09.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3910500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24120142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brennan,%20C.%20W.%20et%20al.%20The%20somatic%20genomic%20landscape%20of%20glioblastoma.%20Cell155,%20462%E2%80%93477%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Inda, M. M. et al. Tumor heterogeneity is an active process maintained by a mutant EGFR-induced cytokine circuit in glioblastoma. <em>Genes Dev.</em><strong>24</strong>, 1731–1745 (2010).
</cite> [<a href="https://doi.org/10.1101/gad.1890510" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2922502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20713517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Inda,%20M.%20M.%20et%20al.%20Tumor%20heterogeneity%20is%20an%20active%20process%20maintained%20by%20a%20mutant%20EGFR-induced%20cytokine%20circuit%20in%20glioblastoma.%20Genes%20Dev.24,%201731%E2%80%931745%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Domagala, J. et al. The tumor Microenvironment—A metabolic obstacle to NK cells’ activity. <em>Cancers</em><strong>12</strong>, 3542 (2020).
</cite> [<a href="https://doi.org/10.3390/cancers12123542" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7761432/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33260925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Domagala,%20J.%20et%20al.%20The%20tumor%20Microenvironment%E2%80%94A%20metabolic%20obstacle%20to%20NK%20cells%E2%80%99%20activity.%20Cancers12,%203542%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Neumaier, E. E., Rothhammer, V. &amp; Linnerbauer, M. The role of midkine in health and disease. <em>Frontiers Immunology</em><strong>14</strong>, 1310094 (2023).</cite> [<a href="https://doi.org/10.3389/fimmu.2023.1310094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10720637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38098484/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Lorente, M. et al. Stimulation of the midkine/alk axis renders glioma cells resistant to cannabinoid antitumoral action. <em>Cell. Death Differ.</em><strong>18</strong>, 959–973 (2011).
</cite> [<a href="https://doi.org/10.1038/cdd.2010.170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3131933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21233844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lorente,%20M.%20et%20al.%20Stimulation%20of%20the%20midkine/alk%20axis%20renders%20glioma%20cells%20resistant%20to%20cannabinoid%20antitumoral%20action.%20Cell.%20Death%20Differ.18,%20959%E2%80%93973%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>López-Valero, I. et al. Midkine signaling maintains the self-renewal and tumorigenic capacity of glioma initiating cells. <em>Theranostics</em><strong>10</strong>, 5120–5136 (2020).
</cite> [<a href="https://doi.org/10.7150/thno.41450" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7163450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32308772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%B3pez-Valero,%20I.%20et%20al.%20Midkine%20signaling%20maintains%20the%20self-renewal%20and%20tumorigenic%20capacity%20of%20glioma%20initiating%20cells.%20Theranostics10,%205120%E2%80%935136%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Hu, B. et al. Midkine promotes glioblastoma progression via PI3K-Akt signaling. <em>Cancer Cell. Int.</em><strong>21</strong>, 509 (2021).
</cite> [<a href="https://doi.org/10.1186/s12935-021-02212-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8461913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34556138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20B.%20et%20al.%20Midkine%20promotes%20glioblastoma%20progression%20via%20PI3K-Akt%20signaling.%20Cancer%20Cell.%20Int.21,%20509%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Satoh, J. et al. Midkine that promotes survival of fetal human neurons is produced by fetal human astrocytes in culture. <em>Brain Res. Dev. Brain Res.</em><strong>75</strong>, 201–205 (1993).
</cite> [<a href="https://doi.org/10.1016/0165-3806(93)90024-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8261612/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Satoh,%20J.%20et%20al.%20Midkine%20that%20promotes%20survival%20of%20fetal%20human%20neurons%20is%20produced%20by%20fetal%20human%20astrocytes%20in%20culture.%20Brain%20Res.%20Dev.%20Brain%20Res.75,%20201%E2%80%93205%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kadomatsu, K., Huang, R. P., Suganuma, T., Murata, F. &amp; Muramatsu, T. A retinoic acid responsive gene MK found in the teratocarcinoma system is expressed in spatially and temporally controlled manner during mouse embryogenesis. <em>J. Cell. Biol.</em><strong>110</strong>, 607–616 (1990).
</cite> [<a href="https://doi.org/10.1083/jcb.110.3.607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2116029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1689730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kadomatsu,%20K.,%20Huang,%20R.%20P.,%20Suganuma,%20T.,%20Murata,%20F.%20&amp;%20Muramatsu,%20T.%20A%20retinoic%20acid%20responsive%20gene%20MK%20found%20in%20the%20teratocarcinoma%20system%20is%20expressed%20in%20spatially%20and%20temporally%20controlled%20manner%20during%20mouse%20embryogenesis.%20J.%20Cell.%20Biol.110,%20607%E2%80%93616%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Muramatsu, T. Midkine, a heparin-binding cytokine with multiple roles in development, repair and diseases. <em>Proc. Jpn Acad. Ser. B Phys. Biol. Sci.</em><strong>86</strong>, 410–425 (2010).
</cite> [<a href="https://doi.org/10.2183/pjab.86.410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3417803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20431264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muramatsu,%20T.%20Midkine,%20a%20heparin-binding%20cytokine%20with%20multiple%20roles%20in%20development,%20repair%20and%20diseases.%20Proc.%20Jpn%20Acad.%20Ser.%20B%20Phys.%20Biol.%20Sci.86,%20410%E2%80%93425%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Ross-Munro, E. et al. Midkine: the who, what, where, and when of a promising neurotrophic therapy for perinatal brain injury. <em>Front. Neurol.</em><strong>11</strong>, 568814 (2020).
</cite> [<a href="https://doi.org/10.3389/fneur.2020.568814" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7642484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33193008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ross-Munro,%20E.%20et%20al.%20Midkine:%20the%20who,%20what,%20where,%20and%20when%20of%20a%20promising%20neurotrophic%20therapy%20for%20perinatal%20brain%20injury.%20Front.%20Neurol.11,%20568814%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Muramatsu, H. et al. alpha4beta1- and alpha6beta1-integrins are functional receptors for midkine, a heparin-binding growth factor. <em>J. Cell. Sci.</em><strong>117</strong>, 5405–5415 (2004).
</cite> [<a href="https://doi.org/10.1242/jcs.01423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15466886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muramatsu,%20H.%20et%20al.%20alpha4beta1-%20and%20alpha6beta1-integrins%20are%20functional%20receptors%20for%20midkine,%20a%20heparin-binding%20growth%20factor.%20J.%20Cell.%20Sci.117,%205405%E2%80%935415%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Stoica, G. E. et al. Midkine binds to anaplastic lymphoma kinase (ALK) and acts as a growth factor for different cell Types *. <em>J. Biol. Chem.</em><strong>277</strong>, 35990–35998 (2002).
</cite> [<a href="https://doi.org/10.1074/jbc.M205749200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12122009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stoica,%20G.%20E.%20et%20al.%20Midkine%20binds%20to%20anaplastic%20lymphoma%20kinase%20(ALK)%20and%20acts%20as%20a%20growth%20factor%20for%20different%20cell%20Types%20*.%20J.%20Biol.%20Chem.277,%2035990%E2%80%9335998%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Mitsiadis, T. A. et al. Expression of the heparin-binding cytokines, midkine (MK) and HB-GAM (pleiotrophin) is associated with epithelial-mesenchymal interactions during fetal development and organogenesis. <em>Development</em><strong>121</strong>, 37–51 (1995).
</cite> [<a href="https://doi.org/10.1242/dev.121.1.37" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7867507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitsiadis,%20T.%20A.%20et%20al.%20Expression%20of%20the%20heparin-binding%20cytokines,%20midkine%20(MK)%20and%20HB-GAM%20(pleiotrophin)%20is%20associated%20with%20epithelial-mesenchymal%20interactions%20during%20fetal%20development%20and%20organogenesis.%20Development121,%2037%E2%80%9351%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Huang, Y. et al. Midkine induces epithelial-mesenchymal transition through Notch2/Jak2-Stat3 signaling in human keratinocytes. <em>Cell. Cycle</em>. <strong>7</strong>, 1613–1622 (2008).
</cite> [<a href="https://doi.org/10.4161/cc.7.11.5952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18469519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20Y.%20et%20al.%20Midkine%20induces%20epithelial-mesenchymal%20transition%20through%20Notch2/Jak2-Stat3%20signaling%20in%20human%20keratinocytes.%20Cell.%20Cycle.%207,%201613%E2%80%931622%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Muramatsu, H. et al. LDL receptor-related protein as a component of the midkine receptor. <em>Biochem. Biophys. Res. Commun.</em><strong>270</strong>, 936–941 (2000).
</cite> [<a href="https://doi.org/10.1006/bbrc.2000.2549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10772929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muramatsu,%20H.%20et%20al.%20LDL%20receptor-related%20protein%20as%20a%20component%20of%20the%20midkine%20receptor.%20Biochem.%20Biophys.%20Res.%20Commun.270,%20936%E2%80%93941%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Maeda, N. et al. A receptor-like protein-tyrosine phosphatase ptpzeta/rptpbeta binds a heparin-binding growth factor midkine. Involvement of arginine 78 of midkine in the high affinity binding to PTPzeta. <em>J. Biol. Chem.</em><strong>274</strong>, 12474–12479 (1999).
</cite> [<a href="https://doi.org/10.1074/jbc.274.18.12474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10212223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maeda,%20N.%20et%20al.%20A%20receptor-like%20protein-tyrosine%20phosphatase%20ptpzeta/rptpbeta%20binds%20a%20heparin-binding%20growth%20factor%20midkine.%20Involvement%20of%20arginine%2078%20of%20midkine%20in%20the%20high%20affinity%20binding%20to%20PTPzeta.%20J.%20Biol.%20Chem.274,%2012474%E2%80%9312479%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Filippou, P. S., Karagiannis, G. S. &amp; Constantinidou, A. Midkine (MDK) growth factor: a key player in cancer progression and a promising therapeutic target. <em>Oncogene</em><strong>39</strong>, 2040–2054 (2020).
</cite> [<a href="https://doi.org/10.1038/s41388-019-1124-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31801970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Filippou,%20P.%20S.,%20Karagiannis,%20G.%20S.%20&amp;%20Constantinidou,%20A.%20Midkine%20(MDK)%20growth%20factor:%20a%20key%20player%20in%20cancer%20progression%20and%20a%20promising%20therapeutic%20target.%20Oncogene39,%202040%E2%80%932054%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Hanahan, D. Hallmarks of cancer: new dimensions. <em>Cancer Discov.</em><strong>12</strong>, 31–46 (2022).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-21-1059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35022204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanahan,%20D.%20Hallmarks%20of%20cancer:%20new%20dimensions.%20Cancer%20Discov.12,%2031%E2%80%9346%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Mishima, K. et al. Increased expression of midkine during the progression of human Astrocytomas. <em>Neurosci. Lett.</em><strong>233</strong>, 29–32 (1997).
</cite> [<a href="https://doi.org/10.1016/s0304-3940(97)00619-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9324232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mishima,%20K.%20et%20al.%20Increased%20expression%20of%20midkine%20during%20the%20progression%20of%20human%20Astrocytomas.%20Neurosci.%20Lett.233,%2029%E2%80%9332%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Cheng, Y. P. et al. Midkine expression in high grade gliomas: correlation of this novel marker with proliferation and survival in human gliomas. <em>Surg. Neurol. Int.</em><strong>5</strong>, 78 (2014).
</cite> [<a href="https://doi.org/10.4103/2152-7806.133205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4061577/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24949221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20Y.%20P.%20et%20al.%20Midkine%20expression%20in%20high%20grade%20gliomas:%20correlation%20of%20this%20novel%20marker%20with%20proliferation%20and%20survival%20in%20human%20gliomas.%20Surg.%20Neurol.%20Int.5,%2078%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Yu, X. et al. MDK induces Temozolomide resistance in glioblastoma by promoting cancer stem-like properties. <em>Am. J. Cancer Res.</em><strong>12</strong>, 4825–4839 (2022).
</cite> [<a href="/articles/PMC9641408/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36381313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20X.%20et%20al.%20MDK%20induces%20Temozolomide%20resistance%20in%20glioblastoma%20by%20promoting%20cancer%20stem-like%20properties.%20Am.%20J.%20Cancer%20Res.12,%204825%E2%80%934839%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Guo, X. et al. Midkine activation of CD8 + T cells establishes a neuron–immune–cancer axis responsible for low-grade glioma growth. <em>Nat. Commun.</em><strong>11</strong>, 2177 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-15770-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7195398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32358581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20X.%20et%20al.%20Midkine%20activation%20of%20CD8%E2%80%89+%E2%80%89T%20cells%20establishes%20a%20neuron%E2%80%93immune%E2%80%93cancer%20axis%20responsible%20for%20low-grade%20glioma%20growth.%20Nat.%20Commun.11,%202177%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Xia, M., Tong, S. &amp; Gao, L. Identification of MDK as a Hypoxia- and Epithelial–Mesenchymal Transition-Related gene biomarker of glioblastoma based on a novel risk model and in vitro experiments. <em>Biomedicines</em><strong>12</strong>, 92 (2024).
</cite> [<a href="https://doi.org/10.3390/biomedicines12010092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10813330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38255198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia,%20M.,%20Tong,%20S.%20&amp;%20Gao,%20L.%20Identification%20of%20MDK%20as%20a%20Hypoxia-%20and%20Epithelial%E2%80%93Mesenchymal%20Transition-Related%20gene%20biomarker%20of%20glioblastoma%20based%20on%20a%20novel%20risk%20model%20and%20in%20vitro%20experiments.%20Biomedicines12,%2092%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Kadomatsu, K., Tomomura, M. &amp; Muramatsu, T. cDNA cloning and sequencing of a new gene intensely expressed in early differentiation stages of embryonal carcinoma cells and in mid-gestation period of mouse embryogenesis. <em>Biochem. Biophys. Res. Commun.</em><strong>151</strong>, 1312–1318 (1988).
</cite> [<a href="https://doi.org/10.1016/s0006-291x(88)80505-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3355557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kadomatsu,%20K.,%20Tomomura,%20M.%20&amp;%20Muramatsu,%20T.%20cDNA%20cloning%20and%20sequencing%20of%20a%20new%20gene%20intensely%20expressed%20in%20early%20differentiation%20stages%20of%20embryonal%20carcinoma%20cells%20and%20in%20mid-gestation%20period%20of%20mouse%20embryogenesis.%20Biochem.%20Biophys.%20Res.%20Commun.151,%201312%E2%80%931318%20(1988)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Tang, S. L., Gao, Y. L. &amp; Chen, X. B. Wnt/β-catenin up-regulates midkine expression in glioma cells. <em>Int. J. Clin. Exp. Med.</em><strong>8</strong>, 12644–12649 (2015).
</cite> [<a href="/articles/PMC4612862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26550177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang,%20S.%20L.,%20Gao,%20Y.%20L.%20&amp;%20Chen,%20X.%20B.%20Wnt/%CE%B2-catenin%20up-regulates%20midkine%20expression%20in%20glioma%20cells.%20Int.%20J.%20Clin.%20Exp.%20Med.8,%2012644%E2%80%9312649%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Gozdz, A. et al. Preservation of the hypoxic transcriptome in glioblastoma Patient-Derived cell lines maintained at Lowered oxygen tension. <em>Cancers (Basel)</em>. <strong>14</strong>, 4852 (2022).
</cite> [<a href="https://doi.org/10.3390/cancers14194852" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9564145/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36230775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gozdz,%20A.%20et%20al.%20Preservation%20of%20the%20hypoxic%20transcriptome%20in%20glioblastoma%20Patient-Derived%20cell%20lines%20maintained%20at%20Lowered%20oxygen%20tension.%20Cancers%20(Basel).%2014,%204852%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Leelatian, N. et al. Preparing viable single cells from human tissue and tumors for cytomic analysis. <em>Curr. Protoc. Mol. Biol.</em><strong>118</strong>, 25C11–25C123 (2017).</cite> [<a href="https://doi.org/10.1002/cpmb.37" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5518778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28369679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leelatian,%20N.%20et%20al.%20Preparing%20viable%20single%20cells%20from%20human%20tissue%20and%20tumors%20for%20cytomic%20analysis.%20Curr.%20Protoc.%20Mol.%20Biol.118,%2025C11%E2%80%9325C123%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Kovaka, S. et al. Transcriptome assembly from long-read RNA-seq alignments with StringTie2. <em>Genome Biol.</em><strong>20</strong>, 278 (2019).
</cite> [<a href="https://doi.org/10.1186/s13059-019-1910-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6912988/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31842956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kovaka,%20S.%20et%20al.%20Transcriptome%20assembly%20from%20long-read%20RNA-seq%20alignments%20with%20StringTie2.%20Genome%20Biol.20,%20278%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Kim, D., Paggi, J. M., Park, C., Bennett, C. &amp; Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. <em>Nat. Biotechnol.</em><strong>37</strong>, 907–915 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0201-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7605509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20D.,%20Paggi,%20J.%20M.,%20Park,%20C.,%20Bennett,%20C.%20&amp;%20Salzberg,%20S.%20L.%20Graph-based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT-genotype.%20Nat.%20Biotechnol.37,%20907%E2%80%93915%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Wickham, H. et al. Welcome to the tidyverse. <em>J. Open. Source Softw.</em><strong>4</strong>, 1686 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wickham,%20H.%20et%20al.%20Welcome%20to%20the%20tidyverse.%20J.%20Open.%20Source%20Softw.4,%201686%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Alquicira-Hernandez, J. &amp; Powell, J. E. Nebulosa recovers single-cell gene expression signals by kernel density Estimation. <em>Bioinformatics</em><strong>37</strong>, 2485–2487 (2021).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33459785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alquicira-Hernandez,%20J.%20&amp;%20Powell,%20J.%20E.%20Nebulosa%20recovers%20single-cell%20gene%20expression%20signals%20by%20kernel%20density%20Estimation.%20Bioinformatics37,%202485%E2%80%932487%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Gu, Z., Eils, R. &amp; Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. <em>Bioinformatics</em><strong>32</strong>, 2847–2849 (2016).</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27207943/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Collado-Torres, L. et al. Reproducible RNA-seq analysis using recount2. <em>Nat. Biotechnol.</em><strong>35</strong>, 319–321 (2017).
</cite> [<a href="https://doi.org/10.1038/nbt.3838" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6742427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28398307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Collado-Torres,%20L.%20et%20al.%20Reproducible%20RNA-seq%20analysis%20using%20recount2.%20Nat.%20Biotechnol.35,%20319%E2%80%93321%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Zhao, Z. et al. Chinese glioma genome atlas (CGGA): A comprehensive resource with functional genomic data from Chinese glioma patients. <em>Genom. Proteom. Bioinform.</em><strong>19</strong>, 1–12 (2021).</cite> [<a href="https://doi.org/10.1016/j.gpb.2020.10.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8498921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33662628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20Z.%20et%20al.%20Chinese%20glioma%20genome%20atlas%20(CGGA):%20A%20comprehensive%20resource%20with%20functional%20genomic%20data%20from%20Chinese%20glioma%20patients.%20Genom.%20Proteom.%20Bioinform.19,%201%E2%80%9312%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Gill, B. J. et al. MRI-localized biopsies reveal subtype-specific differences in molecular and cellular composition at the margins of glioblastoma. <em>Proc. Natl. Acad. Sci. U S A</em>. <strong>111</strong>, 12550–12555 (2014).
</cite> [<a href="https://doi.org/10.1073/pnas.1405839111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4151734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25114226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gill,%20B.%20J.%20et%20al.%20MRI-localized%20biopsies%20reveal%20subtype-specific%20differences%20in%20molecular%20and%20cellular%20composition%20at%20the%20margins%20of%20glioblastoma.%20Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A.%20111,%2012550%E2%80%9312555%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>The Cancer Genome Atlas Research Network. Comprehensive, integrative genomic analysis of diffuse Lower-Grade gliomas. <em>N. Engl. J. Med.</em><strong>372</strong>, 2481–2498 (2015).
</cite> [<a href="https://doi.org/10.1056/NEJMoa1402121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4530011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26061751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?The%20Cancer%20Genome%20Atlas%20Research%20Network.%20Comprehensive,%20integrative%20genomic%20analysis%20of%20diffuse%20Lower-Grade%20gliomas.%20N.%20Engl.%20J.%20Med.372,%202481%E2%80%932498%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Zhang, Y. et al. Purification and characterization of progenitor and mature human astrocytes reveals transcriptional and functional differences with mouse. <em>Neuron</em><strong>89</strong>, 37–53 (2016).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2015.11.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4707064/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26687838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20Purification%20and%20characterization%20of%20progenitor%20and%20mature%20human%20astrocytes%20reveals%20transcriptional%20and%20functional%20differences%20with%20mouse.%20Neuron89,%2037%E2%80%9353%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Zhang, Y. et al. An RNA-Sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex. <em>J. Neurosci.</em><strong>34</strong>, 11929–11947 (2014).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.1860-14.2014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4152602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25186741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20An%20RNA-Sequencing%20transcriptome%20and%20splicing%20database%20of%20glia,%20neurons,%20and%20vascular%20cells%20of%20the%20cerebral%20cortex.%20J.%20Neurosci.34,%2011929%E2%80%9311947%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Leek, J. T., Johnson, W. E., Parker, H. S., Jaffe, A. E. &amp; Storey, J. D. The Sva package for removing batch effects and other unwanted variation in high-throughput experiments. <em>Bioinformatics</em><strong>28</strong>, 882–883 (2012).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3307112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22257669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leek,%20J.%20T.,%20Johnson,%20W.%20E.,%20Parker,%20H.%20S.,%20Jaffe,%20A.%20E.%20&amp;%20Storey,%20J.%20D.%20The%20Sva%20package%20for%20removing%20batch%20effects%20and%20other%20unwanted%20variation%20in%20high-throughput%20experiments.%20Bioinformatics28,%20882%E2%80%93883%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Neftel, C. et al. An integrative model of cellular states, plasticity, and genetics for glioblastoma. <em>Cell</em><strong>178</strong>, 835–849e21 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.06.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6703186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31327527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Neftel,%20C.%20et%20al.%20An%20integrative%20model%20of%20cellular%20states,%20plasticity,%20and%20genetics%20for%20glioblastoma.%20Cell178,%20835%E2%80%93849e21%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Love, M. I., Huber, W. &amp; Anders, S. Moderated Estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biol.</em><strong>15</strong>, 550 (2014).
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love,%20M.%20I.,%20Huber,%20W.%20&amp;%20Anders,%20S.%20Moderated%20Estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.15,%20550%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Ignatiadis, N., Klaus, B., Zaugg, J. B. &amp; Huber, W. Data-driven hypothesis weighting increases detection power in genome-scale multiple testing. <em>Nat. Methods</em>. <strong>13</strong>, 577–580 (2016).
</cite> [<a href="https://doi.org/10.1038/nmeth.3885" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4930141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27240256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ignatiadis,%20N.,%20Klaus,%20B.,%20Zaugg,%20J.%20B.%20&amp;%20Huber,%20W.%20Data-driven%20hypothesis%20weighting%20increases%20detection%20power%20in%20genome-scale%20multiple%20testing.%20Nat.%20Methods.%2013,%20577%E2%80%93580%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Korotkevich, G. et al. Fast gene set enrichment analysis. 060012 Preprint at (2021). 10.1101/060012</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Newman, A. M. et al. Determining cell type abundance and expression from bulk tissues with digital cytometry. <em>Nat. Biotechnol.</em><strong>37</strong>, 773–782 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0114-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6610714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31061481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Newman,%20A.%20M.%20et%20al.%20Determining%20cell%20type%20abundance%20and%20expression%20from%20bulk%20tissues%20with%20digital%20cytometry.%20Nat.%20Biotechnol.37,%20773%E2%80%93782%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Silva, T. C. et al. <em>TCGA Workflow</em>: analyze cancer genomics and epigenomics data using bioconductor packages. Preprint at https://doi.org/ (2016). 10.12688/f1000research.8923.2</cite> [<a href="https://doi.org/10.12688/f1000research.8923.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5302158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28232861/" class="usa-link">PubMed</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Silva, T. C. et al. ELMER v.2: an r/bioconductor package to reconstruct gene regulatory networks from DNA methylation and transcriptome profiles. <em>Bioinformatics</em><strong>35</strong>, 1974–1977 (2019).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty902" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6546131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30364927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Silva,%20T.%20C.%20et%20al.%20ELMER%20v.2:%20an%20r/bioconductor%20package%20to%20reconstruct%20gene%20regulatory%20networks%20from%20DNA%20methylation%20and%20transcriptome%20profiles.%20Bioinformatics35,%201974%E2%80%931977%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Hoffman, P. Seurat: Tools for Single Cell Genomics. (2022).</cite>
</li>
<li id="CR58">
<span class="label">58.</span><cite>Kowalczyk, M. S. et al. Single-cell RNA-seq reveals changes in cell cycle and differentiation programs upon aging of hematopoietic stem cells. <em>Genome Res.</em><strong>25</strong>, 1860–1872 (2015).
</cite> [<a href="https://doi.org/10.1101/gr.192237.115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4665007/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26430063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kowalczyk,%20M.%20S.%20et%20al.%20Single-cell%20RNA-seq%20reveals%20changes%20in%20cell%20cycle%20and%20differentiation%20programs%20upon%20aging%20of%20hematopoietic%20stem%20cells.%20Genome%20Res.25,%201860%E2%80%931872%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Aridome, K. et al. Truncated midkine as a marker of diagnosis and detection of nodal metastases in Gastrointestinal carcinomas. <em>Br. J. Cancer</em>. <strong>78</strong>, 472–477 (1998).
</cite> [<a href="https://doi.org/10.1038/bjc.1998.517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2063076/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9716029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aridome,%20K.%20et%20al.%20Truncated%20midkine%20as%20a%20marker%20of%20diagnosis%20and%20detection%20of%20nodal%20metastases%20in%20Gastrointestinal%20carcinomas.%20Br.%20J.%20Cancer.%2078,%20472%E2%80%93477%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Miyashiro, I. et al. Expression of truncated midkine in human colorectal cancers. <em>Cancer Lett.</em><strong>106</strong>, 287–291 (1996).
</cite> [<a href="https://doi.org/10.1016/0304-3835(96)04333-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8844985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miyashiro,%20I.%20et%20al.%20Expression%20of%20truncated%20midkine%20in%20human%20colorectal%20cancers.%20Cancer%20Lett.106,%20287%E2%80%93291%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Zhao, G. et al. ERβ-mediated estradiol enhances epithelial mesenchymal transition of lung adenocarcinoma through increasing transcription of midkine. <em>Mol. Endocrinol.</em><strong>26</strong>, 1304–1315 (2012).
</cite> [<a href="https://doi.org/10.1210/me.2012-1028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5416977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22669742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20G.%20et%20al.%20ER%CE%B2-mediated%20estradiol%20enhances%20epithelial%20mesenchymal%20transition%20of%20lung%20adenocarcinoma%20through%20increasing%20transcription%20of%20midkine.%20Mol.%20Endocrinol.26,%201304%E2%80%931315%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Weckbach, L. T. et al. Midkine acts as proangiogenic cytokine in hypoxia-induced angiogenesis. <em>Am. J. Physiol. Heart Circ. Physiol.</em><strong>303</strong>, H429–438 (2012).
</cite> [<a href="https://doi.org/10.1152/ajpheart.00934.2011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22707563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weckbach,%20L.%20T.%20et%20al.%20Midkine%20acts%20as%20proangiogenic%20cytokine%20in%20hypoxia-induced%20angiogenesis.%20Am.%20J.%20Physiol.%20Heart%20Circ.%20Physiol.303,%20H429%E2%80%93438%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Wellstein, A. ALK receptor activation, ligands and therapeutic targeting in glioblastoma and in other cancers. <em>Front Oncol</em>. <strong>2</strong>, 192 (2012).</cite> [<a href="https://doi.org/10.3389/fonc.2012.00192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3525999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23267434/" class="usa-link">PubMed</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Jeon, H. M. &amp; Lee, J. MET: roles in epithelial-mesenchymal transition and cancer stemness. <em>Annals Translational Med.</em><strong>5</strong>, 5–5 (2017).</cite> [<a href="https://doi.org/10.21037/atm.2016.12.67" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5253283/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28164090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jeon,%20H.%20M.%20&amp;%20Lee,%20J.%20MET:%20roles%20in%20epithelial-mesenchymal%20transition%20and%20cancer%20stemness.%20Annals%20Translational%20Med.5,%205%E2%80%935%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Kadomatsu, K., Kishida, S. &amp; Tsubota, S. The heparin-binding growth factor midkine: the biological activities and candidate receptors. <em>J. Biochem.</em><strong>153</strong>, 511–521 (2013).
</cite> [<a href="https://doi.org/10.1093/jb/mvt035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23625998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kadomatsu,%20K.,%20Kishida,%20S.%20&amp;%20Tsubota,%20S.%20The%20heparin-binding%20growth%20factor%20midkine:%20the%20biological%20activities%20and%20candidate%20receptors.%20J.%20Biochem.153,%20511%E2%80%93521%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Du, R. et al. Matrix metalloproteinase-2 regulates vascular patterning and growth affecting tumor cell survival and invasion in GBM. <em>Neuro Oncol.</em><strong>10</strong>, 254–264 (2008).
</cite> [<a href="https://doi.org/10.1215/15228517-2008-001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2563048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18359864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Du,%20R.%20et%20al.%20Matrix%20metalloproteinase-2%20regulates%20vascular%20patterning%20and%20growth%20affecting%20tumor%20cell%20survival%20and%20invasion%20in%20GBM.%20Neuro%20Oncol.10,%20254%E2%80%93264%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Hagemann, C., Anacker, J., Ernestus, R. I. &amp; Vince, G. H. A complete compilation of matrix metalloproteinase expression in human malignant gliomas. <em>World J. Clin. Oncol.</em><strong>3</strong>, 67–79 (2012).
</cite> [<a href="https://doi.org/10.5306/wjco.v3.i5.67" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3349915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22582165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hagemann,%20C.,%20Anacker,%20J.,%20Ernestus,%20R.%20I.%20&amp;%20Vince,%20G.%20H.%20A%20complete%20compilation%20of%20matrix%20metalloproteinase%20expression%20in%20human%20malignant%20gliomas.%20World%20J.%20Clin.%20Oncol.3,%2067%E2%80%9379%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Wang, Y. et al. PTCH1 mutation promotes antitumor immunity and the response to immune checkpoint inhibitors in colorectal cancer patients. <em>Cancer Immunol. Immunother</em>. <strong>71</strong>, 111–120 (2022).
</cite> [<a href="https://doi.org/10.1007/s00262-021-02966-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8738454/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34028566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.%20et%20al.%20PTCH1%20mutation%20promotes%20antitumor%20immunity%20and%20the%20response%20to%20immune%20checkpoint%20inhibitors%20in%20colorectal%20cancer%20patients.%20Cancer%20Immunol.%20Immunother.%2071,%20111%E2%80%93120%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Horiba, M. et al. Neointima formation in a restenosis model is suppressed in midkine-deficient mice. <em>J. Clin. Invest.</em><strong>105</strong>, 489–495 (2000).
</cite> [<a href="https://doi.org/10.1172/JCI7208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC289157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10683378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Horiba,%20M.%20et%20al.%20Neointima%20formation%20in%20a%20restenosis%20model%20is%20suppressed%20in%20midkine-deficient%20mice.%20J.%20Clin.%20Invest.105,%20489%E2%80%93495%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Weckbach, L. T. et al. The cytokine midkine supports neutrophil trafficking during acute inflammation by promoting adhesion via β2 integrins (CD11/CD18). <em>Blood</em><strong>123</strong>, 1887–1896 (2014).
</cite> [<a href="https://doi.org/10.1182/blood-2013-06-510875" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24458438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weckbach,%20L.%20T.%20et%20al.%20The%20cytokine%20midkine%20supports%20neutrophil%20trafficking%20during%20acute%20inflammation%20by%20promoting%20adhesion%20via%20%CE%B22%20integrins%20(CD11/CD18).%20Blood123,%201887%E2%80%931896%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Sciumè, G., Santoni, A. &amp; Bernardini, G. Chemokines and glioma: invasion and more. <em>J. Neuroimmunol.</em><strong>224</strong>, 8–12 (2010).
</cite> [<a href="https://doi.org/10.1016/j.jneuroim.2010.05.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20656128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scium%C3%A8,%20G.,%20Santoni,%20A.%20&amp;%20Bernardini,%20G.%20Chemokines%20and%20glioma:%20invasion%20and%20more.%20J.%20Neuroimmunol.224,%208%E2%80%9312%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Yeo, E. C. F., Brown, M. P., Gargett, T. &amp; Ebert, L. M. The role of cytokines and chemokines in shaping the immune microenvironment of glioblastoma: implications for immunotherapy. <em>Cells</em><strong>10</strong>, 607 (2021).
</cite> [<a href="https://doi.org/10.3390/cells10030607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8001644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33803414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yeo,%20E.%20C.%20F.,%20Brown,%20M.%20P.,%20Gargett,%20T.%20&amp;%20Ebert,%20L.%20M.%20The%20role%20of%20cytokines%20and%20chemokines%20in%20shaping%20the%20immune%20microenvironment%20of%20glioblastoma:%20implications%20for%20immunotherapy.%20Cells10,%20607%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Lachota, M. et al. Mapping the chemotactic landscape in NK cells reveals subset-specific synergistic migratory responses to dual chemokine receptor ligation. <em>eBioMedicine</em> 96, (2023).</cite> [<a href="https://doi.org/10.1016/j.ebiom.2023.104811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10520535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37741009/" class="usa-link">PubMed</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Tibbs, E. J. &amp; Cao, X. Pro-tumoral role of granzyme B to aid in invasion and metastasis. <em>J. Immunol.</em><strong>208</strong>, 178.13 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tibbs,%20E.%20J.%20&amp;%20Cao,%20X.%20Pro-tumoral%20role%20of%20granzyme%20B%20to%20aid%20in%20invasion%20and%20metastasis.%20J.%20Immunol.208,%20178.13%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Marshall, L. A. et al. Tumors Establish resistance to immunotherapy by regulating Treg recruitment via CCR4. <em>J. Immunother Cancer</em>. <strong>8</strong>, e000764 (2020).
</cite> [<a href="https://doi.org/10.1136/jitc-2020-000764" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7692993/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33243932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marshall,%20L.%20A.%20et%20al.%20Tumors%20Establish%20resistance%20to%20immunotherapy%20by%20regulating%20Treg%20recruitment%20via%20CCR4.%20J.%20Immunother%20Cancer.%208,%20e000764%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Korbecki, J., Olbromski, M. &amp; Dzięgiel, P. CCL18 in the progression of cancer. <em>Int. J. Mol. Sci.</em><strong>21</strong>, 7955 (2020).
</cite> [<a href="https://doi.org/10.3390/ijms21217955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7663205/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33114763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Korbecki,%20J.,%20Olbromski,%20M.%20&amp;%20Dzi%C4%99giel,%20P.%20CCL18%20in%20the%20progression%20of%20cancer.%20Int.%20J.%20Mol.%20Sci.21,%207955%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Huang, Y. et al. Microglia/macrophage-derived human CCL18 promotes glioma progression via CCR8-ACP5 axis analyzed in humanized slice model. <em>Cell. Rep.</em><strong>39</strong>, 110670 (2022).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2022.110670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35417708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20Y.%20et%20al.%20Microglia/macrophage-derived%20human%20CCL18%20promotes%20glioma%20progression%20via%20CCR8-ACP5%20axis%20analyzed%20in%20humanized%20slice%20model.%20Cell.%20Rep.39,%20110670%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Ravi, V. M. et al. T-cell dysfunction in the glioblastoma microenvironment is mediated by myeloid cells releasing interleukin-10. <em>Nat. Commun.</em><strong>13</strong>, 925 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-28523-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8854421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35177622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ravi,%20V.%20M.%20et%20al.%20T-cell%20dysfunction%20in%20the%20glioblastoma%20microenvironment%20is%20mediated%20by%20myeloid%20cells%20releasing%20interleukin-10.%20Nat.%20Commun.13,%20925%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Kast, R. E. et al. Glioblastoma-synthesized G-CSF and GM-CSF contribute to growth and immunosuppression: potential therapeutic benefit from dapsone, fenofibrate, and ribavirin. <em>Tumour Biol.</em><strong>39</strong>, 1010428317699797 (2017).
</cite> [<a href="https://doi.org/10.1177/1010428317699797" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28459367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kast,%20R.%20E.%20et%20al.%20Glioblastoma-synthesized%20G-CSF%20and%20GM-CSF%20contribute%20to%20growth%20and%20immunosuppression:%20potential%20therapeutic%20benefit%20from%20dapsone,%20fenofibrate,%20and%20ribavirin.%20Tumour%20Biol.39,%201010428317699797%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Kohanbash, G. et al. GM-CSF promotes the immunosuppressive activity of glioma-Infiltrating myeloid cells through interleukin-4 receptor-α. <em>Cancer Res.</em><strong>73</strong> https://doi.org/10.1158/0008-5472.CAN-12–4124 (2013).</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-12-4124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3829000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24030977/" class="usa-link">PubMed</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Yang, T., Kong, Z. &amp; Ma, W. PD-1/PD-L1 immune checkpoint inhibitors in glioblastoma: clinical studies, challenges and potential. <em>Hum Vaccin Immunother</em>. <strong>17</strong>, 546–553 (2020).</cite> [<a href="https://doi.org/10.1080/21645515.2020.1782692" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7899692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32643507/" class="usa-link">PubMed</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Tibbs, E. &amp; Cao, X. Emerging canonical and Non-Canonical roles of granzyme B in health and disease. <em>Cancers (Basel)</em>. <strong>14</strong>, 1436 (2022).
</cite> [<a href="https://doi.org/10.3390/cancers14061436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8946077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35326588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tibbs,%20E.%20&amp;%20Cao,%20X.%20Emerging%20canonical%20and%20Non-Canonical%20roles%20of%20granzyme%20B%20in%20health%20and%20disease.%20Cancers%20(Basel).%2014,%201436%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Mitamura, T. et al. Induction of anti-VEGF therapy resistance by upregulated expression of Microseminoprotein (MSMP). <em>Oncogene</em><strong>37</strong>, 722–731 (2018).
</cite> [<a href="https://doi.org/10.1038/onc.2017.348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6040890/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29059175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitamura,%20T.%20et%20al.%20Induction%20of%20anti-VEGF%20therapy%20resistance%20by%20upregulated%20expression%20of%20Microseminoprotein%20(MSMP).%20Oncogene37,%20722%E2%80%93731%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>She, S. et al. PSMP/MSMP promotes hepatic fibrosis through CCR2 and represents a novel therapeutic target. <em>J. Hepatol.</em><strong>72</strong>, 506–518 (2020).
</cite> [<a href="https://doi.org/10.1016/j.jhep.2019.09.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31813573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?She,%20S.%20et%20al.%20PSMP/MSMP%20promotes%20hepatic%20fibrosis%20through%20CCR2%20and%20represents%20a%20novel%20therapeutic%20target.%20J.%20Hepatol.72,%20506%E2%80%93518%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Maruyama, M. et al. PC3-Secreted microprotein is expressed in glioblastoma Stem-Like cells and human glioma tissues. <em>Biol. Pharm. Bull.</em><strong>44</strong>, 910–919 (2021).
</cite> [<a href="https://doi.org/10.1248/bpb.b20-00868" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33896885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maruyama,%20M.%20et%20al.%20PC3-Secreted%20microprotein%20is%20expressed%20in%20glioblastoma%20Stem-Like%20cells%20and%20human%20glioma%20tissues.%20Biol.%20Pharm.%20Bull.44,%20910%E2%80%93919%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Gilder, A. S. et al. The urokinase receptor induces a mesenchymal gene expression signature in glioblastoma cells and promotes tumor cell survival in neurospheres. <em>Sci. Rep.</em><strong>8</strong>, 2982 (2018).
</cite> [<a href="https://doi.org/10.1038/s41598-018-21358-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5813209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29445239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gilder,%20A.%20S.%20et%20al.%20The%20urokinase%20receptor%20induces%20a%20mesenchymal%20gene%20expression%20signature%20in%20glioblastoma%20cells%20and%20promotes%20tumor%20cell%20survival%20in%20neurospheres.%20Sci.%20Rep.8,%202982%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Yamada, H., Inazumi, T., Tajima, S., Muramatsu, H. &amp; Muramatsu, T. Stimulation of collagen expression and glycosaminoglycan synthesis by midkine in human skin fibroblasts. <em>Arch. Dermatol. Res.</em><strong>289</strong>, 429–433 (1997).
</cite> [<a href="https://doi.org/10.1007/s004030050216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9248623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamada,%20H.,%20Inazumi,%20T.,%20Tajima,%20S.,%20Muramatsu,%20H.%20&amp;%20Muramatsu,%20T.%20Stimulation%20of%20collagen%20expression%20and%20glycosaminoglycan%20synthesis%20by%20midkine%20in%20human%20skin%20fibroblasts.%20Arch.%20Dermatol.%20Res.289,%20429%E2%80%93433%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Gao, Y. F., Zhu, T., Chen, J., Liu, L. &amp; Ouyang, R. Knockdown of collagen α-1(III) inhibits glioma cell proliferation and migration and is regulated by miR128-3p. <em>Oncol. Lett.</em><strong>16</strong>, 1917–1923 (2018).
</cite> [<a href="https://doi.org/10.3892/ol.2018.8830" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6036335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30008884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20Y.%20F.,%20Zhu,%20T.,%20Chen,%20J.,%20Liu,%20L.%20&amp;%20Ouyang,%20R.%20Knockdown%20of%20collagen%20%CE%B1-1(III)%20inhibits%20glioma%20cell%20proliferation%20and%20migration%20and%20is%20regulated%20by%20miR128-3p.%20Oncol.%20Lett.16,%201917%E2%80%931923%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Wang, Y. et al. COL1A2 Inhibition suppresses glioblastoma cell proliferation and invasion. <em>J. Neurosurg.</em><strong>138</strong>, 639–648 (2022).
</cite> [<a href="https://doi.org/10.3171/2022.6.JNS22319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35932265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.%20et%20al.%20COL1A2%20Inhibition%20suppresses%20glioblastoma%20cell%20proliferation%20and%20invasion.%20J.%20Neurosurg.138,%20639%E2%80%93648%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Zheng, Y. et al. Expression of β-globin by cancer cells promotes cell survival during blood-borne dissemination. <em>Nat. Commun.</em><strong>8</strong>, 14344 (2017).
</cite> [<a href="https://doi.org/10.1038/ncomms14344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5321792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28181495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20Y.%20et%20al.%20Expression%20of%20%CE%B2-globin%20by%20cancer%20cells%20promotes%20cell%20survival%20during%20blood-borne%20dissemination.%20Nat.%20Commun.8,%2014344%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Onda, M. et al. Decreased expression of haemoglobin beta (HBB) gene in anaplastic thyroid cancer and recovory of its expression inhibits cell growth. <em>Br. J. Cancer</em>. <strong>92</strong>, 2216–2224 (2005).
</cite> [<a href="https://doi.org/10.1038/sj.bjc.6602634" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2361827/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15956966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Onda,%20M.%20et%20al.%20Decreased%20expression%20of%20haemoglobin%20beta%20(HBB)%20gene%20in%20anaplastic%20thyroid%20cancer%20and%20recovory%20of%20its%20expression%20inhibits%20cell%20growth.%20Br.%20J.%20Cancer.%2092,%202216%E2%80%932224%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Laurentino, T. S., Soares, R. S., Lerario, A. M., Marie, S. K. N. &amp; Oba-Shinjo, S. M. LOXL3 Silencing affected cell adhesion and invasion in U87MG glioma cells. <em>Int. J. Mol. Sci.</em><strong>22</strong>, 8072 (2021).
</cite> [<a href="https://doi.org/10.3390/ijms22158072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8347215/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34360836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laurentino,%20T.%20S.,%20Soares,%20R.%20S.,%20Lerario,%20A.%20M.,%20Marie,%20S.%20K.%20N.%20&amp;%20Oba-Shinjo,%20S.%20M.%20LOXL3%20Silencing%20affected%20cell%20adhesion%20and%20invasion%20in%20U87MG%20glioma%20cells.%20Int.%20J.%20Mol.%20Sci.22,%208072%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Liu, C. et al. Exploring the role of LOX family in glioma progression and immune modulation. <em>Front. Immunol.</em><strong>16</strong>, 1512186 (2025).
</cite> [<a href="https://doi.org/10.3389/fimmu.2025.1512186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12014642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40270974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20C.%20et%20al.%20Exploring%20the%20role%20of%20LOX%20family%20in%20glioma%20progression%20and%20immune%20modulation.%20Front.%20Immunol.16,%201512186%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Tang, H. et al. Lysyl oxidase drives tumour progression by trapping EGF receptors at the cell surface. <em>Nat. Commun.</em><strong>8</strong>, 14909 (2017).
</cite> [<a href="https://doi.org/10.1038/ncomms14909" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5399287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28416796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang,%20H.%20et%20al.%20Lysyl%20oxidase%20drives%20tumour%20progression%20by%20trapping%20EGF%20receptors%20at%20the%20cell%20surface.%20Nat.%20Commun.8,%2014909%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Chang, C. et al. LRP-1 receptor combines EGFR signalling and eHsp90α autocrine to support constitutive breast cancer cell motility in absence of blood supply. <em>Sci. Rep.</em><strong>12</strong>, 12006 (2022).
</cite> [<a href="https://doi.org/10.1038/s41598-022-16161-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9283467/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35835845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang,%20C.%20et%20al.%20LRP-1%20receptor%20combines%20EGFR%20signalling%20and%20eHsp90%CE%B1%20autocrine%20to%20support%20constitutive%20breast%20cancer%20cell%20motility%20in%20absence%20of%20blood%20supply.%20Sci.%20Rep.12,%2012006%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Kadomatsu, K. The midkine family in cancer, inflammation and neural development. <em>Nagoya J. Med. Sci.</em><strong>67</strong>, 71–82 (2005).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/17375473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kadomatsu,%20K.%20The%20midkine%20family%20in%20cancer,%20inflammation%20and%20neural%20development.%20Nagoya%20J.%20Med.%20Sci.67,%2071%E2%80%9382%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Meng, X. et al. DNA damage repair alterations modulate M2 polarization of microglia to remodel the tumor microenvironment via the p53-mediated MDK expression in glioma. <em>EBioMedicine</em><strong>41</strong>, 185–199 (2019).
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2019.01.067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6442002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30773478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meng,%20X.%20et%20al.%20DNA%20damage%20repair%20alterations%20modulate%20M2%20polarization%20of%20microglia%20to%20remodel%20the%20tumor%20microenvironment%20via%20the%20p53-mediated%20MDK%20expression%20in%20glioma.%20EBioMedicine41,%20185%E2%80%93199%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Christensen, J. G. et al. Cytoreductive antitumor activity of PF-2341066, a novel inhibitor of anaplastic lymphoma kinase and c-Met, in experimental models of anaplastic large-cell lymphoma. <em>Mol. Cancer Ther.</em><strong>6</strong>, 3314–3322 (2007).
</cite> [<a href="https://doi.org/10.1158/1535-7163.MCT-07-0365" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18089725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Christensen,%20J.%20G.%20et%20al.%20Cytoreductive%20antitumor%20activity%20of%20PF-2341066,%20a%20novel%20inhibitor%20of%20anaplastic%20lymphoma%20kinase%20and%20c-Met,%20in%20experimental%20models%20of%20anaplastic%20large-cell%20lymphoma.%20Mol.%20Cancer%20Ther.6,%203314%E2%80%933322%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>McDermott, U. et al. Genomic alterations of anaplastic lymphoma kinase May sensitize tumors to anaplastic lymphoma kinase inhibitors. <em>Cancer Res.</em><strong>68</strong>, 3389–3395 (2008).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-07-6186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18451166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McDermott,%20U.%20et%20al.%20Genomic%20alterations%20of%20anaplastic%20lymphoma%20kinase%20May%20sensitize%20tumors%20to%20anaplastic%20lymphoma%20kinase%20inhibitors.%20Cancer%20Res.68,%203389%E2%80%933395%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Le Rhun, E. et al. Patterns of response to Crizotinib in recurrent glioblastoma according to ALK and MET molecular profile in two patients. <em>CNS Oncol.</em><strong>4</strong>, 381–386 (2015).
</cite> [<a href="https://doi.org/10.2217/cns.15.30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6083940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26498130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Le%20Rhun,%20E.%20et%20al.%20Patterns%20of%20response%20to%20Crizotinib%20in%20recurrent%20glioblastoma%20according%20to%20ALK%20and%20MET%20molecular%20profile%20in%20two%20patients.%20CNS%20Oncol.4,%20381%E2%80%93386%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Das, A. et al. Synergistic effects of Crizotinib and Temozolomide in experimental FIG-ROS1 Fusion-Positive glioblastoma. <em>Cancer Growth Metastasis</em><strong>8</strong>, 51–60 (2015). CGM.S32801.</cite> [<a href="https://doi.org/10.4137/CGM.S32801" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4667559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26648752/" class="usa-link">PubMed</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Martínez-García, M. et al. Safety and efficacy of Crizotinib in combination with Temozolomide and radiotherapy in patients with newly diagnosed glioblastoma: phase Ib GEINO 1402 trial. <em>Cancers</em><strong>14</strong>, 2393 (2022).
</cite> [<a href="https://doi.org/10.3390/cancers14102393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9139576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35625997/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mart%C3%ADnez-Garc%C3%ADa,%20M.%20et%20al.%20Safety%20and%20efficacy%20of%20Crizotinib%20in%20combination%20with%20Temozolomide%20and%20radiotherapy%20in%20patients%20with%20newly%20diagnosed%20glioblastoma:%20phase%20Ib%20GEINO%201402%20trial.%20Cancers14,%202393%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Stupp, R. et al. Radiotherapy plus concomitant and adjuvant Temozolomide for glioblastoma. <em>N. Engl. J. Med.</em><strong>352</strong>, 987–996 (2005).
</cite> [<a href="https://doi.org/10.1056/NEJMoa043330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15758009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stupp,%20R.%20et%20al.%20Radiotherapy%20plus%20concomitant%20and%20adjuvant%20Temozolomide%20for%20glioblastoma.%20N.%20Engl.%20J.%20Med.352,%20987%E2%80%93996%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Blakeley, J. O. et al. Phase II study of Iniparib with concurrent chemoradiation in patients with newly diagnosed glioblastoma. <em>Clin. Cancer Res.</em><strong>25</strong>, 73–79 (2019).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-18-0110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6367923/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30131387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blakeley,%20J.%20O.%20et%20al.%20Phase%20II%20study%20of%20Iniparib%20with%20concurrent%20chemoradiation%20in%20patients%20with%20newly%20diagnosed%20glioblastoma.%20Clin.%20Cancer%20Res.25,%2073%E2%80%9379%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Li, P. et al. A phase II study of anlotinib combined with STUPP regimen in the treatment of patients with newly diagnosed glioblastoma (GBM). <em>JCO</em> 39, 2039–2039 (2021).</cite>
</li>
<li id="CR106">
<span class="label">106.</span><cite>Roth, P. et al. EORTC 1709/CCTG CE.8: A phase III trial of Marizomib in combination with temozolomide-based radiochemotherapy versus temozolomide-based radiochemotherapy alone in patients with newly diagnosed glioblastoma. <em>JCO</em><strong>39</strong>, 2004–2004 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Roth,%20P.%20et%20al.%20EORTC%201709/CCTG%20CE.8:%20A%20phase%20III%20trial%20of%20Marizomib%20in%20combination%20with%20temozolomide-based%20radiochemotherapy%20versus%20temozolomide-based%20radiochemotherapy%20alone%20in%20patients%20with%20newly%20diagnosed%20glioblastoma.%20JCO39,%202004%E2%80%932004%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Galanis, E. et al. Phase I/II trial of Vorinostat combined with Temozolomide and radiation therapy for newly diagnosed glioblastoma: results of alliance N0874/ABTC 02. <em>Neuro-Oncology</em><strong>20</strong>, 546–556 (2018).
</cite> [<a href="https://doi.org/10.1093/neuonc/nox161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5909661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29016887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Galanis,%20E.%20et%20al.%20Phase%20I/II%20trial%20of%20Vorinostat%20combined%20with%20Temozolomide%20and%20radiation%20therapy%20for%20newly%20diagnosed%20glioblastoma:%20results%20of%20alliance%20N0874/ABTC%2002.%20Neuro-Oncology20,%20546%E2%80%93556%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368106/bin/41598_2025_16253_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (1.3MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368106/bin/41598_2025_16253_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (1.2MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368106/bin/41598_2025_16253_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (2MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Sequence data have been deposited at the European Genome-phenome Archive (EGA), hosted by the EBI and the CRG, under accession number EGAS00001006267. M.L. will provide the code used for data analysis upon request from the corresponding author.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-16253-5"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_16253.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368106/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368106/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368106%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368106/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368106/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368106/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835683/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368106/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835683/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368106/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368106/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="6YZsyzfJhvYsaUHe6Bv8fJm7A4AFf9JNMt9OHtYcNOvyEuVjf9aNxgTXh8KUzYA7">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
