
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Crystal structures of Salmonella enterica FraB deglycase reveal a conformational heterodimer with remarkable structural plasticity at the active site - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E481DE8AEECAE30581DE0022E07C07.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="prosci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359202/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Protein Science : A Publication of the Protein Society">
<meta name="citation_title" content="Crystal structures of Salmonella enterica FraB deglycase reveal a conformational heterodimer with remarkable structural plasticity at the active site">
<meta name="citation_author" content="Katerina Zakharova">
<meta name="citation_author_institution" content="Department of Biological Chemistry and Pharmacology, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author" content="Jamison D Law">
<meta name="citation_author_institution" content="Ohio State Biochemistry Program, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author" content="Yuan Gao">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author_institution" content="School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA">
<meta name="citation_author_institution" content="Native Mass Spectrometry Guided Structural Biology Center, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author" content="Sravya Kovvali">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author" content="Vicki H Wysocki">
<meta name="citation_author_institution" content="School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA">
<meta name="citation_author_institution" content="Native Mass Spectrometry Guided Structural Biology Center, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author" content="Venkat Gopalan">
<meta name="citation_author_institution" content="Ohio State Biochemistry Program, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author_institution" content="Center for RNA Biology, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author" content="Charles E Bell">
<meta name="citation_author_institution" content="Department of Biological Chemistry and Pharmacology, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author_institution" content="Ohio State Biochemistry Program, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="34">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70260">
<meta name="citation_doi" content="10.1002/pro.70260">
<meta name="citation_pmid" content="40823914">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359202/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359202/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359202/pdf/PRO-34-e70260.pdf">
<meta name="description" content="The fra locus of Salmonella enterica encodes five genes for metabolism of fructose‐asparagine, an Amadori product formed by condensation of asparagine with glucose. In the last step of this pathway, the FraB deglycase cleaves ...">
<meta name="og:title" content="Crystal structures of Salmonella enterica FraB deglycase reveal a conformational heterodimer with remarkable structural plasticity at the active site">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The fra locus of Salmonella enterica encodes five genes for metabolism of fructose‐asparagine, an Amadori product formed by condensation of asparagine with glucose. In the last step of this pathway, the FraB deglycase cleaves ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359202/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12359202">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/pro.70260"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/PRO-34-e70260.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359202%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12359202/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12359202/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359202/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-prosci.gif" alt="Protein Science : A Publication of the Protein Society logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Protein Science : A Publication of the Protein Society" title="Link to Protein Science : A Publication of the Protein Society" shape="default" href="http://www.proteinscience.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Protein Sci</button></div>. 2025 Aug 18;34(9):e70260. doi: <a href="https://doi.org/10.1002/pro.70260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/pro.70260</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protein%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Protein%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Crystal structures of <em>Salmonella enterica</em>
FraB deglycase reveal a conformational heterodimer with remarkable structural plasticity at the active site</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zakharova%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Katerina Zakharova</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Katerina Zakharova</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Biological Chemistry and Pharmacology, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zakharova%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Katerina Zakharova</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Law%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jamison D Law</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jamison D Law</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Ohio State Biochemistry Program, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Law%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">Jamison D Law</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yuan Gao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yuan Gao</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Native Mass Spectrometry Guided Structural Biology Center, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuan Gao</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kovvali%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sravya Kovvali</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sravya Kovvali</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kovvali%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sravya Kovvali</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wysocki%20VH%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Vicki H Wysocki</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Vicki H Wysocki</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Native Mass Spectrometry Guided Structural Biology Center, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wysocki%20VH%22%5BAuthor%5D" class="usa-link"><span class="name western">Vicki H Wysocki</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gopalan%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Venkat Gopalan</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Venkat Gopalan</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Ohio State Biochemistry Program, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>Center for RNA Biology, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gopalan%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Venkat Gopalan</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>6,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bell%20CE%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Charles E Bell</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Charles E Bell</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Biological Chemistry and Pharmacology, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Ohio State Biochemistry Program, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bell%20CE%22%5BAuthor%5D" class="usa-link"><span class="name western">Charles E Bell</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="pro70260-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Biological Chemistry and Pharmacology, The Ohio State University, Columbus, Ohio, USA</div>
<div id="pro70260-aff-0002">
<sup>
<sup>2</sup>
</sup>Ohio State Biochemistry Program, The Ohio State University, Columbus, Ohio, USA</div>
<div id="pro70260-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA</div>
<div id="pro70260-aff-0004">
<sup>
<sup>4</sup>
</sup>School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, Georgia, USA</div>
<div id="pro70260-aff-0005">
<sup>
<sup>5</sup>
</sup>Native Mass Spectrometry Guided Structural Biology Center, The Ohio State University, Columbus, Ohio, USA</div>
<div id="pro70260-aff-0006">
<sup>
<sup>6</sup>
</sup>Center for RNA Biology, The Ohio State University, Columbus, Ohio, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Venkat Gopalan, Department of Chemistry and Biochemistry, The Ohio State University, Columbus, OH 43210, USA. 
Email: <span>gopalan.5@osu.edu</span>
, 
Charles E. Bell, Department of Biological Chemistry and Pharmacology, The Ohio State University, Columbus, OH 43210, USA. 
Email: <span>bell.489@osu.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 23; Received 2025 May 28; Accepted 2025 Jul 25; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Protein Science</em> published by Wiley Periodicals LLC on behalf of The Protein Society.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12359202  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40823914/" class="usa-link">40823914</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The <em>fra</em> locus of <em>Salmonella enterica</em> encodes five genes for metabolism of fructose‐asparagine, an Amadori product formed by condensation of asparagine with glucose. In the last step of this pathway, the FraB deglycase cleaves 6‐phospho‐fructose‐aspartate into glucose‐6‐phosphate and aspartate. In homology models, FraB forms a homodimer with two equivalent active sites located at the dimer interface. E214 and H230, two invariant residues essential for catalysis, project into each active site cleft from opposing subunits of the dimer. Here, we have determined six crystal structures of FraB, three of a variant containing an N‐terminal His<sub>6</sub> tag and two mutations needed for crystallization (hereafter referred to as WT′), two with additional mutations to active site residues (E214A and P232A), and one of a variant with C‐terminal residues 313–325 deleted. Surprisingly, in the WT′ FraB structure, the two catalytic residues, E214 (general base) and H230 (general acid), are positioned ~22 Å apart. In the E214A and C‐terminus‐truncated FraB variants, however, a conformational change in the E214‐residing helix brings E214 and H230* to ~7 Å (* indicates residue from the second protomer that creates the inter‐subunit catalytic center). The loop bearing H230 also exhibits significant variation, ranging from being completely disordered to adopting open or closed states, with the nearby P232* residue being either <em>cis</em> or <em>trans</em>. The C‐terminal residues 313–325 form a flexible “C‐tail” that can be fully disordered, bind in the active site to block access of substrate, or angle across the active site to wrap across the other subunit of the dimer and potentially close over substrate. Collectively, these structures reveal that FraB is a conformational heterodimer with two chemically identical subunits that are constrained to adopt different structures as they come together for catalysis. This plasticity likely involves correlated opening and closure of the two active sites for their respective binding and release of substrates and ligands.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> Amadori compounds, fructose‐asparagine, FraB deglycase, native mass spectrometry, <em>Salmonella</em>, x‐ray crystallography</p></section></section><section id="pro70260-sec-0001"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>The condensation of a carbonyl group in a sugar (e.g., glucose) with an amino group of an amino acid (e.g., Asn) followed by a rearrangement yields sugar‐amino acids [e.g., fructose‐asparagine, F‐Asn, an <em>N</em>‐substituted glycosylamine (Figure <a href="#pro70260-supitem-0001" class="usa-link">S1</a>)] (Mossine &amp; Mawhinney, <a href="#pro70260-bib-0019" class="usa-link" aria-describedby="pro70260-bib-0019">2010</a>). These non‐enzymatic products—called Amadori compounds—are varied and abundant in nature (Li et al., <a href="#pro70260-bib-0017" class="usa-link" aria-describedby="pro70260-bib-0017"><sup>2025a</sup></a><sup>, </sup><a href="#pro70260-bib-0016" class="usa-link" aria-describedby="pro70260-bib-0016"><sup>2025b</sup></a>; Luo et al., <a href="#pro70260-bib-0018" class="usa-link" aria-describedby="pro70260-bib-0018"><sup>2021</sup></a>; Mossine &amp; Mawhinney, <a href="#pro70260-bib-0019" class="usa-link" aria-describedby="pro70260-bib-0019"><sup>2010</sup></a>; Wu et al., <a href="#pro70260-bib-0032" class="usa-link" aria-describedby="pro70260-bib-0032"><sup>2018</sup></a>). A growing appreciation of the nutritional and clinical significance of Amadori compounds continues to inspire investigations into their metabolism (Li et al., <a href="#pro70260-bib-0017" class="usa-link" aria-describedby="pro70260-bib-0017"><sup>2025a</sup></a>; Luo et al., <a href="#pro70260-bib-0018" class="usa-link" aria-describedby="pro70260-bib-0018"><sup>2021</sup></a>; Mossine &amp; Mawhinney, <a href="#pro70260-bib-0019" class="usa-link" aria-describedby="pro70260-bib-0019"><sup>2010</sup></a>). Both prokaryotes and eukaryotes can assimilate Amadori compounds into central metabolic pathways (e.g., glycolysis) that generate energy and metabolic precursors (Deppe et al., <a href="#pro70260-bib-0006" class="usa-link" aria-describedby="pro70260-bib-0006">2011</a>; Van Schaftingen et al., <a href="#pro70260-bib-0029" class="usa-link" aria-describedby="pro70260-bib-0029"><sup>2012</sup></a>). Utilization of Amadori compounds involves three different strategies: oxidases (amadoriases), fructosamine‐3‐kinases, or the combined action of a kinase and deglycase (Deppe et al., <a href="#pro70260-bib-0006" class="usa-link" aria-describedby="pro70260-bib-0006">2011</a>; Van Schaftingen et al., <a href="#pro70260-bib-0029" class="usa-link" aria-describedby="pro70260-bib-0029"><sup>2012</sup></a>). In this study, we focus on the last route, which entails the use of a fructosamine 6‐kinase and a 6‐phosphofructosamine deglycase: while the kinase phosphorylates the C6 position of the sugar, the deglycase splits the phosphorylated fructosamine into glucose‐6‐phosphate (Glc‐6‐P) and the corresponding amino acid.</p>
<p>We have recently directed our attention to how <em>Salmonella enterica</em> subsp. <em>enterica</em> serovar Typhimurium (a causative agent of salmonellosis; hereafter, <em>Salmonella</em>) uses F‐Asn as a nutrient during infection of animal hosts (Ali et al., <a href="#pro70260-bib-0001" class="usa-link" aria-describedby="pro70260-bib-0001">2014</a>; Sabag‐Daigle et al., <a href="#pro70260-bib-0022" class="usa-link" aria-describedby="pro70260-bib-0022"><sup>2016</sup></a>). The <em>Salmonella fra</em> locus that is needed for F‐Asn catabolism encodes five genes: FraR (a transcriptional regulator, not discussed below), FraE, FraA, FraD, and FraB (Figure <a href="#pro70260-supitem-0001" class="usa-link">S1</a>). The catabolic route begins with the action of FraE, a periplasmic asparaginase, that deamidates F‐Asn to F‐Asp. FraA, a Dcu‐type transporter, then facilitates the movement of F‐Asp from the periplasm to the cytoplasm, where the FraD kinase phosphorylates F‐Asp to 6‐phospho‐fructose‐aspartate (6‐P‐F‐Asp) (Ali et al., <a href="#pro70260-bib-0001" class="usa-link" aria-describedby="pro70260-bib-0001">2014</a>; Sabag‐Daigle et al., <a href="#pro70260-bib-0022" class="usa-link" aria-describedby="pro70260-bib-0022"><sup>2016</sup></a>; Sabag‐Daigle et al., <a href="#pro70260-bib-0024" class="usa-link" aria-describedby="pro70260-bib-0024"><sup>2017</sup></a>). In the final step, the FraB deglycase breaks 6‐P‐F‐Asp into Glc‐6‐P and Asp. We previously purified <em>Salmonella</em> FraD and FraB, synthesized their substrates, and confirmed their activities in vitro (Biswas et al., <a href="#pro70260-bib-0002" class="usa-link" aria-describedby="pro70260-bib-0002">2017</a>; Sengupta et al., <a href="#pro70260-bib-0026" class="usa-link" aria-describedby="pro70260-bib-0026"><sup>2019</sup></a>).</p>
<p>Remarkably, deletion of <em>Salmonella</em> FraB causes a build‐up of 6‐P‐F‐Asp (its substrate) and either a bacteriostatic phenotype in vitro or a bactericidal outcome in mice (up to 1000‐fold attenuation) (Boulanger et al., <a href="#pro70260-bib-0004" class="usa-link" aria-describedby="pro70260-bib-0004">2021</a>, <a href="#pro70260-bib-0003" class="usa-link" aria-describedby="pro70260-bib-0003">2022</a>; Sabag‐Daigle et al., <a href="#pro70260-bib-0022" class="usa-link" aria-describedby="pro70260-bib-0022"><sup>2016</sup></a>). Since few species in the human microbiota can utilize F‐Asn and FraB is absent in animals (Sabag‐Daigle et al., <a href="#pro70260-bib-0025" class="usa-link" aria-describedby="pro70260-bib-0025">2018</a>), FraB has thus emerged as an attractive narrow‐spectrum drug target for treating <em>Salmonella</em>‐mediated gastroenteritis (Sabag‐Daigle et al., <a href="#pro70260-bib-0022" class="usa-link" aria-describedby="pro70260-bib-0022">2016</a>). We recently used both cell‐based and biochemical high‐throughput screening assays to identify small molecule inhibitors of FraB and pave the way for the design of potential therapeutics against non‐typhoidal <em>Salmonella</em> (Law et al., <a href="#pro70260-bib-0015" class="usa-link" aria-describedby="pro70260-bib-0015">2025</a>; Sabag‐Daigle et al., <a href="#pro70260-bib-0023" class="usa-link" aria-describedby="pro70260-bib-0023"><sup>2023</sup></a>). Because medicinal chemistry advances are expedited by structure‐based drug discovery (SBDD), we also sought to determine the structure of FraB.</p>
<p>We previously built a RosettaCM‐aided homology model of <em>Salmonella</em> FraB using as templates two bacterial phosphosugar isomerases with ~50% identity to FraB (3FKJ, <em>Salmonella</em>; 3EUA, <em>Bacillus subtilis</em>) (Sengupta et al., <a href="#pro70260-bib-0026" class="usa-link" aria-describedby="pro70260-bib-0026">2019</a>). We then docked the 6‐P‐F‐Asp substrate into this model and performed a 100‐ns molecular dynamics (MD) simulation to identify residues in the substrate‐binding pocket (Figure <a href="#pro70260-fig-0001" class="usa-link">1</a>, and Figure <a href="#pro70260-supitem-0001" class="usa-link">S2</a> for detailed interactions). Based on the inter‐atomic distances between specific side chains and the substrate functional groups (in this model) as well as a sequence alignment of bacterial FraB homologs (conservation shown by ConSurf in Figure <a href="#pro70260-supitem-0001" class="usa-link">S3</a>), we constructed several variants to test the roles of highly conserved residues in FraB substrate binding and catalysis. This work identified E214 and H230 as two invariant residues that are essential for catalysis, likely serving as general base and/or acid, respectively, during the multi‐step deglycase reaction (Sengupta et al., <a href="#pro70260-bib-0026" class="usa-link" aria-describedby="pro70260-bib-0026">2019</a>) (Figure <a href="#pro70260-supitem-0001" class="usa-link">S4</a>). While these studies identified residues essential for FraB function (Sengupta et al., <a href="#pro70260-bib-0026" class="usa-link" aria-describedby="pro70260-bib-0026">2019</a>), an atomic resolution structure of <em>Salmonella</em> FraB is essential for SBDD efforts. Here, we present the first crystal structures of FraB. Although we were unable to get a structure with bound substrate, our surprising findings on the conformational plasticity of FraB, a homodimer, have implications for potential correlated opening and closure of the two active sites for respective binding and release of substrates and ligands (including inhibitors).</p>
<figure class="fig xbox font-sm" id="pro70260-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/54dc/12359202/6b3e9e1069eb/PRO-34-e70260-g003.jpg" loading="lazy" id="jats-graphic-1" height="1174" width="669" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/pro70260-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>FraB homology model with docked 6‐F‐P‐Asp substrate (Sengupta et al., <a href="#pro70260-bib-0026" class="usa-link" aria-describedby="pro70260-bib-0026">2019</a>). (a) Full dimer, (b) Active site zoom. Subunit A, cyan; Subunit B, wheat; C‐tail residues 313–325 of Chain A, dark blue; H230* loop residues 227*–240* of Chain B, brown; Pi helix with E214 of Chain A (residues 209–216), pink; 6‐P‐F‐Asp, green bonds. The E214 to H230* distance of 10.3 Å is indicated by the dashed line.</p></figcaption></figure></section><section id="pro70260-sec-0002"><h2 class="pmc_sec_title">2. RESULTS</h2>
<p>FraB was first overexpressed in <em>Escherichia coli</em> as an N‐terminal His<sub>6</sub>‐tagged protein and purified by immobilized metal affinity chromatography. This protein, with or without the His tag removed (by using an engineered TEV cleavage site), failed to form diffraction‐quality crystals with our collection of ~15 commercial crystal screen kits. To overcome this roadblock, we explored the surface entropy reduction prediction (SERp) approach (Goldschmidt et al., <a href="#pro70260-bib-0011" class="usa-link" aria-describedby="pro70260-bib-0011">2007</a>), which seeks to promote crystal contact formation by replacing with alanines two or three consecutive, non‐conserved surface‐exposed residues predicted to have a propensity for high conformational entropy. Based on the results from the SERp server (Goldschmidt et al., <a href="#pro70260-bib-0011" class="usa-link" aria-describedby="pro70260-bib-0011">2007</a>), we constructed E17A/K18A and K275A/E276A, the two top‐scoring double mutants. Both were successfully purified and found to have activity comparable to WT′. While E17A/K18A failed to form crystals, the K275A/E276A variant yielded several types of well‐diffracting crystals, but only with the His<sub>6</sub>‐tag attached, likely due to increased solubility. Importantly, the K275/E276 residues that were mutated to alanine are distant (&gt;20 Å) from the FraB active site, have only a 2‐fold effect on the k<sub>cat</sub>/K<sub>m</sub> value compared to WT′ FraB (Table <a href="#pro70260-supitem-0001" class="usa-link">S1</a>, Figure <a href="#pro70260-supitem-0001" class="usa-link">S5</a>), and provide a surface that potentiates crystal lattice contacts for at least one of the FraB chains in each crystal form.</p>
<p>Ultimately, we obtained three different WT′ FraB crystal forms (Forms 1, 2, 3) and determined their structures to ~1.8 Å resolution. We also obtained structures of three variants of FraB WT′, two with single point‐mutations to active‐site residues (E214A and P232A) and one with the C‐terminal “tail” residues 313–325 deleted. X‐ray diffraction data and refinement statistics for the six FraB crystal structures are presented in Table <a href="#pro70260-supitem-0001" class="usa-link">S2</a>, and notable structural features of each are summarized in Table <a href="#pro70260-tbl-0001" class="usa-link">1</a>. Despite significant effort, we could not crystallize FraB in complex with either 6‐P‐F‐Asp substrate or several inhibitor compounds – our numerous attempts either failed to produce crystals or more typically yielded crystals whose electron density maps showed no hint of a bound compound. Nonetheless, the unbound FraB structures exhibit a remarkable degree of conformational plasticity in their active site regions and thus provide significant insights into the possible modes of catalysis and regulation.</p>
<section class="tw xbox font-sm" id="pro70260-tbl-0001"><h3 class="obj_head">TABLE 1.</h3>
<div class="caption p"><p>Summary of active site features of the six FraB crystal forms.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Crystal form, chain</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">E214 helix, dist. To H230* (Å)</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">H230 loop residues disordered</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">H230* loop conformation<a href="#pro70260-note-0003" class="usa-link">
<sup>a</sup>
</a>
</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">C‐tail residues disordered</th>
</tr></thead>
<tbody valign="top">
<tr><td colspan="5" align="left" valign="top" rowspan="1">Form 1</td></tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain A</td>
<td align="left" valign="top" rowspan="1" colspan="1">α‐helix, 21.5</td>
<td align="left" valign="top" rowspan="1" colspan="1">235–238</td>
<td align="left" valign="top" rowspan="1" colspan="1">1A (open, cis‐Pro232*)</td>
<td align="left" valign="top" rowspan="1" colspan="1">311–318</td>
</tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain B</td>
<td align="left" valign="top" rowspan="1" colspan="1">α‐helix, 26.9</td>
<td align="left" valign="top" rowspan="1" colspan="1">None</td>
<td align="left" valign="top" rowspan="1" colspan="1">1B (closed, cis‐Pro232*)</td>
<td align="left" valign="top" rowspan="1" colspan="1">311–325</td>
</tr>
<tr><td colspan="5" align="left" valign="top" rowspan="1">Form 2</td></tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain A</td>
<td align="left" valign="top" rowspan="1" colspan="1">α‐helix, 21.8</td>
<td align="left" valign="top" rowspan="1" colspan="1">None</td>
<td align="left" valign="top" rowspan="1" colspan="1">2A (closed, Mg, cis‐Pro232)</td>
<td align="left" valign="top" rowspan="1" colspan="1">311–316</td>
</tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain B</td>
<td align="left" valign="top" rowspan="1" colspan="1">α‐helix, 24.2</td>
<td align="left" valign="top" rowspan="1" colspan="1">None</td>
<td align="left" valign="top" rowspan="1" colspan="1">2B (closed, Mg)</td>
<td align="left" valign="top" rowspan="1" colspan="1">None</td>
</tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain C</td>
<td align="left" valign="top" rowspan="1" colspan="1">α‐helix, 21.9</td>
<td align="left" valign="top" rowspan="1" colspan="1">None</td>
<td align="left" valign="top" rowspan="1" colspan="1">2B (closed)</td>
<td align="left" valign="top" rowspan="1" colspan="1">311–315</td>
</tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain D</td>
<td align="left" valign="top" rowspan="1" colspan="1">α‐helix, 24.4</td>
<td align="left" valign="top" rowspan="1" colspan="1">None</td>
<td align="left" valign="top" rowspan="1" colspan="1">2B (closed, Mg)</td>
<td align="left" valign="top" rowspan="1" colspan="1">312–325</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Form 3</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain A</td>
<td align="left" valign="top" rowspan="1" colspan="1">α‐helix, 21.9</td>
<td align="left" valign="top" rowspan="1" colspan="1">None</td>
<td align="left" valign="top" rowspan="1" colspan="1">3A (closed)</td>
<td align="left" valign="top" rowspan="1" colspan="1">311–321</td>
</tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain B</td>
<td align="left" valign="top" rowspan="1" colspan="1">α‐helix, 21.8</td>
<td align="left" valign="top" rowspan="1" colspan="1">None</td>
<td align="left" valign="top" rowspan="1" colspan="1">3A (closed)</td>
<td align="left" valign="top" rowspan="1" colspan="1">311–323</td>
</tr>
<tr><td colspan="5" align="left" valign="top" rowspan="1">Form 4 (C‐tail deletion)</td></tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain A</td>
<td align="left" valign="top" rowspan="1" colspan="1">π‐helix, 6.9</td>
<td align="left" valign="top" rowspan="1" colspan="1">235–238</td>
<td align="left" valign="top" rowspan="1" colspan="1">1A (open, partially disordered, cis‐Pro232)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Not applicable</td>
</tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain B</td>
<td align="left" valign="top" rowspan="1" colspan="1">π‐helix, 11.6</td>
<td align="left" valign="top" rowspan="1" colspan="1">231–239</td>
<td align="left" valign="top" rowspan="1" colspan="1">Disordered</td>
<td align="left" valign="top" rowspan="1" colspan="1">Not applicable</td>
</tr>
<tr><td colspan="5" align="left" valign="top" rowspan="1">Form 5 (E214A)<a href="#pro70260-note-0004" class="usa-link">
<sup>b</sup>
</a><sup>, </sup><a href="#pro70260-note-0005" class="usa-link">
<sup>c</sup>
</a>
</td></tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain A</td>
<td align="left" valign="top" rowspan="1" colspan="1">π‐helix, 10.6</td>
<td align="left" valign="top" rowspan="1" colspan="1">None</td>
<td align="left" valign="top" rowspan="1" colspan="1">5A (closed but unique)</td>
<td align="left" valign="top" rowspan="1" colspan="1">306–325</td>
</tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain B</td>
<td align="left" valign="top" rowspan="1" colspan="1">π‐helix, 6.2</td>
<td align="left" valign="top" rowspan="1" colspan="1">228–238</td>
<td align="left" valign="top" rowspan="1" colspan="1">Disordered</td>
<td align="left" valign="top" rowspan="1" colspan="1">321–325</td>
</tr>
<tr><td colspan="5" align="left" valign="top" rowspan="1">Form 6 (P232A)</td></tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain A</td>
<td align="left" valign="top" rowspan="1" colspan="1">α‐helix, 21.2</td>
<td align="left" valign="top" rowspan="1" colspan="1">None</td>
<td align="left" valign="top" rowspan="1" colspan="1">6A (closed, similar to 5A)</td>
<td align="left" valign="top" rowspan="1" colspan="1">311–323</td>
</tr>
<tr>
<td align="left" style="padding-left:10%" valign="top" rowspan="1" colspan="1">Chain B</td>
<td align="left" valign="top" rowspan="1" colspan="1">α‐helix, 21.9</td>
<td align="left" valign="top" rowspan="1" colspan="1">232–233</td>
<td align="left" valign="top" rowspan="1" colspan="1">1B (mostly disordered)</td>
<td align="left" valign="top" rowspan="1" colspan="1">313–321</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pro70260-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="pro70260-note-0003">
<sup>
<sup>a</sup>
</sup><p class="display-inline">The entry indicates the prior Chain the H230* loop is similar to, unless it is a new conformation; the open/closed nature and presence of Mg<sup>2+</sup> are also indicated.</p>
</div>
<div class="fn" id="pro70260-note-0004">
<sup>
<sup>b</sup>
</sup><p class="display-inline">For the E214A structures, the glutamate side chain was modeled onto the alanine to estimate the distance with the H230* imidazole ring.</p>
</div>
<div class="fn" id="pro70260-note-0005">
<sup>
<sup>c</sup>
</sup><p class="display-inline">The 5B chain has a unique conformation of the C‐tail, in which it extends across and below the active site cleft to form interactions with the other subunit across the dimer.</p>
</div>
</div></section><p>Collectively, the six crystal structures reveal that FraB forms the expected C2‐symmetric dimer, with each monomer formed by two α/β subdomains with similar topological folds (Figures <a href="#pro70260-fig-0002" class="usa-link">2</a> and <a href="#pro70260-fig-0003" class="usa-link">3</a>). The putative active‐site cleft is formed at the bottom portion of the dimer interface, with the catalytic residues E214 and H230* projecting from opposing subunits (* indicates residue from the second protomer). Two such active sites are formed on each FraB dimer—one at the front and one at the back as viewed in Figure <a href="#pro70260-fig-0003" class="usa-link">3</a>. Interestingly, all six crystal structures have the FraB dimer within the asymmetric unit (two dimers in Form 2), as opposed to being along a crystallographic 2‐fold, suggesting an inherent asymmetry. Furthermore, all structures show significant conformational differences between the two subunits of the dimer, a finding that suggests coupled asymmetry with implications for the catalytic mechanism.</p>
<figure class="fig xbox font-sm" id="pro70260-fig-0002"><h3 class="obj_head">FIGURE 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/54dc/12359202/7749719597b6/PRO-34-e70260-g004.jpg" loading="lazy" id="jats-graphic-3" height="992" width="669" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/pro70260-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Alignment of 14 FraB dimers. (a) The 14 chains are all overlaid on Chain 1A of Form 1 as C‐α traces. Notice that the upper sub‐domains all align closely, but the lower sub‐domains that contain the active sites are more variable. (b) Close‐up of the active site alignment shows three variable segments (H230* loop, E214 helix, C‐tail). The figures were prepared with COOT (Emsley &amp; Cowtan, <a href="#pro70260-bib-0008" class="usa-link" aria-describedby="pro70260-bib-0008">2004</a>).</p></figcaption></figure><figure class="fig xbox font-sm" id="pro70260-fig-0003"><h3 class="obj_head">FIGURE 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/54dc/12359202/d26a63e885d6/PRO-34-e70260-g005.jpg" loading="lazy" id="jats-graphic-5" height="931" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/pro70260-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structure of FraB Crystal Form 1 showing variable regions. (a, b) Front and back views of the Form 1 dimer with active site residues labeled. In this and in all structural figures, Chain A is cyan and Chain B is wheat. (c) Overlay of the two active sites of Form 1 dimer showing variable regions, including the open and closed states of the H230* loop, the presence or absence of the C‐tail (present, Chain A; disordered, Chain B), and the &gt;20 Å distance between E214 and H230*. The last two residues of Chain A, E324 and Y325 bind in the putative active site cleft to interact with H230* and K133, respectively. H230* also interacts with E85 of the opposing chain, and E214 forms an ion pair with R310.</p></figcaption></figure><p>While the overall structures of the seven FraB dimers are all highly similar to one another, aligning to rmsd values ranging from 0.45 to 1.1 Å for all Cα atoms (Figure <a href="#pro70260-fig-0002" class="usa-link">2a</a>), they exhibit significant and intriguing conformational differences at three regions of the active site (Figure <a href="#pro70260-fig-0002" class="usa-link">2b</a>): (Mossine &amp; Mawhinney, <a href="#pro70260-bib-0019" class="usa-link" aria-describedby="pro70260-bib-0019">2010</a>) the helix containing E214, (Li et al., <a href="#pro70260-bib-0017" class="usa-link" aria-describedby="pro70260-bib-0017"><sup>2025a</sup></a>) the loop containing H230, and (Luo et al., <a href="#pro70260-bib-0018" class="usa-link" aria-describedby="pro70260-bib-0018">2021</a>) the C‐terminal residues 313–325, which we termed the “C‐tail”. In the following sections, we describe the range of conformations observed at each of these regions and their implications for function. Most notably, in all structures of WT′ FraB, the side chains of the two catalytic residues E214 (general base) and H230 (general acid) are unexpectedly ~22 Å away from one another (Figure <a href="#pro70260-fig-0003" class="usa-link">3c</a>). However, in the E214A and C‐terminus‐truncated structures, conformational changes bring these two residues much closer together (Figure <a href="#pro70260-fig-0004" class="usa-link">4</a>). In addition, in many of the structures, the last two residues of the C‐tail, E324 and Y325, insert into the active site cleft right where the 6‐P‐F‐Asp substrate is predicted to bind (Figures <a href="#pro70260-fig-0003" class="usa-link">3c</a> and <a href="#pro70260-fig-0005" class="usa-link">5</a>), suggesting a possible role for the C‐terminus in auto‐inhibition.</p>
<figure class="fig xbox font-sm" id="pro70260-fig-0004"><h3 class="obj_head">FIGURE 4.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/54dc/12359202/5bd210f697a2/PRO-34-e70260-g001.jpg" loading="lazy" id="jats-graphic-7" height="958" width="669" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/pro70260-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Conformational change in the Form 4 (C‐terminal deletion) and Form 5 (E214A) structures brings E214 closer to H230* across the dimer interface. (a) Form 4 C‐terminal deletion structure (Chain B, wheat; Chain A, cyan). Due to the Pi conformation of the last two turns of the E214 helix, the E214 side chain now faces H230*, across the dimer interface. The H230* loop of Chain A (cyan) is in the open conformation, such that the E214–H230* distance is still 12.0 Å. (b) Form 5 E214A structure. E214 of Chain B (left, wheat) is present as alanine in the structure, but the E214 side chain is grafted on for the figure to show its approximate position. With the opposing H230* loop of Chain A in a closed conformation, E214 and H230* are now only 6.7 Å from one another, as indicated by the dashed line. The C‐tail of Chain B, shown with M320 as the last ordered residue, now angles across and below the active site to wrap over the H230 loop of the opposing subunit. The side chains of K133, R310, and Gly30‐Gly31 of the helix N‐dipole are shown for reference.</p></figcaption></figure><figure class="fig xbox font-sm" id="pro70260-fig-0005"><h3 class="obj_head">FIGURE 5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/54dc/12359202/6ccda716861e/PRO-34-e70260-g006.jpg" loading="lazy" id="jats-graphic-9" height="1135" width="669" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/pro70260-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Conformational variability of the closed state of the H230* loop. (a) Close‐up of the Form 1 AB active site showing the full sequences of the C‐tail (cyan) and H230* loop (wheat). The ion pair between the C‐terminal carboxylate of Y325 and K133 is shown at the top left. The HGP sequence of the H230* loop has P232* in the <em>cis</em> configuration. Residues R317, Y319, and W321 of the C‐tail contact residues of the opposing H230* loop. (b) Close‐up of the Form 2 AB active site shows the H230* loop in a closed conformation with bound Mg<sup>2+</sup> ion (putative) in magenta and P232* in the <em>trans</em> configuration. Three backbone C=O groups of the H230* loop coordinate the Mg<sup>2+</sup> with distances ranging from 2.0 to 2.3 Å.</p></figcaption></figure><section id="pro70260-sec-0003"><h3 class="pmc_sec_title">2.1. The E214 helix</h3>
<p>As mentioned above, a surprising feature of all three structures of WT′ FraB (Forms 1–3) is that the two key catalytic residues, E214 and H230*, are ~22 Å apart from one another (Figure <a href="#pro70260-fig-0003" class="usa-link">3c</a>), a distance that is inconsistent with the proposed catalytic mechanism (Figure <a href="#pro70260-supitem-0001" class="usa-link">S4</a>). Upon inspection, we observed that this unexpected separation is due to an interesting conformational change that occurs in the helix from which E214 projects. In the FraB homology model (Figure <a href="#pro70260-fig-0001" class="usa-link">1</a>), and in the structures of the homologs on which it was based (3EUA and 3FKJ), the last two turns of the E214‐containing helix adopt the unusual Pi configuration with (i, i + 5) hydrogen bonding instead of the normal (i, i + 4) pattern of the α‐helix. This shift in helix registry places the catalytic E214 on the side of the helix that faces the dimer interface and, importantly, the catalytic histidine (H230* in FraB) from the other protomer. The corresponding catalytic Glu‐His* distances in these structures range from 7 to 10 Å. Notably, the same Pi helix configuration is also present in structures of the more distantly related FrlB deglycase (2A3N and 8GLR), which catalyzes the conversion of 6‐phospho‐fructose‐lysine to Glc‐6‐P and lysine (Kovvali et al., <a href="#pro70260-bib-0014" class="usa-link" aria-describedby="pro70260-bib-0014">2023</a>).</p>
<p>In our structures of WT′ FraB, however, the E214 helix adopts the regular α‐helix conformation with (i, i + 4) hydrogen bonding (Figure <a href="#pro70260-fig-0003" class="usa-link">3c</a>). This feature places E214 on the side of the helix that faces away from the dimer interface and the H230* residue. In this orientation, E214 forms an ion pair with R310, the last residue before the flexible C‐tail, to form an interaction that likely stabilizes this apparently inactive conformation of FraB. Remarkably, in our structures of the E214A or C‐tail deletion FraB variants, the E214‐containing helix switches back to adopt the Pi helix conformation seen for the FraB homologs, placing E214 back on the side of the helix facing H230* (Figure <a href="#pro70260-fig-0004" class="usa-link">4</a>). Coupled to this change, the helix itself moves closer to H230*, a movement that is more exaggerated in the E214A variant than in the C‐terminal truncation. Together, these changes bring E214 and H230* to within 7 Å of one another (Figure <a href="#pro70260-fig-0004" class="usa-link">4b</a>), a distance that seems well suited for catalysis.</p></section><section id="pro70260-sec-0004"><h3 class="pmc_sec_title">2.2. The H230* loop</h3>
<p>Residues 227*–240* of the loop containing the catalytic H230* residue exhibit an even higher degree of structural plasticity than the E214 helix, ranging from completely disordered in two of the FraB chains (Chains 4B and 5B) to adopting at least five distinct conformations in the remaining 12 chains (Figure <a href="#pro70260-fig-0002" class="usa-link">2b</a> and Table <a href="#pro70260-tbl-0001" class="usa-link">1</a>). The ordered states range from an “open” conformation in chains 1A and 4A to variants of a closed conformation in the others (see Figure <a href="#pro70260-fig-0003" class="usa-link">3c</a> for comparison of open and closed). Intriguingly, P232*, a highly conserved residue of the H230* loop (Figure <a href="#pro70260-supitem-0001" class="usa-link">S3</a>), adopts a <em>cis</em> conformation in four of the chains (1A, 1B, 2A, 4A) (shown in Figure <a href="#pro70260-fig-0005" class="usa-link">5a</a> for Chain 1B), including both that have the “open” conformation (1A and 4A), or the more common <em>trans</em> conformation in the remaining chains (shown in Figure <a href="#pro70260-fig-0005" class="usa-link">5b</a> for Chain 2B). Mutation of Pro232* to Ala (in the SER variant K275A/E276A) diminished catalytic efficiency by 70‐fold (Table <a href="#pro70260-supitem-0001" class="usa-link">S1</a> and Figure <a href="#pro70260-supitem-0001" class="usa-link">S5</a>), and mutation of the adjacent G231* to Ala resulted in no detectable activity (Table <a href="#pro70260-supitem-0001" class="usa-link">S1</a>). The 229*–FHGPFE–234* sequence encompassing H230* is highly conserved in FraB homologs (Figure <a href="#pro70260-supitem-0001" class="usa-link">S3</a>), suggesting that the precise folding or interactions of these loop residues are critical for activity. To determine structures with 6‐P‐F‐Asp substrate, we crystallized the P232A variant and determined its structure to 1.48 Å resolution, the highest resolution of all crystal forms, although there was no density for bound substrate. The H230* loop in this structure (Form 6) adopted either a closed conformation in Chain A or was mostly disordered in Chain B.</p>
<p>In all closed conformations, the H230* residue itself is in a similar position, forming an ion pair with E85 across the dimer interface and with E324 when the C‐tail of the opposing chain is ordered. In the open conformations (Figure <a href="#pro70260-fig-0003" class="usa-link">3c</a>), the H230* side chain moves by ~6 Å outward, away from E324, but still in contact with E85. In a previous study, we reported that the E85A variant of FraB exhibited a <em>k</em>
<sub>
<em>ca</em>t</sub>/<em>K</em>
<sub>
<em>m</em>
</sub> that was ~75‐fold lower than WT largely driven by an increased <em>K</em>
<sub>
<em>m</em>
</sub> (Sengupta et al., <a href="#pro70260-bib-0026" class="usa-link" aria-describedby="pro70260-bib-0026">2019</a>). The new structures are consistent with our earlier postulate that H‐bonding between E85 and H230* is critical to anchor the latter and permit its function as a general acid (Sengupta et al., <a href="#pro70260-bib-0026" class="usa-link" aria-describedby="pro70260-bib-0026">2019</a>). Also notable is the finding that the three phenylalanine side chains of the H230* loop can bind in significantly different positions, depending on the overall loop conformation. Such variability is unusual for large apolar residues that are typically part of a hydrophobic core.</p>
<p>Interestingly, in three of the four chains of the Form 2 structure, residues 227*–232* of the H230* loop coordinate what appears to be a divalent metal ion, modeled as Mg<sup>2+</sup>, via three backbone carbonyl groups that all point inwards in a planar configuration (Figure <a href="#pro70260-fig-0005" class="usa-link">5b</a>). This observation is particularly intriguing because Mg<sup>2+</sup> (or other divalent) was not added to the Form 2 crystallization mixture but was present at 100–200 mM for the Form 1 and Form 5 crystals, where no such density was observed. The Mg<sup>2+</sup>‐bound conformation is not likely to be relevant to catalysis, however, as addition of 16 mM EDTA to the enzymatic reaction mixture (lacking MgCl<sub>2</sub>) had no effect on catalysis (data not shown).</p>
<p>Without a structure of FraB with bound substrate, it is difficult to predict which conformation of the H230* loop is most likely to be relevant to catalysis. The closed conformations seen in most of the FraB chains are closest to the conformations of the equivalent loops seen for FraB homologs, albeit in the absence of bound substrate. As the FraB deglycase reaction has different steps (Figure <a href="#pro70260-supitem-0001" class="usa-link">S4</a>), the H230* loop could adopt multiple conformations during catalysis, some of which may be reflected in our structures.</p></section><section id="pro70260-sec-0005"><h3 class="pmc_sec_title">2.3. The C‐terminal tail</h3>
<p>For three of the 14 FraB chains (1B, 2D, 5A), the interpretable electron density ends near R310, and there is no density for C‐tail residues 311–325, which are presumably disordered. For the remaining 11 FraB chains, at least the last two residues of the C‐tail, E324 and Y325, are ordered, and in all cases, they are bound right where the 6‐P‐F‐Asp is predicted to bind in our homology model (compare Figures <a href="#pro70260-fig-0003" class="usa-link">3c</a> and <a href="#pro70260-fig-0005" class="usa-link">5</a>, with Figures <a href="#pro70260-fig-0001" class="usa-link">1</a> and <a href="#pro70260-supitem-0001" class="usa-link">S2</a>). The C‐terminal carboxylate of Y325 forms a close ion pair with K133, and the carboxylate of E324 forms an ion pair with H230* of the opposing subunit (Figures <a href="#pro70260-fig-0003" class="usa-link">3c</a> and <a href="#pro70260-fig-0005" class="usa-link">5</a>). By contrast, in our docked homology model (Figure <a href="#pro70260-supitem-0001" class="usa-link">S2</a>), K133 is predicted to bind to the carboxylate of the Asp portion of 6‐P‐F‐Asp, and the C‐terminal carboxylate of Y325 is predicted to bind to the phosphate. We constructed and purified FraB K133A in the context of the SER variant K275A/E276A and found that its <em>k</em>
<sub>
<em>cat</em>
</sub>/<em>K</em>
<sub>
<em>m</em>
</sub> was reduced by 234‐fold compared to WT′ (Table <a href="#pro70260-supitem-0001" class="usa-link">S1</a> and Figure <a href="#pro70260-supitem-0001" class="usa-link">S5</a>). Whether the interaction of K133 with the C‐terminus or with the substrate is important for activity, or both, remains to be determined.</p>
<p>The negative charges of the two C‐tail carboxylate groups, as well as the backbone carbonyl of E324, are also bound at the N‐end of the helix formed by G30 and G31 to form additional complementary electrostatic interactions (Figure <a href="#pro70260-fig-0003" class="usa-link">3c</a>). This feature is notable because in enzymes more distantly related to FraB for which structures with bound substrate are available, including 6SVM (Ruegenberg et al., <a href="#pro70260-bib-0021" class="usa-link" aria-describedby="pro70260-bib-0021">2020</a>) and 4AMV (Durand et al., <a href="#pro70260-bib-0007" class="usa-link" aria-describedby="pro70260-bib-0007">2008</a>; Mouilleron et al., <a href="#pro70260-bib-0020" class="usa-link" aria-describedby="pro70260-bib-0020"><sup>2006</sup></a>), similar N‐dipole interactions are formed with substrate. The two glycine residues are highly conserved (see Gly30‐Gly31 for FraB in Figure <a href="#pro70260-supitem-0001" class="usa-link">S3</a>) and predicted to bind to 6‐P‐F‐Asp in our docked homology model of FraB (Figure <a href="#pro70260-supitem-0001" class="usa-link">S2</a>). The FraB substrate 6‐P‐F‐Asp has two carboxylate groups, like the C‐terminal Glu‐Tyr pair, such that the latter could mimic the bound substrate.</p>
<p>In six of the FraB chains (1A, 2A, 2B, 2C, 3A, 6B), additional residues of the C‐tail are ordered, as an extended segment that forms additional interactions with the residues of the H230* loop of the opposing subunit (Figure <a href="#pro70260-fig-0005" class="usa-link">5</a>). The conformation of the full C‐tail is best seen in Chain B of Form 2, in which all C‐tail residues are ordered. In particular, the side chains of W321 and Y319 pack against residues 232*–236* of the opposing H230* loop, and R317 forms a potential electrostatic interaction with D237* (Figure <a href="#pro70260-fig-0005" class="usa-link">5</a>). For the other structures, the C‐tail residues that are observed overlay closely with the corresponding residues of Chain 2B, suggesting that the C‐tail in WT′ FraB has a single binding mode that is dynamically moving in and out of position, in contrast with the multiple and more dynamic conformations of the H230* loop.</p>
<p>In the E214A structure (Form 5), the C‐tail of Chain B adopts a unique conformation among the crystal structures that does not plug into the active site but rather angles across and below it to form a distinct set of interactions with the opposing subunit (Figure <a href="#pro70260-fig-0004" class="usa-link">4b</a>). This C‐tail conformation, for which the last ordered residue is M320, is closer to that observed in our FraB homology model (Figure <a href="#pro70260-fig-0001" class="usa-link">1</a>) and in structures of FraB homologs (3FKJ, 3EUA, 2A3N, and others). Thus, the C‐tail conformation observed for E214A could be closer to its conformation with bound substrate during catalysis, presumably closing over the substrate to form interactions such as those predicted in our docked homology model (Figure <a href="#pro70260-supitem-0001" class="usa-link">S2</a>).</p>
<p>In many of the FraB structures, the tip of the C‐tail plugs into the active site to overlap with the predicted position of substrate. It is therefore conceivable that the C‐tail has an auto‐inhibitory role, which in turn could be regulated by post‐translational modifications, interactions with other proteins, or allosteric effectors. To test this idea, the catalytic parameters of the C‐terminally truncated FraB protein consisting of residues 1–312 were measured. We observed that the FraB<sup>1–312</sup> (Δ313–325) protein exhibited a close to normal <em>k</em>
<sub>cat</sub> of &gt;40 min<sup>−1</sup> at 20 mM 6‐P‐F‐Asp (Table <a href="#pro70260-supitem-0001" class="usa-link">S1</a>), but a dramatically elevated <em>K</em>
<sub>
<em>m</em>
</sub> reflecting weakened affinity for substrate upon deletion of the C‐tail. However, we were unable to reliably determine the Michaelis–Menten parameters as substrate concentrations &gt;20 mM led to inhibition under the conditions used. Regardless, this variant is clearly quite capable of bond breakage but suffers from weak substrate binding, consistent with the interactions between the C‐tail and docked substrate predicted in Figure <a href="#pro70260-supitem-0001" class="usa-link">S2</a>.</p>
<p>To better understand these effects on FraB function, we also analyzed the oligomeric state of the FraB<sup>1–312</sup> protein by native mass spectrometry and confirmed that it forms homodimers, but with a higher proportion of monomer than full‐length FraB (Figure <a href="#pro70260-fig-0006" class="usa-link">6a,b</a>). This observation suggests that the inter‐subunit interactions observed for the C‐tail in the crystal structures could be involved in stabilizing the FraB homodimer. For the dimer, we also used ion mobility to compare the arrival time for the 18+ species and observed a higher abundance of an extended conformation for FraB with the C‐tail deleted (FraB<sup>1–312</sup>) than for WT′ (Figure <a href="#pro70260-fig-0006" class="usa-link">6c</a>), again suggesting a change in structure.</p>
<figure class="fig xbox font-sm" id="pro70260-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/54dc/12359202/eab4bad7fd3d/PRO-34-e70260-g002.jpg" loading="lazy" id="jats-graphic-11" height="1815" width="660" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/pro70260-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Native mass spectra for WT′ FraB (a) and the FraB C‐tail deletion variant (b). Monomer observed from FraB C‐tail deletion variant (FraB<sup>1–312</sup>, Δ313–325) shows that deletion of the C‐tail (residues 313–325) weakens the dimer interface. (c) The arrival time (extracted dimer 18+) distribution comparison for FraB WT′ and FraB C‐tail deletion. The arrival time was measured by ion mobility with a single pass on a Waters SELECT Series Cyclic IMS mass spectrometer.</p></figcaption></figure></section></section><section id="pro70260-sec-0006"><h2 class="pmc_sec_title">3. DISCUSSION</h2>
<p>Our rational surface engineering of FraB proved instrumental in mediating crystal contacts and reinforces the value of SER as a useful crystallization strategy. We discuss below the insights gained from our analysis of the six different high‐resolution FraB structures (three WT′ and three different variants).</p>
<p>First, the notable structural differences between the two subunits of the dimer (within the asymmetric unit in all six crystals) establishes FraB as a conformational heterodimer. Despite having the same primary structure, the two FraB protomers within each dimer always exhibit differences in their tertiary structures. Whether or not this asymmetry reflects non‐equivalence of the two active sites (i.e., half‐of‐sites reactivity) with potential for regulation remains to be investigated.</p>
<p>Second, a shift in registry (i.e., transitioning from a regular α‐helix to a Pi‐helix) changes the orientation of E214 to move it to a position that would enable it to act together with H230* as part of a concerted acid–base catalytic mechanism. We had previously shown that when 6‐P‐F‐Asp is docked in a FraB homology model, E214 is positioned &lt;3 Å from the C1 <em>pro</em>‐R hydrogen in 6‐P‐F‐Asp and H230* &lt; 5 Å from the C2 carbonyl oxygen (Figures <a href="#pro70260-fig-0001" class="usa-link">1b</a> and <a href="#pro70260-supitem-0001" class="usa-link">S2</a>). The essentiality of these residues was demonstrated by the findings that E214A and H230A* are inactive in vitro, and that the introduction of either mutation individually in the <em>Salmonella</em> chromosome fully abolished the pathogen's ability to grow on fructose‐asparagine (Sengupta et al., <a href="#pro70260-bib-0026" class="usa-link" aria-describedby="pro70260-bib-0026">2019</a>). In our FraB homology model, and in the structures on which it was based (3EUA and 3FKJ), E214 or its equivalent exists in the unusual Pi helix, which places it close to H230* across the dimer interface. By contrast, in all three structures of WT′ FraB, E214 exists in the more regular α‐helix, which places it on the opposite side of the helix, away from the dimer interface and H230*, and forms an ion pair with R310. This new and unexpected conformation of FraB, which is presumably an inactive conformation, is likely to be the lowest energy (ground) state of FraB, since it is observed in all three structures of WT′ FraB.</p>
<p>Third, our findings raise some key questions. If FraB catalysis depends on the formation of the short Pi‐helical segment with E214, what keeps the Pi helix from forming as the predominantly observed state in the absence of substrate? Furthermore, why would this helix restructuring to the active state occur in the E214A and C‐tail deletion variants, but not in WT′ FraB? As to the first question, it is possible that in WT′ FraB, the Pi and α‐helix conformations of the E214 helix are in conformational equilibrium with one another in solution. While it is conceivable that the crystallization conditions and crystal packing interactions favored one conformation over the other, our observation of the α‐helix for this segment in all three crystal forms of WT′ FraB (Forms 1–3) suggests that it is the lower energy conformation. Interestingly, AlphaFold 3 (AF3) predicts the Pi helix conformation for the E214 helix in WT′ FraB (in the absence of substrate), with an E214 – H230* distance of 7.5 Å (Figure <a href="#pro70260-supitem-0001" class="usa-link">S6</a>). AF3 also predicts a conformation for C‐tail residues 313–325 that mirrors our previous homology model with docked substrate (Figure <a href="#pro70260-supitem-0001" class="usa-link">S2</a>), with the C‐terminal Y325 residue removed from the active site and contacting the other subunit of the dimer. The AF3 models, unlike the crystal structures, likely reflect the structures of the most closely related proteins in the PDB.</p>
<p>Regarding the second question, in the case of E214A, loss of the E214 carboxylate would remove the ion pair with R310, which would presumably destabilize the inactive state. In the case of the C‐terminal deletion (FraB<sup>1–312</sup>), with R310 being close to the new C‐terminus at H312, increased flexibility could also destabilize the ion pair with E214. It is unclear why FraB, but not its homologs, would adopt this inactive state in the first place, although it may reflect a regulatory mechanism that is unique to FraB. Since the inactive conformation with E214 in a regular α‐helix is observed in multiple crystal forms of WT′ FraB, it is not likely to simply be caused by crystal packing interactions. None of the structures of FraB or its homologs described above contain bound substrate. Thus, while the FraB homologs would appear to be poised for catalysis in the ground state (without substrate), FraB itself appears to inherently adopt an inactive conformation. Presumably, binding of substrate to FraB triggers the conformational change to the more active state observed in the E214A and C‐terminal deletion structures, but this feature remains to be determined experimentally.</p>
<p>Fourth, significant conformational changes also occur in the H230* loop of each active site, across the dimer interface from the E214 helix. While the larger scale movements result in open and closed conformations, more subtle but significant changes occur within the different variations of the closed state, including the presence or absence of a Mg<sup>2+</sup> (or other divalent) ion bound by three backbone carbonyl groups and the <em>cis</em>‐<em>trans</em> isomerization of the Pro323 residue. The 229‐FHGPFE‐234 sequence encompassing the catalytic H230* residue is among the most conserved regions of the FraB structure (Figure <a href="#pro70260-supitem-0001" class="usa-link">S3</a>), yet the importance of most of these residues is not evident from our structures, which did not contain bound substrate.</p>
<p>Fifth, our finding that the C‐tail plugs into the active site to overlap with the 6‐P‐F‐Asp substrate motivated us to test the idea that the C‐tail plays an auto‐inhibitory role. We originally hypothesized that deletion of the C‐tail might increase FraB activity. However, as deletion of the last 13 residues left <em>k</em>
<sub>cat</sub> relatively unimpaired and increased the <em>K</em>
<sub>
<em>m</em>
</sub>, we alternatively hypothesize that the C‐tail may have a role in substrate‐binding, perhaps in closing over the substrate once it is bound and thereby shielding the active site from bulk solvent, as seen in the homology model with docked substrate (Figures <a href="#pro70260-fig-0001" class="usa-link">1b</a> and <a href="#pro70260-supitem-0001" class="usa-link">S2</a>). Given that our attempts to co‐crystallize FraB with substrate have so far failed, future experiments are needed to test this revised hypothesis.</p>
<p>Finally, we speculate on a possible basis for the inactive state being the default conformation even though it is arguably primed for catalysis. It is possible that the enzyme is triggered to act only with the <em>correct</em> substrate (6‐P‐F‐Asp). One mechanism to ensure substrate specificity is to use intramolecular contacts that entrap the enzyme in an inactive state and then switch the enzyme to an active state when the same residues involved in autoregulation are also employed for substrate binding. For example, the C‐terminal carboxylate of Y325 forms a close ion pair with K133, but Y325 was postulated to interact with the phosphate moiety in 6‐P‐F‐Asp, the FraB substrate (Figure <a href="#pro70260-supitem-0001" class="usa-link">S2</a>). Thus, in the presence of substrate, the C‐tail could then be released from its contact with K133 and thereby ensure generation of an active state. Such ideas could be tested using FRET and mass spectrometry experiments.</p></section><section id="pro70260-sec-0007"><h2 class="pmc_sec_title">4. MATERIALS AND METHODS</h2>
<section id="pro70260-sec-0008"><h3 class="pmc_sec_title">4.1. Mutagenesis, overexpression and purification of FraB
</h3>
<p>Please see the <a href="#pro70260-supitem-0001" class="usa-link">Supporting Information</a> for details.</p></section><section id="pro70260-sec-0009"><h3 class="pmc_sec_title">4.2. Crystallization and x‐ray structure determination</h3>
<p>For crystallization, purified K275A/E276A FraB protein (un‐cleaved by TEV) was thawed from concentrated aliquots frozen at −80°C and diluted to 10 mg/mL (0.263 mM monomer) in 50 mM Tris–HCl (pH 7.5), 150 mM NaCl, 1 mM DTT. For Form 5 and Form 6 crystals, a 10‐fold molar excess of 6‐P‐F‐Asp (2.63 mM) was added to 10 mg/mL FraB (E214A or P232A) during dilution. Crystal screens were performed in 96‐well hanging drop format using a Mosquito instrument. Hanging drops contained 0.2 μL of complex and 0.2 μL of cocktail solution and were hung over 100 μL of cocktail solution. Cocktail solutions were from standard commercial crystal screens from Hampton Research and Molecular Dimensions. After optimization of hits in 24‐well plates with 2 μL + 2 μL hanging drops and 1 mL of well solution, Form 1 crystals grew from solutions of 15% (v/v) 2‐propanol, 0.1 M MgCl<sub>2</sub>, 0.1 M MES (pH 6.5), and were transferred into well solution supplemented with 30% (v/v) glycerol as cryoprotectant prior to freezing. Form 2 crystals grew from solutions of 21.5% (w/v) PEG 3350, 0.4 M sodium thiocyanate, 0.1 M HEPES (pH 6.3), and were transferred into well solution supplemented with 25% (v/v) glycerol as cryoprotectant. Form 3 crystals grew from 10% to 14% (v/v) PEG 300, 0.1 M MES (pH 6.0–6.5), and were transferred into well solution supplemented to 30% (v/v) PEG 300 as cryoprotectant. Form 4 crystals (of K275A/E276A FraB<sup>1–312</sup>) grew from solutions of 20% (w/v) PEG 3350, 0.2 M sodium sulfate, 0.1 M Bis‐Tris propane (pH 8.0–8.5), and were transferred into well solution supplemented with 30% (v/v) glycerol as cryoprotectant. Form 5 crystals (of E214A/K275A/E276A FraB) grew from solutions of 18% (w/v) PEG 6000, 0.2 M MgCl<sub>2</sub>, 0.1 M MES (pH 6.0), and were transferred into well solution supplemented with 30% (v/v) glycerol as cryoprotectant. Form 6 crystals (of P232A/K275A/E276A FraB) grew from solutions of 13%–19% (v/v) PEG 300, 0.1 M MES, pH 6.5, and were transferred into well solution supplemented with 20% (v/v) glycerol as cryoprotectant. All crystals were mounted in nylon loops (Hampton Research) and plunge frozen in liquid nitrogen.</p>
<p>X‐ray diffraction data were collected at Beamline 31‐ID of the Advanced Photon Source at Argonne National Laboratory. Images were processed with an automatic workflow consisting of autoPROC version 1.1.7 (Vonrhein et al., <a href="#pro70260-bib-0030" class="usa-link" aria-describedby="pro70260-bib-0030">2011</a>), XDS (Kabsch, <a href="#pro70260-bib-0012" class="usa-link" aria-describedby="pro70260-bib-0012">2010</a>) (version Jan 26, 2018, BUILT 20180808 or later), POINTLESS version 1.11.21 or later (Evans, <a href="#pro70260-bib-0009" class="usa-link" aria-describedby="pro70260-bib-0009">2006</a>), AIMLESS version 0.7.4 or later (Evans &amp; Murshudov, <a href="#pro70260-bib-0010" class="usa-link" aria-describedby="pro70260-bib-0010">2013</a>), CCP4 version 7.0.075 or later (Winn et al., <a href="#pro70260-bib-0031" class="usa-link" aria-describedby="pro70260-bib-0031">2011</a>), and STARANISO version 2.3.28 or later (Tickle et al., <a href="#pro70260-bib-0027" class="usa-link" aria-describedby="pro70260-bib-0027">2020</a>). The final anisotropic (ellipsoidal) reduced data sets were used for structure determination and crystallographic refinement. The Form 1 structure was determined by molecular replacement with MOLREP version 11.4.03 (Vagin &amp; Teplyakov, <a href="#pro70260-bib-0028" class="usa-link" aria-describedby="pro70260-bib-0028">1997</a>) using a dimer of the most closely related putative phospho‐sugar isomerase (PDB accession code 3FKJ) as a search model. After refinement (REFMAC5 version 5.8.043 (Kovalevskiy et al., <a href="#pro70260-bib-0013" class="usa-link" aria-describedby="pro70260-bib-0013">2018</a>)), the model was adjusted, and new side chains were built using COOT (Emsley &amp; Cowtan, <a href="#pro70260-bib-0008" class="usa-link" aria-describedby="pro70260-bib-0008">2004</a>). Iterative rounds of refinement and model building yielded the final model with statistics presented in Table <a href="#pro70260-supitem-0001" class="usa-link">S2</a>. Forms 2 to 6 were determined by molecular replacement using the Form 1 structure as a starting model and re‐built and refined as described above for Form 1. Figures depicting protein structures were generated using PyMOL (Delano, <a href="#pro70260-bib-0005" class="usa-link" aria-describedby="pro70260-bib-0005">2002</a>) and COOT (Emsley &amp; Cowtan, <a href="#pro70260-bib-0008" class="usa-link" aria-describedby="pro70260-bib-0008">2004</a>).</p></section><section id="pro70260-sec-0010"><h3 class="pmc_sec_title">4.3. Enzyme assays</h3>
<p>Master mix (15 μL for the C‐tail deletion variant and 29 μL for all others), lacking 6‐P‐F‐Asp, was aliquoted into wells of a 384‐well polystyrene white μClear® microplate (Greiner Bio‐One, #781095), and incubated in an INFINITE M1000Pro (Tecan) at 37°C for 2 min prior to the addition of varying [6‐P‐F‐Asp] (15 μL for the C‐tail deletion variant and 1 μL for all others). Conversion of Glc‐6‐P and NAD<sup>+</sup> into NADH by glucose‐6‐phosphate dehydrogenase (G6PD), as a proxy for FraB activity, was monitored at 37°C by fluorescence (350 nm excitation, 450 nm emission) for a total of 5 min in 2‐s intervals. The reaction mix consisted of 25 mM HEPES (pH 8.0 @ 22°C), 5 mM MgCl<sub>2</sub>, 0.1 mM EGTA, 0.5 mM NAD<sup>+</sup>, 0.1 μM FraB, 0.033 mU/μL G6PD from <em>Leuconostoc mesenteroides</em> (Worthington Biochemicals, #<a href="https://www.ncbi.nlm.nih.gov/nuccore/LS003981" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">LS003981</a>), and varying [6‐P‐F‐Asp]. Relative fluorescence units (RFUs) were plotted against time (min) and a linear regression curve (y = mx + b) was fitted to the linear range of the data. To generate an NADH standard curve, average RFUs of 0 (water), 10, 25, 50, 75, and 100 μM NADH, from triplicate measurements, were plotted against [NADH] (μM) and a linear regression curve (y = mx + b) was fitted to the data. The RFUs/min were converted to initial velocities (μm/min) by dividing the slope (RFUs/min) of the activity assay by the slope (RFUs/μM) of the standard curve. The initial velocities (μM/min) were plotted against [6‐P‐F‐Asp] (mM) using Kaleidagraph (Version 4.5.3, Synergy Software) and the data were fit to <em>v</em>
<sub>
<em>o</em>
</sub> = (<em>V</em>
<sub>
<em>max</em>
</sub> × [S])/(<em>K</em>
<sub>
<em>m</em>
</sub> + [S]) to generate a Michaelis–Menten curve and derive the <em>k</em>
<sub>
<em>cat</em>
</sub> and <em>K</em>
<sub>
<em>m</em>
</sub> values. The goodness of fit (as evident from the correlation coefficient) was 0.99 in all cases. The curve‐fit errors are provided in Table <a href="#pro70260-supitem-0001" class="usa-link">S1</a>.</p></section><section id="pro70260-sec-0011"><h3 class="pmc_sec_title">4.4. Native mass spectrometry</h3>
<p>FraB WT′ and C‐tail deletion samples for native mass spectrometry (nMS) analysis were buffer exchanged into 200 mM ammonium acetate (AmAc) (Sigma‐Aldrich) using micro Bio‐spin P6 spin columns (BioRad). Samples were introduced into the MS instrument by nano‐electrospray ionization, which uses glass capillary tips pulled in‐house using a P‐97 micropipette puller (Sutter Instruments). Samples were diluted to 3 μM (monomer) in AmAc; 3–5 μL of each sample was loaded into the tips. nMS experiments were performed on a Waters SELECT Series Cyclic IMS mass spectrometer. All experiments were operated under ion mobility (IM) mode using the following tuning setting: Cone V: 30 V; IM separation traveling wave (TW) static height and TW velocity: 25 V and 375 m/s. Drift time was measured in the mobility cell with a single pass. nMS and IM data were processed using MassLynx 4.2 (Waters).</p></section></section><section id="pro70260-sec-0015"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Katerina Zakharova:</strong> Writing – review and editing; investigation; methodology. <strong>Jamison D. Law:</strong> Writing – review and editing; investigation; methodology. <strong>Yuan Gao:</strong> Investigation; writing – review and editing; methodology. <strong>Sravya Kovvali:</strong> Investigation; writing – review and editing; methodology. <strong>Vicki H. Wysocki:</strong> Investigation; writing – review and editing; methodology; funding acquisition; supervision; conceptualization. <strong>Venkat Gopalan:</strong> Writing – original draft; conceptualization; investigation; methodology; funding acquisition; supervision. <strong>Charles E. Bell:</strong> Conceptualization; investigation; funding acquisition; writing – original draft; methodology; validation; formal analysis; project administration; data curation; supervision.</p></section><section id="sec13"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pro70260-supitem-0001"><div class="caption p"><p>
<strong>Data S1:</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12359202/bin/PRO-34-e70260-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70260-s001.pdf</a><sup> (888.8KB, pdf) </sup>
</div></div></section></section><section id="pro70260-sec-0012" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This work was funded by grants from the National Institutes of Health (R01AI140541 to V.G., V.H.W., and C.E.B.; P41GM128577 and RM1GM149374 to V.H.W.; T32GM141955 to J.D.L), the Pelotonia Research Foundation (to J.D.L.) and the Behrman Research Fund (to V.G.). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under Contract No. DE‐AC02‐06CH11357. Use of the Lilly Research Laboratories Collaborative Access Team (LRL‐CAT) beamline at Sector 31 of the Advanced Photon Source was provided by Eli Lilly Company, which operates the facility.</p></section><section id="notes1"><p>

Zakharova K, Law JD, Gao Y, Kovvali S, Wysocki VH, Gopalan V, et al. Crystal structures of <em>Salmonella enterica</em>
FraB deglycase reveal a conformational heterodimer with remarkable structural plasticity at the active site. Protein Science. 2025;34(9):e70260. 10.1002/pro.70260

</p>
<section id="pro70260-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="pro70260-note-0001"><p>Katerina Zakharova and Jamison D. Law contributed equally to this work.</p></div>
<div class="fn p" id="pro70260-note-0002"><p>
<strong>Review Editor:</strong> Aitziber L. Cortajarena</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Venkat Gopalan, Email: gopalan.5@osu.edu.</p>
<p>Charles E. Bell, Email: bell.489@osu.edu.</p></section><section id="pro70260-sec-0014"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The data that support the findings of this study are openly available in Protein Data Bank at <a href="https://www.rcsb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org/</a>, reference number 9OTJ, 9OTL, 9OU5, 9OTR, 9OTU, 9OU6.</p></section><section id="pro70260-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="pro70260-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="pro70260-bib-0001">
<cite>
Ali MM, Newsom DL, González JF, Sabag‐Daigle A, Stahl C, Steidley B, et al. Fructose‐asparagine is a primary nutrient during growth of Salmonella in the inflamed intestine. PLoS Pathog. 2014;10:e1004209.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1004209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4072780/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24967579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog&amp;title=Fructose%E2%80%90asparagine%20is%20a%20primary%20nutrient%20during%20growth%20of%20Salmonella%20in%20the%20inflamed%20intestine&amp;author=MM%20Ali&amp;author=DL%20Newsom&amp;author=JF%20Gonz%C3%A1lez&amp;author=A%20Sabag%E2%80%90Daigle&amp;author=C%20Stahl&amp;volume=10&amp;publication_year=2014&amp;pages=e1004209&amp;pmid=24967579&amp;doi=10.1371/journal.ppat.1004209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0002">
<cite>
Biswas PK, Behrman EJ, Gopalan V. Characterization of a Salmonella sugar kinase essential for the utilization of fructose‐asparagine. Biochem Cell Biol. 2017;95:304–309.
</cite> [<a href="https://doi.org/10.1139/bcb-2016-0138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28177776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Cell%20Biol&amp;title=Characterization%20of%20a%20Salmonella%20sugar%20kinase%20essential%20for%20the%20utilization%20of%20fructose%E2%80%90asparagine&amp;author=PK%20Biswas&amp;author=EJ%20Behrman&amp;author=V%20Gopalan&amp;volume=95&amp;publication_year=2017&amp;pages=304-309&amp;pmid=28177776&amp;doi=10.1139/bcb-2016-0138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0003">
<cite>
Boulanger EF, Sabag‐Daigle A, Baniasad M, Kokkinias K, Schwieters A, Wrighton KC, et al. Sugar‐phosphate toxicities attenuate Salmonella fitness in the gut. J Bacteriol. 2022;204:e0034422.
</cite> [<a href="https://doi.org/10.1128/jb.00344-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9765134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36383008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Sugar%E2%80%90phosphate%20toxicities%20attenuate%20Salmonella%20fitness%20in%20the%20gut&amp;author=EF%20Boulanger&amp;author=A%20Sabag%E2%80%90Daigle&amp;author=M%20Baniasad&amp;author=K%20Kokkinias&amp;author=A%20Schwieters&amp;volume=204&amp;publication_year=2022&amp;pages=e0034422&amp;pmid=36383008&amp;doi=10.1128/jb.00344-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0004">
<cite>
Boulanger EF, Sabag‐Daigle A, Thirugnanasambantham P, Gopalan V, Ahmer BMM. Sugar‐phosphate toxicities. Microbiol Mol Biol Rev. 2021;85:e0012321.
</cite> [<a href="https://doi.org/10.1128/MMBR.00123-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8483676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34585982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Mol%20Biol%20Rev&amp;title=Sugar%E2%80%90phosphate%20toxicities&amp;author=EF%20Boulanger&amp;author=A%20Sabag%E2%80%90Daigle&amp;author=P%20Thirugnanasambantham&amp;author=V%20Gopalan&amp;author=BMM%20Ahmer&amp;volume=85&amp;publication_year=2021&amp;pages=e0012321&amp;pmid=34585982&amp;doi=10.1128/MMBR.00123-21&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0005">
<cite>
Delano WL. The PyMOL molecular graphics system. New York: Schrödinger, LLC; 2002.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20PyMOL%20molecular%20graphics%20system&amp;author=WL%20Delano&amp;publication_year=2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0006">
<cite>
Deppe VM, Bongaerts J, O'Connell T, Maurer KH, Meinhardt F. Enzymatic deglycation of Amadori products in bacteria: mechanisms, occurrence and physiological functions. Appl Microbiol Biotechnol. 2011;90:399–406.
</cite> [<a href="https://doi.org/10.1007/s00253-010-3083-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21347729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Microbiol%20Biotechnol&amp;title=Enzymatic%20deglycation%20of%20Amadori%20products%20in%20bacteria:%20mechanisms,%20occurrence%20and%20physiological%20functions&amp;author=VM%20Deppe&amp;author=J%20Bongaerts&amp;author=T%20O'Connell&amp;author=KH%20Maurer&amp;author=F%20Meinhardt&amp;volume=90&amp;publication_year=2011&amp;pages=399-406&amp;pmid=21347729&amp;doi=10.1007/s00253-010-3083-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0007">
<cite>
Durand P, Golinelli‐Pimpaneau B, Mouilleron S, Badet B, Batet‐Denisot MA. Highlights of glucosamine‐6P synthase catalysis. Arch Biochem Biophys. 2008;474:302–317.
</cite> [<a href="https://doi.org/10.1016/j.abb.2008.01.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18279655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&amp;title=Highlights%20of%20glucosamine%E2%80%906P%20synthase%20catalysis&amp;author=P%20Durand&amp;author=B%20Golinelli%E2%80%90Pimpaneau&amp;author=S%20Mouilleron&amp;author=B%20Badet&amp;author=MA%20Batet%E2%80%90Denisot&amp;volume=474&amp;publication_year=2008&amp;pages=302-317&amp;pmid=18279655&amp;doi=10.1016/j.abb.2008.01.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0008">
<cite>
Emsley P, Cowtan K. Coot: model‐building tools for molecular graphics. Acta Crystallogr. 2004;D60:2126–2132.</cite> [<a href="https://doi.org/10.1107/S0907444904019158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15572765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr&amp;title=Coot:%20model%E2%80%90building%20tools%20for%20molecular%20graphics&amp;author=P%20Emsley&amp;author=K%20Cowtan&amp;volume=D60&amp;publication_year=2004&amp;pages=2126-2132&amp;pmid=15572765&amp;doi=10.1107/S0907444904019158&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0009">
<cite>
Evans PR. Scaling and assessment of data quality. Acta Crystallogr. 2006;D62:72–82.</cite> [<a href="https://doi.org/10.1107/S0907444905036693" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16369096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr&amp;title=Scaling%20and%20assessment%20of%20data%20quality&amp;author=PR%20Evans&amp;volume=D62&amp;publication_year=2006&amp;pages=72-82&amp;pmid=16369096&amp;doi=10.1107/S0907444905036693&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0010">
<cite>
Evans PR, Murshudov GN. How good are my data and what is the resolution?
Acta Crystallogr. 2013;D69:1204–1214.</cite> [<a href="https://doi.org/10.1107/S0907444913000061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3689523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23793146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr&amp;title=How%20good%20are%20my%20data%20and%20what%20is%20the%20resolution?&amp;author=PR%20Evans&amp;author=GN%20Murshudov&amp;volume=D69&amp;publication_year=2013&amp;pages=1204-1214&amp;pmid=23793146&amp;doi=10.1107/S0907444913000061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0011">
<cite>
Goldschmidt L, Cooper DR, Derewenda ZS, Eisenberg D. Toward rational protein crystallization: a web server for the design of crystallizable protein variants. Protein Sci. 2007;16:1569–1576.
</cite> [<a href="https://doi.org/10.1110/ps.072914007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2203352/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17656576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Toward%20rational%20protein%20crystallization:%20a%20web%20server%20for%20the%20design%20of%20crystallizable%20protein%20variants&amp;author=L%20Goldschmidt&amp;author=DR%20Cooper&amp;author=ZS%20Derewenda&amp;author=D%20Eisenberg&amp;volume=16&amp;publication_year=2007&amp;pages=1569-1576&amp;pmid=17656576&amp;doi=10.1110/ps.072914007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0012">
<cite>
Kabsch W. XDS. Acta Crystallogr. 2010;D66:125–132.</cite> [<a href="https://doi.org/10.1107/S0907444909047337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2815665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20124692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr&amp;title=XDS&amp;author=W%20Kabsch&amp;volume=D66&amp;publication_year=2010&amp;pages=125-132&amp;pmid=20124692&amp;doi=10.1107/S0907444909047337&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0013">
<cite>
Kovalevskiy O, Nicholls RA, Long F, Murshudov GN. Overview of refinement procedures within REFMAC5: utilizing data from different sources. Acta Crystallogr. 2018;D74:492–505.</cite> [<a href="https://doi.org/10.1107/S2059798318000979" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5947762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29533229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr&amp;title=Overview%20of%20refinement%20procedures%20within%20REFMAC5:%20utilizing%20data%20from%20different%20sources&amp;author=O%20Kovalevskiy&amp;author=RA%20Nicholls&amp;author=F%20Long&amp;author=GN%20Murshudov&amp;volume=D74&amp;publication_year=2018&amp;pages=492-505&amp;pmid=29533229&amp;doi=10.1107/S2059798318000979&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0014">
<cite>
Kovvali S, Gao Y, Cool A, Lindert S, Wysocki VH, Bell CE, et al. Insights into the catalytic mechanism of a bacterial deglycase essential for utilization of fructose‐lysine. Protein Sci. 2023;32:e4695.
</cite> [<a href="https://doi.org/10.1002/pro.4695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10285752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37289023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Insights%20into%20the%20catalytic%20mechanism%20of%20a%20bacterial%20deglycase%20essential%20for%20utilization%20of%20fructose%E2%80%90lysine&amp;author=S%20Kovvali&amp;author=Y%20Gao&amp;author=A%20Cool&amp;author=S%20Lindert&amp;author=VH%20Wysocki&amp;volume=32&amp;publication_year=2023&amp;pages=e4695&amp;pmid=37289023&amp;doi=10.1002/pro.4695&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0015">
<cite>
Law JD, Gao Y, Kovvali S, Thirugnanasambantham P, Wysocki VH, Ahmer BMM, et al. Identification of inhibitors of the Salmonella FraB deglycase, a drug target. FEBS Open Bio. 2025;15:773–792.</cite> [<a href="https://doi.org/10.1002/2211-5463.70001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12051030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39945759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Open%20Bio&amp;title=Identification%20of%20inhibitors%20of%20the%20Salmonella%20FraB%20deglycase,%20a%20drug%20target&amp;author=JD%20Law&amp;author=Y%20Gao&amp;author=S%20Kovvali&amp;author=P%20Thirugnanasambantham&amp;author=VH%20Wysocki&amp;volume=15&amp;publication_year=2025&amp;pages=773-792&amp;pmid=39945759&amp;doi=10.1002/2211-5463.70001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0016">
<cite>
Li T, Yang C, Zhang L. Novel comprehensive perspective on Amadori compounds: preparation, multiple roles and interactions with other compounds. Crit Rev Food Sci Nutr. 2025b;23:1–24.</cite> [<a href="https://doi.org/10.1080/10408398.2025.2494059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40269616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Food%20Sci%20Nutr&amp;title=Novel%20comprehensive%20perspective%20on%20Amadori%20compounds:%20preparation,%20multiple%20roles%20and%20interactions%20with%20other%20compounds&amp;author=T%20Li&amp;author=C%20Yang&amp;author=L%20Zhang&amp;volume=23&amp;publication_year=2025b&amp;pages=1-24&amp;pmid=40269616&amp;doi=10.1080/10408398.2025.2494059&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0017">
<cite>
Li TT, Yang C, Zhang LF. Novel insight into Amadori compounds: fate of Amadori compounds in food supply chain. Compr Rev Food Sci Food Saf. 2025a;24:e70149.
</cite> [<a href="https://doi.org/10.1111/1541-4337.70149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40091644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Compr%20Rev%20Food%20Sci%20Food%20Saf&amp;title=Novel%20insight%20into%20Amadori%20compounds:%20fate%20of%20Amadori%20compounds%20in%20food%20supply%20chain&amp;author=TT%20Li&amp;author=C%20Yang&amp;author=LF%20Zhang&amp;volume=24&amp;publication_year=2025a&amp;pages=e70149&amp;pmid=40091644&amp;doi=10.1111/1541-4337.70149&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0018">
<cite>
Luo Y, Li SM, Ho CT. Key aspects of Amadori rearrangement products as future food additives. Molecules. 2021;26:4314.
</cite> [<a href="https://doi.org/10.3390/molecules26144314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8303902/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34299589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Key%20aspects%20of%20Amadori%20rearrangement%20products%20as%20future%20food%20additives&amp;author=Y%20Luo&amp;author=SM%20Li&amp;author=CT%20Ho&amp;volume=26&amp;publication_year=2021&amp;pages=4314&amp;pmid=34299589&amp;doi=10.3390/molecules26144314&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0019">
<cite>
Mossine VV, Mawhinney TP. 1‐Amino‐1‐deoxy‐D‐fructose (“fructosamine”) and its derivatives. Adv Carbohydr Chem Biochem. 2010;64:291–402.
</cite> [<a href="https://doi.org/10.1016/S0065-2318(10)64006-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20837201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Carbohydr%20Chem%20Biochem&amp;title=1%E2%80%90Amino%E2%80%901%E2%80%90deoxy%E2%80%90D%E2%80%90fructose%20(%E2%80%9Cfructosamine%E2%80%9D)%20and%20its%20derivatives&amp;author=VV%20Mossine&amp;author=TP%20Mawhinney&amp;volume=64&amp;publication_year=2010&amp;pages=291-402&amp;pmid=20837201&amp;doi=10.1016/S0065-2318(10)64006-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0020">
<cite>
Mouilleron S, Badet‐Denisot M‐A, Golinelli‐Pimpaneau B. Glutamine binding opens the ammonia channel and activates glucosamine‐6P synthase. J Biol Chem. 2006;281:4404–4412.
</cite> [<a href="https://doi.org/10.1074/jbc.M511689200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16339762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Glutamine%20binding%20opens%20the%20ammonia%20channel%20and%20activates%20glucosamine%E2%80%906P%20synthase&amp;author=S%20Mouilleron&amp;author=M%E2%80%90A%20Badet%E2%80%90Denisot&amp;author=B%20Golinelli%E2%80%90Pimpaneau&amp;volume=281&amp;publication_year=2006&amp;pages=4404-4412&amp;pmid=16339762&amp;doi=10.1074/jbc.M511689200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0021">
<cite>
Ruegenberg S, Horn M, Pichlo C, Allmeroth K, Baumann U, Denzel MS. Loss of GFAT‐1 feedback regulation activates the hexosamine pathway that modulates protein homeostasis. Nature Commun. 2020;11:687.
</cite> [<a href="https://doi.org/10.1038/s41467-020-14524-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7000685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32019926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Commun&amp;title=Loss%20of%20GFAT%E2%80%901%20feedback%20regulation%20activates%20the%20hexosamine%20pathway%20that%20modulates%20protein%20homeostasis&amp;author=S%20Ruegenberg&amp;author=M%20Horn&amp;author=C%20Pichlo&amp;author=K%20Allmeroth&amp;author=U%20Baumann&amp;volume=11&amp;publication_year=2020&amp;pages=687&amp;pmid=32019926&amp;doi=10.1038/s41467-020-14524-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0022">
<cite>
Sabag‐Daigle A, Blunk HM, Sengupta A, Wu J, Bogard AJ, Ali MM, et al. A metabolic intermediate of the fructose‐asparagine utilization pathway inhibits growth of a Salmonella FraB mutant. Sci Rep. 2016;6:28117.
</cite> [<a href="https://doi.org/10.1038/srep28117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4941530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27403719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=A%20metabolic%20intermediate%20of%20the%20fructose%E2%80%90asparagine%20utilization%20pathway%20inhibits%20growth%20of%20a%20Salmonella%20FraB%20mutant&amp;author=A%20Sabag%E2%80%90Daigle&amp;author=HM%20Blunk&amp;author=A%20Sengupta&amp;author=J%20Wu&amp;author=AJ%20Bogard&amp;volume=6&amp;publication_year=2016&amp;pages=28117&amp;pmid=27403719&amp;doi=10.1038/srep28117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0023">
<cite>
Sabag‐Daigle A, Boulanger EF, Thirugnanasambantham P, Law JD, Bogard AJ, Behrman EJ, et al. Identification of small‐molecule inhibitors of the <em>Salmonella</em> FraB deglycase using a live‐cell assay. Microbiol Spectr. 2023;11:e0460622.
</cite> [<a href="https://doi.org/10.1128/spectrum.04606-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10100877/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36809033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Spectr&amp;title=Identification%20of%20small%E2%80%90molecule%20inhibitors%20of%20the%20Salmonella%20FraB%20deglycase%20using%20a%20live%E2%80%90cell%20assay&amp;author=A%20Sabag%E2%80%90Daigle&amp;author=EF%20Boulanger&amp;author=P%20Thirugnanasambantham&amp;author=JD%20Law&amp;author=AJ%20Bogard&amp;volume=11&amp;publication_year=2023&amp;pages=e0460622&amp;pmid=36809033&amp;doi=10.1128/spectrum.04606-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0024">
<cite>
Sabag‐Daigle A, Sengupta A, Blunk HM, Biswas PK, Cron MC, Bogard AJ, et al. Salmonella FraE, an asparaginase homolog, contributes to fructose‐asparagine but not asparagine utilization. J Bacteriol. 2017;199:e00330−17.
</cite> [<a href="https://doi.org/10.1128/JB.00330-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5648864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28847920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Bacteriol&amp;title=Salmonella%20FraE,%20an%20asparaginase%20homolog,%20contributes%20to%20fructose%E2%80%90asparagine%20but%20not%20asparagine%20utilization&amp;author=A%20Sabag%E2%80%90Daigle&amp;author=A%20Sengupta&amp;author=HM%20Blunk&amp;author=PK%20Biswas&amp;author=MC%20Cron&amp;volume=199&amp;publication_year=2017&amp;pages=e00330%E2%88%9217&amp;pmid=28847920&amp;doi=10.1128/JB.00330-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0025">
<cite>
Sabag‐Daigle A, Wu J, Borton MA, Sengupta A, Gopalan V, Wrighton KC, et al. Identification of bacterial species that can utilize fructose‐asparagine. Appl Environ Microbiol. 2018;84:e01957‐17.
</cite> [<a href="https://doi.org/10.1128/AEM.01957-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5812934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29269489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl%20Environ%20Microbiol&amp;title=Identification%20of%20bacterial%20species%20that%20can%20utilize%20fructose%E2%80%90asparagine&amp;author=A%20Sabag%E2%80%90Daigle&amp;author=J%20Wu&amp;author=MA%20Borton&amp;author=A%20Sengupta&amp;author=V%20Gopalan&amp;volume=84&amp;publication_year=2018&amp;pages=e01957%E2%80%9017&amp;pmid=29269489&amp;doi=10.1128/AEM.01957-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0026">
<cite>
Sengupta A, Wu J, Seffernick JT, Sabag‐Daigle A, Thomsen N, Chen TH, et al. Integrated use of biochemical, native mass spectrometry, computational, and genome‐editing methods to elucidate the mechanism of a Salmonella deglycase. J Mol Biol. 2019;431:4497–4513.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2019.08.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6885551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31493410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Integrated%20use%20of%20biochemical,%20native%20mass%20spectrometry,%20computational,%20and%20genome%E2%80%90editing%20methods%20to%20elucidate%20the%20mechanism%20of%20a%20Salmonella%20deglycase&amp;author=A%20Sengupta&amp;author=J%20Wu&amp;author=JT%20Seffernick&amp;author=A%20Sabag%E2%80%90Daigle&amp;author=N%20Thomsen&amp;volume=431&amp;publication_year=2019&amp;pages=4497-4513&amp;pmid=31493410&amp;doi=10.1016/j.jmb.2019.08.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0027">
<cite>
Tickle IJ, Flensburg C, Keller P, Paciorek W, Sharff A, Vonrhein C, et al. STARANISO. Cambridge, United Kingdom: Global Phasing Ltd; 2020.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=STARANISO&amp;author=IJ%20Tickle&amp;author=C%20Flensburg&amp;author=P%20Keller&amp;author=W%20Paciorek&amp;author=A%20Sharff&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0028">
<cite>
Vagin A, Teplyakov A. MOLREP: an automated program for molecular replacement. J Appl Cyrst. 1997;30:1022–1025.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Cyrst&amp;title=MOLREP:%20an%20automated%20program%20for%20molecular%20replacement&amp;author=A%20Vagin&amp;author=A%20Teplyakov&amp;volume=30&amp;publication_year=1997&amp;pages=1022-1025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0029">
<cite>
Van Schaftingen E, Collard F, Wiame E, Veiga‐da‐Cunha M. Enzymatic repair of Amadori products. Amino Acids. 2012;42:1143–1150.
</cite> [<a href="https://doi.org/10.1007/s00726-010-0780-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20967558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Amino%20Acids&amp;title=Enzymatic%20repair%20of%20Amadori%20products&amp;author=E%20Van%20Schaftingen&amp;author=F%20Collard&amp;author=E%20Wiame&amp;author=M%20Veiga%E2%80%90da%E2%80%90Cunha&amp;volume=42&amp;publication_year=2012&amp;pages=1143-1150&amp;pmid=20967558&amp;doi=10.1007/s00726-010-0780-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0030">
<cite>
Vonrhein C, Flensburg C, Keller P, Sharff A, Smart O, Paciorek W, et al. Data processing and analysis with the autoPROC toolbox. Acta Crystallogr. 2011;D67:293–302.</cite> [<a href="https://doi.org/10.1107/S0907444911007773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3069744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21460447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr&amp;title=Data%20processing%20and%20analysis%20with%20the%20autoPROC%20toolbox&amp;author=C%20Vonrhein&amp;author=C%20Flensburg&amp;author=P%20Keller&amp;author=A%20Sharff&amp;author=O%20Smart&amp;volume=D67&amp;publication_year=2011&amp;pages=293-302&amp;pmid=21460447&amp;doi=10.1107/S0907444911007773&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0031">
<cite>
Winn MD, Ballard CC, Cowtan KD, Dodson EJ, Emsley P, Evans PR, et al. Overview of the CCP4 suite and current developments. Acta Crystallogr. 2011;D67:235–242.</cite> [<a href="https://doi.org/10.1107/S0907444910045749" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3069738/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21460441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr&amp;title=Overview%20of%20the%20CCP4%20suite%20and%20current%20developments&amp;author=MD%20Winn&amp;author=CC%20Ballard&amp;author=KD%20Cowtan&amp;author=EJ%20Dodson&amp;author=P%20Emsley&amp;volume=D67&amp;publication_year=2011&amp;pages=235-242&amp;pmid=21460441&amp;doi=10.1107/S0907444910045749&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70260-bib-0032">
<cite>
Wu J, Sabag‐Daigle A, Metz TO, Deatherage Kaiser BL, Gopalan V, Behrman EJ, et al. Measurement of fructose‐asparagine concentrations in human and animal foods. J Agric Food Chem. 2018;66:212–217.
</cite> [<a href="https://doi.org/10.1021/acs.jafc.7b04237" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6191295/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29232127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Agric%20Food%20Chem&amp;title=Measurement%20of%20fructose%E2%80%90asparagine%20concentrations%20in%20human%20and%20animal%20foods&amp;author=J%20Wu&amp;author=A%20Sabag%E2%80%90Daigle&amp;author=TO%20Metz&amp;author=BL%20Deatherage%20Kaiser&amp;author=V%20Gopalan&amp;volume=66&amp;publication_year=2018&amp;pages=212-217&amp;pmid=29232127&amp;doi=10.1021/acs.jafc.7b04237&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Data S1:</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12359202/bin/PRO-34-e70260-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70260-s001.pdf</a><sup> (888.8KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are openly available in Protein Data Bank at <a href="https://www.rcsb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org/</a>, reference number 9OTJ, 9OTL, 9OU5, 9OTR, 9OTU, 9OU6.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Protein Science : A Publication of the Protein Society are provided here courtesy of <strong>The Protein Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/pro.70260"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/PRO-34-e70260.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12359202/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12359202/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359202%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359202/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12359202/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12359202/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40823914/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12359202/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40823914/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12359202/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12359202/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="BsQbobXYrxrOikO5TmtIdQqtF7CYbY7MSvSOnxWat2GFcRa4BMJzPmQ80VBvNqZ3">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
