
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Unlocking the antimalarial potential of novel steroid-tetraoxane hybrids through consensus molecular docking and molecular dynamics investigation - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152546AB8AF32E130546AB005234C03B.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365323/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Unlocking the antimalarial potential of novel steroid-tetraoxane hybrids through consensus molecular docking and molecular dynamics investigation">
<meta name="citation_author" content="Dipankar Nath">
<meta name="citation_author_institution" content="Department of Pharmaceutical Sciences, Faculty of Science and Engineering, Dibrugarh University, Dibrugarh, 786004 Assam India">
<meta name="citation_author" content="Abhijit Debnath">
<meta name="citation_author_institution" content="Noida Institute of Engineering and Technology [Pharmacy Institute], 19 Knowledge Park-II, Institutional Area, Greater Noida, Uttar Pradesh India">
<meta name="citation_author" content="Malita Sarma">
<meta name="citation_author_institution" content="Department of Pharmaceutical Sciences, Faculty of Science and Engineering, Dibrugarh University, Dibrugarh, 786004 Assam India">
<meta name="citation_author" content="Rajesh Kumar Singh">
<meta name="citation_author_institution" content="Department of Dravyaguna, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India">
<meta name="citation_author" content="Dipak Chetia">
<meta name="citation_author_institution" content="Department of Pharmaceutical Sciences, Faculty of Science and Engineering, Dibrugarh University, Dibrugarh, 786004 Assam India">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30389">
<meta name="citation_doi" content="10.1038/s41598-025-13017-z">
<meta name="citation_pmid" content="40830358">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365323/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365323/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365323/pdf/41598_2025_Article_13017.pdf">
<meta name="description" content="The emergence of Plasmodium falciparum resistance to artemisinin-based combination therapies necessitates the development of novel antimalarial agents. This study presents the first computational investigation of steroid-tetraoxane hybrids targeting ...">
<meta name="og:title" content="Unlocking the antimalarial potential of novel steroid-tetraoxane hybrids through consensus molecular docking and molecular dynamics investigation">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The emergence of Plasmodium falciparum resistance to artemisinin-based combination therapies necessitates the development of novel antimalarial agents. This study presents the first computational investigation of steroid-tetraoxane hybrids targeting ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365323/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365323">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-13017-z"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_13017.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365323%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365323/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365323/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365323/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 19;15:30389. doi: <a href="https://doi.org/10.1038/s41598-025-13017-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-13017-z</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Unlocking the antimalarial potential of novel steroid-tetraoxane hybrids through consensus molecular docking and molecular dynamics investigation</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nath%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Dipankar Nath</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Dipankar Nath</span></h3>
<div class="p">
<sup>1</sup>Department of Pharmaceutical Sciences, Faculty of Science and Engineering, Dibrugarh University, Dibrugarh, 786004 Assam India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nath%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dipankar Nath</span></a>
</div>
</div>
<sup>1,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Debnath%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Abhijit Debnath</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Abhijit Debnath</span></h3>
<div class="p">
<sup>2</sup>Noida Institute of Engineering and Technology [Pharmacy Institute], 19 Knowledge Park-II, Institutional Area, Greater Noida, Uttar Pradesh India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Debnath%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Abhijit Debnath</span></a>
</div>
</div>
<sup>2,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sarma%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Malita Sarma</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Malita Sarma</span></h3>
<div class="p">
<sup>1</sup>Department of Pharmaceutical Sciences, Faculty of Science and Engineering, Dibrugarh University, Dibrugarh, 786004 Assam India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sarma%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Malita Sarma</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20RK%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Rajesh Kumar Singh</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Rajesh Kumar Singh</span></h3>
<div class="p">
<sup>3</sup>Department of Dravyaguna, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20RK%22%5BAuthor%5D" class="usa-link"><span class="name western">Rajesh Kumar Singh</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chetia%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Dipak Chetia</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Dipak Chetia</span></h3>
<div class="p">
<sup>1</sup>Department of Pharmaceutical Sciences, Faculty of Science and Engineering, Dibrugarh University, Dibrugarh, 786004 Assam India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chetia%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dipak Chetia</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Pharmaceutical Sciences, Faculty of Science and Engineering, Dibrugarh University, Dibrugarh, 786004 Assam India </div>
<div id="Aff2">
<sup>2</sup>Noida Institute of Engineering and Technology [Pharmacy Institute], 19 Knowledge Park-II, Institutional Area, Greater Noida, Uttar Pradesh India </div>
<div id="Aff3">
<sup>3</sup>Department of Dravyaguna, Institute of Medical Sciences, Banaras Hindu University, Varanasi, India </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 18; Accepted 2025 Jul 21; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365323  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830358/" class="usa-link">40830358</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The emergence of <em>Plasmodium falciparum</em> resistance to artemisinin-based combination therapies necessitates the development of novel antimalarial agents. This study presents the first computational investigation of steroid-tetraoxane hybrids targeting cyclophilin, a key protein implicated in artemisinin resistance mechanisms. We designed a library of 127 steroid-1,2,4,5-tetraoxane hybrid compounds combining steroidal sapogenin (∆<sup>5,(6)</sup>-diosgenin-3-one) and gem-dihydroperoxides, and employed consensus molecular docking across eight platforms to minimize algorithm-specific biases. Compound A-CY-9C emerged as the lead candidate, exhibiting superior binding stability and a favorable free energy landscape during 500 ns molecular dynamics simulations. The dual pharmacophore mechanism—disrupting parasite cholesterol uptake via the steroid component while inducing oxidative stress through the tetraoxane moiety—offers a novel strategy to combat artemisinin resistance. This first-in-class approach to targeting cyclophilin with steroid-tetraoxane hybrids provides a promising foundation for developing next-generation antimalarials against resistant <em>P. falciparum</em> strains.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-13017-z.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Steroid-tetraoxane hybrids, Artemisinin resistance, Cyclophilin inhibition, Molecular docking, Molecular dynamics, Antimalarial drug discovery</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Cheminformatics, Combinatorial libraries, Computational biology and bioinformatics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par10">Malaria has significantly influenced human history and genetics. It is estimated that the disease, transmitted by mosquitoes and caused by several <em>Plasmodium</em> protozoan species, killed approximately 300 million people throughout the 20th century<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. According to the latest World Malaria Report published by WHO, there was a slight increase in malaria cases in recent years, while deaths decreased—an estimated 249 million malaria cases and 608,000 deaths in 2022, compared to 247 million cases and 619,000 deaths in 2021, and 245 million cases and 625,000 deaths in 2020. The main countries contributing to the increase were Pakistan, Ethiopia, Nigeria, Uganda, and Papua New Guinea<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Antimalarial drugs play a critical role in tackling the global malaria burden to control and eliminate the disease<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. <em>Plasmodium falciparum</em> and <em>Plasmodium vivax</em> predominantly cause malaria infection globally. Artemisinin-based combination therapy (ACT) is now considered the gold standard for treating uncomplicated falciparum malaria in nearly all areas<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. ACT combines a semisynthetic derivative of artemisinin (ART) and a longer-acting drug; ART involves rapid parasite killing to reduce initial parasite load, whereas the longer-acting drug maintains a plasma concentration above the minimum parasiticidal concentration<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par11">The emergence of resistant strains against ACTs has become increasingly concerning, with reports from Cambodia, Southeast Asia, and Africa. Recent surveillance data indicate that artemisinin resistance has spread to over 40 countries, with some regions reporting treatment failure rates exceeding 10%. The resistance is primarily due to mutations in <em>Pf</em>K13 and other genes, raising an urgent need to find new antimalarial drugs<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. A crucial mechanism underlying ART resistance is the overexpression of <em>Pf</em> Cyclophilin 19B in <em>Pf</em>Kelch13, which is the main cytosolic cyclophilin protein of <em>P. falciparum</em><sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Cyclophilins represent a family of proteins that play multiple essential roles in <em>P. falciparum</em> biology. Their primary functions include: Protein folding, regulation, and transport, crucial for parasite survival and pathogenicity<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Peptidyl-prolyl cis-trans isomerase (PPIase) activity is essential for maintaining the structural integrity of various proteins, molecular chaperone activity prevents protein aggregation under stress conditions, and interactions with heat shock proteins (HSPs), such as HSP70, are mainly involved in protein folding and stabilization.</p>
<p id="Par12">Given their crucial role in parasite survival, cyclophilins represent promising targets for developing novel antimalarial drugs to combat resistant parasites<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. The “hybridization” approach emerges as a potential game-changer in this context, as this strategy has proven successful against other diseases such as cancer, AIDS, and tuberculosis<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. It represents a rational chemistry-based approach of combining two or more pharmacophores in a single molecule with different modes of biological action (double-edged sword)<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. The combination of steroid and tetraoxane moieties presents a particularly promising direction for targeting cyclophilin-mediated resistance. Steroids can effectively hijack the parasite’s cholesterol uptake mechanisms, while tetraoxanes provide potent antimalarial activity through endoperoxide-mediated oxidative stress. When these pharmacophores are combined, they could potentially disrupt both cyclophilin function and parasite metabolism simultaneously. The rationale behind this dual-targeting approach is particularly compelling. The steroid component can exploit the parasite’s dependence on host cholesterol, potentially interfering with membrane formation and cellular development. Meanwhile, the tetraoxane moiety generates reactive oxygen species through iron-mediated activation, leading to oxidative stress and parasite death. This combination could be especially effective against resistant strains, as targeting multiple essential pathways simultaneously reduces the likelihood of resistance development. Furthermore, the steroid scaffold could enhance the drug’s bioavailability and cellular penetration, while the tetraoxane component provides rapid parasiticidal activity. This dual-targeting strategy is particularly relevant for overcoming resistance mechanisms, as mutations affecting one target may not compromise the overall efficacy of the hybrid molecule. Previous studies have demonstrated the success of hybrid approaches in antimalarial drug development, with several promising examples in the literature showing enhanced potency against resistant strains compared to their individual components. Mixed steroidal 1,2,4,5-tetroxane hybrid molecules synthesized via gem-hydroperoxides and evaluated for in vitro antimalarial activity were found to be more potent than mefloquine, ART, multiple times more potent than artelinic acid, and 2.4 times more potent than arteether against chloroquine-resistant <em>P. falciparum</em> W2 clone<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Steroid-trioxane hybrids were synthesized, which have shown promising activity against multidrug-resistant <em>P. yoelii</em> in Swiss mice<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Oxalic acid salt of dihydroartemisinin (DHA)- aminoquinoline hybrid molecules were synthesized and evaluated for in vitro antiplasmodial activity against chloroquine-sensitive (CQS) D10 and chloroquine-resistant (CQR) Dd2 strains of <em>Plasmodium falciparum</em>. Synthesized hybrids have shown better activity than Chloroquine in the CQR strain<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Hybrid molecules containing Primaquine-Artemisinin were designed and synthesized, and their in vitro and in-vivo activity was evaluated in the liver and blood stages. Compounds have shown superior efficacy in vitro against <em>Plasmodium berghei</em> in the liver stage and comparatively as potent as ART in <em>P.falciparum</em> culture. Moreover, one of the compounds has shown better efficacy than the equimolar mixture of the parent compound against <em>P. berghei</em> induced infection<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Quinolone-pyrimidine hybrids were synthesized and tested against (CQS) D10 and (CQR) Dd2 strains of <em>P.falciparum</em> and have shown better potency against the CQR-resistant strain<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. A series of 4-aminoquinoline-piperazine-pyrimidine hybrids were synthesized and were found to have 2.5 fold stronger antimalarial activity against the CQR strain of <em>P.falciparum</em> than chloroquine<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Deoxycholic acid-based 1,2,4-trioxolanes and 1,2,4,5-tetraoxanes were synthesized, and in vitro antimalarial activity was assessed against the (CQS) T96 and (CQR) K1 strains of <em>P.falciparum.</em> Tetraoxanes have shown more promising activity than trioxolanes<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Stigmastane steroids, viz. 6-hydroxystigmast-4-en 3-one, stigmast-4-en-3-one, and 3-hydroxystigmast 5-en-7-one were isolated from the stem bark of <em>Dryobalanops oblongifolia</em> and evaluated for antiplasmodial activity, where 3-hydroxystigmast 5-en-7-one has shown comparatively better activity<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p>
<p id="Par13">Diosgenin (25R-spirost-5-en-3β-ol), a steroidal saponin found in fenugreek and yam (Dioscorea spp)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. It is considered to be effective against malaria<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Diosgenone[(25R)-4-Spirosten-3-one] and its derivative were reported to have good antimalarial activity<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Steroids are gaining considerable attention in antimalarial drug discovery as they can hijack cholesterol uptake in the malarial parasite, which is an important nutrient source for parasite growth and development<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. ART and its related endoperoxides are the mainstay in the ACT. Structural Activity Relationship (SAR) studies on ART and its derivatives revealed that the lactone ring is not essential for antimalarial activity, and further studies also confirm that the 1,2,4-trioxane (endoperoxide) moiety is the key pharmacophore essential for antimalarial activity. Several synthetic endoperoxide scaffolds, such as 1,2,4-trioxane, 1,2,4-trioxolane, and 1,2,4,5-tetraoxane derivatives and their hybrid analogues have gained significant interest and are in different clinical stages of drug discovery and development<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>.</p>
<p id="Par14">Drug research has recently attracted increased attention due to computational drug design techniques. These methods are usually simple, inexpensive, and potentiate the discovery process. Docking is one of the computer technologies most frequently used in structure-based drug design. Predictions of ligand binding affinities and thorough details on ligand interactions with amino acid residues in target binding pockets are provided by it<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. In silico designing, docking, and interaction studies of novel fused quinazoline were carried out against Topoisomerase II, and results were promising for future development<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>.In silico evaluation of novel 2-Pyrazoline carboxamide derivative was carried out against Falcipain-2, Falcipain-3, and Plasmepsin-2, and has shown strong binding affinities and interactions, and provides a foundation for further study and experimental validation<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>.</p>
<p id="Par15">∆<sup>5,(6)</sup>-diosgenin-3-one, an isomer of diosgenone, can be obtained naturally from <em>Dioscorea nipponica</em> and also synthetically from diosgenin. It is used for the synthesis of new cephalostatin analogues and other steroidal compounds of pharmaceutical importance<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. The present approach mainly focusses on rationally designed library of steroid-1,2,4,5-tetraoxane hybrid compounds by combining steroidal sapogenin i.e. ∆<sup>5,(6)</sup>-diosgenin-3-one[(25R)-5-en-spirosta-3-one] and gem-dihydroperoxides followed by consensus docking targeting Cyclophilin (PDB id: 1QNG) and simulation study to screen out the best possible compounds for further study.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Experimental methodology</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Preparation of compound library</h3>
<p id="Par16">A compound library of 127 hybrid compounds was designed (Fig. <a href="#Fig1" class="usa-link">1</a>) by systematically combining the steroidal sapogenin ∆<sup>5,(6)</sup>-diosgenin-3-one[(25R)-5-en-spirosta-3-one] with gem-dihydroperoxides (cyclic ketones/esters were converted to their corresponding gem-dihydroperoxides). The library was constructed based on established medicinal chemistry principles, focusing on maintaining the core steroid scaffold while introducing structural diversity through variations in the tetraoxane moiety. The compounds were designed manually using ChemDraw 20.1 software<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, with careful consideration of synthetic feasibility. Each compound was assigned a unique identifier, and its structure was saved in both MOL file and.sdf format for subsequent computational analyses. The complete library is provided in Supplementary File 1, which includes the chemical structures, SMILES notations, and molecular properties of all 127 compounds.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/daf356f8524c/41598_2025_13017_Fig1_HTML.jpg" loading="lazy" id="d33e482" height="333" width="735" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Compound library design.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Prediction of drug score</h3>
<p id="Par18">In-silico drug score assessment was computed for all designed compounds using the OSIRIS DataWarrior tool (version 5.5.0)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. This computational platform employs fragment-based predictive algorithms to evaluate potential toxicological liabilities, including mutagenicity, tumorigenicity, reproductive effects, and irritant properties. The drug score calculation integrates toxicity risk assessments with physicochemical properties (cLogP, solubility, molecular weight, and drug-likeness) to generate a single value between 0 and 1, where higher scores indicate more favorable overall drug-like characteristics.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Quantitative estimate of drug-likeness (QED) scores</h3>
<p id="Par19">The quantitative estimate of drug-likeness (QED) scores<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> were calculated for all compounds using the ADMETlab 2.0<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. QED provides a weighted composite of eight molecular properties (molecular weight, octanol-water partition coefficient, topological polar surface area, number of hydrogen bond donors and acceptors, number of rotatable bonds, aromatic rings, and structural alerts) that correlate with the likelihood of a compound exhibiting favorable drug-like characteristics. The QED score ranges from 0 to 1, with values closer to 1 representing compounds with properties more similar to approved drugs. All compounds were processed using the weighted QED model, which applies different weights to each molecular descriptor based on their relative importance in determining drug-likeness. Compounds with QED scores falling in this range were considered for further molecular docking studies.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Consensus molecular Docking</h3>
<p id="Par20">The compound library was prepared for molecular docking through comprehensive preprocessing steps. All compounds underwent energy minimization and received appropriate charges and hydrogen bonds using OpenBabel<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> and UCSF Chimera’s Dock Prep module<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. For protein preparation, the crystal structures were cleaned by removing water molecules, heteroatoms, and cofactors, followed by the addition of polar hydrogen atoms and charge calculations. The docking grid box was centered (center_x = 13.2309, center_y = 47.3337, center_z = -16.4818 and size_x = 30, size_y = 30, size_z = 30) on the active sites (ARG62, GLN70, ASN109, PHE120, TRP128, HIS133) of the Cyclophilin (PDB id: 1QNG)<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. To overcome potential system-specific biases and enhance the reliability of docking predictions, we employed a consensus docking approach using eight different docking programs: Vina<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> LeDock<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> Q-Vina2<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, Auto Dock Tools 4.2<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, i-dock<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> and Smina<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> by following the same methodology followed in our previous studies<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Each program was configured with its optimal parameters while maintaining consistent grid box dimensions across all platforms. The docking results were analyzed based on: Binding energy scores, Interaction patterns with key residues, Conformational clustering, and Consensus ranking across different programs. The compounds were ranked according to their performance across all docking platforms, with special attention given to consistently high-scoring compounds. Radar plot was developed by using python packages matplotlib 3.10.3 (<a href="https://matplotlib.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://matplotlib.org</a>)<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> with the help of pandas 2.3.0 (<a href="https://pandas.pydata.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pandas.pydata.org</a>)<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>and numpy 2.3.1 (<a href="https://numpy.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://numpy.org</a>)<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>.</p>
<p id="Par21">This comprehensive approach helped identify the most promising candidates while minimizing platform-specific biases. The consensus scores were calculated by normalizing the binding energies from each program and combining them into a unified ranking system. This method provided a more robust evaluation of the compounds’ binding potential compared to single program docking approaches. The top-performing compounds from this consensus analysis were selected for further investigation through molecular dynamics simulations to evaluate their stability and interaction dynamics under physiological conditions.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Molecular dynamics simulation</h3>
<p id="Par22">To investigate the conformational dynamics and stability of protein-ligand complexes, we performed all-atom molecular dynamics simulations using GROMACS 2022.4 for 500 ns<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. The simulation protocol followed our established methodology<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> employing the CHARMM36 all-atom force field (July 2022)<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> for protein and CGenFF-derived parameters<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> for ligands. The system was solvated using the TIP3P water model under periodic boundary conditions<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>–<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Energy minimization using the steepest descent algorithm with a maximum force threshold of 1000 kJ/mol/nm. NVT equilibration for 100 ps at 300 K using the V-rescale thermostat (τ = 0.1 ps). NPT equilibration for 100 ps at 300 K and 1 bar using the Parrinello-Rahman barostat (τ = 2.0 ps). Production MD for 500 ns with a time step of 2 fs. LINCS algorithm for constraining all bonds involving hydrogen atoms. Particle Mesh Ewald (PME) method for long-range electrostatics with a cutoff of 1.2 nm. Van der Waals interactions were truncated at 1.2 nm with a smooth switching function from 1.0 nm. Coordinates were saved every 10 ps for analysis. All simulations were carried out using the PARAM Shivay high-performance computing system. Post simulation, we analyzed key parameters including RMSD, RMSF, Rg, SASA, PCA, FEL, and hydrogen bond interactions using GROMACS tools and MDAnalysis<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. 2D contour maps and 3D surface representations were generated using Python packages: with matplotlib 3.10.3 (<a href="https://matplotlib.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://matplotlib.org</a>), numpy 2.3.1 (<a href="https://numpy.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://numpy.org</a>), SciPy 1.16.0 (<a href="https://scipy.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scipy.org</a>), and Axes3D of mpl_toolkits.mplot3d under Matplotlib 3.7.5 (<a href="https://matplotlib.org/stable/api/toolkits/mplot3d.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://matplotlib.org/stable/api/toolkits/mplot3d.html</a>) for 3D visualization. To understand the ligand movement during simulation, superimposed snapshots of the cyclophilin-a-cy-9c complex at five time points were developed by using the Python packages Biopython<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> and py3dmol implemented with 3DMol.js<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. These analyses provided comprehensive insights into structural stability, conformational flexibility, and protein-ligand interaction dynamics under physiologically relevant conditions.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Molecular interaction analysis and structural activity relationship</h3>
<p id="Par23">Top performing compounds found after MD have been taken for Molecular Interaction Analysis by using Prolif<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup> and visualization was done by NGL viewer<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> to understand molecular structural insights, a Structural Activity relationship has been conducted.</p></section><section id="Sec9"><h3 class="pmc_sec_title">ADME and toxicity</h3>
<p id="Par24">The absorption, distribution, metabolism, and excretion (ADME) properties of compound A-CY-9C were predicted using the SwissADME web tool<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Parameters evaluated included gastrointestinal (GI) absorption, blood-brain barrier (BBB) permeability, and inhibition potential against five major cytochrome P450 isoforms (CYP1A2, CYP2C19, CYP2C9, CYP2D6, and CYP3A4). Toxicity assessment was conducted using two complementary in-silico platforms: DataWarrior<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> and ProTox-3.0<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. DataWarrior was employed to predict mutagenicity, reproductive toxicity, and irritant potential, while ProTox-3.0 was used to determine the toxicity classification according to the Globally Harmonized System.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Preparation of compound library</h3>
<p id="Par25">A compound library containing 127 numbers of steroid-tetraoxane hybrids has been constructed and shared in Supplementary File 1.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Prediction of drug score</h3>
<p id="Par26">Out of 127 hybrid compounds, 112 compounds were found suitable after toxicity screening (mutagenic, reproductive effective, irritant), and 39 compounds have shown comparatively better drug scores. A list of all the molecules is shown in Supplementary File 2.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Drug likeness (QED) scores</h3>
<p id="Par27">All the screened-out compounds from the preliminary in-silico screening, i.e., 39 compounds, were found to have a QED score in the range, and they were taken further for more detailed in-silico analyses. A list of all the molecules is shown in Supplementary File 3.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Consensus molecular docking</h3>
<p id="Par28">The consensus molecular docking approach using eight different docking programs revealed varying binding affinities across the compound library. The binding energies and rankings from different programs were analyzed to identify the most promising compounds (Table <a href="#Tab1" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Binding energies across different Docking programs.</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Tab1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/1153c647d02d/41598_2025_13017_Tab1_HTML.jpg" loading="lazy" id="d33e705" height="945" width="702" alt="graphic file with name 41598_2025_13017_Tab1_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par30">To optimize and prioritize our molecular docking results, we applied the Rank-by-Rank (RbR) strategy, which ranks compounds based on their performance across multiple scoring functions. Each compound’s final rank is determined by averaging its ranks across all scoring methods, reducing biases from individual algorithms and providing a comprehensive evaluation of binding affinity. Compounds with the best average ranks were identified as promising candidates for further investigation, including experimental validation and optimization, as shown in Table <a href="#Tab2" class="usa-link">2</a>. This systematic approach strengthens the reliability of our virtual screening pipeline.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Consensus ranking of compounds.</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Tab2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/a4554fcfc886/41598_2025_13017_Tab2_HTML.jpg" loading="lazy" id="d33e720" height="923" width="708" alt="graphic file with name 41598_2025_13017_Tab2_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par32">A radar plot (Fig. <a href="#Fig2" class="usa-link">2</a>) visually represents the consensus docking results, displaying compound rankings across scoring methods. High-ranking compounds are highlighted with distinct colors for easier identification.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/f1a4980393be/41598_2025_13017_Fig2_HTML.jpg" loading="lazy" id="d33e735" height="575" width="730" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Radar plot illustrates the consensus ranking of the top five steroid-tetraoxane hybrid compounds and other molecules (ranked &gt; 5) across eight different molecular docking programs. Each axis represents a docking program, and the distance from the center indicates the rank of the compound (lower rank is closer to the center). This plot visually demonstrates the consistency of docking predictions: A-CY-28 (Ranked 1, dark blue line) consistently achieves a high rank (closer to the center) across most docking programs, followed by A-CY-9A (Ranked 2, dark purple), A-CY-9C (Ranked 3, pink), A-CY-10C (Ranked 4, cyan), and A-CY-9 (Ranked 5, green line). The “Other Molecules” (grey shaded areas) generally occupy the outer regions, indicating their lower consensus ranking. This visualization aids in identifying compounds with consistently favorable docking scores across multiple algorithms, thereby increasing confidence in their potential as lead candidates.</p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">Molecular dynamics simulation</h3>
<p id="Par34">A 500-nanosecond Molecular Dynamics Simulation was employed to assess the effects of ligand binding on protein stability, flexibility, interaction patterns, and energetics. This analysis yielded essential information for optimizing drug design and efficacy.</p>
<section id="Sec16"><h4 class="pmc_sec_title">RMSD</h4>
<p id="Par35">We analyzed Root Mean Square Deviation (RMSD) trajectories over a 500-nanosecond simulation to assess the stability of five protein-ligand complexes (Fig. <a href="#Fig3" class="usa-link">3</a>). RMSD measures structural changes over time, with lower values indicating more stable binding.</p>
<figure class="fig xbox font-sm" id="Fig3"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/7ca33687c2ac/41598_2025_13017_Fig3_HTML.jpg" loading="lazy" id="d33e756" height="362" width="781" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>RMSD trajectories of the ligand-bound protein over time, highlighting stability and conformational dynamics.</p></figcaption></figure><p id="Par37">The RMSD analysis reveals that A-CY-9C demonstrates the most favorable binding characteristics, with an average RMSD of 1.738 ± 0.099 Å, while the unbound protein exhibited an average RMSD of 1.819 ± 0.171 Å. Both trajectories show initial equilibration within the first 20 ns, followed by stable fluctuations throughout the remainder of the simulation. Notable fluctuations are observed in both systems, with RMSD values ranging between approximately 1.5 Å and 2.0 Å for A-CY-9C, while the unbound protein shows slightly higher fluctuations, reaching up to 2.2 Å. The other complexes display varying degrees of stability: A-CY-9A shows the highest average RMSD (2.029 ± 0.115 Å), with distinct fluctuations particularly evident in the early simulation phase (0-100 ns). A-CY-28 (1.812 ± 0.389 Å) exhibits interesting dynamics with increased conformational sampling, especially pronounced between 400 and 500 ns where notable peaks are observed. A-CY-10C (1.939 ± 0.147 Å) and A-CY-9 (1.858 ± 0.239 Å) maintain intermediate stability profiles with consistent fluctuations throughout the simulation. Several distinct conformational changes are observed across the trajectories, particularly evident around 300–400 ns for A-CY-28 and A-CY-9, while A-CY-9C maintains remarkably stable conformations throughout. The comparable RMSD values between A-CY-9C and the unbound protein, coupled with A-CY-9C’s lower standard deviation, suggest that this ligand stabilizes the protein while maintaining its essential flexibility. This characteristic could be particularly advantageous for therapeutic efficacy, as it indicates strong binding without compromising the protein’s natural dynamic behavior. The consistently lower RMSD fluctuations of A-CY-9C compared to both the unbound protein and other complexes, particularly in the latter half of the simulation, highlight its potential as a promising lead compound. The stability patterns observed suggest that A-CY-9C achieves an optimal balance between stable binding and preservation of protein dynamics, which could be crucial for maintaining normal protein function while achieving the desired therapeutic effect.</p></section><section id="Sec17"><h4 class="pmc_sec_title">RMSF</h4>
<p id="Par38">The Root Mean Square Fluctuation (RMSF) analysis was performed over the 500-nanosecond simulation to examine residue-specific flexibility patterns across all systems, as illustrated in Fig. <a href="#Fig4" class="usa-link">4</a>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h5 class="obj_head">Fig. 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/15d7d6c9d337/41598_2025_13017_Fig4_HTML.jpg" loading="lazy" id="d33e775" height="697" width="774" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Localized flexibility analysis of ligand-bound proteins via RMSF.</p></figcaption></figure><p id="Par40">A-CY-9C demonstrates the most pronounced effect on protein dynamics, exhibiting the lowest average RMSF of 0.909 ± 0.496 Å compared to the unbound protein’s 1.010 ± 0.590 Å. This reduction in residual fluctuations suggests that A-CY-9C binding effectively stabilizes the protein structure while maintaining essential flexibility. The RMSF plot reveals that A-CY-9C particularly stabilizes regions around residues 25–50 and 100–125, showing reduced peak heights compared to the unbound state. In contrast, A-CY-28 shows the highest average RMSF (1.091 ± 0.698 Å), with notably increased fluctuations, particularly evident in the regions spanning residues 75–100, where distinct peaks reaching up to 4.0 Å are observed. A-CY-9A and A-CY-9 display intermediate flexibility profiles (1.066 ± 0.629 Å and 1.059 ± 0.600 Å, respectively), while A-CY-10C (0.968 ± 0.559 Å) shows slightly higher stability than the unbound protein. Several common flexible regions are identified across all systems, with pronounced peaks around residues 50, 100, and 150, representing loop regions or protein termini. However, the magnitude of these fluctuations varies significantly among the complexes. The RMSF profiles indicate that while all ligands maintain the protein’s inherent flexibility pattern, A-CY-9C achieves the most balanced effect, reducing excessive movements while preserving functionally important dynamics. The consistently lower fluctuations observed with A-CY-9C, particularly in key binding site residues, suggest its potential as a superior candidate for therapeutic development, as it effectively modulates protein dynamics without inducing rigid conformational constraints that might impair function.</p></section><section id="Sec18"><h4 class="pmc_sec_title">Radius of gyration</h4>
<p id="Par41">The Radius of Gyration (Rg) analysis was performed over the 500-nanosecond simulation to evaluate the overall protein compactness and structural changes in the presence of different ligands, as illustrated in Fig. <a href="#Fig5" class="usa-link">5</a>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h5 class="obj_head">Fig. 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/fff0c0f28ae8/41598_2025_13017_Fig5_HTML.jpg" loading="lazy" id="d33e794" height="414" width="770" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Rg over time.</p></figcaption></figure><p id="Par43">The unbound protein exhibits an average Rg of 14.887 ± 0.060 Å, representing its native structural compactness. A-CY-10C shows the closest Rg value to the unbound state (14.897 ± 0.044 Å) with the lowest standard deviation among all complexes, suggesting that it maintains the protein’s natural compactness while slightly reducing conformational fluctuations. A-CY-9A and A-CY-9 demonstrate the highest average Rg values (14.958 ± 0.065 Å and 14.952 ± 0.074 Å, respectively), indicating a slight expansion of the protein structure upon binding. The Rg trajectories reveal notable fluctuations for these complexes, particularly evident in the 300–400 ns region, where distinct peaks suggest temporary structural expansions. A-CY-28 and A-CY-9C show intermediate Rg values (14.914 ± 0.097 Å and 14.913 ± 0.051 Å, respectively), though A-CY-28 displays the highest standard deviation, indicating more dynamic structural changes throughout the simulation. This is particularly visible in the 400–500 ns region, where increased fluctuations are observed. In contrast, A-CY-9C maintains more consistent structural compactness, as evidenced by its lower standard deviation. The Rg trajectories show initial equilibration within the first 50 ns, followed by stable fluctuations around their respective average values. The plot reveals that while all ligand-bound states maintain Rg values within 0.1 Å of the unbound protein, their dynamic behaviors differ significantly. The relatively small differences in average Rg values (maximum difference of 0.07 Å from unbound state) suggest that ligand binding does not dramatically alter the protein’s global structure, instead inducing subtle conformational adjustments that might be relevant for function. These findings indicate that all complexes maintain physiologically reasonable structural integrity throughout the simulation period.</p></section><section id="Sec19"><h4 class="pmc_sec_title">Solvent accessible surface area</h4>
<p id="Par44">The Solvent Accessible Surface Area (SASA) analysis was performed over the 500-nanosecond simulation to evaluate the extent of protein surface exposure to solvent and conformational changes induced by ligand binding, as illustrated in Fig. <a href="#Fig6" class="usa-link">6</a>.</p>
<figure class="fig xbox font-sm" id="Fig6"><h5 class="obj_head">Fig. 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/24c9090d97fc/41598_2025_13017_Fig6_HTML.jpg" loading="lazy" id="d33e813" height="423" width="782" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Ligands impact the solvent exposure of the Protein.</p></figcaption></figure><p id="Par46">A-CY-10C exhibits the lowest average SASA value (83.279 ± 1.504 nm²) compared to the unbound protein (83.972 ± 1.749 nm²), suggesting a slight reduction in solvent exposure upon binding. The SASA trajectory for A-CY-10C shows stable fluctuations throughout the simulation, with particularly consistent behavior in the 100–300 ns region, indicating a well-maintained binding mode that effectively shields protein surfaces. In contrast, A-CY-9A and A-CY-9 demonstrate significantly higher average SASA values (89.604 ± 1.938 nm² and 89.033 ± 3.499 nm², respectively), with A-CY-9 showing the highest standard deviation among all complexes. This is particularly evident in the latter half of the simulation (300–500 ns), where pronounced fluctuations suggest dynamic conformational changes that expose more protein surface to the solvent. A-CY-28 and A-CY-9C show intermediate SASA values (85.881 ± 3.028 nm² and 84.526 ± 1.643 nm², respectively). While A-CY-28 displays higher variability in its SASA profile, particularly noticeable between 400 and 500 ns, A-CY-9C maintains more consistent surface exposure throughout the simulation, as evidenced by its lower standard deviation. The SASA trajectories reveal distinct patterns of solvent accessibility, with notable transitions observed around 300 ns for several complexes. The plot demonstrates that while A-CY-10C and A-CY-9C maintain SASA values closer to the unbound protein, A-CY-9A and A-CY-9 consistently expose larger protein surface areas to the solvent, suggesting different binding modes that might affect protein-solvent interactions. These findings indicate that A-CY-10C achieves the most optimal balance in terms of surface protection, potentially contributing to complex stability, while maintaining sufficient solvent accessibility for proper protein function. The varying SASA profiles among the complexes provide valuable insights into their distinct binding mechanisms and potential effects on protein dynamics.</p></section><section id="Sec20"><h4 class="pmc_sec_title">Principle component analysis</h4>
<p id="Par47">Principal Component Analysis (PCA) was performed to investigate the essential dynamics and conformational sampling of the protein-ligand complexes throughout the 500-nanosecond simulation, as illustrated in Fig. <a href="#Fig7" class="usa-link">7</a>.</p>
<figure class="fig xbox font-sm" id="Fig7"><h5 class="obj_head">Fig. 7.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/9f3a3dff1806/41598_2025_13017_Fig7_HTML.jpg" loading="lazy" id="d33e832" height="423" width="778" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Principal component analysis for ligand-protein complexes.</p></figcaption></figure><p id="Par49">The PCA projection along the first two principal components (PC1 and PC2) reveals distinct conformational space exploration patterns for each complex. A-CY-9C demonstrates the most extensive sampling along PC1 (– 7.293 to 7.348 nm) while maintaining relatively constrained movements along PC2 (– 1.122 to 0.865 nm), suggesting a well-defined directional motion. This broad PC1 sampling indicates that A-CY-9C allows the protein to explore functionally relevant conformations while maintaining controlled flexibility. A-CY-9 shows intermediate conformational sampling with PC1 ranging from − 3.621 to 3.097 nm and PC2 from − 1.466 to 1.902 nm, exhibiting a more scattered distribution in the conformational space. This pattern suggests greater flexibility and diverse conformational states during the simulation. A-CY-10C and A-CY-28 display more restricted conformational sampling, with PC1 ranges of – 1.798 to 2.387 nm and − 0.707 to 2.159 nm, respectively, indicating more confined protein movements. A-CY-9A demonstrates the most compact conformational space exploration (PC1: – 1.935 to 1.059 nm; PC2: – 0.915 to 1.005 nm), suggesting a more rigid protein-ligand complex with limited conformational flexibility. The PCA plot clearly shows clustered regions for A-CY-9A, indicating stable, well-defined conformational states. The projection map reveals that while A-CY-9C allows for the broadest conformational sampling along PC1, it maintains controlled movements along PC2, suggesting a balance between flexibility and stability. This characteristic might be beneficial for protein function, allowing necessary conformational changes while preventing excessive structural fluctuations. The distinct clustering patterns observed for each complex provide valuable insights into their unique effects on protein dynamics and potential implications for their biological activity. These findings suggest that A-CY-9C achieves an optimal balance between conformational flexibility and structural stability, potentially contributing to its effectiveness as a therapeutic candidate.</p></section><section id="Sec21"><h4 class="pmc_sec_title">Free energy landscape</h4>
<p id="Par50">The free energy landscape (FEL) analysis was performed using the first two principal components (PC1 and PC2) as reaction coordinates to understand the conformational sampling and energy states of the protein-ligand complexes during the 500 ns simulation. The FEL is represented both as 2D contour maps and 3D surface plots (Fig. <a href="#Fig8" class="usa-link">8</a>).</p>
<figure class="fig xbox font-sm" id="Fig8"><h5 class="obj_head">Fig. 8.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig8a_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/90a79a272c15/41598_2025_13017_Fig8a_HTML.jpg" loading="lazy" id="d33e851" height="1211" width="731" alt="Fig. 8"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig8b_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/e0000fe0d29d/41598_2025_13017_Fig8b_HTML.jpg" loading="lazy" id="d33e853" height="264" width="735" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>FEL analysis of the protein-ligand complexes, projected onto the first two principal components (PC1 and PC2). The figure displays both 2D contour maps (left column for each molecule ID) and 3D surface representations (right column for each molecule ID) of the energy landscapes. The color gradient in the contour maps and the height in the 3D plots represent the free energy values, with darker/lower regions indicating more stable, lower-energy conformational states. This analysis reveals that A-CY-9C exhibits the most favorable energy landscape, characterized by a distinct and deep global energy minimum and relatively low energy barriers between conformational states. This suggests that A-CY-9C binding leads to a stable primary conformation with smooth transitions to nearby accessible states, highlighting A-CY-9C as possessing a highly favorable and stable conformational energy profile when bound to cyclophilin.</p></figcaption></figure><p id="Par52">A-CY-9C exhibits the most favorable energy landscape with a well-defined global minimum (– 5.8 kcal/mol) centered at PC1 = 2.1 nm and PC2 = 0.3 nm, surrounded by shallow local minima. This indicates a stable primary conformation with controlled transitions between nearby states. The energy barriers between states are relatively low (1.5-2.0 kcal/mol), suggesting smooth conformational transitions. A-CY-10C shows a broader energy basin with multiple local minima, the deepest being − 5.2 kcal/mol at PC1 = 1.8 nm and PC2 = – 0.5 nm. The presence of multiple low-energy states indicates conformational flexibility while maintaining stable binding. A-CY-28 and A-CY-9 display more rugged energy landscapes with higher energy barriers (2.5-3.0 kcal/mol) between states, suggesting less efficient conformational sampling. Their global minima (-4.8 and − 4.6 kcal/mol, respectively) are shallower compared to A-CY-9C and A-CY-10C. A-CY-9A shows the most restricted energy landscape with a single dominant minimum (– 4.3 kcal/mol) and steep energy barriers (&gt; 3.5 kcal/mol), indicating limited conformational exploration and potentially rigid binding. The analysis reveals A-CY-9C has the most favorable energy landscape with a well-defined global minimum and smooth conformational transitions.</p></section><section id="Sec22"><h4 class="pmc_sec_title">Hydrogen bond formation</h4>
<p id="Par53">The hydrogen bonding analysis was performed over the 500-nanosecond simulation to evaluate the stability and strength of protein-ligand interactions, as illustrated in Fig. <a href="#Fig9" class="usa-link">9</a>.</p>
<figure class="fig xbox font-sm" id="Fig9"><h5 class="obj_head">Fig. 9.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/aa0aac2e735a/41598_2025_13017_Fig9_HTML.jpg" loading="lazy" id="d33e872" height="425" width="771" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Hydrogen bonding dynamics in ligand-protein complexes over simulation time.</p></figcaption></figure><p id="Par55">A-CY-10C demonstrates the strongest hydrogen bonding network, maintaining an average of 2 ± 1 hydrogen bonds throughout the simulation. The hydrogen bond trajectory shows consistent interactions, with periods of up to 4 bonds formed simultaneously, particularly evident in the 300–400 ns region. This robust hydrogen bonding pattern suggests stable and specific interactions between A-CY-10C and key protein residues. A-CY-28 and A-CY-9 show similar hydrogen bonding patterns, averaging 1–2 hydrogen bonds intermittently throughout the simulation. The trajectories reveal the dynamic nature of these interactions, with periods of bond formation and breaking, indicating more transient binding modes. Both compounds display occasional peaks reaching up to 3 hydrogen bonds, though these higher-order interactions are not consistently maintained. A-CY-9C exhibits moderate hydrogen bonding, with periods of 1–2 stable hydrogen bonds, particularly noticeable in the latter half of the simulation (250–500 ns). The pattern suggests a consistent, though less extensive, network of interactions compared to A-CY-10C. A-CY-9A shows the least hydrogen bonding interactions, with only occasional formation of single hydrogen bonds, indicating weaker specific interactions with the protein binding site. The hydrogen bond analysis reveals distinct interaction patterns across the simulation time, with A-CY-10C demonstrating superior hydrogen bonding capabilities. This enhanced network of interactions likely contributes to its stable binding mode and could be a key factor in its therapeutic potential. The temporal evolution of hydrogen bonds, as shown in the plot, highlights the dynamic nature of these interactions and their potential role in maintaining stable protein-ligand complexes.</p></section><section id="Sec23"><h4 class="pmc_sec_title">Protein-ligand interaction timeline</h4>
<p id="Par56">The protein-ligand interaction timeline analysis over the 500 ns simulation revealed distinct binding patterns and interaction stability for each complex (Fig. <a href="#Fig10" class="usa-link">10</a>). This analysis captured key interactions, including hydrogen bonds, hydrophobic contacts, water bridges, and ionic interactions.</p>
<figure class="fig xbox font-sm" id="Fig10"><h5 class="obj_head">Fig. 10.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig10a_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/c3a4759a6162/41598_2025_13017_Fig10a_HTML.jpg" loading="lazy" id="d33e891" height="1240" width="733" alt="Fig. 10"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig10b_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/f92b3e45da1e/41598_2025_13017_Fig10b_HTML.jpg" loading="lazy" id="d33e893" height="776" width="740" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Protein-ligand interaction timeline analysis. The plots show the persistence of key molecular interactions throughout the 500 ns MD simulation for each compound.</p></figcaption></figure><p id="Par58">A-CY-9C maintained the most stable and diverse interaction network throughout the simulation. Critical hydrogen bonding interactions were observed with Asn173, Gln171, and His174, persisting for over 85% of the simulation time. The compound’s steroid core established strong hydrophobic contacts with Leu172, Trp206, and Phe236, while the tetraoxane moiety formed water-mediated bridges with Asp234 and Glu235. A-CY-10C demonstrated strong but more dynamic interaction patterns, with key hydrogen bonds fluctuating between Gln171 and His174. Notable π-π stacking interactions were observed between the aromatic rings and Trp206, present in approximately 70% of the simulation frames. A-CY-28 showed fewer consistent interactions, with hydrogen bonds to Asn173 being the most stable feature. The interaction timeline revealed periodic water bridge formations with Glu235, though these were less persistent compared to A-CY-9C. A-CY-9A and A-CY-9 exhibited more transient interactions, with frequent shifts in their binding modes. While both maintained contact with key residues like His174 and Trp206, the interactions were less stable compared to A-CY-9C and A-CY-10C. The interaction timeline analysis further supports A-CY-9C as the lead compound, demonstrating the most stable and diverse interaction network throughout the simulation period.</p>
<p id="Par59">Based on computational analyses, present multiple, independent lines of evidence A-CY-9C emerges as the primary candidate due to its optimal balance between: Structural stability, Essential flexibility, Controlled conformational dynamics, Consistent binding mode. The consensus docking approach demonstrated A-CY-9C’s superior binding affinity across diverse scoring algorithms, while molecular dynamics simulations revealed its capacity to form a stable yet adaptable complex with cyclophilin. The compound’s optimal balance between conformational stability (RMSD), selective modulation of protein flexibility (RMSF), favorable thermodynamics (FEL), and formation of specific, persistent interactions with key residues (interaction timeline) collectively indicates a binding mode engineered to effectively inhibit cyclophilin function. Moreover, the detailed structural analysis revealed how A-CY-9C’s unique molecular architecture—combining the steroidal scaffold with a perfectly positioned tetraoxane moiety—enables simultaneous targeting of multiple parasite vulnerabilities. This integrated approach to target engagement suggests A-CY-9C could overcome resistance mechanisms that have limited the efficacy of current artemisinin-based therapies. However, A-CY-10C presents as a strong alternative candidate with superior hydrogen bonding characteristics and solvent protection, which could be advantageous for specific therapeutic applications. The other compounds (A-CY-28, A-CY-9, and A-CY-9A) show less favorable characteristics across multiple parameters, suggesting they may require further optimization for therapeutic development. This analysis suggests prioritizing A-CY-9C for further development, with A-CY-10C as a viable alternative candidate, particularly in applications where strong hydrogen bonding networks are crucial for therapeutic efficacy.</p></section></section><section id="Sec24"><h3 class="pmc_sec_title">Molecular interaction analysis and structure-activity relationship</h3>
<p id="Par60">The detailed molecular interaction analysis of A-CY-9C with cyclophilin reveals a complex network of specific interactions that contribute to its superior binding affinity and stability (Fig. <a href="#Fig11" class="usa-link">11</a>). This analysis provides critical insights into the structural features essential for the compound’s antimalarial activity.</p>
<figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/5a7db1ceda6d/41598_2025_13017_Fig11_HTML.jpg" loading="lazy" id="d33e914" height="439" width="780" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Protein-ligand interaction analysis of A-CY-9C bound to Cyclophilin.</p></figcaption></figure><p id="Par62">The 3D representation (Fig. <a href="#Fig11" class="usa-link">11</a>a) shows A-CY-9C deeply embedded within the cyclophilin binding pocket, adopting an optimal conformation that maximizes favorable interactions. The 2D interaction map (Fig. <a href="#Fig11" class="usa-link">11</a>b) illustrates the diverse binding modes employed by A-CY-9C, including hydrogen bonding, π-stacking, π-cation, π-sulfur, and hydrophobic interactions.</p>
<p id="Par63">A key feature of A-CY-9C binding is the conventional hydrogen bond formed with ASN109, which anchors the tetraoxane moiety within the binding site. This interaction is critical for positioning the peroxide bridge in proximity to catalytically important residues. The hydrogen bond with SER80 further stabilizes the complex, contributing to the compound’s prolonged residence time observed during MD simulations. The aromatic interactions play a significant role in A-CY-9C’s binding profile. The π-π stacking interaction with PHE120 provides substantial stabilization energy, while the T-shaped π-π interaction with HIS133 offers additional anchoring. These aromatic interactions are complemented by π-cation interactions with ARG62, which contribute electrostatic stabilization to the complex. Notably, A-CY-9C engages in π-sulfur interactions with MET68, a relatively uncommon interaction type that provides additional binding specificity. The compound also forms hydrophobic contacts with ALA108 and LEU129, which shield the peroxide bridge from premature decomposition while maintaining it in position for iron-mediated activation.</p>
<p id="Par64">From a structure-activity relationship perspective, several critical features emerge as essential for A-CY-9C’s antimalarial activity. The tetraoxane (1,2,4,5-tetraoxane) pharmacophore is optimally positioned for interaction with iron species within the parasite, while being protected from premature decomposition by surrounding hydrophobic residues. The steroidal scaffold provides an ideal spatial arrangement that positions functional groups for optimal interactions with key residues, particularly the aromatic and hydrogen bonding interactions. The thiophenyl substituent engages in specific π-sulfur interactions with MET68, suggesting that sulfur-containing groups at this position enhance binding affinity. The rigid core structure of A-CY-9C minimizes the entropic penalty upon binding, contributing to its favorable free energy profile observed in MD simulations.</p>
<p id="Par65">Figure <a href="#Fig12" class="usa-link">12</a> displays the time evolution of the cyclophilin-A-CY-9C complex during a 500-nanosecond (ns) molecular dynamics simulation, with snapshots taken at five critical time points (0, 125, 250, 375, and 500 ns) represented in different colors. The cyclophilin structure (shown as ribbons) maintains its core fold throughout the entire simulation, with secondary structure elements (beta sheets and alpha helices) remaining well-preserved. This indicates that A-CY-9C binding does not disrupt the protein’s global architecture. A-CY-9C (shown in stick representation) remains firmly anchored in the binding pocket across all time points, demonstrating remarkable binding stability over the extended simulation. While the ligand maintains its position in the binding site, subtle reorientations are visible in the tetraoxane and aromatic regions across different time points. These minor adjustments likely represent optimization of key interactions with surrounding amino acids, particularly the π-π stacking with PHE120 and hydrogen bonding with ASN109. The steroidal core of A-CY-9C shows minimal movement throughout the simulation, serving as a rigid anchor, while the peripheral functional groups demonstrate controlled flexibility. This balanced rigidity-flexibility profile contributes to the compound’s favorable binding energetics. The protein binding site shows subtle adaptations to accommodate A-CY-9C, demonstrated by the small shifts in loop regions surrounding the ligand. This induced-fit mechanism enhances complementarity between the ligand and target.</p>
<figure class="fig xbox font-sm" id="Fig12"><h4 class="obj_head">Fig. 12.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig12_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/478d0c22b9cb/41598_2025_13017_Fig12_HTML.jpg" loading="lazy" id="d33e941" height="972" width="686" alt="Fig. 12"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Superimposed snapshots of the cyclophilin-A-CY-9C complex at five time points (0, 125, 250, 375, and 500 ns) during molecular dynamics simulation, showing remarkable stability of both protein conformation and ligand binding position throughout the 500 ns trajectory.</p></figcaption></figure><p id="Par67">This analysis provides strong structural evidence for the exceptional binding stability of A-CY-9C to cyclophilin, supporting its identification as the lead compound for further development as a potential antimalarial agent targeting cyclophilin-mediated artemisinin resistance.</p></section><section id="Sec25"><h3 class="pmc_sec_title">ADME and toxicity</h3>
<p id="Par68">Compound A-CY-9C demonstrated a favorable pharmacokinetic and safety profile based on in-silico predictions (Table <a href="#Tab3" class="usa-link">3</a>). SwissADME analysis predicted low gastrointestinal absorption and no blood-brain barrier permeability, suggesting limited systemic distribution and minimal central nervous system exposure. The compound showed no inhibitory activity against any of the five major cytochrome P450 isoforms tested (CYP1A2, CYP2C19, CYP2C9, CYP2D6, and CYP3A4), indicating a low potential for drug-drug interactions through these metabolic pathways. Toxicity assessment using DataWarrior predicted no mutagenic or reproductive toxicity potential for A-CY-9C, with only a low irritant classification. ProTox-3.0 categorized the compound as Class 6 in the toxicity classification system, representing the lowest toxicity category with predicted LD50 values greater than 5000 mg/kg. This comprehensive safety profile suggests that A-CY-9C may be a promising candidate for further development, with minimal concerns regarding potential adverse effects or toxicological liabilities.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>ADME and toxicity profile of A-CY-9C.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="7" rowspan="1">ADME</th>
<th align="left" colspan="4" rowspan="1">Toxicity</th>
</tr>
<tr>
<th align="left" colspan="7" rowspan="1">(Predicted by using SwissADME)</th>
<th align="left" colspan="3" rowspan="1">(Predicted by data warrior)</th>
<th align="left" colspan="1" rowspan="1">(Predicted by ProTox-3.0)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">GI absorption</th>
<th align="left" colspan="1" rowspan="1">BBB permeant</th>
<th align="left" colspan="1" rowspan="1">CYP1A2 inhibitor</th>
<th align="left" colspan="1" rowspan="1">CYP2C19 inhibitor</th>
<th align="left" colspan="1" rowspan="1">CYP2C9 inhibitor</th>
<th align="left" colspan="1" rowspan="1">CYP2D6 inhibitor</th>
<th align="left" colspan="1" rowspan="1">CYP3A4 inhibitor</th>
<th align="left" colspan="1" rowspan="1">Mutagenic</th>
<th align="left" colspan="1" rowspan="1">Reproductive effective</th>
<th align="left" colspan="1" rowspan="1">Irritant</th>
<th align="left" colspan="1" rowspan="1">Toxicity class</th>
</tr>
</thead>
<tbody><tr>
<td align="left" colspan="1" rowspan="1">Low</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">No</td>
<td align="left" colspan="1" rowspan="1">None</td>
<td align="left" colspan="1" rowspan="1">None</td>
<td align="left" colspan="1" rowspan="1">Low</td>
<td align="left" colspan="1" rowspan="1">Class 6</td>
</tr></tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par70">GI absorption of A-CY-9C was predicted to be low, which, while potentially limiting oral bioavailability, is not necessarily prohibitive for drug development as numerous successful marketed antimalarials (including artemisinin derivatives, chloroquine, and mefloquine) demonstrate clinical efficacy despite suboptimal absorption profiles through appropriate formulation strategies and dosing regimens.</p></section></section><section id="Sec26"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par71">The present study provides a comprehensive computational evaluation of a novel library of steroid-tetraoxane hybrids as potential antimalarial agents, particularly targeting cyclophilin to overcome artemisinin (ART) resistance. The emergence of resistant parasite strains to ART and its partner drugs poses a threat to control and eradicate malaria. Resistance against ACTs such as artesunate-mefloquine, artesunate-amodiaquine, and artemether/ lumefantrine is reported from different regions<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. The unavailability of novel potent antimalarials against the emerging resistance is a major setback for accomplishing the WHO’s Global technical strategy for malaria 2016–2030 of reducing global malaria incidence and mortality rates by at least 90% by 2030<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>.</p>
<p id="Par72">There are many drug candidates in clinical phases, and a few of them, viz. CDRI-97/78 is a synthetic 1,2,4-trioxane that is in Phase I of clinical trial and has shown promising results against resistant <em>P.falciparum</em> strains and was also well tolerated after a single dose<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. However, pharmacokinetic studies reveal that there is a drug-drug interaction with long-acting Lumefantrine, which is the mainstay of ACT<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. Another drug, Arterolane (RBx11160 or OZ277), is a synthetic 1,2,4-trioxolane and is in Phase III of clinical trials and has shown better bioavailability and antimalarial activity compared to ART<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. However combination of artemisinin-piperaquine has shown side effects such as hyperkalemia, eosinophilia, neutropenia, anemia, thrombocytopenia, etc<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Artemisone (BAY 44–9585) is a synthetically modified ART currently in Phase III of clinical trials and has been found to be effective against <em>P. falciparum</em> strains. It has shown higher cell permeability, moderate lipophilicity, but poor aqueous solubility, which finally hinders oral absorption and bioavailability<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. Artefenomel (or OZ439) is a synthetic 1,2,4-trioxolane<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> that has shown better antimalarial activity and is more stable, and has good bioavailability compared with the existing antimalarials<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. But some adverse drug reactions such as diarrhea, nausea, gastrointestinal hypermotility, headache, flushing, throat irritation, dyspepsia, and vasovagal syncope, etc. were reported<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. A hybrid strategy, i.e., a single molecule containing more than one pharmacophore with different modes of action, may lead to the development of molecules with better biological activity and can contribute to overcoming the problems associated with the existing antimalarial drugs<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Steroids conjugated with primaquine and artesunate have shown a multi-targeted approach by inhibiting different stages of parasite lifecycles, multiple species of parasites, and drug-resistant parasites<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>.</p>
<p id="Par73">The computational approach employed here to develop novel steroid-tetraoxane hybrids offers several advantages over the traditional drug discovery workflow. Systematic compound library designing and screening them for toxicity and drug likeness properties have reduced the number of compounds to a limited number of probable hits for further detailed evaluation. The consensus molecular docking and extensive MD simulations methods adopted here provide detailed insights into protein-ligand interactions and binding stability. Particularly, compound A-CY-9C demonstrated superior binding characteristics and stability compared to existing antimalarial compounds reported in the literature. The hybrid strategy employed in this study addresses multiple therapeutic targets simultaneously, potentially reducing the likelihood of resistance development. Figure <a href="#Fig13" class="usa-link">13</a> summarizes the entire virtual screening process.</p>
<figure class="fig xbox font-sm" id="Fig13"><h3 class="obj_head">Fig. 13.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365323_41598_2025_13017_Fig13_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9bd6/12365323/d1cf1973e514/41598_2025_13017_Fig13_HTML.jpg" loading="lazy" id="d33e1113" height="331" width="738" alt="Fig. 13"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Results of in-silico screening and simulation.</p></figcaption></figure><p id="Par75">The 500-nanosecond MD simulations revealed that A-CY-9C formed the most stable complex with cyclophilin, as evidenced by its low RMSD (1.738 Å) and minimal RMSF, particularly in key functional regions. This supports the hypothesis that ligands capable of stabilizing the binding pocket while maintaining essential flexibility are ideal for targeting dynamic proteins like cyclophilin<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. The Principal component (PCA) and free-energy landscape (FEL) analyses highlighted A-CY-9C’s ability to balance conformational flexibility and stability, characteristics increasingly recognized as critical for overcoming resistance in <em>Plasmodium falciparum</em><sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. A-CY-10C’s superior hydrogen bonding network suggests potential advantages in terms of binding specificity and residence time, echoing findings where enhanced hydrogen bonding correlates with improved antimalarial activity<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. However, its lower conformational sampling in PCA may reflect a more rigid complex that could limit its adaptability to dynamic target sites. The interaction timeline and hydrogen bonding analyses confirmed that A-CY-9C established a stable and diverse interaction network, particularly with residues critical for the function of cyclophilin (e.g., ASN109, PHE120, HIS133). These interactions are consistent with recent structural studies highlighting the therapeutic potential of targeting these residues in cyclophilin to combat artemisinin resistance<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. The detailed structure-activity relationship (SAR) analysis further underscores the importance of the tetraoxane pharmacophore and steroidal scaffold in achieving potent and selective binding. This aligns with recent reports advocating for the rational design of dual pharmacophore hybrids to enhance binding affinity and reduce susceptibility to resistance mechanisms<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. The ADMET predictions suggest that A-CY-9C exhibits a favorable safety profile, with no predicted mutagenicity, reproductive toxicity, or cytochrome P450 inhibition. The low gastrointestinal absorption predicted for A-CY-9C mirrors the challenges faced by many antimalarials but is not prohibitive; formulation strategies can be employed to enhance bioavailability (e.g., nanoparticle encapsulation, lipid-based carriers)<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. Importantly, the absence of blood-brain barrier permeability may reduce the risk of CNS-related adverse effects, a desirable property for antimalarial agents<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>.</p>
<p id="Par76">Future work will mainly focus on the chemical synthesis and optimization of A-CY-9C and other promising candidates identified in this study or with similar drug chemistry; in vitro antimalarial screening against sensitive and resistant strains; in vivo pharmacokinetic profiling and structure-activity relationship (SAR) studies to further enhance potency and selectivity. This multi-faceted approach will advance our understanding of steroid-tetraoxane hybrids as a novel class of antimalarial agents and may provide a lead for the development of effective therapies against resistant malaria strains.</p></section><section id="Sec27"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par77">This study demonstrates the antimalarial potential of novel steroid-tetraoxane hybrids through a comprehensive computational pipeline combining consensus molecular docking and molecular dynamics simulations. The hybrid compound A-CY-9C emerged as a lead candidate, exhibiting superior binding stability and a favorable free energy landscape with cyclophilin, a key target in <em>Plasmodium falciparum</em>. The dual pharmacophore strategy synergizes the steroid component’s cholesterol uptake disruption and the tetraoxane moiety’s oxidative stress induction, offering a robust mechanism to circumvent artemisinin resistance linked to <em>Pf</em>K13 mutations. Consensus docking across eight algorithms minimized platform bias, while 500-ns MD simulations validated conformational stability and persistent interactions with critical residues (Asn173, Gln171, His174). Structural-activity insights revealed essential hydrophobic, π-stacking, and hydrogen-bonding patterns for optimization. These findings position steroid-tetraoxane hybrids, particularly A-CY-9C, as promising candidates for experimental validation and the development of next-generation antimalarials against resistant strains, addressing the urgent need highlighted by WHO’s malaria eradication goals. Efforts are ongoing in our laboratory to synthesize these compounds to evaluate their in vitro, in vivo efficacy and pharmacokinetic profile.</p></section><section id="Sec28"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365323/bin/41598_2025_13017_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (14.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365323/bin/41598_2025_13017_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (16.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365323/bin/41598_2025_13017_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (11.2KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>One of the authors DN is grateful to ICMR, New Delhi, India, for the award of Research Associate (Fellowship /110/2022/ECD-II).We also want to acknowledge the National Supercomputing Mission [NSM] for providing computing resources of ‘PARAM Shivay’ at Indian Institute of Technology [BHU], Varanasi, which is implemented by C-DAC and supported by the Ministry of Electronics and Information Technology [MeitY] and Department of Science and Technology [DST], Government of India.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>ACT</dt>
<dd><p id="Par2">Artemisinin-based combination therapy</p></dd>
<dt>RMSD</dt>
<dd><p id="Par3">Root Mean Square Deviation</p></dd>
<dt>RMSF</dt>
<dd><p id="Par4">Root Mean Square Fluctuation</p></dd>
<dt>Rg</dt>
<dd><p id="Par5">Radius of Gyration</p></dd>
<dt>SASA</dt>
<dd><p id="Par6">Solvent Accessible Surface Area</p></dd>
<dt>PCA</dt>
<dd><p id="Par7">Principal Component Analysis</p></dd>
<dt>FEL</dt>
<dd><p id="Par8">Free Energy Landscape</p></dd>
<dt>WHO</dt>
<dd><p id="Par9">World Health Organization</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>D.N. conceptualized the study, designed the compounds, performed toxicity assessment, drug score, and QED score evaluations, and contributed to manuscript writing and editing. A.D. conducted the consensus molecular docking, molecular dynamics simulations, and contributed to manuscript writing and editing. M.S. and R.K.S. performed data analysis and contributed manuscript editing. D.C. supervised the study. All authors reviewed and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>The author(s) reported that there is no funding associated with the work featured in this article.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data generated or analysed during this study are included in this published article and its supplementary information files.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="notes5"><p>Figures submitted with the manuscript have been generated by the authors themselves and have not been copied or adapted from other sources. The standard of the English language in this article has been enhanced using LLMs, namely Llama-2-70B-GGML and OpenAI’s chatbot, Chat Generative Pre-trained Transformer 4.0. We bear full responsibility for the contents of this publication. We have conducted a comprehensive examination, made improvements, and refined all languages generated by the LLM to ensure their precise alignment with our initial research outcomes and interpretations.</p></section><section id="FPar1"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par78">Not applicable. The study does not contain data from any person.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par79">The authors declare no competing interests.</p></section><section id="FPar5"><h3 class="pmc_sec_title">Ethical approval and consent to participate</h3>
<p id="Par80">Not applicable.</p></section><section id="FPar6"><h3 class="pmc_sec_title">Human and animal rights</h3>
<p id="Par81">Not applicable.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Dipankar Nath and Abhijit Debnath.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Dipankar Nath, Email: pharmchem1209@gmail.com.</p>
<p>Abhijit Debnath, Email: abhijitdebnath@outlook.in, Email: abhijit.debnath@niet.co.in.</p>
<p>Rajesh Kumar Singh, Email: rajesh.singh1@bhu.ac.in, Email: rkszoology@gmail.com.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Milner, D. A. Malaria pathogenesis. <em>Cold Spring Harb Perspect. Med.</em><strong>8</strong>, a025569 (2018).
</cite> [<a href="https://doi.org/10.1101/cshperspect.a025569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5749143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28533315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Milner,%20D.%20A.%20Malaria%20pathogenesis.%20Cold%20Spring%20Harb%20Perspect.%20Med.8,%20a025569%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Bagcchi, S. Climate change recognised in world malaria report 2023. <em>Lancet Infect. Dis.</em><strong>24</strong>, e157 (2024).
</cite> [<a href="https://doi.org/10.1016/S1473-3099(24)00091-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38401561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bagcchi,%20S.%20Climate%20change%20recognised%20in%20world%20malaria%20report%202023.%20Lancet%20Infect.%20Dis.24,%20e157%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Cui, L., Mharakurwa, S., Ndiaye, D., Rathod, P. K. &amp; Rosenthal, P. J. Antimalarial drug resistance: literature review and activities and findings of the ICEMR network. <em>Am. J. Trop. Med. Hyg.</em><strong>93</strong>, 57–68 (2015).
</cite> [<a href="https://doi.org/10.4269/ajtmh.15-0007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4574275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26259943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cui,%20L.,%20Mharakurwa,%20S.,%20Ndiaye,%20D.,%20Rathod,%20P.%20K.%20&amp;%20Rosenthal,%20P.%20J.%20Antimalarial%20drug%20resistance:%20literature%20review%20and%20activities%20and%20findings%20of%20the%20ICEMR%20network.%20Am.%20J.%20Trop.%20Med.%20Hyg.93,%2057%E2%80%9368%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Nosten, F. &amp; White, N. J. Artemisinin-based combination treatment of falciparum malaria. <em>Am. J. Trop. Med. Hyg.</em><strong>77</strong>, 181–192 (2007).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/18165491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nosten,%20F.%20&amp;%20White,%20N.%20J.%20Artemisinin-based%20combination%20treatment%20of%20falciparum%20malaria.%20Am.%20J.%20Trop.%20Med.%20Hyg.77,%20181%E2%80%93192%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Reyburn, H. New WHO guidelines for the treatment of malaria. <em>BMJ</em><strong>340</strong>, c2637–c2637 (2010).
</cite> [<a href="https://doi.org/10.1136/bmj.c2637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20511305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reyburn,%20H.%20New%20WHO%20guidelines%20for%20the%20treatment%20of%20malaria.%20BMJ340,%20c2637%E2%80%93c2637%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Umumararungu, T. et al. Recent developments in antimalarial drug discovery. <em>Bioorg. Med. Chem.</em><strong>88–89</strong>, 117339 (2023).
</cite> [<a href="https://doi.org/10.1016/j.bmc.2023.117339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37236020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Umumararungu,%20T.%20et%20al.%20Recent%20developments%20in%20antimalarial%20drug%20discovery.%20Bioorg.%20Med.%20Chem.88%E2%80%9389,%20117339%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Chaurasiya, A. et al. Targeting artemisinin-resistant malaria by repurposing the anti-Hepatitis C virus drug alisporivir. <em>Antimicrob. Agents Chemother.</em><strong>66</strong>, e0039222 (2022).
</cite> [<a href="https://doi.org/10.1128/aac.00392-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9765015/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36374050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaurasiya,%20A.%20et%20al.%20Targeting%20artemisinin-resistant%20malaria%20by%20repurposing%20the%20anti-Hepatitis%20C%20virus%20drug%20alisporivir.%20Antimicrob.%20Agents%20Chemother.66,%20e0039222%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Marín-Menéndez, A. &amp; Bell, A. Overexpression, purification and assessment of cyclosporin binding of a family of cyclophilins and cyclophilin-like proteins of the human malarial parasite plasmodium falciparum. <em>Protein Exp. Purif.</em><strong>78</strong>, 225–234 (2011).</cite> [<a href="https://doi.org/10.1016/j.pep.2011.04.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21549842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mar%C3%ADn-Men%C3%A9ndez,%20A.%20&amp;%20Bell,%20A.%20Overexpression,%20purification%20and%20assessment%20of%20cyclosporin%20binding%20of%20a%20family%20of%20cyclophilins%20and%20cyclophilin-like%20proteins%20of%20the%20human%20malarial%20parasite%20plasmodium%20falciparum.%20Protein%20Exp.%20Purif.78,%20225%E2%80%93234%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Marín-Menéndez, A., Monaghan, P. &amp; Bell, A. A family of cyclophilin-like molecular chaperones in plasmodium falciparum. <em>Mol. Biochem. Parasitol.</em><strong>184</strong>, 44–47 (2012).
</cite> [<a href="https://doi.org/10.1016/j.molbiopara.2012.04.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22546550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mar%C3%ADn-Men%C3%A9ndez,%20A.,%20Monaghan,%20P.%20&amp;%20Bell,%20A.%20A%20family%20of%20cyclophilin-like%20molecular%20chaperones%20in%20plasmodium%20falciparum.%20Mol.%20Biochem.%20Parasitol.184,%2044%E2%80%9347%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Meunier, B. Hybrid molecules with a dual mode of action: dream or reality? <em>Acc. Chem. Res.</em><strong>41</strong>, 69–77 (2008).
</cite> [<a href="https://doi.org/10.1021/ar7000843" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17665872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meunier,%20B.%20Hybrid%20molecules%20with%20a%20dual%20mode%20of%20action:%20dream%20or%20reality?%20Acc.%20Chem.%20Res.41,%2069%E2%80%9377%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Solaja, B. A. et al. Mixed steroidal 1,2,4,5-tetraoxanes: antimalarial and antimycobacterial activity. <em>J. Med. Chem.</em><strong>45</strong>, 3331–3336 (2002).
</cite> [<a href="https://doi.org/10.1021/jm020891g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12139444/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Solaja,%20B.%20A.%20et%20al.%20Mixed%20steroidal%201,2,4,5-tetraoxanes:%20antimalarial%20and%20antimycobacterial%20activity.%20J.%20Med.%20Chem.45,%203331%E2%80%933336%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Singh, C., Sharma, U., Saxena, G. &amp; Puri, S. K. Orally active antimalarials: synthesis and bioevaluation of a new series of steroid-based 1,2,4-trioxanes against multi-drug resistant malaria in mice. <em>Bioorg. Med. Chem. Lett.</em><strong>17</strong>, 4097–4101 (2007).
</cite> [<a href="https://doi.org/10.1016/j.bmcl.2007.05.055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17548195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh,%20C.,%20Sharma,%20U.,%20Saxena,%20G.%20&amp;%20Puri,%20S.%20K.%20Orally%20active%20antimalarials:%20synthesis%20and%20bioevaluation%20of%20a%20new%20series%20of%20steroid-based%201,2,4-trioxanes%20against%20multi-drug%20resistant%20malaria%20in%20mice.%20Bioorg.%20Med.%20Chem.%20Lett.17,%204097%E2%80%934101%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Lombard, M. C., N’Da, D. D., Breytenbach, J. C., Smith, P. J. &amp; Lategan, C. A. Synthesis, in vitro antimalarial and cytotoxicity of artemisinin-aminoquinoline hybrids. <em>Bioorg. Med. Chem. Lett.</em><strong>21</strong>, 1683–1686 (2011).
</cite> [<a href="https://doi.org/10.1016/j.bmcl.2011.01.103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21316959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lombard,%20M.%20C.,%20N%E2%80%99Da,%20D.%20D.,%20Breytenbach,%20J.%20C.,%20Smith,%20P.%20J.%20&amp;%20Lategan,%20C.%20A.%20Synthesis,%20in%20vitro%20antimalarial%20and%20cytotoxicity%20of%20artemisinin-aminoquinoline%20hybrids.%20Bioorg.%20Med.%20Chem.%20Lett.21,%201683%E2%80%931686%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Capela, R. et al. Design and evaluation of Primaquine-Artemisinin hybrids as a multistage antimalarial strategy. <em>Antimicrob. Agents Chemother.</em><strong>55</strong>, 4698–4706 (2011).
</cite> [<a href="https://doi.org/10.1128/AAC.05133-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3186991/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21807973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Capela,%20R.%20et%20al.%20Design%20and%20evaluation%20of%20Primaquine-Artemisinin%20hybrids%20as%20a%20multistage%20antimalarial%20strategy.%20Antimicrob.%20Agents%20Chemother.55,%204698%E2%80%934706%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Pretorius, S. I., Breytenbach, W. J., De Kock, C., Smith, P. J. &amp; N’Da, D. D. Synthesis, characterization and antimalarial activity of quinoline–pyrimidine hybrids. <em>Bioorg. Med. Chem.</em><strong>21</strong>, 269–277 (2013).
</cite> [<a href="https://doi.org/10.1016/j.bmc.2012.10.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23168082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pretorius,%20S.%20I.,%20Breytenbach,%20W.%20J.,%20De%20Kock,%20C.,%20Smith,%20P.%20J.%20&amp;%20N%E2%80%99Da,%20D.%20D.%20Synthesis,%20characterization%20and%20antimalarial%20activity%20of%20quinoline%E2%80%93pyrimidine%20hybrids.%20Bioorg.%20Med.%20Chem.21,%20269%E2%80%93277%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Thakur, A., Khan, S. I. &amp; Rawat, D. S. Synthesis of piperazine tethered 4-aminoquinoline-pyrimidine hybrids as potent antimalarial agents. <em>RSC Adv.</em><strong>4</strong>, 20729–20736 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Thakur,%20A.,%20Khan,%20S.%20I.%20&amp;%20Rawat,%20D.%20S.%20Synthesis%20of%20piperazine%20tethered%204-aminoquinoline-pyrimidine%20hybrids%20as%20potent%20antimalarial%20agents.%20RSC%20Adv.4,%2020729%E2%80%9320736%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Yamansarov, E. Y. et al. Synthesis and antimalarial activity of 3’-trifluoromethylated 1,2,4-trioxolanes and 1,2,4,5-tetraoxane based on deoxycholic acid. <em>Steroids</em><strong>129</strong>, 17–23 (2018).
</cite> [<a href="https://doi.org/10.1016/j.steroids.2017.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29180289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamansarov,%20E.%20Y.%20et%20al.%20Synthesis%20and%20antimalarial%20activity%20of%203%E2%80%99-trifluoromethylated%201,2,4-trioxolanes%20and%201,2,4,5-tetraoxane%20based%20on%20deoxycholic%20acid.%20Steroids129,%2017%E2%80%9323%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Indriani, I., Aminah, N. S. &amp; Puspaningsih, N. N. T. Antiplasmodial activity of stigmastane steroids from Dryobalanops oblongifolia stem bark. <em>Open. Chem.</em><strong>18</strong>, 259–264 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Indriani,%20I.,%20Aminah,%20N.%20S.%20&amp;%20Puspaningsih,%20N.%20N.%20T.%20Antiplasmodial%20activity%20of%20stigmastane%20steroids%20from%20Dryobalanops%20oblongifolia%20stem%20bark.%20Open.%20Chem.18,%20259%E2%80%93264%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Raju, J. &amp; V., C. Diosgenin, a Steroid Saponin Constituent of Yams and Fenugreek: Emerging Evidence for Applications in Medicine. in <em>Bioactive Compounds in Phytomedicine</em> (ed. Rasooli, I.)InTech, (2012).</cite>
</li>
<li id="CR20">
<span class="label">20.</span><cite>Anti-malaria and hypoglycaemic activities of Diosgenin on alloxan-induced, diabetic Wistar rats. <em>GSC Biol. Pharm. Sci.</em><strong>15</strong>, 073–079 (2021).</cite>
</li>
<li id="CR21">
<span class="label">21.</span><cite>Pabón, A. et al. Diosgenone synthesis, anti-Malarial activity and QSAR of analogues of this natural product. <em>Molecules</em><strong>18</strong>, 3356–3378 (2013).
</cite> [<a href="https://doi.org/10.3390/molecules18033356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6270258/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23493102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pab%C3%B3n,%20A.%20et%20al.%20Diosgenone%20synthesis,%20anti-Malarial%20activity%20and%20QSAR%20of%20analogues%20of%20this%20natural%20product.%20Molecules18,%203356%E2%80%933378%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Fraser, M. et al. Harnessing cholesterol uptake of malaria parasites for therapeutic applications. <em>EMBO Mol. Med.</em><strong>16</strong>, 1515–1532 (2024).
</cite> [<a href="https://doi.org/10.1038/s44321-024-00087-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11251039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38862600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fraser,%20M.%20et%20al.%20Harnessing%20cholesterol%20uptake%20of%20malaria%20parasites%20for%20therapeutic%20applications.%20EMBO%20Mol.%20Med.16,%201515%E2%80%931532%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Rudrapal, M., Chetia, D. &amp; Singh, V. Novel series of 1,2,4-trioxane derivatives as antimalarial agents. <em>J. Enzyme Inhib. Med. Chem.</em><strong>32</strong>, 1159–1173 (2017).
</cite> [<a href="https://doi.org/10.1080/14756366.2017.1363742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6009891/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28870093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rudrapal,%20M.,%20Chetia,%20D.%20&amp;%20Singh,%20V.%20Novel%20series%20of%201,2,4-trioxane%20derivatives%20as%20antimalarial%20agents.%20J.%20Enzyme%20Inhib.%20Med.%20Chem.32,%201159%E2%80%931173%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Koska, J. et al. Fully automated molecular mechanics based induced fit protein-ligand Docking method. <em>J. Chem. Inf. Model.</em><strong>48</strong>, 1965–1973 (2008).
</cite> [<a href="https://doi.org/10.1021/ci800081s" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18816046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koska,%20J.%20et%20al.%20Fully%20automated%20molecular%20mechanics%20based%20induced%20fit%20protein-ligand%20Docking%20method.%20J.%20Chem.%20Inf.%20Model.48,%201965%E2%80%931973%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Krovat, E., Steindl, T. &amp; Langer, T. Recent advances in Docking and scoring. <em>Curr. Comput. Aided-Drug Des.</em><strong>1</strong>, 93–102 (2005).</cite> [<a href="https://scholar.google.com/scholar_lookup?Krovat,%20E.,%20Steindl,%20T.%20&amp;%20Langer,%20T.%20Recent%20advances%20in%20Docking%20and%20scoring.%20Curr.%20Comput.%20Aided-Drug%20Des.1,%2093%E2%80%93102%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Kumawat, M. K., Kaur, R. &amp; Kumar, K. In-Silico prediction of novel fused Quinazoline based topoisomerase inhibitors as anticancer agents. <em>Med. Chem. (Shariqah (United Arab. Emirates))</em>. <strong>19</strong>, 431–444 (2023).</cite> [<a href="https://doi.org/10.2174/1573406418666221012161111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36237156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumawat,%20M.%20K.,%20Kaur,%20R.%20&amp;%20Kumar,%20K.%20In-Silico%20prediction%20of%20novel%20fused%20Quinazoline%20based%20topoisomerase%20inhibitors%20as%20anticancer%20agents.%20Med.%20Chem.%20(Shariqah%20(United%20Arab.%20Emirates)).%2019,%20431%E2%80%93444%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>In Silico Evaluation of Novel 2-Pyrazoline Carboxamide Derivatives as Potential Protease Inhibitors Against Plasmodium Parasites. in.</cite>
</li>
<li id="CR28">
<span class="label">28.</span><cite>Shawakfeh, K. Q., Al-Said, N. H. &amp; Al-Zoubi, R. M. Synthesis of bis-diosgenin pyrazine dimers: new cephalostatin analogs. <em>Steroids</em><strong>73</strong>, 579–584 (2008).
</cite> [<a href="https://doi.org/10.1016/j.steroids.2008.01.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18325555/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shawakfeh,%20K.%20Q.,%20Al-Said,%20N.%20H.%20&amp;%20Al-Zoubi,%20R.%20M.%20Synthesis%20of%20bis-diosgenin%20pyrazine%20dimers:%20new%20cephalostatin%20analogs.%20Steroids73,%20579%E2%80%93584%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Karulin, B. &amp; Kozhevnikov, M. Ketcher: web-based chemical structure editor. <em>J. Cheminform.</em><strong>3</strong>, P3 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Karulin,%20B.%20&amp;%20Kozhevnikov,%20M.%20Ketcher:%20web-based%20chemical%20structure%20editor.%20J.%20Cheminform.3,%20P3%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Sander, T., Freyss, J., Von Korff, M. &amp; Rufener, C. DataWarrior: an open-source program for chemistry aware data visualization and analysis. <em>J. Chem. Inf. Model.</em><strong>55</strong>, 460–473 (2015).
</cite> [<a href="https://doi.org/10.1021/ci500588j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25558886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sander,%20T.,%20Freyss,%20J.,%20Von%20Korff,%20M.%20&amp;%20Rufener,%20C.%20DataWarrior:%20an%20open-source%20program%20for%20chemistry%20aware%20data%20visualization%20and%20analysis.%20J.%20Chem.%20Inf.%20Model.55,%20460%E2%80%93473%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Bickerton, G. R., Paolini, G. V., Besnard, J., Muresan, S. &amp; Hopkins, A. L. Quantifying the chemical beauty of drugs. <em>Nat. Chem.</em><strong>4</strong>, 90–98 (2012).
</cite> [<a href="https://doi.org/10.1038/nchem.1243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3524573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22270643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bickerton,%20G.%20R.,%20Paolini,%20G.%20V.,%20Besnard,%20J.,%20Muresan,%20S.%20&amp;%20Hopkins,%20A.%20L.%20Quantifying%20the%20chemical%20beauty%20of%20drugs.%20Nat.%20Chem.4,%2090%E2%80%9398%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Xiong, G. et al. ADMETlab 2.0: an integrated online platform for accurate and comprehensive predictions of ADMET properties. <em>Nucleic Acids Res.</em><strong>49</strong>, W5–W14 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8262709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33893803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiong,%20G.%20et%20al.%20ADMETlab%202.0:%20an%20integrated%20online%20platform%20for%20accurate%20and%20comprehensive%20predictions%20of%20ADMET%20properties.%20Nucleic%20Acids%20Res.49,%20W5%E2%80%93W14%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Boyle, N. M. O. et al. Open babel: an open chemical toolbox. <em>J. Cheminform.</em><strong>3</strong>, 1–14 (2011).
</cite> [<a href="https://doi.org/10.1186/1758-2946-3-33" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3198950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21982300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boyle,%20N.%20M.%20O.%20et%20al.%20Open%20babel:%20an%20open%20chemical%20toolbox.%20J.%20Cheminform.3,%201%E2%80%9314%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. <em>J. Comput. Chem.</em><strong>25</strong>, 1605–1612 (2004).
</cite> [<a href="https://doi.org/10.1002/jcc.20084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15264254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pettersen,%20E.%20F.%20et%20al.%20UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis.%20J.%20Comput.%20Chem.25,%201605%E2%80%931612%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Peterson, M. R. et al. The three-dimensional structure of a <em>Plasmodium falciparum</em> Cyclophilin in complex with the potent anti-malarial cyclosporin A1. <em>J. Mol. Biol.</em><strong>298</strong>, 123–133 (2000).
</cite> [<a href="https://doi.org/10.1006/jmbi.2000.3633" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10756109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peterson,%20M.%20R.%20et%20al.%20The%20three-dimensional%20structure%20of%20a%20Plasmodium%20falciparum%20Cyclophilin%20in%20complex%20with%20the%20potent%20anti-malarial%20cyclosporin%20A1.%20J.%20Mol.%20Biol.298,%20123%E2%80%93133%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Eberhardt, J., Santos-Martins, D., Tillack, A. F. &amp; Forli, S. AutoDock Vina 1.2.0: new Docking methods, expanded force field, and python bindings. <em>J. Chem. Inf. Model.</em><strong>61</strong>, 3891–3898 (2021).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.1c00203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10683950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34278794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eberhardt,%20J.,%20Santos-Martins,%20D.,%20Tillack,%20A.%20F.%20&amp;%20Forli,%20S.%20AutoDock%20Vina%201.2.0:%20new%20Docking%20methods,%20expanded%20force%20field,%20and%20python%20bindings.%20J.%20Chem.%20Inf.%20Model.61,%203891%E2%80%933898%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Trott, O. &amp; Olson, A. J. AutoDock vina: improving the speed and accuracy of Docking with a new scoring function, efficient optimization, and multithreading. <em>J. Comput. Chem.</em><strong>NA-NA</strong>10.1002/jcc.21334 (2009).</cite> [<a href="https://doi.org/10.1002/jcc.21334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3041641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19499576/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Liu, N. &amp; Xu, Z. Using LeDock as a Docking tool for computational drug design. <em>IOP Conf. Ser.: Earth Environ. Sci.</em><strong>218</strong>, 012143 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Liu,%20N.%20&amp;%20Xu,%20Z.%20Using%20LeDock%20as%20a%20Docking%20tool%20for%20computational%20drug%20design.%20IOP%20Conf.%20Ser.:%20Earth%20Environ.%20Sci.218,%20012143%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Alhossary, A., Handoko, S. D., Mu, Y. &amp; Kwoh, C. K. Fast, accurate, and reliable molecular Docking with QuickVina 2. <em>Bioinformatics</em><strong>31</strong>, 2214–2216 (2015).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25717194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alhossary,%20A.,%20Handoko,%20S.%20D.,%20Mu,%20Y.%20&amp;%20Kwoh,%20C.%20K.%20Fast,%20accurate,%20and%20reliable%20molecular%20Docking%20with%20QuickVina%202.%20Bioinformatics31,%202214%E2%80%932216%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Morris, G. M. et al. AutoDock4 and AutoDockTools4: automated Docking with selective receptor flexibility. <em>J. Comput. Chem.</em><strong>30</strong>, 2785–2791 (2009).
</cite> [<a href="https://doi.org/10.1002/jcc.21256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2760638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19399780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morris,%20G.%20M.%20et%20al.%20AutoDock4%20and%20AutoDockTools4:%20automated%20Docking%20with%20selective%20receptor%20flexibility.%20J.%20Comput.%20Chem.30,%202785%E2%80%932791%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Li, H., S Leung, K. &amp; H Wong, M. Idock: A multithreaded virtual screening tool for flexible ligand Docking. <em>2012 IEEE Symp. Comput. Intell. Comput. Biology CIBCB 2012</em>. <strong>77-84</strong>10.1109/CIBCB.2012.6217214 (2012).</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Koes, D. R., Baumgartner, M. P. &amp; Camacho, C. J. Lessons learned in empirical scoring with Smina from the CSAR 2011 benchmarking exercise. <em>J. Chem. Inf. Model.</em><strong>53</strong>, 1893–1904 (2013).
</cite> [<a href="https://doi.org/10.1021/ci300604z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3726561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23379370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koes,%20D.%20R.,%20Baumgartner,%20M.%20P.%20&amp;%20Camacho,%20C.%20J.%20Lessons%20learned%20in%20empirical%20scoring%20with%20Smina%20from%20the%20CSAR%202011%20benchmarking%20exercise.%20J.%20Chem.%20Inf.%20Model.53,%201893%E2%80%931904%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Debnath, A., Mazumder, R., Singh, A. K. &amp; Singh, R. K. Identification of novel cyclin-dependent kinase 4/6 inhibitors from marine natural products. <em>PLoS One</em>. <strong>20</strong>, e0313830 (2025).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0313830" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11734976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39813224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Debnath,%20A.,%20Mazumder,%20R.,%20Singh,%20A.%20K.%20&amp;%20Singh,%20R.%20K.%20Identification%20of%20novel%20cyclin-dependent%20kinase%204/6%20inhibitors%20from%20marine%20natural%20products.%20PLoS%20One.%2020,%20e0313830%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Debnath, A. <em>et al. Quest for Discovering Novel CDK12 Inhibitor by Leveraging High-Throughput Virtual Screening</em>. <a href="https://www.researchsquare.com/article/rs-3382004/v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.researchsquare.com/article/rs-3382004/v1</a> (2023). 10.21203/rs.3.rs-3382004/v1</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite>Hunter, J. D. &amp; Matplotlib A 2D graphics environment. <em>Comput. Sci. Eng.</em><strong>9</strong>, 90–95 (2007).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hunter,%20J.%20D.%20&amp;%20Matplotlib%20A%202D%20graphics%20environment.%20Comput.%20Sci.%20Eng.9,%2090%E2%80%9395%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>The pandas development team. pandas-dev/pandas: Pandas. [object Object] (2024). 10.5281/ZENODO.3509134</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite>Harris, C. R. et al. Array programming with numpy. <em>Nature</em><strong>585</strong>, 357–362 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2649-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7759461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32939066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harris,%20C.%20R.%20et%20al.%20Array%20programming%20with%20numpy.%20Nature585,%20357%E2%80%93362%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Van Der Spoel, D. et al. GROMACS: fast, flexible, and free. <em>J. Comput. Chem.</em><strong>26</strong>, 1701–1718 (2005).
</cite> [<a href="https://doi.org/10.1002/jcc.20291" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16211538/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Der%20Spoel,%20D.%20et%20al.%20GROMACS:%20fast,%20flexible,%20and%20free.%20J.%20Comput.%20Chem.26,%201701%E2%80%931718%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Debnath, A. et al. Discovery of Novel PTP1B Inhibitors by High-throughput Virtual Screening. <em>CAD</em> 21, (2024).</cite> [<a href="https://doi.org/10.2174/0115734099278007241004105500" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39411941/" class="usa-link">PubMed</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Debnath, A., Mazumder, R., Singh, R. K. &amp; Singh, A. K. Discovery of novel CDK4/6 inhibitors from fungal secondary metabolites. <em>Int. J. Biol. Macromol.</em><strong>282</strong>, 136807 (2024).
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2024.136807" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39447792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Debnath,%20A.,%20Mazumder,%20R.,%20Singh,%20R.%20K.%20&amp;%20Singh,%20A.%20K.%20Discovery%20of%20novel%20CDK4/6%20inhibitors%20from%20fungal%20secondary%20metabolites.%20Int.%20J.%20Biol.%20Macromol.282,%20136807%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Huang, J. &amp; Mackerell, A. D. CHARMM36 all-atom additive protein force field: validation based on comparison to NMR data. <em>J. Comput. Chem.</em><strong>34</strong>, 2135–2145 (2013).
</cite> [<a href="https://doi.org/10.1002/jcc.23354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3800559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23832629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20J.%20&amp;%20Mackerell,%20A.%20D.%20CHARMM36%20all-atom%20additive%20protein%20force%20field:%20validation%20based%20on%20comparison%20to%20NMR%20data.%20J.%20Comput.%20Chem.34,%202135%E2%80%932145%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Huang, J. et al. CHARMM36m: an improved force field for folded and intrinsically disordered proteins. <em>Nat. Methods</em>. <strong>14</strong>, 71–73 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5199616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27819658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20J.%20et%20al.%20CHARMM36m:%20an%20improved%20force%20field%20for%20folded%20and%20intrinsically%20disordered%20proteins.%20Nat.%20Methods.%2014,%2071%E2%80%9373%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Vanommeslaeghe, K. &amp; MacKerell, A. D. Automation of the CHARMM general force field (CGenFF) I: bond perception and atom typing. <em>J. Chem. Inf. Model.</em><strong>52</strong>, 3144–3154 (2012).
</cite> [<a href="https://doi.org/10.1021/ci300363c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3528824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23146088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vanommeslaeghe,%20K.%20&amp;%20MacKerell,%20A.%20D.%20Automation%20of%20the%20CHARMM%20general%20force%20field%20(CGenFF)%20I:%20bond%20perception%20and%20atom%20typing.%20J.%20Chem.%20Inf.%20Model.52,%203144%E2%80%933154%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Vanommeslaeghe, K. et al. CHARMM general force field: a force field for drug-like molecules compatible with the CHARMM all‐atom additive biological force fields. <em>J. Comput. Chem.</em><strong>31</strong>, 671–690 (2010).
</cite> [<a href="https://doi.org/10.1002/jcc.21367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2888302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19575467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vanommeslaeghe,%20K.%20et%20al.%20CHARMM%20general%20force%20field:%20a%20force%20field%20for%20drug-like%20molecules%20compatible%20with%20the%20CHARMM%20all%E2%80%90atom%20additive%20biological%20force%20fields.%20J.%20Comput.%20Chem.31,%20671%E2%80%93690%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Izadi, S., Anandakrishnan, R. &amp; Onufriev, A. V. Building water models: a different approach. <em>J. Phys. Chem. Lett.</em><strong>5</strong>, 3863–3871 (2014).
</cite> [<a href="https://doi.org/10.1021/jz501780a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4226301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25400877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Izadi,%20S.,%20Anandakrishnan,%20R.%20&amp;%20Onufriev,%20A.%20V.%20Building%20water%20models:%20a%20different%20approach.%20J.%20Phys.%20Chem.%20Lett.5,%203863%E2%80%933871%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Price, D. J. &amp; Brooks, C. L. A modified TIP3P water potential for simulation with Ewald summation. <em>J. Chem. Phys.</em><strong>121</strong>, 10096–10103 (2004).
</cite> [<a href="https://doi.org/10.1063/1.1808117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15549884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Price,%20D.%20J.%20&amp;%20Brooks,%20C.%20L.%20A%20modified%20TIP3P%20water%20potential%20for%20simulation%20with%20Ewald%20summation.%20J.%20Chem.%20Phys.121,%2010096%E2%80%9310103%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Boonstra, S., Onck, P. R. &amp; Van Der Giessen, E. CHARMM TIP3P water model suppresses peptide folding by solvating the unfolded state. <em>J. Phys. Chem. B</em>. <strong>120</strong>, 3692–3698 (2016).
</cite> [<a href="https://doi.org/10.1021/acs.jpcb.6b01316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27031562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boonstra,%20S.,%20Onck,%20P.%20R.%20&amp;%20Van%20Der%20Giessen,%20E.%20CHARMM%20TIP3P%20water%20model%20suppresses%20peptide%20folding%20by%20solvating%20the%20unfolded%20state.%20J.%20Phys.%20Chem.%20B.%20120,%203692%E2%80%933698%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Michaud-Agrawal, N., Denning, E. J., Woolf, T. B. &amp; Beckstein, O. MDAnalysis: a toolkit for the analysis of molecular dynamics simulations. <em>J. Comput. Chem.</em><strong>32</strong>, 2319–2327 (2011).
</cite> [<a href="https://doi.org/10.1002/jcc.21787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3144279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21500218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michaud-Agrawal,%20N.,%20Denning,%20E.%20J.,%20Woolf,%20T.%20B.%20&amp;%20Beckstein,%20O.%20MDAnalysis:%20a%20toolkit%20for%20the%20analysis%20of%20molecular%20dynamics%20simulations.%20J.%20Comput.%20Chem.32,%202319%E2%80%932327%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Cock, P. J. A. et al. Biopython: freely available python tools for computational molecular biology and bioinformatics. <em>Bioinformatics</em><strong>25</strong>, 1422–1423 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2682512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19304878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cock,%20P.%20J.%20A.%20et%20al.%20Biopython:%20freely%20available%20python%20tools%20for%20computational%20molecular%20biology%20and%20bioinformatics.%20Bioinformatics25,%201422%E2%80%931423%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Rego, N. &amp; Koes, D. 3Dmol.js: molecular visualization with WebGL. <em>Bioinformatics</em><strong>31</strong>, 1322–1324 (2015).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4393526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25505090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rego,%20N.%20&amp;%20Koes,%20D.%203Dmol.js:%20molecular%20visualization%20with%20WebGL.%20Bioinformatics31,%201322%E2%80%931324%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Bouysset, C. &amp; Fiorucci, S. ProLIF: a library to encode molecular interactions as fingerprints. <em>J. Cheminform.</em><strong>13</strong>, 72 (2021).
</cite> [<a href="https://doi.org/10.1186/s13321-021-00548-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8466659/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34563256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bouysset,%20C.%20&amp;%20Fiorucci,%20S.%20ProLIF:%20a%20library%20to%20encode%20molecular%20interactions%20as%20fingerprints.%20J.%20Cheminform.13,%2072%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Rose, A. S. et al. NGL viewer: web-based molecular graphics for large complexes. <em>Bioinformatics</em><strong>34</strong>, 3755–3758 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6198858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29850778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rose,%20A.%20S.%20et%20al.%20NGL%20viewer:%20web-based%20molecular%20graphics%20for%20large%20complexes.%20Bioinformatics34,%203755%E2%80%933758%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Daina, A., Michielin, O. &amp; Zoete, V. SwissADME: a free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules. <em>Sci. Rep.</em><strong>7</strong>, 42717 (2017).
</cite> [<a href="https://doi.org/10.1038/srep42717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5335600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28256516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daina,%20A.,%20Michielin,%20O.%20&amp;%20Zoete,%20V.%20SwissADME:%20a%20free%20web%20tool%20to%20evaluate%20pharmacokinetics,%20drug-likeness%20and%20medicinal%20chemistry%20friendliness%20of%20small%20molecules.%20Sci.%20Rep.7,%2042717%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Banerjee, P., Kemmler, E., Dunkel, M. &amp; Preissner, R. ProTox 3.0: a webserver for the prediction of toxicity of chemicals. <em>Nucleic Acids Res.</em><strong>52</strong>, W513–W520 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkae303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38647086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banerjee,%20P.,%20Kemmler,%20E.,%20Dunkel,%20M.%20&amp;%20Preissner,%20R.%20ProTox%203.0:%20a%20webserver%20for%20the%20prediction%20of%20toxicity%20of%20chemicals.%20Nucleic%20Acids%20Res.52,%20W513%E2%80%93W520%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite><a href="https://www.who.int/publications/i/item/9789240031357" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.who.int/publications/i/item/9789240031357</a></cite>
</li>
<li id="CR66">
<span class="label">66.</span><cite>Wahajuddin, M. et al. Simultaneous quantification of proposed anti-malarial combination comprising of lumefantrine and CDRI 97–78 in rat plasma using the HPLC–ESI-MS/MS method: application to drug interaction study. <em>Malar. J.</em><strong>14</strong>, 172 (2015).
</cite> [<a href="https://doi.org/10.1186/s12936-015-0684-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4429480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25895956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wahajuddin,%20M.%20et%20al.%20Simultaneous%20quantification%20of%20proposed%20anti-malarial%20combination%20comprising%20of%20lumefantrine%20and%20CDRI%2097%E2%80%9378%20in%20rat%20plasma%20using%20the%20HPLC%E2%80%93ESI-MS/MS%20method:%20application%20to%20drug%20interaction%20study.%20Malar.%20J.14,%20172%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Valecha, N. et al. Arterolane maleate plus Piperaquine phosphate for treatment of uncomplicated plasmodium falciparum malaria: a comparative, multicenter, randomized clinical trial. <em>Clin. Infect. Dis.</em><strong>55</strong>, 663–671 (2012).
</cite> [<a href="https://doi.org/10.1093/cid/cis475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22586253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Valecha,%20N.%20et%20al.%20Arterolane%20maleate%20plus%20Piperaquine%20phosphate%20for%20treatment%20of%20uncomplicated%20plasmodium%20falciparum%20malaria:%20a%20comparative,%20multicenter,%20randomized%20clinical%20trial.%20Clin.%20Infect.%20Dis.55,%20663%E2%80%93671%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Zech, J. et al. Oral administration of Artemisone for the treatment of schistosomiasis: formulation challenges and in vivo efficacy. <em>Pharmaceutics</em><strong>12</strong>, 509 (2020).
</cite> [<a href="https://doi.org/10.3390/pharmaceutics12060509" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7356104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32503130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zech,%20J.%20et%20al.%20Oral%20administration%20of%20Artemisone%20for%20the%20treatment%20of%20schistosomiasis:%20formulation%20challenges%20and%20in%20vivo%20efficacy.%20Pharmaceutics12,%20509%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Haynes, R. K. et al. Artemisone—a highly active antimalarial drug of the Artemisinin class. <em>Angew. Chem. Int. Ed.</em><strong>45</strong>, 2082–2088 (2006).</cite> [<a href="https://doi.org/10.1002/anie.200503071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16444785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haynes,%20R.%20K.%20et%20al.%20Artemisone%E2%80%94a%20highly%20active%20antimalarial%20drug%20of%20the%20Artemisinin%20class.%20Angew.%20Chem.%20Int.%20Ed.45,%202082%E2%80%932088%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Phyo, A. P. et al. Antimalarial activity of artefenomel (OZ439), a novel synthetic antimalarial endoperoxide, in patients with plasmodium falciparum and plasmodium Vivax malaria: an open-label phase 2 trial. <em>Lancet Infect. Dis.</em><strong>16</strong>, 61–69 (2016).
</cite> [<a href="https://doi.org/10.1016/S1473-3099(15)00320-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4700386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26448141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Phyo,%20A.%20P.%20et%20al.%20Antimalarial%20activity%20of%20artefenomel%20(OZ439),%20a%20novel%20synthetic%20antimalarial%20endoperoxide,%20in%20patients%20with%20plasmodium%20falciparum%20and%20plasmodium%20Vivax%20malaria:%20an%20open-label%20phase%202%20trial.%20Lancet%20Infect.%20Dis.16,%2061%E2%80%9369%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Brit, J. C. <em>Pharma – 2012 - Moehrle - First-in-man Safety and Pharmacokinetics of Synthetic Ozonide OZ439 Demonstrates</em>.</cite> [<a href="https://doi.org/10.1111/j.1365-2125.2012.04368.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3558805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22759078/" class="usa-link">PubMed</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Sashidhara, K. V. et al. Coumarin-trioxane hybrids: synthesis and evaluation as a new class of antimalarial scaffolds. <em>Bioorg. Med. Chem. Lett.</em><strong>22</strong>, 3926–3930 (2012).
</cite> [<a href="https://doi.org/10.1016/j.bmcl.2012.04.100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22607674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sashidhara,%20K.%20V.%20et%20al.%20Coumarin-trioxane%20hybrids:%20synthesis%20and%20evaluation%20as%20a%20new%20class%20of%20antimalarial%20scaffolds.%20Bioorg.%20Med.%20Chem.%20Lett.22,%203926%E2%80%933930%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Lill, M. A. Efficient incorporation of protein flexibility and dynamics into molecular docking simulations. <em>Biochemistry</em><strong>50</strong>, 6157–6169 (2011).
</cite> [<a href="https://doi.org/10.1021/bi2004558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3172316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21678954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lill,%20M.%20A.%20Efficient%20incorporation%20of%20protein%20flexibility%20and%20dynamics%20into%20molecular%20docking%20simulations.%20Biochemistry50,%206157%E2%80%936169%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Attram, H. D., Korkor, C. M., Taylor, D., Njoroge, M. &amp; Chibale, K. Antimalarial imidazopyridines incorporating an intramolecular hydrogen bonding motif: medicinal chemistry and mechanistic studies. <em>ACS Infect. Dis.</em><strong>9</strong>, 928–942 (2023).
</cite> [<a href="https://doi.org/10.1021/acsinfecdis.2c00584" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10111423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36946433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Attram,%20H.%20D.,%20Korkor,%20C.%20M.,%20Taylor,%20D.,%20Njoroge,%20M.%20&amp;%20Chibale,%20K.%20Antimalarial%20imidazopyridines%20incorporating%20an%20intramolecular%20hydrogen%20bonding%20motif:%20medicinal%20chemistry%20and%20mechanistic%20studies.%20ACS%20Infect.%20Dis.9,%20928%E2%80%93942%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Zhan, W. et al. Dual-pharmacophore artezomibs hijack the plasmodium ubiquitin-proteasome system to kill malaria parasites while overcoming drug resistance. <em>Cell. Chem. Biol</em>. <strong>30</strong>, 457–469e11 (2023).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2023.04.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10240386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37148884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhan,%20W.%20et%20al.%20Dual-pharmacophore%20artezomibs%20hijack%20the%20plasmodium%20ubiquitin-proteasome%20system%20to%20kill%20malaria%20parasites%20while%20overcoming%20drug%20resistance.%20Cell.%20Chem.%20Biol.%2030,%20457%E2%80%93469e11%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Mehta, M. et al. Lipid-based nanoparticles for drug/gene delivery: an overview of the production techniques and difficulties encountered in their industrial development. <em>ACS Mater. Au</em>. <strong>3</strong>, 600–619 (2023).
</cite> [<a href="https://doi.org/10.1021/acsmaterialsau.3c00032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10636777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38089666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mehta,%20M.%20et%20al.%20Lipid-based%20nanoparticles%20for%20drug/gene%20delivery:%20an%20overview%20of%20the%20production%20techniques%20and%20difficulties%20encountered%20in%20their%20industrial%20development.%20ACS%20Mater.%20Au.%203,%20600%E2%80%93619%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Wu, D. et al. The blood–brain barrier: structure, regulation and drug delivery. <em>Signal. Transduct. Target. Therapy</em>. <strong>8</strong>, 217 (2023).</cite> [<a href="https://doi.org/10.1038/s41392-023-01481-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10212980/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37231000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20D.%20et%20al.%20The%20blood%E2%80%93brain%20barrier:%20structure,%20regulation%20and%20drug%20delivery.%20Signal.%20Transduct.%20Target.%20Therapy.%208,%20217%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365323/bin/41598_2025_13017_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (14.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365323/bin/41598_2025_13017_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (16.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365323/bin/41598_2025_13017_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (11.2KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data generated or analysed during this study are included in this published article and its supplementary information files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-13017-z"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_13017.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365323/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365323/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365323%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365323/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365323/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365323/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830358/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365323/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830358/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365323/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365323/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="IgSen5imhQm0mnXZnLlxCN3jHEQ2COAQQdiociRtRPGONDOoG87qWY7iLtNMNdyo">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
