
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Charting the spatial transcriptome of the human cerebral cortex at single-cell resolution - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152548238AF35A6305482300294EC14D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365125/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Charting the spatial transcriptome of the human cerebral cortex at single-cell resolution">
<meta name="citation_author" content="Songren Wei">
<meta name="citation_author_institution" content="School of Interdisciplinary Medicine and Engineering, Harbin Medical University, Harbin, 150076 China">
<meta name="citation_author_institution" content="NMPA Key Laboratory for Research and Evaluation of Drug Metabolism&amp; Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou, China">
<meta name="citation_author_institution" content="Center for Brain Science and Brain-Inspired Intelligence, Guangdong-Hong Kong-Macao Greater Bay Area, Guangzhou, 510515 China">
<meta name="citation_author" content="Meng Luo">
<meta name="citation_author_institution" content="School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China">
<meta name="citation_author" content="Pingping Wang">
<meta name="citation_author_institution" content="School of Interdisciplinary Medicine and Engineering, Harbin Medical University, Harbin, 150076 China">
<meta name="citation_author" content="Rui Chen">
<meta name="citation_author_institution" content="Department of Forensic Medicine, Guangdong Medical University, Dongguan, 523808 China">
<meta name="citation_author" content="Xiyun Jin">
<meta name="citation_author_institution" content="School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China">
<meta name="citation_author" content="Chang Xu">
<meta name="citation_author_institution" content="School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China">
<meta name="citation_author" content="Chenyang Li">
<meta name="citation_author_institution" content="Center for Brain Science and Brain-Inspired Intelligence, Guangdong-Hong Kong-Macao Greater Bay Area, Guangzhou, 510515 China">
<meta name="citation_author" content="Xiaoyu Lin">
<meta name="citation_author_institution" content="School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China">
<meta name="citation_author" content="Zhaochun Xu">
<meta name="citation_author_institution" content="School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China">
<meta name="citation_author" content="Hongxin Liu">
<meta name="citation_author_institution" content="School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China">
<meta name="citation_author" content="Rui Cheng">
<meta name="citation_author_institution" content="School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China">
<meta name="citation_author" content="Wenyi Yang">
<meta name="citation_author_institution" content="School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China">
<meta name="citation_author" content="Yideng Cai">
<meta name="citation_author_institution" content="School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China">
<meta name="citation_author" content="Guangfu Xue">
<meta name="citation_author_institution" content="School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China">
<meta name="citation_author" content="Peng Huang">
<meta name="citation_author_institution" content="NMPA Key Laboratory for Research and Evaluation of Drug Metabolism&amp; Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou, China">
<meta name="citation_author_institution" content="Foshan Maternity &amp; Child Healthcare Hospital, Southern Medical University, Foshan, 528000 China">
<meta name="citation_author" content="Zhigang Liu">
<meta name="citation_author_institution" content="Foshan Maternity &amp; Child Healthcare Hospital, Southern Medical University, Foshan, 528000 China">
<meta name="citation_author" content="Haoxiu Sun">
<meta name="citation_author_institution" content="School of Interdisciplinary Medicine and Engineering, Harbin Medical University, Harbin, 150076 China">
<meta name="citation_author" content="Jiangping Xu">
<meta name="citation_author_institution" content="NMPA Key Laboratory for Research and Evaluation of Drug Metabolism&amp; Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou, China">
<meta name="citation_author_institution" content="Center for Brain Science and Brain-Inspired Intelligence, Guangdong-Hong Kong-Macao Greater Bay Area, Guangzhou, 510515 China">
<meta name="citation_author_institution" content="Key Laboratory of Mental Health of the Ministry of Education, Southern Medical University, Guangzhou, 510515 China">
<meta name="citation_author" content="Qinghua Jiang">
<meta name="citation_author_institution" content="School of Interdisciplinary Medicine and Engineering, Harbin Medical University, Harbin, 150076 China">
<meta name="citation_author_institution" content="School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China">
<meta name="citation_author_institution" content="State Key Laboratory of Frigid Zone Cardiovascular Diseases (SKLFZCD), Harbin Medical University, Harbin, 150076 China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7702">
<meta name="citation_doi" content="10.1038/s41467-025-62793-9">
<meta name="citation_pmid" content="40830090">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365125/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365125/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365125/pdf/41467_2025_Article_62793.pdf">
<meta name="description" content="A high-resolution spatial physiological atlas of cortical neurons serves as an essential reference for studying neurological diseases and is crucial for a comprehensive understanding of the cortical functions and physiological mechanisms. Although ...">
<meta name="og:title" content="Charting the spatial transcriptome of the human cerebral cortex at single-cell resolution">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="A high-resolution spatial physiological atlas of cortical neurons serves as an essential reference for studying neurological diseases and is crucial for a comprehensive understanding of the cortical functions and physiological mechanisms. Although ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365125/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365125">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62793-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62793.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365125%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365125/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365125/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365125/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7702. doi: <a href="https://doi.org/10.1038/s41467-025-62793-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62793-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Charting the spatial transcriptome of the human cerebral cortex at single-cell resolution</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Songren Wei</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Songren Wei</span></h3>
<div class="p">
<sup>1</sup>School of Interdisciplinary Medicine and Engineering, Harbin Medical University, Harbin, 150076 China </div>
<div class="p">
<sup>2</sup>NMPA Key Laboratory for Research and Evaluation of Drug Metabolism&amp; Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou, China </div>
<div class="p">
<sup>3</sup>Center for Brain Science and Brain-Inspired Intelligence, Guangdong-Hong Kong-Macao Greater Bay Area, Guangzhou, 510515 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Songren Wei</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Meng Luo</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Meng Luo</span></h3>
<div class="p">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Meng Luo</span></a>
</div>
</div>
<sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Pingping Wang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Pingping Wang</span></h3>
<div class="p">
<sup>1</sup>School of Interdisciplinary Medicine and Engineering, Harbin Medical University, Harbin, 150076 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pingping Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Rui Chen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Rui Chen</span></h3>
<div class="p">
<sup>5</sup>Department of Forensic Medicine, Guangdong Medical University, Dongguan, 523808 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rui Chen</span></a>
</div>
</div>
<sup>5,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jin%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Xiyun Jin</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Xiyun Jin</span></h3>
<div class="p">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jin%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiyun Jin</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Chang Xu</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Chang Xu</span></h3>
<div class="p">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chang Xu</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Chenyang Li</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Chenyang Li</span></h3>
<div class="p">
<sup>3</sup>Center for Brain Science and Brain-Inspired Intelligence, Guangdong-Hong Kong-Macao Greater Bay Area, Guangzhou, 510515 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chenyang Li</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Xiaoyu Lin</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Xiaoyu Lin</span></h3>
<div class="p">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoyu Lin</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Zhaochun Xu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Zhaochun Xu</span></h3>
<div class="p">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhaochun Xu</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Hongxin Liu</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Hongxin Liu</span></h3>
<div class="p">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hongxin Liu</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Rui Cheng</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Rui Cheng</span></h3>
<div class="p">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rui Cheng</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Wenyi Yang</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Wenyi Yang</span></h3>
<div class="p">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenyi Yang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Yideng Cai</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Yideng Cai</span></h3>
<div class="p">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yideng Cai</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xue%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Guangfu Xue</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Guangfu Xue</span></h3>
<div class="p">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xue%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guangfu Xue</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Peng Huang</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Peng Huang</span></h3>
<div class="p">
<sup>2</sup>NMPA Key Laboratory for Research and Evaluation of Drug Metabolism&amp; Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou, China </div>
<div class="p">
<sup>6</sup>Foshan Maternity &amp; Child Healthcare Hospital, Southern Medical University, Foshan, 528000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peng Huang</span></a>
</div>
</div>
<sup>2,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Zhigang Liu</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Zhigang Liu</span></h3>
<div class="p">
<sup>6</sup>Foshan Maternity &amp; Child Healthcare Hospital, Southern Medical University, Foshan, 528000 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhigang Liu</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Haoxiu Sun</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Haoxiu Sun</span></h3>
<div class="p">
<sup>1</sup>School of Interdisciplinary Medicine and Engineering, Harbin Medical University, Harbin, 150076 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haoxiu Sun</span></a>
</div>
</div>
<sup>1,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Jiangping Xu</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Jiangping Xu</span></h3>
<div class="p">
<sup>2</sup>NMPA Key Laboratory for Research and Evaluation of Drug Metabolism&amp; Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou, China </div>
<div class="p">
<sup>3</sup>Center for Brain Science and Brain-Inspired Intelligence, Guangdong-Hong Kong-Macao Greater Bay Area, Guangzhou, 510515 China </div>
<div class="p">
<sup>7</sup>Key Laboratory of Mental Health of the Ministry of Education, Southern Medical University, Guangzhou, 510515 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiangping Xu</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>7,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Qinghua Jiang</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Qinghua Jiang</span></h3>
<div class="p">
<sup>1</sup>School of Interdisciplinary Medicine and Engineering, Harbin Medical University, Harbin, 150076 China </div>
<div class="p">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div class="p">
<sup>8</sup>State Key Laboratory of Frigid Zone Cardiovascular Diseases (SKLFZCD), Harbin Medical University, Harbin, 150076 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qinghua Jiang</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>8,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Interdisciplinary Medicine and Engineering, Harbin Medical University, Harbin, 150076 China </div>
<div id="Aff2">
<sup>2</sup>NMPA Key Laboratory for Research and Evaluation of Drug Metabolism&amp; Guangdong Provincial Key Laboratory of New Drug Screening, School of Pharmaceutical Sciences, Southern Medical University, Guangzhou, China </div>
<div id="Aff3">
<sup>3</sup>Center for Brain Science and Brain-Inspired Intelligence, Guangdong-Hong Kong-Macao Greater Bay Area, Guangzhou, 510515 China </div>
<div id="Aff4">
<sup>4</sup>School of Life Science and Technology, Harbin Institute of Technology, Harbin, 150080 China </div>
<div id="Aff5">
<sup>5</sup>Department of Forensic Medicine, Guangdong Medical University, Dongguan, 523808 China </div>
<div id="Aff6">
<sup>6</sup>Foshan Maternity &amp; Child Healthcare Hospital, Southern Medical University, Foshan, 528000 China </div>
<div id="Aff7">
<sup>7</sup>Key Laboratory of Mental Health of the Ministry of Education, Southern Medical University, Guangzhou, 510515 China </div>
<div id="Aff8">
<sup>8</sup>State Key Laboratory of Frigid Zone Cardiovascular Diseases (SKLFZCD), Harbin Medical University, Harbin, 150076 China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 9; Accepted 2025 Jul 30; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365125Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830090/" class="usa-link">40830090</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">A high-resolution spatial physiological atlas of cortical neurons serves as an essential reference for studying neurological diseases and is crucial for a comprehensive understanding of the cortical functions and physiological mechanisms. Although previous studies have elucidated the cellular basis of individual cortex, the molecular characteristics and precise spatial organization of neural cells within multiple human cortex remain incompletely understood. Subcellular-level spatial transcriptomic sequencing and snRNA-seq enabled mapping of 14 human cortical regions, creating a single-cell atlas with transcriptomic data from 1,121,772 nuclei and spatial profiles from 1,888,306 cells to characterize neural cell organization. The atlas reveals distinct expression patterns and spatial arrangements of neural cell types. Glutamatergic neurons show precise laminar patterns, with similar expression in adjacent cortex. SST neurons fall into two transcriptional categories, corresponding to superficial and deep layer distributions. The atlas, integrated with functional networks, highlights correlations between neural cell types and cortical functions, uncovering cell-cell interactions and ligand-receptor patterns with regional differences in neuron-glia communication. It also deciphers transcriptomic differences and cellular composition in layer 4 and the stable subplate (layer 6b) across regions. Our findings offer insights into the cellular foundations of complex and intelligent regions within the human cortex.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Neuroscience, Genetics of the nervous system</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Human cortical functions rely on intricate spatial arrangements and interactions among neuronal cell types. Here, authors show a comprehensive cellular atlas illustrating detailed neuron distribution and communication patterns across cortical regions.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Diverse high cognitive functions were supported by cellular circuits formed by billion neural cells in different cerebral cortex regions<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>â€“<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. High-resolution â€œcytoarchitectural atlasâ€ of the human cerebral cortex is pivotal for understanding the mechanism of the cortical region performing specific functions<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>â€“<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Substantial application of many recent techniques has revealed the high diversity of neuronal cells, providing an unprecedented opportunity for the unbiased exploration of neural cell types<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>â€“<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. By establishing a physiological atlas of cortical neural cells, our work serves as a powerful reference for future neuro-disease cell atlases<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Furthermore, it enables the identification of taxonomic and spatial heterogeneity in certain cortical neurons during the pathological progression of neurological diseases<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>.</p>
<p id="Par4">In this study, we created a comprehensive cytoarchitectural atlas of the adult human cerebral cortex, encompassing cellular diversity and spatial structure. Using snRNA-seq and Subcellular-level spatial transcriptomics, we mapped this diversity across various cortical regions. We sequenced 307,738 nuclei in 14 cortical regions and aligned and merged the published BICCN neocortex data to obtain 1,121,772 nuclei. Alongside this, we carried out spatial transcriptome sequencing of these cortical regions, incorporating 44 Stereo slides. Our findings revealed that transcriptome-defined cortical L5 extratelencephalic neurons in the anterior cingulate had a similar density compared to other pain-related cortical regions and significantly more cell counting than non-pain cortical regions. This underlines how the combined use of these omics techniques can elucidate the vast variety of cell types and their spatial distributions within the human cortex. Meanwhile, to explore the correlation between the cellular architecture of various cortical regions and their associated functions, we overlaid datasets of cortical region function assessed by functional magnetic resonance imaging (fMRI) with our transcriptional cortical atlas<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. We delineated the cell-type function preference by calculating the correlation between cortical functions and cell-type specificity, providing references for the neural cellular basis of functional specificity of cortical regions. Besides, high-resolution Stereo-seq technology on spatial relationships between cells allows us to predict cell-cell interactions resulting from somatic contacts or paracrine signaling. The cellular atlas of the adult human cortex also empowers us to decipher the cytoarchitecture of layer 4 and cortical subplate layer 6b in 14 cortical regions. Together, our comprehensive atlas of cellular diversity and spatial arrangement in the human cortex provided the molecular and cellular basis of human intelligence.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Transcriptomic cell-type taxonomy in the human cerebral cortex</h3>
<p id="Par5">To understand the cellular diversity in the human cerebral cortex, we conducted snRNA-seq on 42 samples from 5 adult donors without neurological disorders (Fig.Â <a href="#Fig1" class="usa-link">1a</a>) and sequenced 307,738 nuclei in 14 cortical regions. We then combined our data with published BICCN neocortex data, which covered 8 of the regions included in our 14 cortical regions, resulting in a total of 1,121,772 nuclei. Subsequently, in order to expand the snRNA-seq dataset and better discover the heterogeneity of cortical regions, we merged the BICCN published dataset and unified it at the taxonomic granularity (Fig.Â <a href="#Fig1" class="usa-link">1b</a> and <a href="#MOESM1" class="usa-link">Supplementary Figs.</a>). Further to uncover the cortical spatial heterogeneity, we performed spatial transcriptome sequencing on 44 Stereo slides. Our dataset encompassed a broad spectrum of 14 cortical regions, including the frontopolar prefrontal cortex (FPPFC), dorsolateral prefrontal cortex (DLPFC), ventrolateral prefrontal cortex (VLPFC), primary motor cortex (M1), anterior cingulate cortex (ACC), primary somatosensory cortex (S1), primary somatosensory cortex of the eardrum (S1E), superior temporal gyrus (STG), inferior temporal gyrus (ITG), postcentral gyrus (PoCG), supramarginal gyrus (SMG), superior parietal lobule (SPL), angular gyrus (AG), and primary visual cortex (V1) (Fig.Â <a href="#Fig1" class="usa-link">1c</a>, Supplementary DatasetÂ <a href="#MOESM3" class="usa-link">1</a>). These were obtained from 9 donors across 14 cortical regions. The median cell number per slide of Stereo-seq was 42916â€‰Â±â€‰16007 (Supplementary DatasetÂ <a href="#MOESM3" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Comprehensive spatial transcriptomic atlas of the 14 human cortical regions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365125_41467_2025_62793_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7368/12365125/7d3ee349799f/41467_2025_62793_Fig1_HTML.jpg" loading="lazy" id="d33e434" height="854" width="796" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Overview of sampled cortical regions rendered in coronal sections and schematic workflow of snRNA and ST sequencing and analysis. We obtain a comprehensive neural cell census and localize various cell types by the 10X Genomics Chromium snRNA-seq and Stereo spatial transcriptomics technologies. Brain schematic diagram: Created using resources from CC0 public domain image, source: Open Clipart. Coronal brain section: Modified from Allen Institute for Brain Science, available at <a href="https://human.brain-map.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://human.brain-map.org</a>. <sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. <strong>b</strong> UMAP representation of all subclasses in BICCN published neocortex datasets (upper left); UMAP representation of all subclasses in snRNA-seq (upper right); UMAP of subclasses in Merged dataset (lower left); UMAP of all layers in Stereo-seq. Subclasses in all figures are uniformly marked with different colors. <strong>c</strong> The coronal slides cover the following 14 cortical regions by using Stereo-seq. Colors denoted different coronal cortical sections, and the red color shows the part for dissection same in Fig.Â 1a. <strong>d</strong> The transcriptomic taxonomy of 153 clusters organized in a dendrogram. Together we defined 153 transcriptomically distinct neural cell types. Most subclasses of cell types have distinct expression patterns and present clear preferences in spatial distribution. Bar plots showed the cell proportion of each cell type (up). The ridge plot outlined the spatial arrangement of all 153 neural types across regions by convolution of the spatial transcriptome (<em>n</em>â€‰=â€‰44) with snRNA-seq data (<em>n</em>â€‰=â€‰42) (middle); and identification of cluster compositional changes in each cortical region by sccoda from snRNA-seq data (down). The abbreviations: microglia (MICRO), oligodendrocyte (OLIGO), vascular leptomeningeal cell (VLMCs), endothelial cell (ENDO), astrocyte (AST), oligodendrocyte progenitor cell (OPC), near projection neurons (NP), intratelencephalic RORB neurons (L3-L4 IT RORB and L4-L5 IT RORB), intratelencephalic LINC00507 neurons (L2-L3 IT LINC00507), intratelencephalic layer 6 neurons (L6 IT), layer 6 Car3 neurons (L6 CAR3), layer 6b neurons (L6b), corticothalamic layer 6 neurons (L6 CT), extratelencephalic (ET), chandelier neuron (CHANDELIER), parvalbumin neuron (PVALB), somatostatin neuron (SST), lysosome-associated membrane protein 5 Neuron (LAMP5), paired box 6 neuron (PAX6), neuron-derived neurotrophic factor neuron (NDNF), vasoactive intestinal peptide neurons (VIP).</p></figcaption></figure><p id="Par6">Glutamatergic and GABAergic neurons, predominantly located in the cortex, serve as fundamental components in neural circuits by primarily exerting excitatory and inhibitory influences, respectively, within the cortical neural network. Identified transcriptionally, GABAergic neurons exhibit the expression of glutamate decarboxylase (either <em>GAD1</em> or <em>GAD2</em>), which is pivotal for synthesizing the inhibitory neurotransmitter GABA<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Conversely, cells that express <em>SLC17A7</em>, a vesicular glutamate transporter, are typically categorized as glutamatergic neurons due to their capability to release the excitatory neurotransmitter, glutamate<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. While subclasses of GABAergic and glutamatergic neurons can possess distinctive transcriptional profiles, morphologies, and functions, the relationships between these classifications can be complex, as underscored by the notable diversity within GABAergic and glutamatergic neuronal populations, as demonstrated by recent research<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>â€“<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Our atlas, defined by transcriptomic molecular features, aligns glutamatergic neurons with excitatory neurons and GABAergic neurons with inhibitory neurons, reflecting common classifications in neuroscience<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>â€“<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Given potential discrepancies among electrical, morphological, and molecular properties, we advocate primarily adhering to molecular criteria, especially since molecular markers stand out as the most insightful, and some molecular traits persist as lifelong cellular characteristics<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. By integrating snRNA-seq and BICCN data, we defined transcriptionally 22 subclasses under 3 major classes (Fig.Â <a href="#Fig1" class="usa-link">1d</a>, and Supplementary DatasetÂ <a href="#MOESM4" class="usa-link">2</a>): 9 subclasses under glutamatergic neuron class, 7 subclasses under GABAergic neuron class, and 6 subclasses under non-neuronal cell class. And within the subclass, 153 cell types were further divided by transcriptomic characteristics with reference to BICCN data. Each cell type distributed evenly in each donor, indicating the uniformity of sampling in each cortical region. However, we observed that certain cell types exhibit preferential distribution in specific cortical regions. For instance, glutamatergic subtypes LINC00507_3 and LINC00507_4 are more prevalent in the primary motor cortex, while RORB cells, predominantly surrounding the granular layer, are abundant in V1. In contrast, most GABAergic neurons are less abundant in this region. These findings will be discussed in detail in the final section of this research. (Fig.Â <a href="#Fig1" class="usa-link">1d</a> and Supplementary DatasetÂ <a href="#MOESM4" class="usa-link">2</a>). Cajalâ€“Retzius cells were not found due to a ratio of less than 0.1% of L1 neurons in the human cortex<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. The abbreviations for the classification of nerve cells are as follows: Non-neuronal cells include microglia (MICRO), oligodendrocyte (OLIGO), vascular leptomeningeal cell (VLMCs), endothelial cell (ENDO), astrocyte (AST), oligodendrocyte progenitor cell (OPC). Neuronal cells are also divided into two categories: excitatory and inhibitory. Representative categories of glutamatergic neurons include near projection neurons (NP), intratelencephalic RORB neurons (L3-L4 IT RORB and L4-L5 IT RORB), intratelencephalic LINC00507 neurons (L2-L3 IT LINC00507), intratelencephalic layer 6 neurons (L6 IT), layer 6 Car3 neurons (L6 CAR3), layer 6b neurons (L6b), corticothalamic layer 6 neurons (L6 CT), extratelencephalic (ET), while the representative categories of GABAergic neurons are chandelier neuron (CHANDELIER), parvalbumin neuron (PVALB), somatostatin neuron (SST), lysosome-associated membrane protein 5 Neuron (LAMP5), paired box 6 neuron (PAX6), neuron-derived neurotrophic factor neuron (NDNF), vasoactive intestinal peptide neurons (VIP).</p>
<p id="Par7">Previous studies have highlighted sex-related heterogeneity in brain structure and gene expression, suggesting potential influences on cellular composition and function<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.To evaluate the impact of sex in donor sampling, we performed variance analyses on gene expression and cellular distribution data. We partitioned variance contributions from sex, age, spatial domain (layer), subclass, and region. Sex explained &lt;1% of variance in both spatial (gene expression: 0.3%; cellular distribution: 0.8%) and snRNA-seq datasets (gene expression: 0.5%; cellular distribution: non-significant), whereas spatial domain and subclass collectively accounted for &gt;90% (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">1a, b</a>). Despite limited female representation (2/5 spatial donors; 2/7 snRNA-seq donors), permutation tests confirmed no significant sex-driven clustering (FDRâ€‰&gt;â€‰0.1). To investigate sex-associated molecular signatures, we performed differential gene expression analysis on datasets stratified by sex, followed by GO pathway enrichment and overlap analysis with disease-sensitive gene sets, categorized into snRNA-seq (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">2a</a>) and Stereo-seq (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">2b</a>) datasets. In the snRNA-seq dataset, sex-associated differential gene expression was predominantly observed in GABAergic neurons and non-neuronal cells at the subclass level, with these differentially expressed genes (DEGs) showing significant enrichment for autism spectrum disorder (ASD)-related pathways. In contrast, glutamatergic neurons exhibited minimal sex-associated transcriptional differences. For the Stereo-seq dataset, hierarchical analysis revealed relatively uniform sex-associated gene expression across cortical layers. Notably, sex-specific DEGs in each layer, such as <em>KDM5D</em> and <em>ZFY</em> (male-enriched in layer 2), demonstrated substantial overlap with ASD-associated gene sets, highlighting a consistent link between sex-biased gene expression and ASD susceptibility across both datasets.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Laminar structures in the human cerebral cortex revealed by spatial transcriptomics</h3>
<p id="Par8">The spatial resolution of Stereo-seq is approximately 500â€‰nm Ã— 500â€‰nm Ã— 500â€‰nm (i.e., each pixel corresponds to 0.5â€‰Âµm Ã— 0.5â€‰Âµm, with a section thickness of 10â€‰Âµm, though single-molecule localization precision can reach the nanoscale). For subsequent layer-specific analysis (spatial domain analysis), a grid of 100â€‰Âµm Ã— 100â€‰Âµm bins (square bin size 200) was employed. For cell-type analysis, cell binning (masked based on nuclei staining, approximately 10â€“15â€‰Âµm in size) was applied to facilitate cell-type deconvolution. To address concerns regarding potential bias from variable tissue section numbers (1, 4, 10, 13, 16) across donors, we conducted a multi-factor variance decomposition analysis. Using sum of squares (SS) calculations, we quantified contributions of five factors (donor, region, sex, age, spatial domain) to total variance in spatial transcriptomic data (total SSâ€‰=â€‰6,272,860). Donor-specific effects accounted for only 1.03% of total variance (SSâ€‰=â€‰64,347), whereas biological factorsâ€”spatial domain (93.77%, SSâ€‰=â€‰5,880,538) and region (4.62%, SSâ€‰=â€‰290,026)â€”dominated (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">1a</a> and <a href="#MOESM1" class="usa-link">c</a>). To minimize technical confounders, we applied Seurat and Harmony with default parameters, treating donor as a batch factor and region/spatial domain as covariates. UMAP visualization confirmed minimal donor-driven clustering (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">1c</a>). These analyses demonstrate that biological variability vastly outweighs donor-related bias, validating the robustness of our dataset integration. The proportions of glutamatergic neuron subclasses in single-cell data strongly correlate with those observed in Stereo slides (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">1d</a>). Neuron proportions across cortical areas also show high consistency with BICCN published data, further validating our findings (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">1eâ€“h</a>). To accurately identify spatial domains within cortical layers, we integrated multimodal data, including histology (HE and ssDNA staining) and gene expression, into a graph-based deep learning framework (DeepST)<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> for clustering. The resulting clusters were then annotated using laminar markers and deconvoluted cell type distributions, leading to the robust identification of spatial domains (Fig.Â <a href="#Fig2" class="usa-link">2a</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Spatial transcriptomics of cortical regions of humans and layer enrichment of identified laminar marker genes at Stereo-seq.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365125_41467_2025_62793_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7368/12365125/5b18ba2d3435/41467_2025_62793_Fig2_HTML.jpg" loading="lazy" id="d33e631" height="989" width="795" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Single-cell and laminar segmentation for Stereo-seq analysis. H&amp;E and ssDNA staining was performed on the previous section to observe the morphological structure inside the cells, and total mRNA captured from a cortical section by the chip microspheres (Multiple spatial data input); DeepST utilities a deep graph network model to denoise and spatial spots clustering by using both information from image and transcriptomics (Clustering by deep graph network). Then, the results of domain were annotated with traditional laminar markers and cell type distribution deconvoluted by RCTD (Laminar annotation). The slide size is 1Ã—1â€‰cm. Each slide has completed transcriptomic segmentation mostly with L1 to WM. We captured a total of 1,888,306 cells from 44 slides and markers are displayed using the spare bin (100umÃ—100um). <strong>b</strong> Spotplots of normalized expression for sample DLPFC, SMG, STG and V1 for genes <em>MBP</em> (marker for WM), <em>SNAP25</em> (marker for Neuron), <em>CCN2</em> (marker for layer 6b), <em>THEMIS</em> (marker for layer 6), <em>PCP4</em> (marker for around layer 5), <em>RORB</em> (marker for around layer 4), <em>COL5A2</em> (marker for layer 4 and layer 3), <em>CUX2</em> (marker for layer 3 and layer 2), <em>LAMP5</em> (marker for layer 2), <em>PCDH8</em> (marker for layer 2), <em>LINC00507</em> (marker for layer 2) and <em>RELN</em> (marker for layer 1). <strong>c</strong> Heatmap showing scaled expression of representative DEGs enriched in the cortical layer (L1, L2, L3, L4, L5, L6, and WM) (left) and the GO term enrichment analysis for specific regions, highlighting the molecular functions and biological processes (right). <strong>d</strong> Through heatmap above, selected non-traditional laminar markers are analyzed as spotplots of normalized expression across 10 cortical regions. These included <em>CXCL14</em> for layer 1, <em>HPCAL1</em> for layer 2, <em>NEFM</em> for layer 3, <em>NEFH</em> for layer 4, <em>SNCG</em> for layer 5, and both <em>NPTX1</em> and <em>DIRAS2</em> for layer 6. Additionally, <em>ERMN</em> and <em>BCAS1</em> served as markers for white matter (WM). The markers are displayed using the spare bin (100umÃ—100um).</p></figcaption></figure><p id="Par9">Various layers differed dramatically in both expression patterns and layer-specific cell types (Fig.Â <a href="#Fig2" class="usa-link">2b</a>). The transcriptomic expression of spots alters with spatial position, characterizing the spatial transcriptomic continuity in the cortex, consistent with prevalent neurobiological views<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. A series of laminar markers can accurately locate different layers and show expression contiguity along with laminar changing (Fig.Â <a href="#Fig2" class="usa-link">2b</a>, and Supplementary DatasetÂ <a href="#MOESM4" class="usa-link">2</a>): <em>CCN2</em> highlights layer 6 and areas near WM, <em>PCP4</em> highlights layer 5 and the areas near layer 4, <em>CUX2</em> and <em>COL5A2</em> for layer 3, <em>LAMP5</em> for layer 2, <em>LINC00507</em>/<em>PCDH8</em> for layer 2, and <em>RELN</em> for layer 1. Classic markers such as <em>MBP</em> and <em>THEMIS</em> are the top markers for WM and layer 6. The supragranular layers are identified combinatorially by the markers <em>RORB</em> and dense cellular layer staining (Fig.Â <a href="#Fig2" class="usa-link">2b</a>). These markers are exceedingly restricted to specific laminar (Fig.Â <a href="#Fig2" class="usa-link">2b</a>), indicating satisfactory conserveness for precise laminar markers. At the same time, we can analyze the DEGs in each layer of Stereo slides and find out the non-traditional laminar markers (Fig.Â <a href="#Fig2" class="usa-link">2c</a>). Although it is crucial to identify DEGs between layers, some layers shared functional processes, such as the regulation of neurotransmitter secretion across layers 1 to 4 (Fig.Â <a href="#Fig2" class="usa-link">2c</a>). Finally, we present these DEGs on a spatial transcriptome slide, <em>CXCL14</em> is similar to the classic layer 1 marker <em>RELN</em>, while <em>NEFM</em> is similar to the classic layer 3 marker <em>CUX2</em>, and has demonstrated good laminar labeling stability in 14 neocortical regions (Fig.Â <a href="#Fig2" class="usa-link">2d</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Diverse spatial characteristics of glutamatergic neuron</h3>
<p id="Par10">Glutamatergic neurons are characterized by diverse projection patterns through which axons (long, branching processes that transmit electrical signals) send the neurotransmitter glutamate to other brain regions<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. These projection patterns imply the specific brain regions they connect to and the functions they perform in those regions, so subclasses of glutamatergic neurons can be subdivided by projection patterns<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>â€“<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. We leveraged the single-cell transcriptome data to divide glutamatergic neurons into 9 subclasses correspond to different projection types (Figs.Â <a href="#Fig3" class="usa-link">3a and b</a>). Intratelencephalic (IT) neurons formed a continuous transcriptomic spectrum of cells with highly correlated gradual changes in spatial position (Fig.Â <a href="#Fig3" class="usa-link">3c</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3a</a>). Detailed, IT includes 4 subclasses: The L2-L3 IT LINC00507 subclass in the supragranular, the L3-L4 and L4-L5 IT RORB subclasses bridging the supra- and subgranular layers, and the IT L6 subclass beneath layer 5 (Fig.Â <a href="#Fig3" class="usa-link">3c, d</a>). Other transcriptionally defined subclasses include the ET, NP, L6 CAR3, CT, and L6b (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3a</a>). ET, NP, and CT neurons embody distinct classes of projection neurons, each presenting unique roles and projection patterns within the central nervous system. ET neurons in layer 5 express <em>CTIP2</em>/<em>BCL6</em>, which are upregulate genes related to axonal maintenance, long-distance signal transmission, and synaptic connections to distant targets<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. NP neurons specialize in local circuitry, upregulating <em>SYT6</em> for short-range synaptic transmission and specificity<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Lastly, CT neurons in layer 6 express <em>FOXP2</em>, supporting long axons and precise thalamic connections<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Transcriptomic taxonomy of glutamatergic cell types and spatial arrangement in the human cerebral cortex.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365125_41467_2025_62793_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7368/12365125/e941bc5ea27f/41467_2025_62793_Fig3_HTML.jpg" loading="lazy" id="d33e801" height="701" width="799" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The UMAP projection plot of glutamatergic neurons. glutamatergic neurons were divided into 9 subclasses corresponding to their projection types: NP, L6 CAR3, L6 IT, ET, L6 CT, L6b, L2-L3/IT, L3-L4/IT and L4-L5/IT. <strong>b</strong> Detailed UMAP plot showcasing diverse glutamatergic subclasses and the regional information. <strong>c</strong> The spatially resolved single-cell transcriptome of the adult human telencephalon DLPFC as determined by Stereo-seq analyses (left). Scatterplot of Stereo-seq example (DLPFC) showing 4 IT gradient-laminar distributed glutamatergic subclasses (right). Cells are colored by their subclass ID (same color as in Fig.Â 3a). The slide measures 1â€‰cm Ã— 1â€‰cm, with subclasses visualized using cell bins, and cell-type deconvolution conducted with 10â€‰Âµm cell bins derived from Stereo-seq data. <strong>d</strong> Laminar distribution of typical glutamatergic neurons at the Stereo spatial cortical regions. Stereo spatial transcriptome results of the glutamatergic neurons were divided into 5 different subclasses. The color and alpha of spots denote the position of each neuron, same color as in Fig.Â 3a. The subclasses are displayed using the cell bin. <strong>e</strong> Heatmap showing scaled expression of representative DEGs enriched in L2-L3/IT, L3-L4/IT, L4-L5/IT, and L6 IT (left), and line charts showing the expression levels of marker genes in subclasses during the cortical layer from shallow to deep (right).</p></figcaption></figure><p id="Par11">To further investigate cross-regional variation, we evaluated the similarity of these regions based on the transcriptomic separability of glutamatergic neurons and observed the specific and continuous variability of glutamatergic neuron subclasses across cortical regions (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3bâ€“h</a>). Glutamatergic neurons in spatially adjacent cortical regions have greater transcriptomic similarity (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3c, d</a>), especially in the temporal lobe (ITG and STG) and the angular gyrus-connecting parietal triangle (AG, SMG, and SPL) (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3c, d</a>). Next, our analysis focused on the DEGs among the 4 IT subclasses across various cortical regions (Fig.Â <a href="#Fig3" class="usa-link">3e</a>). We found that the four IT subclasses had unique markers that could identify them in 14 cortical regions (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">3eâ€“h</a>), although they were similarly expressed in adjacent cortical regions. Furthermore, we also examined how different factors (region and layer) influence transcriptome changes (Fig.Â <a href="#Fig4" class="usa-link">4a</a>). Our findings revealed that the IT subclasses are generally less influenced by regional specificity. However, the expression differences become more pronounced when considering laminar changes (subclass changes) (Fig.Â <a href="#Fig4" class="usa-link">4a</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Region-specific distribution and functional enrichment of ET neurons in pain-associated cortical regions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365125_41467_2025_62793_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7368/12365125/a330cf3c2744/41467_2025_62793_Fig4_HTML.jpg" loading="lazy" id="d33e874" height="790" width="792" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Between-group sum of squares of factors affecting IT cell type transcriptome variation. Two-sided Anova significance: all <em>p</em>â€‰=â€‰0 (<em>n</em>â€‰=â€‰42). <strong>b</strong> The cellular composition analysis by scCODA show differential subclass in snRNA-seq datasets, highlighting the significant reduction of LINC00507 IT neurons in the ACC. Each dot denotes cells in a subclass in one region compared to all others regions. Only the data with significant output from scCODA will be shown. <strong>c</strong> Scatterplot of Stereo-seq example (ACC) showing 4 gradient-laminar distributed glutamatergic IT subclasses. Cells are colored by their cell density. The slide size is 1x1cm, and the subclasses are displayed using the cell bin. <strong>d</strong> Mature neurons (NeuN) and projection neurons (MAP2) distribution. Left, NeuN and MAP2 staining for neuron. Right, laminar distribution of mature and projection neurons under immunohistochemical staining. <strong>e</strong> The biplot showed the correlation between cortical region function and cell specificity by redundancy analysis (RDA). The angle reflects the correlation between function and cell type, and the length indicates the magnitude of each explicative contribution. A small angle between cell type and function suggests a positive correlation, while a large angle suggests a negative correlation. Here ET neurons show a strong correlation with pain function. <strong>f</strong> Detailed representation of specific cortical regions relevant to pain perception. Zoom-in Stereo slides view for pain-positive (S1, S1E, PoCG, ACC) and pain-negative (ITG, AG, FPPFC) cortical regions, illustrating laminar organization and the distribution of ET. ET densely distributed in pain relevant regions such as S1, S1E, PoCG, while the region with least correlation with pain were found much less ET. The down left plot denotes the pain correlation in cortical regions. <strong>g</strong> Violin plots in ACC (left) and other regions (right) show the expression of differential genes in ET, ANOVA q-value adjusted. ***<em>p</em>â€‰&lt;â€‰0.001, **<em>p</em>â€‰&lt;â€‰0.01, *<em>p</em>â€‰&lt;â€‰0.05. <strong>h</strong> Dot plot represents the specific pathways of ET neurons in different cortical regions, highlighting their unique characteristics across various areas, highlighting the molecular functions and biological processes of ET in multi-cortical regions. (Hypergeometric test (q-value adjusted)).</p></figcaption></figure><p id="Par12">To investigate age-dependent trends in LINC00507 expression, we integrated spatial transcriptomics and snRNA-seq data across donors of varying ages. Spatial analysis revealed a pronounced, pan-regional decline in cytoplasmic LINC00507 expression with age (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">4a</a> and <a href="#MOESM1" class="usa-link">c</a>), corroborated by snRNA-seq (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">4b</a> and d), showing reduced abundance of LINC00507-expressing L2-L3 IT neurons in prefrontal regions (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">4e</a> and <a href="#MOESM1" class="usa-link">f</a>). Linear mixed-effects models identified significant negative correlations between age and LINC00507 expression (adjusted pâ€‰&lt;â€‰0.05), with effect sizes ranging from âˆ’0.21 to âˆ’0.54 (spatial) and âˆ’0.18 to âˆ’0.49 (snRNA-seq), suggesting that bulk-tissue declines arise from diminished neuronal counts and expression<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. These findings align with prior reports and extend them to finer laminar resolutions.</p>
<p id="Par13">In the subsequent analysis, we examined the differences in cell composition across various brain regions and found that the proportion of L2-L3 IT neurons in the ACC was lower in the single-cell data (Fig.Â <a href="#Fig4" class="usa-link">4b</a>), a trend also confirmed by the Stereo slide displays (Fig.Â <a href="#Fig4" class="usa-link">4c</a>). We then performed immunohistochemical staining of these characteristic cortical regions using markers such as NeuN and MAP2. The staining confirmed that ACC shows a lower proportion of superficial IT neurons compared to other regions (V1, FPPFC) and lacks a distinct layer 4 (Fig.Â <a href="#Fig4" class="usa-link">4d</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">4g</a>). Besides, comprehensive observations through immunohistochemistry and Stereo slides also revealed the presence of ET (Von Economo neurons) within the ACC. Notably, layers 5 and 6 in the ACC have a higher proportion of gray matter (Fig.Â <a href="#Fig4" class="usa-link">4d</a>).</p>
<p id="Par14">Meanwhile, diffusion weighted imaging and fiber tractography provide a unique noninvasive technique to study the macro-connectivity structure within the brain<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. To enhance the precision of functional mapping, we employed the Neurosynth framework for meta-analysis, which provides term-region associations at a 2â€‰mmÂ³ resolution<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. This resolution adequately supports the examination of our 14 cortical regions, aligning with their spatial dimensions and facilitating reliable correlations between functional attributes and cell-type distributions<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The 2-mm resolution of Neurosynth enables effective functional assignments across broad cortical areas, with meta-analysis mitigating limitations of individual fMRI studies by integrating data from multiple sources. By quantifying such data, we generate a connectome matrix that reveals that higher-order functions like working memory score highly in the prefrontal lobe, while primary sensory functions (motor and pain back) score higher in the central sulcus (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">4h</a>). Further linkage with our cellular atlas suggests that the higher-order functions of the prefrontal lobe may be related to the higher density of RORB cell types within this cortical region, while the primary sensory functions of the central sulcus may be more relevant to ET (Fig.Â <a href="#Fig4" class="usa-link">4e</a> and Supplementary DatasetÂ <a href="#MOESM5" class="usa-link">3</a>). We found that the pain-related cortical regions in the cortical functional imaging were adjacent to the postcentral sulcus (S1, S1E, and PoCG) and ACC (Fig.Â <a href="#Fig4" class="usa-link">4f</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">4h</a>). We also found that ETs in the postcentral sulcus are densely located in layer 5 (Fig.Â <a href="#Fig4" class="usa-link">4f</a>). It is speculated that the dense presence of ETs in layer 5 in the neocortex is highly related to the pain-related functions of the cortical region. So even though the ACC belongs to the periallocortex, the ACC exhibits pain-related functions because there are more ETs in the wider fifth layer (Fig.Â <a href="#Fig4" class="usa-link">4f</a>). Von Economo neurons (VEN), large spindle-shaped cells, are expected to be ET in the human telencephalon, and they are restricted to the ACC and frontal insula (FI)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The ETs in humans and macaques significantly overexpressed more than half of the known VEN marker genes, whereas other neuronal subclasses did not, suggesting that ETs contain VEN<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. We also found that these VEN marker genes described in the FI region did not apply to ET in the ACC region, and there was only one ET cluster in the ACC, which were generally highly expressed: <em>GNA15</em>, <em>LYPD1</em>, and <em>NEUROD1</em> (Fig.Â <a href="#Fig4" class="usa-link">4g</a>). Gene ontology (GO) terms were performed for ET in the ACC, and they were found to be associated with SRP-dependent cotranslational protein targeting membrane and cytoplasmic translation (Fig.Â <a href="#Fig4" class="usa-link">4h</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Cytoarchitecture of non-neuronal cell and GABAergic neuron</h3>
<p id="Par15">The atlas also covers cortical non-neuronal cells, showing the distribution of non-neuronal cells in cortical regions (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5a, b</a>), as well as their proportions and specific distributions across cortical areas (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5c, d</a>). Compared with non-human primates, humans exhibit a higher proportion of OPCs, suggesting that genes associated with the regulation of OPC maturationâ€”specifically, FOXP2, THEMIS, and PCDH15â€”are more abundantly expressed in the human cortex33. Utilizing Stereo slides, we investigated the distribution of these genes and discovered that their highest overlapping regions are located within layer 6 and even in the WM (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5e</a>). Moreover, our gene density analysis across cortical regions revealed a likely higher prevalence of cells co-expressing FOXP2 and THEMIS in the SPL and DLPFC (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5f</a>). Non-neuronal subclasses vary in layer distribution, reflecting their functions, such as guiding neuronal migration or stabilizing synaptic connections (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5g</a>). OLIGOs are more prevalent in WM, while OPCs are scattered across layers. OPCs play a vital role in nerve conduction and neural development<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>â€“<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, with consistent numbers across cortical regions but varying gene expression patterns (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5h</a>).</p>
<p id="Par16">Besides, we identified five astrocyte clusters through a comparative analysis with published BICCN cell classifications and subsequently focused on examining their distribution across various cortical regions, as well as their spatial arrangement on Stereo slides (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5iâ€“m</a>). Our Stereo-seq analysis revealed that the proportions of Astro4 and Astro5 cells were markedly higher in the ACC compared to other cortical regions (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5i, j</a>), and densities were significantly elevated relative to other neocortical regions (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5k, l</a>). Astro5 cells were more frequently observed in white matter regions than Astro4 cells, indicating potential functional differences between these astrocyte subtypes in distinct cortical environments (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5l</a>). Further DEGs analysis across cortical regions identified <em>TMEM144</em> as a gene specifically expressed in Astro4 and <em>SGK1</em> as a gene specifically expressed in Astro5 within the ACC (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">5m</a>).</p>
<p id="Par17">Additionally, we leveraged atlas-based analysis to decipher the single-cell spatial transcriptomic profiles of GABAergic neurons. GABAergic neurons are primarily involved in local communication within a specific brain region. They form connections with other neurons within the same brain region and play a critical role in the integration and processing of information within that region<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. GABAergic neurons in the taxonomy were split into two groups: caudal ganglionic eminence (CGE)-derived and medial ganglionic eminence (MGE)-derived (Fig.Â <a href="#Fig5" class="usa-link">5a</a>). In detail, we identified 4 subclasses of the CGE branch: LAMP5, NDNF, PAX6, and VIP (Fig.Â <a href="#Fig5" class="usa-link">5a</a>). On the contrary, MGE-derived neurons have 3 subclasses, CHANDELIER, PVALB and SST (Fig.Â <a href="#Fig5" class="usa-link">5a</a>), with developmental fates derived from ventral and dorsal MGEs, respectively<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. In the adult human cortex, the CGE-derived and MGE-derived neurons accounted for 45% and 55% of the total (Fig.Â <a href="#Fig5" class="usa-link">5a</a>), consistent with initial neurodevelopmental origins<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, indicating a relatively stable composition of GABAergic neurons during the whole human life.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Characterization and spatial distribution of GABAergic subclasses in human cortical regions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365125_41467_2025_62793_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7368/12365125/e95f3d0c040e/41467_2025_62793_Fig5_HTML.jpg" loading="lazy" id="d33e1102" height="995" width="763" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> UMAP of GABAergic subclasses. Each subclass is represented by its specific color region. The values represent the number of cells. <strong>b</strong> Distribution of typical CGE-derived subclasses at the stereotaxic cortical regions. The color and alpha of spots denote the position of each cell. The slide size is 1x1cm, and the subclasses are displayed using the cell bin. <strong>c</strong> UMAP plot distinguishing between the shallow and deep families within the GABAergic SST neuron, respectively (up); distribution of SST clusters within shallow and deep families (down). <strong>d</strong> Zoom-in Stereo slides view for deep and shallow SST family in cortical regions, illustrating different distribution of MGE-derived deep and shallow SST family, with the color of spots denoting the position of each cell. The subclasses are displayed using the cell bin. <strong>e</strong> Heat map of DEGs between deep and shallow SST families. The horizontal axis is the DEGs and the vertical axis is the cluster, showing that the clusters in the two types of families are significantly different in terms of DEGs. <strong>f</strong> UMAP plot shows the distribution of <em>CALB</em> in deep and shallow SST families. <strong>g</strong> The enrichment of biological processes in the deep and shallow SST family. <strong>h</strong> The observation of typical gene expression patterns of SST families in both shallow and deep stereocortical regions. Dot plots showcase the gene expression levels for 6 representative shallow and deep SST families in each cell. In these plots, the position of the dots corresponds to the cellâ€™s location, while the color intensity indicates the expression level. The genes are displayed using the spare bin (100umÃ—100um).</p></figcaption></figure><p id="Par18">We found that most CGE-derived clusters favored a supragranular arrangement, especially taxonomic NDNF is present at the superficial layer (Figs.Â <a href="#Fig1" class="usa-link">1</a>d and <a href="#Fig5" class="usa-link">5b</a>), whereas MGE-derived clusters had no apparent laminar restriction. Compared with GABAergic neurons in the mouse cortex, the CGE-derived neurons in the human cortex migrate more shallowly within the tissue and contribute more cells that not only localize exclusively around layer 5 (Fig.Â <a href="#Fig5" class="usa-link">5b</a>)<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>.</p>
<p id="Par19">Through Stereo slides, it was found that MGE-derived SST neurons have upper and lower layering characteristics (Fig.Â <a href="#Fig5" class="usa-link">5c</a>), and MERFISH also confirmed this<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. SST neurons exhibit a division into upper and lower regions, with the L3 region showing relative sparsity (Fig.Â <a href="#Fig5" class="usa-link">5d</a>). We hypothesize that this pattern results from distinct migratory pathways during development. Specifically, superficial SST neurons migrate through the marginal zone (MZ), while deep-layer neurons migrate via the subventricular zone (SVZ), leading to well-defined layering<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Additionally, we found that 20 genes exhibited upregulation in deep SST neurons compared with shallow SST neurons (Fig.Â <a href="#Fig5" class="usa-link">5e</a>). The CALB showed a notably higher expression in superficial SST neurons (Fig.Â <a href="#Fig5" class="usa-link">5f</a>), neurons expressing CALB exhibited a tendency to receive more robust input from excitatory neurons within the same layer compared to calbindin-negative neurons<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. In contrast, SST neurons located in the subgranular layer were predominantly calbindin-negative (Fig.Â <a href="#Fig5" class="usa-link">5f</a>). Furthermore, deep SST neurons are associated with G-protein coupled neurotransmitter receptor activity, while superficial SST neurons are more active in cyclic adenosine monophosphate (cAMP) phosphodiesterase activity (Fig.Â <a href="#Fig5" class="usa-link">5g</a>). Besides, we noticed that marker of shallow SST family <em>PDZD1</em>, <em>GNAL</em>, <em>GRIA4</em>, and <em>CALB1</em> were more expressed in the superficial layer, while <em>DCC</em>, <em>TRHDE</em>, and <em>GRIN3A</em> of deep SST family were co-expressed in deep layer (Fig.Â <a href="#Fig5" class="usa-link">5h</a>).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Cell-cell interactions across 14 human cerebral cortical regions</h3>
<p id="Par20">High-resolution spatial Stereo-seq enabled us to infer cell-cell interactions resulting from somatic contact or paracrine signaling, which is inferred by frequent cell proximity<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> (Fig.Â <a href="#Fig6" class="usa-link">6a</a>). To investigate these potential specific cell-cell interactions, we considered calculating the frequency of observed somatic contact or proximity among cells at the subclass level. Apart from interactions within non-neuronal subclasses, we observed a tendency for non-neuron contact or proximity pairs with certain layer-restricted excitatory neurons (Fig.Â <a href="#Fig6" class="usa-link">6b</a>). We also examined the frequency of interactions between subclasses across 4 cortical lobes and found that the IT subclass showed layer-specific interactions with other neuronal cells in these 4 lobes. Interestingly, the GABAergic subclass, particularly in the infragranular layers (layers 5-6), presented more interactions with IT than with CT and NP, while non-neuronal cells did not exhibit a similar trend (Fig.Â <a href="#Fig6" class="usa-link">6c</a>). Further analysis of interaction patterns across cortical regions revealed that the ACC and V1 differed significantly from other cortical regions, possibly due to their unique laminar relationships (Fig.Â <a href="#Fig6" class="usa-link">6dâ€“f</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Cell-cell interactions in the human cerebral cortex.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365125_41467_2025_62793_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7368/12365125/5190a9cf1691/41467_2025_62793_Fig6_HTML.jpg" loading="lazy" id="d33e1238" height="999" width="780" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Spatial slide of major subclasses of Glutamatergic neuron (EXC), GABAergic neurons (INH), and non-neuronal cells in 4 lobes (left); and zoomed-in view of the boxed region show multiple somatic contact forms of cell-cell interactions at spatial maps (right). Colored shapes are cell nuclei segmentations. The slide size is 1x1cm, and the subclasses are displayed using the cell bin. <strong>b</strong> Enrichment map of pairwise soma contact or proximity for subclasses of cells in the human cortex. The color indicates the observed frequency of soma contact or proximity. <strong>c</strong> Heatmap showing the laminar arrangement of soma contact or proximity interaction patterns (1v1) in the four brain lobes. <strong>d</strong>, <strong>e</strong> Heatmap showing the distribution of frequent interaction patterns across cortical regions. <strong>f</strong> Line chart and heatmap showing the distribution of the somatic contact forms between OLIGO/OPC and various neuronal subclasses in 14 cortical regions. Error bars are provided in the Source Data. <strong>g</strong> Enrichment of ligand-receptor pairs in contacting microglia/astrocytes and neuronal subclasses. <strong>h</strong> Line chart and heatmap showing the distribution of the somatic contact forms between astrocyte and various neuronal subclasses in 14 cortical regions. Error bars are provided in the Source Data. <strong>i</strong> Line chart and heatmap showing the distribution of the somatic contact forms between microglia and various neuronal subclasses in 14 cortical regions. Error bars are provided in the Source Data. <strong>j</strong> Heatmap showing the distribution of pairing contact microglia/astrocytes and INH/EXC in 14 human cortices.</p></figcaption></figure><p id="Par21">In the ligand-receptor analysis, astrocytes primarily engage with other neuronal types through mainly three receptor-ligand pairs: NRG3-ERBB4, NRXN1-LRRTM4, and NRXN1-NLGN1. Conversely, microglia demonstrate a more diverse set of receptor-ligand interactions in their communication with various neuronal types. (Fig.Â <a href="#Fig6" class="usa-link">6g</a>). We also observed various combinations of these two glial cell types across cortical regions, finding that microglia exhibit greater diversity (Fig.Â <a href="#Fig6" class="usa-link">6h, i</a>), whereas the combination of astrocytes with neuronal types showed a specific high proportion in S1E (Fig.Â <a href="#Fig6" class="usa-link">6j</a>).</p></section><section id="Sec8"><h3 class="pmc_sec_title">Spatial transcriptomics of Layer 4 and subplate area in human cerebral cortex</h3>
<p id="Par22">The cytoarchitectural atlas of the adult human cortex provides us with a precise tool for deciphering cortical layers and fine structures. The cellular basis of layer 4 is crucial for understanding sensory processing and related neurological diseases<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Although markers for layer 4 and L4-like neurons have been identified in the M1 region<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, the distribution and expression pattern of layer 4 in 14 cortical regions remain unknown. To explore this further, we analyzed the distribution density of different subclasses within each cortical region (Fig.Â <a href="#Fig7" class="usa-link">7a</a>). Our findings revealed that L3-L4 IT and L4-L5 IT RORB glutamatergic neurons dominate in the V1 region (Figs.Â <a href="#Fig7" class="usa-link">7b, c</a>). Using the arrangement of layers in Stereo slides, we calculated the proportion of each layer within the gray matter. Notably, layer 4 in V1 was the largest, significantly exceeding its proportion in other cortical areas (Fig.Â <a href="#Fig7" class="usa-link">7d</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">6</a>). This distribution pattern was closely associated with the pronounced thickness of layer 4 in the visual cortex, as indicated by the higher abundance of RORB neurons (Fig.Â <a href="#Fig8" class="usa-link">8a</a>). This analysis confirmed this correlation, highlighting the relationship between the thickening of the granular layer and the increased presence of RORB neurons (Fig.Â <a href="#Fig8" class="usa-link">8b</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Spatial distribution and laminar organization of glutamatergic subclasses across 14 cortical regions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365125_41467_2025_62793_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7368/12365125/6ef037352a7a/41467_2025_62793_Fig7_HTML.jpg" loading="lazy" id="d33e1316" height="869" width="765" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The density distribution of each subclass in the cortical regions (<em>n</em>â€‰=â€‰42). The density statistics are shown below. The color follows the color assigned to each subclass in the taxonomy. <strong>b</strong> Heatmap showing the density distribution of each subclass in cortical areas. <strong>c</strong> The biplot showed the correlation between cortical region and glutamatergic subclasses by RDA. The angle reflects the correlation between cortical region and glutamatergic subclasses, and the length indicates the magnitude of each explicative contribution. <strong>d</strong> The boxplot depicted the layer arrangement of grey matter in 14 cortical regions using Stereo slides (nâ€‰=â€‰44, FPPFCâ€‰=â€‰4, DLPFCâ€‰=â€‰7, VLPFCâ€‰=â€‰5, M1â€‰=â€‰3, S1â€‰=â€‰3, S1Eâ€‰=â€‰1, PoCG=1, SPLâ€‰=â€‰3, SMGâ€‰=â€‰4, AGâ€‰=â€‰6, V1â€‰=â€‰3, ITGâ€‰=â€‰1, STGâ€‰=â€‰1, ACCâ€‰=â€‰2, Minimum, maximum, center points, and box boundaries of box plots are provided in the Source Data.) (Two-sided Anova, *<em>p</em>â€‰&lt;â€‰0.05, **<em>p</em>â€‰&lt;â€‰0.01, ***<em>p</em>â€‰&lt;â€‰0.001).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8. Spatial variation of RORB glutamatergic neurons in visual cortex.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365125_41467_2025_62793_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7368/12365125/623565bbf4cb/41467_2025_62793_Fig8_HTML.jpg" loading="lazy" id="d33e1349" height="935" width="780" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> A refined laminar annotation identification of layer 4 in visual cortex (<em>n</em>â€‰=â€‰3), DeepST AI-assisted semi-supervised laminar segmentation based on nucleic acid and cell type annotation. <strong>b</strong> The spatially resolved single-cell transcriptome of the adult human telencephalon V1 (<em>n</em>â€‰=â€‰3) as determined by Stereo-seq analyses (top left). Selected an area with a layered structure (top right). Scatterplot of two Stereo-seq example (V1) showing gradient-laminar distributed subclasses (down). Cells are colored by their subclass ID (same color as in a). The slide size is 1x1cm, and the subclasses are displayed using the cell bin, and cell-type deconvolution is performed using 10 Î¼m cell bins derived from Stereo-seq data. <strong>c</strong> Volcano plots showing layer 4 DEGs in 14 cortical regions. The 3â€“4 most significant genes are shown with their names highlighted (two-sided Wilcoxon test, Bonferroni corrections for multiple comparison; genes with adjusted Pâ€‰&lt;â€‰0.01 and fold change &gt; 1.2 are defined as significant). <strong>d</strong> Volcano plots showing DEGs in the fine structure of layer 4 in V1. The 6 most significant genes are shown with their names highlighted (two-sided Wilcoxon test, Bonferroni corrections for multiple comparison; genes with adjusted Pâ€‰&lt;â€‰0.01 and fold change &gt; 1.2 are defined as significant). <strong>e</strong> Heatmap showing scaled expressions of representative DEGs enriched in L4a, L4b, and L4c (left), and the GO term enrichment analysis for specific regions, highlighting the molecular functions and biological processes (right). <strong>f</strong> Stereo spatial slides of layer 4 in V1 showing expression of markers related to the fine laminar structure. In these plots, the position of the dots corresponds to the marker expression in cells, while the color intensity indicates the expression level.</p></figcaption></figure><p id="Par23">We then examined the DEGs in layer 4 across 14 cortical regions and found significant downregulation of <em>ADAMTS3</em> and <em>FFAR4</em> in V1 (Fig.Â <a href="#Fig8" class="usa-link">8c</a> and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">7</a>), while schizophrenia or neurodegenerative-related genes <em>LRRC73</em> and <em>OGN</em> were expressed in layer 4 of the temporal lobe (ITG, STG) (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">7</a> and Supplementary DatasetÂ <a href="#MOESM6" class="usa-link">4</a>). Building on these findings, we employed high-resolution Stereo-seq to perform semi-supervised laminar clustering, further subdividing layer 4 of V1 into three distinct sub-layers and analyzing the differential gene expression within these sub-layers (Fig.Â <a href="#Fig8" class="usa-link">8d, e</a>). Finally, Stereo slides validated the accuracy of these DEGs as spatial markers, with <em>EGR1</em> marking L4a, <em>NEFH</em> and <em>SYT2</em> marking L4b, and <em>CD74</em> and <em>CNTN5</em> marking L4c (Fig.Â <a href="#Fig8" class="usa-link">8f</a>, and Supplementary DatasetÂ <a href="#MOESM6" class="usa-link">4</a>).</p>
<p id="Par24">In addition to layer 4, we leveraged the atlas to search for finer structures in human cerebral cortex, like layer 6b (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">8a</a>). This structure was transcriptionally defined as layer 6b in the human cerebral cortex, and its neurons differ from those in layer 6 and express markers of cortical subplate neurons such as <em>CTGF</em> (<em>CCN2</em>), <em>SEMA3E</em>, and <em>MGST1</em>, etc (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">8b</a>)<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. These findings demonstrate that layer 6b is not a transient structure and is stabilized in the posterior segment of layer 6 versus the anterior end of the WM<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, suggesting that cortical subplate neurons may be a permanent subpopulation. Cortical subplate neurons have only L6b in layer 6b to regulate the activity of deep cortical circuits<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>, while L6 CT is a transitional cell type that exists between layer 6 and layer 6b (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">8c</a>). Through differential analysis of the L6b region in each cortical region, it can be found that most regions in the neocortex relatively stable express genes with differential features (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">9</a> and Supplementary DatasetÂ <a href="#MOESM6" class="usa-link">4</a>).</p>
<p id="Par25">Besides, we also provide a detailed comparison between our dataset and the BICCN dataset using UMAP visualization, examining the distribution and DEGs at the subclass level (Supplementary Figs.Â <a href="#MOESM1" class="usa-link">10</a>â€“<a href="#MOESM1" class="usa-link">51</a>). Additionally, we present the atypical laminar markers identified through hierarchical differential gene expression analysis in Fig.Â <a href="#Fig2" class="usa-link">2</a>, visualized across 44 Stereo-seq slices (Supplementary Figs.Â <a href="#MOESM1" class="usa-link">52</a>â€“<a href="#MOESM1" class="usa-link">60</a>). Lastly, we perform and present differential gene expression analysis on cortical layers beyond L4 and L6b, further highlighting layer-specific transcriptional profiles (Supplementary Figs.Â <a href="#MOESM1" class="usa-link">61</a>â€“<a href="#MOESM1" class="usa-link">67</a>).</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par26">We profiled the transcriptomes of 307,738 nuclei from 42 fresh samples, even merged 814,034 nuclei from BICCN published data, and 1,888,306 cells for 44 Stereo slides of 9 donors across the entire human cerebral cortex. We conducted a comprehensive census of cellular diversity and spatial arrangements, which facilitated the delineation of the cellular layer distributions of cell types and provided much more spatial and transcriptomic information than traditional approaches such as RNAscope<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>.</p>
<p id="Par27">Our analysis shows that neural cells in the human cortex exhibit high diversity in cell types and heterogeneity in spatial arrangements. There are favorable mappings among different cellular attributions, such as transcriptomically taxonomic classification, molecular features, laminar distribution, and developmental fate. We showed that differences in expression across different types of glutamatergic neurons may help explain the range of projection patterns (Fig.Â <a href="#Fig3" class="usa-link">3</a>). Glutamatergic neurons appeared to be more layer-specific with unimodal distribution, like the RORB neuron type spanning the supra- and subgranular layers and the LINC00507 neuronal type prevalent in the superficial layer. Besides, four types of CGE-derived interneurons were found that tend to arrange in the superficial layer throughout the cortex (Fig.Â <a href="#Fig5" class="usa-link">5b</a>). Moreover, we also used spatial transcriptomics with nanoscale resolution to predict cell-cell interactions caused by somatic contacts or paracrine signaling, and discovered combinatorial patterns in multiple neural cells. Simultaneous cellular atlases also revealed the localization of layers 4 and 6b in multicortical regions, exposing specific marker genes and cellular structures.</p>
<p id="Par28">Previous studies in mice have similarly revealed conserved cellular structures and interactions across cortical layers, employing single-cell transcriptomics and spatial transcriptomics to map cell types and interactions in the mouse brain<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. However, our study extends this framework by uncovering unique features of the human cortex that are absent in mouse models, such as region-specific variations in SST neurons. These neurons are divided into deep and shallow families with distinct transcriptomic profiles and biological processes. The upregulation of genes like <em>CALB</em> in shallow SST neurons and the activation of G-protein-coupled receptors in deep SST neurons represent significant differences from the findings in mouse studies. Moreover, our detailed analysis of layer-specific cell-cell interactions underscores the complexity and heterogeneity of neuronal communication within the human cortex.</p>
<p id="Par29">Our cytoarchitectural atlas covers the entire human cerebral cortex, and samples with high quality and completeness have improved the reliability and robustness of this work. Yet a larger cohort of samples with more cells and genes by more advanced techniques can promote more appropriate and accurate output, which helps find out rarer cell types and bring more insights into potential heterogeneities like genders and ages. Our sex-stratified analyses confirm that the primary findings are robust and independent of sex, with sex-related effects being minimal. By excluding sex-linked genes in our analysis, we minimized potential confounding factors. However, larger cohorts are needed to comprehensively evaluate sex- and age-related interactions. Moreover, we took a robust clustering but conservative in spatial domain identification, which may ignore variability within clusters. Future work should flesh out these differences, as well as link transcriptomic characteristics with morphology and electrophysiology<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>.</p>
<p id="Par30">Overall, our study demonstrates the potential power of combining snRNA-seq and spatial transcriptome sequencing for exploring the cellular basis of sensory perception, cognition, and behavior. Research along these lines may not only help elucidate the mysteries of human cognition but also accelerate our understanding of neurological disorders<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Human brain tissue acquisition</h3>
<p id="Par31">This study was approved by the Ethical Committee of the Harbin Institute of Technology (Approval No. HIT-2021004). Tissues were collected from individuals who died suddenly and had no known history of brain injury or neurological disease. A total of nine neurologically healthy human donors (2 females and 7 males), ranging in age from 5 to 61 years, were included in this study. Among them, five donors were used for Stereo-seq analysis (age: 39â€‰Â±â€‰19.9 years; postmortem interval [PMI]: 6.6â€‰Â±â€‰2.2â€‰hours), and five donors were used for snRNA-seq analysis (age: 51.4â€‰Â±â€‰12.2 years; PMI: 14â€‰Â±â€‰4.2â€‰hours) (meanâ€‰Â±â€‰standard deviation). Written informed consent was obtained from all participants or their legal next of kin. Postmortem brain donation was authorized by the families, who were provided with non-financial compensation. In accordance with legal and ethical regulations prohibiting monetary incentives, compensation was offered in the form of genome sequencing of the donated brain tissue. The results of the sequencingâ€”screening for potential neuropsychiatric or hereditary neurological conditionsâ€”were subsequently communicated to the families. All experiments were performed by following the guidelines of the International Review Board and Institutional Ethics Committee. Brain samples were collected after donor family members signed informed consent in compliance with sample use and storage guidelines. In addition, tissues were used with the informed consent of immediate family members. Coronal sections were used only if they were confirmed not to show neurological or neuropsychiatric disease, based on an assessment by a forensic expert in the Department of Forensic Medicine of Guangdong Medical University. All coronal sections in the study were collected within 48â€‰hours after death and showed an RNA integrity number (RIN) of at least 6.2. Besides, samples for spatial transcriptomic sequencing have RIN over 7 (Supplementary DatasetÂ <a href="#MOESM3" class="usa-link">1</a>).</p></section><section id="Sec12"><h3 class="pmc_sec_title">Ethics &amp; Inclusion statement</h3>
<p id="Par32">This study was conducted in collaboration with local researchers, who contributed to study design, implementation, and data interpretation. Roles and responsibilities were agreed upon prior to the research, and capacity-building opportunities for local partners were prioritized. The study adhered to local ethical guidelines and was approved by the Harbin Institute of Technology (Approval No. HIT-2021004). Research protocols followed higher international standards where local regulations were insufficient. We ensured participant safety and minimized risks of stigmatization or harm. Relevant local and regional literature was incorporated into our analysis to contextualize findings. Benefit-sharing measures were discussed for any transferred materials or knowledge.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Brain tissue processing and staining</h3>
<p id="Par33">Coronal brain sections were suspended in pre-chilled to âˆ’80â€‰Â°C, oxygenated artificial cerebrospinal fluid [0.5â€‰mM calcium chloride (anhydrous), 25 mM <span class="font-variant-small-caps">d</span>-glucose, 20â€‰mM HEPES, 10â€‰mM magnesium sulfate, 1.2â€‰mM sodium phosphate monobasic monohydrate, 92 mM <em>N</em>-methyl-<span class="font-variant-small-caps">d</span>-glucamine chloride (NMDG-Cl), 2.5â€‰mM potassium chloride, 30â€‰mM sodium bicarbonate, 5â€‰mM sodium <span class="font-variant-small-caps">d</span>-ascorbate, 3â€‰mM sodium pyruvate, and 2â€‰mM thiourea]. The sections in artificial cerebrospinal fluid were quick-frozen on dry ice and transported to a pathology dissection table. The following cortical regions were identified in 14 brain sections. These regions were separately extracted from the 14 sections, cut into smaller pieces, immersed in OCT fixative (catalog no. 4583, Tissue-TekÂ® Sakura, Torrance, CA), quick-frozen on dry ice, and stored at âˆ’80â€‰Â°C until cryosectioning.</p>
<p id="Par34">Sections were stained with hematoxylin-eosin as follows. Sections were incubated for 1â€‰min at 37â€‰Â°C, fixed in methanol at âˆ’20â€‰Â°C for 30â€‰min, incubated for 7â€‰min in hematoxylin, for 2â€‰min in Bluing Buffer (catalog no. C0105S, H&amp;E staining reagent test kit, Beyotime, Shanghai, China), and 1â€‰min in eosin. Between each staining step, slides were washed with DNase- and RNase-free water (catalog no. 10977023, Invitrogenâ„¢, MA, USA). Stained sections were imaged under a microscope (ECLIPSE Ti, Nikon, Tokyo, Japan).</p>
<p id="Par35">Sections were stained with Nissl solution as follows. Sections were washed briefly in tap water to remove residual salts, immersed in 2 changes in 100% ethanol (3â€‰min each), defatted through 2â€“3 changes of 100% xylene (15â€‰min each), and rehydrated in 100% ethanol for 10â€‰min, washed in tap water, stained for 4â€“15â€‰min with 0.1% cresyl violet Nissl staining solution (catalog no. G1432, Solarbio, Beijing, China), rinsed quickly in tap water, then washed in 70% ethanol.</p>
<p id="Par36">When necessary, Nissl-stained sections were immersed for 2â€‰min in differentiation solution (catalog no. G1432, Solarbio, Beijing, China), dehydrated through 2 changes of absolute ethanol (3â€‰min each), and cleared through x2 of xylene, then allowed to dry in a fume hood. Cortical layers in the sections were visualized under a fluorescence dissecting microscope (Leica DM6000B, Heidelberger, Germany).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Immunohistochemistry staining</h3>
<p id="Par37">Coronal sections adjacent to Stereo-seq chips were collected for IHC staining with NeuN (Proteintech 26975-1-AP, 1:10000), GAP43 (Abcam ab75810, 1: 3000), and MAP2 (Proteintech 17490-1-AP, 1:2500) antibodies. The 10-Î¼m sections were mounted on gelatinized glass slides and baked to dry for 5â€‰minutes at 37â€‰Â°C. Afterward, the brain tissues mounted on slides were fixed with 4% paraformaldehyde in 0.1â€‰M phosphate buffer (PBS) for 10â€‰minutes. Following three PBS washes, the sections underwent a 15-minute pre-incubation in 0.5% Triton X-100 in PBS, followed by a 1-hour incubation in a blocking solution comprising 10% normal goat serum and 0.1% Triton X-100 in PBS (0.1% PBST). Subsequently, the sections were incubated overnight at 4â€‰Â°C in 0.1% PBST containing the monoclonal antibodies NeuN, GAP43, and MAP2 polyclonal antibody.</p>
<p id="Par38">After another three washes in PBS, the sections underwent a 30-minute incubation in 0.1% PBST containing 0.6% hydrogen peroxide to block endogenous peroxidase that might contribute to background staining. Following three additional PBS washes, the sections were incubated in PBS containing a biotinylated secondary antibody (1:200) for 2â€‰hours. They were then washed three times in PBS and transferred to PBS containing the peroxidase conjugate from the Vectastain CBC kit (Vector Laboratories, Burlingame, CA). After a series of rinses in PBS, the sections were immersed in a solution of 0.05% 3-3â€™diaminobenzidine-4HCl (DAB, Sigma-Aldrich, St Louis, MO) and 0.05% hydrogen peroxide. Once the staining was complete, the sections underwent dehydration in increasing concentrations of ethanol, were cleared in xylene, and finally, coverslipped with DPX medium. Subsequently, the glass-mounted sections were scanned at 5Ã— (0.88 Î¼m/pixel) in a Zeiss scanner to generate images.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Tissue processing for spatial transcriptomic sequencing</h3>
<p id="Par39">Tissue sections for Stereo-seq were adhered to the Stereo-seq chip (generated by BGI, China) surface and incubated at 37 Â°C for 3â€‰minutes. Then, the sections were fixed in methanol and incubated for 30â€‰minutes at âˆ’20 Â°C before Stereo-seq library preparation. Where indicated, the same sections were stained with nucleic acid dye (Thermo Fisher, <a href="https://www.ncbi.nlm.nih.gov/protein/Q10212" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q10212</a>) and imaging was performed with a Motic Custom PA53 FS6 microscope prior to in situ capture at the channel of FITC.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Single-nucleus RNA sequencing</h3>
<p id="Par40">Samples were prepared for snRNA-seq at NovelBio Bio-Pharm Technology (Shanghai, China). Tissue samples were surgically removed as the methods above and snap-frozen in liquid nitrogen, and then nuclei were isolated as described, with some modifications. Briefly, frozen tissue was homogenized in NLB buffer [250â€‰mM sucrose, 10â€‰mM Tris-HCl, 3â€‰mM MgAc2, 0.1% Triton X-100 (Sigma-Aldrich, Saint-Louis, Missouri, USA), 0.1â€‰mM EDTA, 0.2 U/Î¼L RNase Inhibitor (Takara, Tokyo, Japan)]. Then nuclei were isolated on a sucrose gradient, adjusted to approximately 1000 nuclei/Î¼L, and loaded into single channels to generate single-cell Gel Bead-In-Emulsions (GEMs) using the Chromium Controller Instrument (10X Genomics) and Chromium Single Cell 3â€™ Reagent Kit (version 3.1, 10X Genomics). After reverse transcription, GEMs were broken, and barcoded cDNA was purified, amplified, fragmented, A-tailed, ligated with adaptors, and subjected to index PCR. The final libraries were quantified using the Qubit High Sensitivity DNA assay (Thermo Fisher Scientific, Cleveland, OH, USA). The size distribution of the libraries was determined using a High Sensitivity DNA chip on a Bioanalyzer 2200 (Agilent, Santa Clara, CA, USA). All libraries were sequenced by Novaseq6000 (Illumina, San Diego, CA, USA) on a 150â€‰bp paired-end run.</p>
<p id="Par41">Coronal brain sections on slides were used for cell capture, barcoding, reverse transcription, cDNA amplification, and library construction using the Chromium Single Cell 3â€™ Reagent Kit (version 2, catalog no. 120237, 10x Genomics) according to the manufacturerâ€™s instructions.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Quantification and statistical analysis</h3>
<p id="Par42">Minimal sample sizes were not determined. Data were not collected through randomization, nor were analysts blinded to the data conditions, because all data represented a single set of conditions.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Processing of single-cell sequencing data</h3>
<p id="Par43">Single-cell sequencing data were processed using the analysis pipelines in Cell Ranger (version 6.0.2, 10X Genomics)<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Reads were aligned against the human reference genome refdata-gex-GRCh38-2020-A, and feature-barcode matrices were generated using the <em>count</em> function in Cell Ranger. Expression levels were calculated as counts per million reads (CPM). In some analyses, CPM values were transformed by log<sub>2</sub>(CPMâ€‰+â€‰1). Genes were considered to be detected if CPMâ€‰&gt;â€‰0. CPM values depended not only on absolute transcript number but also on gene length. Thus, CPM values for short, abundant transcripts could be similar to those for long, less abundant transcripts.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Quality control</h3>
<p id="Par44">Low-quality cells in each sample were identified and removed using the <em>isOutlier</em> function in the program scatter in R, which identifies outliers based on median absolute deviation (MAD) from the median value of each metric across all cells. Cells were defined to be low-quality if 1). The cell library size (total UMI counts) is smaller than 3 MADs; 2). The number of detected genes is smaller than 3 MADs; 3) the proportion of mitochondrial gene counts is bigger than 3 MADs. Doublets were identified using DoubletFinder<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>, and the expected doublet rate was 0.075.</p>
<p id="Par45">With the data from the remaining high-quality cells, we removed genes expressed in fewer than 10 cells to avoid unnecessary computational work in downstream analyses. Besides, 2 clusters were assigned to donor-specific due to a higher proportion of donors and were excluded from further analysis. These donor-specific clusters came from one donor in the dendrogram, with very low reads and genes detected.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Uniform manifold approximation and projection of single-cell data</h3>
<p id="Par46">We performed a principal component analysis based on the normalized gene expression matrix of 2000 highly variable genes. We used Harmony<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> to perform batch correction across all samples (lambda = 1, theta = 2, donor and library as batch factor), and then uniform manifold approximation and projection (UMAP, default parameters in Seurat RunUMAP) were performed with the <em>RunHarmony</em> function in Seurat<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. Constellation plots depicting global relatedness among cell types were generated using the <em>get_knn_graph</em> and <em>plot_constellation</em> functions in scrattch.hicat<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Single-cell transcriptomic clustering and annotation</h3>
<p id="Par47">In our clustering analysis, we first corrected the data using the Seurat standard clustering pipeline with Harmony for batch correction (lambda = 1, theta = 2, donor and library as batch factor). Initially, we annotated the clusters at the subclass level based on a common set of broad cell-type marker genes: for example, GAD1 or GAD2 expression identified GABAergic neurons; SLC17A7/SATB2 marked glutamatergic neurons; PDGFRA identified oligodendrocyte progenitor cells; AQP4 indicated astrocytes; PLP1/MOBP marked oligodendrocytes; PDGFRB marked vascular smooth muscle cells; FLT1 identified vascular endothelial cells; and APBB1IP indicated microglia.</p>
<p id="Par48">To further refine the annotations and align them with the latest datasets, we first performed a global alignment of our data with external data using the Harmony method (lambda = 1, theta = 2, donor and chip as batch factor), and validated the expression of subclass marker genes. After merging our data with the external dataset, which was downloaded from <a href="https://cellxgene.cziscience.com/collections/d17249d2-0e6e-4500-abb8-e6c93fa1ac6f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cellxgene.cziscience.com/collections/d17249d2-0e6e-4500-abb8-e6c93fa1ac6f</a><sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>, we split the combined data into smaller datasets according to subclasses and then applied Harmony (lambda = 1, theta = 2, donor and chip as batch factor), again to each smaller dataset for more precise alignment. Based on the aligned features obtained from Harmony (lambda = 1, theta = 2, donor and chip as batch factor), we constructed a k-nearest neighbors (KNN, nâ€‰=â€‰15) classifier using the external data and their more detailed cluster labels. This classifier was then applied to the aligned dimensionality-reduced features of our data to assign the corresponding detailed labels to our cells.</p>
<p id="Par49">Additionally, we identified specific marker genes for these finer clusters within each subclass and used them to validate the clusters annotated by the KNN classifier. After several iterations of Harmony parameter adjustments, the alignment became optimal, and our annotated data displayed consistent UMAP visualizations and marker characteristics with the published dataset, demonstrating a high degree of concordance between our dataset and the external reference.</p>
<p id="Par50">We then defined subclasses and clusters based on the taxonomy dendrogram generated from data of 14 cortical regions. Following the convention of nomenclature<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, Clusters were named according to the major cell type (EXC, INH, ASTRO, OLIGO, OPC, VLMC, and MICRO), the maximal expression subclass markers (<em>PAX6, LAMP5, VIP, SST, PVALB, LINC00507, RORB, THEMIS, FEZF2, TYROBP, FGFR3, PDGFRA, OPALIN</em> or <em>NOSTRIN</em>), and the top marker for each cluster.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Regional similarity analysis</h3>
<p id="Par51">The analysis was done at the subclass level based on single-cell transcriptomics. We used randomForest on single-cell transcriptomic data for each category to build the classification model and obtain the appropriate confusion matrix, which can show how dissimilar cellular transcriptomics are from those from other regions. More comparable molecular features from different areas are represented by higher levels of confusion. The confusion matrix was then further clustered by <em>hclust</em> function to show the similarities in transcriptome profiles between various locations. Additionally, <em>FindAllmakers</em> in Seurat<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> was used to examine the genes that differ in expression across regions.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Functional analysis</h3>
<p id="Par52">Meta-analytic functional decoding analysis using Neurosynth (<a href="http://www.neurosynth.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.neurosynth.org</a>) database was conducted with NiMARE (<a href="https://github.com/neurostuff/NiMARE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/neurostuff/NiMARE</a>) to capture the most relevant cognitive functions. Specifically, the Neurosynth ROI association method and feature terms were downloaded from the 50 topic terms (v3;<a href="https://github.com/NeuroanatomyAndConnectivity/gradient_analysis/blob/master/gradient_data/neurosynth/v3-topics-50-keys.txt" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/NeuroanatomyAndConnectivity/gradient_analysis/blob/master/gradient_data/neurosynth/v3-topics-50-keys.txt</a>). Then, we combined these data with the corresponding cell type and subclass distribution data using the redundancy analysis function RDA from the vegan package. Hellinger transformation is used to standardize the data. The output data were scaled further using the function fortify from the ggplot2 package, with the scale parameter set to 2, which emphasizes the correlation between cell type/subclass and functions based on an angle rather than distance.</p>
<p id="Par53">Specifically, our first step includes a fMRI meta-analysis by Neurosynth framework on target areas to derive activation scores for each area and function. Neurosynth is a classic framework used for synthesizing human neuroimaging data, employed in hundreds of fMRI studies to perform robust meta-analyses and generate a voxel-activation database<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Neurosynthâ€™s generative mechanism maps diverse term-position relationships from neuroimaging research onto a 2â€‰mm x 2â€‰mm x 2â€‰mm coordinate system<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, and this resolution is sufficient for our 14 cortical regions. Then we selected voxels within each cortical region and assessed the association between specific terms and regions by Neurosynthâ€™s meta-analytic method, which a two-way chi-square test method was used to assess the association between terms and spatial coordinates (namely â€œassociation testâ€, more detail referred to their code line 165 (<a href="https://neurosynth.org/faq/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://neurosynth.org/faq/</a>) and Method in their article<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Then, we selected functional terms highly frequent in Neurosynth, with ample research data support, thus ensuring the robustness of the region-function association.</p>
<p id="Par54">Subsequently, we correlated these regional functional scores with distributions of region-cell types (originating from snRNA data) to examine the correlation between function terms and cell types. We achieve this by employing RDA to establish a connection between conditions (i.e., functional scores in brain regions) and the distribution of cells in different regions. Redundancy analysis allows studying the relationship between two tables (cell distribution-cortical region; cortical region-function association) by sharing a common variable (cortical region). Lastly, strong correlations between cell types and functions were examined using our Stereo-seq data, such as the number of ET neurons is positively correlated with the pain-term relevance of the cortical region. Building on this foundation, we integrate â€˜cortical region-cell type distributionâ€™ with â€˜cortical region-functionâ€™ data. This integration leverages cortical regions as intermediaries, enabling a detailed probe into the â€˜function-cell type distributionâ€™ relationship. In assessing the association between function terms and positions (â€˜cortical region-functionâ€™), we employed a classical and widely used fMRI meta-analysis framework for the related calculations.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Pathway enrichment within cell-types</h3>
<p id="Par55">We used the Seurat function findMarker to determine the differentially expressed genes at the region, subclass, and region-subclass levels, using FDR as multiple corrections. The KEGG and GO terms enrichment based on the DEGs was accomplished with the help of a Clusterprolifier.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Data analysis and visualization of single-cell data</h3>
<p id="Par56">The single-cell transcriptomic data were analyzed or visualized using R (version 3.5.0 and higher, <a href="https://www.R-project.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R-project.org</a>), RStudio IDE (<a href="https://www.rstudio.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rstudio.com/</a>), Seurat<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>, and scrattch suite (<a href="https://github.com/AllenInstitute/scrattch" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/AllenInstitute/scrattch</a>)<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. The cellular compositional analysis was conducted by scCODA<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. All statistics tests without specifying were conducted by the Wilcox test.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Spatial transcriptomic library generation</h3>
<p id="Par57">For Stereo-seq, tissue sections placed on the chip were permeabilized with 0.1% pepsin (Sigma, P7000) in 0.01â€‰M HCl buffer, incubated at 37â€‰Â°C for 5â€‰minutes, and then washed with 0.1x SSC buffer (Thermo, AM9770) supplemented with 0.05 U/ml RNase inhibitor (NEB, M0314L) (NEB, M0314L). RNA released from the permeabilized tissue and captured by the DNB was reverse transcribed overnight at 42â€‰Â°C using SuperScript II (Invitrogen, 18064-014, 10 U/ml reverse transcriptase, 1â€‰mM dNTPs, 1â€‰M betaine solution PCR reagent, 7.5â€‰mM MgCl2, 5â€‰mM DTT, 2 U/ml RNase inhibitor, 2.5â€‰mM Stereo-seq-TSO, and 1x First-Str After reverse transcription, tissue sections were washed twice with 0.1x SSC buffer and digested with Tissue Removal buffer (10â€‰mM Tris-HCl, 25â€‰mM EDTA, 100â€‰mM NaCl, 0.5% SDS) at 55â€‰Â°C for 10â€‰minutes. cDNA-containing chips were then treated overnight at 55â€‰Â°C with Prepare cDNA Release Mix (enzyme, buffer). VAHTSTM DNA Clean Beads (0.8Ã—) purified cDNA. The concentrations of the PCR products were measured using the Qubit TM dsDNA Assay Kit (Thermo, <a href="https://www.ncbi.nlm.nih.gov/protein/Q32854" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q32854</a>). After fragmenting 20â€‰ng of DNA at 55â€‰Â°C for 10â€‰minutes with in-house Tn5 transposase, the reactions were terminated by adding 0.02% SDS and mixing gently at 37â€‰Â°C for 5â€‰minutes. This is how fragmented products were amplified: 25â€‰ml of fragmentation product, 1x KAPA HiFi Hotstart Ready Mix, 0.3â€‰mM Stereo-seq-Library-F primer, 0.3â€‰mM Stereo-seq-Library-R primer, and nuclease-free H2O in a total volume of 100â€‰ml. The reaction was then carried out as follows: 1 cycle of 95â€‰Â°C for 5â€‰minutes, 13 cycles of 98â€‰Â°C for 20â€‰seconds, 58â€‰Â°C for 20â€‰seconds, and 72â€‰Â°C for 30â€‰seconds, and 1 cycle of 72â€‰Â°C for 5â€‰minutes. PCR products were purified with AMPure XP Beads (0.63 and 0.13), utilized for DNB generation, and then sequenced on an MGI DNBSEQ-Tx sequencer.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Spatial transcriptomic data preprocessing</h3>
<p id="Par58">An MGI DNBSEQ-Tx sequencer was used to generate Stereo-seq fastq files. CID and MID are present in read 1 (CID: 1-25â€‰bp, MID: 26-35â€‰bp), whereas read 2 contains the cDNA sequences. First, the CID sequences on the initial reads were mapped to the designed coordinates of the in situ captured chip obtained from the initial round of sequencing, allowing for 1 base mismatch to correct for sequencing and PCR errors. Reads with a MID containing N or more than 2 bases and a quality score below 10 were filtered out. Each readâ€™s CID and MID were appended to the header of each read. Using STAR (Dobin et al., 2013), retained reads were aligned to the reference genome ensemble 93 and mapped reads with MAPQâ€‰&gt;â€‰10 were counted and annotated with their corresponding genes). UMI with the same CID and gene locus were collapsed, allowing for the correction of sequencing and PCR errors with a single mismatch. Finally, this information was used to generate an expression profile matrix containing CIDs. The entire process has been incorporated into the publicly accessible pipeline SAW, which can be found at <a href="https://github.com/BGIResearch/SAW" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/BGIResearch/SAW</a>. The ssDNA-stained nuclei image was harmonized with its CID-tagged expression matrix using StereoCellâ€™s image registration module (SAW defaults)<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. For cell segmentation, we applied the pretrained â€œcytoâ€ Cellposeâ€‰2 model on the registered image (model_typeâ€‰=â€‰â€˜CPâ€™, diameter=30 px, flow_threshold=0, cellprob_threshold=1.0)<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. The resulting binary maskâ€”distinguishing cellular from extracellular areasâ€”was used to filter point-level UMI data (retaining only cell-associated transcripts) and to assign transcriptome profiles to individual, cell-sized bins. UMI-to-cell assignment then employed the FAST method from the Stereopy toolkit (default distance threshold, single-threaded)<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. Finally, per-cell, per-gene UMI counts were aggregated into a cellâ€“gene expression matrix for downstream analysis.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Spatial domain identification and annotation</h3>
<p id="Par59">To identify the spatial domain robustly, we adopted a deep learning framework, to integrate histological and transcriptomic data<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. First, we segmented images of brain tissue stained with hematoxylin-eosin according to spot coordinates to obtain a partial image, and tissue topography was first processed using a pre-trained deep learning network (CNN). Morphological similarity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e1779"><msub><mrow><mi>M</mi><mi>S</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> between adjacent spots was calculated using this matrix, and the weights of gene expression <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e1794"><msub><mrow><mi>G</mi><mi>C</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> and spatial location <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e1809"><msub><mrow><mi>S</mi><mi>W</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> were merged to re-assign a normalized expression value <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e1824"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold">GE</mi></mrow><mo>~</mo></mover></mrow><mrow><mi>i</mi></mrow></msub></math></span> for each gene inside a spot <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e1842"><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span>. The correlation was applied to calculate the weights of spatial gene expression using cosine distances <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e1856"><mi mathvariant="bold">G</mi><msub><mrow><mi mathvariant="bold">C</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> between a spot <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e1873"><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> and a spot <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e1887"><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span>:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e1902" display="block"><mi mathvariant="bold">G</mi><msub><mrow><mi mathvariant="bold">C</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mn>1</mn><mo>âˆ’</mo><mfrac><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>âˆ’</mo><mover accent="true"><mrow><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo accent="true">Â¯</mo></mover></mrow><mo>)</mo></mrow><mo>â‹…</mo><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>âˆ’</mo><mover accent="true"><mrow><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow><mo accent="true">Â¯</mo></mover></mrow><mo>)</mo></mrow></mrow><mrow><msub><mrow><mo>âˆ¥</mo><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>âˆ’</mo><mover accent="true"><mrow><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo accent="true">Â¯</mo></mover></mrow><mo>)</mo></mrow><mo>âˆ¥</mo></mrow><mrow><mn>2</mn></mrow></msub><msub><mrow><mo>âˆ¥</mo><mrow><mo>(</mo><mrow><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>âˆ’</mo><mover accent="true"><mrow><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow><mo accent="true">Â¯</mo></mover></mrow><mo>)</mo></mrow><mo>âˆ¥</mo></mrow><mrow><mn>2</mn></mrow></msub></mrow></mfrac><mo>.</mo></math></td>
<td class="label">1</td>
</tr></table>
<p id="Par60">To represent the spot morphology, we used principal component analysis to extract the first 50 principal components as latent characteristics. The weights of morphological similarity <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e2024"><mi mathvariant="bold">M</mi><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> between a spot <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e2041"><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> and its adjacent spots <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e2055"><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span> were calculated using the cosine distance:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e2070" display="block"><mi mathvariant="bold">M</mi><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mn>1</mn><mo>âˆ’</mo><mfrac><mrow><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>â‹…</mo><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow><mrow><msub><mrow><mo>âˆ¥</mo><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>âˆ¥</mo></mrow><mrow><mn>2</mn></mrow></msub><msub><mrow><mo>âˆ¥</mo><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>âˆ¥</mo></mrow><mrow><mn>2</mn></mrow></msub></mrow></mfrac><mo>.</mo></math></td>
<td class="label">2</td>
</tr></table>
<p id="Par61">We used spatial coordinates to determine the Euclidean distance between each spot and the remainder, then ordered the distances between the top 3 (optional) adjacent spots to count the radius <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e2132"><mi>Î³</mi></math></span> (mean add variance). For a given spot <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e2141"><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span>, a spot <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e2155"><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span> was a neighbor, so <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e2169"><mi mathvariant="bold">S</mi><msub><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mn>1</mn></math></span> if and only if the Euclidean between two spots is less than the set value. Otherwise <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e2188"><mi mathvariant="bold">S</mi><msub><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mn>2</mn></math></span>.</p>
<p id="Par62">We normalized the gene expression <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e2209"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold">GE</mi></mrow><mo>~</mo></mover></mrow><mrow><mi>i</mi></mrow></msub></math></span> of each spot <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e2226"><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> by incorporating gene expression correlation, spatial neighbors, and morphological similarity:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e2241" display="block"><msub><mrow><mover accent="true"><mrow><mi mathvariant="bold">GE</mi></mrow><mo>~</mo></mover></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><mi mathvariant="bold">G</mi><msub><mrow><mi mathvariant="bold">E</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>+</mo><mfrac><mrow><msubsup><mrow><mo>âˆ‘</mo></mrow><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mspace width="0.3em"></mspace><mi mathvariant="bold">G</mi><msub><mrow><mi mathvariant="bold">E</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>â‹…</mo><mi mathvariant="bold">M</mi><msub><mrow><mi mathvariant="bold">S</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>â‹…</mo><mi mathvariant="bold">G</mi><msub><mrow><mi mathvariant="bold">C</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>â‹…</mo><mi mathvariant="bold">S</mi><msub><mrow><mi mathvariant="bold">W</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow><mrow><mi>n</mi></mrow></mfrac><mo>,</mo></math></td>
<td class="label">3</td>
</tr></table>
<p id="Par63">This also computed a graph adjacency matrix based on spatial coordinates by k-nearest neighbors (KNNs). The number of nearest neighbors was 15. Second, a denoising autoencoder was employed to perform nonlinear mapping on the integrated feature space, thereby generating a low-dimensional representation space that would reduce the risk of model overfitting. Simultaneously, a variational graph autoencoder was inserted into the framework to map spatial associations of spots, generating spatial embedding via integrated representation with the corresponding spatial adjacent spots.</p>
<p id="Par64">The model was trained according to the following parameters: number of model training, 1000 epochs; the number of dimensions of the normalized gene expression matrix (<em>concat_pca_dim)</em>, 50; neural network type, graph convolutional neural network (<em>Conv_type=â€™GCNConvâ€™</em>); and several integrated neighbors (<em>neighbour_k)</em>, 2. Other parameters were default.</p>
<p id="Par65">The domain spot clustering used 4 integrated neighbors (neighbour_kâ€‰=â€‰4) with an adjacent weight of 0.3 for data augmentation, and a KDTree-based graph construction with kâ€‰=â€‰20 neighbors (weights_matrix_nomd). The neural network architecture included linear encoder hidden layers of [64, 16] and convolutional hidden layers of [64, 16]. Pretraining and dimension reduction were enabled, while other parameters were set to their defaults. The model was trained with the output saved to a designated results directory, facilitating downstream spatial analysis.</p>
<p id="Par66">Finally, we obtained a 28-dimensional vector for each spot, for which we computed the Leiden clustering using Scanpy<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. The clustering resolution varied adaptively with the number of spatial domains; it was 7 for human cortical regions.</p>
<p id="Par67">Based on this primary result, we further corrected and annotated spatial domains manually in terms of their structural features, layer marker genes, Nissl-stained images, and cell type deconvolution.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Cell-type deconvolution and Cellular communication</h3>
<p id="Par68">We deconvolute normalized spatial transcriptomic data with the transcriptomic spectrum of each cell type by RCTD<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> (mode = â€œdoubletâ€, filtered out â€œrejectâ€ spot_class and first type as the annotation) to achieve robust results for Stereo-seq data. We selected the top 20 marker genes of pairwise 153 cell types and mapped all cell-type snRNA-seq profiles in spatial spots by a total of 4637 genes. Default values were used for other parameter settings in all analyses. We calculated the probability of each cell type in each spot to calculate their subclass proportion in each pot to draw scatter pie plots. For the cell-type probability boxplot, we calculated the average probability of each cell type in different layers of each slide and performed ANOVA to measure global relatedness and the Wilcox test for specific pairwise comparisons. For the laminar distribution and relative depth, we set 1~7 as the relative depth value for layer 1 to white matter and calculated the standard deviation of probability based on these. For each cell type, we used kernel density estimation to smooth local probability to get weighted density by <em>kde2d.weighted</em> in package ggtern and visualized by <em>geom_raster</em> in ggplot2.</p>
<p id="Par69">To analyze the cellular communication, we first utilized CellChat<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>, a computational framework designed for deciphering intercellular communication networks, to predict one-to-one cell-type interactions in single-cell data. CellChat utilizes a database of ligand-receptor interactions to infer potential signaling pathways and quantify their strengths, providing insights into the communication patterns among various cell types. CellChat was initialized with normalized gene expression data, grouped by subclass (group.by = â€œsubclassâ€), using the human ligand-receptor database (CellChatDB.human). The analysis included subsetting data, identifying overexpressed genes and interactions, and computing communication probabilities using the triMean method with 20 bootstrap iterations (nboot = 20, default). Communication pairs were filtered to include at least 10 cells (min.cells = 10, default). Parallel processing was enabled with 4 workers and a 20 GB memory limit.</p>
<p id="Par70">Following the prediction of cell-cell interactions, we considered two cells within a 15-micrometer range of each other as having somatic contact in spatial data. Based on the cell contacts obtained from all spatial transcriptomic data, we classified them into linked subnetwork units. We then employed the Sequential Pattern Mining Framework (SPMF)<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup> to statistically analyze the frequency of occurrence of connected patterns ranging from 2 to 7 orders (i.e., structures composed of up to 7 interconnected cells).</p></section><section id="Sec30"><h3 class="pmc_sec_title">Spatial transcriptome UMAP</h3>
<p id="Par71">We used the UMAP algorithm from the package Seurat<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> to get the reduction of dimension by treating different samples as different batches and retaining default values for all other parameters. First, we normalized spatial gene expression by integrating spatial location information and tissue morphological information (see â€œSpatial domain identificationâ€ above) and then used principal component analysis to obtain a low-dimensional map of the normalized data, with a default of 50 dimensions. Second, the results were corrected for batch effects using Harmony<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> (lambda = 1, theta = 2, donor and chip as batch factors). The processed data were used to train the model, and this will generate a nonlinear, low-dimensional map of the integrated data. Finally, we applied the UMAP algorithm to this data to generate the 2-dimensional projections.</p></section><section id="Sec31"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par72">Further information on research design is available in theÂ <a href="#MOESM9" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec32"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Infomation</a><sup> (79MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62793_MOESM2_ESM.pdf</a><sup> (95KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 1</a><sup> (59.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 2</a><sup> (49.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 3</a><sup> (13.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 4</a><sup> (788.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM7_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 5</a><sup> (43.6MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM8_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 6</a><sup> (3.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (3.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (9.9MB, pdf) </sup>
</div></div></section></section><section id="Sec33"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM11_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (76.7MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This paper is from the Mesoscopic Brain Mapping Consortium. This work was supported by the National Natural Science Foundation of China (Nos. T2325009) and National Key R&amp;D Program of China (Nos. 2023YFF1204802). We thank NovelBio Bio-Pharm Technology Co., Ltd. (Shanghai, China) for the support of single-cell nucleus sequencing (NovelBrain Cloud Analysis Platform, <a href="http://www.novelbrain.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.novelbrain.com</a>).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: Q.Jiang., J.Xu. and H.Sun.; snRNA-seq and spatial transcriptomic data generation: S.Wei., M.Luo., C.Li., P.Huang., Z.Liu., J.Xu., and Q.Jiang.; Data analysis: S.Wei., M.Luo., P.Wang., X.Jin., C.Xu., C.Li., X.Lin., Z.Xu., H.Liu., R.Cheng., W.Yang., Y.Cai., G.Xue., H.Sun., and Q.Jiang.; Data interpretation: S.Wei., M.Luo., P.Wang., X.Jin., C.Xu., C.Li., X.Lin., H.Sun., and Q.Jiang.; Write manuscript: S.Wei., M.Luo., P.Wang., R.Chen., J.Xu., and Q.Jiang.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par73"><em>Nature Communications</em> thanks Zhichao Miao and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The sequencing data generated in this study have been deposited in the stomicsDB under accession code <a href="https://db.cngb.org/stomics/datasets/STDS0000242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">STDS0000242</a><sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. The immunohistochemical staining images generated in this study have been deposited in Figshare under the accession code <a href="https://doi.org/10.6084/m9.figshare.c.6873943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6873943</a>.Â <a href="#Sec33" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>All codes are available on <a href="https://github.com/lcy1364/Cortex-Atlas-Code" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/lcy1364/Cortex-Atlas-Code</a>. (10.5281/zenodo.14311048)<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup></p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par74">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisherâ€™s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Songren Wei, Meng Luo, Pingping Wang, Rui Chen.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Haoxiu Sun, Email: sunhaoxiu@hrbmu.edu.cn.</p>
<p>Jiangping Xu, Email: jpx@smu.edu.cn.</p>
<p>Qinghua Jiang, Email: qhjiang@hrbmu.edu.cn.</p></section><section id="sec34"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62793-9.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Rakic, P. Evolution of the neocortex: a perspective from developmental biology. <em>Nat. Rev. Neurosci.</em><strong>10</strong>, 724â€“735 (2009).
</cite> [<a href="https://doi.org/10.1038/nrn2719" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2913577/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19763105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rakic,%20P.%20Evolution%20of%20the%20neocortex:%20a%20perspective%20from%20developmental%20biology.%20Nat.%20Rev.%20Neurosci.10,%20724%E2%80%93735%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Van Essen, D. C. &amp; Glasser, M. F. Parcellating cerebral cortex: how invasive animal studies inform noninvasive mapmaking in humans. <em>Neuron</em><strong>99</strong>, 640â€“663 (2018).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2018.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6149530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30138588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Essen,%20D.%20C.%20&amp;%20Glasser,%20M.%20F.%20Parcellating%20cerebral%20cortex:%20how%20invasive%20animal%20studies%20inform%20noninvasive%20mapmaking%20in%20humans.%20Neuron99,%20640%E2%80%93663%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Zeng, H. &amp; Sanes, J. R. Neuronal cell-type classification: challenges, opportunities and the path forward. <em>Nat. Rev. Neurosci.</em><strong>18</strong>, 530â€“546 (2017).
</cite> [<a href="https://doi.org/10.1038/nrn.2017.85" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28775344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20H.%20&amp;%20Sanes,%20J.%20R.%20Neuronal%20cell-type%20classification:%20challenges,%20opportunities%20and%20the%20path%20forward.%20Nat.%20Rev.%20Neurosci.18,%20530%E2%80%93546%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Hodge, R. D. et al. Conserved cell types with divergent features in human versus mouse cortex. <em>Nature</em><strong>573</strong>, 61â€“68 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-1506-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6919571/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31435019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hodge,%20R.%20D.%20et%20al.%20Conserved%20cell%20types%20with%20divergent%20features%20in%20human%20versus%20mouse%20cortex.%20Nature573,%2061%E2%80%9368%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Lake, B. B. et al. Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain. <em>Science</em><strong>352</strong>, 1586â€“1590 (2016).
</cite> [<a href="https://doi.org/10.1126/science.aaf1204" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5038589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27339989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lake,%20B.%20B.%20et%20al.%20Neuronal%20subtypes%20and%20diversity%20revealed%20by%20single-nucleus%20RNA%20sequencing%20of%20the%20human%20brain.%20Science352,%201586%E2%80%931590%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Tasic, B. et al. Adult mouse cortical cell taxonomy revealed by single cell transcriptomics. <em>Nat. Neurosci.</em><strong>19</strong>, 335â€“346 (2016).
</cite> [<a href="https://doi.org/10.1038/nn.4216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4985242/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26727548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tasic,%20B.%20et%20al.%20Adult%20mouse%20cortical%20cell%20taxonomy%20revealed%20by%20single%20cell%20transcriptomics.%20Nat.%20Neurosci.19,%20335%E2%80%93346%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Tasic, B. et al. Shared and distinct transcriptomic cell types across neocortical areas. <em>Nature</em><strong>563</strong>, 72â€“78 (2018).
</cite> [<a href="https://doi.org/10.1038/s41586-018-0654-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6456269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30382198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tasic,%20B.%20et%20al.%20Shared%20and%20distinct%20transcriptomic%20cell%20types%20across%20neocortical%20areas.%20Nature563,%2072%E2%80%9378%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zeisel, A. et al. Brain structure. Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq. <em>Science</em><strong>347</strong>, 1138â€“1142 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aaa1934" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25700174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeisel,%20A.%20et%20al.%20Brain%20structure.%20Cell%20types%20in%20the%20mouse%20cortex%20and%20hippocampus%20revealed%20by%20single-cell%20RNA-seq.%20Science347,%201138%E2%80%931142%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Han, L. et al. Cell transcriptomic atlas of the non-human primate Macaca fascicularis. <em>Nature</em><strong>604</strong>, 723â€“731 (2022).
</cite> [<a href="https://doi.org/10.1038/s41586-022-04587-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35418686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han,%20L.%20et%20al.%20Cell%20transcriptomic%20atlas%20of%20the%20non-human%20primate%20Macaca%20fascicularis.%20Nature604,%20723%E2%80%93731%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Yao, Z. et al. A taxonomy of transcriptomic cell types across the isocortex and hippocampal formation. <em>Cell</em><strong>184</strong>, 3222â€“3241.e3226 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.04.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8195859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34004146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yao,%20Z.%20et%20al.%20A%20taxonomy%20of%20transcriptomic%20cell%20types%20across%20the%20isocortex%20and%20hippocampal%20formation.%20Cell184,%203222%E2%80%933241.e3226%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Chen, X. et al. A brain cell atlas integrating single-cell transcriptomes across human brain regions. <em>Nat. Med.</em> (2024).</cite> [<a href="https://doi.org/10.1038/s41591-024-03150-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11405287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39095595/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Callaway, E. M. et al. A multimodal cell census and atlas of the mammalian primary motor cortex. <em>Nature</em><strong>598</strong>, 86â€“102 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03950-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8494634/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34616075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Callaway,%20E.%20M.%20et%20al.%20A%20multimodal%20cell%20census%20and%20atlas%20of%20the%20mammalian%20primary%20motor%20cortex.%20Nature598,%2086%E2%80%93102%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Yarkoni, T., Poldrack, R. A., Nichols, T. E., Van Essen, D. C. &amp; Wager, T. D. Large-scale automated synthesis of human functional neuroimaging data. <em>Nat. Methods</em><strong>8</strong>, 665â€“670 (2011).
</cite> [<a href="https://doi.org/10.1038/nmeth.1635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3146590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21706013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yarkoni,%20T.,%20Poldrack,%20R.%20A.,%20Nichols,%20T.%20E.,%20Van%20Essen,%20D.%20C.%20&amp;%20Wager,%20T.%20D.%20Large-scale%20automated%20synthesis%20of%20human%20functional%20neuroimaging%20data.%20Nat.%20Methods8,%20665%E2%80%93670%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>MolnÃ¡r, Z. et al. New insights into the development of the human cerebral cortex. <em>J. Anat.</em><strong>235</strong>, 432â€“451 (2019).
</cite> [<a href="https://doi.org/10.1111/joa.13055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6704245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31373394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moln%C3%A1r,%20Z.%20et%20al.%20New%20insights%20into%20the%20development%20of%20the%20human%20cerebral%20cortex.%20J.%20Anat.235,%20432%E2%80%93451%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Bakken, T. E. et al. Comparative cellular analysis of motor cortex in human, marmoset and mouse. <em>Nature</em><strong>598</strong>, 111â€“119 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03465-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8494640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34616062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bakken,%20T.%20E.%20et%20al.%20Comparative%20cellular%20analysis%20of%20motor%20cortex%20in%20human,%20marmoset%20and%20mouse.%20Nature598,%20111%E2%80%93119%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Jorstad, N. L. et al. Transcriptomic cytoarchitecture reveals principles of human neocortex organization. <em>Science</em><strong>382</strong>, eadf6812 (2023).
</cite> [<a href="https://doi.org/10.1126/science.adf6812" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11687949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37824655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jorstad,%20N.%20L.%20et%20al.%20Transcriptomic%20cytoarchitecture%20reveals%20principles%20of%20human%20neocortex%20organization.%20Science382,%20eadf6812%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Siletti, K. et al. Transcriptomic diversity of cell types across the adult human brain. <em>Science</em><strong>382</strong>, eadd7046 (2023).
</cite> [<a href="https://doi.org/10.1126/science.add7046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37824663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Siletti,%20K.%20et%20al.%20Transcriptomic%20diversity%20of%20cell%20types%20across%20the%20adult%20human%20brain.%20Science382,%20eadd7046%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Lee, B. R. et al. Signature morphoelectric properties of diverse GABAergic interneurons in the human neocortex. <em>Science</em><strong>382</strong>, eadf6484 (2023).
</cite> [<a href="https://doi.org/10.1126/science.adf6484" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12229143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37824669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20B.%20R.%20et%20al.%20Signature%20morphoelectric%20properties%20of%20diverse%20GABAergic%20interneurons%20in%20the%20human%20neocortex.%20Science382,%20eadf6484%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Johansen, N. et al. Interindividual variation in human cortical cell type abundance and expression. <em>Science</em><strong>382</strong>, eadf2359 (2023).
</cite> [<a href="https://doi.org/10.1126/science.adf2359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11702338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37824649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johansen,%20N.%20et%20al.%20Interindividual%20variation%20in%20human%20cortical%20cell%20type%20abundance%20and%20expression.%20Science382,%20eadf2359%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Zeng, H. What is a cell type and how to define it?. <em>Cell</em><strong>185</strong>, 2739â€“2755 (2022).
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.06.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9342916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35868277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20H.%20What%20is%20a%20cell%20type%20and%20how%20to%20define%20it?.%20Cell185,%202739%E2%80%932755%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Fang, R. et al. Conservation and divergence of cortical cell organization in human and mouse revealed by MERFISH. <em>Science</em><strong>377</strong>, 56â€“62 (2022).
</cite> [<a href="https://doi.org/10.1126/science.abm1741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9262715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35771910/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang,%20R.%20et%20al.%20Conservation%20and%20divergence%20of%20cortical%20cell%20organization%20in%20human%20and%20mouse%20revealed%20by%20MERFISH.%20Science377,%2056%E2%80%9362%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Belichenko, P. V., Vogt Weisenhorn, D. M., MyklÃ³ssy, J. &amp; Celio, M. R. Calretinin-positive Cajal-Retzius cells persist in the adult human neocortex. <em>Neuroreport</em><strong>6</strong>, 1869â€“1874 (1995).
</cite> [<a href="https://doi.org/10.1097/00001756-199510020-00012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8547587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Belichenko,%20P.%20V.,%20Vogt%20Weisenhorn,%20D.%20M.,%20Mykl%C3%B3ssy,%20J.%20&amp;%20Celio,%20M.%20R.%20Calretinin-positive%20Cajal-Retzius%20cells%20persist%20in%20the%20adult%20human%20neocortex.%20Neuroreport6,%201869%E2%80%931874%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Gur, R. C. et al. Sex differences in brain gray and white matter in healthy young adults: correlations with cognitive performance. <em>J. Neurosci.</em><strong>19</strong>, 4065â€“4072 (1999).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.19-10-04065.1999" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6782697/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10234034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gur,%20R.%20C.%20et%20al.%20Sex%20differences%20in%20brain%20gray%20and%20white%20matter%20in%20healthy%20young%20adults:%20correlations%20with%20cognitive%20performance.%20J.%20Neurosci.19,%204065%E2%80%934072%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Mathys, H. et al. Single-cell transcriptomic analysis of Alzheimerâ€™s disease. <em>Nature</em><strong>570</strong>, 332â€“337 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-1195-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6865822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31042697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mathys,%20H.%20et%20al.%20Single-cell%20transcriptomic%20analysis%20of%20Alzheimer%E2%80%99s%20disease.%20Nature570,%20332%E2%80%93337%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Xu, C. et al. DeepST: identifying spatial domains in spatial transcriptomics by deep learning. <em>Nucleic Acids Res.</em><strong>50</strong>, e131 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkac901" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36250636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20C.%20et%20al.%20DeepST:%20identifying%20spatial%20domains%20in%20spatial%20transcriptomics%20by%20deep%20learning.%20Nucleic%20Acids%20Res.50,%20e131%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Berg, J. et al. Human neocortical expansion involves glutamatergic neuron diversification. <em>Nature</em><strong>598</strong>, 151â€“158 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03813-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8494638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34616067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berg,%20J.%20et%20al.%20Human%20neocortical%20expansion%20involves%20glutamatergic%20neuron%20diversification.%20Nature598,%20151%E2%80%93158%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Pinson, A. &amp; Huttner, W. B. Neocortex expansion in development and evolution-from genes to progenitor cell biology. <em>Curr. Opin. Cell Biol.</em><strong>73</strong>, 9â€“18 (2021).
</cite> [<a href="https://doi.org/10.1016/j.ceb.2021.04.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34098196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pinson,%20A.%20&amp;%20Huttner,%20W.%20B.%20Neocortex%20expansion%20in%20development%20and%20evolution-from%20genes%20to%20progenitor%20cell%20biology.%20Curr.%20Opin.%20Cell%20Biol.73,%209%E2%80%9318%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Harris, K. D. &amp; Shepherd, G. M. The neocortical circuit: themes and variations. <em>Nat. Neurosci.</em><strong>18</strong>, 170â€“181 (2015).
</cite> [<a href="https://doi.org/10.1038/nn.3917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4889215/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25622573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harris,%20K.%20D.%20&amp;%20Shepherd,%20G.%20M.%20The%20neocortical%20circuit:%20themes%20and%20variations.%20Nat.%20Neurosci.18,%20170%E2%80%93181%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Petreanu, L., Mao, T., Sternson, S. M. &amp; Svoboda, K. The subcellular organization of neocortical excitatory connections. <em>Nature</em><strong>457</strong>, 1142â€“1145 (2009).
</cite> [<a href="https://doi.org/10.1038/nature07709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2745650/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19151697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petreanu,%20L.,%20Mao,%20T.,%20Sternson,%20S.%20M.%20&amp;%20Svoboda,%20K.%20The%20subcellular%20organization%20of%20neocortical%20excitatory%20connections.%20Nature457,%201142%E2%80%931145%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Molyneaux, B. J., Arlotta, P., Menezes, J. R. &amp; Macklis, J. D. Neuronal subtype specification in the cerebral cortex. <em>Nat. Rev. Neurosci.</em><strong>8</strong>, 427â€“437 (2007).
</cite> [<a href="https://doi.org/10.1038/nrn2151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17514196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Molyneaux,%20B.%20J.,%20Arlotta,%20P.,%20Menezes,%20J.%20R.%20&amp;%20Macklis,%20J.%20D.%20Neuronal%20subtype%20specification%20in%20the%20cerebral%20cortex.%20Nat.%20Rev.%20Neurosci.8,%20427%E2%80%93437%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Hodge, R. D. et al. Transcriptomic evidence that von Economo neurons are regionally specialized extratelencephalic-projecting excitatory neurons. <em>Nat. Commun.</em><strong>11</strong>, 1172 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-14952-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7054400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32127543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hodge,%20R.%20D.%20et%20al.%20Transcriptomic%20evidence%20that%20von%20Economo%20neurons%20are%20regionally%20specialized%20extratelencephalic-projecting%20excitatory%20neurons.%20Nat.%20Commun.11,%201172%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Cobos, I. &amp; Seeley, W. W. Human von Economo neurons express transcription factors associated with Layer V subcerebral projection neurons. <em>Cereb. Cortex</em><strong>25</strong>, 213â€“220 (2015).
</cite> [<a href="https://doi.org/10.1093/cercor/bht219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4318933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23960210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cobos,%20I.%20&amp;%20Seeley,%20W.%20W.%20Human%20von%20Economo%20neurons%20express%20transcription%20factors%20associated%20with%20Layer%20V%20subcerebral%20projection%20neurons.%20Cereb.%20Cortex25,%20213%E2%80%93220%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Baker, A. et al. Specialized Subpopulations of deep-layer pyramidal neurons in the neocortex: bridging cellular properties to functional consequences. <em>J. Neurosci.</em><strong>38</strong>, 5441â€“5455 (2018).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.0150-18.2018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6001033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29798890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baker,%20A.%20et%20al.%20Specialized%20Subpopulations%20of%20deep-layer%20pyramidal%20neurons%20in%20the%20neocortex:%20bridging%20cellular%20properties%20to%20functional%20consequences.%20J.%20Neurosci.38,%205441%E2%80%935455%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Mills, J. D. et al. LINC00507 Is Specifically Expressed in the Primate Cortex and Has Age-Dependent Expression Patterns. <em>J. Mol. Neurosci.</em><strong>59</strong>, 431â€“439 (2016).
</cite> [<a href="https://doi.org/10.1007/s12031-016-0745-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27059230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mills,%20J.%20D.%20et%20al.%20LINC00507%20Is%20Specifically%20Expressed%20in%20the%20Primate%20Cortex%20and%20Has%20Age-Dependent%20Expression%20Patterns.%20J.%20Mol.%20Neurosci.59,%20431%E2%80%93439%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Christiaens, D. et al. Global tractography of multi-shell diffusion-weighted imaging data using a multi-tissue model. <em>Neuroimage</em><strong>123</strong>, 89â€“101 (2015).
</cite> [<a href="https://doi.org/10.1016/j.neuroimage.2015.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26272729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Christiaens,%20D.%20et%20al.%20Global%20tractography%20of%20multi-shell%20diffusion-weighted%20imaging%20data%20using%20a%20multi-tissue%20model.%20Neuroimage123,%2089%E2%80%93101%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Salo, T. et al. NiMARE: Neuroimaging Meta-Analysis Research Environment. Aperture Neuro <strong>3</strong>, 1â€“32 (2023).</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Zhang, X. H. et al. The cell-type underpinnings of the human functional cortical connectome. <em>Nat. Neurosci.</em><strong>28</strong>, 150â€“160 (2025).
</cite> [<a href="https://doi.org/10.1038/s41593-024-01812-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39572742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20X.%20H.%20et%20al.%20The%20cell-type%20underpinnings%20of%20the%20human%20functional%20cortical%20connectome.%20Nat.%20Neurosci.28,%20150%E2%80%93160%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Yuan, J. et al. Single-nucleus multi-omics analyses reveal cellular and molecular innovations in the anterior cingulate cortex during primate evolution. <em>Cell Genom.</em> 100703, (2024).</cite> [<a href="https://doi.org/10.1016/j.xgen.2024.100703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701334/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39631404/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Rowitch, D. H. &amp; Kriegstein, A. R. Developmental genetics of vertebrate glial-cell specification. <em>Nature</em><strong>468</strong>, 214â€“222 (2010).
</cite> [<a href="https://doi.org/10.1038/nature09611" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21068830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rowitch,%20D.%20H.%20&amp;%20Kriegstein,%20A.%20R.%20Developmental%20genetics%20of%20vertebrate%20glial-cell%20specification.%20Nature468,%20214%E2%80%93222%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Caglayan, E. et al. Molecular features driving cellular complexity of human brain evolution. <em>Nature</em><strong>620</strong>, 145â€“153 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06338-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11161302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37468639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Caglayan,%20E.%20et%20al.%20Molecular%20features%20driving%20cellular%20complexity%20of%20human%20brain%20evolution.%20Nature620,%20145%E2%80%93153%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Clayton, B. L. L. &amp; Tesar, P. J. Oligodendrocyte progenitor cell fate and function in development and disease. <em>Curr. Opin. Cell Biol.</em><strong>73</strong>, 35â€“40 (2021).
</cite> [<a href="https://doi.org/10.1016/j.ceb.2021.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8678156/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34153742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clayton,%20B.%20L.%20L.%20&amp;%20Tesar,%20P.%20J.%20Oligodendrocyte%20progenitor%20cell%20fate%20and%20function%20in%20development%20and%20disease.%20Curr.%20Opin.%20Cell%20Biol.73,%2035%E2%80%9340%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Tremblay, R., Lee, S. &amp; Rudy, B. GABAergic interneurons in the neocortex: from cellular properties to circuits. <em>Neuron</em><strong>91</strong>, 260â€“292 (2016).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2016.06.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4980915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27477017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tremblay,%20R.,%20Lee,%20S.%20&amp;%20Rudy,%20B.%20GABAergic%20interneurons%20in%20the%20neocortex:%20from%20cellular%20properties%20to%20circuits.%20Neuron91,%20260%E2%80%93292%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Inan, M., Welagen, J. &amp; Anderson, S. A. Spatial and temporal bias in the mitotic origins of somatostatin- and parvalbumin-expressing interneuron subgroups and the chandelier subtype in the medial ganglionic eminence. <em>Cereb. Cortex</em><strong>22</strong>, 820â€“827 (2012).
</cite> [<a href="https://doi.org/10.1093/cercor/bhr148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3450921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21693785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Inan,%20M.,%20Welagen,%20J.%20&amp;%20Anderson,%20S.%20A.%20Spatial%20and%20temporal%20bias%20in%20the%20mitotic%20origins%20of%20somatostatin-%20and%20parvalbumin-expressing%20interneuron%20subgroups%20and%20the%20chandelier%20subtype%20in%20the%20medial%20ganglionic%20eminence.%20Cereb.%20Cortex22,%20820%E2%80%93827%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Miyoshi, G., Machold, R. P. &amp; Fishell, G. in <em>Cortical Development: Neural Diversity and Neocortical Organization</em> (eds R. Kageyama &amp; T. Yamamori) 89-126 (Springer Japan, 2013).</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite>Kessaris, N. Human cortical interneuron development unraveled. <em>Science</em><strong>375</strong>, 383â€“384 (2022).
</cite> [<a href="https://doi.org/10.1126/science.abn6333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35084977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kessaris,%20N.%20Human%20cortical%20interneuron%20development%20unraveled.%20Science375,%20383%E2%80%93384%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>LibÃ©-Philippot, B. &amp; Vanderhaeghen, P. Cellular and molecular mechanisms linking human cortical development and evolution. <em>Annu. Rev. Genet.</em><strong>55</strong>, 555â€“581 (2021).
</cite> [<a href="https://doi.org/10.1146/annurev-genet-071719-020705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34535062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lib%C3%A9-Philippot,%20B.%20&amp;%20Vanderhaeghen,%20P.%20Cellular%20and%20molecular%20mechanisms%20linking%20human%20cortical%20development%20and%20evolution.%20Annu.%20Rev.%20Genet.55,%20555%E2%80%93581%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Nery, S., Fishell, G. &amp; Corbin, J. G. The caudal ganglionic eminence is a source of distinct cortical and subcortical cell populations. <em>Nat. Neurosci.</em><strong>5</strong>, 1279â€“1287 (2002).
</cite> [<a href="https://doi.org/10.1038/nn971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12411960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nery,%20S.,%20Fishell,%20G.%20&amp;%20Corbin,%20J.%20G.%20The%20caudal%20ganglionic%20eminence%20is%20a%20source%20of%20distinct%20cortical%20and%20subcortical%20cell%20populations.%20Nat.%20Neurosci.5,%201279%E2%80%931287%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Zhang, M. et al. Spatially resolved cell atlas of the mouse primary motor cortex by MERFISH. <em>Nature</em><strong>598</strong>, 137â€“143 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03705-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8494645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34616063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20M.%20et%20al.%20Spatially%20resolved%20cell%20atlas%20of%20the%20mouse%20primary%20motor%20cortex%20by%20MERFISH.%20Nature598,%20137%E2%80%93143%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Lim, L. et al. Optimization of interneuron function by direct coupling of cell migration and axonal targeting. <em>Nat. Neurosci.</em><strong>21</strong>, 920â€“931 (2018).
</cite> [<a href="https://doi.org/10.1038/s41593-018-0162-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6061935/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29915195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lim,%20L.%20et%20al.%20Optimization%20of%20interneuron%20function%20by%20direct%20coupling%20of%20cell%20migration%20and%20axonal%20targeting.%20Nat.%20Neurosci.21,%20920%E2%80%93931%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Li, W. et al. Molecular and cellular dynamics of the developing human neocortex at single-cell resolution. <em>bioRxiv</em>, 2024.2001.2016.575956, (2024).</cite>
</li>
<li id="CR51">
<span class="label">51.</span><cite>Leng, K. et al. Molecular characterization of selectively vulnerable neurons in Alzheimerâ€™s disease. <em>Nat. Neurosci.</em><strong>24</strong>, 276â€“287 (2021).
</cite> [<a href="https://doi.org/10.1038/s41593-020-00764-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7854528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33432193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leng,%20K.%20et%20al.%20Molecular%20characterization%20of%20selectively%20vulnerable%20neurons%20in%20Alzheimer%E2%80%99s%20disease.%20Nat.%20Neurosci.24,%20276%E2%80%93287%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Bernard, A. et al. Transcriptional architecture of the primate neocortex. <em>Neuron</em><strong>73</strong>, 1083â€“1099 (2012).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2012.03.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3628746/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22445337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bernard,%20A.%20et%20al.%20Transcriptional%20architecture%20of%20the%20primate%20neocortex.%20Neuron73,%201083%E2%80%931099%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Hoerder-Suabedissen, A. &amp; MolnÃ¡r, Z. Molecular diversity of early-born subplate neurons. <em>Cereb. Cortex</em><strong>23</strong>, 1473â€“1483 (2013).
</cite> [<a href="https://doi.org/10.1093/cercor/bhs137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22628460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoerder-Suabedissen,%20A.%20&amp;%20Moln%C3%A1r,%20Z.%20Molecular%20diversity%20of%20early-born%20subplate%20neurons.%20Cereb.%20Cortex23,%201473%E2%80%931483%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Zolnik, T. A. et al. Layer 6b is driven by intracortical long-range projection neurons. <em>Cell Rep.</em><strong>30</strong>, 3492â€“3505.e3495 (2020).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2020.02.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32160552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zolnik,%20T.%20A.%20et%20al.%20Layer%206b%20is%20driven%20by%20intracortical%20long-range%20projection%20neurons.%20Cell%20Rep.30,%203492%E2%80%933505.e3495%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Reep, R. L. Cortical layer VII and persistent subplate cells in mammalian brains. <em>Brain Behav. Evol.</em><strong>56</strong>, 212â€“234 (2000).
</cite> [<a href="https://doi.org/10.1159/000047206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11155000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reep,%20R.%20L.%20Cortical%20layer%20VII%20and%20persistent%20subplate%20cells%20in%20mammalian%20brains.%20Brain%20Behav.%20Evol.56,%20212%E2%80%93234%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Viswanathan, S., Sheikh, A., Looger, L. L. &amp; Kanold, P. O. Molecularly defined subplate neurons project both to thalamocortical recipient layers and Thalamus. <em>Cereb. Cortex</em><strong>27</strong>, 4759â€“4768 (2017).
</cite> [<a href="https://doi.org/10.1093/cercor/bhw271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6059176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27655928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Viswanathan,%20S.,%20Sheikh,%20A.,%20Looger,%20L.%20L.%20&amp;%20Kanold,%20P.%20O.%20Molecularly%20defined%20subplate%20neurons%20project%20both%20to%20thalamocortical%20recipient%20layers%20and%20Thalamus.%20Cereb.%20Cortex27,%204759%E2%80%934768%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Ben-Simon, Y. et al. A direct excitatory projection from entorhinal layer 6b neurons to the hippocampus contributes to spatial coding and memory. <em>Nat. Commun.</em><strong>13</strong>, 4826 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-32559-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9381769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35974109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ben-Simon,%20Y.%20et%20al.%20A%20direct%20excitatory%20projection%20from%20entorhinal%20layer%206b%20neurons%20to%20the%20hippocampus%20contributes%20to%20spatial%20coding%20and%20memory.%20Nat.%20Commun.13,%204826%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Matho, K. S. et al. Genetic dissection of the glutamatergic neuron system in cerebral cortex. <em>Nature</em><strong>598</strong>, 182â€“187 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03955-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8494647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34616069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matho,%20K.%20S.%20et%20al.%20Genetic%20dissection%20of%20the%20glutamatergic%20neuron%20system%20in%20cerebral%20cortex.%20Nature598,%20182%E2%80%93187%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Zeng, H. et al. Spatially resolved single-cell translatomics at molecular resolution. <em>Science</em><strong>380</strong>, eadd3067 (2023).
</cite> [<a href="https://doi.org/10.1126/science.add3067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11146668/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37384709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20H.%20et%20al.%20Spatially%20resolved%20single-cell%20translatomics%20at%20molecular%20resolution.%20Science380,%20eadd3067%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Ortiz, C. et al. Molecular atlas of the adult mouse brain. <em>Sci. Adv.</em><strong>6</strong>, eabb3446 (2020).
</cite> [<a href="https://doi.org/10.1126/sciadv.abb3446" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7319762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32637622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ortiz,%20C.%20et%20al.%20Molecular%20atlas%20of%20the%20adult%20mouse%20brain.%20Sci.%20Adv.6,%20eabb3446%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Peng, H. et al. Morphological diversity of single neurons in molecularly defined cell types. <em>Nature</em><strong>598</strong>, 174â€“181 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03941-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8494643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34616072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peng,%20H.%20et%20al.%20Morphological%20diversity%20of%20single%20neurons%20in%20molecularly%20defined%20cell%20types.%20Nature598,%20174%E2%80%93181%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Shine, J. M., Arnatkeviciute, A., Fornito, A. &amp; Fulcher, B. D. Navigating a complex landscape: using transcriptomics to parcellate the human cortex. <em>Biol. Psychiatry Cogn. Neurosci. Neuroimaging</em><strong>7</strong>, 3â€“4 (2022).
</cite> [<a href="https://doi.org/10.1016/j.bpsc.2021.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34998482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shine,%20J.%20M.,%20Arnatkeviciute,%20A.,%20Fornito,%20A.%20&amp;%20Fulcher,%20B.%20D.%20Navigating%20a%20complex%20landscape:%20using%20transcriptomics%20to%20parcellate%20the%20human%20cortex.%20Biol.%20Psychiatry%20Cogn.%20Neurosci.%20Neuroimaging7,%203%E2%80%934%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Zheng, G. X. et al. Massively parallel digital transcriptional profiling of single cells. <em>Nat. Commun.</em><strong>8</strong>, 14049 (2017).
</cite> [<a href="https://doi.org/10.1038/ncomms14049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5241818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28091601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20G.%20X.%20et%20al.%20Massively%20parallel%20digital%20transcriptional%20profiling%20of%20single%20cells.%20Nat.%20Commun.8,%2014049%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>McGinnis, C. S., Murrow, L. M. &amp; Gartner, Z. J. DoubletFinder: Doublet detection in single-cell RNA sequencing data using artificial nearest neighbors. <em>Cell Syst.</em><strong>8</strong>, 329â€“337.e324 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cels.2019.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6853612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30954475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McGinnis,%20C.%20S.,%20Murrow,%20L.%20M.%20&amp;%20Gartner,%20Z.%20J.%20DoubletFinder:%20Doublet%20detection%20in%20single-cell%20RNA%20sequencing%20data%20using%20artificial%20nearest%20neighbors.%20Cell%20Syst.8,%20329%E2%80%93337.e324%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. <em>Nat. Methods</em><strong>16</strong>, 1289â€“1296 (2019).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0619-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6884693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31740819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Korsunsky,%20I.%20et%20al.%20Fast,%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20Harmony.%20Nat.%20Methods16,%201289%E2%80%931296%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Hao, Y. et al. Integrated analysis of multimodal single-cell data. <em>Cell</em><strong>184</strong>, 3573â€“3587.e3529 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.04.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8238499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao,%20Y.%20et%20al.%20Integrated%20analysis%20of%20multimodal%20single-cell%20data.%20Cell184,%203573%E2%80%933587.e3529%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>BÃ¼ttner, M., Ostner, J., MÃ¼ller, C. L., Theis, F. J. &amp; Schubert, B. scCODA is a Bayesian model for compositional single-cell data analysis. <em>Nat. Commun.</em><strong>12</strong>, 6876 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-27150-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8616929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34824236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?B%C3%BCttner,%20M.,%20Ostner,%20J.,%20M%C3%BCller,%20C.%20L.,%20Theis,%20F.%20J.%20&amp;%20Schubert,%20B.%20scCODA%20is%20a%20Bayesian%20model%20for%20compositional%20single-cell%20data%20analysis.%20Nat.%20Commun.12,%206876%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Li, M. et al. CellBin: a highly accurate single-cell gene expression processing pipeline for high-resolution spatial transcriptomics. <em>bioRxiv</em>, 2023.2002.2028.530414, (2024).</cite>
</li>
<li id="CR69">
<span class="label">69.</span><cite>Pachitariu, M. &amp; Stringer, C. Cellpose 2.0: how to train your own model. <em>Nat. Methods</em><strong>19</strong>, 1634â€“1641 (2022).
</cite> [<a href="https://doi.org/10.1038/s41592-022-01663-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9718665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36344832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pachitariu,%20M.%20&amp;%20Stringer,%20C.%20Cellpose%202.0:%20how%20to%20train%20your%20own%20model.%20Nat.%20Methods19,%201634%E2%80%931641%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Fang, S. et al. Stereopy: modeling comparative and spatiotemporal cellular heterogeneity via multi-sample spatial transcriptomics. <em>Nat. Commun.</em><strong>16</strong>, 3741 (2025).
</cite> [<a href="https://doi.org/10.1038/s41467-025-58079-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12012134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40258830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang,%20S.%20et%20al.%20Stereopy:%20modeling%20comparative%20and%20spatiotemporal%20cellular%20heterogeneity%20via%20multi-sample%20spatial%20transcriptomics.%20Nat.%20Commun.16,%203741%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Wolf, F. A., Angerer, P. &amp; Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. <em>Genome Biol.</em><strong>19</strong>, 15 (2018).
</cite> [<a href="https://doi.org/10.1186/s13059-017-1382-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5802054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29409532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolf,%20F.%20A.,%20Angerer,%20P.%20&amp;%20Theis,%20F.%20J.%20SCANPY:%20large-scale%20single-cell%20gene%20expression%20data%20analysis.%20Genome%20Biol.19,%2015%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Cable, D. M. et al. Robust decomposition of cell type mixtures in spatial transcriptomics. <em>Nat. Biotechnol.</em><strong>40</strong>, 517â€“526 (2022).
</cite> [<a href="https://doi.org/10.1038/s41587-021-00830-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8606190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33603203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cable,%20D.%20M.%20et%20al.%20Robust%20decomposition%20of%20cell%20type%20mixtures%20in%20spatial%20transcriptomics.%20Nat.%20Biotechnol.40,%20517%E2%80%93526%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Jin, S. et al. Inference and analysis of cell-cell communication using CellChat. <em>Nat. Commun.</em><strong>12</strong>, 1088 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-21246-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7889871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33597522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin,%20S.%20et%20al.%20Inference%20and%20analysis%20of%20cell-cell%20communication%20using%20CellChat.%20Nat.%20Commun.12,%201088%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Fournier-Viger, P. et al. SPMF: a Java open-source pattern mining library. <em>J. Mach. Learn. Res.</em><strong>15</strong>, 3389â€“3393 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Fournier-Viger,%20P.%20et%20al.%20SPMF:%20a%20Java%20open-source%20pattern%20mining%20library.%20J.%20Mach.%20Learn.%20Res.15,%203389%E2%80%933393%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Songren W, M. L. et al. (STOmicsDB, 2024). 10.26036/STDS0000242.</cite>
</li>
<li id="CR76">
<span class="label">76.</span><cite>Songren W, M. L. et al. Charting the Spatial Transcriptome of the Human Cerebral Cortex at Single-Cell Resolution v. 1.1 (Zenodo, 2024). 10.5281/zenodo.14311048.</cite>
</li>
<li id="CR77">
<span class="label">77.</span><cite>Allen Institute for Brain Science, Allen Human Brain Atlas, <a href="https://human.brain-map.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://human.brain-map.org</a></cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Infomation</a><sup> (79MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62793_MOESM2_ESM.pdf</a><sup> (95KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 1</a><sup> (59.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 2</a><sup> (49.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 3</a><sup> (13.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 4</a><sup> (788.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM7_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 5</a><sup> (43.6MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM8_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Dataset 6</a><sup> (3.3MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (3.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM10_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (9.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365125/bin/41467_2025_62793_MOESM11_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (76.7MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The sequencing data generated in this study have been deposited in the stomicsDB under accession code <a href="https://db.cngb.org/stomics/datasets/STDS0000242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">STDS0000242</a><sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>. The immunohistochemical staining images generated in this study have been deposited in Figshare under the accession code <a href="https://doi.org/10.6084/m9.figshare.c.6873943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6873943</a>.Â <a href="#Sec33" class="usa-link">Source data</a> are provided with this paper.</p>
<p>All codes are available on <a href="https://github.com/lcy1364/Cortex-Atlas-Code" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/lcy1364/Cortex-Atlas-Code</a>. (10.5281/zenodo.14311048)<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup></p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62793-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62793.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.4Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365125/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365125/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365125%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365125/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365125/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365125/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830090/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365125/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830090/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365125/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365125/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="85awxZodrdFSygwl99gljEkUy1NHJuwQR7HgfHfe0eGFlej2ClJkeMBbzlj2CyTQ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
