
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Prediction of antibiotic resistance from antibiotic susceptibility testing results from surveillance data using machine learning - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5338F78AF313D30538F7003A0341A8.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368220/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Prediction of antibiotic resistance from antibiotic susceptibility testing results from surveillance data using machine learning">
<meta name="citation_author" content="Swetha Valavarasu">
<meta name="citation_author_institution" content="Department of Computer Science and Information Systems, Birla Institute of Technology and Science, Pilani, Pilani Campus, Vidya Vihar, Pilani, 333031 Rajasthan India">
<meta name="citation_author" content="Yasaswini Sangu">
<meta name="citation_author_institution" content="Department of Computer Science and Information Systems, Birla Institute of Technology and Science, Pilani, Pilani Campus, Vidya Vihar, Pilani, 333031 Rajasthan India">
<meta name="citation_author" content="Tanmaya Mahapatra">
<meta name="citation_author_institution" content="Department of Computer Science and Information Systems, Birla Institute of Technology and Science, Pilani, Pilani Campus, Vidya Vihar, Pilani, 333031 Rajasthan India">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30509">
<meta name="citation_doi" content="10.1038/s41598-025-14078-w">
<meta name="citation_pmid" content="40835633">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368220/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368220/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368220/pdf/41598_2025_Article_14078.pdf">
<meta name="description" content="Antimicrobial resistance is a growing global health threat, and artificial intelligence offers a promising avenue for developing advanced tools to address this challenge. In this study, we applied various machine learning techniques to predict ...">
<meta name="og:title" content="Prediction of antibiotic resistance from antibiotic susceptibility testing results from surveillance data using machine learning">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Antimicrobial resistance is a growing global health threat, and artificial intelligence offers a promising avenue for developing advanced tools to address this challenge. In this study, we applied various machine learning techniques to predict ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368220/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368220">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-14078-w"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_14078.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368220%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368220/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368220/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368220/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 20;15:30509. doi: <a href="https://doi.org/10.1038/s41598-025-14078-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-14078-w</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Prediction of antibiotic resistance from antibiotic susceptibility testing results from surveillance data using machine learning</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Valavarasu%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Swetha Valavarasu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Swetha Valavarasu</span></h3>
<div class="p">
<sup>1</sup>Department of Computer Science and Information Systems, Birla Institute of Technology and Science, Pilani, Pilani Campus, Vidya Vihar, Pilani, 333031 Rajasthan India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Valavarasu%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Swetha Valavarasu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sangu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yasaswini Sangu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yasaswini Sangu</span></h3>
<div class="p">
<sup>1</sup>Department of Computer Science and Information Systems, Birla Institute of Technology and Science, Pilani, Pilani Campus, Vidya Vihar, Pilani, 333031 Rajasthan India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sangu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yasaswini Sangu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mahapatra%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Tanmaya Mahapatra</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Tanmaya Mahapatra</span></h3>
<div class="p">
<sup>1</sup>Department of Computer Science and Information Systems, Birla Institute of Technology and Science, Pilani, Pilani Campus, Vidya Vihar, Pilani, 333031 Rajasthan India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mahapatra%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tanmaya Mahapatra</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Computer Science and Information Systems, Birla Institute of Technology and Science, Pilani, Pilani Campus, Vidya Vihar, Pilani, 333031 Rajasthan India </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 23; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368220  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835633/" class="usa-link">40835633</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Antimicrobial resistance is a growing global health threat, and artificial intelligence offers a promising avenue for developing advanced tools to address this challenge. In this study, we applied various machine learning techniques to predict bacterial antibiotic resistance using the Pfizer ATLAS Antibiotics dataset. This comprehensive dataset includes patient demographic data, sample collection details, antibiotic susceptibility test results, and resistance phenotypes for 917,049 bacterial isolates. The dataset was divided into two subsets: Phenotype-Only and Phenotype + Genotype, excluding and including 589,998 isolates with genotype data, respectively. Both subsets underwent exploratory data analysis, preprocessing, machine learning model training, validation, and optimization. XGBoost consistently outperformed other models, achieving AUC values of 0.96 and 0.95 for the Phenotype-Only and Phenotype + Genotype sets, respectively. Hyperparameter tuning yielded slight accuracy improvements, while data balancing techniques notably increased recall. Across all models, the antibiotic used emerged as the most influential feature in predicting resistance outcomes. The SHAP summary plots generated provide insights into model interpretability. Our findings provide valuable insights into global AMR patterns and demonstrate the potential of AI-driven approaches for resistance prediction to help inform clinical decision-making and support the formulation of effective AMR mitigation policies, subject to the availability of highly granular datasets.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-14078-w.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Antibiotic resistance, Antibiotic susceptibility testing, Machine learning, AMR prediction</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Antimicrobial resistance, Computer science</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par5">Antimicrobial resistance (AMR) is a multiscale global public health threat that, if not addressed urgently, could lead to an estimated 39 million deaths in the next 25 years<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. With a significant decline in the number of new antibiotics released in the market and a simultaneous increase in resistance to the ones currently in use, it is imperative that the emergence and spread of AMR be monitored and efforts be made to curb the issue. The different approaches taken to tackle the problem include monitoring molecular dynamics and genetic changes in pathogens using next-generation sequencing (NGS) data, collecting, and analyzing resistance patterns from antibiotic susceptibility testing (AST), developing new antibiotics and antimicrobial peptides (AMPs), drug combinations and adjuvants, and practicing antibiotic stewardship, among others<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. These facets of data have also been used for machine learning (ML) applications<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Models have been built with high accuracy for developing new antibiotics and AMPs, drug combinations and adjuvants, analyzing antibiograms, enhancing AST methods, guiding antimicrobial therapy, analyzing NGS data to identify AMR genes for various pathogens, and predicting AMR to improve prediction-based diagnostics<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>.</p>
<p id="Par6">Comprehensive AMR surveillance programs that monitor and assess ongoing changes in resistance to antimicrobial agents worldwide are administered by government bodies and funded by pharmaceutical industries in an effort to decrease morbidity and mortality caused by AMR in clinical settings by informing treatment guidelines and decisions, improving the effectiveness of intervention strategies, guiding national and international policies and antimicrobial stewardship programs, and directing the development of novel antimicrobial agents. Currently, numerous surveillance programs exist across different geographical locations to collect AMR data of varying scopes and types. Some of the notable ones include World Health Organization’s Global Antimicrobial Resistance and Use Surveillance System (GLASS)<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>, European Antimicrobial Resistance Surveillance Network, National Antimicrobial Resistance Monitoring System, English Surveillance Programme for Antimicrobial Utilisation and Resistance, Japan Nosocomial Infections Surveillance, One Health Trust’s ResistanceMap, Pfizer Antimicrobial Testing Leadership and Surveillance (ATLAS), and Merck Study for Monitoring Antimicrobial Resistance Trends (SMART)<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. These programs primarily collect culture-based AST results (zone diameter or MIC values, which are then interpreted as susceptible, intermediate, or resistant according to international guidelines on clinical breakpoints) of representative clinical samples for various pathogen-antimicrobial agent pairs, with some programs like the Pfizer ATLAS collecting corresponding genotypic data as well for selective samples.</p>
<p id="Par7">The conventional gold-standard method to determine antibiotic susceptibility to antibiotic agents and to identify AMR to treat infections is the in vitro laboratory phenotypic test<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. However, this method is time-consuming and resource-intensive, especially for non-fastidious microbial pathogens and slow-growing bacteria like <em>Mycobacterium tuberculosis</em>. This makes it difficult and impractical to implement in actual clinical practice. An emerging alternative for detecting AMR is utilizing microbial genotype data instead of phenotype information, driven by advancements in sequencing technologies and lower sequencing costs. This genotypic approach offers faster AMR prediction compared to the traditional method by bypassing the need for culture-based laboratory procedures. Additionally, it provides insights into the genetic mechanisms responsible for AMR, enabling early detection of disease transmission<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>.</p>
<p id="Par8">For machine learning-based AMR prediction, there have been studies conducted, majorly in high-resource settings, using clinical, epidemiological, and microbiome data such as demographic characteristics of patients, previous AST results, and prior antibiotic exposure data to predict drug resistance in infected patients, ML-assisted antibiotic prescription, and national and international AMR trends, among other applications<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Utilizing next-generation sequencing data, including whole genome sequencing, metagenomics, and transcriptomics data, AI/ML techniques aid in identifying resistance genes and mutations/variations to enable studying AMR development and support personalized treatment recommendations. With decreasing sequencing costs and expanding availability of genomic data, the information derived for AMR prediction and model interpretability is ever-expanding<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Predictive models for AMR aid in the provision of quick information for prompt AMR mitigation by finding hidden connections in data provided for monitoring AMR patterns using network analysis, time series analysis, and phylogenetic studies. These could help in the early detection of uncommon behaviors and clusters associated with drug resistance and overexpression of antibiotic-resistance genes, which in turn enables the implementation of disease transmission prevention efforts and understanding of the spread of resistance in hospitals and communities<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>.</p>
<p id="Par9">In a review of machine learning prediction of AMR by Kherabi et al.<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>, nineteen out of the thirty-six studies included were based on previous AST results, with sample sizes ranging from a few hundred patients to a maximum of 68,472 patients. These studies were mostly localized to a particular ward, hospital, or country and were performed on a limited set of bacteria. On a global scale, in the study by Mutisya et al. 2024<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, the prediction of AMR using machine learning was performed on the Pfizer ATLAS Antibiotics and Antifungal datasets with data collected from 2004 to 2021. This dataset has also been used to study resistance dynamics and resistance patterns across drug-bacterium pairs<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. In this study, the Pfizer ATLAS Antibiotics dataset with data from 2006 to 19 was analyzed using a multivariable spatiotemporal analysis and found significant associations of AMR with country-level antibiotic sales for fluoroquinolone-resistant <em>E. coli</em> and <em>Pseudomonas</em>, as well as carbapenem-resistant <em>A. baumannii</em>. They also found a correlation between temperature and AMR in Enterobacteriaceae and with health system quality for all bug-drug combinations except for <em>Enterococci</em> and <em>Streptococcus pneumoniae</em>. Such studies showcase the utility of such robust surveillance data and help in understanding the status of AMR globally.</p>
<p id="Par10">In our study, we have explored using machine learning techniques on the Pfizer ATLAS Antibiotics dataset for the prediction of antibiotic resistance to understand how to utilize such robust global surveillance datasets for real-world applications and the challenges encountered. In the process, we have performed exploratory data analysis, training, and testing of various machine learning models, explored methods for handling missing data and data imbalance, and assessed the models based on their performance metrics and optimized them. Feature importance and SHAP analysis were also done to identify which of the features had the most effect on resistance phenotype (susceptible, intermediate, resistant), the target variable, during classification by the model. The results obtained in this study could be a stepping stone for building tools to inform clinical practice and aid in policymaking for the surveillance of AMR on a global scale. This study expands the predictive framework to a globally diverse, multiclass (S/I/R) resistance phenotype using a robust surveillance dataset. Additionally, it explores the implications of both phenotype-only and genotype-inclusive models to evaluate the incremental predictive power of molecular features, along with providing insights into model interpretability.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Exploratory data analysis</h3>
<p id="Par11">EDA was performed using matplotlib, pandas, and seaborn libraries to understand the underlying distributions and correlations in the dataset. The dataset has 917,049 rows with information on resistance interpretations of MIC values when tested against a panel of 50 antibiotic drugs for 324 bacterial pathogen species collected across 83 countries from 2004 to 2022, along with patient demographics. A subset of the dataset with 589,998 isolates contains data on the presence or absence of genotype markers such as CTXM, predominantly for ESBL isolates of the Enterobacteriaceae family.</p>
<p id="Par12">The dataset contained the highest frequency of data collected from the United States of America with the majority share (31%), followed by Spain (12%), France (11.5%), Germany (10.9%), and China (6.1%), aligning with the notion that better monitoring resources are available in economically developed countries<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. AMR is a looming threat in Sub-Saharan Africa<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>; however, there is a significant underrepresentation of data from this region. This disparity highlights the need for increased investments in AMR surveillance infrastructure in low- and middle-income countries (LMICs). Shown in Supplementary Figure <a href="#MOESM1" class="usa-link">S1</a> is the overall global distribution of antibiotic-resistant bacteria captured in the Pfizer ATLAS Antibiotics dataset.</p>
<p id="Par13">Missing data is a major problem during prediction; a heatmap (shown in Fig. <a href="#Fig1" class="usa-link">1</a>) was generated to understand the number of samples in each column where no data was available. In the heatmap, yellow indicates the absence of data, while purple indicates the presence. No data was missing for the initial columns, such as Isolate ID, Family, and Gender. In contrast, a major part of the data was absent for genetic markers like AMPC, CTXM1, etc., which is reasonable given the extensive resources required for genetic marker analysis in routine surveillance systems. This visualization highlights the need for methods to handle missing data while building machine learning models for practical applications. Imputation is a powerful technique for effectively handling missing data in machine learning, as used by Mutisya et al. 2024<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> in their study to predict AMR from surveillance data. Although it increases the prediction accuracy of the models, from a clinical perspective, it needs to be done with caution as it could mislead decision-making processes if assessment of the imputation methods is not done<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368220_41598_2025_14078_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/42a7/12368220/88ca81d09126/41598_2025_14078_Fig1_HTML.jpg" loading="lazy" id="d33e301" height="627" width="743" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Missing data patterns in the dataset for each column are shown on the horizontal axis. Yellow represents missing data, and black represents available data.</p></figcaption></figure><p id="Par15">Temporal analysis was performed to further understand the year-wise distribution of total samples and resistant samples between the years 2004 and 2022. The total number of samples showed a significant increase up to 2014, reflecting better surveillance and data collection efforts over time. However, a minor decline in total samples can be observed after 2016. The resistance rate can be deduced visually by comparing the two-line plots (Fig. <a href="#Fig2" class="usa-link">2</a>). If the resistant line grows at a similar rate to the total line, it implies that the proportion of resistant cases remains largely unchanged, indicating persistent resistance. If the gap between the total and resistant samples increases, it may indicate effective interventions or reduced resistance levels. In contrast, if the lines converge, it could signal an increase in resistance rates. The trend in resistant samples mainly follows the trend of total samples, suggesting a persistent antimicrobial resistance. This could also occur due to sampling bias.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368220_41598_2025_14078_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/42a7/12368220/4a9a99b8628e/41598_2025_14078_Fig2_HTML.jpg" loading="lazy" id="d33e316" height="346" width="607" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Year-wise Total Samples vs. Resistant Samples from 2004 to 2022.</p></figcaption></figure><p id="Par17">Correlation analysis was conducted on the full dataset, including genotypic features indicating the presence or absence of β-lactamase genes such as TEM, AMPC, and NDM<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. These genes are particularly relevant in ESBL-producing organisms like <em>Escherichia coli</em> and <em>Klebsiella</em> spp., where CTX-M, SHV, and TEM serve as hallmark markers<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Co-occurrence of AmpC and carbapenemase genes (e.g., NDM, OXA, IMP) suggests extensive multidrug resistance, limiting therapeutic options<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>.</p>
<p id="Par18">To examine associations between bacterial, demographic, genotypic, and phenotypic variables, a Cramér’s V correlation heatmap was generated (Fig. <a href="#Fig3" class="usa-link">3</a>). This method, suited for categorical data, revealed strong correlations between species and family, and moderate-to-strong associations between key resistance genes and bacterial taxa (e.g., CTX-M1, SHV, TEM). Carbapenemase genes such as NDM and VIM also showed moderate associations with specific species, indicating species-driven gene dissemination patterns<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368220_41598_2025_14078_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/42a7/12368220/4bab01e334bd/41598_2025_14078_Fig3_HTML.jpg" loading="lazy" id="d33e363" height="703" width="755" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cramer’s V correlation plot for all features in the dataset.</p></figcaption></figure><p id="Par19">Antibiotic susceptibility patterns were moderately correlated with AMR outcomes, validating the phenotypic influence of tested antibiotics. In contrast, demographic features like age group, gender, and country exhibited weak correlations with genotypic resistance or AMR phenotype. However, the phenotypic resistance class (e.g., MRSA) is predictive of whether the isolate is Susceptible, Intermediate, or Resistant to an antibiotic, with high correlation values. This reflects biological causality, as phenotypes directly influence susceptibility. Source, Species, and Family also show moderate correlations with Phenotype and AMR. This implies that ecological or biological niches (e.g., certain sources like urine, blood, or respiratory) and taxonomic identity affect resistance patterns.</p>
<p id="Par20">Exploratory analysis of year and country data showed minimal to moderate associations with pathogen and antibiotic variables. This suggests some geographic and temporal variation in testing or prevalence, but not strong enough to drive stratified modeling in this study. These findings support the inclusion of genetic features for improved model interpretability and reinforce the need for global, standardized surveillance of resistance trends.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Model performance</h3>
<p id="Par22">The dataset was pre-processed and was followed by training and testing classification models for the prediction of antibiotic resistance into Susceptible, Intermediate, and Resistant categories. Phenotype-Only and Phenotype + Genotype datasets were derived from the original dataset and processed individually to avoid selection bias since only a subset of the dataset contains genotype data. The models trained included simplistic models like Logistic regression as well as complex ensemble models like XGBoost. The model performance metrics for the initial set of models with all features included and implemented are given in Table <a href="#Tab1" class="usa-link">1</a> for the Phenotype-only and Phenotype + Genotype datasets. Simpler models like logistic regression and KNN have lower accuracies, whereas more complex models like XGBoost and Gradient Boost have better prediction ability and higher scores on all metrics. For the Phenotype-only dataset, SVM has the highest precision. For both datasets, XGBoost was the best-performing model, with its AUC being the highest (95%) and its accuracy slightly enhanced with hyperparameter tuning and ten-fold cross-validation. Supplementary Table S2 shows the parameters used for model development and the tuning parameters used for hyperparameter tuning.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Model performance metrics for Phenotype-Only and Phenotype + Genotype datasets.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Model</th>
<th align="left" colspan="5" rowspan="1">Phenotype-Only</th>
<th align="left" colspan="5" rowspan="1">Phenotype + Genotype</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">F-1 score</th>
<th align="left" colspan="1" rowspan="1">Precision</th>
<th align="left" colspan="1" rowspan="1">Recall</th>
<th align="left" colspan="1" rowspan="1">Accuracy</th>
<th align="left" colspan="1" rowspan="1">ROC-AUC</th>
<th align="left" colspan="1" rowspan="1">F-1 score</th>
<th align="left" colspan="1" rowspan="1">Precision</th>
<th align="left" colspan="1" rowspan="1">Recall</th>
<th align="left" colspan="1" rowspan="1">Accuracy</th>
<th align="left" colspan="1" rowspan="1">ROC-AUC</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Logistic Regression</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.68</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
<td align="left" colspan="1" rowspan="1">0.82</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.72</td>
<td align="left" colspan="1" rowspan="1">0.82</td>
<td align="left" colspan="1" rowspan="1">0.92</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Random Forest Classifier</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.78</td>
<td align="left" colspan="1" rowspan="1">0.71</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.93</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">0.82</td>
<td align="left" colspan="1" rowspan="1">0.76</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">XGBoost</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">0.70</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">0.96</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.76</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.95</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AdaBoost</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.67</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.81</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">0.71</td>
<td align="left" colspan="1" rowspan="1">0.81</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Gradient Boost</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.87</td>
<td align="left" colspan="1" rowspan="1">0.68</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.87</td>
<td align="left" colspan="1" rowspan="1">0.72</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.93</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SVM</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">0.88</td>
<td align="left" colspan="1" rowspan="1">0.69</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">0.87</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">0.74</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">0.93</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KNN (k = 100)</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.87</td>
<td align="left" colspan="1" rowspan="1">0.67</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
<td align="left" colspan="1" rowspan="1">0.94</td>
<td align="left" colspan="1" rowspan="1">0.76</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.62</td>
<td align="left" colspan="1" rowspan="1">0.76</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Naive Bayes (Bernoulli)</td>
<td align="left" colspan="1" rowspan="1">0.77</td>
<td align="left" colspan="1" rowspan="1">0.59</td>
<td align="left" colspan="1" rowspan="1">0.65</td>
<td align="left" colspan="1" rowspan="1">0.77</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">0.77</td>
<td align="left" colspan="1" rowspan="1">0.78</td>
<td align="left" colspan="1" rowspan="1">0.68</td>
<td align="left" colspan="1" rowspan="1">0.77</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
</tr>
<tr><td align="left" colspan="11" rowspan="1">Modifications done on the XGBoost model:</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Undersampling</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.66</td>
<td align="left" colspan="1" rowspan="1">0.80</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.80</td>
<td align="left" colspan="1" rowspan="1">0.74</td>
<td align="left" colspan="1" rowspan="1">0.78</td>
<td align="left" colspan="1" rowspan="1">0.80</td>
<td align="left" colspan="1" rowspan="1">0.80</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Oversampling (SMOTENC)</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.68</td>
<td align="left" colspan="1" rowspan="1">0.78</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.79</td>
<td align="left" colspan="1" rowspan="1">0.78</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bayesian Optimization</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.71</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.78</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cross Validation</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">0.71</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">0.90</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.78</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
<td align="left" colspan="1" rowspan="1">0.86</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>Bayesian optimization +</p>
<p>Cross Validation</p>
</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">0.84</td>
<td align="left" colspan="1" rowspan="1">0.71</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.77</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par24">Shown in Fig. <a href="#Fig4" class="usa-link">4</a> are the ROC-AUC curves of the XGBoost models. ROC-AUC measures the model’s ability to distinguish between classes. A higher AUC indicates a better-performing model, with 1.0 being the best possible score and 0.5 representing random guessing. To handle the multi-class nature of the problem, the One-vs-Rest (OvR) strategy was implemented. For each class, the model was trained to distinguish it from the others, and binary classification was performed on each one. The ROC curve was plotted by calculating the False Positive Rate (FPR) and True Positive Rate (TPR) for each class. The AUC for each class was then calculated, with a higher AUC indicating a better performance at distinguishing that class from the others. To statistically evaluate differences in model performance, the DeLong test was applied to compare the AUC values between models. The results indicated no statistically significant difference in AUC values, suggesting that the models exhibit similar discriminatory power. Additionally, the bootstrapped confidence intervals around the AUC estimates were narrow, indicating stability and low variability in the AUC across resampled datasets. While this supports the robustness of the models’ performance, it also highlights the limitation of relying solely on AUC for model selection. Given the lack of significant difference in AUC, additional metrics—such as precision, recall, and F1-score—as well as model interpretability, were considered to enable a more comprehensive comparison of model behavior and utility.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368220_41598_2025_14078_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/42a7/12368220/9905c7eb95c1/41598_2025_14078_Fig4_HTML.jpg" loading="lazy" id="d33e747" height="294" width="659" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>ROC-AUC curves for XGBoost models for (<strong>a</strong>) Phenotype-Only and (<strong>b</strong>) Phenotype + Genotype datasets. Classes 0,1, and 2 are Intermediate, Resistant, and Susceptible, respectively.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Data balancing</h3>
<p id="Par26">The accuracy of prediction is lowest for the Intermediate category for both Phenotype-only and Phenotype + Genotype datasets, as seen in the confusion matrices of the XGBoost models (Fig. <a href="#Fig5" class="usa-link">5</a>). This is because of data imbalance, with the number of intermediate samples being considerably lower compared to resistant and susceptible samples, which could affect the model’s ability to generalize effectively (S/I/R data distribution shown in Supplementary Figure S2). Imbalanced datasets could cause ML models to bias toward the majority class, misclassify rare but clinically important classes, and underperform in metrics like recall and F1-score for minority classes. Data balancing using under-sampling of susceptible samples and oversampling of intermediate samples done on the XGBoost models improved recall.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368220_41598_2025_14078_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/42a7/12368220/851fc738f045/41598_2025_14078_Fig5_HTML.jpg" loading="lazy" id="d33e771" height="345" width="711" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Confusion matrices of the XGBoost models for (<strong>a</strong>) Phenotype-Only and (<strong>b</strong>) Phenotype + Genotype datasets.</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Feature importance</h3>
<p id="Par28">For models like Logistic Regression, Random Forest, Gradient Boosting, AdaBoost, and XGBoost that support feature analysis, the corresponding scores that indicate how much each feature contributes to the model’s performance were extracted. These feature importance scores reflect how often and how effectively a feature contributes to the decision-making process in the model’s trees. These scores were used to determine which groups of features had the most significant impact on the model’s predictions (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368220_41598_2025_14078_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/42a7/12368220/d2913198fa86/41598_2025_14078_Fig6_HTML.jpg" loading="lazy" id="d33e795" height="268" width="768" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Feature importance plots of XGBoost models for (<strong>a</strong>) Phenotype-Only and (<strong>b</strong>) Phenotype + Genotype datasets.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">SHAP (SHapley additive exPlanations) analysis</h3>
<p id="Par30">SHAP analysis<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> for the XGBoost model trained on the Phenotype-only dataset (summary plot shown in Fig. <a href="#Fig7" class="usa-link">7</a>) revealed that individual features such as Phenotype_ESBL, Phenotype_MRSA, and antibiotics like tigecycline, vancomycin, and oxacillin were among the most influential in predicting AMR phenotypes. These features are known resistance indicators and reflect the model’s reliance on clinically relevant patterns<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Most top features contributed to the prediction of the intermediate class, while fewer features drove predictions for susceptible, suggesting class-wise differences in feature influence. The SHAP plot enhances interpretability by identifying which phenotypic or treatment features the model used most heavily, supporting transparency and clinical relevance.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368220_41598_2025_14078_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/42a7/12368220/0bcfda24b358/41598_2025_14078_Fig7_HTML.jpg" loading="lazy" id="d33e859" height="323" width="799" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SHAP plots of XGBoost models for (<strong>a</strong>) Phenotype-Only and (<strong>b</strong>) Phenotype + Genotype datasets, where classes 0, 1 and 2 are Intermediate, Resistant and Susceptible, respectively. SHAP summary plot illustrating the contribution of each feature to the model’s predictions. Features are ranked by importance (y-axis), and the horizontal distribution shows their impact on the prediction output.</p></figcaption></figure><p id="Par31">The SHAP summary plot for the Phenotype + Genotype dataset (Fig. <a href="#Fig7" class="usa-link">7</a>) provides a global view of the most influential features driving model predictions for antimicrobial resistance phenotypes—Susceptible (Class 2), Resistant (Class 1), and Intermediate (Class 0). Features such as Antibiotic_Colistin, Antibiotic_Ampicillin, and Antibiotic_Tigecycline exhibited the highest average SHAP values, indicating strong contributions to the model’s predictive accuracy. Notably, Colistin and Ampicillin showed the greatest impact on predicting the Intermediate phenotype (Class 0), as seen by the dominance of green bars<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Conversely, features like Ceftazidime avibactam, Cefaroline, and resistance genes such as OXA_NEG and NDM_NEG contributed significantly to identifying Resistant phenotypes (Class 1), suggesting the model correctly leveraged key molecular determinants of resistance<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Susceptible predictions (Class 2) were more associated with features such as <em>Escherichia coli</em>, Cefepime, and absence of resistance genes (KPC_NEG, TEM_NEG), indicating that the model identified typical patterns associated with non-resistant strains<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. These findings not only validate the model’s behavior but also enhance interpretability by aligning with known biological mechanisms. Such model explanations are essential in AMR contexts where understanding the why behind a prediction is as important as the prediction itself. Although genotype features offered modest overall performance gains, they showed disproportionately high importance in ESBL and carbapenem resistance cases. This underscores the clinical value of including molecular features in high-priority resistance scenarios.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par33">Machine learning models were implemented using the Pfizer ATLAS Antibiotics surveillance dataset for the prediction of antibiotic resistance in bacteria within the dataset. Our models predict AMR for single drug-bug combinations and currently do not account for multiple drug resistance. Tree-based models performed better than simpler models<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>, in capturing the resistance patterns from global data for an antibiotic-bacteria pair. The best-performing model for prediction was XGBoost, in agreement with previous studies<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, which was further optimized using feature selection and hyperparameter tuning to improve its accuracy. While prior studies emphasized predictive performance, we contribute to interpretability by integrating SHAP interaction values and comparative model explainability across different data subtypes. This allows more meaningful biological insights, especially in ESBL-producing organisms.</p>
<p id="Par34">Antibiotics are the most critical predictor in the AMR prediction models, which aligns with the current research that antibiotic exposure creates selective pressure, leading to resistance development<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Different antibiotics have varying resistance impacts, reflected in multidrug resistance patterns across bacterial species. Models that utilize antibiotic data perform better as they capture these subtle antibiotic-bacteria interactions<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Geographical variations in AMR are well-documented. Regional factors such as antibiotic stewardship policies, socioeconomic conditions, and antibiotic consumption trends influence resistance patterns<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. However, local resistance trends cannot be captured if data is not available at least at a state level, especially for huge countries like India with resistance patterns varying in different states<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Genetic markers have moderate but essential importance in predicting AMR. For our best-performing model, XGBoost, NDM, CTXM1, and OXA, were present in the top five features. Current research supports this, as NDM (New Delhi Metallo-<em>β</em>-lactamase) is known to produce enzymes that degrade carbapenem antibiotics<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. CTXM1, part of the CTX-M β-lactamase family, confers resistance to third-generation cephalosporins and is commonly found in <em>E. coli</em> and <em>K. pneumoniae</em><sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. OXA-<em>β</em>- lactamases, like OXA-48, are also known to provide resistance to both penicillins and carbapenems<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Studies show that machine learning models benefit from incorporating genetic marker data, especially when predicting resistance across different bacterial strains. Models like Gradient Boosting and XGBoost capture these interactions due to their ability to handle non-linear relationships<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Feature importance is model-specific, and the skew in the raw dataset towards ESBL-producing Enterobacteriaceae could have impacted the output. Feature importance reflects how much a variable contributes to a model’s predictions, and varies across algorithms. In our dataset, the overrepresentation of ESBL-producing <em>Enterobacteriaceae</em> may bias the model to emphasize features associated with these organisms, limiting generalizability. To address this, we applied resampling and caution that the reported importance scores are influenced by this underlying data skew.</p>
<p id="Par35">Demographic features like age and gender have minimal influence on AMR prediction models for data from across the world, although they have been shown to have an effect at the national level<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. While these factors may play a role in infection risk, they do not significantly impact resistance development, as resistance is driven primarily by antibiotic exposure and microbial genetics<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The above analysis aligns with findings that AMR dynamics are better explained by antibiotic usage patterns and pathogen-specific factors rather than patient demographics<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>.</p>
<p id="Par36">There were also some model-specific differences observed in feature importance. Gradient Boosting and XGBoost outperform simpler models like Logistic Regression in capturing complex, non-linear interactions among features. This explains why genetic markers show slightly higher importance in these models<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. Logistic Regression, although simpler, still highlights antibiotics as extremely dominant, reflecting the direct relationship between antibiotic exposure and resistance development. The use of absolute coefficients in Logistic Regression also offers good interpretability, which complements the predictive power of tree-based models<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p>
<p id="Par37">The outcome of any resistance prediction study is as good as the input data, which currently has many caveats and shortcomings, to which the models’ prediction accuracies could be attributed. These stem from noise, biases, and anomalies from discrepancies in AST protocols, resistance phenotype interpretation guidelines, MIC breakpoints, data imbalance with significantly more susceptible phenotypes, as well as a bias towards high-income countries<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. The antibiotic used was the most important feature in AMR prediction, while demographic factors like age and gender did not seem to have a significant effect on the model’s prediction; however, these factors play a major role in determining the socioeconomic effects of AMR. This study shows the potential and challenges of utilizing current AMR surveillance datasets in machine learning applications. Handling missing data is a commendable task in extending this study to build tools for clinical applications and predict resistance trends. We acknowledge that AST methods vary by region, and the dataset’s overrepresentation of certain geographies (e.g., USA, Spain) and susceptible phenotypes may limit generalizability. Our mitigation strategy included data resampling and separate phenotype/genotype models. However, standardized global sampling and AST protocols would greatly enhance model robustness and global applicability.</p>
<p id="Par38">The predictions could be made with more confidence if genomic sequencing data was available for all the isolates, with the possibility of correlating the AST results with AMR genes<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. The availability of these gene sequences could provide critical information on drug-bacteria interaction, which cannot be inferred with just the presence or absence of the genes. Apart from this, AMR surveillance datasets from other industry monitoring systems could be integrated with the Pfizer ATLAS dataset to increase the size of the dataset and its scope for AMR prediction while incorporating checks for differences in methodologies. The models can also be extended to surveillance data from government initiatives and international agencies like the WHO GLASS data to make the most of existing data to predict future AMR patterns across the world<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Also, agencies collecting AMR surveillance data from AST test results could consider including more specific details of patient age, location, and time of sample collection, particularly for being able to implement accurate and usable machine learning models such as time-series forecasting models to predict future trends<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, for clinical use and formulate strategies for global and country-level policy-making to curb AMR. Uniformity in guidelines for AST protocols for AMR surveillance data collection, interpretation of its results, and more representation of isolates from LMICs could immensely enhance the usability and harmonization of these datasets at a global scale for accurate antibiotic resistance predictions using interpretable machine learning models for building future-ready prediction tools. Future work should involve validation on independent datasets—such as those from WHO GLASS or regional health authorities—to assess real-world clinical applicability. Prospective evaluation in clinical decision support systems is needed before deployment. Expanding the input dataset by incorporating data from other AMR surveillance programs and additional genetic features could help increase the model’s robustness and generalizability. Prediction of multidrug resistance is another avenue for exploration.</p>
<p id="Par39">The application of machine learning to surveillance data for AMR prediction raises important ethical considerations related to data governance, patient anonymity, and responsible model deployment. Although the dataset used in this study was fully de-identified and collected under established surveillance protocols, minimizing privacy risks, clinical translation of such models must be approached with caution. Specifically, there is a risk of algorithmic bias arising from uneven geographic or demographic representation, particularly the underrepresentation of LMICs or certain resistance mechanisms. To ensure fair and effective use, machine learning models should be integrated with local epidemiological data and antimicrobial stewardship frameworks. Interpretability tools such as SHAP improve model transparency, but deployment in real-world clinical settings must also involve thorough local validation, regulatory oversight, and context-specific adaptation.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<p id="Par40">The Pfizer ATLAS Antibiotics dataset, one of the many industry monitoring systems for AMR surveillance available on request from the Vivli AMR Register (<a href="https://amr.vivli.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://amr.vivli.org</a>)<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, containing patient data along with extensive metadata, was acquired. The dataset contains MIC values from antibiotic susceptibility assays and their interpretations based on Clinical and Laboratory Standards Institute (CLSI) standards for 917,049 bacterial isolates from 83 countries collected over the years 2004 to 2022.</p>
<p id="Par41">The dataset contains patient demographic data, details of bacterial pathogen isolate collection, MIC values obtained from an in vitro drug susceptibility assay, and resistance interpretations done in accordance with international standards. Exploratory data analysis and the machine learning workflow were performed using Python version 3.8.</p>
<p id="Par42">The raw dataset was preprocessed to ensure compatibility with machine learning workflows. Missing values were replaced with the placeholder ‘NA’ where appropriate, particularly in columns containing MIC values and their corresponding AMR interpretations. These missing entries were not imputed, as their absence typically reflects that susceptibility testing was not performed for the specific antibiotic–bacterium combination, rather than random omission or measurement error. Consequently, such entries were excluded during downstream processing. This approach avoids introducing artificial values into critical microbiological fields, thereby preserving the integrity of the resistance phenotype representation. Initially, the columns Study and Isolate ID were removed as they do not have any effect on the resistance patterns. The column State had values available only for the United States of America and was omitted, considering the other 82 countries did not. The raw MIC values were removed, and the data was transformed for each of the rows to contain the antibiotic tested and its corresponding resistance phenotype interpretation since, in the original dataset, the antibiotic susceptibility results were spread across multiple columns for each of the antibiotics tested.</p>
<p id="Par43">A subset of the dataset with 589,998 isolates had genotypic data, primarily in Enterobacteriaceae isolates with ESBL phenotype, which was made into a separate dataset named Phenotype + Genotype. After transformation, the original dataset was considered separately after the exclusion of genotypic data to eliminate selection bias and named Phenotype-Only. Thus, two distinct datasets were used to implement two distinct sets of machine-learning models, including Logistic Regression, Random Forest Classifier, Naïve Bayes (Bernoulli), K-Nearest Neighbors (KNN), Support Vector Machine (SVM), Adaptive Boosting (AdaBoost), Gradient Boost, and Extreme Gradient Boosting (XGBoost), with an 80:20 train-test split, to maintain a clean train-test split to simulate unseen data. The models classify and predict the probability that the bacterial isolate is susceptible/intermediate/resistant to a particular antibiotic, given the other patient demographics and clinical features. The models were implemented using various libraries - scikit-learn for traditional machine learning algorithms, XGBoost for extreme gradient boosting, and cuML, a RAPIDS library, for GPU implementations of SVM and KNN. All machine learning models were initially trained using the default parameter settings of their respective libraries (e.g., scikit-learn v1.4.0 and XGBoost v1.7.6). A detailed list of the parameter values used for each algorithm is provided in Supplementary Table <em>S1</em>. This includes key parameters such as max_depth, learning_rate, and subsample (for XGBoost), even when default values were retained, to ensure clarity and reproducibility. Model metrics, such as accuracy, precision, F1 score, AUC, and recall, were compared. Model interpretability was evaluated using both feature importance and SHAP summary plots. These provide insight into feature behavior across resistance classes and are essential for policy translation. Among the models trained, the best-performing model, XGBoost, was chosen for further optimizations. The number of susceptible samples was disproportionately higher than resistant and intermediate samples, whereas intermediate samples had very low representation. To balance the data, under-sampling of susceptible samples and oversampling of intermediate samples, using SMOTENC (Synthetic Minority Over-sampling Technique for Nominal and Continuous variables), was done, and the model metrics of these two models were observed. Hyperparameter tuning was performed using the Optuna library, which implements Bayesian optimization to efficiently explore the hyperparameter space. Both standalone and combined strategies involving hyperparameter tuning and k-fold cross-validation (k = 10) were employed during model training. The use of 10-fold cross-validation ensured robust model evaluation by reducing the risk of overfitting and provided a more generalized performance estimate across different subsets of the data. This approach also introduced controlled randomization into the model training process, addressing potential bias associated with a fixed 80:20 train-test split.</p>
<p id="Par44">Based on the results obtained in the initial models, a final set of best-performing models using XGBoost was constructed. From the feature importance charts and lack of any direct effect on the resistance profiles, the features Isolate ID, Study, State, Gender, Age group, and In/Outpatient were removed from the datasets. Hyperparameter tuning was done to improve performance, and the model’s performance was assessed.</p>
<p id="Par45">This study exclusively used de-identified surveillance data publicly available from the Pfizer ATLAS database, ensuring no personal identifiers or protected health information were involved. The ethical use of machine learning models in healthcare settings necessitates transparency, fairness, and responsible implementation. Throughout our model development and evaluation, we adhered to principles outlined in internationally recognized frameworks such as the WHO’s guidance on the ethics and governance of AI for health (2021)<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup> and the OECD AI Principles<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Particular attention was given to mitigating algorithmic bias through resampling techniques and ensuring interpretability using SHAP. Although no formal ethics approval was required for this secondary data analysis, the broader implications of clinical deployment are acknowledged and discussed in the Discussion section.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par46">In this study, we developed machine learning models to predict antimicrobial resistance using surveillance data from the Pfizer ATLAS Antibiotics dataset. The models were evaluated and compared based on their predictive performance using various evaluation metrics. We have also attempted to address the issue of data imbalance using data balancing techniques. Feature importance and SHAP analysis were analyzed for the best-performing XGBoost models to gain insights into key predictive variables. Additionally, we have discussed the challenges and opportunities associated with utilizing large-scale surveillance datasets and outlined future directions, including the incorporation of more granular data and the integration of multiple surveillance sources to enhance predictive accuracy. Although this study does not directly integrate into clinical workflows, it lays a foundation for risk-stratification tools and AMR trend analysis in surveillance contexts. This could support treatment guideline revisions or antimicrobial stewardship policy development, after being subjected to prospective validation. These models highlight variables that could inform surveillance and empirical therapy in settings with similar pathogen/antibiotic profiles.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368220/bin/41598_2025_14078_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (451.5KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors extend their gratitude to the Vivli Platform for sharing valuable data to facilitate research toward efforts to mitigate AMR. Swetha Valavarasu is grateful to BITS, Pilani, for providing the Institute Fellowship.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>S.V.: Conceptualization, methodology, Software, investigation, data curation, Software, analysis, visualization; writing—original draft preparation, writing—review and editing; Y.S.: methodology, data curation, Software, analysis, visualization; T.M.: Conceptualization, methodology, Software, investigation, writing—original draft preparation, writing—review and editing, supervision. All authors reviewed the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>This publication is based on research using data from Pfizer obtained through the Vivli AMR Register https://amr.vivli.org. The Pfizer ATLAS Antibiotics dataset (https://searchamr.vivli.org/datasetDetails/fromSearch/ce7367e1-c91a-4de1-87e4-6e4d8f834085) used in this study is available on request from the Vivli AMR Register. Data requests should be directed to the Vivli AMR platform and are subject to the data-sharing agreements and policies set by the platform. For further details regarding the data request process, refer to the official documentation of the Vivli AMR Register or contact their support team at amr@vivli.org. The source code of the implementation is available at https://github.com/softwareinnovationslabBITS/AMR_AST_ML.</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par47">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Naghavi, M. et al. Global burden of bacterial antimicrobial resistance 1990–2013;2021: a systematic analysis with forecasts to 2050. <em>Lancet</em><strong>404</strong>, 1199–1226 (2024).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(24)01867-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11718157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39299261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naghavi,%20M.%20et%20al.%20Global%20burden%20of%20bacterial%20antimicrobial%20resistance%201990%E2%80%932013;2021:%20a%20systematic%20analysis%20with%20forecasts%20to%202050.%20Lancet404,%201199%E2%80%931226%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Oliveira, M. et al. An overview of the recent advances in antimicrobial resistance. <em>Microorganisms</em><strong>12</strong>, 1920 (2024).
</cite> [<a href="https://doi.org/10.3390/microorganisms12091920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11434382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39338594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oliveira,%20M.%20et%20al.%20An%20overview%20of%20the%20recent%20advances%20in%20antimicrobial%20resistance.%20Microorganisms12,%201920%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Lv, J., Deng, S. &amp; Zhang, L. A review of artificial intelligence applications for antimicrobial resistance. <em>Biosaf. Health</em>. <strong>3</strong>, 22–31 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lv,%20J.,%20Deng,%20S.%20&amp;%20Zhang,%20L.%20A%20review%20of%20artificial%20intelligence%20applications%20for%20antimicrobial%20resistance.%20Biosaf.%20Health.%203,%2022%E2%80%9331%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Bilal, H. et al. The role of artificial intelligence and machine learning in predicting and combating antimicrobial resistance. <em>Comput. Struct. Biotechnol. J.</em><strong>27</strong>, 423–439 (2025).
</cite> [<a href="https://doi.org/10.1016/j.csbj.2025.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11791014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39906157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bilal,%20H.%20et%20al.%20The%20role%20of%20artificial%20intelligence%20and%20machine%20learning%20in%20predicting%20and%20combating%20antimicrobial%20resistance.%20Comput.%20Struct.%20Biotechnol.%20J.27,%20423%E2%80%93439%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Zhou, X., Liu, G., Cao, S. &amp; Lv, J. Deep learning for antimicrobial peptides: computational models and databases. <em>J. Chem. Inf. Model.</em><strong>65</strong>, 1708–1717 (2025).
</cite> [<a href="https://doi.org/10.1021/acs.jcim.5c00006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39927895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20X.,%20Liu,%20G.,%20Cao,%20S.%20&amp;%20Lv,%20J.%20Deep%20learning%20for%20antimicrobial%20peptides:%20computational%20models%20and%20databases.%20J.%20Chem.%20Inf.%20Model.65,%201708%E2%80%931717%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Lv, J. et al. Computational models, databases and tools for antibiotic combinations. <em>Brief. Bioinform</em>. <strong>23</strong>, bbac309 (2022).
</cite> [<a href="https://doi.org/10.1093/bib/bbac309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35915052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lv,%20J.%20et%20al.%20Computational%20models,%20databases%20and%20tools%20for%20antibiotic%20combinations.%20Brief.%20Bioinform.%2023,%20bbac309%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Lv, J. et al. AADB: a manually collected database for combinations of antibiotics with adjuvants. <em>IEEE/ACM Trans. Comput. Biol. Bioinform</em>. <strong>20</strong>, 2827–2836 (2023).
</cite> [<a href="https://doi.org/10.1109/TCBB.2023.3283221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37279138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lv,%20J.%20et%20al.%20AADB:%20a%20manually%20collected%20database%20for%20combinations%20of%20antibiotics%20with%20adjuvants.%20IEEE/ACM%20Trans.%20Comput.%20Biol.%20Bioinform.%2020,%202827%E2%80%932836%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>World Health Organization. Global antimicrobial resistance and use surveillance system (GLASS) report 2022. (2022).</cite>
</li>
<li id="CR9">
<span class="label">9.</span><cite>Cantón, R. et al. Antimicrobial surveillance: a 20-year history of the SMART approach to addressing global antimicrobial resistance into the future. <em>Int. J. Antimicrob. Agents</em>. <strong>62</strong>, 107014 (2023).
</cite> [<a href="https://doi.org/10.1016/j.ijantimicag.2023.107014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37866472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cant%C3%B3n,%20R.%20et%20al.%20Antimicrobial%20surveillance:%20a%2020-year%20history%20of%20the%20SMART%20approach%20to%20addressing%20global%20antimicrobial%20resistance%20into%20the%20future.%20Int.%20J.%20Antimicrob.%20Agents.%2062,%20107014%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Diallo, O. O. et al. Antibiotic resistance surveillance systems: a review. <em>J. Glob Antimicrob. Resist.</em><strong>23</strong>, 430–438 (2020).
</cite> [<a href="https://doi.org/10.1016/j.jgar.2020.10.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33176216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diallo,%20O.%20O.%20et%20al.%20Antibiotic%20resistance%20surveillance%20systems:%20a%20review.%20J.%20Glob%20Antimicrob.%20Resist.23,%20430%E2%80%93438%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Burnham, C. A. D. et al. Diagnosing antimicrobial resistance. <em>Nat. Rev. Microbiol.</em><strong>15</strong>, 697–703 (2017).
</cite> [<a href="https://doi.org/10.1038/nrmicro.2017.103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29021600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burnham,%20C.%20A.%20D.%20et%20al.%20Diagnosing%20antimicrobial%20resistance.%20Nat.%20Rev.%20Microbiol.15,%20697%E2%80%93703%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Yamin, D. et al. Current and future technologies for the detection of antibiotic-resistant bacteria. <em>Diagnostics (Basel)</em>. <strong>13</strong>, 3246 (2023).
</cite> [<a href="https://doi.org/10.3390/diagnostics13203246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10606640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37892067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamin,%20D.%20et%20al.%20Current%20and%20future%20technologies%20for%20the%20detection%20of%20antibiotic-resistant%20bacteria.%20Diagnostics%20(Basel).%2013,%203246%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Kherabi, Y. et al. Machine learning to predict antimicrobial resistance: future applications in clinical practice? <em>Infect. Dis. Now</em>. <strong>54</strong>, 104864 (2024).
</cite> [<a href="https://doi.org/10.1016/j.idnow.2024.104864" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38355048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kherabi,%20Y.%20et%20al.%20Machine%20learning%20to%20predict%20antimicrobial%20resistance:%20future%20applications%20in%20clinical%20practice?%20Infect.%20Dis.%20Now.%2054,%20104864%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Djordjevic, S. P. et al. Genomic surveillance for antimicrobial resistance — a one health perspective. <em>Nat. Rev. Genet.</em><strong>25</strong>, 142–157 (2024).
</cite> [<a href="https://doi.org/10.1038/s41576-023-00649-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37749210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Djordjevic,%20S.%20P.%20et%20al.%20Genomic%20surveillance%20for%20antimicrobial%20resistance%20%E2%80%94%20a%20one%20health%20perspective.%20Nat.%20Rev.%20Genet.25,%20142%E2%80%93157%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Mutisya, F. &amp; Kanguha, R. AntiMicro.ai: an artificial intelligence powered web application for predicting antibacterial/antifungal susceptibility and constructing personalized machine learning models. <em>Wellcome Open. Res.</em><strong>9</strong>, 273 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mutisya,%20F.%20&amp;%20Kanguha,%20R.%20AntiMicro.ai:%20an%20artificial%20intelligence%20powered%20web%20application%20for%20predicting%20antibacterial/antifungal%20susceptibility%20and%20constructing%20personalized%20machine%20learning%20models.%20Wellcome%20Open.%20Res.9,%20273%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Catalán, P. et al. Seeking patterns of antibiotic resistance in ATLAS, an open, Raw MIC database with patient metadata. <em>Nat. Commun.</em><strong>13</strong>, 2917 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-30635-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9133080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35614098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Catal%C3%A1n,%20P.%20et%20al.%20Seeking%20patterns%20of%20antibiotic%20resistance%20in%20ATLAS,%20an%20open,%20Raw%20MIC%20database%20with%20patient%20metadata.%20Nat.%20Commun.13,%202917%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Fears, R. Ter meulen, V. What do we need to do to tackle antimicrobial resistance? <em>Lancet Glob Health</em>. <strong>2</strong>, e11–e12 (2014).
</cite> [<a href="https://doi.org/10.1016/S2214-109X(13)70086-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25104623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fears,%20R.%20Ter%20meulen,%20V.%20What%20do%20we%20need%20to%20do%20to%20tackle%20antimicrobial%20resistance?%20Lancet%20Glob%20Health.%202,%20e11%E2%80%93e12%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kariuki, S. et al. Antimicrobial resistance rates and surveillance in Sub-Saharan africa: where are we now? <em>Infect. Drug Resist.</em><strong>15</strong>, 3589–3609 (2022).
</cite> [<a href="https://doi.org/10.2147/IDR.S342753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9273632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35837538/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kariuki,%20S.%20et%20al.%20Antimicrobial%20resistance%20rates%20and%20surveillance%20in%20Sub-Saharan%20africa:%20where%20are%20we%20now?%20Infect.%20Drug%20Resist.15,%203589%E2%80%933609%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Shadbahr, T. et al. The impact of imputation quality on machine learning classifiers for datasets with missing values. <em>Commun. Med.</em><strong>3</strong>, 139 (2023).
</cite> [<a href="https://doi.org/10.1038/s43856-023-00356-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10558448/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37803172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shadbahr,%20T.%20et%20al.%20The%20impact%20of%20imputation%20quality%20on%20machine%20learning%20classifiers%20for%20datasets%20with%20missing%20values.%20Commun.%20Med.3,%20139%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Bush, K. &amp; Bradford, P. A. β lactams and β lactamase inhibitors: an overview. <em>Cold Spring Harb Perspect. Med.</em><strong>6</strong>, a025247 (2016).
</cite> [<a href="https://doi.org/10.1101/cshperspect.a025247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4968164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27329032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bush,%20K.%20&amp;%20Bradford,%20P.%20A.%20%CE%B2%20lactams%20and%20%CE%B2%20lactamase%20inhibitors:%20an%20overview.%20Cold%20Spring%20Harb%20Perspect.%20Med.6,%20a025247%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Cantón, R., González-Alba, J. M. &amp; Galán, J. C. CTX-M enzymes: origin and diffusion. <em>Front. Microbiol.</em><strong>3</strong>, 110 (2012).
</cite> [<a href="https://doi.org/10.3389/fmicb.2012.00110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3316993/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22485109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cant%C3%B3n,%20R.,%20Gonz%C3%A1lez-Alba,%20J.%20M.%20&amp;%20Gal%C3%A1n,%20J.%20C.%20CTX-M%20enzymes:%20origin%20and%20diffusion.%20Front.%20Microbiol.3,%20110%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Tzouvelekis, L. S. et al. Carbapenemases in <em>Klebsiella pneumoniae</em> and other enterobacteriaceae: an evolving crisis of global dimensions. <em>Clin. Microbiol. Rev.</em><strong>25</strong>, 682–707 (2012).
</cite> [<a href="https://doi.org/10.1128/CMR.05035-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3485753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23034326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tzouvelekis,%20L.%20S.%20et%20al.%20Carbapenemases%20in%20Klebsiella%20pneumoniae%20and%20other%20enterobacteriaceae:%20an%20evolving%20crisis%20of%20global%20dimensions.%20Clin.%20Microbiol.%20Rev.25,%20682%E2%80%93707%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Doi, Y., Iovleva, A. &amp; Bonomo, R. A. The ecology of extended spectrum β lactamases (ESBLs) in the developed world. <em>J. Travel Med.</em><strong>24</strong>, S44–S51 (2017).
</cite> [<a href="https://doi.org/10.1093/jtm/taw102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5731446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28521000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doi,%20Y.,%20Iovleva,%20A.%20&amp;%20Bonomo,%20R.%20A.%20The%20ecology%20of%20extended%20spectrum%20%CE%B2%20lactamases%20(ESBLs)%20in%20the%20developed%20world.%20J.%20Travel%20Med.24,%20S44%E2%80%93S51%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Nordmann, P., Naas, T. &amp; Poirel, L. Global spread of carbapenemase producing Enterobacteriaceae. <em>Emerg. Infect. Dis.</em><strong>17</strong>, 1791–1798 (2011).
</cite> [<a href="https://doi.org/10.3201/eid1710.110655" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3310682/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22000347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nordmann,%20P.,%20Naas,%20T.%20&amp;%20Poirel,%20L.%20Global%20spread%20of%20carbapenemase%20producing%20Enterobacteriaceae.%20Emerg.%20Infect.%20Dis.17,%201791%E2%80%931798%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Lundberg, S. M. &amp; Lee, S. I. A unified approach to interpreting model predictions. <em>Adv. Neural Inf. Process. Syst.</em><strong>30</strong>, 4768–4777 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lundberg,%20S.%20M.%20&amp;%20Lee,%20S.%20I.%20A%20unified%20approach%20to%20interpreting%20model%20predictions.%20Adv.%20Neural%20Inf.%20Process.%20Syst.30,%204768%E2%80%934777%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Paterson, D. L. &amp; Bonomo, R. A. Extended spectrum β lactamases: a clinical update. <em>Clin. Microbiol. Rev.</em><strong>18</strong>, 657–665 (2005).
</cite> [<a href="https://doi.org/10.1128/CMR.18.4.657-686.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1265908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16223952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paterson,%20D.%20L.%20&amp;%20Bonomo,%20R.%20A.%20Extended%20spectrum%20%CE%B2%20lactamases:%20a%20clinical%20update.%20Clin.%20Microbiol.%20Rev.18,%20657%E2%80%93665%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Chambers, H. F. &amp; DeLeo, F. R. Waves of resistance: <em>Staphylococcus aureus</em> in the antibiotic era. <em>Nat. Rev. Microbiol.</em><strong>7</strong>, 629–641 (2009).
</cite> [<a href="https://doi.org/10.1038/nrmicro2200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2871281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19680247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chambers,%20H.%20F.%20&amp;%20DeLeo,%20F.%20R.%20Waves%20of%20resistance:%20Staphylococcus%20aureus%20in%20the%20antibiotic%20era.%20Nat.%20Rev.%20Microbiol.7,%20629%E2%80%93641%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Yaghoubi, S. et al. Tigecycline antibacterial activity, clinical effectiveness, and mechanisms and epidemiology of resistance: narrative review. <em>Eur. J. Clin. Microbiol. Infect. Dis.</em><strong>41</strong>, 1003–1022 (2022).
</cite> [<a href="https://doi.org/10.1007/s10096-020-04121-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7785128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33403565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yaghoubi,%20S.%20et%20al.%20Tigecycline%20antibacterial%20activity,%20clinical%20effectiveness,%20and%20mechanisms%20and%20epidemiology%20of%20resistance:%20narrative%20review.%20Eur.%20J.%20Clin.%20Microbiol.%20Infect.%20Dis.41,%201003%E2%80%931022%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Liu, C. et al. Clinical practice guidelines by the infectious diseases society of America for treatment of methicillin resistant <em>Staphylococcus aureus</em> infections in adults and children. <em>Clin. Infect. Dis.</em><strong>52</strong>, e18–e55 (2011).
</cite> [<a href="https://doi.org/10.1093/cid/ciq146" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21208910/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20C.%20et%20al.%20Clinical%20practice%20guidelines%20by%20the%20infectious%20diseases%20society%20of%20America%20for%20treatment%20of%20methicillin%20resistant%20Staphylococcus%20aureus%20infections%20in%20adults%20and%20children.%20Clin.%20Infect.%20Dis.52,%20e18%E2%80%93e55%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Clinical and Laboratory Standards Institute. Performance standards for antimicrobial susceptibility testing (33rd ed.). (2023).</cite>
</li>
<li id="CR31">
<span class="label">31.</span><cite>Poirel, L., Jayol, A. &amp; Nordmann, P. Polymyxins: antibacterial activity, susceptibility testing, and resistance mechanisms encoded by plasmids or chromosomes. <em>Clin. Microbiol. Rev.</em><strong>30</strong>, 557–596 (2017).
</cite> [<a href="https://doi.org/10.1128/CMR.00064-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5355641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28275006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poirel,%20L.,%20Jayol,%20A.%20&amp;%20Nordmann,%20P.%20Polymyxins:%20antibacterial%20activity,%20susceptibility%20testing,%20and%20resistance%20mechanisms%20encoded%20by%20plasmids%20or%20chromosomes.%20Clin.%20Microbiol.%20Rev.30,%20557%E2%80%93596%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Davies, J. &amp; Davies, D. Origins and evolution of antibiotic resistance. <em>Microbiol. Mol. Biol. Rev.</em><strong>74</strong>, 417–433 (2010).
</cite> [<a href="https://doi.org/10.1128/MMBR.00016-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2937522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20805405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davies,%20J.%20&amp;%20Davies,%20D.%20Origins%20and%20evolution%20of%20antibiotic%20resistance.%20Microbiol.%20Mol.%20Biol.%20Rev.74,%20417%E2%80%93433%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Shields, R. K. et al. Emergence of Ceftazidime avibactam resistance due to plasmid borne blaKPC-3 mutations during treatment of carbapenem resistant <em>Klebsiella pneumoniae</em> infections. <em>Antimicrob. Agents Chemother.</em><strong>61</strong>, e02097–e02016 (2017).
</cite> [<a href="https://doi.org/10.1128/AAC.02097-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5328542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28031201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shields,%20R.%20K.%20et%20al.%20Emergence%20of%20Ceftazidime%20avibactam%20resistance%20due%20to%20plasmid%20borne%20blaKPC-3%20mutations%20during%20treatment%20of%20carbapenem%20resistant%20Klebsiella%20pneumoniae%20infections.%20Antimicrob.%20Agents%20Chemother.61,%20e02097%E2%80%93e02016%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Nicolas-Chanoine, M. H., Bertrand, X. &amp; Madec, J. Y. <em>Escherichia coli</em> ST131, an intriguing clonal group. <em>Clin. Microbiol. Rev.</em><strong>27</strong>, 543–574 (2014).
</cite> [<a href="https://doi.org/10.1128/CMR.00125-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4135899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24982321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nicolas-Chanoine,%20M.%20H.,%20Bertrand,%20X.%20&amp;%20Madec,%20J.%20Y.%20Escherichia%20coli%20ST131,%20an%20intriguing%20clonal%20group.%20Clin.%20Microbiol.%20Rev.27,%20543%E2%80%93574%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Munita, J. M. &amp; Arias, C. A. Mechanisms of antibiotic resistance. <em>Microbiol. Spectr.</em> 4, VMBF-0016-2015 (2016).</cite> [<a href="https://doi.org/10.1128/microbiolspec.VMBF-0016-2015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4888801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27227291/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Bell, B. G. et al. A systematic review and meta-analysis of the effects of antibiotic consumption on antibiotic resistance. <em>BMC Infect. Dis.</em><strong>14</strong>, 13 (2014).
</cite> [<a href="https://doi.org/10.1186/1471-2334-14-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3897982/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24405683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bell,%20B.%20G.%20et%20al.%20A%20systematic%20review%20and%20meta-analysis%20of%20the%20effects%20of%20antibiotic%20consumption%20on%20antibiotic%20resistance.%20BMC%20Infect.%20Dis.14,%2013%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Cherny, S. S. et al. Revealing antibiotic cross-resistance patterns in hospitalized patients through bayesian network modelling. <em>J. Antimicrob. Chemother.</em><strong>76</strong>, 239–248 (2021).
</cite> [<a href="https://doi.org/10.1093/jac/dkaa408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33020811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cherny,%20S.%20S.%20et%20al.%20Revealing%20antibiotic%20cross-resistance%20patterns%20in%20hospitalized%20patients%20through%20bayesian%20network%20modelling.%20J.%20Antimicrob.%20Chemother.76,%20239%E2%80%93248%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Kaur, J., Singh, H. &amp; Sethi, T. Emerging trends in antimicrobial resistance in bloodstream infections: multicentric longitudinal study in India (2017–2022). <em>Lancet Reg. Health Southeast. Asia</em>. <strong>26</strong>, 100412 (2024).
</cite> [<a href="https://doi.org/10.1016/j.lansea.2024.100412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11097075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38757091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaur,%20J.,%20Singh,%20H.%20&amp;%20Sethi,%20T.%20Emerging%20trends%20in%20antimicrobial%20resistance%20in%20bloodstream%20infections:%20multicentric%20longitudinal%20study%20in%20India%20(2017%E2%80%932022).%20Lancet%20Reg.%20Health%20Southeast.%20Asia.%2026,%20100412%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Yong, D. et al. Characterization of a new metallo-beta-lactamase gene, bla(NDM-1), and a novel erythromycin esterase gene carried on a unique genetic structure in <em>Klebsiella pneumoniae</em> sequence type 14 from India. <em>Antimicrob. Agents Chemother.</em><strong>53</strong>, 5046–5055 (2009).
</cite> [<a href="https://doi.org/10.1128/AAC.00774-09" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2786356/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19770275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yong,%20D.%20et%20al.%20Characterization%20of%20a%20new%20metallo-beta-lactamase%20gene,%20bla(NDM-1),%20and%20a%20novel%20erythromycin%20esterase%20gene%20carried%20on%20a%20unique%20genetic%20structure%20in%20Klebsiella%20pneumoniae%20sequence%20type%2014%20from%20India.%20Antimicrob.%20Agents%20Chemother.53,%205046%E2%80%935055%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Cantón, R. &amp; Coque, T. M. The CTX-M beta-lactamase pandemic. <em>Curr. Opin. Microbiol.</em><strong>9</strong>, 466–475 (2006).
</cite> [<a href="https://doi.org/10.1016/j.mib.2006.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16942899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cant%C3%B3n,%20R.%20&amp;%20Coque,%20T.%20M.%20The%20CTX-M%20beta-lactamase%20pandemic.%20Curr.%20Opin.%20Microbiol.9,%20466%E2%80%93475%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Mairi, A. et al. OXA-48-like carbapenemases producing Enterobacteriaceae in different niches. <em>Eur. J. Clin. Microbiol. Infect. Dis.</em><strong>37</strong>, 587–604 (2018).
</cite> [<a href="https://doi.org/10.1007/s10096-017-3112-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28990132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mairi,%20A.%20et%20al.%20OXA-48-like%20carbapenemases%20producing%20Enterobacteriaceae%20in%20different%20niches.%20Eur.%20J.%20Clin.%20Microbiol.%20Infect.%20Dis.37,%20587%E2%80%93604%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Moradigaravand, D. et al. Prediction of antibiotic resistance in <em>Escherichia coli</em> from large-scale pan-genome data. <em>PLoS Comput. Biol.</em><strong>14</strong>, e1006258 (2018).
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1006258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6310291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30550564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moradigaravand,%20D.%20et%20al.%20Prediction%20of%20antibiotic%20resistance%20in%20Escherichia%20coli%20from%20large-scale%20pan-genome%20data.%20PLoS%20Comput.%20Biol.14,%20e1006258%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Waterlow, N. R. et al. Antimicrobial resistance prevalence in bloodstream infection in 29 European countries by age and sex: an observational study. <em>PLoS Med.</em><strong>21</strong>, e1004301 (2024).
</cite> [<a href="https://doi.org/10.1371/journal.pmed.1004301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10939247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38484006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waterlow,%20N.%20R.%20et%20al.%20Antimicrobial%20resistance%20prevalence%20in%20bloodstream%20infection%20in%2029%20European%20countries%20by%20age%20and%20sex:%20an%20observational%20study.%20PLoS%20Med.21,%20e1004301%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Davies, N. G. et al. Within-host dynamics shape antibiotic resistance in commensal bacteria. <em>Nat. Ecol. Evol.</em><strong>3</strong>, 440–449 (2019).
</cite> [<a href="https://doi.org/10.1038/s41559-018-0786-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6420107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30742105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davies,%20N.%20G.%20et%20al.%20Within-host%20dynamics%20shape%20antibiotic%20resistance%20in%20commensal%20bacteria.%20Nat.%20Ecol.%20Evol.3,%20440%E2%80%93449%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Van Camp, P. J. et al. Prediction of antimicrobial resistance in Gram-negative bacteria from whole-genome sequencing data. <em>Front. Microbiol.</em><strong>11</strong>, 1013 (2020).
</cite> [<a href="https://doi.org/10.3389/fmicb.2020.01013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7262952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32528441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Camp,%20P.%20J.%20et%20al.%20Prediction%20of%20antimicrobial%20resistance%20in%20Gram-negative%20bacteria%20from%20whole-genome%20sequencing%20data.%20Front.%20Microbiol.11,%201013%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Ren, Y. et al. Prediction of antimicrobial resistance based on whole-genome sequencing and machine learning. <em>Bioinformatics</em><strong>38</strong>, 325–334 (2022).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8722762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34613360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ren,%20Y.%20et%20al.%20Prediction%20of%20antimicrobial%20resistance%20based%20on%20whole-genome%20sequencing%20and%20machine%20learning.%20Bioinformatics38,%20325%E2%80%93334%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Rahbé, E. et al. Investigating the feasibility and potential of combining industry AMR monitoring systems: a comparison with WHO GLASS. <em>Wellcome Open. Res.</em><strong>9</strong>, 248 (2024).
</cite> [<a href="https://doi.org/10.12688/wellcomeopenres.21181.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11452768/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39372841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rahb%C3%A9,%20E.%20et%20al.%20Investigating%20the%20feasibility%20and%20potential%20of%20combining%20industry%20AMR%20monitoring%20systems:%20a%20comparison%20with%20WHO%20GLASS.%20Wellcome%20Open.%20Res.9,%20248%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Kim, J. et al. Predicting antimicrobial resistance of bacterial pathogens using time series analysis. <em>Front. Microbiol.</em><strong>14</strong>, 1160224 (2023).
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1160224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10213968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37250043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20J.%20et%20al.%20Predicting%20antimicrobial%20resistance%20of%20bacterial%20pathogens%20using%20time%20series%20analysis.%20Front.%20Microbiol.14,%201160224%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Vivli, A. M. R. Register. Available at: <a href="https://amr.vivli.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://amr.vivli.org</a> (2024). Accessed on 30 Jul 2024 .</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>World Health Organization. <em>Ethics and governance of artificial intelligence for health</em> (2021).</cite>
</li>
<li id="CR51">
<span class="label">51.</span><cite>Organisation for Economic Co-operation and Development. <em>Recommendation Council Artif. Intelligence</em>, <strong>C/MIN(2024)16/FINAL</strong> (2024).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368220/bin/41598_2025_14078_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (451.5KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>This publication is based on research using data from Pfizer obtained through the Vivli AMR Register https://amr.vivli.org. The Pfizer ATLAS Antibiotics dataset (https://searchamr.vivli.org/datasetDetails/fromSearch/ce7367e1-c91a-4de1-87e4-6e4d8f834085) used in this study is available on request from the Vivli AMR Register. Data requests should be directed to the Vivli AMR platform and are subject to the data-sharing agreements and policies set by the platform. For further details regarding the data request process, refer to the official documentation of the Vivli AMR Register or contact their support team at amr@vivli.org. The source code of the implementation is available at https://github.com/softwareinnovationslabBITS/AMR_AST_ML.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-14078-w"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_14078.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368220/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368220/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368220%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368220/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368220/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368220/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835633/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368220/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835633/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368220/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368220/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="6T4C2vzX3LAO8ClsueK8yB7Nj9aquFfL6RnlPzYEN3aJqws5KUppLQ2zmq30B4RN">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
