
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Complete mitochondrial genome of Polygonatum cyrtonema Hua reveals variation diversity and evolutionary trends - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152538DA8AF1EB630538DA005C996317.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcps">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369140/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Plant Biology">
<meta name="citation_title" content="Complete mitochondrial genome of Polygonatum cyrtonema Hua reveals variation diversity and evolutionary trends">
<meta name="citation_author" content="Haoxiang Luo">
<meta name="citation_author_institution" content="College of Life Sciences and Medicine, Zhejiang Sci-Tech University, Hangzhou, 310018 PR China">
<meta name="citation_author" content="Fengxiao Ren">
<meta name="citation_author_institution" content="College of Life Sciences and Medicine, Zhejiang Sci-Tech University, Hangzhou, 310018 PR China">
<meta name="citation_author" content="Guochao Cao">
<meta name="citation_author_institution" content="College of Life Sciences and Medicine, Zhejiang Sci-Tech University, Hangzhou, 310018 PR China">
<meta name="citation_author" content="Dekai Wang">
<meta name="citation_author_institution" content="College of Life Sciences and Medicine, Zhejiang Sci-Tech University, Hangzhou, 310018 PR China">
<meta name="citation_author" content="Qiaojun Jia">
<meta name="citation_author_institution" content="College of Life Sciences and Medicine, Zhejiang Sci-Tech University, Hangzhou, 310018 PR China">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="1109">
<meta name="citation_doi" content="10.1186/s12870-025-07074-9">
<meta name="citation_pmid" content="40841885">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369140/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369140/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369140/pdf/12870_2025_Article_7074.pdf">
<meta name="description" content="Polygonatum cyrtonema Hua, a valuable medicinal and edible plant in China, has been increasingly cultivated to produce nutritional and health products to meet market demand. However, the absence of the mitochondrial genome hinders the research of ...">
<meta name="og:title" content="Complete mitochondrial genome of Polygonatum cyrtonema Hua reveals variation diversity and evolutionary trends">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Polygonatum cyrtonema Hua, a valuable medicinal and edible plant in China, has been increasingly cultivated to produce nutritional and health products to meet market demand. However, the absence of the mitochondrial genome hinders the research of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369140/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12369140">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12870-025-07074-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12870_2025_Article_7074.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369140%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12369140/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12369140/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369140/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcps.png" alt="BMC Plant Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Plant Biology" title="Link to BMC Plant Biology" shape="default" href="https://bmcplantbiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Plant Biol</button></div>. 2025 Aug 21;25:1109. doi: <a href="https://doi.org/10.1186/s12870-025-07074-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12870-025-07074-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Plant%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Complete mitochondrial genome of <em>Polygonatum cyrtonema</em> Hua reveals variation diversity and evolutionary trends</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Haoxiang Luo</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Haoxiang Luo</span></h3>
<div class="p">
<sup>1</sup>College of Life Sciences and Medicine, Zhejiang Sci-Tech University, Hangzhou, 310018 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haoxiang Luo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Fengxiao Ren</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Fengxiao Ren</span></h3>
<div class="p">
<sup>1</sup>College of Life Sciences and Medicine, Zhejiang Sci-Tech University, Hangzhou, 310018 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ren%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fengxiao Ren</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Guochao Cao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Guochao Cao</span></h3>
<div class="p">
<sup>1</sup>College of Life Sciences and Medicine, Zhejiang Sci-Tech University, Hangzhou, 310018 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cao%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guochao Cao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Dekai Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Dekai Wang</span></h3>
<div class="p">
<sup>1</sup>College of Life Sciences and Medicine, Zhejiang Sci-Tech University, Hangzhou, 310018 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dekai Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jia%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Qiaojun Jia</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Qiaojun Jia</span></h3>
<div class="p">
<sup>1</sup>College of Life Sciences and Medicine, Zhejiang Sci-Tech University, Hangzhou, 310018 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jia%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qiaojun Jia</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>College of Life Sciences and Medicine, Zhejiang Sci-Tech University, Hangzhou, 310018 PR China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 28; Accepted 2025 Jul 16; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12369140  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841885/" class="usa-link">40841885</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1"><em>Polygonatum cyrtonema Hua</em>, a valuable medicinal and edible plant in China, has been increasingly cultivated to produce nutritional and health products to meet market demand. However, the absence of the mitochondrial genome hinders the research of evolutionary analysis, artificial cultivation, and medicinal resource development. This study aimed to systematically characterize the mitochondrial genome of <em>P. cyrtonema</em> and perform a preliminary bioinformatic analysis.</p></section><section id="sec2"><h3 class="pmc_sec_title">Result</h3>
<p id="Par2">In this study, the mitochondrial genome of <em>P. cyrtonema</em> was assembled and annotated by combining Illumina reads with long-read Oxford Nanopore Technologies, which formed a complex non-circular structure covering 664,991 bp in length, with 33 protein-coding genes, 20 tRNAs, and 3 rRNAs. Overall, a total of 194 simple repeats, 24 tandem repeats, and 294 dispersed repeats were identified, and 599 RNA editing sites were predicted, all of which were C-to-T types. In particular, 29 homologous fragments between the chloroplast and mitochondrial genome were detected and accounted for 2.4% of the mitochondrial genome. In addition, codon usage analysis, nucleotide diversity analysis, and Ka/Ks analysis suggested a slower rate of evolution and a relatively conserved structure of the mitochondrial genome. Phylogenetic relationships based on 33 species demonstrated that <em>P. cyrtonema</em> was most closely related to <em>Polygonatum sibiricum</em>.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par3">Our results provide comprehensive information on the mitochondrial genome of <em>P. cyrtonema</em>, and demonstrate the availability of mitochondrial genome-based taxonomic classification for <em>Polygonatum</em> Mill.. Moreover, it offers a valuable foundation for future research in the cultivation and pharmacological development within <em>Polygonatum</em> species.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12870-025-07074-9.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Mitochondrial genome, Phylogenetic evolution, <em>Polygonatum cyrtonema</em> Hua, Repeated sequences, RNA editing</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par11"><em>Polygonatum cyrtonema</em> Hua (<em>P. cyrtonema</em>), a perennial herb of the Asparagaceae family, grows in forests, scrublands, and shaded hillsides [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>], and is primarily distributed in the middle and lower reaches of the Yangtze River, including the provinces of Guizhou, Zhejiang, Hunan, Hubei, Anhui, and Sichuan [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. <em>P. cyrtonema</em> is rich in active pharmaceutical ingredients, such as polysaccharides, flavonoids, and steroidal saponins, which is one of the official Chinese Pharmacopoeia <em>Polygonatum</em> species (Edition 2020) [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Modern pharmacological studies revealed that <em>P. cyrtonema</em> exerted various physiological effects, including the enhancement of fatigue resistance, antioxidant activity, immune system regulation, modulation of the intestinal microbiome, and anti-cancer properties [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. As a result, <em>P. cyrtonema</em> is widely used in healthcare, food, and medicine with its growing demand. At present, <em>P. cyrtonema</em> sourced from their native wild habitat is gradually being depleted or destroyed, and artificial cultivation is an effective means to meet market demand.</p>
<p id="Par12">Chloroplasts and mitochondria are important organelles in plants. The former are responsible for the process of photosynthesis, which involve the conversion of solar energy into stable chemical energy, and the storage of organic matter to sustain life on Earth [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. The latter function as a source of energy, facilitating the decomposition of organic matter through respiration, a process that releases carbon dioxide and heat energy [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>], which are essential for regulating the growth and circulation of plant cells [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Because of their maternal inheritance, both chloroplasts and mitochondria genomes are widely used in phylogenetics and evolutionary biology [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], which is important for plant cultivation [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. However, the assembly of mitochondrial genomes has lagged behind that of chloroplast genomes because of their highly conserved and complex structure [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Mitochondrial genomes are widely used in biogeographic and population genetic investigation due to its slowly evolution [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Moreover, owing to the development of improved sequencing and assembly technologies, a variety of plant mitochondrial genomes have been published, providing new insights into the evolution of species. <em>P. cyrtonema</em>, <em>Polygonatum filipes</em>, and <em>Polygonatum odoratum</em> are assigned to the sect. <em>Polygonatum</em> clade based on chloroplast genome sequences [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. Unfortunately, the current molecular markers and morphological characteristics don’t satisfy the demands for identifying <em>Polygonatum</em> Mill., possibly due to the limited number of molecular markers and their similar phenotypes. Moreover, only four mitochondrial genomes of <em>Asparagus officinalis</em> [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], <em>Chlorophytum comosum</em> [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>], <em>Polygonatum sibiricum</em> [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] and <em>Polygonatum kingianum</em> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ374235.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ374235.1</a>) in the Asparagaceae family were published, which might impede to investigate systematic evolution of such family. Therefore, it is necessary to characterize the mitochondrial genome of <em>P. cyrtonema</em> to classify <em>Polygonatum</em> Mill. and understand the phylogeny of Asparagaceae.</p>
<p id="Par13">In this study, the mitochondrial genome of <em>P. cyrtonema</em> was assembled using the second and third-generation sequencing strategy. The characteristics of the its mitochondrial genome were analyzed by repeat sequences, RNA editing, codon preference, and so on. The phylogenetic relationships of <em>P. cyrtonema</em> and other species were inferred using mitochondrial protein-coding genes (PCGs). Overall, this study provides mitochondrial genome information for <em>P. cyrtonema</em> to artificial cultivation and germplasm identification.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Plant materials, DNA extraction and sequencing</h3>
<p id="Par14"><em>P. cyrtonema</em> collected from Jiuhuashan, Qingyang County, Chizhou City, Anhui Province, China, were cultivated in Yuhang District, Hangzhou City, Zhejiang Province, China. The well-grown young leaves of <em>P. cyrtonema</em> were harvested and immediately frozen in liquid nitrogen for total genomic DNA extraction.</p>
<p id="Par15">Total genomic DNA was extracted using the Plant Genomic DNA Kit (DP305, Tiangen, Beijing, China), and its concentration and quality was evaluated by electrophoresis and NanoDrop. After successful construction of the library, high-quality genomic DNA was sequenced using Illumina Novaseq6000 (Illumina, San Diego, CA, USA) and the Oxford Nanopore PromethION sequencer (Oxford Nanopore Technologies, Oxford, UK) by Nanjing Genepioneer Biotechnologies Inc. (Nanjing, China). The Illumina and Nanopore sequencing data were filtered using fastp software (v0.20.0) (<a href="https://github.com/OpenGene/fastp" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/OpenGene/fastp</a>) [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] and filtlong software (v0.2.1) (<a href="https://link.zhihu.com/?target=https%3A//github.com/rrwick/Filtlong" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://link.zhihu.com/?target=https%3A//github.com/rrwick/Filtlong</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Mitochondrial genome assembly and annotation</h3>
<p id="Par16">The mitochondrial genome of <em>P. cyrtonema</em> was assembled using a combination of the filtered Illumina and Nanopore strategies. Firstly, the Nanopore sequences were aligned to the references including coding sequences (CDS) and rRNA from published plant mitochondrial genomes using the three-generation comparison software minimap2 (v2.1) (<a href="https://github.com/xul962464/plant_mt_ref_gene" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/xul962464/plant_mt_ref_gene</a>) [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. And the aligned sequences with a comparison length greater than 50 bp were selected as candidate seed sequences. After the Nanopore sequences were aligned with the candidate seed sequences using minimap2 (v2.1) [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>], the seed sequences were selected with an overlap greater than 1 kb and a similarity greater than 70%. The Nanopore sequences were iteratively compared to the seed sequences to obtain all the Nanopore sequences of the mitochondrial genome, which were corrected using the third-generation assembly software Canu (v1.5) [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Bowtie2 (v2.3.5.1) [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] was employed to align the Illumina sequences with the corrected Nanopore sequences. Subsequently, Unicycler (v0.4.8) [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] with default parameters was used to concatenate the above mentioned second-generation data with the corrected third-generation data, and minimap2 (v2.1) [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] was used to compare the corrected third-generation sequencing data with the contig obtained in the second step of Unicycler (v0.4.8) [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Finally, the above Illumina sequences and the corrected Nanopore sequences were concatenated using Unicycler (v0.4.8) [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] with default parameters. The corrected Nanopore sequences were compared to the contigs obtained in the second step of Unicycler (v0.4.8) [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] by using minimap2 (v2.1) [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Branch orientation was determined manually to obtain the final assembly result.</p>
<p id="Par17">The assembled mitochondrial genome of <em>P. cyrtonema</em> was annotated as follows: (1) PCGs and rRNAs were compared with the published plant mitochondrial sequences using default parameter by blast (v2.14.0) [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>], (2) tRNA and ORF were identified using the program tRNAscan-SE [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] and the OpenReading Frame Finder tool with default parameter (<a href="http://www.ncbi.nlm.nih.gov/gorf/gorf.html" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ncbi.nlm.nih.gov/gorf/gorf.html</a>). The assembled mitochondrial genome of <em>P. cyrtonema</em> was mapped and visualized using OrganellarGenomeDraw (<a href="https://chlorobox.mpimp-golm.mpg.de/OGDraw.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://chlorobox.mpimp-golm.mpg.de/OGDraw.html</a>) [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p></section><section id="Sec5"><h3 class="pmc_sec_title">Repeat sequence analysis</h3>
<p id="Par18">Simple sequence repeats (SSRs) were detected using the MISA tool (v1.0) [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] with the parameters: 1 bp-10 repetitions, 2 bp-5 repetitions, 3 bp-4 repetitions, 4 bp-3 repetitions, 5 bp-3 repetitions, and 6 bp-3 repetitions. Tandem repeats were identified using the TRF tool (v4.10.0) [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] with the parameters: Match = 2, Mismatch = 7, Delta = 7, PM = 80, PI = 10, Minscore = 50, Maxperiod = 2000. Dispersed repeats were identified using the blastn tool (v2.14.0) [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] with the parameters: word_size = 7; E-value = 1e-5. The final result was visualized using the Circos tool (v0.69–5) [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>].</p></section><section id="Sec6"><h3 class="pmc_sec_title">Analysis of RNA editing site and homologous sequence among organelle genomes</h3>
<p id="Par19">The potential RNA editing sites of <em>P. cyrtonema</em> PCGs were predicted using a tool PmtREP (<a href="http://112.86.217.82:9919/#/tool/alltool/detail/336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://112.86.217.82:9919/#/tool/alltool/detail/336</a>). Blast (v2.14.0) [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] was used to find the homologous sequences between the mitochondrial and chloroplast genome (<a href="https://www.ncbi.nlm.nih.gov/nuccore/OL436258" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">OL436258</a>) [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. The E-value was set to 1e-5 with the similarity more than 70%. The results of homologous sequence results were visualized by Circos (v0.69–5) [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>].</p></section><section id="Sec7"><h3 class="pmc_sec_title">Analysis of RSCU and nucleotide diversity</h3>
<p id="Par20">The RSCU of the mitochondrial genome of <em>P. cyrtonema</em> was calculated using a self-written Perl script (<a href="http://cloud.genepioneer.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://cloud.genepioneer.com/</a>) based on the PCGs. Homologous genes between <em>P. cyrtonema</em> and five other species (<em>Phoenix dactylifera</em>, <em>Asparagus officinalis</em>, <em>Allium cepa</em>, <em>Chlorophytum comosum</em>, <em>Crocus sativus</em>) were obtained using the comparison rate score of blast and were multiply-aligned by MAFFT software (v7.427) [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Pi values of each genes were calculated using DnaSP5 [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>].</p></section><section id="Sec8"><h3 class="pmc_sec_title">Analysis of phylogenetic tree</h3>
<p id="Par21">Maximum likelihood phylogenetic trees (ML) were constructed using mitochondrial genome PCGs of <em>P. cyrtonema</em> and other 32 selected species (Table S1). Multiple sequence alignment was performed with MAFFT software (v7.427, auto mode) [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Then, the aligned sequences were connected end to end and trimmed by TrimAI (v1.4.rev15) [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. The ML tree was generated using the RAxML software (v8.2.10) [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] with the best-fit model (GTR-GAMMA) determined by Jmodeltest (2.1.10) [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>] at 1000 bootstrap replicas.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Analysis of selective pressure</h3>
<p id="Par22">Multiple alignment of PCGs of <em>P. cyrtonema</em> and five other species (<em>Phoenix dactylifera</em>, <em>Asparagus officinalis</em>, <em>Allium cepa</em>, <em>Chlorophytum comosum</em>, <em>Crocus sativus</em>) were performed by MAFFT software (v7.427) [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. KaKs_Calculator v2.0 [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] software was employed to calculate the ratio of non-synonymous to synonymous substitutions (Ka/Ks) using the MLWL method with default parameter.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Mitochondrial genome characteristics</h3>
<p id="Par23">The complete mitochondrial genome of <em>P. cyrtonema</em> was visualized as a complex and non-circular DNA molecule with 21 contigs by Bandage [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] (Fig. <a href="#Fig1" class="usa-link">1</a>), totaling 664,991 bp with an average GC content of 46.30%. The length of contigs ranged from 209 bp to 125,074 bp with similar depth of coverage. Subsequently, a total of 56 functionally annotated genes were identified in the mitochondrial genome of <em>P. cyrtonema</em>, including 33 protein-coding genes (PCGs), 20 tRNAs, and 3 rRNAs (Fig. <a href="#Fig2" class="usa-link">2</a>). The PCGs could be divided into 9 different functional groups, which were described as follows: ATP synthase (5 genes), cytochrome c biogenesis (4 genes), ubiquinol cytochrome c reductase (1 gene), cytochrome c oxidase (3 genes), maturase (1 gene), transport membrane protein (1 gene), NADH dehydrogenase (9 genes), ribosomal protein (LSU) (2 genes), and ribosomal protein (SSU) (7 genes) (Table <a href="#Tab1" class="usa-link">1</a>). The total length of the PCGs was 43,620 bp, accounting for 6.56% of the whole mitochondrial genome. In particular, we found that <em>ccmFc</em> and <em>rps3</em> contained one intron; <em>cox2</em> and <em>atp1</em> had two introns; <em>nad4</em> contained three introns; <em>nad1</em>, <em>nad2</em>, <em>nad5</em>, and <em>nad7</em> had four introns. Interestingly, the <em>cob</em> gene was devoid of intron and was dispersed in contig 9 and contig 20, which encompassed five genes: <em>cox2</em>, <em>atp1</em>, <em>nad1</em>, <em>nad2</em>, and <em>nad5</em> (Table S2). In addition, there was a large set of 20 tRNAs in the <em>P. cyrtonema</em> mitochondrial genome, among which, <em>trnA-TGC</em>, <em>trnI-GAT</em>, and <em>trnN-GTT</em> participated in protein translation after RNA splicing. In addition, only <em>trnM-CAT</em> had four copies, and the others were single copy (Table <a href="#Tab1" class="usa-link">1</a> and Table S2).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369140_12870_2025_7074_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e230/12369140/b7839e3c5208/12870_2025_7074_Fig1_HTML.jpg" loading="lazy" id="MO1" height="752" width="778" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The assembly result of the mitochondrial genome of <em>P. cyrtonema</em></p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369140_12870_2025_7074_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e230/12369140/be4e4cc785d4/12870_2025_7074_Fig2_HTML.jpg" loading="lazy" id="MO2" height="668" width="778" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Linear graphical display of the genetic maps of 21 contigs in the assembly of the mitochondrial genome of <em>P. cyrtonema</em>. The display most likely does notrepresent the physical structure of the mitogenome in vivo. Contigs names are indicated, e.g., as"contig 1". Contigs lengths are indicated, e.g., as"125,074 bp". The length of the black bars is not proportional to the length of the contigs. Numbers after gene names refer to one or more exons. Potential CDS regions of unknown function are indicated as"ORFs", The map was created with OrganellarGenomeDraw [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] using a linear map stretch factor of 1/4</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Gene organization in the mitochondrial genome of <em>P. cyrtonema</em></p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Group of genes</th>
<th align="left" colspan="1" rowspan="1">Gene name</th>
<th align="left" colspan="1" rowspan="1">Length</th>
<th align="left" colspan="1" rowspan="1">Start codon</th>
<th align="left" colspan="1" rowspan="1">Stop codon</th>
<th align="left" colspan="1" rowspan="1">Amino acid</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="5" colspan="1">ATP synthase</td>
<td align="left" colspan="1" rowspan="1">atp1 (a)</td>
<td align="left" colspan="1" rowspan="1">1530</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TGA</td>
<td align="left" colspan="1" rowspan="1">510</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">atp4</td>
<td align="left" colspan="1" rowspan="1">588</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">196</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">atp6</td>
<td align="left" colspan="1" rowspan="1">921</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">CAA(TAA)</td>
<td align="left" colspan="1" rowspan="1">307</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">atp8</td>
<td align="left" colspan="1" rowspan="1">465</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">155</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">atp9</td>
<td align="left" colspan="1" rowspan="1">225</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">CGA(TGA)</td>
<td align="left" colspan="1" rowspan="1">75</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1">Cytohrome c biogenesis</td>
<td align="left" colspan="1" rowspan="1">ccmB</td>
<td align="left" colspan="1" rowspan="1">621</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TGA</td>
<td align="left" colspan="1" rowspan="1">207</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ccmC</td>
<td align="left" colspan="1" rowspan="1">747</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TGA</td>
<td align="left" colspan="1" rowspan="1">249</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ccmFc (a)</td>
<td align="left" colspan="1" rowspan="1">1353</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">451</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ccmFn</td>
<td align="left" colspan="1" rowspan="1">1788</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAG</td>
<td align="left" colspan="1" rowspan="1">596</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ubichinol cytochrome c reductase</td>
<td align="left" colspan="1" rowspan="1">cob (a)</td>
<td align="left" colspan="1" rowspan="1">1182</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TGA</td>
<td align="left" colspan="1" rowspan="1">394</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Cytochrome c oxidase</td>
<td align="left" colspan="1" rowspan="1">cox1</td>
<td align="left" colspan="1" rowspan="1">1584</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">528</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">cox2 (a)</td>
<td align="left" colspan="1" rowspan="1">783</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">261</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">cox3</td>
<td align="left" colspan="1" rowspan="1">798</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TGA</td>
<td align="left" colspan="1" rowspan="1">266</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Maturases</td>
<td align="left" colspan="1" rowspan="1">matR</td>
<td align="left" colspan="1" rowspan="1">1971</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">657</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Transport membrance protein</td>
<td align="left" colspan="1" rowspan="1">mttB</td>
<td align="left" colspan="1" rowspan="1">828</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAG</td>
<td align="left" colspan="1" rowspan="1">276</td>
</tr>
<tr>
<td align="left" rowspan="9" colspan="1">NADH dehydrogenase</td>
<td align="left" colspan="1" rowspan="1">nad1 (a)</td>
<td align="left" colspan="1" rowspan="1">978</td>
<td align="left" colspan="1" rowspan="1">ACG(ATG)</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">326</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">nad2 (a)</td>
<td align="left" colspan="1" rowspan="1">1467</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">489</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">nad3</td>
<td align="left" colspan="1" rowspan="1">357</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">119</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">nad4 (a)</td>
<td align="left" colspan="1" rowspan="1">1488</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TGA</td>
<td align="left" colspan="1" rowspan="1">496</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">nad4L</td>
<td align="left" colspan="1" rowspan="1">303</td>
<td align="left" colspan="1" rowspan="1">ACG(ATG)</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">101</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">nad5 (a)</td>
<td align="left" colspan="1" rowspan="1">2013</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">671</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">nad6</td>
<td align="left" colspan="1" rowspan="1">780</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">260</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">nad7 (a)</td>
<td align="left" colspan="1" rowspan="1">1251</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAG</td>
<td align="left" colspan="1" rowspan="1">417</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">nad9</td>
<td align="left" colspan="1" rowspan="1">573</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">191</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Ribosomal proteins (LSU)</td>
<td align="left" colspan="1" rowspan="1">rpl16</td>
<td align="left" colspan="1" rowspan="1">411</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">137</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rpl5</td>
<td align="left" colspan="1" rowspan="1">567</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">189</td>
</tr>
<tr>
<td align="left" rowspan="7" colspan="1">Ribosomal proteins (SSU)</td>
<td align="left" colspan="1" rowspan="1">rps1</td>
<td align="left" colspan="1" rowspan="1">609</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">203</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rps12</td>
<td align="left" colspan="1" rowspan="1">378</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TGA</td>
<td align="left" colspan="1" rowspan="1">126</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rps13</td>
<td align="left" colspan="1" rowspan="1">351</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TGA</td>
<td align="left" colspan="1" rowspan="1">117</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rps14</td>
<td align="left" colspan="1" rowspan="1">303</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAG</td>
<td align="left" colspan="1" rowspan="1">101</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rps19</td>
<td align="left" colspan="1" rowspan="1">273</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">91</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rps3</td>
<td align="left" colspan="1" rowspan="1">1647</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TGA</td>
<td align="left" colspan="1" rowspan="1">549</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rps4</td>
<td align="left" colspan="1" rowspan="1">1035</td>
<td align="left" colspan="1" rowspan="1">ATG</td>
<td align="left" colspan="1" rowspan="1">TAA</td>
<td align="left" colspan="1" rowspan="1">345</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Succinate dehydrogenaseRibosomal RNAs</td>
<td align="left" colspan="1" rowspan="1">rrn18</td>
<td align="left" colspan="1" rowspan="1">1786</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rrn26</td>
<td align="left" colspan="1" rowspan="1">3426</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">rrn5</td>
<td align="left" colspan="1" rowspan="1">118</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="20" colspan="1">Transfer RNAs</td>
<td align="left" colspan="1" rowspan="1">trnA-TGC (a)</td>
<td align="left" colspan="1" rowspan="1">65</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnC-GCA</td>
<td align="left" colspan="1" rowspan="1">71</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnD-GTC</td>
<td align="left" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnE-TTC</td>
<td align="left" colspan="1" rowspan="1">72</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnF-GAA</td>
<td align="left" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnH-GTG</td>
<td align="left" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnI-GAT (a)</td>
<td align="left" colspan="1" rowspan="1">72</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnK-TTT</td>
<td align="left" colspan="1" rowspan="1">73</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnM-CAT</td>
<td align="left" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnM-CAT</td>
<td align="left" colspan="1" rowspan="1">73</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnM-CAT</td>
<td align="left" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnM-CAT</td>
<td align="left" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnN-GTT</td>
<td align="left" colspan="1" rowspan="1">72</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnN-GTT (a)</td>
<td align="left" colspan="1" rowspan="1">76</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnP-GGG</td>
<td align="left" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnP-TGG</td>
<td align="left" colspan="1" rowspan="1">75</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnQ-TTG</td>
<td align="left" colspan="1" rowspan="1">72</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnS-GCT</td>
<td align="left" colspan="1" rowspan="1">88</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnW-CCA</td>
<td align="left" colspan="1" rowspan="1">74</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">trnY-GTA</td>
<td align="left" colspan="1" rowspan="1">83</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p21"><p>Gene (a): Refer to the genes containing introns</p></div></div></section></section><section id="Sec12"><h3 class="pmc_sec_title">Repeat sequence authentication</h3>
<p id="Par24">According to sequence structure characteristics, repeat sequences could be divided into simple sequence, tandem, and dispersed repeats [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>–<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. Therefore, three types of the repetitive sequence were analyzed in the mitochondrial genome of <em>P. cyrtonema</em>. The results showed that 194 SSRs were detected and the most common type was tetranucleotide, accounting for 35.6% (<em>n</em> = 69), followed by mononucleotide (25.8%, <em>n</em> = 50), dinucleotide (23.2%, <em>n</em> = 45), trinucleotide (12.4%, <em>n</em> = 69), pentanucleotide (2.6%, <em>n</em> = 5), and hexanucleotide (0.5%, <em>n</em> = 1) (Figs. <a href="#Fig3" class="usa-link">3</a> and <a href="#Fig4" class="usa-link">4</a>). A/T (<em>n</em> = 49) was the most frequent repeat motif, which represented 25.3% of the identified SSRs, secondly for AG/CT (<em>n</em> = 16, 8.2%), GA/TC (<em>n</em> = 16, 8.2%), and AAAG/CTTT (<em>n</em> = 10, 5.2%). In particular, contig1 had the highest number of SSRs among them (Table S3).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369140_12870_2025_7074_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e230/12369140/95ad4974617d/12870_2025_7074_Fig3_HTML.jpg" loading="lazy" id="MO3" height="770" width="770" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution map of repeat sequences in the mitochondrial genome of <em>P. cyrtonema</em>. Note: The outermost circle represents simple sequence repeats, followed by tandem repeat, and the innermost line represents dispersed repeats</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e230/12369140/43bfeb23575f/12870_2025_7074_Fig4_HTML.jpg" loading="lazy" id="MO4" height="507" width="598" alt="Fig. 4"></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of SSRs in the mitochondrial genome of <em>P. cyrtonema</em></p></figcaption></figure><p id="Par25">A total of 24 tandem repeats were obtained in the <em>P. cyrtonema</em> mitochondrial genome, with a length range of 14–43 bp and a matching degree ranging from 69 to 100% (Fig. <a href="#Fig3" class="usa-link">3</a> and Table S4). Five sequences (20.8%) had a matching degree of 100%. The copy number of the tandem repeats exhibited a range of 1.9 to 4.4, indicating that a significant proportion of the tandem repeats were incomplete copies. Similar to the results of SSRs, contig1 also contained the largest number of tandem repeats (<em>n</em> = 8, 33.3%).</p>
<p id="Par26">Dispersed repeats are a kind of scattered repeats distributed in the genome, and have four different patterns of repeats, including forward repeats, reverse repeats, complement repeats and palindromic repeat. A total of 294 dispersed repeats with a mean length of approximately 64 bp were detected in the <em>P. cyrtonema</em> mitochondrial genome, including 153 palindromic repeats and 141 forward repeats, whereas no reverse or complement repeats were identified (Fig. <a href="#Fig3" class="usa-link">3</a> and Table S5). The majority of dispersed repeats ranged from 30 to 69 bp in length. In particular, the number of forward and palindromic repeats ranging from 30 to 39 bp was identical (forward: 33, palindromic: 33; 22.4%) (Fig. <a href="#Fig5" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e230/12369140/14778ee13b98/12870_2025_7074_Fig5_HTML.jpg" loading="lazy" id="MO5" height="485" width="596" alt="Fig. 5"></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Allocation of the lengths of dispersed repeats in the mitochondrial genome of <em>P. cyrtonema</em>. P, palindromic repeat sequences; F, forward repeat sequences; R, Reverse repeat sequences; C, complement repeat sequences</p></figcaption></figure></section><section id="Sec13"><h3 class="pmc_sec_title">RNA editing sites prediction</h3>
<p id="Par27">A total of 599 effective RNA editing sites were identified in 56 PCGs of the <em>P. cyrtonema</em> mitochondrial genome (Fig. <a href="#Fig6" class="usa-link">6</a>). It was worth noting that <em>nad4</em> had the greatest number of RNA editing sites (<em>n</em> = 53), while <em>atp8</em> and <em>rps1</em> had the fewest (n = 2). Among the RNA editing sites, the properties of 42.73% (<em>n</em> = 256) amino acids remained unchanged, 47.58% (<em>n</em> = 285) of the hydrophilic amino acids turned into hydrophobic amino acids, 9.18% (<em>n</em> = 55) of the total sites changed from hydrophobic to hydrophilic amino acids, and 0.50% (<em>n</em> = 3) of amino acid coding sites resulted in stop codons, located in the <em>atp9</em> and <em>ccmFc</em> genes (CGA-TGA), and <em>atp6</em> gene (CAA-TAA), respectively (Table <a href="#Tab2" class="usa-link">2</a>). RNA-editing machinery mainly targeted the second base of codon (<em>n</em> = 378, 63%), followed by the first base of codon (<em>n</em> = 201, 34%). In addition, there were two particular editing cases in which both the first and second bases of the triplet codon were edited, resulting in the conversion of Pro (CCC, CCT) to Phe (TTC, TTT) (<em>n</em> = 20, 3%). Interestingly, the amino acids generated from editing codons were most frequently converted to leucine (<em>n</em> = 248, 41%) after RNA editing, whereas the least were converted to methionine (<em>n</em> = 8, 1%).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369140_12870_2025_7074_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e230/12369140/4dfacd18ba00/12870_2025_7074_Fig6_HTML.jpg" loading="lazy" id="MO6" height="454" width="708" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prediction of RNA editing sites in protein-coding genes in the mitochondrial genome of <em>P. cyrtonema</em>. The horizontal axis refers to the type of gene and the vertical axis refers to the number of RNA-editing sites</p></figcaption></figure><section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Characteristics of hydrophilic and hydrophobic changes in RNA editing sites in the mitochondrial genome of <em>P. cyrtonema</em></p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Type</th>
<th align="left" colspan="1" rowspan="1">RNA-editing</th>
<th align="left" colspan="1" rowspan="1">Number</th>
<th align="left" colspan="1" rowspan="1">Percentage</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="5" colspan="1">hydrophilic-hydrophilic</td>
<td align="left" colspan="1" rowspan="1">CAC (H) = &gt; TAC (Y)</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CAT (H) = &gt; TAT (Y)</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CGC (R) = &gt; TGC (C)</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CGT (R) = &gt; TGT (C)</td>
<td align="left" colspan="1" rowspan="1">34</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">total</td>
<td align="left" colspan="1" rowspan="1">77</td>
<td align="left" colspan="1" rowspan="1">12.85%</td>
</tr>
<tr>
<td align="left" rowspan="10" colspan="1">hydrophilic-hydrophobic</td>
<td align="left" colspan="1" rowspan="1">ACA (T) = &gt; ATA (I)</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ACC (T) = &gt; ATC (I)</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ACG (T) = &gt; ATG (M)</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ACT (T) = &gt; ATT (I)</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CGG (R) = &gt; TGG (W)</td>
<td align="left" colspan="1" rowspan="1">39</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TCA (S) = &gt; TTA (L)</td>
<td align="left" colspan="1" rowspan="1">75</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TCC (S) = &gt; TTC (F)</td>
<td align="left" colspan="1" rowspan="1">43</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TCG (S) = &gt; TTG (L)</td>
<td align="left" colspan="1" rowspan="1">50</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TCT (S) = &gt; TTT (F)</td>
<td align="left" colspan="1" rowspan="1">58</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">total</td>
<td align="left" colspan="1" rowspan="1">285</td>
<td align="left" colspan="1" rowspan="1">47.58%</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">hydrophilic-stop</td>
<td align="left" colspan="1" rowspan="1">CAA (Q) = &gt; TAA (X)</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CGA (R) = &gt; TGA (X)</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">total</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">0.50%</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">hydrophobic-hydrophilic</td>
<td align="left" colspan="1" rowspan="1">CCA (P) = &gt; TCA (S)</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CCC (P) = &gt; TCC (S)</td>
<td align="left" colspan="1" rowspan="1">17</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CCG (P) = &gt; TCG (S)</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CCT (P) = &gt; TCT (S)</td>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">total</td>
<td align="left" colspan="1" rowspan="1">55</td>
<td align="left" colspan="1" rowspan="1">9.18%</td>
</tr>
<tr>
<td align="left" rowspan="12" colspan="1">hydrophobic-hydrophobic</td>
<td align="left" colspan="1" rowspan="1">CCA (P) = &gt; CTA (L)</td>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CCC (P) = &gt; CTC (L)</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CCC (P) = &gt; TTC (F)</td>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CCG (P) = &gt; CTG (L)</td>
<td align="left" colspan="1" rowspan="1">34</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CCT (P) = &gt; CTT (L)</td>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CCT (P) = &gt; TTT (F)</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CTC (L) = &gt; TTC (F)</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CTT (L) = &gt; TTT (F)</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GCC (A) = &gt; GTC (V)</td>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GCG (A) = &gt; GTG (V)</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GCT (A) = &gt; GTT (V)</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">total</td>
<td align="left" colspan="1" rowspan="1">179</td>
<td align="left" colspan="1" rowspan="1">29.88%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">All</td>
<td align="left" colspan="1" rowspan="1">599</td>
<td align="left" colspan="1" rowspan="1">100%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec14"><h3 class="pmc_sec_title">Chloroplast-derived mitochondrial genome sequences</h3>
<p id="Par28">The mitochondrial and chloroplast genomes of <em>P. cyrtonema</em> were compared to provide evidence of sequence transfer (Fig. <a href="#Fig7" class="usa-link">7</a>). Overall, the mitochondrial genome (664,991 bp) of <em>P. cyrtonema</em> was much bigger than the chloroplast genome (155,205 bp) [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. There were 29 transferred fragments between them with a total length of 15,849 bp, covering 2.4% of the mitochondrial genome (Table S6). Interestingly, these transfer fragments mainly located the tRNA (<em>n</em> = 8) and rRNA (<em>n</em> = 4) genes. Among these homologous fragments, only three incomplete tRNA genes (<em>trnN-GTT</em>, <em>trnM-CAT</em>, and <em>trnH-GTG</em>) also appeared in the chloroplast genome synchronously (<em>trnN-GUU</em>, <em>trnM-CAU</em>, and <em>trnH-GUG</em>). Additionally, two chloroplast gene fragments, specifically <em>rrn16</em> and <em>rrn23</em>, were homologous to <em>rrn18</em> and <em>rrn26</em> derived from mitochondrial genome (Table S6).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369140_12870_2025_7074_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e230/12369140/fbbdcfc891d0/12870_2025_7074_Fig7_HTML.jpg" loading="lazy" id="MO7" height="797" width="778" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene transfer events between the chloroplast and mitochondrial genome in the <em>P. cyrtonema</em></p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">Codon usage analysis of PCGs</h3>
<p id="Par29">All the PCGs contained 10,056 codons in the <em>P. cyrtonema</em> mitochondrial genome (Table S7). The most frequent amino acids were Leu (10.3%), Ser (9.6%), and Ile (7.7%), while the expression frequency of other amino acids was more than 100. Therefore, all 64 types of codons were evenly used in the <em>P. cyrtonema</em> mitochondrial genome, and half of them had RSCU values of more than 1.0. Only Trp with 1.0 RSCU value showed no bias in codon usage (Fig. <a href="#Fig8" class="usa-link">8</a>). Met (AUG) with RSCU value of 3, was the only initiator. Similarly, as for stop codons, UAA was the most frequently employed, whose RSCU values = 1.72. Because of the degeneracy of codons, there were different preferences for the codons used in the mitochondrial genome of <em>P. cyrtonema</em>. For example, among the four codons (GCU, GCC, GCA, and GCG) encoded Ala, GCU (RSCU = 1.52) was the most frequently used codon.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369140_12870_2025_7074_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e230/12369140/f0893ed94eb5/12870_2025_7074_Fig8_HTML.jpg" loading="lazy" id="MO8" height="481" width="778" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Relative synonymous codon usage (RSCU) in PCGs of <em>P. cyrtonema</em> mitochondrial genome. Codon families are shown below the graph</p></figcaption></figure></section><section id="Sec16"><h3 class="pmc_sec_title">Nucleotide diversity</h3>
<p id="Par30">To further investigate the selection variation of genes under different selective pressures in the mitochondrial genome of <em>P. cyrtonema</em>, nucleotide diversity was conducted on PCGs and rRNAs. The results showed that Pi values with an average of 0.0328 ranged from 0.0079 (<em>nad7</em>) to 0.1642 (<em>atp8</em>) with a huge gap of 21 times (Fig. <a href="#Fig9" class="usa-link">9</a>). Notably, nearly half of the PCGs (<em>n</em> = 16) and all rRNAs (<em>n</em> = 3) showed high rate of nucleotide diversity (&gt; 0.03). Except for the gene <em>atp8</em>, all PCGs and rRNAs had stable fluctuations in the range of variation. From the perspective of functional enrichment, the ATP synthase function of PCGs exhibited considerable variability, while the others demonstrated lower performance.</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369140_12870_2025_7074_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e230/12369140/41f5af2a3484/12870_2025_7074_Fig9_HTML.jpg" loading="lazy" id="MO9" height="894" width="708" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Nucleotide diversity (Pi) in the mitochondrial genome of <em>P. cyrtonema.</em> The horizontal axis refers to the type of gene and the vertical axis refers to the nucleotide diversity value (Pi) of gene</p></figcaption></figure></section><section id="Sec17"><h3 class="pmc_sec_title">Phylogenetic inference and substitution rates of PCGs</h3>
<p id="Par31">To investigate the evolutionary status of the mitochondrial genome of <em>P. cyrtonema</em>, a phylogenetic tree was constructed on <em>P. cyrtonema</em>, together with 32 other species, using the maximum likelihood (ML) method (Table S1). The results showed that taxa from 22 families in the phylogenetic tree were well clustered (Fig. <a href="#Fig10" class="usa-link">10</a>). In addition, Asparagaceae family clustered together and <em>P. cyrtonema</em> had the closest genetic relationship with <em>Polygonatum sibiricum</em>.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369140_12870_2025_7074_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e230/12369140/8256e5b3e227/12870_2025_7074_Fig10_HTML.jpg" loading="lazy" id="MO10" height="741" width="778" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylogenetic tree inferred based on PCGs of 33 typical plant mitochondrial genomes. Numbers on each node were bootstrap support values. The red branch indicates <em>P. cyrtonema</em>, and the right side shows the family to which the species belongs. <em>Ginkgo biloba</em> was chosen as the outgroup</p></figcaption></figure><p id="Par32">In order to estimate the selection pressure of <em>P. cyrtonema</em> mitochondrial PCGs during the evolution process, all of them were employed to compute the Ka/Ks ratio among the mitochondrial genomes of <em>Phoenix dactylifera</em>, <em>Asparagus officinalis</em>, <em>Chlorophytum comosum</em>, <em>Allium cepa, Crocus sativus,</em> and <em>Epigeneium amplum</em>. From the results, with the exception of the <em>rpl2</em> gene, all Ka/Ks ratios of PCGs were less than 1.0 (Fig. <a href="#Fig11" class="usa-link">11</a>).</p>
<figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369140_12870_2025_7074_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e230/12369140/ec54af8210d1/12870_2025_7074_Fig11_HTML.jpg" loading="lazy" id="MO11" height="380" width="778" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Ka/Ks analysis in the mitochondrial genome of <em>P. cyrtonema</em></p></figcaption></figure></section></section><section id="Sec18"><h2 class="pmc_sec_title">Discussion</h2>
<section id="Sec19"><h3 class="pmc_sec_title">Structural features of the mitochondrial genome in <em>P. cyrtonema</em>
</h3>
<p id="Par33">Most plant mitochondrial genomes were typically assembled and presented as circular maps, as shown in studies on <em>Asparagus officinalis</em> [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], <em>Polygonatum kingianum</em>, and <em>Polygonatum sibiricum</em> [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>], indicating that circular genome-sized molecules were the primary form of plant mitochondrial DNA [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. However, the mitochondrial genome of <em>P. cyrtonema</em> constructed in this study was a complex non-circular form, indicating that the true structure of mitochondria is characterized by complexity and dynamics, consistent with findings from prior investigations in <em>Quercus acutissima</em>, <em>Indigofera amblyantha</em>, and <em>Abies alba</em> [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>–<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. It was reported that the size of the plant mitochondrial genome was incredibly diverse ranging from 200 kb (<em>Brassica napus</em>) to over 10 Mb (<em>Larix sibirica</em>) [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>, <a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. Overall, the published mitochondrial genome sizes of the Asparagaceae family were not prominent (&lt; 1 Mb) (Table S1), indicating relatively conservative evolutionary dynamics. Interestingly, the mitochondrial genome sizes of <em>P. cyrtonema</em> were strikingly similar to other <em>Polygonatum</em> species (<em>P. kingianum</em> and <em>P. sibiricum</em>), whereas their estimated nuclear genome sizes differed significantly by flow cytometry [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>], suggesting that interspecific variation within <em>Polygonatum</em> Mill. resides predominantly in nuclear genes.</p>
<p id="Par34">Although the plant mitochondrial genomes possessed longer lengths and more complex variations relative to their animal mitochondrial counterparts, their PCGs were highly conserved [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>–<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. According to previous studies, 41 PCGs were inferred to exist in the ancestral mitogenome of angiosperms [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. In contrast, 33 PCGs were found in the mitochondrial genome of <em>P. cyrtonema</em>. It seemed that some of them were lost or transfered during evolution [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>, <a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. Through comparative analysis mitochondrial PCGs in <em>P. cyrtonema</em> and 32 additional species, several notable patterns were identified (Table S1). The <em>rps2</em>, <em>rps11</em>, and <em>rpl10</em> genes were lost during the early evolution in most eukaryotic plants, which was predicted by Adams and Kubo [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>, <a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. Asparagaceae species exhibited a higher frequency of <em>rpl2</em> and <em>sdh3</em> genes transfer events from mitochondrial to nuclear genomes throughout evolutionary history [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>, <a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. In <em>Polygonatum</em> Mill., the missing <em>rps7</em> gene was possibly transferred and integrated into the nuclear genomes [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. Additionally, given that <em>P. cyrtonema</em> exhibited the lowest number of PCGs among the published <em>Polygonatum</em> species, it could deduce that <em>P. cyrtonema</em> had drastic genetic changes during evolution from a common ancestor to better adapt to their environment.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Variation and diversity in the mitochondrial genome of <em>P. cyrtonema</em>
</h3>
<p id="Par35">Abundant repetitive sequences played an important role in plant mitochondrial genome evolution and significantly promoted gene recombination within mitochondrial genome [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>, <a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. In this study, a number of 512 repeat sequences were identified in the mitochondrial genome of <em>P. cyrtonema</em>, which not only furnished a substantial number of reference sites for genetic evolution, but also provide a foundation for developing molecular markers for germplasm identification within <em>Polygonatum</em> Mill. [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>–<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. Furthermore, ultralong repetitive sequences were of particular interest, as they not only leaded to isomerization, but were more likely to interact with structural changes in genes to promote evolution [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>, <a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>, <a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. Within the published mitochondrial genome of the Asparagaceae family, ultralong repetitive sequence (&gt; 1000 bp) resulted in the duplication of genes, such as <em>trnM-CAT</em>. Therefore, ultralong repetitive sequences might not only drive the duplication of the <em>trnM-CAT</em> gene into four copies, but also contribute to the structural complexity of the <em>P. cyrtonema</em> mitochondrial genome.</p>
<p id="Par36">RNA editing, a post-transcriptional process in plant mitochondrial and chloroplast genomes, modified nucleotide sequences in mRNA level, which enabled more efficient protein folding [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>, <a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. All of 599 RNA editing sites were found in the mitochondrial genome of <em>P. cyrtonema</em>, which was more than <em>Asparagus officinalis</em> (510) [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] and less than <em>Polygonatum sibiricum</em> (609) [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. The main type of editing change was a transition from C to T in <em>P. cyrtonema</em> RNA editing sites, which was also a general type of RNA editing change in other plants [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. At the same time, the RNA editing of <em>P. cyrtonema</em> mainly happened at the first and second base positions, which was also reported in most plants [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>]. In particular, <em>P. cyrtonema</em> RNA editing greatly increased the proportion of hydrophobic amino acids, which was similar to <em>Polygonatum sibiricum</em> [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]<em>.</em> The increasing proportion of hydrophobic amino acids contributed to the stability of protein structure [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. Furthermore, plant mitochondria mitigated stress-induced damage by modulating RNA editing events in specific PCGs under environmental perturbations [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>–<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]. By combining RNA editing site prediction results with previous studies in <em>Polygonatum sibiricum</em>, cytochrome c biogenesis and NADH dehydrogenase-related PCGs were preferentially edited [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. It could hypothesize that they experienced comparable environmental pressures throughout evolutionary history.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Evolutionary trends of mitochondrial genes in <em>P. cyrtonema</em>
</h3>
<p id="Par37">Codon usage preference might result from species mutation, genetic drift, and translational selection to some extent [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>–<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>]. In the mitochondrial genome of <em>P. cyrtonema</em>, the most preference codon was TTT (phenylalanine), followed by ATT (isoleucine), TTC (phenylalanine), GAA (glutamic acid), ATG (methionine), which was also described in <em>Polygonatum sibiricum</em> [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Intriguingly, like most resolved plant mitochondria, <em>P. cyrtonema</em> exhibited a general preference for Leu and Ser in amino acid utilization, while demonstrating the lowest usage of Trp and Cys. Furthermore, the number of codons ending with A/T outnumbered those ending with C/G (Table S7), which was reported in most plant species [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>, <a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>], indicating that the mitochondrial genes of <em>Polygonatum</em> Mill. had similar preferences for base changes.</p>
<p id="Par38">The construction of a phylogenetic evolutionary tree not only determined the evolutionary position of the species but also elucidated the intrageneric and interspecific phylogeny of the family to which the species belongs [<a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>, <a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>]. In this study, <em>P. cyrtonema</em> was most closely related to <em>Polygonatum sibiricum</em>, followed by <em>Polygonatum kingianum</em>. The similar CG content among <em>Polygonatum</em> Mill. corroborated their distribution in phylogenetic trees, which was consistent with the results of the evolutionary tree based on chloroplast genomes [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. This indicated that classification based on PCGs in the mitochondrial genome was effective for plant classification and confirmed the evolutionary position of <em>P. cyrtonema</em> within <em>Polygonatum</em> Mill.. Additionally, it also provided other references for exploring phylogenetic relationships in Asparagaceae. In conclusion, although significant progress was made in phylogenetic and evolutionary analyses of the mitochondrial genome of <em>P. cyrtonema</em>, more detailed biogeographic studies of related species will be necessary to elucidate their evolutionary history further.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par39">In this study, the complete <em>P. cyrtonema</em> mitochondrial genome was sequenced and assembled based on a combination of long-read Nanopore and Illumina sequencing technologies, which was a complex non-circular form with a length of 664,991 bp, including 56 genes (33 protein-coding genes, 20 tRNA genes, and 3 rRNA genes). Subsequently, the diversity of DNA fragments in the mitochondrial genome of <em>P. cyrtonema</em> was elucidated by analyzing repeat sequence identification, RNA editing site prediction, and gene transfer between chloroplasts and mitochondria. Finally, the genetic status and evolutionary trend of the mitochondrial genome of <em>P. cyrtonema</em> were addressed due to codon bias, synonym and non-synonym substitution rate, nucleotide diversity, and phylogenetic tree construction. In conclusion, this study enhances our understanding of the phylogeny and genetic evolution of <em>P. cyrtonema</em> through its mitochondrial genome, fills a critical gap in its genomic research, and provides a valuable theoretical foundation for future studies, as well as the development and utilization of this important species.</p></section><section id="Sec23"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12369140/bin/12870_2025_7074_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (41.4KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This research was partially supported by"Pioneer"and"Leading Goose"R&amp;D Program of Zhejiang Province (grant no. 2023C02017).</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>CDS</dt>
<dd><p id="Par4">Coding DNA Sequence</p></dd>
<dt>rRNA</dt>
<dd><p id="Par5">Ribosomal RNA</p></dd>
<dt>tRNA</dt>
<dd><p id="Par6">Transfer RNA</p></dd>
<dt>PCGs</dt>
<dd><p id="Par7">Protein-coding genes</p></dd>
<dt>SSRs</dt>
<dd><p id="Par8">Simple sequence repeats</p></dd>
<dt>RSCU</dt>
<dd><p id="Par9">Relative synonymous codon usage ratios</p></dd>
<dt>ML</dt>
<dd><p id="Par10">Maximum Likelihood</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>QJJ and DKW conceived and designed the experiment. HXL prepared figures and tables and wrote the manuscript. QJJ and HXL revised the manuscript. FXR and GCC collected the plant materials and conducted the experiment. QJJ and DKW conduct funding acquisition and project management. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>The Key Research and Development Projects of “Vanguard” and “Leading Goose” in Zhejiang Province (2023C02017).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The calibrated sequencing data for the Illumina and Oxford Nanopore PromethION platforms have been deposited in NCBI (<a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/</a>) with accession numbers PRJNA1195727, SAMN45227193, SAMN45227194, SRR31706100, and SRR31706101. The mitochondrial genome of P. cyrtonema has been deposited in GeneBank (<a href="https://www.ncbi.nlm.nih.gov/genbank/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/genbank/</a>) with accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV007883:PV007903%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV007883-PV007903</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par40">The authors declare that all methods were carried out in accordance with relevant guidelines and regulations. These plant materials do not include any wild species at risk of extinction. No specific permits are required for sample collection in this study. We comply with relevant institutional, national, and international guidelines and legislation for plant study.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par41">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par42">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Si J. Zhu Y. Polygonati rhizoma—a new high-quality crop with great potential and not occupying farmland. Sci Sin Vitae. 2021;51(11):1477–84.</cite>
</li>
<li id="CR2">
<span class="label">2.</span><cite>Maa CD, Chang H, Yang YC, Wang EH, Zhan ZL. Herbal textual research on Polygonati Rhizoma in famous classical formulas. Chin J Exp Tradit Med Formulae. 2022;28(10):193–206.</cite> [<a href="https://scholar.google.com/scholar_lookup?Maa%20CD,%20Chang%20H,%20Yang%20YC,%20Wang%20EH,%20Zhan%20ZL.%20Herbal%20textual%20research%20on%20Polygonati%20Rhizoma%20in%20famous%20classical%20formulas.%20Chin%20J%20Exp%20Tradit%20Med%20Formulae.%202022;28(10):193%E2%80%93206." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Commission CP, Chinese Pharmacopoeia Commission. Chinese pharmacopoeia 2020 edition. Beijing, China: China Medical Science and Technology Press; 2020.</cite> [<a href="https://scholar.google.com/scholar_lookup?Commission%20CP,%20Chinese%20Pharmacopoeia%20Commission.%20Chinese%20pharmacopoeia%202020%20edition.%20Beijing,%20China:%20China%20Medical%20Science%20and%20Technology%20Press;%202020." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Zhang J, Wang YZ, Yang WZ, Yang MQ, Zhang JY. Research progress in chemical constituents in plants of Polygonatum and their pharmacological effects. Zhongguo Zhong Yao Za Zhi. 2019;44(10):1989–2008.
</cite> [<a href="https://doi.org/10.19540/j.cnki.cjcmm.20190222.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31355552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20J,%20Wang%20YZ,%20Yang%20WZ,%20Yang%20MQ,%20Zhang%20JY.%20Research%20progress%20in%20chemical%20constituents%20in%20plants%20of%20Polygonatum%20and%20their%20pharmacological%20effects.%20Zhongguo%20Zhong%20Yao%20Za%20Zhi.%202019;44(10):1989%E2%80%932008." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Chen L, Hu BX, Dai H, Zhang X, Xia CA, Zhang J. Characterizing microbial diversity and community composition of groundwater in a salt-freshwater transition zone. Sci Total Environ. 2019;678:574–84.
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2019.05.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31078848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20L,%20Hu%20BX,%20Dai%20H,%20Zhang%20X,%20Xia%20CA,%20Zhang%20J.%20Characterizing%20microbial%20diversity%20and%20community%20composition%20of%20groundwater%20in%20a%20salt-freshwater%20transition%20zone.%20Sci%20Total%20Environ.%202019;678:574%E2%80%9384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Shen WD, Li XY, Deng YY, Zha XQ, Pan LH, Li QM, Luo JP. <em>Polygonatum cyrtonema</em> Hua polysaccharide exhibits anti-fatigue activity via regulating osteocalcin signaling. Int J Biol Macromol. 2021;175:235–41.
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2021.01.200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33548311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen%20WD,%20Li%20XY,%20Deng%20YY,%20Zha%20XQ,%20Pan%20LH,%20Li%20QM,%20Luo%20JP.%20Polygonatum%20cyrtonema%20Hua%20polysaccharide%20exhibits%20anti-fatigue%20activity%20via%20regulating%20osteocalcin%20signaling.%20Int%20J%20Biol%20Macromol.%202021;175:235%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Li XY, Jiang CL, Zheng C, Hong CZ, Pan LH, Li QM, Luo JP, Zha XQ. <em>Polygonatum cyrtonema</em> Hua polysaccharide alleviates fatigue by modulating osteocalcin-mediated crosstalk between bones and muscles. J Agric Food Chem. 2023;71(16):6468–79.
</cite> [<a href="https://doi.org/10.1021/acs.jafc.2c08192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37043685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20XY,%20Jiang%20CL,%20Zheng%20C,%20Hong%20CZ,%20Pan%20LH,%20Li%20QM,%20Luo%20JP,%20Zha%20XQ.%20Polygonatum%20cyrtonema%20Hua%20polysaccharide%20alleviates%20fatigue%20by%20modulating%20osteocalcin-mediated%20crosstalk%20between%20bones%20and%20muscles.%20J%20Agric%20Food%20Chem.%202023;71(16):6468%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Li J, Wang X, Zhou R, Cheng F, Tang X, Lao J, Xu L, He W, Wan D, Zeng H, et al. <em>Polygonatum cyrtonema</em> Hua polysaccharides protect BV2 microglia relief oxidative stress and ferroptosis by regulating NRF2/HO-1 pathway. Molecules. 2022. 10.3390/molecules27207088.
</cite> [<a href="https://doi.org/10.3390/molecules27207088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9610736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36296678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20J,%20Wang%20X,%20Zhou%20R,%20Cheng%20F,%20Tang%20X,%20Lao%20J,%20Xu%20L,%20He%20W,%20Wan%20D,%20Zeng%20H,%20et%20al.%20Polygonatum%20cyrtonema%20Hua%20polysaccharides%20protect%20BV2%20microglia%20relief%20oxidative%20stress%20and%20ferroptosis%20by%20regulating%20NRF2/HO-1%20pathway.%20Molecules.%202022.%2010.3390/molecules27207088." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Liu D, Hou T, Geng C, Song L, Hou X, Chen Y, Wang F, Wang W, Han B, Gao L. Liposomes enhance the immunological activity of <em>Polygonatum cyrtonema</em> Hua polysaccharides. J Pharm Sci. 2024;113(6):1572–9.
</cite> [<a href="https://doi.org/10.1016/j.xphs.2024.01.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38237668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20D,%20Hou%20T,%20Geng%20C,%20Song%20L,%20Hou%20X,%20Chen%20Y,%20Wang%20F,%20Wang%20W,%20Han%20B,%20Gao%20L.%20Liposomes%20enhance%20the%20immunological%20activity%20of%20Polygonatum%20cyrtonema%20Hua%20polysaccharides.%20J%20Pharm%20Sci.%202024;113(6):1572%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Wang X, Wang X, Gao F, Yang S, Zhen Y, Wang X, Zhu G. Polysaccharides from <em>Polygonatum cyrtonema</em> Hua prevent depression-like behaviors in mice with chronic unpredictable mild stress through refining gut microbiota-lipopolysaccharide-paraventricular nucleus signal axis. Heliyon. 2024;10(19): e38554.
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2024.e38554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11470423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39398073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20X,%20Wang%20X,%20Gao%20F,%20Yang%20S,%20Zhen%20Y,%20Wang%20X,%20Zhu%20G.%20Polysaccharides%20from%20Polygonatum%20cyrtonema%20Hua%20prevent%20depression-like%20behaviors%20in%20mice%20with%20chronic%20unpredictable%20mild%20stress%20through%20refining%20gut%20microbiota-lipopolysaccharide-paraventricular%20nucleus%20signal%20axis.%20Heliyon.%202024;10(19):%20e38554." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Li L, Thakur K, Cao YY, Liao BY, Zhang JG, Wei ZJ. Anticancerous potential of polysaccharides sequentially extracted from <em>Polygonatum cyrtonema</em> Hua in human cervical cancer Hela cells. Int J Biol Macromol. 2020;148:843–50.
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2020.01.223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31982521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20L,%20Thakur%20K,%20Cao%20YY,%20Liao%20BY,%20Zhang%20JG,%20Wei%20ZJ.%20Anticancerous%20potential%20of%20polysaccharides%20sequentially%20extracted%20from%20Polygonatum%20cyrtonema%20Hua%20in%20human%20cervical%20cancer%20Hela%20cells.%20Int%20J%20Biol%20Macromol.%202020;148:843%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Daniell H, Lin CS, Yu M, Chang WJ. Chloroplast genomes: diversity, evolution, and applications in genetic engineering. Genome Biol. 2016;17(1):134.
</cite> [<a href="https://doi.org/10.1186/s13059-016-1004-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4918201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27339192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Daniell%20H,%20Lin%20CS,%20Yu%20M,%20Chang%20WJ.%20Chloroplast%20genomes:%20diversity,%20evolution,%20and%20applications%20in%20genetic%20engineering.%20Genome%20Biol.%202016;17(1):134." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Møller IM, Rasmusson AG, Van Aken O. Plant mitochondria - past, present and future. Plant J. 2021;108(4):912–59.
</cite> [<a href="https://doi.org/10.1111/tpj.15495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34528296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?M%C3%B8ller%20IM,%20Rasmusson%20AG,%20Van%20Aken%20O.%20Plant%20mitochondria%20-%20past,%20present%20and%20future.%20Plant%20J.%202021;108(4):912%E2%80%9359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Liberatore KL, Dukowic-Schulze S, Miller ME, Chen C, Kianian SF. The role of mitochondria in plant development and stress tolerance. Free Radic Biol Med. 2016;100:238–56.
</cite> [<a href="https://doi.org/10.1016/j.freeradbiomed.2016.03.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27036362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liberatore%20KL,%20Dukowic-Schulze%20S,%20Miller%20ME,%20Chen%20C,%20Kianian%20SF.%20The%20role%20of%20mitochondria%20in%20plant%20development%20and%20stress%20tolerance.%20Free%20Radic%20Biol%20Med.%202016;100:238%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Guo L, Wang X, Wang R, Li P. Characterization and comparative analysis of chloroplast genomes of medicinal herb <em>Scrophularia ningpoensis</em> and its common adulterants (Scrophulariaceae). Int J Mol Sci. 2023. 10.3390/ijms241210034.
</cite> [<a href="https://doi.org/10.3390/ijms241210034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10298345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37373180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20L,%20Wang%20X,%20Wang%20R,%20Li%20P.%20Characterization%20and%20comparative%20analysis%20of%20chloroplast%20genomes%20of%20medicinal%20herb%20Scrophularia%20ningpoensis%20and%20its%20common%20adulterants%20(Scrophulariaceae).%20Int%20J%20Mol%20Sci.%202023.%2010.3390/ijms241210034." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Yang LH, Feng YQ, Ding L, Tong LM, Qi ZC, Yan XL. The complete chloroplast genome sequence of <em>Elatostema stewardii</em> Merr. (urticaceae). Mitochondr DNA B Resources. 2022;7(8):1494–6.</cite> [<a href="https://doi.org/10.1080/23802359.2022.2108348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9397455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36016692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20LH,%20Feng%20YQ,%20Ding%20L,%20Tong%20LM,%20Qi%20ZC,%20Yan%20XL.%20The%20complete%20chloroplast%20genome%20sequence%20of%20Elatostema%20stewardii%20Merr.%20(urticaceae).%20Mitochondr%20DNA%20B%20Resources.%202022;7(8):1494%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Xu X, Yao X, Zhang C, Xia P. Characterization of the complete chloroplast genome sequence of <em>Cardamine lyrata</em> Bunge(Brassicaceae). Mitochondr DNA B DNA Resour. 2022;7(6):936–7.</cite> [<a href="https://doi.org/10.1080/23802359.2022.2079106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9176338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35692644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20X,%20Yao%20X,%20Zhang%20C,%20Xia%20P.%20Characterization%20of%20the%20complete%20chloroplast%20genome%20sequence%20of%20Cardamine%20lyrata%20Bunge(Brassicaceae).%20Mitochondr%20DNA%20B%20DNA%20Resour.%202022;7(6):936%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Peng L, Xiao H, Xu Y, Huang Z, Yang X, Lv C, Huang L, Hu J. The pentatricopeptide repeat protein PPR767 modulates plant architecture and drought resistance in rice. Plant Physiol. 2025;7: kiaf187.</cite> [<a href="https://doi.org/10.1093/plphys/kiaf187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40331370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peng%20L,%20Xiao%20H,%20Xu%20Y,%20Huang%20Z,%20Yang%20X,%20Lv%20C,%20Huang%20L,%20Hu%20J.%20The%20pentatricopeptide%20repeat%20protein%20PPR767%20modulates%20plant%20architecture%20and%20drought%20resistance%20in%20rice.%20Plant%20Physiol.%202025;7:%20kiaf187." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Li J, Zhang L, Li C, Chen W, Wang T, Tan L, Qiu Y, Song S, Li B, Li L. The pentatricopeptide repeat protein OsPPR674 regulates rice growth and drought sensitivity by modulating RNA editing of the mitochondrial transcript ccmC. Int J Mol Sci. 2025;26(6): 2646.
</cite> [<a href="https://doi.org/10.3390/ijms26062646" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11941812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40141287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20J,%20Zhang%20L,%20Li%20C,%20Chen%20W,%20Wang%20T,%20Tan%20L,%20Qiu%20Y,%20Song%20S,%20Li%20B,%20Li%20L.%20The%20pentatricopeptide%20repeat%20protein%20OsPPR674%20regulates%20rice%20growth%20and%20drought%20sensitivity%20by%20modulating%20RNA%20editing%20of%20the%20mitochondrial%20transcript%20ccmC.%20Int%20J%20Mol%20Sci.%202025;26(6):%202646." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Sloan DB. One ring to rule them all? Genome sequencing provides new insights into the “master circle” model of plant mitochondrial DNA structure. New Phytol. 2013;200(4):978–85.
</cite> [<a href="https://doi.org/10.1111/nph.12395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24712049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sloan%20DB.%20One%20ring%20to%20rule%20them%20all?%20Genome%20sequencing%20provides%20new%20insights%20into%20the%20%E2%80%9Cmaster%20circle%E2%80%9D%20model%20of%20plant%20mitochondrial%20DNA%20structure.%20New%20Phytol.%202013;200(4):978%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Chen L, Dong X, Huang H, Xu H, Rono PC, Cai X, Hu G. Assembly and comparative analysis of the initial complete mitochondrial genome of <em>Primulina hunanensis</em> (Gesneriaceae): a cave-dwelling endangered plant. BMC Genomics. 2024;25(1):322.
</cite> [<a href="https://doi.org/10.1186/s12864-024-10247-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10983754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38561677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20L,%20Dong%20X,%20Huang%20H,%20Xu%20H,%20Rono%20PC,%20Cai%20X,%20Hu%20G.%20Assembly%20and%20comparative%20analysis%20of%20the%20initial%20complete%20mitochondrial%20genome%20of%20Primulina%20hunanensis%20(Gesneriaceae):%20a%20cave-dwelling%20endangered%20plant.%20BMC%20Genomics.%202024;25(1):322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Wang J, He W, Xiang K, Wu Z, Gu C. Advances in plant phylogeny in the genome era. J Zhejiang A F Univ. 2023;40(1):227.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20He%20W,%20Xiang%20K,%20Wu%20Z,%20Gu%20C.%20Advances%20in%20plant%20phylogeny%20in%20the%20genome%20era.%20J%20Zhejiang%20A%20F%20Univ.%202023;40(1):227." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Liu Y, Cox CJ, Wang W, Goffinet B. Mitochondrial phylogenomics of early land plants: mitigating the effects of saturation, compositional heterogeneity, and codon-usage bias. Syst Biol. 2014;63(6):862–78.
</cite> [<a href="https://doi.org/10.1093/sysbio/syu049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25070972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Y,%20Cox%20CJ,%20Wang%20W,%20Goffinet%20B.%20Mitochondrial%20phylogenomics%20of%20early%20land%20plants:%20mitigating%20the%20effects%20of%20saturation,%20compositional%20heterogeneity,%20and%20codon-usage%20bias.%20Syst%20Biol.%202014;63(6):862%E2%80%9378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Dong S, Chen L, Liu Y, Wang Y, Zhang S, Yang L, Lang X, Zhang S. The draft mitochondrial genome of <em>Magnolia biondii</em> and mitochondrial phylogenomics of angiosperms. PLoS One. 2020;15(4):e0231020.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0231020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7159230/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32294100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong%20S,%20Chen%20L,%20Liu%20Y,%20Wang%20Y,%20Zhang%20S,%20Yang%20L,%20Lang%20X,%20Zhang%20S.%20The%20draft%20mitochondrial%20genome%20of%20Magnolia%20biondii%20and%20mitochondrial%20phylogenomics%20of%20angiosperms.%20PLoS%20One.%202020;15(4):e0231020." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Zhang D, Ren J, Jiang H, Wanga VO, Dong X, Hu G. Comparative and phylogenetic analysis of the complete chloroplast genomes of six <em>Polygonatum</em> species (Asparagaceae). Sci Rep. 2023;13(1):7237.
</cite> [<a href="https://doi.org/10.1038/s41598-023-34083-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10160070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37142659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20D,%20Ren%20J,%20Jiang%20H,%20Wanga%20VO,%20Dong%20X,%20Hu%20G.%20Comparative%20and%20phylogenetic%20analysis%20of%20the%20complete%20chloroplast%20genomes%20of%20six%20Polygonatum%20species%20(Asparagaceae).%20Sci%20Rep.%202023;13(1):7237." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Yao J, Zheng Z, Xu T, Wang D, Pu J, Zhang Y, Zha L. Chloroplast genome sequencing and comparative analysis of six medicinal plants of Polygonatum. Ecol Evol. 2025;15(1): e70831.
</cite> [<a href="https://doi.org/10.1002/ece3.70831" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11718222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39803186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yao%20J,%20Zheng%20Z,%20Xu%20T,%20Wang%20D,%20Pu%20J,%20Zhang%20Y,%20Zha%20L.%20Chloroplast%20genome%20sequencing%20and%20comparative%20analysis%20of%20six%20medicinal%20plants%20of%20Polygonatum.%20Ecol%20Evol.%202025;15(1):%20e70831." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Sheng W, Deng J, Wang C, Kuang Q. The garden asparagus (<em>Asparagus officinalis</em> L.) mitochondrial genome revealed rich sequence variation throughout whole sequencing data. Front Plant Sci. 2023;14:1140043.
</cite> [<a href="https://doi.org/10.3389/fpls.2023.1140043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10084930/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37051082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sheng%20W,%20Deng%20J,%20Wang%20C,%20Kuang%20Q.%20The%20garden%20asparagus%20(Asparagus%20officinalis%20L.)%20mitochondrial%20genome%20revealed%20rich%20sequence%20variation%20throughout%20whole%20sequencing%20data.%20Front%20Plant%20Sci.%202023;14:1140043." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Cui X, Peng Y, Hu L, Shao YY, Song EL, Hou Y, Liu S: Comparative Analysis of the Complete Mitochondrion of Chlorophytum Comosum with 9 Other Species. In: 2021.</cite>
</li>
<li id="CR29">
<span class="label">29.</span><cite>Liu M, Fan R, Wang C, Dai L, Chu S. Complete analysis and phylogenetic analysis of <em>Polygonatum sibiricum</em> mitochondria. BMC Plant Biol. 2025;25(1):471.
</cite> [<a href="https://doi.org/10.1186/s12870-025-06510-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11998138/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40229680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20M,%20Fan%20R,%20Wang%20C,%20Dai%20L,%20Chu%20S.%20Complete%20analysis%20and%20phylogenetic%20analysis%20of%20Polygonatum%20sibiricum%20mitochondria.%20BMC%20Plant%20Biol.%202025;25(1):471." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Chen S, Zhou Y, Chen Y, Gu J. Fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics. 2018;34(17):i884–90.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20S,%20Zhou%20Y,%20Chen%20Y,%20Gu%20J.%20Fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor.%20Bioinformatics.%202018;34(17):i884%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Li H. Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. 2018;34(18):3094–100.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6137996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H.%20Minimap2:%20pairwise%20alignment%20for%20nucleotide%20sequences.%20Bioinformatics.%202018;34(18):3094%E2%80%93100." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Song Y, Du X, Li A, Fan A, He L, Sun Z, Niu Y, Qiao Y. Assembly and analysis of the complete mitochondrial genome of <em>Forsythia suspensa</em> (Thunb.) Vahl. BMC Genomics. 2023;24(1):708.
</cite> [<a href="https://doi.org/10.1186/s12864-023-09821-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10666317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37996801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song%20Y,%20Du%20X,%20Li%20A,%20Fan%20A,%20He%20L,%20Sun%20Z,%20Niu%20Y,%20Qiao%20Y.%20Assembly%20and%20analysis%20of%20the%20complete%20mitochondrial%20genome%20of%20Forsythia%20suspensa%20(Thunb.)%20Vahl.%20BMC%20Genomics.%202023;24(1):708." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Koren S, Walenz BP, Berlin K, Miller JR, Bergman NH, Phillippy AM. Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation. Genome Res. 2017;27(5):722–36.
</cite> [<a href="https://doi.org/10.1101/gr.215087.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5411767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28298431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koren%20S,%20Walenz%20BP,%20Berlin%20K,%20Miller%20JR,%20Bergman%20NH,%20Phillippy%20AM.%20Canu:%20scalable%20and%20accurate%20long-read%20assembly%20via%20adaptive%20k-mer%20weighting%20and%20repeat%20separation.%20Genome%20Res.%202017;27(5):722%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9(4):357–9.
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead%20B,%20Salzberg%20SL.%20Fast%20gapped-read%20alignment%20with%20Bowtie%202.%20Nat%20Methods.%202012;9(4):357%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wick RR, Judd LM, Gorrie CL, Holt KE. Unicycler: resolving bacterial genome assemblies from short and long sequencing reads. PLoS Comput Biol. 2017;13(6):e1005595.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1005595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5481147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28594827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wick%20RR,%20Judd%20LM,%20Gorrie%20CL,%20Holt%20KE.%20Unicycler:%20resolving%20bacterial%20genome%20assemblies%20from%20short%20and%20long%20sequencing%20reads.%20PLoS%20Comput%20Biol.%202017;13(6):e1005595." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Camacho C, Coulouris G, Avagyan V, Ma N, Papadopoulos J, Bealer K, Madden TL. BLAST+: architecture and applications. BMC Bioinformatics. 2009;10:421.
</cite> [<a href="https://doi.org/10.1186/1471-2105-10-421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2803857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20003500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camacho%20C,%20Coulouris%20G,%20Avagyan%20V,%20Ma%20N,%20Papadopoulos%20J,%20Bealer%20K,%20Madden%20TL.%20BLAST+:%20architecture%20and%20applications.%20BMC%20Bioinformatics.%202009;10:421." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Chan PP, Lowe TM. <em>tRNA</em>scan-SE: searching for <em>tRNA</em> genes in genomic sequences. Methods Mol Biol. 2019;1962:1–14.
</cite> [<a href="https://doi.org/10.1007/978-1-4939-9173-0_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6768409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31020551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chan%20PP,%20Lowe%20TM.%20tRNAscan-SE:%20searching%20for%20tRNA%20genes%20in%20genomic%20sequences.%20Methods%20Mol%20Biol.%202019;1962:1%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Greiner S, Lehwark P, Bock R. Organellargenomedraw (OGDRAW) version 1.3.1: expanded toolkit for the graphical visualization of organellar genomes. Nucleic Acids Res. 2019;47(W1):W59-64.
</cite> [<a href="https://doi.org/10.1093/nar/gkz238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6602502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30949694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greiner%20S,%20Lehwark%20P,%20Bock%20R.%20Organellargenomedraw%20(OGDRAW)%20version%201.3.1:%20expanded%20toolkit%20for%20the%20graphical%20visualization%20of%20organellar%20genomes.%20Nucleic%20Acids%20Res.%202019;47(W1):W59-64." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Beier S, Thiel T, Münch T, Scholz U, Mascher M. MISA-web: a web server for microsatellite prediction. Bioinformatics. 2017;33(16):2583–5.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5870701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28398459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beier%20S,%20Thiel%20T,%20M%C3%BCnch%20T,%20Scholz%20U,%20Mascher%20M.%20MISA-web:%20a%20web%20server%20for%20microsatellite%20prediction.%20Bioinformatics.%202017;33(16):2583%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Benson G. Tandem repeats finder: a program to analyze DNA sequences. Nucleic Acids Res. 1999;27(2):573–80.
</cite> [<a href="https://doi.org/10.1093/nar/27.2.573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC148217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9862982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benson%20G.%20Tandem%20repeats%20finder:%20a%20program%20to%20analyze%20DNA%20sequences.%20Nucleic%20Acids%20Res.%201999;27(2):573%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Krzywinski M, Schein J, Birol I, Connors J, Gascoyne R, Horsman D, Jones SJ, Marra MA. Circos: an information aesthetic for comparative genomics. Genome Res. 2009;19(9):1639–45.
</cite> [<a href="https://doi.org/10.1101/gr.092759.109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2752132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19541911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krzywinski%20M,%20Schein%20J,%20Birol%20I,%20Connors%20J,%20Gascoyne%20R,%20Horsman%20D,%20Jones%20SJ,%20Marra%20MA.%20Circos:%20an%20information%20aesthetic%20for%20comparative%20genomics.%20Genome%20Res.%202009;19(9):1639%E2%80%9345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Xie ZN, Lao J, Liu H, Zhang WX, He W, Zhong C, Xie J, Zhang SH, Jin J. Characterization of the chloroplast genome of medicinal herb <em>Polygonatum cyrtonema</em> and identification of molecular markers by comparative analysis. Genome. 2023;66(4):80–90.
</cite> [<a href="https://doi.org/10.1139/gen-2022-0087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36763968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie%20ZN,%20Lao%20J,%20Liu%20H,%20Zhang%20WX,%20He%20W,%20Zhong%20C,%20Xie%20J,%20Zhang%20SH,%20Jin%20J.%20Characterization%20of%20the%20chloroplast%20genome%20of%20medicinal%20herb%20Polygonatum%20cyrtonema%20and%20identification%20of%20molecular%20markers%20by%20comparative%20analysis.%20Genome.%202023;66(4):80%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Katoh K, Standley DM. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol. 2013;30(4):772–80.
</cite> [<a href="https://doi.org/10.1093/molbev/mst010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3603318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23329690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katoh%20K,%20Standley%20DM.%20MAFFT%20multiple%20sequence%20alignment%20software%20version%207:%20improvements%20in%20performance%20and%20usability.%20Mol%20Biol%20Evol.%202013;30(4):772%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Librado P, Rozas J. DnaSP v5: a software for comprehensive analysis of DNA polymorphism data. Bioinformatics. 2009;25(11):1451–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19346325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Librado%20P,%20Rozas%20J.%20DnaSP%20v5:%20a%20software%20for%20comprehensive%20analysis%20of%20DNA%20polymorphism%20data.%20Bioinformatics.%202009;25(11):1451%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Capella-Gutiérrez S, Silla-Martínez JM, Gabaldón T. trimAl: a tool for automated alignment trimming in large-scale phylogenetic analyses. Bioinformatics. 2009;25(15):1972–3.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2712344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Capella-Guti%C3%A9rrez%20S,%20Silla-Mart%C3%ADnez%20JM,%20Gabald%C3%B3n%20T.%20trimAl:%20a%20tool%20for%20automated%20alignment%20trimming%20in%20large-scale%20phylogenetic%20analyses.%20Bioinformatics.%202009;25(15):1972%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Stamatakis A. RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics. 2014;30(9):1312–3.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3998144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24451623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stamatakis%20A.%20RAxML%20version%208:%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies.%20Bioinformatics.%202014;30(9):1312%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Posada D. jModelTest: phylogenetic model averaging. Mol Biol Evol. 2008;25(7):1253–6.
</cite> [<a href="https://doi.org/10.1093/molbev/msn083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18397919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Posada%20D.%20jModelTest:%20phylogenetic%20model%20averaging.%20Mol%20Biol%20Evol.%202008;25(7):1253%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Wang D, Zhang Y, Zhang Z, Zhu J, Yu J. Kaks_calculator 2.0: a toolkit incorporating gamma-series methods and sliding window strategies. Genomics Proteomics Bioinf. 2010;8(1):77–80.</cite> [<a href="https://doi.org/10.1016/S1672-0229(10)60008-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5054116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20451164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20D,%20Zhang%20Y,%20Zhang%20Z,%20Zhu%20J,%20Yu%20J.%20Kaks_calculator%202.0:%20a%20toolkit%20incorporating%20gamma-series%20methods%20and%20sliding%20window%20strategies.%20Genomics%20Proteomics%20Bioinf.%202010;8(1):77%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Wick RR, Schultz MB, Zobel J, Holt KE. Bandage: interactive visualization of de novo genome assemblies. Bioinformatics. 2015;31(20):3350–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4595904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26099265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wick%20RR,%20Schultz%20MB,%20Zobel%20J,%20Holt%20KE.%20Bandage:%20interactive%20visualization%20of%20de%20novo%20genome%20assemblies.%20Bioinformatics.%202015;31(20):3350%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Li Z, Zhao L, Yang T, Tang J, Miao Y, Ren T. Genome-wide simple sequence repeat analysis and specific molecular marker development of rye. BMC Genomics. 2024;25(1):780.
</cite> [<a href="https://doi.org/10.1186/s12864-024-10689-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11318315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39134932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Z,%20Zhao%20L,%20Yang%20T,%20Tang%20J,%20Miao%20Y,%20Ren%20T.%20Genome-wide%20simple%20sequence%20repeat%20analysis%20and%20specific%20molecular%20marker%20development%20of%20rye.%20BMC%20Genomics.%202024;25(1):780." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Rajagopal J, Das S, Khurana DK, Srivastava PS, Lakshmikumaran M. Molecular characterization and distribution of a 145-bp tandem repeat family in the genus <em>Populus</em>. Genome. 1999;42(5):909–18.
</cite> [<a href="https://doi.org/10.1139/g99-013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10584311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rajagopal%20J,%20Das%20S,%20Khurana%20DK,%20Srivastava%20PS,%20Lakshmikumaran%20M.%20Molecular%20characterization%20and%20distribution%20of%20a%20145-bp%20tandem%20repeat%20family%20in%20the%20genus%20Populus.%20Genome.%201999;42(5):909%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Tourasse NJ, Shtaida N, Khozin-Goldberg I, Boussiba S, Vallon O. The complete mitochondrial genome sequence of the green microalga <em>Lobosphaera</em> (<em>Parietochloris</em>) <em>incisa</em> reveals a new type of palindromic repetitive repeat. BMC Genomics. 2015;16(1):580.
</cite> [<a href="https://doi.org/10.1186/s12864-015-1792-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4524435/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26238519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tourasse%20NJ,%20Shtaida%20N,%20Khozin-Goldberg%20I,%20Boussiba%20S,%20Vallon%20O.%20The%20complete%20mitochondrial%20genome%20sequence%20of%20the%20green%20microalga%20Lobosphaera%20(Parietochloris)%20incisa%20reveals%20a%20new%20type%20of%20palindromic%20repetitive%20repeat.%20BMC%20Genomics.%202015;16(1):580." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Li YY, Liu YY, Zeng X, Wu P, Li QM, Guo SX, Hao ZG. Complete mitochondrial genome of <em>Angelica dahurica</em> and its implications on evolutionary analysis of complex mitochondrial genome architecture in Apiaceae. Front Plant Sci. 2024;15:1367299.
</cite> [<a href="https://doi.org/10.3389/fpls.2024.1367299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11074370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38716337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20YY,%20Liu%20YY,%20Zeng%20X,%20Wu%20P,%20Li%20QM,%20Guo%20SX,%20Hao%20ZG.%20Complete%20mitochondrial%20genome%20of%20Angelica%20dahurica%20and%20its%20implications%20on%20evolutionary%20analysis%20of%20complex%20mitochondrial%20genome%20architecture%20in%20Apiaceae.%20Front%20Plant%20Sci.%202024;15:1367299." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Palmer JD, Shields CR. Tripartite structure of the <em>Brassica campestris</em> mitochondrial genome. Nature. 1984;307(5950):437–40.</cite> [<a href="https://scholar.google.com/scholar_lookup?Palmer%20JD,%20Shields%20CR.%20Tripartite%20structure%20of%20the%20Brassica%20campestris%20mitochondrial%20genome.%20Nature.%201984;307(5950):437%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Lonsdale DM, Hodge TP, Fauron CM. The physical map and organisation of the mitochondrial genome from the fertile cytoplasm of maize. Nucleic Acids Res. 1984;12(24):9249–61.
</cite> [<a href="https://doi.org/10.1093/nar/12.24.9249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC320458/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6096824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lonsdale%20DM,%20Hodge%20TP,%20Fauron%20CM.%20The%20physical%20map%20and%20organisation%20of%20the%20mitochondrial%20genome%20from%20the%20fertile%20cytoplasm%20of%20maize.%20Nucleic%20Acids%20Res.%201984;12(24):9249%E2%80%9361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Liu D, Guo H, Zhu J, Qu K, Chen Y, Guo Y, Ding P, Yang H, Xu T, Jing Q, et al. Complex physical structure of complete mitochondrial genome of <em>Quercus acutissima</em> (Fagaceae): a significant energy plant. Genes. 2022. 10.3390/genes13081321.
</cite> [<a href="https://doi.org/10.3390/genes13081321" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9331829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35893058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20D,%20Guo%20H,%20Zhu%20J,%20Qu%20K,%20Chen%20Y,%20Guo%20Y,%20Ding%20P,%20Yang%20H,%20Xu%20T,%20Jing%20Q,%20et%20al.%20Complex%20physical%20structure%20of%20complete%20mitochondrial%20genome%20of%20Quercus%20acutissima%20(Fagaceae):%20a%20significant%20energy%20plant.%20Genes.%202022.%2010.3390/genes13081321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Zhao J, Chen J, Xiong Y, He W, Xiong Y, Xu Y, Ma H, Yu Q, Li Z, Liu L, et al. Organelle genomes of <em>Indigofera amblyantha</em> and <em>Indigofera pseudotinctoria</em>: comparative genome analysis, and intracellular gene transfer. Ind Crop Prod. 2023;198: 116674.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhao%20J,%20Chen%20J,%20Xiong%20Y,%20He%20W,%20Xiong%20Y,%20Xu%20Y,%20Ma%20H,%20Yu%20Q,%20Li%20Z,%20Liu%20L,%20et%20al.%20Organelle%20genomes%20of%20Indigofera%20amblyantha%20and%20Indigofera%20pseudotinctoria:%20comparative%20genome%20analysis,%20and%20intracellular%20gene%20transfer.%20Ind%20Crop%20Prod.%202023;198:%20116674." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Kersten B, Rellstab C, Schroeder H, Brodbeck S, Fladung M, Krutovsky KV, Gugerli F. The mitochondrial genome sequence of <em>Abies alba</em> Mill. reveals a high structural and combinatorial variation. BMC Genomics. 2022;23(1):776.
</cite> [<a href="https://doi.org/10.1186/s12864-022-08993-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9703787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36443651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kersten%20B,%20Rellstab%20C,%20Schroeder%20H,%20Brodbeck%20S,%20Fladung%20M,%20Krutovsky%20KV,%20Gugerli%20F.%20The%20mitochondrial%20genome%20sequence%20of%20Abies%20alba%20Mill.%20reveals%20a%20high%20structural%20and%20combinatorial%20variation.%20BMC%20Genomics.%202022;23(1):776." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Kozik A, Rowan BA, Lavelle D, Berke L, Schranz ME, Michelmore RW, Christensen AC. The alternative reality of plant mitochondrial DNA: one ring does not rule them all. PLoS Genet. 2019;15(8):e1008373.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1008373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6742443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31469821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kozik%20A,%20Rowan%20BA,%20Lavelle%20D,%20Berke%20L,%20Schranz%20ME,%20Michelmore%20RW,%20Christensen%20AC.%20The%20alternative%20reality%20of%20plant%20mitochondrial%20DNA:%20one%20ring%20does%20not%20rule%20them%20all.%20PLoS%20Genet.%202019;15(8):e1008373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Putintseva YA, Bondar EI, Simonov EP, Sharov VV, Oreshkova NV, Kuzmin DA, Konstantinov YM, Shmakov VN, Belkov VI, Sadovsky MG, et al. Siberian larch (<em>Larix sibirica</em> Ledeb.) mitochondrial genome assembled using both short and long nucleotide sequence reads is currently the largest known mitogenome. BMC Genomics. 2020;21(1):654.
</cite> [<a href="https://doi.org/10.1186/s12864-020-07061-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7517811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32972367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Putintseva%20YA,%20Bondar%20EI,%20Simonov%20EP,%20Sharov%20VV,%20Oreshkova%20NV,%20Kuzmin%20DA,%20Konstantinov%20YM,%20Shmakov%20VN,%20Belkov%20VI,%20Sadovsky%20MG,%20et%20al.%20Siberian%20larch%20(Larix%20sibirica%20Ledeb.)%20mitochondrial%20genome%20assembled%20using%20both%20short%20and%20long%20nucleotide%20sequence%20reads%20is%20currently%20the%20largest%20known%20mitogenome.%20BMC%20Genomics.%202020;21(1):654." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Handa H. The complete nucleotide sequence and RNA editing content of the mitochondrial genome of rapeseed (<em>Brassica napus</em> L.): comparative analysis of the mitochondrial genomes of rapeseed and <em>Arabidopsis thaliana</em>. Nucleic Acids Res. 2003;31(20):5907–16.
</cite> [<a href="https://doi.org/10.1093/nar/gkg795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC219474/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14530439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Handa%20H.%20The%20complete%20nucleotide%20sequence%20and%20RNA%20editing%20content%20of%20the%20mitochondrial%20genome%20of%20rapeseed%20(Brassica%20napus%20L.):%20comparative%20analysis%20of%20the%20mitochondrial%20genomes%20of%20rapeseed%20and%20Arabidopsis%20thaliana.%20Nucleic%20Acids%20Res.%202003;31(20):5907%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Zhang SYB, Chen Y, Shen J, Zhu J, LI Haibo. Comparative analysis on genome sizes of four Polygonatum species. Journal of Forest and Environment 2022, 42(2).</cite>
</li>
<li id="CR63">
<span class="label">63.</span><cite>Luo X, Gu C, Gao S, Li M, Zhang H, Zhu S. Complete mitochondrial genome assembly of <em>Zizania latifolia</em> and comparative genome analysis. Front Plant Sci. 2024;15:1381089.
</cite> [<a href="https://doi.org/10.3389/fpls.2024.1381089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11341417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39184575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luo%20X,%20Gu%20C,%20Gao%20S,%20Li%20M,%20Zhang%20H,%20Zhu%20S.%20Complete%20mitochondrial%20genome%20assembly%20of%20Zizania%20latifolia%20and%20comparative%20genome%20analysis.%20Front%20Plant%20Sci.%202024;15:1381089." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Bergthorsson U, Adams KL, Thomason B, Palmer JD. Widespread horizontal transfer of mitochondrial genes in flowering plants. Nature. 2003;424(6945):197–201.
</cite> [<a href="https://doi.org/10.1038/nature01743" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12853958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bergthorsson%20U,%20Adams%20KL,%20Thomason%20B,%20Palmer%20JD.%20Widespread%20horizontal%20transfer%20of%20mitochondrial%20genes%20in%20flowering%20plants.%20Nature.%202003;424(6945):197%E2%80%93201." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Adams KL, Qiu YL, Stoutemyer M, Palmer JD. Punctuated evolution of mitochondrial gene content: high and variable rates of mitochondrial gene loss and transfer to the nucleus during angiosperm evolution. Proc Natl Acad Sci U S A. 2002;99(15):9905–12.
</cite> [<a href="https://doi.org/10.1073/pnas.042694899" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC126597/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12119382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adams%20KL,%20Qiu%20YL,%20Stoutemyer%20M,%20Palmer%20JD.%20Punctuated%20evolution%20of%20mitochondrial%20gene%20content:%20high%20and%20variable%20rates%20of%20mitochondrial%20gene%20loss%20and%20transfer%20to%20the%20nucleus%20during%20angiosperm%20evolution.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202002;99(15):9905%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Petersen G, Cuenca A, Zervas A, Ross GT, Graham SW, Barrett CF, Davis JI, Seberg O. Mitochondrial genome evolution in Alismatales: size reduction and extensive loss of ribosomal protein genes. PLoS One. 2017;12(5):e0177606.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0177606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5435185/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28545148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petersen%20G,%20Cuenca%20A,%20Zervas%20A,%20Ross%20GT,%20Graham%20SW,%20Barrett%20CF,%20Davis%20JI,%20Seberg%20O.%20Mitochondrial%20genome%20evolution%20in%20Alismatales:%20size%20reduction%20and%20extensive%20loss%20of%20ribosomal%20protein%20genes.%20PLoS%20One.%202017;12(5):e0177606." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Guo W, Grewe F, Fan W, Young GJ, Knoop V, Palmer JD, Mower JP. <em>Ginkgo</em> and <em>Welwitschia</em> mitogenomes reveal extreme contrasts in gymnosperm mitochondrial evolution. Mol Biol Evol. 2016;33(6):1448–60.
</cite> [<a href="https://doi.org/10.1093/molbev/msw024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26831941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20W,%20Grewe%20F,%20Fan%20W,%20Young%20GJ,%20Knoop%20V,%20Palmer%20JD,%20Mower%20JP.%20Ginkgo%20and%20Welwitschia%20mitogenomes%20reveal%20extreme%20contrasts%20in%20gymnosperm%20mitochondrial%20evolution.%20Mol%20Biol%20Evol.%202016;33(6):1448%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Adams KL, Palmer JD. Evolution of mitochondrial gene content: gene loss and transfer to the nucleus. Mol Phylogenet Evol. 2003;29(3):380–95.
</cite> [<a href="https://doi.org/10.1016/s1055-7903(03)00194-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14615181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adams%20KL,%20Palmer%20JD.%20Evolution%20of%20mitochondrial%20gene%20content:%20gene%20loss%20and%20transfer%20to%20the%20nucleus.%20Mol%20Phylogenet%20Evol.%202003;29(3):380%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Fajardo D, Schlautman B, Steffan S, Polashock J, Vorsa N, Zalapa J. The American cranberry mitochondrial genome reveals the presence of selenocysteine (tRNA-Sec and SECIS) insertion machinery in land plants. Gene. 2014;536(2):336–43.
</cite> [<a href="https://doi.org/10.1016/j.gene.2013.11.104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24342657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fajardo%20D,%20Schlautman%20B,%20Steffan%20S,%20Polashock%20J,%20Vorsa%20N,%20Zalapa%20J.%20The%20American%20cranberry%20mitochondrial%20genome%20reveals%20the%20presence%20of%20selenocysteine%20(tRNA-Sec%20and%20SECIS)%20insertion%20machinery%20in%20land%20plants.%20Gene.%202014;536(2):336%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Kubo N, Arimura S. Discovery of the rpl10 gene in diverse plant mitochondrial genomes and its probable replacement by the nuclear gene for chloroplast RPL10 in two lineages of angiosperms. DNA Res. 2010;17(1):1–9.
</cite> [<a href="https://doi.org/10.1093/dnares/dsp024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2818186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19934175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kubo%20N,%20Arimura%20S.%20Discovery%20of%20the%20rpl10%20gene%20in%20diverse%20plant%20mitochondrial%20genomes%20and%20its%20probable%20replacement%20by%20the%20nuclear%20gene%20for%20chloroplast%20RPL10%20in%20two%20lineages%20of%20angiosperms.%20DNA%20Res.%202010;17(1):1%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Adams KL, Rosenblueth M, Qiu YL, Palmer JD. Multiple losses and transfers to the nucleus of two mitochondrial succinate dehydrogenase genes during angiosperm evolution. Genetics. 2001;158(3):1289–300.
</cite> [<a href="https://doi.org/10.1093/genetics/158.3.1289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461739/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11454775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adams%20KL,%20Rosenblueth%20M,%20Qiu%20YL,%20Palmer%20JD.%20Multiple%20losses%20and%20transfers%20to%20the%20nucleus%20of%20two%20mitochondrial%20succinate%20dehydrogenase%20genes%20during%20angiosperm%20evolution.%20Genetics.%202001;158(3):1289%E2%80%93300." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Adams KL, Ong HC, Palmer JD. Mitochondrial gene transfer in pieces: fission of the ribosomal protein gene rpl2 and partial or complete gene transfer to the nucleus. Mol Biol Evol. 2001;18(12):2289–97.
</cite> [<a href="https://doi.org/10.1093/oxfordjournals.molbev.a003775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11719578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adams%20KL,%20Ong%20HC,%20Palmer%20JD.%20Mitochondrial%20gene%20transfer%20in%20pieces:%20fission%20of%20the%20ribosomal%20protein%20gene%20rpl2%20and%20partial%20or%20complete%20gene%20transfer%20to%20the%20nucleus.%20Mol%20Biol%20Evol.%202001;18(12):2289%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Small ID, Schallenberg-Rüdinger M, Takenaka M, Mireau H, Ostersetzer-Biran O. Plant organellar RNA editing: what 30 years of research has revealed. Plant J. 2020;101(5):1040–56.
</cite> [<a href="https://doi.org/10.1111/tpj.14578" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31630458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Small%20ID,%20Schallenberg-R%C3%BCdinger%20M,%20Takenaka%20M,%20Mireau%20H,%20Ostersetzer-Biran%20O.%20Plant%20organellar%20RNA%20editing:%20what%2030%20years%20of%20research%20has%20revealed.%20Plant%20J.%202020;101(5):1040%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Bullerwell CE, Gray MW. Evolution of the mitochondrial genome: protist connections to animals, fungi and plants. Curr Opin Microbiol. 2004;7(5):528–34.
</cite> [<a href="https://doi.org/10.1016/j.mib.2004.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15451509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bullerwell%20CE,%20Gray%20MW.%20Evolution%20of%20the%20mitochondrial%20genome:%20protist%20connections%20to%20animals,%20fungi%20and%20plants.%20Curr%20Opin%20Microbiol.%202004;7(5):528%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Chevigny N, Schatz-Daas D, Lotfi F, Gualberto JM. DNA repair and the stability of the plant mitochondrial genome. Int J Mol Sci. 2020. 10.3390/ijms21010328.
</cite> [<a href="https://doi.org/10.3390/ijms21010328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6981420/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31947741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chevigny%20N,%20Schatz-Daas%20D,%20Lotfi%20F,%20Gualberto%20JM.%20DNA%20repair%20and%20the%20stability%20of%20the%20plant%20mitochondrial%20genome.%20Int%20J%20Mol%20Sci.%202020.%2010.3390/ijms21010328." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Shan Y, Li J, Zhang X, Yu J. The complete mitochondrial genome of <em>Amorphophallus albus</em> and development of molecular markers for five <em>Amorphophallus</em> species based on mitochondrial DNA. Front Plant Sci. 2023;14:1180417.
</cite> [<a href="https://doi.org/10.3389/fpls.2023.1180417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10322194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37416891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shan%20Y,%20Li%20J,%20Zhang%20X,%20Yu%20J.%20The%20complete%20mitochondrial%20genome%20of%20Amorphophallus%20albus%20and%20development%20of%20molecular%20markers%20for%20five%20Amorphophallus%20species%20based%20on%20mitochondrial%20DNA.%20Front%20Plant%20Sci.%202023;14:1180417." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Yang H, Ni Y, Zhang X, Li J, Chen H, Liu C. The mitochondrial genomes of <em>Panax notoginseng</em> reveal recombination mediated by repeats associated with DNA replication. Int J Biol Macromol. 2023;252: 126359.
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2023.126359" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37619687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20H,%20Ni%20Y,%20Zhang%20X,%20Li%20J,%20Chen%20H,%20Liu%20C.%20The%20mitochondrial%20genomes%20of%20Panax%20notoginseng%20reveal%20recombination%20mediated%20by%20repeats%20associated%20with%20DNA%20replication.%20Int%20J%20Biol%20Macromol.%202023;252:%20126359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>You L, Yang L, Liu F, Kang L, Chen H, Yang B, Yang Q, Liu Z. Assembly and marker analysis of mitochondrial genomes provide insights into origin, evolution and spread of <em>Brassica juncea</em> (L.) Czern. et Coss. Crop J. 2022;10(3):611–6.</cite> [<a href="https://scholar.google.com/scholar_lookup?You%20L,%20Yang%20L,%20Liu%20F,%20Kang%20L,%20Chen%20H,%20Yang%20B,%20Yang%20Q,%20Liu%20Z.%20Assembly%20and%20marker%20analysis%20of%20mitochondrial%20genomes%20provide%20insights%20into%20origin,%20evolution%20and%20spread%20of%20Brassica%20juncea%20(L.)%20Czern.%20et%20Coss.%20Crop%20J.%202022;10(3):611%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Wynn EL, Christensen AC. Repeats of unusual size in plant mitochondrial genomes: identification, incidence and evolution. G3 Genes Genomes Genet. 2019;9(2):549–59.</cite> [<a href="https://doi.org/10.1534/g3.118.200948" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6385970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30563833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wynn%20EL,%20Christensen%20AC.%20Repeats%20of%20unusual%20size%20in%20plant%20mitochondrial%20genomes:%20identification,%20incidence%20and%20evolution.%20G3%20Genes%20Genomes%20Genet.%202019;9(2):549%E2%80%9359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Bi C, Paterson AH, Wang X, Xu Y, Wu D, Qu Y, Jiang A, Ye Q, Ye N. Analysis of the complete mitochondrial genome sequence of the diploid cotton gossypium raimondii by comparative genomics approaches. Biomed Res Int. 2016;2016:5040598.
</cite> [<a href="https://doi.org/10.1155/2016/5040598" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5099484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27847816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bi%20C,%20Paterson%20AH,%20Wang%20X,%20Xu%20Y,%20Wu%20D,%20Qu%20Y,%20Jiang%20A,%20Ye%20Q,%20Ye%20N.%20Analysis%20of%20the%20complete%20mitochondrial%20genome%20sequence%20of%20the%20diploid%20cotton%20gossypium%20raimondii%20by%20comparative%20genomics%20approaches.%20Biomed%20Res%20Int.%202016;2016:5040598." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Edera AA, Gandini CL, Sanchez-Puerta MV. Towards a comprehensive picture of C-to-U RNA editing sites in angiosperm mitochondria. Plant Mol Biol. 2018;97(3):215–31.
</cite> [<a href="https://doi.org/10.1007/s11103-018-0734-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29761268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edera%20AA,%20Gandini%20CL,%20Sanchez-Puerta%20MV.%20Towards%20a%20comprehensive%20picture%20of%20C-to-U%20RNA%20editing%20sites%20in%20angiosperm%20mitochondria.%20Plant%20Mol%20Biol.%202018;97(3):215%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Giegé P, Brennicke A. RNA editing in Arabidopsis mitochondria effects 441 c to u changes in ORFs. Proc Natl Acad Sci U S A. 1999;96(26):15324–9.
</cite> [<a href="https://doi.org/10.1073/pnas.96.26.15324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC24818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10611383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gieg%C3%A9%20P,%20Brennicke%20A.%20RNA%20editing%20in%20Arabidopsis%20mitochondria%20effects%20441%20c%20to%20u%20changes%20in%20ORFs.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%201999;96(26):15324%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Sloan DB, MacQueen AH, Alverson AJ, Palmer JD, Taylor DR. Extensive loss of RNA editing sites in rapidly evolving <em>Silene</em> mitochondrial genomes: selection vs. retroprocessing as the driving force. Genetics. 2010;185(4):1369–80.
</cite> [<a href="https://doi.org/10.1534/genetics.110.118000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2927763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20479143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sloan%20DB,%20MacQueen%20AH,%20Alverson%20AJ,%20Palmer%20JD,%20Taylor%20DR.%20Extensive%20loss%20of%20RNA%20editing%20sites%20in%20rapidly%20evolving%20Silene%20mitochondrial%20genomes:%20selection%20vs.%20retroprocessing%20as%20the%20driving%20force.%20Genetics.%202010;185(4):1369%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Bentolila S, Alfonso AA, Hanson MR. A pentatricopeptide repeat-containing gene restores fertility to cytoplasmic male-sterile plants. Proc Natl Acad Sci U S A. 2002;99(16):10887–92.
</cite> [<a href="https://doi.org/10.1073/pnas.102301599" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC125068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12136123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bentolila%20S,%20Alfonso%20AA,%20Hanson%20MR.%20A%20pentatricopeptide%20repeat-containing%20gene%20restores%20fertility%20to%20cytoplasmic%20male-sterile%20plants.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202002;99(16):10887%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Feng B, Xiong J, Tao L. How rice plants response to abiotic stresses. Int J Mol Sci. 2023. 10.3390/ijms241612806.
</cite> [<a href="https://doi.org/10.3390/ijms241612806" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10454531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37628985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng%20B,%20Xiong%20J,%20Tao%20L.%20How%20rice%20plants%20response%20to%20abiotic%20stresses.%20Int%20J%20Mol%20Sci.%202023.%2010.3390/ijms241612806." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Huang Y, Xia P. Biomolecular condensates in plant cells: mediating and integrating environmental signals and development. Plant Sci. 2024;347: 112178.
</cite> [<a href="https://doi.org/10.1016/j.plantsci.2024.112178" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38971467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20Y,%20Xia%20P.%20Biomolecular%20condensates%20in%20plant%20cells:%20mediating%20and%20integrating%20environmental%20signals%20and%20development.%20Plant%20Sci.%202024;347:%20112178." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Duan B, Xie X, Jiang Y, Zhu N, Zheng H, Liu Y, Hua X, Zhao Y, Sun Y. GhMYB44 enhances stomatal closure to confer drought stress tolerance in cotton and Arabidopsis. Plant Physiol Biochem. 2023;198: 107692.
</cite> [<a href="https://doi.org/10.1016/j.plaphy.2023.107692" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37058965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duan%20B,%20Xie%20X,%20Jiang%20Y,%20Zhu%20N,%20Zheng%20H,%20Liu%20Y,%20Hua%20X,%20Zhao%20Y,%20Sun%20Y.%20GhMYB44%20enhances%20stomatal%20closure%20to%20confer%20drought%20stress%20tolerance%20in%20cotton%20and%20Arabidopsis.%20Plant%20Physiol%20Biochem.%202023;198:%20107692." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Zhang W, Han Z, Guo Q, Liu Y, Zheng Y, Wu F, Jin W. Identification of maize long non-coding RNAs responsive to drought stress. PLoS One. 2014;9(6):e98958.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0098958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4044008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24892290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20W,%20Han%20Z,%20Guo%20Q,%20Liu%20Y,%20Zheng%20Y,%20Wu%20F,%20Jin%20W.%20Identification%20of%20maize%20long%20non-coding%20RNAs%20responsive%20to%20drought%20stress.%20PLoS%20One.%202014;9(6):e98958." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Duan Y, Xu Z, Liu H, Wang Y, Zou X, Zhang Z, Xu L, Xu M. Genome-wide identification of the TGA gene family and expression analysis under drought stress in <em>Brassica napus</em> L. Int J Mol Sci. 2024. 10.3390/ijms25126355.
</cite> [<a href="https://doi.org/10.3390/ijms25126355" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11203523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38928064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duan%20Y,%20Xu%20Z,%20Liu%20H,%20Wang%20Y,%20Zou%20X,%20Zhang%20Z,%20Xu%20L,%20Xu%20M.%20Genome-wide%20identification%20of%20the%20TGA%20gene%20family%20and%20expression%20analysis%20under%20drought%20stress%20in%20Brassica%20napus%20L.%20Int%20J%20Mol%20Sci.%202024.%2010.3390/ijms25126355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Akashi H. Gene expression and molecular evolution. Curr Opin Genet Dev. 2001;11(6):660–6.
</cite> [<a href="https://doi.org/10.1016/s0959-437x(00)00250-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11682310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akashi%20H.%20Gene%20expression%20and%20molecular%20evolution.%20Curr%20Opin%20Genet%20Dev.%202001;11(6):660%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Akashi H, Eyre-Walker A. Translational selection and molecular evolution. Curr Opin Genet Dev. 1998;8(6):688–93.
</cite> [<a href="https://doi.org/10.1016/s0959-437x(98)80038-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9914211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akashi%20H,%20Eyre-Walker%20A.%20Translational%20selection%20and%20molecular%20evolution.%20Curr%20Opin%20Genet%20Dev.%201998;8(6):688%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Wang Z, San Lucas FA, Qiu P, Liu Y. Improving the sensitivity of sample clustering by leveraging gene co-expression networks in variable selection. BMC Bioinformatics. 2014;15:153.
</cite> [<a href="https://doi.org/10.1186/1471-2105-15-153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4035826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24885641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Z,%20San%20Lucas%20FA,%20Qiu%20P,%20Liu%20Y.%20Improving%20the%20sensitivity%20of%20sample%20clustering%20by%20leveraging%20gene%20co-expression%20networks%20in%20variable%20selection.%20BMC%20Bioinformatics.%202014;15:153." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Ke SJ, Liu DK, Tu XD, He X, Zhang MM, Zhu MJ, Zhang DY, Zhang CL, Lan SR, Liu ZJ. Apostasia mitochondrial genome analysis and monocot mitochondria phylogenomics. Int J Mol Sci. 2023. 10.3390/ijms24097837.
</cite> [<a href="https://doi.org/10.3390/ijms24097837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10178136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37175542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ke%20SJ,%20Liu%20DK,%20Tu%20XD,%20He%20X,%20Zhang%20MM,%20Zhu%20MJ,%20Zhang%20DY,%20Zhang%20CL,%20Lan%20SR,%20Liu%20ZJ.%20Apostasia%20mitochondrial%20genome%20analysis%20and%20monocot%20mitochondria%20phylogenomics.%20Int%20J%20Mol%20Sci.%202023.%2010.3390/ijms24097837." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Li G, Zhang H, Lin Z, Li H, Xu G, Xu Y, Ji R, Luo W, Qiu Y, Qiu S, et al. Comparative analysis of chloroplast and mitochondrial genomes of sweet potato provides evidence of gene transfer. Sci Rep. 2024;14(1):4547.
</cite> [<a href="https://doi.org/10.1038/s41598-024-55150-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10894244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38402284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20G,%20Zhang%20H,%20Lin%20Z,%20Li%20H,%20Xu%20G,%20Xu%20Y,%20Ji%20R,%20Luo%20W,%20Qiu%20Y,%20Qiu%20S,%20et%20al.%20Comparative%20analysis%20of%20chloroplast%20and%20mitochondrial%20genomes%20of%20sweet%20potato%20provides%20evidence%20of%20gene%20transfer.%20Sci%20Rep.%202024;14(1):4547." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Chai W, Li H, Chen X, Qiu J, Xia P. The whole chloroplast genome of <em>Neomartinella yungshunensis</em> (Brassicaceae), an unusual wild plant. Mitochondr DNA B Resources. 2022;7(6):1096–8.</cite> [<a href="https://doi.org/10.1080/23802359.2022.2087543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9246043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35783054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chai%20W,%20Li%20H,%20Chen%20X,%20Qiu%20J,%20Xia%20P.%20The%20whole%20chloroplast%20genome%20of%20Neomartinella%20yungshunensis%20(Brassicaceae),%20an%20unusual%20wild%20plant.%20Mitochondr%20DNA%20B%20Resources.%202022;7(6):1096%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Lu QX, Ren LJ, Wang YN, Xiao Y, Li YD, Ying JN, Zhang Y, Wang R, Qi ZC, You ZY. Characterization of the complete chloroplast genome of <em>Sparganium glomeratum</em> (Typhaceae) from Jilin Province, China and phylogenetic analysis. Mitochondr DNA B Resources. 2021;6(11):3253–4.</cite> [<a href="https://doi.org/10.1080/23802359.2021.1991247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8530496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34693014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20QX,%20Ren%20LJ,%20Wang%20YN,%20Xiao%20Y,%20Li%20YD,%20Ying%20JN,%20Zhang%20Y,%20Wang%20R,%20Qi%20ZC,%20You%20ZY.%20Characterization%20of%20the%20complete%20chloroplast%20genome%20of%20Sparganium%20glomeratum%20(Typhaceae)%20from%20Jilin%20Province,%20China%20and%20phylogenetic%20analysis.%20Mitochondr%20DNA%20B%20Resources.%202021;6(11):3253%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12369140/bin/12870_2025_7074_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (41.4KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The calibrated sequencing data for the Illumina and Oxford Nanopore PromethION platforms have been deposited in NCBI (<a href="https://www.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/</a>) with accession numbers PRJNA1195727, SAMN45227193, SAMN45227194, SRR31706100, and SRR31706101. The mitochondrial genome of P. cyrtonema has been deposited in GeneBank (<a href="https://www.ncbi.nlm.nih.gov/genbank/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/genbank/</a>) with accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV007883:PV007903%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV007883-PV007903</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Plant Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12870-025-07074-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12870_2025_Article_7074.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12369140/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12369140/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369140%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369140/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12369140/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12369140/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841885/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12369140/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841885/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12369140/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12369140/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="BXR9vjL6Hu0LNeT8AvUwVJCFfonxsBjqTY6mA6vTWR80C5UupjIA1KMOOfSj54LB">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
