
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            An unclean slate, discrepancies between food input and recovered protein signal from experimental foodcrusts - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532E758AF21EA3052E75001D0F22EA.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373221/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="An unclean slate, discrepancies between food input and recovered protein signal from experimental foodcrusts">
<meta name="citation_author" content="Joannes Adrianus Antonius Dekker">
<meta name="citation_author_institution" content="Department of Life Sciences and Systems Biology, University of Turin, Turin, Italy">
<meta name="citation_author_institution" content="Section for Geobiology, Globe Institute, University of Copenhagen, Copenhagen, Denmark">
<meta name="citation_author_institution" content="BioArCh, Department of Archaeology, University of York, York, United Kingdom">
<meta name="citation_author" content="Richard Hagan">
<meta name="citation_author_institution" content="BioArCh, Department of Archaeology, University of York, York, United Kingdom">
<meta name="citation_author" content="Matthew Collins">
<meta name="citation_author_institution" content="Section for Geobiology, Globe Institute, University of Copenhagen, Copenhagen, Denmark">
<meta name="citation_author_institution" content="McDonald Institute for Archaeological Research, University of Cambridge, United Kingdom">
<meta name="citation_author" content="Jessica Hendy">
<meta name="citation_author_institution" content="BioArCh, Department of Archaeology, University of York, York, United Kingdom">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0330195">
<meta name="citation_doi" content="10.1371/journal.pone.0330195">
<meta name="citation_pmid" content="40845013">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373221/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373221/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373221/pdf/pone.0330195.pdf">
<meta name="description" content="Organic residues are a rich source of biomolecular information on ancient diets. In particular, foodcrusts, charred residues on ceramics, are commonly analysed for their lipid content and to a lesser extent protein in order to identify foods, ...">
<meta name="og:title" content="An unclean slate, discrepancies between food input and recovered protein signal from experimental foodcrusts">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Organic residues are a rich source of biomolecular information on ancient diets. In particular, foodcrusts, charred residues on ceramics, are commonly analysed for their lipid content and to a lesser extent protein in order to identify foods, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373221/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373221">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0330195"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0330195.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373221%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373221/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373221/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373221/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0330195" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 22;20(8):e0330195. doi: <a href="https://doi.org/10.1371/journal.pone.0330195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330195</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>An unclean slate, discrepancies between food input and recovered protein signal from experimental foodcrusts</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dekker%20JAA%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Joannes Adrianus Antonius Dekker</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Joannes Adrianus Antonius Dekker</span></h3>
<div class="p">
<sup>1</sup>Department of Life Sciences and Systems Biology, University of Turin, Turin, Italy</div>
<div class="p">
<sup>2</sup>Section for Geobiology, Globe Institute, University of Copenhagen, Copenhagen, Denmark</div>
<div class="p">
<sup>3</sup>BioArCh, Department of Archaeology, University of York, York, United Kingdom</div>
<div>Conceptualization, Formal analysis, Investigation, Methodology, Project administration, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dekker%20JAA%22%5BAuthor%5D" class="usa-link"><span class="name western">Joannes Adrianus Antonius Dekker</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hagan%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Richard Hagan</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Richard Hagan</span></h3>
<div class="p">
<sup>3</sup>BioArCh, Department of Archaeology, University of York, York, United Kingdom</div>
<div>Methodology, Resources, Software, Supervision, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hagan%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Richard Hagan</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Collins%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Matthew Collins</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Matthew Collins</span></h3>
<div class="p">
<sup>2</sup>Section for Geobiology, Globe Institute, University of Copenhagen, Copenhagen, Denmark</div>
<div class="p">
<sup>4</sup>McDonald Institute for Archaeological Research, University of Cambridge, United Kingdom</div>
<div>Funding acquisition, Project administration, Resources, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Collins%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Matthew Collins</span></a>
</div>
</div>
<sup>2,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hendy%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jessica Hendy</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jessica Hendy</span></h3>
<div class="p">
<sup>3</sup>BioArCh, Department of Archaeology, University of York, York, United Kingdom</div>
<div>Funding acquisition, Project administration, Resources, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hendy%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jessica Hendy</span></a>
</div>
</div>
<sup>3</sup>
</div>
<div class="cg p">Editor: <span class="name western">Raven Garvey</span><sup>5</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Department of Life Sciences and Systems Biology, University of Turin, Turin, Italy</div>
<div id="aff002">
<sup>2</sup>Section for Geobiology, Globe Institute, University of Copenhagen, Copenhagen, Denmark</div>
<div id="aff003">
<sup>3</sup>BioArCh, Department of Archaeology, University of York, York, United Kingdom</div>
<div id="aff004">
<sup>4</sup>McDonald Institute for Archaeological Research, University of Cambridge, United Kingdom</div>
<div id="edit1">
<sup>5</sup>University of Michigan, UNITED STATES OF AMERICA</div>
<div class="author-notes p">
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>jan@palaeome.org</span></p>
</div>
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Joannes Adrianus Antonius Dekker</span></strong>: <span class="role">Conceptualization, Formal analysis, Investigation, Methodology, Project administration, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Richard Hagan</span></strong>: <span class="role">Methodology, Resources, Software, Supervision, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Matthew Collins</span></strong>: <span class="role">Funding acquisition, Project administration, Resources, Supervision, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jessica Hendy</span></strong>: <span class="role">Funding acquisition, Project administration, Resources, Supervision, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Raven Garvey</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 3; Accepted 2025 Jul 28; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Dekker et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373221  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40845013/" class="usa-link">40845013</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Organic residues are a rich source of biomolecular information on ancient diets. In particular, foodcrusts, charred residues on ceramics, are commonly analysed for their lipid content and to a lesser extent protein in order to identify foods, culinary practices and material culture use in past populations. However, the composition of foodcrusts and the factors behind their formation are not well understood. Here we analyse proteomic data (available via ProteomeXchange with identifier PXD059930) from foodcrusts made using a series of mixtures of protein- (salmon flesh), lipid- (beef fat) and carbohydrate-rich (beetroot) foods to investigate the relationship between the biomolecular composition of the input and the recovered signal using conventional methods applied to archaeological material. Additionally, using 3D modelling we quantify the volume of foodcrust generated by different ingredient combinations The results highlight biases in the data obtained in the analyses of organic residues both in terms of identified resources reflecting the cooked foodstuffs, e.g., an overrepresentation of fish proteins, as well as with regards to the abundance of foodcrust, for example mixtures of only salmon and beef fat resulted in relatively small amounts of foodcrust, and suggest caution in interpreting the composition of residues formed from complex mixtures of foodstuffs.</p></section><section id="sec001"><h2 class="pmc_sec_title">Introduction</h2>
<p>Food residues on or in ceramic vessels are a popular target for studies on ancient diets and foodways. Their potential for revealing components of the diets of past people is demonstrated by studies on the adoption of pottery by hunter-gatherers [<a href="#pone.0330195.ref001" class="usa-link" aria-describedby="pone.0330195.ref001">1</a>] and continuity in diet under political regime change [<a href="#pone.0330195.ref002" class="usa-link" aria-describedby="pone.0330195.ref002">2</a>]. Organic residues on archaeological vessels can manifest in different ways, including calcified deposits [<a href="#pone.0330195.ref003" class="usa-link" aria-describedby="pone.0330195.ref003">3</a>], charred foodcrusts (hereafter foodcrusts) [<a href="#pone.0330195.ref004" class="usa-link" aria-describedby="pone.0330195.ref004">4</a>] and organic residues entrapped within the ceramic matrix [<a href="#pone.0330195.ref005" class="usa-link" aria-describedby="pone.0330195.ref005">5</a>], and more rarely, organic remains preserved on the surface of vessels through arid, frozen or waterlogged conditions [<a href="#pone.0330195.ref006" class="usa-link" aria-describedby="pone.0330195.ref006">6</a>–<a href="#pone.0330195.ref008" class="usa-link" aria-describedby="pone.0330195.ref008">8</a>]. These three different types of residues are formed by different processes and are affected differently by diagenesis. Of the three types of residue, we focus on foodcrusts in this study. Foodcrusts have so far been mostly analysed for their lipid content (including fats, oils and waxes). Lipid analysis is robust and has been successfully applied to materials from a wide range of preservation conditions spanning the globe and stretching back as far as at least 12 000 years ago [<a href="#pone.0330195.ref002" class="usa-link" aria-describedby="pone.0330195.ref002">2</a>,<a href="#pone.0330195.ref009" class="usa-link" aria-describedby="pone.0330195.ref009">9</a>–<a href="#pone.0330195.ref011" class="usa-link" aria-describedby="pone.0330195.ref011">11</a>]. For example, the study of lipids has been successful in studying the emergence and spread of the use of dairy products [<a href="#pone.0330195.ref012" class="usa-link" aria-describedby="pone.0330195.ref012">12</a>–<a href="#pone.0330195.ref014" class="usa-link" aria-describedby="pone.0330195.ref014">14</a>] as well as the continuity in the widespread use of marine resources through the Mesolithic into the Neolithic [<a href="#pone.0330195.ref015" class="usa-link" aria-describedby="pone.0330195.ref015">15</a>–<a href="#pone.0330195.ref018" class="usa-link" aria-describedby="pone.0330195.ref018">18</a>]. However, there are still areas for improvement. The results of compound-specific isotope analysis can be confounded by the mixture of multiple resources and the results are likely biased towards lipid-rich foodstuffs compared to lipid-poor foods, such as most plants [<a href="#pone.0330195.ref019" class="usa-link" aria-describedby="pone.0330195.ref019">19</a>].</p>
<p>The presence of plants in foodcrusts is best investigated using SEM (scanning electron microscopy) [<a href="#pone.0330195.ref020" class="usa-link" aria-describedby="pone.0330195.ref020">20</a>,<a href="#pone.0330195.ref021" class="usa-link" aria-describedby="pone.0330195.ref021">21</a>], revealing, for example, the presence of acorns in ceramic vessels in the Neolithic Netherlands [<a href="#pone.0330195.ref022" class="usa-link" aria-describedby="pone.0330195.ref022">22</a>]. Animal remains, although difficult to taxonomically resolve, can also be observed using SEM [<a href="#pone.0330195.ref020" class="usa-link" aria-describedby="pone.0330195.ref020">20</a>].</p>
<p>In theory, the disadvantages of both lipid analysis and SEM could be circumvented by protein analysis. As proteins are encoded by DNA, they inherently carry phylogenetic information, although the taxonomic specificity of a protein varies greatly between proteins. Beyond their use for taxonomic identification, some proteins can be used to identify the tissues that were cooked, as some proteins are abundant in or exclusive to specific tissues. For example, the frequently observed protein beta-lactoglobulin (BLG) is specific to dairy [<a href="#pone.0330195.ref023" class="usa-link" aria-describedby="pone.0330195.ref023">23</a>–<a href="#pone.0330195.ref026" class="usa-link" aria-describedby="pone.0330195.ref026">26</a>], and is not synthesised by humans at all. Additionally, as proteomic taxonomic identification is based on the presence of diagnostic peptide sequences, protein analysis should be able to distinguish between the different components of a food mixture. However, the application of protein analysis to foodcrusts is in its infancy [<a href="#pone.0330195.ref004" class="usa-link" aria-describedby="pone.0330195.ref004">4</a>,<a href="#pone.0330195.ref007" class="usa-link" aria-describedby="pone.0330195.ref007">7</a>,<a href="#pone.0330195.ref027" class="usa-link" aria-describedby="pone.0330195.ref027">27</a>,<a href="#pone.0330195.ref028" class="usa-link" aria-describedby="pone.0330195.ref028">28</a>] While some success has been observed [<a href="#pone.0330195.ref004" class="usa-link" aria-describedby="pone.0330195.ref004">4</a>,<a href="#pone.0330195.ref029" class="usa-link" aria-describedby="pone.0330195.ref029">29</a>], biases and challenges in this method have been noted [<a href="#pone.0330195.ref012" class="usa-link" aria-describedby="pone.0330195.ref012">12</a>]. For example, the cooking and burning process of foodcrust formation likely impacts the numbers of food proteins, which may or may not be taxonomically diagnostic. Furthermore, in the case of analysing mixtures of ingredients, it is unknown whether the foodcrust preserves an accurate reflection of the input food proteins. Protein analysis will of course favour protein-rich foods, but other foodstuffs less enriched in protein, such as plants, still contain characteristic proteins and have been identified in organic residues [<a href="#pone.0330195.ref003" class="usa-link" aria-describedby="pone.0330195.ref003">3</a>]. Whether there are additional factors beyond the initial protein content determining the likelihood of a particular protein’s survival is unclear. A recent study on protein incorporation and survival in the ceramic matrix and foodcrust tested the influence of several physico-chemical properties, but beyond a slight favour towards hydrophobic peptides, there were no clear universal trends in protein survival [<a href="#pone.0330195.ref030" class="usa-link" aria-describedby="pone.0330195.ref030">30</a>]. Beyond these general limitations, not much is known about the challenges of applying protein analysis to organic residues nor about how the theoretical advantages translate into reality.</p>
<p>This study will focus on foodcrusts and aims to clarify the strengths and limitations of protein analysis using experimental foodcrusts made of a mixture of protein-, lipid- and carbohydrate-rich foods. Specifically, we will investigate how mixtures of different foodstuffs might influence the volume of foodcrust formed and the recoverable proteome. In particular, we will examine the degree to which the ‘extractome’, the suite of recovered proteins [<a href="#pone.0330195.ref031" class="usa-link" aria-describedby="pone.0330195.ref031">31</a>], reflects the original input food mixture, in order to highlight the potential biases involved in the identification of foodstuffs in archaeological material. Additionally, this study aims to map how repeated cooking events and burial influence the extractome’s composition and the volume of generated foodcrust, adding to our understanding of the diagenesis of proteins in charred organic residues.</p>
<p>In order to maximise the relevance of the experiment’s results for archaeological samples the choice of foodstuffs is vital. As this experiment is part of a larger project focused on Mesolithic foodways in Denmark, we selected foodstuffs that would be similar to those utilised by late Danish hunter-gatherers. Secondly, as the three commonly used methods, i.e., lipid analysis, proteomics and SEM, to study organic residues all favour a different class of biomolecule we decided to select one lipid-rich, one protein-rich and one carbohydrate-rich foodstuff to mix. Using foodstuffs with vastly different nutritional compositions facilitates visualising the biases towards either carbohydrates, proteins or lipids. Therefore, care was taken to ensure that the relative proportions of the wet weights of the carbohydrate-rich, protein-rich and lipid-rich foodstuffs matched as closely as possible to the ratio of carbohydrates:proteins:lipids in the total mixture (e.g., a mixture consisting of 20% of the protein-rich food would also contain 20% proteins). Perfect symmetry between the ratios of wet weights and the molecular categories was not possible, especially not when limited to foodstuffs likely available in Mesolithic Denmark, but the closest match was obtained using salmon, beef fat and beetroot (SI 1). However, we acknowledge that this combination of foodstuffs underrepresents the carbohydrate component in particular.</p>
<p>Beetroot was chosen as the carbohydrate-rich food due to the prevalence of its wild ancestor <em>Beta vulgaris maritima</em> at Mesolithic sites such as Tybrind Vig and Møllegabet II [<a href="#pone.0330195.ref032" class="usa-link" aria-describedby="pone.0330195.ref032">32</a>,<a href="#pone.0330195.ref033" class="usa-link" aria-describedby="pone.0330195.ref033">33</a>], as well as the fact that it has a relatively well-annotated reference proteome. Protein identification is heavily dependent on the quality and coverage of reference proteomes. Low-quality reference proteomes only contain a fraction of the total proteome of a species and any proteins not included cannot be identified. To highlight the magnitude of this issue, the median number of proteins listed on UniProt [<a href="#pone.0330195.ref034" class="usa-link" aria-describedby="pone.0330195.ref034">34</a>] for 29 plant species found at Mesolithic northern European sites [<a href="#pone.0330195.ref032" class="usa-link" aria-describedby="pone.0330195.ref032">32</a>,<a href="#pone.0330195.ref035" class="usa-link" aria-describedby="pone.0330195.ref035">35</a>–<a href="#pone.0330195.ref038" class="usa-link" aria-describedby="pone.0330195.ref038">38</a>] is 120 proteins (SI 2), in contrast to the 7679 available for <em>Beta vulgaris</em> or the 130 673 for <em>Triticum aestivum</em> (wheat), a high-quality reference proteome.</p>
<p>We selected beef fat and salmon because the exploitation of aurochs (<em>Bos primigenius</em>) and salmon (<em>Salmo salar</em>) is well attested at a number of Danish late Mesolithic sites [<a href="#pone.0330195.ref036" class="usa-link" aria-describedby="pone.0330195.ref036">36</a>,<a href="#pone.0330195.ref039" class="usa-link" aria-describedby="pone.0330195.ref039">39</a>–<a href="#pone.0330195.ref042" class="usa-link" aria-describedby="pone.0330195.ref042">42</a>]. Additionally, salmon has a well-annotated reference proteome covering 90% of its total proteome. Available reference data for aurochs is much more limited, but due to the short evolutionary distance between them, domestic cattle (<em>Bos taurus</em>), which has a 97% complete reference proteome, was used.</p>
<p>By varying these three ingredients in regular increments we are able to establish the presence of any biases favouring or disadvantaging the recovery of proteins from any of the foods. In turn, this new insight on the presence of biases will facilitate the interpretation of proteins identified in archaeological foodcrusts and thus our understanding of past foodways. The analyses presented here demonstrate that, as expected, there is a bias towards the recovery of proteins from protein-rich foods and that it is likely that not all ingredients can be identified from a food mixture.</p></section><section id="sec002"><h2 class="pmc_sec_title">Materials</h2>
<p>The ceramic vessels used in the cooking experiments were purchased from Potted History, a company specialising in historical ceramic replicas. The specifications of the ceramics are in line with the pottery used in the experiments by Bondetti et al. [<a href="#pone.0330195.ref043" class="usa-link" aria-describedby="pone.0330195.ref043">43</a>]: they were hand thrown with the ‘standard red’ clay (Al2O3, 22.78; Fe2O3, 7.37; CaO, 0.57; MgO, 0.86; K2O, 1.6; and Na2O, 0.1), had a diameter of ~10 cm, a wall thickness of ~0.5 cm and were fired at 700 °C. The foodstuffs used in the experiments were all obtained from local commercial vendors; the beetroot was purchased from Morrisons, the beef fat from M&amp;K Quality Butchers, York, and the salmon was purchased online from LondonGrocery.net (<a href="https://londongrocery.net/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://londongrocery.net/</a>). As what is often sold as ‘salmon’ can actually belong to a variety of species and even genera, the purchased specimens were examined for morphological characteristics [<a href="#pone.0330195.ref044" class="usa-link" aria-describedby="pone.0330195.ref044">44</a>] to ensure that they were in fact <em>Salmo salar</em>.</p></section><section id="sec003"><h2 class="pmc_sec_title">Methods</h2>
<section id="sec004"><h3 class="pmc_sec_title">Cooking experimental protocol</h3>
<p>In order to test how different combinations of carbohydrate-, protein- and lipid-rich foodstuffs influence foodcrust formation it was decided to mix the three selected foodstuffs, beetroot, salmon and beef fat, in all possible combinations using 20% increments, which results in 21 sample conditions (<a href="#pone.0330195.g001" class="usa-link">Fig 1</a>). To each of these unique combinations one ceramic vessel was assigned. Additionally, we included one ceramic vessel as a negative control, which would only be filled with water during cooking. No samples were taken of the negative control for proteomic analysis, as indeed no foodcrust formed on the water filled vessel. Instead, an extraction blank was included in the protein extraction process to serve as negative control.</p>
<figure class="fig xbox font-sm" id="pone.0330195.g001"><h4 class="obj_head">Fig 1. Overview of experimental conditions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373221_pone.0330195.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b934/12373221/2bb11c28da0a/pone.0330195.g001.jpg" loading="lazy" height="540" width="675" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330195.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Each red node indicates an experimental vessel.</p></figcaption></figure><p>Before the first cooking event, all pots were soaked in tap water at room temperature for 15 minutes in order to prevent cracking when heated. The vessels were then filled with the ingredients according to their experimental conditions to a total mass of 100 g. Additionally, 100 mL of distilled water was added to each vessel. The filled vessels were wrapped in aluminium foil, leaving the top open, and placed for an hour in an oven (Binder) preheated to 270°C. The aluminium foil was intended to hold the pots and their contents in place in the event of the ceramic fracturing, as pilot tests had shown this to be a possibility. The cooking temperature of 270°C was selected based on previous research that found that APAAs (ω-(O-alkylphenyl) alkanoic acids) often found in archaeological foodcrusts, formed after heating for one hour at 270°C [<a href="#pone.0330195.ref043" class="usa-link" aria-describedby="pone.0330195.ref043">43</a>]. Indirect heating using a convection oven was chosen in order to ensure a stable and even temperature for all vessels and throughout the cooking event. Direct heating is likely more representative of cooking during Mesolithic times, but introduces substantial variability in the precise temperatures the food is exposed to both during a cooking event as well as between different cooking events and between different vessels within the same cooking event. As we expected temperature to play an important role in the formation of foodcrust, it was preferred to keep it as stable as possible.</p>
<p>After cooking for one hour, the vessels were removed from the oven and allowed to cool down at room temperature. The cooled foodstuffs were then removed from the vessels as we were interested in multiple cooking events. Care was taken to only remove the foodstuffs and to leave the foodcrusts intact. Charred remains that adhered to the ceramic wall, but were macroscopically still recognisable as food, were removed as well. <a href="#pone.0330195.g002" class="usa-link">Fig 2</a> provides some examples of foodcrusts that were formed after the final cooking event.</p>
<figure class="fig xbox font-sm" id="pone.0330195.g002"><h4 class="obj_head">Fig 2. Foodcrusts formed after the ninth cooking event.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373221_pone.0330195.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b934/12373221/d19be1a98cb5/pone.0330195.g002.jpg" loading="lazy" height="737" width="749" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330195.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The number denotes the experimental of each vessel and corresponds to <a href="#pone.0330195.g001" class="usa-link">Fig 1</a>.</p></figcaption></figure><p>After removing the food, the vessels were upturned for roughly an hour to allow any liquid to drip out. Any fat that consolidated on a vessel’s rim during this period was not considered foodcrust and removed. For subsequent cooking episodes, the vessels were no longer soaked in water. Instead, the vessels were immediately placed in the oven after filling them with their ingredients and 100 mL water. In total nine cooking events were performed for each pot and foodcrust samples were taken after the first, fifth and ninth cooking episodes. Foodcrust could form on the rim, body and bottom of the vessel depending on the food input, but the rim of the vessel was the only location that always contained sufficient foodcrust for sampling, and hence foodcrust was consistently sampled from the rim. Where possible samples were still taken from the bottom and body of the vessel.</p>
<p>After the ninth cooking episode, the vessels were buried with their foodcrusts in the York Experimental Archaeological Research (YEAR) Centre for 52 days at a depth of 50 cm. The sediment the vessels were buried in was sandy and the soil pH of the same area was previously measured to be 7.16 [<a href="#pone.0330195.ref043" class="usa-link" aria-describedby="pone.0330195.ref043">43</a>]. For the months of August and September, during which the burial experiment was carried out, the Met Office National Climate Information Centre reported a mean monthly air temperature of 16.9 °C and 13.5 °C for North England respectively [<a href="#pone.0330195.ref045" class="usa-link" aria-describedby="pone.0330195.ref045">45</a>]. The vessels were wrapped loosely in Galvanised wire netting (13 mm mesh size) to prevent bioturbation from any larger creatures. After the burial period had passed, the vessels were excavated and a foodcrust sample was taken from each vessel. All samples were stored at −20°C until protein extraction.</p></section><section id="sec005"><h3 class="pmc_sec_title">Protein extraction</h3>
<p>Foodcrust protein analysis was performed according to a variant of the SP3 protocol [<a href="#pone.0330195.ref046" class="usa-link" aria-describedby="pone.0330195.ref046">46</a>] developed by Dr. Virginia Harvey and Dr. Jessica Hendy [<a href="#pone.0330195.ref047" class="usa-link" aria-describedby="pone.0330195.ref047">47</a>]. In short, 150 μL 6 M guanidine hydrochloride (GuHCl) was added to 20 mg foodcrust, vortexed, centrifuged and incubated at 65°C for one hour. Subsequently, the samples were centrifuged and the foodcrust pellets were discarded. 15 μL of a 100 mM TCEP (Tris(2-carboxyethyl)phosphine) and CAA (2-Chloroacetamide) solution was added to the supernatant, homogenised, centrifuged and incubated at 99°C for 10 minutes. After the incubation, the samples were allowed to cool in room temperature conditions, after which 500 μg of magnetic hydrophilic and hydrophobic beads (Cytiva) and 175 μL of 100% (v/v) ethanol were added to each sample. The samples were then incubated again at 24°C, 1000 RPM for 5 minutes. Protein filtration using the beads was performed by placing the samples on a magnetic rack and allowing the beads to migrate for 2 minutes. The liquid in the samples was then removed until only the beads remained. Samples were taken off the rack and were rinsed with 80% ethanol. This process was repeated three times, the first time 500 μL ethanol was used for the rinsing, the second time 300 μL and the third rinse was done with 200 μL. After the final rinse, the beads were resuspended in 100 μL 50 mM ammonium bicarbonate (ABC). This solution was incubated at 37°C with 750 RPM for 3 minutes, after which the samples were placed back on the magnetic rack. In-solution digestion was performed by adding 1 μL of a 0.2 μg/μL trypsin solution to the samples and incubating them at 37°C, 750 RPM for 18 hours.</p>
<p>After digestion, the samples were centrifuged and placed back on the magnetic rack. Beads and suspended proteins were separated by transferring the solution to another set of Eppendorf tubes. 10 μL of 5% (v/v) trifluoroacetic acid (TFA) was added to the samples to halt digestion. As a last step, the peptides were desalted with C18 ZipTips. The ZipTips were primed twice with a 100 μL 50% acetonitrile (ACN) and 0.1% TFA solution and washed twice with 0.1% TFA. Then the peptides were aspirated from the sample and the ZipTips were washed again twice with 0.1% TFA before eluting the peptides in 50% ACN and 0.1% TFA.</p></section><section id="sec006"><h3 class="pmc_sec_title">Protein mass spectrometry analysis</h3>
<p>The eluted peptides were analysed using an mClass nanoflow UPLC (Ultra Performance Liquid Chromatography) system (Waters) coupled to an Orbitrap Fusion Tribrid mass spectrometer (Thermo Scientific) following the same protocol as published elsewhere [<a href="#pone.0330195.ref048" class="usa-link" aria-describedby="pone.0330195.ref048">48</a>]. In short, the UPLC was equipped with a nanoEaze M/Z Symmetry 100 Å C18, 5 μm trap column (180 μm x 20 mm, Waters) and a PepMap, 2 μm, 100 Å, C18 EasyNano nanocapillary column (75 μm x 500 mm, Thermo Scientific). Chromatographic separation of the peptides was performed by first washing the peptides from the trap column onto the capillary column with 0.05% trifluoroacetic acid aqueous solvent at a flow rate of 15 μL/min. After five minutes the flow was switched to the capillary column, which was first washed for seven minutes with solvent A (0.1% FA (formic acid)) and 3–10% solvent B (0.1% FA in acetonitrile). The concentration of solution B in the mixture was steadily increased over time. After these initial seven minutes, the concentration of solvent B was increased to 35% over 30 minutes and lastly to 99% over five minutes. Finally, the column was washed with 99% solvent B for four minutes.</p>
<p>As the peptides were eluted off the column they were introduced into the mass spectrometer, which was equipped with an EasyNano ionisation source (Thermo Scientific). Xcalibur (version 4.0, Thermo Scientific) was used to acquire ESI-MS and MS2 spectra both in positive mode. The following instrument source settings were used: ion spray voltage, 1,900 V; sweep gas, 0 Arb; ion transfer tube temperature; 275°C. MS1 spectra were acquired in the Orbitrap with: 120,000 resolution, scan range: m/z 375−1,500; AGC target, 4e5; max fill time, 100 ms. Acquisition was data dependent (DDA) and performed in topN mode selecting the 12 most intense precursors with charge states &gt;1. Easy-IC was used for internal calibration. Dynamic exclusion was performed for 50 s post precursor selection and a minimum threshold for fragmentation was set at 5e3. MS2 spectra were acquired in the Orbitrap with: 30,000 resolution, max fill time, 100 ms., HCD; activation energy: 32 NCE. The mass spectrometry proteomics data have been deposited to the ProteomeXchange Consortium via the PRIDE 64 partner repository with the dataset identifier PXD059930 and 10.6019/PXD059930.</p></section><section id="sec007"><h3 class="pmc_sec_title">Protein data analysis</h3>
<p>The resulting raw files were analysed with MetaMorpheus [<a href="#pone.0330195.ref049" class="usa-link" aria-describedby="pone.0330195.ref049">49</a>] using the following settings: files were calibrated with a product mass tolerance of 25 ppm and a precursor tolerance of 15 ppm. A G-PTM-D (Global-Post Translational Modification Discovery) task was performed in order to identify the most common post-translational modifications in the samples. The following PTM groups were included in the G-PTM-D task: common biological (34 PTMs), common artefact (10 PTMs), metal (16 PTMs) and trypsin digested (2 PTMs). The precise PTMs that were included can be found in SI 3. To quantify PTM presence occupancy, that is the fraction of a protein that displays a given PTM at a specific position in the protein sequence [<a href="#pone.0330195.ref050" class="usa-link" aria-describedby="pone.0330195.ref050">50</a>], was calculated automatically at protein level by MetaMorpheus. For the search task a product mass tolerance of 0.5 Da and precursor mass tolerance of 3 ppm was maintained. The maximum number of missed cleavages was 2 and the minimum peptide length was set to 7. Only fully tryptic peptides were accepted.</p>
<p>Using these settings the samples were searched against a custom database consisting of reference proteomes of <em>Beta vulgaris, Chenopodium quinoa, Spinacia oleracea, Salmo salar</em> and <em>Bos taurus</em>, which were downloaded from the Uniprot protein database [<a href="#pone.0330195.ref034" class="usa-link" aria-describedby="pone.0330195.ref034">34</a>]. In addition to the proteomes of the species used in the cooking experiments, it was also decided to include quinoa (<em>Chenopodium quinoa</em>) and spinach (<em>Spinacia oleracea</em>), because of the limited coverage of the <em>Beta vulgaris</em> reference proteome. We feared that our analysis would bias against <em>Beta vulgaris</em> and therefore decided to include the two species with the largest reference proteomes from the Amaranthaceae family, which includes <em>Beta vulgaris</em>. All five proteomes mentioned above were downloaded from UniProt on 03-03-2023 [<a href="#pone.0330195.ref034" class="usa-link" aria-describedby="pone.0330195.ref034">34</a>]. Lastly, we included MetaMorpheus’ inbuilt database of contaminants, cRAP [<a href="#pone.0330195.ref051" class="usa-link" aria-describedby="pone.0330195.ref051">51</a>], in our search. The exact task files and protein database used in this analysis are included in SI 3.</p>
<p>The MetaMorpheus protein identifications were then filtered following the recommendations for palaeoproteomic identifications [<a href="#pone.0330195.ref052" class="usa-link" aria-describedby="pone.0330195.ref052">52</a>], rejecting all proteins which were present in the blanks or which were represented by less than two unique peptides.</p>
<p>Statistical analysis was performed in R <em>v4.1.1</em> [<a href="#pone.0330195.ref053" class="usa-link" aria-describedby="pone.0330195.ref053">53</a>] using the Rstudio environment <em>v2024.4.0.735</em> [<a href="#pone.0330195.ref054" class="usa-link" aria-describedby="pone.0330195.ref054">54</a>].</p></section><section id="sec008"><h3 class="pmc_sec_title">Foodcrust quantification</h3>
<p>In order to investigate the influence of ingredient composition on the amount of foodcrust generated after cooking we quantified the volume of foodcrust using 3D scanning. A 3D scan of each ceramic vessel was acquired before the first cooking event and after the ninth using an Artec Space Spider (Artec 3D). 3D model construction and further analysis of the 3D models were all performed in Artec Studio 16 Professional v16.0.8.2. Volume measurements were obtained by combining the scans for each vessel and time point using the ‘Sharp fusion’ function. Then the volume of the entire 3D model was calculated using the in-built volume calculation function. The foodcrust volume was calculated by subtracting the before-cooking 3D model volume from the after-cooking 3D model volume.</p>
<p>Vessels 1, 7, 11 and 21 were damaged during the cooking process and some fragments had detached from the ceramic wall. To prevent the loss of ceramic fragments from influencing the foodcrust quantification the ceramic wall from which the fragment had detached were deleted from both the before and after model. The resulting hole in the 3D models was closed using the ‘Fix holes’ function. Irregularities in the 3D model resulting from this intervention were resolved using the ‘Small-object filter’ and ‘Hole filling’ ‘Postprocessing’ functions. Foodcrust volume calculations could then be obtained in the same way as for the undamaged experiments.</p></section></section><section id="sec009"><h2 class="pmc_sec_title">Results</h2>
<p>A total list of all proteins recovered from the samples can be found in SI 4. There appeared to be substantial differences in the number of proteins that could be identified between the different experimental conditions and the number of cooking events for a particular ingredient combination (<a href="#pone.0330195.g003" class="usa-link">Fig 3</a>). Additionally, most samples show large differences in the contribution of the different ingredients to the total amount of recovered proteins. Some samples, 6.5, 7.1, 12.1, 12.9, 15.1 and 21.1, also yielded proteins from a species not included in their mixture (<a href="#pone.0330195.t001" class="usa-link">Table 1</a>).</p>
<figure class="fig xbox font-sm" id="pone.0330195.g003"><h3 class="obj_head">Fig 3. Number of proteins identified in each experimental condition split up by number of cooking events and taxon.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373221_pone.0330195.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b934/12373221/71f913974805/pone.0330195.g003.jpg" loading="lazy" height="691" width="750" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330195.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>
PE indicates the post-excavation sample. <em>Beta vulgaris</em>, <em>Chenopodium quinoa</em> and <em>Spinacia oleracea</em> are grouped together as ‘plant proteins’. Each square represents an identified protein.
</p></figcaption></figure><section class="tw xbox font-sm" id="pone.0330195.t001"><h3 class="obj_head">Table 1. Identified proteins belonging to taxa not included in the sample’s mixture.</h3>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Experiment number</th>
<th align="left" rowspan="1" colspan="1">% Beetroot</th>
<th align="left" rowspan="1" colspan="1">% Salmon</th>
<th align="left" rowspan="1" colspan="1">% Beef fat</th>
<th align="left" rowspan="1" colspan="1">Cooking event</th>
<th align="left" rowspan="1" colspan="1">Protein Accession</th>
<th align="left" rowspan="1" colspan="1">Protein Full Name</th>
<th align="left" rowspan="1" colspan="1">Sequence Coverage</th>
<th align="left" rowspan="1" colspan="1">Number of Unique Peptides</th>
<th align="left" rowspan="1" colspan="1">Organism</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">100</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">A0A3Q1LL35</td>
<td align="left" rowspan="1" colspan="1">Uncharacterized protein</td>
<td align="left" rowspan="1" colspan="1">25%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Bos taurus</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">7</td>
<td align="left" rowspan="3" colspan="1">80</td>
<td align="left" rowspan="3" colspan="1">20</td>
<td align="left" rowspan="3" colspan="1">0</td>
<td align="left" rowspan="3" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">A0A1S2X522</td>
<td align="left" rowspan="1" colspan="1">phosphopyruvate hydratase</td>
<td align="left" rowspan="1" colspan="1">63%</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B5DG55</td>
<td align="left" rowspan="1" colspan="1">Alpha-1,4 glucan phosphorylase</td>
<td align="left" rowspan="1" colspan="1">48%</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q3ZLR1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q3ZLR1</a></td>
<td align="left" rowspan="1" colspan="1">Superoxide dismutase [Cu-Zn]</td>
<td align="left" rowspan="1" colspan="1">23%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">12</td>
<td align="left" rowspan="3" colspan="1">60</td>
<td align="left" rowspan="3" colspan="1">0</td>
<td align="left" rowspan="3" colspan="1">40</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">C0H9B8</td>
<td align="left" rowspan="1" colspan="1">Trifunctional enzyme subunit beta, mitochondrial</td>
<td align="left" rowspan="1" colspan="1">9%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">A0A1S3Q7E3</td>
<td align="left" rowspan="1" colspan="1">collagen alpha-1(I) chain-like</td>
<td align="left" rowspan="1" colspan="1">10%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A1S3SM10</td>
<td align="left" rowspan="1" colspan="1">collagen alpha-1(I) chain-like</td>
<td align="left" rowspan="1" colspan="1">8%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">60</td>
<td align="left" rowspan="1" colspan="1">40</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q9LEE0" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9LEE0</a></td>
<td align="left" rowspan="1" colspan="1">phosphopyruvate hydratase</td>
<td align="left" rowspan="1" colspan="1">15%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Spinacia oleracea</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">21</td>
<td align="left" rowspan="3" colspan="1">0</td>
<td align="left" rowspan="3" colspan="1">0</td>
<td align="left" rowspan="3" colspan="1">100</td>
<td align="left" rowspan="3" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">A0A0J7YLM1</td>
<td align="left" rowspan="1" colspan="1">Elongation factor Tu (Fragment)</td>
<td align="left" rowspan="1" colspan="1">13%</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">Beta vulgaris subsp. vulgaris</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A0J8B1E8</td>
<td align="left" rowspan="1" colspan="1">50S ribosomal protein L5, chloroplastic</td>
<td align="left" rowspan="1" colspan="1">14%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Beta vulgaris subsp. vulgaris</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A0K9Q7T6</td>
<td align="left" rowspan="1" colspan="1">S1 motif domain-containing protein</td>
<td align="left" rowspan="1" colspan="1">4%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Spinacia oleracea</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330195.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p><a href="#pone.0330195.g003" class="usa-link">Fig 3</a> also shows that for most experimental conditions the number of protein identifications decreases as the number of cooking events increases, which fits with the expected increase in protein damage due to extended heating. To further explore this pattern we plotted the distribution of protein identifications per sample point (<a href="#pone.0330195.g004" class="usa-link">Fig 4</a>). For each sample we also calculated the protein input in grams based on the ingredients’ protein content [<a href="#pone.0330195.ref055" class="usa-link" aria-describedby="pone.0330195.ref055">55</a>]. As expected the number of proteins identified decreases after the first sampling point and is lowest after the burial, but some of the samples obtained after the ninth cooking event contain more proteins than any sample taken after the fifth event. To test if these differences are statistically significant, a Kruskal Wallis test [<a href="#pone.0330195.ref056" class="usa-link" aria-describedby="pone.0330195.ref056">56</a>] was performed (p = 4.183e<sup>-8</sup>), which indeed suggests that there are significant differences between the number of proteins recovered after the different cooking events. Post hoc analysis using Dunn’s test [<a href="#pone.0330195.ref057" class="usa-link" aria-describedby="pone.0330195.ref057">57</a>,<a href="#pone.0330195.ref058" class="usa-link" aria-describedby="pone.0330195.ref058">58</a>] shows that there are significant differences between cooking events 1 and 5 (p = 5.46*10<sup>−5</sup>), 1 and 9 (p = 1.08*10<sup>−3</sup>), 1 and PE (5.78*10<sup>−7</sup>) and lastly between events 9 and PE (p = 0.0497). The seeming difference in protein recovery between events 5 and 9 is therefore not statistically significant.</p>
<figure class="fig xbox font-sm" id="pone.0330195.g004"><h3 class="obj_head">Fig 4. Distribution of the number of proteins identified per sampling point.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373221_pone.0330195.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b934/12373221/109a485cf5ca/pone.0330195.g004.jpg" loading="lazy" height="565" width="754" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330195.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>In general, the number of identified proteins decreases with increasing cooking events, with two vessels acting as outliers.</p></figcaption></figure><p>To further investigate the relationship between the observed proteome abundance and the cooked ingredients we processed the identified peptides in the following way: first, we filtered the identified peptides to only include those that were observed in the single-species samples (i.e., experiment numbers 1, 6 and 21). As the number of identified peptides, let alone shared-peptides, decreases rapidly with increased cooking, we only included samples from the first cooking event. For each protein in each sample, we then selected the three peptides with the highest intensities (<a href="#pone.0330195.g005" class="usa-link">Fig 5</a>) and summed their intensities per sample, giving a summed intensity as a proxy of their protein abundance. Using these summed intensities as the response variable we created a linear regression model with the nutritional composition as predictors. In the first model we split the fat content of each sample by saturated, mono-unsaturated and poly-unsaturated fatty acids, while in the second model, we grouped these three fatty acid subcategories together. Both models showed a significant relationship (R<sup>2</sup> = 0.7765, F(2,18) = 35.75), the total protein (p = 1.16e<sup>-6</sup>) and carbohydrate (p = 4.84e<sup>-6</sup>) content, in particular, showed a significant positive correlation with the observed sum of intensities.</p>
<figure class="fig xbox font-sm" id="pone.0330195.g005"><h3 class="obj_head">Fig 5. Distribution of the intensity of the three most intense peptides per protein.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373221_pone.0330195.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b934/12373221/75bb17ec7554/pone.0330195.g005.jpg" loading="lazy" height="695" width="723" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330195.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The y-axis shows the log 10 of intensity. For the ingredient combination of each experiment number see <a href="#pone.0330195.g001" class="usa-link">Fig 1</a>.</p></figcaption></figure><p>The same approach also allowed us to identify the proteins that were most abundant across our samples after the first cooking event (<a href="#pone.0330195.t002" class="usa-link">Table 2</a>). Of the ten most intense proteins, eight belong to <em>Salmo salar</em> and the remaining two are from <em>Bos taurus</em>. No plant proteins feature among the most intense proteins. Interestingly, L-lactate dehydrogenase, number nine in the top ten, was only identified in a single sample (vessel 6), which was filled with 100% salmon. Additionally, <em>Salmo salar</em> collagen alpha-1(I) is listed twice in the top ten with two different accession numbers. An alignment of the complete sequences of these two proteins via Clustal [<a href="#pone.0330195.ref059" class="usa-link" aria-describedby="pone.0330195.ref059">59</a>] shows a 93.68% similarity in sequence between the two, indicating that there are a number of differences in the sequences. The UniProt entries of the two proteins indicate that they are both automatically predicted from genomic data, albeit from different gene identifiers [<a href="#pone.0330195.ref034" class="usa-link" aria-describedby="pone.0330195.ref034">34</a>](accessed on 08-07-2025).</p>
<section class="tw xbox font-sm" id="pone.0330195.t002"><h3 class="obj_head">Table 2. Top 10 most intense proteins summed across all samples from cooking event 1.</h3>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Protein accession</th>
<th align="left" rowspan="1" colspan="1">Protein full name</th>
<th align="left" rowspan="1" colspan="1">Taxon</th>
<th align="left" rowspan="1" colspan="1">Average sequence coverage (%)</th>
<th align="left" rowspan="1" colspan="1">Number of samples protein was present in</th>
<th align="left" rowspan="1" colspan="1">Summed intensity</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">B9EP57</td>
<td align="left" rowspan="1" colspan="1">Troponin C, skeletal muscle</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">61%</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">1.47E + 10</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A1S2X522</td>
<td align="left" rowspan="1" colspan="1">Phosphopyruvate hydratase</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">77%</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">9.99E + 09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P02453" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P02453</a></td>
<td align="left" rowspan="1" colspan="1">Collagen alpha-1(I) chain</td>
<td align="left" rowspan="1" colspan="1">Bos taurus</td>
<td align="left" rowspan="1" colspan="1">35%</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">5.31E + 09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q7ZZN0" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q7ZZN0</a></td>
<td align="left" rowspan="1" colspan="1">Myosin regulatory light chain 2</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">84%</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">4.64E + 09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B5DG55</td>
<td align="left" rowspan="1" colspan="1">Alpha-1,4 glucan phosphorylase</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">57%</td>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">4.34E + 09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B5DGL9</td>
<td align="left" rowspan="1" colspan="1">Fructose-bisphosphatase</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">48%</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">3.37E + 09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A1S3Q7E3</td>
<td align="left" rowspan="1" colspan="1">Collagen alpha-1(I) chain-like</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">34%</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">2.72E + 09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P02070" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P02070</a></td>
<td align="left" rowspan="1" colspan="1">Hemoglobin subunit beta</td>
<td align="left" rowspan="1" colspan="1">Bos taurus</td>
<td align="left" rowspan="1" colspan="1">80%</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">2.65E + 09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A1S3MJX7</td>
<td align="left" rowspan="1" colspan="1">L-lactate dehydrogenase</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">82%</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">1.88E + 09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A1S3SM10</td>
<td align="left" rowspan="1" colspan="1">Collagen alpha-1(I) chain-like</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">24%</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">1.41E + 09</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330195.t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The quantification strategy described above is limited by the severe cooking- and burial-induced degradation of the proteins. The number of proteins that survive all cooking events and the burial is limited across all conditions of mixed foods (<a href="#pone.0330195.g003" class="usa-link">Fig 3</a>). Out of the total 623 proteins that were identified across all the cooking experiments, 13 are present in samples from both the first, fifth, ninth cooking event and post-burial (<a href="#pone.0330195.t003" class="usa-link">Table 3</a>). Remarkably, of these 13 persistent proteins, three belong to plant species, while none of the <em>B. taurus</em> proteins were found in samples from each cooking event. An overview of all identified proteins and the number of samples they were recovered in per cooking event (SI 5) shows that there are three additional salmon proteins that were recovered from the post-excavation samples but were not observed in samples from earlier time points. Two, isocitrate dehydrogenase (B5DGS2) and aspartate aminotransferase (B5X142), were not observed in the sample taken after the fifth cooking event, while myosin-7B (A0A1S3QIK3) was not detected after the ninth cooking event. Additionally, some proteins appear to occur consistently and frequently throughout the experiment until the burial, after which they disappear. A notable example is two collagen alpha-1(I)-like entries, A0A1S3Q7E3 and A0A1S3SM10, which are found in 9–11 samples after the first, fifth and ninth cooking event, but in none after burial.</p>
<section class="tw xbox font-sm" id="pone.0330195.t003"><h3 class="obj_head">Table 3. Uniprot accession number, full name and associated species of the proteins found in at least one rim sample from each sampling point. The values indicate the number of samples each protein was identified in per sampling point.</h3>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Protein Accession</th>
<th align="left" rowspan="1" colspan="1">Protein Full Name</th>
<th align="left" rowspan="1" colspan="1">Organism</th>
<th align="left" rowspan="1" colspan="1">1</th>
<th align="left" rowspan="1" colspan="1">5</th>
<th align="left" rowspan="1" colspan="1">9</th>
<th align="left" rowspan="1" colspan="1">PE</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">B5DG55</td>
<td align="left" rowspan="1" colspan="1">Alpha-1,4 glucan phosphorylase</td>
<td align="left" rowspan="1" colspan="1">
<em>Salmo salar</em>
</td>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">4</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A0J8E469</td>
<td align="left" rowspan="1" colspan="1">rRNA N-glycosylase</td>
<td align="left" rowspan="1" colspan="1">
<em>Beta vulgaris</em> subsp. <em>vulgaris</em>
</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B5DFU7</td>
<td align="left" rowspan="1" colspan="1">AMP deaminase</td>
<td align="left" rowspan="1" colspan="1">
<em>Salmo salar</em>
</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A1S3PV75</td>
<td align="left" rowspan="1" colspan="1">Glucose-6-phosphate isomerase</td>
<td align="left" rowspan="1" colspan="1">
<em>Salmo salar</em>
</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A1S3MVJ3</td>
<td align="left" rowspan="1" colspan="1">ATP synthase subunit beta</td>
<td align="left" rowspan="1" colspan="1">
<em>Salmo salar</em>
</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q7ZZN0" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q7ZZN0</a></td>
<td align="left" rowspan="1" colspan="1">Myosin regulatory light chain 2</td>
<td align="left" rowspan="1" colspan="1">
<em>Salmo salar</em>
</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B5X4K4</td>
<td align="left" rowspan="1" colspan="1">L-lactate dehydrogenase</td>
<td align="left" rowspan="1" colspan="1">
<em>Salmo salar</em>
</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A1S3KNQ3</td>
<td align="left" rowspan="1" colspan="1">2-iminobutanoate/2-iminopropanoate deaminase</td>
<td align="left" rowspan="1" colspan="1">
<em>Salmo salar</em>
</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B5DFX8</td>
<td align="left" rowspan="1" colspan="1">Phosphoglycerate kinase</td>
<td align="left" rowspan="1" colspan="1">
<em>Salmo salar</em>
</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B5DGZ1</td>
<td align="left" rowspan="1" colspan="1">ADP/ATP translocase</td>
<td align="left" rowspan="1" colspan="1">
<em>Salmo salar</em>
</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A0J8B2U8</td>
<td align="left" rowspan="1" colspan="1">Uncharacterized protein</td>
<td align="left" rowspan="1" colspan="1">
<em>Beta vulgaris</em> subsp. <em>vulgaris</em>
</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A803LHN7</td>
<td align="left" rowspan="1" colspan="1">Alcohol dehydrogenase</td>
<td align="left" rowspan="1" colspan="1">
<em>Chenopodium quinoa</em>
</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B5XDB2</td>
<td align="left" rowspan="1" colspan="1">D-dopachrome decarboxylase</td>
<td align="left" rowspan="1" colspan="1">
<em>Salmo salar</em>
</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330195.t003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The above results highlight the challenges in linking the abundance of the identified proteins with the species used in the cooking, which was one of the prime aims of this study. Especially, as of the most intense proteins after the first cooking event (<a href="#pone.0330195.t002" class="usa-link">Table 2</a>), only alpha-1,4 glucan phosphorylase (B5DG55) and myosin regulatory light chain 2 (<a href="https://www.ncbi.nlm.nih.gov/protein/Q7ZZN0" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q7ZZN0</a>) could be recovered in samples from all cooking events, highlighting that there are substantial differences in the impact of degradation on particular proteins. Due to the differences in chemical properties between proteins, such variation is not wholly unexpected. Recently, hydrophobicity (refs) has been gaining attention as a potentially important factor in protein survival. Therefore, we calculated the hydrophobicity of the observed fraction of our identified proteins (<a href="#pone.0330195.g006" class="usa-link">Fig 6</a>) using the whole residue values as determined by Wimley and White [<a href="#pone.0330195.ref060" class="usa-link" aria-describedby="pone.0330195.ref060">60</a>].</p>
<figure class="fig xbox font-sm" id="pone.0330195.g006"><h3 class="obj_head">Fig 6. Hydrophobicity of the identified fraction of each protein for each experimental vessel split by cooking event.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373221_pone.0330195.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b934/12373221/6748f7711af2/pone.0330195.g006.jpg" loading="lazy" height="463" width="750" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330195.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Each dot represents a single protein and is coloured by its taxonomic origin. Hydrophobicity was calculated using amino acid hydrophobicity values following Wimley and White [<a href="#pone.0330195.ref060" class="usa-link" aria-describedby="pone.0330195.ref060">60</a>].</p></figcaption></figure><p>It is clear that the overwhelming majority of identified proteins are hydrophilic, or at least the part of the protein sequence that we identify is hydrophilic. This applies to all samples and across all cooking events, even though recent studies suggest that hydrophobicity may aid protein survival [<a href="#pone.0330195.ref030" class="usa-link" aria-describedby="pone.0330195.ref030">30</a>, <a href="#pone.0330195.ref061" class="usa-link" aria-describedby="pone.0330195.ref061">61</a>]. Consequently, the strong bias towards hydrophobic parts of the protein sequences may be more indicative of biases in the protein extraction process rather than resistance against degradation during periods of heating and burial.</p>
<p>Apart from degrading proteins to the point they can no longer be detected, it is well known that processes such as heating can lead to PTMs forming, although the precise effects are not well understood [<a href="#pone.0330195.ref062" class="usa-link" aria-describedby="pone.0330195.ref062">62</a>, <a href="#pone.0330195.ref063" class="usa-link" aria-describedby="pone.0330195.ref063">63</a>]. Our G-PTM-D search identified 43 different PTMs (SI 5), some of which were near ubiquitous, while others were extremely rare. The PTM profile of the identified PTMs on the most intense proteins from the first cooking event seems fairly consistent across all cooking events (<a href="#pone.0330195.g007" class="usa-link">Fig 7</a>). Some changes in the average occupancy of PTMs can be observed, such as the increase in formylation on K and deamidation in hemoglobin subunit beta</p>
<figure class="fig xbox font-sm" id="pone.0330195.g007"><h3 class="obj_head">Fig 7. Average occupancy of PTMs across all cooking events on the 10 proteins that were most intense after the first cooking event.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373221_pone.0330195.g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b934/12373221/7d7f28b6b732/pone.0330195.g007.jpg" loading="lazy" height="864" width="693" alt="Fig 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330195.g007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Only PTMs that had a minimum average occupancy of 10% were included.</p></figcaption></figure><p>(<a href="https://www.ncbi.nlm.nih.gov/protein/P02070" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P02070</a>) and beta-enolase (A0A1S2X522), yet simultaneously N deamidation seems to decrease in L-lactate dehydrogenase (A0A1S3MJX7) and one of the salmon collagen alpha-1(I) chain-like proteins (A0A1S3SM10). Similarly, phosphorylation on S and T seems to increase in bovine collagen alpha-1(I) chain <a href="https://www.ncbi.nlm.nih.gov/protein/P02453" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P02453</a>, but decreases in the other salmon collagen alpha-1(I) chain-like protein (A0A1S3Q7E3).</p>
<p>To test if there was a clearer pattern in PTM occupancy in the larger dataset, we tested for differences between the cooking events and the occupancy using the Kruskal Wallis test for each of the 43 identified PTMs. In case the Kruskal Wallis test indicated a significant difference (p &lt; 0.05), a post-hoc Dunn’s test was performed to identify between what cooking events the occupancy differed. Statistically significant differences were found between:</p>
<ul class="list" style="list-style-type:disc">
<li><p>Hydroxylation on P (ꭓ<sup>2</sup> = 23.72, p = 2.858e<sup>-5</sup>), significant differences (p &lt; 0.05) between all events, except between cooking event 5 and 9.</p></li>
<li><p>Deamidation on N (ꭓ<sup>2</sup> = 28.582, p = 2.742e<sup>-6</sup>), significant differences between cooking events 1 and 5 and 1 and 9.</p></li>
<li><p>Sodium on E (ꭓ<sup>2</sup> = 10.612, p = 0.01402), only differs significantly between cooking events 1 and 5.</p></li>
<li><p>Potassium on E (ꭓ<sup>2</sup> = 6.4568, p = 0.03962), significantly decreases between cooking events 1 and 9.</p></li>
<li><p>Acetylation on K (ꭓ<sup>2</sup> = 10.939, p = 0.004214), significant difference between cooking events 5 and 9.</p></li>
</ul>
<p>Although these five PTMs are statistically significantly different between cooking events, the lack of data for some cooking events for in particular acetylation on K, potassium on E and to a lesser extent sodium on E raise some questions regarding the robustness of these findings (<a href="#pone.0330195.g008" class="usa-link">Fig 8</a>). Similarly, hydroxylation on P seems to consistently display a high occupancy, but was hardly detected in the samples from after the burial.</p>
<figure class="fig xbox font-sm" id="pone.0330195.g008"><h3 class="obj_head">Fig 8. Distribution of the occupancy of the five PTMs statistically significantly different between cooking events.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373221_pone.0330195.g008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b934/12373221/3ebc4475f3bc/pone.0330195.g008.jpg" loading="lazy" height="490" width="660" alt="Fig 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330195.g008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Boxplots show the second and third quartile, as well as the median of the distribution.</p></figcaption></figure><p>Of secondary importance was to identify the tissue types based on the identified proteins. One theoretical advantage of proteins over other biomolecules is that they may reflect particular tissue-types. In order to achieve this we used STRING (version 12.0) [<a href="#pone.0330195.ref064" class="usa-link" aria-describedby="pone.0330195.ref064">64</a>,<a href="#pone.0330195.ref065" class="usa-link" aria-describedby="pone.0330195.ref065">65</a>] to create protein networks for each of the three species and downloaded the tissue expression analysis, which compares the genes associated with the identified proteins to a database (TISSUE) of gene expression profiles per tissue type [<a href="#pone.0330195.ref066" class="usa-link" aria-describedby="pone.0330195.ref066">66</a>]. The output of this analysis is a list of tissue types, gene names and various metrics regarding the robustness of the tissue association. We filtered this list of tissue associations to only include tissues with a false discovery rate below 0.01 and a minimum number of 35 supporting proteins. The remaining tissue-gene associations were linked back to the list of proteins identified per sample (<a href="#pone.0330195.g009" class="usa-link">Fig 9</a>). Unfortunately, the TISSUE database did not include any data for the observed plant proteins, so it was not possible to associate any particular tissues with the plant proteins.</p>
<figure class="fig xbox font-sm" id="pone.0330195.g009"><h3 class="obj_head">Fig 9. Tissue associations of the identified proteins per species.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373221_pone.0330195.g009.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b934/12373221/4e3af8766101/pone.0330195.g009.jpg" loading="lazy" height="576" width="722" alt="Fig 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330195.g009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The tissue associations from detected salmon and beef proteins are mostly unspecific, e.g., “animal” or “whole body”. Several associations with heart-related tissues, such as the heart itself, but also the right atrium and ventricles, were observed. One could argue to group these together as one large cluster associated with the cardiovascular system. We chose to keep them separate, as it highlights a disadvantage of this kind of tissue annotation, namely the unspecificity of the observed proteins. The fact that the average number rounded down of associated tissues per gene in our dataset is 5 ± 2 illustrates well how generic most of these genes are. This also explains why tissues such as the liver and various glands are listed in <a href="#pone.0330195.g006" class="usa-link">Fig 6</a> as well. Despite the lack of a clear muscle-associated proteome, <a href="#pone.0330195.g009" class="usa-link">Fig 9</a> highlights the difficulty in distinguishing between raw fats and muscle based on gene expression annotations, as well as showcases the breadth of tissue associations that can be expected from a foodcrust derived from these tissues.</p>
<p>Foodcrust accumulation showed sizeable variation between samples (<a href="#pone.0330195.g010" class="usa-link">Fig 10</a>). A linear regression model of the percentage of the ingredients and the foodcrust volume yielded no statistically significant relationships and had an adjusted R<sup>2</sup> value of 0.16, showing that the relative abundance of the three ingredients could only explain a small portion of the foodcrust volume variability. A linear regression model of the nutritional content showed a similar pattern. The overall model had a R<sup>2</sup> value of 0.16 and neither the absolute input of proteins, carbohydrates or fats yielded a significant relationship.</p>
<figure class="fig xbox font-sm" id="pone.0330195.g010"><h3 class="obj_head">Fig 10. Volume of foodcrust accumulated after nine cooking events, sorted by experimental conditions listed in <a href="#pone.0330195.g001" class="usa-link">Fig 1</a>.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373221_pone.0330195.g010.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b934/12373221/24878ba08879/pone.0330195.g010.jpg" loading="lazy" height="582" width="703" alt="Fig 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330195.g010/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The input ingredient composition of each of the experiments is shown by the pie chart on top of the bars. Increasing the proportion of one ingredient over another does not seem to have influenced foodcrust formation.</p></figcaption></figure></section><section id="sec010"><h2 class="pmc_sec_title">Discussion</h2>
<section id="sec011"><h3 class="pmc_sec_title">Protein recovery</h3>
<p>Our results show large variations in both the number of recovered proteins as well as the relative taxonomic composition of the recovered proteome between the different samples. However, the only significant relationship that we could observe was a decrease in the number of proteins after more cooking events. Such a decrease in protein recovery is not unexpected as it is well known that proteins are vulnerable to high temperatures [<a href="#pone.0330195.ref067" class="usa-link" aria-describedby="pone.0330195.ref067">67</a>,<a href="#pone.0330195.ref068" class="usa-link" aria-describedby="pone.0330195.ref068">68</a>] and a decrease in protein recovery from fresh foodstuffs to foodcrust has been observed in previous studies [<a href="#pone.0330195.ref030" class="usa-link" aria-describedby="pone.0330195.ref030">30</a>]. In fact, one of the aims of cooking is to make food more easily digestible by opening up closed protein structures [<a href="#pone.0330195.ref063" class="usa-link" aria-describedby="pone.0330195.ref063">63</a>, <a href="#pone.0330195.ref069" class="usa-link" aria-describedby="pone.0330195.ref069">69</a>]. Unfortunately, denatured proteins are more vulnerable to degradation [<a href="#pone.0330195.ref070" class="usa-link" aria-describedby="pone.0330195.ref070">70</a>,<a href="#pone.0330195.ref071" class="usa-link" aria-describedby="pone.0330195.ref071">71</a>], which bodes ill for protein extraction from archaeological vessels. Although the decrease in the number of recovered proteins with increased cooking events demonstrates the negative effects of prolonged heating, the generally low number of proteins found after the first cooking event indicates that even limited exposure to heat can already induce severe protein degradation.</p>
<p>Heating is also known to induce additional PTMs, yet this pattern was not clearly replicated in our data. Five PTMs showed statistically significant differences between the cooking events, but the limited number of observations for sodium on E, potassium on E and acetylation on K call into question the meaningfulness of these statistical relationships. In contrast, the increasing trend in occupancy of N deamidation with more advanced cooking events is supported by a fairly sizeable set of observations. Deamidation has been linked to protein degradation in archaeological samples [<a href="#pone.0330195.ref072" class="usa-link" aria-describedby="pone.0330195.ref072">72</a>–<a href="#pone.0330195.ref074" class="usa-link" aria-describedby="pone.0330195.ref074">74</a>] and an increase in deamidation after increasing cooking duration and burial is in line with our expectations. Although we could observe no statistically significant change in the Q deamidation.</p>
<p>However, as this study does not include unheated samples, it may be that the main effect of heating on PTMs occurs during initial heating rather than prolonged heating after partial charring has already taken place. Instead, the key message of <a href="#pone.0330195.g006" class="usa-link">Fig 6</a> is rather the diversity and high occupancy of many of the PTMs. Open searches, like a G-PTM-D, are not standard practice in palaeoproteomics, and in most studies a select number of PTMs, such as deamidation (NQ) and oxidation (M) are included. Our results show that foodcrusts contain many more PTMs and by calculating the occupancy rather than the percentage of modified residues we can show that the occurrence of PTMs at particular places in the protein sequence is consistent. Although our results do not allow us to conclude that these PTMs are the result of heating, protein identification from foodcrusts may be more limited if a large number of PTMs are not included.</p>
<p>Apart from the heating-induced damage, biases in the incorporation of proteins in the foodcrusts and extraction efficiency may have contributed to the limited size of the extracted proteome. For example, <a href="#pone.0330195.g006" class="usa-link">Fig 6</a> shows that in the case of nearly all recovered proteins, the part of the sequence that we are able to recover is hydrophobic. Additionally, the fact that the standard deviation is roughly equal to the mean number of detected proteins for cooking events 5, 9 and post-excavation, plus the large variation in how different experimental conditions respond to increased cooking in our opinion suggests that there might be considerable variation in protein preservation across the foodcrust in one pot. The degree of spatial variation in foodcrust protein content might best be tested with a spectroscopy-based method, such as NIR [<a href="#pone.0330195.ref075" class="usa-link" aria-describedby="pone.0330195.ref075">75</a>]. A better understanding of the amount of spatial variation as well as the scale at which it occurs could be of great benefit in maximising the proteomic data obtained from these degraded samples.</p>
<p>Furthermore, recent studies highlight that the current state of proteomic bioinformatic analysis only manages to assign a protein identification to a fraction of the extracted compounds [<a href="#pone.0330195.ref076" class="usa-link" aria-describedby="pone.0330195.ref076">76</a>,<a href="#pone.0330195.ref077" class="usa-link" aria-describedby="pone.0330195.ref077">77</a>]. It may be that future proteomic bioinformatic developments will significantly improve our ability to identify proteins in foodcrust samples, yet for now, we were only able to identify a small set of proteins, which for some samples do not represent all the taxa used in the cooking mixture.</p>
<p>All these issues may have impacted our ability to extract and identify proteins from our experimental samples, but ancient foodcrusts may have been exposed to higher temperatures, for longer periods of time and more often. Nevertheless, previous studies have shown that it is possible to extract proteins from archaeological foodcrusts, but the number of recovered proteins is often in the single digits [<a href="#pone.0330195.ref004" class="usa-link" aria-describedby="pone.0330195.ref004">4</a>,<a href="#pone.0330195.ref029" class="usa-link" aria-describedby="pone.0330195.ref029">29</a>]. Considering the limitations of recovering proteins from foodcrusts, it is important to evaluate how well we can succeed in using the limited number of found proteins to identify the taxa and tissues used during the cooking process. Especially in comparison to lipid analysis, which is the most common method for analysing pottery use [<a href="#pone.0330195.ref078" class="usa-link" aria-describedby="pone.0330195.ref078">78</a>].</p></section><section id="sec012"><h3 class="pmc_sec_title">Disentangling mixtures &amp; taxonomic representativeness</h3>
<p>One of the main theoretical advantages of proteomics over lipid analysis is the ability to distinguish the different resources in a mixture. As interpretations of resource use in lipid analysis are often based on compound-specific isotope ratios, samples with values that do not fall in the established range of the references are more challenging to interpret. Lipid biomarkers could help with this problem, but they are relatively limited, both in terms of the number of taxa with established biomarkers as well as the taxonomic specificity of the biomarkers [<a href="#pone.0330195.ref019" class="usa-link" aria-describedby="pone.0330195.ref019">19</a>,<a href="#pone.0330195.ref043" class="usa-link" aria-describedby="pone.0330195.ref043">43</a>,<a href="#pone.0330195.ref079" class="usa-link" aria-describedby="pone.0330195.ref079">79</a>,<a href="#pone.0330195.ref080" class="usa-link" aria-describedby="pone.0330195.ref080">80</a>]. Alternatively, one can apply a mixing model to calculate what combination of foodstuffs could explain the observed values. Mixing models are known to work well in estimating the proportion of a limited number of resources with distinct isotope ratios but struggle more with more complex mixtures of a larger number of foodstuffs. Additionally, species with very similar ecological niches will be nearly impossible to distinguish isotopically [<a href="#pone.0330195.ref081" class="usa-link" aria-describedby="pone.0330195.ref081">81</a>].</p>
<p>In theory, proteomic analysis does not suffer from these limitations. Mixing of multiple foodstuffs should not impact our ability to identify them and proteins can allow for up to species-level identifications [<a href="#pone.0330195.ref082" class="usa-link" aria-describedby="pone.0330195.ref082">82</a>], although this is dependent on the type of recovered proteins and their coverage. Although these theoretical advantages of proteomics might lead to high expectations regarding the performance of proteomic analysis of foodcrust, the aforementioned issues in recovering proteins should temper them. To illustrate this point, 25 out of the 84 samples analysed in this study yielded no proteins at all. 32 samples yielded proteins from all species that were cooked in them. In 18 samples no proteins of one of the cooked species could be found and in 4 samples two species were missing. Lastly, in five samples proteins were found that belonged to a species that was not part of the mixture cooked in that pot (<a href="#pone.0330195.t001" class="usa-link">Table 1</a>). The peptides that were uniquely associated with these proteins were run through BLAST [<a href="#pone.0330195.ref083" class="usa-link" aria-describedby="pone.0330195.ref083">83</a>] to verify their taxonomic identification. The most specific taxon represented by at least two peptides was accepted as the new taxonomic identification (<a href="#pone.0330195.t004" class="usa-link">Table 4</a>). This authentication step indicated that the plant proteins in samples 15.1 and 21.1 were shared up to the level of Eukaryota and could thus well derive from the ingredients cooked in the vessels. Although the BLAST taxonomic identifications are more specific for some of the remaining proteins than the others, it does appear that in samples 6.5, 7.1, 12.1 and 12.9 there are proteins present from a species that was not used in the cooking.</p>
<section class="tw xbox font-sm" id="pone.0330195.t004"><h4 class="obj_head">Table 4. Revised taxonomic identifications using BLAST of potentially intrusive proteins.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Experiment number</th>
<th align="left" rowspan="1" colspan="1">Cooking event</th>
<th align="left" rowspan="1" colspan="1">Protein Accession</th>
<th align="left" rowspan="1" colspan="1">Protein Full Name</th>
<th align="left" rowspan="1" colspan="1">Sequence Coverage</th>
<th align="left" rowspan="1" colspan="1">Number of Unique Peptides</th>
<th align="left" rowspan="1" colspan="1">Organism</th>
<th align="left" rowspan="1" colspan="1">BLAST taxon</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">A0A3Q1LL35</td>
<td align="left" rowspan="1" colspan="1">Uncharacterized protein</td>
<td align="left" rowspan="1" colspan="1">25%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Bos taurus</td>
<td align="left" rowspan="1" colspan="1">Boreoeutheria</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">7</td>
<td align="left" rowspan="3" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">A0A1S2X522</td>
<td align="left" rowspan="1" colspan="1">phosphopyruvate hydratase</td>
<td align="left" rowspan="1" colspan="1">63%</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">Clupeocephala</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B5DG55</td>
<td align="left" rowspan="1" colspan="1">Alpha-1,4 glucan phosphorylase</td>
<td align="left" rowspan="1" colspan="1">48%</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">Salmonidae</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q3ZLR1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q3ZLR1</a></td>
<td align="left" rowspan="1" colspan="1">Superoxide dismutase [Cu-Zn]</td>
<td align="left" rowspan="1" colspan="1">23%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">C0H9B8</td>
<td align="left" rowspan="1" colspan="1">Trifunctional enzyme subunit beta, mitochondrial</td>
<td align="left" rowspan="1" colspan="1">9%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">Teleostei</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">A0A1S3Q7E3</td>
<td align="left" rowspan="1" colspan="1">collagen alpha-1(I) chain-like</td>
<td align="left" rowspan="1" colspan="1">10%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">Salmoninae</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A1S3SM10</td>
<td align="left" rowspan="1" colspan="1">collagen alpha-1(I) chain-like</td>
<td align="left" rowspan="1" colspan="1">8%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Salmo salar</td>
<td align="left" rowspan="1" colspan="1">Salmoninae</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q9LEE0" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q9LEE0</a></td>
<td align="left" rowspan="1" colspan="1">phosphopyruvate hydratase</td>
<td align="left" rowspan="1" colspan="1">15%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Spinacia oleracea</td>
<td align="left" rowspan="1" colspan="1">Eukaryota</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">21</td>
<td align="left" rowspan="3" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">A0A0J7YLM1</td>
<td align="left" rowspan="1" colspan="1">Elongation factor Tu (Fragment)</td>
<td align="left" rowspan="1" colspan="1">13%</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">Beta vulgaris subsp. vulgaris</td>
<td align="left" rowspan="1" colspan="1">Eukaryota</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A0J8B1E8</td>
<td align="left" rowspan="1" colspan="1">50S ribosomal protein L5, chloroplastic</td>
<td align="left" rowspan="1" colspan="1">14%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Beta vulgaris subsp. vulgaris</td>
<td align="left" rowspan="1" colspan="1">Eukaryota</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A0A0K9Q7T6</td>
<td align="left" rowspan="1" colspan="1">S1 motif domain-containing protein</td>
<td align="left" rowspan="1" colspan="1">4%</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">Spinacia oleracea</td>
<td align="left" rowspan="1" colspan="1">Eukaryota</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330195.t004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>There are several explanations for these ‘lost’ proteins, which occur in 5 out of 84 samples. It could be that the contamination occurred during the cooking experiment or during sampling, although sampling equipment was cleaned with ethanol after each sample. Contamination could have occurred during protein extraction, but all proteins present in the blank were not included in further analysis. Another potential explanation is machine carry over. The fish-specific peptides detected in samples 6.1, 12.1, 12.9 were all also detected in the samples run before them. It may have happened that proteins from a previous sample remained stuck on the column and were only eluted with the next sample. Carry over seems a less likely explanation for the <em>B. taurus</em> protein in sample 6.5, as it was the first sample in its batch. Instead, it may be that this protein, A0A3Q1LL35, is background contamination. It is listed as an uncharacterised protein and the taxon assigned by BLAST, Boreoeutheria, encompasses most of the placental mammals. Consequently, this protein could have derived from any number of common contaminants that could have been introduced at any stage of the experiment.</p>
<p>As with the overall number of proteins, the representativeness of the extracted proteome decreases substantially with additional cooking episodes. After the first cooking event 5 of the 21 samples were missing one of the input foods, but after burying the samples 6 out of 7 samples were missing proteins from one or two of the cooked foods. Interestingly, despite 3 of 7 samples yielding protein after burial containing beef fat, none yielded <em>Bos taurus</em> proteins. To check if there was a systematic bias in recovery against any particular species we calculated the recovery rate of each species by dividing the number of samples they were found in by the number of samples that the species was cooked in (<a href="#pone.0330195.t005" class="usa-link">Table 5</a>). The aforementioned 6 samples with proteins from species that were not part of the mixture were excluded from the recovery rate calculation for that particular species. The protein abundance of each species was not taken into account in calculating the recovery rate due to the previously discussed challenges in quantifying protein abundance for degraded samples.</p>
<section class="tw xbox font-sm" id="pone.0330195.t005"><h4 class="obj_head">Table 5. Recovery rate of each species per cooking event.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Cooking event</th>
<th align="left" rowspan="1" colspan="1">Beetroot</th>
<th align="left" rowspan="1" colspan="1">Salmon</th>
<th align="left" rowspan="1" colspan="1">Beef</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">80%</td>
<td align="left" rowspan="1" colspan="1">93%</td>
<td align="left" rowspan="1" colspan="1">87%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">27%</td>
<td align="left" rowspan="1" colspan="1">93%</td>
<td align="left" rowspan="1" colspan="1">67%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">53%</td>
<td align="left" rowspan="1" colspan="1">80%</td>
<td align="left" rowspan="1" colspan="1">73%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PE</td>
<td align="left" rowspan="1" colspan="1">13%</td>
<td align="left" rowspan="1" colspan="1">40%</td>
<td align="left" rowspan="1" colspan="1">0%</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Average</td>
<td align="left" rowspan="1" colspan="1">43%</td>
<td align="left" rowspan="1" colspan="1">77%</td>
<td align="left" rowspan="1" colspan="1">57%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330195.t005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p><a href="#pone.0330195.t005" class="usa-link">Table 5</a> shows clear differences in recovery rate between the foodstuffs. In general, beetroot was the most challenging to recover, which is in line with expectations considering the low protein content of beetroot. Vice versa, the fact that salmon has the highest recovery rate is in line with it having the highest protein content. Additionally, although in general there is a pattern of lower numbers of proteins and decreased recovery rate with an increased number of cooking events, the recovery rate for beetroot and beef is higher for cooking event 9 than 5. However, the Kruskal Wallis and Dunn’s test showed that cooking events 5 and 9 were the only two groups without a statistically significant difference in protein numbers. The lack of a statistically significant difference may be explained by the addition of fresh food at the start of each cooking event. It may be that it is this ‘freshest’ food incorporated into the foodcrust that is yielding us proteins.</p>
<p>Secondly, <a href="#pone.0330195.t005" class="usa-link">Table 5</a> highlights the sudden disappearance of <em>Bos taurus</em> proteins after burial. It is unclear what causes the lack of <em>Bos taurus</em> protein preservation. It could be that the proteins in beef fat (which includes vimentin, creatine kinase B-type and heat shock protein beta-1) are simply more vulnerable to degradation due to their structure. Although <a href="#pone.0330195.g005" class="usa-link">Fig 5</a> shows that the <em>Bos taurus</em> proteins identified are associated with largely the same tissues as the <em>Salmo salar</em> proteins. Alternatively, the vulnerability to degradation may be due to the structure of the foodcrust and how fat-rich foods are incorporated in them rather than the structure of the proteins. There is some reason to suspect other factors might also be at play. A recent study performed a similar experiment, generating and burying foodcrusts made using red deer (<em>Cervus elaphus</em>), salmon (<em>Salmo salar</em>) and sweet chestnut (<em>Castanea sativa</em>) [<a href="#pone.0330195.ref030" class="usa-link" aria-describedby="pone.0330195.ref030">30</a>]. Although our experiment did not use <em>Cervus elaphus</em>, it may be compared to our vessels with <em>Bos taurus</em>. The two species are closely enough related that most of their proteins should be similar, if not their sequences. Of their top 5 most abundant deer proteins in the foodcrust after cooking by peptide count, only fructose-bisphosphate aldolase (A0A3S5ZPB0) was detected in our samples and only 2 samples from the first cooking event. Two of the other common deer proteins, troponin T3 and myosin-1 were not directly detected in our samples, but myosin light chain 1/3 (<a href="https://www.ncbi.nlm.nih.gov/protein/A0JNJ5" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A0JNJ5</a>) and troponin I2 (F6QIC1) were detected, although again in a limited number of samples. As for the most abundant proteins after cooking, only myoglobin was detected in our samples, but bovine myoglobin is the 7th most frequently recovered bovine protein in our samples. However, it was not found in any samples after burial.</p>
<p>There is more overlap in the <em>Salmo salar</em> proteins between our study and Evans <em>et al.</em> (2024). All five most abundant <em>Salmo salar</em> proteins from their after-cooking samples were observed in our samples, although they do not occur in the top 10 of most frequently recovered <em>Salmo salar</em> proteins in our samples. As for the most abundant proteins after burial, four out of five were detected in our samples, but only one, myosin heavy chain, fast skeletal muscle-like (A0A1S3QIK3), in one of our after-burial samples.</p>
<p>The recovery of sweet chestnut and beetroot cannot be compared as easily as <em>Cervus elaphus</em> and <em>Bos taurus</em>, but it is noteworthy that they recover chestnut proteins less frequently than <em>Salmo salar</em> or <em>Cervus elaphus</em> proteins in their buried samples, whereas we do find plant proteins, but no <em>Bos taurus</em> proteins in our buried samples. Of course, their experiment used deer meat, while we used beef fat and it may be that this changes the preservation potential of the proteins contained within the tissue. Regardless of what prevented us from recovering <em>Bos taurus</em> proteins, it is clear that, especially for buried samples, the recovered proteome might not reflect all the foodstuffs cooked in the vessel and that there is substantial variability in our ability to obtain proteins from buried samples.</p>
<p>Compared to other proteomic analyses of organic residues, the results presented here differ somewhat in the diversity of proteins identified. Previous cooking experiments (without burial) on protein extraction from ceramics found mostly myosin, collagen and haemoglobin type proteins [<a href="#pone.0330195.ref084" class="usa-link" aria-describedby="pone.0330195.ref084">84</a>, <a href="#pone.0330195.ref085" class="usa-link" aria-describedby="pone.0330195.ref085">85</a>]. Myosins and collagens have also been found in the limited number of studies on archaeological foodcrust [<a href="#pone.0330195.ref029" class="usa-link" aria-describedby="pone.0330195.ref029">29</a>], as well as parvalbumin and vitellogenin in one specific study [<a href="#pone.0330195.ref004" class="usa-link" aria-describedby="pone.0330195.ref004">4</a>]. Calcified residues have also yielded myosin and haemoglobin proteins [<a href="#pone.0330195.ref086" class="usa-link" aria-describedby="pone.0330195.ref086">86</a>], but in general this type of residue seems to more frequently yield dairy proteins [<a href="#pone.0330195.ref026" class="usa-link" aria-describedby="pone.0330195.ref026">26</a>, <a href="#pone.0330195.ref028" class="usa-link" aria-describedby="pone.0330195.ref028">28</a>]. Collagen, myosin and haemoglobin also feature among the most intense proteins we observed after the first cooking event (<a href="#pone.0330195.t002" class="usa-link">Table 2</a>), but only myosin persisted through all cooking events and burial (<a href="#pone.0330195.t003" class="usa-link">Table 3</a>). Instead, most of the other proteins listed in <a href="#pone.0330195.t003" class="usa-link">Table 3</a> are not commonly reported from organic residues. Data from other types of organic residues, such as from the ceramic matrix or calcified residues are not necessarily directly comparable to foodcrusts. Although these materials also contain dietary proteins, each of them has their own unique properties and processes of protein incorporation. Nonetheless, based on previous studies it seems that for meat-based foodstuffs proteins such as collagens, myosin and haemoglobins form a ‘core’ proteome that can be relatively frequently recovered.</p></section><section id="sec013"><h3 class="pmc_sec_title">Foodcrust abundance</h3>
<p>Our and previous results [<a href="#pone.0330195.ref030" class="usa-link" aria-describedby="pone.0330195.ref030">30</a>] show that the protein contents of foodcrusts do not ‘fairly’ represent all the foodstuffs prepared in the vessel. Biases in protein recovery for particular foodstuffs were expected to a certain extent, but <a href="#pone.0330195.g010" class="usa-link">Fig 10</a> suggests that there are also differences in the quantity of foodcrust that particular foodstuffs generate. <a href="#pone.0330195.g010" class="usa-link">Fig 10</a> shows that there is not one particular food that is driving the formation of more foodcrust. Overall, the beetroot-salmon mixtures produce the most foodcrust, followed by beetroot-salmon-beef fat mixtures. Beetroot-beef fat or salmon-beef fat seems to produce the least foodcrust, suggesting that the presence of such a fat-rich food is not necessarily a driver of foodcrust formation. Indeed, nowadays in contact frying it is common to add a lipid-rich substance, such as oil or butter, to prevent food from sticking to cookware. This common practice is supported by studies showing that increasing amounts of oil reduces the adhesion of food to containers [<a href="#pone.0330195.ref087" class="usa-link" aria-describedby="pone.0330195.ref087">87</a>, <a href="#pone.0330195.ref088" class="usa-link" aria-describedby="pone.0330195.ref088">88</a>]. Lipid-based coatings have even been proposed for pharmaceutical containers to prevent nonspecific binding of the active components of drugs to container walls [<a href="#pone.0330195.ref089" class="usa-link" aria-describedby="pone.0330195.ref089">89</a>]. Beyond food content, experimental studies on organic residue formation, or ‘fouling’ [<a href="#pone.0330195.ref090" class="usa-link" aria-describedby="pone.0330195.ref090">90</a>], show that the material characteristics (e.g., surface roughness) have an substantial effect on the non-stick properties of cookware, as well as the ease of cleaning [<a href="#pone.0330195.ref088" class="usa-link" aria-describedby="pone.0330195.ref088">88</a>, <a href="#pone.0330195.ref091" class="usa-link" aria-describedby="pone.0330195.ref091">91</a>].</p>
<p>The key finding of <a href="#pone.0330195.g010" class="usa-link">Fig 10</a> is perhaps to highlight the importance of carbohydrate and protein-rich foodstuffs in generating foodcrust in the first place, while fat-rich foods seem to play a lesser role. Yet the most common method of analysing these organic residues has so far been lipid analysis, which may bias our interpretation of the processed resources towards fat-rich foodstuffs. Such a variable pattern in foodcrust formation in our dataset emphasises the value of replicates, which we, unfortunately, are not able to provide. Nonetheless, we argue that these results are a valuable addition to our current understanding of the formation of organic residues, especially because as far as we are aware there has been no absolute quantification of the abundance of foodcrust formed up to this date.</p>
<p>Additionally, although we could not obtain quantitative measurements of the volume of foodcrust forming on different parts of the pot (e.g., rim, body or bottom), it is clear from <a href="#pone.0330195.g002" class="usa-link">Fig 2</a> and the 3D scans that foodcrusts predominantly, and in the case of some vessels only, at the rim of the vessel. In particular, foodcrusts seem to form mostly at the interface between the water level and the ceramic, which seems to be in line with observations on the formation of ‘fouling’ in industrial dairy processing contexts [<a href="#pone.0330195.ref092" class="usa-link" aria-describedby="pone.0330195.ref092">92</a>]. A potential explanation for the proclivity of foodcrust formation at this position might be the steep gradient in temperature. Most of the food during the cooking events was submerged in water, which cannot exceed its boiling point of 100 °C, whereas the ceramic itself will be closer to the 270 °C the experiment was conducted at. As the waterline receded through evaporation more of the food would suddenly be exposed to the higher temperatures, potentially prompting the formation of foodcrust. This applies to both pieces of the solid foodstuffs, as well as any soluble components.</p></section></section><section id="sec014"><h2 class="pmc_sec_title">Conclusion</h2>
<p>This experiment had three core aims: firstly, to determine the relationship between the relative ratios of input foods and the proportions between the species of the recovered proteins. Secondly, to measure the impact of repeated cooking and burial on protein preservation and lastly how the different ingredient combinations impacted the amount of foodcrust that formed.</p>
<p>The data shows clearly that the relative abundance of recovered proteins does not mirror the actual input of foodstuffs. Most identified proteins belonged to <em>Salmo salar,</em> followed by <em>Bos taurus</em> and lastly <em>Beta vulgaris</em>. This pattern applied to samples taken after cooking events 1, 5 and 9. However, in the samples taken after the burial no <em>Bos taurus</em> proteins could be recovered, although a small number of <em>Beta vulgaris</em> proteins were identifiable. These results highlight that not only do food input and recovered proteins not necessarily align, the relative abundance of taxa may change due to increased degradation.</p>
<p>Secondly, the number of recovered proteins decreased significantly from the first to the fifth cooking event, plateaued from the fifth to the ninth and decreased again from the ninth event until after burial.</p>
<p>Lastly, no statistically significant predictor of the foodcrust volume could be identified, but the mixtures that formed the most foodcrust in terms of volume were beetroot-salmon, followed by a mixture of all three ingredients, suggesting that lipid-rich foods are not as important for the volume of foodcrust.</p></section><section id="sec015"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pone.0330195.s001"><div class="caption p">
<span>S1 Data. Nutritional composition of the used foods.</span><p>(CSV)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s001.csv" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s001.csv</a><sup> (1.3KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330195.s002"><div class="caption p">
<span>S2 Data. Overview of animal and plant species identified at Danish Ertebølle sites and the availability of public protein reference data for these species.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s002.xlsx</a><sup> (124.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330195.s003"><div class="caption p">
<span>S3 Data. Metadata of the MetaMorpheus database search.</span><p>(ZIP)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s003.zip" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s003.zip</a><sup> (53.7MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330195.s004"><div class="caption p">
<span>S4 Data. List of identified proteins.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s004.xlsx</a><sup> (1.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330195.s005"><div class="caption p">
<span>S5 Data. Overview of the number of the number of foodcrusts per cooking event a protein was identified in.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s005.xlsx</a><sup> (2.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330195.s006"><div class="caption p">
<span>S6 Data. Overview of the identified PTMs, both per protein and summarised per cooking event.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s006.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s006.xlsx</a><sup> (508.4KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank the Bioscience Technology Facility and Chemistry Department at the University of York for mass spectrometry access and support, in particular we would like to thank Dr. Adam Dowle for his support in running the samples and feedback on the manuscript. The instrumentation is part of the York Centre of Excellence in Mass Spectrometry. The centre was created thanks to a major capital investment through Science City York, supported by Yorkshire Forward with funds from the Northern Way Initiative and subsequent support from EPSRC (EP/K039660/1; EP/M028127/1).</p>
<p>We would like to thank the YEAR Centre for their permission to use the Centre’s grounds for the burial experiment and in particular Dr. Gareth Perry for his assistance in the burial and excavation of the experimental vessels.</p>
<p>For the morphological taxonomic verification of the <em>Salmo salar</em> specimens, we would like to thank Dr. Katrien Dierickx and for his assistance in figuring out Sankey diagrams we thank Dr. Jakob Hansen. Additionally, we would like to thank Dr. Claire Koenig for her advice on protein quantification.</p>
<p>Additionally, our gratitude goes to Patrick Gibbs and James Osborn from Heritage360, University of York, for access to their 3D scanning equipment and assistance in the acquisition and analysis of the 3D data.</p>
<p>Lastly, we would like to thank the two anonymous reviewers for their constructive comments, which helped improve the manuscript.</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>A full submission of the acquired mass spectrometry data has been made to the ProteomeXchange Consortium via the PRIDE 64 partner repository with the dataset identifier PXD059930 and <a href="https://doi.org/10.6019/PXD059930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6019/PXD059930</a>. Additional metadata of the MetaMorpheus database search and the 3D model object files of the experimental ceramic vessels used in the cooking experiment can be found on Zenodo under the following DOI: <a href="https://doi.org/10.5281/zenodo.15874520" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15874520</a>.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This project was made possible thanks to funding from the European Union’s Horizon 2020 research and innovation programme under the Marie Skłodowska-Curie grant agreement No 956351. The funders had no role in study design, data collection, analysis, decision to publish or preparation of the manuscript.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0330195.ref001">
<span class="label">1.</span><cite>Oras E, Lucquin A, Lõugas L, Tõrv M, Kriiska A, Craig OE. The adoption of pottery by north-east European hunter-gatherers: Evidence from lipid residue analysis. J Archaeol Sc. 2017;78:112–9. doi: 10.1016/j.jas.2016.11.010</cite> [<a href="https://doi.org/10.1016/j.jas.2016.11.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Archaeol%20Sc&amp;title=The%20adoption%20of%20pottery%20by%20north-east%20European%20hunter-gatherers:%20Evidence%20from%20lipid%20residue%20analysis&amp;author=E%20Oras&amp;author=A%20Lucquin&amp;author=L%20L%C3%B5ugas&amp;author=M%20T%C3%B5rv&amp;author=A%20Kriiska&amp;volume=78&amp;publication_year=2017&amp;pages=112-9&amp;doi=10.1016/j.jas.2016.11.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref002">
<span class="label">2.</span><cite>Lundy J, Drieu L, Orecchioni P, Meo A, Aniceti V, Fiorentino G, et al. Cuisine in transition? Organic residue analysis of domestic containers from 9th-14th century Sicily. R Soc Open Sci. 2023;10(3):221305. doi: 10.1098/rsos.221305

</cite> [<a href="https://doi.org/10.1098/rsos.221305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9993051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36908986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=R%20Soc%20Open%20Sci&amp;title=Cuisine%20in%20transition?%20Organic%20residue%20analysis%20of%20domestic%20containers%20from%209th-14th%20century%20Sicily&amp;author=J%20Lundy&amp;author=L%20Drieu&amp;author=P%20Orecchioni&amp;author=A%20Meo&amp;author=V%20Aniceti&amp;volume=10&amp;issue=3&amp;publication_year=2023&amp;pages=221305&amp;pmid=36908986&amp;doi=10.1098/rsos.221305&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref003">
<span class="label">3.</span><cite>Hendy J, Colonese AC, Franz I, Fernandes R, Fischer R, Orton D, et al. Ancient proteins from ceramic vessels at Çatalhöyük West reveal the hidden cuisine of early farmers. Nat Commun. 2018;9(1):4064. doi: 10.1038/s41467-018-06335-6

</cite> [<a href="https://doi.org/10.1038/s41467-018-06335-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6170438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30283003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Ancient%20proteins%20from%20ceramic%20vessels%20at%20%C3%87atalh%C3%B6y%C3%BCk%20West%20reveal%20the%20hidden%20cuisine%20of%20early%20farmers&amp;author=J%20Hendy&amp;author=AC%20Colonese&amp;author=I%20Franz&amp;author=R%20Fernandes&amp;author=R%20Fischer&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=4064&amp;pmid=30283003&amp;doi=10.1038/s41467-018-06335-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref004">
<span class="label">4.</span><cite>Shevchenko A, Schuhmann A, Thomas H, Wetzel G. Fine Endmesolithic fish caviar meal discovered by proteomics in foodcrusts from archaeological site Friesack 4 (Brandenburg, Germany). PLoS One. 2018;13(11):e0206483. doi: 10.1371/journal.pone.0206483

</cite> [<a href="https://doi.org/10.1371/journal.pone.0206483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6261446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30485287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Fine%20Endmesolithic%20fish%20caviar%20meal%20discovered%20by%20proteomics%20in%20foodcrusts%20from%20archaeological%20site%20Friesack%204%20(Brandenburg,%20Germany)&amp;author=A%20Shevchenko&amp;author=A%20Schuhmann&amp;author=H%20Thomas&amp;author=G%20Wetzel&amp;volume=13&amp;issue=11&amp;publication_year=2018&amp;pmid=30485287&amp;doi=10.1371/journal.pone.0206483&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref005">
<span class="label">5.</span><cite>Chowdhury MP, Campbell S, Buckley M. Proteomic analysis of archaeological ceramics from Tell Khaiber, southern Iraq. J Archaeol Sci. 2021;132:105414. doi: 10.1016/j.jas.2021.105414</cite> [<a href="https://doi.org/10.1016/j.jas.2021.105414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Archaeol%20Sci&amp;title=Proteomic%20analysis%20of%20archaeological%20ceramics%20from%20Tell%20Khaiber,%20southern%20Iraq&amp;author=MP%20Chowdhury&amp;author=S%20Campbell&amp;author=M%20Buckley&amp;volume=132&amp;publication_year=2021&amp;pages=105414&amp;doi=10.1016/j.jas.2021.105414&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref006">
<span class="label">6.</span><cite>Cronin T, Downey L, Synnott C, McSweeney P, Kelly EP, Cahill M, et al. Composition of ancient Irish bog butter. International Dairy Journal. 2007;17(9):1011–20. doi: 10.1016/j.idairyj.2007.01.012</cite> [<a href="https://doi.org/10.1016/j.idairyj.2007.01.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Dairy%20Journal&amp;title=Composition%20of%20ancient%20Irish%20bog%20butter&amp;author=T%20Cronin&amp;author=L%20Downey&amp;author=C%20Synnott&amp;author=P%20McSweeney&amp;author=EP%20Kelly&amp;volume=17&amp;issue=9&amp;publication_year=2007&amp;pages=1011-20&amp;doi=10.1016/j.idairyj.2007.01.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref007">
<span class="label">7.</span><cite>Greco E, El-Aguizy O, Ali MF, Foti S, Cunsolo V, Saletti R, et al. Proteomic Analyses on an Ancient Egyptian Cheese and Biomolecular Evidence of Brucellosis. Anal Chem. 2018;90(16):9673–6. doi: 10.1021/acs.analchem.8b02535

</cite> [<a href="https://doi.org/10.1021/acs.analchem.8b02535" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30044608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Chem&amp;title=Proteomic%20Analyses%20on%20an%20Ancient%20Egyptian%20Cheese%20and%20Biomolecular%20Evidence%20of%20Brucellosis&amp;author=E%20Greco&amp;author=O%20El-Aguizy&amp;author=MF%20Ali&amp;author=S%20Foti&amp;author=V%20Cunsolo&amp;volume=90&amp;issue=16&amp;publication_year=2018&amp;pages=9673-6&amp;pmid=30044608&amp;doi=10.1021/acs.analchem.8b02535&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref008">
<span class="label">8.</span><cite>Yang Y, Shevchenko A, Knaust A, Abuduresule I, Li W, Hu X, et al. Proteomics evidence for kefir dairy in Early Bronze Age China. J Archaeol Sci. 2014;45:178–86. doi: 10.1016/j.jas.2014.02.005</cite> [<a href="https://doi.org/10.1016/j.jas.2014.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Archaeol%20Sci&amp;title=Proteomics%20evidence%20for%20kefir%20dairy%20in%20Early%20Bronze%20Age%20China&amp;author=Y%20Yang&amp;author=A%20Shevchenko&amp;author=A%20Knaust&amp;author=I%20Abuduresule&amp;author=W%20Li&amp;volume=45&amp;publication_year=2014&amp;pages=178-86&amp;doi=10.1016/j.jas.2014.02.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref009">
<span class="label">9.</span><cite>Craig OE, Saul H, Lucquin A, Nishida Y, Taché K, Clarke L, et al. Earliest evidence for the use of pottery. Nature. 2013;496(7445):351–4. doi: 10.1038/nature12109

</cite> [<a href="https://doi.org/10.1038/nature12109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23575637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Earliest%20evidence%20for%20the%20use%20of%20pottery&amp;author=OE%20Craig&amp;author=H%20Saul&amp;author=A%20Lucquin&amp;author=Y%20Nishida&amp;author=K%20Tach%C3%A9&amp;volume=496&amp;issue=7445&amp;publication_year=2013&amp;pages=351-4&amp;pmid=23575637&amp;doi=10.1038/nature12109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref010">
<span class="label">10.</span><cite>Admiraal M, Lucquin A, Drieu L, Casale S, Jordan PD, Craig OE. Leftovers: The presence of manufacture‐derived aquatic lipids in Alaskan pottery. Archaeometry. 2019;62(2):346–61. doi: 10.1111/arcm.12515</cite> [<a href="https://doi.org/10.1111/arcm.12515" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archaeometry&amp;title=Leftovers:%20The%20presence%20of%20manufacture%E2%80%90derived%20aquatic%20lipids%20in%20Alaskan%20pottery&amp;author=M%20Admiraal&amp;author=A%20Lucquin&amp;author=L%20Drieu&amp;author=S%20Casale&amp;author=PD%20Jordan&amp;volume=62&amp;issue=2&amp;publication_year=2019&amp;pages=346-61&amp;doi=10.1111/arcm.12515&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref011">
<span class="label">11.</span><cite>Admiraal M, Colonese AC, Milheira RG, da Rocha Bandeira D, Demathe A, Pereira Dos Santos AM, et al. Chemical analysis of pottery reveals the transition from a maritime to a plant-based economy in pre-colonial coastal Brazil. Sci Rep. 2023;13(1):16771. doi: 10.1038/s41598-023-42662-5

</cite> [<a href="https://doi.org/10.1038/s41598-023-42662-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10556129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37798303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Chemical%20analysis%20of%20pottery%20reveals%20the%20transition%20from%20a%20maritime%20to%20a%20plant-based%20economy%20in%20pre-colonial%20coastal%20Brazil&amp;author=M%20Admiraal&amp;author=AC%20Colonese&amp;author=RG%20Milheira&amp;author=D%20da%20Rocha%20Bandeira&amp;author=A%20Demathe&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=16771&amp;pmid=37798303&amp;doi=10.1038/s41598-023-42662-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref012">
<span class="label">12.</span><cite>Copley MS, Berstan R, Dudd SN, Docherty G, Mukherjee AJ, Straker V, et al. Direct chemical evidence for widespread dairying in prehistoric Britain. Proc Natl Acad Sci U S A. 2003;100(4):1524–9. doi: 10.1073/pnas.0335955100

</cite> [<a href="https://doi.org/10.1073/pnas.0335955100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC149865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12574520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Direct%20chemical%20evidence%20for%20widespread%20dairying%20in%20prehistoric%20Britain&amp;author=MS%20Copley&amp;author=R%20Berstan&amp;author=SN%20Dudd&amp;author=G%20Docherty&amp;author=AJ%20Mukherjee&amp;volume=100&amp;issue=4&amp;publication_year=2003&amp;pages=1524-9&amp;pmid=12574520&amp;doi=10.1073/pnas.0335955100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref013">
<span class="label">13.</span><cite>Copley MS, Berstan R, Dudd SN, Aillaud S, Mukherjee AJ, Straker V, et al. Processing of milk products in pottery vessels through British prehistory. Antiquity. 2005;79(306):895–908. doi: 10.1017/s0003598x00115029</cite> [<a href="https://doi.org/10.1017/s0003598x00115029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antiquity&amp;title=Processing%20of%20milk%20products%20in%20pottery%20vessels%20through%20British%20prehistory&amp;author=MS%20Copley&amp;author=R%20Berstan&amp;author=SN%20Dudd&amp;author=S%20Aillaud&amp;author=AJ%20Mukherjee&amp;volume=79&amp;issue=306&amp;publication_year=2005&amp;pages=895-908&amp;doi=10.1017/s0003598x00115029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref014">
<span class="label">14.</span><cite>Craig OE, Chapman J, Heron C, Willis LH, Bartosiewicz L, Taylor G, et al. Did the first farmers of central and eastern Europe produce dairy foods?
Antiquity. 2005;79(306):882–94. doi: 10.1017/s0003598x00115017</cite> [<a href="https://doi.org/10.1017/s0003598x00115017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antiquity&amp;title=Did%20the%20first%20farmers%20of%20central%20and%20eastern%20Europe%20produce%20dairy%20foods?&amp;author=OE%20Craig&amp;author=J%20Chapman&amp;author=C%20Heron&amp;author=LH%20Willis&amp;author=L%20Bartosiewicz&amp;volume=79&amp;issue=306&amp;publication_year=2005&amp;pages=882-94&amp;doi=10.1017/s0003598x00115017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref015">
<span class="label">15.</span><cite>Craig OE, Steele VJ, Fischer A, Hartz S, Andersen SH, Donohoe P, et al. Ancient lipids reveal continuity in culinary practices across the transition to agriculture in Northern Europe. Proc Natl Acad Sci U S A. 2011;108(44):17910–5. doi: 10.1073/pnas.1107202108

</cite> [<a href="https://doi.org/10.1073/pnas.1107202108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3207664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22025697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Ancient%20lipids%20reveal%20continuity%20in%20culinary%20practices%20across%20the%20transition%20to%20agriculture%20in%20Northern%20Europe&amp;author=OE%20Craig&amp;author=VJ%20Steele&amp;author=A%20Fischer&amp;author=S%20Hartz&amp;author=SH%20Andersen&amp;volume=108&amp;issue=44&amp;publication_year=2011&amp;pages=17910-5&amp;pmid=22025697&amp;doi=10.1073/pnas.1107202108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref016">
<span class="label">16.</span><cite>Bondetti M, Lucquin A, Savel’ev NA, Weber AW, Craig OE, Jordan PD. Resource processing, early pottery and the emergence of Kitoi culture in Cis-Baikal: Insights from lipid residue analysis of an Early Neolithic ceramic assemblage from the Gorelyi Les habitation site, Eastern Siberia. Archaeol Res Asia. 2020;24:100225. doi: 10.1016/j.ara.2020.100225</cite> [<a href="https://doi.org/10.1016/j.ara.2020.100225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archaeol%20Res%20Asia&amp;title=Resource%20processing,%20early%20pottery%20and%20the%20emergence%20of%20Kitoi%20culture%20in%20Cis-Baikal:%20Insights%20from%20lipid%20residue%20analysis%20of%20an%20Early%20Neolithic%20ceramic%20assemblage%20from%20the%20Gorelyi%20Les%20habitation%20site,%20Eastern%20Siberia&amp;author=M%20Bondetti&amp;author=A%20Lucquin&amp;author=NA%20Savel%E2%80%99ev&amp;author=AW%20Weber&amp;author=OE%20Craig&amp;volume=24&amp;publication_year=2020&amp;pages=100225&amp;doi=10.1016/j.ara.2020.100225&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref017">
<span class="label">17.</span><cite>Courel B, Robson HK, Lucquin A, Dolbunova E, Oras E, Adamczak K, et al. Organic residue analysis shows sub-regional patterns in the use of pottery by Northern European hunter–gatherers. R Soc open sci. 2020;7(4):192016. doi: 10.1098/rsos.192016
</cite> [<a href="https://doi.org/10.1098/rsos.192016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7211838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32431883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=R%20Soc%20open%20sci&amp;title=Organic%20residue%20analysis%20shows%20sub-regional%20patterns%20in%20the%20use%20of%20pottery%20by%20Northern%20European%20hunter%E2%80%93gatherers&amp;author=B%20Courel&amp;author=HK%20Robson&amp;author=A%20Lucquin&amp;author=E%20Dolbunova&amp;author=E%20Oras&amp;volume=7&amp;issue=4&amp;publication_year=2020&amp;pages=192016&amp;pmid=32431883&amp;doi=10.1098/rsos.192016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref018">
<span class="label">18.</span><cite>Lucquin A, Robson HK, Oras E, Lundy J, Moretti G, González Carretero L, et al. The impact of farming on prehistoric culinary practices throughout Northern Europe. Proc Natl Acad Sci U S A. 2023;120(43):e2310138120. doi: 10.1073/pnas.2310138120

</cite> [<a href="https://doi.org/10.1073/pnas.2310138120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10614617/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37844237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=The%20impact%20of%20farming%20on%20prehistoric%20culinary%20practices%20throughout%20Northern%20Europe&amp;author=A%20Lucquin&amp;author=HK%20Robson&amp;author=E%20Oras&amp;author=J%20Lundy&amp;author=G%20Moretti&amp;volume=120&amp;issue=43&amp;publication_year=2023&amp;pmid=37844237&amp;doi=10.1073/pnas.2310138120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref019">
<span class="label">19.</span><cite>Colonese AC, Hendy J, Lucquin A, Speller CF, Collins MJ, Carrer F, et al. New criteria for the molecular identification of cereal grains associated with archaeological artefacts. Sci Rep. 2017;7(1):6633. doi: 10.1038/s41598-017-06390-x

</cite> [<a href="https://doi.org/10.1038/s41598-017-06390-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5529501/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28747692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=New%20criteria%20for%20the%20molecular%20identification%20of%20cereal%20grains%20associated%20with%20archaeological%20artefacts&amp;author=AC%20Colonese&amp;author=J%20Hendy&amp;author=A%20Lucquin&amp;author=CF%20Speller&amp;author=MJ%20Collins&amp;volume=7&amp;issue=1&amp;publication_year=2017&amp;pages=6633&amp;pmid=28747692&amp;doi=10.1038/s41598-017-06390-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref020">
<span class="label">20.</span><cite>Courel B, Meadows J, Carretero LG, Lucquin A, McLaughlin R, Bondetti M, et al. The use of early pottery by hunter-gatherers of the Eastern European forest-steppe. Quaternary Science Reviews. 2021;269:107143. doi: 10.1016/j.quascirev.2021.107143</cite> [<a href="https://doi.org/10.1016/j.quascirev.2021.107143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Quaternary%20Science%20Reviews&amp;title=The%20use%20of%20early%20pottery%20by%20hunter-gatherers%20of%20the%20Eastern%20European%20forest-steppe&amp;author=B%20Courel&amp;author=J%20Meadows&amp;author=LG%20Carretero&amp;author=A%20Lucquin&amp;author=R%20McLaughlin&amp;volume=269&amp;publication_year=2021&amp;pages=107143&amp;doi=10.1016/j.quascirev.2021.107143&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref021">
<span class="label">21.</span><cite>Kubiak-Martens L, Oudemans TFM, Brozio JP, Filipović D, Müller J, Kirleis W. Transformation of cereal grains: Botanical and chemical analysis of food residues encrusted on pottery from the Funnel Beaker settlement of Oldenburg LA 77, northern Germany. PLoS One. 2024;19(1):e0296986. doi: 10.1371/journal.pone.0296986

</cite> [<a href="https://doi.org/10.1371/journal.pone.0296986" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10798637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38241419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Transformation%20of%20cereal%20grains:%20Botanical%20and%20chemical%20analysis%20of%20food%20residues%20encrusted%20on%20pottery%20from%20the%20Funnel%20Beaker%20settlement%20of%20Oldenburg%20LA%2077,%20northern%20Germany&amp;author=L%20Kubiak-Martens&amp;author=TFM%20Oudemans&amp;author=JP%20Brozio&amp;author=D%20Filipovi%C4%87&amp;author=J%20M%C3%BCller&amp;volume=19&amp;issue=1&amp;publication_year=2024&amp;pmid=38241419&amp;doi=10.1371/journal.pone.0296986&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref022">
<span class="label">22.</span><cite>Kubiak-Martens L, Brinkkemper O, Oudemans TFM. What’s for dinner? Processed food in the coastal area of the northern Netherlands in the Late Neolithic. Veget Hist Archaeobot. 2014;24(1):47–62. doi: 10.1007/s00334-014-0485-8</cite> [<a href="https://doi.org/10.1007/s00334-014-0485-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Veget%20Hist%20Archaeobot&amp;title=What%E2%80%99s%20for%20dinner?%20Processed%20food%20in%20the%20coastal%20area%20of%20the%20northern%20Netherlands%20in%20the%20Late%20Neolithic&amp;author=L%20Kubiak-Martens&amp;author=O%20Brinkkemper&amp;author=TFM%20Oudemans&amp;volume=24&amp;issue=1&amp;publication_year=2014&amp;pages=47-62&amp;doi=10.1007/s00334-014-0485-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref023">
<span class="label">23.</span><cite>Warinner C, Hendy J, Speller C, Cappellini E, Fischer R, Trachsel C, et al. Direct evidence of milk consumption from ancient human dental calculus. Sci Rep. 2014;4:7104. doi: 10.1038/srep07104

</cite> [<a href="https://doi.org/10.1038/srep07104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4245811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25429530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Direct%20evidence%20of%20milk%20consumption%20from%20ancient%20human%20dental%20calculus&amp;author=C%20Warinner&amp;author=J%20Hendy&amp;author=C%20Speller&amp;author=E%20Cappellini&amp;author=R%20Fischer&amp;volume=4&amp;publication_year=2014&amp;pages=7104&amp;pmid=25429530&amp;doi=10.1038/srep07104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref024">
<span class="label">24.</span><cite>Charlton S, Ramsøe A, Collins M, Craig OE, Fischer R, Alexander M, et al. New insights into Neolithic milk consumption through proteomic analysis of dental calculus. Archaeol Anthropol Sci. 2019;11(11):6183–96. doi: 10.1007/s12520-019-00911-7</cite> [<a href="https://doi.org/10.1007/s12520-019-00911-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archaeol%20Anthropol%20Sci&amp;title=New%20insights%20into%20Neolithic%20milk%20consumption%20through%20proteomic%20analysis%20of%20dental%20calculus&amp;author=S%20Charlton&amp;author=A%20Rams%C3%B8e&amp;author=M%20Collins&amp;author=OE%20Craig&amp;author=R%20Fischer&amp;volume=11&amp;issue=11&amp;publication_year=2019&amp;pages=6183-96&amp;doi=10.1007/s12520-019-00911-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref025">
<span class="label">25.</span><cite>Bleasdale M, Richter KK, Janzen A, Brown S, Scott A, Zech J, et al. Ancient proteins provide evidence of dairy consumption in eastern Africa. Nat Commun. 2021;12(1):632. doi: 10.1038/s41467-020-20682-3

</cite> [<a href="https://doi.org/10.1038/s41467-020-20682-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7841170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33504791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Ancient%20proteins%20provide%20evidence%20of%20dairy%20consumption%20in%20eastern%20Africa&amp;author=M%20Bleasdale&amp;author=KK%20Richter&amp;author=A%20Janzen&amp;author=S%20Brown&amp;author=A%20Scott&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=632&amp;pmid=33504791&amp;doi=10.1038/s41467-020-20682-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref026">
<span class="label">26.</span><cite>Evans M, Lundy J, Lucquin A, Hagan R, Kowalski Ł, Wilczyńki J, et al. Detection of dairy products from multiple taxa in Late Neolithic pottery from Poland: an integrated biomolecular approach. R Soc Open Sci. 2023;10(3):230124. doi: 10.1098/rsos.230124

</cite> [<a href="https://doi.org/10.1098/rsos.230124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10014250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36938542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=R%20Soc%20Open%20Sci&amp;title=Detection%20of%20dairy%20products%20from%20multiple%20taxa%20in%20Late%20Neolithic%20pottery%20from%20Poland:%20an%20integrated%20biomolecular%20approach&amp;author=M%20Evans&amp;author=J%20Lundy&amp;author=A%20Lucquin&amp;author=R%20Hagan&amp;author=%C5%81%20Kowalski&amp;volume=10&amp;issue=3&amp;publication_year=2023&amp;pages=230124&amp;pmid=36938542&amp;doi=10.1098/rsos.230124&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref027">
<span class="label">27.</span><cite>Solazzo C, Fitzhugh WW, Rolando C, Tokarski C. Identification of protein remains in archaeological potsherds by proteomics. Anal Chem. 2008;80(12):4590–7. doi: 10.1021/ac800515v

</cite> [<a href="https://doi.org/10.1021/ac800515v" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18494502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal%20Chem&amp;title=Identification%20of%20protein%20remains%20in%20archaeological%20potsherds%20by%20proteomics&amp;author=C%20Solazzo&amp;author=WW%20Fitzhugh&amp;author=C%20Rolando&amp;author=C%20Tokarski&amp;volume=80&amp;issue=12&amp;publication_year=2008&amp;pages=4590-7&amp;pmid=18494502&amp;doi=10.1021/ac800515v&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref028">
<span class="label">28.</span><cite>Tanasi D, Cucina A, Cunsolo V, Saletti R, Di Francesco A, Greco E, et al. Paleoproteomic profiling of organic residues on prehistoric pottery from Malta. Amino Acids. 2021;53(2):295–312. doi: 10.1007/s00726-021-02946-4

</cite> [<a href="https://doi.org/10.1007/s00726-021-02946-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7910365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33582869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Amino%20Acids&amp;title=Paleoproteomic%20profiling%20of%20organic%20residues%20on%20prehistoric%20pottery%20from%20Malta&amp;author=D%20Tanasi&amp;author=A%20Cucina&amp;author=V%20Cunsolo&amp;author=R%20Saletti&amp;author=A%20Di%20Francesco&amp;volume=53&amp;issue=2&amp;publication_year=2021&amp;pages=295-312&amp;pmid=33582869&amp;doi=10.1007/s00726-021-02946-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref029">
<span class="label">29.</span><cite>Lyu N, Yan L, Wang T, Lin L, Rao H, Yang Y. Characterization of pottery foodcrusts through lipid and proteomic analyses: A case study from the Xiawan site in Yixing city, East China. J Archaeol Sci. 2024;161:105902. doi: 10.1016/j.jas.2023.105902</cite> [<a href="https://doi.org/10.1016/j.jas.2023.105902" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Archaeol%20Sci&amp;title=Characterization%20of%20pottery%20foodcrusts%20through%20lipid%20and%20proteomic%20analyses:%20A%20case%20study%20from%20the%20Xiawan%20site%20in%20Yixing%20city,%20East%20China&amp;author=N%20Lyu&amp;author=L%20Yan&amp;author=T%20Wang&amp;author=L%20Lin&amp;author=H%20Rao&amp;volume=161&amp;publication_year=2024&amp;pages=105902&amp;doi=10.1016/j.jas.2023.105902&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref030">
<span class="label">30.</span><cite>Evans M, Hagan R, Boyd OJ, Bondetti M, Craig OE, Collins MJ, et al. The impact of cooking and burial on proteins: a characterisation of experimental foodcrusts and ceramics. R Soc Open Sci. 2024;11(9):240610. doi: 10.1098/rsos.240610

</cite> [<a href="https://doi.org/10.1098/rsos.240610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11482021/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39416716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=R%20Soc%20Open%20Sci&amp;title=The%20impact%20of%20cooking%20and%20burial%20on%20proteins:%20a%20characterisation%20of%20experimental%20foodcrusts%20and%20ceramics&amp;author=M%20Evans&amp;author=R%20Hagan&amp;author=OJ%20Boyd&amp;author=M%20Bondetti&amp;author=OE%20Craig&amp;volume=11&amp;issue=9&amp;publication_year=2024&amp;pages=240610&amp;pmid=39416716&amp;doi=10.1098/rsos.240610&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref031">
<span class="label">31.</span><cite>Schroeter ER, DeHart CJ, Schweitzer MH, Thomas PM, Kelleher NL. Bone protein “extractomics”: comparing the efficiency of bone protein extractions of Gallus gallus in tandem mass spectrometry, with an eye towards paleoproteomics. PeerJ. 2016;4:e2603. doi: 10.7717/peerj.2603

</cite> [<a href="https://doi.org/10.7717/peerj.2603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5088622/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27812413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Bone%20protein%20%E2%80%9Cextractomics%E2%80%9D:%20comparing%20the%20efficiency%20of%20bone%20protein%20extractions%20of%20Gallus%20gallus%20in%20tandem%20mass%20spectrometry,%20with%20an%20eye%20towards%20paleoproteomics&amp;author=ER%20Schroeter&amp;author=CJ%20DeHart&amp;author=MH%20Schweitzer&amp;author=PM%20Thomas&amp;author=NL%20Kelleher&amp;volume=4&amp;publication_year=2016&amp;pmid=27812413&amp;doi=10.7717/peerj.2603&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref032">
<span class="label">32.</span><cite>Kubiak-Martens L. The plant food component of the diet at the late Mesolithic (Ertebolle) settlement at Tybrind Vig, Denmark. Veget Hist Archaebot. 1999;8(1–2):117–27. doi: 10.1007/bf02042850</cite> [<a href="https://doi.org/10.1007/bf02042850" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Veget%20Hist%20Archaebot&amp;title=The%20plant%20food%20component%20of%20the%20diet%20at%20the%20late%20Mesolithic%20(Ertebolle)%20settlement%20at%20Tybrind%20Vig,%20Denmark&amp;author=L%20Kubiak-Martens&amp;volume=8&amp;publication_year=1999&amp;pages=117-27&amp;doi=10.1007/bf02042850&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref033">
<span class="label">33.</span><cite>Mason S. Archaeobotanical analysis-Møllegabet II. In: Skaarup J, Grøn, editors. Møllegabet II, A submerged Mesolithic settlement in southern Denmark. Archaeopress. 2004:122–43.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=M%C3%B8llegabet%20II,%20A%20submerged%20Mesolithic%20settlement%20in%20southern%20Denmark&amp;author=S%20Mason&amp;publication_year=2004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref034">
<span class="label">34.</span><cite>UniProt Consortium. UniProt: the Universal Protein Knowledgebase in 2023. Nucleic Acids Res. 2023;51(D1):D523–31. doi: 10.1093/nar/gkac1052

</cite> [<a href="https://doi.org/10.1093/nar/gkac1052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36408920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=UniProt:%20the%20Universal%20Protein%20Knowledgebase%20in%202023&amp;volume=51&amp;publication_year=2023&amp;pmid=36408920&amp;doi=10.1093/nar/gkac1052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref035">
<span class="label">35.</span><cite>Kubiak-Martens L. New evidence for the use of root foods in pre-agrarian subsistence recovered from the late Mesolithic site at Halsskov, Denmark. Veget Hist Archaeobot. 2002;11(1–2):23–32. doi: 10.1007/s003340200003</cite> [<a href="https://doi.org/10.1007/s003340200003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Veget%20Hist%20Archaeobot&amp;title=New%20evidence%20for%20the%20use%20of%20root%20foods%20in%20pre-agrarian%20subsistence%20recovered%20from%20the%20late%20Mesolithic%20site%20at%20Halsskov,%20Denmark&amp;author=L%20Kubiak-Martens&amp;volume=11&amp;publication_year=2002&amp;pages=23-32&amp;doi=10.1007/s003340200003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref036">
<span class="label">36.</span><cite>Price DT, Gebauer AB, Hede US, Larsen CS, Noe-Nygaard N, Mason SLR, et al. Smakkerup Huse: A Mesolithic Settlement in NW Zealand, Denmark. J Field Archaeol. 2001;28(1–2):45–67. doi: 10.1179/jfa.2001.28.1-2.45</cite> [<a href="https://doi.org/10.1179/jfa.2001.28.1-2.45" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Field%20Archaeol&amp;title=Smakkerup%20Huse:%20A%20Mesolithic%20Settlement%20in%20NW%20Zealand,%20Denmark&amp;author=DT%20Price&amp;author=AB%20Gebauer&amp;author=US%20Hede&amp;author=CS%20Larsen&amp;author=N%20Noe-Nygaard&amp;volume=28&amp;publication_year=2001&amp;pages=45-67&amp;doi=10.1179/jfa.2001.28.1-2.45&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref037">
<span class="label">37.</span><cite>Holst D. Hazelnut economy of early Holocene hunter–gatherers: a case study from Mesolithic Duvensee, northern Germany. J Archaeol Sci. 2010;37(11):2871–80. doi: 10.1016/j.jas.2010.06.028</cite> [<a href="https://doi.org/10.1016/j.jas.2010.06.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Archaeol%20Sci&amp;title=Hazelnut%20economy%20of%20early%20Holocene%20hunter%E2%80%93gatherers:%20a%20case%20study%20from%20Mesolithic%20Duvensee,%20northern%20Germany&amp;author=D%20Holst&amp;volume=37&amp;issue=11&amp;publication_year=2010&amp;pages=2871-80&amp;doi=10.1016/j.jas.2010.06.028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref038">
<span class="label">38.</span><cite>Robinson DE, Harild JA.
Archaeobotany of an early Ertebölle (late Mesolithic) site at Halsskov, Zealand, Denmark. In: Mason SLR, Hather J, editors. Hunter-Gatherer Archaeobotany: Perspectives from the Northern Temperate Zone. London, England: Routledge. 2009.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Hunter-Gatherer%20Archaeobotany:%20Perspectives%20from%20the%20Northern%20Temperate%20Zone&amp;author=DE%20Robinson&amp;author=JA.%20Harild&amp;author=SLR%20Mason&amp;author=J%20Hather&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref039">
<span class="label">39.</span><cite>Richter J, Noe-Nygaard N. A Late Mesolithic Hunting Station at Agernæs, Fyn, Denmark. ACAR. 2003;74(1):1–32. doi: 10.1111/j.0065-001x.2003.aar740101.x</cite> [<a href="https://doi.org/10.1111/j.0065-001x.2003.aar740101.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACAR&amp;title=A%20Late%20Mesolithic%20Hunting%20Station%20at%20Agern%C3%A6s,%20Fyn,%20Denmark&amp;author=J%20Richter&amp;author=N%20Noe-Nygaard&amp;volume=74&amp;issue=1&amp;publication_year=2003&amp;pages=1-32&amp;doi=10.1111/j.0065-001x.2003.aar740101.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref040">
<span class="label">40.</span><cite>Astrup PM, Benjamin J, Stankiewicz F, Woo K, McCarthy J, Wiseman C, et al. A drowned Mesolithic shell midden complex at Hjarnø Vesterhoved, Denmark and its wider significance. Quaternary Science Reviews. 2021;258:106854. doi: 10.1016/j.quascirev.2021.106854</cite> [<a href="https://doi.org/10.1016/j.quascirev.2021.106854" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Quaternary%20Science%20Reviews&amp;title=A%20drowned%20Mesolithic%20shell%20midden%20complex%20at%20Hjarn%C3%B8%20Vesterhoved,%20Denmark%20and%20its%20wider%20significance&amp;author=PM%20Astrup&amp;author=J%20Benjamin&amp;author=F%20Stankiewicz&amp;author=K%20Woo&amp;author=J%20McCarthy&amp;volume=258&amp;publication_year=2021&amp;pages=106854&amp;doi=10.1016/j.quascirev.2021.106854&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref041">
<span class="label">41.</span><cite>Enghoff I. Regionality and biotope exploitation in Danish Ertebølle and adjoining periods. Det Kongelige Danske Videnskabernes Selskab. 2011.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Regionality%20and%20biotope%20exploitation%20in%20Danish%20Erteb%C3%B8lle%20and%20adjoining%20periods&amp;author=I%20Enghoff&amp;publication_year=2011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref042">
<span class="label">42.</span><cite>Price TD, Ritchie K, Gron KJ, Gebauer AB, Nielsen J. Asnæs Havnemark: a late Mesolithic Ertebølle coastal site in western Sjælland, Denmark. Danish J Archaeology. 2018;7(2):255–76. doi: 10.1080/21662282.2018.1551979</cite> [<a href="https://doi.org/10.1080/21662282.2018.1551979" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Danish%20J%20Archaeology&amp;title=Asn%C3%A6s%20Havnemark:%20a%20late%20Mesolithic%20Erteb%C3%B8lle%20coastal%20site%20in%20western%20Sj%C3%A6lland,%20Denmark&amp;author=TD%20Price&amp;author=K%20Ritchie&amp;author=KJ%20Gron&amp;author=AB%20Gebauer&amp;author=J%20Nielsen&amp;volume=7&amp;issue=2&amp;publication_year=2018&amp;pages=255-76&amp;doi=10.1080/21662282.2018.1551979&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref043">
<span class="label">43.</span><cite>Bondetti M, Scott E, Courel B, Lucquin A, Shoda S, Lundy J, et al. Investigating the formation and diagnostic value of ω-(o-alkylphenyl)alkanoic acids in ancient pottery. Archaeometry. 2021;63(3):594–608. doi: 10.1111/arcm.12631

</cite> [<a href="https://doi.org/10.1111/arcm.12631" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8247306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34219747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archaeometry&amp;title=Investigating%20the%20formation%20and%20diagnostic%20value%20of%20%CF%89-(o-alkylphenyl)alkanoic%20acids%20in%20ancient%20pottery&amp;author=M%20Bondetti&amp;author=E%20Scott&amp;author=B%20Courel&amp;author=A%20Lucquin&amp;author=S%20Shoda&amp;volume=63&amp;issue=3&amp;publication_year=2021&amp;pages=594-608&amp;pmid=34219747&amp;doi=10.1111/arcm.12631&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref044">
<span class="label">44.</span><cite>Page LM, B M B. Peterson Field Guide to Freshwater Fishes of North America North of Mexico. Houghton Mifflin Harcourt. 2011.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Peterson%20Field%20Guide%20to%20Freshwater%20Fishes%20of%20North%20America%20North%20of%20Mexico&amp;author=LM%20Page&amp;author=B%20B%20M&amp;publication_year=2011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref045">
<span class="label">45.</span><cite>Met Office National Climate Information Centre. Areal values from HadUK-Grid 1km gridded climate data from land surface network. 2025. <a href="https://www.metoffice.gov.uk/pub/data/weather/uk/climate/datasets/Tmean/date/England_N.txt" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.metoffice.gov.uk/pub/data/weather/uk/climate/datasets/Tmean/date/England_N.txt</a></cite>
</li>
<li id="pone.0330195.ref046">
<span class="label">46.</span><cite>Cleland TP. Human Bone Paleoproteomics Utilizing the Single-Pot, Solid-Phase-Enhanced Sample Preparation Method to Maximize Detected Proteins and Reduce Humics. J Proteome Res. 2018;17(11):3976–83. doi: 10.1021/acs.jproteome.8b00637

</cite> [<a href="https://doi.org/10.1021/acs.jproteome.8b00637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30336043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Proteome%20Res&amp;title=Human%20Bone%20Paleoproteomics%20Utilizing%20the%20Single-Pot,%20Solid-Phase-Enhanced%20Sample%20Preparation%20Method%20to%20Maximize%20Detected%20Proteins%20and%20Reduce%20Humics&amp;author=TP%20Cleland&amp;volume=17&amp;issue=11&amp;publication_year=2018&amp;pages=3976-83&amp;pmid=30336043&amp;doi=10.1021/acs.jproteome.8b00637&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref047">
<span class="label">47.</span><cite>Harvey VL, Hendy J. SP3 protocol optimised for foodcrust protein extraction from archaeological cooking vessels. <a href="https://www.protocols.io/view/sp3-protocol-optimised-for-foodcrust-protein-extra-dyge7tte.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.protocols.io/view/sp3-protocol-optimised-for-foodcrust-protein-extra-dyge7tte.pdf</a> 2025.</cite>
</li>
<li id="pone.0330195.ref048">
<span class="label">48.</span><cite>Dekker J, Larson T, Tzvetkov J, Harvey VL, Dowle A, Hagan R, et al. Spatial analysis of the ancient proteome of archeological teeth using mass spectrometry imaging. Rapid Commun Mass Spectrom. 2023;37(8):e9486. doi: 10.1002/rcm.9486

</cite> [<a href="https://doi.org/10.1002/rcm.9486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36735645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Commun%20Mass%20Spectrom&amp;title=Spatial%20analysis%20of%20the%20ancient%20proteome%20of%20archeological%20teeth%20using%20mass%20spectrometry%20imaging&amp;author=J%20Dekker&amp;author=T%20Larson&amp;author=J%20Tzvetkov&amp;author=VL%20Harvey&amp;author=A%20Dowle&amp;volume=37&amp;issue=8&amp;publication_year=2023&amp;pmid=36735645&amp;doi=10.1002/rcm.9486&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref049">
<span class="label">49.</span><cite>Solntsev SK, Shortreed MR, Frey BL, Smith LM. Enhanced Global Post-translational Modification Discovery with MetaMorpheus. J Proteome Res. 2018;17(5):1844–51. doi: 10.1021/acs.jproteome.7b00873

</cite> [<a href="https://doi.org/10.1021/acs.jproteome.7b00873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29578715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Proteome%20Res&amp;title=Enhanced%20Global%20Post-translational%20Modification%20Discovery%20with%20MetaMorpheus&amp;author=SK%20Solntsev&amp;author=MR%20Shortreed&amp;author=BL%20Frey&amp;author=LM%20Smith&amp;volume=17&amp;issue=5&amp;publication_year=2018&amp;pages=1844-51&amp;pmid=29578715&amp;doi=10.1021/acs.jproteome.7b00873&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref050">
<span class="label">50.</span><cite>Prus G, Hoegl A, Weinert BT, Choudhary C. Analysis and Interpretation of Protein Post-Translational Modification Site Stoichiometry. Trends Biochem Sci. 2019;44(11):943–60. doi: 10.1016/j.tibs.2019.06.003

</cite> [<a href="https://doi.org/10.1016/j.tibs.2019.06.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31296352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem%20Sci&amp;title=Analysis%20and%20Interpretation%20of%20Protein%20Post-Translational%20Modification%20Site%20Stoichiometry&amp;author=G%20Prus&amp;author=A%20Hoegl&amp;author=BT%20Weinert&amp;author=C%20Choudhary&amp;volume=44&amp;issue=11&amp;publication_year=2019&amp;pages=943-60&amp;pmid=31296352&amp;doi=10.1016/j.tibs.2019.06.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref051">
<span class="label">51.</span><cite>The Global Proteome Machine Organization. cRAP protein sequences. The Global Proteome Machine. <a href="https://www.thegpm.org/crap/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.thegpm.org/crap/</a>. 2012.</cite>
</li>
<li id="pone.0330195.ref052">
<span class="label">52.</span><cite>Hendy J, Welker F, Demarchi B, Speller C, Warinner C, Collins MJ. A guide to ancient protein studies. Nat Ecol Evol. 2018;2(5):791–9. doi: 10.1038/s41559-018-0510-x

</cite> [<a href="https://doi.org/10.1038/s41559-018-0510-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29581591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Ecol%20Evol&amp;title=A%20guide%20to%20ancient%20protein%20studies&amp;author=J%20Hendy&amp;author=F%20Welker&amp;author=B%20Demarchi&amp;author=C%20Speller&amp;author=C%20Warinner&amp;volume=2&amp;issue=5&amp;publication_year=2018&amp;pages=791-9&amp;pmid=29581591&amp;doi=10.1038/s41559-018-0510-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref053">
<span class="label">53.</span><cite>R Core Team. R: A Language and Environment for Statistical Computing. Vienna, Austria: R Foundation for Statistical Computing. 2021.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=R:%20A%20Language%20and%20Environment%20for%20Statistical%20Computing&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref054">
<span class="label">54.</span><cite>Posit team. RStudio: Integrated Development Environment for R. Boston, MA: Posit Software, PBC. 2024.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=RStudio:%20Integrated%20Development%20Environment%20for%20R&amp;author=team%20Posit&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref055">
<span class="label">55.</span><cite>Public Health England. McCance and Widdowson’s The Composition of Foods Integrated Dataset 2021. 2021. Available from: <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/971021/McCance_and_Widdowsons_Composition_of_Foods_integrated_dataset_2021.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/971021/McCance_and_Widdowsons_Composition_of_Foods_integrated_dataset_2021.pdf</a></cite>
</li>
<li id="pone.0330195.ref056">
<span class="label">56.</span><cite>Kruskal WH, Wallis WA. Use of Ranks in One-Criterion Variance Analysis. J Am Stat Assoc. 1952;47(260):583–621. doi: 10.1080/01621459.1952.10483441</cite> [<a href="https://doi.org/10.1080/01621459.1952.10483441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Stat%20Assoc&amp;title=Use%20of%20Ranks%20in%20One-Criterion%20Variance%20Analysis&amp;author=WH%20Kruskal&amp;author=WA%20Wallis&amp;volume=47&amp;issue=260&amp;publication_year=1952&amp;pages=583-621&amp;doi=10.1080/01621459.1952.10483441&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref057">
<span class="label">57.</span><cite>Dunn OJ. Multiple Comparisons among Means. J Am Stat Assoc. 1961;56(293):52–64. doi: 10.1080/01621459.1961.10482090</cite> [<a href="https://doi.org/10.1080/01621459.1961.10482090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Stat%20Assoc&amp;title=Multiple%20Comparisons%20among%20Means&amp;author=OJ%20Dunn&amp;volume=56&amp;issue=293&amp;publication_year=1961&amp;pages=52-64&amp;doi=10.1080/01621459.1961.10482090&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref058">
<span class="label">58.</span><cite>Dinno A. dunn.test: Dunn’s Test of Multiple Comparisons Using Rank Sums. Available from: <a href="https://rdrr.io/cran/dunn.test/man/dunn.test.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://rdrr.io/cran/dunn.test/man/dunn.test.html</a>. 2017.</cite>
</li>
<li id="pone.0330195.ref059">
<span class="label">59.</span><cite>Sievers F, Higgins DG. Clustal Omega for making accurate alignments of many protein sequences. Protein Sci. 2018;27: 135–145. doi: 10.1002/pro.329060</cite> [<a href="https://doi.org/10.1002/pro.329060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5734385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28884485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Clustal%20Omega%20for%20making%20accurate%20alignments%20of%20many%20protein%20sequences&amp;author=F%20Sievers&amp;author=DG%20Higgins&amp;volume=2018&amp;pages=135-145&amp;pmid=28884485&amp;doi=10.1002/pro.329060&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref060">
<span class="label">60.</span><cite>Wimley WC, White SH. Experimentally determined hydrophobicity scale for proteins at membrane interfaces. Nat Struct Biol. 1996;3(10):842–8. doi: 10.1038/nsb1096-842

</cite> [<a href="https://doi.org/10.1038/nsb1096-842" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8836100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Struct%20Biol&amp;title=Experimentally%20determined%20hydrophobicity%20scale%20for%20proteins%20at%20membrane%20interfaces&amp;author=WC%20Wimley&amp;author=SH%20White&amp;volume=3&amp;issue=10&amp;publication_year=1996&amp;pages=842-8&amp;pmid=8836100&amp;doi=10.1038/nsb1096-842&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref061">
<span class="label">61.</span><cite>Morton-Hayward AL, Anderson RP, Saupe EE, Larson G, Cosmidis JG. Human brains preserve in diverse environments for at least 12 000 years. Proc Biol Sci. 2024;291(2019):20232606. doi: 10.1098/rspb.2023.2606

</cite> [<a href="https://doi.org/10.1098/rspb.2023.2606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10950470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38503334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Biol%20Sci&amp;title=Human%20brains%20preserve%20in%20diverse%20environments%20for%20at%20least%2012%20000%20years&amp;author=AL%20Morton-Hayward&amp;author=RP%20Anderson&amp;author=EE%20Saupe&amp;author=G%20Larson&amp;author=JG%20Cosmidis&amp;volume=291&amp;issue=2019&amp;publication_year=2024&amp;pages=20232606&amp;pmid=38503334&amp;doi=10.1098/rspb.2023.2606&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref062">
<span class="label">62.</span><cite>Yu T-Y, Morton JD, Clerens S, Dyer JM. Proteomic investigation of protein profile changes and amino acid residue-level modification in cooked lamb longissimus thoracis et lumborum: The effect of roasting. Meat Sci. 2016;119:80–8. doi: 10.1016/j.meatsci.2016.04.024

</cite> [<a href="https://doi.org/10.1016/j.meatsci.2016.04.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27150797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Meat%20Sci&amp;title=Proteomic%20investigation%20of%20protein%20profile%20changes%20and%20amino%20acid%20residue-level%20modification%20in%20cooked%20lamb%20longissimus%20thoracis%20et%20lumborum:%20The%20effect%20of%20roasting&amp;author=T-Y%20Yu&amp;author=JD%20Morton&amp;author=S%20Clerens&amp;author=JM%20Dyer&amp;volume=119&amp;publication_year=2016&amp;pages=80-8&amp;pmid=27150797&amp;doi=10.1016/j.meatsci.2016.04.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref063">
<span class="label">63.</span><cite>Yu T-Y, Morton JD, Clerens S, Dyer JM. Cooking-Induced Protein Modifications in Meat. Compr Rev Food Sci Food Saf. 2017;16(1):141–59. doi: 10.1111/1541-4337.12243

</cite> [<a href="https://doi.org/10.1111/1541-4337.12243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33371543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Compr%20Rev%20Food%20Sci%20Food%20Saf&amp;title=Cooking-Induced%20Protein%20Modifications%20in%20Meat&amp;author=T-Y%20Yu&amp;author=JD%20Morton&amp;author=S%20Clerens&amp;author=JM%20Dyer&amp;volume=16&amp;issue=1&amp;publication_year=2017&amp;pages=141-59&amp;pmid=33371543&amp;doi=10.1111/1541-4337.12243&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref064">
<span class="label">64.</span><cite>Snel B, Lehmann G, Bork P, Huynen MA. STRING: a web-server to retrieve and display the repeatedly occurring neighbourhood of a gene. Nucleic Acids Res. 2000;28(18):3442–4. doi: 10.1093/nar/28.18.3442

</cite> [<a href="https://doi.org/10.1093/nar/28.18.3442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC110752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10982861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=STRING:%20a%20web-server%20to%20retrieve%20and%20display%20the%20repeatedly%20occurring%20neighbourhood%20of%20a%20gene&amp;author=B%20Snel&amp;author=G%20Lehmann&amp;author=P%20Bork&amp;author=MA%20Huynen&amp;volume=28&amp;issue=18&amp;publication_year=2000&amp;pages=3442-4&amp;pmid=10982861&amp;doi=10.1093/nar/28.18.3442&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref065">
<span class="label">65.</span><cite>Szklarczyk D, Kirsch R, Koutrouli M, Nastou K, Mehryary F, Hachilif R, et al. The STRING database in 2023: protein-protein association networks and functional enrichment analyses for any sequenced genome of interest. Nucleic Acids Res. 2023;51(D1):D638–46. doi: 10.1093/nar/gkac1000

</cite> [<a href="https://doi.org/10.1093/nar/gkac1000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36370105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20STRING%20database%20in%202023:%20protein-protein%20association%20networks%20and%20functional%20enrichment%20analyses%20for%20any%20sequenced%20genome%20of%20interest&amp;author=D%20Szklarczyk&amp;author=R%20Kirsch&amp;author=M%20Koutrouli&amp;author=K%20Nastou&amp;author=F%20Mehryary&amp;volume=51&amp;publication_year=2023&amp;pmid=36370105&amp;doi=10.1093/nar/gkac1000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref066">
<span class="label">66.</span><cite>Palasca O, Santos A, Stolte C, Gorodkin J, Jensen LJ. TISSUES 2.0: an integrative web resource on mammalian tissue expression. Database (Oxford). 2018;2018:bay003. doi: 10.1093/database/bay003

</cite> [<a href="https://doi.org/10.1093/database/bay003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5808782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29617745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Database%20(Oxford)&amp;title=TISSUES%202.0:%20an%20integrative%20web%20resource%20on%20mammalian%20tissue%20expression&amp;author=O%20Palasca&amp;author=A%20Santos&amp;author=C%20Stolte&amp;author=J%20Gorodkin&amp;author=LJ%20Jensen&amp;volume=2018&amp;publication_year=2018&amp;pmid=29617745&amp;doi=10.1093/database/bay003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref067">
<span class="label">67.</span><cite>Ortner DJ, vonEndt DW, Robinson MS. The Effect of Temperature on Protein Decay in Bone: Its Significance in Nitrogen Dating of Archaeological Specimens. Am antiq. 1972;37(4):514–20. doi: 10.2307/278957</cite> [<a href="https://doi.org/10.2307/278957" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20antiq&amp;title=The%20Effect%20of%20Temperature%20on%20Protein%20Decay%20in%20Bone:%20Its%20Significance%20in%20Nitrogen%20Dating%20of%20Archaeological%20Specimens&amp;author=DJ%20Ortner&amp;author=DW%20vonEndt&amp;author=MS%20Robinson&amp;volume=37&amp;issue=4&amp;publication_year=1972&amp;pages=514-20&amp;doi=10.2307/278957&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref068">
<span class="label">68.</span><cite>Collins MJ, Nielsen–Marsh CM, Hiller J, Smith CI, Roberts JP, Prigodich RV, et al. The survival of organic matter in bone: a review. Archaeometry. 2002;44(3):383–94. doi: 10.1111/1475-4754.t01-1-00071</cite> [<a href="https://doi.org/10.1111/1475-4754.t01-1-00071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archaeometry&amp;title=The%20survival%20of%20organic%20matter%20in%20bone:%20a%20review&amp;author=MJ%20Collins&amp;author=CM%20Nielsen%E2%80%93Marsh&amp;author=J%20Hiller&amp;author=CI%20Smith&amp;author=JP%20Roberts&amp;volume=44&amp;issue=3&amp;publication_year=2002&amp;pages=383-94&amp;doi=10.1111/1475-4754.t01-1-00071&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref069">
<span class="label">69.</span><cite>Bax M-L, Aubry L, Ferreira C, Daudin J-D, Gatellier P, Rémond D, et al. Cooking temperature is a key determinant of in vitro meat protein digestion rate: investigation of underlying mechanisms. J Agric Food Chem. 2012;60(10):2569–76. doi: 10.1021/jf205280y

</cite> [<a href="https://doi.org/10.1021/jf205280y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22335241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Agric%20Food%20Chem&amp;title=Cooking%20temperature%20is%20a%20key%20determinant%20of%20in%20vitro%20meat%20protein%20digestion%20rate:%20investigation%20of%20underlying%20mechanisms&amp;author=M-L%20Bax&amp;author=L%20Aubry&amp;author=C%20Ferreira&amp;author=J-D%20Daudin&amp;author=P%20Gatellier&amp;volume=60&amp;issue=10&amp;publication_year=2012&amp;pages=2569-76&amp;pmid=22335241&amp;doi=10.1021/jf205280y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref070">
<span class="label">70.</span><cite>Roberts SJ, Smith CI, Millard A, Collins MJ. The taphonomy of cooked bone: characterizing boiling and its physico–chemical effects. Archaeometry. 2002;44(3):485–94. doi: 10.1111/1475-4754.t01-1-00080</cite> [<a href="https://doi.org/10.1111/1475-4754.t01-1-00080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archaeometry&amp;title=The%20taphonomy%20of%20cooked%20bone:%20characterizing%20boiling%20and%20its%20physico%E2%80%93chemical%20effects&amp;author=SJ%20Roberts&amp;author=CI%20Smith&amp;author=A%20Millard&amp;author=MJ%20Collins&amp;volume=44&amp;issue=3&amp;publication_year=2002&amp;pages=485-94&amp;doi=10.1111/1475-4754.t01-1-00080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref071">
<span class="label">71.</span><cite>Zielbauer BI, Franz J, Viezens B, Vilgis TA. Physical Aspects of Meat Cooking: Time Dependent Thermal Protein Denaturation and Water Loss. Food Biophysics. 2015;11(1):34–42. doi: 10.1007/s11483-015-9410-7</cite> [<a href="https://doi.org/10.1007/s11483-015-9410-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Biophysics&amp;title=Physical%20Aspects%20of%20Meat%20Cooking:%20Time%20Dependent%20Thermal%20Protein%20Denaturation%20and%20Water%20Loss&amp;author=BI%20Zielbauer&amp;author=J%20Franz&amp;author=B%20Viezens&amp;author=TA%20Vilgis&amp;volume=11&amp;issue=1&amp;publication_year=2015&amp;pages=34-42&amp;doi=10.1007/s11483-015-9410-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref072">
<span class="label">72.</span><cite>Ramsøe A, van Heekeren V, Ponce P, Fischer R, Barnes I, Speller C, et al. DeamiDATE 1.0: Site-specific deamidation as a tool to assess authenticity of members of ancient proteomes. J Archaeol Sci. 2020;115:105080. doi: 10.1016/j.jas.2020.105080</cite> [<a href="https://doi.org/10.1016/j.jas.2020.105080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Archaeol%20Sci&amp;title=DeamiDATE%201.0:%20Site-specific%20deamidation%20as%20a%20tool%20to%20assess%20authenticity%20of%20members%20of%20ancient%20proteomes&amp;author=A%20Rams%C3%B8e&amp;author=V%20van%20Heekeren&amp;author=P%20Ponce&amp;author=R%20Fischer&amp;author=I%20Barnes&amp;volume=115&amp;publication_year=2020&amp;pages=105080&amp;doi=10.1016/j.jas.2020.105080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref073">
<span class="label">73.</span><cite>Pal Chowdhury M, Buckley M. Trends in deamidation across archaeological bones, ceramics and dental calculus. Methods. 2022;200:67–79. doi: 10.1016/j.ymeth.2021.08.004

</cite> [<a href="https://doi.org/10.1016/j.ymeth.2021.08.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34450289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Trends%20in%20deamidation%20across%20archaeological%20bones,%20ceramics%20and%20dental%20calculus&amp;author=M%20Pal%20Chowdhury&amp;author=M%20Buckley&amp;volume=200&amp;publication_year=2022&amp;pages=67-79&amp;pmid=34450289&amp;doi=10.1016/j.ymeth.2021.08.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref074">
<span class="label">74.</span><cite>Pal Chowdhury M, Wogelius R, Manning PL, Metz L, Slimak L, Buckley M. Collagen deamidation in archaeological bone as an assessment for relative decay rates. Archaeometry. 2019;61(6):1382–98. doi: 10.1111/arcm.12492</cite> [<a href="https://doi.org/10.1111/arcm.12492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archaeometry&amp;title=Collagen%20deamidation%20in%20archaeological%20bone%20as%20an%20assessment%20for%20relative%20decay%20rates&amp;author=M%20Pal%20Chowdhury&amp;author=R%20Wogelius&amp;author=PL%20Manning&amp;author=L%20Metz&amp;author=L%20Slimak&amp;volume=61&amp;issue=6&amp;publication_year=2019&amp;pages=1382-98&amp;doi=10.1111/arcm.12492&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref075">
<span class="label">75.</span><cite>Maduro Dias CSAM, Nunes HP, Melo TMMV, Rosa HJD, Silva CCG, Borba AES. Application of Near Infrared Reflectance (NIR) spectroscopy to predict the moisture, protein, and fat content of beef for gourmet hamburger preparation. Livestock Science. 2021;254:104772. doi: 10.1016/j.livsci.2021.104772</cite> [<a href="https://doi.org/10.1016/j.livsci.2021.104772" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Livestock%20Science&amp;title=Application%20of%20Near%20Infrared%20Reflectance%20(NIR)%20spectroscopy%20to%20predict%20the%20moisture,%20protein,%20and%20fat%20content%20of%20beef%20for%20gourmet%20hamburger%20preparation&amp;author=CSAM%20Maduro%20Dias&amp;author=HP%20Nunes&amp;author=TMMV%20Melo&amp;author=HJD%20Rosa&amp;author=CCG%20Silva&amp;volume=254&amp;publication_year=2021&amp;pages=104772&amp;doi=10.1016/j.livsci.2021.104772&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref076">
<span class="label">76.</span><cite>Chiang Y, Welker F, Collins MJ. Spectra without stories: reporting 94% dark and unidentified ancient proteomes. Open Res Eur. 2024;4:71. doi: 10.12688/openreseurope.17225.1

</cite> [<a href="https://doi.org/10.12688/openreseurope.17225.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11187534/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38903702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Open%20Res%20Eur&amp;title=Spectra%20without%20stories:%20reporting%2094%%20dark%20and%20unidentified%20ancient%20proteomes&amp;author=Y%20Chiang&amp;author=F%20Welker&amp;author=MJ%20Collins&amp;volume=4&amp;publication_year=2024&amp;pages=71&amp;pmid=38903702&amp;doi=10.12688/openreseurope.17225.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref077">
<span class="label">77.</span><cite>Rodriguez Palomo I, Nair B, Chiang Y, Dekker J, Dartigues B, Mackie M, et al. Benchmarking the identification of a single degraded protein to explore optimal search strategies for ancient proteins. Peer Community Journal. 2024;4. doi: 10.24072/pcjournal.491</cite> [<a href="https://doi.org/10.24072/pcjournal.491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Peer%20Community%20Journal&amp;title=Benchmarking%20the%20identification%20of%20a%20single%20degraded%20protein%20to%20explore%20optimal%20search%20strategies%20for%20ancient%20proteins&amp;author=I%20Rodriguez%20Palomo&amp;author=B%20Nair&amp;author=Y%20Chiang&amp;author=J%20Dekker&amp;author=B%20Dartigues&amp;volume=4&amp;publication_year=2024&amp;doi=10.24072/pcjournal.491&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref078">
<span class="label">78.</span><cite>Cubas M, Becher J, Chiang Y, Antonius Dekker JA, Di Muro A, Doliente JE, et al. Ceramics: Organic Residue Analysis. Encyclopedia of Archaeology (Second Edition). Elsevier. 2024:398–405. doi: 10.1016/b978-0-323-90799-6.00047-1</cite> [<a href="https://doi.org/10.1016/b978-0-323-90799-6.00047-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Encyclopedia%20of%20Archaeology%20(Second%20Edition)&amp;author=M%20Cubas&amp;author=J%20Becher&amp;author=Y%20Chiang&amp;author=JA%20Antonius%20Dekker&amp;author=A%20Di%20Muro&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref079">
<span class="label">79.</span><cite>Whelton HL, Hammann S, Cramp LJE, Dunne J, Roffet-Salque M, Evershed RP. A call for caution in the analysis of lipids and other small biomolecules from archaeological contexts. J Archaeol Sci. 2021;132:105397. doi: 10.1016/j.jas.2021.105397</cite> [<a href="https://doi.org/10.1016/j.jas.2021.105397" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Archaeol%20Sci&amp;title=A%20call%20for%20caution%20in%20the%20analysis%20of%20lipids%20and%20other%20small%20biomolecules%20from%20archaeological%20contexts&amp;author=HL%20Whelton&amp;author=S%20Hammann&amp;author=LJE%20Cramp&amp;author=J%20Dunne&amp;author=M%20Roffet-Salque&amp;volume=132&amp;publication_year=2021&amp;pages=105397&amp;doi=10.1016/j.jas.2021.105397&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref080">
<span class="label">80.</span><cite>Drieu L, Rageot M, Wales N, Stern B, Lundy J, Zerrer M, et al. Is it possible to identify ancient wine production using biomolecular approaches?. STAR: Science &amp; Technology of Archaeological Research. 2020;6(1):16–29. doi: 10.1080/20548923.2020.1738728</cite> [<a href="https://doi.org/10.1080/20548923.2020.1738728" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=STAR:%20Science%20&amp;%20Technology%20of%20Archaeological%20Research&amp;title=Is%20it%20possible%20to%20identify%20ancient%20wine%20production%20using%20biomolecular%20approaches?&amp;author=L%20Drieu&amp;author=M%20Rageot&amp;author=N%20Wales&amp;author=B%20Stern&amp;author=J%20Lundy&amp;volume=6&amp;issue=1&amp;publication_year=2020&amp;pages=16-29&amp;doi=10.1080/20548923.2020.1738728&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref081">
<span class="label">81.</span><cite>Cheung C, Szpak P. Interpreting Past Human Diets Using Stable Isotope Mixing Models. J Archaeol Method Theory. 2020;28(4):1106–42. doi: 10.1007/s10816-020-09492-5</cite> [<a href="https://doi.org/10.1007/s10816-020-09492-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Archaeol%20Method%20Theory&amp;title=Interpreting%20Past%20Human%20Diets%20Using%20Stable%20Isotope%20Mixing%20Models&amp;author=C%20Cheung&amp;author=P%20Szpak&amp;volume=28&amp;issue=4&amp;publication_year=2020&amp;pages=1106-42&amp;doi=10.1007/s10816-020-09492-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref082">
<span class="label">82.</span><cite>Rüther PL, Husic IM, Bangsgaard P, Gregersen KM, Pantmann P, Carvalho M, et al. SPIN enables high throughput species identification of archaeological bone by proteomics. Nat Commun. 2022;13(1):2458. doi: 10.1038/s41467-022-30097-x

</cite> [<a href="https://doi.org/10.1038/s41467-022-30097-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9072323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35513387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=SPIN%20enables%20high%20throughput%20species%20identification%20of%20archaeological%20bone%20by%20proteomics&amp;author=PL%20R%C3%BCther&amp;author=IM%20Husic&amp;author=P%20Bangsgaard&amp;author=KM%20Gregersen&amp;author=P%20Pantmann&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=2458&amp;pmid=35513387&amp;doi=10.1038/s41467-022-30097-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref083">
<span class="label">83.</span><cite>Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. Basic local alignment search tool. J Mol Biol. 1990;215: 403–410. doi:<a href="https://10.1016/S0022-2836(05)80360-284" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">doi:10.1016/S0022-2836(05)80360-284.</a>
</cite> [<a href="https://doi.org/10.1016/S0022-2836(05)80360-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&amp;title=Basic%20local%20alignment%20search%20tool&amp;author=SF%20Altschul&amp;author=W%20Gish&amp;author=W%20Miller&amp;author=EW%20Myers&amp;author=DJ%20Lipman&amp;volume=215&amp;publication_year=1990&amp;pages=403-410&amp;pmid=2231712&amp;doi=10.1016/S0022-2836(05)80360-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref084">
<span class="label">84.</span><cite>Barker A, Venables B, Stevens SM Jr, Seeley KW, Wang P, Wolverton S. An Optimized Approach for Protein Residue Extraction and Identification from Ceramics After Cooking. J Archaeol Method Theory. 2011;19(3):407–39. doi: 10.1007/s10816-011-9120-5</cite> [<a href="https://doi.org/10.1007/s10816-011-9120-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Archaeol%20Method%20Theory&amp;title=An%20Optimized%20Approach%20for%20Protein%20Residue%20Extraction%20and%20Identification%20from%20Ceramics%20After%20Cooking&amp;author=A%20Barker&amp;author=B%20Venables&amp;author=SM%20Stevens&amp;author=KW%20Seeley&amp;author=P%20Wang&amp;volume=19&amp;issue=3&amp;publication_year=2011&amp;pages=407-39&amp;doi=10.1007/s10816-011-9120-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref085">
<span class="label">85.</span><cite>Pal Chowdhury M, Makarewicz C, Piezonka H, Buckley M. Novel Deep Eutectic Solvent-Based Protein Extraction Method for Pottery Residues and Archeological Implications. J Proteome Res. 2022;21(11):2619–34. doi: 10.1021/acs.jproteome.2c00340

</cite> [<a href="https://doi.org/10.1021/acs.jproteome.2c00340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9639204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36268809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Proteome%20Res&amp;title=Novel%20Deep%20Eutectic%20Solvent-Based%20Protein%20Extraction%20Method%20for%20Pottery%20Residues%20and%20Archeological%20Implications&amp;author=M%20Pal%20Chowdhury&amp;author=C%20Makarewicz&amp;author=H%20Piezonka&amp;author=M%20Buckley&amp;volume=21&amp;issue=11&amp;publication_year=2022&amp;pages=2619-34&amp;pmid=36268809&amp;doi=10.1021/acs.jproteome.2c00340&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref086">
<span class="label">86.</span><cite>Hendy J, Warinner C, Bouwman A, Collins MJ, Fiddyment S, Fischer R, et al. Proteomic evidence of dietary sources in ancient dental calculus. Proc Biol Sci. 2018;285(1883):20180977. doi: 10.1098/rspb.2018.0977

</cite> [<a href="https://doi.org/10.1098/rspb.2018.0977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6083251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30051838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Biol%20Sci&amp;title=Proteomic%20evidence%20of%20dietary%20sources%20in%20ancient%20dental%20calculus&amp;author=J%20Hendy&amp;author=C%20Warinner&amp;author=A%20Bouwman&amp;author=MJ%20Collins&amp;author=S%20Fiddyment&amp;volume=285&amp;issue=1883&amp;publication_year=2018&amp;pages=20180977&amp;pmid=30051838&amp;doi=10.1098/rspb.2018.0977&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref087">
<span class="label">87.</span><cite>Magens OM, Liu Y, Hofmans JFA, Nelissen JA, Ian Wilson D. Adhesion and cleaning of foods with complex structure: Effect of oil content and fluoropolymer coating characteristics on the detachment of cake from baking surfaces. Journal of Food Engineering. 2017;197:48–59. doi: 10.1016/j.jfoodeng.2016.11.004</cite> [<a href="https://doi.org/10.1016/j.jfoodeng.2016.11.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Food%20Engineering&amp;title=Adhesion%20and%20cleaning%20of%20foods%20with%20complex%20structure:%20Effect%20of%20oil%20content%20and%20fluoropolymer%20coating%20characteristics%20on%20the%20detachment%20of%20cake%20from%20baking%20surfaces&amp;author=OM%20Magens&amp;author=Y%20Liu&amp;author=JFA%20Hofmans&amp;author=JA%20Nelissen&amp;author=D%20Ian%20Wilson&amp;volume=197&amp;publication_year=2017&amp;pages=48-59&amp;doi=10.1016/j.jfoodeng.2016.11.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref088">
<span class="label">88.</span><cite>Ashokkumar S, Thomsen BR, Hinke J, Møller P, Adler-Nissen J. Cleanability evaluation of different surfaces by fouling from contact frying of foods. 2010. Available from: <a href="https://www.ceb.cam.ac.uk/system/files/documents/45-ashokkumar-fcfp2010.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ceb.cam.ac.uk/system/files/documents/45-ashokkumar-fcfp2010.pdf</a></cite>
</li>
<li id="pone.0330195.ref089">
<span class="label">89.</span><cite>Ma GJ, Yoon BK, Sut TN, Yoo KY, Lee SH, Jeon W, et al. Lipid coating technology: A potential solution to address the problem of sticky containers and vanishing drugs. VIEW. 2021;3(3). doi: 10.1002/viw.20200078</cite> [<a href="https://doi.org/10.1002/viw.20200078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=VIEW&amp;title=Lipid%20coating%20technology:%20A%20potential%20solution%20to%20address%20the%20problem%20of%20sticky%20containers%20and%20vanishing%20drugs&amp;author=GJ%20Ma&amp;author=BK%20Yoon&amp;author=TN%20Sut&amp;author=KY%20Yoo&amp;author=SH%20Lee&amp;volume=3&amp;issue=3&amp;publication_year=2021&amp;doi=10.1002/viw.20200078&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref090">
<span class="label">90.</span><cite>Goode KR, Asteriadou K, Robbins PT, Fryer PJ. Fouling and Cleaning Studies in the Food and Beverage Industry Classified by Cleaning Type. Comp Rev Food Sci Food Safe. 2013;12(2):121–43. doi: 10.1111/1541-4337.12000</cite> [<a href="https://doi.org/10.1111/1541-4337.12000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comp%20Rev%20Food%20Sci%20Food%20Safe&amp;title=Fouling%20and%20Cleaning%20Studies%20in%20the%20Food%20and%20Beverage%20Industry%20Classified%20by%20Cleaning%20Type&amp;author=KR%20Goode&amp;author=K%20Asteriadou&amp;author=PT%20Robbins&amp;author=PJ%20Fryer&amp;volume=12&amp;issue=2&amp;publication_year=2013&amp;pages=121-43&amp;doi=10.1111/1541-4337.12000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref091">
<span class="label">91.</span><cite>Ashokkumar S, Adler-Nissen J. Evaluating non-stick properties of different surface materials for contact frying. Journal of Food Engineering. 2011;105(3):537–44. doi: 10.1016/j.jfoodeng.2011.03.018</cite> [<a href="https://doi.org/10.1016/j.jfoodeng.2011.03.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Food%20Engineering&amp;title=Evaluating%20non-stick%20properties%20of%20different%20surface%20materials%20for%20contact%20frying&amp;author=S%20Ashokkumar&amp;author=J%20Adler-Nissen&amp;volume=105&amp;issue=3&amp;publication_year=2011&amp;pages=537-44&amp;doi=10.1016/j.jfoodeng.2011.03.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330195.ref092">
<span class="label">92.</span><cite>Saget M, de Almeida CF, Fierro V, Celzard A, Delaplace G, Thomy V, et al. A critical review on surface modifications mitigating dairy fouling. Compr Rev Food Sci Food Saf. 2021;20(5):4324–66. doi: 10.1111/1541-4337.12794

</cite> [<a href="https://doi.org/10.1111/1541-4337.12794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34250733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Compr%20Rev%20Food%20Sci%20Food%20Saf&amp;title=A%20critical%20review%20on%20surface%20modifications%20mitigating%20dairy%20fouling&amp;author=M%20Saget&amp;author=CF%20de%20Almeida&amp;author=V%20Fierro&amp;author=A%20Celzard&amp;author=G%20Delaplace&amp;volume=20&amp;issue=5&amp;publication_year=2021&amp;pages=4324-66&amp;pmid=34250733&amp;doi=10.1111/1541-4337.12794&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section></section><article class="sub-article" id="pone.0330195.r001"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330195.r001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330195.r001</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 0</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garvey%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Raven Garvey</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Raven Garvey</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garvey%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Raven Garvey</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Raven Garvey</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.c" class="d-panel p" style="display: none">
<div>© 2025 Raven Garvey</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>15 May 2025</em>
</p>
<p>
PONE-D-25-11386
An unclean slate, discrepancies between food input and recovered protein signal from experimental foodcrusts
PLOS ONE</p>
<p>Dear Dr. Dekker,</p>
<p>Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE’s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>==============================
As you address the reviewers' comments, please pay particular attention to R1's request for more detail regarding direct / indirect heating, burial conditions at York Experimental Archaeological Research Centre, and potential contaminants; and R2's comments regarding accessibility. 
==============================</p>
<p>Please submit your revised manuscript by Jun 29 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <span>plosone@plos.org</span>. When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pone/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<p>Please include the following items when submitting your revised manuscript:
</p>
<ul class="list" style="list-style-type:disc">
<li><p>A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.</p></li>
<li><p>A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p></li>
<li><p>An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p></li>
</ul>
<p>
If you would like to make changes to your financial disclosure, please include your updated statement in your cover letter. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p>
<p>If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: <a href="https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a>. Additionally, PLOS ONE offers an option for publishing peer-reviewed Lab Protocol articles, which describe protocols hosted on protocols.io. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a>.</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Raven Garvey, Ph.D.</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Journal requirements: </p>
<p>When submitting your revision, we need you to address these additional requirements.</p>
<p>1. Please ensure that your manuscript meets PLOS ONE's style requirements, including those for file naming. The PLOS ONE style templates can be found at</p>
<p><a href="https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf</a> and</p>
<p>
<a href="https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf</a>.</p>
<p>2.  We note that the grant information you provided in the ‘Funding Information’ and ‘Financial Disclosure’ sections do not match.</p>
<p>When you resubmit, please ensure that you provide the correct grant numbers for the awards you received for your study in the ‘Funding Information’ section.</p>
<p>3. Thank you for stating the following financial disclosure:</p>
<p> [The research was funded by the Marie Skłodowska-Curie grant agreement No 956351  from the European Union’s Horizon 2020 research and innovation programme.]. </p>
<p>Please state what role the funders took in the study.  If the funders had no role, please state: ""The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.""</p>
<p>If this statement is not correct you must amend it as needed.</p>
<p>Please include this amended Role of Funder statement in your cover letter; we will change the online submission form on your behalf.</p>
<p>5. Please review your reference list to ensure that it is complete and correct. If you have cited papers that have been retracted, please include the rationale for doing so in the manuscript text, or remove these references and replace them with relevant current references. Any changes to the reference list should be mentioned in the rebuttal letter that accompanies your revised manuscript. If you need to cite a retracted article, indicate the article’s retracted status in the References list and also include a citation and full reference for the retraction notice.</p>
<p>[Note: HTML markup is below. Please do not edit.]</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>

<strong>Comments to the Author</strong>
</p>
<p>1. Is the manuscript technically sound, and do the data support the conclusions?</p>
<p>The manuscript must describe a technically sound piece of scientific research with data that supports the conclusions. Experiments must have been conducted rigorously, with appropriate controls, replication, and sample sizes. The conclusions must be drawn appropriately based on the data presented. 
</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>
2. Has the statistical analysis been performed appropriately and rigorously? 
</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>
3. Have the authors made all data underlying the findings in their manuscript fully available?</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a> requires authors to make all data underlying the findings described in their manuscript fully available without restriction, with rare exception (please refer to the Data Availability Statement in the manuscript PDF file). The data should be provided as part of the manuscript or its supporting information, or deposited to a public repository. For example, in addition to summary statistics, the data points behind means, medians and variance measures should be available. If there are restrictions on publicly sharing data—e.g. participant privacy or use of data from a third party—those must be specified.
</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>
4. Is the manuscript presented in an intelligible fashion and written in standard English?</p>
<p>PLOS ONE does not copyedit accepted manuscripts, so the language in submitted articles must be clear, correct, and unambiguous. Any typographical or grammatical errors should be corrected at revision, so please note any specific errors here.
</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>
5. Review Comments to the Author</p>
<p>Please use the space provided to explain your answers to the questions above. You may also include additional comments for the author, including concerns about dual publication, research ethics, or publication ethics. (Please upload your review as an attachment if it exceeds 20,000 characters)
</p>
<p>Reviewer #1: Proteomic analysis of food residues in archaeological ceramics is an emerging field which has the potential to provide us with valuable information regarding utilisation of resources and reconstruction of dietary pathways and food chains. However, progress in the field is hampered by the relative paucity of experimental archaeology studies in this field. As such, the present manuscript is a valuable contribution in the investigation of the relationship between the input resources and the subsequent formation of foodcrusts and the extraction of proteins from them and their interpretation to trace back the resources used.</p>
<p>The manuscript is scientifically sound, and the methodological techniques used are relevant and well-described. The conclusions drawn are broadly supported by the experiments described in the manuscript, and the results are discussed in an archaeological and interpretive context.</p>
<p>However, I have a few minor questions and concerns which I believe need to be addressed before the manuscript can be published.</p>
<p>Materials and Methods –</p>
<p>Can the authors discuss the possible reasons behind the formation of the foodcrust in the rim rather than any other region? I presume this is due to the possible evaporation in the region near the rim, but I will be interested in what the authors think about it.</p>
<p>I assume the cooking simulation experiment was carried using indirect heating, i.e., the ceramics were not placed directly in a heating element. If that ist he case, can the authors please mention and discuss why that approach was chosen rather than direct heating? To me, it appears more likely that archaeological pots would have been heated directly in an open flame, and as such a direct heat would have been a better approximation of the treatment undergone by foodstuff, and hence the formation of foodcrust more representative what likely happened in an archaeological context.</p>
<p>Can the authors please provide descriptions of the burial conditions (namely substrate/soil, approximate temperature, moisture content vs dryness of the substrate etc.) at York Experimental Archaeological Research Centre?</p>
<p>Protein data analysis</p>
<p>Can the authors please search the mass spectrometry data against a database of common contaminants like cRap? This will be helpful in identifying the prevalence of contaminant proteins and any possible misidentitication regarding them, which can be a potential explanantion for the protein A0A3Q1LL35, as the authors mention in line 463.</p>
<p>Discussion</p>
<p>Disentangling mixtures &amp; taxonomic representativeness</p>
<p>Can the authors please discuss the proteins recovered after various cooking events in some greater detail by comparing and contrasting between the proteins recovered after various extraction, with specific focus on proteins recovered after the first cooking event? It appears to me that the first cooking event is perhaps most likely to estimate the conditions of archaeological ceramics- to me, it appears unlikely that the same pot was used for multiple cooking events without scrubbing off any visible charred residue. If that is the case, the residues after the first cooking event are probably most relevant for interpretation of archaeological data, and as such need to discussed in greater detail.</p>
<p>Can the authors please add a small paragraph comparing the nature/type of the various proteins recovered from the foodcrust with the type of proteins recovered in (i) other studies involving proteomics on archaeological samples where evidence of similar foodstuff processing has been recovered (ii) other experimental studies involving recovery of proteins from ceramic matrix and involving similar foodstuff?</p>
<p>The authors go into this in some detail re Evans et al. (line 501), but I believe a similar comparison involving additional studies (and comments on the differences in the types of proteins extracted) will be beneficial. This is particularly relevant since some prior studies have shown the recovery of muscle/blood proteins as evidence of animal processing, contrasting with the disappearance of the Bos taurus proteins here after burial. Is it possible that the nature of the substrate (foodcrust vs calcified deposits vs ceramic matrix) can affect the protein preservation and hence the interpretation of resource utilisation?</p>
<p>Reviewer #2: In this manuscript, Dekker et al. describe an experimental archaeological study to create foodcrusts in clay vessels and characterize their protein content over cooking and a variety of input conditions. This project is very well planned and executed by the authors. I'm impressed with the organization of this relatively complex set of experiments.</p>
<p>Line 88-89: "one lipid-rich, one protein-rich and one carbohydrate-rich to mix" Add material or food after carbohydrate-rich.</p>
<p>Line 182: How were the foodcrusts homogenized?</p>
<p>Line 234-237: For G-PTM-D, at minimum include the number of PTMs included and the PTM groups. If only specific ones were included, add them here or in the supplemental.</p>
<p>Line 240-242: Include the download dates of the Uniprot databases.</p>
<p>Figure 3: Unless the protein input is a continuous measurement, use factor for the scale instead of number.</p>
<p>Table 1: Include a column of the inputs for each cooking event. It may abe difficult to cross reference figure 1 with table 1 in the published version.</p>
<p>Figure 4: The lines between the boxes may not be clear if the figure is small at the final publication.</p>
<p>Line 333-338: Use the protein name instead of the Uniprot accession or put the accession in parentheses. It is hard to tell what protein it is from the accession alone.</p>
<p>Line 534 and 535: Should Fig. 6 be Fig. 7? This seems to be about the foodcrust size and not tissue specificity.</p>
<p>Line 538-539: "Suggesting that the presence of such a fat-rich food is not necessarily a driver of foodcrust formation" I think it may be worth connecting this result to the use of fats in reducing sticking in modern cooking.</p>
<p>Line 541: Confirm that this should be Fig. 5 and not another figure. It could be figure 2 or 7.</p>
<p>SI 5: Use a colorblind accessible gradient instead of the Red-Green one.</p>
<p>**********</p>
<p>
6. PLOS authors have the option to publish the peer review history of their article (<a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a>). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p>
<strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a>.
</p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p>
<p>**********</p>
<p>[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.]</p>
<p>While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a>. PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at <span>figures@plos.org</span>. Please note that Supporting Information files do not need this step.
</p></section></article><article class="sub-article" id="pone.0330195.r002"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. 2025 Aug 22;20(8):e0330195. doi: <a href="https://doi.org/10.1371/journal.pone.0330195.r002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330195.r002</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 1</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.d" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.d" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub2" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>15 Jul 2025</em>
</p>
<p>We would like to thank both reviewers for their constructive reviews.</p>
<p>In the document 'Response to reviewers' you will find a point-by-point response to each of the comments and we hope to have improved the manuscript to your liking.</p>
<section class="sm xbox font-sm" id="pone.0330195.s008"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to reviewers.docx</em>
</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s008.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s008.docx</a><sup> (12.7KB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0330195.r003"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330195.r003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330195.r003</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 1</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garvey%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Raven Garvey</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Raven Garvey</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garvey%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Raven Garvey</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Raven Garvey</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none">
<div>© 2025 Raven Garvey</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>29 Jul 2025</em>
</p>
<p>An unclean slate, discrepancies between food input and recovered protein signal from experimental foodcrusts</p>
<p>PONE-D-25-11386R1</p>
<p>Dear Dr. Dekker,</p>
<p>We’re pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it meets all outstanding technical requirements.</p>
<p>Within one week, you’ll receive an e-mail detailing the required amendments. When these have been addressed, you’ll receive a formal acceptance letter and your manuscript will be scheduled for publication.</p>
<p>An invoice will be generated when your article is formally accepted. Please note, if your institution has a publishing partnership with PLOS and your article meets the relevant criteria, all or part of your publication costs will be covered. Please make sure your user information is up-to-date by logging into Editorial Manager at <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Editorial Manager®</a> and clicking the ‘Update My Information' link at the top of the page. For questions related to billing, please contact <a href="https://plos.my.site.com/s/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">billing support</a>.</p>
<p>If your institution or institutions have a press office, please notify them about your upcoming paper to help maximize its impact. If they’ll be preparing press materials, please inform our press team as soon as possible -- no later than 48 hours after receiving the formal acceptance. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>Kind regards,</p>
<p>Raven Garvey, Ph.D.</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Additional Editor Comments (optional):</p>
<p>Reviewers' comments:</p></section></article><article class="sub-article" id="pone.0330195.r004"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330195.r004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330195.r004</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Acceptance letter</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garvey%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Raven Garvey</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Raven Garvey</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Garvey%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Raven Garvey</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.f" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.f" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Raven Garvey</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.f" class="d-panel p" style="display: none">
<div>© 2025 Raven Garvey</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>PONE-D-25-11386R1</p>
<p>PLOS ONE</p>
<p>Dear Dr. Dekker,</p>
<p>I'm pleased to inform you that your manuscript has been deemed suitable for publication in PLOS ONE. Congratulations! Your manuscript is now being handed over to our production team.</p>
<p>At this stage, our production department will prepare your paper for publication. This includes ensuring the following:</p>
<p>* All references, tables, and figures are properly cited</p>
<p>* All relevant supporting information is included in the manuscript submission,</p>
<p>* There are no issues that prevent the paper from being properly typeset</p>
<p>You will receive further instructions from the production team, including instructions on how to review your proof when it is ready. Please keep in mind that we are working through a large volume of accepted articles, so please give us a few days to review your paper and let you know the next and final steps.</p>
<p>Lastly, if your institution or institutions have a press office, please let them know about your upcoming paper now to help maximize its impact. If they'll be preparing press materials, please inform our press team within the next 48 hours. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>You will receive an invoice from PLOS for your publication fee after your manuscript has reached the completed accept phase. If you receive an email requesting payment before acceptance or for any other service, this may be a phishing scheme. Learn how to identify phishing emails and protect your accounts at <a href="https://explore.plos.org/phishing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://explore.plos.org/phishing</a>.</p>
<p>If we can help with anything else, please email us at customercare@plos.org.</p>
<p>Thank you for submitting your work to PLOS ONE and supporting open access.</p>
<p>Kind regards,</p>
<p>PLOS ONE Editorial Office Staff</p>
<p>on behalf of</p>
<p>Dr Raven Garvey</p>
<p>Academic Editor</p>
<p>PLOS ONE</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Data. Nutritional composition of the used foods.</span><p>(CSV)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s001.csv" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s001.csv</a><sup> (1.3KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S2 Data. Overview of animal and plant species identified at Danish Ertebølle sites and the availability of public protein reference data for these species.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s002.xlsx</a><sup> (124.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S3 Data. Metadata of the MetaMorpheus database search.</span><p>(ZIP)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s003.zip" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s003.zip</a><sup> (53.7MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S4 Data. List of identified proteins.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s004.xlsx</a><sup> (1.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S5 Data. Overview of the number of the number of foodcrusts per cooking event a protein was identified in.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s005.xlsx</a><sup> (2.7MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>S6 Data. Overview of the identified PTMs, both per protein and summarised per cooking event.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s006.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s006.xlsx</a><sup> (508.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to reviewers.docx</em>
</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12373221/bin/pone.0330195.s008.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330195.s008.docx</a><sup> (12.7KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>A full submission of the acquired mass spectrometry data has been made to the ProteomeXchange Consortium via the PRIDE 64 partner repository with the dataset identifier PXD059930 and <a href="https://doi.org/10.6019/PXD059930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6019/PXD059930</a>. Additional metadata of the MetaMorpheus database search and the 3D model object files of the experimental ceramic vessels used in the cooking experiment can be found on Zenodo under the following DOI: <a href="https://doi.org/10.5281/zenodo.15874520" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15874520</a>.</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0330195"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0330195.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373221/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373221/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373221%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373221/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373221/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373221/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40845013/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373221/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40845013/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373221/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373221/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="IFAmMQPQ7ACp17CvrCnXw2BcGPdZ1qkyGnOMDyNv5TLkXRTujseZ5HIc8PvSgn5N">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
