
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Marker‐assisted introgression and characterization of the ph1b mutant in modern US and Australian wheats - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E481798AEEC7F3058179001D71A1E8.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12264458/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Marker‐assisted introgression and characterization of the ph1b mutant in modern US and Australian wheats">
<meta name="citation_author" content="Yadav Gyawali">
<meta name="citation_author_institution" content="Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA">
<meta name="citation_author" content="Wei Zhang">
<meta name="citation_author_institution" content="Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA">
<meta name="citation_author_institution" content="Shanxi Key Laboratory of Minor Crop Germplasm Innovation and Molecular Breeding, Shanxi Agricultural University, Taiyuan, China">
<meta name="citation_author" content="Tatiana V Danilova">
<meta name="citation_author_institution" content="USDA‐ARS Wheat, Sorghum &amp; Forage Research Unit, Lincoln, Nebraska, USA">
<meta name="citation_author" content="Mingyi Zhang">
<meta name="citation_author_institution" content="Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA">
<meta name="citation_author" content="Steven Xu">
<meta name="citation_author_institution" content="USDA‐ARS Crop Improvement and Genetics Research Unit, Western Regional Research Center, Albany, California, USA">
<meta name="citation_author" content="David Van Sanford">
<meta name="citation_author_institution" content="Plant and Soil Sciences Department, University of Kentucky, Lexington, Kentucky, USA">
<meta name="citation_author" content="Arron Carter">
<meta name="citation_author_institution" content="Department of Crop and Soil Sciences, Washington State University, Pullman, Washington, USA">
<meta name="citation_author" content="Elias Elias">
<meta name="citation_author_institution" content="Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA">
<meta name="citation_author" content="Xiwen Cai">
<meta name="citation_author_institution" content="USDA‐ARS Wheat, Sorghum &amp; Forage Research Unit, Lincoln, Nebraska, USA">
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska, USA">
<meta name="citation_publication_date" content="2025 Jul 15">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70065">
<meta name="citation_doi" content="10.1002/tpg2.70065">
<meta name="citation_pmid" content="40665645">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12264458/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12264458/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12264458/pdf/TPG2-18-e70065.pdf">
<meta name="description" content="The ph1b mutant, a large deletion (∼60 Mb) involving Ph1 (pairing homoeologous 1) on chromosome 5B, was incorporated into five major classes of US wheats, including soft white winter wheat, soft red winter wheat, hard red spring wheat, hard white ...">
<meta name="og:title" content="Marker‐assisted introgression and characterization of the ph1b mutant in modern US and Australian wheats">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The ph1b mutant, a large deletion (∼60 Mb) involving Ph1 (pairing homoeologous 1) on chromosome 5B, was incorporated into five major classes of US wheats, including soft white winter wheat, soft red winter wheat, hard red spring wheat, hard white ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12264458/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12264458">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70065"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70065.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12264458%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12264458/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12264458/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12264458/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70065" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 15;18(3):e70065. doi: <a href="https://doi.org/10.1002/tpg2.70065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70065</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Marker‐assisted introgression and characterization of the <em>ph1b</em> mutant in modern US and Australian wheats</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gyawali%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yadav Gyawali</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yadav Gyawali</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gyawali%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yadav Gyawali</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Wei Zhang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wei Zhang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Shanxi Key Laboratory of Minor Crop Germplasm Innovation and Molecular Breeding, Shanxi Agricultural University, Taiyuan, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wei Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Danilova%20TV%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Tatiana V Danilova</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Tatiana V Danilova</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>USDA‐ARS Wheat, Sorghum &amp; Forage Research Unit, Lincoln, Nebraska, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Danilova%20TV%22%5BAuthor%5D" class="usa-link"><span class="name western">Tatiana V Danilova</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mingyi Zhang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mingyi Zhang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingyi Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Steven Xu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Steven Xu</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>USDA‐ARS Crop Improvement and Genetics Research Unit, Western Regional Research Center, Albany, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Steven Xu</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Sanford%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">David Van Sanford</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">David Van Sanford</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Plant and Soil Sciences Department, University of Kentucky, Lexington, Kentucky, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Sanford%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Van Sanford</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carter%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Arron Carter</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Arron Carter</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Department of Crop and Soil Sciences, Washington State University, Pullman, Washington, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carter%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Arron Carter</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elias%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Elias Elias</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Elias Elias</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elias%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elias Elias</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Xiwen Cai</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Xiwen Cai</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>USDA‐ARS Wheat, Sorghum &amp; Forage Research Unit, Lincoln, Nebraska, USA</div>
<div class="p">
<sup>
<sup>7</sup>
</sup>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiwen Cai</span></a>
</div>
</div>
<sup>3,</sup><sup>7,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270065-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Plant Sciences, North Dakota State University, Fargo, North Dakota, USA</div>
<div id="tpg270065-aff-0002">
<sup>
<sup>2</sup>
</sup>Shanxi Key Laboratory of Minor Crop Germplasm Innovation and Molecular Breeding, Shanxi Agricultural University, Taiyuan, China</div>
<div id="tpg270065-aff-0003">
<sup>
<sup>3</sup>
</sup>USDA‐ARS Wheat, Sorghum &amp; Forage Research Unit, Lincoln, Nebraska, USA</div>
<div id="tpg270065-aff-0004">
<sup>
<sup>4</sup>
</sup>USDA‐ARS Crop Improvement and Genetics Research Unit, Western Regional Research Center, Albany, California, USA</div>
<div id="tpg270065-aff-0005">
<sup>
<sup>5</sup>
</sup>Plant and Soil Sciences Department, University of Kentucky, Lexington, Kentucky, USA</div>
<div id="tpg270065-aff-0006">
<sup>
<sup>6</sup>
</sup>Department of Crop and Soil Sciences, Washington State University, Pullman, Washington, USA</div>
<div id="tpg270065-aff-0007">
<sup>
<sup>7</sup>
</sup>Department of Agronomy and Horticulture, University of Nebraska, Lincoln, Nebraska, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Xiwen Cai, USDA‐ARS Wheat, Sorghum &amp; Forage Research Unit, Lincoln, NE 68583, USA. Email: <span>xiwen.cai@usda.gov</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 May 21; Received 2025 Apr 15; Accepted 2025 May 22; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America. This article has been contributed to by U.S. Government employees and their work is in the public domain in the USA.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12264458  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40665645/" class="usa-link">40665645</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The <em>ph1b</em> mutant, a large deletion (∼60 Mb) involving <em>Ph1</em> (pairing homoeologous 1) on chromosome 5B, was incorporated into five major classes of US wheats, including soft white winter wheat, soft red winter wheat, hard red spring wheat, hard white spring wheat, and durum wheat, and an Australian wheat variety with universal susceptibility to rusts. Eight <em>ph1b</em> mutant lines in the five classes of US wheats and the Australia wheat were developed using a molecular marker‐assisted backcross breeding pipeline. All backgrounds containing the <em>ph1b</em> mutation induced meiotic homoeologous pairing and recombination as the original <em>ph1b</em> mutant developed in the Chinese landrace “Chinese Spring” (CS). Thus, they can be utilized for homoeologous recombination‐based alien introgression in the adapted US wheat backgrounds, which will circumvent the use of CS <em>ph1b</em> mutant with undesired agronomic characteristics. Deployment of the <em>ph1b</em> mutant in the universal rust susceptible wheat “Line E” will facilitate rust resistance gene identification and transfer from wild species to wheat. In addition, we developed special DNA markers with a codominance‐like nature specifically for the <em>ph1b</em> deletion and the genomic region spanning the wild‐type allele <em>Ph1</em>, which will improve the utility of the <em>ph1b</em> mutant in alien introgression. This work enables direct gene flow from related grass species into adapted wheats to enrich and diversify their genomes and enhances the genetic potential of wheat in production.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270065-list-10001" class="list" style="list-style-type:disc">
<li><p>The <em>ph1b</em> mutant was incorporated into US wheats and an Australian wheat with universal susceptibility to rusts.</p></li>
<li><p>The <em>ph1b</em> mutant induced meiotic homoeologous pairing and recombination in all wheat genotypes examined.</p></li>
<li><p>The US wheat <em>ph1b</em> mutants will enable gene flow directly from related grasses into US wheats.</p></li>
<li><p>The DNA markers specific for <em>ph1b</em> and <em>Ph1</em> will improve the utility of the <em>ph1b</em> mutant in alien introgression.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270065-dl-0001">
<dt>BLAST</dt>
<dd><p>Basic Local Alignment Search Tool</p></dd>
<dt>CS</dt>
<dd><p>Chinese Spring</p></dd>
<dt>CSphm</dt>
<dd><p>CS <em>ph1b</em> mutant</p></dd>
<dt>DAPI</dt>
<dd><p>4′,6‐diamidino‐2‐phenylindole</p></dd>
<dt>DB</dt>
<dd><p>distal breakpoint</p></dd>
<dt>IWGSC</dt>
<dd><p>International Wheat Genome Sequencing Consortium</p></dd>
<dt>LTR</dt>
<dd><p>long terminal repeat</p></dd>
<dt>MI</dt>
<dd><p>metaphase I</p></dd>
<dt>PACE</dt>
<dd><p>PCR allelic competitive extension</p></dd>
<dt>PB</dt>
<dd><p>proximal breakpoint</p></dd>
<dt>PCR</dt>
<dd><p>polymerase chain reaction</p></dd>
<dt>
<em>Ph1</em>
</dt>
<dd><p>pairing homoeologous 1</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>STS</dt>
<dd><p>sequence‐tagged site</p></dd>
</dl>
<section id="tpg270065-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Cultivated wheat, including durum wheat (<em>Triticum turgidum</em> L. subsp. <em>Durum</em> (Desf.) Husn, 2<em>n</em> = 4<em>x</em> = 28, AABB) and common wheat (<em>Triticum aestivum</em> L., 2<em>n</em> = 6<em>x</em> = 42, AABBDD), are both allopolyploids originated from interspecific crosses of three diploid ancestors and one tetraploid ancestor and subsequent polyploidization (Dvorák et al., <a href="#tpg270065-bib-0010" class="usa-link" aria-describedby="tpg270065-bib-0010">1993</a>; Huang et al., <a href="#tpg270065-bib-0018" class="usa-link" aria-describedby="tpg270065-bib-0018">2002</a>; Kihara, <a href="#tpg270065-bib-0020" class="usa-link" aria-describedby="tpg270065-bib-0020">1944</a>; Levy &amp; Feldman, <a href="#tpg270065-bib-0022" class="usa-link" aria-describedby="tpg270065-bib-0022">2022</a>; Riley et al., <a href="#tpg270065-bib-0033" class="usa-link" aria-describedby="tpg270065-bib-0033">1958</a>). Their sub‐genomes A, B, and D are homoeologous to each other (Kihara, <a href="#tpg270065-bib-0019" class="usa-link" aria-describedby="tpg270065-bib-0019">1919</a>; Sears, <a href="#tpg270065-bib-0035" class="usa-link" aria-describedby="tpg270065-bib-0035">1952</a>).</p>
<p>Both allopolyploids exhibit a diploid‐like chromosome behavior in meiosis because they have a diploidization system in their genomes, which is regulated primarily by the <em>Ph1</em> (pairing homoeologous) gene on chromosome 5B (Sears, <a href="#tpg270065-bib-0036" class="usa-link" aria-describedby="tpg270065-bib-0036">1976</a>, <a href="#tpg270065-bib-0037" class="usa-link" aria-describedby="tpg270065-bib-0037">1977</a>). The diploidization system ensures meiotic synapsis and recombination to occur between homologous chromosomes, not between homoeologous chromosomes in both durum and common wheat. It makes both allopolyploids behave like a diploid in meiosis. Deactivation of the diploidization system by deleting <em>Ph1</em> or suppressing <em>Ph1</em> activity promotes homoeologous chromosomes to synapse and recombine with each other during meiosis (Chen et al., <a href="#tpg270065-bib-0008" class="usa-link" aria-describedby="tpg270065-bib-0008">1994</a>; Dvorak et al., <a href="#tpg270065-bib-0009" class="usa-link" aria-describedby="tpg270065-bib-0009">2006</a>; Giorgi, <a href="#tpg270065-bib-0016" class="usa-link" aria-describedby="tpg270065-bib-0016">1978</a>; Sears, <a href="#tpg270065-bib-0037" class="usa-link" aria-describedby="tpg270065-bib-0037">1977</a>). This has dramatically enhanced meiotic homoeologous recombination‐based alien introgression in wheat and its relatives (Qi et al., <a href="#tpg270065-bib-0028" class="usa-link" aria-describedby="tpg270065-bib-0028">2007</a>; Zhang et al., <a href="#tpg270065-bib-0043" class="usa-link" aria-describedby="tpg270065-bib-0043">2017</a>).</p>
<p>A large deletion mutant of <em>Ph1</em>, designated <em>ph1b</em>, was developed by X‐ray irradiation in Chinese Spring (CS) wheat (K. S. Gill et al., <a href="#tpg270065-bib-0015" class="usa-link" aria-describedby="tpg270065-bib-0015">1993</a>; Gyawali et al., <a href="#tpg270065-bib-0017" class="usa-link" aria-describedby="tpg270065-bib-0017">2019</a>; Sears, <a href="#tpg270065-bib-0037" class="usa-link" aria-describedby="tpg270065-bib-0037">1977</a>). CS is a Chinese soft red spring wheat landrace with many undesired agronomic characteristics. But CS has been considered a standard common wheat genotype and has been widely utilized in various genomic and genetic studies. A variety of genetic stocks and numerous genomic resources have been generated in CS wheat. However, the utility of CS wheat genetic stocks in wheat breeding, including <em>ph1b</em> mutant, has been limited due to its unfavorable agronomic performance. The CS wheat <em>ph1b</em> mutant (CSphm) was transferred to a European winter wheat line (Türkösi et al., <a href="#tpg270065-bib-0040" class="usa-link" aria-describedby="tpg270065-bib-0040">2022</a>) and a US hard red winter wheat line (Friebe et al., <a href="#tpg270065-bib-0012" class="usa-link" aria-describedby="tpg270065-bib-0012">2012</a>), but not other classes of US wheats.</p>
<p>A <em>Ph1</em>‐specific sequence‐tagged site (STS) marker was used for the detection of homozygous <em>ph1b</em> deletion but was unable to differentiate the hemizygote <em>Ph1ph1b</em> from the homozygote <em>Ph1Ph1</em> in the introgression process (Wang et al., <a href="#tpg270065-bib-0041" class="usa-link" aria-describedby="tpg270065-bib-0041">2002</a>). Recently, Gyawali et al. (<a href="#tpg270065-bib-0017" class="usa-link" aria-describedby="tpg270065-bib-0017">2019</a>) identified the deletion breakpoints of CSphm and developed STS markers specific for the wild‐type allele <em>Ph1</em> and <em>ph1b</em> deletion. Those STS markers can differentiate all three genotypes (<em>Ph1Ph1</em>, <em>Ph1ph1b</em>, and <em>ph1bph1b</em>) from each other, which has enhanced the utility of CSphm in alien introgression (Boehm &amp; Cai, <a href="#tpg270065-bib-0005" class="usa-link" aria-describedby="tpg270065-bib-0005">2024</a>). Meanwhile, those allele‐specific markers are useful in the introgression of <em>ph1b</em> mutant from CS into adapted wheat backgrounds, which will further improve the efficacy and throughput of the <em>ph1b</em> mutant in alien introgression and wheat breeding (Boehm &amp; Cai, <a href="#tpg270065-bib-0005" class="usa-link" aria-describedby="tpg270065-bib-0005">2024</a>; Gyawali et al., <a href="#tpg270065-bib-0017" class="usa-link" aria-describedby="tpg270065-bib-0017">2019</a>).</p>
<p>Here, we report marker‐assisted introgression of CSphm into major classes of US wheats and development of <em>ph1b</em> mutant in adapted US wheats. This will enable gene flow from wheat relatives directly into modern US wheats for germplasm and variety development in wheat breeding.</p></section><section id="tpg270065-sec-0020"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270065-sec-0030"><h3 class="pmc_sec_title">2.1. Plant materials and hybridization</h3>
<p>Common wheat CS, CSphm, and representative varieties of the five classes of US wheats, including common wheat “Otto” (soft white winter wheat; adapted to the low rainfall), “Jasper” (soft white winter wheat; adapted to intermediate to high rainfall), “SS8641” (soft red winter wheat), “Pembroke 2014” (soft red winter wheat), “Alsen” (hard red spring wheat), “Argent” (hard white spring wheat), durum wheat “Divide,” and “Line E” (an Australian wheat variety with universal susceptibility to rusts) were employed in this study. Common wheat CS was used as <em>Ph1Ph1</em> control, and CSphm (<em>ph1bph1b</em>) was used as <em>ph1b</em> donor and <em>ph1bph1b</em> control. Otto, Jasper, SS8641, Pembroke 2014, Alsen, Argent, Divide, and Line E were used as the recipients of <em>ph1b</em> introgression. CSphm was provided by the Wheat Genetics Resource Center at Kansas State University, USA.</p>
<p>All materials were grown in the greenhouse for <em>ph1b</em> introgression crosses and backcrosses. The US wheat varieties (Otto, Jasper, SS8641, Pembroke 2014, Alsen, Argent, and Divide) and Australian wheat Line E were used as female parent to cross with CSphm, and the resultant F<sub>1</sub>’s was backcrossed to the respective US wheat parents or Line E. Successive backcrosses were made following a molecular marker‐assisted backcrossing breeding scheme. A total of three backcrosses were made to recover the genetic backgrounds of the recurrent wheat varieties. The F<sub>1</sub> individuals verified as hemizygous (i.e., <em>Ph1ph1b</em>) by the <em>Ph1/ph1b</em>‐specific markers were crossed to the respective US wheat parents for BC<sub>1</sub>F<sub>1</sub> production. Ten to fifteen progenies in each backcross generation were genotyped to select the individuals hemizygous for <em>Ph1/ph1b</em> by the <em>Ph1/ph1b</em>‐specific markers. About four to eight hemizygous plants (<em>Ph1ph1b</em>) were advanced for subsequent backcrosses with each of the US wheat parents. To recover homozygous <em>ph1b</em> mutant, four to eight BC<sub>3</sub>F<sub>1</sub> individuals hemizygous for <em>Ph1/ph1b</em> were selected and self‐pollinated to produce BC<sub>3</sub>F<sub>2</sub> segregating populations with ∼25% homozygous <em>ph1b</em> mutant (<em>ph1bph1b</em>). A total of 30–40 BC<sub>3</sub>F<sub>2</sub> individuals were genotyped by the <em>Ph1/ph1b</em>‐specific markers to select the plants homozygous for <em>ph1b</em> for seed increase.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270065-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270065-list-0001" class="list" style="list-style-type:disc">
<li><p>The <em>ph1b</em> mutant was incorporated into US wheats and an Australian wheat with universal susceptibility to rusts.</p></li>
<li><p>The <em>ph1b</em> mutant induced meiotic homoeologous pairing and recombination in all wheat genotypes examined.</p></li>
<li><p>The US wheat <em>ph1b</em> mutants will enable gene flow directly from related grasses into US wheats.</p></li>
<li><p>The DNA markers specific for <em>ph1b</em> and <em>Ph1</em> will improve the utility of the <em>ph1b</em> mutant in alien introgression.</p></li>
</ul></section></section></section><section id="tpg270065-sec-0050"><h3 class="pmc_sec_title">2.2. Molecular marker analysis</h3>
<section id="tpg270065-sec-0060"><h4 class="pmc_sec_title">2.2.1. Sequence‐tagged sites markers</h4>
<p>Genomic DNA was extracted from young leaves either freshly collected or stored in a −80°C freezer as described by Niu et al. (<a href="#tpg270065-bib-0026" class="usa-link" aria-describedby="tpg270065-bib-0026">2011</a>). The <em>Ph1</em> allele‐specific STS marker (<em>Xwgc2049</em>) together with <em>ph1b</em>‐specific STS markers (<em>Xwgc2106</em>) were used to differentiate <em>Ph1Ph1</em>, <em>Ph1ph1b</em>, and <em>ph1bph1b</em> genotypes from each other in a dual‐plex PCR (Gyawali et al., <a href="#tpg270065-bib-0017" class="usa-link" aria-describedby="tpg270065-bib-0017">2019</a>). The PCR program includes initial denaturation at 94°C for 2 min followed by 30 cycles of 94°C for 30 s, 58°C annealing temperature for 30 s, 72°C for 1 min 30 s and final extension at 72°C for 5 min. PCR was conducted in a 20‐µL reaction mixture containing 100 ng template DNA, 1 µL of 10 µM each primer, and 10 µL of 2× GoTaq Green Master Mix (Promega). PCR products were separated on 1.5% agarose gel electrophoresis with either ethidium bromide or bio‐safe Coomassie stain G‐250 (Bio‐Rad) staining.</p></section><section id="tpg270065-sec-0070"><h4 class="pmc_sec_title">2.2.2. PCR allelic competitive extension markers</h4>
<p>Genomic DNA was isolated from leaves of 2‐week‐old seedlings using a Wizard Magnetic 96 DNA Plant System (Promega) following vendor's protocol modified for wheat. PCR was run in 8 µL of PCR allelic competitive extension (PACE) mixture containing 40–120 ng of DNA, 0.11 µL of 72× primer mix, and 4 µL of a 2× PACE Genotyping Master mix (3CR Bioscience Ltd). Four‐primer PACE assay involving <em>Ph1</em>‐specific (chromosome 5BL‐specific) and <em>ph1b</em> mutant‐specific primer pairs (Table <a href="#tpg270065-tbl-0001" class="usa-link">1</a>) were used to differentiate the genotypes of <em>Ph1Ph1</em>, <em>Ph1ph1b</em>, and <em>ph1bph1b</em> from each other.</p>
<section class="tw xbox font-sm" id="tpg270065-tbl-0001"><h5 class="obj_head">TABLE 1.</h5>
<div class="caption p"><p>Four‐primer PCR allelic competitive extension (PACE) assay discriminating <em>Ph1</em> allele and <em>ph1b</em> deletion on chromosome 5BL.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Marker</th>
<th align="left" rowspan="1" colspan="1">Primers</th>
<th align="left" rowspan="1" colspan="1">Sequences (5′‐3′)</th>
<th align="left" rowspan="1" colspan="1">Amplicon size (bp)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>Xwgc2321</em>
</td>
<td align="left" rowspan="1" colspan="1">Forward <em>Ph1</em>‐5BL‐114</td>
<td align="left" rowspan="1" colspan="1">FAM‐CTCAAGTCAGTGATAACAGTTGTA</td>
<td align="left" rowspan="1" colspan="1">116</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Reverse <em>Ph1</em>‐5BL‐114</td>
<td align="left" rowspan="1" colspan="1">CAACCAGATGTGATTATCAGT</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Forward <em>ph1b</em>‐P2N</td>
<td align="left" rowspan="1" colspan="1">HEX‐GGCCTAGATGCACATAAGTA</td>
<td align="left" rowspan="1" colspan="1">138</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Reverse <em>ph1b</em>‐P2N</td>
<td align="left" rowspan="1" colspan="1">GTATTCGAATCATACCTTCAAC</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270065-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Primers were synthesized by Integrated DNA Technologies, Inc. The 100 µL of 72× primer mix was prepared by mixing 100 µM primer stocks: 12 µL of each forward primer and 30 µL of each reverse primer in 0.1 mM TrisHCl (pH 7.6). PACE PCR was performed in a BioRad T100 thermal cycler under the following conditions: initial denaturation 15 min at 94°C, then 10 cycles of 94°C − 20 s, 62–56°C − 1 min with a temperature drop of 0.6°C per cycle, then 35 cycles of 94°C − 20 s, 56°C − 1 min and held at 10°C. The PCR results were read using a Bio‐Rad CFX96 machine and the Maestro 2.3 software (Bio‐Rad Laboratories, Inc.) after reading cycle: 37°C for 1 min. The STS markers <em>Xwgc2049</em> and <em>Xwgc2111</em> (Gyawali et al., <a href="#tpg270065-bib-0017" class="usa-link" aria-describedby="tpg270065-bib-0017">2019</a>) were used to validate PACE genotyping results.</p></section></section><section id="tpg270065-sec-0080"><h3 class="pmc_sec_title">2.3. Meiotic chromosome preparation and analysis</h3>
<p>Meiotic chromosomes were prepared from anthers at metaphase I (MI) of the <em>ph1b</em> introgression lines and their recipient parents as described by Cai et al. (<a href="#tpg270065-bib-0007" class="usa-link" aria-describedby="tpg270065-bib-0007">1996</a>). Chromosomes were stained with DAPI (4′,6‐diamidino‐2‐phenylindole) and visualized under a Zeiss AxioImager M2 microscopy system with ZEN 3.3 imaging software (Carl Zeiss Microscopy). Meiotic chromosome pairing configurations and frequencies were analyzed in 10–94 meiocytes from three to five plants of the recipient wheat cultivars and their <em>ph1b</em> mutants.</p></section></section><section id="tpg270065-sec-0090"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270065-sec-0100"><h3 class="pmc_sec_title">3.1. Marker‐assisted introgression and production of <em>ph1b</em> mutant in modern US wheats and Australian wheat</h3>
<p>The common wheat CS, Otto, Jasper, SS8641, Pembroke 2014, Alsen, Argent, and Line E, and durum wheat Divide harbor <em>Ph1</em> on chromosome 5B, while CS <em>ph1b</em> mutant has the genomic region of ∼60 Mb spanning <em>Ph1</em> locus deleted (Gyawali et al., <a href="#tpg270065-bib-0017" class="usa-link" aria-describedby="tpg270065-bib-0017">2019</a>). To transfer the <em>ph1b</em> deletion from CSphm to Otto, Jasper, SS8641, Pembroke 2014, Alsen, Argent, Divide and Line E, CSphm was used as male parent to pollinate each of the recipient parents. The F<sub>1</sub> hybrids were backcrossed to the respective recipient parent, and the resultant BC<sub>1</sub>F<sub>1</sub> individuals were genotyped by the <em>Ph1</em>/<em>ph1b</em>‐specific STS markers (<em>Xwgc2049</em> and <em>Xwgc2106</em>) to select the hemizygous segregants containing <em>ph1b</em> deletion (i.e. <em>Ph1ph1b</em>) for the second backcross. Two additional backcrosses were made to the hemizygous BC<sub>1</sub>F<sub>1</sub> individuals (<em>Ph1ph1b</em>) to further recover the genetic backgrounds of the recipient wheat parents and to retain the <em>ph1b</em> deletion through the marker‐assisted introgression pipeline (Figure <a href="#tpg270065-fig-0001" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="tpg270065-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/70c1/12264458/793891e86a05/TPG2-18-e70065-g002.jpg" loading="lazy" id="jats-graphic-1" height="558" width="560" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270065-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Diagram of marker‐assisted introgression of the <em>ph1b</em> mutant from Chinese Spring (CS) into US and Australian wheats.</p></figcaption></figure><p>The STS marker <em>Xwgc2049</em> amplified a segment of 846 bp within the genomic region spanning the <em>Ph1</em> locus on chromosome 5BL, which is absent in the <em>ph1b</em> mutant. Thus, <em>Xwgc2049</em> specifically tags the wild‐type <em>Ph1</em> allele. The STS marker <em>Xwgc2106</em> amplified a segment of 1149 bp across the deletion joint point in the <em>ph1b</em> mutant, which is not amplifiable for the wild‐type <em>Ph1</em> allele because two primer annealing sites within the <em>Ph1</em> genomic region (&gt;60 Mb apart) are far beyond the amplification range of a regular PCR (Gyawali et al., <a href="#tpg270065-bib-0017" class="usa-link" aria-describedby="tpg270065-bib-0017">2019</a>). <em>Xwgc2106</em>, therefore, is a <em>ph1b</em> mutant‐specific marker. Our dual‐plex PCR with <em>Xwgc2049</em> and <em>Xwgc2106</em> clearly differentiated <em>Ph1Ph1</em> from <em>Ph1ph1b</em> in the BC<sub>1</sub>F<sub>1</sub>, BC<sub>2</sub>F<sub>1</sub>, and BC<sub>3</sub>F<sub>1</sub> progenies (Figure <a href="#tpg270065-fig-0002" class="usa-link">2</a>) and <em>Ph1Ph1</em>, <em>Ph1ph1b</em>, and <em>Ph1ph1b</em> from each other in the BC<sub>3</sub>F<sub>2</sub> populations (Figure <a href="#tpg270065-fig-0003" class="usa-link">3</a>). In addition, we did not observe polymorphisms among the genotypes of the introgressions at the <em>Xwgc2049</em> and <em>Xwgc2106</em> loci (Figures <a href="#tpg270065-fig-0002" class="usa-link">2</a> and <a href="#tpg270065-fig-0003" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="tpg270065-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/70c1/12264458/a772d3a8bc88/TPG2-18-e70065-g003.jpg" loading="lazy" id="jats-graphic-3" height="563" width="546" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270065-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Agarose gel electrophoresis images of the <em>Ph1</em>‐specific STS marker <em>Xwgc2049</em> and <em>ph1b</em> deletion‐specific STS marker <em>Xwgc2106</em> amplified by a dual‐plex PCR in the BC<sub>3</sub>F<sub>1</sub>
<em>ph1b</em> introgression populations for the selection of <em>ph1b</em> deletion (hemizygotes) in the introgressions involving CS (A), Divide (B), Line E (C), Otto (D), Pembroke 2014 (E), Alsen (F), Argent (G), SS8641 (H), and Jasper (I). <em>M</em>—Size marker.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270065-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/70c1/12264458/f0697d66342f/TPG2-18-e70065-g004.jpg" loading="lazy" id="jats-graphic-5" height="574" width="546" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270065-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Agarose gel electrophoresis images of the <em>Ph1</em>‐specific STS marker <em>Xwgc2049</em> and <em>ph1b</em> deletion‐specific STS marker <em>Xwgc2106</em> amplified by a dual‐plex PCR in the BC<sub>3</sub>F<sub>2</sub>
<em>ph1b</em> introgression populations for the selection of homozygous <em>ph1bph1b</em> individuals in the introgressions involving CS (A), Divide (B), Line E (C), Otto (D), Pembroke 2014 (E), Alsen (F), Argent (G), SS8641 (H), and Jasper (I). <em>M</em>—Size marker.</p></figcaption></figure><p>Three backcrosses were made for each of the introgressions. We found that the wild‐type allele <em>Ph1</em> and the <em>ph1b</em> deletion had a similar transmission rate in each of the introgression backcrosses and their self‐pollinated populations. Preference transmission was not observed for either of these two alleles in the backcross and self‐pollination generations.</p>
<p>Eight <em>ph1b</em> mutant introgression lines were developed in seven modern varieties representing five classes of US wheats (i.e., Alsen, Divide, Argent, SS8641, Otto, Jasper, and Pembroke 2014) and one Australian wheat (Line E). In addition, we restored the CS genetic background in CS <em>ph1b</em> mutant using the same marker‐assisted introgression pipeline (Figure <a href="#tpg270065-fig-0001" class="usa-link">1</a>). So, a total of nine <em>ph1b</em> mutant introgression lines were produced in this study. Phenotypically, the spring wheat <em>ph1b</em> introgression lines were very similar to their respective recipient parents. However, the winter wheat <em>ph1b</em> introgressions had some phenotypical differences from their respective recipient parents especially on maturity, heading date, and spike morphology after three backcrosses under the greenhouse environment. The phenotypic differences between the winter <em>ph1b</em> introgression lines and their recipient parent might result from the background residual of the CS wheat or its interaction with the recipient backgrounds.</p></section><section id="tpg270065-sec-0110"><h3 class="pmc_sec_title">3.2. Cytological analysis of the <em>ph1b</em> introgressions and their recipient wheat parents</h3>
<p>The seven US modern wheat varieties (Alsen, Divide, Argent, SS8641, Otto, Jasper, and Pembroke 2014), CS, and the Australian wheat line (Line E) showed regular diploid‐like chromosome pairing/synapsis during meiosis in the presence of <em>Ph1</em>. Homologous chromosomes in these wheat genotypes containing <em>Ph1</em> allele normally synapsed to form a bivalent (Figure <a href="#tpg270065-fig-0004" class="usa-link">4</a>). Occasionally, some homologous chromosomes failed to synapse and appeared as a univalent, and some chromosomes formed a multivalent due probably to a structural variation in the presence of <em>Ph1</em> (Table <a href="#tpg270065-tbl-0002" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="tpg270065-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/70c1/12264458/56bcf6018224/TPG2-18-e70065-g001.jpg" loading="lazy" id="jats-graphic-7" height="752" width="720" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270065-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Meiotic chromosome pairing at metaphase I (MI) in PMCs (pollen mother cells) of the homozygous <em>ph1b</em> introgression lines and their recurrent parents of Pembroke 2014 (A1), Pembroke 2014 <em>ph1b</em> mutant (A2), SS8461 (B1), SS8461 <em>ph1b</em> mutant (B2), Otto (C1), Otto <em>ph1b</em> mutant (C2), Jasper (D1), Jasper <em>ph1b</em> mutant (D2), Argent (E1), Argent <em>ph1b</em> mutant (E2), Line E (F1), Line E <em>ph1b</em> mutant (F2), Alsen (G1), Alsen <em>ph1b</em> mutant (G2), Divide (H1), Divide <em>ph1b</em> (H2), CS (I1), and CS <em>ph1b</em> mutant (I2). Arrows point to multivalents. Scale bar 5 µm.</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270065-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>
<em>ph1b</em> mutant‐induced homoeologous pairing configurations and frequencies in Chinese Spring (CS) and modern wheat varieties.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th rowspan="2" align="left" colspan="1">Genotypes</th>
<th rowspan="2" align="left" colspan="1">No. of Meiocytes Analyzed</th>
<th colspan="5" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Multivalents and Univalent</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">III</th>
<th align="left" rowspan="1" colspan="1">IV</th>
<th align="left" rowspan="1" colspan="1">V</th>
<th align="left" rowspan="1" colspan="1">VI</th>
<th align="left" rowspan="1" colspan="1">I</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Pembroke 2014</td>
<td align="left" rowspan="1" colspan="1">32</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Pembroke 2014 <em>ph1b</em>
</td>
<td align="left" rowspan="1" colspan="1">70</td>
<td align="left" rowspan="1" colspan="1">23 (32.86%)</td>
<td align="left" rowspan="1" colspan="1">50 (71.43%)</td>
<td align="left" rowspan="1" colspan="1">8 (11.43%)</td>
<td align="left" rowspan="1" colspan="1">6 (8.57%)</td>
<td align="left" rowspan="1" colspan="1">21 (30.00%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SS8641</td>
<td align="left" rowspan="1" colspan="1">39</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">1 (2.56%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SS8641 <em>ph1b</em>
</td>
<td align="left" rowspan="1" colspan="1">50</td>
<td align="left" rowspan="1" colspan="1">26 (52.00%)</td>
<td align="left" rowspan="1" colspan="1">30 (60.00%)</td>
<td align="left" rowspan="1" colspan="1">3 (6.00%)</td>
<td align="left" rowspan="1" colspan="1">1 (2.00%)</td>
<td align="left" rowspan="1" colspan="1">31 (62.00%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Otto</td>
<td align="left" rowspan="1" colspan="1">42</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Otto <em>ph1b</em>
</td>
<td align="left" rowspan="1" colspan="1">79</td>
<td align="left" rowspan="1" colspan="1">34 (43.04%)</td>
<td align="left" rowspan="1" colspan="1">52 (65.82%)</td>
<td align="left" rowspan="1" colspan="1">15 (18.99%)</td>
<td align="left" rowspan="1" colspan="1">5 (6.33%)</td>
<td align="left" rowspan="1" colspan="1">18 (22.78%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Jasper</td>
<td align="left" rowspan="1" colspan="1">33</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">1 (3.03%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Jasper <em>ph1b</em>
</td>
<td align="left" rowspan="1" colspan="1">32</td>
<td align="left" rowspan="1" colspan="1">22 (68.75%)</td>
<td align="left" rowspan="1" colspan="1">18 (56.25%)</td>
<td align="left" rowspan="1" colspan="1">4 (12.50%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">19 (59.38%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Alsen</td>
<td align="left" rowspan="1" colspan="1">36</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">4 (11.11%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Alsen <em>ph1b</em>
</td>
<td align="left" rowspan="1" colspan="1">22</td>
<td align="left" rowspan="1" colspan="1">7 (31.82%)</td>
<td align="left" rowspan="1" colspan="1">18 (81.82%)</td>
<td align="left" rowspan="1" colspan="1">6 (27.27%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">9 (40.91%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Argent</td>
<td align="left" rowspan="1" colspan="1">66</td>
<td align="left" rowspan="1" colspan="1">1 (1.52%)</td>
<td align="left" rowspan="1" colspan="1">4 (6.06%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">3 (4.55%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Argent <em>ph1b</em>
</td>
<td align="left" rowspan="1" colspan="1">94</td>
<td align="left" rowspan="1" colspan="1">40 (42.55%)</td>
<td align="left" rowspan="1" colspan="1">53 (56.38%)</td>
<td align="left" rowspan="1" colspan="1">3 (3.19%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">43 (45.74%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Divide</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">1 (10.00%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Divide <em>ph1b</em>
</td>
<td align="left" rowspan="1" colspan="1">46</td>
<td align="left" rowspan="1" colspan="1">14 (30.43%)</td>
<td align="left" rowspan="1" colspan="1">6 (13.04)</td>
<td align="left" rowspan="1" colspan="1">1 (2.17%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">19 (41.30%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Line E</td>
<td align="left" rowspan="1" colspan="1">26</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">1 (3.85%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Line E <em>ph1b</em>
</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">14 (82.35%)</td>
<td align="left" rowspan="1" colspan="1">11 (64.71%)</td>
<td align="left" rowspan="1" colspan="1">2 (11.76%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">13 (76.47%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CS (CK)</td>
<td align="left" rowspan="1" colspan="1">59</td>
<td align="left" rowspan="1" colspan="1">2 (3.39%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">5 (8.47%)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CS <em>ph1b</em> (CK)</td>
<td align="left" rowspan="1" colspan="1">29</td>
<td align="left" rowspan="1" colspan="1">19 (65.52%)</td>
<td align="left" rowspan="1" colspan="1">13 (44.83%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">0 (0.00%)</td>
<td align="left" rowspan="1" colspan="1">17 (58.62%)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270065-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270065-tbl2-note-0001"><p>
<em>Note</em>: Number and percentage (in parenthesis) of the meiocytes contain a multivalent and/or univalent. <strong>III—</strong>trivalent; <strong>IV—</strong>quadrivalent; <strong>V</strong>—pentavalent; <strong>VI</strong>—hexavalent; <strong>I</strong>—univalent.</p></div></div></section><p>Multivalents and univalent were frequently observed in all nine <em>ph1b</em> introgressions involving CS, seven modern US wheat varieties (Alsen, Divide, Argent, SS8641, Otto, Jasper, and Pembroke 2014), and one Australian wheat line (Line E), indicating <em>ph1b</em> mutant‐induced homoeologous pairing/synapsis between chromosomes from different wheat sub‐genomes (i.e. A, B, and D) (Figure <a href="#tpg270065-fig-0004" class="usa-link">4</a> and Table <a href="#tpg270065-tbl-0002" class="usa-link">2</a>). These meiotic chromosome pairing/synapsis results confirmed the incorporation of <em>ph1b</em> deletion and removal of <em>Ph1</em> in the introgressions. Additionally, this pairwise cytological analysis indicated that <em>ph1b</em> mutant effectively induced meiotic homoeologous pairing and recombination in those different classes of modern wheats.</p>
<p>All hexaploid <em>ph1b</em> introgressions exhibited a similar homoeologous paring/synapsis pattern in terms of the presence of trivalent (III), quadrivalent (IV), pentavalent (V), hexavalent (VI), and univalent (I). Variation in the frequencies of individual multivalent and univalent was observed in different <em>ph1b</em> introgressions. Overall, the tetraploid <em>ph1b</em> introgression “Divide <em>ph1b</em>” showed lower frequencies of multivalents and univalent than the hexaploid introgressions (Table <a href="#tpg270065-tbl-0002" class="usa-link">2</a>).</p></section><section id="tpg270065-sec-0120"><h3 class="pmc_sec_title">3.3. Development and application of <em>Ph1</em>/<em>ph1b</em>‐specific PACE markers</h3>
<p>The <em>ph1b</em> mutant has been widely used to induce meiotic homoeologous recombination for alien introgression in wheat, which enriches and diversifies the wheat genome to increase the genetic potential of wheat for improvement. To improve the utility of the <em>ph1b</em> mutant in alien introgression, we developed high‐throughput co‐dominant PACE markers to differentiate the <em>Ph1</em> allele from the <em>ph1b</em> deletion based on the DNA sequences within the deletion region and across the deletion joint point. This marker system will improve the efficacy and throughput of the homoeologous recombination‐based chromosome engineering in alien introgression.</p>
<p>To design an SNP‐based PACE assay for the <em>Ph1</em> allele and <em>ph1b</em> deletion, we located the deletion breakpoints on chromosome arm 5BL of CS <em>ph1b</em> mutant according to IWGSC RefSeq v2.1 (Zhu et al., <a href="#tpg270065-bib-0044" class="usa-link" aria-describedby="tpg270065-bib-0044">2021</a>) and data reported by Gyawali et al. (<a href="#tpg270065-bib-0017" class="usa-link" aria-describedby="tpg270065-bib-0017">2019</a>) and analyzed the sequences immediately flanking the deletion joint point. The sequences (∼400 bp) adjacent to the distal and proximal breakpoints (DB and PB) of the <em>ph1b</em> deletion were aligned to multiple locations on each wheat chromosome by Basic Local Alignment Search Tool (BLAST) against IWGSC RefSeq v2.1 (Alaux et al., <a href="#tpg270065-bib-0001" class="usa-link" aria-describedby="tpg270065-bib-0001">2018</a>; Zhu et al., <a href="#tpg270065-bib-0044" class="usa-link" aria-describedby="tpg270065-bib-0044">2021</a>). Subsequent BLAST with the longer queries against GIRI and TREP repeat databases (Kohany et al., <a href="#tpg270065-bib-0021" class="usa-link" aria-describedby="tpg270065-bib-0021">2006</a>; Wicker et al., <a href="#tpg270065-bib-0042" class="usa-link" aria-describedby="tpg270065-bib-0042">2002</a>) showed that the 2080‐bp sequence adjacent to DB (distal to 457,812,183 bp on 5BL) was 84.3% similar to the internal portion of Copia‐type LTR retrotransposon Maximus from barley. The 1700‐bp sequence adjacent to the PB (proximal to 397,791,418 bp on 5BL) was 69.6% similar to the internal portion of a wheat Gypsy‐type LTR retrotransposon from GIRI repeat database and 89% similar to the wheat LTR retrotransposon Abana B of unknown type from TREP database. Thus, the genomic regions surrounding both deletion breakpoints (DB and PB) contain dispersed repeats and can be highly variable among hexaploid and tetraploid wheat varieties, which complicates the design of an SNP‐based assay universal for diverse wheat genotypes.</p>
<p>In attempt to design a three‐primer PACE assay (two forward primers specific for <em>ph1b</em> deletion and <em>Ph1</em> allele respectively and one common reverse primer), we identified 5B‐specific SNPs close to DB and PB of the <em>ph1b</em> deletion by analyzing the 5BL sequences adjacent to the DB and PB of common wheat CS and durum wheat “Svevo,” “Strongfield,” and “Capelli,” and their counterparts on homoeologous group‐5 chromosomes 5A and 5D as well as several other chromosomes (Altschul, <a href="#tpg270065-bib-0002" class="usa-link" aria-describedby="tpg270065-bib-0002">1990</a>). Several primer combinations located on the 480‐bp sequence surrounding DB and PB were designed. Two primer combinations adjacent to PB discriminated <em>Ph1</em> and <em>ph1b</em> in CS by PACE assay, but did not work in the same way on <em>Ph1</em> allele detection in some of the nine diverse wheat varieties (Line E, Alsen, Divide, Argent, SS8641, Otto, Jasper, and Pembroke 2014). To overcome this problem, we developed several four‐primer PACE assays with one primer pair specific for <em>ph1b</em> PB and other pair specific for the <em>Ph1</em> region that is absent in the <em>ph1b</em> deletion. We identified B genome‐specific SNPs present in all nine wheat varieties within the <em>Ph1</em> genomic region of 394,969,000–394,979,000 bp (IWGSC RefSeq v2.1; Gyawali et al., <a href="#tpg270065-bib-0017" class="usa-link" aria-describedby="tpg270065-bib-0017">2019</a>) corresponding to the wheat cDNA tplb0019i10 (<a href="http://www.shigen.nig.ac.jp/wheat/komugi/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.shigen.nig.ac.jp/wheat/komugi/</a>) on 5BL, which led to the development of <em>Ph1</em> region‐specific primer pairs. After testing of several four‐primer PACE assays, one of the PACE assays, designated <em>Xwgc2321</em> (Table <a href="#tpg270065-tbl-0001" class="usa-link">1</a>), clearly discriminated the <em>Ph1</em> allele and <em>ph1b</em> deletion and their resulted genotypes <em>Ph1Ph1</em>, <em>Ph1ph1b</em>, and <em>ph1bph1b</em> in all nine diverse wheat varieties (Figure <a href="#tpg270065-fig-0005" class="usa-link">5</a>). The PACE assay results were validated with the STS markers <em>Xwgc2049</em> and <em>Xwgc2111</em> (Gyawali et al., <a href="#tpg270065-bib-0017" class="usa-link" aria-describedby="tpg270065-bib-0017">2019</a>) in all winter and spring wheat varieties, and consistent results were obtained from the PACE assay and STS marker analysis (Figure <a href="#tpg270065-supinfo-0001" class="usa-link">S1</a>).</p>
<figure class="fig xbox font-sm" id="tpg270065-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/70c1/12264458/74957bca9be9/TPG2-18-e70065-g005.jpg" loading="lazy" id="jats-graphic-9" height="587" width="390" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270065-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Four‐primer PCR allelic competitive extension (PACE) assay with <em>Xwgc2321</em> in the BC<sub>3</sub>F<sub>2</sub>
<em>ph1b</em> introgression populations involving winter wheat recipient varieties “SS8641,” “Otto,” “Jasper,” and “Pembroke 2014” (Top) and those involving spring varieties “CS,” “Alsen,” “Argent,” “Line E” and durum wheat variety “Divide” (Bottom). Allele 1 and Allele 2 refer to the <em>Ph1</em>‐specific and <em>ph1b</em>‐specific alleles, respectively. Individuals homozygous for <em>ph1b</em> (i.e., <em>ph1bph1b</em>) are represented by blue squares, those homozygous for <em>Ph1</em> (i.e., <em>Ph1Ph1</em>) by orange circles, those hemizygous for <em>Ph1</em> (i.e. <em>Ph1ph1b</em>) by green triangles, and negative controls (no DNA template, no call) by black diamonds.</p></figcaption></figure></section></section><section id="tpg270065-sec-0130"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Wheat genome has a narrow genetic variation due to its polyploid origin (Dvorák et al., <a href="#tpg270065-bib-0010" class="usa-link" aria-describedby="tpg270065-bib-0010">1993</a>; Huang et al., <a href="#tpg270065-bib-0018" class="usa-link" aria-describedby="tpg270065-bib-0018">2002</a>; Kihara, <a href="#tpg270065-bib-0020" class="usa-link" aria-describedby="tpg270065-bib-0020">1944</a>; Levy and Feldman <a href="#tpg270065-bib-0022" class="usa-link" aria-describedby="tpg270065-bib-0022">2022</a>; Riley et al., <a href="#tpg270065-bib-0033" class="usa-link" aria-describedby="tpg270065-bib-0033">1958</a>), which has increasingly become a bottleneck for wheat improvement. Wheat ancestors and numerous relatives, including wild and cultivated species, harbor tremendous genetic variation. They represent an invaluable gene reservoir useful in extending the genetic variability of the wheat genome and increasing the genetic potential of wheat in grain production. Meiotic homoeologous recombination‐based chromosome engineering has been considered a most effective approach of bridging gene flow from the relatives into wheat (Boehm &amp; Cai, <a href="#tpg270065-bib-0005" class="usa-link" aria-describedby="tpg270065-bib-0005">2024</a>; B. S. Gill et al., <a href="#tpg270065-bib-0013" class="usa-link" aria-describedby="tpg270065-bib-0013">2011</a>; Qi et al., <a href="#tpg270065-bib-0028" class="usa-link" aria-describedby="tpg270065-bib-0028">2007</a>; Rey et al., <a href="#tpg270065-bib-0030" class="usa-link" aria-describedby="tpg270065-bib-0030">2015</a>).</p>
<p>The <em>ph1b</em> mutant developed in the old wheat landrace CS has been commonly used to induce meiotic homoeologous recombination between wheat and alien chromosomes for gene introgression from the relatives into wheat (Zhang et al., <a href="#tpg270065-bib-0043" class="usa-link" aria-describedby="tpg270065-bib-0043">2017</a>). But the CS wheat <em>ph1b</em> mutant introduces undesired agronomic traits of CS into the introgressions besides inducing homoeologous recombination, which limits the utility of the <em>ph1b</em> mutant in alien introgression. In this study, we deployed the <em>ph1b</em> mutant into modern US wheats. The <em>ph1b</em> mutant was found to induce frequent meiotic homoeologous pairing in all US wheats and Australian wheat Line E as it does in CS, indicating the <em>ph1b</em> mutant normally functions to enhance homoeologous recombination for alien introgression in all these wheat genotypes (Figure <a href="#tpg270065-fig-0004" class="usa-link">4</a> and Table <a href="#tpg270065-tbl-0002" class="usa-link">2</a>). The frequencies of multivalents and univalent in all <em>ph1b</em> introgressions were significantly higher than their wild‐type recipients even though we observed variation among the genotypes (Table <a href="#tpg270065-tbl-0002" class="usa-link">2</a>). The lower homoeologous pairing frequency in the tetraploid durum wheat Divide <em>ph1b</em> mutant than common wheat <em>ph1b</em> mutants probably resulted from the lack of the D sub‐genome in durum wheat comparing to the hexaploid common wheat.</p>
<p>The codominance‐like nature of the multiplex PCR‐based STS and four‐primer PACE markers specific for the wild‐type <em>Ph1</em> allele and <em>ph1b</em> deletion allowed for the differentiation of the three genotypes at the <em>Ph1</em> locus, including <em>Ph1Ph1</em>, <em>Ph1ph1b</em>, and <em>ph1bph1b</em>, in this <em>ph1b</em> introgression study (Figure <a href="#tpg270065-fig-0005" class="usa-link">5</a>). This could not be done with regular dominant markers specific for the <em>Ph1</em> genomic region (K. S. Gill &amp; Gill, <a href="#tpg270065-bib-0014" class="usa-link" aria-describedby="tpg270065-bib-0014">1996</a>; Roberts et al., <a href="#tpg270065-bib-0034" class="usa-link" aria-describedby="tpg270065-bib-0034">1999</a>) before the <em>ph1b</em> deletion breakpoints were demarcated by Gyawali et al. (<a href="#tpg270065-bib-0017" class="usa-link" aria-describedby="tpg270065-bib-0017">2019</a>). These special STS and PACE markers were highly diagnostic for both alleles in all wheat genotypes involved in this study (Figures <a href="#tpg270065-fig-0003" class="usa-link">3</a> and <a href="#tpg270065-fig-0005" class="usa-link">5</a>). They will improve the utility of the US wheat <em>ph1b</em> mutants in alien introgression and wheat breeding.</p>
<p>No polymorphism was detected at the <em>Ph1</em>‐specific STS locus <em>Xwgc2049</em> among the recipient wheat parents included in this study and at the <em>ph1b</em> deletion‐specific STS locus <em>Xwgc2106</em> among the <em>ph1b</em> introgression genotypes in this study (Figures <a href="#tpg270065-fig-0002" class="usa-link">2</a> and <a href="#tpg270065-fig-0003" class="usa-link">3</a>). These findings suggest that the genomic regions at both STS loci might be conservative. Another possibility of no polymorphism at the <em>ph1b</em> deletion‐specific STS locus <em>Xwgc2106</em> among the introgressions would be the incorporation of the entire CS genomic region spanning the <em>ph1b</em> deletion breakpoint into the introgressions. This might result from the interrupted meiotic recombination between the wild‐type <em>Ph1</em> genomic region of the recipient wheat genotypes and CS <em>ph1b</em> deletion, leading to a uniparental transmission of the CS <em>ph1b</em> deletion into the introgressions. The <em>ph1b</em> deletion will be eventually eliminated from the end products of alien introgression because it breaks the stability and integrity of the wheat polyploid genome. That CS genomic region flanking the <em>ph1b</em> deletion will be removed together with the deletion. So, it would not bring in undesired traits into alien introgressions if that is the case. Further studies of this genomic region will provide a better understanding of the <em>ph1b</em> introgression.</p>
<p>The US wheat <em>ph1b</em> mutants developed in this study will enable direct gene flow from wheat relatives into adapted US wheat backgrounds and increase the efficacy of homoeologous recombination‐based alien introgression in wheat breeding. The Australian wheat Line E is universally susceptible to all rust races. So, the Line E <em>ph1b</em> mutant will be an ideal recipient parent for rust resistance gene introgression and identification. All these <em>ph1b</em> introgressions will be released and made available to the wheat breeding and genetic research community for alien introgression, genetic and genome studies, and germplasm/variety development. Thus, this work will enhance gene introgression from wheat‐related grass species and ancestors directly into US wheats and ultimately increase the genetic variability of US wheats to resolve the bottleneck of wheat improvement.</p></section><section id="tpg270065-sec-0140"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Yadav Gyawali</strong>: Data curation; formal analysis; investigation; methodology; resources; validation; visualization; writing—review and editing. <strong>Wei Zhang</strong>: Data curation; formal analysis; investigation; methodology; resources; validation; visualization; writing—review and editing. <strong>Tatiana V. Danilova</strong>: Data curation; formal analysis; investigation; methodology; resources; validation; visualization; writing—original draft; writing—review and editing. <strong>Mingyi Zhang</strong>: Data curation; investigation; resources; writing—review and editing. <strong>Steven Xu</strong>: Methodology; resources; validation; writing—review and editing. <strong>David Van Sanford</strong>: Resources; writing—review and editing. <strong>Arron Carter</strong>: Resources; writing—review and editing. <strong>Elias Elias</strong>: Resources; writing—review and editing. <strong>Xiwen Cai</strong>: Conceptualization; data curation; formal analysis; funding acquisition; investigation; methodology; project administration; resources; software; supervision; validation; visualization; writing—original draft; writing—review and editing.</p></section><section id="tpg270065-sec-0160"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec16"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270065-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270065-supitem-0001"><div class="caption p"><p>
<strong>Figure S1</strong> Validation of PACE results with STS markers <em>Xwgc2049</em> and <em>Xwgc2111</em>. <strong>
<em>N</em>
</strong> – no DNA template, <strong>
<em>M</em>
</strong> – 100bp ladder, <strong>
<em>1</em>
</strong> – CS, <strong>
<em>2</em>
</strong> – CS <em>ph1b</em>, <strong>
<em>3</em>
</strong> – SS8641, <strong>
<em>4</em>
</strong> – SS8641 <em>ph1b</em>, <strong>
<em>5‐22</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving SS8641; <strong>
<em>23</em>
</strong> – Otto, <strong>
<em>24</em>
</strong> – Otto <em>ph1b</em>, <strong>
<em>25‐44</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Otto; <strong>
<em>45</em>
</strong> – Jasper, <strong>
<em>46</em>
</strong> – Jasper <em>ph1b</em>, <strong>
<em>47‐64</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population Jasper; <strong>
<em>65</em>
</strong> – Pembroke 2014, <strong>
<em>66</em>
</strong> – Pembroke 2014 <em>ph1b</em>, <strong>
<em>67‐84</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Pembroke 2014; <strong>
<em>85‐98</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving CS; <strong>
<em>99</em>
</strong> – Alsen, <strong>
<em>100</em>
</strong> – Alsen <em>ph1b</em>, <strong>
<em>101‐115</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Alsen; <strong>
<em>116</em>
</strong> – Argent, <strong>
<em>117</em>
</strong> – Argent <em>ph1b</em>, <strong>
<em>118‐132</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Argent; <strong>
<em>133</em>
</strong> – Divide, <strong>
<em>134</em>
</strong> – Divide <em>ph1b</em>, <strong>
<em>135‐149</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Divide; <strong>
<em>150</em>
</strong> – Line E, <strong>
<em>151</em>
</strong> – Line E <em>ph1b</em>, <strong>
<em>152‐165</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Line E. Arrows point to the PCR products of <em>Xwgc2111</em> (1,067bp) and <em>Xwgc2049</em> (846bp).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12264458/bin/TPG2-18-e70065-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70065-s001.docx</a><sup> (811.5KB, docx) </sup>
</div></div></section></section><section id="tpg270065-sec-0150" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We thank members of the laboratories involved for their help to this research. This project has been supported by Agriculture and Food Research Initiative Competitive Grant no. 2016‐67014‐24455 and 2019‐67013‐35750 from the USDA National Institute of Food and Agriculture and the USDA‐ARS CRIS Project No. 3042‐21000‐035‐000D.</p></section><section id="notes1"><p>


Gyawali, Y.
, 
Zhang, W.
, 
Danilova, T. V.
, 
Zhang, M.
, 
Xu, S.
, 
Van Sanford, D.
, 
Carter, A.
, 
Elias, E.
, &amp; 
Cai, X.
 (2025). Marker‐assisted introgression and characterization of the <em>ph1b</em> mutant in modern US and Australian wheats. The Plant Genome, 18, e70065. 10.1002/tpg2.70065

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="tpg270065-note-0001">
<p>
<strong>Present address</strong>
</p>
<p>Yadav Gyawali, College of Pharmacy, MSC09 5360, 1 University of New Mexico, Albuquerque, NM 87131, USA.</p>
</div>
<div class="fn p" id="tpg270065-note-0002"><p>Assigned to Associate Editor Kulvinder Gill.</p></div>
</div></section></section><section id="tpg270065-sec-0180"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>There are no data associated with this manuscript.</p></section><section id="tpg270065-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270065-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270065-bib-0001">
<cite>

Alaux, M.
, 
Rogers, J.
, 
Letellier, T.
, 
Flores, R.
, 
Alfama, F.
, 
Pommier, C.
, 
Mohellibi, N.
, 
Durand, S.
, 
Kimmel, E.
, 
Michotey, C.
, 
Guerche, C.
, 
Loaec, M.
, 
Lainé, M.
, 
Steinbach, D.
, 
Choulet, F.
, 
Rimbert, H.
, 
Leroy, P.
, 
Guilhot, N.
, 
Salse, J.
, … 
Quesneville, H.
 (2018). Linking the International Wheat Genome Sequencing Consortium bread wheat reference genome sequence to wheat genetic and phenomic data. Genome Biology, 19, 1–10. 10.1186/s13059-018-1491-4

</cite> [<a href="https://doi.org/10.1186/s13059-018-1491-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6097284/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30115101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Linking%20the%20International%20Wheat%20Genome%20Sequencing%20Consortium%20bread%20wheat%20reference%20genome%20sequence%20to%20wheat%20genetic%20and%20phenomic%20data&amp;author=M.%20Alaux&amp;author=J.%20Rogers&amp;author=T.%20Letellier&amp;author=R.%20Flores&amp;author=F.%20Alfama&amp;volume=19&amp;publication_year=2018&amp;pages=1-10&amp;pmid=30115101&amp;doi=10.1186/s13059-018-1491-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0002">
<cite>

Altschul, S.
 (1990). Basic local alignment search tool. Journal of Molecular Biology, 215, 403–410. 10.1016/S0022-2836(05)80360-2

</cite> [<a href="https://doi.org/10.1016/S0022-2836(05)80360-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Biology&amp;title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20Altschul&amp;volume=215&amp;publication_year=1990&amp;pages=403-410&amp;pmid=2231712&amp;doi=10.1016/S0022-2836(05)80360-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0005">
<cite>

Boehm, J. D.
, &amp; 
Cai, X.
 (2024). Enrichment and diversification of the wheat genome by alien introgression. Plants, 13, 339. 10.3390/plants13030339

</cite> [<a href="https://doi.org/10.3390/plants13030339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10857235/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38337872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Enrichment%20and%20diversification%20of%20the%20wheat%20genome%20by%20alien%20introgression&amp;author=J.%20D.%20Boehm&amp;author=X.%20Cai&amp;volume=13&amp;publication_year=2024&amp;pages=339&amp;pmid=38337872&amp;doi=10.3390/plants13030339&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0007">
<cite>

Cai, X. W.
, 
Jones, S. S.
, &amp; 
Murray, T. D.
 (1996). Characterization of an <em>Agropyron elongatum</em> chromosome conferring resistance to Cephalosporium stripe in common wheat. Genome, 39, 56–62. 10.1139/g96-008

</cite> [<a href="https://doi.org/10.1139/g96-008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18469877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome&amp;title=Characterization%20of%20an%20Agropyron%20elongatum%20chromosome%20conferring%20resistance%20to%20Cephalosporium%20stripe%20in%20common%20wheat&amp;author=X.%20W.%20Cai&amp;author=S.%20S.%20Jones&amp;author=T.%20D.%20Murray&amp;volume=39&amp;publication_year=1996&amp;pages=56-62&amp;pmid=18469877&amp;doi=10.1139/g96-008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0008">
<cite>

Chen, P. D.
, 
Tsujimoto, H.
, &amp; 
Gill, B. S.
 (1994). Transfer of <em>Ph (I)</em> genes promoting homoeologous pairing from <em>Triticum speltoides</em> to common wheat. Theoretical and Applied Genetics, 88, 97–101. 10.1007/BF00222400

</cite> [<a href="https://doi.org/10.1007/BF00222400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24185888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Transfer%20of%20Ph%20(I)%20genes%20promoting%20homoeologous%20pairing%20from%20Triticum%20speltoides%20to%20common%20wheat&amp;author=P.%20D.%20Chen&amp;author=H.%20Tsujimoto&amp;author=B.%20S.%20Gill&amp;volume=88&amp;publication_year=1994&amp;pages=97-101&amp;pmid=24185888&amp;doi=10.1007/BF00222400&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0009">
<cite>

Dvorak, J.
, 
Deal, K. R.
, &amp; 
Luo, M. C.
 (2006). Discovery and mapping of wheat <em>Ph1</em> suppressors. Genetics, 174, 17–27. 10.1534/genetics.106.058115

</cite> [<a href="https://doi.org/10.1534/genetics.106.058115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1569802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16702426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Discovery%20and%20mapping%20of%20wheat%20Ph1%20suppressors&amp;author=J.%20Dvorak&amp;author=K.%20R.%20Deal&amp;author=M.%20C.%20Luo&amp;volume=174&amp;publication_year=2006&amp;pages=17-27&amp;pmid=16702426&amp;doi=10.1534/genetics.106.058115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0010">
<cite>

Dvorák, J.
, 
Terlizzi, P. D.
, 
Zhang, H.‐B.
, &amp; 
Resta, P.
 (1993). The evolution of polyploid wheats: Identification of the A genome donor species. Genome, 36, 21–31. 10.1139/g93-004

</cite> [<a href="https://doi.org/10.1139/g93-004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18469969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome&amp;title=The%20evolution%20of%20polyploid%20wheats:%20Identification%20of%20the%20A%20genome%20donor%20species&amp;author=J.%20Dvor%C3%A1k&amp;author=P.%20D.%20Terlizzi&amp;author=H.%E2%80%90B.%20Zhang&amp;author=P.%20Resta&amp;volume=36&amp;publication_year=1993&amp;pages=21-31&amp;pmid=18469969&amp;doi=10.1139/g93-004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0012">
<cite>

Friebe, B.
, 
Qi, L.
, 
Liu, C.
, 
Liu, W.
, &amp; 
Gill, B. S.
 (2012). Registration of a Hard Red Winter Wheat genetic stock homozygous for <em>ph1b</em> for facilitating alien introgression for crop improvement. Journal of Plant Registration, 6, 121–123. 10.3198/jpr2011.05.0273crgs
</cite> [<a href="https://doi.org/10.3198/jpr2011.05.0273crgs" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Plant%20Registration&amp;title=Registration%20of%20a%20Hard%20Red%20Winter%20Wheat%20genetic%20stock%20homozygous%20for%20ph1b%20for%20facilitating%20alien%20introgression%20for%20crop%20improvement&amp;author=B.%20Friebe&amp;author=L.%20Qi&amp;author=C.%20Liu&amp;author=W.%20Liu&amp;author=B.%20S.%20Gill&amp;volume=6&amp;publication_year=2012&amp;pages=121-123&amp;doi=10.3198/jpr2011.05.0273crgs&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0013">
<cite>

Gill, B. S.
, 
Friebe, B. R.
, &amp; 
White, F. F.
 (2011). Alien introgressions represent a rich source of genes for crop improvement. The Proceedings of the National Academy of Sciences, 108(19), 7657–7658. 10.1073/pnas.1104845108
</cite> [<a href="https://doi.org/10.1073/pnas.1104845108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3093505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21527718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Alien%20introgressions%20represent%20a%20rich%20source%20of%20genes%20for%20crop%20improvement&amp;author=B.%20S.%20Gill&amp;author=B.%20R.%20Friebe&amp;author=F.%20F.%20White&amp;volume=108&amp;issue=19&amp;publication_year=2011&amp;pages=7657-7658&amp;pmid=21527718&amp;doi=10.1073/pnas.1104845108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0014">
<cite>

Gill, K. S.
, &amp; 
Gill, B. S.
 (1996). A PCR‐based screening assay of <em>Ph1</em>, the chromosome pairing regulator gene of wheat. Crop Science, 36, 719–722. 10.2135/cropsci1996.0011183X003600030033x
</cite> [<a href="https://doi.org/10.2135/cropsci1996.0011183X003600030033x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=A%20PCR%E2%80%90based%20screening%20assay%20of%20Ph1,%20the%20chromosome%20pairing%20regulator%20gene%20of%20wheat&amp;author=K.%20S.%20Gill&amp;author=B.%20S.%20Gill&amp;volume=36&amp;publication_year=1996&amp;pages=719-722&amp;doi=10.2135/cropsci1996.0011183X003600030033x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0015">
<cite>

Gill, K. S.
, 
Gill, B. S.
, 
Endo, T. R.
, &amp; 
Mukai, Y.
 (1993). Fine physical mapping of <em>Ph1</em>, a chromosome pairing regulator gene in polyploid wheat. Genetics, 134, 1231–1236. 10.1093/genetics/134.4.1231

</cite> [<a href="https://doi.org/10.1093/genetics/134.4.1231" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1205590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8375657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Fine%20physical%20mapping%20of%20Ph1,%20a%20chromosome%20pairing%20regulator%20gene%20in%20polyploid%20wheat&amp;author=K.%20S.%20Gill&amp;author=B.%20S.%20Gill&amp;author=T.%20R.%20Endo&amp;author=Y.%20Mukai&amp;volume=134&amp;publication_year=1993&amp;pages=1231-1236&amp;pmid=8375657&amp;doi=10.1093/genetics/134.4.1231&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0016">
<cite>

Giorgi, B.
 (1978). A homoeologous pairing mutant isolated in <em>Triticum durum</em> cv ‘Cappelli’. Mutation Breeding Newsletter, 11, 4–5.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Mutation%20Breeding%20Newsletter&amp;title=A%20homoeologous%20pairing%20mutant%20isolated%20in%20Triticum%20durum%20cv%20%E2%80%98Cappelli%E2%80%99&amp;author=B.%20Giorgi&amp;volume=11&amp;publication_year=1978&amp;pages=4-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0017">
<cite>

Gyawali, Y.
, 
Zhang, W.
, 
Chao, S.
, 
Xu, S.
, &amp; 
Cai, X.
 (2019). Delimitation of wheat <em>ph1b</em> deletion and development of <em>ph1b</em>‐specific DNA markers. Theoretical and Applied Genetics, 132, 195–204. 10.1007/s00122-018-3207-2

</cite> [<a href="https://doi.org/10.1007/s00122-018-3207-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30343385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Delimitation%20of%20wheat%20ph1b%20deletion%20and%20development%20of%20ph1b%E2%80%90specific%20DNA%20markers&amp;author=Y.%20Gyawali&amp;author=W.%20Zhang&amp;author=S.%20Chao&amp;author=S.%20Xu&amp;author=X.%20Cai&amp;volume=132&amp;publication_year=2019&amp;pages=195-204&amp;pmid=30343385&amp;doi=10.1007/s00122-018-3207-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0018">
<cite>

Huang, X.
, 
Börner, A.
, 
Röder, M.
, &amp; 
Ganal, M.
 (2002). Assessing genetic diversity of wheat (<em>Triticum aestivum</em> L.) germplasm using microsatellite markers. Theoretical and Applied Genetics, 105, 699–707. 10.1007/s00122-002-0959-4

</cite> [<a href="https://doi.org/10.1007/s00122-002-0959-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12582483/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Assessing%20genetic%20diversity%20of%20wheat%20(Triticum%20aestivum%20L.)%20germplasm%20using%20microsatellite%20markers&amp;author=X.%20Huang&amp;author=A.%20B%C3%B6rner&amp;author=M.%20R%C3%B6der&amp;author=M.%20Ganal&amp;volume=105&amp;publication_year=2002&amp;pages=699-707&amp;pmid=12582483&amp;doi=10.1007/s00122-002-0959-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0019">
<cite>

Kihara, H.
 (1919). Üeber cytologische Studien bei einigen Getreidearten. Spezies‐Bastarde des Weizens und Weizenroggen‐Bastard. The Botanical Magazine, 33, 17–38. 10.15281/jplantres1887.33.386_17
</cite> [<a href="https://doi.org/10.15281/jplantres1887.33.386_17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Botanical%20Magazine&amp;title=%C3%9Ceber%20cytologische%20Studien%20bei%20einigen%20Getreidearten.%20Spezies%E2%80%90Bastarde%20des%20Weizens%20und%20Weizenroggen%E2%80%90Bastard&amp;author=H.%20Kihara&amp;volume=33&amp;publication_year=1919&amp;pages=17-38&amp;doi=10.15281/jplantres1887.33.386_17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0020">
<cite>

Kihara, H.
 (1944). Discovery of the DD‐analyser, one of the ancestors of <em>Triticum vulgare</em>
. Agriculture and Horticulture, 19, 13–14.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Agriculture%20and%20Horticulture&amp;title=Discovery%20of%20the%20DD%E2%80%90analyser,%20one%20of%20the%20ancestors%20of%20Triticum%20vulgare&amp;author=H.%20Kihara&amp;volume=19&amp;publication_year=1944&amp;pages=13-14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0021">
<cite>

Kohany, O.
, 
Gentles, A. J.
, 
Hankus, L.
, &amp; 
Jurka, J.
 (2006). Annotation, submission and screening of repetitive elements in Repbase: RepbaseSubmitter and Censor. BMC Bioinformatics, 7, 474. 10.1186/1471-2105-7-474

</cite> [<a href="https://doi.org/10.1186/1471-2105-7-474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1634758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17064419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Annotation,%20submission%20and%20screening%20of%20repetitive%20elements%20in%20Repbase:%20RepbaseSubmitter%20and%20Censor&amp;author=O.%20Kohany&amp;author=A.%20J.%20Gentles&amp;author=L.%20Hankus&amp;author=J.%20Jurka&amp;volume=7&amp;publication_year=2006&amp;pages=474&amp;pmid=17064419&amp;doi=10.1186/1471-2105-7-474&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0022">
<cite>

Levy, A. A.
, &amp; 
Feldman, M.
 (2022). Evolution and origin of bread wheat. Plant Cell, 34, 2549–2567. 10.1093/plcell/koac130

</cite> [<a href="https://doi.org/10.1093/plcell/koac130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9252504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35512194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Evolution%20and%20origin%20of%20bread%20wheat&amp;author=A.%20A.%20Levy&amp;author=M.%20Feldman&amp;volume=34&amp;publication_year=2022&amp;pages=2549-2567&amp;pmid=35512194&amp;doi=10.1093/plcell/koac130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0026">
<cite>

Niu, Z.
, 
Klindworth, D. L.
, 
Friesen, T. L.
, 
Chao, S.
, 
Jin, Y.
, 
Cai, X.
, &amp; 
Xu, S. S.
 (2011). DNA marker‐assisted chromosome engineering of wheat carrying stem rust resistance gene Sr39 derived from <em>Aegilops speltoides</em>
. Genetics, 187, 1011–1021. 10.1534/genetics.110.123588

</cite> [<a href="https://doi.org/10.1534/genetics.110.123588" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3070511/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21242535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=DNA%20marker%E2%80%90assisted%20chromosome%20engineering%20of%20wheat%20carrying%20stem%20rust%20resistance%20gene%20Sr39%20derived%20from%20Aegilops%20speltoides&amp;author=Z.%20Niu&amp;author=D.%20L.%20Klindworth&amp;author=T.%20L.%20Friesen&amp;author=S.%20Chao&amp;author=Y.%20Jin&amp;volume=187&amp;publication_year=2011&amp;pages=1011-1021&amp;pmid=21242535&amp;doi=10.1534/genetics.110.123588&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0028">
<cite>

Qi, L.
, 
Friebe, B.
, 
Zhang, P.
, &amp; 
Gill, B. S.
 (2007). Homoeologous recombination, chromosome engineering and crop improvement. Chromosome Research, 15, 3–19. 10.1007/s10577-006-1108-8

</cite> [<a href="https://doi.org/10.1007/s10577-006-1108-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17295123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chromosome%20Research&amp;title=Homoeologous%20recombination,%20chromosome%20engineering%20and%20crop%20improvement&amp;author=L.%20Qi&amp;author=B.%20Friebe&amp;author=P.%20Zhang&amp;author=B.%20S.%20Gill&amp;volume=15&amp;publication_year=2007&amp;pages=3-19&amp;pmid=17295123&amp;doi=10.1007/s10577-006-1108-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0030">
<cite>

Rey, E.
, 
Molnár, I.
, &amp; 
Doležel, J.
 (2015). Genomics of wild relatives and alien introgressions. In Molnár‐Láng M., Ceoloni C., &amp; Doležel J. (Eds.), Alien introgression in wheat (pp. 347–381). Springer. 10.1007/978-3-319-23494-6_13
</cite> [<a href="https://doi.org/10.1007/978-3-319-23494-6_13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Alien%20introgression%20in%20wheat&amp;author=E.%20Rey&amp;author=I.%20Moln%C3%A1r&amp;author=J.%20Dole%C5%BEel&amp;publication_year=2015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0033">
<cite>

Riley, R.
, 
Unrau, J.
, &amp; 
Chapman, V.
 (1958). Evidence on the origin of the B genome of wheat. Journal of Heredity, 49, 91–98. 10.1093/oxfordjournals.jhered.a106784
</cite> [<a href="https://doi.org/10.1093/oxfordjournals.jhered.a106784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Heredity&amp;title=Evidence%20on%20the%20origin%20of%20the%20B%20genome%20of%20wheat&amp;author=R.%20Riley&amp;author=J.%20Unrau&amp;author=V.%20Chapman&amp;volume=49&amp;publication_year=1958&amp;pages=91-98&amp;doi=10.1093/oxfordjournals.jhered.a106784&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0034">
<cite>

Roberts, M. A.
, 
Reader, S. M.
, 
Dalgliesh, C.
, 
Miller, T. E.
, 
Foote, T. N.
, 
Fish, L. J.
, 
Snape, J. W.
, &amp; 
Moore, G.
 (1999). Induction and characterization of <em>Ph1</em> wheat mutants. Genetics, 153, 1909–1918. 10.1093/genetics/153.4.1909

</cite> [<a href="https://doi.org/10.1093/genetics/153.4.1909" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1460846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10581295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Induction%20and%20characterization%20of%20Ph1%20wheat%20mutants&amp;author=M.%20A.%20Roberts&amp;author=S.%20M.%20Reader&amp;author=C.%20Dalgliesh&amp;author=T.%20E.%20Miller&amp;author=T.%20N.%20Foote&amp;volume=153&amp;publication_year=1999&amp;pages=1909-1918&amp;pmid=10581295&amp;doi=10.1093/genetics/153.4.1909&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0035">
<cite>

Sears, E. R.
 (1952). Homoeologous chromosomes in <em>Triticum aestivum</em>
. Genetics, 37, 624–631.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Homoeologous%20chromosomes%20in%20Triticum%20aestivum&amp;author=E.%20R.%20Sears&amp;volume=37&amp;publication_year=1952&amp;pages=624-631&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0036">
<cite>

Sears, E. R.
 (1976). Genetic control of chromosome pairing in wheat. Annual Review of Genetics, 10, 31–51. 10.1146/annurev.ge.10.120176.000335
</cite> [<a href="https://doi.org/10.1146/annurev.ge.10.120176.000335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/797311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Genetics&amp;title=Genetic%20control%20of%20chromosome%20pairing%20in%20wheat&amp;author=E.%20R.%20Sears&amp;volume=10&amp;publication_year=1976&amp;pages=31-51&amp;pmid=797311&amp;doi=10.1146/annurev.ge.10.120176.000335&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0037">
<cite>

Sears, E. R.
 (1977). An induced mutant with homoeologous pairing in common wheat. Canadian Journal of Genetics and Cytology, 19, 585–593. 10.1139/g77-063
</cite> [<a href="https://doi.org/10.1139/g77-063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Genetics%20and%20Cytology&amp;title=An%20induced%20mutant%20with%20homoeologous%20pairing%20in%20common%20wheat&amp;author=E.%20R.%20Sears&amp;volume=19&amp;publication_year=1977&amp;pages=585-593&amp;doi=10.1139/g77-063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0040">
<cite>

Türkösi, E.
, 
Ivanizs, L.
, 
Farkas, A.
, 
Gaál, E.
, 
Kruppa, K.
, 
Kovács, P.
, 
Szakács, É.
, 
Szőke‐Pázsi, K.
, 
Said, M.
, 
Cápal, P.
, 
Griffiths, S.
, 
Doležel, J.
, &amp; 
Molnár, I.
 (2022). Transfer of the <em>ph1b</em> deletion chromosome 5B from Chinese Spring wheat into a winter wheat line and induction of chromosome rearrangements in wheat‐<em>Aegilops biuncialis</em> hybrids. Frontiers in Plant Science, 13, 875676. 10.3389/fpls.2022.875676

</cite> [<a href="https://doi.org/10.3389/fpls.2022.875676" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9234525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35769292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Transfer%20of%20the%20ph1b%20deletion%20chromosome%205B%20from%20Chinese%20Spring%20wheat%20into%20a%20winter%20wheat%20line%20and%20induction%20of%20chromosome%20rearrangements%20in%20wheat%E2%80%90Aegilops%20biuncialis%20hybrids&amp;author=E.%20T%C3%BCrk%C3%B6si&amp;author=L.%20Ivanizs&amp;author=A.%20Farkas&amp;author=E.%20Ga%C3%A1l&amp;author=K.%20Kruppa&amp;volume=13&amp;publication_year=2022&amp;pages=875676&amp;pmid=35769292&amp;doi=10.3389/fpls.2022.875676&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0041">
<cite>

Wang, X.
, 
Lai, J.
, 
Liu, G.
, &amp; 
Chen, F.
 (2002). Development of a scar marker for the <em>Ph1</em> locus in common wheat and its application. Crop Science, 42, 1365–1368. 10.2135/cropsci2002.1365
</cite> [<a href="https://doi.org/10.2135/cropsci2002.1365" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Development%20of%20a%20scar%20marker%20for%20the%20Ph1%20locus%20in%20common%20wheat%20and%20its%20application&amp;author=X.%20Wang&amp;author=J.%20Lai&amp;author=G.%20Liu&amp;author=F.%20Chen&amp;volume=42&amp;publication_year=2002&amp;pages=1365-1368&amp;doi=10.2135/cropsci2002.1365&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0042">
<cite>

Wicker, T.
, 
Matthews, D. E.
, &amp; 
Keller, B.
 (2002). TREP: A database for Triticeae repetitive elements. Trends in Plant Science, 7, 561–562. 10.1016/S1360-1385(02)02372-5
</cite> [<a href="https://doi.org/10.1016/S1360-1385(02)02372-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=TREP:%20A%20database%20for%20Triticeae%20repetitive%20elements&amp;author=T.%20Wicker&amp;author=D.%20E.%20Matthews&amp;author=B.%20Keller&amp;volume=7&amp;publication_year=2002&amp;pages=561-562&amp;doi=10.1016/S1360-1385(02)02372-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0043">
<cite>

Zhang, W.
, 
Cao, Y.
, 
Zhang, M.
, 
Zhu, X.
, 
Ren, S.
, 
Long, Y.
, 
Gyawali, Y.
, 
Chao, S.
, 
Xu, S. S.
, &amp; 
Cai, X.
 (2017). Meiotic homoeologous recombination‐based alien gene introgression in the genomics era of wheat. Crop Science, 57, 1189–1198. 10.2135/cropsci2016.09.0819
</cite> [<a href="https://doi.org/10.2135/cropsci2016.09.0819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Meiotic%20homoeologous%20recombination%E2%80%90based%20alien%20gene%20introgression%20in%20the%20genomics%20era%20of%20wheat&amp;author=W.%20Zhang&amp;author=Y.%20Cao&amp;author=M.%20Zhang&amp;author=X.%20Zhu&amp;author=S.%20Ren&amp;volume=57&amp;publication_year=2017&amp;pages=1189-1198&amp;doi=10.2135/cropsci2016.09.0819&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270065-bib-0044">
<cite>

Zhu, T.
, 
Wang, L.
, 
Rimbert, H.
, 
Rodriguez, J. C.
, 
Deal, K. R.
, 
De Oliveira, R.
, 
Choulet, F.
, 
Keeble‐Gagnère, G.
, 
Tibbits, J.
, 
Rogers, J.
, &amp; 
Eversole, K.
 (2021). Optical maps refine the bread wheat <em>Triticum aestivum</em> cv. Chinese Spring genome assembly. The Plant Journal, 107(1), 303–314. 10.1111/tpj.15289

</cite> [<a href="https://doi.org/10.1111/tpj.15289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8360199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33893684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=Optical%20maps%20refine%20the%20bread%20wheat%20Triticum%20aestivum%20cv.%20Chinese%20Spring%20genome%20assembly&amp;author=T.%20Zhu&amp;author=L.%20Wang&amp;author=H.%20Rimbert&amp;author=J.%20C.%20Rodriguez&amp;author=K.%20R.%20Deal&amp;volume=107&amp;issue=1&amp;publication_year=2021&amp;pages=303-314&amp;pmid=33893684&amp;doi=10.1111/tpj.15289&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Figure S1</strong> Validation of PACE results with STS markers <em>Xwgc2049</em> and <em>Xwgc2111</em>. <strong>
<em>N</em>
</strong> – no DNA template, <strong>
<em>M</em>
</strong> – 100bp ladder, <strong>
<em>1</em>
</strong> – CS, <strong>
<em>2</em>
</strong> – CS <em>ph1b</em>, <strong>
<em>3</em>
</strong> – SS8641, <strong>
<em>4</em>
</strong> – SS8641 <em>ph1b</em>, <strong>
<em>5‐22</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving SS8641; <strong>
<em>23</em>
</strong> – Otto, <strong>
<em>24</em>
</strong> – Otto <em>ph1b</em>, <strong>
<em>25‐44</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Otto; <strong>
<em>45</em>
</strong> – Jasper, <strong>
<em>46</em>
</strong> – Jasper <em>ph1b</em>, <strong>
<em>47‐64</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population Jasper; <strong>
<em>65</em>
</strong> – Pembroke 2014, <strong>
<em>66</em>
</strong> – Pembroke 2014 <em>ph1b</em>, <strong>
<em>67‐84</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Pembroke 2014; <strong>
<em>85‐98</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving CS; <strong>
<em>99</em>
</strong> – Alsen, <strong>
<em>100</em>
</strong> – Alsen <em>ph1b</em>, <strong>
<em>101‐115</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Alsen; <strong>
<em>116</em>
</strong> – Argent, <strong>
<em>117</em>
</strong> – Argent <em>ph1b</em>, <strong>
<em>118‐132</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Argent; <strong>
<em>133</em>
</strong> – Divide, <strong>
<em>134</em>
</strong> – Divide <em>ph1b</em>, <strong>
<em>135‐149</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Divide; <strong>
<em>150</em>
</strong> – Line E, <strong>
<em>151</em>
</strong> – Line E <em>ph1b</em>, <strong>
<em>152‐165</em>
</strong> – BC<sub>3</sub>F<sub>2</sub> segregating population involving Line E. Arrows point to the PCR products of <em>Xwgc2111</em> (1,067bp) and <em>Xwgc2049</em> (846bp).</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12264458/bin/TPG2-18-e70065-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70065-s001.docx</a><sup> (811.5KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>There are no data associated with this manuscript.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70065"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70065.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12264458/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12264458/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12264458%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12264458/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12264458/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12264458/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40665645/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12264458/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40665645/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12264458/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12264458/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="IgVQ820jiA7TG41sLUOWbEAYbW1ch9Fzcqhz9vB5OW6rEVAGyHntAwrgyR95oJxK">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
