
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A review on multi-omics integration for aiding study design of large scale TCGA cancer datasets - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E483258AEEE723058325006AC47B2B.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcgeno">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372284/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Genomics">
<meta name="citation_title" content="A review on multi-omics integration for aiding study design of large scale TCGA cancer datasets">
<meta name="citation_author" content="Eonyong Han">
<meta name="citation_author_institution" content="School of Computer Science and Engineering, Kyungpook National University, Buk-gu, Daegu, 41566 Republic of Korea">
<meta name="citation_author" content="Hwijun Kwon">
<meta name="citation_author_institution" content="School of Computer Science and Engineering, Kyungpook National University, Buk-gu, Daegu, 41566 Republic of Korea">
<meta name="citation_author" content="Inuk Jung">
<meta name="citation_author_institution" content="School of Computer Science and Engineering, Kyungpook National University, Buk-gu, Daegu, 41566 Republic of Korea">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="769">
<meta name="citation_doi" content="10.1186/s12864-025-11925-y">
<meta name="citation_pmid" content="40847282">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372284/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372284/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372284/pdf/12864_2025_Article_11925.pdf">
<meta name="description" content="Rapid advancements in high-throughput sequencing technologies allow for detailed and accurate measurement of omics features within their biological context. The integration of different omics types creates heterogeneous datasets, presenting ...">
<meta name="og:title" content="A review on multi-omics integration for aiding study design of large scale TCGA cancer datasets">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Rapid advancements in high-throughput sequencing technologies allow for detailed and accurate measurement of omics features within their biological context. The integration of different omics types creates heterogeneous datasets, presenting ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372284/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12372284">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12864-025-11925-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12864_2025_Article_11925.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372284%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12372284/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12372284/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372284/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcgeno.png" alt="BMC Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Genomics" title="Link to BMC Genomics" shape="default" href="https://bmcgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Genomics</button></div>. 2025 Aug 22;26:769. doi: <a href="https://doi.org/10.1186/s12864-025-11925-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12864-025-11925-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A review on multi-omics integration for aiding study design of large scale TCGA cancer datasets</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Eonyong Han</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Eonyong Han</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Engineering, Kyungpook National University, Buk-gu, Daegu, 41566 Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eonyong Han</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwon%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Hwijun Kwon</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Hwijun Kwon</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Engineering, Kyungpook National University, Buk-gu, Daegu, 41566 Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kwon%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hwijun Kwon</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jung%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Inuk Jung</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Inuk Jung</span></h3>
<div class="p">
<sup>1</sup>School of Computer Science and Engineering, Kyungpook National University, Buk-gu, Daegu, 41566 Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jung%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Inuk Jung</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Computer Science and Engineering, Kyungpook National University, Buk-gu, Daegu, 41566 Republic of Korea </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 12; Accepted 2025 Jul 18; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12372284  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847282/" class="usa-link">40847282</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Rapid advancements in high-throughput sequencing technologies allow for detailed and accurate measurement of omics features within their biological context. The integration of different omics types creates heterogeneous datasets, presenting challenges in analysis due to variations in measurement units, sample numbers, and features. Currently, there is a lack of generalized guidelines for making decisions in multi-omics study design (MOSD), such as selecting an appropriate number of samples and features, type of preprocessing and integration for robust analysis results. We propose a suggestive guideline for MOSD, involving nine important factors: sample size, feature selection, preprocessing strategy, noise characterization, class balance, number of classes, cancer subtype combination, omics combination, and clinical features.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">To assess the effectiveness of our proposed MOSD guidelines, we designed and conducted seven benchmark tests using 10 clustering methods on various TCGA cancer datasets with an objective of clustering cancer subtypes. The results indicated robust performance in terms of cancer subtype discrimination when adhering to the following criteria: 26 or more samples per class, selecting less than 10% of omics features, maintaining a sample balance under a 3:1 ratio, and keeping the noise level below 30%. Feature selection was particularly important, improving clustering performance by 34%.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par3">These findings provide evidence-based recommendations for MOSD, enabling researchers to optimize analytical approaches and enhance the reliability of results across cancer datasets. The proposed MOSD framework offers a suggestive guideline addressing both computational and biological factors for multi-omics data integration.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12864-025-11925-y.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Multi-omics, Integration, Study design, Machine learning</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par4">Recent advances in high-throughput sequencing and omics technologies have enabled the generation of extensive multi-omics data, proving valuable in identifying causal relationships for deeper understanding of biological functions [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Multi-omics integration (MOI) involves the simultaneous analysis of genomics, transcriptomics, epigenomics, proteomics, and metabolomics and has substantially advanced investigations into the molecular underpinnings of complex diseases such as cancer [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>].</p>
<p id="Par5">Various MOI tools have been developed for specific combinations of omics data, including IGC [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>], PLRS [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>], Oncodrive-CIS [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>], MethylMix [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>], MMIA [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>], MethGET [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], and linkedOmics [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Tools, such as, MOGSA [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>], ActivePathways [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], multiGSEA [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], and iPanda [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>] have also emerged to facilitate the interpretation of complex multi-omics results. Large-scale multi-omics archives, such as, The Cancer Genome Atlas (TCGA) [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], International Cancer Genome Consortium (ICGC) [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>], Cancer Cell Line Encyclopedia (CCLE) [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>], and Clinical Proteomic Tumor Analysis Consortium (CPTAC) [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>] provide extensive multi-omics data with annotated clinical labels across different cancer types. However, leveraging this integration effectively poses numerous analytical and computational challenges. From a computational perspective, harmonizing distinct data types requires specialized approaches, as different omics data types can have varying distributions and sources of noise [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. For instance, transcript expression follows a binomial distribution, while CpG islands associated with methylation display a bimodal distribution [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Biologically, different omics layers may produce complementary but occasionally conflicting signals, as demonstrated in studies of colorectal carcinomas where methylation profiles were linked to genetic lineages defined by copy number alterations (CNAs), but transcriptional programs showed inconsistent connections to subclonal genetic identities [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>].</p>
<p id="Par6">Previous research has identified several critical challenges in multi-omics integration, including the curse of dimensionality, data heterogeneity, missing data, class imbalances, and scalability [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Various studies have investigated specific aspects of these challenges - from sample size and noise variation effects on clustering performance [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] to the importance of feature selection [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Although these studies offer valuable insights into components of multi-omics analysis, none provide a unified framework that systematically addresses both computational and biological complexities across a broad range of factors [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. The extension of MOI analysis to new multi-omics datasets with smaller size or different omics types presents additional challenges. Questions persist about optimal feature selection, such as determining the number of genes or omics features to use, and establishing minimum sample size requirements for qualitative analysis.</p>
<p id="Par7">To address these gaps, we propose a structured suggestive guideline for Multi-omics Study Design (MOSD). This guideline synthesizes and extends previous work to identify nine key factors across computational and biological domains. Through comprehensive benchmarking across multiple TCGA datasets using ten clustering-based analytical methods, we evaluate how these factors affect MOI analysis across 10 cancer types. Our goal is to provide clear, evidence-based recommendations for researchers engaged in multi-omics research, offering minimal guidelines for designing multi-omics experiments and performing downstream analysis.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Definition of MOSD factors</h3>
<p id="Par8">Through comprehensive literature review and systematic analysis, nine critical factors were identified that fundamentally influence multi-omics integration outcomes. This framework originated from Mirza et al. [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], who initially identified five key challenges in MOI analysis: the curse of dimensionality, data heterogeneity, missing data, class imbalances, and scalability. Building upon these foundations and incorporating insights from subsequent benchmark studies, these factors were organized into computational and biological aspects to provide a more complete analytical framework.</p>
<p id="Par9">The computational factors begin with sample size, which addresses both the curse of dimensionality and scalability challenges. Chauvel et al. [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] demonstrated through both simulation and TCGA breast cancer data how clustering performance changes according to sample numbers, evaluating two clustering methods using the adjusted rand index (ARI) across mRNA, miRNA, methylation and protein omics data from 348 samples. This finding was further supported by Tini et al. [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>], who evaluated five unsupervised algorithms using the F-measure method across multiple datasets including simulation data, murine liver data (BXD), and breast cancer gene expression data. Feature selection is a process where genes or omics features that are likely related to some biological trait of interest are carefully chosen by some algorithm, which is a standard procedure in most omics analysis. Pierre-Jean et al. [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] evaluated 13 unsupervised MOI methods for measuring the impact of feature selection. The study used three distinct simulated datasets (Gaussian, binary, and beta-like omics representations) to examine how the proportion of relevant variables impacts clustering performance. Tini et al. [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] also investigated feature selection’s impact, though their filtering method based on coefficient of variation showed limitations in determining the optimal number of meaningful features for class distinction. Preprocessing and noise characterization of multi-omics data represent vital computational factors. Duan et al. [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] demonstrated this in their benchmark of 10 MOI methods across nine TCGA cancer datasets, where they evaluated robustness by adding Gaussian noise at five different variance levels. Their work emphasized the importance of measuring both noise intensity and its distribution across the dataset. Sample balance across classes, initially identified by Mirza [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], was thoroughly examined by Rappoport et al. [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] in their benchmark of nine MOI methods across 10 TCGA datasets, where they evaluated clustering performance using both survival differences and clinical label correlations.</p>
<p id="Par10">The biological factors incorporate cancer subtype combinations and multi-omics layer integration. Rappoport et al. [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] evaluated this through two measures: scoring survival differences of divided clusters as log ranks and analyzing 41 clinical labels using chi-square and Kruskal–Wallis tests. Duan et al. [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] further explored this aspect by testing 11 different combinations from four omics types to identify optimal configurations for individual and overall cancer subtyping, while also evaluating clustering accuracy, clinical significance, and computational efficiency. The four omics types were gene expression (GE), miRNA (MI), methylation (ME) and copy number variation (CNV). The framework also emphasizes clinical feature correlation, incorporating molecular subtypes, gender, pathological stage, and age. This aspect was particularly highlighted in Pierre-Jean et al.’s work [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] with real datasets (BXD, liver cancer, obesity), though their study noted limitations in applying simulation-derived factors to real clinical data. The framework addresses several limitations identified in previous studies. While Chauvel et al. [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] only performed single tests on TCGA-BRCA data, and Rappoport et al. [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] couldn’t fully grasp the conditions needed to improve MOI method performance, this approach provides a more comprehensive evaluation. Similarly, it addresses the limitations noted in Tini et al.’s work [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] regarding noise application and feature selection, and expands upon Duan et al.’s analysis [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] by incorporating additional data conditions such as size and class balance.</p>
<p id="Par11">Based on the comprehensive review of previous studies (Table <a href="#Tab1" class="usa-link">1</a>), nine factors were identified and categorized into computational and biological factors. The computational factors include: (1) sample size, (2) feature selection, (3) preprocessing strategy, (4) noise characterization, (5) class balance and (6) number of classes. The biological factors comprise: (7) cancer subtype combinations, (8) omics combinations, and (9) clinical feature correlation. These factors provide a structured framework for evaluating and improving multi-omics data integration methods. By consolidating the efforts of previous research and addressing their limitations, we aimed to provide a more complete analytical approach that enhances both the precision and reproducibility of multi-omics analyses. The validation process developed specific benchmarks for each factor and applied them across various TCGA cancer datasets, allowing for comprehensive assessment of their impact on multi-omics integration outcomes.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Related study analysis</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Study</th>
<th align="left" colspan="6" rowspan="1">Computational factors</th>
<th align="left" colspan="3" rowspan="1">Biological factors</th>
<th align="left" rowspan="2" colspan="1">Datasets</th>
<th align="left" rowspan="2" colspan="1">Omics</th>
<th align="left" rowspan="2" colspan="1">Method</th>
<th align="left" rowspan="2" colspan="1">Estimation method</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Sample size</th>
<th align="left" colspan="1" rowspan="1">Feature selection</th>
<th align="left" colspan="1" rowspan="1">Preprocessing</th>
<th align="left" colspan="1" rowspan="1">Noise</th>
<th align="left" colspan="1" rowspan="1">Balance</th>
<th align="left" colspan="1" rowspan="1">Number of classes</th>
<th align="left" colspan="1" rowspan="1">Cancer subtype combination</th>
<th align="left" colspan="1" rowspan="1">Omics combination</th>
<th align="left" colspan="1" rowspan="1">Clinical feature</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="6" colspan="1">Chauvel et al. [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]</td>
<td align="left" rowspan="6" colspan="1">S</td>
<td align="left" rowspan="6" colspan="1">S</td>
<td align="left" rowspan="6" colspan="1">X</td>
<td align="left" rowspan="6" colspan="1">S</td>
<td align="left" rowspan="6" colspan="1">X</td>
<td align="left" rowspan="6" colspan="1">S</td>
<td align="left" rowspan="6" colspan="1">X</td>
<td align="left" rowspan="6" colspan="1">X</td>
<td align="left" rowspan="6" colspan="1">X</td>
<td align="left" rowspan="6" colspan="1">BRCA (348)</td>
<td align="left" colspan="1" rowspan="1">GE</td>
<td align="left" colspan="1" rowspan="1">BCC</td>
<td align="left" rowspan="6" colspan="1">ARI</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ME</td>
<td align="left" colspan="1" rowspan="1">MDI</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MI</td>
<td align="left" colspan="1" rowspan="1">iCluster</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Protein</td>
<td align="left" colspan="1" rowspan="1">moCluster</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">JIVE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">iNMF</td>
</tr>
<tr>
<td align="left" rowspan="10" colspan="1">Rappoport et al. [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" rowspan="10" colspan="1">O</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" rowspan="10" colspan="1">O</td>
<td align="left" colspan="1" rowspan="1">AML (170)</td>
<td align="left" colspan="1" rowspan="1">GE</td>
<td align="left" colspan="1" rowspan="1">k-means</td>
<td align="left" colspan="1" rowspan="1">Survival analysis (logrank)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BIC (621)</td>
<td align="left" colspan="1" rowspan="1">ME</td>
<td align="left" colspan="1" rowspan="1">Spectral</td>
<td align="left" colspan="1" rowspan="1">Clinical distribution</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">COAD (220)</td>
<td align="left" colspan="1" rowspan="1">MI</td>
<td align="left" colspan="1" rowspan="1">LRAcluster</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GBM (274)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">PINS</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KIRC (183)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">SNF</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LIHC (367)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">rMKL-LPP</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LUSC (341)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">MCCA</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SKCM (448)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">MultiNMF</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OV (287)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">iClusterBayes</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SARC (257)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">Tini et al. [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]</td>
<td align="left" rowspan="5" colspan="1">S</td>
<td align="left" rowspan="5" colspan="1">O</td>
<td align="left" rowspan="5" colspan="1">X</td>
<td align="left" rowspan="5" colspan="1">O</td>
<td align="left" rowspan="5" colspan="1">X</td>
<td align="left" rowspan="5" colspan="1">S</td>
<td align="left" rowspan="5" colspan="1">X</td>
<td align="left" rowspan="5" colspan="1">O</td>
<td align="left" rowspan="5" colspan="1">O</td>
<td align="left" colspan="1" rowspan="1">BXD (66)</td>
<td align="left" colspan="1" rowspan="1">GE</td>
<td align="left" colspan="1" rowspan="1">MCCA</td>
<td align="left" rowspan="5" colspan="1">F-measure</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Platelet (12)</td>
<td align="left" colspan="1" rowspan="1">ME</td>
<td align="left" colspan="1" rowspan="1">JIVE</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BRCA (491)</td>
<td align="left" colspan="1" rowspan="1">MI</td>
<td align="left" colspan="1" rowspan="1">MCIA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">MFA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">SNF</td>
</tr>
<tr>
<td align="left" rowspan="13" colspan="1">Pierre-Jean et al. [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]</td>
<td align="left" rowspan="13" colspan="1">S</td>
<td align="left" rowspan="13" colspan="1">S</td>
<td align="left" rowspan="13" colspan="1">S</td>
<td align="left" rowspan="13" colspan="1">O</td>
<td align="left" rowspan="13" colspan="1">S</td>
<td align="left" rowspan="13" colspan="1">S</td>
<td align="left" rowspan="13" colspan="1">X</td>
<td align="left" rowspan="13" colspan="1">O</td>
<td align="left" rowspan="13" colspan="1">O</td>
<td align="left" colspan="1" rowspan="1">BXD (64)</td>
<td align="left" colspan="1" rowspan="1">GE</td>
<td align="left" colspan="1" rowspan="1">RGCCA</td>
<td align="left" colspan="1" rowspan="1">F-measure</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Obesity (13)</td>
<td align="left" colspan="1" rowspan="1">ME</td>
<td align="left" colspan="1" rowspan="1">intNMF</td>
<td align="left" colspan="1" rowspan="1">ARI</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Liver cancer (360)</td>
<td align="left" colspan="1" rowspan="1">CNV</td>
<td align="left" colspan="1" rowspan="1">SNF</td>
<td align="left" colspan="1" rowspan="1">Survival analysis</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Mutation</td>
<td align="left" colspan="1" rowspan="1">LRACluster</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">PINSPlus</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">CC</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">MCIA</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">mixKernel</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">SGCCA</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">iClusterPlus</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">MoCluster</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">CIMLR</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">MOFA</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="10" colspan="1">Duan et al. [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" rowspan="10" colspan="1">O</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" rowspan="10" colspan="1">O</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" rowspan="10" colspan="1">O</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" rowspan="10" colspan="1">O</td>
<td align="left" rowspan="10" colspan="1">X</td>
<td align="left" colspan="1" rowspan="1">ACC (77)</td>
<td align="left" colspan="1" rowspan="1">GE</td>
<td align="left" colspan="1" rowspan="1">SNF</td>
<td align="left" colspan="1" rowspan="1">Precision</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BRCA (760)</td>
<td align="left" colspan="1" rowspan="1">ME</td>
<td align="left" colspan="1" rowspan="1">NEMO</td>
<td align="left" colspan="1" rowspan="1">NMI</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">COAD (291)</td>
<td align="left" colspan="1" rowspan="1">MI</td>
<td align="left" colspan="1" rowspan="1">CIMLR</td>
<td align="left" colspan="1" rowspan="1">ARI</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KIRP (273)</td>
<td align="left" colspan="1" rowspan="1">CNV</td>
<td align="left" colspan="1" rowspan="1">MultiNMF</td>
<td align="left" colspan="1" rowspan="1">F-measure</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">KIRC (314)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">PFA</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LIHC (364)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">LRAcluster</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LUAD (453)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">moCluster</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LUSC (363)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">iClusterBayes</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">THYM (119)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">PINS</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Subtype-GAN</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section id="Sec4"><h4 class="pmc_sec_title">Data acquisition and assembly</h4>
<p id="Par12">A comprehensive analysis of data sourced from TCGA [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>] was used to provide the suggestive MOSD guidelines. This study utilized multi-omics data from TCGA repository, focusing on 3,988 patients across ten cancer types: bladder urothelial carcinoma (BLCA, <em>n</em>=400), breast invasive carcinoma (BRCA, <em>n</em>=592), colon adenocarcinoma (COAD, <em>n</em>=249), head and neck squamous cell carcinoma (HNSC, <em>n</em>=471), kidney renal papillary cell carcinoma (KIRP, <em>n</em>=268), liver hepatocellular carcinoma (LIHC, <em>n</em>=357), lung adenocarcinoma (LUAD, <em>n</em>=442), skin cutaneous melanoma (SKCM, <em>n</em>=350), stomach adenocarcinoma (STAD, <em>n</em>=365), and thyroid carcinoma (THCA, <em>n</em>=494). These cancer types were selected based on the availability of patient-matched omics and clinical data, with a minimum threshold of 100 patients per cancer type to ensure robust statistical analysis. Detailed breakdown of subtype and sample counts is listed in Supplementary Table S1.</p>
<p id="Par13">As shown in Table <a href="#Tab2" class="usa-link">2</a>, the multi-omics layers incorporated in this study comprised gene expression (GE), miRNA (MI), mutation data, copy number variation (CNV), and methylation (ME), representing the commonly utilized data types in multi-omics research [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. The genomic features varied across cancer types, with GE features ranging from 2,097 to 2,229, while LIHC demonstrated the most extensive profile with 17,390 GE features and 21,933 CNV features. Notably, mutation data was particularly comprehensive for LIHC, encompassing 396,065 features. Figure <a href="#Fig1" class="usa-link">1</a> shows the structure of the multi-omics data, which were divided to omics and clinical feature data types for the analysis.</p>
<section class="tw xbox font-sm" id="Tab2"><h5 class="obj_head">Table 2.</h5>
<div class="caption p"><p>Statistics of the TCGA dataset</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Cancer</th>
<th align="left" colspan="4" rowspan="1">Number of omics features</th>
<th align="left" colspan="4" rowspan="1">Number of samples for each clinical feature</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">GE</th>
<th align="left" colspan="1" rowspan="1">ME</th>
<th align="left" colspan="1" rowspan="1">MI</th>
<th align="left" colspan="1" rowspan="1">CNV</th>
<th align="left" colspan="1" rowspan="1">Subtype</th>
<th align="left" colspan="1" rowspan="1">Pathologic-stage</th>
<th align="left" colspan="1" rowspan="1">Gender</th>
<th align="left" colspan="1" rowspan="1">Age</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="5" colspan="1">BLCA</td>
<td align="left" rowspan="52" colspan="1">17390</td>
<td align="left" rowspan="52" colspan="1">396065</td>
<td align="left" rowspan="5" colspan="1">2195</td>
<td align="left" rowspan="52" colspan="1">21933</td>
<td align="left" colspan="1" rowspan="1">1 (41)</td>
<td align="left" colspan="1" rowspan="1">StageII (128)</td>
<td align="left" colspan="1" rowspan="1">MALE (296)</td>
<td align="left" colspan="1" rowspan="1">Middle (124)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2 (41)</td>
<td align="left" colspan="1" rowspan="1">StageIII (136)</td>
<td align="left" colspan="1" rowspan="1">FEMALE (104)</td>
<td align="left" colspan="1" rowspan="1">Old (244)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3 (29)</td>
<td align="left" colspan="1" rowspan="1">StageIV (132)</td>
<td align="left" colspan="1" rowspan="1">Total (400)</td>
<td align="left" colspan="1" rowspan="1">Total (368)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4 (15)</td>
<td align="left" colspan="1" rowspan="1">Total (396)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total (126)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">BRCA</td>
<td align="left" rowspan="5" colspan="1">2222</td>
<td align="left" colspan="1" rowspan="1">LumA (323)</td>
<td align="left" colspan="1" rowspan="1">StageI (96)</td>
<td align="left" rowspan="5" colspan="1">-</td>
<td align="left" colspan="1" rowspan="1">Young (16)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LumB (118)</td>
<td align="left" colspan="1" rowspan="1">StageII (345)</td>
<td align="left" colspan="1" rowspan="1">Middle (221)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Basal (102)</td>
<td align="left" colspan="1" rowspan="1">StageIII (151)</td>
<td align="left" colspan="1" rowspan="1">Old (81)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Her2 (39)</td>
<td align="left" colspan="1" rowspan="1">Total (592)</td>
<td align="left" colspan="1" rowspan="1">Total (318)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total (582)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">COAD</td>
<td align="left" rowspan="5" colspan="1">2097</td>
<td align="left" colspan="1" rowspan="1">CMS1 (34)</td>
<td align="left" colspan="1" rowspan="1">StageI (36)</td>
<td align="left" colspan="1" rowspan="1">MALE (137)</td>
<td align="left" colspan="1" rowspan="1">Middle (99)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CMS2 (67)</td>
<td align="left" colspan="1" rowspan="1">StageII (83)</td>
<td align="left" colspan="1" rowspan="1">FEMALE (112)</td>
<td align="left" colspan="1" rowspan="1">Old (130)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CMS3 (35)</td>
<td align="left" colspan="1" rowspan="1">StageIII (67)</td>
<td align="left" colspan="1" rowspan="1">Total (249)</td>
<td align="left" colspan="1" rowspan="1">Total (229)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CMS4 (63)</td>
<td align="left" colspan="1" rowspan="1">StageIV (29)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total (199)</td>
<td align="left" colspan="1" rowspan="1">Total (215)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">HNSC</td>
<td align="left" rowspan="5" colspan="1">2229</td>
<td align="left" colspan="1" rowspan="1">Basal (74)</td>
<td align="left" colspan="1" rowspan="1">StageI (26)</td>
<td align="left" colspan="1" rowspan="1">MALE (347)</td>
<td align="left" colspan="1" rowspan="1">Middle (261)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mesenchymal (65)</td>
<td align="left" colspan="1" rowspan="1">StageII (71)</td>
<td align="left" colspan="1" rowspan="1">FEMALE (124)</td>
<td align="left" colspan="1" rowspan="1">Old (168)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Atypical (59)</td>
<td align="left" colspan="1" rowspan="1">StageIII (74)</td>
<td align="left" colspan="1" rowspan="1">Total (471)</td>
<td align="left" colspan="1" rowspan="1">Total (429)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Classical (44)</td>
<td align="left" colspan="1" rowspan="1">StageIV (234)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total (242)</td>
<td align="left" colspan="1" rowspan="1">Total (405)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">KIRP</td>
<td align="left" rowspan="5" colspan="1">2099</td>
<td align="left" colspan="1" rowspan="1">C1 (84)</td>
<td align="left" colspan="1" rowspan="1">StageI (164)</td>
<td align="left" colspan="1" rowspan="1">MALE (200)</td>
<td align="left" colspan="1" rowspan="1">Middle (141)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C2a (34)</td>
<td align="left" colspan="1" rowspan="1">StageII (18)</td>
<td align="left" colspan="1" rowspan="1">FEMALE (68)</td>
<td align="left" colspan="1" rowspan="1">Old (91)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C2b (21)</td>
<td align="left" colspan="1" rowspan="1">StageIII (49)</td>
<td align="left" colspan="1" rowspan="1">Total (268)</td>
<td align="left" colspan="1" rowspan="1">Total (232)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total (139)</td>
<td align="left" colspan="1" rowspan="1">StageIV (14)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Total (245)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1">LIHC</td>
<td align="left" rowspan="4" colspan="1">2157</td>
<td align="left" colspan="1" rowspan="1">1 (65)</td>
<td align="left" colspan="1" rowspan="1">StageI (166)</td>
<td align="left" colspan="1" rowspan="1">MALE (243)</td>
<td align="left" colspan="1" rowspan="1">Middle (171)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2 (55)</td>
<td align="left" colspan="1" rowspan="1">StageII (84)</td>
<td align="left" colspan="1" rowspan="1">FEMALE (114)</td>
<td align="left" colspan="1" rowspan="1">Old (129)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3 (63)</td>
<td align="left" colspan="1" rowspan="1">StageIII (61)</td>
<td align="left" colspan="1" rowspan="1">Total (357)</td>
<td align="left" colspan="1" rowspan="1">Total (300)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total (183)</td>
<td align="left" colspan="1" rowspan="1">Total (311)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="7" colspan="1">LUAD</td>
<td align="left" rowspan="7" colspan="1">2211</td>
<td align="left" colspan="1" rowspan="1">1 (17)</td>
<td align="left" colspan="1" rowspan="1">StageI (233)</td>
<td align="left" colspan="1" rowspan="1">MALE (205)</td>
<td align="left" colspan="1" rowspan="1">Middle (174)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2 (22)</td>
<td align="left" colspan="1" rowspan="1">StageII (108)</td>
<td align="left" colspan="1" rowspan="1">FEMALE (237)</td>
<td align="left" colspan="1" rowspan="1">Old (212)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3 (38)</td>
<td align="left" colspan="1" rowspan="1">StageIII (62)</td>
<td align="left" colspan="1" rowspan="1">Total (442)</td>
<td align="left" colspan="1" rowspan="1">Total (386)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4 (26)</td>
<td align="left" colspan="1" rowspan="1">StageIV (19)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5 (43)</td>
<td align="left" colspan="1" rowspan="1">Total (422)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6 (33)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total (179)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">SKCM</td>
<td align="left" rowspan="5" colspan="1">2203</td>
<td align="left" colspan="1" rowspan="1">BRAF (116)</td>
<td align="left" colspan="1" rowspan="1">StageI (72)</td>
<td align="left" colspan="1" rowspan="1">MALE (220)</td>
<td align="left" colspan="1" rowspan="1">Middle (175)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RAS (78)</td>
<td align="left" colspan="1" rowspan="1">StageII (67)</td>
<td align="left" colspan="1" rowspan="1">FEMALE (130)</td>
<td align="left" colspan="1" rowspan="1">Old (103)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TWT (37)</td>
<td align="left" colspan="1" rowspan="1">StageIII (137)</td>
<td align="left" colspan="1" rowspan="1">Total (350)</td>
<td align="left" colspan="1" rowspan="1">Young (29)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NF1 (23)</td>
<td align="left" colspan="1" rowspan="1">StageIV (20)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Total (307)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total (254)</td>
<td align="left" colspan="1" rowspan="1">Total (296)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">STAD</td>
<td align="left" rowspan="5" colspan="1">2161</td>
<td align="left" colspan="1" rowspan="1">CIN (202)</td>
<td align="left" colspan="1" rowspan="1">StageI (46)</td>
<td align="left" colspan="1" rowspan="1">MALE (240)</td>
<td align="left" colspan="1" rowspan="1">Old (177)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MSI (59)</td>
<td align="left" colspan="1" rowspan="1">StageII (116)</td>
<td align="left" colspan="1" rowspan="1">FEMALE (125)</td>
<td align="left" colspan="1" rowspan="1">Middle (133)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GS (46)</td>
<td align="left" colspan="1" rowspan="1">StageIII (163)</td>
<td align="left" colspan="1" rowspan="1">Total (365)</td>
<td align="left" colspan="1" rowspan="1">Total (310)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EBV (29)</td>
<td align="left" colspan="1" rowspan="1">StageIV (28)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total (336)</td>
<td align="left" colspan="1" rowspan="1">Total (353)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="6" colspan="1">THCA</td>
<td align="left" rowspan="6" colspan="1">2201</td>
<td align="left" colspan="1" rowspan="1">1 (131)</td>
<td align="left" colspan="1" rowspan="1">StageI (279)</td>
<td align="left" colspan="1" rowspan="1">MALE (133)</td>
<td align="left" colspan="1" rowspan="1">Middle (237)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2 (67)</td>
<td align="left" colspan="1" rowspan="1">StageII (50)</td>
<td align="left" colspan="1" rowspan="1">FEMALE (361)</td>
<td align="left" colspan="1" rowspan="1">Young (166)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3 (82)</td>
<td align="left" colspan="1" rowspan="1">StageIII (109)</td>
<td align="left" colspan="1" rowspan="1">Total (494)</td>
<td align="left" colspan="1" rowspan="1">Old (45)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4 (105)</td>
<td align="left" colspan="1" rowspan="1">StageIV (46)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Total (448)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5 (90)</td>
<td align="left" colspan="1" rowspan="1">Total (484)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total (475)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig1"><h5 class="obj_head">Fig. 1.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372284_12864_2025_11925_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/ed017d559f6f/12864_2025_11925_Fig1_HTML.jpg" loading="lazy" id="MO1" height="342" width="708" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structure and organization of multi-omics data integration. Multi-omics datasets are structured as a collection of two-dimensional matrices, where each matrix represents a distinct omics layer (e.g., genomics, transcriptomics, proteomics). Rows consistently represent biological samples across all matrices, while columns contain features specific to each omics type (e.g., genes, proteins, methylation probes). A complementary clinical matrix maintains the same sample-wise organization (rows) but contains relevant clinical variables (columns) such as age, sex, and disease status. This standardized structure enables integrated analysis across multiple biological layers while maintaining sample correspondence</p></figcaption></figure><p id="Par15">Clinical annotations were systematically integrated, including molecular subtypes, gender distribution, pathological staging, and age demographics. The molecular subtypes for each cancer type were extracted from TCGA clinical data and varied by cancer type, with some cancers showing distinct classification system. For instance, for BRCA, we applied PAM50-based subtypes [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>] (Luminal A, Luminal B, Basal, HER2), and for COAD, Consensus Molecular Subtype 1-4 (CMS1–4) were classified based on Guinney et al. [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. For other cancer types, including LUAD, STAD, SKCM, KIRP, and HNSC, we utilized the ‘Subtype’ column provided in the TCGA clinical matrices, which included both categorical descriptors and numeric classifications (ranging 1-6 depending on cancer type). These subtype labels were used as-is for benchmarking purposes without reclassification or biological reinterpretation. Age stratification into Young, Middle, and Old categories demonstrated a predominance of middle-aged and elderly patients, with specific age-related patterns varying by cancer type. The University of California Santa Cruz (UCSC) Xena browser [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] was used for collecting the multi-omics data. Patient barcodes were verified through TCGAbiolinks [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] to ensure data matching.</p></section><section id="Sec5"><h4 class="pmc_sec_title">Data preprocessing</h4>
<p id="Par16">The examination of preprocessing effects on integration performance involved the creation of three distinct dataset types, each representing different levels of data refinement. The first category comprises Raw Datasets (RAW), which maintain data in its most basic processed form through log2 transformation of the original values. This approach preserves the inherent structure of the data while enabling basic computational analysis. The second category consists of Normalized Datasets (NORM), which underwent quantile normalization followed by 0-1 scaling. This process establishes uniform distributions across various omics layers, facilitating more standardized comparisons.</p>
<p id="Par17">The third category encompasses Gene Level Datasets (GL), representing an advanced preprocessing approach where normalized data aligns with gene expression characteristics. To enable integrative analysis across heterogeneous omics layers, we converted all omics data into a gene-level format, following a gene-centric integration strategy as described in Jung et al. [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] For methylation, we selected probes located within a 2 kilobase upstream region from the promoter region of each gene. The average beta value across these probes was computed per gene. miRNA expression values were assigned to their corresponding target genes based on experimentally validated miRNA–gene interactions, and the geometric mean of all miRNAs targeting a gene was used as the representative value. For CNV, segmented values overlapping gene bodies were averaged per gene. After alignment across omics layers, each matrix was min–max scaled to a 0–1 range to ensure comparability and avoid dominance from any single data type during integration. This process ensures that all omics matrices share the same gene and sample dimensions, enabling structured gene-level multi-omics representation. The complete preprocessing methodology and associated code reside in the project’s GitHub repository, ensuring transparency and reproducibility.</p>
<p id="Par18">The preprocessing strategy specifically addressed the challenge of missing data, which previous research has identified as detrimental to multi-omics analyses. To ensure consistency across omics layers and avoid the influence of missing values, we adopted a conservative approach by strictly excluding any omics features that contained missing entries. In addition, sample groups with fewer than ten individuals were removed to maintain sufficient representation for statistical analysis. No imputation was applied during preprocessing Such measures maintain robust representation across categories while minimizing potential biases stemming from data sparsity.</p>
<p id="Par19">The processing of clinical data involved careful categorization to enable meaningful correlation with multi-omics findings. The framework retained categorical features including molecular subtype, gender, and pathological stage as natural labels for subsequent analyses. Age data underwent transformation from its continuous form into three discrete categories: young (0-39 years), middle aged (40-64 years), and older (65 years and above), following protocols in cancer research literature [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. This age categorization facilitates the investigation of age specific molecular patterns within each cancer type [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>].</p></section></section><section id="Sec6"><h3 class="pmc_sec_title">MOI analysis methods and implementation</h3>
<p id="Par20">We evaluated ten MOI methods that employ various computational approaches including clustering, matrix factorization, and network integration across ten TCGA cancer datasets to determine optimal conditions for cancer data analysis. These methods were selected based on three criteria: ease of implementation in R or Python, prevalence in academic research, and effectiveness with default parameters. Rather than focusing solely on comparison of method performance, our analysis examined how each method’s characteristics interacted with the nine factors through seven benchmark tests. Through a series of controlled benchmark tests, we identified parameter conditions that consistently improved analysis quality under each method, which we refer to as analytically favorable or recommended setups for multi-omics study.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Similarity network fusion</h3>
<p id="Par21">Similarity Network Fusion (SNF) [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] constructs similarity networks for each omics type using metrics such as Pearson correlation or chi-square and then fuses these networks into a unified representation. Its advantage in MOSD lies in integrating disparate omics layers, handling clinical features of both discrete and continuous types, and tolerating small sample sizes.</p>
<section id="Sec8"><h4 class="pmc_sec_title">Spectrum</h4>
<p id="Par22">Spectrum [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] is a density-aware spectral clustering method that leverages eigenvectors from the multi-omics sample-graph matrix to mitigate noise and illuminate biologically relevant structures. It uses a k-nearest neighbors kernel, obviating the need for extensive parameter tuning across varied datasets. Spectrum’s advantages for MOSD include scalability to large datasets and the ability to capture subtle density variations, aided by a Gaussian Mixture Model (GMM) rather than traditional k-means.</p></section><section id="Sec9"><h4 class="pmc_sec_title">PINSPlus</h4>
<p id="Par23">PINSPlus (PP) [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>], an extension of PINS [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>], integrates multi-omics data by building connectivity graphs for each omics layer and fusing them to identify stable clusters. PP showed to be able to find both known and unknown subtypes with significant clinical implications.</p></section></section><section id="Sec10"><h3 class="pmc_sec_title">Neighborhood-based multi-omics clustering</h3>
<p id="Par24">Neighborhood-based Multi-omics Clustering (NEMO) [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] is a flexible algorithm that operates through three main steps: 1) calculation of pairwise distances between samples within each omics layer using specified distance metrics, 2) construction of k-nearest neighbor graphs for each sample based on local neighborhood patterns, and 3) clustering of samples using graph-based clustering algorithms applied to these neighborhood representations. NEMO supports various feature types including numerical, discrete, ordinal, and imaging data, and excels with partial datasets by eliminating the need for missing data imputation while maintaining faster execution compared to other multi-omics clustering methods.</p>
<section id="Sec11"><h4 class="pmc_sec_title">Cluster of cluster assignments</h4>
<p id="Par25">Cluster of Cluster Assignments (COCA) [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] converts each omics dataset into a one-hot encoded cluster matrix and then applies consensus clustering, typically using Pearson’s correlation to form a similarity matrix. The consensus clustering provides stability and helps identify cross-omics correlations without strict normalization requirements.</p></section><section id="Sec12"><h4 class="pmc_sec_title">Low-rank approximation clustering</h4>
<p id="Par26">Low-Rank Approximation Clustering (LRAcluster) [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] adopts a probabilistic model that treats omics data as Gaussian-distributed variables under a low-rank constraint. It compresses high-dimensional multi-omics data into a more interpretable, low-dimensional representation. LRAcluster accurately detected shared covariance structures across omics types and highlighted critical regulatory signatures and cancer subtypes.</p></section><section id="Sec13"><h4 class="pmc_sec_title">Consensus clustering</h4>
<p id="Par27">Consensus Clustering (CC) [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>] repeatedly applies clustering algorithms to resampled versions of the dataset and aggregates these results into a consensus solution, mitigating random fluctuations. This is particularly valuable in MOSD when handling high-dimensional and heterogeneous data. By quantifying cluster assignment frequencies, CC not only consolidates robust groupings but also reveals the stability of potential cluster solutions across multiple resampling iterations.</p></section><section id="Sec14"><h4 class="pmc_sec_title">Multi-omics factor analysis</h4>
<p id="Par28">Multi-Omics Factor Analysis (MOFA) [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] decomposes multi-omics data into latent factors that capture the main source of variation across different layers. By fitting a linear factor model, it effectively integrates incomplete data, highlights outliers, and identifies molecular signatures driving heterogeneity. MOFA’s interpretability and capacity for imputing missing values offer considerable benefits within MOSD, allowing deeper insight into latent structures that span multiple data types.</p></section><section id="Sec15"><h4 class="pmc_sec_title">iClusterPlus</h4>
<p id="Par29">iClusterPlus [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] is an advanced version of iCluster [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>] that integrates multi-omics data using a probabilistic joint latent-variable model. The method incorporates different omics data types (binary, categorical, and continuous) into a low-dimensional latent representation through estimation of latent variables W and H that capture relationships between samples and features. Model optimization employs a lasso penalty (L1-norm) on H to obtain sparse solutions, enabling dimension reduction by removing uninformative features. This approach improves clustering performance while increasing interpretability and computational efficiency through comprehensive multi-omics integration.</p></section><section id="Sec16"><h4 class="pmc_sec_title">Integrative non-negative matrix factorization</h4>
<p id="Par30">Integrative Non-negative Matrix Factorization (IntNMF) [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] extends classic NMF [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>] by sharing a common basis matrix W across multiple two-dimensional omics datasets, while allowing each data type to have a distinct coefficient matrix H. It enhances the detection of subtypes and leverages resampling-based cross-validation for selecting the optimal number of clusters. IntNMF’s strength lies in its ability to handle large-scale datasets without relying on strict distributional assumptions. Collectively, the ten methods offer a wide range of strategies for multi-omics integration and clustering, each with unique advantages for different data structures and MOSD factors. By systematically benchmarking their performance across TCGA cancer datasets, we aimed to pinpoint favorable conditions, such as sample size requirements, handling of missing values, or optimal omics combinations, that maximize analytical precision and biological relevance under the MOSD framework.</p></section><section id="Sec17"><h4 class="pmc_sec_title">Benchmark design</h4>
<p id="Par31">To address the heterogeneous nature of multi-omics data and evaluate parameter influence on analysis outcomes, a series of comprehensive benchmarks aligned with the previously defined computational and biological factors in MOSD has been developed. Each benchmark test isolated a single study design factor while maintaining all other variables at fixed default settings. This allowed fair comparison of individual factor impacts on clustering performance. The full test design, including parameter ranges, evaluation objectives, target values, and default settings for each benchmark, is summarized in Table S2. This systematic isolation of variables allowed for precise measurement of each factor’s unique contribution to analysis outcomes. The evaluation process employed multiple performance metrics, including Normalized Mutual Information (NMI), Adjusted Rand Index (ARI), and the F-measure. To ensure statistical robustness, each benchmark underwent ten iterations with random sample selection.</p></section><section id="Sec18"><h4 class="pmc_sec_title">Determining the minimal required sample size</h4>
<p id="Par32">A key objective in MOSD is to identify the minimal sample size required for reliable multi-omics clustering. In this test, we randomly selected subsets of increasing size beginning with as few as five samples to track when performance metrics plateaued. As illustrated in Fig. <a href="#Fig2" class="usa-link">2</a>, this plateau reflects the threshold beyond which additional samples provide diminishing returns for accurate cancer subtyping, thus indicating the minimum cohort size needed for robust analysis.</p>
<figure class="fig xbox font-sm" id="Fig2"><h5 class="obj_head">Fig. 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372284_12864_2025_11925_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/8999d1352556/12864_2025_11925_Fig2_HTML.jpg" loading="lazy" id="MO2" height="813" width="778" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Benchmarking designs of key factors. Systematic evaluation of six critical factors in multi-omics integration: <strong>a</strong> Sample size - Performance metrics with increasing sample sizes showing minimum cohort requirements. <strong>b</strong> Feature Selection - Impact of varying feature numbers selected via statistical testing across omics types. <strong>c</strong> Preprocessing test - comparison of RAW (Log2 transformed), NORM (Quantile normalization with 0-1 scaling), and GL (Gene level dataset) approaches on clustering performance. <strong>d</strong> Class balance - effects of classes with imbalanced sample on method accuracy. <strong>e</strong> Noise tolerance - Method resilience to incremental Gaussian noise (10-100%). <strong>f</strong> Cancer subtype combination - Performance analysis across different cancer subtype combinations. Each panel demonstrates the parameter’s influence on MOI performance</p></figcaption></figure></section><section id="Sec19"><h4 class="pmc_sec_title">Optimizing feature selection</h4>
<p id="Par33">Determining and adequate balance between feature quantity and quality is important in MOSD. To assess the effect of feature quantity on MOI performance, we incrementally selected features based on their relevance to sample group labels (i.e., cancer subtypes). Chi-square tests were applied for GE, MI, and ME omics types, while ANOVA was applied to CNV data due to its inclusion of negative values. Features were ranked by <em>p</em>-value, and top-ranked subsets were selected at increasing proportions—1% steps for early iterations, then 10%—up to 100% of features. This strategy was chosen over variance-based or classifier-specific ranking to avoid bias from model dependencies and to provide a group-informed yet algorithm-agnostic selection method. The exact number of features per omics layer in each benchmark iteration is provided in the GitHub repository. The proportion of selected features in each scenario is shown in Fig. <a href="#Fig2" class="usa-link">2</a>.</p></section><section id="Sec20"><h4 class="pmc_sec_title">Measuring the impact of different preprocessing methods</h4>
<p id="Par34">Preprocessing can significantly affect the outcomes of multi-omics analysis. We compared three approaches: RAW, NORM, and GL. RAW data underwent only log2 transformation, while NORM underwent quantile normalization and min-max scaling. GL integration averaged omics features at the gene level, as recommended by Jung et al. [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>] potentially enhancing gene-centric signals yet discarding non-gene-aligned features. By applying these variants to the same sets of MOI methods, we examined whether specific preprocessing strategies yield consistently better clustering performance (Fig. <a href="#Fig2" class="usa-link">2</a>).</p></section></section><section id="Sec21"><h3 class="pmc_sec_title">Evaluating class balance</h3>
<p id="Par35">Class imbalance, where certain cancer subtypes or clinical categories contain fewer samples, can bias clustering and reduce the detection of smaller classes [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. In this benchmark, we progressively introduced imbalance in sample sizes to gauge how well different MOI methods cope with skewed distributions (Fig. <a href="#Fig2" class="usa-link">2</a>).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Assessing noise tolerance</h3>
<p id="Par36">Multi-omics datasets often include noise, or errors, from technical and biological reasons [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. To evaluate its influence, Gaussian noise was introduced in 10% increments up to 100% in each omics layer (Fig. <a href="#Fig2" class="usa-link">2</a>). The objective was to observe the maximum tolerable noise threshold where clustering performance started declining significantly.</p>
<section id="Sec23"><h4 class="pmc_sec_title">Cancer subtype combinations</h4>
<p id="Par37">Different subtypes within the same cancer type frequently display unique molecular signatures. We explored whether varying combinations of these subtypes influence clustering performance, focusing on how distinguishable they are (Fig. <a href="#Fig2" class="usa-link">2</a>). All clustering evaluations were performed on datasets containing at least two distinct subtype labels, ensuring meaningful calculation of performance metrics. The benchmark also examined the number of subtypes assessed—ranging from a low count (2–3) to more diverse sets—to identify at which point distinguishing multiple subtypes becomes challenging or more susceptible to errors.</p></section><section id="Sec24"><h4 class="pmc_sec_title">Omics combination</h4>
<p id="Par38">Finally, we investigated how specific omics-layer combinations affect clustering for four clinical features (molecular subtype, gender, pathological stage, age). Data configurations ranged from single-omics (GE alone) to multi-omics (e.g., GE-ME-CNV or GE-ME-MI-CNV). Although combining more omics can reveal intricate biological insights, it can also amplify heterogeneity, aligning with findings by Duan et al. [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] and Reel et al. [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Hence, determining which combinations yield the most accurate results for each clinical feature is a key focus in MOSD, and our test delineates situations where additional omics layers either enhance or degrade performance. Through these targeted benchmarks, we provide a systematic exploration of how each factor contributed to multi-omics analysis results. This approach offers actionable insights into the design, execution, and interpretation of multi-omics studies.</p></section><section id="Sec25"><h4 class="pmc_sec_title">Performance evaluation for MOI methods</h4>
<p id="Par39">Evaluating the performance of clustering-based MOI methods is an essential component of the MOSD framework. Three metrics were used to evaluate the clustering performance, ARI, NMI, and F-measure [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. ARI is calculated from the Rand Index (RI), which measures clustering accuracy:</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/a1b257324be8/d33e1622.gif" loading="lazy" id="d33e1622" alt="graphic file with name d33e1622.gif"></td></tr></table>
<p id="Par40">Here, TP represents the sample pairs correctly grouped within the same cluster, while TN represents the pairs correctly grouped in different clusters. FP indicates the pairs incorrectly grouped in the same cluster, and FN represents pairs incorrectly grouped in different clusters. ARI adjusts the RI by accounting for chance:</p>
<table class="disp-formula p" id="Equ2"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/a1af79e6111f/d33e1629.gif" loading="lazy" id="d33e1629" alt="graphic file with name d33e1629.gif"></td></tr></table>
<p id="Par41">The Expected(RI) represents the expected RI value by random chance, while Max(RI) indicates the maximum possible RI value. NMI measures the mutual information between the true labels (T) and clustering results (C). The probability distributions are defined as:</p>
<table class="disp-formula p" id="Equ3"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/24fc58f3dbbc/d33e1636.gif" loading="lazy" id="d33e1636" alt="graphic file with name d33e1636.gif"></td></tr></table>
<table class="disp-formula p" id="Equ4"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/fda310a8b9d9/d33e1641.gif" loading="lazy" id="d33e1641" alt="graphic file with name d33e1641.gif"></td></tr></table>
<table class="disp-formula p" id="Equ5"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/95c0f5b6ac3b/d33e1646.gif" loading="lazy" id="d33e1646" alt="graphic file with name d33e1646.gif"></td></tr></table>
<p id="Par42">Here, N represents the total number of elements, while <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/1402db59aca4/d33e1653.gif" loading="lazy" id="d33e1653" alt="Inline graphic"></span> indicates the number of elements in the true class i, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/5c1058b6b439/d33e1659.gif" loading="lazy" id="d33e1659" alt="Inline graphic"></span> represents the number of elements in the predicted cluster j. The intersection <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/eba10a68525e/d33e1665.gif" loading="lazy" id="d33e1665" alt="Inline graphic"></span> denotes the number of elements shared between the true class and predicted cluster. The mutual information is then calculated as:</p>
<table class="disp-formula p" id="Equ6"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/57f33fed5613/d33e1671.gif" loading="lazy" id="d33e1671" alt="graphic file with name d33e1671.gif"></td></tr></table>
<p id="Par43">To normalize MI, we calculate the entropy of both true labels and clustering results. Entropy measures the uncertainty or randomness in the clustering assignments. The entropy for true labels and clustering results are calculated as:</p>
<table class="disp-formula p" id="Equ7"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/3c83c11c3dea/d33e1678.gif" loading="lazy" id="d33e1678" alt="graphic file with name d33e1678.gif"></td></tr></table>
<table class="disp-formula p" id="Equ8"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/60f66dc798b8/d33e1683.gif" loading="lazy" id="d33e1683" alt="graphic file with name d33e1683.gif"></td></tr></table>
<p>where <em>P</em>(<em>i</em>) is the probability of an element belonging to the true class <em>i</em>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/1583dcc6d738/d33e1698.gif" loading="lazy" id="d33e1698" alt="Inline graphic"></span> is the probability of an element being assigned to cluster <em>j</em>. Using these entropy values, we normalized MI to obtain NMI:</p>
<table class="disp-formula p" id="Equ9"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/32d04be26aaf/d33e1708.gif" loading="lazy" id="d33e1708" alt="graphic file with name d33e1708.gif"></td></tr></table>
<p id="Par44">This ensures the NMI value falls between 0 and 1, where 1 indicates perfect clustering alignment with true labels, and 0 indicates completely random clustering. The average of the entropies in the denominator acts as a normalization factor, making NMI more comparable across different clustering scenarios. The F-measure computation involves three set operations:</p>
<table class="disp-formula p" id="Equ10"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/0bcb04eb75f4/d33e1715.gif" loading="lazy" id="d33e1715" alt="graphic file with name d33e1715.gif"></td></tr></table>
<table class="disp-formula p" id="Equ11"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/45b026dc76d6/d33e1720.gif" loading="lazy" id="d33e1720" alt="graphic file with name d33e1720.gif"></td></tr></table>
<table class="disp-formula p" id="Equ12"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/dad3ff614dff/d33e1725.gif" loading="lazy" id="d33e1725" alt="graphic file with name d33e1725.gif"></td></tr></table>
<p id="Par45">The sets U and V represent the predicted clusters and true clusters respectively, where Pairs(X) generates all possible pairs of elements within set X. The F-measure is then calculated as:</p>
<table class="disp-formula p" id="Equ13"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/397c55752f3a/d33e1732.gif" loading="lazy" id="d33e1732" alt="graphic file with name d33e1732.gif"></td></tr></table>
<p id="Par46">To obtain an overall performance metric, we compute a cluster-score as the arithmetic mean of all three metrics:</p>
<table class="disp-formula p" id="Equ14"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/69b7441e07a3/d33e1739.gif" loading="lazy" id="d33e1739" alt="graphic file with name d33e1739.gif"></td></tr></table>
<p id="Par47">Although ARI theoretically ranges from –1 to 1, all observed values in our benchmark were non-negative, eliminating the need for normalization before averaging with NMI and F-measure. This combined cluster-score serves as our primary evaluation metric, providing balanced assessment of clustering performance across all benchmark tests. Individual metrics are examined when detailed analysis is required. We note that future studies encountering negative ARI values should consider normalization to a 0–1 scale to ensure balanced integration into composite scoring.</p></section></section></section><section id="Sec26"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec27"><h3 class="pmc_sec_title">Optimal and suboptimal configurations in benchmark tests</h3>
<p id="Par48">Identifying the most and least effective parameter configurations were searched to characterize the importance of each of the nine key factors. Regarding sample size and class balance, smaller but balanced classes often performed better than larger but imbalanced ones, as exemplified in BLCA where 11 samples per class achieved the highest cluster-score (0.55), while very small classes (i.e., 5 samples) showed poor performance. Feature selection demonstrated that using fewer features generally yielded better results than using all features, which consistently produced the lowest scores across cancer types. The class balance benchmark showed varying optimal configurations across cancer types. Some cancers performed best with equal class sizes (e.g., BRCA with 40 samples per class), while others showed resilience to moderate imbalance. The noise tolerance test revealed that increasing noise levels universally degraded performance, with 100% noise consistently producing the lowest scores (often below 0.10) across all cancer types.</p>
<p id="Par49">For biological factors, certain subtype combinations demonstrated notably stronger signals. For instance, Luminal B-Basal subtypes in BRCA and CIN-EBV in STAD showed particularly high discrimination (score &gt;0.90). For omics combinations, GE-ME pairings frequently emerged as the most effective omics pairs across multiple cancer types, while combinations involving CNV often showed poorer performance. Notably, some tests, such as gender analysis in BRCA, were excluded due to extreme data imbalance, highlighting the importance of considering data distribution in study design.</p>
<p id="Par50">These findings across different parameter configurations provide insights for optimizing multi-omics analysis approaches and aid in parameter selection. The complete detailed results are available in Supplementary Tables S3 and S4.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Impact of sample size</h3>
<p id="Par51">The relationship between sample size and clustering performance was examined across all cancer types (Fig. <a href="#Fig3" class="usa-link">3</a>a). The analysis revealed that increasing sample size from 5 to 38 samples per class had a subtle but positive effect on average cluster scores, as indicated by the gradually rising trend line (red). However, the most notable impact of larger sample sizes was on score consistency rather than absolute performance - the violin plot distributions become narrower as sample size increases, particularly after 26 samples, indicating reduced variance in clustering performance. While the average cluster score shows only modest improvement (i.e., ranging approximately from 0.5 to 0.55), the minimum scores become notably higher with increased sample size, suggesting that larger sample sizes help prevent poor clustering outcomes. Detailed cancer-type specific trends are provided in Supplementary Figure S1.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372284_12864_2025_11925_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/e92614a8daa8/12864_2025_11925_Fig3_HTML.jpg" loading="lazy" id="MO3" height="409" width="778" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Impact of 4 factors on Multi-omics clustering performance. Violin plots showing the distribution of clustering performance scores across different parameters: <strong>a</strong> Sample size effect - Performance trends with increasing sample sizes (5-38 samples per class), showing modest improvement in mean scores (red trend line) but notably decreased variance with larger samples. <strong>b</strong> Feature selection impact - The relationship between feature proportion and clustering performance, demonstrating optimal performance at 1-10% feature selection, decline at 20-50%, and consistent poor performance at 60-100%. <strong>c</strong> Class balance analysis - comparison of clustering performance between nearly balanced (&lt;3:1 ratio) and imbalanced (&gt;3:1 ratio) classes, showing significantly higher performance (****<em>p</em> &lt; 0.0001) in relatively balanced conditions. <strong>d</strong> Noise tolerance - distinct impact of performance response to increasing noise levels: stable (0-50%), decline (50-80%), and severe degradation (80-100%). Red lines indicate overall trends, and violin plot widths represent score distribution density at each parameter level</p></figcaption></figure></section><section id="Sec29"><h3 class="pmc_sec_title">Effect of feature selection</h3>
<p id="Par52">Feature selection appears to influence clustering accuracy, with certain tendencies observed in the relationship between feature proportion and performance (Fig. <a href="#Fig3" class="usa-link">3</a>b). In the analyzed dataset, lower feature proportion (1–10% of features) tended to yield higher cluster scores, averaging around 0.55, albeit with a relatively wide distribution. As the feature proportion increased (20–50%), a gradual decline in performance was observed, followed by a further reduction at higher feature proportion (60–100%), where mean cluster scores approached 0.4. Notably, the distribution patterns change markedly across these phases. The lower feature proportion (1-10%) showed wider distributions, indicating greater variability in outcomes but also higher potential for optimal performance. The transition phase showed gradually narrowing distributions, suggesting more consistent but declining performance. The higher feature proportion (particularly 90-100%) showed both decreased performance and narrower distributions, indicating consistently poorer outcomes when using most or all features.</p>
<p id="Par53">Such pattern suggests that selective feature usage, particularly under 20%, is recommended for optimal clustering performance. The sharp decline in performance with larger feature sets clearly indicates that selecting fewer but biologically relevant ones is beneficial. Detailed cancer-type specific trends are available in Supplementary Figure S1.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Preprocessing influence</h3>
<p id="Par54">The comparative analysis of three preprocessing approaches—RAW, NORM, and GL—revealed nuanced patterns across different cancer types and analytical contexts. While NORM datasets showed a slight overall advantage, the performance differences between methods were generally modest, typically below 0.1, suggesting robustness across the different preprocessing approaches. NORM showed strength in gender-based analyses, with consistently higher performance improvements, ranging from +0.20 to +0.27, across multiple cancer types including BLCA, COAD, HNSC, and LUAD. This suggests that normalization may be especially beneficial when analyzing gender-specific molecular patterns. Cancer-type specific patterns were observed where some types showed consistent preferences for particular preprocessing methods. For instance, LUAD demonstrated better performance with GL preprocessing across multiple test categories, while STAD consistently favored NORM preprocessing, indicating that optimal preprocessing strategies may be cancer-type dependent.</p>
<p id="Par55">Raw data preprocessing showed advantages in noise tolerance tests for several cancer types (notably BLCA, BRCA, and KIRP), potentially indicating that normalization may sometimes weaken the signals relevant to certain biological features. These findings suggest that while the choice of preprocessing can impact analysis outcomes, the effect is often context-dependent, varying by cancer type and analytical objective. The complete comparative analysis is available in Supplementary Table S5.</p></section><section id="Sec31"><h3 class="pmc_sec_title">Influence of class balance</h3>
<p id="Par56">The analysis of sample balance showed distinct difference in clustering performance between nearly balanced and imbalanced classes (Fig. <a href="#Fig3" class="usa-link">3</a>c). Datasets with nearly balanced classes (imbalance ratio &lt; 3:1) yielded significantly higher and more consistent cluster scores compared to imbalanced sample classes (imbalance ratio &gt;3:1), as indicated by the statistically significant difference (adj. <em>p</em> &lt; 0.0001). Classes with nearly balanced samples showed not only higher cluster scores (i.e., 0.58) but also exhibited a wider range of potential high-performance outcomes, as illustrated by the broader upper portion of the violin plot. In contrast, classes with imbalanced samples demonstrated lower performance (0.52) and a distinctive bimodal distribution, suggesting two distinct performance patterns in imbalanced datasets. The narrower distribution in the imbalanced classes indicates more predictable, though generally lower, performance outcomes. Collectively, the overall reliability of clustering results is likely to decline when the sample ratio exceeds 3:1. Detailed cancer-type specific trends are available in Supplementary Figure S1.</p></section><section id="Sec32"><h3 class="pmc_sec_title">Noise-induced effects on clustering outcomes</h3>
<p id="Par57">The analysis of noise tolerance in multi-omics integration showed distinct performance drop after a certain level of noise (Fig. <a href="#Fig3" class="usa-link">3</a>d). The violin plots demonstrate a clear response to increased noise levels. The performance was stable with 0-50% noise where cluster scores maintained around 0.5 with consistent performance. The cluster score started to decline with 50% noise showing gradual decline. Interestingly, the variance in performance decreased substantially with increasing noise, suggesting that high noise levels not only degrade performance but also make outcomes more predictably poor. The sharp decline after 80% noise indicates a critical threshold beyond which reliable clustering becomes nearly impossible, regardless of the MOI method used. Detailed cancer-type specific responses to noise are available in Supplementary Figure S1.</p></section><section id="Sec33"><h3 class="pmc_sec_title">Cancer subtype combination effect</h3>
<p id="Par58">Analysis of cancer subtype combinations across ten TCGA cancer types revealed distinct patterns in clustering performance (Fig. <a href="#Fig4" class="usa-link">4</a>). Cancer-type-specific molecular characteristics significantly influence multi-omics clustering performance, with cluster-score varying across both cancer types and specific subtype combinations. In BRCA, Luminal A vs. Luminal B classification consistently showed lower performance due to their shared estrogen receptor positivity and overlapping expression profiles, differing primarily in proliferation-related genes [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Similarly, COAD clustering of CMS2 and CMS3 subtypes yielded lower accuracy than more distinct pairs like CMS1 and CMS4, reflecting their shared epithelial characteristics and overlapping genomic features [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. These results demonstrate that clustering difficulty correlates with underlying molecular similarity between subtypes, emphasizing the importance of selecting biologically distinct subtypes for reliable integrative clustering outcomes. Method-specific performance patterns emerged across cancer types. SNF, CC, and Spectrum methods consistently maintained higher performance levels across most cancer types, while iClusterPlus showed consistently lower performance when handling multiple subtype distinctions. This pattern was particularly evident in cancers with complex molecular landscapes such as LUAD, which exhibited the most diverse combination patterns and generally lower clustering scores across all methods.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372284_12864_2025_11925_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/237f60e7f5e1/12864_2025_11925_Fig4_HTML.jpg" loading="lazy" id="MO4" height="687" width="778" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Method performance across cancer subtype combinations in ten cancer types. Analysis of clustering performance across different cancer subtype combinations in ten TCGA (The Cancer Genome Atlas) cancer types (BLCA, BRCA, COAD, HNSC, KIRP, LHC, LUAD, SKCM, STAD, and THCA). The x-axis represents different subtype combinations, and the y-axis shows cluster scores. The mean performance (red dashed line) indicates overall trends in clustering capability as subtype complexity increases</p></figcaption></figure><p id="Par59">Cancer types demonstrated varying levels of clustering stability. The KIRP showed relatively stable performance across fewer subtype combinations, while THCA and HNSC displayed more variable performance patterns depending on the specific subtype combinations analyzed. The general trend across all cancer types indicated declining performance as the number of subtypes in the combination increased, particularly when including molecularly similar subtypes. These observations show the importance of carefully considering both cancer type-specific molecular characteristics and method selection when designing multi-omics studies involving subtype analysis. The results suggest that optimal study design should prioritize clearly distinct molecular subtypes when possible, while acknowledging that performance may vary significantly based on the specific cancer type and analysis method chosen.</p></section><section id="Sec34"><h3 class="pmc_sec_title">Optimizing omics combinations for clinical features</h3>
<p id="Par60">Evaluating different omics-layer combinations for Subtype, and Pathological Stage (Fig. <a href="#Fig5" class="usa-link">5</a>) revealed that integrating GE–ME often led to superior clustering performance, especially for Subtype (Fig. <a href="#Fig5" class="usa-link">5</a>a). While GE–MI was also beneficial for Subtype, it did not perform as well for the other test. In contrast, the Pathological Stage (Fig. <a href="#Fig5" class="usa-link">5</a>b) benchmarks generally exhibited lower cluster-scores, suggesting that this clinical feature may be more challenging for MOI methods to distinguish. This pattern is reflected in the narrower regions seen in the corresponding panel (Fig. <a href="#Fig5" class="usa-link">5</a>b), regardless of the method employed. A closer examination of the combination test showed notable method-specific differences. For instance, SNF, COCA, and NEMO show consistently strong performance when classifying Gender using GE–ME, whereas other approaches (e.g., iCluster, MOFA) may excel in different contexts or for different clinical variables. These findings suggest the importance of matching the method to the data type and clinical interest: the interplay between specific omics layers (e.g., GE–ME vs. GE–MI) and particular clinical variables (e.g., Subtype vs. Gender) can substantially influence clustering success. Consequently, researchers need to evaluate a range of MOI approaches and omics-layer combinations to optimize performance for their specific study goals. The additional results for Gender and Age are available in Supplementary Figure S2.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372284_12864_2025_11925_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5acb/12372284/26a8785255dd/12864_2025_11925_Fig5_HTML.jpg" loading="lazy" id="MO5" height="883" width="778" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance analysis of multi-omics combinations across clinical features. Radar chart visualization of clustering performance across different omics combinations and clinical features, presented in two sections: summary charts (top) showing mean performance values and detailed method/cancer-specific charts (bottom). <strong>a</strong> Subtype Analysis - Superior clustering performance observed with combinations including GE-ME-MI (Gene Expression-Methylation-miRNA), demonstrating the value of integrating multiple omics layers for subtype identification. <strong>b</strong> Pathological Stage Classification - Generally lower clustering performance across omics combinations, indicating challenges in stage distinction through multi-omics integration</p></figcaption></figure></section><section id="Sec35"><h3 class="pmc_sec_title">Comparative performance of ten MOI methods</h3>
<p id="Par61">The comparative analysis of ten MOI methods across different test scenarios revealed distinct patterns of performance strengths (Table <a href="#Tab3" class="usa-link">3</a>). SNF demonstrated consistently strong performance in structural tests, leading in sample size (5/10), feature selection (4/10), and subtype combination analyses (5/10). NEMO showed particular strength in clinical feature analysis, achieving top performance in stage-based omics combinations (5/10) and maintaining strong performance across multiple test categories. Method performance varied notably by analytical context. CC showed specific strength in gender-based analyses (4/9), suggesting utility for sex-specific molecular patterns. PP demonstrated robust performance under noise conditions (3/10), while also maintaining competitive performance in various other categories. Other methods showed more specialized performance profiles. LRAcluster and IntNMF performed well in specific clinical contexts, particularly in gender and age-related analyses, while COCA and Spectrum showed moderate but consistent performance across various test scenarios. This performance heterogeneity emphasizes that method selection should be guided by specific analytical objectives rather than general performance metrics alone. Notably, the balanced performance across top methods in several categories (e.g., three methods sharing top performance in balance analysis) suggests that multiple approaches may be viable depending on specific research contexts. These findings highlight the importance of matching MOI method selection to specific analytical goals and data characteristics in the MOSD framework.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Performance ranking of MOI methods across different test scenarios</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Test case</th>
<th align="left" colspan="1" rowspan="1">Top method</th>
<th align="left" colspan="1" rowspan="1">Runner-up</th>
<th align="left" colspan="1" rowspan="1">Notable performer</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Sample size</td>
<td align="left" colspan="1" rowspan="1">SNF (5/10)</td>
<td align="left" colspan="1" rowspan="1">NEMO (2/10)</td>
<td align="left" colspan="1" rowspan="1">PP (2/10)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Feature selection</td>
<td align="left" colspan="1" rowspan="1">SNF (4/10)</td>
<td align="left" colspan="1" rowspan="1">NEMO (2/10)</td>
<td align="left" colspan="1" rowspan="1">PP, CC (2/10 each)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Balance</td>
<td align="left" colspan="1" rowspan="1">SNF, NEMO (3/10 each)</td>
<td align="left" colspan="1" rowspan="1">PP (2/10)</td>
<td align="left" colspan="1" rowspan="1">CC (1/10)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Noise tolerance</td>
<td align="left" colspan="1" rowspan="1">PP (3/10)</td>
<td align="left" colspan="1" rowspan="1">SNF, NEMO, CC (2/10 each)</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Subtype combination</td>
<td align="left" colspan="1" rowspan="1">SNF (5/10)</td>
<td align="left" colspan="1" rowspan="1">NEMO (2/10)</td>
<td align="left" colspan="1" rowspan="1">MOFA, CC (1/10 each)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Omics (Subtype)</td>
<td align="left" colspan="1" rowspan="1">NEMO (4/10)</td>
<td align="left" colspan="1" rowspan="1">SNF, Spectrum (2/10 each)</td>
<td align="left" colspan="1" rowspan="1">CC, IntNMF (1/10 each)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Omics (Gender)</td>
<td align="left" colspan="1" rowspan="1">CC (4/9)</td>
<td align="left" colspan="1" rowspan="1">LRAcluster (2/9)</td>
<td align="left" colspan="1" rowspan="1">SNF, COCA, IntNMF (1/9 each)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Omics (Stage)</td>
<td align="left" colspan="1" rowspan="1">NEMO (5/10)</td>
<td align="left" colspan="1" rowspan="1">PP, CC (2/10 each)</td>
<td align="left" colspan="1" rowspan="1">COCA (1/10)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Omics (Age)</td>
<td align="left" colspan="1" rowspan="1">NEMO (3/10)</td>
<td align="left" colspan="1" rowspan="1">IntNMF, CC, LRAcluster (2/10 each)</td>
<td align="left" colspan="1" rowspan="1">Spectrum (1/10)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p71"><p>A comparison of ten multi-omics integration methods across various test scenarios. Numbers in parentheses indicate the frequency of achieving the highest cluster score out of total cases (e.g., 5/10 means the method achieved the highest score in 5 out of 10 cancer types). Top method indicates the best performing method in each test scenario, followed by the second-best performer (Runner-up) and other methods showing notable performance (Notable Performer). Test cases include fundamental computational assessments (sample size, feature selection, balance, noise tolerance) and biological analyses (subtype and omics combinations with various clinical features). The gender analysis includes only 9 cases due to the exclusion of one cancer type with extreme gender imbalance</p></div></div></section></section></section><section id="Sec36"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par62">MOI has expanded the understanding of cancer biology [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>], while practical challenges persist. Factors such as sample size, feature selection, and omics combination selection influence whether analyses yield reliable or misleading insights [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. Despite advancements in machine learning algorithms, a unifying framework to address variability in data quality, noise, and imbalances has been lacking [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. The proposed MOSD framework encompasses both computational and biological factors, contributing to standardization of MOI approaches and bridging the gap between theory and practice.</p>
<p id="Par63">Feature selection plays a significant role in clustering performance—an aspect sometimes overshadowed by focus on algorithms [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. The findings suggest that feature curation can enhance interpretation and reduce computational overhead, even with modest sample sizes or inherent imbalances. This indicates the importance of considering both quantity and biological relevance of selected features. Benchmark tests revealed that using fewer features (1-10% of the total) generally yielded higher clustering performance, with improvements of up to 34% compared to using all features. This finding aligns with Pierre-Jean et al. [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]’s observation that feature selection impacts clustering quality, while providing more specific guidance on selection ratios. Sample size analysis demonstrated that 26 samples per class supports robust clustering performance, particularly for reducing variance in results. While absolute performance showed modest improvement with larger sample sizes, the consistency of results improved substantially, indicating that larger cohorts help prevent poor clustering outcomes rather than dramatically improving average performance.</p>
<p id="Par64">Class balance emerged as another important factor, with results showing that imbalance ratios exceeding 3:1 degraded clustering performance. This finding expands on Mirza [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]’s identification of class imbalance as a challenge in MOI analysis and offers a specific threshold for multi-omics study design. The comparison of raw, normalized, and gene-level preprocessing approaches demonstrated that preprocessing method selection affects clustering outcomes. The gene-level approach recommended by Jung et al. [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>] showed advantages in certain contexts, particularly when analyzing molecular subtypes, though effectiveness varied by cancer type and analysis method. For biological factors, certain cancer subtype combinations (e.g., Luminal B-Basal in BRCA) showed high discrimination potential, while others were more challenging to distinguish. This pattern was consistent across different MOI methods, highlighting the relevance of considering molecular distinctiveness of subtypes when designing studies.</p>
<p id="Par65">Additionally, combining numerous data types does not automatically improve results—particularly when clinical features are difficult to discriminate based on non-linear relationships [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. The GE-ME pairing frequently emerged as an effective omics combination across multiple cancer types, suggesting that focused integration of complementary omics types may be more effective than comprehensive multi-omics approaches. Method selection also influenced outcomes, with different algorithms showing distinct strengths across various test scenarios. SNF demonstrated consistent performance in structural tests, while NEMO performed well in clinical feature analysis. This method-specific performance variation indicates the importance of matching analytical approaches to specific research questions.</p></section><section id="Sec37"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par66">The MOSD framework provides a guide to prioritizing factors such as sample size, feature selection, and omics integration strategy for reliable and interpretable MOI results. The framework incorporates insights from benchmarking of ten MOI methods across ten TCGA cancer datasets, offering evidence-based recommendations for multi-omics analysis. Guidelines emerging from this study include: maintaining approximately 26 samples per class for reliable results, selecting less than 10% of features to enhance performance, ensuring sample balance under a 3:1 ratio between classes, and keeping noise levels below 30%. For biological factors, distinct molecular subtypes should be prioritized, and omics combinations selected based on the specific clinical features of interest, with GE-ME often providing effective results for subtype identification.</p>
<p id="Par67">These findings have implications for the design, execution, and interpretation of multi-omics studies, potentially improving efficiency and reproducibility of cancer research. Future research could explore framework adaptation to emerging data modalities and additional factors—such as temporal sampling or single-cell resolution—that may further refine multi-omics analyses. Continued refinement of these guidelines may support more efficient translation of complex molecular data into meaningful clinical and biological insights.</p></section><section id="Sec38"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12372284/bin/12864_2025_11925_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (2.8MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors of this paper thank the Division of Healthcare and Artificial Intelligence group within the Korea National Institute of Health for constructing the valuable multi-omics COVID-19 cohort dataset.</p></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>I.J and H.K. conceived the study and designed the experiments. H.K. developed the multi-omics integration methods and performed the data analysis. H.K., E.H., and I.J contributed to the experimental design and interpretation of the results. H.K., E.H., and I.J wrote the manuscript. I.J supported the funding and supervised the study. All authors reviewed and approved the final version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the Korea National Institute of Health (KNIH) research project (project no. 2024-ER-0801-01) and by the National Research Foundation of Korea (NRF) funded by the Korean Government (MSIT) (RS-2024-00440285).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data and associated code supporting the findings of this study are publicly available in our GitHub repository at <a href="https://github.com/cobi-git/MOPARAM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/cobi-git/MOPARAM</a>. The repository includes all necessary materials to reproduce the results presented, along with detailed documentation and instructions for use.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par68">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par69">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par70">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Eonyong Han and Hwijun Kwon contributed equally to this work.</p></div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Karczewski KJ, Snyder MP. Integrative omics for health and disease. Nat Rev Genet. 2018;19(5):299–310.
</cite> [<a href="https://doi.org/10.1038/nrg.2018.4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5990367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29479082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karczewski%20KJ,%20Snyder%20MP.%20Integrative%20omics%20for%20health%20and%20disease.%20Nat%20Rev%20Genet.%202018;19(5):299%E2%80%93310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Ebrahim A, Brunk E, Tan J, O’brien EJ, Kim D, Szubin R, et al. Multi-omic data integration enables discovery of hidden biological regularities. Nat Commun. 2016;7(1):13091.
</cite> [<a href="https://doi.org/10.1038/ncomms13091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5095171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27782110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ebrahim%20A,%20Brunk%20E,%20Tan%20J,%20O%E2%80%99brien%20EJ,%20Kim%20D,%20Szubin%20R,%20et%20al.%20Multi-omic%20data%20integration%20enables%20discovery%20of%20hidden%20biological%20regularities.%20Nat%20Commun.%202016;7(1):13091." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Hasin Y, Seldin M, Lusis A. Multi-omics approaches to disease. Genome Biol. 2017;18:1–15.
</cite> [<a href="https://doi.org/10.1186/s13059-017-1215-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5418815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28476144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hasin%20Y,%20Seldin%20M,%20Lusis%20A.%20Multi-omics%20approaches%20to%20disease.%20Genome%20Biol.%202017;18:1%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Misra BB, Langefeld C, Olivier M, Cox LA. Integrated omics: tools, advances and future approaches. J Mol Endocrinol. 2019;62(1):R21-45.
</cite> [<a href="https://doi.org/10.1530/JME-18-0055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30006342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Misra%20BB,%20Langefeld%20C,%20Olivier%20M,%20Cox%20LA.%20Integrated%20omics:%20tools,%20advances%20and%20future%20approaches.%20J%20Mol%20Endocrinol.%202019;62(1):R21-45." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Nicora G, Vitali F, Dagliati A, Geifman N, Bellazzi R. Integrated multi-omics analyses in oncology: a review of machine learning methods and tools. Front Oncol. 2020;10:1030.
</cite> [<a href="https://doi.org/10.3389/fonc.2020.01030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7338582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32695678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nicora%20G,%20Vitali%20F,%20Dagliati%20A,%20Geifman%20N,%20Bellazzi%20R.%20Integrated%20multi-omics%20analyses%20in%20oncology:%20a%20review%20of%20machine%20learning%20methods%20and%20tools.%20Front%20Oncol.%202020;10:1030." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Kwon MS, Kim Y, Lee S, Namkung J, Yun T, Yi SG, et al. Integrative analysis of multi-omics data for identifying multi-markers for diagnosing pancreatic cancer. BMC Genomics. 2015;16:1–10.
</cite> [<a href="https://doi.org/10.1186/1471-2164-16-S9-S4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4547403/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26328610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kwon%20MS,%20Kim%20Y,%20Lee%20S,%20Namkung%20J,%20Yun%20T,%20Yi%20SG,%20et%20al.%20Integrative%20analysis%20of%20multi-omics%20data%20for%20identifying%20multi-markers%20for%20diagnosing%20pancreatic%20cancer.%20BMC%20Genomics.%202015;16:1%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Chai H, Zhou X, Zhang Z, Rao J, Zhao H, Yang Y. Integrating multi-omics data through deep learning for accurate cancer prognosis prediction. Comput Biol Med. 2021;134: 104481.
</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2021.104481" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33989895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chai%20H,%20Zhou%20X,%20Zhang%20Z,%20Rao%20J,%20Zhao%20H,%20Yang%20Y.%20Integrating%20multi-omics%20data%20through%20deep%20learning%20for%20accurate%20cancer%20prognosis%20prediction.%20Comput%20Biol%20Med.%202021;134:%20104481." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Sammut SJ, Crispin-Ortuzar M, Chin SF, Provenzano E, Bardwell HA, Ma W, et al. Multi-omic machine learning predictor of breast cancer therapy response. Nature. 2022;601(7894):623–9.
</cite> [<a href="https://doi.org/10.1038/s41586-021-04278-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8791834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34875674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sammut%20SJ,%20Crispin-Ortuzar%20M,%20Chin%20SF,%20Provenzano%20E,%20Bardwell%20HA,%20Ma%20W,%20et%20al.%20Multi-omic%20machine%20learning%20predictor%20of%20breast%20cancer%20therapy%20response.%20Nature.%202022;601(7894):623%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Lai YP, Wang LB, Wang WA, Lai LC, Tsai MH, Lu TP, et al. iGC–an integrated analysis package of gene expression and copy number alteration. BMC Bioinformatics. 2017;18:1–9.
</cite> [<a href="https://doi.org/10.1186/s12859-016-1438-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5237550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28088185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lai%20YP,%20Wang%20LB,%20Wang%20WA,%20Lai%20LC,%20Tsai%20MH,%20Lu%20TP,%20et%20al.%20iGC%E2%80%93an%20integrated%20analysis%20package%20of%20gene%20expression%20and%20copy%20number%20alteration.%20BMC%20Bioinformatics.%202017;18:1%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Leday GG, van de Wiel MA. PLRS: a flexible tool for the joint analysis of DNA copy number and mRNA expression data. Bioinformatics. 2013;29(8):1081–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23419375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leday%20GG,%20van%20de%20Wiel%20MA.%20PLRS:%20a%20flexible%20tool%20for%20the%20joint%20analysis%20of%20DNA%20copy%20number%20and%20mRNA%20expression%20data.%20Bioinformatics.%202013;29(8):1081%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Tamborero D, Lopez-Bigas N, Gonzalez-Perez A. Oncodrive-CIS: a method to reveal likely driver genes based on the impact of their copy number changes on expression. PLoS One. 2013;8(2):e55489.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0055489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3568145/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23408991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tamborero%20D,%20Lopez-Bigas%20N,%20Gonzalez-Perez%20A.%20Oncodrive-CIS:%20a%20method%20to%20reveal%20likely%20driver%20genes%20based%20on%20the%20impact%20of%20their%20copy%20number%20changes%20on%20expression.%20PLoS%20One.%202013;8(2):e55489." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Gevaert O. Methylmix: an R package for identifying DNA methylation-driven genes. Bioinformatics. 2015;31(11):1839–41.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4443673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25609794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gevaert%20O.%20Methylmix:%20an%20R%20package%20for%20identifying%20DNA%20methylation-driven%20genes.%20Bioinformatics.%202015;31(11):1839%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Chae H, Rhee S, Nephew KP, Kim S. BioVLAB-MMIA-NGS: microRNA-mRNA integrated analysis using high-throughput sequencing data. Bioinformatics. 2015;31(2):265–7.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu614" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25270639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chae%20H,%20Rhee%20S,%20Nephew%20KP,%20Kim%20S.%20BioVLAB-MMIA-NGS:%20microRNA-mRNA%20integrated%20analysis%20using%20high-throughput%20sequencing%20data.%20Bioinformatics.%202015;31(2):265%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Teng CS, Wu BH, Yen MR, Chen PY. MethGET: web-based bioinformatics software for correlating genome-wide DNA methylation and gene expression. BMC Genomics. 2020;21:1–10.</cite> [<a href="https://doi.org/10.1186/s12864-020-6722-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7257144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32471342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teng%20CS,%20Wu%20BH,%20Yen%20MR,%20Chen%20PY.%20MethGET:%20web-based%20bioinformatics%20software%20for%20correlating%20genome-wide%20DNA%20methylation%20and%20gene%20expression.%20BMC%20Genomics.%202020;21:1%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Vasaikar SV, Straub P, Wang J, Zhang B. Linkedomics: analyzing multi-omics data within and across 32 cancer types. Nucleic Acids Res. 2018;46(D1):D956-63.
</cite> [<a href="https://doi.org/10.1093/nar/gkx1090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5753188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29136207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vasaikar%20SV,%20Straub%20P,%20Wang%20J,%20Zhang%20B.%20Linkedomics:%20analyzing%20multi-omics%20data%20within%20and%20across%2032%20cancer%20types.%20Nucleic%20Acids%20Res.%202018;46(D1):D956-63." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Meng C, Basunia A, Peters B, Gholami AM, Kuster B, Culhane AC. MOGSA: integrative single sample gene-set analysis of multiple omics data. Mol Cell Proteomics. 2019;18(8):S153-68.
</cite> [<a href="https://doi.org/10.1074/mcp.TIR118.001251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6692785/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31243065/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meng%20C,%20Basunia%20A,%20Peters%20B,%20Gholami%20AM,%20Kuster%20B,%20Culhane%20AC.%20MOGSA:%20integrative%20single%20sample%20gene-set%20analysis%20of%20multiple%20omics%20data.%20Mol%20Cell%20Proteomics.%202019;18(8):S153-68." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Paczkowska M, Barenboim J, Sintupisut N, Fox NS, Zhu H, Abd-Rabbo D, et al. Integrative pathway enrichment analysis of multivariate omics data. Nat Commun. 2020;11(1):735.
</cite> [<a href="https://doi.org/10.1038/s41467-019-13983-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7002665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32024846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paczkowska%20M,%20Barenboim%20J,%20Sintupisut%20N,%20Fox%20NS,%20Zhu%20H,%20Abd-Rabbo%20D,%20et%20al.%20Integrative%20pathway%20enrichment%20analysis%20of%20multivariate%20omics%20data.%20Nat%20Commun.%202020;11(1):735." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Canzler S, Hackermüller J. multiGSEA: a GSEA-based pathway enrichment analysis for multi-omics data. BMC Bioinformatics. 2020;21:1–13.
</cite> [<a href="https://doi.org/10.1186/s12859-020-03910-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7720482/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33287694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Canzler%20S,%20Hackerm%C3%BCller%20J.%20multiGSEA:%20a%20GSEA-based%20pathway%20enrichment%20analysis%20for%20multi-omics%20data.%20BMC%20Bioinformatics.%202020;21:1%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Ozerov IV, Lezhnina KV, Izumchenko E, Artemov AV, Medintsev S, Vanhaelen Q, et al. In silico pathway activation network decomposition analysis (iPANDA) as a method for biomarker development. Nat Commun. 2016;7(1):13427.
</cite> [<a href="https://doi.org/10.1038/ncomms13427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5116087/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27848968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ozerov%20IV,%20Lezhnina%20KV,%20Izumchenko%20E,%20Artemov%20AV,%20Medintsev%20S,%20Vanhaelen%20Q,%20et%20al.%20In%20silico%20pathway%20activation%20network%20decomposition%20analysis%20(iPANDA)%20as%20a%20method%20for%20biomarker%20development.%20Nat%20Commun.%202016;7(1):13427." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Weinstein JN, Collisson EA, Mills GB, Shaw KR, Ozenberger BA, Ellrott K, et al. The cancer genome atlas pan-cancer analysis project. Nat Genet. 2013;45(10):1113–20.
</cite> [<a href="https://doi.org/10.1038/ng.2764" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3919969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24071849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weinstein%20JN,%20Collisson%20EA,%20Mills%20GB,%20Shaw%20KR,%20Ozenberger%20BA,%20Ellrott%20K,%20et%20al.%20The%20cancer%20genome%20atlas%20pan-cancer%20analysis%20project.%20Nat%20Genet.%202013;45(10):1113%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Consortium ICG, et al. International network of cancer genome projects. Nature. 2010;464(7291):993.</cite> [<a href="https://doi.org/10.1038/nature08987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2902243/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20393554/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Barretina J, Caponigro G, Stransky N, Venkatesan K, Margolin AA, Kim S, et al. The cancer cell line encyclopedia enables predictive modelling of anticancer drug sensitivity. Nature. 2012;483(7391):603–7.
</cite> [<a href="https://doi.org/10.1038/nature11003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3320027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22460905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barretina%20J,%20Caponigro%20G,%20Stransky%20N,%20Venkatesan%20K,%20Margolin%20AA,%20Kim%20S,%20et%20al.%20The%20cancer%20cell%20line%20encyclopedia%20enables%20predictive%20modelling%20of%20anticancer%20drug%20sensitivity.%20Nature.%202012;483(7391):603%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Edwards NJ, Oberti M, Thangudu RR, Cai S, McGarvey PB, Jacob S, et al. The CPTAC data portal: a resource for cancer proteomics research. J Proteome Res. 2015;14(6):2707–13.
</cite> [<a href="https://doi.org/10.1021/pr501254j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25873244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edwards%20NJ,%20Oberti%20M,%20Thangudu%20RR,%20Cai%20S,%20McGarvey%20PB,%20Jacob%20S,%20et%20al.%20The%20CPTAC%20data%20portal:%20a%20resource%20for%20cancer%20proteomics%20research.%20J%20Proteome%20Res.%202015;14(6):2707%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Tini G, Marchetti L, Priami C, Scott-Boyer MP. Multi-omics integration–a comparison of unsupervised clustering methodologies. Brief Bioinform. 2019;20(4):1269–79.
</cite> [<a href="https://doi.org/10.1093/bib/bbx167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29272335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tini%20G,%20Marchetti%20L,%20Priami%20C,%20Scott-Boyer%20MP.%20Multi-omics%20integration%E2%80%93a%20comparison%20of%20unsupervised%20clustering%20methodologies.%20Brief%20Bioinform.%202019;20(4):1269%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Branciamore S, Chen ZX, Riggs AD, Rodin SN. CpG island clusters and pro-epigenetic selection for CpGs in protein-coding exons of HOX and other transcription factors. Proc Natl Acad Sci U S A. 2010;107(35):15485–90.
</cite> [<a href="https://doi.org/10.1073/pnas.1010506107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2932574/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20716685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Branciamore%20S,%20Chen%20ZX,%20Riggs%20AD,%20Rodin%20SN.%20CpG%20island%20clusters%20and%20pro-epigenetic%20selection%20for%20CpGs%20in%20protein-coding%20exons%20of%20HOX%20and%20other%20transcription%20factors.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202010;107(35):15485%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Nam AS, Chaligne R, Landau DA. Integrating genetic and non-genetic determinants of cancer evolution by single-cell multi-omics. Nat Rev Genet. 2021;22(1):3–18.
</cite> [<a href="https://doi.org/10.1038/s41576-020-0265-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8450921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32807900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nam%20AS,%20Chaligne%20R,%20Landau%20DA.%20Integrating%20genetic%20and%20non-genetic%20determinants%20of%20cancer%20evolution%20by%20single-cell%20multi-omics.%20Nat%20Rev%20Genet.%202021;22(1):3%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Mirza B, Wang W, Wang J, Choi H, Chung NC, Ping P. Machine learning and integrative analysis of biomedical big data. Genes. 2019;10(2):87.
</cite> [<a href="https://doi.org/10.3390/genes10020087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6410075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30696086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mirza%20B,%20Wang%20W,%20Wang%20J,%20Choi%20H,%20Chung%20NC,%20Ping%20P.%20Machine%20learning%20and%20integrative%20analysis%20of%20biomedical%20big%20data.%20Genes.%202019;10(2):87." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Chauvel C, Novoloaca A, Veyre P, Reynier F, Becker J. Evaluation of integrative clustering methods for the analysis of multi-omics data. Brief Bioinform. 2020;21(2):541–52.
</cite> [<a href="https://doi.org/10.1093/bib/bbz015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31220206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chauvel%20C,%20Novoloaca%20A,%20Veyre%20P,%20Reynier%20F,%20Becker%20J.%20Evaluation%20of%20integrative%20clustering%20methods%20for%20the%20analysis%20of%20multi-omics%20data.%20Brief%20Bioinform.%202020;21(2):541%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Pierre-Jean M, Deleuze JF, Le Floch E, Mauger F. Clustering and variable selection evaluation of 13 unsupervised methods for multi-omics data integration. Brief Bioinform. 2020;21(6):2011–30.
</cite> [<a href="https://doi.org/10.1093/bib/bbz138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31792509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pierre-Jean%20M,%20Deleuze%20JF,%20Le%20Floch%20E,%20Mauger%20F.%20Clustering%20and%20variable%20selection%20evaluation%20of%2013%20unsupervised%20methods%20for%20multi-omics%20data%20integration.%20Brief%20Bioinform.%202020;21(6):2011%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Rappoport N, Shamir R. Multi-omic and multi-view clustering algorithms: review and cancer benchmark. Nucleic Acids Res. 2018;46(20):10546–62.
</cite> [<a href="https://doi.org/10.1093/nar/gky889" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6237755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30295871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rappoport%20N,%20Shamir%20R.%20Multi-omic%20and%20multi-view%20clustering%20algorithms:%20review%20and%20cancer%20benchmark.%20Nucleic%20Acids%20Res.%202018;46(20):10546%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Duan R, Gao L, Gao Y, Hu Y, Xu H, Huang M, et al. Evaluation and comparison of multi-omics data integration methods for cancer subtyping. PLoS Comput Biol. 2021;17(8):e1009224.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1009224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8384175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34383739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duan%20R,%20Gao%20L,%20Gao%20Y,%20Hu%20Y,%20Xu%20H,%20Huang%20M,%20et%20al.%20Evaluation%20and%20comparison%20of%20multi-omics%20data%20integration%20methods%20for%20cancer%20subtyping.%20PLoS%20Comput%20Biol.%202021;17(8):e1009224." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Clark K, Vendt B, Smith K, Freymann J, Kirby J, Koppel P, et al. The cancer imaging archive (TCIA): maintaining and operating a public information repository. J Digit Imaging. 2013;26:1045–57.
</cite> [<a href="https://doi.org/10.1007/s10278-013-9622-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3824915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23884657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clark%20K,%20Vendt%20B,%20Smith%20K,%20Freymann%20J,%20Kirby%20J,%20Koppel%20P,%20et%20al.%20The%20cancer%20imaging%20archive%20(TCIA):%20maintaining%20and%20operating%20a%20public%20information%20repository.%20J%20Digit%20Imaging.%202013;26:1045%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Parker JS, Mullins M, Cheang MC, Leung S, Voduc D, Vickery T, et al. Supervised risk predictor of breast cancer based on intrinsic subtypes. J Clin Oncol. 2009;27(8):1160–7.
</cite> [<a href="https://doi.org/10.1200/JCO.2008.18.1370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2667820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19204204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Parker%20JS,%20Mullins%20M,%20Cheang%20MC,%20Leung%20S,%20Voduc%20D,%20Vickery%20T,%20et%20al.%20Supervised%20risk%20predictor%20of%20breast%20cancer%20based%20on%20intrinsic%20subtypes.%20J%20Clin%20Oncol.%202009;27(8):1160%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Guinney J, Dienstmann R, Wang X, De Reynies A, Schlicker A, Soneson C, et al. The consensus molecular subtypes of colorectal cancer. Nat Med. 2015;21(11):1350–6.
</cite> [<a href="https://doi.org/10.1038/nm.3967" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4636487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26457759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guinney%20J,%20Dienstmann%20R,%20Wang%20X,%20De%20Reynies%20A,%20Schlicker%20A,%20Soneson%20C,%20et%20al.%20The%20consensus%20molecular%20subtypes%20of%20colorectal%20cancer.%20Nat%20Med.%202015;21(11):1350%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Goldman MJ, Craft B, Hastie M, Repečka K, McDade F, Kamath A, et al. Visualizing and interpreting cancer genomics data via the Xena platform. Nat Biotechnol. 2020;38(6):675–8.
</cite> [<a href="https://doi.org/10.1038/s41587-020-0546-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7386072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32444850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goldman%20MJ,%20Craft%20B,%20Hastie%20M,%20Repe%C4%8Dka%20K,%20McDade%20F,%20Kamath%20A,%20et%20al.%20Visualizing%20and%20interpreting%20cancer%20genomics%20data%20via%20the%20Xena%20platform.%20Nat%20Biotechnol.%202020;38(6):675%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Silva TC, Colaprico A, Olsen C, D’Angelo F, Bontempi G, Ceccarelli M, et al. TCGA Workflow: Analyze cancer genomics and epigenomics data using Bioconductor packages. F1000Research. 2016;5:1542.</cite> [<a href="https://doi.org/10.12688/f1000research.8923.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5302158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28232861/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Jung I, Kim M, Rhee S, Lim S, Kim S. MONTI: a multi-omics non-negative tensor decomposition framework for gene-level integrative analysis. Front Genet. 2021. 10.3389/fgene.2021.682841.
</cite> [<a href="https://doi.org/10.3389/fgene.2021.682841" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8461247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34567063/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jung%20I,%20Kim%20M,%20Rhee%20S,%20Lim%20S,%20Kim%20S.%20MONTI:%20a%20multi-omics%20non-negative%20tensor%20decomposition%20framework%20for%20gene-level%20integrative%20analysis.%20Front%20Genet.%202021.%2010.3389/fgene.2021.682841." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Andersen NH, Christiansen JA, la Cour K, Aagesen M, Tang LH, Joergensen DS, et al. Differences in functioning between young adults with cancer and older age groups: a cross-sectional study. Eur J Cancer Care. 2022;31(6): e13660.</cite> [<a href="https://doi.org/10.1111/ecc.13660" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9788033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35843622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andersen%20NH,%20Christiansen%20JA,%20la%20Cour%20K,%20Aagesen%20M,%20Tang%20LH,%20Joergensen%20DS,%20et%20al.%20Differences%20in%20functioning%20between%20young%20adults%20with%20cancer%20and%20older%20age%20groups:%20a%20cross-sectional%20study.%20Eur%20J%20Cancer%20Care.%202022;31(6):%20e13660." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Hu Y, Xu Y, Mao L, Lei W, Xiang J, Gao L, et al. Gene expression analysis reveals age and ethnicity signatures between young and old adults in human PBMC. Front Aging. 2022. 10.3389/fragi.2021.797040.
</cite> [<a href="https://doi.org/10.3389/fragi.2021.797040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9261324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35822054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu%20Y,%20Xu%20Y,%20Mao%20L,%20Lei%20W,%20Xiang%20J,%20Gao%20L,%20et%20al.%20Gene%20expression%20analysis%20reveals%20age%20and%20ethnicity%20signatures%20between%20young%20and%20old%20adults%20in%20human%20PBMC.%20Front%20Aging.%202022.%2010.3389/fragi.2021.797040." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Wang B, Mezlini AM, Demir F, Fiume M, Tu Z, Brudno M, et al. Similarity network fusion for aggregating data types on a genomic scale. Nat Methods. 2014;11(3):333–7.
</cite> [<a href="https://doi.org/10.1038/nmeth.2810" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24464287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20B,%20Mezlini%20AM,%20Demir%20F,%20Fiume%20M,%20Tu%20Z,%20Brudno%20M,%20et%20al.%20Similarity%20network%20fusion%20for%20aggregating%20data%20types%20on%20a%20genomic%20scale.%20Nat%20Methods.%202014;11(3):333%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Yan D, Huang L, Jordan MI. Fast approximate spectral clustering. In: Proceedings of the 15th ACM SIGKDD international conference on Knowledge discovery and data mining. New York (NY): Association for Computing Machinery; 2009. p. 907–16. 10.1145/1557019.1557118.</cite>
</li>
<li id="CR42">
<span class="label">42.</span><cite>Nguyen H, Shrestha S, Draghici S, Nguyen T. PINSplus: a tool for tumor subtype discovery in integrated genomic data. Bioinformatics. 2019;35(16):2843–6.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty1049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30590381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen%20H,%20Shrestha%20S,%20Draghici%20S,%20Nguyen%20T.%20PINSplus:%20a%20tool%20for%20tumor%20subtype%20discovery%20in%20integrated%20genomic%20data.%20Bioinformatics.%202019;35(16):2843%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Nguyen T, Tagett R, Diaz D, Draghici S. A novel approach for data integration and disease subtyping. Genome Res. 2017;27(12):2025–39.
</cite> [<a href="https://doi.org/10.1101/gr.215129.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5741060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29066617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen%20T,%20Tagett%20R,%20Diaz%20D,%20Draghici%20S.%20A%20novel%20approach%20for%20data%20integration%20and%20disease%20subtyping.%20Genome%20Res.%202017;27(12):2025%E2%80%9339." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Rappoport N, Shamir R. NEMO: cancer subtyping by integration of partial multi-omic data. Bioinformatics. 2019;35(18):3348–56.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6748715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30698637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rappoport%20N,%20Shamir%20R.%20NEMO:%20cancer%20subtyping%20by%20integration%20of%20partial%20multi-omic%20data.%20Bioinformatics.%202019;35(18):3348%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Hoadley KA, Yau C, Wolf DM, Cherniack AD, Tamborero D, Ng S, et al. Multiplatform analysis of 12 cancer types reveals molecular classification within and across tissues of origin. Cell. 2014;158(4):929–44.
</cite> [<a href="https://doi.org/10.1016/j.cell.2014.06.049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4152462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25109877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoadley%20KA,%20Yau%20C,%20Wolf%20DM,%20Cherniack%20AD,%20Tamborero%20D,%20Ng%20S,%20et%20al.%20Multiplatform%20analysis%20of%2012%20cancer%20types%20reveals%20molecular%20classification%20within%20and%20across%20tissues%20of%20origin.%20Cell.%202014;158(4):929%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Wu D, Wang D, Zhang MQ, Gu J. Fast dimension reduction and integrative clustering of multi-omics data using low-rank approximation: application to cancer molecular classification. BMC Genomics. 2015;16:1–10.
</cite> [<a href="https://doi.org/10.1186/s12864-015-2223-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4667498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26626453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20D,%20Wang%20D,%20Zhang%20MQ,%20Gu%20J.%20Fast%20dimension%20reduction%20and%20integrative%20clustering%20of%20multi-omics%20data%20using%20low-rank%20approximation:%20application%20to%20cancer%20molecular%20classification.%20BMC%20Genomics.%202015;16:1%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Monti S, Tamayo P, Mesirov J, Golub T. Consensus clustering: a resampling-based method for class discovery and visualization of gene expression microarray data. Mach Learn. 2003;52:91–118.</cite> [<a href="https://scholar.google.com/scholar_lookup?Monti%20S,%20Tamayo%20P,%20Mesirov%20J,%20Golub%20T.%20Consensus%20clustering:%20a%20resampling-based%20method%20for%20class%20discovery%20and%20visualization%20of%20gene%20expression%20microarray%20data.%20Mach%20Learn.%202003;52:91%E2%80%93118." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC, et al. Multi-Omics factor analysis–a framework for unsupervised integration of multi-omics data sets. Mol Syst Biol. 2018;14(6): e8124.
</cite> [<a href="https://doi.org/10.15252/msb.20178124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6010767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29925568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Argelaguet%20R,%20Velten%20B,%20Arnol%20D,%20Dietrich%20S,%20Zenz%20T,%20Marioni%20JC,%20et%20al.%20Multi-Omics%20factor%20analysis%E2%80%93a%20framework%20for%20unsupervised%20integration%20of%20multi-omics%20data%20sets.%20Mol%20Syst%20Biol.%202018;14(6):%20e8124." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Mo Q, Wang S, Seshan VE, Olshen AB, Schultz N, Sander C, et al. Pattern discovery and cancer gene identification in integrated cancer genomic data. Proc Natl Acad Sci U S A. 2013;110(11):4245–50.
</cite> [<a href="https://doi.org/10.1073/pnas.1208949110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3600490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23431203/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mo%20Q,%20Wang%20S,%20Seshan%20VE,%20Olshen%20AB,%20Schultz%20N,%20Sander%20C,%20et%20al.%20Pattern%20discovery%20and%20cancer%20gene%20identification%20in%20integrated%20cancer%20genomic%20data.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202013;110(11):4245%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Shen R, Olshen AB, Ladanyi M. Integrative clustering of multiple genomic data types using a joint latent variable model with application to breast and lung cancer subtype analysis. Bioinformatics. 2009;25(22):2906–12.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2800366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19759197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen%20R,%20Olshen%20AB,%20Ladanyi%20M.%20Integrative%20clustering%20of%20multiple%20genomic%20data%20types%20using%20a%20joint%20latent%20variable%20model%20with%20application%20to%20breast%20and%20lung%20cancer%20subtype%20analysis.%20Bioinformatics.%202009;25(22):2906%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Chalise P, Fridley BL. Integrative clustering of multi-level ‘omic data based on non-negative matrix factorization algorithm. PLoS One. 2017;12(5):e0176278.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0176278" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5411077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28459819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chalise%20P,%20Fridley%20BL.%20Integrative%20clustering%20of%20multi-level%20%E2%80%98omic%20data%20based%20on%20non-negative%20matrix%20factorization%20algorithm.%20PLoS%20One.%202017;12(5):e0176278." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Lee D, Seung HS. Algorithms for non-negative matrix factorization. Adv Neural Inf Process Syst. 2000;13:556–562. </cite>
</li>
<li id="CR53">
<span class="label">53.</span><cite>Jeon J, Han EY, Jung I. MOPA: an integrative multi-omics pathway analysis method for measuring omics activity. PLoS One. 2023;18(3):e0278272.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0278272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10019735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36928437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jeon%20J,%20Han%20EY,%20Jung%20I.%20MOPA:%20an%20integrative%20multi-omics%20pathway%20analysis%20method%20for%20measuring%20omics%20activity.%20PLoS%20One.%202023;18(3):e0278272." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Xuan L, Zhigang C, Fan Y. Exploring of clustering algorithm on class-imbalanced data. In: 2013 8th International Conference on Computer Science &amp; Education. Piscataway, NJ, USA: IEEE; 2013. p. 89–93.</cite>
</li>
<li id="CR55">
<span class="label">55.</span><cite>Reel PS, Reel S, Pearson E, Trucco E, Jefferson E. Using machine learning approaches for multi-omics data analysis: a review. Biotechnol Adv. 2021;49: 107739.
</cite> [<a href="https://doi.org/10.1016/j.biotechadv.2021.107739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33794304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reel%20PS,%20Reel%20S,%20Pearson%20E,%20Trucco%20E,%20Jefferson%20E.%20Using%20machine%20learning%20approaches%20for%20multi-omics%20data%20analysis:%20a%20review.%20Biotechnol%20Adv.%202021;49:%20107739." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Pfitzner D, Leibbrandt R, Powers D. Characterization and evaluation of similarity measures for pairs of clusterings. Knowl Inf Syst. 2009;19:361–94.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pfitzner%20D,%20Leibbrandt%20R,%20Powers%20D.%20Characterization%20and%20evaluation%20of%20similarity%20measures%20for%20pairs%20of%20clusterings.%20Knowl%20Inf%20Syst.%202009;19:361%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Wirapati P, Sotiriou C, Kunkel S, Farmer P, Pradervand S, Haibe-Kains B, et al. Meta-analysis of gene expression profiles in breast cancer: toward a unified understanding of breast cancer subtyping and prognosis signatures. Breast Cancer Res. 2008;10:1–11.</cite> [<a href="https://doi.org/10.1186/bcr2124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2575538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18662380/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wirapati%20P,%20Sotiriou%20C,%20Kunkel%20S,%20Farmer%20P,%20Pradervand%20S,%20Haibe-Kains%20B,%20et%20al.%20Meta-analysis%20of%20gene%20expression%20profiles%20in%20breast%20cancer:%20toward%20a%20unified%20understanding%20of%20breast%20cancer%20subtyping%20and%20prognosis%20signatures.%20Breast%20Cancer%20Res.%202008;10:1%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Linnekamp JF, Hooff SRV, Prasetyanti PR, Kandimalla R, Buikhuisen JY, Fessler E, et al. Consensus molecular subtypes of colorectal cancer are recapitulated in in vitro and in vivo models. Cell Death Differ. 2018;25(3):616–33.
</cite> [<a href="https://doi.org/10.1038/s41418-017-0011-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5864202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29305587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Linnekamp%20JF,%20Hooff%20SRV,%20Prasetyanti%20PR,%20Kandimalla%20R,%20Buikhuisen%20JY,%20Fessler%20E,%20et%20al.%20Consensus%20molecular%20subtypes%20of%20colorectal%20cancer%20are%20recapitulated%20in%20in%20vitro%20and%20in%20vivo%20models.%20Cell%20Death%20Differ.%202018;25(3):616%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Ritchie MD, Holzinger ER, Li R, Pendergrass SA, Kim D. Methods of integrating data to uncover genotype-phenotype interactions. Nat Rev Genet. 2015;16(2):85–97.
</cite> [<a href="https://doi.org/10.1038/nrg3868" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25582081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ritchie%20MD,%20Holzinger%20ER,%20Li%20R,%20Pendergrass%20SA,%20Kim%20D.%20Methods%20of%20integrating%20data%20to%20uncover%20genotype-phenotype%20interactions.%20Nat%20Rev%20Genet.%202015;16(2):85%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Leek JT, Scharpf RB, Bravo HC, Simcha D, Langmead B, Johnson WE, et al. Tackling the widespread and critical impact of batch effects in high-throughput data. Nat Rev Genet. 2010;11(10):733–9.
</cite> [<a href="https://doi.org/10.1038/nrg2825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3880143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20838408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leek%20JT,%20Scharpf%20RB,%20Bravo%20HC,%20Simcha%20D,%20Langmead%20B,%20Johnson%20WE,%20et%20al.%20Tackling%20the%20widespread%20and%20critical%20impact%20of%20batch%20effects%20in%20high-throughput%20data.%20Nat%20Rev%20Genet.%202010;11(10):733%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Saeys Y, Inza I, Larranaga P. A review of feature selection techniques in bioinformatics. Bioinformatics. 2007;23(19):2507–17.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17720704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saeys%20Y,%20Inza%20I,%20Larranaga%20P.%20A%20review%20of%20feature%20selection%20techniques%20in%20bioinformatics.%20Bioinformatics.%202007;23(19):2507%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Huang S, Chaudhary K, Garmire LX. More is better: recent progress in multi-omics data integration methods. Front Genet. 2017;8:84.
</cite> [<a href="https://doi.org/10.3389/fgene.2017.00084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5472696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28670325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20S,%20Chaudhary%20K,%20Garmire%20LX.%20More%20is%20better:%20recent%20progress%20in%20multi-omics%20data%20integration%20methods.%20Front%20Genet.%202017;8:84." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12372284/bin/12864_2025_11925_MOESM1_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (2.8MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data and associated code supporting the findings of this study are publicly available in our GitHub repository at <a href="https://github.com/cobi-git/MOPARAM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/cobi-git/MOPARAM</a>. The repository includes all necessary materials to reproduce the results presented, along with detailed documentation and instructions for use.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12864-025-11925-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12864_2025_Article_11925.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12372284/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12372284/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372284%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372284/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12372284/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12372284/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847282/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12372284/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847282/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12372284/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12372284/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="QIxV0QvD5VvgEN23EKz6ipXgaU6PtyZqDXHgo0WJKruHPVtDiZ5R5SKv4hfCrEKc">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
