
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Artificial Intelligence‐Based Multimodal Prediction of Postoperative Adjuvant Immunotherapy Benefit in Urothelial Carcinoma: Results From the Phase III, Multicenter, Randomized, IMvigor010 Trial - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152548238AF35A63054823002FD5B6BC.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="mcomm">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377507/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="MedComm">
<meta name="citation_title" content="Artificial Intelligence‐Based Multimodal Prediction of Postoperative Adjuvant Immunotherapy Benefit in Urothelial Carcinoma: Results From the Phase III, Multicenter, Randomized, IMvigor010 Trial">
<meta name="citation_author" content="Xiatong Huang">
<meta name="citation_author_institution" content="Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Cancer Center, the Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China">
<meta name="citation_author_institution" content="Foshan Key Laboratory of Translational Medicine in Oncology, The Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China">
<meta name="citation_author" content="Wenjun Qiu">
<meta name="citation_author_institution" content="Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Yuyun Kong">
<meta name="citation_author_institution" content="Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Qiyun Ou">
<meta name="citation_author_institution" content="Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Malignant Tumor Epigenetics and Gene Regulation, Department of Ultrasound, Sun Yat‐sen Memorial Hospital, Sun Yat‐sen University, Guangzhou, China">
<meta name="citation_author" content="Qianqian Mao">
<meta name="citation_author_institution" content="Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Cancer Center, the Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China">
<meta name="citation_author_institution" content="Foshan Key Laboratory of Translational Medicine in Oncology, The Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China">
<meta name="citation_author" content="Yiran Fang">
<meta name="citation_author_institution" content="Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Zhouyang Fan">
<meta name="citation_author_institution" content="The First School of Clinical Medicine, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Jiani Wu">
<meta name="citation_author_institution" content="Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Xiansheng Lu">
<meta name="citation_author_institution" content="Department of Pathology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Wenchao Gu">
<meta name="citation_author_institution" content="Department of Diagnostic and Interventional Radiology, University of Tsukuba, Ibaraki, Japan">
<meta name="citation_author" content="Peng Luo">
<meta name="citation_author_institution" content="Department of Oncology, Zhujiang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Junfen Wang">
<meta name="citation_author_institution" content="Department of Gastroenterology, Nanfang Hospital, Southern Medical University, Guangzhou, China">
<meta name="citation_author" content="Jianping Bin">
<meta name="citation_author_institution" content="Department of Cardiology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Yulin Liao">
<meta name="citation_author_institution" content="Department of Cardiology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Min Shi">
<meta name="citation_author_institution" content="Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Zuqiang Wu">
<meta name="citation_author_institution" content="Cancer Center, the Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China">
<meta name="citation_author_institution" content="Foshan Key Laboratory of Translational Medicine in Oncology, The Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China">
<meta name="citation_author" content="Huiying Sun">
<meta name="citation_author_institution" content="Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author" content="Yunfang Yu">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Malignant Tumor Epigenetics and Gene Regulation, Guangdong‐Hong Kong Joint Laboratory for RNA Medicine, Department of Medical Oncology, Sun Yat‐sen Memorial Hospital, Sun Yat‐sen University, Guangzhou, China">
<meta name="citation_author_institution" content="Guangdong Provincial Key Laboratory of Cancer Pathogenesis and Precision Diagnosis and Treatment, AI and joint Big Data Laboratory, Department of Medical Oncology, Sun Yat‐sen Memorial Hospital, Sun Yat‐sen University, Shanwei, China">
<meta name="citation_author_institution" content="Department of Breast Surgery, The First Affiliated Hospital, Jinan University, Guangzhou, China">
<meta name="citation_author_institution" content="Institute for AI in Medicine and faculty of Medicine, Macau University of Science and Technology, Taipa, Macao, China">
<meta name="citation_author" content="Wangjun Liao">
<meta name="citation_author_institution" content="Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Cancer Center, the Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China">
<meta name="citation_author_institution" content="Foshan Key Laboratory of Translational Medicine in Oncology, The Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China">
<meta name="citation_author" content="Dongqiang Zeng">
<meta name="citation_author_institution" content="Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China">
<meta name="citation_author_institution" content="Cancer Center, the Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China">
<meta name="citation_author_institution" content="Foshan Key Laboratory of Translational Medicine in Oncology, The Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="6">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70324">
<meta name="citation_doi" content="10.1002/mco2.70324">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377507/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377507/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377507/pdf/MCO2-6-e70324.pdf">
<meta name="description" content="While circulating tumor DNA (ctDNA) testing has demonstrated utility in identifying muscle‐invasive urothelial carcinoma (MIUC) patients likely to benefit from adjuvant immunotherapy, the prognostic value of transcriptome data from surgical ...">
<meta name="og:title" content="Artificial Intelligence‐Based Multimodal Prediction of Postoperative Adjuvant Immunotherapy Benefit in Urothelial Carcinoma: Results From the Phase III, Multicenter, Randomized, IMvigor010 Trial">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="While circulating tumor DNA (ctDNA) testing has demonstrated utility in identifying muscle‐invasive urothelial carcinoma (MIUC) patients likely to benefit from adjuvant immunotherapy, the prognostic value of transcriptome data from surgical ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377507/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12377507">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/mco2.70324"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/MCO2-6-e70324.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377507%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12377507/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12377507/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377507/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-mcomm.jpg" alt="MedComm logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to MedComm" title="Link to MedComm" shape="default" href="https://onlinelibrary.wiley.com/journal/26882663" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">MedComm (2020)</button></div>. 2025 Aug 25;6(9):e70324. doi: <a href="https://doi.org/10.1002/mco2.70324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/mco2.70324</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22MedComm%20(2020)%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22MedComm%20(2020)%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22MedComm%20(2020)%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22MedComm%20(2020)%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Artificial Intelligence‐Based Multimodal Prediction of Postoperative Adjuvant Immunotherapy Benefit in Urothelial Carcinoma: Results From the Phase III, Multicenter, Randomized, IMvigor010 Trial</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xiatong Huang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xiatong Huang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Cancer Center, the Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Foshan Key Laboratory of Translational Medicine in Oncology, The Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiatong Huang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qiu%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Wenjun Qiu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wenjun Qiu</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qiu%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenjun Qiu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kong%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yuyun Kong</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yuyun Kong</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kong%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuyun Kong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ou%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Qiyun Ou</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Qiyun Ou</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Guangdong Provincial Key Laboratory of Malignant Tumor Epigenetics and Gene Regulation, Department of Ultrasound, Sun Yat‐sen Memorial Hospital, Sun Yat‐sen University, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ou%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qiyun Ou</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mao%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Qianqian Mao</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Qianqian Mao</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Cancer Center, the Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Foshan Key Laboratory of Translational Medicine in Oncology, The Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mao%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qianqian Mao</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yiran Fang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yiran Fang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yiran Fang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fan%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zhouyang Fan</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zhouyang Fan</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>The First School of Clinical Medicine, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fan%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhouyang Fan</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jiani Wu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jiani Wu</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiani Wu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Xiansheng Lu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Xiansheng Lu</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Department of Pathology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiansheng Lu</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gu%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Wenchao Gu</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Wenchao Gu</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>Department of Diagnostic and Interventional Radiology, University of Tsukuba, Ibaraki, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gu%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenchao Gu</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Peng Luo</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Peng Luo</span></h3>
<div class="p">
<sup>
<sup>8</sup>
</sup>Department of Oncology, Zhujiang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peng Luo</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Junfen Wang</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Junfen Wang</span></h3>
<div class="p">
<sup>
<sup>9</sup>
</sup>Department of Gastroenterology, Nanfang Hospital, Southern Medical University, Guangzhou, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junfen Wang</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bin%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Jianping Bin</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Jianping Bin</span></h3>
<div class="p">
<sup>
<sup>10</sup>
</sup>Department of Cardiology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bin%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jianping Bin</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Yulin Liao</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Yulin Liao</span></h3>
<div class="p">
<sup>
<sup>10</sup>
</sup>Department of Cardiology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yulin Liao</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Min Shi</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Min Shi</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Min Shi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Zuqiang Wu</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Zuqiang Wu</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Cancer Center, the Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Foshan Key Laboratory of Translational Medicine in Oncology, The Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zuqiang Wu</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Huiying Sun</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Huiying Sun</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huiying Sun</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Yunfang Yu</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Yunfang Yu</span></h3>
<div class="p">
<sup>
<sup>11</sup>
</sup>Guangdong Provincial Key Laboratory of Malignant Tumor Epigenetics and Gene Regulation, Guangdong‐Hong Kong Joint Laboratory for RNA Medicine, Department of Medical Oncology, Sun Yat‐sen Memorial Hospital, Sun Yat‐sen University, Guangzhou, China</div>
<div class="p">
<sup>
<sup>12</sup>
</sup>Guangdong Provincial Key Laboratory of Cancer Pathogenesis and Precision Diagnosis and Treatment, AI and joint Big Data Laboratory, Department of Medical Oncology, Sun Yat‐sen Memorial Hospital, Sun Yat‐sen University, Shanwei, China</div>
<div class="p">
<sup>
<sup>13</sup>
</sup>Department of Breast Surgery, The First Affiliated Hospital, Jinan University, Guangzhou, China</div>
<div class="p">
<sup>
<sup>14</sup>
</sup>Institute for AI in Medicine and faculty of Medicine, Macau University of Science and Technology, Taipa, Macao, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yunfang Yu</span></a>
</div>
</div>
<sup>11,</sup><sup>12,</sup><sup>13,</sup><sup>14,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Wangjun Liao</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Wangjun Liao</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Cancer Center, the Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Foshan Key Laboratory of Translational Medicine in Oncology, The Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wangjun Liao</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Dongqiang Zeng</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Dongqiang Zeng</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Cancer Center, the Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Foshan Key Laboratory of Translational Medicine in Oncology, The Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dongqiang Zeng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="mco270324-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Oncology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div id="mco270324-aff-0002">
<sup>
<sup>2</sup>
</sup>Cancer Center, the Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div id="mco270324-aff-0003">
<sup>
<sup>3</sup>
</sup>Foshan Key Laboratory of Translational Medicine in Oncology, The Sixth Affiliated Hospital, School of Medicine, South China University of Technology, Foshan, China</div>
<div id="mco270324-aff-0004">
<sup>
<sup>4</sup>
</sup>Guangdong Provincial Key Laboratory of Malignant Tumor Epigenetics and Gene Regulation, Department of Ultrasound, Sun Yat‐sen Memorial Hospital, Sun Yat‐sen University, Guangzhou, China</div>
<div id="mco270324-aff-0005">
<sup>
<sup>5</sup>
</sup>The First School of Clinical Medicine, Southern Medical University, Guangzhou, Guangdong, China</div>
<div id="mco270324-aff-0006">
<sup>
<sup>6</sup>
</sup>Department of Pathology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div id="mco270324-aff-0007">
<sup>
<sup>7</sup>
</sup>Department of Diagnostic and Interventional Radiology, University of Tsukuba, Ibaraki, Japan</div>
<div id="mco270324-aff-0008">
<sup>
<sup>8</sup>
</sup>Department of Oncology, Zhujiang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div id="mco270324-aff-0009">
<sup>
<sup>9</sup>
</sup>Department of Gastroenterology, Nanfang Hospital, Southern Medical University, Guangzhou, China</div>
<div id="mco270324-aff-0010">
<sup>
<sup>10</sup>
</sup>Department of Cardiology, Nanfang Hospital, Southern Medical University, Guangzhou, Guangdong, China</div>
<div id="mco270324-aff-0011">
<sup>
<sup>11</sup>
</sup>Guangdong Provincial Key Laboratory of Malignant Tumor Epigenetics and Gene Regulation, Guangdong‐Hong Kong Joint Laboratory for RNA Medicine, Department of Medical Oncology, Sun Yat‐sen Memorial Hospital, Sun Yat‐sen University, Guangzhou, China</div>
<div id="mco270324-aff-0012">
<sup>
<sup>12</sup>
</sup>Guangdong Provincial Key Laboratory of Cancer Pathogenesis and Precision Diagnosis and Treatment, AI and joint Big Data Laboratory, Department of Medical Oncology, Sun Yat‐sen Memorial Hospital, Sun Yat‐sen University, Shanwei, China</div>
<div id="mco270324-aff-0013">
<sup>
<sup>13</sup>
</sup>Department of Breast Surgery, The First Affiliated Hospital, Jinan University, Guangzhou, China</div>
<div id="mco270324-aff-0014">
<sup>
<sup>14</sup>
</sup>Institute for AI in Medicine and faculty of Medicine, Macau University of Science and Technology, Taipa, Macao, China</div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Apr 17; Received 2024 Sep 26; Accepted 2025 Jun 16; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>MedComm</em> published by Sichuan International Medical Exchange &amp; Promotion Association (SCIMEA) and John Wiley &amp; Sons Australia, Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12377507</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>While circulating tumor DNA (ctDNA) testing has demonstrated utility in identifying muscle‐invasive urothelial carcinoma (MIUC) patients likely to benefit from adjuvant immunotherapy, the prognostic value of transcriptome data from surgical specimens remains underexplored. Using transcriptomic and ctDNA data from the IMvigor010 trial, we developed an artificial intelligence (AI)‐driven biomarker to predict immunotherapy response in urothelial carcinoma, termed UAIscore. Patients with high UAIscore had significantly better outcomes in the atezolizumab arm versus the observation arm. Notably, the predictive performance of the UAIscore consistently outperformed that of ctDNA, tTMB, and PD‐L1, highlighting its value as an independent biomarker. Moreover, combining ctDNA, tTMB, and PD‐L1 with the UAIscore further improved predictive accuracy, underscoring the importance of integrating multi‐modality biomarkers. Further analysis of molecular subtypes revealed that the luminal subtype tends to be sensitive to adjuvant immunotherapy, as it may exhibit the highest level of immune infiltration and the lowest degree of hypoxia. Remarkably, we elucidated the role of the NF‐κB and TNF‐α pathways in mediating immunotherapy resistance within the immune‐enriched tumor microenvironment. These findings stratify patients likely to respond to adjuvant immunotherapy, concurrently providing a mechanistic rationale for combination therapies to augment immunotherapy efficacy in urothelial carcinoma.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> adjuvant immunotherapy, biomarker, decision tree model, predictive score, urothelial carcinoma</p></section></section><section class="abstract" id="mco270324-abs-0001"><hr class="headless">
<p>Based on transcriptome and ctDNA data from the IMvigor010 clinical trial, we sought to identify biomarkers that can predict outcomes following adjuvant immunotherapy for urothelial carcinoma by means of artificial intelligence, which is termed as UAIscore. Combining ctDNA, tTMB, and PD‐L1 with the UAIscore, as multi‐modality biomarkers, further improved predictive accuracy.
</p>
<section class="bt xbox font-sm" id="boxed-text1"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3228/12377507/0bdb17e5b025/MCO2-6-e70324-g002.jpg" loading="lazy" id="jats-graphic-1" height="141" width="180" alt="graphic file with name MCO2-6-e70324-g002.jpg"></p></section></section><section id="mco270324-sec-0010"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Primary urothelial carcinomas include bladder and upper urothelial carcinomas, of which approximately 25% are muscle‐invasive urothelial carcinomas (MIUCs) [<a href="#mco270324-bib-0001" class="usa-link" aria-describedby="mco270324-bib-0001">1</a>, <a href="#mco270324-bib-0002" class="usa-link" aria-describedby="mco270324-bib-0002">2</a>]. The current standard of treatment is radical resection of the bladder or renal ureter [<a href="#mco270324-bib-0001" class="usa-link" aria-describedby="mco270324-bib-0001">1</a>, <a href="#mco270324-bib-0002" class="usa-link" aria-describedby="mco270324-bib-0002">2</a>]. However, MIUC has a high risk of recurrence, with approximately half of the patients experiencing local or distal recurrence or metastasis within 2 years of radical resection and 5‐year survival rates of 60% or less [<a href="#mco270324-bib-0001" class="usa-link" aria-describedby="mco270324-bib-0001">1</a>, <a href="#mco270324-bib-0003" class="usa-link" aria-describedby="mco270324-bib-0003">3</a>]. In recent years, significant breakthroughs have been made in the application of immune checkpoint blockades (ICBs) in urothelial carcinoma. The Food and Drug Administration (FDA) has approved a variety of ICBs for the treatment of advanced urothelial carcinoma, including nivolumab, avelumab, and pembrolizumab [<a href="#mco270324-bib-0003" class="usa-link" aria-describedby="mco270324-bib-0003">3</a>, <a href="#mco270324-bib-0004" class="usa-link" aria-describedby="mco270324-bib-0004">4</a>, <a href="#mco270324-bib-0005" class="usa-link" aria-describedby="mco270324-bib-0005">5</a>]. Nevertheless, the promotion of adjuvant immunotherapy has encountered many challenges. At present, only CheckMate274 [<a href="#mco270324-bib-0003" class="usa-link" aria-describedby="mco270324-bib-0003">3</a>] clinical study has been successful, and nivolumab is the only drug recommended for adjuvant treatment after radical resection of urothelial carcinoma. At a median follow‐up of about 20 months, 48.2% of patients in the nivolumab group and 57.3% of patients in the placebo group had relapsed or died. We observed that in another large randomized clinical study (IMvigor010) [<a href="#mco270324-bib-0006" class="usa-link" aria-describedby="mco270324-bib-0006">6</a>], the ICB (anti‐PD‐L1) treatment group was no better than the observation group. IMvigor010 (<a href="https://clinicaltrials.gov/ct2/show/NCT02450331" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT02450331</a>), the largest and first completed Phase III randomized clinical trial of adjuvant therapy for MIUC, compared the results of the atezolizumab treatment group with the observation group. Unfortunately, the study did not meet its primary study endpoint [<a href="#mco270324-bib-0006" class="usa-link" aria-describedby="mco270324-bib-0006">6</a>, <a href="#mco270324-bib-0007" class="usa-link" aria-describedby="mco270324-bib-0007">7</a>].</p>
<p>Reflecting on the failure of IMvigor010, we realized that finding biomarkers that can predict the efficacy of ICBs and thus screen patients who will respond to ICBs was crucial for accurately improving the prognosis of urothelial carcinoma in recent years. To date, the biomarkers most commonly used to predict the efficacy of ICBs include expression level of programmed cell death‐ligand 1 (PD‐L1) [<a href="#mco270324-bib-0008" class="usa-link" aria-describedby="mco270324-bib-0008">8</a>], tumor mutational burden (TMB) [<a href="#mco270324-bib-0009" class="usa-link" aria-describedby="mco270324-bib-0009">9</a>], microsatellite instability (MSI) [<a href="#mco270324-bib-0010" class="usa-link" aria-describedby="mco270324-bib-0010">10</a>], and circulating tumor DNA (ctDNA) [<a href="#mco270324-bib-0007" class="usa-link" aria-describedby="mco270324-bib-0007">7</a>]. Subsequent exploratory biomarker analysis of IMvigor010 also revealed the predictive ability of ctDNA for ICBs [<a href="#mco270324-bib-0007" class="usa-link" aria-describedby="mco270324-bib-0007">7</a>]. Indeed, ctDNA as a biomarker still has limitations, such as the high cost of ctDNA testing, false‐positive results, and no treatment recommendation for ctDNA‐negative patients.</p>
<p>With the rapid development of artificial intelligence (AI) and multi‐omics, their potential in the medical field has become increasingly evident [<a href="#mco270324-bib-0011" class="usa-link" aria-describedby="mco270324-bib-0011">11</a>, <a href="#mco270324-bib-0012" class="usa-link" aria-describedby="mco270324-bib-0012">12</a>, <a href="#mco270324-bib-0013" class="usa-link" aria-describedby="mco270324-bib-0013">13</a>]. Machine learning and deep learning techniques have been widely applied in various studies related to cancer diagnosis, prognostic prediction, treatment planning, and so on [<a href="#mco270324-bib-0011" class="usa-link" aria-describedby="mco270324-bib-0011">11</a>, <a href="#mco270324-bib-0012" class="usa-link" aria-describedby="mco270324-bib-0012">12</a>, <a href="#mco270324-bib-0014" class="usa-link" aria-describedby="mco270324-bib-0014">14</a>, <a href="#mco270324-bib-0015" class="usa-link" aria-describedby="mco270324-bib-0015">15</a>]. Among these, the random forest classifier is one of the popular models. For urothelial carcinoma, various molecular subtypes have been constructed based on genome, transcriptome, epigenome, and multi‐omics combinations, including transcriptional component types that predict the response to neoadjuvant chemotherapy [<a href="#mco270324-bib-0014" class="usa-link" aria-describedby="mco270324-bib-0014">14</a>], genomic subtypes that predict the efficacy of ICBs in advanced urothelial carcinoma [<a href="#mco270324-bib-0015" class="usa-link" aria-describedby="mco270324-bib-0015">15</a>], and spatial immunophenotype predicting the efficacy of ICBs in urothelial cancers with distant metastases that do not match the primary site [<a href="#mco270324-bib-0016" class="usa-link" aria-describedby="mco270324-bib-0016">16</a>]. However, there are currently no widely available clinical biomarkers for MIUC to predict the response to adjuvant immunotherapy.</p>
<p>In this study, based on the transcriptomic data from IMvigor010 cohort (<em>n</em> = 728 high‐risk MIUC), we constructed a predictive score for survival benefit from adjuvant immunotherapy in urothelial carcinoma (UAIscore), using machine learning algorithms. [<a href="#mco270324-bib-0017" class="usa-link" aria-describedby="mco270324-bib-0017">17</a>, <a href="#mco270324-bib-0018" class="usa-link" aria-describedby="mco270324-bib-0018">18</a>, <a href="#mco270324-bib-0019" class="usa-link" aria-describedby="mco270324-bib-0019">19</a>, <a href="#mco270324-bib-0020" class="usa-link" aria-describedby="mco270324-bib-0020">20</a>] The UAIscore could be used as an independent biomarker and combined with ctDNA as a multi‐modality, which could accurately select patients who are likely to respond to immunotherapy in urothelial carcinoma. Importantly, a variety of influencing factors were comprehensively identified, including the infiltration of tumor immune microenvironment and clinical characteristics such as relapse status as well as tumor stage. In patients who did not respond, drug resistance factors were more common such as hypoxia, angiogenesis, insulin‐like growth factor 1 (IGF‐1), and fibroblast growth factor receptor (FGFR) in accordance with previous studies [<a href="#mco270324-bib-0021" class="usa-link" aria-describedby="mco270324-bib-0021">21</a>, <a href="#mco270324-bib-0022" class="usa-link" aria-describedby="mco270324-bib-0022">22</a>, <a href="#mco270324-bib-0023" class="usa-link" aria-describedby="mco270324-bib-0023">23</a>, <a href="#mco270324-bib-0024" class="usa-link" aria-describedby="mco270324-bib-0024">24</a>]. Interestingly, a few patients did not show differences in prognosis based on variations in the degree of immune infiltration, so it was necessary to uncover the mechanism of other important factors such as nuclear factor kappa‐light‐chain‐enhancer of activated B cells (NF‐κB) and tumor necrosis factor alpha (TNF‐α) pathways leading to resistance in immune‐enriched tumor microenvironment (TME) [<a href="#mco270324-bib-0025" class="usa-link" aria-describedby="mco270324-bib-0025">25</a>, <a href="#mco270324-bib-0026" class="usa-link" aria-describedby="mco270324-bib-0026">26</a>]. Thus, we have initially revealed the possible mechanisms of immune benefit or resistance, providing a preliminary analytical template and potential clinical basis for the precise stratification of immunotherapy and the development of novel combination strategies in urothelial carcinoma.</p></section><section id="mco270324-sec-0020"><h2 class="pmc_sec_title">2. Results</h2>
<section id="mco270324-sec-0030"><h3 class="pmc_sec_title">2.1. The Procedure of the UAIscore Conducted by the Random Forest Algorithm</h3>
<p>We utilized data from the IMvigor010 clinical trial to construct the UAIscore for predicting survival benefit from adjuvant immunotherapy, further demonstrating the independent predictive efficacy of UAIscore and the value of multi‐modality integration. Additionally, we explored the mechanisms of immunotherapy resistance from the perspectives of TME, receptor‐ligand interactions, and single‐cell interactions, providing a foundation for combined treatment strategies (Figure <a href="#mco270324-fig-0001" class="usa-link">1A</a>).</p>
<figure class="fig xbox font-sm" id="mco270324-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3228/12377507/52de631b11c9/MCO2-6-e70324-g003.jpg" loading="lazy" id="jats-graphic-3" height="645" width="546" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/mco270324-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Patients with high UAIscore were more likely to benefit from immunotherapy. (A) The flowchart showed the construction of the UAIscore model and subsequent exploration of its underlying mechanism. Created by BioRender and Vecteezy. (B and C) Kaplan–Meier curves of UAIscore were generated based on DFS survival analysis in immunotherapy (B) and observation (C) cohorts of IMvigor010. (D and E) Kaplan–Meier curves of treatment were generated based on DFS survival analysis in high UAIscore (D) and low UAIscore (E) groups of IMvigor010. DFS, disease‐free survival; OS, overall survival; high, high UAIscore; low, low UAIscore; IO, immunotherapy; Obs, observation; ctDNA, ctDNA tested before treatment, tTMB, tumor mutation burden; PD‐L1, programmed cell death‐ligand 1.</p></figcaption></figure><p>Firstly, the transcriptome data of 728 MIUC patients in the IMvigor010 clinical trial were analyzed, with the patients randomly divided into a training group (<em>n</em> = 475) and a validation group (<em>n</em> = 253) in a 6.5:3.5 ratio. Batch survival analysis was performed for disease‐free survival (DFS) in the training group, and the genes not associated with DFS (<em>p</em> &gt; 0.05) were filtered out. Subgroup survival analysis was performed to select interaction genes whose expression levels caused survival differences in only one treatment group using the Publish R package, resulting in the identification of 91 genes. Then, random forest machine learning was employed to construct a model with 91 survival‐related marker genes to predict the prognosis of adjuvant immunotherapy, named UAIscore (details of the model construction were presented in Section 6 and Figure <a href="#mco270324-supinfo-0001" class="usa-link">S1</a>). Demographics of patients were compared, with no statistically significant differences between high and low UAIscore populations in sex and phenotype (Tables <a href="#mco270324-supinfo-0001" class="usa-link">S1 and S2</a>). Patients were stratified into high and low UAIscore groups according to the median UAIscore in the training set. Combining UAIscore stratification and treatment, all patients were divided into four groups: high UAIscore + immunotherapy (<em>n</em> = 204), high UAIscore + observation (<em>n</em> = 174), low UAIscore + immunotherapy (<em>n</em> = 170), low UAIscore + observation (<em>n</em> = 180).</p></section><section id="mco270324-sec-0040"><h3 class="pmc_sec_title">2.2. Patients With High UAIscore Were More Likely to Benefit From Immunotherapy</h3>
<p>Firstly, we evaluated whether UAIscore was associated with prognosis. The results showed that patients with the high UAIscore presented prolonged DFS (hazard ratio [HR] 2.85, 95% CI: 2.3–3.52, <em>p</em> &lt; 0.0001) and OS (HR 2.88, 95% CI: 2.15–3.85, <em>p</em> &lt; 0.0001) in IMvigor010 clinical trial (Figure <a href="#mco270324-supinfo-0001" class="usa-link">S2A,B</a>). Consistently, the high UAIscore group had significantly better outcomes (HR 1.64, 95% CI: 1.15–2.33, <em>p</em> = 0.0061, Figure <a href="#mco270324-supinfo-0001" class="usa-link">S2C</a>). Thus, the UAIscore could serve as an effective prognostic indicator.</p>
<p>We further examined whether the UAIscore had predictive value with respect to adjuvant immunotherapy, so that the survival analysis was conducted in the training set and the results were verified in the validation set and all patients of IMvigor010 cohort (Figure <a href="#mco270324-fig-0001" class="usa-link">1B–E</a>, Figure <a href="#mco270324-supinfo-0001" class="usa-link">S2D–I</a>). Notably, among patients treated with immunotherapy, the high UAIscore group had notably longer DFS (HR 7.21, 95% CI: 5.21–9.99, <em>p</em> &lt; 0.0001, Figure <a href="#mco270324-fig-0001" class="usa-link">1A</a>) and OS (HR 6.41, 95% CI: 4.04–10.19, <em>p</em> &lt; 0.0001, Figure <a href="#mco270324-supinfo-0001" class="usa-link">S2I</a>). No difference in DFS or OS between UAIscore arms was noted for patients who were keeping observation (Figure <a href="#mco270324-fig-0001" class="usa-link">1C</a> and Figure <a href="#mco270324-supinfo-0001" class="usa-link">S2I</a>).</p>
<p>When considering the UAIscore for treatment selection, patients who were high UAIscore had improved DFS and overall survival in the atezolizumab arm versus the observation arm (DFS HR 0.39, 95% CI: 0.31–0.61, <em>p</em> &lt; 0.0001, Figure <a href="#mco270324-fig-0001" class="usa-link">1D</a>; OS HR 0.39, 95% CI: 0.23–0.64, <em>p</em> = 0.0002; Figure <a href="#mco270324-supinfo-0001" class="usa-link">S2I</a>). In contrast, patients with low UAIscore had favorable DFS (HR 2.02, 95% CI: 1.55–2.62, <em>p</em> &lt; 0.0001, Figure <a href="#mco270324-fig-0001" class="usa-link">1E</a>) and OS (HR 1.7, 95% CI: 1.22–2.38, <em>p</em> = 0.0019, Figure <a href="#mco270324-supinfo-0001" class="usa-link">S2I</a>) when keep observing. Therefore, as the UAIscore could accurately stratify patients, those with high UAIscore were more likely to benefit from immunotherapy, while those with low UAIscore were advised to remain under observation.</p></section><section id="mco270324-sec-0050"><h3 class="pmc_sec_title">2.3. Temporal Analysis Revealed Enhanced Predictive Performance With Multi‐Modality</h3>
<p>Currently, the biomarkers most commonly used to predict the efficacy of ICBs include expression level of PD‐L1 [<a href="#mco270324-bib-0008" class="usa-link" aria-describedby="mco270324-bib-0008">8</a>], tTMB [<a href="#mco270324-bib-0009" class="usa-link" aria-describedby="mco270324-bib-0009">9</a>], and ctDNA [<a href="#mco270324-bib-0007" class="usa-link" aria-describedby="mco270324-bib-0007">7</a>]. Additionally, the importance of multi‐omics combined biomarkers was emphasized. Therefore, the efficacy of different measures in predicting outcomes was assessed in both the immunotherapy and observation cohorts, and further evaluation was conducted to determine whether combining multiple biomarkers could enhance predictive performance. To achieve a comprehensive understanding of how the area under the curve (AUC) of the receiver operating characteristic (ROC) varies across different time points, we analyzed the time‐dependent AUC. We found that the UAIscore consistently demonstrated better predictive capability than ctDNA, tTMB, and PD‐L1 in the immunotherapy cohort (Figure <a href="#mco270324-fig-0002" class="usa-link">2A–F</a>). Moreover, the combined use of ctDNA and UAIscore showed superior predictive ability compared to UAIscore alone during the initial 24 months for DFS, with a similar trend observed for OS during the initial 30 months (Figure <a href="#mco270324-fig-0002" class="usa-link">2A,B</a>). In the observation cohort, although the UAIscore alone did not perform satisfactorily, its integration with ctDNA significantly enhanced the prognostic value (Figure <a href="#mco270324-fig-0002" class="usa-link">2G–H</a>). Furthermore, the integration of ctDNA, tTMB, PD‐L1, and UAIscore improved the predictive accuracy in the later stages of immunotherapy survival, consistently serving as the best prognostic marker in the observation group (Figure <a href="#mco270324-fig-0002" class="usa-link">2G,H</a>). The inclusion of PD‐L1 and tTMB did not markedly enhance the predictive performance of the ctDNA combined with UAIscore consistently, which might be partly due to the fact that the PD‐L1 and tTMB data obtained were binary, affecting the construction of the multi‐modality model. Consequently, enhancing the effectiveness of a multi‐omics model should involve selecting appropriate biomarkers for specific time periods.</p>
<figure class="fig xbox font-sm" id="mco270324-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3228/12377507/9ef5cc5bbfa7/MCO2-6-e70324-g004.jpg" loading="lazy" id="jats-graphic-5" height="623" width="546" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/mco270324-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Temporal analysis revealed enhanced predictive performance with multi‐modality. (A–F) The time‐dependent AUC curve was utilized to evaluate the predictive efficacy with 95% confidence intervals of UAIscore, ctDNA, and the combination of both on DFS and OS in the immunotherapy cohorts. (G–H) The time‐dependent AUC curve was utilized to evaluate the predictive efficacy with 95% confidence intervals of UAIscore, ctDNA, and the combination of both on DFS and OS in the observation cohorts. (I–J) Kaplan–Meier curves of score obtained from the combination of UAIscore and ctDNA based on DFS and OS survival analysis in immunotherapy cohort. (K–L) Kaplan–Meier curves of multi‐modality gain from the combination of UAIscore, ctDNA, tTMB, and PD‐L1 based on DFS and OS survival analysis in immunotherapy cohort. DFS, disease‐free survival; OS, overall survival; high, high UAIscore; low, low UAIscore; IO, immunotherapy; Obs, observation; ctDNA, ctDNA tested before treatment, tTMB, tumor mutation burden; PD‐L1, programmed cell death‐ligand 1.</p></figcaption></figure><p>To more clearly observe the values of AUC at specific time points, we plotted the ROC curves for 1, 2, and 3 years. The results showed that UAIscore alone exhibited superior predictive performance for DFS in the immunotherapy cohort (UAIscore 1/2/3 year AUC: 0.87/0.91/0.96; Figure <a href="#mco270324-supinfo-0001" class="usa-link">S2J–L</a>). In addition, we conducted survival analyses on the results of the combined indicators to determine whether there was a stratification effect. We found that both the combination of ctDNA and UAIscore, as well as the multi‐modality, can accurately identify patients who respond to immunotherapy (Figure <a href="#mco270324-fig-0002" class="usa-link">2I–L</a>). These results demonstrated that the UAIscore could serve as an excellent and independent biomarker, and its predictive capability could be further enhanced through combination with ctDNA, tTMB, and PD‐L1 as a multi‐omic model for immunotherapy in urothelial carcinoma.</p></section><section id="mco270324-sec-0060"><h3 class="pmc_sec_title">2.4. UAIscore Was an Independent Predictive Biomarker for Adjuvant Immunotherapy</h3>
<p>To determine whether UAIscore could serve as an independent predictive biomarker, multivariate analysis was conducted separately on the immunotherapy and observation cohorts. Stratification by UAIscore showed a statistically significant difference in survival and indicated that it could serve as an independent biomarker for predicting the efficacy of immunotherapy (Figure <a href="#mco270324-fig-0003" class="usa-link">3A</a>). However, stratification by UAIscore did not reveal any survival differences in the observational cohort (Figure <a href="#mco270324-fig-0003" class="usa-link">3B</a>). Meanwhile, it was proved that ctDNA is a robust prognostic biomarker regardless of the treatment (Figure <a href="#mco270324-fig-0003" class="usa-link">3A,B</a>). We then divided the immunotherapy cohort into ctDNA positive and ctDNA negative groups and conducted a multivariate analysis again. The results presented that UAIscore had predictive value in both groups, combined with the previous ROC analysis results, which suggested that the predictive performance can be further improved through a combination of multiple indicators (Figure <a href="#mco270324-supinfo-0001" class="usa-link">S3A,B</a>).</p>
<figure class="fig xbox font-sm" id="mco270324-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3228/12377507/b7b7cd80747e/MCO2-6-e70324-g005.jpg" loading="lazy" id="jats-graphic-7" height="617" width="546" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/mco270324-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>UAIscore was an independent predictive biomarker for adjuvant immunotherapy. (A) Multivariate analysis of the immunotherapy cohorts. (B) Multivariate analysis of the observation cohorts. (C and D) Kaplan–Meier curves of UAIscore in ctDNA negative and ctDNA positive groups based on DFS survival analysis in immunotherapy cohort. (E) Kaplan–Meier curves of combine of UAIscore and ctDNA based on DFS survival analysis in immunotherapy cohort. *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01; ***<em>p</em> &lt; 0.001. ctDNA, ctDNA tested before treatment, tTMB, tumor mutation burden; PD‐L1, programmed cell death‐ligand 1.</p></figcaption></figure><p>No survival difference was observed among ctDNA‐negative patients regardless of whether they received immunotherapy in the IMvigor010 cohort [<a href="#mco270324-bib-0007" class="usa-link" aria-describedby="mco270324-bib-0007">7</a>]. Consequently, survival analysis was conducted within the immunotherapy cohort after stratifying by ctDNA. It was found that patients with high UAIscore had the best prognosis in both groups, indicating that UAIscore could provide further guidance for treatment in ctDNA‐negative patients (Figure <a href="#mco270324-fig-0003" class="usa-link">3C,D</a>). Combined analysis of UAIscore and ctDNA revealed that patients with high UAIscore and ctDNA‐negative status had the best therapeutic outcomes, suggesting that the combination of these biomarkers could more accurately identify patients who tended to be sensitive to immunotherapy (Figure <a href="#mco270324-fig-0003" class="usa-link">3E</a>). These findings indicated that the UAIscore has partially compensated for the predictive limitations of ctDNA.</p></section><section id="mco270324-sec-0070"><h3 class="pmc_sec_title">2.5. Clinical Characteristics and Molecular Subtypes and Their Potential Links to Immunotherapy Efficacy</h3>
<p>We conducted further investigations to explore clinical characteristics (Table <a href="#mco270324-supinfo-0001" class="usa-link">S2</a>). The proportion of patients who were positive for ctDNA or increased clearance of ctDNA was higher in the high UAIscore group (Figure <a href="#mco270324-fig-0004" class="usa-link">4A–D</a>). Consistent with previous reports, IC 2/3 PD‐L1 [<a href="#mco270324-bib-0027" class="usa-link" aria-describedby="mco270324-bib-0027">27</a>, <a href="#mco270324-bib-0028" class="usa-link" aria-describedby="mco270324-bib-0028">28</a>] (<em>p</em> &lt; 0.001) and high tGE3 (CD274, IFNγ, and CXCL9) signature [<a href="#mco270324-bib-0029" class="usa-link" aria-describedby="mco270324-bib-0029">29</a>] (<em>p</em> &lt; 0.0001) positively correlated with OS were more prevalent in the high UAIscore group, while high angiogenesis [<a href="#mco270324-bib-0030" class="usa-link" aria-describedby="mco270324-bib-0030">30</a>] (<em>p</em> &lt; 0.0001) negatively correlated with OS was more common in the low UAIscore group (Figure <a href="#mco270324-fig-0004" class="usa-link">4A,E–G</a>). Previous studies have indicated that positive for ctDNA was associated with higher recurrence rates and shorter survival [<a href="#mco270324-bib-0007" class="usa-link" aria-describedby="mco270324-bib-0007">7</a>], consistent with the finding in our study that patients in the low UAIscore group were more likely to relapse (Figure <a href="#mco270324-supinfo-0001" class="usa-link">S4A,B</a>). Interestingly, even among immunotherapy patients, those who were negative for ctDNA and were previously considered less prone to relapse had an increased likelihood of relapse if their UAIscore was low (Figure <a href="#mco270324-supinfo-0001" class="usa-link">S4A,B</a>). These findings suggested that the UAIscore may help identify ctDNA‐negative patients who were at a higher risk of relapse and ctDNA‐positive patients who may be more sensitive to immunotherapy.</p>
<figure class="fig xbox font-sm" id="mco270324-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3228/12377507/250e1cb30648/MCO2-6-e70324-g007.jpg" loading="lazy" id="jats-graphic-9" height="654" width="551" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/mco270324-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clinical characteristics and molecular subtypes and their potential linked to immunotherapy efficacy. (A) The heatmap illustrated the distribution of each biomarker in the four groups, which are defined based on the combination of UAIscore and treatment. The biomarkers include ctDNA tested pretherapy, ctDNA tested 6 weeks after randomization (post‐treatment), ctDNA clearance at two time points, ctDNA changes at two time points (C1–C3), tTMB, PD‐L1, recurrence status, and TCGA subtype. The evaluation of these biomarkers was performed using a chi‐square test. The statistical analysis was conducted using a chi‐square test. (B–H) The bar plot provided the specific percentage of important markers that show significant differences among the four groups. These markers include ctDNA tested pretherapy, ctDNA tested post‐treatment, ctDNA clearance, PD‐L1, tGE3 50, angiogenesis 50, and TCGA subtype. The statistical analysis was conducted using chi‐square test. (I) The distribution of UAIscore among different TCGA subtypes was illustrated using box plots. The Wilcoxon test was conducted to analyze the differences in UAIscore between the two subtypes. (J) Kaplan–Meier curves were generated to perform survival analysis based on DFS for the TCGA subtype in both the immunotherapy cohorts. *<em>p</em> &lt; 0.05; **<em>p</em> &lt; 0.01; ***<em>p</em> &lt; 0.001; ****<em>p</em> &lt; 0.0001. tTMB, tumor mutation burden; PD‐L1, programmed cell death‐ligand 1; high, high UAIscore; low, low UAIscore; IO, immunotherapy; Obs, observation; DFS, disease‐free survival.</p></figcaption></figure><p>We further investigated whether there were significant differences in UAIscore between different subtypes (Figure <a href="#mco270324-fig-0004" class="usa-link">4I</a> and Figure <a href="#mco270324-supinfo-0001" class="usa-link">S4C–E</a>). We focused on the TCGA subtype [<a href="#mco270324-bib-0031" class="usa-link" aria-describedby="mco270324-bib-0031">31</a>] and observed that the luminal subtype was primarily distributed in the high UAIscore group, while the neuronal subtype was more prevalent in the low UAIscore group (Figure <a href="#mco270324-fig-0004" class="usa-link">4A,H</a>). In the IMvigor010 cohort, survival analysis showed that the luminal subtype had the best outcomes, while the neuronal subtype had the worst survival, indicating that the TCGA subtype had prognostic values (Figure <a href="#mco270324-supinfo-0001" class="usa-link">S4F–G</a>). Additionally, survival differences among TCGA subtype were also observed in the immunotherapy cohort, suggesting that the luminal subtype was more likely to respond to immunotherapy compared to other subtypes (Figure <a href="#mco270324-fig-0004" class="usa-link">4J</a> and Figure <a href="#mco270324-supinfo-0001" class="usa-link">S4H</a>). Further analysis of TME revealed that the luminal subtype displayed the highest level of immune infiltration and the lowest degree of hypoxia, which may be reasons for the favorable prognosis and the benefits from immunotherapy (Figure <a href="#mco270324-supinfo-0001" class="usa-link">S4I,J</a>). These data indicated that molecular subgroup analysis of baseline tissues may help better predict patient outcomes.</p></section><section id="mco270324-sec-0080"><h3 class="pmc_sec_title">2.6. Molecular Characteristics and Tumor Immune Microenvironment in Patients Stratified by UAIscore</h3>
<p>We wondered why patients with high UAIscore were more likely to achieve improved survival outcomes with immunotherapy, so further exploration of the underlying biology was needed. For the high UAIscore group in immunotherapy cohort, enrichment analysis revealed its potential association with various immune cells and enzymes involved in the tumor immune microenvironment (Figure <a href="#mco270324-supinfo-0001" class="usa-link">S5A</a> and Figure <a href="#mco270324-fig-0005" class="usa-link">5A,B</a>). Coincidentally, gene set enrichment analysis (GSEA) also suggested that the high UAIscore group exhibited a higher enrichment of immune‐related pathways, including diverse immune cells and the PD‐1/PD‐L1 regulatory pathway (Figure <a href="#mco270324-fig-0005" class="usa-link">5C</a> and Figure <a href="#mco270324-supinfo-0001" class="usa-link">S5B</a>). In contrast, the pathways enriched in the low UAIscore group were more likely characterized by the extracellular matrix and reported drug resistance factors (Figure <a href="#mco270324-fig-0005" class="usa-link">5C,D</a> and Figure <a href="#mco270324-supinfo-0001" class="usa-link">S5B</a>). For instance, IGF‐1, vascular endothelial growth factor receptor (VEGFR), FGFR, and angiogenesis have been reported to be associated with tumor progression and the effect of immunotherapy [<a href="#mco270324-bib-0032" class="usa-link" aria-describedby="mco270324-bib-0032">32</a>, <a href="#mco270324-bib-0033" class="usa-link" aria-describedby="mco270324-bib-0033">33</a>, <a href="#mco270324-bib-0034" class="usa-link" aria-describedby="mco270324-bib-0034">34</a>]. Moreover, we utilized TME analysis and found that B cells, T cells, and tertiary lymphoid structures (TLSs) were significantly enriched in the high UAIscore group as expected, indicating a high degree of immune infiltration (Figure <a href="#mco270324-fig-0005" class="usa-link">5E</a>). With a few exceptions, the existence of TLSs in tumors is generally associated with improved prognosis and clinical outcomes following immunotherapy [<a href="#mco270324-bib-0035" class="usa-link" aria-describedby="mco270324-bib-0035">35</a>, <a href="#mco270324-bib-0036" class="usa-link" aria-describedby="mco270324-bib-0036">36</a>].</p>
<figure class="fig xbox font-sm" id="mco270324-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3228/12377507/fc97116bc50f/MCO2-6-e70324-g001.jpg" loading="lazy" id="jats-graphic-11" height="983" width="795" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/mco270324-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Multi‐dimensional profiling of immunotherapy response and resistance mechanisms. (A and B) From the differentially expressed genes in the high and low UAIscore groups of the immunotherapy cohort, the top 100, 200, and 300 genes in each group were selected for KEGG pathway enrichment analysis (A) and GO enrichment analysis (B). Representative pathways were chosen among those with adjusted <em>p</em> values &lt; 0.05. (C) In the immunotherapy cohort, genes were ranked by the relative expression between the two groups, and gene set enrichment analysis (GSEA) was performed based on Gene Ontology (GO). Representative pathways were selected from those with <em>p</em>‐value &lt; 0.05. (D) Within the representative pathways identified through GSEA enrichment analysis in the low + IO group, the expression differences of core enrichment genes were analysed using the Wilcoxon test. Genes with an adjusted <em>p</em>‐value &lt; 0.05 were visualized in a heatmap. (E) The tumor immune microenvironment of patients in the immunotherapy cohort was analysed using the tumor microenvironment signature from the IOBR package and xCell tumor immune invasion analysis tool. Gene differential analysis was performed between the high and low UAIscore groups using the Wilcoxon test, and the distribution of immune microenvironment characteristics in the two groups was presented through heatmap. (F) The association strength of 867 ligand–receptor pairs was quantified in immunotherapy patients using the Easier package. Gene differences between groups were analyzed with the Wilcoxon test, and ligand–receptor pairs with stronger associations in each group (adjusted <em>p</em> value &lt; 0.05) were selected and visualized in a ring chart. GSEA, gene set enrichment analysis; GO, Gene Ontology; KEGG, Kyoto Encyclopedia of Genes and Genomes; <em>p</em>, <em>p</em> value (in C); <em>p</em>, adjust <em>p</em> value (in A, B, and D–F); high, high UAIscore; low, low UAIscore; IO, immunotherapy; Obs, observation.</p></figcaption></figure><p>The interaction between ligands and receptors (LR) regulates cell–cell (CC) communication to modulate tumor characteristics and antitumor immune responses [<a href="#mco270324-bib-0037" class="usa-link" aria-describedby="mco270324-bib-0037">37</a>]. Thus, we quantified the strength of association for 867 ligand–receptor pairs using the Easier package and conducted a differential analysis (<em>p</em> &lt; 0.05) (Figure <a href="#mco270324-fig-0005" class="usa-link">5F</a>). Consistent with previous results, the collagen family associated with the extracellular matrix structure [<a href="#mco270324-bib-0038" class="usa-link" aria-describedby="mco270324-bib-0038">38</a>] and the fibroblast growth factor (FGF) family involved in FGF were primarily concentrated in the low UAIscore. Furthermore, most of the stronger associations in the low UAIscore exhibited HR &gt; 1 by survival analysis (Figure <a href="#mco270324-supinfo-0001" class="usa-link">S5C,D</a>). To counteract immunotherapy resistance, novel combination strategies to change tumor immune microenvironment or target specific target ligands and receptors could be considered.</p></section><section id="mco270324-sec-0090"><h3 class="pmc_sec_title">2.7. The TNF‐α/NF‐κB Signaling Was Associated With Immunotherapy Resistance in Immune‐Enriched TME</h3>
<p>Unexpectedly, in the immunotherapy cohort, some patients still had high immune infiltration in the high UAIscore, while a few patients had high immune infiltration in the low UAIscore (Figure <a href="#mco270324-supinfo-0001" class="usa-link">S6A</a>). Thus, the immunotherapy cohort was further divided into four categories, including high‐UAIscore immune‐enriched TME (H‐IE), low‐UAIscore immune‐enriched TME (L‐IE), high‐UAIscore immune‐desert TME (H‐ID), and low‐UAIscore immune‐desert TME (L‐ID). Interestingly, patients within the same UAIscore group did not show differences in prognosis based on variations in the degree of immune infiltration (Figure <a href="#mco270324-supinfo-0001" class="usa-link">S6B,C</a>). The H‐IE group exhibited high PD‐L1 expression and substantial T‐cell infiltration (Figures <a href="#mco270324-supinfo-0001" class="usa-link">S6A</a> and <a href="#mco270324-supinfo-0001" class="usa-link">S7A</a>), characteristics that typically predict a positive response to anti‐PD‐L1 therapy, yet did not initially respond, which was defined as primary drug resistance in previous literature [<a href="#mco270324-bib-0039" class="usa-link" aria-describedby="mco270324-bib-0039">39</a>]. These contradictions created challenges for only TME stratification instructing immunotherapy, so that other important factors influencing the response to immunotherapy must be uncovered.</p>
<p>We then focused on the H‐IE and L‐IE groups to investigate the reasons for the different responses to immunotherapy despite similar immune microenvironments. At first, we compared the differences in several clinical indicators between the two groups and found that the recurrence rate in the H‐IE group was significantly higher than that in the L‐IE group (Figures <a href="#mco270324-supinfo-0001" class="usa-link">S6D</a> and <a href="#mco270324-supinfo-0001" class="usa-link">S7B–G</a>), which may be one of the primary reasons for the failure of immunotherapy. In addition, we examined the activation status of 14 key pathways and found that the NF‐κB pathway and the TNF‐α pathway were significantly activated in the H‐IE group (Figures <a href="#mco270324-supinfo-0001" class="usa-link">S6E</a> and <a href="#mco270324-supinfo-0001" class="usa-link">S7H</a>). Previous studies have indicated that NF‐κB is a transcription factor and TNF‐α is a cytokine produced by immune cells, with an inter‐regulatory effect between the TNF‐α pathway and the NF‐κB pathway [<a href="#mco270324-bib-0025" class="usa-link" aria-describedby="mco270324-bib-0025">25</a>, <a href="#mco270324-bib-0026" class="usa-link" aria-describedby="mco270324-bib-0026">26</a>, <a href="#mco270324-bib-0040" class="usa-link" aria-describedby="mco270324-bib-0040">40</a>]. Both TNF‐α and NF‐κB have a bidirectional effect on the shaping of the TME and the occurrence and metastasis of tumors [<a href="#mco270324-bib-0041" class="usa-link" aria-describedby="mco270324-bib-0041">41</a>, <a href="#mco270324-bib-0042" class="usa-link" aria-describedby="mco270324-bib-0042">42</a>, <a href="#mco270324-bib-0043" class="usa-link" aria-describedby="mco270324-bib-0043">43</a>, <a href="#mco270324-bib-0044" class="usa-link" aria-describedby="mco270324-bib-0044">44</a>, <a href="#mco270324-bib-0045" class="usa-link" aria-describedby="mco270324-bib-0045">45</a>, <a href="#mco270324-bib-0046" class="usa-link" aria-describedby="mco270324-bib-0046">46</a>, <a href="#mco270324-bib-0047" class="usa-link" aria-describedby="mco270324-bib-0047">47</a>]. Furthermore, by performing differential gene analysis between the H‐IE as well as L‐IE groups and selecting top ranked genes (<em>n</em> = 100, 200, 300) for enrichment analysis, systemic immune function damage, extracellular matrix‐related factors, and disorders in lipid and protein metabolism likely contribute to the resistance in the H‐IE group (Figures <a href="#mco270324-supinfo-0001" class="usa-link">S6F</a> and <a href="#mco270324-supinfo-0001" class="usa-link">S7I–J</a>).</p></section><section id="mco270324-sec-0100"><h3 class="pmc_sec_title">2.8. The Interaction Strength of the COL1A1‐COL1A2 and CCL5‐CXCR3 Receptor–Ligand Pairs Might Mediate the Efficacy of Immunotherapy at the Single‐Cell Level</h3>
<p>Recent advancements in single‐cell technology have provided a comprehensive understanding of the fundamental properties of various cell types within TME. In this study, we also acquired single‐cell RNA sequencing data from 12 urothelial carcinoma patients who received ureterectomy or radical nephroureterectomy [<a href="#mco270324-bib-0048" class="usa-link" aria-describedby="mco270324-bib-0048">48</a>]. Subsequently, we standardized the data, resulting in the inclusion of 67,392 cells, which were assigned to nine major cell types based on gene expression markers, including T cells, myeloid cells, epithelial cells, fibroblasts, endothelial cells, mast cells, B cells, plasma cells, and natural killer (NK) cells (Figure <a href="#mco270324-fig-0006" class="usa-link">6A</a>).</p>
<figure class="fig xbox font-sm" id="mco270324-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3228/12377507/1a06fa650d4c/MCO2-6-e70324-g006.jpg" loading="lazy" id="jats-graphic-13" height="607" width="546" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/mco270324-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The interaction strength of the COL1A1‐COL1A2 and CCL5‐CXCR3 receptor–ligand pairs might mediate the efficacy of immunotherapy at the single‐cell level. (A) UMAP analysis of single cells from tissue samples of 12 urothelial cancer patients. In the UMAP map, nine cell‐type clusters are labeled by different colors. (B–E) The expression of receptor‒ligand pairs in various cell types were visualized in a bar plot at the single‐cell level. (F–G) Kaplan–Meier curves were generated to analyze DFS for receptor–ligand pairs in the immunotherapy set of IMvigor010 cohort.</p></figcaption></figure><p>To further investigate cell interactions, we analyzed the expression of several crucial ligand–receptor pairs identified in the immunotherapy cohort across different cell types (Figure <a href="#mco270324-fig-0006" class="usa-link">6B–I</a>). Members of the collagen family were commonly appeared in the low UAIscore group, such as collagen type I alpha 1 chain (COL1A1) and collagen type I alpha 2 chain (COL1A2) [<a href="#mco270324-bib-0049" class="usa-link" aria-describedby="mco270324-bib-0049">49</a>], which were significantly higher expressed in fibroblasts, suggesting a potential association between immunotherapy resistance and the active function of fibroblasts (Figure <a href="#mco270324-fig-0006" class="usa-link">6B,C</a>). Conversely, receptor ligands with significantly stronger interactions in the high UAIscore group, such as C‐C motif chemokine ligand 5 (CCL5) and C‐X‐C motif chemokine receptor 3 (CXCR3), were prominently expressed in T cells and NK cells (Figure <a href="#mco270324-fig-0006" class="usa-link">6D,E</a>). Previous studies have suggested that CCL5 produced by tumors and CXCR3 ligands induced by interferon‐gamma (IFN‐γ) from myeloid cells play a crucial role in facilitating T‐cell infiltration in immunoreactive tumors [<a href="#mco270324-bib-0050" class="usa-link" aria-describedby="mco270324-bib-0050">50</a>], indicating that enhanced interactions among these cells may contribute to a better response to immunotherapy in the high UAIscore group. Survival analysis of multiple receptor–ligand pairs in the immunotherapy cohort showed that the interaction strength of COL1A1‐COL1A2 was associated with poorer survival, whereas high CCL5‐CXCR3 interaction was associated with better outcomes, suggesting that these interactions might have mediated the benefit of immunotherapy (Figure <a href="#mco270324-fig-0006" class="usa-link">6F–G</a>). In conclusion, the activity of receptor–ligand pair interactions, such as COL1A1‐COL1A2 in the low UAIscore group and CCL5‐CXCR3 in the high UAIscore group, might play different regulatory roles in the efficacy of immunotherapy.</p></section></section><section id="mco270324-sec-0110"><h2 class="pmc_sec_title">3. Discussion</h2>
<p>Numerous previous studies have utilized biomarkers derived from multiple‐omics to predict the efficacy of chemotherapy and immunotherapy based on genome, transcriptome, epigenome, and multi‐omics combinations in urothelial carcinoma [<a href="#mco270324-bib-0014" class="usa-link" aria-describedby="mco270324-bib-0014">14</a>, <a href="#mco270324-bib-0015" class="usa-link" aria-describedby="mco270324-bib-0015">15</a>]. However, the application of these biomarkers in adjuvant immunotherapy has not yielded satisfactory results, and most studies have failed to establish a connection between data models and resistance mechanisms. ctDNA [<a href="#mco270324-bib-0007" class="usa-link" aria-describedby="mco270324-bib-0007">7</a>] has been proposed as an effective predictor of adjuvant immunotherapy in the follow‐up study of IMvigor010. It showed promising application prospects, but its clinical utilization is still limited due to high cost and measurement standardization. The UAIscore, a machine learning model developed based on transcriptomic data and guided by resistance mechanisms, is the first molecular biomarker for predicting adjuvant immunotherapy outcomes in urothelial carcinoma.</p>
<p>We evaluated the predictive ability of UAIscore and found that it performed exceptionally well in predicting survival and prognosis. Notably, UAIscore outperformed ctDNA, tTMB, and PD‐L1, which suggested that UAIscore could serve as an independent and excellent biomarker of immunotherapy. Furthermore, the predictive efficacy of the combination of ctDNA, tTMB, PD‐L1, and UAIscore was consistently superior to UAIscore alone in a period of time, highlighting the importance of integrating multi‐omic biomarkers. The results of multivariate analysis also presented that UAIscore had predictive value in both ctDNA‐positive and ctDNA‐negative groups. It enabled a more precise and personalized treatment plan and partially compensated for the predictive limitations of ctDNA when used in conjunction with the UAIscore for patient stratification. Specifically, patients with high UAIscore and positive ctDNA positive were more likely to respond to ICBs.</p>
<p>As for other clinical characteristics such as relapse status as well as tumor stage, respond and non‐respond patients performed significantly different. The factors contributing to immunotherapy efficacy have been the subject of numerous studies [<a href="#mco270324-bib-0039" class="usa-link" aria-describedby="mco270324-bib-0039">39</a>, <a href="#mco270324-bib-0051" class="usa-link" aria-describedby="mco270324-bib-0051">51</a>, <a href="#mco270324-bib-0052" class="usa-link" aria-describedby="mco270324-bib-0052">52</a>, <a href="#mco270324-bib-0053" class="usa-link" aria-describedby="mco270324-bib-0053">53</a>], with immune infiltration often highlighted as a critical focus. As expected, a high degree of immune infiltration, including enrichment of cytotoxic T cells and increased PD‐1 expression in TME, was emphasized in most patients who responded to immunotherapy. Further analysis of TCGA subtype revealed that the luminal subtype displayed the highest level of immune infiltration and lowest degree of hypoxia may be a reason for the favorable prognosis and the benefits from immunotherapy. In addition to the immune infiltration, we also captured the involvement of specific resistant molecules, pathways, and receptor–ligand pairs that play crucial roles in determining treatment outcomes, which might remodel the tumor immune microenvironment. These findings have been partially validated at the single‐cell level. Therefore, combined with previous studies, the results of this study can provide a foundation and new ideas for a combination of immunotherapy strategies. For instance, targeting VEGFR can alleviate hypoxia and reduce immunosuppression, and anti‐angiogenesis drugs combined with immunotherapy have been proven in preclinical studies to produce more effective antitumor effects [<a href="#mco270324-bib-0032" class="usa-link" aria-describedby="mco270324-bib-0032">32</a>]. Insulin‐like growth factor 1 receptor (IGF‐1R), which can be inhibited by picropodophyllin (PPP), enhances the therapeutic efficacy of immunotherapy through blockading both immunogenic cytotoxins and PD‐1 [<a href="#mco270324-bib-0033" class="usa-link" aria-describedby="mco270324-bib-0033">33</a>]. Besides, FGF/FGFR promoted the survival of Treg cells and the M2 polarization of tumor‐associated macrophages (TAMs) by facilitating interleukin‐2 (IL‐2)‐mediated signal transducer and activator of transcription (STAT5) phosphorylation, thereby promoting an suppressive immune microenvironment [<a href="#mco270324-bib-0034" class="usa-link" aria-describedby="mco270324-bib-0034">34</a>]. Currently, a series of clinical studies combining immunotherapy with targeted therapy [<a href="#mco270324-bib-0054" class="usa-link" aria-describedby="mco270324-bib-0054">54</a>] and interventions that modify the local TME [<a href="#mco270324-bib-0055" class="usa-link" aria-describedby="mco270324-bib-0055">55</a>, <a href="#mco270324-bib-0056" class="usa-link" aria-describedby="mco270324-bib-0056">56</a>] have been undertaken to enhance immunotherapy efficacy and address drug resistance.</p>
<p>Interestingly, our study revealed that a few patients did not show differences in prognosis based on variations in the degree of immune infiltration, whose contradictions create challenges for only TME stratification instructing immunotherapy. In previous literature, tumors that simultaneously exhibit PD‐L1 expression and TILs infiltration expected to respond to anti‐PD therapy but do not respond initially were defined as primary resistance [<a href="#mco270324-bib-0039" class="usa-link" aria-describedby="mco270324-bib-0039">39</a>]. Although the existence of TILs seems to be necessary, it is unclear which key cellular components dictate primary resistance [<a href="#mco270324-bib-0039" class="usa-link" aria-describedby="mco270324-bib-0039">39</a>]. Solving this issue, we revealed that the interaction between TNF‐α pathway and NF‐κB pathway led to immunotherapy resistance, potentially by inhibiting the function of immune cells within the immune‐enriched TME. Previous studies have shown that both TNF‐α and NF‐κB have a bidirectional effect on shaping the TME and on the occurrence and metastasis of tumors [<a href="#mco270324-bib-0041" class="usa-link" aria-describedby="mco270324-bib-0041">41</a>, <a href="#mco270324-bib-0042" class="usa-link" aria-describedby="mco270324-bib-0042">42</a>, <a href="#mco270324-bib-0043" class="usa-link" aria-describedby="mco270324-bib-0043">43</a>, <a href="#mco270324-bib-0044" class="usa-link" aria-describedby="mco270324-bib-0044">44</a>, <a href="#mco270324-bib-0045" class="usa-link" aria-describedby="mco270324-bib-0045">45</a>, <a href="#mco270324-bib-0046" class="usa-link" aria-describedby="mco270324-bib-0046">46</a>, <a href="#mco270324-bib-0047" class="usa-link" aria-describedby="mco270324-bib-0047">47</a>]. While TNF‐α can inhibit tumor cell growth by inducing apoptosis in cancer cells, it also stimulates the proliferation, survival, migration, and angiogenesis of most cancer cells that are resistant to TNF‐α induced cytotoxicity, leading to tumor development [<a href="#mco270324-bib-0041" class="usa-link" aria-describedby="mco270324-bib-0041">41</a>, <a href="#mco270324-bib-0042" class="usa-link" aria-describedby="mco270324-bib-0042">42</a>, <a href="#mco270324-bib-0043" class="usa-link" aria-describedby="mco270324-bib-0043">43</a>]. NF‐κB has long been considered a potential target for disease treatment. In addition to inhibiting key components of the NF‐κB pathway, targeting TNF‐α, which acts as an activator and effector of the NF‐κB pathway, is also expected to achieve the goal of suppressing tumor growth [<a href="#mco270324-bib-0044" class="usa-link" aria-describedby="mco270324-bib-0044">44</a>, <a href="#mco270324-bib-0045" class="usa-link" aria-describedby="mco270324-bib-0045">45</a>, <a href="#mco270324-bib-0046" class="usa-link" aria-describedby="mco270324-bib-0046">46</a>, <a href="#mco270324-bib-0047" class="usa-link" aria-describedby="mco270324-bib-0047">47</a>]. In addition, the macroscopic tumor environment [<a href="#mco270324-bib-0057" class="usa-link" aria-describedby="mco270324-bib-0057">57</a>, <a href="#mco270324-bib-0058" class="usa-link" aria-describedby="mco270324-bib-0058">58</a>] and intratumoral microbes [<a href="#mco270324-bib-0059" class="usa-link" aria-describedby="mco270324-bib-0059">59</a>] had to be considered, because their stability may improve immunotherapy outcomes in past studies.</p>
<p>In recent years, AI has demonstrated remarkable performance across various clinical fields and has also aided in capturing processes such as tumorigenesis, progression, and treatment resistance [<a href="#mco270324-bib-0013" class="usa-link" aria-describedby="mco270324-bib-0013">13</a>]. AI models can integrate multimodal data to reveal associations between different modalities, such as the relationship between pathology and the origin of primary tumors, the connections between multi‐omics features and cancer subtypes [<a href="#mco270324-bib-0013" class="usa-link" aria-describedby="mco270324-bib-0013">13</a>, <a href="#mco270324-bib-0060" class="usa-link" aria-describedby="mco270324-bib-0060">60</a>, <a href="#mco270324-bib-0061" class="usa-link" aria-describedby="mco270324-bib-0061">61</a>, <a href="#mco270324-bib-0062" class="usa-link" aria-describedby="mco270324-bib-0062">62</a>], and the influence of metabolic signatures on immune dynamics. These associations can help identify new biomarkers or enhance the performance of existing ones, providing more accurate diagnostic and therapeutic predictions [<a href="#mco270324-bib-0063" class="usa-link" aria-describedby="mco270324-bib-0063">63</a>, <a href="#mco270324-bib-0064" class="usa-link" aria-describedby="mco270324-bib-0064">64</a>]. However, prospective trial designs are still necessary to validate the effectiveness of these models, thus providing high‐quality evidence for widespread clinical use. In the future, our study could integrate multi‐omics data, including pathology, radiomics, and microbiomics, to improve the predictive performance of the UAIscore, explore the multi‐omics characteristics of patients stratified by different UAIscore from multiple perspectives, and uncover more specific resistance mechanisms to develop combined treatment strategies.</p></section><section id="mco270324-sec-0120"><h2 class="pmc_sec_title">4. Limitations of the Study</h2>
<p>Admittedly, this study does have certain limitations that must be acknowledged. The availability of publicly accessible datasets for adjuvant immunotherapy is currently limited, which hinders our ability to validate the predictive capability of UAIscore across different datasets thoroughly. Additionally, due to the limited types of omics data included in the IMvigor010 cohort, it was not possible to explore and identify more specific resistance mechanisms from a deeper perspective, thus providing higher level evidence for combined treatments that are most likely to enhance immunotherapy efficacy. We will continue to follow up on the immunotherapy cohort data to verify the predictive value of UAIscore and explore the mechanism of immunotherapy resistance in more independent cohorts.</p></section><section id="mco270324-sec-0130"><h2 class="pmc_sec_title">5. Conclusion</h2>
<p>Our study systematically analyzed the multi‐omics factors influencing adjuvant immunotherapy efficacy, leading to the development of the UAIscore model. Adjuvant atezolizumab may be associated with improved outcomes compared with observation in patients who are detected as having high UAIscore. The mechanisms of resistance were preliminarily explored, and a rationale for combination therapy was suggested in urothelial carcinoma.</p></section><section id="mco270324-sec-0140"><h2 class="pmc_sec_title">6. Materials and Methods</h2>
<section id="mco270324-sec-0150"><h3 class="pmc_sec_title">6.1. Data Acquisition and Processing</h3>
<p>This study was performed using tissue samples from the phase III, multicenter, open‐label, randomized trial IMvigor010 clinical (<a href="https://clinicaltrials.gov/ct2/show/NCT02450331" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT02450331</a>), which aimed to evaluate the efficacy of adjuvant immunotherapy in patients with histologically confirmed MIUC. Eligible patients had undergone radical surgical resection with lymph node dissection and exhibited no evidence of residual disease or metastasis, as confirmed by negative postoperative radiological imaging. Initially, 809 patients from 24 countries or regions were recruited, who were randomized in a 1:1 ratio to receive atezolizumab (1200 mg every 3 weeks) for up to 1 year as adjuvant therapy or undergo observation. A total of 728 patients met the clinical trial criteria, providing tissue samples for transcriptomic sequencing, which fully aligned with our research objectives and were entirely utilized for model construction and subsequent analyses. The primary efficacy endpoint was DFS, which was defined by local (pelvic) or urinary tract recurrence or distant urothelial carcinoma metastasis or death from any cause. The secondary efficacy endpoint was OS, which was defined as the time from randomization to death from any cause. The transcriptome data and primary clinical data of IMvigor010 trial are available after approval at the European genome‐phenome archive (EGA) (<a href="https://ega-archive.org/datasets/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ega‐archive.org/datasets/</a>, accession number: EGAS00001004997) [<a href="#mco270324-bib-0006" class="usa-link" aria-describedby="mco270324-bib-0006">6</a>]. The RNA‐sequencing data of the TCGA‐BLCA dataset were obtained from the TCGA data portal (<a href="https://portal.gdc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov/</a>) using the TCGAbiolinks R package in March 2022, and the clinical data were collected from the research of Liu J et al. [<a href="#mco270324-bib-0065" class="usa-link" aria-describedby="mco270324-bib-0065">65</a>]. The data used in this study are standardized Level 3 data (processed and aggregated results), which remain freely accessible on the TCGA website and are explicitly excluded from current US access restrictions. The independent single‐cell RNA‐seq datasets of 12 urothelial cancer patients who underwent ureterectomy or radical nephroureterectomy are publicly available through the Genome Sequence Archive in the National Genomics Data Center (NGDC) (<a href="https://ngdc.cncb.ac.cn/gsa-human/browse/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ngdc.cncb.ac.cn/gsa‐human/browse/</a>, accession number: HRA001867). [<a href="#mco270324-bib-0048" class="usa-link" aria-describedby="mco270324-bib-0048">48</a>]. All trials were conducted according to Good Clinical Practice and the Declaration of Helsinki. Protocol approval was obtained from independent review boards or ethics committees for each participating site, and informed consent was obtained from all patients. All data were accessed as per institutional protocols and jurisdictional compliance standards.</p></section><section id="mco270324-sec-0160"><h3 class="pmc_sec_title">6.2. The UAIscore Conducted by the Random Forest Algorithm</h3>
<p>The transcriptome data of 24,443 genes from 728 MIUC patients in the IMvigor010 clinical trial were analyzed. The median difference was used to remove 12,221 genes with low variation filtered by the median absolute deviation (MAD) method. A total of 728 patients were divided into a training group (<em>n</em> = 475) and a validation group (<em>n</em> = 253) according to 6.5:3.5 to further screen genes related to survival in all patients. Batch survival analysis was performed for DFS in the training group, and the 2259 genes associated with both DFS and overall survival were selected (<em>p</em> &lt; 0.05). The mean of each gene expression level was taken as the cut‐off value, and the training group samples were divided into high and low UAIscore group. Subgroup survival analysis was performed in the immunotherapy and the observation groups using the Publish R package (<a href="https://github.com/tagteam/Publish" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/tagteam/Publish</a>), and the interaction genes whose expression level caused survival differences in only one treatment group were filtered out (<em>p</em> &lt; 0.05). These survival interaction genes were divided into two parts with the boundary of HR = 1 and combined to obtain 91 marker genes for the final training of the random forest model. By means of data from immunotherapy patients in the training cohort, a risk score was generated based on the expression levels of selected marker genes, named the urothelial carcinoma adjuvant immunotherapy prediction score (UAIscore). The median UAIscore was used to stratify patients into high UAIscore and low UAIscore groups. The R package is available at <a href="https://github.com/LiaoWJLab/uaiscore" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/LiaoWJLab/uaiscore</a>.</p></section><section id="mco270324-sec-0170"><h3 class="pmc_sec_title">6.3. Transcriptomic Data Processing</h3>
<p>Whole‐transcriptome profiles were generated using TruSeq RNA Access technology (Illumina). Initially, quality control (QC) checks were performed on the raw sequence data using fastp (v.0.23.4), which provided insights into the quality metrics such as base quality, sequence quality, and the presence of adaptors. The trimming process ensured that only high‐quality, clean reads were used in subsequent analyses. For the alignment of the trimmed reads, we employed the STAR aligner (v.2.7.1a). Reads were mapped to the human reference genome GRCh38 (v111), utilizing the comprehensive gene annotations provided by the Gencode project (release 44). STAR was configured to perform a two‐pass alignment, which is particularly effective at detecting novel splice junctions and improving alignment rates. Post‐alignment, STAR also generated gene‐level quantification in the form of read counts per gene, using the –quantMode GeneCounts option. This output was crucial for downstream differential gene expression analysis. Finally, the read counts obtained from STAR were used to assemble a gene expression matrix.</p></section><section id="mco270324-sec-0180"><h3 class="pmc_sec_title">6.4. Detection of ctDNA, tTMB, and PD‐L1</h3>
<p>A total of 581 plasma samples were collected on the first day of the first treatment cycle (C1D1), while 495 peripheral blood samples were obtained on the first day of the third treatment cycle (C3D1). ctDNA detection was performed using Natera. After collection, plasma samples were processed within 30 min, separated, and stored at −80°C. Subsequent steps included cell‐free DNA (cfDNA) extraction and library preparation. For those samples where ctDNA was detected, the mean number of tumor molecules per milliliter of plasma was documented and calculated across all variants that met QC standards.</p>
<p>tTMB was evaluated based on the total number of somatic mutations per megabase of the captured exome. Patients whose tTMB was 10 or more mutations per megabase were considered tTMB‐positive, consistent with the average ctDNA BEP.</p>
<p>The central evaluation of PD‐L1 expression was conducted using the VENTANA SP142 immunohistochemistry assay. This assay employed a rabbit monoclonal primary antibody, which was diluted to a total protein concentration of 3 mg/mL. Tumors were classified as PD‐L1 expressing (IC2/3 status) when PD‐L1‐positive immune cells infiltrating the tumor accounted for 5% or more of the tumor area.</p></section><section id="mco270324-sec-0190"><h3 class="pmc_sec_title">6.5. Multivariate Analysis</h3>
<p>Patients were divided into the immunotherapy cohort and the observation cohort based on their treatment background in IMvigor010 clinical trial. A Cox proportional hazards model was utilized for multivariable regression analysis to assess the independent impact of the UAIscore on survival time, incorporating confounding factors such as gender, ctDNA status, tTMB status, PD‐L1 status, tumor stage, nodal status, number of lymph nodes resected, and phenotype.</p></section><section id="mco270324-sec-0200"><h3 class="pmc_sec_title">6.6. Inference of Cell Fraction and Signature Score</h3>
<p>We integrated TME cell type signature collected from several TME computational tools (CIBERSORT, MCP‐counter, ESTIMATE, EPIC, quantiseq, xCell) as well as the Immulo‐Oncology Biological Research (IOBR) R package (<a href="https://github.com/IOBR/IOBR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/IOBR/IOBR</a>) [<a href="#mco270324-bib-0038" class="usa-link" aria-describedby="mco270324-bib-0038">38</a>, <a href="#mco270324-bib-0066" class="usa-link" aria-describedby="mco270324-bib-0066">66</a>] to estimate the infiltration of multiple cell types in the above dataset and subsequent TME clustering was performed based on the above results. Using the gene set variation analysis (GSVA) algorithm, we used the Gene Ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), REACTOME, and HALLMARK gene sets to estimate the pathway enrichment score for each sample. The IOBR R package calculated scores for other popular genetic markers related to TME, tumor innate pathways, and metabolism for each sample. The Pathway RespOnsive GENes for activity Inference (PROGENy) R package (<a href="https://github.com/saezlab/progeny" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/saezlab/progeny</a>) was used to assess the activation level of 14 tumor signaling pathways.</p></section><section id="mco270324-sec-0210"><h3 class="pmc_sec_title">6.7. Identification of TME Subtypes</h3>
<p>Based on the TME cell type results obtained from signature calculations in xCell, MCP‐counter, EPIC, quantiseq, and IOBR (<a href="https://github.com/IOBR/IOBR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/IOBR/IOBR</a>) [<a href="#mco270324-bib-0038" class="usa-link" aria-describedby="mco270324-bib-0038">38</a>, <a href="#mco270324-bib-0066" class="usa-link" aria-describedby="mco270324-bib-0066">66</a>], a consensus clustering algorithm was used to determine the number of clusters in the immunotherapy cohort in IMvigor010. This process is performed using the ConsensusClusterPlus R package (<a href="http://www.bioconductor.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.bioconductor.org/</a>) and is repeated 1000 times to ensure classification stability. Unsupervised cluster analysis was performed on high and low UAIscore patients in the IO cohort, and the best discriminative power of each classification cluster was obtained when <em>k</em> was set at 2.</p></section><section id="mco270324-sec-0220"><h3 class="pmc_sec_title">6.8. Statistical Analysis</h3>
<p>The Shapiro–Wilk normality test was used to test the normality of the variables. For comparisons between two groups, the unpaired Student's <em>t</em>‐test was used to determine the statistical significance of normally distributed variables, and the Mann–Whitney <em>U</em> test (Wilcoxon rank sum test) was used to determine the statistical significance of nonnormally distributed variables. Kruskal–Wallis and one‐way ANOVA tests for nonparametric and parametric methods were used to compare two or more groups. Spearman and distance correlations were used to analyze correlations between normally distributed data. The chi‐squared test and bilateral Fisher exact test were used for contingency table analysis. Survival curves for each subgroup in the dataset were constructed using the Kaplan–Meier method, and the log‐rank (Mantel Cox) test was used to determine whether they were significantly different. A univariate Cox proportional hazards regression model was used to calculate the risk ratio from the univariate analysis. Multivariate Cox regression models were used to identify independent prognostic factors using the survminer R package. All statistical analyses were performed using R V.4.3.1 (<a href="https://www.r-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.r‐project.org/</a>) with a two‐sided <em>p</em> value. A <em>p</em> value less than 0.05 was considered statistically significant. The <em>p</em> value adjusted for multiple testing was calculated using the Benjamini–Hochberg correction method.</p></section></section><section id="mco270324-sec-0230"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>X.T.H</strong>.: conceptualization, methodology, formal analysis, data curation, writing – original draft preparation, visualization. <strong>W.J.Q</strong>.: investigation, resources, data curation, project administration. <strong>Y.Y.K</strong>.: investigation, resources, data curation, project administration. <strong>Q.Y.O</strong>.: investigation, writing and original draft preparation. <strong>Q.Q.M</strong>.: formal analysis. <strong>Y.R.F</strong>: resources, investigation. <strong>Z.Y.F</strong>.: resources, investigation. <strong>J.N.W</strong>.: writing – original draft preparation. <strong>X.S.L</strong>.: writing – original draft preparation. <strong>W.C.G</strong>.: resources, supervision. <strong>P.L</strong>.: resources, project administration. <strong>J.F.W</strong>.: resources, project administration, funding acquisition. <strong>J.P.B</strong>.: resources, supervision. <strong>Y.L.L</strong>.: resources, visualization. <strong>M.S</strong>.: supervision, methodology. <strong>Z.Q.W</strong>.: supervision, project administration. <strong>H.Y.S</strong>.: project administration, methodology. <strong>Y.F.Y</strong>.: project administration, writing – review and editing, software, validation. <strong>W.J.L</strong>.: project administration, conceptualization, resources, supervision, writing – review and editing, funding acquisition. <strong>D.Q.Z</strong>.: project administration, conceptualization, methodology, writing – review and editing, software, validation. All authors have read and approved the final manuscript.</p></section><section id="mco270324-sec-0250"><h2 class="pmc_sec_title">Ethics Statement</h2>
<p>The IMvigor010 study (ClinicalTrials.gov ID: <a href="https://clinicaltrials.gov/ct2/show/NCT02450331" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT02450331</a>), sponsored by Hoffmann‐La Roche, received ethical approval from institutional review boards/ethics committees at all participating sites. The HRA001867 dataset was approved by the Ethics Committee of Peking University First Hospital (Approval No. 2018[186]). All procedures involving human participants complied with the ethical standards of the institutional ethics committees and the Declaration of Helsinki. Informed consent was obtained from all participants or their legal representatives prior to sample collection.</p></section><section id="mco270324-sec-0260"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec26"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="mco270324-supinfo-0001"></section><section class="sm xbox font-sm" id="mco270324-supitem-0001"><div class="caption p">
<p>
<strong>Figure S1</strong>: A flowchart delineating process of model building and Performance verification.</p>
<p>
<strong>Figure S2</strong>: The predictive performance of multi‐omics biomarkers.</p>
<p>
<strong>Figure S3</strong>: Multivariate Cox proportional hazard analyses of clinical variables and treatments for ctDNA subgroup.</p>
<p>
<strong>Figure S4</strong>: Analysis of TCGA‐subtype in IMvigor010 clinical trial.</p>
<p>
<strong>Figure S5</strong>: Benefit and resistance mechanism of immunotherapy cohort.</p>
<p>
<strong>Figure S6</strong>: TNF‐á and the NF‐êB pathway may lead to resistance in immune‐enriched tumor microenvironments.</p>
<p>
<strong>Figure S7</strong>: Clinical characteristics and enrichment of pathways in different tumor immune microenvironment.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377507/bin/MCO2-6-e70324-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">MCO2-6-e70324-s001.pdf</a><sup> (8.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="mco270324-supitem-0002"><div class="caption p"><p>
<strong>Supporting File</strong>: mco270324‐sup‐0002‐Table.xlsx</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377507/bin/MCO2-6-e70324-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">MCO2-6-e70324-s002.xlsx</a><sup> (10.5MB, xlsx) </sup>
</div></div></section></section><section id="mco270324-sec-0240" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>Our gratitude extends to Professor Thomas Powles, Banchereau, R, and Xuesong Li for their valuable dataset contributions to this study. we are grateful to Genentech for supplying the transcriptomic, ctDNA, and clinical data from the IMvigor010 study. The work was supported by the National Natural Science Foundation of China (Nos. 82404016, 82272635, 82404087, and 82402294), the Guang Dong Basic and Applied Basic Research Foundation (Nos. 2018030310051, 2023A1515110193, 2025A1515011563, and 2023A1515010208), and the Guangdong Science and Technology Department (No. 2024B1212030002). We acknowledge Vecteezy (<a href="https://www.vecteezy.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.vecteezy.com/</a>) and BioRender (<a href="https://biorender.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://biorender.com</a>) for providing the graphic resources used in this work. We appreciate the assistance and support provided by AI and Joint Big Data Laboratory, Shenshan Medical Center, Memorial Hospital of Sun Yat‐sen University, UMedEVO Artificial Intelligence Technology (Guangzhou) Co., Ltd., and UMedREVO Artificial Intelligence Technology (Guangzhou) Co., Ltd. in this study.</p></section><section id="notes1"><section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="mco270324-note-0001"><p>Xiatong Huang, Wenjun Qiu, and Yuyun Kong contributed equally to this work.</p></div>
<div class="fn p" id="mco270324-note-0002"><p>
<strong>Funding</strong>: The work was supported by the National Natural Science Foundation of China (Nos. 82404016, 82272635, 82404087, and 82402294), the Guang Dong Basic and Applied Basic Research Foundation (Nos. 2018030310051, 2023A1515110193, 2025A1515011563, and 2023A1515010208), and the Guangdong Science and Technology Department (No. 2024B1212030002).</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Huiying Sun, Email: 1714761643@qq.com.</p>
<p>Yunfang Yu, Email: yuyf9@mail.sysu.edu.cn.</p>
<p>Wangjun Liao, Email: liaowj@smu.edu.cn.</p>
<p>Dongqiang Zeng, Email: interlaken@smu.edu.cn.</p></section><section id="mco270324-sec-0280"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>All datasets presented in this study can be found in online repositories. The data of IMvigor010 clinical trials (accession number: EGAS00001004997) was available after pass the application at the European Genome‐phenome Archive (EGA). The RNA‐sequencing data of the TCGA‐BLCA dataset were gained from the TCGA data portal (<a href="https://portal.gdc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov/</a>), and the clinical data were collected from the research of Liu J et al. [<a href="#mco270324-bib-0065" class="usa-link" aria-describedby="mco270324-bib-0065">65</a>]. The single‐cell RNA‐seq dataset was obtained from the National Genomics Data Center (CNCB; accession number: HRA001867). The data generated in this study are available in the article and its Supporting Information files. The R package is available at <a href="https://github.com/LiaoWJLab/uaiscore" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/LiaoWJLab/uaiscore</a>.</p></section><section id="mco270324-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="mco270324-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="mco270324-bib-0001">
<span class="label">1.</span><cite>
Witjes J. A., Bruins H. M., Cathomas R., et al., “European Association of Urology Guidelines on Muscle‐Invasive and Metastatic Bladder Cancer: Summary of the 2020 Guidelines,” European Urology
79, no. 1 (2021): 82–104.
</cite> [<a href="https://doi.org/10.1016/j.eururo.2020.03.055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32360052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Urology&amp;title=European%20Association%20of%20Urology%20Guidelines%20on%20Muscle%E2%80%90Invasive%20and%20Metastatic%20Bladder%20Cancer:%20Summary%20of%20the%202020%20Guidelines&amp;author=J.%20A.%20Witjes&amp;author=H.%20M.%20Bruins&amp;author=R.%20Cathomas&amp;volume=79&amp;issue=1&amp;publication_year=2021&amp;pages=82-104&amp;pmid=32360052&amp;doi=10.1016/j.eururo.2020.03.055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0002">
<span class="label">2.</span><cite>
Rouprêt M., Seisen T., Birtle A. J., et al., “European Association of Urology Guidelines on Upper Urinary Tract Urothelial Carcinoma: 2023 Update,” European Urology
84, no. 1 (2023): 49–64.
</cite> [<a href="https://doi.org/10.1016/j.eururo.2023.03.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36967359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Urology&amp;title=European%20Association%20of%20Urology%20Guidelines%20on%20Upper%20Urinary%20Tract%20Urothelial%20Carcinoma:%202023%20Update&amp;author=M.%20Roupr%C3%AAt&amp;author=T.%20Seisen&amp;author=A.%20J.%20Birtle&amp;volume=84&amp;issue=1&amp;publication_year=2023&amp;pages=49-64&amp;pmid=36967359&amp;doi=10.1016/j.eururo.2023.03.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0003">
<span class="label">3.</span><cite>
Bajorin D. F., Witjes J. A., Gschwend J. E., et al., “Adjuvant Nivolumab Versus Placebo in Muscle‐Invasive Urothelial Carcinoma,” New England Journal of Medicine
384, no. 22 (2021): 2102–2114.
</cite> [<a href="https://doi.org/10.1056/NEJMoa2034442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8215888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34077643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&amp;title=Adjuvant%20Nivolumab%20Versus%20Placebo%20in%20Muscle%E2%80%90Invasive%20Urothelial%20Carcinoma&amp;author=D.%20F.%20Bajorin&amp;author=J.%20A.%20Witjes&amp;author=J.%20E.%20Gschwend&amp;volume=384&amp;issue=22&amp;publication_year=2021&amp;pages=2102-2114&amp;pmid=34077643&amp;doi=10.1056/NEJMoa2034442&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0004">
<span class="label">4.</span><cite>
Powles T., Park S. H., Voog E., et al., “Avelumab Maintenance Therapy for Advanced or Metastatic Urothelial Carcinoma,” New England Journal of Medicine
383, no. 13 (2020): 1218–1230.
</cite> [<a href="https://doi.org/10.1056/NEJMoa2002788" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32945632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&amp;title=Avelumab%20Maintenance%20Therapy%20for%20Advanced%20or%20Metastatic%20Urothelial%20Carcinoma&amp;author=T.%20Powles&amp;author=S.%20H.%20Park&amp;author=E.%20Voog&amp;volume=383&amp;issue=13&amp;publication_year=2020&amp;pages=1218-1230&amp;pmid=32945632&amp;doi=10.1056/NEJMoa2002788&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0005">
<span class="label">5.</span><cite>
Balar A. V., Castellano D., and O'Donnell P. H., “First‐Line Pembrolizumab in Cisplatin‐Ineligible Patients With Locally Advanced and Unresectable or Metastatic Urothelial Cancer (KEYNOTE‐052): A Multicentre, Single‐Arm, Phase 2 Study,” Lancet Oncology
18, no. 11 (2017): 1483–1492.
</cite> [<a href="https://doi.org/10.1016/S1470-2045(17)30616-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28967485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncology&amp;title=First%E2%80%90Line%20Pembrolizumab%20in%20Cisplatin%E2%80%90Ineligible%20Patients%20With%20Locally%20Advanced%20and%20Unresectable%20or%20Metastatic%20Urothelial%20Cancer%20(KEYNOTE%E2%80%90052):%20A%20Multicentre,%20Single%E2%80%90Arm,%20Phase%202%20Study&amp;author=A.%20V.%20Balar&amp;author=D.%20Castellano&amp;author=P.%20H.%20O'Donnell&amp;volume=18&amp;issue=11&amp;publication_year=2017&amp;pages=1483-1492&amp;pmid=28967485&amp;doi=10.1016/S1470-2045(17)30616-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0006">
<span class="label">6.</span><cite>
Bellmunt J., Hussain M., Gschwend J. E., et al., “Adjuvant Atezolizumab Versus Observation in Muscle‐Invasive Urothelial Carcinoma (IMvigor010): A Multicentre, Open‐Label, Randomised, Phase 3 Trial,” Lancet Oncology
22, no. 4 (2021): 525–537.
</cite> [<a href="https://doi.org/10.1016/S1470-2045(21)00004-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8495594/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33721560/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncology&amp;title=Adjuvant%20Atezolizumab%20Versus%20Observation%20in%20Muscle%E2%80%90Invasive%20Urothelial%20Carcinoma%20(IMvigor010):%20A%20Multicentre,%20Open%E2%80%90Label,%20Randomised,%20Phase%203%20Trial&amp;author=J.%20Bellmunt&amp;author=M.%20Hussain&amp;author=J.%20E.%20Gschwend&amp;volume=22&amp;issue=4&amp;publication_year=2021&amp;pages=525-537&amp;pmid=33721560&amp;doi=10.1016/S1470-2045(21)00004-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0007">
<span class="label">7.</span><cite>
Powles T., Assaf Z. J., Davarpanah N., et al., “ctDNA Guiding Adjuvant Immunotherapy in Urothelial Carcinoma,” Nature
595, no. 7867 (2021): 432–437.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03642-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34135506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=ctDNA%20Guiding%20Adjuvant%20Immunotherapy%20in%20Urothelial%20Carcinoma&amp;author=T.%20Powles&amp;author=Z.%20J.%20Assaf&amp;author=N.%20Davarpanah&amp;volume=595&amp;issue=7867&amp;publication_year=2021&amp;pages=432-437&amp;pmid=34135506&amp;doi=10.1038/s41586-021-03642-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0008">
<span class="label">8.</span><cite>
Doroshow D. B., Bhalla S., Beasley M. B., et al., “PD‐L1 as a Biomarker of Response to Immune‐Checkpoint Inhibitors,” Nature Reviews Clinical Oncology
18, no. 6 (2021): 345–362.</cite> [<a href="https://doi.org/10.1038/s41571-021-00473-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33580222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Clinical%20Oncology&amp;title=PD%E2%80%90L1%20as%20a%20Biomarker%20of%20Response%20to%20Immune%E2%80%90Checkpoint%20Inhibitors&amp;author=D.%20B.%20Doroshow&amp;author=S.%20Bhalla&amp;author=M.%20B.%20Beasley&amp;volume=18&amp;issue=6&amp;publication_year=2021&amp;pages=345-362&amp;pmid=33580222&amp;doi=10.1038/s41571-021-00473-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0009">
<span class="label">9.</span><cite>
Halbert B., and Einstein D. J., “Hot or Not: Tumor Mutational Burden (TMB) as a Biomarker of Immunotherapy Response in Genitourinary Cancers,” Urology
147 (2021): 119–126.
</cite> [<a href="https://doi.org/10.1016/j.urology.2020.10.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33137348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Urology&amp;title=Hot%20or%20Not:%20Tumor%20Mutational%20Burden%20(TMB)%20as%20a%20Biomarker%20of%20Immunotherapy%20Response%20in%20Genitourinary%20Cancers&amp;author=B.%20Halbert&amp;author=D.%20J.%20Einstein&amp;volume=147&amp;publication_year=2021&amp;pages=119-126&amp;pmid=33137348&amp;doi=10.1016/j.urology.2020.10.030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0010">
<span class="label">10.</span><cite>
Zhao P., Li L., Jiang X., and Li Q., “Mismatch Repair Deficiency/Microsatellite Instability‐High as a Predictor for Anti‐PD‐1/PD‐L1 Immunotherapy Efficacy,” Journal of Hematology &amp; Oncology
12, no. 1 (2019): 54.
</cite> [<a href="https://doi.org/10.1186/s13045-019-0738-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6544911/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31151482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Hematology%20&amp;%20Oncology&amp;title=Mismatch%20Repair%20Deficiency/Microsatellite%20Instability%E2%80%90High%20as%20a%20Predictor%20for%20Anti%E2%80%90PD%E2%80%901/PD%E2%80%90L1%20Immunotherapy%20Efficacy&amp;author=P.%20Zhao&amp;author=L.%20Li&amp;author=X.%20Jiang&amp;author=Q.%20Li&amp;volume=12&amp;issue=1&amp;publication_year=2019&amp;pages=54&amp;pmid=31151482&amp;doi=10.1186/s13045-019-0738-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0011">
<span class="label">11.</span><cite>
Swanson K., Wu E., Zhang A., Alizadeh A. A., and Zou J., “From Patterns to Patients: Advances in Clinical Machine Learning for Cancer Diagnosis, Prognosis, and Treatment,” Cell
186, no. 8 (2023): 1772–1791.
</cite> [<a href="https://doi.org/10.1016/j.cell.2023.01.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36905928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=From%20Patterns%20to%20Patients:%20Advances%20in%20Clinical%20Machine%20Learning%20for%20Cancer%20Diagnosis,%20Prognosis,%20and%20Treatment&amp;author=K.%20Swanson&amp;author=E.%20Wu&amp;author=A.%20Zhang&amp;author=A.%20A.%20Alizadeh&amp;author=J.%20Zou&amp;volume=186&amp;issue=8&amp;publication_year=2023&amp;pages=1772-1791&amp;pmid=36905928&amp;doi=10.1016/j.cell.2023.01.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0012">
<span class="label">12.</span><cite>
Yang Y., Zhao Y., Liu X., and Huang J., “Artificial Intelligence for Prediction of Response to Cancer Immunotherapy,” Seminars in Cancer Biology
87 (2022): 137–147.
</cite> [<a href="https://doi.org/10.1016/j.semcancer.2022.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36372326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Seminars%20in%20Cancer%20Biology&amp;title=Artificial%20Intelligence%20for%20Prediction%20of%20Response%20to%20Cancer%20Immunotherapy&amp;author=Y.%20Yang&amp;author=Y.%20Zhao&amp;author=X.%20Liu&amp;author=J.%20Huang&amp;volume=87&amp;publication_year=2022&amp;pages=137-147&amp;pmid=36372326&amp;doi=10.1016/j.semcancer.2022.11.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0013">
<span class="label">13.</span><cite>
Wang D. Q., Feng L. Y., Ye J. G., Zou J. G., and Zheng Y. F., “Accelerating the Integration of ChatGPT and Other Large‐Scale AI Models Into Biomedical Research and Healthcare,” MedComm – Future Medicine
2, no. 2 (2023): e43.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=MedComm%20%E2%80%93%20Future%20Medicine&amp;title=Accelerating%20the%20Integration%20of%20ChatGPT%20and%20Other%20Large%E2%80%90Scale%20AI%20Models%20Into%20Biomedical%20Research%20and%20Healthcare&amp;author=D.%20Q.%20Wang&amp;author=L.%20Y.%20Feng&amp;author=J.%20G.%20Ye&amp;author=J.%20G.%20Zou&amp;author=Y.%20F.%20Zheng&amp;volume=2&amp;issue=2&amp;publication_year=2023&amp;pages=e43&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0014">
<span class="label">14.</span><cite>
Sjödahl G., Abrahamsson J., Holmsten K., et al., “Different Responses to Neoadjuvant Chemotherapy in Urothelial Carcinoma Molecular Subtypes,” European Urology
81, no. 5 (2022): 523–532.
</cite> [<a href="https://doi.org/10.1016/j.eururo.2021.10.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34782206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Urology&amp;title=Different%20Responses%20to%20Neoadjuvant%20Chemotherapy%20in%20Urothelial%20Carcinoma%20Molecular%20Subtypes&amp;author=G.%20Sj%C3%B6dahl&amp;author=J.%20Abrahamsson&amp;author=K.%20Holmsten&amp;volume=81&amp;issue=5&amp;publication_year=2022&amp;pages=523-532&amp;pmid=34782206&amp;doi=10.1016/j.eururo.2021.10.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0015">
<span class="label">15.</span><cite>
Kim J., Kwiatkowski D., McConkey D. J., et al., “The Cancer Genome Atlas Expression Subtypes Stratify Response to Checkpoint Inhibition in Advanced Urothelial Cancer and Identify a Subset of Patients With High Survival Probability,” European Urology
75, no. 6 (2019): 961–964.
</cite> [<a href="https://doi.org/10.1016/j.eururo.2019.02.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30851984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Urology&amp;title=The%20Cancer%20Genome%20Atlas%20Expression%20Subtypes%20Stratify%20Response%20to%20Checkpoint%20Inhibition%20in%20Advanced%20Urothelial%20Cancer%20and%20Identify%20a%20Subset%20of%20Patients%20With%20High%20Survival%20Probability&amp;author=J.%20Kim&amp;author=D.%20Kwiatkowski&amp;author=D.%20J.%20McConkey&amp;volume=75&amp;issue=6&amp;publication_year=2019&amp;pages=961-964&amp;pmid=30851984&amp;doi=10.1016/j.eururo.2019.02.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0016">
<span class="label">16.</span><cite>
Erlmeier F., Klümper N., Landgraf L., et al., “Spatial Immunephenotypes of Distant Metastases but Not Matched Primary Urothelial Carcinomas Predict Response to Immune Checkpoint Inhibition,” European Urology
83, no. 2 (2023): 133–142.
</cite> [<a href="https://doi.org/10.1016/j.eururo.2022.10.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36372626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Urology&amp;title=Spatial%20Immunephenotypes%20of%20Distant%20Metastases%20but%20Not%20Matched%20Primary%20Urothelial%20Carcinomas%20Predict%20Response%20to%20Immune%20Checkpoint%20Inhibition&amp;author=F.%20Erlmeier&amp;author=N.%20Kl%C3%BCmper&amp;author=L.%20Landgraf&amp;volume=83&amp;issue=2&amp;publication_year=2023&amp;pages=133-142&amp;pmid=36372626&amp;doi=10.1016/j.eururo.2022.10.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0017">
<span class="label">17.</span><cite>
Deo R. C., “Machine Learning in Medicine,” Circulation
132, no. 20 (2015): 1920–1930.
</cite> [<a href="https://doi.org/10.1161/CIRCULATIONAHA.115.001593" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5831252/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26572668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Machine%20Learning%20in%20Medicine&amp;author=R.%20C.%20Deo&amp;volume=132&amp;issue=20&amp;publication_year=2015&amp;pages=1920-1930&amp;pmid=26572668&amp;doi=10.1161/CIRCULATIONAHA.115.001593&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0018">
<span class="label">18.</span><cite>
Chen C., Liao C., and Liu Y.‐Y., “Teasing Out Missing Reactions in Genome‐Scale Metabolic Networks Through Hypergraph Learning,” Nature Communications
14, no. 1 (2023): 2375.</cite> [<a href="https://doi.org/10.1038/s41467-023-38110-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10130184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37185345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Teasing%20Out%20Missing%20Reactions%20in%20Genome%E2%80%90Scale%20Metabolic%20Networks%20Through%20Hypergraph%20Learning&amp;author=C.%20Chen&amp;author=C.%20Liao&amp;author=Y.%E2%80%90Y.%20Liu&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=2375&amp;pmid=37185345&amp;doi=10.1038/s41467-023-38110-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0019">
<span class="label">19.</span><cite>
Steyaert S., Pizurica M., Nagaraj D., et al., “Multimodal Data Fusion for Cancer Biomarker Discovery With Deep Learning,” Nature Machine Intelligence
5, no. 4 (2023): 351–362.</cite> [<a href="https://doi.org/10.1038/s42256-023-00633-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10484010/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37693852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Machine%20Intelligence&amp;title=Multimodal%20Data%20Fusion%20for%20Cancer%20Biomarker%20Discovery%20With%20Deep%20Learning&amp;author=S.%20Steyaert&amp;author=M.%20Pizurica&amp;author=D.%20Nagaraj&amp;volume=5&amp;issue=4&amp;publication_year=2023&amp;pages=351-362&amp;pmid=37693852&amp;doi=10.1038/s42256-023-00633-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0020">
<span class="label">20.</span><cite>
Sorin M., Rezanejad M., Karimi E., et al., “Single‐Cell Spatial Landscapes of the Lung Tumour Immune Microenvironment,” Nature
614, no. 7948 (2023): 548–554.
</cite> [<a href="https://doi.org/10.1038/s41586-022-05672-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9931585/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36725934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Single%E2%80%90Cell%20Spatial%20Landscapes%20of%20the%20Lung%20Tumour%20Immune%20Microenvironment&amp;author=M.%20Sorin&amp;author=M.%20Rezanejad&amp;author=E.%20Karimi&amp;volume=614&amp;issue=7948&amp;publication_year=2023&amp;pages=548-554&amp;pmid=36725934&amp;doi=10.1038/s41586-022-05672-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0021">
<span class="label">21.</span><cite>
Wang B., Zhao Q., Zhang Y., et al., “Targeting Hypoxia in the Tumor Microenvironment: A Potential Strategy to Improve Cancer Immunotherapy,” Journal of Experimental &amp; Clinical Cancer Research
40, no. 1 (2021): 24.
</cite> [<a href="https://doi.org/10.1186/s13046-020-01820-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7796640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33422072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20&amp;%20Clinical%20Cancer%20Research&amp;title=Targeting%20Hypoxia%20in%20the%20Tumor%20Microenvironment:%20A%20Potential%20Strategy%20to%20Improve%20Cancer%20Immunotherapy&amp;author=B.%20Wang&amp;author=Q.%20Zhao&amp;author=Y.%20Zhang&amp;volume=40&amp;issue=1&amp;publication_year=2021&amp;pages=24&amp;pmid=33422072&amp;doi=10.1186/s13046-020-01820-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0022">
<span class="label">22.</span><cite>
Apte R. S., Chen D. S., and Ferrara N., “VEGF in Signaling and Disease: Beyond Discovery and Development,” Cell
176, no. 6 (2019): 1248–1264.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.01.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6410740/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30849371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=VEGF%20in%20Signaling%20and%20Disease:%20Beyond%20Discovery%20and%20Development&amp;author=R.%20S.%20Apte&amp;author=D.%20S.%20Chen&amp;author=N.%20Ferrara&amp;volume=176&amp;issue=6&amp;publication_year=2019&amp;pages=1248-1264&amp;pmid=30849371&amp;doi=10.1016/j.cell.2019.01.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0023">
<span class="label">23.</span><cite>
Zhang Y., Gao C., Cao F., et al., “Pan‐Cancer Analysis of IGF‐1 and IGF‐1R as Potential Prognostic Biomarkers and Immunotherapy Targets,” Frontiers in Oncology
11 (2021): 755341.
</cite> [<a href="https://doi.org/10.3389/fonc.2021.755341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8602838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34804946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Oncology&amp;title=Pan%E2%80%90Cancer%20Analysis%20of%20IGF%E2%80%901%20and%20IGF%E2%80%901R%20as%20Potential%20Prognostic%20Biomarkers%20and%20Immunotherapy%20Targets&amp;author=Y.%20Zhang&amp;author=C.%20Gao&amp;author=F.%20Cao&amp;volume=11&amp;publication_year=2021&amp;pages=755341&amp;pmid=34804946&amp;doi=10.3389/fonc.2021.755341&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0024">
<span class="label">24.</span><cite>
Turner N., and Grose R., “Fibroblast Growth Factor Signalling: From Development to Cancer,” Nature Reviews Cancer
10, no. 2 (2010): 116–129.
</cite> [<a href="https://doi.org/10.1038/nrc2780" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20094046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Cancer&amp;title=Fibroblast%20Growth%20Factor%20Signalling:%20From%20Development%20to%20Cancer&amp;author=N.%20Turner&amp;author=R.%20Grose&amp;volume=10&amp;issue=2&amp;publication_year=2010&amp;pages=116-129&amp;pmid=20094046&amp;doi=10.1038/nrc2780&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0025">
<span class="label">25.</span><cite>
Guo Q., Jin Y., Chen X., et al., “NF‐κB in Biology and Targeted Therapy: New Insights and Translational Implications,” Signal Transduction and Targeted Therapy
9, no. 1 (2024): 53.
</cite> [<a href="https://doi.org/10.1038/s41392-024-01757-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10910037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38433280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20and%20Targeted%20Therapy&amp;title=NF%E2%80%90%CE%BAB%20in%20Biology%20and%20Targeted%20Therapy:%20New%20Insights%20and%20Translational%20Implications&amp;author=Q.%20Guo&amp;author=Y.%20Jin&amp;author=X.%20Chen&amp;volume=9&amp;issue=1&amp;publication_year=2024&amp;pages=53&amp;pmid=38433280&amp;doi=10.1038/s41392-024-01757-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0026">
<span class="label">26.</span><cite>
Bonavida B., and Garban H., “Nitric Oxide‐Mediated Sensitization of Resistant Tumor Cells to Apoptosis by Chemo‐Immunotherapeutics,” Redox Biology
6 (2015): 486–494.
</cite> [<a href="https://doi.org/10.1016/j.redox.2015.08.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4596920/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26432660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Redox%20Biology&amp;title=Nitric%20Oxide%E2%80%90Mediated%20Sensitization%20of%20Resistant%20Tumor%20Cells%20to%20Apoptosis%20by%20Chemo%E2%80%90Immunotherapeutics&amp;author=B.%20Bonavida&amp;author=H.%20Garban&amp;volume=6&amp;publication_year=2015&amp;pages=486-494&amp;pmid=26432660&amp;doi=10.1016/j.redox.2015.08.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0027">
<span class="label">27.</span><cite>
Jassem J., de Marinis F., Giaccone G., et al., “Updated Overall Survival Analysis From IMpower110: Atezolizumab Versus Platinum‐Based Chemotherapy in Treatment‐Naive Programmed Death‐Ligand 1–Selected NSCLC,” Journal of Thoracic Oncology
16, no. 11 (2021): 1872–1882.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2021.06.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Thoracic%20Oncology&amp;title=Updated%20Overall%20Survival%20Analysis%20From%20IMpower110:%20Atezolizumab%20Versus%20Platinum%E2%80%90Based%20Chemotherapy%20in%20Treatment%E2%80%90Naive%20Programmed%20Death%E2%80%90Ligand%201%E2%80%93Selected%20NSCLC&amp;author=J.%20Jassem&amp;author=F.%20de%20Marinis&amp;author=G.%20Giaccone&amp;volume=16&amp;issue=11&amp;publication_year=2021&amp;pages=1872-1882&amp;pmid=34265434&amp;doi=10.1016/j.jtho.2021.06.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0028">
<span class="label">28.</span><cite>
Li Y., Liang L., Dai W., et al., “Prognostic Impact of Programed Cell Death‐1 (PD‐1) and PD‐Ligand 1 (PD‐L1) Expression in Cancer Cells and Tumor Infiltrating Lymphocytes in Colorectal Cancer,” Molecular Cancer
15, no. 1 (2016): 55.
</cite> [<a href="https://doi.org/10.1186/s12943-016-0539-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4995750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27552968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer&amp;title=Prognostic%20Impact%20of%20Programed%20Cell%20Death%E2%80%901%20(PD%E2%80%901)%20and%20PD%E2%80%90Ligand%201%20(PD%E2%80%90L1)%20Expression%20in%20Cancer%20Cells%20and%20Tumor%20Infiltrating%20Lymphocytes%20in%20Colorectal%20Cancer&amp;author=Y.%20Li&amp;author=L.%20Liang&amp;author=W.%20Dai&amp;volume=15&amp;issue=1&amp;publication_year=2016&amp;pages=55&amp;pmid=27552968&amp;doi=10.1186/s12943-016-0539-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0029">
<span class="label">29.</span><cite>
Powles T., Durán I., van der Heijden M. S., et al., “Atezolizumab Versus Chemotherapy in Patients With Platinum‐Treated Locally Advanced or Metastatic Urothelial Carcinoma (IMvigor211): A Multicentre, Open‐Label, Phase 3 Randomised Controlled Trial,” Lancet (London, England)
391, no. 10122 (2018): 748–757.
</cite> [<a href="https://doi.org/10.1016/S0140-6736(17)33297-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29268948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet%20(London,%20England)&amp;title=Atezolizumab%20Versus%20Chemotherapy%20in%20Patients%20With%20Platinum%E2%80%90Treated%20Locally%20Advanced%20or%20Metastatic%20Urothelial%20Carcinoma%20(IMvigor211):%20A%20Multicentre,%20Open%E2%80%90Label,%20Phase%203%20Randomised%20Controlled%20Trial&amp;author=T.%20Powles&amp;author=I.%20Dur%C3%A1n&amp;author=M.%20S.%20van%20der%20Heijden&amp;volume=391&amp;issue=10122&amp;publication_year=2018&amp;pages=748-757&amp;pmid=29268948&amp;doi=10.1016/S0140-6736(17)33297-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0030">
<span class="label">30.</span><cite>
Fontanini G., Vignati S., Basolo F., et al., “Angiogenesis as a Prognostic Indicator of Survival in Non‐Small‐Cell Lung Carcinoma: A Prospective Study,” JNCI: Journal of the National Cancer Institute
89, no. 12 (1997): 881–886.
</cite> [<a href="https://doi.org/10.1093/jnci/89.12.881" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9196255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JNCI:%20Journal%20of%20the%20National%20Cancer%20Institute&amp;title=Angiogenesis%20as%20a%20Prognostic%20Indicator%20of%20Survival%20in%20Non%E2%80%90Small%E2%80%90Cell%20Lung%20Carcinoma:%20A%20Prospective%20Study&amp;author=G.%20Fontanini&amp;author=S.%20Vignati&amp;author=F.%20Basolo&amp;volume=89&amp;issue=12&amp;publication_year=1997&amp;pages=881-886&amp;pmid=9196255&amp;doi=10.1093/jnci/89.12.881&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0031">
<span class="label">31.</span><cite>
Robertson A. G., Kim J., Al‐Ahmadie H., et al., “Comprehensive Molecular Characterization of Muscle‐Invasive Bladder Cancer,” Cell
171, no. 3 (2017): 540–556. e525.
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.09.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5687509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28988769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Comprehensive%20Molecular%20Characterization%20of%20Muscle%E2%80%90Invasive%20Bladder%20Cancer&amp;author=A.%20G.%20Robertson&amp;author=J.%20Kim&amp;author=H.%20Al%E2%80%90Ahmadie&amp;volume=171&amp;issue=3&amp;publication_year=2017&amp;pages=540-556&amp;pmid=28988769&amp;doi=10.1016/j.cell.2017.09.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0032">
<span class="label">32.</span><cite>
Khan K. A., and Kerbel R. S., “Improving Immunotherapy Outcomes With Anti‐Angiogenic Treatments and Vice Versa,” Nature Reviews Clinical Oncology
15, no. 5 (2018): 310–324.</cite> [<a href="https://doi.org/10.1038/nrclinonc.2018.9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29434333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Clinical%20Oncology&amp;title=Improving%20Immunotherapy%20Outcomes%20With%20Anti%E2%80%90Angiogenic%20Treatments%20and%20Vice%20Versa&amp;author=K.%20A.%20Khan&amp;author=R.%20S.%20Kerbel&amp;volume=15&amp;issue=5&amp;publication_year=2018&amp;pages=310-324&amp;pmid=29434333&amp;doi=10.1038/nrclinonc.2018.9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0033">
<span class="label">33.</span><cite>
Wu Q., Tian A.‐L., Li B., et al., “IGF1 Receptor Inhibition Amplifies the Effects of Cancer Drugs by Autophagy and Immune‐Dependent Mechanisms,” Journal for ImmunoTherapy of Cancer
9, no. 6 (2021): e002722.
</cite> [<a href="https://doi.org/10.1136/jitc-2021-002722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8204183/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34127545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20for%20ImmunoTherapy%20of%20Cancer&amp;title=IGF1%20Receptor%20Inhibition%20Amplifies%20the%20Effects%20of%20Cancer%20Drugs%20by%20Autophagy%20and%20Immune%E2%80%90Dependent%20Mechanisms&amp;author=Q.%20Wu&amp;author=A.%E2%80%90L.%20Tian&amp;author=B.%20Li&amp;volume=9&amp;issue=6&amp;publication_year=2021&amp;pages=e002722&amp;pmid=34127545&amp;doi=10.1136/jitc-2021-002722&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0034">
<span class="label">34.</span><cite>
Ruan R., Li L., Li X., et al., “Unleashing the Potential of Combining FGFR Inhibitor and Immune Checkpoint Blockade for FGF/FGFR Signaling in Tumor Microenvironment,” Molecular Cancer
22, no. 1 (2023): 60.
</cite> [<a href="https://doi.org/10.1186/s12943-023-01761-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10039534/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36966334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer&amp;title=Unleashing%20the%20Potential%20of%20Combining%20FGFR%20Inhibitor%20and%20Immune%20Checkpoint%20Blockade%20for%20FGF/FGFR%20Signaling%20in%20Tumor%20Microenvironment&amp;author=R.%20Ruan&amp;author=L.%20Li&amp;author=X.%20Li&amp;volume=22&amp;issue=1&amp;publication_year=2023&amp;pages=60&amp;pmid=36966334&amp;doi=10.1186/s12943-023-01761-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0035">
<span class="label">35.</span><cite>
Sautès‐Fridman C., Petitprez F., Calderaro J., and Fridman W. H., “Tertiary Lymphoid Structures in the Era of Cancer Immunotherapy,” Nature Reviews Cancer
19, no. 6 (2019): 307–325.
</cite> [<a href="https://doi.org/10.1038/s41568-019-0144-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31092904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Cancer&amp;title=Tertiary%20Lymphoid%20Structures%20in%20the%20Era%20of%20Cancer%20Immunotherapy&amp;author=C.%20Saut%C3%A8s%E2%80%90Fridman&amp;author=F.%20Petitprez&amp;author=J.%20Calderaro&amp;author=W.%20H.%20Fridman&amp;volume=19&amp;issue=6&amp;publication_year=2019&amp;pages=307-325&amp;pmid=31092904&amp;doi=10.1038/s41568-019-0144-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0036">
<span class="label">36.</span><cite>
Schumacher T. N., and Thommen D. S., “Tertiary Lymphoid Structures in Cancer,” Science
375, no. 6576 (2022): eabf9419.
</cite> [<a href="https://doi.org/10.1126/science.abf9419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34990248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Tertiary%20Lymphoid%20Structures%20in%20Cancer&amp;author=T.%20N.%20Schumacher&amp;author=D.%20S.%20Thommen&amp;volume=375&amp;issue=6576&amp;publication_year=2022&amp;pages=eabf9419&amp;pmid=34990248&amp;doi=10.1126/science.abf9419&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0037">
<span class="label">37.</span><cite>
Gouin K. H., Ing N., Plummer J. T., et al., “An N‐Cadherin 2 Expressing Epithelial Cell Subpopulation Predicts Response to Surgery, Chemotherapy and Immunotherapy in Bladder Cancer,” Nature Communications
12, no. 1 (2021): 4906.</cite> [<a href="https://doi.org/10.1038/s41467-021-25103-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8361097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34385456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=An%20N%E2%80%90Cadherin%202%20Expressing%20Epithelial%20Cell%20Subpopulation%20Predicts%20Response%20to%20Surgery,%20Chemotherapy%20and%20Immunotherapy%20in%20Bladder%20Cancer&amp;author=K.%20H.%20Gouin&amp;author=N.%20Ing&amp;author=J.%20T.%20Plummer&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=4906&amp;pmid=34385456&amp;doi=10.1038/s41467-021-25103-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0038">
<span class="label">38.</span><cite>
Su H., and Karin M., “Collagen Architecture and Signaling Orchestrate Cancer Development,” Trends in Cancer
9, no. 9 (2023): 764–773.
</cite> [<a href="https://doi.org/10.1016/j.trecan.2023.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37400314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Cancer&amp;title=Collagen%20Architecture%20and%20Signaling%20Orchestrate%20Cancer%20Development&amp;author=H.%20Su&amp;author=M.%20Karin&amp;volume=9&amp;issue=9&amp;publication_year=2023&amp;pages=764-773&amp;pmid=37400314&amp;doi=10.1016/j.trecan.2023.06.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0039">
<span class="label">39.</span><cite>
Kim T. K., Vandsemb E. N., Herbst R. S., and Chen L., “Adaptive Immune Resistance at the Tumour Site: Mechanisms and Therapeutic Opportunities,” Nature Reviews Drug Discovery
21, no. 7 (2022): 529–540.
</cite> [<a href="https://doi.org/10.1038/s41573-022-00493-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35701637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Drug%20Discovery&amp;title=Adaptive%20Immune%20Resistance%20at%20the%20Tumour%20Site:%20Mechanisms%20and%20Therapeutic%20Opportunities&amp;author=T.%20K.%20Kim&amp;author=E.%20N.%20Vandsemb&amp;author=R.%20S.%20Herbst&amp;author=L.%20Chen&amp;volume=21&amp;issue=7&amp;publication_year=2022&amp;pages=529-540&amp;pmid=35701637&amp;doi=10.1038/s41573-022-00493-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0040">
<span class="label">40.</span><cite>
Aggarwal B. B., Gupta S. C., and Kim J. H., “Historical Perspectives on Tumor Necrosis Factor and Its Superfamily: 25 Years Later, a Golden Journey,” Blood
119, no. 3 (2012): 651–665.
</cite> [<a href="https://doi.org/10.1182/blood-2011-04-325225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3265196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22053109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Historical%20Perspectives%20on%20Tumor%20Necrosis%20Factor%20and%20Its%20Superfamily:%2025%20Years%20Later,%20a%20Golden%20Journey&amp;author=B.%20B.%20Aggarwal&amp;author=S.%20C.%20Gupta&amp;author=J.%20H.%20Kim&amp;volume=119&amp;issue=3&amp;publication_year=2012&amp;pages=651-665&amp;pmid=22053109&amp;doi=10.1182/blood-2011-04-325225&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0041">
<span class="label">41.</span><cite>
Wu L., Jin Y., Zhao X., et al., “Tumor Aerobic Glycolysis Confers Immune Evasion Through Modulating Sensitivity to T Cell‐Mediated Bystander Killing via TNF‐α,” Cell Metabolism
35, no. 9 (2023): 1580–1596. e1589.
</cite> [<a href="https://doi.org/10.1016/j.cmet.2023.07.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37506695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metabolism&amp;title=Tumor%20Aerobic%20Glycolysis%20Confers%20Immune%20Evasion%20Through%20Modulating%20Sensitivity%20to%20T%20Cell%E2%80%90Mediated%20Bystander%20Killing%20via%20TNF%E2%80%90%CE%B1&amp;author=L.%20Wu&amp;author=Y.%20Jin&amp;author=X.%20Zhao&amp;volume=35&amp;issue=9&amp;publication_year=2023&amp;pages=1580-1596&amp;pmid=37506695&amp;doi=10.1016/j.cmet.2023.07.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0042">
<span class="label">42.</span><cite>
Jing Y., Sun K., Liu W., et al., “Tumor Necrosis Factor‐α Promotes Hepatocellular Carcinogenesis Through the Activation of Hepatic Progenitor Cells,” Cancer Letters
434 (2018): 22–32.
</cite> [<a href="https://doi.org/10.1016/j.canlet.2018.07.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29981431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Letters&amp;title=Tumor%20Necrosis%20Factor%E2%80%90%CE%B1%20Promotes%20Hepatocellular%20Carcinogenesis%20Through%20the%20Activation%20of%20Hepatic%20Progenitor%20Cells&amp;author=Y.%20Jing&amp;author=K.%20Sun&amp;author=W.%20Liu&amp;volume=434&amp;publication_year=2018&amp;pages=22-32&amp;pmid=29981431&amp;doi=10.1016/j.canlet.2018.07.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0043">
<span class="label">43.</span><cite>
Tan W., Luo X., Li W., et al., “TNF‐α Is a Potential Therapeutic Target to Overcome Sorafenib Resistance in Hepatocellular Carcinoma,” EBioMedicine
40 (2019): 446–456.
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2018.12.047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6412090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30594557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EBioMedicine&amp;title=TNF%E2%80%90%CE%B1%20Is%20a%20Potential%20Therapeutic%20Target%20to%20Overcome%20Sorafenib%20Resistance%20in%20Hepatocellular%20Carcinoma&amp;author=W.%20Tan&amp;author=X.%20Luo&amp;author=W.%20Li&amp;volume=40&amp;publication_year=2019&amp;pages=446-456&amp;pmid=30594557&amp;doi=10.1016/j.ebiom.2018.12.047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0044">
<span class="label">44.</span><cite>
Shinde A., Chandak N., Singh J., et al., “TNF‐α Induced NF‐κB Mediated LYRM7 Expression Modulates the Tumor Growth and Metastatic Ability in Breast Cancer,” Free Radical Biology and Medicine
211 (2024): 158–170.
</cite> [<a href="https://doi.org/10.1016/j.freeradbiomed.2023.12.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38104742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Free%20Radical%20Biology%20and%20Medicine&amp;title=TNF%E2%80%90%CE%B1%20Induced%20NF%E2%80%90%CE%BAB%20Mediated%20LYRM7%20Expression%20Modulates%20the%20Tumor%20Growth%20and%20Metastatic%20Ability%20in%20Breast%20Cancer&amp;author=A.%20Shinde&amp;author=N.%20Chandak&amp;author=J.%20Singh&amp;volume=211&amp;publication_year=2024&amp;pages=158-170&amp;pmid=38104742&amp;doi=10.1016/j.freeradbiomed.2023.12.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0045">
<span class="label">45.</span><cite>
Liu W., Lu X., Shi P., et al., “TNF‐α Increases Breast Cancer Stem‐Like Cells Through Up‐Regulating TAZ Expression via the Non‐Canonical NF‐κB Pathway,” Scientific Reports
10, no. 1 (2020): 1804.
</cite> [<a href="https://doi.org/10.1038/s41598-020-58642-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7000832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32019974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=TNF%E2%80%90%CE%B1%20Increases%20Breast%20Cancer%20Stem%E2%80%90Like%20Cells%20Through%20Up%E2%80%90Regulating%20TAZ%20Expression%20via%20the%20Non%E2%80%90Canonical%20NF%E2%80%90%CE%BAB%20Pathway&amp;author=W.%20Liu&amp;author=X.%20Lu&amp;author=P.%20Shi&amp;volume=10&amp;issue=1&amp;publication_year=2020&amp;pages=1804&amp;pmid=32019974&amp;doi=10.1038/s41598-020-58642-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0046">
<span class="label">46.</span><cite>
Shinde A., Tang X., Singh R., and Brindley D. N., “Infliximab, a Monoclonal Antibody Against TNF‐α, Inhibits NF‐κB Activation, Autotaxin Expression and Breast Cancer Metastasis to Lungs,” Cancers
16, no. 1 (2023): 52.
</cite> [<a href="https://doi.org/10.3390/cancers16010052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10778319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38201482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=Infliximab,%20a%20Monoclonal%20Antibody%20Against%20TNF%E2%80%90%CE%B1,%20Inhibits%20NF%E2%80%90%CE%BAB%20Activation,%20Autotaxin%20Expression%20and%20Breast%20Cancer%20Metastasis%20to%20Lungs&amp;author=A.%20Shinde&amp;author=X.%20Tang&amp;author=R.%20Singh&amp;author=D.%20N.%20Brindley&amp;volume=16&amp;issue=1&amp;publication_year=2023&amp;pages=52&amp;pmid=38201482&amp;doi=10.3390/cancers16010052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0047">
<span class="label">47.</span><cite>
Liu B.‐X., Xie Y., Zhang J., et al., “SERPINB5 Promotes Colorectal Cancer Invasion and Migration by Promoting EMT and Angiogenesis via the TNF‐α/NF‐κB Pathway,” International Immunopharmacology
131 (2024): 111759.
</cite> [<a href="https://doi.org/10.1016/j.intimp.2024.111759" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38460302/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Immunopharmacology&amp;title=SERPINB5%20Promotes%20Colorectal%20Cancer%20Invasion%20and%20Migration%20by%20Promoting%20EMT%20and%20Angiogenesis%20via%20the%20TNF%E2%80%90%CE%B1/NF%E2%80%90%CE%BAB%20Pathway&amp;author=B.%E2%80%90X.%20Liu&amp;author=Y.%20Xie&amp;author=J.%20Zhang&amp;volume=131&amp;publication_year=2024&amp;pages=111759&amp;pmid=38460302&amp;doi=10.1016/j.intimp.2024.111759&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0048">
<span class="label">48.</span><cite>
Liang Y., Tan Y., Guan B., et al., “Single‐Cell Atlases Link Macrophages and CD8+ T‐Cell Subpopulations to Disease Progression and Immunotherapy Response in Urothelial Carcinoma,” Theranostics
12, no. 18 (2022): 7745–7759.
</cite> [<a href="https://doi.org/10.7150/thno.77281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9706581/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36451860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theranostics&amp;title=Single%E2%80%90Cell%20Atlases%20Link%20Macrophages%20and%20CD8+%20T%E2%80%90Cell%20Subpopulations%20to%20Disease%20Progression%20and%20Immunotherapy%20Response%20in%20Urothelial%20Carcinoma&amp;author=Y.%20Liang&amp;author=Y.%20Tan&amp;author=B.%20Guan&amp;volume=12&amp;issue=18&amp;publication_year=2022&amp;pages=7745-7759&amp;pmid=36451860&amp;doi=10.7150/thno.77281&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0049">
<span class="label">49.</span><cite>
Marini J. C., Forlino A., Bächinger H. P., et al., “Osteogenesis Imperfecta,” Nature Reviews Disease Primers
3, no. 1 (2017): 17052.</cite> [<a href="https://doi.org/10.1038/nrdp.2017.52" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28820180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Disease%20Primers&amp;title=Osteogenesis%20Imperfecta&amp;author=J.%20C.%20Marini&amp;author=A.%20Forlino&amp;author=H.%20P.%20B%C3%A4chinger&amp;volume=3&amp;issue=1&amp;publication_year=2017&amp;pages=17052&amp;pmid=28820180&amp;doi=10.1038/nrdp.2017.52&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0050">
<span class="label">50.</span><cite>
Dangaj D., Bruand M., Grimm A. J., et al., “Cooperation Between Constitutive and Inducible Chemokines Enables T Cell Engraftment and Immune Attack in Solid Tumors,” Cancer Cell
35, no. 6 (2019): 885–900. e810.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2019.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6961655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31185212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Cooperation%20Between%20Constitutive%20and%20Inducible%20Chemokines%20Enables%20T%20Cell%20Engraftment%20and%20Immune%20Attack%20in%20Solid%20Tumors&amp;author=D.%20Dangaj&amp;author=M.%20Bruand&amp;author=A.%20J.%20Grimm&amp;volume=35&amp;issue=6&amp;publication_year=2019&amp;pages=885-900&amp;pmid=31185212&amp;doi=10.1016/j.ccell.2019.05.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0051">
<span class="label">51.</span><cite>
Dong H., Strome S. E., Salomao D. R., et al., “Tumor‐Associated B7‐H1 Promotes T‐Cell Apoptosis: A Potential Mechanism of Immune Evasion,” Nature Medicine
8, no. 8 (2002): 793–800.</cite> [<a href="https://doi.org/10.1038/nm730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12091876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&amp;title=Tumor%E2%80%90Associated%20B7%E2%80%90H1%20Promotes%20T%E2%80%90Cell%20Apoptosis:%20A%20Potential%20Mechanism%20of%20Immune%20Evasion&amp;author=H.%20Dong&amp;author=S.%20E.%20Strome&amp;author=D.%20R.%20Salomao&amp;volume=8&amp;issue=8&amp;publication_year=2002&amp;pages=793-800&amp;pmid=12091876&amp;doi=10.1038/nm730&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0052">
<span class="label">52.</span><cite>
Chen L., “Co‐inhibitory Molecules of the B7‐CD28 Family in the Control of T‐Cell Immunity,” Nature Reviews Immunology
4, no. 5 (2004): 336–347.</cite> [<a href="https://doi.org/10.1038/nri1349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15122199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Immunology&amp;title=Co%E2%80%90inhibitory%20Molecules%20of%20the%20B7%E2%80%90CD28%20Family%20in%20the%20Control%20of%20T%E2%80%90Cell%20Immunity&amp;author=L.%20Chen&amp;volume=4&amp;issue=5&amp;publication_year=2004&amp;pages=336-347&amp;pmid=15122199&amp;doi=10.1038/nri1349&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0053">
<span class="label">53.</span><cite>
Zou W., and Chen L., “Inhibitory B7‐Family Molecules in the Tumour Microenvironment,” Nature Reviews Immunology
8, no. 6 (2008): 467–477.</cite> [<a href="https://doi.org/10.1038/nri2326" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18500231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Immunology&amp;title=Inhibitory%20B7%E2%80%90Family%20Molecules%20in%20the%20Tumour%20Microenvironment&amp;author=W.%20Zou&amp;author=L.%20Chen&amp;volume=8&amp;issue=6&amp;publication_year=2008&amp;pages=467-477&amp;pmid=18500231&amp;doi=10.1038/nri2326&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0054">
<span class="label">54.</span><cite>
Rini B. I., Plimack E. R., Stus V., et al., “Pembrolizumab Plus Axitinib Versus Sunitinib for Advanced Renal‐Cell Carcinoma,” New England Journal of Medicine
380, no. 12 (2019): 1116–1127.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1816714" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30779529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&amp;title=Pembrolizumab%20Plus%20Axitinib%20Versus%20Sunitinib%20for%20Advanced%20Renal%E2%80%90Cell%20Carcinoma&amp;author=B.%20I.%20Rini&amp;author=E.%20R.%20Plimack&amp;author=V.%20Stus&amp;volume=380&amp;issue=12&amp;publication_year=2019&amp;pages=1116-1127&amp;pmid=30779529&amp;doi=10.1056/NEJMoa1816714&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0055">
<span class="label">55.</span><cite>
Liang Y., Tang H., Guo J., et al., “Targeting IFNα to Tumor by Anti‐PD‐L1 Creates Feedforward Antitumor Responses to Overcome Checkpoint Blockade Resistance,” Nature Communications
9, no. 1 (2018): 4586.</cite> [<a href="https://doi.org/10.1038/s41467-018-06890-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6214895/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30389912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Targeting%20IFN%CE%B1%20to%20Tumor%20by%20Anti%E2%80%90PD%E2%80%90L1%20Creates%20Feedforward%20Antitumor%20Responses%20to%20Overcome%20Checkpoint%20Blockade%20Resistance&amp;author=Y.%20Liang&amp;author=H.%20Tang&amp;author=J.%20Guo&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=4586&amp;pmid=30389912&amp;doi=10.1038/s41467-018-06890-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0056">
<span class="label">56.</span><cite>
Ager C. R., Reilley M. J., Nicholas C., Bartkowiak T., Jaiswal A. R., and Curran M. A., “Intratumoral STING Activation With T‐Cell Checkpoint Modulation Generates Systemic Antitumor Immunity,” Cancer Immunology Research
5, no. 8 (2017): 676–684.
</cite> [<a href="https://doi.org/10.1158/2326-6066.CIR-17-0049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5547907/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28674082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Immunology%20Research&amp;title=Intratumoral%20STING%20Activation%20With%20T%E2%80%90Cell%20Checkpoint%20Modulation%20Generates%20Systemic%20Antitumor%20Immunity&amp;author=C.%20R.%20Ager&amp;author=M.%20J.%20Reilley&amp;author=C.%20Nicholas&amp;author=T.%20Bartkowiak&amp;author=A.%20R.%20Jaiswal&amp;volume=5&amp;issue=8&amp;publication_year=2017&amp;pages=676-684&amp;pmid=28674082&amp;doi=10.1158/2326-6066.CIR-17-0049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0057">
<span class="label">57.</span><cite>
Allen B. M., Hiam K. J., Burnett C. E., et al., “Systemic Dysfunction and Plasticity of the Immune Macroenvironment in Cancer Models,” Nature Medicine
26, no. 7 (2020): 1125–1134.</cite> [<a href="https://doi.org/10.1038/s41591-020-0892-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7384250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32451499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Medicine&amp;title=Systemic%20Dysfunction%20and%20Plasticity%20of%20the%20Immune%20Macroenvironment%20in%20Cancer%20Models&amp;author=B.%20M.%20Allen&amp;author=K.%20J.%20Hiam&amp;author=C.%20E.%20Burnett&amp;volume=26&amp;issue=7&amp;publication_year=2020&amp;pages=1125-1134&amp;pmid=32451499&amp;doi=10.1038/s41591-020-0892-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0058">
<span class="label">58.</span><cite>
Dieterich L. C., and Bikfalvi A., “The Tumor Organismal Environment: Role in Tumor Development and Cancer Immunotherapy,” Seminars in Cancer Biology
65 (2020): 197–206.
</cite> [<a href="https://doi.org/10.1016/j.semcancer.2019.12.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31891779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Seminars%20in%20Cancer%20Biology&amp;title=The%20Tumor%20Organismal%20Environment:%20Role%20in%20Tumor%20Development%20and%20Cancer%20Immunotherapy&amp;author=L.%20C.%20Dieterich&amp;author=A.%20Bikfalvi&amp;volume=65&amp;publication_year=2020&amp;pages=197-206&amp;pmid=31891779&amp;doi=10.1016/j.semcancer.2019.12.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0059">
<span class="label">59.</span><cite>
Sepich‐Poore G. D., Zitvogel L., Straussman R., Hasty J., Wargo J. A., and Knight R., “The Microbiome and human Cancer,” Science
371, no. 6536 (2021): eabc4552.
</cite> [<a href="https://doi.org/10.1126/science.abc4552" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8767999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33766858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20Microbiome%20and%20human%20Cancer&amp;author=G.%20D.%20Sepich%E2%80%90Poore&amp;author=L.%20Zitvogel&amp;author=R.%20Straussman&amp;author=J.%20Hasty&amp;author=J.%20A.%20Wargo&amp;volume=371&amp;issue=6536&amp;publication_year=2021&amp;pages=eabc4552&amp;pmid=33766858&amp;doi=10.1126/science.abc4552&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0060">
<span class="label">60.</span><cite>
Lu M. Y., Chen T. Y., Williamson D. F. K., et al., “AI‐Based Pathology Predicts Origins for Cancers of Unknown Primary,” Nature
594, no. 7861 (2021): 106–110.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03512-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33953404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=AI%E2%80%90Based%20Pathology%20Predicts%20Origins%20for%20Cancers%20of%20Unknown%20Primary&amp;author=M.%20Y.%20Lu&amp;author=T.%20Y.%20Chen&amp;author=D.%20F.%20K.%20Williamson&amp;volume=594&amp;issue=7861&amp;publication_year=2021&amp;pages=106-110&amp;pmid=33953404&amp;doi=10.1038/s41586-021-03512-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0061">
<span class="label">61.</span><cite>
Zhang X., Wang J., Lu J., et al., “Robust Prognostic Subtyping of Muscle‐Invasive Bladder Cancer Revealed by Deep Learning‐Based Multi‐Omics Data Integration,” Frontiers in Oncology
11 (2021): 689626.
</cite> [<a href="https://doi.org/10.3389/fonc.2021.689626" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8378227/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34422643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Oncology&amp;title=Robust%20Prognostic%20Subtyping%20of%20Muscle%E2%80%90Invasive%20Bladder%20Cancer%20Revealed%20by%20Deep%20Learning%E2%80%90Based%20Multi%E2%80%90Omics%20Data%20Integration&amp;author=X.%20Zhang&amp;author=J.%20Wang&amp;author=J.%20Lu&amp;volume=11&amp;publication_year=2021&amp;pages=689626&amp;pmid=34422643&amp;doi=10.3389/fonc.2021.689626&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0062">
<span class="label">62.</span><cite>
Ou Q., Lu Z., Cai G., et al., “Unraveling the Influence of Metabolic Signatures on Immune Dynamics for Predicting Immunotherapy Response and Survival in Cancer,” MedComm – Future Medicine
3, no. 2 (2024): 89.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=MedComm%20%E2%80%93%20Future%20Medicine&amp;title=Unraveling%20the%20Influence%20of%20Metabolic%20Signatures%20on%20Immune%20Dynamics%20for%20Predicting%20Immunotherapy%20Response%20and%20Survival%20in%20Cancer&amp;author=Q.%20Ou&amp;author=Z.%20Lu&amp;author=G.%20Cai&amp;volume=3&amp;issue=2&amp;publication_year=2024&amp;pages=89&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0063">
<span class="label">63.</span><cite>
Lipkova J., Chen R. J., Chen B., et al., “Artificial Intelligence for Multimodal Data Integration in Oncology,” Cancer Cell
40, no. 10 (2022): 1095–1110.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2022.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10655164/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36220072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Artificial%20Intelligence%20for%20Multimodal%20Data%20Integration%20in%20Oncology&amp;author=J.%20Lipkova&amp;author=R.%20J.%20Chen&amp;author=B.%20Chen&amp;volume=40&amp;issue=10&amp;publication_year=2022&amp;pages=1095-1110&amp;pmid=36220072&amp;doi=10.1016/j.ccell.2022.09.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0064">
<span class="label">64.</span><cite>
Prelaj A., Miskovic V., Zanitti M., et al., “Artificial Intelligence for Predictive Biomarker Discovery in Immuno‐Oncology: A Systematic Review,” Annals of Oncology
35, no. 1 (2024): 29–65.
</cite> [<a href="https://doi.org/10.1016/j.annonc.2023.10.125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37879443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Oncology&amp;title=Artificial%20Intelligence%20for%20Predictive%20Biomarker%20Discovery%20in%20Immuno%E2%80%90Oncology:%20A%20Systematic%20Review&amp;author=A.%20Prelaj&amp;author=V.%20Miskovic&amp;author=M.%20Zanitti&amp;volume=35&amp;issue=1&amp;publication_year=2024&amp;pages=29-65&amp;pmid=37879443&amp;doi=10.1016/j.annonc.2023.10.125&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0065">
<span class="label">65.</span><cite>
Liu J., Lichtenberg T., Hoadley K. A., et al., “An Integrated TCGA Pan‐Cancer Clinical Data Resource to Drive High‐Quality Survival Outcome Analytics,” Cell
173, no. 2 (2018): 400–416. e411.
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.02.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6066282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29625055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=An%20Integrated%20TCGA%20Pan%E2%80%90Cancer%20Clinical%20Data%20Resource%20to%20Drive%20High%E2%80%90Quality%20Survival%20Outcome%20Analytics&amp;author=J.%20Liu&amp;author=T.%20Lichtenberg&amp;author=K.%20A.%20Hoadley&amp;volume=173&amp;issue=2&amp;publication_year=2018&amp;pages=400-416&amp;pmid=29625055&amp;doi=10.1016/j.cell.2018.02.052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270324-bib-0066">
<span class="label">66.</span><cite>
Zeng D., Fang Y., Luo P., et al., “IOBR2: Multidimensional Decoding Tumor Microenvironment for Immuno‐Oncology Research,” BioRxiv (2024). 10.1101/2024.01.13.575484.</cite> [<a href="https://doi.org/10.1101/2024.01.13.575484" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11704618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39626665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BioRxiv&amp;title=IOBR2:%20Multidimensional%20Decoding%20Tumor%20Microenvironment%20for%20Immuno%E2%80%90Oncology%20Research&amp;author=D.%20Zeng&amp;author=Y.%20Fang&amp;author=P.%20Luo&amp;publication_year=2024&amp;pmid=39626665&amp;doi=10.1101/2024.01.13.575484&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>
<strong>Figure S1</strong>: A flowchart delineating process of model building and Performance verification.</p>
<p>
<strong>Figure S2</strong>: The predictive performance of multi‐omics biomarkers.</p>
<p>
<strong>Figure S3</strong>: Multivariate Cox proportional hazard analyses of clinical variables and treatments for ctDNA subgroup.</p>
<p>
<strong>Figure S4</strong>: Analysis of TCGA‐subtype in IMvigor010 clinical trial.</p>
<p>
<strong>Figure S5</strong>: Benefit and resistance mechanism of immunotherapy cohort.</p>
<p>
<strong>Figure S6</strong>: TNF‐á and the NF‐êB pathway may lead to resistance in immune‐enriched tumor microenvironments.</p>
<p>
<strong>Figure S7</strong>: Clinical characteristics and enrichment of pathways in different tumor immune microenvironment.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377507/bin/MCO2-6-e70324-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">MCO2-6-e70324-s001.pdf</a><sup> (8.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Supporting File</strong>: mco270324‐sup‐0002‐Table.xlsx</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377507/bin/MCO2-6-e70324-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">MCO2-6-e70324-s002.xlsx</a><sup> (10.5MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All datasets presented in this study can be found in online repositories. The data of IMvigor010 clinical trials (accession number: EGAS00001004997) was available after pass the application at the European Genome‐phenome Archive (EGA). The RNA‐sequencing data of the TCGA‐BLCA dataset were gained from the TCGA data portal (<a href="https://portal.gdc.cancer.gov/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portal.gdc.cancer.gov/</a>), and the clinical data were collected from the research of Liu J et al. [<a href="#mco270324-bib-0065" class="usa-link" aria-describedby="mco270324-bib-0065">65</a>]. The single‐cell RNA‐seq dataset was obtained from the National Genomics Data Center (CNCB; accession number: HRA001867). The data generated in this study are available in the article and its Supporting Information files. The R package is available at <a href="https://github.com/LiaoWJLab/uaiscore" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/LiaoWJLab/uaiscore</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from MedComm are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/mco2.70324"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/MCO2-6-e70324.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (11.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12377507/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12377507/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377507%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377507/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12377507/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12377507/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12377507/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12377507/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12377507/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="CIFbzqfd85WWnwyeJkxlXmS8sDIyrdJZWDWBncGeM15mhnWrLtS1EmdzbJo2hWnQ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
