
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            miRNA biomarkers for prognosis and therapy monitoring in a multi-ethnic cohort with SARS-CoV-2 infection - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A2288AF1E33305A228005C10EA7E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371099/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="miRNA biomarkers for prognosis and therapy monitoring in a multi-ethnic cohort with SARS-CoV-2 infection">
<meta name="citation_author" content="Farah Mustafa">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE">
<meta name="citation_author_institution" content="Zayed Bin Sultan Center for Health Sciences, UAE University, Al Ain, UAE">
<meta name="citation_author_institution" content="ASPIRE Research Institute in Precision Medicine, Abu Dhabi, UAE">
<meta name="citation_author" content="Waqar Ahmad">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE">
<meta name="citation_author" content="Bushra Gull">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE">
<meta name="citation_author" content="Jasmin Baby">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE">
<meta name="citation_author" content="Neena G Panicker">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE">
<meta name="citation_author" content="Thanumol Abdul Khader">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE">
<meta name="citation_author" content="Hala Abdul Baki">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE">
<meta name="citation_author" content="Erum Rehman">
<meta name="citation_author_institution" content="Rashid Hospital Dubai Health, Dubai, UAE">
<meta name="citation_author" content="Asif M Salim">
<meta name="citation_author_institution" content="Rashid Hospital Dubai Health, Dubai, UAE">
<meta name="citation_author" content="Rubina L G Ahmed">
<meta name="citation_author_institution" content="Al Jalila Children’s Specialty Hospital, Dubai, UAE">
<meta name="citation_author" content="Hamda H Khansaheb">
<meta name="citation_author_institution" content="Mohammed Bin Rashid University of Medicine and Health Sciences (MBRU), Dubai, UAE">
<meta name="citation_author_institution" content="Dubai Academic Health Corporation (Dubai Health), Dubai, UAE">
<meta name="citation_author" content="Maya Habous">
<meta name="citation_author_institution" content="Rashid Hospital Dubai Health, Dubai, UAE">
<meta name="citation_author" content="Laila M J A AlDabal">
<meta name="citation_author_institution" content="Rashid Hospital Dubai Health, Dubai, UAE">
<meta name="citation_author" content="Soumeya Jaballah">
<meta name="citation_author_institution" content="Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE">
<meta name="citation_author_institution" content="ASPIRE Research Institute in Precision Medicine, Abu Dhabi, UAE">
<meta name="citation_author" content="Saif S Alqassim">
<meta name="citation_author_institution" content="Mohammed Bin Rashid University of Medicine and Health Sciences (MBRU), Dubai, UAE">
<meta name="citation_author" content="Alawi Alsheikh-Ali">
<meta name="citation_author_institution" content="Mohammed Bin Rashid University of Medicine and Health Sciences (MBRU), Dubai, UAE">
<meta name="citation_author_institution" content="Dubai Academic Health Corporation (Dubai Health), Dubai, UAE">
<meta name="citation_author" content="Tahir A Rizvi">
<meta name="citation_author_institution" content="Zayed Bin Sultan Center for Health Sciences, UAE University, Al Ain, UAE">
<meta name="citation_author_institution" content="ASPIRE Research Institute in Precision Medicine, Abu Dhabi, UAE">
<meta name="citation_author_institution" content="Department of Microbiology &amp; Immunology, College of Medicine and Health Sciences (CMHS), Tawam Hospital Complex, CMHS, UAE University, P.O. Box 15551, Al Ain, UAE">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30815">
<meta name="citation_doi" content="10.1038/s41598-025-15248-6">
<meta name="citation_pmid" content="40841549">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371099/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371099/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371099/pdf/41598_2025_Article_15248.pdf">
<meta name="description" content="This study aimed to identify miRNA-based biomarkers in a multi-ethnic cohort of SARS-CoV-2-infected individuals to enhance preparedness for future variants of concern. A total of 31 healthy controls and 154 infected patients were enrolled, from ...">
<meta name="og:title" content="miRNA biomarkers for prognosis and therapy monitoring in a multi-ethnic cohort with SARS-CoV-2 infection">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="This study aimed to identify miRNA-based biomarkers in a multi-ethnic cohort of SARS-CoV-2-infected individuals to enhance preparedness for future variants of concern. A total of 31 healthy controls and 154 infected patients were enrolled, from ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371099/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371099">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-15248-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_15248.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371099%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371099/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371099/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371099/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 21;15:30815. doi: <a href="https://doi.org/10.1038/s41598-025-15248-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-15248-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>miRNA biomarkers for prognosis and therapy monitoring in a multi-ethnic cohort with SARS-CoV-2 infection</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mustafa%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Farah Mustafa</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Farah Mustafa</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE </div>
<div class="p">
<sup>2</sup>Zayed Bin Sultan Center for Health Sciences, UAE University, Al Ain, UAE </div>
<div class="p">
<sup>3</sup>ASPIRE Research Institute in Precision Medicine, Abu Dhabi, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mustafa%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Farah Mustafa</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahmad%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Waqar Ahmad</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Waqar Ahmad</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahmad%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Waqar Ahmad</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gull%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Bushra Gull</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Bushra Gull</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gull%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bushra Gull</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baby%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jasmin Baby</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jasmin Baby</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baby%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jasmin Baby</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Panicker%20NG%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Neena G Panicker</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Neena G Panicker</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Panicker%20NG%22%5BAuthor%5D" class="usa-link"><span class="name western">Neena G Panicker</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khader%20TA%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Thanumol Abdul Khader</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Thanumol Abdul Khader</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khader%20TA%22%5BAuthor%5D" class="usa-link"><span class="name western">Thanumol Abdul Khader</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baki%20HA%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Hala Abdul Baki</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Hala Abdul Baki</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Baki%20HA%22%5BAuthor%5D" class="usa-link"><span class="name western">Hala Abdul Baki</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rehman%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Erum Rehman</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Erum Rehman</span></h3>
<div class="p">
<sup>4</sup>Rashid Hospital Dubai Health, Dubai, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rehman%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Erum Rehman</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Salim%20AM%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Asif M Salim</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Asif M Salim</span></h3>
<div class="p">
<sup>4</sup>Rashid Hospital Dubai Health, Dubai, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Salim%20AM%22%5BAuthor%5D" class="usa-link"><span class="name western">Asif M Salim</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahmed%20RLG%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Rubina L G Ahmed</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Rubina L G Ahmed</span></h3>
<div class="p">
<sup>5</sup>Al Jalila Children’s Specialty Hospital, Dubai, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahmed%20RLG%22%5BAuthor%5D" class="usa-link"><span class="name western">Rubina L G Ahmed</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khansaheb%20HH%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Hamda H Khansaheb</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Hamda H Khansaheb</span></h3>
<div class="p">
<sup>6</sup>Mohammed Bin Rashid University of Medicine and Health Sciences (MBRU), Dubai, UAE </div>
<div class="p">
<sup>7</sup>Dubai Academic Health Corporation (Dubai Health), Dubai, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khansaheb%20HH%22%5BAuthor%5D" class="usa-link"><span class="name western">Hamda H Khansaheb</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Habous%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Maya Habous</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Maya Habous</span></h3>
<div class="p">
<sup>4</sup>Rashid Hospital Dubai Health, Dubai, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Habous%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maya Habous</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22AlDabal%20LMJA%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Laila M J A AlDabal</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Laila M J A AlDabal</span></h3>
<div class="p">
<sup>4</sup>Rashid Hospital Dubai Health, Dubai, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22AlDabal%20LMJA%22%5BAuthor%5D" class="usa-link"><span class="name western">Laila M J A AlDabal</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaballah%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Soumeya Jaballah</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Soumeya Jaballah</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE </div>
<div class="p">
<sup>3</sup>ASPIRE Research Institute in Precision Medicine, Abu Dhabi, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jaballah%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Soumeya Jaballah</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alqassim%20SS%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Saif S Alqassim</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Saif S Alqassim</span></h3>
<div class="p">
<sup>6</sup>Mohammed Bin Rashid University of Medicine and Health Sciences (MBRU), Dubai, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alqassim%20SS%22%5BAuthor%5D" class="usa-link"><span class="name western">Saif S Alqassim</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alsheikh-Ali%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Alawi Alsheikh-Ali</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Alawi Alsheikh-Ali</span></h3>
<div class="p">
<sup>6</sup>Mohammed Bin Rashid University of Medicine and Health Sciences (MBRU), Dubai, UAE </div>
<div class="p">
<sup>7</sup>Dubai Academic Health Corporation (Dubai Health), Dubai, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alsheikh-Ali%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alawi Alsheikh-Ali</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rizvi%20TA%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Tahir A Rizvi</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Tahir A Rizvi</span></h3>
<div class="p">
<sup>2</sup>Zayed Bin Sultan Center for Health Sciences, UAE University, Al Ain, UAE </div>
<div class="p">
<sup>3</sup>ASPIRE Research Institute in Precision Medicine, Abu Dhabi, UAE </div>
<div class="p">
<sup>8</sup>Department of Microbiology &amp; Immunology, College of Medicine and Health Sciences (CMHS), Tawam Hospital Complex, CMHS, UAE University, P.O. Box 15551, Al Ain, UAE </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rizvi%20TA%22%5BAuthor%5D" class="usa-link"><span class="name western">Tahir A Rizvi</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>8,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Biochemistry &amp; Molecular Biology, College of Medicine &amp; Health Sciences (CMHS), United Arab Emirates (UAE) University, Al Ain, UAE </div>
<div id="Aff2">
<sup>2</sup>Zayed Bin Sultan Center for Health Sciences, UAE University, Al Ain, UAE </div>
<div id="Aff3">
<sup>3</sup>ASPIRE Research Institute in Precision Medicine, Abu Dhabi, UAE </div>
<div id="Aff4">
<sup>4</sup>Rashid Hospital Dubai Health, Dubai, UAE </div>
<div id="Aff5">
<sup>5</sup>Al Jalila Children’s Specialty Hospital, Dubai, UAE </div>
<div id="Aff6">
<sup>6</sup>Mohammed Bin Rashid University of Medicine and Health Sciences (MBRU), Dubai, UAE </div>
<div id="Aff7">
<sup>7</sup>Dubai Academic Health Corporation (Dubai Health), Dubai, UAE </div>
<div id="Aff8">
<sup>8</sup>Department of Microbiology &amp; Immunology, College of Medicine and Health Sciences (CMHS), Tawam Hospital Complex, CMHS, UAE University, P.O. Box 15551, Al Ain, UAE </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 26; Accepted 2025 Aug 6; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371099  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841549/" class="usa-link">40841549</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par33">This study aimed to identify miRNA-based biomarkers in a multi-ethnic cohort of SARS-CoV-2-infected individuals to enhance preparedness for future variants of concern. A total of 31 healthy controls and 154 infected patients were enrolled, from which 13 matched controls and 38 infected nasal swab samples were analyzed using miRNA sequencing, followed by qRT-PCR validation. Among the 1788 miRNAs detected, 14 differentially expressed miRNAs and four novel miRNAs were identified, with novel-miR-264-5p showing a ≥ 2-fold change. Correlation with clinical markers highlighted several miRNAs as potential prognostic biomarkers. Seven miRNAs, including miR-146b-3p, miR-154-5p, miR-5010-3p, miR-127-3p, miR-335-3p, miR-30c-5p, and miR-202-5p, showed strong prognostic potential. Combined ROC analysis demonstrated that a panel of top-performing miRNAs significantly enhanced diagnostic accuracy (AUC 0.939–0.972; <em>p</em> &lt; 0.0001). Moreover, integrating miRNA biomarkers with clinical parameters further improved performance (AUC = 0.982; <em>p</em> &lt; 0.0001). miR-146b-3p, detected exclusively in infected patients, emerged as a highly specific biomarker. Several nasal miRNAs mirrored blood profiles, highlighting the utility of nasal swabs for non-invasive monitoring. Collectively, these findings suggest that miRNA-based biomarkers, alone or combined with clinical markers, offer a promising platform for COVID-19 prognosis and diagnosis, and lay groundwork for future miRNA-based antiviral strategies.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-15248-6.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> COVID-19, SARS-CoV-2, MiRNAs, Biomarkers, Nasopharyngeal swabs, MiRNAseq, Variants-of-Concern (VOC), Respiratory diseases, Infectious diseases</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Bioinformatics, Gene expression analysis, Genomic analysis, Microbiology, Molecular biology, Biomarkers, Health care, Medical research, Molecular medicine</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par34">Severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) is the cause of coronavirus disease 2019 (COVID-19) and infected over 778 million individuals globally and caused over 7 million deaths until June 14, 2025 (<a href="https://covid19.who.int/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://covid19.who.int/</a>). SARS-CoV-2 infection may result in symptoms ranging from being completely asymptomatic to having severe disease manifestations, such as the severe pulmonary disease, acute respiratory distress syndrome (ARDS)<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. It can also induce inflammatory responses, such as cytokine storm and activation of chemokines, hypoxemia, disruption of kidney and liver enzymes among individuals, etc.,<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. The infection occurs in both genders and can lead to multi-organ failure. However, men are more prone to severe COVID-19 than women, irrespective of age<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Co-morbidities, including previous lung infections, diabetes, hypertension, heart-associated risk factors, obesity, older age, immune disorders, and other organs manifestations, are all associated with induction of severe disease<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<p id="Par35">Since the emergence of COVID-19, numerous studies have attempted to establish the crucial role of host responses at the molecular and cellular levels to the severity of the disease in the infected patients. Viral infections in general can alter host-immune as well as cellular transcriptional responses to facilitate their replication and infectivity. This can result in molecular changes at cellular levels that can be detected clinically<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Interestingly, not every patient shows a similar host-response pattern and there is always a need to find molecular biomarkers that may differentiate COVID-19 infected patients from non-infected ones through simple and straightforward clinical tests<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>.</p>
<p id="Par36">Currently, researchers are emphasizing the exploration of microRNAs (miRNAs) as biomarkers for disease diagnosis and prognosis, especially due to their stability and resistance to degradation in routinely examined body fluids, including nasopharyngeal discharge, saliva, urine, blood, plasma, and serum<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. In viral infections, miRNA-based biomarkers can be especially useful since host miRNAs are induced as a host’s natural innate anti-viral response, better known as RNA interference (RNAi), to control virus infections<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Viruses, on the other hand, can exploit host miRNAs to subvert this response to facilitate their own replication or manipulate the host’s immune responses<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. This has been observed in a number of clinically important viruses, such as the human immunodeficiency virus (HIV), hepatitis B &amp; C viruses, influenza, dengue,  and others<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. An added advantage of identifying dysregulated miRNAs in diseases is the use of this information for eventual development of miRNA-based therapeutics<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. MiRNAs are small, non-coding functional RNAs encoded by genes and expressed in cells and by viruses as key regulators of gene expression. They function in a post-transcriptional manner by attacking specific host mRNAs via sequence complementarity, causing their translational inhibition or degradation<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Thus, miRNAs regulate gene expression in a highly targeted manner and the best way to identify and quantitate them are approaches based on next generation sequencing (NGS)<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p>
<p id="Par37">At present, most of the published data in this area of research is mainly driven from miRNAs isolated from blood samples; however, nasopharyngeal sample collection was the most widely used method to detect SARS-CoV-2 infection globally<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>–<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. It is well known that miRNAs are expressed in a tissue-specific manner and released in body fluids in a manner that each type of body fluid reveals its own canvas of miRNAs, a profile that changes upon disease induction and its severity. In COVID-19, there are only a few studies that report miRNA expression analysis from the nasopharyngeal samples in SARS-CoV-2 infected patients, despite the wide use of this sample in diagnosis<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>–<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Most of these studies were limited to only a selection of miRNAs and did not investigate the whole miRNA expression profile.</p>
<p id="Par38">To overcome this limitation, we used miRNAseq followed by quantitative reverse transcriptase PCR (RT-qPCR) to characterize miRNAs expressed during SARS-CoV-2 infection. While COVID-19 is no longer as deadly as it initially was and seems to have adapted well to the human population, the danger persists that a variant-of-concern (VOC) may emerge anytime that could cause another major pandemic, as is the case with other respiratory viruses like influenza. Thus, the aim of this study was to identify miRNAs that are uniquely expressed in a consistent manner following SARS-CoV-2 infection so that such miRNAs can be used as biomarkers for not only disease prognosis, but also the ability to develop novel RNA-based anti-virals and therapeutics that can be created rapidly to tackle any future pandemics.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Ethical permission and informed consent</h3>
<p id="Par39">The study was approved by the UAE University (UAEU) Human Research Ethics Committee (HREC approval no: ERH-2020-7219 2020-22) and the Dubai Scientific Research Ethics Committee (DSREC) of the Dubai Health Authority (DHA) (DSREC approval nos: DSREC-11/2020_16 and DSREC-11/2021_08). Informed Consent was taken from each subject following DHA and UAEU established procedures using approved consent forms available both in English and Arabic. Moreover, we confirmed that all methods were carried out in accordance with relevant guidelines and regulations.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Control and patient enrollment</h3>
<p id="Par40">The patients included in this study were confirmed positive for the virus by the COVID-19 nasal swab RT-qPCR test approved by the national health authorities prior to enrollment into the study. They were admitted to the Rashid Hospital, Dubai, the largest tertiary care hospital managing patients with COVID-19 infection since the start of the pandemic in the UAE. The control subjects were recruited from the staff of the Rashid Hospital as well as the UAE University. Each subject tested negative for SARS-CoV-2 infection by the COVID-19 RT-qPCR test performed on samples taken from nasal swab. A total of 154 infected patient samples were collected from <strong>September 2021- March 2022</strong>, while 31 control samples were collected for the study.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Clinical data</h3>
<p id="Par41">Clinical and demographic data for the enrolled patients was collected from the electronic medical records of the Rashid Hospital, Dubai. The patients were assessed for any medical symptoms (either acute or chronic) at the time of enrollment. Most of the symptoms included cough, diarrhea, chest problems, fever, and nausea. The comorbidities reported by infected individuals included diabetes, hypertension, thyroid issues, lipidemia, anemia, asthma, viral infections, kidney problems, previous history of chest infections, and heart-related issues. Chest X-rays were performed, depending upon the severity of disease symptoms. Routine laboratory tests were performed as per hospital protocol.</p></section><section id="Sec6"><h3 class="pmc_sec_title">RNA extraction and MiRNA sequencing</h3>
<p id="Par42">MiRNAs were extracted from nasal swab samples using the Macherey-Nagel NucleoSpin miRNA mini kit (cat. no. 740971), as described by the manufacturer, and quantified using Nano Drop in house. The miRNA sample integrity was tested by Agilent 2100 Bioanalyzer and sequenced at the Beijing Genomics Institute (BGI, Hong Kong) using UMI small RNA Library prep via the DNBSEQ platform (SE 50-single end; 50 bp size) at 20 M (20 million) clean read resolution.</p></section><section id="Sec7"><h3 class="pmc_sec_title">RNAseq data analysis</h3>
<p id="Par43">RNAseq was performed on 38 SARS-CoV-2 infected and 13 control (non-infected) individuals, as described above. The raw data was filtered using BGI in-house software known as SOAPnuke v1.5.2<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> and the impurities in the data like sequences without inserts, sequences that were too long or of low-quality, poly A sequences, small fragment sequences, and sequences with adaptor contaminations were removed. The clean sequences were subjected to filtration based on read length and the sequences ranging from 18 to 30 nt were selected for the next step. The Pearson correlation coefficients were calculated to compare the quality of gene expression between each sample, which showed higher similarities between each sample within a group. Principal component analysis (PCA) was used to remove any outliers from samples. The filtered clean reads were then subjected to quality control using FASTQ<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>.</p>
<p id="Par44">After sequencing, an average of 26.93 million reads (of ~ 50 nt length) were generated per sample and subjected to quality control. The clean reads were aligned to the reference genome sequence GCF_000001405.39_GRCh38.p13 (<em>Homo sapiens</em>), with an average alignment of 78.63% using Bowtie2<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. The results were submitted to the BGI in-house software Dr. Tom accessed through an online server for further analysis. The raw and analyzed data (BioProject accession number: PRJNA1049129) can be downloaded from the server (<a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1049129" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1049129</a>) for data re-analysis and further processing.</p></section><section id="Sec8"><h3 class="pmc_sec_title">RNAseq: DEGs analysis</h3>
<p id="Par45">After sequencing and cleaning (removal of adaptors), the data in fastq file(s) format were analyzed using the automated Dr. Tom software from BGI that allowed visualization and analysis of raw data. The DESeq2 method<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup> (Q-value/adjusted <em>p-</em>value ≤ 0.05) was used to detect differentially expressed miRNAs. The miRNAs with adjusted <em>p-</em>value of ≤ 0.05 and log2FC ≥ 2/≤ −2 were considered as differentially expressed genes (DEGs) and presented in red (upregulated) or green (downregulated) throughout the manuscript. All of the data were downloaded from Dr. Tom and re-analyzed using Microsoft Excel for any discrepancies. Venn diagrams were drawn for the overlapping miRNAs/transcripts using the online platform Bioinformatics &amp; Evolutionary Genomics from Van de Peer laboratory website (<a href="http://bioinformatics.psb.ugent.be/webtools/Venn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bioinformatics.psb.ugent.be/webtools/Venn/</a>). Heatmaps were generated using Multiple Experiment Viewer v4.9.0<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. The volcano and other plots used in this study were generated through Dr. Tom and were further improved accordingly.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Prediction of possible targeted mRNAs and associated pathways</h3>
<p id="Par46">MiRNAs function by targeting mRNAs; however, the number of experimentally verified mRNA targets of miRNAs are still limited. To overcome this <em>lacuna</em>, there are several methods (either supervised or semi-supervised) based on support vector machine (SVM). Most of these methods predict miRNA-mRNA interactions based on high-throughput sequencing experiments or from other verified sources. Three commonly used databases include: miRDB<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> (<a href="https://mirdb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://mirdb.org/</a>), TargetScan<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> (<a href="https://www.targetscan.org/vert_80/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.targetscan.org/vert_80/</a>), and DIANA-microT-CDS<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> (<a href="http://www.microrna.gr/microT-CDS" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.microrna.gr/microT-CDS</a>). These databases were employed to predict targets for the miRNAs identified in this study primarily due to limited available target-interaction data.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Identification of experimentally validated MiRNAs targets</h3>
<p id="Par47">Experimentally-validated miRNA targets were extracted from the miRTarBase<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> (<a href="https://mirtarbase.cuhk.edu.cn/~miRTarBase/miRTarBase_2022/php/index.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://mirtarbase.cuhk.edu.cn/~miRTarBase/miRTarBase_2022/php/index.php</a>), miRpathDB<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup> (<a href="https://mpd.bioinf.uni-sb.de/mirnas.html?organism=hsa" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://mpd.bioinf.uni-sb.de/mirnas.html?organism=hsa</a>.), and miRwayDB<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> (<a href="http://www.mirway.iitkgp.ac.in/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.mirway.iitkgp.ac.in/</a>) databases. These databases collect miRNA-target interactions after surveying existing literature and classify them as strong or poor targets based on reporter assays. In this study, we chose the targets associated with strong experimental evidence only.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Functional enrichment of gene ontology (GO) and pathway analysis</h3>
<p id="Par48">The list of predicted and experimentally identified targeted genes were separately uploaded to DAVID (<a href="https://david.abcc.ncifcrf.gov/tools.jsp" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://david.abcc.ncifcrf.gov/tools.jsp</a><sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> for gene ontology (GO) and pathway analysis using <em>Homo sapiens</em> as the reference species. The GO analysis plots genes according to their function, biological processes, cellular presence, and molecular functions. DAVID generated the list of genes involved in several biological pathways using KEGG (Kyoto Encyclopedia of Genes and Genomes: <a href="https://www.genome.jp/kegg/pathway.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genome.jp/kegg/pathway.html</a>)<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup> and Reactome (<a href="https://reactome.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://reactome.org/</a>)<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> pathway databases.</p></section><section id="Sec12"><h3 class="pmc_sec_title">RT-qPCR assays</h3>
<p id="Par49">Real time RT-qPCR was used to validate the miRNAseq results obtained. This was accomplished by retesting 15 patient miRNA samples analyzed by miRNAseq for 16 miRNAs using miRNA-based TaqMan assays from Applied Biosystems (Thermo Fisher Scientific, Waltham, MA USA). Briefly, the extracted RNAs were first converted into cDNAs using the TaqMan Advanced miRNA cDNA Synthesis Kit, as per manufacturer’s directions (cat. no. <a href="https://www.ncbi.nlm.nih.gov/protein/A28007" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">A28007</a>). This was followed by conducting TaqMan Advanced miRNA Assays for the miRNAs listed in Table <a href="#Tab1" class="usa-link">1</a> using the TaqMan Fast Advanced Master Mix (cat. no. 4444964) and the QuantStudios 7 Flex Real-Time PCR system. A 384-plate format was used to test the miRNAs in triplicates.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>List of the miRNAs tested by TaqMan assays for relative expression.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Serial Number</th>
<th align="left" colspan="1" rowspan="1">miRNA tested</th>
<th align="left" colspan="1" rowspan="1">Serial number</th>
<th align="left" colspan="1" rowspan="1">miRNA tested</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-550a-3p</td>
<td align="center" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-92b-5p</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-132-3p</td>
<td align="center" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-30c-5p</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-877-3p</td>
<td align="center" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-328-3p</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-95-3p</td>
<td align="center" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-106b-5p</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-486-5p</td>
<td align="center" colspan="1" rowspan="1">13</td>
<td align="left" colspan="1" rowspan="1">has-miR-5010-3p</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-221-3p</td>
<td align="center" colspan="1" rowspan="1">14</td>
<td align="left" colspan="1" rowspan="1">has-miR-125b-5p</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-2110</td>
<td align="center" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-16-5p</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-941</td>
<td align="center" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">hsa-miR-191-5p</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec13"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par51">To distinguish miRNAs expression between control and infected patients, a <em>p</em> value &lt; 0.05 (student’s t-test) and <em>Q</em> value &lt; 0.05 (False Discovery Rate (FDR) using Storey-Tibshirani procedure) were used. Levene’s test for equality of variances was used for comparing clinical data using SPSS v.26. GraphPad Prism v8 software was used to create Receiver Operating Characteristic (ROC) curves using the expression profiles of the miRNAs, as well as to generate other graphs presented in this study. The area under the curve (AUC) with a <em>p</em> value &lt; 0.05 was used to assess the ability of a miRNA to distinguish between control and infected patients. AUC with <em>p</em> value &lt; 0.05 for any miRNA could predict the usefulness of a miRNA to distinguish between control and infected patients effectively. To further account for multiple comparisons and reduce the risk of false positives, FDR correction using the Benjamini-Hochberg method was applied to ROC <em>p</em>-values. Only FDR-adjusted <em>p</em>-values (<em>p</em>-adjusted) below 0.05 were considered statistically significant. For other graphical data, a <em>p</em> value &lt; 0.05 was considered as a significant change.</p></section></section><section id="Sec14"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec15"><h3 class="pmc_sec_title">Patient demographics</h3>
<p id="Par52">This study encompassed a total of 31 control individuals (23 males (74.2%) and 8 females (25.8%), and 154 infected patients (116 males (75.3%) and 38 females (24.7%). The United Arab Emirates (UAE) is a diverse and multinational country, home to people from various parts of the world. While the participants included in this research hailed from 40 different countries (Fig. <a href="#Fig1" class="usa-link">1</a>a), a significant portion of them originated from India (<em>n</em> = 44; 23.8%), Philippines (<em>n</em> = 22; 11.9%), Pakistan (<em>n</em> = 18; 9.7%), UAE (<em>n</em> = 10; 5.4%), Egypt (<em>n</em> = 9; 4.9%), and Iran (<em>n</em> = 7; 3.8%). Additionally, there were 6 participants from Jordan and Iraq (3.2%), 5 from Bangladesh (2.7%), as well as 4 from Lebanon and Palestine (2.2%). Furthermore, Belgium, China, Nepal, Poland, Senegal, Sudan, and Syria were represented by 3 participants each (1.62%), while Australia, Ethiopia, Russia, South Africa, Sri Lanka, UK, and USA had 2 participants each (1.08%). Finally, Afghanistan, Armenia, Burundi, the Czech Republic, Eritrea, Germany, Ghana, Indonesia, Malaysia, Morocco, Switzerland, Thailand, Vietnam, and Yemen each had 1 participant, with one participant remaining unclassified (0.54%).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371099_41598_2025_15248_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3f6a/12371099/b6211fdad1d6/41598_2025_15248_Fig1_HTML.jpg" loading="lazy" id="MO1" height="736" width="736" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Visual representation of demographic and clinical outcomes of the participants. <strong>(a)</strong> A map displays the diverse origin of the participants of this study, representing 40 different countries across the globe. The intensity of green colors on the map indicates varying participant numbers from each country, with darker shades representing more participants and lighter shades indicating fewer. The world map was created using Microsoft Excel (Microsoft Excel LTSC Professional Plus 2024, 64-bit). <strong>(b)</strong> Comparative analysis of age, BMI, plasma levels of IL-6, ferritin, D-dimer, CRP, lymphocyte, WBC, and platelet count, hemoglobin levels, ALT, AST, urea, creatinine, and glomerular filtration rate (GFR) between uninfected and SARS-CoV-2 infected patients. The red or green scatter plots represent significantly up- or downregulated clinical markers, while orange indicates non-significant change.</p></figcaption></figure></section><section id="Sec16"><h3 class="pmc_sec_title">Clinical data shows distorted levels of ferritin, CRP, WBC, ALT, urea, &amp; GFR in infected patients</h3>
<p id="Par53">Figure 1b shows the comparison of other demographic and clinical parameters between the control and infected groups. The mean age of the control and infected groups was 44.23 ± 11.3 and 46.67 ± 13.1 years, respectively. The age variation was not statistically different within either control or infected groups (Fig. <a href="#Fig1" class="usa-link">1</a>; Table <a href="#Tab2" class="usa-link">2</a>). Both the control and infected groups showed similar body mass index (BMI) of ~ 27–28, revealing an overall overweight cohort. Although the controls showed presence of common morbidities like hypertension and diabetes, they were never reported to be previously infected with SARS-CoV-2 at the time of data collection. Some clinical laboratory data for the control group could not be included in this study due to the unavailability of test reports or because these tests were not conducted since the individuals were not experiencing any illness. Nonetheless, the available data for the control group fell within the already defined normal value range. Among the 154 infected patients at the time of infection, 55 (35.7%) had received vaccinations. The vaccines administered included Sinopharm BIBP (<em>n</em> = 38), Pfizer-BioNTech (<em>n</em> = 10), and other vaccines (<em>n</em> = 7), with the majority having received two doses of any vaccine (<em>n</em> = 44). A majority of the infected participants (<em>n</em> = 86; 55.8%) had cough, fatigue, fever, and shortness of breath at the time of infection. While X-ray reports were not available for controls, they were taken for 104/154 (67.5%) of the infected patients. Among the enrolled patients, 35 (22.7%) required respiratory support with oxygen delivery rates of up to 6 L/min and 51 (33.1%) with &gt; 6 L/min.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Clinical characteristics of the patients (<em>n</em> = 154) and control (<em>n</em> = 31) subjects.</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371099_41598_2025_15248_Tab2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3f6a/12371099/66a3ed8d7ce4/41598_2025_15248_Tab2_HTML.jpg" loading="lazy" id="MO2" height="486" width="750" alt="graphic file with name 41598_2025_15248_Tab2_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par55"> Routine or pre-defined laboratory tests were conducted for each infected patient. These tests involved the measurement of various clinical markers in patient blood, including ferritin, D-dimer, IL-6, C-reactive protein (CRP), white blood cell (WBC) &amp; lymphocyte count, platelet count, hemoglobin levels, alanine transaminase (ALT), aspartate aminotransferase (AST), urea, creatinine, and glomerular filtration rate (GFR). WBCs represent a broader category of cells involved in the body’s immune response than lymphocytes which are a specific subtype of white blood cells that play key roles in adaptive immunity, including T cells, B cells, and natural killer cells. For example, lymphopenia, which is a decrease in the number of lymphocytes in the blood, is a common finding in patients with severe COVID-19, while the total WBC count can vary in the same patients, depending on factors such as the presence of secondary bacterial infections or other inflammatory conditions. The levels of these clinical markers were then compared with either control or established normal ranges (Fig. <a href="#Fig1" class="usa-link">1</a>; Table <a href="#Tab2" class="usa-link">2</a>). A significant elevation was identified in the plasma concentrations of ferritin, CRP, and ALT among infected patients, surpassing the defined normal range. Induced levels of WBC, hemoglobin, and urea were also noted, but within the normal range, compared to the control. A marked reduction in GFR was observed compared to controls despite remaining within normal ranges, while a slight reduction in lymphocytes was observed. Although elevated levels of D-dimer and IL-6 were observed in infected patients, insufficient data precluded their inclusion in subsequent analysis.</p>
<p id="Par56">Finally, in terms of vaccines, at the time of sample collection, SARS-CoV-2 vaccination was not yet widespread. Among the 13 healthy controls, 2 individuals (15.4%) had received Pfizer vaccinations, while the remaining 11 were unvaccinated. In the infected group, 17 of our infected patients (44.7%) were vaccinated: 6 patients had received the Pfizer vaccine, 9 had received Sinopharm, and 2 had received Covishield. The remaining 21 infected patients were unvaccinated.</p></section><section id="Sec17"><h3 class="pmc_sec_title">SARS-CoV-2 infection significantly alters expression of a small subset of host MiRNAs</h3>
<p id="Par57">RNA was extracted from the nasopharyngeal swabs. Depending upon the amount and quality of the RNA, a select group of samples were analyzed by miRNAseq. Nevertheless, statistically significant number of samples were selected, including 13 control and 38 individuals infected with SARS-CoV-2. The average age of the control and infected patient was 47.62 ± 12.76 and 47.66 ± 12.92 years, respectively, while the male/female ratio was 3:1 for both groups (Table <a href="#Tab3" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Gender and age of the patients selected for MiRNAseq data analysis.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Patient groups</th>
<th align="left" colspan="1" rowspan="1">Gender (M/F)</th>
<th align="left" colspan="1" rowspan="1">Average Age</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Control</strong> (<em>n</em> = 13)</td>
<td align="left" colspan="1" rowspan="1">10 M/3F (3:1)</td>
<td align="left" colspan="1" rowspan="1">47.62 ± 12.76</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Infected</strong> (<em>n</em> = 38)</td>
<td align="left" colspan="1" rowspan="1">29 M/9F (3:1)</td>
<td align="left" colspan="1" rowspan="1">47.66 ± 12.92</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p32"><p>There was no significant difference in the average age of the two groups (<em>p</em> &gt; 0.992).</p></div></div></section><p id="Par59">Figure <a href="#Fig2" class="usa-link">2</a> shows the quality of data and the overall results obtained from the miRNAseq analysis. The heatmap displaying the raw expression data (Fig. <a href="#Fig2" class="usa-link">2</a>a), the bar graphs representing transcript expression profiles from individual samples (Fig. <a href="#Fig2" class="usa-link">2</a>b), as well as the two groups with data represented as box plots (Fig. <a href="#Fig2" class="usa-link">2</a>c), demonstrated consistent distribution within each sample from both groups, highlighting reliability of the data. The initial data analysis yielded a total of 1788 transcripts, with 1456 of them being previously known and 332 identified as novel miRNAs (Supplementary Data S1). Among the miRNAs detected, 1218 (67%) were expressed in both the control and infected groups, 78 (4.3%) miRNAs were uniquely expressed in the control group, while 492 (27.1%) were uniquely expressed in the infected group (Fig. <a href="#Fig2" class="usa-link">2</a>d). Pearson correlation of miRNA expression changes between control and infected groups revealed that SARS-CoV-2 infection altered ~ 2.6% of the host miRNAs (Fig. <a href="#Fig2" class="usa-link">2</a>e). Of the significantly altered miRNAs, 44 were known and 4 were novel miRNAs (29 up- and 15 downregulated; with <em>p</em> and <em>Q</em> values &lt; 0.05) that may be able to distinguish between control from infected groups (Fig. <a href="#Fig2" class="usa-link">2</a>f). Despite miRNAseq being a well-established and reliable technique that may not necessitate additional validation<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>, we took the extra step of confirming our overall miRNAseq findings. This was achieved by randomly selecting 15 RNA samples (7 control and 8 infected) that had been sequenced and subjecting them to TaqMan RT-qPCR assays for 16 miRNAs (two as endogenous controls). The results from our RT-qPCR analysis indicated that, with the exception of two miRNAs (miR-5010-3p and miR-2110), the remaining samples exhibited a similar trend in miRNA expression (non-significant variance) to what was observed in the miRNAseq analysis (Fig. <a href="#Fig3" class="usa-link">3</a>). Our findings demonstrate a superiority over the previously established normal range, with a 15–20% non-concordance in gene expression between RT-qPCR and miRNAseq.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371099_41598_2025_15248_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3f6a/12371099/5e9de8ab0c8b/41598_2025_15248_Fig2_HTML.jpg" loading="lazy" id="MO3" height="454" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Summary of the miRNA expression analysis of the raw data. In this study, 13 control and 38 infected samples were subject to miRNAseq. MiRNA sequencing resulted in 1788 transcripts that were identified as potential miRNAs. The data was analyzed using the fully automated program Dr. Tom from BGI. <strong>(a)</strong> Heatmap of the hierarchal clustered raw data representing control and infected groups. <strong>(b)</strong> Tukey box plots showing the expression of miRNAs in individual control and infected samples after normalization. <strong>(c)</strong> Tukey box plot comparing the whole group level distributions of miRNAs expression data after normalization. <strong>(d)</strong> Pearson’s correlation plot representing the correlation (<em>r</em>) values between control and infected groups. <strong>(e)</strong> Venn diagram of intersection of miRNAs expressed in control and infected groups representing 1296 control and 1710 infected miRNAs in sub groups with fold change (FC) &gt; 0, 1 and 2. <strong>(f)</strong> Volcano plot of the differentially regulated miRNAs. Red and green dots represent up- and downregulated miRNAs with FC ≥ 1.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371099_41598_2025_15248_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3f6a/12371099/c0317af562d4/41598_2025_15248_Fig3_HTML.jpg" loading="lazy" id="MO5" height="765" width="792" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Quantitative RT-PCR validation of the miRNAseq data. The relative normalized expression values (log 2 of infected/control) of the individual samples obtained from the miRNAseq analysis (<em>n</em> = 13 control and 38 infected) were compared with those obtained from the RT-qPCR data (<em>n</em> = 7 control and 8 infected). The red and green box and violin graphs show up- and down-regulated miRNAs, while the blue depicts the results that contradict the findings of the miRNAseq analysis. The floating line for every bar represents the mean expression value. The <em>p</em> value shows difference in mean (student’s <em>t</em>-test) for each miRNA in the miRNAseq and RT-qPCR cohort.</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">Characterization of the differentially regulated miRNAs observed in infected patients</h3>
<p id="Par61">Next, we closely analyzed the 48 miRNAs with <em>p/Q</em> values &lt; 0.05 observed in the infected patients. As shown, among the 44 known miRNAs, 29 were upregulated and 15 were downregulated (Fig. <a href="#Fig4" class="usa-link">4</a>a and b). However, only 36 of the 44 showed a fold change of &gt; 1, with 24 being upregulated and 12 downregulated (Fig. <a href="#Fig4" class="usa-link">4</a>b). Additionally, 14 miRNAs (12 upregulated and 2 downregulated) showed a fold change greater than ± 2 (Fig. <a href="#Fig4" class="usa-link">4</a>b). Notably, miR-146b-3p and miR-365b-3p were significantly upregulated, while miR-202-5p was markedly downregulated (as shown in Fig. <a href="#Fig4" class="usa-link">4</a>c; Table <a href="#Tab4" class="usa-link">4</a>). Among the four novel miRNAs examined, novel-miR-285-5p displayed significant upregulation, whereas the other three novel miRNAs, miR-115-5p, miR-189-5p, and miR-264-3p, were downregulated in patients infected with SARS-CoV-2 (Table <a href="#Tab4" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371099_41598_2025_15248_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3f6a/12371099/fd390b22d65d/41598_2025_15248_Fig4_HTML.jpg" loading="lazy" id="MO6" height="449" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Summary of the differentially regulated miRNAs in the infected patients. DEG analysis identified 48 miRNAs with <em>p</em>/<em>Q</em> values &lt; 0.05 with 44 known and 4 novel miRNAs. <strong>(a)</strong> Heatmap of the hierarchal clustered DEGs in individual infected samples. The blue boxes represent downregulated miRNAs, while yellow boxes show upregulated miRNAs in infected samples when compared to the control. These boxes represent log2 (Infected/Control) values for individual infected samples. <strong>(b)</strong> Bar graph representing up- and downregulated miRNAs in infected vs. control groups. Red bars represent up- while green bar represent downregulated miRNAs. <strong>(c)</strong> Volcano plot of differentially expressed up- (red dots) and down- (green dots) regulated 14 miRNAs in the infected group when compared to the control.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Differentially regulated known MiRNAs (<em>n</em> = 44) in infected patients (FC ≥ 0.5, <em>Q</em> &lt; 0.05)*, **.</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371099_41598_2025_15248_Tab4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3f6a/12371099/ef50e460fa24/41598_2025_15248_Tab4_HTML.jpg" loading="lazy" id="MO4" height="1170" width="772" alt="graphic file with name 41598_2025_15248_Tab4_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec19"><h3 class="pmc_sec_title">miRNAs may serve as possible prognostic markers in defining SARS-CoV-2 infected patients</h3>
<p id="Par65">The primary objective of this study was to evaluate the potential of differentially regulated miRNAs as prognostic markers when identifying SARS-CoV-2-infected patients. This was accomplished by conducting ROC analysis of each miRNA that helps discriminate the true negatives from true positives. Thus, ROC curves were created for each known miRNA that showed differential regulation in our study (with a fold change of ± 2 or more), as well as for the novel miRNAs identified during our research. Furthermore, we constructed ROC curves for four previously-reported miRNAs with an FC &lt; 2 in our study (miR125-5p, miR-151b, miR590-3p, and miR-625-5p) (Table <a href="#Tab5" class="usa-link">5</a>), but that had been shown to have potential as biomarkers to test their overall diagnostic performance in our SARS-CoV-2-infected patients<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. These ROC curves were drawn using normalized miRNA read counts levels observed in both the control and infected groups. The analysis of the ROC area under the curve (AUC) was used as a feature used to measure the accuracy of our biomarkers. Several miRNAs, including miR-146b-3p (AUC = 0.999, <em>p</em> &lt; 0.0001), miR-154-5p (AUC = 0.891, <em>p</em> = 0.002), miR-335-3p (AUC = 0.874, <em>p</em> &lt; 0.0001), and miR-30c-5p (AUC = 0.761, <em>p</em> = 0.004), demonstrated excellent discriminative ability between infected and control samples. (Fig. <a href="#Fig5" class="usa-link">5</a>a). Among the novel miRNAs, N-miR-264-5p showed a high diagnostic value with an AUC of 0.902 (<em>p</em> &lt; 0.0001), while N-miR-115-5p also displayed strong potential (AUC = 0.792, <em>p</em> = 0.001). (Fig. <a href="#Fig5" class="usa-link">5</a>b).</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Experimentally verified target genes associated with differentially regulated miRNAs.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">miRNA(s)</th>
<th align="left" colspan="1" rowspan="1">Number of genes</th>
<th align="left" colspan="1" rowspan="1">Experimentally verified genes name</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">hsa-miR-30c-5p</td>
<td align="left" colspan="1" rowspan="1">40</td>
<td align="left" colspan="1" rowspan="1">SNAI1, SERPINE1, UBE2I, SMAD1, IL11, VIM, TWF1, MTA1, HSPA4, TGIF2, HDAC4, SOCS3, BCL9, DDIT4, DLL4, CCND2, IDH1, RARB, NCOR2, RFX6, CASP3, RUNX2, JAK1, MTTP, SNAI2, NOTCH1, TP53, FASN, EIF2S1, BECN1, DNMT1, RASAL2, CAMK2D, IER2, CDC42, PAK1, FOXO3, CTGF, MCL1, TRADD</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hsa-miR-335-3p</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">NOS3, ESR1, PAX6SP1, MET, SP1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hsa-miR-132-3p</td>
<td align="left" colspan="1" rowspan="1">34</td>
<td align="left" colspan="1" rowspan="1">ARHGAP32, SIRT1, CDKN1A, RB1, CRK, HBEGF, RASA1, TJAP1, TLN2, SPRED1, MUC13, CCNA2, CCNB1, FOXO1, JPT1, BDNF, KLHL11, MAPK1, SOX4, SOX5, IRAK4, EGFR, RAF1, SPRY1, MMP9, GDF5, YY1AP1, AGO2, PIK3R3, SOX6, SMAD2, SLC2A1, FOXO3, PTEN,</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hsa-miR-127-3p</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">MAPK4, MMP13, PRDM1, XBP1, BCL6, KMT5A, XRCC3, SKI, SEPT7, RGMA, ZWINT, SFRP1, SERPINB9, MGMT, BAG5, IGF1R, IRS1, BLVRB</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hsa-miR-103a-2-5p</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">SMAD4, PDCD10</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hsa-miR-146b-3p</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">PER1, MAT2A, NPAS4, IRAK1, SMAD4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hsa-miR-202-5p</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">TGFBR1, TGFBR2, TNFSF13B</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hsa-miR-18a-3p</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">KRAS, CBX7, ATM, HIF1α, IRF2, S6k1, 4EBPL</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hsa-miR-154-5p</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">TLR2, ZEB2, HMGA2, CCND2, WNT5A, E2F5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hsa-miR-28-3p</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">STAT5B, TP53, BAG1, IGF1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371099_41598_2025_15248_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3f6a/12371099/b0e4551d7d98/41598_2025_15248_Fig5_HTML.jpg" loading="lazy" id="MO7" height="551" width="757" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Receiver Operating Characteristic (ROC) curve analysis of differentially regulated miRNAs in COVID-19 patients compared to healthy controls. ROC analysis was conducted using RNA sequencing data from 51 nasal swab samples (control: <em>n</em> = 13, infected: <em>n</em> = 38). <strong>(a)</strong> ROC curves for 12 upregulated and 2 down regulated known miRNAs showing FC ≥ ± 2 identified through RNA sequencing. <strong>(b)</strong> ROC curves of 4 novel miRNAs identified in this study. <strong>(c)</strong> ROC curves for previously reported circulating miRNAs validated using expression data from our cohort. (<strong>d</strong>) Combined ROC curves showing the improved discriminative power when multiple miRNAs were combined as biomarker panels. ROC curves were drawn for these reported miRNAs using read count expression data from our study. In all ROC plots, red and green lines represent predictive curves for upregulated and downregulated miRNAs, respectively; the dashed blue line indicates the random classifier (reference line), whereas orange line represents predictive curves for combined miRNAs. The Area Under the Curve (AUC) values and corresponding <em>p</em>-values are indicated for each miRNA.</p></figcaption></figure><p id="Par67">The ROC curves for the previously reported circulating miRNAs associated with viral infections were evaluated using expression data from our study. MiRNAs such as miR-125-5p (AUC = 0.746, <em>p</em> = 0.007), miR-151b (AUC = 0.749, <em>p</em> = 0.006), and miR-590-3p (AUC = 0.823, <em>p</em> &lt; 0.0001) demonstrated robust predictive performance, validating previous reports (Fig. <a href="#Fig5" class="usa-link">5</a>c). The optimal sensitivity and specificity values for identified markers in our study ranged from 65.91 to 100% and 61.54–100%, respectively, revealing high specificity with low to zero false rates since an AUC value of 1 indicates 100% accuracy (Supplementary Data S2)<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. The cut-off values for differentially regulated miRNA biomarkers were also determined based on normalized read counts to achieve an optimal balance between sensitivity and specificity using ROC curve analysis (Supplementary Data S2). This approach provides a robust estimate of the classification potential of the identified miRNAs within our cohort, aligning with methodologies previously described in biomarker research<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p>
<p id="Par68">To account for multiple comparisons, the FDR correction using the Benjamini-Hochberg method was applied to ROC <em>p</em>-values. Only FDR-adjusted <em>p</em>-values (<em>p</em>-adjusted) below 0.05 were considered statistically significant. This correction ensured that the significance of miRNAs as biomarkers is not due to random chance when evaluating multiple candidates. After correction, miR-146b-3p, miR-154-5p, miR-5010-3p, miR-127-3p, miR-335-3p, miR-30c-5p and miR-202-5p emerged as the most promising biomarker candidates (Fig. <a href="#Fig5" class="usa-link">5</a>a). These miRNAs demonstrated strong discriminative power, with AUC values ranging from 0.75 to 0.99 and statistically significant <em>p</em>-values after multiple testing correction. Finally, combined ROC curve analyses, integrating multiple top-performing miRNAs (such as miR-146b-3p, miR-154-5p, miR-335-3p, miR-127-3p, miR-30c-5p, and miR-202-5p) significantly enhanced diagnostic performance, achieving AUC values between 0.939 and 0.972 (<em>p</em> &lt; 0.0001) (Fig. <a href="#Fig5" class="usa-link">5</a>d). These findings highlight the potential advantage of using multi-miRNA panels over individual biomarkers for improving COVID-19 diagnosis from nasal swab samples.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Clinical markers and miRNAs showed significant correlation among each other</h3>
<p id="Par69">In our study, we noticed notable changes in both clinical indicators and miRNA expression among individuals infected with SARS-CoV-2 when compared to those who were healthy. To delve deeper into their connections, we conducted a correlation analysis by calculating the Pearson’s coefficient that evaluates the linear relationship between variables (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371099_41598_2025_15248_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3f6a/12371099/69f354d31a85/41598_2025_15248_Fig6_HTML.jpg" loading="lazy" id="MO8" height="557" width="762" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Correlation analysis and diagnostic performance of clinical markers and miRNA biomarkers in SARS-CoV-2 infection. Pearson’s correlation coefficient (r), <em>p</em>-values, and R² values were calculated to assess associations between clinical markers and differentially expressed miRNAs. Analyses were conducted using RNA sequencing data and clinical parameters from up to 51 samples (controls: <em>n</em> = 13; infected: <em>n</em> = 38). <strong>(a)</strong> Heatmap representing Pearson’s correlation coefficients among clinical markers, including ferritin, C-reactive protein (CRP), white blood cell count (WBC), alanine aminotransferase (ALT), urea, and glomerular filtration rate (GFR). Significant correlations are indicated by asterisks (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01). <strong>(b)</strong> Heatmap showing Pearson correlation values among selected potential biomarker miRNAs. <strong>(c)</strong> Scatter plots illustrating significant individual correlations between specific clinical markers (ferritin, CRP, GFR, urea) and candidate miRNAs (N-miR-115-5p, N-miR-264-5p, N-miR-30c-5p, and miR-5010-3p). (<strong>d</strong>) ROC curve analysis assessing the diagnostic performance of individual clinical markers (ferritin, CRP, urea, and GFR), their combined performance, and performance in combination with selected miRNAs. Integration of miRNA expression profiles with clinical markers markedly improved diagnostic accuracy (AUC = 0.982, <em>p</em> &lt; 0.0001).</p></figcaption></figure><p id="Par71">This analysis uncovered significant associations not only within the clinical markers or miRNA expression levels, but also between these two categories. Within the clinical markers, we observed both positive and negative correlations. For instance, ferritin displayed significant positive correlations with CRP, WBC, and ALT, while CRP exhibited a positive correlation with urea but a negative correlation with GFR. Additionally, GFR showed a strong negative correlation with urea levels in COVID-19 infected patients (Fig. <a href="#Fig6" class="usa-link">6</a>a).</p>
<p id="Par72">Notably, we also detected significant correlation among differentially regulated miRNAs. For this analysis, we selected miRNAs with an AUC greater than 0.75 and a <em>p</em>-value less than 0.05. These miRNAs included miR-154-5p, miR-5010-3p, miR-335-3p, miR-30c-5p, miR-202-5p, miR-590-3p, miR-625-3p, novel-miR-115-5p, and novel-miR-264-5p (Supplementary Data S2 and Fig. <a href="#Fig5" class="usa-link">5</a>). Our results revealed positive correlation between several miRNAs pairs, such as miR-154-5p/miR-5010-5p, miR-5010-3p/miR-30c-5p, miR-335-3p/miR-30c-5p, miR-30c-5p/miR-5010-3p, miR-202-5p/miR-625-5p, miR-590-3p/miR-625-5p, novel-miR-115-5p/novel-miR-264-5p, as well as miR-625-5p/miR-202-5p (Fig. <a href="#Fig6" class="usa-link">6</a>b). Additionally, significant negative correlations were observed between miR-154-5p/novel-miR-264-5p, miR-5010-3p/novel-miR-115-5p, miR-335-3p/novel-miR-264-5p, miR-30c-5p/novel-miR-115-5p and novel-miR-264-5p (Fig. <a href="#Fig6" class="usa-link">6</a>b).</p>
<p id="Par73">Remarkably, we also detected meaningful associations between the expression values of clinical markers and miRNAs expression levels (Fig. <a href="#Fig6" class="usa-link">6</a>c). Individual scatter plot analyses further confirmed significant correlations between selected miRNAs and key clinical markers. N-miR-115-5p levels negatively correlated with ferritin concentrations, while N-miR-30c-5p levels positively correlated with urea and CRP. Furthermore, miR-5010-3p demonstrated a positive association with urea and a negative association with GFR. Similarly, N-miR-264-5p levels were significantly associated with GFR, highlighting their potential relevance to kidney function during SARS-CoV-2 infection (Fig. <a href="#Fig6" class="usa-link">6</a>c).</p>
<p id="Par74">Next, ROC curve analysis was conducted to evaluate the diagnostic performance of individual clinical markers and their combination with miRNAs (Fig. <a href="#Fig6" class="usa-link">6</a>d). Among the clinical markers, ferritin (AUC = 0.931), CRP (AUC = 0.844), GFR (AUC = 0.777), and urea (AUC = 0.753) demonstrated reasonable discriminative ability between infected and control groups. When clinical markers were combined, diagnostic performance improved (AUC = 0.951, <em>p</em> = 0.002). Importantly, integrating selected miRNA biomarkers with clinical markers further enhanced diagnostic accuracy, achieving an AUC of 0.982 (<em>p</em> &lt; 0.0001), suggesting that a combined miRNA-clinical marker panel could serve as a highly sensitive and specific approach for detecting SARS-CoV-2 infection. Overall, our findings demonstrate a notable connection between the expression levels of specific miRNAs and clinical markers, revealing a biological association between the two. This association has the potential to serve as valuable prognostic markers for distinguishing SARS-CoV-2 infected patients from healthy individuals.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Differentially-expressed miRNAs share crucial biological pathways associated with disease severity</h3>
<p id="Par75">Next, we aimed at identifying the targets of the differentially-regulated miRNAs observed in our study. While numerous computer-based tools are available for predicting potential miRNA targets, we chose to focus solely on experimentally verified target genes for our specific miRNAs. To identify these targets, we conducted a thorough search of online databases, including miRTarBase, miRpathDB, and miRwayDB, which revealed that out of 14 miRNA that exhibited FC &gt; ± 2, ten miRNAs, including miR-30c-5p, miR-132-3p, miR-127-3p, miR-18a-3p, miR-154-5p, miR-335-3p, miR-146b-3p, miR-202-5p, and miR-103a-2-5p targeted a total of 40, 34, 18, 7, 6, 5, 5, 3, and 2 human genes, respectively, while there were no experimentally-validated targets identified for the remaining four miRNAs (Table <a href="#Tab5" class="usa-link">5</a>).</p>
<p id="Par76">To gain further insights into whether the genes we predicted as experimental targets share various biological pathways, we initially constructed a protein network encompassing all proteins transcribed by these genes using the STRING database. Among the 122 genes, 118 were found to be interconnected (resulting in 118 nodes and 344 edges) at higher confidence STRING settings (Fig. <a href="#Fig7" class="usa-link">7</a>a). Subsequently, we conducted searches for associated KEGG pathways, biological processes, molecular functions, and diseases using the DAVID database. DAVID analysis yielded 112 biological pathways (Supplementary Data S3) with a <em>p</em>-value &lt; 0.05, including prominent pathways such as FoxO, MAPK, neurotrophin, PI3-AKT, prolactin, relaxin, TGF-β, AGE-RAGE, ErbB, and apelin signaling (Fig. <a href="#Fig7" class="usa-link">7</a>b). The number of genes associated with each pathway are indicated by the size of the circle shown (Fig. <a href="#Fig7" class="usa-link">7</a>b).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371099_41598_2025_15248_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3f6a/12371099/732e2458100e/41598_2025_15248_Fig7_HTML.jpg" loading="lazy" id="MO9" height="442" width="730" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene Ontology (GO) and pathway analysis of experimentally verified targeted genes. Ten of the differentially regulated miRNAs (FC &gt; ± 2) identified in our study were used to predict their target genes and associated biological pathways. <strong>(a)</strong> Protein-protein interactions in miRNAs-associated targets during SARS-CoV-2 infection using STRING. Proteins showing two or more interactions were included in this figure. <strong>(b)</strong> Biological pathways associated with target genes in our study. <strong>(c)</strong> Interaction analysis demonstrating the connections between miRNAs (shown in <strong>blue)</strong>, associated genes (shown in <strong>black)</strong>, and the targeted pathways (shown in <strong>red</strong>). The <strong>blue</strong> lines show the connection between the miRNAs and their target genes, while the <strong>green</strong> lines represent the connections between miRNAs and the targeted pathways. This network of identified pathways, genes, and miRNAs was constructed and visualized using Cytoscape and resulted in 54 nodes and 154 edges. The genes associated with only one miRNA or pathway were excluded. <strong>(d)</strong> Biological processes associated with the targeted genes. <strong>(e)</strong> Molecular functions associated with the targeted genes. <strong>(f)</strong> Association between the targeted genes and various diseases.</p></figcaption></figure><p id="Par78">Network analysis further revealed that many genes associated with multiple pathways could be regulated by similar miRNAs (Fig. <a href="#Fig7" class="usa-link">7</a>c &amp; Supplementary Data S4). For example, among the miRNAs identified in this study, the MAPK signaling pathway could be influenced by miR-132-3p, miR-18a-3p, miR-28-3p, miR-30c-5p, miR-202-5p, and miR-146-3p. Similarly, the crucial PI3-AKT pathway is linked to miR-132-3p, miR-18a-3p, miR-28-3p, miR-30c-5p, miR-335-3p, and miR-154-5p. The same was true of many other genes, including AGE-RAGE, Apelin, FoxO, Relaxin, Neurotrophin, Prolactin, ErbB, and chemokine.</p>
<p id="Par79">Gene ontology analysis demonstrated that the predicted experimentally verified genes in this study regulate numerous biological processes, including transcription, apoptosis, morphogenesis, protein phosphorylation, and cell proliferation, among others, as shown in Fig. <a href="#Fig7" class="usa-link">7</a>d &amp; Supplementary Data S5) and molecular functions, such as protein binding, DNA binding, enzyme binding, miRNA binding, and transcription factor regulation, as illustrated in Fig. <a href="#Fig7" class="usa-link">7</a>e &amp; Supplementary Data S6). Furthermore, expression of these genes was associated with various diseases, especially many different types of cancers (breast, lung, ovarian, colorectal, prostate, oral, stomach, etc.), bone mineral density disease (osteoporosis), hypercholesterolemia, kidney failure, and human papillomavirus infection (Fig. <a href="#Fig7" class="usa-link">7</a>f &amp; Supplementary Data S7). These findings underscore the potential implications of dysregulated miRNAs in SARS-CoV-2 infected patients as they may have significant consequences.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par82">The aim of this study was to identify miRNA-based biomarkers in our ethnically-rich population infected with SARS-CoV-2 for prognosis/diagnosis purposes and beyond (Fig. <a href="#Fig1" class="usa-link">1</a>a). SARS-CoV-2 infection often leads to abnormal inflammatory and biochemical markers, including elevated D-dimers, ferritin, CRP, urea, creatinine, and abnormal cell counts, contributing to organ dysfunction and ICU admissions<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. In our cohort, ferritin, hemoglobin, CRP, WBC, ALT, and urea were significantly elevated, while GFR was reduced. IL-6 and D-dimer levels exceeded normal thresholds, but lacked control group comparisons (Fig. <a href="#Fig1" class="usa-link">1</a>b; Table <a href="#Tab2" class="usa-link">2</a>). Notably, ferritin correlated with CRP, WBC, ALT, and GFR, while urea was linked to lower GFR and higher CRP. These correlations highlight their potential as screening markers for infected patients (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<p id="Par83">Our study is unique due to its comprehensive data collection and diverse cohort, comprising individuals from over 40 countries in the UAE (Fig. <a href="#Fig1" class="usa-link">1</a>a). This diversity enhances insights into SARS-CoV-2-induced clinical and biochemical alterations. Due to sample quality limitations, sequencing analysis was conducted on 57 samples (control: <em>n</em> = 13, infected: <em>n</em> = 38), with additional validation performed on 15 RT-PCR samples. While the cohort size may impact statistical power, similar studies have used comparable sample sizes (Table <a href="#Tab6" class="usa-link">6</a>), and the application of FDR correction, inclusion of age- and sex-matched controls, and RT-PCR validation strengthens the reliability of our findings.</p>
<section class="tw xbox font-sm" id="Tab6"><h3 class="obj_head">Table 6.</h3>
<div class="caption p"><p>Comparison of studies using nasal swab samples for miRNA discovery.</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371099_41598_2025_15248_Tab6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3f6a/12371099/2a7aa3d29346/41598_2025_15248_Tab6_HTML.jpg" loading="lazy" id="MO11" height="566" width="793" alt="graphic file with name 41598_2025_15248_Tab6_HTML.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par84">We analyzed 14 differentially expressed miRNAs (≥ 2-fold change), along with four novel and four known miRNAs, as potential biomarkers using ROC analysis (Figs. <a href="#Fig2" class="usa-link">2</a>, <a href="#Fig3" class="usa-link">3</a> and <a href="#Fig4" class="usa-link">4</a>; Table <a href="#Tab5" class="usa-link">5</a>). Seven miRNAs, including miR-146b-3p, miR-154-5p, miR-5010-3p, miR-127-3p, miR-335-3p, miR-30c-5p and miR-202-5p, showed strong prognostic potential (AUC: 0.75–1, <em>p</em> &lt; 0.05). Additionally, four previously reported miRNAs, though less dysregulated, exhibited consistent expression (miR125-5p, miR-151b, miR590-3p, and miR-625-5p)<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>, reinforcing their biomarker potential (Figs. <a href="#Fig5" class="usa-link">5</a>, Table S2). Despite being less significantly dysregulated, these four miRNAs were also reported to be dysregulated in blood samples<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>–<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>.</p>
<p id="Par85">In this study, cut-off values for miRNA biomarkers were established using normalized read counts to balance sensitivity and specificity through ROC curve analysis (Supplementary Data S2). Higher sensitivity thresholds were emphasized for early screening to minimize false negatives, whereas specificity was prioritized in the context of disease severity assessment or prognostic evaluation, to avoid overestimating risk and ensuring accurate clinical decision-making. These thresholds serve as reference points within our cohort and should be further validated in future studies tailored to specific clinical applications. Furthermore, to ensure consistency across samples and comparability, all read counts were normalized for library size and distribution prior to ROC analysis, enhancing the robustness of classification estimates. While absolute expression levels may vary across detection platforms, normalization methods, and different cohorts, the consistent directionality of deregulated miRNAs (their up- or down-regulation) supports their reproducibility as biomarkers. Therefore, while the specific cut-off values identified here may require external validation, they offer a reliable reference for assessing clinical utility within this study.</p>
<p id="Par86">Interestingly, our combined ROC curve analyses further demonstrated the advantage of using multi-miRNA panels for diagnostic purposes. Integration of multiple top-performing miRNAs, including miR-146b-3p, miR-154-5p, miR-335-3p, miR-127-3p, miR-30c-5p, and miR-202-5p, significantly enhanced diagnostic performance, with AUC values ranging from 0.939 to 0.972 (<em>p</em> &lt; 0.0001). These results suggest that a multi-miRNA panel may offer superior sensitivity and specificity compared to individual biomarkers. Such panels could be particularly valuable in improving the detection of SARS-CoV-2 infection from nasal swab samples, providing a more reliable molecular diagnostic tool in clinical practice.</p>
<p id="Par87">Next, we compared our findings with differentially regulated miRNAs in WBCs, plasma, and serum (Fig. <a href="#Fig8" class="usa-link">8</a>). While our findings provide valuable insights into the role of miRNAs identified using nasal swab, it is important to consider the potential relationship between miRNA profiles in blood samples. Blood-based miRNA profiling is widely used for biomarker discovery due to the stability and systemic nature of circulating miRNAs; however, nasal swabs offer a more direct and localized assessment of respiratory tract infections. Despite the fact that most studies used miRNA sequencing, the expression profiles varied, with only a few miRNAs overlapping (Fig. <a href="#Fig8" class="usa-link">8</a>). Notably, Chen et al., 2020<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> and Fernandez-Pato et al., 2022<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup> showed 86% and 100% concordance with our nasal swab findings using WBCs and plasma, respectively. Interestingly, WBCs and plasma profiles were more similar to nasal swabs than serum, suggesting the use of coagulants can affect results. miR-125b-5p was frequently dysregulated across all sample types, while miR-127-3p and miR-151b were upregulated in nasal swabs, WBCs, and plasma but absent in serum. Furthermore, miR-590-3p was consistently downregulated in nasal swabs, WBCs, and serum but unreported in plasma. These findings highlight distinct miRNA expression patterns across sample types and underscore the potential of nasal swabs as biomarkers for COVID-19 therapeutic monitoring. Even though our study focused on nasal swabs, future studies comparing miRNA profiles from paired blood and nasal samples could enhance the reliability and clinical applicability of these biomarkers. Such comparisons may help determine whether specific miRNAs identified in nasal swabs also exhibit similar diagnostic potential in blood-based assays.</p>
<figure class="fig xbox font-sm" id="Fig8"><h3 class="obj_head">Fig. 8.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371099_41598_2025_15248_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3f6a/12371099/14b072d4f6de/41598_2025_15248_Fig8_HTML.jpg" loading="lazy" id="MO10" height="470" width="770" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative analysis of miRNA expression levels between our nasal swab (NS) samples and those obtained from various studies on blood. We examined the expression levels of fourteen identified miRNAs showing &gt; 2-fold change in expression from our study in comparison to samples from blood, including WBC, plasma, and serum. Red boxes denote miRNAs that were upregulated, while green boxes denote those that were downregulated following COVID-19 infection. The miRNAs highlighted in blue showed the same expression profile in our nasal swab samples as those observed in the WBCs, plasma, or serum reported from other studies, while the miRNA in orange was the most consistently dysregulated miRNA reported in most studies listed here. This is contrary to the profile of serum miRNAs reported that were mostly dysregulated in the opposite manner; i.,e., upregulated in nasal swabs, but downregulated in serum. All the studies reported in this figure used RNAseq except the one in the last column reported by Li et al., 2022 that used RT-qPCR.</p></figcaption></figure><p id="Par89">Understanding host responses to viral infection is crucial for managing patient health and predicting outcomes. Circulatory miRNAs have emerged as potential biomarkers for diagnosing SARS-CoV-2 infection and assessing disease severity<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. Combining miRNA expression with clinical markers may improve disease monitoring and prognosis. Our analysis revealed significant correlations between identified miRNAs and clinical parameters (Fig. <a href="#Fig6" class="usa-link">6</a>). MiR-30c-5p and miR-5010-3p correlated with high urea and low GFR, indicating poor prognosis, while novel-miR-264-5p correlated with low CRP and high GFR, suggesting a good prognosis. These findings highlight the potential of miRNA-based markers combined with clinical data for improved patient assessment. While individual clinical markers such as ferritin, CRP, GFR, and urea demonstrated reasonable discriminatory ability, combining these clinical markers significantly enhanced diagnostic performance (AUC = 0.951, <em>p</em> = 0.002). Notably, the addition of selected miRNA biomarkers to the clinical parameters further improved diagnostic accuracy, achieving an impressive AUC of 0.982 (<em>p</em> &lt; 0.0001). These findings highlight that miRNA expression profiles, when integrated with routine clinical data, could offer a highly sensitive and specific approach for identifying SARS-CoV-2 infection. This combinatorial strategy may overcome the limitations of using clinical markers alone and provide a more robust platform for early and precise diagnosis, particularly in cases where clinical symptoms are ambiguous or overlapping with other respiratory infections.</p>
<p id="Par90">We also compared our results with eight studies analyzing miRNA profiles in nasopharyngeal samples<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>–<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>,<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>–<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>, and one in bronchial aspirates<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup>. While no single miRNA was shared across all studies, several miRNAs, including miR-486-5p, let-7 family, miR-34b, miR-200 family, miR-21, and miR-27a-3p, appeared in multiple studies with similar or contrasting expression patterns (Table <a href="#Tab6" class="usa-link">6</a>). Notably, miR-486-5p and miR-27-3p were common between a nasal swab and bronchial aspirate study but absent in other nasal swab studies. These findings highlight shared and unique miRNA dysregulation patterns in COVID-19 across different respiratory sample types. We could identify only one miRNA that was common to one published nasal swab study that had the same expression profile as we reported, miR-21. To the contrary, we identified three miRNAs (miR-378a-3p, miR-146-3p, and miR-30c-5p) that were identified in another study, but with opposing expression profile (Table <a href="#Tab6" class="usa-link">6</a>). This inconsistency was not exclusive to the current analysis of nasopharyngeal samples, but was also observed in studies involving blood or serum samples (reviewed in<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. Based on these observations, it is plausible that validated biomarkers in one study may not align with those in another conducted elsewhere, as factors such as ethnicity, gender, age, presence of comorbidities, medications, types of COVID-19 treatments &amp; vaccines administered, and the strain of SARS-CoV-2, etc., could influence miRNA expression profiles in COVID-19 patients<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>,<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>.</p>
<p id="Par91">It has been evident that SARS-CoV-2 infection alters host miRNA expression, impacting biological functions and clinical outcomes<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. It is widely recognized that miRNAs target genes within interconnected pathways rather than randomly across the transcriptome<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>. Our analysis identified miRNA-targeted genes involved in key signaling pathways, including FoXO, MAPK, PI3-Akt, prolactin, TGF-β, and chemokine activation (Table <a href="#Tab6" class="usa-link">6</a>; Fig. <a href="#Fig7" class="usa-link">7</a>). These pathways regulate essential cellular processes such as transcription, apoptosis, proliferation, differentiation, morphogenesis, enzyme activity, and receptor functions, highlighting the critical role of miRNAs in COVID-19 pathogenesis and potential therapeutic targets. While bioinformatics predictions provide mechanistic insights, functional validation through RT-qPCR, luciferase assays, and knockdown experiments is warranted to confirm miRNA-mRNA interactions and their roles in SARS-CoV-2 infection.</p>
<p id="Par92">A majority of the miRNAs identified in this study (Table <a href="#Tab5" class="usa-link">5</a>) have previously been linked to either host immune responses or oncogenesis. For instance, miR-146b-3p, which we and others have found to be dysregulated in SARS-CoV-2 infected patients<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>, has been associated with pro-inflammatory markers, such as IL-6 and IL-8<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>,<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup>. Inhibition of miR-146b-3p has been shown to result in a 49% reduction in IL-6 expression, making it a promising inflammatory marker<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup>.</p>
<p id="Par93">Another identified miRNA, miR-154-5p, known as a regulator of allergic inflammation, has been proposed as a potential marker in patients with endometriosis<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>,<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup>. The upregulated miR-103a-5p in our study has been linked to T-cell activation in COVID-19 patients at rehabilitation stage and found to be upregulated in the plasma of COVID-19 patients<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>,<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>. Additionally, the upregulated miR-132-3p reported in our study is known to mediate anti-inflammatory effects by targeting acetylcholine and could impact processes like autophagy and calcium signaling<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup>.</p>
<p id="Par94">Three upregulated miRNAs—miR-1307-5p, miR-127-3p, and miR-30c-5p—are known to mediate host-viral responses and oncogenesis<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. miR-1307 promotes NF-κβ/MAPK signaling, enhancing cell proliferation<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>. miR-127-3p downregulation over-activates type I IFN signaling, affecting transcription and antiviral defense<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup>. Meanwhile, miR-30c-5p inhibits hepatocellular carcinoma progression and improves tamoxifen response in breast cancer<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>. Other upregulated miRNAs, including miR-365b-3p, miR-6887-3p, miR-5010-3p, and miR-335-3p, regulate apoptosis, MAPK/JAK/STAT signaling, and tumor development<sup><a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>,<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup>. Another upregulated miRNA in our study, miR-18a-3p is linked to p53-mediated ceRNA networks, where its knockdown reduces tumor growth and promotes differentiation<sup><a href="#CR100" class="usa-link" aria-describedby="CR100">100</a></sup>.</p>
<p id="Par95">Two of our identified miRNAs, miR-3663-5p and miR-202-5p, showed downregulation upon SARS-CoV-2 infection. MiR-3663-5p has previously been reported as dysregulated in non-alcoholic fatty liver disease (NAFLD) and may be linked to insulin signaling, metabolic regulation, and inflammation<sup><a href="#CR101" class="usa-link" aria-describedby="CR101">101</a></sup>. On the other hand, the downregulated miR-202-5p has been identified as an tumor suppressor and a potential diagnostic biomarker with reduced levels associated with cancer progression<sup><a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup>. Overall, the potential outcomes of the markedly disrupted miRNAs observed in this study indicate that SARS-CoV-2 infection not only initiates a host immune response and inflammation, but also encourages tissue damage that could be associated with organ failure, and in the long term, even cancer<sup><a href="#CR103" class="usa-link" aria-describedby="CR103">103</a></sup>. Our study suggests that miRNA dysregulation might induce oncogenic responses by activating IL-6 and TNF-α, while also potentially inciting anti-tumor responses following infection, implying a multifaceted relationship between SARS-CoV-2 and the progression of cancer.</p>
<p id="Par96">Beyond their prognostic value, the differentially expressed miRNAs identified in this study may offer therapeutic potential. The emergence of non-coding RNAs as promising candidates for therapeutic applications across infectious and non-infectious diseases is well established<sup><a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>–<a href="#CR110" class="usa-link" aria-describedby="CR110">110</a></sup>. Restoring the expression of downregulated miRNAs like miR-202-5p could enhance antiviral defenses, while targeting upregulated miRNAs such as miR-146b-3p, miR-132-3p, miR-103a-5p, miR-127-3p, and miR-30c-5p could modulate immune and antiviral pathways in SARS-CoV-2 infected patients. For instance, miRNA-132-3p, showing an upregulation in this study, has been shown to facilitate influenza A infection and enhance viral gene expression<sup><a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>,<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a></sup>. If miRNA-132-3p has a similar effect during COVID-19, downregulation of this miRNA could potentially be a valuable therapeutic target in COVID-19. Our findings thus suggest diverse roles of these miRNAs in host response and viral pathogenesis. Future mechanistic studies on their direct functional impact on SARS-CoV-2 could open avenues for promising RNA-based anti-COVID strategies, as has been seen in other diseases. For example, Liu et al.<sup><a href="#CR113" class="usa-link" aria-describedby="CR113">113</a></sup> demonstrated that miR-146b-3p attenuates ARDS in septic mice by targeting and inhibiting the PI3K/AKT signaling pathway, thus reducing inflammation and tissue damage. Thus, future studies are needed to validate the therapeutic potential of the identified miRNAs, though currently, their strongest application remains in prognosis and disease monitoring.</p>
<p id="Par97"><strong>Conclusions</strong>.</p>
<p id="Par98">We have identified 14 known, 4 novel, and 4 previously-identified miRNAs with the potential to serve as biomarkers for COVID-19 infection, miRNAs that exhibit differential regulation following SARS-CoV-2 infection in nasal swab samples. This research is unique in its presentation of data from a diverse global cohort residing in the UAE, making these biomarkers more suitable to a wide range of ethnicities. Many of the miRNAs identified demonstrated promising potential for distinguishing individuals infected with SARS-CoV-2 from uninfected individuals in combination with clinical parameters. Our results are consistent with previously reported data, and the newly discovered miRNAs in this study should be considered as promising candidates for future research. These biomarkers should also be valuable towards the rapid development of novel anti-virals and therapeutics against future variants-of-concern (VOC) that may emerge with pandemic potential.</p>
<section id="Sec23"><h3 class="pmc_sec_title">Limitations and future directions</h3>
<p id="Par99">This study identifies important miRNA-based biomarkers for future studies; however, with certain limitations. For instance, sample size &amp; quality, and potential confounders such as medications, vaccination status, and circulating viral variants could have influenced miRNA profiles and host responses. Furthermore, a longitudinal sampling design could have better captured the temporal changes in miRNA expression with disease progression along with an analysis of paired blood samples to determine systemic biomarker signatures. Therefore, future research should track miRNA dynamics across the acute, recovery, and post-COVID-19 phases to enhance biomarker relevance and correlate with disease severity. Future studies are also warranted with larger, stratified cohorts and detailed clinical metadata to refine biomarker reliability. Finally, while our findings highlight miRNAs with strong prognostic potential, it is important to distinguish their biomarker role from direct antiviral functions. Although prior studies, such as Haddad et al.<sup><a href="#CR114" class="usa-link" aria-describedby="CR114">114</a></sup>, suggest miRNAs may influence viral transmission, we did not directly assess such functional impacts. In short, our results lay a foundation for future functional investigations, with the immediate clinical utility of these miRNAs lying in prognosis and disease monitoring.</p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371099/bin/41598_2025_15248_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (2MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We are truly grateful to the staff and patients of Rashid Hospital, Dubai and the College of Medicine &amp; Health Sciences, UAE University (CMHS, UAEU) for the donation of samples and medical records to conduct the study. Without this generosity of spirit, this study would not have been possible. Special thanks go to Mr. Shiraz Anjum &amp; Mr. Sam Daniel at the Al Jalila Children’s Specialty Hospital for help with sample collection and processing, and Ms. Ekhlass Mohammed of the Department of Pediatrics, CMHS, UAEU for help in taking samples from the control individuals at CMHS, UAEU. WA &amp; FM contributed equally to this work.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>FM, TAR conceptualized the study. FM, TAR, AAA, SSA acquired funding for the project. FM &amp; TAR supervised the study. BG, JB, NGP, TAK, FM performed the experiments. BG, JB, WA, SJ, MH, LMJAA, ER, AMS, HHK, HAB, and RLGA collected clinical information &amp; data, coordinated project activities, and collected samples. FM, ER, AMS, WA, BG, and HHK designed and collected the questionnaire. WA, FM, performed data analysis. WA, FM, TAR interpreted the data and wrote the manuscript. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This project was funded by the Al Jalila Foundation (AJF2020006), Dubai, UAE and in part from funds from the Abu Dhabi Department of Education and Knowledge (ADEK) ASPIRE Grant (AARE20-344) and by ASPIRE, the technology program management pillar of Abu Dhabi’s Advanced Technology Research Council (ATRC), via the ASPIRE “Abu Dhabi Precision Medicine ARI” (VRI-20-10).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The control and patient samples are being further analyzed for other biomolecules and hence not available. The datasets generated and/or analysed during the current study are available in the BioProject repository under accession number: PRJNA1049129 (https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1049129).</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par100">The authors declare no competing interests.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par101">The study was approved by the UAE University (UAEU) Human Research Ethics Committee (HREC approval no: ERH-2020-7219 2020-22) and the Dubai Scientific Research Ethics Committee (DSREC) of the Dubai Health Authority (DHA) (DSREC approval nos: DSREC-11/2020_16 and DSREC-11/2021_08). Consent was taken from each subject following DHA and UAEU established procedures.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par102">All authors read and approved the final manuscript.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>These authors equally contributed to this work: Farah Mustafa and Waqar Ahmad.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Farah Mustafa, Email: fmustafa@uaeu.ac.ae.</p>
<p>Tahir A. Rizvi, Email: tarizvi@uaeu.ac.ae</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Celik, I. &amp; Ozturk, R. From asymptomatic to critical illness: decoding various clinical stages of COVID-19. <em>Turk. J. Med. Sci.</em><strong>51</strong> (SI-1), 3284–3300 (2021).
</cite> [<a href="https://doi.org/10.3906/sag-2107-137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8771010/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34445855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Celik,%20I.%20&amp;%20Ozturk,%20R.%20From%20asymptomatic%20to%20critical%20illness:%20decoding%20various%20clinical%20stages%20of%20COVID-19.%20Turk.%20J.%20Med.%20Sci.51%20(SI-1),%203284%E2%80%933300%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Hu, B. et al. Characteristics of SARS-CoV-2 and COVID-19. <em>Nat. Rev. Microbiol.</em><strong>19</strong> (3), 141–154 (2021).
</cite> [<a href="https://doi.org/10.1038/s41579-020-00459-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7537588/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33024307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20B.%20et%20al.%20Characteristics%20of%20SARS-CoV-2%20and%20COVID-19.%20Nat.%20Rev.%20Microbiol.19%20(3),%20141%E2%80%93154%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Clerbaux, L. A. et al. <em>Factors Modulating COVID-19: A Mechanistic Understanding Based on the Adverse Outcome Pathway Framework.</em> J Clin Med, 11(15). (2022).</cite> [<a href="https://doi.org/10.3390/jcm11154464" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9369763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35956081/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Jiang, Y. et al. <em>Inflammatory pathways in COVID-19: Mechanism and therapeutic interventions.</em> MedComm 2022. 3(3): p. e154. (2020).</cite> [<a href="https://doi.org/10.1002/mco2.154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9340488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35923762/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Rabaan, A. A. et al. Diverse immunological factors influencing pathogenesis in patients with COVID-19: A review on viral dissemination, immunotherapeutic options to counter cytokine storm and inflammatory responses. <em>Pathogens</em><strong>10</strong>(5), 565 (2021).</cite> [<a href="https://doi.org/10.3390/pathogens10050565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8150955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34066983/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Qudus, M. S. et al. The roles of critical pro-inflammatory cytokines in the drive of cytokine storm during SARS-CoV-2 infection. <em>J. Med. Virol.</em><strong>95</strong> (4), e28751 (2023).
</cite> [<a href="https://doi.org/10.1002/jmv.28751" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37185833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qudus,%20M.%20S.%20et%20al.%20The%20roles%20of%20critical%20pro-inflammatory%20cytokines%20in%20the%20drive%20of%20cytokine%20storm%20during%20SARS-CoV-2%20infection.%20J.%20Med.%20Virol.95%20(4),%20e28751%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Hojyo, S. et al. How COVID-19 induces cytokine storm with high mortality. <em>Inflamm. Regen</em>. <strong>40</strong>, 37 (2020).
</cite> [<a href="https://doi.org/10.1186/s41232-020-00146-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7527296/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33014208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hojyo,%20S.%20et%20al.%20How%20COVID-19%20induces%20cytokine%20storm%20with%20high%20mortality.%20Inflamm.%20Regen.%2040,%2037%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Farahani, M. et al. Molecular pathways involved in COVID-19 and potential pathway-based therapeutic targets. <em>Biomed. Pharmacother</em>. <strong>145</strong>, 112420 (2022).
</cite> [<a href="https://doi.org/10.1016/j.biopha.2021.112420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8585639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34801852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Farahani,%20M.%20et%20al.%20Molecular%20pathways%20involved%20in%20COVID-19%20and%20potential%20pathway-based%20therapeutic%20targets.%20Biomed.%20Pharmacother.%20145,%20112420%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Wu, H. H. L. et al. COVID-19 and hepatorenal syndrome. <em>World J. Gastroenterol.</em><strong>28</strong> (39), 5666–5678 (2022).
</cite> [<a href="https://doi.org/10.3748/wjg.v28.i39.5666" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9627428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36338894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20H.%20H.%20L.%20et%20al.%20COVID-19%20and%20hepatorenal%20syndrome.%20World%20J.%20Gastroenterol.28%20(39),%205666%E2%80%935678%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Fabiao, J. et al. Why do men have worse COVID-19-related outcomes? A systematic review and meta-analysis with sex adjusted for age. <em>Braz J. Med. Biol. Res.</em><strong>55</strong>, e11711 (2022).
</cite> [<a href="https://doi.org/10.1590/1414-431X2021e11711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8856598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35195196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fabiao,%20J.%20et%20al.%20Why%20do%20men%20have%20worse%20COVID-19-related%20outcomes?%20A%20systematic%20review%20and%20meta-analysis%20with%20sex%20adjusted%20for%20age.%20Braz%20J.%20Med.%20Biol.%20Res.55,%20e11711%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Iannizzi, C. et al. Convalescent plasma for people with COVID-19: a living systematic review. <em>Cochrane Database Syst. Rev.</em><strong>5</strong> (5), CD013600 (2023).
</cite> [<a href="https://doi.org/10.1002/14651858.CD013600.pub6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10171886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37162745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Iannizzi,%20C.%20et%20al.%20Convalescent%20plasma%20for%20people%20with%20COVID-19:%20a%20living%20systematic%20review.%20Cochrane%20Database%20Syst.%20Rev.5%20(5),%20CD013600%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Kartsonaki, C. et al. Characteristics and outcomes of an international cohort of 600 000 hospitalized patients with COVID-19. <em>Int. J. Epidemiol.</em><strong>52</strong> (2), 355–376 (2023).
</cite> [<a href="https://doi.org/10.1093/ije/dyad012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10114094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36850054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kartsonaki,%20C.%20et%20al.%20Characteristics%20and%20outcomes%20of%20an%20international%20cohort%20of%20600%20000%20hospitalized%20patients%20with%20COVID-19.%20Int.%20J.%20Epidemiol.52%20(2),%20355%E2%80%93376%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Ahmad, W. &amp; Shabbiri, K. Two years of SARS-CoV-2 infection (2019–2021): structural biology, vaccination, and current global situation. <em>Egypt. J. Intern. Med.</em><strong>34</strong> (1), 5 (2022).
</cite> [<a href="https://doi.org/10.1186/s43162-021-00092-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8759062/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35043040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahmad,%20W.%20&amp;%20Shabbiri,%20K.%20Two%20years%20of%20SARS-CoV-2%20infection%20(2019%E2%80%932021):%20structural%20biology,%20vaccination,%20and%20current%20global%20situation.%20Egypt.%20J.%20Intern.%20Med.34%20(1),%205%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Herbert, J. A. &amp; Panagiotou, S. In <em>Immune Response To Viruses. Encyclopedia of Infection and Immunity</em>. Vol. 1 (ed. Rezaei, N.) 429-444 (Elsevier, 2022).</cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Moreno-Altamirano, M. M. B., Kolstoe, S. E. &amp; Sanchez-Garcia, F. J. Virus control of cell metabolism for replication and evasion of host immune responses. <em>Front. Cell. Infect. Microbiol.</em><strong>9</strong>, 95 (2019).
</cite> [<a href="https://doi.org/10.3389/fcimb.2019.00095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6482253/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31058096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moreno-Altamirano,%20M.%20M.%20B.,%20Kolstoe,%20S.%20E.%20&amp;%20Sanchez-Garcia,%20F.%20J.%20Virus%20control%20of%20cell%20metabolism%20for%20replication%20and%20evasion%20of%20host%20immune%20responses.%20Front.%20Cell.%20Infect.%20Microbiol.9,%2095%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Adhya, D. &amp; Basu, A. Epigenetic modulation of host: new insights into immune evasion by viruses. <em>J. Biosci.</em><strong>35</strong> (4), 647–663 (2010).
</cite> [<a href="https://doi.org/10.1007/s12038-010-0072-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21289446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adhya,%20D.%20&amp;%20Basu,%20A.%20Epigenetic%20modulation%20of%20host:%20new%20insights%20into%20immune%20evasion%20by%20viruses.%20J.%20Biosci.35%20(4),%20647%E2%80%93663%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Hilleman, M. R. Strategies and mechanisms for host and pathogen survival in acute and persistent viral infections. <em>Proc. Natl. Acad. Sci. U S A</em>. <strong>101</strong> (Suppl 2(Suppl 2), 14560–14566 (2004).
</cite> [<a href="https://doi.org/10.1073/pnas.0404758101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC521982/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15297608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hilleman,%20M.%20R.%20Strategies%20and%20mechanisms%20for%20host%20and%20pathogen%20survival%20in%20acute%20and%20persistent%20viral%20infections.%20Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A.%20101%20(Suppl%202(Suppl%202),%2014560%E2%80%9314566%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Gavriatopoulou, M. et al. Organ-specific manifestations of COVID-19 infection. <em>Clin. Exp. Med.</em><strong>20</strong> (4), 493–506 (2020).
</cite> [<a href="https://doi.org/10.1007/s10238-020-00648-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7383117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32720223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gavriatopoulou,%20M.%20et%20al.%20Organ-specific%20manifestations%20of%20COVID-19%20infection.%20Clin.%20Exp.%20Med.20%20(4),%20493%E2%80%93506%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Ahmad, W. et al. Differentially-regulated MiRNAs in COVID-19: A systematic review. <em>Rev. Med. Virol.</em> 33(4), e2449 (2023).</cite> [<a href="https://doi.org/10.1002/rmv.2449" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37145095/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Mathew, D. et al. Deep immune profiling of COVID-19 patients reveals distinct immunotypes with therapeutic implications. <em>Science</em><strong>369</strong>(6508), eabc8511 (2020).</cite> [<a href="https://doi.org/10.1126/science.abc8511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7402624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32669297/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Battaglini, D. et al. Laboratory biomarkers for diagnosis and prognosis in COVID-19. <em>Front. Immunol.</em><strong>13</strong>, 857573 (2022).
</cite> [<a href="https://doi.org/10.3389/fimmu.2022.857573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9091347/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35572561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Battaglini,%20D.%20et%20al.%20Laboratory%20biomarkers%20for%20diagnosis%20and%20prognosis%20in%20COVID-19.%20Front.%20Immunol.13,%20857573%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Samprathi, M. &amp; Jayashree, M. Biomarkers in COVID-19: an Up-To-Date review. <em>Front. Pediatr.</em><strong>8</strong>, 607647 (2020).
</cite> [<a href="https://doi.org/10.3389/fped.2020.607647" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8042162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33859967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Samprathi,%20M.%20&amp;%20Jayashree,%20M.%20Biomarkers%20in%20COVID-19:%20an%20Up-To-Date%20review.%20Front.%20Pediatr.8,%20607647%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Luo, S. et al. Exploring potential biomarkers and molecular mechanisms of ischemic cardiomyopathy and COVID-19 comorbidity based on bioinformatics and systems biology. <em>Int. J. Mol. Sci.</em><strong>24</strong>(7), 6511 (2023).</cite> [<a href="https://doi.org/10.3390/ijms24076511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10094917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37047484/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Su, B. et al. Potential application of MicroRNA profiling to the diagnosis and prognosis of HIV-1 infection. <em>Front. Microbiol.</em><strong>9</strong>, 3185 (2018).
</cite> [<a href="https://doi.org/10.3389/fmicb.2018.03185" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6308129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30619232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su,%20B.%20et%20al.%20Potential%20application%20of%20MicroRNA%20profiling%20to%20the%20diagnosis%20and%20prognosis%20of%20HIV-1%20infection.%20Front.%20Microbiol.9,%203185%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Stanczyk, J. et al. Altered expression of MicroRNA in synovial fibroblasts and synovial tissue in rheumatoid arthritis. <em>Arthritis Rheum.</em><strong>58</strong> (4), 1001–1009 (2008).
</cite> [<a href="https://doi.org/10.1002/art.23386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18383392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stanczyk,%20J.%20et%20al.%20Altered%20expression%20of%20MicroRNA%20in%20synovial%20fibroblasts%20and%20synovial%20tissue%20in%20rheumatoid%20arthritis.%20Arthritis%20Rheum.58%20(4),%201001%E2%80%931009%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Andorfer, C. A. et al. MicroRNA signatures: clinical biomarkers for the diagnosis and treatment of breast cancer. <em>Trends Mol. Med.</em><strong>17</strong> (6), 313–319 (2011).
</cite> [<a href="https://doi.org/10.1016/j.molmed.2011.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21376668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andorfer,%20C.%20A.%20et%20al.%20MicroRNA%20signatures:%20clinical%20biomarkers%20for%20the%20diagnosis%20and%20treatment%20of%20breast%20cancer.%20Trends%20Mol.%20Med.17%20(6),%20313%E2%80%93319%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Huang, W. MicroRNAs: biomarkers, diagnostics, and therapeutics. <em>Methods Mol. Biol.</em><strong>1617</strong>, 57–67 (2017).
</cite> [<a href="https://doi.org/10.1007/978-1-4939-7046-9_4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28540676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20W.%20MicroRNAs:%20biomarkers,%20diagnostics,%20and%20therapeutics.%20Methods%20Mol.%20Biol.1617,%2057%E2%80%9367%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Leon-Icaza, S. A., Zeng, M. &amp; Rosas-Taraco, A. G. MicroRNAs in viral acute respiratory infections: immune regulation, biomarkers, therapy, and vaccines. <em>ExRNA</em><strong>1</strong> (1), 1 (2019).
</cite> [<a href="https://doi.org/10.1186/s41544-018-0004-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7149109/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34171007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leon-Icaza,%20S.%20A.,%20Zeng,%20M.%20&amp;%20Rosas-Taraco,%20A.%20G.%20MicroRNAs%20in%20viral%20acute%20respiratory%20infections:%20immune%20regulation,%20biomarkers,%20therapy,%20and%20vaccines.%20ExRNA1%20(1),%201%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Trobaugh, D. W. &amp; Klimstra, W. B. MicroRNA regulation of RNA virus replication and pathogenesis. <em>Trends Mol. Med.</em><strong>23</strong> (1), 80–93 (2017).
</cite> [<a href="https://doi.org/10.1016/j.molmed.2016.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5836316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27989642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trobaugh,%20D.%20W.%20&amp;%20Klimstra,%20W.%20B.%20MicroRNA%20regulation%20of%20RNA%20virus%20replication%20and%20pathogenesis.%20Trends%20Mol.%20Med.23%20(1),%2080%E2%80%9393%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Takahashi, T., Heaton, S. M. &amp; Parrish, N. F. Mammalian antiviral systems directed by small RNA. <em>PLoS Pathog</em>. <strong>17</strong> (12), e1010091 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1010091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8675686/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34914813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takahashi,%20T.,%20Heaton,%20S.%20M.%20&amp;%20Parrish,%20N.%20F.%20Mammalian%20antiviral%20systems%20directed%20by%20small%20RNA.%20PLoS%20Pathog.%2017%20(12),%20e1010091%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Trobaugh, D. W. et al. RNA viruses can hijack vertebrate MicroRNAs to suppress innate immunity. <em>Nature</em><strong>506</strong> (7487), 245–248 (2014).
</cite> [<a href="https://doi.org/10.1038/nature12869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4349380/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24352241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trobaugh,%20D.%20W.%20et%20al.%20RNA%20viruses%20can%20hijack%20vertebrate%20MicroRNAs%20to%20suppress%20innate%20immunity.%20Nature506%20(7487),%20245%E2%80%93248%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Ostrycharz, E. &amp; Hukowska-Szematowicz, B. Micro-Players of great Significance-Host MicroRNA signature in viral infections in humans and animals. <em>Int. J. Mol. Sci.</em><strong>23</strong>(18), 10536 (2022).</cite> [<a href="https://doi.org/10.3390/ijms231810536" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9504570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36142450/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Nguyen, T. H. et al. Potential role of MicroRNAs in the regulation of antiviral responses to influenza infection. <em>Front. Immunol.</em><strong>9</strong>, 1541 (2018).
</cite> [<a href="https://doi.org/10.3389/fimmu.2018.01541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6039551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30022983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen,%20T.%20H.%20et%20al.%20Potential%20role%20of%20MicroRNAs%20in%20the%20regulation%20of%20antiviral%20responses%20to%20influenza%20infection.%20Front.%20Immunol.9,%201541%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wong, R. R. et al. Role of MicroRNAs in antiviral responses to dengue infection. <em>J. Biomed. Sci.</em><strong>27</strong> (1), 4 (2020).
</cite> [<a href="https://doi.org/10.1186/s12929-019-0614-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6941309/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31898495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wong,%20R.%20R.%20et%20al.%20Role%20of%20MicroRNAs%20in%20antiviral%20responses%20to%20dengue%20infection.%20J.%20Biomed.%20Sci.27%20(1),%204%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Megahed, F. et al. MicroRNAs: small molecules with significant functions, particularly in the context of viral hepatitis B and C infection. <em>Med. (Kaunas)</em><strong>59</strong>(1), 173. (2023).</cite> [<a href="https://doi.org/10.3390/medicina59010173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9862007/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36676797/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Shimakami, T. et al. Stabilization of hepatitis C virus RNA by an Ago2-miR-122 complex. <em>Proc. Natl. Acad. Sci. U S A</em>. <strong>109</strong> (3), 941–946 (2012).
</cite> [<a href="https://doi.org/10.1073/pnas.1112263109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3271899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22215596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shimakami,%20T.%20et%20al.%20Stabilization%20of%20hepatitis%20C%20virus%20RNA%20by%20an%20Ago2-miR-122%20complex.%20Proc.%20Natl.%20Acad.%20Sci.%20U%20S%20A.%20109%20(3),%20941%E2%80%93946%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Chinniah, R. et al. The effect of MiRNA gene regulation on HIV disease. <em>Front. Genet.</em><strong>13</strong>, 862642 (2022).
</cite> [<a href="https://doi.org/10.3389/fgene.2022.862642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9117004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35601502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chinniah,%20R.%20et%20al.%20The%20effect%20of%20MiRNA%20gene%20regulation%20on%20HIV%20disease.%20Front.%20Genet.13,%20862642%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Balasubramaniam, M., Pandhare, J. &amp; Dash, C. Are MicroRNAs important players in HIV-1 infection?? An update. <em>Viruses</em><strong>10</strong>(3), 110 (2018).</cite> [<a href="https://doi.org/10.3390/v10030110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5869503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29510515/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Diener, C., Keller, A. &amp; Meese, E. Emerging concepts of MiRNA therapeutics: from cells to clinic. <em>Trends Genet.</em><strong>38</strong> (6), 613–626 (2022).
</cite> [<a href="https://doi.org/10.1016/j.tig.2022.02.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35303998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diener,%20C.,%20Keller,%20A.%20&amp;%20Meese,%20E.%20Emerging%20concepts%20of%20MiRNA%20therapeutics:%20from%20cells%20to%20clinic.%20Trends%20Genet.38%20(6),%20613%E2%80%93626%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Hombach, S. &amp; Kretz, M. Non-coding rnas: classification, biology and functioning. <em>Adv. Exp. Med. Biol.</em><strong>937</strong>, 3–17 (2016).
</cite> [<a href="https://doi.org/10.1007/978-3-319-42059-2_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27573892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hombach,%20S.%20&amp;%20Kretz,%20M.%20Non-coding%20rnas:%20classification,%20biology%20and%20functioning.%20Adv.%20Exp.%20Med.%20Biol.937,%203%E2%80%9317%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Blakely, J. P. Guest editorial: changes in radiology training. <em>Invest. Radiol.</em><strong>8</strong> (5), 357–359 (1973).
</cite> [<a href="https://doi.org/10.1097/00004424-197309000-00011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4744438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blakely,%20J.%20P.%20Guest%20editorial:%20changes%20in%20radiology%20training.%20Invest.%20Radiol.8%20(5),%20357%E2%80%93359%20(1973)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Dexheimer, P. J. &amp; Cochella, L. MicroRNAs: from mechanism to organism. <em>Front. Cell. Dev. Biol.</em><strong>8</strong>, 409 (2020).
</cite> [<a href="https://doi.org/10.3389/fcell.2020.00409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7283388/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32582699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dexheimer,%20P.%20J.%20&amp;%20Cochella,%20L.%20MicroRNAs:%20from%20mechanism%20to%20organism.%20Front.%20Cell.%20Dev.%20Biol.8,%20409%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Skalsky, R. L. &amp; Cullen, B. R. Viruses, micrornas, and host interactions. <em>Annu. Rev. Microbiol.</em><strong>64</strong>, 123–141 (2010).
</cite> [<a href="https://doi.org/10.1146/annurev.micro.112408.134243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3621958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20477536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Skalsky,%20R.%20L.%20&amp;%20Cullen,%20B.%20R.%20Viruses,%20micrornas,%20and%20host%20interactions.%20Annu.%20Rev.%20Microbiol.64,%20123%E2%80%93141%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Benesova, S., Kubista, M. &amp; Valihrach, L. Small RNA-Sequencing: approaches and considerations for MiRNA analysis. <em>Diagnostics (Basel)</em><strong>11</strong>(6), 964 (2021).</cite> [<a href="https://doi.org/10.3390/diagnostics11060964" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8229417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34071824/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>MubarakAli, D. Comprehensive review on rapid diagnosis of new infection COVID-19. <em>Appl. Biochem. Biotechnol.</em><strong>194</strong> (3), 1390–1400 (2022).
</cite> [<a href="https://doi.org/10.1007/s12010-021-03728-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8553914/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34714500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?MubarakAli,%20D.%20Comprehensive%20review%20on%20rapid%20diagnosis%20of%20new%20infection%20COVID-19.%20Appl.%20Biochem.%20Biotechnol.194%20(3),%201390%E2%80%931400%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Tarim, E. A. et al. Microfluidic-based virus detection methods for respiratory diseases. <em>Emergent Mater.</em><strong>4</strong> (1), 143–168 (2021).
</cite> [<a href="https://doi.org/10.1007/s42247-021-00169-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7992628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33786415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tarim,%20E.%20A.%20et%20al.%20Microfluidic-based%20virus%20detection%20methods%20for%20respiratory%20diseases.%20Emergent%20Mater.4%20(1),%20143%E2%80%93168%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Pritchard, C. C., Cheng, H. H. &amp; Tewari, M. MicroRNA profiling: approaches and considerations. <em>Nat. Rev. Genet.</em><strong>13</strong> (5), 358–369 (2012).
</cite> [<a href="https://doi.org/10.1038/nrg3198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4517822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22510765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pritchard,%20C.%20C.,%20Cheng,%20H.%20H.%20&amp;%20Tewari,%20M.%20MicroRNA%20profiling:%20approaches%20and%20considerations.%20Nat.%20Rev.%20Genet.13%20(5),%20358%E2%80%93369%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Eichmeier, A. et al. Conserved MicroRNAs in human nasopharynx tissue samples from swabs are differentially expressed in response to SARS-CoV-2. <em>Genes (Basel)</em><strong>13</strong>(2), 348 (2022).</cite> [<a href="https://doi.org/10.3390/genes13020348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8871708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35205390/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Farr, R. J. et al. Detection of SARS-CoV-2 infection by MicroRNA profiling of the upper respiratory tract. <em>PLoS One</em>. <strong>17</strong> (4), e0265670 (2022).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0265670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8982876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35381016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Farr,%20R.%20J.%20et%20al.%20Detection%20of%20SARS-CoV-2%20infection%20by%20MicroRNA%20profiling%20of%20the%20upper%20respiratory%20tract.%20PLoS%20One.%2017%20(4),%20e0265670%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Latini, A. et al. Expression analysis of MiRNA hsa-let7b-5p in naso-oropharyngeal swabs of COVID-19 patients supports its role in regulating ACE2 and DPP4 receptors. <em>J. Cell. Mol. Med.</em><strong>26</strong> (19), 4940–4948 (2022).
</cite> [<a href="https://doi.org/10.1111/jcmm.17492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9538662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36073344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Latini,%20A.%20et%20al.%20Expression%20analysis%20of%20MiRNA%20hsa-let7b-5p%20in%20naso-oropharyngeal%20swabs%20of%20COVID-19%20patients%20supports%20its%20role%20in%20regulating%20ACE2%20and%20DPP4%20receptors.%20J.%20Cell.%20Mol.%20Med.26%20(19),%204940%E2%80%934948%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>McDonald, J. T. et al. Role of miR-2392 in driving SARS-CoV-2 infection. <em>Cell. Rep.</em><strong>37</strong> (3), 109839 (2021).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.109839" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8481092/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34624208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McDonald,%20J.%20T.%20et%20al.%20Role%20of%20miR-2392%20in%20driving%20SARS-CoV-2%20infection.%20Cell.%20Rep.37%20(3),%20109839%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Chen, Y. et al. SOAPnuke: a mapreduce acceleration-supported software for integrated quality control and preprocessing of high-throughput sequencing data. <em>Gigascience</em><strong>7</strong> (1), 1–6 (2018).
</cite> [<a href="https://doi.org/10.1093/gigascience/gix120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5788068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29220494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Y.%20et%20al.%20SOAPnuke:%20a%20mapreduce%20acceleration-supported%20software%20for%20integrated%20quality%20control%20and%20preprocessing%20of%20high-throughput%20sequencing%20data.%20Gigascience7%20(1),%201%E2%80%936%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Wingett, S. W. &amp; Andrews, S. <em>FastQ Screen: A tool for multi-genome mapping and quality control.</em> F1000Res, 7: p. 1338. (2018).</cite> [<a href="https://doi.org/10.12688/f1000research.15931.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6124377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30254741/" class="usa-link">PubMed</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with bowtie 2. <em>Nat. Methods</em>. <strong>9</strong> (4), 357–359 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead,%20B.%20&amp;%20Salzberg,%20S.%20L.%20Fast%20gapped-read%20alignment%20with%20bowtie%202.%20Nat.%20Methods.%209%20(4),%20357%E2%80%93359%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Love, M. I., Huber, W. &amp; Anders, S. Moderated Estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biol.</em><strong>15</strong> (12), 550 (2014).
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love,%20M.%20I.,%20Huber,%20W.%20&amp;%20Anders,%20S.%20Moderated%20Estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.15%20(12),%20550%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Saeed, A. I. et al. TM4: a free, open-source system for microarray data management and analysis. <em>Biotechniques</em><strong>34</strong> (2), 374–378 (2003).
</cite> [<a href="https://doi.org/10.2144/03342mt01" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12613259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saeed,%20A.%20I.%20et%20al.%20TM4:%20a%20free,%20open-source%20system%20for%20microarray%20data%20management%20and%20analysis.%20Biotechniques34%20(2),%20374%E2%80%93378%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Chen, Y. &amp; Wang, X. MiRDB: an online database for prediction of functional MicroRNA targets. <em>Nucleic Acids Res.</em><strong>48</strong> (D1), D127–D131 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkz757" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6943051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31504780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Y.%20&amp;%20Wang,%20X.%20MiRDB:%20an%20online%20database%20for%20prediction%20of%20functional%20MicroRNA%20targets.%20Nucleic%20Acids%20Res.48%20(D1),%20D127%E2%80%93D131%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>McGeary, S. E. et al. The biochemical basis of MicroRNA targeting efficacy. <em>Science</em><strong>366</strong>(6472), eaav1741 (2019).</cite> [<a href="https://doi.org/10.1126/science.aav1741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7051167/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31806698/" class="usa-link">PubMed</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Tastsoglou, S. et al. <em>DIANA-microT 2023: Including Predicted Targets of Virally Encoded miRNAs</em> (Nucleic Acids Res, 2023).</cite> [<a href="https://doi.org/10.1093/nar/gkad283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37094027/" class="usa-link">PubMed</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Huang, H. Y. et al. MiRTarBase update 2022: an informative resource for experimentally validated miRNA-target interactions. <em>Nucleic Acids Res.</em><strong>50</strong> (D1), D222–D230 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20H.%20Y.%20et%20al.%20MiRTarBase%20update%202022:%20an%20informative%20resource%20for%20experimentally%20validated%20miRNA-target%20interactions.%20Nucleic%20Acids%20Res.50%20(D1),%20D222%E2%80%93D230%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Kehl, T. et al. MiRPathDB 2.0: a novel release of the MiRNA pathway dictionary database. <em>Nucleic Acids Res.</em><strong>48</strong> (D1), D142–D147 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkz1022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31691816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kehl,%20T.%20et%20al.%20MiRPathDB%202.0:%20a%20novel%20release%20of%20the%20MiRNA%20pathway%20dictionary%20database.%20Nucleic%20Acids%20Res.48%20(D1),%20D142%E2%80%93D147%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Das, S. S., Saha, P. &amp; Chakravorty, N. <em>miRwayDB: a database for experimentally validated microRNA-pathway associations in pathophysiological conditions.</em> Database (Oxford), 2018. (2018).</cite> [<a href="https://doi.org/10.1093/database/bay023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5829561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29688364/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Huang da, W., Sherman, B. T. &amp; Lempicki, R. A. Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources. <em>Nat. Protoc.</em><strong>4</strong> (1), 44–57 (2009).
</cite> [<a href="https://doi.org/10.1038/nprot.2008.211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19131956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20da,%20W.,%20Sherman,%20B.%20T.%20&amp;%20Lempicki,%20R.%20A.%20Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources.%20Nat.%20Protoc.4%20(1),%2044%E2%80%9357%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Kanehisa, M. et al. KEGG as a reference resource for gene and protein annotation. <em>Nucleic Acids Res.</em><strong>44</strong> (D1), D457–D462 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkv1070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4702792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26476454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa,%20M.%20et%20al.%20KEGG%20as%20a%20reference%20resource%20for%20gene%20and%20protein%20annotation.%20Nucleic%20Acids%20Res.44%20(D1),%20D457%E2%80%93D462%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Griss, J. et al. ReactomeGSA - Efficient Multi-Omics comparative pathway analysis. <em>Mol. Cell. Proteom.</em><strong>19</strong> (12), 2115–2125 (2020).</cite> [<a href="https://doi.org/10.1074/mcp.TIR120.002155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7710148/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32907876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Griss,%20J.%20et%20al.%20ReactomeGSA%20-%20Efficient%20Multi-Omics%20comparative%20pathway%20analysis.%20Mol.%20Cell.%20Proteom.19%20(12),%202115%E2%80%932125%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Coenye, T. Do results obtained with RNA-sequencing require independent verification? <em>Biofilm</em><strong>3</strong>, 100043 (2021).
</cite> [<a href="https://doi.org/10.1016/j.bioflm.2021.100043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7823214/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33665610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coenye,%20T.%20Do%20results%20obtained%20with%20RNA-sequencing%20require%20independent%20verification?%20Biofilm3,%20100043%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Nahm, F. S. Receiver operating characteristic curve: overview and practical use for clinicians. <em>Korean J. Anesthesiol</em>. <strong>75</strong> (1), 25–36 (2022).
</cite> [<a href="https://doi.org/10.4097/kja.21209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8831439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35124947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nahm,%20F.%20S.%20Receiver%20operating%20characteristic%20curve:%20overview%20and%20practical%20use%20for%20clinicians.%20Korean%20J.%20Anesthesiol.%2075%20(1),%2025%E2%80%9336%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Unal, I. Defining an optimal Cut-Point value in ROC analysis: an alternative approach. <em>Comput. Math. Methods Med.</em><strong>2017</strong>, 3762651 (2017).
</cite> [<a href="https://doi.org/10.1155/2017/3762651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5470053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28642804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Unal,%20I.%20Defining%20an%20optimal%20Cut-Point%20value%20in%20ROC%20analysis:%20an%20alternative%20approach.%20Comput.%20Math.%20Methods%20Med.2017,%203762651%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Florkowski, C. M. Sensitivity, specificity, receiver-operating characteristic (ROC) curves and likelihood ratios: communicating the performance of diagnostic tests. <em>Clin. Biochem. Rev.</em><strong>29</strong> (Suppl 1(Suppl 1), S83–S87 (2008).
</cite> [<a href="/articles/PMC2556590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18852864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Florkowski,%20C.%20M.%20Sensitivity,%20specificity,%20receiver-operating%20characteristic%20(ROC)%20curves%20and%20likelihood%20ratios:%20communicating%20the%20performance%20of%20diagnostic%20tests.%20Clin.%20Biochem.%20Rev.29%20(Suppl%201(Suppl%201),%20S83%E2%80%93S87%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Araya, S. et al. Organ function biomarker abnormalities, associated factors and disease outcome among hospitalized patients with COVID-19. <em>Biomark. Med.</em><strong>16</strong> (6), 417–426 (2022).
</cite> [<a href="https://doi.org/10.2217/bmm-2021-0681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8890361/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35234521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Araya,%20S.%20et%20al.%20Organ%20function%20biomarker%20abnormalities,%20associated%20factors%20and%20disease%20outcome%20among%20hospitalized%20patients%20with%20COVID-19.%20Biomark.%20Med.16%20(6),%20417%E2%80%93426%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Ahmad, W. et al. Differentially-regulated MiRNAs in COVID-19: A systematic review. <em>Rev. Med. Virol.</em><strong>33</strong> (4), e2449 (2023).
</cite> [<a href="https://doi.org/10.1002/rmv.2449" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37145095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahmad,%20W.%20et%20al.%20Differentially-regulated%20MiRNAs%20in%20COVID-19:%20A%20systematic%20review.%20Rev.%20Med.%20Virol.33%20(4),%20e2449%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Chen, Y. M. et al. Blood molecular markers associated with COVID-19 immunopathology and multi-organ damage. <em>EMBO J.</em><strong>39</strong> (24), e105896 (2020).
</cite> [<a href="https://doi.org/10.15252/embj.2020105896" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7737620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33140861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Y.%20M.%20et%20al.%20Blood%20molecular%20markers%20associated%20with%20COVID-19%20immunopathology%20and%20multi-organ%20damage.%20EMBO%20J.39%20(24),%20e105896%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Farr, R. J. et al. Altered MicroRNA expression in COVID-19 patients enables identification of SARS-CoV-2 infection. <em>PLoS Pathog</em>. <strong>17</strong> (7), e1009759 (2021).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1009759" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8318295/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34320031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Farr,%20R.%20J.%20et%20al.%20Altered%20MicroRNA%20expression%20in%20COVID-19%20patients%20enables%20identification%20of%20SARS-CoV-2%20infection.%20PLoS%20Pathog.%2017%20(7),%20e1009759%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Fernandez-Pato, A. et al. Plasma MiRNA profile at COVID-19 onset predicts severity status and mortality. <em>Emerg. Microbes Infect.</em><strong>11</strong> (1), 676–688 (2022).
</cite> [<a href="https://doi.org/10.1080/22221751.2022.2038021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8890551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35130828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fernandez-Pato,%20A.%20et%20al.%20Plasma%20MiRNA%20profile%20at%20COVID-19%20onset%20predicts%20severity%20status%20and%20mortality.%20Emerg.%20Microbes%20Infect.11%20(1),%20676%E2%80%93688%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Garcia-Giralt, N. et al. Circulating MicroRNA profiling is altered in the acute respiratory distress syndrome related to SARS-CoV-2 infection. <em>Sci. Rep.</em><strong>12</strong> (1), 6929 (2022).
</cite> [<a href="https://doi.org/10.1038/s41598-022-10738-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9047579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35484171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garcia-Giralt,%20N.%20et%20al.%20Circulating%20MicroRNA%20profiling%20is%20altered%20in%20the%20acute%20respiratory%20distress%20syndrome%20related%20to%20SARS-CoV-2%20infection.%20Sci.%20Rep.12%20(1),%206929%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Grehl, C. et al. Detection of SARS-CoV-2 derived small RNAs and changes in Circulating small RNAs associated with COVID-19. <em>Viruses</em><strong>13</strong>(8), 1593 (2021).</cite> [<a href="https://doi.org/10.3390/v13081593" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8402885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34452458/" class="usa-link">PubMed</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Li, C. et al. Differential MicroRNA expression in the peripheral blood from human patients with COVID-19. <em>J. Clin. Lab. Anal.</em><strong>34</strong> (10), e23590 (2020).
</cite> [<a href="https://doi.org/10.1002/jcla.23590" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7536972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32960473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20C.%20et%20al.%20Differential%20MicroRNA%20expression%20in%20the%20peripheral%20blood%20from%20human%20patients%20with%20COVID-19.%20J.%20Clin.%20Lab.%20Anal.34%20(10),%20e23590%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Li, X. et al. Potential predictive value of miR-125b-5p, miR-155-5p and their target genes in the course of COVID-19. <em>Infect. Drug Resist.</em><strong>15</strong>, 4079–4091 (2022).
</cite> [<a href="https://doi.org/10.2147/IDR.S372420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9346419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35937783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20X.%20et%20al.%20Potential%20predictive%20value%20of%20miR-125b-5p,%20miR-155-5p%20and%20their%20target%20genes%20in%20the%20course%20of%20COVID-19.%20Infect.%20Drug%20Resist.15,%204079%E2%80%934091%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Nicoletti, A. S. et al. Differentially expressed plasmatic MicroRNAs in Brazilian patients with coronavirus disease 2019 (COVID-19): preliminary results. <em>Mol. Biol. Rep.</em><strong>49</strong> (7), 6931–6943 (2022).
</cite> [<a href="https://doi.org/10.1007/s11033-022-07338-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8929466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35301654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nicoletti,%20A.%20S.%20et%20al.%20Differentially%20expressed%20plasmatic%20MicroRNAs%20in%20Brazilian%20patients%20with%20coronavirus%20disease%202019%20(COVID-19):%20preliminary%20results.%20Mol.%20Biol.%20Rep.49%20(7),%206931%E2%80%936943%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Zheng, H. Y. et al. Longitudinal transcriptome analyses show robust T cell immunity during recovery from COVID-19. <em>Signal. Transduct. Target. Ther.</em><strong>5</strong> (1), 294 (2020).
</cite> [<a href="https://doi.org/10.1038/s41392-020-00457-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7758413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33361761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20H.%20Y.%20et%20al.%20Longitudinal%20transcriptome%20analyses%20show%20robust%20T%20cell%20immunity%20during%20recovery%20from%20COVID-19.%20Signal.%20Transduct.%20Target.%20Ther.5%20(1),%20294%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Gutmann, C. et al. Association of cardiometabolic MicroRNAs with COVID-19 severity and mortality. <em>Cardiovasc. Res.</em><strong>118</strong> (2), 461–474 (2022).
</cite> [<a href="https://doi.org/10.1093/cvr/cvab338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8689968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34755842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gutmann,%20C.%20et%20al.%20Association%20of%20cardiometabolic%20MicroRNAs%20with%20COVID-19%20severity%20and%20mortality.%20Cardiovasc.%20Res.118%20(2),%20461%E2%80%93474%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Tang, H. et al. The noncoding and coding transcriptional landscape of the peripheral immune response in patients with COVID-19. <em>Clin. Transl Med.</em><strong>10</strong> (6), e200 (2020).
</cite> [<a href="https://doi.org/10.1002/ctm2.200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7548099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33135345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang,%20H.%20et%20al.%20The%20noncoding%20and%20coding%20transcriptional%20landscape%20of%20the%20peripheral%20immune%20response%20in%20patients%20with%20COVID-19.%20Clin.%20Transl%20Med.10%20(6),%20e200%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Zhao, Q. et al. Identification of a SARS-CoV-2 virus-encoded small non-coding RNA in association with the neurological disorders in COVID-19 patients. <em>Signal. Transduct. Target. Ther.</em><strong>7</strong> (1), 107 (2022).
</cite> [<a href="https://doi.org/10.1038/s41392-022-00969-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8967939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35361748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20Q.%20et%20al.%20Identification%20of%20a%20SARS-CoV-2%20virus-encoded%20small%20non-coding%20RNA%20in%20association%20with%20the%20neurological%20disorders%20in%20COVID-19%20patients.%20Signal.%20Transduct.%20Target.%20Ther.7%20(1),%20107%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Soltane, R. et al. A comparative analysis of MicroRNA expression in mild, moderate, and severe COVID-19: insights from urine, serum, and nasopharyngeal samples. <em>Biomolecules</em><strong>13</strong>(12), 1681 (2023).</cite> [<a href="https://doi.org/10.3390/biom13121681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10742216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38136554/" class="usa-link">PubMed</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Garnier, N. et al. Altered MicroRNA expression in severe COVID-19: potential prognostic and pathophysiological role. <em>Clin. Transl Med.</em><strong>12</strong> (6), e899 (2022).
</cite> [<a href="https://doi.org/10.1002/ctm2.899" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9191908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35696619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garnier,%20N.%20et%20al.%20Altered%20MicroRNA%20expression%20in%20severe%20COVID-19:%20potential%20prognostic%20and%20pathophysiological%20role.%20Clin.%20Transl%20Med.12%20(6),%20e899%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Wu, W. et al. Changes of small Non-coding RNAs by severe acute respiratory syndrome coronavirus 2 infection. <em>Front. Mol. Biosci.</em><strong>9</strong>, 821137 (2022).
</cite> [<a href="https://doi.org/10.3389/fmolb.2022.821137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8905365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35281271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20W.%20et%20al.%20Changes%20of%20small%20Non-coding%20RNAs%20by%20severe%20acute%20respiratory%20syndrome%20coronavirus%202%20infection.%20Front.%20Mol.%20Biosci.9,%20821137%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Molinero, M. et al. Bronchial Aspirate-Based profiling identifies MicroRNA signatures associated with COVID-19 and fatal disease in critically ill patients. <em>Front. Med. (Lausanne)</em>. <strong>8</strong>, 756517 (2021).
</cite> [<a href="https://doi.org/10.3389/fmed.2021.756517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8850692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35186962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Molinero,%20M.%20et%20al.%20Bronchial%20Aspirate-Based%20profiling%20identifies%20MicroRNA%20signatures%20associated%20with%20COVID-19%20and%20fatal%20disease%20in%20critically%20ill%20patients.%20Front.%20Med.%20(Lausanne).%208,%20756517%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Jankovic, M. et al. MiRNAs as a potential biomarker in the COVID-19 infection and complications course, severity, and outcome. <em>Diagnostics (Basel)</em>, <strong>13</strong>(6), 1091 (2023).</cite> [<a href="https://doi.org/10.3390/diagnostics13061091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10047241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36980399/" class="usa-link">PubMed</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Backes, C. et al. A dictionary on MicroRNAs and their putative target pathways. <em>Nucleic Acids Res.</em><strong>38</strong> (13), 4476–4486 (2010).
</cite> [<a href="https://doi.org/10.1093/nar/gkq167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2910047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20299343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Backes,%20C.%20et%20al.%20A%20dictionary%20on%20MicroRNAs%20and%20their%20putative%20target%20pathways.%20Nucleic%20Acids%20Res.38%20(13),%204476%E2%80%934486%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Sheedy, F. J. &amp; O’Neill, L. A. Adding fuel to fire: MicroRNAs as a new class of mediators of inflammation. <em>Ann. Rheum. Dis.</em><strong>67</strong> (Suppl 3), iii50–iii55 (2008).
</cite> [<a href="https://doi.org/10.1136/ard.2008.100289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19022814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sheedy,%20F.%20J.%20&amp;%20O%E2%80%99Neill,%20L.%20A.%20Adding%20fuel%20to%20fire:%20MicroRNAs%20as%20a%20new%20class%20of%20mediators%20of%20inflammation.%20Ann.%20Rheum.%20Dis.67%20(Suppl%203),%20iii50%E2%80%93iii55%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Pfeiffer, D. et al. miR-146a, miR-146b, and miR-155 increase expression of IL-6 and IL-8 and support HSP10 in an in vitro sepsis model. <em>PLoS One</em>. <strong>12</strong> (6), e0179850 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0179850" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5491059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28662100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pfeiffer,%20D.%20et%20al.%20miR-146a,%20miR-146b,%20and%20miR-155%20increase%20expression%20of%20IL-6%20and%20IL-8%20and%20support%20HSP10%20in%20an%20in%20vitro%20sepsis%20model.%20PLoS%20One.%2012%20(6),%20e0179850%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Kim, M. et al. MiR-154-5p-MCP1 axis regulates allergic inflammation by mediating cellular interactions. <em>Front. Immunol.</em><strong>12</strong>, 663726 (2021).
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.663726" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8201518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34135893/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20M.%20et%20al.%20MiR-154-5p-MCP1%20axis%20regulates%20allergic%20inflammation%20by%20mediating%20cellular%20interactions.%20Front.%20Immunol.12,%20663726%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Pateisky, P. et al. hsa-miRNA-154-5p expression in plasma of endometriosis patients is a potential diagnostic marker for the disease. <em>Reprod. Biomed. Online</em>. <strong>37</strong> (4), 449–466 (2018).
</cite> [<a href="https://doi.org/10.1016/j.rbmo.2018.05.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29857988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pateisky,%20P.%20et%20al.%20hsa-miRNA-154-5p%20expression%20in%20plasma%20of%20endometriosis%20patients%20is%20a%20potential%20diagnostic%20marker%20for%20the%20disease.%20Reprod.%20Biomed.%20Online.%2037%20(4),%20449%E2%80%93466%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Shaked, I. et al. MicroRNA-132 potentiates cholinergic anti-inflammatory signaling by targeting acetylcholinesterase. <em>Immunity</em><strong>31</strong> (6), 965–973 (2009).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2009.09.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20005135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shaked,%20I.%20et%20al.%20MicroRNA-132%20potentiates%20cholinergic%20anti-inflammatory%20signaling%20by%20targeting%20acetylcholinesterase.%20Immunity31%20(6),%20965%E2%80%93973%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Du, X. et al. MiR-1307-5p targeting TRAF3 upregulates the MAPK/NF-kappaB pathway and promotes lung adenocarcinoma proliferation. <em>Cancer Cell. Int.</em><strong>20</strong>, 502 (2020).
</cite> [<a href="https://doi.org/10.1186/s12935-020-01595-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7552495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33061854/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Du,%20X.%20et%20al.%20MiR-1307-5p%20targeting%20TRAF3%20upregulates%20the%20MAPK/NF-kappaB%20pathway%20and%20promotes%20lung%20adenocarcinoma%20proliferation.%20Cancer%20Cell.%20Int.20,%20502%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Wu, L. et al. Downregulation of renal Hsa-miR-127-3p contributes to the overactivation of type I interferon signaling pathway in the kidney of lupus nephritis. <em>Front. Immunol.</em><strong>12</strong>, 747616 (2021).
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.747616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8566726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34745118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20L.%20et%20al.%20Downregulation%20of%20renal%20Hsa-miR-127-3p%20contributes%20to%20the%20overactivation%20of%20type%20I%20interferon%20signaling%20pathway%20in%20the%20kidney%20of%20lupus%20nephritis.%20Front.%20Immunol.12,%20747616%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Hu, S. et al. Identification of MicroRNA hsa-miR-30c-5p as an inhibitory factor in the progression of hepatocellular carcinoma and investigation of its regulatory network via comprehensive analysis. <em>Bioengineered</em><strong>12</strong> (1), 7165–7177 (2021).
</cite> [<a href="https://doi.org/10.1080/21655979.2021.1979439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8806565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34503377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20S.%20et%20al.%20Identification%20of%20MicroRNA%20hsa-miR-30c-5p%20as%20an%20inhibitory%20factor%20in%20the%20progression%20of%20hepatocellular%20carcinoma%20and%20investigation%20of%20its%20regulatory%20network%20via%20comprehensive%20analysis.%20Bioengineered12%20(1),%207165%E2%80%937177%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Nie, J. et al. microRNA-365, down-regulated in colon cancer, inhibits cell cycle progression and promotes apoptosis of colon cancer cells by probably targeting Cyclin D1 and Bcl-2. <em>Carcinogenesis</em><strong>33</strong> (1), 220–225 (2012).
</cite> [<a href="https://doi.org/10.1093/carcin/bgr245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22072615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nie,%20J.%20et%20al.%20microRNA-365,%20down-regulated%20in%20colon%20cancer,%20inhibits%20cell%20cycle%20progression%20and%20promotes%20apoptosis%20of%20colon%20cancer%20cells%20by%20probably%20targeting%20Cyclin%20D1%20and%20Bcl-2.%20Carcinogenesis33%20(1),%20220%E2%80%93225%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Ye, L. et al. Functions and targets of miR-335 in cancer. <em>Onco Targets Ther.</em><strong>14</strong>, 3335–3349 (2021).
</cite> [<a href="https://doi.org/10.2147/OTT.S305098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8144171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34045870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye,%20L.%20et%20al.%20Functions%20and%20targets%20of%20miR-335%20in%20cancer.%20Onco%20Targets%20Ther.14,%203335%E2%80%933349%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Hsu, T. I. et al. MicroRNA-18a is elevated in prostate cancer and promotes tumorigenesis through suppressing STK4 in vitro and in vivo. <em>Oncogenesis</em><strong>3</strong> (4), e99 (2014).
</cite> [<a href="https://doi.org/10.1038/oncsis.2014.12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4007194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24752237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hsu,%20T.%20I.%20et%20al.%20MicroRNA-18a%20is%20elevated%20in%20prostate%20cancer%20and%20promotes%20tumorigenesis%20through%20suppressing%20STK4%20in%20vitro%20and%20in%20vivo.%20Oncogenesis3%20(4),%20e99%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Soronen, J. et al. <em>Novel hepatic MicroRNAs upregulated in human nonalcoholic fatty liver disease</em>. <em>Physiol. Rep.</em><strong>4</strong>(1), e12661 (2016).</cite> [<a href="https://doi.org/10.14814/phy2.12661" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4760405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26733244/" class="usa-link">PubMed</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Ahmed, E. A., Rajendran, P. &amp; Scherthan, H. The microRNA-202 as a diagnostic biomarker and a potential tumor suppressor. <em>Int. J. Mol. Sci.</em><strong>23</strong>(11), 5870 (2022).</cite> [<a href="https://doi.org/10.3390/ijms23115870" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9180238/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35682549/" class="usa-link">PubMed</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Costanzo, M., De Giglio, M. A. R. &amp; Roviello, G. N. Deciphering the relationship between SARS-CoV-2 and cancer. <em>Int. J. Mol. Sci.</em><strong>24</strong>(9), 7803 (2023).</cite> [<a href="https://doi.org/10.3390/ijms24097803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10178366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37175509/" class="usa-link">PubMed</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Kimura, M. et al. MicroRNAs in infectious diseases: potential diagnostic biomarkers and therapeutic targets. <em>Clin. Microbiol. Rev.</em><strong>36</strong> (4), e0001523 (2023).
</cite> [<a href="https://doi.org/10.1128/cmr.00015-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10732047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37909789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kimura,%20M.%20et%20al.%20MicroRNAs%20in%20infectious%20diseases:%20potential%20diagnostic%20biomarkers%20and%20therapeutic%20targets.%20Clin.%20Microbiol.%20Rev.36%20(4),%20e0001523%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Liu, X. et al. Non-coding RNAs expression in SARS-CoV-2 infection: pathogenesis, clinical significance, and therapeutic targets. <em>Signal. Transduct. Target. Ther.</em><strong>8</strong> (1), 441 (2023).
</cite> [<a href="https://doi.org/10.1038/s41392-023-01669-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10700414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38057315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20X.%20et%20al.%20Non-coding%20RNAs%20expression%20in%20SARS-CoV-2%20infection:%20pathogenesis,%20clinical%20significance,%20and%20therapeutic%20targets.%20Signal.%20Transduct.%20Target.%20Ther.8%20(1),%20441%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Janga, H. et al. Non-coding RNA networks in infection. <em>Methods Mol. Biol.</em><strong>2883</strong>, 53–77 (2025).
</cite> [<a href="https://doi.org/10.1007/978-1-0716-4290-0_3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39702704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Janga,%20H.%20et%20al.%20Non-coding%20RNA%20networks%20in%20infection.%20Methods%20Mol.%20Biol.2883,%2053%E2%80%9377%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Tamgue, O. et al. Non-Coding RNAs in the etiology and control of major and neglected human tropical diseases. <em>Front. Immunol.</em><strong>12</strong>, 703936 (2021).
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.703936" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8560798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34737736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tamgue,%20O.%20et%20al.%20Non-Coding%20RNAs%20in%20the%20etiology%20and%20control%20of%20major%20and%20neglected%20human%20tropical%20diseases.%20Front.%20Immunol.12,%20703936%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>Zhang, Y. et al. Comprehensive review for non-coding rnas: from mechanisms to therapeutic applications. <em>Biochem. Pharmacol.</em><strong>224</strong>, 116218 (2024).
</cite> [<a href="https://doi.org/10.1016/j.bcp.2024.116218" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38643906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20Comprehensive%20review%20for%20non-coding%20rnas:%20from%20mechanisms%20to%20therapeutic%20applications.%20Biochem.%20Pharmacol.224,%20116218%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Baptista, B. et al. Non-coding rnas: emerging from the discovery to therapeutic applications. <em>Biochem. Pharmacol.</em><strong>189</strong>, 114469 (2021).
</cite> [<a href="https://doi.org/10.1016/j.bcp.2021.114469" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33577888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baptista,%20B.%20et%20al.%20Non-coding%20rnas:%20emerging%20from%20the%20discovery%20to%20therapeutic%20applications.%20Biochem.%20Pharmacol.189,%20114469%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Garbo, S. et al. Next RNA therapeutics: the mine of Non-Coding. <em>Int. J. Mol. Sci.</em><strong>23</strong>(13), 7471 (2022).</cite> [<a href="https://doi.org/10.3390/ijms23137471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9267739/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35806476/" class="usa-link">PubMed</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Lagos, D. et al. miR-132 regulates antiviral innate immunity through suppression of the p300 transcriptional co-activator. <em>Nat. Cell. Biol.</em><strong>12</strong> (5), 513–519 (2010).
</cite> [<a href="https://doi.org/10.1038/ncb2054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20418869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lagos,%20D.%20et%20al.%20miR-132%20regulates%20antiviral%20innate%20immunity%20through%20suppression%20of%20the%20p300%20transcriptional%20co-activator.%20Nat.%20Cell.%20Biol.12%20(5),%20513%E2%80%93519%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Zhang, F. et al. MicroRNA-132-3p suppresses type I IFN response through targeting IRF1 to facilitate H1N1 influenza A virus infection. <em>Biosci. Rep.</em><strong>39</strong>(12), BSR20192769 (2019).</cite> [<a href="https://doi.org/10.1042/BSR20192769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6904772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31746331/" class="usa-link">PubMed</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite>Liu, Y. et al. Up-regulation of miR-146b-3p protects septic mice with acute respiratory distress syndrome by inhibiting PI3K/AKT signaling pathway. <em>J. Bioenerg Biomembr.</em><strong>52</strong> (4), 229–236 (2020).
</cite> [<a href="https://doi.org/10.1007/s10863-020-09839-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7266652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32488541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20Up-regulation%20of%20miR-146b-3p%20protects%20septic%20mice%20with%20acute%20respiratory%20distress%20syndrome%20by%20inhibiting%20PI3K/AKT%20signaling%20pathway.%20J.%20Bioenerg%20Biomembr.52%20(4),%20229%E2%80%93236%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>Haddad, H. &amp; Walid, A. Z. MiRNA target prediction might explain the reduced transmission of SARS-CoV-2 in jordan, middle East. <em>Noncoding RNA Res.</em><strong>5</strong> (3), 135–143 (2020).
</cite> [<a href="https://doi.org/10.1016/j.ncrna.2020.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7439007/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32839745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haddad,%20H.%20&amp;%20Walid,%20A.%20Z.%20MiRNA%20target%20prediction%20might%20explain%20the%20reduced%20transmission%20of%20SARS-CoV-2%20in%20jordan,%20middle%20East.%20Noncoding%20RNA%20Res.5%20(3),%20135%E2%80%93143%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371099/bin/41598_2025_15248_MOESM1_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (2MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The control and patient samples are being further analyzed for other biomolecules and hence not available. The datasets generated and/or analysed during the current study are available in the BioProject repository under accession number: PRJNA1049129 (https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA1049129).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-15248-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_15248.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (8.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371099/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371099/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371099%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371099/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371099/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371099/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841549/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371099/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841549/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371099/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371099/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="IqN46TQ21PbM1N4p9BJliGvWQMuEB36EiDom2QjFHwRzakOQ8MMzGpVSWeAJ1FaM">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
