
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Phenotypic plasticity in maize grain yield: Genetic and environmental insights of response to environmental gradients - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5312328AEF27630512320002FA0D25.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332327/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Phenotypic plasticity in maize grain yield: Genetic and environmental insights of response to environmental gradients">
<meta name="citation_author" content="Fatma Ozair">
<meta name="citation_author_institution" content="Interdisciplinary Graduate Program in Genetics and Genomics (Department of Biochemistry and Biophysics), Texas A&amp;M University, College Station, Texas, USA">
<meta name="citation_author" content="Alper Adak">
<meta name="citation_author_institution" content="Department of Soil and Crop Sciences, Texas A&amp;M University, College Station, Texas, USA">
<meta name="citation_author" content="Seth C Murray">
<meta name="citation_author_institution" content="Department of Soil and Crop Sciences, Texas A&amp;M University, College Station, Texas, USA">
<meta name="citation_author" content="Ryan T Alpers">
<meta name="citation_author_institution" content="Department of Plant and Agroecosystem Sciences, University of Wisconsin–Madison, Madison, Wisconsin, USA">
<meta name="citation_author" content="Alejandro C Aviles">
<meta name="citation_author_institution" content="Bayer Crop Science, St. Louis, Missouri, USA">
<meta name="citation_author" content="Dayane C Lima">
<meta name="citation_author_institution" content="Department of Plant and Agroecosystem Sciences, University of Wisconsin–Madison, Madison, Wisconsin, USA">
<meta name="citation_author" content="Jode Edwards">
<meta name="citation_author_institution" content="USDA‐ARS Corn Insect and Crop Genetics Research Unit, Ames, Iowa, USA">
<meta name="citation_author" content="David Ertl">
<meta name="citation_author_institution" content="Iowa Corn Promotion Board, Johnston, Iowa, USA">
<meta name="citation_author" content="Michael A Gore">
<meta name="citation_author_institution" content="Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, USA">
<meta name="citation_author" content="Candice N Hirsch">
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota, USA">
<meta name="citation_author" content="Joseph E Knoll">
<meta name="citation_author_institution" content="USDA‐ARS Crop Genetics and Breeding Research Unit, Tifton, Georgia, USA">
<meta name="citation_author" content="James C Schnable">
<meta name="citation_author_institution" content="Department of Agronomy and Horticulture, University of Nebraska–Lincoln, Lincoln, Nebraska, USA">
<meta name="citation_author" content="Maninder P Singh">
<meta name="citation_author_institution" content="Department of Plant, Soil and Microbial Sciences, Michigan State University, East Lansing, Michigan, USA">
<meta name="citation_author" content="Erin E Sparks">
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware, USA">
<meta name="citation_author" content="Addie Thompson">
<meta name="citation_author_institution" content="Department of Plant, Soil and Microbial Sciences, Michigan State University, East Lansing, Michigan, USA">
<meta name="citation_author" content="Teclemariam Weldekidan">
<meta name="citation_author_institution" content="Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware, USA">
<meta name="citation_author" content="Wenwei Xu">
<meta name="citation_author_institution" content="Texas A&amp;M AgriLife Research, Texas A&amp;M University System, Lubbock, Texas, USA">
<meta name="citation_publication_date" content="2025 Aug 7">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70078">
<meta name="citation_doi" content="10.1002/tpg2.70078">
<meta name="citation_pmid" content="40776458">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332327/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332327/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332327/pdf/TPG2-18-e70078.pdf">
<meta name="description" content="Understanding genotype‐by‐environment (G × E) interactions that underlie phenotypic variation, when observed for complex traits in multi‐environment trials, is important for biological discovery and for crop improvement. The regression‐on‐the‐mean ...">
<meta name="og:title" content="Phenotypic plasticity in maize grain yield: Genetic and environmental insights of response to environmental gradients">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Understanding genotype‐by‐environment (G × E) interactions that underlie phenotypic variation, when observed for complex traits in multi‐environment trials, is important for biological discovery and for crop improvement. The regression‐on‐the‐mean ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332327/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332327">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70078"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70078.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332327%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332327/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332327/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332327/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70078" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 7;18(3):e70078. doi: <a href="https://doi.org/10.1002/tpg2.70078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70078</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Phenotypic plasticity in maize grain yield: Genetic and environmental insights of response to environmental gradients</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ozair%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Fatma Ozair</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Fatma Ozair</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Interdisciplinary Graduate Program in Genetics and Genomics (Department of Biochemistry and Biophysics), Texas A&amp;M University, College Station, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ozair%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fatma Ozair</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Adak%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Alper Adak</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Alper Adak</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Soil and Crop Sciences, Texas A&amp;M University, College Station, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Adak%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alper Adak</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Murray%20SC%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Seth C Murray</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Seth C Murray</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Soil and Crop Sciences, Texas A&amp;M University, College Station, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Murray%20SC%22%5BAuthor%5D" class="usa-link"><span class="name western">Seth C Murray</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alpers%20RT%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ryan T Alpers</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ryan T Alpers</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Plant and Agroecosystem Sciences, University of Wisconsin–Madison, Madison, Wisconsin, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alpers%20RT%22%5BAuthor%5D" class="usa-link"><span class="name western">Ryan T Alpers</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aviles%20AC%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Alejandro C Aviles</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Alejandro C Aviles</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Bayer Crop Science, St. Louis, Missouri, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aviles%20AC%22%5BAuthor%5D" class="usa-link"><span class="name western">Alejandro C Aviles</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lima%20DC%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Dayane C Lima</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Dayane C Lima</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Plant and Agroecosystem Sciences, University of Wisconsin–Madison, Madison, Wisconsin, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lima%20DC%22%5BAuthor%5D" class="usa-link"><span class="name western">Dayane C Lima</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Edwards%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jode Edwards</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jode Edwards</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>USDA‐ARS Corn Insect and Crop Genetics Research Unit, Ames, Iowa, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Edwards%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jode Edwards</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ertl%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">David Ertl</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">David Ertl</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Iowa Corn Promotion Board, Johnston, Iowa, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ertl%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Ertl</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gore%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Michael A Gore</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Michael A Gore</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gore%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael A Gore</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hirsch%20CN%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Candice N Hirsch</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Candice N Hirsch</span></h3>
<div class="p">
<sup>
<sup>8</sup>
</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hirsch%20CN%22%5BAuthor%5D" class="usa-link"><span class="name western">Candice N Hirsch</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Knoll%20JE%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Joseph E Knoll</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Joseph E Knoll</span></h3>
<div class="p">
<sup>
<sup>9</sup>
</sup>USDA‐ARS Crop Genetics and Breeding Research Unit, Tifton, Georgia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Knoll%20JE%22%5BAuthor%5D" class="usa-link"><span class="name western">Joseph E Knoll</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schnable%20JC%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">James C Schnable</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">James C Schnable</span></h3>
<div class="p">
<sup>
<sup>10</sup>
</sup>Department of Agronomy and Horticulture, University of Nebraska–Lincoln, Lincoln, Nebraska, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schnable%20JC%22%5BAuthor%5D" class="usa-link"><span class="name western">James C Schnable</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20MP%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Maninder P Singh</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Maninder P Singh</span></h3>
<div class="p">
<sup>
<sup>11</sup>
</sup>Department of Plant, Soil and Microbial Sciences, Michigan State University, East Lansing, Michigan, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20MP%22%5BAuthor%5D" class="usa-link"><span class="name western">Maninder P Singh</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sparks%20EE%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Erin E Sparks</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Erin E Sparks</span></h3>
<div class="p">
<sup>
<sup>12</sup>
</sup>Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sparks%20EE%22%5BAuthor%5D" class="usa-link"><span class="name western">Erin E Sparks</span></a>
</div>
</div>
<sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thompson%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Addie Thompson</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Addie Thompson</span></h3>
<div class="p">
<sup>
<sup>11</sup>
</sup>Department of Plant, Soil and Microbial Sciences, Michigan State University, East Lansing, Michigan, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thompson%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Addie Thompson</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weldekidan%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Teclemariam Weldekidan</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Teclemariam Weldekidan</span></h3>
<div class="p">
<sup>
<sup>12</sup>
</sup>Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weldekidan%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Teclemariam Weldekidan</span></a>
</div>
</div>
<sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Wenwei Xu</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Wenwei Xu</span></h3>
<div class="p">
<sup>
<sup>13</sup>
</sup>Texas A&amp;M AgriLife Research, Texas A&amp;M University System, Lubbock, Texas, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenwei Xu</span></a>
</div>
</div>
<sup>13</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270078-aff-0001">
<sup>
<sup>1</sup>
</sup>Interdisciplinary Graduate Program in Genetics and Genomics (Department of Biochemistry and Biophysics), Texas A&amp;M University, College Station, Texas, USA</div>
<div id="tpg270078-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Soil and Crop Sciences, Texas A&amp;M University, College Station, Texas, USA</div>
<div id="tpg270078-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Plant and Agroecosystem Sciences, University of Wisconsin–Madison, Madison, Wisconsin, USA</div>
<div id="tpg270078-aff-0004">
<sup>
<sup>4</sup>
</sup>Bayer Crop Science, St. Louis, Missouri, USA</div>
<div id="tpg270078-aff-0005">
<sup>
<sup>5</sup>
</sup>USDA‐ARS Corn Insect and Crop Genetics Research Unit, Ames, Iowa, USA</div>
<div id="tpg270078-aff-0006">
<sup>
<sup>6</sup>
</sup>Iowa Corn Promotion Board, Johnston, Iowa, USA</div>
<div id="tpg270078-aff-0007">
<sup>
<sup>7</sup>
</sup>Plant Breeding and Genetics Section, School of Integrative Plant Science, Cornell University, Ithaca, New York, USA</div>
<div id="tpg270078-aff-0008">
<sup>
<sup>8</sup>
</sup>Department of Agronomy and Plant Genetics, University of Minnesota, St. Paul, Minnesota, USA</div>
<div id="tpg270078-aff-0009">
<sup>
<sup>9</sup>
</sup>USDA‐ARS Crop Genetics and Breeding Research Unit, Tifton, Georgia, USA</div>
<div id="tpg270078-aff-0010">
<sup>
<sup>10</sup>
</sup>Department of Agronomy and Horticulture, University of Nebraska–Lincoln, Lincoln, Nebraska, USA</div>
<div id="tpg270078-aff-0011">
<sup>
<sup>11</sup>
</sup>Department of Plant, Soil and Microbial Sciences, Michigan State University, East Lansing, Michigan, USA</div>
<div id="tpg270078-aff-0012">
<sup>
<sup>12</sup>
</sup>Department of Plant and Soil Sciences, University of Delaware, Newark, Delaware, USA</div>
<div id="tpg270078-aff-0013">
<sup>
<sup>13</sup>
</sup>Texas A&amp;M AgriLife Research, Texas A&amp;M University System, Lubbock, Texas, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Seth C. Murray, Department of Soil and Crop Sciences, Texas A&amp;M University, College Station, TX 77843‐2474, USA. Email: <span>sethmurray@tamu.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Apr 16; Received 2024 Aug 19; Accepted 2025 Jun 2; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America. This article has been contributed to by U.S. Government employees and their work is in the public domain in the USA.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332327  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40776458/" class="usa-link">40776458</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Understanding genotype‐by‐environment (G × E) interactions that underlie phenotypic variation, when observed for complex traits in multi‐environment trials, is important for biological discovery and for crop improvement. The regression‐on‐the‐mean model is an approach to observe G × E trends for complex traits across a gradient of environmental inputs. Biologically relevant environmental index values can be utilized to quantify phenotypic plasticity of individuals by correlating environmental means and environmental parameters within specific time windows. By accounting for trait stability, improvements can be made in genome‐wide association studies and genomic prediction models involving data with high volumes of environments, genotypes, and their interaction effects. Here, field data collected through the national hybrid maize (<em>Zea mays</em> L.) Genomes‐to‐Fields project was analyzed. Reaction norm parameters were obtained from photothermal ratio (PTR) indices for hybrid grain yield (GY) using three separate tester populations across 29 diverse environments. The PTR time windows most correlated with the average GY were discovered to differ by tester but were confounded by region. Using 100,000 single‐nucleotide polymorphisms (SNPs), we discovered 96 quantitative trait loci (QTLs) significantly associated with GY and six QTLs significantly associated with GY stability. The modified, regression‐on‐the‐mean genomic prediction model using PTR‐estimated reaction norm parameters of each hybrid worked nearly as well as a traditional, additive genomic prediction model using the G × E interaction terms but took 192× less time. The PTR genomic prediction model predicted untested environment performance (0.57–0.71) better than untested hybrid performance (0.26–0.37). This study suggests improved potential for multi‐environment genomic predictions by incorporating environmental measures to dissect the complexities of differential performance of genotypes across environments.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270078-list-10001" class="list" style="list-style-type:disc">
<li><p>Developmentally sensitive timepoints for grain yield were revealed by temporal photothermal ratio (PTR) data.</p></li>
<li><p>Hybrid grain yield plasticity was quantified by modeling genotype‐by‐environment (G × E) interactions with a PTR index.</p></li>
<li><p>PTR‐derived values allowed for hybrid genomic prediction scenarios for grain yield without prior performance data.</p></li>
<li><p>The regression‐on‐the‐mean model had comparable GY prediction ability to the additive model but with reduced computational burden.</p></li>
<li><p>Loci and candidate genes accounting for grain yield plasticity were identified by PTR‐derived data.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270078-dl-0001">
<dt>BLUP</dt>
<dd><p>best linear unbiased predictor</p></dd>
<dt>G × E</dt>
<dd><p>genotype‐by‐environment</p></dd>
<dt>G2F</dt>
<dd><p>Genomes‐to‐Fields</p></dd>
<dt>GDD</dt>
<dd><p>growing degree days</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association studies</p></dd>
<dt>GY</dt>
<dd><p>grain yield</p></dd>
<dt>MAGIC</dt>
<dd><p>multiparent advanced generation intercross</p></dd>
<dt>PTR</dt>
<dd><p>photothermal ratio</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>SNP</dt>
<dd><p>single‐nucleotide polymorphism</p></dd>
</dl>
<section id="tpg270078-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>The phenotypic development and growth of crops are largely influenced by the environmental conditions they are exposed to during their life cycle. Crops, being stationary sessile species, evolved to adapt to their local surroundings by sensing environmental stimuli and producing biological responses best suited for their survival. (Pierik &amp; Testerink, <a href="#tpg270078-bib-0040" class="usa-link" aria-describedby="tpg270078-bib-0040">2014</a>). Genotype‐by‐environment (G × E) interactions and phenotypic plasticity are key concepts in the study of crop adaptation and resilience. The G × E interaction refers to the differential performance of genotypes across various environmental conditions, indicating that the success of a particular genetic makeup is highly dependent on the environment in which it is placed. Phenotypic plasticity, on the other hand, is the ability of a single genotype to produce different phenotypes in response to changing environmental conditions, thus playing a crucial role in how crops adapt to various stresses and climatic variations. The critical role of G × E interactions and phenotypic plasticity has been shown in the adaptation, resilience, and yield stability of crops. Understanding these dynamics is vital for genetics, crop improvement, and plant breeding programs to effectively respond to the challenges posed by climate change and environmental variability. The collection of all abiotic and biotic factors imposed on a crop throughout growth directly contributes to the trends observed for G × E and phenotypic plasticity; therefore, it is crucial to consider a multitude of genotypes and environments to uncover these patterns. One approach to quantify phenotypic plasticity is by examining the reaction norms of genotypes, which are graphical representations showing the range of phenotypes expressed by a single genotype across a gradient of environmental conditions (Fu &amp; Wang, <a href="#tpg270078-bib-0014" class="usa-link" aria-describedby="tpg270078-bib-0014">2023</a>; Guo et al., <a href="#tpg270078-bib-0017" class="usa-link" aria-describedby="tpg270078-bib-0017">2020</a>; Li et al., <a href="#tpg270078-bib-0024" class="usa-link" aria-describedby="tpg270078-bib-0024">2018</a>). The Finlay–Wilkinson method for regression analysis of phenotypes, also known as the regression‐on‐the‐mean model, has been widely used as an approach to evaluate trait stability by regressing the trait value of individual genotypes against an index of the mean performance of a trait across several environments (Finlay &amp; Wilkinson, <a href="#tpg270078-bib-0013" class="usa-link" aria-describedby="tpg270078-bib-0013">1963</a>). The slope of a genotype's reaction norm is particularly insightful because it provides a measure of the degree of plasticity: Steeper slopes indicate greater plasticity, as the phenotype changes more markedly with environmental variation. Various methods of defining a continuous, environmental component to incorporate into reaction norms have been proposed to better understand underlying patterns of G × E in crop species (Choquette et al., <a href="#tpg270078-bib-0007" class="usa-link" aria-describedby="tpg270078-bib-0007">2023</a>; de los Campos et al., <a href="#tpg270078-bib-0011" class="usa-link" aria-describedby="tpg270078-bib-0011">2020</a>; Jarquín et al., <a href="#tpg270078-bib-0019" class="usa-link" aria-describedby="tpg270078-bib-0019">2017</a>).</p>
<p>Maize (<em>Zea mays</em> L.) is among the most notable model species studied for phenotypic plasticity due to its importance as a common crop grown across diverse environments (Kusmec et al., <a href="#tpg270078-bib-0023" class="usa-link" aria-describedby="tpg270078-bib-0023">2017</a>; Oostra et al., <a href="#tpg270078-bib-0037" class="usa-link" aria-describedby="tpg270078-bib-0037">2018</a>). Historical cultivation of maize landraces adapted to different geographic regions, beyond its center of origin in Central America, demonstrates adaptation potential while understanding the genetic and phenotypic variability of populations (Arteaga et al., <a href="#tpg270078-bib-0005" class="usa-link" aria-describedby="tpg270078-bib-0005">2016</a>; Prasanna, <a href="#tpg270078-bib-0041" class="usa-link" aria-describedby="tpg270078-bib-0041">2012</a>; Rivas et al., <a href="#tpg270078-bib-0042" class="usa-link" aria-describedby="tpg270078-bib-0042">2022</a>). Research on phenotypic plasticity and reaction norms in maize have significantly advanced our understanding of how crops can be bred and managed for better performance under diverse conditions. For instance, studying the effect of artificial selection on phenotypic plasticity in maize highlighted the importance of the genetic basis of G × E interactions to identify adaptive limitations in modern germplasm suited to temperate North American regions (Gage et al., <a href="#tpg270078-bib-0015" class="usa-link" aria-describedby="tpg270078-bib-0015">2017</a>)​. Their approach used genome‐wide association studies (GWAS) to explore the genomic locations associated with these interactions, providing insights into the regions of the genome involved in phenotypic plasticity. While plant breeders and researchers are aware that maize traits display variable sensitivity to different environmental parameters, the underlying physiology and alleles causing these differences remain largely unknown and untested across diverse environments for agronomic traits of interest. Many environments are needed to dissect the various environmental parameters influencing maize traits at the gene and genotype levels.</p>
<p>A separate study in maize, focusing on quantifying phenotypic plasticity to improve maize yields, emphasized the complexity of breeding decisions due to the intricate relationship between yield, plastic responses to multiple stresses, and yield‐related genes (Kusmec et al., <a href="#tpg270078-bib-0022" class="usa-link" aria-describedby="tpg270078-bib-0022">2018</a>)​​. The findings suggested that identifying loci conferring plasticity or stability from exotic germplasm is essential for maximizing future adaptation and yield of maize hybrids. Elite maize lines selected in optimal US Midwest environments and cultivation for specific traits have diminished the genetic diversity of hybrids throughout successive generations of breeding (Gerke et al., <a href="#tpg270078-bib-0016" class="usa-link" aria-describedby="tpg270078-bib-0016">2015</a>; Khoury et al., <a href="#tpg270078-bib-0021" class="usa-link" aria-describedby="tpg270078-bib-0021">2021</a>). The introgression of rare, favorable alleles from exotic germplasm unlocks the potential to improve traits influenced by G × E and plasticity through the selection process (Wang et al., <a href="#tpg270078-bib-0047" class="usa-link" aria-describedby="tpg270078-bib-0047">2017</a>).</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270078-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270078-list-0001" class="list" style="list-style-type:disc">
<li><p>Developmentally sensitive timepoints for grain yield were revealed by temporal photothermal ratio (PTR) data.</p></li>
<li><p>Hybrid grain yield plasticity was quantified by modeling genotype‐by‐environment (G × E) interactions with a PTR index.</p></li>
<li><p>PTR‐derived values allowed for hybrid genomic prediction scenarios for grain yield without prior performance data.</p></li>
<li><p>The regression‐on‐the‐mean model had comparable GY prediction ability to the additive model but with reduced computational burden.</p></li>
<li><p>Loci and candidate genes accounting for grain yield plasticity were identified by PTR‐derived data.</p></li>
</ul></section></section><p>Phenotypic plasticity's contribution to maize adaptation and heterosis further demonstrates the significance of understanding underlying genetic architecture (N. Liu et al., <a href="#tpg270078-bib-0029" class="usa-link" aria-describedby="tpg270078-bib-0029">2020</a>). By modeling genotype and environment interactions, valuable insights have been made into the effects of specific genetic loci on adaptation‐related traits and heterosis, underlining the importance of quantitative trait loci (QTLs) or quantitative trait nucleotides effects across different environments. Optimization of hybrids tolerating specific environmental stresses is possible in maize by leveraging heterosis in addition to the selection for additive genetic trait plasticity.</p>
<p>A complex genetic architecture underlying the plasticity of maize agronomic traits has revealed a dynamic interplay between genetic diversity, environmental variation, and their interactions​​ (Jin et al., <a href="#tpg270078-bib-0020" class="usa-link" aria-describedby="tpg270078-bib-0020">2023</a>). For example, altitude‐related environmental factors were found to be drivers for variation in flowering time but not in plant architecture and yield traits, underscoring the importance of site‐specific breeding strategies for local adaptation for various traits.</p>
<p>The genetic interconnection between, for instance, yield and yield stability is a cornerstone of crop breeding and agronomy, demonstrating the intricate balance between local genetic selection and environmental adaptability. This linkage suggests that the successful selection of crops for high yield does not come synergistically with or at the expense of yield stability, but rather, these two crucial traits can be improved concurrently (Tollenaar &amp; Lee, <a href="#tpg270078-bib-0044" class="usa-link" aria-describedby="tpg270078-bib-0044">2002</a>).</p>
<p>Coordinated research initiatives across numerous broad environments, like the Genomes‐to‐Fields (G2F) G × E project (<a href="https://www.genomes2fields.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genomes2fields.org/</a>), are needed to further explore and validate the genetic relationships between yield and yield stability across a wide range of environments. The G2F G × E project from 2014 to 2023 has trialed approximately 180,000 field plots involving more than 2500 maize hybrids across 162 unique environments in North America, the largest public sector cooperative project of its type. The hybrids tested, however, have changed across the years (Lima, Washburn, et al., <a href="#tpg270078-bib-0027" class="usa-link" aria-describedby="tpg270078-bib-0027">2023</a>).</p>
<p>To understand the genetic basis of the interplay between G × E and phenotypic plasticity, G2F datasets on maize tester hybrids across 29 environments in 2020 and 2021 were analyzed because these years used the same hybrids. Comprehensive insight into the genetic architecture of phenotypic plasticity was investigated for grain yield (GY). This study aimed to (i) observe trends in the variation of GY across multiple environments with diverse gradients, (ii) determine environmental index values for reaction norm modeling using the mean values of a weather parameter across time windows that are highly correlated with the mean GY across all environments, (iii) examine the reaction norm for GY using parameters (e.g., slope, intercept) of the linear model by fitting the individual hybrids’ performances to the environmental index, (iv) identify QTL and candidate genes associated with GY and its stability, and (v) assess the findings of this study by comparing the metrics of different G × E genomic prediction models.</p></section><section id="tpg270078-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270078-sec-0040"><h3 class="pmc_sec_title">2.1. Plant material and growing environments</h3>
<p>In this study, the 2020 and 2021 G2F project's maize hybrids, detailed in Lima, Aviles, et al. (<a href="#tpg270078-bib-0026" class="usa-link" aria-describedby="tpg270078-bib-0026">2023</a>), were planted in 29 environments from a combination of year:location. A subset of 220 inbreds, selected from 779 doubled haploid inbreds derived from the Stiff Stalk MAGIC (multiparent advanced generation intercross) population (Michel et al., <a href="#tpg270078-bib-0033" class="usa-link" aria-describedby="tpg270078-bib-0033">2022</a>), were used to advance hybrids grown across G2F environments in 2020 and 2021. These 220 inbreds were common across each of the three hybrid populations, which were created using three non‐Stiff Stalk expired plant variety protection tester lines (PHK76 [Pioneer Hi‐Bred International, Inc., <a href="#tpg270078-bib-0049" class="usa-link" aria-describedby="tpg270078-bib-0049">1988</a>], PHP02 [Pioneer Hi‐Bred International, Inc., <a href="#tpg270078-bib-0051" class="usa-link" aria-describedby="tpg270078-bib-0051">1989</a>], and PHZ51 [Pioneer Hi‐Bred International, Inc., <a href="#tpg270078-bib-0050" class="usa-link" aria-describedby="tpg270078-bib-0050">1987</a>]). The three hybrid populations with their tester and environment information are given in Figure <a href="#tpg270078-fig-0001" class="usa-link">1A</a>.</p>
<figure class="fig xbox font-sm" id="tpg270078-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7928/12332327/81c29057b9fb/TPG2-18-e70078-g002.jpg" loading="lazy" id="jats-graphic-1" height="306" width="546" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270078-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Data on Genomes‐to‐Fields (G2F) growing environments during 2020 and 2021, color‐coded by tester information for cultivated hybrids. (B) The arrangement of grain yield values for hybrids in ascending order. (C) Relative contribution of variances for grain yield (measured in t/ha). The variance of range effects was too small to be shown in the figure.</p></figcaption></figure><p>Measurements for GY were recorded in each environment based on standard operating procedures (SOP). SOP details were determined within the G2F project guidelines (<a href="https://www.genomes2fields.org/resources/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.genomes2fields.org/resources/</a>). In general, PHZ51 hybrids are adapted to southern regions, PHP02 to northern regions, and PHK76 to intermediate regions in the Midwest (Figure <a href="#tpg270078-fig-0001" class="usa-link">1A</a>). PHZ51, PHP02, and PHK76 were represented in 20, 14, and 11 environments, respectively. Overall, six locations (seven environments) featured all three tester populations together, such as Iowa and Delaware, and two locations featured two tester populations together. There were a total of 45 tester:environment combinations.</p></section><section id="tpg270078-sec-0050"><h3 class="pmc_sec_title">2.2. Best linear unbiased predictors (BLUPs) for GY</h3>
<p>The linear model fit to the data was of the following form: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline"><mrow><mrow><mrow><mi>Grain</mi><mspace width="0.28em"></mspace><mi>Yiel</mi></mrow><msub><mi mathvariant="normal">d</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub><mo>=</mo><mi mathvariant="normal">μ</mi><mo>+</mo><msub><mi>Genotype</mi><mi>i</mi></msub><mo>+</mo><msub><mi>Environment</mi><mi>j</mi></msub><mo>+</mo><msub><mrow><mspace width="0.28em"></mspace><mi>GE</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>+</mo><msub><mi>Range</mi><mi>k</mi></msub><mo>+</mo><msub><mi>Row</mi><mi>l</mi></msub><mo>+</mo><mi>Rep</mi><msub><mrow><mo stretchy="false">[</mo><mi>E</mi><mo stretchy="false">]</mo></mrow><mrow><mi>k</mi><mi>j</mi></mrow></msub><mo>+</mo><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><mrow><mi>Grain</mi><msub><mi>Yield</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub></mrow></mrow></math></span> represents the phenotypic data of GY (t/ha) for the <em>i</em>th hybrid in the <em>j</em>th environment, <em>k</em>th range, and <em>l</em>th row, <em>μ</em> is the grand mean of the trait, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><mrow><msub><mi>Genotype</mi><mi>i</mi></msub><mfrac><mrow><mi>i</mi><mi>i</mi><mi>d</mi></mrow><mo>∼</mo></mfrac><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><msub><mi>G</mi><mi>i</mi></msub><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> is the random effect of hybrid <em>i</em>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mrow><msub><mi>Environment</mi><mi>j</mi></msub><mfrac><mrow><mi>i</mi><mi>i</mi><mi>d</mi></mrow><mo>∼</mo></mfrac><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><msub><mi>E</mi><mi>j</mi></msub><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> is the random effect of environment <em>j</em>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><mrow><msub><mi>GE</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mfrac><mrow><mi>i</mi><mi>i</mi><mi>d</mi></mrow><mo>∼</mo></mfrac><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><msub><mi>GE</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> is the random effect of interactions between hybrid <em>i</em> and environment <em>j</em>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><mrow><msub><mi>Range</mi><mi>k</mi></msub><mfrac><mrow><mi>i</mi><mi>i</mi><mi>d</mi></mrow><mo>∼</mo></mfrac><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><msub><mi>Range</mi><mi>k</mi></msub><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mrow><msub><mi>Row</mi><mi>l</mi></msub><mfrac><mrow><mi>i</mi><mi>i</mi><mi>d</mi></mrow><mo>∼</mo></mfrac><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><msub><mi>Row</mi><mi>l</mi></msub><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> are the horizontal and lateral grids of plots within each environment (rows are parallel to planted rows and ranges are perpendicular), <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><mrow><mi>Rep</mi><msub><mrow><mo stretchy="false">[</mo><mi>E</mi><mo stretchy="false">]</mo></mrow><mrow><mi>k</mi><mi>j</mi></mrow></msub><mfrac><mrow><mi>i</mi><mi>i</mi><mi>d</mi></mrow><mo>∼</mo></mfrac><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>Rep</mi><msub><mrow><mo stretchy="false">[</mo><mi>E</mi><mo stretchy="false">]</mo></mrow><mrow><mi>k</mi><mi>j</mi></mrow></msub></mrow><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> is the replication effects found in each environment since each environment contains two replications, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><mrow><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub><mfrac><mrow><mi>i</mi><mi>i</mi><mi>d</mi></mrow><mo>∼</mo></mfrac><mi>N</mi><mrow><mo stretchy="false">(</mo><mrow><mn>0</mn><mo>,</mo><msubsup><mi mathvariant="normal">σ</mi><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow></msub><mn>2</mn></msubsup></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math></span> refers to the residual error. All model effects except the mean, <em>μ</em>, were considered random. Variance components were estimated from the full dataset with restricted maximum likelihood using the lmer() function contained in the lme4 R package (Bates et al., <a href="#tpg270078-bib-0002" class="usa-link" aria-describedby="tpg270078-bib-0002">2015</a>). The heritability for GY was calculated as follows:
</p>
<table class="disp-formula p" id="tpg270078-disp-0001"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7928/12332327/a54af97afa7f/TPG2-18-e70078-e011.jpg" loading="lazy" alt="2.2."></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi>i</mi><mn>2</mn></msubsup></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>i</mi><mi>j</mi></mrow><mn>2</mn></msubsup></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><msub><mi mathvariant="normal">σ</mi><msubsup><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi><mi>l</mi></mrow><mn>2</mn></msubsup></msub></mrow></math></span> are the variance components of the overall genotype effects, the interaction between genotype and environment, and the error term, respectively. BLUP values for GY were obtained from the combined effects of genotypes, environments, and their interaction (G × E). The BLUP values were retained for the subsequent genomic prediction and reaction norm calculations.</p></section><section id="tpg270078-sec-0060"><h3 class="pmc_sec_title">2.3. Environmental index</h3>
<p>The EnvRtype package in R was implemented to obtain daily weather data derived from the NASA Prediction of Worldwide Energy Resources (POWER) project (Costa‐Neto et al., <a href="#tpg270078-bib-0009" class="usa-link" aria-describedby="tpg270078-bib-0009">2021</a>). The environmental index was calculated based on the Pearson correlations between the photothermal ratio (PTR) and the mean value of the G × E BLUPs for GY across the growing environments separated by the three hybrid populations. The PTR is an environmental parameter that represents the ratio of day length (hour) divided by growing degree days (GDD) (Li et al., <a href="#tpg270078-bib-0025" class="usa-link" aria-describedby="tpg270078-bib-0025">2021</a>; B. Liu &amp; Heins, <a href="#tpg270078-bib-0028" class="usa-link" aria-describedby="tpg270078-bib-0028">2002</a>). The GDD was calculated based on <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mrow><mrow><mo stretchy="false">(</mo><mfrac><mrow><msub><mi>T</mi><mi>max</mi></msub><mo>−</mo><msub><mi>T</mi><mi>min</mi></msub></mrow><mn>2</mn></mfrac><mo stretchy="false">)</mo></mrow><mo>−</mo><msub><mi>T</mi><mi>base</mi></msub></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><msub><mi>T</mi><mi>max</mi></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><msub><mi>T</mi><mi>min</mi></msub></mrow></math></span> are the maximum and minimum temperatures (°C), respectively, recorded without thresholding, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline"><mrow><msub><mi>T</mi><mi>base</mi></msub></mrow></math></span> is the base temperature, set as 10°C (Costa‐Neto et al., <a href="#tpg270078-bib-0009" class="usa-link" aria-describedby="tpg270078-bib-0009">2021</a>; Soltani &amp; Sinclair, <a href="#tpg270078-bib-0043" class="usa-link" aria-describedby="tpg270078-bib-0043">2012</a>). Mean PTR values were calculated for windows of days of all sizes from a window of 1 day to 120 days. For each window size, average PTR values were calculated for all possible windows from 1 day after planting in 1‐day increments. For example, for a 5‐day window, the mean PTR value was calculated from Day 1 to 5, Day 2 to 6, …, up to Day 116 to 120 after planting. Subsequently, the mean PTR value from the window that demonstrated the highest correlation to the yield estimates in each environment was utilized as the environmental index for reaction norms.</p></section><section id="tpg270078-sec-0070"><h3 class="pmc_sec_title">2.4. Stability analysis</h3>
<p>The stability of GY was calculated based on the linear fit of scaled and centered GY of hybrids across the environmental index of environments. Then, slope and intercept were retrieved for each hybrid and used as a merit of stability and performance (Sjoberg et al., <a href="#tpg270078-bib-0052" class="usa-link" aria-describedby="tpg270078-bib-0052">2020</a>) using regression where the G × E BLUPs were set as the dependent variable and the PTR index values were set as the independent variable. The stability can be classified according to the following slope values: slope &lt; 1 where the performance was higher than expected in unfavorable environments compared to favorable environments based on the population average performance, slope = 0 where the performance was consistent across unfavorable and favorable environments, slope = 1 where the performance was consistent across environmental means, and slope &gt; 1 where the performance was higher than expected in favorable environments compared to unfavorable environments based on the population average performance (Bernardo, <a href="#tpg270078-bib-0053" class="usa-link" aria-describedby="tpg270078-bib-0053">2002</a>).</p></section><section id="tpg270078-sec-0080"><h3 class="pmc_sec_title">2.5. QTLs analysis</h3>
<p>Genomic data from the 220 inbreds in 2020 and 2021 obtained from CyVerse (<a href="https://doi.org/10.25739/hzzs-a865" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.25739/hzzs‐a865</a> and <a href="https://doi.org/10.25739/5ae3-sw62" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.25739/5ae3‐sw62</a>) were employed for QTL mapping. A total of 100,000 single‐nucleotide polymorphisms (SNPs) were selected, evenly distributed across 10 chromosomes. Further details related to genomic data can be found in Michel et al. (<a href="#tpg270078-bib-0033" class="usa-link" aria-describedby="tpg270078-bib-0033">2022</a>). Three types of phenotypes were used for dissecting GY of the three hybrid populations: (i) the mean GY of the hybrids, (ii) the slope value of the hybrids, and (iii) the intercept values of the hybrids. QTL mapping was conducted using R/qtl2, where the population type was set as “dh6,” in accordance with the development design of the inbreds of the six‐way doubled haploid Wisconsin MAGIC population (Broman et al., <a href="#tpg270078-bib-0006" class="usa-link" aria-describedby="tpg270078-bib-0006">2019</a>). R/qtl2::scan1perm() function was used to calculate the threshold for significant QTLs using 1000 permutations, and resulted in a limit of detection threshold of five to identify significant regions. The candidate genes within these intervals were identified using the Zm‐B73‐NAM‐5.0 reference genome on MaizeGDB (<a href="https://jbrowse.maizegdb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://jbrowse.maizegdb.org/</a>).</p></section><section id="tpg270078-sec-0090"><h3 class="pmc_sec_title">2.6. Environmental index facilitated genomic prediction</h3>
<p>An environmental index facilitated genomic prediction was conducted to predict the GY, validated using four common prediction scenarios related to a plant breeding program (Jarquín et al., <a href="#tpg270078-bib-0019" class="usa-link" aria-describedby="tpg270078-bib-0019">2017</a>). Bayesian generalized kernel regression genomic prediction models were applied to the three different hybrid populations separately using reproducing kernel Hilbert spaces from the BGLR package (Perez &amp; de los Campos, <a href="#tpg270078-bib-0038" class="usa-link" aria-describedby="tpg270078-bib-0038">2014</a>). The steps were as follows: (i) 220 hybrids were partitioned into tested and untested groups with percent ratios of approximately 70% and 30%, respectively. (ii) Environments were similarly separated into tested and untested categories using the leave‐one‐out method, where one environment was removed for the next step, designated as the untested environment, while the others were termed as tested environments. (iii) A linear fit was applied for each tested hybrid using their GY data and the environmental index (see Section <a href="#tpg270078-sec-0060" class="usa-link">2.3</a>) of the tested environments, followed by calculating the slope and intercept values. (iv) The genomic prediction model was trained using the slope and intercept values of the tested hybrids in the tested environments. (v) The slope and intercept values of both untested and tested hybrids were predicted using a genomic relationship matrix derived from 100,000 genomic markers using ridge regression (rrBLUP package) (Endelman, <a href="#tpg270078-bib-0012" class="usa-link" aria-describedby="tpg270078-bib-0012">2011</a>). The markers with a minor allele frequency less than 0.05 were excluded, and missing markers were imputed using expectation maximization. (vi) GY values of both untested and tested hybrids were calculated using their predicted slope and intercept values, along with the genomic relationship matrix estimated from the 100,000 SNPs. The two models assessed for prediction accuracy were based on (Malosetti, <a href="#tpg270078-bib-0054" class="usa-link" aria-describedby="tpg270078-bib-0054">2013</a>):</p>
<p>
<em>Model 1 (M1)</em>: Additive effects model
</p>
<table class="disp-formula p" id="tpg270078-disp-0002"><tr><td class="formula"><math id="jats-math-18" display="block"><mrow><mrow><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>G</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">G</mi><msub><mi mathvariant="normal">E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></math></span> is the GY of the <em>i</em>th hybrid in the <em>j</em>th environment, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><mi mathvariant="normal">μ</mi></mrow></math></span> is the overall mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-21" display="inline"><mrow><mrow><msub><mi>G</mi><mi>i</mi></msub><mspace width="0.28em"></mspace></mrow></mrow></math></span> is the genotype of the <em>i</em>th hybrid, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-22" display="inline"><mrow><mrow><msub><mi>E</mi><mi>j</mi></msub><mspace width="0.28em"></mspace></mrow></mrow></math></span> is the <em>j</em>th environment, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-23" display="inline"><mrow><mrow><mi mathvariant="normal">G</mi><msub><mi mathvariant="normal">E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></span> is the G × E of the <em>i</em>th hybrid in the <em>j</em>th environment, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-24" display="inline"><mrow><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></math></span> is the residual error.</p>
<p>
<em>Model 2 (M2)</em>: Regression on the mean model
</p>
<table class="disp-formula p" id="tpg270078-disp-0003"><tr><td class="formula"><math id="jats-math-25" display="block"><mrow><mrow><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>G</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>b</mi><mi>i</mi></msub><msub><mi>E</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi mathvariant="normal">ε</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow></math></td></tr></table>
<p>M2 retains factors from the M1 model, except <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-26" display="inline"><mrow><mrow><mi mathvariant="normal">G</mi><msub><mi mathvariant="normal">E</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mspace width="0.28em"></mspace></mrow></mrow></math></span> is replaced by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-27" display="inline"><mrow><mrow><msub><mi>b</mi><mi>i</mi></msub><msub><mi>E</mi><mi>j</mi></msub></mrow></mrow></math></span>, which is the <em>i</em>th hybrid's specific reaction norm parameter in the <em>j</em>th environment. Both models were run using Texas A&amp;M's High Performance Research Computer.</p>
<p>Subsequently, four types of cross‐validation schemes between the actual GY and the predicted GY were calculated for tested hybrids in tested environments, untested hybrids in tested environments, tested hybrids in untested environments, and untested hybrids in untested environments. This procedure was iterated 100 times for each cross‐validation scenario using different hybrids and environments summarized as the mean estimates and confidence intervals for each hybrid and environment for each set of testers.</p></section></section><section id="tpg270078-sec-0100"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270078-sec-0110"><h3 class="pmc_sec_title">3.1. Sources of variation for GY</h3>
<p>The variance component estimates for GY in Figure <a href="#tpg270078-fig-0001" class="usa-link">1C</a> revealed that the environment explained the highest variation (∼30%), which aligns with the amount of diverse environments included in the study. The G × E effect explained about 12% of the variation in GY. The effects of G × E on GY were further explored after confirming its contribution to phenotypic variation across the hybrids. The moderate model <em>R</em>
<sup>2</sup> value of ∼0.75 is likely reduced by the high contribution of unexplained variation in the residual error. The environments in which all three populations were planted consistently had lower BLUPs for PHZ51 and higher BLUPs for PHP02 (Figure <a href="#tpg270078-fig-0001" class="usa-link">1B</a>). The southern environments using PHZ51, which were distributed over the widest geographic locations, tended to have the lowest BLUP values overall.</p></section><section id="tpg270078-sec-0120"><h3 class="pmc_sec_title">3.2. Identifying the environmental index linked to GY</h3>
<p>Based on the sliding window correlation analysis between PTR and BLUPs of GY from each environment, specific windows were identified that showed the strongest correlations with GY separately across the three maize hybrid populations (PHK76, PHP02, and PHZ51) and across all tested environment (Figure <a href="#tpg270078-fig-0002" class="usa-link">2A–C</a>). In the case of PHK76, the greatest correlation was observed in the 2‐day window between 100 and 101 days (Figure <a href="#tpg270078-fig-0002" class="usa-link">2A</a>). For PHP02, the highest correlation was found in the 2‐day window of 67–68 days (Figure <a href="#tpg270078-fig-0002" class="usa-link">2B</a>). In the case of PHZ51, the most significant correlation spanned a broader period, specifically the 45‐day window from Day 46 to Day 90 (Figure <a href="#tpg270078-fig-0002" class="usa-link">2C</a>). The mean PTR value in these specific windows was designated as the environmental index. Later, the stability of hybrid GY was analyzed using linear fits built between the environmental index and GY (centered and scaled) of each environment, revealing different reaction norms of hybrids (Figure <a href="#tpg270078-fig-0002" class="usa-link">2D</a>).</p>
<figure class="fig xbox font-sm" id="tpg270078-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7928/12332327/47813d871580/TPG2-18-e70078-g005.jpg" loading="lazy" id="jats-graphic-3" height="304" width="546" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270078-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Environmental index using sliding window correlation between photothermal ratio (PTR) and average grain yield (t/ha) across environments for the three hybrid populations and their stability. Figure Panels (A) (PHK76), (B) (PHP02), and (C) (PHZ51) show the correlations between values of all possible windows of PTR with the grain yield (left), highlighting the highest correlations between specific windows and the mean grain yield across environments in hybrid populations. (D) Predicted grain yield (predicted GY, on the <em>y</em>‐axis) calculated by linearly regressing the scaled and centered grain yield of each hybrid on the environmental index (on the <em>x</em>‐axis) across each growing environment of three hybrid populations. Points are the predicted grain yield values. Each line, representing a hybrid, was colored based on the slope parameter of the linear fit to visualize the reaction norms of the hybrids.</p></figcaption></figure></section><section id="tpg270078-sec-0130"><h3 class="pmc_sec_title">3.3. QTLs linked to GY and its stability</h3>
<p>Several QTLs were discovered for GY across different environments and years, depending on the hybrid populations (Figure <a href="#tpg270078-fig-0003" class="usa-link">3</a>); their corresponding candidate genes are provided in Table <a href="#tpg270078-supinfo-0001" class="usa-link">S1</a>. A total of 96 unique QTLs were significantly associated with GY, with 24 belonging to PHK76, 34 to PHP02, and 38 to PHZ51. As most QTLs were unique to environments and testers, the PHZ51 population had the most QTLs across the chromosomes, likely due to this population being planted in the most environments compared to the other populations.</p>
<figure class="fig xbox font-sm" id="tpg270078-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7928/12332327/55fef88ff6f9/TPG2-18-e70078-g001.jpg" loading="lazy" id="jats-graphic-5" height="164" width="546" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270078-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Quantitative trait loci (QTLs) linked to grain yield in three different hybrid populations advanced by PHK76, PHP02, and PHZ51 testers across various growing environments and years.</p></figcaption></figure><p>The merit of GY stability across locations and years (29 environments) was quantified by slope and intercept values. Among the QTLs discovered, six relevant QTLs were identified related to GY stability as shown in Figure <a href="#tpg270078-fig-0004" class="usa-link">4</a> and Table <a href="#tpg270078-supinfo-0001" class="usa-link">S2</a>. Two of these loci were detected for both slope and intercept (PHK76 Chr 7 and PHZ51 Chr 7). The detected positions of these Chr 7 loci between the two testers were 6.7 Mb apart. The function of potential candidate genes related to GY stability for three hybrid populations is also given in Table <a href="#tpg270078-tbl-0001" class="usa-link">1</a>. In total, six candidate genes related to GY stability were provided across 29 environments and three hybrid populations.</p>
<figure class="fig xbox font-sm" id="tpg270078-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7928/12332327/813af87682b0/TPG2-18-e70078-g004.jpg" loading="lazy" id="jats-graphic-7" height="331" width="546" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270078-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Quantitative trait loci (QTLs) identified for slope and intercept of hybrids across three testers. Testers for each population are provided on the right. A limit of detection (LOD) threshold of five was used for significance.</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270078-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Quantitative trait loci (QTLs) discovered along with candidate genes and their functions for grain yield, slope, and intercept for three different maize populations with PHK76, PHP02 and PHZ51 testers.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Population</th>
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">SNP position (Mbp)</th>
<th align="left" rowspan="1" colspan="1">Chr</th>
<th align="left" rowspan="1" colspan="1">No. genes in interval</th>
<th align="left" rowspan="1" colspan="1">Likely candidate gene</th>
<th align="left" rowspan="1" colspan="1">Function</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">PHK76</td>
<td align="left" rowspan="1" colspan="1">Slope</td>
<td align="left" rowspan="1" colspan="1">142217610–142236552</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">
<em>Zm00001eb317590</em>
</td>
<td align="left" rowspan="1" colspan="1">E3 ubiquitin‐protein ligase RFWD2 (RING/U‐box superfamily protein) (ubiquitin‐protein ligase/zinc ion‐binding protein)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PHK76</td>
<td align="left" rowspan="1" colspan="1">Intercept</td>
<td align="left" rowspan="1" colspan="1">114599617–114602053</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">41</td>
<td align="left" rowspan="1" colspan="1">
<em>Zm00001eb089110</em>
</td>
<td align="left" rowspan="1" colspan="1">Alpha‐humulene/(−)‐(E)‐beta‐caryophyllene synthase</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PHK76</td>
<td align="left" rowspan="1" colspan="1">Intercept</td>
<td align="left" rowspan="1" colspan="1">181334786–181336694</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">50</td>
<td align="left" rowspan="1" colspan="1">
<em>Zm00001eb098510</em>
</td>
<td align="left" rowspan="1" colspan="1">60S ribosomal protein L33‐B (60S ribosomal protein L35a‐2)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PHK76</td>
<td align="left" rowspan="1" colspan="1">Intercept</td>
<td align="left" rowspan="1" colspan="1">142217610–142236552</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">
<em>Zm00001eb317590</em>
</td>
<td align="left" rowspan="1" colspan="1">E3 ubiquitin‐protein ligase RFWD2 (RING/U‐box superfamily protein) (ubiquitin‐protein ligase/zinc ion‐binding protein)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PHP02</td>
<td align="left" rowspan="1" colspan="1">Slope</td>
<td align="left" rowspan="1" colspan="1">171965158–171966322</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">
<em>Zm00001eb293730</em>
</td>
<td align="left" rowspan="1" colspan="1">Uncharacterized protein</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PHP02</td>
<td align="left" rowspan="1" colspan="1">Intercept</td>
<td align="left" rowspan="1" colspan="1">21039872–21045940</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">
<em>Zm00001eb376840</em>
</td>
<td align="left" rowspan="1" colspan="1">NB‐ARC domain‐containing protein</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PHZ51</td>
<td align="left" rowspan="1" colspan="1">Slope</td>
<td align="left" rowspan="1" colspan="1">148935616–148937490</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">69</td>
<td align="left" rowspan="1" colspan="1">
<em>Zm00001eb319470</em>
</td>
<td align="left" rowspan="1" colspan="1">Auxin‐responsive protein (IAA33–Aux/IAA‐transcription factor 33)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PHZ51</td>
<td align="left" rowspan="1" colspan="1">Intercept</td>
<td align="left" rowspan="1" colspan="1">148935616–148937490</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">84</td>
<td align="left" rowspan="1" colspan="1">
<em>Zm00001eb319470</em>
</td>
<td align="left" rowspan="1" colspan="1">Auxin‐responsive protein (IAA33–Aux/IAA‐transcription factor 33)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270078-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270078-tbl1-note-0001"><p>Abbreviation: SNP, single‐nucleotide polymorphism.</p></div></div></section></section><section id="tpg270078-sec-0140"><h3 class="pmc_sec_title">3.4. Environmental index‐based genomic prediction for GY</h3>
<p>A specific PTR window differentiated the environments for the three hybrid populations. The reaction norm of GY was then revealed as a response to day length and GDD, thanks to the linear regression of the hybrids' GY on the specific window of PTR across distinct environments (Figure <a href="#tpg270078-fig-0002" class="usa-link">2D</a>). Slope and intercept are the merit of the plasticity of each hybrid in two aspects: (i) slope is the identification of which hybrids are more or less favorable across varying environmental conditions and (ii) intercept is the identification of hybrids’ baseline yield potential. Environmental index facilitated genomic prediction provided different prediction abilities with small differences between PHK76, PHP02, and PHZ51 testers grown across 11, 14, and 20 environments (Figure <a href="#tpg270078-fig-0005" class="usa-link">5</a>). Both the M1 (additive effects model, with G × E interaction effects) and M2 (regression on the PTR mean model) performed similarly in most cases. M1 was better on tested hybrids in tested environments across all three tester populations. For both models, in tested hybrids in untested environments, the prediction ability was between ∼0.6 and 0.8 depending on the hybrid population, and M1 performed better in all cases. In untested hybrids in tested environments, the prediction ability dropped to ∼0.3–0.4, and M1 was numerically better for two of three testers. The most challenging prediction scenario, untested hybrids in untested environments, provided prediction abilities ranging between ∼0.2 and 0.4, and the M1 model performed numerically better for two of three testers.</p>
<figure class="fig xbox font-sm" id="tpg270078-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7928/12332327/6adf0480b017/TPG2-18-e70078-g003.jpg" loading="lazy" id="jats-graphic-9" height="549" width="546" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270078-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of two models for genomic prediction response. (A) Genomic prediction of grain yield facilitated using M1 (additive model with genotype‐by‐environment [G × E]) for four prediction scenarios illustrated by each facet and three hybrid populations illustrated by legend. The standard error is represented by the vertical lines above the bars. (B) Genomic prediction of grain yield facilitated using M2 (regression on the photothermal ratio [PTR] mean model).</p></figcaption></figure></section><section id="tpg270078-sec-0150"><h3 class="pmc_sec_title">3.5. Computational costs between the M1 and M2 models</h3>
<p>Although both of the genomic models incorporated G × E effects to predict GY, the computational burden with the M2 model was greatly reduced in comparison to the M1 model. Using the same computing infrastructure, the M1 model took 192× longer to run (24 days vs. 3 h). The M2 relies on G × E inputs derived from the slope and intercept values of reaction norms; thus, it consists of a matrix with the same amount of observations (rows) as the number of genotypes. On the other hand, the M1 model relies on fitting all the combinations of genotypes by environment, thus greatly increasing the number of observations as the amount of genotypes/environments increases. As shown in Figure <a href="#tpg270078-fig-0005" class="usa-link">5</a>, both models give comparable prediction accuracies. Therefore, the M2 model is preferable when using an optimal environmental parameter, datasets are large, and run time is important in decision‐making.</p></section></section><section id="tpg270078-sec-0160"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Overall, this MAGIC‐derived hybrid population was grown in a high number of environments and uniquely used multiple tester lines to ensure adaptation of the tested hybrids was relevant to the target environments. GY had moderately high heritability, in part due to the fact these were doubled haploid lines that were not pre‐selected for high GY. This high heritability allowed for the dissection of environmental factors and the genetic mapping of numerous loci.</p>
<section id="tpg270078-sec-0170"><h3 class="pmc_sec_title">4.1. Identifying the environmental index linked to GY</h3>
<p>The hybrids used in the G2F project data demonstrated substantial G × E interactions and phenotypic plasticity. Maize's adaptability and stability across diverse environmental conditions are crucial for maximizing GY over many diverse target environments and over unpredictable weather. This study uniquely contributes to the understanding of GY in maize by identifying specific environmental windows that correlated with yield variation across diverse environments. This highlights the significance of the temporal dimensions of environmental factors in maize development and GY, and the interaction with genetic factors. Identifying timepoints with the greatest effects on G × E interactions is useful to understand when to measure physiological aspects of the crop. Like this study, prior research has shown that maize and sorghum (<em>Sorghum bicolor</em> (L.) Moench) exhibit significant phenotypic plasticity in response to environmental stimuli (Jin et al., <a href="#tpg270078-bib-0020" class="usa-link" aria-describedby="tpg270078-bib-0020">2023</a>; Kusmec et al., <a href="#tpg270078-bib-0023" class="usa-link" aria-describedby="tpg270078-bib-0023">2017</a>; Li et al., <a href="#tpg270078-bib-0024" class="usa-link" aria-describedby="tpg270078-bib-0024">2018</a>; Millet et al., <a href="#tpg270078-bib-0034" class="usa-link" aria-describedby="tpg270078-bib-0034">2019</a>; Mu et al., <a href="#tpg270078-bib-0035" class="usa-link" aria-describedby="tpg270078-bib-0035">2022</a>). Photoperiodicity as well as heat sensitivity can affect crucial stages of vegetative and flowering development that ultimately affect GY (Bonhomme et al., <a href="#tpg270078-bib-0003" class="usa-link" aria-describedby="tpg270078-bib-0003">1994</a>; Lobell et al., <a href="#tpg270078-bib-0030" class="usa-link" aria-describedby="tpg270078-bib-0030">2013</a>; Wang et al., <a href="#tpg270078-bib-0046" class="usa-link" aria-describedby="tpg270078-bib-0046">2010</a>). Therefore, differentiating populations based on a specific environmental index that is biologically meaningful and variable can help to dissect the target environment G × E patterns at play during crucial developmental stages when assessing the performance of hybrids. Incorporation of environmental data in the form of numerically ordered reaction norms along with phenotypic data may also reveal the underlying variation among hybrids based on their sensitivity to local conditions (Piepho &amp; Blancon, <a href="#tpg270078-bib-0039" class="usa-link" aria-describedby="tpg270078-bib-0039">2023</a>). Each tester used for hybrid formation had unique genetic makeups that showed clear differences in the window sizes for PTR correlations to environmental mean yields. This was partially expected considering the range of environmental locations where the hybrids were planted and the inherent adaptations of the individual testers. However, the dramatic differences observed in window sizes across the populations, as early as 45 days for hybrids with PHZ51 testers and 100–101 days for hybrids with PHK76 testers, were surprising. Plotting each environment's weather parameters over time, we did not observe any distinct spikes or obvious causes for these specific windows, or the differences between them. These varying windows may be due to genetic differences, environmental factors, or their interactions, which we could not separate since these testers were confounded with tested regions in all but two locations. Among environmental factors, the day length affecting photoperiod and the confounding associations with temperature had a pronounced difference depending on geographic latitudes. Southern locations tend to have longer day lengths than the northern regions early in the year, but the opposite is true in summer. This, combined with differences in planting dates, results in southern oddities like Texas (a southern location), flowering as the days are getting longer. In contrast, hybrids in most US maize production and G2F locations flower as the days are getting shorter. The southern conditions for both photoperiod and temperature are not particularly suitable for the temperate‐adapted inbreds of the Wisconsin MAGIC population. This was intentionally modulated through tester choice in hybrids, which allowed hybrids with the same population of lines to be appropriately evaluated in all locations. Thus, an environmental index that captures the inherent environmental difference of multi‐environment field trials is crucial to establish relevant findings for a trait of interest to assess critical windows of development.</p></section><section id="tpg270078-sec-0180"><h3 class="pmc_sec_title">4.2. QTLs linked to GY and its stability</h3>
<p>The discovery of QTLs related to GY varied significantly across different environments (Figure <a href="#tpg270078-fig-0003" class="usa-link">3</a>; <a href="#tpg270078-supinfo-0001" class="usa-link">Data S1</a>), reflecting the varied genetic segregants impacting GY variation in maize. It should be noted that because segregating lines were evaluated on the same tester, QTLs reflect genetic segregation in the lines that are not masked by the tester. This is more likely to discover practically relevant QTLs as farmers grow hybrids, not inbreds. However, it is also expected that fewer QTLs would be discovered using hybrids instead of inbred lines per se. Only two QTLs of 96 corresponded across individual environments or testers. In terms of stability QTL, there was no correspondence of loci between testers, with the possible exception of the PHK76 and PHZ51 Chr 7 QTL. However, a 6.7 Mb difference in location is unlikely to have the same cause within this higher resolution population. This result of inconsistent loci detected for a complex trait like yield is expected and has been shown in previous genetic mapping studies of both inbred lines and hybrids (Mural et al., <a href="#tpg270078-bib-0036" class="usa-link" aria-describedby="tpg270078-bib-0036">2022</a>; Melchinger et al., <a href="#tpg270078-bib-0055" class="usa-link" aria-describedby="tpg270078-bib-0055">1998</a>). The three tester lines have different loci and alleles from each other, which are expected to mask different segregating loci in the doubled haploid MAGIC lines. Differences in environmental interactions can further cause different segregating loci in the doubled haploid lines and hybrids to have differential effects. The inconsistency of QTL is a major challenge for marker‐assisted breeding of quantitative traits in maize, especially in cumulative traits like GY.</p>
<p>The MAGIC approach to doubled haploid development allowed high resolution of QTL (smaller QTL intervals with fewer candidate genes) as predicted by theory and empirically demonstrated (Anderson et al., <a href="#tpg270078-bib-0056" class="usa-link" aria-describedby="tpg270078-bib-0056">2018</a>). The resulting improved resolution reduced the number of annotated genes in each QTL interval and the likely candidate genes that were investigated. The presence of candidate genes like E3 ubiquitin‐protein ligase RFWD2 and auxin‐responsive proteins in identified QTLs suggests complex regulatory mechanisms influencing yield stability. An auxin biosynthesis pathway was previously identified related to stability performance of plant height and flowering time in maize (Fu &amp; Wang, <a href="#tpg270078-bib-0014" class="usa-link" aria-describedby="tpg270078-bib-0014">2023</a>); similarly, we discovered a segregating auxin‐responsive candidate gene (<em>Zm00001eb319470</em>) corresponding to the slope of the GY in hybrid population with PHZ51 tester, grown in the greatest number of environments. Auxin‐responsive genes are related to plant growth and development and are responsive to environmental stimuli (Luo et al., <a href="#tpg270078-bib-0031" class="usa-link" aria-describedby="tpg270078-bib-0031">2018</a>) and were likely to have played a major role on plasticity in G2F maize hybrids used in this study. The <em>Zm00001eb319470</em> locus encodes the IAA33–Aux/IAA‐transcription factor 33, known to interact with auxin response factors (ARFs) and inhibit the transcription of genes activated by ARFs. This interaction is essential in the auxin signaling pathway (Luo et al., <a href="#tpg270078-bib-0031" class="usa-link" aria-describedby="tpg270078-bib-0031">2018</a>; Lv et al., <a href="#tpg270078-bib-0032" class="usa-link" aria-describedby="tpg270078-bib-0032">2020</a>), influencing numerous phenotypic traits such as the height from ground to top ear node in maize (Wallace et al., <a href="#tpg270078-bib-0045" class="usa-link" aria-describedby="tpg270078-bib-0045">2014</a>). Variations in <em>Zm00001eb319470</em>, as indicated by the environmental index (46–90 days of PTR across 20 environments in Figure <a href="#tpg270078-fig-0002" class="usa-link">2C</a>), might be associated with the genetic basis for taller plant height and increased yield observed in northern environments compared to southern ones. This hypothesis aligns with the idea that IAA33, encoded by <em>Zm00001eb319470</em>, could have enhanced activity in northern environments where longer days relative to GDD could influence its contribution to the stability of GY through plant growth and development processes.</p></section><section id="tpg270078-sec-0190"><h3 class="pmc_sec_title">4.3. Environmental index‐based genomic prediction for GY</h3>
<p>Genomic prediction and selection have been widely implemented for the purpose of selecting the best individuals for overall crop improvement and genetic gain (Crossa et al., <a href="#tpg270078-bib-0010" class="usa-link" aria-describedby="tpg270078-bib-0010">2014</a>; Hunt et al., <a href="#tpg270078-bib-0018" class="usa-link" aria-describedby="tpg270078-bib-0018">2018</a>). Inbred lines and their derived hybrids are easier to analyze for genomic prediction models (and GWAS) than hybrids due to simpler additive models for homozygosity, but inbred lines per se limits the scope of application, and the predictions would not be relevant for more genetically complex hybrids. In addition to additive variation, hybrids are prone to exhibit differential performance across environments resulting from dominance, variable epistatic interactions, and differential gene expression. However, some challenges associated with hybrid genomic prediction may be overcome with sufficient data that account for the phenotypic plasticity, such as multi‐environmental trials on the same set of hybrids. This study demonstrated the ability to predict GY of untested hybrids in diverse maize hybrid populations and environments using a combination of an environmental index and genomic prediction analysis, uniquely through the stability of GY determined by its slope and intercept (Figure <a href="#tpg270078-fig-0002" class="usa-link">2</a>). The genomic prediction models showed that GY predictions made with untested genotypes in tested environments had reduced prediction accuracy compared to the tested genotypes in untested environments. This could be because the same genotypes performed relatively consistently across all environments or because the hybrids shared similar genetic backgrounds since they were crossed with the same testers, reducing variability in G versus E. However, the proportion of G × E variance (∼12%) was comparable to the amount of genetic variance (∼20%) across this study (Figure <a href="#tpg270078-fig-0001" class="usa-link">1</a>). Considering the expected genetic variation between the testers in hybrid populations, it is expected that without sufficient performance data on the individual hybrids, the model would be less accurate for untested genotypes. Ideally, a high prediction accuracy value for the untested genotypes in untested environments scenario would provide knowledge for trait stabilities since they are complex and not easily detectable without extensive and costly field trials. This methodology, reflecting the complex interplay between genotype, phenotype, and environment, could allow improved prediction of complex traits with high phenotypic plasticity derived by a combination of environmental clues. Further physiological modeling of yield component traits could provide further gains (Cooper et al., <a href="#tpg270078-bib-0008" class="usa-link" aria-describedby="tpg270078-bib-0008">2016</a>).</p>
<p>The approach of using sliding window‐based environmental indices has been successfully integrated before in predicting flowering times in sorghum and rice (<em>Oryza sativa</em> L.) (Li et al., <a href="#tpg270078-bib-0024" class="usa-link" aria-describedby="tpg270078-bib-0024">2018</a>). The novel approach of using a PTR‐based environmental index for genomic prediction was promising based on previous studies in oat (<em>Avena sativa</em> L.) and wheat (<em>Triticum aestivum</em> L.), where PTR explained the highest amount of variation for mean yield across environments (Guo et al., <a href="#tpg270078-bib-0017" class="usa-link" aria-describedby="tpg270078-bib-0017">2020</a>). Here, PTR was used in maize GY for the first time to our knowledge. The incorporation of a PTR index differs from other genomic prediction studies commonly used to assess the performance of maize lines by also using a biologically relevant sliding window and an extensive set of environments (Choquette et al., <a href="#tpg270078-bib-0007" class="usa-link" aria-describedby="tpg270078-bib-0007">2023</a>; de los Campos et al., <a href="#tpg270078-bib-0011" class="usa-link" aria-describedby="tpg270078-bib-0011">2020</a>; Jarquín et al., <a href="#tpg270078-bib-0019" class="usa-link" aria-describedby="tpg270078-bib-0019">2017</a>). The use of environmental indices generally, and PTR specifically here, provides insights into how specific environmental parameters can be integrated into strategies to enhance yield predictability and stability by determining the timeframe at which an environmental parameter has the most impact on a trait across contrasting environments. As previously mentioned, variability in day length and temperature (GDD) are crucial environmental parameters throughout growth and maturity but are partly confounded; thus, a PTR index can capture both environmental components simultaneously without confounding to assess the performance of a genotype.</p></section><section id="tpg270078-sec-0200"><h3 class="pmc_sec_title">4.4. Use of these findings</h3>
<p>The incorporation of data derived from environmental indices alongside genomic data offers a useful development in methodologies in predictive plant breeding, considering the evolving challenges of climate change and its effects on crop productivity. Utilizing these predictive capabilities can aid researchers in better understanding the diverse trajectories of reaction norms of complex traits. Continued exploration promises to enhance these models further, potentially extending their relevance to a wider variety of crops and future environmental scenarios. Phenomic prediction models, which incorporate temporal phenomic markers of genotypes, may also provide insight on diverse environmental parameters’ effects on plasticity at different stages of a trait (Adak et al., <a href="#tpg270078-bib-0004" class="usa-link" aria-describedby="tpg270078-bib-0004">2023</a>). Rather than focusing on the relationship between temporal environmental parameters affecting a single timepoint related to the terminal stage, temporal phenomic data and crop growth modeling have the potential to capture trait values across different growth periods, which would highlight the stages of a trait that are more correlated with a specific environmental parameter. The combination of G × E data derived from this study along with data from different domains throughout the breeding process may be examined simultaneously using deep learning algorithms to improve existing predictive models that would otherwise present limitations for data‐driven breeding practices (Xu et al., <a href="#tpg270078-bib-0048" class="usa-link" aria-describedby="tpg270078-bib-0048">2022</a>). This could effectively contribute to advancements in crop genetics and breeding practices.</p></section></section><section id="tpg270078-sec-0210"><h2 class="pmc_sec_title">5. CONCLUSION</h2>
<p>Understanding the underlying role of G × E for complex traits is beneficial for quantifying the phenotypic variation seen within individuals across diverse environments using reaction norms. When combined with genotype data, this framework can uncover loci responsible for trait stabilities. This study sheds light on the varied genetic architecture of GY stability of three hybrid populations exposed to a varying range of PTR values in natural environments. The extent of environments used to derive correlations between PTR and mean GY has not been previously implemented for genomic prediction models in maize. The moderate‐to‐high prediction accuracies calculated for the tested hybrids in tested and untested environments reveal that future breeding efforts can be tailored to accommodate known hybrids based on the patterns of their phenotypic plasticity for GY. This can help future breeding efforts in designing field trials for lines that are either locally adapted to environments or for selecting lines that are stable across multiple environments based on performance predictions from environmental and genetic predictors.</p></section><section id="tpg270078-sec-0220"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Fatma Ozair</strong>: Conceptualization; data curation; formal analysis; investigation; visualization; writing—original draft; writing—review and editing. <strong>Alper Adak</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; project administration; supervision; visualization; writing—original draft; writing—review and editing. <strong>Seth C. Murray</strong>: Conceptualization; data curation; funding acquisition; investigation; methodology; project administration; resources; supervision; writing—original draft; writing—review and editing. <strong>Ryan T. Alpers</strong>: Data curation; project administration; resources. <strong>Alejandro C. Aviles</strong>: Data curation; project administration; resources. <strong>Dayane C. Lima</strong>: Data curation; project administration; resources. <strong>Jode Edwards</strong>: Funding acquisition; resources; validation; writing—review and editing. <strong>David Ertl</strong>: Funding acquisition; project administration; resources; writing—review and editing. <strong>Michael A. Gore</strong>: Data curation; investigation; project administration; resources; writing—review and editing. <strong>Candice N. Hirsch</strong>: Data curation; funding acquisition; investigation; project administration; resources. <strong>Joseph E. Knoll</strong>: Data curation; funding acquisition; project administration; resources. <strong>James C. Schnable</strong>: Data curation; investigation; project administration; resources. <strong>Maninder P. Singh</strong>: Data curation; investigation; project administration; resources; writing—review and editing. <strong>Erin E. Sparks</strong>: Project administration; resources; supervision. <strong>Addie Thompson</strong>: Data curation; funding acquisition; investigation; project administration; resources. <strong>Teclemariam Weldekidan</strong>: Data curation; investigation; project administration; resources. <strong>Wenwei Xu</strong>: Data curation; investigation; project administration; resources.</p></section><section id="tpg270078-sec-0240"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec24"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270078-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270078-supitem-0001"><div class="caption p"><p>Supplemental Table 1 (S1) contains the significant quantitative trait loci (QTLs) discovered using a LOD threshold of 5, along with candidate genes and their functions for grain yield for three different maize tester populations. Supplemental Table 2 (S2) contains the significant quantitative trait loci (QTLs) discovered using a LOD threshold of 5, along with candidate genes and their functions for grain yield slope and intercept for the same three populations as above.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12332327/bin/TPG2-18-e70078-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70078-s001.docx</a><sup> (210.1KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270078-supitem-0002"><div class="caption p"><p>Supplemental Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12332327/bin/TPG2-18-e70078-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70078-s002.pdf</a><sup> (3.1MB, pdf) </sup>
</div></div></section></section><section id="tpg270078-sec-0230" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This research was supported in part by the US Department of Agriculture, Agricultural Research Service (USDA ARS) Project No. 5030‐21000‐073‐000D; along with the Foundation for Food and Agriculture Research (FFAR) with the project entitled CERCA—On‐Farm Nitrogen Recycling No. 58‐8062‐4‐003. Mention of trade names or commercial products in this publication is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the US Department of Agriculture. USDA is an equal opportunity provider and employer. The advanced computing resources provided by Texas A&amp;M High Performance Research Computing (HPRC) were used to run portions of the R code.</p></section><section id="notes1"><p>


Ozair, F.
, 
Adak, A.
, 
Murray, S. C.
, 
Alpers, R. T.
, 
Aviles, A. C.
, 
Lima, D. C.
, 
Edwards, J.
, 
Ertl, D.
, 
Gore, M. A.
, 
Hirsch, C. N.
, 
Knoll, J. E.
, 
Schnable, J. C.
, 
Singh, M. P.
, 
Sparks, E. E.
, 
Thompson, A.
, 
Weldekidan, T.
, &amp; 
Xu, W.
 (2025). Phenotypic plasticity in maize grain yield: Genetic and environmental insights of response to environmental gradients. The Plant Genome, 18, e70078. 10.1002/tpg2.70078

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270078-note-0001"><p>Assigned to Associate Editor Diego Jarquin.</p></div></div></section></section><section id="tpg270078-sec-0260"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All R codes and input files for this research are stored in GitHub (<a href="https://github.com/fatmaoz25/ReactionNormGenomicPrediction" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/fatmaoz25/ReactionNormGenomicPrediction</a>) for readers to access.</p></section><section id="tpg270078-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270078-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270078-bib-0004">
<cite>

Adak, A.
, 
Kang, M.
, 
Anderson, S. L.
, 
Murray, S. C.
, 
Jarquin, D.
, 
Wong, R. K.
, &amp; 
Katzfuß, M.
 (2023). Phenomic data‐driven biological prediction of maize through field‐based high‐throughput phenotyping integration with genomic data. Journal of Experimental Botany, 74(17), 5307–5326. 10.1093/jxb/erad216

</cite> [<a href="https://doi.org/10.1093/jxb/erad216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37279568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Phenomic%20data%E2%80%90driven%20biological%20prediction%20of%20maize%20through%20field%E2%80%90based%20high%E2%80%90throughput%20phenotyping%20integration%20with%20genomic%20data&amp;author=A.%20Adak&amp;author=M.%20Kang&amp;author=S.%20L.%20Anderson&amp;author=S.%20C.%20Murray&amp;author=D.%20Jarquin&amp;volume=74&amp;issue=17&amp;publication_year=2023&amp;pages=5307-5326&amp;pmid=37279568&amp;doi=10.1093/jxb/erad216&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0056">
<cite>

Anderson, S. L.
, 
Mahan, A. L.
, 
Murray, S. C.
, &amp; 
Klein, P. E.
 (2018). Four parent maize (FPM) population: Effects of mating designs on linkage disequilibrium and mapping quantitative traits. The Plant Genome, 11(2), 170102. 10.3835/plantgenome2017.11.0102
</cite> [<a href="https://doi.org/10.3835/plantgenome2017.11.0102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30025026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Four%20parent%20maize%20(FPM)%20population:%20Effects%20of%20mating%20designs%20on%20linkage%20disequilibrium%20and%20mapping%20quantitative%20traits&amp;author=S.%20L.%20Anderson&amp;author=A.%20L.%20Mahan&amp;author=S.%20C.%20Murray&amp;author=P.%20E.%20Klein&amp;volume=11&amp;issue=2&amp;publication_year=2018&amp;pages=170102&amp;pmid=30025026&amp;doi=10.3835/plantgenome2017.11.0102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0005">
<cite>

Arteaga, M. C.
, 
Moreno‐Letelier, A.
, 
Mastretta‐Yanes, A.
, 
Vázquez‐Lobo, A.
, 
Breña‐Ochoa, A.
, 
Moreno‐Estrada, A.
, 
Eguiarte, L. E.
, &amp; 
Piñero, D.
 (2016). Genomic variation in recently collected maize landraces from Mexico. Genomics Data, 7, 38–45. 10.1016/j.gdata.2015.11.002

</cite> [<a href="https://doi.org/10.1016/j.gdata.2015.11.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4778591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26981357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics%20Data&amp;title=Genomic%20variation%20in%20recently%20collected%20maize%20landraces%20from%20Mexico&amp;author=M.%20C.%20Arteaga&amp;author=A.%20Moreno%E2%80%90Letelier&amp;author=A.%20Mastretta%E2%80%90Yanes&amp;author=A.%20V%C3%A1zquez%E2%80%90Lobo&amp;author=A.%20Bre%C3%B1a%E2%80%90Ochoa&amp;volume=7&amp;publication_year=2016&amp;pages=38-45&amp;pmid=26981357&amp;doi=10.1016/j.gdata.2015.11.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0002">
<cite>

Bates, D.
, 
Mächler, M.
, 
Bolker, B.
, &amp; 
Walker, S.
 (2015). Fitting linear mixed‐effects models using lme4. Journal of Statistical Software, 67(1), 1–48. 10.18637/jss.v067.i01
</cite> [<a href="https://doi.org/10.18637/jss.v067.i01" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=Fitting%20linear%20mixed%E2%80%90effects%20models%20using%20lme4&amp;author=D.%20Bates&amp;author=M.%20M%C3%A4chler&amp;author=B.%20Bolker&amp;author=S.%20Walker&amp;volume=67&amp;issue=1&amp;publication_year=2015&amp;pages=1-48&amp;doi=10.18637/jss.v067.i01&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0003">
<cite>

Bonhomme, R.
, 
Derieux, M.
, &amp; 
Edmeades, G. O.
 (1994). Flowering of diverse maize cultivars in relation to temperature and photoperiod in multilocation field trials. Crop Science, 34(1), 156–164. 10.2135/cropsci1994.0011183X003400010028x
</cite> [<a href="https://doi.org/10.2135/cropsci1994.0011183X003400010028x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Flowering%20of%20diverse%20maize%20cultivars%20in%20relation%20to%20temperature%20and%20photoperiod%20in%20multilocation%20field%20trials&amp;author=R.%20Bonhomme&amp;author=M.%20Derieux&amp;author=G.%20O.%20Edmeades&amp;volume=34&amp;issue=1&amp;publication_year=1994&amp;pages=156-164&amp;doi=10.2135/cropsci1994.0011183X003400010028x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0053">
<cite>

Bernardo, R.
 (2002). Breeding for quantitative traits in plants (Vol. 1). Stemma Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Breeding%20for%20quantitative%20traits%20in%20plants&amp;author=R.%20Bernardo&amp;publication_year=2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0006">
<cite>

Broman, K. W.
, 
Gatti, D. M.
, 
Simecek, P.
, 
Furlotte, N. A.
, 
Prins, P.
, 
Sen, Ś.
, 
Yandell, B. S.
, &amp; 
Churchill, G. A.
 (2019). R/qtl2: Software for mapping quantitative trait loci with high‐dimensional data and multiparent populations. Genetics, 211, 495–502. 10.1534/genetics.118.301595

</cite> [<a href="https://doi.org/10.1534/genetics.118.301595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6366910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30591514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=R/qtl2:%20Software%20for%20mapping%20quantitative%20trait%20loci%20with%20high%E2%80%90dimensional%20data%20and%20multiparent%20populations&amp;author=K.%20W.%20Broman&amp;author=D.%20M.%20Gatti&amp;author=P.%20Simecek&amp;author=N.%20A.%20Furlotte&amp;author=P.%20Prins&amp;volume=211&amp;publication_year=2019&amp;pages=495-502&amp;pmid=30591514&amp;doi=10.1534/genetics.118.301595&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0007">
<cite>

Choquette, N. E.
, 
Holland, J. B.
, 
Weldekidan, T.
, 
Drouault, J.
, 
Flint‐Garcia, S.
, 
Lauter, N.
, 
Murray, S. C.
, 
Xu, W.
, &amp; 
Wisser, R. J.
 (2023). Environment‐specific selection alters flowering‐time plasticity and results in pervasive pleiotropic responses in maize. New Phytologist, 238(2), 737–749. 10.1111/nph.18769

</cite> [<a href="https://doi.org/10.1111/nph.18769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36683443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Environment%E2%80%90specific%20selection%20alters%20flowering%E2%80%90time%20plasticity%20and%20results%20in%20pervasive%20pleiotropic%20responses%20in%20maize&amp;author=N.%20E.%20Choquette&amp;author=J.%20B.%20Holland&amp;author=T.%20Weldekidan&amp;author=J.%20Drouault&amp;author=S.%20Flint%E2%80%90Garcia&amp;volume=238&amp;issue=2&amp;publication_year=2023&amp;pages=737-749&amp;pmid=36683443&amp;doi=10.1111/nph.18769&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0008">
<cite>

Cooper, M.
, 
Technow, F.
, 
Messina, C.
, 
Gho, C.
, &amp; 
Totir, L. R.
 (2016). Use of crop growth models with whole‐genome prediction: Application to a maize multienvironment trial. Crop Science, 56(5), 2141–2156. 10.2135/cropsci2015.08.0512
</cite> [<a href="https://doi.org/10.2135/cropsci2015.08.0512" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Use%20of%20crop%20growth%20models%20with%20whole%E2%80%90genome%20prediction:%20Application%20to%20a%20maize%20multienvironment%20trial&amp;author=M.%20Cooper&amp;author=F.%20Technow&amp;author=C.%20Messina&amp;author=C.%20Gho&amp;author=L.%20R.%20Totir&amp;volume=56&amp;issue=5&amp;publication_year=2016&amp;pages=2141-2156&amp;doi=10.2135/cropsci2015.08.0512&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0009">
<cite>

Costa‐Neto, G.
, 
Galli, G.
, 
Carvalho, H. F.
, 
Crossa, J.
, &amp; 
Fritsche‐Neto, R.
 (2021). EnvRtype: A software to interplay enviromics and quantitative genomics in agriculture. G3 Genes|Genomes|Genetics, 11(4), Article jkab040. 10.1093/g3journal/jkab040

</cite> [<a href="https://doi.org/10.1093/g3journal/jkab040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8049414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33835165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=EnvRtype:%20A%20software%20to%20interplay%20enviromics%20and%20quantitative%20genomics%20in%20agriculture&amp;author=G.%20Costa%E2%80%90Neto&amp;author=G.%20Galli&amp;author=H.%20F.%20Carvalho&amp;author=J.%20Crossa&amp;author=R.%20Fritsche%E2%80%90Neto&amp;volume=11&amp;issue=4&amp;publication_year=2021&amp;pages=jkab040&amp;pmid=33835165&amp;doi=10.1093/g3journal/jkab040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0010">
<cite>

Crossa, J.
, 
Pérez, P.
, 
Hickey, J.
, 
Burgueño, J.
, 
Ornella, L.
, 
Zhang, X.
, 
Dreisigacker, S.
, 
Babu, R.
, 
Li, Y.
, 
Bonnett, D.
, &amp; 
Mathews, K.
 (2014). Genomic prediction in CIMMYT maize and wheat breeding programs. Heredity, 112(1), 48–60. 10.1038/hdy.2013.16

</cite> [<a href="https://doi.org/10.1038/hdy.2013.16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3860161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23572121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Genomic%20prediction%20in%20CIMMYT%20maize%20and%20wheat%20breeding%20programs&amp;author=J.%20Crossa&amp;author=P.%20P%C3%A9rez&amp;author=J.%20Hickey&amp;author=J.%20Burgue%C3%B1o&amp;author=L.%20Ornella&amp;volume=112&amp;issue=1&amp;publication_year=2014&amp;pages=48-60&amp;pmid=23572121&amp;doi=10.1038/hdy.2013.16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0011">
<cite>

de los Campos, G.
, 
Pérez‐Rodríguez, P.
, 
Bogard, M.
, 
Gouache, D.
, &amp; 
Crossa, J.
 (2020). A data‐driven simulation platform to predict cultivars’ performances under uncertain weather conditions. Nature Communications, 11, Article 4876. 10.1038/s41467-020-18480-y
</cite> [<a href="https://doi.org/10.1038/s41467-020-18480-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7519145/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32978378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=A%20data%E2%80%90driven%20simulation%20platform%20to%20predict%20cultivars%E2%80%99%20performances%20under%20uncertain%20weather%20conditions&amp;author=G.%20de%20los%20Campos&amp;author=P.%20P%C3%A9rez%E2%80%90Rodr%C3%ADguez&amp;author=M.%20Bogard&amp;author=D.%20Gouache&amp;author=J.%20Crossa&amp;volume=11&amp;publication_year=2020&amp;pages=4876&amp;pmid=32978378&amp;doi=10.1038/s41467-020-18480-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0012">
<cite>

Endelman, J. B.
 (2011). Ridge regression and other kernels for genomic selection with R package rrBLUP. The Plant Genome, 4, 250–255. 10.3835/plantgenome2011.08.0024
</cite> [<a href="https://doi.org/10.3835/plantgenome2011.08.0024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Ridge%20regression%20and%20other%20kernels%20for%20genomic%20selection%20with%20R%20package%20rrBLUP&amp;author=J.%20B.%20Endelman&amp;volume=4&amp;publication_year=2011&amp;pages=250-255&amp;doi=10.3835/plantgenome2011.08.0024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0013">
<cite>

Finlay, K.
, &amp; 
Wilkinson, G.
 (1963). The analysis of adaptation in a plant‐breeding programme. Crop &amp; Pasture Science, 14, 742–754. 10.1071/AR9630742
</cite> [<a href="https://doi.org/10.1071/AR9630742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20&amp;%20Pasture%20Science&amp;title=The%20analysis%20of%20adaptation%20in%20a%20plant%E2%80%90breeding%20programme&amp;author=K.%20Finlay&amp;author=G.%20Wilkinson&amp;volume=14&amp;publication_year=1963&amp;pages=742-754&amp;doi=10.1071/AR9630742&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0014">
<cite>

Fu, R.
, &amp; 
Wang, X.
 (2023). Modeling the influence of phenotypic plasticity on maize hybrid performance. Plant Communications, 4, Article 100548. 10.1016/j.xplc.2023.100548

</cite> [<a href="https://doi.org/10.1016/j.xplc.2023.100548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10203382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36635964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Communications&amp;title=Modeling%20the%20influence%20of%20phenotypic%20plasticity%20on%20maize%20hybrid%20performance&amp;author=R.%20Fu&amp;author=X.%20Wang&amp;volume=4&amp;publication_year=2023&amp;pages=100548&amp;pmid=36635964&amp;doi=10.1016/j.xplc.2023.100548&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0015">
<cite>

Gage, J. L.
, 
Jarquin, D.
, 
Romay, C.
, 
Lorenz, A.
, 
Buckler, E. S.
, 
Kaeppler, S.
, 
Alkhalifah, N.
, 
Bohn, M.
, 
Campbell, D. A.
, &amp; 
Edwards, J.
 (2017). The effect of artificial selection on phenotypic plasticity in maize. Nature Communications, 8, Article 1348. 10.1038/s41467-017-01450-2
</cite> [<a href="https://doi.org/10.1038/s41467-017-01450-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5677005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29116144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=The%20effect%20of%20artificial%20selection%20on%20phenotypic%20plasticity%20in%20maize&amp;author=J.%20L.%20Gage&amp;author=D.%20Jarquin&amp;author=C.%20Romay&amp;author=A.%20Lorenz&amp;author=E.%20S.%20Buckler&amp;volume=8&amp;publication_year=2017&amp;pages=1348&amp;pmid=29116144&amp;doi=10.1038/s41467-017-01450-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0016">
<cite>

Gerke, J. P.
, 
Edwards, J. W.
, 
Guill, K. E.
, 
Ross‐Ibarra, J.
, &amp; 
McMullen, M. D.
 (2015). The genomic impacts of drift and selection for hybrid performance in maize. Genetics, 201, 1201–1211. 10.1534/genetics.115.182410

</cite> [<a href="https://doi.org/10.1534/genetics.115.182410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4649645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26385980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The%20genomic%20impacts%20of%20drift%20and%20selection%20for%20hybrid%20performance%20in%20maize&amp;author=J.%20P.%20Gerke&amp;author=J.%20W.%20Edwards&amp;author=K.%20E.%20Guill&amp;author=J.%20Ross%E2%80%90Ibarra&amp;author=M.%20D.%20McMullen&amp;volume=201&amp;publication_year=2015&amp;pages=1201-1211&amp;pmid=26385980&amp;doi=10.1534/genetics.115.182410&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0017">
<cite>

Guo, T.
, 
Mu, Q.
, 
Wang, J.
, 
Vanous, A. E.
, 
Onogi, A.
, 
Iwata, H.
, 
Li, X.
, &amp; 
Yu, J.
 (2020). Dynamic effects of interacting genes underlying rice flowering‐time phenotypic plasticity and global adaptation. Genome Research, 30, 673–683. 10.1101/gr.255703.119

</cite> [<a href="https://doi.org/10.1101/gr.255703.119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7263186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32299830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Dynamic%20effects%20of%20interacting%20genes%20underlying%20rice%20flowering%E2%80%90time%20phenotypic%20plasticity%20and%20global%20adaptation&amp;author=T.%20Guo&amp;author=Q.%20Mu&amp;author=J.%20Wang&amp;author=A.%20E.%20Vanous&amp;author=A.%20Onogi&amp;volume=30&amp;publication_year=2020&amp;pages=673-683&amp;pmid=32299830&amp;doi=10.1101/gr.255703.119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0018">
<cite>

Hunt, C. H.
, 
Mace, E. S.
, 
Hayes, B. J.
, &amp; 
Jordan, D. R.
 (2018). Development of genomic prediction in sorghum. Crop Science, 58(2), 690–700. 10.2135/cropsci2017.08.0469
</cite> [<a href="https://doi.org/10.2135/cropsci2017.08.0469" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Development%20of%20genomic%20prediction%20in%20sorghum&amp;author=C.%20H.%20Hunt&amp;author=E.%20S.%20Mace&amp;author=B.%20J.%20Hayes&amp;author=D.%20R.%20Jordan&amp;volume=58&amp;issue=2&amp;publication_year=2018&amp;pages=690-700&amp;doi=10.2135/cropsci2017.08.0469&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0019">
<cite>

Jarquín, D.
, 
Lemes da Silva, C.
, 
Gaynor, R. C.
, 
Poland, J.
, 
Fritz, A.
, 
Howard, R.
, 
Battenfield, S.
, &amp; 
Crossa, J.
 (2017). Increasing genomic‐enabled prediction accuracy by modeling genotype × environment interactions in Kansas wheat. The Plant Genome, 10, Article plantgenome2016.12.0130. 10.3835/plantgenome2016.12.0130
</cite> [<a href="https://doi.org/10.3835/plantgenome2016.12.0130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28724062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Increasing%20genomic%E2%80%90enabled%20prediction%20accuracy%20by%20modeling%20genotype%20%C3%97%20environment%20interactions%20in%20Kansas%20wheat&amp;author=D.%20Jarqu%C3%ADn&amp;author=C.%20Lemes%20da%20Silva&amp;author=R.%20C.%20Gaynor&amp;author=J.%20Poland&amp;author=A.%20Fritz&amp;volume=10&amp;publication_year=2017&amp;pmid=28724062&amp;doi=10.3835/plantgenome2016.12.0130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0020">
<cite>

Jin, M.
, 
Liu, H.
, 
Liu, X.
, 
Guo, T.
, 
Guo, J.
, 
Yin, Y.
, 
Ji, Y.
, 
Li, Z.
, 
Zhang, J.
, 
Wang, X.
, 
Qiao, F.
, 
Xiao, Y.
, 
Zan, Y.
, &amp; 
Yan, J.
 (2023). Complex genetic architecture underlying the plasticity of maize agronomic traits. Plant Communications, 4, Article 100473. 10.1016/j.xplc.2022.100473

</cite> [<a href="https://doi.org/10.1016/j.xplc.2022.100473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10203269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36642074/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Communications&amp;title=Complex%20genetic%20architecture%20underlying%20the%20plasticity%20of%20maize%20agronomic%20traits&amp;author=M.%20Jin&amp;author=H.%20Liu&amp;author=X.%20Liu&amp;author=T.%20Guo&amp;author=J.%20Guo&amp;volume=4&amp;publication_year=2023&amp;pages=100473&amp;pmid=36642074&amp;doi=10.1016/j.xplc.2022.100473&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0021">
<cite>

Khoury, C. K.
, 
Brush, S.
, 
Costich, D. E.
, 
Curry, H. A.
, 
Engels, J. M. M.
, 
Guarino, L.
, 
Hoban, S.
, 
Mercer, K. L.
, 
Miller, A. J.
, 
Nabhan, G. P.
, 
Perales, H. R.
, 
Richards, C.
, 
Riggins, C.
, &amp; 
Thormann, I.
 (2022). Crop genetic erosion: Understanding and responding to loss of crop diversity. New Phytologist, 233(1), 84–118. 10.1111/nph.17733

</cite> [<a href="https://doi.org/10.1111/nph.17733" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34515358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Crop%20genetic%20erosion:%20Understanding%20and%20responding%20to%20loss%20of%20crop%20diversity&amp;author=C.%20K.%20Khoury&amp;author=S.%20Brush&amp;author=D.%20E.%20Costich&amp;author=H.%20A.%20Curry&amp;author=J.%20M.%20M.%20Engels&amp;volume=233&amp;issue=1&amp;publication_year=2022&amp;pages=84-118&amp;pmid=34515358&amp;doi=10.1111/nph.17733&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0022">
<cite>

Kusmec, A.
, 
de Leon, N.
, &amp; 
Schnable, P. S.
 (2018). Harnessing phenotypic plasticity to improve maize yields. Frontiers in Plant Science, 9, Article 410186. 10.3389/fpls.2018.01377
</cite> [<a href="https://doi.org/10.3389/fpls.2018.01377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6156439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30283485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Harnessing%20phenotypic%20plasticity%20to%20improve%20maize%20yields&amp;author=A.%20Kusmec&amp;author=N.%20de%20Leon&amp;author=P.%20S.%20Schnable&amp;volume=9&amp;publication_year=2018&amp;pages=410186&amp;pmid=30283485&amp;doi=10.3389/fpls.2018.01377&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0023">
<cite>

Kusmec, A.
, 
Srinivasan, S.
, 
Nettleton, D.
, &amp; 
Schnable, P. S.
 (2017). Distinct genetic architectures for phenotype means and plasticities in <em>Zea mays</em>
. Nature Plants, 3, 715–723. 10.1038/s41477-017-0007-7

</cite> [<a href="https://doi.org/10.1038/s41477-017-0007-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6209453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29150689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=Distinct%20genetic%20architectures%20for%20phenotype%20means%20and%20plasticities%20in%20Zea%20mays&amp;author=A.%20Kusmec&amp;author=S.%20Srinivasan&amp;author=D.%20Nettleton&amp;author=P.%20S.%20Schnable&amp;volume=3&amp;publication_year=2017&amp;pages=715-723&amp;pmid=29150689&amp;doi=10.1038/s41477-017-0007-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0024">
<cite>

Li, X.
, 
Guo, T.
, 
Mu, Q.
, 
Li, X.
, &amp; 
Yu, J.
 (2018). Genomic and environmental determinants and their interplay underlying phenotypic plasticity. Proceedings of the National Academy of Sciences of the USA, 115, 6679–6684. 10.1073/pnas.1718326115

</cite> [<a href="https://doi.org/10.1073/pnas.1718326115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6042117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29891664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20USA&amp;title=Genomic%20and%20environmental%20determinants%20and%20their%20interplay%20underlying%20phenotypic%20plasticity&amp;author=X.%20Li&amp;author=T.%20Guo&amp;author=Q.%20Mu&amp;author=X.%20Li&amp;author=J.%20Yu&amp;volume=115&amp;publication_year=2018&amp;pages=6679-6684&amp;pmid=29891664&amp;doi=10.1073/pnas.1718326115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0025">
<cite>

Li, X.
, 
Guo, T.
, 
Wang, J.
, 
Bekele, W. A.
, 
Sukumaran, S.
, 
Vanous, A.
, 
McNellie, J. P.
, 
Tibbs, L. E.
, 
Julia, M.
, 
Lamkey, K. R.
, 
Westgate, M. E.
, 
McKay, J. K.
, 
Archontoulis, S. V.
, 
Reynolds, M. R.
, 
Tinker, N. A.
, 
Schnable, P. S.
, &amp; 
Yu, J.
 (2021). An integrated framework reinstating the environmental dimension for GWAS and genomic selection in crops. Molecular Plant, 14(6), 874–887. 10.1016/j.molp.2021.03.010

</cite> [<a href="https://doi.org/10.1016/j.molp.2021.03.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33713844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=An%20integrated%20framework%20reinstating%20the%20environmental%20dimension%20for%20GWAS%20and%20genomic%20selection%20in%20crops&amp;author=X.%20Li&amp;author=T.%20Guo&amp;author=J.%20Wang&amp;author=W.%20A.%20Bekele&amp;author=S.%20Sukumaran&amp;volume=14&amp;issue=6&amp;publication_year=2021&amp;pages=874-887&amp;pmid=33713844&amp;doi=10.1016/j.molp.2021.03.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0026">
<cite>

Lima, D. C.
, 
Aviles, A. C.
, 
Alpers, R. T.
, 
Perkins, A.
, 
Schoemaker, D. L.
, 
Costa, M.
, 
Michel, K. J.
, 
Kaeppler, S.
, 
Ertl, D.
, 
Romay, M. C.
, 
Gage, J. L.
, 
Holland, J.
, 
Beissinger, T.
, 
Bohn, M.
, 
Buckler, E.
, 
Edwards, J.
, 
Flint‐Garcia, S.
, 
Gore, M. A.
, 
Hirsch, C. N.
, … 
de Leon, N.
 (2023). 2020–2021 field seasons of Maize G × E project within the Genomes to Fields Initiative. BMC Research Notes, 16, Article 219. 10.1186/s13104-023-06430-y

</cite> [<a href="https://doi.org/10.1186/s13104-023-06430-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10502993/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37710302/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Research%20Notes&amp;title=2020%E2%80%932021%20field%20seasons%20of%20Maize%20G%20%C3%97%20E%20project%20within%20the%20Genomes%20to%20Fields%20Initiative&amp;author=D.%20C.%20Lima&amp;author=A.%20C.%20Aviles&amp;author=R.%20T.%20Alpers&amp;author=A.%20Perkins&amp;author=D.%20L.%20Schoemaker&amp;volume=16&amp;publication_year=2023&amp;pages=219&amp;pmid=37710302&amp;doi=10.1186/s13104-023-06430-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0027">
<cite>

Lima, D. C.
, 
Washburn, J. D.
, 
Varela, J. I.
, 
Chen, Q.
, 
Gage, J. L.
, 
Romay, M. C.
, 
Holland, J.
, 
Ertl, D.
, 
Lopez‐Cruz, M.
, 
Aguate, F. M.
, 
de Los Campos, G.
, 
Kaeppler, S.
, 
Beissinger, T.
, 
Bohn, M.
, 
Buckler, E.
, 
Edwards, J.
, 
Flint‐Garcia, S.
, 
Gore, M. A.
, 
Hirsch, C. N.
, … 
de Leon, N.
 (2023). Genomes to fields 2022 maize genotype by environment prediction competition. BMC Research Notes, 16(1), Article 148. 10.1186/s13104-023-06421-z

</cite> [<a href="https://doi.org/10.1186/s13104-023-06421-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10353085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37461058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Research%20Notes&amp;title=Genomes%20to%20fields%202022%20maize%20genotype%20by%20environment%20prediction%20competition&amp;author=D.%20C.%20Lima&amp;author=J.%20D.%20Washburn&amp;author=J.%20I.%20Varela&amp;author=Q.%20Chen&amp;author=J.%20L.%20Gage&amp;volume=16&amp;issue=1&amp;publication_year=2023&amp;pages=148&amp;pmid=37461058&amp;doi=10.1186/s13104-023-06421-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0028">
<cite>

Liu, B.
, &amp; 
Heins, R. D.
 (2002). Photothermal ratio affects plant quality in ‘Freedom’ poinsettia. Journal of the American Society for Horticultural Science, 127(1), 20–26. 10.21273/JASHS.127.1.20
</cite> [<a href="https://doi.org/10.21273/JASHS.127.1.20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Horticultural%20Science&amp;title=Photothermal%20ratio%20affects%20plant%20quality%20in%20%E2%80%98Freedom%E2%80%99%20poinsettia&amp;author=B.%20Liu&amp;author=R.%20D.%20Heins&amp;volume=127&amp;issue=1&amp;publication_year=2002&amp;pages=20-26&amp;doi=10.21273/JASHS.127.1.20&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0029">
<cite>

Liu, N.
, 
Du, Y.
, 
Warburton, M. L.
, 
Xiao, Y.
, &amp; 
Yan, J.
 (2020). Phenotypic plasticity contributes to maize adaptation and heterosis. Molecular Biology and Evolution, 38, 1262–1275. 10.1093/molbev/msaa283
</cite> [<a href="https://doi.org/10.1093/molbev/msaa283" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8480182/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33212480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=Phenotypic%20plasticity%20contributes%20to%20maize%20adaptation%20and%20heterosis&amp;author=N.%20Liu&amp;author=Y.%20Du&amp;author=M.%20L.%20Warburton&amp;author=Y.%20Xiao&amp;author=J.%20Yan&amp;volume=38&amp;publication_year=2020&amp;pages=1262-1275&amp;pmid=33212480&amp;doi=10.1093/molbev/msaa283&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0030">
<cite>

Lobell, D. B.
, 
Hammer, G. L.
, 
McLean, G.
, 
Messina, C.
, 
Roberts, M. J.
, &amp; 
Schlenker, W.
 (2013). The critical role of extreme heat for maize production in the United States. Nature Climate Change, 3(5), 497–501. 10.1038/nclimate1832
</cite> [<a href="https://doi.org/10.1038/nclimate1832" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Climate%20Change&amp;title=The%20critical%20role%20of%20extreme%20heat%20for%20maize%20production%20in%20the%20United%20States&amp;author=D.%20B.%20Lobell&amp;author=G.%20L.%20Hammer&amp;author=G.%20McLean&amp;author=C.%20Messina&amp;author=M.%20J.%20Roberts&amp;volume=3&amp;issue=5&amp;publication_year=2013&amp;pages=497-501&amp;doi=10.1038/nclimate1832&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0031">
<cite>

Luo, J.
, 
Zhou, J.‐J.
, &amp; 
Zhang, J.‐Z.
 (2018). Aux/IAA gene family in plants: Molecular structure, regulation, and function. International Journal of Molecular Sciences, 19, Article 259. 10.3390/ijms19010259

</cite> [<a href="https://doi.org/10.3390/ijms19010259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5796205/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29337875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Aux/IAA%20gene%20family%20in%20plants:%20Molecular%20structure,%20regulation,%20and%20function&amp;author=J.%20Luo&amp;author=J.%E2%80%90J.%20Zhou&amp;author=J.%E2%80%90Z.%20Zhang&amp;volume=19&amp;publication_year=2018&amp;pages=259&amp;pmid=29337875&amp;doi=10.3390/ijms19010259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0032">
<cite>

Lv, B.
, 
Yu, Q.
, 
Liu, J.
, 
Wen, X.
, 
Yan, Z.
, 
Hu, K.
, 
Li, H.
, 
Kong, X.
, 
Li, C.
, 
Tian, H.
, 
De Smet, I.
, 
Zhang, X. S.
, &amp; 
Ding, Z.
 (2020). Non‐canonical AUX/IAA protein IAA33 competes with canonical AUX/IAA repressor IAA5 to negatively regulate auxin signaling. The EMBO Journal, 39, Article e101515. 10.15252/embj.2019101515

</cite> [<a href="https://doi.org/10.15252/embj.2019101515" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6939196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31617603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20EMBO%20Journal&amp;title=Non%E2%80%90canonical%20AUX/IAA%20protein%20IAA33%20competes%20with%20canonical%20AUX/IAA%20repressor%20IAA5%20to%20negatively%20regulate%20auxin%20signaling&amp;author=B.%20Lv&amp;author=Q.%20Yu&amp;author=J.%20Liu&amp;author=X.%20Wen&amp;author=Z.%20Yan&amp;volume=39&amp;publication_year=2020&amp;pages=e101515&amp;pmid=31617603&amp;doi=10.15252/embj.2019101515&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0054">
<cite>

Malosetti, M.
, 
Ribaut, J. M.
, &amp; 
van Eeuwijk, F. A.
 (2013). The statistical analysis of multi‐environment data: modeling genotype‐by‐environment interaction and its genetic basis. Frontiers in Physiology, 4, 44.
</cite> [<a href="https://doi.org/10.3389/fphys.2013.00044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3594989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23487515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Physiology&amp;title=The%20statistical%20analysis%20of%20multi%E2%80%90environment%20data:%20modeling%20genotype%E2%80%90by%E2%80%90environment%20interaction%20and%20its%20genetic%20basis&amp;author=M.%20Malosetti&amp;author=J.%20M.%20Ribaut&amp;author=F.%20A.%20van%20Eeuwijk&amp;volume=4&amp;publication_year=2013&amp;pages=44&amp;pmid=23487515&amp;doi=10.3389/fphys.2013.00044&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0055">
<cite>

Melchinger, A. E.
, &amp; 
Gumber, R. K.
 (1998). Overview of heterosis and heterotic groups in agronomic crops. In Larnkey K. R. &amp; Staub J. E. (Eds.), Concepts and breeding of heterosis in crop plants (Vol. 25, pp. 29–44). Wiley.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Concepts%20and%20breeding%20of%20heterosis%20in%20crop%20plants&amp;author=A.%20E.%20Melchinger&amp;author=R.%20K.%20Gumber&amp;publication_year=1998&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0033">
<cite>

Michel, K. J.
, 
Lima, D. C.
, 
Hundley, H.
, 
Singan, V.
, 
Yoshinaga, Y.
, 
Daum, C.
, 
Barry, K.
, 
Broman, K. W.
, 
Robin Buell, C.
, 
de Leon, N.
, &amp; 
Kaeppler, S. M.
 (2022). Genetic mapping and prediction of flowering time and plant height in a maize Stiff Stalk MAGIC population. Genetics, 221, Article iyac063. 10.1093/genetics/iyac063

</cite> [<a href="https://doi.org/10.1093/genetics/iyac063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9157087/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35441688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genetic%20mapping%20and%20prediction%20of%20flowering%20time%20and%20plant%20height%20in%20a%20maize%20Stiff%20Stalk%20MAGIC%20population&amp;author=K.%20J.%20Michel&amp;author=D.%20C.%20Lima&amp;author=H.%20Hundley&amp;author=V.%20Singan&amp;author=Y.%20Yoshinaga&amp;volume=221&amp;publication_year=2022&amp;pages=iyac063&amp;pmid=35441688&amp;doi=10.1093/genetics/iyac063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0034">
<cite>

Millet, E. J.
, 
Kruijer, W.
, 
Coupel‐Ledru, A.
, 
Alvarez Prado, S.
, 
Cabrera‐Bosquet, L.
, 
Lacube, S.
, 
Charcosset, A.
, 
Welcker, C.
, 
van Eeuwijk, F.
, &amp; 
Tardieu, F.
 (2019). Genomic prediction of maize yield across European environmental conditions. Nature Genetics, 51, 952–956. 10.1038/s41588-019-0414-y

</cite> [<a href="https://doi.org/10.1038/s41588-019-0414-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31110353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Genomic%20prediction%20of%20maize%20yield%20across%20European%20environmental%20conditions&amp;author=E.%20J.%20Millet&amp;author=W.%20Kruijer&amp;author=A.%20Coupel%E2%80%90Ledru&amp;author=S.%20Alvarez%20Prado&amp;author=L.%20Cabrera%E2%80%90Bosquet&amp;volume=51&amp;publication_year=2019&amp;pages=952-956&amp;pmid=31110353&amp;doi=10.1038/s41588-019-0414-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0035">
<cite>

Mu, Q.
, 
Guo, T.
, 
Li, X.
, &amp; 
Yu, J.
 (2022). Phenotypic plasticity in plant height shaped by interaction between genetic loci and diurnal temperature range. New Phytologist, 233, 1768–1779. 10.1111/nph.17904

</cite> [<a href="https://doi.org/10.1111/nph.17904" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34870847/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Phenotypic%20plasticity%20in%20plant%20height%20shaped%20by%20interaction%20between%20genetic%20loci%20and%20diurnal%20temperature%20range&amp;author=Q.%20Mu&amp;author=T.%20Guo&amp;author=X.%20Li&amp;author=J.%20Yu&amp;volume=233&amp;publication_year=2022&amp;pages=1768-1779&amp;pmid=34870847&amp;doi=10.1111/nph.17904&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0036">
<cite>

Mural, R. V.
, 
Sun, G.
, 
Grzybowski, M.
, 
Tross, M. C.
, 
Jin, H.
, 
Smith, C.
, 
Newton, L.
, 
Andorf, C. M.
, 
Woodhouse, M. R.
, 
Thompson, A. M.
, 
Sigmon, B.
, &amp; 
Schnable, J. C.
 (2022). Association mapping across a multitude of traits collected in diverse environments in maize. GigaScience, 11, Article giac080. 10.1093/gigascience/giac080

</cite> [<a href="https://doi.org/10.1093/gigascience/giac080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9396454/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35997208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=Association%20mapping%20across%20a%20multitude%20of%20traits%20collected%20in%20diverse%20environments%20in%20maize&amp;author=R.%20V.%20Mural&amp;author=G.%20Sun&amp;author=M.%20Grzybowski&amp;author=M.%20C.%20Tross&amp;author=H.%20Jin&amp;volume=11&amp;publication_year=2022&amp;pages=giac080&amp;pmid=35997208&amp;doi=10.1093/gigascience/giac080&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0037">
<cite>

Oostra, V.
, 
Saastamoinen, M.
, 
Zwaan, B. J.
, &amp; 
Wheat, C. W.
 (2018). Strong phenotypic plasticity limits potential for evolutionary responses to climate change. Nature Communications, 9, Article 1005. 10.1038/s41467-018-03384-9
</cite> [<a href="https://doi.org/10.1038/s41467-018-03384-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5843647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29520061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Strong%20phenotypic%20plasticity%20limits%20potential%20for%20evolutionary%20responses%20to%20climate%20change&amp;author=V.%20Oostra&amp;author=M.%20Saastamoinen&amp;author=B.%20J.%20Zwaan&amp;author=C.%20W.%20Wheat&amp;volume=9&amp;publication_year=2018&amp;pages=1005&amp;pmid=29520061&amp;doi=10.1038/s41467-018-03384-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0038">
<cite>

Perez, P.
, &amp; 
de los Campos, G.
 (2014). Genome‐wide regression and prediction with the BGLR statistical package. Genetics, 198(2), 483–495. 10.1534/genetics.114.164442

</cite> [<a href="https://doi.org/10.1534/genetics.114.164442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4196607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25009151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genome%E2%80%90wide%20regression%20and%20prediction%20with%20the%20BGLR%20statistical%20package&amp;author=P.%20Perez&amp;author=G.%20de%20los%20Campos&amp;volume=198&amp;issue=2&amp;publication_year=2014&amp;pages=483-495&amp;pmid=25009151&amp;doi=10.1534/genetics.114.164442&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0039">
<cite>

Piepho, P.
, &amp; 
Blancon, J.
 (2023). Extending Finlay–Wilkinson regression with environmental covariates. Plant Breeding, 142(5), 621–631. 10.1111/pbr.13130
</cite> [<a href="https://doi.org/10.1111/pbr.13130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Extending%20Finlay%E2%80%93Wilkinson%20regression%20with%20environmental%20covariates&amp;author=P.%20Piepho&amp;author=J.%20Blancon&amp;volume=142&amp;issue=5&amp;publication_year=2023&amp;pages=621-631&amp;doi=10.1111/pbr.13130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0040">
<cite>

Pierik, R.
, &amp; 
Testerink, C.
 (2014). The art of being flexible: How to escape from shade, salt, and drought. Plant Physiology, 166, 5–22. 10.1104/pp.114.239160

</cite> [<a href="https://doi.org/10.1104/pp.114.239160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4149730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24972713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=The%20art%20of%20being%20flexible:%20How%20to%20escape%20from%20shade,%20salt,%20and%20drought&amp;author=R.%20Pierik&amp;author=C.%20Testerink&amp;volume=166&amp;publication_year=2014&amp;pages=5-22&amp;pmid=24972713&amp;doi=10.1104/pp.114.239160&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0050">
<cite>
Pioneer Hi‐Bred International
. (1987). 
<em>Corn ‘PHZ51’</em> (U.S. Plant Patent No. 8600132). U.S. Patent and Trademark Office.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pioneer%20Hi%E2%80%90Bred%20International%20.%20(1987).%20Corn%20%E2%80%98PHZ51%E2%80%99%20(U.S.%20Plant%20Patent%20No.%208600132).%20U.S.%20Patent%20and%20Trademark%20Office." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0049">
<cite>
Pioneer Hi‐Bred International
. (1988). 
<em>Corn ‘PHK76’</em> (U.S. Plant Patent No. 8800036). U.S. Patent and Trademark Office.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pioneer%20Hi%E2%80%90Bred%20International%20.%20(1988).%20Corn%20%E2%80%98PHK76%E2%80%99%20(U.S.%20Plant%20Patent%20No.%208800036).%20U.S.%20Patent%20and%20Trademark%20Office." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0051">
<cite>
Pioneer Hi‐Bred International
. (1989). 
<em>Corn ‘PHP02’</em> (U.S. Plant Patent No. 8800212). U.S. Patent and Trademark Office.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pioneer%20Hi%E2%80%90Bred%20International%20.%20(1989).%20Corn%20%E2%80%98PHP02%E2%80%99%20(U.S.%20Plant%20Patent%20No.%208800212).%20U.S.%20Patent%20and%20Trademark%20Office." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0041">
<cite>

Prasanna, B. M.
 (2012). Diversity in global maize germplasm: Characterization and utilization. Journal of Biosciences, 37, 843–855. 10.1007/s12038-012-9227-1

</cite> [<a href="https://doi.org/10.1007/s12038-012-9227-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23107920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biosciences&amp;title=Diversity%20in%20global%20maize%20germplasm:%20Characterization%20and%20utilization&amp;author=B.%20M.%20Prasanna&amp;volume=37&amp;publication_year=2012&amp;pages=843-855&amp;pmid=23107920&amp;doi=10.1007/s12038-012-9227-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0042">
<cite>

Rivas, J. G.
, 
Gutierrez, A. V.
, 
Defacio, R. A.
, 
Schimpf, J.
, 
Vicario, A. L.
, 
Hopp, H. E.
, 
Paniego, N. B.
, &amp; 
Lia, V. V.
 (2022). Morphological and genetic diversity of maize landraces along an altitudinal gradient in the Southern Andes. PLoS ONE, 17(12), Article e0271424. 10.1371/journal.pone.0271424

</cite> [<a href="https://doi.org/10.1371/journal.pone.0271424" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9770441/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36542628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Morphological%20and%20genetic%20diversity%20of%20maize%20landraces%20along%20an%20altitudinal%20gradient%20in%20the%20Southern%20Andes&amp;author=J.%20G.%20Rivas&amp;author=A.%20V.%20Gutierrez&amp;author=R.%20A.%20Defacio&amp;author=J.%20Schimpf&amp;author=A.%20L.%20Vicario&amp;volume=17&amp;issue=12&amp;publication_year=2022&amp;pages=e0271424&amp;pmid=36542628&amp;doi=10.1371/journal.pone.0271424&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0052">
<cite>

Sjoberg, S. M.
, 
Carter, A. H.
, 
Steber, C. M.
, &amp; 
Campbell, K. A. G.
 (2020). Unraveling complex traits in wheat: Approaches for analyzing genotype × environment interactions in a multienvironment study of falling numbers. Crop Science, 60(6), 3013–3026. 10.1002/csc2.20133
</cite> [<a href="https://doi.org/10.1002/csc2.20133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Unraveling%20complex%20traits%20in%20wheat:%20Approaches%20for%20analyzing%20genotype%20%C3%97%20environment%20interactions%20in%20a%20multienvironment%20study%20of%20falling%20numbers&amp;author=S.%20M.%20Sjoberg&amp;author=A.%20H.%20Carter&amp;author=C.%20M.%20Steber&amp;author=K.%20A.%20G.%20Campbell&amp;volume=60&amp;issue=6&amp;publication_year=2020&amp;pages=3013-3026&amp;doi=10.1002/csc2.20133&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0043">
<cite>

Soltani, A.
, &amp; 
Sinclair, T. R.
 (2012). Modeling physiology of crop development, growth and yield. CAB International. 10.1079/9781845939700.0000
</cite> [<a href="https://doi.org/10.1079/9781845939700.0000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Modeling%20physiology%20of%20crop%20development,%20growth%20and%20yield&amp;author=A.%20Soltani&amp;author=T.%20R.%20Sinclair&amp;publication_year=2012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0044">
<cite>

Tollenaar, M.
, &amp; 
Lee, E.
 (2002). Yield potential, yield stability and stress tolerance in maize. Field Crops Research, 75(2–3), 161–169. 10.1016/S0378-4290(02)00024-2
</cite> [<a href="https://doi.org/10.1016/S0378-4290(02)00024-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Field%20Crops%20Research&amp;title=Yield%20potential,%20yield%20stability%20and%20stress%20tolerance%20in%20maize&amp;author=M.%20Tollenaar&amp;author=E.%20Lee&amp;volume=75&amp;issue=2%E2%80%933&amp;publication_year=2002&amp;pages=161-169&amp;doi=10.1016/S0378-4290(02)00024-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0045">
<cite>

Wallace, J. G.
, 
Bradbury, P. J.
, 
Zhang, N.
, 
Gibon, Y.
, 
Stitt, M.
, &amp; 
Buckler, E. S.
 (2014). Association mapping across numerous traits reveals patterns of functional variation in maize. PLOS Genetics, 10, Article e1004845. 10.1371/journal.pgen.1004845

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004845" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4256217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25474422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=Association%20mapping%20across%20numerous%20traits%20reveals%20patterns%20of%20functional%20variation%20in%20maize&amp;author=J.%20G.%20Wallace&amp;author=P.%20J.%20Bradbury&amp;author=N.%20Zhang&amp;author=Y.%20Gibon&amp;author=M.%20Stitt&amp;volume=10&amp;publication_year=2014&amp;pages=e1004845&amp;pmid=25474422&amp;doi=10.1371/journal.pgen.1004845&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0046">
<cite>

Wang, C.
, 
Chen, Y.
, 
Ku, L.
, 
Wang, T.
, 
Sun, Z.
, 
Cheng, F.
, &amp; 
Wu, L.
 (2010). Mapping QTL associated with photoperiod sensitivity and assessing the importance of QTL×environment interaction for flowering time in maize. PLoS ONE, 5(11), Article e14068. 10.1371/journal.pone.0014068

</cite> [<a href="https://doi.org/10.1371/journal.pone.0014068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2988818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21124912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Mapping%20QTL%20associated%20with%20photoperiod%20sensitivity%20and%20assessing%20the%20importance%20of%20QTL%C3%97environment%20interaction%20for%20flowering%20time%20in%20maize&amp;author=C.%20Wang&amp;author=Y.%20Chen&amp;author=L.%20Ku&amp;author=T.%20Wang&amp;author=Z.%20Sun&amp;volume=5&amp;issue=11&amp;publication_year=2010&amp;pages=e14068&amp;pmid=21124912&amp;doi=10.1371/journal.pone.0014068&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0047">
<cite>

Wang, C.
, 
Hu, S.
, 
Gardner, C.
, &amp; 
Lübberstedt, T.
 (2017). Emerging avenues for utilization of exotic germplasm. Trends in Plant Science, 22, 624–637. 10.1016/j.tplants.2017.04.002

</cite> [<a href="https://doi.org/10.1016/j.tplants.2017.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28476651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=Emerging%20avenues%20for%20utilization%20of%20exotic%20germplasm&amp;author=C.%20Wang&amp;author=S.%20Hu&amp;author=C.%20Gardner&amp;author=T.%20L%C3%BCbberstedt&amp;volume=22&amp;publication_year=2017&amp;pages=624-637&amp;pmid=28476651&amp;doi=10.1016/j.tplants.2017.04.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270078-bib-0048">
<cite>

Xu, Y.
, 
Zhang, X.
, 
Li, H.
, 
Zheng, H.
, 
Zhang, J.
, 
Olsen, M. S.
, &amp; 
Qian, Q.
 (2022). Smart breeding driven by big data, artificial intelligence, and integrated genomic‐enviromic prediction. Molecular Plant, 15(11), 1664–1695. 10.1016/j.molp.2022.09.001

</cite> [<a href="https://doi.org/10.1016/j.molp.2022.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36081348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Smart%20breeding%20driven%20by%20big%20data,%20artificial%20intelligence,%20and%20integrated%20genomic%E2%80%90enviromic%20prediction&amp;author=Y.%20Xu&amp;author=X.%20Zhang&amp;author=H.%20Li&amp;author=H.%20Zheng&amp;author=J.%20Zhang&amp;volume=15&amp;issue=11&amp;publication_year=2022&amp;pages=1664-1695&amp;pmid=36081348&amp;doi=10.1016/j.molp.2022.09.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Supplemental Table 1 (S1) contains the significant quantitative trait loci (QTLs) discovered using a LOD threshold of 5, along with candidate genes and their functions for grain yield for three different maize tester populations. Supplemental Table 2 (S2) contains the significant quantitative trait loci (QTLs) discovered using a LOD threshold of 5, along with candidate genes and their functions for grain yield slope and intercept for the same three populations as above.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12332327/bin/TPG2-18-e70078-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70078-s001.docx</a><sup> (210.1KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>Supplemental Material</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12332327/bin/TPG2-18-e70078-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70078-s002.pdf</a><sup> (3.1MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All R codes and input files for this research are stored in GitHub (<a href="https://github.com/fatmaoz25/ReactionNormGenomicPrediction" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/fatmaoz25/ReactionNormGenomicPrediction</a>) for readers to access.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70078"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70078.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332327/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332327/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332327%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332327/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332327/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332327/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40776458/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332327/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40776458/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332327/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332327/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="XiQcJRwlfrNMbQO6nth4WCTe27RSpi6SbW7Bgym3elzBm7NMgbreSXO3WFEvm6mr">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
