
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            S2ALM: Sequence-Structure Pre-trained Large Language Model for Comprehensive Antibody Representation Learning - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E483258AEEE7230583250068A26705.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="research">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364524/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Research">
<meta name="citation_title" content="S2ALM: Sequence-Structure Pre-trained Large Language Model for Comprehensive Antibody Representation Learning">
<meta name="citation_author" content="Mingze Yin">
<meta name="citation_author_institution" content="College of Computer Science and Technology, Zhejiang University, Hangzhou, China.">
<meta name="citation_author_institution" content="Institute of Wenzhou, Zhejiang University, Wenzhou, China.">
<meta name="citation_author" content="Hanjing Zhou">
<meta name="citation_author_institution" content="Zhejiang Cuisine Bird Supply Chain Management Co., Ltd., Hangzhou, China.">
<meta name="citation_author" content="Jialu Wu">
<meta name="citation_author_institution" content="College of Pharmaceutical Sciences, Zhejiang University, Hangzhou, China.">
<meta name="citation_author" content="Yiheng Zhu">
<meta name="citation_author_institution" content="College of Computer Science and Technology, Zhejiang University, Hangzhou, China.">
<meta name="citation_author" content="Yuxuan Zhan">
<meta name="citation_author_institution" content="School of Public Health, Zhejiang University, Hangzhou, China.">
<meta name="citation_author" content="Zitai Kong">
<meta name="citation_author_institution" content="College of Computer Science and Technology, Zhejiang University, Hangzhou, China.">
<meta name="citation_author" content="Hongxia Xu">
<meta name="citation_author_institution" content="College of Pharmaceutical Sciences, Zhejiang University, Hangzhou, China.">
<meta name="citation_author" content="Chang-Yu Hsieh">
<meta name="citation_author_institution" content="College of Pharmaceutical Sciences, Zhejiang University, Hangzhou, China.">
<meta name="citation_author" content="Jintai Chen">
<meta name="citation_author_institution" content="AI Thrust, Information Hub, HKUST(GZ), Guangzhou, China.">
<meta name="citation_author" content="Tingjun Hou">
<meta name="citation_author_institution" content="College of Pharmaceutical Sciences, Zhejiang University, Hangzhou, China.">
<meta name="citation_author" content="Jian Wu">
<meta name="citation_author_institution" content="Institute of Wenzhou, Zhejiang University, Wenzhou, China.">
<meta name="citation_author_institution" content="School of Public Health, Zhejiang University, Hangzhou, China.">
<meta name="citation_author_institution" content="Zhejiang Key Laboratory of Medical Imaging Artificial Intelligence, Hangzhou, China.">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="8">
<meta name="citation_firstpage" content="0721">
<meta name="citation_doi" content="10.34133/research.0721">
<meta name="citation_pmid" content="40837877">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364524/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364524/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364524/pdf/research.0721.pdf">
<meta name="description" content="Antibodies safeguard our health through their precise and potent binding to specific antigens, demonstrating promising therapeutic efficacy in the treatment of numerous diseases, including COVID-19. Recent advancements in biomedical language models ...">
<meta name="og:title" content="S2ALM: Sequence-Structure Pre-trained Large Language Model for Comprehensive Antibody Representation Learning">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Antibodies safeguard our health through their precise and potent binding to specific antigens, demonstrating promising therapeutic efficacy in the treatment of numerous diseases, including COVID-19. Recent advancements in biomedical language models ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364524/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12364524">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.34133/research.0721"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/research.0721.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12364524%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12364524/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12364524/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364524/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-research.png" alt="Research logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Research" title="Link to Research" shape="default" href="https://spj.sciencemag.org/research/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Research (Wash D C)</button></div>. 2025 Aug 19;8:0721. doi: <a href="https://doi.org/10.34133/research.0721" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.34133/research.0721</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Research%20(Wash%20D%20C)%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Research%20(Wash%20D%20C)%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Research%20(Wash%20D%20C)%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Research%20(Wash%20D%20C)%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>S<sup>2</sup>ALM: Sequence-Structure Pre-trained Large Language Model for Comprehensive Antibody Representation Learning</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mingze Yin</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mingze Yin</span></h3>
<div class="p">
<sup><sup>1</sup></sup>College of Computer Science and Technology, 
Zhejiang University, Hangzhou, China.</div>
<div class="p">
<sup><sup>2</sup></sup>Institute of Wenzhou, 
Zhejiang University, Wenzhou, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingze Yin</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Hanjing Zhou</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Hanjing Zhou</span></h3>
<div class="p">
<sup><sup>3</sup></sup>
Zhejiang Cuisine Bird Supply Chain Management Co., Ltd., Hangzhou, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hanjing Zhou</span></a>
</div>
</div>
<sup>3,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jialu Wu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jialu Wu</span></h3>
<div class="p">
<sup><sup>4</sup></sup>College of Pharmaceutical Sciences, 
Zhejiang University, Hangzhou, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jialu Wu</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yiheng Zhu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yiheng Zhu</span></h3>
<div class="p">
<sup><sup>1</sup></sup>College of Computer Science and Technology, 
Zhejiang University, Hangzhou, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yiheng Zhu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhan%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yuxuan Zhan</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yuxuan Zhan</span></h3>
<div class="p">
<sup><sup>5</sup></sup>School of Public Health, 
Zhejiang University, Hangzhou, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhan%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuxuan Zhan</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kong%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Zitai Kong</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Zitai Kong</span></h3>
<div class="p">
<sup><sup>1</sup></sup>College of Computer Science and Technology, 
Zhejiang University, Hangzhou, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kong%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zitai Kong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Hongxia Xu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Hongxia Xu</span></h3>
<div class="p">
<sup><sup>4</sup></sup>College of Pharmaceutical Sciences, 
Zhejiang University, Hangzhou, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hongxia Xu</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hsieh%20CY%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Chang-Yu Hsieh</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Chang-Yu Hsieh</span></h3>
<div class="p">
<sup><sup>4</sup></sup>College of Pharmaceutical Sciences, 
Zhejiang University, Hangzhou, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hsieh%20CY%22%5BAuthor%5D" class="usa-link"><span class="name western">Chang-Yu Hsieh</span></a>
</div>
</div>
<sup>4,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Jintai Chen</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Jintai Chen</span></h3>
<div class="p">
<sup><sup>6</sup></sup>
AI Thrust, Information Hub, HKUST(GZ), Guangzhou, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jintai Chen</span></a>
</div>
</div>
<sup>6,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hou%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Tingjun Hou</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Tingjun Hou</span></h3>
<div class="p">
<sup><sup>4</sup></sup>College of Pharmaceutical Sciences, 
Zhejiang University, Hangzhou, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hou%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tingjun Hou</span></a>
</div>
</div>
<sup>4,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Jian Wu</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Jian Wu</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Institute of Wenzhou, 
Zhejiang University, Wenzhou, China.</div>
<div class="p">
<sup><sup>5</sup></sup>School of Public Health, 
Zhejiang University, Hangzhou, China.</div>
<div class="p">
<sup><sup>7</sup></sup>
Zhejiang Key Laboratory of Medical Imaging Artificial Intelligence, Hangzhou, China.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jian Wu</span></a>
</div>
</div>
<sup>2,</sup><sup>5,</sup><sup>7,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>College of Computer Science and Technology, 
Zhejiang University, Hangzhou, China.</div>
<div id="aff2">
<sup><sup>2</sup></sup>Institute of Wenzhou, 
Zhejiang University, Wenzhou, China.</div>
<div id="aff3">
<sup><sup>3</sup></sup>
Zhejiang Cuisine Bird Supply Chain Management Co., Ltd., Hangzhou, China.</div>
<div id="aff4">
<sup><sup>4</sup></sup>College of Pharmaceutical Sciences, 
Zhejiang University, Hangzhou, China.</div>
<div id="aff5">
<sup><sup>5</sup></sup>School of Public Health, 
Zhejiang University, Hangzhou, China.</div>
<div id="aff6">
<sup><sup>6</sup></sup>
AI Thrust, Information Hub, HKUST(GZ), Guangzhou, China.</div>
<div id="aff7">
<sup><sup>7</sup></sup>
Zhejiang Key Laboratory of Medical Imaging Artificial Intelligence, Hangzhou, China.</div>
<div class="author-notes p">
<div class="fn" id="corr1">
<sup>*</sup><p class="display-inline">Address correspondence to: <span>jtchen721@gmail.com</span> (J.C.); <span>kimhsieh@zju.edu.cn</span> (C.-Y.H.); <span>tingjunhou@zju.edu.cn</span> (T.H.); <span>wujian2000@zju.edu.cn</span> (J.W.)</p>
</div>
<div class="fn" id="afn1">
<sup>†</sup><p class="display-inline">These authors contributed equally to this work.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 15; Revised 2025 Apr 9; Accepted 2025 May 7; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 Mingze Yin et al.</div>
<p>Exclusive licensee Science and Technology Review Publishing House. No claim to original U.S. Government Works. Distributed under a <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License (CC BY 4.0)</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12364524  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40837877/" class="usa-link">40837877</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Antibodies safeguard our health through their precise and potent binding to specific antigens, demonstrating promising therapeutic efficacy in the treatment of numerous diseases, including COVID-19. Recent advancements in biomedical language models have shown the great potential to interpret complex biological structures and functions. However, existing antibody-specific models have a notable limitation that they lack explicit consideration for antibody structural information, despite the fact that both 1-dimensional sequence and 3-dimensional structure carry unique and complementary insights into antibody behavior and functionality. This paper proposes the <strong>S</strong>equence-<strong>S</strong>tructure multi-level pre-trained <strong>A</strong>ntibody <strong>L</strong>anguage <strong>M</strong>odel (S<sup>2</sup>ALM), combining holistic sequential and structural information in one unified, generic antibody foundation model. We construct a hierarchical pre-training paradigm incorporated with 2 customized multi-level training objectives to facilitate the modeling of comprehensive antibody representations. S<sup>2</sup>ALM’s representation space uncovers inherent functional binding mechanisms, biological evolution properties, and structural interaction patterns. Pre-trained over 75 million sequences and 11.7 million structures, S<sup>2</sup>ALM can be adopted for diverse downstream tasks: accurately predicting antigen–antibody binding affinities, precisely distinguishing B cell maturation stages, identifying antibody crucial binding positions, and specifically designing novel coronavirus-binding antibodies. Remarkably, S<sup>2</sup>ALM outperforms well-established and renowned baselines and sets new state-of-the-art performance across extensive antibody-specific understanding and generation tasks. S<sup>2</sup>ALM’s ability to model comprehensive and generalized representations further positions its potential to advance real-world therapeutic antibody development, potentially addressing unmet academic, industrial, and clinical needs.</p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Antibodies, also known as immunoglobulins, are specialized proteins produced by the immune system to protect against a range of diseases (e.g., severe acute respiratory syndrome coronavirus 2 [SARS-CoV-2]), which fulfill the responsibility as guardians in the human body via evolving to complement their structures with the corresponding antigen structures [<a href="#B1" class="usa-link" aria-describedby="B1">1</a>–<a href="#B4" class="usa-link" aria-describedby="B4">4</a>]. Due to the high specificity and low adverse effect of antibodies, antibody drugs have important clinical significance and account for nearly one-fifth of new drug approvals by the Food and Drug Administration each year [<a href="#B5" class="usa-link" aria-describedby="B5">5</a>]. By mimicking the actions of the immune system, these drugs specifically target harmful agents like viruses and cancer cells, to detect viral infections or stimulate T cell immunity in cancer treatment [<a href="#B6" class="usa-link" aria-describedby="B6">6</a>–<a href="#B9" class="usa-link" aria-describedby="B9">9</a>]. Therefore, deciphering the information stored in antibody sequences and structures is crucial for understanding immune responses and disease development, and may accelerate the development of therapeutic antibodies for broad-spectrum disease treatment.</p>
<p>To alleviate the burden of time-consuming wet-lab experiments, in recent years, the computer-aided antibody engineering has emerged to improve the efficiency of antibody evolution and identify promising therapeutic antibodies with desirable developability profiles. While interpretable, traditional representation learning approaches depend on inefficient hand-crafted features that may miss hidden or latent patterns in diverse data. Drawing inspiration from natural language processing with the introduction of pre-trained foundational models, similar artificial intelligence technologies have emerged as the pivotal technology to interpret the language of biology [<a href="#B10" class="usa-link" aria-describedby="B10">10</a>–<a href="#B14" class="usa-link" aria-describedby="B14">14</a>]. Pre-trained on large-scale antibody corpora, antibody-specific large language models (ALMs) [<a href="#B6" class="usa-link" aria-describedby="B6">6</a>,<a href="#B7" class="usa-link" aria-describedby="B7">7</a>,<a href="#B15" class="usa-link" aria-describedby="B15">15</a>–<a href="#B18" class="usa-link" aria-describedby="B18">18</a>] have exhibited powerful capabilities in advancing the understanding of antibody structures and functions. Additionally, ALMs consistently outperform general-purpose protein language models (PLMs) because the mechanism of antibody evolution is strongly biased to the target antigen. This finding inspires us to dive deep into the development of comprehensive antibody-specific pre-training.</p>
<p>The central tenet of molecular biology is that an antibody’s amino acid sequence determines its 3-dimensional (3D) structure, and the 3D structure determines its biological function. The spatial structure plays an integral role for antibody functional characterization, as the principle of antigen–antibody structural binding indicates that antibody structures directly determine biological functions. Consequently, injecting structural information into antibody pre-training emerges a highly compelling endeavor. In this paper, we propose the <strong>S</strong>equence-<strong>S</strong>tructure multi-level pre-trained <strong>A</strong>ntibody <strong>L</strong>anguage <strong>M</strong>odel (S<sup>2</sup>ALM), incorporating antibody sequential and structural information to construct a comprehensive antibody foundation model. To accomplish this, we collect a holistic pre-training dataset with massive 1D sequences and 3D structures, and propose a hierarchical pre-training paradigm (Fig. <a href="#F1" class="usa-link">1</a>), accompanied by 2 customized training objectives: sequence-structure matching (SSM) and cross-level reconstruction (CLR). These methodologies allow the model to concurrently process and analyze antibody sequence and structure data, further modeling comprehensive antibody representations to facilitate the deciphering of the biological language.</p>
<figure class="fig xbox font-sm" id="F1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce22/12364524/da99b9388b39/research.0721.fig.001.jpg" loading="lazy" height="547" width="660" alt="Fig. 1."></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of the proposed hierarchical pre-training paradigm containing 2 stages. (A) In stage I, S<sup>2</sup>ALM aims at general sequence-structure learning with protein sequences and structures. In stage II, S<sup>2</sup>ALM learns antibody-specific multi-level knowledge using antibody sequences and structures. (B) Masked language modeling (MLM) reconstructs the masked tokens based on the contextualized information. (C) Sequence-structure matching (SSM) identifies the matching relationships between 1D and 3Di sequences. (D) Cross-level reconstruction (CLR) reconstructs the corrupted tokens based on hybrid information from both 1D and 3Di sequences.</p></figcaption></figure><p>Through extensive interpretability analyses, S<sup>2</sup>ALM exhibited an emergent understanding of unobserved patterns inherent in antibody structures and functions. Furthermore, to illustrate S<sup>2</sup>ALM ’s practical effectiveness, we conducted in-depth examinations to assess the performance of S<sup>2</sup>ALM across a range of downstream scenarios for antibody analysis. The outstanding performance verifies the S<sup>2</sup>ALM’s substantial potential to serve as the novel antibody foundation model and highlights its applicability in real-world therapeutic development.</p>
<p>The main contributions of S<sup>2</sup>ALM are 3-fold:</p>
<ul class="list" style="list-style-type:none">
<li>
<span class="label">•</span><p class="display-inline">Pre-trained on a large-scale multi-level dataset containing 75 million 1D sequences and 11.7 million 3D structures from protein and antibody domains, S<sup>2</sup>ALM learns comprehensive antibody representations uncovering inherent patterns within structures and functions.</p>
</li>
<li>
<span class="label">•</span><p class="display-inline">A hierarchical pre-training paradigm incorporated with 2 customized training objectives, sequence-structure matching and cross-level reconstruction, is proposed to promote the injection of antibody structural information.</p>
</li>
<li>
<span class="label">•</span><p class="display-inline">S<sup>2</sup>ALM consistently exceeds state-of-the-art performance on extensive tasks including antigen binding prediction, B cell maturation analysis, antibody paratope prediction, binding affinity prediction, and computational antibody design, exhibiting its broad applicability in therapeutic antibody development and immune response analysis.</p>
</li>
</ul></section><section id="sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="sec3"><h3 class="pmc_sec_title">S<sup>2</sup>ALM: pre-trained ALM integrating sequences and structures</h3>
<p>In this paper, we pre-trained a large-scale antibody-specific language model S<sup>2</sup>ALM, which integrates multi-level information of 1D sequences and 3D structures for comprehensive antibody representation learning. To accomplish this, the model architecture is built upon ESM-2 [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>], comprising 650 million trainable parameters to facilitate the large-scale pre-training. Additionally, we collect a large-scale comprehensive training dataset, composed of sequences and structures from both protein and antibody domains. Furthermore, Foldseek [<a href="#B20" class="usa-link" aria-describedby="B20">20</a>] is introduced to execute the efficient encoding of protein and antibody 3D structures, transforming 3D structures to 3Di sequences. Given sufficient 1D and 3Di sequences of proteins and antibodies, we propose a hierarchical pre-training paradigm, where 2 pre-training stages are constructed to foster holistic understandings of general proteins and specific antibodies correspondingly. To further combine sequential and structural information, we develop 2 customized pre-training objectives, namely, sequence-structure matching and cross-level reconstruction. Equipped with a comprehensive dataset, an efficient encoding technique, a hierarchical pre-training paradigm, and tailored pre-training objectives, the resulting S<sup>2</sup>ALM acquires general-purpose antibody representations containing biological information derived from both 1D sequences and 3D structures. To our best knowledge, S<sup>2</sup>ALM is the first large-scale pre-trained antibody-specific language model that encodes sequences and structures in a hybrid manner for comprehensive representation learning. In contrast to ALMs solely trained on massive sequences, our main contribution to the biological community lies in the additional infusion of structural information into S<sup>2</sup>ALM. (See detailed advantages over other large language models [LLMs] in Note S4.) The structure-enhanced representations demonstrate a strong potential to broadly benefit antibody understanding and generation applications.</p></section><section id="sec4"><h3 class="pmc_sec_title">Observation of multi-scale organization in antibody representations</h3>
<p>The variation observed in large antibody sequence datasets is influenced by processes at many scales, including functional properties that affect binding to specific antigens, species selection biases, and isotype biases that reflect different B cell maturation stages. Uncovering inherent patterns and properties is crucial for comprehending antibody properties. Unsupervised learning captures latent factors that, while unobserved, prove instrumental in elucidating the biological sequence variation [<a href="#B9" class="usa-link" aria-describedby="B9">9</a>]. We investigate the representation space of the pre-trained large language models at aforementioned multiple scales to look for signatures of biological organization.</p>
<p>For a more intuitive comparison, the t-SNE algorithm [<a href="#B21" class="usa-link" aria-describedby="B21">21</a>] is employed to visualize the antibody representations generated by the last layer of LLMs. To comprehensively evaluate the effectiveness of large-scale pre-training, it is necessary to compare representations encoded by the untrained S<sup>2</sup>ALM with randomly initialized weights and the pre-trained S<sup>2</sup>ALM [<a href="#B7" class="usa-link" aria-describedby="B7">7</a>]. We also include the well-established and powerful ESM-2 [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>] (pre-trained on protein sequences) for comparison to further verify the superiority of the constructed S<sup>2</sup>ALM.</p>
<section id="sec5"><h4 class="pmc_sec_title">Pre-training encodes functional specificity</h4>
<p>Antibodies have emerged as essential therapeutic agents in the treatment of various autoimmune, infectious, and metabolic diseases, mainly owing to their ability to specifically bind to corresponding antigens [<a href="#B6" class="usa-link" aria-describedby="B6">6</a>,<a href="#B22" class="usa-link" aria-describedby="B22">22</a>]. Precise identification of antibody functional specificity will greatly enhance the progress of antibody development and optimization. From the Observed Antibody Space (OAS) database [<a href="#B23" class="usa-link" aria-describedby="B23">23</a>], we systematically identify and filter out 6,000 antibodies that exhibit specific binding affinities toward 3 distinct pathogens: HIV, Ebola virus, and SARS-CoV-2.</p>
<p>We hereby present the t-SNE visualization analyses in Fig. <a href="#F2" class="usa-link">2</a>A. Surprisingly, ESM-2 [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>] somewhat succeeds in distinguishing among these 3 types of antibodies, but there remains some confusion regarding antibodies targeting SARS-CoV-2 and the Ebola virus. The pre-trained S<sup>2</sup>ALM produces more clearly aggregated antibody representations for all 3 pathogens, contrasting with scattered representations from the untrained model and ESM-2. This further verifies the necessity of our pre-training phase and indicates that the representations extracted by S<sup>2</sup>ALM contain antibody functional specificity information.</p>
<figure class="fig xbox font-sm" id="F2"><h5 class="obj_head">Fig. 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce22/12364524/a782298374e8/research.0721.fig.002.jpg" loading="lazy" height="597" width="660" alt="Fig. 2."></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The t-SNE visualization results. Different colors indicate antibodies with different categories correspondingly. Untrained S<sup>2</sup>ALM and pre-trained ESM-2 [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>] are included for comparison. The visualization analyses demonstrate that S<sup>2</sup>ALM contains information about functional specificity, biological species, and evolutionary isotypes in its comprehensive encoded representations.</p></figcaption></figure></section><section id="sec6"><h4 class="pmc_sec_title">Pre-training encodes biological species</h4>
<p>Due to substantial differences in their genetic information and immune function mechanisms, antibodies from different species are expected to exhibit notable distribution disparities. We randomly sample and collect 15,000 antibody sequences from the OAS database [<a href="#B23" class="usa-link" aria-describedby="B23">23</a>], ensuring an equal number of distinct species. The primary sources of biological species are taken into account for antibody sequencing, including camel, human, mouse, rabbit, and rat.</p>
<p>The t-SNE visualization is shown in Fig. <a href="#F2" class="usa-link">2</a>B. We first observe that, when encoded by untrained S<sup>2</sup>ALM, all antibodies from different species tend to mix together in the t-SNE 2-dimensional space. Additionally, ESM-2 [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>] successfully uncovers some clue about the species origins of various antibodies, whereas there are still some confounding issues regarding the boundaries of the clusters. Notably, antibodies of distinct species are nicely clustered together after being encoded by pre-trained S<sup>2</sup>ALM, indicating that the learned representations are encoded with rich biological species information.</p></section><section id="sec7"><h4 class="pmc_sec_title">Pre-training encodes evolutionary isotypes</h4>
<p>Antibodies of different isotypes activate distinct effector mechanisms, manifesting at different stages of the immune response, and differing in structures and locations. Isotype expression reflects the maturation stage of a B cell. Naive B cells express immunoglobulin M (IgM) and IgD isotypes with unmutated variable genes, while expression of other antibody isotypes (IgG, IgA, and IgE) occurs after antigen exposure [<a href="#B24" class="usa-link" aria-describedby="B24">24</a>]. Distinguishing antibody isotypes evaluates the injection of evolutionary information during the model pre-training phase. We select 2,000 antibody sequences from the OAS database [<a href="#B23" class="usa-link" aria-describedby="B23">23</a>] per distinct isotype, resulting in a total of 10,000 sequences for t-SNE visualization analyses.</p>
<p>As illustrated in Fig. <a href="#F2" class="usa-link">2</a>C, when exploiting representations extracted by untrained S<sup>2</sup>ALM and the pre-trained ESM-2 [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>] for t-SNE projection, the organizations of antibodies with various isotypes are predominantly diffuse. This may be attributed to the fact that isotype classification heavily relies on antibody-specific evolutionary information, which the untrained S<sup>2</sup>ALM and the pre-trained ESM-2 are unable to capture. In contrast, we can observe a distinct clustering phenomenon utilizing antibody representations with different isotypes derived from the pre-trained S<sup>2</sup>ALM. The visualization provides a sanity check for the ability of S<sup>2</sup>ALM to extract the valuable information of antibody evolutionary isotypes.</p></section></section><section id="sec8"><h3 class="pmc_sec_title">S<sup>2</sup>ALM captures clues on antibody structure</h3>
<p>It is essential for antibody-specific language models to identify and focus on crucial interaction sites during the representation learning process. The multi-head self-attention mechanism, which specifically focuses on the different aspects of antibodies, has the potential to capture sophisticated interdependencies within the antibody structure. As a practical demonstration of S<sup>2</sup>ALM’s proficiency in modeling the antibody structural interaction patterns, we execute a structural interpretability analysis, and we find that residue pairs with high self-attention scores can accurately reveal long-range structural contacts. Specifically, the antibody STE90-C11 (Protein Data Bank [PDB]: 7B3O), a SARS-CoV-2 neutralizing antibody that binds to the ACE2-RBD interface, is selected as an example input. The self-attention scores from the last hidden layer in S<sup>2</sup>ALM are extracted to build the heatmap, and we identify the potential hydrogen bond locations within the antibody structure. This allows assessing whether S<sup>2</sup>ALM identifies antibody residue structural interactions. As shown in Fig. <a href="#F3" class="usa-link">3</a>A, the heatmap displays a high self-attention score between residues TRP157 and SER183 (highlighted in orange), serving as an indicator of potential structural associations among these 2 residues. Correspondingly, the intricate crystal structure precisely substantiates the existence of a hydrogen bond between residues TRP157 and SER183. While the interpretation of attention mechanisms remains an active area of research, this analysis provides an intuitive way to gain profound comprehension of the model’s functional mechanism. S<sup>2</sup>ALM showcases its ability to meaningfully highlight the key interaction sites within antibody structures, reaffirming the superiority of incorporating structural information during ALM pre-training.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce22/12364524/7497385bbd63/research.0721.fig.003.jpg" loading="lazy" height="634" width="660" alt="Fig. 3."></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>S<sup>2</sup>ALM exhibits superior performance on antibody understanding and generation tasks. (A) Interpretability analysis of S<sup>2</sup>ALM in capturing antibody structural interaction patterns. The heatmap reveals the self-attention values of the STE90-C11’s heavy chain, derived from the last hidden layer of the third head in S<sup>2</sup>ALM. The crystal structure of STE90-C11 (PDB: 7B3O) confirms the interaction mediated by the hydrogen bond between TRP157 and SER183. (B) Evaluation results of the generated antibodies on antibody CDR design task. S<sup>2</sup>ALM simultaneously balances the generative PPL, AAR, and DIV. (C and D) Experimental performance on antigen binding capability prediction, B cell maturation analysis, and antibody paratope prediction tasks. Antibody paratope prediction datasets are from Wang et al. [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>] in (C) and Leem et al. [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>] in (D). S<sup>2</sup>ALM consistently achieves state-of-the-art performance across all included evaluation metrics compared to all baseline models. (E) Structural evaluation results of the generated antibodies, utilizing AlphaFold3 [<a href="#B25" class="usa-link" aria-describedby="B25">25</a>] for 3D structure prediction. S<sup>2</sup>ALM surpasses other baseline models in terms of pLDDT, pTM, and ipTM. (F) 3D structure visualization of the generated antigen–antibody complexes. Targeting 3 specific pathogens (i.e., Vaccinia virus, <em>Neisseria meningitidis</em>, and Influenza B virus), S<sup>2</sup>ALM is employed to design the antibody CDR-H3 (highlighted in red). The stable and regular 3D structures of designed antigen–antibody complexes fully demonstrate the superiority of S<sup>2</sup>ALM .</p></figcaption></figure></section><section id="sec9"><h3 class="pmc_sec_title">S<sup>2</sup>ALM accurately predicts antigen binding capacity</h3>
<p>Establishing the precise binding specificity between antibodies and their target antigens is central to accelerating advancements in therapeutic antibody optimization and deepening our comprehension of the intricacies inherent to the immune response. Consequently, rapid and accurate evaluation of the antigen binding capacity stands as a pressing and fundamental requirement in the realm of antibody research and development. Antigen binding capacity prediction is a binary sequence classification task to determine binding capacity with the specific antigen, where each antibody is mapped to a neutralization or non-neutralization label. Specifically, we input antibody sequences into the pre-trained S<sup>2</sup>ALM to extract biological representations. The classification head (i.e., a linear projection layer) is added on top of the pre-trained model to output logits for identifying neutralization or non-neutralization classes. With a keen focus on the interaction between the target antigen human epidermal growth factor receptor 2 (HER2) and the clinically approved wild-type antibody trastuzumab, we compile a dataset of antibody-expressing sequences. These sequences are designed by replacing the original trastuzumab sequence with a myriad of variant CDR-H3 fragments from the heavy chain, as detailed in Refs. [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>,<a href="#B26" class="usa-link" aria-describedby="B26">26</a>]. The resulting dataset boasts an impressive scale of 21,612 unique antibody sequences and applies the training/validation/test split of 15,128/3,242/3,242 (i.e., 70%/15%/15%). Our comparative analysis encompasses baselines of different types, including those pre-trained on protein sequences (i.e., ESM-1b [<a href="#B27" class="usa-link" aria-describedby="B27">27</a>] and MSA-1b [<a href="#B28" class="usa-link" aria-describedby="B28">28</a>]) and those pre-trained on antibody sequences (i.e., AbLang-H [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>], AbLang-L [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>], AntiBERTa [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>], and EATLM [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>]). Three classification metrics are utilized for evaluation. AUC is the area under the receiver operating characteristic curve, indicating the performance at all classification thresholds. F1 is the average weighted score of precision and recall. Matthews correlation coefficient (MCC) is the coefficient between true and predicted values.</p>
<p>We report the evaluation results of the proposed S<sup>2</sup>ALM and various baselines in Table <a href="#T1" class="usa-link">1</a> (left). On the one hand, ESM-1b and MSA-1b yield relatively inferior performance due to the domain knowledge gap between proteins and antibodies. On the other hand, among baselines pre-trained on antibody sequences, AbLang-H outperforms AbLang-L and AntiBERTa, indicating the advantage of separate training for heavy and light chain sequences. However, the true standout in these evaluations is S<sup>2</sup>ALM, which surpasses all baselines and sets a new state-of-the-art performance in this task. Such accomplishment underscores the capacity of S<sup>2</sup>ALM to transcend traditional sequence-based models in antigen binding prediction.</p>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Evaluation performance on antigen binding capacity prediction, B cell maturation analysis, and antibody paratope prediction tasks. The best experimental results are highlighted in bold. S<sup>2</sup>ALM achieves considerable improvements across all 3 antibody-specific tasks. This reveals the potential of S<sup>2</sup>ALM as the antibody-specific foundation model constructed for comprehensive antibody representation learning.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th rowspan="2" align="left" colspan="1">Method</th>
<th colspan="3" align="center" rowspan="1">Antigen binding prediction</th>
<th align="center" rowspan="1" colspan="1">Maturation analysis</th>
<th colspan="3" align="center" rowspan="1">Antibody paratope prediction</th>
</tr>
<tr>
<th align="center" rowspan="1" colspan="1">AUC</th>
<th align="center" rowspan="1" colspan="1">F1</th>
<th align="center" rowspan="1" colspan="1">MCC</th>
<th align="center" rowspan="1" colspan="1">ACC</th>
<th align="center" rowspan="1" colspan="1">AUC</th>
<th align="center" rowspan="1" colspan="1">F1</th>
<th align="center" rowspan="1" colspan="1">MCC</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">ESM-1b [<a href="#B27" class="usa-link" aria-describedby="B27">27</a>]</td>
<td align="center" rowspan="1" colspan="1">0.917</td>
<td align="center" rowspan="1" colspan="1">0.854</td>
<td align="center" rowspan="1" colspan="1">0.689</td>
<td align="center" rowspan="1" colspan="1">0.503</td>
<td align="center" rowspan="1" colspan="1">0.886</td>
<td align="center" rowspan="1" colspan="1">0.669</td>
<td align="center" rowspan="1" colspan="1">0.547</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">MSA-1b [<a href="#B28" class="usa-link" aria-describedby="B28">28</a>]</td>
<td align="center" rowspan="1" colspan="1">0.921</td>
<td align="center" rowspan="1" colspan="1">0.857</td>
<td align="center" rowspan="1" colspan="1">0.689</td>
<td align="center" rowspan="1" colspan="1">0.416</td>
<td align="center" rowspan="1" colspan="1">0.887</td>
<td align="center" rowspan="1" colspan="1">0.679</td>
<td align="center" rowspan="1" colspan="1">0.557</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AbLang-H [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>]</td>
<td align="center" rowspan="1" colspan="1">0.918</td>
<td align="center" rowspan="1" colspan="1">0.861</td>
<td align="center" rowspan="1" colspan="1">0.704</td>
<td align="center" rowspan="1" colspan="1">0.570</td>
<td align="center" rowspan="1" colspan="1">0.878</td>
<td align="center" rowspan="1" colspan="1">0.674</td>
<td align="center" rowspan="1" colspan="1">0.546</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AbLang-L [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>]</td>
<td align="center" rowspan="1" colspan="1">0.917</td>
<td align="center" rowspan="1" colspan="1">0.856</td>
<td align="center" rowspan="1" colspan="1">0.682</td>
<td align="center" rowspan="1" colspan="1">0.546</td>
<td align="center" rowspan="1" colspan="1">0.882</td>
<td align="center" rowspan="1" colspan="1">0.680</td>
<td align="center" rowspan="1" colspan="1">0.553</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AntiBERTa [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>]</td>
<td align="center" rowspan="1" colspan="1">0.918</td>
<td align="center" rowspan="1" colspan="1">0.843</td>
<td align="center" rowspan="1" colspan="1">0.678</td>
<td align="center" rowspan="1" colspan="1">0.565</td>
<td align="center" rowspan="1" colspan="1">0.879</td>
<td align="center" rowspan="1" colspan="1">0.690</td>
<td align="center" rowspan="1" colspan="1">0.559</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">EATLM [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>]</td>
<td align="center" rowspan="1" colspan="1">0.922</td>
<td align="center" rowspan="1" colspan="1">0.862</td>
<td align="center" rowspan="1" colspan="1">0.699</td>
<td align="center" rowspan="1" colspan="1">0.581</td>
<td align="center" rowspan="1" colspan="1">0.887</td>
<td align="center" rowspan="1" colspan="1">0.698</td>
<td align="center" rowspan="1" colspan="1">0.575</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">S<sup>2</sup>ALM</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.931</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.868</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.705</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.588</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.893</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.708</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.583</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec10"><h3 class="pmc_sec_title">S<sup>2</sup>ALM precisely distinguishes B cell maturation states</h3>
<p>B cells occupy a central position in the immune system’s protective arsenal, due to their distinctive ability to generate antibodies. In the human body, these antibodies serve as a vital line of defense against invading pathogens [<a href="#B7" class="usa-link" aria-describedby="B7">7</a>]. Therefore, exploring and analyzing the process of B cell antibody maturation remains crucial, enhancing our understanding of the intricate mechanisms that unfold during immune system evolution (i.e., a critical biological process affecting the function and antigen binding specificity of antibodies) [<a href="#B29" class="usa-link" aria-describedby="B29">29</a>,<a href="#B30" class="usa-link" aria-describedby="B30">30</a>]. B cell maturation analysis is a 6-category classification task. Its core objective lies in accurately distinguishing the maturation states of B cell antibody sequences. Each antibody sequence belongs to one of {<em>immature, transitional, mature, plasmacytes, memory IgD+, memory IgD-</em>} states. Accomplishing this necessitates a model capable of learning an evolution-aware representation sensitive to different B cell maturation states, with a linear projection layer serving as the classification head. We utilize a total of 88,094 antibody sequences from [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>,<a href="#B31" class="usa-link" aria-describedby="B31">31</a>] with 6 maturation states and apply the default data split. The evaluation metric is accuracy (ACC), calculating the ratio of correct predictions.</p>
<p>Table <a href="#T1" class="usa-link">1</a> (center) demonstrates the findings of model ability to distinguish between multiple B cell mature states. For the baselines pre-trained on protein sequences, both ESM-1b [<a href="#B27" class="usa-link" aria-describedby="B27">27</a>] and MSA-1b [<a href="#B28" class="usa-link" aria-describedby="B28">28</a>] present inferior performance. Even though armed with evolution-aware representations learned from multiple protein sequence alignments, MSA-1b performs poorly in this antibody maturation related task. Such phenomenon suggests that there is a gap between protein and antibody domains. For those pre-trained on antibody-specific sequences (i.e., AbLang-H [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>], AbLang-L [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>], AntiBERTa [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>], and EATLM [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>]), they deliver promising results in this task. Notably, S<sup>2</sup>ALM, pre-trained on both antibody sequences and structures, achieves a substantial leap forward and surpasses all baselines by a large margin. The evaluation result highlights the efficacy of S<sup>2</sup>ALM in improving the comprehension of B cell maturation and immune evolutionary mechanisms.</p></section><section id="sec11"><h3 class="pmc_sec_title">S<sup>2</sup>ALM thoroughly identifies antibody paratopes</h3>
<p>In immunology research, identifying the amino acids in immunoglobulins that specifically bind to antigens (i.e., the antibody paratope) is one of the most crucial and challenging tasks [<a href="#B32" class="usa-link" aria-describedby="B32">32</a>]. The antibody paratope, usually composed of multiple amino acids in the complementarity-determining regions, forms the antibody binding site, which is essential for antigen interactions in the immune response [<a href="#B33" class="usa-link" aria-describedby="B33">33</a>]. Antibody paratope prediction could significantly help to enhance our understanding of the binding mechanisms of therapeutic antibodies. It is a binary token labeling task that identifies specific binding positions within antibody sequences, assigning a 0/1 label to each amino acid residue. We add a linear projection layer atop the pre-trained model to act as the prediction head. For paratope prediction, we utilize 2 antibody datasets sourced from Refs. [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>,<a href="#B18" class="usa-link" aria-describedby="B18">18</a>], which comprise 277 and 900 antibody sequences annotated with token-wise paratope regions. Numerous baseline models are utilized, including Sapiens-H [<a href="#B34" class="usa-link" aria-describedby="B34">34</a>], ProtBERT [<a href="#B35" class="usa-link" aria-describedby="B35">35</a>], AntiBERTa [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>], AntiBERTy [<a href="#B16" class="usa-link" aria-describedby="B16">16</a>], ESM-1b [<a href="#B27" class="usa-link" aria-describedby="B27">27</a>], ESM-2 [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>], BALM [<a href="#B9" class="usa-link" aria-describedby="B9">9</a>], MSA-1b [<a href="#B28" class="usa-link" aria-describedby="B28">28</a>], AbLang-H [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>], AbLang-L [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>], and EATLM [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>]. Experimental metrics used to evaluate the prediction capability include AUC, F1 score, and MCC.</p>
<p>Table <a href="#T1" class="usa-link">1</a> (right) reports results on antibody paratope prediction benchmark [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>], and we observe that S<sup>2</sup>ALM achieves considerable improvements while other baselines show no significant difference in performance. Fig. <a href="#F3" class="usa-link">3</a>D depicts results on antibody paratope prediction benchmark [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>], where S<sup>2</sup>ALM stands out by outperforming all other baselines across all criteria. The holistic experimental results showcase the superiority of our model in efficiently capturing features associated with antibody binding sites. This further highlights S<sup>2</sup>ALM’s high specificity, accuracy, and in-depth understanding of antibody complex binding mechanisms.</p></section><section id="sec12"><h3 class="pmc_sec_title">S<sup>2</sup>ALM accurately predicts antigen–antibody binding affinity</h3>
<p>Therapeutic antibody development has become an increasingly popular strategy for drug discovery, but still involves intensive laboratory experiments [<a href="#B36" class="usa-link" aria-describedby="B36">36</a>]. Recently, deep learning methods have emerged as a promising approach to accelerate such process by identifying high-affinity antibody candidates [<a href="#B36" class="usa-link" aria-describedby="B36">36</a>,<a href="#B37" class="usa-link" aria-describedby="B37">37</a>]. The property of binding affinity, which quantifies the binding strength between antigen–antibody pairs, plays a critical role in antibody development and optimization. To assess the model’s performance in predicting the binding affinity, we first utilize 2 datasets, 14H and 14L [<a href="#B38" class="usa-link" aria-describedby="B38">38</a>], which contain abundant antibodies labeled with binding affinity values. Within these 2 datasets, the affinity values quantify the binding strength of antibodies to a stable peptide in the HR2 region of SARS-CoV-2. To further expand the diversity of binding antigens and thereby enhance both the comprehensiveness and reliability of this experiment, we also include the BioMap [<a href="#B39" class="usa-link" aria-describedby="B39">39</a>] dataset for the evaluation of binding affinity prediction. The BioMap dataset contains 1,706 antigen–antibody pairing data with labeled binding affinity values, in which most antibodies stem from human and mouse sources. As for the experiment execution, the 14H, 14L, and BioMap datasets are each divided into the training/validation/test set according to the 80%/10%/10% split following He et al. [<a href="#B7" class="usa-link" aria-describedby="B7">7</a>]. Based on the test sets, we record experimental results for fair comparison with extensive baselines (i.e., Vanilla BERT [<a href="#B40" class="usa-link" aria-describedby="B40">40</a>], Ens-Grad [<a href="#B41" class="usa-link" aria-describedby="B41">41</a>], AbMAP [<a href="#B42" class="usa-link" aria-describedby="B42">42</a>], AntiBERTa2 [<a href="#B43" class="usa-link" aria-describedby="B43">43</a>], ESM-F [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>], and A2binder [<a href="#B7" class="usa-link" aria-describedby="B7">7</a>]). In this binding affinity prediction task, we use the Pearson correlation and the Spearman correlation as evaluation metrics. Building upon pre-trained S<sup>2</sup>ALM, we extend and enhance the model architecture to enable the accurate learning of antigen–antibody binding rules. As illustrated in Fig. <a href="#F4" class="usa-link">4</a> (left), S<sup>2</sup>ALM is employed to extract features of antibodies, while ESM-2 [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>] is introduced for the feature encoding of antigens. The antibody and antigen features extracted by pre-trained models are subsequently integrated through concatenation. Finally, the MLP (a multilayer perceptron) model utilizes the combined feature representation to predict binding affinity.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce22/12364524/f8cf12172e55/research.0721.fig.004.jpg" loading="lazy" height="478" width="660" alt="Fig. 4."></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of the S<sup>2</sup>ALM experimental workflow. Left: Detailed architecture of antigen–antibody binding affinity prediction. S<sup>2</sup>ALM is applied to extract antibody features, while ESM-2 is used for antigen feature encoding. The encoded features are concatenated and input into the MLP model to predict binding affinity. Right: Elaborate workflow of computational antibody design. S<sup>2</sup>ALM generates antibodies via sequence infilling, retaining top 3 samples with the highest naturalness. AlphaFold3 is incorporated for predicting 3D structures of antigen–antibody complexes.</p></figcaption></figure><p>Table <a href="#T2" class="usa-link">2</a> illustrates the performance comparison of models in predicting the antigen–antibody binding affinity. S<sup>2</sup>ALM consistently achieves state-of-the-art performance in Pearson correlation and Spearman correlation metrics. We observe that S<sup>2</sup>ALM significantly surpasses all baselines, except for the BioMap dataset, where it achieves the same Spearman correlation as A2binder. The experimental results further verify the heightened precision of S<sup>2</sup>ALM in predicting the antigen–antibody binding affinity. By integrating both sequence and structure data for large-scale pre-training, S<sup>2</sup>ALM effectively captures the specific functional information of antibodies. We believe that S<sup>2</sup>ALM has the great potential to serve as the antibody-specific foundation model to promote the therapeutic antibody development in real-world scenarios.</p>
<section class="tw xbox font-sm" id="T2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>The performance comparison on antigen–antibody binding affinity prediction tasks. We highlight the best experimental results in bold. S<sup>2</sup>ALM achieves state-of-the-art performance across all evaluation metrics on the 3 antibody datasets (i.e., 14H, 14L and BioMap).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th rowspan="2" align="left" colspan="1">Method</th>
<th colspan="2" align="center" rowspan="1">14H</th>
<th colspan="2" align="center" rowspan="1">14L</th>
<th colspan="2" align="center" rowspan="1">BioMap</th>
</tr>
<tr>
<th align="center" rowspan="1" colspan="1">Pearson</th>
<th align="center" rowspan="1" colspan="1">Spearman</th>
<th align="center" rowspan="1" colspan="1">Pearson</th>
<th align="center" rowspan="1" colspan="1">Spearman</th>
<th align="center" rowspan="1" colspan="1">Pearson</th>
<th align="center" rowspan="1" colspan="1">Spearman</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Vanilla BERT [<a href="#B40" class="usa-link" aria-describedby="B40">40</a>]</td>
<td align="center" rowspan="1" colspan="1">0.594 (0.021)</td>
<td align="center" rowspan="1" colspan="1">0.480 (0.025)</td>
<td align="center" rowspan="1" colspan="1">0.607 (0.024)</td>
<td align="center" rowspan="1" colspan="1">0.611 (0.027)</td>
<td align="center" rowspan="1" colspan="1">0.492 (0.036)</td>
<td align="center" rowspan="1" colspan="1">0.498 (0.037)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ens-Grad [<a href="#B41" class="usa-link" aria-describedby="B41">41</a>]</td>
<td align="center" rowspan="1" colspan="1">0.601 (0.016)</td>
<td align="center" rowspan="1" colspan="1">0.476 (0.023)</td>
<td align="center" rowspan="1" colspan="1">0.637 (0.019)</td>
<td align="center" rowspan="1" colspan="1">0.645 (0.023)</td>
<td align="center" rowspan="1" colspan="1">0.645 (0.031)</td>
<td align="center" rowspan="1" colspan="1">0.664 (0.033)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AbMAP [<a href="#B42" class="usa-link" aria-describedby="B42">42</a>]</td>
<td align="center" rowspan="1" colspan="1">0.606 (0.015)</td>
<td align="center" rowspan="1" colspan="1">0.510 (0.015)</td>
<td align="center" rowspan="1" colspan="1">0.674 (0.012)</td>
<td align="center" rowspan="1" colspan="1">0.685 (0.016)</td>
<td align="center" rowspan="1" colspan="1">0.637 (0.027)</td>
<td align="center" rowspan="1" colspan="1">0.673 (0.029)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AntiBERTa2 [<a href="#B43" class="usa-link" aria-describedby="B43">43</a>]</td>
<td align="center" rowspan="1" colspan="1">0.623 (0.011)</td>
<td align="center" rowspan="1" colspan="1">0.545 (0.008)</td>
<td align="center" rowspan="1" colspan="1">0.673 (0.013)</td>
<td align="center" rowspan="1" colspan="1">0.684 (0.012)</td>
<td align="center" rowspan="1" colspan="1">0.633 (0.022)</td>
<td align="center" rowspan="1" colspan="1">0.670 (0.031)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ESM-F [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>]</td>
<td align="center" rowspan="1" colspan="1">0.634 (0.007)</td>
<td align="center" rowspan="1" colspan="1">0.516 (0.010)</td>
<td align="center" rowspan="1" colspan="1">0.674 (0.011)</td>
<td align="center" rowspan="1" colspan="1">0.681 (0.014)</td>
<td align="center" rowspan="1" colspan="1">0.628 (0.028)</td>
<td align="center" rowspan="1" colspan="1">0.644 (0.024)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">A2binder [<a href="#B7" class="usa-link" aria-describedby="B7">7</a>]</td>
<td align="center" rowspan="1" colspan="1">0.642 (0.012)</td>
<td align="center" rowspan="1" colspan="1">0.553 (0.011)</td>
<td align="center" rowspan="1" colspan="1">0.683 (0.010)</td>
<td align="center" rowspan="1" colspan="1">0.688 (0.015)</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.701 (0.024)</strong>
</td>
<td align="center" rowspan="1" colspan="1">0.746 (0.025)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">S<sup>2</sup>ALM</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.650 (0.013)</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.558 (0.014)</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.687 (0.015)</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.693 (0.017)</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.701 (0.022)</strong>
</td>
<td align="center" rowspan="1" colspan="1">
<strong>0.749 (0.028)</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec13"><h3 class="pmc_sec_title">S<sup>2</sup>ALM enables computational antibody CDR design</h3>
<p>Given the extraordinary performance on antibody understanding tasks related to the evolutionary process and functional mechanism, we additionally incorporate a novel generation task to further evaluate the model’s capability of antibody generation and optimization. The complementarity-determining region (CDR), also known as the hypervariable region, primarily determines the antibody’s affinity and specificity for its target. Given the crucial role of CDRs, computational antibody design primarily focuses on generating these regions. Specifically, antibody CDR design is a sequence infilling task to generate masked CDRs (more exactly, CDR-H3) based on the contextualized representation. For antibody CDR design, the dataset curated by Jin et al. [<a href="#B44" class="usa-link" aria-describedby="B44">44</a>] follows the training/validation/test split of 2,282/291/291 samples derived from the Coronavirus Antibody Database (CoV-AbDab) [<a href="#B45" class="usa-link" aria-describedby="B45">45</a>]. CoV-AbDab is a public database documenting molecular information (sequences and structures) of all published and patented antibodies and nanobodies that can bind to coronaviruses, including SARS-CoV-2, SARS-CoV-1, and MERS-CoV [<a href="#B45" class="usa-link" aria-describedby="B45">45</a>]. During this experiment, we mask the CDRs in both 1D and 3Di antibody sequences for S<sup>2</sup>ALM to reconstruct. To benchmark the quality of antibody sequences generated by S<sup>2</sup>ALM, we employ ProtBERT [<a href="#B35" class="usa-link" aria-describedby="B35">35</a>], AbLang-H [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>], and ReprogBERT [<a href="#B22" class="usa-link" aria-describedby="B22">22</a>] for comparison. Three metrics are utilized for holistic evaluation. Perplexity (PPL) measures how well the language model predicts the generative tokens. Here, we exploit off-the-shelf ProGen [<a href="#B46" class="usa-link" aria-describedby="B46">46</a>] for calculation following Melnyk et al. [<a href="#B22" class="usa-link" aria-describedby="B22">22</a>]. Lower values signify better performance, indicating stronger naturalness of the generated CDRs. Amino acid recovery (AAR) is computed for the specific sequence region of interest, measuring the percent of the exact matches between ground truth and sampled sequences. The higher the AAR, the more accurate the recovery. Diversity (DIV) computes the complement of the average recovery of all pairwise comparisons using sampled CDRs. A higher value corresponds to an increased dissimilarity among the samples themselves.</p>
<p>The evaluation results are shown in Fig. <a href="#F3" class="usa-link">3</a>B. In spite of the high AAR, ProtBERT struggles in low generation diversity. AbLang-H and ReprogBERT are particularly talented in generation diversity but demonstrate relatively high perplexity. In contrast, S<sup>2</sup>ALM achieves substantially high AAR and the lowest perplexity while maintaining the comparable high diversity to ReprogBERT, accomplishing the balance of all 3 generative metrics. Leveraging the CDR infilling paradigm, S<sup>2</sup>ALM successfully generates CDR loop libraries with improved in silico developability profiles. This strongly indicates that S<sup>2</sup>ALM could serve as an effective and insightful tool for computational antibody design, benefiting the biomedical research community.</p></section><section id="sec14"><h3 class="pmc_sec_title">3D structural visualization</h3>
<p>To further manifest the advantages of incorporating structural information during ALM pre-training, we conduct 3D visualization of antigen–antibody complexes generated by S<sup>2</sup>ALM. The thorough workflow for antibody sequence generation and structure visualization is depicted in Fig. <a href="#F4" class="usa-link">4</a> (right). Based on a holistic antigen–antibody complex dataset of 60 antigen–antibody pairs constructed by Adolf-Bryfogle et al. [<a href="#B47" class="usa-link" aria-describedby="B47">47</a>], S<sup>2</sup>ALM designs antibody CDR-H3 via sequence infilling. For each antigen–antibody complex, 100 CDR-H3 samples are generated and combined with standard framework regions to obtain 100 full-length antibody heavy chains. We select the top 3 antibody heavy chains with the highest naturalness using ProGen2 [<a href="#B48" class="usa-link" aria-describedby="B48">48</a>], following the standard protocol [<a href="#B7" class="usa-link" aria-describedby="B7">7</a>,<a href="#B22" class="usa-link" aria-describedby="B22">22</a>]. Subsequently, AlphaFold3 [<a href="#B25" class="usa-link" aria-describedby="B25">25</a>], highly capable of predicting the antigen–antibody joint structure, is applied to generate the 3D structures of complexes comprising antigen and generated antibodies (including antibody heavy chains and corresponding light chains). Evaluation metrics of the folded 3D structure are the predicted template-modeling (pTM) score, the interface pTM (ipTM), and the predicted local distance difference test (pLDDT). Moreover, to facilitate a thorough comparison, we involve a variety of language models: ESM-2 [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>], pre-trained on general protein sequences; SaProt [<a href="#B49" class="usa-link" aria-describedby="B49">49</a>], which harnesses both protein sequences and structures for pre-training; and 2 dedicated ALMs, AntiBERTa [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>] and AbLang-H [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>]. The structural evaluation results through AlphaFold3 are illustrated in Fig. <a href="#F3" class="usa-link">3</a>E. SaProt and AbLang-H stand out among all baselines, verifying the advantages of injecting structural information and promoting antibody-specific learning. S<sup>2</sup>ALM consistently achieves the optimal pTM, ipTM, and pLDDT scores compared to other methods, demonstrating that the generated antibodies are more likely to target the corresponding antigens and form stable binding complexes. Furthermore, we visualize the generated 3D complex structures targeting 3 clinically important pathogens (i.e., Vaccinia virus, <em>Neisseria meningitidis</em>, and Influenza B virus). Fig. <a href="#F3" class="usa-link">3</a>F depicts some visualization examples. The stable and regular 3D structures of designed antigen–antibody complexes fully exhibit the superior generative capability of S<sup>2</sup>ALM, which pioneeringly incorporates 3D structural information into ALM pre-training.</p></section></section><section id="sec15"><h2 class="pmc_sec_title">Discussion</h2>
<p>Antibodies are vital proteins produced by the immune system to safeguard against and combat with a variety of diseases [<a href="#B1" class="usa-link" aria-describedby="B1">1</a>]. There are abundant clinical applications of antibody analysis in therapeutic drug discovery and finding novel diagnostics [<a href="#B6" class="usa-link" aria-describedby="B6">6</a>,<a href="#B7" class="usa-link" aria-describedby="B7">7</a>,<a href="#B15" class="usa-link" aria-describedby="B15">15</a>,<a href="#B50" class="usa-link" aria-describedby="B50">50</a>]. Most analyses have focused on comparing high-level differences between cohorts to facilitate antibody understanding, such as number of somatic hypermutations, isotype subclass usage, and V(D)J gene segment usage. With the advancements of machine learning techniques, it has been verified in various studies that transformer-based LLMs significantly promote the understanding and decoding of biological language [<a href="#B6" class="usa-link" aria-describedby="B6">6</a>,<a href="#B7" class="usa-link" aria-describedby="B7">7</a>,<a href="#B10" class="usa-link" aria-describedby="B10">10</a>,<a href="#B51" class="usa-link" aria-describedby="B51">51</a>,<a href="#B52" class="usa-link" aria-describedby="B52">52</a>]. The biological function of an antibody is directly determined by its 3D structure; thus, the modalities of sequence and structure should be tightly coupled to generate a comprehensive representation of the antibody. However, existing works fall short in effectively injecting structural information during antibody pre-training, which substantially hinders the further progress.</p>
<p>In this study, our endeavor herein offers an antibody-specific foundation model that posits the potential to facilitate comprehensive antibody understanding and generation. Extensive machine learning paradigms and techniques are utilized in developing S<sup>2</sup>ALM. To our best knowledge, S<sup>2</sup>ALM stands out as the first ALM to integrate hybrid information of 1D sequences and 3D structures during large-scale pre-training. S<sup>2</sup>ALM extracts comprehensive and meaningful antibody representations, uncovering inherent patterns and properties of functional specificity, biological species, and evolutionary isotypes. Additionally, as a supplement to interpretability analysis, S<sup>2</sup>ALM precisely captures crucial interaction sites within 3D structures. Evaluated on diverse computational scenarios, S<sup>2</sup>ALM exhibits a state-of-the-art performance in antibody representation learning: accurately predicting antigen binding capacity, precisely distinguishing B cell maturation states, thoroughly identifying antibody paratopes, and accurately predicting antigen–antibody binding affinity. To further explore the potential applications, we employ S<sup>2</sup>ALM for computational antibody design leveraging the CDR infilling paradigm. The generated stable and regular 3D structures further indicate the superior generative capability of S<sup>2</sup>ALM. We hold the expectation that our work will usher in a new era of antibody multi-level pre-training via explicitly modeling the profound interrelation between sequences and structures, fostering the development of therapeutic antibody drugs in real-world scenarios.</p>
<p>While our work demonstrates promising and outstanding results in extensive antibody-specific downstream tasks, there are still some areas for improvement and future exploration: (a) Due to computational constraints, the model size of S<sup>2</sup>ALM may not have reached its maximum capacity. (b) Although we circumnavigate such constraints by introducing extra protein structures in our hierarchical pre-training paradigm, the limitation of antibody structure data cannot be overlooked. In the future, we eagerly anticipate the emergence of large-scale antibody structure databases with sufficient experimentally determined 3D structures. Such efforts will fill the void of large-scale antibody structure corpus, thereby fueling multi-level antibody pre-training. (c) The performance of structural pre-training heavily depends on Foldseek [<a href="#B20" class="usa-link" aria-describedby="B20">20</a>], which aims to balance search efficiency and encoding accuracy. Thus, there is still room for improving the representation capability of all methods building upon Foldseek, including ours.</p>
<p>To conclude, this paper documents our efforts to build a comprehensive antibody-specific foundation model to represent the intricacies of the biological world. The capabilities demonstrated by S<sup>2</sup>ALM exhibit considerable promise and underscore several areas in antibody understanding and generation that necessitate substantial advancements. Such a multi-level pre-trained foundational model, integrating antibody sequential and structural information, will prove immensely valuable in accelerating immunotherapy development and enhancing our comprehension of biological phenomena.</p></section><section id="sec16"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="sec17"><h3 class="pmc_sec_title">Pre-training data</h3>
<p>In the field of antibody pre-training, there is an abundance of sequential data, whereas the quantity of structure data remains limited, especially those determined by experiments. To compensate for the inadequacy of experimentally determined antibody structures, we additionally introduce computationally predicted antibody structures and general protein structures for comprehensive large-scale pre-training. Eventually, our holistic pre-training data encompass various levels and span multiple domains, including 75 million 1D sequences and 11.7 million 3D structures from protein and antibody domains. Fig. <a href="#F5" class="usa-link">5</a>A and B presents compositional ratios of various parts in the pre-training dataset, and details of each part are as follows.</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce22/12364524/8830be08e1fd/research.0721.fig.005.jpg" loading="lazy" height="268" width="659" alt="Fig. 5."></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Illustrations of compositional ratios of the pre-training data and the structural encoding protocol. (A) The protein data contains 3 parts: sequences, experimentally determined structures, and computationally predicted structures. (B) The antibody data contains 4 parts: sequences, experimentally determined structures, and computationally predicted structures from ABodyBuilder2 [<a href="#B57" class="usa-link" aria-describedby="B57">57</a>] and IgFold [<a href="#B58" class="usa-link" aria-describedby="B58">58</a>]. (C) Efficient encoding protocol of 3D structures. Foldseek [<a href="#B20" class="usa-link" aria-describedby="B20">20</a>] is employed to discretize the target 3D structure into the 3Di sequences, deciphering 3Di states that describe the tertiary interaction between a residue and its nearest neighbor.</p></figcaption></figure><section id="sec18"><h4 class="pmc_sec_title">Protein sequence and structure data</h4>
<p>Extensive protein sequence and structure data are incorporated for general sequence-structure learning. Specifically, for the primary structural information (i.e., sequential information), 65 million protein sequences are derived from UniRef50 [<a href="#B53" class="usa-link" aria-describedby="B53">53</a>], which is a clustering of UniRef90 seed sequences at 50% sequence identity. Moreover, to alleviate the insufficiency of antibody structure data, the extra protein structure data are incorporated. For the secondary and tertiary structural information, we utilize 0.2 million experimentally determined general protein 3D structures sourced from the PDB [<a href="#B54" class="usa-link" aria-describedby="B54">54</a>] and 10 million computationally predicted protein 3D structures from the AlphaFold Protein Structure Database (AFDB) [<a href="#B55" class="usa-link" aria-describedby="B55">55</a>].</p></section><section id="sec19"><h4 class="pmc_sec_title">Antibody sequence data</h4>
<p>The pre-training antibody sequence data come from the OAS database [<a href="#B23" class="usa-link" aria-describedby="B23">23</a>], which contains over 2.4 billion unpaired sequences and 3 million paired sequences of antibody heavy and light chains. The OAS database boasts a rich collection of antibody sequences from 825 unique subjects spanning across 6 different species, where humans account for 88% and mice account for 11%. We downloaded the OAS database of unpaired and paired antibody sequences on 2024 January 11, and pre-processed the antibody sequence data by filtering, cleaning, and clustering. First, all duplicate antibody sequences were filtered out and removed. Additionally, we excluded antibody sequences containing unusual residues (selenocysteine and pyrrolysine), along with those having any framework region shorter than IMGT defined. Furthermore, to mitigate data leakage, we clustered antibody sequences at 70% sequence identity over the whole sequence using the MMseq2 algorithm [<a href="#B56" class="usa-link" aria-describedby="B56">56</a>]. Alignment coverage in MMseq2 was calculated with respect to the target sequence (“cov-mode 1”), with all other parameters set to their default values. Ultimately, the antibody sequence data comprise 43,551,946 unique antibody sequences, from which we randomly select 10 million sequences for training.</p></section><section id="sec20"><h4 class="pmc_sec_title">Antibody structure data</h4>
<p>Given the central tenet that antibody structure substantially governs its function, we innovatively explore to inject antibody-specific structural knowledge into ALMs. To achieve this, we collect experimentally determined antibody structure data based on the Structural Antibody Database [<a href="#B59" class="usa-link" aria-describedby="B59">59</a>], which is composed of approximately 10 thousand antibody structures retrieved from the PDB [<a href="#B54" class="usa-link" aria-describedby="B54">54</a>]. Considering the limited quantity of experimentally determined antibody structures, we seek antibody structure prediction models for assistance to provide ample computationally predicted structures. Concretely, approximately 150 thousand antibody structures predicted by ABodyBuilder2 [<a href="#B57" class="usa-link" aria-describedby="B57">57</a>] and 1.4 million antibody structures predicted by IgFold [<a href="#B58" class="usa-link" aria-describedby="B58">58</a>] are incorporated for pre-training. Overall, the aforementioned 3 data sources collectively contribute to antibody structure learning.</p></section></section><section id="sec21"><h3 class="pmc_sec_title">Structural encoding technique</h3>
<p>Foldseek [<a href="#B20" class="usa-link" aria-describedby="B20">20</a>] is a computational tool for fast and accurate protein structure search, merging features such as amino acid spatial angles, distances, and sequence positions. It utilizes the vector quantized-variational autoencoder (VQ-VAE) model [<a href="#B60" class="usa-link" aria-describedby="B60">60</a>] to encode the 3D protein structure as distinct and information-rich 3Di tokens. Each amino acid is assigned a token based on its distance and relative position to the nearest amino acids in the folded protein structure. Foldseek achieves this transformation through the identification of the nearest neighbors and the extraction of distinctive features for individual residues [<a href="#B49" class="usa-link" aria-describedby="B49">49</a>]. In this paper, we introduce Foldseek to accomplish the efficient encoding of 3D structures, as depicted in Fig. <a href="#F5" class="usa-link">5</a>C. Concretely, we adopt Foldseek with the default setting of 20 3Di tokens. Aligning all residue sites, protein and antibody 3D structures are transformed into 3Di sequences (i.e., 1D sequences storing 3D structural information). Such technique fills the void in effective encoding of structure data and enables S<sup>2</sup>ALM to handle sequence-structure multi-level information in a hybrid and unified manner, promoting comprehensive antibody representation learning.</p></section><section id="sec22"><h3 class="pmc_sec_title">Hierarchical pre-training paradigm</h3>
<section id="sec23"><h4 class="pmc_sec_title">Stage I: General sequence-structure learning</h4>
<p>In pre-training stage I, we first tokenize the sequences and structures of proteins by an innovative multi-level vocabulary <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M1" display="inline" overflow="linebreak"><mi mathvariant="script">F</mi><mo>=</mo><msub><mi mathvariant="script">V</mi><mn>1</mn></msub><mo>+</mo><msub><mi mathvariant="script">V</mi><mn>2</mn></msub></math></span>. The original sequence alphabet <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M2" display="inline" overflow="linebreak"><msub><mi mathvariant="script">V</mi><mn>1</mn></msub></math></span> consists of 20 standard amino acids. A protein or antibody sequence can be denoted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M3" display="inline" overflow="linebreak"><mi>s</mi><mo>=</mo><mfenced open="{" close="}" separators=",,,"><msub><mi>s</mi><mn>1</mn></msub><mrow><mspace width=".15em"></mspace><msub><mi>s</mi><mn>2</mn></msub></mrow><mo>…</mo><mrow><mspace width=".15em"></mspace><msub><mi>s</mi><mi>n</mi></msub></mrow></mfenced></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M4" display="inline" overflow="linebreak"><msub><mi>s</mi><mi>i</mi></msub><mo>∈</mo><msub><mi mathvariant="script">V</mi><mn>1</mn></msub></math></span> represents the 1D residue token at the <em>i</em>th site. Building upon the concept of Foldseek [<a href="#B20" class="usa-link" aria-describedby="B20">20</a>], the newly built structure alphabet <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M5" display="inline" overflow="linebreak"><msub><mi mathvariant="script">V</mi><mn>2</mn></msub></math></span> utilizes 20 distinct 3Di tokens to accomplish the generation of pseudo structural sequences (i.e., 3Di sequences). A protein or antibody structure can be denoted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M6" display="inline" overflow="linebreak"><mi>a</mi><mo>=</mo><mfenced open="{" close="}" separators=",,,"><msub><mi>a</mi><mn>1</mn></msub><mrow><mspace width=".15em"></mspace><msub><mi>a</mi><mn>2</mn></msub></mrow><mo>…</mo><mrow><mspace width=".15em"></mspace><msub><mi>a</mi><mi>m</mi></msub></mrow></mfenced></math></span>, with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M7" display="inline" overflow="linebreak"><msub><mi>a</mi><mi>j</mi></msub><mo>∈</mo><msub><mi mathvariant="script">V</mi><mn>2</mn></msub></math></span> indicating the 3Di token at the <em>j</em>th site.</p>
<p>Building on the multi-level vocabulary, we obtain 1D and 3Di sequences and feed them into the model alternately. During pre-training stage I, we train S<sup>2</sup>ALM using the BERT-style masked language modeling (MLM) objective [<a href="#B40" class="usa-link" aria-describedby="B40">40</a>] to integratively learn from the 1D and 3Di sequences, enabling support for both sequence-level and structure-level tasks:</p>
<table class="disp-formula p" id="EQ1"><tr>
<td class="formula"><math id="M8" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="script">L</mi><mrow><mn>1</mn><mi mathvariant="normal">D</mi><mo>−</mo><mi>MLM</mi></mrow></msub><mo>=</mo><mi mathvariant="double-struck">E</mi><mfenced open="[" close="]"><mrow><munder><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>∈</mo><mi mathvariant="script">M</mi></mrow></munder><mo>−</mo><mtext>log</mtext><mspace width=".2em"></mspace><mi>p</mi><mfenced open="(" close=")" separators="|"><msub><mi>s</mi><mi>i</mi></msub><msub><mi mathvariant="bold">s</mi><mrow><mo>/</mo><mi mathvariant="script">M</mi></mrow></msub></mfenced></mrow></mfenced><mo>,</mo></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<table class="disp-formula p" id="EQ2"><tr>
<td class="formula"><math id="M9" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="script">L</mi><mrow><mn>3</mn><mi>Di</mi><mo>−</mo><mi>MLM</mi></mrow></msub><mo>=</mo><mi mathvariant="double-struck">E</mi><mfenced open="[" close="]"><mrow><munder><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>∈</mo><mi mathvariant="script">M</mi></mrow></munder><mo>−</mo><mtext>log</mtext><mspace width=".2em"></mspace><mi>p</mi><mfenced open="(" close=")" separators="|"><msub><mi>a</mi><mi>i</mi></msub><msub><mi mathvariant="bold">a</mi><mrow><mo>/</mo><mi mathvariant="script">M</mi></mrow></msub></mfenced></mrow></mfenced><mo>,</mo></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M10" display="inline" overflow="linebreak"><mi mathvariant="script">M</mi></math></span> is randomly masked positions and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M11" display="inline" overflow="linebreak"><msub><mi mathvariant="bold">s</mi><mrow><mo>/</mo><mi mathvariant="script">M</mi></mrow></msub><mo>,</mo><msub><mi mathvariant="bold">a</mi><mrow><mo>/</mo><mi mathvariant="script">M</mi></mrow></msub></math></span> denote the masked sequence where all masked positions have been replaced by the [MASK] token.</p>
<p>Specifically, for each 1D or 3Di sequence, 15% of the corresponding tokens are randomly masked and then predicted based on the remaining contextualized representation. Consequently, the loss function <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M12" display="inline" overflow="linebreak"><msub><mi mathvariant="script">L</mi><mi mathvariant="normal">I</mi></msub></math></span> in pre-training stage I is formulated as follows, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M13" display="inline" overflow="linebreak"><msub><mi mathvariant="script">L</mi><mrow><mn>1</mn><mi mathvariant="normal">D</mi><mo>−</mo><mi>MLM</mi></mrow></msub></math></span> stands for 1D sequences and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M14" display="inline" overflow="linebreak"><msub><mi mathvariant="script">L</mi><mrow><mn>3</mn><mi>Di</mi><mo>−</mo><mi>MLM</mi></mrow></msub></math></span> stands for 3Di sequences.</p>
<table class="disp-formula p" id="EQ3"><tr>
<td class="formula"><math id="M15" display="block" overflow="linebreak"><msub><mi mathvariant="script">L</mi><mi mathvariant="normal">I</mi></msub><mo>=</mo><msub><mi mathvariant="script">L</mi><mrow><mn>1</mn><mi mathvariant="normal">D</mi><mo>−</mo><mi>MLM</mi></mrow></msub><mo>+</mo><msub><mi mathvariant="script">L</mi><mrow><mn>3</mn><mi>Di</mi><mo>−</mo><mi>MLM</mi></mrow></msub><mo>.</mo></math></td>
<td class="label">(3)</td>
</tr></table>
<p>Pre-training stage I endows the model with the capability to simultaneously identify both 1D and 3Di sequences. Furthermore, the efficient utilization of protein data in pre-training stage I effectively alleviates issues arising from insufficient antibody structure data. Due to the powerful generalization ability of large language models, the global structural constraints from proteins learned in stage I set the foundation for antibody-specific learning of sub-domain local constraints in stage II.</p></section><section id="sec24"><h4 class="pmc_sec_title">Stage II: Antibody-specific multi-level learning</h4>
<p>After pre-training stage I, S<sup>2</sup>ALM has thoroughly comprehended 1D and 3Di sequences across the general protein domain. Subsequently in pre-training stage II, we can primarily focus on multi-level representation learning in the target antibody sub-domain. To better absorb comprehensive knowledge of antibody sequences and structures, exploring new pre-training mechanisms is worthwhile. Two multi-level learning objectives are introduced to inject different granularities of antibody-specific sequential and structural information into an ALM: sequence-structure matching (SSM) and cross-level reconstruction (CLR). The customized learning objectives facilitate the extraction of complex patterns and interdependency inherent in antibody sequences and structures.</p>
<p>SSM captures the coarse-grained alignment between antibody sequential and structural information. It is a binary classification task to predict whether a sequence-structure pair is matching or unmatching, as depicted in Fig. <a href="#F1" class="usa-link">1</a>C. The model extracts representations of the corrupted antibody 1D and 3Di sequences and then exploits a linear layer to make classification on their matching relationships. SSM optimizes the following loss function:</p>
<table class="disp-formula p" id="EQ4"><tr>
<td class="formula"><math id="M16" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="script">L</mi><mi>SSM</mi></msub><mo>=</mo><munder><mo movablelimits="false">∑</mo><mrow><mfenced open="(" close=")" separators=","><mover accent="true"><mi>s</mi><mo stretchy="false">˜</mo></mover><mrow><mspace width=".15em"></mspace><mover accent="true"><mi>a</mi><mo stretchy="false">˜</mo></mover></mrow></mfenced><mo>∈</mo><mfenced open="(" close=")"><mrow><mover accent="true"><mrow><mi>S</mi><mo>,</mo><mspace width=".10em"></mspace><mi>A</mi></mrow><mo stretchy="false">˜</mo></mover></mrow></mfenced></mrow></munder><mi>H</mi><mfenced open="[" close="]" separators=",,"><mrow><mi>y</mi><mfenced open="(" close=")" separators=","><mover accent="true"><mi>s</mi><mo stretchy="false">˜</mo></mover><mrow><mspace width=".15em"></mspace><mover accent="true"><mi>a</mi><mo stretchy="false">˜</mo></mover></mrow></mfenced></mrow><mrow><mi>p</mi><mo stretchy="false">(</mo><msub><mi mathvariant="script">E</mi><mi>θ</mi></msub><mo stretchy="false">(</mo><mover accent="true"><mi>s</mi><mo stretchy="false">˜</mo></mover></mrow><mrow><mspace width=".15em"></mspace><mover accent="true"><mi>a</mi><mo stretchy="false">˜</mo></mover><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow></mfenced><mo>,</mo></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M17" display="inline" overflow="linebreak"><mfenced open="(" close=")"><mrow><mover accent="true"><mrow><mi>S</mi><mo>,</mo><mspace width=".10em"></mspace><mi>A</mi></mrow><mo stretchy="false">˜</mo></mover></mrow></mfenced></math></span> is the corrupted antibody dataset containing matched and unmatched sequence-structure pairs and <em>H</em> denotes the cross entropy loss function. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M18" display="inline" overflow="linebreak"><msub><mi mathvariant="script">E</mi><mi>θ</mi></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M19" display="inline" overflow="linebreak"><mi>y</mi></math></span> represent parameterized encoder and matching ground truth, respectively.</p>
<p>CLR focuses on improving fine-grained understanding in antibody sequence-structure pre-training, which differs in reconstruction conditions from MLM in the “Stage I: General sequence-structure learning” section. Concretely, the paired antibody 1D and 3Di sequences are separately encoded by the model. Then, we randomly replace 15% of tokens with the [MASK] token in the 1D or 3Di sequence, while keeping the other level sequence unmasked. Eventually, as illustrated in Fig. <a href="#F1" class="usa-link">1</a>D, hybrid information from both 1D and 3Di sequences serves as the CLR condition to make token-level predictions through a linear layer. CLR encourages the model to recover the corrupted 1D or 3Di sequences based on information from both levels, explicitly capturing the interrelated mechanism between antibody sequences and structures:</p>
<table class="disp-formula p" id="EQ5"><tr>
<td class="formula"><math id="M20" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="script">L</mi><mrow><mn>1</mn><mi mathvariant="normal">D</mi><mo>−</mo><mi>CLR</mi></mrow></msub><mo>=</mo><mi mathvariant="double-struck">E</mi><mfenced open="[" close="]"><mrow><munder><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>∈</mo><mi mathvariant="script">M</mi></mrow></munder><mo>−</mo><mtext>log</mtext><mspace width=".2em"></mspace><mi>p</mi><mfenced open="(" close=")" separators="|;"><msub><mi>s</mi><mi>i</mi></msub><msub><mi mathvariant="bold">s</mi><mrow><mo>/</mo><mi mathvariant="script">M</mi></mrow></msub><mi mathvariant="bold">a</mi></mfenced></mrow></mfenced><mo>,</mo></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<table class="disp-formula p" id="EQ6"><tr>
<td class="formula"><math id="M21" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="script">L</mi><mrow><mn>3</mn><mi>Di</mi><mo>−</mo><mi>CLR</mi></mrow></msub><mo>=</mo><mi mathvariant="double-struck">E</mi><mfenced open="[" close="]"><mrow><munder><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>∈</mo><mi mathvariant="script">M</mi></mrow></munder><mo>−</mo><mtext>log</mtext><mspace width=".2em"></mspace><mi>p</mi><mfenced open="(" close=")" separators="|;"><msub><mi>a</mi><mi>i</mi></msub><mi mathvariant="bold">s</mi><msub><mi mathvariant="bold">a</mi><mrow><mo>/</mo><mi mathvariant="script">M</mi></mrow></msub></mfenced></mrow></mfenced><mo>,</mo></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M22" display="inline" overflow="linebreak"><mi mathvariant="script">M</mi></math></span> is randomly masked positions in 1D or 3Di sequences. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M23" display="inline" overflow="linebreak"><msub><mi mathvariant="script">L</mi><mrow><mn>1</mn><mi mathvariant="normal">D</mi><mo>−</mo><mi>CLR</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M24" display="inline" overflow="linebreak"><msub><mi mathvariant="script">L</mi><mrow><mn>3</mn><mi>Di</mi><mo>−</mo><mi>CLR</mi></mrow></msub></math></span> subsequently measure the reconstruction costs of 1D and 3Di sequences. Therefore, the ultimate loss function <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M25" display="inline" overflow="linebreak"><msub><mi mathvariant="script">L</mi><mi>II</mi></msub></math></span> in pre-training stage II is formulated as:</p>
<table class="disp-formula p" id="EQ7"><tr>
<td class="formula"><math id="M26" display="block" overflow="linebreak"><mrow><msub><mi mathvariant="script">L</mi><mi>II</mi></msub><mo>=</mo><msub><mi mathvariant="script">L</mi><mi>SSM</mi></msub><mo>+</mo><msub><mi mathvariant="script">L</mi><mrow><mn>1</mn><mi mathvariant="normal">D</mi><mo>−</mo><mi>CLR</mi></mrow></msub><mo>+</mo><msub><mi mathvariant="script">L</mi><mrow><mn>3</mn><mi>Di</mi><mo>−</mo><mi>CLR</mi></mrow></msub></mrow></math></td>
<td class="label">(7)</td>
</tr></table>
<p>With the groundwork laid in pre-training stage I, the tailored pre-training objectives in stage II facilitate S<sup>2</sup>ALM to effectively integrate antibody sequential and structural information, modeling comprehensive antibody representations. Overall, within this hierarchical pre-training paradigm, these 2 pre-training stages are complementary and indispensable to each other. Their synergy makes S<sup>2</sup>ALM a powerful antibody foundation model, further fostering holistic antibody understanding and generation. (See thorough pre-training details in Note <a href="#supplementary-material-1" class="usa-link">S3</a>.)</p></section></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We sincerely thank the College of Computer Science and Technology, Zhejiang University and the School of Medicine, Zhejiang University for their support.</p>
<p><strong>Funding:</strong> This research was partially supported by the National Natural Science Foundation of China under grant no. 82202984, the Zhejiang Key R&amp;D Program of China under grant no. 2023C03053, and the Zhejiang Key Laboratory of Medical Imaging Artificial Intelligence.</p>
<p><strong>Author contributions:</strong> M.Y., H.Z., and J.C. conceived the idea. M.Y., H.Z., Jialu Wu, and Y. Zhu developed the methodology and designed the experiments. M.Y., Y. Zhan, and Z.K. collected the data. M.Y., Y. Zhu, and Y. Zhan conducted the experiments. Y. Zhan, Z.K., and H.X. conducted the visualization. M.Y., H.Z., Jialu Wu, and J.C. wrote the manuscript. C.-Y.H., T.H., and Jian Wu supervised the study. All authors revised and approved the final version of the manuscript.</p>
<p><strong>Competing interests:</strong> The authors declare that they have no competing interests.</p></section><section id="sec25"><h2 class="pmc_sec_title">Data Availability</h2>
<p>The source code and data are freely available at <a href="https://github.com/KDurant-123/S2ALM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/KDurant-123/S2ALM</a>.</p></section><section id="supplementary-material-1"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="supp-1"><div class="caption p">
<span>Supplementary 1</span><p>Notes S1 to S6</p>
<p>Tables S1 and S2</p>
<p>References [<a href="#B61" class="usa-link" aria-describedby="B61">61</a>–<a href="#B70" class="usa-link" aria-describedby="B70">70</a>]</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12364524/bin/research.0721.f1.pdf" data-ga-action="click_feat_suppl" class="usa-link">research.0721.f1.pdf</a><sup> (3.9MB, pdf) </sup>
</div></div></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>Taylor PC, Adams AC, Hufford MM, Torre I, Winthrop K, Gottlieb RL. 
Neutralizing monoclonal antibodies for treatment of COVID-19. Nat Rev Immunol. 2021;21(6):382–393.
</cite> [<a href="https://doi.org/10.1038/s41577-021-00542-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8054133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33875867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&amp;title=Neutralizing%20monoclonal%20antibodies%20for%20treatment%20of%20COVID-19&amp;volume=21&amp;issue=6&amp;publication_year=2021&amp;pages=382-393&amp;pmid=33875867&amp;doi=10.1038/s41577-021-00542-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>Ma X, Liang J, Zhu G, Bhoria P, Shoara AA, MacKeigan DT, Khoury CJ, Slavkovic S, Lin L, Karakas D, et al. 
SARS-CoV-2 RBD and its variants can induce platelet activation and clearance: Implications for antibody therapy and vaccinations against COVID-19. Research. 2023;6:0124.
</cite> [<a href="https://doi.org/10.34133/research.0124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10202384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37223472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research&amp;title=SARS-CoV-2%20RBD%20and%20its%20variants%20can%20induce%20platelet%20activation%20and%20clearance:%20Implications%20for%20antibody%20therapy%20and%20vaccinations%20against%20COVID-19&amp;volume=6&amp;publication_year=2023&amp;pages=0124&amp;pmid=37223472&amp;doi=10.34133/research.0124&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>Li P, Yang Y, Wang Y, Zheng J, Chen F, Jiang M, Chou CK, Cong W, Li Z, Chen X. 
Anti-TNFR2 antibody-conjugated PLGA nanoparticles for targeted delivery of adriamycin in mouse colon cancer. Research. 2024;7:0444.
</cite> [<a href="https://doi.org/10.34133/research.0444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11377996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39247806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research&amp;title=Anti-TNFR2%20antibody-conjugated%20PLGA%20nanoparticles%20for%20targeted%20delivery%20of%20adriamycin%20in%20mouse%20colon%20cancer&amp;volume=7&amp;publication_year=2024&amp;pages=0444&amp;pmid=39247806&amp;doi=10.34133/research.0444&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>Yin M, Zhou H, Zhu Y, Wu J, Wu W, Li M, Fu K, Wang Z, Hsieh C-Y, Hou T,et al. Synergy of GFlowNet and Protein Language Model Makes a Diverse Antibody Designer. Paper presented at: Proceedings of the AAAI Conference on Artificial Intelligence; 2025; Philadelphia, PA, USA.</cite>
</li>
<li id="B5">
<span class="label">5.</span><cite>Mullard A. 
FDA approves 100th monoclonal antibody product. Nat Rev Drug Discov. 2021;20(7):491–495.
</cite> [<a href="https://doi.org/10.1038/d41573-021-00079-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33953368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&amp;title=FDA%20approves%20100th%20monoclonal%20antibody%20product&amp;volume=20&amp;issue=7&amp;publication_year=2021&amp;pages=491-495&amp;pmid=33953368&amp;doi=10.1038/d41573-021-00079-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>Shuai RW, Ruffolo JA, Gray JJ. 
IgLM: Infilling language modeling for antibody sequence design. Cell Syst. 2023;14(11):979–989.e4.
</cite> [<a href="https://doi.org/10.1016/j.cels.2023.10.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11018345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37909045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=IgLM:%20Infilling%20language%20modeling%20for%20antibody%20sequence%20design&amp;volume=14&amp;issue=11&amp;publication_year=2023&amp;pages=979-989.e4&amp;pmid=37909045&amp;doi=10.1016/j.cels.2023.10.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>He H, He B, Guan L, Zhao Y, Jiang F, Chen G, Zhu Q, Chen CYC, Li T, Yao J. 
De novo generation of SARS-CoV-2 antibody CDRH3 with a pre-trained generative large language model. Nat Commun. 2024;15(1):6867.
</cite> [<a href="https://doi.org/10.1038/s41467-024-50903-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11316817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39127753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=De%20novo%20generation%20of%20SARS-CoV-2%20antibody%20CDRH3%20with%20a%20pre-trained%20generative%20large%20language%20model&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=6867&amp;pmid=39127753&amp;doi=10.1038/s41467-024-50903-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>Luo Y, Liu XY, Yang K, Huang K, Hong M, Zhang J, Wu Y, Nie Z. 
Toward unified AI drug discovery with multimodal knowledge. Health Data Sci. 2024;4:0113.
</cite> [<a href="https://doi.org/10.34133/hds.0113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10886071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38486623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Health%20Data%20Sci&amp;title=Toward%20unified%20AI%20drug%20discovery%20with%20multimodal%20knowledge&amp;volume=4&amp;publication_year=2024&amp;pages=0113&amp;pmid=38486623&amp;doi=10.34133/hds.0113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>Jing H, Gao Z, Xu S, Shen T, Peng Z, He S, You T, Ye S, Lin W, Sun S. 
Accurate prediction of antibody function and structure using bio-inspired antibody language model. Brief Bioinform. 2024;25(4):bbae245.
</cite> [<a href="https://doi.org/10.1093/bib/bbae245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11128484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38797969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform&amp;title=Accurate%20prediction%20of%20antibody%20function%20and%20structure%20using%20bio-inspired%20antibody%20language%20model&amp;volume=25&amp;issue=4&amp;publication_year=2024&amp;pages=bbae245&amp;pmid=38797969&amp;doi=10.1093/bib/bbae245&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>Nguyen E, Poli M, Durrant MG, Kang B, Katrekar D, Li DB, Bartie LJ, Thomas AW, King SH, Brixi G, et al. 
Sequence modeling and design from molecular to genome scale with Evo. Science. 2024;386(6723):eado9336.
</cite> [<a href="https://doi.org/10.1126/science.ado9336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12057570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39541441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Sequence%20modeling%20and%20design%20from%20molecular%20to%20genome%20scale%20with%20Evo&amp;volume=386&amp;issue=6723&amp;publication_year=2024&amp;pages=eado9336&amp;pmid=39541441&amp;doi=10.1126/science.ado9336&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>Mou M, Pan Z, Zhou Z, Zheng L, Zhang H, Shi S, Li F, Sun X, Zhu F. 
A transformer-based ensemble framework for the prediction of protein-protein interaction sites. Research. 2023;6:0240.
</cite> [<a href="https://doi.org/10.34133/research.0240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10528219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37771850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research&amp;title=A%20transformer-based%20ensemble%20framework%20for%20the%20prediction%20of%20protein-protein%20interaction%20sites&amp;volume=6&amp;publication_year=2023&amp;pages=0240&amp;pmid=37771850&amp;doi=10.34133/research.0240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>Zhang Y, Guan J, Li C, Wang Z, Deng Z, Gasser RB, Song J, Ou HY. 
DeepSecE: A deep-learning-based framework for multiclass prediction of secreted proteins in gram-negative bacteria. Research. 2023;6:0258.
</cite> [<a href="https://doi.org/10.34133/research.0258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10599158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37886621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research&amp;title=DeepSecE:%20A%20deep-learning-based%20framework%20for%20multiclass%20prediction%20of%20secreted%20proteins%20in%20gram-negative%20bacteria&amp;volume=6&amp;publication_year=2023&amp;pages=0258&amp;pmid=37886621&amp;doi=10.34133/research.0258&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>Zhou H, Yin M, Wu W, Li M, Fu K, Chen J, Wu J,Wang Z. ProtCLIP: Function-informed protein multi-modal learning. Paper presented at: Proceedings of the AAAI Conference on Artificial Intelligence; 2025; Philadelphia, PA, USA.</cite>
</li>
<li id="B14">
<span class="label">14.</span><cite>Liu Z, Qian W, Cai W, Song W, Wang W, Maharjan DT, Cheng W, Chen J, Wang H, Xu D, et al. 
Inferring the effects of protein variants on protein-protein interactions with interpretable transformer representations. Research. 2023;6:0219.
</cite> [<a href="https://doi.org/10.34133/research.0219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10494974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37701056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research&amp;title=Inferring%20the%20effects%20of%20protein%20variants%20on%20protein-protein%20interactions%20with%20interpretable%20transformer%20representations&amp;volume=6&amp;publication_year=2023&amp;pages=0219&amp;pmid=37701056&amp;doi=10.34133/research.0219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>Leem J, Mitchell LS, Farmery JH, Barton J, Galson JD. 
Deciphering the language of antibodies using self-supervised learning. Patterns. 2022;3(7):
Article 100513.
</cite> [<a href="https://doi.org/10.1016/j.patter.2022.100513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9278498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35845836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Patterns&amp;title=Deciphering%20the%20language%20of%20antibodies%20using%20self-supervised%20learning&amp;volume=3&amp;issue=7&amp;publication_year=2022&amp;pages=Article%20100513&amp;pmid=35845836&amp;doi=10.1016/j.patter.2022.100513&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>Ruffolo JA, Gray JJ, Sulam J. Deciphering antibody affinity maturation with language models and weakly supervised learning. arXiv. 2021. 10.48550/arXiv.2112.07782</cite> [<a href="https://doi.org/10.48550/arXiv.2112.07782" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>Olsen TH, Moal IH, Deane CM. 
AbLang: An antibody language model for completing antibody sequences. Bioinform Adv. 2022;2(1):vbac046.
</cite> [<a href="https://doi.org/10.1093/bioadv/vbac046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9710568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36699403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinform%20Adv&amp;title=AbLang:%20An%20antibody%20language%20model%20for%20completing%20antibody%20sequences&amp;volume=2&amp;issue=1&amp;publication_year=2022&amp;pages=vbac046&amp;pmid=36699403&amp;doi=10.1093/bioadv/vbac046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>Wang D, Ye F, Zhou H. On pre-trained language models for antibody. Paper presented at: Proceedings of the 11th International Conference on Learning Representations; 2023; Kigali, Rwanda.</cite>
</li>
<li id="B19">
<span class="label">19.</span><cite>Lin Z, Akin H, Rao R, Hie B, Zhu Z, Lu W, Smetanin N, Verkuil R, Kabeli O, Shmueli Y, et al. 
Evolutionary-scale prediction of atomic-level protein structure with a language model. Science. 2023;379(6637):1123–1130.
</cite> [<a href="https://doi.org/10.1126/science.ade2574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36927031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Evolutionary-scale%20prediction%20of%20atomic-level%20protein%20structure%20with%20a%20language%20model&amp;volume=379&amp;issue=6637&amp;publication_year=2023&amp;pages=1123-1130&amp;pmid=36927031&amp;doi=10.1126/science.ade2574&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20">
<span class="label">20.</span><cite>Kempen M, Kim SS, Tumescheit C, Mirdita M, Lee J, Gilchrist CLM, Söding J, Steinegger M. 
Fast and accurate protein structure search with Foldseek. Nat Biotechnol. 2024;42(2):243–246.
</cite> [<a href="https://doi.org/10.1038/s41587-023-01773-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10869269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37156916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Fast%20and%20accurate%20protein%20structure%20search%20with%20Foldseek&amp;volume=42&amp;issue=2&amp;publication_year=2024&amp;pages=243-246&amp;pmid=37156916&amp;doi=10.1038/s41587-023-01773-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B21">
<span class="label">21.</span><cite>Van der Maaten L, Hinton G. 
Visualizing data using t-SNE. J Mach Learn Res. 2008;9(86):2579–2605.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mach%20Learn%20Res&amp;title=Visualizing%20data%20using%20t-SNE&amp;volume=9&amp;issue=86&amp;publication_year=2008&amp;pages=2579-2605&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B22">
<span class="label">22.</span><cite>Melnyk I, Chenthamarakshan V, Chen PY, Das P, Dhurandhar A, Padhi I, Das D. Reprogramming pretrained language models for antibody sequence infilling. Paper presented at: Proceedings of the 40th International Conference on Machine Learning; 2023; Honolulu, HI.</cite>
</li>
<li id="B23">
<span class="label">23.</span><cite>Olsen TH, Boyles F, Deane CM. 
Observed antibody space: A diverse database of cleaned, annotated, and translated unpaired and paired antibody sequences. Protein Sci. 2022;31(1):141–146.
</cite> [<a href="https://doi.org/10.1002/pro.4205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8740823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34655133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Observed%20antibody%20space:%20A%20diverse%20database%20of%20cleaned,%20annotated,%20and%20translated%20unpaired%20and%20paired%20antibody%20sequences&amp;volume=31&amp;issue=1&amp;publication_year=2022&amp;pages=141-146&amp;pmid=34655133&amp;doi=10.1002/pro.4205&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24">
<span class="label">24.</span><cite>Fagarasan S, Kinoshita K, Muramatsu M, Ikuta K, Honjo T. 
In situ class switching and differentiation to IgA-producing cells in the gut lamina propria. Nature. 2001;413(6856):639–643.
</cite> [<a href="https://doi.org/10.1038/35098100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11675788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=In%20situ%20class%20switching%20and%20differentiation%20to%20IgA-producing%20cells%20in%20the%20gut%20lamina%20propria&amp;volume=413&amp;issue=6856&amp;publication_year=2001&amp;pages=639-643&amp;pmid=11675788&amp;doi=10.1038/35098100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25">
<span class="label">25.</span><cite>Abramson J, Adler J, Dunger J, Evans R, Green T, Pritzel A, Ronneberger O, Willmore L, Ballard AJ, Bambrick J, et al. 
Accurate structure prediction of biomolecular interactions with AlphaFold 3. Nature. 2024;630(8016):493–500.
</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20AlphaFold%203&amp;volume=630&amp;issue=8016&amp;publication_year=2024&amp;pages=493-500&amp;pmid=38718835&amp;doi=10.1038/s41586-024-07487-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26">
<span class="label">26.</span><cite>Mason DM, Friedensohn S, Weber CR, Jordi C, Wagner B, Meng SM, Ehling RA, Bonati L, Dahinden J, Gainza P, et al. 
Optimization of therapeutic antibodies by predicting antigen specificity from antibody sequence via deep learning. Nat Biomed Eng. 2021;5(6):600–612.
</cite> [<a href="https://doi.org/10.1038/s41551-021-00699-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33859386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biomed%20Eng&amp;title=Optimization%20of%20therapeutic%20antibodies%20by%20predicting%20antigen%20specificity%20from%20antibody%20sequence%20via%20deep%20learning&amp;volume=5&amp;issue=6&amp;publication_year=2021&amp;pages=600-612&amp;pmid=33859386&amp;doi=10.1038/s41551-021-00699-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27">
<span class="label">27.</span><cite>Rives A, Meier J, Sercu T, Goyal S, Lin Z, Liu J, Guo D, Ott M, Zitnick CL, Ma J, et al. 
Biological structure and function emerge from scaling unsupervised learning to 250 million protein sequences. Proc Natl Acad Sci USA. 2021;118(15):
Article e2016239118.
</cite> [<a href="https://doi.org/10.1073/pnas.2016239118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8053943/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33876751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Biological%20structure%20and%20function%20emerge%20from%20scaling%20unsupervised%20learning%20to%20250%20million%20protein%20sequences&amp;volume=118&amp;issue=15&amp;publication_year=2021&amp;pages=Article%20e2016239118&amp;pmid=33876751&amp;doi=10.1073/pnas.2016239118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28">
<span class="label">28.</span><cite>Rao RM, Liu J, Verkuil R, Meier J, Canny J, Abbeel Sercu T, Rives A. MSA transformer. Paper presented at: Proceedings of the 38th International Conference on Machine Learning; 2021; virtual.</cite>
</li>
<li id="B29">
<span class="label">29.</span><cite>Ghraichy M, Niederhäusern V, Kovaltsuk A, Galson JD, Deane CM, Trück J. 
Different B cell subpopulations show distinct patterns in their IgH repertoire metrics. eLife. 2021;10:
Article e73111.
</cite> [<a href="https://doi.org/10.7554/eLife.73111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8560093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34661527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Different%20B%20cell%20subpopulations%20show%20distinct%20patterns%20in%20their%20IgH%20repertoire%20metrics&amp;volume=10&amp;publication_year=2021&amp;pages=Article%20e73111&amp;pmid=34661527&amp;doi=10.7554/eLife.73111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B30">
<span class="label">30.</span><cite>Meffre E, Casellas R, Nussenzweig MC. 
Antibody regulation of B cell development. Nat Immunol. 2000;1(5):379–385.
</cite> [<a href="https://doi.org/10.1038/80816" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11062496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&amp;title=Antibody%20regulation%20of%20B%20cell%20development&amp;volume=1&amp;issue=5&amp;publication_year=2000&amp;pages=379-385&amp;pmid=11062496&amp;doi=10.1038/80816&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31">
<span class="label">31.</span><cite>Mroczek ES, Ippolito GC, Rogosch T, Hoi KH, Hwangpo TA, Brand MG, Zhuang Y, Liu CR, Schneider DA, Zemlin M, et al. 
Differences in the composition of the human antibody repertoire by b cell subsets in the blood. Front Immunol. 2014;5:96.
</cite> [<a href="https://doi.org/10.3389/fimmu.2014.00096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3958703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24678310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=Differences%20in%20the%20composition%20of%20the%20human%20antibody%20repertoire%20by%20b%20cell%20subsets%20in%20the%20blood&amp;volume=5&amp;publication_year=2014&amp;pages=96&amp;pmid=24678310&amp;doi=10.3389/fimmu.2014.00096&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32">
<span class="label">32.</span><cite>Bashford-Rogers R, Bergamaschi L, McKinney E, Pombal DC, Mescia F, Lee JC, Thomas DC, Flint SM, Kellam P, Jayne DRW, et al. 
Analysis of the B cell receptor repertoire in six immune-mediated diseases. Nature. 2019;574(7776):122–126.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1595-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6795535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31554970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Analysis%20of%20the%20B%20cell%20receptor%20repertoire%20in%20six%20immune-mediated%20diseases&amp;volume=574&amp;issue=7776&amp;publication_year=2019&amp;pages=122-126&amp;pmid=31554970&amp;doi=10.1038/s41586-019-1595-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33">
<span class="label">33.</span><cite>Kunik V, Ashkenazi S, Ofran Y. 
Paratome: An online tool for systematic identification of antigen-binding regions in antibodies based on sequence or structure. Nucleic Acids Res. 2012;40:W521–W524.
</cite> [<a href="https://doi.org/10.1093/nar/gks480" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3394289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22675071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Paratome:%20An%20online%20tool%20for%20systematic%20identification%20of%20antigen-binding%20regions%20in%20antibodies%20based%20on%20sequence%20or%20structure&amp;volume=40&amp;publication_year=2012&amp;pages=W521-W524&amp;pmid=22675071&amp;doi=10.1093/nar/gks480&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34">
<span class="label">34.</span><cite>Prihoda D, Maamary J, Waight A, Juan V, Fayadat-Dilman L, Svozil D, Bitton DA. 
BioPhi: A platform for antibody design, humanization, and humanness evaluation based on natural antibody repertoires and deep learning. MAbs. 2022;14(1):2020203.
</cite> [<a href="https://doi.org/10.1080/19420862.2021.2020203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8837241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35133949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MAbs&amp;title=BioPhi:%20A%20platform%20for%20antibody%20design,%20humanization,%20and%20humanness%20evaluation%20based%20on%20natural%20antibody%20repertoires%20and%20deep%20learning&amp;volume=14&amp;issue=1&amp;publication_year=2022&amp;pages=2020203&amp;pmid=35133949&amp;doi=10.1080/19420862.2021.2020203&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35">
<span class="label">35.</span><cite>Elnaggar A, Heinzinger M, Dallago C, Rehawi G, Wang Y, Jones L, Gibbs T, Feher T, Angerer C, Steinegger M, et al. 
ProtTrans: Toward understanding the language of life through self-supervised learning. IEEE Trans Pattern Anal Mach Intell. 2022;44(10):7112–7127.
</cite> [<a href="https://doi.org/10.1109/TPAMI.2021.3095381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34232869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Trans%20Pattern%20Anal%20Mach%20Intell&amp;title=ProtTrans:%20Toward%20understanding%20the%20language%20of%20life%20through%20self-supervised%20learning&amp;volume=44&amp;issue=10&amp;publication_year=2022&amp;pages=7112-7127&amp;pmid=34232869&amp;doi=10.1109/TPAMI.2021.3095381&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36">
<span class="label">36.</span><cite>Li L, Gupta E, Spaeth J, Shing L, Bepler T, Caceres RS. Antibody representation learning for drug discovery. arXiv. 2022. 10.48550/arXiv.2210.02881</cite> [<a href="https://doi.org/10.48550/arXiv.2210.02881" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="B37">
<span class="label">37.</span><cite>Kenlay H, Dreyer FA, Kovaltsuk A, Miketa D, Pires D, Deane CM. Large scale paired antibody language models. arXiv. 2024. 10.48550/arXiv.2403.17889</cite> [<a href="https://doi.org/10.48550/arXiv.2403.17889" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11654935/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39642174/" class="usa-link">PubMed</a>]</li>
<li id="B38">
<span class="label">38.</span><cite>Engelhart E, Emerson R, Shing L, Lennartz C, Guion D, Kelley M, Lin C, Lopez R, Younger D, Walsh ME. 
A dataset comprised of binding interactions for 104,972 antibodies against a SARS-CoV-2 peptide. Sci Data. 2022;9(1):653.
</cite> [<a href="https://doi.org/10.1038/s41597-022-01779-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9606274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36289234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Data&amp;title=A%20dataset%20comprised%20of%20binding%20interactions%20for%20104,972%20antibodies%20against%20a%20SARS-CoV-2%20peptide&amp;volume=9&amp;issue=1&amp;publication_year=2022&amp;pages=653&amp;pmid=36289234&amp;doi=10.1038/s41597-022-01779-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B39">
<span class="label">39.</span><cite>Chen B, Cheng X, Geng Y, Li S, Zeng X, Wang B, Gong J, Liu C, Zeng A, Dong Y, et al. xTrimoPGLM: Unified 100B-scale pre-trained transformer for deciphering the language of protein. bioRxiv. 2024. 10.1101/2023.07.05.547496</cite> [<a href="https://doi.org/10.1101/2023.07.05.547496" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40181110/" class="usa-link">PubMed</a>]</li>
<li id="B40">
<span class="label">40.</span><cite>Devlin J, Chang MW, Lee K, Toutanova K. BERT: Pre-training of deep bidirectional transformers for language understanding. arXiv. 2019. 10.48550/arXiv.1810.04805</cite> [<a href="https://doi.org/10.48550/arXiv.1810.04805" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="B41">
<span class="label">41.</span><cite>Liu G, Zeng H, Mueller J, Carter B, Wang Z, Schilz J, Horny G, Birnbaum ME, Ewert S, Gifford DK. 
Antibody complementarity determining region design using high-capacity machine learning. Bioinformatics. 2020;36(7):2126–2133.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz895" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7141872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31778140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Antibody%20complementarity%20determining%20region%20design%20using%20high-capacity%20machine%20learning&amp;volume=36&amp;issue=7&amp;publication_year=2020&amp;pages=2126-2133&amp;pmid=31778140&amp;doi=10.1093/bioinformatics/btz895&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B42">
<span class="label">42.</span><cite>Singh R, Im C, Qiu Y, Mackness B, Gupta A, Sorenson T, Sledzieski S, Erlach L, Wendt M, Nanfack YF, et al. Learning the language of antibody hypervariability. bioRxiv. 2023. 10.1101/2023.04.26.538476</cite> [<a href="https://doi.org/10.1101/2023.04.26.538476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11725859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39793083/" class="usa-link">PubMed</a>]</li>
<li id="B43">
<span class="label">43.</span><cite>Barton J, Galson JD, Leem J. Enhancing antibody language models with structural information. bioRxiv. 2024. 10.1101/2023.12.12.569610</cite> [<a href="https://doi.org/10.1101/2023.12.12.569610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="B44">
<span class="label">44.</span><cite>Jin W, Wohlwend J, Barzilay R, Jaakkola TS. Iterative refinement graph neural network for antibody sequence-structure co-design. Paper presented at: Proceedings of the 10th International Conference on Learning Representations; 2022; virtual.</cite>
</li>
<li id="B45">
<span class="label">45.</span><cite>Raybould MIJ, Kovaltsuk A, Marks C, Deane CM. 
CoV-AbDab: The coronavirus antibody database. Bioinformatics. 2021;37(5):734–735.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7558925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32805021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=CoV-AbDab:%20The%20coronavirus%20antibody%20database&amp;volume=37&amp;issue=5&amp;publication_year=2021&amp;pages=734-735&amp;pmid=32805021&amp;doi=10.1093/bioinformatics/btaa739&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B46">
<span class="label">46.</span><cite>Madani A, Krause B, Greene ER, Subramanian S, Mohr BP, Holton JM, Olmos JL Jr, Xiong C, Sun ZZ, Socher R, et al. 
Large language models generate functional protein sequences across diverse families. Nat Biotechnol. 2023;41(8):1099–1106.
</cite> [<a href="https://doi.org/10.1038/s41587-022-01618-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10400306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36702895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Large%20language%20models%20generate%20functional%20protein%20sequences%20across%20diverse%20families&amp;volume=41&amp;issue=8&amp;publication_year=2023&amp;pages=1099-1106&amp;pmid=36702895&amp;doi=10.1038/s41587-022-01618-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B47">
<span class="label">47.</span><cite>Adolf-Bryfogle J, Kalyuzhniy O, Kubitz M, Weitzner BD, Hu X, Adachi Y, Schief WR, Dunbrack RL. 
RosettaAntibodyDesign (RAbD): A general framework for computational antibody design. PLOS Comput Biol. 2018;14(4):
Article e1006112.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1006112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5942852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29702641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Comput%20Biol&amp;title=RosettaAntibodyDesign%20(RAbD):%20A%20general%20framework%20for%20computational%20antibody%20design&amp;volume=14&amp;issue=4&amp;publication_year=2018&amp;pages=Article%20e1006112&amp;pmid=29702641&amp;doi=10.1371/journal.pcbi.1006112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B48">
<span class="label">48.</span><cite>Nijkamp E, Ruffolo JA, Weinstein EN, Naik N, Madani A. 
ProGen2: Exploring the boundaries of protein language models. Cell Syst. 2023;14(11):968–978.e3.
</cite> [<a href="https://doi.org/10.1016/j.cels.2023.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37909046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=ProGen2:%20Exploring%20the%20boundaries%20of%20protein%20language%20models&amp;volume=14&amp;issue=11&amp;publication_year=2023&amp;pages=968-978.e3&amp;pmid=37909046&amp;doi=10.1016/j.cels.2023.10.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B49">
<span class="label">49.</span><cite>Su J, Han C, Zhou Y, Shan J, Zhou X, Yuan F. SaProt: Protein language modeling with structure-aware vocabulary. Paper presented at: Proceedings of the 12th International Conference on Learning Representations; 2024; Vienna, Austria.</cite>
</li>
<li id="B50">
<span class="label">50.</span><cite>Yin M, Zhou H, Zhu Y, Lin M, Wu Y, Wu J, Xu H, Hsieh CY, Hou T, Chen J, et al. Multi-modal CLIP-informed protein editing. arXiv. 2024. 10.48550/arXiv.2407.19296</cite> [<a href="https://doi.org/10.48550/arXiv.2407.19296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11658819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39703565/" class="usa-link">PubMed</a>]</li>
<li id="B51">
<span class="label">51.</span><cite>Zhu Y, Wu J, Li Q, Yan J, Yin M, Wu W, Li M, Ye J, Wang Z, Wu J. Bridge-IF: Learning inverse protein folding with Markov bridges. arXiv. 2024. 10.48550/arXiv.2411.02120</cite> [<a href="https://doi.org/10.48550/arXiv.2411.02120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="B52">
<span class="label">52.</span><cite>Shi Z, Deng R, Yuan Q, Mao Z, Wang R, Li H, Liao X, Ma H. 
Enzyme commission number prediction and benchmarking with hierarchical dual-core multitask learning framework. Research. 2023;6:0153.
</cite> [<a href="https://doi.org/10.34133/research.0153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10232324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37275124/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research&amp;title=Enzyme%20commission%20number%20prediction%20and%20benchmarking%20with%20hierarchical%20dual-core%20multitask%20learning%20framework&amp;volume=6&amp;publication_year=2023&amp;pages=0153&amp;pmid=37275124&amp;doi=10.34133/research.0153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B53">
<span class="label">53.</span><cite>Suzek BE, Wang Y, Huang H, McGarvey PB, Wu CH, Consortium U . 
UniRef clusters: A comprehensive and scalable alternative for improving sequence similarity searches. Bioinformatics. 2015;31(6):926–932.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4375400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25398609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=UniRef%20clusters:%20A%20comprehensive%20and%20scalable%20alternative%20for%20improving%20sequence%20similarity%20searches&amp;volume=31&amp;issue=6&amp;publication_year=2015&amp;pages=926-932&amp;pmid=25398609&amp;doi=10.1093/bioinformatics/btu739&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B54">
<span class="label">54.</span><cite>Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, Shindyalov IN, Bourne PE. 
The protein data bank. Nucleic Acids Res. 2000;28(1):235–242.
</cite> [<a href="https://doi.org/10.1093/nar/28.1.235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20protein%20data%20bank&amp;volume=28&amp;issue=1&amp;publication_year=2000&amp;pages=235-242&amp;pmid=10592235&amp;doi=10.1093/nar/28.1.235&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B55">
<span class="label">55.</span><cite>Varadi M, Anyango S, Deshpande M, Nair S, Natassia C, Yordanova G, Yuan D, Stroe O, Wood G, Laydon A, et al. 
AlphaFold protein structure database: Massively expanding the structural coverage of protein-sequence space with high-accuracy models. Nucleic Acids Res. 2022;50(D1):D439–D444.
</cite> [<a href="https://doi.org/10.1093/nar/gkab1061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34791371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=AlphaFold%20protein%20structure%20database:%20Massively%20expanding%20the%20structural%20coverage%20of%20protein-sequence%20space%20with%20high-accuracy%20models&amp;volume=50&amp;issue=D1&amp;publication_year=2022&amp;pages=D439-D444&amp;pmid=34791371&amp;doi=10.1093/nar/gkab1061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B56">
<span class="label">56.</span><cite>Steinegger M, Söding J. 
Clustering huge protein sequence sets in linear time. Nat Commun. 2018;9(1):2542.
</cite> [<a href="https://doi.org/10.1038/s41467-018-04964-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6026198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29959318/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Clustering%20huge%20protein%20sequence%20sets%20in%20linear%20time&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=2542&amp;pmid=29959318&amp;doi=10.1038/s41467-018-04964-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B57">
<span class="label">57.</span><cite>Abanades B, Wong WK, Boyles F, Georges G, Bujotzek A, Deane CM. 
ImmuneBuilder: Deep-learning models for predicting the structures of immune proteins. Commun Biol. 2023;6:575.
</cite> [<a href="https://doi.org/10.1038/s42003-023-04927-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10227038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37248282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Commun%20Biol&amp;title=ImmuneBuilder:%20Deep-learning%20models%20for%20predicting%20the%20structures%20of%20immune%20proteins&amp;volume=6&amp;publication_year=2023&amp;pages=575&amp;pmid=37248282&amp;doi=10.1038/s42003-023-04927-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B58">
<span class="label">58.</span><cite>Ruffolo JA, Chu LS, Mahajan SP, Gray JJ. 
Fast, accurate antibody structure prediction from deep learning on massive set of natural antibodies. Nat Commun. 2023;14(1):2389.
</cite> [<a href="https://doi.org/10.1038/s41467-023-38063-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10129313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37185622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Fast,%20accurate%20antibody%20structure%20prediction%20from%20deep%20learning%20on%20massive%20set%20of%20natural%20antibodies&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=2389&amp;pmid=37185622&amp;doi=10.1038/s41467-023-38063-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B59">
<span class="label">59.</span><cite>Dunbar J, Krawczyk K, Leem J, Baker T, Fuchs A, Georges G, Shi J, Deane CM. 
SAbDab: The structural antibody database. Nucleic Acids Res. 2014;42:D1140–D1146.
</cite> [<a href="https://doi.org/10.1093/nar/gkt1043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3965125/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24214988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=SAbDab:%20The%20structural%20antibody%20database&amp;volume=42&amp;publication_year=2014&amp;pages=D1140-D1146&amp;pmid=24214988&amp;doi=10.1093/nar/gkt1043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B60">
<span class="label">60.</span><cite>Aaron vdO, Oriol V, Koray K. Neural discrete representation learning. Paper presented at: Proceedings of the 31st International Conference on Neural Information Processing Systems; 2017; Long Beach, CA.</cite>
</li>
<li id="B61">
<span class="label">61.</span><cite>Achiam J, Adler S, Agarwal S, Ahmad L, Akkaya I, Aleman FL, Almeida D, Altenschmidt J, Altman S, et al. GPT-4 technical report. arXiv. 2023. 10.48550/arXiv.2303.08774</cite> [<a href="https://doi.org/10.48550/arXiv.2303.08774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="B62">
<span class="label">62.</span><cite>Radford A, Kim JW, Hallacy C, Ramesh A, Goh G, Agarwal S, Sastry G, Askell A, Mishkin P, Clark J, et al. Learning transferable visual models from natural language supervision. Paper presented at: Proceedings of the 38th International Conference on Machine Learning; 2021; virtual.</cite>
</li>
<li id="B63">
<span class="label">63.</span><cite>Paszke A, Gross S, Massa F, Lerer A, Bradbury J, Chanan G, Killeen T, Lin Z, Gimelshein N, Antiga L, et al. PyTorch: An imperative style high-performance deep learning library. Paper presented at: Proceedings of the 33rd International Conference on Neural Information Processing Systems; 2019; Vancouver, BC, Canada.</cite>
</li>
<li id="B64">
<span class="label">64.</span><cite>Rajbhandari S, Rasley J, Ruwase O, He Y. ZeRO: Memory optimizations toward training trillion parameter models. Paper presented at: Proceedings of the International Conference for High Performance Computing, Networking, Storage and Analysis; 2020; Atlanta, GA.</cite>
</li>
<li id="B65">
<span class="label">65.</span><cite>Wang Z, Zhang Q, Yu H, et al. Multi-level protein structure pre-training with prompt learning. Paper presented at: Proceedings of the 11th International Conference on Learning Representations; 2023; Kigali, Rwanda.</cite>
</li>
<li id="B66">
<span class="label">66.</span><cite>Chen C, Zhou J, Wang F, Liu X, Dou D. 
Structure-aware protein self-supervised learning. Bioinformatics. 2023;39(4):btad189.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btad189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10139775/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37052532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Structure-aware%20protein%20self-supervised%20learning&amp;volume=39&amp;issue=4&amp;publication_year=2023&amp;pages=btad189&amp;pmid=37052532&amp;doi=10.1093/bioinformatics/btad189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B67">
<span class="label">67.</span><cite>Zhang Z, Xu M, Lozano A, Chenthamarakshan V, Das P, Tang J. Enhancing protein language model with structure-based encoder and pre-training. Paper presented at: Proceedings of the 11th International Conference on Learning Representations Machine Learning for Drug Discovery Workshop; 2023; virtual.</cite>
</li>
<li id="B68">
<span class="label">68.</span><cite>Lee Y, Yu H, Lee J, Kim J. Pre-training sequence, structure, and surface features for comprehensive protein representation learning. Paper presented at: Proceedings of the 11th International Conference on Learning Representations; 2023; Kigali, Rwanda.</cite>
</li>
<li id="B69">
<span class="label">69.</span><cite>Heinzinger M, Weissenow K, Sanchez JG, Henkel A, Steinegger M, Rost B. ProstT5: Bilingual language model for protein sequence and structure. bioRxiv. 2023. 10.1101/2023.07.23.550085</cite> [<a href="https://doi.org/10.1101/2023.07.23.550085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11616678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39633723/" class="usa-link">PubMed</a>]</li>
<li id="B70">
<span class="label">70.</span><cite>Gao K, Wu L, Zhu J, Zhu J, Peng T, Xia Y, He L, Xie S, Qin T, Liu H, He K, et al. Pre-training antibody language models for antigen-specific computational antibody design. Paper presented at: Proceedings of the 29th ACM SIGKDD Conference on Knowledge Discovery and Data Mining; 2023; Long Beach, CA.</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>Supplementary 1</span><p>Notes S1 to S6</p>
<p>Tables S1 and S2</p>
<p>References [<a href="#B61" class="usa-link" aria-describedby="B61">61</a>–<a href="#B70" class="usa-link" aria-describedby="B70">70</a>]</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12364524/bin/research.0721.f1.pdf" data-ga-action="click_feat_suppl" class="usa-link">research.0721.f1.pdf</a><sup> (3.9MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The source code and data are freely available at <a href="https://github.com/KDurant-123/S2ALM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/KDurant-123/S2ALM</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Research are provided here courtesy of <strong>American Association for the Advancement of Science (AAAS) and Science and Technology Review Publishing House</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.34133/research.0721"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/research.0721.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12364524/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12364524/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12364524%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12364524/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12364524/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12364524/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40837877/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12364524/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40837877/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12364524/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12364524/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="4tF0qZEKbSzfTXJ5SURUeYDKjNt2RkyLtBRLruON46vio1qSyrWciuV22IYzxKGQ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
