
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Transcriptome sequencing reveals the evolutionary histories and gene expression evolution in two related Pagurus species - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE533AD08AF33413053AD000360FE679.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367144/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="Transcriptome sequencing reveals the evolutionary histories and gene expression evolution in two related Pagurus species">
<meta name="citation_author" content="Zakea Sultana">
<meta name="citation_author_institution" content="Fisheries Division, Japan International Research Center for Agricultural Sciences, Tsukuba, Ibaraki, Japan">
<meta name="citation_author" content="Isaac Adeyemi Babarinde">
<meta name="citation_author_institution" content="Laboratory of Inflammation and Vaccines (LIV), Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, Guangdong Province, China">
<meta name="citation_author" content="Masafumi Nozawa">
<meta name="citation_author_institution" content="Department of Biological Sciences, Tokyo Metropolitan University, Tokyo, Japan">
<meta name="citation_author_institution" content="Research Center for Genomics and Bioinformatics, Tokyo Metropolitan University, Tokyo, Japan">
<meta name="citation_author" content="Kazuho Ikeo">
<meta name="citation_author_institution" content="Laboratory for DNA Data Analysis, National Institute of Genetics, Mishima, Shizuoka, Japan">
<meta name="citation_author_institution" content="Department of Genetics, School of Life Science, The Graduate University for Advanced Studies (SOKENDAI), Mishima, Shizuoka, Japan">
<meta name="citation_author" content="Akira Asakura">
<meta name="citation_author_institution" content="Seto Marine Biological Laboratory, Field Science Education and Research Center, Kyoto University, Nishimuro, Wakayama, Japan">
<meta name="citation_author" content="Tomoyuki Nakano">
<meta name="citation_author_institution" content="Seto Marine Biological Laboratory, Field Science Education and Research Center, Kyoto University, Nishimuro, Wakayama, Japan">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0330170">
<meta name="citation_doi" content="10.1371/journal.pone.0330170">
<meta name="citation_pmid" content="40834015">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367144/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367144/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367144/pdf/pone.0330170.pdf">
<meta name="description" content="Pagurus lanuginosus (De Haan, 1833–1850) and Pagurus maculosus (Komai and Imafuku, 1996) are two closely related species of the genus Pagurus, generally referred to as “right-handed hermit crabs”. Previously thought to be color morphs of the same ...">
<meta name="og:title" content="Transcriptome sequencing reveals the evolutionary histories and gene expression evolution in two related Pagurus species">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Pagurus lanuginosus (De Haan, 1833–1850) and Pagurus maculosus (Komai and Imafuku, 1996) are two closely related species of the genus Pagurus, generally referred to as “right-handed hermit crabs”. Previously thought to be color morphs of the same ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367144/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12367144">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0330170"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0330170.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367144%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12367144/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12367144/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367144/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0330170" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 20;20(8):e0330170. doi: <a href="https://doi.org/10.1371/journal.pone.0330170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330170</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Transcriptome sequencing reveals the evolutionary histories and gene expression evolution in two related <em>Pagurus</em> species</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sultana%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zakea Sultana</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zakea Sultana</span></h3>
<div class="p">
<sup>1</sup>Fisheries Division, Japan International Research Center for Agricultural Sciences, Tsukuba, Ibaraki, Japan</div>
<div>Conceptualization, Data curation, Investigation, Methodology, Validation, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sultana%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zakea Sultana</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Babarinde%20IA%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Isaac Adeyemi Babarinde</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Isaac Adeyemi Babarinde</span></h3>
<div class="p">
<sup>2</sup>Laboratory of Inflammation and Vaccines (LIV), Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, Guangdong Province, China</div>
<div>Data curation, Formal analysis, Investigation, Methodology, Resources, Software, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Babarinde%20IA%22%5BAuthor%5D" class="usa-link"><span class="name western">Isaac Adeyemi Babarinde</span></a>
</div>
</div>
<sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nozawa%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Masafumi Nozawa</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Masafumi Nozawa</span></h3>
<div class="p">
<sup>3</sup>Department of Biological Sciences, Tokyo Metropolitan University, Tokyo, Japan</div>
<div class="p">
<sup>4</sup>Research Center for Genomics and Bioinformatics, Tokyo Metropolitan University, Tokyo, Japan</div>
<div>Conceptualization, Investigation, Methodology, Validation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nozawa%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Masafumi Nozawa</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ikeo%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Kazuho Ikeo</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kazuho Ikeo</span></h3>
<div class="p">
<sup>5</sup>Laboratory for DNA Data Analysis, National Institute of Genetics, Mishima, Shizuoka, Japan</div>
<div class="p">
<sup>6</sup>Department of Genetics, School of Life Science, The Graduate University for Advanced Studies (SOKENDAI), Mishima, Shizuoka, Japan</div>
<div>Conceptualization, Data curation, Methodology, Resources, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ikeo%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kazuho Ikeo</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Asakura%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Akira Asakura</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Akira Asakura</span></h3>
<div class="p">
<sup>7</sup>Seto Marine Biological Laboratory, Field Science Education and Research Center, Kyoto University, Nishimuro, Wakayama, Japan</div>
<div>Conceptualization, Investigation, Methodology, Resources, Supervision, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Asakura%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Akira Asakura</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakano%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Tomoyuki Nakano</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Tomoyuki Nakano</span></h3>
<div class="p">
<sup>7</sup>Seto Marine Biological Laboratory, Field Science Education and Research Center, Kyoto University, Nishimuro, Wakayama, Japan</div>
<div>Conceptualization, Data curation, Funding acquisition, Methodology, Project administration, Resources, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakano%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tomoyuki Nakano</span></a>
</div>
</div>
<sup>7,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Feng ZHANG</span><sup>8</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Fisheries Division, Japan International Research Center for Agricultural Sciences, Tsukuba, Ibaraki, Japan</div>
<div id="aff002">
<sup>2</sup>Laboratory of Inflammation and Vaccines (LIV), Shenzhen Institutes of Advanced Technology, Chinese Academy of Sciences, Shenzhen, Guangdong Province, China</div>
<div id="aff003">
<sup>3</sup>Department of Biological Sciences, Tokyo Metropolitan University, Tokyo, Japan</div>
<div id="aff004">
<sup>4</sup>Research Center for Genomics and Bioinformatics, Tokyo Metropolitan University, Tokyo, Japan</div>
<div id="aff005">
<sup>5</sup>Laboratory for DNA Data Analysis, National Institute of Genetics, Mishima, Shizuoka, Japan</div>
<div id="aff006">
<sup>6</sup>Department of Genetics, School of Life Science, The Graduate University for Advanced Studies (SOKENDAI), Mishima, Shizuoka, Japan</div>
<div id="aff007">
<sup>7</sup>Seto Marine Biological Laboratory, Field Science Education and Research Center, Kyoto University, Nishimuro, Wakayama, Japan</div>
<div id="edit1">
<sup>8</sup>Nanjing Agricultural University, CHINA</div>
<div class="author-notes p">
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>adeyemi@siat.ac.cn</span> (IAB); <span>nakano.tomoyuki.2a@kyoto-u.ac.jp</span> (TN)</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Zakea Sultana</span></strong>: <span class="role">Conceptualization, Data curation, Investigation, Methodology, Validation, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Isaac Adeyemi Babarinde</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Methodology, Resources, Software, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Masafumi Nozawa</span></strong>: <span class="role">Conceptualization, Investigation, Methodology, Validation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Kazuho Ikeo</span></strong>: <span class="role">Conceptualization, Data curation, Methodology, Resources, Supervision, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Akira Asakura</span></strong>: <span class="role">Conceptualization, Investigation, Methodology, Resources, Supervision, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Tomoyuki Nakano</span></strong>: <span class="role">Conceptualization, Data curation, Funding acquisition, Methodology, Project administration, Resources, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Feng ZHANG</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 4; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Sultana et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12367144  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40834015/" class="usa-link">40834015</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p><em>Pagurus lanuginosus</em> (De Haan, 1833–1850) and <em>Pagurus maculosus</em> (Komai and Imafuku, 1996) are two closely related species of the genus <em>Pagurus</em>, generally referred to as “right-handed hermit crabs”. Previously thought to be color morphs of the same species, recent studies have itemized their unique features. To investigate the molecular and gene expression evolution that have followed the divergence of these two species, we performed transcriptome sequencing and <em>de novo</em> transcriptome assembly using cephalothorax (head) and pereopod (leg) of the two species. Synonymous and nonsynonymous divergence between the two species was estimated to be 0.0246 and 0.0066, respectively. About 5% of the protein-coding transcripts had signatures of positive selection. The phylogenetic analyses showed that the two species are indeed closely related and diverged about 6.24 million years ago. Comparison of tissue expression showed that head expressions were more conserved, and that more genes were upregulated in the legs of <em>P. maculosus</em> leg; while more pigmentation genes were found to be upregulated in the legs of <em>P. lanuginosus</em>. Genes associated with the extracellular matrix and space, chitin binding and chitin-based larval cuticles were found to have higher expression in the legs of <em>P. maculosus</em>, suggesting the roles in morphological differences. This study sheds light on the evolutionary history of the two species at the molecular level.</p></section><section id="sec001"><h2 class="pmc_sec_title">Introduction</h2>
<p>The genus <em>Pagurus</em> [<a href="#pone.0330170.ref001" class="usa-link" aria-describedby="pone.0330170.ref001">1</a>], generally referred to as ‘right-handed hermit crabs’, is widely distributed in almost all marine habitats ranging from shallow to deep waters, worldwide. The genus is found in diverse biotopes along hard rocky shores to intertidal muddy flats. Thus, the species of this genus exhibit extremely high morphological diversity [<a href="#pone.0330170.ref002" class="usa-link" aria-describedby="pone.0330170.ref002">2</a>]. The morphological diversity and the ecological heterogeneity associated with the species of this genus make it difficult to identify and classify <em>Pagurus</em> species. Consequently, many sibling species remain cryptic species that are difficult to recognize using classic systematic methods [<a href="#pone.0330170.ref003" class="usa-link" aria-describedby="pone.0330170.ref003">3</a>]. Many new species are being described and added to this genus continuously, even as many more are yet to be recognized [<a href="#pone.0330170.ref004" class="usa-link" aria-describedby="pone.0330170.ref004">4</a>]. Therefore, the taxonomic classification of this ever-proliferative genus <em>Pagurus</em> is far from being resolved [<a href="#pone.0330170.ref005" class="usa-link" aria-describedby="pone.0330170.ref005">5</a>].</p>
<p>This concern is exemplified in two Japanese water intertidal hermit crab species, <em>Pagurus lanuginosus</em> [<a href="#pone.0330170.ref006" class="usa-link" aria-describedby="pone.0330170.ref006">6</a>] and <em>P. maculosus</em> [<a href="#pone.0330170.ref007" class="usa-link" aria-describedby="pone.0330170.ref007">7</a>]. Because of their superficial similarities, the two species were previously considered to be two color morphs of one species, <em>P. lanuginosus</em> [<a href="#pone.0330170.ref008" class="usa-link" aria-describedby="pone.0330170.ref008">8</a>]. The major difference was the color of their chromatophores scattered on the pereopods (legs) and cephalothorax (head). The chromatophores are white in <em>P. maculosus</em> and black in <em>P. lanuginosus</em> [<a href="#pone.0330170.ref007" class="usa-link" aria-describedby="pone.0330170.ref007">7</a>,<a href="#pone.0330170.ref008" class="usa-link" aria-describedby="pone.0330170.ref008">8</a>]. They both inhabit intertidal rocky shores and sometimes exhibit cohabitation [<a href="#pone.0330170.ref009" class="usa-link" aria-describedby="pone.0330170.ref009">9</a>]. However, the distribution of <em>P. lanuginosus</em> is broader than that of <em>P. maculosus</em>. Additional differences in morphological characters such as the number of calcareous teeth on the dactyl and the presence or absence of a large tubercle on the merus of the right cheliped in adults were described by Komai and Imafuku [<a href="#pone.0330170.ref007" class="usa-link" aria-describedby="pone.0330170.ref007">7</a>]. Thus, Komai and Imafuku concluded that these hermit crabs are two different species [<a href="#pone.0330170.ref007" class="usa-link" aria-describedby="pone.0330170.ref007">7</a>]. Further, careful studies of the larval and post-larval development of these two species revealed differences in the number of setae or spines on the antennule, maxillule and maxilla, and the presence of chromatophores on the carapace [<a href="#pone.0330170.ref010" class="usa-link" aria-describedby="pone.0330170.ref010">10</a>,<a href="#pone.0330170.ref011" class="usa-link" aria-describedby="pone.0330170.ref011">11</a>].</p>
<p>Morphological characters have been classically used for the identification and descriptions of decapod crustaceans. However, in some cases, morphological characters were very similar between congeners, raising uncertainty about their independent species status. With the advancement in nucleic acid extraction [<a href="#pone.0330170.ref012" class="usa-link" aria-describedby="pone.0330170.ref012">12</a>] and sequencing technologies [<a href="#pone.0330170.ref013" class="usa-link" aria-describedby="pone.0330170.ref013">13</a>], recent attention has been focused on the evolutionary studies of diverse decapod groups, utilizing the molecular approaches to disentangle various unresolved questions on the origins of diversity, habitat shifts, and diversification [<a href="#pone.0330170.ref014" class="usa-link" aria-describedby="pone.0330170.ref014">14</a>–<a href="#pone.0330170.ref017" class="usa-link" aria-describedby="pone.0330170.ref017">17</a>]. Molecular evolution has now been used as the basis for investigating morphological evolution [<a href="#pone.0330170.ref018" class="usa-link" aria-describedby="pone.0330170.ref018">18</a>–<a href="#pone.0330170.ref020" class="usa-link" aria-describedby="pone.0330170.ref020">20</a>].</p>
<p>Previous molecular evolution attempts using mitochondrial loci showed that <em>P. maculosus</em> and <em>P. lanuginosus</em> were indeed phylogenetically very close [<a href="#pone.0330170.ref009" class="usa-link" aria-describedby="pone.0330170.ref009">9</a>,<a href="#pone.0330170.ref021" class="usa-link" aria-describedby="pone.0330170.ref021">21</a>]. In this study, we aimed to investigate the phylogenetic relationships of the two species with other related species with additional focus on the gene expression evolution using transcriptome data. To achieve this, we generated head and leg transcriptomes of the two species. We conducted <em>de novo</em> transcript assembly and functionally annotated the newly assembled transcripts. We used the assembled transcripts to reveal some insights into the molecular evolution, phylogenetic relationship and gene expression evolution in the two hermit crab species.</p></section><section id="sec002"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="sec003"><h3 class="pmc_sec_title">Sampling location and permit</h3>
<p>Specimens of <em>Pagurus lanuginosus</em> and <em>P. maculosus</em> were collected from the same location at a rocky shore at Shirahama (33°41’ N, 135°23’ E), Wakayama Prefecture, along the North Pacific, Japan, in November 2023, and were transported to the laboratory alive. Sample collection permit for experimental research with number 5−2 was granted by Wakayama Prefecture, Japan. Researches on <em>Pagurus</em> species do not require any special approval. All experiments complied with institutional regulations and Japanese policy on animal use [<a href="#pone.0330170.ref022" class="usa-link" aria-describedby="pone.0330170.ref022">22</a>].</p></section><section id="sec004"><h3 class="pmc_sec_title">RNA extraction and sequencing</h3>
<p>Three individuals of each species (<em><em>P.</em>
<em>maculosus</em></em> and <em>P. lanuginosus</em>) were sampled for RNA extraction. For each individual, RNA was extracted from two distinct tissues in which pigmentation is most prevalent: the cephalothorax (head) and pereopod (leg), as illustrated in <a href="#pone.0330170.g001" class="usa-link">Fig 1A</a> and <a href="#pone.0330170.g001" class="usa-link">Fig 1B</a>. This resulted in three biological replicates per tissue type for each species. Total RNA was extracted by homogenizing approximately 25 mg of head or leg in 500 µl of Binding Buffer (4 M Guanidine Thiocyanate; 25 mM Na Citrate; 0.5% SDSC; 0.1 M Mercaptoethanol) with a tissue homogenizer (BEADS CRUSHER µT-12, TAITEC). The homogenate was processed following the manufacturer’s protocol of PureLink RNA Mini Kits (Qiagen). RNA purity and integrity were assessed visually in agarose gels, and quantified in spectrophotometry (Nanodrop ND-1000, NanoDrop Technologies, USA). High-quality total RNA samples were sent to an outside facility (Macrogen, Japan) for subsequent steps including library preparation and next generations sequencing. Library preparation was done using TruSeq Stranded Total RNA Library Preparation Kit (Illumina) and the sequencing was done on the NovaSeq 6000 system (Illumina) platform with 150 base pair paired-end (PE) reads. Head and leg of each species were sequenced in triplicates, with 10Gb of data per sample. All raw reads generated in this study are available in the DDBJ Sequence Read Archive (DRA) under the BioProject accession number PRJDB35803. The data sets supporting the results presented here are available with the accession numbers DRR708008-DRR708019.</p>
<figure class="fig xbox font-sm" id="pone.0330170.g001"><h4 class="obj_head">Fig 1. Transcriptome sequencing of <em>Pagurus maculosus</em> and <em>P. lanuginosus.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367144_pone.0330170.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5288/12367144/6f568eb38859/pone.0330170.g001.jpg" loading="lazy" height="900" width="775" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330170.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>A. The picture of <em>P. maculosus</em> showing the parts from which RNA was extracted. B. The picture of <em>P. lanuginosus</em> showing the parts from which RNA was extracted. C. Alignment rates of reads that could be mapped back to the assembled transcripts. D. The distribution of coding and noncoding transcripts and expression. The transcript pie charts show the distribution of the transcript counts while the expression pie charts show contributions to the expression levels. E. Homology of a species in the other species. Unique transcripts did not have significant similarity with any transcripts in the other species. Transcripts with homology were divided into “one-to-one” or “other homologs” based on one-to-one correspondence in the two species. F. Coverage of one-to-one transcripts in the other species.</p></figcaption></figure></section><section id="sec005"><h3 class="pmc_sec_title">Retrieval of publicly available annotation data and crustacean short-read data</h3>
<p>Publicly available data were retrieved for annotation and evolutionary analyses. Nonredundant (nr) amino acid sequences were retrieved from the NCBI database. The crustacean species with the highest homology were selected. The short-read raw RNA-seq data of the top homologous crustacean species, with at least 20,000 homologous transcripts, were then retrieved from the NCBI short-read archives. In addition, we retrieved the short-read RNA-seq data from the project PRJNA562428 [<a href="#pone.0330170.ref023" class="usa-link" aria-describedby="pone.0330170.ref023">23</a>]. Both newly generated RNA-seq data and publicly available data were then independently assembled using the same <em>de novo</em> transcript assembly pipeline.</p></section><section id="sec006"><h3 class="pmc_sec_title">Transcriptome assembly and its evaluation</h3>
<p>The quality of the sequenced reads was examined using version v0.11.9 of fastQC (<a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioinformatics.babraham.ac.uk/projects/fastqc</a>). For transcript assembly, raw reads of all tissues and replicates of the same species were concatenated. <em>De novo</em> transcript assembly was performed using the default settings of the version v2.13.2 of Trinity [<a href="#pone.0330170.ref024" class="usa-link" aria-describedby="pone.0330170.ref024">24</a>,<a href="#pone.0330170.ref025" class="usa-link" aria-describedby="pone.0330170.ref025">25</a>]. Initial transcript expression quantification was done using bowtie2 in RSEM [<a href="#pone.0330170.ref026" class="usa-link" aria-describedby="pone.0330170.ref026">26</a>], with the settings “<em>--bowtie2 --paired-end --bowtie2-sensitivity-level very_sensitive</em>”. Transcripts with less than 2 RSEM’s expected counts were discarded. To evaluate the quality of the assembly, bowtie2 [<a href="#pone.0330170.ref027" class="usa-link" aria-describedby="pone.0330170.ref027">27</a>] was used to map the raw reads back to the assembled transcripts to quantify percentage of reads that were used for the actual assembly. The filtered transcript sets were then used for the downstream analyses.</p></section><section id="sec007"><h3 class="pmc_sec_title">Transcript expression quantification</h3>
<p>Transcript expression quantification was performed using RSEM with bowtie2 alignment option. First, RSEM reference was built for the transcript assembly of each species. The raw reads were cleaned with fastp [<a href="#pone.0330170.ref028" class="usa-link" aria-describedby="pone.0330170.ref028">28</a>] with the settings “<em>-q 20 -u 10</em>”. The fastp-filtered reads were then used for the expression quantification. The RSEM settings used were “<em>--bowtie2 --paired-end --bowtie2-sensitivity-level very_sensitive</em>”.</p></section><section id="sec008"><h3 class="pmc_sec_title">Transcript annotation</h3>
<p>Transcript annotation was focused on the predicted protein-coding transcripts. First, the TransDecoder [<a href="#pone.0330170.ref025" class="usa-link" aria-describedby="pone.0330170.ref025">25</a>] was used to extract all the six open reading frames (ORFs) with at least 50 amino acids. The most likely protein-coding ORF was then predicted by TransDecoder in Trinity Package. TransDecoder-predicted amino acid sequences were then used as queries for homology search to identify the homologous annotated proteins in NCBI Reference Sequence (RefSeq) non-redundant (nr) [<a href="#pone.0330170.ref029" class="usa-link" aria-describedby="pone.0330170.ref029">29</a>,<a href="#pone.0330170.ref030" class="usa-link" aria-describedby="pone.0330170.ref030">30</a>], Uniprot reference (UniRef90) [<a href="#pone.0330170.ref031" class="usa-link" aria-describedby="pone.0330170.ref031">31</a>] and UniProtKB/TrEMBL [<a href="#pone.0330170.ref032" class="usa-link" aria-describedby="pone.0330170.ref032">32</a>] databases. Homology search was done using DIAMOND [<a href="#pone.0330170.ref033" class="usa-link" aria-describedby="pone.0330170.ref033">33</a>] with “<em>--sensitive</em>” mode.</p>
<p>Functional annotation of the predicted amino acids was done using InterProScan 5 [<a href="#pone.0330170.ref034" class="usa-link" aria-describedby="pone.0330170.ref034">34</a>]. The predicted gene ontology (GO) terms and pathways were also included in the output. For the annotation of pigmentation-related genes, we employed a published strategy [<a href="#pone.0330170.ref035" class="usa-link" aria-describedby="pone.0330170.ref035">35</a>]. Briefly, the term “pigmentation” was searched in the gene ontology database [<a href="#pone.0330170.ref036" class="usa-link" aria-describedby="pone.0330170.ref036">36</a>] to obtain the list of genes with pigmentation functions in any species. The sequences of the genes were then obtained from UniProtKB/TrEMBL database to obtain the pigmentation gene reference sequences. Predicted amino acids were then searched in the pigmentation reference using BLASTP [<a href="#pone.0330170.ref037" class="usa-link" aria-describedby="pone.0330170.ref037">37</a>] with “<em>--evalue 0.001</em>” to obtain the list of pigment-related crab genes.</p></section><section id="sec009"><h3 class="pmc_sec_title">Gene ontology enrichment analyses</h3>
<p>Gene ontology (GO) enrichment analyses were performed using the results of InterProScan. Genes with no annotated GO term was not included in the enrichment analyses. Similarly, GO terms with no annotated gene were excluded. GO analyses were computed for positively selected gene set and differentially expressed genes. For each GO term, the enrichment and hypergeometric probability were computed. Multiple testing was corrected with Bonferroni procedure [<a href="#pone.0330170.ref038" class="usa-link" aria-describedby="pone.0330170.ref038">38</a>]. Adjusted p value &lt; 0.05 was set as the threshold for significantly enriched GO term.</p></section><section id="sec010"><h3 class="pmc_sec_title">Evolutionary analyses</h3>
<p>Evolutionary analyses were conducted at both the nucleotide and amino acid levels. Nucleotide homology search was done with BLASTN [<a href="#pone.0330170.ref037" class="usa-link" aria-describedby="pone.0330170.ref037">37</a>] while amino acid searches were done with DIAMOND or BLASTP, depending on the size of the database. Reciprocal best hits, assessed by the bit scores, between the two species were termed “one-to-one”. The identification of the coding sequences (CDS) was done with TransDecoder. Stop codons were striped from the CDS. Each cluster of one-to-one genes were aligned with CLUSTALW [<a href="#pone.0330170.ref039" class="usa-link" aria-describedby="pone.0330170.ref039">39</a>]. Custom python scripts were used to concatenate the sequences, and partition them into different codon positions. The best amino acid and nucleotide models were estimated using maximum likelihood models in MEGA [<a href="#pone.0330170.ref040" class="usa-link" aria-describedby="pone.0330170.ref040">40</a>]. The rates of synonymous (dS) and nonsynonymous (dN) substitutions using both Nie-Gojobori [<a href="#pone.0330170.ref041" class="usa-link" aria-describedby="pone.0330170.ref041">41</a>] and Yang-Nielsen [<a href="#pone.0330170.ref042" class="usa-link" aria-describedby="pone.0330170.ref042">42</a>] methods were computed with <em>yn00</em> in version 4.10.6 of PAML [<a href="#pone.0330170.ref043" class="usa-link" aria-describedby="pone.0330170.ref043">43</a>]. The evolutionary pressures acting on the transcripts were estimated using the estimated and error values of dS and dN computed from Yang-Nielsen method [<a href="#pone.0330170.ref042" class="usa-link" aria-describedby="pone.0330170.ref042">42</a>]. For positively selected transcripts, dN-dN<sub>err</sub> &gt; dS + dS<sub>err</sub>, where dN<sub>err</sub> and dS<sub>err</sub> are error estimates for dN and dS, respectively. For purifying selection, dS-dS<sub>err </sub>&gt; dN + dN<sub>err</sub>. Other transcripts were classified as neutrally evolving.</p></section><section id="sec011"><h3 class="pmc_sec_title">Phylogenetics and divergence time estimates</h3>
<p>Because of the computational requirements of the maximum likelihood, only one fourth of the amino acid positions of the concatenated amino acids were used for the phylogenetic tree. To extract the alignment subset, the amino acids of the first position in 4-step moving window were extracted using a custom Python script. The phylogenetic analyses were conducted using maximum likelihood method in MEGA. Aligned sites with gaps or unknown amino acids were completely deleted. Gamma distribution with invariant sites (G + I) and gamma = 5, was used for the rate among sites. The initial tree was made with NJ/BioNJ method. Weak branch swap filter was set. LG model with frequencies were used. Branch support was evaluated with 1,000 bootstrap replicates.</p>
<p>Divergence time estimates were computed using MCMCtree program available in version 4.10.6 of PAML following previously published methods [<a href="#pone.0330170.ref044" class="usa-link" aria-describedby="pone.0330170.ref044">44</a>–<a href="#pone.0330170.ref046" class="usa-link" aria-describedby="pone.0330170.ref046">46</a>]. First, the overall substitution rates were estimated using <em>baseml</em> in PAML with the root age set to 450 million years ago (MYA). Next, the gradient (g) and Hessian (H) were estimated using the phylogenetic tree and alignments of the multiple species. Finally, the divergence times were estimated using the computed gradient, Hessian, alignments and the fossil calibrations. Nine calibration points were retrieved from published studies [<a href="#pone.0330170.ref017" class="usa-link" aria-describedby="pone.0330170.ref017">17</a>,<a href="#pone.0330170.ref047" class="usa-link" aria-describedby="pone.0330170.ref047">47</a>–<a href="#pone.0330170.ref049" class="usa-link" aria-describedby="pone.0330170.ref049">49</a>]. To assess the convergence, the divergence time estimation was repeated three times with different random seed numbers.</p></section><section id="sec012"><h3 class="pmc_sec_title">Differential gene expression</h3>
<p>The consistencies of the replicates were first examined by principal component analyses and clustering analyses using the expression data normalized by DESeq2 [<a href="#pone.0330170.ref050" class="usa-link" aria-describedby="pone.0330170.ref050">50</a>]. The two most consistent samples for each species-tissue combination were used for differential gene expression analyses. Differential gene expressions were computed for each tissue between the two species using DESeq2. Differential gene expression was done independently for head and leg. Statistical significance was set at adjusted p value of 0.05.</p></section><section id="sec013"><h3 class="pmc_sec_title">Statistical tests and graphical representations</h3>
<p>Statistical tests were computed using Scipy package in Python or R statistical package. For gene enrichment analyses, hypergeometric tests were conducted to test the significance of the enrichment of specific gene ontology and term. Multiple tests were corrected using Bonferroni method [<a href="#pone.0330170.ref038" class="usa-link" aria-describedby="pone.0330170.ref038">38</a>]. Graphical representations were done mostly by Matplotlib in Python and occasionally by R.</p></section></section><section id="sec014"><h2 class="pmc_sec_title">Results</h2>
<section id="sec015"><h3 class="pmc_sec_title">Head and leg transcriptome assemblies of <em>P. maculosus</em> and <em>P. lanuginosus</em>
</h3>
<p>RNA sequencing was performed with three biological replicates from each tissue type and species. For each replicate, ~ 21,000,000–27,000,000 paired-end reads were sequenced (<a href="#pone.0330170.s002" class="usa-link">S1 Table</a>). A total of 139,881,387 <em>P. lanuginosus</em> and 148,561,226 <em>P. maculosus</em> paired-end reads were generated (<a href="#pone.0330170.t001" class="usa-link">Table 1</a>). Transcriptome assembly produced 373,660 and 468,052 initial transcripts for <em>P. maculosus</em> and <em>P. lanuginosus,</em> respectively. Deeper sequencing has been reported to be positively correlated with the number of assembled transcripts, including potentially noisy transcripts [<a href="#pone.0330170.ref051" class="usa-link" aria-describedby="pone.0330170.ref051">51</a>]. Removal of potentially noisy transcripts produced 203,793 transcripts, including 140,954 genes for <em>P. maculosus,</em> and 259,375 transcripts, including 179,685 genes for <em>P. lanuginosus</em> (<a href="#pone.0330170.t001" class="usa-link">Table 1</a>). More than 95% of raw reads could be mapped back to the filtered assembled transcripts (<a href="#pone.0330170.g001" class="usa-link">Fig 1C</a>), showing that the transcriptomes were sufficiently represented in the transcript assemblies.</p>
<section class="tw xbox font-sm" id="pone.0330170.t001"><h4 class="obj_head">Table 1. <em>De novo</em> transcript assembly statistics.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Metrics</th>
<th align="left" rowspan="1" colspan="1">
<em>Pagurus lanuginosus</em>
</th>
<th align="left" rowspan="1" colspan="1">
<em>Pagurus maculosus</em>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Total raw read count</td>
<td align="left" rowspan="1" colspan="1">139881387</td>
<td align="left" rowspan="1" colspan="1">148561226</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Assembled gene count</td>
<td align="left" rowspan="1" colspan="1">179685</td>
<td align="left" rowspan="1" colspan="1">140954</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Assembled transcript count</td>
<td align="left" rowspan="1" colspan="1">259375</td>
<td align="left" rowspan="1" colspan="1">203793</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Transcript N50</td>
<td align="left" rowspan="1" colspan="1">1029</td>
<td align="left" rowspan="1" colspan="1">1084</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Transcript median length (bp)</td>
<td align="left" rowspan="1" colspan="1">470</td>
<td align="left" rowspan="1" colspan="1">498</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Transcript average length (bp)</td>
<td align="left" rowspan="1" colspan="1">765</td>
<td align="left" rowspan="1" colspan="1">804</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Predicted coding transcripts*</td>
<td align="left" rowspan="1" colspan="1">87037</td>
<td align="left" rowspan="1" colspan="1">69511</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330170.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t001fn001"><p><em>*</em>Transcripts with open reading frame (orf) with at least 50-amino acid were predicted to be protein-coding<em>.</em></p></div></div></section><p>Transcriptome assembly has the potential to retrieve both coding and noncoding transcripts. We therefore checked the proportion of both coding and noncoding transcripts in our transcriptome assembly (see Materials and Methods). In both species, about 34% of the transcripts were coding (<a href="#pone.0330170.g001" class="usa-link">Fig 1D</a>). However, expression quantifications showed that coding transcripts constituted about 79% and 75% of the <em>P. maculosus</em> and <em>P. lanuginosus</em> transcriptomes<em>,</em> respectively, reflecting higher expression levels of coding transcripts, as previously reported in human [<a href="#pone.0330170.ref052" class="usa-link" aria-describedby="pone.0330170.ref052">52</a>,<a href="#pone.0330170.ref053" class="usa-link" aria-describedby="pone.0330170.ref053">53</a>].</p>
<p>Next, we checked the homology between the assembled <em>P. maculosus</em> and <em>P. lanuginosus</em> transcripts. At the nucleotide levels, &gt; 95% of <em>P. lanuginosus</em> transcripts had homologs in <em>P. maculosus</em> (<a href="#pone.0330170.g001" class="usa-link">Fig 1E</a>). On the other hand, &gt; 97% <em>P. maculosus</em> transcripts had homologs in <em>P. lanuginosus.</em> At the amino acid levels, 83% of the <em>P. lanuginosus</em> transcripts and 86% of <em>P. maculosus</em> transcripts had detectable homologs in the other species, at the specified search thresholds. We defined one-to-one homology as the reciprocal best hit in the two species in reciprocal BLAST searches. Interestingly, although the majority of the transcripts had homologs in the two species, one-to-one relationship could not be established for many transcripts. The inability to establish one-to-one relationship could suggest the presence of transcriptional noise, false positives or false negatives reflecting the limitation of transcript assembly and annotation procedures from short-read RNA-seq [<a href="#pone.0330170.ref051" class="usa-link" aria-describedby="pone.0330170.ref051">51</a>,<a href="#pone.0330170.ref053" class="usa-link" aria-describedby="pone.0330170.ref053">53</a>]. This observation could also highlight the roles of independent evolution of gene duplications, losses and/or splicing events after the divergence of the two species. For transcripts with one-to-one correspondence, a substantial proportion of the transcripts had 90–100% alignment coverage in the two species (<a href="#pone.0330170.g001" class="usa-link">Fig 1F</a>), highlighting the confidence of our assembly and annotation, at least for transcripts with one-to-one relationship. These results highlight the properties of the assembled transcripts and the evolution of gene expressions after the divergence of the two species.</p></section><section id="sec016"><h3 class="pmc_sec_title">Annotation of the assembled <em>Pagurus</em> transcripts</h3>
<p>To functionally annotate the assembled transcripts, we searched for the predicted amino acids in NCBI RefSeq nr, UniRef90 and UniProtKB TrEMBL amino acid databases. We defined homology by significant similarity with the annotated proteins. About 48,000 <em>P. lanuginosus</em> and about 39,000 <em>P. maculosus</em> transcripts had homology in the databases (<a href="#pone.0330170.s003" class="usa-link">S2 Table</a>, upper panel of <a href="#pone.0330170.g002" class="usa-link">Fig 2A</a>). Interestingly, 48,712 <em>P. lanuginosus</em> transcripts had homolog in at least one of the investigated databases, including 47,653 (97.8%) with homology in the three investigated databases (<a href="#pone.0330170.s003" class="usa-link">S2 Table</a>). Similarly, 39,639 <em>P. maculosus</em> transcripts had homolog in at least one of the three databases, 38,805 (97.9%) of which were found in all three databases. The consistency across databases was also found at gene levels. By attributing the assembled transcript to the annotated protein with the highest score, <em>P. lanuginosus</em> transcripts had homology with 26,006 unique proteins in RefSeq nr, 24,027 for UniRef90 and 24,178 for UniProtKB TrEMBL database (lower panel of <a href="#pone.0330170.g002" class="usa-link">Fig 2A</a>). For <em>P. maculosus</em> transcripts, homology was found for 21,228, 19,574 and 19,721 unique proteins in NCBI nr, UniRef90 and UniProtKB TrEMBL, respectively. Majority of the amino acid sequences with homology had very high coverage with the annotated proteins in the database (<a href="#pone.0330170.g002" class="usa-link">Fig 2B</a>). These data showed that a substantial number of predicted amino acid sequences had homology with known proteins.</p>
<figure class="fig xbox font-sm" id="pone.0330170.g002"><h4 class="obj_head">Fig 2. Annotation of the assembled transcripts with predicted coding abilities.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367144_pone.0330170.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5288/12367144/c94bf6a4e3e7/pone.0330170.g002.jpg" loading="lazy" height="916" width="746" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330170.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>A. The number of assembled transcripts (upper panel) with significant homologs in the listed database and the corresponding number of unique homologous proteins in the database (lower panel). B. Coverage of the annotated proteins in the assembled transcripts. C. The number of transcripts found in the top 30 represented species. D. The list of the top annotated species with at least 20,000 homologs in the assembled transcripts.</p></figcaption></figure><p>In order to more efficiently annotate the assembled transcripts, we decided to identify the phylogenetically closely related well-annotated species that could potentially be used as the reference. To identify the closest species with known amino acid sequences, we identified the top 30 species with homologous proteins in both <em>P. lanuginosus</em> and <em>P. maculosus.</em> Eleven species each had homology to more than 20,000 predicted <em>P. lanuginosus</em> and <em>P. maculosus</em> transcripts (<a href="#pone.0330170.g002" class="usa-link">Fig 2C</a>). Interestingly, the same species were the top closest known species to both <em>P. lanuginosus</em> and <em>P. maculosus.</em> Based on the number of homologous amino acid sequences, the closest two species were American lobster, <em>Homarus americanus</em> and Porcelain crab, <em>Petrolisthes cinctipes.</em> The two species probably diverged from <em>Pagurus sp.</em> more than 200 million years ago (MYA) [<a href="#pone.0330170.ref054" class="usa-link" aria-describedby="pone.0330170.ref054">54</a>]. Therefore, the species with the highest numbers of homologous proteins do not share recent evolutionary histories with <em>Pagurus</em> species. These results highlight the need for concentrated efforts on the annotation of the transcripts of <em>Pagurus</em> sp<em>.</em></p></section><section id="sec017"><h3 class="pmc_sec_title">Phylogenetic analyses and divergence time estimates</h3>
<p>The divergence times between two <em>Pagurus</em> species and the closest crustacean species suggests that the closest species have not been sequenced or annotated to be included in the databases. We therefore, retrieved the raw reads of multiple crustacean species published as part of the crustacean annotated transcriptome (CAT) database [<a href="#pone.0330170.ref023" class="usa-link" aria-describedby="pone.0330170.ref023">23</a>]. Using the same pipeline, we assembled the transcripts and predicted amino acids sequences. To have a uniform processing, we also retrieved the short-read sequences of the top 11 species from the database (<a href="#pone.0330170.g002" class="usa-link">Figs 2B</a> and <a href="#pone.0330170.g002" class="usa-link">2C</a>) and perform <em>de novo</em> transcriptome assembly. In total, 20 species had proteins with homology to more than 20,000 <em>Pagurus</em> transcripts<em>.</em> We then extracted transcripts with one-to-one homology between <em>P. maculosus</em> and each of the 21 species, including <em>P. lanuginosus</em> and the 20 other species. The transcripts were aligned, concatenated and used for phylogenetic analyses. To ensure that the same sites were used in all the species, positions with alignment gaps in any species were completely deleted. A total of 2,035 one-to-one transcripts covering 147,186 gapless amino acids were extracted. These amino acid sequences were used for phylogenetic analyses.</p>
<p>The phylogenetic analyses revealed that <em>P. maculosus</em> and <em>P. lanuginosus</em> were sister species (<a href="#pone.0330170.g003" class="usa-link">Fig 3</a>), consistent with the previous results with the mitochondrial genomes [<a href="#pone.0330170.ref009" class="usa-link" aria-describedby="pone.0330170.ref009">9</a>,<a href="#pone.0330170.ref021" class="usa-link" aria-describedby="pone.0330170.ref021">21</a>]. We also found that the next closest species to <em>P. maculosus</em> and <em>P. lanuginosus</em> was a red king crab, <em>Paralithodes camtschaticus</em> which was found to be more closely related than another <em>Pagurus</em> species, <em>Pagurus longicarpus.</em> This result further confirmed the previous studies reporting “hermit-to-king” hypothesis that <em>Paralithodes</em> species evolved from <em>Pagurus</em> lineage [<a href="#pone.0330170.ref055" class="usa-link" aria-describedby="pone.0330170.ref055">55</a>–<a href="#pone.0330170.ref058" class="usa-link" aria-describedby="pone.0330170.ref058">58</a>]. Importantly, we confirmed that the closest species to the <em>P. maculosus</em> and <em>P. lanuginosus</em> are not well annotated in the public amino acid databases, highlighting the gap being filled by this study.</p>
<figure class="fig xbox font-sm" id="pone.0330170.g003"><h4 class="obj_head">Fig 3. Phylogenetic analyses and divergence times estimates of the investigated species.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367144_pone.0330170.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5288/12367144/0ff34f0a3cfc/pone.0330170.g003.jpg" loading="lazy" height="786" width="780" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330170.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The initial phylogenetic tree was made using maximum likelihood method with 1000 bootstraps. The tree was made using 147,186 gapless amino acid positions from 2,035 aligned one-to-one transcripts. The numbers on the nodes represent the bootstrap values in percentage. Nodes with no indicated bootstrap values had 100% bootstrap support. The calibration points are marked in alphabets a-i, and are described in <a href="#pone.0330170.s004" class="usa-link">S3 Table</a>. The divergence times were estimated using MCMCtree.</p></figcaption></figure><p>Next, we computed the divergence times involving 22 species using MCMC tree. Nine calibration points (<a href="#pone.0330170.s004" class="usa-link">S3 Table</a>) were used. Our analysis showed that <em>P. maculosus</em> and <em>P. lanuginosus</em> diverged around 6.24 (4.43–8.16 95% confidence interval) MYA (<a href="#pone.0330170.g003" class="usa-link">Fig 3</a>). The two species diverged from the red king crab, <em>Par. camtschaticus</em> around 47.63 (36.14–59.77 95% confidence interval) MYA. The closest identified species pair was two porcelain crabs, <em>Pet. cinctipes</em> and <em>Pet. manimaculis</em> which were found to have diverged less than 1 MYA. These two <em>Petrolisthes</em> species were estimated to have diverged from <em>Pagurus</em> species 249.86 (227.16–272.02 95% confidence interval) MYA. Even more diverged was the American lobster, <em>H. americanus</em> which was found to have diverged from <em>Pagurus</em> species more than 350 MYA. These results showed that <em>P. maculosus</em> and <em>P. lanuginosus</em> are sister species, and highlight the phylogenetic relationships between <em>Pagurus</em> species and selected crustacean species with transcriptome data.</p></section><section id="sec018"><h3 class="pmc_sec_title">Selection pressures after <em>P. maculosus</em> and <em>P. lanuginosus</em> divergence</h3>
<p>After confirming that both <em>Pagurus</em> species were sister species, we proceeded to investigate the evolutionary pressures that have shaped the evolution of the two species after their divergence. We first computed the synonymous (dS) and nonsynomous (dN) substitutions between the two species using 22,516 transcripts with a minimum gapless length of 50 amino acids. The estimations of dS and dN from both Yang-Nielsen [<a href="#pone.0330170.ref042" class="usa-link" aria-describedby="pone.0330170.ref042">42</a>] and Nei-Gojobori [<a href="#pone.0330170.ref041" class="usa-link" aria-describedby="pone.0330170.ref041">41</a>] methods were very consistent with very high correlation coefficients (<a href="#pone.0330170.g004" class="usa-link">Fig 4A</a>). The median dS and dN values from Nei-Gojobori method were 0.0284 and 0.0062, respectively while the vales from Yang- Nielsen method were 0.0246 and 0.0066, respectively (<a href="#pone.0330170.g004" class="usa-link">Fig 4B</a>). These results showed that both methods produced similar results.</p>
<figure class="fig xbox font-sm" id="pone.0330170.g004"><h4 class="obj_head">Fig 4. Evolutionary analyses of the assembled transcripts with protein-coding potentials.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367144_pone.0330170.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5288/12367144/bc54ddc97980/pone.0330170.g004.jpg" loading="lazy" height="669" width="743" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330170.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>A. Scatter plots showing the relationships between Yang-Nielsen and Nei-Gojobori methods for synonymous (left) and nonsynonymous (right) substitutions. r = Pearson’s correlation coefficient. B. The distributions of synonymous and nonsynonymous substitutions computed with Yang-Nielsen and Nei-Gojobori methods. C. The distributions of dN/dS computed from Yang-Nielsen method. The transcripts were grouped into neutral, negative and positive based on the values and error estimates of the dN and dS. D. Pie charts showing the distributions of the transcripts based on the selection pressures. E. The percentages of positively selected transcripts with homology to pigmentation genes. F. Venn diagram showing the overlap between one-to-one and pigmentation genes. G. Venn diagram showing the positively selected pigmentation genes. H. Gene ontology enrichments for positively selected transcripts.</p></figcaption></figure><p>Having established the consistencies of the two methods, we decided to use Yamg-Nielsen method for the subsequent analyses because the method provides error estimate for each value. We computed the dN/dS ratio to identify the evolutionary pressures acting on each predicted protein-coding transcript. Evolutionary selections of the transcripts were classified into neutral, negative and positive based on the value and error estimates of dS and dN (see Materials and Methods section for details). For neutral transcripts, the estimates of dN and dS and the corresponding errors overlapped. Positively selected transcripts had higher dN values (including the error estimates) while transcripts under purifying selection had higher dS values (including the error estimates). Although higher dN than dS values could also result from relaxed purifying selection [<a href="#pone.0330170.ref059" class="usa-link" aria-describedby="pone.0330170.ref059">59</a>,<a href="#pone.0330170.ref060" class="usa-link" aria-describedby="pone.0330170.ref060">60</a>], we simply referred to transcripts with significantly higher dN than dS values as positively selected. The distribution of the dN/dS ratio revealed that dN/dS ratio was less than one for most of the transcripts. (<a href="#pone.0330170.g004" class="usa-link">Fig 4C</a>). Because of the inclusion of error estimates in classifying the transcripts into different categories, transcripts from different categories may have similar dN/dS values. Interestingly, dN/dS values for some negatively and positively selected transcripts overlapped the values for some neutrally evolving transcripts while other evolutionarily selected transcripts had substantially higher values, possibly reflecting two distinct levels of selection as presented in Ohta’s nearly neutral theory of molecular evolution [<a href="#pone.0330170.ref061" class="usa-link" aria-describedby="pone.0330170.ref061">61</a>]. The distribution of selection pressures showed that about 55% of the transcripts were under purifying selection while only 5% were under positive selection (<a href="#pone.0330170.g004" class="usa-link">Fig 4D</a>), suggesting that purifying selection dominates the evolutionary pressures shaping the gene evolution.</p>
<p>The two species were initially thought to be color morphs of the same species. We therefore checked if the pigmentation genes were positively selected. We first identified assembled <em>Pagurus</em> genes that were homologous to known pigmentation genes (<a href="#pone.0330170.s005" class="usa-link">S4 Table</a>). Indeed, we found 52 pigmentation genes to be positively selected (<a href="#pone.0330170.g004" class="usa-link">Fig 4E</a>). However, the 52 genes only represented about 1% of total one-to-one pigment-related genes (<a href="#pone.0330170.g004" class="usa-link">Figs 4E</a> and <a href="#pone.0330170.g004" class="usa-link">4F</a>). Specifically, while 4,260 one-to-one pigmentation genes were found (<a href="#pone.0330170.g004" class="usa-link">Fig 4F</a>), only 52 one-to-one pigmentation genes were found to be positively selected (<a href="#pone.0330170.g004" class="usa-link">Figs 4E</a> and <a href="#pone.0330170.g004" class="usa-link">4G</a>). The GO enrichment analyses showed that the most significantly enriched term in the positively selected genes was extracellular region (GO:0.0005576) (<a href="#pone.0330170.g004" class="usa-link">Fig 4H</a>). These results revealed the dynamics of evolutionary selection after the divergence of <em>P. maculosus</em> and <em>P. lanuginosus.</em></p></section><section id="sec019"><h3 class="pmc_sec_title">Evolution of gene expressions in <em>P. maculosus</em> and <em>P. lanuginosus</em>
</h3>
<p>To investigate the evolution of expression levels in the two species, we took advantage of the high sequence similarity of the two species. It is important to note that the median synonymous divergence was lower than 3%, and the non-synonymous divergence was even lower (0.6%) (<a href="#pone.0330170.g004" class="usa-link">Fig 4B</a>). We wanted to see if the assembled transcripts from a species could be used as reference for the quantification of the tissues from the other species. We first asked if the proportion of the homologous genes in the transcriptome of both species were substantial enough. Interestingly, close to 100% of the transcriptomes in both species came from homologous genes (<a href="#pone.0330170.g005" class="usa-link">Fig 5A</a>). However, the percentages of transcriptomes with one-to-one homology were lower, with the value dropping to 70% and 79% in a <em>P. maculosus</em> leg and head, respectively. This suggests that gene duplications and/or alternative splicing events might have shaped gene evolutions in these two species. Furthermore, the transcriptome percentages when transcripts were considered were even lower, reiterating the potential contribution of splicing to the evolution of gene expression in the two species.</p>
<figure class="fig xbox font-sm" id="pone.0330170.g005"><h4 class="obj_head">Fig 5. <em>Pagurus</em> gene expression evolution.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367144_pone.0330170.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5288/12367144/19d204bbd4a1/pone.0330170.g005.jpg" loading="lazy" height="671" width="759" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330170.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>A. The percentages of <em>P. maculosus</em> and <em>P. lanuginosus</em> reads that can be mapped to the genes of other species. Close to 100% percent of the reads could be mapped to the transcripts of the other species. However, when genes/transcripts with one-to-one relationships were used, the percentages dropped. <strong>B</strong>. 2-D hexagonal binning plots showing the relationships between gene expression patterns of the <em>P. maculosus</em> (PM) and <em>P. lanuginosus</em> (PL) in the head (upper panel) and the leg (lower panel). r = Pearson’s correlation coefficient while the red lines represent the linear regression lines. Log<sub>2</sub>-transfromed TMM-normalized expressions were used for the plots C. Volcano plots showing differential gene expression analyses between <em>P. maculosus</em> (PM) and <em>P. lanuginosus</em> (PL) in the head (upper) and leg (lower). D. Venn diagram showing genes that were consistently upregulated in <em>P. maculosus</em> (PM, left panel) and <em>P. lanuginosus</em> (PL, right panel) head and leg. E. Percentages of pigmentation genes in head (upper) and leg (lower) differentially expressed genes. P values were computed using Fisher exact tests. F. Gene ontology enrichment analyses for genes upregulated in <em>P. maculosus</em> leg. The bar color corresponded to the Bonferroni-corrected hypergeometry p value.</p></figcaption></figure><p>Having established the high sequence similarity and percentages of transcriptomes in the two species, we performed expression quantification again using <em>P. lanuginosus</em> reference. We clustered the samples using principal component analyses and heatmaps to identify the two most consistent samples. Using the average expression of the most consistent duplicates, we found that the expression correlation in the head (Pearson’s correlation coefficient = 0.76) of the two species was higher than the leg correlation (Pearson’s correlation coefficient = 0.52), suggesting higher expression evolution in the genes expressed in the leg (<a href="#pone.0330170.g005" class="usa-link">Fig 5B</a>). When only the predicted protein-coding genes were used for the correlations (<a href="#pone.0330170.s001" class="usa-link">S1A Fig</a>), the correlations became stronger (Pearson’s correlation coefficients were 0.84 and 0.60, respectively for head and leg), suggesting divergence expression in the legs across the two species. Interestingly, the trend of higher head correlation persisted when only positively selected genes were used (<a href="#pone.0330170.s001" class="usa-link">S1B Fig</a>). Differential gene expression analyses further showed more differentially expressed genes in the leg, compared to the head (<a href="#pone.0330170.g005" class="usa-link">Fig 5C</a>), further confirming more divergent expression in the leg and conserved expression in the head across these two species. While 4,985 differentially expressed genes (DEGs) were found in the head, 7,585 DEGs were found in the leg. Also, the patterns of head DEGs were similar across the two species (Fisher exact p value = 0.155) while more genes were upregulated in <em>P. maculosus</em> leg (Fisher exact p value = 3.66 × 10<sup>-22</sup>) (<a href="#pone.0330170.g005" class="usa-link">Figs 5C</a> and <a href="#pone.0330170.g005" class="usa-link">5D</a>), suggesting tissue-specific evolution of gene expression patterns.</p>
<p>We next asked if the expressions of the same set of genes evolved in both head and leg. Log-fold changes for differentially expressed genes in leg and head were correlated (Spearman’s rank coefficient = 0.54) (<a href="#pone.0330170.s001" class="usa-link">S1C Fig</a>). For genes that were significantly differentially expressed in both leg and head, the patterns of differential expression were mostly consistent. For genes with higher expression in <em>P. maculosus,</em> 896 were found to be common to both head and leg differential expression (<a href="#pone.0330170.g005" class="usa-link">Fig 5D</a>). However, a larger proportion of the genes were found to be specific to head or leg comparison. Similar patterns were found for genes with higher expression in <em>P. lanuginosus</em>, suggesting that <em>Pagurus</em> gene expression evolution involved both shared and tissue-specific components.</p>
<p>To identify the potential functions of the differentially expressed genes, we checked the enrichments of pigmentation genes. Head DEGs included 149 <em>P. maculosus</em>-enriched pigmentation genes and 167 <em>P. lanuginosus</em>-enriched pigmentation genes (<a href="#pone.0330170.g005" class="usa-link">Fig 5E</a>). However, there was no significant bias in pigmentation genes differentially expressed in the head across the two species (p value = 0.64). More pigmentation genes were found in the leg differentially expressed genes. Importantly, more pigmentation genes were found to have higher expression in <em>P. lanuginosus</em> leg (p value = 1.76 × 10<sup>−08</sup>). We checked the enrichment of positively selected genes in deferentially expressed gene sets but found no bias in either of the species (<a href="#pone.0330170.s001" class="usa-link">S1D Fig</a>), suggesting that positive selection on gene expression was not specifically biased to either species.</p>
<p>Finally, we performed gene ontology enrichments for the differentially expressed genes. As the assembled genes have not been annotated, we first performed GO annotation of the genes using InterProScan 5 to predict the likely gene functions and the potential GO terms for the assembled protein-coding transcripts. Of the 55,714 <em>P. lanuginosus</em> with InterProScan annotation, 29,186 could be mapped to at least one GO term (<a href="#pone.0330170.s006" class="usa-link">S5 Table</a>). For <em>P. maculosus</em>, 23,703 of the 45,038 annotated transcripts could be mapped to at least one GO term. In total, 5,916 (<em>P. lanuginosus</em>) and 5,326 (<em>P. maculosus</em>) GO terms were found.</p>
<p>GO enrichment analyses showed that the largest enrichments were found in the genes that were upregulated in <em>P. maculosus</em> leg (<a href="#pone.0330170.g005" class="usa-link">Fig 5F</a>). Gene involved in extracellular activities were significantly enriched. Additionally, genes involved in the chitin-based larval cuticle and chitin binding were upregulated in <em>P. maculosus</em> leg. On the contrary, genes upregulated in <em>P. lanuginosus</em> leg were enriched in the nucleus, and in peptidase inhibitor and catalytic activities (<a href="#pone.0330170.s001" class="usa-link">S1E Fig</a>). Next, we checked the GO enrichment of <em>P. maculosus</em>-enriched head DEGs and found that the genes were enriched in ion exchange (<a href="#pone.0330170.s001" class="usa-link">S1F Fig</a>). Finally, we checked the GO enrichment of <em>P. lanuginosus</em>-upregulated head genes and found that genes involved in sulfation, sulfotransferase and transmembrane transporter activities were enriched (<a href="#pone.0330170.s001" class="usa-link">S1E Fig</a>). These results showed that genes upregulated in different species and different tissues performed different functions. Taken together, the results highlight how gene expressions have evolved after the divergence of <em>P. maculosus and P. lanuginosus.</em></p></section></section><section id="sec020"><h2 class="pmc_sec_title">Discussion</h2>
<p><em>Pagurus maculosus</em> and <em>P. lanuginosus</em> are two intertidal hermit crab species found in the Japanese water. They were initially thought to be color morphs of the same species. However, subsequent studies demonstrated that they are two separate species with unique features [<a href="#pone.0330170.ref007" class="usa-link" aria-describedby="pone.0330170.ref007">7</a>,<a href="#pone.0330170.ref009" class="usa-link" aria-describedby="pone.0330170.ref009">9</a>–<a href="#pone.0330170.ref011" class="usa-link" aria-describedby="pone.0330170.ref011">11</a>,<a href="#pone.0330170.ref021" class="usa-link" aria-describedby="pone.0330170.ref021">21</a>]. In this study, we report the analyses of the head (cephalothorax) and leg (pereopod) transcriptomes of the two species. We performed independent <em>de novo</em> transcript assembly for the two species, and confirmed that our transcript assembly did not lead to substantial loss of transcriptome data. Although almost all reads of one species could be mapped to the transcripts of the other species, a sizable proportion of the transcripts did not have one-to-one correspondence between the two species. Also, consideration at gene levels had higher proportion of one-to-one correspondence between the two species. These suggest that gene duplication and alternative splicing have played important roles in shaping the evolution of gene expressions in the two species.</p>
<p>We quantified the genetic distance between the two species. Using Yang-Nielsen method [<a href="#pone.0330170.ref042" class="usa-link" aria-describedby="pone.0330170.ref042">42</a>], the synonymous and nonsynonymous substitution rates for the two species were estimated to be 2.46% and 0.66%, respectively. These values were lower than 3.4–4.1% reported from nucleotide sequences of selected mitochondrial loci [<a href="#pone.0330170.ref021" class="usa-link" aria-describedby="pone.0330170.ref021">21</a>]. This discrepancy might be due to the differences in the evolutionary rates of mitochondrial and nuclear genes [<a href="#pone.0330170.ref062" class="usa-link" aria-describedby="pone.0330170.ref062">62</a>]. In addition, while the mitochondrial value was computed using Kimura 2-parameter on mitochondrial regions without segregation into synonymous and nonsynonymous rates, the evolutionary distances computed in this study were computed from predicted protein-coding transcripts. Based on synonymous and nonsynonymous substitutions, majority of the genes were found to undergo purifying selection, and only 5% were found to be undergoing positive selection. Interestingly, we found that substantial number of genes under selection are undergoing slightly neutral evolution [<a href="#pone.0330170.ref061" class="usa-link" aria-describedby="pone.0330170.ref061">61</a>,<a href="#pone.0330170.ref063" class="usa-link" aria-describedby="pone.0330170.ref063">63</a>]. This observation is interesting as the two species are sometimes found in the same ecosystem, suggesting that the two species might not have been subjected to drastically different pressures. The role of geographical isolation in shaping speciation and molecular evolution after speciation would be an interesting future study.</p>
<p>The search of public databases revealed that the closest annotated species to <em>P. lanuginosus</em> was the porcelain crab, <em>Pet. cinctipes,</em> which was estimated to have diverged around 250 MYA. However, our analyses revealed that the pair of <em>P. lanuginosus</em> and <em>P. maculosus</em> diverged from <em>Par. camtschaticus</em> about 36.1–59.8 MYA. These suggest that species closer to <em>Pagurus</em> have not been well annotated or their annotations have not been integrated into public protein databases. Supporting the later possibility, indeed transcriptome data of a number of species have recently been published [<a href="#pone.0330170.ref023" class="usa-link" aria-describedby="pone.0330170.ref023">23</a>,<a href="#pone.0330170.ref064" class="usa-link" aria-describedby="pone.0330170.ref064">64</a>–<a href="#pone.0330170.ref066" class="usa-link" aria-describedby="pone.0330170.ref066">66</a>]. Using the transcriptome of multiple related species, we estimated the divergence time of <em>P. lanuginosus</em> and <em>P. maculosus</em> to be 6.24 MYA (4.43–8.16; 95% confidence interval). This divergence time gives the rate of synonymous substitutions of 3.9 × 10<sup>−9</sup>, a rate similar to what was found in mammals [<a href="#pone.0330170.ref046" class="usa-link" aria-describedby="pone.0330170.ref046">46</a>]. Consistent with the previous reports that genus <em>Pagurus</em> is not monophyletic [<a href="#pone.0330170.ref004" class="usa-link" aria-describedby="pone.0330170.ref004">4</a>,<a href="#pone.0330170.ref009" class="usa-link" aria-describedby="pone.0330170.ref009">9</a>,<a href="#pone.0330170.ref067" class="usa-link" aria-describedby="pone.0330170.ref067">67</a>,<a href="#pone.0330170.ref068" class="usa-link" aria-describedby="pone.0330170.ref068">68</a>], we showed that the red king crab, <em>Par. camtschaticus</em> is more closely related to <em>P. maculosus</em> than <em>P. longicarpus</em>.</p>
<p>Gene expression evolution was found to be different across tissues with the head having higher correlation, suggesting slower gene expression evolution in the head when compared to the leg. This observation was similar to mammalian studies that investigated conserved noncoding sequences to infer that gene expression evolution varied across tissues [<a href="#pone.0330170.ref069" class="usa-link" aria-describedby="pone.0330170.ref069">69</a>–<a href="#pone.0330170.ref071" class="usa-link" aria-describedby="pone.0330170.ref071">71</a>]. Interestingly, we found that gene expression evolution was not symmetric. Genes involved in pigmentation were found to be significantly upregulated in <em>P. lanuginosus.</em> We found that genes upregulated in <em>P. maculosus</em> leg were enriched in extracellular space, larval cuticle, chitin binding and extracellular matrix, suggesting relationships between the gene expression changes and previously reported morphological changes [<a href="#pone.0330170.ref007" class="usa-link" aria-describedby="pone.0330170.ref007">7</a>,<a href="#pone.0330170.ref010" class="usa-link" aria-describedby="pone.0330170.ref010">10</a>,<a href="#pone.0330170.ref011" class="usa-link" aria-describedby="pone.0330170.ref011">11</a>]. Taken together, this study uncovers the molecular changes that have ensued the divergence of <em>P. maculosus</em> and <em>P. lanuginosus</em> and the potential relationships with the reported morphological changes.</p></section><section id="sec021"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pone.0330170.s001"><div class="caption p">
<span>S1 Fig. Gene expression evolution and the nature of genes enriched across the species.</span><p>A. 2-D hexagonal binning plots showing the relationships between protein-coding gene expression patterns of the <em>P. maculosus</em> (PM) and <em>P. lanuginosus</em> (PL) in the head (upper panel) and the leg (lower panel). B. 2-D hexagonal binning plots showing the relationships between the positively selected protein-coding gene expression patterns of the <em>P. maculosus</em> (PM) and <em>P. lanuginosus</em> (PL) in the head (upper panel) and the leg (lower panel). For A and B, r = Pearson’s correlation coefficient while the red lines represent the linear regression lines. Log<sub>2</sub>-transfromed TMM-normalized expressions were used for the plots C. Scattered plot showing the relationship between the log-fold changes of genes differentially expressed in head and/or leg. Each dot represents a gene. Negative values represent <em>P. lanuginosus</em> enrichment while positive values represent <em>P. maculosus</em> enrichment. Rho = Spearman’r correlation coefficient. D. Species bias of positively selected differentially expressed head (upper) and leg (lower) genes. P values were computed using Fisher exact tests. Gene ontology enrichments of genes are upregulated in <em>P. lanuginosus</em> leg (E), <em>P. maculosus</em> head (F) and <em>P. lanuginosus</em> head (G). The bar color corresponded to the Bonferroni-corrected hypergeometry p value.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s001.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s001.tif</a><sup> (2.4MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330170.s002"><div class="caption p">
<span>S1 Table. Details of Transcriptome sequencing.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s002.xlsx</a><sup> (10.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330170.s003"><div class="caption p">
<span>S2 Table. Number of homologous transcripts and genes in different databases.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s003.xlsx</a><sup> (10.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330170.s004"><div class="caption p">
<span>S3 Table. Calibrations used for divergence time estimates.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s004.xlsx</a><sup> (10.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330170.s005"><div class="caption p">
<span>S4 Table. The list of known pigmentation genes.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s005.xlsx</a><sup> (41.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0330170.s006"><div class="caption p">
<span>S5 Table. InterproScan annotation statistics for the assembled transcripts.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s006.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s006.xlsx</a><sup> (10.1KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors are thankful to Mr. Nakano, R., Ms. Nakano, M., and the members of SMBL for their help in collecting the study animals.</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>All data generated in the study have been made publicly available and deposited to the DNA Data Bank of Japan under the project accession number PRJDB35803, and can be accessed on the NCBI database (<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJDB35803/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/PRJDB35803/</a>). All short-read data have been described in S1 Table.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This study was supported by the funding of Research Organization of Information and Systems (ROIS) for special collaboration research student program of National Institute of Genetics (NIG) and Kyoto University, Japan awarded to ZS, and by the Operating Expenses (運営費) from Kyoto University, Japan awarded to AA and TN. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0330170.ref001">
<span class="label">1.</span><cite>Fabricius JC. Systema entomologiae: sistens insectorvm classes, ordines, genera, species, adiectis synonymis, locis, descriptionibvs, observationibvs.  Systema entomologiae: sistens insectorvm classes, ordines, genera, species, adiectis synonymis, locis, descriptionibvs, observationibvs/ Io. Christ. Fabricii. Flensbvrgi et Lipsiae: In Officina Libraria Kortii; 1775. doi: 10.5962/bhl.title.36510</cite> [<a href="https://doi.org/10.5962/bhl.title.36510" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Systema%20entomologiae:%20sistens%20insectorvm%20classes,%20ordines,%20genera,%20species,%20adiectis%20synonymis,%20locis,%20descriptionibvs,%20observationibvs/%20Io.%20Christ.%20Fabricii.%20Flensbvrgi%20et%20Lipsiae:%20In%20Officina%20Libraria%20Kortii&amp;title=Systema%20entomologiae:%20sistens%20insectorvm%20classes,%20ordines,%20genera,%20species,%20adiectis%20synonymis,%20locis,%20descriptionibvs,%20observationibvs.&amp;author=JC%20Fabricius&amp;publication_year=1775&amp;doi=10.5962/bhl.title.36510&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref002">
<span class="label">2.</span><cite>Young AM, Torres C, Mack JE, Cunningham CW. Morphological and genetic evidence for vicariance and refugium in Atlantic and Gulf of Mexico populations of the hermit crab Pagurus longicarpus. Marine Biology. 2002;140(5):1059–66. doi: 10.1007/s00227-002-0780-2</cite> [<a href="https://doi.org/10.1007/s00227-002-0780-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Biology&amp;title=Morphological%20and%20genetic%20evidence%20for%20vicariance%20and%20refugium%20in%20Atlantic%20and%20Gulf%20of%20Mexico%20populations%20of%20the%20hermit%20crab%20Pagurus%20longicarpus&amp;author=AM%20Young&amp;author=C%20Torres&amp;author=JE%20Mack&amp;author=CW%20Cunningham&amp;volume=140&amp;issue=5&amp;publication_year=2002&amp;pages=1059-66&amp;doi=10.1007/s00227-002-0780-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref003">
<span class="label">3.</span><cite>Knowlton N. Cryptic and Sibling Species among the Decapod Crustacea. Journal of Crustacean Biology. 1986;6(3):356. doi: 10.2307/1548175</cite> [<a href="https://doi.org/10.2307/1548175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Crustacean%20Biology&amp;title=Cryptic%20and%20Sibling%20Species%20among%20the%20Decapod%20Crustacea&amp;author=N%20Knowlton&amp;volume=6&amp;issue=3&amp;publication_year=1986&amp;pages=356&amp;doi=10.2307/1548175&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref004">
<span class="label">4.</span><cite>Mantelatto FL, Pardo LM, Pileggi LG, Felder DL. Taxonomic re-examination of the hermit crab species Pagurus forceps and Pagurus comptus (Decapoda: Paguridae) by molecular analysis. Zootaxa. 2009;2133(1). doi: 10.11646/zootaxa.2133.1.2</cite> [<a href="https://doi.org/10.11646/zootaxa.2133.1.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Zootaxa&amp;title=Taxonomic%20re-examination%20of%20the%20hermit%20crab%20species%20Pagurus%20forceps%20and%20Pagurus%20comptus%20(Decapoda:%20Paguridae)%20by%20molecular%20analysis&amp;author=FL%20Mantelatto&amp;author=LM%20Pardo&amp;author=LG%20Pileggi&amp;author=DL%20Felder&amp;volume=2133&amp;issue=1&amp;publication_year=2009&amp;doi=10.11646/zootaxa.2133.1.2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref005">
<span class="label">5.</span><cite>Matzen da Silva J, Creer S, dos Santos A, Costa AC, Cunha MR, Costa FO, et al. Systematic and evolutionary insights derived from mtDNA COI barcode diversity in the Decapoda (Crustacea: Malacostraca). PLoS One. 2011;6(5):e19449. doi: 10.1371/journal.pone.0019449

</cite> [<a href="https://doi.org/10.1371/journal.pone.0019449" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3093375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21589909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Systematic%20and%20evolutionary%20insights%20derived%20from%20mtDNA%20COI%20barcode%20diversity%20in%20the%20Decapoda%20(Crustacea:%20Malacostraca)&amp;author=J%20Matzen%20da%20Silva&amp;author=S%20Creer&amp;author=A%20dos%20Santos&amp;author=AC%20Costa&amp;author=MR%20Cunha&amp;volume=6&amp;issue=5&amp;publication_year=2011&amp;pmid=21589909&amp;doi=10.1371/journal.pone.0019449&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref006">
<span class="label">6.</span><cite>De Haan W.
Crustacea. In: von Siebold PF, Fauna Japonica sive Descriptio Animalium, quae in Itinere per Japoniam, Jussu et Auspiciis Superiorum, qui Summum in India Batava Imperium tenent, Suscepto, Annis 1823–1830 Collegit, Notis, Observationibus e Adumbrationibus Illustravit. Von Siebold, editor. 1849.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Fauna%20Japonica%20sive%20Descriptio%20Animalium,%20quae%20in%20Itinere%20per%20Japoniam,%20Jussu%20et%20Auspiciis%20Superiorum,%20qui%20Summum%20in%20India%20Batava%20Imperium%20tenent,%20Suscepto,%20Annis%201823%E2%80%931830%20Collegit,%20Notis,%20Observationibus%20e%20Adumbrationibus%20Illustravit.%20Von%20Siebold,%20editor&amp;author=W.%20De%20Haan&amp;author=PF%20von%20Siebold&amp;publication_year=1849&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref007">
<span class="label">7.</span><cite>Komai T, Imafuku M. Redescription of Pagurus lanuginosus with the Establishment of a Neotype, and Description of a New Closely Related Species (Decapoda: Anomura: Paguridae). Journal of Crustacean Biology. 1996;16(4):782. doi: 10.2307/1549196</cite> [<a href="https://doi.org/10.2307/1549196" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Crustacean%20Biology&amp;title=Redescription%20of%20Pagurus%20lanuginosus%20with%20the%20Establishment%20of%20a%20Neotype,%20and%20Description%20of%20a%20New%20Closely%20Related%20Species%20(Decapoda:%20Anomura:%20Paguridae)&amp;author=T%20Komai&amp;author=M%20Imafuku&amp;volume=16&amp;issue=4&amp;publication_year=1996&amp;pages=782&amp;doi=10.2307/1549196&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref008">
<span class="label">8.</span><cite>Imafuku M, Ikeda H. On Two Types of the Hermit Crab Pagurus lanuginosus (Crustacea: Anomura: Paguridae). Publications of the Seto Marine Biological Laboratory; 1995. p. 36. doi: 10.5134/176246</cite> [<a href="https://doi.org/10.5134/176246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=On%20Two%20Types%20of%20the%20Hermit%20Crab%20Pagurus%20lanuginosus%20(Crustacea:%20Anomura:%20Paguridae).%20Publications%20of%20the%20Seto%20Marine%20Biological%20Laboratory&amp;author=M%20Imafuku&amp;author=H%20Ikeda&amp;publication_year=1995&amp;pages=36&amp;doi=10.5134/176246&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref009">
<span class="label">9.</span><cite>Sultana Z, Babarinde IA, Asakura A. Diversity and Molecular Phylogeny of Pagurid Hermit Crabs (Anomura: Paguridae: Pagurus). Diversity. 2022;14(2):141. doi: 10.3390/d14020141</cite> [<a href="https://doi.org/10.3390/d14020141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diversity&amp;title=Diversity%20and%20Molecular%20Phylogeny%20of%20Pagurid%20Hermit%20Crabs%20(Anomura:%20Paguridae:%20Pagurus)&amp;author=Z%20Sultana&amp;author=IA%20Babarinde&amp;author=A%20Asakura&amp;volume=14&amp;issue=2&amp;publication_year=2022&amp;pages=141&amp;doi=10.3390/d14020141&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref010">
<span class="label">10.</span><cite>Sultana Z, Asakura A. The complete larval development of Pagurus maculosus Komai &amp; Imafuku, 1996 (Decapoda, Anomura, Paguridae) reared in the laboratory, and a comparison with sympatric species. Zootaxa. 2015;3947(3):301–26. doi: 10.11646/zootaxa.3947.3.1

</cite> [<a href="https://doi.org/10.11646/zootaxa.3947.3.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25947739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Zootaxa&amp;title=The%20complete%20larval%20development%20of%20Pagurus%20maculosus%20Komai%20&amp;%20Imafuku,%201996%20(Decapoda,%20Anomura,%20Paguridae)%20reared%20in%20the%20laboratory,%20and%20a%20comparison%20with%20sympatric%20species&amp;author=Z%20Sultana&amp;author=A%20Asakura&amp;volume=3947&amp;issue=3&amp;publication_year=2015&amp;pages=301-26&amp;pmid=25947739&amp;doi=10.11646/zootaxa.3947.3.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref011">
<span class="label">11.</span><cite>Sultana Z, Asakura A. The complete larval development of Pagurus lanuginosus De Haan, 1849 (Decapoda, Anomura, Paguridae) reared in the laboratory, with emphasis on the post-larval stage. Zootaxa. 2015;3915: 206–32. doi: 10.11646/zootaxa.3915.2.2
</cite> [<a href="https://doi.org/10.11646/zootaxa.3915.2.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25662120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Zootaxa&amp;title=The%20complete%20larval%20development%20of%20Pagurus%20lanuginosus%20De%20Haan,%201849%20(Decapoda,%20Anomura,%20Paguridae)%20reared%20in%20the%20laboratory,%20with%20emphasis%20on%20the%20post-larval%20stage&amp;author=Z%20Sultana&amp;author=A%20Asakura&amp;volume=3915&amp;publication_year=2015&amp;pages=206-32&amp;pmid=25662120&amp;doi=10.11646/zootaxa.3915.2.2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref012">
<span class="label">12.</span><cite>Akinwole MT, Babarinde IA. Assessing tissue lysis with sodium dodecyl sulphate for DNA extraction from frozen animal tissue. Journal of Forensic Research. 2019;10.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Forensic%20Research&amp;title=Assessing%20tissue%20lysis%20with%20sodium%20dodecyl%20sulphate%20for%20DNA%20extraction%20from%20frozen%20animal%20tissue&amp;author=MT%20Akinwole&amp;author=IA%20Babarinde&amp;volume=10&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref013">
<span class="label">13.</span><cite>Satam H, Joshi K, Mangrolia U, Waghoo S, Zaidi G, Rawool S, et al. Next-Generation Sequencing Technology: Current Trends and Advancements. Biology (Basel). 2023;12(7):997. doi: 10.3390/biology12070997

</cite> [<a href="https://doi.org/10.3390/biology12070997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10376292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37508427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biology%20(Basel)&amp;title=Next-Generation%20Sequencing%20Technology:%20Current%20Trends%20and%20Advancements&amp;author=H%20Satam&amp;author=K%20Joshi&amp;author=U%20Mangrolia&amp;author=S%20Waghoo&amp;author=G%20Zaidi&amp;volume=12&amp;issue=7&amp;publication_year=2023&amp;pages=997&amp;pmid=37508427&amp;doi=10.3390/biology12070997&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref014">
<span class="label">14.</span><cite>Berggren MS. Decapod crustacean phylogenetics. Marine Biology Research. 2010;6(2):221–2. doi: 10.1080/17451001003604796</cite> [<a href="https://doi.org/10.1080/17451001003604796" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Biology%20Research&amp;title=Decapod%20crustacean%20phylogenetics&amp;author=MS%20Berggren&amp;volume=6&amp;issue=2&amp;publication_year=2010&amp;pages=221-2&amp;doi=10.1080/17451001003604796&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref015">
<span class="label">15.</span><cite>Toon A, Finley M, Staples J, Crandall K. Decapod Phylogenetics and Molecular Evolution. Crustacean Issues. CRC Press. 2009. p. 15–29. doi: 10.1201/9781420092592-c2</cite> [<a href="https://doi.org/10.1201/9781420092592-c2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crustacean%20Issues.%20CRC%20Press&amp;title=Decapod%20Phylogenetics%20and%20Molecular%20Evolution&amp;author=A%20Toon&amp;author=M%20Finley&amp;author=J%20Staples&amp;author=K%20Crandall&amp;publication_year=2009&amp;pages=15-29&amp;doi=10.1201/9781420092592-c2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref016">
<span class="label">16.</span><cite>Porter ML, Pérez-Losada M, Crandall KA. Model-based multi-locus estimation of decapod phylogeny and divergence times. Mol Phylogenet Evol. 2005;37(2):355–69. doi: 10.1016/j.ympev.2005.06.021

</cite> [<a href="https://doi.org/10.1016/j.ympev.2005.06.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16112880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Phylogenet%20Evol&amp;title=Model-based%20multi-locus%20estimation%20of%20decapod%20phylogeny%20and%20divergence%20times&amp;author=ML%20Porter&amp;author=M%20P%C3%A9rez-Losada&amp;author=KA%20Crandall&amp;volume=37&amp;issue=2&amp;publication_year=2005&amp;pages=355-69&amp;pmid=16112880&amp;doi=10.1016/j.ympev.2005.06.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref017">
<span class="label">17.</span><cite>Wolfe JM, Breinholt JW, Crandall KA, Lemmon AR, Lemmon EM, Timm LE, et al. A phylogenomic framework, evolutionary timeline and genomic resources for comparative studies of decapod crustaceans. Proc Biol Sci. 2019;286(1901):20190079. doi: 10.1098/rspb.2019.0079

</cite> [<a href="https://doi.org/10.1098/rspb.2019.0079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6501934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31014217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Biol%20Sci&amp;title=A%20phylogenomic%20framework,%20evolutionary%20timeline%20and%20genomic%20resources%20for%20comparative%20studies%20of%20decapod%20crustaceans&amp;author=JM%20Wolfe&amp;author=JW%20Breinholt&amp;author=KA%20Crandall&amp;author=AR%20Lemmon&amp;author=EM%20Lemmon&amp;volume=286&amp;issue=1901&amp;publication_year=2019&amp;pages=20190079&amp;pmid=31014217&amp;doi=10.1098/rspb.2019.0079&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref018">
<span class="label">18.</span><cite>Blaxter M. Molecular systematics: Counting angels with DNA. Nature. 2003;421(6919):122–4. doi: 10.1038/421122a

</cite> [<a href="https://doi.org/10.1038/421122a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12520286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Molecular%20systematics:%20Counting%20angels%20with%20DNA&amp;author=M%20Blaxter&amp;volume=421&amp;issue=6919&amp;publication_year=2003&amp;pages=122-4&amp;pmid=12520286&amp;doi=10.1038/421122a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref019">
<span class="label">19.</span><cite>Antil S, Abraham JS, Sripoorna S, Maurya S, Dagar J, Makhija S, et al. DNA barcoding, an effective tool for species identification: a review. Mol Biol Rep. 2023;50(1):761–75. doi: 10.1007/s11033-022-08015-7

</cite> [<a href="https://doi.org/10.1007/s11033-022-08015-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36308581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Rep&amp;title=DNA%20barcoding,%20an%20effective%20tool%20for%20species%20identification:%20a%20review&amp;author=S%20Antil&amp;author=JS%20Abraham&amp;author=S%20Sripoorna&amp;author=S%20Maurya&amp;author=J%20Dagar&amp;volume=50&amp;issue=1&amp;publication_year=2023&amp;pages=761-75&amp;pmid=36308581&amp;doi=10.1007/s11033-022-08015-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref020">
<span class="label">20.</span><cite>Guo M, Yuan C, Tao L, Cai Y, Zhang W. Life barcoded by DNA barcodes. Conservation Genet Resour. 2022;14(4):351–65. doi: 10.1007/s12686-022-01291-2</cite> [<a href="https://doi.org/10.1007/s12686-022-01291-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9377290/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35991367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Conservation%20Genet%20Resour&amp;title=Life%20barcoded%20by%20DNA%20barcodes&amp;author=M%20Guo&amp;author=C%20Yuan&amp;author=L%20Tao&amp;author=Y%20Cai&amp;author=W%20Zhang&amp;volume=14&amp;issue=4&amp;publication_year=2022&amp;pages=351-65&amp;pmid=35991367&amp;doi=10.1007/s12686-022-01291-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref021">
<span class="label">21.</span><cite>Sultana Z, Asakura A, Kinjo S, Nozawa M, Nakano T, Ikeo K. Molecular phylogeny of ten intertidal hermit crabs of the genus Pagurus inferred from multiple mitochondrial genes, with special emphasis on the evolutionary relationship of Pagurus lanuginosus and Pagurus maculosus. Genetica. 2018;146(4–5):369–81. doi: 10.1007/s10709-018-0029-8

</cite> [<a href="https://doi.org/10.1007/s10709-018-0029-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29992482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetica&amp;title=Molecular%20phylogeny%20of%20ten%20intertidal%20hermit%20crabs%20of%20the%20genus%20Pagurus%20inferred%20from%20multiple%20mitochondrial%20genes,%20with%20special%20emphasis%20on%20the%20evolutionary%20relationship%20of%20Pagurus%20lanuginosus%20and%20Pagurus%20maculosus&amp;author=Z%20Sultana&amp;author=A%20Asakura&amp;author=S%20Kinjo&amp;author=M%20Nozawa&amp;author=T%20Nakano&amp;volume=146&amp;publication_year=2018&amp;pages=369-81&amp;pmid=29992482&amp;doi=10.1007/s10709-018-0029-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref022">
<span class="label">22.</span><cite>Kurosawa TM. Japanese regulation of laboratory animal care with 3Rs. AATEX. 2007;14:317–21.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=AATEX&amp;title=Japanese%20regulation%20of%20laboratory%20animal%20care%20with%203Rs&amp;author=TM%20Kurosawa&amp;volume=14&amp;publication_year=2007&amp;pages=317-21&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref023">
<span class="label">23.</span><cite>Nong W, Chai ZYH, Jiang X, Qin J, Ma KY, Chan KM, et al. A crustacean annotated transcriptome (CAT) database. BMC Genomics. 2020;21(1):32. doi: 10.1186/s12864-019-6433-3

</cite> [<a href="https://doi.org/10.1186/s12864-019-6433-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6953184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31918660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=A%20crustacean%20annotated%20transcriptome%20(CAT)%20database&amp;author=W%20Nong&amp;author=ZYH%20Chai&amp;author=X%20Jiang&amp;author=J%20Qin&amp;author=KY%20Ma&amp;volume=21&amp;issue=1&amp;publication_year=2020&amp;pages=32&amp;pmid=31918660&amp;doi=10.1186/s12864-019-6433-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref024">
<span class="label">24.</span><cite>Grabherr MG, Haas BJ, Yassour M, Levin JZ, Thompson DA, Amit I. Trinity: reconstructing a full-length transcriptome without a genome from RNA-Seq data HHS Public Access. Nat Biotechnol. 2011;29.</cite> [<a href="https://doi.org/10.1038/nbt.1883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3571712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21572440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Trinity:%20reconstructing%20a%20full-length%20transcriptome%20without%20a%20genome%20from%20RNA-Seq%20data%20HHS%20Public%20Access&amp;author=MG%20Grabherr&amp;author=BJ%20Haas&amp;author=M%20Yassour&amp;author=JZ%20Levin&amp;author=DA%20Thompson&amp;volume=29&amp;publication_year=2011&amp;pmid=21572440&amp;doi=10.1038/nbt.1883&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref025">
<span class="label">25.</span><cite>Haas BJ, Papanicolaou A, Yassour M, Grabherr M, Blood PD, Bowden J, et al. De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis. Nat Protoc. 2013;8(8):1494–512. doi: 10.1038/nprot.2013.084

</cite> [<a href="https://doi.org/10.1038/nprot.2013.084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3875132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23845962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&amp;title=De%20novo%20transcript%20sequence%20reconstruction%20from%20RNA-seq%20using%20the%20Trinity%20platform%20for%20reference%20generation%20and%20analysis&amp;author=BJ%20Haas&amp;author=A%20Papanicolaou&amp;author=M%20Yassour&amp;author=M%20Grabherr&amp;author=PD%20Blood&amp;volume=8&amp;issue=8&amp;publication_year=2013&amp;pages=1494-512&amp;pmid=23845962&amp;doi=10.1038/nprot.2013.084&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref026">
<span class="label">26.</span><cite>Li B, Dewey CN. RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome. BMC Bioinformatics. 2011;12:323. doi: 10.1186/1471-2105-12-323

</cite> [<a href="https://doi.org/10.1186/1471-2105-12-323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3163565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21816040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=RSEM:%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome&amp;author=B%20Li&amp;author=CN%20Dewey&amp;volume=12&amp;publication_year=2011&amp;pages=323&amp;pmid=21816040&amp;doi=10.1186/1471-2105-12-323&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref027">
<span class="label">27.</span><cite>Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9(4):357–9. doi: 10.1038/nmeth.1923

</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B%20Langmead&amp;author=SL%20Salzberg&amp;volume=9&amp;issue=4&amp;publication_year=2012&amp;pages=357-9&amp;pmid=22388286&amp;doi=10.1038/nmeth.1923&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref028">
<span class="label">28.</span><cite>Chen S, Zhou Y, Chen Y, Gu J. fastp: an ultra-fast all-in-one FASTQ preprocessor. Bioinformatics. 2018;34(17):i884–90. doi: 10.1093/bioinformatics/bty560

</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor&amp;author=S%20Chen&amp;author=Y%20Zhou&amp;author=Y%20Chen&amp;author=J%20Gu&amp;volume=34&amp;issue=17&amp;publication_year=2018&amp;pmid=30423086&amp;doi=10.1093/bioinformatics/bty560&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref029">
<span class="label">29.</span><cite>Pruitt KD, Tatusova T, Maglott DR. NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acids Res. 2007;35(Database issue):D61-5. doi: 10.1093/nar/gkl842

</cite> [<a href="https://doi.org/10.1093/nar/gkl842" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1716718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17130148/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=NCBI%20reference%20sequences%20(RefSeq):%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes,%20transcripts%20and%20proteins&amp;author=KD%20Pruitt&amp;author=T%20Tatusova&amp;author=DR%20Maglott&amp;volume=35&amp;publication_year=2007&amp;pmid=17130148&amp;doi=10.1093/nar/gkl842&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref030">
<span class="label">30.</span><cite>O’Leary NA, Wright MW, Brister JR, Ciufo S, Haddad D, McVeigh R, et al. Reference sequence (RefSeq) database at NCBI: current status, taxonomic expansion, and functional annotation. Nucleic Acids Res. 2015;44(D1):D733–45. doi: 10.1093/nar/gkv1189</cite> [<a href="https://doi.org/10.1093/nar/gkv1189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4702849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26553804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Reference%20sequence%20(RefSeq)%20database%20at%20NCBI:%20current%20status,%20taxonomic%20expansion,%20and%20functional%20annotation&amp;author=NA%20O%E2%80%99Leary&amp;author=MW%20Wright&amp;author=JR%20Brister&amp;author=S%20Ciufo&amp;author=D%20Haddad&amp;volume=44&amp;publication_year=2015&amp;pmid=26553804&amp;doi=10.1093/nar/gkv1189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref031">
<span class="label">31.</span><cite>Suzek BE, Huang H, McGarvey P, Mazumder R, Wu CH. UniRef: comprehensive and non-redundant UniProt reference clusters. Bioinformatics. 2007;23(10):1282–8. doi: 10.1093/bioinformatics/btm098

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17379688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=UniRef:%20comprehensive%20and%20non-redundant%20UniProt%20reference%20clusters&amp;author=BE%20Suzek&amp;author=H%20Huang&amp;author=P%20McGarvey&amp;author=R%20Mazumder&amp;author=CH%20Wu&amp;volume=23&amp;issue=10&amp;publication_year=2007&amp;pages=1282-8&amp;pmid=17379688&amp;doi=10.1093/bioinformatics/btm098&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref032">
<span class="label">32.</span><cite>Bateman A, Martin MJ, Orchard S, Magrane M, Ahmad S, Alpi E, et al. UniProt: the Universal Protein Knowledgebase in 2023. Nucleic Acids Res. 2023;51(D1):D523–31. doi: 10.1093/nar/gkac1052

</cite> [<a href="https://doi.org/10.1093/nar/gkac1052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36408920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=UniProt:%20the%20Universal%20Protein%20Knowledgebase%20in%202023&amp;author=A%20Bateman&amp;author=MJ%20Martin&amp;author=S%20Orchard&amp;author=M%20Magrane&amp;author=S%20Ahmad&amp;volume=51&amp;publication_year=2023&amp;pmid=36408920&amp;doi=10.1093/nar/gkac1052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref033">
<span class="label">33.</span><cite>Buchfink B, Xie C, Huson DH. Fast and sensitive protein alignment using DIAMOND. Nat Methods. 2015;12(1):59–60. doi: 10.1038/nmeth.3176

</cite> [<a href="https://doi.org/10.1038/nmeth.3176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25402007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Fast%20and%20sensitive%20protein%20alignment%20using%20DIAMOND&amp;author=B%20Buchfink&amp;author=C%20Xie&amp;author=DH%20Huson&amp;volume=12&amp;issue=1&amp;publication_year=2015&amp;pages=59-60&amp;pmid=25402007&amp;doi=10.1038/nmeth.3176&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref034">
<span class="label">34.</span><cite>Jones P, Binns D, Chang H-Y, Fraser M, Li W, McAnulla C, et al. InterProScan 5: genome-scale protein function classification. Bioinformatics. 2014;30(9):1236–40. doi: 10.1093/bioinformatics/btu031

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3998142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24451626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=InterProScan%205:%20genome-scale%20protein%20function%20classification&amp;author=P%20Jones&amp;author=D%20Binns&amp;author=H-Y%20Chang&amp;author=M%20Fraser&amp;author=W%20Li&amp;volume=30&amp;issue=9&amp;publication_year=2014&amp;pages=1236-40&amp;pmid=24451626&amp;doi=10.1093/bioinformatics/btu031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref035">
<span class="label">35.</span><cite>Baxter LL, Watkins-Chow DE, Pavan WJ, Loftus SK. A curated gene list for expanding the horizons of pigmentation biology. Pigment Cell Melanoma Res. 2019;32(3):348–58. doi: 10.1111/pcmr.12743

</cite> [<a href="https://doi.org/10.1111/pcmr.12743" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10413850/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30339321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pigment%20Cell%20Melanoma%20Res&amp;title=A%20curated%20gene%20list%20for%20expanding%20the%20horizons%20of%20pigmentation%20biology&amp;author=LL%20Baxter&amp;author=DE%20Watkins-Chow&amp;author=WJ%20Pavan&amp;author=SK%20Loftus&amp;volume=32&amp;issue=3&amp;publication_year=2019&amp;pages=348-58&amp;pmid=30339321&amp;doi=10.1111/pcmr.12743&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref036">
<span class="label">36.</span><cite>Gene Ontology Consortium, Aleksander SA, Balhoff J, Carbon S, Cherry JM, Drabkin HJ, et al. The Gene Ontology knowledgebase in 2023. Genetics. 2023;224(1):iyad031. doi: 10.1093/genetics/iyad031

</cite> [<a href="https://doi.org/10.1093/genetics/iyad031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10158837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36866529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The%20Gene%20Ontology%20knowledgebase%20in%202023&amp;author=Consortium%20Gene%20Ontology&amp;author=SA%20Aleksander&amp;author=J%20Balhoff&amp;author=S%20Carbon&amp;author=JM%20Cherry&amp;volume=224&amp;issue=1&amp;publication_year=2023&amp;pmid=36866529&amp;doi=10.1093/genetics/iyad031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref037">
<span class="label">37.</span><cite>Altschul SF, Madden TL, Schäffer AA, Zhang J, Zhang Z, Miller W, et al. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. 1997;25(17):3389–402. doi: 10.1093/nar/25.17.3389

</cite> [<a href="https://doi.org/10.1093/nar/25.17.3389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC146917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9254694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Gapped%20BLAST%20and%20PSI-BLAST:%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=SF%20Altschul&amp;author=TL%20Madden&amp;author=AA%20Sch%C3%A4ffer&amp;author=J%20Zhang&amp;author=Z%20Zhang&amp;volume=25&amp;issue=17&amp;publication_year=1997&amp;pages=3389-402&amp;pmid=9254694&amp;doi=10.1093/nar/25.17.3389&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref038">
<span class="label">38.</span><cite>Neyman J, Pearson ES. On the Use and Interpretation of Certain Test Criteria for Purposes of Statistical Inference: Part II. Biometrika. 1928;20A(3/4):263. doi: 10.2307/2332112</cite> [<a href="https://doi.org/10.2307/2332112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrika&amp;title=On%20the%20Use%20and%20Interpretation%20of%20Certain%20Test%20Criteria%20for%20Purposes%20of%20Statistical%20Inference:%20Part%20II&amp;author=J%20Neyman&amp;author=ES%20Pearson&amp;publication_year=1928&amp;pages=263&amp;doi=10.2307/2332112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref039">
<span class="label">39.</span><cite>Larkin MA, Blackshields G, Brown NP, Chenna R, McGettigan PA, McWilliam H, et al. Clustal W and Clustal X version 2.0. Bioinformatics. 2007;23(21):2947–8. doi: 10.1093/bioinformatics/btm404

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17846036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Clustal%20W%20and%20Clustal%20X%20version%202.0&amp;author=MA%20Larkin&amp;author=G%20Blackshields&amp;author=NP%20Brown&amp;author=R%20Chenna&amp;author=PA%20McGettigan&amp;volume=23&amp;issue=21&amp;publication_year=2007&amp;pages=2947-8&amp;pmid=17846036&amp;doi=10.1093/bioinformatics/btm404&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref040">
<span class="label">40.</span><cite>Tamura K, Stecher G, Kumar S. MEGA11: Molecular Evolutionary Genetics Analysis Version 11. Mol Biol Evol. 2021;38(7):3022–7. doi: 10.1093/molbev/msab120

</cite> [<a href="https://doi.org/10.1093/molbev/msab120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8233496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33892491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=MEGA11:%20Molecular%20Evolutionary%20Genetics%20Analysis%20Version%2011&amp;author=K%20Tamura&amp;author=G%20Stecher&amp;author=S%20Kumar&amp;volume=38&amp;issue=7&amp;publication_year=2021&amp;pages=3022-7&amp;pmid=33892491&amp;doi=10.1093/molbev/msab120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref041">
<span class="label">41.</span><cite>Simple methods for estimating the numbers of synonymous and nonsynonymous nucleotide substitutions. Molecular Biology and Evolution. 1986. doi: 10.1093/oxfordjournals.molbev.a040410</cite> [<a href="https://doi.org/10.1093/oxfordjournals.molbev.a040410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3444411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=Simple%20methods%20for%20estimating%20the%20numbers%20of%20synonymous%20and%20nonsynonymous%20nucleotide%20substitutions&amp;publication_year=1986&amp;pmid=3444411&amp;doi=10.1093/oxfordjournals.molbev.a040410&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref042">
<span class="label">42.</span><cite>Yang Z, Nielsen R. Estimating synonymous and nonsynonymous substitution rates under realistic evolutionary models. Mol Biol Evol. 2000;17(1):32–43. doi: 10.1093/oxfordjournals.molbev.a026236

</cite> [<a href="https://doi.org/10.1093/oxfordjournals.molbev.a026236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10666704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=Estimating%20synonymous%20and%20nonsynonymous%20substitution%20rates%20under%20realistic%20evolutionary%20models&amp;author=Z%20Yang&amp;author=R%20Nielsen&amp;volume=17&amp;issue=1&amp;publication_year=2000&amp;pages=32-43&amp;pmid=10666704&amp;doi=10.1093/oxfordjournals.molbev.a026236&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref043">
<span class="label">43.</span><cite>Yang Z. PAML 4: phylogenetic analysis by maximum likelihood. Mol Biol Evol. 2007;24(8):1586–91. doi: 10.1093/molbev/msm088

</cite> [<a href="https://doi.org/10.1093/molbev/msm088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17483113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=PAML%204:%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z%20Yang&amp;volume=24&amp;issue=8&amp;publication_year=2007&amp;pages=1586-91&amp;pmid=17483113&amp;doi=10.1093/molbev/msm088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref044">
<span class="label">44.</span><cite>dos Reis M, Yang Z. Approximate likelihood calculation on a phylogeny for Bayesian estimation of divergence times. Mol Biol Evol. 2011;28(7):2161–72. doi: 10.1093/molbev/msr045

</cite> [<a href="https://doi.org/10.1093/molbev/msr045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21310946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=Approximate%20likelihood%20calculation%20on%20a%20phylogeny%20for%20Bayesian%20estimation%20of%20divergence%20times&amp;author=M%20dos%20Reis&amp;author=Z%20Yang&amp;volume=28&amp;issue=7&amp;publication_year=2011&amp;pages=2161-72&amp;pmid=21310946&amp;doi=10.1093/molbev/msr045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref045">
<span class="label">45.</span><cite>Inoue JG, Miya M, Lam K, Tay B-H, Danks JA, Bell J, et al. Evolutionary origin and phylogeny of the modern holocephalans (Chondrichthyes: Chimaeriformes): a mitogenomic perspective. Mol Biol Evol. 2010;27(11):2576–86. doi: 10.1093/molbev/msq147

</cite> [<a href="https://doi.org/10.1093/molbev/msq147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20551041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=Evolutionary%20origin%20and%20phylogeny%20of%20the%20modern%20holocephalans%20(Chondrichthyes:%20Chimaeriformes):%20a%20mitogenomic%20perspective&amp;author=JG%20Inoue&amp;author=M%20Miya&amp;author=K%20Lam&amp;author=B-H%20Tay&amp;author=JA%20Danks&amp;volume=27&amp;issue=11&amp;publication_year=2010&amp;pages=2576-86&amp;pmid=20551041&amp;doi=10.1093/molbev/msq147&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref046">
<span class="label">46.</span><cite>Babarinde IA, Saitou N. The Dynamics, Causes, and Impacts of Mammalian Evolutionary Rates Revealed by the Analyses of Capybara Draft Genome Sequences. Genome Biol Evol. 2020;12(8):1444–58. doi: 10.1093/gbe/evaa157

</cite> [<a href="https://doi.org/10.1093/gbe/evaa157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7502209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32835375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol%20Evol&amp;title=The%20Dynamics,%20Causes,%20and%20Impacts%20of%20Mammalian%20Evolutionary%20Rates%20Revealed%20by%20the%20Analyses%20of%20Capybara%20Draft%20Genome%20Sequences&amp;author=IA%20Babarinde&amp;author=N%20Saitou&amp;volume=12&amp;issue=8&amp;publication_year=2020&amp;pages=1444-58&amp;pmid=32835375&amp;doi=10.1093/gbe/evaa157&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref047">
<span class="label">47.</span><cite>Bracken-Grissom HD, Ahyong ST, Wilkinson RD, Feldmann RM, Schweitzer CE, Breinholt JW, et al. The emergence of lobsters: phylogenetic relationships, morphological evolution and divergence time comparisons of an ancient group (decapoda: achelata, astacidea, glypheidea, polychelida). Syst Biol. 2014;63(4):457–79. doi: 10.1093/sysbio/syu008

</cite> [<a href="https://doi.org/10.1093/sysbio/syu008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24562813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Syst%20Biol&amp;title=The%20emergence%20of%20lobsters:%20phylogenetic%20relationships,%20morphological%20evolution%20and%20divergence%20time%20comparisons%20of%20an%20ancient%20group%20(decapoda:%20achelata,%20astacidea,%20glypheidea,%20polychelida)&amp;author=HD%20Bracken-Grissom&amp;author=ST%20Ahyong&amp;author=RD%20Wilkinson&amp;author=RM%20Feldmann&amp;author=CE%20Schweitzer&amp;volume=63&amp;issue=4&amp;publication_year=2014&amp;pages=457-79&amp;pmid=24562813&amp;doi=10.1093/sysbio/syu008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref048">
<span class="label">48.</span><cite>Hedges SB, Marin J, Suleski M, Paymer M, Kumar S. Tree of life reveals clock-like speciation and diversification. Mol Biol Evol. 2015;32(4):835–45. doi: 10.1093/molbev/msv037

</cite> [<a href="https://doi.org/10.1093/molbev/msv037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4379413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25739733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=Tree%20of%20life%20reveals%20clock-like%20speciation%20and%20diversification&amp;author=SB%20Hedges&amp;author=J%20Marin&amp;author=M%20Suleski&amp;author=M%20Paymer&amp;author=S%20Kumar&amp;volume=32&amp;issue=4&amp;publication_year=2015&amp;pages=835-45&amp;pmid=25739733&amp;doi=10.1093/molbev/msv037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref049">
<span class="label">49.</span><cite>Bracken-Grissom HD, Cannon ME, Cabezas P, Feldmann RM, Schweitzer CE, Ahyong ST, et al. A comprehensive and integrative reconstruction of evolutionary history for Anomura (Crustacea: Decapoda). BMC Evol Biol. 2013;13:128. doi: 10.1186/1471-2148-13-128

</cite> [<a href="https://doi.org/10.1186/1471-2148-13-128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3708748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23786343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evol%20Biol&amp;title=A%20comprehensive%20and%20integrative%20reconstruction%20of%20evolutionary%20history%20for%20Anomura%20(Crustacea:%20Decapoda)&amp;author=HD%20Bracken-Grissom&amp;author=ME%20Cannon&amp;author=P%20Cabezas&amp;author=RM%20Feldmann&amp;author=CE%20Schweitzer&amp;volume=13&amp;publication_year=2013&amp;pages=128&amp;pmid=23786343&amp;doi=10.1186/1471-2148-13-128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref050">
<span class="label">50.</span><cite>Love MI, Huber W, Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014;15(12):550. doi: 10.1186/s13059-014-0550-8

</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;author=MI%20Love&amp;author=W%20Huber&amp;author=S%20Anders&amp;volume=15&amp;issue=12&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref051">
<span class="label">51.</span><cite>Babarinde IA, Hutchins AP. The effects of sequencing depth on the assembly of coding and noncoding transcripts in the human genome. BMC Genomics. 2022;23(1). doi: 10.1186/s12864-022-08717-z</cite> [<a href="https://doi.org/10.1186/s12864-022-08717-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9251931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35787153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=The%20effects%20of%20sequencing%20depth%20on%20the%20assembly%20of%20coding%20and%20noncoding%20transcripts%20in%20the%20human%20genome&amp;author=IA%20Babarinde&amp;author=AP%20Hutchins&amp;volume=23&amp;issue=1&amp;publication_year=2022&amp;pmid=35787153&amp;doi=10.1186/s12864-022-08717-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref052">
<span class="label">52.</span><cite>Babarinde IA, Ma G, Li Y, Deng B, Luo Z, Liu H, et al. Transposable element sequence fragments incorporated into coding and noncoding transcripts modulate the transcriptome of human pluripotent stem cells. Nucleic Acids Research. 2021;49(16):9132–53. doi: 10.1093/nar/gkab710
</cite> [<a href="https://doi.org/10.1093/nar/gkab710" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8450112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34390351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Transposable%20element%20sequence%20fragments%20incorporated%20into%20coding%20and%20noncoding%20transcripts%20modulate%20the%20transcriptome%20of%20human%20pluripotent%20stem%20cells&amp;author=IA%20Babarinde&amp;author=G%20Ma&amp;author=Y%20Li&amp;author=B%20Deng&amp;author=Z%20Luo&amp;volume=49&amp;issue=16&amp;publication_year=2021&amp;pages=9132-53&amp;pmid=34390351&amp;doi=10.1093/nar/gkab710&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref053">
<span class="label">53.</span><cite>Babarinde IA, Li Y, Hutchins AP. Computational Methods for Mapping, Assembly and Quantification for Coding and Non-coding Transcripts. Comput Struct Biotechnol J. 2019;17:628–37. doi: 10.1016/j.csbj.2019.04.012

</cite> [<a href="https://doi.org/10.1016/j.csbj.2019.04.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6526290/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31193391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Struct%20Biotechnol%20J&amp;title=Computational%20Methods%20for%20Mapping,%20Assembly%20and%20Quantification%20for%20Coding%20and%20Non-coding%20Transcripts&amp;author=IA%20Babarinde&amp;author=Y%20Li&amp;author=AP%20Hutchins&amp;volume=17&amp;publication_year=2019&amp;pages=628-37&amp;pmid=31193391&amp;doi=10.1016/j.csbj.2019.04.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref054">
<span class="label">54.</span><cite>Kumar S, Suleski M, Craig JM, Kasprowicz AE, Sanderford M, Li M, et al. TimeTree 5: An Expanded Resource for Species Divergence Times. Mol Biol Evol. 2022;39(8):msac174. doi: 10.1093/molbev/msac174

</cite> [<a href="https://doi.org/10.1093/molbev/msac174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9400175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35932227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=TimeTree%205:%20An%20Expanded%20Resource%20for%20Species%20Divergence%20Times&amp;author=S%20Kumar&amp;author=M%20Suleski&amp;author=JM%20Craig&amp;author=AE%20Kasprowicz&amp;author=M%20Sanderford&amp;volume=39&amp;issue=8&amp;publication_year=2022&amp;pmid=35932227&amp;doi=10.1093/molbev/msac174&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref055">
<span class="label">55.</span><cite>Cunningham CW, Blackstone NW, Buss LW. Evolution of king crabs from hermit crab ancestors. Nature. 1992;355(6360):539–42. doi: 10.1038/355539a0

</cite> [<a href="https://doi.org/10.1038/355539a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1741031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Evolution%20of%20king%20crabs%20from%20hermit%20crab%20ancestors&amp;author=CW%20Cunningham&amp;author=NW%20Blackstone&amp;author=LW%20Buss&amp;volume=355&amp;issue=6360&amp;publication_year=1992&amp;pages=539-42&amp;pmid=1741031&amp;doi=10.1038/355539a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref056">
<span class="label">56.</span><cite>Bracken-Grissom HD, Cannon ME, Cabezas P, Feldmann RM, Schweitzer CE, Ahyong ST, et al. A comprehensive and integrative reconstruction of evolutionary history for Anomura (Crustacea: Decapoda). BMC Evol Biol. 2013;13:128. doi: 10.1186/1471-2148-13-128

</cite> [<a href="https://doi.org/10.1186/1471-2148-13-128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3708748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23786343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evol%20Biol&amp;title=A%20comprehensive%20and%20integrative%20reconstruction%20of%20evolutionary%20history%20for%20Anomura%20(Crustacea:%20Decapoda)&amp;author=HD%20Bracken-Grissom&amp;author=ME%20Cannon&amp;author=P%20Cabezas&amp;author=RM%20Feldmann&amp;author=CE%20Schweitzer&amp;volume=13&amp;publication_year=2013&amp;pages=128&amp;pmid=23786343&amp;doi=10.1186/1471-2148-13-128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref057">
<span class="label">57.</span><cite>Chow S, Hamasaki K, Konishi K, Yanagimoto T, Wagatsuma R, Takeyama H. Variation of length and sequence of the nuclear ribosomal DNA internal transcribed spacer 1 supports “hermit-to-king” crab hypothesis. Crust Res. 2023;52(0):31–48. doi: 10.18353/crustacea.52.0_31</cite> [<a href="https://doi.org/10.18353/crustacea.52.0_31" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crust%20Res&amp;title=Variation%20of%20length%20and%20sequence%20of%20the%20nuclear%20ribosomal%20DNA%20internal%20transcribed%20spacer%201%20supports%20%E2%80%9Chermit-to-king%E2%80%9D%20crab%20hypothesis&amp;author=S%20Chow&amp;author=K%20Hamasaki&amp;author=K%20Konishi&amp;author=T%20Yanagimoto&amp;author=R%20Wagatsuma&amp;volume=52&amp;issue=0&amp;publication_year=2023&amp;pages=31-48&amp;doi=10.18353/crustacea.52.0_31&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref058">
<span class="label">58.</span><cite>Morrison CL, Harvey AW, Lavery S, Tieu K, Huang Y, Cunningham CW. Mitochondrial gene rearrangements confirm the parallel evolution of the crab-like form. Proc Biol Sci. 2002;269(1489):345–50. doi: 10.1098/rspb.2001.1886

</cite> [<a href="https://doi.org/10.1098/rspb.2001.1886" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1690904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11886621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Biol%20Sci&amp;title=Mitochondrial%20gene%20rearrangements%20confirm%20the%20parallel%20evolution%20of%20the%20crab-like%20form&amp;author=CL%20Morrison&amp;author=AW%20Harvey&amp;author=S%20Lavery&amp;author=K%20Tieu&amp;author=Y%20Huang&amp;volume=269&amp;issue=1489&amp;publication_year=2002&amp;pages=345-50&amp;pmid=11886621&amp;doi=10.1098/rspb.2001.1886&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref059">
<span class="label">59.</span><cite>Zwonitzer KD, Iverson ENK, Sterling JE, Weaver RJ, Maclaine BA, Havird JC. Disentangling Positive Selection from Relaxed Selection in Animal Mitochondrial Genomes. Am Nat. 2023;202(4):E121–9. doi: 10.1086/725805

</cite> [<a href="https://doi.org/10.1086/725805" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10955554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37792916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20Nat&amp;title=Disentangling%20Positive%20Selection%20from%20Relaxed%20Selection%20in%20Animal%20Mitochondrial%20Genomes&amp;author=KD%20Zwonitzer&amp;author=ENK%20Iverson&amp;author=JE%20Sterling&amp;author=RJ%20Weaver&amp;author=BA%20Maclaine&amp;volume=202&amp;issue=4&amp;publication_year=2023&amp;pmid=37792916&amp;doi=10.1086/725805&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref060">
<span class="label">60.</span><cite>Wertheim JO, Murrell B, Smith MD, Kosakovsky Pond SL, Scheffler K. RELAX: detecting relaxed selection in a phylogenetic framework. Mol Biol Evol. 2015;32(3):820–32. doi: 10.1093/molbev/msu400

</cite> [<a href="https://doi.org/10.1093/molbev/msu400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4327161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25540451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=RELAX:%20detecting%20relaxed%20selection%20in%20a%20phylogenetic%20framework&amp;author=JO%20Wertheim&amp;author=B%20Murrell&amp;author=MD%20Smith&amp;author=SL%20Kosakovsky%20Pond&amp;author=K%20Scheffler&amp;volume=32&amp;issue=3&amp;publication_year=2015&amp;pages=820-32&amp;pmid=25540451&amp;doi=10.1093/molbev/msu400&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref061">
<span class="label">61.</span><cite>Ohta T. The Nearly Neutral Theory of Molecular Evolution. Annu Rev Ecol Syst. 1992;23(1):263–86. doi: 10.1146/annurev.es.23.110192.001403</cite> [<a href="https://doi.org/10.1146/annurev.es.23.110192.001403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Ecol%20Syst&amp;title=The%20Nearly%20Neutral%20Theory%20of%20Molecular%20Evolution&amp;author=T%20Ohta&amp;volume=23&amp;issue=1&amp;publication_year=1992&amp;pages=263-86&amp;doi=10.1146/annurev.es.23.110192.001403&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref062">
<span class="label">62.</span><cite>Miyata T, Hayashida H, Kikuno R, Hasegawa M, Kobayashi M, Koike K. Molecular clock of silent substitution: at least six-fold preponderance of silent changes in mitochondrial genes over those in nuclear genes. J Mol Evol. 1982;19(1):28–35. doi: 10.1007/BF02100221

</cite> [<a href="https://doi.org/10.1007/BF02100221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7161808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Evol&amp;title=Molecular%20clock%20of%20silent%20substitution:%20at%20least%20six-fold%20preponderance%20of%20silent%20changes%20in%20mitochondrial%20genes%20over%20those%20in%20nuclear%20genes&amp;author=T%20Miyata&amp;author=H%20Hayashida&amp;author=R%20Kikuno&amp;author=M%20Hasegawa&amp;author=M%20Kobayashi&amp;volume=19&amp;issue=1&amp;publication_year=1982&amp;pages=28-35&amp;pmid=7161808&amp;doi=10.1007/BF02100221&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref063">
<span class="label">63.</span><cite>Ohta T. Slightly deleterious mutant substitutions in evolution. Nature. 1973;246(5428):96–8. doi: 10.1038/246096a0

</cite> [<a href="https://doi.org/10.1038/246096a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4585855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Slightly%20deleterious%20mutant%20substitutions%20in%20evolution&amp;author=T%20Ohta&amp;volume=246&amp;issue=5428&amp;publication_year=1973&amp;pages=96-8&amp;pmid=4585855&amp;doi=10.1038/246096a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref064">
<span class="label">64.</span><cite>Tagmount A, Wang M, Lindquist E, Tanaka Y, Teranishi KS, Sunagawa S, et al. The porcelain crab transcriptome and PCAD, the porcelain crab microarray and sequence database. PLoS One. 2010;5(2):e9327. doi: 10.1371/journal.pone.0009327

</cite> [<a href="https://doi.org/10.1371/journal.pone.0009327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2824831/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20174471/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=The%20porcelain%20crab%20transcriptome%20and%20PCAD,%20the%20porcelain%20crab%20microarray%20and%20sequence%20database&amp;author=A%20Tagmount&amp;author=M%20Wang&amp;author=E%20Lindquist&amp;author=Y%20Tanaka&amp;author=KS%20Teranishi&amp;volume=5&amp;issue=2&amp;publication_year=2010&amp;pmid=20174471&amp;doi=10.1371/journal.pone.0009327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref065">
<span class="label">65.</span><cite>Cheng C-H, Ma H-L, Deng Y-Q, Feng J, Chen X-L, Guo Z-X. Transcriptome analysis and histopathology of the mud crab (Scylla paramamosain) after air exposure. Comp Biochem Physiol C Toxicol Pharmacol. 2020;228:108652. doi: 10.1016/j.cbpc.2019.108652

</cite> [<a href="https://doi.org/10.1016/j.cbpc.2019.108652" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31683012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comp%20Biochem%20Physiol%20C%20Toxicol%20Pharmacol&amp;title=Transcriptome%20analysis%20and%20histopathology%20of%20the%20mud%20crab%20(Scylla%20paramamosain)%20after%20air%20exposure&amp;author=C-H%20Cheng&amp;author=H-L%20Ma&amp;author=Y-Q%20Deng&amp;author=J%20Feng&amp;author=X-L%20Chen&amp;volume=228&amp;publication_year=2020&amp;pages=108652&amp;pmid=31683012&amp;doi=10.1016/j.cbpc.2019.108652&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref066">
<span class="label">66.</span><cite>Yednock BK, Sullivan TJ, Neigel JE. De novo assembly of a transcriptome from juvenile blue crabs (Callinectes sapidus) following exposure to surrogate Macondo crude oil. BMC Genomics. 2015;16(1):521. doi: 10.1186/s12864-015-1739-2

</cite> [<a href="https://doi.org/10.1186/s12864-015-1739-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4499174/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26162747/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=De%20novo%20assembly%20of%20a%20transcriptome%20from%20juvenile%20blue%20crabs%20(Callinectes%20sapidus)%20following%20exposure%20to%20surrogate%20Macondo%20crude%20oil&amp;author=BK%20Yednock&amp;author=TJ%20Sullivan&amp;author=JE%20Neigel&amp;volume=16&amp;issue=1&amp;publication_year=2015&amp;pages=521&amp;pmid=26162747&amp;doi=10.1186/s12864-015-1739-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref067">
<span class="label">67.</span><cite>Ingle RW. Northeastern Atlantic and Mediterranean hermit crabs (Crustacea: Anomura: Paguroidea: Paguridae). I. The genusPagurusFabricius, 1775. Journal of Natural History. 1985;19(4):745–69. doi: 10.1080/00222938500770461</cite> [<a href="https://doi.org/10.1080/00222938500770461" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Natural%20History&amp;title=Northeastern%20Atlantic%20and%20Mediterranean%20hermit%20crabs%20(Crustacea:%20Anomura:%20Paguroidea:%20Paguridae).%20I.%20The%20genusPagurusFabricius,%201775&amp;author=RW%20Ingle&amp;volume=19&amp;issue=4&amp;publication_year=1985&amp;pages=745-69&amp;doi=10.1080/00222938500770461&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref068">
<span class="label">68.</span><cite>Castaño C. A new species of Pagurus Fabricius, 1775 from the Pacific coast of Colombia, with a checklist of eastern Pacific species of the genus. Nauplius. 2004;12.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Nauplius&amp;title=A%20new%20species%20of%20Pagurus%20Fabricius,%201775%20from%20the%20Pacific%20coast%20of%20Colombia,%20with%20a%20checklist%20of%20eastern%20Pacific%20species%20of%20the%20genus&amp;author=C%20Casta%C3%B1o&amp;volume=12&amp;publication_year=2004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref069">
<span class="label">69.</span><cite>Babarinde IA. Conserved noncoding sequences: Evolving puzzles. iDarwin. 2021;1:3–36. <a href="http://idarwin.org/docs/iDarwin_1_3-36_BABARINDE_review.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://idarwin.org/docs/iDarwin_1_3-36_BABARINDE_review.pdf</a></cite> [<a href="https://scholar.google.com/scholar_lookup?journal=iDarwin&amp;title=Conserved%20noncoding%20sequences:%20Evolving%20puzzles&amp;author=IA%20Babarinde&amp;volume=1&amp;publication_year=2021&amp;pages=3-36&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref070">
<span class="label">70.</span><cite>Babarinde IA, Saitou N. Genomic Locations of Conserved Noncoding Sequences and Their Proximal Protein-Coding Genes in Mammalian Expression Dynamics. Mol Biol Evol. 2016;33(7):1807–17. doi: 10.1093/molbev/msw058

</cite> [<a href="https://doi.org/10.1093/molbev/msw058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27017584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=Genomic%20Locations%20of%20Conserved%20Noncoding%20Sequences%20and%20Their%20Proximal%20Protein-Coding%20Genes%20in%20Mammalian%20Expression%20Dynamics&amp;author=IA%20Babarinde&amp;author=N%20Saitou&amp;volume=33&amp;issue=7&amp;publication_year=2016&amp;pages=1807-17&amp;pmid=27017584&amp;doi=10.1093/molbev/msw058&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330170.ref071">
<span class="label">71.</span><cite>Babarinde IA, Saitou N. Heterogeneous tempo and mode of conserved noncoding sequence evolution among four mammalian orders. Genome Biol Evol. 2013;5(12):2330–43. doi: 10.1093/gbe/evt177

</cite> [<a href="https://doi.org/10.1093/gbe/evt177" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3879966/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24259317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol%20Evol&amp;title=Heterogeneous%20tempo%20and%20mode%20of%20conserved%20noncoding%20sequence%20evolution%20among%20four%20mammalian%20orders&amp;author=IA%20Babarinde&amp;author=N%20Saitou&amp;volume=5&amp;issue=12&amp;publication_year=2013&amp;pages=2330-43&amp;pmid=24259317&amp;doi=10.1093/gbe/evt177&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section></section><article class="sub-article" id="pone.0330170.r001"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330170.r001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330170.r001</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 0</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ZHANG%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Feng ZHANG</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Feng ZHANG</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ZHANG%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Feng ZHANG</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Feng ZHANG</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.c" class="d-panel p" style="display: none">
<div>© 2025 Feng ZHANG</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>27 Jun 2025</em>
</p>
<p>Dear Dr. Babarinde,</p>
<p>Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE’s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>Please submit your revised manuscript by Aug 11 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <span>plosone@plos.org</span> . When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pone/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<ul class="list" style="list-style-type:disc">
<li><p>A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.</p></li>
<li><p>A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p></li>
<li><p>An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p></li>
</ul>
<p>If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: <a href="https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a> . Additionally, PLOS ONE offers an option for publishing peer-reviewed Lab Protocol articles, which describe protocols hosted on protocols.io. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a> .</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Feng ZHANG, Ph.D.</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Journal Requirements:</p>
<p>When submitting your revision, we need you to address these additional requirements.</p>
<p>1. Please ensure that your manuscript meets PLOS ONE's style requirements, including those for file naming. The PLOS ONE style templates can be found at</p>
<p><a href="https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf</a> and</p>
<p>
<a href="https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf</a>
</p>
<p>2.  In your Methods section, please provide additional information regarding the permits you obtained for the work. Please ensure you have included the full name of the authority that approved the field site access and, if no permits were required, a brief statement explaining why.</p>
<p>3. We note that the grant information you provided in the ‘Funding Information’ and ‘Financial Disclosure’ sections do not match.</p>
<p>When you resubmit, please ensure that you provide the correct grant numbers for the awards you received for your study in the ‘Funding Information’ section.</p>
<p>4. Thank you for stating the following financial disclosure:</p>
<p>“This study was supported by the funding of Research Organization of Information and Systems (ROIS) for special collaboration research student program of National Institute of Genetics (NIG) and Kyoto University, Japan”</p>
<p>Please state what role the funders took in the study. If the funders had no role, please state: "The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript."</p>
<p>If this statement is not correct you must amend it as needed.</p>
<p>Please include this amended Role of Funder statement in your cover letter; we will change the online submission form on your behalf.</p>
<p>5. Thank you for stating the following in the Acknowledgments Section of your manuscript:</p>
<p>“The authors are thankful to Mr. Nakano, R., Ms. Nakano, M., and the members of SMBL for their help in collecting the study animals. This study was supported by the funding of Research Organization of Information and Systems (ROIS) for special collaboration research student program of National Institute of Genetics (NIG) and Kyoto University, Japan.”</p>
<p>We note that you have provided funding information that is not currently declared in your Funding Statement. However, funding information should not appear in the Acknowledgments section or other areas of your manuscript. We will only publish funding information present in the Funding Statement section of the online submission form.</p>
<p>Please remove any funding-related text from the manuscript and let us know how you would like to update your Funding Statement. Currently, your Funding Statement reads as follows:</p>
<p>“This study was supported by the funding of Research Organization of Information and Systems (ROIS) for special collaboration research student program of National Institute of Genetics (NIG) and Kyoto University, Japan”</p>
<p>Please include your amended statements within your cover letter; we will change the online submission form on your behalf.</p>
<p>6. When completing the data availability statement of the submission form, you indicated that you will make your data available on acceptance. We strongly recommend all authors decide on a data sharing plan before acceptance, as the process can be lengthy and hold up publication timelines. Please note that, though access restrictions are acceptable now, your entire data will need to be made freely accessible if your manuscript is accepted for publication. This policy applies to all data except where public deposition would breach compliance with the protocol approved by your research ethics board. If you are unable to adhere to our open data policy, please kindly revise your statement to explain your reasoning and we will seek the editor's input on an exemption. Please be assured that, once you have provided your new statement, the assessment of your exemption will not hold up the peer review process.</p>
<p>7. PLOS requires an ORCID iD for the corresponding author in Editorial Manager on papers submitted after December 6th, 2016. Please ensure that you have an ORCID iD and that it is validated in Editorial Manager. To do this, go to ‘Update my Information’ (in the upper left-hand corner of the main menu), and click on the Fetch/Validate link next to the ORCID field. This will take you to the ORCID site and allow you to create a new iD or authenticate a pre-existing iD in Editorial Manager.</p>
<p>8. We are unable to open your Supporting Information file [HermitCrab_SupplementaryTables.xlsx and S1Fig.eps]. Please kindly revise as necessary and re-upload.</p>
<p>9. Please review your reference list to ensure that it is complete and correct. If you have cited papers that have been retracted, please include the rationale for doing so in the manuscript text, or remove these references and replace them with relevant current references. Any changes to the reference list should be mentioned in the rebuttal letter that accompanies your revised manuscript. If you need to cite a retracted article, indicate the article’s retracted status in the References list and also include a citation and full reference for the retraction notice.</p>
<p>[Note: HTML markup is below. Please do not edit.]</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>
<strong>Comments to the Author</strong>
</p>
<p>1. Is the manuscript technically sound, and do the data support the conclusions?</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>2. Has the statistical analysis been performed appropriately and rigorously? --&gt;?&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>3. Have the authors made all data underlying the findings in their manuscript fully available??&gt;</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a></p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>4. Is the manuscript presented in an intelligible fashion and written in standard English??&gt;</p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>Reviewer #1: I believe that this paper is valuable for publication. However, as mentioned below, it needs some revisions.</p>
<p>L.38, 39: leg of xxx leg?</p>
<p>L.40: chin-based?</p>
<p>L.52: what is “those species”?</p>
<p>L.52: “Sibling species and cryptic species are sometimes not properly identified”. Because taxa not known are cryptic species.</p>
<p>L.93: “RNA samples were extracted separately from the head and leg” Both parts contain muscle, membrane, and exoskeleton or something else, but RNA was extracted from muscle. First, the material must be properly identified.</p>
<p>BLAST not blast</p>
<p>L.211-212: This must be placed in Materials and Methods.</p>
<p>L.215: remove “for each sample” How many individuals per species were used?</p>
<p>L215-216: This must be placed in Materials and Methods.</p>
<p>L.219-221: This must be placed in Materials and Methods.</p>
<p>L.242-244: This must be placed in Materials and Methods.</p>
<p>L.321-324: References 9 and 18 addressed only the genus Pagurus. Therefore, if the authors are referring to the non-monophyletic status of Pagurus in relation to the family Lithodidae, they should cite other references (e.g., Cunningham et al., 1992; Zaklin, 2001; Morrison et al., 2002; Blacken-Grissom et al., 2013).</p>
<p>L.505: respectively</p>
<p>The generic abbreviations for Pagurus, Paralithodes, and Petrolisthes should be clear. For example, they could be Pag., Par., and Pet.</p>
<p>L.505-: I guess that almost all nucleotide substitutions in protein-coding mitochondrial DNA between the two Pagurus species are synonymous.</p>
<p>L.519: “found” should be “estimated”</p>
<p>L.525: specify the two species.</p>
<p>L.529: A recent study on ITS1 sequence divergence and length (Chow et al. 2023, Crust. Res. 52) showed well separate status between Pagurus and Lithodid. No discussion for this?</p>
<p>Reviewer #2: The MS investigates the molecular and gene expression evolution of two closely related Pagurus species using transcriptome sequencing. The results sheds light on the evolutionary history of the two species at the molecular level. This MS demonstrates substantive content, sound data analysis, and reliable results, presented through well-structured writing that meets the criteria for publication readiness.</p>
<p>**********</p>
<p><a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a></p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p>
<p>**********</p>
<p>[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.]</p>
<p>While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a> . PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at <span>figures@plos.org</span></p></section></article><article class="sub-article" id="pone.0330170.r002"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. 2025 Aug 20;20(8):e0330170. doi: <a href="https://doi.org/10.1371/journal.pone.0330170.r002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330170.r002</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 1</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.d" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.d" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub2" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>25 Jul 2025</em>
</p>
<p>We appreciate the editor and the reviewers for the time and efforts spent in reviewing our manuscript. We are particularly grateful for the suggestions towards the improvement of the manuscript. We have meticulously revised the manuscript based on the suggestions and comments from the editor and the reviewers. We have also corrected other typographical or grammatical errors that we encountered. All changes have been marked in one of the uuploaded documents. Specific response and changes made in response to each comment are itemized below (in blue font).</p>
<p>Response to the editor</p>
<p>We are grateful to the editor for the comments. We have responded to each raised comment with blue font below.</p>
<p>1. Please ensure that your manuscript meets PLOS ONE's style requirements, including those for file naming. The PLOS ONE style templates can be found at</p>
<p><a href="https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf</a> and</p>
<p>
<a href="https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf</a>
</p>
<p>We appreciate the editor for pointing this out. The first page has been revised to be consistent with PLOS ONE’s style. We have also revised the listing of supplementary figures and tables to be consistent with the PLOS ONE’s style.</p>
<p>2. In your Methods section, please provide additional information regarding the permits you obtained for the work. Please ensure you have included the full name of the authority that approved the field site access and, if no permits were required, a brief statement explaining why.</p>
<p>We have updated the Materials and Methods section by adding the following sentences. “Sample collection permit for experimental research with number 5-2 was granted by Wakayama Prefecture, Japan. Researches on Pagurus species do not require any special approval. All experiments complied with institutional regulations and Japanese policy on animal use [22].” We also corrected the sampling date to reflect the actual date when the specific sequenced samples were collected.</p>
<p>3. We note that the grant information you provided in the ‘Funding Information’ and ‘Financial Disclosure’ sections do not match.</p>
<p>When you resubmit, please ensure that you provide the correct grant numbers for the awards you received for your study in the ‘Funding Information’ section.</p>
<p>The grants used for the study do not have any special grant number as they are mostly in-house grants.</p>
<p>4. Thank you for stating the following financial disclosure:</p>
<p>“This study was supported by the funding of Research Organization of Information and Systems (ROIS) for special collaboration research student program of National Institute of Genetics (NIG) and Kyoto University, Japan”</p>
<p>Please state what role the funders took in the study. If the funders had no role, please state: "The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript."</p>
<p>If this statement is not correct you must amend it as needed.</p>
<p>Please include this amended Role of Funder statement in your cover letter; we will change the online submission form on your behalf.</p>
<p>We declare that "The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript."</p>
<p>5. Thank you for stating the following in the Acknowledgments Section of your manuscript:</p>
<p>“The authors are thankful to Mr. Nakano, R., Ms. Nakano, M., and the members of SMBL for their help in collecting the study animals. This study was supported by the funding of Research Organization of Information and Systems (ROIS) for special collaboration research student program of National Institute of Genetics (NIG) and Kyoto University, Japan.”</p>
<p>We note that you have provided funding information that is not currently declared in your Funding Statement. However, funding information should not appear in the Acknowledgments section or other areas of your manuscript. We will only publish funding information present in the Funding Statement section of the online submission form.</p>
<p>Please remove any funding-related text from the manuscript and let us know how you would like to update your Funding Statement. Currently, your Funding Statement reads as follows:</p>
<p>“This study was supported by the funding of Research Organization of Information and Systems (ROIS) for special collaboration research student program of National Institute of Genetics (NIG) and Kyoto University, Japan”</p>
<p>Please include your amended statements within your cover letter; we will change the online submission form on your behalf.</p>
<p>As suggested by the editor, we have deleted the funding-related text from the manuscript. And we would like to leave the funding statement as it is, that is, “This study was supported by the funding of Research Organization of Information and Systems (ROIS) for special collaboration research student program of National Institute of Genetics (NIG) and Kyoto University, Japan”</p>
<p>6. When completing the data availability statement of the submission form, you indicated that you will make your data available on acceptance. We strongly recommend all authors decide on a data sharing plan before acceptance, as the process can be lengthy and hold up publication timelines. Please note that, though access restrictions are acceptable now, your entire data will need to be made freely accessible if your manuscript is accepted for publication. This policy applies to all data except where public deposition would breach compliance with the protocol approved by your research ethics board. If you are unable to adhere to our open data policy, please kindly revise your statement to explain your reasoning and we will seek the editor's input on an exemption. Please be assured that, once you have provided your new statement, the assessment of your exemption will not hold up the peer review process.</p>
<p>We have made the raw read data publicly available (as from July 22nd, 2025) using the new accessions provided in the manuscript. The data can now be assessed from <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJDB35803" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/PRJDB35803</a>. We would like to point out that we discovered that the biological replicates were not correctly entered in the original submission (<a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJDB7354" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJDB7354</a>). That is why we had to revise the submission and now have new accessions for the project and the samples. The manuscript has been appropriately revised to state “All raw reads generated in this study are available in the DDBJ Sequence Read Archive (DRA) under the BioProject accession number PRJDB35803. The data sets supporting the results presented here are available with the accession numbers DRR708008-DRR708019.”</p>
<p>7. PLOS requires an ORCID iD for the corresponding author in Editorial Manager on papers submitted after December 6th, 2016. Please ensure that you have an ORCID iD and that it is validated in Editorial Manager. To do this, go to ‘Update my Information’ (in the upper left-hand corner of the main menu), and click on the Fetch/Validate link next to the ORCID field. This will take you to the ORCID site and allow you to create a new iD or authenticate a pre-existing iD in Editorial Manager.</p>
<p>ORCID ID has been linked and validated in Editorial Manager.</p>
<p>8. We are unable to open your Supporting Information file [HermitCrab_SupplementaryTables.xlsx and S1Fig.eps]. Please kindly revise as necessary and re-upload.</p>
<p>We have confirmed that the files are still in good shape. We suspect that the problem might have arisen during file uploading. We will re-upload the files during the submission of the revised manuscript. To ensure that the files can be opened, supplementary figure will be saved in two formats (.eps and .tiff). An additional compressed file of the supplementary figure and table files will also be attached. Multiple uploading of the same files is to ensure the ability of the files to be opened.</p>
<p>9. Please review your reference list to ensure that it is complete and correct. If you have cited papers that have been retracted, please include the rationale for doing so in the manuscript text, or remove these references and replace them with relevant current references. Any changes to the reference list should be mentioned in the rebuttal letter that accompanies your revised manuscript. If you need to cite a retracted article, indicate the article’s retracted status in the References list and also include a citation and full reference for the retraction notice.</p>
<p>The revised manuscript now included references 18-20 supporting the growing use of molecular barcodes. We have listed reference 22 about institutional regulations and Japanese policy on animal use. In addition, we have cited new papers (references 55-58) suggested by a reviewer.</p>
<p>Response to reviewers</p>
<p>Reviewer #1: I believe that this paper is valuable for publication. However, as mentioned below, it needs some revisions.</p>
<p>We appreciate the reviewer for the time and efforts towards the improvement of our manuscripts. We are sincerely grateful for the multiple corrections that the reviewer has suggested. We have incorporated the reviewer’s suggestions and corrections in the revised version of the manuscript. We have also listed point-by-point response to the reviewer’s suggestion and comments.</p>
<p>L.38, 39: leg of xxx leg?</p>
<p>This has been revised by deleting “leg”.</p>
<p>L.40: chin-based?</p>
<p>“chin-based” has been changed to “chitin-based”</p>
<p>L.52: what is “those species”?</p>
<p>“those species” has been changed to “Pagurus species”</p>
<p>L.52: “Sibling species and cryptic species are sometimes not properly identified”. Because taxa not known are cryptic species.</p>
<p>This sentence has been changed to “Consequently, many sibling species remain cryptic species that are difficult to recognize using traditional classic methods”</p>
<p>L.93: “RNA samples were extracted separately from the head and leg” Both parts contain muscle, membrane, and exoskeleton or something else, but RNA was extracted from muscle. First, the material must be properly identified.</p>
<p>We apologize for this confusion. RNA extraction was actually done using the whole leg or head as pointed out in Figures 1A and B, and not just for the muscle. We have replaced “muscle” with “head or leg” in the Methods section.</p>
<p>BLAST not blast</p>
<p>All instances of “blast” to “BLAST”, including “BLASTN” and “BLASTP”.</p>
<p>L.211-212: This must be placed in Materials and Methods.</p>
<p>This has been moved.</p>
<p>L.215: remove “for each sample” How many individuals per species were used?</p>
<p>This has been revised. Three individuals were used per species.</p>
<p>L215-216: This must be placed in Materials and Methods.</p>
<p>This has been removed.</p>
<p>L.219-221: This must be placed in Materials and Methods.</p>
<p>This has been deleted. We have revised the following sentence to read “Removal of potentially noisy transcripts produced 203,793 transcripts, including 140,954 genes for P. maculosus, and 259,375 transcripts, including 179,685 genes for P. lanuginosus (Table 1)”</p>
<p>L.242-244: This must be placed in Materials and Methods.</p>
<p>This has been deleted. We have added the following to give the background to the result: “Transcriptome assembly has the potential to retrieve both coding and noncoding transcripts. We therefore checked the proportion of both coding and noncoding transcripts in our transcriptome assembly (see Materials and Methods)”</p>
<p>L.321-324: References 9 and 18 addressed only the genus Pagurus. Therefore, if the authors are referring to the non-monophyletic status of Pagurus in relation to the family Lithodidae, they should cite other references (e.g., Cunningham et al., 1992; Zaklin, 2001; Morrison et al., 2002; Blacken-Grissom et al., 2013).</p>
<p>We appreciate the reviewer. We have rephrased the sentence to focus on the evolution of Paralithodes species from Pagurus lineage. As suggested by the reviewer, we have added Cunningham et al. (1992), Morrison et al. (2002) and Bracken-Grissom et al. (2013), as suggested by the reviewer. We also added Chow et al. (2023) which supports the same hypothesis. The sentence now reads “This result further confirmed the previous studies reporting “hermit-to-king” hypothesis that Paralithodes species evolved from Pagurus lineage [55–58].”.</p>
<p>L.505: respectively</p>
<p>“respective” has been changed to “respectively”</p>
<p>The generic abbreviations for Pagurus, Paralithodes, and Petrolisthes should be clear. For example, they could be Pag., Par., and Pet.</p>
<p>We appreciate the reviewer for this suggestion. Since our focus is on Pagurus, we have retained P. for Pagurus genus while using Par. and Pet. for Paralithodes, and Petrolisthes, respectively, as suggested by the reviewer.</p>
<p>L.505-: I guess that almost all nucleotide substitutions in protein-coding mitochondrial DNA between the two Pagurus species are synonymous.</p>
<p>In Sultana et al., two rRNAs, two ATPs, Cox1, Cox2, Cox3, Nn2, and Nd3 mitochondrial regions were analyzed to compute overall nucleotide distance. Although the mitochondrial genes included those that code and those that do not code for proteins, all regions were analyzed together without considering nucleotide substitution type. Indeed, one of the motivations for this study was to more systematically investigate nucleotide evolution. In this study, we have separately computed synonymous and nonsynonymous substitution rates in larger gene set.</p>
<p>L.519: “found” should be “estimated”</p>
<p>This has been corrected.</p>
<p>L.525: specify the two species.</p>
<p>“the two species” has been changed to “P. lanuginosus and P. maculosus”</p>
<p>L.529: A recent study on ITS1 sequence divergence and length (Chow et al. 2023, Crust. Res. 52) showed well separate status between Pagurus and Lithodid. No discussion for this?</p>
<p>We appreciate the reviewer for calling our attention to this paper. Interestingly, the paper showed that lithodoid species had unique internal transcribed spacer 1 (ITS1), which evolved from those found among paguroid species. Their results support the evolution of monophyletic Lithodid from Pagurus lineage. In other words, their findings support the “hermit-to-king” crab hypothesis, as our findings do. Please see Fig. 8 of Chow et al. (2023) which presents the hypothetical evolutionary relationship among hermit crab families. We have included the reference in our manuscript.</p>
<p>Again, we thank the reviewer for their contribution. We hope all the concerns of the reviewer has been addressed.</p>
<p>Reviewer #2: The MS investigates the molecular and gene expression evolution of two closely related Pagurus species using transcriptome sequencing. The results sheds light on the evolutionary history of the two species at the molecular level. This MS demonstrates substantive content, sound data analysis, and reliable results, presented through well-structured writing that meets the criteria for publication readiness.</p>
<p>We appreciate the reviewer for the positive feedback about our manuscript. Thank you.</p>
<section class="sm xbox font-sm" id="pone.0330170.s008"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to Reviewers.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s008.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s008.docx</a><sup> (39.2KB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0330170.r003"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330170.r003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330170.r003</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 1</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ZHANG%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Feng ZHANG</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Feng ZHANG</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ZHANG%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Feng ZHANG</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Feng ZHANG</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none">
<div>© 2025 Feng ZHANG</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>29 Jul 2025</em>
</p>
<p>Transcriptome sequencing reveals the evolutionary histories and gene expression evolution in two related Pagurus species</p>
<p>PONE-D-25-30439R1</p>
<p>Dear Dr. Isaac Adeyemi BABARINDE,</p>
<p>We’re pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it meets all outstanding technical requirements.</p>
<p>Within one week, you’ll receive an e-mail detailing the required amendments. When these have been addressed, you’ll receive a formal acceptance letter and your manuscript will be scheduled for publication.</p>
<p>An invoice will be generated when your article is formally accepted. Please note, if your institution has a publishing partnership with PLOS and your article meets the relevant criteria, all or part of your publication costs will be covered. Please make sure your user information is up-to-date by logging into Editorial Manager at <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Editorial Manager®</a>  and clicking the ‘Update My Information' link at the top of the page. For questions related to billing, please contact <a href="https://plos.my.site.com/s/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">billing support</a> .</p>
<p>If your institution or institutions have a press office, please notify them about your upcoming paper to help maximize its impact. If they’ll be preparing press materials, please inform our press team as soon as possible -- no later than 48 hours after receiving the formal acceptance. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>Kind regards,</p>
<p>Feng ZHANG, Ph.D.</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Additional Editor Comments (optional):</p>
<p>Reviewers' comments:</p></section></article><article class="sub-article" id="pone.0330170.r004"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330170.r004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330170.r004</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Acceptance letter</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ZHANG%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Feng ZHANG</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Feng ZHANG</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ZHANG%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Feng ZHANG</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.f" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.f" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Feng ZHANG</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.f" class="d-panel p" style="display: none">
<div>© 2025 Feng ZHANG</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>PONE-D-25-30439R1</p>
<p>PLOS ONE</p>
<p>Dear Dr. Babarinde,</p>
<p>I'm pleased to inform you that your manuscript has been deemed suitable for publication in PLOS ONE. Congratulations! Your manuscript is now being handed over to our production team.</p>
<p>At this stage, our production department will prepare your paper for publication. This includes ensuring the following:</p>
<p>* All references, tables, and figures are properly cited</p>
<p>* All relevant supporting information is included in the manuscript submission,</p>
<p>* There are no issues that prevent the paper from being properly typeset</p>
<p>You will receive further instructions from the production team, including instructions on how to review your proof when it is ready. Please keep in mind that we are working through a large volume of accepted articles, so please give us a few days to review your paper and let you know the next and final steps.</p>
<p>Lastly, if your institution or institutions have a press office, please let them know about your upcoming paper now to help maximize its impact. If they'll be preparing press materials, please inform our press team within the next 48 hours. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>You will receive an invoice from PLOS for your publication fee after your manuscript has reached the completed accept phase. If you receive an email requesting payment before acceptance or for any other service, this may be a phishing scheme. Learn how to identify phishing emails and protect your accounts at <a href="https://explore.plos.org/phishing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://explore.plos.org/phishing</a>.</p>
<p>If we can help with anything else, please email us at customercare@plos.org.</p>
<p>Thank you for submitting your work to PLOS ONE and supporting open access.</p>
<p>Kind regards,</p>
<p>PLOS ONE Editorial Office Staff</p>
<p>on behalf of</p>
<p>Dr. Feng ZHANG</p>
<p>Academic Editor</p>
<p>PLOS ONE</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Fig. Gene expression evolution and the nature of genes enriched across the species.</span><p>A. 2-D hexagonal binning plots showing the relationships between protein-coding gene expression patterns of the <em>P. maculosus</em> (PM) and <em>P. lanuginosus</em> (PL) in the head (upper panel) and the leg (lower panel). B. 2-D hexagonal binning plots showing the relationships between the positively selected protein-coding gene expression patterns of the <em>P. maculosus</em> (PM) and <em>P. lanuginosus</em> (PL) in the head (upper panel) and the leg (lower panel). For A and B, r = Pearson’s correlation coefficient while the red lines represent the linear regression lines. Log<sub>2</sub>-transfromed TMM-normalized expressions were used for the plots C. Scattered plot showing the relationship between the log-fold changes of genes differentially expressed in head and/or leg. Each dot represents a gene. Negative values represent <em>P. lanuginosus</em> enrichment while positive values represent <em>P. maculosus</em> enrichment. Rho = Spearman’r correlation coefficient. D. Species bias of positively selected differentially expressed head (upper) and leg (lower) genes. P values were computed using Fisher exact tests. Gene ontology enrichments of genes are upregulated in <em>P. lanuginosus</em> leg (E), <em>P. maculosus</em> head (F) and <em>P. lanuginosus</em> head (G). The bar color corresponded to the Bonferroni-corrected hypergeometry p value.</p>
<p>(TIF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s001.tif" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s001.tif</a><sup> (2.4MB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S1 Table. Details of Transcriptome sequencing.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s002.xlsx</a><sup> (10.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S2 Table. Number of homologous transcripts and genes in different databases.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s003.xlsx</a><sup> (10.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S3 Table. Calibrations used for divergence time estimates.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s004.xlsx</a><sup> (10.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S4 Table. The list of known pigmentation genes.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s005.xlsx</a><sup> (41.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>S5 Table. InterproScan annotation statistics for the assembled transcripts.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s006.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s006.xlsx</a><sup> (10.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to Reviewers.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367144/bin/pone.0330170.s008.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330170.s008.docx</a><sup> (39.2KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>All data generated in the study have been made publicly available and deposited to the DNA Data Bank of Japan under the project accession number PRJDB35803, and can be accessed on the NCBI database (<a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJDB35803/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/PRJDB35803/</a>). All short-read data have been described in S1 Table.</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0330170"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0330170.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12367144/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12367144/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367144%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367144/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12367144/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12367144/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40834015/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12367144/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40834015/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12367144/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12367144/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="aZ5IiaJvTxmNHBrWvGtYNoGRHpY2lRU0yXEUMDZdhz0YDhzRjfdnjavNLBeRtxVI">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
