
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Improving risk stratification of PI-RADS 3 + 1 lesions of the peripheral zone: expert lexicon of terms, multi-reader performance and contribution of artificial intelligence - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE533F7F8AF3A043053F7F004AA3EE0D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="canimag">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366217/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Cancer Imaging">
<meta name="citation_title" content="Improving risk stratification of PI-RADS 3 + 1 lesions of the peripheral zone: expert lexicon of terms, multi-reader performance and contribution of artificial intelligence">
<meta name="citation_author" content="Philip A Glemser">
<meta name="citation_author_institution" content="Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany">
<meta name="citation_author" content="Nils Netzer">
<meta name="citation_author_institution" content="Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany">
<meta name="citation_author_institution" content="Department of Radiation Oncology, Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author_institution" content="Heidelberg University Medical School, Heidelberg, Germany">
<meta name="citation_author" content="Christian H Ziener">
<meta name="citation_author_institution" content="Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany">
<meta name="citation_author" content="Markus Wilhelm">
<meta name="citation_author_institution" content="Breast Diagnostics Munich, Sonnenstraße 29, Munich, Germany">
<meta name="citation_author" content="Thomas Hielscher">
<meta name="citation_author_institution" content="Division of Biostatistics, German Cancer Research Center (DKFZ), Heidelberg, Germany">
<meta name="citation_author" content="Kevin Sun Zhang">
<meta name="citation_author_institution" content="Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany">
<meta name="citation_author" content="Magdalena Görtz">
<meta name="citation_author_institution" content="Department of Urology, Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author_institution" content="Junior Clinical Cooperation Unit ‘Multiparametric Methods for Early Detection of Prostate Cancer’, German Cancer Research Center (DKFZ), Heidelberg, Germany">
<meta name="citation_author" content="Viktoria Schütz">
<meta name="citation_author_institution" content="Department of Urology, Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author" content="Albrecht Stenzinger">
<meta name="citation_author_institution" content="Institute of Pathology, University of Heidelberg Medical Center, Heidelberg, Germany">
<meta name="citation_author" content="Markus Hohenfellner">
<meta name="citation_author_institution" content="Department of Urology, Heidelberg University Hospital, Heidelberg, Germany">
<meta name="citation_author" content="Heinz-Peter Schlemmer">
<meta name="citation_author_institution" content="Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany">
<meta name="citation_author_institution" content="National Center for Tumor Diseases (NCT) Heidelberg, Heidelberg, Germany">
<meta name="citation_author" content="David Bonekamp">
<meta name="citation_author_institution" content="Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany">
<meta name="citation_author_institution" content="Heidelberg University Medical School, Heidelberg, Germany">
<meta name="citation_author_institution" content="National Center for Tumor Diseases (NCT) Heidelberg, Heidelberg, Germany">
<meta name="citation_author_institution" content="German Cancer Consortium (DKTK), Heidelberg, Germany">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="102">
<meta name="citation_doi" content="10.1186/s40644-025-00916-7">
<meta name="citation_pmid" content="40830988">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366217/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366217/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366217/pdf/40644_2025_Article_916.pdf">
<meta name="description" content="According to PI-RADS v2.1, peripheral PI-RADS 3 lesions are upgraded to PI-RADS 4 if dynamic contrast-enhanced MRI is positive (3+1 lesions), however those lesions are radiologically challenging. We aimed to define criteria by expert consensus and ...">
<meta name="og:title" content="Improving risk stratification of PI-RADS 3 + 1 lesions of the peripheral zone: expert lexicon of terms, multi-reader performance and contribution of artificial intelligence">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="According to PI-RADS v2.1, peripheral PI-RADS 3 lesions are upgraded to PI-RADS 4 if dynamic contrast-enhanced MRI is positive (3+1 lesions), however those lesions are radiologically challenging. We aimed to define criteria by expert consensus and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366217/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366217">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s40644-025-00916-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/40644_2025_Article_916.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366217%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366217/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366217/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366217/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-canimag.png" alt="Cancer Imaging logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Cancer Imaging" title="Link to Cancer Imaging" shape="default" href="https://cancerimagingjournal.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Cancer Imaging</button></div>. 2025 Aug 19;25:102. doi: <a href="https://doi.org/10.1186/s40644-025-00916-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s40644-025-00916-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Cancer%20Imaging%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cancer%20Imaging%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Cancer%20Imaging%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Cancer%20Imaging%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Improving risk stratification of PI-RADS 3 + 1 lesions of the peripheral zone: expert lexicon of terms, multi-reader performance and contribution of artificial intelligence</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22A%20Glemser%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Philip A Glemser</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Philip A Glemser</span></h3>
<div class="p">
<sup>1</sup>Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22A%20Glemser%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Philip A Glemser</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Netzer%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Nils Netzer</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Nils Netzer</span></h3>
<div class="p">
<sup>1</sup>Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany </div>
<div class="p">
<sup>2</sup>Department of Radiation Oncology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">
<sup>3</sup>Heidelberg University Medical School, Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Netzer%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nils Netzer</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22H%20Ziener%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Christian  H Ziener</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Christian  H Ziener</span></h3>
<div class="p">
<sup>1</sup>Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22H%20Ziener%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian  H Ziener</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilhelm%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Markus  Wilhelm</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Markus  Wilhelm</span></h3>
<div class="p">
<sup>4</sup>Breast Diagnostics Munich, Sonnenstraße 29, Munich, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilhelm%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Markus  Wilhelm</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hielscher%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Thomas Hielscher</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Thomas Hielscher</span></h3>
<div class="p">
<sup>5</sup>Division of Biostatistics, German Cancer Research Center (DKFZ), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hielscher%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas Hielscher</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20Zhang%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kevin  Sun Zhang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kevin  Sun Zhang</span></h3>
<div class="p">
<sup>1</sup>Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20Zhang%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kevin  Sun Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%B6rtz%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Magdalena Görtz</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Magdalena Görtz</span></h3>
<div class="p">
<sup>6</sup>Department of Urology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">
<sup>7</sup>Junior Clinical Cooperation Unit ‘Multiparametric Methods for Early Detection of Prostate Cancer’, German Cancer Research Center (DKFZ), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%B6rtz%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Magdalena Görtz</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sch%C3%BCtz%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Viktoria Schütz</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Viktoria Schütz</span></h3>
<div class="p">
<sup>6</sup>Department of Urology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sch%C3%BCtz%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Viktoria Schütz</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stenzinger%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Albrecht Stenzinger</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Albrecht Stenzinger</span></h3>
<div class="p">
<sup>8</sup>Institute of Pathology, University of Heidelberg Medical Center, Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stenzinger%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Albrecht Stenzinger</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hohenfellner%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Markus Hohenfellner</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Markus Hohenfellner</span></h3>
<div class="p">
<sup>6</sup>Department of Urology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hohenfellner%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Markus Hohenfellner</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schlemmer%20HP%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Heinz-Peter Schlemmer</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Heinz-Peter Schlemmer</span></h3>
<div class="p">
<sup>1</sup>Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany </div>
<div class="p">
<sup>9</sup>National Center for Tumor Diseases (NCT) Heidelberg, Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schlemmer%20HP%22%5BAuthor%5D" class="usa-link"><span class="name western">Heinz-Peter Schlemmer</span></a>
</div>
</div>
<sup>1,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bonekamp%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">David Bonekamp</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">David Bonekamp</span></h3>
<div class="p">
<sup>1</sup>Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany </div>
<div class="p">
<sup>3</sup>Heidelberg University Medical School, Heidelberg, Germany </div>
<div class="p">
<sup>9</sup>National Center for Tumor Diseases (NCT) Heidelberg, Heidelberg, Germany </div>
<div class="p">
<sup>10</sup>German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bonekamp%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Bonekamp</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>9,</sup><sup>10,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Division of Radiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany </div>
<div id="Aff2">
<sup>2</sup>Department of Radiation Oncology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div id="Aff3">
<sup>3</sup>Heidelberg University Medical School, Heidelberg, Germany </div>
<div id="Aff4">
<sup>4</sup>Breast Diagnostics Munich, Sonnenstraße 29, Munich, Germany </div>
<div id="Aff5">
<sup>5</sup>Division of Biostatistics, German Cancer Research Center (DKFZ), Heidelberg, Germany </div>
<div id="Aff6">
<sup>6</sup>Department of Urology, Heidelberg University Hospital, Heidelberg, Germany </div>
<div id="Aff7">
<sup>7</sup>Junior Clinical Cooperation Unit ‘Multiparametric Methods for Early Detection of Prostate Cancer’, German Cancer Research Center (DKFZ), Heidelberg, Germany </div>
<div id="Aff8">
<sup>8</sup>Institute of Pathology, University of Heidelberg Medical Center, Heidelberg, Germany </div>
<div id="Aff9">
<sup>9</sup>National Center for Tumor Diseases (NCT) Heidelberg, Heidelberg, Germany </div>
<div id="Aff10">
<sup>10</sup>German Cancer Consortium (DKTK), Heidelberg, Germany </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 11; Accepted 2025 Jul 24; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366217  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830988/" class="usa-link">40830988</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">According to PI-RADS v2.1, peripheral PI-RADS 3 lesions are upgraded to PI-RADS 4 if dynamic contrast-enhanced MRI is positive (3+1 lesions), however those lesions are radiologically challenging. We aimed to define criteria by expert consensus and test applicability by other radiologists for sPC prediction of PI-RADS 3+1 lesions and determine their value in integrated regression models.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">From consecutive 3 Tesla MR examinations performed between 08/2016 to 12/2018 we identified 85 MRI examinations from 83 patients with a total of 94 PI-RADS 3+1 lesions in the official clinical report. Lesions were retrospectively assessed by expert consensus with construction of a newly devised feature catalogue which was utilized subsequently by two additional radiologists specialized in prostate MRI for independent lesion assessment. With reference to extended fused targeted and systematic TRUS/MRI-biopsy histopathological correlation, relevant catalogue features were identified by univariate analysis and put into context to typically available clinical features and automated AI image assessment utilizing lasso-penalized logistic regression models, also focusing on the contribution of DCE imaging (feature-based, bi- and multiparametric AI-enhanced and solely bi- and multiparametric AI-driven).</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">The feature catalog enabled image-based lesional risk stratification for all readers. Expert consensus provided 3 significant features in univariate analysis (adj. p-value &lt;0.05; most relevant feature T2w configuration: “irregular/microlobulated/spiculated”, OR 9.0 (95%CI 2.3-44.3); adj. p-value: 0.016). These remained after lasso penalized regression based feature reduction, while the only selected clinical feature was prostate volume (OR&lt;1), enabling nomogram construction. While DCE-derived consensus features did not enhance model performance (bootstrapped AUC), there was a trend for increased performance by including multiparametric AI, but not biparametric AI into models, both for combined and AI-only models.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par4">PI-RADS 3+1 lesions can be risk-stratified using lexicon terms and a key feature nomogram. AI potentially benefits more from DCE imaging than experienced prostate radiologists.</p></section><section id="sec5"><h3 class="pmc_sec_title">Clinical trial number</h3>
<p id="Par5">Not applicable.</p></section><section id="sec6"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s40644-025-00916-7.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Prostate cancer, Risk stratification, Lexicon terms, Deep learning, Extended fused biopsy</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par46">The prostate imaging reporting and data system (PI-RADS) has led to standardization of multiparametric prostate MRI (mpMRI) interpretation and reporting and plays a crucial role for detection and staging of prostate cancer (PC). In the current PI-RADS version 2.1, a PI-RADS lesion in the peripheral zone (PZ) receives a total score of 4 even if the DWI/ADC-subscore is 3, if the dynamic contrast-enhanced MRI (DCE-MRI) is focally positive for early contrast enhancement (PI-RADS 3 + 1 lesion) [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. However, it has been shown that PI-RADS 3 + 1 lesions carry a lower risk for harboring significant prostate cancer (sPC) than PI-RADS 4 lesions with an DWI/ADC-subscore of 4 [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Presence of focal enhancement problematically produces a decision between PI-RADS 2 and 4 in situations where there is a marginal range between non-focal and focal findings on the T2w and ADC images. In these cases, differentiation between PI-RADS 2 and PI-RADS 4 (3 + 1) results is challenging in daily practice, leading to unnecessary biopsies or undetected sPC.</p>
<p id="Par47">The benefit of integrating DCE sequences as part of a multiparametric MRI for prostate imaging overall, as well as in terms of risk stratification of indeterminate ADC-PI-RADS 3 lesions, is subject to growing debate and has not yet been answered [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>].</p>
<p id="Par48">In a first attempt to risk-stratifying PZ lesions, Chatterjee et al. showed that a higher percentage of T2w wedge-shaped PZ lesions were malignant than previously assumed and suggested preferring quantitative ADC values for distinguishing malignant vs. benign conditions [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. In another study with focus on PI-RADS 4 lesions in the PZ, lesion shape according to the PI-RADS v2 lexicon or different patterns of PZ sparing, i.e. normal PZ visible between lesion and prostate capsule (peripheral sparing) or between lesion and transition zone (TZ) (central sparing), were not in general predictive for sPC [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>].</p>
<p id="Par49">Clear sPC feature predictors in the gray area between PI-RADS 2 and 4 are lacking and the definition of a comprehensive feature set for stratification of PI-RADS 3 + 1 lesions is pending. For general PI-RADS lexicon terms the association with sPC, PC or benign prostate changes has been addressed previously [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>] and recent results from the use of deep learning in prostate imaging have been promising [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Nonetheless, additional lexicon terms, clinical features, the discriminatory power of CNN approaches and the standard lexicon terms specifically applied in PI-RADS 3 + 1 lesions remain to be clarified.</p>
<p id="Par50">While it is important to validate any diagnostic imaging approach in a multi-center, multi-vendor setup or even better in different clinical settings before recommending it for integration into clinical practice, this approach may carry difficulties. Indeed, the value of multi-center, multi-vendor studies lies in the consideration of added data heterogeneity, but this always introduces the challenge of maintaining overall compatible high imaging and reporting quality and complicates statistical analyses as the added variability usually requires larger cohort sizes. In this context, there is justifiable reason to evaluate new ideas and concepts first in a single-center, single-vendor setting. In particular, regarding the challenging issue of PI-RADS 3 + 1 at the threshold of inflammatory and incipient suspicious changes, we assume that a homogeneous single-center, single-vendor study performed in an experienced academic center with long-term prostate imaging expertise and use of high-quality MRI protocols may potentially address the complex issue of lesion interpretation.</p>
<p id="Par51">We hypothesized that MR-based criteria can be defined that allow stratification of PI-RADS 3 + 1 lesions into subgroups with different sPC risk. The aim of this study was to retrospectively establish a comprehensive feature set by expert consensus review of mpMRI with reference to extended MRI/TRUS-biopsy ground truth and to evaluate the discriminatory power of the feature set in 3 + 1 lesions through application of the defined criteria by two additional experienced prostate radiologists. We also aimed to put the potential added benefit of the derived features into context with typically available clinical features and with automated AI-based image analysis differentiation both, a biparametric (DWI, T2w) and a multiparametric (DWI, T2w, DCE) information context. Based on these assessments, we set out to determine the optimal strategy for risk assessment utilizing a combination of radiological features, clinical features and AI for improved prediction accuracy and simplification of the decision between biopsy and follow-up.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Patient characteristics</h3>
<p id="Par52">This retrospective study was approved by the institutional ethics committee (institutional ethics approval number S-164/2019) and conducted in accordance to the declaration of Helsinki in its current form. Inclusion criteria of patients were as follows: (1) included in prior patient cohort of our group previously published by Schrader et al. [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>] (2) examination in the time-frame from 08/2016 to 12/2018 and (3) at least one PI-RADS 3 + 1 lesion call in the initial clinical read. Significant PC was defined according to International Society of Urological Pathology (ISUP) grade of ≥ 2. Patient selection flow chart is given in Supplemental Fig. <a href="#MOESM1" class="usa-link">1</a>. The patient cohort has been reported in several previous publications dealing with artificial intelligence and prostate cancer prediction [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>], however, data have not been used in a systematic sub-cohort analysis of PI-RADS 3 + 1 lesions so far.</p></section><section id="Sec4"><h3 class="pmc_sec_title">MRI protocol</h3>
<p id="Par53">Multiparametric MRI including T2w, diffusion-weighted (DWI) and dynamic contrast-enhanced MRI series (DCE) were performed on a 3 Tesla MRI scanner system (Prisma, Siemens Healthcare, Erlangen, Germany) under usage of the standard multichannel body coil and integrated spine-phase-array coil, based on PI-RADS recommendations [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] and guidelines of the European Society of Urogenital Radiology [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. Detailed MRI protocol information can be found in Supplemental Material 1.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Histological correlation with MRI/TRUS-fusion biopsies</h3>
<p id="Par54">All patients underwent sextant-specific systematic extended transperineal MRI/TRUS-fusion biopsies (mean 24.7 cores, IQR 22–28) according to the Ginsburg biopsy scheme as well as fusion-targeted biopsies of suspicious MRI lesions in the initial clinical read with a PI-RADS score ≥ 3 (mean 4.7 cores per lesion, IQR 4.7–6). For details see Supplemental Material <a href="#MOESM1" class="usa-link">S2</a>. This extended biopsy approach at our institutions has showed adequate concordance with radical prostatectomy specimen in previous validations [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>].</p></section><section id="Sec6"><h3 class="pmc_sec_title">PIRADS 3 + 1 lesions: imaging feature set (lexicon of terms) and reassessment of clinically reported lesions in consensus and single reads</h3>
<p id="Par55">Two prostate MRI experts with 14 respectively 8 years of experience in prostate imaging defined a feature set for PIRADS 3 + 1 lesions including general, sequence-based (T2w, DWI and DCE) inter-sequence/correlation-based criteria as mentioned in Fig. <a href="#Fig1" class="usa-link">1</a>. For each feature (i.e. qualitative size) several options (lexicon of terms) were phrased and only the best fitting phrase should be assigned. Features are grouped into new and PI-RADS related features according to Supplemental Table <a href="#MOESM1" class="usa-link">1</a>. All clinically called PI-RADS 3 + 1 lesions were reassessed by consensus read by applying the abovementioned criteria and determining a lesion-based PI-RADS score (2, 3, 3 + 1, 4, 5) to the respective lesion(s) of each patient. In a second step, two board-certified radiologists with &gt; 5 years of experience in prostate imaging were instructed about the feature set and evaluated those lesions independently. All readers were blinded to patients’ clinical and laboratory data.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366217_40644_2025_916_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c235/12366217/6e6e99b9771f/40644_2025_916_Fig1_HTML.jpg" loading="lazy" id="d33e637" height="1092" width="787" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Feature set (lexicon of terms) for risk-stratifying PI-RADS 3 + 1 lesions including general, sequence-based (T2w, DWI and DCE) and inter-sequence/correlation-based feature criteria. For each lesion the best fitting phrase in each category was assigned in the reading sections</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">Biparametric and multiparametric AI analysis</h3>
<p id="Par57">For each examination, multi-parametric axial images were collected: T2-weighted (T2w), DWI with low and high b-values, ADC maps, and 4D-DCE-MRI images. Prostate masks were generated for T2w, DWI, and DCE-MRI using nnUNet-based custom models for each modality. The earliest contrast appearance in DCE, deemed critical by PI-RADS guidelines, was identified using mean signal intensity within the prostate mask. Four time steps surrounding this “wash-in point” were selected to focus on baseline and early enhancement characteristics of DCE. These images were then prepared as input for neural networks. The collected images and their corresponding prostate masks were resampled to 0.58 × 0.58 × 3 mm dimensions. All images were normalized by subtracting their mean and dividing by the standard deviation, in the case of ADC maps using global dataset statistics. Since only T2w annotations were utilized, precise cross-modality registration was used, with image co-registration as previously described [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. The Dice coefficient was used to assess the overlap between fixed T2w and moving DWI/DCE prostate masks. If the overlap was suboptimal (&lt; 0.70), automatic rigid alignment using the masks’ centers of mass was initiated. Training was conducted with a batch size of 6 over 1000 epochs, each containing 250 batches, in a five-fold cross-validation setup. Lesion segmentation networks were created for bpMRI and mpMRI using nnUNet, accommodating either bi-parametric four-parameter (T2w, low- and high b value, ADC map) or 8-parameter multi-parametric (bi-parametric plus four DCE timepoints) inputs. Training and calibration were performed on a dataset of <em>N</em> = 1456, excluding the examinations used for 3 + 1 lesion assessment in this study. After training, inference was performed on the current dataset, and the CNN prediction probability of sPC determined as the maximum softmax value within the manually segmented 3 + 1 lesion mask (denoted as <em>lesion-max</em>).</p></section><section id="Sec8"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par58">Descriptive statistics of the cohort were calculated and tabulated. In a first step, univariable analysis of feature categories was performed, and evaluated using Fisher’s exact test (p-value two-sided). Subsequently, univariate analysis of individual features was performed using Fisher’s exact test (p-value, two-sided). For this analysis, individual features were pre-selected, keeping only those with a prevalence between 10% and 90% and any pairwise correlation below 0.8. Multiple testing adjustment was applied controlling the false discovery rate. The global test approach according to Goeman [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] using a logistic model was applied for identifying potential groups of individual features associated with sPC. To further select relevant features, we performed lasso-penalized logistic regression choosing lambda 1SE (standard error) as penalty based on 20-fold cross-validation.</p>
<p id="Par59">The ROC AUC performance of lasso penalized logistic regression, with respect to sPC association, was evaluated using 4 different models and we also evaluated the biparametric and multiparametric softmax output individually. Each regression also included the following clinical variables/features: “PSA”, “patient’s age”, “histological results of any previous biopsies” (none versus no malignancy versus Gleason 6) and “log-transformed prostate volume” as potential predictors. This resulted in the following models: <em>model 1</em> (all pre-selected consensus read features + clinical features), <em>model 2</em> (all pre-selected consensus read features excluding DCE features + clinical features), <em>model 3</em> (model 2 + biparametric convolutional neural network (bpCNN) lesion-max), <em>model 4</em> (model 1 + multiparametric CNN (mpCNN) lesion-max) as well as <em>solely biparametric CNN</em> (bpCNN lesion-max) and <em>multiparametric CNN</em> (mpCNN lesion-max) as mentioned above. Model performance was assessed based on bootstrapped ROC-AUC. Inter-rater agreement was assessed using Cohen’s weighted kappa. Analyses were performed with software R using add-on packages rms, glmnet and riskRegression.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Study population</h3>
<p id="Par60">Of 673 MR examinations within the inclusion period between August 2016 and December 2018 followed by biopsy at our institutions, 85 examinations in 83 patients (median patient age:64, 45–85 years) were included in the study. Of those patients, 56 (67.5%) were biopsy-naïve, 27 (32.5%) received prior biopsies with evidence of Gleason 6 in 10 patients (12%) and 18 (22%) were enrolled in active surveillance. 83 patients in the study sample had 94 PI-RADS 3 + 1 lesions according to the initial clinical report. 27 of 94 lesions (29%) were positive for sPC at biopsy. Demographic data and patient characteristics are given in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Demographic and clinical characteristics of 83 included men</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Variable</th>
<th align="left" colspan="1" rowspan="1">Cohort <em>n</em> = 83 patients</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Age (years) median (IQR)</td>
<td align="left" colspan="1" rowspan="1">64 (58–69)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PSA (ng/ml) median (IQR)</td>
<td align="left" colspan="1" rowspan="1">7.4 (5.0–10.0)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PSA density median (IQR)</td>
<td align="left" colspan="1" rowspan="1">0.1 (0.1–0.2)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Timing (days) between mpMRI and biopsy median (IQR)</td>
<td align="left" colspan="1" rowspan="1">1 (1-6.25)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>ISUP grading of PIRADS 3 + 1 lesions (<em>n</em> = 94)</p>
<p>PER LESION</p>
</td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No cancer</td>
<td align="left" colspan="1" rowspan="1">48</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISUP 1</td>
<td align="left" colspan="1" rowspan="1">19</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISUP 2</td>
<td align="left" colspan="1" rowspan="1">19</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISUP 3</td>
<td align="left" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISUP 4</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISUP 5</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">No sPC</td>
<td align="left" colspan="1" rowspan="1">67</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">sPC</td>
<td align="left" colspan="1" rowspan="1">27</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p28"><p>Abbreviations PSA = prostate-specific-antigen, ISUP = International Society of Urological Pathology, sPC = clinically significant prostate cancer</p></div></div></section></section><section id="Sec11"><h3 class="pmc_sec_title">Correlation of lesion-based PI-RADS scores by different readers with extended biopsy ground-truth</h3>
<p id="Par62">As stated in Table <a href="#Tab2" class="usa-link">2</a> the expert consensus read resulted in a lesion-based sensitivity and specificity of 96.3% and 26.9% for PI-RADS threshold ≥ 3, 92.6% and 31.3% for PI-RADS threshold ≥ 3 + 1 and 48.1% and 89.6% for PI-RADS threshold ≥ 4. This results in one false-negative lesion for PI-RADS threshold ≥ 3 and two false-negative lesions for PI-RADS threshold ≥ 3 + 1 in the consensus read. The assessment of additional Reader 1 (R1) showed lower sensitivity and higher specificity with 40.7% and 77.6% for PI-RADS threshold ≥ 3, 40.7% and 80.6% for PI-RADS threshold ≥ 3 + 1 and 29.6% and 89.6% for PI-RADS threshold ≥ 4. This yielded 16 falsely negative evaluated lesions each for PI-RADS threshold ≥ 3 and ≥ 3 + 1. The reading assessment of additional single reader 2 showed slightly lower sensitivity as well as partially higher specificity compared to consensus read with 85.2% and 34.3% for PI-RADS threshold ≥ 3, 81.5% and 41.8% for PI-RADS threshold ≥ 3 + 1, and 37 and 70.1% for PI-RADS threshold ≥ 4. Four and accordingly five false negative lesions resulted here for PI-RADS threshold ≥ 3 and PI-RADS threshold ≥ 3 + 1.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Lesion-based diagnostic reader performance (consensus read vs. single-reads) against extended targeted and systematic MRI/TRUS-biopsy histopathology ground truth. Statistical evaluation of reading section under regard of PI-RADS thresholds ≥ 3, ≥3 + 1 and ≥ 4</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1">PI-RADS threshold</th>
<th align="left" colspan="1" rowspan="1">Sensitivity</th>
<th align="left" colspan="1" rowspan="1">Specificity</th>
<th align="left" colspan="1" rowspan="1">tn</th>
<th align="left" colspan="1" rowspan="1">tp</th>
<th align="left" colspan="1" rowspan="1">fn</th>
<th align="left" colspan="1" rowspan="1">fp</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="3" colspan="1">Consensus read</td>
<td align="left" colspan="1" rowspan="1">≥ 3</td>
<td align="center" colspan="1" rowspan="1">96.3</td>
<td align="center" colspan="1" rowspan="1">26.9</td>
<td align="center" colspan="1" rowspan="1">18</td>
<td align="center" colspan="1" rowspan="1">26</td>
<td align="center" colspan="1" rowspan="1">1</td>
<td align="center" colspan="1" rowspan="1">49</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">≥ 3 + 1</td>
<td align="center" colspan="1" rowspan="1">92.6</td>
<td align="center" colspan="1" rowspan="1">31.3</td>
<td align="center" colspan="1" rowspan="1">21</td>
<td align="center" colspan="1" rowspan="1">25</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="center" colspan="1" rowspan="1">46</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">≥ 4</td>
<td align="center" colspan="1" rowspan="1">48.1</td>
<td align="center" colspan="1" rowspan="1">89.6</td>
<td align="center" colspan="1" rowspan="1">60</td>
<td align="center" colspan="1" rowspan="1">13</td>
<td align="center" colspan="1" rowspan="1">14</td>
<td align="center" colspan="1" rowspan="1">7</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Single reader 1</td>
<td align="left" colspan="1" rowspan="1">≥ 3</td>
<td align="center" colspan="1" rowspan="1">40.7</td>
<td align="center" colspan="1" rowspan="1">77.6</td>
<td align="center" colspan="1" rowspan="1">52</td>
<td align="center" colspan="1" rowspan="1">11</td>
<td align="center" colspan="1" rowspan="1">16</td>
<td align="center" colspan="1" rowspan="1">15</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">≥ 3 + 1</td>
<td align="center" colspan="1" rowspan="1">40.7</td>
<td align="center" colspan="1" rowspan="1">80.6</td>
<td align="center" colspan="1" rowspan="1">54</td>
<td align="center" colspan="1" rowspan="1">11</td>
<td align="center" colspan="1" rowspan="1">16</td>
<td align="center" colspan="1" rowspan="1">13</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">≥ 4</td>
<td align="center" colspan="1" rowspan="1">29.6</td>
<td align="center" colspan="1" rowspan="1">89.6</td>
<td align="center" colspan="1" rowspan="1">60</td>
<td align="center" colspan="1" rowspan="1">8</td>
<td align="center" colspan="1" rowspan="1">19</td>
<td align="center" colspan="1" rowspan="1">7</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Single Reader 2</td>
<td align="left" colspan="1" rowspan="1">≥ 3</td>
<td align="center" colspan="1" rowspan="1">85.2</td>
<td align="center" colspan="1" rowspan="1">34.3</td>
<td align="center" colspan="1" rowspan="1">23</td>
<td align="center" colspan="1" rowspan="1">23</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="center" colspan="1" rowspan="1">44</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">≥ 3 + 1</td>
<td align="center" colspan="1" rowspan="1">81.5</td>
<td align="center" colspan="1" rowspan="1">41.8</td>
<td align="center" colspan="1" rowspan="1">28</td>
<td align="center" colspan="1" rowspan="1">22</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">39</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">≥ 4</td>
<td align="center" colspan="1" rowspan="1">37.0</td>
<td align="center" colspan="1" rowspan="1">70.1</td>
<td align="center" colspan="1" rowspan="1">47</td>
<td align="center" colspan="1" rowspan="1">10</td>
<td align="center" colspan="1" rowspan="1">17</td>
<td align="center" colspan="1" rowspan="1">20</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p32"><p>Abbreviations: tn = true negative, tp = true positive, fn = false negative, fp = false positive</p></div></div></section></section><section id="Sec12"><h3 class="pmc_sec_title">Inter-rater agreement</h3>
<p id="Par64">Inter-rater agreement according to Cohen’s kappa between consensus and R1 was 0.2 (slight to fair), between consensus and R2 0.26 (fair) and between R1 and R2 0.22 (fair). As shown in Supplemental Tables <a href="#MOESM1" class="usa-link">2</a>–<a href="#MOESM1" class="usa-link">4</a>, this corresponds to identical PI-RADS decisions between consensus and R1 in 43% of lesions, between consensus and R2 in 62% of lesions and between R1 and R2 in 47% of lesions.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Univariable analysis of feature categories</h3>
<p id="Par65">In the univariate analysis of feature categories based on the consensus read, the categories “DWI suspicion”, “T2w configuration”, “T2w texture”, “DWI configuration” and “qualitative size” were significantly associated with sPC (adj. p-value, all <em>p</em> = 0.041); while the other categories, including “DCE lesion timing (p = 0.058)”, “leading sequence (p = 0.101)”, “DCE match (p = 0.161)”, “DCE configuration (p = 0.214)”, “T2w to DWI match (p = 0.302)”,“DCE margin (p = 0.432)”, “T2w biplanar detection (p = 0.481)”, “localization (p = 0.894)” and “T2w to DCE match (p = 1.0)” were not.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Univariable analysis of individual features</h3>
<p id="Par66">Univariable analysis of individual features from the consensus read before multiplicity adjustment (see Supplemental Table <a href="#MOESM1" class="usa-link">5</a>) identified eight features demonstrating significant effects with regard to sPC prediction. The significant lesion parameters (uncorrected p-value ≤ 0.05) were “T2w configuration irregular/microlobulated/spiculated (T2cims)” (<em>P</em> &lt; 0.001), “T2w texture homogeneous suspicious T2w hypointense (T2thsT2h)” (<em>p</em> = 0.003), “DWI suspicion black and white (DWIbaw)” (<em>p</em> = 0.004), “DWI suspicion ADC subtle/BVAL subtle (DWIsADCsBVALs)” (<em>p</em> = 0.009), “leading sequence T2w (lsT2)” (<em>p</em> = 0.012), “DCE lesion timing earlier than rest (DCEltetr)” (<em>p</em> = 0.018), “T2w texture linear or reticular T2w bright (T2tlorT2b)” (<em>p</em> = 0.02) and “T2w configuration concave sharp borders no perfect wedge (T2ccsbnpw)” (<em>p</em> = 0.03). After subsequent multiplicity adjustment, three features remained significant: “T2cims” (adj. <em>p</em> = 0.016) and subordinate “T2thsT2h” and “DWIbaw” (adj. <em>p</em> = 0.046 each).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Global test approach</h3>
<p id="Par67">Using Goeman’s global test [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] (see Fig. <a href="#Fig2" class="usa-link">2</a>) the parameters “T2cims”, “T2thsT2h”, “DWIbaw” and “DCEltetr” were associated with sPC by decreasing order. Otherwise, the parameters “DWIsADCsBVALs”, “lsT2”, “T2tlorT2b” and “T2ccsbnpw” revealed association with non sPC by decreasing order. As such, there was mostly agreement between the individual feature contribution and the univariable analyses with uncorrected p-values above. When applying multiple testing correction, a significant cluster consisting of “T2cims” and “T2thsT2h” was identified.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366217_40644_2025_916_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c235/12366217/864fda0cfe19/40644_2025_916_Fig2_HTML.jpg" loading="lazy" id="d33e1041" height="711" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Bar chart for selected radiological feature contribution (positive association with sPC in green, negative in red) based on the Goeman test. 4 features each were significantly associated with sPC (p-value &lt; 0.05; “T2w configuration irregular/ microlobulated/ spiculated”, “T2w texture homogeneous suspicious T2w hypointense”, “DWI suspicion black and white” and “DCE lesion timing earlier than rest”) and with non sPC in descending order (“DWI suspicion ADC subtle/BVAL subtle”, “leading sequence T2w”, “T2w texture linear or reticular T2w bright” and “T2w configuration concave sharp borders no perfect wedge”). A significant cluster following multiple-testing adjustment of the individual features “T2w configuration irregular/microlobulated/spiculated” and “T2w texture homogeneous suspicious T2w hypointense” was found.</p></figcaption></figure></section><section id="Sec16"><h3 class="pmc_sec_title">Lasso penalized regression models</h3>
<p id="Par69">Lasso penalized logistic regression models enabled the consideration of clinical features and the reduction of the extensive consensus-based feature models to just a few features (see Table <a href="#Tab3" class="usa-link">3</a>) and underlined the relevance of the feature “T2cims” that remained the first-ranked expert consensus-based feature in all models according to the Odds Ratio. The features “T2thsT2h“ and “DWIbaw” were represented subordinately in all feature-based (model 1 and 2) or feature-based/AI assisted models (model 3 and 4). In model 3, the feature “bpCNN lesion-max” was not selected and the features “T2thsT2h” and “DWIbaw” revealed comparatively small model coefficients. In model 4, with a bootstrapped ROC AUC performance of 0.71, multiparametric CNN (mpCNN lesion-max) was selected and another feature “lsT2” emerged fifth-ranked and negatively correlated with sPC (OR &lt; 1). Of all clinical features, only the feature “log-transformed prostate volume” was selected subordinately and negatively correlated with sPC (OR &lt; 1) in model 1–4.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Lasso penalized logistic regression analysis with remaining individual semantic and clinical features in different feature-based and feature-based/AI-enhanced models (lasso 1SE). In this approach, the number of features could be reduced significantly, with 3 radiologist features and 1 clinical feature remaining even in the same arrangement with regard to OR values in models 1–3. In model 4 the AI-feature mpCNN lesion-max was highest-ranked but the radiological semantic features “T2w configuration: irregular/microlobulated/spiculated”, “DWI suspicion: black and white” and “T2w texture: homogeneous, suspicious T2w hypointense” also remained with an OR &gt; 1 in model 4. The semantic feature “leading sequence: T2w” and the clinical feature “log-transformed prostate volume” in model 4 were selected negatively correlated with sPC (OR &lt; 1)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Model</th>
<th align="left" colspan="1" rowspan="1">Remaining features</th>
<th align="left" colspan="1" rowspan="1">OR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Model 1</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>T2w configuration: irregular/microlobulated/spiculated</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>1.81</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">(radiologist defined features only + clinical features)</td>
<td align="left" colspan="1" rowspan="1">T2w texture: homogeneous, suspicious T2w hypointense</td>
<td align="center" colspan="1" rowspan="1">1.11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">DWI suspicion: black and white</td>
<td align="center" colspan="1" rowspan="1">1.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Log-transformed prostate volume</td>
<td align="center" colspan="1" rowspan="1">0.89</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Model 2</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>T2w configuration: irregular/microlobulated/spiculated</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>2.1</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">(radiologist defined features only excluding DCE features + clinical features)</td>
<td align="left" colspan="1" rowspan="1">T2w texture: homogeneous, suspicious T2w hypointense</td>
<td align="center" colspan="1" rowspan="1">1.26</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">DWI suspicion: black and white</td>
<td align="center" colspan="1" rowspan="1">1.25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Log-transformed prostate volume</td>
<td align="center" colspan="1" rowspan="1">0.76</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Model 3</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>T2w configuration: irregular/microlobulated/spiculated</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>1.81</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">(Model 2 + bp CNN lesion-max)</td>
<td align="left" colspan="1" rowspan="1">T2w texture: homogeneous, suspicious T2w hypointense</td>
<td align="center" colspan="1" rowspan="1">1.11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">DWI suspicion: black and white</td>
<td align="center" colspan="1" rowspan="1">1.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Log-transformed prostate volume</td>
<td align="center" colspan="1" rowspan="1">0.89</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Model 4</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>mpCNN lesion-max</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>3.32</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">(Model 1 + mp CNN lesion-max)</td>
<td align="left" colspan="1" rowspan="1">T2w configuration: irregular/microlobulated/spiculated</td>
<td align="center" colspan="1" rowspan="1">2.33</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">DWI suspicion: black and white</td>
<td align="center" colspan="1" rowspan="1">1.18</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">T2w texture: homogeneous, suspicious T2w hypointense</td>
<td align="center" colspan="1" rowspan="1">1.09</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Leading sequence: T2w</td>
<td align="center" colspan="1" rowspan="1">0.92</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Log-transformed prostate volume</td>
<td align="center" colspan="1" rowspan="1">0.81</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p44"><p>Abbreviations: bp = biparametric, mp = multiparametric, OR = odds ratio, DCE = dynamic contrast enhancement, CNN = convolutional neural networks, DCE = dynamic contrast-enhanced, DWI = diffusion-weighted imaging</p></div></div></section><p id="Par70">Regarding the solely AI-driven models, the multiparametric AI was superior to biparametric AI with AUC of 0.68 vs. 0.64 (Fig. <a href="#Fig3" class="usa-link">3</a>). Case examples of radiological/AI analysis of PI-RADS 3 + 1 lesions are depicted in Figs. <a href="#Fig4" class="usa-link">4</a> and <a href="#Fig5" class="usa-link">5</a>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366217_40644_2025_916_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c235/12366217/1e22e60328b3/40644_2025_916_Fig3_HTML.jpg" loading="lazy" id="d33e1262" height="678" width="744" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Receiver operating characteristic (ROC) curves averaged across bootstrap runs are shown for lasso-penalized logistic regression models 1–4 as well as for solely AI-driven biparametric and multiparametric CNNs (AUC values with corresponding 95% confidence intervals in brackets). Based solely on the consensus read and clinical features, biparametric model 2 even showed a slightly better performance than the multiparametric model 1, indicating no inferiority of the model that excluded all DCE features. AI-assisted biparametric (model 3) and multiparametric (model 4) approaches showed a slight advantage for model 4 with 0.71 vs. 0.69 in terms of AUC while a more pronounced benefit was observed for the solely AI-driven multiparametric CNN compared to the biparametric CNN with 0.68 vs. 0.64. Abbreviations: AUC = area under the curve</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366217_40644_2025_916_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c235/12366217/8fe37843067f/40644_2025_916_Fig4_HTML.jpg" loading="lazy" id="d33e1294" height="458" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Histology-confirmed an ISUP 2 carcinoma. 45 year-old patient with PSA-elevation (5.7 ng/ml, PSA density: 0.31 ng/ml<sup>2</sup>); clinical lesion assessment: PI-RADS 3 + 1 = 4 in the posteromedial peripheral zone (PZpm) in the right apex. This focal lesion revealed T2w hypointense (<strong>A</strong>), with an irregular and partially microlobulated/spiculated T2w configuration and subtle diffusion restriction with signal decrease in the corresponding ADC-map (<strong>B</strong>). In dynamic acquisition (DCE) a distinct early focal enhancement in comparison of initial (<strong>D</strong>) and optimal enhancement phase (<strong>E</strong>) was shown. In expert consensus read it was also classified as PI-RADS 3 + 1 = 4. AI probability score aligns well with the MRI findings and depicts a high probability score for PCa (DL-PI-RADS 4) in bi- and multiparametric CNN (<strong>C</strong> and <strong>F</strong>; axial T2w with respective CNN overlay) with a higher sPC probability score in the DL-PI-RADS 4 continuum for mpCNN.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366217_40644_2025_916_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c235/12366217/2c39157b8322/40644_2025_916_Fig5_HTML.jpg" loading="lazy" id="d33e1326" height="459" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>69-year old patient with PSA-elevation (18 ng/ml, PSA density: 0.18 ng/ml<sup>2</sup>); clinical lesion assessment: PI-RADS 3 + 1 = 4 in the left posteromedial/-lateral peripheral zone in the prostate mid. This focal lesion revealed patchy/macrolobulated with a suspicious homogeneous T2w hypointense T2w texture (<strong>A</strong>) and a subtle diffusion restriction with slight signal decrease in the corresponding ADC-map (<strong>B</strong>). In dynamic acquisition (DCE) a clear early focal enhancement in the initial enhancement phase (<strong>D</strong>) was shown, in optimal enhancement phase (<strong>E</strong>) the surrounding PZ shows nonfocal enhancement with the lesion itself appearing more focally accentuated. In expert consensus read it was classified as PI-RADS 3 + 1 = 4. AI probability score fits well with the MRI findings and shows a high probability score for PCa (DL-PI-RADS 4) in bi- and multiparametric CNN (<strong>C</strong> and <strong>F</strong>; axial T2w with respective CNN overlay) with a higher sPC probability score in the DL-PI-RADS 4 continuum for mpCNN once more. Histology showed no malignancy</p></figcaption></figure><p id="Par71">A technical roadmap summarizing main aspects of methodology and results of our study can be found in Supplemental Fig. <a href="#MOESM1" class="usa-link">2</a>.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Nomogram model - multivariable logistic regression model using selected predictors</h3>
<p id="Par76">Expert read based histopathological correlation of significant lexicon features is mentioned in Table <a href="#Tab4" class="usa-link">4</a>. Based on the three selected semantic and the one clinical feature “prostate volume”, a nomogram for lesion-based prediction of sPC probability was build using a multivariable logistic regression model (as given in Supplemental Fig. <a href="#MOESM1" class="usa-link">3</a>). The predicted sPC probability can be derived from the following formula:</p>
<table class="disp-formula p" id="Equa"><tr><td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c235/12366217/823469b25265/d33e1338.gif" loading="lazy" id="d33e1338" alt="graphic file with name d33e1338.gif"></td></tr></table>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Histopathological correlation of significant lexicon features (key features) based on expert consensus and systematic and targeted extended transperineal MRI/TRUS-fusion biopsy approach. Significant individual features are analyzed according to their ISUP score</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Lesion based ISUP score</th>
<th align="left" colspan="2" rowspan="1">Feature 1<br>T2w configuration: irregular /microlobulated/ spiculated</th>
<th align="left" colspan="2" rowspan="1">Feature 2<br>T2w texture: homogeneous suspicious T2w hypointense</th>
<th align="left" colspan="2" rowspan="1">Feature 3<br>DWI suspicion: black and white</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Feature called (+) / not called (-) in expert consensus read (number of times)</td>
<td align="left" colspan="1" rowspan="1">+ (14)</td>
<td align="left" colspan="1" rowspan="1">− (80)</td>
<td align="left" colspan="1" rowspan="1">+ (27)</td>
<td align="left" colspan="1" rowspan="1">− (67)</td>
<td align="left" colspan="1" rowspan="1">+ (19)</td>
<td align="left" colspan="1" rowspan="1">− (75)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">no cancer</td>
<td align="left" colspan="1" rowspan="1">2 (14.3%)</td>
<td align="left" colspan="1" rowspan="1">46 (57.5%)</td>
<td align="left" colspan="1" rowspan="1">8 (29.6%)</td>
<td align="left" colspan="1" rowspan="1">40 (59.7%)</td>
<td align="left" colspan="1" rowspan="1">5 (26.3%)</td>
<td align="left" colspan="1" rowspan="1">43 (57.3%)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISUP 1</td>
<td align="left" colspan="1" rowspan="1">2 (14.3%)</td>
<td align="left" colspan="1" rowspan="1">17 (21.2%)</td>
<td align="left" colspan="1" rowspan="1">5 (18.5%)</td>
<td align="left" colspan="1" rowspan="1">14 (20.9)</td>
<td align="left" colspan="1" rowspan="1">3 (15.8%)</td>
<td align="left" colspan="1" rowspan="1">16 (21.3%)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISUP 2</td>
<td align="left" colspan="1" rowspan="1">8 (57.1%)</td>
<td align="left" colspan="1" rowspan="1">11 (13.8%)</td>
<td align="left" colspan="1" rowspan="1">8 (29.6%)</td>
<td align="left" colspan="1" rowspan="1">11 (16.4%)</td>
<td align="left" colspan="1" rowspan="1">6 (31.6%)</td>
<td align="left" colspan="1" rowspan="1">13 (17.3%)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISUP 3</td>
<td align="left" colspan="1" rowspan="1">1 (7.1%)</td>
<td align="left" colspan="1" rowspan="1">3 (3.8%)</td>
<td align="left" colspan="1" rowspan="1">3 (11.1%)</td>
<td align="left" colspan="1" rowspan="1">1 (1.5%)</td>
<td align="left" colspan="1" rowspan="1">3 (15.8%)</td>
<td align="left" colspan="1" rowspan="1">1 (1.3%)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISUP 4</td>
<td align="left" colspan="1" rowspan="1">1 (7.1%)</td>
<td align="left" colspan="1" rowspan="1">1 (1.2%)</td>
<td align="left" colspan="1" rowspan="1">2 (7.4%)</td>
<td align="left" colspan="1" rowspan="1">0 (0%)</td>
<td align="left" colspan="1" rowspan="1">1 (5.3%)</td>
<td align="left" colspan="1" rowspan="1">1 (1.3%)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ISUP 5</td>
<td align="left" colspan="1" rowspan="1">0 (0%)</td>
<td align="left" colspan="1" rowspan="1">2 (2.5%)</td>
<td align="left" colspan="1" rowspan="1">1 (3.7%)</td>
<td align="left" colspan="1" rowspan="1">1 (1.5%)</td>
<td align="left" colspan="1" rowspan="1">1 (5.3%)</td>
<td align="left" colspan="1" rowspan="1">1 (1.3%)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par77">with.</p>
<p id="Par78">linear predictor = + 3.454</p>
<p id="Par79">+ 1.953 x (presence of feature “T2cims”)</p>
<p id="Par80">+ 1.238 x (presence of feature “DWIbaw”)</p>
<p id="Par81">+ 0.915 x (presence of feature “T2thsT2h”)</p>
<p id="Par82">− 1.409 x (log-transformed prostate volume)</p></section></section><section id="Sec18"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par84">In this study, lexicon terms were defined and evaluated in consensus and subsequently utilized independently by two radiologists with different reading styles, demonstrating the ability to stratify lesions that clinically all received a DCE-upgraded PI-RADS 4 score (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h3 class="obj_head">Fig. 6.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366217_40644_2025_916_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c235/12366217/d61ed7d9a411/40644_2025_916_Fig6_HTML.jpg" loading="lazy" id="d33e1505" height="627" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Risk stratification of clinically called PI-RADS 3 + 1 lesions based on expert review with generation of a feature catalog that was able to achieve risk stratification of lesions on basis of the consensus read. In a further step, learning and applying the expert criteria also enabled moderate to good risk stratification of other experienced prostate readers. Overall, the criteria make it possible to further differentiate an initially homogeneous, not further diversified lesion cohort</p></figcaption></figure><p id="Par86">Based on expert consensus, five significant feature categories out of 14 were extracted including lesion size and two T2w and DWI categories each. Interestingly, DCE feature categories did not reach significance. This aligns with a previous study by Brancato et al. who highlighted the notable contributions of T2w and ADC radiomic features for stratification of equivocal PI-RADS 3 and upgraded PI-RADS 4 PZ lesions [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. In univariate analysis of the individual features of the lexicon of terms, similar to feature categories, T2w and DWI features dominated compared to the DCE features (5 T2w features, 2 DWI features, 1 DCE feature) before multiplicity adjustment. The fact that the feature “T2cims” remained most significant after multiplicity adjustment indicates that a microlobulated or spiculated lesion appearance should imply a high level of suspicion in clinical assessment of PI-RADS 3 + 1 lesions. This complies with study findings of Rudolph et al., who reported rather high positive predictive values (PPV) for the PZ lexicon descriptors “irregular border” (43.8%), “spiculated border” (56.7%), and “lobulated shape” (56.8%). The marginally significant feature ”DWIbaw”, formally appears to be a PI-RADS 4 category descriptor, but lesions seem to have appeared less conspicuous during clinical interpretation, possibly due to their small size or superimposed inflammation. Lasso penalized regression models based on consensus read and additional clinical features as mentioned above did not show improved outcomes with the use of DCE lexicon terms when evaluated on a feature basis. Hence, the consensus radiologists did not gain a stratification benefit from consideration of DCE variables in the reading section. Prospective studies have already demonstrated the non-inferiority of biparametric MRI compared to mpMRI in detecting sPC while concurrently reducing the detection rates of non-sPC and the number of unnecessary biopsies [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Regarding previous studies in larger screening cohorts without focus on PIRADS 3 + 1 lesions, a notable redundancy of DCE findings (i.e. focal lesion enhancement) compared to findings in T2w and DWI/ADC has been demonstrated. Visschere et al. showed a redundancy of DCE findings in 80.8% of patients, and recommend a multiparametric protocol only in patients with a PI-RADS score of 3 and without suspicious TZ lesions [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. The latter statement contradicts the results of Tavakoli et al. who revealed that neither the quantitative nor the binary qualitative DCE analysis was beneficial in the prediction of clinical sPC in ADC PI-RADS 3 lesions and did not improve the distinction of lesions with intermediate suspicion [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Chatterjee et al. revealed that wedge-shaped PZ lesions, a common PI-RADS 3 + 1 morphologic appearance, do not benefit from utilization of early enhancement in distinguishing between benign and malignant lesions [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Aiming to improve MRI accuracy and reduce false positives, Messina proposed to biopsy PIRADS 3 and PIRADS 3up lesions only in cases of a PSA densitiy ≥ 0.15ng/ml<sup>2</sup> [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. However, the study protocol revealed several relevant limitations, i.e. a debatable PSA density cutoff and lack of biopsies in the PIRADS 3/3 + 1 subcohort without elevated PSA density [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>] leading to a possibility of undetected prostate carcinomas in the cohort [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Thus, the role of PSA density in 3 + 1 lesions is still unclear. When we corrected for PSA density in our models (data not shown), the features “T2cims” and “T2thsT2h” were still selected into the models, showing that these imaging-based features are not just a surrogate of PSAD. In contrast, based on our own previous data, PI-RADS 3 lesions could be risk-stratified by adding a PSA density cutoff of &lt; 0.1 ng/ml² with a 98% probability of excluding significant prostate cancer [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Of all considered clinical features in our study solely “log-transformed prostate volume” was selected into the models, however with lower contribution than the best semantic feature. This finding is plausible, as PSA elevation in patients with increased prostate volume may lead to the indication for MR evalution while BPH is an independent and non-malignant factor for PSA elevation.</p>
<p id="Par87">In AI-based analysis, DCE resulted in a trend for improvement in diagnostic accuracy (ROC AUC increase from 0.64 to 0.68, non-significant in bootstrap AUC analysis), in agreement with previous studies [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>, <a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Matsuoka revealed improvements in specificity and positive predictive value within the PZ and fewer false-positives compared to biparametric AI approach [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] in multiparametric models. In our study, when radiologist-identified features were combined with AI predictions, the mpMRI AI model (using DCE information) provided a trend for a benefit to radiologists using lexicon terms, while the bpMRI model did not. Thus, while for radiologist assessment, DCE did not improve diagnostic performance, mpCNN may be able to provide complementary information.</p>
<p id="Par88">The provided nomogram may provide decision support whether to perform prostate biopsy in the future, however requires confirmation in further studies.</p>
<p id="Par89">Limitations of our study include the retrospective, single-institution, single-vendor setup, albeit in a high-volume imaging center with a high level of diagnostic experience and continued review of images in combination with extended histopathology results as discussed above. We are aware that retrospective studies carry the risk for underlying bias, which would not arise in prospective analysis. We have tried to design the study setup and the inclusion criteria in such a manner that the risk for bias was removed as best we could. There was no specific patient selection for this study than including prostate MRI examinations within a continuous time frame of 29 months that had at least one peripheral PI-RADS 3 + 1 lesion in the official clinical radiology report. We also assume that the study design was chosen carefully enough to avoid the risk for recall bias i.e. by long-enough wash-out periods comparable to those found in the literature. Multi-centric and multi-vendor study settings are certainly advantageous and necessary to transfer study results into clinical practice. Those setups are widely discussed in the AI community and primarily serve to further investigate performances of models that have already shown good results in mono-institutional studies. Given the scarce evidence of studies addressing the topic of PI-RADS 3 + 1, we believe our study highlights open questions which have not been addressed to more extent so far, even in a mono-institutional study setup. We even believe that the homogeneity of data can actually be of advantage. Multi-vendor and multi-institution studies would in turn mean added variability and data heterogeneity that would increase the number of prostate MRI cases to be assessed to discover potential underlying relationships. Furthermore an even larger sample size would have been beneficial e.g. with regard to even more robust models and statistical power. However, it should be noted that the scarcity of PI-RADS 3 + 1 cases in itself limits the available data. Otherwise, the sample size in our study enabled a detailed evaluation of PI-RADS 3 + 1 lesions by a total of 4 experienced radiologists. We utilized in-house developed and trained deep learning pipelines, which have not been validated in many institutions. However, validation has been performed to extra-institutional data, both using public datasets [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>] and bi-institutionally [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>].</p>
<p id="Par90">In conclusion, we addressed a large number of clinical PI-RADS 3 + 1 lesions by re-evaluating them both by expert consensus and independent board-certified radiologist re-assessment, thereby establishing radiological and clinical key features for nomogram building that exhibit potential to aid in improved sPC detection. By comparing them to deep-learning performance, we find no significant contribution of DCE, although trends are discovered suggesting that deep learning may contribute only to radiological feature assessment when multiparametric information, but not when only biparametric information is utilized in DL models, findings that require further evaluation in larger cohorts. We believe that this study performed in a homogeneous setting provides important novel insights into the assessment of PI-RADS 3 + 1 lesions that may inform larger future multi-centric, multi-vendor studies.</p></section><section id="Sec19"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366217/bin/40644_2025_916_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (370.6KB, docx) </sup>
</div></div></section></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>adj.</dt>
<dd><p id="Par6">Adjusted</p></dd>
<dt>ADC</dt>
<dd><p id="Par7">Apparent diffusion coefficient</p></dd>
<dt>AI</dt>
<dd><p id="Par8">Artificial intelligence</p></dd>
<dt>AUC</dt>
<dd><p id="Par9">Area under the receiver operating characteristics curve</p></dd>
<dt>bpMRI</dt>
<dd><p id="Par10">Biparametric magnetic resonance imaging</p></dd>
<dt>CNN</dt>
<dd><p id="Par11">Convolutional neural network</p></dd>
<dt>(c)sPC</dt>
<dd><p id="Par12">(clinically) significant prostate cancer</p></dd>
<dt>DCE</dt>
<dd><p id="Par13">Dynamic contrast enhancement</p></dd>
<dt>DCEltetr</dt>
<dd><p id="Par14">DCE lesion timing earlier than rest</p></dd>
<dt>DL</dt>
<dd><p id="Par15">Deep learning</p></dd>
<dt>DWI</dt>
<dd><p id="Par16">Diffusion-weighted images</p></dd>
<dt>DWIbaw</dt>
<dd><p id="Par17">DWI suspicion: black and white</p></dd>
<dt>DWIsADCsBVALs</dt>
<dd><p id="Par18">DWI suspicion: ADC subtle/BVAL subtle</p></dd>
<dt>ESUR</dt>
<dd><p id="Par19">European Society of Urogenital Radiology</p></dd>
<dt>ICC</dt>
<dd><p id="Par20">Intraclass correlation coefficient</p></dd>
<dt>ISUP</dt>
<dd><p id="Par21">International Society of Urological Pathology</p></dd>
<dt>lsT2</dt>
<dd><p id="Par22">Leading sequence: T2w</p></dd>
<dt>MR</dt>
<dd><p id="Par23">Magnetic resonance</p></dd>
<dt>MRI</dt>
<dd><p id="Par24">Magnetic resonance imaging</p></dd>
<dt>mpMRI</dt>
<dd><p id="Par25">Multiparametric magnetic resonance imaging</p></dd>
<dt>nnUNet</dt>
<dd><p id="Par26">no new U-Net</p></dd>
<dt>OR</dt>
<dd><p id="Par27">Odds ratio</p></dd>
<dt>PC</dt>
<dd><p id="Par28">Prostate cancer</p></dd>
<dt>PI-RADS</dt>
<dd><p id="Par29">Prostate Imaging Reporting and Data System</p></dd>
<dt>PI-RADS</dt>
<dd><p id="Par30">3up upgraded PI-RADS 3 lesion, alternative term to PI-RADS 3 + 1</p></dd>
<dt>PPV</dt>
<dd><p id="Par31">Positive predictive value</p></dd>
<dt>PSA(D)</dt>
<dd><p id="Par32">Prostate-specific antigen (density)</p></dd>
<dt>PZ</dt>
<dd><p id="Par33">Peripheral zone</p></dd>
<dt>R1</dt>
<dd><p id="Par34">Reader 1</p></dd>
<dt>R2</dt>
<dd><p id="Par35">Reader 2</p></dd>
<dt>ROC</dt>
<dd><p id="Par36">Receiver operating characteristics</p></dd>
<dt>SE</dt>
<dd><p id="Par37">Standard error</p></dd>
<dt>sPC</dt>
<dd><p id="Par38">Significant prostate cancer</p></dd>
<dt>T2w</dt>
<dd><p id="Par39">T2-weighted</p></dd>
<dt>T2cims</dt>
<dd><p id="Par40">T2w configuration: irregular/microlobulated/spiculated</p></dd>
<dt>T2ccsbnpw</dt>
<dd><p id="Par41">T2w configuration: concave sharp borders no perfect wedge</p></dd>
<dt>T2thsT2h</dt>
<dd><p id="Par42">T2w texture: homogeneous suspicious T2w hypointense</p></dd>
<dt>T2tlorT2b</dt>
<dd><p id="Par43">T2w texture: linear or reticular T2w bright</p></dd>
<dt>TRUS</dt>
<dd><p id="Par44">Transrectal ultrasound</p></dd>
<dt>TZ</dt>
<dd><p id="Par45">Transitional zone</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>P. G. and D.B. designed and supervised the study, performed the expert consensus read, analyzed the data and drafted the manuscript. N. N. performed the AI analyses, contributed to data interpretation and drafted the manuscript. T. H. performed the statistical analysis and drafted the manuscript. C. H. Z. and M. W. took part in lesion analysis as single readers and revised the manuscript. M.G., V. S., M. H., A. S., K. S. Z. and H. P. S. contributed to data interpretation and revised the manuscript. The author(s) read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open Access funding enabled and organized by Projekt DEAL. This research project received research support from Bundesministerium für Wirtschaft und Klimaschutz (BMWK): grant no. 01MT21004B.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data used in this manuscript are subject to local privacy and data protection laws and not publicly available. Requests for anonymized data review can be directed to the authors and will be reviewed in co-operation with the responsible data protection officers.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par91">The study was approved by the local ethics committee (institutional ethics approval number S-164/2019) in accordance with the 1964 Helsinki declaration and its later amendments or comparable ethical standards. Informed consent requirement was waived.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par92">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par93">Philip A Glemser, Nils Netzer, Christian H Ziener, Markus Wilhelm, Thomas Hielscher, Kevin S Zhang, Magdalena Görtz, Viktoria Schütz, Albrecht Stenzinger, Markus Hohenfellner: No relevant relationships.Heinz-Peter Schlemmer: Grants from Federal Ministry for Economy and German Aerospace Center, honoria from Bayer Health, Bracco, and Siemens; travel reimbursement from Siemens Healthineers; Bayer Vital and Bayer Health; advisory board, Siemens Healthineers.David Bonekamp: Lecture payment from Bayer Vital.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Turkbey B, Rosenkrantz AB, Haider MA, Padhani AR, Villeirs G, Macura KJ, et al. Eur Urol. 2019;76(3):340–51. Prostate Imaging Reporting and Data System Version 2.1: 2019 Update of Prostate Imaging Reporting and Data System Version 2.</cite> [<a href="https://doi.org/10.1016/j.eururo.2019.02.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30898406/" class="usa-link">PubMed</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Shankar PR, Curci NE, Davenport MS. Characteristics of PI-RADS 4 lesions within the prostatic peripheral zone: a retrospective diagnostic accuracy study evaluating 170 lesions. Abdom Radiol (NY). 2018;43(8):2176–82.
</cite> [<a href="https://doi.org/10.1007/s00261-017-1415-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29198006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shankar%20PR,%20Curci%20NE,%20Davenport%20MS.%20Characteristics%20of%20PI-RADS%204%20lesions%20within%20the%20prostatic%20peripheral%20zone:%20a%20retrospective%20diagnostic%20accuracy%20study%20evaluating%20170%20lesions.%20Abdom%20Radiol%20(NY).%202018;43(8):2176%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Rosenkrantz AB, Babb JS, Taneja SS, Ream JM. Proposed adjustments to PI-RADS version 2 decision rules: impact on prostate cancer detection. Radiology. 2017;283(1):119–29.
</cite> [<a href="https://doi.org/10.1148/radiol.2016161124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27783538/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosenkrantz%20AB,%20Babb%20JS,%20Taneja%20SS,%20Ream%20JM.%20Proposed%20adjustments%20to%20PI-RADS%20version%202%20decision%20rules:%20impact%20on%20prostate%20cancer%20detection.%20Radiology.%202017;283(1):119%E2%80%9329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Greer MD, Shih JH, Lay N, Barrett T, Kayat Bittencourt L, Borofsky S, et al. Validation of the dominant sequence paradigm and role of dynamic Contrast-enhanced imaging in PI-RADS version 2. Radiology. 2017;285(3):859–69.
</cite> [<a href="https://doi.org/10.1148/radiol.2017161316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5708285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28727501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greer%20MD,%20Shih%20JH,%20Lay%20N,%20Barrett%20T,%20Kayat%20Bittencourt%20L,%20Borofsky%20S,%20et%20al.%20Validation%20of%20the%20dominant%20sequence%20paradigm%20and%20role%20of%20dynamic%20Contrast-enhanced%20imaging%20in%20PI-RADS%20version%202.%20Radiology.%202017;285(3):859%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Druskin SC, Ward R, Purysko AS, Young A, Tosoian JJ, Ghabili K, et al. Dynamic contrast enhanced magnetic resonance imaging improves classification of prostate lesions: a study of pathological outcomes on targeted prostate biopsy. J Urol. 2017;198(6):1301–8.
</cite> [<a href="https://doi.org/10.1016/j.juro.2017.07.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28709889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Druskin%20SC,%20Ward%20R,%20Purysko%20AS,%20Young%20A,%20Tosoian%20JJ,%20Ghabili%20K,%20et%20al.%20Dynamic%20contrast%20enhanced%20magnetic%20resonance%20imaging%20improves%20classification%20of%20prostate%20lesions:%20a%20study%20of%20pathological%20outcomes%20on%20targeted%20prostate%20biopsy.%20J%20Urol.%202017;198(6):1301%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Kuhl CK, Bruhn R, Krämer N, Nebelung S, Heidenreich A, Schrading S. Abbreviated biparametric prostate MR imaging in men with elevated prostate-specific antigen. Radiology. 2017;285(2):493–505.
</cite> [<a href="https://doi.org/10.1148/radiol.2017170129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28727544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuhl%20CK,%20Bruhn%20R,%20Kr%C3%A4mer%20N,%20Nebelung%20S,%20Heidenreich%20A,%20Schrading%20S.%20Abbreviated%20biparametric%20prostate%20MR%20imaging%20in%20men%20with%20elevated%20prostate-specific%20antigen.%20Radiology.%202017;285(2):493%E2%80%93505." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>van der Leest M, Israël B, Cornel EB, Zámecnik P, Schoots IG, van der Lelij H, et al. High diagnostic performance of short magnetic resonance imaging protocols for prostate cancer detection in Biopsy-naïve men: the next step in magnetic resonance imaging accessibility. Eur Urol. 2019;76(5):574–81.
</cite> [<a href="https://doi.org/10.1016/j.eururo.2019.05.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31167748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?van%20der%20Leest%20M,%20Isra%C3%ABl%20B,%20Cornel%20EB,%20Z%C3%A1mecnik%20P,%20Schoots%20IG,%20van%20der%20Lelij%20H,%20et%20al.%20High%20diagnostic%20performance%20of%20short%20magnetic%20resonance%20imaging%20protocols%20for%20prostate%20cancer%20detection%20in%20Biopsy-na%C3%AFve%20men:%20the%20next%20step%20in%20magnetic%20resonance%20imaging%20accessibility.%20Eur%20Urol.%202019;76(5):574%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Xu L, Zhang G, Shi B, Liu Y, Zou T, Yan W, et al. Comparison of biparametric and multiparametric MRI in the diagnosis of prostate cancer. Cancer Imaging. 2019;19(1):90.
</cite> [<a href="https://doi.org/10.1186/s40644-019-0274-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6925429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31864408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20L,%20Zhang%20G,%20Shi%20B,%20Liu%20Y,%20Zou%20T,%20Yan%20W,%20et%20al.%20Comparison%20of%20biparametric%20and%20multiparametric%20MRI%20in%20the%20diagnosis%20of%20prostate%20cancer.%20Cancer%20Imaging.%202019;19(1):90." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Ullrich T, Quentin M, Arsov C, Laqua N, Abrar D, Hiester A, et al. Value of dynamic Contrast-Enhanced (DCE) MR imaging in peripheral lesions in PI-RADS-4 patients. Rofo. 2020;192(5):441–7.
</cite> [<a href="https://doi.org/10.1055/a-1020-4026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31622990/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ullrich%20T,%20Quentin%20M,%20Arsov%20C,%20Laqua%20N,%20Abrar%20D,%20Hiester%20A,%20et%20al.%20Value%20of%20dynamic%20Contrast-Enhanced%20(DCE)%20MR%20imaging%20in%20peripheral%20lesions%20in%20PI-RADS-4%20patients.%20Rofo.%202020;192(5):441%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Ullrich T, Schimmöller L. Perspective: a critical assessment of PI-RADS 2.1. Abdom Radiol (NY). 2020;45(12):3961–8.
</cite> [<a href="https://doi.org/10.1007/s00261-020-02424-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32009207/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ullrich%20T,%20Schimm%C3%B6ller%20L.%20Perspective:%20a%20critical%20assessment%20of%20PI-RADS%202.1.%20Abdom%20Radiol%20(NY).%202020;45(12):3961%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Bass EJ, Pantovic A, Connor M, Gabe R, Padhani AR, Rockall A, et al. A systematic review and meta-analysis of the diagnostic accuracy of biparametric prostate MRI for prostate cancer in men at risk. Prostate Cancer Prostatic Dis. 2021;24(3):596–611.
</cite> [<a href="https://doi.org/10.1038/s41391-020-00298-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33219368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bass%20EJ,%20Pantovic%20A,%20Connor%20M,%20Gabe%20R,%20Padhani%20AR,%20Rockall%20A,%20et%20al.%20A%20systematic%20review%20and%20meta-analysis%20of%20the%20diagnostic%20accuracy%20of%20biparametric%20prostate%20MRI%20for%20prostate%20cancer%20in%20men%20at%20risk.%20Prostate%20Cancer%20Prostatic%20Dis.%202021;24(3):596%E2%80%93611." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Tavakoli AA, Hielscher T, Badura P, Görtz M, Kuder TA, Gnirs R, et al. Contribution of dynamic Contrast-enhanced and diffusion MRI to PI-RADS for detecting clinically significant prostate cancer. Radiology. 2023;306(1):186–99.
</cite> [<a href="https://doi.org/10.1148/radiol.212692" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35972360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tavakoli%20AA,%20Hielscher%20T,%20Badura%20P,%20G%C3%B6rtz%20M,%20Kuder%20TA,%20Gnirs%20R,%20et%20al.%20Contribution%20of%20dynamic%20Contrast-enhanced%20and%20diffusion%20MRI%20to%20PI-RADS%20for%20detecting%20clinically%20significant%20prostate%20cancer.%20Radiology.%202023;306(1):186%E2%80%9399." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Messina E, Pecoraro M, Laschena L, Bicchetti M, Proietti F, Ciardi A, et al. Low cancer yield in PI-RADS 3 upgraded to 4 by dynamic contrast-enhanced MRI: is it time to reconsider scoring categorization? Eur Radiol. 2023;33(8):5828–39.
</cite> [<a href="https://doi.org/10.1007/s00330-023-09605-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10326099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37045981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Messina%20E,%20Pecoraro%20M,%20Laschena%20L,%20Bicchetti%20M,%20Proietti%20F,%20Ciardi%20A,%20et%20al.%20Low%20cancer%20yield%20in%20PI-RADS%203%20upgraded%20to%204%20by%20dynamic%20contrast-enhanced%20MRI:%20is%20it%20time%20to%20reconsider%20scoring%20categorization?%20Eur%20Radiol.%202023;33(8):5828%E2%80%9339." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Chatterjee A, Tokdemir S, Gallan AJ, Yousuf A, Antic T, Karczmar GS, et al. Multiparametric MRI features and pathologic outcome of Wedge-Shaped lesions in the peripheral zone on T2-Weighted images of the prostate. AJR Am J Roentgenol. 2019;212(1):124–9.
</cite> [<a href="https://doi.org/10.2214/AJR.18.19742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6437777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30403532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chatterjee%20A,%20Tokdemir%20S,%20Gallan%20AJ,%20Yousuf%20A,%20Antic%20T,%20Karczmar%20GS,%20et%20al.%20Multiparametric%20MRI%20features%20and%20pathologic%20outcome%20of%20Wedge-Shaped%20lesions%20in%20the%20peripheral%20zone%20on%20T2-Weighted%20images%20of%20the%20prostate.%20AJR%20Am%20J%20Roentgenol.%202019;212(1):124%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Rudolph MM, Baur ADJ, Haas M, Cash H, Miller K, Mahjoub S, et al. Validation of the PI-RADS language: predictive values of PI-RADS lexicon descriptors for detection of prostate cancer. Eur Radiol. 2020;30(8):4262–71.
</cite> [<a href="https://doi.org/10.1007/s00330-020-06773-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7338829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32219507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rudolph%20MM,%20Baur%20ADJ,%20Haas%20M,%20Cash%20H,%20Miller%20K,%20Mahjoub%20S,%20et%20al.%20Validation%20of%20the%20PI-RADS%20language:%20predictive%20values%20of%20PI-RADS%20lexicon%20descriptors%20for%20detection%20of%20prostate%20cancer.%20Eur%20Radiol.%202020;30(8):4262%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Schelb P, Kohl S, Radtke JP, Wiesenfarth M, Kickingereder P, Bickelhaupt S, et al. Classification of cancer at prostate MRI: deep learning versus clinical PI-RADS assessment. Radiology. 2019;293(3):607–17.
</cite> [<a href="https://doi.org/10.1148/radiol.2019190938" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31592731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schelb%20P,%20Kohl%20S,%20Radtke%20JP,%20Wiesenfarth%20M,%20Kickingereder%20P,%20Bickelhaupt%20S,%20et%20al.%20Classification%20of%20cancer%20at%20prostate%20MRI:%20deep%20learning%20versus%20clinical%20PI-RADS%20assessment.%20Radiology.%202019;293(3):607%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Schelb P, Wang X, Radtke JP, Wiesenfarth M, Kickingereder P, Stenzinger A, et al. Simulated clinical deployment of fully automatic deep learning for clinical prostate MRI assessment. Eur Radiol. 2021;31(1):302–13.
</cite> [<a href="https://doi.org/10.1007/s00330-020-07086-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7755653/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32767102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schelb%20P,%20Wang%20X,%20Radtke%20JP,%20Wiesenfarth%20M,%20Kickingereder%20P,%20Stenzinger%20A,%20et%20al.%20Simulated%20clinical%20deployment%20of%20fully%20automatic%20deep%20learning%20for%20clinical%20prostate%20MRI%20assessment.%20Eur%20Radiol.%202021;31(1):302%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Bonekamp D, Kohl S, Wiesenfarth M, Schelb P, Radtke JP, Götz M, et al. Radiomic machine learning for characterization of prostate lesions with MRI: comparison to ADC values. Radiology. 2018;289(1):128–37.
</cite> [<a href="https://doi.org/10.1148/radiol.2018173064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30063191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bonekamp%20D,%20Kohl%20S,%20Wiesenfarth%20M,%20Schelb%20P,%20Radtke%20JP,%20G%C3%B6tz%20M,%20et%20al.%20Radiomic%20machine%20learning%20for%20characterization%20of%20prostate%20lesions%20with%20MRI:%20comparison%20to%20ADC%20values.%20Radiology.%202018;289(1):128%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Netzer N, Weißer C, Schelb P, Wang X, Qin X, Görtz M, et al. Fully automatic deep learning in Bi-institutional prostate magnetic resonance imaging: effects of cohort size and heterogeneity. Invest Radiol. 2021;56(12):799–808.
</cite> [<a href="https://doi.org/10.1097/RLI.0000000000000791" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34049336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Netzer%20N,%20Wei%C3%9Fer%20C,%20Schelb%20P,%20Wang%20X,%20Qin%20X,%20G%C3%B6rtz%20M,%20et%20al.%20Fully%20automatic%20deep%20learning%20in%20Bi-institutional%20prostate%20magnetic%20resonance%20imaging:%20effects%20of%20cohort%20size%20and%20heterogeneity.%20Invest%20Radiol.%202021;56(12):799%E2%80%93808." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Schrader A, Netzer N, Hielscher T, Görtz M, Zhang KS, Schütz V et al. Prostate cancer risk assessment and avoidance of prostate biopsies using fully automatic deep learning in prostate MRI: comparison to PI-RADS and integration with clinical data in nomograms. Eur Radiol. 2024; 34(12):7909-2920</cite> [<a href="https://doi.org/10.1007/s00330-024-10818-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11557625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38955845/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Zhang KS, Schelb P, Kohl S, Radtke JP, Wiesenfarth M, Schimmöller L, et al. Improvement of PI-RADS-dependent prostate cancer classification by quantitative image assessment using radiomics or mean ADC. Magn Reson Imaging. 2021;82:9–17.
</cite> [<a href="https://doi.org/10.1016/j.mri.2021.06.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34147597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20KS,%20Schelb%20P,%20Kohl%20S,%20Radtke%20JP,%20Wiesenfarth%20M,%20Schimm%C3%B6ller%20L,%20et%20al.%20Improvement%20of%20PI-RADS-dependent%20prostate%20cancer%20classification%20by%20quantitative%20image%20assessment%20using%20radiomics%20or%20mean%20ADC.%20Magn%20Reson%20Imaging.%202021;82:9%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Zhang KS, Schelb P, Netzer N, Tavakoli AA, Keymling M, Wehrse E, et al. Pseudoprospective paraclinical interaction of radiology residents with a deep learning system for prostate cancer detection: experience, performance, and identification of the need for intermittent recalibration. Invest Radiol. 2022;57(9):601–12.
</cite> [<a href="https://doi.org/10.1097/RLI.0000000000000878" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35467572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20KS,%20Schelb%20P,%20Netzer%20N,%20Tavakoli%20AA,%20Keymling%20M,%20Wehrse%20E,%20et%20al.%20Pseudoprospective%20paraclinical%20interaction%20of%20radiology%20residents%20with%20a%20deep%20learning%20system%20for%20prostate%20cancer%20detection:%20experience,%20performance,%20and%20identification%20of%20the%20need%20for%20intermittent%20recalibration.%20Invest%20Radiol.%202022;57(9):601%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Netzer N, Eith C, Bethge O, Hielscher T, Schwab C, Stenzinger A, et al. Application of a validated prostate MRI deep learning system to independent same-vendor multi-institutional data: demonstration of transferability. Eur Radiol. 2023;33(11):7463–76.
</cite> [<a href="https://doi.org/10.1007/s00330-023-09882-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10598076/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37507610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Netzer%20N,%20Eith%20C,%20Bethge%20O,%20Hielscher%20T,%20Schwab%20C,%20Stenzinger%20A,%20et%20al.%20Application%20of%20a%20validated%20prostate%20MRI%20deep%20learning%20system%20to%20independent%20same-vendor%20multi-institutional%20data:%20demonstration%20of%20transferability.%20Eur%20Radiol.%202023;33(11):7463%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Weinreb JC, Barentsz JO, Choyke PL, Cornud F, Haider MA, Macura KJ, et al. Eur Urol. 2016;69(1):16–40. PI-RADS Prostate Imaging - Reporting and Data System: 2015, Version 2.</cite> [<a href="https://doi.org/10.1016/j.eururo.2015.08.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6467207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26427566/" class="usa-link">PubMed</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Barentsz JO, Richenberg J, Clements R, Choyke P, Verma S, Villeirs G, et al. ESUR prostate MR guidelines 2012. Eur Radiol. 2012;22(4):746–57.
</cite> [<a href="https://doi.org/10.1007/s00330-011-2377-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3297750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22322308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barentsz%20JO,%20Richenberg%20J,%20Clements%20R,%20Choyke%20P,%20Verma%20S,%20Villeirs%20G,%20et%20al.%20ESUR%20prostate%20MR%20guidelines%202012.%20Eur%20Radiol.%202012;22(4):746%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Dickinson L, Ahmed HU, Allen C, Barentsz JO, Carey B, Futterer JJ, et al. Magnetic resonance imaging for the detection, localisation, and characterisation of prostate cancer: recommendations from a European consensus meeting. Eur Urol. 2011;59(4):477–94.
</cite> [<a href="https://doi.org/10.1016/j.eururo.2010.12.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21195536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dickinson%20L,%20Ahmed%20HU,%20Allen%20C,%20Barentsz%20JO,%20Carey%20B,%20Futterer%20JJ,%20et%20al.%20Magnetic%20resonance%20imaging%20for%20the%20detection,%20localisation,%20and%20characterisation%20of%20prostate%20cancer:%20recommendations%20from%20a%20European%20consensus%20meeting.%20Eur%20Urol.%202011;59(4):477%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Radtke JP, Schwab C, Wolf MB, Freitag MT, Alt CD, Kesch C, et al. Multiparametric magnetic resonance imaging (MRI) and MRI-Transrectal ultrasound fusion biopsy for index tumor detection: correlation with radical prostatectomy specimen. Eur Urol. 2016;70(5):846–53.
</cite> [<a href="https://doi.org/10.1016/j.eururo.2015.12.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26810346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Radtke%20JP,%20Schwab%20C,%20Wolf%20MB,%20Freitag%20MT,%20Alt%20CD,%20Kesch%20C,%20et%20al.%20Multiparametric%20magnetic%20resonance%20imaging%20(MRI)%20and%20MRI-Transrectal%20ultrasound%20fusion%20biopsy%20for%20index%20tumor%20detection:%20correlation%20with%20radical%20prostatectomy%20specimen.%20Eur%20Urol.%202016;70(5):846%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Goeman JJ, van de Geer SA, de Kort F, van Houwelingen HC. A global test for groups of genes: testing association with a clinical outcome. Bioinformatics. 2004;20(1):93–9.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btg382" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14693814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goeman%20JJ,%20van%20de%20Geer%20SA,%20de%20Kort%20F,%20van%20Houwelingen%20HC.%20A%20global%20test%20for%20groups%20of%20genes:%20testing%20association%20with%20a%20clinical%20outcome.%20Bioinformatics.%202004;20(1):93%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Brancato V, Aiello M, Basso L, Monti S, Palumbo L, Di Costanzo G, et al. Evaluation of a multiparametric MRI radiomic-based approach for stratification of equivocal PI-RADS 3 and upgraded PI-RADS 4 prostatic lesions. Sci Rep. 2021;11(1):643.
</cite> [<a href="https://doi.org/10.1038/s41598-020-80749-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7804929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33436929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brancato%20V,%20Aiello%20M,%20Basso%20L,%20Monti%20S,%20Palumbo%20L,%20Di%20Costanzo%20G,%20et%20al.%20Evaluation%20of%20a%20multiparametric%20MRI%20radiomic-based%20approach%20for%20stratification%20of%20equivocal%20PI-RADS%203%20and%20upgraded%20PI-RADS%204%20prostatic%20lesions.%20Sci%20Rep.%202021;11(1):643." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Sonni I, Felker ER, Lenis AT, Sisk AE, Bahri S, Allen-Auerbach M, et al. Head-to-Head comparison of (68)Ga-PSMA-11 PET/CT and MpMRI with a histopathology gold standard in the detection, intraprostatic localization, and determination of local extension of primary prostate cancer: results from a prospective Single-Center imaging trial. J Nucl Med. 2022;63(6):847–54.
</cite> [<a href="https://doi.org/10.2967/jnumed.121.262398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9157724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34649942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sonni%20I,%20Felker%20ER,%20Lenis%20AT,%20Sisk%20AE,%20Bahri%20S,%20Allen-Auerbach%20M,%20et%20al.%20Head-to-Head%20comparison%20of%20(68)Ga-PSMA-11%20PET/CT%20and%20MpMRI%20with%20a%20histopathology%20gold%20standard%20in%20the%20detection,%20intraprostatic%20localization,%20and%20determination%20of%20local%20extension%20of%20primary%20prostate%20cancer:%20results%20from%20a%20prospective%20Single-Center%20imaging%20trial.%20J%20Nucl%20Med.%202022;63(6):847%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Hao S, Discacciati A, Eklund M, Heintz E, Östensson E, Elfström KM, et al. Cost-effectiveness of prostate cancer screening using magnetic resonance imaging or standard biopsy based on the STHLM3-MRI study. JAMA Oncol. 2022;9(1):88–94.
</cite> [<a href="https://doi.org/10.1001/jamaoncol.2022.5252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9650623/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36355382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao%20S,%20Discacciati%20A,%20Eklund%20M,%20Heintz%20E,%20%C3%96stensson%20E,%20Elfstr%C3%B6m%20KM,%20et%20al.%20Cost-effectiveness%20of%20prostate%20cancer%20screening%20using%20magnetic%20resonance%20imaging%20or%20standard%20biopsy%20based%20on%20the%20STHLM3-MRI%20study.%20JAMA%20Oncol.%202022;9(1):88%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>De Visschere P, Lumen N, Ost P, Decaestecker K, Pattyn E, Villeirs G. Dynamic contrast-enhanced imaging has limited added value over T2-weighted imaging and diffusion-weighted imaging when using PI-RADSv2 for diagnosis of clinically significant prostate cancer in patients with elevated PSA. Clin Radiol. 2017;72(1):23–32.
</cite> [<a href="https://doi.org/10.1016/j.crad.2016.09.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27726850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Visschere%20P,%20Lumen%20N,%20Ost%20P,%20Decaestecker%20K,%20Pattyn%20E,%20Villeirs%20G.%20Dynamic%20contrast-enhanced%20imaging%20has%20limited%20added%20value%20over%20T2-weighted%20imaging%20and%20diffusion-weighted%20imaging%20when%20using%20PI-RADSv2%20for%20diagnosis%20of%20clinically%20significant%20prostate%20cancer%20in%20patients%20with%20elevated%20PSA.%20Clin%20Radiol.%202017;72(1):23%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Asbach P, Padhani AR. Are upgraded DCE-positive PI-RADS 3 lesions truly suspicious for clinically significant prostate cancer? Eur Radiol. 2023;33(8):5825–7.
</cite> [<a href="https://doi.org/10.1007/s00330-023-09711-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37154954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Asbach%20P,%20Padhani%20AR.%20Are%20upgraded%20DCE-positive%20PI-RADS%203%20lesions%20truly%20suspicious%20for%20clinically%20significant%20prostate%20cancer?%20Eur%20Radiol.%202023;33(8):5825%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Sigle A, Borkowetz A, von Hardenberg J, Drerup M, Kornienko K, Kwe J, et al. Prediction of significant prostate cancer in equivocal magnetic resonance imaging lesions: A High-volume international multicenter study. Eur Urol Focus. 2023;9(4):606–13.
</cite> [<a href="https://doi.org/10.1016/j.euf.2023.01.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36804191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sigle%20A,%20Borkowetz%20A,%20von%20Hardenberg%20J,%20Drerup%20M,%20Kornienko%20K,%20Kwe%20J,%20et%20al.%20Prediction%20of%20significant%20prostate%20cancer%20in%20equivocal%20magnetic%20resonance%20imaging%20lesions:%20A%20High-volume%20international%20multicenter%20study.%20Eur%20Urol%20Focus.%202023;9(4):606%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Görtz M, Radtke JP, Hatiboglu G, Schütz V, Tosev G, Güttlein M, et al. The value of prostate-specific antigen density for prostate Imaging-Reporting and data system 3 lesions on multiparametric magnetic resonance imaging: a strategy to avoid unnecessary prostate biopsies. Eur Urol Focus. 2021;7(2):325–31.
</cite> [<a href="https://doi.org/10.1016/j.euf.2019.11.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31839564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?G%C3%B6rtz%20M,%20Radtke%20JP,%20Hatiboglu%20G,%20Sch%C3%BCtz%20V,%20Tosev%20G,%20G%C3%BCttlein%20M,%20et%20al.%20The%20value%20of%20prostate-specific%20antigen%20density%20for%20prostate%20Imaging-Reporting%20and%20data%20system%203%20lesions%20on%20multiparametric%20magnetic%20resonance%20imaging:%20a%20strategy%20to%20avoid%20unnecessary%20prostate%20biopsies.%20Eur%20Urol%20Focus.%202021;7(2):325%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Aldoj N, Lukas S, Dewey M, Penzkofer T. Semi-automatic classification of prostate cancer on multi-parametric MR imaging using a multi-channel 3D convolutional neural network. Eur Radiol. 2020;30(2):1243–53.
</cite> [<a href="https://doi.org/10.1007/s00330-019-06417-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31468158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aldoj%20N,%20Lukas%20S,%20Dewey%20M,%20Penzkofer%20T.%20Semi-automatic%20classification%20of%20prostate%20cancer%20on%20multi-parametric%20MR%20imaging%20using%20a%20multi-channel%203D%20convolutional%20neural%20network.%20Eur%20Radiol.%202020;30(2):1243%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Matsuoka Y, Ueno Y, Uehara S, Tanaka H, Kobayashi M, Tanaka H, et al. Deep-learning prostate cancer detection and segmentation on biparametric versus multiparametric magnetic resonance imaging: added value of dynamic contrast-enhanced imaging. Int J Urol. 2023;30(12):1103–11.
</cite> [<a href="https://doi.org/10.1111/iju.15280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37605627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matsuoka%20Y,%20Ueno%20Y,%20Uehara%20S,%20Tanaka%20H,%20Kobayashi%20M,%20Tanaka%20H,%20et%20al.%20Deep-learning%20prostate%20cancer%20detection%20and%20segmentation%20on%20biparametric%20versus%20multiparametric%20magnetic%20resonance%20imaging:%20added%20value%20of%20dynamic%20contrast-enhanced%20imaging.%20Int%20J%20Urol.%202023;30(12):1103%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366217/bin/40644_2025_916_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (370.6KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data used in this manuscript are subject to local privacy and data protection laws and not publicly available. Requests for anonymized data review can be directed to the authors and will be reviewed in co-operation with the responsible data protection officers.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Cancer Imaging are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s40644-025-00916-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/40644_2025_Article_916.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366217/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366217/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366217%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366217/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366217/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366217/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830988/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366217/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830988/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366217/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366217/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="U2ggyyUdsDahRO9qbjyXjNaIWwz6xPT0979FfkhCCl4dtJZKBZveOMpVhA15TEKN">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
