
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Mechanism and cellular actions of the potent AMPK inhibitor BAY-3827 - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532FF38AF23383052FF30005EDB6A7.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sciadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372887/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Science Advances">
<meta name="citation_title" content="Mechanism and cellular actions of the potent AMPK inhibitor BAY-3827">
<meta name="citation_author" content="Conchita Fraguas Bringas">
<meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen 2200, Denmark.">
<meta name="citation_author" content="Mohd Syed Ahangar">
<meta name="citation_author_institution" content="Astbury Centre for Structural Molecular Biology, School of Molecular and Cellular Biology, Faculty of Biological Sciences, University of Leeds, Leeds LS2 9JT, UK.">
<meta name="citation_author" content="Joyceline Cuenco">
<meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen 2200, Denmark.">
<meta name="citation_author" content="Hongling Liu">
<meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen 2200, Denmark.">
<meta name="citation_author" content="Alex B Addinsall">
<meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen 2200, Denmark.">
<meta name="citation_author" content="Maria Lindahl">
<meta name="citation_author_institution" content="Department of Experimental Medical Science, Lund University, Lund 22184, Sweden.">
<meta name="citation_author" content="Ashley J Ovens">
<meta name="citation_author_institution" content="St.Vincent&#x27;s Institute of Medical Research, Fitzroy, Victoria 3065, Australia.">
<meta name="citation_author" content="Mark A Febbraio">
<meta name="citation_author_institution" content="Drug Discovery Biology, Monash Institute of Pharmaceutical Sciences, Parkville, Victoria 3052, Australia.">
<meta name="citation_author" content="Marc Foretz">
<meta name="citation_author_institution" content="Université Paris Cité, CNRS, Inserm, Institut Cochin, Paris 75014, France.">
<meta name="citation_author" content="Olga Göransson">
<meta name="citation_author_institution" content="Department of Experimental Medical Science, Lund University, Lund 22184, Sweden.">
<meta name="citation_author" content="John W Scott">
<meta name="citation_author_institution" content="St.Vincent&#x27;s Institute of Medical Research, Fitzroy, Victoria 3065, Australia.">
<meta name="citation_author_institution" content="Drug Discovery Biology, Monash Institute of Pharmaceutical Sciences, Parkville, Victoria 3052, Australia.">
<meta name="citation_author" content="Elton Zeqiraj">
<meta name="citation_author_institution" content="Astbury Centre for Structural Molecular Biology, School of Molecular and Cellular Biology, Faculty of Biological Sciences, University of Leeds, Leeds LS2 9JT, UK.">
<meta name="citation_author" content="Kei Sakamoto">
<meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen 2200, Denmark.">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="11">
<meta name="citation_issue" content="34">
<meta name="citation_firstpage" content="eadx2434">
<meta name="citation_doi" content="10.1126/sciadv.adx2434">
<meta name="citation_pmid" content="40845097">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372887/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372887/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372887/pdf/sciadv.adx2434.pdf">
<meta name="description" content="Inhibition of adenosine 5′-monophosphate (AMP)–activated protein kinase (AMPK) is under increasing investigation for its therapeutic potential in many diseases. Existing AMPK inhibitors are however limited, with poor selectivity and substantial ...">
<meta name="og:title" content="Mechanism and cellular actions of the potent AMPK inhibitor BAY-3827">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Inhibition of adenosine 5′-monophosphate (AMP)–activated protein kinase (AMPK) is under increasing investigation for its therapeutic potential in many diseases. Existing AMPK inhibitors are however limited, with poor selectivity and substantial ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372887/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12372887">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1126/sciadv.adx2434"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/sciadv.adx2434.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372887%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12372887/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12372887/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372887/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sciadv.gif" alt="Science Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Science Advances" title="Link to Science Advances" shape="default" href="http://advances.sciencemag.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Adv</button></div>. 2025 Aug 22;11(34):eadx2434. doi: <a href="https://doi.org/10.1126/sciadv.adx2434" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1126/sciadv.adx2434</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Mechanism and cellular actions of the potent AMPK inhibitor BAY-3827</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bringas%20CF%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Conchita Fraguas Bringas</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Conchita Fraguas Bringas</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen 2200, Denmark.</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Project administration, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bringas%20CF%22%5BAuthor%5D" class="usa-link"><span class="name western">Conchita Fraguas Bringas</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahangar%20MS%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mohd Syed Ahangar</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mohd Syed Ahangar</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Astbury Centre for Structural Molecular Biology, School of Molecular and Cellular Biology, Faculty of Biological Sciences, University of Leeds, Leeds LS2 9JT, UK.</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Resources, Validation, Visualization, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahangar%20MS%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohd Syed Ahangar</span></a>
</div>
</div>
<sup>2,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cuenco%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Joyceline Cuenco</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Joyceline Cuenco</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen 2200, Denmark.</div>
<div>Formal analysis, Investigation, Validation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cuenco%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Joyceline Cuenco</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Hongling Liu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Hongling Liu</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen 2200, Denmark.</div>
<div>Investigation, Validation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hongling Liu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Addinsall%20AB%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Alex B Addinsall</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Alex B Addinsall</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen 2200, Denmark.</div>
<div>Formal analysis, Investigation, Validation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Addinsall%20AB%22%5BAuthor%5D" class="usa-link"><span class="name western">Alex B Addinsall</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lindahl%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Maria Lindahl</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Maria Lindahl</span></h3>
<div class="p">
<sup><sup>3</sup></sup>Department of Experimental Medical Science, Lund University, Lund 22184, Sweden.</div>
<div>Formal analysis, Investigation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lindahl%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maria Lindahl</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ovens%20AJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Ashley J Ovens</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ashley J Ovens</span></h3>
<div class="p">
<sup><sup>4</sup></sup>St.Vincent's Institute of Medical Research, Fitzroy, Victoria 3065, Australia.</div>
<div>Conceptualization, Investigation, Methodology, Resources, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ovens%20AJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Ashley J Ovens</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Febbraio%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Mark A Febbraio</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Mark A Febbraio</span></h3>
<div class="p">
<sup><sup>5</sup></sup>Drug Discovery Biology, Monash Institute of Pharmaceutical Sciences, Parkville, Victoria 3052, Australia.</div>
<div>Funding acquisition, Resources</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Febbraio%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Mark A Febbraio</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foretz%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Marc Foretz</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Marc Foretz</span></h3>
<div class="p">
<sup><sup>6</sup></sup>Université Paris Cité, CNRS, Inserm, Institut Cochin, Paris 75014, France.</div>
<div>Resources, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Foretz%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marc Foretz</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%B6ransson%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Olga Göransson</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Olga Göransson</span></h3>
<div class="p">
<sup><sup>3</sup></sup>Department of Experimental Medical Science, Lund University, Lund 22184, Sweden.</div>
<div>Formal analysis, Methodology, Resources, Supervision, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22G%C3%B6ransson%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Olga Göransson</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scott%20JW%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">John W Scott</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">John W Scott</span></h3>
<div class="p">
<sup><sup>4</sup></sup>St.Vincent's Institute of Medical Research, Fitzroy, Victoria 3065, Australia.</div>
<div class="p">
<sup><sup>5</sup></sup>Drug Discovery Biology, Monash Institute of Pharmaceutical Sciences, Parkville, Victoria 3052, Australia.</div>
<div>Formal analysis, Funding acquisition, Investigation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scott%20JW%22%5BAuthor%5D" class="usa-link"><span class="name western">John W Scott</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeqiraj%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Elton Zeqiraj</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Elton Zeqiraj</span></h3>
<div class="p">
<sup><sup>2</sup></sup>Astbury Centre for Structural Molecular Biology, School of Molecular and Cellular Biology, Faculty of Biological Sciences, University of Leeds, Leeds LS2 9JT, UK.</div>
<div>Conceptualization, Data curation, Funding acquisition, Methodology, Project administration, Resources, Supervision, Validation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeqiraj%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elton Zeqiraj</span></a>
</div>
</div>
<sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sakamoto%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Kei Sakamoto</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Kei Sakamoto</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen 2200, Denmark.</div>
<div>Conceptualization, Formal analysis, Funding acquisition, Methodology, Project administration, Resources, Supervision, Validation, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sakamoto%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kei Sakamoto</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>Novo Nordisk Foundation Center for Basic Metabolic Research, Faculty of Health and Medical Sciences, University of Copenhagen, Copenhagen 2200, Denmark.</div>
<div id="aff2">
<sup><sup>2</sup></sup>Astbury Centre for Structural Molecular Biology, School of Molecular and Cellular Biology, Faculty of Biological Sciences, University of Leeds, Leeds LS2 9JT, UK.</div>
<div id="aff3">
<sup><sup>3</sup></sup>Department of Experimental Medical Science, Lund University, Lund 22184, Sweden.</div>
<div id="aff4">
<sup><sup>4</sup></sup>St.Vincent's Institute of Medical Research, Fitzroy, Victoria 3065, Australia.</div>
<div id="aff5">
<sup><sup>5</sup></sup>Drug Discovery Biology, Monash Institute of Pharmaceutical Sciences, Parkville, Victoria 3052, Australia.</div>
<div id="aff6">
<sup><sup>6</sup></sup>Université Paris Cité, CNRS, Inserm, Institut Cochin, Paris 75014, France.</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">Corresponding author. Email: <span>kei.sakamoto@sund.ku.dk</span> (K.S.); <span>e.zeqiraj@leeds.ac.uk</span> (E.Z.)</p>
</div>
<div class="fn" id="afn1">
<sup>†</sup><p class="display-inline">These authors contributed equally to this work.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Conchita Fraguas Bringas</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Project administration, Software, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mohd Syed Ahangar</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Resources, Validation, Visualization, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Joyceline Cuenco</span></strong>: <span class="role">Formal analysis, Investigation, Validation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Hongling Liu</span></strong>: <span class="role">Investigation, Validation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Alex B Addinsall</span></strong>: <span class="role">Formal analysis, Investigation, Validation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Maria Lindahl</span></strong>: <span class="role">Formal analysis, Investigation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ashley J Ovens</span></strong>: <span class="role">Conceptualization, Investigation, Methodology, Resources, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mark A Febbraio</span></strong>: <span class="role">Funding acquisition, Resources</span>
</div>
<div>
<strong class="contrib"><span class="name western">Marc Foretz</span></strong>: <span class="role">Resources, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Olga Göransson</span></strong>: <span class="role">Formal analysis, Methodology, Resources, Supervision, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">John W Scott</span></strong>: <span class="role">Formal analysis, Funding acquisition, Investigation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Elton Zeqiraj</span></strong>: <span class="role">Conceptualization, Data curation, Funding acquisition, Methodology, Project administration, Resources, Supervision, Validation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Kei Sakamoto</span></strong>: <span class="role">Conceptualization, Formal analysis, Funding acquisition, Methodology, Project administration, Resources, Supervision, Validation, Writing - original draft, Writing - review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 14; Accepted 2025 Jul 23; Collection date 2025 Aug 22.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. Distributed under a Creative Commons Attribution License 4.0 (CC BY).</div>
<p>This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution license</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12372887  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40845097/" class="usa-link">40845097</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Inhibition of adenosine 5′-monophosphate (AMP)–activated protein kinase (AMPK) is under increasing investigation for its therapeutic potential in many diseases. Existing AMPK inhibitors are however limited, with poor selectivity and substantial off-target effects. Here, we provide mechanistic insights and describe the cellular selectivity of the recently identified AMPK inhibitor BAY-3827. A 2.5-Å cocrystal structure of the AMPK kinase domain with BAY-3827 revealed distinct features including a disulfide bridge between the αD helix Cys<sup>106</sup> and the activation loop residue Cys<sup>174</sup>. This bridge appears to stabilize the activation loop such that Asn<sup>162</sup> repositions the Asp-Phe-Gly (DFG) motif Phe<sup>158</sup> toward the C-terminal lobe, displacing His<sup>137</sup> and disrupting the regulatory spine, promoting an inactive kinase state. In hepatocytes, BAY-3827 blocked AMPK activator (MK-8722)–mediated phosphorylation of ACC1 and corresponding inhibition of lipogenesis. Transcriptome analysis revealed that BAY-3827 down-regulated ~30% of MK-8722–stimulated AMPK-dependent genes. We establish the molecular and cellular basis of BAY-3827’s selectivity and utility for delineating AMPK functions while highlighting its limitations.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>The mechanism and unique molecular/cellular features of the potent and selective AMPK inhibitor BAY-3827 are unraveled.</p></section><section id="sec1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>Adenosine 5′-monophosphate (AMP)–activated protein kinase (AMPK) governs cellular energy metabolism and coordinates a myriad of cellular functions to maintain organismal homeostasis (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>–<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>). AMPK is ubiquitously expressed in eukaryotic cells as heterotrimeric complexes comprising a catalytic α and regulatory β and γ subunits. In mammals, the three subunits are encoded by multiple genes, giving rise to seven subunit isoforms (α1, α2; β1, β2; and γ1, γ2, and γ3) that can generate up to 12 distinct heterotrimeric complexes (<a href="#R4" class="usa-link" aria-describedby="R4"><em>4</em></a>, <a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>). AMPK is activated through phosphorylation of Thr<sup>172</sup> within the activation loop of the α subunit kinase domain (KD). The upstream kinases phosphorylating Thr<sup>172</sup> are liver kinase B1 (LKB1) and Ca<sup>2+</sup>-calmodulin–dependent protein kinase kinase-2 (CaMKK2) (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>, <a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>). Thr<sup>172</sup> is located in the activation loop of the catalytic α subunit, and the negative charge added upon residue phosphorylation leads to a conformational change that prepares AMPK for substrate phosphorylation, acquiring an active kinase state. The Thr<sup>172</sup>-containing activation loop is stabilized via charge interactions with positively charged αC helix Lys<sup>60</sup> and Arg<sup>138</sup> and with Asn<sup>162</sup>, swinging outward to direct the γ-phosphate of adenosine 5′-triphosphate (ATP) toward the peptide substrate. The conserved Asp-Phe-Gly (DFG) kinase motif will point away from the ATP-binding pocket, interacting with the conserved Lys<sup>45</sup> residue and with ATP-Mg<sup>2+</sup> interacting with Asp<sup>157</sup> (<a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>, <a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>), allowing for phosphoryl transfer to take place. The γ subunits contain four tandem cystathionine β-synthase (CBS) repeats, creating three functional nucleotide-binding sites that provide AMPK with its energy-sensing capabilities (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>). The binding of adenosine 5′-diphosphate (ADP) and/or AMP to CBS motifs causes conformational changes that increase net Thr<sup>172</sup> phosphorylation. In addition, the binding of AMP further increases AMPK activity through allosteric stimulation. Although ADP has been shown to act as a modest allosteric activator of AMPK under certain conditions, AMP is still considered the more physiologically relevant activator in vivo (<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>). AMPK activity is therefore controlled by multiple regulatory mechanisms, often involving a combination of posttranslational modifications and allosteric regulation.</p>
<p>AMPK has long held promise as a therapeutic target for metabolic syndrome, as its physiological and pharmacological activation in multiple metabolic tissues has resulted in the amelioration of insulin resistance and reversal of hyperglycemia in preclinical studies (<a href="#R9" class="usa-link" aria-describedby="R9"><em>9</em></a>). For example, active AMPK inhibits fatty acid and cholesterol biosynthesis in the liver through phosphorylation and inactivation of acetyl–coenzyme A (CoA) carboxylase-1 (ACC1) and 3-hydroxy-3-methylglutaryl (HMG)–CoA reductase, respectively (<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>, <a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>). The activation of AMPK leads to increased fatty acid oxidation through phosphorylation of ACC2 and also promotion of plasma membrane localization of GLUT4 partly via TBC1D1 phosphorylation in an insulin-independent manner, promoting glucose uptake in skeletal muscle (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>, <a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>–<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>). These beneficial effects of AMPK activation prompted the pharmaceutical development of small-molecule activators of AMPK, resulting in the identification of several potent and specific allosteric activators, including A-769662 (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>, <a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>), 991 (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>), MK-8722 (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>, <a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>), and PF-739 (<a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>). These compounds bind into a pocket termed the allosteric drug and metabolite (ADaM) site (<a href="#R7" class="usa-link" aria-describedby="R7"><em>7</em></a>) located between the α and β subunits (<a href="#R9" class="usa-link" aria-describedby="R9"><em>9</em></a>, <a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>). To date, some ADaM site binding compounds have demonstrated promising efficacy in reversing hyperglycemia in preclinical proof-of-concept studies (<a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>, <a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>). Although current allosteric pan-AMPK activators have the potential risk of causing cardiac hypertrophy, isoform-selective activators are being developed to mitigate safety issues (<a href="#R9" class="usa-link" aria-describedby="R9"><em>9</em></a>). Moreover, these highly potent and selective allosteric activators serve as valuable research tools and have profoundly contributed to delineating AMPK functions.</p>
<p>In cancer, AMPK has a dual role, as a tumor promoter or suppressor, depending on, for instance, subcellular context, trimeric complex formation, and upstream regulators (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>, <a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>). Previously, AMPK was viewed as a tumor suppressor, based on its ability to inhibit anabolic processes and mediate the tumor suppressive effects of LKB1, such as promoting cell growth through the inactivation of mTORC1 (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>–<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>). Conversely, more recent studies have supported the notion that increased AMPK activity may help to promote tumor growth and survival, alongside conferring drug resistance and resilience under tumor hypoxia and nutrient deprivation (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>, <a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>). In such conditions in established tumors, AMPK inhibitors might be efficacious tools for cancer treatment. Specifically, they may be particularly effective in cases where the <em>PRKAA1</em> or <em>PRKAB2</em> genes are amplified, causing aberrantly high expression of AMPK. (<a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>, <a href="#R26" class="usa-link" aria-describedby="R26"><em>26</em></a>).</p>
<p>In contrast to activators, the availability of selective AMPK inhibitors is limited. The first and most commonly used AMPK inhibitor, the pyrazolopyrimidine derivative compound C, was identified in an in vitro high-throughput screen, and although its selectivity was poorly characterized, it was used to probe for AMPK-dependent metabolic actions of the antidiabetes drug metformin in hepatocytes (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>). Compound C was “rediscovered” in an in vivo compound screen in zebrafish embryos as an inhibitor of the bone morphogenetic protein (BMP) pathway (e.g., BMP type 1 receptors ALK2, 3, and 6) and was renamed “dorsomorphin” as it induced dorsalization in embryos (<a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>). Cocrystal structure studies have established that compound C/dorsomorphin binds to the highly conserved ATP-binding pocket in the KD of AMPKα (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>) and ALK2 (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>). However, multiple in vitro kinase activity screens studying inhibitor selectivity have verified that compound C is not a selective inhibitor of AMPK nor BMP receptor kinases (<a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>–<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>), as, for example, it has been reported to inhibit the activities of 34 of 119 kinases more potently than AMPK at 1 μM (<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>). In line with this, numerous cellular studies to date have shown that compound C affects a diverse range of biological processes via AMPK-independent mechanisms (<a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>).</p>
<p>SBI-0206965, a 2-aminopyrimidine derivative, was originally identified through a cell-based screen and described as a selective inhibitor of the autophagy initiator kinase ULK1, with the ability to inhibit ULK signaling and ULK1-mediated survival of cancer cells (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>). Recent studies have reported that SBI-0206965 inhibits AMPK and ULK1/2 with a similar potency (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>) and in a cell-free assay, SBI-0206965 inhibited AMPK with a median inhibitory concentration (IC<sub>50</sub>) of 0.16 μM at 20 μM ATP, compared to a value of 1.9 μM with compound C (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>). Although SBI-0206965 demonstrated much higher potency and selectivity toward AMPK in cell-free assays than compound C, several other kinases including NUAK1, FAK, MLK1/3, and MARK3/4, all of which contain a methionine in their gatekeeper position in the KD, were also potently inhibited (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). In cellular studies, SBI-0206965 dose-dependently blocked 991-induced phosphorylation of ACC1 and inhibition of lipogenesis in mouse primary hepatocytes. Notably, SBI-0206965 also inhibited glucose and nucleoside transport systems in adipocytes and myotubes (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>).</p>
<p>More recently, a potent and selective AMPK inhibitor termed BAY-3827 was identified in a high- throughput compound screen and underwent chemical optimization. In cell-free assays, BAY-3827 inhibited AMPK with IC<sub>50</sub> values of 1.4 nM at low (10 μM) ATP and 15 nM at high (2 mM) ATP concentrations (with 2 μM AMP under both conditions), although it inhibited 90-kDa ribosomal S6 kinase (RSK) isoforms with a similar potency (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>). In cellular assays, BAY-3827 inhibited phosphorylation of ACC1 and showed antiproliferative effects in a subset of tumor models, namely androgen-dependent prostate cancer and myeloma cell lines (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>). However, it is unknown whether this antiproliferative effect of BAY-3827 is mediated through inhibition of AMPK. To date, there is limited information about the molecular mechanisms of AMPK inhibition by BAY-3827. Recent biochemical analysis has characterized BAY-3827 as a mixed-type inhibitor (<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>); however, the structural mechanisms underlying its inhibitory action and the basis for its kinase specificity remain largely undefined. Here, we describe the molecular basis for BAY-3827’s inhibitory effect on AMPK through the cocrystallization of the AMPKα2 KD (α2KD) with BAY-3827. In addition, we provide cellular potency and selectivity data of BAY-3827 with a specific emphasis on AMPK-dependent transcriptional and metabolic processes (fatty acid synthesis) in AMPKα1α2<sup>−/−</sup> double knockout (DKO) and AMPKα1α2<sup>+/+</sup> (control) primary mouse hepatocytes.</p></section><section id="sec2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="sec3"><h3 class="pmc_sec_title">Kinase inhibition selectivity profile of BAY-3827 and its structurally related BAY-974</h3>
<p>We determined the selectivity of BAY-3827 alongside its inactive control compound BAY-974, which shares a similar chemical scaffold with BAY-3827, by performing an in vitro kinase activity assay across a panel of 140 human protein kinases (<a href="#F1" class="usa-link">Fig. 1, A and B</a>). Both compounds were tested at 0.1 μM in the assay and consistent with previous data (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>), AMPK was the most inhibited (&gt;90%) by BAY-3827, followed by PHK and RSK1 (~80 to 85%). Aurora B, MINK1, TBK1, and IRAK1 were also inhibited (~60%) by BAY-3827 (<a href="#F1" class="usa-link">Fig. 1A</a> and table S1). The inhibitory profile of BAY-3827 in the tested panel displayed a preference for the human kinome CAMK group (which includes AMPK and PHK) and the AGC group containing RSK family and MSK1. Furthermore, Aurora kinase family Aurora B and STE group kinases MINK1 and MST3 were also highly inhibited alongside IRAK1, which belongs to the TKL group (<a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>). As anticipated, BAY-974 had almost no effect on AMPK; however, it inhibited (~55 to 90%) other kinases mainly from the TK and TKL groups, including SYK, YES1, IRAK1, and ULK1 from the ULK family (<a href="#F1" class="usa-link">Fig. 1B</a> and table S1). These results demonstrate that BAY-3827, but not its structurally related BAY-974, selectively inhibits AMPK. This can be attributed to specific chemical alterations; BAY-974 has a missing fluoride group in the indazole ring compared to BAY-3827, a methyl group in the benzamide ring instead of BAY-3827’s ethyl group and finally a dimethyl dihydropyridine ring compared to a trimethyl H-pyridine ring present in BAY-3827 (<a href="#F1" class="usa-link">Fig. 1C</a>). The inhibitory profiles between BAY-3827 and BAY-974 across the panel of kinases with ≤50% remaining activity revealed a minimal overlap (<a href="#F1" class="usa-link">Fig. 1D</a>), making BAY-974 a suitable control compound to help dissect BAY-3827-specific effects.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Fig. 1. BAY-3827 but not BAY-974 selectively inhibits AMPK.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372887_sciadv.adx2434-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d1e6/12372887/2d397d404f14/sciadv.adx2434-f1.jpg" loading="lazy" height="1017" width="718" alt="Fig. 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Kinase activity assays were conducted across a panel of 140 human kinases, and compound selectivity of (<strong>A</strong>) BAY-3827 (0.1 μM) and (<strong>B</strong>) BAY-974 (0.1 μM) was assessed, reporting remaining kinase activity (%) from <em>n</em> = 2 in means ± SEM. (<strong>C</strong>) Chemical structures of BAY-3827 (N-[5-(3,5-dicyano-1,2,6-trimethyl-4H-pyridin-4-yl)-6-fluoro-7-methyl-1H-indazol-3-yl]-2-ethylbenzamide) and BAY-974 (N-[5-(3,5-dicyano-2,6-dimethyl-1,4-dihydropyridin-4-yl)-1-methylindazol-3-yl]-2-methylbenzamide) drawn in Marvin (24.3.1), 2024, ChemAxon (<a href="http://chemaxon.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://chemaxon.com</a>). (<strong>D</strong>) Schematic showing the inhibitory overlap between BAY-3827 and BAY-974 of kinases with ≤50% remaining activity as well as the corresponding kinase groups they belong to. (<strong>E</strong>) In silico residue conservation analysis based on inhibited (≤50% remaining activity) and noninhibited (≥99% remaining activity) kinases by BAY-3827 at 1 μM across a 140-human kinase panel cross-referenced with available data (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>). (<strong>F</strong>) Ribbon representation of α2KD [Protein Data Bank (PDB): 7MYJ, chain A] showing computed ConSurf conservation scale (1 to 9) colors (<a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>, <a href="#R41" class="usa-link" aria-describedby="R41"><em>41</em></a>) between BAY-3827–inhibited kinase group (left) and noninhibited kinase group (right) displaying residues with a three-point or higher scale difference between groups.</p></figcaption></figure><p>Achieving a specific kinase inhibitor often involves targeting residues that are unique to the kinase of interest or poorly conserved across the kinome. On the basis of the selectivity profile of BAY-3827, we attempted to identify the pattern of this compound’s preference by studying residue conservation in the KD of highly inhibited kinases by BAY-3827 compared to those that were not inhibited. To do this comprehensively, we screened a higher concentration of BAY-3827 (1 μM) across the same kinase panel, and these data were cross-referenced with available kinase screen data at the same concentration (<a href="#F1" class="usa-link">Fig. 1E</a>) (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>). We defined a BAY-3827–inhibited kinase group containing kinases that had 50% or less activity remaining after BAY-3827 treatment, and a noninhibited kinase group representing those with 99% or more remaining activity after BAY-3827 treatment. AMPK, BRSK2, MELK, NUAK1, MSK1, RSK1, and RSK2 composed the BAY-3827–inhibited group (<a href="#F1" class="usa-link">Fig. 1E</a>, shown in blue) and kinases ABL1, PIM3, INSR, IRAK4, WNK1, PIM2, IKKβ, CDK2, TTBK1, and TAK1 were selected as representatives of the noninhibited group (<a href="#F1" class="usa-link">Fig. 1E</a>, shown in green). A sequence alignment of human KDs was conducted in each group, and their conservation was studied using AMPKα1 as a reference sequence, with the inhibited group displaying 30% or higher percentage sequence identity up to 47.8% in BRSK2, whereas the noninhibited group ranged from 29.7 to 24.1% identity to AMPK (table S2). On the basis of residue conservation, a score was computed using the ConSurf server (<a href="#F1" class="usa-link">Fig. 1, E and F</a>, and table S2) (<a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>, <a href="#R41" class="usa-link" aria-describedby="R41"><em>41</em></a>), revealing notably different residue conservation patterns between the two kinase groups (fig. S1, A and B). We therefore hypothesized that highly conserved residues in the BAY-3827–inhibited, but not in the noninhibited kinase group, would be important for the inhibitory mechanism of BAY-3827. Residues with at least a three-point difference in the conservation scale (1 to 9) between kinase groups were displayed in the structure of AMPKα2. These were localized in four clusters: (i) in the N-terminal lobe such as Tyr<sup>16</sup> and Arg<sup>49</sup>; (ii) near the ATP-binding pocket, including residues such as Gly<sup>99</sup>, Leu<sup>146</sup>, and Asp<sup>148</sup>; (iii) in the C-terminal lobe helices such as His<sup>265</sup> and Val<sup>251</sup>; and (iv) in the activation loop, where Thr<sup>172</sup> had a four-point scale decrease in conservation in the noninhibited group, and residues Cys<sup>174</sup> and Leu<sup>170</sup> had a notable six-point decrease in conservation compared to the BAY-3827–inhibited kinase group (<a href="#F1" class="usa-link">Fig. 1E</a>, fig. S1C, and table S2). Collectively, these analyses suggest that there is a conservation basis for the mechanistic inhibitory action of BAY-3827. Residues surrounding the ATP-binding pocket and the activation loop were identified as key contributors to BAY-3827’s inhibitory action, showing high conservation among BAY-3827–sensitive kinases but with low or variable conservation in BAY-3827–insensitive kinases.</p></section><section id="sec4"><h3 class="pmc_sec_title">AMPKα2 binds BAY-3827 in a DFG-in inactive kinase conformation</h3>
<p>To understand the precise molecular mechanism behind the inhibition of AMPK by BAY-3827, the cocrystal structure of BAY-3827 with wild-type (WT) α2KD was obtained at a resolution of 2.5 Å (<a href="#T1" class="usa-link">Table 1</a> and <a href="#F2" class="usa-link">Fig. 2A</a>). BAY-3827 binds into the conserved ATP-binding pocket between the N- and C-terminal lobes, as revealed by an unambiguous compound density map (<a href="#F2" class="usa-link">Fig. 2B</a>), binding with a dissociation constant (<em>K</em><sub>d</sub>) value of 1.23 μM as measured by a spectral shift binding assay (fig. S2A). In this structure, the activation loop (highlighted in magenta) instead of swinging out in an active conformation (fig. S2B) (<a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>), it hovers over the kinase core below the β1 sheet and interacts with the αD helix in the C-lobe through Cys<sup>174</sup>, constituting an inactive kinase conformation (<a href="#F2" class="usa-link">Fig. 2, C and D</a>). The indazole ring in BAY-3827 hydrogen bonds with hinge region residues, with a single bond formed with Glu<sup>94</sup> and two with Val<sup>96</sup> alongside two additional bonds made with N-lobe residues Leu<sup>22</sup> and Lys<sup>45</sup> (<a href="#F2" class="usa-link">Fig. 2C</a> and fig. S2C). Leu<sup>22</sup> bonding to BAY-3827’s benzamide moiety is water-mediated and the conserved Lys<sup>45</sup> in the β3 sheet interacts with the pyridine ring of BAY-3827 (<a href="#F2" class="usa-link">Fig. 2C</a>). Since BAY-3827 interacts directly with Lys<sup>45</sup>, the typical salt bridge with Glu<sup>64</sup> in the αC helix is no longer possible, having a Lys-Glu distance of 11.7 Å (<a href="#T2" class="usa-link">Table 2</a>), and an αC helix out conformation (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>). Residues that accommodate and engage in hydrophobic contacts with BAY-3827 include hinge region residues Tyr<sup>95</sup> and Ser<sup>97</sup>, as well as N-lobe residues Gly<sup>99</sup>, Val<sup>30</sup>, and Asp<sup>166</sup>. In the C-lobe, residues Glu<sup>100</sup>, Gly<sup>23</sup>, Ile<sup>77</sup>, and Ala<sup>43</sup> and the gatekeeper Met<sup>93</sup> also contribute to ligand hydrophobic contacts, as well as Leu<sup>146</sup>, which faces the benzamine moiety of BAY-3827. Moreover, activation loop residues Asn<sup>144</sup> and Ala<sup>156</sup>, alongside Asn<sup>162</sup>, Met<sup>164</sup>, and Ser<sup>165</sup> face toward the pyridine ring of BAY-3827 and engage in hydrophobic contacts (fig. S2C), with BAY-3827 occupying a space in the ATP-binding pocket that does not extend further past gatekeeper Met<sup>93</sup>. Focusing on the DFG motif, Asp<sup>157</sup> presents with an out conformation and the DFG motif Phe<sup>158</sup> residue has shifted down toward the C-lobe compared to the kinase active state in AMPK (<a href="#F2" class="usa-link">Fig. 2C</a> and fig. S2B) (<a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>). In the C-lobe, a key interaction occurs between αD helix Cys<sup>106</sup> and activation loop residue Cys<sup>174</sup>, which forms a disulfide bridge (<a href="#F2" class="usa-link">Fig. 2D</a>). To confirm the relevance of this interaction to BAY-3827’s mechanism, we performed binding analyses in the presence or absence of dithiothreitol (DTT) alongside the phosphomimetic AMPKα2 (T172D)–α2KD mutant protein to evaluate the impact of this mutation on BAY-3827 binding affinity (fig. S2, D to G). The T172D mutation modestly reduced binding affinity compared to WT AMPKα2-KD (fig. S2F). Notably, treatment with 3 mM DTT markedly diminished BAY-3827 binding in the WT protein but had no substantial effect on the T172D- α2KD mutant (fig. S2, E to G), highlighting the role of disulfide bridge formation as well as Thr<sup>172</sup> residue in stabilizing BAY-3827 binding. Superimposition of α2KD–BAY-3827 with available compound C and SBI-0206965 structures (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>, <a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>), showed a partial overlap with compound C’s binding site in the ATP-binding pocket. However, the binding space of BAY-3827 and SBI-0206965 on AMPK was shared, whereas compound C extended outside of this pocket (<a href="#F2" class="usa-link">Fig. 2E</a> and fig. S2J).</p>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">Table 1. Crystallography data collection and refinement statistics.</h4>
<div class="caption p"><p>Crystallography data collection for α2KD with BAY-3827. Statistics for the highest-resolution shell are shown in parentheses.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th scope="col" rowspan="1" colspan="1"></th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">α2 KD-BAY-3827</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="top" align="left" rowspan="1" colspan="1">Wavelength</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.6199 Å</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Resolution range</td>
<td valign="top" align="center" rowspan="1" colspan="1">58.89 to 2.5 (2.56 to 2.50)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Space group</td>
<td valign="top" align="center" rowspan="1" colspan="1">P 21 21 2</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Unit cell</td>
<td valign="top" align="center" rowspan="1" colspan="1">59.176 117.778 38.327 90 90 90</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Total reflections</td>
<td valign="top" align="center" rowspan="1" colspan="1">65,446 (6013)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Unique reflections</td>
<td valign="top" align="center" rowspan="1" colspan="1">10,021 (890)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Multiplicity</td>
<td valign="top" align="center" rowspan="1" colspan="1">6.5 (6.8)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Completeness (%)</td>
<td valign="top" align="center" rowspan="1" colspan="1">99.9 (100.00)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Mean I/sigma(I)</td>
<td valign="top" align="center" rowspan="1" colspan="1">9.07 (0.81)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Wilson B-factor</td>
<td valign="top" align="center" rowspan="1" colspan="1">39.04</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">R-merge</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.074 (0.418)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">R-meas</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.081 (0.454)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">R-pim</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.0332 (0.174)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">CC1/2</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.997 (0.944)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">CC*</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.999 (0.973)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Reflections used in refinement</td>
<td valign="top" align="center" rowspan="1" colspan="1">9799 (2397)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Reflections used for R-free</td>
<td valign="top" align="center" rowspan="1" colspan="1">497 (119)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">R-work</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.1975 (2433)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">R-free</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.2350 (0.3053)</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Number of nonhydrogen atoms</td>
<td valign="top" align="center" rowspan="1" colspan="1">2256</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Macromolecules</td>
<td valign="top" align="center" rowspan="1" colspan="1">2142</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Ligands</td>
<td valign="top" align="center" rowspan="1" colspan="1">43</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Solvent</td>
<td valign="top" align="center" rowspan="1" colspan="1">71</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Protein residues</td>
<td valign="top" align="center" rowspan="1" colspan="1">267</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">RMS (bonds)</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.002</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">RMS (angles)</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.52</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Ramachandran favored (%)</td>
<td valign="top" align="center" rowspan="1" colspan="1">95.44</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Ramachandran allowed (%)</td>
<td valign="top" align="center" rowspan="1" colspan="1">3.04</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Ramachandran outliers (%)</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.52</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Rotamer outliers (%)</td>
<td valign="top" align="center" rowspan="1" colspan="1">6.06</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Clash score</td>
<td valign="top" align="center" rowspan="1" colspan="1">5.52</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Average B-factor</td>
<td valign="top" align="center" rowspan="1" colspan="1">53.50</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Macromolecules</td>
<td valign="top" align="center" rowspan="1" colspan="1">53.90</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Ligands</td>
<td valign="top" align="center" rowspan="1" colspan="1">48.62</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Solvent</td>
<td valign="top" align="center" rowspan="1" colspan="1">44.43</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2. Structure of α2KD bound to BAY-3827.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372887_sciadv.adx2434-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d1e6/12372887/dc3cb420e598/sciadv.adx2434-f2.jpg" loading="lazy" height="1146" width="794" alt="Fig. 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Cocrystal structure of α2KD (shown as ribbons) with BAY-3827 (shown as yellow sticks). The activation loop (AL) segment is colored magenta. (<strong>B</strong>) Stereo-image of the 2Fo-Fc omit map of BAY-3827 shown at 1σ. (<strong>C</strong>) Magnified view of the BAY-3827 binding site with interactions indicated as gray dash lines. The side chains of Asp<sup>157</sup> and Phe<sup>158</sup> of the DFG motif are shown as sticks with Phe<sup>158</sup> in an “FG down” conformation. A water molecule (W) is shown as a red sphere. A disulfide bond (Cys<sup>106</sup>-Cys<sup>174</sup>) is shown as sticks. (<strong>D</strong>) Stereo image of the Fo-Fc polder map (Phenix) of the indicated disulfide bond between Cys<sup>106</sup> from αD and Cys<sup>174</sup> from activation loop (AL). (<strong>E</strong>) Surface view of α2KD bound to BAY-3827 with superimposed compound C (light cyan, PDB:3AQV) and SBI-0206965 green, PDB:6BX6) inhibitors shown with sticks (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>, <a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>, <a href="#R80" class="usa-link" aria-describedby="R80"><em>80</em></a>). (<strong>F</strong>) Zoomed-in view of the DFG motif superimposed with compound C (light cyan, PDB: 3AQV) and SBI-0206965 (green, PDB:6BX6), displaying the backbone (as ribbons) of α2KD bound to BAY-3827 showing key regulatory spine residues.</p></figcaption></figure><section class="tw xbox font-sm" id="T2"><h4 class="obj_head">Table 2. Atomic distance measurements to define kinase conformation.</h4>
<div class="caption p"><p>Distance measurements performed as previously described (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>) in PyMOL (<a href="#R80" class="usa-link" aria-describedby="R80"><em>80</em></a>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="left" scope="col" rowspan="1" colspan="1">Structures</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">D1: Leu<sup>68</sup>.CA-Phe<sup>158</sup>.A CZ</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">D2: Lys<sup>45</sup>.CA-Phe<sup>158</sup>.CZ</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Lys<sup>45</sup>.CB-Glu<sup>64</sup>.CB</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="top" align="left" rowspan="1" colspan="1">α2KD–BAY-3827</td>
<td valign="top" align="center" rowspan="1" colspan="1">12.4 Å</td>
<td valign="top" align="center" rowspan="1" colspan="1">18.4 Å</td>
<td valign="top" align="center" rowspan="1" colspan="1">11.7 Å</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">AMPKα2 (T172D)–KD-compound C (PDB: 3AQV)</td>
<td valign="top" align="center" rowspan="1" colspan="1">11.4 Å</td>
<td valign="top" align="center" rowspan="1" colspan="1">16.6 Å</td>
<td valign="top" align="center" rowspan="1" colspan="1">11.3 Å</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">AMPKα2 (T172D)–KD-SBI-0206965 (PDB: 6BX6)</td>
<td valign="top" align="center" rowspan="1" colspan="1">13.1 Å</td>
<td valign="top" align="center" rowspan="1" colspan="1">18.2 Å</td>
<td valign="top" align="center" rowspan="1" colspan="1">11.8 Å</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>A recent classification of kinase structural states based on DFG motif conformation used the positions of the αC helix and the DFG Phe<sup>158</sup> ring relative to Lys<sup>45</sup> (D1 and D2 distances) (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>). According to these measurements (<a href="#T2" class="usa-link">Table 2</a>), α2KD–BAY-3827 and AMPKα2(T172D) inhibitor structures of compound C and SBI-0206965 clustered together with other reported “DFG-in” structures (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>). To further classify the DFG conformation of the α2KD–BAY-3827 structure, the Phi/Psi (Φ/Ψ) angles of the x-DFG-x motif residues were also measured (<a href="#T3" class="usa-link">Table 3</a>). However, we could not place the structure in an established cluster, suggesting that this structure belongs to a specific DFG-in conformation with an inactive kinase state that has not yet been defined (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>). The SBI-0206965–bound structure (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>) had nearly identical Φ/Ψ angle measurements compared to α2KD–BAY-3827 (<a href="#T3" class="usa-link">Table 3</a>) and a “Phe<sup>158</sup>-down” orientation toward the C-lobe compared to the active AMPK conformation (<a href="#F2" class="usa-link">Fig. 2F</a> and fig. S2B). In AMPKα2(T172D)–compound C (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>), Phe<sup>158</sup> also pointed down toward the C-lobe but adopted a different conformation than in BAY-3827/SBI-0206965 structures (<a href="#F2" class="usa-link">Fig. 2F</a> and <a href="#T3" class="usa-link">Table 3</a>).</p>
<section class="tw xbox font-sm" id="T3"><h4 class="obj_head">Table 3. DFG motif Phe<sup>158</sup> Phi, Psi (Φ, ψ) angle measurements.</h4>
<div class="caption p"><p>Measurements were conducted in PyMOL (<a href="#R80" class="usa-link" aria-describedby="R80"><em>80</em></a>) retrieving available compound C (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>) and SBI-0206965–bound (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>) AMPKα2 (T172D) KD structures.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="left" scope="col" rowspan="1" colspan="1">Structures</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Ala<sup>156</sup> (Φ, ψ)</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Asp<sup>157</sup> (Φ, ψ)</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Phe<sup>158</sup> (Φ, ψ)</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Gly<sup>159</sup> (Φ, ψ)</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Leu<sup>160</sup> (Φ, ψ)</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="top" align="left" rowspan="1" colspan="1">α2KD–BAY-3827</td>
<td valign="top" align="center" rowspan="1" colspan="1">−77.8°, 133.2°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−64.3°, 162.0°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−90.6°, 42.6°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−91.5°, 174.0°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−69.5°, −13.6°</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">AMPKα2 (T172D)–KD compound C (PDB: 3AQV)</td>
<td valign="top" align="center" rowspan="1" colspan="1">−77.6°, 137.5°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−58.4°, 143.4°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−127.8°, 21.4°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−78.7°, −145.7°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−61.1°, −34.9°</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">AMPKα2 (T172D)–KD-SBI-0206965 (PDB: 6BX6)</td>
<td valign="top" align="center" rowspan="1" colspan="1">−83.3°, 141.7°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−56.9°, 150.6°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−90.9°, 68.7°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−125.6°, 179.3°</td>
<td valign="top" align="center" rowspan="1" colspan="1">−68.2°, −25.5°</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>In α2KD–BAY-3827, activation-loop residue Asn<sup>162</sup> occupies a position below Lys<sup>45</sup> in the N-lobe that is stabilized by hydrogen bonds with DFG residues Asp<sup>157</sup> and Gly<sup>159</sup> (<a href="#F2" class="usa-link">Fig. 2F</a>). This differs from its proximity to His<sup>137</sup> in the C-lobe observed in active AMPK (fig. S2B) (<a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>). As a result, Phe<sup>158</sup> is displaced downward toward the αE helix, and His<sup>137</sup> is pushed outward, away from its typical stack on Asp<sup>196</sup>, resulting in a broken hydrophobic spine characteristic of an inactive kinase (<a href="#F2" class="usa-link">Fig. 2F</a> and fig. S2K) (<a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>). Notably, in BAY-3827–bound α2KD, His<sup>137</sup> points toward the αC helix in the N-lobe, whereas in SBI-0206965– and compound C-bound structures, it faces the C-lobe (<a href="#F2" class="usa-link">Fig. 2F</a> and fig. S2K), highlighting two different ways of breaking the regulatory spine. Superimposing these three inhibitors also revealed a preserved αC helix conformation (<a href="#F2" class="usa-link">Fig. 2F</a> and fig. S2J). Overall, given their overlapping binding sites and DFG conformations (<a href="#F2" class="usa-link">Fig. 2, E and F</a>), the “FG-down” arrangement may represent a pivotal DFG-in transitional state in AMPK structural dynamics that is exploited by small molecule inhibitors.</p></section><section id="sec5"><h3 class="pmc_sec_title">BAY-3827 inhibits different AMPK trimeric complexes with similar potency</h3>
<p>Given the diverse assemblies that AMPK complexes can take depending on their subunit isoforms, the inhibitory action of BAY-3827 alongside BAY-974 compound control was studied using cell-free assays with different AMPK heterotrimeric complexes, namely α2β1γ1, α1β1γ1, and α2β2γ1 (<a href="#F3" class="usa-link">Fig. 3, A to E</a>). The α1β1γ1 and α2β2γ1 complexes showed similar IC<sub>50</sub> values in response to BAY-3827 treatment when tested at high (200 μM) ATP concentrations, with IC<sub>50</sub> values of 31.2 and 36.5 nM, respectively, in line with previous cell-free assay data with these two complexes (<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>), whereas the α2β1γ1 complex had a moderately higher IC<sub>50</sub> of 57 nM. At lower ATP (20 μM), BAY-3827–treated α1β1γ1 and α2β2γ1 had values of 1.8 and 2.8 nM, with the α2β1γ1 complex having an IC<sub>50</sub> value of 4.1 nM (<a href="#F3" class="usa-link">Fig. 3E</a>). Despite this modest difference, BAY-3827 had an inhibitory effect in the same order of magnitude and kinetics across all three tested AMPK complexes (<a href="#F3" class="usa-link">Fig. 3, A to C</a>), whereas BAY-974 showed no effect (<a href="#F3" class="usa-link">Fig. 3, D and E</a>). To examine the effect of different AMPK complexes on BAY-3827’s binding affinity, we also performed binding analyses on nonphosphorylated AMPKα2β2γ1 and AMPKα2β2γ3 complexes, which were obtained at a comparable purity level (fig. S2D). AMPKα2β2γ1 showed a slightly lower <em>K</em><sub>d</sub> value compared to the AMPKα2β2γ3 complex; however, both had a binding affinity found between an ~10 and 30 μM range (fig. S2, H and I). In addition, we also tested the effects of increasing concentrations of BAY-3827 and BAY-974 on selected kinases; ULK1 [known to be potently inhibited by SBI-0206965 (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>)] and CaMKK2 (upstream kinase of AMPK), and confirmed no inhibitory effect (fig. S3, A and B).</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3. BAY-3827 and BAY-974 kinetics.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372887_sciadv.adx2434-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d1e6/12372887/b6f064616cf0/sciadv.adx2434-f3.jpg" loading="lazy" height="1018" width="745" alt="Fig. 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong> to <strong>C</strong>) Inhibition of AMPK complexes in cell-free assays by BAY-3827 in high (200 μM) and low (20 μM) ATP concentrations and by (<strong>D</strong>) BAY-974 at 200 μM ATP. Data are <em>n</em> = 3 represented as means ± SEM. (<strong>E</strong>) IC<sub>50</sub> values (nM) of BAY-3827 inhibition of AMPK complexes in different ATP conditions. (<strong>F</strong> and <strong>G</strong>) HTRF assays in U2OS cells measuring phosphorylation (%) of Ser<sup>79</sup>-ACC1 following BAY-3827 or BAY-974 treatment and phosphorylation (%) of (<strong>H</strong> and <strong>I</strong>) Thr<sup><strong>172</strong></sup>-AMPKα in the presence or absence of AMPK activator MK-8722 (10 μM). (<strong>J</strong>) Estimated IC<sub>50</sub> and EC<sub>50</sub> values (μM) shown, respectively, from (F) to (I). Data are means ± SEM with <em>n</em> = 3 technical replicates from three independent experiments.</p></figcaption></figure></section><section id="sec6"><h3 class="pmc_sec_title">Cellular kinetics of BAY-3827 inhibitory action on AMPK</h3>
<p>Before using BAY-3827 and the control compound BAY-974 in cellular functional studies, their dose-dependent effect on AMPK activity was investigated in U2OS cells, an osteosarcoma cell line. Homogeneous time-resolved fluorescence (HTRF) is a high-throughput fluorescence resonance energy transfer–based technique previously benchmarked as a sensitive and quantitative assay for detection of ACC1 Ser<sup>79</sup> phosphorylation (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>, <a href="#R45" class="usa-link" aria-describedby="R45"><em>45</em></a>) and is an established surrogate marker for cellular AMPK activation. HTRF assays were performed following treatment with BAY-3827 and BAY-974 in the presence and absence of allosteric AMPK activator MK-8722 (<a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>). Vehicle compared to MK-8722 control treatment showed an expected significant increase in both pSer<sup>79</sup>-ACC1 and pThr<sup>172</sup>-AMPK (fig. S3, C and D). ACC1 phosphorylation dose-dependently decreased upon BAY-3827 treatment, with an IC<sub>50</sub> of 0.93 μM, which increased to a value of 6.36 μM in cells which were also treated with MK-8722. No inhibition was observed with BAY-974 even at the highest tested concentration of 100 μM (<a href="#F3" class="usa-link">Fig. 3, F, G, and J</a>). AMPK phosphorylation dose-dependently increased with BAY-3827, but not BAY-974, with a median effective concentration (EC<sub>50</sub>) of 1.93 and 0.57 μM in the presence of MK-8722 (<a href="#F3" class="usa-link">Fig. 3, H to J</a>). This observed increase in pAMPK is consistent with the reported effects of BAY-3827 on AMPK phosphorylation levels, possibly due to its protective effect against Thr<sup>172</sup> dephosphorylation (<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>).</p></section><section id="sec7"><h3 class="pmc_sec_title">BAY-3827 inhibits AMPK and blocks its suppressive effect on lipogenesis in hepatocytes</h3>
<p>We next sought to establish whether BAY-3827 inhibited an AMPK-dependent process in a more physiologically relevant cell system. Mouse primary hepatocytes were treated with increasing doses of BAY-3827 with or without MK-8722 (10 μM). Immunoblot analysis revealed a dose-dependent reduction in phosphorylation of ACC1 and Raptor, with full inhibition in the presence of MK-8722 achieved at 5 and 2.5 μM. (<a href="#F4" class="usa-link">Fig. 4, A to D</a>). Increased pThr<sup>172</sup>AMPK levels were only significant under MK-8722–treated conditions at 5 μM (<a href="#F4" class="usa-link">Fig. 4E</a>). BAY-974 (5 μM) showed no effect on levels of pACC1, pRaptor, or pAMPK with or without MK-8722 (<a href="#F4" class="usa-link">Fig. 4, C to E</a>).</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4. BAY-3827 but not BAY-974 dose-dependently reverses AMPK-inhibited lipogenesis in primary hepatocytes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372887_sciadv.adx2434-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d1e6/12372887/29960707c9e1/sciadv.adx2434-f4.jpg" loading="lazy" height="917" width="775" alt="Fig. 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Representative blots of AMPKα1α2<sup>+/+</sup> (control) hepatocytes treated with BAY-3827 and BAY-974 in basal conditions and (<strong>B</strong>) treated with 10 μM MK-8722 after compound treatment for 15 min, with selected antibodies describing AMPK signaling. (<strong>C</strong> to <strong>E</strong>) Calculated phosphorylation ratios following band intensity quantification of data from (A) and (B) showing means ± SEM from four separate experiments with <em>n</em> = 2 to 3 biological replicates subjected to a two-way analysis of variance (ANOVA) test with *<em>P</em> &lt; 0.05 (vehicle versus treatments) and <sup>#</sup><em>P</em> &lt; 0.05 (basal versus MK-8722). (<strong>F</strong>) Lipogenesis assay of control and AMPKα1α2<sup>−/−</sup> DKO primary hepatocytes treated with BAY-3827 and inactive BAY-974, where <em>n</em> = 3 to 5 biological replicates from five separate experiments. A two-way ANOVA test was conducted, where *<em>P</em> &lt; 0.05 (vehicle versus indicated treatment), #<em>P</em> &lt; 0.05 (vehicle + MK-8722 versus in indicated treatment) and †<em>P</em> &lt; 0.05 (control versus DKO treatments in basal and + MK-8722 conditions).</p></figcaption></figure><p>One of the best-characterized physiological consequences of AMPK activation in hepatocytes is the suppression of fatty acid synthesis through phosphorylation of Ser<sup>79</sup> on ACC1 (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>, <a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>). Mouse primary hepatocytes derived from AMPKα1α2<sup>+/+</sup> control and liver-specific AMPKα1α2<sup>−/−</sup> DKO mice were treated with BAY-3827 (2.5 and 5 μM) with or without MK-8722 (10 μM), and the [<sup>14</sup>C]-acetate incorporation into fatty acids was assessed. In control hepatocytes, BAY-3827 alone significantly stimulated (~70 to 90%) lipogenesis, while MK-8722 alone robustly suppressed it (&gt;80%) (<a href="#F4" class="usa-link">Fig. 4F</a>). BAY-3827 dose-dependently rescued the MK-8722–mediated inhibition of lipogenesis. In AMPKα1α2<sup>−/−</sup> DKO cells, basal lipogenesis was significantly higher than vehicle-treated, but similar to BAY-3827–treated control hepatocytes. In AMPKα1α2<sup>−/−</sup> DKO hepatocytes, BAY-3827 and MK-8722 treatment had no effect on lipogenesis. In addition, BAY-974 treatment (5 μM) had no effect on lipogenesis in both vehicle- and MK-8722–treated conditions (<a href="#F4" class="usa-link">Fig. 4F</a>). We also tested whether compound C was able to rescue MK-8722–mediated inhibition of hepatic lipogenesis and observed that it only negligibly (10%) restored it at 25 μM, confirming its low potency in cells (fig. S3E). These results establish BAY-3827 as a potent and selective compound to probe the AMPK-dependent regulation of lipogenesis in response to pharmacological and genetic modulations in hepatocytes.</p>
<p>We have previously reported that SBI-0206965 not only inhibits AMPK-related kinases (NUAK1 and MARK3/4) equally or more potently than AMPK or ULK1 but also inhibits critical cellular functions such as basal and insulin-stimulated glucose uptake in adipocytes and skeletal muscle (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). We treated mouse primary adipocytes and isolated mouse skeletal muscle ex vivo with varying doses of BAY-3827 with or without MK-8722 or insulin (fig. S3, F to K). We observed a dose-dependent reduction in Raptor phosphorylation in response to BAY-3827 irrespective of MK-8722’s presence (fig. S3, F and G). Both basal and insulin-stimulated phosphorylation of Akt and TBC1D4, as well as glucose uptake were not significantly affected in adipocytes (fig. S3, H to J). In ex vivo skeletal muscle, BAY-3827 dose-dependently inhibited MK-8722–stimulated glucose uptake and showed a full inhibition at 5 μM (fig. S3K). BAY-3827 attenuated insulin-stimulated (AMPK-independent) glucose uptake in ex vivo skeletal muscle, indicating a potential off-target effect.</p></section><section id="sec8"><h3 class="pmc_sec_title">BAY-3827 diminishes drug-stimulated AMPK-dependent gene expression</h3>
<p>To further examine the effect of BAY-3827 on AMPK signaling and gene expression in an unbiased manner, we treated control and AMPKα1α2<sup>−/−</sup> DKO mouse primary hepatocytes with 5 μM BAY-3827 in the presence or absence of MK-8722 and performed RNA sequencing (RNA-seq) analysis. We focused on 1841 genes that were significantly up-regulated in MK-8722 treatment compared to basal conditions (vehicle-treated) in control hepatocytes. To confidently define them as AMPK-dependent genes, we selected genes that were significantly down-regulated in MK-8722–treated AMPKα1α2<sup>−/−</sup> DKO cells compared to control. This yielded 845 genes that were both MK-8722 stimulated and AMPK dependent, with their expression visualized in a heatmap where the MK-8722–stimulated gene expression in control cells was almost completely abolished in AMPKα1α2<sup>−/−</sup> DKO cells (<a href="#F5" class="usa-link">Fig. 5A</a>). In hepatocytes treated with BAY-3827 and MK-8722, approximately 30% of MK-stimulated AMPK genes were down-regulated (524 genes of 845 predefined MK-8722-AMPK genes) (<a href="#F5" class="usa-link">Fig. 5, A and B</a>, and fig. S4A). In control hepatocytes treated with BAY-3827 and MK-8722, 2511 significant genes were differentially expressed compared to MK-8722, with 61.2% (1539 genes) down-regulated by BAY-3827 treatment in AMPK-activated conditions. Top significant [fold change (FC) &lt; −1.3; false discovery rate (FDR) &lt; 0.05] down-regulated genes included <em>PLCE1</em>, <em>ZFP36</em>, <em>FIGNL2</em>, <em>ADAMTSL4</em>, and <em>TIMP3</em> (<a href="#F5" class="usa-link">Fig. 5C</a> and table S3). In MK-8722–treated AMPKα1α2<sup>−/−</sup> DKO cells compared to control, <em>ADAMTSL4</em> and <em>ZFP36</em> were also among the top down-regulated genes (fig. S4B).</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Fig. 5. Unbiased transcriptome analysis of BAY-3827 ± MK-8722–treated AMPKα1α2<sup>+/+</sup> (control) and AMPKα1α2<sup>−/−</sup> DKO primary hepatocytes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372887_sciadv.adx2434-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d1e6/12372887/05539f67f44a/sciadv.adx2434-f5.jpg" loading="lazy" height="1018" width="719" alt="Fig. 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Heatmap of gene expression of 845 significant genes that are stimulated by MK-8722 and are AMPK-dependent. (<strong>B</strong>) Venn diagram showing the overlap between genes with a significantly reduced expression when treated with BAY-3827 in MK-8722–stimulated control cells compared to MK-8722–stimulated/AMPK-regulated genes, with a 30.1% overlap. (<strong>C</strong>) Volcano plot showing top significant (FDR &lt; 0.05) up-regulated (FC ≥ 1.3; red), down-regulated (FC &lt; −1.3; blue), and nonsignificant (ns) regulated genes by BAY-3827 in MK-8722–stimulated control cells compared to MK-8722 treatment alone. (<strong>D</strong>) GO enrichment analysis of biological processes of down-regulated significant genes (FDR &lt; 0.05) by BAY-3827 + MK-8722 treatment and (<strong>E</strong>) by the loss of AMPKα1α2 in MK-8722–treated cells. (<strong>F</strong>) Schematic of the down-regulated GO processes showing the functional overlap between MK-8722–stimulated AMPKα1α2<sup>−/−</sup> DKO versus control cells and BAY-3827 + MK-8722 treated versus MK-8722 in control cells. Created in BioRender. C. Fraguas (2025) (<a href="https://BioRender.com/j60q096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/j60q096</a>).</p></figcaption></figure><p>Gene ontology (GO) enrichment analysis of biological processes was performed to query the gene functions of those down-regulated when AMPK is inactivated either via genetic (AMPKα1α2<sup>−/−</sup> DKO) or pharmacological (BAY-3827) means, in the presence of MK-8722 (<a href="#F5" class="usa-link">Fig. 5, D to F</a>). The top 25 categories (FDR &lt; 0.05) of down-regulated genes in MK-8722–stimulated AMPKα1α2<sup>−/−</sup> DKO and BAY-3827–treated control cells revealed a large overlap in known AMPK-regulated functions. These functions, which BAY-3827 also inhibited, included lipid oxidation and modulation, organic acid catabolic processes, and response to nutrient levels and starvation (<a href="#F5" class="usa-link">Fig. 5F</a>). Specific functions of BAY-3827–down-regulated genes in MK-8722–stimulated control cells included nucleoside biphosphate and ketone metabolism, as well as the regulation of peroxisome proliferator–activated receptor signaling pathway (<a href="#F5" class="usa-link">Fig. 5, D and F</a>). Uniquely down-regulated functions in MK-8722–induced AMPKα1α2<sup>−/−</sup> DKO compared to control cells included autophagy-related functions as well as steroid and secondary alcohol metabolic processes (<a href="#F5" class="usa-link">fig. 5, E and F</a>). Treatment of BAY-3827 in the presence of MK-8722 resulted in a significant decrease of AMPK-dependent genes involving lipid metabolic genes, which in MK-8722 alone conditions are normally up-regulated, and promote lipid oxidation and other catabolic processes (fig. S4C). AMPK-dependent genes that were down-regulated by BAY-3827 treatment in MK-8722–treated control cells included fatty acid metabolism genes <em>PLIN5</em>, <em>LPIN2</em>, <em>EHHADH</em>, <em>AVPR1A</em>, and <em>ADORA2B</em> as well as liver metabolism regulator <em>SDS</em> and histidine catabolism gene <em>HAL</em>, establishing the role of BAY-3827 in suppressing the expression of AMPK-regulated lipid oxidative genes (<a href="#T4" class="usa-link">Table 4</a> and fig. S4C). Apart from lipid metabolic genes, BAY-3827 also reduced the expression of genes involved in transcription regulation, such as <em>SP1,</em> which encodes a transcription factor proposed to be AMPK regulated (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>). Another example includes the <em>TBCD15</em> and <em>TBCD17</em> genes with roles in autophagy, with encoded Rab-GTPase activating protein TBCD17 recently described as an AMPK substrate (table S3) (<a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>, <a href="#R50" class="usa-link" aria-describedby="R50"><em>50</em></a>).</p>
<section class="tw xbox font-sm" id="T4"><h4 class="obj_head">Table 4. Examples of AMPK-dependent genes down-regulated in BAY-3827 in MK-8722–stimulated control hepatocytes.</h4>
<div class="caption p"><p>The complete list of differentially expressed genes in BAY-3827 + MK-8722 versus MK-8722 in control hepatocytes is recorded in table S3.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="left" scope="col" rowspan="1" colspan="1">Gene</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Name</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">LogFC</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">FDR</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="top" align="left" rowspan="1" colspan="1">
<em>LIPE</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">Lipase, hormone sensitive</td>
<td valign="top" align="center" rowspan="1" colspan="1">−0.28159</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.004450324</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>PFKFB2</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">6-phosphofructo-2-kinase/fructose-2,6-biphosphatase 2</td>
<td valign="top" align="center" rowspan="1" colspan="1">−0.54205</td>
<td valign="top" align="center" rowspan="1" colspan="1">3.09 × 10<sup>−08</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>HMGCR</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">3-hydroxy-3-methylglutaryl–CoA reductase</td>
<td valign="top" align="center" rowspan="1" colspan="1">−0.16208</td>
<td valign="top" align="center" rowspan="1" colspan="1">0.00451032</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>PLIN5</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">Perilipin 5</td>
<td valign="top" align="center" rowspan="1" colspan="1">−1.29957</td>
<td valign="top" align="center" rowspan="1" colspan="1">5.66092 × 10<sup>−18</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>LPIN2</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">Lipin 2</td>
<td valign="top" align="center" rowspan="1" colspan="1">−0.52774</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.10 × 10<sup>−08</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>PLIN4</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">Perilipin 4</td>
<td valign="top" align="center" rowspan="1" colspan="1">−1.23494</td>
<td valign="top" align="center" rowspan="1" colspan="1">4.01982 × 10<sup>−14</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>SP1</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">trans-acting transcription factor 1</td>
<td valign="top" align="center" rowspan="1" colspan="1">−0.43973</td>
<td valign="top" align="center" rowspan="1" colspan="1">7.4359 × 10<sup>−13</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>HPGD</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">Hydroxyprostaglandin dehydrogenase 15 (NAD)</td>
<td valign="top" align="center" rowspan="1" colspan="1">−1.745</td>
<td valign="top" align="center" rowspan="1" colspan="1">5.54633 × 10<sup>−14</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>EHHADH</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">Enoyl-CoA, hydratase/3-hydroxyacyl CoA dehydrogenase</td>
<td valign="top" align="center" rowspan="1" colspan="1">−1.5955</td>
<td valign="top" align="center" rowspan="1" colspan="1">2.79131 × 10<sup>−17</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>AVPR1A</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">Arginine vasopressin receptor 1A</td>
<td valign="top" align="center" rowspan="1" colspan="1">−1.70797</td>
<td valign="top" align="center" rowspan="1" colspan="1">7.31686 × 10<sup>−13</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>ADORA2B</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">Adenosine A2b receptor</td>
<td valign="top" align="center" rowspan="1" colspan="1">−1.3062</td>
<td valign="top" align="center" rowspan="1" colspan="1">7.77511 × 10<sup>−11</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>TBC1D15</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">TBC1 domain family, member 15</td>
<td valign="top" align="center" rowspan="1" colspan="1">−0.48954</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.47094 × 10<sup>−15</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>TBC1D17</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">TBC1 domain family, member 17</td>
<td valign="top" align="center" rowspan="1" colspan="1">−0.559835768</td>
<td valign="top" align="center" rowspan="1" colspan="1">9.72654 × 10<sup>−17</sup>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>HAL</em>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">Histidine ammonia lyase</td>
<td valign="top" align="center" rowspan="1" colspan="1">−1.34842</td>
<td valign="top" align="center" rowspan="1" colspan="1">1.52157 × 10<sup>−09</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>In addition, given AMPK’s established roles in mitochondrial biogenesis, to validate our dataset, we visualized the normalized counts of the <em>PPARGC1A</em> gene across sample treatments with vehicle [dimethyl sulfoxide (DMSO)], BAY-3827, MK-8722, or BAY-3827 + MK-8722 in control and AMPKα1α2<sup>−/−</sup> DKO genotypes. We observed a significant count increase in the MK-8722 condition compared to vehicle in control cells but not in AMPKα1α2<sup>−/−</sup> DKO cells, and, furthermore, BAY-3827 + MK-8722 significantly reduced gene counts compared to MK-8722 treatment alone in AMPKα1α2<sup>+/+</sup> cells (fig. S4D). Moreover, <em>PPARGC1A</em> was among the top significant down-regulated genes by BAY-3827 + MK-8722 compared to MK-8722 treatment alone in control hepatocytes, with a <em>P</em>-adjusted value of 6.5083 × 10<sup>−05</sup> (table S3). Together, these data indicate that BAY-3827 inhibits AMPK-induced increase in <em>PPARGC1A</em>, a key regulator of mitochondrial biogenesis.</p>
<p>Yang <em>et al.</em> (<a href="#R51" class="usa-link" aria-describedby="R51"><em>51</em></a>) recently described the transcriptional profiles of RSK1- and RSK2 knockout glioma-based cell lines. We found proposed RSK-regulated genes that were down-regulated by BAY-3827 in MK-8722–stimulated control cells. These included <em>STAT5B</em> and <em>CALCOCO1</em> (RSK2) as well as <em>NEK2</em>, <em>DSN1</em>, <em>MASTL</em>, and <em>SPC25</em> (RSK1) among others (fig. S4E and table S3) (<a href="#R51" class="usa-link" aria-describedby="R51"><em>51</em></a>).</p></section><section id="sec9"><h3 class="pmc_sec_title">BAY-3827 inhibits RSK in cells</h3>
<p>Although previous (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>) and current in vitro (cell-free) data (<a href="#F1" class="usa-link">Fig. 1A</a>) have shown that BAY-3827 potently inhibits RSK isoforms, this has not been verified in a cellular context. We treated serum-starved human embryonic kidney–293 (HEK293) cells with varying concentrations of BAY-3827 or the RSK inhibitor BI-D1870 (<a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>) with or without phorbol 12-myristate 13-acetate (PMA), an activator of extracellular signal–regulated kinase (ERK) 1/2 and RSK isoforms, and assessed phosphorylation of glycogen synthase kinase (GSK) 3α/β as a surrogate marker for cellular RSK activity (<a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>). Since phosphorylation of GSK3α/β is also regulated by Akt, we included the Akt inhibitor MK-2206 to specifically assess RSK-mediated (i.e., PMA-stimulated) phosphorylation of GSK3α/β. As anticipated, BAY-3827 treatment resulted in a dose-dependent reduction of ACC1 phosphorylation, while BI-D1870 treatment had no effect (fig. S5, A and B). MK-2206 treatment reduced basal pGSK3αβ, and PMA robustly increased pERK1/2 and pGSK3αβ levels under MK-2206–treated conditions (fig. S5, C and D). PMA-stimulated pGSK3αβ (in the presence of MK-2206) was inhibited to the same extent by 10 μM BAY-3827 and BI-D1870 (fig. S5D), indicating that BAY-3827 has an inhibitory potency similar to BI-D1870 in suppressing RSK in HEK293 cells.</p>
<p>To explore the molecular basis for RSK inhibition by BAY-3827, we conducted an in silico superimposition of the BAY-3827–α2KD structure with available structures of RSK2 N-terminal (NT) KD in complex with BI-D1870 and SL0101 RSK inhibitors (<a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>, <a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>). Both compounds had overlapping binding sites with BAY-3827–α2KD as well as conserved interacting residues with AMPK (fig. S5, E and F). In addition, both inhibitors make hydrogen bond contacts with hinge region residues Glu<sup>94</sup> (Asp<sup>148</sup> in RSK2) and Val<sup>96</sup> (Leu<sup>150</sup> in RSK2) where the latter in SL0101 is water mediated (fig. S5, E and F). This suggests that the effect of BAY-3827 on RSK inhibition is likely derived from these conserved structural features as well as from the compound binding mode.</p></section></section><section id="sec10"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>Here, we provide the structure-based mechanism of action of BAY-3827 in the inhibition of AMPK, describing distinct and previously unidentified molecular features. We also have comprehensively evaluated the cellular potency and selectivity (i.e., on/off-target effects) of BAY-3827 alongside its inactive control BAY-974 in signaling, transcriptomic, and metabolic output levels by using control and AMPKα1α2<sup>−/−</sup> DKO primary hepatocytes. Consistent with a previous study (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>), we show that BAY-3827 potently inhibits the AGC group kinases RSK1 and RSK2 in cell-free assays, albeit to a lower extent than AMPK, as shown across a panel of 140 human protein kinases (<a href="#F1" class="usa-link">Fig. 1A</a> and table S1). Compound C and SBI-0206965, based on their reported selectivity profiles (<a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>, <a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>, <a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>) did not affect the RSK family, although other compounds reported to inhibit AMPK have also shown a preference for this kinase family. For instance, the ATP-competitive PAK4 inhibitor PF-03758309 has been reported to potently inhibit AMPK with an IC<sub>50</sub> of 1 nM in a cell-free assay. PF-03758309 is also known to potently inhibit RSK1 and RSK2 with an IC<sub>50</sub> of 2 nM (<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>), where in silico docking to AMPK KD predicted three hydrogen bonding interactions in the hinge region with Glu<sup>94</sup> and Val<sup>96</sup> residues (<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>) in the same fashion as in the current α2KD–BAY-3827 structure. RSK inhibitors BI-D1870 and SL0101 both make contacts with these two hinge residues, which correspond to Asp<sup>148</sup> and Leu<sup>150</sup> in the reported RSK2 structures (fig. S5, E and F) (<a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>, <a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>), sharing some interaction residues with BAY-3827’s binding mode, pointing to a structural basis for RSK inhibition. In line with this notion, and with results from current and previous (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>) cell-free assays, our cellular studies revealed that BAY-3827 treatment inhibited PMA-stimulated RSK activation to the same extent as BI-D1870 (fig. S5, A and D). Thus, RSK isoforms are validated off-targets of BAY-3827, and RSK inhibitors could be included as an additional control alongside BAY-974 (which does not inhibit RSKs) to mitigate the involvement of RSK isoforms.</p>
<p>The cocrystal structure of BAY-3827 with inactive WT α2KD reveals the molecular mechanism allowing compound inhibition, with past inhibitor structures (compound C, SBI-0206965) were instead obtained using a phosphomimetic KD; AMPKα2 (T172D). BAY-3827 binds the ATP-binding pocket at a site that overlaps with SBI-0206965, consistent with past biochemical data deeming both SBI-0206965 and BAY-3827 compounds as mixed-type inhibitors (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a><em>,</em>
<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>). Superimposition of α2KD–BAY-3827 with SBI-0206965– and compound C-bound structures revealed an activation loop conformation characteristic of an inactive kinase, with a conserved αC helix-out state (<a href="#F2" class="usa-link">Fig. 2F</a>). However, in the BAY-3827–bound conformation, activation loop residue Asn<sup>162</sup> is found in a position that disrupts the AMPK regulatory spine and where His<sup>137</sup> is displaced (fig. S2K). Phe<sup>158</sup> residue faces down toward the C-lobe compared to the active state, with a conserved conformation with the SBI-0206965–bound structure (<a href="#F2" class="usa-link">Fig. 2F</a> and fig. S2B), which is likely due to their shared binding sites.</p>
<p>The AMPKα2 (T172D)–SBI-0206965 structure (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>) was first described as a type IIB structure in a noncanonical DFG-out conformation; however, distance and Φ/Ψ angle measurements following an updated kinase nomenclature (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>) revealed that it clusters together with the BAY-3827–bound structure with other DFG-in states (<a href="#T2" class="usa-link">Table 2</a>). DFG-in structures are often associated with active kinases; however, they are also found in inactive states (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>, <a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>, <a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>). In these structures, an inactive state is achieved either due to αC helix movement or activation loop conformation (<a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>). In the case of BAY-3827 binding, we find that both factors contribute to this, mainly by disrupting the regulatory spine that is otherwise uninterrupted in an active kinase. Given the conserved orientation of the Phe<sup>158</sup> ring toward the C-lobe, we propose that an FG-down conformation, known to be associated with DFG-in states (<a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>), might be important in AMPK structural dynamics upon compound-based inhibition. We further observed a cysteine disulfide bridge unique to the α2KD–BAY-3827 structure between the αD Cys<sup>106</sup> and activation loop Cys<sup>174</sup>, which we hypothesize helps to stabilize activation loop conformation in an unproductive kinase state upon BAY-3827 binding (<a href="#F2" class="usa-link">Fig. 2D</a>). This interaction has not been observed previously in available α2KD-inhibitor co-structures, and Cys<sup>174</sup> alongside the close-by Phe<sup>102</sup>, which is displaced as a result of this interaction, were among the top residues predicted as highly conserved in BAY-3827–inhibited kinases but were lowly conserved in noninhibited kinases (<a href="#F1" class="usa-link">Fig. 1, E and F</a>, and table S2), suggesting that the A-loop stabilization is a contributing factor to BAY-3827 inhibition of AMPK. Although Cys<sup>174</sup> is a highly conserved activation loop residue across CAMK and AGC Ser/Thr kinases (<a href="#R58" class="usa-link" aria-describedby="R58"><em>58</em></a>), our data indicate that BAY-3827’s kinase selectivity is mediated by a cohort of residues (table S2 and fig. S1C), and, hence, selectivity is not solely imparted by the disulfide bridge, but rather, this interaction points to a stabilizing feature that we find contributes to BAY-3827’s binding affinity (fig. S2, E to G). Previous crystal structure studies with other kinases have shown intramolecular disulfides in the activation segment region (<a href="#R59" class="usa-link" aria-describedby="R59"><em>59</em></a>); however, the Akt2 KD structure lacked solved residues between the two interacting cysteines (Cys<sup>297</sup>-Cys<sup>311</sup>), which was not the case in the AMPK α2KD–BAY-3827 structure, and Cys<sup>297</sup> is not conserved in AMPK. Cells are known to maintain high reducing environments, raising the possibility that the observed disulfide Cys<sup>106</sup>-Cys<sup>174</sup> bridge could be affected under such conditions. To investigate this, we studied BAY-3827–binding affinity in the presence or absence of 3 mM DTT in both WT and T172D AMPKα2-KD and found that a reducing environment lowered BAY-3827’s binding affinity in WT but was unaffected in T172D-α2KD compared to no-DTT (fig. S2, E to G).</p>
<p>We also report that BAY-3827 increases AMPK-Thr<sup>172</sup> phosphorylation in U2OS cells, mouse primary hepatocytes, and mouse primary adipocytes (<a href="#F4" class="usa-link">Fig. 4</a> and fig. S3). It was recently suggested that the binding of BAY-3827 (and also SBI-0206965) (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>, <a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>) induces a protective effect toward phosphatase-meditated dephosphorylation of Thr<sup>172</sup> (<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>). Given the dose-dependent reduction in AMPK downstream target phosphorylation such as ACC1 and Raptor, we find that it does not reflect functional activation status of AMPK. Rather, BAY-3827 likely stabilizes an activation loop conformation that contributes to phosphatase access shielding as well as an inactive conformation for substrate engagement.</p>
<p>We have demonstrated that BAY-3827 is capable of fully reversing MK-8722–induced inhibition of fatty acid synthesis and down-regulating ~30% of MK-8722–stimulated AMPK-regulated genes (<a href="#F4" class="usa-link">Fig. 4B</a>) involved in known AMPK-governing functions such as fatty acid oxidation and lipid catabolism (<a href="#F4" class="usa-link">Fig. 4, D to F</a>). Lemos <em>et al.</em> (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>) treated androgen-dependent prostate cancer cells (LNCaP) with BAY-3827 and reported several down-regulated genes involved in lipid metabolism that we also observed were down-regulated by BAY-3827 treatment in MK-8722–stimulated control primary hepatocytes, namely <em>LIPE</em>, <em>PFKFB2</em>, and <em>HMGCR</em>. Our analysis also uncovered other lipid metabolism–related genes with their expression dependent on the presence of functional AMPK, which were significantly down-regulated following the treatment with BAY-3827 when AMPK had been pharmacologically activated (fig. S4, A and C). The RSK family encompasses a conserved group of serine/threonine protein kinases formed by human isoforms RSK1, RSK2, RSK3, and RSK4 alongside RSK-like mitogen- and stress-activated kinases 1 (MSK1) and 2 (MSK2), with key functions in cellular growth and proliferation such as cell survival promotion, protein translation, and ribosome biogenesis (<a href="#R60" class="usa-link" aria-describedby="R60"><em>60</em></a>). Because of their growth-promoting roles alongside ensuring cell survival, RSKs are also established targets in anticancer therapies and are associated with increased expression in certain types of cancer (<a href="#R61" class="usa-link" aria-describedby="R61"><em>61</em></a>). We searched for RSK-regulated genes (<a href="#R51" class="usa-link" aria-describedby="R51"><em>51</em></a>) in our dataset and found that BAY-3827 in MK-8722–stimulated control hepatocytes significantly down-regulated some RSK-proposed genes, with a larger number of previously defined RSK1 genes present in our dataset compared to RSK2 (fig. S4D). This is consistent with our data showing higher inhibition of RSK1 compared to RSK2 isoform (table S1) and warrants further investigations.</p>
<p>As emerging roles and therapeutic implications of AMPK arise, it is critical to perform robust proof-of-concept experiments with multifaceted approaches. Although there are well-established preclinical genetic models (<a href="#R62" class="usa-link" aria-describedby="R62"><em>62</em></a>, <a href="#R63" class="usa-link" aria-describedby="R63"><em>63</em></a>) and allosteric activators (<a href="#R9" class="usa-link" aria-describedby="R9"><em>9</em></a>), availability of potent and selective inhibitors is largely limited. Here, we have delineated the molecular basis of BAY-3827’s inhibition of AMPK and robustly characterized AMPK-regulated transcriptomic and metabolic functions, alongside building on the knowledge of AMPK-specific structural features, which will facilitate future efforts in inhibitor discovery and rational drug design. We have also described the current limitations of BAY-3827 (e.g., potent inhibition of RSK) and have extensively characterized BAY-974 across different cell lines and concentration range (0.03 to 100 μM), deeming this compound as a good experimental control for BAY-3827 use. In addition, we propose the context-dependent use of RSK inhibitors. In summary, BAY-3827 offers an increased potency and selectivity compared to prevalently used inhibitors (e.g., compound C, SBI-0206965), making it the preferred choice for cell-based studies. Still, potent inhibition of RSK isoforms and poor pharmacokinetic properties limit its in vivo use, and these issues will need to be addressed in future efforts to identify AMPK inhibitors.</p></section><section id="sec11"><h2 class="pmc_sec_title">MATERIALS AND METHODS</h2>
<section id="sec12"><h3 class="pmc_sec_title">Animals</h3>
<p>Animal experiments were conducted in accordance with the European directive 2010/63/EU of the European Parliament and of the Council of the protection of animals used for scientific purposes. Ethical approval was given by the Danish Animal Experiments Inspectorate (license number #2021-15-0201-01059). For primary hepatocytes (signaling studies) and ex vivo skeletal muscle incubation experiments, WT C57BL/ 6N Tac male mice (10 to 16 weeks old) were obtained from Taconic Biosciences and housed in the animal facility at the Faculty of Health and Medical Sciences (University of Copenhagen). Generation of AMPKα1α2<sup>lox/lox</sup> and liver-specific AMPKα1α2 DKO (C57BL/6 background) for lipogenesis and RNA-seq experiments were described previously (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). Animals were anesthetized with Avertin [stock of tribromoethanol (1 g/ml; #<a href="https://www.ncbi.nlm.nih.gov/nuccore/T48402" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">T48402</a>, Merck MilliporeSigma)] in 2-methyl-2-butanol (#152463, Merck MilliporeSigma), diluted 1:20 in saline, and dosed at 10 μl/g body weight via intraperitoneal injection before the procedure.</p>
<p>For adipocyte experiments, animal experiments were approved by the Regional Ethical Committee on Animal Experiments in Malmö/Lund (approval number 5.8.18–19111/2023). WT C57BL6/J male mice (10 to 11 weeks old) were obtained from Taconic Biosciences and kept in the animal facility at the Biomedical Centre (Lund University, Sweden). All the animals were kept and maintained according to local regulations under a light/dark cycle of 12 hours, 22° ± 1°C, and had free access to water and standard chow diet.</p></section><section id="sec13"><h3 class="pmc_sec_title">Mouse primary hepatocyte isolation and lipogenesis assay</h3>
<p>Mouse hepatocytes were isolated by collagenase perfusion as previously described (<a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>). Hepatocytes were seeded in medium Eagle-199 (MEM-199) (#41150, Thermo Fisher Scientific) containing penicillin G (100 U/ml), streptomycin (100 μg/ml), and 10% (v/v) fetal bovine serum (FBS). Hepatocytes were left for attachment (3 to 4 hours) and serum-starved overnight at 37°C with 5% CO<sub>2</sub> in MEM-199 supplemented with penicillin G (100 U/ml), streptomycin (100 μg/ml), 10 nM insulin, and 100 nM dexamethasone. Isolated hepatocytes were seeded and after 16 to 18 hours incubated for 3 hours for signaling studies as previously described (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>) or were incubated with MEM-199 media supplemented with 0.6 μCi/ml [<sup>14</sup>C]-acetate (PerkinElmer, #NEC084H001MC) in the presence of BAY-3827 (MedChemExpress, #HY-112083) or inactive BAY-974 (donated probe from the Structural Genomics Consortium; SGC-Frankfurt) cotreated with MK-8722 (MedChemExpress, HY-111363) or vehicle (0.1% DMSO), respectively. Cells were then harvested in 0.5 ml of phosphate-buffered saline (PBS), transferred into 1 ml of 40% KOH and 2 ml of methanol followed by 1-hour incubation at 80°C. Lipids were saponified by acidifying the samples in 37% HCl and extracted with petroleum ether. Extracts were allowed to evaporate to dryness and then dissolved in Ultima Gold scintillation fluid for determination of [<sup>14</sup>C]-acetate incorporation into lipids.</p></section><section id="sec14"><h3 class="pmc_sec_title">Mouse primary adipocyte isolation, compound treatment, and glucose uptake</h3>
<p>Epidydimal adipose tissue excised from mice was digested with collagenase (1 mg/ml) in Krebs-Ringer medium (120 mM NaCl, 4.7 mM KCl, 1.2 mM KH<sub>2</sub>PO<sub>4</sub>, and 1.2 mM MgSO<sub>4</sub>) containing 25 mM Hepes (pH 7.4), 200 nM adenosine, 2 mM glucose, and 3% (w/v) bovine serum albumin (BSA) (KRH buffer) at 37°C in a shaking incubator. Adipocytes were isolated by filtering and washing in KRH buffer. For preparation of protein extracts for Western blot, adipocyte suspensions [0.7 to 1 ml of 10% (v/v) cells] were preincubated with BAY-3827 or BAY-974 compounds or 0.1% DMSO for 1 hour at 37°C in a shaking (80 rpm) water bath followed by either 1 hour of incubation with 10 μM MK-8722 or 30 min with 0.1 nM insulin (Novo Nordisk) in KRH buffer, as indicated in the figure legends. Cells were then washed in KRH buffer without BSA and lysed in 50 mM tris-HCl (pH 7.5), 0.27 M sucrose, 1 mM EDTA, 1 mM EGTA, 5 mM sodium pyrophosphate, 1 mM sodium orthovanadate, 50 mM sodium fluoride, 1 mM dithiothreitol, 1% (w/v) NP-40, and complete protease inhibitor cocktail (one tablet/50 ml) included in lysis buffer. Lysates were centrifuged at 13,000<em>g</em> for 15 min (4°C), and supernatant protein concentration was determined by the Bradford assay. Freshly isolated mouse adipocytes were washed in glucose-free buffer containing 30 mM Hepes (pH 7.4), 120 mM NaCl, 4 mM KH<sub>2</sub>OPO<sub>4</sub>, 1 mM MgSO<sub>4</sub>, 0.75 mM CaCl<sub>2</sub>, 10 mM NaHCO<sub>3</sub>, 200 nM adenosine, and 3% (w/v) BSA (KRBH buffer). Adipocyte suspensions [400 μl of 3.75% (v/v) cells] were preincubated with BAY compounds or 0.1% DMSO for 1 hour at 37°C in a shaking (80 rpm) water bath, before being stimulated with 0.1 insulin (Novo Nordisk) or 10 μM cytochalasin B (Sigma-Aldrich) for 30 min. Subsequently, 100 μl KRBH buffer containing 0.25 μl (0.025 μCi) [<sup>14</sup>C]-glucose (275 mCi/mmol glucose; final glucose concentration = 0.18 μM) was added, and the cells were incubated for a further 30 min. Reactions were stopped by aliquoting 300 μl of the total 500-μl adipocyte suspension to Beckman microtubes containing 75 μl of dinonylphtalate. The adipocyte suspension was centrifuged at 6000<em>g</em> and frozen at −20°C before adipocytes were collected and subjected to scintillation counting. The assay was performed in triplicate for each condition.</p></section><section id="sec15"><h3 class="pmc_sec_title">Ex vivo skeletal muscle incubation for signaling and glucose uptake assays</h3>
<p>Extensor digitorum longus muscles were rapidly dissected and mounted in oxygenated (95% O<sub>2</sub> and 5% CO<sub>2</sub>), and warmed (30°C) Krebs Ringer buffer (KRB) containing 117 mM NaCl, 4.7 mM KCl, 2.5 mM CaCl<sub>2</sub>*2H<sub>2</sub>O, 1.2 mM KH<sub>2</sub>PO<sub>4</sub>, 1.2 mM MgSO<sub>4</sub>*7H<sub>2</sub>O, 24.6 mM NaHCO<sub>3</sub> (pH 7.5) supplemented with 2 mM pyruvate (Sigma-Aldrich, P-2256) in the presence of the BAY-3827 or vehicle (0.1% DMSO) for 30 min. The muscles were then coincubated for 50 min with BAY-3827 and ± 10 μM MK-8722 or 100 nM insulin (Novo Nordisk).The muscles were further incubated for 10 min in the presence of vehicle, compounds, or insulin in glucose uptake buffer composed of KRB buffer containing 1.5 μCi/ml [<sup>3</sup>H]-2-deoxy-<span class="font-variant-small-caps">d</span>-glucose (NET549005MC lot #2644198 2DG H3), 1 mM 2-deoxy-<span class="font-variant-small-caps">d</span>-glucose, 0.45 μCi/ml [<sup>14</sup>C]-mannitol (NEC852250UC lot #2741082 D-Mannitol C14), and 7 mM mannitol. At the end of incubation period, the muscles were frozen in liquid nitrogen and subsequently processed for glucose uptake [as described in (<a href="#R64" class="usa-link" aria-describedby="R64"><em>64</em></a>)] and Western blot analysis.</p></section><section id="sec16"><h3 class="pmc_sec_title">Kinase selectivity assays</h3>
<p>BAY-3827 and BAY-974 were assayed against a 140-human kinase panel at 0.1 μM and at 1 μM (BAY-3827) concentration in a [γ-<sup>33</sup>P] ATP–based radioactive filter binding assay (<a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>, <a href="#R65" class="usa-link" aria-describedby="R65"><em>65</em></a>) setup in duplicates with reported mean remaining kinase percentage activity alongside SD (table S1). Assays were performed at either 5, 20, or 50 μM ATP to be at or below the Michaelis constant for ATP for each enzyme and conducted by the services offered by the International Centre for Kinase Profiling at MRC Protein Phosphorylation and Ubiquitylation Unit, University of Dundee (<a href="https://www.kinase-screen.mrc.ac.uk/services/premier-screen" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.kinase-screen.mrc.ac.uk/services/premier-screen</a>).</p></section><section id="sec17"><h3 class="pmc_sec_title">Protein production and kinase assays</h3>
<p>Heterotrimeric human AMPK FLAG- α1β1γ1, FLAG- α2β1γ1, and FLAG- α2β2γ1, as well as human FLAG-CaMKK2 and FLAG-ULK1, were produced in mammalian HEK293 cells as described (<a href="#R66" class="usa-link" aria-describedby="R66"><em>66</em></a>, <a href="#R67" class="usa-link" aria-describedby="R67"><em>67</em></a>). For AMPK expression, the cells were triply transfected at 60% confluency using FuGene HD (Roche Applied Science) and 1 μg of pcDNA3 plasmid expression constructs for AMPK Flag-α1, β1-Myc, β2-Myc, and HA-γ1. For CaMKK2 expression, the cells were transfected with 1 μg of pcDNA3 Flag-CaMKK2 plasmid. For ULK1 expression, the cells were transfected with 2 mg of pcDNA3 FLAG-ULK1 plasmid. After 48 hours, the transfected cells were harvested by rinsing with ice-cold PBS, followed by rapid lysis using 500 μl of lysis buffer. AMPK, CaMKK2, and ULK1 kinase activities were determined by phosphorylation of synthetic peptide substrates: SAMS (HMRSAMSGLHLVKRR), CaMKKtide (LSNLYHQGKFLQTFCGAPLYRRR), and S108tide. (KLPLTRSHNNFVARRR), respectively. Briefly, recombinant AMPK, CaMKK2, or ULK1 were immunoprecipitated from 10 μg of transfected cultured cell lysate using 10 μl of anti-FLAG M2 agarose beads [50% (v/v)] (Merck MilliporeSigma), washed, and then added to a 25-μl reaction containing assay buffer [50 mM Hepes-NaOH (pH 7.4), 1 mM DTT, and 0.02% (v/v) Brij-35], 200 μM synthetic peptide substrate (SAMS, CaMKKtide, or S108tide), 200 or 20 μM [γ-<sup>32</sup>P]-ATP (PerkinElmer), 5 mM MgCl<sub>2</sub>, in the presence of 0 to 1 μM BAY-3827 or BAY-974. Reactions were performed at 30°C and terminated after 10 min by spotting 15 μl onto phosphocellulose paper. Radioactivity was quantified by liquid scintillation counting. Data visualization and nonlinear regression fitting were performed in GraphPad Prism v10.</p></section><section id="sec18"><h3 class="pmc_sec_title">Sequence conservation of BAY-3827 inhibited and noninhibited kinases</h3>
<p>Human KD sequences of AMPKα1, AMPKα2, RSK1, RSK2, MSK1, NUAK1, MELK, BRSK2, ABL1, PIM3, INSR, IRAK4, WNK1, PIM2, IKKβ, CDK2, TTBK1, and TAK1 were retrieved from KinBase (<a href="http://kinase.com/web/current/kinbase" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://kinase.com/web/current/kinbase</a>) (<a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>). For RSK1, RSK2, and MSK1 kinases, the two KDs were included in the analysis. A multiple sequence alignment of BAY-3827–inhibited and noninhibited kinase groups was performed in Jalview (<a href="#R68" class="usa-link" aria-describedby="R68"><em>68</em></a>) containing both AMPKα1 and AMPKα2, aligned using Tcoffee with default settings. Pairwise alignment scores in reference to AMPKα1 were calculated (<a href="#R68" class="usa-link" aria-describedby="R68"><em>68</em></a>) with respective kinases in each group (table S2). FASTA alignment files were inputted into ConSurf web server (<a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>, <a href="#R41" class="usa-link" aria-describedby="R41"><em>41</em></a>), and the template Protein Data Bank (PDB): 7MYJ (chain A) was selected, with sequence reported in AMPKα2 numbering. Running parameters included phylogenetic tree construction using Neighbor Joining with ML distance, conservation score calculation using the Bayesian method and with amino acid substitution model chosen by best fit. For the BAY-3827–inhibited kinase group, the average pairwise distance was 1.32, whereas for the noninhibited group, it was 1.82. The ConSurf color scale (9-conserved; 1-variable) was assigned on the basis of calculated conservation scores and computed confidence intervals. When confidence interval scores &gt;3 or the MSA number &lt;6, this was flagged as insufficient data with an asterisk (*) to signify a low confidence score. The difference in conservation between kinase groups was calculated, omitting low confidence scores and focusing on conservation score changes at 3 or above (table S2).</p></section><section id="sec19"><h3 class="pmc_sec_title">HEK293 cell culture and RSK signaling experiments</h3>
<p>HEK293 cells (Invitrogen, <a href="https://www.ncbi.nlm.nih.gov/nuccore/R75007" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">R75007</a>) were seeded at a density of 500,000 cells per well in six-well cell culture plates (Corning Incorporated Costar #3516) in Dulbecco’s modified Eagle medium (DMEM) high-glucose + GlutaMAX Supplement, pyruvate (Thermo Fisher Scientific, catalog no. 31966-021) containing penicillin G (100 U/ml), streptomycin (100 μg/ml), and 10% (v/v) FBS. Once attached, the next day, the media were changed to serum-free and the cells were starved for 16 hours followed by compound treatment with BAY-3827, BI-D1870 (MedChemExpress, #HY-10510) ± 10 μM MK-2206 dihydrochloride (MedChemExpress, #HY-10358) with 30 min incubation at 37°C followed by coincubation with 0.1 PMA μM (MedChemExpress, #HY-18739) or vehicle for additional 30 min with final assay DMSO concentration of 0.13%. All cell lines used here were tested monthly for mycoplasma and tested negative.</p></section><section id="sec20"><h3 class="pmc_sec_title">Western blotting</h3>
<p>Following compound treatment, the cells were washed with room temperature PBS and lysed on ice with the addition of lysis buffer [50 mM tris-HCl (pH 7.5), 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 0.27 M sucrose, 1% w/v Triton X-100, 20 mM glycerol-2 phosphate disodium, 50 mM NaF, and 5 mM Na<sub>4</sub>P<sub>2</sub>O<sub>7</sub>.10 H<sub>2</sub>O] containing proteases and phosphatase inhibitors [0.5 mM phenylmethylsulfonyl fluoride (PMSF), 1 mM benzamidine HCl, leupeptin (1 μg/ml), pepstatin A (1 μg/ml), 1 μM microcystin-LR, 1 mM Na<sub>3</sub>VO<sub>4</sub>, and 1 mM DTT]. Lysates were centrifuged at 10,000<em>g</em>, 4°C for 10 min, and protein concentration of the supernatant was determined with the Pierce Bradford Protein Assay Kit (Thermo Fisher Scientific, #23200) and read on the Hidex Plate Reader (software version 0.5.64.0). Samples were prepared in Laemmli sample buffer (tris 200 mM, 8% SDS, EDTA 2 mM, and 40% glycerol) supplemented with 10% DTT and boiled for 5 min at 100°C. Ten micrograms of protein per sample was run on a Criterion tris-glycine polyacrylamide gel (Bio-Rad, #5678095) and transferred to a nitrocellulose membrane. For cell signaling experiments in HEK293 cells, the samples were run on a Criterion TGX stain-free precast gel (Bio-Rad, catalog no. 5678085) with transfer to a polyvinylidene difluoride membrane performed in Bio-Rad Criterion blotter (#1704071) using tris-glycine transfer buffer (Bio-Rad, #1610734) following the manufacturer’s instructions. The membranes were probed at 4°C overnight in a 4% BSA TBS-T solution containing 0.02% sodium azide with the following antibodies where appropriate: Phospho-ACC (Ser<sup>79</sup>) (1:1000, Cell Signaling Technology, #3661S), ACC (C83B10) (1:1000, Cell Signaling Technology, #3676S), phospho-Raptor (Ser<sup>792</sup>) (1:1000, Cell Signaling Technology, #2083S), Raptor (24C12) (1:1000, Cell Signaling Technology, #2280S), GSK3αβ (1:2000, Sigma-Aldrich, #04-903), phospho-GSK3αβ (Ser<sup>21/9</sup>) (D17D2) (1:1000, Cell Signaling Technology, #8566S), ERK1/2 (137F5) (1:1000, Cell Signaling Technology, #4695), phospho-ERK1/2 (Thr<sup>202</sup>/Tyr<sup>204</sup>) (D13.14.4E) (1:1000, Cell Signaling Technology, #4370), phospho-AMPKα (Thr<sup>172</sup>) (40H9) (1:1000, Cell Signaling Technology, #2535S), AMPKα (1:1000, Cell Signaling Technology, #2532S), anti–glyceraldehyde-3-phosphate dehydrogenase antibody (1:5000, Cell Signaling Technology, #G8795), and vinculin (1:1000, Cell Signaling Technology, #13901). Membranes were scanned using the Odyssey Fc LiCor system (exposure time, 30 s) with densitometry analysis performed using LiCor Image Studio Lite normalizing to control loading protein. Figures were made using GraphPad Prism v10.</p>
<p>For adipocyte experiments, cell lysates (25 μg protein) were heated at 95°C for 3 min in LDS sample buffer, subjected to polyacrylamide gel electrophoresis on precast Novex gradient gels (Thermo Fisher Scientific) and electrotransferred to nitrocellulose membranes. Membranes were blocked for 30 min in 50 mM tris-HCl (pH 7.6), 137 mM NaCl, and 0.1% (w/v) Tween 20 (TBS-T) containing 10% (w/v) skimmed milk and then probed with the indicated antibodies in TBS-T containing 5% (w/v) BSA for 16 hours at 4°C. The following antibodies were used: anti-AMPK (#2603, 1:1000), anti–AMPK-pT172 (#2535, 1:1000), anti-Raptor (#2280, 1:1000), anti–Raptor-pS792 (#2083, 1:1000), anti-ACC1 (#3662, 1:1000), anti–ACC1-pS79 (#3661, 1:1000), anti-PKB/Akt (#9272, 1:1000), anti–PKB/Akt-pT308 (#9275, 1:1000), and anti-GSK3α/β-pS21/9 (#9331, 1:1000), which were all purchased from Cell Signaling Technology (Danvers, USA). Anti-AS160 (#07-741, 1:1000) was from Merck MilliporeSigma, anti–AS160-pT649 (#44-1071G, 1:1000) from Thermo Fisher Scientific, and anti-HSP90 from BD Biosciences (San Jose, CA, United States). Detection of primary antibodies was performed using horseradish peroxidase–conjugated secondary antibodies, and SuperSignal West Pico or Femto Chemiluminescent Substrates (Thermo Fisher Scientific). Luminescence signals were visualized in a ChemiDoc XRS+ (Bio-Rad) and quantified by densitometry using the software Image LabTM 5.1 (Bio-Rad). Signals were normalized to the loading control HSP90.</p></section><section id="sec21"><h3 class="pmc_sec_title">HTRF assays</h3>
<p>Human osteosarcoma U2OS WT cells were obtained from John Rouse (MRC Protein Phosphorylation and Ubiquitylation Unit, University of Dundee) and were cultured in DMEM high glucose + GlutaMAX Supplement, pyruvate (Thermo Fisher Scientific, catalog no. 31966-021) containing penicillin G (100 U/ml), streptomycin (100 μg/ml), and 10% (v/v) FBS seeding with 50,000 cells per well in a 50-μl volume in culture 96-well cell culture plates (Corning Incorporated costar, #3599). The next day, the media were changed to 50 μl of serum-free media, and 50 μl of serum-free media containing BAY-3827 or BAY-974 was dispensed and left to incubate at 37°C for 30 min. The media were aspirated and replaced with 50 μl of serum-free media containing either MK-8722 (10 μM) or vehicle and incubated at 37°C for 1 hour, with a final DMSO assay concentration of 1.03%. HTRF assays using either phospho-Ser<sup>79</sup>-ACC1 kit (Revvity, #64ACCPEG) or phospho-Thr<sup>172</sup>-AMPK kit (Revvity, #64AMPKPEG) were performed in triplicate in HTRF 96-well low volume white plates (Revvity, #66PL96001) following manufacturer’s instructions. Sealed plates incubated overnight in darkness conditions were scanned at room temperature in Multimode Plate Reader EnVision (PerkinElmer) machine following HTRF setup recommendations for EnVision (Revvity), equipped with a top mirror LANCE/DELFIA single mirror (#412), a UV2 (TRF) 320 excitation filter (#111), emission filter APC 665 (#205), and second emission filter Europium 615 (#203), with a 2000-μs cycle and 60-μs delay, with the number of flashes set to 100 as well as for the 2nd detector. Results were viewed in EnVision Manager software (version 1.13.3009.1401) where 665/615 ratios were calculated, and data were normalized to control condition as %. All assay ratios between control lysate signal/nonspecific signal were greater than 2 as expected of a successful assay following the manufacturer’s instructions. Data were visualized in GraphPad Prism v10.</p></section><section id="sec22"><h3 class="pmc_sec_title">Primary hepatocyte RNA isolation, sequencing, and bioinformatics analysis</h3>
<p>Primary hepatocytes were pretreated for 15 min with BAY-3827 5 μM at 37°C in MEM-199 supplemented with penicillin G (100 U/ml), streptomycin (100 mg/ml), and 100 nM dexamethasone followed by addition of MK-8722 10 μM treatment. Six hours posttreatment, the cells were washed twice with room-temperature PBS. RNA was isolated using QIAwave RNAeasy kit (catalog no. 74104) with on-column deoxyribonuclease (DNAse) treatment using Qiagen RNase-free DNase set (catalog no. 79256) following the manufacturer’s protocol. Messenger RNA-seq was performed by the Single-Cell Omics platform at the Novo Nordisk Foundation Center for Basic Metabolic Research. Libraries were prepared using the Universal Plus mRNA-seq protocol (Tecan) as recommended by the manufacturer. Libraries were quantified with NuQuant using the CLARIOstar Plate Reader (BMG Labtech), quality checked using a TapeStation instrument (Agilent Technologies), and subjected to 52–base pair paired-end sequencing on a NovaSeq 6000 (Illumina). The sequencing analysis pipeline nf-core (<a href="#R69" class="usa-link" aria-describedby="R69"><em>69</em></a>) was used, with alignment and quantification selection of STAR and Salmon, and mouse GRCm38 set as the reference genome. Unique molecular identifier (UMI)-based deduplication was performed using UMI tools, and sorting and index alignments were completed with SAMtools as part of the pipeline. Differential expression analysis was carried out using the R package edgeR (<a href="#R70" class="usa-link" aria-describedby="R70"><em>70</em></a>) and a batch-effect removal was performed in limma (<a href="#R71" class="usa-link" aria-describedby="R71"><em>71</em></a>) to account for the variability induced by two mice (291 + 295) from which cells with the same treatments had been pooled to obtain the required cell confluence. Venn diagrams were produced using the R package ggvenn (<a href="#R72" class="usa-link" aria-describedby="R72"><em>72</em></a>). Volcano plots were produced using the R package ggplot2, and significant up-regulated or down-regulated genes were described as having an FC ≥1.3 and FDR &lt; 0.05. Heatmaps of log<sub>2</sub> counts per million normalized counts were produced using R package pheatmap (<a href="#R73" class="usa-link" aria-describedby="R73"><em>73</em></a>). GO enrichment analysis was performed using the R package clusterProfiler version 4.6.2 (<a href="#R74" class="usa-link" aria-describedby="R74"><em>74</em></a>) setting FDR &lt; 0.05 with reference organism specified as mouse, pAdjustMethod was set to BH, and ontology category set to biological process, with a qvalueCutoff of 0.05. Visualization analyses were conducted in RStudio version 4.2.0 (<a href="#R75" class="usa-link" aria-describedby="R75"><em>75</em></a>).</p></section><section id="sec23"><h3 class="pmc_sec_title">Plasmid preparation for AMPK construction</h3>
<p>The coding sequence for the human AMPK-α2 KD (α2KD; residues 6 to 280) and its phosphomimetic mutant T172D-α2KD were custom synthesized (GENEWIZ, Azenta Life Sciences) with codon-optimized for <em>Escherichia coli</em> expression and cloned into the pET-14b vector with an N-terminal cleavable 6 × His-AviTag. For expression of full-length AMPKα2β2γ1, α2β2γ2, and α2β2γ3 complexes, polycistronic constructs encoding the three subunits were assembled into the pET-30b vector with a TEV-cleavable 6 × His–glutathione <em>S</em>-transferase (GST) tag on the γ subunit. Constructs were transformed into <em>E. coli</em> BL21(DE3)-RIL cells (Agilent) and cultured in LB medium at 37°C to an optical density at 600 nm of 0.7. Protein expression was induced with 0.2 μM isopropyl-β-<span class="font-variant-small-caps">d</span>-thiogalactopyranoside, followed by overnight incubation at 18°C with shaking at 200 rpm. The cells were harvested by centrifugation, resuspended in lysis buffer [25 mM tris-HCl (pH 8.0), 500 mM NaCl, 10% (v/v) glycerol, 20 mM imidazole, 2 mM TCEP, 5 mM MgCl<sub>2</sub>, 0.2 mM PMSF, 1 mM benzamidine, and lysozyme (0.3 mg/ml)], and lysed by sonication (10 min total, 20 s on/40 s off) on ice. Lysates were clarified by centrifugation at 30,000<em>g</em> for 30 min at 4°C and filtered through a 0.45-μm syringe filter before loading onto the His-Trap column.</p></section><section id="sec24"><h3 class="pmc_sec_title">Purification of AMPKα2 and T172D-α2 KDs</h3>
<p>Clarified lysates were loaded onto a 5-ml HisTrap HP column (Cytiva) preequilibrated with lysis buffer. The column was washed with 10 column volumes (CV) of wash buffer containing tris 25 mM, NaCl 500 mM, glycerol 10%, imidazole 20 mM, TCEP 2 mM, MgCl<sub>2</sub> 5 mM (pH 8.0). Proteins were eluted with a linear imidazole gradient (20 to 300 mM) more than 35 CV using an ÄKTA system. For crystallization, α2KD pooled fractions were incubated with TEV protease at a 1:50 molar ratio and dialyzed overnight at 4°C in buffer containing 25 mM tris-HCl (pH 8.0), 200 mM NaCl, 10% glycerol, 2 mM β-mercaptoethanol, and 5 mM MgCl<sub>2</sub>. This step of dialysis and TEV-cleavage was skipped for protein preparation that was used for Dianthus assays. Eluted fractions were pooled and further purified by size-exclusion chromatography on a Superdex S-75 16/600 column (Cytiva) equilibrated in 25 mM tris-HCl (pH 8.0), 200 mM NaCl, 10% glycerol, and 1 mM TCEP. The pure fractions of the proteins were concentrated to 5 mg/ml (α2KD) and 2 mg/ml (T172D-α2KD), and flash frozen for storage for further use.</p></section><section id="sec25"><h3 class="pmc_sec_title">Purification of full-length AMPK complexes</h3>
<p>Clarified lysates containing AMPK complexes were incubated with 5 ml of Ni–nitrilotriacetic acid (NTA) agarose resin (Qiagen) for 2 hours at 4°C. The resin was washed ≥six times with wash buffer and eluted with 300 mM imidazole-containing buffer. Eluted proteins were incubated with 5 ml of GST Sepharose (Cytiva), washed, and eluted in buffer containing 20 mM reduced glutathione.</p>
<p>Final purification was performed by gel filtration chromatography using a Superdex-S200 16/600 column (Cytiva) equilibrated in 25 mM Hepes (pH 7.5), 200 mM NaCl, 5% glycerol, and 1 mM TCEP. Peak fractions were concentrated using 30-kDa MWCO centrifugal concentrators (Millipore) with final concentration obtained at &lt;1 mg/ml and flash-frozen in liquid nitrogen for storage at −80°C.</p></section><section id="sec26"><h3 class="pmc_sec_title">Crystallization and structure determination</h3>
<p>The crystallization screening was set up with various commercially available crystal screens with a protein concentration of 5 mg/ml in a sitting-drop vapor diffusion setup. Four hundred–nanoliter drops were setup with 1:1 ratio of protein to reservoir solution in a 96-well two-drop MRC plates. For α2KD–BAY-3827 cocrystallization, the protein at 0.5 mg/ml was mixed with the BAY-3827 compound at a final concentration of 0.1 mM, which was then concentrated to 5 mg/ml. The α2KD–BAY-3827 cocrystals were grown in a condition containing 0.1 M ammonium sulphate, 0.3 M sodium formate, 0.1 M sodium cacodylate (pH 6.5), 3% (w/v) γ-PGA, 3% (w/v) polyethylene glycol, molecular weight 20,000. The crystals were harvested and flash frozen in liquid nitrogen. Diffraction data were collected at 100 K at I24 beamline, Diamond light source. The data were processed by autoPROC (<a href="#R76" class="usa-link" aria-describedby="R76"><em>76</em></a>) in space group <a href="https://www.ncbi.nlm.nih.gov/protein/P21212" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P21212</a>. The structure of α2KD-BAY-3827 was solved by molecular replacement using the program PHASER (<a href="#R77" class="usa-link" aria-describedby="R77"><em>77</em></a>) with a search model 2H6D (Apo-α2KD structure). The structure was built in coot (<a href="#R78" class="usa-link" aria-describedby="R78"><em>78</em></a>) and further refined in phenix-refine (<a href="#R79" class="usa-link" aria-describedby="R79"><em>79</em></a>). The figures were generated in PyMOL (<a href="#R80" class="usa-link" aria-describedby="R80"><em>80</em></a>).</p></section><section id="sec27"><h3 class="pmc_sec_title">Dianthus binding assays</h3>
<p>The binding affinity of BAY-3827 to α2KD, AMPK (T172D)-α2KD, or AMPK complexes α2β2γ1 and α2β2γ3 was measured by Dianthus (Nano Temper Technologies). BAY-3827 was diluted from a 100 mM stock in DMSO to 0.5 mM working concentration in the assay buffer (PBS with 0.005% Tween-20) with a final DMSO concentration of 2.5%, which was kept constant throughout the concentration gradient of the compound. For additional experiments, assay buffer containing Hepes 25 mM, NaCl 250 mM, and 0.005% Tween-20 was used to optimize the assay, enhancing signal-to-noise ratios. His-tagged α2KD, T172D-α2KD, or His-tagged AMPK-α2β2γ1 or α2β2γ3 complexes were labeled with His-tag labeling dye (His-Tag Labeling Kit RED-tris-NTA 2nd Generation). The ratio of fluorescence intensities (670/650 nm) over BAY-3827 concentration gradient ranging from 250 μM down to 0.015 μM. The concentration gradient was prepared by 1:1 serial dilution over 16 steps, with a setup of two to three technical replicates for each concentration. The binding experiments were repeated three times, and the mean <em>K</em><sub>d</sub> values of the inhibitor are reported in the appropriate figure alongside ± SEM (micromolar) displaying a representative graph. A concentration of 50 nM of the corresponding protein target was used in each performed assay. The ratio of fluorescence intensities was measured over BAY-3827 concentration gradient in GraphPad Prism using nonlinear regression (one site-total binding). Data quality was assessed by monitoring XYZ scans for each well to detect air bubbles or aggregation where present following the manufacturer’s instructions.</p></section><section id="sec28"><h3 class="pmc_sec_title">Statistical analyses</h3>
<p>Data points are plotted as means ± SEM. Data were inputted, and all statistical tests were performed using GraphPad Prism, version 10.1.0. One- or two-way analysis of variance (ANOVA) statistical tests were conducted using Tukey’s correction for multiple comparisons; specific test will be stated in the figure legend. Significance was accepted as <em>P</em> &lt; 0.05. <em>N</em> number and statistical details for each experiment are reported in each figure legend. For lipogenesis generated data, a one-way Welch and Brown-Forsythe ANOVA test was conducted to assess statistical significance, which does not assume that all the groups were sampled from populations with equal variances (skew in data), with Sunnett T3 multiple comparisons correction. To estimate IC<sub>50</sub> values, nonlinear regression fitting was conducted using the model (Inhibitor) versus response—variable slope in GraphPad Prism (version 10.1.0) and estimating values reported in nanomolar. To estimate binding affinities (<em>K</em><sub>d</sub>) from performed spectral shift binding assays, data were fitted using a one-site total model [Y = <em>B</em><sub>max</sub><em>X/(K</em><sub>d</sub> <em>+ X) + NS</em>X + Background] where <em>X</em> is the ligand concentration, <em>B</em><sub>max</sub> represents the maximum binding, and NS is the slope of the nonlinear regression of <em>Y</em>/<em>X</em>. Values are reported in means ± SEM in micromolar.</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank the Scientific Genome Consortium (SGC)–Frankfurt for donation of chemical probe BAY-974. We acknowledge L. R. Ingerslev, C. Grønbæk, M. Mechta, and the staff at the Single-Cell Omics platform at the Novo Nordisk Foundation Center for Basic Metabolic Research (CBMR) for technical and computational expertise and support. We acknowledge T. M. Frimurer and M. Lückmann at the Computational Chemistry Unit at CBMR for computational expertise and training support in virtual docking simulations. We acknowledge assistance and support from computational resources and infrastructure via the Health Data Science Sandbox (<a href="https://hds-sandbox.github.io" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hds-sandbox.github.io</a>), funded by the Novo Nordisk Foundation grant NNF20OC0063268. We thank the Diamond Light Source for beamtime (proposals MX29074 and MX35120), and the staff of beamlines I04 and I24 for assistance with crystal testing and data collection.</p>
<section id="sec29"><p><strong>Funding:</strong> This work was financially supported by the Novo Nordisk Foundation grant NNF18CC0034900 and NNF23SA0084103 (K.S.), Wellcome Trust Senior Fellowship 222531/Z/21/Z (E.Z.), National Health and Medical Research Council of Australia Ideas grant 2001817 (J.W.S.), Discovery Grant 210102840 from the Australian Research Council (J.W.S.), Swedish Foundation for Strategic Research Dnr IRC15-0067 (O.G. and M.L.), the Swedish Diabetes Foundation, and the Påhlsson Foundation (O.G. and M.L.). The Copenhagen Bioscience PhD program is funded by Novo Nordisk Foundation grant NNF20SA003558 (C.F.B.). This work was further supported by Investigator Grants (APP1194141, APP2040938) from the National Health &amp; Medical Research Council (NHMRC) of Australia (M.A.F.).</p></section><section id="sec30"><p><strong>Author contributions:</strong> Conceptualization: K.S., E.Z., C.F.B., and M.S.A. Experimental design: K.S., E.Z., C.F.B., M.S.A., J.W.S., and O.G. Investigation: C.F.B., M.S.A., J.W.S., M.L., J.C., A.B.A., H.L., and A.J.O. Data analysis: C.F.B., M.S.A., J.C., M.L., A.B.A., O.G., J.W.S., E.Z., and K.S. Research model development and establishment: M.F. Visualization: C.F.B. and M.S.A. Project administration: C.F.B. and K.S. Supervision: K.S., E.Z., O.G., and M.A.F. Writing—original draft: C.F.B. and K.S. Writing—review and editing: All authors.</p></section><section id="sec31"><p><strong>Competing interests:</strong> M.A.F. is a scientific advisor for Vitaleon Pharma and is the founder and shareholder of Celesta Therapeutics. All other authors declare that they have no competing interests.</p></section><section id="sec32"><p><strong>Data and materials availability:</strong> All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials. RNA-seq data pertaining to this project have been deposited in the Gene Expression Omnibus (GEO) repository under accession <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE290022" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE290022</a>. All code related to this project is available in Zenodo repository (<a href="https://doi.org/10.5281/zenodo.15854039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15854039</a>) and GitHub (<a href="https://github.com/cfraguas/BAY-code-notebooks" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/cfraguas/BAY-code-notebooks</a>). X-ray crystallography data have been deposited in the Worldwide Protein Data Bank (PDB) under accession 9IC2. AMPK-related cDNA constructs can be provided by E.Z. pending scientific review and a completed material transfer agreement. Requests for the AMPK cDNA constructs should be submitted to E.Z. (<span>e.zeqiraj@leeds.ac.uk</span>). The AMPKα1/α2 floxed ± Alfp-Cre mice can be provided by M.F. pending scientific review and a completed material transfer agreement. Requests for the AMPKα1/α2 floxed ± Alfp-Cre mice should be submitted to M.F. (<span>marc.foretz@inserm.fr</span>).</p></section></section><section id="sec33"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section id="sec34"><h3 class="pmc_sec_title">This PDF file includes:</h3>
<section class="sm xbox font-sm" id="supplementary-material1"><div class="caption p">
<p>Figs. S1 to S5</p>
<p>Tables S1 to S3</p>
<p>References</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372887/bin/sciadv.adx2434_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adx2434_sm.pdf</a><sup> (15.8MB, pdf) </sup>
</div></div></section></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES AND NOTES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Hardie D. G., Ross F. A., Hawley S. A., 
AMPK: A nutrient and energy sensor that maintains energy homeostasis. Nat. Rev. Mol. Cell Biol.
13, 
251–262 (2012).
</cite> [<a href="https://doi.org/10.1038/nrm3311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5726489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22436748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;title=AMPK:%20A%20nutrient%20and%20energy%20sensor%20that%20maintains%20energy%20homeostasis&amp;volume=13&amp;publication_year=2012&amp;pages=251-262&amp;pmid=22436748&amp;doi=10.1038/nrm3311&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Steinberg G. R., Hardie D. G., 
New insights into activation and function of the AMPK. Nat. Rev. Mol. Cell Biol.
24, 
255–272 (2023).
</cite> [<a href="https://doi.org/10.1038/s41580-022-00547-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36316383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;title=New%20insights%20into%20activation%20and%20function%20of%20the%20AMPK&amp;volume=24&amp;publication_year=2023&amp;pages=255-272&amp;pmid=36316383&amp;doi=10.1038/s41580-022-00547-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Hardie D. G., Sakamoto K., 
AMPK: A key sensor of fuel and energy status in skeletal muscle. Physiology
21, 
48–60 (2006).
</cite> [<a href="https://doi.org/10.1152/physiol.00044.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16443822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiology&amp;title=AMPK:%20A%20key%20sensor%20of%20fuel%20and%20energy%20status%20in%20skeletal%20muscle&amp;volume=21&amp;publication_year=2006&amp;pages=48-60&amp;pmid=16443822&amp;doi=10.1152/physiol.00044.2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Ross F. A., MacKintosh C., Hardie D. G., 
AMP-activated protein kinase: A cellular energy sensor that comes in 12 flavours. FEBS J.
283, 
2987–3001 (2016).
</cite> [<a href="https://doi.org/10.1111/febs.13698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4995730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26934201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20J.&amp;title=AMP-activated%20protein%20kinase:%20A%20cellular%20energy%20sensor%20that%20comes%20in%2012%20flavours&amp;volume=283&amp;publication_year=2016&amp;pages=2987-3001&amp;pmid=26934201&amp;doi=10.1111/febs.13698&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Smiles W. J., Ovens A. J., Oakhil J. S., Kofler B., 
The metabolic sensor AMPK: Twelve enzymes in one. <em>Molecular</em>. Metabolism
90, 
102042 (2024).</cite> [<a href="https://doi.org/10.1016/j.molmet.2024.102042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11752127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39362600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Metabolism&amp;title=The%20metabolic%20sensor%20AMPK:%20Twelve%20enzymes%20in%20one.%20Molecular&amp;volume=90&amp;publication_year=2024&amp;pages=102042&amp;pmid=39362600&amp;doi=10.1016/j.molmet.2024.102042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Xiao B., Sanders M. J., Underwood E., Heath R., Mayer F. V., Carmena D., Jing C., Walker P. A., Eccleston J. F., Haire L. F., Saiu P., Howell S., Aasland R., Martin S., Carling D., Gamblin S., 
Structure of mammalian AMPK and its regulation by ADP. Nature
472, 
230–233 (2011).
</cite> [<a href="https://doi.org/10.1038/nature09932" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3078618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21399626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure%20of%20mammalian%20AMPK%20and%20its%20regulation%20by%20ADP&amp;volume=472&amp;publication_year=2011&amp;pages=230-233&amp;pmid=21399626&amp;doi=10.1038/nature09932&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Langendorf C. G., Kemp B. E., 
Choreography of AMPK activation. Cell Res.
25, 
5–6 (2015).
</cite> [<a href="https://doi.org/10.1038/cr.2014.163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4650591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25475061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Res.&amp;title=Choreography%20of%20AMPK%20activation&amp;volume=25&amp;publication_year=2015&amp;pages=5-6&amp;pmid=25475061&amp;doi=10.1038/cr.2014.163&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Hawley S. A., Russell F. M., Hardie D. G., 
AMP-activated protein kinase can be allosterically activated by ADP but AMP remains the key activating ligand. Biochem. J.
8, 
587–599 (2024).</cite> [<a href="https://doi.org/10.1042/BCJ20240082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11088877/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38592738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=AMP-activated%20protein%20kinase%20can%20be%20allosterically%20activated%20by%20ADP%20but%20AMP%20remains%20the%20key%20activating%20ligand&amp;volume=8&amp;publication_year=2024&amp;pages=587-599&amp;pmid=38592738&amp;doi=10.1042/BCJ20240082&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Steinberg G. R., Carling D., 
AMP-activated protein kinase: The current landscape for drug development. Nat. Rev. Drug Discov.
18, 
527–551 (2019).
</cite> [<a href="https://doi.org/10.1038/s41573-019-0019-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30867601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discov.&amp;title=AMP-activated%20protein%20kinase:%20The%20current%20landscape%20for%20drug%20development&amp;volume=18&amp;publication_year=2019&amp;pages=527-551&amp;pmid=30867601&amp;doi=10.1038/s41573-019-0019-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Carling D., Zammit V. A., Hardie D. G., 
A common bicyclic protein kinase cascade inactivates the regulatory enzymes of fatty acid and cholesterol biosynthesis. FEBS Lett.
223, 
217–222 (1987).
</cite> [<a href="https://doi.org/10.1016/0014-5793(87)80292-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2889619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&amp;title=A%20common%20bicyclic%20protein%20kinase%20cascade%20inactivates%20the%20regulatory%20enzymes%20of%20fatty%20acid%20and%20cholesterol%20biosynthesis&amp;volume=223&amp;publication_year=1987&amp;pages=217-222&amp;pmid=2889619&amp;doi=10.1016/0014-5793(87)80292-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Hardie D. G., 
AMP-activated protein kinase — A journey from 1 to 100 downstream targets. Biochem. J.
479, 
2327–2343 (2022).
</cite> [<a href="https://doi.org/10.1042/BCJ20220255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9704532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36383046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=AMP-activated%20protein%20kinase%20%E2%80%94%20A%20journey%20from%201%20to%20100%20downstream%20targets&amp;volume=479&amp;publication_year=2022&amp;pages=2327-2343&amp;pmid=36383046&amp;doi=10.1042/BCJ20220255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Sakamoto K., Holman G. D., 
Emerging role for AS160/TBC1D4 and TBC1D1 in the regulation of GLUT4 traffic. Am. J. Physiol. Endocrinol. Metab.
295, 
E29–E37 (2008).
</cite> [<a href="https://doi.org/10.1152/ajpendo.90331.2008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2493596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18477703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.%20Endocrinol.%20Metab.&amp;title=Emerging%20role%20for%20AS160/TBC1D4%20and%20TBC1D1%20in%20the%20regulation%20of%20GLUT4%20traffic&amp;volume=295&amp;publication_year=2008&amp;pages=E29-E37&amp;pmid=18477703&amp;doi=10.1152/ajpendo.90331.2008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Chen Q., Xie B., Zhu S., Rong P., Sheng Y., Ducommun S., Chen L., Quan C., Li M., Sakamoto K., MacKintosh C., Chen S., Wang H., 
A Tbc1d1 Ser231Ala-knockin mutation partially impairs AICAR- but not exercise-induced muscle glucose uptake in mice. Diabetologia
60, 
336–345 (2017).
</cite> [<a href="https://doi.org/10.1007/s00125-016-4151-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27826658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=A%20Tbc1d1%20Ser231Ala-knockin%20mutation%20partially%20impairs%20AICAR-%20but%20not%20exercise-induced%20muscle%20glucose%20uptake%20in%20mice&amp;volume=60&amp;publication_year=2017&amp;pages=336-345&amp;pmid=27826658&amp;doi=10.1007/s00125-016-4151-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Kjøbsted R., Roll J. L. W., Jørgensen N. O., Birk J. B., Foretz M., Viollet B., Chadt A., Al-Hasani H., Wojtaszewski J. F. P., 
AMPK and TBC1D1 regulate muscle glucose uptake after, but not during, exercise and contraction. Diabetes
68, 
1427–1440 (2019).
</cite> [<a href="https://doi.org/10.2337/db19-0050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31010958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=AMPK%20and%20TBC1D1%20regulate%20muscle%20glucose%20uptake%20after,%20but%20not%20during,%20exercise%20and%20contraction&amp;volume=68&amp;publication_year=2019&amp;pages=1427-1440&amp;pmid=31010958&amp;doi=10.2337/db19-0050&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Cool B., Zinker B., Chiou W., Kifle L., Cao N., Perham M., Dickinson R., Adler A., Gagne G., Iyengar R., Zhao G., Marsh K., Kym P., Jung P., Camp H., Frevert E., 
Identification and characterization of a small molecule AMPK activator that treats key components of type 2 diabetes and the metabolic syndrome. Cell Metab.
3, 
403–416 (2006).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2006.05.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16753576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab.&amp;title=Identification%20and%20characterization%20of%20a%20small%20molecule%20AMPK%20activator%20that%20treats%20key%20components%20of%20type%202%20diabetes%20and%20the%20metabolic%20syndrome&amp;volume=3&amp;publication_year=2006&amp;pages=403-416&amp;pmid=16753576&amp;doi=10.1016/j.cmet.2006.05.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Göransson O., McBride A., Hawley S. A., Ross F. A., Shpiro N., Foretz M., Viollet B., Hardie D. G., Sakamoto K., 
Mechanism of action of A-769662, a valuable tool for activation of AMP-activated protein kinase. J. Biol. Chem.
282, 
32549–32560 (2007).
</cite> [<a href="https://doi.org/10.1074/jbc.M706536200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2156105/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17855357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Mechanism%20of%20action%20of%20A-769662,%20a%20valuable%20tool%20for%20activation%20of%20AMP-activated%20protein%20kinase&amp;volume=282&amp;publication_year=2007&amp;pages=32549-32560&amp;pmid=17855357&amp;doi=10.1074/jbc.M706536200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Xiao B., Sanders M. J., Carmena D., Bright N. J., Haire L. F., Underwood E., Patel B. R., Heath R. B., Walker P. A., Hallen S., Giordanetto F., Martin S., Carling D., Gamblin S., 
Structural basis of AMPK regulation by small molecule activators. Nat. Commun.
4, 
3017 (2013).
</cite> [<a href="https://doi.org/10.1038/ncomms4017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3905731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24352254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Structural%20basis%20of%20AMPK%20regulation%20by%20small%20molecule%20activators&amp;volume=4&amp;publication_year=2013&amp;pages=3017&amp;pmid=24352254&amp;doi=10.1038/ncomms4017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Myers R. W., Guan H. P., Ehrhart J., Petrov A., Prahalada S., Tozzo E., Yang X., Kurtz M. M., Trujillo M., Trotter D. G., Feng D., Xu S., Eiermann G., Holahan M. A., Rubins D., Conarello S., Niu X., Souza S. C., Miller C., Liu J., Lu K., Feng W., Li Y., Painter R., Milligan J., He H., Liu F., Ogawa A., Wisniewski D., Rohm R., Wang L., Bunzel M., Qian Y., Zhu W., Wang H., Bennet B., Scheuch L., Fernandez G., Li C., Klimas M., Zhou G., Van Heek M., Biftu T., Weber A., Kelley D., Thornberry N., Erion M., Kemp D., Sebhat I., 
Systemic pan-AMPK activator MK-8722 improves glucose homeostasis but induces cardiac hypertrophy. Science
357, 
507–511 (2017).
</cite> [<a href="https://doi.org/10.1126/science.aah5582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28705990/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Systemic%20pan-AMPK%20activator%20MK-8722%20improves%20glucose%20homeostasis%20but%20induces%20cardiac%20hypertrophy&amp;volume=357&amp;publication_year=2017&amp;pages=507-511&amp;pmid=28705990&amp;doi=10.1126/science.aah5582&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Cokorinos E. C., Delmore J., Reyes A. R., Albuquerque B., Kjøbsted R., Jørgensen N. O., Tran J. L., Jatkar A., Cialdea K., Esquejo R. M., Meissen J., Calabrese M. F., Cordes J., Moccia R., Tess D., Salatto C. T., Coskran T. M., Opsahl A. C., Flynn D., Blatnik M., Li W., Kindt E., Foretz M., Viollet B., Ward J., Kurumbail R., Kalgutkar A., Wojtaszewski J., Cameron K., Miller R., 
Activation of skeletal muscle AMPK promotes glucose disposal and glucose lowering in non-human primates and mice. Cell Metab.
25, 
1147–1159.e10 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2017.04.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28467931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab.&amp;title=Activation%20of%20skeletal%20muscle%20AMPK%20promotes%20glucose%20disposal%20and%20glucose%20lowering%20in%20non-human%20primates%20and%20mice&amp;volume=25&amp;publication_year=2017&amp;pages=1147-1159.e10&amp;pmid=28467931&amp;doi=10.1016/j.cmet.2017.04.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Strang J. E., Astridge D. D., Nguyen V. T., Reigan P., 
Small molecule modulators of AMP-activated protein kinase (AMPK) activity and their potential in cancer therapy. J. Med. Chem.
68, 
2238–2254 (2025).
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.4c02354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11831681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39879193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Small%20molecule%20modulators%20of%20AMP-activated%20protein%20kinase%20(AMPK)%20activity%20and%20their%20potential%20in%20cancer%20therapy&amp;volume=68&amp;publication_year=2025&amp;pages=2238-2254&amp;pmid=39879193&amp;doi=10.1021/acs.jmedchem.4c02354&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Vara-Ciruelos D., Russell F. M., Grahame Hardie D., 
The strange case of AMPK and cancer: Dr Jekyll or Mr Hyde?
Open Biol.
9, 
190099 (2019).
</cite> [<a href="https://doi.org/10.1098/rsob.190099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6685927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31288625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Open%20Biol.&amp;title=The%20strange%20case%20of%20AMPK%20and%20cancer:%20Dr%20Jekyll%20or%20Mr%20Hyde?&amp;volume=9&amp;publication_year=2019&amp;pages=190099&amp;pmid=31288625&amp;doi=10.1098/rsob.190099&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Alessi D. R., Sakamoto K., Bayascas J. R., 
LKB1-dependent signaling pathways. Annu. Rev. Biochem.
75, 
137–163 (2006).
</cite> [<a href="https://doi.org/10.1146/annurev.biochem.75.103004.142702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16756488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Biochem.&amp;title=LKB1-dependent%20signaling%20pathways&amp;volume=75&amp;publication_year=2006&amp;pages=137-163&amp;pmid=16756488&amp;doi=10.1146/annurev.biochem.75.103004.142702&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Shaw R. J., Bardeesy N., Manning B. D., Lopez L., Kosmatka M., DePinho R. A., Cantley L. C., 
The LKB1 tumor suppressor negatively regulates mTOR signaling. Cancer Cell
6, 
91–99 (2004).
</cite> [<a href="https://doi.org/10.1016/j.ccr.2004.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15261145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=The%20LKB1%20tumor%20suppressor%20negatively%20regulates%20mTOR%20signaling&amp;volume=6&amp;publication_year=2004&amp;pages=91-99&amp;pmid=15261145&amp;doi=10.1016/j.ccr.2004.06.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Huang X., Wullschleger S., Shpiro M., McGuire V. A., Sakamoto K., Woods Y. L., McBurnie W., Fleming S., Alessi D. R., 
Important role of the LKB1-AMPK pathway in suppressing tumorigenesis in PTEN-deficient mice. Biochem. J.
412, 
211–221 (2008).
</cite> [<a href="https://doi.org/10.1042/BJ20080557" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18387000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=Important%20role%20of%20the%20LKB1-AMPK%20pathway%20in%20suppressing%20tumorigenesis%20in%20PTEN-deficient%20mice&amp;volume=412&amp;publication_year=2008&amp;pages=211-221&amp;pmid=18387000&amp;doi=10.1042/BJ20080557&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Russell F. M., Hardie D. G., 
AMP-activated protein kinase: Do we need activators or inhibitors to treat or prevent cancer?
Int. J. Mol. Sci.
22, 
186 (2020).
</cite> [<a href="https://doi.org/10.3390/ijms22010186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7795930/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33375416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=AMP-activated%20protein%20kinase:%20Do%20we%20need%20activators%20or%20inhibitors%20to%20treat%20or%20prevent%20cancer?&amp;volume=22&amp;publication_year=2020&amp;pages=186&amp;pmid=33375416&amp;doi=10.3390/ijms22010186&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Monteverde T., Muthalagu N., Port J., Murphy D. J., 
Evidence of cancer-promoting roles for AMPK and related kinases. FEBS J.
282, 
4658–4671 (2015).
</cite> [<a href="https://doi.org/10.1111/febs.13534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26426570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20J.&amp;title=Evidence%20of%20cancer-promoting%20roles%20for%20AMPK%20and%20related%20kinases&amp;volume=282&amp;publication_year=2015&amp;pages=4658-4671&amp;pmid=26426570&amp;doi=10.1111/febs.13534&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Zhou G., Myers R., Li Y., Chen Y., Shen X., Fenyk-Melody J., Wu M., Ventre J., Doebber T., Fujii N., Musi N., Hirshman M. F., Goodyear L. J., Moller D. E., 
Role of AMP-activated protein kinase in mechanism of metformin action. J. Clin. Investig.
108, 
1167–1174 (2001).
</cite> [<a href="https://doi.org/10.1172/JCI13505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC209533/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11602624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Investig.&amp;title=Role%20of%20AMP-activated%20protein%20kinase%20in%20mechanism%20of%20metformin%20action&amp;volume=108&amp;publication_year=2001&amp;pages=1167-1174&amp;pmid=11602624&amp;doi=10.1172/JCI13505&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Yu P. B., Hong C. C., Sachidanandan C., Babitt J. L., Deng D. Y., Hoyng S. A., Lin H. Y., Bloch K. D., Peterson R. T., 
Dorsomorphin inhibits BMP signals required for embryogenesis and iron metabolism. Nat. Chem. Biol.
4, 
33–41 (2008).
</cite> [<a href="https://doi.org/10.1038/nchembio.2007.54" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2727650/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18026094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&amp;title=Dorsomorphin%20inhibits%20BMP%20signals%20required%20for%20embryogenesis%20and%20iron%20metabolism&amp;volume=4&amp;publication_year=2008&amp;pages=33-41&amp;pmid=18026094&amp;doi=10.1038/nchembio.2007.54&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Handa N., Takagi T., Saijo S., Kishishita S., Takaya D., Toyama M., Terada T., Shirouzu M., Suzuki A., Lee S., Yamauchi T., Okada-Iwabu M., Iwabu M., Kadowaki T., Minokoshi Y., Yokoyama S., 
Structural basis for compound C inhibition of the human AMP-activated protein kinase α2 subunit kinase domain. Acta Crystallogr. D Biol. Crystallogr.
67, 
480–487 (2011).
</cite> [<a href="https://doi.org/10.1107/S0907444911010201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21543851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.&amp;title=Structural%20basis%20for%20compound%20C%20inhibition%20of%20the%20human%20AMP-activated%20protein%20kinase%20%CE%B12%20subunit%20kinase%20domain&amp;volume=67&amp;publication_year=2011&amp;pages=480-487&amp;pmid=21543851&amp;doi=10.1107/S0907444911010201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Chaikuad A., Alfano I., Kerr G., Sanvitale C. E., Boergermann J. H., Triffitt J. T., Von Delft F., Knapp S., Knaus P., Bullock A. N., 
Structure of the bone morphogenetic protein receptor ALK2 and implications for fibrodysplasia ossificans progressiva. J. Biol. Chem.
287, 
36990–36998 (2012).
</cite> [<a href="https://doi.org/10.1074/jbc.M112.365932" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3481300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22977237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Structure%20of%20the%20bone%20morphogenetic%20protein%20receptor%20ALK2%20and%20implications%20for%20fibrodysplasia%20ossificans%20progressiva&amp;volume=287&amp;publication_year=2012&amp;pages=36990-36998&amp;pmid=22977237&amp;doi=10.1074/jbc.M112.365932&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Bain J., Plater L., Elliott M., Shpiro N., Hastie C. J., Mclauchlan H., Klevernic I., Arthur J. S. C., Alessi D. R., Cohen P., 
The selectivity of protein kinase inhibitors: A further update. Biochem. J.
408, 
297–315 (2007).
</cite> [<a href="https://doi.org/10.1042/BJ20070797" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2267365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17850214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=The%20selectivity%20of%20protein%20kinase%20inhibitors:%20A%20further%20update&amp;volume=408&amp;publication_year=2007&amp;pages=297-315&amp;pmid=17850214&amp;doi=10.1042/BJ20070797&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Vogt J., Traynor R., Sapkota G. P., 
The specificities of small molecule inhibitors of the TGFß and BMP pathways. Cell. Signal.
23, 
1831–1842 (2011).
</cite> [<a href="https://doi.org/10.1016/j.cellsig.2011.06.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21740966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell.%20Signal.&amp;title=The%20specificities%20of%20small%20molecule%20inhibitors%20of%20the%20TGF%C3%9F%20and%20BMP%20pathways&amp;volume=23&amp;publication_year=2011&amp;pages=1831-1842&amp;pmid=21740966&amp;doi=10.1016/j.cellsig.2011.06.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Dite T. A., Langendorf C. G., Hoque A., Galic S., Rebello R. J., Ovens A. J., Lindqvist L. M., Ngoei K. R. W., Ling N. X. Y., Furic L., Kemp B. E., Scott J. W., Oakhill J. S., 
AMP-activated protein kinase selectively inhibited by the type II inhibitor SBI-0206965. J. Biol. Chem.
293, 
8874–8885 (2018).
</cite> [<a href="https://doi.org/10.1074/jbc.RA118.003547" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5995511/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29695504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=AMP-activated%20protein%20kinase%20selectively%20inhibited%20by%20the%20type%20II%20inhibitor%20SBI-0206965&amp;volume=293&amp;publication_year=2018&amp;pages=8874-8885&amp;pmid=29695504&amp;doi=10.1074/jbc.RA118.003547&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>B. Dasgupta, W. Seibel, Compound C/Dorsomorphin: Its use and misuse as an AMPK inhibitor, in <em>Methods in Molecular Biology</em>, (Humana Press Inc., 2018), pp. 195–202.</cite> [<a href="https://doi.org/10.1007/978-1-4939-7598-3_12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29480476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?B.%20Dasgupta,%20W.%20Seibel,%20Compound%20C/Dorsomorphin:%20Its%20use%20and%20misuse%20as%20an%20AMPK%20inhibitor,%20in%20Methods%20in%20Molecular%20Biology,%20(Humana%20Press%20Inc.,%202018),%20pp.%20195%E2%80%93202." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Egan D. F., Chun M. G. H., Vamos M., Zou H., Rong J., Miller C. J., Lou H. J., Raveendra-Panickar D., Yang C. C., Sheffler D. J., Teriete P., Asara J., Turk B., Cosford N., Shaw R., 
Small molecule inhibition of the autophagy kinase ULK1 and identification of ULK1 substrates. Mol. Cell
59, 
285–297 (2015).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2015.05.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4530630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26118643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Small%20molecule%20inhibition%20of%20the%20autophagy%20kinase%20ULK1%20and%20identification%20of%20ULK1%20substrates&amp;volume=59&amp;publication_year=2015&amp;pages=285-297&amp;pmid=26118643&amp;doi=10.1016/j.molcel.2015.05.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Ahwazi D., Neopane K., Markby G. R., Kopietz F., Ovens A. J., Dall M., Hassing A. S., Gräsle P., Alshuweishi Y., Treebak J. T., Salt I., Göransson O., Zeqiraj E., Scott J., Sakamoto K., 
Investigation of the specificity and mechanism of action of the ULK1/AMPK inhibitor SBI-0206965. Biochem. J.
478, 
2977–2997 (2021).
</cite> [<a href="https://doi.org/10.1042/BCJ20210284" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8370752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34259310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=Investigation%20of%20the%20specificity%20and%20mechanism%20of%20action%20of%20the%20ULK1/AMPK%20inhibitor%20SBI-0206965&amp;volume=478&amp;publication_year=2021&amp;pages=2977-2997&amp;pmid=34259310&amp;doi=10.1042/BCJ20210284&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Lemos C., Schulze V. K., Baumgart S. J., Nevedomskaya E., Heinrich T., Lefranc J., Bader B., Christ C. D., Briem H., Kuhnke L. P., Holton S., Bömer U., Lienau P., von Nussbaum F., Nising C., Bauser M., Hägebarth A., Mumberg D., Haendler B., 
The potent AMPK inhibitor BAY-3827 shows strong efficacy in androgen-dependent prostate cancer models. Cell. Oncol.
44, 
581–594 (2021).</cite> [<a href="https://doi.org/10.1007/s13402-020-00584-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33492659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell.%20Oncol.&amp;title=The%20potent%20AMPK%20inhibitor%20BAY-3827%20shows%20strong%20efficacy%20in%20androgen-dependent%20prostate%20cancer%20models&amp;volume=44&amp;publication_year=2021&amp;pages=581-594&amp;pmid=33492659&amp;doi=10.1007/s13402-020-00584-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Hawley S. A., Russell F. M., Ross F. A., Hardie D. G., 
BAY-3827 and SBI-0206965: Potent ampk inhibitors that paradoxically increase Thr172 phosphorylation. Int. J. Mol. Sci.
25, 
453 (2024).</cite> [<a href="https://doi.org/10.3390/ijms25010453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10778976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38203624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=BAY-3827%20and%20SBI-0206965:%20Potent%20ampk%20inhibitors%20that%20paradoxically%20increase%20Thr172%20phosphorylation&amp;volume=25&amp;publication_year=2024&amp;pages=453&amp;pmid=38203624&amp;doi=10.3390/ijms25010453&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Manning G., Whyte D. B., Martinez R., Hunter T., Sudarsanam S., 
The protein kinase complement of the human genome. Science
298, 
1912–1934 (2002).
</cite> [<a href="https://doi.org/10.1126/science.1075762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12471243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20protein%20kinase%20complement%20of%20the%20human%20genome&amp;volume=298&amp;publication_year=2002&amp;pages=1912-1934&amp;pmid=12471243&amp;doi=10.1126/science.1075762&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Ashkenazy H., Abadi S., Martz E., Chay O., Mayrose I., Pupko T., Ben-Tal N., 
ConSurf 2016: An improved methodology to estimate and visualize evolutionary conservation in macromolecules. Nucleic Acids Res.
44, 
W344–W350 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkw408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4987940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27166375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=ConSurf%202016:%20An%20improved%20methodology%20to%20estimate%20and%20visualize%20evolutionary%20conservation%20in%20macromolecules&amp;volume=44&amp;publication_year=2016&amp;pages=W344-W350&amp;pmid=27166375&amp;doi=10.1093/nar/gkw408&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Yariv B., Yariv E., Kessel A., Masrati G., Chorin A. B., Martz E., Mayrose I., Pupko T., Ben-Tal N., 
Using evolutionary data to make sense of macromolecules with a “face-lifted” ConSurf. Protein Sci.
32, 
e4582 (2023).
</cite> [<a href="https://doi.org/10.1002/pro.4582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9942591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36718848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=Using%20evolutionary%20data%20to%20make%20sense%20of%20macromolecules%20with%20a%20%E2%80%9Cface-lifted%E2%80%9D%20ConSurf&amp;volume=32&amp;publication_year=2023&amp;pages=e4582&amp;pmid=36718848&amp;doi=10.1002/pro.4582&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Langendorf C. G., Ngoei K. R. W., Scott J. W., Ling N. X. Y., Issa S. M. A., Gorman M. A., Parker M. W., Sakamoto K., Oakhill J. S., Kemp B. E., 
Structural basis of allosteric and synergistic activation of AMPK by furan-2-phosphonic derivative C2 binding. Nat. Commun.
7, 
10912 (2016).
</cite> [<a href="https://doi.org/10.1038/ncomms10912" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4786773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26952388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Structural%20basis%20of%20allosteric%20and%20synergistic%20activation%20of%20AMPK%20by%20furan-2-phosphonic%20derivative%20C2%20binding&amp;volume=7&amp;publication_year=2016&amp;pages=10912&amp;pmid=26952388&amp;doi=10.1038/ncomms10912&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Modi V., Dunbrack R. L., 
Defining a new nomenclature for the structures of active and inactive kinases. Proc. Natl. Acad. Sci. U.S.A.
116, 
6818–6827 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1814279116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6452665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30867294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Defining%20a%20new%20nomenclature%20for%20the%20structures%20of%20active%20and%20inactive%20kinases&amp;volume=116&amp;publication_year=2019&amp;pages=6818-6827&amp;pmid=30867294&amp;doi=10.1073/pnas.1814279116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Taylor S. S., Keshwani M. M., Steichen J. M., Kornev A. P., 
Evolution of the eukaryotic protein kinases as dynamic molecular switches. Philos. Trans. R. Soc. Lond. B. Biol. Sci.
367, 
2517–2528 (2012).
</cite> [<a href="https://doi.org/10.1098/rstb.2012.0054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3415842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22889904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B.%20Biol.%20Sci.&amp;title=Evolution%20of%20the%20eukaryotic%20protein%20kinases%20as%20dynamic%20molecular%20switches&amp;volume=367&amp;publication_year=2012&amp;pages=2517-2528&amp;pmid=22889904&amp;doi=10.1098/rstb.2012.0054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Sanders M. J., Ratinaud Y., Neopane K., Bonhoure N., Day E. A., Ciclet O., Lassueur S., Pinta M. N., Deak M., Brinon B., Christen S., Steinberg G., Barron D., Sakamoto K., 
Natural (dihydro)phenanthrene plant compounds are direct activators of AMPK through its allosteric drug and metabolite-binding site. J. Biol. Chem.
298, 
101852 (2022).
</cite> [<a href="https://doi.org/10.1016/j.jbc.2022.101852" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9108889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35331736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Natural%20(dihydro)phenanthrene%20plant%20compounds%20are%20direct%20activators%20of%20AMPK%20through%20its%20allosteric%20drug%20and%20metabolite-binding%20site&amp;volume=298&amp;publication_year=2022&amp;pages=101852&amp;pmid=35331736&amp;doi=10.1016/j.jbc.2022.101852&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Boudaba N., Marion A., Huet C., Pierre R., Viollet B., Foretz M., 
Ampk re-activation suppresses hepatic steatosis but its downregulation does not promote fatty liver development. EBioMedicine
28, 
194–209 (2018).
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2018.01.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5835560/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29343420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EBioMedicine&amp;title=Ampk%20re-activation%20suppresses%20hepatic%20steatosis%20but%20its%20downregulation%20does%20not%20promote%20fatty%20liver%20development&amp;volume=28&amp;publication_year=2018&amp;pages=194-209&amp;pmid=29343420&amp;doi=10.1016/j.ebiom.2018.01.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Hunter R. W., Foretz M., Bultot L., Fullerton M. D., Deak M., Ross F. A., Hawley S. A., Shpiro N., Viollet B., Barron D., Kemp B., Steinberg G., Hardie D., Sakamoto K., 
Mechanism of action of compound-13: An α1-selective small molecule activator of AMPK. Chem. Biol.
21, 
866–879 (2014).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2014.05.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4104029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25036776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&amp;title=Mechanism%20of%20action%20of%20compound-13:%20An%20%CE%B11-selective%20small%20molecule%20activator%20of%20AMPK&amp;volume=21&amp;publication_year=2014&amp;pages=866-879&amp;pmid=25036776&amp;doi=10.1016/j.chembiol.2014.05.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Hashimoto T., Urushihara Y., Murata Y., Fujishima Y., Hosoi Y., 
AMPK increases expression of ATM through transcriptional factor Sp1 and induces radioresistance under severe hypoxia in glioblastoma cell lines. Biochem. Biophys. Res. Commun.
590, 
82–88 (2022).
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2021.12.076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34973534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;title=AMPK%20increases%20expression%20of%20ATM%20through%20transcriptional%20factor%20Sp1%20and%20induces%20radioresistance%20under%20severe%20hypoxia%20in%20glioblastoma%20cell%20lines&amp;volume=590&amp;publication_year=2022&amp;pages=82-88&amp;pmid=34973534&amp;doi=10.1016/j.bbrc.2021.12.076&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Rao X. S., Cong X. X., Gao X. K., Shi Y. P., Shi L. J., Wang J. F., Ni C. Y., He M. J., Xu Y., Yi C., Meng Z., Liu J., Lin P., Zheng L., Zhou Y., 
AMPK-mediated phosphorylation enhances the auto-inhibition of TBC1D17 to promote Rab5-dependent glucose uptake. Cell Death Differ.
28, 
3214–3234 (2021).
</cite> [<a href="https://doi.org/10.1038/s41418-021-00809-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8630067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34045668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&amp;title=AMPK-mediated%20phosphorylation%20enhances%20the%20auto-inhibition%20of%20TBC1D17%20to%20promote%20Rab5-dependent%20glucose%20uptake&amp;volume=28&amp;publication_year=2021&amp;pages=3214-3234&amp;pmid=34045668&amp;doi=10.1038/s41418-021-00809-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Yamano K., Fogel A. I., Wang C., van der Bliek A. M., Youle R. J., 
Mitochondrial Rab GAPs govern autophagosome biogenesis during mitophagy. eLife
3, 
e01612 (2014).
</cite> [<a href="https://doi.org/10.7554/eLife.01612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3930140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24569479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Mitochondrial%20Rab%20GAPs%20govern%20autophagosome%20biogenesis%20during%20mitophagy&amp;volume=3&amp;publication_year=2014&amp;pages=e01612&amp;pmid=24569479&amp;doi=10.7554/eLife.01612&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Yang W. S., Caliva M. J., Khadka V. S., Tiirikainen M., Matter M. L., Deng Y., Ramos J. W., 
RSK1 and RSK2 serine/threonine kinases regulate different transcription programs in cancer. Front. Cell Dev. Biol.
10, 
1015665 (2023).
</cite> [<a href="https://doi.org/10.3389/fcell.2022.1015665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9845784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36684450/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell%20Dev.%20Biol.&amp;title=RSK1%20and%20RSK2%20serine/threonine%20kinases%20regulate%20different%20transcription%20programs%20in%20cancer&amp;volume=10&amp;publication_year=2023&amp;pages=1015665&amp;pmid=36684450&amp;doi=10.3389/fcell.2022.1015665&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Sapkota G. P., Cummings L., Newell F. S., Armstrong C., Bain J., Frodin M., Grauert M., Hoffmann M., Schnapp G., Steegmaier M., Cohen P., Alessi D., 
BI-D1870 is a specific inhibitor of the p90 RSK (ribosomal S6 kinase) isoforms in vitro and in vivo. Biochem. J.
401, 
29–38 (2007).
</cite> [<a href="https://doi.org/10.1042/BJ20061088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1698666/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17040210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=BI-D1870%20is%20a%20specific%20inhibitor%20of%20the%20p90%20RSK%20(ribosomal%20S6%20kinase)%20isoforms%20in%20vitro%20and%20in%20vivo&amp;volume=401&amp;publication_year=2007&amp;pages=29-38&amp;pmid=17040210&amp;doi=10.1042/BJ20061088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Utepbergenov D., Derewenda U., Olekhnovich N., Szukalska G., Banerjee B., Hilinski M. K., Lannigan D. A., Stukenberg P. T., Derewenda Z. S., 
Insights into the inhibition of the p90 ribosomal S6 kinase (RSK) by the flavonol glycoside SL0101 from the 1.5 Å crystal structure of the N-terminal domain of RSK2 with bound inhibitor. Biochemistry
51, 
6499–6510 (2012).
</cite> [<a href="https://doi.org/10.1021/bi300620c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3462495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22846040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Insights%20into%20the%20inhibition%20of%20the%20p90%20ribosomal%20S6%20kinase%20(RSK)%20by%20the%20flavonol%20glycoside%20SL0101%20from%20the%201.5%20%C3%85%20crystal%20structure%20of%20the%20N-terminal%20domain%20of%20RSK2%20with%20bound%20inhibitor&amp;volume=51&amp;publication_year=2012&amp;pages=6499-6510&amp;pmid=22846040&amp;doi=10.1021/bi300620c&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Jain R., Mathur M., Lan J., Costales A., Atallah G., Ramurthy S., Subramanian S., Setti L., Feucht P., Warne B., Doyle L., Basham S., Jefferson A., Lindvall M., Appleton B., Shafer C., 
Discovery of potent and selective RSK inhibitors as biological probes. J. Med. Chem.
58, 
6766–6783 (2015).
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.5b00450" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26270416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Discovery%20of%20potent%20and%20selective%20RSK%20inhibitors%20as%20biological%20probes&amp;volume=58&amp;publication_year=2015&amp;pages=6766-6783&amp;pmid=26270416&amp;doi=10.1021/acs.jmedchem.5b00450&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>Murray B. W., Guo C., Piraino J., Westwick J. K., Zhang C., Lamerdin J., Dagostino E., Knighton D., Loi C. M., Zager M., Kraynov E., Popoff I., Christensen J., Martinez R., Kephart S., Marakovits J., Karlicek S., Bergqvist S., Smeal T., 
Small-molecule p21-activated kinase inhibitor PF-3758309 is a potent inhibitor of oncogenic signaling and tumor growth. Proc. Natl. Acad. Sci. U.S.A.
107, 
9446–9451 (2010).
</cite> [<a href="https://doi.org/10.1073/pnas.0911863107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2889050/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20439741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Small-molecule%20p21-activated%20kinase%20inhibitor%20PF-3758309%20is%20a%20potent%20inhibitor%20of%20oncogenic%20signaling%20and%20tumor%20growth&amp;volume=107&amp;publication_year=2010&amp;pages=9446-9451&amp;pmid=20439741&amp;doi=10.1073/pnas.0911863107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Schneider C., Hilbert J., Genevaux F., Höfer S., Krauß L., Schicktanz F., Contreras C. T., Jansari S., Papargyriou A., Richter T., Alfayomy A. M., Falcomatà C., Schneeweis C., Orben F., Öllinger R., Wegwitz F., Boshnakovska A., Rehling P., Müller D., Ströbel P., Ellenrieder V., Conradi L., Hessmann E., Ghadimi M., Grade M., Wirth M., Steiger K., Rad R., Kuster B., Sippl W., Reichert M., Saur D., Schneider G., 
A novel AMPK inhibitor sensitizes pancreatic cancer cells to ferroptosis induction. Adv. Sci.
11, 
2307695 (2024).</cite> [<a href="https://doi.org/10.1002/advs.202307695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11336956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38885414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Sci.&amp;title=A%20novel%20AMPK%20inhibitor%20sensitizes%20pancreatic%20cancer%20cells%20to%20ferroptosis%20induction&amp;volume=11&amp;publication_year=2024&amp;pages=2307695&amp;pmid=38885414&amp;doi=10.1002/advs.202307695&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>Möbitz H., 
The ABC of protein kinase conformations. Biochim. Biophys. Acta
1854, 
1555–1566 (2015).
</cite> [<a href="https://doi.org/10.1016/j.bbapap.2015.03.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25839999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&amp;title=The%20ABC%20of%20protein%20kinase%20conformations&amp;volume=1854&amp;publication_year=2015&amp;pages=1555-1566&amp;pmid=25839999&amp;doi=10.1016/j.bbapap.2015.03.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Byrne D. P., Shrestha S., Galler M., Cao M., Daly L. A., Campbell A. E., Eyers C. E., Veal E. A., Kannan N., Eyers P. A., 
Aurora A regulation by reversible cysteine oxidation reveals evolutionarily conserved redox control of Ser/Thr protein kinase activity. Sci. Signal.
639, 
2713 (2020).</cite> [<a href="https://doi.org/10.1126/scisignal.aax2713" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32636306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Signal.&amp;title=Aurora%20A%20regulation%20by%20reversible%20cysteine%20oxidation%20reveals%20evolutionarily%20conserved%20redox%20control%20of%20Ser/Thr%20protein%20kinase%20activity&amp;volume=639&amp;publication_year=2020&amp;pages=2713&amp;pmid=32636306&amp;doi=10.1126/scisignal.aax2713&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>Huang X., Begley M., Morgenstern K. A., Gu Y., Rose P., Zhao H., Zhu X., 
Crystal structure of an inactive Akt2 kinase domain. Structure
1, 
21–30 (2003).</cite> [<a href="https://doi.org/10.1016/s0969-2126(02)00937-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12517337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Crystal%20structure%20of%20an%20inactive%20Akt2%20kinase%20domain&amp;volume=1&amp;publication_year=2003&amp;pages=21-30&amp;pmid=12517337&amp;doi=10.1016/s0969-2126(02)00937-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Anjum R., Blenis J., 
The RSK family of kinases: Emerging roles in cellular signalling. Nat. Rev. Mol. Cell Biol.
9, 
747–758 (2008).
</cite> [<a href="https://doi.org/10.1038/nrm2509" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18813292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;title=The%20RSK%20family%20of%20kinases:%20Emerging%20roles%20in%20cellular%20signalling&amp;volume=9&amp;publication_year=2008&amp;pages=747-758&amp;pmid=18813292&amp;doi=10.1038/nrm2509&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Koutsougianni F., Alexopoulou D., Uvez A., Lamprianidou A., Sereti E., Tsimplouli C., Ilkay Armutak E., Dimas K., 
P90 ribosomal S6 kinases: A bona fide target for novel targeted anticancer therapies?
Biochem. Pharmacol.
210, 
115488 (2023).
</cite> [<a href="https://doi.org/10.1016/j.bcp.2023.115488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36889445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Pharmacol.&amp;title=P90%20ribosomal%20S6%20kinases:%20A%20bona%20fide%20target%20for%20novel%20targeted%20anticancer%20therapies?&amp;volume=210&amp;publication_year=2023&amp;pages=115488&amp;pmid=36889445&amp;doi=10.1016/j.bcp.2023.115488&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Viollet B., Foretz M., 
Animal models to study AMPK. EXS
107, 
441–469 (2016).
</cite> [<a href="https://doi.org/10.1007/978-3-319-43589-3_18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27812991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EXS&amp;title=Animal%20models%20to%20study%20AMPK&amp;volume=107&amp;publication_year=2016&amp;pages=441-469&amp;pmid=27812991&amp;doi=10.1007/978-3-319-43589-3_18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R63">
<span class="label">63.</span><cite>Ashraf N., van Nostrand J. L., 
Fine-tuning AMPK in physiology and disease using point-mutant mouse models. Dis. Model Mech.
17, 
dmm050798 (2024).
</cite> [<a href="https://doi.org/10.1242/dmm.050798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11340815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39136185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dis.%20Model%20Mech.&amp;title=Fine-tuning%20AMPK%20in%20physiology%20and%20disease%20using%20point-mutant%20mouse%20models&amp;volume=17&amp;publication_year=2024&amp;pages=dmm050798&amp;pmid=39136185&amp;doi=10.1242/dmm.050798&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R64">
<span class="label">64.</span><cite>Sakamoto K., McCarthy A., Smith D., Green K. A., Hardie D. G., Ashworth A., Alessi D. R., 
Deficiency of LKB1 in skeletal muscle prevents AMPK activation and glucose uptake during contraction. EMBO J.
24, 
1810–1820 (2005).
</cite> [<a href="https://doi.org/10.1038/sj.emboj.7600667" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1142598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15889149/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&amp;title=Deficiency%20of%20LKB1%20in%20skeletal%20muscle%20prevents%20AMPK%20activation%20and%20glucose%20uptake%20during%20contraction&amp;volume=24&amp;publication_year=2005&amp;pages=1810-1820&amp;pmid=15889149&amp;doi=10.1038/sj.emboj.7600667&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R65">
<span class="label">65.</span><cite>Hastie C. J., McLauchlan H. J., Cohen P., 
Assay of protein kinases using radiolabeled ATP: A protocol. Nat. Protoc.
1, 
968–971 (2006).
</cite> [<a href="https://doi.org/10.1038/nprot.2006.149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17406331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Assay%20of%20protein%20kinases%20using%20radiolabeled%20ATP:%20A%20protocol&amp;volume=1&amp;publication_year=2006&amp;pages=968-971&amp;pmid=17406331&amp;doi=10.1038/nprot.2006.149&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R66">
<span class="label">66.</span><cite>Dite T. A., Ling N. X. Y., Scott J. W., Hoque A., Galic S., Parker B. L., Ngoei K. R. W., Langendorf C. G., O’Brien M. T., Kundu M., Steinberg G. R., Sakamoto K., Kemp B. E., Oakhill J. S., 
The autophagy initiator ULK1 sensitizes AMPK to allosteric drugs. Nat. Commun.
8, 
571 (2017).
</cite> [<a href="https://doi.org/10.1038/s41467-017-00628-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5603566/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28924239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=The%20autophagy%20initiator%20ULK1%20sensitizes%20AMPK%20to%20allosteric%20drugs&amp;volume=8&amp;publication_year=2017&amp;pages=571&amp;pmid=28924239&amp;doi=10.1038/s41467-017-00628-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R67">
<span class="label">67.</span><cite>Scott J. W., Park E., Rodriguiz R. M., Oakhill J. S., Issa S. M. A., Obrien M. T., Dite T. A., Langendorf C. G., Wetsel W. C., Means A. R., Kemp B. E., 
Autophosphorylation of CaMKK2 generates autonomous activity that is disrupted by a T85S mutation linked to anxiety and bipolar disorder. Sci. Rep.
5, 
14436 (2015).
</cite> [<a href="https://doi.org/10.1038/srep14436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4585769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26395653/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Autophosphorylation%20of%20CaMKK2%20generates%20autonomous%20activity%20that%20is%20disrupted%20by%20a%20T85S%20mutation%20linked%20to%20anxiety%20and%20bipolar%20disorder&amp;volume=5&amp;publication_year=2015&amp;pages=14436&amp;pmid=26395653&amp;doi=10.1038/srep14436&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R68">
<span class="label">68.</span><cite>Waterhouse A. M., Procter J. B., Martin D. M. A., Clamp M., Barton G. J., 
Jalview Version 2–A multiple sequence alignment editor and analysis workbench. Bioinformatics
25, 
1189–1191 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2672624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19151095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Jalview%20Version%202%E2%80%93A%20multiple%20sequence%20alignment%20editor%20and%20analysis%20workbench&amp;volume=25&amp;publication_year=2009&amp;pages=1189-1191&amp;pmid=19151095&amp;doi=10.1093/bioinformatics/btp033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R69">
<span class="label">69.</span><cite>Ewels P. A., Peltzer A., Fillinger S., Patel H., Alneberg J., Wilm A., Garcia M. U., Di Tommaso P., Nahnsen S., 
The nf-core framework for community-curated bioinformatics pipelines. Nat. Biotechnol.
38, 
276–278 (2020).
</cite> [<a href="https://doi.org/10.1038/s41587-020-0439-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32055031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=The%20nf-core%20framework%20for%20community-curated%20bioinformatics%20pipelines&amp;volume=38&amp;publication_year=2020&amp;pages=276-278&amp;pmid=32055031&amp;doi=10.1038/s41587-020-0439-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R70">
<span class="label">70.</span><cite>Robinson M. D., McCarthy D. J., Smyth G. K., 
edgeR: A bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics
26, 
139–140 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2796818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=edgeR:%20A%20bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&amp;volume=26&amp;publication_year=2009&amp;pages=139-140&amp;pmid=19910308&amp;doi=10.1093/bioinformatics/btp616&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R71">
<span class="label">71.</span><cite>Ritchie M. E., Phipson B., Wu D., Hu Y., Law C. W., Shi W., Smyth G. K., 
Limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res.
43, 
e47 (2015).
</cite> [<a href="https://doi.org/10.1093/nar/gkv007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4402510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25605792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Limma%20powers%20differential%20expression%20analyses%20for%20RNA-sequencing%20and%20microarray%20studies&amp;volume=43&amp;publication_year=2015&amp;pages=e47&amp;pmid=25605792&amp;doi=10.1093/nar/gkv007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R72">
<span class="label">72.</span><cite>L. Yan, ggvenn: Draw Venn Diagram by ‘ggplot2’. R package version 0.1.10 (2025); <a href="https://github.com/yanlinlin82/ggvenn" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/yanlinlin82/ggvennhttps://github.com/yanlinlin82/ggvenn</a>.</cite>
</li>
<li id="R73">
<span class="label">73.</span><cite>R. Kolde, pheatmap: Pretty Heatmaps. R package version 1.0.12 (2019); <a href="https://CRAN.R-project.org/package=pheatmaphttps://CRAN.R-project.org/package=pheatmap" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R-project.org/package=pheatmaphttps://CRAN.R-project.org/package=pheatmap</a>.</cite>
</li>
<li id="R74">
<span class="label">74.</span><cite>Xu S., Hu E., Cai Y., Xie Z., Luo X., Zhan L., Tang W., Wang Q., Liu B., Wang R., Xie W., Wu T., Xie L., Yu G., 
Using clusterProfiler to characterize multiomics data. Nat. Protoc.
19, 
3292–3320 (2024).
</cite> [<a href="https://doi.org/10.1038/s41596-024-01020-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39019974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Using%20clusterProfiler%20to%20characterize%20multiomics%20data&amp;volume=19&amp;publication_year=2024&amp;pages=3292-3320&amp;pmid=39019974&amp;doi=10.1038/s41596-024-01020-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R75">
<span class="label">75.</span><cite>R Core Team, R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria (2021); <a href="https://R-project.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://R-project.org</a>.</cite>
</li>
<li id="R76">
<span class="label">76.</span><cite>Vonrhein C., Flensburg C., Keller P., Sharff A., Smart O., Paciorek W., Womack T., Bricogne G., 
Data processing and analysis with the autoPROC toolbox. Acta Crystallogr. D Biol. Crystallogr.
67, 
293–302 (2011).
</cite> [<a href="https://doi.org/10.1107/S0907444911007773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3069744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21460447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.&amp;title=Data%20processing%20and%20analysis%20with%20the%20autoPROC%20toolbox&amp;volume=67&amp;publication_year=2011&amp;pages=293-302&amp;pmid=21460447&amp;doi=10.1107/S0907444911007773&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R77">
<span class="label">77.</span><cite>McCoy A. J., Grosse-Kunstleve R. W., Adams P. D., Winn M. D., Storoni L. C., Read R. J., 
Phaser crystallographic software. J. Appl. Cryst.
40, 
658–674 (2007).
</cite> [<a href="https://doi.org/10.1107/S0021889807021206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2483472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19461840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Cryst.&amp;title=Phaser%20crystallographic%20software&amp;volume=40&amp;publication_year=2007&amp;pages=658-674&amp;pmid=19461840&amp;doi=10.1107/S0021889807021206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R78">
<span class="label">78.</span><cite>Emsley P., Lohkamp B., Scott W. G., Cowtan K., 
Features and development of coot. Acta Crystallogr. D Biol. Crystallogr.
66, 
486–501 (2010).
</cite> [<a href="https://doi.org/10.1107/S0907444910007493" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2852313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.&amp;title=Features%20and%20development%20of%20coot&amp;volume=66&amp;publication_year=2010&amp;pages=486-501&amp;pmid=20383002&amp;doi=10.1107/S0907444910007493&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R79">
<span class="label">79.</span><cite>Afonine P. V., Grosse-Kunstleve R. W., Echols N., Headd J. J., Moriarty N. W., Mustyakimov M., Terwilliger T. C., Urzhumtsev A., Zwart P. H., Adams P. D., 
Towards automated crystallographic structure refinement with phenix.refine. Acta Crystallogr. D Biol. Crystallogr.
68, 
352–367 (2012).
</cite> [<a href="https://doi.org/10.1107/S0907444912001308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322595/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22505256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.%20D%20Biol.%20Crystallogr.&amp;title=Towards%20automated%20crystallographic%20structure%20refinement%20with%20phenix.refine&amp;volume=68&amp;publication_year=2012&amp;pages=352-367&amp;pmid=22505256&amp;doi=10.1107/S0907444912001308&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R80">
<span class="label">80.</span><cite>The PyMOL Molecular Graphics System, Version 3.0 Schrödinger, LLC.</cite> [<a href="https://scholar.google.com/scholar_lookup?The%20PyMOL%20Molecular%20Graphics%20System,%20Version%203.0%20Schr%C3%B6dinger,%20LLC." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R81">
<span class="label">81.</span><cite>Pettersen E. F., Goddard T. D., Huang C. C., Couch G. S., Greenblatt D. M., Meng E. C., Ferrin T. E., 
UCSF chimera - A visualization system for exploratory research and analysis. J. Comput. Chem.
25, 
1605–1612 (2004).
</cite> [<a href="https://doi.org/10.1002/jcc.20084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15264254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=UCSF%20chimera%20-%20A%20visualization%20system%20for%20exploratory%20research%20and%20analysis&amp;volume=25&amp;publication_year=2004&amp;pages=1605-1612&amp;pmid=15264254&amp;doi=10.1002/jcc.20084&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>Figs. S1 to S5</p>
<p>Tables S1 to S3</p>
<p>References</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12372887/bin/sciadv.adx2434_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adx2434_sm.pdf</a><sup> (15.8MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Science Advances are provided here courtesy of <strong>American Association for the Advancement of Science</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1126/sciadv.adx2434"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/sciadv.adx2434.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12372887/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12372887/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372887%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372887/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12372887/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12372887/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40845097/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12372887/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40845097/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12372887/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12372887/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="GXNMWXz6iz5l0hMc607FDomBKra6AnGEsQdtHdKT8qeN2azZDgVo5iW9fPl43V9Y">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
