
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A simple guide to the use of Student’s t-test, Mann-Whitney U test, Chi-squared test, and Kruskal-Wallis test in biostatistics - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5338458AF307030538450027983C99.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="biodmin">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366075/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BioData Mining">
<meta name="citation_title" content="A simple guide to the use of Student’s t-test, Mann-Whitney U test, Chi-squared test, and Kruskal-Wallis test in biostatistics">
<meta name="citation_author" content="Davide Chicco">
<meta name="citation_author_institution" content="Università di Milano-Bicocca, Milan, Italy">
<meta name="citation_author_institution" content="University of Toronto, Toronto, Ontario, Canada">
<meta name="citation_author" content="Andrea Sichenze">
<meta name="citation_author_institution" content="Università di Milano-Bicocca, Milan, Italy">
<meta name="citation_author" content="Giuseppe Jurman">
<meta name="citation_author_institution" content="Humanitas University, Milan, Italy">
<meta name="citation_author_institution" content="Fondazione Bruno Kessler, Trento, Italy">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="18">
<meta name="citation_firstpage" content="56">
<meta name="citation_doi" content="10.1186/s13040-025-00465-6">
<meta name="citation_pmid" content="40835959">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366075/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366075/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366075/pdf/13040_2025_Article_465.pdf">
<meta name="description" content="In an age when machine learning and artificial intelligence are broadly employed, traditional statistics can still provide insightful information and results quickly and at a low computational cost. Statistics, in fact, offers many useful tools to ...">
<meta name="og:title" content="A simple guide to the use of Student’s t-test, Mann-Whitney U test, Chi-squared test, and Kruskal-Wallis test in biostatistics">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="In an age when machine learning and artificial intelligence are broadly employed, traditional statistics can still provide insightful information and results quickly and at a low computational cost. Statistics, in fact, offers many useful tools to ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366075/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366075">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s13040-025-00465-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/13040_2025_Article_465.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366075%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366075/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366075/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366075/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-biodmin.png" alt="BioData Mining logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BioData Mining" title="Link to BioData Mining" shape="default" href="https://biodatamining.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BioData Min</button></div>. 2025 Aug 20;18:56. doi: <a href="https://doi.org/10.1186/s13040-025-00465-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s13040-025-00465-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BioData%20Min%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BioData%20Min%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BioData%20Min%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BioData%20Min%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A simple guide to the use of Student’s <em>t</em>-test, Mann-Whitney <em>U</em> test, Chi-squared test, and Kruskal-Wallis test in biostatistics</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chicco%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Davide Chicco</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Davide Chicco</span></h3>
<div class="p">
<sup>1</sup>Università di Milano-Bicocca, Milan, Italy </div>
<div class="p">
<sup>2</sup>University of Toronto, Toronto, Ontario, Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chicco%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Davide Chicco</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sichenze%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Andrea Sichenze</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Andrea Sichenze</span></h3>
<div class="p">
<sup>1</sup>Università di Milano-Bicocca, Milan, Italy </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sichenze%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrea Sichenze</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jurman%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Giuseppe Jurman</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Giuseppe Jurman</span></h3>
<div class="p">
<sup>3</sup>Humanitas University, Milan, Italy </div>
<div class="p">
<sup>4</sup>Fondazione Bruno Kessler, Trento, Italy </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jurman%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Giuseppe Jurman</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Università di Milano-Bicocca, Milan, Italy </div>
<div id="Aff2">
<sup>2</sup>University of Toronto, Toronto, Ontario, Canada </div>
<div id="Aff3">
<sup>3</sup>Humanitas University, Milan, Italy </div>
<div id="Aff4">
<sup>4</sup>Fondazione Bruno Kessler, Trento, Italy </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 3; Accepted 2025 Jul 20; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366075  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835959/" class="usa-link">40835959</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">In an age when machine learning and artificial intelligence are broadly employed, traditional statistics can still provide insightful information and results quickly and at a low computational cost. Statistics, in fact, offers many useful tools to researchers, including a series of univariate statistical tests that can identify relationships between pairs of numeric samples: Student’s <em>t</em>-test, Mann-Whitney <em>U</em> test, Chi-squared test, and Kruskal-Wallis test. These tests generate several outcomes, including probability values (<em>p</em>-values) that can express a numerical quantity which accepts or rejects the null hypothesis, based on a certain threshold used. Although effective, these tests are often misused or employed in the wrong contexts, especially among biostatistics studies. Many scientific researchers do not seem to know how to choose one test over the others, and this misuse can lead to incorrect results and wrong conclusions. Here we present a simple theoretical and practical guide to the use of these four tests, first describing their theoretical properties and then displaying the results obtained by applying these tests to real-world medical datasets. Eventually, we explain when and how to use each test based on the data types of the samples considered. Our study can have a strong impact on scientific research by potentially influencing future studies involving these tests. Our recommendations, in turn, can help researchers produce more reliable and sound scientific results, thus increasing the quality of multiple scientific studies across various fields.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Biostatistics, Chi-squared test, Kruskal-Wallis test, Mann-Whitney <em>U</em> test, Statistical significance, Student’s <em>t</em>-test, Univariate statistical tests</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par33">Today, machine learning and artificial intelligence have spread in all scientific fields, paving the way to a huge number of new scientific discoveries [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. These tools, even if useful, suffer from several flaws: their mathematical models often are complex and obscure, their application can be slow and computationally expensive, and their incorrect use can lead to several negative consequences [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>].</p>
<p id="Par34">Biostatistics, on the contrary, offers several statistical tests that can serve to prove a relationship between variables, at a low computational cost, quickly, and in human-understandable way. For these reasons, univariate statistical tests are broadly employed by researchers in several scientific fields nowadays. These statistical tests, however, sometimes are chosen in the wrong way or out of context, and this wrong selection can produce bad results and outcomes eventually. In this study, we try to get some order in this confused situation by providing a guide on four common univariate statistical tests: Student’s <em>t</em>-test, Mann-Whitney <em>U</em> test, Chi-squared test, and Kruskal-Wallis test.</p>
<p id="Par35">First, we provide an informal and purely explanatory definition of the statistical tests that will be analyzed later, accompanied by conceptual examples. The objective is to offer a general and straightforward understanding of each test.</p>
<p id="Par36">The Student’s <em>t</em>-statistic is a statistical procedure used to determine if there is a significant difference between two groups. For example, if we have two groups of patients and one receives drug A while the other receives drug B, we want to understand if there is a significant difference in their effects. There are three different types of <em>t</em>-tests: one-sample, independent samples, and paired samples. The one-sample <em>t</em>-test is used when we want to compare the mean of a sample to a known reference value. For instance, if a chocolate bar company claims that each bar weighs an average of fifty grams, we could take a sample of thirty bars that have an average weight of forty-eight grams. The one-sample <em>t</em>-test would help us determine if this average of forty-eight grams is significantly different from fifty grams. The independent samples <em>t</em>-test is applied when we want to compare the means of two independent samples to check if there is a significant difference between them. For example, if we want to test the effectiveness of two painkillers, A and B, we could use a sample of sixty people divided into two groups, one receiving drug A and the other receiving drug B. This type of <em>t</em>-test would allow us to assess if there is a significant difference in the effectiveness of the two drugs. The paired samples <em>t</em>-test is used when we want to compare the means of two dependent samples to see if there is a significant difference. For instance, to evaluate the effectiveness of a particular diet, we could weigh the same group of people before and after the diet and then observe the weight difference for each individual. It’s important to highlight the difference between independent and paired samples: in the case of paired samples, the measurements are taken in pairs from the same individuals, such as results obtained from the same people before and after a treatment; in contrast, for independent samples, the individuals in the two groups are unrelated to each other. The one-sample test is very similar to the paired sample test, where we can think of paired samples as two samples taken at different times. We calculate the difference between paired values, obtaining a value and determining its significance based on how much this value deviates from a reference value [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>].</p>
<p id="Par37">The Mann-Whitney test is a nonparametric test that checks the difference between two independent samples. For example: is there a significant difference in reaction times between men and women? The difference between the independent samples <em>t</em>-test and Mann-Whitney is that the <em>t</em>-test uses the value of the mean between the two groups, while Mann-Whitney <em>U</em> test uses the sum of ranks. To calculate the sum of ranks: order the subjects from lowest to highest value. The subject with the lowest value gets rank 1, the second-lowest rank 2, and so on, obtaining two groups with different ranks. The data must therefore be rankable. The advantage of using the sum of ranks compared to the mean difference is that the data do not need to follow a normal distribution [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>].</p>
<p id="Par38">The Chi-squared test is a statistical technique used to compare observed data with expected distributions based on a specific hypothesis. Its main purpose is to determine whether the differences between observed and expected values can be attributed to chance or if they suggest a significant relationship between variables. There are three common applications of the Chi-squared test: test of independence, goodness-of-fit test and test for homogeneity. The Chi-squared test of Independence is used to assess whether there is a relationship between two categorical variables. For example, researchers might want to investigate if gender influences the likelihood of having a Netflix subscription. In this case, a survey could be conducted to record whether individuals of different genders have a subscription. The data would then be arranged in a contingency table, allowing researchers to compare the observed frequencies to the expected frequencies calculated under the assumption that gender and subscription status are independent of each other. Another application is the Chi-squared goodness-of-fit test, which is used to evaluate if the observed frequencies for a categorical variable match the expected frequencies based on a known distribution. For instance, if a market researcher wants to find out whether the distribution of video streaming service subscriptions—such as Netflix, Amazon, and Disney—in a specific city aligns with the national distribution, this test would be appropriate. By comparing the observed data from the city with the expected frequencies derived from national statistics, the researcher can determine if the distribution significantly differs. In the end, the Chi-squared test for Homogeneity is used to check if different populations share the same distribution for a categorical variable. As an example, a survey might be conducted to explore if subscription rates for streaming services vary across different age groups. The observed data from each age group would be compared to determine if the distributions are similar or significantly different. The key differences among these tests lie in the number of variables involved and the research questions they address: the test of independence examines relationships between two variables, the goodness-of-fit test checks if a single variable matches a known distribution, and the test for homogeneity compares distributions across multiple groups In conclusion, the Chi-squared test proves to be a versatile tool for analyzing relationships between categorical variables, assessing distributions, and comparing different populations [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>].</p>
<p id="Par39">The Kruskal-Wallis test is a non-parametric statistical method used to determine if there are significant differences between the medians of three or more independent groups. This test is particularly useful when the data do not follow a normal distribution, allowing researchers to make valid comparisons without relying on the assumptions required for parametric tests. For example, if a researcher who wants to find out if three different diets lead to different amounts of weight loss, the Kruskal-Wallis test can compare the median weight loss across these diet groups to see if the differences are statistically significant. This test is especially appropriate in situations where we have three or more independent groups to compare and when data are either ordinal or continuous but not normally distributed. It is also useful as a test of choice when we are interested in comparing medians rather than means, making it a flexible option for various types of data. For instance, a doctor might want to test if three different types of physical therapy result in different recovery times for patients after surgery. By dividing the patients into three groups, each receiving a different therapy, the doctor could use the Kruskal-Wallis test to assess if there are significant differences in recovery times among these groups. The way the Kruskal-Wallis test works is straightforward yet effective. It begins by ranking all the data from the smallest to the largest, regardless of the group each data point belongs to. Then, it calcul ates the sum of these ranks for each group and checks if the sums differ significantly. If the test finds a significant difference, Kruskal-Wallis test suggests that at least one group is different from the others. In such cases, additional post-hoc tests can be performed to identify which specific groups differ. Overall, the Kruskal-Wallis test is a powerful tool for analyzing data that do not meet the assumptions required for parametric tests, making it a valuable technique in a wide range of research fields [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>].</p>
<section id="Sec2"><h3 class="pmc_sec_title">Literature review</h3>
<p id="Par40">This section examines the four tests by reviewing their application in medical research using data acquired from EHRs. In the following text, we explain how each test has been employed, the type of data computed with the test and the results of the study. Of course, the list of articles is a subpart of all the studies involving the use of the four tests analyzed.</p>
<section id="FPar1"><h4 class="pmc_sec_title">Student’s <em>t</em>-test</h4>
<p id="Par41">Daniel Moynihan and coauthors [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>] conducted a study on the diagnostic processes for rare diseases, specifically Fabry Disease and Familial Hypercholesterolemia, using data from approximately one million patients across three hospitals in Singapore. A two-sample <em>t</em>-test for LDL-C levels was applied to compare confirmed cases (true positives) and suspect cohorts of FH, revealing no statistically significant difference.</p>
<p>Heather Patton et al. [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>] conducted a study on a nonalcoholic fatty liver disease care pathway, evaluating adherence to components such as patient education, vibration controlled transient elastography examination, hepatology consultation, and weight management referral. They assessed the impact of the pathway on weight and ALT changes in 632 patients over a one-year period. The Student’s <em>t</em>-test was used to analyze the impact of weight management participation and liver stiffness on weight change and ALT change, finding that referral to WM was associated with significant weight loss and ALT reduction</p>
<p>Cindy K. Blair and coauthors [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>] used Student’s <em>t</em>-test to compare Body Mass Index, tumor size, and age at diagnosis across groups defined by tumor characteristics (for example, stage, grade), finding significant associations between obesity and poor prognosis tumors.</p>
<p>Charlotte A. Nelson et al. [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] developed a method to predict chronic diseases, specifically multiple sclerosis, up to five years prior to clinical diagnosis by embedding electronic health record data into a biomedical knowledge graph and applying a random forest classifier. The Student’s <em>t</em>-test was employed to compare the rank distributions of nodes within the SPOKE graph between patients with multiple sclerosis and patients without multiple sclerosis.</p>
<p>Heather Patton et al. [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>] studied a care pathway for nonalcoholic fatty liver disease, assessing adherence to key steps and their effects on weight and ALT levels in over six hundred patients. The Student’s <em>t</em>-test showed that referral to weight management was linked to significant reductions in both weight and ALT.</p>
<p>Cindy K. Blair et al. [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>] used Student’s <em>t</em>-test to compare body mass index, tumor size, and age by tumor traits, finding obesity significantly linked to poor-prognosis tumors.</p>
<p>Charlotte A. Nelson et al. [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] predicted multiple sclerosis up to five years before diagnosis using a biomedical knowledge graph and random forest. Student’s <em>t</em>-test compared graph node ranks between patients with and without the disease.</p>
<p>Charles-Henri David and coauthors [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>] conducted a retrospective study to assess the outcomes of patients with postcardiotomy cardiogenic shock supported with a left ventricular assist device on a 5-year period. A total of 29 patients (mean age 63) were included, with survival to discharge observed in more than half of cases. The mean duration of the support was 9 days, and 70% of patients were successfully weaned from the device.</p>
<p>Jing Gao and coauthors [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>] studied the relationship between gut microbes, plasma trimethylamine N-oxide levels, and cardiovascular events in sixty participants, including patients with unstable angina, post-ST-segment elevation myocardial infarction, and healthy controls. Metagenomic sequencing and TMAO measurements revealed an association between specific gut microbial taxa and serum TMAO levels. A Student’s <em>t</em>-test showed that elevated serum TMAO levels were linked to a higher risk of cardiovascular events, with certain microbial taxa acting as potential biomarkers for acute coronary syndrome diagnosis.</p>
<p>Rachel B. Issaka et al. [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>] conducted a study to explore factors influencing colonoscopy completion after a positive fecal immunochemical test in a safety-net healthcare system. They analyzed data from over two thousand individuals aged between fifty and seventy-five who had an abnormal test result on a 3-year period. The study found that just over half of the patients completed their colonoscopy within one year of the positive test. The Student’s <em>t</em>-test was used to compare the colonoscopy completion rates between different groups based on these factors, finding significant differences in the completion rates. The study highlighted the need for improved follow-up and documentation practices to ensure timely colonoscopy completion.</p>
<p>Andrew C. Storm and coauthors [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>] conducted a study to assess the impact of implementing a new electronic health record system on patient safety and staff satisfaction in endoscopy suites. They compared procedure times and staff perceptions before and after the new system was introduced. The study found that procedure times significantly increased, and nurses spent less time directly monitoring patients after the implementation. The Student’s <em>t</em>-test was used to analyze changes in procedure time and monitoring time, revealing significant differences.</p>
<p>Jiao-Zhi Zhou et al. [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] conducted a study on the associations between workplace bullying, burnout, and depression among clinical nurses in China, surveying 415 nurses across nine hospitals in October 2023. The study found that 20% of participants exhibited depression symptoms, with the depression group scoring significantly higher on the Negative Acts Questionnaire than the control group. The Student’s <em>t</em>-test was used to compare the depression group and the control group across variables such as age, length of service, and professional title.</p>
<p>Guangda Wang and coauthors [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>] conducted a study involving 10 patients diagnosed with pre-malignant lesions and early-stage gastric cardia adenocarcinoma. The study found that in early-stage GCA, mutations in some of the genes were prevalent. The study also highlighted potential therapeutic targets based on genetic mutations, with 80% of patients showing promising treatment options. The Student’s <em>t</em>-test was applied to compare the tumor mutation burden between different cohorts, showing no statistically significant differences between the HB cohort and the TCGA GCA cohort for most mutations; the test revealed significant differences in specific mutations, such as those in the EPHA2 gene.</p></section><section id="FPar2"><h4 class="pmc_sec_title">Mann-Whitney <em>U</em> test</h4>
<p id="Par54">Tam M. Do et al. [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>] examined the relationship between dietary factors and breast cancer risk in Vietnamese women, using the Mann-Whitney <em>U</em> test to compare dietary intake and body mass index, between cases and controls. Significant differences were observed in the intake of vegetables, fruits, soybean products, coffee, and eggs, with higher consumption of these foods associated with lower breast cancer risk.</p>
<p>Brittany R. Lapin and coauthors [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] conducted a study involving over six thousand patients who completed patient-reported outcome measurements and satisfaction surveys at neurological clinics. The study found a significant positive association between PROM experience and visit satisfaction, with stronger effects observed among nonwhite patients, individuals with lower income, and those with more comorbidities. The Mann-Whitney <em>U</em> test was applied to compare PROMIS health scores and depression levels between patients who completed satisfaction surveys and those who did not, revealing significant differences in mental and physical health metrics between these groups.</p>
<p>Salvador Lugo-Perez et al. [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] conducted a study on rheumatoid arthritis patients without heart disease to investigate the relationship between antibody levels and left ventricular remodeling. The study revealed that patients with altered left ventricular geometry had significantly higher antibody titers, with anti-cyclic citrullinated peptide levels showing the strongest correlation with left ventricular remodeling. The Mann-Whitney <em>U</em> test was applied to compare echocardiographic variables and antibody levels between patients with and without left ventricular remodeling, highlighting statistically significant differences in these measures.</p>
<p>Ingo Steinbrück and coauthors [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>] conducted a randomized controlled trial involving multiple centers to compare the safety and outcomes of cold versus hot endoscopic mucosal resection for non-pedunculated colorectal polyps. The study demonstrated that cold resection resulted in significantly fewer major adverse events, such as perforation and post-endoscopic bleeding, compared to hot resection. The Mann-Whitney <em>U</em> test was applied to compare continuous variables, including polyp diameter and resection speed, between the two groups, revealing significant differences in adverse event rates and procedural outcomes.</p>
<p>Ying Qian et al. [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] conducted a study to evaluate the effectiveness of copper bianstone scraping combined with a Chinese modified hypertension dietary therapy program. The study included hundreds of hypertensive patients divided into a comparison group and an observation group. The results showed that the observation group had significantly greater improvements in blood pressure, body mass index, and blood glucose levels. The Mann-Whitney <em>U</em> test revealed significant differences between the groups, particularly in glycosylated hemoglobin, fasting blood glucose, and 2-hour postprandial blood glucose.</p>
<p>Jinghong Meng and coauthors [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] conducted a study on the incidence of surgical site infection in elective foot and ankle surgeries. More than a thousand of patients were included, and the 2% developed SSIs. The most common causative bacteria were pseudomonas aeruginosa and methicillin-resistant Staphylococcus aureus. Five factors were identified as independent risk factors for SSI: prolonged preoperative stay, use of allograft or bone substitute, elevated fasting blood glucose levels, low albumin levels, and abnormal neutrophil count. The Mann-Whitney <em>U</em> test was applied to data like age, hospital stay duration, and levels of albumin, fasting blood glucose, and other biological parameters, in order to compare the distributions between the groups with and without surgical site infection.</p>
<p>Brittany Lapin et al. [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] conducted a study to quantify the neurologic patient experience with patient-reported outcome measures across six neurology clinics. Over 16 thousands patients participated, completing both generic and condition-specific measures. The study identified that a majority of patients found the questions easy to understand, useful, and improved communication and control of their care. The Mann-Whitney <em>U</em> test was applied to compare the distributions of demographic factors, including age, income, and clinical characteristics, between groups of patients with varying experiences and satisfaction levels with measures.</p>
<p>Mitsuru Esaki and coauthors [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] conducted a study comparing endoscopic submucosal dissection outcomes in the postoperative stomach to those in the non-operative stomach for early gastric cancer, using propensity score matching to adjust for differences in baseline characteristics. Over one thousand patients were reviewed, with forty-one in the postoperative group and one thousand five in the non-operative group. The study found that the postoperative procedures required significantly more time but achieved similar rates of <em>en bloc</em>, complete, and curative resections with low adverse event rates. The Mann-Whitney <em>U</em> test was applied to compare procedure time and continuous variables like age, tumor size, and lesion depth.</p>
<p>Ausra Zelviene and Algirdas Boguseviciu [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] conducted a study to evaluate the reliability and validity of the revised Champion’s Health Belief Model Scale in measuring Lithuanian women’s beliefs about breast cancer and screening, using data from three hundred fifty women aged forty to sixty-nine, with no prior mammograms or breast cancer history. The Mann-Whitney <em>U</em> test was used to compare the responses between women who participated in mammography screenings and those who did not. The test revealed that women participating in mammography screenings perceived lower susceptibility and severity but reported reduced confidence in performing breast self-examinations, while non-participants exhibited more barriers to mammography and lower perceived benefits. In bioinformatics, the Mann-Whitney <em>U</em> test has been used also for gene set enrichment analysis [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>].</p></section><section id="FPar3"><h4 class="pmc_sec_title">Chi-squared test</h4>
<p id="Par63">Siyi Zhu et al. [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>] conducted a study to assess the influence of obesity on the clinical and pathological characteristics of breast cancer and its impact on endocrine therapy effectiveness among Chinese patients. They analyzed data from nearly three thousand patients with luminal or human epidermal growth factor receptor two-negative early breast cancer, dividing them into obese and non-obese groups based on body mass index. The Chi-squared test was used to compare categorical variables, such as menopausal status, comorbidities, expression of estrogen and progesterone receptors, and luminal subtypes, between the two groups. Results indicated that obese patients had significantly higher rates of comorbidities and progesterone receptor positivity but showed no differences in other characteristics. This study highlights how obesity influences breast cancer features and endocrine therapy outcomes.</p>
<p>Di Zhao and coauthors [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] conducted a study to investigate the association between high body mass index and breast cancer characteristics across different age groups, utilizing Chi-squared tests to analyze relationships between body mass index and variables such as tumor grade and HER2 positivity. The results showed that in patients younger than fifty-five years, high body mass index was significantly associated with HER2 positivity and worse progression-free survival, while in patients older than fifty-five years, high body mass index was linked to lower tumor grades.</p>
<p>José Pablo Leone et al. [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] conducted an observational study using data from a large cancer registry to examine the effects of the COVID-19 pandemic on breast cancer diagnosis, treatment patterns, and short-term mortality. They analyzed over thirty-seven thousand cases of ductal carcinoma in situ and almost two hundred thousand cases of invasive breast cancer between 2018 and 2020. The Chi-squared test was used to evaluate differences in treatment patterns, specifically the distribution of surgery, chemotherapy, and radiation therapy across the years studied. The results showed that during 2020, there was a significant decline in breast cancer diagnoses, particularly for early-stage cancers. Treatment patterns also shifted, with reduced use of surgery and radiation and increased use of chemotherapy. However, the twelve-month mortality rates did not differ significantly between 2020 and the previous years.</p>
<p>The RECOVERY Collaborative Group [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] conducted a randomised, controlled trial to evaluate the safety and efficacy of convalescent plasma therapy for patients with COVID-19. Patients were randomly assigned to receive either usual care or usual care plus high-titre convalescent plasma. The primary outcome, 28-day mortality, was not significantly different between the two groups, with no substantial differences in other clinical outcomes such as hospital discharge or progression to invasive mechanical ventilation. A Chi-squared test was used to analyze observed effects in subgroups based on characteristics at randomization, including age, sex, and presence of anti-SARS-CoV-2 antibodies.</p>
<p>Juan Jin and coauthors [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] conducted a study to examine the characteristics of HER2-low breast cancer in comparison with HER2-zero and HER2-positive tumors, with a focus on clinical and molecular traits. The analysis included clinical and genomic data of five hundred seventy-nine metastatic breast cancer patients, and the HER2-low subtype was identified as more commonly associated with hormone receptor-positive tumors. The results indicated that, despite the differences in metastasis patterns between HER2-low and HER2-positive patients in the hormone receptor-positive subgroup, the clinicopathological characteristics were largely influenced by hormone receptor status. Importantly, HER2-low tumors showed no significant differences in mutation alterations or copy number variations compared to HER2-zero tumors. Additionally, the study identified a higher prevalence of germline BRCA2 mutations in HER2-low patients. A Chi-squared test was employed to compare categorical variables, such as tumor characteristics and metastasis patterns, between the different HER2 subgroups.</p>
<p>Francesca Magnoni et al. [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] investigated the clinicopathological characteristics and outcomes of patients with pure and mixed invasive micropapillary breast cancer (IMPC) compared to invasive ductal cancer (IDC). The study included over thirty thousand IDC patients and more than two hundred IMPC patients. They found that both pure and mixed IMPC patients had a higher incidence of locally advanced disease and vascular invasion compared to IDC. After matching, pure IMPC patients showed worse overall survival and breast cancer-specific survival compared to IDC, whereas mixed IMPC patients had outcomes similar to IDC. The Chi-squared test was used to compare categorical variables, including tumor grade, lymph node status, vascular invasion, tumor molecular subtype, and type of surgery (mastectomy vs conservative surgery)</p>
<p>Kevser Tarı Selçuk and coauthors [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] conducted a study to determine the breast cancer screening behavior of women and investigate the relationship between health beliefs and screening behaviors. The study included 416 women aged forty and above. The results showed that the participation rates in breast self-examination, clinical breast examination, and mammography were relatively low. They found strong associations between perceived susceptibility, seriousness, self-efficacy, benefits, health motivation, and perceived barriers with screening behaviors. The Chi-squared test was used to compare screening behaviors with sociodemographic characteristics, and the variables considered in the analysis were age, education level, perceived economic status, and family history of breast cancer. The study also utilized multivariate binary logistic regression to examine how health beliefs influenced screening behaviors, and the Hosmer-Lemeshow test was applied to assess model fit.</p>
<p>Oindrila Bhattacharyya et al. [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] compared statewide health information exchange (HIE) data with survey self-report (SR) for measuring cancer screening. The study included Indiana residents and assessed screenings like colonoscopy, fecal immunochemical test, HPV and Pap tests, and mammography. Chi-squared tests were used to compare data on variables such as age, gender, and health status. Results showed fair to substantial concordance between HIE and SR, with higher sensitivity for procedures (for example, colonoscopy, mammography) and lower sensitivity for lab tests (for example, FIT, HPV). The Chi-squared test was used to evaluate differences in screening data.</p>
<p>Burcu Cengiz and coauthors [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] examined the effects of home-based nursing interventions, informed by the Health Belief Model, on individuals with stomas. The study included 30 participants in the experimental group and 31 in the control group. Data analysis involved Chi-squared, Mann-Whitney <em>U</em>, Wilcoxon, and Friedman tests to evaluate various outcomes. Results showed that home nursing interventions significantly improved compliance with stoma management and reduced complication rates in the experimental group compared to the control group, but no significant difference was found in quality-of-life scores. The nursing interventions led to significant cost savings for the experimental group.</p>
<p>Yirong Wu et al. [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>] developed prediction models using electronic health records to identify the “most harmful” breast cancers, focusing on variables like demographics, diagnoses, symptoms, procedures, medications, and laboratory results. They used Ridge Logistic Regression and Lasso Logistic Regression models, both of which showed strong performance with area under the ROC curve values of 0.818 and 0.839, respectively. Both models outperformed individual records components. The study also found that key features such as tobacco use and screening choices were associated with more harmful breast cancer cases.</p></section><section id="FPar4"><h4 class="pmc_sec_title">Kruskal-Wallis test</h4>
<p id="Par73">Jacob C. Clifton and coauthors [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] conducted a single-center retrospective study to evaluate the impact of an automated order reminder system on the timeliness of postanesthesia care unit order placement, analyzing over one hundred thousand surgical cases. Using the Kruskal-Wallis test, they assessed differences in procedural variables, including anesthesia duration, between patients receiving reminders and those who did not. The study found that reminders increased the likelihood of timely order placement and reduced delays in order submission, correlating with shorter PACU stays and improved pain management outcomes.</p>
<p>Mehrdad Karajizadeh et al. [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] conducted a study to identify the essential information needs for rapid response team electronic records in a major organ transplant center in Shiraz, Iran, through a cross-sectional survey of clinical roles. The Kruskal-Wallis test revealed significant differences among roles regarding data element priorities, and the Mann-Whitney <em>U</em> test highlighted distinctions specifically between registered anesthetist nurses and other groups.</p>
<p>Franklin Dexter and coauthors [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] analyzed workloads in veterinary dental clinics across forty-four workdays, using the Kruskal-Wallis test to compare anesthetist workloads among different days. They examined the workloads to identify significant variations in workload distribution and calculated upper confidence limits for workloads exceeding allocated times. The Kruskal-Wallis test was specifically applied to assess differences in workload data across various days, ensuring that variances in work schedules were adequately considered for accurate time allocation.</p>
<p>Irene L. Katzan et al. [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] conducted a survey to assess neurologic provider satisfaction with the systematic electronic collection of patient-reported outcome measures (PROMs), including disease-specific measures and depression screening using the Patient Health Questionnaire (PHQ-9). The survey was sent to two hundred ninety-nine staff physicians and advanced practice providers, with two hundred six responding. They used the Kruskal-Wallis test to evaluate differences in provider responses across age categories and provider types, and the Mann-Whitney <em>U</em> test to compare perceived usefulness between disease-specific information and the PHQ-9 depression screen. The results showed that PROM collection was deemed useful for patient care, research, and quality improvement, with respondents expressing similar perceptions of the clinical usefulness of both disease-specific information and the PHQ-9 depression screen. The Kruskal-Wallis test revealed no significant differences in the usefulness between the two types of data within each center, suggesting that both were similarly valued by providers.</p>
<p>Daniel Clay Williams and coauthors [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>] conducted a study to assess physician satisfaction with electronic health record (EHR) systems, particularly focusing on how they affect patient care. The study surveyed one hundred fifty-seven physicians at a quaternary care academic hospital. The results showed that overall satisfaction with the EHR system and its perceived impact on patient care were generally positive. The Kruskal-Wallis test was used for bivariate comparisons, and linear regression models identified that physician characteristics, including age and clinical role, were associated with EHR satisfaction. The perceived efficiency in using the EHR was found to be the most significant factor influencing overall satisfaction and the perception of its impact on patient care.</p>
<p>Karim Jabali et al. [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] conducted a study on anesthesiologists’ perceptions of Electronic Health Records (EHRs) in Saudi Arabia, focusing on their impact, benefits, ease of use, and anesthesia-specific features. Sixty-seven anesthesiologists participated, with younger and more experienced users showing more favorable views. The study found that EHRs were seen as beneficial to healthcare delivery, though job rank negatively impacted perceptions of ease of use. Specific anesthesiology features were generally well-received, although integration challenges were noted. The Kruskal-Wallis test was applied to assess the influence of demographic factors on perceptions, revealing significant differences between age groups, but no differences based on job rank or experience in most aspects of EHR use. Statistical tests showed that age and EHR experience significantly influenced perceptions.</p>
<p>Martha Fors and coauthors [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] conducted a study to evaluate sex-dependent differences in the contribution of the neutrophil-to-lymphocyte ratio, platelet-to-lymphocyte ratio, lymphocyte-to-monocyte ratio, and mean platelet volume-to-platelet ratio to the severity and mortality of COVID-19 upon hospital admission. They analyzed data from 3,280 confirmed COVID-19 cases in Quito, Ecuador, and identified differences in the hematology ratios between men and women. Severe COVID-19 pneumonia and non-surviving patients had higher levels of these ratios, with men showing higher values than women. The Kruskal-Wallis test was used to assess the association between these ratios and mortality and severity, revealing significant differences between the sexes. Dunn’s post hoc tests indicated differences in most groups, but no significant differences in NLR levels between surviving women and men. The results suggest that the hematology ratios could be useful biomarkers for predicting COVID-19 severity and mortality, with varying performances between men and women.</p>
<p>Luis E. Tollinche et al. [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>] conducted a study on the implementation of an automated Clinical Alert System for improving the documentation compliance of the Immediate Preoperative Assessment (IPOA) at Memorial Sloan Kettering Cancer Center. A retrospective analysis of 42,285 cases, spanning a year before and after the alert system implementation, showed that the compliance rate increased significantly from seventy-six percent to eighty-eight percent (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e784"><mrow><mi>p</mi><mo>&lt;</mo><mn>0.001</mn></mrow></math></span>). The analysis revealed that compliance was more prominent in surgeries that were shorter in duration and occurred earlier in the day. The study suggests that automated alerts can effectively enhance adherence to documentation standards, meeting regulatory requirements like those of the Centers for Medicare and Medicaid Services and The Joint Commission.</p>
<p>Sigal Shafran-Tikva and coauthors [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] conducted a study to investigate community-acquired pressure injuries (CAPIs) in older individuals using big data. Data were collected from over 44 thousand electronic medical records of patients admitted to internal medicine departments at two general medical centers between January 2016 and December 2018. The study found that the majority of CAPIs were located in the buttocks, with smaller occurrences in areas like the sacrum and ankle. Various tissue types, including necrotic and infected tissue, were identified. A significant portion of the CAPIs were second-degree, followed by first-degree and third-degree. The study revealed several patient characteristics associated with CAPIs, such as age, oxygen use, and albumin levels. Logistic regression was used to estimate the odds ratios of clinical variables for CAPIs, with a multivariate model highlighting variables that were clinically and statistically significant. The results underscore the importance of data-driven approaches in identifying and preventing CAPIs, particularly in elderly patients in community care settings.</p></section><section id="FPar5"><h4 class="pmc_sec_title">This study</h4>
<p id="Par82">This work is primarily intended for clinicians and biomedical researchers who do not have advanced statistical training and may lack the methodological tools to independently determine the appropriate use of these biostatistics univariate tests. Our aim is to equip these researchers with a structured and rigorous framework for selecting and distinguishing among the four tests examined, based on the nature of the data, the underlying research question, and key assumptions such as sample size and distributional properties</p>
<p>Even if some studies on guidelines to statistics exist [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>–<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>], to the best of our knowledge no article provides a handbook to the four univariate statistical tests provided here. We fill this gap with our investigation. The objective of this study is to provide clarity on the correct use of these four tests by explaining when each can be applied, for which types of data, and under what conditions (specifically, sample size and normal distribution) they yield the most powerful results. Specifically, we aim to help researchers select the most appropriate test to detect differences in means between groups or to assess the association between categorical variables.</p>
<p>We organize the rest of the manuscript as follows. After this Introduction, we explain the mathematical properties of the four tests in the <a href="#Sec3" class="usa-link">Methods</a> section, where we also offer guidance on which univariate statistical test to select based on the data types of the samples analyzed. Afterwards, we report and describe the results of the application of these tests on toy artificial data and on real-world medical data in the “<a href="#Sec14" class="usa-link">Results</a>” section. Eventually, we summarize and discuss the main findings of our study and describe some limitations and future directions in the “<a href="#Sec17" class="usa-link">Discussion and conclusions</a>” section.</p></section></section></section><section id="Sec3"><h2 class="pmc_sec_title">Methods</h2>
<p id="Par85">In this section, we introduce the formal definitions, mathematical properties, and assumptions underlying the statistical tests used in this analysis. We also discuss the calculation and interpretation of the <em>p</em>-value, as well as the conditions required for each test to be valid. Additionally, we examine the tests for normality, which help determine whether parametric or non-parametric tests are more appropriate. This information will allow us to better understand the strengths and limitations of each test in the context of the datasets analyzed.</p>
<section id="FPar6"><h3 class="pmc_sec_title">The null hypothesis is the uninteresting scenario</h3>
<p id="Par86">All the four tests produce probability values (called <em>p</em>-values) that can be used to confirm or reject the null hypothesis (denoted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e839"><msub><mi>H</mi><mn>0</mn></msub></math></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e850"><msub><mi>h</mi><mn>0</mn></msub></math></span>). A null hypothesis is a statement which says there is no effect or no difference between two samples or events, and it serves as a starting point for statistical testing.</p>
<p>We clarify the meaning of this concept through an example. Let us consider the scenario where a pharmaceutical company wants to test whether a new medication is effective in lowering cholesterol compared to a placebo. In this scenario, the null hypothesis would be that the new medication has no effect on cholesterol compared to the placebo.</p>
<p>In statistical terms, this can be expressed as: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e865"><msub><mi>H</mi><mn>0</mn></msub></math></span>: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e876"><mrow><msub><mi>μ</mi><mrow><mi mathvariant="italic">drug</mi></mrow></msub><mo>=</mo><msub><mi>μ</mi><mrow><mi mathvariant="italic">placebo</mi></mrow></msub></mrow></math></span></p>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e897"><msub><mi>μ</mi><mrow><mi mathvariant="italic">drug</mi></mrow></msub></math></span> represents the mean cholesterol level of the group taking the new drug and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e910"><msub><mi>μ</mi><mrow><mi mathvariant="italic">placebo</mi></mrow></msub></math></span> the mean cholesterol level of the group taking the placebo. In this study, the researchers would collect cholesterol data from two groups (one receiving the new drug and the other receiving the placebo) and perform a statistical test (one of the four tests described in this study) to determine if there is enough evidence to reject the null hypothesis in favor of an alternative hypothesis, which would suggest that the new drug does have an effect on lowering cholesterol. That means, if the <em>p</em>-value produced by the statistical test applied to the cholesterol levels of the group taking the new drug (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e926"><mrow><msub><mi>μ</mi><mrow><mi mathvariant="italic">drug</mi></mrow></msub><mrow><mo stretchy="false">)</mo></mrow></mrow></math></span> and the cholesterol levels of the group taking the placebo (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e943"><mrow><msub><mi>μ</mi><mrow><mi mathvariant="italic">placebo</mi></mrow></msub><mrow><mo stretchy="false">)</mo></mrow></mrow></math></span> is lower than the significance 0.005 threshold [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>], we can reject the hypothesis that the new medical has no effect on cholesterol compared to the placebo.</p>
<p>In pseudo-code, we can write the previous claim this way:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par91">p_value <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e970"><mo stretchy="false">←</mo></math></span> test(drug_group.cholesterol_levels, placebo_group.cholesterol_levels)</p></li>
<li><p id="Par92">if(p_value &lt; 0.005) then { null_hypothesis <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e983"><mo stretchy="false">←</mo></math></span> false }</p></li>
</ul>
<p>In turn, this statement means that we can accept the alternative hypothesis saying that there the new medicine has some effect on lowering cholesterol compared to the placebo. On the contrary, if the <em>p</em>-value were higher than the 0.005 threshold, we could not reject the null hypothesis (we will explain significance thresholds later on more precisely).</p>
<p>One can think of null hypotheses as uninteresting, irrelevant, normal scenarios [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>].</p></section><section id="Sec4"><h3 class="pmc_sec_title">Properties and theoretical aspects of statistical tests</h3>
<p id="Par95">In this section, we explain the theory beyond the four statistical tests analyzed in this study.</p>
<section id="Sec5"><h4 class="pmc_sec_title">Student’s <em>t</em>-test</h4>
<p id="Par96">The Student’s <em>t</em>-test is a statistical test used to verify the statistical significance of the difference in responses between two groups. It is based on the Student’s <em>t</em>-distribution, used with small sample sizes and a reference population with a normal (or approximately normal) distribution. The test compares the means of two groups of data and evaluates whether the difference between them is statistically significant relative to the variability within the groups themselves.</p>
<p id="Par97">The test provides a <em>t</em>-value and an associated <em>p</em>-value: the <em>t</em>-value represents how large the difference between the means of the two groups is relative to the variability within the groups; the <em>p</em>-value represents the probability that the observed difference is due to chance, assuming there is no real difference between the group means. If the <em>p</em>-value is low (in our case, less than 0.005), it indicates that the difference between the means is statistically significant.</p>
<p id="Par98">Assumptions:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par99">One-sample <em>t</em>-test: requires one sample and a reference value.</p></li>
<li><p id="Par100">Independent samples <em>t</em>-test: requires two independent samples. The variances must be approximately equal, which can be checked using Levene’s test [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>].</p></li>
<li><p id="Par101">Paired samples <em>t</em>-test: requires paired samples.</p></li>
</ul>
<p>We can evaluate differences only on metrics, not on categorical values.</p>
<p id="Par102">Hypotheses:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par103">One-sample <em>t</em>-test: Null hypothesis: the sample mean is equal to the reference value.</p></li>
<li><p id="Par104">Independent samples <em>t</em>-test: Null hypothesis: the means of the two samples are equal.</p></li>
<li><p id="Par105">Paired samples <em>t</em>-test: Null hypothesis: the mean difference between paired groups is zero.</p></li>
</ul>
<p id="Par106">How to calculate the <em>t</em>-value:</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><math id="d33e1089" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mi>t</mi><mo>=</mo><mfrac><mrow><mtext>difference between sample means</mtext></mrow><mrow><mtext>standard error (standard deviation from the mean)</mtext></mrow></mfrac></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par107">One-sample <em>t</em>-test: <table class="disp-formula p" id="Equ2"><tr><td class="formula"><math id="d33e1115" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mi>t</mi><mo>=</mo><mfrac><mrow><mi>X</mi><mo>-</mo><mi>μ</mi></mrow><mrow><mi>S</mi><mo stretchy="false">/</mo><msqrt><mi>n</mi></msqrt></mrow></mfrac></mrow></mtd></mtr></mtable></mrow></math></td></tr></table> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e1141"><mi>X</mi></math></span> is the sample mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e1150"><mi>μ</mi></math></span> is the known mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e1159"><mi>S</mi></math></span> is the standard deviation of the data, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e1169"><mi>n</mi></math></span> is the sample size. We recall that, in the previous formula, the denominator division is computed before the general division.</div></li>
<li><div class="p" id="Par108">Independent samples <em>t</em>-test: <table class="disp-formula p" id="Equ3"><tr><td class="formula"><math id="d33e1184" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mi>t</mi><mo>=</mo><mfrac><mrow><msub><mover accent="true"><mrow><mi>X</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mn>1</mn></msub><mo>-</mo><msub><mover accent="true"><mrow><mi>X</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mn>2</mn></msub></mrow><msqrt><mfenced close=")" open="("><mfrac><msubsup><mi>s</mi><mn>1</mn><mn>2</mn></msubsup><msub><mi>n</mi><mn>1</mn></msub></mfrac><mo>+</mo><mfrac><msubsup><mi>s</mi><mn>2</mn><mn>2</mn></msubsup><msub><mi>n</mi><mn>2</mn></msub></mfrac></mfenced></msqrt></mfrac></mrow></mtd></mtr></mtable></mrow></math></td></tr></table> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e1237"><msub><mover accent="true"><mrow><mi>X</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mi>i</mi></msub></math></span> is the sample mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e1253"><msub><mi>s</mi><mi>i</mi></msub></math></span> is the standard deviation, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e1264"><msub><mi>n</mi><mi>i</mi></msub></math></span> is the sample size for group <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e1276"><mi>i</mi></math></span>.</div></li>
<li><div class="p" id="Par109">Paired samples <em>t</em>-test: <table class="disp-formula p" id="Equ4"><tr><td class="formula"><math id="d33e1291" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mi>t</mi><mo>=</mo><mfrac><mover accent="true"><mrow><mi>X</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mrow><mi>S</mi><mo stretchy="false">/</mo><msqrt><mi>n</mi></msqrt></mrow></mfrac></mrow></mtd></mtr></mtable></mrow></math></td></tr></table> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e1319"><mover accent="true"><mrow><mi>X</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover></math></span> is the mean of the differences, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e1333"><mi>S</mi></math></span> is the standard deviation of the differences, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e1342"><mi>n</mi></math></span> is the number of pairs.</div></li>
</ul>
<p id="Par110">Results of the <em>t</em>-statistic: The higher the <em>t</em>-value, the greater the difference between the means of the samples (the same applies in reverse). The greater the dispersion of the mean, that is, the standard error, the less significant the difference in the means between the data.</p>
<p id="Par111">To accept or reject the null hypothesis, we can either consult a critical <em>t</em>-value table or calculate the <em>p</em>-value.</p>
<p id="Par112">The <em>p</em>-value is the probability of observing the experimental data or the test results, assuming the null hypothesis is true. If the <em>p</em>-value is smaller than a chosen threshold (in our case, 0.005), it means we have sufficient evidence to reject the null hypothesis and conclude that there is a difference between the two groups.</p>
<p id="Par113">Regarding the <em>t</em>-value table, we choose a significance level (0.005), then look at the value for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e1380"><mrow><mn>1</mn><mo>-</mo><mn>0.005</mn><mo>=</mo><mn>0.995</mn></mrow></math></span>. Next, we choose the degrees of freedom (<em>df</em>). For one-sample and paired tests, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e1397"><mrow><mi>d</mi><mi>f</mi><mo>=</mo><mi>n</mi><mo>-</mo><mn>1</mn></mrow></math></span>, while for independent samples, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e1412"><mrow><mi>d</mi><mi>f</mi><mo>=</mo><msub><mi>n</mi><mn>1</mn></msub><mo>+</mo><msub><mi>n</mi><mn>2</mn></msub><mo>-</mo><mn>2</mn></mrow></math></span>.</p>
<p id="Par114">If the calculated <em>t</em>-value is greater than the critical <em>t</em>-value, we reject the null hypothesis.</p>
<p id="Par115">We also need to consider the difference between direct and indirect hypotheses. In an indirect hypothesis, if we verify that there is a difference between the means of two groups, we do not know which group has a higher mean than the other. In a direct hypothesis, we know this information (Table <a href="#Tab1" class="usa-link">1</a>).
</p>
<section class="tw xbox font-sm" id="Tab1"><h5 class="obj_head">Table 1.</h5>
<div class="caption p"><p>Differentiated Student’s <em>t</em>-tests. df: degrees of freedom. The degree of freedom in statistical tests refers to the number of independent values or quantities that can vary in an analysis without breaking any constraints</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Sample</th>
<th align="left" colspan="1" rowspan="1">Difference</th>
<th align="left" colspan="1" rowspan="1">Hypothesis</th>
<th align="left" colspan="1" rowspan="1">Formula</th>
<th align="left" colspan="1" rowspan="1">DF</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">one sample</td>
<td align="left" colspan="1" rowspan="1">one sample with a known reference value <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e1481"><mi>μ</mi></math></span>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e1493"><mrow><msub><mi>h</mi><mn>0</mn></msub><mtext>:</mtext><mspace width="0.333333em"></mspace><mi>μ</mi><mo>=</mo><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mo>,</mo><mspace width="0.166667em"></mspace><msub><mi>h</mi><mn>1</mn></msub><mtext>:</mtext><mspace width="0.333333em"></mspace><mi>μ</mi><mo>≠</mo><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e1534"><mrow><mi>t</mi><mo>=</mo><mfrac><mrow><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mo>-</mo><mi>μ</mi></mrow><mrow><mi>s</mi><mo stretchy="false">/</mo><msqrt><mi>n</mi></msqrt></mrow></mfrac></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e1564"><mrow><mi>n</mi><mo>-</mo><mn>1</mn></mrow></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">independent samples</td>
<td align="left" colspan="1" rowspan="1">two independent samples with similar variances</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e1584"><mrow><msub><mi>h</mi><mn>0</mn></msub><mtext>:</mtext><mspace width="0.333333em"></mspace><msub><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mn>1</mn></msub><mo>=</mo><msub><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mn>2</mn></msub><mo>,</mo><mspace width="0.166667em"></mspace><msub><mi>h</mi><mn>1</mn></msub><mtext>:</mtext><mspace width="0.333333em"></mspace><msub><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mn>1</mn></msub><mo>≠</mo><msub><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mn>2</mn></msub></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e1643"><mrow><mi>t</mi><mo>=</mo><mfrac><mrow><msub><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mn>1</mn></msub><mo>-</mo><msub><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mn>2</mn></msub></mrow><msqrt><mfenced close=")" open="("><mfrac><msubsup><mi>s</mi><mn>1</mn><mn>2</mn></msubsup><msub><mi>n</mi><mn>1</mn></msub></mfrac><mo>+</mo><mfrac><msubsup><mi>s</mi><mn>2</mn><mn>2</mn></msubsup><msub><mi>n</mi><mn>2</mn></msub></mfrac></mfenced></msqrt></mfrac></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e1695"><mrow><msub><mi>n</mi><mn>1</mn></msub><mo>+</mo><msub><mi>n</mi><mn>2</mn></msub><mo>-</mo><mn>2</mn></mrow></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">paired samples</td>
<td align="left" colspan="1" rowspan="1">differences in values from the same sample at different times <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e1719"><mi>δ</mi></math></span>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e1731"><mrow><msub><mi>h</mi><mn>0</mn></msub><mtext>:</mtext><mspace width="0.333333em"></mspace><mi>δ</mi><mo>=</mo><mn>0</mn><mo>,</mo><mspace width="0.166667em"></mspace><msub><mi>h</mi><mn>1</mn></msub><mtext>:</mtext><mspace width="0.333333em"></mspace><mi>δ</mi><mo>≠</mo><mn>0</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e1762"><mrow><mi>t</mi><mo>=</mo><mfrac><mover accent="true"><mrow><mi>x</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mrow><mi>s</mi><mo stretchy="false">/</mo><msqrt><mi>n</mi></msqrt></mrow></mfrac></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e1789"><mrow><mi>n</mi><mo>-</mo><mn>1</mn></mrow></math></span>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par116">For these different types of the Student’s <em>t</em>-test, it is possible to calculate the values that the <em>t</em>-statistic tends toward as the variables change (boundary values). Recall the definition of the variables involved in the calculation:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par117"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e1812"><mover accent="true"><mrow><mi>X</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover></math></span>: sample mean</p></li>
<li><p id="Par118"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e1828"><mi>μ</mi></math></span>: known mean (only for the one-sample <em>t</em>-test)</p></li>
<li><p id="Par119"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e1842"><mi>S</mi></math></span>: standard deviation</p></li>
<li><p id="Par120"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e1853"><mi>n</mi></math></span>: sample size</p></li>
</ul>
<p id="Par121">In general, we note the following common limits:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par122"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e1867"><mrow><mi>S</mi><mo stretchy="false">→</mo><mn>0</mn></mrow></math></span>: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e1880"><mrow><mi>t</mi><mo stretchy="false">→</mo><mi>∞</mi></mrow></math></span>: a very small standard deviation erroneously suggests strong statistical significance.</p></li>
<li><p id="Par123"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e1895"><mrow><mi>n</mi><mo stretchy="false">→</mo><mi>∞</mi></mrow></math></span>: very large samples make even very small differences significant.</p></li>
<li><p id="Par124"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e1910"><mrow><mi>n</mi><mo stretchy="false">→</mo><mn>0</mn></mrow></math></span>: very small samples make it difficult to obtain significance even with large differences.</p></li>
</ul></section><section id="Sec6"><h4 class="pmc_sec_title">Mann-Whitney <em>U</em> test</h4>
<p id="Par125">The Mann-Whitney <em>U</em> test, also known as the Wilcoxon-Mann-Whitney test, is a non-parametric statistical test used to evaluate the significance of differences between two independent groups. Unlike the Student’s <em>t</em>-test, it does not require the assumption of normality in the underlying population distributions, making it suitable for data that may not follow a normal distribution or when sample sizes are small.</p>
<p id="Par126">The test compares the ranks of the data rather than their raw values, assessing whether one group tends to have systematically higher or lower ranks than the other. It is particularly effective for ordinal or continuous data that violate parametric assumptions.</p>
<p id="Par127">The test is based on the following assumptions:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par128">The two samples are independent.</p></li>
<li><p id="Par129">Observations are ordinal or continuous.</p></li>
<li><p id="Par130">The distributions of the two groups are similar in shape; otherwise, the test may capture differences in shape or dispersion in addition to central tendency.</p></li>
</ul>
<p id="Par131">The hypotheses for the Mann-Whitney <em>U</em> test are:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par132">Null hypothesis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e1960"><msub><mi>H</mi><mn>0</mn></msub></math></span>): The distributions of the two groups are identical, implying no systematic difference in ranks.</p></li>
<li><p id="Par133">Alternative hypothesis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e1974"><msub><mi>H</mi><mn>1</mn></msub></math></span>): The distributions of the two groups differ, indicating a systematic rank difference between groups.</p></li>
</ul>
<p id="Par134">The test produces the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e1987"><mi>U</mi></math></span> statistic, which represents the sum of ranks assigned to one of the groups, adjusted for the sample sizes. This statistic is then converted into a <em>p</em>-value to assess significance.</p>
<p id="Par135">The test statistic <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e2001"><mi>U</mi></math></span> is computed through the following formula:</p>
<table class="disp-formula p" id="Equ5"><tr><td class="formula"><math id="d33e2010" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mi>U</mi><mo>=</mo><msub><mi>n</mi><mn>1</mn></msub><mo>·</mo><msub><mi>n</mi><mn>2</mn></msub><mo>+</mo><mfrac><mrow><msub><mi>n</mi><mn>1</mn></msub><mo>·</mo><mrow><mo stretchy="false">(</mo><msub><mi>n</mi><mn>1</mn></msub><mo>+</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></mrow><mn>2</mn></mfrac><mo>-</mo><msub><mi>R</mi><mn>1</mn></msub></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<p>where:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par136"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e2057"><msub><mi>n</mi><mn>1</mn></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><math id="d33e2068"><msub><mi>n</mi><mn>2</mn></msub></math></span> are the sample sizes of the two groups.</p></li>
<li><p id="Par137"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><math id="d33e2081"><msub><mi>R</mi><mn>1</mn></msub></math></span> is the sum of ranks in the first group [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>].</p></li>
</ul>
<p id="Par138">Regarding interpretation, it is worth noticing that a smaller <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><math id="d33e2097"><mi>U</mi></math></span> value indicates a larger difference between groups. Additionally, the <em>p</em>-value derived from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><math id="d33e2109"><mi>U</mi></math></span> determines whether the observed rank differences are statistically significant. A <em>p</em>-value below the significance level (for instance, 0.005) leads to rejecting <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><math id="d33e2121"><msub><mi>H</mi><mn>0</mn></msub></math></span>, supporting that the groups differ significantly.</p>
<p id="Par139">This test has several key features: robustness, since the test does not assume equal variances or normality in the data; applicability, since this test is suitable for both small and large sample sizes but requires independent observations between groups; and flexibility, since this test can handle ordinal data and non-normal continuous data [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>].</p>
<p id="Par140">The <em>U</em> test presents several advantages. It is applicable to skewed data or data with outliers, it is Suitable for small sample sizes, and it does not assume homogeneity of variances.</p>
<p id="Par141">However, this test has some drawbacks as well: it is Less powerful than the <em>t</em>-test for normal distributions, and it may lose sensitivity when data contains ties or ranks overlap significantly [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>].</p></section><section id="Sec7"><h4 class="pmc_sec_title">Chi-squared test</h4>
<p id="Par142">The Chi-squared test is a statistical method used to evaluate the differences between observed and expected frequencies in categorical data. It was first introduced by Karl Pearson in 1900 as a measure of discrepancy between observed and theoretical distributions. The test is widely applied in contingency table analysis and goodness-of-fit testing [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>].</p>
<p id="Par143">The Chi-squared statistic is calculated as:</p>
<table class="disp-formula p" id="Equ6"><tr><td class="formula"><math id="d33e2159" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><msup><mi>χ</mi><mn>2</mn></msup><mo>=</mo><munderover><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mfrac><msup><mrow><mo stretchy="false">(</mo><msub><mi>O</mi><mi>i</mi></msub><mo>-</mo><msub><mi>E</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><msub><mi>E</mi><mi>i</mi></msub></mfrac></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<p>where:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par144"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><math id="d33e2205"><msub><mi>O</mi><mi>i</mi></msub></math></span> represents the observed frequency in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><math id="d33e2216"><mi>i</mi></math></span>-th category,</p></li>
<li><p id="Par145"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><math id="d33e2227"><msub><mi>E</mi><mi>i</mi></msub></math></span> represents the expected frequency in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><math id="d33e2238"><mi>i</mi></math></span>-th category under the null hypothesis [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>].</p></li>
</ul>
<p id="Par146">The Chi-squared test has several relevant characteristics. First, it evaluates whether the observed frequencies significantly deviate from the expected frequencies. The null hypothesis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><math id="d33e2252"><msub><mi>H</mi><mn>0</mn></msub></math></span>) states that the observed frequencies match the expected frequencies. Moreover, the alternative hypothesis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><math id="d33e2263"><msub><mi>H</mi><mn>1</mn></msub></math></span>) states that the observed frequencies differ from the expected frequencies.</p>
<p id="Par147">This test makes some assumptions as well. Expected frequencies should be sufficiently large, typically at least 5 in each category, to ensure the validity of the test. Observations must be independent, meaning that each data point should not influence or depend on others. Data must be derived from random samples representing the population of interest.</p>
<p id="Par148">The Chi-squared statistic follows a Chi-squared distribution with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><math id="d33e2278"><mrow><mi>k</mi><mo>-</mo><mn>1</mn></mrow></math></span> degrees of freedom, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><math id="d33e2290"><mi>k</mi></math></span> is the number of categories. The critical value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><math id="d33e2299"><msup><mi>χ</mi><mn>2</mn></msup></math></span> depends on the chosen significance threshold (for instance, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><math id="d33e2310"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span>) and the degrees of freedom.</p>
<p id="Par149">Regarding interpretation, we can state that a high <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><math id="d33e2324"><msup><mi>χ</mi><mn>2</mn></msup></math></span> value indicates a large difference between observed and expected frequencies. Additionally If the <em>p</em>-value associated with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><math id="d33e2338"><msup><mi>χ</mi><mn>2</mn></msup></math></span> is less than the significance level, the null hypothesis is rejected, suggesting significant differences. Conversely, if the <em>p</em>-value is greater than the significance level, there is insufficient evidence to reject the null hypothesis.</p>
<p id="Par150">This Chi-squared test suffers from some limitations, too. When <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><math id="d33e2354"><mrow><msub><mi>O</mi><mi>i</mi></msub><mo>=</mo><msub><mi>E</mi><mi>i</mi></msub></mrow></math></span> for all <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><math id="d33e2370"><mi>i</mi></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><math id="d33e2379"><mrow><msup><mi>χ</mi><mn>2</mn></msup><mo>=</mo><mn>0</mn></mrow></math></span>, indicating no difference between observed and expected frequencies. As <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><math id="d33e2393"><mrow><mo stretchy="false">(</mo><msub><mi>O</mi><mi>i</mi></msub><mo>-</mo><msub><mi>E</mi><mi>i</mi></msub><mo stretchy="false">)</mo><mo stretchy="false">→</mo><mi>∞</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><math id="d33e2416"><mrow><msup><mi>χ</mi><mn>2</mn></msup><mo stretchy="false">→</mo><mi>∞</mi></mrow></math></span>, reflecting increasing discrepancies. If <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><math id="d33e2432"><msub><mi>E</mi><mi>i</mi></msub></math></span> values are too small, the Chi-squared statistic may not follow the theoretical Chi-squared distribution, compromising the test’s reliability [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>].</p></section><section id="Sec8"><h4 class="pmc_sec_title">Kruskal-Wallis test</h4>
<p id="Par151">The Kruskal-Wallis test, also called Kruskal-Wallis <em>H</em>-test sometimes, is a nonparametric statistical procedure used to assess whether there are significant differences among three or more independent groups based on ordinal or continuous dependent data. It is often considered an extension of the Mann-Whitney <em>U</em> test, which is limited to comparing two groups. The Kruskal-Wallis test evaluates whether the distributions of ranks within groups are statistically equivalent or if at least one group deviates significantly from the others; this test is particularly useful for comparing independent groups when the assumptions of one-way ANOVA test are not met, such as when data are non-normal or when sample sizes are unequal [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>].</p>
<p id="Par152">The test operates on the following principles:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par153">Observations within each group are ranked across the entire dataset, assigning rank 1 to the smallest value, rank 2 to the next smallest, and so on.</p></li>
<li><p id="Par154">The test statistic <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><math id="d33e2471"><mi>H</mi></math></span> is computed using the sum of ranks for each group and measures the degree to which the ranks differ among groups.</p></li>
<li><p id="Par155">The null hypothesis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><math id="d33e2483"><msub><mi>H</mi><mn>0</mn></msub></math></span>) states that all independent groups share the same central tendency, indicating they come from the same population distribution.</p></li>
<li><p id="Par156">The alternative hypothesis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><math id="d33e2497"><msub><mi>H</mi><mn>1</mn></msub></math></span>) posits that at least one group exhibits a different central tendency, suggesting it originates from a distinct population distribution.</p></li>
</ul>
<p id="Par157">Assumptions of the Kruskal-Wallis test include:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par158">Observations within groups must be independent.</p></li>
<li><p id="Par159">Groups must have distributions with the same shape, though not necessarily the same median.</p></li>
<li><p id="Par160">Data must be at least ordinal and originate from random independent samples.</p></li>
</ul>
<p id="Par161">The test statistic <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><math id="d33e2522"><mi>H</mi></math></span> is calculated as:</p>
<table class="disp-formula p" id="Equ7"><tr><td class="formula"><math id="d33e2531" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mi>H</mi><mo>=</mo><mfrac><mrow><mo stretchy="false">(</mo><mi>n</mi><mo>-</mo><mn>1</mn><mo stretchy="false">)</mo></mrow><mn>12</mn></mfrac><munderover><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>k</mi></munderover><mfrac><mrow><msub><mi>n</mi><mi>i</mi></msub><msup><mfenced close=")" open="("><msub><mover accent="true"><mrow><mi>R</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mi>i</mi></msub><mo>-</mo><msub><mi>E</mi><mi>R</mi></msub></mfenced><mn>2</mn></msup></mrow><msubsup><mi>σ</mi><mi>R</mi><mn>2</mn></msubsup></mfrac></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<p>where:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par162"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><math id="d33e2591"><mi>n</mi></math></span> is the total number of observations,</p></li>
<li><p id="Par163"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><math id="d33e2602"><mi>k</mi></math></span> is the number of groups,</p></li>
<li><p id="Par164"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><math id="d33e2613"><msub><mi>n</mi><mi>i</mi></msub></math></span> is the sample size of group <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><math id="d33e2624"><mi>i</mi></math></span>,</p></li>
<li><p id="Par165"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq81"><math id="d33e2635"><msub><mover accent="true"><mrow><mi>R</mi></mrow><mrow><mo stretchy="false">¯</mo></mrow></mover><mi>i</mi></msub></math></span> is the average rank for group <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq82"><math id="d33e2651"><mi>i</mi></math></span>,</p></li>
<li><p id="Par166"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq83"><math id="d33e2662"><msub><mi>E</mi><mi>R</mi></msub></math></span> is the expected rank under the null hypothesis, given by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq84"><math id="d33e2673"><mrow><msub><mi>E</mi><mi>R</mi></msub><mo>=</mo><mfrac><mrow><mi>n</mi><mo>+</mo><mn>1</mn></mrow><mn>2</mn></mfrac></mrow></math></span>,</p></li>
<li><p id="Par167"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq85"><math id="d33e2694"><msubsup><mi>σ</mi><mi>R</mi><mn>2</mn></msubsup></math></span> is the rank variance, calculated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq86"><math id="d33e2706"><mrow><msubsup><mi>σ</mi><mi>R</mi><mn>2</mn></msubsup><mo>=</mo><mfrac><mrow><msup><mi>n</mi><mn>2</mn></msup><mo>-</mo><mn>1</mn></mrow><mn>12</mn></mfrac></mrow></math></span>.</p></li>
</ul>
<p id="Par168">Degrees of freedom for the test are defined as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq87"><math id="d33e2730"><mrow><mtext>df</mtext><mo>=</mo><mi>k</mi><mo>-</mo><mn>1</mn></mrow></math></span>. The computed <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq88"><math id="d33e2744"><mi>H</mi></math></span> statistic is compared against the critical value from the Chi-squared distribution table with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq89"><math id="d33e2753"><mtext>df</mtext></math></span> degrees of freedom at the desired significance level.</p>
<p id="Par169">Regarding interpretation, if the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq90"><math id="d33e2764"><mi>H</mi></math></span> statistic exceeds the critical value, or if the <em>p</em>-value is less than <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq91"><math id="d33e2776"><mi>α</mi></math></span>, the null hypothesis is rejected, indicating significant differences among the groups. If <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq92"><math id="d33e2785"><mi>H</mi></math></span> is below the critical value, or the <em>p</em>-value is greater than <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq93"><math id="d33e2798"><mi>α</mi></math></span>, there is insufficient evidence to reject the null hypothesis.</p>
<p id="Par170">Of course, the <em>H</em> statistic should not be confused with the null hypothesis (usually indicated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq94"><math id="d33e2812"><msub><mi>H</mi><mn>0</mn></msub></math></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq95"><math id="d33e2823"><msub><mi>h</mi><mn>0</mn></msub></math></span>).</p>
<p id="Par171">This Kruskal-Wallis test also presents several limitations. When the distributions of all groups are identical, the test statistic <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq96"><math id="d33e2836"><mi>H</mi></math></span> is close to zero, indicating no difference in medians. As the rank differences between groups increase, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq97"><math id="d33e2845"><mrow><mi>H</mi><mo stretchy="false">→</mo><mi>∞</mi></mrow></math></span>, reflecting stronger evidence against the null hypothesis [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. However, when sample sizes are small or unequal, or when there are many tied ranks, the approximation of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq98"><math id="d33e2861"><mi>H</mi></math></span> to a chi-squared distribution becomes less accurate, potentially reducing the test’s reliability [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>].</p></section></section><section id="Sec9"><h3 class="pmc_sec_title">Python software packages</h3>
<p id="Par172">In our computational analyses, we utilized several Python packages to handle data manipulation, statistical testing, and visualization:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par173">Pandas: for data manipulation and analysis (reading datasets, handling dataframes, and cleaning data) [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>];</p></li>
<li><p id="Par174">SciPy: for Student’s <em>t</em>, Mann-Whitney <em>U</em> and Chi-squared tests [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>];</p></li>
<li><p id="Par175">Matplotlib: to create bar charts displaying the <em>p</em>-values from statistical tests [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>];</p></li>
<li><p id="Par176">NumPy: for normality testing (Shapiro-Wilk test and Kolmogorov-Smirnov test) [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>].</p></li>
</ul>
<p id="Par177">We decided to use the Python programming language because it is free and open source, allowing the universal reproducibility of our experiments [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>, <a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. Additionally, Python is considered the most widely used programming language in the world, by the PYPL index of July 2025 [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>], the TIOBE index of July 2025 [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>] and the 2022 Kaggle survey [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>].</p></section><section id="Sec10"><h3 class="pmc_sec_title">Summary of the four tests considered</h3>
<p id="Par178">We can recap the meanings of the four statistical tests considered this way. The Student’s <em>t</em>-statistic measures the standardized difference between two group means: a value of zero indicates no difference, while larger positive or negative values suggest greater separation. The Mann-Whitney <em>U</em> statistic compares the ranks of values from two independent groups: values near zero indicate minimal difference, and values approaching the product of the group sizes indicate strong ordering in favor of one group. The Chi-squared statistic compares observed and expected frequencies in contingency tables; it starts at zero when distributions match perfectly and increases with greater discrepancies. Similarly, the Kruskal-Wallis H statistic evaluates whether rank distributions differ across multiple groups, starting at zero when distributions are identical and increasing with larger differences. Higher values of these statistics generally provide stronger evidence against the null hypothesis, but their significance must be evaluated using corresponding <em>p</em>-values and degrees of freedom: these values are not <em>p</em>-values, but rather the raw statistics calculated during the test. We recap the properties of the four analyzed statistical tests in Table <a href="#Tab2" class="usa-link">2</a> and their assumptions and limitations in Table <a href="#Tab3" class="usa-link">3</a>.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Summary of statistical tests analyzed. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq106"><math id="d33e3107"><msub><mi>n</mi><mn>1</mn></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq107"><math id="d33e3118"><msub><mi>n</mi><mn>2</mn></msub></math></span> are the sample sizes of the two groups in the comparison. Normality required: Shapiro-Wilk test significant for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq108"><math id="d33e3129"><mrow><mi>n</mi><mo>&lt;</mo><mn>50</mn></mrow></math></span> or Kolmogorov-Smirnov test significant for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq109"><math id="d33e3141"><mrow><mi>n</mi><mo>≥</mo><mn>50</mn></mrow></math></span></p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Test</th>
<th align="left" colspan="1" rowspan="1">Data types for both samples</th>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">Normality required</th>
<th align="left" colspan="1" rowspan="1">Output</th>
<th align="left" colspan="1" rowspan="1">purpose</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Student’s <em>t</em>
</td>
<td align="left" colspan="1" rowspan="1">numerical</td>
<td align="left" colspan="1" rowspan="1">means of 2 groups</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq110"><math id="d33e3189"><mrow><mi>t</mi><mo>∈</mo><mo stretchy="false">(</mo><mo>-</mo><mi>∞</mi><mo>;</mo><mo>+</mo><mi>∞</mi><mo stretchy="false">)</mo></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">differences between group means</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mann-Whitney <em>U</em>
</td>
<td align="left" colspan="1" rowspan="1">numerical, ordinal, or categorical</td>
<td align="left" colspan="1" rowspan="1">ranks of 2 groups</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq111"><math id="d33e3226"><mrow><mi>U</mi><mo>∈</mo><mo stretchy="false">[</mo><mn>0</mn><mo>;</mo><msub><mi>n</mi><mn>1</mn></msub><mo>·</mo><msub><mi>n</mi><mn>2</mn></msub><mo stretchy="false">]</mo></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">compare ranks between groups. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq112"><math id="d33e3253"><msub><mi>n</mi><mn>1</mn></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq113"><math id="d33e3264"><msub><mi>n</mi><mn>2</mn></msub></math></span> are the sample sizes of the two groups in the comparison</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Chi-squared</td>
<td align="left" colspan="1" rowspan="1">categorical</td>
<td align="left" colspan="1" rowspan="1">observed versus expected frequencies</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq114"><math id="d33e3287"><mrow><msup><mi>χ</mi><mn>2</mn></msup><mo>∈</mo><mrow><mo stretchy="false">[</mo><mn>0</mn><mo>;</mo><mo>+</mo><mi>∞</mi><mo stretchy="false">)</mo></mrow></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">association or independence between variables</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Kruskal-Wallis</td>
<td align="left" colspan="1" rowspan="1">numerical, ordinal, or categorical</td>
<td align="left" colspan="1" rowspan="1">ranks of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq115"><math id="d33e3319"><mrow><mo>≥</mo><mn>2</mn></mrow></math></span> groups</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq116"><math id="d33e3335"><mrow><mi>H</mi><mo>∈</mo><mo stretchy="false">[</mo><mn>0</mn><mo>;</mo><mo>+</mo><mi>∞</mi><mo stretchy="false">)</mo></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">compare ranks across multiple groups</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Assumptions and limitations of the statistical tests considered in this study</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Test</th>
<th align="left" colspan="1" rowspan="1">Assumptions</th>
<th align="left" colspan="1" rowspan="1">Limitations</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Student’s <em>t</em>
</td>
<td align="left" colspan="1" rowspan="1">normally distributed data, equal variances across groups</td>
<td align="left" colspan="1" rowspan="1">sensitive to outliers</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mann-Whitney <em>U</em>
</td>
<td align="left" colspan="1" rowspan="1">independent observations, data not necessarily normal</td>
<td align="left" colspan="1" rowspan="1">does not provide direct information on the difference between groups, less powerful than the Student’s <em>t</em>-test</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Chi-squared</td>
<td align="left" colspan="1" rowspan="1">data not necessarily normal, independent observations</td>
<td align="left" colspan="1" rowspan="1">does not provide information about the direction or magnitude of the difference</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Kruskal-Wallis</td>
<td align="left" colspan="1" rowspan="1">independent observations</td>
<td align="left" colspan="1" rowspan="1">does not provide information about the direction or magnitude of the difference</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par179">The four tests produce not only a probability value (<em>p</em>-value), but also other values. As we explained earlier (“<a href="#Sec4" class="usa-link">Properties and theoretical aspects of statistical tests</a>” section), the Student’s <em>t</em>-test generate the <em>t</em>-value, the Mann-Whitney <em>U</em> test generate the <em>U</em> statistic, the Chi-squared test delivers the Chi-squared statistic, and the Kruskal-Wallis test outputs the <em>H</em> statistic. We report the lower limits and the upper limits of these statistics in Table <a href="#Tab4" class="usa-link">4</a>.</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Lower and upper boundary cases for the considered statistical tests. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq117"><math id="d33e3422"><msub><mi>n</mi><mn>1</mn></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq118"><math id="d33e3433"><msub><mi>n</mi><mn>2</mn></msub></math></span> are the sample sizes of the two groups in the comparison. <em>H</em>: statistic produced by the Kruskal-Wallis test, that indicates the degree of difference among the groups being compared, and should not be confused with the null hypothesis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq119"><math id="d33e3447"><msub><mi>H</mi><mn>0</mn></msub></math></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq120"><math id="d33e3458"><msub><mi>h</mi><mn>0</mn></msub></math></span>)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Test</th>
<th align="left" colspan="1" rowspan="1">Lower limit</th>
<th align="left" colspan="1" rowspan="1">Upper limit</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Student’s <em>t</em>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq121"><math id="d33e3491"><mrow><mn>0</mn><mo>=</mo><mtext>no difference between groups</mtext></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq122"><math id="d33e3506"><mrow><mi>t</mi><mo stretchy="false">→</mo><mo>+</mo><mi>∞</mi></mrow></math></span> or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq123"><math id="d33e3520"><mrow><mi>t</mi><mo stretchy="false">→</mo><mo>-</mo><mi>∞</mi></mrow></math></span>: larger differences between means
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mann-Whitney <em>U</em>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq124"><math id="d33e3543"><mrow><mn>0</mn><mo>=</mo><mtext>no difference between groups</mtext></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq125"><math id="d33e3558"><mrow><mi>U</mi><mo>=</mo><msub><mi>n</mi><mn>1</mn></msub><mo>·</mo><msub><mi>n</mi><mn>2</mn></msub></mrow></math></span>: maximum possible comparisons
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Chi-squared</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq126"><math id="d33e3582"><mrow><mn>0</mn><mo>=</mo><mtext>no difference between groups</mtext></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">no defined upper limit; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq127"><math id="d33e3597"><msup><mi>χ</mi><mn>2</mn></msup></math></span> increases with discrepancies</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Kruskal-Wallis</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq128"><math id="d33e3614"><mrow><mn>0</mn><mo>=</mo><mtext>no difference between groups</mtext></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">no defined upper limit; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq129"><math id="d33e3629"><mi>H</mi></math></span> increases with discrepancies</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par180">Even if these statistics can be useful in several contexts, we mainly consider these four statistical tests for the probability values they produce. The <em>p</em>-value is a statistical measure that quantifies the probability of observing an effect or a difference as extreme as, or more extreme than, what is obtained from the data, assuming the null hypothesis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq99"><math id="d33e2992"><msub><mi>H</mi><mn>0</mn></msub></math></span>) is true. In other words, it is an indicator of the strength of evidence against <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq100"><math id="d33e3003"><msub><mi>H</mi><mn>0</mn></msub></math></span>.</p>
<p id="Par181">To summarize:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par182">A small <em>p</em>-value indicates that the observed data are unlikely under <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq101"><math id="d33e3023"><msub><mi>H</mi><mn>0</mn></msub></math></span>, suggesting the null hypothesis should be rejected in favor of the alternative hypothesis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq102"><math id="d33e3034"><msub><mi>H</mi><mn>1</mn></msub></math></span>).</p></li>
<li><p id="Par183">A large <em>p</em>-value suggests there is insufficient evidence to reject <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq103"><math id="d33e3051"><msub><mi>H</mi><mn>0</mn></msub></math></span>, but it does not necessarily confirm its validity [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>].</p></li>
</ul>
<p>In our study we will use the significance threshold at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq104"><math id="d33e3066"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span>, that actually is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq105"><math id="d33e3078"><mrow><mn>5</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>3</mn></mrow></msup></mrow></math></span>, as suggested by Daniel J. Benjamin et al. [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]: this choice is made to improve the reproducibility of scientific research and reduce the false positive rates in any scientific field (Table <a href="#Tab5" class="usa-link">5</a>).
</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Our interpretation of the <em>p</em>-values</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">
<em>P</em>-value range</th>
<th align="left" colspan="1" rowspan="1">Judgment</th>
<th align="left" colspan="1" rowspan="1">Interpretation</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">[0; 0.005)</td>
<td align="left" colspan="1" rowspan="1">small</td>
<td align="left" colspan="1" rowspan="1">strong statistical evidence against <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq130"><math id="d33e3673"><msub><mi>H</mi><mn>0</mn></msub></math></span>; a very significant effect or difference between groups.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">[0.005; 0.01)</td>
<td align="left" colspan="1" rowspan="1">moderate</td>
<td align="left" colspan="1" rowspan="1">some statistical evidence against <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq131"><math id="d33e3692"><msub><mi>H</mi><mn>0</mn></msub></math></span>; some significant effect or difference between groups.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">[0.01; 0.05)</td>
<td align="left" colspan="1" rowspan="1">large</td>
<td align="left" colspan="1" rowspan="1">limited statistical evidence against <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq132"><math id="d33e3711"><msub><mi>H</mi><mn>0</mn></msub></math></span>; marginally significant effect or difference between groups.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">[0.05; 1)</td>
<td align="left" colspan="1" rowspan="1">very large</td>
<td align="left" colspan="1" rowspan="1">no statistical evidence against <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq133"><math id="d33e3730"><msub><mi>H</mi><mn>0</mn></msub></math></span>; the observed effect or difference might be due to chance.</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section id="FPar7"><h4 class="pmc_sec_title">Misuses of <em>p</em>-values</h4>
<p id="Par187">Probability values produced by statistical tests are the cornerstone of thousand of scientific studies worldwide, but their use can sometimes be wrong or misleading. In particular, the meaning of resulting <em>p</em>-values can be misunderstood [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>–<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>] or the threshold to decide if a <em>p</em>-value is significant or not can be arbitrarily chosen [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>–<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>].</p>
<p>Although widely used across scientific disciplines, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq134"><math id="d33e3770"><mi>p</mi></math></span>-values are often misunderstood and misapplied. A common misconception is that a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq135"><math id="d33e3779"><mi>p</mi></math></span>-value indicates the probability that the null hypothesis is true, when in fact it represents the probability of obtaining results at least as extreme as the observed ones under the assumption that the null hypothesis is correct [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>–<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. This subtle but crucial distinction is frequently overlooked, leading to erroneous conclusions, such as equating <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq136"><math id="d33e3794"><mrow><mi>p</mi><mo>&gt;</mo><mn>0.05</mn></mrow></math></span> with evidence of no effect, or interpreting <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq137"><math id="d33e3807"><mrow><mi>p</mi><mo>&lt;</mo><mn>0.05</mn></mrow></math></span> as confirmation of the alternative hypothesis [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>].</p>
<p>The threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq138"><math id="d33e3824"><mrow><mi>p</mi><mo>&lt;</mo><mn>0.05</mn></mrow></math></span>, though historically influential, is arbitrary and can create a false dichotomy between “significant” and “non-significant” results [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>, <a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. As Altman et al. [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>] and Di Leo et al. [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>] point out, this practice not only ignores the continuity of evidence but also discourages nuanced interpretations based on confidence intervals, effect sizes, and prior plausibility. Even when <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq139"><math id="d33e3849"><mrow><mi>p</mi><mo>&lt;</mo><mn>0.05</mn></mrow></math></span>, the evidence may be weak—for instance, a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq140"><math id="d33e3861"><mi>p</mi></math></span>-value of 0.05 corresponds to a Bayes factor upper bound suggesting the alternative hypothesis is only about 2.5 times more likely than the null [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>].</p>
<p>Moreover, the fixation on arbitrary significance thresholds fosters questionable research practices such as <em>p</em>-hacking, where analytical flexibility is exploited to reach statistical significance [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]. Evidence of widespread <em>p</em>-hacking has been documented through the analysis of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq141"><math id="d33e3885"><mi>p</mi></math></span>-curve distributions, revealing a disproportionate number of results just below the 0.05 threshold, particularly in biomedical and multidisciplinary research [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>].</p>
<p>In response to these issues, the American Statistical Association and numerous scholars have called for abandoning the binary notion of <em>statistical significance</em> [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]: their members advocate instead for more informative inferential approaches that emphasize compatibility intervals, transparent reporting, prior evidence, and contextual interpretation of results. Lowering the significance threshold (for example, to 0.005) or integrating Bayesian reasoning and estimates of false discovery rates can also help mitigate the risks of spurious findings [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>, <a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. Ultimately, the meaningful use of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq142"><math id="d33e3911"><mi>p</mi></math></span>-values demands a shift from threshold-based decisions to comprehensive inferential reasoning, where uncertainty, effect size, study design, and scientific plausibility are given equal weight in interpreting results.</p></section></section><section id="Sec11"><h3 class="pmc_sec_title">Selection of statistical tests</h3>
<p id="Par192">These tests are employed depending on the nature of the variables under consideration. The following table summarizes the appropriate choice of test based on the type of comparison between the variables. In the following sections, <em>n</em> will indicate the sample size and <em>k</em> the number of groups in the comparison, which is the number of unique elements in a data samples. For example, a sample containing the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq143"><math id="d33e3930"><mrow><mo stretchy="false">{</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mn>3</mn><mo>,</mo><mn>4</mn><mo>,</mo><mn>5</mn><mo>,</mo><mn>5</mn><mo>,</mo><mn>5</mn><mo stretchy="false">}</mo></mrow></math></span> data elements has <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq144"><math id="d33e3957"><mrow><mi>n</mi><mo>=</mo><mn>7</mn></mrow></math></span> (which is the size of the sample) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq145"><math id="d33e3969"><mrow><mi>k</mi><mo>=</mo><mn>5</mn></mrow></math></span> (which is the number of unique data elements of the sample).</p>
<p id="Par193">In this section, we set the record straight about which statistical tests should be used in which cases. We summarize all our guidelines in Table <a href="#Tab6" class="usa-link">6</a>, Figs. <a href="#Fig1" class="usa-link">1</a>, and <a href="#Fig2" class="usa-link">2</a>.
</p>
<section class="tw xbox font-sm" id="Tab6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>List of statistics tests to be used for comparing pairs of samples, depending on the data type of the samples and on the number of distinct data elements of the samples. Comparison of statistical tests based on different types of variables, sample size <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq146"><math id="d33e3998"><mi>n</mi></math></span>, and number of groups <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq147"><math id="d33e4007"><mi>k</mi></math></span>. Group: number of unique data elements in a sample (<em>k</em> in the text). The case numbers, Diamond 1, and Diamond 2 refer to Figs. <a href="#Fig1" class="usa-link">1</a> and to <a href="#Fig2" class="usa-link">2</a>. *Even when the one-way ANOVA test assumptions are valid, using the Kruskal-Wallis test is still correct, even if less powerful than the one-way ANOVA test</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Case number</th>
<th align="left" colspan="1" rowspan="1">Groups</th>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">Recommended test</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq148"><math id="d33e4047"><mrow><mo>=</mo><mn>2</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">ordinal versus ordinal</td>
<td align="left" colspan="1" rowspan="1">Chi-squared test</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq149"><math id="d33e4068"><mrow><mo>=</mo><mn>2</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">categorical versus categorical</td>
<td align="left" colspan="1" rowspan="1">Chi-squared test</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq150"><math id="d33e4089"><mrow><mo>=</mo><mn>2</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">categorical versus ordinal</td>
<td align="left" colspan="1" rowspan="1">Chi-squared test</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq151"><math id="d33e4110"><mrow><mo>=</mo><mn>2</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">numeric versus numeric</td>
<td align="left" colspan="1" rowspan="1">if <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq152"><math id="d33e4126"><mrow><mi>n</mi><mo>&lt;</mo><mn>50</mn></mrow></math></span> and Shapiro-Wilk test is significant or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq153"><math id="d33e4138"><mrow><mi>n</mi><mo>≥</mo><mn>50</mn></mrow></math></span> and Kolmogorov-Smirnov test is significant (Diamond 1):</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Student’s <em>t</em>-test (only with normal distribution, more powerful) else:</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Mann-Whitney <em>U</em> test (for any distribution, including normal)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq154"><math id="d33e4176"><mrow><mo>=</mo><mn>2</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">numeric versus categorical</td>
<td align="left" colspan="1" rowspan="1">Mann-Whitney <em>U</em> test</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq155"><math id="d33e4201"><mrow><mo>=</mo><mn>2</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">ordinal versus numeric</td>
<td align="left" colspan="1" rowspan="1">Mann-Whitney <em>U</em> test</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq156"><math id="d33e4226"><mrow><mo>≥</mo><mn>3</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">numeric versus numeric</td>
<td align="left" colspan="1" rowspan="1">if data follows a normal distribution &amp; homogeneity of variances (Diamond 2): one-way ANOVA test (more powerful) else:</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">Kruskal-Wallis test*</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq157"><math id="d33e4253"><mrow><mo>≥</mo><mn>3</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">numeric versus categorical</td>
<td align="left" colspan="1" rowspan="1">Kruskal-Wallis test</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq158"><math id="d33e4275"><mrow><mo>≥</mo><mn>3</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">categorical versus categorical</td>
<td align="left" colspan="1" rowspan="1">Kruskal-Wallis test</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq159"><math id="d33e4296"><mrow><mo>≥</mo><mn>3</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">categorical versus ordinal</td>
<td align="left" colspan="1" rowspan="1">Kruskal-Wallis test</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq160"><math id="d33e4317"><mrow><mo>≥</mo><mn>3</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">ordinal versus numeric</td>
<td align="left" colspan="1" rowspan="1">Kruskal-Wallis test</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq161"><math id="d33e4338"><mrow><mo>≥</mo><mn>3</mn></mrow></math></span>
</td>
<td align="left" colspan="1" rowspan="1">ordinal versus ordinal</td>
<td align="left" colspan="1" rowspan="1">Kruskal-Wallis test</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ecf3/12366075/843d7b184c43/13040_2025_465_Fig1_HTML.jpg" loading="lazy" id="MO1" height="790" width="778" alt="Fig. 1"></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flowchart all possible statistical tests choices for pairs of samples containing data of 2 distinct groups. The numbers in the yellow boxes correspond to the case numbers in Table <a href="#Tab6" class="usa-link">6</a>. Groups: number of unique elements in a sample</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ecf3/12366075/5275686894ae/13040_2025_465_Fig2_HTML.jpg" loading="lazy" id="MO2" height="825" width="778" alt="Fig. 2"></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flowchart representing all possible statistical tests choices for pairs of samples containing data of 3 distinct groups or more. The numbers in the yellow boxes correspond to the case numbers in Table <a href="#Tab6" class="usa-link">6</a>. Groups: number of unique elements in a sample. *Actually, even when normality, homogeneity, and independence in both samples are present, the Kruskal-Wallis test is still valid and has been proved by Jamie Gleasonto be slightly less powerful than the one-way ANOVA test [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]</p></figcaption></figure><section id="FPar8"><h4 class="pmc_sec_title">Comparison for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq162"><math id="d33e4356"><mrow><mi>k</mi><mo>=</mo><mn>2</mn></mrow></math></span> groups.</h4>
<p id="Par194">When the pair of samples contains only two different data elements, all the tests’ selections are straightforward except the numeric versus numeric comparison. The Chi-squared test should be employed for the ordindal-ordinal case, the categorical-categorical case, and the categorical-ordinal case. The Mann-Whitney <em>U</em> test should be utilized for the numeric-categorical and the ordinal-numeric test (Table <a href="#Tab6" class="usa-link">6</a> and Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<p>The situation is different for the numeric-numeric statistical comparison, where two tests can be utilized. The choice between the Student’s <em>t</em>-test and the Mann-Whitney <em>U</em> test is primarily based on the assumption of normality of the data. If the data follow a normal distribution, the Student’s <em>t</em>-test should be preferred. However, if the data do not meet this assumption, the Mann-Whitney <em>U</em> test is the appropriate test to use. While the Mann-Whitney test can be employed even with normally distributed data, it may result in a loss of statistical power, which could impair the detection of differences between the groups.</p>
<p>The Student’s <em>t</em>-test and the Mann-Whitney <em>U</em> test have some differences in calculation. The Student’s <em>t</em>-test calculates the difference between the means of the two groups and compares it with the variability (variance) within each group. The formula takes into account the sample size <em>n</em> and the variance of both groups. On the other hand, in the Mann-Whitney <em>U</em> test, instead of working with raw numerical values, the data are ranked from smallest to largest. The sum of ranks for each group is computed, and the ranks (rather than the original data points) are compared between the groups.</p>
<p>Difference raise also when dealing with outliers. The Student’s <em>t</em>-test is sensitive to outliers, as it is based on the mean of the data, which can be heavily influenced by extreme values. In contrast, the Mann-Whitney <em>U</em> test is less sensitive to outliers, as it operates on ranks rather than raw values.</p>
<p>We recap our pieces of advice in Table <a href="#Tab6" class="usa-link">6</a> and Fig. <a href="#Fig1" class="usa-link">1</a>.</p></section><section id="FPar9"><h4 class="pmc_sec_title">Comparison for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq163"><math id="d33e4429"><mrow><mi>k</mi><mo>≥</mo><mn>3</mn></mrow></math></span> groups.</h4>
<p id="Par199">In cases where three groups or more need to be compared, the choice of the statistical test is between one-way ANOVA test and Kruskal-Wallis test. The one-way ANOVA test is preferred when the assumptions of normality, homogeneity of variances, and independence are met, providing slightly higher statistical power compared to the Kruskal-Wallis test. However, studies have shown that the difference in power between one-way ANOVA test and Kruskal-Wallis test is minimal under normality conditions, with Kruskal-Wallis presenting a disadvantage of approximately 0.01 to 0.02 in power. When the assumptions for one-way ANOVA test are violated, the Kruskal-Wallis test often demonstrates substantially higher power, especially under non-normal distributions such as the Chi-squared distribution with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq164"><math id="d33e4443"><mrow><mi>d</mi><mi>f</mi><mo>=</mo><mn>2</mn></mrow></math></span>. Given that perfect normality is rarely achieved in practice, the Kruskal-Wallis test offers a robust alternative with little to lose in terms of power, making it a reliable choice for non-normally distributed data or when assumptions of homogeneity of variances are in question [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. We report our recommendations in Table <a href="#Tab6" class="usa-link">6</a> and Fig. <a href="#Fig2" class="usa-link">2</a>.</p>
<p>Since the strict conditions for a proper use of the one-way ANOVA test are rarely met among biomedical data, we decided not to focus on this particular test in this study. We mention the existence of the one-way ANOVA test for full disclosure and for clarity purposes.</p></section><section id="Sec12"><h4 class="pmc_sec_title">Normality testing</h4>
<p id="Par202">It is essential to confirm the normality of the data before selecting the appropriate test. Two common normality tests used for this purpose are the Shapiro-Wilk test [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>] and the Kolmogorov-Smirnov test [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. The condition of normality is crucial in deciding which test to use, and to have statistical certainty about the normality of the data, these two tests are commonly employed.</p>
<p id="Par203">The Shapiro-Wilk test is one of the most common tests used to check if a dataset follows a normal distribution. It is particularly suitable when you have a small or medium-sized sample (up to about two thousand observations). How it works:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par204">Null hypothesis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq165"><math id="d33e4512"><msub><mi>H</mi><mn>0</mn></msub></math></span>): the data follows a normal distribution.</p></li>
<li><p id="Par205">Alternative hypothesis (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq166"><math id="d33e4526"><msub><mi>H</mi><mn>1</mn></msub></math></span>): the data does not follow a normal distribution.</p></li>
</ul>
<p>Procedure:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par206">Test statistics are calculated based on the difference between your observed data and the values you would expect if the data were normally distributed.</p></li>
<li><p id="Par207">The test returns a <em>p</em>-value.</p></li>
<li><p id="Par208">If the <em>p</em>-value is greater than the significance level, you do not reject the null hypothesis, meaning the data are considered compatible with a normal distribution.</p></li>
<li><p id="Par209">If the <em>p</em>-value is less than the significance level, you reject the null hypothesis and conclude that the data do not follow a normal distribution [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>].</p></li>
</ul>
<p>Limitations of the test:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par210">Sensitivity to large samples: with very large samples, even small deviations from normality may be statistically significant, even if these deviations are not practically significant.</p></li>
<li><p id="Par211">Sensitivity to small samples: with very small samples, the test may not have enough power to detect deviations from normality [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>].</p></li>
</ul>
<p>When to use it: The Shapiro-Wilk test is useful to determine whether you can use parametric tests (which require normality, like the <em>t</em>-test or one-way ANOVA test) or non-parametric tests (like the Mann-Whitney or Kruskal-Wallis test). Note: Since this is a test for normality, it is only valid for continuous numeric variables and not applicable to ordinal or categorical variables.</p>
<p id="Par212">The Shapiro-Wilk test is generally preferred over the Kolmogorov-Smirnov test for assessing the normality of data in specific situations:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par213">Higher statistical power for small samples: The Shapiro-Wilk test is known to be more powerful in detecting deviations from normality when working with small samples (typically fewer than 50 observations), whereas the Kolmogorov-Smirnov test may not be as sensitive in these situations.</p></li>
<li><p id="Par214">Test specifically for normality: The Kolmogorov-Smirnov test compares an empirical distribution with a reference distribution (which could be normal or another), whereas the Shapiro-Wilk test is specifically designed to test whether the data follow a normal distribution. Therefore, the Shapiro-Wilk test is more targeted when the goal is to assess normality.</p></li>
<li><p id="Par215">Asymptotic distribution: The Kolmogorov-Smirnov test is a non-parametric test and may be more suitable for large samples, but in the case of small samples, the asymptotic distribution it is based on may not provide results as accurate as those from the Shapiro-Wilk test [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>].</p></li>
</ul>
<p id="Par216">In summary, the Shapiro-Wilk test is preferable for small samples and when the goal is solely to check normality, while the Kolmogorov-Smirnov test is better suited for comparing an empirical distribution with a theoretical distribution, especially with larger samples.</p>
<p id="Par217">We report the use of these tests in the decision Diamond 1 indicated in Table <a href="#Tab6" class="usa-link">6</a> and depicted in Fig. <a href="#Fig1" class="usa-link">1</a>.</p>
<section id="FPar10"><h5 class="pmc_sec_title">Whitney <em>U</em> test versus Student’s <em>t</em> test</h5>
<p id="Par218">Regarding the use of the Whitney <em>U</em> test over the Student’s <em>t</em> test Zaal Kikvidze et al. [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>] evaluated the performance of <em>t</em> tests, Mann–Whitney <em>U</em> tests, and randomization methods on simulated ecological data with skewed distributions, unequal variances, and unbalanced sample sizes; they found that the Mann–Whitney test failed under unequal variances in large samples, while the <em>t</em> test for unequal variances lost power when smaller samples had lower variability.</p></section></section></section></section><section id="Sec13"><h2 class="pmc_sec_title">Datasets</h2>
<p id="Par219">To understand how to properly use the four tests described in this study, we report the results of these tests first on artificial data and then on real-world medical datasets. In this section, we briefly describe the five datasets derived from electronic medical records that we analyze later in this study.</p>
<p id="Par220">We decided to use data derived from electronic health records (EHRs) in this study for several reasons [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>]. First, EHRs are relatively easy to access, especially through open datasets made available for research. Second, they contain clinical information collected for medical purposes, which increases the real-world relevance of the data. Third, EHRs are inherently complex, including heterogeneous variables, missing data, and non-standardized formats. This complexity fit perfectly in the aim of modelling and explaining; finally, working with open EHR data ensures transparency and reproducibility, which are essential for scientific validation.</p>
<section id="FPar11"><h3 class="pmc_sec_title">Original studies on the datasets considered</h3>
<p id="Par221">The purpose of this section is to introduce the datasets used to compare the four statistical tests under analysis. These datasets were carefully chosen to demonstrate how each test can be applied effectively in different scenarios, highlighting their strengths and suitability for various types of data.</p>
<p>Yangyan Ma and coauthors [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>] analyzed clinical and pathological characteristics, MYCN gene status, surgical methods, and prognosis in neuroblastoma patients from Eastern China. MYCN amplification was associated with significantly lower overall survival, and gross total resection improved survival in advanced-stage cases.</p>
<p>Bulent Gucyetmez et al. [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>, <a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>] analyzed inflammatory markers in over one thousand intensive care patients to evaluate whether C-reactive protein levels and blood count parameters can differentiate sepsis from non-sepsis systemic inflammatory response syndrome. They found that high C-reactive protein levels combined with low lymphocyte and platelet counts significantly increased the likelihood of sepsis, while other common markers such as white blood cell count and neutrophil count were not reliable discriminators.</p>
<p>Bhautesh Dinesh Jani and coauthors [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>] investigated the impact of depression on clinical outcomes in heart failure patients from a community cohort. Depression was associated with increased risks of hospitalization and death, and its inclusion in predictive models significantly improved risk stratification.</p>
<p>Rosa Requena-Morales et al. [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>] examined out-of-hospital cardiac arrest mortality in Alicante, Spain, identifying male gender, asystole, longer emergency response time, and cardiac arrest at home as significant risk factors. The findings highlight the need for enhanced CPR training policies to reduce mortality.</p>
<p>Yuichi Takashi and coauthors [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>, <a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>] examined the relationship between serum osteocalcin levels and body fat percentage in Japanese young adults with childhood-onset type one diabetes. The study found an inverse correlation between osteocalcin concentrations and body fat, which remained significant after adjusting for clinical factors. No associations were observed with adiponectin, testosterone, muscle strength, or insulin dose.</p></section></section><section id="Sec14"><h2 class="pmc_sec_title">Results</h2>
<p id="Par227">In the previous sections, we described the theoretical and mathematical properties and the assumptions of the four statistical tests. Here we move from theory to practice. In this section, we first report the results obtained the the considered statistical tests on artificial data (“<a href="#Sec15" class="usa-link">Tests on artificial data</a>” section) and then on medical datasets (“<a href="#Sec16" class="usa-link">Tests on medical data</a>” section).</p>
<section id="Sec15"><h3 class="pmc_sec_title">Tests on artificial data</h3>
<p id="Par228">Before applying the statistical tests to real-world datasets, we performed several sanity checks on artificial data for each test. This step ensures that the tests behave as expected under controlled conditions, helping to verify their proper implementation and interpretation.</p>
<p id="Par229">In Table <a href="#Tab7" class="usa-link">7</a> we report the results of the tests conducted for the one-sample Student’s <em>t</em>-test using two fixed samples with the same mean, varying the known population mean <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq167"><math id="d33e4699"><msub><mi>μ</mi><mn>0</mn></msub></math></span> to test the method’s responsiveness. To design the sanity checks, we started with samples characterized by low and then higher variance, and tested them against known population means that were initially identical to the sample mean and then progressively more distant, up to values far outside the observed data range. When the known mean equals the sample mean (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq168"><math id="d33e4710"><mrow><msub><mi>μ</mi><mn>0</mn></msub><mo>=</mo><mn>2</mn></mrow></math></span>), we expect no difference and a very high <em>p</em>-value. This outcome is confirmed by the result <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq169"><math id="d33e4728"><mrow><mi>p</mi><mo>=</mo><mn>1.000</mn></mrow></math></span>. With a slightly different known mean (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq170"><math id="d33e4740"><mrow><msub><mi>μ</mi><mn>0</mn></msub><mo>=</mo><mn>3</mn></mrow></math></span>), a moderate <em>p</em>-value is expected, still above the threshold. The result <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq171"><math id="d33e4757"><mrow><mi>p</mi><mo>=</mo><mn>0.200</mn></mrow></math></span> confirms this. As the known mean increases further (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq172"><math id="d33e4769"><mrow><msub><mi>μ</mi><mn>0</mn></msub><mo>=</mo><mn>5</mn></mrow></math></span>), we expect the <em>p</em>-value to decrease, approaching significance. Indeed, we obtain <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq173"><math id="d33e4787"><mrow><mi>p</mi><mo>=</mo><mn>0.010</mn></mrow></math></span>, consistent with expectations. Finally, with a distant known mean (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq174"><math id="d33e4799"><mrow><msub><mi>μ</mi><mn>0</mn></msub><mo>=</mo><mn>15</mn></mrow></math></span>), we expect a highly significant result. The very low value <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq175"><math id="d33e4813"><mrow><mi>p</mi><mo>=</mo><mn>3.732</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>5</mn></mrow></msup></mrow></math></span> confirms this. Overall, the results behave as expected: larger discrepancies between the sample and the known mean lead to progressively smaller <em>p</em>-values, validating the test.
</p>
<section class="tw xbox font-sm" id="Tab7"><h4 class="obj_head">Table 7.</h4>
<div class="caption p"><p>One sample Student’s <em>t</em>-test – sanity check. The symbol <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq176"><math id="d33e4843"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span> indicates statistical significance at the 0.005 level. Lower <em>p</em>-values indicate stronger evidence of a significant difference between the sample and the known mean. As expected, comparisons involving increasingly different values yield progressively smaller <em>p</em>-values</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">Batches</th>
<th align="left" colspan="1" rowspan="1">Characteristics</th>
<th align="left" colspan="1" rowspan="1">
<em>P</em>-value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">identity</td>
<td align="left" colspan="1" rowspan="1">sample1 = [2, 2.1, 1.9, 2.2, 1.8]</td>
<td align="left" colspan="1" rowspan="1">identical data to the known mean, no difference, the test should not be significant.</td>
<td align="left" colspan="1" rowspan="1">1.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">known mean1 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq177"><math id="d33e4900"><msub><mi>μ</mi><mn>0</mn></msub></math></span> = 2</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mean close to the sample</td>
<td align="left" colspan="1" rowspan="1">sample2 = [2, 0.1, 3.9, 1.2, 2.8]</td>
<td align="left" colspan="1" rowspan="1">known mean close to the sample data, the test does not detect a significant difference.</td>
<td align="left" colspan="1" rowspan="1">0.200</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">known mean2 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq178"><math id="d33e4927"><msub><mi>μ</mi><mn>0</mn></msub></math></span> = 3</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mean far from the sample</td>
<td align="left" colspan="1" rowspan="1">sample2 = [2, 0.1, 3.9, 1.2, 2.8]</td>
<td align="left" colspan="1" rowspan="1">known mean distant from the sample data, the test detects a significant difference.</td>
<td align="left" colspan="1" rowspan="1">0.010</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">known mean3 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq179"><math id="d33e4954"><msub><mi>μ</mi><mn>0</mn></msub></math></span> = 5</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">mean very far from the sample</td>
<td align="left" colspan="1" rowspan="1">sample2 = [2, 0.1, 3.9, 1.2, 2.8]</td>
<td align="left" colspan="1" rowspan="1">known mean extremely distant from the sample, the test detects a highly significant difference.</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq180"><math id="d33e4977"><mrow><mn>3.732</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>5</mn></mrow></msup></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq181"><math id="d33e4993"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">known mean4 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq182"><math id="d33e5016"><msub><mi>μ</mi><mn>0</mn></msub></math></span> = 15</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par230">In Table <a href="#Tab8" class="usa-link">8</a>, we disclose the results of the tests carried out for the independent samples Student’s <em>t</em>-test by comparing different pairs of samples under controlled scenarios. To design the sanity checks, we generated independent sample pairs with increasing differences in variance, and then progressively altered the group means, from identical values to increasingly distant ones, to evaluate the test’s sensitivity to between-group differences in central tendency. When the two samples are identical, we expect no difference and thus a <em>p</em>-value of 1.000, which is exactly what we observe in the first row of Table <a href="#Tab8" class="usa-link">8</a>. In the second case, although the variances differs, the sample means are similar; we expect the test to remain non-significant. The result (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq183"><math id="d33e5043"><mrow><mi>p</mi><mo>=</mo><mn>1.000</mn></mrow></math></span>) confirms this expectation. With two samples showing large differences in variance but still similar central tendencies, we envision a non-significant result again. The obtained <em>p</em>-value of 0.872 confirms that variance alone does not lead to significance in this context. In the fourth case (sample5 and sample6), the two groups have clearly different means. We expect a statistically significant result. The test confirms this expectation by producing <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq184"><math id="d33e5059"><mrow><mi>p</mi><mo>=</mo><mn>2.358</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>3</mn></mrow></msup></mrow></math></span>, which is below the threshold of 0.005. Finally, when comparing two groups with both very different means and variances, we expect a highly significant difference. The result is indeed strongly significant (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq185"><math id="d33e5077"><mrow><mi>p</mi><mo>=</mo><mn>9.179</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>9</mn></mrow></msup></mrow></math></span>), validating the test’s sensitivity. Overall, the observed outcomes match theoretical assumptions: as the difference between group means increases, the <em>p</em>-value decreases accordingly, confirming the correct behavior of the independent samples <em>t</em>-test.
</p>
<section class="tw xbox font-sm" id="Tab8"><h4 class="obj_head">Table 8.</h4>
<div class="caption p"><p>Independent samples Student’s <em>t</em>-test – sanity check. The symbol <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq186"><math id="d33e5110"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span> indicates statistical significance at the 0.005 level. Lower <em>p</em>-values indicate stronger evidence of a significant difference between the two samples. As expected, comparisons involving samples with increasingly different values yield progressively smaller <em>p</em>-values</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">Batches</th>
<th align="left" colspan="1" rowspan="1">Characteristics</th>
<th align="left" colspan="1" rowspan="1">
<em>P</em>-value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">identity</td>
<td align="left" colspan="1" rowspan="1">sample1 = [2, 2.1, 1.9, 2.2, 1.8]</td>
<td align="left" colspan="1" rowspan="1">identical data, no difference between groups, the test should not be significant.</td>
<td align="left" colspan="1" rowspan="1">1.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample1 = [2, 2.1, 1.9, 2.2, 1.8]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">different variances</td>
<td align="left" colspan="1" rowspan="1">sample1 = [2, 2.1, 1.9, 2.2, 1.8]</td>
<td align="left" colspan="1" rowspan="1">samples with different variances but similar means, the test does not detect a significant difference.</td>
<td align="left" colspan="1" rowspan="1">1.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample2 = [2, 0.1, 3.9, 1.2, 2.8]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">highly different variances</td>
<td align="left" colspan="1" rowspan="1">sample3 = [35, 45, 50, 55, 65]</td>
<td align="left" colspan="1" rowspan="1">samples with highly different variances, the test does not detect a significant difference.</td>
<td align="left" colspan="1" rowspan="1">0.872</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample4 = [–150, 0, 50, 100, 200]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">distant groups</td>
<td align="left" colspan="1" rowspan="1">sample5 = [1, 2, 7, 8, 5]</td>
<td align="left" colspan="1" rowspan="1">groups with clearly different means, the test detects a significant difference.</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq187"><math id="d33e5208"><mrow><mn>2.358</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>3</mn></mrow></msup></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq188"><math id="d33e5224"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample6 = [10, 12, 15, 18, 20]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">very distant groups</td>
<td align="left" colspan="1" rowspan="1">sample5 = [1, 2, 7, 8, 5]</td>
<td align="left" colspan="1" rowspan="1">groups with very distant means and variances, the test detects a highly significant difference.</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq189"><math id="d33e5258"><mrow><mn>9.179</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>9</mn></mrow></msup></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq190"><math id="d33e5274"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample7 = [60, 65, 68, 63, 70]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par231">In Table <a href="#Tab9" class="usa-link">9</a>, we detail the results of the sanity check experiments for the paired samples Student’s <em>t</em>-test by comparing fixed sample pairs under increasingly divergent conditions. To design the sanity checks, we created paired samples starting from identical values, then varied the within-pair differences by modifying the variance, and finally introduced progressively larger shifts in the paired means to assess how the test responds to increasing paired discrepancies. In the first case, the same sample is compared with itself. Since all paired differences are exactly zero, the test statistic is mathematically undefined due to division by zero. This <em>undefined</em> outcome is what we expected to see. In the second case (sample1 and sample2), we compare paired samples with similar means but different variances. Since the differences between pairs are small, we expect a non-significant result. The test confirms this prospect by generating <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq191"><math id="d33e5309"><mrow><mi>p</mi><mo>=</mo><mn>1.000</mn></mrow></math></span>. In the third case, the variances differ substantially, but the means are still close. The expected outcome is a insignificant result, which is confirmed by the value <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq192"><math id="d33e5321"><mrow><mi>p</mi><mo>=</mo><mn>0.859</mn></mrow></math></span>, clearly above the 0.005 threshold. In the fourth case, the paired samples differ clearly in their means. We expect the test to detect a significant difference. The test returns <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq193"><math id="d33e5334"><mrow><mi>p</mi><mo>=</mo><mn>1.000</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>3</mn></mrow></msup></mrow></math></span>, below the threshold of 0.005, as forecast. Finally, when the means are very distant (sample5 and sample7), we expect a strongly significant result. The test returns <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq194"><math id="d33e5352"><mrow><mi>p</mi><mo>=</mo><mn>3.877</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>6</mn></mrow></msup></mrow></math></span>, confirming the expected behavior. These results show that the paired <em>t</em>-test correctly detects differences in means while being robust to variance shifts in the context of paired data. As the magnitude of the mean difference increases, the <em>p</em>-value decreases accordingly.
</p>
<section class="tw xbox font-sm" id="Tab9"><h4 class="obj_head">Table 9.</h4>
<div class="caption p"><p>Paired samples Student’s <em>t</em>-test – sanity check. The symbol <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq195"><math id="d33e5385"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span> indicates statistical significance at the 0.005 level. Lower <em>p</em>-values indicate stronger evidence of a significant difference between the two samples. As expected, comparisons involving samples with increasingly different values yield progressively smaller <em>p</em>-values.<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq196"><math id="d33e5409"><mmultiscripts><mrow></mrow><mrow></mrow><mi>α</mi></mmultiscripts></math></span>When the paired samples are identical (in our case, sample1 and sample1), all differences between observations are zero. The paired <em>t</em>-test statistic, which divides the average difference by the standard deviation of the differences divided by the square root of the sample size, involves a division by zero. Since the denominator becomes zero, the test is mathematically undefined</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">Batches</th>
<th align="left" colspan="1" rowspan="1">Characteristics</th>
<th align="left" colspan="1" rowspan="1">
<em>P</em>-value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">identity</td>
<td align="left" colspan="1" rowspan="1">sample1 = [2, 2.1, 1.9, 2.2, 1.8]</td>
<td align="left" colspan="1" rowspan="1">identical data, no difference between groups, the test should not make sense.</td>
<td align="left" colspan="1" rowspan="1">Undefined<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq197"><math id="d33e5454"><mmultiscripts><mrow></mrow><mrow></mrow><mi>α</mi></mmultiscripts></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample1 = [2, 2.1, 1.9, 2.2, 1.8]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">different variances</td>
<td align="left" colspan="1" rowspan="1">sample1 = [2, 2.1, 1.9, 2.2, 1.8]</td>
<td align="left" colspan="1" rowspan="1">differences in variance in paired data but no significant difference in means.</td>
<td align="left" colspan="1" rowspan="1">1.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample2 = [2, 0.1, 3.9, 1.2, 2.8]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">highly different variances</td>
<td align="left" colspan="1" rowspan="1">sample3 = [35, 45, 50, 55, 65]</td>
<td align="left" colspan="1" rowspan="1">significant variance differences in paired data, the test does not detect a significant difference.</td>
<td align="left" colspan="1" rowspan="1">0.859</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample4 = [–150, 0, 50, 100, 200]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">distant groups</td>
<td align="left" colspan="1" rowspan="1">sample5 = [1, 2, 7, 8, 5]</td>
<td align="left" colspan="1" rowspan="1">paired data with clearly different means, the test detects a significant difference.</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq198"><math id="d33e5512"><mrow><mn>1.000</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>3</mn></mrow></msup></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq199"><math id="d33e5528"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample6 = [10, 12, 15, 18, 20]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">very distant groups</td>
<td align="left" colspan="1" rowspan="1">sample5 = [1, 2, 7, 8, 5]</td>
<td align="left" colspan="1" rowspan="1">paired groups with highly different means, the test detects a highly significant difference.</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq200"><math id="d33e5562"><mrow><mn>3.877</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>6</mn></mrow></msup></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq201"><math id="d33e5578"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample7 = [60, 65, 68, 63, 70]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par232">In Table <a href="#Tab10" class="usa-link">10</a>, we outline the results obtained thorughthe sanity checks for the Mann-Whitney <em>U</em> test, by comparing samples under various controlled conditions to assess the test’s robustness and limitations. To design the sanity checks, we constructed pairs of independent samples with varying levels of separation, variance, and rank distribution. We began with identical samples, then introduced cases with complete separation but tied ranks, followed by unequal sample sizes, overlapping data, and highly asymmetric distributions, to explore the test’s sensitivity and limitations under diverse non-parametric conditions. When the two samples are identical, we expect the test not to detect any difference. The result <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq202"><math id="d33e5610"><mrow><mi>p</mi><mo>=</mo><mn>1.000</mn></mrow></math></span> confirms this belief, as we forecast. In the second case, although the two groups are perfectly separated in value, they have minimal variance and tied ranks, which may limit the test’s sensitivity. Despite the large difference in values, the result is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq203"><math id="d33e5622"><mrow><mi>p</mi><mo>=</mo><mn>0.398</mn></mrow></math></span>, not statistically significant, which deviates from expectations and highlights a potential limitation of the test in small, uniform samples. With unequal sample sizes (sample11 and sample12), instead, we expect the test to remain usable but possibly less stable. The result <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq204"><math id="d33e5634"><mrow><mi>p</mi><mo>=</mo><mn>0.024</mn></mrow></math></span> indicates a moderate difference but does not cross the 0.005 threshold, aligning with a lower power due to the imbalance. In the fourth scenario, the two samples overlap considerably. We expect a non-significant result due to reduced rank separation. The test confirms this prediction with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq205"><math id="d33e5647"><mrow><mi>p</mi><mo>=</mo><mn>0.343</mn></mrow></math></span>. Finally, when comparing highly asymmetric samples (sample15 and sample16), we expect a strong signal of difference. Despite the skewness, the Mann-Whitney test returns a significant result (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq206"><math id="d33e5659"><mrow><mi>p</mi><mo>=</mo><mn>4.847</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>3</mn></mrow></msup></mrow></math></span>), just below the significance threshold, confirming its sensitivity even in the presence of distributional asymmetry. Overall, the test behaves mostly as expected, though limitations emerge in cases of extreme separation with no variance or when sample sizes are small and homogeneous.</p>
<section class="tw xbox font-sm" id="Tab10"><h4 class="obj_head">Table 10.</h4>
<div class="caption p"><p>Mann-Whitney <em>U</em> test – sanity check. The symbol <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq219"><math id="d33e5873"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span> indicates statistical significance at the 0.005 level. Lower <em>p</em>-values indicate stronger evidence of a significant difference between the two samples. As expected, comparisons involving samples with increasingly different values yield progressively smaller <em>p</em>-values</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">Samples</th>
<th align="left" colspan="1" rowspan="1">Characteristics</th>
<th align="left" colspan="1" rowspan="1">
<em>P</em>-value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">identical samples</td>
<td align="left" colspan="1" rowspan="1">sample6 = [10, 12, 15, 18, 20]</td>
<td align="left" colspan="1" rowspan="1">identical data, no difference between groups, the test should not be significant.</td>
<td align="left" colspan="1" rowspan="1">1.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample6 = [10, 12, 15, 18, 20]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">separate samples</td>
<td align="left" colspan="1" rowspan="1">sample8 = [1, 1, 1, 1, 1]</td>
<td align="left" colspan="1" rowspan="1">significant separation between groups; the test should detect a significant difference.</td>
<td align="left" colspan="1" rowspan="1">0.398</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample9 = [10, 10, 10, 10, 10]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">unequal sample sizes</td>
<td align="left" colspan="1" rowspan="1">sample10 = [12, 13, 15]</td>
<td align="left" colspan="1" rowspan="1">unequal sample sizes but independent; the test can be used but might be less reliable.</td>
<td align="left" colspan="1" rowspan="1">0.024</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample11 = [20, 22, 21, 23, 25, 19]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">overlapping samples</td>
<td align="left" colspan="1" rowspan="1">sample12 = [4, 5, 7, 8, 9]</td>
<td align="left" colspan="1" rowspan="1">overlapping data reduces test power; results may not be significant.</td>
<td align="left" colspan="1" rowspan="1">0.343</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample13 = [6, 7, 8, 9, 10]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">asymmetric samples</td>
<td align="left" colspan="1" rowspan="1">sample14 = [1, 1, 2, 2, 3, 3]</td>
<td align="left" colspan="1" rowspan="1">asymmetric data may distort results; the test may not be significant due to skewed distributions.</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq220"><math id="d33e5986"><mrow><mn>4.847</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>3</mn></mrow></msup></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq221"><math id="d33e6002"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample15 = [100, 200, 300, 400, 500, 600]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par233">In the Table <a href="#Tab11" class="usa-link">11</a>, we outline the results of the sanity check experiments for the Chi-squared test, comparing categorical data distributions to assess the test’s sensitivity to imbalance. To design the sanity checks, we created pairs of categorical distributions with increasing imbalance. We started with uniform and nearly uniform frequency distributions, then introduced small and moderate deviations, and finally constructed highly skewed and extreme distributions to assess the test’s responsiveness to categorical association under varying levels of discrepancy. When comparing two uniform distributions, we expect no significant difference. The result <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq207"><math id="d33e5682"><mrow><mi>p</mi><mo>=</mo><mn>0.427</mn></mrow></math></span> confirms this. Similarly, in the case of a slight deviation from uniformity, we expect the test to remain non-significant. The test yields <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq208"><math id="d33e5694"><mrow><mi>p</mi><mo>=</mo><mn>0.982</mn></mrow></math></span>, which confirms the expectation. When small variations are introduced between groups, as in the third case, we again expect a non-significant result. The value <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq209"><math id="d33e5706"><mrow><mi>p</mi><mo>=</mo><mn>0.999</mn></mrow></math></span> confirms this lack of evidence against the null hypothesis. In the fourth case, the two groups show a strong difference in frequency values, but with small sample sizes. We expect a possible sign of imbalance, though limited power may affect detection. The result <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq210"><math id="d33e5718"><mrow><mi>p</mi><mo>=</mo><mn>0.256</mn></mrow></math></span> indicates no significant difference, aligning with this uncertainty. In the fifth scenario, the distributions are highly skewed. We expect a significant result due to the extreme imbalance. The test confirms this with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq211"><math id="d33e5731"><mrow><mi>p</mi><mo>=</mo><mn>1.000</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>7</mn></mrow></msup></mrow></math></span>. Finally, when one group is dominated by a single value and the other is uniform, we expect the test to clearly detect the difference. The test yields <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq212"><math id="d33e5749"><mrow><mi>p</mi><mo>=</mo><mn>2.202</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>5</mn></mrow></msup></mrow></math></span>, confirming our expectation. Overall, the Chi-squared test behaves as expected: it does not detect differences under uniformity or minimal variation, while it correctly identifies statistically significant discrepancies when the imbalance becomes substantial.</p>
<section class="tw xbox font-sm" id="Tab11"><h4 class="obj_head">Table 11.</h4>
<div class="caption p"><p>Chi-squared test – sanity check. The symbol <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq222"><math id="d33e6034"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span> indicates statistical significance at the 0.005 level. Lower <em>p</em>-values indicate stronger evidence of a significant difference between the two samples. As expected, comparisons involving samples with increasingly different values yield progressively smaller <em>p</em>-values</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">Batches</th>
<th align="left" colspan="1" rowspan="1">Characteristics</th>
<th align="left" colspan="1" rowspan="1">
<em>P</em>-value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">uniform distribution</td>
<td align="left" colspan="1" rowspan="1">sample16 = [25, 20, 35, 20]</td>
<td align="left" colspan="1" rowspan="1">both distributions are close to uniform; the test should not be significant.</td>
<td align="left" colspan="1" rowspan="1">0.427</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample17 = [25, 25, 25, 25]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">imbalanced distribution</td>
<td align="left" colspan="1" rowspan="1">sample18 = [10, 12, 8, 10, 10]</td>
<td align="left" colspan="1" rowspan="1">slight deviations from uniformity; the test should not be significant.</td>
<td align="left" colspan="1" rowspan="1">0.982</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample9 = [10, 10, 10, 10, 10]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">small sample variation</td>
<td align="left" colspan="1" rowspan="1">sample19 = [5, 6, 4, 5, 5, 5]</td>
<td align="left" colspan="1" rowspan="1">minimal variance between groups; the test should not be significant.</td>
<td align="left" colspan="1" rowspan="1">0.999</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample20 = [5, 5, 5, 5, 5, 5]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">strong imbalance</td>
<td align="left" colspan="1" rowspan="1">sample21 = [4, 10, 4, 10]</td>
<td align="left" colspan="1" rowspan="1">strong differences between groups; the test may detect an imbalance.</td>
<td align="left" colspan="1" rowspan="1">0.256</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample17 = [25, 25, 25, 25]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">highly unbalanced</td>
<td align="left" colspan="1" rowspan="1">sample22 = [2, 20, 2, 20, 2]</td>
<td align="left" colspan="1" rowspan="1">highly skewed distribution; the test should detect a significant difference.</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq223"><math id="d33e6147"><mrow><mn>1.000</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>7</mn></mrow></msup></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq224"><math id="d33e6163"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample23 = [100, 100, 100, 100, 100]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">extreme imbalance</td>
<td align="left" colspan="1" rowspan="1">sample24 = [1, 1, 1, 1, 26]</td>
<td align="left" colspan="1" rowspan="1">one category dominates; the test should detect a significant difference.</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq225"><math id="d33e6197"><mrow><mn>2.202</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>5</mn></mrow></msup></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq226"><math id="d33e6213"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample25 = [6, 6, 6, 6, 6]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par234">In the Table <a href="#Tab12" class="usa-link">12</a>, we report the results achieved through sanity checks for the Kruskal-Wallis test by comparing three independent samples under various conditions, assessing how rank-based differences affect significance. To design the sanity checks, we generated three-group comparisons with increasing differences in central tendency and variance. We started with identical samples, then varied the spread while keeping similar ranks, and finally introduced progressively more distinct mean and rank patterns to evaluate the test’s ability to detect rank-based differences across multiple groups. In the first case, all three samples are identical. We expect no rank difference and a non-significant result, which is confirmed by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq213"><math id="d33e5772"><mrow><mi>p</mi><mo>=</mo><mn>1.000</mn></mrow></math></span>. In the second scenario, the samples differ in variance but not in rank distribution. We expect the test not to detect a difference: the <em>p</em>-value of 0.978 confirms assumption. In the third case, despite highly different variances, the samples have nearly identical means and aligned ranks. The test again yields <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq214"><math id="d33e5787"><mrow><mi>p</mi><mo>=</mo><mn>1.000</mn></mrow></math></span>, confirming our expectation. When comparing samples with different means and variances but similar rank distributions, we do not expect significance. The test result of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq215"><math id="d33e5799"><mrow><mi>p</mi><mo>=</mo><mn>0.645</mn></mrow></math></span> aligns with this prospect. In the fifth case, the groups have clearly different means and values spread far apart. We expect to obseverve statistical significance, and the test returns <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq216"><math id="d33e5812"><mrow><mi>p</mi><mo>=</mo><mn>0.023</mn></mrow></math></span>, which confirms this although it is just above the stricter threshold of 0.005. With more extreme group differences, we expect strong significance. The result <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq217"><math id="d33e5824"><mrow><mi>p</mi><mo>=</mo><mn>1.930</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>3</mn></mrow></msup></mrow></math></span> confirms this prospect. Surprisingly, the final comparison involves similar distributions without evident shifts in ranks, yet the test yields a highly significant result (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq218"><math id="d33e5842"><mrow><mi>p</mi><mo>=</mo><mn>2.490</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>6</mn></mrow></msup></mrow></math></span>). This outcome suggests that even subtle, systematic rank shifts can be detected as significant, possibly due to the larger sample size. Overall, the Kruskal-Wallis test behaves as expected in most cases: it remains robust to variance changes when ranks align, and becomes sensitive as rank distributions diverge, although it may sometimes over-detect differences in structured but similar data.</p>
<section class="tw xbox font-sm" id="Tab12"><h4 class="obj_head">Table 12.</h4>
<div class="caption p"><p>Kruskal-Wallis test – sanity check. The symbol <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq227"><math id="d33e6246"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span> indicates statistical significance at the 0.005 level. Lower <em>p</em>-values indicate stronger evidence of a significant difference between the two samples. As expected, comparisons involving samples with increasingly different values yield progressively smaller <em>p</em>-values</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">Samples</th>
<th align="left" colspan="1" rowspan="1">Characteristics</th>
<th align="left" colspan="1" rowspan="1">
<em>P</em>-value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">identity</td>
<td align="left" colspan="1" rowspan="1">sample1 = [2, 2.1, 1.9, 2.2, 1.8]</td>
<td align="left" colspan="1" rowspan="1">identical data, no difference between groups, the test is not significant.</td>
<td align="left" colspan="1" rowspan="1">1.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample1 = [2, 2.1, 1.9, 2.2, 1.8]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample1 = [2, 2.1, 1.9, 2.2, 1.8]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">different variances</td>
<td align="left" colspan="1" rowspan="1">sample1 = [2, 2.1, 1.9, 2.2, 1.8]</td>
<td align="left" colspan="1" rowspan="1">differences in variance, no significant difference in ranks, the test is not significant.</td>
<td align="left" colspan="1" rowspan="1">0.978</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample2 = [2, 0.1, 3.9, 1.2, 2.8]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample26 = [2, 0.1, 0.9, 4.2, 5.8]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">equal means, highly different variances</td>
<td align="left" colspan="1" rowspan="1">sample27 = [51.5, 52, 52.5, 53, 53.5]</td>
<td align="left" colspan="1" rowspan="1">equal means but highly different variances, no significant difference, the test is not significant.</td>
<td align="left" colspan="1" rowspan="1">1.000</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample28 = [47.5, 50, 52.5, 55, 57.5]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample29 = [10, 30, 52.5, 75, 95]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">different means and variances</td>
<td align="left" colspan="1" rowspan="1">sample27 = [51.5, 52, 52.5, 53, 53.5]</td>
<td align="left" colspan="1" rowspan="1">different means and variances, no significant difference in ranks, the test is not significant.</td>
<td align="left" colspan="1" rowspan="1">0.645</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample30 = [42.5, 47.5, 50, 52.5, 57.5]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample31 = [10, 30, 50, 70, 90]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">distant groups</td>
<td align="left" colspan="1" rowspan="1">sample5 = [1, 2, 7, 8, 5]</td>
<td align="left" colspan="1" rowspan="1">paired data with clearly different means, the test is significant.</td>
<td align="left" colspan="1" rowspan="1">0.023</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample7 = [60, 65, 68, 63, 70]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample32 = [252, 22.1, 91.9, 2.2, 41.8]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">very distant groups</td>
<td align="left" colspan="1" rowspan="1">sample5 = [1, 2, 7, 8, 5]</td>
<td align="left" colspan="1" rowspan="1">highly distant groups, the test is significant.</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq228"><math id="d33e6405"><mrow><mn>1.930</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>3</mn></mrow></msup></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq229"><math id="d33e6421"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample7 = [60, 65, 68, 63, 70]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample33 = [252, 722.1, 951.9, 782.2, 941.8]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">similar groups</td>
<td align="left" colspan="1" rowspan="1">sample34 = [80, 85, 90, 88, 84, 87, 89, 91, 86, 82]</td>
<td align="left" colspan="1" rowspan="1">similar groups with no clear difference, the test is highly significant.</td>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq230"><math id="d33e6461"><mrow><mn>2.490</mn><mo>×</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>6</mn></mrow></msup></mrow></math></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq231"><math id="d33e6477"><mmultiscripts><mrow></mrow><mrow></mrow><mrow><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample35 = [60, 65, 70, 68, 64, 67, 69, 71, 66, 62]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">sample36 = [40, 45, 50, 48, 44, 47, 49, 51, 46, 42]</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par235"><strong>Artificial data results recap</strong>. Overall, the results obtained from the sanity checks confirm the theoretical expectations: each statistical test behaved consistently with its underlying assumptions, demonstrating appropriate sensitivity to differences in central tendency, rank distribution, or categorical imbalance, and highlighting specific conditions under which their performance may be limited.
</p></section><section id="Sec16"><h3 class="pmc_sec_title">Tests on medical data</h3>
<p id="Par238">The next figures represent barcharts illustrating the <em>p</em>-values obtained from statistical tests (<em>t</em>-test, Chi-squared test, Mann-Whitney <em>U</em> test, Kruskal-Wallis test) conducted on the variables analyzed in relation to the target variable (Figs. <a href="#Fig3" class="usa-link">3</a>, <a href="#Fig4" class="usa-link">4</a>, <a href="#Fig5" class="usa-link">5</a>, <a href="#Fig6" class="usa-link">6</a>, <a href="#Fig7" class="usa-link">7</a>). Each bar represents the <em>p</em>-value of a variable, ordered from highest to lowest. Bars are colored so that variables with <em>p</em>-values greater than the significance threshold (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq232"><math id="d33e6542"><mrow><mi>p</mi><mo>&gt;</mo><mn>0.005</mn></mrow></math></span>) are shown in sky blue, while those with <em>p</em>-values below the threshold are shown in red, indicating statistical significance.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366075_13040_2025_465_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ecf3/12366075/efbd5e8963bd/13040_2025_465_Fig3_HTML.jpg" loading="lazy" id="MO3" height="472" width="778" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Barchart of <em>p</em>-values – Neuroblastoma dataset. *** indicate significant <em>p</em>-values <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq259"><math id="d33e7084"><mrow><mo>&lt;</mo><mn>0.005</mn></mrow></math></span> threshold. chi: Chi-squared test. MW: Mann-Whitney <em>U</em> test. We reported the meanings of the clinical features in Table <a href="#Tab13" class="usa-link">13</a>. More information on this dataset can be found in the original article [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366075_13040_2025_465_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ecf3/12366075/ea66e2b8154c/13040_2025_465_Fig4_HTML.jpg" loading="lazy" id="MO4" height="470" width="778" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Barchart of <em>p</em>-values – Sepsis and SIRS dataset. *** indicate significant <em>p</em>-values <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq260"><math id="d33e7122"><mrow><mo>&lt;</mo><mn>0.005</mn></mrow></math></span> threshold. chi: Chi-squared test. MW: Mann-Whitney <em>U</em> test. We reported the meanings of the clinical features in Table <a href="#Tab14" class="usa-link">14</a>. More information on this dataset can be found in the original article [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>]</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366075_13040_2025_465_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ecf3/12366075/9eb777da4056/13040_2025_465_Fig5_HTML.jpg" loading="lazy" id="MO5" height="473" width="778" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Barchart of <em>p</em>-values – Depression and Heart Failure dataset. *** indicate significant <em>p</em>-values <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq261"><math id="d33e7161"><mrow><mo>&lt;</mo><mn>0.005</mn></mrow></math></span> threshold. chi: Chi-squared test. MW: Mann-Whitney <em>U</em> test. We reported the meanings of the clinical features in Table <a href="#Tab15" class="usa-link">15</a>. More information on this dataset can be found in the original article [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>]</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366075_13040_2025_465_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ecf3/12366075/0432e1f7a9df/13040_2025_465_Fig6_HTML.jpg" loading="lazy" id="MO6" height="478" width="778" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Barchart of <em>p</em>-values – Cardiac Arrest dataset. *** indicate significant <em>p</em>-values <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq262"><math id="d33e7199"><mrow><mo>&lt;</mo><mn>0.005</mn></mrow></math></span> threshold. chi: Chi-squared test. MW: Mann-Whitney <em>U</em> test. We reported the meanings of the clinical features in Table <a href="#Tab16" class="usa-link">16</a>. More information on this dataset can be found in the original article [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>]</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366075_13040_2025_465_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ecf3/12366075/1b6d9c8729e8/13040_2025_465_Fig7_HTML.jpg" loading="lazy" id="MO7" height="471" width="778" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Barchart of <em>p</em>-values – Diabetes Type One dataset. *** indicate significant <em>p</em>-values <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq263"><math id="d33e7237"><mrow><mo>&lt;</mo><mn>0.005</mn></mrow></math></span> threshold. chi: Chi-squared test. MW: Mann-Whitney <em>U</em> test. We reported the meanings of the clinical features in Table <a href="#Tab17" class="usa-link">17</a>. More information on this dataset can be found in the original article [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>]</p></figcaption></figure><p id="Par239">A dashed line marks the significance threshold to clearly distinguish between significant and non-significant variables. Significant variables are labeled with a star next to their name. Additionally, significant <em>p</em>-values are annotated in scientific notation next to their respective bars.</p>
<p id="Par240">This visualization allows for a quick identification of variables that significantly impact the outcome of the analysis, making it easier to interpret the determining factors in the statistical evaluation.</p>
<section id="FPar12"><h4 class="pmc_sec_title">Neuroblastoma dataset</h4>
<p id="Par241">Figure <a href="#Fig3" class="usa-link">3</a> illustrates the <em>p</em>-values of various variables in relation to the outcome, using Chi-squared tests (chi) and the Mann-Whitney <em>U</em> test (MW). Variables with significant <em>p</em>-values are highlighted in blue, with asterisks denoting their level of significance (*** for <em>p</em>-values &lt;0.005). The red dashed line represents the significance threshold set at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq233"><math id="d33e6585"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span>. The variables “risk”, “MYCN status”, and “UH or FH” show highly significant relationships with the outcome, with <em>p</em>-values highly below the 0.005 threshold. These findings are consistent with the original study results, which have also identified these variables as significant predictors of the outcome [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]. Additionally, these results are consistent with other published studies: the correlation between the outcome and MYCN status has been verified by Damiano Bartolucci and coauthors [<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>] and the correlation between outcome and favorable histology (FH) and unfavorable histology (UH) has been verified by Atsuko Nakazawa et al. [<a href="#CR97" class="usa-link" aria-describedby="CR97">97</a>] The remaining variables did not show significance under the stringent <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq234"><math id="d33e6609"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span> threshold: this stricter threshold, compared to the commonly used <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq235"><math id="d33e6622"><mrow><mi>α</mi><mo>=</mo><mn>0.05</mn></mrow></math></span>, reduces the likelihood of type one errors, leading to fewer significant results. In the original study of this article [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>], the normality threshold has not been specified, and we assume it to be <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq236"><math id="d33e6637"><mrow><mi>α</mi><mo>=</mo><mn>0.05</mn></mrow></math></span>. The variables “risk”, “MYCN status”, and “UH or FH” are highly significant, in agreement with the original publication [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>] and confirming prior findings [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>, <a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>, <a href="#CR97" class="usa-link" aria-describedby="CR97">97</a>].</p></section><section id="FPar13"><h4 class="pmc_sec_title">Sepsis and SIRS dataset</h4>
<p id="Par242">Figure <a href="#Fig4" class="usa-link">4</a> depicts the <em>p</em>-values of various clinical and demographic variables in relation to the diagnosis of septic versus non-septic SIRS, using Chi-squared tests (chi) for categorical variables and the Mann-Whitney <em>U</em> test (MW) for continuous or ordinal ones. Variables with significant <em>p</em>-values are highlighted in blue, with asterisks denoting their significance level (*** for <em>p</em>-values <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq237"><math id="d33e6682"><mrow><mo>&lt;</mo><mn>0.005</mn></mrow></math></span>). The red dashed line indicates the significance threshold set at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq238"><math id="d33e6693"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span>. The variables “Group”, “Mortality”, “LOS-ICU”, “SOFA”, “EOC”, and “APACHE II” show strong associations with the diagnosis, with <em>p</em>-values far below the 0.005 threshold. These findings confirm the conclusions of the original study [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>], which identified these variables as key markers in discriminating septic from non-septic patients. For example, the correlation between diagnosis and mortality has also been identified by Michael Bauer et al. [<a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>]. The remaining variables, including PLTC, CRP, sex, and various blood cell counts, did not reach statistical significance under the strict <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq239"><math id="d33e6714"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span> threshold. This threshold, which is more conservative than the commonly used <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq240"><math id="d33e6727"><mrow><mi>α</mi><mo>=</mo><mn>0.05</mn></mrow></math></span>, reduces false positives but may overlook relevant predictors. Based on the original study [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>], variables such as CRP, LymC, and PLTC were nonetheless considered important in multivariate combinations. No normality assumption was stated in the original paper; we assume <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq241"><math id="d33e6742"><mrow><mi>α</mi><mo>=</mo><mn>0.05</mn></mrow></math></span> as the general threshold unless otherwise specified. The variables resulting more relevant through the biostatistics tests are known to be associated with neuroblastoma in the scientific literature.</p></section><section id="FPar14"><h4 class="pmc_sec_title">Depression and Heart Failure dataset</h4>
<p id="Par243">Figure <a href="#Fig5" class="usa-link">5</a> shows the <em>p</em>-values of various clinical and demographic variables in relation to two-year mortality in patients diagnosed with heart failure, using Chi-squared tests (chi) for categorical variables and the Mann-Whitney <em>U</em> test (MW) for continuous or ordinal ones. Variables with significant <em>p</em>-values are shown in blue, with asterisks indicating significance levels (*** for <em>p</em>-values &lt;0.005). The red dashed line represents the threshold for significance, set at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq242"><math id="d33e6774"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span>. The variables “Hospitalized”, “PHQ-9”, “BNP/NT-BNP levels”, and “Etiology HF” are associated with highly significant <em>p</em>-values, confirming the original study’s findings that depression, hospitalization history, and cardiac biomarkers are strong predictors of mortality [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>]. The variables “Hospitalized” (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq243"><math id="d33e6792"><mrow><mi>p</mi><mo>=</mo><mn>0.0064</mn></mrow></math></span>), “PHQ-9” (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq244"><math id="d33e6804"><mrow><mi>p</mi><mo>=</mo><mn>0.0118</mn></mrow></math></span>), “BNP/NT-BNP levels” (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq245"><math id="d33e6816"><mrow><mi>p</mi><mo>=</mo><mn>0.0119</mn></mrow></math></span>), and “Etiology HF” (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq246"><math id="d33e6829"><mrow><mi>p</mi><mo>=</mo><mn>0.0302</mn></mrow></math></span>) did not reach statistical significance under the stricter threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq247"><math id="d33e6841"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span>, but their <em>p</em>-values were close to this cutoff. These results remain directionally consistent with the original study’s findings [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>], which identified depression, hospitalization history, and cardiac biomarkers as relevant predictors of mortality in heart failure. The correlation between death and Patient Health Questionnaire-9 score (PHQ-9) has also been detected by Scott R. Beach et al. [<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>]. Other variables such as age, estimated glomerular filtration rate, and time from diagnosis to death or hospitalization were not significant at this threshold. As in the previous dataset, the stricter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq248"><math id="d33e6862"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span> threshold reduces the probability of false positives, potentially excluding variables that could contribute meaningfully in multivariate settings. The original analysis showed that PHQ-9 in particular improves model performance when included, emphasizing the importance of considering weakly associated variables in broader prediction frameworks [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>]. The dataset original study does not explicitly state a normality assumption, so we infer a default threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq249"><math id="d33e6878"><mrow><mi>α</mi><mo>=</mo><mn>0.05</mn></mrow></math></span>.</p></section><section id="FPar15"><h4 class="pmc_sec_title">Cardiac Arrest dataset</h4>
<p id="Par244">Figure <a href="#Fig6" class="usa-link">6</a> represents the <em>p</em>-values of various clinical and demographic variables in relation to pre-hospital mortality (“Exitus” variable) following out-of-hospital cardiac arrest, using Chi-squared tests (chi) for categorical variables and the Mann-Whitney <em>U</em> test (MW) for continuous variables. Significant variables are highlighted in blue, and asterisks indicate the level of significance (*** for <em>p</em>-values &lt;0.005). The red dashed line marks the significance threshold, set at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq250"><math id="d33e6906"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span>. The variables “Asystole” and “Cardiac arrest at home” display <em>p</em>-values well below the threshold, indicating a strong statistical association with the mortality outcome. These findings match those reported in the original study [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>], where both conditions were shown to significantly increase the odds of death before hospital arrival. The correlation between “Exitus” and “Asystole” has also been verified by Junki Ishii et al. [<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>] Other variables like age, sex, time to EMS, and bystander CPR did not reach the stringent threshold, despite some being considered relevant in multivariate analysis. The use of a conservative <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq251"><math id="d33e6928"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span> level limits type I errors but may overlook weaker predictors. The original dataset study [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>] also discussed functional status and emergency response delay as important but multifactorial contributors. Normality was not assumed or tested explicitly, so we presume a general threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq252"><math id="d33e6943"><mrow><mi>α</mi><mo>=</mo><mn>0.05</mn></mrow></math></span>.</p></section><section id="FPar16"><h4 class="pmc_sec_title">Diabetes Type One dataset</h4>
<p id="Par245">Figure <a href="#Fig7" class="usa-link">7</a> represents the <em>p</em>-values of various biological and clinical variables in relation to body fat percentage in patients with type one diabetes, tested using Chi-squared tests (chi) for categorical variables and the Mann-Whitney <em>U</em> test (MW) for continuous ones. Significant <em>p</em>-values are shown in blue, and levels of significance are indicated by asterisks (*** for <em>p</em>-values &lt;0.005). The red dashed line marks the threshold for significance at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq253"><math id="d33e6975"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span>. The variables “body mass index”, “sex”, and “HbA1c” exhibit strong associations with the body fat outcome, with <em>p</em>-values well below the threshold. These results are consistent with the original study [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>], which emphasized the role of glycemic control and anthropometric factors in fat accumulation. The correlation between percent body fat and HbA1c has also been confirmed by a study by Julie Bower et al. [<a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>]. None of the tested variables, including “body mass index”, “sex”, and “HbA1c”, reached statistical significance at the predefined threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq254"><math id="d33e6996"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span>. Nonetheless, body mass index (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq255"><math id="d33e7008"><mrow><mi>p</mi><mo>=</mo><mn>0.0306</mn></mrow></math></span>) and sex (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq256"><math id="d33e7021"><mrow><mi>p</mi><mo>=</mo><mn>0.055</mn></mrow></math></span>) showed <em>p</em>-values closer to the cutoff and may still be of interest. These findings are only partially consistent with the original study [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>], which emphasized the influence of glycemic control and anthropometric factors on fat accumulation. While the association between percent body fat and HbA1c has been reported elsewhere [<a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>], it was not confirmed in our analysis. Other variables, including ucOC and OC, did not reach significance in this univariate analysis, but were found to have an inverse relationship with body fat in the original multivariate regression. Additional variables such as adiponectin, testosterone, and insulin dosage also failed to show significance under the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq257"><math id="d33e7042"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span> criterion. As with other datasets, this stricter threshold reduces type I error risk, though it may exclude weak but meaningful predictors. The original dataset study [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>] does not specify any test for normality, so we assume a conventional <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq258"><math id="d33e7058"><mrow><mi>α</mi><mo>=</mo><mn>0.05</mn></mrow></math></span> baseline.</p></section><section id="FPar17"><h4 class="pmc_sec_title">EHRs results recap</h4>
<p id="Par251">scrivere qua riassunto Using a significance threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq264"><math id="d33e7265"><mrow><mi>α</mi><mo>=</mo><mn>0.005</mn></mrow></math></span>, only a subset of variables reached statistical significance in univariate tests. However, several others, such as “PHQ-9” in heart failure and “Body Mass Index” in type one diabetes, showed <em>p</em>-values close to the cutoff and may still hold predictive value in multivariate models. Overall, the results align with the original studies: key predictors such as “MYCN status” in neuroblastoma and “SOFA” in sepsis were confirmed. Some expected associations did not reach significance, likely due to the conservative threshold and the univariate nature of the analysis. In absence of normality assumptions in the original sources, non-parametric methods were used, and comparisons with the conventional <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq265"><math id="d33e7280"><mrow><mi>α</mi><mo>=</mo><mn>0.05</mn></mrow></math></span> suggest that certain borderline variables could be reconsidered in broader modeling frameworks. Consistencies with external studies support the clinical relevance of several findings, even when <em>p</em>-values were not below the threshold.</p></section></section></section><section id="Sec17"><h2 class="pmc_sec_title">Discussion and conclusions</h2>
<section id="FPar18"><h3 class="pmc_sec_title">A manual on univariate statistical tests</h3>
<p id="Par252">In an era where machine learning and artificial intelligence are becoming increasingly prevalent, biostatistics can serve as a useful and simpler tool for assessing the validity of scientific results obtained through computational analyses. Biostatistics offers a range of statistical tests that are low in computational cost and can be executed quickly, providing researchers with insights into potential strong relationships between two samples. However, these statistical tests are often chosen incorrectly, applied out of context, or used in a misleading manner, particularly in biomedical informatics. In this study, we try to alleviate this problem by analyzing the four most common univariate statistical tests that we have employed and observed in our scientific activities: Student’s <em>t</em>-test, Chi-squared test, Kruskal-Wallis test, and Mann-Whitney <em>U</em> test. Our goal is to bring some order to this topic and provide a general guide on when to choose one test over another.</p>
<p>We first introduce these four biostatistics tests by explaining their mathematical properties, assumptions, produced statistics, and the meanings of their <em>p</em>-values. Then we provide some guidelines on when to choose which test, based on the datatypes of the samples considered. Afterwards, moving from theory to practice we applied these four tests to artificial data and to real-world medical data derived from electronic medical records. This way, we displayed the behavior of the analyzed four tests on a practical application.
</p>
<section class="tw xbox font-sm" id="Tab13"><h4 class="obj_head">Table 13.</h4>
<div class="caption p"><p>Description of the variables used in the Neuroblastoma dataset. The target variable we consider is “outcome”, used to assess survival</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Variable</th>
<th align="left" colspan="1" rowspan="1">Description and possible values</th>
<th align="left" colspan="1" rowspan="1">Type</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Age</td>
<td align="left" colspan="1" rowspan="1">Any positive integer</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sex</td>
<td align="left" colspan="1" rowspan="1">0 = Female, 1 = Male</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Site</td>
<td align="left" colspan="1" rowspan="1">0, 1, 2,... (codes for tumor primary sites)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Stage</td>
<td align="left" colspan="1" rowspan="1">1, 2, 3, 4</td>
<td align="left" colspan="1" rowspan="1">Ordinal</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Risk</td>
<td align="left" colspan="1" rowspan="1">0 = Low risk, 1 = High risk</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">time months</td>
<td align="left" colspan="1" rowspan="1">Any positive integer</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">autologous stem cell transplantation</td>
<td align="left" colspan="1" rowspan="1">0 = No, 1 = Yes</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">radiation</td>
<td align="left" colspan="1" rowspan="1">0 = No, 1 = Yes</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">degree of differentiation</td>
<td align="left" colspan="1" rowspan="1">0 = Well differentiated, 1 = Moderately differentiated, 2 = Poorly differentiated</td>
<td align="left" colspan="1" rowspan="1">Ordinal</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UH or FH</td>
<td align="left" colspan="1" rowspan="1">0 = Absent, 1 = Present</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MYCN status</td>
<td align="left" colspan="1" rowspan="1">0 = Normal, 1 = Amplified</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">surgical methods</td>
<td align="left" colspan="1" rowspan="1">0 = No surgery, 1 = Surgery performed</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">outcome</td>
<td align="left" colspan="1" rowspan="1">0 = Not survived, 1 = Survived</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab14"><h4 class="obj_head">Table 14.</h4>
<div class="caption p"><p>Description of the variables used in the Sepsis &amp; SIRS dataset. The target variable we consider is “diagnosis_0EC_1M_2AC”, used to assess the diagnosis in the patients</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Variable</th>
<th align="left" colspan="1" rowspan="1">Description and possible values</th>
<th align="left" colspan="1" rowspan="1">Type</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Age</td>
<td align="left" colspan="1" rowspan="1">Patient’s age in years (positive integers)</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">sex_woman</td>
<td align="left" colspan="1" rowspan="1">0 = Male, 1 = Female</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">diagnosis 0EC 1M 2AC</td>
<td align="left" colspan="1" rowspan="1">0 = Elective surgery, 1 = Medical disease, 2 = Emergency surgery</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">APACHE II</td>
<td align="left" colspan="1" rowspan="1">Acute Physiology and Chronic Health Evaluation score at ICU admission</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SOFA</td>
<td align="left" colspan="1" rowspan="1">Sequential Organ Failure Assessment score at ICU admission</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CRP</td>
<td align="left" colspan="1" rowspan="1">C-Reactive Protein measured in mg/dL at ICU admission</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WBCC</td>
<td align="left" colspan="1" rowspan="1">White Blood Cell Count measured in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq266"><math id="d33e7490"><msup><mn>10</mn><mn>3</mn></msup></math></span> cells/<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq267"><math id="d33e7501"><mi>μ</mi></math></span>L at ICU admission</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NeuC</td>
<td align="left" colspan="1" rowspan="1">Neutrophil Count measured in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq268"><math id="d33e7518"><msup><mn>10</mn><mn>3</mn></msup></math></span> cells/<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq269"><math id="d33e7529"><mi>μ</mi></math></span>L at ICU admission</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LymC</td>
<td align="left" colspan="1" rowspan="1">Lymphocyte Count measured in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq270"><math id="d33e7546"><msup><mn>10</mn><mn>3</mn></msup></math></span> cells/<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq271"><math id="d33e7557"><mi>μ</mi></math></span>L at ICU admission</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EOC</td>
<td align="left" colspan="1" rowspan="1">Eosinophil Count measured in cells/<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq272"><math id="d33e7574"><mi>μ</mi></math></span>L at ICU admission</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">NLCR</td>
<td align="left" colspan="1" rowspan="1">Neutrophil-to-Lymphocyte Count Ratio at ICU admission</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PLTC</td>
<td align="left" colspan="1" rowspan="1">Platelet Count measured in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq273"><math id="d33e7599"><msup><mn>10</mn><mn>3</mn></msup></math></span> cells/<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq274"><math id="d33e7610"><mi>μ</mi></math></span>L at ICU admission</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MPV</td>
<td align="left" colspan="1" rowspan="1">Mean Platelet Volume measured in femtoliters (fL) at ICU admission</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Group</td>
<td align="left" colspan="1" rowspan="1">0 = Non-sepsis SIRS, 1 = Sepsis</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LOS-ICU</td>
<td align="left" colspan="1" rowspan="1">Length of ICU stay in days (positive integers)</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mortality</td>
<td align="left" colspan="1" rowspan="1">0 = Alive at ICU discharge, 1 = Deceased during ICU stay</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab14/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab15"><h4 class="obj_head">Table 15.</h4>
<div class="caption p"><p>Description of the variables used in the Depression and Heart Failure dataset. The target variable we consider is “Death”, used to assess death within two years</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Variable</th>
<th align="left" colspan="1" rowspan="1">Description and possible values</th>
<th align="left" colspan="1" rowspan="1">Type</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Age (years)</td>
<td align="left" colspan="1" rowspan="1">Age in years</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Male</td>
<td align="left" colspan="1" rowspan="1">Patient gender (1 = Male, 0 = Female)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PHQ-9</td>
<td align="left" colspan="1" rowspan="1">Patient Health Questionnaire score</td>
<td align="left" colspan="1" rowspan="1">Ordinal</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Systolic BP (mm Hg)</td>
<td align="left" colspan="1" rowspan="1">Systolic blood pressure in mm Hg</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Estimated glomerular filtration rate</td>
<td align="left" colspan="1" rowspan="1">Estimated glomerular filtration rate</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Ejection fraction (%)</td>
<td align="left" colspan="1" rowspan="1">Heart ejection fraction (percentage)</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Serum sodium (mmol/l)</td>
<td align="left" colspan="1" rowspan="1">Blood sodium concentration in mmol/l</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Blood urea nitrogen (mg/dl)</td>
<td align="left" colspan="1" rowspan="1">Blood urea nitrogen in mg/dl</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Etiology HF</td>
<td align="left" colspan="1" rowspan="1">Etiology of heart failure (1 = Yes, 0 = No)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Prior diabetes mellitus</td>
<td align="left" colspan="1" rowspan="1">Presence of pre-existing diabetes mellitus</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Elevated level of BNP/NT-BNP</td>
<td align="left" colspan="1" rowspan="1">Elevated BNP or NT-BNP levels (1 = Yes, 0 = No)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Time from HF to Death (days)</td>
<td align="left" colspan="1" rowspan="1">Time from heart failure to death (in days)</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Death</td>
<td align="left" colspan="1" rowspan="1">Death within two years (1 = Yes, 0 = No)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Time from HF to hospitalization (days)</td>
<td align="left" colspan="1" rowspan="1">Time from heart failure to hospitalization (in days)</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Hospitalized</td>
<td align="left" colspan="1" rowspan="1">Hospitalization within two years (1 = Yes, 0 = No)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab15/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab16"><h4 class="obj_head">Table 16.</h4>
<div class="caption p"><p>Description of the variables used in the Cardiac Arrest dataset. The target variable we consider is “Exitus”, used to assess survival before hospital arrival</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Variable</th>
<th align="left" colspan="1" rowspan="1">Description and possible values</th>
<th align="left" colspan="1" rowspan="1">Type</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Exitus</td>
<td align="left" colspan="1" rowspan="1">Outcome of cardiac arrest (0 = Deceased, 1 = Survived)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">sex woman</td>
<td align="left" colspan="1" rowspan="1">Patient’s sex (0 = Male, 1 = Female)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Age years</td>
<td align="left" colspan="1" rowspan="1">Patient’s age in years (positive integers)</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Endotracheal intubation</td>
<td align="left" colspan="1" rowspan="1">Endotracheal intubation performed (0 = No, 1 = Yes)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Functional_status</td>
<td align="left" colspan="1" rowspan="1">Pre-arrest functional status (scale from 0 to 3)</td>
<td align="left" colspan="1" rowspan="1">Ordinal</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Asystole</td>
<td align="left" colspan="1" rowspan="1">Presence of asystole (0 = No, 1 = Yes)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cardiac arrest at home</td>
<td align="left" colspan="1" rowspan="1">Location of cardiac arrest (0 = Other, 1 = Home)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Bystander</td>
<td align="left" colspan="1" rowspan="1">Bystander CPR provided (0 = No, 1 = Yes)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Time min</td>
<td align="left" colspan="1" rowspan="1">Time from arrest to EMS arrival (in minutes)</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cardiogenic</td>
<td align="left" colspan="1" rowspan="1">Cardiac etiology of arrest (0 = No, 1 = Yes)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab16/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab17"><h4 class="obj_head">Table 17.</h4>
<div class="caption p"><p>Description of the variables used in the Diabete Type One dataset. The target variable we consider is “perc.body.fat”, used to assess the body fat percentage</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Variable</th>
<th align="left" colspan="1" rowspan="1">Description and possible values</th>
<th align="left" colspan="1" rowspan="1">Type</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">age</td>
<td align="left" colspan="1" rowspan="1">Patient’s age in years</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">duration of diabetes</td>
<td align="left" colspan="1" rowspan="1">Duration of diabetes in years</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">body mass index</td>
<td align="left" colspan="1" rowspan="1">Body Mass Index (BMI)</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">TDD</td>
<td align="left" colspan="1" rowspan="1">Total daily dose of insulin</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">basal</td>
<td align="left" colspan="1" rowspan="1">Basal insulin dose</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">bolus</td>
<td align="left" colspan="1" rowspan="1">Rapid-acting (bolus) insulin dose</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HbA1c</td>
<td align="left" colspan="1" rowspan="1">Glycated hemoglobin</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">eGFR</td>
<td align="left" colspan="1" rowspan="1">Estimated glomerular filtration rate</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">perc body fat</td>
<td align="left" colspan="1" rowspan="1">Body fat percentage</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">adiponectin</td>
<td align="left" colspan="1" rowspan="1">Serum adiponectin level</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">free testosterone</td>
<td align="left" colspan="1" rowspan="1">Serum free testosterone level</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SMI</td>
<td align="left" colspan="1" rowspan="1">Skeletal muscle mass index</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">grip strength</td>
<td align="left" colspan="1" rowspan="1">Hand grip strength</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">knee extension strength</td>
<td align="left" colspan="1" rowspan="1">Knee extension strength</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">gait speed</td>
<td align="left" colspan="1" rowspan="1">Walking speed</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ucOC</td>
<td align="left" colspan="1" rowspan="1">Serum undercarboxylated osteocalcin level</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">OC</td>
<td align="left" colspan="1" rowspan="1">Serum osteocalcin level</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">weight kg</td>
<td align="left" colspan="1" rowspan="1">Body weight in kilograms</td>
<td align="left" colspan="1" rowspan="1">Numerical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">insulin regimen binary</td>
<td align="left" colspan="1" rowspan="1">Insulin regimen (0 = No insulin, 1 = Insulin)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">sex 0man 1woman</td>
<td align="left" colspan="1" rowspan="1">Sex (0 = Male, 1 = Female)</td>
<td align="left" colspan="1" rowspan="1">Categorical</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab17/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>This study has highlighted that the choice of statistical test is not arbitrary but instead highly dependent on several key factors. Among these, the type of data—whether numerical, categorical, or ordinal—plays a crucial role. Particular attention should be given to binary variables, for which the decision to treat them as categorical or ordinal depends on both their nature and the context of analysis. Additionally, the normality of numerical data and the sample size significantly influence which test is most appropriate. We believe our study can have a strong impact on the scientific community: potentially, anyone about to use a biostatistics univariate test will be able to take advantage of our study guide to make the proper selection. This idoneous choice, in turn, can help produce more reliable and robust scientific results, increasing the quality of scientific findings in any field. In theory, our study can serve thousands of scientific researchers worldwide.</p>
<p>We invite anyone performing a scientific study involving univariate statistical tests to take into consideration this present guide, and to use it to carefully choose the most suitable test for their case. The flowchart listed before (Figs. <a href="#Fig1" class="usa-link">1</a> and <a href="#Fig2" class="usa-link">2</a>) should guide any researcher before making a decision on which univariate statistical test to employ in a specific scenario-</p></section><section id="FPar19"><h3 class="pmc_sec_title">Limitations</h3>
<p id="Par260">Several limitations of the study must be acknowledged. First, the analysis was restricted to data derived exclusively from electronic health records, thereby excluding other potentially informative sources. Second, the study focused on only five datasets, which constitutes a relatively small sample and may limit the robustness and generalizability of the findings.</p>
<p>Our choice of using 50 as a threshold for the normality testing can also be questioned. We selected this number based on a traditional choice carried out in the biostatistics community [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>, <a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>], but we acknowledge that other values can be utilized for this purpose as well.</p>
<p>The scope of the study was intentionally narrowed to just four well-known univariate tests: the Student’s <em>t</em>-test, the Mann-Whitney <em>U</em> test, the Chi-squared test, and the Kruskal-Wallis test. This focus excluded other valid approaches that might be suitable for specific data types or research questions. Furthermore, we did not comnpare the results of our tests on artificial data and on medical data with any ground truth, to quantify how much they were right. We did not have this possibility because a real ground truth for these data does not exist. For the medical data, we found evidence about the associations between clinical features in the scientific literature, but we could not state how correct the tests’ <em>p</em>-values were.</p></section><section id="FPar20"><h3 class="pmc_sec_title">Future developments</h3>
<p id="Par263">Future research could aim to generalize these results beyond the medical domain and apply them to datasets not derived from electronic health records. Expanding the range of tests and software tools considered could also provide a more comprehensive evaluation framework and offer deeper insights into the practical implications of test selection in applied research. The present guide is focused on the univariate tests; we envision a future new study on multivariate tests, such as the Hotelling’s <em>t</em>-squared test [<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>].</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors acknowldedge the use of Ecosia AI Chat for English proof-reading of the text of this article.</p>
<section id="FPar21"><h3 class="pmc_sec_title">Software code availability</h3>
<p id="Par264">Our software code is publicly available under the GPL-3.0 license on GitHub at <a href="https://github.com/AndreaSichenze/Biostatistics_tests" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/AndreaSichenze/Biostatistics_tests</a> and on Zenodo at <a href="https://doi.org/10.5281/zenodo.15544014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15544014</a>.</p></section></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>ALT</dt>
<dd><p id="Par2">Alanine aminotransferase</p></dd>
<dt>ANOVA</dt>
<dd><p id="Par3">Analysis of variance</p></dd>
<dt>BMI</dt>
<dd><p id="Par4">Body mass index</p></dd>
<dt>BRCA</dt>
<dd><p id="Par5">Breast cancer gene</p></dd>
<dt>CPR</dt>
<dd><p id="Par6">Cardiopulmonary resuscitation</p></dd>
<dt>COVID-19</dt>
<dd><p id="Par7">Coronavirus disease 2019</p></dd>
<dt>DNA</dt>
<dd><p id="Par8">Deoxyribonucleic acid</p></dd>
<dt>EHRs</dt>
<dd><p id="Par9">Electronic health records</p></dd>
<dt>FH</dt>
<dd><p id="Par10">Familial hypercholesterolemia</p></dd>
<dt>FIT</dt>
<dd><p id="Par11">Fecal immunochemical test</p></dd>
<dt>GCA</dt>
<dd><p id="Par12">Gastric cardia adenocarcinoma</p></dd>
<dt>GPL-3</dt>
<dd><p id="Par13">GNU General Public License version 3</p></dd>
<dt>HER2</dt>
<dd><p id="Par14">Human epidermal growth factor receptor 2</p></dd>
<dt>HIE</dt>
<dd><p id="Par15">Health information exchange</p></dd>
<dt>HPV</dt>
<dd><p id="Par16">Human papillomavirus</p></dd>
<dt>IDC</dt>
<dd><p id="Par17">Invasive ductal carcinoma</p></dd>
<dt>IMPC</dt>
<dd><p id="Par18">Invasive micropapillary carcinoma</p></dd>
<dt>KW</dt>
<dd><p id="Par19">Kruskal-Wallis test</p></dd>
<dt>LDL-C</dt>
<dd><p id="Par20">Low-density lipoprotein cholesterol</p></dd>
<dt>MS</dt>
<dd><p id="Par21">Multiple sclerosis</p></dd>
<dt>MYCN</dt>
<dd><p id="Par22">MYCN Proto-Oncogene, BHLH Transcription Factor</p></dd>
<dt>MW</dt>
<dd><p id="Par23">Mann-Whitney <em>U</em> test</p></dd>
<dt>PACU</dt>
<dd><p id="Par24">Post-anesthesia care unit</p></dd>
<dt>PHQ-9</dt>
<dd><p id="Par25">Patient health questionnaire-9</p></dd>
<dt>PROM</dt>
<dd><p id="Par26">Patient-reported outcome measurement</p></dd>
<dt>ROC</dt>
<dd><p id="Par27">Receiver operating characteristic curve</p></dd>
<dt>SL</dt>
<dd><p id="Par28">Serum levels</p></dd>
<dt>SSIs</dt>
<dd><p id="Par29">Surgical site infections</p></dd>
<dt>TCGA</dt>
<dd><p id="Par30">The Cancer Genome Atlas</p></dd>
<dt>TMAO</dt>
<dd><p id="Par31">Trimethylamine n-oxide</p></dd>
<dt>WM</dt>
<dd><p id="Par32">Weight management</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>D.C. conceived and supervised the study, contributed to the writing of the manuscript, and reviewed the manuscript. A.S. performed the tests, wrote the software code, designed the experiments, wrote the literature review, wrote several parts of the manuscript, and reviewed the manuscript. G.J. supervised the study, provided feedback on the article’s contents, and reviewed the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>The work of D.C. is partially funded by the Italian Ministero Italiano delle Imprese e del Made in Italy under the Digital Intervention in Psychiatric and Psychologist Services (DIPPS) programme and is partially supported by Ministero dell’Università e della Ricerca of Italy under the “Dipartimenti di Eccellenza 2023-2027” ReGAInS grant assigned to Dipartimento di Informatica Sistemistica e Comunicazione at Università di Milano-Bicocca. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data availability The EHRs datasets employed in this study are publicly available in the supplementary ma- terials of the original dataset publications [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>, <a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>, <a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>–<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>] under the CC BY 4.0 license at the following URLs:</p>
<p> – Sepsis and SIRS [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>] <a href="https://figshare.com/articles/dataset/_C_Reactive_Protein_and_Hemogram_Parameters_for_the_Non_Sepsis_Systemic_Inflammatory_Response_Syndrome_and_Sepsis_What_Do_They_Mean_/1644426?file=2637248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/_C_Reactive_Protein_and_Hemogram_Parameters_for_the_Non_Sepsis_Systemic_Inflammatory_Response_Syndrome_and_Sepsis_What_Do_They_Mean_/1644426?file=2637248</a></p>
<p> – Depression and Heart Failure [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>] <a href="https://figshare.com/articles/dataset/Comorbid_Depression_and_Heart_Failure_A_Community_Cohort_Study/3916224?file=6130425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Comorbid_Depression_and_Heart_Failure_A_Community_Cohort_Study/3916224?file=6130425</a></p>
<p> – Cardiac Arrest [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>] <a href="https://figshare.com/articles/dataset/Mortality_after_out-of-hospital_cardiac_arrest_in_a_Spanish_Region/4876247?file=8166893" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Mortality_after_out-of-hospital_cardiac_arrest_in_a_Spanish_Region/4876247?file=8166893</a></p>
<p> – Neuroblastoma [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>] <a href="https://doi.org/10.7717/peerj.5665/supp-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.7717/peerj.5665/supp-5</a></p>
<p> – Diabetes Type One [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>] <a href="https://figshare.com/articles/dataset/Circulating_osteocalcin_as_a_bone-derived_hormone_is_inversely_correlated_with_body_fat_in_patients_with_type_1_diabetes/8079389?file=15057092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Circulating_osteocalcin_as_a_bone-derived_hormone_is_inversely_correlated_with_body_fat_in_patients_with_type_1_diabetes/8079389?file=15057092</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar22"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par265">Permission to collect and analyze the data of patients’ involved in this study has been obtained by the original datasets curators from ethical committees of their hospitals, as stated in the [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>, <a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>, <a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>–<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>] original articles.</p></section><section id="FPar23"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par266">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Rajkomar A, Dean J, Kohane I. Machine learning in medicine. N Engl J Med. 2019;380(14):1347–58. 10.1056/nejmra1814259.
</cite> [<a href="https://doi.org/10.1056/NEJMra1814259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30943338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rajkomar%20A,%20Dean%20J,%20Kohane%20I.%20Machine%20learning%20in%20medicine.%20N%20Engl%20J%20Med.%202019;380(14):1347%E2%80%9358.%2010.1056/nejmra1814259." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Alves R, Pasquier C, Pasquier N. The pervasiveness of machine learning in omics science. In: ECML PKDD 2014 – International Conference on Machine Learning and Principles and Practice of Knowledge Discovery in Databases Tutorial T3. 2014. <a href="https://hal.science/hal-01330594v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hal.science/hal-01330594v1</a>. Accessed 1st July 2025.</cite>
</li>
<li id="CR3">
<span class="label">3.</span><cite>Sambasivan N, Kapania S, Highfill H, Akrong D, Paritosh P, Aroyo LM. “Everyone wants to do the model work, not the data work”: data cascades in High-Stakes AI. In: Proceedings of CHI ’21 – the 2021 CHI Conference on Human Factors in Computing Systems. ACM; 2021. pp. 1–15. 10.1145/3411764.3445518.</cite>
</li>
<li id="CR4">
<span class="label">4.</span><cite>Rudin C. Stop explaining black box machine learning models for high stakes decisions and use interpretable models instead. Nat Mach Intel. 2019;1(5):206–15. 10.1038/s42256-019-0048-x.</cite> [<a href="https://doi.org/10.1038/s42256-019-0048-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9122117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35603010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rudin%20C.%20Stop%20explaining%20black%20box%20machine%20learning%20models%20for%20high%20stakes%20decisions%20and%20use%20interpretable%20models%20instead.%20Nat%20Mach%20Intel.%202019;1(5):206%E2%80%9315.%2010.1038/s42256-019-0048-x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Gosset WSS. The probable error of a mean. Biometrika. 1908:1–25. 10.2307/2331554.</cite>
</li>
<li id="CR6">
<span class="label">6.</span><cite>DATAtab Team. t-Test. 2025. <a href="https://datatab.net/tutorial/t-test" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://datatab.net/tutorial/t-test</a>. URL visited on 2nd May 2025. Accessed 1st July 2025.</cite>
</li>
<li id="CR7">
<span class="label">7.</span><cite>Mann HB, Whitney DR. On a test of whether one of two random variables is stochastically larger than the other. Ann Math Stat. 1947:50–60. <a href="https://www.jstor.org/stable/2236101%20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.jstor.org/stable/2236101</a>. Accessed 1st July 2025.</cite>
</li>
<li id="CR8">
<span class="label">8.</span><cite>DATAtab Team. Mann-Whitney U-Test. 2025. <a href="https://datatab.net/tutorial/mann-whitney-u-test" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://datatab.net/tutorial/mann-whitney-u-test</a>. URL visited on 2nd May 2025. Accessed 1st July 2025.</cite>
</li>
<li id="CR9">
<span class="label">9.</span><cite>Cochran WG. The <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ecf3/12366075/0acaf120f904/13040_2025_465_Article_IEq63.gif" loading="lazy" alt="Inline graphic"> Test of Goodness of Fit. Ann Math Stat. 1952;23:315–345. <a href="https://api.semanticscholar.org/CorpusID:121913312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://api.semanticscholar.org/CorpusID:121913312</a>. Accessed 1st July 2025.</cite>
</li>
<li id="CR10">
<span class="label">10.</span><cite>DATAtab Team. Chi-Square test. 2025. <a href="https://datatab.net/tutorial/chi-square-test" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://datatab.net/tutorial/chi-square-test</a>. URL visited on 2nd May 2025. Accessed 1st July 2025.</cite>
</li>
<li id="CR11">
<span class="label">11.</span><cite>Kruskal WH, Wallis WA. Use of Ranks in One-Criterion Variance Analysis. J Am Stat Assoc. 1952;47:583–621. <a href="https://api.semanticscholar.org/CorpusID:51902974" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://api.semanticscholar.org/CorpusID:51902974</a>. Accessed 1st July 2025.</cite>
</li>
<li id="CR12">
<span class="label">12.</span><cite>DATAtab Team. Kruskal-Wallis-Test. 2025. <a href="https://datatab.net/tutorial/kruskal-wallis-test" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://datatab.net/tutorial/kruskal-wallis-test</a>. URL visited on 2nd May 2025. Accessed 1st July 2025.</cite>
</li>
<li id="CR13">
<span class="label">13.</span><cite>Moynihan D, Monaco S, Ting TW, Narasimhalu K, Hsieh J, Kam S, et al. Author Correction: Analysis and visualisation of electronic health records data to identify undiagnosed patients with rare genetic diseases. Sci Rep. 2024;14(1):10084. 10.1038/s41598-024-55424-8.
</cite> [<a href="https://doi.org/10.1038/s41598-024-60776-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11066083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38698204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moynihan%20D,%20Monaco%20S,%20Ting%20TW,%20Narasimhalu%20K,%20Hsieh%20J,%20Kam%20S,%20et%20al.%20Author%20Correction:%20Analysis%20and%20visualisation%20of%20electronic%20health%20records%20data%20to%20identify%20undiagnosed%20patients%20with%20rare%20genetic%20diseases.%20Sci%20Rep.%202024;14(1):10084.%2010.1038/s41598-024-55424-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Patton H, Burchette R, Tovar S, Pio J, Shi J, Nyberg LM. Retrospective analysis of a dedicated care pathway for nonalcoholic fatty liver disease in an integrated US healthcare system demonstrates support of weight management and improved ALT. BMC Gastroenterol. 2020;20(1):362. 10.1186/s12876-020-01492-9.
</cite> [<a href="https://doi.org/10.1186/s12876-020-01492-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7603663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33129272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patton%20H,%20Burchette%20R,%20Tovar%20S,%20Pio%20J,%20Shi%20J,%20Nyberg%20LM.%20Retrospective%20analysis%20of%20a%20dedicated%20care%20pathway%20for%20nonalcoholic%20fatty%20liver%20disease%20in%20an%20integrated%20US%20healthcare%20system%20demonstrates%20support%20of%20weight%20management%20and%20improved%20ALT.%20BMC%20Gastroenterol.%202020;20(1):362.%2010.1186/s12876-020-01492-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Blair CK, Wiggins CL, Nibbe AM, Storlie CB, Prossnitz ER, Royce M, et al. Obesity and survival among a cohort of breast cancer patients is partially mediated by tumor characteristics. NPJ Breast Cancer. 2019;5(1):33. 10.1038/s41523-019-0128-4.
</cite> [<a href="https://doi.org/10.1038/s41523-019-0128-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6775111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31602394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blair%20CK,%20Wiggins%20CL,%20Nibbe%20AM,%20Storlie%20CB,%20Prossnitz%20ER,%20Royce%20M,%20et%20al.%20Obesity%20and%20survival%20among%20a%20cohort%20of%20breast%20cancer%20patients%20is%20partially%20mediated%20by%20tumor%20characteristics.%20NPJ%20Breast%20Cancer.%202019;5(1):33.%2010.1038/s41523-019-0128-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Nelson CA, Bove R, Butte AJ, Baranzini SE. Embedding electronic health records onto a knowledge network recognizes prodromal features of multiple sclerosis and predicts diagnosis. J Am Med Inform Assoc. 2021;29(3):424–34. 10.1093/jamia/ocab270.</cite> [<a href="https://doi.org/10.1093/jamia/ocab270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8800523/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34915552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nelson%20CA,%20Bove%20R,%20Butte%20AJ,%20Baranzini%20SE.%20Embedding%20electronic%20health%20records%20onto%20a%20knowledge%20network%20recognizes%20prodromal%20features%20of%20multiple%20sclerosis%20and%20predicts%20diagnosis.%20J%20Am%20Med%20Inform%20Assoc.%202021;29(3):424%E2%80%9334.%2010.1093/jamia/ocab270." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>David CH, Quessard A, Mastroianni C, Hekimian G, Amour J, Leprince P, et al. Mechanical circulatory support with the Impella 5.0 and the Impella Left Direct pumps for postcardiotomy cardiogenic shock at La Pitié-Salpêtrière Hospital. Eur J Cardiothorac Surg. 2019;57(1):183–188. 10.1093/ejcts/ezz179.</cite> [<a href="https://doi.org/10.1093/ejcts/ezz179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31270535/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Gao J, Yan KT, Wang JX, Dou J, Wang J, Ren M, et al. Gut microbial taxa as potential predictive biomarkers for acute coronary syndrome and post-STEMI cardiovascular events. Sci Rep. 2020;10(1):2639. 10.1038/s41598-020-59235-5.
</cite> [<a href="https://doi.org/10.1038/s41598-020-59235-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7021689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32060329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20J,%20Yan%20KT,%20Wang%20JX,%20Dou%20J,%20Wang%20J,%20Ren%20M,%20et%20al.%20Gut%20microbial%20taxa%20as%20potential%20predictive%20biomarkers%20for%20acute%20coronary%20syndrome%20and%20post-STEMI%20cardiovascular%20events.%20Sci%20Rep.%202020;10(1):2639.%2010.1038/s41598-020-59235-5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Issaka RB, Singh MH, Oshima SM, Laleau VJ, Rachocki CD, Chen EH, et al. Inadequate Utilization of Diagnostic Colonoscopy Following Abnormal FIT Results in an Integrated Safety-Net System. Am J Gastroenterol. 2016;112(2):375–82. 10.1038/ajg.2016.555.
</cite> [<a href="https://doi.org/10.1038/ajg.2016.555" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6597438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28154400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Issaka%20RB,%20Singh%20MH,%20Oshima%20SM,%20Laleau%20VJ,%20Rachocki%20CD,%20Chen%20EH,%20et%20al.%20Inadequate%20Utilization%20of%20Diagnostic%20Colonoscopy%20Following%20Abnormal%20FIT%20Results%20in%20an%20Integrated%20Safety-Net%20System.%20Am%20J%20Gastroenterol.%202016;112(2):375%E2%80%9382.%2010.1038/ajg.2016.555." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Storm AC, Ryou M, Thompson CC. Multicenter implementation of a new electronic medical record system leads to longer procedure times and poor staff satisfaction. Clin Endosc. 2019;52(1):87–9. 10.5946/ce.2018.080.
</cite> [<a href="https://doi.org/10.5946/ce.2018.080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6370922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30130844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Storm%20AC,%20Ryou%20M,%20Thompson%20CC.%20Multicenter%20implementation%20of%20a%20new%20electronic%20medical%20record%20system%20leads%20to%20longer%20procedure%20times%20and%20poor%20staff%20satisfaction.%20Clin%20Endosc.%202019;52(1):87%E2%80%939.%2010.5946/ce.2018.080." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Zhou JZ, Liu X, Ye GJ. The impact of workplace bullying on depression among clinical nurses in China: a comparative analysis. Medicine. 2025;104(2). 10.1097/md.0000000000041246.</cite> [<a href="https://doi.org/10.1097/MD.0000000000041246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11729269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39792771/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Wang G, Liu L, Zhao Y, Lin Y, Er L. Comparative genomic analysis unveiling the mutational landscape associated with premalignant lesions and early-stage gastric cardia cancer. Medicine. 2025;104(2). 10.1097/md.0000000000040332.</cite> [<a href="https://doi.org/10.1097/MD.0000000000040332" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11731115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39792765/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Do TM, Nguyen QHN, Le NHD, Nguyen HD, Phung AHT, Tran TS, et al. Association between dietary factors and breast cancer risk: a matched case-control study in Vietnam. BMC Cancer. 2024;24(1):1224. 10.1186/s12885-024-12918-y.
</cite> [<a href="https://doi.org/10.1186/s12885-024-12918-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11448273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39363187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Do%20TM,%20Nguyen%20QHN,%20Le%20NHD,%20Nguyen%20HD,%20Phung%20AHT,%20Tran%20TS,%20et%20al.%20Association%20between%20dietary%20factors%20and%20breast%20cancer%20risk:%20a%20matched%20case-control%20study%20in%20Vietnam.%20BMC%20Cancer.%202024;24(1):1224.%2010.1186/s12885-024-12918-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Lapin BR, Honomichl RD, Thompson NR, Rose S, Sugano D, Udeh B, et al. Association Between Patient Experience With Patient-Reported Outcome Measurements and Overall Satisfaction With Care in Neurology. Value Health. 2019;22(5):555–63. 10.1016/j.jval.2019.02.007.
</cite> [<a href="https://doi.org/10.1016/j.jval.2019.02.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31104734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lapin%20BR,%20Honomichl%20RD,%20Thompson%20NR,%20Rose%20S,%20Sugano%20D,%20Udeh%20B,%20et%20al.%20Association%20Between%20Patient%20Experience%20With%20Patient-Reported%20Outcome%20Measurements%20and%20Overall%20Satisfaction%20With%20Care%20in%20Neurology.%20Value%20Health.%202019;22(5):555%E2%80%9363.%2010.1016/j.jval.2019.02.007." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Lugo-Perez S, Azpiri-Lopez JR, Galarza-Delgado DA, Colunga-Pedraza IJ, Cardenas-De La Garza JA, Arvizu-Rivera RI, et al. Cardiac remodeling in rheumatoid arthritis: assessing the influence of anti-CCP and rheumatoid factor. Eur Heart J. 2024;45(Supplement_1):ehae666.2729. 10.1093/eurheartj/ehae666.2729.</cite>
</li>
<li id="CR26">
<span class="label">26.</span><cite>Steinbruck I, Ebigbo A, Kuellmer A, Schmidt A, Kouladouros K, Brand M, et al. Cold Versus Hot Snare Endoscopic Resection of Large Nonpedunculated Colorectal Polyps: randomized Controlled German CHRONICLE Trial. Gastroenterology. 2024;167(4):764–77. 10.1053/j.gastro.2024.05.013.
</cite> [<a href="https://doi.org/10.1053/j.gastro.2024.05.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38795735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Steinbruck%20I,%20Ebigbo%20A,%20Kuellmer%20A,%20Schmidt%20A,%20Kouladouros%20K,%20Brand%20M,%20et%20al.%20Cold%20Versus%20Hot%20Snare%20Endoscopic%20Resection%20of%20Large%20Nonpedunculated%20Colorectal%20Polyps:%20randomized%20Controlled%20German%20CHRONICLE%20Trial.%20Gastroenterology.%202024;167(4):764%E2%80%9377.%2010.1053/j.gastro.2024.05.013." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Qian Y, Yuan L, Zhang X. Comparative study on blood pressure and metabolic improvements in hypertensive patients using copper bianstone scraping. Medicine. 2025;104(2). 10.1097/md.0000000000041133.</cite> [<a href="https://doi.org/10.1097/MD.0000000000041133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11730402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39792768/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Meng J, Zhu Y, Li Y, Sun T, Zhang F, Qin S, et al. Incidence and risk factors for surgical site infection following elective foot and ankle surgery: a retrospective study. J Orthop Surg Res. 2020;15(1):449. 10.1186/s13018-020-01972-4.
</cite> [<a href="https://doi.org/10.1186/s13018-020-01972-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7528469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33004062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meng%20J,%20Zhu%20Y,%20Li%20Y,%20Sun%20T,%20Zhang%20F,%20Qin%20S,%20et%20al.%20Incidence%20and%20risk%20factors%20for%20surgical%20site%20infection%20following%20elective%20foot%20and%20ankle%20surgery:%20a%20retrospective%20study.%20J%20Orthop%20Surg%20Res.%202020;15(1):449.%2010.1186/s13018-020-01972-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Lapin B, Udeh B, Bautista JF, Katzan IL. Patient experience with patient-reported outcome measures in neurologic practice. Neurology. 2018;91(12):e1135–51. 10.1212/wnl.0000000000006198.
</cite> [<a href="https://doi.org/10.1212/WNL.0000000000006198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30135254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lapin%20B,%20Udeh%20B,%20Bautista%20JF,%20Katzan%20IL.%20Patient%20experience%20with%20patient-reported%20outcome%20measures%20in%20neurologic%20practice.%20Neurology.%202018;91(12):e1135%E2%80%9351.%2010.1212/wnl.0000000000006198." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Esaki M, Suzuki S, Hayashi Y, Yokoyama A, Abe S, Hosokawa T, et al. Propensity score-matching analysis to compare clinical outcomes of endoscopic submucosal dissection for early gastric cancer in the postoperative and non-operative stomachs. BMC Gastroenterol. 2018;18(1):125. 10.1186/s12876-018-0855-2.
</cite> [<a href="https://doi.org/10.1186/s12876-018-0855-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6080519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30081824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Esaki%20M,%20Suzuki%20S,%20Hayashi%20Y,%20Yokoyama%20A,%20Abe%20S,%20Hosokawa%20T,%20et%20al.%20Propensity%20score-matching%20analysis%20to%20compare%20clinical%20outcomes%20of%20endoscopic%20submucosal%20dissection%20for%20early%20gastric%20cancer%20in%20the%20postoperative%20and%20non-operative%20stomachs.%20BMC%20Gastroenterol.%202018;18(1):125.%2010.1186/s12876-018-0855-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Zelviene A, Bogusevicius A. Reliability and validity of the Champion’s Health Belief Model Scale among Lithuanian women. Cancer Nurs. 2007;30(3):E20-8. 10.1097/01.ncc.0000270711.72413.a6.
</cite> [<a href="https://doi.org/10.1097/01.NCC.0000270711.72413.a6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17510578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zelviene%20A,%20Bogusevicius%20A.%20Reliability%20and%20validity%20of%20the%20Champion%E2%80%99s%20Health%20Belief%20Model%20Scale%20among%20Lithuanian%20women.%20Cancer%20Nurs.%202007;30(3):E20-8.%2010.1097/01.ncc.0000270711.72413.a6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Cerulo L, Pagnotta SM. massiveGST: a Mann-Whitney-Wilcoxon gene-set test tool that gives meaning to gene-set enrichment analysis. Entropy. 2022;24(5):739. 10.3390/e24050739.
</cite> [<a href="https://doi.org/10.3390/e24050739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9140214/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35626622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cerulo%20L,%20Pagnotta%20SM.%20massiveGST:%20a%20Mann-Whitney-Wilcoxon%20gene-set%20test%20tool%20that%20gives%20meaning%20to%20gene-set%20enrichment%20analysis.%20Entropy.%202022;24(5):739.%2010.3390/e24050739." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Zhu S, Tong Y, Chen W, Chen X, Shen K. Association of obesity and luminal subtypes in prognosis and adjuvant endocrine treatment effectiveness prediction in Chinese breast cancer patients. 2021. 10.3389/fonc.2022.862224.</cite> [<a href="https://doi.org/10.3389/fonc.2022.862224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9117630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35600356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20S,%20Tong%20Y,%20Chen%20W,%20Chen%20X,%20Shen%20K.%20Association%20of%20obesity%20and%20luminal%20subtypes%20in%20prognosis%20and%20adjuvant%20endocrine%20treatment%20effectiveness%20prediction%20in%20Chinese%20breast%20cancer%20patients.%202021.%2010.3389/fonc.2022.862224." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Zhao D, Wang X, Beeraka NM, Zhou R, Zhang H, Liu Y, et al. High body mass index was associated with human epidermal growth factor receptor 2-positivity, histological grade and disease progression differently by age. World J Oncol. 2023;14(1):75–83. 10.14740/wjon1543.</cite> [<a href="https://doi.org/10.14740/wjon1543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9990731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36895993/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Leone JP, Leone J, Hassett MJ, Freedman RA, Avila J, Vallejo CT, et al. Incidence, treatment patterns, and mortality for patients with breast cancer during the first year of the COVID-19 pandemic: a population-based study. Breast Cancer Res Treat. 2024. 10.1007/s10549-024-07562-w.
</cite> [<a href="https://doi.org/10.1007/s10549-024-07562-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39621161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leone%20JP,%20Leone%20J,%20Hassett%20MJ,%20Freedman%20RA,%20Avila%20J,%20Vallejo%20CT,%20et%20al.%20Incidence,%20treatment%20patterns,%20and%20mortality%20for%20patients%20with%20breast%20cancer%20during%20the%20first%20year%20of%20the%20COVID-19%20pandemic:%20a%20population-based%20study.%20Breast%20Cancer%20Res%20Treat.%202024.%2010.1007/s10549-024-07562-w." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>RECOVERY Collaborative Group. Convalescent plasma in patients admitted to hospital with COVID-19 (RECOVERY): a randomised controlled, open-label, platform trial. Lancet. 2021;397(10289):2049–59. 10.1016/S0140-6736(21)00897-7.
</cite> [<a href="https://doi.org/10.1016/S0140-6736(21)00897-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8121538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34000257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?RECOVERY%20Collaborative%20Group.%20Convalescent%20plasma%20in%20patients%20admitted%20to%20hospital%20with%20COVID-19%20(RECOVERY):%20a%20randomised%20controlled,%20open-label,%20platform%20trial.%20Lancet.%202021;397(10289):2049%E2%80%9359.%2010.1016/S0140-6736(21)00897-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Jin J, Li B, Cao J, Li T, Zhang J, Cao J, et al. Analysis of clinical features, genomic landscapes and survival outcomes in HER2-low breast cancer. J Transl Med. 2023;21(1):360. 10.1186/s12967-023-04076-9.
</cite> [<a href="https://doi.org/10.1186/s12967-023-04076-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10236705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37264417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin%20J,%20Li%20B,%20Cao%20J,%20Li%20T,%20Zhang%20J,%20Cao%20J,%20et%20al.%20Analysis%20of%20clinical%20features,%20genomic%20landscapes%20and%20survival%20outcomes%20in%20HER2-low%20breast%20cancer.%20J%20Transl%20Med.%202023;21(1):360.%2010.1186/s12967-023-04076-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Magnoni F, Bianchi B, Pagan E, Corso G, Sala I, Bagnardi V, et al. Long-term outcome of invasive pure micropapillary breast cancer compared with invasive mixed micropapillary and invasive ductal breast cancer: a matched retrospective study. Breast Cancer Res Treat. 2024;208(2):333–47. 10.1007/s10549-024-07422-7.
</cite> [<a href="https://doi.org/10.1007/s10549-024-07422-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38963526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Magnoni%20F,%20Bianchi%20B,%20Pagan%20E,%20Corso%20G,%20Sala%20I,%20Bagnardi%20V,%20et%20al.%20Long-term%20outcome%20of%20invasive%20pure%20micropapillary%20breast%20cancer%20compared%20with%20invasive%20mixed%20micropapillary%20and%20invasive%20ductal%20breast%20cancer:%20a%20matched%20retrospective%20study.%20Breast%20Cancer%20Res%20Treat.%202024;208(2):333%E2%80%9347.%2010.1007/s10549-024-07422-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Tarı Selçuk K, Avcı D, Yılmaz Dündar G, Mercan Y. Breast cancer screening behaviors in women aged 40 years and over in a semi-urban region in Turkey: relationships with health beliefs. Healthcare. 2020;8(2):171. 10.3390/healthcare8020171.
</cite> [<a href="https://doi.org/10.3390/healthcare8020171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7349266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32545805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tar%C4%B1%20Sel%C3%A7uk%20K,%20Avc%C4%B1%20D,%20Y%C4%B1lmaz%20D%C3%BCndar%20G,%20Mercan%20Y.%20Breast%20cancer%20screening%20behaviors%20in%20women%20aged%2040%20years%20and%20over%20in%20a%20semi-urban%20region%20in%20Turkey:%20relationships%20with%20health%20beliefs.%20Healthcare.%202020;8(2):171.%2010.3390/healthcare8020171." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Bhattacharyya O, Rawl SM, Dickinson SL, Haggstrom DA. Comparison of health information exchange data with self-report in measuring cancer screening. BMC Med Res Methodol. 2023;23(1):172. 10.1186/s12874-023-01907-7.
</cite> [<a href="https://doi.org/10.1186/s12874-023-01907-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10367403/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37491208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bhattacharyya%20O,%20Rawl%20SM,%20Dickinson%20SL,%20Haggstrom%20DA.%20Comparison%20of%20health%20information%20exchange%20data%20with%20self-report%20in%20measuring%20cancer%20screening.%20BMC%20Med%20Res%20Methodol.%202023;23(1):172.%2010.1186/s12874-023-01907-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Cengiz B, Bahar Z, Canda AE. The effects of patient care results of applied nursing intervention to individuals with stoma according to the Health Belief Model. Cancer Nurs. 2020;43(2):E87–96. 10.1097/ncc.0000000000000678.
</cite> [<a href="https://doi.org/10.1097/NCC.0000000000000678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30543570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cengiz%20B,%20Bahar%20Z,%20Canda%20AE.%20The%20effects%20of%20patient%20care%20results%20of%20applied%20nursing%20intervention%20to%20individuals%20with%20stoma%20according%20to%20the%20Health%20Belief%20Model.%20Cancer%20Nurs.%202020;43(2):E87%E2%80%9396.%2010.1097/ncc.0000000000000678." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Wu Y, Fan J, Peissig P, Berg R, Tafti AP, Yin J, et al. Quantifying predictive capability of electronic health records for the most harmful breast cancer. In: Medical Imaging 2018: Image Perception, Observer Performance, and Technology Assessment, vol. 10577. SPIE; 2018. pp. 112–120. 10.1117/12.2293954.</cite> [<a href="https://doi.org/10.1117/12.2293954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5914175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29706685/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Clifton JC, Ende HB, Rathnam C, Freundlich RE, Sandberg WS, Wanderer JP. A mobile post anesthesia care unit order reminder system improves timely order entry. J Med Syst. 2024;48(1):60. 10.1007/s10916-024-02079-7.
</cite> [<a href="https://doi.org/10.1007/s10916-024-02079-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11164754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38856813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clifton%20JC,%20Ende%20HB,%20Rathnam%20C,%20Freundlich%20RE,%20Sandberg%20WS,%20Wanderer%20JP.%20A%20mobile%20post%20anesthesia%20care%20unit%20order%20reminder%20system%20improves%20timely%20order%20entry.%20J%20Med%20Syst.%202024;48(1):60.%2010.1007/s10916-024-02079-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Karajizadeh M, Nikandish R, Zalpour Z, Roozrokh Arshadi Montazer M, Soleimanijafarbiglo M, Mazaher Y, et al. Identification of the Information Needs of a Nurse-led Rapid Response Team to Design and Develop an Electronic Medical Record System. Health Manag Inf Sci. 2022;9(4):236–242. 10.30476/jhmi.2023.96214.1143.</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite>Dexter F, Pinho RH, Pang DSJ. Modeling daily veterinary anesthetist patient care hours and probabilities of exceeding critical thresholds. Am J Vet Res. 2024;85(5):1–10. 10.2460/ajvr.23.09.0196.
</cite> [<a href="https://doi.org/10.2460/ajvr.23.09.0196" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38408432/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dexter%20F,%20Pinho%20RH,%20Pang%20DSJ.%20Modeling%20daily%20veterinary%20anesthetist%20patient%20care%20hours%20and%20probabilities%20of%20exceeding%20critical%20thresholds.%20Am%20J%20Vet%20Res.%202024;85(5):1%E2%80%9310.%2010.2460/ajvr.23.09.0196." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Katzan IL, Thompson NR, Dunphy C, Urchek J, Lapin B. Neurologic provider views on patient-reported outcomes including depression screening. Neurol Clin Pract. 2018;8(2):86–92. 10.1212/cpj.0000000000000438.
</cite> [<a href="https://doi.org/10.1212/CPJ.0000000000000438" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5914757/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29708224/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katzan%20IL,%20Thompson%20NR,%20Dunphy%20C,%20Urchek%20J,%20Lapin%20B.%20Neurologic%20provider%20views%20on%20patient-reported%20outcomes%20including%20depression%20screening.%20Neurol%20Clin%20Pract.%202018;8(2):86%E2%80%9392.%2010.1212/cpj.0000000000000438." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Williams DC, Warren RW, Ebeling M, Andrews AL, Teufel RJ II. Physician use of electronic health records: Survey study assessing factors associated with provider reported satisfaction and perceived patient impact. JMIR Med Inform. 2019;7(2):e10949. 10.2196/10949.
</cite> [<a href="https://doi.org/10.2196/10949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6470463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30946023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Williams%20DC,%20Warren%20RW,%20Ebeling%20M,%20Andrews%20AL,%20Teufel%20RJ%20II.%20Physician%20use%20of%20electronic%20health%20records:%20Survey%20study%20assessing%20factors%20associated%20with%20provider%20reported%20satisfaction%20and%20perceived%20patient%20impact.%20JMIR%20Med%20Inform.%202019;7(2):e10949.%2010.2196/10949." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Jabali AK. Predictors of Anesthesiologists’ attitude toward EHRs in Saudi Arabia for clinical practice. Inform Med Unlocked. 2021;23(100555):100555. 10.3389/fdgth.2023.1252227.</cite> [<a href="https://scholar.google.com/scholar_lookup?Jabali%20AK.%20Predictors%20of%20Anesthesiologists%E2%80%99%20attitude%20toward%20EHRs%20in%20Saudi%20Arabia%20for%20clinical%20practice.%20Inform%20Med%20Unlocked.%202021;23(100555):100555.%2010.3389/fdgth.2023.1252227." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Fors M, Ballaz S, Ramírez H, Mora FX, Pulgar-Sánchez M, Chamorro K, et al. Sex-dependent performance of the neutrophil-to-lymphocyte, monocyte-to-lymphocyte, platelet-to-lymphocyte and mean platelet volume-to-platelet ratios in discriminating COVID-19 severity. Front Cardiovasc Med. 2022;9:822556. 10.3389/fcvm.2022.822556.
</cite> [<a href="https://doi.org/10.3389/fcvm.2022.822556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9023889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35463770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fors%20M,%20Ballaz%20S,%20Ram%C3%ADrez%20H,%20Mora%20FX,%20Pulgar-S%C3%A1nchez%20M,%20Chamorro%20K,%20et%20al.%20Sex-dependent%20performance%20of%20the%20neutrophil-to-lymphocyte,%20monocyte-to-lymphocyte,%20platelet-to-lymphocyte%20and%20mean%20platelet%20volume-to-platelet%20ratios%20in%20discriminating%20COVID-19%20severity.%20Front%20Cardiovasc%20Med.%202022;9:822556.%2010.3389/fcvm.2022.822556." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Tollinche LE, Shi R, Hannum M, McCormick P, Thorne A, Tan KS, et al. The impact of real-time clinical alerts on the compliance of anesthesia documentation: A retrospective observational study. Comput Methods Prog Biomed. 2020;191(105399):105399. 10.1016/j.cmpb.2020.105399.</cite> [<a href="https://doi.org/10.1016/j.cmpb.2020.105399" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7283007/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32146211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tollinche%20LE,%20Shi%20R,%20Hannum%20M,%20McCormick%20P,%20Thorne%20A,%20Tan%20KS,%20et%20al.%20The%20impact%20of%20real-time%20clinical%20alerts%20on%20the%20compliance%20of%20anesthesia%20documentation:%20A%20retrospective%20observational%20study.%20Comput%20Methods%20Prog%20Biomed.%202020;191(105399):105399.%2010.1016/j.cmpb.2020.105399." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Shafran-Tikva S, Gabay G, Kagan I. Transformative insights into community-acquired pressure injuries among the elderly: a big data analysis. Healthcare. 2025;13(2). 10.3390/healthcare13020153.</cite> [<a href="https://doi.org/10.3390/healthcare13020153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11764954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39857180/" class="usa-link">PubMed</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Makin TR, Orban de Xivry JJ. Ten common statistical mistakes to watch out for when writing or reviewing a manuscript. eLife. 2019;8:e48175. 10.7554/elife.48175.</cite> [<a href="https://doi.org/10.7554/eLife.48175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6785265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31596231/" class="usa-link">PubMed</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>D’Arrigo G, El Hafeez SA, Mezzatesta S, Abelardo D, Provenzano FP, Vilasi A, et al. Common mistakes in biostatistics. Clin Kidney J. 2024;17(7):sfae197. 10.1093/ckj/sfae197.</cite> [<a href="https://doi.org/10.1093/ckj/sfae197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11333961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39165900/" class="usa-link">PubMed</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>AbdulRaheem Y. Statistics in medical research: common mistakes. J Taibah Univ Med Sci. 2023;18(6):1197. 10.1016/j.jtumed.2023.04.004.
</cite> [<a href="https://doi.org/10.1016/j.jtumed.2023.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10205532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37234723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?AbdulRaheem%20Y.%20Statistics%20in%20medical%20research:%20common%20mistakes.%20J%20Taibah%20Univ%20Med%20Sci.%202023;18(6):1197.%2010.1016/j.jtumed.2023.04.004." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Hanna M. Statistics: common mistakes. How to Write Better Medical Papers. 2019:73–81. 10.1007/978-3-030-02955-5_14.</cite>
</li>
<li id="CR56">
<span class="label">56.</span><cite>Cowger CD. Correcting misuse is the best defense of statistical tests of significance. Soc Serv Rev. 1987;61(1):170–172. <a href="https://www.jstor.org/stable/30011875" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.jstor.org/stable/30011875</a>. Accessed 1st July 2025.</cite>
</li>
<li id="CR57">
<span class="label">57.</span><cite>Streiner DL. Best (but oft-forgotten) practices: the multiple problems of multiplicity–whether and how to correct for many statistical tests. Am J Clin Nutr. 2015;102(4):721–8. 10.3945/ajcn.115.113548.
</cite> [<a href="https://doi.org/10.3945/ajcn.115.113548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26245806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Streiner%20DL.%20Best%20(but%20oft-forgotten)%20practices:%20the%20multiple%20problems%20of%20multiplicity%E2%80%93whether%20and%20how%20to%20correct%20for%20many%20statistical%20tests.%20Am%20J%20Clin%20Nutr.%202015;102(4):721%E2%80%938.%2010.3945/ajcn.115.113548." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Schatz P, Jay KA, McComb J, McLaughlin J. Misuse of statistical tests in publications. Arch Clin Neuropsychol. 2005;20(8):1053–9. 10.1016/j.acn.2005.06.006.
</cite> [<a href="https://doi.org/10.1016/j.acn.2005.06.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16095871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schatz%20P,%20Jay%20KA,%20McComb%20J,%20McLaughlin%20J.%20Misuse%20of%20statistical%20tests%20in%20publications.%20Arch%20Clin%20Neuropsychol.%202005;20(8):1053%E2%80%939.%2010.1016/j.acn.2005.06.006." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Benjamin DJ, Berger JO, Johannesson M, Nosek BA, Wagenmakers EJ, Berk R, et al. Redefine statistical significance. Nat Hum Behav. 2018;2(1):6–10. 10.1038/s41562-017-0189-z.
</cite> [<a href="https://doi.org/10.1038/s41562-017-0189-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30980045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benjamin%20DJ,%20Berger%20JO,%20Johannesson%20M,%20Nosek%20BA,%20Wagenmakers%20EJ,%20Berk%20R,%20et%20al.%20Redefine%20statistical%20significance.%20Nat%20Hum%20Behav.%202018;2(1):6%E2%80%9310.%2010.1038/s41562-017-0189-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Khan Academy. Examples of null and alternative hypotheses. 2025. <a href="https://www.khanacademy.org/math/ap-statistics/xfb5d8e68:inference-categorical-proportions/idea-significance-tests/v/examples-of-null-and-alternative-hypotheses" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.khanacademy.org/math/ap-statistics/xfb5d8e68:inference-categorical-proportions/idea-significance-tests/v/examples-of-null-and-alternative-hypotheses</a>. URL visited on 2nd May. Accessed 1st July 2025.</cite>
</li>
<li id="CR61">
<span class="label">61.</span><cite>Levene H. Robust tests for equality of variances. Contributions to probability and statistics. Technical Report, Stanford University, 1960:278–92.</cite>
</li>
<li id="CR62">
<span class="label">62.</span><cite>Conover WJ. Practical Nonparametric Statistics, vol. 350. John Wiley &amp; Sons; 1999. <a href="https://www.wiley.com/en-us/Practical+Nonparametric+Statistics%2C+3rd+Edition-p-9780471160687" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.wiley.com/en-us/Practical+Nonparametric+Statistics%2C+3rd+Edition-p-9780471160687</a>. Accessed 1st July 2025</cite>
</li>
<li id="CR63">
<span class="label">63.</span><cite>Sidney S. Nonparametric statistics for the behavioral sciences. J Nerv Ment Dis. 1957;125(3):497. <a href="https://journals.lww.com/jonmd/citation/1957/07000/NONPARAMETRIC_STATISTICS_FOR_THE_BEHAVIORAL.32.aspx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.lww.com/jonmd/citation/1957/07000/NONPARAMETRIC_STATISTICS_FOR_THE_BEHAVIORAL.32.aspx</a>. Accessed 1st July 2025.</cite>
</li>
<li id="CR64">
<span class="label">64.</span><cite>Pearson KX. On the criterion that a given system of deviations from the probable in the case of a correlated system of variables is such that it can be reasonably supposed to have arisen from random sampling. Lond Edinb Dublin Philos Mag J Sci. 1900;50(302):157–75. 10.1080/14786440009463897.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pearson%20KX.%20On%20the%20criterion%20that%20a%20given%20system%20of%20deviations%20from%20the%20probable%20in%20the%20case%20of%20a%20correlated%20system%20of%20variables%20is%20such%20that%20it%20can%20be%20reasonably%20supposed%20to%20have%20arisen%20from%20random%20sampling.%20Lond%20Edinb%20Dublin%20Philos%20Mag%20J%20Sci.%201900;50(302):157%E2%80%9375.%2010.1080/14786440009463897." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Kruskal WH, and WAW. Use of Ranks in One-Criterion Variance Analysis. J Am Stat Assoc. 1952;47(260):583–621. 10.1080/01621459.1952.10483441.</cite>
</li>
<li id="CR66">
<span class="label">66.</span><cite>Team TPD. Pandas: Python Data Analysis Library. <a href="https://pandas.pydata.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pandas.pydata.org/</a>. URL visited on 19th February 2025. Accessed 1st July 2025.</cite>
</li>
<li id="CR67">
<span class="label">67.</span><cite>Community TS. SciPy: scientific computing library for Python. <a href="https://scipy.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scipy.org/</a>. URL visited on 19th February 2025. Accessed 1st July 2025.</cite>
</li>
<li id="CR68">
<span class="label">68.</span><cite>Team TMD. Matplotlib: visualization with Python. <a href="https://matplotlib.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://matplotlib.org/</a>. URL visited on 19th February 2025. Accessed 1st July 2025.</cite>
</li>
<li id="CR69">
<span class="label">69.</span><cite>Developers TN. NumPy: The Fundamental Package for Scientific Computing with Python. <a href="https://numpy.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://numpy.org/</a>. URL visited on 19th February 2025. Accessed 1st July 2025.</cite>
</li>
<li id="CR70">
<span class="label">70.</span><cite>Sandve GK, Nekrutenko A, Taylor J, Hovig E. Ten simple rules for reproducible computational research. PLoS Comput Biol. 2013;9(10):e1003285. 10.1371/journal.pcbi.1003285.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1003285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3812051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24204232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sandve%20GK,%20Nekrutenko%20A,%20Taylor%20J,%20Hovig%20E.%20Ten%20simple%20rules%20for%20reproducible%20computational%20research.%20PLoS%20Comput%20Biol.%202013;9(10):e1003285.%2010.1371/journal.pcbi.1003285." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Danchev V. Reproducible data science with Python: an open learning resource. J Open Source Educ. 2022. 10.21105/jose.00156.</cite>
</li>
<li id="CR72">
<span class="label">72.</span><cite>PYPL. PopularitY of Programming Language for May 2025. 2025. <a href="https://pypl.github.io/PYPL.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pypl.github.io/PYPL.html</a>. URL visited on 14th May. Accessed 1st July 2025.</cite>
</li>
<li id="CR73">
<span class="label">73.</span><cite>TIOBE. TIOBE Index May 2025. 2025. <a href="https://www.tiobe.com/tiobe-index/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.tiobe.com/tiobe-index/</a>. URL visited on 14th May. Accessed 1st July 2025.</cite>
</li>
<li id="CR74">
<span class="label">74.</span><cite>Kaggle. State of data science and machine learning 2022. 2025. <a href="https://www.kaggle.com/kaggle-survey-2022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.kaggle.com/kaggle-survey-2022</a>. URL visited on 14th May. Accessed 1st July 2025.</cite>
</li>
<li id="CR75">
<span class="label">75.</span><cite>Gibbons JD, Pratt JW. P-values: interpretation and methodology. Am Stat. 1975;29(1):20–5. 10.1080/00031305.1975.10479106.</cite> [<a href="https://scholar.google.com/scholar_lookup?Gibbons%20JD,%20Pratt%20JW.%20P-values:%20interpretation%20and%20methodology.%20Am%20Stat.%201975;29(1):20%E2%80%935.%2010.1080/00031305.1975.10479106." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Altman N, Krzywinski M. Interpreting P values. Nat Methods. 2017;14(3):213–4. 10.1038/nmeth.4210.</cite> [<a href="https://scholar.google.com/scholar_lookup?Altman%20N,%20Krzywinski%20M.%20Interpreting%20P%20values.%20Nat%20Methods.%202017;14(3):213%E2%80%934.%2010.1038/nmeth.4210." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Goodman S. A dirty dozen: twelve P-value misconceptions. Semin Hematol. 2008;45(3):135–40. 10.1053/j.seminhematol.2008.04.003.
</cite> [<a href="https://doi.org/10.1053/j.seminhematol.2008.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18582619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goodman%20S.%20A%20dirty%20dozen:%20twelve%20P-value%20misconceptions.%20Semin%20Hematol.%202008;45(3):135%E2%80%9340.%2010.1053/j.seminhematol.2008.04.003." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Andrade C. The P value and statistical significance: misunderstandings, explanations, challenges, and alternatives. Indian J Psychol Med. 2019;41(3):210–5. 10.4103/IJPSYM.IJPSYM_193_19.
</cite> [<a href="https://doi.org/10.4103/IJPSYM.IJPSYM_193_19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6532382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31142921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andrade%20C.%20The%20P%20value%20and%20statistical%20significance:%20misunderstandings,%20explanations,%20challenges,%20and%20alternatives.%20Indian%20J%20Psychol%20Med.%202019;41(3):210%E2%80%935.%2010.4103/IJPSYM.IJPSYM_193_19." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Di Leo G, Sardanelli F. Statistical significance: p value, 0.05 threshold, and applications to radiomics—reasons for a conservative approach. Eur Radiol Exp. 2020;4(1). 10.1186/s41747-020-0145-y.</cite> [<a href="https://doi.org/10.1186/s41747-020-0145-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7064671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32157489/" class="usa-link">PubMed</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Wasserstein RL, Schirm AL, Lazar NA. Moving to a World Beyond “p <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ecf3/12366075/a68969b7247a/13040_2025_465_Article_IEq278.gif" loading="lazy" alt="Inline graphic"> 0.05”. Am Stat. 2019;73(sup1):1–19. 10.1080/00031305.2019.1583913.</cite>
</li>
<li id="CR81">
<span class="label">81.</span><cite>Head ML, Holman L, Lanfear R, Kahn AT, Jennions MD. The extent and consequences of <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ecf3/12366075/78f687d765bd/13040_2025_465_Article_IEq134.gif" loading="lazy" alt="Inline graphic">-hacking in science. PLoS Biol. 2015;13(3):e1002106. 10.1371/journal.pbio.1002106.</cite> [<a href="https://doi.org/10.1371/journal.pbio.1002106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4359000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25768323/" class="usa-link">PubMed</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Gleason J. Comparative power of the ANOVA, randomization ANOVA, and Kruskal-Wallis Test [PhD Dissertation]. Wayne State University; 2013. <a href="https://digitalcommons.wayne.edu/oa_dissertations/658" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://digitalcommons.wayne.edu/oa_dissertations/658</a>. Accessed 1st July 2025.</cite>
</li>
<li id="CR83">
<span class="label">83.</span><cite>Shapiro SS, Wilk MB. An analysis of variance test for normality (complete samples). Biometrika. 1965;52(3–4):591–611. 10.1093/biomet/52.3-4.591.</cite> [<a href="https://scholar.google.com/scholar_lookup?Shapiro%20SS,%20Wilk%20MB.%20An%20analysis%20of%20variance%20test%20for%20normality%20(complete%20samples).%20Biometrika.%201965;52(3%E2%80%934):591%E2%80%93611.%2010.1093/biomet/52.3-4.591." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Berger VW, Zhou Y. Kolmogorov-Smirnov test: overview. Wiley StatsRef: Statistics Reference Online. 2014. 10.1002/9781118445112.stat06558.</cite> [<a href="https://scholar.google.com/scholar_lookup?Berger%20VW,%20Zhou%20Y.%20Kolmogorov-Smirnov%20test:%20overview.%20Wiley%20StatsRef:%20Statistics%20Reference%20Online.%202014.%2010.1002/9781118445112.stat06558." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Ghasemi A, Zahediasl S. Normality tests for statistical analysis: a guide for non-statisticians. Int J Endocrinol Metab. 2012;10(2):486. 10.5812/ijem.3505.
</cite> [<a href="https://doi.org/10.5812/ijem.3505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3693611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23843808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ghasemi%20A,%20Zahediasl%20S.%20Normality%20tests%20for%20statistical%20analysis:%20a%20guide%20for%20non-statisticians.%20Int%20J%20Endocrinol%20Metab.%202012;10(2):486.%2010.5812/ijem.3505." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Razali NM, Wah YB. Power comparisons of Shapiro-Wilk, Kolmogorov-Smirnov Lilliefors and Anderson-Darling tests. J Stat Model Anal. 2011;2(1):21–33. <a href="http://www.de.ufpb.br/%7eulisses/disciplinas/normality_tests_comparison.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.de.ufpb.br/~ulisses/disciplinas/normality_tests_comparison.pdf</a>. Accessed 1st July 2025.</cite>
</li>
<li id="CR87">
<span class="label">87.</span><cite>Kikvidze Z, Moya-Laraño J. Unexpected failures of recommended tests in basic statistical analyses of ecological data. Web Ecol. 2008;8(1):67–73. 10.5194/we-8-67-2008.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kikvidze%20Z,%20Moya-Lara%C3%B1o%20J.%20Unexpected%20failures%20of%20recommended%20tests%20in%20basic%20statistical%20analyses%20of%20ecological%20data.%20Web%20Ecol.%202008;8(1):67%E2%80%9373.%2010.5194/we-8-67-2008." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Tang AS, Woldemariam SR, Miramontes S, Norgeot B, Oskotsky TT, Sirota M. Harnessing EHR data for health research. Nat Med. 2024;30(7):1847–55. 10.1038/s41591-024-03074-8.
</cite> [<a href="https://doi.org/10.1038/s41591-024-03074-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38965433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20AS,%20Woldemariam%20SR,%20Miramontes%20S,%20Norgeot%20B,%20Oskotsky%20TT,%20Sirota%20M.%20Harnessing%20EHR%20data%20for%20health%20research.%20Nat%20Med.%202024;30(7):1847%E2%80%9355.%2010.1038/s41591-024-03074-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Ma Y, Zheng J, Feng J, Chen L, Dong K, Xiao X. Neuroblastomas in eastern China: a retrospective series study of 275 cases in a regional center. PeerJ. 2018;6:e5665. 10.7717/peerj.5665.
</cite> [<a href="https://doi.org/10.7717/peerj.5665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6148411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30245940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma%20Y,%20Zheng%20J,%20Feng%20J,%20Chen%20L,%20Dong%20K,%20Xiao%20X.%20Neuroblastomas%20in%20eastern%20China:%20a%20retrospective%20series%20study%20of%20275%20cases%20in%20a%20regional%20center.%20PeerJ.%202018;6:e5665.%2010.7717/peerj.5665." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Gucyetmez B, Atalan HK. C-reactive protein and hemogram parameters for the non-sepsis systemic inflammatory response syndrome and sepsis: what do they mean? PLoS ONE. 2016;11(2):e0148699. 10.1371/journal.pone.0148699.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0148699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4749228/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26863002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gucyetmez%20B,%20Atalan%20HK.%20C-reactive%20protein%20and%20hemogram%20parameters%20for%20the%20non-sepsis%20systemic%20inflammatory%20response%20syndrome%20and%20sepsis:%20what%20do%20they%20mean?%20PLoS%20ONE.%202016;11(2):e0148699.%2010.1371/journal.pone.0148699." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Mollura M, Chicco D, Paglialonga A, Barbieri R. Identifying prognostic factors for survival in intensive care unit patients with SIRS or sepsis by machine learning analysis on electronic health records. PLoS Digit Health. 2024;3(3):e0000459. 10.1371/journal.pdig.0000459.
</cite> [<a href="https://doi.org/10.1371/journal.pdig.0000459" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10942078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38489347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mollura%20M,%20Chicco%20D,%20Paglialonga%20A,%20Barbieri%20R.%20Identifying%20prognostic%20factors%20for%20survival%20in%20intensive%20care%20unit%20patients%20with%20SIRS%20or%20sepsis%20by%20machine%20learning%20analysis%20on%20electronic%20health%20records.%20PLoS%20Digit%20Health.%202024;3(3):e0000459.%2010.1371/journal.pdig.0000459." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Jani BD, Mair FS, Roger VL, Weston SA, Jiang R, Chamberlain AM. Comorbid depression and heart failure: a community cohort study. PLoS ONE. 2016;11(6):e0158570. 10.1371/journal.pone.0158570.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0158570" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4928788/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27362359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jani%20BD,%20Mair%20FS,%20Roger%20VL,%20Weston%20SA,%20Jiang%20R,%20Chamberlain%20AM.%20Comorbid%20depression%20and%20heart%20failure:%20a%20community%20cohort%20study.%20PLoS%20ONE.%202016;11(6):e0158570.%2010.1371/journal.pone.0158570." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Requena-Morales R, Palazón-Bru A, Rizo-Baeza MM, Adsuar-Quesada JM, Gil-Guillén VF, Cortés-Castell E. Mortality after out-of-hospital cardiac arrest in a Spanish region. PLoS ONE. 2017;12(4):e0175818. 10.1371/journal.pone.0175818.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0175818" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5391106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28407020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Requena-Morales%20R,%20Palaz%C3%B3n-Bru%20A,%20Rizo-Baeza%20MM,%20Adsuar-Quesada%20JM,%20Gil-Guill%C3%A9n%20VF,%20Cort%C3%A9s-Castell%20E.%20Mortality%20after%20out-of-hospital%20cardiac%20arrest%20in%20a%20Spanish%20region.%20PLoS%20ONE.%202017;12(4):e0175818.%2010.1371/journal.pone.0175818." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Takashi Y, Ishizu M, Mori H, Miyashita K, Sakamoto F, Katakami N, et al. Circulating osteocalcin as a bone-derived hormone is inversely correlated with body fat in patients with type 1 diabetes. PLoS ONE. 2019;14(5):e0216416. 10.1371/journal.pone.0216416.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0216416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6499427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31050684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takashi%20Y,%20Ishizu%20M,%20Mori%20H,%20Miyashita%20K,%20Sakamoto%20F,%20Katakami%20N,%20et%20al.%20Circulating%20osteocalcin%20as%20a%20bone-derived%20hormone%20is%20inversely%20correlated%20with%20body%20fat%20in%20patients%20with%20type%201%20diabetes.%20PLoS%20ONE.%202019;14(5):e0216416.%2010.1371/journal.pone.0216416." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Cerono G, Chicco D. Ensemble machine learning reveals key features for diabetes duration from electronic health records. PeerJ Comput Sci. 2024;10:e1896. 10.7717/peerj-cs.1896.
</cite> [<a href="https://doi.org/10.7717/peerj-cs.1896" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10909161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38435625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cerono%20G,%20Chicco%20D.%20Ensemble%20machine%20learning%20reveals%20key%20features%20for%20diabetes%20duration%20from%20electronic%20health%20records.%20PeerJ%20Comput%20Sci.%202024;10:e1896.%2010.7717/peerj-cs.1896." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Bartolucci D, Montemurro L, Raieli S, Lampis S, Pession A, Hrelia P, et al. MYCN impact on high-risk neuroblastoma: from diagnosis and prognosis to targeted treatment. Cancers. 2022;14(18):4421. 10.3390/cancers14184421.
</cite> [<a href="https://doi.org/10.3390/cancers14184421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9496712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36139583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bartolucci%20D,%20Montemurro%20L,%20Raieli%20S,%20Lampis%20S,%20Pession%20A,%20Hrelia%20P,%20et%20al.%20MYCN%20impact%20on%20high-risk%20neuroblastoma:%20from%20diagnosis%20and%20prognosis%20to%20targeted%20treatment.%20Cancers.%202022;14(18):4421.%2010.3390/cancers14184421." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Nakazawa A, Haga C, Ohira M, Okita H, Kamijo T, Nakagawara A. Correlation between the International Neuroblastoma Pathology Classification and genomic signature in neuroblastoma. Cancer Sci. 2015;106(6):766–71. 10.1111/cas.12665.
</cite> [<a href="https://doi.org/10.1111/cas.12665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4471784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25827934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakazawa%20A,%20Haga%20C,%20Ohira%20M,%20Okita%20H,%20Kamijo%20T,%20Nakagawara%20A.%20Correlation%20between%20the%20International%20Neuroblastoma%20Pathology%20Classification%20and%20genomic%20signature%20in%20neuroblastoma.%20Cancer%20Sci.%202015;106(6):766%E2%80%9371.%2010.1111/cas.12665." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Bauer M, Gerlach H, Vogelmann T, Preissing F, Stiefel J, Adam D. Mortality in sepsis and septic shock in Europe, North America and Australia between 2009 and 2019-Results from a systematic review and meta-analysis. Crit Care. 2020;24(1):239. 10.1186/s13054-020-02950-2.
</cite> [<a href="https://doi.org/10.1186/s13054-020-02950-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7236499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32430052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bauer%20M,%20Gerlach%20H,%20Vogelmann%20T,%20Preissing%20F,%20Stiefel%20J,%20Adam%20D.%20Mortality%20in%20sepsis%20and%20septic%20shock%20in%20Europe,%20North%20America%20and%20Australia%20between%202009%20and%202019-Results%20from%20a%20systematic%20review%20and%20meta-analysis.%20Crit%20Care.%202020;24(1):239.%2010.1186/s13054-020-02950-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Beach SR, Januzzi JL, Mastromauro CA, Healy BC, Beale EE, Celano CM, et al. Patient Health Questionnaire-9 score and adverse cardiac outcomes in patients hospitalized for acute cardiac disease. J Psychosom Res. 2013;75(5):409–13. 10.1016/j.jpsychores.2013.08.001.
</cite> [<a href="https://doi.org/10.1016/j.jpsychores.2013.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24182627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beach%20SR,%20Januzzi%20JL,%20Mastromauro%20CA,%20Healy%20BC,%20Beale%20EE,%20Celano%20CM,%20et%20al.%20Patient%20Health%20Questionnaire-9%20score%20and%20adverse%20cardiac%20outcomes%20in%20patients%20hospitalized%20for%20acute%20cardiac%20disease.%20J%20Psychosom%20Res.%202013;75(5):409%E2%80%9313.%2010.1016/j.jpsychores.2013.08.001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Ishii J, Nishikimi M, Kikutani K, Ohki S, Ota K, Anzai T, et al. Resuscitation attempt and outcomes in patients with asystole out-of-hospital cardiac arrest. JAMA Netw Open. 2024;7(11):e2445543. 10.1001/jamanetworkopen.2024.45543.
</cite> [<a href="https://doi.org/10.1001/jamanetworkopen.2024.45543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11574695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39556393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ishii%20J,%20Nishikimi%20M,%20Kikutani%20K,%20Ohki%20S,%20Ota%20K,%20Anzai%20T,%20et%20al.%20Resuscitation%20attempt%20and%20outcomes%20in%20patients%20with%20asystole%20out-of-hospital%20cardiac%20arrest.%20JAMA%20Netw%20Open.%202024;7(11):e2445543.%2010.1001/jamanetworkopen.2024.45543." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Bower JK, Meadows RJ, Foster MC, Foraker RE, Shoben AB. The association of percent body fat and lean mass with HbA1c in US adults. J Endocr Soc. 2017;1(6):600–8. 10.1210/js.2017-00046.
</cite> [<a href="https://doi.org/10.1210/js.2017-00046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5686694/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29264513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bower%20JK,%20Meadows%20RJ,%20Foster%20MC,%20Foraker%20RE,%20Shoben%20AB.%20The%20association%20of%20percent%20body%20fat%20and%20lean%20mass%20with%20HbA1c%20in%20US%20adults.%20J%20Endocr%20Soc.%202017;1(6):600%E2%80%938.%2010.1210/js.2017-00046." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Hotelling H. The generalization of Student’s ratio. Ann Math Stat. 1931;2(3):360–78. 10.1214/aoms/1177732979.</cite> [<a href="https://scholar.google.com/scholar_lookup?Hotelling%20H.%20The%20generalization%20of%20Student%E2%80%99s%20ratio.%20Ann%20Math%20Stat.%201931;2(3):360%E2%80%9378.%2010.1214/aoms/1177732979." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data availability The EHRs datasets employed in this study are publicly available in the supplementary ma- terials of the original dataset publications [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>, <a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>, <a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>–<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>] under the CC BY 4.0 license at the following URLs:</p>
<p> – Sepsis and SIRS [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>] <a href="https://figshare.com/articles/dataset/_C_Reactive_Protein_and_Hemogram_Parameters_for_the_Non_Sepsis_Systemic_Inflammatory_Response_Syndrome_and_Sepsis_What_Do_They_Mean_/1644426?file=2637248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/_C_Reactive_Protein_and_Hemogram_Parameters_for_the_Non_Sepsis_Systemic_Inflammatory_Response_Syndrome_and_Sepsis_What_Do_They_Mean_/1644426?file=2637248</a></p>
<p> – Depression and Heart Failure [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>] <a href="https://figshare.com/articles/dataset/Comorbid_Depression_and_Heart_Failure_A_Community_Cohort_Study/3916224?file=6130425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Comorbid_Depression_and_Heart_Failure_A_Community_Cohort_Study/3916224?file=6130425</a></p>
<p> – Cardiac Arrest [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>] <a href="https://figshare.com/articles/dataset/Mortality_after_out-of-hospital_cardiac_arrest_in_a_Spanish_Region/4876247?file=8166893" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Mortality_after_out-of-hospital_cardiac_arrest_in_a_Spanish_Region/4876247?file=8166893</a></p>
<p> – Neuroblastoma [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>] <a href="https://doi.org/10.7717/peerj.5665/supp-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.7717/peerj.5665/supp-5</a></p>
<p> – Diabetes Type One [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>] <a href="https://figshare.com/articles/dataset/Circulating_osteocalcin_as_a_bone-derived_hormone_is_inversely_correlated_with_body_fat_in_patients_with_type_1_diabetes/8079389?file=15057092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/articles/dataset/Circulating_osteocalcin_as_a_bone-derived_hormone_is_inversely_correlated_with_body_fat_in_patients_with_type_1_diabetes/8079389?file=15057092</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BioData Mining are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s13040-025-00465-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/13040_2025_Article_465.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366075/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366075/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366075%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366075/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366075/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366075/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835959/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366075/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835959/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366075/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366075/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ToGJgeXAmkqfOvlPE311VirN0WwT9DudU7hrUtM0OF0lLRiFbpnX20tPHmxbhOcS">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
