
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Taxonomic quasi‐primes: peptides charting lineage‐specific adaptations and disease‐relevant loci - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53129E8AEF2D2305129E000D3AA0A6.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="prosci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375989/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Protein Science : A Publication of the Protein Society">
<meta name="citation_title" content="Taxonomic quasi‐primes: peptides charting lineage‐specific adaptations and disease‐relevant loci">
<meta name="citation_author" content="Eleftherios Bochalis">
<meta name="citation_author_institution" content="Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, Pennsylvania, USA">
<meta name="citation_author_institution" content="Department of Internal Medicine, Division of Endocrinology, Medical School, University of Patras, Patras, Greece">
<meta name="citation_author" content="Michail Patsakis">
<meta name="citation_author_institution" content="Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, Pennsylvania, USA">
<meta name="citation_author_institution" content="Huck Institute of the Life Sciences, Pennsylvania State University, University Park, Pennsylvania, USA">
<meta name="citation_author" content="Nikol Chantzi">
<meta name="citation_author_institution" content="Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, Pennsylvania, USA">
<meta name="citation_author_institution" content="Huck Institute of the Life Sciences, Pennsylvania State University, University Park, Pennsylvania, USA">
<meta name="citation_author" content="Ioannis Mouratidis">
<meta name="citation_author_institution" content="Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, Pennsylvania, USA">
<meta name="citation_author_institution" content="Huck Institute of the Life Sciences, Pennsylvania State University, University Park, Pennsylvania, USA">
<meta name="citation_author" content="Dionysios V Chartoumpekis">
<meta name="citation_author_institution" content="Department of Internal Medicine, Division of Endocrinology, Medical School, University of Patras, Patras, Greece">
<meta name="citation_author" content="Ilias Georgakopoulos‐Soares">
<meta name="citation_author_institution" content="Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, Pennsylvania, USA">
<meta name="citation_author_institution" content="Huck Institute of the Life Sciences, Pennsylvania State University, University Park, Pennsylvania, USA">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="34">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70241">
<meta name="citation_doi" content="10.1002/pro.70241">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375989/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375989/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375989/pdf/PRO-34-e70241.pdf">
<meta name="description" content="The identification of succinct, universal fingerprints that enable the characterization of individual taxonomies can reveal insights into trait development. Here, we introduce taxonomic quasi‐primes, peptide k‐mer sequences that are exclusively ...">
<meta name="og:title" content="Taxonomic quasi‐primes: peptides charting lineage‐specific adaptations and disease‐relevant loci">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The identification of succinct, universal fingerprints that enable the characterization of individual taxonomies can reveal insights into trait development. Here, we introduce taxonomic quasi‐primes, peptide k‐mer sequences that are exclusively ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375989/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375989">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/pro.70241"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/PRO-34-e70241.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375989%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375989/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375989/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375989/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-prosci.gif" alt="Protein Science : A Publication of the Protein Society logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Protein Science : A Publication of the Protein Society" title="Link to Protein Science : A Publication of the Protein Society" shape="default" href="http://www.proteinscience.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Protein Sci</button></div>. 2025 Aug 25;34(9):e70241. doi: <a href="https://doi.org/10.1002/pro.70241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/pro.70241</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protein%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Protein%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Taxonomic quasi‐primes: peptides charting lineage‐specific adaptations and disease‐relevant loci</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bochalis%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Eleftherios Bochalis</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Eleftherios Bochalis</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, Pennsylvania, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Internal Medicine, Division of Endocrinology, Medical School, University of Patras, Patras, Greece</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bochalis%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eleftherios Bochalis</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patsakis%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Michail Patsakis</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Michail Patsakis</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, Pennsylvania, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Huck Institute of the Life Sciences, Pennsylvania State University, University Park, Pennsylvania, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Patsakis%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michail Patsakis</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chantzi%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Nikol Chantzi</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Nikol Chantzi</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, Pennsylvania, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Huck Institute of the Life Sciences, Pennsylvania State University, University Park, Pennsylvania, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chantzi%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nikol Chantzi</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mouratidis%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ioannis Mouratidis</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ioannis Mouratidis</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, Pennsylvania, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Huck Institute of the Life Sciences, Pennsylvania State University, University Park, Pennsylvania, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mouratidis%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ioannis Mouratidis</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chartoumpekis%20DV%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Dionysios V Chartoumpekis</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Dionysios V Chartoumpekis</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Internal Medicine, Division of Endocrinology, Medical School, University of Patras, Patras, Greece</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chartoumpekis%20DV%22%5BAuthor%5D" class="usa-link"><span class="name western">Dionysios V Chartoumpekis</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Georgakopoulos%E2%80%90Soares%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Ilias Georgakopoulos‐Soares</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Ilias Georgakopoulos‐Soares</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, Pennsylvania, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Huck Institute of the Life Sciences, Pennsylvania State University, University Park, Pennsylvania, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Georgakopoulos%E2%80%90Soares%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ilias Georgakopoulos‐Soares</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="pro70241-aff-0001">
<sup>
<sup>1</sup>
</sup>Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, Pennsylvania, USA</div>
<div id="pro70241-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Internal Medicine, Division of Endocrinology, Medical School, University of Patras, Patras, Greece</div>
<div id="pro70241-aff-0003">
<sup>
<sup>3</sup>
</sup>Huck Institute of the Life Sciences, Pennsylvania State University, University Park, Pennsylvania, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Ilias Georgakopoulos‐Soares, Institute for Personalized Medicine, Department of Biochemistry and Molecular Biology, The Pennsylvania State University College of Medicine, Hershey, PA, USA; Huck Institute of the Life Sciences, Pennsylvania State University, University Park, PA, USA. 
Email: <span>izg5139@psu.edu</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 17; Received 2025 Mar 24; Accepted 2025 Jul 11; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Protein Science</em> published by Wiley Periodicals LLC on behalf of The Protein Society.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375989</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The identification of succinct, universal fingerprints that enable the characterization of individual taxonomies can reveal insights into trait development. Here, we introduce taxonomic quasi‐primes, peptide k‐mer sequences that are exclusively present in a specific taxonomy and absent from all others. By analyzing 24,073 reference proteomes, we identified these unique peptides at the superkingdom, kingdom, and phylum ranks. These sequences exhibit remarkable uniqueness at six‐ and seven‐amino‐acid lengths. For instance, the seven‐mer SAPNYCY is found in 98.11% of eukaryotic species, while being completely absent from archaeal, bacterial, and viral reference proteomes. Functional analysis demonstrated that taxonomic quasi‐prime containing proteins are enriched for processes defining a lineage, such as synaptic signaling in Chordata. Structural analysis revealed that these peptides are preferentially located within proteins, participating directly in enzymatic active sites, mediating protein–protein interactions, and stabilizing ligand binding. Moreover, we show that in human proteins, highly conserved Chordata quasi‐prime loci are 2.08‐fold more likely to harbor pathogenic variants than surrounding regions, directly linking these evolutionary signatures to disease. This study establishes taxonomic quasi‐primes as markers that illuminate evolutionary pathways and provide a powerful method for identifying functionally indispensable and disease‐relevant loci, which warrant further therapeutic and diagnostic investigation.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> evolution, K‐mers, reference proteomes, taxonomic quasi‐primes, taxonomies, tree of life</p></section></section><section id="pro70241-sec-0001"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>The number of available reference proteomes has rapidly increased in recent years, a trend that is expected to continue in the foreseeable future (UniProt Consortium, <a href="#pro70241-bib-0051" class="usa-link" aria-describedby="pro70241-bib-0051">2023</a>). The availability of a large and diverse set of proteomes provides an opportunity to increase our understanding of protein sequence and functional diversity in nature across taxonomic ranks. Such research could reveal insights into trait development through findings pertaining to mechanisms of sequence conservation and divergence, the emergence of proteins with novel functional roles, and can have applications in biomarker discovery and pathogen surveillance (Al‐Amrani et al., <a href="#pro70241-bib-0002" class="usa-link" aria-describedby="pro70241-bib-0002">2021</a>; Lacerda &amp; Reardon, <a href="#pro70241-bib-0032" class="usa-link" aria-describedby="pro70241-bib-0032"><sup>2009</sup></a>). These advances can be facilitated if the availability of ever‐expanding proteomic information is coupled with novel and insightful algorithms to process this abundance of biological information.</p>
<p>Peptide k‐mers are defined as oligopeptide sequences of length k and are often used in proteomics analyses (Moeckel et al., <a href="#pro70241-bib-0039" class="usa-link" aria-describedby="pro70241-bib-0039">2024</a>). The number of possible peptide k‐mers exponentiates with k‐mer length, leading to oligopeptide sequence uniqueness even at low k‐mer lengths (Georgakopoulos‐Soares et al., <a href="#pro70241-bib-0020" class="usa-link" aria-describedby="pro70241-bib-0020">2021</a>; Mouratidis et al., <a href="#pro70241-bib-0040" class="usa-link" aria-describedby="pro70241-bib-0040"><sup>2024</sup></a>). Because of their ease of identification, peptide k‐mers have numerous applications, including mass‐spectrometry‐based proteomics (Chapman, <a href="#pro70241-bib-0010" class="usa-link" aria-describedby="pro70241-bib-0010">2013</a>), motif search and evolutionary studies (Wen et al., <a href="#pro70241-bib-0056" class="usa-link" aria-describedby="pro70241-bib-0056">2014</a>), taxonomic classification, antimicrobial resistance and pathogen detection (ValizadehAslani et al., <a href="#pro70241-bib-0052" class="usa-link" aria-describedby="pro70241-bib-0052">2020</a>), and the identification of therapeutic targets (Hajisharifi et al., <a href="#pro70241-bib-0021" class="usa-link" aria-describedby="pro70241-bib-0021">2014</a>; Wu et al., <a href="#pro70241-bib-0058" class="usa-link" aria-describedby="pro70241-bib-0058"><sup>2019</sup></a>) among others.</p>
<p>We recently described the concept of quasi‐prime peptides, which are the shortest peptides that are unique to a species's proteome and are absent from every other known proteome (Mouratidis et al., <a href="#pro70241-bib-0041" class="usa-link" aria-describedby="pro70241-bib-0041">2023</a>). Here, we have extended the concept of quasi‐prime peptides to incorporate taxonomic ranks, identifying k‐mer peptides that are present in one or multiple species of a taxonomic rank, but absent from every known proteome outside that rank. Using 24,073 reference proteomes, we provide proof of the existence of k‐mer peptides with this property at the superkingdom, kingdom, and phylum ranks (Figure <a href="#pro70241-fig-0001" class="usa-link">1</a>). We demonstrate the role of taxonomic quasi‐prime peptides in evolutionary divergence and taxonomic adaptations and provide evidence that Chordata quasi‐prime loci are more likely to harbor pathogenic variants in human proteins.</p>
<figure class="fig xbox font-sm" id="pro70241-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0080/12375989/7b69bfe469b6/PRO-34-e70241-g008.jpg" loading="lazy" id="jats-graphic-1" height="654" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/pro70241-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of taxonomic quasi‐prime peptide identification and analysis pipeline. Identification of taxonomic quasi‐prime peptides was performed for 24,073 reference proteomes at the superkingdom, kingdom, and phylum ranks. To understand the roles of taxonomic quasi‐prime peptides, a thorough characterization was performed, including Gene Ontology enrichment analyses, protein domain and family analyses, and structural examinations.</p></figcaption></figure></section><section id="pro70241-sec-0002"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="pro70241-sec-0003"><h3 class="pmc_sec_title">2.1. Proteomic datasets</h3>
<p>Reference proteomes were obtained from UniProt (Release 2024_01), comprising a total of 24,073 species, including 364 Archaeal, 9438 Bacterial, 2489 Eukaryotic, and 11,782 viral proteomes.</p>
<p>Peptide k‐mer extraction was performed as previously described in Mouratidis et al. (<a href="#pro70241-bib-0041" class="usa-link" aria-describedby="pro70241-bib-0041">2023</a>), for k‐mer lengths of five to seven amino acids. We defined <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline" overflow="linebreak"><mrow><mrow><mi>T</mi></mrow></mrow></math></span> as the superset of all considered taxonomies, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline" overflow="linebreak"><mrow><mrow><mi>K</mi></mrow></mrow></math></span> as a given k‐mer and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline" overflow="linebreak"><mrow><mrow><mi>P</mi></mrow></mrow></math></span> as a proteome. K‐mer lengths below five amino acids were not considered, as tetrapeptides are highly prevalent and in the human proteome all possible tetrapeptides are observed (Chantzi et al., <a href="#pro70241-bib-0009" class="usa-link" aria-describedby="pro70241-bib-0009">2024</a>; Georgakopoulos‐Soares et al., <a href="#pro70241-bib-0020" class="usa-link" aria-describedby="pro70241-bib-0020"><sup>2021</sup></a>). Octapeptides and longer oligopeptides were not considered as the possible proteome space becomes extremely large (Wen et al., <a href="#pro70241-bib-0056" class="usa-link" aria-describedby="pro70241-bib-0056">2014</a>), limiting the set of k‐mers that are shared between multiple species in a taxonomy.</p>
<p>
<em>Definition of k‐mer</em>: We say that a k‐mer <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline" overflow="linebreak"><mrow><mrow><mi>K</mi></mrow></mrow></math></span> belongs to a taxonomic rank <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>T</mi><mi>i</mi></msub></mrow></mrow></math></span> if and only if there exists at least one species <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline" overflow="linebreak"><mrow><mrow><mi>S</mi></mrow></mrow></math></span> in <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>T</mi><mi>i</mi></msub></mrow></mrow></math></span> such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline" overflow="linebreak"><mrow><mrow><mi>K</mi></mrow></mrow></math></span> is found in <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline" overflow="linebreak"><mrow><mrow><mi>S</mi></mrow></mrow></math></span>, that is:</p>
<table class="disp-formula p" id="pro70241-disp-0001"><tr>
<td class="formula"><math id="jats-math-10" display="block" overflow="linebreak"><mrow><mrow><mi>K</mi><mo>∈</mo><msub><mi>T</mi><mi>i</mi></msub><mspace width="0.52em"></mspace><mtext>if and only if</mtext><mspace width="0.52em"></mspace><mo>∃</mo><mi>S</mi><mo>∈</mo><msub><mi>T</mi><mi>i</mi></msub><mo>:</mo><mi>K</mi><mo>∈</mo><mi>S</mi></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>
<em>Definition of taxonomy</em>: For species classification, a taxonomy organizes species into hierarchical categories such as Superkingdoms, Kingdoms, and Phyla.</p>
<p>
<em>Definition of ε‐score</em>: The ε‐score for a k‐mer <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline" overflow="linebreak"><mrow><mrow><mi>K</mi></mrow></mrow></math></span> in a taxonomic rank <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>T</mi><mi>i</mi></msub></mrow></mrow></math></span> is a measure of the frequency with which <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline" overflow="linebreak"><mrow><mrow><mi>K</mi></mrow></mrow></math></span> appears in the proteomes of taxonomic rank <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>T</mi><mi>i</mi></msub></mrow></mrow></math></span>.</p>
<table class="disp-formula p" id="pro70241-disp-0002"><tr>
<td class="formula"><math id="jats-math-15" display="block" overflow="linebreak"><mrow><mrow><mi>ε</mi><mfenced open="(" close=")" separators=","><mi>K</mi><msub><mi>T</mi><mi>i</mi></msub></mfenced><mo linebreak="goodbreak">=</mo><mfrac><mfenced open="|" close="|"><mfenced open="{" close="}"><mrow><mi>P</mi><mo>∈</mo><msub><mi>T</mi><mi>i</mi></msub><mo>:</mo><mi>K</mi><mo>∈</mo><mi>P</mi></mrow></mfenced></mfenced><mfenced open="|" close="|"><msub><mi>T</mi><mi>i</mi></msub></mfenced></mfrac><mo linebreak="goodbreak">×</mo><mn>100</mn><mo>,</mo><mi>ε</mi><mfenced open="(" close=")" separators=","><mi>K</mi><msub><mi>T</mi><mi>i</mi></msub></mfenced><mo linebreak="goodbreak">=</mo><mn>0</mn><mo>,</mo><mi>…</mi><mo>,</mo><mn>100</mn></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>An ε‐score equal to zero indicates complete absence of the k‐mer <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline" overflow="linebreak"><mrow><mrow><mi>K</mi></mrow></mrow></math></span> across all species of the taxonomic rank <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>T</mi><mi>i</mi></msub></mrow></mrow></math></span>, while an ε‐score of one hundred indicates universal presence of k‐mer <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-18" display="inline" overflow="linebreak"><mrow><mrow><mi>K</mi></mrow></mrow></math></span> across all member species of the taxonomic rank <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>T</mi><mi>i</mi></msub></mrow></mrow></math></span>.</p>
<p>
<em>Definition of taxonomic quasi‐prime peptides</em>: Taxonomic quasi‐prime peptides were defined as peptide sequences present in species of a taxonomic rank <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>T</mi><mi>i</mi></msub></mrow></mrow></math></span> and absent from all species outside that taxonomic rank.</p>
<table class="disp-formula p" id="pro70241-disp-0003"><tr>
<td class="formula"><math id="jats-math-21" display="block" overflow="linebreak"><mrow><mrow><mi>Q</mi><mfenced open="(" close=")"><msub><mi>T</mi><mi>i</mi></msub></mfenced><mo linebreak="goodbreak">=</mo><mfenced open="{" close="}" separators="|"><mi>K</mi><mrow><mi>K</mi><mo>∈</mo><msub><mi>T</mi><mi>i</mi></msub><mo>∧</mo><mo>∀</mo><mi>j</mi><mo>≠</mo><mi>i</mi><mo>:</mo><mi>K</mi><mo>∉</mo><msub><mi>T</mi><mi>j</mi></msub></mrow></mfenced></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table></section><section id="pro70241-sec-0004"><h3 class="pmc_sec_title">2.2. Taxonomic quasi‐prime peptide protein matching</h3>
<p>The Peptide Match command line tool (Chen et al., <a href="#pro70241-bib-0011" class="usa-link" aria-describedby="pro70241-bib-0011">2013</a>) was used to map taxonomic quasi‐prime peptides to the protein sequences containing them. The Lucene index needed for Peptide Match was created using UniProt reference proteome sequences containing SwissProt and TrEMBL entries. After the mapping process, a file containing the taxonomic quasi‐prime peptides of interest, the corresponding UniProt entry identifier, and the sequence coordinates containing the peptide was obtained.</p></section><section id="pro70241-sec-0005"><h3 class="pmc_sec_title">2.3. Species clustering based on phylum ε‐scores</h3>
<p>Uniform Manifold Approximation and Projection (UMAP; McInnes et al., <a href="#pro70241-bib-0036" class="usa-link" aria-describedby="pro70241-bib-0036"><sup>2018</sup></a>) was employed using the Python library umap‐learn (version 0.5.7) to analyze phylum taxonomic quasi‐prime seven‐mers corresponding to the top 50th percentile of ε‐scores within each phylum. This approach aimed to visualize how species clustered in two‐dimensional space based on their taxonomic quasi‐prime composition. A semi‐supervised, density‐based UMAP was implemented, incorporating a target weight of 0.25 for clustering based on labels. The algorithm parameters were configured with 30 neighbors and a minimum distance of 0.1 to optimize cluster resolution.</p></section><section id="pro70241-sec-0006"><h3 class="pmc_sec_title">2.4. Multi‐species gene ontology (GO) enrichment analysis</h3>
<p>Multi‐species GO enrichment analysis (GOEA) was performed on taxonomic quasi‐prime peptide‐containing proteins using GOATools (version 1.4.12) (Klopfenstein et al., <a href="#pro70241-bib-0029" class="usa-link" aria-describedby="pro70241-bib-0029">2018</a>) at the phylum level. A GOEA (Ashburner et al., <a href="#pro70241-bib-0003" class="usa-link" aria-describedby="pro70241-bib-0003">2000</a>) was performed for each species, and results were combined using appropriate statistical methods (see “Enrichment score combination across same phylum species” section). We selected taxonomic quasi‐prime peptides with an ε‐score above 90.00% for each phylum in Archaea, Bacteria, and Eukaryotes. The study population for each analysis consisted of the taxonomic quasi‐prime peptide‐containing proteins, whereas the background population was represented by all the proteins expressed from the corresponding species. The Open Biological and Biomedical Ontologies (OBO) 1.4 file (.obo), containing ontology information and needed for the GOATools package, was obtained from the Gene Ontology Resource (Gene Ontology Consortium et al., <a href="#pro70241-bib-0019" class="usa-link" aria-describedby="pro70241-bib-0019">2023</a>) (<a href="https://geneontology.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://geneontology.org/</a>, release 2024‐09‐08) and included a total of 40,939 GO terms and 7,894,411 annotations for 5,426 species. Finally, the GO annotation file representing the relationship between UniProt entry and GO terms was downloaded from the Gene Ontology Annotation Database (version 222, released on August 05, 2024, <a href="https://ftp.ebi.ac.uk/pub/databases/GO/goa/UNIPROT/goa_uniprot_all.gaf.gz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.ebi.ac.uk/pub/databases/GO/goa/UNIPROT/goa_uniprot_all.gaf.gz</a>). Species were filtered to keep only those that possessed a protein count greater than 10 to ensure statistical robustness. Study population protein counts were not propagated up the GO graph during GOEA. <em>P</em>‐values were adjusted using the Benjamini–Hochberg method, and only GO terms with an adjusted <em>p</em>‐value less than 0.05 were selected, comprising the statistically significant GO terms.</p></section><section id="pro70241-sec-0007"><h3 class="pmc_sec_title">2.5. GO result clustering</h3>
<p>Statistically significant GO terms were clustered into broader representative terms to minimize redundancy and noise and highlight functional themes present across phyla. Each GO term class (biological process (BP), molecular function (MF), and cellular component (CC)) within each phylum was handled separately and Wang's semantic similarity measure (Wang et al., <a href="#pro70241-bib-0054" class="usa-link" aria-describedby="pro70241-bib-0054">2007</a>) was used to calculate the pairwise similarity of terms. These similarity values were converted to a distance matrix for agglomerative hierarchical clustering with the average linkage method. The optimal clustering threshold specifically for each GO term class was set to 0.54 for BP, 0.535 for MF and 0.52 for CC (Bettembourg et al., <a href="#pro70241-bib-0006" class="usa-link" aria-describedby="pro70241-bib-0006">2015</a>). Post‐clustering a representative term for each cluster was selected based on the adjusted p‐value and ties between terms were resolved using the presence percentage across species belonging to an individual phylum and the combined LOR, where terms with the highest value were characterized as representative. For the representative terms a weighted average LOR was calculated using the standard error as weight followed by winsorization at the 95th percentile to minimize the effect of extreme values. GO terms that were not assigned to any cluster were also retained, if the term's adjusted <em>p</em>‐value was less than 0.05.</p></section><section id="pro70241-sec-0008"><h3 class="pmc_sec_title">2.6. Multi‐species protein entry enrichment analysis</h3>
<p>Taxonomic quasi‐prime peptide‐containing proteins were subjected to multi‐species protein entry enrichment analysis. During the analysis, the presence of taxonomic quasi‐primes within functional protein domains and the protein family composition proteins containing them was analyzed. Protein entry data were obtained from the InterPro database (version 101.0 updated on July 25, 2024) (Paysan‐Lafosse et al., <a href="#pro70241-bib-0045" class="usa-link" aria-describedby="pro70241-bib-0045">2023</a>), which contains 14,950 domain and 26,089 family entries. For the enrichment analysis, Fisher's exact test was implemented, from which an odds ratio (OR) and a <em>p</em>‐value was calculated for each entry. Haldane–Anscombe correction (Agresti, <a href="#pro70241-bib-0001" class="usa-link" aria-describedby="pro70241-bib-0001">1999</a>) was applied to all cells of the 2 × 2 contingency table. Only entries with a <em>p</em>‐value less than 0.05 and an OR greater than 1.0, to analyze only enriched entries downstream, and with presence across more than 1 species were subjected to effect combination, whereas entries present only in one species were retained, if they passed the <em>p</em>‐value and OR cut‐off.</p></section><section id="pro70241-sec-0009"><h3 class="pmc_sec_title">2.7. Enrichment score combination across same phylum species</h3>
<p>Single species GOEA and protein entry enrichment analysis results were combined into a phylum representative unified score (one for each analysis). This was achieved using a meta‐analytic random effects model. The model involved calculating the natural logarithm of OR (LOR) and the corresponding initial fixed‐effect weights. Then using Cochran's Q and the subsequent Tau‐squared we calculated the final adjusted weights that also represent the between‐study variance. These weights were used to determine the phylum‐wide enrichment value for each item. The statistical significance of each enrichment score was assessed using a modified <em>z</em>‐score and <em>p</em>‐values derived from a <em>t</em>‐distribution. Multiple testing correction was implemented through the Benjamini–Hochberg procedure, and items with an adjusted <em>p</em>‐value greater than 0.05 were filtered out. More details about the mathematical procedures behind the random‐effects model can be found in the Supplementary Material.</p></section><section id="pro70241-sec-0010"><h3 class="pmc_sec_title">2.8. Secondary and tertiary structure analysis</h3>
<p>The secondary structures of taxonomic quasi‐prime peptides from key model and global health risk organisms were analyzed. Protein PDB files were obtained from the AlphaFold Protein Structure Database (Jumper et al., <a href="#pro70241-bib-0025" class="usa-link" aria-describedby="pro70241-bib-0025">2021</a>; Varadi et al., <a href="#pro70241-bib-0053" class="usa-link" aria-describedby="pro70241-bib-0053"><sup>2024</sup></a>) updated as of September 2024. For each protein, taxonomic quasi‐prime peptide regions were extracted and their secondary structure was identified using the STRIDE (STRuctural IDEntification) algorithm (Frishman &amp; Argos, <a href="#pro70241-bib-0018" class="usa-link" aria-describedby="pro70241-bib-0018">1995</a>) as provided by the ssbio (version 0.9.9) tool package. Peptides that presented with no hydrogen‐bonds were characterized to have a Disordered conformation. For the structural analysis, PDB files presenting the interaction between selected proteins and ligands were downloaded from the RCSB Protein Data Bank (Berman et al., <a href="#pro70241-bib-0005" class="usa-link" aria-describedby="pro70241-bib-0005">2000</a>) (updated as of 2024, download timestamp: November 21, 2024). Only conformations obtained through Cryo‐EM or x‐ray were selected. Protein visualization and hydrogen‐bond detection was performed using UCSF ChimeraX (version 1.8) software (Meng et al., <a href="#pro70241-bib-0038" class="usa-link" aria-describedby="pro70241-bib-0038">2023</a>).</p></section><section id="pro70241-sec-0011"><h3 class="pmc_sec_title">2.9. Assessing the biochemical robustness of taxonomic quasi‐prime peptides</h3>
<p>To address if the uniqueness of the taxonomic quasi‐prime peptides extended into biochemical properties other than k‐mer sequence, we performed substitution matrix analysis. We selected Chordata taxonomic quasi‐prime seven‐mers with an ε‐score exceeding 90.00% and for each peptide we generated all possible single‐amino acid variants that have a favorable score. We employed the BLOSUM62 (Henikoff &amp; Henikoff, <a href="#pro70241-bib-0022" class="usa-link" aria-describedby="pro70241-bib-0022">1992</a>) substitution matrix implemented through the blosum (version 2.0.3) Python package, where a peptide variant was classified as biologically conservative and was retained for further analysis, if the substitution had a positive score in the matrix. We considered substitutes only for canonical amino acids, excluding the ambiguity codes “X,” “*,” “B,” “Z,” and “J.” The set of biologically conservative peptides was then mapped against the proteins of all 24,073 reference proteomes.</p></section><section id="pro70241-sec-0012"><h3 class="pmc_sec_title">2.10. Protein ortholog mapping and multiple sequence alignment</h3>
<p>Protein orthologs for human proteins across the Chordata phylum were obtained through the EggNOG (version 5.0.0) (Huerta‐Cepas et al., <a href="#pro70241-bib-0023" class="usa-link" aria-describedby="pro70241-bib-0023">2019</a>) database. The orthologous groups of choice were later subjected to multiple sequence alignment (MSA) using the MAFFT (version 7) command line tool (Katoh &amp; Standley, <a href="#pro70241-bib-0028" class="usa-link" aria-describedby="pro70241-bib-0028">2013</a>) with the options: ‐‐localpair ‐‐maxiterate 1000 ‐‐amino ‐‐thread 5. The resulting alignments were trimmed using ClipKIT (version 2.3.0) (Steenwyk et al., <a href="#pro70241-bib-0050" class="usa-link" aria-describedby="pro70241-bib-0050">2020</a>) with the options ‐smart‐gap to remove poorly aligned protein regions and improve the phylogenetic signal by focusing on well‐conserved segments. The trimmed results were visualized using Jalview (version 2.11.4.1) (Waterhouse et al., <a href="#pro70241-bib-0055" class="usa-link" aria-describedby="pro70241-bib-0055">2009</a>). Sequences were ordered based on pairwise similarity, and taxonomic quasi‐prime‐containing regions were highlighted whereas distant regions with no peptides of interest were hidden from the visualization.</p></section><section id="pro70241-sec-0013"><h3 class="pmc_sec_title">2.11. Pathogenicity prediction in taxonomic quasi‐prime protein regions</h3>
<p>To predict the pathogenicity of single‐nucleotide missense variants, we employed AlphaMissense (Cheng et al., <a href="#pro70241-bib-0012" class="usa-link" aria-describedby="pro70241-bib-0012">2023</a>), (version 3, updated as of September 19, 2023, <a href="https://zenodo.org/records/10813168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/10813168</a>). The analysis included all possible missense variants (approximately 71 million) derived from 19,000 canonical protein‐coding transcripts in the human genome (hg38 build). Our study specifically focused on human proteins containing taxonomic quasi‐prime seven‐mers with ε‐scores exceeding 90.00%. We conducted a comparative analysis of the pathogenicity associated with missense mutations located within taxonomic quasi‐prime loci and those occurring outside these loci. Mutations with AlphaMissense scores below 0.1 were classified as likely benign, while those with scores above 0.9 were designated as highly pathogenic.</p></section></section><section id="pro70241-sec-0014"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="pro70241-sec-0015"><h3 class="pmc_sec_title">3.1. Derivation of superkingdom‐ and kingdom‐specific quasi‐prime peptides</h3>
<p>Taxonomic quasi‐prime five‐mers are identifiable at the superkingdom level and are exclusively found within the Eukaryota. We identified 12 distinct five‐mers with ε‐scores ranging from 0.52% to 3.74%, and a median ε‐score (ε<sub>M</sub>) of 1.67% (Supplementary Figure <a href="#pro70241-supitem-0004" class="usa-link">1</a>). The sequences of these 12 peptides are detailed in Supplementary Table <a href="#pro70241-supitem-0004" class="usa-link">1</a>. At the six‐mer level, we find that the median percentage of species containing these six amino acid peptides is 0.01% in viruses, 0.02% in Bacteria, 0.24% in Eukaryotes, and 0.27% in Archaea (Figure <a href="#pro70241-fig-0002" class="usa-link">2a</a>.). Similar patterns are observed for taxonomic quasi‐prime seven‐mers (Figure <a href="#pro70241-fig-0002" class="usa-link">2a</a>). The larger number of taxonomic quasi‐primes identified in Eukaryotes stems from their larger proteome sizes (Spearman's correlation coefficient <em>ρ</em> = 0.963, <em>p</em>‐value &lt;0.001) (Supplementary Figure <a href="#pro70241-supitem-0004" class="usa-link">S4</a>). Additionally, due to the large k‐mer space, these findings translate to a considerable number of superkingdom‐specific peptides. Specifically, at k‐mer lengths six and seven, we observe thousands of peptides that are only found in species of a single superkingdom (Figure <a href="#pro70241-fig-0002" class="usa-link">2a</a>). For instance, we observe 85,373 and 86,483,511 peptides found uniquely in bacterial proteomes at six and seven amino acids k‐mer lengths, respectively (Figure <a href="#pro70241-fig-0002" class="usa-link">2a</a>).</p>
<figure class="fig xbox font-sm" id="pro70241-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0080/12375989/41517317886e/PRO-34-e70241-g006.jpg" loading="lazy" id="jats-graphic-3" height="580" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/pro70241-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Taxonomic quasi‐prime peptide counts and ε‐score distribution. (a) Kernel density estimate plots showing taxonomic quasi‐prime peptide ε‐score distribution at superkingdom level. Dotted lines represent the median ε‐score (ε<sub>M</sub>) and solid lines represent the maximum ε‐score (ε<sub>max</sub>) value. Tables inside each plot display the number of unique taxonomic quasi‐prime peptide counts. (b) Taxonomic quasi‐prime peptide ε‐score distribution and peptide count at kingdom level. Left to right: Letter‐value plot of ε‐score distribution, where the ε<sub>M</sub> value is depicted as a solid black line and the ε<sub>max</sub> value as a rhombus. Bar plot of the unique taxonomic quasi‐prime peptide counts. (Data for taxonomic quasi‐prime six‐mers are displayed to the left and seven‐mers are displayed to the right).</p></figcaption></figure><p>The maximum ε‐score (ε<sub>max</sub>) of taxonomic quasi‐prime six‐mers was 0.59%, 10.10%, 17.31%, and 92.33% in viruses, Bacteria, Archaea, and Eukaryotes respectively. This means that the same taxonomic quasi‐prime six‐mer is found in 92.33% of reference Eukaryotes and is otherwise absent from all Bacteria, Archaea, and viruses. For taxonomic quasi‐prime seven‐mers, peptide SAPNYCY had an ε<sub>max</sub> of 98.11% for eukaryotic species, which maps to proteins of the serine/threonine phosphatase family, which are highly conserved in Eukaryotes (Ohama, <a href="#pro70241-bib-0043" class="usa-link" aria-describedby="pro70241-bib-0043">2019</a>) (Figure <a href="#pro70241-fig-0002" class="usa-link">2a</a>, Supplementary Figure <a href="#pro70241-supitem-0004" class="usa-link">3a</a>).</p>
<p>Next, we analyzed the distribution of taxonomic quasi‐primes across various organismal and viral kingdoms, observing significant variability within individual groups. This was particularly pronounced in viral kingdoms, where ε<sub>M</sub> values ranged from 0.02% to 10.00% at the six‐mer level and from 0.02% to 5.00% at the seven‐mer level in Heunggongvirae and Zilligvirae, respectively (Figure <a href="#pro70241-fig-0002" class="usa-link">2b</a>). Among eukaryotic kingdoms, the lowest ε<sub>M</sub> value was found in Fungi (0.09%), while the highest was observed in Protista (0.55%) (Figure <a href="#pro70241-fig-0002" class="usa-link">2b</a>). The ε<sub>max</sub> of taxonomic quasi‐prime six‐mers ranged from 17.03% to 87.18%, while for seven‐mers, values ranged from 37.91% to 98.78% in Protista and Metazoa, respectively (Supplementary Table <a href="#pro70241-supitem-0004" class="usa-link">2</a>). For Metazoa, the taxonomic quasi‐prime seven‐mer with the highest ε‐score was CKGFFKR, which mapped to proteins associated with the nuclear hormone receptor family. Proteins included in this family show strong sequence conservation and little evidence for positive selection in Metazoans (Krasowski et al., <a href="#pro70241-bib-0030" class="usa-link" aria-describedby="pro70241-bib-0030">2005</a>) (Figure <a href="#pro70241-fig-0002" class="usa-link">2b</a>, Supplementary Figure <a href="#pro70241-supitem-0004" class="usa-link">3b</a>).</p></section><section id="pro70241-sec-0016"><h3 class="pmc_sec_title">3.2. Derivation of phylum‐specific quasi‐prime peptides</h3>
<p>We investigated the presence and distribution of taxonomic quasi‐primes across various phyla. At the six‐amino‐acid sequence length, ε<sub>M</sub> values displayed substantial variation. In eukaryotic phyla, values ranged from 0.14% in Ascomycota to 100.00% in Foraminifera and 16 other eukaryotic phyla. For bacterial phyla, ε<sub>M</sub> values spanned from 0.03% in Pseudomonadota to 100.00% in Abditibacteriota and 17 additional bacterial phyla. Archaeal phyla exhibited values between 0.43% in Euryarchaeota and 100.00% in seven candidate archaeal phyla. In viral phyla, ε<sub>M</sub> values ranged from 0.02% in Uroviricota to 10.00% in Taleaviricota (Figure <a href="#pro70241-fig-0003" class="usa-link">3</a>; Supplementary Figures <a href="#pro70241-supitem-0004" class="usa-link">4</a> and <a href="#pro70241-supitem-0004" class="usa-link">5</a>). A similar distribution pattern was observed for the seven‐amino‐acid sequence length. In eukaryotic phyla, Ascomycota again exhibited the lowest ε<sub>M</sub> value at 0.14%, while Porifera reached the maximum value of 100.00%, shared with 16 other eukaryotic phyla. Among bacterial phyla, ε<sub>M</sub> values ranged from 0.03% for Pseudomonadota to 100.00% for Abditibacteriota and 18 additional bacterial phyla. For archaeal phyla, Euryarchaeota showed the lowest ε<sub>M</sub> value of 0.43%, while Candidatus Bathyarchaeota and seven other candidate archaeal phyla attained ε<sub>M</sub> of 100.00%. Viral phyla exhibited values ranging from 0.02% in Uroviricota to 12.50% in Dividoviricota (Figure <a href="#pro70241-fig-0003" class="usa-link">3</a>; Supplementary Figures <a href="#pro70241-supitem-0004" class="usa-link">4</a> and <a href="#pro70241-supitem-0004" class="usa-link">5</a>). We also observed that superkingdoms cluster by taxonomic quasi‐primes (Figure <a href="#pro70241-fig-0003" class="usa-link">3b</a>). The complete distribution of ε<sub>M</sub> values alongside ε<sub>max</sub> for each phylum is available at the Supplementary <a href="#pro70241-supitem-0001" class="usa-link">File 1</a>.</p>
<figure class="fig xbox font-sm" id="pro70241-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0080/12375989/650b3b7a0e55/PRO-34-e70241-g007.jpg" loading="lazy" id="jats-graphic-5" height="1073" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/pro70241-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Taxonomic quasi‐prime peptide statistics at phylum level. (a) Circos plot depicting the taxonomic organization of superkingdoms, kingdoms, and phyla at the archaeal, bacterial, eukaryotic, and viral levels (center). The number of unique taxonomic quasi‐prime peptides, alongside the median ε‐score value (ε<sub>M</sub>) at six‐mer and seven‐mer lengths, is also shown. Inner to outer: ε<sub>M</sub> for six‐mer taxonomic quasi‐prime peptides; total six‐mer taxonomic quasi‐prime peptide counts; ε<sub>M</sub> for seven‐mer taxonomic quasi‐prime peptides; total seven‐mer taxonomic quasi‐prime peptide counts. (b) UMAP plot depicting the clustering of reference proteomes based on their taxonomic quasi‐prime seven‐mers. Only the top 50th percentile of taxonomic quasi‐primes based on their ε‐score was used for the clustering. C, Candidatus; I.S, incertae sedis.</p></figcaption></figure><p>Across all three cellular superkingdoms, several phyla exhibited an ε<sub>max</sub> of 100.00%, whereas for viral phyla the ε<sub>max</sub> was capped at 50.00%. This is likely the result of viruses' rapid evolution in response to host immune pressures resulting in viral phyla encompassing greater genetic variation than cellular phyla. For both Streptophyta and Nematoda, we analyzed the proteins from which these highly phylum‐specific peptide seven‐mers originated. In Streptophyta, the taxonomic quasi‐primes TPWPGNN, EHFCIHA, and THHEYIQ were identified with an ε<sub>max</sub> of 99.11%. These peptides were found in <em>Arabidopsis thaliana</em> within the proteins Cellulose Synthase A Catalytic Subunit 4 (UDP‐forming), 3‐Ketoacyl‐CoA Synthase 1, and Callose Synthase 5, respectively (Supplementary Figure <a href="#pro70241-supitem-0004" class="usa-link">5c–e</a>). In Nematoda, a single peptide, ICPKYCA, was identified with an ε<sub>max</sub> of 97.65%. This peptide is located in the cuticle collagen DPY‐13 protein of <em>Caenorhabditis elegans</em> (Supplementary Figure <a href="#pro70241-supitem-0004" class="usa-link">5f</a>), which is crucial for cuticle formation, serving both as an exoskeleton and a protective barrier against environmental challenges.</p></section><section id="pro70241-sec-0017"><h3 class="pmc_sec_title">3.3. Taxonomic quasi‐primes enable the detection of loci divergence and functional adaptations across taxa</h3>
<p>To explore the functional roles of taxonomic quasi‐prime peptides and their associated proteins, we conducted a GOEA tailored to individual taxa. As expected, eukaryotic phyla, such as Chordata and Streptophyta, exhibited the highest number of statistically significant (adjusted <em>p</em>‐value &lt;0.05) enriched GO terms, while Archaea and Bacteria showed smaller comparable numbers. This finding in Eukaryotes is consistent with their larger proteomes and developmental complexity (Figure <a href="#pro70241-fig-0004" class="usa-link">4a</a>).</p>
<figure class="fig xbox font-sm" id="pro70241-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0080/12375989/b01fd2ad088b/PRO-34-e70241-g002.jpg" loading="lazy" id="jats-graphic-7" height="682" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/pro70241-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Multi‐phylum gene ontology (GO) enrichment analysis at the superkingdom level. (a) Stacked bar chart illustrating the total number of enriched GO terms, categorized by GO class, at the superkingdom level. (b)–(d) Heatmaps highlighting the top 10 GO class specific terms most broadly enriched across phyla within each of the three cellular organism superkingdoms. Dot size indicates the prevalence of each GO term among species within a given phylum, while dot color denotes the combined natural log (ln) of enrichment values, reflecting the strength of enrichment. Heatmaps are organized as follows: (b) biological processes, (c) molecular functions, (d) cellular components.</p></figcaption></figure><p>The results of the GOEA correctly pinpointed GO terms in major phyla that are fundamental to their known biology. In Chordata, we observed significant enrichment for terms such as ion transport, receptor tyrosine kinase signaling, synaptic vesicle, and postsynapse, all of which are hallmarks of the complex nervous system that characterizes this phylum (Figures <a href="#pro70241-fig-0004" class="usa-link">4b</a>, <a href="#pro70241-fig-0005" class="usa-link">5a–c</a>). Expanding on this track, we correctly identified ephrin receptor binding, which is known to be associated with nervous system development (Kao &amp; Kania, <a href="#pro70241-bib-0027" class="usa-link" aria-describedby="pro70241-bib-0027">2011</a>; Kullander et al., <a href="#pro70241-bib-0031" class="usa-link" aria-describedby="pro70241-bib-0031"><sup>2001</sup></a>) and angiogenesis (Cheng et al., <a href="#pro70241-bib-0013" class="usa-link" aria-describedby="pro70241-bib-0013">2002</a>), and clathrin binding, known for clathrin‐mediated endocytosis (McMahon &amp; Boucrot, <a href="#pro70241-bib-0037" class="usa-link" aria-describedby="pro70241-bib-0037">2011</a>), to be highly associated with taxonomic quasi‐prime peptides (Figure <a href="#pro70241-fig-0005" class="usa-link">5b</a>).</p>
<figure class="fig xbox font-sm" id="pro70241-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0080/12375989/891c709f62ce/PRO-34-e70241-g003.jpg" loading="lazy" id="jats-graphic-9" height="620" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/pro70241-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene ontology (GO) term cluster enrichment analysis across superkingdom representative phyla, with taxonomic quasi‐prime peptides. Lollipop plots display the mean enrichment of the top three most enriched clustered GO terms for representative phyla of the cellular organism superkingdoms, categorized into: (a) Biological processes, (b) molecular functions, (c) cellular components. Only GO terms with an adjusted <em>p</em>‐value less than 0.05 and a species representation greater than 5% are included. Dot size represents cluster size, lollipop color represents the −log10 adjusted <em>p</em>‐value, and error bars show the 95% confidence interval of the calculations. GO terms have been grouped into broader clusters using Wang's semantic similarity measure combined with hierarchical clustering.</p></figcaption></figure><p>Our validating results are not limited to Chordata. For instance, in Mollusca, estrogen signaling and estrogen response binding are correctly found enriched, confirming their reliance on environmental uptake of estrogen hormones (Balbi et al., <a href="#pro70241-bib-0004" class="usa-link" aria-describedby="pro70241-bib-0004">2019</a>) (Figure <a href="#pro70241-fig-0005" class="usa-link">5a, b</a>). The enrichment of taxonomic quasi‐primes in the microtubules and axonemal microtubules of Euglenozoa and Cryptomycota, respectively, aligns perfectly with the critical role these components play in flagellar motility and host interactions (Jones et al., <a href="#pro70241-bib-0024" class="usa-link" aria-describedby="pro70241-bib-0024">2011</a>). In archaeal and bacterial phyla, 4Fe‐4S cluster binding was found enriched.</p>
<p>Beyond this validation, our analysis also revealed more nuanced and novel insights. Membrane and transmembrane transport were universally enriched, highlighting how this fundamental process is fine‐tuned through lineage‐specific adaptations (Figure <a href="#pro70241-fig-0004" class="usa-link">4b, c</a>). More strikingly, we identified unexpected enrichment of eukaryotic‐like functions in Poribacteria, a bacterial phylum known to be a symbiot of Porifera. The enrichment of GO terms related to the RNA polymerase II transcription initiation BP and the dopamine monooxygenase MF (Figure <a href="#pro70241-fig-0005" class="usa-link">5a, b</a>) suggests that the known mimicry of eukaryotic proteins by this bacterium (Kamke et al., <a href="#pro70241-bib-0026" class="usa-link" aria-describedby="pro70241-bib-0026">2014</a>) extends down to the level of its unique taxonomic quasi‐primes, a novel insight provided by our methodology.</p></section><section id="pro70241-sec-0018"><h3 class="pmc_sec_title">3.4. Taxonomic quasi‐primes detect superkingdom‐ and phylum‐level protein adaptations</h3>
<p>To investigate taxon‐specific functional changes, we analyzed the distribution of taxonomic quasi‐primes within protein domains and families, collectively termed as protein entries. Across all three superkingdoms, the major facilitator superfamily domain (MFS_dom), critical for transmembrane transport (Pao et al., <a href="#pro70241-bib-0044" class="usa-link" aria-describedby="pro70241-bib-0044">1998</a>), emerged as a highly enriched domain (Figure <a href="#pro70241-fig-0006" class="usa-link">6a</a>), underscoring its pivotal role in transport processes essential for cellular survival and adaptation (Complete protein domain names can be found at: Supplementary <a href="#pro70241-supitem-0001" class="usa-link">[Link]</a>, <a href="#pro70241-supitem-0002" class="usa-link">[Link]</a>). This aligns with previously found enriched transmembrane transport BP shared across all three superkingdoms.</p>
<figure class="fig xbox font-sm" id="pro70241-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0080/12375989/33085a23d41a/PRO-34-e70241-g005.jpg" loading="lazy" id="jats-graphic-11" height="682" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/pro70241-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Preferential presence of taxonomic quasi‐primes in specific protein domains. (a) Size‐ and color‐coded heatmap displaying the top 10 protein domains with taxonomic quasi‐primes enriched across most phyla within each of the three cellular organism superkingdoms. Dot size indicates the prevalence of each protein domain within species of the respective phylum, while dot color corresponds to the combined ln(Enrichment) value for each domain. (b)–(d) Circos plots of the top three enriched protein domains per phylum with an adjusted <em>p</em>‐value less than 0.05 and a species prevalence greater than 5% for each cellular organism superkingdom. (b) Archaea, (c) Bacteria, and (d) Eukaryotes. Inner to outer: Phylum name; Heatmap depicting the percentage of the species within the respective phylum that have the specific protein domain enriched; Barplot showing the ln(Enrichment) value with error bars representing the 95% confidence interval of the calculation.</p></figcaption></figure><p>Archaea and Bacteria displayed similar enrichment patterns in protein entries, which were largely absent in Eukaryota, and vice versa (Figures <a href="#pro70241-fig-0006" class="usa-link">6a</a> and <a href="#pro70241-fig-0007" class="usa-link">7a</a>). This indicates evolutionary conservation of protein entries across these two superkingdoms, due to their shared environmental challenges, horizontal gene transfer events, and adaptive responses. Enriched domains included those associated with oxidoreductase activity, transferase activity, catalytic activity, and metal ion binding. Particularly, the radical S‐adenosyl‐L‐methionine (rSAM) domain and its associated families (e.g., Elp3/MiaA/NifB‐like, PqqE‐like) (Figures <a href="#pro70241-fig-0006" class="usa-link">6a</a> and <a href="#pro70241-fig-0007" class="usa-link">7a</a>) were enriched, reflecting their roles in essential enzymatic functions. These families belong to the rSAM enzyme superfamily (Frey et al., <a href="#pro70241-bib-0017" class="usa-link" aria-describedby="pro70241-bib-0017">2008</a>) and involve 4Fe‐4S cluster binding (4Fe4S_Fe‐S‐bd), critical for catalysis.</p>
<figure class="fig xbox font-sm" id="pro70241-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0080/12375989/8cb9c6dc0087/PRO-34-e70241-g004.jpg" loading="lazy" id="jats-graphic-13" height="704" width="709" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/pro70241-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Analysis of the predominant taxonomic quasi‐primes in superkingdom specific protein families. (a) Size‐ and color‐coded heatmap identifying the top 10 enriched protein families widely present across phyla within each of the three cellular organism superkingdoms. Dot size indicates the prevalence of each protein family within species of the respective phylum, while dot color corresponds to the combined ln(Enrichment) value for each family. (b)–(d) Circos plots showcasing the top three most enriched protein families per phylum with an adjusted <em>p</em>‐value less than 0.05 and a species prevalence greater than 5% in: (b) Archaea, (c) Bacteria, and (d) Eukaryotes. Inner to outer: Phylum name; Heatmap depicting the percentage of the species within the respective phylum that have the specific protein family enriched; Barplot showing the ln(Enrichment) value with error bars representing the 95% confidence interval of the calculation.</p></figcaption></figure><p>Aminoacyl‐tRNA synthetase domains were enriched in Archaea and Bacteria (aa‐tRNA‐synth Ia and aa‐tRNA‐synth II), with bacterial‐specific enrichment of the class II G/P/S/T subtype (Figure <a href="#pro70241-fig-0006" class="usa-link">6a</a>). Three class Ia tRNA ligase protein families (Valine‐, Leucine and Isoleucine‐tRNA ligases) are found enriched predominantly in most archaeal and bacterial phyla (Figure <a href="#pro70241-fig-0007" class="usa-link">7a</a>), supporting protein synthesis under extreme conditions. These adaptations compensate for the limited diversity of post‐transcriptional modifications in prokaryotes.</p>
<p>In Eukaryotes, enriched entries included the ion transport domain (Figure <a href="#pro70241-fig-0006" class="usa-link">6a</a>), found in sodium, potassium, and calcium ion channels, the TRIAD supradomain present in the E3 ubiquitin ligase RBR family, and the myosin head motor domain alongside the dynein heavy chain family (Figure <a href="#pro70241-fig-0007" class="usa-link">7a</a>). E3 ubiquitin ligases are highly conserved across Eukaryotes since they are a part of the ubiquitin‐proteasome system, involved in protein degradation and the maintenance of cellular homeostasis (Yang et al., <a href="#pro70241-bib-0059" class="usa-link" aria-describedby="pro70241-bib-0059">2021</a>). Enrichment of motor protein entries highlights the necessity of quasi‐prime adaptations for intracellular transport and cellular motility.</p>
<p>We also identified the top three enriched entries within specific phyla. In Cyanobacteria, entries related to the circadian clock oscillator protein family (e.g., KaiC) (Figures <a href="#pro70241-fig-0006" class="usa-link">6b</a> and <a href="#pro70241-fig-0007" class="usa-link">7b</a>) are enriched, suggesting the high evolutionary conservation of taxonomic quasi‐primes in the regulation of day‐night cycles (Markson &amp; O'Shea, <a href="#pro70241-bib-0034" class="usa-link" aria-describedby="pro70241-bib-0034">2009</a>). In Candidatus Paceibacterota, the UV‐induced DNA damage repair photolyase family was enriched, warranting further study (Figure <a href="#pro70241-fig-0007" class="usa-link">7b</a>). In Chordata, the heparan sulfate‐N‐deacetylase protein domain, critical for the biosynthesis of heparan sulfate (Sarrazin et al., <a href="#pro70241-bib-0048" class="usa-link" aria-describedby="pro70241-bib-0048">2011</a>), is highlighted (Figure <a href="#pro70241-fig-0006" class="usa-link">6</a>c). Also, the sodium channel A8 and A1 subunit families, alongside the gamma‐aminobutyric‐acid A receptor, alpha 1 subunit, are present across most Chordata species, further facilitating the involvement of taxonomic quasi‐primes in the function of the nervous system. Leucine‐rich repeat families were prominent in Arthropoda (Figure <a href="#pro70241-fig-0007" class="usa-link">7c</a>) underscoring roles in immune defense and development, with emphasis on Toll‐like receptors (Dey et al., <a href="#pro70241-bib-0014" class="usa-link" aria-describedby="pro70241-bib-0014">2022</a>) and extracellular matrix integrity (Matsushima et al., <a href="#pro70241-bib-0035" class="usa-link" aria-describedby="pro70241-bib-0035">2021</a>). Little is known about how these families function in Arthropods, and the existence of highly specific taxonomic quasi‐primes emphasizes the need for further study.</p></section><section id="pro70241-sec-0019"><h3 class="pmc_sec_title">3.5. Secondary structure analysis of taxonomic quasi‐primes in diverse organisms</h3>
<p>We next analyzed the secondary structures of protein loci containing taxonomic quasi‐prime seven‐mers with an ε‐score greater than 90.00% across 13 model organisms and 17 pathogens of global health concern. Our findings revealed a distinct enrichment of taxonomic quasi‐primes in protein coils, followed by alpha helices and turns (Supplementary Figure <a href="#pro70241-supitem-0004" class="usa-link">6</a>). We evaluated the statistical significance of this structural preference using Kruskal–Wallis tests. For coiled‐like conformations (encompassing alpha helices, coils, and turns), the analysis yielded an H‐statistic of 213,788.82 with a <em>p</em>‐value &lt;0.001 in model organisms, and an H‐statistic of 1352.19 with a <em>p</em>‐value &lt;0.001 in pathogen proteomes. Nevertheless, we report significant differences depending on the organism studied; for instance, <em>Schizosaccharomyces pombe</em> shows a preference for taxonomic quasi‐primes in alpha helices and <em>Plasmodium falciparum</em> in disordered secondary structures (Supplementary Figure <a href="#pro70241-supitem-0004" class="usa-link">6</a>).</p></section><section id="pro70241-sec-0020"><h3 class="pmc_sec_title">3.6. Multiple Sequence Alignment (MSA) and structural insights into chordata proteins</h3>
<p>To further investigate the significance of highly conserved taxonomic quasi‐prime peptides in molecular function, we performed tertiary analysis on four representative Chordata proteins. These proteins were selected because they are prime representatives of the enriched Chordata GO terms and protein entries identified in the previous analyses. Also, these proteins contained taxonomic quasi‐prime seven‐mers with an ε‐score greater than 90.00% (Supplementary Table <a href="#pro70241-supitem-0004" class="usa-link">3</a>).</p>
<p>The four selected proteins include:</p>
<ul id="pro70241-list-0001" class="list" style="list-style-type:disc">
<li id="pro70241-li-0001"><p>Sodium channel 8A (SCN8A) (Figure <a href="#pro70241-fig-0008" class="usa-link">8a</a>): Represents the enriched ion transport BP (Figure <a href="#pro70241-fig-0004" class="usa-link">4b</a>) and the sodium channel α8 family (Figure <a href="#pro70241-fig-0007" class="usa-link">7d</a>).</p></li>
<li id="pro70241-li-0002"><p>E3 ubiquitin‐protein ligase ARIH2 (ARIH2) (Figure <a href="#pro70241-fig-0008" class="usa-link">8b</a>): Represents the enriched E3 ubiquitin ligase RBR family (Figure <a href="#pro70241-fig-0007" class="usa-link">7a</a>) and its associated TRIAD supradomain (Figure <a href="#pro70241-fig-0006" class="usa-link">6a</a>), which are linked to the enriched metal ion binding MF (Figure <a href="#pro70241-fig-0004" class="usa-link">4c</a>).</p></li>
<li id="pro70241-li-0003"><p>Heparan sulfate N‐deacetylase‐N‐sulfotransferase 1 (NDST1) (Figure <a href="#pro70241-fig-0008" class="usa-link">8c</a>): Selected due to the enrichment of the heparan SO<sub>4</sub> deacetylase domain (Figure <a href="#pro70241-fig-0006" class="usa-link">6d</a>).</p></li>
<li id="pro70241-li-0004"><p>Myosin 7 (MYH7) (Figure <a href="#pro70241-fig-0008" class="usa-link">8d</a>): Represents the enriched myosin complex CC (Figure <a href="#pro70241-fig-0004" class="usa-link">4d</a>) and the myosin head motor dom‐like domain (Figure <a href="#pro70241-fig-0006" class="usa-link">6a</a>), which are fundamental for motor functions like muscle contraction.</p></li>
</ul>
<figure class="fig xbox font-sm" id="pro70241-fig-0008"><h4 class="obj_head">FIGURE 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0080/12375989/cad16721b97c/PRO-34-e70241-g009.jpg" loading="lazy" id="jats-graphic-15" height="687" width="709" alt="FIGURE 8"></p>
<div class="p text-right font-secondary"><a href="figure/pro70241-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structural profiling of taxonomic quasi‐primes across Chordata representative proteins. Multiple sequence alignment and structural representations of proteins containing taxonomic quasi‐primes with ε‐score greater than 90% in: (a) Human sodium channel 8A (<em>SCN8A</em>). (b) Human E3 ubiquitin‐protein ligase ARIH2 (<em>ARIH2</em>). (c) Human heparan sulfate N‐deacetylase‐N‐sulfotransferase 1 (<em>NDST1</em>). (d) Human myosin 7 (<em>MYH7</em>). Taxonomic quasi‐prime peptide region is displayed in yellow. Inset panels provide a detailed view of these regions, illustrating their specific locations within proteins, where they play a functional role. A3P: Adenosine‐3′‐5′‐Diphosphate.</p></figcaption></figure><p>Twelve taxonomic quasi‐primes, spread across various regions of the SCN8A protein, were identified (Figure <a href="#pro70241-fig-0008" class="usa-link">8a</a>), suggesting a predominantly structural role rather than a functional one, as these regions are not known to actively interact with lipid substrates. In the ARI2 protein, three taxonomic quasi‐primes were found to interact directly with zinc ions through coordinate covalent bonds at the Cys257, Cys260, and His265 sites. These sites have been characterized as essential for the structural stability and catalytic activity of the enzyme (Figure <a href="#pro70241-fig-0008" class="usa-link">8b</a>), since they are present in the IBR linker domain (Duda et al., <a href="#pro70241-bib-0015" class="usa-link" aria-describedby="pro70241-bib-0015">2013</a>). Eleven taxonomic quasi‐primes are identified in the NDST1 protein at regions related to calcium and adenosine‐3′‐5′‐diphosphate (A3P) binding. Particularly, the residues His389 and His393 both interact with the calcium ion through coordination covalent bonds, while Phe816 interacts with A3P through π‐stacking interactions (Figure <a href="#pro70241-fig-0008" class="usa-link">8c</a>). The sequence coordinates of taxonomic quasi‐primes overlap with the sulfotransferase and the deacetylase active sites, which are known to stabilize the tertiary structure, enabling the correct placement of the N‐acetyl‐heparan sulfate (Mycroft‐West et al., <a href="#pro70241-bib-0042" class="usa-link" aria-describedby="pro70241-bib-0042">2024</a>). Five quasi‐primes were identified in the MYH7 protein, interacting with the myosin regulatory light chain 2, ventricular/cardiac muscle isoform (MLC2v) (Figure <a href="#pro70241-fig-0008" class="usa-link">8d</a>) through one hydrogen bond at the Trp827 site and electrostatic interactions at the positively charged Lys831. This interaction is found to regulate motor function, stabilize the myosin complex, and enhance calcium sensitivity, ensuring efficient cardiac muscle contraction (Rayment et al., <a href="#pro70241-bib-0046" class="usa-link" aria-describedby="pro70241-bib-0046">1993</a>).</p>
<p>This functional importance is not limited only to human proteins. MSA results display that the highly conserved taxonomic quasi‐primes are present within the same coordinates across Chordata orthologs (Figure <a href="#pro70241-fig-0008" class="usa-link">8</a>). This finding is a strong indicator of purifying selection, suggesting that these specific peptide sequences are indispensable for the protein's function and have been maintained throughout evolution.</p></section><section id="pro70241-sec-0021"><h3 class="pmc_sec_title">3.7. The uniqueness of taxonomic quasi‐primes is highly dependent on exact sequence identity</h3>
<p>After having successfully identified taxonomic quasi‐primes across superkingdoms, kingdoms, and phyla, we set out to test if this uniqueness expanded to biochemical properties other than sequence motifs. We focused on the Chordata quasi‐prime seven‐mers with an ε‐score greater than 90.00%. This set of taxonomic quasi‐primes amounted to 2,652 unique peptides. Using the BLOSUM62 substitution matrix, we generated all biologically conservative single amino‐acid variants, where the BLOSUM score was above 0, which were later mapped against all 24,073 reference proteomes.</p>
<p>Of the 37,330 unique conservative variants generated, 67.27% were found in at least one proteome outside of Chordata, 23.72% were mapped to no proteins, and only 9.01% were identified exclusively within the Chordata phylum (Supplementary Figure <a href="#pro70241-supitem-0004" class="usa-link">7b</a>). In total, these variants appeared in 1,007,246 proteins. 83.41% of these were Chordata proteins, followed by 2.92% in Arthropoda, 2.27% in Streptophyta, and 1.66% in Ascomycota, all eukaryotic phyla (Supplementary Figure <a href="#pro70241-supitem-0004" class="usa-link">7a</a>). For each original Chordata quasi‐prime, the vast majority of its peptide variants mapped to eukaryotic proteins (Supplementary Figure <a href="#pro70241-supitem-0004" class="usa-link">7c</a>). We found three taxonomic quasi‐prime seven‐mers, whose variants mapped exclusively to eukaryotic proteins. These peptides were CCEEWVC with 11 variants, CPKRCVC with 8 variants, and CCTPHRT with six variants. The taxonomic quasi‐prime HEQCCWT, with its 11 variants, was identified exclusively in Chordata species, exhibiting an ε‐score of 93.23%. Notably, none of its variants mapped to any proteins across the four superkingdoms. This specific taxonomic quasi‐prime is situated within the human H(+)/Cl(−) exchange transporter 4, spanning sequence coordinates 111–117.</p></section><section id="pro70241-sec-0022"><h3 class="pmc_sec_title">3.8. Pathogenic variants are enriched at human taxonomic quasi‐prime loci</h3>
<p>Finally, we examined if taxonomic quasi‐prime loci are more likely to harbor pathogenic variants than surrounding sequences. We used human proteome‐wide missense variant effect prediction maps (Cheng et al., <a href="#pro70241-bib-0012" class="usa-link" aria-describedby="pro70241-bib-0012">2023</a>), provided by AlphaMissense, and investigated if the subset of missense variants that are pathogenic is more likely to be found at taxonomic quasi‐prime loci. We find that the distribution of variants overlapping taxonomic quasi‐primes is significantly sifted toward pathogenic effects (Kolmogorov–Smirnov test <em>p</em>‐value &lt;0.001, Cliff's delta = 0.448) with taxonomic quasi‐prime loci being 2.08‐fold more likely to overlap pathogenic variants than expected (Figure <a href="#pro70241-fig-0009" class="usa-link">9</a>). The finding that pathogenic missense variants are significantly enriched within taxonomic quasi‐prime loci underscores their critical importance to protein function. This strong link between lineage‐specific sequences and functional significance makes these regions valuable targets for therapeutic development, including small‐molecule inhibitors or highly specific antibody‐based therapies. From a diagnostic perspective, this implies that mutations within the genomic regions that encode taxonomic quasi‐primes should be prioritized during clinical variant analysis to better assess disease risk and a patient's health status.</p>
<figure class="fig xbox font-sm" id="pro70241-fig-0009"><h4 class="obj_head">FIGURE 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0080/12375989/7e93ad535ded/PRO-34-e70241-g010.jpg" loading="lazy" id="jats-graphic-17" height="527" width="672" alt="FIGURE 9"></p>
<div class="p text-right font-secondary"><a href="figure/pro70241-fig-0009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Missense mutations in highly conserved quasi‐prime regions are more likely to be pathogenic. The kernel density estimate plot compares the distribution of AlphaMissense‐predicted pathogenicity scores for missense mutations within human taxonomic quasi‐prime loci (for Chordata quasi‐prime seven‐mers with ε‐scores above 90%) to those outside these loci within the same proteins. Highlighted ranges indicate highly benign mutations (pathogenicity scores &lt;0.1) and highly pathogenic mutations (pathogenicity scores &gt;0.9). Statistical metrics: Kolmogorov–Smirnov (K–S) statistic, Cliff's delta (δ), and <em>p</em>‐value (<em>p</em>).</p></figcaption></figure></section></section><section id="pro70241-sec-0023"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>The presence of short peptide k‐mers that are highly specific to individual taxonomies has not been studied to date. Here, we provide evidence that based on the tens of thousands of available reference proteomes, such sequences can be systematically discovered across taxonomic levels. We identify peptides that exhibit remarkable taxonomic uniqueness at six‐ and seven‐amino‐acid lengths, offering insights into evolutionary divergence, lineage‐specific adaptations, and proteomic diversity.</p>
<p>We observe large variations in the number and frequency of taxonomic quasi‐primes when comparing different taxonomies. Protista display substantially lower ε<sub>max</sub> scores when compared to other eukaryotic kingdoms, indicating that their characteristic taxonomic quasi‐prime peptides appear only in a small subset of the corresponding species. This can be attributed to the highly diverse polyphyletic nature of Protista, which encompasses both multicellular and unicellular organisms. Expanding on this, Protista exhibit a wide variety of environments, from marine and terrestrial ecosystems to parasitic niches, increasing the need to adopt different mechanisms for survivability (Burki et al., <a href="#pro70241-bib-0008" class="usa-link" aria-describedby="pro70241-bib-0008">2021</a>).</p>
<p>The identification of taxonomic quasi‐primes offers a novel way to study sequence divergence, speciation, and trait development. By identifying k‐mer peptides unique to specific taxa, these sequences can serve as molecular markers of evolutionary processes and lineage‐specific adaptations. Analyzing the proteins harboring taxonomic quasi‐prime peptides may reveal how unique traits emerge, such as specialized metabolic pathways or complex physiological processes. Future research could integrate phylogenetic analyses with taxonomic quasi‐prime peptide distributions to trace evolutionary histories and uncover the genetic basis of speciation events, shedding light on how sequence‐level variations drive biological diversity and the evolution of novel traits.</p>
<p>Tertiary analysis revealed that taxonomic quasi‐primes are not randomly located within the protein structure. Instead, they are preferentially identified in functionally critical regions, where they also facilitate direct roles in the protein's function. They are involved in catalysis and structural stability; they stabilize enzymatic active sites, they mediate essential protein–protein interactions, and finally, they contribute to the stable conformation of large protein complexes.</p>
<p>Although we could not pinpoint any taxonomic quasi‐primes with all variants exclusively within the Chordata phylum, our examination of variant peptides from highly conserved Chordata quasi‐prime seven‐mers revealed that 1,740 out of 2,652 taxonomic quasi‐primes had some corresponding variants exclusively mapped to Chordata and present in our original dataset. To illustrate this, we selected the peptide MWDCMEV (ε‐score = 98.27%), which had 16 variants, 8 of which were specific to the Chordata phylum. This selection was based on MWDCMEV having one of the highest numbers of Chordata‐specific variants and the highest ε‐score.</p>
<p>We propose transitioning from exact taxonomic quasi‐prime sequences to a consensus taxonomic quasi‐prime pattern that encompasses these variants. For instance, the MWDCMEV peptide can be reformatted into the taxonomic quasi‐prime pattern [LM][FW][DEN]CM[EKQ][ILV], where bracketed amino acids represent possible residues at that position. This approach aligns with previous work on pattern‐based categorization, such as'seqlets’ (Smith &amp; Smith, <a href="#pro70241-bib-0049" class="usa-link" aria-describedby="pro70241-bib-0049">1990</a>), recurring, conserved amino acid patterns discovered from large, unaligned protein databases using the TEIRESIAS algorithm (Rigoutsos et al., <a href="#pro70241-bib-0047" class="usa-link" aria-describedby="pro70241-bib-0047">1999</a>). A similar methodology could advance the concept of taxonomic quasi‐primes toward a pattern‐based direction.</p>
<p>Furthermore, inspired by the Aho‐Corasick algorithm, these taxonomic quasi‐prime patterns can be represented as prefix trees for efficient taxonomic categorization of newly discovered proteomes. A curated set of highly conserved taxonomic quasi‐primes, each with its variants represented as a prefix tree (Figure <a href="#pro70241-fig-0010" class="usa-link">10</a>), can be created for each taxonomic rank. By scanning a novel proteome for these patterns, we can generate a quantitative taxonomic profile for the organism. Counting pattern hits for each major taxonomy (e.g., Chordata, Arthropoda, Bacteria) allows us to calculate a likelihood score, assigning the novel proteome to its most probable lineage. This offers a powerful and efficient method for sequence‐based classification, where matches with higher ε‐scores provide information about the general rank, and associated matches with lower ε‐scores indicate the closest species within that taxonomic rank.</p>
<figure class="fig xbox font-sm" id="pro70241-fig-0010"><h3 class="obj_head">FIGURE 10.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0080/12375989/46a951d6adad/PRO-34-e70241-g001.jpg" loading="lazy" id="jats-graphic-19" height="452" width="709" alt="FIGURE 10"></p>
<div class="p text-right font-secondary"><a href="figure/pro70241-fig-0010/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prefix tree for the Chordata quasi‐prime MWDCMEV. End nodes are colored based on ε‐score, with higher scores having dark blue color.</p></figcaption></figure><p>Taxonomic quasi‐prime applications are not limited to just taxonomic classification. Immunological methods based on antibodies can face limitations in sensitivity and specificity, particularly due to cross‐reactivity (Wild, <a href="#pro70241-bib-0057" class="usa-link" aria-describedby="pro70241-bib-0057">2013</a>). By prioritizing antigens that harbor taxonomic quasi‐prime peptides, the sensitivity and specificity of designed antibodies can be increased. Given their short length and unique taxonomic specificity, taxonomic quasi‐prime peptides are well suited for integration into mass‐spectrometry workflows (Birhanu, <a href="#pro70241-bib-0007" class="usa-link" aria-describedby="pro70241-bib-0007">2023</a>; Dupree et al., <a href="#pro70241-bib-0016" class="usa-link" aria-describedby="pro70241-bib-0016"><sup>2020</sup></a>). Their inherent specificity can enhance the precision of proteomic analyses, enabling more efficient identification of proteins in complex biological samples (Li et al., <a href="#pro70241-bib-0033" class="usa-link" aria-describedby="pro70241-bib-0033">2023</a>). In agriculture, taxonomic quasi‐prime peptides can serve as highly specific biomarkers for pathogen detection. Examination of taxonomic quasi‐prime peptides specific to microbial communities or ecological niches could also provide insights into microbiome dynamics and their role in ecosystem function. In diagnostics, taxonomic quasi‐prime peptides could facilitate the identification of pathogenic organisms in clinical samples, contributing to faster and more precise diagnostics for infectious diseases.</p>
<p>The number of available proteomes represents only a small subset of the species known. Therefore, future work is required to examine how these findings change as more reference proteomes of different organisms become available. The use of TrEMBL proteins limits our findings to some extent, further necessitating the need for future work where highest quality protein annotations will be present. Furthermore, the incorporation of protein isoforms and population variants could influence our conclusions, and further exploration toward these directions is needed, particularly in eukaryotic proteomes, when such data become available across multiple organisms.</p></section><section id="pro70241-sec-0028"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Eleftherios Bochalis:</strong> Conceptualization; writing – original draft; writing – review and editing; visualization; methodology; software; data curation; formal analysis; validation; investigation. <strong>Michail Patsakis:</strong> Methodology; software; data curation; formal analysis; writing – original draft. <strong>Nikol Chantzi:</strong> Conceptualization; investigation; validation; methodology. <strong>Ioannis Mouratidis:</strong> Writing – review and editing; funding acquisition; supervision; validation; investigation. <strong>Dionysios V. Chartoumpekis:</strong> Writing – review and editing; supervision. <strong>Ilias Georgakopoulos‐Soares:</strong> Supervision; writing – original draft; writing – review and editing; funding acquisition; project administration; validation; conceptualization.</p></section><section id="pro70241-sec-0025"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no competing interests.</p></section><section id="sec26"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pro70241-supitem-0001"><div class="caption p"><p>
Supplementary File 1:
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375989/bin/PRO-34-e70241-s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70241-s004.xlsx</a><sup> (19.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pro70241-supitem-0002"><div class="caption p"><p>
Supplementary File 2:
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375989/bin/PRO-34-e70241-s003.csv" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70241-s003.csv</a><sup> (149.5KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="pro70241-supitem-0003"><div class="caption p"><p>
Supplementary File 3:
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375989/bin/PRO-34-e70241-s001.csv" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70241-s001.csv</a><sup> (278.3KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="pro70241-supitem-0004"><div class="caption p"><p>
<strong>Data S1:</strong> Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375989/bin/PRO-34-e70241-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70241-s002.docx</a><sup> (5.3MB, docx) </sup>
</div></div></section></section><section id="pro70241-sec-0024" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>Research reported in this publication was supported by the Huck Institutes of the Life Sciences and the National Institute of General Medical Sciences of the National Institutes of Health under Award Number R35GM155468. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.</p></section><section id="notes1"><p>

Bochalis E, Patsakis M, Chantzi N, Mouratidis I, Chartoumpekis DV, Georgakopoulos‐Soares I. Taxonomic quasi‐primes: peptides charting lineage‐specific adaptations and disease‐relevant loci. Protein Science. 2025;34(9):e70241. 10.1002/pro.70241

</p>
<section id="pro70241-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="pro70241-note-0001"><p>
<strong>Review Editor:</strong> Nir Ben‐Tal</p></div></div></section></section><section id="pro70241-sec-0027"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>The data that support the findings of this study are openly available in Taxonomic quasi‐prime raw data at <a href="https://zenodo.org/records/14385095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/14385095</a>. Code used for this study is openly available at <a href="https://github.com/Georgakopoulos-Soares-lab/taxonomic_quasi_primes" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Georgakopoulos-Soares-lab/taxonomic_quasi_primes</a>.</p></section><section id="pro70241-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="pro70241-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="pro70241-bib-0001">
<cite>
Agresti A. On logit confidence intervals for the odds ratio with small samples. Biometrics. 1999;55:597–602.
</cite> [<a href="https://doi.org/10.1111/j.0006-341x.1999.00597.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11318220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=On%20logit%20confidence%20intervals%20for%20the%20odds%20ratio%20with%20small%20samples&amp;author=A%20Agresti&amp;volume=55&amp;publication_year=1999&amp;pages=597-602&amp;pmid=11318220&amp;doi=10.1111/j.0006-341x.1999.00597.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0002">
<cite>
Al‐Amrani S, Al‐Jabri Z, Al‐Zaabi A, Alshekaili J, Al‐Khabori M. Proteomics: concepts and applications in human medicine. World J Biol Chem. 2021;12:57–69.
</cite> [<a href="https://doi.org/10.4331/wjbc.v12.i5.57" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8473418/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34630910/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=World%20J%20Biol%20Chem&amp;title=Proteomics:%20concepts%20and%20applications%20in%20human%20medicine&amp;author=S%20Al%E2%80%90Amrani&amp;author=Z%20Al%E2%80%90Jabri&amp;author=A%20Al%E2%80%90Zaabi&amp;author=J%20Alshekaili&amp;author=M%20Al%E2%80%90Khabori&amp;volume=12&amp;publication_year=2021&amp;pages=57-69&amp;pmid=34630910&amp;doi=10.4331/wjbc.v12.i5.57&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0003">
<cite>
Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, et al. Gene ontology: tool for the unification of biology. Nat Genet. 2000;25:25–29.
</cite> [<a href="https://doi.org/10.1038/75556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3037419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10802651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Gene%20ontology:%20tool%20for%20the%20unification%20of%20biology&amp;author=M%20Ashburner&amp;author=CA%20Ball&amp;author=JA%20Blake&amp;author=D%20Botstein&amp;author=H%20Butler&amp;volume=25&amp;publication_year=2000&amp;pages=25-29&amp;pmid=10802651&amp;doi=10.1038/75556&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0004">
<cite>
Balbi T, Ciacci C, Canesi L. Estrogenic compounds as exogenous modulators of physiological functions in molluscs: signaling pathways and biological responses. Comp Biochem Physiol C Toxicol Pharmacol. 2019;222:135–144.
</cite> [<a href="https://doi.org/10.1016/j.cbpc.2019.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31055067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comp%20Biochem%20Physiol%20C%20Toxicol%20Pharmacol&amp;title=Estrogenic%20compounds%20as%20exogenous%20modulators%20of%20physiological%20functions%20in%20molluscs:%20signaling%20pathways%20and%20biological%20responses&amp;author=T%20Balbi&amp;author=C%20Ciacci&amp;author=L%20Canesi&amp;volume=222&amp;publication_year=2019&amp;pages=135-144&amp;pmid=31055067&amp;doi=10.1016/j.cbpc.2019.05.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0005">
<cite>
Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, et al. The Protein Data Bank. Nucleic Acids Res. 2000;28:235–242.
</cite> [<a href="https://doi.org/10.1093/nar/28.1.235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20Protein%20Data%20Bank&amp;author=HM%20Berman&amp;author=J%20Westbrook&amp;author=Z%20Feng&amp;author=G%20Gilliland&amp;author=TN%20Bhat&amp;volume=28&amp;publication_year=2000&amp;pages=235-242&amp;pmid=10592235&amp;doi=10.1093/nar/28.1.235&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0006">
<cite>
Bettembourg C, Diot C, Dameron O. Optimal threshold determination for interpreting semantic similarity and particularity: application to the comparison of gene sets and metabolic pathways using GO and ChEBI. PLoS One. 2015;10:e0133579.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0133579" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4521860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26230274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Optimal%20threshold%20determination%20for%20interpreting%20semantic%20similarity%20and%20particularity:%20application%20to%20the%20comparison%20of%20gene%20sets%20and%20metabolic%20pathways%20using%20GO%20and%20ChEBI&amp;author=C%20Bettembourg&amp;author=C%20Diot&amp;author=O%20Dameron&amp;volume=10&amp;publication_year=2015&amp;pages=e0133579&amp;pmid=26230274&amp;doi=10.1371/journal.pone.0133579&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0007">
<cite>
Birhanu AG. Mass spectrometry‐based proteomics as an emerging tool in clinical laboratories. Clin Proteomics. 2023;20:1–20.
</cite> [<a href="https://doi.org/10.1186/s12014-023-09424-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10464495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37633929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Proteomics&amp;title=Mass%20spectrometry%E2%80%90based%20proteomics%20as%20an%20emerging%20tool%20in%20clinical%20laboratories&amp;author=AG%20Birhanu&amp;volume=20&amp;publication_year=2023&amp;pages=1-20&amp;pmid=37633929&amp;doi=10.1186/s12014-023-09424-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0008">
<cite>
Burki F, Sandin MM, Jamy M. Diversity and ecology of protists revealed by metabarcoding. Curr Biol. 2021;31:R1267–R1280.
</cite> [<a href="https://doi.org/10.1016/j.cub.2021.07.066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34637739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Biol&amp;title=Diversity%20and%20ecology%20of%20protists%20revealed%20by%20metabarcoding&amp;author=F%20Burki&amp;author=MM%20Sandin&amp;author=M%20Jamy&amp;volume=31&amp;publication_year=2021&amp;pages=R1267-R1280&amp;pmid=34637739&amp;doi=10.1016/j.cub.2021.07.066&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0009">
<cite>
Chantzi N, Mareboina M, Konnaris MA, Montgomery A, Patsakis M, Mouratidis I, et al. The determinants of the rarity of nucleic and peptide short sequences in nature. NAR Genom Bioinform. 2024;6:lqae029.
</cite> [<a href="https://doi.org/10.1093/nargab/lqae029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10993293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38584871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NAR%20Genom%20Bioinform&amp;title=The%20determinants%20of%20the%20rarity%20of%20nucleic%20and%20peptide%20short%20sequences%20in%20nature&amp;author=N%20Chantzi&amp;author=M%20Mareboina&amp;author=MA%20Konnaris&amp;author=A%20Montgomery&amp;author=M%20Patsakis&amp;volume=6&amp;publication_year=2024&amp;pages=lqae029&amp;pmid=38584871&amp;doi=10.1093/nargab/lqae029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0010">
<cite>
Chapman JR. Protein and peptide analysis by mass spectrometry. New Jersey, United States: Humana Press; 2013.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Protein%20and%20peptide%20analysis%20by%20mass%20spectrometry&amp;author=JR%20Chapman&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0011">
<cite>
Chen C, Li Z, Huang H, Suzek BE, Wu CH, UniProt Consortium
. A fast peptide match service for UniProt knowledgebase. Bioinformatics. 2013;29:2808–2809.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt484" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3799477/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23958731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20fast%20peptide%20match%20service%20for%20UniProt%20knowledgebase&amp;author=C%20Chen&amp;author=Z%20Li&amp;author=H%20Huang&amp;author=BE%20Suzek&amp;author=CH%20Wu&amp;volume=29&amp;publication_year=2013&amp;pages=2808-2809&amp;pmid=23958731&amp;doi=10.1093/bioinformatics/btt484&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0012">
<cite>
Cheng J, Novati G, Pan J, Bycroft C, Žemgulytė A, Applebaum T, et al. Accurate proteome‐wide missense variant effect prediction with AlphaMissense. Science. 2023; 381(6664):1284–1295. 10.1126/science.adg7492

</cite> [<a href="https://doi.org/10.1126/science.adg7492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37733863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Accurate%20proteome%E2%80%90wide%20missense%20variant%20effect%20prediction%20with%20AlphaMissense&amp;author=J%20Cheng&amp;author=G%20Novati&amp;author=J%20Pan&amp;author=C%20Bycroft&amp;author=A%20%C5%BDemgulyt%C4%97&amp;volume=381&amp;issue=6664&amp;publication_year=2023&amp;pages=1284-1295&amp;pmid=37733863&amp;doi=10.1126/science.adg7492&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0013">
<cite>
Cheng N, Brantley DM, Chen J. The ephrins and Eph receptors in angiogenesis. Cytokine Growth Factor Rev. 2002;13:75–85.
</cite> [<a href="https://doi.org/10.1016/s1359-6101(01)00031-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11750881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine%20Growth%20Factor%20Rev&amp;title=The%20ephrins%20and%20Eph%20receptors%20in%20angiogenesis&amp;author=N%20Cheng&amp;author=DM%20Brantley&amp;author=J%20Chen&amp;volume=13&amp;publication_year=2002&amp;pages=75-85&amp;pmid=11750881&amp;doi=10.1016/s1359-6101(01)00031-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0014">
<cite>
Dey D, Dhar D, Das S, Maulik A, Basu S. Structural and functional implications of leucine‐rich repeats in toll‐like receptor1 subfamily. J Biosci. 2022; 47:59: Available from: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36222137" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/pubmed/36222137</a>

</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/36222137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biosci&amp;title=Structural%20and%20functional%20implications%20of%20leucine%E2%80%90rich%20repeats%20in%20toll%E2%80%90like%20receptor1%20subfamily&amp;author=D%20Dey&amp;author=D%20Dhar&amp;author=S%20Das&amp;author=A%20Maulik&amp;author=S%20Basu&amp;volume=47&amp;publication_year=2022&amp;pages=59&amp;pmid=36222137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0015">
<cite>
Duda DM, Olszewski JL, Schuermann JP, Kurinov I, Miller DJ, Nourse A, et al. Structure of HHARI, a RING‐IBR‐RING ubiquitin ligase: autoinhibition of an Ariadne‐family E3 and insights into ligation mechanism. Structure. 2013;21:1030–1041.
</cite> [<a href="https://doi.org/10.1016/j.str.2013.04.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3747818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23707686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Structure%20of%20HHARI,%20a%20RING%E2%80%90IBR%E2%80%90RING%20ubiquitin%20ligase:%20autoinhibition%20of%20an%20Ariadne%E2%80%90family%20E3%20and%20insights%20into%20ligation%20mechanism&amp;author=DM%20Duda&amp;author=JL%20Olszewski&amp;author=JP%20Schuermann&amp;author=I%20Kurinov&amp;author=DJ%20Miller&amp;volume=21&amp;publication_year=2013&amp;pages=1030-1041&amp;pmid=23707686&amp;doi=10.1016/j.str.2013.04.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0016">
<cite>
Dupree EJ, Jayathirtha M, Yorkey H, Mihasan M, Petre BA, Darie CC. A critical review of bottom‐up proteomics: the good, the bad, and the future of this field. Proteomes. 2020;8:14. 10.3390/proteomes8030014

</cite> [<a href="https://doi.org/10.3390/proteomes8030014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7564415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32640657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomes&amp;title=A%20critical%20review%20of%20bottom%E2%80%90up%20proteomics:%20the%20good,%20the%20bad,%20and%20the%20future%20of%20this%20field&amp;author=EJ%20Dupree&amp;author=M%20Jayathirtha&amp;author=H%20Yorkey&amp;author=M%20Mihasan&amp;author=BA%20Petre&amp;volume=8&amp;publication_year=2020&amp;pages=14&amp;pmid=32640657&amp;doi=10.3390/proteomes8030014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0017">
<cite>
Frey PA, Hegeman AD, Ruzicka FJ. The radical SAM superfamily. Crit Rev Biochem Mol Biol. 2008;43:63–88.
</cite> [<a href="https://doi.org/10.1080/10409230701829169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18307109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Biochem%20Mol%20Biol&amp;title=The%20radical%20SAM%20superfamily&amp;author=PA%20Frey&amp;author=AD%20Hegeman&amp;author=FJ%20Ruzicka&amp;volume=43&amp;publication_year=2008&amp;pages=63-88&amp;pmid=18307109&amp;doi=10.1080/10409230701829169&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0018">
<cite>
Frishman D, Argos P. Knowledge‐based protein secondary structure assignment. Proteins. 1995;23:566–579.
</cite> [<a href="https://doi.org/10.1002/prot.340230412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8749853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Knowledge%E2%80%90based%20protein%20secondary%20structure%20assignment&amp;author=D%20Frishman&amp;author=P%20Argos&amp;volume=23&amp;publication_year=1995&amp;pages=566-579&amp;pmid=8749853&amp;doi=10.1002/prot.340230412&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0019">
<cite>
Gene Ontology Consortium
, Aleksander SA, Balhoff J, Carbon S, Cherry JM, Drabkin HJ, et al. The gene ontology knowledgebase in 2023. Genetics. 2023;224(1):iyad031. 10.1093/genetics/iyad031

</cite> [<a href="https://doi.org/10.1093/genetics/iyad031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10158837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36866529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The%20gene%20ontology%20knowledgebase%20in%202023&amp;author=SA%20Aleksander&amp;author=J%20Balhoff&amp;author=S%20Carbon&amp;author=JM%20Cherry&amp;author=HJ%20Drabkin&amp;volume=224&amp;issue=1&amp;publication_year=2023&amp;pages=iyad031&amp;pmid=36866529&amp;doi=10.1093/genetics/iyad031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0020">
<cite>
Georgakopoulos‐Soares I, Yizhar‐Barnea O, Mouratidis I, Hemberg M, Ahituv N. Absent from DNA and protein: genomic characterization of nullomers and nullpeptides across functional categories and evolution. Genome Biol. 2021;22:245.
</cite> [<a href="https://doi.org/10.1186/s13059-021-02459-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8386077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34433494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Absent%20from%20DNA%20and%20protein:%20genomic%20characterization%20of%20nullomers%20and%20nullpeptides%20across%20functional%20categories%20and%20evolution&amp;author=I%20Georgakopoulos%E2%80%90Soares&amp;author=O%20Yizhar%E2%80%90Barnea&amp;author=I%20Mouratidis&amp;author=M%20Hemberg&amp;author=N%20Ahituv&amp;volume=22&amp;publication_year=2021&amp;pages=245&amp;pmid=34433494&amp;doi=10.1186/s13059-021-02459-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0021">
<cite>
Hajisharifi Z, Piryaiee M, Mohammad Beigi M, Behbahani M, Mohabatkar H. Predicting anticancer peptides with Chou's pseudo amino acid composition and investigating their mutagenicity via Ames test. J Theor Biol. 2014;341:34–40.
</cite> [<a href="https://doi.org/10.1016/j.jtbi.2013.08.037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24035842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Theor%20Biol&amp;title=Predicting%20anticancer%20peptides%20with%20Chou's%20pseudo%20amino%20acid%20composition%20and%20investigating%20their%20mutagenicity%20via%20Ames%20test&amp;author=Z%20Hajisharifi&amp;author=M%20Piryaiee&amp;author=M%20Mohammad%20Beigi&amp;author=M%20Behbahani&amp;author=H%20Mohabatkar&amp;volume=341&amp;publication_year=2014&amp;pages=34-40&amp;pmid=24035842&amp;doi=10.1016/j.jtbi.2013.08.037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0022">
<cite>
Henikoff S, Henikoff JG. Amino acid substitution matrices from protein blocks. Proc Natl Acad Sci USA. 1992;89:10915–10919.
</cite> [<a href="https://doi.org/10.1073/pnas.89.22.10915" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC50453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1438297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Amino%20acid%20substitution%20matrices%20from%20protein%20blocks&amp;author=S%20Henikoff&amp;author=JG%20Henikoff&amp;volume=89&amp;publication_year=1992&amp;pages=10915-10919&amp;pmid=1438297&amp;doi=10.1073/pnas.89.22.10915&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0023">
<cite>
Huerta‐Cepas J, Szklarczyk D, Heller D, Hernández‐Plaza A, Forslund SK, Cook H, et al. eggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res. 2019;47:D309–D314.
</cite> [<a href="https://doi.org/10.1093/nar/gky1085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30418610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=eggNOG%205.0:%20a%20hierarchical,%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses&amp;author=J%20Huerta%E2%80%90Cepas&amp;author=D%20Szklarczyk&amp;author=D%20Heller&amp;author=A%20Hern%C3%A1ndez%E2%80%90Plaza&amp;author=SK%20Forslund&amp;volume=47&amp;publication_year=2019&amp;pages=D309-D314&amp;pmid=30418610&amp;doi=10.1093/nar/gky1085&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0024">
<cite>
Jones MDM, Forn I, Gadelha C, Egan MJ, Bass D, Massana R, et al. Discovery of novel intermediate forms redefines the fungal tree of life. Nature. 2011;474:200–203.
</cite> [<a href="https://doi.org/10.1038/nature09984" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21562490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Discovery%20of%20novel%20intermediate%20forms%20redefines%20the%20fungal%20tree%20of%20life&amp;author=MDM%20Jones&amp;author=I%20Forn&amp;author=C%20Gadelha&amp;author=MJ%20Egan&amp;author=D%20Bass&amp;volume=474&amp;publication_year=2011&amp;pages=200-203&amp;pmid=21562490&amp;doi=10.1038/nature09984&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0025">
<cite>
Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure prediction with AlphaFold. Nature. 2021;596:583–589.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;author=J%20Jumper&amp;author=R%20Evans&amp;author=A%20Pritzel&amp;author=T%20Green&amp;author=M%20Figurnov&amp;volume=596&amp;publication_year=2021&amp;pages=583-589&amp;pmid=34265844&amp;doi=10.1038/s41586-021-03819-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0026">
<cite>
Kamke J, Rinke C, Schwientek P, Mavromatis K, Ivanova N, Sczyrba A, et al. The candidate phylum Poribacteria by single‐cell genomics: new insights into phylogeny, cell‐compartmentation, eukaryote‐like repeat proteins, and other genomic features. PLoS One. 2014;9:e87353.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0087353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3909097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24498082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=The%20candidate%20phylum%20Poribacteria%20by%20single%E2%80%90cell%20genomics:%20new%20insights%20into%20phylogeny,%20cell%E2%80%90compartmentation,%20eukaryote%E2%80%90like%20repeat%20proteins,%20and%20other%20genomic%20features&amp;author=J%20Kamke&amp;author=C%20Rinke&amp;author=P%20Schwientek&amp;author=K%20Mavromatis&amp;author=N%20Ivanova&amp;volume=9&amp;publication_year=2014&amp;pages=e87353&amp;pmid=24498082&amp;doi=10.1371/journal.pone.0087353&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0027">
<cite>
Kao T‐J, Kania A. Ephrin‐mediated cis‐attenuation of Eph receptor signaling is essential for spinal motor axon guidance. Neuron. 2011;71:76–91.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2011.05.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21745639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Ephrin%E2%80%90mediated%20cis%E2%80%90attenuation%20of%20Eph%20receptor%20signaling%20is%20essential%20for%20spinal%20motor%20axon%20guidance&amp;author=T%E2%80%90J%20Kao&amp;author=A%20Kania&amp;volume=71&amp;publication_year=2011&amp;pages=76-91&amp;pmid=21745639&amp;doi=10.1016/j.neuron.2011.05.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0028">
<cite>
Katoh K, Standley DM. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol Biol Evol. 2013;30:772–780.
</cite> [<a href="https://doi.org/10.1093/molbev/mst010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3603318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23329690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=MAFFT%20multiple%20sequence%20alignment%20software%20version%207:%20improvements%20in%20performance%20and%20usability&amp;author=K%20Katoh&amp;author=DM%20Standley&amp;volume=30&amp;publication_year=2013&amp;pages=772-780&amp;pmid=23329690&amp;doi=10.1093/molbev/mst010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0029">
<cite>
Klopfenstein DV, Zhang L, Pedersen BS, Ramírez F, Warwick Vesztrocy A, Naldi A, et al. GOATOOLS: a python library for gene ontology analyses. Sci Rep. 2018;8:10872.
</cite> [<a href="https://doi.org/10.1038/s41598-018-28948-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6052049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30022098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=GOATOOLS:%20a%20python%20library%20for%20gene%20ontology%20analyses&amp;author=DV%20Klopfenstein&amp;author=L%20Zhang&amp;author=BS%20Pedersen&amp;author=F%20Ram%C3%ADrez&amp;author=A%20Warwick%20Vesztrocy&amp;volume=8&amp;publication_year=2018&amp;pages=10872&amp;pmid=30022098&amp;doi=10.1038/s41598-018-28948-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0030">
<cite>
Krasowski MD, Yasuda K, Hagey LR, Schuetz EG. Evolutionary selection across the nuclear hormone receptor superfamily with a focus on the NR1I subfamily (vitamin D, pregnane X, and constitutive androstane receptors). Nucl Recept. 2005;3:2.
</cite> [<a href="https://doi.org/10.1186/1478-1336-3-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1262763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16197547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucl%20Recept&amp;title=Evolutionary%20selection%20across%20the%20nuclear%20hormone%20receptor%20superfamily%20with%20a%20focus%20on%20the%20NR1I%20subfamily%20(vitamin%20D,%20pregnane%20X,%20and%20constitutive%20androstane%20receptors)&amp;author=MD%20Krasowski&amp;author=K%20Yasuda&amp;author=LR%20Hagey&amp;author=EG%20Schuetz&amp;volume=3&amp;publication_year=2005&amp;pages=2&amp;pmid=16197547&amp;doi=10.1186/1478-1336-3-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0031">
<cite>
Kullander K, Mather NK, Diella F, Dottori M, Boyd AW, Klein R. Kinase‐dependent and kinase‐independent functions of EphA4 receptors in major axon tract formation in vivo. Neuron. 2001;29:73–84.
</cite> [<a href="https://doi.org/10.1016/s0896-6273(01)00181-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11182082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Kinase%E2%80%90dependent%20and%20kinase%E2%80%90independent%20functions%20of%20EphA4%20receptors%20in%20major%20axon%20tract%20formation%20in%20vivo&amp;author=K%20Kullander&amp;author=NK%20Mather&amp;author=F%20Diella&amp;author=M%20Dottori&amp;author=AW%20Boyd&amp;volume=29&amp;publication_year=2001&amp;pages=73-84&amp;pmid=11182082&amp;doi=10.1016/s0896-6273(01)00181-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0032">
<cite>
Lacerda CMR, Reardon KF. Environmental proteomics: applications of proteome profiling in environmental microbiology and biotechnology. Brief Funct Genomic Proteomic. 2009;8:75–87.
</cite> [<a href="https://doi.org/10.1093/bfgp/elp005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19279070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brief%20Funct%20Genomic%20Proteomic&amp;title=Environmental%20proteomics:%20applications%20of%20proteome%20profiling%20in%20environmental%20microbiology%20and%20biotechnology&amp;author=CMR%20Lacerda&amp;author=KF%20Reardon&amp;volume=8&amp;publication_year=2009&amp;pages=75-87&amp;pmid=19279070&amp;doi=10.1093/bfgp/elp005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0033">
<cite>
Li L, Wang T, Ning Z, Zhang X, Butcher J, Serrana JM, et al. Revealing proteome‐level functional redundancy in the human gut microbiome using ultra‐deep metaproteomics. Nat Commun. 2023;14:3428.
</cite> [<a href="https://doi.org/10.1038/s41467-023-39149-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10257714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37301875/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Revealing%20proteome%E2%80%90level%20functional%20redundancy%20in%20the%20human%20gut%20microbiome%20using%20ultra%E2%80%90deep%20metaproteomics&amp;author=L%20Li&amp;author=T%20Wang&amp;author=Z%20Ning&amp;author=X%20Zhang&amp;author=J%20Butcher&amp;volume=14&amp;publication_year=2023&amp;pages=3428&amp;pmid=37301875&amp;doi=10.1038/s41467-023-39149-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0034">
<cite>
Markson JS, O'Shea EK. The molecular clockwork of a protein‐based circadian oscillator. FEBS Lett. 2009;583:3938–3947.
</cite> [<a href="https://doi.org/10.1016/j.febslet.2009.11.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2810098/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19913541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&amp;title=The%20molecular%20clockwork%20of%20a%20protein%E2%80%90based%20circadian%20oscillator&amp;author=JS%20Markson&amp;author=EK%20O'Shea&amp;volume=583&amp;publication_year=2009&amp;pages=3938-3947&amp;pmid=19913541&amp;doi=10.1016/j.febslet.2009.11.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0035">
<cite>
Matsushima N, Miyashita H, Kretsinger RH. Sequence features, structure, ligand interaction, and diseases in small leucine rich repeat proteoglycans. J Cell Commun Signal. 2021;15:519–531.
</cite> [<a href="https://doi.org/10.1007/s12079-021-00616-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8642563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33860400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Commun%20Signal&amp;title=Sequence%20features,%20structure,%20ligand%20interaction,%20and%20diseases%20in%20small%20leucine%20rich%20repeat%20proteoglycans&amp;author=N%20Matsushima&amp;author=H%20Miyashita&amp;author=RH%20Kretsinger&amp;volume=15&amp;publication_year=2021&amp;pages=519-531&amp;pmid=33860400&amp;doi=10.1007/s12079-021-00616-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0036"><cite>
McInnes L, Healy J, Melville J. UMAP: uniform manifold approximation and projection for dimension reduction. 2018.  Available from: <a href="http://arxiv.org/abs/1802.03426" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://arxiv.org/abs/1802.03426</a>
</cite></li>
<li id="pro70241-bib-0037">
<cite>
McMahon HT, Boucrot E. Molecular mechanism and physiological functions of clathrin‐mediated endocytosis. Nat Rev Mol Cell Biol. 2011;12:517–533.
</cite> [<a href="https://doi.org/10.1038/nrm3151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21779028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;title=Molecular%20mechanism%20and%20physiological%20functions%20of%20clathrin%E2%80%90mediated%20endocytosis&amp;author=HT%20McMahon&amp;author=E%20Boucrot&amp;volume=12&amp;publication_year=2011&amp;pages=517-533&amp;pmid=21779028&amp;doi=10.1038/nrm3151&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0038">
<cite>
Meng EC, Goddard TD, Pettersen EF, Couch GS, Pearson ZJ, Morris JH, et al. UCSF ChimeraX: tools for structure building and analysis. Protein Sci. 2023;32:e4792.
</cite> [<a href="https://doi.org/10.1002/pro.4792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10588335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37774136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=UCSF%20ChimeraX:%20tools%20for%20structure%20building%20and%20analysis&amp;author=EC%20Meng&amp;author=TD%20Goddard&amp;author=EF%20Pettersen&amp;author=GS%20Couch&amp;author=ZJ%20Pearson&amp;volume=32&amp;publication_year=2023&amp;pages=e4792&amp;pmid=37774136&amp;doi=10.1002/pro.4792&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0039">
<cite>
Moeckel C, Mareboina M, Konnaris MA, Chan CSY, Mouratidis I, Montgomery A, et al. A survey of k‐mer methods and applications in bioinformatics. Comput Struct Biotechnol J. 2024;23:2289–2303.
</cite> [<a href="https://doi.org/10.1016/j.csbj.2024.05.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11152613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38840832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Struct%20Biotechnol%20J&amp;title=A%20survey%20of%20k%E2%80%90mer%20methods%20and%20applications%20in%20bioinformatics&amp;author=C%20Moeckel&amp;author=M%20Mareboina&amp;author=MA%20Konnaris&amp;author=CSY%20Chan&amp;author=I%20Mouratidis&amp;volume=23&amp;publication_year=2024&amp;pages=2289-2303&amp;pmid=38840832&amp;doi=10.1016/j.csbj.2024.05.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0040">
<cite>
Mouratidis I, Baltoumas FA, Chantzi N, Patsakis M, Chan CSY, Montgomery A, et al. kmerDB: a database encompassing the set of genomic and proteomic sequence information for each species. Comput Struct Biotechnol J. 2024;23:1919–1928.
</cite> [<a href="https://doi.org/10.1016/j.csbj.2024.04.050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11070822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38711760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Struct%20Biotechnol%20J&amp;title=kmerDB:%20a%20database%20encompassing%20the%20set%20of%20genomic%20and%20proteomic%20sequence%20information%20for%20each%20species&amp;author=I%20Mouratidis&amp;author=FA%20Baltoumas&amp;author=N%20Chantzi&amp;author=M%20Patsakis&amp;author=CSY%20Chan&amp;volume=23&amp;publication_year=2024&amp;pages=1919-1928&amp;pmid=38711760&amp;doi=10.1016/j.csbj.2024.04.050&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0041">
<cite>
Mouratidis I, Chan CSY, Chantzi N, Tsiatsianis GC, Hemberg M, Ahituv N, et al. Quasi‐prime peptides: identification of the shortest peptide sequences unique to a species. NAR Genom Bioinform. 2023;5:lqad039.
</cite> [<a href="https://doi.org/10.1093/nargab/lqad039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10124967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37101657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NAR%20Genom%20Bioinform&amp;title=Quasi%E2%80%90prime%20peptides:%20identification%20of%20the%20shortest%20peptide%20sequences%20unique%20to%20a%20species&amp;author=I%20Mouratidis&amp;author=CSY%20Chan&amp;author=N%20Chantzi&amp;author=GC%20Tsiatsianis&amp;author=M%20Hemberg&amp;volume=5&amp;publication_year=2023&amp;pages=lqad039&amp;pmid=37101657&amp;doi=10.1093/nargab/lqad039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0042">
<cite>
Mycroft‐West CJ, Abdelkarim S, Duyvesteyn HME, Gandhi NS, Skidmore MA, Owens RJ, et al. Structural and mechanistic characterization of bifunctional heparan sulfate N‐deacetylase‐N‐sulfotransferase 1. Nat Commun. 2024;15:1–17.
</cite> [<a href="https://doi.org/10.1038/s41467-024-45419-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10864358/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38351061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Structural%20and%20mechanistic%20characterization%20of%20bifunctional%20heparan%20sulfate%20N%E2%80%90deacetylase%E2%80%90N%E2%80%90sulfotransferase%201&amp;author=CJ%20Mycroft%E2%80%90West&amp;author=S%20Abdelkarim&amp;author=HME%20Duyvesteyn&amp;author=NS%20Gandhi&amp;author=MA%20Skidmore&amp;volume=15&amp;publication_year=2024&amp;pages=1-17&amp;pmid=38351061&amp;doi=10.1038/s41467-024-45419-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0043">
<cite>
Ohama T. The multiple functions of protein phosphatase 6. Biochim Biophys Acta Mol Cell Res. 2019;1866:74–82.
</cite> [<a href="https://doi.org/10.1016/j.bbamcr.2018.07.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30036567/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta%20Mol%20Cell%20Res&amp;title=The%20multiple%20functions%20of%20protein%20phosphatase%206&amp;author=T%20Ohama&amp;volume=1866&amp;publication_year=2019&amp;pages=74-82&amp;pmid=30036567&amp;doi=10.1016/j.bbamcr.2018.07.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0044">
<cite>
Pao SS, Paulsen IT, Saier MH Jr. Major facilitator superfamily. Microbiol Mol Biol Rev. 1998;62:1–34.
</cite> [<a href="https://doi.org/10.1128/mmbr.62.1.1-34.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC98904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9529885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Mol%20Biol%20Rev&amp;title=Major%20facilitator%20superfamily&amp;author=SS%20Pao&amp;author=IT%20Paulsen&amp;author=MH%20Saier&amp;volume=62&amp;publication_year=1998&amp;pages=1-34&amp;pmid=9529885&amp;doi=10.1128/mmbr.62.1.1-34.1998&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0045">
<cite>
Paysan‐Lafosse T, Blum M, Chuguransky S, Grego T, Pinto BL, Salazar GA, et al. InterPro in 2022. Nucleic Acids Res
<a href="https://pubmed.ncbi.nlm.nih.gov/36350672/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubmed.ncbi.nlm.nih.gov/36350672/</a>. 2023;51:D418–D427.
</cite> [<a href="https://doi.org/10.1093/nar/gkac993" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36350672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=InterPro%20in%202022&amp;author=T%20Paysan%E2%80%90Lafosse&amp;author=M%20Blum&amp;author=S%20Chuguransky&amp;author=T%20Grego&amp;author=BL%20Pinto&amp;volume=51&amp;publication_year=2023&amp;pages=D418-D427&amp;pmid=36350672&amp;doi=10.1093/nar/gkac993&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0046">
<cite>
Rayment I, Rypniewski WR, Schmidt‐Bäse K, Smith R, Tomchick DR, Benning MM, et al. Three‐dimensional structure of myosin subfragment‐1: a molecular motor. Science. 1993;261:50–58.
</cite> [<a href="https://doi.org/10.1126/science.8316857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8316857/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Three%E2%80%90dimensional%20structure%20of%20myosin%20subfragment%E2%80%901:%20a%20molecular%20motor&amp;author=I%20Rayment&amp;author=WR%20Rypniewski&amp;author=K%20Schmidt%E2%80%90B%C3%A4se&amp;author=R%20Smith&amp;author=DR%20Tomchick&amp;volume=261&amp;publication_year=1993&amp;pages=50-58&amp;pmid=8316857&amp;doi=10.1126/science.8316857&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0047">
<cite>
Rigoutsos I, Floratos A, Ouzounis C, Gao Y, Parida L. Dictionary building via unsupervised hierarchical motif discovery in the sequence space of natural proteins. Proteins. 1999;37:264–277. Available from: <a href="https://pubmed.ncbi.nlm.nih.gov/10584071/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubmed.ncbi.nlm.nih.gov/10584071/</a>

</cite> [<a href="https://doi.org/10.1002/(sici)1097-0134(19991101)37:2&lt;264::aid-prot11&gt;3.0.co;2-c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10584071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Dictionary%20building%20via%20unsupervised%20hierarchical%20motif%20discovery%20in%20the%20sequence%20space%20of%20natural%20proteins&amp;author=I%20Rigoutsos&amp;author=A%20Floratos&amp;author=C%20Ouzounis&amp;author=Y%20Gao&amp;author=L%20Parida&amp;volume=37&amp;publication_year=1999&amp;pages=264-277&amp;pmid=10584071&amp;doi=10.1002/(sici)1097-0134(19991101)37:2&lt;264::aid-prot11&gt;3.0.co;2-c&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0048">
<cite>
Sarrazin S, Lamanna WC, Esko JD. Heparan sulfate proteoglycans. Cold Spring Harb Perspect Biol. 2011;3:a004952. 10.1101/cshperspect.a004952

</cite> [<a href="https://doi.org/10.1101/cshperspect.a004952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3119907/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21690215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb%20Perspect%20Biol&amp;title=Heparan%20sulfate%20proteoglycans&amp;author=S%20Sarrazin&amp;author=WC%20Lamanna&amp;author=JD%20Esko&amp;volume=3&amp;publication_year=2011&amp;pages=a004952&amp;pmid=21690215&amp;doi=10.1101/cshperspect.a004952&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0049">
<cite>
Smith RF, Smith TF. Automatic generation of primary sequence patterns from sets of related protein sequences. Proc Natl Acad Sci USA. 1990;87:118–122.
</cite> [<a href="https://doi.org/10.1073/pnas.87.1.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC53211/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2296575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Automatic%20generation%20of%20primary%20sequence%20patterns%20from%20sets%20of%20related%20protein%20sequences&amp;author=RF%20Smith&amp;author=TF%20Smith&amp;volume=87&amp;publication_year=1990&amp;pages=118-122&amp;pmid=2296575&amp;doi=10.1073/pnas.87.1.118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0050">
<cite>
Steenwyk JL, Buida TJ 3rd, Li Y, Shen X‐X, Rokas A. ClipKIT: a multiple sequence alignment trimming software for accurate phylogenomic inference. PLoS Biol. 2020;18:e3001007.
</cite> [<a href="https://doi.org/10.1371/journal.pbio.3001007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7735675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33264284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol&amp;title=ClipKIT:%20a%20multiple%20sequence%20alignment%20trimming%20software%20for%20accurate%20phylogenomic%20inference&amp;author=JL%20Steenwyk&amp;author=TJ%20Buida&amp;author=Y%20Li&amp;author=X%E2%80%90X%20Shen&amp;author=A%20Rokas&amp;volume=18&amp;publication_year=2020&amp;pages=e3001007&amp;pmid=33264284&amp;doi=10.1371/journal.pbio.3001007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0051">
<cite>
UniProt Consortium
. UniProt: the universal protein knowledgebase in 2023. Nucleic Acids Res. 2023;51:D523–D531.
</cite> [<a href="https://doi.org/10.1093/nar/gkac1052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36408920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=UniProt:%20the%20universal%20protein%20knowledgebase%20in%202023&amp;volume=51&amp;publication_year=2023&amp;pages=D523-D531&amp;pmid=36408920&amp;doi=10.1093/nar/gkac1052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0052">
<cite>
ValizadehAslani T, Zhao Z, Sokhansanj BA, Rosen GL. Amino acid ‐mer feature extraction for quantitative antimicrobial resistance (AMR) prediction by machine learning and model interpretation for biological insights. Biology. 2020;9:365. 10.3390/biology9110365

</cite> [<a href="https://doi.org/10.3390/biology9110365" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7694136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33126516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biology&amp;title=Amino%20acid%20%E2%80%90mer%20feature%20extraction%20for%20quantitative%20antimicrobial%20resistance%20(AMR)%20prediction%20by%20machine%20learning%20and%20model%20interpretation%20for%20biological%20insights&amp;author=T%20ValizadehAslani&amp;author=Z%20Zhao&amp;author=BA%20Sokhansanj&amp;author=GL%20Rosen&amp;volume=9&amp;publication_year=2020&amp;pages=365&amp;pmid=33126516&amp;doi=10.3390/biology9110365&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0053">
<cite>
Varadi M, Bertoni D, Magana P, Paramval U, Pidruchna I, Radhakrishnan M, et al. AlphaFold protein structure database in 2024: providing structure coverage for over 214 million protein sequences. Nucleic Acids Res. 2024;52:D368–D375.
</cite> [<a href="https://doi.org/10.1093/nar/gkad1011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37933859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=AlphaFold%20protein%20structure%20database%20in%202024:%20providing%20structure%20coverage%20for%20over%20214%20million%20protein%20sequences&amp;author=M%20Varadi&amp;author=D%20Bertoni&amp;author=P%20Magana&amp;author=U%20Paramval&amp;author=I%20Pidruchna&amp;volume=52&amp;publication_year=2024&amp;pages=D368-D375&amp;pmid=37933859&amp;doi=10.1093/nar/gkad1011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0054">
<cite>
Wang JZ, Du Z, Payattakool R, Yu PS, Chen C‐F. A new method to measure the semantic similarity of GO terms. Bioinformatics. 2007;23:1274–1281.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17344234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20new%20method%20to%20measure%20the%20semantic%20similarity%20of%20GO%20terms&amp;author=JZ%20Wang&amp;author=Z%20Du&amp;author=R%20Payattakool&amp;author=PS%20Yu&amp;author=C%E2%80%90F%20Chen&amp;volume=23&amp;publication_year=2007&amp;pages=1274-1281&amp;pmid=17344234&amp;doi=10.1093/bioinformatics/btm087&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0055">
<cite>
Waterhouse AM, Procter JB, Martin DMA, Clamp M, Barton GJ. Jalview version 2—a multiple sequence alignment editor and analysis workbench. Bioinformatics. 2009;25:1189–1191.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2672624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19151095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Jalview%20version%202%E2%80%94a%20multiple%20sequence%20alignment%20editor%20and%20analysis%20workbench&amp;author=AM%20Waterhouse&amp;author=JB%20Procter&amp;author=DMA%20Martin&amp;author=M%20Clamp&amp;author=GJ%20Barton&amp;volume=25&amp;publication_year=2009&amp;pages=1189-1191&amp;pmid=19151095&amp;doi=10.1093/bioinformatics/btp033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0056">
<cite>
Wen J, Chan RHF, Yau S‐C, He RL, Yau SST. K‐mer natural vector and its application to the phylogenetic analysis of genetic sequences. Gene. 2014;546:25–34.
</cite> [<a href="https://doi.org/10.1016/j.gene.2014.05.043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4096558/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24858075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=K%E2%80%90mer%20natural%20vector%20and%20its%20application%20to%20the%20phylogenetic%20analysis%20of%20genetic%20sequences&amp;author=J%20Wen&amp;author=RHF%20Chan&amp;author=S%E2%80%90C%20Yau&amp;author=RL%20He&amp;author=SST%20Yau&amp;volume=546&amp;publication_year=2014&amp;pages=25-34&amp;pmid=24858075&amp;doi=10.1016/j.gene.2014.05.043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0057">
<cite>
Wild D. The immunoassay handbook: theory and applications of ligand binding, ELISA and Related techniques. Oxford, United Kingdom: Newnes; 2013.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20immunoassay%20handbook:%20theory%20and%20applications%20of%20ligand%20binding,%20ELISA%20and%20Related%20techniques&amp;author=D%20Wild&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0058">
<cite>
Wu C, Gao R, Zhang Y, De Marinis Y. PTPD: predicting therapeutic peptides by deep learning and word2vec. BMC Bioinform. 2019;20:456.</cite> [<a href="https://doi.org/10.1186/s12859-019-3006-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6728961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31492094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinform&amp;title=PTPD:%20predicting%20therapeutic%20peptides%20by%20deep%20learning%20and%20word2vec&amp;author=C%20Wu&amp;author=R%20Gao&amp;author=Y%20Zhang&amp;author=Y%20De%20Marinis&amp;volume=20&amp;publication_year=2019&amp;pages=456&amp;pmid=31492094&amp;doi=10.1186/s12859-019-3006-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70241-bib-0059">
<cite>
Yang Q, Zhao J, Chen D, Wang Y. E3 ubiquitin ligases: styles, structures and functions. Mol Biomed. 2021;2:23.
</cite> [<a href="https://doi.org/10.1186/s43556-021-00043-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8607428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35006464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biomed&amp;title=E3%20ubiquitin%20ligases:%20styles,%20structures%20and%20functions&amp;author=Q%20Yang&amp;author=J%20Zhao&amp;author=D%20Chen&amp;author=Y%20Wang&amp;volume=2&amp;publication_year=2021&amp;pages=23&amp;pmid=35006464&amp;doi=10.1186/s43556-021-00043-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
Supplementary File 1:
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375989/bin/PRO-34-e70241-s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70241-s004.xlsx</a><sup> (19.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
Supplementary File 2:
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375989/bin/PRO-34-e70241-s003.csv" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70241-s003.csv</a><sup> (149.5KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
Supplementary File 3:
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375989/bin/PRO-34-e70241-s001.csv" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70241-s001.csv</a><sup> (278.3KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>
<strong>Data S1:</strong> Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375989/bin/PRO-34-e70241-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70241-s002.docx</a><sup> (5.3MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are openly available in Taxonomic quasi‐prime raw data at <a href="https://zenodo.org/records/14385095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/records/14385095</a>. Code used for this study is openly available at <a href="https://github.com/Georgakopoulos-Soares-lab/taxonomic_quasi_primes" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Georgakopoulos-Soares-lab/taxonomic_quasi_primes</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Protein Science : A Publication of the Protein Society are provided here courtesy of <strong>The Protein Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/pro.70241"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/PRO-34-e70241.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375989/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375989/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375989%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375989/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375989/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375989/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375989/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375989/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375989/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Sm74bnAplHYSfYrq7Rgm2AzQC49pmMNH2wE1Y8b4AdXeFOskv86WQJx2Gyi0k3Z5">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
