
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Single−cell transcriptomic landscape of sciatic nerve after transection injury - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251C6F8AEEFAC3051C6F00616708FA.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="jneuro">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374453/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal of Neuroinflammation">
<meta name="citation_title" content="Single−cell transcriptomic landscape of sciatic nerve after transection injury">
<meta name="citation_author" content="Yiben Ouyang">
<meta name="citation_author_institution" content="School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Mingqian Yu">
<meta name="citation_author_institution" content="School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China">
<meta name="citation_author" content="Tieyuan Zhang">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author_institution" content="Cheeloo College of Medicine, Shandong University, 44 West Wenhua Road, Lixia District, Jinan, 250012 Shandong P.R. China">
<meta name="citation_author" content="Haofeng Cheng">
<meta name="citation_author_institution" content="School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Liang Zuo">
<meta name="citation_author_institution" content="School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Haolin Liu">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Yanjun Guan">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Ao Liu">
<meta name="citation_author_institution" content="School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Jiajie Chen">
<meta name="citation_author_institution" content="School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Ruichao He">
<meta name="citation_author_institution" content="School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Sice Wang">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Tianqi Su">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Yixiao Tan">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Yuhui Cu">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Xiaochun Zhang">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Xiaoyang Fu">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Junli Wang">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author_institution" content="Jinzhou Medical University, No. 40, Songpo Road, San Duan, Linghe District, Jinzhou, Liaoning China">
<meta name="citation_author" content="Jinjuan Zhao">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author" content="Jiang Peng">
<meta name="citation_author_institution" content="School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author_institution" content="Co−innovation Center of Neuroregeneration, Nantong University, Nantong, 226007 Jiangsu Province PR China">
<meta name="citation_author" content="Yu Wang">
<meta name="citation_author_institution" content="School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China">
<meta name="citation_author_institution" content="Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China">
<meta name="citation_author_institution" content="Co−innovation Center of Neuroregeneration, Nantong University, Nantong, 226007 Jiangsu Province PR China">
<meta name="citation_publication_date" content="2025 Aug 23">
<meta name="citation_volume" content="22">
<meta name="citation_firstpage" content="205">
<meta name="citation_doi" content="10.1186/s12974-025-03514-3">
<meta name="citation_pmid" content="40849669">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374453/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374453/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374453/pdf/12974_2025_Article_3514.pdf">
<meta name="description" content="Peripheral nerve injuries, particularly those affecting the sciatic nerve, often result in incomplete functional recovery due to the limited regenerative capacity of adult peripheral nerves. To elucidate the cellular and molecular mechanisms ...">
<meta name="og:title" content="Single−cell transcriptomic landscape of sciatic nerve after transection injury">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Peripheral nerve injuries, particularly those affecting the sciatic nerve, often result in incomplete functional recovery due to the limited regenerative capacity of adult peripheral nerves. To elucidate the cellular and molecular mechanisms ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374453/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12374453">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12974-025-03514-3"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12974_2025_Article_3514.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374453%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12374453/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12374453/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374453/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-jneuro.png" alt="Journal of Neuroinflammation logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Journal of Neuroinflammation" title="Link to Journal of Neuroinflammation" shape="default" href="https://jneuroinflammation.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Neuroinflammation</button></div>. 2025 Aug 23;22:205. doi: <a href="https://doi.org/10.1186/s12974-025-03514-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12974-025-03514-3</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Neuroinflammation%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Neuroinflammation%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Neuroinflammation%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Neuroinflammation%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Single−cell transcriptomic landscape of sciatic nerve after transection injury</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ouyang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yiben Ouyang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yiben Ouyang</span></h3>
<div class="p">
<sup>1</sup>School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China </div>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ouyang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yiben Ouyang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mingqian Yu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mingqian Yu</span></h3>
<div class="p">
<sup>1</sup>School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mingqian Yu</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Tieyuan Zhang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Tieyuan Zhang</span></h3>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">
<sup>4</sup>Cheeloo College of Medicine, Shandong University, 44 West Wenhua Road, Lixia District, Jinan, 250012 Shandong P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tieyuan Zhang</span></a>
</div>
</div>
<sup>2,</sup><sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Haofeng Cheng</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Haofeng Cheng</span></h3>
<div class="p">
<sup>1</sup>School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China </div>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haofeng Cheng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zuo%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Liang Zuo</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Liang Zuo</span></h3>
<div class="p">
<sup>1</sup>School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China </div>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zuo%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Liang Zuo</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Haolin Liu</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Haolin Liu</span></h3>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haolin Liu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guan%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Yanjun Guan</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Yanjun Guan</span></h3>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guan%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yanjun Guan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ao Liu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ao Liu</span></h3>
<div class="p">
<sup>1</sup>School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China </div>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ao Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Jiajie Chen</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Jiajie Chen</span></h3>
<div class="p">
<sup>1</sup>School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China </div>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiajie Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Ruichao He</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Ruichao He</span></h3>
<div class="p">
<sup>1</sup>School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China </div>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ruichao He</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Sice Wang</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Sice Wang</span></h3>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sice Wang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Su%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Tianqi Su</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Tianqi Su</span></h3>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Su%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tianqi Su</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tan%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Yixiao Tan</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Yixiao Tan</span></h3>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tan%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yixiao Tan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Yuhui Cu</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Yuhui Cu</span></h3>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuhui Cu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Xiaochun Zhang</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Xiaochun Zhang</span></h3>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaochun Zhang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Xiaoyang Fu</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Xiaoyang Fu</span></h3>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoyang Fu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Junli Wang</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Junli Wang</span></h3>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">
<sup>5</sup>Jinzhou Medical University, No. 40, Songpo Road, San Duan, Linghe District, Jinzhou, Liaoning China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junli Wang</span></a>
</div>
</div>
<sup>2,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Jinjuan Zhao</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Jinjuan Zhao</span></h3>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jinjuan Zhao</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Jiang Peng</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Jiang Peng</span></h3>
<div class="p">
<sup>1</sup>School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China </div>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">
<sup>3</sup>Co−innovation Center of Neuroregeneration, Nantong University, Nantong, 226007 Jiangsu Province PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiang Peng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Yu Wang</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Yu Wang</span></h3>
<div class="p">
<sup>1</sup>School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China </div>
<div class="p">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div class="p">
<sup>3</sup>Co−innovation Center of Neuroregeneration, Nantong University, Nantong, 226007 Jiangsu Province PR China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yu Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Medicine, Nankai University, No. 94, Weijin Road, Nankai District, Tianjin, 300071 PR China </div>
<div id="Aff2">
<sup>2</sup>Institute of Orthopedics, Beijing Key Lab of Regenerative Medicine in Orthopedics, Key Laboratory of Musculoskeletal Trauma &amp; War Injuries PLA, The Fourth Medical Center of Chinese PLA General Hospital, No. 51 Fucheng Road, Beijing, 100048 PR China </div>
<div id="Aff3">
<sup>3</sup>Co−innovation Center of Neuroregeneration, Nantong University, Nantong, 226007 Jiangsu Province PR China </div>
<div id="Aff4">
<sup>4</sup>Cheeloo College of Medicine, Shandong University, 44 West Wenhua Road, Lixia District, Jinan, 250012 Shandong P.R. China </div>
<div id="Aff5">
<sup>5</sup>Jinzhou Medical University, No. 40, Songpo Road, San Duan, Linghe District, Jinzhou, Liaoning China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 3; Accepted 2025 Jul 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12374453  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40849669/" class="usa-link">40849669</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Peripheral nerve injuries, particularly those affecting the sciatic nerve, often result in incomplete functional recovery due to the limited regenerative capacity of adult peripheral nerves. To elucidate the cellular and molecular mechanisms underlying nerve regeneration, we performed single−cell RNA sequencing (scRNA−seq) on rat sciatic nerve tissues at seven time points (Days 0, 1, 3, 5, 7, 10, and 14) following transection injury. Through unsupervised clustering, we identified four major cellular compartments—neurofibroblasts (NFs), glial cells (Glis), immune cells, and vascular cells—and delineated their dynamic trajectories during regeneration. Early responses were dominated by macrophage (Mac) and granulocyte infiltration (Day 1), followed by proliferative expansion of proliferating mesenchymal fibroblasts (NF5) and repair Schwann cells (Gli0) by Days 3–5. Vascular remodeling commenced from Day 7, while Glis progressively transitioned to mature myelinating states (Gli2/Gli5) by Day 14. Pseudotime analysis revealed subtype−specific reprogramming in both Macs and Glis, and cell–cell communication analysis uncovered key ligand–receptor interactions—particularly collagen and PTN signaling between Macs, NFs, and Glis. Bulk transcriptomic validation confirmed sustained and spatially distinct activation of the <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e296.gif" loading="lazy" id="d33e296" alt="Inline graphic"></span> signaling pathway across cell types and anatomical locations. Comparative analysis with a sciatic nerve crush injury model revealed a stronger early immune response and delayed Gli recovery in transection injury, indicating a narrowed therapeutic window. Together, this work provides a time−resolved single−cell atlas of peripheral nerve regeneration, defines key regulatory circuits within the immune–NF–Gli axis, and identifies phase−specific therapeutic targets—such as early Mac heterogeneity, NF4−mediated matrix remodeling, and Schwann cell remyelination—for enhancing functional recovery following severe nerve injury.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12974-025-03514-3.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Sciatic nerve transection, Single−cell RNA−seq, Glial cell reprogramming, Macrophage heterogeneity, Neurofibroblast–glia interaction, <em>TGF−β</em> signaling, Nerve regeneration</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Peripheral nerve injuries, particularly those involving the sciatic nerve, pose significant clinical challenges with profound socioeconomic impacts. Globally, sciatic nerve trauma has an incidence of 7.7% in specific high−risk populations (e.g., acetabular fracture patients), among whom iatrogenic injuries account for 12.87% of postoperative complications [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. The primary etiologies include traumatic accidents [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>], surgical interventions [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>], and metabolic disorders such as diabetic neuropathy [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Despite advances in microsurgical techniques, functional recovery rates remain suboptimal, with only 50% of patients achieving clinically meaningful recovery, while persistent motor deficits and chronic neuropathic pain—strongly correlated with reduced quality of life in physical functioning and social participation domains [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]—continue to burden survivors. This incomplete recovery underscores the urgent need to dissect the multicellular dynamics that underlie nerve repair—particularly the role of macrophage (Mac) subpopulations and transcriptional reprogramming in axonal regeneration—as revealed by recent single−cell RNA sequencing (scRNA−seq) studies [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>].</p>
<p id="Par3">The primary therapeutic strategies for sciatic nerve injury, such as nerve grafting and neurotrophic factor administration, remain limited by incomplete axonal regeneration and misdirected reinnervation [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. These limitations arise primarily due to the inability of traditional bulk RNA sequencing approaches to resolve the spatiotemporal coordination among Schwann cells, fibroblasts, immune cells, and vascular cells during regeneration. By averaging signals across heterogeneous cell populations, bulk RNA sequencing obscures critical cell type−specific responses [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. Schwann cells exhibit dynamic phenotypic transitions between dedifferentiated, pro−regenerative, and repair states [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>], while Macs undergo polarization from pro−inflammatory to pro−repair subtypes [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]—processes that bulk RNA sequencing fails to resolve.</p>
<p id="Par4">Emerging scRNA−seq technologies now enable unprecedented cellular−resolution dissection of neural repair mechanisms. In central nervous system injury models—such as ischemic stroke and spinal cord injury—scRNA−seq has revealed transitional cellular states and intercellular crosstalk networks that dictate regenerative outcomes [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. More recently, in peripheral nerve injury models, scRNA−seq has been employed to unravel the heterogeneity and functional states of Schwann cells, fibroblasts, immune subsets, and vascular cells. For instance, distinct transcriptional alterations have been identified in both myelinating and non−myelinating Schwann cells under autoimmune conditions, providing insights into glial plasticity [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Endothelial Plexin−D1 has been shown to play dual roles in peripheral nerve repair by not only guiding the directional growth of endothelial cells (ECs) but also regulating angiogenic patterning [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. In parallel, studies using Aire−deficient mouse models demonstrated that T cell–derived <em>IFN−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/020274b3a072/d33e387.gif" loading="lazy" id="d33e387" alt="Inline graphic"></span> induces Mac <em>TNF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/73975112fc4a/d33e396.gif" loading="lazy" id="d33e396" alt="Inline graphic"></span> expression, thereby driving Mac phenotype switching and amplifying inflammatory responses [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Furthermore, in trauma−induced heterotopic ossification, scRNA−seq revealed a high degree of spatial colocalization between peripheral nerves and blood vessels [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], implicating coordinated neurovascular remodeling. Collectively, these findings underscore the power of scRNA−seq to disclose cellular plasticity, spatial interactions, and regulatory networks that remain obscured in conventional bulk transcriptomic analysis.</p>
<p id="Par5">Critical knowledge gaps persist regarding the spatiotemporal regulation of cellular responses during sciatic nerve regeneration. Despite recent advances identifying transcription factors (e.g., <em>Sox10</em> [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>] and <em>Zeb2</em> [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]) and signaling pathways (e.g., <em>Neuregulin−1/ErbB</em> [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] and <em>Wnt/</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e431.gif" loading="lazy" id="d33e431" alt="Inline graphic"></span><em>−catenin</em> [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]) as key regulators of Schwann cell development, the molecular mechanisms governing Schwann cell fate decisions—particularly the transition into repair−promoting phenotypes following nerve injury—remain incompletely understood.​ Immune cell dynamics, including the recruitment of bone marrow−derived Macs and neutrophils, exhibit temporal specificity—Macs peak at Day 3 post−injury, while neutrophils surge within 24 h—a pattern consistent with the time−dependent immune activation originally observed in the sciatic nerve crush injury model [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Vascular ECs in the sciatic nerve demonstrate distinct subtypes, including epineurial, endoneurial, and lymphatic endothelial cells (lyECs)—each characterized by unique gene expression profiles. Marker genes such as <em>Spock2</em>, <em>Rgcc</em>, and <em>Lrg1</em> have been validated in vivo for the identification of these subtypes, offering improved specificity over classical pan−endothelial markers like <em>Pecam1</em> [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>].</p>
<p id="Par6">To address the unresolved complexity of peripheral nerve regeneration, we conducted a longitudinal scRNA−seq study of rat sciatic nerves at multiple time points following transection injury. By mapping the dynamic transitions of major cellular compartments—including glial cells (Glis), neurofibroblasts (NFs), immune cells, and vascular populations—we systematically characterized the cellular choreography underlying nerve repair. Particular emphasis was placed on dissecting how Glis influence NFs behavior and how intercellular communication evolves across regenerative phases. Pseudotime trajectory analyses were used to reconstruct lineage transitions, notably the differentiation of monocytes (Mos) into functionally distinct Mac subsets and the reprogramming of Schwann cells from repair to remyelinating phenotypes. To further validate the cell type–specific signaling changes observed at the single−cell level, we performed bulk RNA−seq on anatomically defined nerve segments and demonstrated sustained, spatially distinct activation of the transforming growth factor beta (<em>TGF</em><em>−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e467.gif" loading="lazy" id="d33e467" alt="Inline graphic"></span>) signaling pathway across the repair timeline. Moreover, by comparing cellular and molecular responses between transection and crush injury models, we uncovered both shared and injury−specific regenerative strategies. Together, our study presents a comprehensive, time−resolved cellular and transcriptomic atlas of sciatic nerve regeneration and identifies phase−specific signaling programs and cell–cell interactions—particularly involving <em>TGF</em><em>−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e477.gif" loading="lazy" id="d33e477" alt="Inline graphic"></span> and NF–Gli crosstalk—that may inform targeted therapeutic interventions for peripheral nerve repair.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Sciatic nerve transection model</h3>
<p id="Par7">Adult female Sprague–Dawley rats (8–10 weeks old, 220–250 g) were purchased from Beijing Vital River Laboratory Animal Technology Co., Ltd. All experimental procedures involving animals were approved by the Institutional Animal Care and Use Committee of PLA General Hospital (approval number: 2023‑x4‑19) and conducted in accordance with national and international guidelines, including those from the International Council for Laboratory Animal Science. All efforts were made to minimize animal suffering and reduce the number of animals used.</p>
<p id="Par8">Rats were anesthetized by intraperitoneal injection of sodium pentobarbital (30 mg/kg). Under sterile conditions, the right sciatic nerve was exposed via a mid‑thigh skin incision, freed from surrounding tissue, and sharply transected approximately 10 mm proximal to its trifurcation. The muscle fascia and skin were then closed in two layers with 4‑0 absorbable sutures. After recovery from anesthesia, animals were housed in standard plastic cages with a 12‑hour light/12‑hour dark cycle and free access to food and water. Five rats were used per time point.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Reference sciatic nerve crush model (Zhao et al., 2022)</h3>
<p id="Par9">The sciatic nerve crush model and corresponding data used in this study were originally established and generated by Zhao et al. (2022) [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Although we did not perform the sciatic nerve crush procedure ourselves, here we briefly summarize their methodology for clarity. In their study, adult C57BL/6 mice (8–16 weeks old) were anesthetized and the right sciatic nerve exposed via a dorsal thigh incision. The nerve was crushed approximately 5 mm distal to the piriformis muscle using fine forceps (Dumont #55) for 15 s under stable pressure, producing axonal injury while preserving the epineurium. Sham controls underwent nerve exposure without crush. Postoperative analgesia and tissue collection procedures followed standard protocols as detailed by Zhao et al. (2022).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Tissue collection and single−cell dissociation</h3>
<p id="Par10">At each designated post‑injury time point (Day 1, 3, 5, 7, 10, and 14), five rats were re‑anesthetized and the original incision reopened (Additional file 1: Fig. <a href="#MOESM1" class="usa-link">S1</a>a). For Day 0 (immediately after transection), a ± 2 mm segment proximal and distal to the transection site was harvested. For Day 1–3 (before nerve‑bridge formation), a 2 mm segment was collected from each stump (proximal and distal to the transection site). For Day 5–14 (after nerve‑bridge formation), a ± 2 mm segment of the newly formed inter-stump nervebridge was harvested. Tissues from the five animals at each time point were pooled,immediately placed in ice-cold Hank’s Balanced Salt Solution (HBSS), and enzymaticallydigested in collagenase IV (1 mg/mL) and DNase I (50 U/mL) at 37 °C for 30 min. Theresulting cell suspension was filtered through a 40 μm cell strainer, centrifuged at 300 × gfor 5 min, and resuspended in phosphate−buffered saline (PBS) containing 0.04% bovineserum albumin (BSA) for subsequent scRNA−seq library preparation.</p></section><section id="Sec6"><h3 class="pmc_sec_title">scRNA−seq and preprocessing</h3>
<p id="Par11">scRNA−seq was performed using the Chromium Single Cell Gene Expression Solution (10x Genomics), which enables the isolation and labeling of 500−10,000 individual cells. This technology is based on the GemCode microfluidics platform, where barcoded gel beads and single cells are encapsulated within oil droplets (Gel Bead−In − EMulsions, GEMs). Within each GEM, gel beads dissolve, and cells undergo lysis, releasing mRNA that is reverse−transcribed into barcoded cDNA. After breaking the emulsion, cDNA was amplified via PCR, followed by quality control to assess fragment size and yield. The amplified cDNA was then fragmented to 200–300 bp, end−repaired, A−tailed, and ligated with sequencing adapters before undergoing index PCR amplification. Library quality was validated before sequencing on the Illumina NovaSeq 6000 platform to obtain high−throughput single−cell gene expression data.</p>
<p id="Par12">Raw sequencing reads were processed using the Cell Ranger pipeline (10x Genomics), including read alignment to the rat genome (Rnor_6.0), barcode assignment, and unique molecular identifier (UMI) counting. Low−quality cells with high mitochondrial gene content (&gt; 10%) or low total UMI counts (&lt; 500) were removed. Doublet detection was performed using <em>DoubletFinder</em>, and doublets were excluded from downstream analysis.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Tissue collection for bulk RNA−seq</h3>
<p id="Par13">To spatially profile transcriptomic changes during nerve regeneration, we collected 2−mm segments from both the proximal and distal stumps of the transected sciatic nerve at designated time points (Days 1, 3, 5, 7, and 14), along with uninjured control samples. For each time point, tissues from 3 rats were pooled for each anatomical region. Samples were immediately snap−frozen in liquid nitrogen and stored at − 80 °C until RNA extraction.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Bulk RNA−seq library preparation and sequencing</h3>
<p id="Par14">Total RNA was extracted from tissue samples using a standard protocol, and RNA integrity was assessed using the Agilent 2100 Bioanalyzer. Samples with sufficient yield and RNA integrity were used for downstream bulk RNA−seq analysis. mRNA was enriched using oligo(dT) magnetic beads to capture polyadenylated transcripts or, alternatively, rRNA was depleted to retain a broader transcript profile. The enriched RNA was fragmented and reverse−transcribed to cDNA. Libraries were prepared using either standard or strand−specific protocols, the latter incorporating dUTP during second−strand synthesis to retain strand information [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. After end repair, A−tailing, adaptor ligation, size selection, and PCR amplification, libraries were sequenced on an Illumina platform with paired−end 150 bp reads.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Clustering and cell type annotation</h3>
<p id="Par15">Dimensionality reduction was performed using principal component analysis (PCA) on the top 3000 variable genes. The first 30 principal components were used for uniform manifold approximation and projection (UMAP) visualization. Unsupervised clustering was performed using the <em>Seurat</em> package [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] (version 5.1.0) with the Louvain algorithm at a resolution of 0.8. Cell types were annotated based on the expression of established marker genes: NFs (<em>Col1a1</em>,<em> Dcn</em>,<em> Col3a1</em>), Glis (<em>Mpz</em>,<em> S100b</em>,<em> Mag</em>), immune cells (<em>Aif1</em>,<em> Cd68</em>,<em> Cd3e</em>), and vascular cells (<em>Vtn</em>,<em> Esam</em>,<em> Plvap</em>). Further subclustering within major cell populations was performed to identify distinct cellular subtypes.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Bioinformatic analysis and statistics</h3>
<p id="Par16">We employed Seurat for downstream analysis, following a structured pipeline to process and analyse the scRNA−seq data. The analysis was conducted in seven key steps:</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par18">Normalization: Raw gene expression counts were normalized on a per − cell basis using log−normalization (log1p transformation), in which the natural logarithm of 1 plus the counts per 10,000 was computed. This step ensures that expression levels are comparable across cells and suitable for downstream analyses.</p></li>
<li><p id="Par19">Highly Variable Gene Selection and Batch Effect Correction: The top 3,000 highly variable genes were identified using the <em>FindVariableFeatures</em> function, capturing genes with the greatest variability across the dataset and likely reflecting biologically meaningful signals. To mitigate batch effects, integration anchors were first identified using the <em>FindIntegrationAnchors</em> function and then used to integrate the datasets with <em>IntegrateData</em>.</p></li>
<li><p id="Par20">Data Scaling: Gene expression values were standardized across all cells using Z−score transformation via the <em>ScaleData</em> function. This step adjusts for differences in average gene expression levels, facilitating cross−cell comparisons and downstream dimensionality reduction.</p></li>
<li><p id="Par21">PCA: PCA was performed on the scaled expression matrix of the highly variable genes to reduce dimensionality and capture the primary axes of variation in the dataset.</p></li>
<li><p id="Par22">UMAP Visualization: UMAP was used to project the high−dimensional data into a two−dimensional space for visualization. The <em>RunUMAP</em> function in Seurat was executed using principal components 1 through 15 (dims = 1:15), as determined from the PCA on the subsetted data.</p></li>
<li><p id="Par23">Clustering: Cell clustering was carried out using a graph−based approach with the Louvain algorithm, as implemented in the <em>FindClusters</em> function. A shared nearest neighbor (SNN) graph was first constructed using <em>FindNeighbors</em>, and clustering was performed with a <em>resolution</em> parameter of 0.3 to define discrete cell populations within the sciatic nerve dataset.</p></li>
<li><p id="Par24">Differential Gene Expression (DGE) Analysis: To identify differentially expressed genes (DEGs) for each cluster, the Wilcoxon rank−sum test was applied using the <em>FindMarkers</em> function. The analysis was conducted on the log−normalized expression matrix, with <em>min.pct</em> = 0.25 set to include genes expressed in at least 25% of cells in either group. All other parameters were kept at their default values, including <em>only.pos</em> = TRUE, <em>logfc.threshold</em> = 0.1, and <em>max.cells.per.ident</em> = Inf. This strategy enabled robust identification of cluster−specific marker genes while maintaining sensitivity to subtle expression differences.</p></li>
</ol></section><section id="Sec11"><h3 class="pmc_sec_title">Cell type annotation</h3>
<p id="Par25">For unsupervised cell type annotation, we utilized the <em>SingleR</em> package [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>] (version 2.8.0) with the crush sciatic nerve injury single−cell dataset (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198582" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE198582</a> [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]) as a reference. Cell type assignments were further manually validated by examining the DEGs for the presence of canonical marker genes for each cell type. Based on this analysis, we assigned metacells to various cell types, including NF, Gli, PC, SMC, Mac, Gran, DC, and T /NK/B cell.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Subclustering and further resolution</h3>
<p id="Par26">To achieve higher resolution of cell states, subclustering was performed on major cell types by first subsetting specific populations (e.g., NFs) from the integrated dataset. For each subset, data normalization and scaling were repeated using the <em>SCTransform</em> function to ensure consistency in preprocessing. Dimensionality reduction was carried out via PCA, and the appropriate number of principal components (PCs) was determined using <em>ElbowPlot</em>. The selected PCs (typically 1–10/15) were then used for constructing a SNN graph and performing graph−based clustering using the <em>FindNeighbors</em> and <em>FindClusters</em> functions, with an appropriate resolution to capture finer subpopulations. Low−dimensional embeddings were generated using UMAP based on the selected PCs.</p>
<p id="Par27">DEGs between subclusters were identified using the <em>FindMarkers</em> function with the parameter <em>min.pct</em> = 0.25, while other parameters remained at their default settings. DEGs were ranked by log2 fold−change values to identify representative marker genes. For visualization, z−scores were computed based on the average expression levels of each gene across subclusters, and the resulting matrix was visualized using heatmaps, enabling clear comparison of transcriptional profiles and aiding in the refinement of subpopulation annotations.</p></section><section id="Sec13"><h3 class="pmc_sec_title">DGE and enrichment analysis</h3>
<p id="Par28">DGE analysis was performed using the <em>FindMarkers</em> function in Seurat. For identifying marker genes of general cell subtypes, genes were considered significant if they were both upregulated in the target cluster and had a <em>p</em>−value &lt; 0.05. This dual criterion ensured that selected genes were not only statistically significant but also biologically relevant in distinguishing cell populations. This analysis enabled the classification of distinct cellular subtypes and facilitated the identification of key regulatory genes within each lineage.</p>
<p id="Par29">For the comparative analysis between crush injury and transection injury, a stricter threshold was applied to identify DEGs that distinguished the two injury models. After using <em>FindMarkers</em>, genes were considered significantly different if they met the criteria of fold−change &gt; 1 and adjusted <em>p−value</em> &lt; 0.01. These DEGs were subsequently subjected to Gene ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analysis using <em>clusterProfiler</em> [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] to identify biological pathways that were commonly activated in both injury types or uniquely enriched in either crush or transection injuries. This approach enabled a detailed characterization of both shared and distinct molecular mechanisms underlying nerve repair in different injury models.</p>
<p id="Par30">In addition to single−cell−based DGE analysis, we performed gene set enrichment analysis (GSEA) on bulk RNA−seq data to assess pathway activation dynamics at multiple anatomical locations and time points. Gene sets were referenced from the 2024 mouse version of the MSigDB database, and enrichment was calculated based on ranked gene expression profiles. Particular focus was placed on <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e721.gif" loading="lazy" id="d33e721" alt="Inline graphic"></span> signaling–related pathways, whose activation patterns were compared across injury sites (e.g., proximal stump, distal stump) and different post−injury days. This bulk transcriptomic analysis confirmed the temporal regulation of <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e729.gif" loading="lazy" id="d33e729" alt="Inline graphic"></span> signaling during nerve regeneration and provided orthogonal validation for the findings derived from single−cell data.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Pseudotime and trajectory analysis</h3>
<p id="Par31">To investigate dynamic cellular transitions during nerve repair, pseudotime trajectory analysis was performed using <em>Monocle</em> [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] for Glis and <em>slingshot</em> [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] for Mo to Mac differentiation. For Glis, <em>Monocle</em> was used to model the lineage trajectory from repair Schwann cells (Gli0) to myelinating Schwann cells (Gli5). After filtering out Day 0 samples, cells from Day 5 to Day 14 were selected for analysis. Cells were re−embedded based on highly variable genes, and a subset of genes expressed in at least 350 cells was retained. To define ordering genes for trajectory inference, we performed differential expression analysis across timepoints using <em>differentialGeneTest</em>, and genes with <em>p</em> &lt; <em>1e</em><sup><em>− 15</em></sup> were selected. The top pseudotime−associated genes were then analyzed for expression trends using <em>plot_pseudotime_heatmap</em>, and k−means clustering (<em>k</em> = 3) was applied to identify co−expression modules. These clusters were interpreted as representing early−stage regulatory genes, intermediate−phase modulators, and late−stage effectors, corresponding to successive waves of gene activation during Gli reprogramming and remyelination. To explore the biological significance of these gene modules, we conducted GO enrichment analysis for each gene cluster using <em>clusterProfiler</em> [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], revealing functional themes associated with glial plasticity, differentiation, and remyelination. For the immune compartment, <em>slingshot</em> was used to reconstruct lineage relationships between infiltrating Mos and differentiated Mac subtypes. The trajectory was inferred based on UMAP embeddings, with pseudotime values extracted along inferred lineages. For downstream analysis, cells with valid pseudotime values were filtered, and a generalized additive model (GAM) was fitted to gene expression using the tradeSeq framework. Genes exhibiting significant changes along the trajectory were identified using <em>associationTest</em> and <em>startVsEndTest</em>, highlighting transcriptional programs that govern Mo to Mac fate transitions. Top−ranked genes visualized with <em>plotSmoothers</em> were prioritized for further interpretation. Together, these analyses provided comprehensive insights into the temporal orchestration of glial reprogramming and immune differentiation during peripheral nerve regeneration.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Cell−cell communication analysis</h3>
<p id="Par32">Cell−cell communication analysis was performed to investigate intercellular signaling dynamics during sciatic nerve repair. The <em>CellChat</em> package [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] was used to infer ligand−receptor interactions among different cell populations. Normalized single−cell RNA−seq data were used as input, and the analysis was conducted separately for different time points to track temporal changes in signaling activity. First, the expression of known ligand−receptor pairs was assessed across all major cell types, including NFs, Glis, immune cells (Macs, Mos, Grans, T cells, B cells, NK cells, and DCs), and vascular cells (PCs, ECs, SMCs, and lyECs). Communication networks were constructed based on the strength and specificity of ligand−receptor interactions. The <em>computeCommunProb</em> function was applied to calculate interaction probabilities, followed by <em>computeCommunProbPathway</em> to identify active signaling pathways. To visualize signaling patterns, network diagrams and heatmaps were generated to depict interactions between cell types. The <em>netVisual_circle</em> function was used to illustrate global intercellular communication, while <em>netVisual_bubble</em> provided insights into specific ligand−receptor pairs. Pathway−level analysis was conducted to identify key signaling cascades, with a focus on those implicated in nerve regeneration, including collagen−related signaling, <em>PTN</em> (Pleiotrophin) signaling between Glis and NFs, and Mac−mediated extracellular matrix remodeling. The strength and directionality of communication were assessed by examining changes in outgoing and incoming signaling patterns for specific cell populations over time. Statistical analyses were corrected for multiple comparisons using the Benjamini−Hochberg method to control the false discovery rate.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Comparative analysis of crush injury and transection injury models</h3>
<p id="Par33">To compare the cellular and molecular responses between crush injury and transection injury models, we conducted a detailed comparative analysis using scRNA−seq data. The scRNA−seq data from both injury models were integrated using the <em>FindIntegrationAnchors</em> function in Seurat, which identifies mutual nearest neighbors (anchors) between datasets to correct for batch effects and technical variations while preserving biological differences. Following anchor identification, the datasets were harmonized using the <em>IntegrateData</em> function, generating a combined dataset with minimized batch effects. The integrated dataset was log−normalized using the natural log1p normalization, and the 3,000 most variable genes were identified using the <em>FindVariableFeatures</em> function. Expression values were standardized across cells using Z−score transformation, and PCA was performed on the scaled variable gene matrix. Clustering was performed using the Louvain algorithm implemented in the <em>FindClusters</em> function, with a resolution setting of 0.7 to identify distinct cell populations. Cell type annotations were based on the classification derived from the crush injury dataset, which included various functional subtypes such as fibroblasts (e.g., proliferating, differentiating, and matrix−stabilizing subtypes), Schwann cells (e.g., proliferating, repairing, and myelinating subtypes), Macs (e.g., pro−inflammatory, pro−repair, and proliferating subtypes), Mo, Gran, T cells, B cells, ECs, and PCs. These annotations were manually validated by examining the expression of canonical marker genes for each cell type. DGE analysis was conducted using the <em>FindMarkers</em> function, with genes considered significantly differentially expressed if they exhibited a fold change &gt; 1 and an adjusted <em>p</em>−value &lt; 0.01. GO and KEGG pathway enrichment analysis were conducted using the <em>clusterProfiler</em> package, with pathways considered significantly enriched if they had an adjusted <em>p</em>−value &lt; 0.05. This comprehensive comparative analysis provided insights into the distinct cellular and molecular mechanisms underlying nerve repair in crush injury and transection injury models, highlighting potential therapeutic targets for enhancing nerve regeneration in different injury contexts.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Immunofluorescence staining of sciatic nerve sections</h3>
<p id="Par34">At each designated post−injury time point (Day 0, 1, 3, 5, 7, 10, and 14), rats were deeply anesthetized with sodium pentobarbital (50 mg/kg, intraperitoneally) and euthanized by overdose. The sciatic nerve, including the proximal stump, distal stump, and the regenerating nerve bridge (if present), was carefully dissected under a stereomicroscope. Tissues were immediately immersed in 4% paraformaldehyde (PFA) at 4 °C for 24 h. Following fixation, samples were cryoprotected in 30% sucrose (w/v in PBS) at 4 °C until fully equilibrated (i.e., tissues sank). Cryoprotected tissues were embedded in optimal cutting temperature (OCT) compound, frozen, and sectioned longitudinally at a thickness of 10 μm using a cryostat (Leica CM1950). Sections were mounted onto glass slides and air−dried at room temperature.</p>
<p id="Par35">Immunofluorescence staining was performed in two separate rounds. In the first round, sections were permeabilized and blocked with 10% normal goat serum containing 0.3% Triton X−100 in PBS for 1 h at room temperature. Sections were then incubated overnight at 4 °C with primary antibodies against S100b (1:200, Abcam, ab52642), a marker for Schwann cells, and NF200 (1:300, Sigma–Aldrich, N0142), a marker for axons. After PBS washes, sections were incubated for 1 h at room temperature in the dark with goat anti–mouse Alexa Fluor 488 (1:1000, Invitrogen, A11029) and goat anti–rabbit Alexa Fluor 594 (1:1000, Invitrogen, A11012). In the second round, separate sections were stained with anti–TGF–β1 (1:200, Abmart, TA1027) to detect TGF–β1 expression. The same blocking and permeabilization protocol was used, and the sections were incubated overnight at 4 °C with the primary antibody, followed by goat anti–rabbit Alexa Fluor 594 (1:1000, Invitrogen, A11012) for secondary detection. In both staining procedures, nuclei were counterstained with DAPI and coverslipped using antifade mounting medium.</p></section></section><section id="Sec18"><h2 class="pmc_sec_title">Result</h2>
<section id="Sec19"><h3 class="pmc_sec_title">Single−cell transcriptomics reveals dynamic cellular heterogeneity and intercellular communication in sciatic nerve repair</h3>
<p id="Par36">We conducted scRNA−seq on rat sciatic nerve tissues following transection injury by analyzing 58,943 high−quality cells across seven time points (Day 0, 1, 3, 5, 7, 10, 14) (Additional file 1: Fig. <a href="#MOESM1" class="usa-link">S1</a>). Unsupervised clustering and UMAP dimensionality reduction revealed four major cellular compartments: NFs, Glis, immune cells (Mac, Mo, Gran, T/B/NK/DC cells), and vascular cells (PC, EC, SMC, lyEC) (Fig. <a href="#Fig1" class="usa-link">1</a> and Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>a). Subclustering and annotation analysis revealed distinct cellular subtypes within each major compartment (Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a> and Additional file 3: Fig. <a href="#MOESM3" class="usa-link">S3</a>a). In the NF lineage, we identified fibroblasts and mesenchymal cell populations, along with their respective subtypes (Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>b). Glis exhibited functional heterogeneity, comprising proliferating, repairing, myelinating, and non−myelinating subtypes (Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>c). Immune cell diversity was characterized by distinct Mac populations, including proliferating Macs, as well as plasmacytoid and mature/migrating dendritic cells (DCs) (Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>f−k). Within the vascular compartment, we identified arterial pericytes (PCs), proliferating PCs and additional SMC−associated subtypes (Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>d, e). These findings highlight the cellular complexity and dynamic responses underlying sciatic nerve repair.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374453_12974_2025_3514_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/7d8e4e7765bf/12974_2025_3514_Fig1_HTML.jpg" loading="lazy" id="d33e918" height="1015" width="718" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Temporal Cell Type Heterogeneity Following Sciatic Nerve Transection. (<strong>a</strong>) UMAP visualization of integrated single–cell RNA sequencing (scRNA–seq) data across different post–injury time points. Each dot represents a single cell, colored by time point. (<strong>b</strong>) Left: UMAP showing clustering into four major cell types—neurofibroblasts (NFs), immune cells, glial cells (Glis), and vascular cells. Right: Cell type distribution across time points. (<strong>c</strong>) Bar plot of relative cell type proportions at each time point. (<strong>d</strong>) Dot plot of canonical marker genes across cell types. Dot size represents the proportion of cells expressing each gene (≥ 1 UMI); color indicates average expression level. (<strong>e</strong>) Heatmap of differentially expressed genes (DEGs) among primary cell types. Red: high expression; gray: low expression. (<strong>f</strong>) Volcano plots of DEGs for each major cell type. Top 10 DEGs are labeled; y–axis shows log₂ fold change (log₂FC). (<strong>g</strong>) Immunofluorescence staining for S100 (red, Schwann cells) and NF200 (green, axons) in uninjured and injured sciatic nerves (Days 3, 5, 7, 10, 14), showing spatiotemporal changes in Gli activity and axonal structure. Scale bar: 500 μm</p></figcaption></figure><p id="Par38">Dynamic changes in cellular composition were observed following sciatic nerve transection (Fig. <a href="#Fig1" class="usa-link">1</a>a−c). At Day 0, NFs constituted the predominant population, accounting for 81.8% of the total cells, followed by vascular and immune cells, with Glis being the least abundant (Additional file 3: Fig. <a href="#MOESM3" class="usa-link">S3</a>c). NFs were characterized by the expression of key extracellular matrix−related genes such as <em>Col1a1</em>,<em> Dcn</em>,<em> Col3a1</em>,<em> Col6a2</em>, and <em>Lum</em>, which play a fundamental role in maintaining tissue integrity (Fig. <a href="#Fig1" class="usa-link">1</a>d). Vascular cells were identified by markers including <em>Vtn</em>,<em> Esam</em>,<em> Plvap</em>,<em> Acta2</em>, and <em>Des</em>, while immune cells exhibited distinct signatures such as <em>Aif1</em>,<em> Cd68</em>,<em> Cd3e</em>,<em> Cd3g</em>,<em> Inpp5d</em>, and <em>Adgre1</em>. Glial cells, primarily Schwann cells, expressed <em>Mpz</em>,<em> S100b</em>,<em> Mbp</em>, and <em>Mag</em>, underscoring their role in nerve support and myelination. By Day 1 post−injury, immune cells expanded dramatically, exceeding 95% of the total cell population (Additional file 3: Fig. <a href="#MOESM3" class="usa-link">S3</a>c). This early inflammatory response was dominated by Macs and granulocytes (Grans), consistent with their essential roles in debris clearance and initiating the repair process [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Mac subsets were characterized by markers such as <em>F13a1</em>,<em> Pf4</em>,<em> C1qc</em>,<em> C1qb</em>,<em> Ms4a7</em>, and <em>Folr2</em>, whereas Grans displayed gene signatures including <em>S100a8</em>,<em> S100a9</em>,<em> Il1r2</em>, and <em>Dgat2</em> (Additional file 5: Fig. <a href="#MOESM5" class="usa-link">S5</a>e). The marked immune infiltration suggested a rapid activation of innate immune mechanisms to facilitate the removal of myelin debris [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. By Day 5, NFs re−emerged in substantial numbers, marking a transition from the inflammatory phase to the regenerative phase. This increase in NFs coincided with a notable rise in Glis, which play a critical role in axonal regeneration and remyelination. At the same time, immune cell numbers began to decline, indicating a shift toward tissue remodeling and repair. By Day 7, vascular cells nearly doubled, primarily comprising PCs and ECs (Fig. <a href="#Fig1" class="usa-link">1</a>c and Additional file 3: Fig. <a href="#MOESM3" class="usa-link">S3</a>c). This vascular expansion likely reflects an increase in angiogenesis and the establishment of a supportive microenvironment for regenerating axons. Given the observed strong cell–cell communication between Glis and vascular cells (Additional file 3: Fig. <a href="#MOESM3" class="usa-link">S3</a>b), their coordinated function may be essential in restoring nerve homeostasis and promoting functional recovery [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>].</p>
<p id="Par40">The analysis of cellular communication during nerve repair reveals complex interactions between different cell types (Additional file 3: Fig. <a href="#MOESM3" class="usa-link">S3</a>b). Glis and vascular cells exhibit strong cell−cell communication both in normal and injured states, suggesting their pivotal roles in maintaining nerve homeostasis and promoting tissue repair. NFs and Glis show robust signaling interactions, with potential regulatory pathways that facilitate cellular differentiation and tissue regeneration [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. While immune cells play a crucial role in the early stages of injury, their cell−cell communication with other cell types is relatively weaker both in normal tissue and throughout the repair process [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. The intricate communication networks observed during the repair process highlight the need for coordinated signaling between multiple cell types to ensure efficient tissue regeneration and functional recovery.</p>
<p id="Par41">In summary, this study provides a comprehensive overview of the cellular landscape during sciatic nerve injury and repair, revealing dynamic shifts in cellular proportions and highlighting the complex cellular interactions that drive tissue regeneration. The identification of key cell subtype markers and their associated signaling pathways offers insights into the molecular mechanisms underlying nerve repair, which may inform future therapeutic strategies aimed at enhancing nerve regeneration.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Temporal dynamics and subtype−specific remodeling of the immune landscape during sciatic nerve repair</h3>
<p id="Par42">At Day 0, Macs constitute the predominant immune cell population, followed by T cells and DCs (Fig. <a href="#Fig2" class="usa-link">2</a>a, b). Following injury, the immune response undergoes dynamic changes over time. During the early phase (Day 1–Day 3), the immune landscape is dominated by Macs and Grans, reflecting their crucial role in the immediate inflammatory response and debris clearance [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. These early immune events are primarily driven by myeloid−derived cells. As the repair process progresses (Day 5–Day 14), the proportion of T cells and DCs increases, while Macs gradually decline, indicating a shift toward a lymphoid−and DC−driven immune environment. This transition highlights the shift from an inflammatory environment to an adaptive immune response and tissue remodeling [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374453_12974_2025_3514_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/c169c7ba1cd9/12974_2025_3514_Fig2_HTML.jpg" loading="lazy" id="d33e1391" height="841" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clustering and Functional Profiling of Myeloid Immune Subtypes. (<strong>a</strong>) UMAP plot showing immune cell types across all time points: granulocytes (Grans), macrophages (Macs), monocytes (Mos), dendritic cells (DCs), T cells, B cells, and NK cells. (<strong>b</strong>) Bar plot of immune cell type proportions over time. (<strong>c</strong>–<strong>e</strong>) Unsupervised subclustering of myeloid cells. Left: UMAP of subtypes; Right: temporal distribution. (<strong>c</strong>) Gran0–Gran3. (<strong>d</strong>) Mac0–Mac4. (<strong>e</strong>) Mo0–Mo1. (<strong>f</strong>–<strong>h</strong>) Heatmaps of upregulated marker genes (Z–score, log scale) for each subtype. (<strong>f</strong>) Grans, (<strong>g</strong>) Macs, (<strong>h</strong>) Mos. (<strong>i</strong>) GO enrichment analysis of each Gran, Mac, and Mo subtype</p></figcaption></figure><section id="Sec21"><h4 class="pmc_sec_title">Myeloid−derived populations: Grans, Macs, and Mos</h4>
<p id="Par44">Grans exhibit significant changes post−injury (Fig. <a href="#Fig2" class="usa-link">2</a>c). At Day 0, the predominant Gran subsets are Gran0 and Gran1, with Gran0 increasing and Gran1 decreasing during the course of nerve repair. Additionally, two new Gran subtypes, Gran2 and Gran3, emerge post−injury. Marker gene analysis identifies Gran0 as expressing <em>Riok3</em>, <em>Gadd45a</em>, and <em>S100a8</em>, while Gran1 expresses <em>Rps28</em> and <em>Naca</em>. Gran2 cells upregulate <em>Ccl2</em> and <em>Gpnmb</em>, whereas Gran3 expresses <em>Atp6v0d1</em>, <em>Psmb3</em>, and <em>Capg</em> (Fig. <a href="#Fig2" class="usa-link">2</a>f). Functional enrichment analysis reveals distinct roles among Gran subtypes during nerve repair (Fig. <a href="#Fig2" class="usa-link">2</a>i). Gran0 is primarily associated with redox homeostasis and metabolic regulation, suggesting a role in detoxification and chemical balance. Gran1 is enriched in pathways related to translation and Toll−like receptor signaling, indicating its involvement in early innate immune responses. Gran2 shows signatures linked to vascular modulation and negative regulation of <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e1134.gif" loading="lazy" id="d33e1134" alt="Inline graphic"></span> signaling, implicating it in immunoregulation and endothelial interactions. Gran3 is associated with sphingolipid metabolism, pointing to potential roles in membrane remodeling and inflammatory resolution. These findings highlight the functional heterogeneity of Grans and their dynamic contributions to different phases of peripheral nerve regeneration.</p>
<p id="Par45">During the early phase (Day 1), the Mac0 subtype dominates (Fig. <a href="#Fig2" class="usa-link">2</a>d). These cells are characterized not only by their involvement in acute−phase responses and metabolic processes related to amino acid metabolism (Fig. <a href="#Fig2" class="usa-link">2</a>i) but also by a unique marker profile that includes <em>Cl1</em>, <em>Cxcl3</em>, <em>Ereg</em>, <em>Slc2a6</em>, <em>Slpi</em>, <em>Ass1</em>, <em>Vcan</em>, <em>Ccl24</em>, and <em>Nrg1</em> (Fig. <a href="#Fig2" class="usa-link">2</a>g). By Day 3, there is a marked shift with Mac1 and Mac2 subtypes becoming more prominent. Mac1 cells—displaying markers such as <em>Fxyd2</em>, <em>Hpse</em>, <em>Asgr2</em>, <em>Kctd4</em>, <em>Pdgfc</em>, <em>Akr1b8</em>, <em>Gdf15</em>, <em>Gsta1</em>, <em>Htr2b</em>, and <em>C6</em>—likely represent a substantial fraction of blood−derived Macs, while Mac2 cells are defined by the expression of <em>Dcn</em>, <em>Col3a1</em>, and <em>Col1a1</em> (Fig. <a href="#Fig2" class="usa-link">2</a>g and Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>f). These subtypes are involved in lipid homeostasis, cytokine regulation, and extracellular matrix remodeling (Fig. <a href="#Fig2" class="usa-link">2</a>i). Their numbers decline by Day 5 and nearly vanish by Day 7, coinciding with the progressive increase in Mac3 and Mac4 populations. The Mac4 subset, identified by a distinct proliferative marker profile (<em>Cep55</em>, <em>Mastl</em>, <em>Hist1h1b</em>, <em>Kif20b</em>, <em>Hmmr</em>, <em>Aspm</em>, <em>Kif4a</em>, <em>Sgo2</em>, <em>Pclaf</em>, and <em>Espl1</em>), gradually increases in the later repair stages (Day 7, Day 10, and Day 14), underscoring its importance in cell proliferation and tissue regeneration (Fig. <a href="#Fig2" class="usa-link">2</a>g).</p>
<p id="Par46">Enrichment analysis further support these findings (Fig. <a href="#Fig2" class="usa-link">2</a>i). Specifically, Mac0 cells are enriched in pathways related to acute−phase response and both proteinogenic and non−proteinogenic amino acid metabolism. In contrast, Mac1 cells are associated with negative regulation of cytokine production, lipid homeostasis, and complement activation, among other immune regulatory processes. Mac2 cells are linked to responses to mechanical stimuli and collagen fibril organization. Meanwhile, Mac3 cells contribute to the regulation of hemopoiesis and non−membrane−bound organelle assembly, and Mac4 cells not only support the proliferative capacity by engaging in chromosome segregation and DNA repair−dependent chromatin remodeling but also modulate epigenetic regulation through pathways such as constitutive heterochromatin formation and negative regulation of the cGAS/STING signaling pathway.</p>
<p id="Par47">Mos, largely absent at Day 0, appear in significant numbers at the lesion site during the subsequent stages of repair (Fig. <a href="#Fig2" class="usa-link">2</a>e, h). Two major Mo subsets are observed: Mo0 and Mo1. Mo0 cells are marked by <em>Cdc42ep4</em>, <em>Polr2j</em>, <em>Dap</em>, <em>Polr2e</em>, <em>Cct3</em>, <em>Ccl2</em>, <em>Cd81</em>, <em>Card19</em>, <em>Rhoc</em>, and <em>Sh3kbp1</em>, and their enrichment analysis shows a strong association with mRNA metabolism, nucleic acid catabolic processes, and T cell−mediated immunity, among other functions. In comparison, Mo1 cells—exhibiting markers such as <em>Cops2</em>, <em>Dhx58</em>, <em>Psmg4</em>, <em>Isg15</em>, <em>Tsta3</em>, <em>Phc2</em>, <em>Pik3cd</em>, <em>Krt75</em>, <em>Gps1</em>, and <em>Polr3k</em>—are enriched in pathways involved in RNA catabolism and the cellular response to increased oxygen levels (Fig. <a href="#Fig2" class="usa-link">2</a>h, i). The stable presence and balanced proportions of these Mo subsets throughout the repair process highlight their crucial role in shaping the heterogeneity and functionality of the Mac populations.</p></section><section id="Sec22"><h4 class="pmc_sec_title">DCs</h4>
<p id="Par48">DCs display dynamic changes in their subpopulations (Fig. <a href="#Fig3" class="usa-link">3</a>a). At Day 0, the sciatic nerve primarily contains DC0 cells, which decrease during repair, while DC1 cells progressively increase. Subtype characterization reveals that DC0 cells, expressing <em>Ccl17</em>, <em>Mfge8</em>, and <em>Clec4a1</em>, are Mo−derived DCs (Fig. <a href="#Fig3" class="usa-link">3</a>e and Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>h). DC1 cells, marked by <em>Mctp2</em>, <em>Slco4a1</em>, and <em>Siglech</em>, predominantly represent plasmacytoid DCs, whereas DC3 cells (<em>Gls2</em>, <em>Fscn1</em>, and <em>Lad1</em>) correspond to mature/migrating DCs, and DC2 cells (<em>Naaa</em>, <em>Slpi</em>, and <em>Xcr1</em>) align with conventional DCs (Fig. <a href="#Fig3" class="usa-link">3</a>e and Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>h). Functional analysis indicates that DC0 cells are involved in biotic stimulus responses and Toll−like receptor signaling, DC1 cells participate in endoplasmic reticulum stress responses and nuclear receptor signaling, DC2 cells regulate innate immune responses and lipid absorption, and DC3 cells mediate actin filament organization and Toll−like receptor signaling (Fig. <a href="#Fig3" class="usa-link">3</a>i). These findings suggest a coordinated DC response that facilitates antigen presentation and immune modulation during nerve repair.</p>
<figure class="fig xbox font-sm" id="Fig3"><h5 class="obj_head">Fig. 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374453_12974_2025_3514_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/caabeff6e656/12974_2025_3514_Fig3_HTML.jpg" loading="lazy" id="d33e1520" height="827" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clustering and Functional Profiling of DC and Lymphoid Subtypes. (<strong>a</strong>–<strong>d</strong>) Unsupervised subclustering of lymphoid and DC subtypes. Left: UMAP; Right: temporal proportions. (<strong>a</strong>) Dendritic cells: DC0–DC3. (<strong>b</strong>) T cells: T0–T3. (<strong>c</strong>) B cells: B0–B2. (<strong>d</strong>) NK cells: NK0–NK1. (<strong>e</strong>–<strong>h</strong>) Heatmaps of upregulated marker genes (Z–score, log) for each subtype. (<strong>e</strong>) DCs, (<strong>f</strong>) T cells, (<strong>g</strong>) B cells, (<strong>h</strong>) NK cells. (<strong>i</strong>–<strong>j</strong>) GO enrichment analysis. (<strong>i</strong>) DC subtypes. (<strong>j</strong>) <strong>T</strong>, <strong>B</strong>, and NK cell subtypes</p></figcaption></figure></section><section id="Sec23"><h4 class="pmc_sec_title">Lymphoid−derived populations: T, B, and NK cells</h4>
<p id="Par50">T cells exhibit distinct subtypes that vary throughout the repair process (Fig. <a href="#Fig3" class="usa-link">3</a>b). At Day 0, the sciatic nerve primarily harbors T0 and T2 cells, with T0 gradually decreasing post−injury. In contrast, T1, T2, and T3 subpopulations expand during the repair phase. Marker gene analysis reveals that T0 cells are characterized by <em>Calhm6</em> and <em>S1pr1</em> expression, whereas T1 cells express <em>Ccl5</em>, <em>Ccl3</em>, and <em>Ccl4</em>, suggesting their involvement in immune activation and recruitment (Fig. <a href="#Fig3" class="usa-link">3</a>f). T2 cells, marked by <em>Btrc</em>, <em>Tnfrsf4</em>, and <em>Foxp3</em>, may correspond to regulatory T cells contributing to immune modulation. T3 cells, enriched in genes related to chromosome segregation and mitotic activity, such as <em>Ncapg</em>, <em>Cdca3</em>, and <em>Top2a</em>, likely represent proliferative T cell subsets. Functional enrichment analysis further supports these findings (Fig. <a href="#Fig3" class="usa-link">3</a>j), as T0 cells are associated with type II interferon production and ATP export, T1 cells with lymphocyte−mediated immunity and dopamine biosynthesis, T2 cells with cytokine regulation and mononuclear cell proliferation, and T3 cells with chromatin remodeling and amino acid metabolism.</p>
<p id="Par51">B cells, although present at low levels throughout the repair process (Fig. <a href="#Fig3" class="usa-link">3</a>c), undergo subtype−specific changes. The primary subsets include B0, B1, and B2 cells. Marker gene analysis indicates that B0 cells express <em>Cd40</em>, <em>Lbh</em>, and <em>Ms4a1</em>, while B1 cells upregulate <em>Gpr171</em>, <em>Grn</em>, and <em>Cd7</em> (Fig. <a href="#Fig3" class="usa-link">3</a>g). Notably, B2 cells, characterized by <em>Nipal1</em>, <em>Ctla4</em>, and <em>Tnfrsf17 </em>expression, primarily correspond to plasma blasts (Fig. <a href="#Fig3" class="usa-link">3</a>g and Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>k). Functional enrichment analysis suggests that B0 cells participate in ribosome biogenesis and interleukin−2 regulation, B1 cells contribute to autophagosome assembly and lectin receptor signaling, and B2 cells engage in endoplasmic reticulum stress responses and protein localization (Fig. <a href="#Fig3" class="usa-link">3</a>j).</p>
<p id="Par52">Natural killer (NK) cells are present at low levels in the sciatic nerve from Day 0 onward, with no significant changes in their proportions observed throughout the repair process (Fig. <a href="#Fig3" class="usa-link">3</a>d). Two major NK cell subtypes are identified: NK0 and NK1. NK0 cells are enriched in <em>Gata3</em>, <em>Cd27</em>, and <em>Gpr183</em>, while NK1 cells express <em>Batf</em>, <em>Il21r</em>, and <em>Gzma</em> (Fig. <a href="#Fig3" class="usa-link">3</a>h). Functional annotation suggests that NK0 cells contribute to T cell chemotaxis and calcium ion transport, whereas NK1 cells are involved in Gran chemotaxis and tissue disruption, implying a role in immune surveillance and cytotoxic activity (Fig. <a href="#Fig3" class="usa-link">3</a>j).</p>
<p id="Par53">Collectively, these results demonstrate that immune cell populations undergo distinct temporal and subtype−specific changes during sciatic nerve repair. The early phase is dominated by Macs and Grans, facilitating debris clearance and acute inflammation. As repair progresses, T cells, DCs, and B cells expand, contributing to adaptive immunity and tissue remodeling. These findings provide insights into the immune landscape following nerve injury, with potential implications for therapeutic strategies targeting immune modulation in peripheral nerve repair.</p></section></section><section id="Sec24"><h3 class="pmc_sec_title">Mo−derived Mac diversification and functional trajectories shape the immune microenvironment during sciatic nerve repair</h3>
<p id="Par54">Macs and Mos play critical roles in the repair process following sciatic nerve injury, orchestrating a dynamic response that evolves over time. At Day 0, Macs are predominantly of the Mac3 subtype, characterized by high expression of genes such as <em>Slco2b1</em>, <em>Cd4</em>, and <em>Selenop</em> (Fig. <a href="#Fig2" class="usa-link">2</a>d, g). Following injury, the immune landscape undergoes significant shifts, with distinct Mac subtypes emerging at different stages.</p>
<p id="Par55">Pseudotime trajectory analysis reveals two distinct differentiation pathways for Mo−derived Macs, both originating from Mos (the root state) and progressing through intermediate stages (Fig. <a href="#Fig4" class="usa-link">4</a>c, f). Trajectory 1 follows a sequential progression whereby Mos transition through early−stage Mac0, intermediate−stage Mac2, and ultimately differentiate into late−stage Mac1 and Mac3 subsets. In Trajectory 2, the cells share the early to intermediate stages (Mac0 <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/3419b6f67981/d33e1669.gif" loading="lazy" id="d33e1669" alt="Inline graphic"></span> Mac2) but diverge in the late phase, giving rise to a mix of Mac1, Mac3, and the proliferative Mac4 subpopulation. These trajectories are accompanied by dynamic transcriptional reprogramming. For instance, genes such as <em>Apoe</em>,<em> Pltp</em>,<em> C1qa</em>,<em> C1qb</em>, and <em>C1qc</em> peak in late−stage Mac of Trajectory 1 and then decline, indicating transient yet essential roles in lipid metabolism, cholesterol efflux, and complement system regulation (Fig. <a href="#Fig4" class="usa-link">4</a>e). In Trajectory 2, the late−phase upregulation of proliferation−associated genes—including <em>Tuba1b</em>,<em> Hmgb2l1</em>,<em> Tubb5</em>, and <em>Cst3</em>—highlights the support for microtubule dynamics, chromatin remodeling, and lysosomal functions in promoting the expansion of the Mac4 subset. The fluctuating expression of additional markers such as <em>Ctsz</em> and <em>Vim</em> further underscores their roles in proteolysis and cytoskeletal reorganization during tissue repair (Fig. <a href="#Fig4" class="usa-link">4</a>h).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374453_12974_2025_3514_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/56f20c1efe69/12974_2025_3514_Fig4_HTML.jpg" loading="lazy" id="d33e1755" height="455" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mo–to–Mac Trajectories and Cell–Cell Interactions. (<strong>a</strong>) UMAP clustering of Mos and Macs with annotated subtypes. (<strong>b</strong>) Chord diagram of CellChat–inferred collagen signaling between Macs and NFs at different time points. Edge thickness: interaction strength; color: sender cluster. (<strong>c</strong>, <strong>f</strong>) Slingshot trajectory analysis showing pseudotime for trajectories 1 and 2. (<strong>d</strong>, <strong>g</strong>) Density plots of Mac and Mo subtypes along each pseudotime trajectory. (<strong>e</strong>, <strong>h</strong>) Smoothed expression dynamics of representative genes along pseudotime (GAM fitting). Each dot represents a single cell, colored by its corresponding time point</p></figcaption></figure><p id="Par57">Finally, cell−cell communication analysis reveals that collagen−mediated signaling between Macs and NFs is significantly enhanced during the repair process (Fig. <a href="#Fig4" class="usa-link">4</a>b). Although this interaction is minimal at Day 0 and absent at Day 1, it intensifies markedly from Day 3 onward. This observation reinforces the significance of collagen in extracellular matrix remodeling and nerve regeneration, as well as the critical interplay between Macs and NFs in facilitating tissue repair. The integration of marker gene analysis with functional and temporal data not only refines our understanding of Macs and Mos heterogeneity but also provides important insights for developing therapeutic strategies aimed at harnessing Mac−mediated regenerative processes.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Temporal dynamics and functional specialization of vascular cell populations during sciatic nerve repair</h3>
<p id="Par58">At Day 0, vascular cell populations were predominantly composed of smooth muscle cells (SMCs) (Fig. <a href="#Fig5" class="usa-link">5</a>a, b). Following nerve injury, vascular cell dynamics underwent significant shifts across different time points. Notably, no vascular cell infiltration was observed on Day 1 post−injury. From Day 3 to Day 14, ECs remained the dominant vascular cell type, with PCs emerging on Day 5 and subsequently maintaining a stable proportion throughout the repair process. A substantial increase in vascular cell numbers was observed from Day 7, showing a rise that persisted in the later stages. In contrast, SMCs and lyECs remained relatively scarce throughout the repair timeline.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374453_12974_2025_3514_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/42dabdfe4c6a/12974_2025_3514_Fig5_HTML.jpg" loading="lazy" id="d33e2046" height="1015" width="718" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Vascular Cell Subtypes and Functional Characteristics. (<strong>a</strong>) UMAP clustering of vascular cells into four types: endothelial cells (ECs), lymphatic ECs (lyECs), pericytes (PCs), and smooth muscle cells (SMCs). (<strong>b</strong>) Bar plot of vascular cell type proportions over time. (<strong>c</strong>–<strong>f</strong>) Subclustering of vascular types. Left: UMAP; Right: subtype proportions over time. (<strong>c</strong>) EC0–EC2. (<strong>d</strong>) lyEC0–lyEC1. (<strong>e</strong>) PC0–PC3. (<strong>f</strong>) SMC0–SMC1. (<strong>g</strong>–<strong>j</strong>) Heatmaps of upregulated markers (Z–score, log) per subtype. (<strong>g</strong>) ECs, (<strong>h</strong>) lyECs, (<strong>i</strong>) PCs, (<strong>j</strong>) SMCs. (<strong>k</strong>–<strong>l</strong>) GO enrichment results. (<strong>k</strong>) EC and lyEC subtypes. (<strong>l</strong>) PC and SMC subtypes</p></figcaption></figure><p id="Par59">EC subtypes exhibited distinct temporal patterns and functional enrichments during nerve repair (Fig. <a href="#Fig5" class="usa-link">5</a>c, g). At Day 0, ECs were rare, and no ECs were detected at Day 1 post−injury. From Day 3 to Day 10, EC0 and EC1 constituted the predominant endothelial populations contributing to repair. By Day 14, EC2 became the dominant subtype. GO analysis revealed functional specialization among these subpopulations (Fig. <a href="#Fig5" class="usa-link">5</a>k). EC0 was enriched in pathways associated with epithelial tube morphogenesis, negative regulation of cell migration, and response to reactive oxygen species, highlighting its role in early−stage tissue remodeling. EC1 displayed enrichment in inflammatory response regulation, nitric oxide metabolism, and maintenance of blood vessel diameter, indicating its involvement in modulating vascular tone and oxidative stress response. EC2, emerging in the later stages, was linked to apoptotic signaling regulation, nucleic acid catabolism, and toll−like receptor signaling, suggesting a role in resolving inflammation and tissue remodeling.</p>
<p id="Par60">lyECs were scarcely present in the normal sciatic nerve, with lyEC1 constituting the primary population (Fig. <a href="#Fig5" class="usa-link">5</a>d, h). Following injury, lyECs began to emerge at Day 5, predominantly consisting of lyEC0, and showed a notable numerical increase by Day 14, suggesting their involvement in the later phases of repair. Marker gene analysis revealed that lyEC0 expressed <em>Plaur</em> [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>], <em>Ccl2</em> [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>], <em>Nr4a3</em> [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>], <em>Hmox1</em> [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>], and <em>Serpine1</em> [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]—genes involved in immune cell recruitment, oxidative stress regulation, and extracellular matrix remodeling (Fig. <a href="#Fig5" class="usa-link">5</a>h, k). These features are consistent with the functional enrichment of lyEC0 in apoptotic signaling regulation, <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e1814.gif" loading="lazy" id="d33e1814" alt="Inline graphic"></span> receptor signaling, and nucleic acid catabolic processes, indicating a role in immune modulation and tissue adaptation during early regeneration. In contrast, lyEC1, although present at baseline, became relatively less prominent post−injury but retained expression of genes such as <em>Cytl1</em>, <em>Slco2a1</em>, <em>Smad4</em>, and <em>Rbp7</em>. These genes are associated with immune signaling (e.g., <em>Cytl1</em> [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>], <em>Slco2a1</em> [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]), transcriptional regulation (<em>Smad4</em> [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]), and metabolic function (<em>Rbp7</em> [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]), suggesting that lyEC1 may contribute to maintaining endothelial homeostasis and modulating repair−associated responses during the late stages of nerve regeneration (Fig. <a href="#Fig5" class="usa-link">5</a>h, k). Both lyEC subtypes showed enrichment in <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e1863.gif" loading="lazy" id="d33e1863" alt="Inline graphic"></span> receptor signaling pathways, reinforcing the pathway’s central importance across endothelial phenotypes. Moreover, cell–cell communication analysis revealed that lyECs maintained robust and persistent interactions with non−immune cell types, including vascular cells, Glis, and NFs, throughout the regenerative timeline (Additional file 3: Fig. <a href="#MOESM3" class="usa-link">S3</a>b). This suggests that lyECs may contribute to coordinating lymphatic remodeling, immune resolution, and stromal stabilization in the evolving nerve microenvironment.</p>
<p id="Par61">PCs exhibited a distinct temporal pattern, with minimal presence at Day 0 (Fig. <a href="#Fig5" class="usa-link">5</a>e). PC populations emerged at Day 5 and followed a dynamic shift in subtypes over time. PC1 was identified as arterial pericytes, and PC3 as proliferative pericytes, each displaying distinct temporal and functional features (Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>d). PC3, the proliferative subset characterized by marker genes such as <em>Esco2</em>, <em>Cdca3</em>, <em>Kif2c</em>, <em>Top2a</em>, and <em>Nusap1</em>, emerged as early as Day 5 and declined progressively, indicating an early role in proliferative expansion, chromosomal segregation, and stress adaptation (Fig. <a href="#Fig5" class="usa-link">5</a>e, i, l). PC1, marked by <em>Pln</em>, <em>Entpd1</em>, <em>Slit3</em>, <em>Col4a5</em>, and <em>Fmod</em>, began to appear from Day 7 and maintained a stable proportion thereafter, consistent with its annotation as arterial pericytes and its role in structural vascular support and vascular tone regulation (Fig. <a href="#Fig5" class="usa-link">5</a>e, i, l). Functionally, PC1 was enriched in muscle cell differentiation, cholesterol metabolism, and viral response pathways, suggesting involvement in both metabolic support and immune modulation (Fig. <a href="#Fig5" class="usa-link">5</a>l). In the early repair phase (Day 5–Day 10), PC0 was the predominant subtype. It expressed <em>RT1−A1</em>, <em>Il33</em>, and <em>Lum</em>, and was enriched in extracellular matrix organization, regulation of blood circulation, and <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e1934.gif" loading="lazy" id="d33e1934" alt="Inline graphic"></span> signaling, implicating it in early vascular stabilization and immune modulation (Fig. <a href="#Fig5" class="usa-link">5</a>e, i, l). By Day 14, PC2 became the dominant population. PC2−specific markers included <em>Adra2b</em>, <em>Ets2</em>, and <em>Tfpi2</em>, with enrichment in apoptotic signaling, nuclear protein import, and pigmentation regulation, suggesting a transition toward late−stage tissue remodeling and homeostatic maintenance (Fig. <a href="#Fig5" class="usa-link">5</a>e, i, l). Furthermore, intercellular communication analysis revealed that PCs maintained close interactions with vascular cells, NFs, and Glis during the repair process. Notably, analysis of NF subtype–PC subtype communication patterns revealed persistent and aberrant activation of the CD99 signaling pathway, with PCs as the sending and NFs as the receiving cells, implying a potential role in pathological remodeling or chronic repair signaling [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] (Additional file 4: Fig. <a href="#MOESM4" class="usa-link">S4</a>c).</p>
<p id="Par62">SMCs were present in very low numbers throughout the repair process, appearing only at Day 7 post−injury (Fig. <a href="#Fig5" class="usa-link">5</a>f, j). SMC0 and SMC1 were identified as the primary subtypes, each displaying distinct functional properties. SMC0 exhibited enrichment in extracellular matrix organization, muscle cell proliferation, nucleocytoplasmic transport, and calcium ion transmembrane transport, suggesting its involvement in vascular support and contractile function (Fig. <a href="#Fig5" class="usa-link">5</a>l). SMC1 shared enrichment in extracellular matrix organization and <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e1972.gif" loading="lazy" id="d33e1972" alt="Inline graphic"></span> signaling but also displayed enrichment in amino acid metabolic processes, potentially linking it to metabolic regulation within the vascular microenvironment during nerve repair.</p>
<p id="Par63">Overall, vascular cell dynamics during sciatic nerve repair revealed a highly coordinated response, with ECs playing a central role in early repair, followed by the recruitment of PCs and the late emergence of SMCs. The functional diversity of these subtypes suggests specialized roles in angiogenesis, inflammatory modulation, and extracellular matrix remodeling, highlighting their critical contributions to the regeneration process.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Dynamic shifts in NF subpopulations reveal phase−specific roles in nerve regeneration</h3>
<p id="Par64">Dynamic changes in NF subpopulations were observed throughout the sciatic nerve repair process, highlighting their diverse functional roles in different phases of regeneration (Fig. <a href="#Fig6" class="usa-link">6</a>a−c). At Day 0, NF2 cells, identified by marker genes <em>Crispld2</em>,<em> Sqle</em>,<em> Aldh1a1</em>,<em> Idi1</em>,<em> Ralgps2</em>,<em> Kcnk2</em>,<em> Myoc</em>,<em> Cttnbp2</em>,<em> Col9a1</em>, and <em>Col9a2</em>, were the predominant population (Fig. <a href="#Fig6" class="usa-link">6</a>d). These cells were classified as endoneurial mesenchymal cells and were primarily involved in extracellular matrix organization and <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2092.gif" loading="lazy" id="d33e2092" alt="Inline graphic"></span> receptor superfamily signaling pathways, supporting the structural integrity of the nerve and facilitating initial cellular responses post−injury (Fig. <a href="#Fig6" class="usa-link">6</a>e and Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>b).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374453_12974_2025_3514_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/a1c592bd584c/12974_2025_3514_Fig6_HTML.jpg" loading="lazy" id="d33e2132" height="1016" width="744" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Neurofibroblast Subtype Dynamics During Nerve Repair. (<strong>a</strong>) UMAP of NFs across time points. (<strong>b</strong>) Left: Subtype clustering of NF0–NF6. Right: Temporal distribution of each subtype. (<strong>c</strong>) Proportional changes in NF subtypes post–injury. (<strong>d</strong>) Heatmap of upregulated marker genes (Z–score, log). (<strong>e</strong>) GO enrichment of NF subtypes. (<strong>f</strong>) GO enrichment of glial subtypes (Gli0–Gli5) for comparison</p></figcaption></figure><p id="Par66">In the early inflammatory phase (Day 1 to Day 3), NF5 cells, characterized by <em>Ncapg</em>,<em> Tpx2</em>,<em> Hmmr</em>,<em> Ect2</em>,<em> Kif4a</em>,<em> Cenpu</em>,<em> Cenpf</em>,<em> Sgo2</em>,<em> Pclaf</em>, and <em>Kif20b</em>, exhibited significant expansion (Fig. <a href="#Fig6" class="usa-link">6</a>c, d). These cells, categorized as proliferating mesenchymal cells, played a critical role in chromosome segregation and non−membrane−bounded organelle assembly (Fig. <a href="#Fig6" class="usa-link">6</a>e and Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>b) [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. Their increased activity suggested a surge in cell proliferation, likely contributing to the rapid remodeling of the extracellular environment to accommodate immune cell infiltration and debris clearance [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>].</p>
<p id="Par67">By Day 5, NF0 cells, defined by <em>Spon1</em>, displayed a notable peak, surpassing their levels at Day 3 and Day 7 (Fig. <a href="#Fig6" class="usa-link">6</a>c, d). NF0 cells, along with NF1 and NF3, were classified as differentiating mesenchymal cells (Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>b). NF0 cells were specifically enriched in pathways related to extracellular matrix organization, <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2202.gif" loading="lazy" id="d33e2202" alt="Inline graphic"></span> receptor signaling and nuclear receptor−mediated signaling (Fig. <a href="#Fig6" class="usa-link">6</a>e). The increased presence of NF0 at this stage indicated their crucial role in transitioning from the inflammatory to the regenerative phase, facilitating extracellular matrix remodeling and tissue stabilization. Simultaneously, NF4 cells, characterized by the expression of <em>Lrg1</em> and <em>Plvap</em>—genes commonly implicated in vascular biology [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]—were enriched in collagen fibril organization and glycolytic pathways, indicating a role in matrix remodeling and metabolic adaptation essential for repair (Fig. <a href="#Fig6" class="usa-link">6</a>c−e) [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>].</p>
<p id="Par68">During the mid−to−late regenerative phase (Day 7 to Day 14), NF0 cells continued to increase gradually, whereas NF3 cells, marked by <em>Sbsn</em>,<em> Cdkn2a</em>,<em> Bhlhe22</em>,<em> Apod</em>,<em> Rdh10</em>,<em> Slc16a11</em>,<em> A2m</em>,<em> Scn3b</em>,<em> Plcxd3</em>, and <em>Mrap2</em>, showed a progressive decline (Fig. <a href="#Fig6" class="usa-link">6</a>c, d). NF3 cells were involved in epithelial cell proliferation regulation and <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2269.gif" loading="lazy" id="d33e2269" alt="Inline graphic"></span> receptor signaling, suggesting their involvement in early repair mechanisms that diminished as regeneration progressed (Fig. <a href="#Fig6" class="usa-link">6</a>e). These cells played a key role in the final stages of repair by contributing to the reconstruction of the extracellular matrix and supporting the structural and functional restoration of the nerve tissue [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>].</p>
<p id="Par69">Overall, the dynamic shifts in NF subpopulations underscore their essential contributions to different phases of nerve repair. The transition from NF5−driven proliferative responses to NF4−mediated metabolic transformation and NF0−associated extracellular matrix remodeling highlights the coordinated interplay of these NF subsets. Their involvement in key signaling pathways, particularly <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2288.gif" loading="lazy" id="d33e2288" alt="Inline graphic"></span> receptor superfamily signaling [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>], suggests that modulating these pathways could be a potential therapeutic strategy to enhance nerve regeneration.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Dynamic heterogeneity and temporal fate transitions of Glis during sciatic nerve regeneration</h3>
<p id="Par70">Dynamic changes in Gli populations were observed during the repair process following sciatic nerve transection (Fig. <a href="#Fig7" class="usa-link">7</a>a−c). At Day 0, the predominant Gli subtype was Gli5, which represents myelinating Schwann cells (Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>c). Following nerve injury, Glis underwent significant phenotypic transitions, with Gli0, identified as repair−associated Schwann cells, emerging as the dominant subtype during the early phase of regeneration (Fig. <a href="#Fig7" class="usa-link">7</a>c and Additional file 2: Fig. <a href="#MOESM2" class="usa-link">S2</a>c). As the repair process progressed, Gli0 cells gradually declined, while Gli2 cells increased, suggesting a shift toward remyelination and structural recovery (Fig. <a href="#Fig7" class="usa-link">7</a>c). Gli0, Gli1, and Gli4 contribute to nerve regeneration by engaging the <em>TGF</em>−<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2320.gif" loading="lazy" id="d33e2320" alt="Inline graphic"></span> receptor superfamily signaling pathway to modulate cellular responses, orchestrating extracellular matrix organization to reshape the tissue microenvironment, and regulating axon ensheathment to support the restoration of nerve function (Fig. <a href="#Fig6" class="usa-link">6</a>f).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374453_12974_2025_3514_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/c952da3135d5/12974_2025_3514_Fig7_HTML.jpg" loading="lazy" id="d33e2369" height="845" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Glial Cell Subtype Differentiation and Signaling Activity. (<strong>a</strong>) UMAP of Glis across time points. (<strong>b</strong>) Left: Subtype clustering of Gli0–Gli5. Right: Distribution over time. (<strong>c</strong>) Changes in Gli subtype proportions post–injury. (<strong>d</strong>) Heatmap of upregulated markers (Z–score, log). (<strong>e</strong>) Pseudotime trajectories from Day 5–14 identifying two distinct differentiation paths. (<strong>f</strong>) Heatmaps of variable gene expression along each trajectory. (<strong>g</strong>) Density plots of cell distributions along trajectories. (<strong>h</strong>) Bubble plots of pathway activation over pseudotime. (<strong>i</strong>) CellChat chord diagram of PTN signaling from Gli to NF cells. (<strong>j</strong>) Bubble plot showing predicted Gli–to–NF signaling interactions by pathway</p></figcaption></figure><p id="Par72">Single−cell transcriptomic analysis identified distinct marker genes associated with different Gli subtypes (Fig. <a href="#Fig7" class="usa-link">7</a>d). Gli5, characteristic of myelinating Schwann cells, expressed markers such as <em>Ncmap</em>,<em> Sema5a</em>,<em> Mt1 and Kcna1</em>. In contrast, Gli0 (repair Schwann cells) exhibited increased expression of genes such as <em>Clcf1</em>,<em> Met</em>,<em> Artn</em>,<em> and Runx2</em>, which are associated with regeneration and extracellular matrix reorganization (Fig. <a href="#Fig6" class="usa-link">6</a>f and <a href="#Fig7" class="usa-link">7</a>d). Gli3 (proliferating Schwann cells) exhibited high expression of <em>Ube2c</em>,<em> Kif14</em>,<em> Kif4a</em> and <em>Plk1</em>, indicating active cell cycle progression and proliferation (Fig. <a href="#Fig6" class="usa-link">6</a>f and <a href="#Fig7" class="usa-link">7</a>d). The transition toward remyelination was marked by the emergence of Gli2, which expressed genes such as <em>Nefm</em>,<em> Cuedc2</em>,<em> Cldn19</em>, and <em>Mag</em>, indicating a functional shift toward axon ensheathment and nerve fiber stabilization [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>] (Fig. <a href="#Fig6" class="usa-link">6</a>f and <a href="#Fig7" class="usa-link">7</a>d).</p>
<p id="Par73">Functional enrichment analysis further elucidated the biological roles of different Gli subtypes during the repair process (Fig. <a href="#Fig6" class="usa-link">6</a>f). Gli0 was associated with pathways involved in <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2456.gif" loading="lazy" id="d33e2456" alt="Inline graphic"></span> receptor signaling, neuron projection guidance, and glycoprotein metabolism, all of which are critical for early nerve repair. Gli1, which plays a role in extracellular matrix remodeling, was enriched for pathways related to <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2464.gif" loading="lazy" id="d33e2464" alt="Inline graphic"></span> signaling, peptide cross−linking, and interferon−mediated signaling. Gli2 exhibited functional enrichment in myelination−related processes, including amine transport and apical protein localization, consistent with its role in late−stage nerve repair. Gli3 was primarily associated with cell cycle regulation, with enrichment in chromosome segregation and organelle assembly. Gli4 was linked to extracellular matrix organization and nuclear receptor−mediated signaling, suggesting involvement in tissue remodeling. Finally, Gli5, as the mature myelinating Schwann cell population, was enriched for pathways regulating axon ensheathment, potassium ion transport, and sterol biosynthesis, all of which are crucial for maintaining functional nerve architecture.</p>
<p id="Par74">Pseudotime trajectory analysis revealed two major regenerative pathways governing Gli fate transitions (Fig. <a href="#Fig7" class="usa-link">7</a>e−h and Additional file 7: Fig. <a href="#MOESM7" class="usa-link">S7</a>a−b). The first trajectory was characterized by increased expression of genes associated with myelination, proliferation, autophagy, and metabolic regulation, including <em>Col3a1</em>,<em> Csrp2</em>,<em> Mbp</em>,<em> Mpz</em>,<em> posten</em>, and <em>Pmp22</em> (Additional file 7: Fig. <a href="#MOESM7" class="usa-link">S7</a>b). These genes were upregulated during later stages of repair, highlighting their role in structural restoration [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>–<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. The second trajectory was primarily associated with growth factor secretion and extracellular matrix modulation, with early upregulation of genes such as <em>Apod</em>,<em> Apoe</em>,<em> Col1a1</em>, and <em>Fn1</em> (Additional file 7: Fig. <a href="#MOESM7" class="usa-link">S7</a>b) [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>, <a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. The dynamic interplay between these two pathways underscores the complex cellular mechanisms governing Gli function during nerve regeneration.</p>
<p id="Par75">Additionally, cell−cell communication analysis indicated that Glis actively interacted with NFs during repair, particularly through enhanced PTN signaling (Fig. <a href="#Fig7" class="usa-link">7</a>i and Additional file 7: Fig. <a href="#MOESM7" class="usa-link">S7</a>c), which is consistent with the known roles of PTN in promoting neurite outgrowth, axon regeneration, and cell–cell communication in the nervous system [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. This interaction was not only observed during nerve regeneration but has also been implicated in pathological conditions such as neurofibromatosis, where Gli−NF communication is dysregulated (Fig. <a href="#Fig7" class="usa-link">7</a>j and Additional file 8: Fig. <a href="#MOESM8" class="usa-link">S8</a>) [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. Collectively, these findings provide a comprehensive understanding of Gli heterogeneity, their temporal dynamics, and their critical roles in coordinating nerve repair following injury.</p></section><section id="Sec28"><h3 class="pmc_sec_title">
<em>TGF</em>−<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2555.gif" loading="lazy" id="d33e2555" alt="Inline graphic"></span> signaling exhibits sustained and spatially distinct activation during nerve repair</h3>
<p id="Par76">To validate the cell−type−specific <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2565.gif" loading="lazy" id="d33e2565" alt="Inline graphic"></span> signaling activation observed in our single−cell transcriptomic analysis, we conducted a complementary bulk RNA−seq experiment. We collected 2−mm proximal and distal segments of the sciatic nerve at Day 1, 3, 5, 7, and 14 post−transection, along with uninjured control nerves, and performed transcriptome−wide differential expression analysis (Fig. <a href="#Fig8" class="usa-link">8</a>a). Across all time points, proximal and distal segments displayed highly similar gene expression dynamics relative to uninjured controls.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374453_12974_2025_3514_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/748769b6f7c8/12974_2025_3514_Fig8_HTML.jpg" loading="lazy" id="d33e2643" height="840" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Region – and Time–Resolved Bulk Transcriptomic Profiling of <em>TGF–</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2584.gif" loading="lazy" id="d33e2584" alt="Inline graphic"></span> signaling after nerve transection. (<strong>a</strong>) Scatter plots comparing log₂FC in proximal vs. distal segments (relative to uninjured controls) at Days 1, 3, 5, 7, and 14. Red: genes upregulated in both; green: downregulated in both. Labeled genes are <em>TGF–</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2595.gif" loading="lazy" id="d33e2595" alt="Inline graphic"></span>–related and significantly changed in both regions. (<strong>b</strong>) GO enrichment of genes from (<strong>a</strong>), categorized as shared, proximal–specific, or distal–specific upregulated genes. (<strong>c</strong>) GSEA heatmap of normalized enrichment scores (NES) for <em>TGF–</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2613.gif" loading="lazy" id="d33e2613" alt="Inline graphic"></span>–related pathways across regions and time points. (<strong>d</strong>) Heatmap of key <em>TGF–</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2624.gif" loading="lazy" id="d33e2624" alt="Inline graphic"></span> signaling components (ligands, receptors, mediators, targets) across regions and time points (Z–score, log). (<strong>e</strong>) Immunofluorescence staining of <em>TGF–</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2636.gif" loading="lazy" id="d33e2636" alt="Inline graphic"></span> in uninjured and injured nerves (Days 3–14), showing spatial–temporal changes in protein expression. Scale bar: 500 μm</p></figcaption></figure><p id="Par78">Enrichment analysis of shared DEGs revealed consistent activation of the <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2649.gif" loading="lazy" id="d33e2649" alt="Inline graphic"></span> signaling pathway throughout the regeneration process (Fig. <a href="#Fig8" class="usa-link">8</a>b). Notably, genes specifically upregulated in the proximal segment at Day 14 remained significantly enriched for <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2660.gif" loading="lazy" id="d33e2660" alt="Inline graphic"></span> signaling, whereas distal–specific genes did not, suggesting more sustained transcriptional activation in the proximal region. GSEA analysis further highlighted this spatial distinction: the distal nerve segment exhibited strong early activation of <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2668.gif" loading="lazy" id="d33e2668" alt="Inline graphic"></span> signaling at Day 1, which decreased over time, while the proximal segment maintained elevated activity across later time points (Fig. <a href="#Fig8" class="usa-link">8</a>c).</p>
<p id="Par79">This pattern was partially reflected at the protein level. Immunofluorescence staining revealed robust <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2682.gif" loading="lazy" id="d33e2682" alt="Inline graphic"></span> upregulation in both proximal and distal segments post–injury, with expression persisting longer in the proximal region (Fig. <a href="#Fig8" class="usa-link">8</a>e). However, transcriptomic heatmaps of canonical <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2693.gif" loading="lazy" id="d33e2693" alt="Inline graphic"></span> pathway components—including ligands (<em>Tgfb1</em>,<em> Tgfb2</em>), receptors (<em>Tgfbr1</em>,<em> Tgfbr2</em>), intracellular mediators (<em>Smad2</em>,<em> Smad3</em>,<em> Smad4</em>), and downstream targets (<em>Serpine1</em>,<em> Ctgf</em>,<em> Fn1</em>,<em> Col1a1</em>,<em> Acta2</em>)—revealed a noteworthy pattern: by Day 14, distal segments exhibited higher mRNA expression of many <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2740.gif" loading="lazy" id="d33e2740" alt="Inline graphic"></span>–related genes compared to the proximal side (Fig. <a href="#Fig8" class="usa-link">8</a>d). This discrepancy between mRNA and protein levels likely reflects region – and phase–specific post–transcriptional regulation and feedback control mechanisms, suggesting that <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2751.gif" loading="lazy" id="d33e2751" alt="Inline graphic"></span> signaling persists transcriptionally in the distal stump even after protein–level activity has diminished.</p>
<p id="Par80">Together, these findings integrate protein and transcriptomic dynamics to reveal a complex spatiotemporal regulation of <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2761.gif" loading="lazy" id="d33e2761" alt="Inline graphic"></span> signaling, involving both early and sustained activation in different compartments. Moreover, enrichment of “negative regulation of <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2769.gif" loading="lazy" id="d33e2769" alt="Inline graphic"></span> signaling” at Day 1 in the distal stump and Day 7 in the proximal stump suggests location – and phase–specific feedback modulation. These bulk transcriptomic results are highly consistent with our single–cell data, which showed widespread <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2777.gif" loading="lazy" id="d33e2777" alt="Inline graphic"></span> pathway activity across major cell types including Glis, NFs, and immune cells, and align with previous reports highlighting the essential role of <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2786.gif" loading="lazy" id="d33e2786" alt="Inline graphic"></span> signaling in orchestrating inflammation, extracellular matrix remodeling, and cellular differentiation during peripheral nerve repair [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>, <a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>].</p></section><section id="Sec29"><h3 class="pmc_sec_title">Divergent cellular and molecular repair programs in crush versus transection models of sciatic nerve injury</h3>
<p id="Par81">The comparative analysis of crush and transection injuries revealed distinct cellular responses and molecular pathways involved in the repair process (Fig. <a href="#Fig9" class="usa-link">9</a>). In the early stages following crush injury, Macs were the predominant immune cell type, maintaining a stable presence from Day 1 to Day 7 (Fig. <a href="#Fig9" class="usa-link">9</a>b−e). In contrast, in transection injury, Mac presence was more pronounced in the early phases, with a higher proportion on Day 1 and Day 3 compared to crush injury. However, by Day 7, the proportion of Macs had significantly declined. Additionally, Gran were more abundant in the early stages of transection injury (Day 1 and Day 3) than in crush injury, but their presence diminished markedly by Day 7.</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374453_12974_2025_3514_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f4a74db6cb8f/12974_2025_3514_Fig9_HTML.jpg" loading="lazy" id="d33e2851" height="742" width="669" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative Analysis of Crush vs. Transection Injuries in Single–Cell Resolution. (<strong>a</strong>) UMAP of cells from sciatic nerve crush injury (SNCI) and transection injury (SNTI) at Days 0, 1, 3, 7. (<strong>b</strong>) Cell type annotation using SingleR with the Injured Sciatic Nerve Atlas (iSNAT) as reference. (<strong>c</strong>–<strong>d</strong>) UMAPs of SNCI (<strong>c</strong>) and SNTI (<strong>d</strong>) separately, colored by cell type. (<strong>e</strong>) Bar plot comparing cell type proportions between SNCI and SNTI at each time point. (<strong>f</strong>) Scatter plots of DEGs (log₂FC &gt; ± 1, FDR &lt; 0.05) in SNCI vs. SNTI at Days 1, 3, 7. Red: downregulated in both; green: upregulated in both. (<strong>g</strong>) GO enrichment of commonly upregulated genes (green in <strong>f</strong>). (<strong>h</strong>) GO enrichment of injury–specific genes (log₂FC &gt; 1 in only SNCI or SNTI)</p></figcaption></figure><p id="Par83">Schwann cell dynamics also varied between the two injury models (Fig. <a href="#Fig9" class="usa-link">9</a>b−e). In crush injury, Schwann cells progressively increased from Day 1 to Day 7, contributing to nerve regeneration. Conversely, in transection injury, Schwann cells were scarce in the early stages (Day 1 and Day 3), with a notable presence only emerging by Day 7. These findings suggest that Schwann cell recruitment and proliferation are delayed in transection injury, potentially affecting the overall repair process.</p>
<p id="Par84">Mesenchymal stem cells exhibited a distinct pattern in both models (Fig. <a href="#Fig9" class="usa-link">9</a>b−e). In crush injury, they were continuously present throughout the repair timeline, suggesting a sustained role in tissue remodeling and repair. In contrast, in transection injury, mesenchymal stem cells were present at lower levels during the early stages (Day 1 and Day 3) but showed a substantial increase by Day 7, indicating a delayed yet significant involvement in the repair process.</p>
<p id="Par85">The enrichment analysis of shared upregulated genes between crush and transection injuries highlighted several key pathways (Fig. <a href="#Fig9" class="usa-link">9</a>g). On Day 1, common pathways included leukocyte migration, positive regulation of cytosolic calcium ion concentration, interferon−mediated signaling, and toll−like receptor signaling. By Day 3, pathways related to chromosome segregation, multicellular homeostasis, and M phase regulation were enriched, reflecting active cell division and immune responses. On Day 7, pathways such as chromosome segregation and structural disruption in another organism remained prevalent, indicating ongoing cellular proliferation and inflammatory responses.</p>
<p id="Par86">In contrast, pathways specifically enriched in transection injury revealed distinct molecular mechanisms (Fig. <a href="#Fig9" class="usa-link">9</a>h). On Day 1, pathways such as regulation of angiogenesis, extracellular matrix organization, inflammatory response modulation, and neuron apoptotic processes were significantly enriched, suggesting a robust early vascular and immune response. By Day 3, <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2875.gif" loading="lazy" id="d33e2875" alt="Inline graphic"></span> receptor signaling and potassium ion transport were notably active, highlighting their role in tissue remodeling. On Day 7, pathways involved in cytosolic calcium ion regulation, interleukin−1 response, and fatty acid biosynthesis were enriched, indicating a shift toward metabolic and immune modulation during later repair phases.</p>
<p id="Par87">These findings underscore the fundamental differences between crush and transection injuries in terms of immune cell recruitment, Schwann cell involvement, and mesenchymal stem cell dynamics. The enrichment analysis further elucidates the distinct molecular pathways governing nerve repair in each model, with transection injury demonstrating a more complex and delayed regenerative process. The differential activation of immune and metabolic pathways suggests potential therapeutic targets to enhance nerve regeneration in severe injury conditions.</p></section></section><section id="Sec30"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par88">The present study integrates our single−cell transcriptomic findings into a comprehensive narrative of sciatic nerve repair post−transection, revealing a dynamic, multi−phasic process that diverges from traditional theories of nerve regeneration. In contrast to classical views that primarily emphasize Wallerian degeneration and subsequent axonal regrowth, our results demonstrate that multiple cell types participate in a temporally coordinated response. Notably, the immediate post−injury phase (Day 1–3) is marked by a significant influx of immune cells. During this critical window, distinct Mac subsets emerge—characterized by the expression of <em>Slpi</em> [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>] and <em>Ass1</em> [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]—alongside Gran identifiable by markers such as <em>S100a8/</em>9 [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>], <em>Anxa1</em> [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>], <em>Mmp8</em> [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>], and <em>Pglyrp1</em> [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. These observations concur with earlier studies indicating that early immune activation is indispensable for efficient debris clearance via phagocytosis and proteolytic mechanisms.</p>
<p id="Par89">As repair commences, the inflammatory landscape gradually shifts. Immune cells, which dominate the early stages with over 95% of the cell population by Day 1, give way to a robust re−emergence of NFs and Glis. Detailed subclustering of NF populations reveals that the NFs, which are predominant in the uninjured nerve, begin to undergo significant reprogramming post−injury. By Day 5, a specialized NF subpopulation—NF4, distinguished by high <em>Lrg1</em> [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>, <a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>] and <em>Plvap</em> [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>, <a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>] expression—appears. This NF4 subset is hypothesized to be a pivotal intermediary, potentially serving as a key transitional cell type that integrates signals from the early immune response and fosters the subsequent emergence of mature fibroblasts, Glis, and vascular cells. Such a role is consistent with the observation that NF4 marks the transformation of NFs from a proliferative state towards a more differentiated, mature phenotype as part of the evolving tissue repair process. In parallel, NF0 cells—marked by Spon1 and peaking at Day 5—play a distinct and essential role in remodeling the extracellular matrix and stabilizing the regenerating tissue microenvironment. These cells are classified as differentiating mesenchymal cells and are specifically enriched in pathways related to extracellular matrix organization, <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2948.gif" loading="lazy" id="d33e2948" alt="Inline graphic"></span> receptor signaling, and nuclear receptor−mediated transcriptional regulation. Unlike NF4, which is characterized by metabolic adaptation and vascular−associated gene expression, NF0 cells contribute to structural remodeling and act as mediators in the transition from inflammatory response to regenerative stabilization. Their gradual increase during the mid−to−late phase of repair (Day 7 to Day 14) further highlights their sustained involvement in tissue reorganization and suggests that NF0 fibroblasts may act as a scaffold−forming subpopulation that collaborates with other cell types to reestablish nerve architecture. Together, the complementary roles of NF0 and NF4 underscore the temporal coordination between ECM remodeling and metabolic reprogramming during sciatic nerve repair, revealing the functional heterogeneity within the NF lineage and emphasizing the potential of NF0 as a therapeutic target for enhancing matrix regeneration.</p>
<p id="Par90">Concurrently, Glis exhibit significant phenotypic transitions that underscore their essential role in restoring nerve function. In uninjured tissue, myelinating Schwann cells dominate; however, following injury, repair−associated subtypes rapidly emerge. The initial surge of Gli0 cells, characterized by the upregulation of regeneration−associated genes such as <em>Clcf1</em> [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>] and <em>Runx</em> [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>, <a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>], facilitates early axonal guidance and extracellular matrix reorganization. As regeneration progresses, a shift occurs with the increase of Gli2 cells, which express markers including <em>Mag</em> [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>] and <em>Cldn19</em> [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>], signifying a transition towards remyelination and stabilization of the axonal architecture. This sequential switch—from an initial repair−focused profile to one committed to remyelination—underscores a dynamic process in which Gli functionality is finely tuned to meet the evolving requirements of the regenerating nerve.</p>
<p id="Par91">Vascular remodeling also plays a critical role in the repair process. Although vascular cells are relatively sparse in the immediate aftermath of injury, ECs begin to accumulate from Day 3 onward, and by Day 7, there is a dramatic surge in PC numbers. The robust intercellular communication between Glis and vascular cells, coupled with the emergence of PC subsets involved in <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e2988.gif" loading="lazy" id="d33e2988" alt="Inline graphic"></span> signaling, establishes a supportive microenvironment essential for angiogenesis. Such vascular expansion not only restores blood flow but also creates a niche that is vital for sustaining axonal regrowth and overall tissue homeostasis.</p>
<p id="Par92">Moreover, the trajectory analysis of Mo−derived Macs reveals distinct differentiation pathways that are intimately linked with the regenerative process. Early Mac subsets engaged in inflammatory clearance gradually give way to populations involved in tissue remodeling and extracellular matrix deposition. The interplay between these immune cells and NFs, mediated by enhanced collagen signaling, highlights an integrated network that regulates both inflammation resolution and tissue repair.</p>
<p id="Par93">To further validate the cell type–specific <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e3000.gif" loading="lazy" id="d33e3000" alt="Inline graphic"></span> signaling activation observed in our scRNA−seq analysis, we performed bulk RNA−seq on anatomically defined sciatic nerve segments. Consistent with our single−cell findings, we observed sustained and spatially distinct activation of <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e3008.gif" loading="lazy" id="d33e3008" alt="Inline graphic"></span> signaling pathways, particularly in the proximal stump. Enrichment analysis and GSEA revealed prolonged <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e3016.gif" loading="lazy" id="d33e3016" alt="Inline graphic"></span> activity at Day 14 in proximal regions, whereas distal segments showed a more transient response. This spatial asymmetry may reflect differing regenerative capacities or mechanical environments across nerve segments and supports the notion that proximal Glis and NFs maintain a prolonged pro−regenerative state. These bulk transcriptomic results provide orthogonal validation of the single−cell data and underscore <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e3025.gif" loading="lazy" id="d33e3025" alt="Inline graphic"></span> signaling as a key regulator orchestrating extracellular matrix remodeling, immune modulation, and Gli reprogramming during peripheral nerve repair.</p>
<p id="Par94">In summary, our time−resolved single−cell atlas of sciatic nerve transection injury reveals a coordinated, multi−phasic repair program that progresses through three principal biological phases: early immune activation, extracellular matrix remodeling, and Schwann cell−driven remyelination. Initially, the robust infiltration of specialized Macs and Grans not only facilitates debris clearance but also establishes a pro−regenerative cytokine environment. This is followed by a transitional phase marked by the emergence of NF4 fibroblasts and proliferative mesenchymal subsets, which remodel the extracellular matrix through <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e3035.gif" loading="lazy" id="d33e3035" alt="Inline graphic"></span> and collagen−related signaling, setting the foundation for tissue repair. In the later phase, Glis exhibit dynamic fate transitions—from repair−associated Gli0 to myelinating Gli2 subtypes—underscoring their essential role in axonal ensheathment and functional restoration.</p>
<p id="Par95">Importantly, our cross−species integration with human neurofibroma data highlights conserved PTN signaling between NFs and Glis, implicating a broader relevance of the NF–Gli axis in both regenerative and pathological contexts. <em>PTN</em> exerts its effects by binding to receptors such as <em>PTP1</em> and <em>ALK</em>, thereby moduating signaling pathways involved in neuronal migration, survival, and extracellular matrix remodeling [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. Its significant upregulation following peripheral nerve injury further supports its critical role in orchestrating a pro−regenerative microenvironment [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>]. Moreover, by comparing crush and transection injury models, we demonstrate that the delayed engagement of Schwann cells and mesenchymal subtypes in transection injury likely contributes to impaired regeneration, pointing to a narrower therapeutic window for intervention.</p>
<p id="Par96">Together, these findings provide not only a granular understanding of the cellular and molecular mechanisms orchestrating nerve repair, but also identify temporal checkpoints—such as early Mac heterogeneity, mid−phase NF4 activation, and late Schwann cell remyelination—as actionable targets for stage−specific therapeutic modulation. Notably, the sustained and region−specific activation of <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e3062.gif" loading="lazy" id="d33e3062" alt="Inline graphic"></span> signaling revealed by both single−cell and bulk transcriptomic analyses highlights this pathway as a unifying axis that coordinates immune regulation, extracellular matrix remodeling, and Schwann cell fate transitions. This work lays a foundation for developing time−tuned regenerative therapies tailored to the specific needs of each phase of peripheral nerve repair.</p></section><section id="Sec31"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par97">In conclusion, our study establishes a comprehensive, time−resolved single−cell atlas of sciatic nerve transection injury, revealing three sequential and highly coordinated regenerative phases. In the acute phase (within 24 h), the lesion environment is dominated by infiltration of pro−inflammatory Macs and Grans, alongside the expansion of proliferative mesenchymal fibroblasts (NF5), which jointly initiate debris clearance and pro−regenerative cytokine signaling. During the intermediate phase (Days 3–7), activated fibroblasts (NF4/NF0) and repair Schwann cells (Gli0) drive extracellular matrix remodeling and reinitiate axonal guidance, largely via <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e3074.gif" loading="lazy" id="d33e3074" alt="Inline graphic"></span> and collagen signaling pathways. In the late phase (Day 7 onward), Glis progressively transition into myelinating phenotypes (Gli2/5), vascular cells expand to re−establish perfusion, and Macs adopt resolution−associated profiles (Mac3/4), culminating in the reconstruction of nerve architecture by Day 14. Importantly, comparative analysis with crush injury revealed that transection triggers a more intense early immune response and delays Schwann cell remyelination, narrowing the therapeutic window for intervention. Notably, bulk RNA−seq of proximal and distal nerve segments confirmed the spatially sustained activation of <em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e3082.gif" loading="lazy" id="d33e3082" alt="Inline graphic"></span> signaling, reinforcing its central role in orchestrating the immune–stromal–glial axis during regeneration. These findings define temporal checkpoints and molecular targets—such as early Mac heterogeneity, mid−phase NF activation, and late Gli differentiation—that may guide the development of time−adapted regenerative strategies for peripheral nerve injury.</p></section><section id="Sec32"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (3.5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (1.4MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (1.7MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (2.6MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM5_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (2.9MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM6_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (2.6MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM7_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7</a><sup> (2.1MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM8_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8</a><sup> (2.8MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM9_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9</a><sup> (2.4MB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Beijing Capital Biotechnology Co., Ltd. for assistance with single−cell RNA sequencing. We also acknowledge the use of publicly available datasets (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE198582" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE198582</a>, human neurofibroma scRNA−seq data) which greatly supported the cross−validation and annotation processes. The authors thank the developers of CellChat, Monocle, Seurat, and related tools for their contributions to open science.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>B</dt>
<dd><p id="Par100">B cell</p></dd>
<dt>BSA</dt>
<dd><p id="Par101">Bovine serum albumin</p></dd>
<dt>DAPI</dt>
<dd><p id="Par102">4’,6−diamidino−2−phenylindole</p></dd>
<dt>DC</dt>
<dd><p id="Par103">Dendritic cell</p></dd>
<dt>DEG</dt>
<dd><p id="Par104">Differentially expressed gene</p></dd>
<dt>DGE</dt>
<dd><p id="Par105">Differential gene expression</p></dd>
<dt>EC</dt>
<dd><p id="Par106">Endothelial cell</p></dd>
<dt>Gli</dt>
<dd><p id="Par107">Glial cell</p></dd>
<dt>GO</dt>
<dd><p id="Par108">Gene Ontology</p></dd>
<dt>Gran</dt>
<dd><p id="Par109">Granulocyte</p></dd>
<dt>GSEA</dt>
<dd><p id="Par110">Gene Set Enrichment Analysis</p></dd>
<dt>KEGG</dt>
<dd><p id="Par111">Kyoto Encyclopedia of Genes and Genomes</p></dd>
<dt>lyEC</dt>
<dd><p id="Par112">Lymphatic endothelial cells</p></dd>
<dt>Mac</dt>
<dd><p id="Par113">Macrophage</p></dd>
<dt>Mo</dt>
<dd><p id="Par114">Monocyte</p></dd>
<dt>NF</dt>
<dd><p id="Par115">Neurofibroblast</p></dd>
<dt>NK</dt>
<dd><p id="Par116">Natural killer cell</p></dd>
<dt>PBS</dt>
<dd><p id="Par117">Phosphate−buffered saline</p></dd>
<dt>PC</dt>
<dd><p id="Par118">Pericyte</p></dd>
<dt>PCA</dt>
<dd><p id="Par119">Principal component analysis</p></dd>
<dt>scRNA−seq</dt>
<dd><p id="Par120">Single−cell RNA sequencing</p></dd>
<dt>SMC</dt>
<dd><p id="Par121">Smooth muscle cell</p></dd>
<dt>SNN</dt>
<dd><p id="Par122">Shared nearest neighbor</p></dd>
<dt>T</dt>
<dd><p id="Par123">T cell</p></dd>
<dt>
<em>TGF−</em><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9667/12374453/f5e0e8aa5fab/d33e3244.gif" loading="lazy" id="d33e3244" alt="Inline graphic"></span>
</dt>
<dd><p id="Par124">Transforming growth factor beta</p></dd>
<dt>UMAP</dt>
<dd><p id="Par125">Uniform manifold approximation and projection</p></dd>
<dt>UMI</dt>
<dd><p id="Par126">Unique molecular identifier</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Yiben Ouyang, Mingqian Yu, Tieyuan Zhang, and Haofeng Cheng contributed equally to this work. The study was conceptualized and designed by Yiben Ouyang, Jiang Peng and Yu Wang. Methodology development, animal experiments, and immunofluorescence staining were performed by Tieyuan Zhang, Haolin Liu, Haofeng Cheng, Liang Zuo, and Yiben Ouyang. Single-cell library preparation and sequencing were carried out by Yanjun Guan and Sice Wang. Data processing and formal analysis were conducted by Mingqian Yu, Ao Liu, Ruichao He, Xiaoyang Fu, and Jiajie Chen, with bioinformatics and pseudotime trajectory analyses by Mingqian Yu, Yixiao Tan, Yuhui Cui, Junli Wang, and Yiben Ouyang. Cell–cell communication and enrichment analyses were performed by Jinjuan Zhao, Ao Liu, Xiaochun Zhang and Tianqi Su, and visualization was produced by Mingqian Yu and Yiben Ouyang. The original draft was written by Yiben Ouyang and Mingqian Yu, and all authors contributed to review and editing. Supervision and funding acquisition were provided by Jiang Peng and Yu Wang.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was funded by the Key Technologies Research and Development Program (2024YFC3406806) and the National Natural Science Foundation of China (32171356).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The raw sequencing data generated in this study have been deposited in the NCBI Sequence Read Archive (SRA). The single-cell RNA-seq data are available under BioProject accession number SAMN48188757, and the bulk RNA-seq data (spatiotemporal profiling of proximal and distal sciatic nerve segments) are available under BioProject accession number SAMN49603701.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar4"><h3 class="pmc_sec_title">Ethics approval</h3>
<p id="Par98">All experimental procedures involving animals were approved by the International Council for Laboratory Animal Science and conducted in accordance with guidelines for the care and use of laboratory animals. Efforts were made to minimize animal suffering and reduce the number of animals used in the study. All procedures were approved by the Institutional Animal Care and Use Committee of PLA General Hospital (approval number: 2023‑x4‑19) and conformed to national guidelines for animal care.</p></section><section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par99">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Yiben Ouyang, Mingqian Yu, Tieyuan Zhang and Haofeng Cheng contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Jiang Peng, Email: pengjiang301@126.com.</p>
<p>Yu Wang, Email: wangwangdian628@126.com.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Arbash M, Alzobi OZ, Salameh M, Alkhayarin M, Ahmed G. Incidence, risk factors, and prognosis of sciatic nerve injury in acetabular fractures: a retrospective cross-sectional study. Int Orthop. 2024;48:849–56.
</cite> [<a href="https://doi.org/10.1007/s00264-024-06087-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10902080/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38195944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arbash%20M,%20Alzobi%20OZ,%20Salameh%20M,%20Alkhayarin%20M,%20Ahmed%20G.%20Incidence,%20risk%20factors,%20and%20prognosis%20of%20sciatic%20nerve%20injury%20in%20acetabular%20fractures:%20a%20retrospective%20cross-sectional%20study.%20Int%20Orthop.%202024;48:849%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Liu Z, Fu B, Xu W, Liu F, Dong J, Li L, Zhou D, Hao Z, Lu S. Incidence of traumatic sciatic nerve injury in association with acetabular fracture: a retrospective observational single-center study. Int J Gen Med. 2022;15:7417–25.
</cite> [<a href="https://doi.org/10.2147/IJGM.S385995" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9512635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36172087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Z,%20Fu%20B,%20Xu%20W,%20Liu%20F,%20Dong%20J,%20Li%20L,%20Zhou%20D,%20Hao%20Z,%20Lu%20S.%20Incidence%20of%20traumatic%20sciatic%20nerve%20injury%20in%20association%20with%20acetabular%20fracture:%20a%20retrospective%20observational%20single-center%20study.%20Int%20J%20Gen%20Med.%202022;15:7417%E2%80%9325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Callaghan BC, Cheng HT, Stables CL, Smith AL, Feldman EL. Diabetic neuropathy: clinical manifestations and current treatments. Lancet Neurol. 2012;11:521–34.
</cite> [<a href="https://doi.org/10.1016/S1474-4422(12)70065-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4254767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22608666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Callaghan%20BC,%20Cheng%20HT,%20Stables%20CL,%20Smith%20AL,%20Feldman%20EL.%20Diabetic%20neuropathy:%20clinical%20manifestations%20and%20current%20treatments.%20Lancet%20Neurol.%202012;11:521%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Vieira AS, Baptista AF, Mendes L, Silva KS, Gois SC, Lima FM, Souza I, Sa KN. Impact of neuropathic pain at the population level. J Clin Med Res. 2014;6:111–9.
</cite> [<a href="https://doi.org/10.14740/jocmr1675w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3935519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24578752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vieira%20AS,%20Baptista%20AF,%20Mendes%20L,%20Silva%20KS,%20Gois%20SC,%20Lima%20FM,%20Souza%20I,%20Sa%20KN.%20Impact%20of%20neuropathic%20pain%20at%20the%20population%20level.%20J%20Clin%20Med%20Res.%202014;6:111%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Litovchenko TA, Borodai OM, Tondiy OL. Quality of life of patients with post-traumatic neuropathies and plexopathies accompanied by chronic neuropathic pain syndrome. Int Neurol J. 2021;17:21–4.</cite> [<a href="https://scholar.google.com/scholar_lookup?Litovchenko%20TA,%20Borodai%20OM,%20Tondiy%20OL.%20Quality%20of%20life%20of%20patients%20with%20post-traumatic%20neuropathies%20and%20plexopathies%20accompanied%20by%20chronic%20neuropathic%20pain%20syndrome.%20Int%20Neurol%20J.%202021;17:21%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Zhao XF, Huffman LD, Hafner H, Athaiya M, Finneran MC, Kalinski AL, Kohen R, Flynn C, Passino R, Johnson CN et al. The injured sciatic nerve atlas (iSNAT), insights into the cellular and molecular basis of neural tissue degeneration and regeneration. Elife. 2022;11.</cite> [<a href="https://doi.org/10.7554/eLife.80881" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9829412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36515985/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Deumens R, Bozkurt A, Meek MF, Marcus MA, Joosten EA, Weis J, Brook GA. Repairing injured peripheral nerves: bridging the gap. Prog Neurobiol. 2010;92:245–76.
</cite> [<a href="https://doi.org/10.1016/j.pneurobio.2010.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20950667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deumens%20R,%20Bozkurt%20A,%20Meek%20MF,%20Marcus%20MA,%20Joosten%20EA,%20Weis%20J,%20Brook%20GA.%20Repairing%20injured%20peripheral%20nerves:%20bridging%20the%20gap.%20Prog%20Neurobiol.%202010;92:245%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Siham HR, MA.: From DNA to data: transcending beyond the double Helix and demystifying the genetic alchemy of life through NGS to empower precision medicine. J Clin Med Res. 2023;5:162−2189.</cite>
</li>
<li id="CR9">
<span class="label">9.</span><cite>Jessen KR, Mirsky R. The repair Schwann cell and its function in regenerating nerves. J Physiol. 2016;594:3521–31.
</cite> [<a href="https://doi.org/10.1113/JP270874" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4929314/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26864683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jessen%20KR,%20Mirsky%20R.%20The%20repair%20Schwann%20cell%20and%20its%20function%20in%20regenerating%20nerves.%20J%20Physiol.%202016;594:3521%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Torres-Mejía E, Trümbach D, Kleeberger C, Dornseifer U, Orschmann T, Bäcker T, Brenke JK, Hadian K, Wurst W, López-Schier H, Desbordes SC. Sox2 controls Schwann cell self-organization through fibronectin fibrillogenesis. Sci Rep. 2020;10:1984.</cite> [<a href="https://doi.org/10.1038/s41598-019-56877-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7005302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32029747/" class="usa-link">PubMed</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Chen P, Cescon M, Zuccolotto G, Nobbio L, Colombelli C, Filaferro M, Vitale G, Feltri ML, Bonaldo P. Collagen VI regulates peripheral nerve regeneration by modulating macrophage recruitment and polarization. Acta Neuropathol. 2015;129:97–113.
</cite> [<a href="https://doi.org/10.1007/s00401-014-1369-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25421425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20P,%20Cescon%20M,%20Zuccolotto%20G,%20Nobbio%20L,%20Colombelli%20C,%20Filaferro%20M,%20Vitale%20G,%20Feltri%20ML,%20Bonaldo%20P.%20Collagen%20VI%20regulates%20peripheral%20nerve%20regeneration%20by%20modulating%20macrophage%20recruitment%20and%20polarization.%20Acta%20Neuropathol.%202015;129:97%E2%80%93113." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Bormann D, Knoflach M, Poreba E, Riedl CJ, Testa G, Orset C, Levilly A, Cottereau A, Jauk P, Hametner S, et al. Single-nucleus RNA sequencing reveals glial cell type-specific responses to ischemic stroke in male rodents. Nat Commun. 2024;15:6232.
</cite> [<a href="https://doi.org/10.1038/s41467-024-50465-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11266704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39043661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bormann%20D,%20Knoflach%20M,%20Poreba%20E,%20Riedl%20CJ,%20Testa%20G,%20Orset%20C,%20Levilly%20A,%20Cottereau%20A,%20Jauk%20P,%20Hametner%20S,%20et%20al.%20Single-nucleus%20RNA%20sequencing%20reveals%20glial%20cell%20type-specific%20responses%20to%20ischemic%20stroke%20in%20male%20rodents.%20Nat%20Commun.%202024;15:6232." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Garcia-Bonilla L, Shahanoor Z, Sciortino R, Nazarzoda O, Racchumi G, Iadecola C, Anrather J. Analysis of brain and blood single-cell transcriptomics in acute and subacute phases after experimental stroke. Nat Immunol. 2024;25:357–70.
</cite> [<a href="https://doi.org/10.1038/s41590-023-01711-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38177281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garcia-Bonilla%20L,%20Shahanoor%20Z,%20Sciortino%20R,%20Nazarzoda%20O,%20Racchumi%20G,%20Iadecola%20C,%20Anrather%20J.%20Analysis%20of%20brain%20and%20blood%20single-cell%20transcriptomics%20in%20acute%20and%20subacute%20phases%20after%20experimental%20stroke.%20Nat%20Immunol.%202024;25:357%E2%80%9370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Skinnider MA, Gautier M, Teo AYY, Kathe C, Hutson TH, Laskaratos A, de Coucy A, Regazzi N, Aureli V, James ND, et al. Single-cell and spatial atlases of spinal cord injury in the tabulae paralytica. Nature. 2024;631:150–63.
</cite> [<a href="https://doi.org/10.1038/s41586-024-07504-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38898272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Skinnider%20MA,%20Gautier%20M,%20Teo%20AYY,%20Kathe%20C,%20Hutson%20TH,%20Laskaratos%20A,%20de%20Coucy%20A,%20Regazzi%20N,%20Aureli%20V,%20James%20ND,%20et%20al.%20Single-cell%20and%20spatial%20atlases%20of%20spinal%20cord%20injury%20in%20the%20tabulae%20paralytica.%20Nature.%202024;631:150%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Wolbert J, Li X, Heming M, Mausberg AK, Akkermann D, Frydrychowicz C, Fledrich R, Groeneweg L, Schulz C, Stettner M, et al. Redefining the heterogeneity of peripheral nerve cells in health and autoimmunity. Proc Natl Acad Sci U S A. 2020;117:9466–76.
</cite> [<a href="https://doi.org/10.1073/pnas.1912139117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7196786/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32295886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolbert%20J,%20Li%20X,%20Heming%20M,%20Mausberg%20AK,%20Akkermann%20D,%20Frydrychowicz%20C,%20Fledrich%20R,%20Groeneweg%20L,%20Schulz%20C,%20Stettner%20M,%20et%20al.%20Redefining%20the%20heterogeneity%20of%20peripheral%20nerve%20cells%20in%20health%20and%20autoimmunity.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202020;117:9466%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Bhat GP, Maurizio A, Motta A, Podini P, Diprima S, Malpighi C, Brambilla I, Martins L, Badaloni A, Boselli D, et al. Structured wound angiogenesis instructs mesenchymal barrier compartments in the regenerating nerve. Neuron. 2024;112:209–e229211.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2023.10.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37972594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bhat%20GP,%20Maurizio%20A,%20Motta%20A,%20Podini%20P,%20Diprima%20S,%20Malpighi%20C,%20Brambilla%20I,%20Martins%20L,%20Badaloni%20A,%20Boselli%20D,%20et%20al.%20Structured%20wound%20angiogenesis%20instructs%20mesenchymal%20barrier%20compartments%20in%20the%20regenerating%20nerve.%20Neuron.%202024;112:209%E2%80%93e229211." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Wang Y, Guo L, Yin X, McCarthy EC, Cheng MI, Hoang AT, Chen HC, Patel AY, Allard Trout D, Xu E et al. Pathogenic TNF-α drives peripheral nerve inflammation in an aire-deficient model of autoimmunity. Proc Natl Acad Sci U S A. 2022;119.</cite> [<a href="https://doi.org/10.1073/pnas.2114406119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8795502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35058362/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Qin Q, Gomez-Salazar M, Cherief M, Pagani CA, Lee S, Hwang C, Tower RJ, Onggo S, Sun Y, Piplani A, et al. Neuron-to-vessel signaling is a required feature of aberrant stem cell commitment after soft tissue trauma. Bone Res. 2022;10:43.
</cite> [<a href="https://doi.org/10.1038/s41413-022-00216-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9156761/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35641477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qin%20Q,%20Gomez-Salazar%20M,%20Cherief%20M,%20Pagani%20CA,%20Lee%20S,%20Hwang%20C,%20Tower%20RJ,%20Onggo%20S,%20Sun%20Y,%20Piplani%20A,%20et%20al.%20Neuron-to-vessel%20signaling%20is%20a%20required%20feature%20of%20aberrant%20stem%20cell%20commitment%20after%20soft%20tissue%20trauma.%20Bone%20Res.%202022;10:43." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Stefan B, Derk EG, Dieter R, Reto IP, Moritz R, Klaus-Armin N, Carmen B, Michael W. The transcription factor Sox10 is a key regulator of peripheral glial development. Genes Dev. 2001;15:66–78.
</cite> [<a href="https://doi.org/10.1101/gad.186601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC312607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11156606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stefan%20B,%20Derk%20EG,%20Dieter%20R,%20Reto%20IP,%20Moritz%20R,%20Klaus-Armin%20N,%20Carmen%20B,%20Michael%20W.%20The%20transcription%20factor%20Sox10%20is%20a%20key%20regulator%20of%20peripheral%20glial%20development.%20Genes%20Dev.%202001;15:66%E2%80%9378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Susanne Q, Bastian GB, Madlen E, Franziska F, Theresa K, Friederike AA, Victor T, Danny H, Dies M, Ueli S, et al. Zeb2 is essential for Schwann cell differentiation, myelination and nerve repair. Nat Neurosci. 2016;19:1050–9.
</cite> [<a href="https://doi.org/10.1038/nn.4321" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4964942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27294512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Susanne%20Q,%20Bastian%20GB,%20Madlen%20E,%20Franziska%20F,%20Theresa%20K,%20Friederike%20AA,%20Victor%20T,%20Danny%20H,%20Dies%20M,%20Ueli%20S,%20et%20al.%20Zeb2%20is%20essential%20for%20Schwann%20cell%20differentiation,%20myelination%20and%20nerve%20repair.%20Nat%20Neurosci.%202016;19:1050%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Katja SG, Hagen W, Florian EP, Cyril C, Alistair NG, Sandra Z, Konstantin F, Daniel B, Herbert S, Elior P et al. The tyrosine phosphatase Shp2 (PTPN11) directs Neuregulin-1/ErbB signaling throughout Schwann cell development. Proceedings of the National Academy of Sciences. 2009;106:16704–16709.</cite> [<a href="https://doi.org/10.1073/pnas.0904336106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2742405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19805360/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Tamara G, Simone S, Jingjing Q, Hagen W, Alistair NG, Klaus-Armin N, Carmen B, Walter B. Wnt/Rspondin/β-catenin signals control axonal sorting and lineage progression in Schwann cell development. Proceedings of the National Academy of Sciences. 2013;110:18174–18179.</cite> [<a href="https://doi.org/10.1073/pnas.1310490110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3831430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24151333/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Chen B, Banton MC, Singh L, Parkinson DB, Dun XP. Single cell transcriptome data analysis defines the heterogeneity of peripheral nerve cells in homeostasis and regeneration. Front Cell Neurosci. 2021;15:624826.
</cite> [<a href="https://doi.org/10.3389/fncel.2021.624826" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8019921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33828460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20B,%20Banton%20MC,%20Singh%20L,%20Parkinson%20DB,%20Dun%20XP.%20Single%20cell%20transcriptome%20data%20analysis%20defines%20the%20heterogeneity%20of%20peripheral%20nerve%20cells%20in%20homeostasis%20and%20regeneration.%20Front%20Cell%20Neurosci.%202021;15:624826." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Parkhomchuk D, Borodina T, Amstislavskiy V, Banaru M, Hallen L, Krobitsch S, Lehrach H, Soldatov A. Transcriptome analysis by strand-specific sequencing of complementary DNA. Nucleic Acids Res. 2009;37:e123.
</cite> [<a href="https://doi.org/10.1093/nar/gkp596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2764448/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19620212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Parkhomchuk%20D,%20Borodina%20T,%20Amstislavskiy%20V,%20Banaru%20M,%20Hallen%20L,%20Krobitsch%20S,%20Lehrach%20H,%20Soldatov%20A.%20Transcriptome%20analysis%20by%20strand-specific%20sequencing%20of%20complementary%20DNA.%20Nucleic%20Acids%20Res.%202009;37:e123." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Hao Y, Stuart T, Kowalski MH, Choudhary S, Hoffman P, Hartman A, Srivastava A, Molla G, Madad S, Fernandez-Granda C, Satija R. Dictionary learning for integrative, multimodal and scalable single-cell analysis. Nat Biotechnol. 2024;42:293–304.
</cite> [<a href="https://doi.org/10.1038/s41587-023-01767-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10928517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37231261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao%20Y,%20Stuart%20T,%20Kowalski%20MH,%20Choudhary%20S,%20Hoffman%20P,%20Hartman%20A,%20Srivastava%20A,%20Molla%20G,%20Madad%20S,%20Fernandez-Granda%20C,%20Satija%20R.%20Dictionary%20learning%20for%20integrative,%20multimodal%20and%20scalable%20single-cell%20analysis.%20Nat%20Biotechnol.%202024;42:293%E2%80%93304." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Aran D, Looney AP, Liu L, Wu E, Fong V, Hsu A, Chak S, Naikawadi RP, Wolters PJ, Abate AR, et al. Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. Nat Immunol. 2019;20:163–72.
</cite> [<a href="https://doi.org/10.1038/s41590-018-0276-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6340744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30643263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aran%20D,%20Looney%20AP,%20Liu%20L,%20Wu%20E,%20Fong%20V,%20Hsu%20A,%20Chak%20S,%20Naikawadi%20RP,%20Wolters%20PJ,%20Abate%20AR,%20et%20al.%20Reference-based%20analysis%20of%20lung%20single-cell%20sequencing%20reveals%20a%20transitional%20profibrotic%20macrophage.%20Nat%20Immunol.%202019;20:163%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Xu S, Hu E, Cai Y, Xie Z, Luo X, Zhan L, Tang W, Wang Q, Liu B, Wang R, et al. Using clusterprofiler to characterize multiomics data. Nat Protoc. 2024;19:3292–320.
</cite> [<a href="https://doi.org/10.1038/s41596-024-01020-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39019974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20S,%20Hu%20E,%20Cai%20Y,%20Xie%20Z,%20Luo%20X,%20Zhan%20L,%20Tang%20W,%20Wang%20Q,%20Liu%20B,%20Wang%20R,%20et%20al.%20Using%20clusterprofiler%20to%20characterize%20multiomics%20data.%20Nat%20Protoc.%202024;19:3292%E2%80%93320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Trapnell C, Cacchiarelli D, Grimsby J, Pokharel P, Li S, Morse M, Lennon NJ, Livak KJ, Mikkelsen TS, Rinn JL. The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. Nat Biotechnol. 2014;32:381–6.
</cite> [<a href="https://doi.org/10.1038/nbt.2859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4122333/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24658644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trapnell%20C,%20Cacchiarelli%20D,%20Grimsby%20J,%20Pokharel%20P,%20Li%20S,%20Morse%20M,%20Lennon%20NJ,%20Livak%20KJ,%20Mikkelsen%20TS,%20Rinn%20JL.%20The%20dynamics%20and%20regulators%20of%20cell%20fate%20decisions%20are%20revealed%20by%20pseudotemporal%20ordering%20of%20single%20cells.%20Nat%20Biotechnol.%202014;32:381%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Street K, Risso D, Fletcher RB, Das D, Ngai J, Yosef N, Purdom E, Dudoit S. Slingshot: cell lineage and pseudotime inference for single-cell transcriptomics. BMC Genomics. 2018;19:477.
</cite> [<a href="https://doi.org/10.1186/s12864-018-4772-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6007078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29914354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Street%20K,%20Risso%20D,%20Fletcher%20RB,%20Das%20D,%20Ngai%20J,%20Yosef%20N,%20Purdom%20E,%20Dudoit%20S.%20Slingshot:%20cell%20lineage%20and%20pseudotime%20inference%20for%20single-cell%20transcriptomics.%20BMC%20Genomics.%202018;19:477." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Jin S, Guerrero-Juarez CF, Zhang L, Chang I, Ramos R, Kuan CH, Myung P, Plikus MV, Nie Q. Inference and analysis of cell-cell communication using cellchat. Nat Commun. 2021;12:1088.
</cite> [<a href="https://doi.org/10.1038/s41467-021-21246-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7889871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33597522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin%20S,%20Guerrero-Juarez%20CF,%20Zhang%20L,%20Chang%20I,%20Ramos%20R,%20Kuan%20CH,%20Myung%20P,%20Plikus%20MV,%20Nie%20Q.%20Inference%20and%20analysis%20of%20cell-cell%20communication%20using%20cellchat.%20Nat%20Commun.%202021;12:1088." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Yan L, Wang J, Cai X, Liou YC, Shen HM, Hao J, Huang C, Luo G, He W. Macrophage plasticity: signaling pathways, tissue repair, and regeneration. MedComm (2020). 2024;5:e658.</cite> [<a href="https://doi.org/10.1002/mco2.658" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11292402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39092292/" class="usa-link">PubMed</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Tiffany B, Nicola H. Neutrophil–macrophage cooperation and its impact on tissue repair. Immunol Cell Biology. 2019;97:289–98.</cite> [<a href="https://doi.org/10.1111/imcb.12241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30710448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tiffany%20B,%20Nicola%20H.%20Neutrophil%E2%80%93macrophage%20cooperation%20and%20its%20impact%20on%20tissue%20repair.%20Immunol%20Cell%20Biology.%202019;97:289%E2%80%9398." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Cunha MI, Su M, Cantuti-Castelvetri L, Müller SA, Schifferer M, Djannatian M, Alexopoulos I, van der Meer F, Winkler A, van Ham TJ et al. Pro-inflammatory activation following demyelination is required for Myelin clearance and oligodendrogenesis. J Exp Med. 2020;217.</cite> [<a href="https://doi.org/10.1084/jem.20191390" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7201919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32078678/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wang G, Lu P, Qiao P, Zhang P, Cai X, Tang L, Qian T, Wang H. Blood vessel remodeling in late stage of vascular network reconstruction is essential for peripheral nerve regeneration. Bioeng Transl Med. 2022;7:e10361.
</cite> [<a href="https://doi.org/10.1002/btm2.10361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9472024/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36176610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20G,%20Lu%20P,%20Qiao%20P,%20Zhang%20P,%20Cai%20X,%20Tang%20L,%20Qian%20T,%20Wang%20H.%20Blood%20vessel%20remodeling%20in%20late%20stage%20of%20vascular%20network%20reconstruction%20is%20essential%20for%20peripheral%20nerve%20regeneration.%20Bioeng%20Transl%20Med.%202022;7:e10361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wang Y, Li D, Wang G, Chen L, Chen J, Liu Z, Zhang Z, Shen H, Jin Y, Shen Z. The effect of co-transplantation of nerve fibroblasts and Schwann cells on peripheral nerve repair. Int J Biol Sci. 2017;13:1507–19.
</cite> [<a href="https://doi.org/10.7150/ijbs.21976" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5723917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29230099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Y,%20Li%20D,%20Wang%20G,%20Chen%20L,%20Chen%20J,%20Liu%20Z,%20Zhang%20Z,%20Shen%20H,%20Jin%20Y,%20Shen%20Z.%20The%20effect%20of%20co-transplantation%20of%20nerve%20fibroblasts%20and%20Schwann%20cells%20on%20peripheral%20nerve%20repair.%20Int%20J%20Biol%20Sci.%202017;13:1507%E2%80%9319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Haynes ME, Sullivan DP, Muller WA. Neutrophil infiltration and function in the pathogenesis of inflammatory airspace disease. Am J Pathol. 2024;194:628–36.
</cite> [<a href="https://doi.org/10.1016/j.ajpath.2023.12.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11074974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38309429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haynes%20ME,%20Sullivan%20DP,%20Muller%20WA.%20Neutrophil%20infiltration%20and%20function%20in%20the%20pathogenesis%20of%20inflammatory%20airspace%20disease.%20Am%20J%20Pathol.%202024;194:628%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Gawriluk TR, Simkin J, Hacker CK, Kimani JM, Kiama SG, Ezenwa VO, Seifert AW. Complex tissue regeneration in mammals is associated with reduced inflammatory cytokines and an influx of T cells. Front Immunol. 2020;11:1695.
</cite> [<a href="https://doi.org/10.3389/fimmu.2020.01695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7427103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32849592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gawriluk%20TR,%20Simkin%20J,%20Hacker%20CK,%20Kimani%20JM,%20Kiama%20SG,%20Ezenwa%20VO,%20Seifert%20AW.%20Complex%20tissue%20regeneration%20in%20mammals%20is%20associated%20with%20reduced%20inflammatory%20cytokines%20and%20an%20influx%20of%20T%20cells.%20Front%20Immunol.%202020;11:1695." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Zanna MY, Yasmin AR, Omar AR, Arshad SS, Mariatulqabtiah AR, Nur-Fazila SH, Mahiza MIN. Review of dendritic cells, their role in clinical immunology, and distribution in various animal species. Int J Mol Sci. 2021;22.</cite> [<a href="https://doi.org/10.3390/ijms22158044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8348663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34360810/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Liu M, Chen S, Zhang A, Zheng Q, Fu J. PLAUR as a potential biomarker associated with immune infiltration in bladder urothelial carcinoma. J Inflamm Res. 2021;14:4629–41.
</cite> [<a href="https://doi.org/10.2147/JIR.S326559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8450190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34552345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20M,%20Chen%20S,%20Zhang%20A,%20Zheng%20Q,%20Fu%20J.%20PLAUR%20as%20a%20potential%20biomarker%20associated%20with%20immune%20infiltration%20in%20bladder%20urothelial%20carcinoma.%20J%20Inflamm%20Res.%202021;14:4629%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Wang F, Zhao C, Jing Z, Wang Q, Li M, Lu B, Huo A, Liang W, Hu W, Fu X. The dual roles of chemokines in peripheral nerve injury and repair. Inflamm Regen. 2025;45:11.
</cite> [<a href="https://doi.org/10.1186/s41232-025-00375-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11987372/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40217284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20F,%20Zhao%20C,%20Jing%20Z,%20Wang%20Q,%20Li%20M,%20Lu%20B,%20Huo%20A,%20Liang%20W,%20Hu%20W,%20Fu%20X.%20The%20dual%20roles%20of%20chemokines%20in%20peripheral%20nerve%20injury%20and%20repair.%20Inflamm%20Regen.%202025;45:11." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Zheng SK, Zhao XK, Wu H, He DW, Xiong L, Cheng XG. Oxidative stress-induced EGR1 upregulation promotes NR4A3-mediated nucleus pulposus cells apoptosis in intervertebral disc degeneration. Aging. 2024;16:10216–38.
</cite> [<a href="https://doi.org/10.18632/aging.205920" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11236312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38943627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng%20SK,%20Zhao%20XK,%20Wu%20H,%20He%20DW,%20Xiong%20L,%20Cheng%20XG.%20Oxidative%20stress-induced%20EGR1%20upregulation%20promotes%20NR4A3-mediated%20nucleus%20pulposus%20cells%20apoptosis%20in%20intervertebral%20disc%20degeneration.%20Aging.%202024;16:10216%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Raghunandan S, Ramachandran S, Ke E, Miao Y, Lal R, Chen ZB, Subramaniam S. Heme oxygenase-1 at the nexus of endothelial cell fate decision under oxidative stress. Front Cell Dev Biol. 2021;9:702974.
</cite> [<a href="https://doi.org/10.3389/fcell.2021.702974" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8476872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34595164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raghunandan%20S,%20Ramachandran%20S,%20Ke%20E,%20Miao%20Y,%20Lal%20R,%20Chen%20ZB,%20Subramaniam%20S.%20Heme%20oxygenase-1%20at%20the%20nexus%20of%20endothelial%20cell%20fate%20decision%20under%20oxidative%20stress.%20Front%20Cell%20Dev%20Biol.%202021;9:702974." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Yang Z, Zhong D, Hu X, Chen W, Liao Y, He X. NFATC2/SERPINE1/JAK3/STAT3 signaling feedback loop in gastric cancer: immune evasion and anti-PD-1 resistance. Cell Biol Toxicol. 2025;41:102.
</cite> [<a href="https://doi.org/10.1007/s10565-025-10050-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12166020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40512406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20Z,%20Zhong%20D,%20Hu%20X,%20Chen%20W,%20Liao%20Y,%20He%20X.%20NFATC2/SERPINE1/JAK3/STAT3%20signaling%20feedback%20loop%20in%20gastric%20cancer:%20immune%20evasion%20and%20anti-PD-1%20resistance.%20Cell%20Biol%20Toxicol.%202025;41:102." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Tao L, Cui Y, Sun J, Cao Y, Dai Z, Ge X, Zhang L, Ma R, Liu Y. Bioinformatics-based analysis reveals elevated CYTL1 as a potential therapeutic target for BRAF-mutated melanoma. Front Cell Dev Biol. 2023;11:1171047.
</cite> [<a href="https://doi.org/10.3389/fcell.2023.1171047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10516578/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37745303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tao%20L,%20Cui%20Y,%20Sun%20J,%20Cao%20Y,%20Dai%20Z,%20Ge%20X,%20Zhang%20L,%20Ma%20R,%20Liu%20Y.%20Bioinformatics-based%20analysis%20reveals%20elevated%20CYTL1%20as%20a%20potential%20therapeutic%20target%20for%20BRAF-mutated%20melanoma.%20Front%20Cell%20Dev%20Biol.%202023;11:1171047." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Li C, Xu C, Guan R, Jiao R, Wang Y, Cui C, Cao S, Chang F, Wei R, Li Z, et al. Spatial transcriptomics reveal tumor microenvironment and SLCO2A1 correlated with tumor suppression in hypopharyngeal squamous cell carcinoma. Int Immunopharmacol. 2024;142:113243.
</cite> [<a href="https://doi.org/10.1016/j.intimp.2024.113243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39340989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20C,%20Xu%20C,%20Guan%20R,%20Jiao%20R,%20Wang%20Y,%20Cui%20C,%20Cao%20S,%20Chang%20F,%20Wei%20R,%20Li%20Z,%20et%20al.%20Spatial%20transcriptomics%20reveal%20tumor%20microenvironment%20and%20SLCO2A1%20correlated%20with%20tumor%20suppression%20in%20hypopharyngeal%20squamous%20cell%20carcinoma.%20Int%20Immunopharmacol.%202024;142:113243." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Attisano L, Silvestri C, Izzi L, Labbé E. The transcriptional role of Smads and FAST (FoxH1) in TGFbeta and activin signalling. Mol Cell Endocrinol. 2001;180:3–11.
</cite> [<a href="https://doi.org/10.1016/s0303-7207(01)00524-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11451566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Attisano%20L,%20Silvestri%20C,%20Izzi%20L,%20Labb%C3%A9%20E.%20The%20transcriptional%20role%20of%20Smads%20and%20FAST%20(FoxH1)%20in%20TGFbeta%20and%20activin%20signalling.%20Mol%20Cell%20Endocrinol.%202001;180:3%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Ahn J, Kim DH, Suh Y, Lee JW, Lee K. Adipose-specific expression of mouse Rbp7 gene and its developmental and metabolic changes. Gene. 2018;670:38–45.
</cite> [<a href="https://doi.org/10.1016/j.gene.2018.05.101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29803924/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahn%20J,%20Kim%20DH,%20Suh%20Y,%20Lee%20JW,%20Lee%20K.%20Adipose-specific%20expression%20of%20mouse%20Rbp7%20gene%20and%20its%20developmental%20and%20metabolic%20changes.%20Gene.%202018;670:38%E2%80%9345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Zhou Y, Zhou Y, Chen H, Zhang L, Bi S. Potential role of CD99 signaling pathway in Schwann cell dysfunction in diabetic foot ulcers based on single-cell transcriptome analysis. J Diabetes Res. 2025;2025:9935400.
</cite> [<a href="https://doi.org/10.1155/jdr/9935400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12103954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40420926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20Y,%20Zhou%20Y,%20Chen%20H,%20Zhang%20L,%20Bi%20S.%20Potential%20role%20of%20CD99%20signaling%20pathway%20in%20Schwann%20cell%20dysfunction%20in%20diabetic%20foot%20ulcers%20based%20on%20single-cell%20transcriptome%20analysis.%20J%20Diabetes%20Res.%202025;2025:9935400." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Zhang H, Xie P, An R. Abstract 2789: NCAPG promotes triple-negative breast cancer stemness by a positive feedback loop with HIF-1α. Cancer Res. 2024;84:3.37902414
</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhang%20H,%20Xie%20P,%20An%20R.%20Abstract%202789:%20NCAPG%20promotes%20triple-negative%20breast%20cancer%20stemness%20by%20a%20positive%20feedback%20loop%20with%20HIF-1%CE%B1.%20Cancer%20Res.%202024;84:3.37902414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Wei P, Zhang N, Xu Y, Li X, Shi D, Wang Y, Li D, Cai S. TPX2 is a novel prognostic marker for the growth and metastasis of colon cancer. J Transl Med. 2013;11:313.
</cite> [<a href="https://doi.org/10.1186/1479-5876-11-313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3878622/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24341487/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei%20P,%20Zhang%20N,%20Xu%20Y,%20Li%20X,%20Shi%20D,%20Wang%20Y,%20Li%20D,%20Cai%20S.%20TPX2%20is%20a%20novel%20prognostic%20marker%20for%20the%20growth%20and%20metastasis%20of%20colon%20cancer.%20J%20Transl%20Med.%202013;11:313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Aggarwal S, Pittenger MF. Modulation of immune cell responses by human mesenchymal stem cells. Blood. 2004;104:1288–1288.</cite> [<a href="https://doi.org/10.1182/blood-2004-04-1559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15494428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aggarwal%20S,%20Pittenger%20MF.%20Modulation%20of%20immune%20cell%20responses%20by%20human%20mesenchymal%20stem%20cells.%20Blood.%202004;104:1288%E2%80%931288." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Genovese L, Brendolan A. Lymphoid tissue mesenchymal stromal cells in development and tissue remodeling. Stem Cells Int. 2016;2016:8419104.</cite> [<a href="https://doi.org/10.1155/2016/8419104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4846763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27190524/" class="usa-link">PubMed</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Strickland LA, Jubb AM, Hongo JA, Zhong F, Burwick J, Fu L, Frantz GD, Koeppen H. Plasmalemmal vesicle-associated protein (PLVAP) is expressed by tumour endothelium and is upregulated by vascular endothelial growth factor-A (VEGF). J Pathol. 2005;206:466–75.
</cite> [<a href="https://doi.org/10.1002/path.1805" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15971170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Strickland%20LA,%20Jubb%20AM,%20Hongo%20JA,%20Zhong%20F,%20Burwick%20J,%20Fu%20L,%20Frantz%20GD,%20Koeppen%20H.%20Plasmalemmal%20vesicle-associated%20protein%20(PLVAP)%20is%20expressed%20by%20tumour%20endothelium%20and%20is%20upregulated%20by%20vascular%20endothelial%20growth%20factor-A%20(VEGF).%20J%20Pathol.%202005;206:466%E2%80%9375." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Wang X, Abraham S, McKenzie JAG, Jeffs N, Swire M, Tripathi VB, Luhmann UFO, Lange CAK, Zhai Z, Arthur HM, et al. LRG1 promotes angiogenesis by modulating endothelial TGF-β signalling. Nature. 2013;499:306–11.
</cite> [<a href="https://doi.org/10.1038/nature12345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3836402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23868260/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20X,%20Abraham%20S,%20McKenzie%20JAG,%20Jeffs%20N,%20Swire%20M,%20Tripathi%20VB,%20Luhmann%20UFO,%20Lange%20CAK,%20Zhai%20Z,%20Arthur%20HM,%20et%20al.%20LRG1%20promotes%20angiogenesis%20by%20modulating%20endothelial%20TGF-%CE%B2%20signalling.%20Nature.%202013;499:306%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Park HN, Song MJ, Choi YE, Lee DH, Chung JH, Lee ST. LRG1 promotes ECM integrity by activating the TGF-β signaling pathway in fibroblasts. Int J Mol Sci. 2023;24.</cite> [<a href="https://doi.org/10.3390/ijms241512445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10418909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37569820/" class="usa-link">PubMed</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Cui N, Hu M, Khalil RA. Biochemical and biological attributes of matrix metalloproteinases. Prog Mol Biol Transl Sci. 2017;147:1–73.
</cite> [<a href="https://doi.org/10.1016/bs.pmbts.2017.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5430303/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28413025/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cui%20N,%20Hu%20M,%20Khalil%20RA.%20Biochemical%20and%20biological%20attributes%20of%20matrix%20metalloproteinases.%20Prog%20Mol%20Biol%20Transl%20Sci.%202017;147:1%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Cash JN, Angerman EB, Keutmann HT, Thompson TB. Characterization of follistatin-type domains and their contribution to myostatin and activin A antagonism. Mol Endocrinol. 2012;26:1167–78.
</cite> [<a href="https://doi.org/10.1210/me.2012-1061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3385792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22593183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cash%20JN,%20Angerman%20EB,%20Keutmann%20HT,%20Thompson%20TB.%20Characterization%20of%20follistatin-type%20domains%20and%20their%20contribution%20to%20myostatin%20and%20activin%20A%20antagonism.%20Mol%20Endocrinol.%202012;26:1167%E2%80%9378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Ding Z, Jiang M, Qian J, Gu D, Bai H, Cai M, Yao D. Role of transforming growth factor-β in peripheral nerve regeneration. Neural Regen Res. 2024;19:380–6.
</cite> [<a href="https://doi.org/10.4103/1673-5374.377588" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10503632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37488894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ding%20Z,%20Jiang%20M,%20Qian%20J,%20Gu%20D,%20Bai%20H,%20Cai%20M,%20Yao%20D.%20Role%20of%20transforming%20growth%20factor-%CE%B2%20in%20peripheral%20nerve%20regeneration.%20Neural%20Regen%20Res.%202024;19:380%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Wingo AP, Dammer EB, Breen MS, Logsdon BA, Duong DM, Troncosco JC, Thambisetty M, Beach TG, Serrano GE, Reiman EM, et al. Large-scale proteomic analysis of human brain identifies proteins associated with cognitive trajectory in advanced age. Nat Commun. 2019;10:1619.
</cite> [<a href="https://doi.org/10.1038/s41467-019-09613-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6453881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30962425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wingo%20AP,%20Dammer%20EB,%20Breen%20MS,%20Logsdon%20BA,%20Duong%20DM,%20Troncosco%20JC,%20Thambisetty%20M,%20Beach%20TG,%20Serrano%20GE,%20Reiman%20EM,%20et%20al.%20Large-scale%20proteomic%20analysis%20of%20human%20brain%20identifies%20proteins%20associated%20with%20cognitive%20trajectory%20in%20advanced%20age.%20Nat%20Commun.%202019;10:1619." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Schnaar RL, Lopez PH. Myelin-associated glycoprotein and its axonal receptors. J Neurosci Res. 2009;87:3267–76.
</cite> [<a href="https://doi.org/10.1002/jnr.21992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2892843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19156870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schnaar%20RL,%20Lopez%20PH.%20Myelin-associated%20glycoprotein%20and%20its%20axonal%20receptors.%20J%20Neurosci%20Res.%202009;87:3267%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Kuivaniemi H, Tromp G. Type III collagen (COL3A1): gene and protein structure, tissue distribution, and associated diseases. Gene. 2019;707:151–71.
</cite> [<a href="https://doi.org/10.1016/j.gene.2019.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6579750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31075413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuivaniemi%20H,%20Tromp%20G.%20Type%20III%20collagen%20(COL3A1):%20gene%20and%20protein%20structure,%20tissue%20distribution,%20and%20associated%20diseases.%20Gene.%202019;707:151%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Pashkova N, Peterson TA, Ptak CP, Winistorfer SC, Guerrero-Given D, Kamasawa N, Ahern CA, Shy ME, Piper RC. Disrupting the transmembrane domain interface between PMP22 and MPZ causes peripheral neuropathy. bioRxiv. 2024.</cite> [<a href="https://doi.org/10.1016/j.isci.2024.110989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11700639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39759075/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Shih CH, Lacagnina M, Leuer-Bisciotti K, Pröschel C. Astroglial-derived periostin promotes axonal regeneration after spinal cord injury. J Neurosci. 2014;34:2438–43.
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.2947-13.2014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6802751/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24523534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shih%20CH,%20Lacagnina%20M,%20Leuer-Bisciotti%20K,%20Pr%C3%B6schel%20C.%20Astroglial-derived%20periostin%20promotes%20axonal%20regeneration%20after%20spinal%20cord%20injury.%20J%20Neurosci.%202014;34:2438%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Castellano JM, Kim J, Stewart FR, Jiang H, DeMattos RB, Patterson BW, Fagan AM, Morris JC, Mawuenyega KG, Cruchaga C, et al. Human ApoE isoforms differentially regulate brain amyloid-β peptide clearance. Sci Transl Med. 2011;3:89ra57.
</cite> [<a href="https://doi.org/10.1126/scitranslmed.3002156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3192364/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21715678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Castellano%20JM,%20Kim%20J,%20Stewart%20FR,%20Jiang%20H,%20DeMattos%20RB,%20Patterson%20BW,%20Fagan%20AM,%20Morris%20JC,%20Mawuenyega%20KG,%20Cruchaga%20C,%20et%20al.%20Human%20ApoE%20isoforms%20differentially%20regulate%20brain%20amyloid-%CE%B2%20peptide%20clearance.%20Sci%20Transl%20Med.%202011;3:89ra57." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Sharma A, Schwarzbauer JE. Differential regulation of neurite outgrowth and growth cone morphology by 3D fibronectin and fibronectin-collagen extracellular matrices. Mol Neurobiol. 2022;59:1112–23.
</cite> [<a href="https://doi.org/10.1007/s12035-021-02637-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8858852/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34845592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma%20A,%20Schwarzbauer%20JE.%20Differential%20regulation%20of%20neurite%20outgrowth%20and%20growth%20cone%20morphology%20by%203D%20fibronectin%20and%20fibronectin-collagen%20extracellular%20matrices.%20Mol%20Neurobiol.%202022;59:1112%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Rauvala H, Peng HB. HB-GAM (heparin-binding growth-associated molecule) and heparin-type glycans in the development and plasticity of neuron-target contacts. Prog Neurobiol. 1997;52:127–44.
</cite> [<a href="https://doi.org/10.1016/s0301-0082(97)00007-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9185236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rauvala%20H,%20Peng%20HB.%20HB-GAM%20(heparin-binding%20growth-associated%20molecule)%20and%20heparin-type%20glycans%20in%20the%20development%20and%20plasticity%20of%20neuron-target%20contacts.%20Prog%20Neurobiol.%201997;52:127%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Brosseau JP, Sathe AA, Wang Y, Nguyen T, Glass DA 2nd, Xing C, Le LQ. Human cutaneous neurofibroma matrisome revealed by single-cell RNA sequencing. Acta Neuropathol Commun. 2021;9:11.
</cite> [<a href="https://doi.org/10.1186/s40478-020-01103-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7792184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33413690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brosseau%20JP,%20Sathe%20AA,%20Wang%20Y,%20Nguyen%20T,%20Glass%20DA%202nd,%20Xing%20C,%20Le%20LQ.%20Human%20cutaneous%20neurofibroma%20matrisome%20revealed%20by%20single-cell%20RNA%20sequencing.%20Acta%20Neuropathol%20Commun.%202021;9:11." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Jiang M, Ding Z, Huang Y, Jiang T, Xia Y, Gu D, Gu X, Bai H, Yao D. TGF-β1 improves nerve regeneration and functional recovery after sciatic nerve injury by alleviating inflammation. Biomedicines. 2025;13.</cite> [<a href="https://doi.org/10.3390/biomedicines13040872" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12024759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40299436/" class="usa-link">PubMed</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Ye Z, Wei J, Zhan C, Hou J. Role of transforming growth factor beta in peripheral nerve regeneration: cellular and molecular mechanisms. Front Neurosci. 2022;16:917587.
</cite> [<a href="https://doi.org/10.3389/fnins.2022.917587" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9234557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35769702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye%20Z,%20Wei%20J,%20Zhan%20C,%20Hou%20J.%20Role%20of%20transforming%20growth%20factor%20beta%20in%20peripheral%20nerve%20regeneration:%20cellular%20and%20molecular%20mechanisms.%20Front%20Neurosci.%202022;16:917587." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Gomez SA, Argüelles CL, Guerrieri D, Tateosian NL, Amiano NO, Slimovich R, Maffia PC, Abbate E, Musella RM, Garcia VE, Chuluyan HE. Secretory leukocyte protease inhibitor: a secreted pattern recognition receptor for mycobacteria. Am J Respir Crit Care Med. 2009;179:247–53.
</cite> [<a href="https://doi.org/10.1164/rccm.200804-615OC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19011154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gomez%20SA,%20Arg%C3%BCelles%20CL,%20Guerrieri%20D,%20Tateosian%20NL,%20Amiano%20NO,%20Slimovich%20R,%20Maffia%20PC,%20Abbate%20E,%20Musella%20RM,%20Garcia%20VE,%20Chuluyan%20HE.%20Secretory%20leukocyte%20protease%20inhibitor:%20a%20secreted%20pattern%20recognition%20receptor%20for%20mycobacteria.%20Am%20J%20Respir%20Crit%20Care%20Med.%202009;179:247%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Mao Y, Shi D, Li G, Jiang P. Citrulline depletion by ASS1 is required for proinflammatory macrophage activation and immune responses. Mol Cell. 2022;82:527–e541527.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2021.12.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35016033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao%20Y,%20Shi%20D,%20Li%20G,%20Jiang%20P.%20Citrulline%20depletion%20by%20ASS1%20is%20required%20for%20proinflammatory%20macrophage%20activation%20and%20immune%20responses.%20Mol%20Cell.%202022;82:527%E2%80%93e541527." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Pruenster M, Immler R, Roth J, Kuchler T, Bromberger T, Napoli M, Nussbaumer K, Rohwedder I, Wackerbarth LM, Piantoni C, et al. E-selectin-mediated rapid NLRP3 inflammasome activation regulates S100A8/S100A9 release from neutrophils via transient gasdermin D pore formation. Nat Immunol. 2023;24:2021–31.
</cite> [<a href="https://doi.org/10.1038/s41590-023-01656-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10681899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37903858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pruenster%20M,%20Immler%20R,%20Roth%20J,%20Kuchler%20T,%20Bromberger%20T,%20Napoli%20M,%20Nussbaumer%20K,%20Rohwedder%20I,%20Wackerbarth%20LM,%20Piantoni%20C,%20et%20al.%20E-selectin-mediated%20rapid%20NLRP3%20inflammasome%20activation%20regulates%20S100A8/S100A9%20release%20from%20neutrophils%20via%20transient%20gasdermin%20D%20pore%20formation.%20Nat%20Immunol.%202023;24:2021%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Sugimoto MA, Vago JP, Teixeira MM, Sousa LP. Annexin A1 and the resolution of inflammation: modulation of neutrophil recruitment, apoptosis, and clearance. J Immunol Res. 2016;2016:8239258.</cite> [<a href="https://doi.org/10.1155/2016/8239258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4738713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26885535/" class="usa-link">PubMed</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Tester AM, Cox JH, Connor AR, Starr AE, Dean RA, Puente XS, López-Otín C, Overall CM. LPS responsiveness and neutrophil chemotaxis in vivo require PMN MMP-8 activity. PLoS ONE. 2007;2:e312.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0000312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1819564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17375198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tester%20AM,%20Cox%20JH,%20Connor%20AR,%20Starr%20AE,%20Dean%20RA,%20Puente%20XS,%20L%C3%B3pez-Ot%C3%ADn%20C,%20Overall%20CM.%20LPS%20responsiveness%20and%20neutrophil%20chemotaxis%20in%20vivo%20require%20PMN%20MMP-8%20activity.%20PLoS%20ONE.%202007;2:e312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Schnell A, Huang L, Regan BML, Singh V, Vonficht D, Bollhagen A, Wang M, Hou Y, Bod L, Sobel RA, et al. Targeting PGLYRP1 promotes antitumor immunity while inhibiting autoimmune neuroinflammation. Nat Immunol. 2023;24:1908–20.
</cite> [<a href="https://doi.org/10.1038/s41590-023-01645-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10864036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37828379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schnell%20A,%20Huang%20L,%20Regan%20BML,%20Singh%20V,%20Vonficht%20D,%20Bollhagen%20A,%20Wang%20M,%20Hou%20Y,%20Bod%20L,%20Sobel%20RA,%20et%20al.%20Targeting%20PGLYRP1%20promotes%20antitumor%20immunity%20while%20inhibiting%20autoimmune%20neuroinflammation.%20Nat%20Immunol.%202023;24:1908%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Denzer L, Muranyi W, Schroten H, Schwerk C. The role of PLVAP in endothelial cells. Cell Tissue Res. 2023;392:393–412.
</cite> [<a href="https://doi.org/10.1007/s00441-023-03741-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10172233/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36781482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Denzer%20L,%20Muranyi%20W,%20Schroten%20H,%20Schwerk%20C.%20The%20role%20of%20PLVAP%20in%20endothelial%20cells.%20Cell%20Tissue%20Res.%202023;392:393%E2%80%93412." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Zhu J, Huang Y, Lei L, Zhao Z, Wang H, Li Z, Liu H, Wang J, Zheng Z. CRLF1/CLCF1 heterodimer involvement in intervertebral disc degeneration via exacerbation of extracellular matrix degradation and nucleus pulposus cell senescence. Osteoarthritis Cartilage. 2025;33:574−589.</cite> [<a href="https://doi.org/10.1016/j.joca.2025.02.773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39986601/" class="usa-link">PubMed</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Hu R, Dun X, Singh L, Banton MC. Runx2 regulates peripheral nerve regeneration to promote Schwann cell migration and re-myelination. Neural Regen Res. 2024;19:1575–83.
</cite> [<a href="https://doi.org/10.4103/1673-5374.387977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10883509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38051902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu%20R,%20Dun%20X,%20Singh%20L,%20Banton%20MC.%20Runx2%20regulates%20peripheral%20nerve%20regeneration%20to%20promote%20Schwann%20cell%20migration%20and%20re-myelination.%20Neural%20Regen%20Res.%202024;19:1575%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Bao M, Chen Y, Liu JT, Bao H, Wang WB, Qi YX, Lv F. Extracellular matrix stiffness controls VEGF(165) secretion and neuroblastoma angiogenesis via the YAP/RUNX2/SRSF1 axis. Angiogenesis. 2022;25:71–86.
</cite> [<a href="https://doi.org/10.1007/s10456-021-09804-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34170441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bao%20M,%20Chen%20Y,%20Liu%20JT,%20Bao%20H,%20Wang%20WB,%20Qi%20YX,%20Lv%20F.%20Extracellular%20matrix%20stiffness%20controls%20VEGF(165)%20secretion%20and%20neuroblastoma%20angiogenesis%20via%20the%20YAP/RUNX2/SRSF1%20axis.%20Angiogenesis.%202022;25:71%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Bartsch U. Neural CAMS and their role in the development and organization of myelin sheaths. Front Biosci. 2003;8:d477–490.
</cite> [<a href="https://doi.org/10.2741/1028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12456309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bartsch%20U.%20Neural%20CAMS%20and%20their%20role%20in%20the%20development%20and%20organization%20of%20myelin%20sheaths.%20Front%20Biosci.%202003;8:d477%E2%80%93490." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Chen JT, Hu X, Otto IUC, Schürger C, von Bieberstein BR, Doppler K, Krug SM, Hankir MK, Blasig R, Sommer C, et al. Myelin barrier breakdown, mechanical hypersensitivity, and painfulness in polyneuropathy with claudin-12 deficiency. Neurobiol Dis. 2023;185:106246.
</cite> [<a href="https://doi.org/10.1016/j.nbd.2023.106246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37527762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20JT,%20Hu%20X,%20Otto%20IUC,%20Sch%C3%BCrger%20C,%20von%20Bieberstein%20BR,%20Doppler%20K,%20Krug%20SM,%20Hankir%20MK,%20Blasig%20R,%20Sommer%20C,%20et%20al.%20Myelin%20barrier%20breakdown,%20mechanical%20hypersensitivity,%20and%20painfulness%20in%20polyneuropathy%20with%20claudin-12%20deficiency.%20Neurobiol%20Dis.%202023;185:106246." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Kadomatsu K, Muramatsu T. Midkine and pleiotrophin in neural development and cancer. Cancer Lett. 2004;204:127–43.
</cite> [<a href="https://doi.org/10.1016/S0304-3835(03)00450-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15013213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kadomatsu%20K,%20Muramatsu%20T.%20Midkine%20and%20pleiotrophin%20in%20neural%20development%20and%20cancer.%20Cancer%20Lett.%202004;204:127%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Jin L, Jianghai C, Juan L, Hao K. Pleiotrophin and peripheral nerve injury. Neurosurg Rev. 2009;32:387–93.
</cite> [<a href="https://doi.org/10.1007/s10143-009-0202-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19424734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin%20L,%20Jianghai%20C,%20Juan%20L,%20Hao%20K.%20Pleiotrophin%20and%20peripheral%20nerve%20injury.%20Neurosurg%20Rev.%202009;32:387%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (3.5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM2_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (1.4MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (1.7MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (2.6MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM5_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (2.9MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM6_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (2.6MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM7_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7</a><sup> (2.1MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM8_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8</a><sup> (2.8MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374453/bin/12974_2025_3514_MOESM9_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 9</a><sup> (2.4MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The raw sequencing data generated in this study have been deposited in the NCBI Sequence Read Archive (SRA). The single-cell RNA-seq data are available under BioProject accession number SAMN48188757, and the bulk RNA-seq data (spatiotemporal profiling of proximal and distal sciatic nerve segments) are available under BioProject accession number SAMN49603701.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Journal of Neuroinflammation are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12974-025-03514-3"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12974_2025_Article_3514.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (16.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12374453/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12374453/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374453%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374453/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12374453/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12374453/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40849669/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12374453/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40849669/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12374453/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12374453/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="S2dO7cKlXONRd4eVylRQsCYeBf0QwjFDa8lsnlVm8qpJNvgpWgg54ocvffuXpj1z">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
