
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            HIF-1 attenuates high-fiber diet-mediated proliferation and stemness of colonic epithelium - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15254D4C8AF3DAC3054D4C000402265A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="gmic">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369635/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Gut Microbes">
<meta name="citation_title" content="HIF-1 attenuates high-fiber diet-mediated proliferation and stemness of colonic epithelium">
<meta name="citation_author" content="Pollyana Ribeiro Castro">
<meta name="citation_author_institution" content="Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Renan Oliveira Corrêa">
<meta name="citation_author_institution" content="Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author_institution" content="Université Paris Cité, Imagine Institute, INSERM UMR1163, Laboratory of Intestinal Immunity, Paris, France">
<meta name="citation_author" content="Monara Kaélle Sérvulo Cruz Angelim">
<meta name="citation_author_institution" content="Laboratory of Immunometabolism, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Vinícius de Rezende Rodovalho">
<meta name="citation_author_institution" content="Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author_institution" content="Bioinformatics Laboratory (LABINFO), National Laboratory for Scientific Computing (LNCC), Petrópolis, Brazil">
<meta name="citation_author" content="Mariane Font Fernandes">
<meta name="citation_author_institution" content="Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Vinícius Dias Nirello">
<meta name="citation_author_institution" content="Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Arilson Bernardo dos Santos Pereira Gomes">
<meta name="citation_author_institution" content="Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Jaqueline de Souza Felipe">
<meta name="citation_author_institution" content="Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Lais Passarielo Pral">
<meta name="citation_author_institution" content="Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Sarah de Oliveira">
<meta name="citation_author_institution" content="Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Patrícia Brito Rodrigues">
<meta name="citation_author_institution" content="Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="José Luís Fachi">
<meta name="citation_author_institution" content="Department of Pathology and Immunology, Washington University School of Medicine in Saint Louis, Saint Louis, MO, USA">
<meta name="citation_author" content="Guilherme Reis-de-Oliveira">
<meta name="citation_author_institution" content="Laboratory of Neuroproteomics, Department of Biochemistry and Tissue Biology, Institute of Biology, University of Campinas (UNICAMP), Campinas, SP, Brazil">
<meta name="citation_author_institution" content="Boldrini Research Center, Campinas, SP, Brazil">
<meta name="citation_author" content="Bradley J Smith">
<meta name="citation_author_institution" content="Laboratory of Neuroproteomics, Department of Biochemistry and Tissue Biology, Institute of Biology, University of Campinas (UNICAMP), Campinas, SP, Brazil">
<meta name="citation_author" content="Victor C Carregari">
<meta name="citation_author_institution" content="Laboratory of Neuroproteomics, Department of Biochemistry and Tissue Biology, Institute of Biology, University of Campinas (UNICAMP), Campinas, SP, Brazil">
<meta name="citation_author" content="Nicolas G Shealy">
<meta name="citation_author_institution" content="Department of Pathology, Microbiology &amp; Immunology, Vanderbilt University Medical Center, Nashville, TN, USA">
<meta name="citation_author" content="Catherine Shelton">
<meta name="citation_author_institution" content="Department of Pathology, Microbiology &amp; Immunology, Vanderbilt University Medical Center, Nashville, TN, USA">
<meta name="citation_author" content="Daniel Martins-de-Souza">
<meta name="citation_author_institution" content="Laboratory of Neuroproteomics, Department of Biochemistry and Tissue Biology, Institute of Biology, University of Campinas (UNICAMP), Campinas, SP, Brazil">
<meta name="citation_author_institution" content="Experimental Medicine Research Cluster (EMRC), Campinas, SP, Brazil">
<meta name="citation_author_institution" content="National Institute of Science and Technology in Modeling Human Complex Diseases with 3D Platforms (INCT Model3D), São Paulo, São Paulo, Brazil">
<meta name="citation_author" content="Pedro M Moraes–Vieira">
<meta name="citation_author_institution" content="Laboratory of Immunometabolism, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author_institution" content="Experimental Medicine Research Cluster (EMRC), Campinas, SP, Brazil">
<meta name="citation_author_institution" content="Obesity and Comorbidities Research Center (OCRC), University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author" content="Mariana Xavier Byndloss">
<meta name="citation_author_institution" content="Department of Pathology, Microbiology &amp; Immunology, Vanderbilt University Medical Center, Nashville, TN, USA">
<meta name="citation_author_institution" content="Howard Hughes Medical Institute, Vanderbilt University Medical Center, Nashville, TN, USA">
<meta name="citation_author" content="Patrick Varga-Weisz">
<meta name="citation_author_institution" content="International Laboratory for Microbiome Host Epigenetics, Department of Genetics, Evolution, Microbiology, and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author_institution" content="School of Life Sciences, University of Essex, Colchester, UK">
<meta name="citation_author" content="Marco Aurélio Ramirez Vinolo">
<meta name="citation_author_institution" content="Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author_institution" content="Experimental Medicine Research Cluster (EMRC), Campinas, SP, Brazil">
<meta name="citation_author_institution" content="National Institute of Science and Technology in Modeling Human Complex Diseases with 3D Platforms (INCT Model3D), São Paulo, São Paulo, Brazil">
<meta name="citation_author_institution" content="Obesity and Comorbidities Research Center (OCRC), University of Campinas, Campinas, SP, Brazil">
<meta name="citation_author_institution" content="International Laboratory for Microbiome Host Epigenetics, Department of Genetics, Evolution, Microbiology, and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="17">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="2543123">
<meta name="citation_doi" content="10.1080/19490976.2025.2543123">
<meta name="citation_pmid" content="40827535">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369635/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369635/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369635/pdf/KGMI_17_2543123.pdf">
<meta name="description" content="The complex interplay between diet, microbiota, and host is exemplified by the effects of dietary fiber on the intestine. Inulin ingestion triggers epithelial changes in the colon that depend on microbiota-derived molecules, including enhanced ...">
<meta name="og:title" content="HIF-1 attenuates high-fiber diet-mediated proliferation and stemness of colonic epithelium">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The complex interplay between diet, microbiota, and host is exemplified by the effects of dietary fiber on the intestine. Inulin ingestion triggers epithelial changes in the colon that depend on microbiota-derived molecules, including enhanced ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369635/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12369635">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1080/19490976.2025.2543123"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/KGMI_17_2543123.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369635%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12369635/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12369635/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369635/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-gmic.png" alt="Gut Microbes logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Gut Microbes" title="Link to Gut Microbes" shape="default" href="https://doi.org/10.1080/19490976.2025.2543123" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Gut Microbes</button></div>. 2025 Aug 19;17(1):2543123. doi: <a href="https://doi.org/10.1080/19490976.2025.2543123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1080/19490976.2025.2543123</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Gut%20Microbes%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gut%20Microbes%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Gut%20Microbes%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Gut%20Microbes%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>HIF-1 attenuates high-fiber diet-mediated proliferation and stemness of colonic epithelium</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ribeiro%20Castro%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Pollyana Ribeiro Castro</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Pollyana Ribeiro Castro</span></h3>
<div class="p">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ribeiro%20Castro%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pollyana Ribeiro Castro</span></a>
</div>
</div>
<sup>a,</sup><sup>✉,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Corr%C3%AAa%20RO%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Renan Oliveira Corrêa</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Renan Oliveira Corrêa</span></h3>
<div class="p">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div class="p">
<sup>b</sup>Université Paris Cité, Imagine Institute, INSERM UMR1163, Laboratory of Intestinal Immunity, Paris, France</div>
<div>Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Corr%C3%AAa%20RO%22%5BAuthor%5D" class="usa-link"><span class="name western">Renan Oliveira Corrêa</span></a>
</div>
</div>
<sup>a,</sup><sup>b,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22S%C3%A9rvulo%20Cruz%20Angelim%20MK%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Monara Kaélle Sérvulo Cruz Angelim</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Monara Kaélle Sérvulo Cruz Angelim</span></h3>
<div class="p">
<sup>c</sup>Laboratory of Immunometabolism, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div>Investigation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22S%C3%A9rvulo%20Cruz%20Angelim%20MK%22%5BAuthor%5D" class="usa-link"><span class="name western">Monara Kaélle Sérvulo Cruz Angelim</span></a>
</div>
</div>
<sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Rezende%20Rodovalho%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Vinícius de Rezende Rodovalho</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Vinícius de Rezende Rodovalho</span></h3>
<div class="p">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div class="p">
<sup>d</sup>Bioinformatics Laboratory (LABINFO), National Laboratory for Scientific Computing (LNCC), Petrópolis, Brazil</div>
<div>Data curation, Formal analysis, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Rezende%20Rodovalho%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vinícius de Rezende Rodovalho</span></a>
</div>
</div>
<sup>a,</sup><sup>d</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Font%20Fernandes%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Mariane Font Fernandes</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Mariane Font Fernandes</span></h3>
<div class="p">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div>Formal analysis, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Font%20Fernandes%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariane Font Fernandes</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dias%20Nirello%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Vinícius Dias Nirello</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Vinícius Dias Nirello</span></h3>
<div class="p">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div>Formal analysis, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dias%20Nirello%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vinícius Dias Nirello</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22dos%20Santos%20Pereira%20Gomes%20AB%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Arilson Bernardo dos Santos Pereira Gomes</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Arilson Bernardo dos Santos Pereira Gomes</span></h3>
<div class="p">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div>Investigation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22dos%20Santos%20Pereira%20Gomes%20AB%22%5BAuthor%5D" class="usa-link"><span class="name western">Arilson Bernardo dos Santos Pereira Gomes</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Souza%20Felipe%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jaqueline de Souza Felipe</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jaqueline de Souza Felipe</span></h3>
<div class="p">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div>Methodology, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Souza%20Felipe%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jaqueline de Souza Felipe</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Passarielo%20Pral%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Lais Passarielo Pral</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Lais Passarielo Pral</span></h3>
<div class="p">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div>Investigation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Passarielo%20Pral%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lais Passarielo Pral</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Oliveira%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Sarah de Oliveira</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Sarah de Oliveira</span></h3>
<div class="p">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div>Investigation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22de%20Oliveira%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarah de Oliveira</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodrigues%20PB%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Patrícia Brito Rodrigues</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Patrícia Brito Rodrigues</span></h3>
<div class="p">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div>Investigation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodrigues%20PB%22%5BAuthor%5D" class="usa-link"><span class="name western">Patrícia Brito Rodrigues</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fachi%20JL%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">José Luís Fachi</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">José Luís Fachi</span></h3>
<div class="p">
<sup>e</sup>Department of Pathology and Immunology, Washington University School of Medicine in Saint Louis, Saint Louis, MO, USA</div>
<div>Investigation, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fachi%20JL%22%5BAuthor%5D" class="usa-link"><span class="name western">José Luís Fachi</span></a>
</div>
</div>
<sup>e</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reis-de-Oliveira%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Guilherme Reis-de-Oliveira</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Guilherme Reis-de-Oliveira</span></h3>
<div class="p">
<sup>f</sup>Laboratory of Neuroproteomics, Department of Biochemistry and Tissue Biology, Institute of Biology, University of Campinas (UNICAMP), Campinas, SP, Brazil</div>
<div class="p">
<sup>g</sup>Boldrini Research Center, Campinas, SP, Brazil</div>
<div>Visualization, Formal analysis, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reis-de-Oliveira%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guilherme Reis-de-Oliveira</span></a>
</div>
</div>
<sup>f,</sup><sup>g</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20BJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Bradley J Smith</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Bradley J Smith</span></h3>
<div class="p">
<sup>f</sup>Laboratory of Neuroproteomics, Department of Biochemistry and Tissue Biology, Institute of Biology, University of Campinas (UNICAMP), Campinas, SP, Brazil</div>
<div>Investigation, Formal analysis, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Smith%20BJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Bradley J Smith</span></a>
</div>
</div>
<sup>f</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carregari%20VC%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Victor C Carregari</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Victor C Carregari</span></h3>
<div class="p">
<sup>f</sup>Laboratory of Neuroproteomics, Department of Biochemistry and Tissue Biology, Institute of Biology, University of Campinas (UNICAMP), Campinas, SP, Brazil</div>
<div>Investigation, Formal analysis, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carregari%20VC%22%5BAuthor%5D" class="usa-link"><span class="name western">Victor C Carregari</span></a>
</div>
</div>
<sup>f</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shealy%20NG%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Nicolas G Shealy</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Nicolas G Shealy</span></h3>
<div class="p">
<sup>h</sup>Department of Pathology, Microbiology &amp; Immunology, Vanderbilt University Medical Center, Nashville, TN, USA</div>
<div>Investigation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shealy%20NG%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicolas G Shealy</span></a>
</div>
</div>
<sup>h</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shelton%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Catherine Shelton</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Catherine Shelton</span></h3>
<div class="p">
<sup>h</sup>Department of Pathology, Microbiology &amp; Immunology, Vanderbilt University Medical Center, Nashville, TN, USA</div>
<div>Investigation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shelton%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Catherine Shelton</span></a>
</div>
</div>
<sup>h</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Martins-de-Souza%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Daniel Martins-de-Souza</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Daniel Martins-de-Souza</span></h3>
<div class="p">
<sup>f</sup>Laboratory of Neuroproteomics, Department of Biochemistry and Tissue Biology, Institute of Biology, University of Campinas (UNICAMP), Campinas, SP, Brazil</div>
<div class="p">
<sup>i</sup>Experimental Medicine Research Cluster (EMRC), Campinas, SP, Brazil</div>
<div class="p">
<sup>j</sup>National Institute of Science and Technology in Modeling Human Complex Diseases with 3D Platforms (INCT Model3D), São Paulo, São Paulo, Brazil</div>
<div>Funding acquisition, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Martins-de-Souza%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniel Martins-de-Souza</span></a>
</div>
</div>
<sup>f,</sup><sup>i,</sup><sup>j</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moraes%E2%80%93Vieira%20PM%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Pedro M Moraes–Vieira</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Pedro M Moraes–Vieira</span></h3>
<div class="p">
<sup>c</sup>Laboratory of Immunometabolism, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div class="p">
<sup>i</sup>Experimental Medicine Research Cluster (EMRC), Campinas, SP, Brazil</div>
<div class="p">
<sup>k</sup>Obesity and Comorbidities Research Center (OCRC), University of Campinas, Campinas, SP, Brazil</div>
<div>Funding acquisition, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moraes%E2%80%93Vieira%20PM%22%5BAuthor%5D" class="usa-link"><span class="name western">Pedro M Moraes–Vieira</span></a>
</div>
</div>
<sup>c,</sup><sup>i,</sup><sup>k</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xavier%20Byndloss%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Mariana Xavier Byndloss</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Mariana Xavier Byndloss</span></h3>
<div class="p">
<sup>h</sup>Department of Pathology, Microbiology &amp; Immunology, Vanderbilt University Medical Center, Nashville, TN, USA</div>
<div class="p">
<sup>l</sup>Howard Hughes Medical Institute, Vanderbilt University Medical Center, Nashville, TN, USA</div>
<div>Funding acquisition, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xavier%20Byndloss%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariana Xavier Byndloss</span></a>
</div>
</div>
<sup>h,</sup><sup>l</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Varga-Weisz%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Patrick Varga-Weisz</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Patrick Varga-Weisz</span></h3>
<div class="p">
<sup>m</sup>International Laboratory for Microbiome Host Epigenetics, Department of Genetics, Evolution, Microbiology, and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div class="p">
<sup>n</sup>School of Life Sciences, University of Essex, Colchester, UK</div>
<div>Conceptualization, Funding acquisition, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Varga-Weisz%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Patrick Varga-Weisz</span></a>
</div>
</div>
<sup>m,</sup><sup>n,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramirez%20Vinolo%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Marco Aurélio Ramirez Vinolo</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Marco Aurélio Ramirez Vinolo</span></h3>
<div class="p">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div class="p">
<sup>i</sup>Experimental Medicine Research Cluster (EMRC), Campinas, SP, Brazil</div>
<div class="p">
<sup>j</sup>National Institute of Science and Technology in Modeling Human Complex Diseases with 3D Platforms (INCT Model3D), São Paulo, São Paulo, Brazil</div>
<div class="p">
<sup>k</sup>Obesity and Comorbidities Research Center (OCRC), University of Campinas, Campinas, SP, Brazil</div>
<div class="p">
<sup>m</sup>International Laboratory for Microbiome Host Epigenetics, Department of Genetics, Evolution, Microbiology, and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div>Conceptualization, Funding acquisition, Methodology, Resources, Supervision, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramirez%20Vinolo%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Marco Aurélio Ramirez Vinolo</span></a>
</div>
</div>
<sup>a,</sup><sup>i,</sup><sup>j,</sup><sup>k,</sup><sup>m,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff0001">
<sup>a</sup>Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div id="aff0002">
<sup>b</sup>Université Paris Cité, Imagine Institute, INSERM UMR1163, Laboratory of Intestinal Immunity, Paris, France</div>
<div id="aff0003">
<sup>c</sup>Laboratory of Immunometabolism, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div id="aff0004">
<sup>d</sup>Bioinformatics Laboratory (LABINFO), National Laboratory for Scientific Computing (LNCC), Petrópolis, Brazil</div>
<div id="aff0005">
<sup>e</sup>Department of Pathology and Immunology, Washington University School of Medicine in Saint Louis, Saint Louis, MO, USA</div>
<div id="aff0006">
<sup>f</sup>Laboratory of Neuroproteomics, Department of Biochemistry and Tissue Biology, Institute of Biology, University of Campinas (UNICAMP), Campinas, SP, Brazil</div>
<div id="aff0007">
<sup>g</sup>Boldrini Research Center, Campinas, SP, Brazil</div>
<div id="aff0008">
<sup>h</sup>Department of Pathology, Microbiology &amp; Immunology, Vanderbilt University Medical Center, Nashville, TN, USA</div>
<div id="aff0009">
<sup>i</sup>Experimental Medicine Research Cluster (EMRC), Campinas, SP, Brazil</div>
<div id="aff0010">
<sup>j</sup>National Institute of Science and Technology in Modeling Human Complex Diseases with 3D Platforms (INCT Model3D), São Paulo, São Paulo, Brazil</div>
<div id="aff0011">
<sup>k</sup>Obesity and Comorbidities Research Center (OCRC), University of Campinas, Campinas, SP, Brazil</div>
<div id="aff0012">
<sup>l</sup>Howard Hughes Medical Institute, Vanderbilt University Medical Center, Nashville, TN, USA</div>
<div id="aff0013">
<sup>m</sup>International Laboratory for Microbiome Host Epigenetics, Department of Genetics, Evolution, Microbiology, and Immunology, Institute of Biology, University of Campinas, Campinas, SP, Brazil</div>
<div id="aff0014">
<sup>n</sup>School of Life Sciences, University of Essex, Colchester, UK</div>
<div class="author-notes p">
<div class="fn" id="an0001">
<sup>✉</sup><p class="display-inline">CONTACT Pollyana Ribeiro Castro <span>pollyanarc@gmail.com</span></p>
</div>
<div class="fn" id="an0002">
<sup>✉</sup><p class="display-inline">Marco Aurélio Ramirez Vinolo <span>mvinolo@unicamp.br</span> Laboratory of Immunoinflammation, Department of Genetics, Evolution, Microbiology and Immunology, Institute of Biology, University of Campinas, Campinas, SP
13083-862, Brazil</p>
</div>
<div class="fn" id="an0003">
<sup>✉</sup><p class="display-inline">Patrick Varga-Weisz <span>patrick.varga-weisz@essex.ac.uk</span> International Laboratory for Microbiome Host Epigenetics, Department of Genetics, Evolution, Microbiology, and Immunology, Institute of Biology, University of Campinas, Campinas, SP
13083-862, Brazil</p>
</div>
<div class="fn" id="an0004">
<sup>*</sup><p class="display-inline">These authors contributed equally.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Pollyana Ribeiro Castro</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Renan Oliveira Corrêa</span></strong>: <span class="role">Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Monara Kaélle Sérvulo Cruz Angelim</span></strong>: <span class="role">Investigation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Vinícius de Rezende Rodovalho</span></strong>: <span class="role">Data curation, Formal analysis, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mariane Font Fernandes</span></strong>: <span class="role">Formal analysis, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Vinícius Dias Nirello</span></strong>: <span class="role">Formal analysis, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Arilson Bernardo dos Santos Pereira Gomes</span></strong>: <span class="role">Investigation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jaqueline de Souza Felipe</span></strong>: <span class="role">Methodology, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Lais Passarielo Pral</span></strong>: <span class="role">Investigation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Sarah de Oliveira</span></strong>: <span class="role">Investigation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Patrícia Brito Rodrigues</span></strong>: <span class="role">Investigation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">José Luís Fachi</span></strong>: <span class="role">Investigation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Guilherme Reis-de-Oliveira</span></strong>: <span class="role">Visualization, Formal analysis, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Bradley J Smith</span></strong>: <span class="role">Investigation, Formal analysis, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Victor C Carregari</span></strong>: <span class="role">Investigation, Formal analysis, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Nicolas G Shealy</span></strong>: <span class="role">Investigation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Catherine Shelton</span></strong>: <span class="role">Investigation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Daniel Martins-de-Souza</span></strong>: <span class="role">Funding acquisition, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Pedro M Moraes–Vieira</span></strong>: <span class="role">Funding acquisition, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mariana Xavier Byndloss</span></strong>: <span class="role">Funding acquisition, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Patrick Varga-Weisz</span></strong>: <span class="role">Conceptualization, Funding acquisition, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Marco Aurélio Ramirez Vinolo</span></strong>: <span class="role">Conceptualization, Funding acquisition, Methodology, Resources, Supervision, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 15; Accepted 2025 Jul 30; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). Published with license by Taylor &amp; Francis Group, LLC.</div>
<p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited. The terms on which this article has been published allow the posting of the Accepted Manuscript in a repository by the author(s) or with their consent.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12369635  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40827535/" class="usa-link">40827535</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>The complex interplay between diet, microbiota, and host is exemplified by the effects of dietary fiber on the intestine. Inulin ingestion triggers epithelial changes in the colon that depend on microbiota-derived molecules, including enhanced proliferation, increased mucus production, and elevated antimicrobial peptide secretion. Here we employed a multilayered and multi-omics approach, including dietary interventions, intestinal organoids, and both genetic and pharmacological interventions to investigate the impact of inulin on two aspects of diet-microbiota-host interactions: intestinal hypoxia and hypoxia-inducible factor (HIF)-1 signaling in intestinal epithelial cells (IECs). We found that inulin, a soluble fiber, promotes intestinal hypoxia, stabilizing HIF-1 in IECs in a microbiota- and host-dependent manner. Furthermore, we show that HIF-1 stabilization modulates intestinal stem cell (ISC) function through metabolic reprogramming in a microbiota-dependent manner. Our findings reveal an unrecognized role for HIF-1 in orchestrating microbiota-dependent epithelial metabolism and proliferation in the colon, underscoring the complexity of diet-microbiota-host interactions.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>KEYWORDS:</strong> Hypoxia, dietary fiber, inulin, metabolism, microbiota, intestinal stem cells</p></section></section><section id="s0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The gut epithelium consists of a single layer of cells derived from intestinal stem cells (ISCs) located at the base of the intestinal crypts.<sup><a href="#cit0001" class="usa-link" aria-describedby="cit0001">1</a></sup> Beyond its absorptive and secretory roles, the epithelium serves as a physical barrier, protecting the lamina propria from potentially harmful components in the intestinal lumen. To
maintain host homeostasis, this dynamic structure continuously adapts to environmental factors such as microbiota composition, dietary intake, and oxygen levels.<sup><a href="#cit0002" class="usa-link" aria-describedby="cit0002">2</a>,<a href="#cit0003" class="usa-link" aria-describedby="cit0003">3</a></sup> Understanding how these factors communicate with each other in a steady state is important and our study seeks to bring more clarity to these complex interactions.</p>
<p>A healthy microbiota features high taxonomic diversity, microbial gene richness, and a stable core, essential for epithelial functions.<sup><a href="#cit0004" class="usa-link" aria-describedby="cit0004">4</a></sup> The microbiota maintains barrier integrity, facilitates nutrient metabolism, regulates immune responses, promotes cellular regeneration, and provides colonization resistance, collectively fostering intestinal health and preventing intestinal diseases.<sup><a href="#cit0005" class="usa-link" aria-describedby="cit0005">5</a></sup> Intestinal epithelial cells (IECs), in turn, express pattern recognition receptors (PRRs), such as Toll-like receptors (TLRs), which detect microbial components and mediate responses to both beneficial and harmful microbes.<sup><a href="#cit0006" class="usa-link" aria-describedby="cit0006">6</a></sup> Furthermore, IECs sense metabolites derived from microbial breakdown of dietary components, such as short-chain fatty acids (SCFAs) or aryl hydrocarbon receptor (AhR) ligands including tryptophan derivatives like kynurenine, indole-3-acetic acid, and indole-3-propionic acid. These metabolites play critical roles in energy metabolism, production of mucus and antimicrobial peptides, and mucosal healing.<sup><a href="#cit0007" class="usa-link" aria-describedby="cit0007">7–9</a></sup></p>
<p>Additionally, dietary choices greatly influence the intestinal environment. Consumption of prebiotics, such as dietary fiber inulin or fructooligosaccharides, offers health benefits by modulating specific bacterial strains and boosting the production of microbial-derived metabolites.<sup><a href="#cit0010" class="usa-link" aria-describedby="cit0010">10</a></sup> Recent findings reveal that inulin intake remodels the colonic epithelium by enhancing ISC proliferation and differentiation, a process dependent on coordinated interactions between the gut microbiota and the immune system.<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a></sup> In addition to fibers, several other dietary signals have been shown to significantly influence ISC fate decisions,<sup><a href="#cit0012" class="usa-link" aria-describedby="cit0012">12</a></sup> highlighting the pivotal role of diet in shaping epithelial renewal, composition and functions.</p>
<p>Fluctuations in intestinal oxygen levels, shaped by local metabolic activity and physiological conditions, impact the microbiota and epithelial homeostasis.<sup><a href="#cit0013" class="usa-link" aria-describedby="cit0013">13</a>,<a href="#cit0014" class="usa-link" aria-describedby="cit0014">14</a></sup> Oxygen levels in the colonic epithelium decline from ~ 3% in the submucosa to &lt; 1% in the luminal surface.<sup><a href="#cit0015" class="usa-link" aria-describedby="cit0015">15</a></sup> The SCFA butyrate sustains hypoxia by promoting oxidative metabolism in IECs,<sup><a href="#cit0016" class="usa-link" aria-describedby="cit0016">16</a></sup> consuming oxygen, and stabilizing hypoxia-inducible factor 1 (HIF-1), which in turn regulates glucose use and mitochondrial biogenesis, supporting epithelial homeostasis under hypoxic conditions.<sup><a href="#cit0017" class="usa-link" aria-describedby="cit0017">17</a>,<a href="#cit0018" class="usa-link" aria-describedby="cit0018">18</a></sup></p>
<p>Intestinal hypoxia and HIF-1 activation by the gut microbiota are essential for maintaining barrier integrity,<sup><a href="#cit0019" class="usa-link" aria-describedby="cit0019">19</a>,<a href="#cit0020" class="usa-link" aria-describedby="cit0020">20</a></sup> with disruptions driving pathological conditions. For instance, reduced butyrate-producing <em>Clostridia</em> after antibiotic treatment increase epithelial oxygenation, allowing expansion of pathogenic, aerobic <em>Salmonella</em>.<sup><a href="#cit0021" class="usa-link" aria-describedby="cit0021">21</a></sup> Similarly, high-fat diets impair IECs mitochondrial oxygen uptake, increasing susceptibility to anaerobic pathobionts like <em>E. coli</em>.<sup><a href="#cit0022" class="usa-link" aria-describedby="cit0022">22</a></sup> Hypoxia also worsens intestinal inflammation and epithelial damage during <em>Salmonella enterica</em> serovar Typhimurium infection,<sup><a href="#cit0023" class="usa-link" aria-describedby="cit0023">23</a></sup> emphasizing the need to understand how hypoxia mediates beneficial or harmful responses.</p>
<p>In this study, we explored how dietary fiber, microbiota, and hypoxia influence epithelial responses. Our findings revealed that both microbiota- and host-mediated mechanisms contribute to increased colonic hypoxia and HIF-1 activation in intestinal epithelial cells after inulin ingestion. Using conditional knockout mice, we identified HIF-1α as a key regulator of cell proliferation and intestinal stem cell function in response to high-fiber diet. We find these effects are associated with metabolic regulation of epithelial cells. Together, our findings reveal the intricate role of dietary and microbiota-derived factors in shaping oxygen availability and intestinal epithelium function.</p></section><section id="s0002"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="s0002-s2001"><h3 class="pmc_sec_title">2.1. Mice</h3>
<p>All animal procedures described in this project were approved by the Ethic Committee on the Use of Animals (CEUA) of the University of Campinas (Protocols #4537–1/2017, #5881–1/2021, #5967–1/2024) and Vanderbilt University Medical Center. C57BL/6 mice were purchased from the Multidisciplinary Center for Biological Research (CEMIB – IB/UNICAMP). ODD-luciferase, HIF-1α<sup>fl/fl</sup>, Vhl<sup>fl/fl</sup>, Pparg<sup>fl/fl</sup>, Lgr5-EGFP-IRES-creERT2 and Villin-cre mice were purchased from Jackson Laboratory. Conditional knockout mice (HIF-1α<sup>ΔIEC</sup>, VHL<sup>ΔIEC</sup> and Pparg<sup>ΔIEC</sup>) were generated after crossing mice with the floxed genes (HIF-1α<sup>fl/fl</sup>, VHL<sup>fl/fl</sup> and Pparg<sup>fl/fl</sup>) and mice that express the Cre recombinase enzyme under the
control of the Villin promoter (Vil-Cre) or Lgr5 (Lgr5-EGFP-IRES-creERT2, Lgr5-Cre). HIF-1α<sup>ΔISC</sup> mice were generated after crossing Lgr5-Cre with Ai14 (B6.Cg-Gt(ROSA)26Sortm14(CAG-tdTomato)Hze/J) and HIF-1α<sup>fl/fl</sup>. Other strains were purchased from Jackson Laboratory or provided by collaborators from the Department of Microbiology, Institute of Biological Science of the Federal University of Minas Gerais. All mice were kept in the Animal Facility of the Department of Genetics, Evolution, Microbiology and Immunology of the Institute of Biology of the University of Campinas in cages with a filter on the lid and had <em>ad libitum</em> access to water and sterilized food. C57Bl/6N germ-free mice (Taconic, B6NTac – GF) were bred in-house and moved into a positive pressure cage system at 3 weeks old in the Animal Facility at Vanderbilt University Medical Center. Adult male and female mice aged 8 to 12 weeks were used in all experiments.</p></section><section id="s0002-s2002"><h3 class="pmc_sec_title">2.2. Dietary approaches</h3>
<p>Control and inulin diets were prepared at the Laboratory of Cereals, Roots, and Tubers (FEA/UNICAMP) according to the American Institute of Nutrition recommendations (AIN93).<sup><a href="#cit0024" class="usa-link" aria-describedby="cit0024">24</a></sup> The control diet was composed of 5% insoluble fiber cellulose, and the inulin-enriched diet was composed of 5% cellulose + 10% inulin (soluble fiber). Details of each diet composition can be found in our previous publication.<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a></sup> Mice were fed for 3–4 weeks, unless otherwise mentioned. Body weight was measured for each animal before and after the diets. After euthanasia, intestines were harvested and measured, with their values normalized by each individual animal’s body weight. For antibiotic experiments, C57BL/6 mice received a mix of 1 g/L ampicillin, 1 g/L neomycin, 1 g/L metronidazole, and 0.5 g/L vancomycin<sup><a href="#cit0025" class="usa-link" aria-describedby="cit0025">25</a></sup> in drinking water for 30 days, while being fed with the inulin-enriched diet. For etomoxir intervention, mice on the inulin diet were treated with etomoxir (20 mg/kg of body weight in 100 µL) or vehicle (100 µL of PBS) intraperitoneally, 24 hours and 2 hours before euthanasia.</p></section><section id="s0002-s2003"><h3 class="pmc_sec_title">2.3. Analysis of bacteria load</h3>
<p>To confirm microbiota depletion, fecal samples were collected from mice before and after diet and antibiotic treatments to assess bacterial load. Briefly, 50 mg of fecal material was used for genomic DNA extraction using the PureLink Microbiome DNA Purification Kit (Thermo Fisher Scientific). Bacterial load was quantified by quantitative real-time PCR (qPCR) using primers targeting a conserved region of the Eubacterial 16S rDNA gene (E338F forward: 5′-ACTCCTACGGGAGGCAGCAGT-3′ and U1407R reverse: 5′-ATTACCGCGGCGGCTGCTGGC-3′), as previously described by Durand et al. (2010).<sup><a href="#cit0026" class="usa-link" aria-describedby="cit0026">26</a></sup> A standard curve was established using serial dilutions of <em>Escherichia coli</em> genomic DNA.</p></section><section id="s0002-s2004"><h3 class="pmc_sec_title">2.4. Gnotobiotic mice</h3>
<p>Germ-free mice were monocolonized with individual bacterial strains via oral gavage using an inoculum of 1 × 10<sup>9</sup> CFU in 100 µL. Briefly, <em>Bacteroides ovatus</em> (ATCC 8483) and <em>Bacteroides thetaiotaomicron</em> (ATCC 29148D) strains were cultured anaerobically at 37°C in Brain Heart Infusion supplemented (BHIS) broth or on BHIS agar plates. For the inoculum preparation, bacteria were streaked onto BHIS agar plates. Single colonies were used to start cultures in BHIS broth. Bacteria were harvested by centrifugation at 3,000 ×g for 8 min and then pellets were resuspended in 1 mL fresh media. The optical density (OD) at 600 nm (OD600) was measured, and the number of bacteria was adjusted to 1 × 10<sup>10</sup> CFU/mL. A 0.1 mL suspension was then administered to each mouse. Fecal pellets were collected weekly under sterile conditions, weighed, and immediately homogenized in 1 mL of PBS. Serial dilutions were prepared and plated on BHIS agar to quantify bacterial colonization by colony counting.</p></section><section id="s0002-s2005"><h3 class="pmc_sec_title">2.5. Analysis of luciferase activity</h3>
<p>HIF-1 reporter mice (ODD-luciferase) were fed a control or inulin-enriched diet for 30 days. In accordance with a published protocol,<sup><a href="#cit0027" class="usa-link" aria-describedby="cit0027">27</a></sup> mice were anesthetized with isoflurane followed by intraperitoneal injection of luciferin (50 mg/kg). Images were taken every min for 5 min on the IVIS Spectrum In Vivo Imaging System.
Representative images shown in this manuscript were taken 5 min post luciferin inoculation. After euthanasia, the intestine of these mice was harvested, and a 1.5 cm of the proximal portion of the small intestine and colon was collected for the detection of luciferase activity <em>ex vivo</em>. To do so, tissues were first processed with 125 μL of lysis buffer +100 μM cobalt chloride using a tissue homogenizer. Next, samples were snap-frozen in liquid nitrogen and thawed at 37°C for 1 min each, three times. After centrifugation (16,000 ×g, 15 sec, 4 °C), the supernatant was collected, and luminescence (590/35) was measured (Synergy HT Biotek) immediately after the addition of the substrate. Results were normalized to total protein content, as determined by the Bradford assay.</p></section><section id="s0002-s2006"><h3 class="pmc_sec_title">2.6. Crypts isolation and single cell dissociation</h3>
<p>Colons were harvested, opened longitudinally, washed in cold PBS to remove feces. They were then cut into small pieces and placed in microtubes with Hanks’ balanced salt solution (HBSS) containing 10 mM EDTA and 1% P/S (penicillin/streptomycin). The samples were incubated with agitation (1000 rpm, 37 °C) for 20 min, after which the tissue pieces were removed, and the tubes were centrifuged for 5 min (300×g, 4°C). The pellet was washed with cold HBSS, centrifuged again, and resuspended in HBSS supplemented with 0.04% BSA, 1% P/S, and 10 mM dithiothreitol (DTT) to remove the mucus. After 10 min of incubation at room temperature, cold PBS supplemented with 0.04% BSA was added, and the suspension was filtered through a 70-μm cell strainer.</p></section><section id="s0002-s2007"><h3 class="pmc_sec_title">2.7. Histological analysis and hypoxyprobe</h3>
<p>Colon samples were fixed in 4% formalin followed by paraffin embedding. Tissue slides were obtained and stained with Hematoxylin – Eosin (H&amp;E) for histological analysis. Hypoxia detection was performed using the Hypoxyprobe™-1 kit (Hypoxyprobe, Inc.). Briefly, mice were injected intraperitoneally with pimonidazole solution (PMDZ) (60 mg/kg of body weight) 90 min before euthanasia. Colon tissues were fixed in 10% buffered formalin, paraffin-embedded, and incubated with monoclonal rabbit anti-pimonidazole IgG1 (Mab 4.3.11.3). Slides were washed and then incubated with a secondary antibody (Cy5-conjugated goat anti-rabbit antibody from Jackson ImmunoResearch Laboratories). The slides were mounted using SlowFade Gold Mountant with DAPI (Molecular Probes). Hypoxia scoring was performed based on the degree of epithelial staining: 0 (no hypoxia), 1 (mild focal hypoxia), 2 (moderate multifocal hypoxia), and 3 (intense diffuse hypoxia), as previously described.<sup><a href="#cit0028" class="usa-link" aria-describedby="cit0028">28</a></sup> Images were acquired using Cytation 9 (Synergy HT Biotek) and analyzed with ImageJ software.<sup><a href="#cit0029" class="usa-link" aria-describedby="cit0029">29</a></sup></p></section><section id="s0002-s2008"><h3 class="pmc_sec_title">2.8. EdU staining</h3>
<p>5-ethynyl-2´-deoxyuridine (EdU) was prepared at 10 mM in PBS and 100 µL was injected i.p. per animal 90 min before euthanasia. Colon tissues were fixed in 4% formalin followed by paraffin embedding. The slides were labeled with Click-It Plus EdU Kit (ThermoFisher Scientific) for incorporation of fluorescent azide (AlexaFluor) followed by counterstaining with DAPI using SlowFade Gold Mountant (Molecular Probes). Images were obtained using Cytation9 equipment (Synergy HT Biotek) and analyzed using ImageJ software. For <em>in vitro</em> experiments, organoids were incubated with 100 µM of EdU for 90 min (5% CO<sub>2</sub> and 37 °C). After this, the medium was removed, and the organoids were collected and labeled with Click-It Plus EdU Kit for flow cytometry analysis.</p></section><section id="s0002-s2009"><h3 class="pmc_sec_title">2.9. Intestinal organoids</h3>
<p>The distal portion of the colon was collected, washed in cold PBS, and incubated in PBS +10 mM EDTA for 30 min, at 37°C, shaking at 1000 rpm. After PBS washes, isolated crypts were counted and then seeded in Matrigel™ (Corning). Culture medium contained: Advanced DMEM (Gibco), Noggin (200 ng/mL; Peprotech), R-spondin (500 ng/mL; R&amp;D or Sino Biological), epidermal growth factor (EGF; 40 ng/mL; R&amp;D), N-acetyl-L-cysteine (1 μM; Sigma-Aldrich), Chiron (10 μM; Sigma-Aldrich), B27 (1X; Life Technologies), and Y-27632 dihydrochloride monohydrate (20 ng/mL;
Sigma-Aldrich). Culture was maintained at 37°C in fully humidified chambers containing 5% CO<sub>2</sub>. Clonogenicity and organoid volume were determined 5 days after seeding. Etomoxir (3 or 12.5 µM), oligomycin (1 µM), antimycin (1 µM), rotenone (100 nM) or 2-deoxyglucose (2-DG; 50 mM) were added to the organoids culture 24 h after initial plating. EdU (100 µM) was added to the organoid cultures 90 min prior to harvesting to assess cell proliferation by flow cytometry. To collect the organoids, the medium was removed, and 1 mL of ice-cold PBS was added to the wells containing the organoids. After this, the Matrigel® was broken with the aid of a pipette tip and the supernatant was collected and centrifuged (300×g, 5 min, 4°C). The resulting pellet was used in different analyses. Detection of EdU-labeled cells was performed using the Click-iT™ Plus EdU Flow Cytometry Kit (ThermoFisher Scientific).</p></section><section id="s0002-s2010"><h3 class="pmc_sec_title">2.10. MitoTracker green assay</h3>
<p>To evaluate the mitochondrial mass of organoids, we used the probe MitoTracker green (Invitrogen). For this, organoids were incubated with 50 nM of this probe for 30 min, then the medium was removed, and the organoids were washed with 1 mL of cold PBS. After this, we proceed with cell viability and EdU labeling. Cells were then analyzed on a flow cytometer, as described below.</p></section><section id="s0002-s2011"><h3 class="pmc_sec_title">2.11. Flow cytometry</h3>
<p>Intestinal epithelial cells from colon and organoids cells were obtained as previously described by Corrêa and cols (2023)<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a></sup>. To exclude non-viable cells, a live/dead viability assay (Zombie Aqua<sup>TM</sup> Fixable Viability Kit, Biolegend) was used. Intestinal epithelial cells were gated in live/dead negative population and EdU and/or MitoTracker positive cells (FITC positive) to evaluate proliferation and mitochondrial mass. All cells were diluted in Fluorescence-Activated Cell Sorting (FACS) buffer. Staining of the live/dead and surface antigens was performed in the dark, at 4°C, for 20 min. HIF-1α and Ki67 were stained with monoclonal antibodies after fixation and permeabilization of isolated cells. Samples were analyzed using a FACSymphony™ (BD Biosciences) and FACSDiva™ Software (BD biosciences). All FACS data were analyzed using FlowJo software (v10.1; Becton Dickinson).</p></section><section id="s0002-s2012"><h3 class="pmc_sec_title">2.12. Cell sorting</h3>
<p>Reporter mice Lgr5-Cre; Ai14; HIF-1α<sup>fl/fl</sup> with specific deletion of HIF-1α in ISCs (HIF-1α<sup>ΔISC</sup>) and their controls Lgr5-Cre; Ai14; HIF-1α<sup>fl/-</sup> (HIF-1α<sup>fl/-</sup>) were fed with inulin diet for 3 weeks and received a single tamoxifen (80 mg/kg) injection i.p. 24 hours before the euthanasia. Intestinal epithelial cells from colon and organoids cells were obtained as previously described. To exclude non-viable cells, a live/dead viability assay (Zombie Aqua<sup>TM</sup> Fixable Viability Kit, Biolegend) was used. IECs were gated in live/dead negative population and tdTomato positive cells. Cells were sorted using FACSDiscover<sup>TM</sup> S8 (BD Biosciences) and collected in culture media for organoids. FACS-sorted Tomato+ cells were plated for organoids culture as previously described.</p></section><section id="s0002-s2013"><h3 class="pmc_sec_title">2.13. Seahorse assay</h3>
<p>For real-time analysis of OCR (Oxygen Consumption Rate), colon organoids were analyzed with an XFe96 Extracellular Flux Analyzer (Seahorse Bioscience) according to manufacturer’s instructions and as described by Ludikhuize et al. (2021).<sup><a href="#cit0030" class="usa-link" aria-describedby="cit0030">30</a></sup> Briefly, 20 organoids in 3 µL of Matrigel® per well were seeded onto an XF96 plate (Seahorse). Due to differences in the growth rates between HIF-1α<sup>ΔIEC</sup> and HIF-1α<sup>fl/fl</sup> organoids, the HIF-1α<sup>fl/fl</sup> organoids were plated two days earlier than the HIF-1α<sup>ΔIEC</sup> organoids and subsequently grown in complete organoid culture media for 4 days prior to the analysis. This approach ensured that, at the time of measurement, organoids from both groups were of comparable size. Three or more consecutive measurements were obtained under baseline conditions and after the sequential addition of specific inhibitors/activators from the metabolic tests. For the mitochondrial stress test, we added 1 µM of oligomycin, 2 µM of FCCP (fluoro-carbonyl cyanide phenylhydrazone), and 100 mM of rotenone +1 µM of
antimycin A (all Sigma-Aldrich reagents). The results of the Seahorse measurements were normalized based on the number of organoids present in the culture.</p></section><section id="s0002-s2014"><h3 class="pmc_sec_title">2.14. Single-cell RNA sequencing data analysis</h3>
<p>Single-cell RNA sequencing (scRNA-seq) data was obtained from our previous study.<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a></sup> All cell annotations and dimensionality reductions used in this analysis were retained, as described in the original publication. Differentially expressed genes (DEGs) were identified using the FindMarkers function from the Seurat package (version 5.1.0). This function employs the Wilcoxon Rank-Sum test to compare two groups of cells. Default parameters were used unless stated otherwise, and p-values were adjusted for multiple testing using the Bonferroni correction method. Gene ontology analysis was performed using DEGs. The R package enrichR was employed for Gene Ontology (GO) function annotation and Kyoto Encyclopedia of Genes and Genomes pathway (KEGG) enrichment analysis. Enriched status was determined based on a false discovery rate (FDR) &lt; 0.05. Only GO terms with 2 to 1000 assigned genes were considered for enrichment status. Redundancy reduction analysis was performed using the rrvgo package in R, where the calculate SimMatrix() function estimated semantic similarities between GO terms via the Wang method, and terms with a similarity threshold higher than 0.5 were grouped by the reduceSimMatrix() function. The smallest adjusted P-value was assigned for different GO terms grouped under the same parental GO term, and duplicates were removed. Only the top 5 terms related to intestine were included in the graphs. To verify the overlap of expressed genes between the cells, Venn diagrams were generated using the ggvenn package (<a href="https://github.com/yanlinlin82/ggvenn" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/yanlinlin82/ggvenn</a>.). Expression levels of oxidative phosphorylation (OXPHOS)-related genes, previously identified through gene ontology analysis, were averaged using Seurat by cell type, treatment, and sample ID. Heatmaps were generated with the dittoSeq package, displaying Z-score normalized expression across intestinal stem cells, transit-amplifying cells, and enterocytes.</p></section><section id="s0002-s2015"><h3 class="pmc_sec_title">2.15. Quantitative gene expression</h3>
<p>Total RNA was extracted from colonic epithelial cells using the PureLink™ RNA Mini Kit (Thermo Fisher Scientific), and its concentration and purity were assessed by NanoDrop spectrophotometer. cDNA was synthesized from 1 µg of RNA using the High-Capacity cDNA Reverse Transcription Kit (Applied Biosystems). RT-qPCR was performed using Power SYBR™ Green PCR Master Mix (Applied Biosystems) on a CFX384 Touch Real-Time PCR System (Bio-Rad). Reactions were conducted in triplicate in a 10 µL volume, with gene-specific primers (<a href="#t0001" class="usa-link">Table 1</a>). The thermal protocol consisted of 95°C for 10 min, followed by 40 cycles of 95°C for 15 s and 60°C for 1 min. Specificity was verified by melting curve analysis. Gene expression was normalized to β2-microglobulin as the housekeeping gene, and relative expression levels were calculated using the 2^−ΔΔCt method. Data analysis was performed on CFX Manager software (Bio-Rad) with β2-microglobulin (<em>B2m</em>) serving as the reference gene due to its stability and invariant expression
across groups.</p>
<section class="tw xbox font-sm" id="t0001"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Primer sequences used for RT-qPCR analysis.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="left" span="1">
<col width="1*" align="left" span="1">
</colgroup>
<thead><tr>
<th align="left" colspan="1" rowspan="1">Gene</th>
<th align="center" colspan="1" rowspan="1">Primer direction</th>
<th align="center" colspan="1" rowspan="1">Sequence (5‘→ 3’)</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="2" align="left" colspan="1"><em><strong>Hif1a</strong></em></td>
<td align="left" colspan="1" rowspan="1">Forward</td>
<td align="left" colspan="1" rowspan="1">ATCTCGGCGAAGCAAAGAGTC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Reverse</td>
<td align="left" colspan="1" rowspan="1">TGGGGAAGTGGCAACTGAT</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1"><em><strong>Slc2a1</strong></em></td>
<td align="left" colspan="1" rowspan="1">Forward</td>
<td align="left" colspan="1" rowspan="1">CTTTGTGGCCTTCTTTGAAGT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Reverse</td>
<td align="left" colspan="1" rowspan="1">CCACACAGTTGCTCCACAT</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1"><em><strong>Pfkfb3</strong></em></td>
<td align="left" colspan="1" rowspan="1">Forward</td>
<td align="left" colspan="1" rowspan="1">GGAGGTCGGCATGTTGAAGA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Reverse</td>
<td align="left" colspan="1" rowspan="1">CTTTGGAAGGGCCTGAGAGG</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1"><em><strong>Muc2</strong></em></td>
<td align="left" colspan="1" rowspan="1">Forward</td>
<td align="left" colspan="1" rowspan="1">CGACTGTGAGCAGTGTGTCT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Reverse</td>
<td align="left" colspan="1" rowspan="1">GGGTAGGGTCACCTCCATCT</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1"><em><strong>Tff3</strong></em></td>
<td align="left" colspan="1" rowspan="1">Forward</td>
<td align="left" colspan="1" rowspan="1">TGCAGATTACGTTGGCCTGT</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Reverse</td>
<td align="left" colspan="1" rowspan="1">TGCAGAGGTTTGAAGCACCA</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1"><em><strong>Cldn1</strong></em></td>
<td align="left" colspan="1" rowspan="1">Forward</td>
<td align="left" colspan="1" rowspan="1">CACTCCCAGACTCCACCACC</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Reverse</td>
<td align="left" colspan="1" rowspan="1">CGATCCATCCCAGAGAAGCC</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1"><em><strong>B2m</strong></em></td>
<td align="left" colspan="1" rowspan="1">Forward</td>
<td align="left" colspan="1" rowspan="1">CCCAGTGAGACTGATACATACG</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Reverse</td>
<td align="left" colspan="1" rowspan="1">CGATCCCAGTAGACGGTCTTG</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/t0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="s0002-s2016"><h3 class="pmc_sec_title">2.16. 16S rRNA gene processing and analysis</h3>
<p>DNA was extracted from colon luminal content using the PureLink™ Microbiome DNA Purification Kit from Thermo Fisher. The V3-V4 variable regions of the 16S rRNA gene were amplified and sequenced as 250-bp paired-end reads. The nf-core/ampliseq workflow version 2.11.0 was used to process the generated data.<sup><a href="#cit0031" class="usa-link" aria-describedby="cit0031">31–33</a></sup> Briefly, the quality of the data was evaluated with FastQC (Andrews, 2010 - <a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>.) and summarized with MultiQC.<sup><a href="#cit0034" class="usa-link" aria-describedby="cit0034">34</a></sup> DADA2 was used to process the sequences<sup><a href="#cit0035" class="usa-link" aria-describedby="cit0035">35</a></sup> eliminating PhiX contamination, discarding reads with more than 2 expected errors, correcting errors, merging read pairs, and removing PCR chimeras. Taxonomic classification was performed by DADA2, using the Silva 138.1 prokaryotic SSU database.<sup><a href="#cit0036" class="usa-link" aria-describedby="cit0036">36</a></sup> The amplicon sequencing variants (ASVs) sequences, abundance, and taxonomic assignments were loaded into QIIME2<sup><a href="#cit0037" class="usa-link" aria-describedby="cit0037">37</a></sup> and the ASVs with taxonomic strings containing mitochondria, chloroplast, or archaea were removed. Within QIIME2, the microbial community data was visualized in barplots and evaluated for sufficient sequencing depth with alpha rarefaction curves. Final visualizations were obtained as bar plots with the R libraries phyloseq,<sup><a href="#cit0038" class="usa-link" aria-describedby="cit0038">38</a></sup> microViz<sup><a href="#cit0039" class="usa-link" aria-describedby="cit0039">39</a></sup> and MicrobiotaProcess.<sup><a href="#cit0040" class="usa-link" aria-describedby="cit0040">40</a></sup> After rarefaction, Shannon’s index was used to assess alpha diversity, and Bray-Curtis distances were used to assess beta diversity. LEfSe (Linear discriminant analysis Effect Size) analysis was performed using the run_lefse function from the microbiomeMarker R package. The analysis was conducted at the genus taxonomic level with the statistical significance thresholds set as follows: Kruskal-Wallis test cutoff (kw_cutoff) of 0.05 to detect features with significant differences among classes, Wilcoxon rank-sum test cutoff (Wilcoxon_cutoff) of 0.05 for pairwise comparisons, and an LDA score cutoff (lda_cutoff) of 2.0 to estimate the effect size of each differentially abundant taxon.</p></section><section id="s0002-s2017"><h3 class="pmc_sec_title">2.17. Proteomic analysis</h3>
<p>Colon epithelial cells from wild-type and knockout mice (<em>n</em> = 10) were submitted to protein extraction using lysis buffer (100 mM Tris HCL, 150 mM NaCl, 1 mM EDTA, 1% Triton X) with freshly added protease inhibitors (cOmplete Mini Protease Inhibitor Cocktail, Roche). Tissue samples were incubated in lysis buffer for 30 min on ice before mechanical lysis by maceration, followed by ultrasonication (3 cycles of 20s, output 1.3) with a probe ultrasonic homogenizer (Cole-Parmer). Membranes and cellular debris were pelleted by centrifugation at 2000 x <em>g</em> for 10 min at 4°C and quantified using a microBCA kit (Thermo Fisher Scientific) with bovine serum albumin as the quantification curve. 20 micrograms of protein were washed, reduced, alkylated and digested using a modified filter-aided sample preparation (FASP) protocol.<sup><a href="#cit0041" class="usa-link" aria-describedby="cit0041">41</a></sup></p>
<p>Digested peptides from each sample were desiccated by SpeedVac vacuum concentrator (Eppendorf) and resuspended in 0.1% formic acid (FA). The separation of tryptic peptides was performed on an ACQUITY M-Class System (Waters Corporation). One microgram of each digested sample was loaded onto a Symmetry C18 trapping column (5 μm, 180 μm × 20 mm; Waters Corp.) and subsequently separated by a 120-min reversed phase chromatography method at 300 nL/min (linear gradient, 3–55% ACN over 90 min) using a HSS T3 C18 analytical column (1.8 μm, 75 μm × 150 mm; Waters Corp.) maintained at 40°C. For the gradient elution, water: formic acid (99.9:0.1, v/v) was used as eluent A and acetonitrile: formic acid (99.9:0.1, v/v) as eluent B. The separated peptides were analyzed by a Synapt G2-S<em>i</em> mass spectrometer directly coupled online to the chromatographic system. The eluting peptides were ionized by nano-electrospray ionization in positive mode (nESI+) with a capillary voltage of 2.5 kV. Peptide and fragment ions were acquired using full-spectrum (m/z 50–2000), data-independent acquisition of ions with alternating low (4 eV) and high (ramping 25–60 eV) energies in the collision cell with ion mobility separation (HDMSe) with a scan time of 1.0 s. Ion mobility separation was performed with a wave velocity of 1000 m/s and transfer wave velocity of 175 m/s. A time-aligned inventory of accurate mass flight times in the TOF separation chamber was performed using a reference lock mass ([Glu1]-Fibrinopeptide B Standard, Waters Corp.). Each sample was run in three or four aligned instrumental replicates to improve precursor identification.</p>
<p>Continuum LC-MS data from three replicate experiments for each sample were processed for peak picking, retention time alignment, normalization, and protein identification and quantitation using Progenesis QI for Proteomics (Version 4.0.6x; Waters Corp.). The protein identification was performed using a database search algorithm against the reviewed <em>Mus musculus</em> database (UNIPROT Proteome ID
UP000000589, retrieved June 2021, 17085 hits). Search parameters were set as: tolerance of 20 ppm for precursor ions and 10 ppm for product ions, maximum protein mass of 400 kDa, minimum 1 fragment ion matched per peptide, minimum 3 fragment ions matched per protein, minimum 1 unique peptide matched per protein, up to 2 missed cleavages, carbamidomethylation of cysteine as a fixed modification and oxidation of methionine as a variable modification, and 1% for false discovery rate (FDR), calculated using a reverse database library generated on-the-fly.</p>
<p>Label-free quantitative data were obtained using the relative abundance intensity integrated into Progenesis software, using all identified peptides for normalization. The expression analysis was performed considering technical replicates available for each experimental condition following the hypothesis that each group is an independent variable. Filtered tables were generated to include proteins found at least in two out of three technical replicates and to exclude proteins showing less than 20% change and those showing no statistical significance (ANOVA ≥0.05).</p>
<p>Downstream proteomics data analyses, including differential proteomic and enrichment analyses, were performed using OmicScope.<sup><a href="#cit0042" class="usa-link" aria-describedby="cit0042">42</a></sup> For enrichment analysis, we searched differentially regulated proteins against Biological Process – Gene Ontology database. Figures were generated with the OmicScope web application (<a href="https://omicscope.ib.unicamp.br" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://omicscope.ib.unicamp.br</a>.) and Cytoscape environment.<sup><a href="#cit0043" class="usa-link" aria-describedby="cit0043">43</a></sup></p></section><section id="s0002-s2018"><h3 class="pmc_sec_title">2.18. Statistical analysis</h3>
<p>Except for proteomic, scRNA-seq and 16s rRNA data, all statistical analyses were performed using GraphPad Prism 8.0 software. The results are presented as mean and standard deviation. Comparisons between two groups were conducted using unpaired two-tailed t-tests. For multiple group comparisons, one-way or two-way ANOVA was applied, as appropriate. Details of the statistical tests used are provided in the figure legends. Statistical significance was set at <em>p</em> &lt; 0.05.</p></section></section><section id="s0003"><h2 class="pmc_sec_title">3. Results</h2>
<section id="s0003-s2001"><h3 class="pmc_sec_title">3.1. Inulin ingestion triggers distinct transcriptional changes in colonic enterocytes and proliferating cells</h3>
<p>Our previous study highlighted inulin’s capacity to remodel the colonic epithelium through complex interactions between the microbiota and immune system.<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a></sup> In the original study, we generated a single-cell RNA sequencing dataset to examine shifts in the proportions of intestinal epithelial subtypes and the expression of cell cycle – related genes. Here, to assess the impact of a high-fiber diet, we re-analyzed the data,<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a></sup> comparing mice fed an inulin-supplemented diet versus a cellulose-fiber control diet, and evaluated transcriptional changes across epithelial cell subsets. For this analysis, we retained the original cell annotations and dimensionality reduction, as described in the original publication.<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a></sup></p>
<p>Inulin intake significantly influenced gene expression across absorptive (AC; 1,379 genes), proliferative (PC; 761 genes), and secretory (SEC; 243 genes) intestinal cell populations (Figure S1A-C). Although some DEGs, particularly those that were downregulated (Figure S1B), were shared among the epithelial populations, the majority of DEGs were unique to each cell type, suggesting that the inulin diet exerts cell-type-specific effects on IECs. Enrichment analysis using the KEGG database revealed that inulin-induced genes were associated with the ‘protein processing in the endoplasmic reticulum’ pathway, which was common to both SECs and ACs (Figure S1C). In ACs, other pathways were found to be enriched, including ‘RNA transport’ and metabolic pathways, namely ‘alanine, aspartate, and glutamate metabolism,’ ‘propanoate metabolism,’ and the ‘citrate cycle’ (Figure S1C). Conversely, downregulated genes were linked to terms such as ‘thermogenesis’ and ‘oxidative phosphorylation’ in PCs and SECs, while the genes in ‘ribosome’ were consistently repressed across all cell populations (Figure S1C). These results indicate that the inulin diet drives distinct transcriptional profiles in different epithelial cell types, potentially affecting key biological processes in the gut.</p>
<p>To better clarify these findings, we analyzed DEGs and functional terms within specific subsets of intestinal epithelial cells, uncovering contrasting effects of inulin-diet between enterocytes (ECs), which make up over 60% of the cells in our dataset, and the two main proliferative subtypes: cycling transit-
amplifying cells (TAs) and ISCs. A comparison of the DEGs across these three IEC subsets revealed that most upregulated genes were specific to enterocytes (<a href="#f0001" class="usa-link">Figure 1A</a>), whereas a significant portion of the genes downregulated by inulin were shared between TAs and ISCs (approximately 50% of the total DEGs), with a much smaller proportion ( &lt; 15%) shared with enterocytes (<a href="#f0001" class="usa-link">Figure 1B</a>). This data suggests that inulin has markedly different effects on gene expression in ECs compared to progenitor and proliferative cell populations. This difference becomes even more evident when looking at the KEGG terms related to the DEGs. Although inulin uniformly suppressed ‘ribosome’ genes across all cell subtypes, its effect on key metabolic pathways varies. In the EC compartment, an enrichment of tricarboxylic acid cycle (TCA) and oxidative phosphorylation pathways was observed in inulin-fed mice. In contrast, genes of oxidative phosphorylation were significantly downregulated in intestinal ISCs and TAs (<a href="#f0001" class="usa-link">Figure 1C,D</a>). Inulin diet reduced the expression of genes encoding the alpha subunit of ATP synthase F1, subunits of the cytochrome c oxidase complex (Cox), and components of complex I of the mitochondrial oxidative phosphorylation chain in ISCs and TAs (<a href="#f0001" class="usa-link">Figure 1D</a>). These changes were not observed in ECs (<a href="#f0001" class="usa-link">Figure 1D</a>). Together, these findings suggest that inulin intake alters intestinal epithelial cell metabolism, potentially reducing oxidative metabolism in ISCs and transit-amplifying cells, while increasing it in enterocytes. This is consistent with the elevated butyrate production observed in mice fed with an inulin-rich diet<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a>,<a href="#cit0044" class="usa-link" aria-describedby="cit0044">44</a>,<a href="#cit0045" class="usa-link" aria-describedby="cit0045">45</a></sup> and may reflect the zonation/distribution of these bacterial-derived metabolites along the crypt axis.<sup><a href="#cit0046" class="usa-link" aria-describedby="cit0046">46</a></sup>
</p>
<figure class="fig xbox font-sm" id="f0001"><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e69/12369635/f305a5a8ab5f/KGMI_A_2543123_F0001_OC.jpg" loading="lazy" height="993" width="800" alt="Figure 1."></p>
<div class="p text-right font-secondary"><a href="figure/f0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Inulin ingestion induces distinct transcriptional profiles in enterocytes and proliferating cells in the colon. (A) Venn diagram comparing the upregulated genes among intestinal stem cells (ISC), transit-amplifying cells (TA) and enterocytes (EC) obtained from the scRNA-seq analysis of intestinal epithelial cells (IECs) in inulin-treated animals. (B) Venn diagram comparing the downregulated genes among ISC, TA and EC obtained from the scRNA-seq analysis of IECs in inulin-treated animals. (C) KEGG pathway analysis by enrichR highlighting the top five terms for the up- (red) and downregulated (blue) genes among ISC, cycling TA and EC obtained from the scRNA-seq analysis of IECs in inulin-treated mice. (D) Average expression of selected oxidative phosphorylation (OXPHOS) genes in ISC, TA, and EC populations from control and inulin-fed mice. Genes were selected from significantly enriched pathways identified by gene ontology analysis. Color scale represents Z-score normalized expression values.</p></figcaption></figure></section><section id="s0003-s2002"><h3 class="pmc_sec_title">3.2. Inulin diet amplifies intestinal hypoxia and stabilizes HIF-1 in gut epithelial cells</h3>
<p>Inulin fermentation by the gut microbiota generates several bacterial metabolites, including SCFAs, which are essential for maintaining epithelial barrier integrity and metabolic homeostasis.<sup><a href="#cit0047" class="usa-link" aria-describedby="cit0047">47</a>,<a href="#cit0048" class="usa-link" aria-describedby="cit0048">48</a></sup> Among them, butyrate has been shown to induce intestinal hypoxia by increasing mitochondrial-dependent oxygen consumption by IECs, which in turn stabilize HIF-1, a master cellular oxygen sensor that plays a key role in the maintenance of barrier function in the colon.<sup><a href="#cit0020" class="usa-link" aria-describedby="cit0020">20</a></sup></p>
<p>Given this, we next investigated whether inulin ingestion influences intestinal epithelial cells by modulating intestinal hypoxia and the activity of HIF-1. To do so, mice were fed for three weeks with the control or inulin-enriched diets (<a href="#f0002" class="usa-link">Figure 2A</a>). Animals exhibited no differences in food intake or body weight (data not shown). Gut hypoxia was assessed using PMDZ staining. PMDZ is a 2-nitroimidazole that is specifically activated in hypoxic cells and tissues, forming stable complexes with thiol groups on proteins, peptides, and amino acids. The intake of the inulin diet increased colonic hypoxia, with greater PMDZ intensity and deeper distribution along the crypt axis compared to controls (<a href="#f0002" class="usa-link">Figure 2B,C</a>).
</p>
<figure class="fig xbox font-sm" id="f0002"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e69/12369635/c6db7c4589dc/KGMI_A_2543123_F0002_OC.jpg" loading="lazy" height="1076" width="800" alt="Figure 2."></p>
<div class="p text-right font-secondary"><a href="figure/f0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Inulin ingestion increases intestinal hypoxia and HIF-1a stabilization in intestinal epithelial cells. (A) Experimental design illustrating diets treatments and analyses performed in mice. (B) PMDZ staining was quantified by scoring sections of proximal colon from conventional mice fed with a control or a 10% inulin diet (<em>n</em> = 6). (C) Representative images of colonic sections stained with PMDZ from conventional mice treated with control and inulin diet for 3 weeks. Bars = 200 µm, 20x magnification. Zoom insets were taken from different areas or sections. Scale bars in the insets represent 100 µm (D) Representative images captured on the IVIS-SPECTRUM equipment 5 min after luciferin injection in ODD-mice fed with
control or inulin diets (<em>n</em> = 3–5). (E) Luminescence quantification over time after luciferin injection in mice fed with control and inulin diet (<em>n</em> = 3–5). (F) Normalized HIF-1 luciferase activity in the small intestine and colon from mice treated with control or inulin diet (<em>n</em> = 3–5). (G) Relative quantification of <em>Hif1a</em> and its target genes expression on colonic epithelial cells by RT-qPCR (<em>n</em> = 8). Data were analyzed using Student’s <em>t</em>-test. In all graphs, each point represents an individual mouse. *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001.</p></figcaption></figure><p>Next, we assessed HIF-1 stabilization using the ODD-luciferase transgenic mice, which express luciferase under the control of the oxygen-dependent degradation domain of HIF-1α.<sup><a href="#cit0049" class="usa-link" aria-describedby="cit0049">49</a></sup> Following luciferin administration, inulin-fed mice exhibited stronger chemiluminescence in the abdominal area compared to controls, indicating increased HIF-1α stabilization (<a href="#f0002" class="usa-link">Figure 2D–E</a>). In inulin-fed mice, we also observed a signal originating from a region located above the lungs, the nature of which will require further investigation (<a href="#f0002" class="usa-link">Figure 2D</a>). After euthanasia, luminescence was higher in colonic epithelial cells than in those from the small intestine, consistent with the gastrointestinal oxygen gradient<sup><a href="#cit0014" class="usa-link" aria-describedby="cit0014">14</a></sup> (<a href="#f0002" class="usa-link">Figure 2F</a>). Additionally, colonic IECs from inulin-fed mice showed greater HIF-1α stabilization than controls (<a href="#f0002" class="usa-link">Figure 2G</a>). This effect was accompanied by elevated expression of <em>Hif1a</em> and some HIF-1 target genes, including mucin-2 (<em>Muc2</em>) and trefoil factor-3 (<em>Tff3</em>) in colonic epithelial cells (<a href="#f0002" class="usa-link">Figure 2G</a>). Together, these data indicate that inulin ingestion intensifies colonic hypoxia, leading to increased HIF-1α expression and activity in epithelial cells. The partial upregulation of HIF-1 target genes suggests a limited response, potentially modulated by secondary regulatory mechanisms.</p></section><section id="s0003-s2003"><h3 class="pmc_sec_title">3.3. The effects of inulin on colonic hypoxia and HIF-1 stabilization depend on both gut microbiota and epithelial PPARγ</h3>
<p>Next, we investigated the role of the gut microbiota in mediating the effects of inulin-diet on the colonic epithelium, as fermentation of soluble fibers relies on the intestinal bacterial community. For that, mice were fed an inulin-rich diet and treated with a cocktail of broad-spectrum antibiotics (ABX) to deplete the gut microbiota<sup><a href="#cit0025" class="usa-link" aria-describedby="cit0025">25</a></sup> (<a href="#f0003" class="usa-link">Figure 3A</a>). The depletion of microbiota was confirmed by the reduced bacterial load in
feces of mice treated with the ABX cocktail (Figure S2A). ABX treatment reduced HIF-1α levels in IECs from inulin-treated mice (<a href="#f0003" class="usa-link">Figure 3B</a> and S2B), as well as the expression of <em>Hif1a</em> and some of its target genes (<a href="#f0003" class="usa-link">Figure 3C</a>). To further examine the microbiota’s impact on HIF-1, we conducted experiments with germ-free (GF), specific pathogen-free (SPF), and conventionalized (CV) mice. GF mice exhibited lower <em>Hif1a</em> expression and reduced levels of its target genes in colonic epithelial cells (<a href="#f0003" class="usa-link">Figure 3D</a>). Conventionalization via cecal content gavage and concurrent co-housing with SPF mice restored the expression of <em>Hif1a</em> and some target genes (<a href="#f0003" class="usa-link">Figure 3D</a>). These findings demonstrate that microbiota is necessary for regulating intestinal hypoxia and increasing <em>Hif1a</em> expression and its activity in response to inulin intake.
</p>
<figure class="fig xbox font-sm" id="f0003"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e69/12369635/5ed2b2bee433/KGMI_A_2543123_F0003_OC.jpg" loading="lazy" height="1157" width="800" alt="Figure 3."></p>
<div class="p text-right font-secondary"><a href="figure/f0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>HIF-1α stabilization in response to inulin diet is dependent on colon microbiota and epithelial Pparg. (A) Experimental design illustrating diet and ABX treatments in C57BL/6 mice. The ABX mix consisted of 1 g/L ampicillin, 1 g/L neomycin, 1 g/L
metronidazole, and 0.5 g/L vancomycin. (B) Median fluorescence intensity (MFI) for HIF-1α<sup>+</sup> labeling in CD45<sup>−</sup>/EpCAM<sup>+</sup> cells, as determined by flow cytometry, in mice fed control or inulin diet and treated with a mix of antibiotics (ABX + inulin) (<em>n</em> = 5). (C) Relative expression of <em>Hif1a</em> and HIF-1α target genes in mice fed with an inulin diet and treated with an antibiotic mix (<em>n</em> = 5). (D) Relative expression of <em>Hif1a</em> and HIF-1α target genes in specific pathogen-free (SPF), germ-free (GF) and conventionalized (CV) mice fed with a normal chow diet (<em>n</em> = 4). (E) <em>In vitro</em> growth curves of BOV and BT in media containing different amounts of inulin (<em>n</em> = 3). (F) Experimental design illustrating gnotobiotic mice colonized with <em>B. ovatus</em> (BOV) or <em>B. thetaiotaomicron</em> (BT), fed a control or inulin diet, followed by pimonidazole injection. (G) Hypoxia score in monocolonized mice fed control or inulin diet (<em>n</em> = 6). (H) Representative images of colonic sections stained with PMDZ from monocolonized mice treated with control and inulin diet for 3 weeks. Bars = 200 µm, 20x magnification. (I) Experimental design illustrating diets treatments and analyses performed in Pparg<sup>fl/fl</sup> and Pparg<sup>∆IEC</sup> mice. (J) Hypoxia score in Pparg<sup>fl/fl</sup> and Pparg<sup>ΔIEC</sup> fed with control or inulin diet (<em>n</em> = 4–7). (K) Relative mRNA expression of <em>Hif1a</em> in colonic epithelial cells from Pparg<sup>fl/fl</sup> and Pparg<sup>ΔIEC</sup>, as determined by RT-qPCR. (<em>n</em> = 4–7). Data were analyzed using one-way ANOVA followed by Tukey’s multiple comparisons test (B and D), Student’s <em>t</em>-test (C) or two-way ANOVA followed by Sidak’s multiple comparisons test (J and K). In all graphs, each point represents an individual mouse. *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001.</p></figcaption></figure><p>To better understand the role of inulin fermentation in our system, we conducted experiments using monocolonized mice. For this, we selected two individual bacterial species, <em>Bacteroides ovatus</em> (BOV) and <em>Bacteroides thetaiotaomicron</em> (BT). These species exhibited distinct growth capacities in media in which inulin was the sole carbon source (<a href="#f0003" class="usa-link">Figure 3E</a>). This is consistent with previous reports showing that BOV can metabolize inulin, whereas BT cannot<sup><a href="#cit0050" class="usa-link" aria-describedby="cit0050">50</a></sup>. Monocolonized mice were kept on either a control or inulin diet for 3 weeks and the hypoxia levels of the proximal colon were measured using the PMDZ technique (<a href="#f0003" class="usa-link">Figure 3F</a>). Both mice groups exhibited similar levels of bacterial colonization, indicating that BOV and BT have comparable capacities to colonize the gnotobiotic mouse colon (Fig S2C). Nonetheless, increased colonic hypoxia was only observed in mice monocolonized with BOV and fed with inulin (<a href="#f0003" class="usa-link">Figure 3</a><a href="#f0003" class="usa-link">G-H</a>), indicating that inulin metabolism is mandatory for inducing intestinal hypoxia and, therefore, HIF-1 activity.</p>
<p>Previous studies by our group and others have shown that inulin ingestion increases butyrate production,<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a>,<a href="#cit0051" class="usa-link" aria-describedby="cit0051">51</a>,<a href="#cit0052" class="usa-link" aria-describedby="cit0052">52</a></sup> a key metabolite that regulates various epithelial barrier functions.<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a>,<a href="#cit0020" class="usa-link" aria-describedby="cit0020">20</a>,<a href="#cit0053" class="usa-link" aria-describedby="cit0053">53</a></sup> Butyrate promotes β-oxidation in epithelial cells via activation of peroxisome proliferator-activated receptor gamma (PPARγ), enhancing oxygen consumption and maintaining intestinal hypoxia.<sup><a href="#cit0016" class="usa-link" aria-describedby="cit0016">16</a></sup> To investigate whether PPARγ plays a significant role in the inulin-induced epithelial hypoxia, we studied littermate mice with epithelial-specific deletion of <em>Pparg</em> (Pparg<sup>fl/fl</sup> x VilCre<sup>+</sup>, Pparg<sup>ΔIEC</sup>) and wild-type controls (Pparg<sup>fl/fl</sup> x VilCre<sup>−</sup>, Pparg<sup>fl/fl</sup>) (<a href="#f0003" class="usa-link">Figure 3I</a>). After three weeks on either a control or inulin diet, we assessed intestinal hypoxia. As expected, Pparg<sup>fl/fl</sup> mice on the inulin diet showed a significant increase in intestinal hypoxia evaluated by PMDZ staining and <em>Hif1a</em> expression in IECs (<a href="#f0003" class="usa-link">Figure 3J–K</a>). However, these effects were absent in Pparg<sup>ΔIEC</sup> mice, indicating that the PPARγ is directly involved in the development of hypoxia and subsequent increase in HIF-1α following inulin intake. Together, these findings suggest that inulin fermentation by the microbiota produces metabolites that activate PPARγ, driving mitochondrial β-oxidation and oxygen consumption, leading to increased intestinal hypoxia.</p></section><section id="s0003-s2004"><h3 class="pmc_sec_title">3.4. HIF-1 regulates colon epithelial cell proliferation in response to the inulin diet</h3>
<p>HIF-1 is highly expressed in intestinal epithelial cells, where it regulates genes related to antimicrobial production, barrier function, mucin secretion, and energy metabolism.<sup><a href="#cit0014" class="usa-link" aria-describedby="cit0014">14</a></sup> Given the stabilization of HIF-1α in response to the inulin diet, we next sought to evaluate the biological significance of this effect on the intestinal epithelium. To do so, we utilized mice with a targeted deletion of <em>Hif1a</em> in intestinal epithelial cells (HIF-1α<sup>fl/fl</sup> x VilCre<sup>+</sup>, HIF-1α<sup>ΔIEC</sup>) alongside floxed littermate controls without Cre recombinase (HIF-1α<sup>fl/fl</sup> x VilCre<sup>−</sup>, HIF-1α<sup>fl/fl</sup>) (<a href="#f0004" class="usa-link">Figure 4A</a>). HIF-1α deletion in HIF-1α<sup>ΔIEC</sup> mice was confirmed by quantitative PCR analysis of colon tissues from animals on the inulin diet (Figure S3A) and by assessing HIF-1α levels via flow cytometry (Figure S3B and S3C). No differences were observed in body weight, small intestine or cecum lengths (Figure S3D-F) in HIF-1α<sup>ΔIEC</sup> mice when compared to littermate controls. Nonetheless, when maintained on inulin diet, HIF-1α<sup>ΔIEC</sup> mice displayed increased colon length and deeper intestinal crypts compared to controls (<a href="#f0004" class="usa-link">Figure 4B–D</a>). Moreover, there was a marked increase in proliferative cells, as shown by EdU (S phase) and Ki67 (G1, S, G2, and M phases) staining (<a href="#f0004" class="usa-link">Figure 4E,F</a>, and S3G-I). Additionally, we
demonstrated that the increment on intestinal proliferative cells and crypt size in HIF-1α<sup>ΔIEC</sup> mice, compared to HIF-1α<sup>fl/fl</sup> mice, was dependent of the inulin-diet, as HIF-1α<sup>ΔIEC</sup> mice on a control diet did not present this response (Figure S3J). We also analyzed the gut microbiota composition of these mice and found that, although there were no significant changes in overall community structure, as assessed by alpha and beta diversity (Figure S4A – C), LEfSe revealed specific taxonomic shifts, including an increased abundance of the <em>Muribaculaceae</em> family in HIF-1α<sup>ΔIEC</sup> mice (Figure S4D). Together, these data indicate the interplay between diet and the gene regulatory machinery involving HIF-1 in regulating the colonic epithelium.
</p>
<figure class="fig xbox font-sm" id="f0004"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e69/12369635/f69ea30bb6d1/KGMI_A_2543123_F0004_OC.jpg" loading="lazy" height="1025" width="800" alt="Figure 4."></p>
<div class="p text-right font-secondary"><a href="figure/f0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Epithelial deletion of HIF-1α increases intestinal epithelial proliferation and modulates metabolism pathways in response to inulin diet. (A) Experimental design illustrating diets treatments and analyses performed in HIF-1α<sup>fl/fl</sup> and HIF-1α<sup>∆IEC</sup> mice. (B) Colon length normalized by body weight in HIF-1α<sup>fl/fl</sup> and HIF-1α<sup>∆IEC</sup> mice fed with inulin diet (<em>n</em> = 8–9). (C) Crypts length normalized by body weight in HIF-1α<sup>fl/fl</sup> and HIF-1α<sup>∆IEC</sup> mice fed with inulin diet (<em>n</em> = 8–9). (D) Representative images of colonic sections stained with H&amp;E from mice fed with inulin diet for 3 weeks. Bars = 100 µm, 20x magnification (<em>n</em> = 8–9). (E) Quantification of EdU-positive cells per crypt (<em>n</em> = 4). (F) Visualization of EdU-positive cells in colonic crypts by fluorescence microscopy following EdU Click-iT reaction . Bars = 100 µm, 20 ×magnification. (G) Biological process enrichment map from differentially regulated proteins (HIF-1α<sup>ΔIEC</sup> vs HIF-1α<sup>fl/fl</sup>) in mice fed with inulin diet, highlighting the modules associated with
cell metabolism (<em>n</em> = 5). (H) Process-protein network showing proteins differentially regulated (blue-red gradient for down- and upregulation, respectively) across metabolic modules. The red edges highlight key proteins and processes linking metabolic effects with epithelial development. Data were analyzed using Student’s <em>t</em>-test. In all graphs, each point represents an individual mouse. *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01.</p></figcaption></figure><p>To investigate the mechanism behind the increased proliferation in HIF-1α<sup>ΔIEC</sup> mice, we performed an unbiased proteomic analysis of IECs from HIF-1α<sup>ΔIEC</sup> and their littermates on the inulin diet (<a href="#f0004" class="usa-link">Figure 4G–H</a>, and S5). This analysis revealed differential regulation of approximately one-fourth of the identified proteins (435 out of 1601), with 361 downregulated and 74 upregulated in HIF-1α<sup>ΔIEC</sup> compared to HIF-1α<sup>fl/fl</sup> mice (Figure S5). Biological process enrichment map from differentially regulated proteins (HIF-1α<sup>ΔIEC</sup> vs HIF-1α<sup>fl/fl</sup>) in mice fed with inulin highlighted important modulation of modules associated with cell metabolism including mitochondrial organization, fatty acids catabolic processes and glycolytic processes in addition to modulation of epithelium development (<a href="#f0004" class="usa-link">Figure 4G</a>). Furthermore, process-protein network showed proteins differentially regulated across metabolic modules. Some proteins involved in glycolytic process such as aldolases (Aldoa and Aldoc) and hexokinases (Hk2–3 and Hkdc1), oxidative phosphorylation such as NADH dehydrogenase [ubiquinone] iron-sulfur family (Ndufs subunits 4, 7, 9, 10 and 11) and cytochrome c oxidase family (COX) and fatty acid catabolic process such as carnitine palmitoyltransferase 2 (Cpt2) and short-chain specific acyl-CoA dehydrogenase (Acads) were seen downregulated in HIF-1α<sup>∆IEC</sup> epithelium (<a href="#f0004" class="usa-link">Figure 4H</a>). On the other hand, most of the upregulated proteins in HIF-1α<sup>ΔIEC</sup> epithelium were associated with mitochondrial carrier protein family (Timm23, Timm50, Mtch1 and Gfm1), glycolytic process and cellular response to stress (Rpia and Dera, respectively), besides cell cycle control proteins (Cdc42). Interestingly, most of these differentially regulated proteins link metabolic effects with epithelial development, consistent with an important role of HIF-1 in regulating metabolic pathways, which in turn modulate the proliferation and differentiation of the intestinal epithelium.</p></section><section id="s0003-s2005"><h3 class="pmc_sec_title">3.5. HIF-1 regulates stem cell activity in response to inulin diet</h3>
<p>Intestinal epithelial barrier homeostasis relies on the balance between self-renewal and differentiation of ISCs, processes regulated by both intrinsic and extrinsic signals including diet and microbiota.<sup><a href="#cit0054" class="usa-link" aria-describedby="cit0054">54</a></sup> Our previous study showed that inulin ingestion increases the proliferation and activity of colonic stem cells compared to a control diet.<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a></sup> Based on these findings, we tested if HIF-1 is involved in these effects of inulin specifically on ISCs. HIF-1α<sup>ΔIEC</sup> mice and their floxed controls were fed an inulin diet, followed by colonic crypts isolation for clonogenicity assay, according to the protocol by Beyaz et al. (2016).<sup><a href="#cit0055" class="usa-link" aria-describedby="cit0055">55</a></sup> We found that the conversion of crypts into organoids was significantly higher in knockout animals compared to their littermates (<a href="#f0005" class="usa-link">Figure 5A and C</a>), which also exhibited larger volume and growth (<a href="#f0005" class="usa-link">Figure 5B</a>), thus indicating stronger ISCs activity in the absence of HIF.
</p>
<figure class="fig xbox font-sm" id="f0005"><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e69/12369635/d202e4e5f715/KGMI_A_2543123_F0005_OC.jpg" loading="lazy" height="972" width="800" alt="Figure 5."></p>
<div class="p text-right font-secondary"><a href="figure/f0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>HIF-1α plays a critical role in intestinal stem cell (ISC) function by modulating oxidative metabolism. (A) Quantification of the clonogenicity capacity of colon crypts in mice with and without epithelial deletion of HIF-1α (<em>n</em> = 9). (B) Growth kinetics of organoids (<em>n</em> = 9). (C) Representative images of organoids from mice with HIF-1α epithelial deletion. (D) Experimental design for sorting cell experiments with HIF-1α<sup>ΔISC</sup> mice (<em>n</em> = 5–6). (E) Clonogenicity capacity of sorted-ISCs from HIF-1α<sup>ΔISC</sup> mice and their controls (<em>n</em> = 5–6). (F) Organoid volume average from HIF-1α<sup>ΔISC</sup> mice and their controls (<em>n</em> = 5–6). (G) Representative images of organoids from HIF-1α<sup>ΔISC</sup> mice and their controls (<em>n</em> = 5–6). (H) Oxygen consumption rates (OCRs) of organoids, as measured by CellMito stress test and a Seahorse XF extracellular flux analyzer (<em>n</em> = 3). (I) Basal respiration of colonic organoids (<em>n</em> = 3). (J) ATP-linked respiration of colonic organoids (<em>n</em> = 3). (K) Maximal respiration of colonic organoids (<em>n</em> = 3). Data were analyzed using Student’s <em>t</em>-test. In all graphs, each point represents an individual mouse. *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01.</p></figcaption></figure><p>To further validate the role of HIF-1α, we performed a similar analysis on von Hippel-Lindau tumor suppressor (VHL)-deficient mice (VHL<sup>ΔIEC</sup>) (Figure S6A). The VHL protein regulates the stability of HIF-(1/2)α, and its deletion leads to constitutive HIF-1α stabilization in IECs by inhibiting its proteasomal degradation.<sup><a href="#cit0056" class="usa-link" aria-describedby="cit0056">56</a></sup> VHL<sup>ΔIEC</sup> mice presented no significant differences in body weight, colon size, or colon crypt length (Figure S6B-D). However, consistent with the results from HIF-1α-deficient mice, VHL<sup>ΔIEC</sup> mice without inulin treatment exhibited reduced epithelial cell proliferation, as observed by a significant lower number of EdU-positive cells (Figure S6E). Similarly, VHL<sup>ΔIEC</sup> crypts showed reduced clonogenicity capacity but with no changes in organoid volume (Figure S6F-G), indicating that constitutive expression of HIF-1 impairs ISCs activity.</p>
<p>We next performed experiments using an ISC-specific HIF-1α-deficient mouse model (Lgr5-Cre; Ai14; HIF-1α<sup>fl/fl</sup>, termed HIF-1α<sup>ΔISC</sup>) to confirm whether the effect of HIF-1α on organoid formation was intrinsic to ISCs. Both HIF-1α<sup>ΔISC</sup> mice and their controls (Lgr5-Cre; Ai14; HIF-1α<sup>fl/-</sup>, termed HIF-1α<sup>fl/-</sup>) were maintained on an inulin diet during the entire protocol and received tamoxifen (i.p.) 24 hours before the experiment to induce tomato fluorescence (<a href="#f0005" class="usa-link">Figure 5D</a>). We then FACS-sorted viable cells with high tdTomato signal and plated them for organoid formation (<a href="#f0005" class="usa-link">Figure 5D</a>). We found that organoids formation was more efficient in HIF-1α-deficient ISCs compared to controls expressing normal levels of HIF-1α (<a href="#f0005" class="usa-link">Figure 5E–G</a>) confirming that HIF-1α regulates ISCs functions in response to inulin diet.</p></section><section id="s0003-s2006"><h3 class="pmc_sec_title">3.6. HIF-1α modulates intestinal epithelial proliferation through metabolic reprogramming</h3>
<p>Since our proteomics data indicated that HIF-1α deletion leads to changes in the metabolic pattern of the intestinal epithelium (<a href="#f0004" class="usa-link">Figure 4G–H</a>), we sought to determine the impact of its absence on mitochondrial function and, consequently, on the oxidative metabolism in intestinal organoids. The mitochondrial respiratory capacity of organoids from HIF-1α<sup>ΔIEC</sup> mice and their controls was evaluated using a Cell Mito Stress Test on a Seahorse XF extracellular flux analyzer. We found oxygen consumption rates (OCRs), ATP-linked, and maximal mitochondrial respiration were significantly elevated in HIF-1α<sup>ΔIEC</sup> organoids (<a href="#f0005" class="usa-link">Figure 5H–K</a>). These changes were not accompanied by alterations in mitochondrial mass, which was examined by MitoTracker green (Figure S7A-D), indicating that the effect of HIF-1 deletion in the IECs is not dependent on the mitochondrial mass, but on mitochondrial activity.</p>
<p>Fatty acid oxidation (FAO) is an important source of intermediates for the TCA cycle and provides electron carriers (NADH and FADH<sub>2</sub>) that fuel the mitochondrial electron transport chain, leading to ATP production. This process has been linked with ISC stemness.<sup><a href="#cit0057" class="usa-link" aria-describedby="cit0057">57</a></sup> To investigate if FAO is connected to the increased proliferation in HIF-1α<sup>ΔIEC</sup> mice, we treated organoids from HIF-1α<sup>ΔIEC</sup> and HIF-1α<sup>fl/fl</sup> mice with low doses of etomoxir, an FAO inhibitor that targets carnitine palmitoyl transferase I (CPT-1).<sup><a href="#cit0058" class="usa-link" aria-describedby="cit0058">58</a></sup> Once again, HIF-1α<sup>ΔIEC</sup> derived organoids presented a higher number of proliferating cells, as assessed by EdU staining (<a href="#f0006" class="usa-link">Figure 6A</a>), and HIF-1α<sup>ΔIEC</sup> colonic crypts showed higher clonogenicity, forming larger organoids (<a href="#f0006" class="usa-link">Figure 6B,C</a>) when compared to the control condition. These effects were abrogated in HIF-1α<sup>ΔIEC</sup>-derived organoids treated with etomoxir, while the drug had no effect in HIF-1α<sup>fl/fl</sup> organoids. Furthermore, <em>in vivo</em> treatment with etomoxir also reduced colonic epithelial proliferation in HIF-1α<sup>ΔIEC</sup> mice fed with inulin, as observed by reduced number of EdU-positive cells (<a href="#f0006" class="usa-link">Figure 6D–F</a>), and a diminished ability of the crypts to generate organoids (<a href="#f0006" class="usa-link">Figure 6G</a>). These data suggest that FAO contributes to the increased ISC proliferation and activity in the absence of HIF-1. Additionally, we examined how HIF-1α<sup>ΔIEC</sup>-derived organoids responded to mitochondrial inhibitors and the glycolysis inhibitor 2-deoxyglucose (2-DG) (<a href="#f0006" class="usa-link">Figure 6H</a>). Treatment with 2-DG, antimycin (complex III inhibitor), oligomycin (complex V inhibitor), or rotenone (complex I inhibitor), at the concentrations used in our study, had no effect on the growth of HIF-1α<sup>fl/fl</sup> organoids. However, these treatments impaired the phenotype observed in HIF-1α<sup>ΔIEC</sup>-derived organoids, which no longer showed any differences when compared to the control condition (<a href="#f0006" class="usa-link">Figure 6I,J</a> and S7E).
These findings highlight the crucial role of HIF-1 in regulating ISC function through metabolic reprogramming in the colon.
</p>
<figure class="fig xbox font-sm" id="f0006"><h4 class="obj_head">Figure 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8e69/12369635/47572fbc711a/KGMI_A_2543123_F0006_OC.jpg" loading="lazy" height="1100" width="800" alt="Figure 6."></p>
<div class="p text-right font-secondary"><a href="figure/f0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>HIF-1 controls colon cell proliferation by regulating oxidative phosphorylation and mitochondrial function in ISCs. (A) Percentage of live EdU-positive cells in colon organoids after treatment with the FAO inhibitor, etomoxir (<em>n</em> = 4). (B) Clonogenicity capacity of colon crypts treated with etomoxir (<em>n</em> = 4). (C) Measurement of organoids volume after 4 days of etomoxir treatment (<em>n</em> = 4). (D) Experimental design of etomoxir <em>in vivo</em> treatment. (E) EdU-positive cells in colon sections from mice treated with etomoxir (20 mg/kg i.p.) (<em>n</em> = 4). (F) Representative images of EdU-positive cells in the colon of mice
treated with PBS or etomoxir (<em>n</em> = 4). Bars = 100 µm, 20 ×magnification. (G) Clonogenicity capacity of colon crypts from mice treated with etomoxir (<em>n</em> = 4). (H) Experimental design of organoids treatment with different inhibitors of glycolysis and mitochondrial complexes. (I) Clonogenicity capacity of colon crypts treated with glycolysis inhibitor, 2-deoxyglucose (2-DG), and inhibitors of mitochondrial complexes (<em>n</em> = 3–5). (J) Organoid volume average from organoids treated with oligomycin, antimycin, rotenone or 2-DG (<em>n</em> = 3–5). Data were analyzed using two-way ANOVA followed by Sidak’s multiple comparisons test (A, B, C, I and J) or Student’s <em>t</em>-test (E and G). In all graphs, each point represents an individual mouse. *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001.</p></figcaption></figure></section></section><section id="s0004"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>We describe a previously unrecognized function of HIF-1α in regulating host-microbiota interactions, expanding its known functions in the intestinal epithelium, which include maintaining barrier integrity, inducing antimicrobial peptides, and modulating immune responses.<sup><a href="#cit0020" class="usa-link" aria-describedby="cit0020">20</a></sup> We have previously shown that the ingestion of an inulin-enriched diet promotes epithelial proliferation via modulation of ISCs activity in a microbiota-dependent manner.<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a></sup> Here, we now demonstrate that this proliferative phenotype is significantly amplified in mice with epithelial-specific deletion of HIF-1α. These mice exhibit elevated ISCs activity and markedly increased epithelial proliferation compared to their wild-type counterparts. Therefore, our findings suggest that, under homeostatic conditions, HIF-1α acts as a regulatory brake, fine-tuning the impact of the dietary fiber inulin on intestinal epithelial dynamics.</p>
<p>We link the role of HIF-1α in the colon epithelium to its impact on regulation of stem and proliferative cell activity. HIF-1 plays a pivotal role in regulating ISCs and modulating their responses under both physiological and pathological conditions by directly interacting with key signaling pathways essential for ISC function, including Wnt/β-catenin, Notch, and JAK/STAT pathways.<sup><a href="#cit0059" class="usa-link" aria-describedby="cit0059">59–61</a></sup> Additionally, HIF-1 can indirectly influence ISC activity by acting on the stem cell niche within the intestinal crypts, promoting the release of growth factors and cytokines that stimulate proliferation and inhibit apoptosis.<sup><a href="#cit0062" class="usa-link" aria-describedby="cit0062">62</a>,<a href="#cit0063" class="usa-link" aria-describedby="cit0063">63</a></sup> Some of the roles of HIF-1α in stem cells may be shared in other somatic stem cell compartments. A study of mouse muscle regeneration indicated that HIF1-signaling regulates stem cell proliferation and showed that disrupting the HIF signaling led to enhanced proliferation with potential depletion of the stem cell pool and reduced regenerative potential of the muscle and this has been also linked to the role of HIF in regulating Notch.<sup><a href="#cit0064" class="usa-link" aria-describedby="cit0064">64</a>,<a href="#cit0065" class="usa-link" aria-describedby="cit0065">65</a></sup> Future research needs to illuminate shared and tissue-specific functions of HIF in regulating stem cell biology.</p>
<p>The regulatory role of HIF-1α in homeostasis contrasts with its established function in inflammation and tumorigenesis, where it typically enhances cell proliferation and survival.<sup><a href="#cit0066" class="usa-link" aria-describedby="cit0066">66–68</a></sup> In such pathological settings, increased HIF-1α activity is often attributed to reduced oxygen availability in the tumor microenvironment<sup><a href="#cit0069" class="usa-link" aria-describedby="cit0069">69</a></sup> and/or a metabolic shift known as the Warburg effect, in which cancer cells preferentially use glycolysis over oxidative phosphorylation even in the presence of oxygen.<sup><a href="#cit0070" class="usa-link" aria-describedby="cit0070">70</a></sup> However, some studies have shown that the absence of HIF-1α can paradoxically stimulate growth and proliferation in colon cells.<sup><a href="#cit0071" class="usa-link" aria-describedby="cit0071">71</a></sup> The conditional deletion of HIF-1α in IECs can lead to accelerated tumor growth in certain models of colorectal cancer, suggesting that HIF-1α can also play a role in inhibiting tumor progression.<sup><a href="#cit0072" class="usa-link" aria-describedby="cit0072">72</a></sup> Therefore, together with the studies described above, our findings highlight that the effects of HIF-1 on the regulation of cell proliferation are context-dependent and influenced by factors such as the types of cells involved, the presence of inflammation or dysbiosis, and the metabolic state of the tissue.</p>
<p>We show that an inulin-rich diet promotes microbial fermentation, leading to increased intestinal hypoxia and activation of HIF-1 signaling in epithelial cells. This, in turn, regulates epithelial metabolism and influences ISC proliferation and maintenance. Prior studies have shown that the SCFA butyrate has a pivotal role in modulating intestinal oxygen levels.<sup><a href="#cit0020" class="usa-link" aria-describedby="cit0020">20</a></sup> As a primary energy source for IECs, butyrate enhances mitochondrial oxidative phosphorylation, increases oxygen consumption, and contributes to a hypoxic microenvironment.<sup><a href="#cit0016" class="usa-link" aria-describedby="cit0016">16</a></sup> This hypoxia is crucial for preventing dysbiosis and sustaining epithelial integrity.<sup><a href="#cit0014" class="usa-link" aria-describedby="cit0014">14</a>,<a href="#cit0016" class="usa-link" aria-describedby="cit0016">16</a>,<a href="#cit0021" class="usa-link" aria-describedby="cit0021">21</a>,<a href="#cit0073" class="usa-link" aria-describedby="cit0073">73</a>,<a href="#cit0074" class="usa-link" aria-describedby="cit0074">74</a></sup> In our previous work, we demonstrated that inulin consumption increases the levels of acetate, propionate, and butyrate compared to a cellulose-based control diet.<sup><a href="#cit0011" class="usa-link" aria-describedby="cit0011">11</a></sup> In this follow-up study, we indirectly link this SCFA enrichment
to elevated intestinal hypoxia and enhanced HIF-1 signaling in IECs. While future metagenomic and metabolomic analysis are needed to identify the precise microbial metabolites involved, our data provide evidence that inulin, via microbial fermentation, differentially affects the metabolic activity of IEC subsets. Specifically, it suppresses oxidative phosphorylation in the proliferative compartments of the crypts, including on ISCs and TAs, while promoting it in differentiated enterocytes. Together, these findings support the concept of spatial gradients in both oxygen and microbial metabolites along the crypt axis, which drive distinct metabolic responses in the functionally and spatially diverse populations of IECs that make up the intestinal epithelium.<sup><a href="#cit0046" class="usa-link" aria-describedby="cit0046">46</a>,<a href="#cit0075" class="usa-link" aria-describedby="cit0075">75</a>,<a href="#cit0076" class="usa-link" aria-describedby="cit0076">76</a></sup> Future studies should dissect the metabolic landscape across distinct epithelial cell populations, including mature, progenitor/proliferative and stem cells, which, as suggested by our transcriptomic data, exhibit differential metabolic responses to inulin. Single-cell metabolomic approaches will be particularly valuable in resolving the cell-type – specific changes.</p>
<p>Our proteomic analyses of epithelial cells with and without HIF-1 on inulin diet revealed significant alterations in structural proteins associated with the mitochondrial membrane, findings that were corroborated through functional assays using selective inhibitors of mitochondrial respiratory complexes. These results suggest that HIF-1 signaling is a key mediator linking energy metabolism to epithelial cell turnover. Furthermore, these findings highlight the pivotal role of mitochondria as hub that integrates metabolic signals and relays these into regulation of cellular proliferation. The important roles of mitochondria in generating energy, calcium storage and regulating cell death are well established. Less in known about the roles of mitochondria in regulating stem cell renewal and differentiation.<sup><a href="#cit0077" class="usa-link" aria-describedby="cit0077">77</a>,<a href="#cit0078" class="usa-link" aria-describedby="cit0078">78</a></sup> In <em>Drosophila</em>, mitochondrial electron chain transport activity promotes ISC proliferation and epithelial homeostasis through the transcriptional regulator FOXO<sup><a href="#cit0079" class="usa-link" aria-describedby="cit0079">79</a></sup>. Other studies, also in <em>Drosophila</em>, implicated mitochondrial fusion dynamics in the regulation of ISC biology<sup><a href="#cit0080" class="usa-link" aria-describedby="cit0080">80</a></sup>. Recent studies have shown that mitochondria play a key role in regulating intestinal stem cell (ISC) maintenance and differentiation in the mouse intestine.<sup><a href="#cit0030" class="usa-link" aria-describedby="cit0030">30</a>,<a href="#cit0081" class="usa-link" aria-describedby="cit0081">81</a></sup> How HIF-1 regulates mitochondrial activity and affects ISCs functions will require further analysis.</p>
<p>What might be the biological role, if any, of increased proliferation in the intestinal epithelium and deepening of the crypt? The crypt has been suggested to be an invagination that protects the intestinal stem cells from toxic activities derived from microbial metabolites, including butyrate.<sup><a href="#cit0046" class="usa-link" aria-describedby="cit0046">46</a></sup> Therefore, it is tempting to speculate that the role of HIF-1 in regulating proliferative activity in the intestinal epithelium and crypt depth is linked to this protective role. One possible scenario might be that reactive oxygen species (ROS) play an important role in our observations. ROS are generated by oxidative phosphorylation of SCFAs and might trigger proliferation, e.g., via signaling pathways, possibly promoting protection of the stem cells from their detrimental activity, similar to what has been described in the <em>Drosophila</em> system.<sup><a href="#cit0082" class="usa-link" aria-describedby="cit0082">82</a>,<a href="#cit0083" class="usa-link" aria-describedby="cit0083">83</a></sup> Indeed, NOX1-dependent redox signaling has been shown to promote colonic cell proliferation.<sup><a href="#cit0084" class="usa-link" aria-describedby="cit0084">84</a></sup></p>
<p>Given this multifaceted regulation, it remains challenging to delineate the precise mechanisms through which HIF-1 modulates ISC behavior and, consequently, epithelial architecture in response to dynamic luminal cues such as dietary composition, energy availability, and microbiota-derived signals. Several potential limitations of our study should be considered. First, we have not identified the specific inulin diet-dependent microbial metabolites that influence proliferation in the colonic epithelium. While butyrate has been linked to intestinal hypoxia, other metabolites, such as propionate, may also contribute. Another aspect that needs further investigation is the spatial gradient of oxygen and metabolites along the crypt axis, which we speculate may induce differential effects across epithelial compartments. An additional limitation is that, although we demonstrate that metabolic reprogramming is involved in the phenotype observed following HIF-1α deletion, we did not elucidate the underlying mechanisms. In this context, factors such as mitochondrial activity and the potential role of reactive oxygen species (ROS) generation should be further explored. Nonetheless, despite the complexity of this regulatory landscape, our findings underscore the essential role of HIF-1 in maintaining ISC function and, by extension, intestinal epithelial homeostasis in response to inulin intake, serving as a molecular break preventing cell over proliferation. Future work should evaluate these findings in the context of colorectal cancer, e.g., using appropriate mouse models.</p></section><section id="s0005"><h2 class="pmc_sec_title">5. Conclusion</h2>
<p>In conclusion, our data highlights the impact of inulin on intestinal epithelial hypoxia and intestinal stem cell metabolism. It also demonstrates a significant novel role of HIF-1 in limiting intestinal epithelial proliferation in response to microbiota-derived metabolites produced by high-fiber diets, thereby contributing to the maintenance of colonic epithelial homeostasis.</p></section><section id="sec30"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="SM5400"><div class="caption p"><span>Supplemental Material</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12369635/bin/KGMI_A_2543123_SM5400.zip" data-ga-action="click_feat_suppl" class="usa-link">KGMI_A_2543123_SM5400.zip</a><sup> (22.9MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Dr. Maria Teresa Pedrosa Silva Clerici and Ulliana Marques Sampaio for the formulation of diets, Dr. Valquiria Aparecida Matheus for histology support, Dr. Flaviano dos Santos Martins for providing germ-free mice, and Mirella Cristiny Pereira de Andrade for experimental support. We also thank the staff of the LaCTAD from State University of Campinas (UNICAMP), for the cell sorting experiments and from the Translational Pathology Shared Resource for histological support in PMDZ experiments.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This study was supported by National Institutes of Health (NIH) [1R01DK126969-01], Fundação de Amparo à Pesquisa do Estado de São Paulo (FAPESP) [#2017/16280-3, #2017/25588-1, #2018/15313-8, #2019/00098-7, #2020/14071-0, #2022/04576-3, #2022/04576-3, #2023/14946-5], Fundação de Desenvolvimento da Unicamp (FUNCAMP), National Council for Scientific and Technological Development (CNPq) [301658/2022-6], and Coordenação de Aperfeiçoamento de Pessoal de Nível Superior - Brasil (CAPES) Finance Code 001. P.R.C., R.O.C., M.F.F., V.D.N., A.B.S.P.G., L.P.P., S.O., and V.C.C., are recipients of grants from FAPESP [#2020/02919-5, #2016/23142-3, #2019/02640-3, #2021/10478-1, #2021/00393-9, #2020/02312-3, #2020/13689-0, #2021/09155-3, #2019/05155-9, respectively]. J.L.F. is a recipient of the Pew Latin American fellowship [#00035299]. N.G.S. was supported by Howard Hughes Medical Institute Gilliam Fellowship (GT15104). C.D.S. was supported by NIH Ruth L. Kirschstein Predoctoral Individual Fellowship (1F31AI161882-01A1). M.X.B is an Howard Hughes Medical Institute Freeman Hrabowski Scholar. Work in M.X.B.‘s laboratory was supported by the National Institutes of Health [R01DK131104-01 and 1R01AI168302-01A1], The Pew Charitable Trusts [2022-A-19568] and Burroughs Welcome Fund [022792]. P.V.W.’s work is supported by the University of Essex, by FAPESP São Paulo Excellence Chair special grant [#2019/16113-5]. The Translational Pathology Shared Resource is supported by NCI/NIH Cancer Center Grant [P30CA068485].</p></section><section id="s0006"><h2 class="pmc_sec_title">Disclosure statement</h2>
<p>No potential conflict of interest was reported by the author(s).</p></section><section id="s0007"><h2 class="pmc_sec_title">Availability of data and materials</h2>
<ol id="list_0001" class="list" style="list-style-type:decimal">
<li><p>- Single-cell RNA sequencing data is deposited at NCBI BioProject: PRJNA856646</p></li>
<li><p>- 16S sequencing data is deposited at NCBI BioProject: PRJNA1188097.</p></li>
<li><p>- The mass spectrometry proteomics data is deposited to the ProteomeXchange Consortium (<a href="http://proteomecentral.proteomexchange.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://proteomecentral.proteomexchange.org</a>.) via the PRIDE partner repository<sup><a href="#cit0085" class="usa-link" aria-describedby="cit0085">85</a></sup> with the dataset identifier PXD059757. Project DOI: 10.6019/PXD059757</p></li>
</ol></section><section id="s0008"><h2 class="pmc_sec_title">Consent for publication</h2>
<p>All authors have read and agreed with the manuscript.</p></section><section id="s0009"><h2 class="pmc_sec_title">Supplementary material</h2>
<p>Supplemental data for this article can be accessed online at <a href="https://doi.org/10.1080/19490976.2025.2543123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1080/19490976.2025.2543123</a>.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="cit0001">
<span class="label">1.</span><cite>Gehart H, Clevers H.. 
Tales from the crypt: new insights into intestinal stem cells. Nat Rev Gastroenterol Hepatol [Internet]. 2019.  [cited 2025 May
12]. 16(1):19–26. doi: 10.1038/s41575-018-0081-y.
</cite> [<a href="https://doi.org/10.1038/s41575-018-0081-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30429586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Gastroenterol%20Hepatol%20%5BInternet%5D&amp;title=Tales%20from%20the%20crypt:%20new%20insights%20into%20intestinal%20stem%20cells&amp;volume=16&amp;issue=1&amp;publication_year=2019&amp;pages=19-26&amp;pmid=30429586&amp;doi=10.1038/s41575-018-0081-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0002">
<span class="label">2.</span><cite>Kayama H, Okumura R, Takeda K. Interaction between the microbiota, epithelia, and immune cells in the intestine. Annu Rev Immunol [Internet]. 2020.  [cited 2025 May
12]. 38(1):23–48. doi: 10.1146/annurev-immunol-070119-115104.
</cite> [<a href="https://doi.org/10.1146/annurev-immunol-070119-115104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32340570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Immunol%20%5BInternet%5D&amp;title=Interaction%20between%20the%20microbiota,%20epithelia,%20and%20immune%20cells%20in%20the%20intestine&amp;volume=38&amp;issue=1&amp;publication_year=2020&amp;pages=23-48&amp;pmid=32340570&amp;doi=10.1146/annurev-immunol-070119-115104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0003">
<span class="label">3.</span><cite>Konjar Š, Pavšič M, Veldhoen M. Regulation of oxygen homeostasis at the intestinal epithelial barrier site. IJMS [Internet]. 2021.  [cited 2025 May
12]. 22(17):9170.
</cite> [<a href="https://doi.org/10.3390/ijms22179170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8431628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34502078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IJMS%20%5BInternet%5D&amp;title=Regulation%20of%20oxygen%20homeostasis%20at%20the%20intestinal%20epithelial%20barrier%20site&amp;volume=22&amp;issue=17&amp;publication_year=2021&amp;pages=9170&amp;pmid=34502078&amp;doi=10.3390/ijms22179170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0004">
<span class="label">4.</span><cite>Fan Y, Pedersen O. Gut microbiota in human metabolic health and disease. Nat Rev Microbiol [Internet]. 2021.  [cited 2025 May
12]. 19(1):55–71. doi: 10.1038/s41579-020-0433-9.
</cite> [<a href="https://doi.org/10.1038/s41579-020-0433-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32887946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Microbiol%20%5BInternet%5D&amp;title=Gut%20microbiota%20in%20human%20metabolic%20health%20and%20disease&amp;volume=19&amp;issue=1&amp;publication_year=2021&amp;pages=55-71&amp;pmid=32887946&amp;doi=10.1038/s41579-020-0433-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0005">
<span class="label">5.</span><cite>Wiertsema SP, Van Bergenhenegouwen J, Garssen J, Knippels LMJ. The interplay between the gut microbiome and the immune system in the context of infectious diseases throughout Life and the role of Nutrition in optimizing treatment strategies. Nutrients [Internet]. 2021.  [cited 2025 May
12]. 13(3):886. doi: 10.3390/nu13030886.
</cite> [<a href="https://doi.org/10.3390/nu13030886" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8001875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33803407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nutrients%20%5BInternet%5D&amp;title=The%20interplay%20between%20the%20gut%20microbiome%20and%20the%20immune%20system%20in%20the%20context%20of%20infectious%20diseases%20throughout%20Life%20and%20the%20role%20of%20Nutrition%20in%20optimizing%20treatment%20strategies&amp;volume=13&amp;issue=3&amp;publication_year=2021&amp;pages=886&amp;pmid=33803407&amp;doi=10.3390/nu13030886&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0006">
<span class="label">6.</span><cite>Sharma R, Young C, Neu J. Molecular modulation of intestinal epithelial barrier: contribution of microbiota. J Appl Psychol Biomedicine And Biotechnology [Internet]. 2010.  [cited 2025 May
12]. 2010:1–15. doi: 10.1155/2010/305879.</cite> [<a href="https://doi.org/10.1155/2010/305879" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2817557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20150966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Psychol%20Biomedicine%20And%20Biotechnology%20%5BInternet%5D&amp;title=Molecular%20modulation%20of%20intestinal%20epithelial%20barrier:%20contribution%20of%20microbiota&amp;volume=2010&amp;publication_year=2010&amp;pages=1-15&amp;pmid=20150966&amp;doi=10.1155/2010/305879&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0007">
<span class="label">7.</span><cite>Marchix J, Goddard G, Helmrath MA. Host-gut microbiota crosstalk in intestinal adaptation. Cellular Mol Gastroenterol Hepatol [Internet]. 2018.  [cited 2025 May
12]. 6(2):149–162. doi: 10.1016/j.jcmgh.2018.01.024.</cite> [<a href="https://doi.org/10.1016/j.jcmgh.2018.01.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6047313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30023411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cellular%20Mol%20Gastroenterol%20Hepatol%20%5BInternet%5D&amp;title=Host-gut%20microbiota%20crosstalk%20in%20intestinal%20adaptation&amp;volume=6&amp;issue=2&amp;publication_year=2018&amp;pages=149-162&amp;pmid=30023411&amp;doi=10.1016/j.jcmgh.2018.01.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0008">
<span class="label">8.</span><cite>Zhang Y, Chen R, Zhang D, Qi S, Liu Y. Metabolite interactions between host and microbiota during health and disease: which feeds the other?
Biomed &amp; Pharmacother. 2023.  [cited 2025 May
12]. 160:114295. <a href="https://linkinghub.elsevier.com/retrieve/pii/S0753332223000835" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://linkinghub.elsevier.com/retrieve/pii/S0753332223000835</a>.</cite> [<a href="https://doi.org/10.1016/j.biopha.2023.114295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36709600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomed%20&amp;%20Pharmacother&amp;title=Metabolite%20interactions%20between%20host%20and%20microbiota%20during%20health%20and%20disease:%20which%20feeds%20the%20other?&amp;volume=160&amp;publication_year=2023&amp;pages=114295&amp;pmid=36709600&amp;doi=10.1016/j.biopha.2023.114295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0009">
<span class="label">9.</span><cite>Dong F, Perdew GH. The aryl hydrocarbon receptor as a mediator of host-microbiota interplay. Gut Microbes [Internet]. 2020.  [cited 2025 May
12]. 12(1):1859812. doi: 10.1080/19490976.2020.1859812.
</cite> [<a href="https://doi.org/10.1080/19490976.2020.1859812" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7781536/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33382356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut%20Microbes%20%5BInternet%5D&amp;title=The%20aryl%20hydrocarbon%20receptor%20as%20a%20mediator%20of%20host-microbiota%20interplay&amp;volume=12&amp;issue=1&amp;publication_year=2020&amp;pages=1859812&amp;pmid=33382356&amp;doi=10.1080/19490976.2020.1859812&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0010">
<span class="label">10.</span><cite>Leeming ER, Johnson AJ, Spector TD, Le Roy CI. Effect of diet on the gut microbiota: rethinking intervention duration. Nutrients [Internet]. 2019.  [cited 2025 May
12]. 11(12):2862. doi: 10.3390/nu11122862.
</cite> [<a href="https://doi.org/10.3390/nu11122862" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6950569/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31766592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nutrients%20%5BInternet%5D&amp;title=Effect%20of%20diet%20on%20the%20gut%20microbiota:%20rethinking%20intervention%20duration&amp;volume=11&amp;issue=12&amp;publication_year=2019&amp;pages=2862&amp;pmid=31766592&amp;doi=10.3390/nu11122862&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0011">
<span class="label">11.</span><cite>Corrêa RO, Castro PR, Fachi JL, Nirello VD, El-Sahhar S, Imada S, Pereira GV, Pral LP, Araújo NVP, Fernandes MF, et al. Inulin diet uncovers complex diet-microbiota-immune cell interactions remodeling the gut epithelium. Microbiome [Internet]. 2023.  [cited 2025 May
12]. 11(1):90. doi: 10.1186/s40168-023-01520-2.
</cite> [<a href="https://doi.org/10.1186/s40168-023-01520-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10131329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37101209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome%20%5BInternet%5D&amp;title=Inulin%20diet%20uncovers%20complex%20diet-microbiota-immune%20cell%20interactions%20remodeling%20the%20gut%20epithelium&amp;volume=11&amp;issue=1&amp;publication_year=2023&amp;pages=90&amp;pmid=37101209&amp;doi=10.1186/s40168-023-01520-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0012">
<span class="label">12.</span><cite>Calibasi-Kocal G, Mashinchian O, Basbinar Y, Ellidokuz E, Cheng C-W, Yilmaz ÖH. Nutritional control of intestinal stem cells in homeostasis and Tumorigenesis. Trends In Endocrinol &amp; Metab. 2021.  [cited 2025 May
12]. 32(1):20–35. <a href="https://linkinghub.elsevier.com/retrieve/pii/S1043276020302216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://linkinghub.elsevier.com/retrieve/pii/S1043276020302216</a>.
</cite> [<a href="https://doi.org/10.1016/j.tem.2020.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33277157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20In%20Endocrinol%20&amp;%20Metab&amp;title=Nutritional%20control%20of%20intestinal%20stem%20cells%20in%20homeostasis%20and%20Tumorigenesis&amp;volume=32&amp;issue=1&amp;publication_year=2021&amp;pages=20-35&amp;pmid=33277157&amp;doi=10.1016/j.tem.2020.11.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0013">
<span class="label">13.</span><cite>Singhal R, Shah YM. Oxygen battle in the gut: hypoxia and hypoxia-inducible factors in metabolic and inflammatory responses in the intestine. J Biol Chem [Internet]. 2020.  [cited 2025 May
12]. 295:10493–10505. doi: 10.1074/jbc.REV120.011188.
</cite> [<a href="https://doi.org/10.1074/jbc.REV120.011188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7383395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32503843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem%20%5BInternet%5D&amp;title=Oxygen%20battle%20in%20the%20gut:%20hypoxia%20and%20hypoxia-inducible%20factors%20in%20metabolic%20and%20inflammatory%20responses%20in%20the%20intestine&amp;volume=295&amp;publication_year=2020&amp;pages=10493-10505&amp;pmid=32503843&amp;doi=10.1074/jbc.REV120.011188&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0014">
<span class="label">14.</span><cite>Pral LP, Fachi JL, Corrêa RO, Colonna M, Vinolo MAR. Hypoxia and HIF-1 as key regulators of gut microbiota and host interactions. Trends In Immunol [Internet]. 2021.  [cited 2025 May
12]. 42(7):604–621. doi: 10.1016/j.it.2021.05.004.
</cite> [<a href="https://doi.org/10.1016/j.it.2021.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8283795/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34171295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20In%20Immunol%20%5BInternet%5D&amp;title=Hypoxia%20and%20HIF-1%20as%20key%20regulators%20of%20gut%20microbiota%20and%20host%20interactions&amp;volume=42&amp;issue=7&amp;publication_year=2021&amp;pages=604-621&amp;pmid=34171295&amp;doi=10.1016/j.it.2021.05.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0015">
<span class="label">15.</span><cite>Walaas GA, Gopalakrishnan S, Bakke I, Skovdahl HK, Flatberg A, Østvik AE, Sandvik AK, Bruland T. Physiological hypoxia improves growth and functional differentiation of human intestinal epithelial organoids. Front Immunol [Internet]. 2023.  [cited 2025 May
12]. 14:1095812. <a href="https://www.frontiersin.org/articles/10.3389/fimmu.2023.1095812/full" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frontiersin.org/articles/10.3389/fimmu.2023.1095812/full</a>.
</cite> [<a href="https://doi.org/10.3389/fimmu.2023.1095812" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9922616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36793710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol%20%5BInternet%5D&amp;title=Physiological%20hypoxia%20improves%20growth%20and%20functional%20differentiation%20of%20human%20intestinal%20epithelial%20organoids&amp;volume=14&amp;publication_year=2023&amp;pages=1095812&amp;pmid=36793710&amp;doi=10.3389/fimmu.2023.1095812&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0016">
<span class="label">16.</span><cite>Byndloss MX, Olsan EE, Rivera-Chávez F, Tiffany CR, Cevallos SA, Lokken KL, Torres TP, Byndloss AJ, Faber F, Gao Y, et al. Microbiota-activated PPAR-γ signaling inhibits dysbiotic Enterobacteriaceae expansion. Science [Internet]. 2017.  [cited 2025 May
12]. 357(6351):570–575. doi: 10.1126/science.aam9949.
</cite> [<a href="https://doi.org/10.1126/science.aam9949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5642957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28798125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20%5BInternet%5D&amp;title=Microbiota-activated%20PPAR-%CE%B3%20signaling%20inhibits%20dysbiotic%20Enterobacteriaceae%20expansion&amp;volume=357&amp;issue=6351&amp;publication_year=2017&amp;pages=570-575&amp;pmid=28798125&amp;doi=10.1126/science.aam9949&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0017">
<span class="label">17.</span><cite>Kierans SJ, Fagundes RR, Malkov MI, Sparkes R, Dillon ET, Smolenski A, Faber KN, Taylor CT. Hypoxia induces a glycolytic complex in intestinal epithelial cells independent of HIF-1-driven glycolytic gene expression. Proc Natl Acad Sci USA. 2023.  [cited 2025 May
12]. 120(35):e2208117120. doi: 10.1073/pnas.2208117120.
</cite> [<a href="https://doi.org/10.1073/pnas.2208117120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10469334/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37603756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Hypoxia%20induces%20a%20glycolytic%20complex%20in%20intestinal%20epithelial%20cells%20independent%20of%20HIF-1-driven%20glycolytic%20gene%20expression&amp;volume=120&amp;issue=35&amp;publication_year=2023&amp;pages=e2208117120&amp;pmid=37603756&amp;doi=10.1073/pnas.2208117120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0018">
<span class="label">18.</span><cite>Taylor CT, Scholz CC. The effect of HIF on metabolism and immunity. Nat Rev Nephrol [Internet]. 2022.  [cited 2025 May
12]. 18(9):573–587. doi: 10.1038/s41581-022-00587-8.
</cite> [<a href="https://doi.org/10.1038/s41581-022-00587-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9208707/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35726016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Nephrol%20%5BInternet%5D&amp;title=The%20effect%20of%20HIF%20on%20metabolism%20and%20immunity&amp;volume=18&amp;issue=9&amp;publication_year=2022&amp;pages=573-587&amp;pmid=35726016&amp;doi=10.1038/s41581-022-00587-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0019">
<span class="label">19.</span><cite>Glover LE, Colgan SP. Hypoxia and metabolic factors that influence inflammatory bowel disease pathogenesis. Gastroenterol [Internet]. 2011.  [cited 2025 May
12]. 140(6):1748–1755. doi: 10.1053/j.gastro.2011.01.056.</cite> [<a href="https://doi.org/10.1053/j.gastro.2011.01.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3093411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21530741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterol%20%5BInternet%5D&amp;title=Hypoxia%20and%20metabolic%20factors%20that%20influence%20inflammatory%20bowel%20disease%20pathogenesis&amp;volume=140&amp;issue=6&amp;publication_year=2011&amp;pages=1748-1755&amp;pmid=21530741&amp;doi=10.1053/j.gastro.2011.01.056&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0020">
<span class="label">20.</span><cite>Kelly CJ, Zheng L, Campbell EL, Saeedi B, Scholz CC, Bayless AJ, Wilson KE, Glover LE, Kominsky DJ, Magnuson A, et al. Crosstalk between microbiota-derived short-chain fatty acids and intestinal epithelial HIF augments tissue barrier function. Cell Host &amp; Microbe. 2015.  [cited 2025 May
12]. 17(5):662–671. <a href="https://linkinghub.elsevier.com/retrieve/pii/S1931312815001225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://linkinghub.elsevier.com/retrieve/pii/S1931312815001225</a>.
</cite> [<a href="https://doi.org/10.1016/j.chom.2015.03.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4433427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25865369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20&amp;%20Microbe&amp;title=Crosstalk%20between%20microbiota-derived%20short-chain%20fatty%20acids%20and%20intestinal%20epithelial%20HIF%20augments%20tissue%20barrier%20function&amp;volume=17&amp;issue=5&amp;publication_year=2015&amp;pages=662-671&amp;pmid=25865369&amp;doi=10.1016/j.chom.2015.03.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0021">
<span class="label">21.</span><cite>Rivera-Chávez F, Zhang LF, Faber F, Lopez CA, Byndloss MX, Olsan EE, Xu G, Velazquez EM, Lebrilla CB, Winter SE, et al. Depletion of butyrate-producing clostridia from the gut microbiota drives an aerobic luminal expansion of salmonella. Cell Host &amp; Microbe [Internet]. 2016.  [cited 2025 May
12]. 19(4):443–454. doi: 10.1016/j.chom.2016.03.004.
</cite> [<a href="https://doi.org/10.1016/j.chom.2016.03.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4832419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27078066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20&amp;%20Microbe%20%5BInternet%5D&amp;title=Depletion%20of%20butyrate-producing%20clostridia%20from%20the%20gut%20microbiota%20drives%20an%20aerobic%20luminal%20expansion%20of%20salmonella&amp;volume=19&amp;issue=4&amp;publication_year=2016&amp;pages=443-454&amp;pmid=27078066&amp;doi=10.1016/j.chom.2016.03.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0022">
<span class="label">22.</span><cite>Yoo W, Zieba JK, Foegeding NJ, Torres TP, Shelton CD, Shealy NG, Byndloss AJ, Cevallos SA, Gertz E, Tiffany CR, et al. High-fat diet–induced colonocyte dysfunction escalates microbiota-derived trimethylamine N -oxide. Sci [Internet]. 2021.  [cited 2025 May
12]. 373(6556):813–818. doi: 10.1126/science.aba3683.</cite> [<a href="https://doi.org/10.1126/science.aba3683" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8506909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34385401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20%5BInternet%5D&amp;title=High-fat%20diet%E2%80%93induced%20colonocyte%20dysfunction%20escalates%20microbiota-derived%20trimethylamine%20N%20-oxide&amp;volume=373&amp;issue=6556&amp;publication_year=2021&amp;pages=813-818&amp;pmid=34385401&amp;doi=10.1126/science.aba3683&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0023">
<span class="label">23.</span><cite>Zhu Q, Han Y, Wang X, Jia R, Zhang J, Liu M, Zhang W. Hypoxia exacerbates intestinal injury and inflammatory response mediated by myeloperoxidase during salmonella typhimurium infection in mice. Gut Pathog [Internet]. 2023.  [cited 2025 May
12]. 15(1):62. doi: 10.1186/s13099-023-00586-5.
</cite> [<a href="https://doi.org/10.1186/s13099-023-00586-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10688069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38037141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut%20Pathog%20%5BInternet%5D&amp;title=Hypoxia%20exacerbates%20intestinal%20injury%20and%20inflammatory%20response%20mediated%20by%20myeloperoxidase%20during%20salmonella%20typhimurium%20infection%20in%20mice&amp;volume=15&amp;issue=1&amp;publication_year=2023&amp;pages=62&amp;pmid=38037141&amp;doi=10.1186/s13099-023-00586-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0024">
<span class="label">24.</span><cite>Reeves PG, Nielsen FH, Fahey GC. AIN-93 purified diets for laboratory rodents: final report of the American Institute of nutrition ad hoc writing committee on the reformulation of the AIN-76A rodent diet. J Sport Hist Of Nutrition [Internet]. 1993.  [cited 2025 May
12]. 123(11):1939–1951. doi: 10.1093/jn/123.11.1939.</cite> [<a href="https://doi.org/10.1093/jn/123.11.1939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8229312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Sport%20Hist%20Of%20Nutrition%20%5BInternet%5D&amp;title=AIN-93%20purified%20diets%20for%20laboratory%20rodents:%20final%20report%20of%20the%20American%20Institute%20of%20nutrition%20ad%20hoc%20writing%20committee%20on%20the%20reformulation%20of%20the%20AIN-76A%20rodent%20diet&amp;volume=123&amp;issue=11&amp;publication_year=1993&amp;pages=1939-1951&amp;pmid=8229312&amp;doi=10.1093/jn/123.11.1939&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0025">
<span class="label">25.</span><cite>Tuganbaev T, Mor U, Bashiardes S, Liwinski T, Nobs SP, Leshem A, Dori-Bachash M, Thaiss CA, Pinker EY, Ratiner K, et al. Diet diurnally regulates small intestinal microbiome-epithelial-immune homeostasis and enteritis. Cell [Internet]. 2020.  [cited 2025 May
12]. 182(6):1441–1459.e21. doi: 10.1016/j.cell.2020.08.027.
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.08.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32888430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20%5BInternet%5D&amp;title=Diet%20diurnally%20regulates%20small%20intestinal%20microbiome-epithelial-immune%20homeostasis%20and%20enteritis&amp;volume=182&amp;issue=6&amp;publication_year=2020&amp;pages=1441-1459.e21&amp;pmid=32888430&amp;doi=10.1016/j.cell.2020.08.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0026">
<span class="label">26.</span><cite>Durand L, Zbinden M, Cueff-Gauchard V, Duperron S, Roussel EG, Shillito B, Cambon-Bonavita M-A. Microbial diversity associated with the hydrothermal shrimp rimicaris exoculata gut and occurrence of a resident microbial community. FEMS Microbiol Ecol [Internet]. 2010.  [cited 2025 May
12]. 71(2):291–303. doi: 10.1111/j.1574-6941.2009.00806.x.
</cite> [<a href="https://doi.org/10.1111/j.1574-6941.2009.00806.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19951370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol%20Ecol%20%5BInternet%5D&amp;title=Microbial%20diversity%20associated%20with%20the%20hydrothermal%20shrimp%20rimicaris%20exoculata%20gut%20and%20occurrence%20of%20a%20resident%20microbial%20community&amp;volume=71&amp;issue=2&amp;publication_year=2010&amp;pages=291-303&amp;pmid=19951370&amp;doi=10.1111/j.1574-6941.2009.00806.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0027">
<span class="label">27.</span><cite>Goldman SJ, Chen E, Taylor R, Zhang S, Petrosky W, Reiss M, Jin S, Agoulnik I. Use of the ODD-Luciferase transgene for the non-invasive imaging of spontaneous tumors in mice. PLOS ONE [Internet]. 2011.  [cited 2025 May
12]. 6(3):e18269. doi: 10.1371/journal.pone.0018269.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0018269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3066234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21479246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE%20%5BInternet%5D&amp;title=Use%20of%20the%20ODD-Luciferase%20transgene%20for%20the%20non-invasive%20imaging%20of%20spontaneous%20tumors%20in%20mice&amp;volume=6&amp;issue=3&amp;publication_year=2011&amp;pages=e18269&amp;pmid=21479246&amp;doi=10.1371/journal.pone.0018269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0028">
<span class="label">28.</span><cite>Lee J-Y, Cevallos SA, Byndloss MX, Tiffany CR, Olsan EE, Butler BP, Young BM, Rogers AWL, Nguyen H, Kim K, et al. High-fat diet and antibiotics cooperatively impair mitochondrial bioenergetics to trigger dysbiosis that exacerbates pre-inflammatory bowel disease. Cell Host &amp; Microbe [Internet]. 2020.  [cited 2025 May
12]. 28(2):273–284.e6. <a href="https://linkinghub.elsevier.com/retrieve/pii/S1931312820303036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://linkinghub.elsevier.com/retrieve/pii/S1931312820303036</a>.
</cite> [<a href="https://doi.org/10.1016/j.chom.2020.06.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7429289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32668218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20&amp;%20Microbe%20%5BInternet%5D&amp;title=High-fat%20diet%20and%20antibiotics%20cooperatively%20impair%20mitochondrial%20bioenergetics%20to%20trigger%20dysbiosis%20that%20exacerbates%20pre-inflammatory%20bowel%20disease&amp;volume=28&amp;issue=2&amp;publication_year=2020&amp;pages=273-284.e6&amp;pmid=32668218&amp;doi=10.1016/j.chom.2020.06.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0029">
<span class="label">29.</span><cite>Schneider CA, Rasband WS, Eliceiri KW. NIH image to ImageJ: 25 years of image analysis. Nat Methods [Internet]. 2012.  [cited 2025 May
12]. 9(7):671–675. doi: 10.1038/nmeth.2089.
</cite> [<a href="https://doi.org/10.1038/nmeth.2089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5554542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods%20%5BInternet%5D&amp;title=NIH%20image%20to%20ImageJ:%2025%20years%20of%20image%20analysis&amp;volume=9&amp;issue=7&amp;publication_year=2012&amp;pages=671-675&amp;pmid=22930834&amp;doi=10.1038/nmeth.2089&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0030">
<span class="label">30.</span><cite>Ludikhuize MC, Meerlo M, Gallego MP, Xanthakis D, Burgaya Julià M, Nguyen NTB, Brombacher EC, Liv N, Maurice MM, Paik J, et al. Mitochondria define intestinal stem cell differentiation downstream of a FOXO/Notch axis. Cell Metab [Internet]. 2020.  [cited 2025 May
12]. 32(5):889–900.e7. doi: 10.1016/j.cmet.2020.10.005.
</cite> [<a href="https://doi.org/10.1016/j.cmet.2020.10.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33147486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab%20%5BInternet%5D&amp;title=Mitochondria%20define%20intestinal%20stem%20cell%20differentiation%20downstream%20of%20a%20FOXO/Notch%20axis&amp;volume=32&amp;issue=5&amp;publication_year=2020&amp;pages=889-900.e7&amp;pmid=33147486&amp;doi=10.1016/j.cmet.2020.10.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0031">
<span class="label">31.</span><cite>Ewels PA, Peltzer A, Fillinger S, Patel H, Alneberg J, Wilm A, Garcia MU, Di Tommaso P, Nahnsen S. The nf-core framework for community-curated bioinformatics pipelines. Nat Biotechnol [Internet]. 2020.  [cited 2025 May
12]. 38(3):276–278. doi: 10.1038/s41587-020-0439-x.
</cite> [<a href="https://doi.org/10.1038/s41587-020-0439-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32055031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol%20%5BInternet%5D&amp;title=The%20nf-core%20framework%20for%20community-curated%20bioinformatics%20pipelines&amp;volume=38&amp;issue=3&amp;publication_year=2020&amp;pages=276-278&amp;pmid=32055031&amp;doi=10.1038/s41587-020-0439-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0032">
<span class="label">32.</span><cite>Straub D, Blackwell N, Langarica-Fuentes A, Peltzer A, Nahnsen S, Kleindienst S. Interpretations of environmental microbial community studies are biased by the selected 16S rRNA (gene) amplicon sequencing pipeline. Front Microbiol. 2020.  [cited 2025 May
12]. 11:550420. <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2020.550420/full" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frontiersin.org/articles/10.3389/fmicb.2020.550420/full</a>.
</cite> [<a href="https://doi.org/10.3389/fmicb.2020.550420" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7645116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33193131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Interpretations%20of%20environmental%20microbial%20community%20studies%20are%20biased%20by%20the%20selected%2016S%20rRNA%20(gene)%20amplicon%20sequencing%20pipeline&amp;volume=11&amp;publication_year=2020&amp;pages=550420&amp;pmid=33193131&amp;doi=10.3389/fmicb.2020.550420&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0033">
<span class="label">33.</span><cite>Straub D, Tångrot J, Lundin D, Peltzer A, Core Bot NF, Emnilsson S, Bennett A, Sundh J, Brambilla D. Nf-core/ampliseq: ampliseq version 2.13.0 [Internet]. 2025.  [cited 2025 May
12]. <a href="https://zenodo.org/doi/10.5281/zenodo.1493841" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://zenodo.org/doi/10.5281/zenodo.1493841</a>.</cite>
</li>
<li id="cit0034">
<span class="label">34.</span><cite>Ewels P, Magnusson M, Lundin S, Käller M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinf [Internet]. 2016.  [cited 2025 May
12]. 32(19):3047–3048. doi: 10.1093/bioinformatics/btw354.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5039924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27312411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinf%20%5BInternet%5D&amp;title=MultiQC:%20summarize%20analysis%20results%20for%20multiple%20tools%20and%20samples%20in%20a%20single%20report&amp;volume=32&amp;issue=19&amp;publication_year=2016&amp;pages=3047-3048&amp;pmid=27312411&amp;doi=10.1093/bioinformatics/btw354&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0035">
<span class="label">35.</span><cite>Callahan BJ, McMurdie PJ, Rosen MJ, Han AW, Johnson AJA, Holmes SP. DADA2: high-resolution sample inference from illumina amplicon data. Nat Methods [Internet]. 2016.  [cited 2025 May
12]. 13(7):581–583. <a href="https://www.nature.com/articles/nmeth.3869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/nmeth.3869</a>.
</cite> [<a href="https://doi.org/10.1038/nmeth.3869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4927377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27214047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods%20%5BInternet%5D&amp;title=DADA2:%20high-resolution%20sample%20inference%20from%20illumina%20amplicon%20data&amp;volume=13&amp;issue=7&amp;publication_year=2016&amp;pages=581-583&amp;pmid=27214047&amp;doi=10.1038/nmeth.3869&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0036">
<span class="label">36.</span><cite>Quast C, Pruesse E, Yilmaz P, Gerken J, Schweer T, Yarza P, Peplies J, Glöckner FO. The SILVA ribosomal RNA gene database project: improved data processing and web-based tools. Nucleic Acids Res [Internet]. 2012.  [cited 2025 May
12]. 41(D1):D590–6. doi: 10.1093/nar/gks1219.
</cite> [<a href="https://doi.org/10.1093/nar/gks1219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23193283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res%20%5BInternet%5D&amp;title=The%20SILVA%20ribosomal%20RNA%20gene%20database%20project:%20improved%20data%20processing%20and%20web-based%20tools&amp;volume=41&amp;issue=D1&amp;publication_year=2012&amp;pages=D590-6&amp;pmid=23193283&amp;doi=10.1093/nar/gks1219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0037">
<span class="label">37.</span><cite>Bolyen E, Rideout JR, Dillon MR, Bokulich NA, Abnet CC, Al-Ghalith GA, Alexander H, Alm EJ, Arumugam M, Asnicar F, et al. Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2. Nat Biotechnol [Internet]. 2019.  [cited 2025 May
12]. 37(8):852–857. doi: 10.1038/s41587-019-0209-9.
</cite> [<a href="https://doi.org/10.1038/s41587-019-0209-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7015180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31341288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol%20%5BInternet%5D&amp;title=Reproducible,%20interactive,%20scalable%20and%20extensible%20microbiome%20data%20science%20using%20QIIME%202&amp;volume=37&amp;issue=8&amp;publication_year=2019&amp;pages=852-857&amp;pmid=31341288&amp;doi=10.1038/s41587-019-0209-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0038">
<span class="label">38.</span><cite>McMurdie PJ, Holmes S, Watson M. Phyloseq: an R package for reproducible interactive analysis and graphics of microbiome census data. PLOS ONE [Internet]. 2013.  [cited 2025 May
12]. 8(4):e61217. doi: 10.1371/journal.pone.0061217.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0061217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3632530/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23630581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE%20%5BInternet%5D&amp;title=Phyloseq:%20an%20R%20package%20for%20reproducible%20interactive%20analysis%20and%20graphics%20of%20microbiome%20census%20data&amp;volume=8&amp;issue=4&amp;publication_year=2013&amp;pages=e61217&amp;pmid=23630581&amp;doi=10.1371/journal.pone.0061217&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0039">
<span class="label">39.</span><cite>Barnett D, Arts I, Penders J. microViz: an R package for microbiome data visualization and statistics. JOSS [Internet]. 2021.  [cited 2025 May
12]. 6(63):3201. doi: 10.21105/joss.03201.</cite> [<a href="https://doi.org/10.21105/joss.03201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JOSS%20%5BInternet%5D&amp;title=microViz:%20an%20R%20package%20for%20microbiome%20data%20visualization%20and%20statistics&amp;volume=6&amp;issue=63&amp;publication_year=2021&amp;pages=3201&amp;doi=10.21105/joss.03201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0040">
<span class="label">40.</span><cite>Xu S, Zhan L, Tang W, Wang Q, Dai Z, Zhou L, Feng T, Chen M, Wu T, Hu E, et al. Microbiota process: a comprehensive R package for deep mining microbiome. The Innov [Internet]. 2023.  [cited 2025 May
12]. 4(2):100388. <a href="https://linkinghub.elsevier.com/retrieve/pii/S2666675823000164" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://linkinghub.elsevier.com/retrieve/pii/S2666675823000164</a>.</cite> [<a href="https://doi.org/10.1016/j.xinn.2023.100388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9988672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36895758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Innov%20%5BInternet%5D&amp;title=Microbiota%20process:%20a%20comprehensive%20R%20package%20for%20deep%20mining%20microbiome&amp;volume=4&amp;issue=2&amp;publication_year=2023&amp;pages=100388&amp;pmid=36895758&amp;doi=10.1016/j.xinn.2023.100388&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0041">
<span class="label">41.</span><cite>Marini F, Carregari VC, Greco V, Ronci M, Iavarone F, Persichilli S, Castagnola M, Urbani A, Pieroni L. Exploring the HeLa dark mitochondrial proteome. Front Cell Dev Biol [Internet]. 2020.  [cited 2025 May
12]. 8:137. <a href="https://www.frontiersin.org/article/10.3389/fcell.2020.00137/full" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frontiersin.org/article/10.3389/fcell.2020.00137/full</a>.
</cite> [<a href="https://doi.org/10.3389/fcell.2020.00137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7066081/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32195257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Dev%20Biol%20%5BInternet%5D&amp;title=Exploring%20the%20HeLa%20dark%20mitochondrial%20proteome&amp;volume=8&amp;publication_year=2020&amp;pages=137&amp;pmid=32195257&amp;doi=10.3389/fcell.2020.00137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0042">
<span class="label">42.</span><cite>Reis-de-Oliveira G, Carregari VC, Sousa GRDRD, Martins-de-Souza D. Sousa GR dos R de, Martins-de-Souza D. OmicScope unravels systems-level insights from quantitative proteomics data. Nat Commun [Internet]. 2024.  [cited 2025 May
12]. 15(1):6510. doi: 10.1038/s41467-024-50875-z.
</cite> [<a href="https://doi.org/10.1038/s41467-024-50875-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11297029/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39095347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun%20%5BInternet%5D&amp;title=Sousa%20GR%20dos%20R%20de,%20Martins-de-Souza%20D.%20OmicScope%20unravels%20systems-level%20insights%20from%20quantitative%20proteomics%20data&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=6510&amp;pmid=39095347&amp;doi=10.1038/s41467-024-50875-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0043">
<span class="label">43.</span><cite>Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, Amin N, Schwikowski B, Ideker T. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res [Internet]. 2003.  [cited 2025 May
12]. 13(11):2498–2504. doi: 10.1101/gr.1239303.
</cite> [<a href="https://doi.org/10.1101/gr.1239303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC403769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res%20%5BInternet%5D&amp;title=Cytoscape:%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&amp;volume=13&amp;issue=11&amp;publication_year=2003&amp;pages=2498-2504&amp;pmid=14597658&amp;doi=10.1101/gr.1239303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0044">
<span class="label">44.</span><cite>Boets E, Deroover L, Houben E, Vermeulen K, Gomand S, Delcour J, Verbeke K. Quantification of in vivo colonic short chain fatty acid production from inulin. Nutrients [Internet]. 2015.  [cited 2025 May
12]. 7(11):8916–8929. doi: 10.3390/nu7115440.
</cite> [<a href="https://doi.org/10.3390/nu7115440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4663568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26516911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nutrients%20%5BInternet%5D&amp;title=Quantification%20of%20in%20vivo%20colonic%20short%20chain%20fatty%20acid%20production%20from%20inulin&amp;volume=7&amp;issue=11&amp;publication_year=2015&amp;pages=8916-8929&amp;pmid=26516911&amp;doi=10.3390/nu7115440&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0045">
<span class="label">45.</span><cite>Holmes ZC, Villa MM, Durand HK, Jiang S, Dallow EP, Petrone BL, Silverman JD, Lin P-H, David LA. Microbiota responses to different prebiotics are conserved within individuals and associated with habitual fiber intake. Microbiome [Internet]. 2022.  [cited 2025 May
12]. 10(1):114. doi: 10.1186/s40168-022-01307-x.
</cite> [<a href="https://doi.org/10.1186/s40168-022-01307-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9336045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35902900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome%20%5BInternet%5D&amp;title=Microbiota%20responses%20to%20different%20prebiotics%20are%20conserved%20within%20individuals%20and%20associated%20with%20habitual%20fiber%20intake&amp;volume=10&amp;issue=1&amp;publication_year=2022&amp;pages=114&amp;pmid=35902900&amp;doi=10.1186/s40168-022-01307-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0046">
<span class="label">46.</span><cite>Kaiko GE, Ryu SH, Koues OI, Collins PL, Solnica-Krezel L, Pearce EJ, Pearce EL, Oltz EM, Stappenbeck TS. The colonic crypt protects stem cells from microbiota-derived metabolites. Cell [Internet]. 2016.  [cited 2025 May
12]. 165(7):1708–1720. doi: 10.1016/j.cell.2016.05.018.
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.05.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5026192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27264604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20%5BInternet%5D&amp;title=The%20colonic%20crypt%20protects%20stem%20cells%20from%20microbiota-derived%20metabolites&amp;volume=165&amp;issue=7&amp;publication_year=2016&amp;pages=1708-1720&amp;pmid=27264604&amp;doi=10.1016/j.cell.2016.05.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0047">
<span class="label">47.</span><cite>Salvi PS, Cowles RA. Butyrate and the intestinal epithelium: modulation of proliferation and inflammation in homeostasis and disease. Cells [Internet]. 2021.  [cited 2025 May
12]. 10(7):1775. doi: 10.3390/cells10071775.
</cite> [<a href="https://doi.org/10.3390/cells10071775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8304699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34359944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells%20%5BInternet%5D&amp;title=Butyrate%20and%20the%20intestinal%20epithelium:%20modulation%20of%20proliferation%20and%20inflammation%20in%20homeostasis%20and%20disease&amp;volume=10&amp;issue=7&amp;publication_year=2021&amp;pages=1775&amp;pmid=34359944&amp;doi=10.3390/cells10071775&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0048">
<span class="label">48.</span><cite>Wang RX, Henen MA, Lee JS, Vögeli B, Colgan SP. Microbiota-derived butyrate is an endogenous HIF prolyl hydroxylase inhibitor. Gut Microbes [Internet]. 2021.  [cited 2025 May
12]. 13(1):1938380. doi: 10.1080/19490976.2021.1938380.
</cite> [<a href="https://doi.org/10.1080/19490976.2021.1938380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8253137/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34190032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut%20Microbes%20%5BInternet%5D&amp;title=Microbiota-derived%20butyrate%20is%20an%20endogenous%20HIF%20prolyl%20hydroxylase%20inhibitor&amp;volume=13&amp;issue=1&amp;publication_year=2021&amp;pages=1938380&amp;pmid=34190032&amp;doi=10.1080/19490976.2021.1938380&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0049">
<span class="label">49.</span><cite>Safran M, Kim WY, O’Connell F, Flippin L, Günzler V, Horner JW, DePinho RA, Kaelin WG. Mouse model for noninvasive imaging of HIF prolyl hydroxylase activity: assessment of an oral agent that stimulates erythropoietin production. Proc Natl Acad Sci USA. 2006.  [cited 2025 May
12]. 103(1):105–110. doi: 10.1073/pnas.0509459103.
</cite> [<a href="https://doi.org/10.1073/pnas.0509459103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1324998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16373502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Mouse%20model%20for%20noninvasive%20imaging%20of%20HIF%20prolyl%20hydroxylase%20activity:%20assessment%20of%20an%20oral%20agent%20that%20stimulates%20erythropoietin%20production&amp;volume=103&amp;issue=1&amp;publication_year=2006&amp;pages=105-110&amp;pmid=16373502&amp;doi=10.1073/pnas.0509459103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0050">
<span class="label">50.</span><cite>Sonnenburg ED, Zheng H, Joglekar P, Higginbottom SK, Firbank SJ, Bolam DN, Sonnenburg JL. Specificity of polysaccharide use in intestinal bacteroides species determines diet-induced microbiota alterations. Cell [Internet]. 2010.  [cited 2025 May
12]. 141(7):1241–1252. doi: 10.1016/j.cell.2010.05.005.
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.05.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2900928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20603004/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20%5BInternet%5D&amp;title=Specificity%20of%20polysaccharide%20use%20in%20intestinal%20bacteroides%20species%20determines%20diet-induced%20microbiota%20alterations&amp;volume=141&amp;issue=7&amp;publication_year=2010&amp;pages=1241-1252&amp;pmid=20603004&amp;doi=10.1016/j.cell.2010.05.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0051">
<span class="label">51.</span><cite>Hutchinson NT, Wang SS, Rund LA, Caetano-Silva ME, Allen JM, Johnson RW, Woods JA. Effects of an inulin fiber diet on the gut microbiome, colon, and inflammatory biomarkers in aged mice. Exp Gerontolo [Internet]. 2023.  [cited 2025 May
12]. 176:112164. doi: 10.1016/j.exger.2023.112164.</cite> [<a href="https://doi.org/10.1016/j.exger.2023.112164" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10159939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37011713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp%20Gerontolo%20%5BInternet%5D&amp;title=Effects%20of%20an%20inulin%20fiber%20diet%20on%20the%20gut%20microbiome,%20colon,%20and%20inflammatory%20biomarkers%20in%20aged%20mice&amp;volume=176&amp;publication_year=2023&amp;pages=112164&amp;pmid=37011713&amp;doi=10.1016/j.exger.2023.112164&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0052">
<span class="label">52.</span><cite>Guimarães JB, Rodrigues VF, Pereira ÍS, Manso GMDC, Elias-Oliveira J, Leite JA, Waldetario MCGM, De Oliveira S, Gomes ABDSP, Faria AMC, et al. Inulin prebiotic ameliorates type 1 diabetes dictating regulatory T cell homing via CCR4 to pancreatic islets and butyrogenic gut microbiota in murine model. J Leukoc Biol [Internet]. 2024.  [cited 2025 May
12]. 115(3):483–496. <a href="https://academic.oup.com/jleukbio/article/115/3/483/7382080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/jleukbio/article/115/3/483/7382080</a>.
</cite> [<a href="https://doi.org/10.1093/jleuko/qiad132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37947010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Leukoc%20Biol%20%5BInternet%5D&amp;title=Inulin%20prebiotic%20ameliorates%20type%201%20diabetes%20dictating%20regulatory%20T%20cell%20homing%20via%20CCR4%20to%20pancreatic%20islets%20and%20butyrogenic%20gut%20microbiota%20in%20murine%20model&amp;volume=115&amp;issue=3&amp;publication_year=2024&amp;pages=483-496&amp;pmid=37947010&amp;doi=10.1093/jleuko/qiad132&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0053">
<span class="label">53.</span><cite>Parada Venegas D, De La Fuente MK, Landskron G, González MJ, Quera R, Dijkstra G, Harmsen HJM, Faber KN, Hermoso MA. Short chain fatty acids (SCFAs)-mediated gut epithelial and immune regulation and its relevance for inflammatory bowel diseases. Front Immunol [Internet]. 2019.  [cited 2025 May
12]. 10:277. <a href="https://www.frontiersin.org/article/10.3389/fimmu.2019.00277/full" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frontiersin.org/article/10.3389/fimmu.2019.00277/full</a>.
</cite> [<a href="https://doi.org/10.3389/fimmu.2019.00277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6421268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30915065/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol%20%5BInternet%5D&amp;title=Short%20chain%20fatty%20acids%20(SCFAs)-mediated%20gut%20epithelial%20and%20immune%20regulation%20and%20its%20relevance%20for%20inflammatory%20bowel%20diseases&amp;volume=10&amp;publication_year=2019&amp;pages=277&amp;pmid=30915065&amp;doi=10.3389/fimmu.2019.00277&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0054">
<span class="label">54.</span><cite>Wang D, Odle J, Liu Y. Metabolic regulation of intestinal stem cell homeostasis. Trends Cell Biol [Internet]. 2021.  [cited 2025 May
12]. 31(5):325–327. doi: 10.1016/j.tcb.2021.02.001.
</cite> [<a href="https://doi.org/10.1016/j.tcb.2021.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33648839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cell%20Biol%20%5BInternet%5D&amp;title=Metabolic%20regulation%20of%20intestinal%20stem%20cell%20homeostasis&amp;volume=31&amp;issue=5&amp;publication_year=2021&amp;pages=325-327&amp;pmid=33648839&amp;doi=10.1016/j.tcb.2021.02.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0055">
<span class="label">55.</span><cite>Beyaz S, Mana MD, Roper J, Kedrin D, Saadatpour A, Hong S-J, Bauer-Rowe KE, Xifaras ME, Akkad A, Arias E, et al. High-fat diet enhances stemness and tumorigenicity of intestinal progenitors. Nat [Internet]. 2016.  [cited 2025 May
12]. 531(7592):53–58. doi: 10.1038/nature17173.</cite> [<a href="https://doi.org/10.1038/nature17173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4846772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26935695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20%5BInternet%5D&amp;title=High-fat%20diet%20enhances%20stemness%20and%20tumorigenicity%20of%20intestinal%20progenitors&amp;volume=531&amp;issue=7592&amp;publication_year=2016&amp;pages=53-58&amp;pmid=26935695&amp;doi=10.1038/nature17173&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0056">
<span class="label">56.</span><cite>Shah YM, Matsubara T, Ito S, Yim S-H, Gonzalez FJ. Intestinal hypoxia-inducible transcription factors are essential for iron absorption following iron deficiency. Cell Metab [Internet]. 2009.  [cited 2025 May
12]. 9(2):152–164. doi: 10.1016/j.cmet.2008.12.012.
</cite> [<a href="https://doi.org/10.1016/j.cmet.2008.12.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2659630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19147412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metab%20%5BInternet%5D&amp;title=Intestinal%20hypoxia-inducible%20transcription%20factors%20are%20essential%20for%20iron%20absorption%20following%20iron%20deficiency&amp;volume=9&amp;issue=2&amp;publication_year=2009&amp;pages=152-164&amp;pmid=19147412&amp;doi=10.1016/j.cmet.2008.12.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0057">
<span class="label">57.</span><cite>Chen L, Vasoya RP, Toke NH, Parthasarathy A, Luo S, Chiles E, Flores J, Gao N, Bonder EM, Su X, et al. HNF4 regulates fatty acid oxidation and is required for renewal of intestinal stem cells in mice. Gastroenterol [Internet]. 2020.  [cited 2025 May
12]. 158(4):985–999.e9. doi: 10.1053/j.gastro.2019.11.031.</cite> [<a href="https://doi.org/10.1053/j.gastro.2019.11.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7062567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31759926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterol%20%5BInternet%5D&amp;title=HNF4%20regulates%20fatty%20acid%20oxidation%20and%20is%20required%20for%20renewal%20of%20intestinal%20stem%20cells%20in%20mice&amp;volume=158&amp;issue=4&amp;publication_year=2020&amp;pages=985-999.e9&amp;pmid=31759926&amp;doi=10.1053/j.gastro.2019.11.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0058">
<span class="label">58.</span><cite>Shim J-K, Choi S, Yoon S-J, Choi RJ, Park J, Lee EH, Cho HJ, Lee S, Teo W-Y, Moon JH, et al. Etomoxir, a carnitine palmitoyltransferase 1 inhibitor, combined with temozolomide reduces stemness and invasiveness in patient-derived glioblastoma tumorspheres. Cancer Cell Int [Internet]. 2022.  [cited 2025 May
12]. 22(1):309. doi: 10.1186/s12935-022-02731-7.
</cite> [<a href="https://doi.org/10.1186/s12935-022-02731-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9552483/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36221088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell%20Int%20%5BInternet%5D&amp;title=Etomoxir,%20a%20carnitine%20palmitoyltransferase%201%20inhibitor,%20combined%20with%20temozolomide%20reduces%20stemness%20and%20invasiveness%20in%20patient-derived%20glioblastoma%20tumorspheres&amp;volume=22&amp;issue=1&amp;publication_year=2022&amp;pages=309&amp;pmid=36221088&amp;doi=10.1186/s12935-022-02731-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0059">
<span class="label">59.</span><cite>Liu W, Fan X, Jian B, Wen D, Wang H, Liu Z, Li B. The signaling pathway of hypoxia inducible factor in regulating gut homeostasis. Front Microbiol. 2023.  [cited 2025 May
12]. 14. <a href="https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1289102/full" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2023.1289102/full</a>.</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1289102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10641782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37965556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=The%20signaling%20pathway%20of%20hypoxia%20inducible%20factor%20in%20regulating%20gut%20homeostasis&amp;volume=14&amp;publication_year=2023&amp;pmid=37965556&amp;doi=10.3389/fmicb.2023.1289102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0060">
<span class="label">60.</span><cite>Lan X, Qiu P, Mou C. Hypoxia impacts small intestinal organoid stemness and differentiation [Internet]. 2023.  [cited 2025 May
12]. doi: 10.1101/2023.12.30.573689.</cite> [<a href="https://doi.org/10.1101/2023.12.30.573689" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Lan%20X,%20Qiu%20P,%20Mou%20C.%20Hypoxia%20impacts%20small%20intestinal%20organoid%20stemness%20and%20differentiation%20%5BInternet%5D.%202023.%20%5Bcited%202025%20May%2012%5D.%20doi:%2010.1101/2023.12.30.573689." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0061">
<span class="label">61.</span><cite>Li L, Liu Y, Zhi N, Ji Y, Xu J, Mao G, Wang Y, Ma J, Wang Y. Hypoxic preconditioning accelerates the healing of ischemic intestinal injury by activating HIF-1α/PPARα pathway-mediated fatty acid oxidation. Cell Death Discov. 2024.  [cited 2025 May
12];10(1):164. <a href="https://www.nature.com/articles/s41420-024-01937-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nature.com/articles/s41420-024-01937-0</a>.
</cite> [<a href="https://doi.org/10.1038/s41420-024-01937-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10994932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38575595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Discov&amp;title=Hypoxic%20preconditioning%20accelerates%20the%20healing%20of%20ischemic%20intestinal%20injury%20by%20activating%20HIF-1%CE%B1/PPAR%CE%B1%20pathway-mediated%20fatty%20acid%20oxidation&amp;volume=10&amp;issue=1&amp;publication_year=2024&amp;pages=164&amp;pmid=38575595&amp;doi=10.1038/s41420-024-01937-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0062">
<span class="label">62.</span><cite>Di Luccia B, Acampora V, Saggese A, Calabrò V, Vivo M, Angrisano T, Baccigalupi L, Ricca E, Pollice A. Modulation of intestinal epithelial cell proliferation and apoptosis by lactobacillus gasseri SF1183. Sci Rep [Internet]. 2022.  [cited 2025 May
12];12(1):20248. doi: 10.1038/s41598-022-24483-0.
</cite> [<a href="https://doi.org/10.1038/s41598-022-24483-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9691729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36424419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep%20%5BInternet%5D&amp;title=Modulation%20of%20intestinal%20epithelial%20cell%20proliferation%20and%20apoptosis%20by%20lactobacillus%20gasseri%20SF1183&amp;volume=12&amp;issue=1&amp;publication_year=2022&amp;pages=20248&amp;pmid=36424419&amp;doi=10.1038/s41598-022-24483-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0063">
<span class="label">63.</span><cite>Van Der Flier LG, Clevers H. Stem cells, self-renewal, and differentiation in the intestinal epithelium. Annu Rev Physiol [Internet]. 2009.  [cited 2025 May
12];71(1):241–260. doi: 10.1146/annurev.physiol.010908.163145.
</cite> [<a href="https://doi.org/10.1146/annurev.physiol.010908.163145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18808327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Physiol%20%5BInternet%5D&amp;title=Stem%20cells,%20self-renewal,%20and%20differentiation%20in%20the%20intestinal%20epithelium&amp;volume=71&amp;issue=1&amp;publication_year=2009&amp;pages=241-260&amp;pmid=18808327&amp;doi=10.1146/annurev.physiol.010908.163145&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0064">
<span class="label">64.</span><cite>Yang X, Yang S, Wang C, Kuang S. The hypoxia-inducible factors HIF1α and HIF2α are dispensable for embryonic muscle development but essential for postnatal muscle regeneration. J Biol Chem [Internet]. 2017.  [cited 2025 May
12];292(14):5981–5991. doi: 10.1074/jbc.M116.756312.
</cite> [<a href="https://doi.org/10.1074/jbc.M116.756312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5392588/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28232488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem%20%5BInternet%5D&amp;title=The%20hypoxia-inducible%20factors%20HIF1%CE%B1%20and%20HIF2%CE%B1%20are%20dispensable%20for%20embryonic%20muscle%20development%20but%20essential%20for%20postnatal%20muscle%20regeneration&amp;volume=292&amp;issue=14&amp;publication_year=2017&amp;pages=5981-5991&amp;pmid=28232488&amp;doi=10.1074/jbc.M116.756312&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0065">
<span class="label">65.</span><cite>Enguita FJ, Martins LO, Henriques AO, Carrondo MA. Crystal structure of a bacterial endospore coat component. J Biol Chem [Internet]. 2003.  [cited 2025 May
12];278(21):19416–19425. doi: 10.1074/jbc.M301251200.
</cite> [<a href="https://doi.org/10.1074/jbc.M301251200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12637519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem%20%5BInternet%5D&amp;title=Crystal%20structure%20of%20a%20bacterial%20endospore%20coat%20component&amp;volume=278&amp;issue=21&amp;publication_year=2003&amp;pages=19416-19425&amp;pmid=12637519&amp;doi=10.1074/jbc.M301251200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0066">
<span class="label">66.</span><cite>Peng J, Shen S, Wang J, Jiang H, Wang Y. Ηypoxia-inducible factor 1-α promotes colon cell proliferation and migration by upregulating AMPK-related protein kinase 5 under hypoxic conditions. Oncol Lett [Internet]. 2018.  [cited 2025 May
12]. doi: 10.3892/ol.2018.7748.</cite> [<a href="https://doi.org/10.3892/ol.2018.7748" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5796283/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29467884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol%20Lett%20%5BInternet%5D&amp;title=%CE%97ypoxia-inducible%20factor%201-%CE%B1%20promotes%20colon%20cell%20proliferation%20and%20migration%20by%20upregulating%20AMPK-related%20protein%20kinase%205%20under%20hypoxic%20conditions&amp;publication_year=2018&amp;pmid=29467884&amp;doi=10.3892/ol.2018.7748&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0067">
<span class="label">67.</span><cite>Simiantonaki N, Taxeidis M, Jayasinghe C, Kurzik-Dumke U, Kirkpatrick CJ. Hypoxia-inducible factor 1 alpha expression increases during colorectal carcinogenesis and tumor progression. BMC Cancer [Internet]. 2008.  [cited 2025 May
12];8(1):320. doi: 10.1186/1471-2407-8-320.
</cite> [<a href="https://doi.org/10.1186/1471-2407-8-320" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2584660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18983642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer%20%5BInternet%5D&amp;title=Hypoxia-inducible%20factor%201%20alpha%20expression%20increases%20during%20colorectal%20carcinogenesis%20and%20tumor%20progression&amp;volume=8&amp;issue=1&amp;publication_year=2008&amp;pages=320&amp;pmid=18983642&amp;doi=10.1186/1471-2407-8-320&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0068">
<span class="label">68.</span><cite>Mladenova DN, Dahlstrom JE, Tran PN, Benthani F, Bean EG, Ng I, Pangon L, Currey N, Kohonen-Corish MRJ. HIF1α deficiency reduces colon inflammation in a mouse model of proximal colon cancer. Disease Model &amp; Mechanisms [Internet]. 2015.  [cited 2025 May
12];dmm.019000. doi: 10.1242/dmm.019000.</cite> [<a href="https://doi.org/10.1242/dmm.019000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4582097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26183215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Disease%20Model%20&amp;%20Mechanisms%20%5BInternet%5D&amp;title=HIF1%CE%B1%20deficiency%20reduces%20colon%20inflammation%20in%20a%20mouse%20model%20of%20proximal%20colon%20cancer&amp;publication_year=2015&amp;pmid=26183215&amp;doi=10.1242/dmm.019000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0069">
<span class="label">69.</span><cite>Azimi I. The interplay between HIF-1 and calcium signalling in cancer. Int J Biochem Cell Biol [Internet]. 2018.  [cited 2025 May
12];97:73–77. doi: 10.1016/j.biocel.2018.02.001.
</cite> [<a href="https://doi.org/10.1016/j.biocel.2018.02.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29407528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Biochem%20Cell%20Biol%20%5BInternet%5D&amp;title=The%20interplay%20between%20HIF-1%20and%20calcium%20signalling%20in%20cancer&amp;volume=97&amp;publication_year=2018&amp;pages=73-77&amp;pmid=29407528&amp;doi=10.1016/j.biocel.2018.02.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0070">
<span class="label">70.</span><cite>Courtnay R, Ngo DC, Malik N, Ververis K, Tortorella SM, Karagiannis TC. Cancer metabolism and the Warburg effect: the role of HIF-1 and PI3K. Mol Biol Rep [Internet]. 2015.  [cited 2025 May
12];42(4):841–851. doi: 10.1007/s11033-015-3858-x.
</cite> [<a href="https://doi.org/10.1007/s11033-015-3858-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25689954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Rep%20%5BInternet%5D&amp;title=Cancer%20metabolism%20and%20the%20Warburg%20effect:%20the%20role%20of%20HIF-1%20and%20PI3K&amp;volume=42&amp;issue=4&amp;publication_year=2015&amp;pages=841-851&amp;pmid=25689954&amp;doi=10.1007/s11033-015-3858-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0071">
<span class="label">71.</span><cite>Dang DT, Chen F, Gardner LB, Cummins JM, Rago C, Bunz F, Kantsevoy SV, Dang LH. Hypoxia-inducible factor-1α promotes nonhypoxia-mediated proliferation in colon cancer cells and xenografts. Cancer Res [Internet]. 2006;[cited 2025 May
12];66(3):1684–1693. doi: 10.1158/0008-5472.CAN-05-2887.
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-05-2887" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16452228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res%20%5BInternet%5D&amp;title=Hypoxia-inducible%20factor-1%CE%B1%20promotes%20nonhypoxia-mediated%20proliferation%20in%20colon%20cancer%20cells%20and%20xenografts&amp;volume=66&amp;issue=3&amp;publication_year=2006&amp;pages=1684-1693&amp;pmid=16452228&amp;doi=10.1158/0008-5472.CAN-05-2887&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0072">
<span class="label">72.</span><cite>Rohwer N, Jumpertz S, Erdem M, Egners A, Warzecha KT, Fragoulis A, Kühl AA, Kramann R, Neuss S, Rudolph I, et al. Non-canonical HIF-1 stabilization contributes to intestinal tumorigenesis. Oncogene [Internet]. 2019.  [cited 2025 May
12];38(28):5670–5685. doi: 10.1038/s41388-019-0816-4.
</cite> [<a href="https://doi.org/10.1038/s41388-019-0816-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31043706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene%20%5BInternet%5D&amp;title=Non-canonical%20HIF-1%20stabilization%20contributes%20to%20intestinal%20tumorigenesis&amp;volume=38&amp;issue=28&amp;publication_year=2019&amp;pages=5670-5685&amp;pmid=31043706&amp;doi=10.1038/s41388-019-0816-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0073">
<span class="label">73.</span><cite>Fachi JL, Felipe JDS, Pral LP, Da Silva BK, Corrêa RO, De Andrade MCP, Da Fonseca DM, Basso PJ, Câmara NOS, De Sales E, et al. Butyrate protects mice from clostridium difficile-induced colitis through an HIF-1-Dependent mechanism. Cell Rep [Internet]. 2019.  [cited 2025 May
12];27(3):750–761.e7. <a href="https://linkinghub.elsevier.com/retrieve/pii/S2211124719303845" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://linkinghub.elsevier.com/retrieve/pii/S2211124719303845</a>.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.03.054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30995474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep%20%5BInternet%5D&amp;title=Butyrate%20protects%20mice%20from%20clostridium%20difficile-induced%20colitis%20through%20an%20HIF-1-Dependent%20mechanism&amp;volume=27&amp;issue=3&amp;publication_year=2019&amp;pages=750-761.e7&amp;pmid=30995474&amp;doi=10.1016/j.celrep.2019.03.054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0074">
<span class="label">74.</span><cite>Chen G, Ran X, Li B, Li Y, He D, Huang B, Fu S, Liu J, Wang W. Sodium butyrate inhibits inflammation and maintains epithelium barrier integrity in a TNBS-induced inflammatory bowel disease mice model. eBioMedicine [Internet]. 2018.  [cited 2025 May
12];30:317–325. doi: 10.1016/j.ebiom.2018.03.030.
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2018.03.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5952406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29627390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eBioMedicine%20%5BInternet%5D&amp;title=Sodium%20butyrate%20inhibits%20inflammation%20and%20maintains%20epithelium%20barrier%20integrity%20in%20a%20TNBS-induced%20inflammatory%20bowel%20disease%20mice%20model&amp;volume=30&amp;publication_year=2018&amp;pages=317-325&amp;pmid=29627390&amp;doi=10.1016/j.ebiom.2018.03.030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0075">
<span class="label">75.</span><cite>Albenberg L, Esipova TV, Judge CP, Bittinger K, Chen J, Laughlin A, Grunberg S, Baldassano RN, Lewis JD, Li H, et al. Correlation between intraluminal oxygen gradient and radial partitioning of intestinal microbiota. Gastroenterol [Internet]. 2014.  [cited 2025 May
12];147(5):1055–1063.e8. doi: 10.1053/j.gastro.2014.07.020.</cite> [<a href="https://doi.org/10.1053/j.gastro.2014.07.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4252572/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25046162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterol%20%5BInternet%5D&amp;title=Correlation%20between%20intraluminal%20oxygen%20gradient%20and%20radial%20partitioning%20of%20intestinal%20microbiota&amp;volume=147&amp;issue=5&amp;publication_year=2014&amp;pages=1055-1063.e8&amp;pmid=25046162&amp;doi=10.1053/j.gastro.2014.07.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0076">
<span class="label">76.</span><cite>Broman E, Izabel-Shen D, Rodríguez-Gijón A, Bonaglia S, Garcia SL, Nascimento FJA. Microbial functional genes are driven by gradients in sediment stoichiometry, oxygen, and salinity across the Baltic benthic ecosystem. Microbiome [Internet]. 2022.  [cited 2025 May
12];10(1):126. doi: 10.1186/s40168-022-01321-z.
</cite> [<a href="https://doi.org/10.1186/s40168-022-01321-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9377124/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35965333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiome%20%5BInternet%5D&amp;title=Microbial%20functional%20genes%20are%20driven%20by%20gradients%20in%20sediment%20stoichiometry,%20oxygen,%20and%20salinity%20across%20the%20Baltic%20benthic%20ecosystem&amp;volume=10&amp;issue=1&amp;publication_year=2022&amp;pages=126&amp;pmid=35965333&amp;doi=10.1186/s40168-022-01321-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0077">
<span class="label">77.</span><cite>Lei X, Xu Z, Huang Y, Huang L, Lang J, Qu M, Liu D. Regulation of mitochondrial quality control of intestinal stem cells in homeostasis and diseases. Antioxid &amp; Redox Signaling [Internet]. 2025.  [cited 2025 May
12];42(10–12):494–511. doi: 10.1089/ars.2023.0489.</cite> [<a href="https://doi.org/10.1089/ars.2023.0489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39225500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antioxid%20&amp;%20Redox%20Signaling%20%5BInternet%5D&amp;title=Regulation%20of%20mitochondrial%20quality%20control%20of%20intestinal%20stem%20cells%20in%20homeostasis%20and%20diseases&amp;volume=42&amp;issue=10%E2%80%9312&amp;publication_year=2025&amp;pages=494-511&amp;pmid=39225500&amp;doi=10.1089/ars.2023.0489&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0078">
<span class="label">78.</span><cite>Lampert MA, Gustafsson ÅB. Mitochondria and autophagy in adult stem cells: proliferate or differentiate. J Muscle Res Cell Motil [Internet]. 2020.  [cited 2025 May
12];41(4):355–362. doi: 10.1007/s10974-019-09542-w.
</cite> [<a href="https://doi.org/10.1007/s10974-019-09542-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6962576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31313217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Muscle%20Res%20Cell%20Motil%20%5BInternet%5D&amp;title=Mitochondria%20and%20autophagy%20in%20adult%20stem%20cells:%20proliferate%20or%20differentiate&amp;volume=41&amp;issue=4&amp;publication_year=2020&amp;pages=355-362&amp;pmid=31313217&amp;doi=10.1007/s10974-019-09542-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0079">
<span class="label">79.</span><cite>Zhang F, Pirooznia M, Xu H, Fox T. Mitochondria regulate intestinal stem cell proliferation and epithelial homeostasis through FOXO. MBoC [Internet]. 2020.  [cited 2025 May
12];31(14):1538–1549. doi: 10.1091/mbc.E19-10-0560.
</cite> [<a href="https://doi.org/10.1091/mbc.E19-10-0560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7359575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32374658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MBoC%20%5BInternet%5D&amp;title=Mitochondria%20regulate%20intestinal%20stem%20cell%20proliferation%20and%20epithelial%20homeostasis%20through%20FOXO&amp;volume=31&amp;issue=14&amp;publication_year=2020&amp;pages=1538-1549&amp;pmid=32374658&amp;doi=10.1091/mbc.E19-10-0560&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0080">
<span class="label">80.</span><cite>Dai Z, Li D, Du X, Ge Y, Hursh DA, Bi X, Lu B. Drosophila Caliban preserves intestinal homeostasis and lifespan through regulating mitochondrial dynamics and redox state in enterocytes. PLOS Genet [Internet]. 2020.  [cited 2025 May
12];16(10):e1009140. doi: 10.1371/journal.pgen.1009140.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1009140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7591072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33057338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genet%20%5BInternet%5D&amp;title=Drosophila%20Caliban%20preserves%20intestinal%20homeostasis%20and%20lifespan%20through%20regulating%20mitochondrial%20dynamics%20and%20redox%20state%20in%20enterocytes&amp;volume=16&amp;issue=10&amp;publication_year=2020&amp;pages=e1009140&amp;pmid=33057338&amp;doi=10.1371/journal.pgen.1009140&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0081">
<span class="label">81.</span><cite>Rodríguez-Colman MJ, Schewe M, Meerlo M, Stigter E, Gerrits J, Pras-Raves M, Sacchetti A, Hornsveld M, Oost KC, Snippert HJ, et al. Interplay between metabolic identities in the intestinal crypt supports stem cell function. Nat [Internet]. 2017.  [cited 2025 May
12];543(7645):424–427. doi: 10.1038/nature21673.</cite> [<a href="https://doi.org/10.1038/nature21673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28273069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20%5BInternet%5D&amp;title=Interplay%20between%20metabolic%20identities%20in%20the%20intestinal%20crypt%20supports%20stem%20cell%20function&amp;volume=543&amp;issue=7645&amp;publication_year=2017&amp;pages=424-427&amp;pmid=28273069&amp;doi=10.1038/nature21673&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0082">
<span class="label">82.</span><cite>Reedy AR, Luo L, Neish AS, Jones RM. Commensal microbiota-induced redox signaling activates proliferative signals in the intestinal stem cell microenvironment. Devel [Internet]. 2019.  [cited 2025 May
12];146(3):dev171520. doi: 10.1242/dev.171520.</cite> [<a href="https://doi.org/10.1242/dev.171520" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6382009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30658986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Devel%20%5BInternet%5D&amp;title=Commensal%20microbiota-induced%20redox%20signaling%20activates%20proliferative%20signals%20in%20the%20intestinal%20stem%20cell%20microenvironment&amp;volume=146&amp;issue=3&amp;publication_year=2019&amp;pages=dev171520&amp;pmid=30658986&amp;doi=10.1242/dev.171520&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0083">
<span class="label">83.</span><cite>Iatsenko I, Boquete J-P, Lemaitre B. Microbiota-derived lactate activates production of reactive oxygen species by the intestinal NADPH oxidase nox and shortens drosophila lifespan. Immun [Internet]. 2018.  [cited 2025 May
12];49(5):929–942.e5. doi: 10.1016/j.immuni.2018.09.017.</cite> [<a href="https://doi.org/10.1016/j.immuni.2018.09.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30446385/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immun%20%5BInternet%5D&amp;title=Microbiota-derived%20lactate%20activates%20production%20of%20reactive%20oxygen%20species%20by%20the%20intestinal%20NADPH%20oxidase%20nox%20and%20shortens%20drosophila%20lifespan&amp;volume=49&amp;issue=5&amp;publication_year=2018&amp;pages=929-942.e5&amp;pmid=30446385&amp;doi=10.1016/j.immuni.2018.09.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0084">
<span class="label">84.</span><cite>Van Der Post S, Birchenough GMH, Held JM. NOX1-dependent redox signaling potentiates colonic stem cell proliferation to adapt to the intestinal microbiota by linking EGFR and TLR activation. Cell Rep [Internet]. 2021.  [cited 2025 May
12];35(1):108949. doi: 10.1016/j.celrep.2021.108949.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.108949" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10327654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33826887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep%20%5BInternet%5D&amp;title=NOX1-dependent%20redox%20signaling%20potentiates%20colonic%20stem%20cell%20proliferation%20to%20adapt%20to%20the%20intestinal%20microbiota%20by%20linking%20EGFR%20and%20TLR%20activation&amp;volume=35&amp;issue=1&amp;publication_year=2021&amp;pages=108949&amp;pmid=33826887&amp;doi=10.1016/j.celrep.2021.108949&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit0085">
<span class="label">85.</span><cite>Perez-Riverol Y, Bandla C, Kundu DJ, Kamatchinathan S, Bai J, Hewapathirana S, John NS, Prakash A, Walzer M, Wang S, et al. The PRIDE database at 20 years: 2025 update. Nucleic Acids Res [Internet]. 2025.  [cited 2025 May
12];53(D1):D543–53. <a href="https://academic.oup.com/nar/article/53/D1/D543/7874848" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://academic.oup.com/nar/article/53/D1/D543/7874848</a>.
</cite> [<a href="https://doi.org/10.1093/nar/gkae1011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11701690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39494541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res%20%5BInternet%5D&amp;title=The%20PRIDE%20database%20at%2020%20years:%202025%20update&amp;volume=53&amp;issue=D1&amp;publication_year=2025&amp;pages=D543-53&amp;pmid=39494541&amp;doi=10.1093/nar/gkae1011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Supplemental Material</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12369635/bin/KGMI_A_2543123_SM5400.zip" data-ga-action="click_feat_suppl" class="usa-link">KGMI_A_2543123_SM5400.zip</a><sup> (22.9MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<ol class="list" style="list-style-type:decimal">
<li><p>- Single-cell RNA sequencing data is deposited at NCBI BioProject: PRJNA856646</p></li>
<li><p>- 16S sequencing data is deposited at NCBI BioProject: PRJNA1188097.</p></li>
<li><p>- The mass spectrometry proteomics data is deposited to the ProteomeXchange Consortium (<a href="http://proteomecentral.proteomexchange.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://proteomecentral.proteomexchange.org</a>.) via the PRIDE partner repository<sup><a href="#cit0085" class="usa-link" aria-describedby="cit0085">85</a></sup> with the dataset identifier PXD059757. Project DOI: 10.6019/PXD059757</p></li>
</ol></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Gut Microbes are provided here courtesy of <strong>Taylor & Francis</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1080/19490976.2025.2543123"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/KGMI_17_2543123.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12369635/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12369635/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369635%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369635/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12369635/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12369635/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40827535/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12369635/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40827535/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12369635/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12369635/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="w1QEQMtbyP0cnauhmRSY8hiIvIotsSD6AvuGex7mqrPiMNWhTIJv6hTw56AdOEWU">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
