
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Increasing pathogenic germline variant diagnosis rates in precision medicine: current best practices and future opportunities - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5315468AEF66530515460012F535ED.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="humgenomics">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374290/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Human Genomics">
<meta name="citation_title" content="Increasing pathogenic germline variant diagnosis rates in precision medicine: current best practices and future opportunities">
<meta name="citation_author" content="Sonam Dukda">
<meta name="citation_author_institution" content="Centre for Tropical Bioinformatics and Molecular Biology, College Science and Engineering, James Cook University, Cairns, QLD Australia">
<meta name="citation_author" content="Manoharan Kumar">
<meta name="citation_author_institution" content="Centre for Tropical Bioinformatics and Molecular Biology, College Science and Engineering, James Cook University, Cairns, QLD Australia">
<meta name="citation_author" content="Andrew Calcino">
<meta name="citation_author_institution" content="Centre for Tropical Bioinformatics and Molecular Biology, College Science and Engineering, James Cook University, Cairns, QLD Australia">
<meta name="citation_author" content="Ulf Schmitz">
<meta name="citation_author_institution" content="Centre for Tropical Bioinformatics and Molecular Biology, College Science and Engineering, James Cook University, Cairns, QLD Australia">
<meta name="citation_author_institution" content="Centenary Institute, The University of Sydney, Camperdown, Australia">
<meta name="citation_author_institution" content="Computational Biomedicine Lab, James Cook University, Townsville, Australia">
<meta name="citation_author" content="Matt A Field">
<meta name="citation_author_institution" content="Centre for Tropical Bioinformatics and Molecular Biology, College Science and Engineering, James Cook University, Cairns, QLD Australia">
<meta name="citation_author_institution" content="Immunogenomics Lab, Garvan Institute of Medical Research, Darlinghurst, NSW Australia">
<meta name="citation_author_institution" content="Menzies School of Health Research, Charles Darwin University, Darwin, Australia">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="19">
<meta name="citation_firstpage" content="97">
<meta name="citation_doi" content="10.1186/s40246-025-00811-z">
<meta name="citation_pmid" content="40847372">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374290/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374290/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374290/pdf/40246_2025_Article_811.pdf">
<meta name="description" content="The accurate diagnosis of pathogenic variants is essential for effective clinical decision making within precision medicine programs. Despite significant advances in both the quality and quantity of molecular patient data, diagnostic rates remain ...">
<meta name="og:title" content="Increasing pathogenic germline variant diagnosis rates in precision medicine: current best practices and future opportunities">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The accurate diagnosis of pathogenic variants is essential for effective clinical decision making within precision medicine programs. Despite significant advances in both the quality and quantity of molecular patient data, diagnostic rates remain ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374290/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12374290">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s40246-025-00811-z"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/40246_2025_Article_811.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374290%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12374290/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12374290/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374290/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-humgenomics.png" alt="Human Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Human Genomics" title="Link to Human Genomics" shape="default" href="https://humgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Hum Genomics</button></div>. 2025 Aug 22;19:97. doi: <a href="https://doi.org/10.1186/s40246-025-00811-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s40246-025-00811-z</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Hum%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hum%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Hum%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Hum%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Increasing pathogenic germline variant diagnosis rates in precision medicine: current best practices and future opportunities</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dukda%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sonam Dukda</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sonam Dukda</span></h3>
<div class="p">
<sup>1</sup>Centre for Tropical Bioinformatics and Molecular Biology, College Science and Engineering, James Cook University, Cairns, QLD Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dukda%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sonam Dukda</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Manoharan Kumar</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Manoharan Kumar</span></h3>
<div class="p">
<sup>1</sup>Centre for Tropical Bioinformatics and Molecular Biology, College Science and Engineering, James Cook University, Cairns, QLD Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Manoharan Kumar</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calcino%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Andrew Calcino</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Andrew Calcino</span></h3>
<div class="p">
<sup>1</sup>Centre for Tropical Bioinformatics and Molecular Biology, College Science and Engineering, James Cook University, Cairns, QLD Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calcino%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew Calcino</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schmitz%20U%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ulf Schmitz</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ulf Schmitz</span></h3>
<div class="p">
<sup>1</sup>Centre for Tropical Bioinformatics and Molecular Biology, College Science and Engineering, James Cook University, Cairns, QLD Australia </div>
<div class="p">
<sup>2</sup>Centenary Institute, The University of Sydney, Camperdown, Australia </div>
<div class="p">
<sup>3</sup>Computational Biomedicine Lab, James Cook University, Townsville, Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schmitz%20U%22%5BAuthor%5D" class="usa-link"><span class="name western">Ulf Schmitz</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Field%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Matt A Field</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Matt A Field</span></h3>
<div class="p">
<sup>1</sup>Centre for Tropical Bioinformatics and Molecular Biology, College Science and Engineering, James Cook University, Cairns, QLD Australia </div>
<div class="p">
<sup>4</sup>Immunogenomics Lab, Garvan Institute of Medical Research, Darlinghurst, NSW Australia </div>
<div class="p">
<sup>5</sup>Menzies School of Health Research, Charles Darwin University, Darwin, Australia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Field%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Matt A Field</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Centre for Tropical Bioinformatics and Molecular Biology, College Science and Engineering, James Cook University, Cairns, QLD Australia </div>
<div id="Aff2">
<sup>2</sup>Centenary Institute, The University of Sydney, Camperdown, Australia </div>
<div id="Aff3">
<sup>3</sup>Computational Biomedicine Lab, James Cook University, Townsville, Australia </div>
<div id="Aff4">
<sup>4</sup>Immunogenomics Lab, Garvan Institute of Medical Research, Darlinghurst, NSW Australia </div>
<div id="Aff5">
<sup>5</sup>Menzies School of Health Research, Charles Darwin University, Darwin, Australia </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jul 7; Accepted 2025 Aug 6; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12374290  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847372/" class="usa-link">40847372</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The accurate diagnosis of pathogenic variants is essential for effective clinical decision making within precision medicine programs. Despite significant advances in both the quality and quantity of molecular patient data, diagnostic rates remain suboptimal for many inherited diseases. As such, prioritisation and identification of pathogenic disease-causing variants remains a complex and rapidly evolving field. This review explores the latest technological and computational options being used to increase genetic diagnosis rates in precision medicine programs.</p>
<p id="Par2">While interpreting genetic variation via standards such as ACMG guidelines is increasingly being recognized as a gold standard approach, the underlying datasets and algorithms recommended are often slow to incorporate additional data types and methodologies. For example, new technological developments, particularly in single-cell and long-read sequencing, offer great opportunity to improve genetic diagnosis rates, however, how to best interpret and integrate increasingly complex multi-omics patient data remains unclear. Further, advances in artificial intelligence and machine learning applications in biomedical research offer enormous potential, however they require careful consideration and benchmarking given the clinical nature of the data. This review covers the current state of the art in available sequencing technologies, software methodologies for variant annotation/prioritisation, pedigree-based strategies and the potential role of machine learning applications. We describe a key set of design principles required for a modern multi-omic precision medicine framework that is robust, modular, secure, flexible, and scalable. Creating a next generation framework will ensure we realise the full potential of precision medicine into the future.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Identifying targetable disease-causing genetic variants lies at the heart of advancing precision medicine, improving clinical diagnostics, and enhancing our understanding of genetic contributions to diseases [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Precision medicine entails a healthcare delivery model that relies extensively on patient specific data points to guide the development of customised therapies [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. One key driver of such initiatives is the ability to pinpoint pathogenic variants that greatly improves our mechanistic understanding of the disease process [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. This progress has been largely enabled by the increasing affordability and accessibility of high-quality sequence data. Despite this progress, conclusively linking genetic variants with disease remains resource-intensive and time-consuming [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. The most significant challenge remains differentiating the key genetic drivers from the large volumes of background genetic variation naturally present in every person. Further challenges exist with variant detection, annotation and prioritisation methods with the lack of global standards resulting in an over reliance on variable bespoke in-house solutions [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Groups such as the American College of Medical Genetics and Genomics (ACMG) are addressing this by providing guidelines on variant detection and interpretation; for example they propose guidelines to establish consistent cataloguing of genetic variants, classifying variants into five categories based on the strength of the evidence for disease causation. Despite these standards, many variants remain annotated as variants of uncertain significance (VUS), lacking sufficient functional assay data required for reliable classification [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Additionally, recurrent false positive variants can be included in clinical databases [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>].</p>
<p id="Par4">There is also an increasing recognition of the role of large and repetitive genetic variants in driving disease, however these remain challenging to detect with current short-read sequencing technologies and are better suited to more expensive long-read sequencing approaches [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. Additionally, non-coding variants are being recognised in driving disease including intronic variants which create cryptic splice sites as well as variants modifying regulatory elements such as enhancers and promotors [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Another challenge specific to complex disease is the often-little understood interactions between genetic and environmental factors. Factors such as lifestyle and surrounding environment can contribute significantly to disease development and progression; however, our understanding of these processes is limited [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>].</p>
<p id="Par5">Segregation analysis of germline variants within families plays a critical role in precision medicine by enabling the accurate interpretation of genetic findings in the context of inherited disease risk [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. By studying how a specific variant co-segregates with a disease phenotype across multiple family members, clinicians and researchers can distinguish pathogenic mutations from benign polymorphisms, thus improving diagnostic accuracy [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. This analysis not only helps validate the clinical relevance of a variant but also informs risk assessment, surveillance strategies, and therapeutic decisions for both affected individuals and at-risk relatives. In precision medicine, where individualized care hinges on the precise understanding of genetic contributions to disease, segregation analysis remains a cornerstone for translating genomic data into meaningful clinical outcomes. Overall family data provides evidence needed to reclassify variants, either supporting pathogenicity through clear segregation or suggesting benign status through inconsistent patterns. This improved classification directly impacts clinical decision-making, from diagnosis and treatment selection to family planning [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Family analysis enables confident clinical recommendations, reduces uncertainty in genetic counselling, and identifies at-risk family members who would benefit from testing or enhanced surveillance, exemplifying the promise of precision medicine.</p>
<p id="Par6">Despite steady progress, genetic diagnostic discovery rates for many complex diseases using traditional approaches remain low and capturing “missing heritability” requires multi-pronged approaches. These include a variety of sequencing-based (e.g. long-read, single-cell sequencing) and computer-based approaches (e.g. machine learning, multi-omic workflows). Robust frameworks capable of integrating huge volumes of complex patient data, genetic variant and annotation information are urgently needed [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>].</p></section><section id="Sec2"><h2 class="pmc_sec_title">Sequencing technology</h2>
<p id="Par7">A diverse selection of sequencing technologies are now available for identifying genetic variants [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. DNA-based solutions include whole genome sequencing (WGS), whole exome sequencing (WES), and targeted gene panels, while genetic variants can also be detected in cDNA used for RNA sequencing (RNA-Seq) [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Newer approaches include long-read sequencing, suitable for identifying more complex, larger genetic variants [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], and single-cell sequencing to identify rare or cell-type-specific variants [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>].</p>
<section id="Sec3"><h3 class="pmc_sec_title">Current DNA-based sequencing options</h3>
<p id="Par8">Targeted gene panel sequencing is appropriate when driver genes are largely known for a disease. In such cases, gene panels can obtain high diagnostic rates and the simple deployment, interpretation, and lower costs, offers an attractive alternative to WES/WGS [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Targeted panels are typically sequenced at a high depth to identify rare variants, a process which can be combined with unique molecular identifier (UMI)-based approaches to further increase resolution [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. There are limitations of this approach however such as their inability to identify novel variants and large genetic variants [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>].</p>
<p id="Par9">WES is effective in finding both known [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] and novel driver mutations [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] in coding regions of the genome. WES employs a targeted approach via a capture array containing most known coding exons, thus covering the majority of the coding regions [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Despite only accounting for ~ 1% of the genome, an estimated 85% of mutations responsible for diseases are thought to occur within exons [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. WES is attractive with regard to price and sequence depth relative to WGS [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>].</p>
<p id="Par10">WGS is an unbiased method that provides sequence data across the entire genome [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. WGS is increasingly becoming the first choice for patient sequencing due to advantages including the ability to detect small and large genetic variants as well as achieving relatively even sequence coverage [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. The diagnostic superiority of WGS to chromosomal microarray (CMA), karyotyping, targeted sequencing assays and WES [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>–<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] has been demonstrated. Accordingly, precision medicine programs are employing WGS as the first option resulting in the development of increasingly standardised methodologies [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>].</p></section><section id="Sec4"><h3 class="pmc_sec_title">Current RNA-based sequencing options</h3>
<p id="Par11">Bulk RNA-seq is a high-throughput method used to examine the complete set of RNA transcripts within a biological sample [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Bulk RNA-Seq typically obtains sequence data from a mixed heterogenous population of cells in contrast to tagged individual cells as is done in single-cell RNA-Seq (scRNA-Seq) [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. The clinical utility has been demonstrated largely for the ability to identify dysregulated genes that warrant further investigation within the genome [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. Additionally, RNA-Seq allows the identification of aberrant splicing events such as retained introns or skipped exons and gene fusions.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Single-cell sequencing</h3>
<p id="Par12">In contrast to traditional bulk sequencing methods, single-cell technologies incorporate cell-specific barcodes to obtain per-cell sequence information for thousands of cells simultaneously. To date, most single-cell platforms utilise RNA as input (scRNA-Seq) however a growing number of platforms offer single-cell DNA sequencing (scDNA-Seq).</p>
<p id="Par13">Single-cell RNA sequencing is an advanced technology able to evaluate transcriptional similarities and variances within a population of cells, revealing cell-type-specific levels of heterogeneity previously undetectable by bulk sequencing methods [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>–<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Nonetheless, scRNA-Seq remains technically challenging with limitations including generation of doublets and dead cells, lower sequencing depth per cell, data sparsity, high input cell requirements, and high cost. Despite these challenges, scRNA-Seq offers an unprecedented opportunity to track disease progression within heterogenous cell populations.</p>
<p id="Par14">Single-cell DNA sequencing allows the detection of per-cell or cell-type-specific rare genetic variants from mixed heterogenous input samples [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. Many of the same limitations are shared with scRNA-Seq, however variant detection is feasible for targeted gene panels using technologies such as Mission Bio’s Tapestri platform. Additionally, significant amplification is typically required [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>], a process known to introduce errors and uneven coverage resulting in challenges in downstream data analysis [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>].</p></section><section id="Sec6"><h3 class="pmc_sec_title">Long-read sequencing</h3>
<p id="Par15">Third generation single molecule long-read sequencing overcomes many of the limitations of short-read technologies [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. Initially plagued by high error rates, continual improvements are producing progressively longer and higher quality reads, with lengths of up to 2 Megabase pairs now possible [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>, <a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. The third-generation sequencing market is primarily dominated by two technologies: (i) Pacific Biosciences and (ii) Oxford Nanopore Technologies (ONT) [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>], both of which offer DNA or RNA based sequencing. Long-read DNA sequencing is critical if driver variants are repetitive or complex in nature (e.g. long tandem repeats, copy number variants) or occur in repetitive gene families, GC-rich regions or pseudogenes. Long-read RNA sequencing captures full-length isoforms and can identify novel transcripts, skipped exons, retained introns and gene fusion products [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>].</p></section></section><section id="Sec7"><h2 class="pmc_sec_title">Prioritisation strategies</h2>
<p id="Par16">All sequencing approaches generate large numbers of genetic variants, the majority of which are not relevant for the underlying disease. Reducing the genetic search space for causal variants can be done through a combination of careful patient selection, variant annotation (at the level of variant, gene and gene network), and software development and optimisation (Fig. <a href="#Fig1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374290_40246_2025_811_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bc99/12374290/f9e75df7a6a6/40246_2025_811_Fig1_HTML.jpg" loading="lazy" id="d33e437" height="593" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Variant prioritisation strategies</p></figcaption></figure><section id="Sec8"><h3 class="pmc_sec_title">Sample selection strategies</h3>
<p id="Par18">Sample selection strategies are critical to increasing the likelihood of identifying disease causing variants. Common strategies include grouping unrelated individuals by phenotype, selecting patients with early onset and/or extreme phenotype and pedigree sequencing for families.</p>
<p id="Par19">For single or unrelated individuals, strategies include sequencing samples with early onset or extreme phenotypes as well grouping patients with similar phenotypes that potentially share an underlying genetic cause [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. Choosing unrelated individuals with a shared well-characterised phenotype requires annotation with standardised human phenotype ontology (HPO) terms [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Further defining inclusion and exclusion criteria based on clinical features - disease progression, or other relevant parameters – allows the generation of a study cohort that is relatively homogenous and well suited for detecting shared driver variants (or pathways) linked to the observed phenotype [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. The significance of careful patient selection is highlighted in numerous studies as an effective way to increase diagnosis rates [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>, <a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>].</p></section><section id="Sec9"><h3 class="pmc_sec_title">Pedigree-based strategies</h3>
<p id="Par20">For related individuals, utilization of pedigree sequencing is an extremely effective strategy for reducing the genomic search space for causal variants. Pedigree sequencing is particularly useful for the identification of rare familial variants which segregate with the phenotype of interest [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>, <a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. This approach yields additional information including inheritance modes and can track the segregation of variants within families, however custom software is required [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Sequencing of a proband child with healthy parents is very successful for rare, early onset diseases by focusing on a small number <em>de novo</em> variants. Similarly, consanguineous pedigree sequencing reduces the search space to homozygous variants [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>].</p>
<p id="Par21">Segregation analysis plays a vital role in interpreting and prioritising genetic variants by examining how a variant is inherited within a family and whether it consistently appears in affected individuals while being absent in those unaffected. Such co-segregation patterns strengthens the case for pathogenicity, particularly when it is aligned with the expected pattern of inheritance [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>, <a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. In the line with ACMG/AMP guidelines, segregation data can count towards variant classification as ‘supporting evidence (PP1)’, with increasing weight assigned as more segregation evidence is observed [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. Recognition of segregating <em>de novo</em> or very rare variants in dominant conditions further strengthens the case for pathogenicity (PS2) [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. In contrast, variants that do not segregate with disease phenotype in affected family members may indicate benign status, reduced penetrance, or the presence of phenocopies—individuals who exhibit symptoms due to non-genetic factors or different underlying genetic causes [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. Such scenarios may point to potential locus heterogeneity, where similar phenotypes arise from mutations in different genes [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>, <a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. The identification of such cases is vital for accurate genetic diagnosis. When causal variants are present in asymptomatic carriers, we can estimate penetrance and variability in disease severity thus enabling personalized monitoring strategies and more accurate prognoses for at-risk relatives [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>].</p></section><section id="Sec10"><h3 class="pmc_sec_title">Annotation-based strategies – variant level</h3>
<p id="Par22">Variants are first annotated based on the exact genomic coordinate and the observed nucleotide change. This includes assessing impact on transcripts, estimating impact on protein function, comparing to population variant levels and overlapping with clinical variant databases.</p>
<p id="Par23">Variants are typically overlapped to a transcript model and stratified according to their impact using popular tools such as SnpEff, VEP or ANNOVAR [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>–<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. High level genomic overlaps are first considered (e.g. inter-genic, intronic, exonic, etc.) and further refined by transcript effect when appropriate (e.g. exonic SNVs classified as synonymous or non-synonymous, exonic indels classified as frameshift or non-frameshift). Next, variants are run through functional inference tools to predict potential functional consequence of genetic variants. These tools utilise information including evolutionary conservation patterns, protein annotations and structural information to make predictions about the functional impact of genetic variants. Many tools exist including PolyPhen2 and SIFT [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>, <a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>] for missense mutations and CADD for all variant types [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. While these tools are useful, their results should be combined with other types of evidence as they are known to suffer from high false positive rates, particularly for variant subclasses such as pharmacogenetic variants [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>, <a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>].</p>
<p id="Par24">Until recently, a significant limitation in the annotation of protein features was the lack of comprehensive protein structures for all human proteins. AlphaFold successfully generated extremely accurate protein structures for all human proteins offering new opportunities for variant prioritisation [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. The accuracy of functional impact tools can be increased with AlphaFold integration by considering the structural context of genetic variants [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. AlphaMissense was subsequently developed based on AlphaFold2 predictions and fine-tuned using human and primate variant frequency databases [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. Over time AlphaMissense (and subsequent developments) are likely to play a big role in improving functional impact metrics [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]. Overall there are many options for variant functional annotation (e.g. ENSEMBL Variant Effect Predictor (VEP), ANNOVAR, SnpEff) and functional inference prediction (Polyphen2, SIFT, CADD, AlphaMissense). Table <a href="#Tab1" class="usa-link">1</a> summarises several options including their relative strengths and weaknesses.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Variant annotation tools</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Software (Class)</th>
<th align="left" colspan="1" rowspan="1">Strengths</th>
<th align="left" colspan="1" rowspan="1">Weaknesses</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">VEP (Functional annotation)</td>
<td align="left" colspan="1" rowspan="1">Open source; supports web, CLI, and API usage; annotates coding/non-coding variants via Ensembl &amp; RefSeq models; integrates allele frequencies, pathogenicity scores &amp; phenotype databases; customizable output &amp; filtering options; regularly updated.</td>
<td align="left" colspan="1" rowspan="1">Complex output with multiple transcript annotations; requires filtering to simplify; non-coding annotations require additional configuration; plugin setup can be complex; slower performance on large datasets without caching.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ANNOVAR (Functional annotation)</td>
<td align="left" colspan="1" rowspan="1">Broad annotation support (e.g., RefSeq, gnomAD, CADD); flexible framework; efficient variant filtering; fast runtime.</td>
<td align="left" colspan="1" rowspan="1">Lower HGVS accuracy (93.3% concordance); limited support for complex/structural variants; requires manual database updates; not optimised for polygenic traits; collapses transcript isoforms.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SnpEff (Functional annotation)</td>
<td align="left" colspan="1" rowspan="1">Fast annotation for high-throughput pipelines; supports multiple genomes &amp; transcript models; Coding annotation accuracy (~ 89.8%) comparable to VEP.</td>
<td align="left" colspan="1" rowspan="1">Lower concordance for indels &amp; frameshifts (&lt; 75%); protein syntax often mismatches references;</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Polyphen2 (Functional inference prediction)</td>
<td align="left" colspan="1" rowspan="1">Predicts impact of missense mutations based on protein structure and evolutionary conservation.</td>
<td align="left" colspan="1" rowspan="1">High false-positive rate for certain variant classes, does not handle non-missense variants.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SIFT (Functional inference prediction)</td>
<td align="left" colspan="1" rowspan="1">Missense variants prediction based on sequence conservation.</td>
<td align="left" colspan="1" rowspan="1">Lower accuracy for less conserved regions of proteins.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CADD (Functional inference prediction)</td>
<td align="left" colspan="1" rowspan="1">Combines 60 + annotations into a single impact score; ranks deleteriousness across coding &amp; non-coding variants; Incorporates both simulated &amp; observed variants for robust training; Machine learning (SVM) framework improves generalizability and prioritization.</td>
<td align="left" colspan="1" rowspan="1">Lacks variant-type specificity (e.g. splicing vs. missense); less precise for rare or population-specific variants ; computationally demanding for non-pre-computed variants; may inflate scores for non-coding variants.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AlphaMissense (Functional inference prediction)</td>
<td align="left" colspan="1" rowspan="1">Combines structure &amp; conservation; high concordance with REVEL/CADD; effective for prioritizing pathogenic missense variants.</td>
<td align="left" colspan="1" rowspan="1">Still emerging; may inflate pathogenicity scores in some domains; needs further validation; accuracy varies across genes/proteins classes.</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par26">Population level databases of variant frequency are a powerful tool for understanding potential biological impact of genetic variation both globally and within matched ethnicities. Assigning variant allele frequencies enables the identification of rare or novel variants, a group enriched for pathogenic variants [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. Variant databases have grown progressively larger over time, however historically most variants were of European origin [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. Databases such as dbSNP and 1000 Genomes are two of the earliest databases and have proved invaluable for assigning variant frequencies [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>, <a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. The Genome Aggregation Database, commonly known as gnomAD, is a more recent entry providing a comprehensive and publicly accessible repository that aggregates genomic data from a diverse range of populations [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>]. It provides a wealth of information regarding the frequency and distribution of genetic variants across the human genome [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>]. Critically, population databases have recognised the importance of incorporating non-European individuals, however many groups remain underrepresented [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>].</p>
<p id="Par27">Disease focused variant databases are another critical tool for variant prioritisation. While population level databases serve to filter out large numbers of common variants, disease variant databases help identify candidate pathogenic variants. These databases contribute significantly to the interpretation of genetic variants in a clinical context, aiding in the identification of variants associated with diseases thus informing clinical decision-making. Some of the larger databases include ClinVar [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>], Human Gene Mutation Database (HGMD) [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>] and Leiden Open Variation Database (LOVD) [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>]. ClinVar is one of the largest clinical genomic databases, serving as a repository for variant data from clinical laboratories, clinicians, expert groups, patients, researchers, and other databases; it is a freely accessible, publicly curated database maintained by the National Centre for Biotechnology Information (NCBI) [<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>]. ClinVar ranks variants based on evidence such as functional assays providing a consistent scoring system across all potential clinically relevant variants. Similarly, the Human Gene Mutation Database (HGMD) aims to catalogue all mutations associated with inherited diseases [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>, <a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>]. The mutation data in HGMD are sourced solely from scientific literature and undergo rigorous manual curation using manual screening of journals and automated text mining [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>]. Finally, LOVD is a freely available web-based platform for the collection, display, and curation of DNA variants in locus-specific databases (LSDBs) [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>]. The design of LOVD system includes flexibility and seamless integration with other locus-specific LOVD instances, and introduced the “custom column” feature, enabling curators to tailor field setups according to their needs [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>].</p></section><section id="Sec11"><h3 class="pmc_sec_title">Annotation based strategies – gene level</h3>
<p id="Par28">A critical decision in any annotation workflow is the selection of the gene model. There are many efforts to standardise both gene sets and naming conventions however many challenges persist. Some of the popular models include ENSEMBL [<a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>], GENCODE [<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>], RefSeq [<a href="#CR97" class="usa-link" aria-describedby="CR97">97</a>], UCSC [<a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>] and UniProt [<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>]. Table <a href="#Tab2" class="usa-link">2</a> highlights the models’ strengths and weaknesses.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Gene model options</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Gene Model</th>
<th align="left" colspan="1" rowspan="1">Strengths</th>
<th align="left" colspan="1" rowspan="1">Weaknesses</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">ENSEMBL</td>
<td align="left" colspan="1" rowspan="1">Reliable cross-species gene annotation via combined manual &amp; automated methods; supports transcript diversity &amp; comparative genomics; regularly updated with VEP &amp; BioMart links; core genome browser support.</td>
<td align="left" colspan="1" rowspan="1">Annotation varies by species; complex transcripts could be inconsistently modelled; dependent on quality of assembly genome</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GENCODE</td>
<td align="left" colspan="1" rowspan="1">High-quality gene annotation for human/mouse; includes lncRNAs, pseudogenes, &amp; transcripts; integrates manual curation &amp; automation; captures transcript diversity; used within Ensembl, RefSeq, &amp; UCSC</td>
<td align="left" colspan="1" rowspan="1">Incomplete experimental support for all transcripts; redundancy &amp; unclear function in many lncRNAs / pseudogenes; manual curation limits scalability; inter-version differences may affect coordinate tracking.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RefSeq</td>
<td align="left" colspan="1" rowspan="1">
<p>High-quality, curated reference sequences for genomic, transcript, &amp; protein data; consistent across species annotations; integrate</p>
<p>manual curation with scalable automation;</p>
<p>widely adopted in tools like VEP, ANNOVAR, GATK.</p>
</td>
<td align="left" colspan="1" rowspan="1">RefSeq tends to be conservative and includes fewer transcript isoforms; RefSeq updates less frequently than other models; RefSeq is centrally managed by NCBI and no community input</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">UCSC</td>
<td align="left" colspan="1" rowspan="1">
<p>Curated gene models from mRNA/protein alignments enhances RNA-seq quantification; emphasizes</p>
<p>reliable transcripts &amp; simplifies isoform sets for reproducible gene counts; integrated with UCSC Genome Browser.</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>Limited transcript diversity &amp; isoforms &amp; non-coding RNAs.</p>
<p>Fewer splice junctions reduce RNA-seq accuracy;</p>
<p>biased toward canonical genes</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Uniprot</td>
<td align="left" colspan="1" rowspan="1">Provides detailed protein-level annotation including domains, function, and subcellular localization</td>
<td align="left" colspan="1" rowspan="1">Does not directly annotate variants or regulatory elements; protein-focused</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par30">In addition to variant-specific annotations, gene-level annotations are subsequently applied. In many cases a specific variant may not have been explicitly linked to disease pathogenesis, however the role the gene plays in driving the disease is well characterised. Gene level annotations largely consist of disease databases and gene ontology (GO) term annotation and enrichment analysis. Databases are typically curated repositories of gene-disease associations and help identify whether gene dysfunction has previously been implicated in similar diseases. There are many such databases, the largest being the Online Mendelian Inheritance in Man (OMIM) database [<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>].</p>
<p id="Par31">If, however, a gene has not been directly implicated in causing the disease, gene ontology (GO) is able to identify the function for each gene of interest potentially linking GO terms to the observed disease phenotype. GO annotation is standardised through large international efforts such the Gene Ontology Consortium (<a href="http://www.geneontology.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.geneontology.org</a>) which enable quick and easy GO annotation of structured domain-specific ontologies [<a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>]. A common application using GO terms is enrichment analysis, which aims to identify over-represented biological process, molecular function or cellular component shared by genes implicated in driving a polygenic disease.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Annotation based strategies – gene network level</h3>
<p id="Par32">Beyond gene-level annotations, the gene’s role within larger biological networks can be examined. To do this there are a variety of gene network analysis tools designed to analyse and interpret biological data, particularly gene expression data, in the context of biological networks. These tools aim to uncover relationships and interactions between genes to gain insights into the underlying biological processes. They contribute significantly to the understanding of the complex relationships within biological systems, helping researchers unravel the functional implications of gene interactions. Several popular tools in this space are STRING and Ingenuity Pathway Analysis (IPA). STRING (Search Tool for the Retrieval of Interacting Genes/Proteins) is an online platform created to identify protein-protein interactions (PPIs) and functional associations [<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>]. The STRING database, available at <a href="https://string-db.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://string-db.org/</a>, systematically compiles and integrates protein–protein interactions, encompassing both physical interactions and functional associations. The database is populated from various sources, including automated text mining of scientific literature, computational predictions based on co-expression and conserved genomic context, databases containing interaction experiments, and curated sources describing known complexes and pathways [<a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>]. IPA is a proprietary tool developed by QIAGEN with similar functionality that is employed for applications including biomarker discovery, metabolomics, microRNA research, next-generation sequencing data analysis, proteomics, toxicogenomics, and transcriptomics [<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>]. While most current gene network tools incorporate well-characterised large, curated datasets new tools that construct custom networks from genome/transcriptome patient data for a single patient offer potential for custom treatments [<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>].</p></section><section id="Sec13"><h3 class="pmc_sec_title">Comprehensive annotation tools</h3>
<p id="Par33">While many pipelines overlap annotation datasets consecutively in series, aggregate tools are becoming increasing popular to manage the increasing number of disparate annotation resources. With continuously updated databases, it is increasingly important to apply consistent, up to date annotations [<a href="#CR106" class="usa-link" aria-describedby="CR106">106</a>]. ENSEMBL’s Variant Effect Predictor (VEP) is a prominent aggregate tool in the landscape of functional annotation. VEP provides comprehensive annotations for genetic variants, including their functional consequences, conservation scores, and potential associations with known diseases [<a href="#CR107" class="usa-link" aria-describedby="CR107">107</a>]. The tool is known for its user-friendly interface and frequent updates, ensuring that researchers have access to the latest genomic information by linking results to ENSEMBL’s latest gene model. VEP’s ability to handle diverse types of genomic variants and its integration with various databases make it a valuable resource in annotation improvement efforts [<a href="#CR108" class="usa-link" aria-describedby="CR108">108</a>]. Additional tools like ANNOVAR perform a similar role [<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>].</p></section><section id="Sec14"><h3 class="pmc_sec_title">Complete end-to-end workflows</h3>
<p id="Par34">Beyond aggregate annotation tools, there are an increasing number of complete end-to-end variant prioritisation workflows. These tools typically integrate functionalities including variant annotation, filtering, and interpretation aiming to identify potentially pathogenic variants directly from input variant lists with little to no manual interpretation required. Example include VarSeq, a commercial variant analysis software tool that is designed to streamline the entire workflow, from variant discovery to interpretation for gene panels, exomes or genomes [<a href="#CR110" class="usa-link" aria-describedby="CR110">110</a>]. Non-commercial options include WANNOVAR, a web-based tool designed for annotation and functional prediction of genetic variants and VariantDB which is designed for the annotation, prioritisation and analysis of genetic variants [<a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>]. Seqr is an increasingly popular option developed by the Broad Institute [<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>].</p>
<p id="Par35">While this next generation of tools are promising, challenges with installation and lack of configurability hamper their widespread uptake with researchers and clinicians often favouring to combine multiple tools to perform concurrent steps in bespoke workflows [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p></section></section><section id="Sec15"><h2 class="pmc_sec_title">Machine learning applications</h2>
<p id="Par36">Addressing the significant challenges associated with pathogenic variant identification requires a multi-faceted approach. One potential solution being considered is the development of advanced machine learning (ML) algorithms. By leveraging ML, algorithms can potentially improve diagnosis rates by identifying underlying complex relationships within biological systems [<a href="#CR113" class="usa-link" aria-describedby="CR113">113</a>]. ML algorithms are gaining traction in life sciences due to the capacity to deal efficiently with complex genomic data patterns [<a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>]. Early works suggest that ML algorithms have the potential to learn from, and act upon complex heterogeneous datasets by identifying new biological patterns that increase diagnosis accuracy [<a href="#CR115" class="usa-link" aria-describedby="CR115">115</a>–<a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>]. It is likely that ML algorithms will play an increasingly important role in detecting pathogenic variants. Some recently published ML algorithms for precision medicine are listed in Table <a href="#Tab3" class="usa-link">3</a> .</p>
<section class="tw xbox font-sm" id="Tab3"><h3 class="obj_head">Table 3.</h3>
<div class="caption p"><p>ML applications in precision medicine</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Software</th>
<th align="left" colspan="1" rowspan="1">Function</th>
<th align="left" colspan="1" rowspan="1">ML approach</th>
<th align="left" colspan="1" rowspan="1">Website</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">AlphaMissense</td>
<td align="left" colspan="1" rowspan="1">Functional inference</td>
<td align="left" colspan="1" rowspan="1">Deep learning</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://alphamissense.hegelab.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://alphamissense.hegelab.org/</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DeepVariant</td>
<td align="left" colspan="1" rowspan="1">Variant detection</td>
<td align="left" colspan="1" rowspan="1">CNN</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://github.com/google/deepvariant" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/google/deepvariant</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">M-CAP</td>
<td align="left" colspan="1" rowspan="1">Variant prioritisation</td>
<td align="left" colspan="1" rowspan="1">Supervised ML</td>
<td align="left" colspan="1" rowspan="1">
<a href="http://bejerano.stanford.edu/MCAP/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bejerano.stanford.edu/MCAP/</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MLVar</td>
<td align="left" colspan="1" rowspan="1">Variant prioritisation</td>
<td align="left" colspan="1" rowspan="1">Method/Pipeline using ML</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://github.com/GiovannaNicora/MLVar" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/GiovannaNicora/MLVar</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">REVEL</td>
<td align="left" colspan="1" rowspan="1">Functional inference</td>
<td align="left" colspan="1" rowspan="1">Random Forest (RF)</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://sites.google.com/site/revelgenomics/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sites.google.com/site/revelgenomics/</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>BayesDel/</p>
<p>PEARCH</p>
</td>
<td align="left" colspan="1" rowspan="1">Functional inference</td>
<td align="left" colspan="1" rowspan="1">Likelihood Based approach</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://fenglab.chpc.utah.edu/BayesDel.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://fenglab.chpc.utah.edu/BayesDel.html</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ClinPred</td>
<td align="left" colspan="1" rowspan="1">Functional inference</td>
<td align="left" colspan="1" rowspan="1">Random forest and Gradient boosting models</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://sites.google.com/site/clinpred/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sites.google.com/site/clinpred/</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MAVERICK</td>
<td align="left" colspan="1" rowspan="1">Functional inference</td>
<td align="left" colspan="1" rowspan="1">Neural network-based</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://github.com/ZuchnerLab/Maverick" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ZuchnerLab/Maverick</a>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">EvAgg</td>
<td align="left" colspan="1" rowspan="1">Clinical curation</td>
<td align="left" colspan="1" rowspan="1">LLM model</td>
<td align="left" colspan="1" rowspan="1">
<a href="https://github.com/microsoft/healthfutures-evagg" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/microsoft/healthfutures-evagg</a>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section id="Sec16"><h3 class="pmc_sec_title">Variant pathogenicity</h3>
<p id="Par38">One of the most common current applications of ML is predicting variant pathogenicity, with numerous computational tools available [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>–<a href="#CR129" class="usa-link" aria-describedby="CR129">129</a>]. ML models are typically trained on both known pathogenic and benign variants and deployed to predict the functional consequences of genetic variants on both protein structure and function [<a href="#CR130" class="usa-link" aria-describedby="CR130">130</a>]. Depending on the composition of training data, these approaches can be divided into genome-wide, disease-specific, or even gene-specific categories [<a href="#CR131" class="usa-link" aria-describedby="CR131">131</a>]. Popular tools offering genome-wide data predictions include Rare Exome Variant Ensemble Learner (REVEL) [<a href="#CR124" class="usa-link" aria-describedby="CR124">124</a>], BayesDel [<a href="#CR132" class="usa-link" aria-describedby="CR132">132</a>], ClinPred [<a href="#CR133" class="usa-link" aria-describedby="CR133">133</a>] and AlphaMissense [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>].</p></section><section id="Sec17"><h3 class="pmc_sec_title">Variant prioritisation</h3>
<p id="Par39">Other common ML applications include variant detection, prioritisation and feature discovery. DeepVariant is an increasingly popular variant calling workflow compatible with Illumina, PacBio HiFi, and Oxford Nanopore sequence data [<a href="#CR134" class="usa-link" aria-describedby="CR134">134</a>]. DeepTrio is built upon DeepVariant and uses neural networks to identify variants specifically in pedigree of two or three members. M-CAP is a prioritisation tool that eliminates uncertain variants and reports 95% sensitivity levels [<a href="#CR130" class="usa-link" aria-describedby="CR130">130</a>]. MLVar is a another variant prioritisation workflow that follows ACMG guidelines and uses variant annotation features to predict probabilistic pathogenicity scores [<a href="#CR135" class="usa-link" aria-describedby="CR135">135</a>]. MAVERICK uses a neural network approach to predict pathogenic variants for Mendelian monogenic diseases [<a href="#CR136" class="usa-link" aria-describedby="CR136">136</a>]. Identifying cryptic biological features is another important ML application, for example the prediction and recognition of transcription start sites (TSSs) [<a href="#CR137" class="usa-link" aria-describedby="CR137">137</a>], splice sites [<a href="#CR138" class="usa-link" aria-describedby="CR138">138</a>], promoters [<a href="#CR139" class="usa-link" aria-describedby="CR139">139</a>], enhancers [<a href="#CR140" class="usa-link" aria-describedby="CR140">140</a>], and nucleosome position [<a href="#CR141" class="usa-link" aria-describedby="CR141">141</a>]. Larger end-to-end workflows employing ML are an active area of development.</p>
<p id="Par40">Genome-wide association studies (GWAS) are another active area of ML algorithm development. Tools such as genomic best linear unbiased prediction (gBLUP) [<a href="#CR142" class="usa-link" aria-describedby="CR142">142</a>], support vector machine (SVM) [<a href="#CR143" class="usa-link" aria-describedby="CR143">143</a>], xGBoost [<a href="#CR144" class="usa-link" aria-describedby="CR144">144</a>], and random forest (RF) [<a href="#CR145" class="usa-link" aria-describedby="CR145">145</a>] are widely used to identify relevant traits in GWAS, with the large, well annotated GWAS data suitable for training purposes. Similarly, large datasets divided into disease and controls serve as suitable training data for ML algorithms that are able to predict traits and identify enriched variants [<a href="#CR146" class="usa-link" aria-describedby="CR146">146</a>]. Many tools however fail to generalise to specific diseases. To address this, studies often run multiple ML methods using a consensus-based approach.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Large Language model</h3>
<p id="Par41">Large language models (LLM) show promise in a variety of precision medicine applications such as reducing literature search time for variant classification and interpretation. Microsoft developed the generative AI tool the EvAgg, which reports improvements in the sensitivity and specificity of pathogenic variant identification [<a href="#CR147" class="usa-link" aria-describedby="CR147">147</a>]. EvAgg reduced the manual curation time by 34% and increased the number of papers, variants, and cases evaluated per unit time. Such works have led to papers concluding that variant classification will be standardized in the near future, however achieving this requires overcoming significant challenges [<a href="#CR148" class="usa-link" aria-describedby="CR148">148</a>]. A recent benchmarking study considered four LLMs across ten fictional oncology patients and encouragingly found that LLMs were able to identify several important treatment strategies and provide some reasonable suggestions not easily found by experts [<a href="#CR149" class="usa-link" aria-describedby="CR149">149</a>]. However, they concluded that LLMs are not yet applicable for routine clinical analysis as an aid for clinical decision-making in oncology. Analysing complex germline disease represents an even bigger challenge.</p>
<p id="Par42">While ML holds great promise for precision medicine, it is not without challenges due to the complexity and uniqueness of an individual’s genome highlighting the need for accurate, robust and interpretable models. Challenges include issues with the accuracy of existing variant classifications as well as the rising number of variants of uncertain significance (VUS) [<a href="#CR150" class="usa-link" aria-describedby="CR150">150</a>]. Building robust ML models requires large, high-quality data that has been extensively benchmarked using both simulated and established reference data sets [<a href="#CR151" class="usa-link" aria-describedby="CR151">151</a>]. Generating such data sets is compounded by the inherent complexity of the human genome, with numerous non-genetic factors contributing to complex disease. While ML algorithms are increasingly able to process complex genomic information to identify novel patterns and associations relevant to variant interpretation, patient datasets are increasingly heterogeneous in terms of data type and source [<a href="#CR152" class="usa-link" aria-describedby="CR152">152</a>]. Increasingly, data sets contain a number of data modalities (e.g. transcriptomics, epigenomics, and clinical data) requiring updates and changes to existing ML models trained exclusively on other types of data [<a href="#CR153" class="usa-link" aria-describedby="CR153">153</a>]. ML algorithms face additional challenges with their need for large diverse and homogenous training datasets without potential biases, a known challenge with many complex molecular datasets [<a href="#CR154" class="usa-link" aria-describedby="CR154">154</a>]. Overall, the most significant barriers to widespread clinical adoption of ML approaches are model interpretability, model validation and data harmonisation.</p>
<p id="Par43">Model interpretability remains one of the most significant barriers to clinical uptake as any critical treatment decisions require a clear understanding of how models arrive at recommendations. Deep learning models, while highly effective at pattern recognition in genomic data and medical imaging, often function as “black boxes” where the decision-making process remains opaque. This is particularly problematic in precision medicine, where treatment decisions involve weighing complex risk-benefit profiles for individual patients. For example, a neural network might accurately predict cancer treatment response, but if clinicians cannot understand the biological rationale, they will be reluctant to act on it. New tools such as StratoMod [<a href="#CR155" class="usa-link" aria-describedby="CR155">155</a>] are using interpretable ML to predict variant calling and sequencing errors however without additional orthogonal validation data, clinical uptake remains unlikely. To address ‘black box’ challenges in security sensitive applications new tools like EnEXP are using interpretable ensemble tree approach to achieve a global interpretation of the entire dataset through the aggregation of individual sample insights [<a href="#CR156" class="usa-link" aria-describedby="CR156">156</a>]. Recent advances in explainable AI, including LIME and SHAP, offer promising approaches, but these post-hoc explanations may not accurately reflect the model’s true decision-making process [<a href="#CR157" class="usa-link" aria-describedby="CR157">157</a>].</p>
<p id="Par44">Data validation is limiting clinical uptake as it presents unique challenges beyond traditional evaluation metrics. While models might demonstrate excellent performance on test sets, real-world clinical translation requires additional considerations. Temporal validation represents a particular challenge as medical practice and treatment protocols evolve continuously [<a href="#CR158" class="usa-link" aria-describedby="CR158">158</a>], for example models trained on historical data may not perform reliably on current patients in rapidly advancing fields like oncology where new biomarkers are regularly discovered. External validation across different healthcare systems and patient populations is also essential but difficult to achieve in practice. Models developed at academic centres may not generalize to community hospitals with different demographics. Overall, the validation process must account for clinical decision-making’s dynamic nature, where model predictions influence subsequent patient management, creating complex feedback loops difficult to capture in traditional frameworks.</p>
<p id="Par45">Data harmonization is another significant barrier to clinical uptake. Genomic data harmonization involves reconciling different sequencing platforms, analytical pipelines, and annotation standards which can introduce systematic biases if not properly addressed [<a href="#CR159" class="usa-link" aria-describedby="CR159">159</a>]. Clinical data harmonization faces complexities from varying electronic health record systems, coding standards, and documentation practices. Laboratory records may use different units, clinical observations different terminologies, and treatment protocols may vary significantly across institutions. Temporal alignment of multi-modal data presents another significant challenge as genomic data is collected at specific time points while clinical data accumulates continuously, making it difficult to create coherent longitudinal patient profiles for ML training. Patient data is often siloed within healthcare systems due to privacy regulations and competitive concerns. Federated learning approaches offer potential solutions but introduce additional technical complexities [<a href="#CR160" class="usa-link" aria-describedby="CR160">160</a>]. Applying consistent guidelines and principles for data structure harmonization are critical; for example Findable, Accessible, Interoperable, and Reusable (FAIR) principles for data sharing [<a href="#CR161" class="usa-link" aria-describedby="CR161">161</a>] are gaining popularity. There have also been some advances in scalable approaches [<a href="#CR162" class="usa-link" aria-describedby="CR162">162</a>] and guidelines on using ML ethically [<a href="#CR163" class="usa-link" aria-describedby="CR163">163</a>]. However, these need more empirical validation before implementing within health care systems.</p></section></section><section id="Sec19"><h2 class="pmc_sec_title">Core principles for best practices</h2>
<p id="Par46">The state of art for best practices in pathogenic variant detection is a rapidly moving target, however core design principles are key in creating a robust and flexible framework able to integrate new modalities as they gain traction in precision medicine. Here we describe several core principles needed to develop a system appropriate for both the current and future needs in precision medicine.</p>
<p id="Par47">For general design considerations, it is critical to design a workflow that is modular, scalable, parallelisable, secure, reproducible and flexible. A modular design that can incorporate new data types and algorithms as needed is a key requirement when working in this rapidly evolving space. Design flexibility is also key to ensure longevity. For example the ability to run multiple tools and employ a consensus based approach is increasingly being recognised in a variety of applications including variant detection and RNA-Seq data analysis [<a href="#CR164" class="usa-link" aria-describedby="CR164">164</a>]. It is also preferable to incorporate well-tested standardised tools when available to avoid introducing unintentional errors arising from less well tested internally developed software. Another key consideration is the ability to parallelise and scale analysis components, important for reducing turnaround time for individual patients and for handling an increasing volume of samples. Consistency regarding input format requirements and outputs is also critical in ensuring backwards compatibility and the ability for re-analysis.</p>
<p id="Par48">More specific considerations for precision medicine are around security, interactivity and the ability to generate concise clinical reports. Handling patient data securely is critical for many reasons. Patient data that includes genomic information, medical records and family histories requires the utmost care and sensitivity to meet patient expectations. Misuse of such data may lead to discrimination and potential legal ramifications. De-identification is a common approach however it must be done properly to not allow re-identification via cross referencing of metadata or other public datasets. Ideally de-identification capability needs to be managed via an additional layer of access control. Security needs to be balanced with the development of interactive systems for clinicians who interrogate the data to identify pathogenic variants. Such web-based tools are critical however they need to ensure data is secure and protected throughout. The interface needs to support variant filtering and display variant summaries with the information needed to assess pathogenicity. The interface should support free text entry where clinicians record their determination and describe the evidence justifying the classification. A final consideration is the ability to develop robust clinical reports. Developing user friendly reports requires many iterations with clinicians to determine both the filters to employ and the level of detail to include for each candidate driver. The report design requires flexibility to incorporate addition, often disease specific, information as needed.</p>
<p id="Par49">Arguably the most critical component in precision medicine system is reproducibility. There are many practical considerations needed to ensure complete reproducibility, which can be achieved through a combination of code versioning, log file generation, robust testing suites and employing a software pipeline manager. Versioning all code and config files during development coupled with thorough logging of all commands lays the foundation for reproducible workflows. Extensive documentation of code and protocols is also critical particularly when multiple team members are involved. The development of a robust testing suite with gold standard datasets will help ensure any changes will not generate unintended downstream consequences. Finally, it is recommended to employ a modern pipeline manager tool such as NextFlow or Snakemake to facilitate running the workflow in a variety of hardware infrastructures including local infrastructure, HPC or cloud. Collectively following these design principles will drastically increase the reproducibility and longevity of the workflow.</p></section><section id="Sec20"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par50">While precision medicine has increased diagnosis rates around the world, current challenges exist including handling sensitive patient data and the lack of disease specific annotation, data standards and ethnically matched variant annotations. Future challenges include the need of integration of multi-omic data and the incorporation of new NGS data types (Fig. <a href="#Fig2" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374290_40246_2025_811_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bc99/12374290/5d9c479a1874/40246_2025_811_Fig2_HTML.jpg" loading="lazy" id="d33e1169" height="401" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Current and future challenges</p></figcaption></figure><section id="Sec21"><h3 class="pmc_sec_title">Current challenges</h3>
<p id="Par52">The systematic annotation of genetic variants within precision medicine programs has led to the discovery of large numbers of pathogenic variants however it has become clear no single strategy is universally effective for all diseases. Success within a disease class depends on multiple factors including sequence technology, patient selection and annotation strategy. For example, pedigree sequencing has been critical in identifying disease-causing <em>de novo</em> mutations driving neurodevelopmental disorders [<a href="#CR165" class="usa-link" aria-describedby="CR165">165</a>] and autism spectrum disorders [<a href="#CR166" class="usa-link" aria-describedby="CR166">166</a>] and in tracing inheritance patterns across multiple generations in condition like Huntington’s disease [<a href="#CR167" class="usa-link" aria-describedby="CR167">167</a>]. In autoimmune disease research, B-cell or T-cell receptor repertoire sequencing is increasingly utilised to identify pathogenic clonal lineages [<a href="#CR168" class="usa-link" aria-describedby="CR168">168</a>]. Additionally, certain mutation types are strongly associated with a specific disease class such as copy number variations (CNVs) in neurological and autoimmune disorders [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR169" class="usa-link" aria-describedby="CR169">169</a>] and splicing variants in Duchenne muscular dystrophy [<a href="#CR170" class="usa-link" aria-describedby="CR170">170</a>]. Careful patient selection based on genomic profiling has been instrumental in matching patients with targeted therapies as demonstrated by studies such as the NCI-MATCH (Molecular Analysis for Therapy Choice) trial [<a href="#CR171" class="usa-link" aria-describedby="CR171">171</a>]. Collectively these findings underscore the need for additional disease-specific considerations to improve genetic diagnosis yields.</p>
<p id="Par53">A major obstacle to incorporating genomic data into clinical practice is the lack of standard procedures for both analysing NGS data and summarising the relevant information into clinical reports. While progress has been made, consistent and reproducible methods remains essential to ensure the reliability of genomic results [<a href="#CR172" class="usa-link" aria-describedby="CR172">172</a>]. While it is widely acknowledged that ACMG standards and guidelines serve as the de facto gold standard for genetic therapy, their recommended datasets and workflows are often distributed across different online platforms and databases [<a href="#CR173" class="usa-link" aria-describedby="CR173">173</a>]. Further, laboratories often employ different tools and cutoffs, resulting in discrepancies in variant classification with such inconsistencies impacting patient care [<a href="#CR174" class="usa-link" aria-describedby="CR174">174</a>].</p>
<p id="Par54">Harmonizing interpretation guidelines is crucial for providing clinicians with reliable genomic information to guide personalized medical interventions [<a href="#CR175" class="usa-link" aria-describedby="CR175">175</a>]. However, any standardization must remain flexible given the fast pace of technological advancement and evolving data types [<a href="#CR176" class="usa-link" aria-describedby="CR176">176</a>]. Establishing global collaborations and encouraging data sharing initiatives can contribute to the development of comprehensive, standardized guidelines such as the Global Alliance for Genomics and Health (GA4GH) [<a href="#CR177" class="usa-link" aria-describedby="CR177">177</a>]. Standardized guidelines enhance consistency across laboratories, improve accuracy in variant classification, and ultimately contribute to the reliability of genomic data.</p>
<p id="Par55">The global genetic landscape is shaped by ethnic diversity and influenced by historical, geographical, and demographic factors [<a href="#CR178" class="usa-link" aria-describedby="CR178">178</a>]. To account for this, it is important to establish population-specific databases to capture the unique genetic variations across diverse ethnic groups. GNOMAD, for instance, has been instrumental in cataloguing genetic variations across varied populations, offering a valuable resource for ethnicity level variant frequency estimations [<a href="#CR179" class="usa-link" aria-describedby="CR179">179</a>]. This information can influence not only disease susceptibility but treatment response; for example, certain pharmacogenetic variants affect drug metabolism differently in various populations [<a href="#CR180" class="usa-link" aria-describedby="CR180">180</a>]. However establishing population-specific databases comes with challenges, including ethical considerations, data privacy, and ensuring adequate representation [<a href="#CR181" class="usa-link" aria-describedby="CR181">181</a>]. Initiatives such as ‘All of Us Research Program’ aims to address these issues by building inclusive, large-scale dataset that reflects the global genetic diversity. Such efforts are crucial in unravelling the complexities of genetic variants across distinct ethnic groups.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Future challenges</h3>
<p id="Par56">The rise of affordable sequencing technology has led to a growing reliance on data generated across various biological levels [<a href="#CR182" class="usa-link" aria-describedby="CR182">182</a>]. For example, the microbiome is increasingly being linked to human health outcomes with composition shifts observed during the onset of many diseases such as type II diabetes [<a href="#CR183" class="usa-link" aria-describedby="CR183">183</a>–<a href="#CR186" class="usa-link" aria-describedby="CR186">186</a>]. Integrating metagenomic and other multi-omic patient data with clinical information has the potential to improve prognostics and predictive accuracy of disease phenotypes ultimately leading to better treatment and prevention strategies [<a href="#CR187" class="usa-link" aria-describedby="CR187">187</a>, <a href="#CR188" class="usa-link" aria-describedby="CR188">188</a>]. However, the analysis of these complex datasets remains a challenge due to the inherent heterogeneity in individual omics datasets and the computational resources required for analysis and integration [<a href="#CR182" class="usa-link" aria-describedby="CR182">182</a>]. The rapid pace of change within the multi-omics space means benchmarking studies are essential to ensure appropriate tools are chosen to address specific biological questions [<a href="#CR189" class="usa-link" aria-describedby="CR189">189</a>–<a href="#CR191" class="usa-link" aria-describedby="CR191">191</a>]. Future developments should prioritise reducing complexity, enhancing interoperability, and creating user-friendly frameworks to consolidate multi-omics data.</p>
<p id="Par57">The effective application of precision medicine depends on precise, evidence-driven interpretation of genetic data, ensuring proper clinical management and care, while avoiding flawed conclusions that could lead to harm [<a href="#CR192" class="usa-link" aria-describedby="CR192">192</a>]. Genomic data is inherently dynamic and influenced by fast moving technological advancements meaning variants that were once classified as benign may need re-evaluation as new evidence emerges [<a href="#CR193" class="usa-link" aria-describedby="CR193">193</a>]. Advanced informatics and ML algorithms are poised to enable real-time data integration of diverse datasets, identifying clinically-relevant patterns that contribute to the continuous refinement of variant interpretation [<a href="#CR194" class="usa-link" aria-describedby="CR194">194</a>]. Such systems will empower clinicians with the most up-to-date variant interpretations, enhancing the precision and effectiveness of personalized healthcare. Central to this vision is working with Electronic Health Records (EHRs), which serve as comprehensive repositories of patient-specific data, encompassing medical histories, treatment responses, and other relevant information. Incorporating EHR data into the genetic variant prioritisation process enables a holistic view of the patient’s health journey [<a href="#CR195" class="usa-link" aria-describedby="CR195">195</a>]. Embracing a patient-centric framework will allow healthcare providers to tailor genetic interpretations that align with individual needs, ultimately realising the potential of personalised patient care.</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par58">Prioritising disease-causing genetic variants is fundamental for progressing personalized medicine, improving clinical diagnostics, and understanding genetic contributions to diseases. The adoption of precision medicine programs underscores the importance of prioritizing genetic variants, tailoring patient care based on genetic makeup and individual characteristics. While the affordability of quality sequence data has improved, substantiating the link between genes and diseases remains resource intensive. Accurate identification of disease-causing variants enhances diagnostic precision, aiding in early detection and targeted interventions for genetic disorders. Addressing current challenges today will ensure better precision medicine in the future.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>M.A.F and A.C conceived the review. S.D and M.A.F wrote the main text with contributions from M.K, U.S. and A.C. All authors reviewed the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>No datasets were generated or analysed during the current study.</p></section><section id="notes3"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par59">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Yang Y, Lyu J, Wang R, Wen Q, Zhao L, Chen W, et al. A digital mask to safeguard patient privacy. Nat Med. 2022;28(9):1883–92.
</cite> [<a href="https://doi.org/10.1038/s41591-022-01966-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9499857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36109638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20Y,%20Lyu%20J,%20Wang%20R,%20Wen%20Q,%20Zhao%20L,%20Chen%20W,%20et%20al.%20A%20digital%20mask%20to%20safeguard%20patient%20privacy.%20Nat%20Med.%202022;28(9):1883%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Ginsburg GS, Phillips KA. Precision medicine: from science to value. Health Aff (Millwood). 2018;37(5):694–701.
</cite> [<a href="https://doi.org/10.1377/hlthaff.2017.1624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5989714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29733705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ginsburg%20GS,%20Phillips%20KA.%20Precision%20medicine:%20from%20science%20to%20value.%20Health%20Aff%20(Millwood).%202018;37(5):694%E2%80%93701." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Petersen B-S, Fredrich B, Hoeppner MP, Ellinghaus D, Franke A. Opportunities and challenges of whole-genome and -exome sequencing. BMC Genet. 2017;18(1):14.
</cite> [<a href="https://doi.org/10.1186/s12863-017-0479-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5307692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28193154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petersen%20B-S,%20Fredrich%20B,%20Hoeppner%20MP,%20Ellinghaus%20D,%20Franke%20A.%20Opportunities%20and%20challenges%20of%20whole-genome%20and%20-exome%20sequencing.%20BMC%20Genet.%202017;18(1):14." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Magger O, Waldman YY, Ruppin E, Sharan R. Enhancing the prioritization of disease-causing genes through tissue specific protein interaction networks. PLoS Comput Biol. 2012;8(9):e1002690.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1002690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3459874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23028288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Magger%20O,%20Waldman%20YY,%20Ruppin%20E,%20Sharan%20R.%20Enhancing%20the%20prioritization%20of%20disease-causing%20genes%20through%20tissue%20specific%20protein%20interaction%20networks.%20PLoS%20Comput%20Biol.%202012;8(9):e1002690." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Field MA, Cho V, Andrews TD, Goodnow CC. Reliably detecting clinically important variants requires both combined variant calls and optimized filtering strategies. PLoS ONE. 2015;10(11):e0143199.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0143199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4658170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26600436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Field%20MA,%20Cho%20V,%20Andrews%20TD,%20Goodnow%20CC.%20Reliably%20detecting%20clinically%20important%20variants%20requires%20both%20combined%20variant%20calls%20and%20optimized%20filtering%20strategies.%20PLoS%20ONE.%202015;10(11):e0143199." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Berger SM, Appelbaum PS, Siegel K, Wynn J, Saami AM, Brokamp E, et al. Challenges of variant reinterpretation: opinions of stakeholders and need for guidelines. Genet Med. 2022;24(9):1878–87.
</cite> [<a href="https://doi.org/10.1016/j.gim.2022.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10407574/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35767006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berger%20SM,%20Appelbaum%20PS,%20Siegel%20K,%20Wynn%20J,%20Saami%20AM,%20Brokamp%20E,%20et%20al.%20Challenges%20of%20variant%20reinterpretation:%20opinions%20of%20stakeholders%20and%20need%20for%20guidelines.%20Genet%20Med.%202022;24(9):1878%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Field MA, Burgio G, Chuah A, Al Shekaili J, Hassan B, Al Sukaiti N, et al. Recurrent miscalling of missense variation from short-read genome sequence data. BMC Genomics. 2019;20(Suppl 8):546.
</cite> [<a href="https://doi.org/10.1186/s12864-019-5863-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6631443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31307400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Field%20MA,%20Burgio%20G,%20Chuah%20A,%20Al%20Shekaili%20J,%20Hassan%20B,%20Al%20Sukaiti%20N,%20et%20al.%20Recurrent%20miscalling%20of%20missense%20variation%20from%20short-read%20genome%20sequence%20data.%20BMC%20Genomics.%202019;20(Suppl%208):546." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Chen J, Li X, Zhong H, Meng Y, Du H. Systematic comparison of germline variant calling pipelines cross multiple next-generation sequencers. Sci Rep. 2019;9(1):9345.
</cite> [<a href="https://doi.org/10.1038/s41598-019-45835-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6597787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31249349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20J,%20Li%20X,%20Zhong%20H,%20Meng%20Y,%20Du%20H.%20Systematic%20comparison%20of%20germline%20variant%20calling%20pipelines%20cross%20multiple%20next-generation%20sequencers.%20Sci%20Rep.%202019;9(1):9345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Murphy DA, Elyashiv E, Amster G, Sella G. Broad-scale variation in human genetic diversity levels is predicted by purifying selection on coding and non-coding elements. Elife. 2023;12.</cite> [<a href="https://doi.org/10.7554/eLife.76065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10299832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36196994/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Virolainen SJ, VonHandorf A, Viel K, Weirauch MT, Kottyan LC. Gene-environment interactions and their impact on human health. Genes Immun. 2023;24(1):1–11.
</cite> [<a href="https://doi.org/10.1038/s41435-022-00192-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9801363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36585519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Virolainen%20SJ,%20VonHandorf%20A,%20Viel%20K,%20Weirauch%20MT,%20Kottyan%20LC.%20Gene-environment%20interactions%20and%20their%20impact%20on%20human%20health.%20Genes%20Immun.%202023;24(1):1%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Jeffreys AJ, Wilson V, Thein SL, Weatherall DJ, Ponder BA. DNA fingerprints and segregation analysis of multiple markers in human pedigrees. Am J Hum Genet. 1986;39(1):11–24.
</cite> [<a href="/articles/PMC1684027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3019128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jeffreys%20AJ,%20Wilson%20V,%20Thein%20SL,%20Weatherall%20DJ,%20Ponder%20BA.%20DNA%20fingerprints%20and%20segregation%20analysis%20of%20multiple%20markers%20in%20human%20pedigrees.%20Am%20J%20Hum%20Genet.%201986;39(1):11%E2%80%9324." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Field MA, Cho V, Cook MC, Enders A, Vinuesa C, Whittle B et al. Reducing the search space for causal genetic variants with VASP: variant analysis of sequenced pedigrees. Bioinformatics. 2015.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4495293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25755272/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Aureliano W. Difficult decisions and possible choices: rare diseases, genetic inheritance and reproduction of the family. Soc Sci Med. 2024;363:117380.
</cite> [<a href="https://doi.org/10.1016/j.socscimed.2024.117380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39561432/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aureliano%20W.%20Difficult%20decisions%20and%20possible%20choices:%20rare%20diseases,%20genetic%20inheritance%20and%20reproduction%20of%20the%20family.%20Soc%20Sci%20Med.%202024;363:117380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Strianese O, Rizzo F, Ciccarelli M, Galasso G, D’Agostino Y, Salvati A et al. Precision and Personalized Medicine: How Genomic Approach Improves the Management of Cardiovascular and Neurodegenerative Disease. Genes (Basel). 2020;11(7).</cite> [<a href="https://doi.org/10.3390/genes11070747" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7397223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32640513/" class="usa-link">PubMed</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Rehm HL, Fowler DM. Keeping up with the genomes: scaling genomic variant interpretation. Genome Med. 2019;12(1):5.
</cite> [<a href="https://doi.org/10.1186/s13073-019-0700-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6938604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31892366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rehm%20HL,%20Fowler%20DM.%20Keeping%20up%20with%20the%20genomes:%20scaling%20genomic%20variant%20interpretation.%20Genome%20Med.%202019;12(1):5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Pei XM, Yeung MHY, Wong ANN, Tsang HF, Yu ACS, Yim AKY et al. Targeted sequencing approach and its clinical applications for the molecular diagnosis of human diseases. Cells. 2023;12(3).</cite> [<a href="https://doi.org/10.3390/cells12030493" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9913990/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36766834/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Merker JD, Wenger AM, Sneddon T, Grove M, Zappala Z, Fresard L, et al. Long-read genome sequencing identifies causal structural variation in a Mendelian disease. Genet Med. 2018;20(1):159–63.
</cite> [<a href="https://doi.org/10.1038/gim.2017.86" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5741540/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28640241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Merker%20JD,%20Wenger%20AM,%20Sneddon%20T,%20Grove%20M,%20Zappala%20Z,%20Fresard%20L,%20et%20al.%20Long-read%20genome%20sequencing%20identifies%20causal%20structural%20variation%20in%20a%20Mendelian%20disease.%20Genet%20Med.%202018;20(1):159%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Young C, Singh M, Jackson KJL, Field MA, Peters TJ, Angioletti-Uberti S et al. A triad of somatic mutagenesis converges in self-reactive B cells to cause a virus-induced autoimmune disease. Immunity. 2025.</cite> [<a href="https://doi.org/10.1016/j.immuni.2024.12.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39818208/" class="usa-link">PubMed</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Singh M, Louie RHY, Samir J, Field MA, Milthorpe C, Adikari T, et al. Expanded T cell clones with lymphoma driver somatic mutations accumulate in refractory Celiac disease. Sci Transl Med. 2025;17(798):eadp6812.
</cite> [<a href="https://doi.org/10.1126/scitranslmed.adp6812" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40367192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh%20M,%20Louie%20RHY,%20Samir%20J,%20Field%20MA,%20Milthorpe%20C,%20Adikari%20T,%20et%20al.%20Expanded%20T%20cell%20clones%20with%20lymphoma%20driver%20somatic%20mutations%20accumulate%20in%20refractory%20Celiac%20disease.%20Sci%20Transl%20Med.%202025;17(798):eadp6812." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>McCabe MJ, Gauthier M-EA, Chan C-L, Thompson TJ, De Sousa SMC, Puttick C, et al. Development and validation of a targeted gene sequencing panel for application to disparate cancers. Sci Rep. 2019;9(1):17052.
</cite> [<a href="https://doi.org/10.1038/s41598-019-52000-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6864073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31745186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McCabe%20MJ,%20Gauthier%20M-EA,%20Chan%20C-L,%20Thompson%20TJ,%20De%20Sousa%20SMC,%20Puttick%20C,%20et%20al.%20Development%20and%20validation%20of%20a%20targeted%20gene%20sequencing%20panel%20for%20application%20to%20disparate%20cancers.%20Sci%20Rep.%202019;9(1):17052." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Rehm HL. Disease-targeted sequencing: a cornerstone in the clinic. Nat Rev Genet. 2013;14(4):295–300.
</cite> [<a href="https://doi.org/10.1038/nrg3463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3786217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23478348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rehm%20HL.%20Disease-targeted%20sequencing:%20a%20cornerstone%20in%20the%20clinic.%20Nat%20Rev%20Genet.%202013;14(4):295%E2%80%93300." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Andrews TD, Jeelall Y, Talaulikar D, Goodnow CC, Field MA. DeepSNVMiner: a sequence analysis tool to detect emergent, rare mutations in subsets of cell populations. PeerJ. 2016;4:e2074.
</cite> [<a href="https://doi.org/10.7717/peerj.2074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4888318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27257550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andrews%20TD,%20Jeelall%20Y,%20Talaulikar%20D,%20Goodnow%20CC,%20Field%20MA.%20DeepSNVMiner:%20a%20sequence%20analysis%20tool%20to%20detect%20emergent,%20rare%20mutations%20in%20subsets%20of%20cell%20populations.%20PeerJ.%202016;4:e2074." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Yu H, Yu H, Zhang R, Peng D, Yan D, Gu Y et al. Targeted gene panel provides advantages over whole-exome sequencing for diagnosing obesity and diabetes mellitus. J Mol Cell Biol. 2023.</cite> [<a href="https://doi.org/10.1093/jmcb/mjad040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10847719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37327085/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Johar AS, Mastronardi C, Rojas-Villarraga A, Patel HR, Chuah A, Peng K, et al. Novel and rare functional genomic variants in multiple autoimmune syndrome and sjogren’s syndrome. J Transl Med. 2015;13:173.
</cite> [<a href="https://doi.org/10.1186/s12967-015-0525-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4450850/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26031516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johar%20AS,%20Mastronardi%20C,%20Rojas-Villarraga%20A,%20Patel%20HR,%20Chuah%20A,%20Peng%20K,%20et%20al.%20Novel%20and%20rare%20functional%20genomic%20variants%20in%20multiple%20autoimmune%20syndrome%20and%20sjogren%E2%80%99s%20syndrome.%20J%20Transl%20Med.%202015;13:173." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Dunkerton S, Field M, Cho V, Bertram E, Whittle B, Groves A et al. A de Novo mutation in KMT2A (MLL) in monozygotic twins with Wiedemann-Steiner syndrome. Am J Med Genet A. 2015.</cite> [<a href="https://doi.org/10.1002/ajmg.a.37130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25929198/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Bamshad MJ, Ng SB, Bigham AW, Tabor HK, Emond MJ, Nickerson DA, et al. Exome sequencing as a tool for Mendelian disease gene discovery. Nat Rev Genet. 2011;12(11):745–55.
</cite> [<a href="https://doi.org/10.1038/nrg3031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21946919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bamshad%20MJ,%20Ng%20SB,%20Bigham%20AW,%20Tabor%20HK,%20Emond%20MJ,%20Nickerson%20DA,%20et%20al.%20Exome%20sequencing%20as%20a%20tool%20for%20Mendelian%20disease%20gene%20discovery.%20Nat%20Rev%20Genet.%202011;12(11):745%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Botstein D, Risch N. Discovering genotypes underlying human phenotypes: past successes for Mendelian disease, future approaches for complex disease. Nat Genet. 2003;33(Suppl):228–37.
</cite> [<a href="https://doi.org/10.1038/ng1090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12610532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Botstein%20D,%20Risch%20N.%20Discovering%20genotypes%20underlying%20human%20phenotypes:%20past%20successes%20for%20Mendelian%20disease,%20future%20approaches%20for%20complex%20disease.%20Nat%20Genet.%202003;33(Suppl):228%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Majewski J, Schwartzentruber J, Lalonde E, Montpetit A, Jabado N. What can exome sequencing do for you? J Med Genet. 2011;48(9):580–9.
</cite> [<a href="https://doi.org/10.1136/jmedgenet-2011-100223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21730106/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Majewski%20J,%20Schwartzentruber%20J,%20Lalonde%20E,%20Montpetit%20A,%20Jabado%20N.%20What%20can%20exome%20sequencing%20do%20for%20you?%20J%20Med%20Genet.%202011;48(9):580%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Ng SB, Buckingham KJ, Lee C, Bigham AW, Tabor HK, Dent KM, et al. Exome sequencing identifies the cause of a Mendelian disorder. Nat Genet. 2010;42(1):30–5.
</cite> [<a href="https://doi.org/10.1038/ng.499" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2847889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19915526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ng%20SB,%20Buckingham%20KJ,%20Lee%20C,%20Bigham%20AW,%20Tabor%20HK,%20Dent%20KM,%20et%20al.%20Exome%20sequencing%20identifies%20the%20cause%20of%20a%20Mendelian%20disorder.%20Nat%20Genet.%202010;42(1):30%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Rabbani B, Tekin M, Mahdieh N. The promise of whole-exome sequencing in medical genetics. J Hum Genet. 2014;59(1):5–15.
</cite> [<a href="https://doi.org/10.1038/jhg.2013.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24196381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rabbani%20B,%20Tekin%20M,%20Mahdieh%20N.%20The%20promise%20of%20whole-exome%20sequencing%20in%20medical%20genetics.%20J%20Hum%20Genet.%202014;59(1):5%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Nakagawa H, Wardell CP, Furuta M, Taniguchi H, Fujimoto A. Cancer whole-genome sequencing: present and future. Oncogene. 2015;34(49):5943–50.
</cite> [<a href="https://doi.org/10.1038/onc.2015.90" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25823020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakagawa%20H,%20Wardell%20CP,%20Furuta%20M,%20Taniguchi%20H,%20Fujimoto%20A.%20Cancer%20whole-genome%20sequencing:%20present%20and%20future.%20Oncogene.%202015;34(49):5943%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Scocchia A, Wigby KM, Masser-Frye D, Del Campo M, Galarreta CI, Thorpe E, et al. Clinical whole genome sequencing as a first-tier test at a resource-limited dysmorphology clinic in Mexico. NPJ Genom Med. 2019;4:5.
</cite> [<a href="https://doi.org/10.1038/s41525-018-0076-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6375919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30792901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scocchia%20A,%20Wigby%20KM,%20Masser-Frye%20D,%20Del%20Campo%20M,%20Galarreta%20CI,%20Thorpe%20E,%20et%20al.%20Clinical%20whole%20genome%20sequencing%20as%20a%20first-tier%20test%20at%20a%20resource-limited%20dysmorphology%20clinic%20in%20Mexico.%20NPJ%20Genom%20Med.%202019;4:5." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Belkadi A, Bolze A, Itan Y, Cobat A, Vincent QB, Antipenko A, et al. Whole-genome sequencing is more powerful than whole-exome sequencing for detecting exome variants. Proc Natl Acad Sci USA. 2015;112(17):5473–8.
</cite> [<a href="https://doi.org/10.1073/pnas.1418631112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4418901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25827230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Belkadi%20A,%20Bolze%20A,%20Itan%20Y,%20Cobat%20A,%20Vincent%20QB,%20Antipenko%20A,%20et%20al.%20Whole-genome%20sequencing%20is%20more%20powerful%20than%20whole-exome%20sequencing%20for%20detecting%20exome%20variants.%20Proc%20Natl%20Acad%20Sci%20USA.%202015;112(17):5473%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Lelieveld SH, Spielmann M, Mundlos S, Veltman JA, Gilissen C. Comparison of exome and genome sequencing technologies for the complete capture of Protein-Coding regions. Hum Mutat. 2015;36(8):815–22.
</cite> [<a href="https://doi.org/10.1002/humu.22813" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4755152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25973577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lelieveld%20SH,%20Spielmann%20M,%20Mundlos%20S,%20Veltman%20JA,%20Gilissen%20C.%20Comparison%20of%20exome%20and%20genome%20sequencing%20technologies%20for%20the%20complete%20capture%20of%20Protein-Coding%20regions.%20Hum%20Mutat.%202015;36(8):815%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Lionel AC, Costain G, Monfared N, Walker S, Reuter MS, Hosseini SM, et al. Improved diagnostic yield compared with targeted gene sequencing panels suggests a role for whole-genome sequencing as a first-tier genetic test. Genet Med. 2018;20(4):435–43.
</cite> [<a href="https://doi.org/10.1038/gim.2017.119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5895460/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28771251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lionel%20AC,%20Costain%20G,%20Monfared%20N,%20Walker%20S,%20Reuter%20MS,%20Hosseini%20SM,%20et%20al.%20Improved%20diagnostic%20yield%20compared%20with%20targeted%20gene%20sequencing%20panels%20suggests%20a%20role%20for%20whole-genome%20sequencing%20as%20a%20first-tier%20genetic%20test.%20Genet%20Med.%202018;20(4):435%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Bertoli-Avella AM, Beetz C, Ameziane N, Rocha ME, Guatibonza P, Pereira C, et al. Successful application of genome sequencing in a diagnostic setting: 1007 index cases from a clinically heterogeneous cohort. Eur J Hum Genet. 2021;29(1):141–53.
</cite> [<a href="https://doi.org/10.1038/s41431-020-00713-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7852664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32860008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bertoli-Avella%20AM,%20Beetz%20C,%20Ameziane%20N,%20Rocha%20ME,%20Guatibonza%20P,%20Pereira%20C,%20et%20al.%20Successful%20application%20of%20genome%20sequencing%20in%20a%20diagnostic%20setting:%201007%20index%20cases%20from%20a%20clinically%20heterogeneous%20cohort.%20Eur%20J%20Hum%20Genet.%202021;29(1):141%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Stavropoulos DJ, Merico D, Jobling R, Bowdin S, Monfared N, Thiruvahindrapuram B, et al. Whole genome sequencing expands diagnostic utility and improves clinical management in pediatric medicine. NPJ Genom Med. 2016;1:15012.
</cite> [<a href="https://doi.org/10.1038/npjgenmed.2015.12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5447450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28567303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stavropoulos%20DJ,%20Merico%20D,%20Jobling%20R,%20Bowdin%20S,%20Monfared%20N,%20Thiruvahindrapuram%20B,%20et%20al.%20Whole%20genome%20sequencing%20expands%20diagnostic%20utility%20and%20improves%20clinical%20management%20in%20pediatric%20medicine.%20NPJ%20Genom%20Med.%202016;1:15012." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Willig LK, Petrikin JE, Smith LD, Saunders CJ, Thiffault I, Miller NA, et al. Whole-genome sequencing for identification of Mendelian disorders in critically ill infants: a retrospective analysis of diagnostic and clinical findings. Lancet Respir Med. 2015;3(5):377–87.
</cite> [<a href="https://doi.org/10.1016/S2213-2600(15)00139-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4479194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25937001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Willig%20LK,%20Petrikin%20JE,%20Smith%20LD,%20Saunders%20CJ,%20Thiffault%20I,%20Miller%20NA,%20et%20al.%20Whole-genome%20sequencing%20for%20identification%20of%20Mendelian%20disorders%20in%20critically%20ill%20infants:%20a%20retrospective%20analysis%20of%20diagnostic%20and%20clinical%20findings.%20Lancet%20Respir%20Med.%202015;3(5):377%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Ostrander BEP, Butterfield RJ, Pedersen BS, Farrell AJ, Layer RM, Ward A, et al. Whole-genome analysis for effective clinical diagnosis and gene discovery in early infantile epileptic encephalopathy. Npj Genomic Med. 2018;3(1):22.</cite> [<a href="https://doi.org/10.1038/s41525-018-0061-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6089881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30109124/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ostrander%20BEP,%20Butterfield%20RJ,%20Pedersen%20BS,%20Farrell%20AJ,%20Layer%20RM,%20Ward%20A,%20et%20al.%20Whole-genome%20analysis%20for%20effective%20clinical%20diagnosis%20and%20gene%20discovery%20in%20early%20infantile%20epileptic%20encephalopathy.%20Npj%20Genomic%20Med.%202018;3(1):22." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Rajagopalan R, Gilbert MA, McEldrew DA, Nassur JA, Loomes KM, Piccoli DA, et al. Genome sequencing increases diagnostic yield in clinically diagnosed Alagille syndrome patients with previously negative test results. Genet Sci. 2021;23(2):323–30.</cite> [<a href="https://doi.org/10.1038/s41436-020-00989-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7862053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33077891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rajagopalan%20R,%20Gilbert%20MA,%20McEldrew%20DA,%20Nassur%20JA,%20Loomes%20KM,%20Piccoli%20DA,%20et%20al.%20Genome%20sequencing%20increases%20diagnostic%20yield%20in%20clinically%20diagnosed%20Alagille%20syndrome%20patients%20with%20previously%20negative%20test%20results.%20Genet%20Sci.%202021;23(2):323%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Austin-Tse CA, Jobanputra V, Perry DL, Bick D, Taft RJ, Venner E, et al. Best practices for the interpretation and reporting of clinical whole genome sequencing. Npj Genomic Med. 2022;7(1):27.</cite> [<a href="https://doi.org/10.1038/s41525-022-00295-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8993917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35395838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Austin-Tse%20CA,%20Jobanputra%20V,%20Perry%20DL,%20Bick%20D,%20Taft%20RJ,%20Venner%20E,%20et%20al.%20Best%20practices%20for%20the%20interpretation%20and%20reporting%20of%20clinical%20whole%20genome%20sequencing.%20Npj%20Genomic%20Med.%202022;7(1):27." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Hamzeh AR, Andrews TD, Field MA. Detecting causal variants in Mendelian disorders using Whole-Genome sequencing. Methods Mol Biol. 2021;2243:1–25.
</cite> [<a href="https://doi.org/10.1007/978-1-0716-1103-6_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33606250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hamzeh%20AR,%20Andrews%20TD,%20Field%20MA.%20Detecting%20causal%20variants%20in%20Mendelian%20disorders%20using%20Whole-Genome%20sequencing.%20Methods%20Mol%20Biol.%202021;2243:1%E2%80%9325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Hegenbarth J-C, Lezzoche G, De Windt LJ, Stoll M. Perspectives on Bulk-Tissue RNA sequencing and Single-Cell RNA sequencing for cardiac transcriptomics. Front Mol Med. 2022;2.</cite> [<a href="https://doi.org/10.3389/fmmed.2022.839338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11285642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39086967/" class="usa-link">PubMed</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Cummings BB, Marshall JL, Tukiainen T, Lek M, Donkervoort S, Foley AR et al. Improving genetic diagnosis in Mendelian disease with transcriptome sequencing. Sci Transl Med. 2017;9(386).</cite> [<a href="https://doi.org/10.1126/scitranslmed.aal5209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5548421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28424332/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Deng Q, Ramsköld D, Reinius B, Sandberg R. Single-cell RNA-seq reveals dynamic, random monoallelic gene expression in mammalian cells. Science. 2014;343(6167):193–6.
</cite> [<a href="https://doi.org/10.1126/science.1245316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24408435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deng%20Q,%20Ramsk%C3%B6ld%20D,%20Reinius%20B,%20Sandberg%20R.%20Single-cell%20RNA-seq%20reveals%20dynamic,%20random%20monoallelic%20gene%20expression%20in%20mammalian%20cells.%20Science.%202014;343(6167):193%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Jaitin DA, Kenigsberg E, Keren-Shaul H, Elefant N, Paul F, Zaretsky I, et al. Massively parallel single-cell RNA-seq for marker-free decomposition of tissues into cell types. Science. 2014;343(6172):776–9.
</cite> [<a href="https://doi.org/10.1126/science.1247651" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4412462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24531970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jaitin%20DA,%20Kenigsberg%20E,%20Keren-Shaul%20H,%20Elefant%20N,%20Paul%20F,%20Zaretsky%20I,%20et%20al.%20Massively%20parallel%20single-cell%20RNA-seq%20for%20marker-free%20decomposition%20of%20tissues%20into%20cell%20types.%20Science.%202014;343(6172):776%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Yan L, Yang M, Guo H, Yang L, Wu J, Li R, et al. Single-cell RNA-Seq profiling of human preimplantation embryos and embryonic stem cells. Nat Struct Mol Biol. 2013;20(9):1131–9.
</cite> [<a href="https://doi.org/10.1038/nsmb.2660" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23934149/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan%20L,%20Yang%20M,%20Guo%20H,%20Yang%20L,%20Wu%20J,%20Li%20R,%20et%20al.%20Single-cell%20RNA-Seq%20profiling%20of%20human%20preimplantation%20embryos%20and%20embryonic%20stem%20cells.%20Nat%20Struct%20Mol%20Biol.%202013;20(9):1131%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Goldman SL, MacKay M, Afshinnekoo E, Melnick AM, Wu S, Mason CE. The impact of heterogeneity on Single-Cell sequencing. Front Genet. 2019;10:8.
</cite> [<a href="https://doi.org/10.3389/fgene.2019.00008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6405636/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30881372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goldman%20SL,%20MacKay%20M,%20Afshinnekoo%20E,%20Melnick%20AM,%20Wu%20S,%20Mason%20CE.%20The%20impact%20of%20heterogeneity%20on%20Single-Cell%20sequencing.%20Front%20Genet.%202019;10:8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Brittain HK, Scott R, Thomas E. The rise of the genome and personalised medicine. Clin Med (Lond). 2017;17(6):545–51.
</cite> [<a href="https://doi.org/10.7861/clinmedicine.17-6-545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6297695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29196356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brittain%20HK,%20Scott%20R,%20Thomas%20E.%20The%20rise%20of%20the%20genome%20and%20personalised%20medicine.%20Clin%20Med%20(Lond).%202017;17(6):545%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Gawad C, Koh W, Quake SR. Single-cell genome sequencing: current state of the science. Nat Rev Genet. 2016;17(3):175–88.
</cite> [<a href="https://doi.org/10.1038/nrg.2015.16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26806412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gawad%20C,%20Koh%20W,%20Quake%20SR.%20Single-cell%20genome%20sequencing:%20current%20state%20of%20the%20science.%20Nat%20Rev%20Genet.%202016;17(3):175%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Satam H, Joshi K, Mangrolia U, Waghoo S, Zaidi G, Rawool S et al. Next-Generation sequencing technology: current trends and advancements. Biology (Basel). 2023;12(7).</cite> [<a href="https://doi.org/10.3390/biology12070997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10376292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37508427/" class="usa-link">PubMed</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Payne A, Holmes N, Rakyan V, Loose M. BulkVis: a graphical viewer for Oxford nanopore bulk FAST5 files. Bioinformatics. 2019;35(13):2193–8.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty841" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6596899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30462145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Payne%20A,%20Holmes%20N,%20Rakyan%20V,%20Loose%20M.%20BulkVis:%20a%20graphical%20viewer%20for%20Oxford%20nanopore%20bulk%20FAST5%20files.%20Bioinformatics.%202019;35(13):2193%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Wang Y, Zhao Y, Bollas A, Wang Y, Au KF. Nanopore sequencing technology, bioinformatics and applications. Nat Biotechnol. 2021;39(11):1348–65.
</cite> [<a href="https://doi.org/10.1038/s41587-021-01108-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8988251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34750572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Y,%20Zhao%20Y,%20Bollas%20A,%20Wang%20Y,%20Au%20KF.%20Nanopore%20sequencing%20technology,%20bioinformatics%20and%20applications.%20Nat%20Biotechnol.%202021;39(11):1348%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Athanasopoulou K, Boti MA, Adamopoulos PG, Skourou PC, Scorilas A. Third-Generation sequencing: the spearhead towards the radical transformation of modern genomics. Life (Basel). 2021;12(1).</cite> [<a href="https://doi.org/10.3390/life12010030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8780579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35054423/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Uhrig S, Ellermann J, Walther T, Burkhardt P, Fröhlich M, Hutter B, et al. Accurate and efficient detection of gene fusions from RNA sequencing data. Genome Res. 2021;31(3):448–60.
</cite> [<a href="https://doi.org/10.1101/gr.257246.119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7919457/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33441414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uhrig%20S,%20Ellermann%20J,%20Walther%20T,%20Burkhardt%20P,%20Fr%C3%B6hlich%20M,%20Hutter%20B,%20et%20al.%20Accurate%20and%20efficient%20detection%20of%20gene%20fusions%20from%20RNA%20sequencing%20data.%20Genome%20Res.%202021;31(3):448%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Johar AS, Anaya JM, Andrews D, Patel HR, Field M, Goodnow C et al. Candidate gene discovery in autoimmunity by using extreme phenotypes, next generation sequencing and whole exome capture. Autoimmun Rev. 2014.</cite> [<a href="https://doi.org/10.1016/j.autrev.2014.10.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25447288/" class="usa-link">PubMed</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Kohler S, Carmody L, Vasilevsky N, Jacobsen JOB, Danis D, Gourdine JP, et al. Expansion of the human phenotype ontology (HPO) knowledge base and resources. Nucleic Acids Res. 2019;47(D1):D1018–27.
</cite> [<a href="https://doi.org/10.1093/nar/gky1105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30476213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kohler%20S,%20Carmody%20L,%20Vasilevsky%20N,%20Jacobsen%20JOB,%20Danis%20D,%20Gourdine%20JP,%20et%20al.%20Expansion%20of%20the%20human%20phenotype%20ontology%20(HPO)%20knowledge%20base%20and%20resources.%20Nucleic%20Acids%20Res.%202019;47(D1):D1018%E2%80%9327." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Patino CM, Ferreira JC. Inclusion and exclusion criteria in research studies: definitions and why they matter. J Bras Pneumol. 2018;44(2):84.
</cite> [<a href="https://doi.org/10.1590/S1806-37562018000000088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6044655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29791550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patino%20CM,%20Ferreira%20JC.%20Inclusion%20and%20exclusion%20criteria%20in%20research%20studies:%20definitions%20and%20why%20they%20matter.%20J%20Bras%20Pneumol.%202018;44(2):84." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Cirulli ET, Goldstein DB. Uncovering the roles of rare variants in common disease through whole-genome sequencing. Nat Rev Genet. 2010;11(6):415–25.
</cite> [<a href="https://doi.org/10.1038/nrg2779" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20479773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cirulli%20ET,%20Goldstein%20DB.%20Uncovering%20the%20roles%20of%20rare%20variants%20in%20common%20disease%20through%20whole-genome%20sequencing.%20Nat%20Rev%20Genet.%202010;11(6):415%E2%80%9325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Manolio TA, Collins FS, Cox NJ, Goldstein DB, Hindorff LA, Hunter DJ, et al. Finding the missing heritability of complex diseases. Nature. 2009;461(7265):747–53.
</cite> [<a href="https://doi.org/10.1038/nature08494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2831613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19812666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manolio%20TA,%20Collins%20FS,%20Cox%20NJ,%20Goldstein%20DB,%20Hindorff%20LA,%20Hunter%20DJ,%20et%20al.%20Finding%20the%20missing%20heritability%20of%20complex%20diseases.%20Nature.%202009;461(7265):747%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Tennessen JA, Bigham AW, O’Connor TD, Fu W, Kenny EE, Gravel S, et al. Evolution and functional impact of rare coding variation from deep sequencing of human exomes. Science. 2012;337(6090):64–9.
</cite> [<a href="https://doi.org/10.1126/science.1219240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3708544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22604720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tennessen%20JA,%20Bigham%20AW,%20O%E2%80%99Connor%20TD,%20Fu%20W,%20Kenny%20EE,%20Gravel%20S,%20et%20al.%20Evolution%20and%20functional%20impact%20of%20rare%20coding%20variation%20from%20deep%20sequencing%20of%20human%20exomes.%20Science.%202012;337(6090):64%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Chong JX, Buckingham KJ, Jhangiani SN, Boehm C, Sobreira N, Smith JD, et al. The genetic basis of Mendelian phenotypes: discoveries, challenges, and opportunities. Am J Hum Genet. 2015;97(2):199–215.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2015.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4573249/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26166479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chong%20JX,%20Buckingham%20KJ,%20Jhangiani%20SN,%20Boehm%20C,%20Sobreira%20N,%20Smith%20JD,%20et%20al.%20The%20genetic%20basis%20of%20Mendelian%20phenotypes:%20discoveries,%20challenges,%20and%20opportunities.%20Am%20J%20Hum%20Genet.%202015;97(2):199%E2%80%93215." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Al Sukaiti N, AbdelRahman K, AlShekaili J, Al Oraimi S, Al Sinani A, Al Rahbi N, et al. Agammaglobulinaemia despite terminal B-cell differentiation in a patient with a novel LRBA mutation. Clin Translational Immunol. 2017;6(5):e144.</cite> [<a href="https://doi.org/10.1038/cti.2017.20" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5493589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28690850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Al%20Sukaiti%20N,%20AbdelRahman%20K,%20AlShekaili%20J,%20Al%20Oraimi%20S,%20Al%20Sinani%20A,%20Al%20Rahbi%20N,%20et%20al.%20Agammaglobulinaemia%20despite%20terminal%20B-cell%20differentiation%20in%20a%20patient%20with%20a%20novel%20LRBA%20mutation.%20Clin%20Translational%20Immunol.%202017;6(5):e144." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Jarvik GPBB. Consideration of Cosegregation in the Pathogenicity Classification of Genomic Variants. 2016.</cite> [<a href="https://doi.org/10.1016/j.ajhg.2016.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4908147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27236918/" class="usa-link">PubMed</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Richards SAN, Bale S et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American college of medical genetics and genomics and the association for molecular pathology. 2015.</cite> [<a href="https://doi.org/10.1038/gim.2015.30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4544753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25741868/" class="usa-link">PubMed</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Biesecker LG, Byrne AB, Harrison SM, Pesaran T, Schaffer AA, Shirts BH, et al. ClinGen guidance for use of the PP1/BS4 co-segregation and PP4 phenotype specificity criteria for sequence variant pathogenicity classification. Am J Hum Genet. 2024;111(1):24–38.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2023.11.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10806742/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38103548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Biesecker%20LG,%20Byrne%20AB,%20Harrison%20SM,%20Pesaran%20T,%20Schaffer%20AA,%20Shirts%20BH,%20et%20al.%20ClinGen%20guidance%20for%20use%20of%20the%20PP1/BS4%20co-segregation%20and%20PP4%20phenotype%20specificity%20criteria%20for%20sequence%20variant%20pathogenicity%20classification.%20Am%20J%20Hum%20Genet.%202024;111(1):24%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>MacArthur DGMT, Dimmock DP et al. Guidelines for investigating causality of sequence variants in human disease. 2014.</cite> [<a href="https://doi.org/10.1038/nature13127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4180223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24759409/" class="usa-link">PubMed</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Biesecker LGGR. Diagnostic clinical genome and exome sequencing. 2014.</cite> [<a href="https://doi.org/10.1056/NEJMra1312543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24941179/" class="usa-link">PubMed</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Ng KPB, Kiat Puar TH et al. COVID-19 and the Risk to Health Care Workers: A Case Report. 2020.</cite> [<a href="https://doi.org/10.7326/L20-0175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7081171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32176257/" class="usa-link">PubMed</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Cooper CSN, Livingston G et al. Ethnic inequalities in the use of health services for common mental disorders in England. 2013.</cite> [<a href="https://doi.org/10.1007/s00127-012-0565-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22893107/" class="usa-link">PubMed</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>McLaren W, Gil L, Hunt SE, Riat HS, Ritchie GRS, Thormann A, et al. The ensembl variant effect predictor. Genome Biol. 2016;17(1):122.
</cite> [<a href="https://doi.org/10.1186/s13059-016-0974-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4893825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27268795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McLaren%20W,%20Gil%20L,%20Hunt%20SE,%20Riat%20HS,%20Ritchie%20GRS,%20Thormann%20A,%20et%20al.%20The%20ensembl%20variant%20effect%20predictor.%20Genome%20Biol.%202016;17(1):122." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Cingolani P, Platts A, Wang le L, Coon M, Nguyen T, Wang L, et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, snpeff: SNPs in the genome of drosophila melanogaster strain w1118; iso-2; iso-3. Fly (Austin). 2012;6(2):80–92.
</cite> [<a href="https://doi.org/10.4161/fly.19695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3679285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22728672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cingolani%20P,%20Platts%20A,%20Wang%20le%20L,%20Coon%20M,%20Nguyen%20T,%20Wang%20L,%20et%20al.%20A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms,%20snpeff:%20SNPs%20in%20the%20genome%20of%20drosophila%20melanogaster%20strain%20w1118;%20iso-2;%20iso-3.%20Fly%20(Austin).%202012;6(2):80%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 2010;38(16):e164.
</cite> [<a href="https://doi.org/10.1093/nar/gkq603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2938201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20601685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20K,%20Li%20M,%20Hakonarson%20H.%20ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data.%20Nucleic%20Acids%20Res.%202010;38(16):e164." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Ng PC, Henikoff S. SIFT: predicting amino acid changes that affect protein function. Nucleic Acids Res. 2003;31(13):3812–4.
</cite> [<a href="https://doi.org/10.1093/nar/gkg509" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC168916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12824425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ng%20PC,%20Henikoff%20S.%20SIFT:%20predicting%20amino%20acid%20changes%20that%20affect%20protein%20function.%20Nucleic%20Acids%20Res.%202003;31(13):3812%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Adzhubei IA, Schmidt S, Peshkin L, Ramensky VE, Gerasimova A, Bork P, et al. A method and server for predicting damaging missense mutations. Nat Methods. 2010;7(4):248–9.
</cite> [<a href="https://doi.org/10.1038/nmeth0410-248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2855889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20354512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adzhubei%20IA,%20Schmidt%20S,%20Peshkin%20L,%20Ramensky%20VE,%20Gerasimova%20A,%20Bork%20P,%20et%20al.%20A%20method%20and%20server%20for%20predicting%20damaging%20missense%20mutations.%20Nat%20Methods.%202010;7(4):248%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Kircher M, Witten DM, Jain P, O’Roak BJ, Cooper GM, Shendure J. A general framework for estimating the relative pathogenicity of human genetic variants. Nat Genet. 2014;46(3):310–5.
</cite> [<a href="https://doi.org/10.1038/ng.2892" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3992975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24487276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kircher%20M,%20Witten%20DM,%20Jain%20P,%20O%E2%80%99Roak%20BJ,%20Cooper%20GM,%20Shendure%20J.%20A%20general%20framework%20for%20estimating%20the%20relative%20pathogenicity%20of%20human%20genetic%20variants.%20Nat%20Genet.%202014;46(3):310%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>McConnell H, Andrews TD, Field MA. Efficacy of computational predictions of the functional effect of idiosyncratic Pharmacogenetic variants. PeerJ. 2021;9:e11774.
</cite> [<a href="https://doi.org/10.7717/peerj.11774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8286708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34316407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McConnell%20H,%20Andrews%20TD,%20Field%20MA.%20Efficacy%20of%20computational%20predictions%20of%20the%20functional%20effect%20of%20idiosyncratic%20Pharmacogenetic%20variants.%20PeerJ.%202021;9:e11774." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Miosge LA, Field MA, Sontani Y, Cho V, Johnson S, Palkova A, et al. Comparison of predicted and actual consequences of missense mutations. Proceedings of the National Academy of Sciences of the United States of America; 2015.</cite> [<a href="https://doi.org/10.1073/pnas.1511585112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4577149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26269570/" class="usa-link">PubMed</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Senior AW, Evans R, Jumper J, Kirkpatrick J, Sifre L, Green T, et al. Improved protein structure prediction using potentials from deep learning. Nature. 2020;577(7792):706–10.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1923-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31942072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Senior%20AW,%20Evans%20R,%20Jumper%20J,%20Kirkpatrick%20J,%20Sifre%20L,%20Green%20T,%20et%20al.%20Improved%20protein%20structure%20prediction%20using%20potentials%20from%20deep%20learning.%20Nature.%202020;577(7792):706%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure prediction with alphafold. Nature. 2021;596(7873):583–9.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumper%20J,%20Evans%20R,%20Pritzel%20A,%20Green%20T,%20Figurnov%20M,%20Ronneberger%20O,%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20alphafold.%20Nature.%202021;596(7873):583%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Schmidt A, Röner S, Mai K, Klinkhammer H, Kircher M, Ludwig KU. Predicting the pathogenicity of missense variants using features derived from AlphaFold2. Bioinformatics. 2023;39(5).</cite> [<a href="https://doi.org/10.1093/bioinformatics/btad280" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10203375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37084271/" class="usa-link">PubMed</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Gudmundsson S, Singer-Berk M, Watts NA, Phu W, Goodrich JK, Solomonson M, et al. Variant interpretation using population databases: lessons from GnomAD. Hum Mutat. 2022;43(8):1012–30.
</cite> [<a href="https://doi.org/10.1002/humu.24309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9160216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34859531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gudmundsson%20S,%20Singer-Berk%20M,%20Watts%20NA,%20Phu%20W,%20Goodrich%20JK,%20Solomonson%20M,%20et%20al.%20Variant%20interpretation%20using%20population%20databases:%20lessons%20from%20GnomAD.%20Hum%20Mutat.%202022;43(8):1012%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, Gibbs RA, et al. A map of human genome variation from population-scale sequencing. Nature. 2010;467(7319):1061–73.
</cite> [<a href="https://doi.org/10.1038/nature09534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3042601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20981092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abecasis%20GR,%20Altshuler%20D,%20Auton%20A,%20Brooks%20LD,%20Durbin%20RM,%20Gibbs%20RA,%20et%20al.%20A%20map%20of%20human%20genome%20variation%20from%20population-scale%20sequencing.%20Nature.%202010;467(7319):1061%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Sherry ST, Ward MH, Kholodov M, Baker J, Phan L, Smigielski EM, et al. DbSNP: the NCBI database of genetic variation. Nucleic Acids Res. 2001;29(1):308–11.
</cite> [<a href="https://doi.org/10.1093/nar/29.1.308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC29783/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11125122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sherry%20ST,%20Ward%20MH,%20Kholodov%20M,%20Baker%20J,%20Phan%20L,%20Smigielski%20EM,%20et%20al.%20DbSNP:%20the%20NCBI%20database%20of%20genetic%20variation.%20Nucleic%20Acids%20Res.%202001;29(1):308%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Karczewski KJ, Francioli LC, Tiao G, Cummings BB, Alföldi J, Wang Q et al. Variation across 141,456 human exomes and genomes reveals the spectrum of loss-of-function intolerance across human protein-coding genes. BioRxiv. 2019:531210.</cite>
</li>
<li id="CR86">
<span class="label">86.</span><cite>Lek M, Karczewski KJ, Minikel EV, Samocha KE, Banks E, Fennell T, et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature. 2016;536(7616):285–91.
</cite> [<a href="https://doi.org/10.1038/nature19057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5018207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27535533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lek%20M,%20Karczewski%20KJ,%20Minikel%20EV,%20Samocha%20KE,%20Banks%20E,%20Fennell%20T,%20et%20al.%20Analysis%20of%20protein-coding%20genetic%20variation%20in%2060,706%20humans.%20Nature.%202016;536(7616):285%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Calcino A, Cooke I, Cowman P, Higgie M, Massault C, Schmitz U, et al. Harnessing genomic technologies for one health solutions in the tropics. Global Health. 2024;20(1):78.
</cite> [<a href="https://doi.org/10.1186/s12992-024-01083-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11566161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39543642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calcino%20A,%20Cooke%20I,%20Cowman%20P,%20Higgie%20M,%20Massault%20C,%20Schmitz%20U,%20et%20al.%20Harnessing%20genomic%20technologies%20for%20one%20health%20solutions%20in%20the%20tropics.%20Global%20Health.%202024;20(1):78." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Landrum MJ, Lee JM, Riley GR, Jang W, Rubinstein WS, Church DM, et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Nucleic Acids Res. 2014;42(Database issue):D980–5.
</cite> [<a href="https://doi.org/10.1093/nar/gkt1113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3965032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24234437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Landrum%20MJ,%20Lee%20JM,%20Riley%20GR,%20Jang%20W,%20Rubinstein%20WS,%20Church%20DM,%20et%20al.%20ClinVar:%20public%20archive%20of%20relationships%20among%20sequence%20variation%20and%20human%20phenotype.%20Nucleic%20Acids%20Res.%202014;42(Database%20issue):D980%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Stenson PD, Ball EV, Mort M, Phillips AD, Shiel JA, Thomas NS, et al. Human gene mutation database (HGMD): 2003 update. Hum Mutat. 2003;21(6):577–81.
</cite> [<a href="https://doi.org/10.1002/humu.10212" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12754702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stenson%20PD,%20Ball%20EV,%20Mort%20M,%20Phillips%20AD,%20Shiel%20JA,%20Thomas%20NS,%20et%20al.%20Human%20gene%20mutation%20database%20(HGMD):%202003%20update.%20Hum%20Mutat.%202003;21(6):577%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Fokkema IF, Taschner PE, Schaafsma GC, Celli J, Laros JF, den Dunnen JT. LOVD v.2.0: the next generation in gene variant databases. Hum Mutat. 2011;32(5):557–63.
</cite> [<a href="https://doi.org/10.1002/humu.21438" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21520333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fokkema%20IF,%20Taschner%20PE,%20Schaafsma%20GC,%20Celli%20J,%20Laros%20JF,%20den%20Dunnen%20JT.%20LOVD%20v.2.0:%20the%20next%20generation%20in%20gene%20variant%20databases.%20Hum%20Mutat.%202011;32(5):557%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Shabani M, Dyke SOM, Marelli L, Borry P. Variant data sharing by clinical laboratories through public databases: consent, privacy and further contact for research policies. Genet Sci. 2019;21(5):1031–7.</cite> [<a href="https://doi.org/10.1038/s41436-018-0316-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30293992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shabani%20M,%20Dyke%20SOM,%20Marelli%20L,%20Borry%20P.%20Variant%20data%20sharing%20by%20clinical%20laboratories%20through%20public%20databases:%20consent,%20privacy%20and%20further%20contact%20for%20research%20policies.%20Genet%20Sci.%202019;21(5):1031%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, Korbel JO, et al. A global reference for human genetic variation. Nature. 2015;526(7571):68–74.
</cite> [<a href="https://doi.org/10.1038/nature15393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4750478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Auton%20A,%20Brooks%20LD,%20Durbin%20RM,%20Garrison%20EP,%20Kang%20HM,%20Korbel%20JO,%20et%20al.%20A%20global%20reference%20for%20human%20genetic%20variation.%20Nature.%202015;526(7571):68%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Stenson PD, Mort M, Ball EV, Evans K, Hayden M, Heywood S, et al. The human gene mutation database: towards a comprehensive repository of inherited mutation data for medical research, genetic diagnosis and next-generation sequencing studies. Hum Genet. 2017;136(6):665–77.
</cite> [<a href="https://doi.org/10.1007/s00439-017-1779-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5429360/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28349240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stenson%20PD,%20Mort%20M,%20Ball%20EV,%20Evans%20K,%20Hayden%20M,%20Heywood%20S,%20et%20al.%20The%20human%20gene%20mutation%20database:%20towards%20a%20comprehensive%20repository%20of%20inherited%20mutation%20data%20for%20medical%20research,%20genetic%20diagnosis%20and%20next-generation%20sequencing%20studies.%20Hum%20Genet.%202017;136(6):665%E2%80%9377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Fokkema IFAC, Kroon M, López Hernández JA, Asscheman D, Lugtenburg I, Hoogenboom J, et al. The LOVD3 platform: efficient genome-wide sharing of genetic variants. Eur J Hum Genet. 2021;29(12):1796–803.
</cite> [<a href="https://doi.org/10.1038/s41431-021-00959-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8632977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34521998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fokkema%20IFAC,%20Kroon%20M,%20L%C3%B3pez%20Hern%C3%A1ndez%20JA,%20Asscheman%20D,%20Lugtenburg%20I,%20Hoogenboom%20J,%20et%20al.%20The%20LOVD3%20platform:%20efficient%20genome-wide%20sharing%20of%20genetic%20variants.%20Eur%20J%20Hum%20Genet.%202021;29(12):1796%E2%80%93803." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Aken BL, Ayling S, Barrell D, Clarke L, Curwen V, Fairley S et al. The Ensembl gene annotation system. Database (Oxford). 2016;2016.</cite> [<a href="https://doi.org/10.1093/database/baw093" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4919035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27337980/" class="usa-link">PubMed</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Frankish A, Diekhans M, Ferreira AM, Johnson R, Jungreis I, Loveland J, et al. GENCODE reference annotation for the human and mouse genomes. Nucleic Acids Res. 2019;47(D1):D766–73.
</cite> [<a href="https://doi.org/10.1093/nar/gky955" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30357393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Frankish%20A,%20Diekhans%20M,%20Ferreira%20AM,%20Johnson%20R,%20Jungreis%20I,%20Loveland%20J,%20et%20al.%20GENCODE%20reference%20annotation%20for%20the%20human%20and%20mouse%20genomes.%20Nucleic%20Acids%20Res.%202019;47(D1):D766%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Pruitt KD, Tatusova T, Maglott DR. NCBI reference sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acids Res. 2005;33(Database issue):D501–4.
</cite> [<a href="https://doi.org/10.1093/nar/gki025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC539979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15608248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pruitt%20KD,%20Tatusova%20T,%20Maglott%20DR.%20NCBI%20reference%20sequence%20(RefSeq):%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes,%20transcripts%20and%20proteins.%20Nucleic%20Acids%20Res.%202005;33(Database%20issue):D501%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Hsu F, Kent WJ, Clawson H, Kuhn RM, Diekhans M, Haussler D. The UCSC known genes. Bioinformatics. 2006;22(9):1036–46.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btl048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16500937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hsu%20F,%20Kent%20WJ,%20Clawson%20H,%20Kuhn%20RM,%20Diekhans%20M,%20Haussler%20D.%20The%20UCSC%20known%20genes.%20Bioinformatics.%202006;22(9):1036%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>UniProt Consortium T. UniProt: the universal protein knowledgebase. Nucleic Acids Res. 2018;46(5):2699.
</cite> [<a href="https://doi.org/10.1093/nar/gky092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5861450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29425356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?UniProt%20Consortium%20T.%20UniProt:%20the%20universal%20protein%20knowledgebase.%20Nucleic%20Acids%20Res.%202018;46(5):2699." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Hamosh A, Amberger JS, Bocchini C, Scott AF, Rasmussen SA. Online Mendelian inheritance in man (OMIM<sup>®</sup>): victor mckusick’s magnum opus. Am J Med Genet A. 2021;185(11):3259–65.
</cite> [<a href="https://doi.org/10.1002/ajmg.a.62407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8596664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34169650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hamosh%20A,%20Amberger%20JS,%20Bocchini%20C,%20Scott%20AF,%20Rasmussen%20SA.%20Online%20Mendelian%20inheritance%20in%20man%20(OMIM%C2%AE):%20victor%20mckusick%E2%80%99s%20magnum%20opus.%20Am%20J%20Med%20Genet%20A.%202021;185(11):3259%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Blake JA, Dolan M, Drabkin H, Hill DP, Li N, Sitnikov D, et al. Gene ontology annotations and resources. Nucleic Acids Res. 2013;41(Database issue):D530–5.
</cite> [<a href="https://doi.org/10.1093/nar/gks1050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23161678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blake%20JA,%20Dolan%20M,%20Drabkin%20H,%20Hill%20DP,%20Li%20N,%20Sitnikov%20D,%20et%20al.%20Gene%20ontology%20annotations%20and%20resources.%20Nucleic%20Acids%20Res.%202013;41(Database%20issue):D530%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Zhuang Y, Xing F, Ghosh D, Banaei-Kashani F, Bowler RP, Kechris K. An augmented High-Dimensional graphical Lasso method to incorporate prior biological knowledge for global network learning. Front Genet. 2022;12.</cite> [<a href="https://doi.org/10.3389/fgene.2021.760299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8829118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35154240/" class="usa-link">PubMed</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Szklarczyk D, Kirsch R, Koutrouli M, Nastou K, Mehryary F, Hachilif R, et al. The STRING database in 2023: protein-protein association networks and functional enrichment analyses for any sequenced genome of interest. Nucleic Acids Res. 2023;51(D1):D638–46.
</cite> [<a href="https://doi.org/10.1093/nar/gkac1000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36370105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szklarczyk%20D,%20Kirsch%20R,%20Koutrouli%20M,%20Nastou%20K,%20Mehryary%20F,%20Hachilif%20R,%20et%20al.%20The%20STRING%20database%20in%202023:%20protein-protein%20association%20networks%20and%20functional%20enrichment%20analyses%20for%20any%20sequenced%20genome%20of%20interest.%20Nucleic%20Acids%20Res.%202023;51(D1):D638%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Kramer A, Green J, Pollard J Jr., Tugendreich S. Causal analysis approaches in ingenuity pathway analysis. Bioinformatics. 2014;30(4):523–30.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3928520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24336805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kramer%20A,%20Green%20J,%20Pollard%20J%20Jr.,%20Tugendreich%20S.%20Causal%20analysis%20approaches%20in%20ingenuity%20pathway%20analysis.%20Bioinformatics.%202014;30(4):523%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Gillman R, Field MA, Schmitz U, Karamatic R, Hebbard L. Identifying cancer driver genes in individual tumours. Comput Struct Biotechnol J. 2023;21:5028–38.
</cite> [<a href="https://doi.org/10.1016/j.csbj.2023.10.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10589724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37867967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gillman%20R,%20Field%20MA,%20Schmitz%20U,%20Karamatic%20R,%20Hebbard%20L.%20Identifying%20cancer%20driver%20genes%20in%20individual%20tumours.%20Comput%20Struct%20Biotechnol%20J.%202023;21:5028%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>American Academy of Microbiology Colloquia Reports. In: Washington DC, editor. An experimental approach to genome annotation: this report is based on a colloquium sponsored by the American academy of microbiology held July 19–20, 2004. Washington (DC): American Society for Microbiology Copyright 2004 American Academy of Microbiology; 2004.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/33001599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?American%20Academy%20of%20Microbiology%20Colloquia%20Reports.%20In:%20Washington%20DC,%20editor.%20An%20experimental%20approach%20to%20genome%20annotation:%20this%20report%20is%20based%20on%20a%20colloquium%20sponsored%20by%20the%20American%20academy%20of%20microbiology%20held%20July%2019%E2%80%9320,%202004.%20Washington%20(DC):%20American%20Society%20for%20Microbiology%20Copyright%202004%20American%20Academy%20of%20Microbiology;%202004." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Tuteja S, Kadri S, Yap KL. A performance evaluation study: variant annotation tools - the enigma of clinical next generation sequencing (NGS) based genetic testing. J Pathol Inf. 2022;13:100130.</cite> [<a href="https://doi.org/10.1016/j.jpi.2022.100130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9577137/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36268089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tuteja%20S,%20Kadri%20S,%20Yap%20KL.%20A%20performance%20evaluation%20study:%20variant%20annotation%20tools%20-%20the%20enigma%20of%20clinical%20next%20generation%20sequencing%20(NGS)%20based%20genetic%20testing.%20J%20Pathol%20Inf.%202022;13:100130." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>McLaren W, Gil L, Hunt SE, Riat HS, Ritchie GR, Thormann A, et al. The ensembl variant effect predictor. Genome Biol. 2016;17(1):122.
</cite> [<a href="https://doi.org/10.1186/s13059-016-0974-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4893825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27268795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McLaren%20W,%20Gil%20L,%20Hunt%20SE,%20Riat%20HS,%20Ritchie%20GR,%20Thormann%20A,%20et%20al.%20The%20ensembl%20variant%20effect%20predictor.%20Genome%20Biol.%202016;17(1):122." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 2010;38(16):e164–e.
</cite> [<a href="https://doi.org/10.1093/nar/gkq603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2938201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20601685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20K,%20Li%20M,%20Hakonarson%20H.%20ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data.%20Nucleic%20Acids%20Res.%202010;38(16):e164%E2%80%93e." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Pranckėnienė L, Jakaitienė A, Ambrozaitytė L, Kavaliauskienė I, Kučinskas V. Insights into de Novo mutation variation in Lithuanian exome. Front Genet. 2018;9:315.
</cite> [<a href="https://doi.org/10.3389/fgene.2018.00315" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6102505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30154829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pranck%C4%97nien%C4%97%20L,%20Jakaitien%C4%97%20A,%20Ambrozaityt%C4%97%20L,%20Kavaliauskien%C4%97%20I,%20Ku%C4%8Dinskas%20V.%20Insights%20into%20de%20Novo%20mutation%20variation%20in%20Lithuanian%20exome.%20Front%20Genet.%202018;9:315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Vandeweyer G, Van Laer L, Loeys B, Van den Bulcke T, Kooy RF. VariantDB: a flexible annotation and filtering portal for next generation sequencing data. Genome Med. 2014;6(10):74.
</cite> [<a href="https://doi.org/10.1186/s13073-014-0074-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4210545/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25352915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vandeweyer%20G,%20Van%20Laer%20L,%20Loeys%20B,%20Van%20den%20Bulcke%20T,%20Kooy%20RF.%20VariantDB:%20a%20flexible%20annotation%20and%20filtering%20portal%20for%20next%20generation%20sequencing%20data.%20Genome%20Med.%202014;6(10):74." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Pais LS, Snow H, Weisburd B, Zhang S, Baxter SM, DiTroia S, et al. Seqr: A web-based analysis and collaboration tool for rare disease genomics. Hum Mutat. 2022;43(6):698–707.
</cite> [<a href="https://doi.org/10.1002/humu.24366" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9903206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35266241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pais%20LS,%20Snow%20H,%20Weisburd%20B,%20Zhang%20S,%20Baxter%20SM,%20DiTroia%20S,%20et%20al.%20Seqr:%20A%20web-based%20analysis%20and%20collaboration%20tool%20for%20rare%20disease%20genomics.%20Hum%20Mutat.%202022;43(6):698%E2%80%93707." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite>Wang R, Jiang Y, Jin J, Yin C, Yu H, Wang F, et al. DeepBIO: an automated and interpretable deep-learning platform for high-throughput biological sequence prediction, functional annotation and visualization analysis. Nucleic Acids Res. 2023;51(7):3017–29.
</cite> [<a href="https://doi.org/10.1093/nar/gkad055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10123094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36796796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20R,%20Jiang%20Y,%20Jin%20J,%20Yin%20C,%20Yu%20H,%20Wang%20F,%20et%20al.%20DeepBIO:%20an%20automated%20and%20interpretable%20deep-learning%20platform%20for%20high-throughput%20biological%20sequence%20prediction,%20functional%20annotation%20and%20visualization%20analysis.%20Nucleic%20Acids%20Res.%202023;51(7):3017%E2%80%9329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>Routhier E, Mozziconacci J. Genomics enters the deep learning era. PeerJ. 2022;10:e13613.
</cite> [<a href="https://doi.org/10.7717/peerj.13613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9235815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35769139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Routhier%20E,%20Mozziconacci%20J.%20Genomics%20enters%20the%20deep%20learning%20era.%20PeerJ.%202022;10:e13613." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR115">
<span class="label">115.</span><cite>Schaefer J, Lehne M, Schepers J, Prasser F, Thun S. The use of machine learning in rare diseases: a scoping review. Orphanet J Rare Dis. 2020;15:1–10.
</cite> [<a href="https://doi.org/10.1186/s13023-020-01424-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7285453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32517778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schaefer%20J,%20Lehne%20M,%20Schepers%20J,%20Prasser%20F,%20Thun%20S.%20The%20use%20of%20machine%20learning%20in%20rare%20diseases:%20a%20scoping%20review.%20Orphanet%20J%20Rare%20Dis.%202020;15:1%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR116">
<span class="label">116.</span><cite>Setty ST, Scott-Boyer M-P, Cuppens T, Droit A. New developments and possibilities in reanalysis and reinterpretation of whole exome sequencing datasets for unsolved rare diseases using machine learning approaches. Int J Mol Sci. 2022;23(12):6792.
</cite> [<a href="https://doi.org/10.3390/ijms23126792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9224427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35743235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Setty%20ST,%20Scott-Boyer%20M-P,%20Cuppens%20T,%20Droit%20A.%20New%20developments%20and%20possibilities%20in%20reanalysis%20and%20reinterpretation%20of%20whole%20exome%20sequencing%20datasets%20for%20unsolved%20rare%20diseases%20using%20machine%20learning%20approaches.%20Int%20J%20Mol%20Sci.%202022;23(12):6792." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Cohen AS, Farrow EG, Abdelmoity AT, Alaimo JT, Amudhavalli SM, Anderson JT, et al. Genomic answers for children: dynamic analyses of &gt; 1000 pediatric rare disease genomes. Genet Sci. 2022;24(6):1336–48.</cite> [<a href="https://doi.org/10.1016/j.gim.2022.02.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35305867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cohen%20AS,%20Farrow%20EG,%20Abdelmoity%20AT,%20Alaimo%20JT,%20Amudhavalli%20SM,%20Anderson%20JT,%20et%20al.%20Genomic%20answers%20for%20children:%20dynamic%20analyses%20of%20&gt;%E2%80%891000%20pediatric%20rare%20disease%20genomes.%20Genet%20Sci.%202022;24(6):1336%E2%80%9348." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>Okazaki A, Ott J. Machine learning approaches to explore digenic inheritance. Trends Genet. 2022;38(10):1013–8.
</cite> [<a href="https://doi.org/10.1016/j.tig.2022.04.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35581032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Okazaki%20A,%20Ott%20J.%20Machine%20learning%20approaches%20to%20explore%20digenic%20inheritance.%20Trends%20Genet.%202022;38(10):1013%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Miki Y, Swensen J, Shattuck-Eidens D, Futreal PA, Harshman K, Tavtigian S, et al. A strong candidate for the breast and ovarian cancer susceptibility gene BRCA1. Science. 1994;266(5182):66–71.
</cite> [<a href="https://doi.org/10.1126/science.7545954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7545954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miki%20Y,%20Swensen%20J,%20Shattuck-Eidens%20D,%20Futreal%20PA,%20Harshman%20K,%20Tavtigian%20S,%20et%20al.%20A%20strong%20candidate%20for%20the%20breast%20and%20ovarian%20cancer%20susceptibility%20gene%20BRCA1.%20Science.%201994;266(5182):66%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Aljarf R, Shen M, Pires DEV, Ascher DB. Understanding and predicting the functional consequences of missense mutations in BRCA1 and BRCA2. Sci Rep. 2022;12(1):10458.
</cite> [<a href="https://doi.org/10.1038/s41598-022-13508-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9213547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35729312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aljarf%20R,%20Shen%20M,%20Pires%20DEV,%20Ascher%20DB.%20Understanding%20and%20predicting%20the%20functional%20consequences%20of%20missense%20mutations%20in%20BRCA1%20and%20BRCA2.%20Sci%20Rep.%202022;12(1):10458." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Crockett DK, Piccolo SR, Ridge PG, Margraf RL, Lyon E, Williams MS, et al. Predicting phenotypic severity of uncertain gene variants in the RET proto-oncogene. PLoS ONE. 2011;6(3):e18380.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0018380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3068179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21479187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crockett%20DK,%20Piccolo%20SR,%20Ridge%20PG,%20Margraf%20RL,%20Lyon%20E,%20Williams%20MS,%20et%20al.%20Predicting%20phenotypic%20severity%20of%20uncertain%20gene%20variants%20in%20the%20RET%20proto-oncogene.%20PLoS%20ONE.%202011;6(3):e18380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR122">
<span class="label">122.</span><cite>Evans P, Wu C, Lindy A, McKnight DA, Lebo M, Sarmady M, et al. Genetic variant pathogenicity prediction trained using disease-specific clinical sequencing data sets. Genome Res. 2019;29(7):1144–51.
</cite> [<a href="https://doi.org/10.1101/gr.240994.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6633260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31235655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Evans%20P,%20Wu%20C,%20Lindy%20A,%20McKnight%20DA,%20Lebo%20M,%20Sarmady%20M,%20et%20al.%20Genetic%20variant%20pathogenicity%20prediction%20trained%20using%20disease-specific%20clinical%20sequencing%20data%20sets.%20Genome%20Res.%202019;29(7):1144%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR123">
<span class="label">123.</span><cite>Hart SN, Polley EC, Shimelis H, Yadav S, Couch FJ. Prediction of the functional impact of missense variants in BRCA1 and BRCA2 with BRCA-ML. NPJ Breast Cancer. 2020;6:13.
</cite> [<a href="https://doi.org/10.1038/s41523-020-0159-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7190647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32377563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hart%20SN,%20Polley%20EC,%20Shimelis%20H,%20Yadav%20S,%20Couch%20FJ.%20Prediction%20of%20the%20functional%20impact%20of%20missense%20variants%20in%20BRCA1%20and%20BRCA2%20with%20BRCA-ML.%20NPJ%20Breast%20Cancer.%202020;6:13." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR124">
<span class="label">124.</span><cite>Ioannidis NM, Rothstein JH, Pejaver V, Middha S, McDonnell SK, Baheti S, et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. Am J Hum Genet. 2016;99(4):877–85.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2016.08.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5065685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27666373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ioannidis%20NM,%20Rothstein%20JH,%20Pejaver%20V,%20Middha%20S,%20McDonnell%20SK,%20Baheti%20S,%20et%20al.%20REVEL:%20an%20ensemble%20method%20for%20predicting%20the%20pathogenicity%20of%20rare%20missense%20variants.%20Am%20J%20Hum%20Genet.%202016;99(4):877%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR125">
<span class="label">125.</span><cite>Lai C, Zimmer AD, O’Connor R, Kim S, Chan R, van den Akker J, et al. LEAP: using machine learning to support variant classification in a clinical setting. Hum Mutat. 2020;41(6):1079–90.
</cite> [<a href="https://doi.org/10.1002/humu.24011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7317941/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32176384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lai%20C,%20Zimmer%20AD,%20O%E2%80%99Connor%20R,%20Kim%20S,%20Chan%20R,%20van%20den%20Akker%20J,%20et%20al.%20LEAP:%20using%20machine%20learning%20to%20support%20variant%20classification%20in%20a%20clinical%20setting.%20Hum%20Mutat.%202020;41(6):1079%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR126">
<span class="label">126.</span><cite>Crockett DK, Lyon E, Williams MS, Narus SP, Facelli JC, Mitchell JA. Utility of gene-specific algorithms for predicting pathogenicity of uncertain gene variants. J Am Med Inf Assoc. 2012;19(2):207–11.</cite> [<a href="https://doi.org/10.1136/amiajnl-2011-000309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3277614/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22037892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crockett%20DK,%20Lyon%20E,%20Williams%20MS,%20Narus%20SP,%20Facelli%20JC,%20Mitchell%20JA.%20Utility%20of%20gene-specific%20algorithms%20for%20predicting%20pathogenicity%20of%20uncertain%20gene%20variants.%20J%20Am%20Med%20Inf%20Assoc.%202012;19(2):207%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR127">
<span class="label">127.</span><cite>Karalidou V, Kalfakakou D, Papathanasiou A, Fostira F, Matsopoulos GK. MARGINAL: an automatic classification of variants in BRCA1 and BRCA2 genes using a machine learning model. Biomolecules. 2022;12(11).</cite> [<a href="https://doi.org/10.3390/biom12111552" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9687470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36358902/" class="usa-link">PubMed</a>]</li>
<li id="CR128">
<span class="label">128.</span><cite>Khandakji MN, Mifsud B. Gene-specific machine learning model to predict the pathogenicity of BRCA2 variants. Front Genet. 2022;13:982930.
</cite> [<a href="https://doi.org/10.3389/fgene.2022.982930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9561395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36246618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khandakji%20MN,%20Mifsud%20B.%20Gene-specific%20machine%20learning%20model%20to%20predict%20the%20pathogenicity%20of%20BRCA2%20variants.%20Front%20Genet.%202022;13:982930." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR129">
<span class="label">129.</span><cite>Padilla N, Moles-Fernández A, Riera C, Montalban G, Özkan S, Ootes L, et al. BRCA1- and BRCA2-specific in Silico tools for variant interpretation in the CAGI 5 ENIGMA challenge. Hum Mutat. 2019;40(9):1593–611.
</cite> [<a href="https://doi.org/10.1002/humu.23802" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6744361/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31112341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Padilla%20N,%20Moles-Fern%C3%A1ndez%20A,%20Riera%20C,%20Montalban%20G,%20%C3%96zkan%20S,%20Ootes%20L,%20et%20al.%20BRCA1-%20and%20BRCA2-specific%20in%20Silico%20tools%20for%20variant%20interpretation%20in%20the%20CAGI%205%20ENIGMA%20challenge.%20Hum%20Mutat.%202019;40(9):1593%E2%80%93611." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR130">
<span class="label">130.</span><cite>Jagadeesh KA, Wenger AM, Berger MJ, Guturu H, Stenson PD, Cooper DN, et al. M-CAP eliminates a majority of variants of uncertain significance in clinical exomes at high sensitivity. Nat Genet. 2016;48(12):1581–6.
</cite> [<a href="https://doi.org/10.1038/ng.3703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27776117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jagadeesh%20KA,%20Wenger%20AM,%20Berger%20MJ,%20Guturu%20H,%20Stenson%20PD,%20Cooper%20DN,%20et%20al.%20M-CAP%20eliminates%20a%20majority%20of%20variants%20of%20uncertain%20significance%20in%20clinical%20exomes%20at%20high%20sensitivity.%20Nat%20Genet.%202016;48(12):1581%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR131">
<span class="label">131.</span><cite>Kang M, Kim S, Lee D-B, Hong C, Hwang K-B. Gene-specific machine learning for pathogenicity prediction of rare BRCA1 and BRCA2 missense variants. Sci Rep. 2023;13(1):10478.
</cite> [<a href="https://doi.org/10.1038/s41598-023-37698-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10307865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37380723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kang%20M,%20Kim%20S,%20Lee%20D-B,%20Hong%20C,%20Hwang%20K-B.%20Gene-specific%20machine%20learning%20for%20pathogenicity%20prediction%20of%20rare%20BRCA1%20and%20BRCA2%20missense%20variants.%20Sci%20Rep.%202023;13(1):10478." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR132">
<span class="label">132.</span><cite>Feng BJ. PERCH: A unified framework for disease gene prioritization. Hum Mutat. 2017;38(3):243–51.
</cite> [<a href="https://doi.org/10.1002/humu.23158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5299048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27995669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng%20BJ.%20PERCH:%20A%20unified%20framework%20for%20disease%20gene%20prioritization.%20Hum%20Mutat.%202017;38(3):243%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR133">
<span class="label">133.</span><cite>Alirezaie N, Kernohan KD, Hartley T, Majewski J, Hocking TD. ClinPred: prediction tool to identify Disease-Relevant nonsynonymous Single-Nucleotide variants. Am J Hum Genet. 2018;103(4):474–83.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2018.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6174354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30220433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alirezaie%20N,%20Kernohan%20KD,%20Hartley%20T,%20Majewski%20J,%20Hocking%20TD.%20ClinPred:%20prediction%20tool%20to%20identify%20Disease-Relevant%20nonsynonymous%20Single-Nucleotide%20variants.%20Am%20J%20Hum%20Genet.%202018;103(4):474%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR134">
<span class="label">134.</span><cite>Poplin R, Chang P-C, Alexander D, Schwartz S, Colthurst T, Ku A, et al. A universal SNP and small-indel variant caller using deep neural networks. Nat Biotechnol. 2018;36(10):983–7.
</cite> [<a href="https://doi.org/10.1038/nbt.4235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30247488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poplin%20R,%20Chang%20P-C,%20Alexander%20D,%20Schwartz%20S,%20Colthurst%20T,%20Ku%20A,%20et%20al.%20A%20universal%20SNP%20and%20small-indel%20variant%20caller%20using%20deep%20neural%20networks.%20Nat%20Biotechnol.%202018;36(10):983%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR135">
<span class="label">135.</span><cite>Nicora G, Zucca S, Limongelli I, Bellazzi R, Magni P. A machine learning approach based on ACMG/AMP guidelines for genomic variant classification and prioritization. Sci Rep. 2022;12(1):2517.
</cite> [<a href="https://doi.org/10.1038/s41598-022-06547-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8847497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35169226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nicora%20G,%20Zucca%20S,%20Limongelli%20I,%20Bellazzi%20R,%20Magni%20P.%20A%20machine%20learning%20approach%20based%20on%20ACMG/AMP%20guidelines%20for%20genomic%20variant%20classification%20and%20prioritization.%20Sci%20Rep.%202022;12(1):2517." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR136">
<span class="label">136.</span><cite>Danzi MC, Dohrn MF, Fazal S, Beijer D, Rebelo AP, Cintra V, et al. Deep structured learning for variant prioritization in Mendelian diseases. Nat Commun. 2023;14(1):4167.
</cite> [<a href="https://doi.org/10.1038/s41467-023-39306-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10345112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37443090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danzi%20MC,%20Dohrn%20MF,%20Fazal%20S,%20Beijer%20D,%20Rebelo%20AP,%20Cintra%20V,%20et%20al.%20Deep%20structured%20learning%20for%20variant%20prioritization%20in%20Mendelian%20diseases.%20Nat%20Commun.%202023;14(1):4167." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR137">
<span class="label">137.</span><cite>Ohler U, Liao G-c, Niemann H, Rubin GM. Computational analysis of core promoters in the drosophila genome. Genome Biol. 2002;3(12):research00871.</cite> [<a href="https://doi.org/10.1186/gb-2002-3-12-research0087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC151189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12537576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ohler%20U,%20Liao%20G-c,%20Niemann%20H,%20Rubin%20GM.%20Computational%20analysis%20of%20core%20promoters%20in%20the%20drosophila%20genome.%20Genome%20Biol.%202002;3(12):research00871." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR138">
<span class="label">138.</span><cite>Degroeve S, De Baets B, Van de Peer Y, Rouzé P. Feature subset selection for splice site prediction. Bioinformatics. 2002;18(Suppl 2):S75–83.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/18.suppl_2.s75" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12385987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Degroeve%20S,%20De%20Baets%20B,%20Van%20de%20Peer%20Y,%20Rouz%C3%A9%20P.%20Feature%20subset%20selection%20for%20splice%20site%20prediction.%20Bioinformatics.%202002;18(Suppl%202):S75%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR139">
<span class="label">139.</span><cite>Bucher P. Weight matrix descriptions of four eukaryotic RNA polymerase II promoter elements derived from 502 unrelated promoter sequences. J Mol Biol. 1990;212(4):563–78.
</cite> [<a href="https://doi.org/10.1016/0022-2836(90)90223-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2329577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bucher%20P.%20Weight%20matrix%20descriptions%20of%20four%20eukaryotic%20RNA%20polymerase%20II%20promoter%20elements%20derived%20from%20502%20unrelated%20promoter%20sequences.%20J%20Mol%20Biol.%201990;212(4):563%E2%80%9378." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR140">
<span class="label">140.</span><cite>Heintzman ND, Stuart RK, Hon G, Fu Y, Ching CW, Hawkins RD, et al. Distinct and predictive chromatin signatures of transcriptional promoters and enhancers in the human genome. Nat Genet. 2007;39(3):311–8.
</cite> [<a href="https://doi.org/10.1038/ng1966" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17277777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heintzman%20ND,%20Stuart%20RK,%20Hon%20G,%20Fu%20Y,%20Ching%20CW,%20Hawkins%20RD,%20et%20al.%20Distinct%20and%20predictive%20chromatin%20signatures%20of%20transcriptional%20promoters%20and%20enhancers%20in%20the%20human%20genome.%20Nat%20Genet.%202007;39(3):311%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR141">
<span class="label">141.</span><cite>Segal E, Fondufe-Mittendorf Y, Chen L, Thåström A, Field Y, Moore IK, et al. A genomic code for nucleosome positioning. Nature. 2006;442(7104):772–8.
</cite> [<a href="https://doi.org/10.1038/nature04979" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2623244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16862119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Segal%20E,%20Fondufe-Mittendorf%20Y,%20Chen%20L,%20Th%C3%A5str%C3%B6m%20A,%20Field%20Y,%20Moore%20IK,%20et%20al.%20A%20genomic%20code%20for%20nucleosome%20positioning.%20Nature.%202006;442(7104):772%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR142">
<span class="label">142.</span><cite>Akbarzadeh M, Dehkordi SR, Roudbar MA, Sargolzaei M, Guity K, Sedaghati-khayat B, et al. GWAS findings improved genomic prediction accuracy of lipid profile traits: Tehran cardiometabolic genetic study. Sci Rep. 2021;11(1):5780.
</cite> [<a href="https://doi.org/10.1038/s41598-021-85203-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7952573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33707626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akbarzadeh%20M,%20Dehkordi%20SR,%20Roudbar%20MA,%20Sargolzaei%20M,%20Guity%20K,%20Sedaghati-khayat%20B,%20et%20al.%20GWAS%20findings%20improved%20genomic%20prediction%20accuracy%20of%20lipid%20profile%20traits:%20Tehran%20cardiometabolic%20genetic%20study.%20Sci%20Rep.%202021;11(1):5780." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR143">
<span class="label">143.</span><cite>Mittag F, Büchel F, Saad M, Jahn A, Schulte C, Bochdanovits Z, et al. Use of support vector machines for disease risk prediction in genome-wide association studies: concerns and opportunities. Hum Mutat. 2012;33(12):1708–18.
</cite> [<a href="https://doi.org/10.1002/humu.22161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5968822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22777693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mittag%20F,%20B%C3%BCchel%20F,%20Saad%20M,%20Jahn%20A,%20Schulte%20C,%20Bochdanovits%20Z,%20et%20al.%20Use%20of%20support%20vector%20machines%20for%20disease%20risk%20prediction%20in%20genome-wide%20association%20studies:%20concerns%20and%20opportunities.%20Hum%20Mutat.%202012;33(12):1708%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR144">
<span class="label">144.</span><cite>Guo Y, Wu C, Yuan Z, Wang Y, Liang Z, Wang Y, et al. Gene-Based testing of interactions using XGBoost in Genome-Wide association studies. Front Cell Dev Biol. 2021;9:801113.
</cite> [<a href="https://doi.org/10.3389/fcell.2021.801113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8716787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34977040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20Y,%20Wu%20C,%20Yuan%20Z,%20Wang%20Y,%20Liang%20Z,%20Wang%20Y,%20et%20al.%20Gene-Based%20testing%20of%20interactions%20using%20XGBoost%20in%20Genome-Wide%20association%20studies.%20Front%20Cell%20Dev%20Biol.%202021;9:801113." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR145">
<span class="label">145.</span><cite>Maples BK, Gravel S, Kenny EE, Bustamante CD. RFMix: a discriminative modeling approach for rapid and robust local-ancestry inference. Am J Hum Genet. 2013;93(2):278–88.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2013.06.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3738819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23910464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maples%20BK,%20Gravel%20S,%20Kenny%20EE,%20Bustamante%20CD.%20RFMix:%20a%20discriminative%20modeling%20approach%20for%20rapid%20and%20robust%20local-ancestry%20inference.%20Am%20J%20Hum%20Genet.%202013;93(2):278%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR146">
<span class="label">146.</span><cite>Badré A, Zhang L, Muchero W, Reynolds JC, Pan C. Deep neural network improves the Estimation of polygenic risk scores for breast cancer. J Hum Genet. 2021;66(4):359–69.
</cite> [<a href="https://doi.org/10.1038/s10038-020-00832-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33009504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Badr%C3%A9%20A,%20Zhang%20L,%20Muchero%20W,%20Reynolds%20JC,%20Pan%20C.%20Deep%20neural%20network%20improves%20the%20Estimation%20of%20polygenic%20risk%20scores%20for%20breast%20cancer.%20J%20Hum%20Genet.%202021;66(4):359%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR147">
<span class="label">147.</span><cite>Twede H, Conard AM, Pais L, Bryen S, O’Heir E, Smith G et al. Evidence Aggregator: AI reasoning applied to rare disease diagnostics. bioRxiv. 2025:2025.03.10.642480.</cite>
</li>
<li id="CR148">
<span class="label">148.</span><cite>Fowler DM, Rehm HL. Will variants of uncertain significance still exist in 2030? Am J Hum Genet. 2024;111(1):5–10.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2023.11.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10806733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38086381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fowler%20DM,%20Rehm%20HL.%20Will%20variants%20of%20uncertain%20significance%20still%20exist%20in%202030?%20Am%20J%20Hum%20Genet.%202024;111(1):5%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR149">
<span class="label">149.</span><cite>Benary M, Wang XD, Schmidt M, Soll D, Hilfenhaus G, Nassir M, et al. Leveraging large Language models for decision support in personalized oncology. JAMA Netw Open. 2023;6(11):e2343689.
</cite> [<a href="https://doi.org/10.1001/jamanetworkopen.2023.43689" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10656647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37976064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Benary%20M,%20Wang%20XD,%20Schmidt%20M,%20Soll%20D,%20Hilfenhaus%20G,%20Nassir%20M,%20et%20al.%20Leveraging%20large%20Language%20models%20for%20decision%20support%20in%20personalized%20oncology.%20JAMA%20Netw%20Open.%202023;6(11):e2343689." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR150">
<span class="label">150.</span><cite>McInnes G, Sharo AG, Koleske ML, Brown JEH, Norstad M, Adhikari AN, et al. Opportunities and challenges for the computational interpretation of rare variation in clinically important genes. Am J Hum Genet. 2021;108(4):535–48.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2021.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8059338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33798442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McInnes%20G,%20Sharo%20AG,%20Koleske%20ML,%20Brown%20JEH,%20Norstad%20M,%20Adhikari%20AN,%20et%20al.%20Opportunities%20and%20challenges%20for%20the%20computational%20interpretation%20of%20rare%20variation%20in%20clinically%20important%20genes.%20Am%20J%20Hum%20Genet.%202021;108(4):535%E2%80%9348." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR151">
<span class="label">151.</span><cite>Nicholls HL, John CR, Watson DS, Munroe PB, Barnes MR, Cabrera CP. Reaching the End-Game for GWAS: machine learning approaches for the prioritization of complex disease loci. Front Genet. 2020;11:350.
</cite> [<a href="https://doi.org/10.3389/fgene.2020.00350" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7174742/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32351543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nicholls%20HL,%20John%20CR,%20Watson%20DS,%20Munroe%20PB,%20Barnes%20MR,%20Cabrera%20CP.%20Reaching%20the%20End-Game%20for%20GWAS:%20machine%20learning%20approaches%20for%20the%20prioritization%20of%20complex%20disease%20loci.%20Front%20Genet.%202020;11:350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR152">
<span class="label">152.</span><cite>Tufail S, Riggs H, Tariq M, Sarwat AI. Advancements and challenges in machine learning: A comprehensive review of models, libraries, applications, and algorithms. Electronics. 2023;12(8):1789.</cite> [<a href="https://scholar.google.com/scholar_lookup?Tufail%20S,%20Riggs%20H,%20Tariq%20M,%20Sarwat%20AI.%20Advancements%20and%20challenges%20in%20machine%20learning:%20A%20comprehensive%20review%20of%20models,%20libraries,%20applications,%20and%20algorithms.%20Electronics.%202023;12(8):1789." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR153">
<span class="label">153.</span><cite>Mirza B, Wang W, Wang J, Choi H, Chung NC, Ping P. Machine learning and integrative analysis of biomedical big data. Genes (Basel). 2019;10(2).</cite> [<a href="https://doi.org/10.3390/genes10020087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6410075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30696086/" class="usa-link">PubMed</a>]</li>
<li id="CR154">
<span class="label">154.</span><cite>Wang R, Chaudhari P, Davatzikos C. Bias in machine learning models can be significantly mitigated by careful training: evidence from neuroimaging studies. Proc Natl Acad Sci. 2023;120(6):e2211613120.
</cite> [<a href="https://doi.org/10.1073/pnas.2211613120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9962919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36716365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20R,%20Chaudhari%20P,%20Davatzikos%20C.%20Bias%20in%20machine%20learning%20models%20can%20be%20significantly%20mitigated%20by%20careful%20training:%20evidence%20from%20neuroimaging%20studies.%20Proc%20Natl%20Acad%20Sci.%202023;120(6):e2211613120." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR155">
<span class="label">155.</span><cite>Dwarshuis N, Tonner P, Olson ND, Sedlazeck FJ, Wagner J, Zook JM. StratoMod: predicting sequencing and variant calling errors with interpretable machine learning. Commun Biology. 2024;7(1):1316.</cite> [<a href="https://doi.org/10.1038/s42003-024-06981-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11471861/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39397114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dwarshuis%20N,%20Tonner%20P,%20Olson%20ND,%20Sedlazeck%20FJ,%20Wagner%20J,%20Zook%20JM.%20StratoMod:%20predicting%20sequencing%20and%20variant%20calling%20errors%20with%20interpretable%20machine%20learning.%20Commun%20Biology.%202024;7(1):1316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR156">
<span class="label">156.</span><cite>Lee Y-S, Yen S-J, Jiang W, Chen J, Chang C-Y. Illuminating the black box: an interpretable machine learning based on ensemble trees. Expert Syst Appl. 2025;272:126720.</cite> [<a href="https://scholar.google.com/scholar_lookup?Lee%20Y-S,%20Yen%20S-J,%20Jiang%20W,%20Chen%20J,%20Chang%20C-Y.%20Illuminating%20the%20black%20box:%20an%20interpretable%20machine%20learning%20based%20on%20ensemble%20trees.%20Expert%20Syst%20Appl.%202025;272:126720." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR157">
<span class="label">157.</span><cite>Karim MR, Islam T, Shajalal M, Beyan O, Lange C, Cochez M et al. Explainable AI for bioinformatics: methods, tools and applications. Brief Bioinform. 2023;24(5).</cite> [<a href="https://doi.org/10.1093/bib/bbad236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37478371/" class="usa-link">PubMed</a>]</li>
<li id="CR158">
<span class="label">158.</span><cite>Patharkar A, Cai F, Al-Hindawi F, Wu T. Predictive modeling of biomedical Temporal data in healthcare applications: review and future directions. Front Physiol. 2024;15:1386760.
</cite> [<a href="https://doi.org/10.3389/fphys.2024.1386760" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11519529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39473609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patharkar%20A,%20Cai%20F,%20Al-Hindawi%20F,%20Wu%20T.%20Predictive%20modeling%20of%20biomedical%20Temporal%20data%20in%20healthcare%20applications:%20review%20and%20future%20directions.%20Front%20Physiol.%202024;15:1386760." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR159">
<span class="label">159.</span><cite>Field MA. Bioinformatic challenges detecting genetic variation in precision medicine programs. Front Med. 2022;9.</cite> [<a href="https://doi.org/10.3389/fmed.2022.806696" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9024231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35463004/" class="usa-link">PubMed</a>]</li>
<li id="CR160">
<span class="label">160.</span><cite>von Gerich H, Chomutare T, Peltonen LM. Building bridges for federated learning in healthcare: review on approaches for common data model development. Stud Health Technol Inf. 2024;315:711–2.</cite> [<a href="https://doi.org/10.3233/SHTI240292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39049393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?von%20Gerich%20H,%20Chomutare%20T,%20Peltonen%20LM.%20Building%20bridges%20for%20federated%20learning%20in%20healthcare:%20review%20on%20approaches%20for%20common%20data%20model%20development.%20Stud%20Health%20Technol%20Inf.%202024;315:711%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR161">
<span class="label">161.</span><cite>Wilkinson MD, Dumontier M, Aalbersberg IJ, Appleton G, Axton M, Baak A, et al. The FAIR guiding principles for scientific data management and stewardship. Sci Data. 2016;3(1):160018.
</cite> [<a href="https://doi.org/10.1038/sdata.2016.18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4792175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26978244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilkinson%20MD,%20Dumontier%20M,%20Aalbersberg%20IJ,%20Appleton%20G,%20Axton%20M,%20Baak%20A,%20et%20al.%20The%20FAIR%20guiding%20principles%20for%20scientific%20data%20management%20and%20stewardship.%20Sci%20Data.%202016;3(1):160018." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR162">
<span class="label">162.</span><cite>Rajkomar A, Oren E, Chen K, Dai AM, Hajaj N, Hardt M, et al. Scalable and accurate deep learning with electronic health records. Npj Digit Med. 2018;1(1):18.
</cite> [<a href="https://doi.org/10.1038/s41746-018-0029-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6550175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31304302/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rajkomar%20A,%20Oren%20E,%20Chen%20K,%20Dai%20AM,%20Hajaj%20N,%20Hardt%20M,%20et%20al.%20Scalable%20and%20accurate%20deep%20learning%20with%20electronic%20health%20records.%20Npj%20Digit%20Med.%202018;1(1):18." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR163">
<span class="label">163.</span><cite>Chen IY, Pierson E, Rose S, Joshi S, Ferryman K, Ghassemi M. Ethical machine learning in healthcare. Annu Rev Biomed Data Sci. 2021;4:123–44.
</cite> [<a href="https://doi.org/10.1146/annurev-biodatasci-092820-114757" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8362902/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34396058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20IY,%20Pierson%20E,%20Rose%20S,%20Joshi%20S,%20Ferryman%20K,%20Ghassemi%20M.%20Ethical%20machine%20learning%20in%20healthcare.%20Annu%20Rev%20Biomed%20Data%20Sci.%202021;4:123%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR164">
<span class="label">164.</span><cite>Waardenberg AJ, Field MA. ConsensusDE: an R package for assessing consensus of multiple RNA-seq algorithms with RUV correction. PeerJ. 2019;7:e8206.
</cite> [<a href="https://doi.org/10.7717/peerj.8206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6913255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31844586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waardenberg%20AJ,%20Field%20MA.%20ConsensusDE:%20an%20R%20package%20for%20assessing%20consensus%20of%20multiple%20RNA-seq%20algorithms%20with%20RUV%20correction.%20PeerJ.%202019;7:e8206." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR165">
<span class="label">165.</span><cite>Iossifov I, O’Roak BJ, Sanders SJ, Ronemus M, Krumm N, Levy D, et al. The contribution of de Novo coding mutations to autism spectrum disorder. Nature. 2014;515(7526):216–21.
</cite> [<a href="https://doi.org/10.1038/nature13908" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4313871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25363768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Iossifov%20I,%20O%E2%80%99Roak%20BJ,%20Sanders%20SJ,%20Ronemus%20M,%20Krumm%20N,%20Levy%20D,%20et%20al.%20The%20contribution%20of%20de%20Novo%20coding%20mutations%20to%20autism%20spectrum%20disorder.%20Nature.%202014;515(7526):216%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR166">
<span class="label">166.</span><cite>Sanders SJ, Murtha MT, Gupta AR, Murdoch JD, Raubeson MJ, Willsey AJ, et al. De Novo mutations revealed by whole-exome sequencing are strongly associated with autism. Nature. 2012;485(7397):237–41.
</cite> [<a href="https://doi.org/10.1038/nature10945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3667984/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22495306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanders%20SJ,%20Murtha%20MT,%20Gupta%20AR,%20Murdoch%20JD,%20Raubeson%20MJ,%20Willsey%20AJ,%20et%20al.%20De%20Novo%20mutations%20revealed%20by%20whole-exome%20sequencing%20are%20strongly%20associated%20with%20autism.%20Nature.%202012;485(7397):237%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR167">
<span class="label">167.</span><cite>CAG Repeat Not. Polyglutamine length determines timing of huntington’s disease onset. Cell. 2019;178(4):887–e90014.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.06.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6700281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31398342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?CAG%20Repeat%20Not.%20Polyglutamine%20length%20determines%20timing%20of%20huntington%E2%80%99s%20disease%20onset.%20Cell.%202019;178(4):887%E2%80%93e90014." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR168">
<span class="label">168.</span><cite>Field MA. Detecting pathogenic variants in autoimmune diseases using high-throughput sequencing. Immunol Cell Biol. 2020.</cite> [<a href="https://doi.org/10.1111/imcb.12372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7891608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32623783/" class="usa-link">PubMed</a>]</li>
<li id="CR169">
<span class="label">169.</span><cite>Jiang SH, Mercan S, Papa I, Moldovan M, Walters GD, Koina M, et al. Deletions in VANGL1 are a risk factor for antibody-mediated kidney disease. Cell Rep Med. 2021;2(12):100475.
</cite> [<a href="https://doi.org/10.1016/j.xcrm.2021.100475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8714939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35028616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20SH,%20Mercan%20S,%20Papa%20I,%20Moldovan%20M,%20Walters%20GD,%20Koina%20M,%20et%20al.%20Deletions%20in%20VANGL1%20are%20a%20risk%20factor%20for%20antibody-mediated%20kidney%20disease.%20Cell%20Rep%20Med.%202021;2(12):100475." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR170">
<span class="label">170.</span><cite>Aartsma-Rus A, Krieg AM. FDA approves Eteplirsen for Duchenne muscular dystrophy: the next chapter in the Eteplirsen Saga. Nucleic Acid Ther. 2017;27(1):1–3.
</cite> [<a href="https://doi.org/10.1089/nat.2016.0657" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5312460/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27929755/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aartsma-Rus%20A,%20Krieg%20AM.%20FDA%20approves%20Eteplirsen%20for%20Duchenne%20muscular%20dystrophy:%20the%20next%20chapter%20in%20the%20Eteplirsen%20Saga.%20Nucleic%20Acid%20Ther.%202017;27(1):1%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR171">
<span class="label">171.</span><cite>Tsimberidou AM, Iskander NG, Hong DS, Wheler JJ, Falchook GS, Fu S, et al. Personalized medicine in a phase I clinical trials program: the MD Anderson cancer center initiative. Clin Cancer Res. 2012;18(22):6373–83.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-12-1627" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4454458/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22966018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsimberidou%20AM,%20Iskander%20NG,%20Hong%20DS,%20Wheler%20JJ,%20Falchook%20GS,%20Fu%20S,%20et%20al.%20Personalized%20medicine%20in%20a%20phase%20I%20clinical%20trials%20program:%20the%20MD%20Anderson%20cancer%20center%20initiative.%20Clin%20Cancer%20Res.%202012;18(22):6373%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR172">
<span class="label">172.</span><cite>Carter TC, He MM. Challenges of Identifying Clinically Actionable Genetic Variants for Precision Medicine. J Healthc Eng. 2016;2016.</cite> [<a href="https://doi.org/10.1155/2016/3617572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4955563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27195526/" class="usa-link">PubMed</a>]</li>
<li id="CR173">
<span class="label">173.</span><cite>Wang Z, Zhao G, Zhu Z, Wang Y, Xiang X, Zhang S, et al. VarCards2: an integrated genetic and clinical database for ACMG-AMP variant-interpretation guidelines in the human whole genome. Nucleic Acids Res. 2024;52(D1):D1478–89.
</cite> [<a href="https://doi.org/10.1093/nar/gkad1061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37956311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Z,%20Zhao%20G,%20Zhu%20Z,%20Wang%20Y,%20Xiang%20X,%20Zhang%20S,%20et%20al.%20VarCards2:%20an%20integrated%20genetic%20and%20clinical%20database%20for%20ACMG-AMP%20variant-interpretation%20guidelines%20in%20the%20human%20whole%20genome.%20Nucleic%20Acids%20Res.%202024;52(D1):D1478%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR174">
<span class="label">174.</span><cite>Richards S, Aziz N, Bale S, Bick D, Das S, Gastier-Foster J, et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American college of medical genetics and genomics and the association for molecular pathology. Genet Med. 2015;17(5):405–24.
</cite> [<a href="https://doi.org/10.1038/gim.2015.30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4544753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25741868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Richards%20S,%20Aziz%20N,%20Bale%20S,%20Bick%20D,%20Das%20S,%20Gastier-Foster%20J,%20et%20al.%20Standards%20and%20guidelines%20for%20the%20interpretation%20of%20sequence%20variants:%20a%20joint%20consensus%20recommendation%20of%20the%20American%20college%20of%20medical%20genetics%20and%20genomics%20and%20the%20association%20for%20molecular%20pathology.%20Genet%20Med.%202015;17(5):405%E2%80%9324." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR175">
<span class="label">175.</span><cite>Amendola LM, Jarvik GP, Leo MC, McLaughlin HM, Akkari Y, Amaral MD, et al. Performance of ACMG-AMP Variant-Interpretation guidelines among nine laboratories in the clinical sequencing exploratory research consortium. Am J Hum Genet. 2016;98(6):1067–76.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2016.03.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4908185/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27181684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amendola%20LM,%20Jarvik%20GP,%20Leo%20MC,%20McLaughlin%20HM,%20Akkari%20Y,%20Amaral%20MD,%20et%20al.%20Performance%20of%20ACMG-AMP%20Variant-Interpretation%20guidelines%20among%20nine%20laboratories%20in%20the%20clinical%20sequencing%20exploratory%20research%20consortium.%20Am%20J%20Hum%20Genet.%202016;98(6):1067%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR176">
<span class="label">176.</span><cite>Harrison SM, Riggs ER, Maglott DR, Lee JM, Azzariti DR, Niehaus A et al. Using ClinVar as a Resource to Support Variant Interpretation. Curr Protoc Hum Genet. 2016;89:8.16.1-8.23.</cite> [<a href="https://doi.org/10.1002/0471142905.hg0816s89" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4832236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27037489/" class="usa-link">PubMed</a>]</li>
<li id="CR177">
<span class="label">177.</span><cite>Rehm HL, Page AJH, Smith L, Adams JB, Alterovitz G, Babb LJ et al. GA4GH: international policies and standards for data sharing across genomic research and healthcare. Cell Genom. 2021;1(2).</cite> [<a href="https://doi.org/10.1016/j.xgen.2021.100029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8774288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35072136/" class="usa-link">PubMed</a>]</li>
<li id="CR178">
<span class="label">178.</span><cite>Campbell MC, Tishkoff SA. African genetic diversity: implications for human demographic history, modern human origins, and complex disease mapping. Annu Rev Genomics Hum Genet. 2008;9:403–33.
</cite> [<a href="https://doi.org/10.1146/annurev.genom.9.081307.164258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2953791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18593304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Campbell%20MC,%20Tishkoff%20SA.%20African%20genetic%20diversity:%20implications%20for%20human%20demographic%20history,%20modern%20human%20origins,%20and%20complex%20disease%20mapping.%20Annu%20Rev%20Genomics%20Hum%20Genet.%202008;9:403%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR179">
<span class="label">179.</span><cite>Auton A, Abecasis GR, Altshuler DM, Durbin RM, Abecasis GR, Bentley DR, et al. A global reference for human genetic variation. Nature. 2015;526(7571):68–74.
</cite> [<a href="https://doi.org/10.1038/nature15393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4750478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26432245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Auton%20A,%20Abecasis%20GR,%20Altshuler%20DM,%20Durbin%20RM,%20Abecasis%20GR,%20Bentley%20DR,%20et%20al.%20A%20global%20reference%20for%20human%20genetic%20variation.%20Nature.%202015;526(7571):68%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR180">
<span class="label">180.</span><cite>Rosas LG, Nasrallah C, Park VT, Vasquez JJ, Duron Y, Garrick O, et al. Perspectives on precision health among racial/ethnic minority communities and the physicians that serve them. Ethn Dis. 2020;30(Suppl 1):137–48.
</cite> [<a href="https://doi.org/10.18865/ed.30.S1.137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7138446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32269455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosas%20LG,%20Nasrallah%20C,%20Park%20VT,%20Vasquez%20JJ,%20Duron%20Y,%20Garrick%20O,%20et%20al.%20Perspectives%20on%20precision%20health%20among%20racial/ethnic%20minority%20communities%20and%20the%20physicians%20that%20serve%20them.%20Ethn%20Dis.%202020;30(Suppl%201):137%E2%80%9348." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR181">
<span class="label">181.</span><cite>Ienca M, Ferretti A, Hurst S, Puhan M, Lovis C, Vayena E. Considerations for ethics review of big data health research: A scoping review. PLoS ONE. 2018;13(10):e0204937.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0204937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6181558/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30308031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ienca%20M,%20Ferretti%20A,%20Hurst%20S,%20Puhan%20M,%20Lovis%20C,%20Vayena%20E.%20Considerations%20for%20ethics%20review%20of%20big%20data%20health%20research:%20A%20scoping%20review.%20PLoS%20ONE.%202018;13(10):e0204937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR182">
<span class="label">182.</span><cite>Subramanian I, Verma S, Kumar S, Jere A, Anamika K. Multi-omics data integration, interpretation, and its application. Bioinform Biol Insights. 2020;14:1177932219899051.
</cite> [<a href="https://doi.org/10.1177/1177932219899051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7003173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32076369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Subramanian%20I,%20Verma%20S,%20Kumar%20S,%20Jere%20A,%20Anamika%20K.%20Multi-omics%20data%20integration,%20interpretation,%20and%20its%20application.%20Bioinform%20Biol%20Insights.%202020;14:1177932219899051." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR183">
<span class="label">183.</span><cite>Thang MWC, Chua XY, Price G, Gorse D, Field MA. MetaDEGalaxy: galaxy workflow for differential abundance analysis of 16s metagenomic data. F1000Res. 2019;8:726.
</cite> [<a href="https://doi.org/10.12688/f1000research.18866.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6807170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31737256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thang%20MWC,%20Chua%20XY,%20Price%20G,%20Gorse%20D,%20Field%20MA.%20MetaDEGalaxy:%20galaxy%20workflow%20for%20differential%20abundance%20analysis%20of%2016s%20metagenomic%20data.%20F1000Res.%202019;8:726." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR184">
<span class="label">184.</span><cite>Khudhair Z, Alhallaf R, Eichenberger RM, Field M, Krause L, Sotillo J et al. Administration of hookworm excretory/secretory proteins improves glucose tolerance in a mouse model of type 2 diabetes. Biomolecules. 2022;12(5).</cite> [<a href="https://doi.org/10.3390/biom12050637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9138508/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35625566/" class="usa-link">PubMed</a>]</li>
<li id="CR185">
<span class="label">185.</span><cite>Pierce DR, McDonald M, Merone L, Becker L, Thompson F, Lewis C, et al. Effect of experimental hookworm infection on insulin resistance in people at risk of type 2 diabetes. Nat Commun. 2023;14(1):4503.
</cite> [<a href="https://doi.org/10.1038/s41467-023-40263-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10372076/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37495576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pierce%20DR,%20McDonald%20M,%20Merone%20L,%20Becker%20L,%20Thompson%20F,%20Lewis%20C,%20et%20al.%20Effect%20of%20experimental%20hookworm%20infection%20on%20insulin%20resistance%20in%20people%20at%20risk%20of%20type%202%20diabetes.%20Nat%20Commun.%202023;14(1):4503." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR186">
<span class="label">186.</span><cite>Khudhair Z, Alhallaf R, Eichenberger RM, Whan J, Kupz A, Field M, et al. Gastrointestinal helminth infection improves insulin sensitivity, decreases systemic inflammation, and alters the composition of gut microbiota in distinct mouse models of type 2 diabetes. Front Endocrinol (Lausanne). 2020;11:606530.
</cite> [<a href="https://doi.org/10.3389/fendo.2020.606530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7892786/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33613446/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khudhair%20Z,%20Alhallaf%20R,%20Eichenberger%20RM,%20Whan%20J,%20Kupz%20A,%20Field%20M,%20et%20al.%20Gastrointestinal%20helminth%20infection%20improves%20insulin%20sensitivity,%20decreases%20systemic%20inflammation,%20and%20alters%20the%20composition%20of%20gut%20microbiota%20in%20distinct%20mouse%20models%20of%20type%202%20diabetes.%20Front%20Endocrinol%20(Lausanne).%202020;11:606530." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR187">
<span class="label">187.</span><cite>Yan J, Risacher SL, Shen L, Saykin AJ. Network approaches to systems biology analysis of complex disease: integrative methods for multi-omics data. Brief Bioinform. 2018;19(6):1370–81.
</cite> [<a href="https://doi.org/10.1093/bib/bbx066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6454489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28679163/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan%20J,%20Risacher%20SL,%20Shen%20L,%20Saykin%20AJ.%20Network%20approaches%20to%20systems%20biology%20analysis%20of%20complex%20disease:%20integrative%20methods%20for%20multi-omics%20data.%20Brief%20Bioinform.%202018;19(6):1370%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR188">
<span class="label">188.</span><cite>Hasin Y, Seldin M, Lusis A. Multi-omics approaches to disease. Genome Biol. 2017;18(1):83.
</cite> [<a href="https://doi.org/10.1186/s13059-017-1215-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5418815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28476144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hasin%20Y,%20Seldin%20M,%20Lusis%20A.%20Multi-omics%20approaches%20to%20disease.%20Genome%20Biol.%202017;18(1):83." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR189">
<span class="label">189.</span><cite>Rappoport N, Shamir R. Multi-omic and multi-view clustering algorithms: review and cancer benchmark. Nucleic Acids Res. 2018;46(20):10546–62.
</cite> [<a href="https://doi.org/10.1093/nar/gky889" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6237755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30295871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rappoport%20N,%20Shamir%20R.%20Multi-omic%20and%20multi-view%20clustering%20algorithms:%20review%20and%20cancer%20benchmark.%20Nucleic%20Acids%20Res.%202018;46(20):10546%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR190">
<span class="label">190.</span><cite>Tini G, Marchetti L, Priami C, Scott-Boyer MP. Multi-omics integration-a comparison of unsupervised clustering methodologies. Brief Bioinform. 2019;20(4):1269–79.
</cite> [<a href="https://doi.org/10.1093/bib/bbx167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29272335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tini%20G,%20Marchetti%20L,%20Priami%20C,%20Scott-Boyer%20MP.%20Multi-omics%20integration-a%20comparison%20of%20unsupervised%20clustering%20methodologies.%20Brief%20Bioinform.%202019;20(4):1269%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR191">
<span class="label">191.</span><cite>Chauvel C, Novoloaca A, Veyre P, Reynier F, Becker J. Evaluation of integrative clustering methods for the analysis of multi-omics data. Brief Bioinform. 2020;21(2):541–52.
</cite> [<a href="https://doi.org/10.1093/bib/bbz015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31220206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chauvel%20C,%20Novoloaca%20A,%20Veyre%20P,%20Reynier%20F,%20Becker%20J.%20Evaluation%20of%20integrative%20clustering%20methods%20for%20the%20analysis%20of%20multi-omics%20data.%20Brief%20Bioinform.%202020;21(2):541%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR192">
<span class="label">192.</span><cite>Zouk H, Yu W, Oza A, Hawley M, Vijay Kumar PK, Koch C, et al. Reanalysis of eMERGE phase III sequence variants in 10,500 participants and infrastructure to support the automated return of knowledge updates. Genet Med. 2022;24(2):454–62.
</cite> [<a href="https://doi.org/10.1016/j.gim.2021.10.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10128874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34906510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zouk%20H,%20Yu%20W,%20Oza%20A,%20Hawley%20M,%20Vijay%20Kumar%20PK,%20Koch%20C,%20et%20al.%20Reanalysis%20of%20eMERGE%20phase%20III%20sequence%20variants%20in%2010,500%20participants%20and%20infrastructure%20to%20support%20the%20automated%20return%20of%20knowledge%20updates.%20Genet%20Med.%202022;24(2):454%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR193">
<span class="label">193.</span><cite>Manolio TA, Abramowicz M, Al-Mulla F, Anderson W, Balling R, Berger AC, et al. Global implementation of genomic medicine: we are not alone. Sci Transl Med. 2015;7(290):290ps13.
</cite> [<a href="https://doi.org/10.1126/scitranslmed.aab0194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4898888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26041702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manolio%20TA,%20Abramowicz%20M,%20Al-Mulla%20F,%20Anderson%20W,%20Balling%20R,%20Berger%20AC,%20et%20al.%20Global%20implementation%20of%20genomic%20medicine:%20we%20are%20not%20alone.%20Sci%20Transl%20Med.%202015;7(290):290ps13." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR194">
<span class="label">194.</span><cite>Wang Y, Liu L, Wang C. Trends in using deep learning algorithms in biomedical prediction systems. Front Neurosci. 2023;17:1256351.
</cite> [<a href="https://doi.org/10.3389/fnins.2023.1256351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10665494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38027475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Y,%20Liu%20L,%20Wang%20C.%20Trends%20in%20using%20deep%20learning%20algorithms%20in%20biomedical%20prediction%20systems.%20Front%20Neurosci.%202023;17:1256351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR195">
<span class="label">195.</span><cite>Adler-Milstein J, Raphael K, Bonner A, Pelton L, Fulmer T. Hospital adoption of electronic health record functions to support age-friendly care: results from a National survey. J Am Med Inf Assoc. 2020;27(8):1206–13.</cite> [<a href="https://doi.org/10.1093/jamia/ocaa129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7647349/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32772089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Adler-Milstein%20J,%20Raphael%20K,%20Bonner%20A,%20Pelton%20L,%20Fulmer%20T.%20Hospital%20adoption%20of%20electronic%20health%20record%20functions%20to%20support%20age-friendly%20care:%20results%20from%20a%20National%20survey.%20J%20Am%20Med%20Inf%20Assoc.%202020;27(8):1206%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>No datasets were generated or analysed during the current study.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Human Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s40246-025-00811-z"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/40246_2025_Article_811.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12374290/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12374290/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374290%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374290/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12374290/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12374290/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847372/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12374290/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847372/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12374290/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12374290/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="xsPW43UEM5jvSOZf6Wt55BuKeheK6wfzZx3CyuN42J9Wa7FKn23uMpzGae9CupkG">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
